<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <title>James Routley | Feed</title>
    <link
      rel="stylesheet"
      type="text/css"
      href="../styles.css"
      media="screen"
    />
  </head>
  <body>
    <a href="/index.html">Back</a>
    <a href="https://www.nature.com/articles/s41467-023-38876-w">Original</a>
    <h1>A biological camera that captures and stores images directly into DNA</h1>
    
    <div id="readability-page-1" class="page"><div>
                <section data-title="Introduction"><div id="Sec1-section"><h2 id="Sec1">Introduction</h2><div id="Sec1-content"><p>DNA is a key biomaterial that forms the basis of biological life on earth. It serves as the storage of genetic information that encodes for the multitude of proteins which fulfil various functions of life. Due to this ability to act as a storage of information, along with its simple, repeating code of 4 nucleotides, ATCG, DNA has been proposed and subsequently explored as a form of digital information storage<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 1" title="Church, G. M., Gao, Y. &amp; Kosuri, S. Next-generation digital information storage in DNA. Science 337, 1628–1628 (2012)." href="https://www.nature.com/articles/s41467-023-38876-w#ref-CR1" id="ref-link-section-d19454052e470">1</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 2" title="Goldman, N. et al. Towards practical, high-capacity, low-maintenance information storage in synthesized DNA. Nature 494, 77–80 (2013)." href="https://www.nature.com/articles/s41467-023-38876-w#ref-CR2" id="ref-link-section-d19454052e473">2</a></sup>. This is due to its extreme density (petabytes per gram<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 3" title="Zhirnov, V., Zadegan, R. M., Sandhu, G. S., Church, G. M. &amp; Hughes, W. L. Nucleic acid memory. Nat. Mater. 15, 366–370 (2016)." href="https://www.nature.com/articles/s41467-023-38876-w#ref-CR3" id="ref-link-section-d19454052e477">3</a></sup>), longevity (DNA has been retrieved from samples millions of years old<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 4" title="van der Valk, T. et al. Million-year-old DNA sheds light on the genomic history of mammoths. Nature, 1–5. 
                  https://doi.org/10.1038/s41586-021-03224-9
                  
                 (2021)." href="https://www.nature.com/articles/s41467-023-38876-w#ref-CR4" id="ref-link-section-d19454052e481">4</a></sup>), and continued relevance to study due to its underpinning of biological life. These properties have led to a recent boom in developing different workflows<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Lim, C. K., Nirantar, S., Yew, W. S. &amp; Poh, C. L. Novel Modalities in DNA Data Storage. Trends Biotechnol. 39, 990–1003 (2021)." href="#ref-CR5" id="ref-link-section-d19454052e485">5</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Tabatabaei, S. K. et al. DNA punch cards for storing data on native DNA sequences via enzymatic nicking. Nat. Commun. 11, 1–10 (2020)." href="#ref-CR6" id="ref-link-section-d19454052e485_1">6</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Chen, K., Zhu, J., Bošković, F. &amp; Keyser, U. F. Nanopore-based DNA hard drives for rewritable and secure data storage. Nano Lett. 
                  https://doi.org/10.1021/acs.nanolett.0c00755
                  
                 (2020)." href="#ref-CR7" id="ref-link-section-d19454052e485_2">7</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Tomek, K. J. et al. Driving the scalability of DNA-based information storage systems. ACS Synth. Biol. 8, 1241–1248 (2019)." href="#ref-CR8" id="ref-link-section-d19454052e485_3">8</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 9" title="Yim, S. S. et al. Robust direct digital-to-biological data storage in living cells. Nat. Chem. Biol. 17, 1–8 (2021)." href="https://www.nature.com/articles/s41467-023-38876-w#ref-CR9" id="ref-link-section-d19454052e488">9</a></sup> for converting digital data into DNA and vice versa, which has been increasingly relevant and urgent due to the impending shortage of silica necessary for manufacturing storage devices required to accommodate our projected data storage requirements<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 3" title="Zhirnov, V., Zadegan, R. M., Sandhu, G. S., Church, G. M. &amp; Hughes, W. L. Nucleic acid memory. Nat. Mater. 15, 366–370 (2016)." href="https://www.nature.com/articles/s41467-023-38876-w#ref-CR3" id="ref-link-section-d19454052e492">3</a></sup>.</p><p>Current DNA storage workflows largely rely on in vitro synthesis of DNA strands, which are costly and require complex instrumentation<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Ceze, L., Nivala, J. &amp; Strauss, K. Molecular digital data storage using DNA. Nat. Rev. Genet. 20, 456–466 (2019)." href="#ref-CR10" id="ref-link-section-d19454052e499">10</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Lee, H. H., Kalhor, R., Goela, N., Bolot, J. &amp; Church, G. M. Terminator-free template-independent enzymatic DNA synthesis for digital information storage. Nat. Commun. 10, 1–12 (2019)." href="#ref-CR11" id="ref-link-section-d19454052e499_1">11</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 12" title="Takahashi, C. N., Nguyen, B. H., Strauss, K. &amp; Ceze, L. Demonstration of end-to-end automation of DNA Data Storage. Sci. Rep. 9, 1–5 (2019)." href="https://www.nature.com/articles/s41467-023-38876-w#ref-CR12" id="ref-link-section-d19454052e502">12</a></sup>. Errors in the synthesis process are also common. While there have been substantial advances in accelerating this process by developing enzymatic synthesis methods<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 13" title="Palluk, S. et al. De novo DNA synthesis using polymerase-nucleotide conjugates. Nat. Biotechnol. 36, 645–650 (2018)." href="https://www.nature.com/articles/s41467-023-38876-w#ref-CR13" id="ref-link-section-d19454052e506">13</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 14" title="Lu, X. et al. Enzymatic DNA synthesis by engineering terminal deoxynucleotidyl transferase. ACS Catal. 2988–2997. 
                  https://doi.org/10.1021/acscatal.1c04879
                  
                 (2022)." href="https://www.nature.com/articles/s41467-023-38876-w#ref-CR14" id="ref-link-section-d19454052e509">14</a></sup>, miniaturizing electrochemical synthesis<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Xu, C. et al. Electrochemical DNA synthesis and sequencing on a single electrode with scalability for integrated data storage. Sci. Adv. 7, eabk0100 (2021)." href="#ref-CR15" id="ref-link-section-d19454052e513">15</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Nguyen, B. H. et al. Scaling DNA data storage with nanoscale electrode wells. Sci. Adv. 7, eabi6714 (2021)." href="#ref-CR16" id="ref-link-section-d19454052e513_1">16</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 17" title="Antkowiak, P. L. et al. Low cost DNA data storage using photolithographic synthesis and advanced information reconstruction and error correction. Nat. Commun. 11, 5345 (2020)." href="https://www.nature.com/articles/s41467-023-38876-w#ref-CR17" id="ref-link-section-d19454052e516">17</a></sup>, or developing more robust encoding methods<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Erlich, Y. &amp; Zielinski, D. DNA Fountain enables a robust and efficient storage architecture. Science 355, 950–954 (2017)." href="#ref-CR18" id="ref-link-section-d19454052e520">18</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Wang, Y. et al. High capacity DNA data storage with variable-length Oligonucleotides using repeat accumulate code and hybrid mapping. J. Biol. Eng. 13, 89 (2019)." href="#ref-CR19" id="ref-link-section-d19454052e520_1">19</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 20" title="Anavy, L., Vaknin, I., Atar, O., Amit, R. &amp; Yakhini, Z. Data storage in DNA with fewer synthesis cycles using composite DNA letters. Nat. Biotechnol. 37, 1229–1236 (2019)." href="https://www.nature.com/articles/s41467-023-38876-w#ref-CR20" id="ref-link-section-d19454052e523">20</a></sup>, DNA synthesis remains a bottleneck in the adoption of DNA as a data storage medium. There is thus significant interest in developing ways of encoding information into DNA that can either supersede or circumvent DNA synthesis in its current form.</p><p>The abundance of DNA in living cells have been considered as a potential source of DNA that can be encoded with information via the use of molecular biology tools and biological systems. Utilizing living cells as a way to incorporate and record external signals such as the presence of chemicals or electrical inputs into DNA have also been demonstrated, with recording systems ranging from recombinases<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Sheets, M. B., Wong, W. W. &amp; Dunlop, M. J. Light-inducible recombinases for bacterial optogenetics. ACS Synth. Biol. 9, 227–235 (2020)." href="#ref-CR21" id="ref-link-section-d19454052e530">21</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Yang, L. et al. Permanent genetic memory with &gt;1-byte capacity. Nat. Methods 11, 1261–1266 (2014)." href="#ref-CR22" id="ref-link-section-d19454052e530_1">22</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 23" title="Bonnet, J., Subsoontorn, P. &amp; Endy, D. Rewritable digital data storage in live cells via engineered control of recombination directionality. Proc. Natl Acad. Sci. USA 109, 8884–8889 (2012)." href="https://www.nature.com/articles/s41467-023-38876-w#ref-CR23" id="ref-link-section-d19454052e533">23</a></sup> to CRISPR-based modifications<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Shipman, S. L., Nivala, J., Macklis, J. D. &amp; Church, G. M. Molecular recordings by directed CRISPR spacer acquisition. Science 353, aaf1175 (2016)." href="#ref-CR24" id="ref-link-section-d19454052e537">24</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Shipman, S. L., Nivala, J., Macklis, J. D. &amp; Church, G. M. CRISPR–Cas encoding of a digital movie into the genomes of a population of living bacteria. Nature 547, 345–349 (2017)." href="#ref-CR25" id="ref-link-section-d19454052e537_1">25</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 26" title="Tang, W. &amp; Liu, D. R. Rewritable multi-event analog recording in bacterial and mammalian cells. Science 360, eaap8992 (2018)." href="https://www.nature.com/articles/s41467-023-38876-w#ref-CR26" id="ref-link-section-d19454052e540">26</a></sup>. Most notably, recent work has showcased the storage of 72 bits of information directly into the genomic DNA of living cells via external electrical input signals<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 9" title="Yim, S. S. et al. Robust direct digital-to-biological data storage in living cells. Nat. Chem. Biol. 17, 1–8 (2021)." href="https://www.nature.com/articles/s41467-023-38876-w#ref-CR9" id="ref-link-section-d19454052e544">9</a></sup>, as well as the use of DNA structural barcodes that bind to the M13mp18 bacteriophage DNA scaffold to encode information<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Chen, K., Zhu, J., Bošković, F. &amp; Keyser, U. F. Nanopore-based DNA hard drives for rewritable and secure data storage. Nano Lett. 
                  https://doi.org/10.1021/acs.nanolett.0c00755
                  
                 (2020)." href="https://www.nature.com/articles/s41467-023-38876-w#ref-CR7" id="ref-link-section-d19454052e548">7</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 27" title="Chen, K. et al. Digital data storage using DNA nanostructures and solid-state nanopores. Nano Lett. 19, 1210–1215 (2019)." href="https://www.nature.com/articles/s41467-023-38876-w#ref-CR27" id="ref-link-section-d19454052e551">27</a></sup>. However, such systems have not been able to accurately capture spatial information, due to the lack of mechanisms that can simultaneously encode spatial information as well as external input signals. Furthermore, existing encoding systems also tend to utilize low-throughput chemical processes, which severely limits scalability.</p><p>In this paper, we present a workflow that allows the direct capture of both spatial information and input signals via light into DNA itself as a means to store digital information such as images into DNA. We utilize a blue light-responsive recombinase system that responds to the presence or absence of blue light as an external signal, and subsequently records that response into DNA itself via site-specific DNA editing. To enable the encoding of spatial information along with the recorded signal, we implemented a barcoding scheme to enable differentiation of individual wells containing cells with records of light exposure, thereby ‘digitizing’ the recorded image and allowing for deconvolution upon retrieval of the DNA sequences by sequencing. As a result, this recombinase-based DNA recorder, coupled with the aforementioned barcoding scheme that can preserve spatial information, enables the direct capture of both spatial information as well as input signals via light into DNA itself. While previous work has demonstrated the usage of light-responsive systems that can capture light and display this input as a corresponding light output, akin to an analogue camera<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 28" title="Fernandez-Rodriguez, J., Moser, F., Song, M. &amp; Voigt, C. A. Engineering RGB color vision into Escherichia coli. Nat. Chem. Biol. 13, 706–708 (2017)." href="https://www.nature.com/articles/s41467-023-38876-w#ref-CR28" id="ref-link-section-d19454052e558">28</a></sup>, our process instead creates a biological analogue to a digital camera which we term as ‘BacCam’. As the DNA encoded with different images can be pooled and stored together, we further characterize this workflow by demonstrating random access and multiplexing, and utilize outlier detection and reassignment techniques, along with unsupervised clustering algorithms from the field of machine learning to accurately reconstruct the stored data. Taking advantage of the multiplexing capabilities offered by light, we further layer an orthogonal red light system with the blue light system to allow encoding of 2 images simultaneously, increasing the scalability and density of the workflow and enabling multicolor image capture. Utilizing molecular biology methods, optogenetics, barcoding techniques, and biological systems, we thus provide a framework for the integration of biological and digital interfaces.</p></div></div></section><section data-title="Results"><div id="Sec2-section"><h2 id="Sec2">Results</h2><div id="Sec2-content"><h3 id="Sec3">BacCam enables the direct-to-DNA storage and retrieval of images</h3><p>Light has the distinct advantages of being cheap, massively parallelizable, rapid, highly programmable, and easily multiplexed, with little effort or cost needed to scale or generate patterns of increasing complexity. There is therefore an interest in utilizing light as a patterning mechanism in biology, such as in photolithographic DNA synthesis<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 17" title="Antkowiak, P. L. et al. Low cost DNA data storage using photolithographic synthesis and advanced information reconstruction and error correction. Nat. Commun. 11, 5345 (2020)." href="https://www.nature.com/articles/s41467-023-38876-w#ref-CR17" id="ref-link-section-d19454052e574">17</a></sup>, as well as optogenetic circuits for multiple purposes such as bioproduction or controlling cellular behavior on a spatiotemporal level<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Jayaraman, P. et al. Blue light-mediated transcriptional activation and repression of gene expression in bacteria. Nucleic Acids Res. 44, 6994–7005 (2016)." href="#ref-CR29" id="ref-link-section-d19454052e578">29</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Pouzet, S. et al. The promise of optogenetics for bioproduction: dynamic control strategies and scale-up instruments. Bioengineering 7, 151 (2020)." href="#ref-CR30" id="ref-link-section-d19454052e578_1">30</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 31" title="Mansouri, M. &amp; Fussenegger, M. Synthetic biology-based optogenetic approaches to control therapeutic designer cells. Curr. Opin. Syst. Biol. 28, 100396 (2021)." href="https://www.nature.com/articles/s41467-023-38876-w#ref-CR31" id="ref-link-section-d19454052e581">31</a></sup>. We therefore sought a way to utilize light as an input to encode information into DNA. We hypothesized that cells containing optogenetic circuits that can record the presence or absence of light within DNA can be perceived as analogous to a digital camera that captures images via light exposure and records said exposure in a digital format. We thus utilized an Opto-Cre-Vvd recombinase system, whereby a Cre-Lox recombinase protein was engineered to be light inducible by splitting the recombinase and attaching photodimers that bring the split protein together upon exposure to blue light<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 21" title="Sheets, M. B., Wong, W. W. &amp; Dunlop, M. J. Light-inducible recombinases for bacterial optogenetics. ACS Synth. Biol. 9, 227–235 (2020)." href="https://www.nature.com/articles/s41467-023-38876-w#ref-CR21" id="ref-link-section-d19454052e585">21</a></sup>. The recombinase, upon activation, excises a predefined section of DNA that are flanked by LoxP sites, resulting in an alteration in the sequence. We posited that this alteration is analogous to the encoding of a bit. To determine whether a bit has been encoded, upon sequencing, the total number of reads that possess the excised LoxP sequence is compared to the total number of reads with the full LoxP sequence. A high ratio of excised reads to unexcised ones would correspond to a bit state of ‘1’ being encoded, while the opposite would correspond to a bit state of ‘0’.</p><p>We then posited that a 96-well plate, with each well being appropriately barcoded, and containing cells with the Opto-Cre-Vvd and LoxP genetic circuits, would be analogous to a digital camera with image sensors each containing individual pixels that have their own unique identifier, storing information corresponding to light exposure. To scale the one-bit digital drive within each cell to that of a multi-pixel image, a population of <i>Escherichia</i> <i>coli</i> (<i>E. coli</i>) bacteria containing the above-mentioned circuit was spatially separated in individual wells of a black, clear-bottomed 96-well plate, upon which a predefined pattern of 465 nm wavelength light was projected from below.</p><p>The projected pattern is then preserved by the addition of unique oligonucleotide barcodes, which we term as ‘well-codes’, that bind at the region preceding the LoxP recording site. The nucleotide sequence of the well-code is mapped to predefined spatial locations, with said mappings saved in a separate table, thereby resulting in the linkage of the spatial location of the recording bacteria in each isolated well along with the recorded digital drive that each bacteria possesses. The addition of the well-code to the recording sequence is conducted via PCR. Further oligonucleotide sequences were then appended via PCR to prepare the individual samples for Next-Generation Sequencing (NGS). Subsequently, all samples are then pooled together for storage, which obviates the need for preserving the original well configuration as the unique well-codes have tied this configuration with the recorded bit state. This results in a pool of DNA oligonucleotide sequences that stores the information of the image that was captured (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41467-023-38876-w#Fig1">1</a>). In practice, this barcoding system enables the capture of 2-dimensional images, where each pixel’s location is represented by a well-code, and the bit state of the pixel is determined by the presence or absence of the DNA excision site. This resulting data can be stored robustly in dried form on a benchtop at room temperature, or frozen in a −20 °C fridge.</p><div data-test="figure" data-container-section="figure" id="figure-1" data-title="Workflow of BacCam."><figure><figcaption><b id="Fig1" data-test="figure-caption-text">Fig. 1: Workflow of BacCam.</b></figcaption><div><div><a data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="https://www.nature.com/articles/s41467-023-38876-w/figures/1" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41467-023-38876-w/MediaObjects/41467_2023_38876_Fig1_HTML.png?as=webp"/><img aria-describedby="Fig1" src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41467-023-38876-w/MediaObjects/41467_2023_38876_Fig1_HTML.png" alt="figure 1" loading="lazy" width="685" height="427"/></picture></a></div><p><b>A</b> Encoding of light into bacteria and barcoding for preservation of spatial information. <b>B</b> Sequencing and decoding of stored DNA back into original image. <b>C</b> Initial proof of concept of BacCam using the ‘BACCAM’ pattern. Successful reconstruction of the image was done with 93/96 accuracy. Source data are provided as a Source Data file.</p></div></figure></div><p>To retrieve the image, sequencing of the pooled DNA was then conducted. The resulting sequencing data generated was then deconvoluted by determining the total number of sequences retrieved for each barcode, and the bit state determined by the ratio of sequences lacking the region between the LoxP sites to sequences containing the region. These ratios indicate the proportion of DNA that were excised compared to ones that were intact. High ratios would thus indicate that a larger proportion of DNA was excised, which would then imply that light exposure was ‘recorded’ and so recording a ‘1’ signal, and vice versa for lower ratios, which would then record a ‘0’ signal. The threshold determining the ratio that demarcates assignment of a ‘1’ or a ‘0’ signal was derived using clustering methods that compare ratios across each plate. This deconvolution was done for all well-codes used, and the resulting bit-well-code pairs were then reassembled back, according to a previously stored mapping table of well-codes to spatial locations, to form a digitized version of the projected image.</p><p>Utilizing the above-mentioned process, we have successfully demonstrated the viability of various aspects of the workflow detailed above. We demonstrated the recording of a predefined pattern into DNA and retrieved it via sequencing (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41467-023-38876-w#Fig1">1</a>). Deconvolution of the sequenced reads was done by mapping the well-code back to its spatial location, counting the number of excised and intact LoxP reads corresponding to each well-code, and determining the bit state of each well via the above-mentioned ratio of excised to intact LoxP reads.</p><h3 id="Sec4">Multiple images can be stored in a complex pool and retrieved with high accuracy</h3><p>We subsequently tested the multiplexing capability of our workflow, generating multiple images and pooling them together, and deconvoluting each image from the DNA sequencing data generated via sequencing of said pool. We hypothesized that each full image can also be barcoded with its own ‘meta-barcode’ that separates images from one another in a heterogenous DNA pool of multiple images, despite using the same well-codes that separate individual wells, allowing for multiplexing. We thus incorporated this second layer of indexing using indexing barcodes provided by Illumina. Each 96 well plate, after the addition of the initial well-codes, has the same 2 indexing barcodes appended on the 5′ and 3′ end of the sequence in each well. These barcodes are combinatorial in nature, thus increasing the possible number of total images that can be stored together.</p><p>To determine if this second indexing layer can be used to deconvolute multiple images from the same sample, we exposed 5 unique patterns to 5 different 96-well plates, thereby creating a set of 5 images (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41467-023-38876-w#Fig2">2A</a>). 4 of these patterns (NUS, SYNCTI, BacCam, and Smiley) illustrate the ability of the system to capture images. The last pattern, ‘Heloo wo{|d!’ serves to illustrate the ability of BacCam to encode information (such as letters and symbols represented in 8-bit ASCII code format) by allocating each well as a bit and encoding the information in a 96-well format. Hence, each column (8 wells) will encode one ASCII code. In this case, BacCam can also serve to encode any sort of information, as long as it is projected with the appropriate pattern.</p><div data-test="figure" data-container-section="figure" id="figure-2" data-title="Storage and retrieval of multiple images in a single complex pool."><figure><figcaption><b id="Fig2" data-test="figure-caption-text">Fig. 2: Storage and retrieval of multiple images in a single complex pool.</b></figcaption><div><div><a data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="https://www.nature.com/articles/s41467-023-38876-w/figures/2" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41467-023-38876-w/MediaObjects/41467_2023_38876_Fig2_HTML.png?as=webp"/><img aria-describedby="Fig2" src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41467-023-38876-w/MediaObjects/41467_2023_38876_Fig2_HTML.png" alt="figure 2" loading="lazy" width="685" height="553"/></picture></a></div><p>Red boxes indicate wells with allocated bits that do not correspond with the projected image. <b>A</b> Projected patterns that are stored with the BacCam process. Images are projected on different plates, barcoded individually with index sequences corresponding to each image, and resulting products are pooled together into a single tube. <b>B</b> Retrieved images from a single sequencing run of a mixed pool, deconvoluted with the corresponding indexes of each image. <b>C</b> Retrieved images from multiple sequencing runs, whereby random access of each image was conducted from the mixed pool by using specific primers to amplify desired images before sequencing. <b>D</b> Reconstruction of selectively amplified single images from mixed libraries of increasing dilutions, demonstrating robustness of the amplification technique in selecting specific images from an ever-decreasing initial amount of DNA. Source data are provided as a Source Data file.</p></div></figure></div><p>We then added the same set of well-codes to each of them via PCR, before extending each image with indexing barcodes. Each image used a different combination of indexing barcodes, which were then saved in a mapping table that links the unique index sequences with its corresponding image. These indexed images were then pooled together into the same pool of DNA. We then determined if a mixed, heterogenous pool of DNA can be sequenced as a whole library, with the individual images deconvoluted from the total raw reads that are produced by sequencing. We demonstrate that a mixed pool of at least 5 different images can be deconvoluted and reconstructed with an accuracy of at least 90% for all images, showcasing the multiplexing capability of our workflow (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41467-023-38876-w#Fig2">2B</a>).</p><p>We subsequently tested the ability of our workflow to implement random access of images, whereby each image can be selectively amplified and decoded without the need to sequence the entire mixed DNA pool. We hypothesized that designing primers that selectively bind to the indexed sequences corresponding to the desired image would be sufficient for random access, adequately amplifying each individual ‘pixel’ associated with the image, while avoiding amplification of sequences that do not possess the selected index. To test the hypothesis, we demonstrated that all images (with accuracy &gt; 80%) in a mixed pool of 5 can be selectively amplified and accessed by using the corresponding indexing primers and conducting a touchdown PCR to ensure precise binding and amplification of desired sequences<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 32" title="Korbie, D. J. &amp; Mattick, J. S. Touchdown PCR for increased specificity and sensitivity in PCR amplification. Nat. Protoc. 3, 1452–1456 (2008)." href="https://www.nature.com/articles/s41467-023-38876-w#ref-CR32" id="ref-link-section-d19454052e700">32</a></sup> (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41467-023-38876-w#Fig2">2C</a>), thus obviating the need to sequence the entire pool of DNA. Taken together, these results show that multiple images can be easily tagged, stored together and subsequently demultiplexed by using a simple indexing process that complements existing NGS workflows.</p><h3 id="Sec5">Information can be accurately reconstructed from small initial quantity of samples</h3><p>To determine the minimum amount of DNA required to reconstruct images accurately, we conducted a series of dilution experiments. We diluted a mixed pool of samples progressively, with each dilution being tenfold less than the previous. Sequencing of each dilution for a particular image for deconvolution and reconstruction demonstrated that images were accurately reconstructed at dilutions of a hundred-fold from the initial concentration (2.66 nM). At dilutions of a 1000-fold (2.66 pM), individual reads for each barcode numbered in the low single digits, with many having no reads at all. As a result, the fidelity of the image drops significantly (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41467-023-38876-w#Fig3">3A</a>). For large scale multiplexing of images, the concentration of DNA representing a single image drops as the number of images in each pool increases. This dilution assay thus provides us with an approximate proxy for the number of images that can be concurrently retrieved in a sequencing run, assuming a fixed volume of the pooled library was sequenced, and sequencing coverage remained the same. The results imply that the number of different images that can be stored in a pool and retrieved in a single run is between 100 and 1000. Increasing the number of distinct images that can be retrieved this way would therefore require an increase in sequencing coverage and/or increasing concentrations of each image.</p><div data-test="figure" data-container-section="figure" id="figure-3" data-title="Testing the boundaries of BacCam."><figure><figcaption><b id="Fig3" data-test="figure-caption-text">Fig. 3: Testing the boundaries of BacCam.</b></figcaption><div><div><a data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="https://www.nature.com/articles/s41467-023-38876-w/figures/3" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41467-023-38876-w/MediaObjects/41467_2023_38876_Fig3_HTML.png?as=webp"/><img aria-describedby="Fig3" src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41467-023-38876-w/MediaObjects/41467_2023_38876_Fig3_HTML.png" alt="figure 3" loading="lazy" width="685" height="237"/></picture></a></div><p><b>A</b> Dilution experiments demonstrating the theoretical maximum information capacity of BacCam. Images can be successfully retrieved up to a dilution of 100× from the original fully amplified pool, with 1000x dilution having significantly higher number of errors due to the low number of reads. <b>B</b> No loss in information for ‘Smiley’ stored in both liquid and dried form. <b>C</b> No loss of information in ‘Heloo wo{|d!’ stored in varying conditions for extended periods of time. Source data are provided as a Source Data file.</p></div></figure></div><p>However, we also hypothesized that PCR amplification of diluted reads via random access might be sufficient to overcome the above-mentioned limit without an increase in sequencing coverage, saving costs while still allowing for multiplexed storage in the same pool. We therefore conducted serial dilutions on the mixed pool of images and utilized indexing primers for random access of selected images via PCR, amplifying the diluted DNA and sending it for sequencing. We demonstrated that the NUS image was selectively amplified with the N701 and N501 indexes (index sequences in Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41467-023-38876-w#MOESM1">1</a>), from a 50× dilution of an initial concentration of 17.98 nM to 0.36 nM, up to a dilution of 50,000× (0.36 pM), showcasing the robustness of image retrieval from a complex pool of 5 images, and demonstrating consistent re-amplification of images from a small quantity of initial samples (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41467-023-38876-w#Fig2">2D</a>). Attempts to retrieve the other images contained in the mixed library after selective amplification were unsuccessful, demonstrating the capabilities of targeted amplification of selective images. It is also apparent that at lower dilutions, there is an increasing skew in terms of the ratio values—suggesting that amplification from minute amounts of sample tends to dramatically increase the contrast of reads that are excised compared to unexcised ones.</p><h3 id="Sec6">Images stored are durable in varying environmental conditions</h3><p>We further explored the limits of BacCam by subjecting it to a battery of physical challenges. One way to improve the density and stability of DNA is to dry it, due to the hydrolytic activity of water molecules on the phosphate backbone. This also has the effect of reducing volume, resulting in a higher overall density. To test the viability of our workflow for drying, we spun dried a volume of DNA corresponding to the ‘Smiley’ image before rehydration and sequencing. We showed that images in dried form could be successfully retrieved with zero loss in accuracy compared to liquid, frozen images (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41467-023-38876-w#Fig3">3B</a>).</p><p>We also subjected a DNA sample with the ‘Heloo wo{|d!’ image to accelerated aging experiments. This was done by comparing identical samples, both in aqueous form with one at room temperature (RT) and another that was kept in a 60 °C oven, both for a duration of 1 week. We also exposed a freshly thawed frozen sample to UV light for 1 hr. It was previously demonstrated that data encoded in dried DNA stored at a temperature of 70 °C for a duration of 7 days was not able to be retrieved, and that duration was equivalent to storing DNA at 9.4 °C for 2000 years<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 33" title="Grass, R. N., Heckel, R., Puddu, M., Paunescu, D. &amp; Stark, W. J. Robust chemical preservation of digital information on DNA in silica with error-correcting codes. Angew. Chem. Int. Ed. 54, 2552–2555 (2015)." href="https://www.nature.com/articles/s41467-023-38876-w#ref-CR33" id="ref-link-section-d19454052e769">33</a></sup>. We demonstrated a successful retrieval of said image from all samples, with no loss of information despite varying conditions, showcasing the inherent robustness and redundancy of the workflow (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41467-023-38876-w#Fig3">3C</a>).</p><h3 id="Sec7">Optimization of image deconvolution with computational methods</h3><p>Upon receipt of sequencing data, proportions of excised and unexcised DNA was calculated and allocated to appropriate wells. To determine the correct threshold that will allow for accurate separation of recorded light signals, we utilized a manual thresholding process based on foreknowledge of the encoded image. This method was used to assess the accuracy of the overall workflow in storing a projected image into DNA. While this method suffices due to foreknowledge of the pattern encoded, to enable a true information encoding and retrieval system, a method that can reconstruct a potential image solely from the sequenced data would be necessary. We theorized that computational clustering methods such as unsupervised machine learning techniques could be utilized to automatically identify the distinct groups/clusters. To preprocess the sequencing data before implementing the clustering methods, we applied an outlier detection technique also known as unsupervised anomaly detection to detect the outliers located at the low-density regions<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 34" title="Breunig, M. M., Kriegel, H.-P., Ng, R. T. &amp; Sander, J. LOF: identifying density-based local outliers. ACM SIGMOD Rec. 29, 93–104 (2000)." href="https://www.nature.com/articles/s41467-023-38876-w#ref-CR34" id="ref-link-section-d19454052e785">34</a></sup>. We proceeded to reassign the outliers with the nearest values detected from the inliers. This outlier detection and reassignment data preprocessing demonstrated improvement in removing small low-density clusters and enhancing the edges among larger clusters. Post-processing techniques were also implemented to satisfy the edge cases of having fully ‘ON’ or ‘OFF’ images and to perform cluster grouping to retrieve these final binary state images. The entire automated workflow for image deconvolution is outlined in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41467-023-38876-w#Fig4">4A</a>.</p><div data-test="figure" data-container-section="figure" id="figure-4" data-title="Developing a machine learning clustering-based workflow for automated image deconvolution."><figure><figcaption><b id="Fig4" data-test="figure-caption-text">Fig. 4: Developing a machine learning clustering-based workflow for automated image deconvolution.</b></figcaption><div><div><a data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="https://www.nature.com/articles/s41467-023-38876-w/figures/4" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41467-023-38876-w/MediaObjects/41467_2023_38876_Fig4_HTML.png?as=webp"/><img aria-describedby="Fig4" src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41467-023-38876-w/MediaObjects/41467_2023_38876_Fig4_HTML.png" alt="figure 4" loading="lazy" width="685" height="487"/></picture></a></div><p><b>A</b> An automated workflow incorporating outlier detection and reassignment method, machine learning unsupervised clustering technique, full ON-OFF criteria assessment, and cluster grouping for automated image deconvolution. <b>B</b> An example implemented using GMM to demonstrate the results acquired at different phases. The dots in the graphs represent the raw data or the curated data from the 96 wells. The ‘1’ and ‘−1’ obtained from the Local Outlier Factor denote the inliers and outliers detected respectively from the raw data. The curated data were acquired after reassigning the outliers to the nearest values of inliers. The M0–M2 indicate the computed mean values of the individual clusters for full ON–OFF criteria assessment. The deconvoluted image was then plotted based on the three clusters and after clusters grouping into the final binary state image (‘0’: light blue; ‘1’: dark blue) with 3 error bits (orange). <b>C</b> Validation of the automated workflow on other patterns including full ‘ON’ and ‘OFF’ datasets.</p></div></figure></div><p>We tested multiple clustering methods under different parameter settings on the existing datasets, and compared the accuracy of the automated clustering methods with manual thresholding based on foreknowledge of the existing image (Table <a data-track="click" data-track-label="link" data-track-action="table anchor" href="https://www.nature.com/articles/s41467-023-38876-w#Tab1">1</a>). The results showed that the OPTICS clustering method and 3-component Gaussian Mixture Model (GMM) produced clusters that most closely emulated manual thresholds, with near perfect accuracy (Supplementary <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41467-023-38876-w#MOESM1">S2</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41467-023-38876-w#MOESM1">S3</a>). We tested the results acquired using GMM at different phases of the workflow (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41467-023-38876-w#Fig4">4B</a>), showing how each phase contributes to building an accurate end result, and demonstrated the robustness of the automated workflow by showcasing all the successfully deconvoluted images with high accuracy (&gt;0.9) (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41467-023-38876-w#Fig4">4C</a>, Supplementary <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41467-023-38876-w#MOESM1">S2</a>).</p><div data-test="inline-table" data-container-section="table" id="table-1"><figure><figcaption><b id="Tab1" data-test="table-caption">Table 1 Comparison of methods used for automated thresholding</b></figcaption></figure></div><h3 id="Sec8">Multiplexing with multiple colors of light</h3><p>One significant advantage of using light is the ability to multiplex in a simple fashion with the addition of different light wavelengths. We redesigned a red light sensitive Cre recombinase (pBbS5a-RLCre) initially developed in yeast systems to add the red wavelength to the existing blue light BacCam workflow<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 35" title="Hochrein, L., Mitchell, L. A., Schulz, K., Messerschmidt, K. &amp; Mueller-Roeber, B. L-SCRaMbLE as a tool for light-controlled Cre-mediated recombination in yeast. Nat. Commun. 9, 1931 (2018)." href="https://www.nature.com/articles/s41467-023-38876-w#ref-CR35" id="ref-link-section-d19454052e1307">35</a></sup>. To allow for differentiation between the two light systems in the workflow, we designed another recording plasmid (pBbAW4k-Spacer1Barcoding-loxP-TT-loxP-ho1-pcyA) that contained another intermediate barcode, thus differentiating edits caused by red light exposure to those caused by blue light exposure (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41467-023-38876-w#Fig5">5A</a>, Supplementary <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41467-023-38876-w#MOESM1">S1</a>). We also designed a programmable light illumination device (OptoBox, Supplementary <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41467-023-38876-w#MOESM1">S8</a>) to project multiple colors of light in the same well. We show that a co-culture of the blue light and red light-responsive bacteria in the same well is able to sense multiple wavelengths of light at the same time, and encode 2 separate images simultaneously. Images were encoded in two separate ways. One way involved the overnight projection of a blue light pattern along with a different red light pattern in an alternating fashion, with each pattern being projected for 10 min before switching. Another way involved projecting both images simultaneously (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41467-023-38876-w#Fig5">5B</a>). We demonstrate the successful encoding and retrieval of two different images with different wavelengths of light, with both methods of projection being viable approaches, with a minimum accuracy of more than 90% for each image encoded (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41467-023-38876-w#Fig5">5C</a>, Supplementary <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41467-023-38876-w#MOESM1">S5</a>–<a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41467-023-38876-w#MOESM1">S7</a>). These results demonstrate high orthogonality of both light systems (Supplementary <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41467-023-38876-w#MOESM1">S9</a>).</p><div data-test="figure" data-container-section="figure" id="figure-5" data-title="Multiplexing the BacCam workflow using multiple colors of light."><figure><figcaption><b id="Fig5" data-test="figure-caption-text">Fig. 5: Multiplexing the BacCam workflow using multiple colors of light.</b></figcaption><div><div><a data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="https://www.nature.com/articles/s41467-023-38876-w/figures/5" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41467-023-38876-w/MediaObjects/41467_2023_38876_Fig5_HTML.png?as=webp"/><img aria-describedby="Fig5" src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41467-023-38876-w/MediaObjects/41467_2023_38876_Fig5_HTML.png" alt="figure 5" loading="lazy" width="685" height="1289"/></picture></a></div><p><b>A</b> Design of a co-culture of engineered <i>E. coli</i> with blue and red light sensitive recombinases respectively. The blue light Opto-Cre-Vvd system and the redesigned red light recombinase recording system were separately transformed into <i>E. coli</i>. <b>B</b> Experimental workflow of the co-culture. Both resulting strains of <i>E. coli</i> were co-cultured together on the same plate, and exposed to either alternating blue and red light projections, or simultaneous projections of both blue and red light. <b>C</b> Images obtained after sequencing and deconvolution, separating red and blue with the red light specific barcode and subsequently subjecting it to our deconvolution algorithm workflow based on 3-component Gaussian Mixture Model (GMM). Source data are provided as a Source Data file.</p></div></figure></div></div></div></section><section data-title="Discussion"><div id="Sec9-section"><h2 id="Sec9">Discussion</h2><div id="Sec9-content"><p>In this work, we have proposed and demonstrated an alternative workflow for DNA data storage—a direct capture of images into DNA that is analogous to the creation of a digital camera. We demonstrated scaling of this workflow using a single color light (blue) wavelength to five 96 bit images, totaling 60 bytes, and showed that the theoretical limits of this system potentially scales to more than one hundred 96 bit images in a single heterogenous pool. We have also shown the ability for random access of individual images, even from a highly dilute concentration of 50,000x lesser than the initial concentration, and the reliable recovery of data from diluted, heat treated, UV-exposed and dried pools. We have implemented computational pipelines that serve to deconvolute the encoded information and correct for errors in a reliable fashion. To scale this workflow beyond a single wavelength of light, we incorporated an additional wavelength of light, doubling the amount of data that can be stored in a single, simultaneous capture and demonstrating the multiplexing potential of the system. This system further expands the boundaries of the field outside of preexisting DNA synthesis and sequencing workflows.</p><p>We have utilized living cells as the encoder for DNA data storage in this work. This offers several advantages over de novo in vitro DNA synthesis-based data storage formats. Living cells are an ever-renewing source of DNA, making it cost effective to produce at scale. Cells also possess a multitude of systems that can interface with DNA, such as transcription factor binding proteins, and are also able to respond to a multitude of stimuli ranging from chemical to light and electrical means that can also be recorded into DNA in vivo. Furthermore, in BacCam, the writing step is done in parallel, as each bit is encoded at the same time. As such, latency of writing is greatly reduced.</p><p>While living cells have been previously used as encoders for storing images, BacCam differs primarily from existing work by combining spatial addressability along with optogenetic systems to digitally encode information directly into DNA. This is in contrast to previous work such as that of Shipman et al.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 24" title="Shipman, S. L., Nivala, J., Macklis, J. D. &amp; Church, G. M. Molecular recordings by directed CRISPR spacer acquisition. Science 353, aaf1175 (2016)." href="https://www.nature.com/articles/s41467-023-38876-w#ref-CR24" id="ref-link-section-d19454052e1388">24</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 25" title="Shipman, S. L., Nivala, J., Macklis, J. D. &amp; Church, G. M. CRISPR–Cas encoding of a digital movie into the genomes of a population of living bacteria. Nature 547, 345–349 (2017)." href="https://www.nature.com/articles/s41467-023-38876-w#ref-CR25" id="ref-link-section-d19454052e1391">25</a></sup>, in which image information was encoded via de novo DNA synthesis before insertion into the bacterial genome. Furthermore, the multiplexing ability of light allows us to utilize different wavelengths of light to encode additional layers of information, providing significant flexibility in encoding and offering potential avenues for capacity expansion.</p><p>The resolution of images encoded is a function of the number of unique well-codes generated, as well as the physical separation and isolation of individually addressable bacterial populations. As a proof of concept, we have thus far utilized 96-well plates for convenience and cost. However, we envision that subsequent implementations of the workflow can be further miniaturized and scaled up to 384, 1536 or even higher resolution systems, leveraging advances in liquid handling devices, microchips<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 36" title="Chen, B. et al. High-throughput analysis and protein engineering using microcapillary arrays. Nat. Chem. Biol. 12, 76–81 (2016)." href="https://www.nature.com/articles/s41467-023-38876-w#ref-CR36" id="ref-link-section-d19454052e1398">36</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 37" title="Zhang, Y. et al. Accurate high-throughput screening based on digital protein synthesis in a massively parallel femtoliter droplet array. Sci. Adv. 5, eaav8185 (2019)." href="https://www.nature.com/articles/s41467-023-38876-w#ref-CR37" id="ref-link-section-d19454052e1401">37</a></sup>, as well as microfluidic setups<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 38" title="Antkowiak, P. L. et al. Integrating DNA encapsulates and digital microfluidics for automated data storage in DNA. Small 18, 2107381 (2022)." href="https://www.nature.com/articles/s41467-023-38876-w#ref-CR38" id="ref-link-section-d19454052e1405">38</a></sup>. The robustness of PCR amplification in retrieving information from a complex pool of oligonucleotides has been demonstrated earlier, and indicates that there is still significant potential capacity for data storage. Implementation of various PCR or DNA assembly methods that do not require thermocycling for appending well-codes and indexing barcodes can also increase the throughput of the process while reducing the footprint and complexity of the workflow<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 39" title="Lin, K. N., Volkel, K., Tuck, J. M. &amp; Keung, A. J. Dynamic and scalable DNA-based information storage. Nat. Commun. 11, 2981 (2020)." href="https://www.nature.com/articles/s41467-023-38876-w#ref-CR39" id="ref-link-section-d19454052e1409">39</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 40" title="Banal, J. L. et al. Random access DNA memory using Boolean search in an archival file storage system. Nat. Mater. 1–9. 
                  https://doi.org/10.1038/s41563-021-01021-3
                  
                 (2021)." href="https://www.nature.com/articles/s41467-023-38876-w#ref-CR40" id="ref-link-section-d19454052e1412">40</a></sup>. Advances in barcode multiplexing methods, as well as metadata tagging, can potentially lead to a searchable image database, as proposed by previous groups<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 40" title="Banal, J. L. et al. Random access DNA memory using Boolean search in an archival file storage system. Nat. Mater. 1–9. 
                  https://doi.org/10.1038/s41563-021-01021-3
                  
                 (2021)." href="https://www.nature.com/articles/s41467-023-38876-w#ref-CR40" id="ref-link-section-d19454052e1416">40</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 41" title="Bee, C. et al. Molecular-level similarity search brings computing to DNA data storage. Nat. Commun. 12, 4764 (2021)." href="https://www.nature.com/articles/s41467-023-38876-w#ref-CR41" id="ref-link-section-d19454052e1419">41</a></sup>.</p><p>While we utilize light-responsive recombinases in our workflow as the main interface between light and DNA recording, it is by no means the only DNA writing system that can be used. Other advances in DNA writing and editing tools can also serve to act as the intermediary for the conversion of light to information stored in DNA, and possess further advantages such as increased orthogonality. This could lead to an increase in density and the expansion in the types of images that can be stored.</p><p>As the field of DNA data storage continues to progress, there is an increasing interest in bridging the interface between biological and digital systems. Our work showcases further applications of DNA data storage that recreate existing information capture devices in a biological form, providing the basis for continued innovation in information recording and storage.</p></div></div></section><section data-title="Methods"><div id="Sec10-section"><h2 id="Sec10">Methods</h2><div id="Sec10-content"><h3 id="Sec11">Capturing of blue light patterns into DNA</h3><p>Two plasmids, pBbAW4k-loxP-TT-loxP-mRFP1 (pLoxP) and pBbE5a-Opto-Cre-Vvd-2 (pOptoCre) were used. pBbAW4k-loxP-TT-loxP-mRFP1 was a gift from Mary Dunlop (Addgene plasmid # 134405; <a href="http://n2t.net/addgene:134405">http://n2t.net/addgene:134405</a>; RRID:Addgene_134405) and pBbS5a-Opto-Cre-Vvd-2 was a gift from Mary Dunlop (Addgene plasmid # 160400; <a href="http://n2t.net/addgene:160400">http://n2t.net/addgene:160400</a>; RRID:Addgene_160400). These plasmids were transformed together into XL1-Blue <i>E.  coli</i> chemically competent cells. pLoxP contains two LoxP sites within the plasmid in the same orientation, with an intervening sequence containing 2 terminator sequences. pOptoCre contains a genetic circuit that utilizes the Lac promoter to control production of the Opto-Cre-Vvd-2 construct. The transformed strain was inoculated into a culture tube with 5 ml of LB medium supplemented with 100 mg/ml of ampicillin and 50 mg/ml of kanamycin and grown overnight in a shaking incubator at 37 °C, aerobically. The culture was diluted 1:100 into a fresh culture tube with 10 ml of LB medium supplemented with 100 mg/ml of ampicillin and 50 mg/ml of kanamycin and induced with 0.1 µM of IPTG for 2 h in a 37 °C shaking incubator. Eighty microliters of refreshed and induced cells were aliquoted into each well of a 96-well, black and flat clear-bottomed plate. A light pattern was created by selectively blocking out the bottoms of wells with aluminum foil, and the plate was exposed to light from the bottom with a blue LED light pane (HQRP). The plate was incubated atop the device overnight at room temperature to ensure sufficient exposure to light. However, this duration can be shortened to 30 min if necessary as demonstrated in Supplementary <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41467-023-38876-w#MOESM1">S4</a>.</p><h3 id="Sec12">Barcoding of DNA and library prep for illumina sequencing</h3><p>Ninety-six different PCR reactions were set up. Each reaction well contains 0.5 µl of a specific known well-code at a concentration of 10 µM that is mapped to the <i>X</i>–<i>Y</i> coordinates of that well. The unique barcoding sequences were generated computationally using the ‘DNABarcodes’ R package in Bioconductor (version 1.28.0)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 42" title="Buschmann, T. &amp; Bystrykh, L. V. Levenshtein error-correcting barcodes for multiplexed DNA sequencing. BMC Bioinforma. 14, 272 (2013)." href="https://www.nature.com/articles/s41467-023-38876-w#ref-CR42" id="ref-link-section-d19454052e1476">42</a></sup>. One microliter of cells from each well of the 96-well plate was then used for the PCR reactions. PCRs were performed using Taq DNA polymerase (NEB) with each reaction containing 2 µl of 10X ThermoPol buffer (NEB), 0.1 µl of 100 µM of each forward and reverse primer pair (IDT), 0.2 µl of Taq DNA polymerase (NEB), 0.2 µl of 20 mM dNTP mix (BioBasic), 1 µl of cells, and topped up with ddH<sub>2</sub>O for a total reaction volume of 20 µl. Individual reactions were run on 1.2% agarose gels for product verification. Appending of adapter ends for sequencing was also done using PCR, with 0.2 µl of each barcoded mix from the previous reaction added into a reaction containing 0.2 µl Taq DNA polymerase (NEB), 2 µl of 10X ThermoPol buffer (NEB), 0.1 µl each of 100 µM inner sequencing index primers (IDT), 0.3 µl of 100 µM outer sequencing index primers (IDT), 0.2 µl of 20 mM dNTP mix (BioBasic), and topped up with ddH<sub>2</sub>O to a total volume of 20 µl for each reaction. Products were subsequently verified on agarose gel before aliquoting 10 µl from each reaction and pooled together into a single tube. Two hundred microliters of this mixed pool was sent for NovaSeq dual index paired-end sequencing with an external sequencing service provider (NovogeneAIT).</p><h3 id="Sec13">Deconvolution of raw reads into images</h3><p>The resulting raw data generated was deconvoluted with appropriate index sequences and separated accordingly by the external sequencing provider. Subsequent analysis of the data was done using an in-house R script (code provided in the Supplementary Software file) (R version 4.2.2<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 43" title="R Core Team. R: the R project for statistical computing. 
                  https://www.r-project.org/
                  
                 (2022)." href="https://www.nature.com/articles/s41467-023-38876-w#ref-CR43" id="ref-link-section-d19454052e1492">43</a></sup>, Bioconductor version 3.16<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 44" title="Huber, W. et al. Orchestrating high-throughput genomic analysis with Bioconductor. Nat. Methods 12, 115–121 (2015)." href="https://www.nature.com/articles/s41467-023-38876-w#ref-CR44" id="ref-link-section-d19454052e1496">44</a></sup>, Biostrings version 2.66.0<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 45" title="Pagès, H., Aboyoun, P., Gentleman, R. &amp; DebRoy, S. Biostrings: efficient manipulation of biological strings. 
                  https://doi.org/10.18129/B9.bioc.Biostrings
                  
                 (2023)." href="https://www.nature.com/articles/s41467-023-38876-w#ref-CR45" id="ref-link-section-d19454052e1500">45</a></sup>, ShortRead version 1.56.1<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 46" title="Morgan, M. et al. ShortRead: a bioconductor package for input, quality assessment and exploration of high-throughput sequence data. Bioinformatics 25, 2607–2608 (2009)." href="https://www.nature.com/articles/s41467-023-38876-w#ref-CR46" id="ref-link-section-d19454052e1504">46</a></sup> and stringr version 1.5.0<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 47" title="Wickham, H. stringr: Simple, Consistent Wrappers for Common String Operations. 
                  https://stringr.tidyverse.org
                  
                , 
                  https://github.com/tidyverse/stringr
                  
                 (2022)." href="https://www.nature.com/articles/s41467-023-38876-w#ref-CR47" id="ref-link-section-d19454052e1508">47</a></sup> packages). Briefly, the script derived the total number of reads, the number of reads with excised intervening LoxP sequences, and number of reads with full LoxP sequences, that were linked to each unique well-code. To determine whether that unique barcoded population was exposed to light or not, the ratio of truncated to full reads was determined. This ratio is calculated for each well in the entire plate, and serves to define the bit state of the well. A higher ratio indicates a larger proportion of truncated reads, which implies that a large number of the population of cells that were barcoded was exposed to light and vice versa. To reconstruct the images, each the aforementioned ratio was converted to either 1 or 0, with the threshold between both determined with GMM clustering to define the two clusters and the threshold set via the boundary between the two clusters with the lowest variance score. This is then linked back to the well-code, which was then mapped to its predefined spatial location, thus creating an image with a pixel depth of 1 bit and a resolution of 96 pixels. Accuracy of the final image was defined as the number of wells that were appropriately demarcated as 1 or 0 as compared to the original projected pattern. Differentiating between two colors of light was done by calculating ratios of sequences specific to the blue light recording plasmid as well as the red light recording plasmid separately from one another.</p><h3 id="Sec14">Random access of images</h3><p>Primers specifically designed for random access with index sequences targeted at selected indexes were synthesized and used to enable random access. Eight PCR reactions with the same conditions were set up to generate sufficient DNA for sequencing. 1 µl of template from the mixed pool was used for each reaction totaling 50 µl, and a touchdown PCR with decreasing annealing temperatures from 65 to 55 °C for eight cycles and constant annealing temperatures for 20 cycles was conducted, using specific random access primers that bind to desired image to be amplified. Resulting samples were appended with Illumina adapter sequences and sent for sequencing. Diluting experiments were conducted by taking 1 µl of template from the mixed pool and diluting in 9, 99, and 999 µl of PBS before taking 1 µl from each dilution for PCR as per the above protocol. N701 and N501 indexing primers selective for the ‘NUS’ pattern were used for this particular random access dilution.</p><h3 id="Sec15">Environmental challenge experiments</h3><p>A pool of DNA containing the ‘Smiley’ pattern was used for drying and resuspension experiments. Twenty microliters of the pool was spun dry using a centrifuge, and subsequently resuspended in 50 µl ddH<sub>2</sub>O before being submitted for sequencing.</p><p>For environmental challenge experiments, 200 µl of the ‘Heloo wo{|d!’ pool was kept in varying conditions in a cryotube, wrapped in aluminum foil. Conditions included frozen in −20 °C, at room temperature, and in a 60 °C oven, all for 7 days. One last sample was kept in a 1.5 ml Eppendorf tube which was then exposed to UV light for 1 hour. Samples were then sent for sequencing.</p><h3 id="Sec16">Dilution experiments</h3><p>Two forms of dilution experiments were conducted. One involved diluting a pool of DNA containing the ‘Smiley’ pattern 3 consecutive times, at a 10 times dilution each, with each dilution being sent for sequencing. The other involved diluting a mixed pool of DNA with multiple samples 3 consecutive times, with each dilution being a 10 times dilution as described above. Random access was then conducted on each dilution as detailed above before sending the amplified products for sequencing.</p><h3 id="Sec17">Clustering of deconvoluted images</h3><p>The clustering workflow was developed and implemented in Python 3<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 48" title="The Python Language Reference. Python documentation. 
                  https://docs.python.org/3/reference/index.html
                  
                ." href="https://www.nature.com/articles/s41467-023-38876-w#ref-CR48" id="ref-link-section-d19454052e1550">48</a></sup> (version 3.11.1) (code provided in the Supplementary Software file). Four unsupervised clustering techniques (K-means algorithm<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 49" title="Arthur, D. &amp; Vassilvitskii, S. k-means++: the advantages of careful seeding. In Proceedings of the eighteenth annual ACM-SIAM symposium on Discrete algorithms 1027–1035 (Society for Industrial and Applied Mathematics, 2007)." href="https://www.nature.com/articles/s41467-023-38876-w#ref-CR49" id="ref-link-section-d19454052e1554">49</a></sup>, DBSCAN<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 50" title="Schubert, E., Sander, J., Ester, M., Kriegel, H. P. &amp; Xu, X. DBSCAN revisited, revisited: why and how you should (Still) use DBSCAN. ACM Trans. Database Syst. 42, 1–21 (2017)." href="https://www.nature.com/articles/s41467-023-38876-w#ref-CR50" id="ref-link-section-d19454052e1558">50</a></sup>, OPTICS<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 51" title="Ankerst, M., Breunig, M. M., Kriegel, H.-P. &amp; Sander, J. OPTICS: ordering points to identify the clustering structure. ACM SIGMOD Rec. 28, 49–60 (1999)." href="https://www.nature.com/articles/s41467-023-38876-w#ref-CR51" id="ref-link-section-d19454052e1562">51</a></sup>, and Gaussian Mixture Model<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 52" title="Yang, M.-S., Lai, C.-Y. &amp; Lin, C.-Y. A robust EM clustering algorithm for Gaussian mixture models. Pattern Recognit. 45, 3950–3961 (2012)." href="https://www.nature.com/articles/s41467-023-38876-w#ref-CR52" id="ref-link-section-d19454052e1566">52</a></sup>) were adopted from scikit-learn (version 1.2.0) machine learning packages under cluster and mixture modules<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 53" title="Pedregosa, F. et al. Scikit-learn: Machine Learning in Python. J. Mach. Learn. Res. 12, 2825–2830 (2011)." href="https://www.nature.com/articles/s41467-023-38876-w#ref-CR53" id="ref-link-section-d19454052e1571">53</a></sup>. The unsupervised outlier detection algorithm was implemented using Local Outlier Factor (LOF) from scikit-learn neighbors module. The parameter ‘n_neighbors’ (blue light = 20; red light = 10) was tuned accordingly to improve the deconvolution accuracy through better reassignment of outliers near to the cutoff threshold for better clustering performance (Supplementary <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41467-023-38876-w#MOESM1">S6</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41467-023-38876-w#MOESM1">S7</a>). The cutoff threshold for the full ‘ON’ and ‘OFF’ was determined from the means computed from clusters from all existing datasets. The dataset would be classified as full ‘ON’ or ‘OFF’ when the lowest mean of clusters is above the cutoff threshold, or the highest mean is below the cutoff threshold. To generate the final binary ‘0’ or ‘1’ images, cluster grouping was executed for clustering techniques that identified more than 2 clusters to group them into two clusters designated as ‘ON’ and ‘OFF’ states. For the blue light patterns, the second and higher mean clusters were grouped into a single ‘ON’ cluster whereas the initial cluster with the lowest mean would be considered as ‘OFF’ states. Meanwhile, for the red light patterns, the two clusters with the lower mean were considered as a single ‘OFF’ cluster instead whereas the rest of the clusters would be considered as ‘ON’ states due to differences in the distribution of the raw data.</p><h3 id="Sec18">Development of the red light bacterial recombinase system</h3><p>Two plasmids were designed for the red light bacterial recombinase system. pBbS5a-RLCre, a red light sensitive split Cre-recombinase based off the L-SCRaMbLE system previously developed in yeast<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 35" title="Hochrein, L., Mitchell, L. A., Schulz, K., Messerschmidt, K. &amp; Mueller-Roeber, B. L-SCRaMbLE as a tool for light-controlled Cre-mediated recombination in yeast. Nat. Commun. 9, 1931 (2018)." href="https://www.nature.com/articles/s41467-023-38876-w#ref-CR35" id="ref-link-section-d19454052e1589">35</a></sup> was designed, codon-optimized for <i>E. coli</i> and synthesized (Twist Bioscience). pBbAW4k-Spacer1Barcoding-loxP-TT-loxP-ho1-pcyA, a recording plasmid, was also designed, consisting of LoxP recognition sites, a unique preceding barcode (Spacer1Barcoding), as well a ho1-pcyA component that produces phycocyanobilin necessary for the function of the red light split recombinase. Both plasmids were transformed simultaneously into XL1 Blue <i>E. coli</i> chemically competent cells.</p><h3 id="Sec19">Design and construction of a custom light illumination device (OptoBox)</h3><p>To test the co-culture of red and blue light-responsive <i>E. coli</i> under various blue and red light patterns, a 96-well programmable LED device was developed (design illustrated in Supplementary <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41467-023-38876-w#MOESM1">S8</a>). This device is powered and controlled by an Arduino Uno Microcontroller and includes 8 strips of 12 Adafruit Neopixel SMD 5050 RGB LEDs totalling 96 individual LEDs. Each of these LEDs corresponds to each well of the 96-well plate. The overall construction of the device comprises a 96-tip pipette tip container. These pipette tip containers comprise two separable layers stacked on top of one another. This setup allows the LED strips to be immobilized between the layers. The wells of the pipette tip container also prevent residual light spillover from the LEDs, resulting in spatial isolation when coupled with a clear-bottomed black 96-well plate.</p><p>For our application, these hollow tip-holes are used to fit and secure the SMD 5050 LEDs in place by fitting the LED chips into the underside of each of these corresponding holes. This allows each individual LED to be held in place through a friction fit, with the diodes that make up the SMD 5050 LED centered to the wells of the 96-well plate. The LED strips were soldered to wires supplying power and electrical control. Each strip’s data-wire was connected to an individual Arduino Uno data pin, allowing individual control of each LED. Programming of the LEDs was done using the available FastLED Arduino library.</p><h3 id="Sec20">Capture of multicolor images with a co-culture</h3><p>Strains containing the red light and blue light recombinase systems were inoculated separately into culture tubes with 5 ml of LB medium supplemented with 100 mg/ml of ampicillin and 50 mg/ml of kanamycin and grown overnight in a shaking incubator at 37 °C, aerobically. These cultures were diluted 1:100 into fresh culture tubes with 10 ml of LB medium supplemented with 100 mg/ml of ampicillin and 50 mg/ml of kanamycin and induced with 0.1 µM of IPTG for 2 h in a 37 °C shaking incubator. 40 µl of each culture were aliquoted into each well of a 96-well, black and flat clear-bottomed plate. A red (620 nm) and a blue (465 nm) light pattern was created by the in-house custom-built light illumination device (OptoBox), and either projected periodically for 10 min in an alternating fashion overnight, or projected simultaneously overnight, both from the bottom of the plate and at equal intensities. Barcoding, sequencing and deconvolution follows the same procedure as detailed in the preceding sections.</p><h3 id="Sec21">Statistics and reproducibility</h3><p>No statistical method was used to predetermine sample size. No data were excluded from the analyses. The experiments were not randomized. The Investigators were not blinded to allocation during experiments and outcome assessment.</p><h3 id="Sec22">Reporting summary</h3><p>Further information on research design is available in the <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41467-023-38876-w#MOESM2">Nature Portfolio Reporting Summary</a> linked to this article.</p></div></div></section>
                </div><div>
            <section data-title="Data availability"><div id="data-availability-section"><h2 id="data-availability">Data availability</h2><p>The sequencing data generated in this study have been deposited in a Figshare repository at: <a href="https://doi.org/10.6084/m9.figshare.22678534">https://doi.org/10.6084/m9.figshare.22678534</a>. The same data have also been deposited in the NCBI Sequence Read Archive under the PRJNA970212 BioProject at <a href="https://www.ncbi.nlm.nih.gov/bioproject/PRJNA970212">https://www.ncbi.nlm.nih.gov/bioproject/PRJNA970212</a>, with the individual accession numbers available in the supplementary file under Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41467-023-38876-w#MOESM1">2</a>. <a data-track="click" data-track-label="link" data-track-action="section anchor" href="https://www.nature.com/articles/s41467-023-38876-w#Sec24">Source data</a> are provided with this paper. Barcodes and primers used in this work are detailed in the Source Data file.</p></div></section><section data-title="Code availability"><div id="code-availability-section"><h2 id="code-availability">Code availability</h2><p>The code used for deconvolution of reads as well as clustering is available in the <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41467-023-38876-w#MOESM5">Supplementary Software</a> file.</p></div></section><div id="MagazineFulltextArticleBodySuffix"><section aria-labelledby="Bib1" data-title="References"><div id="Bib1-section"><h2 id="Bib1">References</h2><div id="Bib1-content"><div data-container-section="references"><ol data-track-component="outbound reference"><li data-counter="1."><p id="ref-CR1">Church, G. M., Gao, Y. &amp; Kosuri, S. Next-generation digital information storage in DNA. <i>Science</i> <b>337</b>, 1628–1628 (2012).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1126/science.1226355" data-track-action="article reference" href="https://doi.org/10.1126%2Fscience.1226355" aria-label="Article reference 1" data-doi="10.1126/science.1226355">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2012Sci...337.1628C" aria-label="ADS reference 1">ADS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38Xhtl2ntrrL" aria-label="CAS reference 1">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22903519" aria-label="PubMed reference 1">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 1" href="http://scholar.google.com/scholar_lookup?&amp;title=Next-generation%20digital%20information%20storage%20in%20DNA&amp;journal=Science&amp;doi=10.1126%2Fscience.1226355&amp;volume=337&amp;pages=1628-1628&amp;publication_year=2012&amp;author=Church%2CGM&amp;author=Gao%2CY&amp;author=Kosuri%2CS">
                    Google Scholar</a> 
                </p></li><li data-counter="2."><p id="ref-CR2">Goldman, N. et al. Towards practical, high-capacity, low-maintenance information storage in synthesized DNA. <i>Nature</i> <b>494</b>, 77–80 (2013).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/nature11875" data-track-action="article reference" href="https://doi.org/10.1038%2Fnature11875" aria-label="Article reference 2" data-doi="10.1038/nature11875">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2013Natur.494...77G" aria-label="ADS reference 2">ADS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXhsVygsLg%3D" aria-label="CAS reference 2">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23354052" aria-label="PubMed reference 2">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3672958" aria-label="PubMed Central reference 2">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 2" href="http://scholar.google.com/scholar_lookup?&amp;title=Towards%20practical%2C%20high-capacity%2C%20low-maintenance%20information%20storage%20in%20synthesized%20DNA&amp;journal=Nature&amp;doi=10.1038%2Fnature11875&amp;volume=494&amp;pages=77-80&amp;publication_year=2013&amp;author=Goldman%2CN">
                    Google Scholar</a> 
                </p></li><li data-counter="3."><p id="ref-CR3">Zhirnov, V., Zadegan, R. M., Sandhu, G. S., Church, G. M. &amp; Hughes, W. L. Nucleic acid memory. <i>Nat. Mater.</i> <b>15</b>, 366–370 (2016).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/nmat4594" data-track-action="article reference" href="https://doi.org/10.1038%2Fnmat4594" aria-label="Article reference 3" data-doi="10.1038/nmat4594">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2016NatMa..15..366Z" aria-label="ADS reference 3">ADS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XkvVCitb0%3D" aria-label="CAS reference 3">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27005909" aria-label="PubMed reference 3">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6361517" aria-label="PubMed Central reference 3">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 3" href="http://scholar.google.com/scholar_lookup?&amp;title=Nucleic%20acid%20memory&amp;journal=Nat.%20Mater.&amp;doi=10.1038%2Fnmat4594&amp;volume=15&amp;pages=366-370&amp;publication_year=2016&amp;author=Zhirnov%2CV&amp;author=Zadegan%2CRM&amp;author=Sandhu%2CGS&amp;author=Church%2CGM&amp;author=Hughes%2CWL">
                    Google Scholar</a> 
                </p></li><li data-counter="4."><p id="ref-CR4">van der Valk, T. et al. Million-year-old DNA sheds light on the genomic history of mammoths. <i>Nature</i>, 1–5. <a href="https://doi.org/10.1038/s41586-021-03224-9">https://doi.org/10.1038/s41586-021-03224-9</a> (2021).</p></li><li data-counter="5."><p id="ref-CR5">Lim, C. K., Nirantar, S., Yew, W. S. &amp; Poh, C. L. Novel Modalities in DNA Data Storage. <i>Trends Biotechnol</i>. <b>39</b>, 990–1003 (2021).</p></li><li data-counter="6."><p id="ref-CR6">Tabatabaei, S. K. et al. DNA punch cards for storing data on native DNA sequences via enzymatic nicking. <i>Nat. Commun.</i> <b>11</b>, 1–10 (2020).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/s41467-020-15588-z" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41467-020-15588-z" aria-label="Article reference 6" data-doi="10.1038/s41467-020-15588-z">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 6" href="http://scholar.google.com/scholar_lookup?&amp;title=DNA%20punch%20cards%20for%20storing%20data%20on%20native%20DNA%20sequences%20via%20enzymatic%20nicking&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-020-15588-z&amp;volume=11&amp;pages=1-10&amp;publication_year=2020&amp;author=Tabatabaei%2CSK">
                    Google Scholar</a> 
                </p></li><li data-counter="7."><p id="ref-CR7">Chen, K., Zhu, J., Bošković, F. &amp; Keyser, U. F. Nanopore-based DNA hard drives for rewritable and secure data storage. <i>Nano Lett</i>. <a href="https://doi.org/10.1021/acs.nanolett.0c00755">https://doi.org/10.1021/acs.nanolett.0c00755</a> (2020).</p></li><li data-counter="8."><p id="ref-CR8">Tomek, K. J. et al. Driving the scalability of DNA-based information storage systems. <i>ACS Synth. Biol.</i> <b>8</b>, 1241–1248 (2019).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1021/acssynbio.9b00100" data-track-action="article reference" href="https://doi.org/10.1021%2Facssynbio.9b00100" aria-label="Article reference 8" data-doi="10.1021/acssynbio.9b00100">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXpvVCmur4%3D" aria-label="CAS reference 8">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31117362" aria-label="PubMed reference 8">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 8" href="http://scholar.google.com/scholar_lookup?&amp;title=Driving%20the%20scalability%20of%20DNA-based%20information%20storage%20systems&amp;journal=ACS%20Synth.%20Biol.&amp;doi=10.1021%2Facssynbio.9b00100&amp;volume=8&amp;pages=1241-1248&amp;publication_year=2019&amp;author=Tomek%2CKJ">
                    Google Scholar</a> 
                </p></li><li data-counter="9."><p id="ref-CR9">Yim, S. S. et al. Robust direct digital-to-biological data storage in living cells. <i>Nat. Chem. Biol.</i> <b>17</b>, 1–8 (2021).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/s41589-020-00711-4" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41589-020-00711-4" aria-label="Article reference 9" data-doi="10.1038/s41589-020-00711-4">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 9" href="http://scholar.google.com/scholar_lookup?&amp;title=Robust%20direct%20digital-to-biological%20data%20storage%20in%20living%20cells&amp;journal=Nat.%20Chem.%20Biol.&amp;doi=10.1038%2Fs41589-020-00711-4&amp;volume=17&amp;pages=1-8&amp;publication_year=2021&amp;author=Yim%2CSS">
                    Google Scholar</a> 
                </p></li><li data-counter="10."><p id="ref-CR10">Ceze, L., Nivala, J. &amp; Strauss, K. Molecular digital data storage using DNA. <i>Nat. Rev. Genet.</i> <b>20</b>, 456–466 (2019).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/s41576-019-0125-3" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41576-019-0125-3" aria-label="Article reference 10" data-doi="10.1038/s41576-019-0125-3">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXptFeqsrs%3D" aria-label="CAS reference 10">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31068682" aria-label="PubMed reference 10">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 10" href="http://scholar.google.com/scholar_lookup?&amp;title=Molecular%20digital%20data%20storage%20using%20DNA&amp;journal=Nat.%20Rev.%20Genet.&amp;doi=10.1038%2Fs41576-019-0125-3&amp;volume=20&amp;pages=456-466&amp;publication_year=2019&amp;author=Ceze%2CL&amp;author=Nivala%2CJ&amp;author=Strauss%2CK">
                    Google Scholar</a> 
                </p></li><li data-counter="11."><p id="ref-CR11">Lee, H. H., Kalhor, R., Goela, N., Bolot, J. &amp; Church, G. M. Terminator-free template-independent enzymatic DNA synthesis for digital information storage. <i>Nat. Commun.</i> <b>10</b>, 1–12 (2019).</p><p><a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 11" href="http://scholar.google.com/scholar_lookup?&amp;title=Terminator-free%20template-independent%20enzymatic%20DNA%20synthesis%20for%20digital%20information%20storage&amp;journal=Nat.%20Commun.&amp;volume=10&amp;pages=1-12&amp;publication_year=2019&amp;author=Lee%2CHH&amp;author=Kalhor%2CR&amp;author=Goela%2CN&amp;author=Bolot%2CJ&amp;author=Church%2CGM">
                    Google Scholar</a> 
                </p></li><li data-counter="12."><p id="ref-CR12">Takahashi, C. N., Nguyen, B. H., Strauss, K. &amp; Ceze, L. Demonstration of end-to-end automation of DNA Data Storage. <i>Sci. Rep.</i> <b>9</b>, 1–5 (2019).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/s41598-019-41228-8" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41598-019-41228-8" aria-label="Article reference 12" data-doi="10.1038/s41598-019-41228-8">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 12" href="http://scholar.google.com/scholar_lookup?&amp;title=Demonstration%20of%20end-to-end%20automation%20of%20DNA%20Data%20Storage&amp;journal=Sci.%20Rep.&amp;doi=10.1038%2Fs41598-019-41228-8&amp;volume=9&amp;pages=1-5&amp;publication_year=2019&amp;author=Takahashi%2CCN&amp;author=Nguyen%2CBH&amp;author=Strauss%2CK&amp;author=Ceze%2CL">
                    Google Scholar</a> 
                </p></li><li data-counter="13."><p id="ref-CR13">Palluk, S. et al. De novo DNA synthesis using polymerase-nucleotide conjugates. <i>Nat. Biotechnol.</i> <b>36</b>, 645–650 (2018).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/nbt.4173" data-track-action="article reference" href="https://doi.org/10.1038%2Fnbt.4173" aria-label="Article reference 13" data-doi="10.1038/nbt.4173">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtFGjs7%2FI" aria-label="CAS reference 13">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29912208" aria-label="PubMed reference 13">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 13" href="http://scholar.google.com/scholar_lookup?&amp;title=De%20novo%20DNA%20synthesis%20using%20polymerase-nucleotide%20conjugates&amp;journal=Nat.%20Biotechnol.&amp;doi=10.1038%2Fnbt.4173&amp;volume=36&amp;pages=645-650&amp;publication_year=2018&amp;author=Palluk%2CS">
                    Google Scholar</a> 
                </p></li><li data-counter="14."><p id="ref-CR14">Lu, X. et al. Enzymatic DNA synthesis by engineering terminal deoxynucleotidyl transferase. <i>ACS Catal</i>. 2988–2997. <a href="https://doi.org/10.1021/acscatal.1c04879">https://doi.org/10.1021/acscatal.1c04879</a> (2022).</p></li><li data-counter="15."><p id="ref-CR15">Xu, C. et al. Electrochemical DNA synthesis and sequencing on a single electrode with scalability for integrated data storage. <i>Sci. Adv.</i> <b>7</b>, eabk0100 (2021).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1126/sciadv.abk0100" data-track-action="article reference" href="https://doi.org/10.1126%2Fsciadv.abk0100" aria-label="Article reference 15" data-doi="10.1126/sciadv.abk0100">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2021SciA....7..100X" aria-label="ADS reference 15">ADS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXis1Ghsb3E" aria-label="CAS reference 15">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34767438" aria-label="PubMed reference 15">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8589306" aria-label="PubMed Central reference 15">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 15" href="http://scholar.google.com/scholar_lookup?&amp;title=Electrochemical%20DNA%20synthesis%20and%20sequencing%20on%20a%20single%20electrode%20with%20scalability%20for%20integrated%20data%20storage&amp;journal=Sci.%20Adv.&amp;doi=10.1126%2Fsciadv.abk0100&amp;volume=7&amp;publication_year=2021&amp;author=Xu%2CC">
                    Google Scholar</a> 
                </p></li><li data-counter="16."><p id="ref-CR16">Nguyen, B. H. et al. Scaling DNA data storage with nanoscale electrode wells. <i>Sci. Adv.</i> <b>7</b>, eabi6714 (2021).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1126/sciadv.abi6714" data-track-action="article reference" href="https://doi.org/10.1126%2Fsciadv.abi6714" aria-label="Article reference 16" data-doi="10.1126/sciadv.abi6714">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2021SciA....7.6714N" aria-label="ADS reference 16">ADS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXislCntLrK" aria-label="CAS reference 16">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34818035" aria-label="PubMed reference 16">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8612674" aria-label="PubMed Central reference 16">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 16" href="http://scholar.google.com/scholar_lookup?&amp;title=Scaling%20DNA%20data%20storage%20with%20nanoscale%20electrode%20wells&amp;journal=Sci.%20Adv.&amp;doi=10.1126%2Fsciadv.abi6714&amp;volume=7&amp;publication_year=2021&amp;author=Nguyen%2CBH">
                    Google Scholar</a> 
                </p></li><li data-counter="17."><p id="ref-CR17">Antkowiak, P. L. et al. Low cost DNA data storage using photolithographic synthesis and advanced information reconstruction and error correction. <i>Nat. Commun.</i> <b>11</b>, 5345 (2020).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/s41467-020-19148-3" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41467-020-19148-3" aria-label="Article reference 17" data-doi="10.1038/s41467-020-19148-3">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2020NatCo..11.5345A" aria-label="ADS reference 17">ADS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXitF2js7rI" aria-label="CAS reference 17">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33093494" aria-label="PubMed reference 17">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7582880" aria-label="PubMed Central reference 17">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 17" href="http://scholar.google.com/scholar_lookup?&amp;title=Low%20cost%20DNA%20data%20storage%20using%20photolithographic%20synthesis%20and%20advanced%20information%20reconstruction%20and%20error%20correction&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-020-19148-3&amp;volume=11&amp;publication_year=2020&amp;author=Antkowiak%2CPL">
                    Google Scholar</a> 
                </p></li><li data-counter="18."><p id="ref-CR18">Erlich, Y. &amp; Zielinski, D. DNA Fountain enables a robust and efficient storage architecture. <i>Science</i> <b>355</b>, 950–954 (2017).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1126/science.aaj2038" data-track-action="article reference" href="https://doi.org/10.1126%2Fscience.aaj2038" aria-label="Article reference 18" data-doi="10.1126/science.aaj2038">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2017Sci...355..950E" aria-label="ADS reference 18">ADS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXjsVCgsrc%3D" aria-label="CAS reference 18">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28254941" aria-label="PubMed reference 18">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 18" href="http://scholar.google.com/scholar_lookup?&amp;title=DNA%20Fountain%20enables%20a%20robust%20and%20efficient%20storage%20architecture&amp;journal=Science&amp;doi=10.1126%2Fscience.aaj2038&amp;volume=355&amp;pages=950-954&amp;publication_year=2017&amp;author=Erlich%2CY&amp;author=Zielinski%2CD">
                    Google Scholar</a> 
                </p></li><li data-counter="19."><p id="ref-CR19">Wang, Y. et al. High capacity DNA data storage with variable-length Oligonucleotides using repeat accumulate code and hybrid mapping. <i>J. Biol. Eng.</i> <b>13</b>, 89 (2019).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1186/s13036-019-0211-2" data-track-action="article reference" href="https://doi.org/10.1186%2Fs13036-019-0211-2" aria-label="Article reference 19" data-doi="10.1186/s13036-019-0211-2">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31832092" aria-label="PubMed reference 19">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6868767" aria-label="PubMed Central reference 19">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 19" href="http://scholar.google.com/scholar_lookup?&amp;title=High%20capacity%20DNA%20data%20storage%20with%20variable-length%20Oligonucleotides%20using%20repeat%20accumulate%20code%20and%20hybrid%20mapping&amp;journal=J.%20Biol.%20Eng.&amp;doi=10.1186%2Fs13036-019-0211-2&amp;volume=13&amp;publication_year=2019&amp;author=Wang%2CY">
                    Google Scholar</a> 
                </p></li><li data-counter="20."><p id="ref-CR20">Anavy, L., Vaknin, I., Atar, O., Amit, R. &amp; Yakhini, Z. Data storage in DNA with fewer synthesis cycles using composite DNA letters. <i>Nat. Biotechnol.</i> <b>37</b>, 1229–1236 (2019).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/s41587-019-0240-x" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41587-019-0240-x" aria-label="Article reference 20" data-doi="10.1038/s41587-019-0240-x">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhsleltLvL" aria-label="CAS reference 20">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31501560" aria-label="PubMed reference 20">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 20" href="http://scholar.google.com/scholar_lookup?&amp;title=Data%20storage%20in%20DNA%20with%20fewer%20synthesis%20cycles%20using%20composite%20DNA%20letters&amp;journal=Nat.%20Biotechnol.&amp;doi=10.1038%2Fs41587-019-0240-x&amp;volume=37&amp;pages=1229-1236&amp;publication_year=2019&amp;author=Anavy%2CL&amp;author=Vaknin%2CI&amp;author=Atar%2CO&amp;author=Amit%2CR&amp;author=Yakhini%2CZ">
                    Google Scholar</a> 
                </p></li><li data-counter="21."><p id="ref-CR21">Sheets, M. B., Wong, W. W. &amp; Dunlop, M. J. Light-inducible recombinases for bacterial optogenetics. <i>ACS Synth. Biol.</i> <b>9</b>, 227–235 (2020).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1021/acssynbio.9b00395" data-track-action="article reference" href="https://doi.org/10.1021%2Facssynbio.9b00395" aria-label="Article reference 21" data-doi="10.1021/acssynbio.9b00395">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXht1Wiu74%3D" aria-label="CAS reference 21">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31961670" aria-label="PubMed reference 21">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7393974" aria-label="PubMed Central reference 21">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 21" href="http://scholar.google.com/scholar_lookup?&amp;title=Light-inducible%20recombinases%20for%20bacterial%20optogenetics&amp;journal=ACS%20Synth.%20Biol.&amp;doi=10.1021%2Facssynbio.9b00395&amp;volume=9&amp;pages=227-235&amp;publication_year=2020&amp;author=Sheets%2CMB&amp;author=Wong%2CWW&amp;author=Dunlop%2CMJ">
                    Google Scholar</a> 
                </p></li><li data-counter="22."><p id="ref-CR22">Yang, L. et al. Permanent genetic memory with &gt;1-byte capacity. <i>Nat. Methods</i> <b>11</b>, 1261–1266 (2014).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/nmeth.3147" data-track-action="article reference" href="https://doi.org/10.1038%2Fnmeth.3147" aria-label="Article reference 22" data-doi="10.1038/nmeth.3147">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXhvVSqsbnE" aria-label="CAS reference 22">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25344638" aria-label="PubMed reference 22">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4245323" aria-label="PubMed Central reference 22">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 22" href="http://scholar.google.com/scholar_lookup?&amp;title=Permanent%20genetic%20memory%20with%20%3E1-byte%20capacity&amp;journal=Nat.%20Methods&amp;doi=10.1038%2Fnmeth.3147&amp;volume=11&amp;pages=1261-1266&amp;publication_year=2014&amp;author=Yang%2CL">
                    Google Scholar</a> 
                </p></li><li data-counter="23."><p id="ref-CR23">Bonnet, J., Subsoontorn, P. &amp; Endy, D. Rewritable digital data storage in live cells via engineered control of recombination directionality. <i>Proc. Natl Acad. Sci. USA</i> <b>109</b>, 8884–8889 (2012).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1073/pnas.1202344109" data-track-action="article reference" href="https://doi.org/10.1073%2Fpnas.1202344109" aria-label="Article reference 23" data-doi="10.1073/pnas.1202344109">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2012PNAS..109.8884B" aria-label="ADS reference 23">ADS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XovF2gt70%3D" aria-label="CAS reference 23">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22615351" aria-label="PubMed reference 23">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3384180" aria-label="PubMed Central reference 23">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 23" href="http://scholar.google.com/scholar_lookup?&amp;title=Rewritable%20digital%20data%20storage%20in%20live%20cells%20via%20engineered%20control%20of%20recombination%20directionality&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;doi=10.1073%2Fpnas.1202344109&amp;volume=109&amp;pages=8884-8889&amp;publication_year=2012&amp;author=Bonnet%2CJ&amp;author=Subsoontorn%2CP&amp;author=Endy%2CD">
                    Google Scholar</a> 
                </p></li><li data-counter="24."><p id="ref-CR24">Shipman, S. L., Nivala, J., Macklis, J. D. &amp; Church, G. M. Molecular recordings by directed CRISPR spacer acquisition. <i>Science</i> <b>353</b>, aaf1175 (2016).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1126/science.aaf1175" data-track-action="article reference" href="https://doi.org/10.1126%2Fscience.aaf1175" aria-label="Article reference 24" data-doi="10.1126/science.aaf1175">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27284167" aria-label="PubMed reference 24">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4994893" aria-label="PubMed Central reference 24">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 24" href="http://scholar.google.com/scholar_lookup?&amp;title=Molecular%20recordings%20by%20directed%20CRISPR%20spacer%20acquisition&amp;journal=Science&amp;doi=10.1126%2Fscience.aaf1175&amp;volume=353&amp;publication_year=2016&amp;author=Shipman%2CSL&amp;author=Nivala%2CJ&amp;author=Macklis%2CJD&amp;author=Church%2CGM">
                    Google Scholar</a> 
                </p></li><li data-counter="25."><p id="ref-CR25">Shipman, S. L., Nivala, J., Macklis, J. D. &amp; Church, G. M. CRISPR–Cas encoding of a digital movie into the genomes of a population of living bacteria. <i>Nature</i> <b>547</b>, 345–349 (2017).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/nature23017" data-track-action="article reference" href="https://doi.org/10.1038%2Fnature23017" aria-label="Article reference 25" data-doi="10.1038/nature23017">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2017Natur.547..345S" aria-label="ADS reference 25">ADS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhtFOjtrfN" aria-label="CAS reference 25">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28700573" aria-label="PubMed reference 25">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5842791" aria-label="PubMed Central reference 25">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 25" href="http://scholar.google.com/scholar_lookup?&amp;title=CRISPR%E2%80%93Cas%20encoding%20of%20a%20digital%20movie%20into%20the%20genomes%20of%20a%20population%20of%20living%20bacteria&amp;journal=Nature&amp;doi=10.1038%2Fnature23017&amp;volume=547&amp;pages=345-349&amp;publication_year=2017&amp;author=Shipman%2CSL&amp;author=Nivala%2CJ&amp;author=Macklis%2CJD&amp;author=Church%2CGM">
                    Google Scholar</a> 
                </p></li><li data-counter="26."><p id="ref-CR26">Tang, W. &amp; Liu, D. R. Rewritable multi-event analog recording in bacterial and mammalian cells. <i>Science</i> <b>360</b>, eaap8992 (2018).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1126/science.aap8992" data-track-action="article reference" href="https://doi.org/10.1126%2Fscience.aap8992" aria-label="Article reference 26" data-doi="10.1126/science.aap8992">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29449507" aria-label="PubMed reference 26">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5898985" aria-label="PubMed Central reference 26">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 26" href="http://scholar.google.com/scholar_lookup?&amp;title=Rewritable%20multi-event%20analog%20recording%20in%20bacterial%20and%20mammalian%20cells&amp;journal=Science&amp;doi=10.1126%2Fscience.aap8992&amp;volume=360&amp;publication_year=2018&amp;author=Tang%2CW&amp;author=Liu%2CDR">
                    Google Scholar</a> 
                </p></li><li data-counter="27."><p id="ref-CR27">Chen, K. et al. Digital data storage using DNA nanostructures and solid-state nanopores. <i>Nano Lett.</i> <b>19</b>, 1210–1215 (2019).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1021/acs.nanolett.8b04715" data-track-action="article reference" href="https://doi.org/10.1021%2Facs.nanolett.8b04715" aria-label="Article reference 27" data-doi="10.1021/acs.nanolett.8b04715">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2019NanoL..19.1210C" aria-label="ADS reference 27">ADS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXisFyksrfO" aria-label="CAS reference 27">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30585490" aria-label="PubMed reference 27">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 27" href="http://scholar.google.com/scholar_lookup?&amp;title=Digital%20data%20storage%20using%20DNA%20nanostructures%20and%20solid-state%20nanopores&amp;journal=Nano%20Lett.&amp;doi=10.1021%2Facs.nanolett.8b04715&amp;volume=19&amp;pages=1210-1215&amp;publication_year=2019&amp;author=Chen%2CK">
                    Google Scholar</a> 
                </p></li><li data-counter="28."><p id="ref-CR28">Fernandez-Rodriguez, J., Moser, F., Song, M. &amp; Voigt, C. A. Engineering RGB color vision into <i>Escherichia coli</i>. <i>Nat. Chem. Biol.</i> <b>13</b>, 706–708 (2017).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/nchembio.2390" data-track-action="article reference" href="https://doi.org/10.1038%2Fnchembio.2390" aria-label="Article reference 28" data-doi="10.1038/nchembio.2390">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXotVOltL0%3D" aria-label="CAS reference 28">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28530708" aria-label="PubMed reference 28">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 28" href="http://scholar.google.com/scholar_lookup?&amp;title=Engineering%20RGB%20color%20vision%20into%20Escherichia%20coli&amp;journal=Nat.%20Chem.%20Biol.&amp;doi=10.1038%2Fnchembio.2390&amp;volume=13&amp;pages=706-708&amp;publication_year=2017&amp;author=Fernandez-Rodriguez%2CJ&amp;author=Moser%2CF&amp;author=Song%2CM&amp;author=Voigt%2CCA">
                    Google Scholar</a> 
                </p></li><li data-counter="29."><p id="ref-CR29">Jayaraman, P. et al. Blue light-mediated transcriptional activation and repression of gene expression in bacteria. <i>Nucleic Acids Res</i>. <b>44</b>, 6994–7005 (2016).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1093/nar/gkw548" data-track-action="article reference" href="https://doi.org/10.1093%2Fnar%2Fgkw548" aria-label="Article reference 29" data-doi="10.1093/nar/gkw548">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XhslWrsLzF" aria-label="CAS reference 29">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27353329" aria-label="PubMed reference 29">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5001607" aria-label="PubMed Central reference 29">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 29" href="http://scholar.google.com/scholar_lookup?&amp;title=Blue%20light-mediated%20transcriptional%20activation%20and%20repression%20of%20gene%20expression%20in%20bacteria&amp;journal=Nucleic%20Acids%20Res&amp;doi=10.1093%2Fnar%2Fgkw548&amp;volume=44&amp;pages=6994-7005&amp;publication_year=2016&amp;author=Jayaraman%2CP">
                    Google Scholar</a> 
                </p></li><li data-counter="30."><p id="ref-CR30">Pouzet, S. et al. The promise of optogenetics for bioproduction: dynamic control strategies and scale-up instruments. <i>Bioengineering</i> <b>7</b>, 151 (2020).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.3390/bioengineering7040151" data-track-action="article reference" href="https://doi.org/10.3390%2Fbioengineering7040151" aria-label="Article reference 30" data-doi="10.3390/bioengineering7040151">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXmsVSjsro%3D" aria-label="CAS reference 30">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33255280" aria-label="PubMed reference 30">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7712799" aria-label="PubMed Central reference 30">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 30" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20promise%20of%20optogenetics%20for%20bioproduction%3A%20dynamic%20control%20strategies%20and%20scale-up%20instruments&amp;journal=Bioengineering&amp;doi=10.3390%2Fbioengineering7040151&amp;volume=7&amp;publication_year=2020&amp;author=Pouzet%2CS">
                    Google Scholar</a> 
                </p></li><li data-counter="31."><p id="ref-CR31">Mansouri, M. &amp; Fussenegger, M. Synthetic biology-based optogenetic approaches to control therapeutic designer cells. <i>Curr. Opin. Syst. Biol.</i> <b>28</b>, 100396 (2021).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.coisb.2021.100396" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.coisb.2021.100396" aria-label="Article reference 31" data-doi="10.1016/j.coisb.2021.100396">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XhtlSmt7fN" aria-label="CAS reference 31">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 31" href="http://scholar.google.com/scholar_lookup?&amp;title=Synthetic%20biology-based%20optogenetic%20approaches%20to%20control%20therapeutic%20designer%20cells&amp;journal=Curr.%20Opin.%20Syst.%20Biol.&amp;doi=10.1016%2Fj.coisb.2021.100396&amp;volume=28&amp;publication_year=2021&amp;author=Mansouri%2CM&amp;author=Fussenegger%2CM">
                    Google Scholar</a> 
                </p></li><li data-counter="32."><p id="ref-CR32">Korbie, D. J. &amp; Mattick, J. S. Touchdown PCR for increased specificity and sensitivity in PCR amplification. <i>Nat. Protoc.</i> <b>3</b>, 1452–1456 (2008).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/nprot.2008.133" data-track-action="article reference" href="https://doi.org/10.1038%2Fnprot.2008.133" aria-label="Article reference 32" data-doi="10.1038/nprot.2008.133">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXhtVOnsL7F" aria-label="CAS reference 32">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18772872" aria-label="PubMed reference 32">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 32" href="http://scholar.google.com/scholar_lookup?&amp;title=Touchdown%20PCR%20for%20increased%20specificity%20and%20sensitivity%20in%20PCR%20amplification&amp;journal=Nat.%20Protoc.&amp;doi=10.1038%2Fnprot.2008.133&amp;volume=3&amp;pages=1452-1456&amp;publication_year=2008&amp;author=Korbie%2CDJ&amp;author=Mattick%2CJS">
                    Google Scholar</a> 
                </p></li><li data-counter="33."><p id="ref-CR33">Grass, R. N., Heckel, R., Puddu, M., Paunescu, D. &amp; Stark, W. J. Robust chemical preservation of digital information on DNA in silica with error-correcting codes. <i>Angew. Chem. Int. Ed.</i> <b>54</b>, 2552–2555 (2015).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1002/anie.201411378" data-track-action="article reference" href="https://doi.org/10.1002%2Fanie.201411378" aria-label="Article reference 33" data-doi="10.1002/anie.201411378">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXis1Snsr8%3D" aria-label="CAS reference 33">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 33" href="http://scholar.google.com/scholar_lookup?&amp;title=Robust%20chemical%20preservation%20of%20digital%20information%20on%20DNA%20in%20silica%20with%20error-correcting%20codes&amp;journal=Angew.%20Chem.%20Int.%20Ed.&amp;doi=10.1002%2Fanie.201411378&amp;volume=54&amp;pages=2552-2555&amp;publication_year=2015&amp;author=Grass%2CRN&amp;author=Heckel%2CR&amp;author=Puddu%2CM&amp;author=Paunescu%2CD&amp;author=Stark%2CWJ">
                    Google Scholar</a> 
                </p></li><li data-counter="34."><p id="ref-CR34">Breunig, M. M., Kriegel, H.-P., Ng, R. T. &amp; Sander, J. LOF: identifying density-based local outliers. <i>ACM SIGMOD Rec.</i> <b>29</b>, 93–104 (2000).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1145/335191.335388" data-track-action="article reference" href="https://doi.org/10.1145%2F335191.335388" aria-label="Article reference 34" data-doi="10.1145/335191.335388">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 34" href="http://scholar.google.com/scholar_lookup?&amp;title=LOF%3A%20identifying%20density-based%20local%20outliers&amp;journal=ACM%20SIGMOD%20Rec.&amp;doi=10.1145%2F335191.335388&amp;volume=29&amp;pages=93-104&amp;publication_year=2000&amp;author=Breunig%2CMM&amp;author=Kriegel%2CH-P&amp;author=Ng%2CRT&amp;author=Sander%2CJ">
                    Google Scholar</a> 
                </p></li><li data-counter="35."><p id="ref-CR35">Hochrein, L., Mitchell, L. A., Schulz, K., Messerschmidt, K. &amp; Mueller-Roeber, B. L-SCRaMbLE as a tool for light-controlled Cre-mediated recombination in yeast. <i>Nat. Commun.</i> <b>9</b>, 1931 (2018).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/s41467-017-02208-6" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41467-017-02208-6" aria-label="Article reference 35" data-doi="10.1038/s41467-017-02208-6">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2018NatCo...9.1931H" aria-label="ADS reference 35">ADS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29789561" aria-label="PubMed reference 35">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5964156" aria-label="PubMed Central reference 35">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 35" href="http://scholar.google.com/scholar_lookup?&amp;title=L-SCRaMbLE%20as%20a%20tool%20for%20light-controlled%20Cre-mediated%20recombination%20in%20yeast&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-017-02208-6&amp;volume=9&amp;publication_year=2018&amp;author=Hochrein%2CL&amp;author=Mitchell%2CLA&amp;author=Schulz%2CK&amp;author=Messerschmidt%2CK&amp;author=Mueller-Roeber%2CB">
                    Google Scholar</a> 
                </p></li><li data-counter="36."><p id="ref-CR36">Chen, B. et al. High-throughput analysis and protein engineering using microcapillary arrays. <i>Nat. Chem. Biol.</i> <b>12</b>, 76–81 (2016).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/nchembio.1978" data-track-action="article reference" href="https://doi.org/10.1038%2Fnchembio.1978" aria-label="Article reference 36" data-doi="10.1038/nchembio.1978">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXhvFemsbrL" aria-label="CAS reference 36">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26641932" aria-label="PubMed reference 36">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 36" href="http://scholar.google.com/scholar_lookup?&amp;title=High-throughput%20analysis%20and%20protein%20engineering%20using%20microcapillary%20arrays&amp;journal=Nat.%20Chem.%20Biol.&amp;doi=10.1038%2Fnchembio.1978&amp;volume=12&amp;pages=76-81&amp;publication_year=2016&amp;author=Chen%2CB">
                    Google Scholar</a> 
                </p></li><li data-counter="37."><p id="ref-CR37">Zhang, Y. et al. Accurate high-throughput screening based on digital protein synthesis in a massively parallel femtoliter droplet array. <i>Sci. Adv.</i> <b>5</b>, eaav8185 (2019).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1126/sciadv.aav8185" data-track-action="article reference" href="https://doi.org/10.1126%2Fsciadv.aav8185" aria-label="Article reference 37" data-doi="10.1126/sciadv.aav8185">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2019SciA....5.8185Z" aria-label="ADS reference 37">ADS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhtlars7%2FM" aria-label="CAS reference 37">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31457078" aria-label="PubMed reference 37">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6703874" aria-label="PubMed Central reference 37">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 37" href="http://scholar.google.com/scholar_lookup?&amp;title=Accurate%20high-throughput%20screening%20based%20on%20digital%20protein%20synthesis%20in%20a%20massively%20parallel%20femtoliter%20droplet%20array&amp;journal=Sci.%20Adv.&amp;doi=10.1126%2Fsciadv.aav8185&amp;volume=5&amp;publication_year=2019&amp;author=Zhang%2CY">
                    Google Scholar</a> 
                </p></li><li data-counter="38."><p id="ref-CR38">Antkowiak, P. L. et al. Integrating DNA encapsulates and digital microfluidics for automated data storage in DNA. <i>Small</i> <b>18</b>, 2107381 (2022).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1002/smll.202107381" data-track-action="article reference" href="https://doi.org/10.1002%2Fsmll.202107381" aria-label="Article reference 38" data-doi="10.1002/smll.202107381">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XlvVaksrk%3D" aria-label="CAS reference 38">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 38" href="http://scholar.google.com/scholar_lookup?&amp;title=Integrating%20DNA%20encapsulates%20and%20digital%20microfluidics%20for%20automated%20data%20storage%20in%20DNA&amp;journal=Small&amp;doi=10.1002%2Fsmll.202107381&amp;volume=18&amp;publication_year=2022&amp;author=Antkowiak%2CPL">
                    Google Scholar</a> 
                </p></li><li data-counter="39."><p id="ref-CR39">Lin, K. N., Volkel, K., Tuck, J. M. &amp; Keung, A. J. Dynamic and scalable DNA-based information storage. <i>Nat. Commun.</i> <b>11</b>, 2981 (2020).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/s41467-020-16797-2" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41467-020-16797-2" aria-label="Article reference 39" data-doi="10.1038/s41467-020-16797-2">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2020NatCo..11.2981L" aria-label="ADS reference 39">ADS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhtFylsL3I" aria-label="CAS reference 39">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32532979" aria-label="PubMed reference 39">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7293219" aria-label="PubMed Central reference 39">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 39" href="http://scholar.google.com/scholar_lookup?&amp;title=Dynamic%20and%20scalable%20DNA-based%20information%20storage&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-020-16797-2&amp;volume=11&amp;publication_year=2020&amp;author=Lin%2CKN&amp;author=Volkel%2CK&amp;author=Tuck%2CJM&amp;author=Keung%2CAJ">
                    Google Scholar</a> 
                </p></li><li data-counter="40."><p id="ref-CR40">Banal, J. L. et al. Random access DNA memory using Boolean search in an archival file storage system. <i>Nat. Mater</i>. 1–9. <a href="https://doi.org/10.1038/s41563-021-01021-3">https://doi.org/10.1038/s41563-021-01021-3</a> (2021).</p></li><li data-counter="41."><p id="ref-CR41">Bee, C. et al. Molecular-level similarity search brings computing to DNA data storage. <i>Nat. Commun.</i> <b>12</b>, 4764 (2021).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/s41467-021-24991-z" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41467-021-24991-z" aria-label="Article reference 41" data-doi="10.1038/s41467-021-24991-z">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2021NatCo..12.4764B" aria-label="ADS reference 41">ADS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhvVSls73J" aria-label="CAS reference 41">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34362913" aria-label="PubMed reference 41">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8346626" aria-label="PubMed Central reference 41">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 41" href="http://scholar.google.com/scholar_lookup?&amp;title=Molecular-level%20similarity%20search%20brings%20computing%20to%20DNA%20data%20storage&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-021-24991-z&amp;volume=12&amp;publication_year=2021&amp;author=Bee%2CC">
                    Google Scholar</a> 
                </p></li><li data-counter="42."><p id="ref-CR42">Buschmann, T. &amp; Bystrykh, L. V. Levenshtein error-correcting barcodes for multiplexed DNA sequencing. <i>BMC Bioinforma.</i> <b>14</b>, 272 (2013).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1186/1471-2105-14-272" data-track-action="article reference" href="https://doi.org/10.1186%2F1471-2105-14-272" aria-label="Article reference 42" data-doi="10.1186/1471-2105-14-272">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 42" href="http://scholar.google.com/scholar_lookup?&amp;title=Levenshtein%20error-correcting%20barcodes%20for%20multiplexed%20DNA%20sequencing&amp;journal=BMC%20Bioinforma.&amp;doi=10.1186%2F1471-2105-14-272&amp;volume=14&amp;publication_year=2013&amp;author=Buschmann%2CT&amp;author=Bystrykh%2CLV">
                    Google Scholar</a> 
                </p></li><li data-counter="43."><p id="ref-CR43">R Core Team. R: the R project for statistical computing. <a href="https://www.r-project.org/">https://www.r-project.org/</a> (2022).</p></li><li data-counter="44."><p id="ref-CR44">Huber, W. et al. Orchestrating high-throughput genomic analysis with Bioconductor. <i>Nat. Methods</i> <b>12</b>, 115–121 (2015).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/nmeth.3252" data-track-action="article reference" href="https://doi.org/10.1038%2Fnmeth.3252" aria-label="Article reference 44" data-doi="10.1038/nmeth.3252">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXjvVCrurg%3D" aria-label="CAS reference 44">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25633503" aria-label="PubMed reference 44">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4509590" aria-label="PubMed Central reference 44">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 44" href="http://scholar.google.com/scholar_lookup?&amp;title=Orchestrating%20high-throughput%20genomic%20analysis%20with%20Bioconductor&amp;journal=Nat.%20Methods&amp;doi=10.1038%2Fnmeth.3252&amp;volume=12&amp;pages=115-121&amp;publication_year=2015&amp;author=Huber%2CW">
                    Google Scholar</a> 
                </p></li><li data-counter="45."><p id="ref-CR45">Pagès, H., Aboyoun, P., Gentleman, R. &amp; DebRoy, S. Biostrings: efficient manipulation of biological strings. <a href="https://doi.org/10.18129/B9.bioc.Biostrings">https://doi.org/10.18129/B9.bioc.Biostrings</a> (2023).</p></li><li data-counter="46."><p id="ref-CR46">Morgan, M. et al. ShortRead: a bioconductor package for input, quality assessment and exploration of high-throughput sequence data. <i>Bioinformatics</i> <b>25</b>, 2607–2608 (2009).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1093/bioinformatics/btp450" data-track-action="article reference" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtp450" aria-label="Article reference 46" data-doi="10.1093/bioinformatics/btp450">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXhtFyhur7I" aria-label="CAS reference 46">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19654119" aria-label="PubMed reference 46">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2752612" aria-label="PubMed Central reference 46">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 46" href="http://scholar.google.com/scholar_lookup?&amp;title=ShortRead%3A%20a%20bioconductor%20package%20for%20input%2C%20quality%20assessment%20and%20exploration%20of%20high-throughput%20sequence%20data&amp;journal=Bioinformatics&amp;doi=10.1093%2Fbioinformatics%2Fbtp450&amp;volume=25&amp;pages=2607-2608&amp;publication_year=2009&amp;author=Morgan%2CM">
                    Google Scholar</a> 
                </p></li><li data-counter="47."><p id="ref-CR47">Wickham, H. stringr: Simple, Consistent Wrappers for Common String Operations. <a href="https://stringr.tidyverse.org">https://stringr.tidyverse.org</a>, <a href="https://github.com/tidyverse/stringr">https://github.com/tidyverse/stringr</a> (2022).</p></li><li data-counter="48."><p id="ref-CR48">The Python Language Reference. <i>Python documentation.</i> <a href="https://docs.python.org/3/reference/index.html">https://docs.python.org/3/reference/index.html</a>.</p></li><li data-counter="49."><p id="ref-CR49">Arthur, D. &amp; Vassilvitskii, S. k-means++: the advantages of careful seeding. In <i>Proceedings of the eighteenth annual ACM-SIAM symposium on Discrete algorithms</i> 1027–1035 (Society for Industrial and Applied Mathematics, 2007).</p></li><li data-counter="50."><p id="ref-CR50">Schubert, E., Sander, J., Ester, M., Kriegel, H. P. &amp; Xu, X. DBSCAN revisited, revisited: why and how you should (Still) use DBSCAN. <i>ACM Trans. Database Syst.</i> <b>42</b>, 1–21 (2017).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1145/3068335" data-track-action="article reference" href="https://doi.org/10.1145%2F3068335" aria-label="Article reference 50" data-doi="10.1145/3068335">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="mathscinet reference" href="http://www.ams.org/mathscinet-getitem?mr=3622239" aria-label="MathSciNet reference 50">MathSciNet</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 50" href="http://scholar.google.com/scholar_lookup?&amp;title=DBSCAN%20revisited%2C%20revisited%3A%20why%20and%20how%20you%20should%20%28Still%29%20use%20DBSCAN&amp;journal=ACM%20Trans.%20Database%20Syst.&amp;doi=10.1145%2F3068335&amp;volume=42&amp;pages=1-21&amp;publication_year=2017&amp;author=Schubert%2CE&amp;author=Sander%2CJ&amp;author=Ester%2CM&amp;author=Kriegel%2CHP&amp;author=Xu%2CX">
                    Google Scholar</a> 
                </p></li><li data-counter="51."><p id="ref-CR51">Ankerst, M., Breunig, M. M., Kriegel, H.-P. &amp; Sander, J. OPTICS: ordering points to identify the clustering structure. <i>ACM SIGMOD Rec.</i> <b>28</b>, 49–60 (1999).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1145/304181.304187" data-track-action="article reference" href="https://doi.org/10.1145%2F304181.304187" aria-label="Article reference 51" data-doi="10.1145/304181.304187">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 51" href="http://scholar.google.com/scholar_lookup?&amp;title=OPTICS%3A%20ordering%20points%20to%20identify%20the%20clustering%20structure&amp;journal=ACM%20SIGMOD%20Rec.&amp;doi=10.1145%2F304181.304187&amp;volume=28&amp;pages=49-60&amp;publication_year=1999&amp;author=Ankerst%2CM&amp;author=Breunig%2CMM&amp;author=Kriegel%2CH-P&amp;author=Sander%2CJ">
                    Google Scholar</a> 
                </p></li><li data-counter="52."><p id="ref-CR52">Yang, M.-S., Lai, C.-Y. &amp; Lin, C.-Y. A robust EM clustering algorithm for Gaussian mixture models. <i>Pattern Recognit.</i> <b>45</b>, 3950–3961 (2012).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.patcog.2012.04.031" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.patcog.2012.04.031" aria-label="Article reference 52" data-doi="10.1016/j.patcog.2012.04.031">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2012PatRe..45.3950Y" aria-label="ADS reference 52">ADS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="math reference" href="http://www.emis.de/MATH-item?1242.68260" aria-label="MATH reference 52">MATH</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 52" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20robust%20EM%20clustering%20algorithm%20for%20Gaussian%20mixture%20models&amp;journal=Pattern%20Recognit.&amp;doi=10.1016%2Fj.patcog.2012.04.031&amp;volume=45&amp;pages=3950-3961&amp;publication_year=2012&amp;author=Yang%2CM-S&amp;author=Lai%2CC-Y&amp;author=Lin%2CC-Y">
                    Google Scholar</a> 
                </p></li><li data-counter="53."><p id="ref-CR53">Pedregosa, F. et al. Scikit-learn: Machine Learning in Python. <i>J. Mach. Learn. Res</i>. <b>12</b>, 2825–2830 (2011).</p></li></ol><p><a data-track="click" data-track-action="download citation references" data-track-label="link" rel="nofollow" href="https://citation-needed.springer.com/v2/references/10.1038/s41467-023-38876-w?format=refman&amp;flavour=references">Download references<svg width="16" height="16" focusable="false" role="img" aria-hidden="true"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-download"></use></svg></a></p></div></div></div></section></div><section data-title="Acknowledgements"><div id="Ack1-section"><h2 id="Ack1">Acknowledgements</h2><p>We thank the Yew and Poh Lab members for constructive criticism critical to the research. This work was supported by Ministry of Education, Singapore, under its AcRF Tier 2 Grant (MOE-T2EP30221-0014). C.K.L. is funded by the NUS Integrative Sciences and Engineering Programme.</p></div></section><section aria-labelledby="author-information" data-title="Author information"><div id="author-information-section"><h2 id="author-information">Author information</h2><div id="author-information-content"><h3 id="affiliations">Authors and Affiliations</h3><ol><li id="Aff1"><p>Synthetic Biology for Clinical and Technological Innovation, National University of Singapore, 28 Medical Drive, Singapore, 117456, Singapore</p><p>Cheng Kai Lim, Jing Wui Yeoh, Aurelius Andrew Kunartama, Wen Shan Yew &amp; Chueh Loo Poh</p></li><li id="Aff2"><p>Synthetic Biology Translational Research Programme, Yong Loo Lin School of Medicine, National University of Singapore, 14 Medical Drive, Singapore, 117599, Singapore</p><p>Cheng Kai Lim &amp; Wen Shan Yew</p></li><li id="Aff3"><p>Department of Biochemistry, Yong Loo Lin School of Medicine, National University of Singapore, 8 Medical Drive, Singapore, 117597, Singapore</p><p>Cheng Kai Lim &amp; Wen Shan Yew</p></li><li id="Aff4"><p>Department of Biomedical Engineering, College of Design and Engineering, National University of Singapore, Singapore, Singapore</p><p>Cheng Kai Lim, Jing Wui Yeoh, Aurelius Andrew Kunartama &amp; Chueh Loo Poh</p></li><li id="Aff5"><p>Integrative Sciences and Engineering Programme (ISEP), NUS Graduate School, National University of Singapore, Singapore, Singapore</p><p>Cheng Kai Lim</p></li></ol><h3 id="contributions">Contributions</h3><p>C.K.L. and C.L.P. developed the initial concept. C.K.L. performed the experiments and analyzed the results under the supervision of C.L.P. and W.S.Y. C.K.L., J.W.Y. and C.L.P. developed and constructed the computational pipeline for image reconstruction. A.A.K. designed and developed the OptoBox device. C.K.L., J.W.Y., A.A.K. and C.L.P. wrote the manuscript with input from all authors.</p><h3 id="corresponding-author">Corresponding author</h3><p id="corresponding-author-list">Correspondence to
                <a id="corresp-c1" href="mailto:poh.chuehloo@nus.edu.sg">Chueh Loo Poh</a>.</p></div></div></section><section data-title="Ethics declarations"><div id="ethics-section"><h2 id="ethics">Ethics declarations</h2><div id="ethics-content">
              
                <h3 id="FPar2">Competing interests</h3>
                <p>The authors declare no competing interests.</p>
              
            </div></div></section><section data-title="Peer review"><div id="peer-review-section"><h2 id="peer-review">Peer review</h2><div id="peer-review-content">
              
              
                <h3 id="FPar1">Peer review information</h3>
                <p><i>Nature Communications</i> thanks the anonymous reviewer(s) for their contribution to the peer review of this work. A peer review file is available.</p>
              
            </div></div></section><section data-title="Additional information"><div id="additional-information-section"><h2 id="additional-information">Additional information</h2><p><b>Publisher’s note</b> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div></section><section data-title="Supplementary information"><div id="Sec23-section"><h2 id="Sec23">Supplementary information</h2></div></section><section data-title="Source data"></section><section data-title="Rights and permissions"><div id="rightslink-section"><h2 id="rightslink">Rights and permissions</h2><div id="rightslink-content">
                <p><b>Open Access</b>  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this license, visit <a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>.</p>
              <p><a data-track="click" data-track-action="view rights and permissions" data-track-label="link" href="https://s100.copyright.com/AppDispatchServlet?title=A%20biological%20camera%20that%20captures%20and%20stores%20images%20directly%20into%20DNA&amp;author=Cheng%20Kai%20Lim%20et%20al&amp;contentID=10.1038%2Fs41467-023-38876-w&amp;copyright=The%20Author%28s%29&amp;publication=2041-1723&amp;publicationDate=2023-07-03&amp;publisherName=SpringerNature&amp;orderBeanReset=true&amp;oa=CC%20BY">Reprints and Permissions</a></p></div></div></section><section aria-labelledby="article-info" data-title="About this article"><div id="article-info-section"><h2 id="article-info">About this article</h2><div id="article-info-content"><div><p><a data-crossmark="10.1038/s41467-023-38876-w" target="_blank" rel="noopener" href="https://crossmark.crossref.org/dialog/?doi=10.1038/s41467-023-38876-w" data-track="click" data-track-action="Click Crossmark" data-track-label="link" data-test="crossmark"><img width="57" height="81" alt="Check for updates. Verify currency and authenticity via CrossMark" src="data:image/svg+xml;base64,<svg height="81" width="57" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="m17.35 35.45 21.3-14.2v-17.03h-21.3" fill="#989898"/><path d="m38.65 35.45-21.3-14.2v-17.03h21.3" fill="#747474"/><path d="m28 .5c-12.98 0-23.5 10.52-23.5 23.5s10.52 23.5 23.5 23.5 23.5-10.52 23.5-23.5c0-6.23-2.48-12.21-6.88-16.62-4.41-4.4-10.39-6.88-16.62-6.88zm0 41.25c-9.8 0-17.75-7.95-17.75-17.75s7.95-17.75 17.75-17.75 17.75 7.95 17.75 17.75c0 4.71-1.87 9.22-5.2 12.55s-7.84 5.2-12.55 5.2z" fill="#535353"/><path d="m41 36c-5.81 6.23-15.23 7.45-22.43 2.9-7.21-4.55-10.16-13.57-7.03-21.5l-4.92-3.11c-4.95 10.7-1.19 23.42 8.78 29.71 9.97 6.3 23.07 4.22 30.6-4.86z" fill="#9c9c9c"/><path d="m.2 58.45c0-.75.11-1.42.33-2.01s.52-1.09.91-1.5c.38-.41.83-.73 1.34-.94.51-.22 1.06-.32 1.65-.32.56 0 1.06.11 1.51.35.44.23.81.5 1.1.81l-.91 1.01c-.24-.24-.49-.42-.75-.56-.27-.13-.58-.2-.93-.2-.39 0-.73.08-1.05.23-.31.16-.58.37-.81.66-.23.28-.41.63-.53 1.04-.13.41-.19.88-.19 1.39 0 1.04.23 1.86.68 2.46.45.59 1.06.88 1.84.88.41 0 .77-.07 1.07-.23s.59-.39.85-.68l.91 1c-.38.43-.8.76-1.28.99-.47.22-1 .34-1.58.34-.59 0-1.13-.1-1.64-.31-.5-.2-.94-.51-1.31-.91-.38-.4-.67-.9-.88-1.48-.22-.59-.33-1.26-.33-2.02zm8.4-5.33h1.61v2.54l-.05 1.33c.29-.27.61-.51.96-.72s.76-.31 1.24-.31c.73 0 1.27.23 1.61.71.33.47.5 1.14.5 2.02v4.31h-1.61v-4.1c0-.57-.08-.97-.25-1.21-.17-.23-.45-.35-.83-.35-.3 0-.56.08-.79.22-.23.15-.49.36-.78.64v4.8h-1.61zm7.37 6.45c0-.56.09-1.06.26-1.51.18-.45.42-.83.71-1.14.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.36c.07.62.29 1.1.65 1.44.36.33.82.5 1.38.5.29 0 .57-.04.83-.13s.51-.21.76-.37l.55 1.01c-.33.21-.69.39-1.09.53-.41.14-.83.21-1.26.21-.48 0-.92-.08-1.34-.25-.41-.16-.76-.4-1.07-.7-.31-.31-.55-.69-.72-1.13-.18-.44-.26-.95-.26-1.52zm4.6-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.07.45-.31.29-.5.73-.58 1.3zm2.5.62c0-.57.09-1.08.28-1.53.18-.44.43-.82.75-1.13s.69-.54 1.1-.71c.42-.16.85-.24 1.31-.24.45 0 .84.08 1.17.23s.61.34.85.57l-.77 1.02c-.19-.16-.38-.28-.56-.37-.19-.09-.39-.14-.61-.14-.56 0-1.01.21-1.35.63-.35.41-.52.97-.52 1.67 0 .69.17 1.24.51 1.66.34.41.78.62 1.32.62.28 0 .54-.06.78-.17.24-.12.45-.26.64-.42l.67 1.03c-.33.29-.69.51-1.08.65-.39.15-.78.23-1.18.23-.46 0-.9-.08-1.31-.24-.4-.16-.75-.39-1.05-.7s-.53-.69-.7-1.13c-.17-.45-.25-.96-.25-1.53zm6.91-6.45h1.58v6.17h.05l2.54-3.16h1.77l-2.35 2.8 2.59 4.07h-1.75l-1.77-2.98-1.08 1.23v1.75h-1.58zm13.69 1.27c-.25-.11-.5-.17-.75-.17-.58 0-.87.39-.87 1.16v.75h1.34v1.27h-1.34v5.6h-1.61v-5.6h-.92v-1.2l.92-.07v-.72c0-.35.04-.68.13-.98.08-.31.21-.57.4-.79s.42-.39.71-.51c.28-.12.63-.18 1.04-.18.24 0 .48.02.69.07.22.05.41.1.57.17zm.48 5.18c0-.57.09-1.08.27-1.53.17-.44.41-.82.72-1.13.3-.31.65-.54 1.04-.71.39-.16.8-.24 1.23-.24s.84.08 1.24.24c.4.17.74.4 1.04.71s.54.69.72 1.13c.19.45.28.96.28 1.53s-.09 1.08-.28 1.53c-.18.44-.42.82-.72 1.13s-.64.54-1.04.7-.81.24-1.24.24-.84-.08-1.23-.24-.74-.39-1.04-.7c-.31-.31-.55-.69-.72-1.13-.18-.45-.27-.96-.27-1.53zm1.65 0c0 .69.14 1.24.43 1.66.28.41.68.62 1.18.62.51 0 .9-.21 1.19-.62.29-.42.44-.97.44-1.66 0-.7-.15-1.26-.44-1.67-.29-.42-.68-.63-1.19-.63-.5 0-.9.21-1.18.63-.29.41-.43.97-.43 1.67zm6.48-3.44h1.33l.12 1.21h.05c.24-.44.54-.79.88-1.02.35-.24.7-.36 1.07-.36.32 0 .59.05.78.14l-.28 1.4-.33-.09c-.11-.01-.23-.02-.38-.02-.27 0-.56.1-.86.31s-.55.58-.77 1.1v4.2h-1.61zm-47.87 15h1.61v4.1c0 .57.08.97.25 1.2.17.24.44.35.81.35.3 0 .57-.07.8-.22.22-.15.47-.39.73-.73v-4.7h1.61v6.87h-1.32l-.12-1.01h-.04c-.3.36-.63.64-.98.86-.35.21-.76.32-1.24.32-.73 0-1.27-.24-1.61-.71-.33-.47-.5-1.14-.5-2.02zm9.46 7.43v2.16h-1.61v-9.59h1.33l.12.72h.05c.29-.24.61-.45.97-.63.35-.17.72-.26 1.1-.26.43 0 .81.08 1.15.24.33.17.61.4.84.71.24.31.41.68.53 1.11.13.42.19.91.19 1.44 0 .59-.09 1.11-.25 1.57-.16.47-.38.85-.65 1.16-.27.32-.58.56-.94.73-.35.16-.72.25-1.1.25-.3 0-.6-.07-.9-.2s-.59-.31-.87-.56zm0-2.3c.26.22.5.37.73.45.24.09.46.13.66.13.46 0 .84-.2 1.15-.6.31-.39.46-.98.46-1.77 0-.69-.12-1.22-.35-1.61-.23-.38-.61-.57-1.13-.57-.49 0-.99.26-1.52.77zm5.87-1.69c0-.56.08-1.06.25-1.51.16-.45.37-.83.65-1.14.27-.3.58-.54.93-.71s.71-.25 1.08-.25c.39 0 .73.07 1 .2.27.14.54.32.81.55l-.06-1.1v-2.49h1.61v9.88h-1.33l-.11-.74h-.06c-.25.25-.54.46-.88.64-.33.18-.69.27-1.06.27-.87 0-1.56-.32-2.07-.95s-.76-1.51-.76-2.65zm1.67-.01c0 .74.13 1.31.4 1.7.26.38.65.58 1.15.58.51 0 .99-.26 1.44-.77v-3.21c-.24-.21-.48-.36-.7-.45-.23-.08-.46-.12-.7-.12-.45 0-.82.19-1.13.59-.31.39-.46.95-.46 1.68zm6.35 1.59c0-.73.32-1.3.97-1.71.64-.4 1.67-.68 3.08-.84 0-.17-.02-.34-.07-.51-.05-.16-.12-.3-.22-.43s-.22-.22-.38-.3c-.15-.06-.34-.1-.58-.1-.34 0-.68.07-1 .2s-.63.29-.93.47l-.59-1.08c.39-.24.81-.45 1.28-.63.47-.17.99-.26 1.54-.26.86 0 1.51.25 1.93.76s.63 1.25.63 2.21v4.07h-1.32l-.12-.76h-.05c-.3.27-.63.48-.98.66s-.73.27-1.14.27c-.61 0-1.1-.19-1.48-.56-.38-.36-.57-.85-.57-1.46zm1.57-.12c0 .3.09.53.27.67.19.14.42.21.71.21.28 0 .54-.07.77-.2s.48-.31.73-.56v-1.54c-.47.06-.86.13-1.18.23-.31.09-.57.19-.76.31s-.33.25-.41.4c-.09.15-.13.31-.13.48zm6.29-3.63h-.98v-1.2l1.06-.07.2-1.88h1.34v1.88h1.75v1.27h-1.75v3.28c0 .8.32 1.2.97 1.2.12 0 .24-.01.37-.04.12-.03.24-.07.34-.11l.28 1.19c-.19.06-.4.12-.64.17-.23.05-.49.08-.76.08-.4 0-.74-.06-1.02-.18-.27-.13-.49-.3-.67-.52-.17-.21-.3-.48-.37-.78-.08-.3-.12-.64-.12-1.01zm4.36 2.17c0-.56.09-1.06.27-1.51s.41-.83.71-1.14c.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.37c.08.62.29 1.1.65 1.44.36.33.82.5 1.38.5.3 0 .58-.04.84-.13.25-.09.51-.21.76-.37l.54 1.01c-.32.21-.69.39-1.09.53s-.82.21-1.26.21c-.47 0-.92-.08-1.33-.25-.41-.16-.77-.4-1.08-.7-.3-.31-.54-.69-.72-1.13-.17-.44-.26-.95-.26-1.52zm4.61-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.08.45-.31.29-.5.73-.57 1.3zm3.01 2.23c.31.24.61.43.92.57.3.13.63.2.98.2.38 0 .65-.08.83-.23s.27-.35.27-.6c0-.14-.05-.26-.13-.37-.08-.1-.2-.2-.34-.28-.14-.09-.29-.16-.47-.23l-.53-.22c-.23-.09-.46-.18-.69-.3-.23-.11-.44-.24-.62-.4s-.33-.35-.45-.55c-.12-.21-.18-.46-.18-.75 0-.61.23-1.1.68-1.49.44-.38 1.06-.57 1.83-.57.48 0 .91.08 1.29.25s.71.36.99.57l-.74.98c-.24-.17-.49-.32-.73-.42-.25-.11-.51-.16-.78-.16-.35 0-.6.07-.76.21-.17.15-.25.33-.25.54 0 .14.04.26.12.36s.18.18.31.26c.14.07.29.14.46.21l.54.19c.23.09.47.18.7.29s.44.24.64.4c.19.16.34.35.46.58.11.23.17.5.17.82 0 .3-.06.58-.17.83-.12.26-.29.48-.51.68-.23.19-.51.34-.84.45-.34.11-.72.17-1.15.17-.48 0-.95-.09-1.41-.27-.46-.19-.86-.41-1.2-.68z" fill="#535353"/></g></svg>"/></a></p><div><h3 id="citeas">Cite this article</h3><p>Lim, C.K., Yeoh, J.W., Kunartama, A.A. <i>et al.</i> A biological camera that captures and stores images directly into DNA.
                    <i>Nat Commun</i> <b>14</b>, 3921 (2023). https://doi.org/10.1038/s41467-023-38876-w</p><p><a data-test="citation-link" data-track="click" data-track-action="download article citation" data-track-label="link" data-track-external="" rel="nofollow" href="https://citation-needed.springer.com/v2/references/10.1038/s41467-023-38876-w?format=refman&amp;flavour=citation">Download citation<svg width="16" height="16" focusable="false" role="img" aria-hidden="true"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-download"></use></svg></a></p><ul data-test="publication-history"><li><p>Received<span>: </span><span><time datetime="2022-04-19">19 April 2022</time></span></p></li><li><p>Accepted<span>: </span><span><time datetime="2023-05-19">19 May 2023</time></span></p></li><li><p>Published<span>: </span><span><time datetime="2023-07-03">03 July 2023</time></span></p></li><li><p><abbr title="Digital Object Identifier">DOI</abbr><span>: </span><span>https://doi.org/10.1038/s41467-023-38876-w</span></p></li></ul></div></div></div></div></section>
            </div></div>
  </body>
</html>
