<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <title>James Routley | Feed</title>
    <link
      rel="stylesheet"
      type="text/css"
      href="../styles.css"
      media="screen"
    />
  </head>
  <body>
    <a href="/index.html">Back</a>
    <a href="https://www.nature.com/articles/s41586-023-06806-x">Original</a>
    <h1>A human embryonic limb cell atlas resolved in space and time</h1>
    
    <div id="readability-page-1" class="page"><div>
                    <section data-title="Main"><div id="Sec1-section"><h2 id="Sec1">Main</h2><div id="Sec1-content"><p>Human limb buds emerge by the end of the fourth post-conception week (PCW) and develop to form arms and legs during the first trimester. By studying model organisms such as the mouse and the chick, it is known that development of the limb bud begins in the form of two major components. The parietal lateral plate mesodermal (LPM) cells condense into the skeletal system as well as forming tendon, fibrous and smooth muscle populations, whereas skeletal muscle progenitor (SkMP) cells migrate from the paraxial mesoderm to the limb field, forming striated muscle<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 3" title="Chevallier, A., Kieny, M. &amp; Mauger, A. Limb-somite relationship: origin of the limb musculature. J. Embryol. Exp. Morphol. 41, 245–258 (1977)." href="https://www.nature.com/articles/s41586-023-06806-x#ref-CR3" id="ref-link-section-d90000874e1111">3</a></sup>. The mesoderm is encapsulated within a thin layer of ectoderm, a subset of which (the apical ectodermal ridge) governs mesenchymal proliferation and aids in the establishment of the limb axes through fibroblast growth factor (FGF) signalling<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 4" title="Tabin, C. &amp; Wolpert, L. Rethinking the proximodistal axis of the vertebrate limb in the molecular era. Genes Dev. 21, 1433–1442 (2007)." href="https://www.nature.com/articles/s41586-023-06806-x#ref-CR4" id="ref-link-section-d90000874e1115">4</a></sup>. Limb maturation continues in a proximal–distal manner, controlled by a complex system of temporally and spatially restricted gene expression programmes, in which small perturbations can result in profound changes to the structure and function of the limb<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 1" title="Zuniga, A. Next generation limb development and evolution: old questions, new perspectives. Development 142, 3810–3820 (2015)." href="https://www.nature.com/articles/s41586-023-06806-x#ref-CR1" id="ref-link-section-d90000874e1119">1</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 5" title="McQueen, C. &amp; Towers, M. Establishing the pattern of the vertebrate limb. Development 147, dev177956 (2020)." href="https://www.nature.com/articles/s41586-023-06806-x#ref-CR5" id="ref-link-section-d90000874e1122">5</a></sup>. Indeed, approximately 1 in 500 humans are born with congenital limb malformations<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 2" title="Wilkie, A. O. M. Why study human limb malformations? J. Anat. 202, 27–35 (2003)." href="https://www.nature.com/articles/s41586-023-06806-x#ref-CR2" id="ref-link-section-d90000874e1126">2</a></sup>. Although model organisms have provided key insights into cell fates and morphogenesis, how precisely their biology translates to human development and disease remains unclear. The lack of complementary spatial information in such studies further precludes the assembly of a comprehensive tissue catalogue of human limb development.</p><p>Here we performed single-cell transcriptomic RNA sequencing (scRNA-seq) and spatial transcriptomic sequencing to detail the development of the human hindlimb (or lower limb) in space and time. We identified 67 distinct cell clusters from 125,955 captured single cells, and spatially mapped them across four first trimester timepoints to shed new light on limb development. At PCW8, we applied VisiumStitcher to map cells to a sagittal section of an entire fetal hindlimb. In addition, our spatial transcriptomic data provide insights into the key patterning events in the developing limb, with a focus on genes associated with limb malformation. We performed scRNA-seq on mouse embryonic limbs to compare limb development across species, revealing extensive homology between a classical model organism and human. Our data can be freely accessed at <a href="https://developmental.cellatlas.io/embryonic-limb">https://developmental.cellatlas.io/embryonic-limb</a>.</p></div></div></section><section data-title="Cellular heterogeneity of the developing limb in space and time"><div id="Sec2-section"><h2 id="Sec2">Cellular heterogeneity of the developing limb in space and time</h2><div id="Sec2-content"><p>To track the contribution of the different lineages to the developing limb, we collected single-cell embryonic limb profiles from PCW5 to PCW9 (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig1">1a</a> and Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41586-023-06806-x#MOESM8">5</a>). In total, we analysed 125,955 single cells that passed quality control filters and identified 67 distinct cell clusters (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig1">1b</a>, Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41586-023-06806-x#MOESM4">1</a> for marker genes and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig7">1a,b</a>). Thirty-four clusters were derived from the LPM. They contain mesenchymal, chondrocyte, osteoblast, fibroblast and smooth muscle cell states, consistent with previous studies<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 6" title="Xi, H. et al. A human skeletal muscle atlas identifies the trajectories of stem and progenitor cells across development and from human pluripotent stem cells. Cell Stem Cell 27, 158–176.e10 (2020)." href="https://www.nature.com/articles/s41586-023-06806-x#ref-CR6" id="ref-link-section-d90000874e1164">6</a></sup>. A further eight states formed the muscle lineage, derived from the somite. Other non-LPM cell clusters included haematopoietic (<i>n</i> = 14), endothelial (<i>n</i> = 3), neural crest-derived (<i>n</i> = 5) and epithelial (<i>n</i> = 3), including one apical ectodermal ridge-like cluster, which broadly expressed <i>SP8</i> and <i>WNT6</i> and contained nine <i>FGF8</i><sup>+</sup> cells originating from PCW5 and PCW6 (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig8">2a–d</a>), supported by RNA in situ hybridization (RNA-ISH) (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig8">2e</a>). The cellular composition of the developing limb changed markedly over time; progenitor states for each lineage were chiefly dominating in PCW5 and PCW6, with more differentiated cell states emerging thereafter (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig9">3a,b</a>).</p><div data-test="figure" data-container-section="figure" id="figure-1" data-title="A single-cell temporal–spatial atlas of the human embryonic limb."><figure><figcaption><b id="Fig1" data-test="figure-caption-text">Fig. 1: A single-cell temporal–spatial atlas of the human embryonic limb.</b></figcaption><div><div><a data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="https://www.nature.com/articles/s41586-023-06806-x/figures/1" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41586-023-06806-x/MediaObjects/41586_2023_6806_Fig1_HTML.png?as=webp"/><img aria-describedby="Fig1" src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41586-023-06806-x/MediaObjects/41586_2023_6806_Fig1_HTML.png" alt="figure 1" loading="lazy" width="685" height="656"/></picture></a></div><div data-test="bottom-caption" id="figure-1-desc"><p><b>a</b>, Overview of samples and the experimental scheme. The stars indicate timepoints. D, distal; M, middle; P, proximal. <b>b</b>, Uniform manifold approximation and projection (UMAP) visualization of 125,955 human embryonic limb cells with cluster labels (see <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41586-023-06806-x#MOESM1">Supplementary Information</a>). <b>c</b>,<b>d</b>, Spatially resolved heatmaps across tissue sections from the PCW6.2 (<b>c</b>) and PCW8.1 (<b>d</b>) human hindlimbs assembled from three slides showing population abundance and corresponding marker genes.</p><p><a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41586-023-06806-x#MOESM10">Source Data</a></p></div></div></figure></div><p>To give spatial context to this cellular heterogeneity, we performed spatial transcriptomic experiments using the 10x Visium assay, generating high-quality transcriptomic profiles for samples from PCW5 to PCW8 (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig7">1c</a>). We then deconvolved Visium voxels against the scRNA-seq data (see <a data-track="click" data-track-label="link" data-track-action="section anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Sec9">Methods</a>; Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig7">1d</a> for quality control). This demarcated the tissue sections into distinct regions, separating three distal progenitor populations that we named ‘distal’ (<i>LHX2</i><sup>+</sup><i>MSX1</i><sup>+</sup><i>SP9</i><sup>+</sup>), ‘<i>RDH10</i><sup>+</sup>distal’ (<i>RDH10</i><sup>+</sup><i>LHX2</i><sup>+</sup><i>MSX1</i><sup>+</sup>) and ‘transitional’ (<i>IRX1</i><sup>+</sup><i>MSX1</i><sup>+</sup>) mesenchyme (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig1">1c,d</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig10">4a–d,f,g</a>). The distal mesenchymal cells are located at the distal periphery of the limb. Proximal to it are the transitional mesenchyme together with chondrocyte progenitors of the developing autopod (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig1">1c</a>). Although all of them sit in proliferative regions (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig10">4e</a>), subtle transcriptomic differences exist, with the distal mesenchyme expressing digit patterning genes, including <i>LHX2</i> and <i>TFAP2B</i> (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig1">1c</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig10">4a–d</a>). Mutations in <i>TFAP2B</i> cause Char syndrome, a feature of which is postaxial polydactyly<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Satoda, M., Pierpont, M. E., Diaz, G. A., Bornemeier, R. A. &amp; Gelb, B. D. Char syndrome, an inherited disorder with patent ductus arteriosus, maps to chromosome 6p12-p21. Circulation 99, 3036–3042 (1999)." href="https://www.nature.com/articles/s41586-023-06806-x#ref-CR7" id="ref-link-section-d90000874e1320">7</a></sup>. The <i>RDH10</i><sup>+</sup> distal mesenchyme strongly expresses <i>RDH10</i>, encoding the primary enzyme of retinaldehyde synthesis, which is critical in interdigital cell death<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 8" title="Cunningham, T. J., Chatzi, C., Sandell, L. L., Trainor, P. A. &amp; Duester, G. Rdh10 mutants deficient in limb field retinoic acid signaling exhibit normal limb patterning but display interdigital webbing. Dev. Dyn. 240, 1142–1150 (2011)." href="https://www.nature.com/articles/s41586-023-06806-x#ref-CR8" id="ref-link-section-d90000874e1331">8</a></sup> (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig10">4f,g</a>). The transitional mesenchyme expresses <i>IRX1</i> and <i>IRX2</i>, which are key genes in digit formation and chondrogenic boundary definition<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 9" title="Díaz-Hernández, M. E., Bustamante, M., Galván-Hernández, C. I. &amp; Chimal-Monroy, J. Irx1 and Irx2 are coordinately expressed and regulated by retinoic acid, TGFβ and FGF signaling during chick hindlimb development. PLoS ONE 8, e58549 (2013)." href="https://www.nature.com/articles/s41586-023-06806-x#ref-CR9" id="ref-link-section-d90000874e1345">9</a></sup> (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig1">1c</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig10">4d,f</a>). We further examined the distributions of these genes at PCW5 and PCW6 in three dimensions using tissue clearing and light-sheet fluorescence microscopy (Supplementary Video <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41586-023-06806-x#MOESM9">1</a>). <i>IHH</i><sup>+</sup> prehypertrophic chondrocytes (PHCs) localized to the mid-diaphysis of the forming tibia and the metatarsals (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig1">1c</a>). At the proximal limit of the sample, both <i>MEIS2</i><sup><i>+</i></sup><i>WT1</i><sup><i>+</i></sup> proximal mesenchymal cells and <i>CITED1</i><sup>+</sup> mesenchymal cells (Mes3) were observed (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig1">1c</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig10">4f</a>).</p><p>At PCW8, we placed three anatomically continuous sections from the hindlimb on separate capture areas of the same Visium chip. We subsequently integrated data from this chip to obtain a spatial transcriptomic readout of a complete sagittal section of the limb (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig1">1d</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig11">5a</a>). At this stage, articular chondrocytes mapped to the articular surfaces of the developing joints, whereas osteoblasts mapped to the mid-diaphyseal bone collar of the tibia and femur. Perichondrial cells matched to a comparable region, although they extended along the full length of the tibia and femur (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig1">1d</a>), a finding confirmed by immunofluorescence staining for <i>RUNX2</i> and <i>THBS2</i> alongside <i>COL2A1</i> (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig11">5b</a>). <i>COL10A1</i><sup>+</sup> hypertrophic chondrocytes (HCCs) mapped to the mid-diaphysis of the tibia (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig1">1d</a>). Glial cells expressing myelin genes (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig7">1b</a>) were co-located with a <i>FOXS1</i><sup>+</sup> fibroblast subtype (‘neural fibroblast’) in the periphery of the sciatic and tibial nerves (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig1">1d</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig11">5c–e</a>). We captured few neurons (<i>n</i> = 28) in our single-cell data, probably due to the distant location of their cell bodies within the spinal ganglia.</p><p>Cell states with related (but not identical) transcriptomic profiles did not necessarily occupy the same location. For example, one group of three fibroblast clusters were co-located with basal and periderm cells<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 10" title="Richardson, R. J. et al. Periderm prevents pathological epithelial adhesions during embryogenesis. J. Clin. Invest. 124, 3891–3900 (2014)." href="https://www.nature.com/articles/s41586-023-06806-x#ref-CR10" id="ref-link-section-d90000874e1440">10</a></sup>, prompting their annotation as dermal fibroblasts (DermFiB) and their precursors (<i>F10</i><sup>+</sup>DermFiBP and <i>HOXC5</i><sup>+</sup>DermFiBP)(Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig8">2f,g</a>). Conversely, another group of two fibroblast clusters expressing ADH family members (<i>ADH</i><sup>+</sup>Fibro, InterMusFibro) colocalized with muscle cells (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig8">2h–j</a>). Similarly, two clusters expressed the tendon markers scleraxis (<i>SCX</i>) and tenomodulin (<i>TNMD</i>), with one expressing the extracellular matrix genes biglycan (<i>BGN</i>) and keratocan (<i>KERA</i>), whereas the other expressed pro-glucagon (<i>GCG</i>) (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig11">5f,g</a>). The former cluster matched to the hamstrings, quadriceps and patellar tendons, whereas the latter cluster matched to the perimysium surrounding the muscles (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig1">1d</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig11">5h</a>). We therefore annotated these clusters as tenocyte and perimysium, respectively. This integrated analysis serves as an example of how spatial transcriptomic methodologies can improve our understanding of tissue architecture and locate cell states within a dynamic anatomical structure such as the developing limb.</p></div></div></section><section data-title="Patterning, morphogenesis and developmental disorders in the limb"><div id="Sec3-section"><h2 id="Sec3">Patterning, morphogenesis and developmental disorders in the limb</h2><div id="Sec3-content"><p>We utilized spatial transcriptomic data to investigate patterning genes and found consistency with classical expression patterns in model organisms (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig12">6a–e</a>). This included genes that govern proximal identity, such as <i>MEIS</i><i>1</i>, <i>MEIS2</i>, <i>PBX1</i> and <i>IRX3</i>, as well as genes regulating limb outgrowth and distal morphogenesis such as <i>WNT5A</i>, <i>GREM1</i>, <i>ETV4</i> and <i>SALL1</i> (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 5" title="McQueen, C. &amp; Towers, M. Establishing the pattern of the vertebrate limb. Development 147, dev177956 (2020)." href="https://www.nature.com/articles/s41586-023-06806-x#ref-CR5" id="ref-link-section-d90000874e1526">5</a></sup>) (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig12">6b,c</a>). Similarly, classical mammalian anterior–posterior genes were captured, including <i>HAND1</i>, <i>PAX9</i>, <i>ALX4</i> and <i>ZIC3</i> (anterior) and <i>HAND2</i>, <i>SHH</i>, <i>PTCH1</i> and <i>GLI1</i> (posterior)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 5" title="McQueen, C. &amp; Towers, M. Establishing the pattern of the vertebrate limb. Development 147, dev177956 (2020)." href="https://www.nature.com/articles/s41586-023-06806-x#ref-CR5" id="ref-link-section-d90000874e1559">5</a></sup>,<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="McGlinn, E. et al. Pax9 and Jagged1 act downstream of Gli3 in vertebrate limb development. Mech. Dev. 122, 1218–1233 (2005)." href="#ref-CR11" id="ref-link-section-d90000874e1563">11</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Kuijper, S. et al. Function and regulation of Alx4 in limb development: complex genetic interactions with Gli3 and Shh. Dev. Biol. 285, 533–544 (2005)." href="#ref-CR12" id="ref-link-section-d90000874e1563_1">12</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Quinn, M. E., Haaning, A. &amp; Ware, S. M. Preaxial polydactyly caused by Gli3 haploinsufficiency is rescued by Zic3 loss of function in mice. Hum. Mol. Genet. 21, 1888–1896 (2012)." href="#ref-CR13" id="ref-link-section-d90000874e1563_2">13</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Riddle, R. D., Johnson, R. L., Laufer, E. &amp; Tabin, C. Sonic hedgehog mediates the polarizing activity of the ZPA. Cell 75, 1401–1416 (1993)." href="#ref-CR14" id="ref-link-section-d90000874e1563_3">14</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 15" title="Huangfu, D. &amp; Anderson, K. V. Signaling from Smo to Ci/Gli: conservation and divergence of Hedgehog pathways from Drosophila to vertebrates. Development 133, 3–14 (2006)." href="https://www.nature.com/articles/s41586-023-06806-x#ref-CR15" id="ref-link-section-d90000874e1566">15</a></sup> (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig12">6d,e</a>). Our spatial transcriptomic data captured the expression patterns of the HOXA and HOXD gene clusters at PCW5.6 (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig12">6f</a>). As expected, their expression matched the second wave of Hox expression in mice, with a loss of asymmetry in the HoxA cluster and its maintenance in the HoxD cluster, which showed increased restriction to the posterior margin of the limb with increased 5′ position. For both clusters, an increase in group number corresponded to more distally restricted expression, with group 13 genes limited to the autopod. An exception was <i>HOXA11</i>, which showed no overlap with <i>HOXA13</i>, in keeping with their expression patterns in mice. Our data revealed a flip to the antisense transcript of <i>HOXA11</i> in the distal limb (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig12">6f</a>), which may be due to HOXA13- and HOXD13-dependent activation of the <i>HOXA11-AS</i> enhancer<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 16" title="Kherdjemil, Y. et al. Evolution of Hoxa11 regulation in vertebrates is linked to the pentadactyl state. Nature 539, 89–92 (2016)." href="https://www.nature.com/articles/s41586-023-06806-x#ref-CR16" id="ref-link-section-d90000874e1592">16</a></sup>.</p><p>To investigate gene expression patterns during digit formation, we obtained coronal sections through a PCW6.2 foot plate to reveal the forming digits and interdigital space (IDS; Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig2">2a</a>). We annotated digital, interdigital and distal mesenchyme and other regions (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig2">2a</a>; see <a data-track="click" data-track-label="link" data-track-action="section anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Sec9">Methods</a>). Differential expression testing between the digital space and IDS demonstrated an enrichment of genes involved in IDS cell death, such as <i>BMP</i><i>7</i>, <i>BMP2</i> and <i>ADAMTS1</i> (refs. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 17" title="McCulloch, D. R. et al. ADAMTS metalloproteases generate active versican fragments that regulate interdigital web regression. Dev. Cell 17, 687–698 (2009)." href="https://www.nature.com/articles/s41586-023-06806-x#ref-CR17" id="ref-link-section-d90000874e1620">17</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 18" title="Kaltcheva, M. M., Anderson, M. J., Harfe, B. D. &amp; Lewandoski, M. BMPs are direct triggers of interdigital programmed cell death. Dev. Biol. 411, 266–276 (2016)." href="https://www.nature.com/articles/s41586-023-06806-x#ref-CR18" id="ref-link-section-d90000874e1623">18</a></sup>) (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig2">2b</a>). IDS also showed an enrichment of <i>RDH10</i> and <i>CRABP1</i>, whereas <i>CYP26B1</i>, encoding a retinoic acid-metabolizing enzyme, was upregulated in the digital regions, highlighting the role of retinoic acid in triggering IDS cell death<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 19" title="Díaz-Hernández, M. E., Rios-Flores, A. J., Abarca-Buis, R. F., Bustamante, M. &amp; Chimal-Monroy, J. Molecular control of interdigital cell death and cell differentiation by retinoic acid during digit development. J. Dev. Biol. 2, 138–157 (2014)." href="https://www.nature.com/articles/s41586-023-06806-x#ref-CR19" id="ref-link-section-d90000874e1640">19</a></sup> (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig2">2b,c</a>). Other digit-specific genes included <i>TGFB2</i>, a vital molecule in interphalangeal joint specification, and <i>WWP2</i>, a regulator of chondrogenesis<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 20" title="Spagnoli, A. et al. TGF-β signaling is essential for joint morphogenesis. J. Cell Biol. 177, 1105–1117 (2007)." href="https://www.nature.com/articles/s41586-023-06806-x#ref-CR20" id="ref-link-section-d90000874e1653">20</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 21" title="Mokuda, S. et al. Wwp2 maintains cartilage homeostasis through regulation of Adamts5. Nat. Commun. 10, 2429 (2019)." href="https://www.nature.com/articles/s41586-023-06806-x#ref-CR21" id="ref-link-section-d90000874e1656">21</a></sup>. In addition, <i>PIEZO2</i>, which promotes bone formation via calcium-dependent activation of NFATc1, YAP1 and β-catenin, was restricted to the digits, together with <i>C1QL1</i> (encoding a calcium-binding molecule), which correlates with <i>COL2A1</i> expression during in vitro chondrogenesis<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 22" title="Wu, C.-L. et al. Single cell transcriptomic analysis of human pluripotent stem cell chondrogenesis. Nat. Commun. 12, 362 (2021)." href="https://www.nature.com/articles/s41586-023-06806-x#ref-CR22" id="ref-link-section-d90000874e1670">22</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 23" title="Zhou, T. et al. Piezo1/2 mediate mechanotransduction essential for bone formation through concerted activation of NFAT–YAP1–β-catenin. eLife 9, e52779 (2020)." href="https://www.nature.com/articles/s41586-023-06806-x#ref-CR23" id="ref-link-section-d90000874e1673">23</a></sup> (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig2">2b</a>).</p><div data-test="figure" data-container-section="figure" id="figure-2" data-title="Spatial expression pattern of genes involved in digit formation and phenotype."><figure><figcaption><b id="Fig2" data-test="figure-caption-text">Fig. 2: Spatial expression pattern of genes involved in digit formation and phenotype.</b></figcaption><div><div><a data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="https://www.nature.com/articles/s41586-023-06806-x/figures/2" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41586-023-06806-x/MediaObjects/41586_2023_6806_Fig2_HTML.png?as=webp"/><img aria-describedby="Fig2" src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41586-023-06806-x/MediaObjects/41586_2023_6806_Fig2_HTML.png" alt="figure 2" loading="lazy" width="685" height="663"/></picture></a></div><div data-test="bottom-caption" id="figure-2-desc"><p><b>a</b>, Scheme to identify genes involved in digit formation and interdigital cell death (ICD). DEG, differentially expressed gene; DistMes, distal mesenchyme; IDS, interdigital space. <b>b</b>,<b>e</b>, Spatial expression (normalized and log-transformed) of genes promoting ICD (<b>b</b>, left panel) and digital tissue survival (<b>b</b>, right panel) and genes associated with digit malformation (<b>e</b>) in the PCW6.2 human hindlimb, and their distributions in IDS and digit regions (<i>P</i> values were determined by Wilcoxon rank sum test). OSMED, otospondylomegaepiphyseal dysplasia. <b>c</b>, RNA-ISH of <i>RDH10</i>, <i>CYP26B1</i> and <i>TGFB2</i> in the human hindlimb. Scale bars, 1 mm. <b>d</b>, Heatmap showing the expression (<i>Z</i> scores) of genes associated with digit malformation.</p><p><a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41586-023-06806-x#MOESM11">Source Data</a></p></div></div></figure></div><p>Finally, we annotated each digit to search for genes that vary with digit identity (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig12">6i</a>). Anterior genes <i>ID2</i> and <i>ZNF503</i>, as well as the regulators of cell proliferation <i>PLK2</i> and <i>LEMD</i><i>1</i> (refs. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Lorda-Diez, C. I., Torre-Pérez, N., García-Porrero, J. A., Hurle, J. M. &amp; Montero, J. A. Expression of Id2 in the developing limb is associated with zones of active BMP signaling and marks the regions of growth and differentiation of the developing digits. Int. J. Dev. Biol. 53, 1495–1502 (2009)." href="#ref-CR24" id="ref-link-section-d90000874e1765">24</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="McGlinn, E. et al. Expression of the NET family member Zfp503 is regulated by hedgehog and BMP signaling in the limb. Dev. Dyn. 237, 1172–1182 (2008)." href="#ref-CR25" id="ref-link-section-d90000874e1765_1">25</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 26" title="Ma, S., Charron, J. &amp; Erikson, R. L. Role of Plk2 (Snk) in mouse development and cell proliferation. Mol. Cell. Biol. 23, 6936–6943 (2003)." href="https://www.nature.com/articles/s41586-023-06806-x#ref-CR26" id="ref-link-section-d90000874e1768">26</a></sup>) were upregulated in the great toe; whereas <i>HOXD11</i> was downregulated as is found in mice and chicks. We found no differentially expressed genes in the remaining digits, although statistical power was limited by the sample size.</p><p>We next cross-referenced the list of spatial differentially expressed genes against a list of 2,300 genetic conditions. Genes involved in several types of non-syndromic brachydactyly were upregulated in the digits, including <i>NOG</i> (brachydactyly type B2), <i>PTH1R</i> (Eiken syndrome), <i>COL11A2</i> (otospondylomegaepiphyseal dysplasia), <i>SOX9</i> (Cooks syndrome) and <i>FGFR3</i> (achondroplasia)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Falardeau, F., Camurri, M. V. &amp; Campeau, P. M. Genomic approaches to diagnose rare bone disorders. Bone 102, 5–14 (2017)." href="#ref-CR27" id="ref-link-section-d90000874e1794">27</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Cooks, R. G., Hertz, M., Katznelson, M. B. &amp; Goodman, R. M. A new nail dysplasia syndrome with onychonychia and absence and/or hypoplasia of distal phalanges. Clin. Genet. 27, 85–91 (1985)." href="#ref-CR28" id="ref-link-section-d90000874e1794_1">28</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 29" title="Temtamy, S. A. &amp; Aglan, M. S. Brachydactyly. Orphanet J. Rare Dis. 3, 15 (2008)." href="https://www.nature.com/articles/s41586-023-06806-x#ref-CR29" id="ref-link-section-d90000874e1797">29</a></sup> (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig2">2d,e</a> and Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41586-023-06806-x#MOESM5">2</a> for all differentially expressed genes). Conversely, genes implicated in syndromes featuring syndactyly were significantly upregulated in the IDS and distal mesenchyme. These include <i>DLX5</i> (split hand–foot malformation), <i>MYCN</i> (Feingold syndrome type 1) and <i>TWIST1</i> (Saethre–Chotzen syndrome)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Bahubali, D. &amp; Gane, P. N. Split-hand/feet malformation: a rare syndrome. J. Family Med. Prim. Care 5, 168–169 (2016)." href="#ref-CR30" id="ref-link-section-d90000874e1817">30</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Marcelis, C. L. M. &amp; de Brouwer, A. P. M. Feingold Syndrome 1. In GeneReviews® [Internet]. (eds Adam, M. P. et al.) 
                www.ncbi.nlm.nih.gov/books/NBK7050
                
               (Univ. Washington, 2019)." href="#ref-CR31" id="ref-link-section-d90000874e1817_1">31</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Bakrania, P. et al. Mutations in BMP4 cause eye, brain, and digit developmental anomalies: overlap between the BMP4 and Hedgehog signaling pathways. Am. J. Hum. Genet. 82, 304–319 (2008)." href="#ref-CR32" id="ref-link-section-d90000874e1817_2">32</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 33" title="El Ghouzzi, V. et al. Saethre–Chotzen mutations cause TWIST protein degradation or impaired nuclear location. Hum. Mol. Genet. 9, 813–819 (2000)." href="https://www.nature.com/articles/s41586-023-06806-x#ref-CR33" id="ref-link-section-d90000874e1820">33</a></sup>. Where mouse models of these conditions exist, their phenotype is broadly comparable with the human models (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41586-023-06806-x#MOESM6">3</a>). Thus, our spatial atlas provides a valuable reference of gene expression under homeostatic conditions for comparison with genetic variations for which phenotypes may begin to penetrate during embryonic development.</p></div></div></section><section data-title="Regulation of cell-fate decisions of mesenchymal-derived lineages"><div id="Sec4-section"><h2 id="Sec4">Regulation of cell-fate decisions of mesenchymal-derived lineages</h2><div id="Sec4-content"><p>To better understand what may control their specification, we inferred cellular trajectories in the mesenchyme-associated states by combining diffusion maps, partition-based graph abstraction and force-directed graph (see <a data-track="click" data-track-label="link" data-track-action="section anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Sec9">Methods</a>). The global embedding revealed clusters of lineage-committed cells radiating outward from a hub of six mesenchymal states (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig3">3a,b</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig10">4f</a>). A first mesenchymal population, proximal mesenchymal cells, expressed the regulator of stylopod identity, <i>MEIS2</i>, together with <i>WT1</i>, which marks the point of limb–torso junction<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 34" title="Moore, A. W. et al. YAC transgenic analysis reveals Wilms’ tumour 1 gene activity in the proliferating coelomic epithelium, developing diaphragm and limb. Mech. Dev. 79, 169–184 (1998)." href="https://www.nature.com/articles/s41586-023-06806-x#ref-CR34" id="ref-link-section-d90000874e1851">34</a></sup> (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig3">3a,b</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig10">4f,h</a>). Mes1 cells exhibited a similar expression profile but lacked <i>WT1</i>, probably representing the mesenchyme just distal to the limb–torso junction (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig3">3a,b</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig10">4f,h</a>). We also identified a mesenchymal population within the posterior aspect of the developing hindlimb that expressed <i>ISL1</i> (<i>ISL1</i><sup><i>+</i></sup>Mes) in addition to <i>MEIS2</i>, but not <i>WT1</i> (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig10">4f,h</a>). Two further clusters (Mes2 and Mes3) expressed <i>CITED1</i>, a gene encoding a molecule that localizes to the proximal domain of the limb and has an unclear role in limb development<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 35" title="Dunwoodie, S. L., Rodriguez, T. A. &amp; Beddington, R. S. Msg1 and Mrg1, founding members of a gene family, show distinct patterns of gene expression during mouse embryogenesis. Mech. Dev. 72, 27–40 (1998)." href="https://www.nature.com/articles/s41586-023-06806-x#ref-CR35" id="ref-link-section-d90000874e1893">35</a></sup>. Mes2 also expressed <i>MEIS2</i>, suggesting a proximal–anterior location (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig10">4f,h</a>). The Mes4 cluster exhibited similar expression patterns to distal and transitional mesenchymal cells but lacked <i>LHX2</i> and <i>IRX1</i>. These six cell states form the majority of the cells during PCW5 (particularly notable at PCW5.1 (PCW5 plus 1 day) and PCW5.4, at 85% and 65%, respectively; Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig13">7c</a>), supporting their early mesenchymal identity. However, their numbers declined thereafter, with almost none present by PCW8.</p><div data-test="figure" data-container-section="figure" id="figure-3" data-title="Cell lineage diversification and transcription factor specificity of LPM during human embryonic limb development."><figure><figcaption><b id="Fig3" data-test="figure-caption-text">Fig. 3: Cell lineage diversification and transcription factor specificity of LPM during human embryonic limb development.</b></figcaption><div><div><a data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="https://www.nature.com/articles/s41586-023-06806-x/figures/3" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41586-023-06806-x/MediaObjects/41586_2023_6806_Fig3_HTML.png?as=webp"/><img aria-describedby="Fig3" src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41586-023-06806-x/MediaObjects/41586_2023_6806_Fig3_HTML.png" alt="figure 3" loading="lazy" width="685" height="615"/></picture></a></div><div data-test="bottom-caption" id="figure-3-desc"><p><b>a</b>, Force-directed graph layout of cells associated with the LPM, coloured by cell clusters. The black arrows indicate differentiation directions. Cluster abbreviations are the same as in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig1">1</a>. FA, force atlas. <b>b</b>, Dot plot showing selected marker genes for each cell cluster. The colour bar indicates the average expression level in linearly scaled values. <b>c</b>, Heatmap illustrating the vertically normalized mean activity of selected genes encoding transcription factors for each cell cluster. <b>d</b>, Force-directed graph (top) and Visium heatmaps (bottom) from the human hindlimb showing the expression of genes encoding transcription factors (normalized and log-transformed).</p><p><a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41586-023-06806-x#MOESM12">Source Data</a></p></div></div></figure></div><p>We next looked for modules of active transcription factor networks associated with progression through each lineage (see <a data-track="click" data-track-label="link" data-track-action="section anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Sec9">Methods</a>; Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig3">3c,d</a>, Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41586-023-06806-x#MOESM7">4</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig13">7a,b</a>). In addition to <i>WT1</i>, <i>MEIS1</i> and <i>MEIS2</i>, <i>GATA3</i> (detected in the proximal developing mouse limb) and <i>GATA5</i> (a putative proximal–distal patterning gene in the <i>Xenopus</i> limb) were both predicted to be active in the proximal mesenchyme<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 36" title="Hudson, D. T. et al. Gene expression analysis of the Xenopus laevis early limb bud proximodistal axis. Dev. Dyn. 251, 1880–1896 (2022)." href="https://www.nature.com/articles/s41586-023-06806-x#ref-CR36" id="ref-link-section-d90000874e1985">36</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 37" title="Sheng, G. &amp; Stern, C. D. Gata2 and Gata3: novel markers for early embryonic polarity and for non-neural ectoderm in the chick embryo. Mech. Dev. 87, 213–216 (1999)." href="https://www.nature.com/articles/s41586-023-06806-x#ref-CR37" id="ref-link-section-d90000874e1988">37</a></sup>. <i>HOXA11</i>, which defines the zeugopod, was active in Mes3. The distal mesenchyme showed activation of <i>LHX2</i>, <i>MSX1</i> and <i>MSX2</i>, as previously described in the mouse, as well as <i>HOXA13</i>, which defines the autopod<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 38" title="Tzchori, I. et al. LIM homeobox transcription factors integrate signaling events that control three-dimensional limb patterning and growth. Development 136, 1375–1385 (2009)." href="https://www.nature.com/articles/s41586-023-06806-x#ref-CR38" id="ref-link-section-d90000874e2008">38</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 39" title="Bensoussan-Trigano, V., Lallemand, Y., Saint, C. C. &amp; Robert, B. Msx1 and Msx2 in limb mesenchyme modulate digit number and identity. Dev. Dyn. 240, 1190–1202 (2011)." href="https://www.nature.com/articles/s41586-023-06806-x#ref-CR39" id="ref-link-section-d90000874e2011">39</a></sup>. <i>HIC1</i> was predicted to have activity in several mesenchymal populations (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig3">3c,d</a>). <i>HIC1</i><sup><i>+</i></sup> mesenchymal cells are known to migrate into the limb from the hypaxial somite, differentiating into a range of tissues such as chondrocytes and tenocytes, while maintaining <i>HIC1</i> expression<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 40" title="Arostegui, M., Scott, R. W., Böse, K. &amp; Underhill, T. M. Cellular taxonomy of Hic1+ mesenchymal progenitor derivatives in the limb: from embryo to adult. Nat. Commun. 13, 4989 (2022)." href="https://www.nature.com/articles/s41586-023-06806-x#ref-CR40" id="ref-link-section-d90000874e2031">40</a></sup>. Indeed, <i>HIC1</i> was active in chondrocyte and tendon populations (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig3">3c,d</a>).</p><p>The chondrocyte lineage increased in number over time, shifting from progenitors to more mature clusters during the period studied (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig13">7c</a>). Mesenchymal condensate cells, <i>SOX</i><sup><i>low</i></sup><i>COL2A1</i><sup><i>low</i></sup><i>PRRX1</i><sup><i>hi</i></sup> osteochondral progenitors and <i>SOX9</i><sup><i>hi</i></sup><i>COL2A1</i><sup><i>low</i></sup> chondrocyte progenitors gave way to three populations of <i>SOX9</i><sup><i>hi</i></sup><i>COL2A1</i><sup><i>hi</i></sup> chondrocytes: <i>UCMA</i><sup>+</sup> resting, <i>UCMA</i><sup>−</sup><i>IHH</i><sup>−</sup> proliferating (with a greater proportion of cells in G2, M and S phases) and <i>IHH</i><sup>+</sup> PHCs (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig3">3b,c</a>, Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41586-023-06806-x#MOESM4">1</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig13">7d–g</a>). In addition, we captured 14 <i>COL10A1</i><sup>+</sup><i>MMP13</i><sup>+</sup> HCCs (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig13">7d</a>). Curiously, both partition-based graph abstraction and RNA velocity analyses suggested chondrocyte progenitors for an individual sample may progress to either the resting state before proliferation or proceed directly to proliferation (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig3">3a</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig13">7e,f</a>), although further work is required to investigate this finding. The transition from mesenchymal condensate to committed chondrocytes (with the latter localizing to chondrocyte condensations at PCW5.6 and the developing long bones at PCW6.2) was associated with activity in <i>SOX5</i>, <i>SOX</i><i>6</i> and <i>SOX</i><i>9</i>, as well as <i>MAFF</i> and <i>NKX3-2</i>, which encode chondrogenic transcription factors<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Nishimura, R., Hata, K., Matsubara, T., Wakabayashi, M. &amp; Yoneda, T. Regulation of bone and cartilage development by network between BMP signalling and transcription factors. J. Biochem. 151, 247–254 (2012)." href="#ref-CR41" id="ref-link-section-d90000874e2150">41</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Kawato, Y. et al. Nkx3.2 promotes primary chondrogenic differentiation by upregulating Col2a1 transcription. PLoS ONE 7, e34703 (2012)." href="#ref-CR42" id="ref-link-section-d90000874e2150_1">42</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 43" title="Hong, E., Di Cesare, P. E. &amp; Haudenschild, D. R. Role of c-Maf in chondrocyte differentiation: a review. Cartilage 2, 27–35 (2011)." href="https://www.nature.com/articles/s41586-023-06806-x#ref-CR43" id="ref-link-section-d90000874e2153">43</a></sup> (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig3">3d</a>). Several regulators of chondrocyte hypertrophy were active in PHCs and HCCs (the latter localizing to the tibial diaphysis at PCW6.2), including <i>SP7</i>, <i>DLX2/3</i> and <i>RUNX3</i> (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 41" title="Nishimura, R., Hata, K., Matsubara, T., Wakabayashi, M. &amp; Yoneda, T. Regulation of bone and cartilage development by network between BMP signalling and transcription factors. J. Biochem. 151, 247–254 (2012)." href="https://www.nature.com/articles/s41586-023-06806-x#ref-CR41" id="ref-link-section-d90000874e2170">41</a></sup>). <i>RUNX2</i> was predicted to be active in the perichondrium and osteoblasts, in addition to PHCs and HCCs (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig1">1d</a>). The osteogenic regulator <i>SATB2</i> was highly specific to osteoblasts<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 44" title="Zhang, J. et al. Roles of SATB2 in osteogenic differentiation and bone regeneration. Tissue Eng. Part A 17, 1767–1776 (2011)." href="https://www.nature.com/articles/s41586-023-06806-x#ref-CR44" id="ref-link-section-d90000874e2183">44</a></sup>.</p><p>To capture cells of the interzone, mesenchymal cells that reside at the sites of future synovial joints and give rise to their constituent parts, we sectioned two forelimbs and two hindlimbs into proximal, middle (containing the knee and elbow regions) and distal segments. These data contained a cluster expressing the interzone marker <i>GDF5</i> and articular chondrocytes expressing lubricin (<i>PRG4</i>; log fold change = 5.15, <i>P</i> = 4.1 × 10<sup>−26</sup>) (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig3">3b</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig13">7d</a>). Notably, <i>FOXP2</i> (a negative regulator of endochondral ossification) and <i>ERG</i> (a positive regulator of articular chondrocyte) were active in interzone cells but <i>SOX5</i>, <i>SOX6</i> and <i>SOX9</i> were not<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 45" title="Iwamoto, M. et al. Transcription factor ERG and joint and articular cartilage formation during mouse limb and spine skeletogenesis. Dev. Biol. 305, 40–51 (2007)." href="https://www.nature.com/articles/s41586-023-06806-x#ref-CR45" id="ref-link-section-d90000874e2224">45</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 46" title="Zhao, H. et al. Foxp1/2/4 regulate endochondral ossification as a suppresser complex. Dev. Biol. 398, 242–254 (2015)." href="https://www.nature.com/articles/s41586-023-06806-x#ref-CR46" id="ref-link-section-d90000874e2227">46</a></sup> (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig3">3c</a>).</p><p><i>SCX</i><sup><i>hi</i></sup><i>TNMD</i><sup><i>low</i></sup> tendon progenitors emerged during PCW5 before being replaced by <i>TNMD</i><sup><i>hi</i></sup> tenocytes and perimysium from PCW7 onwards (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig3">3b</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig13">7c</a>). <i>SCX</i><sup>+</sup><i>SOX9</i><sup>+</sup> cells previously shown to give rise to the entheses were also captured, a finding confirmed by spatial transcriptomic and RNA-ISH<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 47" title="Blitz, E., Sharir, A., Akiyama, H. &amp; Zelzer, E. Tendon-bone attachment unit is formed modularly by a distinct pool of Scx- and Sox9-positive progenitors. Development 140, 2680–2690 (2013)." href="https://www.nature.com/articles/s41586-023-06806-x#ref-CR47" id="ref-link-section-d90000874e2268">47</a></sup> (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig13">7h–k</a>). Several tenogenic transcription factors were predicted to be active in these clusters, including <i>ETV4</i> and <i>NFIX</i><sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 48" title="Yang, F. &amp; Richardson, D. W. Comparative analysis of tenogenic gene expression in tenocyte-derived induced pluripotent stem cells and bone marrow-derived mesenchymal stem cells in response to biochemical and biomechanical stimuli. Stem Cells Int. 2021, 8835576 (2021)." href="https://www.nature.com/articles/s41586-023-06806-x#ref-CR48" id="ref-link-section-d90000874e2281">48</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 49" title="Rossi, G. et al. Nfix regulates temporal progression of muscle regeneration through modulation of myostatin expression. Cell Rep. 14, 2238–2249 (2016)." href="https://www.nature.com/articles/s41586-023-06806-x#ref-CR49" id="ref-link-section-d90000874e2284">49</a></sup> (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig3">3c</a>). Finally, fibroblast and smooth muscle cell populations within the limb exhibited clearly distinct transcription factor activities (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig13">7a,b</a>). Dermal fibroblasts showed activity in known regulators of this lineage, including <i>RUNX1</i> and <i>TFAP2C</i><sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 50" title="Hoi, C. S. L. et al. Runx1 directly promotes proliferation of hair follicle stem cells and epithelial tumor formation in mouse skin. Mol. Cell. Biol. 30, 2518–2536 (2010)." href="https://www.nature.com/articles/s41586-023-06806-x#ref-CR50" id="ref-link-section-d90000874e2299">50</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 51" title="Li, L. et al. TFAP2C- and p63-dependent networks sequentially rearrange chromatin landscapes to drive human epidermal lineage commitment. Cell Stem Cell 24, 271–284.e8 (2019)." href="https://www.nature.com/articles/s41586-023-06806-x#ref-CR51" id="ref-link-section-d90000874e2302">51</a></sup>, whereas smooth muscle cells and their precursors both showed activity in <i>GATA6</i>, which is thought to regulate their synthetic function<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 52" title="Lepore, J. J., Cappola, T. P., Mericko, P. A., Morrisey, E. E. &amp; Parmacek, M. S. GATA-6 regulates genes promoting synthetic functions in vascular smooth muscle cells. Arterioscler. Thromb. Vasc. Biol. 25, 309–314 (2005)." href="https://www.nature.com/articles/s41586-023-06806-x#ref-CR52" id="ref-link-section-d90000874e2310">52</a></sup>. In addition, smooth muscle cells showed activity in transcription factors with known roles in smooth muscle function, such as <i>ARNTL</i><sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 53" title="Xie, Z. et al. Smooth-muscle BMAL1 participates in blood pressure circadian rhythm regulation. J. Clin. Invest. 125, 324–336 (2015)." href="https://www.nature.com/articles/s41586-023-06806-x#ref-CR53" id="ref-link-section-d90000874e2316">53</a></sup>.</p></div></div></section><section data-title="Regulation of embryonic and fetal myogenesis"><div id="Sec5-section"><h2 id="Sec5">Regulation of embryonic and fetal myogenesis</h2><div id="Sec5-content"><p>Limb muscle formation begins with delamination and migration from the somite regulated by <i>PAX3</i> and co-regulators such as <i>LBX1</i> and <i>MEOX2</i>. Two waves of myogenesis follow: embryonic and fetal<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 3" title="Chevallier, A., Kieny, M. &amp; Mauger, A. Limb-somite relationship: origin of the limb musculature. J. Embryol. Exp. Morphol. 41, 245–258 (1977)." href="https://www.nature.com/articles/s41586-023-06806-x#ref-CR3" id="ref-link-section-d90000874e2337">3</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 54" title="Buckingham, M. &amp; Rigby, P. W. J. Gene regulatory networks and transcriptional mechanisms that control myogenesis. Dev. Cell 28, 225–238 (2014)." href="https://www.nature.com/articles/s41586-023-06806-x#ref-CR54" id="ref-link-section-d90000874e2340">54</a></sup>. During embryonic myogenesis, a portion of <i>PAX3</i><sup>+</sup> embryonic skeletal muscle progenitors differentiate and fuse into multinucleated myotubes. These primary fibres act as the scaffold for the formation of secondary fibres derived from <i>PAX7</i><sup>+</sup> fetal skeletal muscle progenitors, which are themselves derived from <i>PAX3</i><sup>+</sup> muscle progenitors<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 54" title="Buckingham, M. &amp; Rigby, P. W. J. Gene regulatory networks and transcriptional mechanisms that control myogenesis. Dev. Cell 28, 225–238 (2014)." href="https://www.nature.com/articles/s41586-023-06806-x#ref-CR54" id="ref-link-section-d90000874e2357">54</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 55" title="Hutcheson, D. A., Zhao, J., Merrell, A. &amp; Haldar, M. Embryonic and fetal limb myogenic cells are derived from developmentally distinct progenitors and have different requirements for β-catenin. Genes 10.1101/gad.1769009 (2009)." href="https://www.nature.com/articles/s41586-023-06806-x#ref-CR55" id="ref-link-section-d90000874e2360">55</a></sup>. To dissect these developmental trajectories in humans, we re-embedded muscle cells using diffusion mapping combined with partition-based graph abstraction and force-directed graph. Three distinct trajectories with an origin in <i>PAX3</i><sup>+</sup>SkMP emerged (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig4">4a</a>). The first (labelled first myogenesis, in keeping with embryonic myogenesis) progresses from <i>PAX3</i><sup>+</sup>SkMP to an embryonic myoblast state (MyoB1) followed by an early embryonic myocyte state (MyoC1), and finally mature embryonic myocytes. The second runs from <i>PAX3</i><sup>+</sup>SkMP to <i>PAX3</i><sup>+</sup><i>PAX7</i><sup>+</sup> cells, followed by a heterogeneous pool of mostly <i>MyoD</i><sup><i>low</i></sup><i>PAX7</i><sup>+</sup> SkMP cells (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig4">4a,b</a>). This represents a developmental path that generates progenitors for subsequent muscle formation and regeneration. The final trajectory (labelled second myogenesis) connects cell states that express <i>PAX7</i> first to fetal myoblasts (MyoB2), then to early fetal myocytes (MyoC2), and finally to mature fetal myocytes (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig4">4a,c</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig14">8b</a>).</p><div data-test="figure" data-container-section="figure" id="figure-4" data-title="Cell trajectory and transcription factors of embryonic and fetal limb myogenesis."><figure><figcaption><b id="Fig4" data-test="figure-caption-text">Fig. 4: Cell trajectory and transcription factors of embryonic and fetal limb myogenesis.</b></figcaption><div><div><a data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="https://www.nature.com/articles/s41586-023-06806-x/figures/4" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41586-023-06806-x/MediaObjects/41586_2023_6806_Fig4_HTML.png?as=webp"/><img aria-describedby="Fig4" src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41586-023-06806-x/MediaObjects/41586_2023_6806_Fig4_HTML.png" alt="figure 4" loading="lazy" width="685" height="446"/></picture></a></div><div data-test="bottom-caption" id="figure-4-desc"><p><b>a</b>, Force-directed graph layout of cells associated with the myogenesis, coloured by cell clusters. The green and pink arrows indicate the direction of first and second myogenesis, separately. MyoB, myoblast; MyoC, myocyte; MyoProg, myogenic progenitor. <b>b</b>, Dot plot showing expression pattern of selected marker genes. The colour bar indicates the average expression level in linearly scaled values. <b>c</b>, Fraction of cell type per timepoint. <b>d</b>, Heatmap illustrating the vertically normalized mean activity of filtered genes encoding transcription factors for each cell cluster. <b>e</b>, Violin plot showing the expression level of <i>PITX1</i> in human forelimb and hindlimb at PCW5.6. <b>f</b>, Immunofluorescence co-staining (scale bar, 50 μm) of PITX1 and PAX3 on hindlimb (top panels) and forelimb (bottom panels) sections (scale bar, 200 μm). Hindlimb, <i>n</i> = 4; forelimb, <i>n</i> = 2. <b>g</b>, RT–qPCR analysis of the fold-enrichment myocyte genes upon knockdown of <i>MSC</i> in human primary embryonic myoblasts. Data are presented as mean ± s.e.m. <i>P</i> values are from two-sided Student’s <i>t</i>-tests. <i>n</i> = 2 embryos and 3 independent experiments with similar results.</p><p><a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41586-023-06806-x#MOESM13">Source Data</a></p></div></div></figure></div><p>Comparing these pathways, <i>PAX3</i> is almost absent in fetal myogenesis, whereas it persists to late states along the embryonic pathway, consistent with a previous study that captured <i>Pax3</i><sup>+</sup><i>Myog</i><sup>+</sup> cells in the mouse limb<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 56" title="Singh, A. J. et al. FACS-seq analysis of Pax3-derived cells identifies non-myogenic lineages in the embryonic forelimb. Sci. Rep. 8, 7670 (2018)." href="https://www.nature.com/articles/s41586-023-06806-x#ref-CR56" id="ref-link-section-d90000874e2490">56</a></sup> (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig4">4b</a>). <i>ID2</i> and <i>ID3</i>, which have been shown to attenuate myogenic regulatory factors<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 57" title="Benezra, R., Davis, R. L., Lockshon, D., Turner, D. L. &amp; Weintraub, H. The protein Id: a negative regulator of helix-loop-helix DNA binding proteins. Cell 61, 49–59 (1990)." href="https://www.nature.com/articles/s41586-023-06806-x#ref-CR57" id="ref-link-section-d90000874e2504">57</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 58" title="Roschger, C. &amp; Cabrele, C. The Id-protein family in developmental and cancer-associated pathways. Cell Commun. Signal. 15, 7 (2017)." href="https://www.nature.com/articles/s41586-023-06806-x#ref-CR58" id="ref-link-section-d90000874e2507">58</a></sup>, were highly expressed in embryonic myogenesis, which may imply different upstream regulatory networks. Additional genes such as <i>FST</i>, <i>RGS4</i>, <i>NEFM</i> and <i>SAMD11</i> were also identified to be marking the first myogenic pathway, whereas <i>TNFSF13B</i>, <i>KRT31</i> and <i>RGR</i> mark the second myogenic pathway (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig4">4b</a>). In fact, keratin genes have been shown to facilitate sarcomere organization<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 59" title="Muriel, J. M. et al. Keratin 18 is an integral part of the intermediate filament network in murine skeletal muscle. Am. J. Physiol. Cell Physiol. 318, C215–C224 (2020)." href="https://www.nature.com/articles/s41586-023-06806-x#ref-CR59" id="ref-link-section-d90000874e2536">59</a></sup>.</p><p>Next, we applied SCENIC to search for transcription factors driving each myogenic stage (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig4">4d</a>, Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41586-023-06806-x#MOESM7">4</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig14">8a</a>). Although <i>MYOD1</i> and <i>MYOG</i> showed similar activities across fetal and embryonic myogenesis, several transcription factors were predicted to have higher activity in one or the other. For example, <i>PITX2</i> exhibited a higher activity score and abundance during embryonic myogenesis (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig4">4d</a>), possibly related to its different regulatory roles<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 60" title="Hernandez-Torres, F., Rodríguez-Outeiriño, L., Franco, D. &amp; Aranega, A. E. Pitx2 in embryonic and adult myogenesis. Front. Cell Dev. Biol. 5, 46 (2017)." href="https://www.nature.com/articles/s41586-023-06806-x#ref-CR60" id="ref-link-section-d90000874e2565">60</a></sup>. By contrast, <i>PITX1</i> exhibited comparable activity in both trajectories. Despite its known role as a hindlimb-specific transcription factor, <i>PITX1</i> is expressed in both forelimb and hindlimb muscle cells, including <i>PAX3</i><sup>+</sup> cells as early as PCW5 (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig4">4e,f</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig14">8c</a>), suggesting a regulatory role in embryonic myogenesis. Other genes encoding transcription factors specific to embryonic myogenesis included <i>MSX1</i>, which maintains the early progenitor pool, the MyoD activator <i>SIX2</i> and the satellite cell homeodomain factor <i>BARX2</i> (refs. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Lee, H., Habas, R. &amp; Abate-Shen, C. MSX1 cooperates with histone H1b for inhibition of transcription and myogenesis. Science 304, 1675–1678 (2004)." href="#ref-CR61" id="ref-link-section-d90000874e2596">61</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Relaix, F. et al. Six homeoproteins directly activate Myod expression in the gene regulatory networks that control early myogenesis. PLoS Genet. 9, e1003425 (2013)." href="#ref-CR62" id="ref-link-section-d90000874e2596_1">62</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 63" title="Meech, R. et al. Barx2 is expressed in satellite cells and is required for normal muscle growth and regeneration. Stem Cells 30, 253–265 (2012)." href="https://www.nature.com/articles/s41586-023-06806-x#ref-CR63" id="ref-link-section-d90000874e2599">63</a></sup>). Next, we investigated transcriptional repressor expression in muscle. We observed specific expressions of <i>ID1</i>, <i>ID2</i>, <i>ID3</i>, <i>HEY1</i>, <i>MSC</i> and <i>HES1</i> in <i>PAX7</i><sup>+</sup> skeletal progenitors (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig4">4b</a>). The most prominent repressor, <i>MSC</i> (also known as musculin, <i>ABF1</i> or <i>MyoR</i>) encodes a basic helix–loop–helix transcription factor that has been shown to inhibit the ability of MyoD to activate myogenesis in 10T1/2 fibroblasts<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 64" title="Lu, J., Webb, R., Richardson, J. A. &amp; Olson, E. N. MyoR: a muscle-restricted basic helix–loop–helix transcription factor that antagonizes the actions of MyoD. Proc. Natl Acad. Sci. USA 96, 552–557 (1999)." href="https://www.nature.com/articles/s41586-023-06806-x#ref-CR64" id="ref-link-section-d90000874e2639">64</a></sup> and rhabdomyosarcoma cells<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 65" title="MacQuarrie, K. L., Yao, Z., Fong, A. P. &amp; Tapscott, S. J. Genome-wide binding of the basic helix-loop-helix myogenic inhibitor musculin has substantial overlap with MyoD: implications for buffering activity. Skelet. Muscle 3, 26 (2013)." href="https://www.nature.com/articles/s41586-023-06806-x#ref-CR65" id="ref-link-section-d90000874e2643">65</a></sup>. In C2C12 murine myoblasts, <i>MSC</i> facilitates the inhibition of myogenesis by NOTCH (although it appears to exhibit functional redundancy in this role)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 66" title="Buas, M. F., Kabak, S. &amp; Kadesch, T. Inhibition of myogenesis by Notch: evidence for multiple pathways. J. Cell. Physiol. 218, 84–93 (2009)." href="https://www.nature.com/articles/s41586-023-06806-x#ref-CR66" id="ref-link-section-d90000874e2650">66</a></sup>. To test whether human <i>MSC</i> also has a role in repressing <i>PAX7</i><sup>+</sup> progenitor maturation, we knocked down <i>MSC</i> in primary human embryonic limb myoblasts. Quantitative PCR with reverse transcription (RT–qPCR) results showed upregulation of late myocyte genes, suggesting that it has a key role in maintaining limb muscle progenitor identity (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig4">4g</a>).</p></div></div></section><section data-title="Spatially resolved microenvironments exhibit distinct patterns of cell–cell communication"><div id="Sec6-section"><h2 id="Sec6">Spatially resolved microenvironments exhibit distinct patterns of cell–cell communication</h2><div id="Sec6-content"><p>We next looked for stage-specific ligand–receptor interactions in co-located cell populations (see <a data-track="click" data-track-label="link" data-track-action="section anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Sec9">Methods</a>). This highlighted the role of the WNT signalling pathway in early limb morphogenesis. <i>WNT5A</i> exhibited a proximal–distal expression gradient, peaking in the distal mesenchyme (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig5">5a–e</a>). Its receptor <i>FZD10</i> was expressed in the distal ectoderm of the limb at PCW6, with weak mesenchymal expression, although at this comparatively late stage, it appears to be no longer restricted to posterior regions, as has been reported in early limb development in the mouse<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 67" title="Nunnally, A. P. &amp; Parr, B. A. Analysis of Fz10 expression in mouse embryos. Dev. Genes Evol. 214, 144–148 (2004)." href="https://www.nature.com/articles/s41586-023-06806-x#ref-CR67" id="ref-link-section-d90000874e2689">67</a></sup> (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig5">5c</a>). Furthermore, our single-cell data revealed high expression of <i>FZD4</i> in the mesenchymal condensate, a finding supported by RNA-ISH (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig5">5a,d,e</a>). This supports the suggestion from in vitro studies that <i>FZD4</i> has a role in initiating chondrogenesis when mesenchymal condensate reaches a critical mass<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 68" title="Sarem, M., Otto, O., Tanaka, S. &amp; Shastri, V. P. Cell number in mesenchymal stem cell aggregates dictates cell stiffness and chondrogenesis. Stem Cell Res. Ther. 10, 10 (2019)." href="https://www.nature.com/articles/s41586-023-06806-x#ref-CR68" id="ref-link-section-d90000874e2706">68</a></sup>.</p><div data-test="figure" data-container-section="figure" id="figure-5" data-title="Spatially resolved cell–cell communication."><figure><figcaption><b id="Fig5" data-test="figure-caption-text">Fig. 5: Spatially resolved cell–cell communication.</b></figcaption><div><div><a data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="https://www.nature.com/articles/s41586-023-06806-x/figures/5" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41586-023-06806-x/MediaObjects/41586_2023_6806_Fig5_HTML.png?as=webp"/><img aria-describedby="Fig5" src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41586-023-06806-x/MediaObjects/41586_2023_6806_Fig5_HTML.png" alt="figure 5" loading="lazy" width="685" height="669"/></picture></a></div><div data-test="bottom-caption" id="figure-5-desc"><p><b>a</b>, Dot plots showing expression (<i>Z</i> score) of ligands and cognate receptors in cell clusters (top), and heatmaps showing predicted cell-type abundance (bottom). <b>b</b>,<b>d</b>,<b>f</b>, Visium heatmaps of the hindlimb at PCW5.6 showing expression (normalized and log-transformed) of <i>WNT5A</i> (<b>b</b>,<b>d</b>), <i>JAG1</i> (<b>f</b>) and their cognate receptors <i>FZD10</i> (<b>b</b>), <i>FZD4</i> (<b>d</b>) and <i>NOTCH1</i> (<b>f</b>). The yellow stars indicate both the ligand and its receptor expressed. <b>c</b>,<b>e</b>,<b>g</b>, RNA-ISH expression of <i>WNT5A</i> (<b>c</b>,<b>e</b>), <i>JAG1</i> (<b>g</b>) and their cognate receptors <i>FZD10</i> (<b>c</b>), <i>FZD4</i> (<b>e</b>) and <i>NOTCH1</i> (<b>g</b>) in situ. Scale bars, 1 mm. <b>h</b>, Dot plots of <i>FGFR2</i> expression and its ligands (top), and Visium heatmaps of a PCW6.2 human hindlimb showing spatially resolved selected mesenchymal cell cluster (separated by colour) signatures (bottom). <b>i</b>, Visium heatmaps of a PCW6.2 human hindlimb showing expression of <i>FGF8</i>, <i>FGF10</i> and <i>FGFR2</i>. The yellow stars indicate that both <i>FGF8</i> and <i>FGFR2</i> are expressed. The white stars denote that both <i>FGF10</i> and <i>FGFR2</i> are expressed. <b>j</b>, RNA-ISH of <i>FGF8</i>, <i>FGF10</i> and <i>FGFR2</i> expression in the PCW6.2 hindlimb.</p><p><a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41586-023-06806-x#MOESM14">Source Data</a></p></div></div></figure></div><p>In the early (PCW5.6) limb, NOTCH signalling was predicted to occur in its distal posterior aspect through the <i>SHH</i>-induced<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 11" title="McGlinn, E. et al. Pax9 and Jagged1 act downstream of Gli3 in vertebrate limb development. Mech. Dev. 122, 1218–1233 (2005)." href="https://www.nature.com/articles/s41586-023-06806-x#ref-CR11" id="ref-link-section-d90000874e2878">11</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 69" title="Crosnier, C. et al. JAGGED1 gene expression during human embryogenesis elucidates the wide phenotypic spectrum of Alagille syndrome. Hepatology 32, 574–581 (2000)." href="https://www.nature.com/articles/s41586-023-06806-x#ref-CR69" id="ref-link-section-d90000874e2881">69</a></sup> canonical ligand <i>JAG1</i> (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig5">5a,f</a>) and was confirmed with RNA-ISH (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig5">5g</a>). This interaction occurs between adjacent cells, triggering proteolytic cleavage of the intracellular domain of NOTCH<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 70" title="Schroeter, E. H., Kisslinger, J. A. &amp; Kopan, R. Notch-1 signalling requires ligand-induced proteolytic release of intracellular domain. Nature 393, 382–386 (1998)." href="https://www.nature.com/articles/s41586-023-06806-x#ref-CR70" id="ref-link-section-d90000874e2895">70</a></sup>. Through colocalization analysis (see <a data-track="click" data-track-label="link" data-track-action="section anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Sec9">Methods</a>), we observed that <i>NOTCH1</i> expression closely follows <i>JAG1</i> with a probability of co-existence in each pixel (0.14 × 0.14 μm) of 0.75 (2.63 × 10<sup>7</sup> dual-positive pixels out of 3.51 × 10<sup>7</sup> total pixels containing <i>JAG1</i>). Analysis of single-cell data showed that <i>JAG1</i> and <i>NOTCH1</i> were expressed by several mesenchymal populations within the early limb (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig5">5a</a>). This finding sheds further light on limb morphogenesis and has implications for conditions in which this signalling axis is disrupted, such as the posterior digit absence of Adams–Oliver syndrome and fifth finger clinodactyly of Alagille syndrome<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 71" title="Mašek, J. &amp; Andersson, E. R. The developmental biology of genetic Notch disorders. Development 144, 1743–1763 (2017)." href="https://www.nature.com/articles/s41586-023-06806-x#ref-CR71" id="ref-link-section-d90000874e2925">71</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 72" title="Turnpenny, P. D. &amp; Ellard, S. Alagille syndrome: pathogenesis, diagnosis and management. Eur. J. Hum. Genet. 20, 251–257 (2011)." href="https://www.nature.com/articles/s41586-023-06806-x#ref-CR72" id="ref-link-section-d90000874e2928">72</a></sup>.</p><p>We captured weak but reproducible signals of <i>FGF8</i> in the apical ectodermal ridge across various timepoints, whereas <i>FGF10</i> was detected in the adjacent distal mesenchyme (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig5">5h–i</a>). <i>FGF8</i> and <i>FGF10</i> have been shown to be expressed in the limb ectoderm and distal mesenchyme, respectively, and to form a feedback loop through <i>FGFR2</i> that is essential for limb induction<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 73" title="Xu, X. et al. Fibroblast growth factor receptor 2 (FGFR2)-mediated reciprocal regulation loop between FGF8 and FGF10 is essential for limb induction. Development 10.1242/dev.125.4.753 (1998)." href="https://www.nature.com/articles/s41586-023-06806-x#ref-CR73" id="ref-link-section-d90000874e2954">73</a></sup>. Ectodermal <i>FGF8</i> expression was confirmed via RNA-ISH (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig5">5j</a>). <i>FGF10</i> expression was notably restricted in the foot plate adjacent to the forming phalanges and was excluded in the IDS and <i>RDH10</i><sup>+</sup> distal mesenchyme (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig5">5h,j</a>). This is consistent with expression in mouse, where conditional knockdown leads to short, webbed digits<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 74" title="Agha, E. E. et al. Characterization of a novel fibroblast growth factor 10 (Fgf10) knock-in mouse line to target mesenchymal progenitors during embryonic development. Pneumologie 10.1055/s-0032-1315504 (2012)." href="https://www.nature.com/articles/s41586-023-06806-x#ref-CR74" id="ref-link-section-d90000874e2975">74</a></sup>. <i>FGF10</i>, which induces chondrogenesis via <i>FGFR2</i>, was expressed in the chondrocyte progenitors (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig5">5h</a>). RNA-ISH confirmed this throughout the skeleton of the forming limb (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig5">5j</a>). <i>FGFR2</i> colocalized with <i>FGF8</i> (probability = 0.62; 7.31 × 10<sup>5</sup> dual positive; 1.17 × 10<sup>6</sup> total <i>FGF8</i>) and similarly, <i>FGFR2</i> colocalized with <i>FGF10</i> (probability = 0.89; 1.56 × 10<sup>6</sup> dual positive; 1.75 × 10<sup>6</sup> total <i>FGF10</i>). Finally, the co-expression of <i>FGF10</i> and <i>FGF8</i> in the same pixel was infrequent (probability = 0.05; 9.68 × 10<sup>4</sup> dual positive; 1.17 × 10<sup>6</sup> total <i>FGF8</i>). The importance of this receptor in skeletal development is highlighted by the limb phenotypes observed in the <i>FGFR2</i>-related craniosynostoses<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 75" title="Azoury, S. C., Reddy, S., Shukla, V. &amp; Deng, C.-X. Fibroblast growth factor receptor 2 (FGFR2) mutation related syndromic craniosynostosis. Int. J. Biol. Sci. 13, 1479–1488 (2017)." href="https://www.nature.com/articles/s41586-023-06806-x#ref-CR75" id="ref-link-section-d90000874e3036">75</a></sup>.</p></div></div></section><section data-title="Homology and divergence between human and mouse limb development"><div id="Sec7-section"><h2 id="Sec7">Homology and divergence between human and mouse limb development</h2><div id="Sec7-content"><p>Limb development has long been studied in model organisms, whereas studies of human samples are few. To explore differences between mice and humans, we collected 13 mouse limb samples for scRNA-seq and combined our newly generated data with 18 high-quality limb datasets from 3 published studies<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="He, P. et al. The changing mouse embryo transcriptome at whole tissue and single-cell resolution. Nature 583, 760–767 (2020)." href="#ref-CR76" id="ref-link-section-d90000874e3048">76</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Kelly, N. H., Huynh, N. P. T. &amp; Guilak, F. Single cell RNA-sequencing reveals cellular heterogeneity and trajectories of lineage specification during murine embryonic limb development. Matrix Biol. 10.1016/j.matbio.2019.12.004 (2020)." href="#ref-CR77" id="ref-link-section-d90000874e3048_1">77</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 78" title="Allou, L. et al. Non-coding deletions identify Maenli lncRNA as a limb-specific En1 regulator. Nature 592, 93–98 (2021)." href="https://www.nature.com/articles/s41586-023-06806-x#ref-CR78" id="ref-link-section-d90000874e3051">78</a></sup> (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig6">6a</a>, Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41586-023-06806-x#MOESM8">5</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig15">9a–c</a>). We used matched orthologues to align the two species  and included non-orthologous genes for embedding (see <a data-track="click" data-track-label="link" data-track-action="section anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Sec9">Methods</a>) (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig6">6b</a>). The resulting integrated atlas showed highly conserved cell composition between humans and mice, with similar developmental trajectories of the skeletal muscle and LPM (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig6">6c,d</a> and Extended Data Figs. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig14">8d–g</a>, <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig15">9d</a> and <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig16">10a–c</a>).</p><div data-test="figure" data-container-section="figure" id="figure-6" data-title="Comparison of a single-cell atlas between human and mouse limb."><figure><figcaption><b id="Fig6" data-test="figure-caption-text">Fig. 6: Comparison of a single-cell atlas between human and mouse limb.</b></figcaption><div><div><a data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="https://www.nature.com/articles/s41586-023-06806-x/figures/6" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41586-023-06806-x/MediaObjects/41586_2023_6806_Fig6_HTML.png?as=webp"/><img aria-describedby="Fig6" src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41586-023-06806-x/MediaObjects/41586_2023_6806_Fig6_HTML.png" alt="figure 6" loading="lazy" width="685" height="505"/></picture></a></div><div data-test="bottom-caption" id="figure-6-desc"><p><b>a</b>, Overview of mouse sampling and the experimental scheme. The stars indicate timepoints. <b>b</b>, Overview of the analysis pipeline to integrate human and mouse scRNA-seq data. <b>c</b>, MultiMAP layout of integrated cells, coloured by integrated cell-type annotation or species (bottom right). Cluster abbreviations are the same as in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig1">1</a>. <b>d</b>, Broad cell-type proportions of each scRNA-seq library, with dissection region, location and species labelled at the bottom. NC, neural crest. <b>e</b>,<b>f</b>, Triangular diagrams (<b>e</b>) showing the cell-type proportion biases towards the proximal, middle or distal region of the human and mouse forelimb (left) and hindlimb (right), and a scatter plot (<b>f</b>) showing the fraction of hindlimb representation of each cell type. Each cell type or mean is represented by a circle (human) and a square (mouse), with size (square of diameter) denoting the average number of cells per segment (proximal, middle or distal).</p><p><a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41586-023-06806-x#MOESM15">Source Data</a></p></div></div></figure></div><p>Some species differences were most likely technical, such as the greater abundance of <i>PAX3</i><sup>+</sup> myogenic progenitors in the mouse and the presence of two mouse-enriched mesenchymal populations, ‘early DistMes’ and ‘early ProxMes’, which originated from samples before embryonic day 12 (E12), the equivalent stage to human PCW5 (Extended Data Figs. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig14">8d–e,g</a> and <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig16">10a–c</a>). Similarly, the lack of <i>Wt1</i> expression in the mouse proximal mesenchyme, despite previous description (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig16">10d</a>), is probably due to dissection not including the trunk<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 34" title="Moore, A. W. et al. YAC transgenic analysis reveals Wilms’ tumour 1 gene activity in the proliferating coelomic epithelium, developing diaphragm and limb. Mech. Dev. 79, 169–184 (1998)." href="https://www.nature.com/articles/s41586-023-06806-x#ref-CR34" id="ref-link-section-d90000874e3154">34</a></sup>.</p><p>However, we also identified biological features. Mouse limbs contained a higher percentage of epithelial and immune cells (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig6">6d</a>), possibly due to faster maturation of these systems in the mouse. In skeletal muscle, <i>PAX3</i><sup>+</sup><i>PAX7</i><sup>+</sup> myogenic progenitor cells were more abundant in humans than in mice (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig14">8d–f</a>). Although the <i>PAX3</i><sup>+</sup> pools were transcriptomically similar, the mouse data showed low expression of promyogenic factors <i>Fst</i> and <i>Uchl1</i>, although this may again be due to differences in sample stage<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 79" title="Gilson, H. et al. Follistatin induces muscle hypertrophy through satellite cell proliferation and inhibition of both myostatin and activin. Am. J. Physiol. Endocrinol. Metab. 297, E157–E164 (2009)." href="https://www.nature.com/articles/s41586-023-06806-x#ref-CR79" id="ref-link-section-d90000874e3185">79</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 80" title="Gao, H. et al. UCHL1 regulates muscle fibers and mTORC1 activity in skeletal muscle. Life Sci. 233, 116699 (2019)." href="https://www.nature.com/articles/s41586-023-06806-x#ref-CR80" id="ref-link-section-d90000874e3188">80</a></sup> (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig14">8h</a>). The mesenchymal compartments of both species were highly analogous (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig16">10d</a>). Notably, in both species <i>FGF8</i> was expressed in the proximal mesenchyme; its presence in mesenchyme has been reported only in urodeles<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 81" title="Purushothaman, S., Elewa, A. &amp; Seifert, A. W. Fgf-signaling is compartmentalized within the mesenchyme and controls proliferation during salamander limb development. eLife 10.7554/eLife.48507 (2019)." href="https://www.nature.com/articles/s41586-023-06806-x#ref-CR81" id="ref-link-section-d90000874e3202">81</a></sup> (Extended Data Figs. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig8">2c–e</a> and <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig16">10e,f</a>).</p><p>To explore the proximal–distal axis and forelimb versus hindlimb identities across species, we dissected forelimbs and hindlimbs from each species into proximal, middle and distal segments (Figs. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig1">1a</a> and <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig6">6a</a>). Overall cluster compositions along the proximal–distal axis were highly similar, with proximal and distal/transitional mesenchymal cells enriched in the proximal and distal samples, respectively, and interzone enriched in the middle sections (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig6">6e</a>). Forelimb and hindlimb composition was also highly similar in both species (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig6">6f</a>). Finally, we took cells from the 34 LPM-derived states and compared orthologue expression signatures between proximal versus distal segments and forelimb versus hindlimb samples in mice and humans. Both species recapitulated known proximal–distal-biased genes such as <i>MEIS2</i> (proximal), <i>LHX2</i> (distal) and HOX genes (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig15">9e</a>). Known forelimb/hindlimb-biased genes were also captured, such as <i>TBX5</i> specific to the forelimb and <i>TBX4</i>, <i>PITX1</i> and <i>ISL1</i> specific to the hindlimb (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig15">9f</a>). Overall, we showed that the spatial expression patterns of genes controlling forelimb/hindlimb and proximal–distal identity are highly similar between mice and humans.</p></div></div></section><section data-title="Discussion"><div id="Sec8-section"><h2 id="Sec8">Discussion</h2><p>Our developmental limb atlas combines single-cell and spatial transcriptomics to form the first detailed characterization of human limb development across space and time, identifying and placing 67 clusters of cells into anatomical context<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 76" title="He, P. et al. The changing mouse embryo transcriptome at whole tissue and single-cell resolution. Nature 583, 760–767 (2020)." href="https://www.nature.com/articles/s41586-023-06806-x#ref-CR76" id="ref-link-section-d90000874e3261">76</a></sup>. In doing so, we uncovered several new cell states. We identified neural fibroblasts surrounding the sciatic nerve and two clusters in the tendon lineage mapped to the tendon and perimysium. In the autopod, we described three clusters of cells with subtly different transcriptomes mapped to distinct regions: distal (<i>LHX2</i><sup>+</sup><i>MSX1</i><sup>+</sup><i>SP9</i><sup>+</sup>), <i>RDH10</i><sup><i>+</i></sup>distal (<i>RDH10</i><sup>+</sup><i>LHX2</i><sup>+</sup><i>MSX1</i><sup>+</sup>) and transitional (<i>IRX1</i><sup>+</sup><i>MSX1</i><sup>+</sup>) mesenchyme. Through spatial transcriptomic analysis of the foot plate, we connected physiological gene expression patterns to genetic conditions with a digit phenotype, demonstrating the clinical relevance of developmental cell atlas projects. Our study also presents a refined model of the overlapping processes of primary and secondary myogenesis in the limb, while identifying and validating <i>MSC</i> as a key player in muscle stem cell maintenance. We further maximized the utility of this study by presenting an integrated cross-species atlas with unified annotations. It is important to acknowledge that our findings are limited by the lack of earliest-stage limb samples, the 3′ end bias of our transcriptomic assays, differences in cell-type coverages between scRNA-seq and spatial transcriptomics, as well as the low spatial resolution of spatial transcriptomics (see <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41586-023-06806-x#MOESM1">Supplementary Information</a>). In summary, our work presents one of the first cell atlases of an entire human embryonic tissue, presents detailed spatiotemporal models of its development and provides rich resources for single-cell, spatial and developmental biology communities.</p></div></section><section data-title="Methods"><div id="Sec9-section"><h2 id="Sec9">Methods</h2><div id="Sec9-content"><h3 id="Sec10">Human tissue sample collection</h3><p>First trimester human embryonic tissue was collected from elective termination of pregnancy procedures at Addenbrookes Hospital (Cambridge, UK) under full ethical approval from the East of England–Cambridge Central Research Ethics Committee (REC-96/085; for scRNA-seq and Visium), and at Guangzhou Women and Children’s Medical Center (China) under approval of the Research Ethics Committee of Zhongshan School of Medicine (ZSSOM), Sun Yat-sen University (ZSSOM-2019-075) and Guangzhou Women and Children’s Medical Center (2022-050A01, for in situ hybridization and immunohistochemistry). Consent was obtained after the decision was made to terminate pregnancy, in advance of the procedure. Experiments also followed the 2021 International Society for Stem Cell Research (ISSCR) guidelines in working on human embryos. Informed written consent was obtained from all donors before termination of pregnancy and tissue collection. No developmental abnormalities were visible or known in any of the embryos collected. All human data generated from China were registered at the China National Center for Bioinformation (PRJCA012474) and have been approved by the Chinese Ministry of Science and Technology for the Review and the Approval of Human Genetic Resources (2023BAT0445). For light-sheet fluorescence microscopy, tissues were obtained through INSERM’s HuDeCA Biobank and made available in accordance with the French bylaw. Permission to use human tissues was obtained from the French agency for biomedical research (Agence de la Biomédecine, Saint-Denis La Plaine, France; no. PFS19-012) and the INSERM Ethics Committee (IRB00003888). Written, informed consent was given for tissue collection by all patients. Embryonic age (PCW) was estimated using the independent measurement of the crown rump length, using the formula PCW (days) = 0.9022 × crown rump length (mm) + 27.372. PCW was recorded as week and day, separated by a decimal point; for example, PCW5.6 translates to 5 weeks and 6 days.</p><h3 id="Sec11">Human tissue processing and scRNA-seq data generation</h3><p>Embryonic limbs were dissected from the trunk under a microscope using sterile microsurgical instruments. To capture cells of the interzone, four samples (a hindlimb and a forelimb from both PCW5.6 and PCW6.1) were then further dissected into proximal, middle (containing undisturbed interzone) and distal thirds before dissociation. For the PCW5.1 sample, no further dissection was performed due to small size and the limb was dissociated as a whole. For all other samples, the limb was dissected into proximal and distal halves before dissociation.</p><p>Dissected tissues were mechanically chopped into a mash, and then were digested in Liberase TH solution (50 μg ml<sup>−1</sup>; 05401135001, Roche) at 37 °C for 30–40 min till no tissue piece was visible. Digested tissues were filtered through 40-μm cell strainers followed by centrifugation at 750<i>g</i> for 5 min at 4 °C. Cell pellets were resuspended with 2% FBS in PBS if the embryos were younger than PCW8, otherwise red blood cell lysis (00-4300, eBioscience) was performed. The single-cell suspensions derived from each sample were then loaded onto separate channels of a Chromium 10x Genomics single-cell 3′ version 2 library chip as per the manufacturer’s protocol (PN-120233, 10x Genomics). cDNA sequencing libraries were prepared as per the manufacturer’s protocol and sequenced using an Illumina Hi-seq 4000 with 2 x 150-bp paired-end reads.</p><h3 id="Sec12">Mouse tissue sample collection and scRNA-seq data generation</h3><p>Timed pregnant C57BL/6J wild-type mice were ordered from Jackson Laboratories. On arrival, timed pregnant mice were housed singly and maintained in solid-bottom Zyfone individually ventilated microisolator caging ((Lab Products) 7′′ wide × 12′′ length × 6′′ height). All cages were sanitized in a cagewash facility with a final rinse temperature of at least 180 F° before use. Each cage contained autoclaved hardwood chip bedding (Aspen Chip Bedding, Northeastern Products) and two sheets of tissue paper for nest building enrichment. All mice were fed irradiated standard rodent diet (PicoLab Rodent Diet 5053, PMI Nutrition International), and provided with ad libitum reverse osmosis water via water pouches (Hydropac, Lab Products) on arrival, before the start of any experimental manipulation. Animal rooms were maintained on a 14:10 h light–dark cycle with an hour-long dawn–dusk period with humidity ranging from 30% to 70% and temperatures ranging from 71 °F to 75 °F in compliance with the Guide for the Care and Use of Laboratory Animals. Animals were checked daily by the animal care staff to check for health and the availability of food, water and cage conditions. Embryos were collected at E12.5, E13.5 and E16.5. Only right-side forelimbs and hindlimbs were used in this study: <i>n</i> = 5 at the E12.5 timepoint, <i>n</i> = 5 at E13.5 and <i>n</i> = 2 at E16.5. No randomization, blinding or sample size choice were done. The sex of the embryos was not known or selected. Hindlimbs and forelimbs were pooled separately in ice-cold HBSS (14175-095, Gibco), and dissected into proximal, middle and distal limb regions, which were again separately pooled in 200 μl of HBSS placed in a drop in the centre of a 6-cm culture plate. Tissues were then minced with a razor blade and incubated with an addition of 120 μl of diluted DNase solution (04716728001, Roche) at 37 °C for 15 min. The DNase solution consisted of 1 ml UltraPure water (10977-015, Invitrogen), 110 μl 10× DNase buffer and 70 μl DNase stock solution. Of diluted Liberase TH (05401151001, Roche), 2 ml was then added to the plate, and the minced tissue suspension was pipetted into a 15-ml conical centrifuge tube. The culture plate was rinsed with 2 ml, and again with 1 ml of fresh Liberase TH, which was serially collected and added to the cell suspension. The suspension was incubated at 37 °C for 15 min, triturated with a P1000 tip and incubated for an additional 15 min at 37 °C. For the Liberase TH solution, 50X stock was prepared by adding 2 ml PBS to 5 mg of Liberase TH. Working solution was made by adding 100 μl 50X stock to 4.9 ml PBS. After a final gentle trituration of the tissue with a P1000 tip, the suspension was spun at 380<i>g</i> in a swinging bucket rotor at 4 °C for 5 min. After removing the supernatant, cells were resuspended in 5 ml of 2% FBS in PBS, and filtered through a pre-wetted 40-μm filter (352340, Falcon). After spinning again at 380<i>g</i> at 4 °C for 5 min, the supernatant was removed and cells were resuspended in 200 µl 2% FBS in PBS. A small aliquot was diluted 1:10 in 2% FBS/PBS and mixed with an equal volume of Trypan Blue for counting on a haemocytometer. The full suspension was diluted to 1.2 million cells per millilitre for processing on the 10x Genomics Chromium Controller, with a target of 8,000 cells per library. Libraries were processed according to the manufacturer’s protocol, using the v3 Chromium reagents. All animal procedures were performed according to protocols approved by the Institutional Animal Care and Use Committee at the California Institute of Technology. Animals were housed in an AAALAC-accredited facility in accordance with the Guide for the Care and Use of Laboratory Animals.</p><h3 id="Sec13">Visium spatial transcriptomic experiments of human tissue</h3><p>Whole embryonic limb samples at PCW6–PCW8 were embedded in OCT within cryo wells and flash-frozen using an isopentane and dry ice slurry. Ten-micron-thick cryosections were then cut in the desired plane and transferred onto Visium slides before haematoxylin and eosin staining and imaged at ×20 magnification on a Hamamatsu Nanozoomer 2.0 HT Brightfield. These slides were then further processed according to the 10x Genomics Visium protocol, using a permeabilization time of 18 min for the PCW6 samples and 24 min for older samples. Images were exported as tiled tiffs for analysis. Dual-indexed libraries were prepared as in the 10x Genomics protocol, pooled at 2.25 nM and sequenced four samples per Illumina Novaseq SP flow cell with read lengths of 28 bp R1, 10 bp i7 index, 10 bp i5 index and 90 bp R2.</p><h3 id="Sec14">Digit region analysis of Visium data</h3><p>The Visium data were clustered by the Louvain algorithm after filtering genes that were expressed in less than one spot and performing normalization and logarithmization. After that, the spot clusters of interest were annotated based on haematoxylin and eosin histology and marker genes. The differential expression testing was performed by Wilcoxon test using Scanpy (sc.tl.rank_gene_group).</p><h3 id="Sec15">Alignment, quantification and quality control of scRNA-seq data</h3><p>Droplet-based (10x) sequencing data were aligned and quantified using the Cell Ranger Single-Cell Software Suite (v3.0.2, 10x Genomics). The human reference is the hg38 genome refdata-cellranger-GRCh38-3.0.0, available at: <a href="http://cf.10xgenomics.com/supp/cell-exp/refdata-cellranger-GRCh38-3.0.0.tar.gz">http://cf.10xgenomics.com/supp/cell-exp/refdata-cellranger-GRCh38-3.0.0.tar.gz</a>. The mouse reference is the mm10 reference genome refdata-gex-mm10-2020-A, available at: <a href="https://cf.10xgenomics.com/supp/cell-exp/refdata-gex-mm10-2020-A.tar.gz">https://cf.10xgenomics.com/supp/cell-exp/refdata-gex-mm10-2020-A.tar.gz</a>. Published mouse scRNA-seq FASTQ files were downloaded from ENCODE’s portal and the Gene Expression Omnibus<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="He, P. et al. The changing mouse embryo transcriptome at whole tissue and single-cell resolution. Nature 583, 760–767 (2020)." href="#ref-CR76" id="ref-link-section-d90000874e3396">76</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Kelly, N. H., Huynh, N. P. T. &amp; Guilak, F. Single cell RNA-sequencing reveals cellular heterogeneity and trajectories of lineage specification during murine embryonic limb development. Matrix Biol. 10.1016/j.matbio.2019.12.004 (2020)." href="#ref-CR77" id="ref-link-section-d90000874e3396_1">77</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 78" title="Allou, L. et al. Non-coding deletions identify Maenli lncRNA as a limb-specific En1 regulator. Nature 592, 93–98 (2021)." href="https://www.nature.com/articles/s41586-023-06806-x#ref-CR78" id="ref-link-section-d90000874e3399">78</a></sup>. The following quality control steps were performed: (1) cells that expressed fewer than 200 genes (low quality) were excluded; (2) genes expressed by less than five cells were removed; and (3) cells in which over 10% of unique molecular identifiers were derived from the mitochondrial genome were removed.</p><h3 id="Sec16">Alignment and quantification of human Visium data</h3><p>Raw FASTQ files and histology images were processed, aligned and quantified by sample using the Space Ranger software v.1.1.0, which uses STAR v.2.5.1b52 for genome alignment, against the Cell Ranger hg38 reference genome refdata-cellranger-GRCh38-3.0.0, available at: <a href="http://cf.10xgenomics.com/supp/cell-exp/refdata-cellranger-GRCh38-3.0.0.tar.gz">http://cf.10xgenomics.com/supp/cell-exp/refdata-cellranger-GRCh38-3.0.0.tar.gz</a>.</p><h3 id="Sec17">Doublet detection of scRNA-seq data</h3><p>Doublets were detected with an approach adapted from a previous study<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 82" title="Popescu, D. M. et al. Decoding human fetal liver haematopoiesis. Nature 574, 365–371 (2019)." href="https://www.nature.com/articles/s41586-023-06806-x#ref-CR82" id="ref-link-section-d90000874e3426">82</a></sup>. In the first step of the process, each 10x lane was processed independently using the Scrublet to obtain per-cell doublet scores. In the second step of the process, the standard Scanpy processing pipeline was performed up to the clustering stage, using default parameters<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 83" title="Wolock, S. L., Lopez, R. &amp; Klein, A. M. Scrublet: computational identification of cell doublets in single-cell transcriptomic data. Cell Syst. 8, 281–291.e9 (2019)." href="https://www.nature.com/articles/s41586-023-06806-x#ref-CR83" id="ref-link-section-d90000874e3430">83</a></sup>. Each cluster was subsequently separately clustered again, yielding an over-clustered manifold, and each of the resulting clusters had its Scrublet scores replaced by the median of the observed values. The resulting scores were assessed for statistical significance, with <i>P</i> values computed using a right tailed test from a normal distribution centred on the score median and a median absolute deviation-derived standard deviation estimate. The median absolute deviation was computed from above-median values to circumvent zero truncation. The <i>P</i> values were corrected for false discovery rate with the Benjamini–Hochberg procedure and were used to assess doublet level. The clusters from batch-corrected overall clustering across all the samples that have median scores lower than 0.1 and are supported by an absence of exclusive marker genes or literature were manually curated and removed (1,450 doublets were removed in human data and 958 in mouse data).</p><h3 id="Sec18">Data preprocessing and integration of scRNA-seq data</h3><p>Preprocessing included data normalization (pp.normalize_per_cell with 10,000 counts per cell after normalization), logarithmization (pp.log1p), highly variable genes detection (pp.highly_variable_genes and select for highly correlated ones as previously described<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 76" title="He, P. et al. The changing mouse embryo transcriptome at whole tissue and single-cell resolution. Nature 583, 760–767 (2020)." href="https://www.nature.com/articles/s41586-023-06806-x#ref-CR76" id="ref-link-section-d90000874e3448">76</a></sup>) per batch and merging, data feature scaling (pp.scale), cell cycle and technical variance regressing (tl.score_gene_cell_cycle and pp.regress_out(adata,[‘S_score’, ‘G2M_score’, ‘n_counts’, ‘percent_mito’])), and principal component analysis (tl.pca with 100 components) performed using the Python package Scanpy (v.1.8.2). bbknn (v.1.5.1) was used to correct for batch effect between sample identities with the following parameters (n_pcs = 100, metric = ‘Euclidean’, neighbors_within_batch = 3, trim = 299, approx = false). Following this, further dimension reduction was performed using uniform manifold approximation and projection (UMAP) (scanpy tl.umap with default parameters) based on the corrected neighbourhood graph of bbknn.</p><h3 id="Sec19">Clustering and annotation of scRNA-seq data</h3><p>We first applied Leiden graph-based clustering (scanpy tl.leiden with default parameters) to perform unsupervised cell classification. Each cluster was then subclustered if heterogeneity was still observed and was manually annotated (see Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41586-023-06806-x#MOESM4">1</a> for marker genes) and curated as previously described<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 84" title="He, P. et al. A human fetal lung cell atlas uncovers proximal-distal gradients of differentiation and key regulators of epithelial fates. Cell 185, 4841–4860.e25 (2022)." href="https://www.nature.com/articles/s41586-023-06806-x#ref-CR84" id="ref-link-section-d90000874e3463">84</a></sup>. To make sure all the curated Leiden clusters could clearly be mapped onto their UMAP embedding coordinates, we performed the partition-based graph abstraction (PAGA) (tl.paga with the Leiden clusters) and reran UMAP with the initial position from the PAGA.</p><h3 id="Sec20">Deconvolution of human Visium data using cell2location</h3><p>To map clusters of cells identified by scRNA-seq in the profiled spatial transcriptomics slides, we used the cell2location method<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 85" title="Kleshchevnikov, V. et al. Cell2location maps fine-grained cell types in spatial transcriptomics. Nat. Biotechnol. 40, 661–671 (2022)." href="https://www.nature.com/articles/s41586-023-06806-x#ref-CR85" id="ref-link-section-d90000874e3476">85</a></sup>. In brief, this involved first training a negative binomial regression model to estimate reference transcriptomic profiles for all the scRNA-seq clusters in the developing limb. Next, lowly expressed genes were excluded as per recommendations for use of cell2location, leaving 13,763 genes for downstream analysis. Next, we estimated the abundance of each cluster in the spatial transcriptomics slides using the reference transcriptomic profiles of different clusters. This was applied to all slides simultaneously, using the sample ID as the batch_key and categorical_covariate_keys. To identify microenvironments of colocalizing cell clusters, we used non-negative matrix factorization implementation in scikit-learn, utilizing the wrapper in the cell2location package<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 86" title="Pedregosa, F. et al. Scikit-learn: machine learning in Python. J. Mach. Learn. Res. 12, 2825–2830 (2011)." href="https://www.nature.com/articles/s41586-023-06806-x#ref-CR86" id="ref-link-section-d90000874e3480">86</a></sup>. A cell type was considered part of a microenvironment if the fraction of that cell type in said environment was over 0.2.</p><h3 id="Sec21">Alignment and merging of multiple Visium sections using VisiumStitcher</h3><p>To analyse the whole PCW8.1 human hindlimb, we took three consecutive 10-µm sections from different regions and placed them on different capture areas of the same Visium library preparation slide. The first section spanned the distal femur, knee joint and proximal tibia (sample C42A1), the second the proximal thigh (sample C42B1) and the third the distal tibia, ankle and foot (sample C42C1).</p><p>The images from these three Visium capture areas were then aligned using the TrackEM plugin (Fiji)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 87" title="Schindelin, J. et al. Fiji: an open-source platform for biological-image analysis. Nat. Methods 9, 676–682 (2012)." href="https://www.nature.com/articles/s41586-023-06806-x#ref-CR87" id="ref-link-section-d90000874e3495">87</a></sup>. Following affine transformations of C42B1 and C42C1 to C42A1, the transformation matrices were exported to an in-house pipeline (<a href="https://github.com/Teichlab/limbcellatlas">https://github.com/Teichlab/limbcellatlas</a>) for complementary alignment of the spot positions from the SpaceRanger output to the reconstructed space. In addition, we arbitrarily decided that overlapping regions would keep the image from the centre portion (see Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig12">6a</a>) while keeping all the spots in the data matrix. Next, we merged the three library files and matched the reconstructed image to the unified AnnData object.</p><h3 id="Sec22">Trajectory analysis of human scRNA-seq data</h3><p>Development trajectories were inferred by combining diffusion maps, PAGA and force-directed graph. The first step of this process was to perform the first nonlinear dimensionality reduction using diffusion maps (scanpy tl.diffmap with 15 components) and recompute the neighbourhood graph (scanpy pp.neighbors) based on the 15 components of diffusion maps. In the second step of this process, PAGA (scanpy tl.paga) was performed to generate an abstracted graph of partitions. Finally, force-directed graph was performed with the initial position from PAGA (scanpt tl.draw_graph) to visualize the development trajectories.</p><h3 id="Sec23">RNA velocity calculations for mesenchymal compartment</h3><p>The scVelo version 0.24 package for Python was used to calculate a ratio of spliced-to-unspliced mRNA abundances in the dataset<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 88" title="Bergen, V., Lange, M., Peidli, S., Wolf, F. A. &amp; Theis, F. J. Generalizing RNA velocity to transient cell states through dynamical modeling. Nat. Biotechnol. 38, 1408–1414 (2020)." href="https://www.nature.com/articles/s41586-023-06806-x#ref-CR88" id="ref-link-section-d90000874e3525">88</a></sup>. The data were subclustered to the mesenchymal compartment for a single sample (PCW7.2). The data were then processed using default parameters following preprocessing as described in Scanpy scVelo implementation. The samples were preprocessed using functions for detection of minimum number of counts, filtering and normalization using scv.pp.filter_and_normalize and followed by scv.pp.moments function using default parameters. The gene-specific velocities were then calculated using scv.tl.velocity with mode set to stochastic and scv.tl.velocity_graph functions, and visualized using scv.pl.velocity_graph function.</p><h3 id="Sec24">Cell–cell communication analysis of human scRNA-seq data</h3><p>Cell–cell communication analysis was performed using CellPhoneDB.org (v.2.1.4) for each dataset at the same stage of development<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 89" title="Vento-Tormo, R. et al. Single-cell reconstruction of the early maternal–fetal interface in humans. Nature 563, 347–353 (2018)." href="https://www.nature.com/articles/s41586-023-06806-x#ref-CR89" id="ref-link-section-d90000874e3537">89</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 90" title="Efremova, M., Vento-Tormo, M., Teichmann, S. A. &amp; Vento-Tormo, R. CellPhoneDB: inferring cell-cell communication from combined expression of multi-subunit ligand–receptor complexes. Nat. Protoc. 15, 1484–1506 (2020)." href="https://www.nature.com/articles/s41586-023-06806-x#ref-CR90" id="ref-link-section-d90000874e3540">90</a></sup>. The stage-matched Visium data were used to validate the spatial distance and expression pattern of significant (<i>P</i> &lt; 0.05) ligand–receptor interactions.</p><h3 id="Sec25">Regulon analysis of transcription factors</h3><p>To carry out transcription factor network inference, analysis was performed as previously described<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 91" title="Van de Sande, B. et al. A scalable SCENIC workflow for single-cell gene regulatory network analysis. Nat. Protoc. 15, 2247–2276 (2020)." href="https://www.nature.com/articles/s41586-023-06806-x#ref-CR91" id="ref-link-section-d90000874e3555">91</a></sup> using the pySCENIC Python package (v.0.10.3). For the input data, we filtered out the genes that were expressed in less than 10% of the cells in each cell cluster. Then, we performed the standard procedure including deriving co-expression modules (pyscenic grn), finding enriched motifs (pyscenic ctx) and quantifying activity (pyscenic aucell).</p><h3 id="Sec26">Integration of human and mouse scRNA-seq data</h3><p>Mouse orthologues were first ‘translated’ to human genes using MGI homology database (<a href="https://www.informatics.jax.org/homology.shtml">https://www.informatics.jax.org/homology.shtml</a>). Processed human and mouse data were then merged together using outer join of all the genes. The matched dataset was then integrated by MultiMAP<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 92" title="Jain, M. S. et al. MultiMAP: dimensionality reduction and integration of multimodal data. Genome Biol. 22, 346 (2021)." href="https://www.nature.com/articles/s41586-023-06806-x#ref-CR92" id="ref-link-section-d90000874e3575">92</a></sup> (the MultiMAP_Integration() function), using separately pre-calculated principal components and the union set of previously calculated mouse and human feature genes (including both orthologues and non-orthologues) to maximize biological variance. Downstream clustering and embedding were performed in the same way as previously described and cell-type annotation was based on marker genes. Cell-type composition of proximal, middle and distal segments of the same limb was visualized using plotly.express.scatter_ternary() function. To capture the differential expression of sparsely captured genes, the odds ratio of the percentages of non-zero cells between groups of cells was used to select for proximal/distal or forelimb/hindlimb biased genes with a cut-off at 30-fold and 3-fold, respectively.</p><h3 id="Sec27">Immunohistochemistry</h3><p>The limb samples were post-fixed in 4% paraformaldehyde for 24 h at 4 °C followed by paraffin embedding. A thickness of 4-μm sections were boiled in 0.01 M citrate buffer (pH 6.0) after dewaxing. Immunofluorescence staining was then carried out as previously described<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 93" title="Wang, S. et al. Muscle stem cell immunostaining. Curr. Protoc. Mouse Biol. 8, e47 (2018)." href="https://www.nature.com/articles/s41586-023-06806-x#ref-CR93" id="ref-link-section-d90000874e3587">93</a></sup>. Primary antibodies for RUNX2 (1:50; sc-390715, Santa Cruz), THBS2 (1:100; PA5-76418, Thermo Fisher), COL2A1 (1:200; sc-52658, Santa Cruz), PITX1 (1:30; Ab244308, Abcam), PAX3 (1:1; AB_528426 supernatant, DSHB), ALDH1A3 (1:50; 25167-1-AP, Proteintech) and MYH3 (1:3; AB_528358 supernatant, DSHB) and anti-KERA (1:1,000; HPA039321, Sigma-Aldrich) were incubated overnight at 4 °C. After washing, sections were incubated with appropriate secondary antibodies Alexa Flour 488 goat anti-mouse IgG1 (1:400; A-21121, Invitrogen), Alexa Flour 647 goat anti-mouse IgG2b (1:400; A-21242, Invitrogen), Alexa Flour 488 goat anti-mouse IgG (H + L) (1:400; A-11029, Invitrogen) and Alexa Flour 546 goat anti-rabbit IgG (H + L) (1:400; A-11035, Invitrogen) at room temperature for 1 h, and were mounted using FluorSave Reagent (345789, Calbiochem). For 3,3-diaminobenzidine staining, we used a streptavidin–peroxidase broad spectrum kit (SP-0022, Bioss) and 3,3-diaminobenzidine solution (ZLI-9017, ZSGB-BIO) following the manuals from the manufacturers. The primary antibodies PI16 (1:500; HPA043763, Sigma-Aldrich), FGF19 (1:500; DF2651, Affinity) and NEFH (1:1,000; 2836, Cell Signaling) were applied. Single-plane images were acquired using an inverted microscope (DMi8, Leica).</p><h3 id="Sec28">RNA-ISH</h3><p>Fresh tissue samples were embedded in OCT and frozen at −80 °C until analysis. Cryosections were cut at a thickness of 10 μm or 12 μm using a cryostat (Leica CM1950 or CM3050). Before staining, tissue sections were post-fixed in 4% paraformaldehyde for 15 min at 4 °C. After a series of 50%, 70%, 100% and 100% ethanol dehydration for 5 min each, tissue sections were treated with hydrogen peroxide for 10 min. Next, the sections were digested with protease IV (322336, ACD) for 20–30 min at room temperature; alternatively, they were digested with protease III (322337, ACD) for 15 min after heat-induced epitope retrieval. RNA-ISH was then carried out manually or using BOND RX (Leica) by using the RNAscope Multiplex Fluorescent Reagent Kit v2 Assay (323110, ACD) or the PinpoRNA multiplex fluorescent RNA in situ hybridization kit (PIF2000, GD Pinpoease) according to the instructions by the manufacturers. To visualize targeted RNAs from individual channels, different tyramide signal amplification (TSA) fluorescent substrates were incubated. Two sets of fluorophores TSA520, TSA570 and TSA650 (PANOVUE) and Opal 520, Opal 570 and Opal 650 (Akoya Biosciences) were used and consistent results were obtained. For the staining of four probes, the RNAscope 4-plex Ancillary Kit (323120, ACD) was applied additionally, and a combination of fluorophores TSA520, TSA570, Opal620 and Opal690 were used. The stained sections were imaged with either AxioScan.Z1 (Zeiss) or the Opera Phenix High-Content Screening System (PerkinElmer).</p><h3 id="Sec29">RNA-ISH colocalization analysis</h3><p>Colocalization analysis was performed by first identifying the expressed genes on raw images through the utilization of a pixel classifier trained with the software ilastik<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 94" title="Berg, S. et al. ilastik: Interactive machine learning for (bio)image analysis. Nat. Methods 16, 1226–1232 (2019)." href="https://www.nature.com/articles/s41586-023-06806-x#ref-CR94" id="ref-link-section-d90000874e3607">94</a></sup>. Subsequently, the predicted mask image was subjected to analysis, with the probability of co-occurrence determined by tallying the instances in which one gene coexists with another at the same 0.14 × 0.14 μm pixel, and dividing this by the total number of pixels in which the gene of interest was expressed, regardless of the presence of the other gene.</p><h3 id="Sec30">Light-sheet fluorescence microscopy</h3><p>Embryonic and fetal limbs were dissected from morphologically normal specimens collected from PCW5 to PCW6.5. Candidate antibodies were screened by immunofluorescence on cryosections obtained from OCT-embedded specimens as previously described<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 95" title="Belle, M. et al. Tridimensional visualization and analysis of early human development. Cell 169, 161–173.e12 (2017)." href="https://www.nature.com/articles/s41586-023-06806-x#ref-CR95" id="ref-link-section-d90000874e3619">95</a></sup>. Whole-mount immunostaining of the limbs was performed as previously described, with primary antibody incubation at 37 °C reduced to 3 days followed by 1 day in secondary antibodies. Samples were embedded in 1.5% agarose and optically cleared with solvents using the iDisco+ method. Cleared samples were imaged with a Blaze light-sheet microscope (Miltenyi Biotec) equipped with a 5.5MP sCMOS camera controlled by Imspector Pro 7.5.3 acquisition software. A ×12 objective with ×0.6 or ×1 magnification (MI plan NA 0.53) was used. Imaris (v10.0, BitPlane) was used for image conversion, processing and video production.</p><h3 id="Sec31">The antibodies used for light-sheet fluorescence microscopy</h3><p>IRX1 Sigma-Aldrich cat. no. HPA043160, RRID: AB_10794771 (1/200e); MSX1 R&amp;D Systems cat. no. AF5045, RRID: AB_2148804 (1/500e); LHX2 Abcam cat. no. ab184337, RRID: AB_2916270 (1/1,000e); SOX9 Abcam cat. no. ab196184, RRID: AB_2813853 (1/500e); MAFB Abcam cat. no. ab223744, RRID: AB_2894837 (1/500e); donkey anti-rabbit IgG H&amp;L (Alexa Fluor 555) Abcam cat. no. ab150062, RRID: AB_2801638 (1/800e); and donkey anti-goat IgG H&amp;L (Alexa Fluor 750) Abcam cat. no. ab175745, RRID: AB_2924800 (1/300e).</p><h3 id="Sec32">
                        <i>MSC</i> knockdown in human primary myoblasts</h3><h4 id="Sec33">Isolation of human primary myoblast cells</h4><p>The thighs from human embryos were processed as previously described<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 96" title="Lapan, A. D. &amp; Gussoni, E. Isolation and characterization of human fetal myoblasts. Methods Mol. Biol. 798, 3–19 (2012)." href="https://www.nature.com/articles/s41586-023-06806-x#ref-CR96" id="ref-link-section-d90000874e3648">96</a></sup>, except that the dissociated cells were not treated with erythrocyte lysis solution, and were incubated with anti-human CD31 (12-0319-41, eBioscience), CD45 (12-0459-41, eBioscience) and CD184 (17-9999-41, eBioscience) antibodies for cell sorting. Fluorescent activated cell sorting (BD, influx) sorted CD31<sup>−</sup>CD45<sup>−</sup>CD184<sup>+</sup> cells were cultured in complete growth medium DMEM supplemented with 20% FCS and 1% penicillin–streptomycin (15140122, Gibco).</p><h4 id="Sec34">Small interfering RNA transfection</h4><p>Human primary myoblasts were seeded into a six-well plate one night before transfection. When the cell density reached approximately 50% confluence, oligos of small interfering RNA against <i>MSC</i> and negative control were transfected using Lipofectamine 3000 reagent (L3000015, Invitrogen) at a final concentration of 37.5 nM. After incubation for 16 h, the growth medium was replaced with differentiation medium containing 2% horse serum and 1% penicillin–streptomycin in DMEM. After culturing for an additional 6–8 h, the cells were collected for RNA extraction. Initially, three siRNA oligos (9242-1, 9242-2 and 9242-3, Bioneer) were tested, and the third one with sense sequences 5′-GAAGUUUCCGCAGCCAACA-3′ were used in this study.</p><h4 id="Sec35">RNA extraction and qPCR</h4><p>Total cell RNA was extracted with the EZ-press RNA purification kit (B0004D, EZBioscience), and the cDNA was synthetized using the PrimeScript RT Master Mix Kit (RR036A, Takara). The qPCR was performed using PerfectStart Green qPCR Super Mix (AQ601, TransGen Biotech) on a real-time PCR detection system (LightCycle480 II, Roche). <i>RPLP0</i> served as an internal control, and the fold enrichment was calculated using the formula 2<sup>−ΔΔCt</sup>. The following primers (5′−3′) were used:</p><p><i>RPLP0</i> forward: ATGCAGCAGATCCGCATGT, reverse: TTGCGCATCATGGTGTTCTT; <i>MSC</i> forward: CAGGAGGACCGCTATGAGAA, reverse: GCGGTGGTTCCACATAGTCT; <i>MYOG</i> forward: AGTGCCATCCAGTACATCGAGC, reverse: AGGCGCTGTGAGAGCTGCATTC; <i>MYH2</i> forward: GGAGGACAAAGTCAACACCCTG, reverse: GCCCTTTCTAGGTCCATGCGAA; <i>MYH3</i> forward: CTGGAGGATGAATGCTCAGAGC, reverse: CCCAGAGAGTTCCTCAGTAAGG; <i>MYH4</i> forward: CGGGAGGTTCACACAAAAGTCATA, reverse: CCTTGATATACAGGACAGTGACAA; <i>TNNT1</i> forward: AACGCGAACGTCAGGCTAAGCT, reverse: CTTGACCAGGTAGCCGCCAAAA.</p><h3 id="Sec36">Ethics statement</h3><p>The work done in the UK was supported by the National Institute for Health and Care Research Cambridge Biomedical Research Centre (NIHR203312) and provided by the Cambridge Biorepository for Translational Medicine (<a href="https://www.cbtm.group.cam.ac.uk">https://www.cbtm.group.cam.ac.uk</a>). Human fetal samples were provided by the National Institute for Health and Care Research Cambridge Biomedical Research Centre and collected under the Research Ethics Committee-approved study 96/085. The work done in China was approved by the Research Ethics Committee of Zhongshan School of Medicine (ZSSOM), Sun Yat-sen University (ZSSOM-2019-075) and Guangzhou Women and Children’s Medical Center (2022-050A01). At both centres, consent was obtained from the patient following the decision to terminate the pregnancy and in advance of the procedure. All animal procedures were performed according to protocols approved by the Institutional Animal Care and Use Committee at the California Institute of Technology. Details of human tissue sample collection are in the Methods section of this article. The views expressed are those of the authors and not necessarily those of the National Institute for Health and Care Research or the Department of Health and Social Care.</p><h3 id="Sec37">Reporting summary</h3><p>Further information on research design is available in the <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41586-023-06806-x#MOESM2">Nature Portfolio Reporting Summary</a> linked to this article.</p></div></div></section>
                </div><div>
            <section data-title="Data availability"><div id="data-availability-section"><h2 id="data-availability">Data availability</h2><p>All of our newly generated raw data are publicly available on ArrayExpress (mouse scRNA-seq, <a href="http://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-10514/">E-MTAB-10514</a>; human Visium, <a href="http://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-10367/">E-MTAB-10367</a>; and human scRNA-seq, <a href="http://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-8813/">E-MTAB-8813</a>). Previously published raw data can be found from the ENCODE portal (<a href="https://www.encodeproject.org/publication-data/ENCSR713GIS/">ENCSR713GIS</a>) and the Gene Expression Omnibus (<a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE137335">GSE137335</a> and <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE142425">GSE142425</a>). Processed data can be downloaded and visualized at our data portal (<a href="https://limb-dev.cellgeni.sanger.ac.uk/">https://limb-dev.cellgeni.sanger.ac.uk/</a>). The data deposited and made public are compliant with the regulations of Ministry of Science and Technology of the People’s Republic of China. <a data-track="click" data-track-label="link" data-track-action="section anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Sec41">Source data</a> are provided with this paper.</p></div></section><section data-title="Code availability"><div id="code-availability-section"><h2 id="code-availability">Code availability</h2><div id="code-availability-content">
            
            <p>All in-house code can be found on GitHub (<a href="https://github.com/Teichlab/limbcellatlas/">https://github.com/Teichlab/limbcellatlas/</a>), which is central to our conclusions.</p>
          </div></div></section><section data-title="Change history"><div id="change-history-section"><h2 id="change-history">Change history</h2><div id="change-history-content"><ul><li id="chg1"><ins datetime="2023-12-12"><h3>12 December 2023</h3><p>In the initially published version of this article, Present address footnotes were incorrectly listed for the first four authors, which are now removed in the HTML and PDF versions of the article.</p></ins></li></ul></div></div></section><div id="MagazineFulltextArticleBodySuffix"><section aria-labelledby="Bib1" data-title="References"><div id="Bib1-section"><h2 id="Bib1">References</h2><div id="Bib1-content"><div data-container-section="references"><ol data-track-component="outbound reference"><li data-counter="1."><p id="ref-CR1">Zuniga, A. Next generation limb development and evolution: old questions, new perspectives. <i>Development</i> <b>142</b>, 3810–3820 (2015).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1242/dev.125757" data-track-action="article reference" href="https://doi.org/10.1242%2Fdev.125757" aria-label="Article reference 1" data-doi="10.1242/dev.125757">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XmvFGiur0%3D" aria-label="CAS reference 1">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26577204" aria-label="PubMed reference 1">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 1" href="http://scholar.google.com/scholar_lookup?&amp;title=Next%20generation%20limb%20development%20and%20evolution%3A%20old%20questions%2C%20new%20perspectives&amp;journal=Development&amp;doi=10.1242%2Fdev.125757&amp;volume=142&amp;pages=3810-3820&amp;publication_year=2015&amp;author=Zuniga%2CA">
                    Google Scholar</a> 
                </p></li><li data-counter="2."><p id="ref-CR2">Wilkie, A. O. M. Why study human limb malformations? <i>J. Anat.</i> <b>202</b>, 27–35 (2003).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1046/j.1469-7580.2003.00130.x" data-track-action="article reference" href="https://doi.org/10.1046%2Fj.1469-7580.2003.00130.x" aria-label="Article reference 2" data-doi="10.1046/j.1469-7580.2003.00130.x">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=12587917" aria-label="PubMed reference 2">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1571054" aria-label="PubMed Central reference 2">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 2" href="http://scholar.google.com/scholar_lookup?&amp;title=Why%20study%20human%20limb%20malformations%3F&amp;journal=J.%20Anat.&amp;doi=10.1046%2Fj.1469-7580.2003.00130.x&amp;volume=202&amp;pages=27-35&amp;publication_year=2003&amp;author=Wilkie%2CAOM">
                    Google Scholar</a> 
                </p></li><li data-counter="3."><p id="ref-CR3">Chevallier, A., Kieny, M. &amp; Mauger, A. Limb-somite relationship: origin of the limb musculature. <i>J. Embryol. Exp. Morphol.</i> <b>41</b>, 245–258 (1977).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:STN:280:DyaE1c%2FmvVShtw%3D%3D" aria-label="CAS reference 3">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=591873" aria-label="PubMed reference 3">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 3" href="http://scholar.google.com/scholar_lookup?&amp;title=Limb-somite%20relationship%3A%20origin%20of%20the%20limb%20musculature&amp;journal=J.%20Embryol.%20Exp.%20Morphol.&amp;volume=41&amp;pages=245-258&amp;publication_year=1977&amp;author=Chevallier%2CA&amp;author=Kieny%2CM&amp;author=Mauger%2CA">
                    Google Scholar</a> 
                </p></li><li data-counter="4."><p id="ref-CR4">Tabin, C. &amp; Wolpert, L. Rethinking the proximodistal axis of the vertebrate limb in the molecular era. <i>Genes Dev.</i> <b>21</b>, 1433–1442 (2007).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1101/gad.1547407" data-track-action="article reference" href="https://doi.org/10.1101%2Fgad.1547407" aria-label="Article reference 4" data-doi="10.1101/gad.1547407">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2sXntlCrtb0%3D" aria-label="CAS reference 4">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17575045" aria-label="PubMed reference 4">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 4" href="http://scholar.google.com/scholar_lookup?&amp;title=Rethinking%20the%20proximodistal%20axis%20of%20the%20vertebrate%20limb%20in%20the%20molecular%20era&amp;journal=Genes%20Dev.&amp;doi=10.1101%2Fgad.1547407&amp;volume=21&amp;pages=1433-1442&amp;publication_year=2007&amp;author=Tabin%2CC&amp;author=Wolpert%2CL">
                    Google Scholar</a> 
                </p></li><li data-counter="5."><p id="ref-CR5">McQueen, C. &amp; Towers, M. Establishing the pattern of the vertebrate limb. <i>Development</i> <b>147</b>, dev177956 (2020).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1242/dev.177956" data-track-action="article reference" href="https://doi.org/10.1242%2Fdev.177956" aria-label="Article reference 5" data-doi="10.1242/dev.177956">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhvFygtLfP" aria-label="CAS reference 5">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32917670" aria-label="PubMed reference 5">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 5" href="http://scholar.google.com/scholar_lookup?&amp;title=Establishing%20the%20pattern%20of%20the%20vertebrate%20limb&amp;journal=Development&amp;doi=10.1242%2Fdev.177956&amp;volume=147&amp;publication_year=2020&amp;author=McQueen%2CC&amp;author=Towers%2CM">
                    Google Scholar</a> 
                </p></li><li data-counter="6."><p id="ref-CR6">Xi, H. et al. A human skeletal muscle atlas identifies the trajectories of stem and progenitor cells across development and from human pluripotent stem cells. <i>Cell Stem Cell</i> <b>27</b>, 158–176.e10 (2020).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.stem.2020.04.017" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.stem.2020.04.017" aria-label="Article reference 6" data-doi="10.1016/j.stem.2020.04.017">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXptlegsrg%3D" aria-label="CAS reference 6">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32396864" aria-label="PubMed reference 6">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7367475" aria-label="PubMed Central reference 6">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 6" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20human%20skeletal%20muscle%20atlas%20identifies%20the%20trajectories%20of%20stem%20and%20progenitor%20cells%20across%20development%20and%20from%20human%20pluripotent%20stem%20cells&amp;journal=Cell%20Stem%20Cell&amp;doi=10.1016%2Fj.stem.2020.04.017&amp;volume=27&amp;pages=158-176.e10&amp;publication_year=2020&amp;author=Xi%2CH">
                    Google Scholar</a> 
                </p></li><li data-counter="7."><p id="ref-CR7">Satoda, M., Pierpont, M. E., Diaz, G. A., Bornemeier, R. A. &amp; Gelb, B. D. Char syndrome, an inherited disorder with patent ductus arteriosus, maps to chromosome 6p12-p21. <i>Circulation</i> <b>99</b>, 3036–3042 (1999).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1161/01.CIR.99.23.3036" data-track-action="article reference" href="https://doi.org/10.1161%2F01.CIR.99.23.3036" aria-label="Article reference 7" data-doi="10.1161/01.CIR.99.23.3036">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:STN:280:DyaK1M3ps1ClsQ%3D%3D" aria-label="CAS reference 7">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=10368122" aria-label="PubMed reference 7">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 7" href="http://scholar.google.com/scholar_lookup?&amp;title=Char%20syndrome%2C%20an%20inherited%20disorder%20with%20patent%20ductus%20arteriosus%2C%20maps%20to%20chromosome%206p12-p21&amp;journal=Circulation&amp;doi=10.1161%2F01.CIR.99.23.3036&amp;volume=99&amp;pages=3036-3042&amp;publication_year=1999&amp;author=Satoda%2CM&amp;author=Pierpont%2CME&amp;author=Diaz%2CGA&amp;author=Bornemeier%2CRA&amp;author=Gelb%2CBD">
                    Google Scholar</a> 
                </p></li><li data-counter="8."><p id="ref-CR8">Cunningham, T. J., Chatzi, C., Sandell, L. L., Trainor, P. A. &amp; Duester, G. Rdh10 mutants deficient in limb field retinoic acid signaling exhibit normal limb patterning but display interdigital webbing. <i>Dev. Dyn.</i> <b>240</b>, 1142–1150 (2011).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1002/dvdy.22583" data-track-action="article reference" href="https://doi.org/10.1002%2Fdvdy.22583" aria-label="Article reference 8" data-doi="10.1002/dvdy.22583">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXkvFGgsrY%3D" aria-label="CAS reference 8">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21360789" aria-label="PubMed reference 8">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3081420" aria-label="PubMed Central reference 8">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 8" href="http://scholar.google.com/scholar_lookup?&amp;title=Rdh10%20mutants%20deficient%20in%20limb%20field%20retinoic%20acid%20signaling%20exhibit%20normal%20limb%20patterning%20but%20display%20interdigital%20webbing&amp;journal=Dev.%20Dyn.&amp;doi=10.1002%2Fdvdy.22583&amp;volume=240&amp;pages=1142-1150&amp;publication_year=2011&amp;author=Cunningham%2CTJ&amp;author=Chatzi%2CC&amp;author=Sandell%2CLL&amp;author=Trainor%2CPA&amp;author=Duester%2CG">
                    Google Scholar</a> 
                </p></li><li data-counter="9."><p id="ref-CR9">Díaz-Hernández, M. E., Bustamante, M., Galván-Hernández, C. I. &amp; Chimal-Monroy, J. Irx1 and Irx2 are coordinately expressed and regulated by retinoic acid, TGFβ and FGF signaling during chick hindlimb development. <i>PLoS ONE</i> <b>8</b>, e58549 (2013).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1371/journal.pone.0058549" data-track-action="article reference" href="https://doi.org/10.1371%2Fjournal.pone.0058549" aria-label="Article reference 9" data-doi="10.1371/journal.pone.0058549">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2013PLoSO...858549D" aria-label="ADS reference 9">ADS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23505533" aria-label="PubMed reference 9">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3594311" aria-label="PubMed Central reference 9">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 9" href="http://scholar.google.com/scholar_lookup?&amp;title=Irx1%20and%20Irx2%20are%20coordinately%20expressed%20and%20regulated%20by%20retinoic%20acid%2C%20TGF%CE%B2%20and%20FGF%20signaling%20during%20chick%20hindlimb%20development&amp;journal=PLoS%20ONE&amp;doi=10.1371%2Fjournal.pone.0058549&amp;volume=8&amp;publication_year=2013&amp;author=D%C3%ADaz-Hern%C3%A1ndez%2CME&amp;author=Bustamante%2CM&amp;author=Galv%C3%A1n-Hern%C3%A1ndez%2CCI&amp;author=Chimal-Monroy%2CJ">
                    Google Scholar</a> 
                </p></li><li data-counter="10."><p id="ref-CR10">Richardson, R. J. et al. Periderm prevents pathological epithelial adhesions during embryogenesis. <i>J. Clin. Invest.</i> <b>124</b>, 3891–3900 (2014).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1172/JCI71946" data-track-action="article reference" href="https://doi.org/10.1172%2FJCI71946" aria-label="Article reference 10" data-doi="10.1172/JCI71946">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXhsFalu7nJ" aria-label="CAS reference 10">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25133425" aria-label="PubMed reference 10">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4151209" aria-label="PubMed Central reference 10">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 10" href="http://scholar.google.com/scholar_lookup?&amp;title=Periderm%20prevents%20pathological%20epithelial%20adhesions%20during%20embryogenesis&amp;journal=J.%20Clin.%20Invest.&amp;doi=10.1172%2FJCI71946&amp;volume=124&amp;pages=3891-3900&amp;publication_year=2014&amp;author=Richardson%2CRJ">
                    Google Scholar</a> 
                </p></li><li data-counter="11."><p id="ref-CR11">McGlinn, E. et al. Pax9 and Jagged1 act downstream of Gli3 in vertebrate limb development. <i>Mech. Dev.</i> <b>122</b>, 1218–1233 (2005).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.mod.2005.06.012" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.mod.2005.06.012" aria-label="Article reference 11" data-doi="10.1016/j.mod.2005.06.012">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2MXhtFeiur%2FP" aria-label="CAS reference 11">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16169709" aria-label="PubMed reference 11">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 11" href="http://scholar.google.com/scholar_lookup?&amp;title=Pax9%20and%20Jagged1%20act%20downstream%20of%20Gli3%20in%20vertebrate%20limb%20development&amp;journal=Mech.%20Dev.&amp;doi=10.1016%2Fj.mod.2005.06.012&amp;volume=122&amp;pages=1218-1233&amp;publication_year=2005&amp;author=McGlinn%2CE">
                    Google Scholar</a> 
                </p></li><li data-counter="12."><p id="ref-CR12">Kuijper, S. et al. Function and regulation of Alx4 in limb development: complex genetic interactions with Gli3 and Shh. <i>Dev. Biol.</i> <b>285</b>, 533–544 (2005).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.ydbio.2005.06.017" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.ydbio.2005.06.017" aria-label="Article reference 12" data-doi="10.1016/j.ydbio.2005.06.017">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2MXhtVymsLjL" aria-label="CAS reference 12">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16039644" aria-label="PubMed reference 12">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 12" href="http://scholar.google.com/scholar_lookup?&amp;title=Function%20and%20regulation%20of%20Alx4%20in%20limb%20development%3A%20complex%20genetic%20interactions%20with%20Gli3%20and%20Shh&amp;journal=Dev.%20Biol.&amp;doi=10.1016%2Fj.ydbio.2005.06.017&amp;volume=285&amp;pages=533-544&amp;publication_year=2005&amp;author=Kuijper%2CS">
                    Google Scholar</a> 
                </p></li><li data-counter="13."><p id="ref-CR13">Quinn, M. E., Haaning, A. &amp; Ware, S. M. Preaxial polydactyly caused by Gli3 haploinsufficiency is rescued by Zic3 loss of function in mice. <i>Hum. Mol. Genet.</i> <b>21</b>, 1888–1896 (2012).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1093/hmg/dds002" data-track-action="article reference" href="https://doi.org/10.1093%2Fhmg%2Fdds002" aria-label="Article reference 13" data-doi="10.1093/hmg/dds002">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XkvVemtrc%3D" aria-label="CAS reference 13">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22234993" aria-label="PubMed reference 13">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3313802" aria-label="PubMed Central reference 13">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 13" href="http://scholar.google.com/scholar_lookup?&amp;title=Preaxial%20polydactyly%20caused%20by%20Gli3%20haploinsufficiency%20is%20rescued%20by%20Zic3%20loss%20of%20function%20in%20mice&amp;journal=Hum.%20Mol.%20Genet.&amp;doi=10.1093%2Fhmg%2Fdds002&amp;volume=21&amp;pages=1888-1896&amp;publication_year=2012&amp;author=Quinn%2CME&amp;author=Haaning%2CA&amp;author=Ware%2CSM">
                    Google Scholar</a> 
                </p></li><li data-counter="14."><p id="ref-CR14">Riddle, R. D., Johnson, R. L., Laufer, E. &amp; Tabin, C. Sonic hedgehog mediates the polarizing activity of the ZPA. <i>Cell</i> <b>75</b>, 1401–1416 (1993).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/0092-8674(93)90626-2" data-track-action="article reference" href="https://doi.org/10.1016%2F0092-8674%2893%2990626-2" aria-label="Article reference 14" data-doi="10.1016/0092-8674(93)90626-2">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK2cXhvVOmsb8%3D" aria-label="CAS reference 14">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=8269518" aria-label="PubMed reference 14">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 14" href="http://scholar.google.com/scholar_lookup?&amp;title=Sonic%20hedgehog%20mediates%20the%20polarizing%20activity%20of%20the%20ZPA&amp;journal=Cell&amp;doi=10.1016%2F0092-8674%2893%2990626-2&amp;volume=75&amp;pages=1401-1416&amp;publication_year=1993&amp;author=Riddle%2CRD&amp;author=Johnson%2CRL&amp;author=Laufer%2CE&amp;author=Tabin%2CC">
                    Google Scholar</a> 
                </p></li><li data-counter="15."><p id="ref-CR15">Huangfu, D. &amp; Anderson, K. V. Signaling from Smo to Ci/Gli: conservation and divergence of Hedgehog pathways from <i>Drosophila</i> to vertebrates. <i>Development</i> <b>133</b>, 3–14 (2006).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1242/dev.02169" data-track-action="article reference" href="https://doi.org/10.1242%2Fdev.02169" aria-label="Article reference 15" data-doi="10.1242/dev.02169">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD28XhtFGgtLc%3D" aria-label="CAS reference 15">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16339192" aria-label="PubMed reference 15">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 15" href="http://scholar.google.com/scholar_lookup?&amp;title=Signaling%20from%20Smo%20to%20Ci%2FGli%3A%20conservation%20and%20divergence%20of%20Hedgehog%20pathways%20from%20Drosophila%20to%20vertebrates&amp;journal=Development&amp;doi=10.1242%2Fdev.02169&amp;volume=133&amp;pages=3-14&amp;publication_year=2006&amp;author=Huangfu%2CD&amp;author=Anderson%2CKV">
                    Google Scholar</a> 
                </p></li><li data-counter="16."><p id="ref-CR16">Kherdjemil, Y. et al. Evolution of Hoxa11 regulation in vertebrates is linked to the pentadactyl state. <i>Nature</i> <b>539</b>, 89–92 (2016).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/nature19813" data-track-action="article reference" href="https://doi.org/10.1038%2Fnature19813" aria-label="Article reference 16" data-doi="10.1038/nature19813">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2016Natur.539...89K" aria-label="ADS reference 16">ADS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28Xhs1amu7vP" aria-label="CAS reference 16">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27706137" aria-label="PubMed reference 16">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5558051" aria-label="PubMed Central reference 16">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 16" href="http://scholar.google.com/scholar_lookup?&amp;title=Evolution%20of%20Hoxa11%20regulation%20in%20vertebrates%20is%20linked%20to%20the%20pentadactyl%20state&amp;journal=Nature&amp;doi=10.1038%2Fnature19813&amp;volume=539&amp;pages=89-92&amp;publication_year=2016&amp;author=Kherdjemil%2CY">
                    Google Scholar</a> 
                </p></li><li data-counter="17."><p id="ref-CR17">McCulloch, D. R. et al. ADAMTS metalloproteases generate active versican fragments that regulate interdigital web regression. <i>Dev. Cell</i> <b>17</b>, 687–698 (2009).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.devcel.2009.09.008" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.devcel.2009.09.008" aria-label="Article reference 17" data-doi="10.1016/j.devcel.2009.09.008">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXhsFKltb7N" aria-label="CAS reference 17">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19922873" aria-label="PubMed reference 17">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2780442" aria-label="PubMed Central reference 17">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 17" href="http://scholar.google.com/scholar_lookup?&amp;title=ADAMTS%20metalloproteases%20generate%20active%20versican%20fragments%20that%20regulate%20interdigital%20web%20regression&amp;journal=Dev.%20Cell&amp;doi=10.1016%2Fj.devcel.2009.09.008&amp;volume=17&amp;pages=687-698&amp;publication_year=2009&amp;author=McCulloch%2CDR">
                    Google Scholar</a> 
                </p></li><li data-counter="18."><p id="ref-CR18">Kaltcheva, M. M., Anderson, M. J., Harfe, B. D. &amp; Lewandoski, M. BMPs are direct triggers of interdigital programmed cell death. <i>Dev. Biol.</i> <b>411</b>, 266–276 (2016).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.ydbio.2015.12.016" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.ydbio.2015.12.016" aria-label="Article reference 18" data-doi="10.1016/j.ydbio.2015.12.016">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XhsFygtrc%3D" aria-label="CAS reference 18">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26826495" aria-label="PubMed reference 18">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7028150" aria-label="PubMed Central reference 18">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 18" href="http://scholar.google.com/scholar_lookup?&amp;title=BMPs%20are%20direct%20triggers%20of%20interdigital%20programmed%20cell%20death&amp;journal=Dev.%20Biol.&amp;doi=10.1016%2Fj.ydbio.2015.12.016&amp;volume=411&amp;pages=266-276&amp;publication_year=2016&amp;author=Kaltcheva%2CMM&amp;author=Anderson%2CMJ&amp;author=Harfe%2CBD&amp;author=Lewandoski%2CM">
                    Google Scholar</a> 
                </p></li><li data-counter="19."><p id="ref-CR19">Díaz-Hernández, M. E., Rios-Flores, A. J., Abarca-Buis, R. F., Bustamante, M. &amp; Chimal-Monroy, J. Molecular control of interdigital cell death and cell differentiation by retinoic acid during digit development. <i>J. Dev. Biol.</i> <b>2</b>, 138–157 (2014).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.3390/jdb2020138" data-track-action="article reference" href="https://doi.org/10.3390%2Fjdb2020138" aria-label="Article reference 19" data-doi="10.3390/jdb2020138">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 19" href="http://scholar.google.com/scholar_lookup?&amp;title=Molecular%20control%20of%20interdigital%20cell%20death%20and%20cell%20differentiation%20by%20retinoic%20acid%20during%20digit%20development&amp;journal=J.%20Dev.%20Biol.&amp;doi=10.3390%2Fjdb2020138&amp;volume=2&amp;pages=138-157&amp;publication_year=2014&amp;author=D%C3%ADaz-Hern%C3%A1ndez%2CME&amp;author=Rios-Flores%2CAJ&amp;author=Abarca-Buis%2CRF&amp;author=Bustamante%2CM&amp;author=Chimal-Monroy%2CJ">
                    Google Scholar</a> 
                </p></li><li data-counter="20."><p id="ref-CR20">Spagnoli, A. et al. TGF-β signaling is essential for joint morphogenesis. <i>J. Cell Biol.</i> <b>177</b>, 1105–1117 (2007).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1083/jcb.200611031" data-track-action="article reference" href="https://doi.org/10.1083%2Fjcb.200611031" aria-label="Article reference 20" data-doi="10.1083/jcb.200611031">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2sXmvF2qsr4%3D" aria-label="CAS reference 20">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17576802" aria-label="PubMed reference 20">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2064369" aria-label="PubMed Central reference 20">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 20" href="http://scholar.google.com/scholar_lookup?&amp;title=TGF-%CE%B2%20signaling%20is%20essential%20for%20joint%20morphogenesis&amp;journal=J.%20Cell%20Biol.&amp;doi=10.1083%2Fjcb.200611031&amp;volume=177&amp;pages=1105-1117&amp;publication_year=2007&amp;author=Spagnoli%2CA">
                    Google Scholar</a> 
                </p></li><li data-counter="21."><p id="ref-CR21">Mokuda, S. et al. Wwp2 maintains cartilage homeostasis through regulation of Adamts5. <i>Nat. Commun.</i> <b>10</b>, 2429 (2019).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/s41467-019-10177-1" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41467-019-10177-1" aria-label="Article reference 21" data-doi="10.1038/s41467-019-10177-1">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2019NatCo..10.2429M" aria-label="ADS reference 21">ADS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31160553" aria-label="PubMed reference 21">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6546747" aria-label="PubMed Central reference 21">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 21" href="http://scholar.google.com/scholar_lookup?&amp;title=Wwp2%20maintains%20cartilage%20homeostasis%20through%20regulation%20of%20Adamts5&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-019-10177-1&amp;volume=10&amp;publication_year=2019&amp;author=Mokuda%2CS">
                    Google Scholar</a> 
                </p></li><li data-counter="22."><p id="ref-CR22">Wu, C.-L. et al. Single cell transcriptomic analysis of human pluripotent stem cell chondrogenesis. <i>Nat. Commun.</i> <b>12</b>, 362 (2021).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/s41467-020-20598-y" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41467-020-20598-y" aria-label="Article reference 22" data-doi="10.1038/s41467-020-20598-y">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2021NatCo..12..362W" aria-label="ADS reference 22">ADS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhsFGlt7s%3D" aria-label="CAS reference 22">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33441552" aria-label="PubMed reference 22">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7806634" aria-label="PubMed Central reference 22">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 22" href="http://scholar.google.com/scholar_lookup?&amp;title=Single%20cell%20transcriptomic%20analysis%20of%20human%20pluripotent%20stem%20cell%20chondrogenesis&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-020-20598-y&amp;volume=12&amp;publication_year=2021&amp;author=Wu%2CC-L">
                    Google Scholar</a> 
                </p></li><li data-counter="23."><p id="ref-CR23">Zhou, T. et al. Piezo1/2 mediate mechanotransduction essential for bone formation through concerted activation of NFAT–YAP1–β-catenin. <i>eLife</i> <b>9</b>, e52779 (2020).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.7554/eLife.52779" data-track-action="article reference" href="https://doi.org/10.7554%2FeLife.52779" aria-label="Article reference 23" data-doi="10.7554/eLife.52779">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhslWiur3O" aria-label="CAS reference 23">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32186512" aria-label="PubMed reference 23">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7112954" aria-label="PubMed Central reference 23">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 23" href="http://scholar.google.com/scholar_lookup?&amp;title=Piezo1%2F2%20mediate%20mechanotransduction%20essential%20for%20bone%20formation%20through%20concerted%20activation%20of%20NFAT%E2%80%93YAP1%E2%80%93%CE%B2-catenin&amp;journal=eLife&amp;doi=10.7554%2FeLife.52779&amp;volume=9&amp;publication_year=2020&amp;author=Zhou%2CT">
                    Google Scholar</a> 
                </p></li><li data-counter="24."><p id="ref-CR24">Lorda-Diez, C. I., Torre-Pérez, N., García-Porrero, J. A., Hurle, J. M. &amp; Montero, J. A. Expression of Id2 in the developing limb is associated with zones of active BMP signaling and marks the regions of growth and differentiation of the developing digits. <i>Int. J. Dev. Biol.</i> <b>53</b>, 1495–1502 (2009).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1387/ijdb.072415cl" data-track-action="article reference" href="https://doi.org/10.1387%2Fijdb.072415cl" aria-label="Article reference 24" data-doi="10.1387/ijdb.072415cl">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXjtVyksrk%3D" aria-label="CAS reference 24">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19247943" aria-label="PubMed reference 24">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 24" href="http://scholar.google.com/scholar_lookup?&amp;title=Expression%20of%20Id2%20in%20the%20developing%20limb%20is%20associated%20with%20zones%20of%20active%20BMP%20signaling%20and%20marks%20the%20regions%20of%20growth%20and%20differentiation%20of%20the%20developing%20digits&amp;journal=Int.%20J.%20Dev.%20Biol.&amp;doi=10.1387%2Fijdb.072415cl&amp;volume=53&amp;pages=1495-1502&amp;publication_year=2009&amp;author=Lorda-Diez%2CCI&amp;author=Torre-P%C3%A9rez%2CN&amp;author=Garc%C3%ADa-Porrero%2CJA&amp;author=Hurle%2CJM&amp;author=Montero%2CJA">
                    Google Scholar</a> 
                </p></li><li data-counter="25."><p id="ref-CR25">McGlinn, E. et al. Expression of the NET family member Zfp503 is regulated by hedgehog and BMP signaling in the limb. <i>Dev. Dyn.</i> <b>237</b>, 1172–1182 (2008).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1002/dvdy.21508" data-track-action="article reference" href="https://doi.org/10.1002%2Fdvdy.21508" aria-label="Article reference 25" data-doi="10.1002/dvdy.21508">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXltFarsL8%3D" aria-label="CAS reference 25">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18351672" aria-label="PubMed reference 25">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 25" href="http://scholar.google.com/scholar_lookup?&amp;title=Expression%20of%20the%20NET%20family%20member%20Zfp503%20is%20regulated%20by%20hedgehog%20and%20BMP%20signaling%20in%20the%20limb&amp;journal=Dev.%20Dyn.&amp;doi=10.1002%2Fdvdy.21508&amp;volume=237&amp;pages=1172-1182&amp;publication_year=2008&amp;author=McGlinn%2CE">
                    Google Scholar</a> 
                </p></li><li data-counter="26."><p id="ref-CR26">Ma, S., Charron, J. &amp; Erikson, R. L. Role of Plk2 (Snk) in mouse development and cell proliferation. <i>Mol. Cell. Biol.</i> <b>23</b>, 6936–6943 (2003).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1128/MCB.23.19.6936-6943.2003" data-track-action="article reference" href="https://doi.org/10.1128%2FMCB.23.19.6936-6943.2003" aria-label="Article reference 26" data-doi="10.1128/MCB.23.19.6936-6943.2003">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3sXnvVyis7Y%3D" aria-label="CAS reference 26">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=12972611" aria-label="PubMed reference 26">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC193943" aria-label="PubMed Central reference 26">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 26" href="http://scholar.google.com/scholar_lookup?&amp;title=Role%20of%20Plk2%20%28Snk%29%20in%20mouse%20development%20and%20cell%20proliferation&amp;journal=Mol.%20Cell.%20Biol.&amp;doi=10.1128%2FMCB.23.19.6936-6943.2003&amp;volume=23&amp;pages=6936-6943&amp;publication_year=2003&amp;author=Ma%2CS&amp;author=Charron%2CJ&amp;author=Erikson%2CRL">
                    Google Scholar</a> 
                </p></li><li data-counter="27."><p id="ref-CR27">Falardeau, F., Camurri, M. V. &amp; Campeau, P. M. Genomic approaches to diagnose rare bone disorders. <i>Bone</i> <b>102</b>, 5–14 (2017).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.bone.2016.07.020" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.bone.2016.07.020" aria-label="Article reference 27" data-doi="10.1016/j.bone.2016.07.020">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28Xht12isrvJ" aria-label="CAS reference 27">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27474525" aria-label="PubMed reference 27">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 27" href="http://scholar.google.com/scholar_lookup?&amp;title=Genomic%20approaches%20to%20diagnose%20rare%20bone%20disorders&amp;journal=Bone&amp;doi=10.1016%2Fj.bone.2016.07.020&amp;volume=102&amp;pages=5-14&amp;publication_year=2017&amp;author=Falardeau%2CF&amp;author=Camurri%2CMV&amp;author=Campeau%2CPM">
                    Google Scholar</a> 
                </p></li><li data-counter="28."><p id="ref-CR28">Cooks, R. G., Hertz, M., Katznelson, M. B. &amp; Goodman, R. M. A new nail dysplasia syndrome with onychonychia and absence and/or hypoplasia of distal phalanges. <i>Clin. Genet.</i> <b>27</b>, 85–91 (1985).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1111/j.1399-0004.1985.tb00189.x" data-track-action="article reference" href="https://doi.org/10.1111%2Fj.1399-0004.1985.tb00189.x" aria-label="Article reference 28" data-doi="10.1111/j.1399-0004.1985.tb00189.x">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:STN:280:DyaL2M7lt1Gkug%3D%3D" aria-label="CAS reference 28">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=3978841" aria-label="PubMed reference 28">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 28" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20new%20nail%20dysplasia%20syndrome%20with%20onychonychia%20and%20absence%20and%2For%20hypoplasia%20of%20distal%20phalanges&amp;journal=Clin.%20Genet.&amp;doi=10.1111%2Fj.1399-0004.1985.tb00189.x&amp;volume=27&amp;pages=85-91&amp;publication_year=1985&amp;author=Cooks%2CRG&amp;author=Hertz%2CM&amp;author=Katznelson%2CMB&amp;author=Goodman%2CRM">
                    Google Scholar</a> 
                </p></li><li data-counter="29."><p id="ref-CR29">Temtamy, S. A. &amp; Aglan, M. S. Brachydactyly. <i>Orphanet J. Rare Dis.</i> <b>3</b>, 15 (2008).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1186/1750-1172-3-15" data-track-action="article reference" href="https://doi.org/10.1186%2F1750-1172-3-15" aria-label="Article reference 29" data-doi="10.1186/1750-1172-3-15">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18554391" aria-label="PubMed reference 29">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2441618" aria-label="PubMed Central reference 29">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 29" href="http://scholar.google.com/scholar_lookup?&amp;title=Brachydactyly&amp;journal=Orphanet%20J.%20Rare%20Dis.&amp;doi=10.1186%2F1750-1172-3-15&amp;volume=3&amp;publication_year=2008&amp;author=Temtamy%2CSA&amp;author=Aglan%2CMS">
                    Google Scholar</a> 
                </p></li><li data-counter="30."><p id="ref-CR30">Bahubali, D. &amp; Gane, P. N. Split-hand/feet malformation: a rare syndrome. <i>J. Family Med. Prim. Care</i> <b>5</b>, 168–169 (2016).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.4103/2249-4863.184656" data-track-action="article reference" href="https://doi.org/10.4103%2F2249-4863.184656" aria-label="Article reference 30" data-doi="10.4103/2249-4863.184656">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 30" href="http://scholar.google.com/scholar_lookup?&amp;title=Split-hand%2Ffeet%20malformation%3A%20a%20rare%20syndrome&amp;journal=J.%20Family%20Med.%20Prim.%20Care&amp;doi=10.4103%2F2249-4863.184656&amp;volume=5&amp;pages=168-169&amp;publication_year=2016&amp;author=Bahubali%2CD&amp;author=Gane%2CPN">
                    Google Scholar</a> 
                </p></li><li data-counter="31."><p id="ref-CR31">Marcelis, C. L. M. &amp; de Brouwer, A. P. M. Feingold Syndrome 1. In <i>GeneReviews</i>® [Internet]. (eds Adam, M. P. et al.) <a href="https://www.ncbi.nlm.nih.gov/books/NBK7050" data-track="click" data-track-action="external reference" data-track-label="https://www.ncbi.nlm.nih.gov/books/NBK7050">www.ncbi.nlm.nih.gov/books/NBK7050</a> (Univ. Washington, 2019).</p></li><li data-counter="32."><p id="ref-CR32">Bakrania, P. et al. Mutations in BMP4 cause eye, brain, and digit developmental anomalies: overlap between the BMP4 and Hedgehog signaling pathways. <i>Am. J. Hum. Genet.</i> <b>82</b>, 304–319 (2008).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.ajhg.2007.09.023" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.ajhg.2007.09.023" aria-label="Article reference 32" data-doi="10.1016/j.ajhg.2007.09.023">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXit1Sju74%3D" aria-label="CAS reference 32">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18252212" aria-label="PubMed reference 32">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2427285" aria-label="PubMed Central reference 32">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 32" href="http://scholar.google.com/scholar_lookup?&amp;title=Mutations%20in%20BMP4%20cause%20eye%2C%20brain%2C%20and%20digit%20developmental%20anomalies%3A%20overlap%20between%20the%20BMP4%20and%20Hedgehog%20signaling%20pathways&amp;journal=Am.%20J.%20Hum.%20Genet.&amp;doi=10.1016%2Fj.ajhg.2007.09.023&amp;volume=82&amp;pages=304-319&amp;publication_year=2008&amp;author=Bakrania%2CP">
                    Google Scholar</a> 
                </p></li><li data-counter="33."><p id="ref-CR33">El Ghouzzi, V. et al. Saethre–Chotzen mutations cause TWIST protein degradation or impaired nuclear location. <i>Hum. Mol. Genet.</i> <b>9</b>, 813–819 (2000).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1093/hmg/9.5.813" data-track-action="article reference" href="https://doi.org/10.1093%2Fhmg%2F9.5.813" aria-label="Article reference 33" data-doi="10.1093/hmg/9.5.813">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3cXitlaisL8%3D" aria-label="CAS reference 33">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=10749989" aria-label="PubMed reference 33">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 33" href="http://scholar.google.com/scholar_lookup?&amp;title=Saethre%E2%80%93Chotzen%20mutations%20cause%20TWIST%20protein%20degradation%20or%20impaired%20nuclear%20location&amp;journal=Hum.%20Mol.%20Genet.&amp;doi=10.1093%2Fhmg%2F9.5.813&amp;volume=9&amp;pages=813-819&amp;publication_year=2000&amp;author=El%20Ghouzzi%2CV">
                    Google Scholar</a> 
                </p></li><li data-counter="34."><p id="ref-CR34">Moore, A. W. et al. YAC transgenic analysis reveals Wilms’ tumour 1 gene activity in the proliferating coelomic epithelium, developing diaphragm and limb. <i>Mech. Dev.</i> <b>79</b>, 169–184 (1998).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/S0925-4773(98)00188-9" data-track-action="article reference" href="https://doi.org/10.1016%2FS0925-4773%2898%2900188-9" aria-label="Article reference 34" data-doi="10.1016/S0925-4773(98)00188-9">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=1998asto.book.....M" aria-label="ADS reference 34">ADS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK1cXnvVWitr4%3D" aria-label="CAS reference 34">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=10349631" aria-label="PubMed reference 34">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 34" href="http://scholar.google.com/scholar_lookup?&amp;title=YAC%20transgenic%20analysis%20reveals%20Wilms%E2%80%99%20tumour%201%20gene%20activity%20in%20the%20proliferating%20coelomic%20epithelium%2C%20developing%20diaphragm%20and%20limb&amp;journal=Mech.%20Dev.&amp;doi=10.1016%2FS0925-4773%2898%2900188-9&amp;volume=79&amp;pages=169-184&amp;publication_year=1998&amp;author=Moore%2CAW">
                    Google Scholar</a> 
                </p></li><li data-counter="35."><p id="ref-CR35">Dunwoodie, S. L., Rodriguez, T. A. &amp; Beddington, R. S. Msg1 and Mrg1, founding members of a gene family, show distinct patterns of gene expression during mouse embryogenesis. <i>Mech. Dev.</i> <b>72</b>, 27–40 (1998).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/S0925-4773(98)00011-2" data-track-action="article reference" href="https://doi.org/10.1016%2FS0925-4773%2898%2900011-2" aria-label="Article reference 35" data-doi="10.1016/S0925-4773(98)00011-2">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK1cXhvVShtb4%3D" aria-label="CAS reference 35">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=9533950" aria-label="PubMed reference 35">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 35" href="http://scholar.google.com/scholar_lookup?&amp;title=Msg1%20and%20Mrg1%2C%20founding%20members%20of%20a%20gene%20family%2C%20show%20distinct%20patterns%20of%20gene%20expression%20during%20mouse%20embryogenesis&amp;journal=Mech.%20Dev.&amp;doi=10.1016%2FS0925-4773%2898%2900011-2&amp;volume=72&amp;pages=27-40&amp;publication_year=1998&amp;author=Dunwoodie%2CSL&amp;author=Rodriguez%2CTA&amp;author=Beddington%2CRS">
                    Google Scholar</a> 
                </p></li><li data-counter="36."><p id="ref-CR36">Hudson, D. T. et al. Gene expression analysis of the <i>Xenopus laevis</i> early limb bud proximodistal axis. <i>Dev. Dyn.</i> <b>251</b>, 1880–1896 (2022).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1002/dvdy.517" data-track-action="article reference" href="https://doi.org/10.1002%2Fdvdy.517" aria-label="Article reference 36" data-doi="10.1002/dvdy.517">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XitFCju7%2FK" aria-label="CAS reference 36">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=35809036" aria-label="PubMed reference 36">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9796579" aria-label="PubMed Central reference 36">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 36" href="http://scholar.google.com/scholar_lookup?&amp;title=Gene%20expression%20analysis%20of%20the%20Xenopus%20laevis%20early%20limb%20bud%20proximodistal%20axis&amp;journal=Dev.%20Dyn.&amp;doi=10.1002%2Fdvdy.517&amp;volume=251&amp;pages=1880-1896&amp;publication_year=2022&amp;author=Hudson%2CDT">
                    Google Scholar</a> 
                </p></li><li data-counter="37."><p id="ref-CR37">Sheng, G. &amp; Stern, C. D. Gata2 and Gata3: novel markers for early embryonic polarity and for non-neural ectoderm in the chick embryo. <i>Mech. Dev.</i> <b>87</b>, 213–216 (1999).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/S0925-4773(99)00150-1" data-track-action="article reference" href="https://doi.org/10.1016%2FS0925-4773%2899%2900150-1" aria-label="Article reference 37" data-doi="10.1016/S0925-4773(99)00150-1">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK1MXmtVyjtLY%3D" aria-label="CAS reference 37">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=10495290" aria-label="PubMed reference 37">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 37" href="http://scholar.google.com/scholar_lookup?&amp;title=Gata2%20and%20Gata3%3A%20novel%20markers%20for%20early%20embryonic%20polarity%20and%20for%20non-neural%20ectoderm%20in%20the%20chick%20embryo&amp;journal=Mech.%20Dev.&amp;doi=10.1016%2FS0925-4773%2899%2900150-1&amp;volume=87&amp;pages=213-216&amp;publication_year=1999&amp;author=Sheng%2CG&amp;author=Stern%2CCD">
                    Google Scholar</a> 
                </p></li><li data-counter="38."><p id="ref-CR38">Tzchori, I. et al. LIM homeobox transcription factors integrate signaling events that control three-dimensional limb patterning and growth. <i>Development</i> <b>136</b>, 1375–1385 (2009).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1242/dev.026476" data-track-action="article reference" href="https://doi.org/10.1242%2Fdev.026476" aria-label="Article reference 38" data-doi="10.1242/dev.026476">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXmtVakurs%3D" aria-label="CAS reference 38">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19304889" aria-label="PubMed reference 38">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2687467" aria-label="PubMed Central reference 38">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 38" href="http://scholar.google.com/scholar_lookup?&amp;title=LIM%20homeobox%20transcription%20factors%20integrate%20signaling%20events%20that%20control%20three-dimensional%20limb%20patterning%20and%20growth&amp;journal=Development&amp;doi=10.1242%2Fdev.026476&amp;volume=136&amp;pages=1375-1385&amp;publication_year=2009&amp;author=Tzchori%2CI">
                    Google Scholar</a> 
                </p></li><li data-counter="39."><p id="ref-CR39">Bensoussan-Trigano, V., Lallemand, Y., Saint, C. C. &amp; Robert, B. Msx1 and Msx2 in limb mesenchyme modulate digit number and identity. <i>Dev. Dyn.</i> <b>240</b>, 1190–1202 (2011).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1002/dvdy.22619" data-track-action="article reference" href="https://doi.org/10.1002%2Fdvdy.22619" aria-label="Article reference 39" data-doi="10.1002/dvdy.22619">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXkvFGgsLo%3D" aria-label="CAS reference 39">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21465616" aria-label="PubMed reference 39">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 39" href="http://scholar.google.com/scholar_lookup?&amp;title=Msx1%20and%20Msx2%20in%20limb%20mesenchyme%20modulate%20digit%20number%20and%20identity&amp;journal=Dev.%20Dyn.&amp;doi=10.1002%2Fdvdy.22619&amp;volume=240&amp;pages=1190-1202&amp;publication_year=2011&amp;author=Bensoussan-Trigano%2CV&amp;author=Lallemand%2CY&amp;author=Saint%2CCC&amp;author=Robert%2CB">
                    Google Scholar</a> 
                </p></li><li data-counter="40."><p id="ref-CR40">Arostegui, M., Scott, R. W., Böse, K. &amp; Underhill, T. M. Cellular taxonomy of Hic1<sup>+</sup> mesenchymal progenitor derivatives in the limb: from embryo to adult. <i>Nat. Commun.</i> <b>13</b>, 4989 (2022).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/s41467-022-32695-1" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41467-022-32695-1" aria-label="Article reference 40" data-doi="10.1038/s41467-022-32695-1">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2022NatCo..13.4989A" aria-label="ADS reference 40">ADS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XitlSgur%2FO" aria-label="CAS reference 40">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=36008423" aria-label="PubMed reference 40">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9411605" aria-label="PubMed Central reference 40">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 40" href="http://scholar.google.com/scholar_lookup?&amp;title=Cellular%20taxonomy%20of%20Hic1%2B%20mesenchymal%20progenitor%20derivatives%20in%20the%20limb%3A%20from%20embryo%20to%20adult&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-022-32695-1&amp;volume=13&amp;publication_year=2022&amp;author=Arostegui%2CM&amp;author=Scott%2CRW&amp;author=B%C3%B6se%2CK&amp;author=Underhill%2CTM">
                    Google Scholar</a> 
                </p></li><li data-counter="41."><p id="ref-CR41">Nishimura, R., Hata, K., Matsubara, T., Wakabayashi, M. &amp; Yoneda, T. Regulation of bone and cartilage development by network between BMP signalling and transcription factors. <i>J. Biochem.</i> <b>151</b>, 247–254 (2012).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1093/jb/mvs004" data-track-action="article reference" href="https://doi.org/10.1093%2Fjb%2Fmvs004" aria-label="Article reference 41" data-doi="10.1093/jb/mvs004">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XjtlKktr0%3D" aria-label="CAS reference 41">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22253449" aria-label="PubMed reference 41">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 41" href="http://scholar.google.com/scholar_lookup?&amp;title=Regulation%20of%20bone%20and%20cartilage%20development%20by%20network%20between%20BMP%20signalling%20and%20transcription%20factors&amp;journal=J.%20Biochem.&amp;doi=10.1093%2Fjb%2Fmvs004&amp;volume=151&amp;pages=247-254&amp;publication_year=2012&amp;author=Nishimura%2CR&amp;author=Hata%2CK&amp;author=Matsubara%2CT&amp;author=Wakabayashi%2CM&amp;author=Yoneda%2CT">
                    Google Scholar</a> 
                </p></li><li data-counter="42."><p id="ref-CR42">Kawato, Y. et al. Nkx3.2 promotes primary chondrogenic differentiation by upregulating Col2a1 transcription. <i>PLoS ONE</i> <b>7</b>, e34703 (2012).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1371/journal.pone.0034703" data-track-action="article reference" href="https://doi.org/10.1371%2Fjournal.pone.0034703" aria-label="Article reference 42" data-doi="10.1371/journal.pone.0034703">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2012PLoSO...734703K" aria-label="ADS reference 42">ADS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XmtFGitb4%3D" aria-label="CAS reference 42">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22511961" aria-label="PubMed reference 42">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3325257" aria-label="PubMed Central reference 42">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 42" href="http://scholar.google.com/scholar_lookup?&amp;title=Nkx3.2%20promotes%20primary%20chondrogenic%20differentiation%20by%20upregulating%20Col2a1%20transcription&amp;journal=PLoS%20ONE&amp;doi=10.1371%2Fjournal.pone.0034703&amp;volume=7&amp;publication_year=2012&amp;author=Kawato%2CY">
                    Google Scholar</a> 
                </p></li><li data-counter="43."><p id="ref-CR43">Hong, E., Di Cesare, P. E. &amp; Haudenschild, D. R. Role of c-Maf in chondrocyte differentiation: a review. <i>Cartilage</i> <b>2</b>, 27–35 (2011).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1177/1947603510377464" data-track-action="article reference" href="https://doi.org/10.1177%2F1947603510377464" aria-label="Article reference 43" data-doi="10.1177/1947603510377464">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXkt1Cmsrg%3D" aria-label="CAS reference 43">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26069566" aria-label="PubMed reference 43">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4300789" aria-label="PubMed Central reference 43">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 43" href="http://scholar.google.com/scholar_lookup?&amp;title=Role%20of%20c-Maf%20in%20chondrocyte%20differentiation%3A%20a%20review&amp;journal=Cartilage&amp;doi=10.1177%2F1947603510377464&amp;volume=2&amp;pages=27-35&amp;publication_year=2011&amp;author=Hong%2CE&amp;author=Cesare%2CPE&amp;author=Haudenschild%2CDR">
                    Google Scholar</a> 
                </p></li><li data-counter="44."><p id="ref-CR44">Zhang, J. et al. Roles of SATB2 in osteogenic differentiation and bone regeneration. <i>Tissue Eng. Part A</i> <b>17</b>, 1767–1776 (2011).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1089/ten.tea.2010.0503" data-track-action="article reference" href="https://doi.org/10.1089%2Ften.tea.2010.0503" aria-label="Article reference 44" data-doi="10.1089/ten.tea.2010.0503">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXnvFOrsbg%3D" aria-label="CAS reference 44">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21385070" aria-label="PubMed reference 44">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3118613" aria-label="PubMed Central reference 44">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 44" href="http://scholar.google.com/scholar_lookup?&amp;title=Roles%20of%20SATB2%20in%20osteogenic%20differentiation%20and%20bone%20regeneration&amp;journal=Tissue%20Eng.%20Part%20A&amp;doi=10.1089%2Ften.tea.2010.0503&amp;volume=17&amp;pages=1767-1776&amp;publication_year=2011&amp;author=Zhang%2CJ">
                    Google Scholar</a> 
                </p></li><li data-counter="45."><p id="ref-CR45">Iwamoto, M. et al. Transcription factor ERG and joint and articular cartilage formation during mouse limb and spine skeletogenesis. <i>Dev. Biol.</i> <b>305</b>, 40–51 (2007).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.ydbio.2007.01.037" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.ydbio.2007.01.037" aria-label="Article reference 45" data-doi="10.1016/j.ydbio.2007.01.037">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2sXksV2ksrs%3D" aria-label="CAS reference 45">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17336282" aria-label="PubMed reference 45">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2104487" aria-label="PubMed Central reference 45">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 45" href="http://scholar.google.com/scholar_lookup?&amp;title=Transcription%20factor%20ERG%20and%20joint%20and%20articular%20cartilage%20formation%20during%20mouse%20limb%20and%20spine%20skeletogenesis&amp;journal=Dev.%20Biol.&amp;doi=10.1016%2Fj.ydbio.2007.01.037&amp;volume=305&amp;pages=40-51&amp;publication_year=2007&amp;author=Iwamoto%2CM">
                    Google Scholar</a> 
                </p></li><li data-counter="46."><p id="ref-CR46">Zhao, H. et al. Foxp1/2/4 regulate endochondral ossification as a suppresser complex. <i>Dev. Biol.</i> <b>398</b>, 242–254 (2015).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.ydbio.2014.12.007" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.ydbio.2014.12.007" aria-label="Article reference 46" data-doi="10.1016/j.ydbio.2014.12.007">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXitFygt7bO" aria-label="CAS reference 46">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25527076" aria-label="PubMed reference 46">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 46" href="http://scholar.google.com/scholar_lookup?&amp;title=Foxp1%2F2%2F4%20regulate%20endochondral%20ossification%20as%20a%20suppresser%20complex&amp;journal=Dev.%20Biol.&amp;doi=10.1016%2Fj.ydbio.2014.12.007&amp;volume=398&amp;pages=242-254&amp;publication_year=2015&amp;author=Zhao%2CH">
                    Google Scholar</a> 
                </p></li><li data-counter="47."><p id="ref-CR47">Blitz, E., Sharir, A., Akiyama, H. &amp; Zelzer, E. Tendon-bone attachment unit is formed modularly by a distinct pool of Scx- and Sox9-positive progenitors. <i>Development</i> <b>140</b>, 2680–2690 (2013).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1242/dev.093906" data-track-action="article reference" href="https://doi.org/10.1242%2Fdev.093906" aria-label="Article reference 47" data-doi="10.1242/dev.093906">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXht1OmsbbM" aria-label="CAS reference 47">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23720048" aria-label="PubMed reference 47">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 47" href="http://scholar.google.com/scholar_lookup?&amp;title=Tendon-bone%20attachment%20unit%20is%20formed%20modularly%20by%20a%20distinct%20pool%20of%20Scx-%20and%20Sox9-positive%20progenitors&amp;journal=Development&amp;doi=10.1242%2Fdev.093906&amp;volume=140&amp;pages=2680-2690&amp;publication_year=2013&amp;author=Blitz%2CE&amp;author=Sharir%2CA&amp;author=Akiyama%2CH&amp;author=Zelzer%2CE">
                    Google Scholar</a> 
                </p></li><li data-counter="48."><p id="ref-CR48">Yang, F. &amp; Richardson, D. W. Comparative analysis of tenogenic gene expression in tenocyte-derived induced pluripotent stem cells and bone marrow-derived mesenchymal stem cells in response to biochemical and biomechanical stimuli. <i>Stem Cells Int.</i> <b>2021</b>, 8835576 (2021).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1155/2021/8835576" data-track-action="article reference" href="https://doi.org/10.1155%2F2021%2F8835576" aria-label="Article reference 48" data-doi="10.1155/2021/8835576">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33510795" aria-label="PubMed reference 48">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7825360" aria-label="PubMed Central reference 48">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 48" href="http://scholar.google.com/scholar_lookup?&amp;title=Comparative%20analysis%20of%20tenogenic%20gene%20expression%20in%20tenocyte-derived%20induced%20pluripotent%20stem%20cells%20and%20bone%20marrow-derived%20mesenchymal%20stem%20cells%20in%20response%20to%20biochemical%20and%20biomechanical%20stimuli&amp;journal=Stem%20Cells%20Int.&amp;doi=10.1155%2F2021%2F8835576&amp;volume=2021&amp;publication_year=2021&amp;author=Yang%2CF&amp;author=Richardson%2CDW">
                    Google Scholar</a> 
                </p></li><li data-counter="49."><p id="ref-CR49">Rossi, G. et al. Nfix regulates temporal progression of muscle regeneration through modulation of myostatin expression. <i>Cell Rep.</i> <b>14</b>, 2238–2249 (2016).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.celrep.2016.02.014" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.celrep.2016.02.014" aria-label="Article reference 49" data-doi="10.1016/j.celrep.2016.02.014">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28Xjt1yrsLc%3D" aria-label="CAS reference 49">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26923583" aria-label="PubMed reference 49">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4793149" aria-label="PubMed Central reference 49">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 49" href="http://scholar.google.com/scholar_lookup?&amp;title=Nfix%20regulates%20temporal%20progression%20of%20muscle%20regeneration%20through%20modulation%20of%20myostatin%20expression&amp;journal=Cell%20Rep.&amp;doi=10.1016%2Fj.celrep.2016.02.014&amp;volume=14&amp;pages=2238-2249&amp;publication_year=2016&amp;author=Rossi%2CG">
                    Google Scholar</a> 
                </p></li><li data-counter="50."><p id="ref-CR50">Hoi, C. S. L. et al. Runx1 directly promotes proliferation of hair follicle stem cells and epithelial tumor formation in mouse skin. <i>Mol. Cell. Biol.</i> <b>30</b>, 2518–2536 (2010).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1128/MCB.01308-09" data-track-action="article reference" href="https://doi.org/10.1128%2FMCB.01308-09" aria-label="Article reference 50" data-doi="10.1128/MCB.01308-09">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXmtFymsbs%3D" aria-label="CAS reference 50">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20308320" aria-label="PubMed reference 50">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2863705" aria-label="PubMed Central reference 50">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 50" href="http://scholar.google.com/scholar_lookup?&amp;title=Runx1%20directly%20promotes%20proliferation%20of%20hair%20follicle%20stem%20cells%20and%20epithelial%20tumor%20formation%20in%20mouse%20skin&amp;journal=Mol.%20Cell.%20Biol.&amp;doi=10.1128%2FMCB.01308-09&amp;volume=30&amp;pages=2518-2536&amp;publication_year=2010&amp;author=Hoi%2CCSL">
                    Google Scholar</a> 
                </p></li><li data-counter="51."><p id="ref-CR51">Li, L. et al. TFAP2C- and p63-dependent networks sequentially rearrange chromatin landscapes to drive human epidermal lineage commitment. <i>Cell Stem Cell</i> <b>24</b>, 271–284.e8 (2019).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.stem.2018.12.012" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.stem.2018.12.012" aria-label="Article reference 51" data-doi="10.1016/j.stem.2018.12.012">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhvFSlt7g%3D" aria-label="CAS reference 51">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30686763" aria-label="PubMed reference 51">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7135956" aria-label="PubMed Central reference 51">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 51" href="http://scholar.google.com/scholar_lookup?&amp;title=TFAP2C-%20and%20p63-dependent%20networks%20sequentially%20rearrange%20chromatin%20landscapes%20to%20drive%20human%20epidermal%20lineage%20commitment&amp;journal=Cell%20Stem%20Cell&amp;doi=10.1016%2Fj.stem.2018.12.012&amp;volume=24&amp;pages=271-284.e8&amp;publication_year=2019&amp;author=Li%2CL">
                    Google Scholar</a> 
                </p></li><li data-counter="52."><p id="ref-CR52">Lepore, J. J., Cappola, T. P., Mericko, P. A., Morrisey, E. E. &amp; Parmacek, M. S. GATA-6 regulates genes promoting synthetic functions in vascular smooth muscle cells. <i>Arterioscler. Thromb. Vasc. Biol.</i> <b>25</b>, 309–314 (2005).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1161/01.ATV.0000152725.76020.3c" data-track-action="article reference" href="https://doi.org/10.1161%2F01.ATV.0000152725.76020.3c" aria-label="Article reference 52" data-doi="10.1161/01.ATV.0000152725.76020.3c">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2MXmtFCktg%3D%3D" aria-label="CAS reference 52">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15591218" aria-label="PubMed reference 52">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 52" href="http://scholar.google.com/scholar_lookup?&amp;title=GATA-6%20regulates%20genes%20promoting%20synthetic%20functions%20in%20vascular%20smooth%20muscle%20cells&amp;journal=Arterioscler.%20Thromb.%20Vasc.%20Biol.&amp;doi=10.1161%2F01.ATV.0000152725.76020.3c&amp;volume=25&amp;pages=309-314&amp;publication_year=2005&amp;author=Lepore%2CJJ&amp;author=Cappola%2CTP&amp;author=Mericko%2CPA&amp;author=Morrisey%2CEE&amp;author=Parmacek%2CMS">
                    Google Scholar</a> 
                </p></li><li data-counter="53."><p id="ref-CR53">Xie, Z. et al. Smooth-muscle BMAL1 participates in blood pressure circadian rhythm regulation. <i>J. Clin. Invest.</i> <b>125</b>, 324–336 (2015).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1172/JCI76881" data-track-action="article reference" href="https://doi.org/10.1172%2FJCI76881" aria-label="Article reference 53" data-doi="10.1172/JCI76881">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25485682" aria-label="PubMed reference 53">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 53" href="http://scholar.google.com/scholar_lookup?&amp;title=Smooth-muscle%20BMAL1%20participates%20in%20blood%20pressure%20circadian%20rhythm%20regulation&amp;journal=J.%20Clin.%20Invest.&amp;doi=10.1172%2FJCI76881&amp;volume=125&amp;pages=324-336&amp;publication_year=2015&amp;author=Xie%2CZ">
                    Google Scholar</a> 
                </p></li><li data-counter="54."><p id="ref-CR54">Buckingham, M. &amp; Rigby, P. W. J. Gene regulatory networks and transcriptional mechanisms that control myogenesis. <i>Dev. Cell</i> <b>28</b>, 225–238 (2014).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.devcel.2013.12.020" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.devcel.2013.12.020" aria-label="Article reference 54" data-doi="10.1016/j.devcel.2013.12.020">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXitlenu7g%3D" aria-label="CAS reference 54">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24525185" aria-label="PubMed reference 54">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 54" href="http://scholar.google.com/scholar_lookup?&amp;title=Gene%20regulatory%20networks%20and%20transcriptional%20mechanisms%20that%20control%20myogenesis&amp;journal=Dev.%20Cell&amp;doi=10.1016%2Fj.devcel.2013.12.020&amp;volume=28&amp;pages=225-238&amp;publication_year=2014&amp;author=Buckingham%2CM&amp;author=Rigby%2CPWJ">
                    Google Scholar</a> 
                </p></li><li data-counter="55."><p id="ref-CR55">Hutcheson, D. A., Zhao, J., Merrell, A. &amp; Haldar, M. Embryonic and fetal limb myogenic cells are derived from developmentally distinct progenitors and have different requirements for β-catenin. <i>Genes</i> 10.1101/gad.1769009 (2009).</p></li><li data-counter="56."><p id="ref-CR56">Singh, A. J. et al. FACS-seq analysis of Pax3-derived cells identifies non-myogenic lineages in the embryonic forelimb. <i>Sci. Rep.</i> <b>8</b>, 7670 (2018).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/s41598-018-25998-1" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41598-018-25998-1" aria-label="Article reference 56" data-doi="10.1038/s41598-018-25998-1">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2018NatSR...8.7670S" aria-label="ADS reference 56">ADS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29769607" aria-label="PubMed reference 56">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5956100" aria-label="PubMed Central reference 56">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 56" href="http://scholar.google.com/scholar_lookup?&amp;title=FACS-seq%20analysis%20of%20Pax3-derived%20cells%20identifies%20non-myogenic%20lineages%20in%20the%20embryonic%20forelimb&amp;journal=Sci.%20Rep.&amp;doi=10.1038%2Fs41598-018-25998-1&amp;volume=8&amp;publication_year=2018&amp;author=Singh%2CAJ">
                    Google Scholar</a> 
                </p></li><li data-counter="57."><p id="ref-CR57">Benezra, R., Davis, R. L., Lockshon, D., Turner, D. L. &amp; Weintraub, H. The protein Id: a negative regulator of helix-loop-helix DNA binding proteins. <i>Cell</i> <b>61</b>, 49–59 (1990).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/0092-8674(90)90214-Y" data-track-action="article reference" href="https://doi.org/10.1016%2F0092-8674%2890%2990214-Y" aria-label="Article reference 57" data-doi="10.1016/0092-8674(90)90214-Y">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK3cXkslehsbg%3D" aria-label="CAS reference 57">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=2156629" aria-label="PubMed reference 57">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 57" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20protein%20Id%3A%20a%20negative%20regulator%20of%20helix-loop-helix%20DNA%20binding%20proteins&amp;journal=Cell&amp;doi=10.1016%2F0092-8674%2890%2990214-Y&amp;volume=61&amp;pages=49-59&amp;publication_year=1990&amp;author=Benezra%2CR&amp;author=Davis%2CRL&amp;author=Lockshon%2CD&amp;author=Turner%2CDL&amp;author=Weintraub%2CH">
                    Google Scholar</a> 
                </p></li><li data-counter="58."><p id="ref-CR58">Roschger, C. &amp; Cabrele, C. The Id-protein family in developmental and cancer-associated pathways. <i>Cell Commun. Signal.</i> <b>15</b>, 7 (2017).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1186/s12964-016-0161-y" data-track-action="article reference" href="https://doi.org/10.1186%2Fs12964-016-0161-y" aria-label="Article reference 58" data-doi="10.1186/s12964-016-0161-y">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28122577" aria-label="PubMed reference 58">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5267474" aria-label="PubMed Central reference 58">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 58" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20Id-protein%20family%20in%20developmental%20and%20cancer-associated%20pathways&amp;journal=Cell%20Commun.%20Signal.&amp;doi=10.1186%2Fs12964-016-0161-y&amp;volume=15&amp;publication_year=2017&amp;author=Roschger%2CC&amp;author=Cabrele%2CC">
                    Google Scholar</a> 
                </p></li><li data-counter="59."><p id="ref-CR59">Muriel, J. M. et al. Keratin 18 is an integral part of the intermediate filament network in murine skeletal muscle. <i>Am. J. Physiol. Cell Physiol.</i> <b>318</b>, C215–C224 (2020).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1152/ajpcell.00279.2019" data-track-action="article reference" href="https://doi.org/10.1152%2Fajpcell.00279.2019" aria-label="Article reference 59" data-doi="10.1152/ajpcell.00279.2019">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXjsFChtLk%3D" aria-label="CAS reference 59">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31721615" aria-label="PubMed reference 59">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 59" href="http://scholar.google.com/scholar_lookup?&amp;title=Keratin%2018%20is%20an%20integral%20part%20of%20the%20intermediate%20filament%20network%20in%20murine%20skeletal%20muscle&amp;journal=Am.%20J.%20Physiol.%20Cell%20Physiol.&amp;doi=10.1152%2Fajpcell.00279.2019&amp;volume=318&amp;pages=C215-C224&amp;publication_year=2020&amp;author=Muriel%2CJM">
                    Google Scholar</a> 
                </p></li><li data-counter="60."><p id="ref-CR60">Hernandez-Torres, F., Rodríguez-Outeiriño, L., Franco, D. &amp; Aranega, A. E. Pitx2 in embryonic and adult myogenesis. <i>Front. Cell Dev. Biol.</i> <b>5</b>, 46 (2017).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.3389/fcell.2017.00046" data-track-action="article reference" href="https://doi.org/10.3389%2Ffcell.2017.00046" aria-label="Article reference 60" data-doi="10.3389/fcell.2017.00046">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28507987" aria-label="PubMed reference 60">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5410577" aria-label="PubMed Central reference 60">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 60" href="http://scholar.google.com/scholar_lookup?&amp;title=Pitx2%20in%20embryonic%20and%20adult%20myogenesis&amp;journal=Front.%20Cell%20Dev.%20Biol.&amp;doi=10.3389%2Ffcell.2017.00046&amp;volume=5&amp;publication_year=2017&amp;author=Hernandez-Torres%2CF&amp;author=Rodr%C3%ADguez-Outeiri%C3%B1o%2CL&amp;author=Franco%2CD&amp;author=Aranega%2CAE">
                    Google Scholar</a> 
                </p></li><li data-counter="61."><p id="ref-CR61">Lee, H., Habas, R. &amp; Abate-Shen, C. MSX1 cooperates with histone H1b for inhibition of transcription and myogenesis. <i>Science</i> <b>304</b>, 1675–1678 (2004).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1126/science.1098096" data-track-action="article reference" href="https://doi.org/10.1126%2Fscience.1098096" aria-label="Article reference 61" data-doi="10.1126/science.1098096">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2004Sci...304.1675L" aria-label="ADS reference 61">ADS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2cXks1Cgsrw%3D" aria-label="CAS reference 61">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15192231" aria-label="PubMed reference 61">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 61" href="http://scholar.google.com/scholar_lookup?&amp;title=MSX1%20cooperates%20with%20histone%20H1b%20for%20inhibition%20of%20transcription%20and%20myogenesis&amp;journal=Science&amp;doi=10.1126%2Fscience.1098096&amp;volume=304&amp;pages=1675-1678&amp;publication_year=2004&amp;author=Lee%2CH&amp;author=Habas%2CR&amp;author=Abate-Shen%2CC">
                    Google Scholar</a> 
                </p></li><li data-counter="62."><p id="ref-CR62">Relaix, F. et al. Six homeoproteins directly activate Myod expression in the gene regulatory networks that control early myogenesis. <i>PLoS Genet.</i> <b>9</b>, e1003425 (2013).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1371/journal.pgen.1003425" data-track-action="article reference" href="https://doi.org/10.1371%2Fjournal.pgen.1003425" aria-label="Article reference 62" data-doi="10.1371/journal.pgen.1003425">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXnt1aqsro%3D" aria-label="CAS reference 62">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23637613" aria-label="PubMed reference 62">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3636133" aria-label="PubMed Central reference 62">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 62" href="http://scholar.google.com/scholar_lookup?&amp;title=Six%20homeoproteins%20directly%20activate%20Myod%20expression%20in%20the%20gene%20regulatory%20networks%20that%20control%20early%20myogenesis&amp;journal=PLoS%20Genet.&amp;doi=10.1371%2Fjournal.pgen.1003425&amp;volume=9&amp;publication_year=2013&amp;author=Relaix%2CF">
                    Google Scholar</a> 
                </p></li><li data-counter="63."><p id="ref-CR63">Meech, R. et al. Barx2 is expressed in satellite cells and is required for normal muscle growth and regeneration. <i>Stem Cells</i> <b>30</b>, 253–265 (2012).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1002/stem.777" data-track-action="article reference" href="https://doi.org/10.1002%2Fstem.777" aria-label="Article reference 63" data-doi="10.1002/stem.777">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38Xjs1Shsrs%3D" aria-label="CAS reference 63">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22076929" aria-label="PubMed reference 63">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 63" href="http://scholar.google.com/scholar_lookup?&amp;title=Barx2%20is%20expressed%20in%20satellite%20cells%20and%20is%20required%20for%20normal%20muscle%20growth%20and%20regeneration&amp;journal=Stem%20Cells&amp;doi=10.1002%2Fstem.777&amp;volume=30&amp;pages=253-265&amp;publication_year=2012&amp;author=Meech%2CR">
                    Google Scholar</a> 
                </p></li><li data-counter="64."><p id="ref-CR64">Lu, J., Webb, R., Richardson, J. A. &amp; Olson, E. N. MyoR: a muscle-restricted basic helix–loop–helix transcription factor that antagonizes the actions of MyoD. <i>Proc. Natl Acad. Sci. USA</i> <b>96</b>, 552–557 (1999).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1073/pnas.96.2.552" data-track-action="article reference" href="https://doi.org/10.1073%2Fpnas.96.2.552" aria-label="Article reference 64" data-doi="10.1073/pnas.96.2.552">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=1999PNAS...96..552L" aria-label="ADS reference 64">ADS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK1MXmtlCmug%3D%3D" aria-label="CAS reference 64">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=9892671" aria-label="PubMed reference 64">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC15174" aria-label="PubMed Central reference 64">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 64" href="http://scholar.google.com/scholar_lookup?&amp;title=MyoR%3A%20a%20muscle-restricted%20basic%20helix%E2%80%93loop%E2%80%93helix%20transcription%20factor%20that%20antagonizes%20the%20actions%20of%20MyoD&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;doi=10.1073%2Fpnas.96.2.552&amp;volume=96&amp;pages=552-557&amp;publication_year=1999&amp;author=Lu%2CJ&amp;author=Webb%2CR&amp;author=Richardson%2CJA&amp;author=Olson%2CEN">
                    Google Scholar</a> 
                </p></li><li data-counter="65."><p id="ref-CR65">MacQuarrie, K. L., Yao, Z., Fong, A. P. &amp; Tapscott, S. J. Genome-wide binding of the basic helix-loop-helix myogenic inhibitor musculin has substantial overlap with MyoD: implications for buffering activity. <i>Skelet. Muscle</i> <b>3</b>, 26 (2013).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1186/2044-5040-3-26" data-track-action="article reference" href="https://doi.org/10.1186%2F2044-5040-3-26" aria-label="Article reference 65" data-doi="10.1186/2044-5040-3-26">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24175993" aria-label="PubMed reference 65">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4177542" aria-label="PubMed Central reference 65">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 65" href="http://scholar.google.com/scholar_lookup?&amp;title=Genome-wide%20binding%20of%20the%20basic%20helix-loop-helix%20myogenic%20inhibitor%20musculin%20has%20substantial%20overlap%20with%20MyoD%3A%20implications%20for%20buffering%20activity&amp;journal=Skelet.%20Muscle&amp;doi=10.1186%2F2044-5040-3-26&amp;volume=3&amp;publication_year=2013&amp;author=MacQuarrie%2CKL&amp;author=Yao%2CZ&amp;author=Fong%2CAP&amp;author=Tapscott%2CSJ">
                    Google Scholar</a> 
                </p></li><li data-counter="66."><p id="ref-CR66">Buas, M. F., Kabak, S. &amp; Kadesch, T. Inhibition of myogenesis by Notch: evidence for multiple pathways. <i>J. Cell. Physiol.</i> <b>218</b>, 84–93 (2009).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1002/jcp.21571" data-track-action="article reference" href="https://doi.org/10.1002%2Fjcp.21571" aria-label="Article reference 66" data-doi="10.1002/jcp.21571">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXhsFahsbbK" aria-label="CAS reference 66">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18727102" aria-label="PubMed reference 66">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4281488" aria-label="PubMed Central reference 66">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 66" href="http://scholar.google.com/scholar_lookup?&amp;title=Inhibition%20of%20myogenesis%20by%20Notch%3A%20evidence%20for%20multiple%20pathways&amp;journal=J.%20Cell.%20Physiol.&amp;doi=10.1002%2Fjcp.21571&amp;volume=218&amp;pages=84-93&amp;publication_year=2009&amp;author=Buas%2CMF&amp;author=Kabak%2CS&amp;author=Kadesch%2CT">
                    Google Scholar</a> 
                </p></li><li data-counter="67."><p id="ref-CR67">Nunnally, A. P. &amp; Parr, B. A. Analysis of Fz10 expression in mouse embryos. <i>Dev. Genes Evol.</i> <b>214</b>, 144–148 (2004).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1007/s00427-004-0386-4" data-track-action="article reference" href="https://doi.org/10.1007%2Fs00427-004-0386-4" aria-label="Article reference 67" data-doi="10.1007/s00427-004-0386-4">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2cXisFSrtbo%3D" aria-label="CAS reference 67">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=14758554" aria-label="PubMed reference 67">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 67" href="http://scholar.google.com/scholar_lookup?&amp;title=Analysis%20of%20Fz10%20expression%20in%20mouse%20embryos&amp;journal=Dev.%20Genes%20Evol.&amp;doi=10.1007%2Fs00427-004-0386-4&amp;volume=214&amp;pages=144-148&amp;publication_year=2004&amp;author=Nunnally%2CAP&amp;author=Parr%2CBA">
                    Google Scholar</a> 
                </p></li><li data-counter="68."><p id="ref-CR68">Sarem, M., Otto, O., Tanaka, S. &amp; Shastri, V. P. Cell number in mesenchymal stem cell aggregates dictates cell stiffness and chondrogenesis. <i>Stem Cell Res. Ther.</i> <b>10</b>, 10 (2019).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1186/s13287-018-1103-y" data-track-action="article reference" href="https://doi.org/10.1186%2Fs13287-018-1103-y" aria-label="Article reference 68" data-doi="10.1186/s13287-018-1103-y">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhsFGkur3O" aria-label="CAS reference 68">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30630531" aria-label="PubMed reference 68">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6329065" aria-label="PubMed Central reference 68">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 68" href="http://scholar.google.com/scholar_lookup?&amp;title=Cell%20number%20in%20mesenchymal%20stem%20cell%20aggregates%20dictates%20cell%20stiffness%20and%20chondrogenesis&amp;journal=Stem%20Cell%20Res.%20Ther.&amp;doi=10.1186%2Fs13287-018-1103-y&amp;volume=10&amp;publication_year=2019&amp;author=Sarem%2CM&amp;author=Otto%2CO&amp;author=Tanaka%2CS&amp;author=Shastri%2CVP">
                    Google Scholar</a> 
                </p></li><li data-counter="69."><p id="ref-CR69">Crosnier, C. et al. JAGGED1 gene expression during human embryogenesis elucidates the wide phenotypic spectrum of Alagille syndrome. <i>Hepatology</i> <b>32</b>, 574–581 (2000).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1053/jhep.2000.16600" data-track-action="article reference" href="https://doi.org/10.1053%2Fjhep.2000.16600" aria-label="Article reference 69" data-doi="10.1053/jhep.2000.16600">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3cXmsF2murc%3D" aria-label="CAS reference 69">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=10960452" aria-label="PubMed reference 69">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 69" href="http://scholar.google.com/scholar_lookup?&amp;title=JAGGED1%20gene%20expression%20during%20human%20embryogenesis%20elucidates%20the%20wide%20phenotypic%20spectrum%20of%20Alagille%20syndrome&amp;journal=Hepatology&amp;doi=10.1053%2Fjhep.2000.16600&amp;volume=32&amp;pages=574-581&amp;publication_year=2000&amp;author=Crosnier%2CC">
                    Google Scholar</a> 
                </p></li><li data-counter="70."><p id="ref-CR70">Schroeter, E. H., Kisslinger, J. A. &amp; Kopan, R. Notch-1 signalling requires ligand-induced proteolytic release of intracellular domain. <i>Nature</i> <b>393</b>, 382–386 (1998).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/30756" data-track-action="article reference" href="https://doi.org/10.1038%2F30756" aria-label="Article reference 70" data-doi="10.1038/30756">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=1998Natur.393..382S" aria-label="ADS reference 70">ADS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK1cXjsFKgt78%3D" aria-label="CAS reference 70">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=9620803" aria-label="PubMed reference 70">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 70" href="http://scholar.google.com/scholar_lookup?&amp;title=Notch-1%20signalling%20requires%20ligand-induced%20proteolytic%20release%20of%20intracellular%20domain&amp;journal=Nature&amp;doi=10.1038%2F30756&amp;volume=393&amp;pages=382-386&amp;publication_year=1998&amp;author=Schroeter%2CEH&amp;author=Kisslinger%2CJA&amp;author=Kopan%2CR">
                    Google Scholar</a> 
                </p></li><li data-counter="71."><p id="ref-CR71">Mašek, J. &amp; Andersson, E. R. The developmental biology of genetic Notch disorders. <i>Development</i> <b>144</b>, 1743–1763 (2017).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1242/dev.148007" data-track-action="article reference" href="https://doi.org/10.1242%2Fdev.148007" aria-label="Article reference 71" data-doi="10.1242/dev.148007">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28512196" aria-label="PubMed reference 71">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 71" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20developmental%20biology%20of%20genetic%20Notch%20disorders&amp;journal=Development&amp;doi=10.1242%2Fdev.148007&amp;volume=144&amp;pages=1743-1763&amp;publication_year=2017&amp;author=Ma%C5%A1ek%2CJ&amp;author=Andersson%2CER">
                    Google Scholar</a> 
                </p></li><li data-counter="72."><p id="ref-CR72">Turnpenny, P. D. &amp; Ellard, S. Alagille syndrome: pathogenesis, diagnosis and management. <i>Eur. J. Hum. Genet.</i> <b>20</b>, 251–257 (2011).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/ejhg.2011.181" data-track-action="article reference" href="https://doi.org/10.1038%2Fejhg.2011.181" aria-label="Article reference 72" data-doi="10.1038/ejhg.2011.181">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21934706" aria-label="PubMed reference 72">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3283172" aria-label="PubMed Central reference 72">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 72" href="http://scholar.google.com/scholar_lookup?&amp;title=Alagille%20syndrome%3A%20pathogenesis%2C%20diagnosis%20and%20management&amp;journal=Eur.%20J.%20Hum.%20Genet.&amp;doi=10.1038%2Fejhg.2011.181&amp;volume=20&amp;pages=251-257&amp;publication_year=2011&amp;author=Turnpenny%2CPD&amp;author=Ellard%2CS">
                    Google Scholar</a> 
                </p></li><li data-counter="73."><p id="ref-CR73">Xu, X. et al. Fibroblast growth factor receptor 2 (FGFR2)-mediated reciprocal regulation loop between FGF8 and FGF10 is essential for limb induction. <i>Development</i> 10.1242/dev.125.4.753 (1998).</p></li><li data-counter="74."><p id="ref-CR74">Agha, E. E. et al. Characterization of a novel fibroblast growth factor 10 (Fgf10) knock-in mouse line to target mesenchymal progenitors during embryonic development. <i>Pneumologie</i> 10.1055/s-0032-1315504 (2012).</p></li><li data-counter="75."><p id="ref-CR75">Azoury, S. C., Reddy, S., Shukla, V. &amp; Deng, C.-X. Fibroblast growth factor receptor 2 (FGFR2) mutation related syndromic craniosynostosis. <i>Int. J. Biol. Sci.</i> <b>13</b>, 1479–1488 (2017).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.7150/ijbs.22373" data-track-action="article reference" href="https://doi.org/10.7150%2Fijbs.22373" aria-label="Article reference 75" data-doi="10.7150/ijbs.22373">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXitFegsrfK" aria-label="CAS reference 75">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29230096" aria-label="PubMed reference 75">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5723914" aria-label="PubMed Central reference 75">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 75" href="http://scholar.google.com/scholar_lookup?&amp;title=Fibroblast%20growth%20factor%20receptor%202%20%28FGFR2%29%20mutation%20related%20syndromic%20craniosynostosis&amp;journal=Int.%20J.%20Biol.%20Sci.&amp;doi=10.7150%2Fijbs.22373&amp;volume=13&amp;pages=1479-1488&amp;publication_year=2017&amp;author=Azoury%2CSC&amp;author=Reddy%2CS&amp;author=Shukla%2CV&amp;author=Deng%2CC-X">
                    Google Scholar</a> 
                </p></li><li data-counter="76."><p id="ref-CR76">He, P. et al. The changing mouse embryo transcriptome at whole tissue and single-cell resolution. <i>Nature</i> <b>583</b>, 760–767 (2020).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/s41586-020-2536-x" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41586-020-2536-x" aria-label="Article reference 76" data-doi="10.1038/s41586-020-2536-x">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2020Natur.583..760H" aria-label="ADS reference 76">ADS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhsVygtb7M" aria-label="CAS reference 76">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32728245" aria-label="PubMed reference 76">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7410830" aria-label="PubMed Central reference 76">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 76" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20changing%20mouse%20embryo%20transcriptome%20at%20whole%20tissue%20and%20single-cell%20resolution&amp;journal=Nature&amp;doi=10.1038%2Fs41586-020-2536-x&amp;volume=583&amp;pages=760-767&amp;publication_year=2020&amp;author=He%2CP">
                    Google Scholar</a> 
                </p></li><li data-counter="77."><p id="ref-CR77">Kelly, N. H., Huynh, N. P. T. &amp; Guilak, F. Single cell RNA-sequencing reveals cellular heterogeneity and trajectories of lineage specification during murine embryonic limb development. <i>Matrix Biol.</i> 10.1016/j.matbio.2019.12.004 (2020).</p></li><li data-counter="78."><p id="ref-CR78">Allou, L. et al. Non-coding deletions identify Maenli lncRNA as a limb-specific En1 regulator. <i>Nature</i> <b>592</b>, 93–98 (2021).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2021Natur.592...93A" aria-label="ADS reference 78">ADS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXjslKntbg%3D" aria-label="CAS reference 78">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33568816" aria-label="PubMed reference 78">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 78" href="http://scholar.google.com/scholar_lookup?&amp;title=Non-coding%20deletions%20identify%20Maenli%20lncRNA%20as%20a%20limb-specific%20En1%20regulator&amp;journal=Nature&amp;volume=592&amp;pages=93-98&amp;publication_year=2021&amp;author=Allou%2CL">
                    Google Scholar</a> 
                </p></li><li data-counter="79."><p id="ref-CR79">Gilson, H. et al. Follistatin induces muscle hypertrophy through satellite cell proliferation and inhibition of both myostatin and activin. <i>Am. J. Physiol. Endocrinol. Metab.</i> <b>297</b>, E157–E164 (2009).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1152/ajpendo.00193.2009" data-track-action="article reference" href="https://doi.org/10.1152%2Fajpendo.00193.2009" aria-label="Article reference 79" data-doi="10.1152/ajpendo.00193.2009">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXovVKjt7Y%3D" aria-label="CAS reference 79">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19435857" aria-label="PubMed reference 79">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 79" href="http://scholar.google.com/scholar_lookup?&amp;title=Follistatin%20induces%20muscle%20hypertrophy%20through%20satellite%20cell%20proliferation%20and%20inhibition%20of%20both%20myostatin%20and%20activin&amp;journal=Am.%20J.%20Physiol.%20Endocrinol.%20Metab.&amp;doi=10.1152%2Fajpendo.00193.2009&amp;volume=297&amp;pages=E157-E164&amp;publication_year=2009&amp;author=Gilson%2CH">
                    Google Scholar</a> 
                </p></li><li data-counter="80."><p id="ref-CR80">Gao, H. et al. UCHL1 regulates muscle fibers and mTORC1 activity in skeletal muscle. <i>Life Sci.</i> <b>233</b>, 116699 (2019).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.lfs.2019.116699" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.lfs.2019.116699" aria-label="Article reference 80" data-doi="10.1016/j.lfs.2019.116699">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhsFWmtLjE" aria-label="CAS reference 80">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31356902" aria-label="PubMed reference 80">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6718320" aria-label="PubMed Central reference 80">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 80" href="http://scholar.google.com/scholar_lookup?&amp;title=UCHL1%20regulates%20muscle%20fibers%20and%20mTORC1%20activity%20in%20skeletal%20muscle&amp;journal=Life%20Sci.&amp;doi=10.1016%2Fj.lfs.2019.116699&amp;volume=233&amp;publication_year=2019&amp;author=Gao%2CH">
                    Google Scholar</a> 
                </p></li><li data-counter="81."><p id="ref-CR81">Purushothaman, S., Elewa, A. &amp; Seifert, A. W. Fgf-signaling is compartmentalized within the mesenchyme and controls proliferation during salamander limb development. <i>eLife</i> 10.7554/eLife.48507 (2019).</p></li><li data-counter="82."><p id="ref-CR82">Popescu, D. M. et al. Decoding human fetal liver haematopoiesis. <i>Nature</i> <b>574</b>, 365–371 (2019).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/s41586-019-1652-y" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41586-019-1652-y" aria-label="Article reference 82" data-doi="10.1038/s41586-019-1652-y">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2019Natur.574..365P" aria-label="ADS reference 82">ADS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhvFCis7jN" aria-label="CAS reference 82">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31597962" aria-label="PubMed reference 82">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6861135" aria-label="PubMed Central reference 82">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 82" href="http://scholar.google.com/scholar_lookup?&amp;title=Decoding%20human%20fetal%20liver%20haematopoiesis&amp;journal=Nature&amp;doi=10.1038%2Fs41586-019-1652-y&amp;volume=574&amp;pages=365-371&amp;publication_year=2019&amp;author=Popescu%2CDM">
                    Google Scholar</a> 
                </p></li><li data-counter="83."><p id="ref-CR83">Wolock, S. L., Lopez, R. &amp; Klein, A. M. Scrublet: computational identification of cell doublets in single-cell transcriptomic data. <i>Cell Syst.</i> <b>8</b>, 281–291.e9 (2019).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.cels.2018.11.005" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.cels.2018.11.005" aria-label="Article reference 83" data-doi="10.1016/j.cels.2018.11.005">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXosVyhtbw%3D" aria-label="CAS reference 83">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30954476" aria-label="PubMed reference 83">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6625319" aria-label="PubMed Central reference 83">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 83" href="http://scholar.google.com/scholar_lookup?&amp;title=Scrublet%3A%20computational%20identification%20of%20cell%20doublets%20in%20single-cell%20transcriptomic%20data&amp;journal=Cell%20Syst.&amp;doi=10.1016%2Fj.cels.2018.11.005&amp;volume=8&amp;pages=281-291.e9&amp;publication_year=2019&amp;author=Wolock%2CSL&amp;author=Lopez%2CR&amp;author=Klein%2CAM">
                    Google Scholar</a> 
                </p></li><li data-counter="84."><p id="ref-CR84">He, P. et al. A human fetal lung cell atlas uncovers proximal-distal gradients of differentiation and key regulators of epithelial fates. <i>Cell</i> <b>185</b>, 4841–4860.e25 (2022).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.cell.2022.11.005" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.cell.2022.11.005" aria-label="Article reference 84" data-doi="10.1016/j.cell.2022.11.005">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XjtVOqsbzE" aria-label="CAS reference 84">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=36493756" aria-label="PubMed reference 84">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 84" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20human%20fetal%20lung%20cell%20atlas%20uncovers%20proximal-distal%20gradients%20of%20differentiation%20and%20key%20regulators%20of%20epithelial%20fates&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2022.11.005&amp;volume=185&amp;pages=4841-4860.e25&amp;publication_year=2022&amp;author=He%2CP">
                    Google Scholar</a> 
                </p></li><li data-counter="85."><p id="ref-CR85">Kleshchevnikov, V. et al. Cell2location maps fine-grained cell types in spatial transcriptomics. <i>Nat. Biotechnol.</i> <b>40</b>, 661–671 (2022).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/s41587-021-01139-4" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41587-021-01139-4" aria-label="Article reference 85" data-doi="10.1038/s41587-021-01139-4">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38Xht1KqsL8%3D" aria-label="CAS reference 85">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=35027729" aria-label="PubMed reference 85">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 85" href="http://scholar.google.com/scholar_lookup?&amp;title=Cell2location%20maps%20fine-grained%20cell%20types%20in%20spatial%20transcriptomics&amp;journal=Nat.%20Biotechnol.&amp;doi=10.1038%2Fs41587-021-01139-4&amp;volume=40&amp;pages=661-671&amp;publication_year=2022&amp;author=Kleshchevnikov%2CV">
                    Google Scholar</a> 
                </p></li><li data-counter="86."><p id="ref-CR86">Pedregosa, F. et al. Scikit-learn: machine learning in Python. <i>J. Mach. Learn. Res.</i> <b>12</b>, 2825–2830 (2011).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="mathscinet reference" href="http://www.ams.org/mathscinet-getitem?mr=2854348" aria-label="MathSciNet reference 86">MathSciNet</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="math reference" href="http://www.emis.de/MATH-item?1280.68189" aria-label="MATH reference 86">MATH</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 86" href="http://scholar.google.com/scholar_lookup?&amp;title=Scikit-learn%3A%20machine%20learning%20in%20Python&amp;journal=J.%20Mach.%20Learn.%20Res.&amp;volume=12&amp;pages=2825-2830&amp;publication_year=2011&amp;author=Pedregosa%2CF">
                    Google Scholar</a> 
                </p></li><li data-counter="87."><p id="ref-CR87">Schindelin, J. et al. Fiji: an open-source platform for biological-image analysis. <i>Nat. Methods</i> <b>9</b>, 676–682 (2012).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/nmeth.2019" data-track-action="article reference" href="https://doi.org/10.1038%2Fnmeth.2019" aria-label="Article reference 87" data-doi="10.1038/nmeth.2019">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XhtVKnurbJ" aria-label="CAS reference 87">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22743772" aria-label="PubMed reference 87">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 87" href="http://scholar.google.com/scholar_lookup?&amp;title=Fiji%3A%20an%20open-source%20platform%20for%20biological-image%20analysis&amp;journal=Nat.%20Methods&amp;doi=10.1038%2Fnmeth.2019&amp;volume=9&amp;pages=676-682&amp;publication_year=2012&amp;author=Schindelin%2CJ">
                    Google Scholar</a> 
                </p></li><li data-counter="88."><p id="ref-CR88">Bergen, V., Lange, M., Peidli, S., Wolf, F. A. &amp; Theis, F. J. Generalizing RNA velocity to transient cell states through dynamical modeling. <i>Nat. Biotechnol.</i> <b>38</b>, 1408–1414 (2020).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/s41587-020-0591-3" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41587-020-0591-3" aria-label="Article reference 88" data-doi="10.1038/s41587-020-0591-3">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhsFait7%2FE" aria-label="CAS reference 88">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32747759" aria-label="PubMed reference 88">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 88" href="http://scholar.google.com/scholar_lookup?&amp;title=Generalizing%20RNA%20velocity%20to%20transient%20cell%20states%20through%20dynamical%20modeling&amp;journal=Nat.%20Biotechnol.&amp;doi=10.1038%2Fs41587-020-0591-3&amp;volume=38&amp;pages=1408-1414&amp;publication_year=2020&amp;author=Bergen%2CV&amp;author=Lange%2CM&amp;author=Peidli%2CS&amp;author=Wolf%2CFA&amp;author=Theis%2CFJ">
                    Google Scholar</a> 
                </p></li><li data-counter="89."><p id="ref-CR89">Vento-Tormo, R. et al. Single-cell reconstruction of the early maternal–fetal interface in humans. <i>Nature</i> <b>563</b>, 347–353 (2018).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/s41586-018-0698-6" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41586-018-0698-6" aria-label="Article reference 89" data-doi="10.1038/s41586-018-0698-6">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2018Natur.563..347V" aria-label="ADS reference 89">ADS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXit1SnsbzI" aria-label="CAS reference 89">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30429548" aria-label="PubMed reference 89">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7612850" aria-label="PubMed Central reference 89">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 89" href="http://scholar.google.com/scholar_lookup?&amp;title=Single-cell%20reconstruction%20of%20the%20early%20maternal%E2%80%93fetal%20interface%20in%20humans&amp;journal=Nature&amp;doi=10.1038%2Fs41586-018-0698-6&amp;volume=563&amp;pages=347-353&amp;publication_year=2018&amp;author=Vento-Tormo%2CR">
                    Google Scholar</a> 
                </p></li><li data-counter="90."><p id="ref-CR90">Efremova, M., Vento-Tormo, M., Teichmann, S. A. &amp; Vento-Tormo, R. CellPhoneDB: inferring cell-cell communication from combined expression of multi-subunit ligand–receptor complexes. <i>Nat. Protoc.</i> <b>15</b>, 1484–1506 (2020).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/s41596-020-0292-x" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41596-020-0292-x" aria-label="Article reference 90" data-doi="10.1038/s41596-020-0292-x">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXjvFynsL0%3D" aria-label="CAS reference 90">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32103204" aria-label="PubMed reference 90">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 90" href="http://scholar.google.com/scholar_lookup?&amp;title=CellPhoneDB%3A%20inferring%20cell-cell%20communication%20from%20combined%20expression%20of%20multi-subunit%20ligand%E2%80%93receptor%20complexes&amp;journal=Nat.%20Protoc.&amp;doi=10.1038%2Fs41596-020-0292-x&amp;volume=15&amp;pages=1484-1506&amp;publication_year=2020&amp;author=Efremova%2CM&amp;author=Vento-Tormo%2CM&amp;author=Teichmann%2CSA&amp;author=Vento-Tormo%2CR">
                    Google Scholar</a> 
                </p></li><li data-counter="91."><p id="ref-CR91">Van de Sande, B. et al. A scalable SCENIC workflow for single-cell gene regulatory network analysis. <i>Nat. Protoc.</i> <b>15</b>, 2247–2276 (2020).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/s41596-020-0336-2" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41596-020-0336-2" aria-label="Article reference 91" data-doi="10.1038/s41596-020-0336-2">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32561888" aria-label="PubMed reference 91">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 91" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20scalable%20SCENIC%20workflow%20for%20single-cell%20gene%20regulatory%20network%20analysis&amp;journal=Nat.%20Protoc.&amp;doi=10.1038%2Fs41596-020-0336-2&amp;volume=15&amp;pages=2247-2276&amp;publication_year=2020&amp;author=Sande%2CB">
                    Google Scholar</a> 
                </p></li><li data-counter="92."><p id="ref-CR92">Jain, M. S. et al. MultiMAP: dimensionality reduction and integration of multimodal data. <i>Genome Biol.</i> <b>22</b>, 346 (2021).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1186/s13059-021-02565-y" data-track-action="article reference" href="https://doi.org/10.1186%2Fs13059-021-02565-y" aria-label="Article reference 92" data-doi="10.1186/s13059-021-02565-y">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34930412" aria-label="PubMed reference 92">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8686224" aria-label="PubMed Central reference 92">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 92" href="http://scholar.google.com/scholar_lookup?&amp;title=MultiMAP%3A%20dimensionality%20reduction%20and%20integration%20of%20multimodal%20data&amp;journal=Genome%20Biol.&amp;doi=10.1186%2Fs13059-021-02565-y&amp;volume=22&amp;publication_year=2021&amp;author=Jain%2CMS">
                    Google Scholar</a> 
                </p></li><li data-counter="93."><p id="ref-CR93">Wang, S. et al. Muscle stem cell immunostaining. <i>Curr. Protoc. Mouse Biol.</i> <b>8</b>, e47 (2018).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1002/cpmo.47" data-track-action="article reference" href="https://doi.org/10.1002%2Fcpmo.47" aria-label="Article reference 93" data-doi="10.1002/cpmo.47">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30106515" aria-label="PubMed reference 93">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 93" href="http://scholar.google.com/scholar_lookup?&amp;title=Muscle%20stem%20cell%20immunostaining&amp;journal=Curr.%20Protoc.%20Mouse%20Biol.&amp;doi=10.1002%2Fcpmo.47&amp;volume=8&amp;publication_year=2018&amp;author=Wang%2CS">
                    Google Scholar</a> 
                </p></li><li data-counter="94."><p id="ref-CR94">Berg, S. et al. ilastik: Interactive machine learning for (bio)image analysis. <i>Nat. Methods</i> <b>16</b>, 1226–1232 (2019).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/s41592-019-0582-9" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41592-019-0582-9" aria-label="Article reference 94" data-doi="10.1038/s41592-019-0582-9">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhvVOltbbF" aria-label="CAS reference 94">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31570887" aria-label="PubMed reference 94">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 94" href="http://scholar.google.com/scholar_lookup?&amp;title=ilastik%3A%20Interactive%20machine%20learning%20for%20%28bio%29image%20analysis&amp;journal=Nat.%20Methods&amp;doi=10.1038%2Fs41592-019-0582-9&amp;volume=16&amp;pages=1226-1232&amp;publication_year=2019&amp;author=Berg%2CS">
                    Google Scholar</a> 
                </p></li><li data-counter="95."><p id="ref-CR95">Belle, M. et al. Tridimensional visualization and analysis of early human development. <i>Cell</i> <b>169</b>, 161–173.e12 (2017).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.cell.2017.03.008" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.cell.2017.03.008" aria-label="Article reference 95" data-doi="10.1016/j.cell.2017.03.008">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXltVWnsL8%3D" aria-label="CAS reference 95">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28340341" aria-label="PubMed reference 95">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 95" href="http://scholar.google.com/scholar_lookup?&amp;title=Tridimensional%20visualization%20and%20analysis%20of%20early%20human%20development&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2017.03.008&amp;volume=169&amp;pages=161-173.e12&amp;publication_year=2017&amp;author=Belle%2CM">
                    Google Scholar</a> 
                </p></li><li data-counter="96."><p id="ref-CR96">Lapan, A. D. &amp; Gussoni, E. Isolation and characterization of human fetal myoblasts. <i>Methods Mol. Biol.</i> <b>798</b>, 3–19 (2012).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1007/978-1-61779-343-1_1" data-track-action="article reference" href="https://doi.org/10.1007%2F978-1-61779-343-1_1" aria-label="Article reference 96" data-doi="10.1007/978-1-61779-343-1_1">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XhsFagsLvO" aria-label="CAS reference 96">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22130828" aria-label="PubMed reference 96">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3249651" aria-label="PubMed Central reference 96">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 96" href="http://scholar.google.com/scholar_lookup?&amp;title=Isolation%20and%20characterization%20of%20human%20fetal%20myoblasts&amp;journal=Methods%20Mol.%20Biol.&amp;doi=10.1007%2F978-1-61779-343-1_1&amp;volume=798&amp;pages=3-19&amp;publication_year=2012&amp;author=Lapan%2CAD&amp;author=Gussoni%2CE">
                    Google Scholar</a> 
                </p></li></ol><p><a data-track="click" data-track-action="download citation references" data-track-label="link" rel="nofollow" href="https://citation-needed.springer.com/v2/references/10.1038/s41586-023-06806-x?format=refman&amp;flavour=references">Download references<svg width="16" height="16" focusable="false" role="img" aria-hidden="true"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-download-medium"></use></svg></a></p></div></div></div></section></div><section data-title="Acknowledgements"><div id="Ack1-section"><h2 id="Ack1">Acknowledgements</h2><p>We thank M. Thomson’s laboratory for help with mouse 10x loading; members of the Teichmann laboratory, Zhang laboratory, Marioni laboraory, Haniffa laboratory and Behjati laboratory for discussion and feedback; O. Pourquié for feedback on muscle development; A. Pyle, H. Xi and J. Langerman for sharing their raw data with us; and K. To for proofreading the manuscript. This work was supported by the Wellcome Trust grants 206194, 108413/A/15/D and 211276/Z/18/Z, as well as the National Key Research and Development Program (grants 2019YFA0801703 and 2022YFA1104904), the National Natural Science Foundation of China (grants 31871370 and 82203952), the Science and Technology Program of Guangzhou (grant 202002030429) (to H.Z.); and the China Postdoctoral Science Foundation (grant 2021M700936) and the Natural Science Foundation of Guangdong (grant 2019A1515011342) (to S.W.). P.H. holds a non-stipendiary research fellowship at St Edmund’s College, University of Cambridge. J.E.G.L. is funded by the Wellcome Trust under the clinical PhD programme (grant 222902/Z/21/Z) and supported by Darwin College, Cambridge through a Geoffrey Fisk Studentship. A.C. is funded by the INSERM cross-cutting programme HuDeCA 2018. R.B. was a recipient of a fellowship from the ‘Fondation pour la Recherche Médicale’ (FRM). This publication is part of the Human Cell Atlas (<a href="https://www.humancellatlas.org/publications/">www.humancellatlas.org/publications/#</a>).</p></div></section><section aria-labelledby="author-information" data-title="Author information"><div id="author-information-section"><h2 id="author-information">Author information</h2><div id="author-information-content"><p><span id="author-notes">Author notes</span></p><ol><li id="nAff18"><p>Lira Mamanova</p><p>Present address: Enhanc3D Genomics Ltd, Cambridge, UK</p></li><li id="nAff19"><p>Liam Bolt</p><p>Present address: Genomics England, London, UK</p></li><li id="nAff20"><p>Eirini S. Fasouli</p><p>Present address: Basic Research Center, Biomedical Research Foundation, Academy of Athens, Athens, Greece</p></li><li id="na1"><p>These authors contributed equally: Bao Zhang, Peng He, John E. G. Lawrence, Shuaiyu Wang</p></li></ol><h3 id="affiliations">Authors and Affiliations</h3><ol><li id="Aff1"><p>The Key Laboratory for Stem Cells and Tissue Engineering, Ministry of Education, Zhongshan School of Medicine, Sun Yat-sen University, Guangzhou, China</p><p>Bao Zhang, Shuaiyu Wang, Yixi Fu, Hao Yang, Chen Liang &amp; Hongbo Zhang</p></li><li id="Aff2"><p>European Molecular Biology Laboratory, European Bioinformatics Institute (EMBL-EBI), Wellcome Genome Campus, Hinxton, UK</p><p>Peng He, Nadav Yayon &amp; John C. Marioni</p></li><li id="Aff3"><p>Wellcome Sanger Institute, Wellcome Genome Campus, Hinxton, UK</p><p>Peng He, John E. G. Lawrence, Elizabeth Tuck, Kenny Roberts, Vitalii Kleshchevnikov, Lira Mamanova, Liam Bolt, Krzysztof Polanski, Tong Li, Rasa Elmentaite, Eirini S. Fasouli, Martin Prete, Nadav Yayon, Helen Firth, Omer Ali Bayraktar, John C. Marioni &amp; Sarah A. Teichmann</p></li><li id="Aff4"><p>Department of Trauma and Orthopaedics, Cambridge University Hospitals NHS Foundation Trust, Addenbrooke’s Hospital, Cambridge, UK</p><p>John E. G. Lawrence</p></li><li id="Aff5"><p>Department of Obstetrics, Guangzhou Institute of Pediatrics, Guangzhou Women and Children’s Medical Center, Guangzhou Medical University, Guangzhou, China</p><p>Shuaiyu Wang</p></li><li id="Aff6"><p>Division of Biology and Biological Engineering, California Institute of Technology, Pasadena, CA, USA</p><p>Brian A. Williams &amp; Barbara J. Wold</p></li><li id="Aff7"><p>John van Geest Centre for Brain Repair, Department of Clinical Neurosciences, University of Cambridge, Cambridge, UK</p><p>Xiaoling He &amp; Roger A. Barker</p></li><li id="Aff8"><p>Wellcome-MRC Cambridge Stem Cell Institute, University of Cambridge, Cambridge, UK</p><p>Xiaoling He, Roger A. Barker &amp; Mekayla A. Storer</p></li><li id="Aff9"><p>Institute of Human Virology, Key Laboratory of Tropical Disease Control of Ministry of Education, Zhongshan School of Medicine, Sun Yat-sen University, Guangzhou, China</p><p>Hui Zhang</p></li><li id="Aff10"><p>Sorbonne Université, INSERM, CNRS, Institut de la Vision, Paris, France</p><p>Raphael Blain &amp; Alain Chedotal</p></li><li id="Aff11"><p>Institut de pathologie, groupe hospitalier Est, hospices civils de Lyon, Lyon, France</p><p>Alain Chedotal</p></li><li id="Aff12"><p>University Claude Bernard Lyon 1, MeLiS, CNRS UMR5284, INSERM U1314, Lyon, France</p><p>Alain Chedotal</p></li><li id="Aff13"><p>MRC Human Genetics Unit, IGC, University of Edinburgh, WGH, Edinburgh, UK</p><p>David R. FitzPatrick</p></li><li id="Aff14"><p>Department of Clinical Neurosciences, Cambridge University Hospitals NHS Foundation, Cambridge, UK</p><p>Andrew Dean</p></li><li id="Aff15"><p>Advanced Medical Technology Center, the First Affiliated Hospital, Zhongshan School of Medicine, Sun Yat-sen University, Guangzhou, China</p><p>Hongbo Zhang</p></li><li id="Aff16"><p>Department of Histology and Embryology, Zhongshan School of Medicine, Sun Yat-sen University, Guangzhou, China</p><p>Hongbo Zhang</p></li><li id="Aff17"><p>Theory of Condensed Matter Group, Department of Physics, Cavendish Laboratory, University of Cambridge, Cambridge, UK</p><p>Sarah A. Teichmann</p></li></ol><h3 id="contributions">Contributions</h3><p>S.A.T. and H.Z. supervised the project. S.A.T. initiated and designed the project. X.H. and Y.F. carried out human tissue collection. B.A.W. carried out mouse tissue collection. B.A.W., L.M., L.B., R.E. and E.S.F. performed scRNA-seq. E.T. performed the Visium spatial experiments. S.W., K.R. and E.T. carried out the in situ staining and functional experiments. R.B. performed the light-sheet fluorescence microscopy. H.Y., C.L. and H.Z. provided experimental support. B.Z., P.H. and J.E.G.L. analysed the sequencing data and generated figures. V.K., K.P., M.P., N.Y. and T.L. provided computational support. D.R.F., H.F., A.D., M.A.S. and B.J.W. contributed to interpretation of the results. B.Z., P.H., J.E.G.L., S.W., H.Z. and S.A.T. wrote the manuscript. A.C., O.A.B., J.C.M., R.A.B., H.Z. and S.A.T. supervised team members. All authors contributed to the discussion and editing of the manuscript.</p><h3 id="corresponding-author">Corresponding authors</h3><p id="corresponding-author-list">Correspondence to
                <a id="corresp-c1" href="mailto:zhanghongbo@mail.sysu.edu.cn">Hongbo Zhang</a> or <a id="corresp-c2" href="mailto:st9@sanger.ac.uk">Sarah A. Teichmann</a>.</p></div></div></section><section data-title="Ethics declarations"><div id="ethics-section"><h2 id="ethics">Ethics declarations</h2><div id="ethics-content">
            
              <h3 id="FPar2">Competing interests</h3>
              <p>In the past 3 years, S.A.T. has consulted for or been a member of scientific advisory boards at Qiagen, Sanofi, GlaxoSmithKline and ForeSite Labs. She is a consultant and equity holder for TransitionBio and EnsoCell. J.C.M has been an employee of Genentech, Inc. since September 2022. The remaining authors declare no competing interests.</p>
            
          </div></div></section><section data-title="Peer review"><div id="peer-review-section"><h2 id="peer-review">Peer review</h2><div id="peer-review-content">
            
            
              <h3 id="FPar1">Peer review information</h3>
              <p><i>Nature</i> thanks the anonymous reviewers for their contribution to the peer review of this work. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41586-023-06806-x#MOESM3">Peer reviewer reports</a> are available.</p>
            
          </div></div></section><section data-title="Additional information"><div id="additional-information-section"><h2 id="additional-information">Additional information</h2><p><b>Publisher’s note</b> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div></section><section data-title="Extended data figures and tables"><div id="Sec39-section"><h2 id="Sec39">Extended data figures and tables</h2><div id="Sec39-content"><div data-test="supplementary-info"><div data-test="supp-item" id="Fig7"><h3><a data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 1 data quality and preprocessin" href="https://www.nature.com/articles/s41586-023-06806-x/figures/7" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41586-023-06806-x/MediaObjects/41586_2023_6806_Fig7_ESM.jpg">Extended Data Fig. 1 Data quality and preprocessing of human scRNA-seq and spatial visium data.</a></h3><div data-component="thumbnail-container"><p><b>a</b>, Bar plot and violin plot showing the sample size and per-cell quality of each library, separately, coloured by stage. <b>b</b>, Dot plot showing the expression level of marker genes for each cell cluster. The colour bar indicates the linearly scaled mean of expression level. Cluster abbreviations same as Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig1">1</a>. <b>c</b>, Bar plot and violin plot showing the number of voxels and per-voxel quality of each 10x Visium library, separately, coloured by stage (n = 1 library per bar/violin). <b>d</b>, Scatter plot showing the reconstruction accuracy of cell2location.</p><p><a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41586-023-06806-x#MOESM16">Source Data</a></p></div></div><div data-test="supp-item" id="Fig8"><h3><a data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 2 the heterogeneity of epidermi" href="https://www.nature.com/articles/s41586-023-06806-x/figures/8" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41586-023-06806-x/MediaObjects/41586_2023_6806_Fig8_ESM.jpg">Extended Data Fig. 2 The heterogeneity of epidermis and fibroblast.</a></h3><div data-component="thumbnail-container"><p><b>a</b>, Uniform manifold approximation and projection (UMAP) visualization of AER-basal, basal and periderm cells. <b>b</b>, Violin plot showing the normalised and log-transformed expression level of <i>WNT6</i>, <i>CYP26A1</i>, <i>SP8</i>, <i>ETV4</i> and <i>FGF8</i> in AER-basal, basal and periderm cells. <b>c</b>, UMAP (left panel) and violin plot (right panel) showing the normalised and log-transformed expression level of <i>FGF8</i> in the human limb. ProxMes, proximal mesenchyme; TenoProg, tendon progenitor; MyoProg, myogenic progenitor; MyoB, myoblast; MyoC, Myocyte. <b>d</b>, Heatmap across tissue section from PCW6.2 (post conception week 6 plus 2 days) human hindlimb showing <i>FGF8</i> expression. <b>e</b>, RNA-ISH of tissue sections from human hind limb showing the expression pattern of <i>FGF8</i>. Scale bar, 500 μm. <b>f</b>, <b>h</b>, Dot plot showing the expression level of marker genes for selected fibroblast (Fibro) clusters (<b>f</b>) and muscle interstitial fibroblast (InterMusFibro) (<b>h</b>). The colour bar indicates the linearly scaled values of expression level. DermFibro, dermal Fibro; DermFibroProg, DermFibro progenitor. <b>g</b>, <b>i</b>, Heatmaps across tissue sections from PCW8.1 human hindlimb showing inferred abundance of each fibroblast cluster (<b>g</b>) and InterMusFibro (<b>i</b>). <b>j</b>, Immunofluorescence staining of MYH3 and ALDH1A3 on the skeletal muscle tissue (as also shown by H&amp;E staining) from a PCW9 longitudinal section. n = 2. Scale bar, 50 μm.</p><p><a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41586-023-06806-x#MOESM17">Source Data</a></p></div></div><div data-test="supp-item" id="Fig9"><h3><a data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 3 dynamic changes in cell clust" href="https://www.nature.com/articles/s41586-023-06806-x/figures/9" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41586-023-06806-x/MediaObjects/41586_2023_6806_Fig9_ESM.jpg">Extended Data Fig. 3 Dynamic changes in cell clusters of the human embryonic limb over developmental time.</a></h3><div data-component="thumbnail-container"><p><b>a</b>, Fraction of cell cluster per time point, coloured by cell clusters and grouped by cell compartment. Cluster abbreviations same as Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig1">1</a>. <b>b</b>, Uniform manifold approximation and projection (UMAP) visualisation of cells per post conception week (PCW), coloured by cell cluster in <b>a</b>.</p><p><a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41586-023-06806-x#MOESM18">Source Data</a></p></div></div><div data-test="supp-item" id="Fig10"><h3><a data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 4 the heterogeneity of mesenchy" href="https://www.nature.com/articles/s41586-023-06806-x/figures/10" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41586-023-06806-x/MediaObjects/41586_2023_6806_Fig10_ESM.jpg">Extended Data Fig. 4 The heterogeneity of Mesenchyme.</a></h3><div data-component="thumbnail-container"><p><b>a</b>, Uniform manifold approximation and projection (UMAP) plot showing the cell clusters of Chondrogenic progenitor (ChondroProg), mesenchymal condensate cell (MesCond), transitional mesenchyme (TransMes) and distal mesenchyme (DistalMes) (left panel) as well as the expression level of <i>SP9</i>, <i>LHX2</i>, <i>MSX1</i>, <i>IRX1</i> and <i>SOX9</i>. The colour bar indicates the normalised and log-transformed expression values. <b>b</b>, Heatmaps across tissue sections from human hindlimb at stage of PCW5.6 (post conception week 5 plus 6 days) and PCW6.2 showing the spatial expression pattern of <i>SP9</i>, <i>LHX2</i>, <i>MSX1</i>, <i>IRX1</i> and <i>SOX9</i>. The colour bar indicates the normalised log-transformed expression values. <b>c</b>, <b>d</b>, RNA-ISH of tissue sections from human hindlimb showing the spatial expression pattern of <i>SP9</i>, <i>LHX2</i> and <i>MSX1</i> (<b>c</b>), as well as <i>MSX1</i>, <i>IRX1</i> and <i>SOX9</i> (<b>d</b>) at different stage. Scale bar, 1 mm. <b>e</b>, Heatmaps across tissue sections from human hindlimb at stage of PCW5.6 showing the cell cycle of G1, G2M and S phase. <b>f</b>, Dot plot showing the expression level of marker genes for different cell clusters of mesenchyme. The colour bar indicates the linearly scaled mean of expression level. ProxMes, proximal Mes. <b>g</b>, <b>h</b>, RNA-ISH of tissue sections from human hindlimb showing the spatial expression pattern of <i>IRX1</i>, <i>SP9</i> and <i>RDH10</i> (<b>g</b>), as well as <i>MEIS2</i>, <i>WT1</i>, <i>CITED1</i> and <i>ISL1</i> (<b>h</b>) n = 2-4 for RNA-ISH. Scale bar, 1 mm.</p><p><a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41586-023-06806-x#MOESM19">Source Data</a></p></div></div><div data-test="supp-item" id="Fig11"><h3><a data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 5 identification of novel cell " href="https://www.nature.com/articles/s41586-023-06806-x/figures/11" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41586-023-06806-x/MediaObjects/41586_2023_6806_Fig11_ESM.jpg">Extended Data Fig. 5 Identification of novel cell types at spatial and single-cell level.</a></h3><div data-component="thumbnail-container"><p><b>a</b>, The VisiumStitcher workflow for merging the 10x Visium spatial data of the human limb. <b>b</b>, Immunofuorescence staining of RUNX2, THBS2 and COL2A1 on the longitudinal section of the tibia from a PCW7 embryo. n = 3. Scale bar, 50 μm. <b>c</b>, Uniform manifold approximation and projection (UMAP) visualization of smooth muscle progenitor (SMProg), neural fibroblasts (NeuralFibro), pericyte and smooth muscle (SMC). <b>d</b>, Violin plot showing the expression level of <i>FOXS1</i>, <i>PI16</i>, <i>FGF19</i>, <i>KCNJ8</i> and <i>ACTA2</i> in SMProg, NeuralFibro, pericyte and SMC, using normalised and log-transformed values. <b>e</b>, Immunohistochemical staining of PI16 and FGF19 showing the NeuralFibro in the sciatic nerve at PCW9. The neurofilament was stained with NEFH antibody. A neighbouring section stained with H&amp;E solution is also shown. n = 4. Scale bar, 50 μm. <b>f</b>, UMAP visualization of tendon progenitor (TenoProg), tenocytes (Teno) and perimysium cells. <b>g</b>, Violin plot showing the expression level of <i>SCX</i>, <i>TNMD</i>, <i>GCG</i>, <i>BGN</i> and <i>KERA</i> in TenoProg, Perimysium, and Teno. The expression level of genes is the normalised and log-transformed values. <b>h</b>, RNA-ISH (<i>GCG</i>) combined with immunohistochemistry (MYH3 and KERA) of tissue sections from human hind limb showing the spatial expression pattern of <i>GCG</i>, MYH3 and KERA. n = 2. Scale bar, 1 mm.</p><p><a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41586-023-06806-x#MOESM20">Source Data</a></p></div></div><div data-test="supp-item" id="Fig12"><h3><a data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 6 spatial expression patterns o" href="https://www.nature.com/articles/s41586-023-06806-x/figures/12" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41586-023-06806-x/MediaObjects/41586_2023_6806_Fig12_ESM.jpg">Extended Data Fig. 6 Spatial expression patterns of genes that determine human limb axis formation and morphogenesis.</a></h3><div data-component="thumbnail-container"><p><b>a</b>, Overview of analysis workflow to identify genes specific to spatial location. <b>b-e</b>, Heatmaps across tissue section from the human hindlimb at stage of PCW5.6 showing spatial expression pattern of genes specific to proximal (<b>b</b>), distal (<b>c</b>), anterior (<b>d</b>) or posterior (<b>e</b>) regions. The expression level of genes is the normalised and log-transformed values. <b>f</b>, Heatmaps across tissue section from human hindlimb at stage of PCW5.6 showing spatial expression pattern of homeobox (HOX) A (top panel) and D (bottom panel) family genes. <b>g</b>, <b>h</b>, Heatmaps across tissue sections from the human hindlimb at stage PCW6.2 showing inferred abundance of macrophage (<b>g</b>) and endothelial cells (vein endothelial cells (VeinEndo) and arterial endothelial cells (ArterialEndo), <b>h</b>) as well as expression of maker genes. Anter, anterior; Post, posterior; Prox, proximal; Dist, distal. The expression level of genes is the normalised and logarithmic value of raw counts. <b>i</b>, Spatially resolved heatmaps across tissue section from the human hindlimb at stage of PCW6.2 showing spatial expression pattern of digit-associated genes in normalized and log-transformed values.</p><p><a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41586-023-06806-x#MOESM21">Source Data</a></p></div></div><div data-test="supp-item" id="Fig13"><h3><a data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 7 the transcriptional regulatio" href="https://www.nature.com/articles/s41586-023-06806-x/figures/13" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41586-023-06806-x/MediaObjects/41586_2023_6806_Fig13_ESM.jpg">Extended Data Fig. 7 The transcriptional regulation of LPM differentiation in the human limb.</a></h3><div data-component="thumbnail-container"><p><b>a</b>, Heatmap illustrating the vertically normalised mean activity of selected transcription factors for each cell type from soft connective lineage of LPM. <b>b</b>, Force-directed graphs (top panel) and heatmaps across tissue section from the human hindlimb at PCW5.6 and PCW6.2 (bottom panel) showing the expression pattern of representative transcription factors in normalised and log-transformed values. <b>c</b>, Stacked bar chart showing the fraction of cell cluster per time point, coloured by cell type and grouped by tissue type. PCW, post conception week. SmM, smooth muscle group; other abbreviations as per Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06806-x#Fig1">1</a>. <b>d</b>, Dot plot showing the expression level of marker genes of osteochondral cell clusters. The colour bar indicates the mean normalised expression level. <b>e</b>, Force-directed graph of cells per time point, coloured by cell type in <b>a</b>. <b>f</b>, Uniform manifold approximation and projection (UMAP) visualization of the chondrocyte lineage with arrows representing inferred differentiation directions (See Methods). <b>g</b>, Stacked bar chart showing the fraction of phase of cell cycle per osteochondral cell cluster. <b>h</b>, Force directed graph showing the expression level of <i>SCX</i> and <i>SOX9</i>. The colour bar indicates the normalised and log-transformed expression values. <b>i</b>, Scatter plots showing the expression level of <i>SCX</i> and <i>SOX9</i> expression in all LPM-derived cells in normalised and log-transformed values. The percentages of double positive cells are given. <b>j</b>, Heatmap across tissue section from the human hindlimb at PCW6.2 showing <i>SCX</i> and <i>SOX9</i> expression in normalised and log-transformed values. The voxels marked with yellow asterisks express both <i>SCX</i> and <i>SOX9</i>. <b>k</b>, RNA-ISH of tissue sections from the human hindlimb showing the expression of <i>SCX</i> and <i>SOX9</i> in situ.</p><p><a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41586-023-06806-x#MOESM22">Source Data</a></p></div></div><div data-test="supp-item" id="Fig14"><h3><a data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 8 the transcriptional regulatio" href="https://www.nature.com/articles/s41586-023-06806-x/figures/14" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41586-023-06806-x/MediaObjects/41586_2023_6806_Fig14_ESM.jpg">Extended Data Fig. 8 The transcriptional regulation of myogenesis in human and mouse.</a></h3><div data-component="thumbnail-container"><p><b>a</b>, Dot plot showing expression level of transcription factors per cell cluster in humans, coloured by group (green, first myogenesis; pink, second myogenesis; yellow, both). The colour bar indicates the linearly scaled mean of expression level. MyoProg, myogenic progenitor; MyoB, myoblast; MyoC, myocyte. <b>b</b>, Force-directed graph showing muscle populations per time point. <b>c</b>, Force-directed graph showing the expression of <i>PITX1</i> between forelimb (left panel) and hindlimb (right panel) in cells derived from mesenchyme and skeletal muscle lineage. The expression level of genes is the normalised and logarithmic value of raw counts. <b>d</b>, Force-directed graph of human and mouse skeletal muscle cells, coloured by cell clusters. <b>e</b>, Stacked bar chart showing the fraction of mouse cell clusters per time point, followed by the colour code of mouse cell clusters in d. <b>f</b>, Scatter plots of <i>PAX3</i> and <i>PAX7</i> expression (normalised and log-transformed) in mouse and human skeletal muscle cells. The percentages of double positive cells are given. <b>g</b>, Force-directed graph of mouse cells per time point, coloured by cell cluster in <b>d</b>. <b>h</b>, Dotplots of selected genes expressed in humans and mice. The colour bar indicates linearly scaled average expression levels.</p><p><a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41586-023-06806-x#MOESM23">Source Data</a></p></div></div><div data-test="supp-item" id="Fig15"><h3><a data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 9 comparing human and mouse emb" href="https://www.nature.com/articles/s41586-023-06806-x/figures/15" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41586-023-06806-x/MediaObjects/41586_2023_6806_Fig15_ESM.jpg">Extended Data Fig. 9 Comparing human and mouse embryonic limbs.</a></h3><div data-component="thumbnail-container"><p><b>a</b>, Violin plots of sample quality for all the scRNA-seq data in the integrated atlas, coloured by library ID and group by dataset at the bottom. <b>b</b>, <b>c</b>, The integrated mouse scRNA-seq data projected on a shared UMAP plane, coloured by cell clusters (<b>b</b>) or metadata (<b>c</b>). <b>d</b>, Cell-cluster proportions of each scRNA-seq library with dissection region, location and species labelled at the bottom. <b>e</b>, Genes enriched in proximal or distal segments in human and mouse. <b>f</b>, Genes enriched in the forelimb or hindlimb in human and mouse.</p><p><a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41586-023-06806-x#MOESM24">Source Data</a></p></div></div><div data-test="supp-item" id="Fig16"><h3><a data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 10 the lpm lineage in human and" href="https://www.nature.com/articles/s41586-023-06806-x/figures/16" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41586-023-06806-x/MediaObjects/41586_2023_6806_Fig16_ESM.jpg">Extended Data Fig. 10 The LPM lineage in human and mouse.</a></h3><div data-component="thumbnail-container"><p><b>a</b>, Force-directed graph of human and mouse LPM-derived cells, coloured by cell clusters. <b>b</b>, Stacked bar chart showing the fraction of mouse cell clusters per time point, followed by the colour code of mouse cell clusters in <b>a</b>. <b>c</b>, Force-directed graph of mouse cells per time point, coloured by cell cluster in <b>a</b>. <b>d</b>, Dotplot of selected genes expressed in human and mouse. The colour bar indicates the linearly scaled average expression levels. <b>e</b>, UMAP plot showing the expression of <i>Fgf8</i> in mouse limb cell atlas. The expression level of gene on the left is the normalised and logarithmic value of raw counts. <b>f</b>, Violin plot showing the normalised and log-transformed expression level of <i>Fgf8</i> in cell clusters of mouse.</p><p><a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41586-023-06806-x#MOESM25">Source Data</a></p></div></div></div></div></div></section><section data-title="Supplementary information"><div id="Sec40-section"><h2 id="Sec40">Supplementary information</h2><div id="Sec40-content"><div data-test="supplementary-info"><div data-test="supp-item" id="MOESM1"><h3><a data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="supplementary information" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41586-023-06806-x/MediaObjects/41586_2023_6806_MOESM1_ESM.docx" data-supp-info-image="">Supplementary Information</a></h3><p>This file contains a table containing cluster label abbreviations, Supplementary Figure 1 (gating strategy for sorting myoblasts), Supplementary Discussion, full legend for Supplementary Video 1 and Supplementary References for Supplementary Table 3.</p></div><div data-test="supp-item" id="MOESM4"><h3><a data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="supplementary table 1" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41586-023-06806-x/MediaObjects/41586_2023_6806_MOESM4_ESM.xlsx" data-supp-info-image="">Supplementary Table 1</a></h3><p>Marker genes for human limb cell clusters. Computed marker genes and literature-supported marker genes for annotation are included in separate tabs.</p></div><div data-test="supp-item" id="MOESM5"><h3><a data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="supplementary table 2" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41586-023-06806-x/MediaObjects/41586_2023_6806_MOESM5_ESM.xlsx" data-supp-info-image="">Supplementary Table 2</a></h3><p>Digit related differentially expressed genes. DEGs and disease-associated DEGs calculated from Visium data are included in separate tabs.</p></div><div data-test="supp-item" id="MOESM6"><h3><a data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="supplementary table 3" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41586-023-06806-x/MediaObjects/41586_2023_6806_MOESM6_ESM.xlsx" data-supp-info-image="">Supplementary Table 3</a></h3><p>Mouse-human phenotype comparison. Phenotypes of 10 classic human musculoskeletal disease genes and those of their homologs in mice.</p></div><div data-test="supp-item" id="MOESM7"><h3><a data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="supplementary table 4" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41586-023-06806-x/MediaObjects/41586_2023_6806_MOESM7_ESM.xlsx" data-supp-info-image="">Supplementary Table 4</a></h3><p>SCENIC scores for LPM and SkM. SCENIC scores of transcription factors in each cell cluster of the LPM and the skeletal muscle compartments.</p></div><div data-test="supp-item" id="MOESM8"><h3><a data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="supplementary table 5" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41586-023-06806-x/MediaObjects/41586_2023_6806_MOESM8_ESM.xlsx" data-supp-info-image="">Supplementary Table 5</a></h3><p>Sample metadata. Stage, hospital pseudo ID, Sample ID, specimen ID, cell number, gender, measurements, kit version and anatomical description of each sequencing library.</p></div><div data-test="supp-item" id="MOESM9"><h3><a data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="supplementary video 1" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41586-023-06806-x/MediaObjects/41586_2023_6806_MOESM9_ESM.mp4" data-supp-info-image="">Supplementary Video 1</a></h3><p>Light sheet fluorescence microscopy of marker genes for mesenchyme populations in the developing autopod.</p></div></div></div></div></section><section data-title="Source data"></section><section data-title="Rights and permissions"><div id="rightslink-section"><h2 id="rightslink">Rights and permissions</h2><div id="rightslink-content">
              <p><b>Open Access</b>  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>.</p>
            <p><a data-track="click" data-track-action="view rights and permissions" data-track-label="link" href="https://s100.copyright.com/AppDispatchServlet?title=A%20human%20embryonic%20limb%20cell%20atlas%20resolved%20in%20space%20and%20time&amp;author=Bao%20Zhang%20et%20al&amp;contentID=10.1038%2Fs41586-023-06806-x&amp;copyright=The%20Author%28s%29&amp;publication=0028-0836&amp;publicationDate=2023-12-06&amp;publisherName=SpringerNature&amp;orderBeanReset=true&amp;oa=CC%20BY">Reprints and Permissions</a></p></div></div></section><section aria-labelledby="article-info" data-title="About this article"><div id="article-info-section"><h2 id="article-info">About this article</h2><div id="article-info-content"><div><p><a data-crossmark="10.1038/s41586-023-06806-x" target="_blank" rel="noopener" href="https://crossmark.crossref.org/dialog/?doi=10.1038/s41586-023-06806-x" data-track="click" data-track-action="Click Crossmark" data-track-label="link" data-test="crossmark"><img width="57" height="81" alt="Check for updates. Verify currency and authenticity via CrossMark" src="data:image/svg+xml;base64,<svg height="81" width="57" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="m17.35 35.45 21.3-14.2v-17.03h-21.3" fill="#989898"/><path d="m38.65 35.45-21.3-14.2v-17.03h21.3" fill="#747474"/><path d="m28 .5c-12.98 0-23.5 10.52-23.5 23.5s10.52 23.5 23.5 23.5 23.5-10.52 23.5-23.5c0-6.23-2.48-12.21-6.88-16.62-4.41-4.4-10.39-6.88-16.62-6.88zm0 41.25c-9.8 0-17.75-7.95-17.75-17.75s7.95-17.75 17.75-17.75 17.75 7.95 17.75 17.75c0 4.71-1.87 9.22-5.2 12.55s-7.84 5.2-12.55 5.2z" fill="#535353"/><path d="m41 36c-5.81 6.23-15.23 7.45-22.43 2.9-7.21-4.55-10.16-13.57-7.03-21.5l-4.92-3.11c-4.95 10.7-1.19 23.42 8.78 29.71 9.97 6.3 23.07 4.22 30.6-4.86z" fill="#9c9c9c"/><path d="m.2 58.45c0-.75.11-1.42.33-2.01s.52-1.09.91-1.5c.38-.41.83-.73 1.34-.94.51-.22 1.06-.32 1.65-.32.56 0 1.06.11 1.51.35.44.23.81.5 1.1.81l-.91 1.01c-.24-.24-.49-.42-.75-.56-.27-.13-.58-.2-.93-.2-.39 0-.73.08-1.05.23-.31.16-.58.37-.81.66-.23.28-.41.63-.53 1.04-.13.41-.19.88-.19 1.39 0 1.04.23 1.86.68 2.46.45.59 1.06.88 1.84.88.41 0 .77-.07 1.07-.23s.59-.39.85-.68l.91 1c-.38.43-.8.76-1.28.99-.47.22-1 .34-1.58.34-.59 0-1.13-.1-1.64-.31-.5-.2-.94-.51-1.31-.91-.38-.4-.67-.9-.88-1.48-.22-.59-.33-1.26-.33-2.02zm8.4-5.33h1.61v2.54l-.05 1.33c.29-.27.61-.51.96-.72s.76-.31 1.24-.31c.73 0 1.27.23 1.61.71.33.47.5 1.14.5 2.02v4.31h-1.61v-4.1c0-.57-.08-.97-.25-1.21-.17-.23-.45-.35-.83-.35-.3 0-.56.08-.79.22-.23.15-.49.36-.78.64v4.8h-1.61zm7.37 6.45c0-.56.09-1.06.26-1.51.18-.45.42-.83.71-1.14.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.36c.07.62.29 1.1.65 1.44.36.33.82.5 1.38.5.29 0 .57-.04.83-.13s.51-.21.76-.37l.55 1.01c-.33.21-.69.39-1.09.53-.41.14-.83.21-1.26.21-.48 0-.92-.08-1.34-.25-.41-.16-.76-.4-1.07-.7-.31-.31-.55-.69-.72-1.13-.18-.44-.26-.95-.26-1.52zm4.6-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.07.45-.31.29-.5.73-.58 1.3zm2.5.62c0-.57.09-1.08.28-1.53.18-.44.43-.82.75-1.13s.69-.54 1.1-.71c.42-.16.85-.24 1.31-.24.45 0 .84.08 1.17.23s.61.34.85.57l-.77 1.02c-.19-.16-.38-.28-.56-.37-.19-.09-.39-.14-.61-.14-.56 0-1.01.21-1.35.63-.35.41-.52.97-.52 1.67 0 .69.17 1.24.51 1.66.34.41.78.62 1.32.62.28 0 .54-.06.78-.17.24-.12.45-.26.64-.42l.67 1.03c-.33.29-.69.51-1.08.65-.39.15-.78.23-1.18.23-.46 0-.9-.08-1.31-.24-.4-.16-.75-.39-1.05-.7s-.53-.69-.7-1.13c-.17-.45-.25-.96-.25-1.53zm6.91-6.45h1.58v6.17h.05l2.54-3.16h1.77l-2.35 2.8 2.59 4.07h-1.75l-1.77-2.98-1.08 1.23v1.75h-1.58zm13.69 1.27c-.25-.11-.5-.17-.75-.17-.58 0-.87.39-.87 1.16v.75h1.34v1.27h-1.34v5.6h-1.61v-5.6h-.92v-1.2l.92-.07v-.72c0-.35.04-.68.13-.98.08-.31.21-.57.4-.79s.42-.39.71-.51c.28-.12.63-.18 1.04-.18.24 0 .48.02.69.07.22.05.41.1.57.17zm.48 5.18c0-.57.09-1.08.27-1.53.17-.44.41-.82.72-1.13.3-.31.65-.54 1.04-.71.39-.16.8-.24 1.23-.24s.84.08 1.24.24c.4.17.74.4 1.04.71s.54.69.72 1.13c.19.45.28.96.28 1.53s-.09 1.08-.28 1.53c-.18.44-.42.82-.72 1.13s-.64.54-1.04.7-.81.24-1.24.24-.84-.08-1.23-.24-.74-.39-1.04-.7c-.31-.31-.55-.69-.72-1.13-.18-.45-.27-.96-.27-1.53zm1.65 0c0 .69.14 1.24.43 1.66.28.41.68.62 1.18.62.51 0 .9-.21 1.19-.62.29-.42.44-.97.44-1.66 0-.7-.15-1.26-.44-1.67-.29-.42-.68-.63-1.19-.63-.5 0-.9.21-1.18.63-.29.41-.43.97-.43 1.67zm6.48-3.44h1.33l.12 1.21h.05c.24-.44.54-.79.88-1.02.35-.24.7-.36 1.07-.36.32 0 .59.05.78.14l-.28 1.4-.33-.09c-.11-.01-.23-.02-.38-.02-.27 0-.56.1-.86.31s-.55.58-.77 1.1v4.2h-1.61zm-47.87 15h1.61v4.1c0 .57.08.97.25 1.2.17.24.44.35.81.35.3 0 .57-.07.8-.22.22-.15.47-.39.73-.73v-4.7h1.61v6.87h-1.32l-.12-1.01h-.04c-.3.36-.63.64-.98.86-.35.21-.76.32-1.24.32-.73 0-1.27-.24-1.61-.71-.33-.47-.5-1.14-.5-2.02zm9.46 7.43v2.16h-1.61v-9.59h1.33l.12.72h.05c.29-.24.61-.45.97-.63.35-.17.72-.26 1.1-.26.43 0 .81.08 1.15.24.33.17.61.4.84.71.24.31.41.68.53 1.11.13.42.19.91.19 1.44 0 .59-.09 1.11-.25 1.57-.16.47-.38.85-.65 1.16-.27.32-.58.56-.94.73-.35.16-.72.25-1.1.25-.3 0-.6-.07-.9-.2s-.59-.31-.87-.56zm0-2.3c.26.22.5.37.73.45.24.09.46.13.66.13.46 0 .84-.2 1.15-.6.31-.39.46-.98.46-1.77 0-.69-.12-1.22-.35-1.61-.23-.38-.61-.57-1.13-.57-.49 0-.99.26-1.52.77zm5.87-1.69c0-.56.08-1.06.25-1.51.16-.45.37-.83.65-1.14.27-.3.58-.54.93-.71s.71-.25 1.08-.25c.39 0 .73.07 1 .2.27.14.54.32.81.55l-.06-1.1v-2.49h1.61v9.88h-1.33l-.11-.74h-.06c-.25.25-.54.46-.88.64-.33.18-.69.27-1.06.27-.87 0-1.56-.32-2.07-.95s-.76-1.51-.76-2.65zm1.67-.01c0 .74.13 1.31.4 1.7.26.38.65.58 1.15.58.51 0 .99-.26 1.44-.77v-3.21c-.24-.21-.48-.36-.7-.45-.23-.08-.46-.12-.7-.12-.45 0-.82.19-1.13.59-.31.39-.46.95-.46 1.68zm6.35 1.59c0-.73.32-1.3.97-1.71.64-.4 1.67-.68 3.08-.84 0-.17-.02-.34-.07-.51-.05-.16-.12-.3-.22-.43s-.22-.22-.38-.3c-.15-.06-.34-.1-.58-.1-.34 0-.68.07-1 .2s-.63.29-.93.47l-.59-1.08c.39-.24.81-.45 1.28-.63.47-.17.99-.26 1.54-.26.86 0 1.51.25 1.93.76s.63 1.25.63 2.21v4.07h-1.32l-.12-.76h-.05c-.3.27-.63.48-.98.66s-.73.27-1.14.27c-.61 0-1.1-.19-1.48-.56-.38-.36-.57-.85-.57-1.46zm1.57-.12c0 .3.09.53.27.67.19.14.42.21.71.21.28 0 .54-.07.77-.2s.48-.31.73-.56v-1.54c-.47.06-.86.13-1.18.23-.31.09-.57.19-.76.31s-.33.25-.41.4c-.09.15-.13.31-.13.48zm6.29-3.63h-.98v-1.2l1.06-.07.2-1.88h1.34v1.88h1.75v1.27h-1.75v3.28c0 .8.32 1.2.97 1.2.12 0 .24-.01.37-.04.12-.03.24-.07.34-.11l.28 1.19c-.19.06-.4.12-.64.17-.23.05-.49.08-.76.08-.4 0-.74-.06-1.02-.18-.27-.13-.49-.3-.67-.52-.17-.21-.3-.48-.37-.78-.08-.3-.12-.64-.12-1.01zm4.36 2.17c0-.56.09-1.06.27-1.51s.41-.83.71-1.14c.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.37c.08.62.29 1.1.65 1.44.36.33.82.5 1.38.5.3 0 .58-.04.84-.13.25-.09.51-.21.76-.37l.54 1.01c-.32.21-.69.39-1.09.53s-.82.21-1.26.21c-.47 0-.92-.08-1.33-.25-.41-.16-.77-.4-1.08-.7-.3-.31-.54-.69-.72-1.13-.17-.44-.26-.95-.26-1.52zm4.61-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.08.45-.31.29-.5.73-.57 1.3zm3.01 2.23c.31.24.61.43.92.57.3.13.63.2.98.2.38 0 .65-.08.83-.23s.27-.35.27-.6c0-.14-.05-.26-.13-.37-.08-.1-.2-.2-.34-.28-.14-.09-.29-.16-.47-.23l-.53-.22c-.23-.09-.46-.18-.69-.3-.23-.11-.44-.24-.62-.4s-.33-.35-.45-.55c-.12-.21-.18-.46-.18-.75 0-.61.23-1.1.68-1.49.44-.38 1.06-.57 1.83-.57.48 0 .91.08 1.29.25s.71.36.99.57l-.74.98c-.24-.17-.49-.32-.73-.42-.25-.11-.51-.16-.78-.16-.35 0-.6.07-.76.21-.17.15-.25.33-.25.54 0 .14.04.26.12.36s.18.18.31.26c.14.07.29.14.46.21l.54.19c.23.09.47.18.7.29s.44.24.64.4c.19.16.34.35.46.58.11.23.17.5.17.82 0 .3-.06.58-.17.83-.12.26-.29.48-.51.68-.23.19-.51.34-.84.45-.34.11-.72.17-1.15.17-.48 0-.95-.09-1.41-.27-.46-.19-.86-.41-1.2-.68z" fill="#535353"/></g></svg>"/></a></p><div><h3 id="citeas">Cite this article</h3><p>Zhang, B., He, P., Lawrence, J.E.G. <i>et al.</i> A human embryonic limb cell atlas resolved in space and time.
                    <i>Nature</i>  (2023). https://doi.org/10.1038/s41586-023-06806-x</p><p><a data-test="citation-link" data-track="click" data-track-action="download article citation" data-track-label="link" data-track-external="" rel="nofollow" href="https://citation-needed.springer.com/v2/references/10.1038/s41586-023-06806-x?format=refman&amp;flavour=citation">Download citation<svg width="16" height="16" focusable="false" role="img" aria-hidden="true"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-download-medium"></use></svg></a></p><ul data-test="publication-history"><li><p>Received<span>: </span><span><time datetime="2022-04-15">15 April 2022</time></span></p></li><li><p>Accepted<span>: </span><span><time datetime="2023-10-31">31 October 2023</time></span></p></li><li><p>Published<span>: </span><span><time datetime="2023-12-06">06 December 2023</time></span></p></li><li><p><abbr title="Digital Object Identifier">DOI</abbr><span>: </span><span>https://doi.org/10.1038/s41586-023-06806-x</span></p></li></ul></div></div></div></div></section>
            </div></div>
  </body>
</html>
