<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <title>James Routley | Feed</title>
    <link
      rel="stylesheet"
      type="text/css"
      href="../styles.css"
      media="screen"
    />
  </head>
  <body>
    <a href="/index.html">Back</a>
    <a href="https://www.nature.com/articles/s41586-022-05028-x">Original</a>
    <h1>A physical wiring diagram for the human immune system</h1>
    
    <div id="readability-page-1" class="page"><div>
            <section aria-labelledby="Abs1" data-title="Abstract" lang="en"><div id="Abs1-section"><h2 id="Abs1">Abstract</h2><p>The human immune system is composed of a distributed network of cells circulating throughout the body, which must dynamically form physical associations and communicate using interactions between their cell-surface proteomes<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 1" title="Rieckmann, J. C. et al. Social network architecture of human immune cells unveiled by quantitative proteomics. Nat. Immunol. 18, 583–593 (2017)." href="https://www.nature.com/articles/s41586-022-05028-x#ref-CR1" id="ref-link-section-d169881924e465">1</a></sup>. Despite their therapeutic potential<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 2" title="Yao, S., Zhu, Y. &amp; Chen, L. Advances in targeting cell surface signalling molecules for immune modulation. Nat. Rev. Drug Discov. 12, 130–146 (2013)." href="https://www.nature.com/articles/s41586-022-05028-x#ref-CR2" id="ref-link-section-d169881924e469">2</a></sup>, our map of these surface interactions remains incomplete<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 3" title="Germain, R. N., Robey, E. A. &amp; Cahalan, M. D. A decade of imaging cellular motility and interaction dynamics in the immune system. Science 336, 1676–1681 (2012)." href="https://www.nature.com/articles/s41586-022-05028-x#ref-CR3" id="ref-link-section-d169881924e473">3</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 4" title="Grandclaudon, M. et al. A quantitative multivariate model of human dendritic cell–T helper cell communication. Cell 179, 432–447 (2019)." href="https://www.nature.com/articles/s41586-022-05028-x#ref-CR4" id="ref-link-section-d169881924e476">4</a></sup>. Here, using a high-throughput surface receptor screening method, we systematically mapped the direct protein interactions across a recombinant library that encompasses most of the surface proteins that are detectable on human leukocytes. We independently validated and determined the biophysical parameters of each novel interaction, resulting in a high-confidence and quantitative view of the receptor wiring that connects human immune cells. By integrating our interactome with expression data, we identified trends in the dynamics of immune interactions and constructed a reductionist mathematical model that predicts cellular connectivity from basic principles. We also developed an interactive multi-tissue single-cell atlas that infers immune interactions throughout the body, revealing potential functional contexts for new interactions and hubs in multicellular networks. Finally, we combined targeted protein stimulation of human leukocytes with multiplex high-content microscopy to link our receptor interactions to functional roles, in terms of both modulating immune responses and maintaining normal patterns of intercellular associations. Together, our work provides a systematic perspective on the intercellular wiring of the human immune system that extends from systems-level principles of immune cell connectivity down to mechanistic characterization of individual receptors, which could offer opportunities for therapeutic intervention.</p></div></section>
            

                
            
                <section data-title="Main"><div id="Sec1-section"><h2 id="Sec1">Main</h2><div id="Sec1-content"><p>The human immune system must maintain the same coordination and cohesion as the body’s other homeostatic organ systems despite being composed of highly migratory and circulating cell types that are distributed throughout the body. Diverse arrays of cell-surface proteins organize immune cells into interconnected cellular communities, linking cells through physical interactions that act both for signalling communication and for structural adhesion<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 5" title="Bausch-Fluck, D. et al. The in silico human surfaceome. Proc. Natl Acad. Sci. USA 115, E10988–E10997 (2018)." href="https://www.nature.com/articles/s41586-022-05028-x#ref-CR5" id="ref-link-section-d169881924e497">5</a></sup>. The immune system has been described from one perspective as carefully coordinated networks of cell types<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 6" title="Gawel, D. R. et al. A validated single-cell-based strategy to identify diagnostic and therapeutic targets in complex diseases. Genome Med. 11, 47 (2019)." href="https://www.nature.com/articles/s41586-022-05028-x#ref-CR6" id="ref-link-section-d169881924e501">6</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Bergthaler, A. &amp; Menche, J. The immune system as a social network. Nat. Immunol. 18, 481–482 (2017)." href="https://www.nature.com/articles/s41586-022-05028-x#ref-CR7" id="ref-link-section-d169881924e504">7</a></sup>, where by extension it is these physical linkages that hold the network together<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 3" title="Germain, R. N., Robey, E. A. &amp; Cahalan, M. D. A decade of imaging cellular motility and interaction dynamics in the immune system. Science 336, 1676–1681 (2012)." href="https://www.nature.com/articles/s41586-022-05028-x#ref-CR3" id="ref-link-section-d169881924e508">3</a></sup>. Consequently, immune receptors regulate virtually all stages of cellular activation and are appreciated as critical mediators of a variety of homeostatic and pathological processes, which range from tumour surveillance, to autoimmunity, to infection control. For these reasons, along with their accessibility to systemically administered medicines, immune surface proteins and their interactions are particularly attractive therapeutic targets<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 2" title="Yao, S., Zhu, Y. &amp; Chen, L. Advances in targeting cell surface signalling molecules for immune modulation. Nat. Rev. Drug Discov. 12, 130–146 (2013)." href="https://www.nature.com/articles/s41586-022-05028-x#ref-CR2" id="ref-link-section-d169881924e512">2</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 8" title="Overington, J. P., Al-Lazikani, B. &amp; Hopkins, A. L. How many drug targets are there? Nat. Rev. Drug Discov. 5, 993–996 (2006)." href="https://www.nature.com/articles/s41586-022-05028-x#ref-CR8" id="ref-link-section-d169881924e515">8</a></sup>.</p><p>Although the interaction networks that involve secreted proteins have already been systematically catalogued<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 9" title="von Hundelshausen, P. et al. Chemokine interactome mapping enables tailored intervention in acute and chronic inflammation. Sci. Transl. Med. 9, eaah6650 (2017)." href="https://www.nature.com/articles/s41586-022-05028-x#ref-CR9" id="ref-link-section-d169881924e522">9</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 10" title="Lin, H. et al. Discovery of a cytokine and its receptor by functional screening of the extracellular proteome. Science 320, 807–811 (2008)." href="https://www.nature.com/articles/s41586-022-05028-x#ref-CR10" id="ref-link-section-d169881924e525">10</a></sup>, in the immune system and more generally across existing protein interaction databases, there remains a substantial under-representation of the interactions between cell-surface proteins<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 11" title="Braun, P. et al. An experimentally derived confidence score for binary protein-protein interactions. Nat. Methods 6, 91–97 (2009)." href="https://www.nature.com/articles/s41586-022-05028-x#ref-CR11" id="ref-link-section-d169881924e529">11</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 12" title="Kotlyar, M. et al. In silico prediction of physical protein interactions and characterization of interactome orphans. Nat. Methods 12, 79–84 (2015)." href="https://www.nature.com/articles/s41586-022-05028-x#ref-CR12" id="ref-link-section-d169881924e532">12</a></sup>. Specialized methods have been developed that tackle individual challenges stemming from membrane-embedded surface proteins, such as their typically weak binding affinities<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 13" title="Özkan, E. et al. An extracellular interactome of immunoglobulin and LRR proteins reveals receptor-ligand networks. Cell 154, 228–239 (2013)." href="https://www.nature.com/articles/s41586-022-05028-x#ref-CR13" id="ref-link-section-d169881924e536">13</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 14" title="Bushell, K. M., Söllner, C., Schuster-Boeckler, B., Bateman, A. &amp; Wright, G. J. Large-scale screening for novel low-affinity extracellular protein interactions. Genome Res. 18, 622–630 (2008)." href="https://www.nature.com/articles/s41586-022-05028-x#ref-CR14" id="ref-link-section-d169881924e539">14</a></sup> and the low tractability of these proteins for many classic biochemical approaches<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 15" title="Sokolina, K. et al. Systematic protein-protein interaction mapping for clinically relevant human GPCRs. Mol. Syst. Biol. 13, 918 (2017)." href="https://www.nature.com/articles/s41586-022-05028-x#ref-CR15" id="ref-link-section-d169881924e543">15</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 16" title="Snider, J. et al. Fundamentals of protein interaction network mapping. Mol. Syst. Biol. 11, 848 (2015)." href="https://www.nature.com/articles/s41586-022-05028-x#ref-CR16" id="ref-link-section-d169881924e546">16</a></sup>. These methods, however, generally lack the throughput to systematically characterize whole cell-surface proteomes, or have only had success for specific protein families rather than the full diverse spectrum of surface protein topologies and complexes<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 17" title="Wojtowicz, W. M. et al. A human IgSF cell-surface interactome reveals a complex network of protein–protein interactions. Cell 182, 1027–1043 (2020)." href="https://www.nature.com/articles/s41586-022-05028-x#ref-CR17" id="ref-link-section-d169881924e550">17</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 18" title="Verschueren, E. et al. The immunoglobulin superfamily receptome defines cancer-relevant networks associated with clinical outcome. Cell 182, 329–344 (2020)." href="https://www.nature.com/articles/s41586-022-05028-x#ref-CR18" id="ref-link-section-d169881924e553">18</a></sup>. Thus, how complete our understanding is of extracellular immune receptor interactions has remained unknown. Moreover, many immune receptors of clinical importance have been left as ‘orphans’, with their physiological ligands undiscovered despite in some cases decades of study<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Evans, E. J. et al. The T cell surface-how well do we know it? Immunity 19, 213–223 (2003)." href="#ref-CR19" id="ref-link-section-d169881924e558">19</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Estefanía, E. et al. Human KIR2DL5 is an inhibitory receptor expressed on the surface of NK and T lymphocyte subsets. J. Immunol. 178, 4402–4410 (2007)." href="#ref-CR20" id="ref-link-section-d169881924e558_1">20</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Lines, J. L., Sempere, L. F., Broughton, T., Wang, L. &amp; Noelle, R. VISTA Is a novel broad-spectrum negative checkpoint regulator for cancer immunotherapy. Cancer Immunol. Res. 2, 510–517 (2014)." href="#ref-CR21" id="ref-link-section-d169881924e558_2">21</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 22" title="Kang, X. et al. Inhibitory leukocyte immunoglobulin-like receptors: immune checkpoint proteins and tumor sustaining factors. Cell Cycle 15, 25–40 (2015)." href="https://www.nature.com/articles/s41586-022-05028-x#ref-CR22" id="ref-link-section-d169881924e561">22</a></sup>. Without a systematic picture of the physical interactions that link immune cells, any efforts at present to generate truly systems-level views of immune function will remain patchwork at best.</p></div></div></section><section data-title="Building a surface protein interactome"><div id="Sec2-section"><h2 id="Sec2">Building a surface protein interactome</h2><div id="Sec2-content"><p>To enable a systematic survey of the surface protein interactions between immune cells at scales that approach the size of whole cell-surface proteomes, we first developed an optimized method for testing binary interactions of all possible pairings of recombinant surface proteins (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-022-05028-x#Fig1">1a</a>). Our method, the scalable arrayed multi-valent extracellular interaction screen (SAVEXIS), simultaneously addresses several key limitations of previous methods to make it possible to screen hundreds of thousands of interactions while consuming minute amounts of protein (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-022-05028-x#Fig5">1</a>). By exploiting multimerization around streptavidin, both immobilized ‘baits’ and reporter-linked ‘preys’ for screening can be produced from a single construct instead of two, and the design is attuned for detecting even low-affinity interactions across the range of structural classes that cell-surface proteins span.</p><div data-test="figure" data-container-section="figure" id="figure-1" data-title="A leukocyte receptor network by systematic protein interaction mapping."><figure><figcaption><b id="Fig1" data-test="figure-caption-text">Fig. 1: A leukocyte receptor network by systematic protein interaction mapping.</b></figcaption><div><div><a data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="https://www.nature.com/articles/s41586-022-05028-x/figures/1" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41586-022-05028-x/MediaObjects/41586_2022_5028_Fig1_HTML.png?as=webp"/><img aria-describedby="Fig1" src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41586-022-05028-x/MediaObjects/41586_2022_5028_Fig1_HTML.png" alt="figure 1" loading="lazy" width="685" height="606"/></picture></a></div><div data-test="bottom-caption" id="figure-1-desc"><p><b>a</b>, SAVEXIS enables efficient and high-throughput screening for protein binding interactions between recombinant extracellular domains. <b>b</b>, Schematic showing the diverse structural architectures of leukocyte surface proteins within the pan-leukocyte library of 630 proteins. The number of proteins from each class is noted above, and the recombinant expression strategy is illustrated below. <b>c</b>, Summarized matrix of protein–protein pairs for immune receptors with interactions either identified by screening or previously reported in the literature. The average signal intensity for a given bait–prey measurement orientation across the primary and secondary screens is indicated by the shaded intensity, and the colour indicates which interactions are novel. <b>d</b>, Screening successfully finds most previously reported interactions with minimal false positives. Receiver operating characteristic (ROC) curve for average measurements of protein–protein pairs against reference sets of expected positive and randomized negative interactions. AUC, area under the curve. <b>e</b>, Organized interaction network of immune receptor interactions. The colour indicates which interactions are novel, and the line thickness is proportional to the magnitude of evidence from the screening measurements.</p><p><a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41586-022-05028-x#MOESM6">Source data</a></p></div></div></figure></div><p>We assembled a detailed library that encompassed the full ectodomains of cell-surface proteins detectable in a previous high-resolution proteomics survey of peripheral immune cells<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 1" title="Rieckmann, J. C. et al. Social network architecture of human immune cells unveiled by quantitative proteomics. Nat. Immunol. 18, 583–593 (2017)." href="https://www.nature.com/articles/s41586-022-05028-x#ref-CR1" id="ref-link-section-d169881924e620">1</a></sup> plus all CD-numbered proteins that were compatible with our recombinant expression platform (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41586-022-05028-x#MOESM4">1</a>). The design of each expression construct was tailored to accommodate the structural class of the protein, with six different bespoke designs for functionally expressing different receptor topologies and multicomponent complexes (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-022-05028-x#Fig1">1b</a>). This library of 630 different proteins or protein complexes (such as all known integrin combinations) was expressed in human cells, purified and quality-checked (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-022-05028-x#Fig6">2a,b</a> and Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41586-022-05028-x#MOESM3">1</a>). We tested all possible protein pairings in both bait–prey orientations for every protein expressed (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-022-05028-x#Fig6">2c</a> and Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41586-022-05028-x#MOESM4">2</a>). Positive interactions identified from this comprehensive primary screen were then re-tested in a secondary screen with independent protein preparations from a separate cell strain, consisting of an all-versus-all matrix of 187 proteins (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-022-05028-x#Fig6">2d</a>) togive a final matrix of reproducible interactions (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-022-05028-x#Fig1">1c</a>). In benchmarks against a hand-curation of the published literature (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41586-022-05028-x#MOESM4">3</a>), our screen independently captured a majority of all previously reported interactions at a false positive rate below 1 in 10,000 (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-022-05028-x#Fig1">1d</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-022-05028-x#Fig7">3a,b</a>). We identified 28 new interactions that were not in our literature curation, expanding the total number of known high-confidence interactions in the human immune system by 20% (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-022-05028-x#Fig1">1e</a>). Notably, these include endogenous non-tumour ligands for the previously orphan immune checkpoint receptor VISTA, comprising the non-classical MHC molecules HLA-E and HLA-F. As these measurements constitute a highly systematic view of surface interactions for an exemplar cell system, our data also suggest answers to the general properties of surface receptor networks. In the set of immune receptors, 57% of binding pairs are unique, without either protein having another binding partner. Exclusivity is particularly common among proteins that are generally considered to have primarily signalling roles, whereas the largest interconnected group features integrins and other adhesion molecules.</p></div></div></section><section data-title="Quantifying total receptor engagement"><div id="Sec3-section"><h2 id="Sec3">Quantifying total receptor engagement</h2><div id="Sec3-content"><p>To validate these discoveries, every interaction was assessed by orthogonal approaches. First, we tested protein binding to the complementary receptor when displayed on a human cell surface following transfection with a cDNA that encodes the receptor (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-022-05028-x#Fig2">2a</a>). This was followed by two rounds of surface plasmon resonance (SPR) to characterize direct binding (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-022-05028-x#Fig2">2b</a> and Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41586-022-05028-x#MOESM3">2</a>). All of the top-ranked 28 interactions were supported by at least one additional method (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-022-05028-x#Fig7">3c</a>). We realized that we could combine the binding affinities measured in our SPR data with measurements that we methodically extracted from the literature to assemble not only a systematic physical interaction network, but also a uniquely quantitative one (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-022-05028-x#Fig2">2c</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-022-05028-x#Fig8">4</a>). We integrated this quantitative receptor interaction network with proteomics expression in leukocytes to gain insight into the patterns of binding kinetics across the immune system (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-022-05028-x#Fig2">2d</a> and Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41586-022-05028-x#MOESM4">4</a>). For example, in their contacts with antigen-presenting cells, we found that circulating T lymphocytes show a subtle preference for higher-affinity receptors when pairing with B cells compared to dendritic cells (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-022-05028-x#Fig9">5a</a>). We also established that the overall distribution of surface interactions has affinities centred in the low micromolar range, although with a long tail of higher-affinity interactions (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-022-05028-x#Fig9">5b</a>). In a test of previous theoretical predictions<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 23" title="van der Merwe, P. A. &amp; Barclay, A. N. Transient intercellular adhesion: the importance of weak protein-protein interactions. Trends Biochem. Sci. 19, 354–358 (1994)." href="https://www.nature.com/articles/s41586-022-05028-x#ref-CR23" id="ref-link-section-d169881924e701">23</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 24" title="Wright, G. J. Signal initiation in biological systems: the properties and detection of transient extracellular protein interactions. Mol. Biosyst. 5, 1405–1412 (2009)." href="https://www.nature.com/articles/s41586-022-05028-x#ref-CR24" id="ref-link-section-d169881924e704">24</a></sup> we found that higher expression levels do negatively correlate with binding strength, although only weakly (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-022-05028-x#Fig9">5c,d</a>). Of note, we also found that immune activation is accompanied by a broad transition in cellular interaction strengths. Higher-affinity interactions predominate in an inflamed state, and these are replaced with more transient interactions in a resting state, possibly to support the need for more dynamic responses when sensing potential threats (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-022-05028-x#Fig2">2e</a>). We confirmed this ‘affinity switch’ in human leukocyte receptor preferences using an independent transcriptomic dataset (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-022-05028-x#Fig10">6</a>).</p><div data-test="figure" data-container-section="figure" id="figure-2" data-title="Validating and assembling a quantitative immune interactome."><figure><figcaption><b id="Fig2" data-test="figure-caption-text">Fig. 2: Validating and assembling a quantitative immune interactome.</b></figcaption><div><div><a data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="https://www.nature.com/articles/s41586-022-05028-x/figures/2" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41586-022-05028-x/MediaObjects/41586_2022_5028_Fig2_HTML.png?as=webp"/><img aria-describedby="Fig2" src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41586-022-05028-x/MediaObjects/41586_2022_5028_Fig2_HTML.png" alt="figure 2" loading="lazy" width="685" height="630"/></picture></a></div><div data-test="bottom-caption" id="figure-2-desc"><p><b>a</b>, Novel receptor interactions are detectable on the surfaces of live human cells. For six examples that encompass different architectural classes, flow cytometry traces are shown for the binding of fluorescent-conjugated protein to HEK293 cells overexpressing its identified counter-receptor (blue) or control cells (red). <b>b</b>, SPR substantiates and quantifies the binding of novel leukocyte receptor–ligand pairs. For the same six example protein pairs, sensorgram data (black) are shown with Langmuir model fitting curves overlaid (red) for all interactions for which a robust fit could be calculated. Ig-SF, immunoglobulin superfamily; LRR-SF, leucine rich repeat superfamily. <b>c</b>, The quantitative interactome of immune cell-surface proteins. Proteins are shown as circular charts indicating the proportion of expression in each leukocyte population. Binding affinity between proteins is indicated by the size and intensity of red edges (expressed in terms of the binding dissociation constant (<i>K</i><sub>D</sub>), where smaller values reflect stronger binding). Abbreviations for cell type names are defined in Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41586-022-05028-x#MOESM4">5</a>. <b>d</b>, Immune cell subsets use related but varying distributions of binding affinities when communicating with other cell types. For each pairing of two cells, a histogram of inferred interactions is shown alongside a colour shade that indicates the average affinity. <b>e</b>, Inflammatory activation broadly reconfigures receptors towards those with less-transient binding kinetics. After differential expression testing of surface proteins between activated and stimulated leukocytes (<i>n</i> = 4 samples per condition), the binding affinities of interactions involving downregulated (downreg) receptors are compared to the binding affinities of upregulated (upreg) receptors. Data are shown as Tukey box plots with Holm-corrected <i>P</i> values from a two-sided Welch’s <i>t</i>-test. <b>f</b>, Intercellular connectivity can be mathematically predicted by integrating protein expression, binding affinity and cell parameters using physics-based equations. A detailed description of the model can be found in the <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41586-022-05028-x#MOESM1">Supplementary Equations</a>. <b>g</b>, Model predictions for baseline rates of immune cell association agree with published data measuring in vitro immune cell association. Each data point has two colours that correspond to the two physically interacting cell types. Shading depicts the 95% compatibility interval for the least-squares linear regression fit.</p><p><a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41586-022-05028-x#MOESM6">Source data</a></p></div></div></figure></div><p>Our quantitative wiring diagram, if truly systematic to the point of approaching completeness, should make it possible to derive a reductionist model that explains how circulating immune cells associate with each other solely from receptor-binding mechanisms and physics-based formulas. We built a coarse yet principled mathematical model that integrates quantitative proteomics expression, binding kinetics and published cell parameters to summarize the contributions of individual protein interactions to a given cellular interaction (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-022-05028-x#Fig2">2f</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41586-022-05028-x#MOESM1">Supplementary Equations</a>). Using equations based on the law of mass action, the model then computes how the overall probability of binding between two cell types emerges from the distinct spectrum of cell-surface receptors that connect them (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-022-05028-x#Fig11">7</a>). Although greatly simplistic, our model could still infer the relative frequencies at which human immune cells physically interact with sufficient precision to be consistent with published empirical measurements<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 25" title="Vladimer, G. I. et al. Global survey of the immunomodulatory potential of common drugs. Nat. Chem. Biol. 13, 681–690 (2017)." href="https://www.nature.com/articles/s41586-022-05028-x#ref-CR25" id="ref-link-section-d169881924e795">25</a></sup> (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-022-05028-x#Fig2">2g</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-022-05028-x#Fig11">7d</a>).</p></div></div></section><section data-title="Integrating multicellular networks"><div id="Sec4-section"><h2 id="Sec4">Integrating multicellular networks</h2><div id="Sec4-content"><p>Although our library used circulating immune cells as a source, the immune system traverses a broad repertoire of organs, each of which may be key to understanding the biological role of an interaction. We therefore sought to contextualize the interactions of our network by creating an interactive atlas that charts where these receptor and ligand pairs have been detected across single-cell expression datasets of human tissues (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-022-05028-x#Fig3">3a</a>). Our integrated atlas is available at <a href="https://www.sanger.ac.uk/tool/immune-interaction/immune-interaction">https://www.sanger.ac.uk/tool/immune-interaction/immune-interaction</a> and allows multiple kinds of analysis, which range from summarizing the overall cellular connectivity of different tissue immune populations to inferring which cell–cell pairs are capable of carrying out a particular receptor interaction (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-022-05028-x#Fig3">3b</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-022-05028-x#Fig12">8a</a>).</p><div data-test="figure" data-container-section="figure" id="figure-3" data-title="An interactive atlas of immune cell connections across the human body."><figure><figcaption><b id="Fig3" data-test="figure-caption-text">Fig. 3: An interactive atlas of immune cell connections across the human body.</b></figcaption><div><div><a data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="https://www.nature.com/articles/s41586-022-05028-x/figures/3" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41586-022-05028-x/MediaObjects/41586_2022_5028_Fig3_HTML.png?as=webp"/><img aria-describedby="Fig3" src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41586-022-05028-x/MediaObjects/41586_2022_5028_Fig3_HTML.png" alt="figure 3" loading="lazy" width="685" height="672"/></picture></a></div><div data-test="bottom-caption" id="figure-3-desc"><p><b>a</b>, Systematic integration of single-cell datasets to map cellular connectivity across tissues with substantial immune populations. Cell types are positioned around each circle, with each position along it marking a cell-surface protein. Linkages formed by physically interacting surface proteins between cell types are marked by curved lines, coloured by interaction identity. Full abbreviations are listed in Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41586-022-05028-x#MOESM4">5</a>. <b>b</b>, Functionalities available through our interactive atlas of physical immune interactions (<a href="https://www.sanger.ac.uk/tool/immune-interaction/immune-interaction">https://www.sanger.ac.uk/tool/immune-interaction/immune-interaction</a>). <b>c</b>, Myeloid cells act as interaction hubs in immune tissues. Eigenvector centrality metrics of myeloid cells compared to all other populations after converting the total interaction count for all cell–cell pairs into a weighted undirected graph. Data are shown as Tukey box plots with Benjamini–Hochberg <i>P</i> values calculated from a two-sided Welch’s <i>t</i>-test. <b>d</b>, Spatial transcriptomics of a human lymph node confirms that our identified interaction partners are physically colocalized in situ. An example data point of a tissue section analysed for the JAG1 + VASN interaction is shown. The percentage of measured spots in which the expression of one protein of an interacting pair is spatially connected to the other protein of the pair is compared for previously reported interactions (green), novel interactions (blue) and a negative control of the same proteins with interaction links randomly permuted (yellow) (<i>n</i> = 100). Data are shown as Tukey box plots with <i>P</i> values calculated from a Tukey&#39;s honest significance test. <b>e</b>, Single-molecule RNA hybridization on human lymph nodes defines regions in which newly identified interaction partners are expressed in spatially bordering cells. A single lymphoid follicle enriched in <i>CD45</i><sup>+</sup> leukocytes is magnified (left), showing the zonation of <i>JAG1</i>- and <i>VASN</i>-expressing cells into the corona and the germinal centre, respectively (middle). An inset (right) highlights a region of bordering cells expressing each marker. Scale bars, 200 μm (left); 100 μm (middle); 50 μm (right).</p><p><a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41586-022-05028-x#MOESM6">Source data</a></p></div></div></figure></div><p>Through this systematic multi-organ atlas, we could determine whether immune receptor interactions proceed through shared structures or are distinct between tissues. We found a recurring motif in which myeloid-lineage cells act as hubs across several cellular interaction networks. Quantified across multiple primary and secondary lymphoid tissues, myeloid cells have consistently higher network centrality scores (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-022-05028-x#Fig3">3c</a>), despite expressing similar numbers of surface ligands to other cell types (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-022-05028-x#Fig12">8b</a>). This suggests that resident myeloid cells may adapt their receptor repertoire to serve as central integrators of local interactions in their tissue niche. Considering the breadth of pathological conditions that exhibit immune dysregulation, we reasoned that these same integrated approaches could also inform how the physical interactions that we catalogued between immune cells may change in disease. We incorporated paired diseased and reference samples where available in our atlas, which can be used to generate hypotheses on which interactions (cellular or molecular) may differentially appear in diseased states. For example, we could see phagocyte populations shifting a large fraction of their total cellular contacts within the tumour microenvironment of kidney samples (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-022-05028-x#Fig12">8c</a>), including upregulation of APLP2 and APP ligands that we characterized (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41586-022-05028-x#MOESM4">6</a>). Our novel receptor interactions could also be integrated with known signalling pathways to infer cellular communication pathways that appear differentially active in diseased states<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 26" title="Browaeys, R., Saelens, W. &amp; Saeys, Y. NicheNet: modeling intercellular communication by linking ligands to target genes. Nat. Methods 17, 159–162 (2020)." href="https://www.nature.com/articles/s41586-022-05028-x#ref-CR26" id="ref-link-section-d169881924e917">26</a></sup>, which implicated the newly discovered JAML interaction in potentially regulating anti-tumour immunity (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-022-05028-x#Fig12">8d</a>). To investigate whether the interactions that we discovered in our network manifest themselves under physiological settings in the human body, we finally examined the spatial colocalization of receptors and ligands in a lymph-node spatial transcriptomics dataset. Although this reflects only a snapshot of these dynamic cellular populations, both our set of new interactions and the previously reported interactions were significantly more likely to be spatially proximal than randomized networks (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-022-05028-x#Fig3">3d</a>). In follow-up experiments, we could verify that novel interacting pairs we identified, such as JAG1–VASN, are distributed in bordering regions of the immune-cell-rich compartments of human lymph nodes (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-022-05028-x#Fig3">3e</a>). This confirms that these interactions have the potential to occur between immune cells in vivo (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-022-05028-x#Fig13">9a</a>).</p></div></div></section><section data-title="Assigning functions to binding targets"><div id="Sec5-section"><h2 id="Sec5">Assigning functions to binding targets</h2><div id="Sec5-content"><p>We finally asked whether the reagents and receptor interactions that we characterized could have a potential clinical use in modulating the immune system. Soluble recombinant proteins with activity to bind specific immune cell receptors have shown promise as immunotherapies<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 27" title="Fitzpatrick, E. A., Wang, J. &amp; Strome, S. E. Engineering of Fc multimers as a protein therapy for autoimmune disease. Front. Immunol. 11, 496 (2020)." href="https://www.nature.com/articles/s41586-022-05028-x#ref-CR27" id="ref-link-section-d169881924e942">27</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 28" title="Gu, D., Ao, X., Yang, Y., Chen, Z. &amp; Xu, X. Soluble immune checkpoints in cancer: production, function and biological significance. J. Immunother. Cancer 6, 132 (2018)." href="https://www.nature.com/articles/s41586-022-05028-x#ref-CR28" id="ref-link-section-d169881924e945">28</a></sup>. As a first step towards potential therapeutic applications and functional classifications on modes of action, we performed high-throughput cellular phenotyping assays on isolated human immune cells that were treated with recombinant forms of the proteins identified in our molecular wiring map. To reflect the wide array of interactions and leukocyte subtypes included in our investigation, we adapted an approach for measuring leukocyte activation and cellular interaction phenotypes by high-content microscopy<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 29" title="Severin, Y. et al. Multiplexed high-throughput immune cell imaging reveals molecular health-associated phenotypes. Preprint at bioRxiv 
                https://doi.org/10.1101/2021.12.03.471105
                
               (2021)." href="https://www.nature.com/articles/s41586-022-05028-x#ref-CR29" id="ref-link-section-d169881924e949">29</a></sup>, which captures all major cell populations in a single multiplex experiment (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-022-05028-x#Fig13">9c,d</a>). We incubated pools of human leukocytes in the presence of purified proteins from our recombinant surface receptor library, imaging at 4 and 24 h to measure changes in both cell–cell interactions and the proportions of activated cells elicited by each protein (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-022-05028-x#Fig4">4a</a> and Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41586-022-05028-x#MOESM3">3</a>). Because the function of interactions may not be revealed unless in the proper context, we measured resting leukocytes and those stimulated by low levels of lipopolysaccharide (LPS). Proteins from our set of novel interactors elicited diverse responses, including generalized T cell activation by the formerly orphan<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 30" title="Benschop, R., Wei, T. &amp; Na, S. Tumor necrosis factor receptor superfamily member 21: TNFR-related death receptor-6, DR6. Adv. Exp. Med. Biol. 647, 186–194 (2009)." href="https://www.nature.com/articles/s41586-022-05028-x#ref-CR30" id="ref-link-section-d169881924e963">30</a></sup> TNFRSF21, as well as the adhesion proteins CHL1 and CD320 (along with CD58 as previously described<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 31" title="Liu, L. L. et al. Critical role of CD2 co-stimulation in adaptive natural killer cell responses revealed in NKG2C-deficient humans. Cell Rep. 15, 1088–1099 (2016)." href="https://www.nature.com/articles/s41586-022-05028-x#ref-CR31" id="ref-link-section-d169881924e967">31</a></sup>) facilitating natural killer (NK) cell activation (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-022-05028-x#Fig4">4b</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-022-05028-x#Fig14">10a</a>). These phenotypes were generally consistent with the cell-type expression of each protein’s newly identified receptor partners, suggesting direct effects as opposed to indirect mechanisms mediated by an intermediate leukocyte type within the mixed pools of cells (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-022-05028-x#Fig14">10b</a>).</p><div data-test="figure" data-container-section="figure" id="figure-4" data-title="Multiplex leukocyte assays identify functional pathways for receptor proteins."><figure><figcaption><b id="Fig4" data-test="figure-caption-text">Fig. 4: Multiplex leukocyte assays identify functional pathways for receptor proteins.</b></figcaption><div><div><a data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="https://www.nature.com/articles/s41586-022-05028-x/figures/4" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41586-022-05028-x/MediaObjects/41586_2022_5028_Fig4_HTML.png?as=webp"/><img aria-describedby="Fig4" src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41586-022-05028-x/MediaObjects/41586_2022_5028_Fig4_HTML.png" alt="figure 4" loading="lazy" width="685" height="491"/></picture></a></div><div data-test="bottom-caption" id="figure-4-desc"><p><b>a</b>, High-content microscopy set-up for perturbing human peripheral blood mononuclear cells (PBMCs) with recombinant proteins and measuring changes in cellular activation and connectivity. Scale bar, 30 µm. <b>b</b>, Proteins with identified receptor interactions elicit responses on lymphocyte action. Polarization of lymphocyte populations in resting and weakly activating background conditions (<i>y</i> axis) after addition of soluble protein extracellular domains (<i>x</i> axis). Stimulation (red) or inhibition (blue) relative to control is shown of a cell polarization marker of lymphocyte activation. Phenotypes that have <i>P</i> values below the adjusted significance threshold are outlined in bold. <i>n</i> = 10 samples. <b>c</b>, Interacting cellular communities can be extracted from high-content imaging data. Representative microscopy fields (left) and computed physical cell contacts (right, white lines) are depicted for leukocytes perturbed with recombinant SEMA4D and SIRPA as examples. Scale bar, 30 µm. <b>d</b>, Rewiring of cellular interactions by perturbing receptor pathways. Measured changes in cell–cell interactions (<i>x</i> axis) induced by recombinant proteins (panels) across measurement time points and background conditions (<i>y</i> axis). The same colour scale as in <b>c</b> is used to identify cell pairs along the <i>x</i> axis. <i>n</i> = 10 samples. <b>e</b>, Observed interaction changes conform to mathematical model predictions. Average magnitudes of cell–cell interaction changes (<i>y</i> axis) after the addition of recombinant proteins are compared for cell pairs predicted by the model to be likely to change after perturbing that surface protein (‘true’) and those predicted not to change (‘false’). Each panel considers a different recombinant protein added in the experiment and the corresponding model predictions for that same protein. The two colours for each data point depict the identity of the cell pair according to the colour scale in <b>c</b>. <i>n</i> = 10 samples for the experimental data.</p><p><a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41586-022-05028-x#MOESM6">Source data</a></p></div></div></figure></div><p>The perturbation experiments also provided high-dimensional data on changes in cellular connectivity triggered by infusing soluble receptor-binding proteins (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-022-05028-x#Fig4">4c</a>). The landscape of immune interactions across these conditions provides a rare view into the functional roles of previously described and novel immune surface proteins. This includes large-scale shifts away from cellular contacts between T cell populations from CD58 along with increased monocyte contacts from SIRPA (consistent with our previous biological understanding<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 32" title="Dustin, M. L., Sanders, M. E., Shaw, S. &amp; Springer, T. A. Purified lymphocyte function-associated antigen 3 binds to CD2 and mediates T lymphocyte adhesion. J. Exp. Med. 165, 677–692 (1987)." href="https://www.nature.com/articles/s41586-022-05028-x#ref-CR32" id="ref-link-section-d169881924e1062">32</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 33" title="Hayes, B. H. et al. Macrophages show higher levels of engulfment after disruption of cis interactions between CD47 and the checkpoint receptor SIRPα. J. Cell Sci. 133, jcs237800 (2020)." href="https://www.nature.com/articles/s41586-022-05028-x#ref-CR33" id="ref-link-section-d169881924e1065">33</a></sup>); an inhibition of interactions between monocyte–lymphocyte pairs from SLITRK4; and several ligands that we implicated in adhesion such as NRCAM, CD93 and CDH2 converging on facilitating early B cell interactions (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-022-05028-x#Fig4">4d</a>). In line with results reported previously for changes in the interactions of leukocytes after drug administration<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 25" title="Vladimer, G. I. et al. Global survey of the immunomodulatory potential of common drugs. Nat. Chem. Biol. 13, 681–690 (2017)." href="https://www.nature.com/articles/s41586-022-05028-x#ref-CR25" id="ref-link-section-d169881924e1072">25</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 29" title="Severin, Y. et al. Multiplexed high-throughput immune cell imaging reveals molecular health-associated phenotypes. Preprint at bioRxiv 
                https://doi.org/10.1101/2021.12.03.471105
                
               (2021)." href="https://www.nature.com/articles/s41586-022-05028-x#ref-CR29" id="ref-link-section-d169881924e1075">29</a></sup>, infusing our recombinant proteins triggered changes that generally fit within a distinct set of modules of action (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-022-05028-x#Fig14">10c</a>). Our in silico model of immune connectivity allowed us to rationalize which molecular changes would have been likely to lead to the observed phenotypes. We compared these experimentally measured perturbations to mathematical predictions as to which cell–cell pairs would have the greatest perturbations. Although as before there were some gaps in the accuracy of our model for particular conditions, nevertheless, for most stimuli we observed significantly greater magnitudes of connectivity perturbations in our experimental data for cell pairs that the model predicted would be most perturbed (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-022-05028-x#Fig4">4e</a>).</p></div></div></section><section data-title="Discussion"><div id="Sec6-section"><h2 id="Sec6">Discussion</h2><div id="Sec6-content"><p>The immune system is distinctive for being a distributed system. It is not fixed to a single localized organ in the body, but rather is made up of numerous specialized cell types that must adaptably organize their intercellular connections to respond to pathogens and other threats wherever they may appear. We provide a systematic and quantitative view of the cell-surface proteins that enable immune cells to dynamically wire their interactions. The receptor interactions that we report in our network each merit further individualized study to characterize their full roles in health and disease. Of particular note are our discovery of HLA-E and HLA-F (but probably not HLA-G) as endogenous non-tumour ligands for the immune checkpoint receptor VISTA; the ability of vasorin to act as a receptor for Jagged ligands; and immunoglobulin family receptors binding members of the amyloid precursor protein family. Notably, members of this same family of non-classical MHC class I molecules that includes HLA-E and HLA-F have previously been identified as key ligands for maintaining innate immune quiescence<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 34" title="Braud, V. M. et al. HLA-E binds to natural killer cell receptors CD94/NKG2A, B and C. Nature 391, 795–799 (1998)." href="https://www.nature.com/articles/s41586-022-05028-x#ref-CR34" id="ref-link-section-d169881924e1095">34</a></sup>, which our findings extend to raise the possibility that they may similarly act as regulators of adaptive immune quiescence through VISTA<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 35" title="ElTanbouly, M. A. et al. VISTA is a checkpoint regulator for naïve T cell quiescence and peripheral tolerance. Science 367, eaay0524 (2020)." href="https://www.nature.com/articles/s41586-022-05028-x#ref-CR35" id="ref-link-section-d169881924e1099">35</a></sup>. Our functional screening on blood immune cells further points to pathways worth greater consideration; for example, a role of SLITRK4 in lymphocyte responses. While we were preparing this study, independent groups have provided supporting evidence for several of the interactions that we characterized here<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 17" title="Wojtowicz, W. M. et al. A human IgSF cell-surface interactome reveals a complex network of protein–protein interactions. Cell 182, 1027–1043 (2020)." href="https://www.nature.com/articles/s41586-022-05028-x#ref-CR17" id="ref-link-section-d169881924e1103">17</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 18" title="Verschueren, E. et al. The immunoglobulin superfamily receptome defines cancer-relevant networks associated with clinical outcome. Cell 182, 329–344 (2020)." href="https://www.nature.com/articles/s41586-022-05028-x#ref-CR18" id="ref-link-section-d169881924e1106">18</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 36" title="Husain, B. et al. A platform for extracellular interactome discovery identifies novel functional binding partners for the immune receptors B7-H3/CD276 and PVR/CD155. Mol. Cell. Proteomics 18, 2310–2323 (2019)." href="https://www.nature.com/articles/s41586-022-05028-x#ref-CR36" id="ref-link-section-d169881924e1109">36</a></sup>, including PVR as the ligand for the formerly orphan KIR2DL5A and CD146 as an adhesive ligand for CNTN1.</p><p>Because our physical wiring diagram encapsulates the diversity of surface protein architectures found across all major subsets of leukocytes, it can be integrated with publicly available expression data both qualitatively and quantitatively. The physical interaction landscape that we provide in our single-cell expression atlas offers an interactive platform for deriving insights out of our systematic receptor network. Although other studies have provided useful views of interactions that are differentially regulated in particular cell types<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 37" title="Vento-Tormo, R. et al. Single-cell reconstruction of the early maternal-fetal interface in humans. Nature 563, 347–353 (2018)." href="https://www.nature.com/articles/s41586-022-05028-x#ref-CR37" id="ref-link-section-d169881924e1116">37</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 38" title="Smillie, C. S. et al. Intra- and inter-cellular rewiring of the human colon during ulcerative colitis. Cell 178, 714–730 (2019)." href="https://www.nature.com/articles/s41586-022-05028-x#ref-CR38" id="ref-link-section-d169881924e1119">38</a></sup>, we provide a systematized catalogue of all biological contexts in which an interaction is inferred to be possible, including complementary views directedat particular cell subsets or receptor proteins. This provides material for generating hypotheses about multicellular immune circuits. These same integrated datasets can resolve broader questions in the field. For example, we have shown that immune cell activation is accompanied by broad shifts in the affinity profiles of cell-surface interactions, with rapid kinetics predominating during the early stages of immune responses (matching previous conceptual models for optimal antigen sampling<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 39" title="Sinai, P. et al. T/B-cell interactions are more transient in response to weak stimuli in SLE-prone mice. Eur. J. Immunol. 44, 3522–3531 (2014)." href="https://www.nature.com/articles/s41586-022-05028-x#ref-CR39" id="ref-link-section-d169881924e1123">39</a></sup> and propagating signals to other cells<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 40" title="Hens, C., Harush, U., Haber, S., Cohen, R. &amp; Barzel, B. Spatiotemporal signal propagation in complex networks. Nat. Phys. 15, 403–412 (2019)." href="https://www.nature.com/articles/s41586-022-05028-x#ref-CR40" id="ref-link-section-d169881924e1127">40</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 41" title="Kumar, R., Clermont, G., Vodovotz, Y. &amp; Chow, C. C. The dynamics of acute inflammation. J. Theor. Biol. 230, 145–155 (2004)." href="https://www.nature.com/articles/s41586-022-05028-x#ref-CR41" id="ref-link-section-d169881924e1130">41</a></sup>), which switchto higher-affinity contacts in an inflamed state to match the changing demands of a forceful inflammatory response. This raises parallels with findings of disease-associated variants that modulate the balance of adhesive receptor affinity<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Semmrich, M. et al. Importance of integrin LFA-1 deactivation for the generation of immune responses. J. Exp. Med. 201, 1987–1998 (2005)." href="#ref-CR42" id="ref-link-section-d169881924e1134">42</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="MacPherson, M., Lek, H. S., Prescott, A. &amp; Fagerholm, S. C. A systemic lupus erythematosus-associated R77H substitution in the CD11b chain of the Mac-1 integrin compromises leukocyte adhesion and phagocytosis. J. Biol. Chem. 286, 17303–17310 (2011)." href="#ref-CR43" id="ref-link-section-d169881924e1134_1">43</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 44" title="Faridi, M. H. et al. CD11b activation suppresses TLR-dependent inflammation and autoimmunity in systemic lupus erythematosus. J. Clin. Invest. 127, 1271–1283 (2017)." href="https://www.nature.com/articles/s41586-022-05028-x#ref-CR44" id="ref-link-section-d169881924e1137">44</a></sup>. We could also show that the previously theorized link between receptor affinity and abundance—in which low-affinity receptors are compensated for by their high abundance<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 24" title="Wright, G. J. Signal initiation in biological systems: the properties and detection of transient extracellular protein interactions. Mol. Biosyst. 5, 1405–1412 (2009)." href="https://www.nature.com/articles/s41586-022-05028-x#ref-CR24" id="ref-link-section-d169881924e1141">24</a></sup>—may indeed exist, but appears not to be sufficient to account for why certain receptors have evolved the binding kinetic rates we observe.</p><p>Out of this integrative approach, we could construct a proof-of-concept mathematical model that predicts the behaviour of collections of leukocytes from first principles. Although considerable study has been devoted to particular specialized cell-to-cell contacts such as the immunological synapse<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 45" title="Dustin, M. L., Chakraborty, A. K. &amp; Shaw, A. S. Understanding the structure and function of the immunological synapse. Cold Spring Harb. Perspect. Biol. 2, a002311 (2010)." href="https://www.nature.com/articles/s41586-022-05028-x#ref-CR45" id="ref-link-section-d169881924e1148">45</a></sup>, the overall connectivity of immune cells and their dynamic approaches and disengagements has been neglected. Our core model opens up considerable scope for dissecting in a principled way the mechanisms by which immune cells directly associate. In addition, the discrepancies between the model compared to experiments offer opportunities for refining our mechanistic understanding of cell–cell interactions by comparing how elaborations to this core model further improve prediction accuracy.</p><p>More broadly, the integrated approaches that we pioneered here for disentangling the immune system provide a framework for future systematic investigations. Using our high-throughput biochemical method for interaction screening (SAVEXIS) and the strategies that we describe here to characterize interactions by combinations of multiplex cellular assays and genomics datasets, a range of other cellular communities in the human body could similarly be quantitatively mapped. To our knowledge, our study is among the first to systematically map and model how the collective actions of individual receptor molecules through physical laws could explain and predict cellular connectivity on a scale as large as the circulating immune system. Our analysis and the methods that we developed provide a template for future studies looking at physical cell wiring networks in detail. From these combined approaches, we may finally begin to disentangle cellular circuits in immunity and beyond, bridging from individual protein molecules to multicellular behaviour.</p></div></div></section><section data-title="Methods"><div id="Sec7-section"><h2 id="Sec7">Methods</h2><div id="Sec7-content"><h3 id="Sec8">Culture and transfection of HEK293 cells</h3><p>Human embryonic kidney 293 (HEK293) cells were maintained in suspension culture with Freestyle medium (Gibco 12338018) supplemented by 1% heat-inactivated fetal bovine serum (Sigma F2442) at 37 °C, 5% carbon dioxide and 70% humidity as previously described<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 46" title="Shilts, J., Crozier, T. W. M., Greenwood, E. J. D., Lehner, P. J. &amp; Wright, G. J. No evidence for basigin/CD147 as a direct SARS-CoV-2 spike binding receptor. Sci. Rep. 11, 413 (2021)." href="https://www.nature.com/articles/s41586-022-05028-x#ref-CR46" id="ref-link-section-d169881924e1167">46</a></sup>. For all transient transfections, cells were seeded 24 h before transfection at a density of 2.5  × 10<sup>5</sup> cells per ml, then transfected with 0.5 µg DNA per ml cells as previously described<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 46" title="Shilts, J., Crozier, T. W. M., Greenwood, E. J. D., Lehner, P. J. &amp; Wright, G. J. No evidence for basigin/CD147 as a direct SARS-CoV-2 spike binding receptor. Sci. Rep. 11, 413 (2021)." href="https://www.nature.com/articles/s41586-022-05028-x#ref-CR46" id="ref-link-section-d169881924e1173">46</a></sup>. Protein expression was done in vented conical flasks (Corning) ranging from 30 ml cells for the primary screen to 100 ml cells for other applications. Transfections for cell-binding assays were typically done in a volume of 1 ml cells in 96-well deep plates (Corning 3960). Where expressed proteins were to be enzymatically biotinylated, the culture medium was supplemented with <span>d</span>-biotin (Sigma B4501) to 100 µM and a plasmid encoding secreted BirA biotin ligase was co-transfected at 30 ng per ml cells<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 47" title="Parrott, M. B. &amp; Barry, M. A. Metabolic biotinylation of secreted and cell surface proteins from mammalian cells. Biochem. Biophys. Res. Commun. 281, 993–1000 (2001)." href="https://www.nature.com/articles/s41586-022-05028-x#ref-CR47" id="ref-link-section-d169881924e1180">47</a></sup>. Transfections for protein complexes such as integrins were done using equimolar ratios of plasmids encoding both chains, except for HLA-related complexes for which only 10% of the total plasmid encoded β<sub>2</sub> microglobulin. For most experiments, the EBNA1-containing HEK293-E cell line was used, except for proteins produced for the secondary binding screen, which were expressed in the serum-free modified HEK293-6E line<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 48" title="Loignon, M. et al. Stable high volumetric production of glycosylated human recombinant IFNα2b in HEK293 cells. BMC Biotechnol. 8, 65 (2008)." href="https://www.nature.com/articles/s41586-022-05028-x#ref-CR48" id="ref-link-section-d169881924e1187">48</a></sup>. Both cell lines were provided by Y. Durocher. Cell lines were regularly tested for mycoplasma (Surrey Diagnostics) and found to be negative.</p><h3 id="Sec9">Purification of recombinant proteins</h3><p>After transfected cells were incubated for times ranging from 90 to 120 h, they were centrifuged at 2,000<i>g</i> for 20 min. Supernatants were filtered through 0.22-µm filters and purified by nickel-ion affinity chromatography, with the exact procedure differing slightly by the intended downstream experiment. Proteins for high-throughput interaction screening were purified using His MultiTrap plates (GE Healthcare 28-4009-89) via a previously described 96-position pneumatic press<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 49" title="Sun, Y., Gallagher-Jones, M., Barker, C. &amp; Wright, G. J. A benchmarked protein microarray-based platform for the identification of novel low-affinity extracellular protein interactions. Anal. Biochem. 424, 45–53 (2012)." href="https://www.nature.com/articles/s41586-022-05028-x#ref-CR49" id="ref-link-section-d169881924e1202">49</a></sup>. Each supernatant was supplemented to a final imidazole concentration of 16 mM and an NaCl concentration of 250 mM before purification, and the plates were prepared by rinsing with 500 µl pure water and washing twice with 500 µl 20 mM imidazole phosphate buffer following the manufacturer’s instructions. Once all samples had loaded, plates were washed again twice with 500 µl 20 mM imidazole phosphate buffer and eluted with 200 µl 200 mM imidazole phosphate buffer. Proteins for the secondary interaction screen and SPR were purified using an ÄKTA Pure automated chromatography instrument (GE Healthcare) loaded with 1 ml HisTrap HP columns (GE Healthcare) as previously described<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 50" title="Müller-Sienerth, N. et al. A panel of recombinant proteins from human-infective Plasmodium species for serological surveillance. Malar. J. 19, 31 (2020)." href="https://www.nature.com/articles/s41586-022-05028-x#ref-CR50" id="ref-link-section-d169881924e1206">50</a></sup>. Proteins for cell-based experiments were manually purified with HisPur Ni-NTA resin (Thermo Fisher Scientific 88221). Resins were pre-washed twice in 20 mM imidazole phosphate buffer, then resuspended in supernatant supplemented with 1 mM imidazole. The slurry was incubated rotating overnight at 4 °C, before two further washes in 20 mM imidazole phosphate buffer and finally eluted in 200 mM imidazole phosphate buffer. After initial purifications, proteins intended for use in immune cell assays were dialysed against phosphate-buffered saline overnight at 4 °C using a 12–14-kDa molecular mass cut-off D-tube dialyser (Millipore 71505). Proteins intended for use as SPR analytes were further resolved by gel filtration with an ÄKTA Express machine on a Superdex 200 Increase 10/300 GL size-exclusion column (GE Healthcare) to remove any aggregated protein that may interfere with kinetic measurements. To avoid precipitation on the column, analyte samples were pre-dialysed in HBS-EP buffer (GE Healthcare BR100669). All proteins were stored at 4 °C until use.</p><h3 id="Sec10">Protein gel electrophoresis</h3><p>Samples were denatured in lithium dodecyl sulfate (NuPAGE NP0007) and dithiothreitol (NuPAGE NP0004) to a total volume of 10 µl before heating at 70 °C for 10 min. Samples were resolved on Bis-Tris 4–12% polyacrylamide gradient SDS–PAGE gels (NuPAGE NP0329) at 200 V for 50 min in MOPS buffer (NuPAGE NP0001). As a marker, 4 µl of pre-stained SeeBlue Plus2 protein standard (Invitrogen LC5925) was added to each gel. For Coomassie staining of total protein, gels were briefly rinsed in water then incubated in brilliant blue G-250 dye (Abcam ab119211) over 24–36 h. Before imaging under visible light, gels were briefly rinsed twice in water. For immunoblots, proteins were transferred to a methanol-activated PVDF membrane (GE Healthcare 10600029) in transfer buffer (NuPAGE NP0006) over 60 min at 300 mA. Blots were briefly washed in HEPES buffered saline (HBS) with 0.1% (v/v) Tween-20 (Sigma P2287), blocked with 2% (m/v) bovine serum albumin (BSA; Sigma A9647), stained with 1:5,000 anti-His tag C terminus antibody conjugated to HRP (Invitrogen R931-25, clone 3D5) for 16 h at 4 °C, washed an additional three times, and exposed to chemiluminescent substrate (Thermo Fisher Scientific PI34577). Images were developed on photographic film (GE Healthcare 28906835) for 45–90 s. To automate our comparisons of the molecular masses observed by electrophoresis to the computationally expected masses, we made a custom Python script that translates each expression construct using the Biopython library and identifies post-translational processing sites through automated queries to Uniprot.</p><h3 id="Sec11">Bradford concentration normalization</h3><p>Standard curves of pure BSA (Pierce 23209) spanning from 1,200 to 7 ng µl<sup>−1</sup> were prepared in clear flat-bottom 96-well plates (Thermo Fisher Scientific 11349163). In general, 20 µl of each standard curve and purified protein sample was added to wells of the plate. All wells had 250 µl of Bradford reagent (Pierce 23236) added and were gently agitated for 60 s, then incubated for 30 min before measuring the absorbance at 595 nm with a Tecan Spark plate reader. All raw absorbance signals had the average background of buffer-only controls subtracted before further processing. The standard curve data were fit with fifth-order polynomials with the intercept fixed at (0,0). The curves were manually inspected for their fits and residuals before using the polynomial equation to calculate the concentration of each sample. These values were converted to molar concentrations by using the molecular masses calculated through the automated Python script, except excluding the added mass of glycan modifications.</p><h3 id="Sec12">ELISA tetramerization assays</h3><p>Both as a check on Bradford-based molarity calculations and for determining how to form cell-staining tetramers, biotinylated recombinant proteins were quantified by a competitive ELISA. Dilution series of the target proteins were made in 96-well plates, then incubated with a known constant quantity of a conjugated avidin (SAV–HRP for binding assay calculations, SAV–PE for cell-binding cytometry reagents and neutravidin for immune cell perturbation assays). After at least 45 min at 20 °C, these pre-incubated samples were transferred to a BSA-blocked streptavidin-coated 96-well plate (Greiner 655990). After another 45 min or longer incubation, plates were washed three times with HBS and 0.1% (v/v) Tween-20. Primary antibody, typically 1 µg ml<sup>−1</sup> of an OX68 monoclonal against the protein epitope tags<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 51" title="Staudt, N., Müller-Sienerth, N. &amp; Wright, G. J. Development of an antigen microarray for high throughput monoclonal antibody selection. Biochem. Biophys. Res. Commun. 445, 785–790 (2014)." href="https://www.nature.com/articles/s41586-022-05028-x#ref-CR51" id="ref-link-section-d169881924e1239">51</a></sup>, was prepared in 100 µl 2% (m/v) BSA in HBS, incubated for 60 min and then washed three times. Secondary antibody of goat anti-mouse IgG conjugated to alkaline phosphatase (Sigma A3562) was similarly incubated at 0.2 μg ml<sup>−1</sup> for 30 min before another three washes were performed. Finally, 60 µl of 1.5 mg ml<sup>−1</sup> <i>p</i>-nitrophenyl phosphate (Sigma P4744) in diethanolamine buffer was added and the absorbance at 405 nm was measured after 30–60 min by a Tecan Spark plate reader.</p><h3 id="Sec13">High-throughput SAVEXIS screening</h3><p>The following is the procedure for the final SAVEXIS technique used for screening, as established by extensive optimization of each assay parameter following several iterations during its development. Streptavidin-coated 384-well plates (Greiner 781990) were washed once briefly in 80 μl HBS with 0.1% Tween-20 (HBS-T), then blocked in 2% (m/v) BSA in HBS (10 mM HEPES, 1 mM MgCl<sub>2</sub>, 2 mM CaCl<sub>2</sub>, 5 mM KCl, 140 mM NaCl, pH 7.4) for at least 30 min at 20 °C. Purified bait proteins were diluted in 2% BSA in HBS such that each 50-μl well would contain 100 femtomoles of biotinylated protein. All screening plates were arrayed from a set of stock source plates using a custom-programmed Biomek FXp robot (Beckman Coulter) and manually inspected to correct any wells missed. Baits were left to be captured for 16 h at 4 °C. In parallel, multimeric preys were assembled by mixing 6.25 fmol of streptavidin–HRP (Pierce 21130) with the calculated stoichiometric equivalent of 25 fmol recombinant protein. Preys were prepared in fresh 2% BSA in HBS for at least 30 min at 20 °C before applying to plates (or alternatively at least 60 min at 4 °C). After removing baits from plates, they were washed three times in 50 μl HBS-T supplemented with 0.8 μM desthiobiotin (Sigma D1411) to gently block any unoccupied biotin-binding sites. To each well, 50 μl of prepared prey multimers were then added.After a 60-min incubation at 20 °C, two further 75-μl washes in HBS-T with desthiobiotin were followed by a final wash in 75 μl HBS. Immediately after, 30 μl of TMB chromogenic substrate (Millipore ES001) was dispensed and allowed to incubate at 20 °C for 40 min. To stabilize the signal at this standardized time point, the reaction was halted by adding an additional 30 μl of 0.3% (m/v) NaF (Sigma 201154). Plates were then measured on a Tecan Spark plate reader for absorbance at 650 nm. The identity of each plate and well was then unblinded by a custom R script that stitched all measurements together and matched their numeric barcodes to the proteins they correspond to. All washing steps were performed with multidrop dispenser units (for example, Thermo Fisher Scientific 5840300), and in between washes, plates were centrifuged at 10<i>g</i> for 30 s upside-down with absorbent padding (Kimberly-Clark 7338) layered underneath to remove all trapped bubbles. As controls, every plate had at least one well with no bait added, one well with a tag-only bait construct and two wells of the <i>Plasmodium falciparum</i> P41 protein over which dilutions of the complementary protein P12 presented as a prey were manually added as positive controls for sensitivity<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 52" title="Taechalertpaisarn, T. et al. Biochemical and functional analysis of two Plasmodium falciparum blood-stage 6-cys proteins: P12 and P41. PLoS One 7, e41937 (2012)." href="https://www.nature.com/articles/s41586-022-05028-x#ref-CR52" id="ref-link-section-d169881924e1268">52</a></sup>.</p><h3 id="Sec14">Cell-based binding assays</h3><p>HEK293-E cells were transiently transfected with full-length human cDNA expression constructs (OriGene) for 42–46 h before staining. Ninety microlitres of cells were transferred to U-bottom 96-well plates (Greiner 650161), then washed on ice in 110 µl DPBS (HyClone SH30264). Cells were centrifuged at 300<i>g</i> for 3 min, the supernatant was removed, then resuspended in tetramers of biotinylated recombinant protein complexed with R-phycoerythrin-conjugated streptavidin (BioLegend 405245). Typically, a series of four tetramer quantities were used, ranging from 30 pmol to 1 pmol in a volume of 100 µl DPBS (including around 1 mM calcium and magnesium ions) with 1% BSA. Cells were incubated with tetramers for 45 min on ice before washing with DPBS. Washing consisted of topping each well with an additional 150 µl DPBS, centrifuging, resuspending in 250 µl cold DPBS, then centrifuging again to remove supernatant. Cells were resuspended in 100 µl DPBS with 1% BSA before being analysed on a LSR Fortessa flow cytometer (BD). Recorded events were gated for size and to remove doublets by their forward scatter and side scatter profiles using FlowJo software (v.10.6.1) as previously described<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 46" title="Shilts, J., Crozier, T. W. M., Greenwood, E. J. D., Lehner, P. J. &amp; Wright, G. J. No evidence for basigin/CD147 as a direct SARS-CoV-2 spike binding receptor. Sci. Rep. 11, 413 (2021)." href="https://www.nature.com/articles/s41586-022-05028-x#ref-CR46" id="ref-link-section-d169881924e1283">46</a></sup>. Generally, around 20,000 gated events were collected for each well condition.</p><h3 id="Sec15">Surface plasmon resonance</h3><p>Both kinetic and equilibrium measurements were done on a Biacore 8K SPR instrument as previously described<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 53" title="Galaway, F., Yu, R., Constantinou, A., Prugnolle, F. &amp; Wright, G. J. Resurrection of the ancestral RH5 invasion ligand provides a molecular explanation for the origin of P. falciparum malaria in humans. PLoS Biol. 17, e3000490 (2019)." href="https://www.nature.com/articles/s41586-022-05028-x#ref-CR53" id="ref-link-section-d169881924e1295">53</a></sup>. Biotinylated protein ligands were immobilized on streptavidin-coated Series S CAP chips (GE Life Sciences 28920234) to approximately 200 response units or the closest achievable level. A tag-only negative control was immobilized to an approximately equimolar level. Within 24 h of being purified by size-exclusion chromatography, analytes were injected at 100 μl per minute to derive kinetic data, or at 20 μl per minute for equilibrium measurements. Experiments were performed at 37 °C in HBS-EP buffer (GE Healthcare BR100669). A dilution series of analytes were tested along with at least one concentration in duplicate to check for consistency, plus a buffer-only cycle as a negative control. Response traces were analysed using the manufacturer’s evaluation software (v.1.1). Response units were reference-subtracted and sensorgram data was fit using default parameters.</p><h3 id="Sec16">Assembly of expression constructs</h3><p>The full library of constructs for recombinant expression was assembled by a combination of cloning and gene synthesis. For cloned sequences, cDNA templates (OriGene) were amplified by PCR with primers delineating the extracellular domain. Overhangs on the primers introduced NotI and AscI restriction sites, which enabled ligation into the appropriate vector backbone. All assembled inserts not produced de novo by synthesis were verified by Sanger sequencing. Constructs that were not cloned from existing DNA templates were ordered as synthetic DNA (Twist Biosciences and Thermo Fisher Scientific GeneArt). Synthesized codons were optimized for human cell expression. Optimal Kozak consensus nucleotide sequences were included in all constructs, which occasionally required mutating the second amino acid of the endogenous signal peptide to alanine. Plasmids were prepared to transfection-grade quantity using midiprep or maxiprep kits (for example, Invitrogen K210007).</p><h3 id="Sec17">Protein library design</h3><p>The cell-surface proteomes of blood immune cells were defined by two sources. First, the full dataset of a previous high-resolution proteomics survey of 28 leukocyte populations in resting and activated states (for 44 cell types and states total, covering all major categories)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 1" title="Rieckmann, J. C. et al. Social network architecture of human immune cells unveiled by quantitative proteomics. Nat. Immunol. 18, 583–593 (2017)." href="https://www.nature.com/articles/s41586-022-05028-x#ref-CR1" id="ref-link-section-d169881924e1315">1</a></sup> was merged against a previously established manually curated list of every cell-surface protein in the human genome<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 54" title="Efremova, M., Vento-Tormo, M., Teichmann, S. A. &amp; Vento-Tormo, R. CellPhoneDB: inferring cell–cell communication from combined expression of multi-subunit ligand-receptor complexes. Nat. Protoc. 15, 1484–1506 (2020)." href="https://www.nature.com/articles/s41586-022-05028-x#ref-CR54" id="ref-link-section-d169881924e1319">54</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 55" title="Wood, L. &amp; Wright, G. J. High-content imaging for large-scale detection of low-affinity extracellular protein interactions. SLAS Discov. 24, 987–999 (2019)." href="https://www.nature.com/articles/s41586-022-05028-x#ref-CR55" id="ref-link-section-d169881924e1322">55</a></sup>. The cell-surface proteome list was manually reviewed to verify that each protein does not have publications measuring its localization that contradict a presence on the cell surface. Every protein that was detected was included regardless of how low the expression counts were, with the exception of highly polymorphic proteins such as HLA-A. Second, we added all proteins with a designated CD number as of the 10th human cell differentiation molecule workshop<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 56" title="Engel, P. et al. CD nomenclature 2015: human leukocyte differentiation antigen workshops as a driving force in immunology. J. Immunol. 195, 4555–4563 (2015)." href="https://www.nature.com/articles/s41586-022-05028-x#ref-CR56" id="ref-link-section-d169881924e1326">56</a></sup>. The amino acid sequences and topologies<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 57" title="Omasits, U., Ahrens, C. H., Müller, S. &amp; Wollscheid, B. Protter: interactive protein feature visualization and integration with experimental proteomic data. Bioinformatics 30, 884–886 (2014)." href="https://www.nature.com/articles/s41586-022-05028-x#ref-CR57" id="ref-link-section-d169881924e1330">57</a></sup> of these proteins were manually inspected to determine the extracellular regions and in which structural class the protein belonged (out of type I single-pass/GPI-anchored, type II single-pass, multi-pass, and proteins that function as obligate dimers such as integrins). Proteins that lacked a single contiguous extracellular region of at least 20 amino acids after signal peptide processing were excluded. Similarly, multi-pass proteins without a clear contiguous extracellular domain to express were excluded as incompatible with our recombinant expression system. Constructs were produced as synthetic DNA sequences by Twist Biosciences or Thermo Fisher Scientific GeneArt, optimizing codons for human cell expression. As previously described, proteins were cloned into pTT3 expression vectors matching the intended topology<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 49" title="Sun, Y., Gallagher-Jones, M., Barker, C. &amp; Wright, G. J. A benchmarked protein microarray-based platform for the identification of novel low-affinity extracellular protein interactions. Anal. Biochem. 424, 45–53 (2012)." href="https://www.nature.com/articles/s41586-022-05028-x#ref-CR49" id="ref-link-section-d169881924e1334">49</a></sup>. Single-pass proteins with N-terminal extracellular domains retained their endogenous signal peptides in cases in which a well-annotated or SignalP-predicted signal peptide could be found. Otherwise, an exogenous signal peptide based on the mouse kappa antibody secretion sequence was inserted. Proteins with N-terminal domains had tags attached to their C terminus, whereas the inverted design was used for type II C-terminal proteins<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 58" title="Sun, Y. et al. A human platelet receptor protein microarray identifies the high affinity immunoglobulin E receptor subunit α (FcεR1α) as an activating platelet endothelium aggregation receptor 1 (PEAR1) ligand. Mol. Cell. Proteomics 14, 1265–1274 (2015)." href="https://www.nature.com/articles/s41586-022-05028-x#ref-CR58" id="ref-link-section-d169881924e1339">58</a></sup>. All proteins were produced as fusions with an established recombinant linker comprising domains 3 and 4 of rat CD4 in place of the original transmembrane sequence (termed rCD4)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 59" title="Voulgaraki, D. et al. Multivalent recombinant proteins for probing functions of leucocyte surface proteins such as the CD200 receptor. Immunology 115, 337–346 (2005)." href="https://www.nature.com/articles/s41586-022-05028-x#ref-CR59" id="ref-link-section-d169881924e1343">59</a></sup>, along with a biotin-acceptor site for covalent modification and a hexahistidine tag for purification purposes as described<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 14" title="Bushell, K. M., Söllner, C., Schuster-Boeckler, B., Bateman, A. &amp; Wright, G. J. Large-scale screening for novel low-affinity extracellular protein interactions. Genome Res. 18, 622–630 (2008)." href="https://www.nature.com/articles/s41586-022-05028-x#ref-CR14" id="ref-link-section-d169881924e1347">14</a></sup>. In the case of proteins that exist as dimeric complexes, including integrins, HLA-related molecules, CD1, CD8, GPIb, CD79 and CD94 family NK receptors, we relied on a previously determined design in which one chain would lack any tags, thus directing the purification of full complexes, particularly in cases in which the tagged chain is not secreted on its own<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 60" title="Dundas, K. et al. Alpha-v-containing integrins are host receptors for the Plasmodium falciparum sporozoite surface protein, TRAP. Proc. Natl Acad. Sci. USA 115, 4477–4482 (2018)." href="https://www.nature.com/articles/s41586-022-05028-x#ref-CR60" id="ref-link-section-d169881924e1351">60</a></sup>. Proteins with an intrinsic multimerizing ability owing to intermolecular disulfide bonds, including the TNF-superfamily CD27 and HVEM trimers or B7-family ICOS and CD28 dimers, were expressed with the appropriate cysteine residues intact and allowed to form functional complexes after cell secretion.</p><h3 id="Sec18">Processing of data from the binding screen</h3><p>The matrix of bait versus prey raw absorbance values from the SAVEXIS screen was processed through a two-way median polish. The two separate phases of screening were processed separately then combined to remove per-phase batch effects. Median-polished signals for each protein pair were summed across the two bait–prey orientations (or doubled in the case of homophilic interactions), then all interactions that gave a signal of at least 1.0 were selected to be re-measured in the secondary screen, with the exception of proteins that gave a highly variable background signal in the primary screen. Median-polished signals from the secondary and primary screens were combined by a weighted sum that valued the lower-throughput secondary screen measurements three times more than the high-throughput primary screen measurements. Every protein that gave a clearly reproducible signal in at least one orientation (32 interactions total, identified using our ROC analysis) was then followed-up with validation assays. We excluded interactions with proteins that were highly promiscuous in the screen; that is, appearing more than 20 times in the top 1,000 ranked interaction pairs. Most of these have known mechanisms explaining their lack of specificity, such as being known lectins. Our list of proteins that frequently recur in binding assays may be useful in guiding the interpretation of future screen results, as we note that many of them (for example, CLEC receptors, certain CEACAMs, NRP1, IGF2R, FGFRs and LDLR) are frequently also reported to be binding partners in the context of other published studies.</p><h3 id="Sec19">Manual curation of literature interactions</h3><p>Interactions involving proteins in our immune library were systematically compiled in a three-step process. First, every protein was checked across published reference manuals of immune surface molecules to see whether any interactions were claimed<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 61" title="Zola, H. et al. Leukocyte and Stromal Cell Molecules: the CD Markers (Wiley-Liss, 2007)." href="https://www.nature.com/articles/s41586-022-05028-x#ref-CR61" id="ref-link-section-d169881924e1372">61</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 62" title="Barclay, A. N. et al. The Leucocyte Antigen Factsbook 2nd edn (Academic Press, 1997)." href="https://www.nature.com/articles/s41586-022-05028-x#ref-CR62" id="ref-link-section-d169881924e1375">62</a></sup>. Second, for each protein, its name and applicable synonyms were searched in Google and PubMed using standardized search terms including &lt;“protein name” AND (binding OR interaction OR affinity)&gt; and &lt;“protein name” AND (SPR OR kinetics)&gt;. Finally, existing databases were evaluated, including CellphoneDB, IntAct, PCDq, BioGRID, OmniPath and other published lists<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 63" title="Ramilowski, J. A. et al. A draft network of ligand-receptor-mediated multicellular signalling in human. Nat. Commun. 6, 7866 (2015)." href="https://www.nature.com/articles/s41586-022-05028-x#ref-CR63" id="ref-link-section-d169881924e1379">63</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 64" title="Teku, G. N., Ortutay, C. &amp; Vihinen, M. Identification of core T cell network based on immunome interactome. BMC Syst. Biol. 8, 17 (2014)." href="https://www.nature.com/articles/s41586-022-05028-x#ref-CR64" id="ref-link-section-d169881924e1382">64</a></sup>. Claims identified through these methods were manually verified by identifying the original publications behind each claim. Only interactions supported by citable experimental results were included. In the process, false positives in these databases were removed, such as common falsely claimed interactions based on mouse experiments that are demonstrated not to be conserved in humans, mistakes made by databases mapping protein names, issues arising from considering single genes instead of functional surface protein complexes, or outdated interaction claims that have since been rejected by a consensus of later studies. Wherever quantitative measurements of monomeric binding affinities were available, these were extracted from the original papers. In the process of this manual curation, other relevant results mentioned in the papers were also documented, including if an interaction measurement gave a demonstrably negative result.</p><h3 id="Sec20">Benchmarking screen results</h3><p>Processed binding signals from the arrayed screens were benchmarked against reference sets derived from the detailed manual curation of the literature. Positive reference sets were defined either as every interaction with a claim in the literature, or more stringently as every claim with validation by either a quantitative method (for example, SPR, analytical ultracentrifugation or radiolabelling) or a co-crystal structure (for example, X-ray crystallography or cryogenic electron microscopy). Negative reference sets were either based on experimentally measured negative interactions, or by defining a random negative reference set as previously recommended<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 11" title="Braun, P. et al. An experimentally derived confidence score for binary protein-protein interactions. Nat. Methods 6, 91–97 (2009)." href="https://www.nature.com/articles/s41586-022-05028-x#ref-CR11" id="ref-link-section-d169881924e1394">11</a></sup>. ROC and precision-recall curves were calculated using the PRROC package in R. For these ROC and precision-recall curves, the performance of every possible threshold for converting the median-polished absorbance measurements from the screens into a binary classification of ‘interacting’ or ‘not interacting’ is evaluated against the respective benchmark. The area under the curve (AUC) is reported as an overall summary of screen performance. Only proteins with detectable evidence of recombinant expression were considered when calculating classification performance in the main figures.</p><h3 id="Sec21">Integration of expression data and binding matrices</h3><p>Because our protein–protein interaction network largely represents molecular connections that would occur between cells, we integrated our interaction matrices identifying which cell-surface proteins bound each other with expression data identifying which cell-surface proteins were present on different cells. Expression data included proteomics from bulk-sorted immune cell types and single-cell RNA datasets. We iterated through all possible pairs of cell types in the expression dataset and all pairs of proteins with identified interactions in our binding dataset between those cell-type pairs. From this, we created a master data key that lists all detected molecular interactions between all cell pairs. The mappings of gene identifiers to the Uniprot accessions used in the interaction network files were manually verified to ensure no errors or missed values. From the full listing of cell–cell interactions and the molecules mediating them, we could then perform quantitative or qualitative analyses by either using the expression values or binarized lists of detected interactions. For binarization, gene or protein expression matrices were generally expressed in the form of percentages of replicates (for bulk datasets) or cells (for single-cell datasets) in which expression was detectable in that cell type. Binarization of expression then could be standardized by setting a minimum per cent threshold (that is, for most bulk datasets, expression detected in at least a majority of replicates; or for single-cell datasets, following a common precedent of thresholding at 10% detection<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 54" title="Efremova, M., Vento-Tormo, M., Teichmann, S. A. &amp; Vento-Tormo, R. CellPhoneDB: inferring cell–cell communication from combined expression of multi-subunit ligand-receptor complexes. Nat. Protoc. 15, 1484–1506 (2020)." href="https://www.nature.com/articles/s41586-022-05028-x#ref-CR54" id="ref-link-section-d169881924e1406">54</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 65" title="Armingol, E., Officer, A., Harismendy, O. &amp; Lewis, N. E. Deciphering cell–cell interactions and communication from gene expression. Nat. Rev. Genet. 22, 71–88 (2021)." href="https://www.nature.com/articles/s41586-022-05028-x#ref-CR65" id="ref-link-section-d169881924e1409">65</a></sup>). When comparing diseased to control tissue samples, this process would be repeated separately for the control and disease expression data, and then for each interaction whether it was detected according to these standards in one, both or neither condition could be determined. All integrations were performed in R.</p><h3 id="Sec22">Processing of single-cell RNA-sequencing data</h3><p>Single-cell RNA-sequencing (RNA-seq) datasets were processed following a standard data-cleaning pipeline using the Scanpy package in Python (v.1.4.5)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 66" title="Luecken, M. D. &amp; Theis, F. J. Current best practices in single-cell RNA-seq analysis: a tutorial. Mol. Syst. Biol. 15, e8746 (2019)." href="https://www.nature.com/articles/s41586-022-05028-x#ref-CR66" id="ref-link-section-d169881924e1421">66</a></sup>. Cells with more than 10% of all reads coming from mitochondrial sequences were removed, as were cells with fewer than a minimum of 200 genes or more than a maximum of 3,000 genes. Genes that were detected in fewer than two cells were not considered. Cell-type labels taken from the original published studies were always retained when available. Cell types in the bone marrow dataset were manually annotated after Louvain clustering on the top 1,000 highly variable genes following a previously described pipeline<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 67" title="Zheng, G. X. Y. et al. Massively parallel digital transcriptional profiling of single cells. Nat. Commun. 8, 14049 (2017)." href="https://www.nature.com/articles/s41586-022-05028-x#ref-CR67" id="ref-link-section-d169881924e1425">67</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 68" title="Satija, R., Farrell, J. A., Gennert, D., Schier, A. F. &amp; Regev, A. Spatial reconstruction of single-cell gene expression data. Nat. Biotechnol. 33, 495–502 (2015)." href="https://www.nature.com/articles/s41586-022-05028-x#ref-CR68" id="ref-link-section-d169881924e1428">68</a></sup>. Cell-type clusters with fewer than 10 total cells were not included in subsequent analyses. For Circos-style plots, the ShinyCircos package was used to display the integrated single-cell RNA and interaction matrix data. Linkages on the Circos-style plots are drawn where two cell types express an interacting cell-surface protein pair above a threshold requiring a minimum of 10% of single cells in a cell-type cluster to have at least one mRNA read detected for the surface protein. For visualization, ubiquitous interactions are not displayed, but users can explore different visualization criteria on our interactive website. Signalling analysis was done using the NicheNet package in R (v.1.0.0) with the ligand–target matrix constructed off the immune receptor interactome (known plus novel) described in this manuscript using all default settings for custom model construction with no parameter optimization<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 26" title="Browaeys, R., Saelens, W. &amp; Saeys, Y. NicheNet: modeling intercellular communication by linking ligands to target genes. Nat. Methods 17, 159–162 (2020)." href="https://www.nature.com/articles/s41586-022-05028-x#ref-CR26" id="ref-link-section-d169881924e1432">26</a></sup>. Differential expression testing between diseased and paired reference tissue was done with the Seurat package (v.3.1.5)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 69" title="Stuart, T. et al. Comprehensive integration of single-cell data. Cell 177, 1888–1902 (2019)." href="https://www.nature.com/articles/s41586-022-05028-x#ref-CR69" id="ref-link-section-d169881924e1436">69</a></sup> using a non-parametric Wilcoxon rank sum test. Genome-wide multiple testing correction was applied.</p><h3 id="Sec23">Immune activation differential expression</h3><p>Cell types with proteomics measurements in both activated and resting states were used for differential expression calculations. The DESeq2 package in R was used to model expression counts and compute Wald test statistics<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 70" title="Love, M. I., Huber, W. &amp; Anders, S. Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. Genome Biol. 15, 550 (2014)." href="https://www.nature.com/articles/s41586-022-05028-x#ref-CR70" id="ref-link-section-d169881924e1448">70</a></sup>. From these results, the set of differentially expressed cell-surface proteins upon immune activation were determined by setting a fold change threshold of greater than 2. Alternatively, we also compared using adjusted <i>P</i> value thresholds, which gave similar results (that is, Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-022-05028-x#Fig10">6</a>). Each protein was mapped to its measured binding affinity for the interaction(s) it participates in. For proteins with multiple interactions, this ambiguity was addressed by including the affinity of every binding interaction as separate points. These affinity values for each cell type were then grouped on the basis of whether their corresponding proteins were upregulated or downregulated upon immune activation. The affinities of upregulated and downregulated interactions were compared by Welch’s <i>t</i>-test.</p><h3 id="Sec24">Mathematical model based on cell-binding kinetics</h3><p>Details of the kinetic model and derivations of equations can be found in the <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41586-022-05028-x#MOESM1">Supplementary Equations</a>. For each blood immune cell type to be modelled, published parameters about its physical geometry, proportions and protein expression were compiled. When cell types were to be matched with experimental data containing less subtype resolution than the proteomics expression dataset, expression values were estimated as the weighted average of all of a cell type’s constituent subtypes, weighted by their measured proportions in blood (for example, if total CD56<sup>+</sup> NK cells were measured in the experiment, the ‘NK dim’ and ‘NK bright’ subtypes measured in the proteomics would be proportionally averaged in the model). Absolute protein counts per cell from the expression data were converted to average protein density per surface area by assuming that all protein is present on an approximately spherical cell surface. Per quantified interaction, a relative equilibrium density of bound protein molecules was calculated through the Michaelis–Menten equation<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 71" title="Bell, G. I. Models for the specific adhesion of cells to cells. Science 200, 618–627 (1978)." href="https://www.nature.com/articles/s41586-022-05028-x#ref-CR71" id="ref-link-section-d169881924e1475">71</a></sup>. To determine the relative connection affinities of different cell types, the sums of all interactions calculated were compared for different pairs of cells.</p><h3 id="Sec25">Binding-perturbation model using differential equations</h3><p>Although the core kinetics model can calculate relative cellular affinities, by formulation it cannot on its own predict the outcomes of specific perturbations to proteins (for example, its predictions from removing a surface protein would all uniformly be decreases in binding, or predictions from a strengthening in affinity would all be increases). Thus, when making perturbation predictions, the relative cellular affinities were passed as parameters to a system of differential equations based on the law of mass action. Specifically, all cell types were assumed to collide and form connections at a constant rate, and the dissociation rate for that cell–cell bond was inversely proportional to the relative affinity determined by the core kinetics model. As initial conditions, all cells were assumed to be unbound at frequencies that match literature-reported values for human blood. Numerical integration proceeded until equilibrium was reached. Calculations were performed using the PySB package in Python (v.1.11)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 72" title="Lopez, C. F., Muhlich, J. L., Bachman, J. A. &amp; Sorger, P. K. Programming biological models in Python using PySB. Mol. Syst. Biol. 9, 646 (2013)." href="https://www.nature.com/articles/s41586-022-05028-x#ref-CR72" id="ref-link-section-d169881924e1487">72</a></sup>. Our initial perturbation studies have simulated removing particular surface proteins by setting the expression values for that protein to zero across all cell types.</p><h3 id="Sec26">Network centrality calculations</h3><p>Counts of binarized interactions following integration of expression datasets with the interaction table were converted into a weighted undirected network graph using the igraph package in R (v.1.2.5). Eigenvector centralities were calculated for each cell type in the graph. Each tissue with a single-cell resolution dataset available was computed separately. To compare myeloid cell populations to other lineages, we performed two-sided Welch’s <i>t</i>-tests on the centrality metrics. <i>P</i> value corrections for the multiple tissues tested were done by the Benjamini–Hochberg procedure. For these and all other box plots, the central box displays the 25th, 50th and 75th percentiles, with whiskers extending to 1.5 times the interquartile range.</p><h3 id="Sec27">Spatial transcriptomics analysis</h3><p>Lymph-node spatial transcriptomic data were downloaded from 10x Genomics and processed using a standard cleaning pipeline in Python with the Scanpy package. Measured spots on the array were constrained to have between 4,000 and 36,000 total transcript counts, with fewer than 20% of reads derived from mitochondrial sequences, and at least 2,000 different genes detected. Genes were constrained to be detected in at least five spots. A radius of 150 units on the spatial coordinates was empirically determined to encompass only the immediately connected neighbouring spots to a given spot’s centre, and used when determining neighbour relationships. Iterating through all protein pairs in the interaction network, spots were marked for whether they detectably expressed at least one count of either a single protein of the pair, both proteins or neither. The number of instances in which both proteins of an interacting pair were detected in physically connected spots (either directly adjacent spots or the same spot containing cells expressing both, owing to how the resolution of spots means that each spot on average contains more than one cell) was tabulated, as was the number of physically connected spots in which both expressed only the same protein but not an interacting pair. To calculate colocalization scores for each pair of protein-coding genes in a receptor–ligand interaction, the fraction of pairings that are ‘interaction-capable’ (that is, having physically neighbouring expression of the receptor and ligand) was calculated over the entire lymph-node tissue section. To test our experimentally found interaction list against a null hypothesis of randomly paired cell-surface proteins, we took the same proteins as in our true interaction network and randomly permuted which were paired together. The colocalization scores from this permuted null distribution were compared against both the scores from our literature-curated interaction list and our empirically discovered interactions. Statistical testing consisted first of an omnibus one-way ANOVA between these three sets of colocalization scores, followed by a post-hoc Tukey’s honestly significant difference test.</p><h3 id="Sec28">In situ hybridization of lymph nodes</h3><p>Fresh, unfixed tissue samples were flash-frozen in OCT using dry-ice-cooled isopentane and morphology was checked by haematoxylin and eosin staining. For RNAScope, 10-μm-thick cryosections were cut onto SuperFrost Plus slides, fixed for 15 min with chilled 4% paraformaldehyde (PFA) followed by 90 min in room temperature in 4% PFA, then dehydrated through an ethanol series (50%, 70%, 100% and 100% ethanol). Slides were then processed using the RNAScope 2.5 LS multiplex fluorescent assay (ACD, Bio-Techne) on the Leica BOND RX system (Leica) with protease III treatment. Initially, RNAScope positive and negative control probes were tested on sections, before proceeding to probes of interest with fluorophores opal 520, opal 570 and opal 650 at 1:1,000 concentration. All probes were previously established and are commercially available under the catalogue numbers 546188-C3 (<i>JAG1</i>), 845158-C1 (<i>VASN</i>), 491518-C4 (<i>VISTA</i>; also known as <i>VSIR</i>), 457368-C1 (<i>HLA-E</i>), 460048-C2 (<i>PLXNA4</i>), 418328-C1 (<i>APP</i>), 442598-C1 (<i>CNTN1</i>), 601738-C3 (<i>MCAM</i>) and 601998-C4 (<i>CD45</i>; also known as <i>PTPRC</i>). These were then imaged at 20× magnification on a Perkin Elmer Opera Phenix High Content Screening System with water immersion.</p><h3 id="Sec29">Isolation of human PBMCs</h3><p>Blood buffy coat from a healthy donor was obtained by the Blutspende Zurich, under a study protocol approved by the cantonal ethical committee Zurich (KEK Zurich, BASEC-Nr 2019-01579). To obtain PBMCs, the sample was diluted 1:1 in PBS (Gibco) and cells were isolated with a Histopaque-1077 density gradient (Sigma-Aldrich) according to the manufacturer&#39;s instructions. Subsequently, cells at the interface were collected, washed once with PBS and resuspended in RPMI 1640 + GlutaMax medium (Gibco) supplemented with 10% human serum (Chemie Brunschwig). Immune cells were seeded and cultured in CellCarrier 384 Ultra, clear-bottom, tissue-culture-treated plates (PerkinElmer) at a density of 2 × 10<sup>4</sup> cells per well in 50 μl per well and incubated at 37 °C and 5% CO<sub>2</sub>. Cell number and viability was determined by use of a Countess II Cell Counter (Thermo Fisher Scientific).</p><h3 id="Sec30">Leukocyte protein perturbation and fixation</h3><p>Isolated leukocytes were incubated with purified recombinant proteins tetramerized around neutravidin (Thermo Fisher Scientific 31000) at doses of 80 pmol (1.6 μM) to 200 pmol (4 μM) per well. Additional negative controls of buffer-only, a tag-only tetramer and the elution material from the supernatant of mock-transfected cells were included. All proteins for which a novel interaction was identified and that could be sufficiently expressed to provide a 4 μM concentration were included, along with CD209, CD58, ICAM1 and SIRPA as previously characterized controls. Assay plates were created in a fully randomized layout and prepared by an Echo 555 Liquid Handler. Four assays were done with five replicates per condition per assay, including 4 h and 24 h time points with and without the addition of 1 pg µl<sup>−1</sup> LPS. The assay was stopped by fixing and permeabilizing the cells with 20 μl per well of a solution containing 0.5% (w/v) formalin (Sigma-Aldrich), 0.05% (v/v) Triton X-100 (Sigma-Aldrich), 10 mM sodium(meta)periodate (Sigma-Aldrich) and 75 mM <span>l</span>-lysine monohydrochloride (Sigma-Aldrich). After a 20-min incubation at room temperature, the fixative-containing medium was aspirated by use of a HydroSpeed plate washer (Tecan). The cells were then blocked (50 μl per well) with PBS supplemented with 5% fetal bovine serum (Gibco) and photobleached for 4 to 24 h at 4 °C to reduce background fluorescence by illuminating the fixed cells with conventional white light LED panels.</p><h3 id="Sec31">Immunostaining and imaging</h3><p>For immunohistochemistry staining, all primary antibodies were diluted 1:300 in PBS with 6 µM DAPI (Sigma-Aldrich) for nuclear detection. Antibodies used were anti-CD3 Alexa Fluor 647 (BioLegend, Clone UCHT1), anti-CD4 FITC (BioLegend, clone SK3), anti-CD8 PE (BD Biosciences, clone SK1), anti-CD19 FITC (BioLegend, clone SJ25C1), anti-CD56 PE (Beckman Coulter, clone N901), anti-CD16 PE (BioLegend, clone 3G8), anti-CD14 Alexa Fluor 647 (BioLegend, clone HCD14) and anti-CD20 PE (BD Biosciences, clone 2H7). Per well, 20 μl of the antibody cocktail was added and incubated for 1 h at room temperature. For imaging, a PerkinElmer Opera Phenix automated spinning-disk confocal microscope was used and each well of a 348-well plate was imaged at 20× magnification with 5 × 5 non-overlapping images, covering the whole well surface. The images were taken sequentially from the bright-field (650–760 nm), DAPI/nuclear signal (435–480 nm), GFP signal (500–550 nm), PE signal (570–630 nm) and APC signal (650–760 nm) channels. Raw .tiff files were exported for analysis.</p><h3 id="Sec32">Image processing and quality filtering</h3><p>Cell detection and single-cell image analysis was performed using CellProfiler (v.2)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 73" title="Carpenter, A. E. et al. CellProfiler: image analysis software for identifying and quantifying cell phenotypes. Genome Biol. 7, R100 (2006)." href="https://www.nature.com/articles/s41586-022-05028-x#ref-CR73" id="ref-link-section-d169881924e1597">73</a></sup>. Nuclear segmentation was performed by thresholding on DAPI intensity. Cellular outlines were estimated by a circular expansion from the outlines of the nucleus. In addition, a second and larger expansion from the nuclei was performed to measure the local area around each single cell (local cellular background). Standard CellProfiler-based intensity, shape and texture features of the nucleus, cytoplasm and the local cell proximity were extracted for each measured channel. Raw fluorescent intensities were log<sub>10</sub>-transformed and normalized towards the local cellular background as previously described<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 25" title="Vladimer, G. I. et al. Global survey of the immunomodulatory potential of common drugs. Nat. Chem. Biol. 13, 681–690 (2017)." href="https://www.nature.com/articles/s41586-022-05028-x#ref-CR25" id="ref-link-section-d169881924e1603">25</a></sup>.</p><h3 id="Sec33">Cell-type image classifier</h3><p>An 8-class 71-layer deep convolutional neural network with an adapted ResNet architecture<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 74" title="He, K., Zhang, X., Ren, S. &amp; Sun, J. Deep residual learning for image recognition. In 2016 IEEE Conference on Computer Vision and Pattern Recognition (CVPR) (eds. Agapito, L. et al.) 770–778 (IEEE, 2016)." href="https://www.nature.com/articles/s41586-022-05028-x#ref-CR74" id="ref-link-section-d169881924e1615">74</a></sup> was implemented using 48 × 48 × 5 input images in MATLAB’s Neural Network Toolbox (v.R2020a). For all morphology classifiers (B cell, NK cell, T cell) a 2-class 39-layer deep convolutional neural network (CNN) with an adapted ResNet architecture was used. Input images of 48 × 48 × 3 were used, in which all images contained the DAPI and the bright-field channel whereas the third channel contained the respective channel with the lineage marker. In the case of dendritic cells, the absence of all other lineage markers was used. In all CNN classifications, 48 × 48 pixel sub-images around each nuclei centre were generated. Cells closer than 24 pixels to the border of an image were excluded from all classifications. Network training, evaluation and classification were performed as previously described<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 29" title="Severin, Y. et al. Multiplexed high-throughput immune cell imaging reveals molecular health-associated phenotypes. Preprint at bioRxiv 
                https://doi.org/10.1101/2021.12.03.471105
                
               (2021)." href="https://www.nature.com/articles/s41586-022-05028-x#ref-CR29" id="ref-link-section-d169881924e1619">29</a></sup>.</p><h3 id="Sec34">Image processing for cell interactions</h3><p>For extracting cell–cell interactions from image data, a simplified version of a previously published<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 25" title="Vladimer, G. I. et al. Global survey of the immunomodulatory potential of common drugs. Nat. Chem. Biol. 13, 681–690 (2017)." href="https://www.nature.com/articles/s41586-022-05028-x#ref-CR25" id="ref-link-section-d169881924e1631">25</a></sup> method was used. Cell–cell interaction analysis was conducted over all different image sites within the same well. Cells were scored as interacting if their nuclear centroids were within a Euclidean distance of 40 pixels. To calculate the interaction score of a cell with type A interacting with a cell of type B, we first calculated specific interactions and total interactions per well. We define specific interactions as the total count of type B cells within the defined radius around a cell of type A. Total interactions are considered as the total count of all interacting cells in that well. To calculate the final interaction score, specific interactions were divided by the product of (the fraction of type A cells of all cells) × (the fraction of type B cells of all cells) × total interactions.</p><h3 id="Sec35">Pharmacoscopy data processing and analysis</h3><p>For graphical displays, observed cell state and cell–cell interaction frequencies were first normalized against their respective controls. Recombinant protein conditions were normalized against an average of the control wells for each respective time point, dose and background immune activation. Protein controls consisted of a buffer-only mock treatment well, a well with carryover from purifying an empty transfection and a well stimulated with only the protein epitope tags. As previously described for pharmacoscopy experiments, normalizations were calculated as the observed value minus the control average, divided by the maximum of those two values to give a metric bounded from [−1, 1] with 0 representing no change relative to control. For statistical analysis, the raw measurements across all available control conditions were compared against all replicates of each corresponding perturbation condition by Welch’s <i>t</i>-test. To adjust for multiple testing, the Benjamini–Hochberg test was used and a 10% false discovery rate threshold was set for delineating significant effects. These calculations are shown both for each individual dose applied, as well as when all doses of a given protein treatment were aggregated. The same overall analysis procedure was done for both the cell state frequencies and the cell–cell interaction datasets, primarily differing in which time point (24 h or 4 h, respectively) was chosen for focused statistical analyses. The non-classical CD16-positive monocyte population was omitted from all final plots as the extremely low numbers of these cells that were detected in each experiment led to inconsistent and often non-finite effect sizes (for example, changes from 0 cells found to 1 cell found between condition and control).</p><h3 id="Sec36">Model comparison to microscopy interaction data</h3><p>The cell types measured in the pharmacoscopy experiments were all incorporated into the system of differential equations in the model to determine both baseline cell–cell interaction frequencies and to predict changes when single surface proteins were removed from the model. The model-calculated frequencies of cells in unbound and all possible interacting paired configurations were processed into interaction scores by following the same equations used to process the microscopy image data. Data from the 4-h time point with no background LPS stimulation were used for all protein conditions. Because our recombinant proteins could either trigger or suppress a given receptor pathway, we compared the absolute magnitudes of the normalized interaction scores to quantify the extent of perturbation. For the final analyses, we restricted the set of proteins to compare to only those for which the model predicted that perturbing that protein would produce meaningful changes in cell–cell interactions (defined as the model prediction maximum for a condition being no lower than one-fifth of the median perturbation prediction maximum across the full set of proteins for which we gathered experimental data). The top third of cell pairs that were predicted to have the greatest perturbation magnitude according to the model were contrasted with the remaining cell pairs that were predicted not to be strongly changed after perturbation. A one-sided <i>t</i>-test was then done to determine whether the observed changes in interaction score after the addition of recombinant proteins in our experiment for the cell pairs that were predicted by our model to be perturbed were significantly greater than the baseline interaction score changes of all other cell pairs that were not predicted to change. <i>P</i> values were adjusted using the Benjamini–Hochberg procedure.</p><h3 id="Sec37">Access to human tissue</h3><p>Human lymph nodes were obtained from deceased transplant organ donors by the Cambridge Biorepository for Translational Medicine (CBTM) with informed consent from the donor families and approval from the NRES Committee of East of England – Cambridge South (15/EE/0152). This consent includes the generation of open-access genetic sequencing data and publication in open-access journals in line with Wellcome Trust policy. CBTM operates in accordance with the guidelines of the UK Human Tissue Authority. Blood samples from anonymized healthy donors were provided by the Blutspende Zurich, under a study protocol approved by the cantonal ethical committee Zurich (KEK Zurich, BASEC-Nr 2019-01579).</p><h3 id="Sec38">Reporting summary</h3><p>Further information on research design is available in the <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41586-022-05028-x#MOESM2">Nature Research Reporting Summary</a> linked to this paper.</p></div></div></section>
            

            <section data-title="Data availability"><div id="data-availability-section"><h2 id="data-availability">Data availability</h2><div id="data-availability-content">
            
            <p>The data files required to generate the figures and analyses in this paper are provided in the GitHub repository accompanying this manuscript: <a href="https://github.com/jshilts/shilts-et-al-2022-immunoreceptors">https://github.com/jshilts/shilts-et-al-2022-immunoreceptors</a>.</p>
          </div></div></section><section data-title="Code availability"><div id="code-availability-section"><h2 id="code-availability">Code availability</h2><div id="code-availability-content">
            
            <p>The custom code files used for this study are publicly available in the GitHub repository accompanying this manuscript: <a href="https://github.com/jshilts/shilts-et-al-2022-immunoreceptors">https://github.com/jshilts/shilts-et-al-2022-immunoreceptors</a>.</p>
          </div></div></section><div id="MagazineFulltextArticleBodySuffix"><section aria-labelledby="Bib1" data-title="References"><div id="Bib1-section"><h2 id="Bib1">References</h2><div id="Bib1-content"><div data-container-section="references"><ol><li data-counter="1."><p id="ref-CR1">Rieckmann, J. C. et al. Social network architecture of human immune cells unveiled by quantitative proteomics. <i>Nat. Immunol.</i> <b>18</b>, 583–593 (2017).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXkvFagtrc%3D" aria-label="CAS reference 1">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28263321" aria-label="PubMed reference 1">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://doi.org/10.1038%2Fni.3693" aria-label="Article reference 1">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 1" href="http://scholar.google.com/scholar_lookup?&amp;title=Social%20network%20architecture%20of%20human%20immune%20cells%20unveiled%20by%20quantitative%20proteomics&amp;journal=Nat.%20Immunol.&amp;volume=18&amp;pages=583-593&amp;publication_year=2017&amp;author=Rieckmann%2CJC">
                    Google Scholar</a> 
                </p></li><li data-counter="2."><p id="ref-CR2">Yao, S., Zhu, Y. &amp; Chen, L. Advances in targeting cell surface signalling molecules for immune modulation. <i>Nat. Rev. Drug Discov.</i> <b>12</b>, 130–146 (2013).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXhslaju7c%3D" aria-label="CAS reference 2">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23370250" aria-label="PubMed reference 2">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3698571" aria-label="PubMed Central reference 2">PubMed Central</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://doi.org/10.1038%2Fnrd3877" aria-label="Article reference 2">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 2" href="http://scholar.google.com/scholar_lookup?&amp;title=Advances%20in%20targeting%20cell%20surface%20signalling%20molecules%20for%20immune%20modulation&amp;journal=Nat.%20Rev.%20Drug%20Discov.&amp;volume=12&amp;pages=130-146&amp;publication_year=2013&amp;author=Yao%2CS&amp;author=Zhu%2CY&amp;author=Chen%2CL">
                    Google Scholar</a> 
                </p></li><li data-counter="3."><p id="ref-CR3">Germain, R. N., Robey, E. A. &amp; Cahalan, M. D. A decade of imaging cellular motility and interaction dynamics in the immune system. <i>Science</i> <b>336</b>, 1676–1681 (2012).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2012Sci...336.1676G" aria-label="ADS reference 3">ADS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XptFWmsr4%3D" aria-label="CAS reference 3">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22745423" aria-label="PubMed reference 3">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3405774" aria-label="PubMed Central reference 3">PubMed Central</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://doi.org/10.1126%2Fscience.1221063" aria-label="Article reference 3">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 3" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20decade%20of%20imaging%20cellular%20motility%20and%20interaction%20dynamics%20in%20the%20immune%20system&amp;journal=Science&amp;volume=336&amp;pages=1676-1681&amp;publication_year=2012&amp;author=Germain%2CRN&amp;author=Robey%2CEA&amp;author=Cahalan%2CMD">
                    Google Scholar</a> 
                </p></li><li data-counter="4."><p id="ref-CR4">Grandclaudon, M. et al. A quantitative multivariate model of human dendritic cell–T helper cell communication. <i>Cell</i> <b>179</b>, 432–447 (2019).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhvFers7zK" aria-label="CAS reference 4">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31585082" aria-label="PubMed reference 4">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://doi.org/10.1016%2Fj.cell.2019.09.012" aria-label="Article reference 4">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 4" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20quantitative%20multivariate%20model%20of%20human%20dendritic%20cell%E2%80%93T%20helper%20cell%20communication&amp;journal=Cell&amp;volume=179&amp;pages=432-447&amp;publication_year=2019&amp;author=Grandclaudon%2CM">
                    Google Scholar</a> 
                </p></li><li data-counter="5."><p id="ref-CR5">Bausch-Fluck, D. et al. The in silico human surfaceome. <i>Proc. Natl Acad. Sci. USA</i> <b>115</b>, E10988–E10997 (2018).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXit1SgtL%2FK" aria-label="CAS reference 5">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30373828" aria-label="PubMed reference 5">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6243280" aria-label="PubMed Central reference 5">PubMed Central</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://doi.org/10.1073%2Fpnas.1808790115" aria-label="Article reference 5">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 5" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20in%20silico%20human%20surfaceome&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=115&amp;pages=E10988-E10997&amp;publication_year=2018&amp;author=Bausch-Fluck%2CD">
                    Google Scholar</a> 
                </p></li><li data-counter="6."><p id="ref-CR6">Gawel, D. R. et al. A validated single-cell-based strategy to identify diagnostic and therapeutic targets in complex diseases. <i>Genome Med.</i> <b>11</b>, 47 (2019).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31358043" aria-label="PubMed reference 6">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6664760" aria-label="PubMed Central reference 6">PubMed Central</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://doi.org/10.1186%2Fs13073-019-0657-3" aria-label="Article reference 6">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 6" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20validated%20single-cell-based%20strategy%20to%20identify%20diagnostic%20and%20therapeutic%20targets%20in%20complex%20diseases&amp;journal=Genome%20Med.&amp;volume=11&amp;publication_year=2019&amp;author=Gawel%2CDR">
                    Google Scholar</a> 
                </p></li><li data-counter="7."><p id="ref-CR7">Bergthaler, A. &amp; Menche, J. The immune system as a social network. <i>Nat. Immunol.</i> <b>18</b>, 481–482 (2017).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXmtFKiurw%3D" aria-label="CAS reference 7">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28418389" aria-label="PubMed reference 7">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://doi.org/10.1038%2Fni.3727" aria-label="Article reference 7">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 7" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20immune%20system%20as%20a%20social%20network&amp;journal=Nat.%20Immunol.&amp;volume=18&amp;pages=481-482&amp;publication_year=2017&amp;author=Bergthaler%2CA&amp;author=Menche%2CJ">
                    Google Scholar</a> 
                </p></li><li data-counter="8."><p id="ref-CR8">Overington, J. P., Al-Lazikani, B. &amp; Hopkins, A. L. How many drug targets are there? <i>Nat. Rev. Drug Discov.</i> <b>5</b>, 993–996 (2006).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD28Xht1Kju7jM" aria-label="CAS reference 8">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17139284" aria-label="PubMed reference 8">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://doi.org/10.1038%2Fnrd2199" aria-label="Article reference 8">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 8" href="http://scholar.google.com/scholar_lookup?&amp;title=How%20many%20drug%20targets%20are%20there%3F&amp;journal=Nat.%20Rev.%20Drug%20Discov.&amp;volume=5&amp;pages=993-996&amp;publication_year=2006&amp;author=Overington%2CJP&amp;author=Al-Lazikani%2CB&amp;author=Hopkins%2CAL">
                    Google Scholar</a> 
                </p></li><li data-counter="9."><p id="ref-CR9">von Hundelshausen, P. et al. Chemokine interactome mapping enables tailored intervention in acute and chronic inflammation. <i>Sci. Transl. Med.</i> <b>9</b>, eaah6650 (2017).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://doi.org/10.1126%2Fscitranslmed.aah6650" aria-label="Article reference 9">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXjvVaht7k%3D" aria-label="CAS reference 9">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 9" href="http://scholar.google.com/scholar_lookup?&amp;title=Chemokine%20interactome%20mapping%20enables%20tailored%20intervention%20in%20acute%20and%20chronic%20inflammation&amp;journal=Sci.%20Transl.%20Med.&amp;volume=9&amp;publication_year=2017&amp;author=Hundelshausen%2CP">
                    Google Scholar</a> 
                </p></li><li data-counter="10."><p id="ref-CR10">Lin, H. et al. Discovery of a cytokine and its receptor by functional screening of the extracellular proteome. <i>Science</i> <b>320</b>, 807–811 (2008).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2008Sci...320..807L" aria-label="ADS reference 10">ADS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXlsVGltrc%3D" aria-label="CAS reference 10">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18467591" aria-label="PubMed reference 10">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://doi.org/10.1126%2Fscience.1154370" aria-label="Article reference 10">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 10" href="http://scholar.google.com/scholar_lookup?&amp;title=Discovery%20of%20a%20cytokine%20and%20its%20receptor%20by%20functional%20screening%20of%20the%20extracellular%20proteome&amp;journal=Science&amp;volume=320&amp;pages=807-811&amp;publication_year=2008&amp;author=Lin%2CH">
                    Google Scholar</a> 
                </p></li><li data-counter="11."><p id="ref-CR11">Braun, P. et al. An experimentally derived confidence score for binary protein-protein interactions. <i>Nat. Methods</i> <b>6</b>, 91–97 (2009).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXhsVKlt7%2FF" aria-label="CAS reference 11">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19060903" aria-label="PubMed reference 11">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://doi.org/10.1038%2Fnmeth.1281" aria-label="Article reference 11">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 11" href="http://scholar.google.com/scholar_lookup?&amp;title=An%20experimentally%20derived%20confidence%20score%20for%20binary%20protein-protein%20interactions&amp;journal=Nat.%20Methods&amp;volume=6&amp;pages=91-97&amp;publication_year=2009&amp;author=Braun%2CP">
                    Google Scholar</a> 
                </p></li><li data-counter="12."><p id="ref-CR12">Kotlyar, M. et al. In silico prediction of physical protein interactions and characterization of interactome orphans. <i>Nat. Methods</i> <b>12</b>, 79–84 (2015).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXhvFKlsrzK" aria-label="CAS reference 12">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25402006" aria-label="PubMed reference 12">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://doi.org/10.1038%2Fnmeth.3178" aria-label="Article reference 12">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 12" href="http://scholar.google.com/scholar_lookup?&amp;title=In%20silico%20prediction%20of%20physical%20protein%20interactions%20and%20characterization%20of%20interactome%20orphans&amp;journal=Nat.%20Methods&amp;volume=12&amp;pages=79-84&amp;publication_year=2015&amp;author=Kotlyar%2CM">
                    Google Scholar</a> 
                </p></li><li data-counter="13."><p id="ref-CR13">Özkan, E. et al. An extracellular interactome of immunoglobulin and LRR proteins reveals receptor-ligand networks. <i>Cell</i> <b>154</b>, 228–239 (2013).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23827685" aria-label="PubMed reference 13">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3756661" aria-label="PubMed Central reference 13">PubMed Central</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://doi.org/10.1016%2Fj.cell.2013.06.006" aria-label="Article reference 13">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXhtVGktLvN" aria-label="CAS reference 13">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 13" href="http://scholar.google.com/scholar_lookup?&amp;title=An%20extracellular%20interactome%20of%20immunoglobulin%20and%20LRR%20proteins%20reveals%20receptor-ligand%20networks&amp;journal=Cell&amp;volume=154&amp;pages=228-239&amp;publication_year=2013&amp;author=%C3%96zkan%2CE">
                    Google Scholar</a> 
                </p></li><li data-counter="14."><p id="ref-CR14">Bushell, K. M., Söllner, C., Schuster-Boeckler, B., Bateman, A. &amp; Wright, G. J. Large-scale screening for novel low-affinity extracellular protein interactions. <i>Genome Res.</i> <b>18</b>, 622–630 (2008).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXks1alsLY%3D" aria-label="CAS reference 14">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18296487" aria-label="PubMed reference 14">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2279249" aria-label="PubMed Central reference 14">PubMed Central</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://doi.org/10.1101%2Fgr.7187808" aria-label="Article reference 14">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 14" href="http://scholar.google.com/scholar_lookup?&amp;title=Large-scale%20screening%20for%20novel%20low-affinity%20extracellular%20protein%20interactions&amp;journal=Genome%20Res.&amp;volume=18&amp;pages=622-630&amp;publication_year=2008&amp;author=Bushell%2CKM&amp;author=S%C3%B6llner%2CC&amp;author=Schuster-Boeckler%2CB&amp;author=Bateman%2CA&amp;author=Wright%2CGJ">
                    Google Scholar</a> 
                </p></li><li data-counter="15."><p id="ref-CR15">Sokolina, K. et al. Systematic protein-protein interaction mapping for clinically relevant human GPCRs. <i>Mol. Syst. Biol.</i> <b>13</b>, 918 (2017).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28298427" aria-label="PubMed reference 15">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5371730" aria-label="PubMed Central reference 15">PubMed Central</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://doi.org/10.15252%2Fmsb.20167430" aria-label="Article reference 15">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXlt1yqt74%3D" aria-label="CAS reference 15">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 15" href="http://scholar.google.com/scholar_lookup?&amp;title=Systematic%20protein-protein%20interaction%20mapping%20for%20clinically%20relevant%20human%20GPCRs&amp;journal=Mol.%20Syst.%20Biol.&amp;volume=13&amp;publication_year=2017&amp;author=Sokolina%2CK">
                    Google Scholar</a> 
                </p></li><li data-counter="16."><p id="ref-CR16">Snider, J. et al. Fundamentals of protein interaction network mapping. <i>Mol. Syst. Biol.</i> <b>11</b>, 848 (2015).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26681426" aria-label="PubMed reference 16">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4704491" aria-label="PubMed Central reference 16">PubMed Central</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://doi.org/10.15252%2Fmsb.20156351" aria-label="Article reference 16">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XnvVKjtw%3D%3D" aria-label="CAS reference 16">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 16" href="http://scholar.google.com/scholar_lookup?&amp;title=Fundamentals%20of%20protein%20interaction%20network%20mapping&amp;journal=Mol.%20Syst.%20Biol.&amp;volume=11&amp;publication_year=2015&amp;author=Snider%2CJ">
                    Google Scholar</a> 
                </p></li><li data-counter="17."><p id="ref-CR17">Wojtowicz, W. M. et al. A human IgSF cell-surface interactome reveals a complex network of protein–protein interactions. <i>Cell</i> <b>182</b>, 1027–1043 (2020).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhs1Gqsb3P" aria-label="CAS reference 17">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32822567" aria-label="PubMed reference 17">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7440162" aria-label="PubMed Central reference 17">PubMed Central</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://doi.org/10.1016%2Fj.cell.2020.07.025" aria-label="Article reference 17">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 17" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20human%20IgSF%20cell-surface%20interactome%20reveals%20a%20complex%20network%20of%20protein%E2%80%93protein%20interactions&amp;journal=Cell&amp;volume=182&amp;pages=1027-1043&amp;publication_year=2020&amp;author=Wojtowicz%2CWM">
                    Google Scholar</a> 
                </p></li><li data-counter="18."><p id="ref-CR18">Verschueren, E. et al. The immunoglobulin superfamily receptome defines cancer-relevant networks associated with clinical outcome. <i>Cell</i> <b>182</b>, 329–344 (2020).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXht1KitrrI" aria-label="CAS reference 18">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32589946" aria-label="PubMed reference 18">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://doi.org/10.1016%2Fj.cell.2020.06.007" aria-label="Article reference 18">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 18" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20immunoglobulin%20superfamily%20receptome%20defines%20cancer-relevant%20networks%20associated%20with%20clinical%20outcome&amp;journal=Cell&amp;volume=182&amp;pages=329-344&amp;publication_year=2020&amp;author=Verschueren%2CE">
                    Google Scholar</a> 
                </p></li><li data-counter="19."><p id="ref-CR19">Evans, E. J. et al. The T cell surface-how well do we know it? <i>Immunity</i> <b>19</b>, 213–223 (2003).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3sXotlGqsbw%3D" aria-label="CAS reference 19">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=12932355" aria-label="PubMed reference 19">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://doi.org/10.1016%2FS1074-7613%2803%2900198-5" aria-label="Article reference 19">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 19" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20T%20cell%20surface-how%20well%20do%20we%20know%20it%3F&amp;journal=Immunity&amp;volume=19&amp;pages=213-223&amp;publication_year=2003&amp;author=Evans%2CEJ">
                    Google Scholar</a> 
                </p></li><li data-counter="20."><p id="ref-CR20">Estefanía, E. et al. Human KIR2DL5 is an inhibitory receptor expressed on the surface of NK and T lymphocyte subsets. <i>J. Immunol.</i> <b>178</b>, 4402–4410 (2007).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17371997" aria-label="PubMed reference 20">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://doi.org/10.4049%2Fjimmunol.178.7.4402" aria-label="Article reference 20">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 20" href="http://scholar.google.com/scholar_lookup?&amp;title=Human%20KIR2DL5%20is%20an%20inhibitory%20receptor%20expressed%20on%20the%20surface%20of%20NK%20and%20T%20lymphocyte%20subsets&amp;journal=J.%20Immunol.&amp;volume=178&amp;pages=4402-4410&amp;publication_year=2007&amp;author=Estefan%C3%ADa%2CE">
                    Google Scholar</a> 
                </p></li><li data-counter="21."><p id="ref-CR21">Lines, J. L., Sempere, L. F., Broughton, T., Wang, L. &amp; Noelle, R. VISTA Is a novel broad-spectrum negative checkpoint regulator for cancer immunotherapy. <i>Cancer Immunol. Res.</i> <b>2</b>, 510–517 (2014).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXpsVGgtLw%3D" aria-label="CAS reference 21">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24894088" aria-label="PubMed reference 21">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4085258" aria-label="PubMed Central reference 21">PubMed Central</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://doi.org/10.1158%2F2326-6066.CIR-14-0072" aria-label="Article reference 21">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 21" href="http://scholar.google.com/scholar_lookup?&amp;title=VISTA%20Is%20a%20novel%20broad-spectrum%20negative%20checkpoint%20regulator%20for%20cancer%20immunotherapy&amp;journal=Cancer%20Immunol.%20Res.&amp;volume=2&amp;pages=510-517&amp;publication_year=2014&amp;author=Lines%2CJL&amp;author=Sempere%2CLF&amp;author=Broughton%2CT&amp;author=Wang%2CL&amp;author=Noelle%2CR">
                    Google Scholar</a> 
                </p></li><li data-counter="22."><p id="ref-CR22">Kang, X. et al. Inhibitory leukocyte immunoglobulin-like receptors: immune checkpoint proteins and tumor sustaining factors. <i>Cell Cycle</i> <b>15</b>, 25–40 (2015).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2015ASPC..499...25K" aria-label="ADS reference 22">ADS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4825776" aria-label="PubMed Central reference 22">PubMed Central</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://doi.org/10.1080%2F15384101.2015.1121324" aria-label="Article reference 22">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28Xps1ejsA%3D%3D" aria-label="CAS reference 22">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 22" href="http://scholar.google.com/scholar_lookup?&amp;title=Inhibitory%20leukocyte%20immunoglobulin-like%20receptors%3A%20immune%20checkpoint%20proteins%20and%20tumor%20sustaining%20factors&amp;journal=Cell%20Cycle&amp;volume=15&amp;pages=25-40&amp;publication_year=2015&amp;author=Kang%2CX">
                    Google Scholar</a> 
                </p></li><li data-counter="23."><p id="ref-CR23">van der Merwe, P. A. &amp; Barclay, A. N. Transient intercellular adhesion: the importance of weak protein-protein interactions. <i>Trends Biochem. Sci.</i> <b>19</b>, 354–358 (1994).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=7985226" aria-label="PubMed reference 23">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://doi.org/10.1016%2F0968-0004%2894%2990109-0" aria-label="Article reference 23">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 23" href="http://scholar.google.com/scholar_lookup?&amp;title=Transient%20intercellular%20adhesion%3A%20the%20importance%20of%20weak%20protein-protein%20interactions&amp;journal=Trends%20Biochem.%20Sci.&amp;volume=19&amp;pages=354-358&amp;publication_year=1994&amp;author=Merwe%2CPA&amp;author=Barclay%2CAN">
                    Google Scholar</a> 
                </p></li><li data-counter="24."><p id="ref-CR24">Wright, G. J. Signal initiation in biological systems: the properties and detection of transient extracellular protein interactions. <i>Mol. Biosyst.</i> <b>5</b>, 1405–1412 (2009).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXhtl2ks7zK" aria-label="CAS reference 24">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19593473" aria-label="PubMed reference 24">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898632" aria-label="PubMed Central reference 24">PubMed Central</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://doi.org/10.1039%2Fb903580j" aria-label="Article reference 24">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 24" href="http://scholar.google.com/scholar_lookup?&amp;title=Signal%20initiation%20in%20biological%20systems%3A%20the%20properties%20and%20detection%20of%20transient%20extracellular%20protein%20interactions&amp;journal=Mol.%20Biosyst.&amp;volume=5&amp;pages=1405-1412&amp;publication_year=2009&amp;author=Wright%2CGJ">
                    Google Scholar</a> 
                </p></li><li data-counter="25."><p id="ref-CR25">Vladimer, G. I. et al. Global survey of the immunomodulatory potential of common drugs. <i>Nat. Chem. Biol.</i> <b>13</b>, 681–690 (2017).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXmvVCltbg%3D" aria-label="CAS reference 25">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28437395" aria-label="PubMed reference 25">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5438060" aria-label="PubMed Central reference 25">PubMed Central</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://doi.org/10.1038%2Fnchembio.2360" aria-label="Article reference 25">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 25" href="http://scholar.google.com/scholar_lookup?&amp;title=Global%20survey%20of%20the%20immunomodulatory%20potential%20of%20common%20drugs&amp;journal=Nat.%20Chem.%20Biol.&amp;volume=13&amp;pages=681-690&amp;publication_year=2017&amp;author=Vladimer%2CGI">
                    Google Scholar</a> 
                </p></li><li data-counter="26."><p id="ref-CR26">Browaeys, R., Saelens, W. &amp; Saeys, Y. NicheNet: modeling intercellular communication by linking ligands to target genes. <i>Nat. Methods</i> <b>17</b>, 159–162 (2020).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXitlCht7%2FP" aria-label="CAS reference 26">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31819264" aria-label="PubMed reference 26">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://doi.org/10.1038%2Fs41592-019-0667-5" aria-label="Article reference 26">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 26" href="http://scholar.google.com/scholar_lookup?&amp;title=NicheNet%3A%20modeling%20intercellular%20communication%20by%20linking%20ligands%20to%20target%20genes&amp;journal=Nat.%20Methods&amp;volume=17&amp;pages=159-162&amp;publication_year=2020&amp;author=Browaeys%2CR&amp;author=Saelens%2CW&amp;author=Saeys%2CY">
                    Google Scholar</a> 
                </p></li><li data-counter="27."><p id="ref-CR27">Fitzpatrick, E. A., Wang, J. &amp; Strome, S. E. Engineering of Fc multimers as a protein therapy for autoimmune disease. <i>Front. Immunol.</i> <b>11</b>, 496 (2020).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhvVars7zE" aria-label="CAS reference 27">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32269572" aria-label="PubMed reference 27">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7109252" aria-label="PubMed Central reference 27">PubMed Central</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://doi.org/10.3389%2Ffimmu.2020.00496" aria-label="Article reference 27">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 27" href="http://scholar.google.com/scholar_lookup?&amp;title=Engineering%20of%20Fc%20multimers%20as%20a%20protein%20therapy%20for%20autoimmune%20disease&amp;journal=Front.%20Immunol.&amp;volume=11&amp;publication_year=2020&amp;author=Fitzpatrick%2CEA&amp;author=Wang%2CJ&amp;author=Strome%2CSE">
                    Google Scholar</a> 
                </p></li><li data-counter="28."><p id="ref-CR28">Gu, D., Ao, X., Yang, Y., Chen, Z. &amp; Xu, X. Soluble immune checkpoints in cancer: production, function and biological significance. <i>J. Immunother. Cancer</i> <b>6</b>, 132 (2018).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30482248" aria-label="PubMed reference 28">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6260693" aria-label="PubMed Central reference 28">PubMed Central</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://doi.org/10.1186%2Fs40425-018-0449-0" aria-label="Article reference 28">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 28" href="http://scholar.google.com/scholar_lookup?&amp;title=Soluble%20immune%20checkpoints%20in%20cancer%3A%20production%2C%20function%20and%20biological%20significance&amp;journal=J.%20Immunother.%20Cancer&amp;volume=6&amp;publication_year=2018&amp;author=Gu%2CD&amp;author=Ao%2CX&amp;author=Yang%2CY&amp;author=Chen%2CZ&amp;author=Xu%2CX">
                    Google Scholar</a> 
                </p></li><li data-counter="29."><p id="ref-CR29">Severin, Y. et al. Multiplexed high-throughput immune cell imaging reveals molecular health-associated phenotypes. Preprint at <i>bioRxiv</i> <a href="https://doi.org/10.1101/2021.12.03.471105">https://doi.org/10.1101/2021.12.03.471105</a> (2021).</p></li><li data-counter="30."><p id="ref-CR30">Benschop, R., Wei, T. &amp; Na, S. Tumor necrosis factor receptor superfamily member 21: TNFR-related death receptor-6, DR6. <i>Adv. Exp. Med. Biol.</i> <b>647</b>, 186–194 (2009).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXhtF2itLjN" aria-label="CAS reference 30">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19760075" aria-label="PubMed reference 30">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://doi.org/10.1007%2F978-0-387-89520-8_13" aria-label="Article reference 30">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 30" href="http://scholar.google.com/scholar_lookup?&amp;title=Tumor%20necrosis%20factor%20receptor%20superfamily%20member%2021%3A%20TNFR-related%20death%20receptor-6%2C%20DR6&amp;journal=Adv.%20Exp.%20Med.%20Biol.&amp;volume=647&amp;pages=186-194&amp;publication_year=2009&amp;author=Benschop%2CR&amp;author=Wei%2CT&amp;author=Na%2CS">
                    Google Scholar</a> 
                </p></li><li data-counter="31."><p id="ref-CR31">Liu, L. L. et al. Critical role of CD2 co-stimulation in adaptive natural killer cell responses revealed in NKG2C-deficient humans. <i>Cell Rep.</i> <b>15</b>, 1088–1099 (2016).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XmslSjtLk%3D" aria-label="CAS reference 31">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27117418" aria-label="PubMed reference 31">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4858565" aria-label="PubMed Central reference 31">PubMed Central</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://doi.org/10.1016%2Fj.celrep.2016.04.005" aria-label="Article reference 31">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 31" href="http://scholar.google.com/scholar_lookup?&amp;title=Critical%20role%20of%20CD2%20co-stimulation%20in%20adaptive%20natural%20killer%20cell%20responses%20revealed%20in%20NKG2C-deficient%20humans&amp;journal=Cell%20Rep.&amp;volume=15&amp;pages=1088-1099&amp;publication_year=2016&amp;author=Liu%2CLL">
                    Google Scholar</a> 
                </p></li><li data-counter="32."><p id="ref-CR32">Dustin, M. L., Sanders, M. E., Shaw, S. &amp; Springer, T. A. Purified lymphocyte function-associated antigen 3 binds to CD2 and mediates T lymphocyte adhesion. <i>J. Exp. Med.</i> <b>165</b>, 677–692 (1987).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaL2sXhsFCnsrk%3D" aria-label="CAS reference 32">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=3102676" aria-label="PubMed reference 32">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://doi.org/10.1084%2Fjem.165.3.677" aria-label="Article reference 32">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 32" href="http://scholar.google.com/scholar_lookup?&amp;title=Purified%20lymphocyte%20function-associated%20antigen%203%20binds%20to%20CD2%20and%20mediates%20T%20lymphocyte%20adhesion&amp;journal=J.%20Exp.%20Med.&amp;volume=165&amp;pages=677-692&amp;publication_year=1987&amp;author=Dustin%2CML&amp;author=Sanders%2CME&amp;author=Shaw%2CS&amp;author=Springer%2CTA">
                    Google Scholar</a> 
                </p></li><li data-counter="33."><p id="ref-CR33">Hayes, B. H. et al. Macrophages show higher levels of engulfment after disruption of cis interactions between CD47 and the checkpoint receptor SIRPα. <i>J. Cell Sci.</i> <b>133</b>, jcs237800 (2020).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXptlOrt7c%3D" aria-label="CAS reference 33">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31964705" aria-label="PubMed reference 33">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7064788" aria-label="PubMed Central reference 33">PubMed Central</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://doi.org/10.1242%2Fjcs.237800" aria-label="Article reference 33">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 33" href="http://scholar.google.com/scholar_lookup?&amp;title=Macrophages%20show%20higher%20levels%20of%20engulfment%20after%20disruption%20of%20cis%20interactions%20between%20CD47%20and%20the%20checkpoint%20receptor%20SIRP%CE%B1&amp;journal=J.%20Cell%20Sci.&amp;volume=133&amp;publication_year=2020&amp;author=Hayes%2CBH">
                    Google Scholar</a> 
                </p></li><li data-counter="34."><p id="ref-CR34">Braud, V. M. et al. HLA-E binds to natural killer cell receptors CD94/NKG2A, B and C. <i>Nature</i> <b>391</b>, 795–799 (1998).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=1998Natur.391..795B" aria-label="ADS reference 34">ADS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK1cXhtlCjtbo%3D" aria-label="CAS reference 34">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=9486650" aria-label="PubMed reference 34">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://doi.org/10.1038%2F35869" aria-label="Article reference 34">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 34" href="http://scholar.google.com/scholar_lookup?&amp;title=HLA-E%20binds%20to%20natural%20killer%20cell%20receptors%20CD94%2FNKG2A%2C%20B%20and%20C&amp;journal=Nature&amp;volume=391&amp;pages=795-799&amp;publication_year=1998&amp;author=Braud%2CVM">
                    Google Scholar</a> 
                </p></li><li data-counter="35."><p id="ref-CR35">ElTanbouly, M. A. et al. VISTA is a checkpoint regulator for naïve T cell quiescence and peripheral tolerance. <i>Science</i> <b>367</b>, eaay0524 (2020).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhs12ltLk%3D" aria-label="CAS reference 35">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31949051" aria-label="PubMed reference 35">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7391053" aria-label="PubMed Central reference 35">PubMed Central</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://doi.org/10.1126%2Fscience.aay0524" aria-label="Article reference 35">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 35" href="http://scholar.google.com/scholar_lookup?&amp;title=VISTA%20is%20a%20checkpoint%20regulator%20for%20na%C3%AFve%20T%20cell%20quiescence%20and%20peripheral%20tolerance&amp;journal=Science&amp;volume=367&amp;publication_year=2020&amp;author=ElTanbouly%2CMA">
                    Google Scholar</a> 
                </p></li><li data-counter="36."><p id="ref-CR36">Husain, B. et al. A platform for extracellular interactome discovery identifies novel functional binding partners for the immune receptors B7-H3/CD276 and PVR/CD155. <i>Mol. Cell. Proteomics</i> <b>18</b>, 2310–2323 (2019).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31308249" aria-label="PubMed reference 36">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6823854" aria-label="PubMed Central reference 36">PubMed Central</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://doi.org/10.1074%2Fmcp.TIR119.001433" aria-label="Article reference 36">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 36" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20platform%20for%20extracellular%20interactome%20discovery%20identifies%20novel%20functional%20binding%20partners%20for%20the%20immune%20receptors%20B7-H3%2FCD276%20and%20PVR%2FCD155&amp;journal=Mol.%20Cell.%20Proteomics&amp;volume=18&amp;pages=2310-2323&amp;publication_year=2019&amp;author=Husain%2CB">
                    Google Scholar</a> 
                </p></li><li data-counter="37."><p id="ref-CR37">Vento-Tormo, R. et al. Single-cell reconstruction of the early maternal-fetal interface in humans. <i>Nature</i> <b>563</b>, 347–353 (2018).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2018Natur.563..347V" aria-label="ADS reference 37">ADS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXit1SnsbzI" aria-label="CAS reference 37">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30429548" aria-label="PubMed reference 37">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7612850" aria-label="PubMed Central reference 37">PubMed Central</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://doi.org/10.1038%2Fs41586-018-0698-6" aria-label="Article reference 37">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 37" href="http://scholar.google.com/scholar_lookup?&amp;title=Single-cell%20reconstruction%20of%20the%20early%20maternal-fetal%20interface%20in%20humans&amp;journal=Nature&amp;volume=563&amp;pages=347-353&amp;publication_year=2018&amp;author=Vento-Tormo%2CR">
                    Google Scholar</a> 
                </p></li><li data-counter="38."><p id="ref-CR38">Smillie, C. S. et al. Intra- and inter-cellular rewiring of the human colon during ulcerative colitis. <i>Cell</i> <b>178</b>, 714–730 (2019).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhsVOgsL7K" aria-label="CAS reference 38">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31348891" aria-label="PubMed reference 38">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6662628" aria-label="PubMed Central reference 38">PubMed Central</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://doi.org/10.1016%2Fj.cell.2019.06.029" aria-label="Article reference 38">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 38" href="http://scholar.google.com/scholar_lookup?&amp;title=Intra-%20and%20inter-cellular%20rewiring%20of%20the%20human%20colon%20during%20ulcerative%20colitis&amp;journal=Cell&amp;volume=178&amp;pages=714-730&amp;publication_year=2019&amp;author=Smillie%2CCS">
                    Google Scholar</a> 
                </p></li><li data-counter="39."><p id="ref-CR39">Sinai, P. et al. T/B-cell interactions are more transient in response to weak stimuli in SLE-prone mice. <i>Eur. J. Immunol.</i> <b>44</b>, 3522–3531 (2014).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXlt1Ght7w%3D" aria-label="CAS reference 39">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25209945" aria-label="PubMed reference 39">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4261040" aria-label="PubMed Central reference 39">PubMed Central</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://doi.org/10.1002%2Feji.201444602" aria-label="Article reference 39">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 39" href="http://scholar.google.com/scholar_lookup?&amp;title=T%2FB-cell%20interactions%20are%20more%20transient%20in%20response%20to%20weak%20stimuli%20in%20SLE-prone%20mice&amp;journal=Eur.%20J.%20Immunol.&amp;volume=44&amp;pages=3522-3531&amp;publication_year=2014&amp;author=Sinai%2CP">
                    Google Scholar</a> 
                </p></li><li data-counter="40."><p id="ref-CR40">Hens, C., Harush, U., Haber, S., Cohen, R. &amp; Barzel, B. Spatiotemporal signal propagation in complex networks. <i>Nat. Phys.</i> <b>15</b>, 403–412 (2019).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXmtVyisrY%3D" aria-label="CAS reference 40">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://doi.org/10.1038%2Fs41567-018-0409-0" aria-label="Article reference 40">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 40" href="http://scholar.google.com/scholar_lookup?&amp;title=Spatiotemporal%20signal%20propagation%20in%20complex%20networks&amp;journal=Nat.%20Phys.&amp;volume=15&amp;pages=403-412&amp;publication_year=2019&amp;author=Hens%2CC&amp;author=Harush%2CU&amp;author=Haber%2CS&amp;author=Cohen%2CR&amp;author=Barzel%2CB">
                    Google Scholar</a> 
                </p></li><li data-counter="41."><p id="ref-CR41">Kumar, R., Clermont, G., Vodovotz, Y. &amp; Chow, C. C. The dynamics of acute inflammation. <i>J. Theor. Biol.</i> <b>230</b>, 145–155 (2004).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2004JThBi.230..145K" aria-label="ADS reference 41">ADS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ams.org/mathscinet-getitem?mr=2089074" aria-label="MathSciNet reference 41">MathSciNet</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2cXmsVCrtbY%3D" aria-label="CAS reference 41">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15321710" aria-label="PubMed reference 41">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.emis.de/MATH-item?1447.92093" aria-label="MATH reference 41">MATH</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://doi.org/10.1016%2Fj.jtbi.2004.04.044" aria-label="Article reference 41">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 41" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20dynamics%20of%20acute%20inflammation&amp;journal=J.%20Theor.%20Biol.&amp;volume=230&amp;pages=145-155&amp;publication_year=2004&amp;author=Kumar%2CR&amp;author=Clermont%2CG&amp;author=Vodovotz%2CY&amp;author=Chow%2CCC">
                    Google Scholar</a> 
                </p></li><li data-counter="42."><p id="ref-CR42">Semmrich, M. et al. Importance of integrin LFA-1 deactivation for the generation of immune responses. <i>J. Exp. Med.</i> <b>201</b>, 1987–1998 (2005).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2MXls1Wiurw%3D" aria-label="CAS reference 42">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15955836" aria-label="PubMed reference 42">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2212031" aria-label="PubMed Central reference 42">PubMed Central</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://doi.org/10.1084%2Fjem.20041850" aria-label="Article reference 42">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 42" href="http://scholar.google.com/scholar_lookup?&amp;title=Importance%20of%20integrin%20LFA-1%20deactivation%20for%20the%20generation%20of%20immune%20responses&amp;journal=J.%20Exp.%20Med.&amp;volume=201&amp;pages=1987-1998&amp;publication_year=2005&amp;author=Semmrich%2CM">
                    Google Scholar</a> 
                </p></li><li data-counter="43."><p id="ref-CR43">MacPherson, M., Lek, H. S., Prescott, A. &amp; Fagerholm, S. C. A systemic lupus erythematosus-associated R77H substitution in the CD11b chain of the Mac-1 integrin compromises leukocyte adhesion and phagocytosis. <i>J. Biol. Chem.</i> <b>286</b>, 17303–17310 (2011).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXlslKksb0%3D" aria-label="CAS reference 43">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21454473" aria-label="PubMed reference 43">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3089572" aria-label="PubMed Central reference 43">PubMed Central</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://doi.org/10.1074%2Fjbc.M110.182998" aria-label="Article reference 43">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 43" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20systemic%20lupus%20erythematosus-associated%20R77H%20substitution%20in%20the%20CD11b%20chain%20of%20the%20Mac-1%20integrin%20compromises%20leukocyte%20adhesion%20and%20phagocytosis&amp;journal=J.%20Biol.%20Chem.&amp;volume=286&amp;pages=17303-17310&amp;publication_year=2011&amp;author=MacPherson%2CM&amp;author=Lek%2CHS&amp;author=Prescott%2CA&amp;author=Fagerholm%2CSC">
                    Google Scholar</a> 
                </p></li><li data-counter="44."><p id="ref-CR44">Faridi, M. H. et al. CD11b activation suppresses TLR-dependent inflammation and autoimmunity in systemic lupus erythematosus. <i>J. Clin. Invest.</i> <b>127</b>, 1271–1283 (2017).</p></li><li data-counter="45."><p id="ref-CR45">Dustin, M. L., Chakraborty, A. K. &amp; Shaw, A. S. Understanding the structure and function of the immunological synapse. <i>Cold Spring Harb. Perspect. Biol.</i> <b>2</b>, a002311 (2010).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXhtlWgtr3M" aria-label="CAS reference 45">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20843980" aria-label="PubMed reference 45">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2944359" aria-label="PubMed Central reference 45">PubMed Central</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://doi.org/10.1101%2Fcshperspect.a002311" aria-label="Article reference 45">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 45" href="http://scholar.google.com/scholar_lookup?&amp;title=Understanding%20the%20structure%20and%20function%20of%20the%20immunological%20synapse&amp;journal=Cold%20Spring%20Harb.%20Perspect.%20Biol.&amp;volume=2&amp;publication_year=2010&amp;author=Dustin%2CML&amp;author=Chakraborty%2CAK&amp;author=Shaw%2CAS">
                    Google Scholar</a> 
                </p></li><li data-counter="46."><p id="ref-CR46">Shilts, J., Crozier, T. W. M., Greenwood, E. J. D., Lehner, P. J. &amp; Wright, G. J. No evidence for basigin/CD147 as a direct SARS-CoV-2 spike binding receptor. <i>Sci. Rep.</i> <b>11</b>, 413 (2021).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhsVeht7k%3D" aria-label="CAS reference 46">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33432067" aria-label="PubMed reference 46">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7801465" aria-label="PubMed Central reference 46">PubMed Central</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://doi.org/10.1038%2Fs41598-020-80464-1" aria-label="Article reference 46">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 46" href="http://scholar.google.com/scholar_lookup?&amp;title=No%20evidence%20for%20basigin%2FCD147%20as%20a%20direct%20SARS-CoV-2%20spike%20binding%20receptor&amp;journal=Sci.%20Rep.&amp;volume=11&amp;publication_year=2021&amp;author=Shilts%2CJ&amp;author=Crozier%2CTWM&amp;author=Greenwood%2CEJD&amp;author=Lehner%2CPJ&amp;author=Wright%2CGJ">
                    Google Scholar</a> 
                </p></li><li data-counter="47."><p id="ref-CR47">Parrott, M. B. &amp; Barry, M. A. Metabolic biotinylation of secreted and cell surface proteins from mammalian cells. <i>Biochem. Biophys. Res. Commun.</i> <b>281</b>, 993–1000 (2001).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3MXhs1Churg%3D" aria-label="CAS reference 47">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=11237761" aria-label="PubMed reference 47">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://doi.org/10.1006%2Fbbrc.2001.4437" aria-label="Article reference 47">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 47" href="http://scholar.google.com/scholar_lookup?&amp;title=Metabolic%20biotinylation%20of%20secreted%20and%20cell%20surface%20proteins%20from%20mammalian%20cells&amp;journal=Biochem.%20Biophys.%20Res.%20Commun.&amp;volume=281&amp;pages=993-1000&amp;publication_year=2001&amp;author=Parrott%2CMB&amp;author=Barry%2CMA">
                    Google Scholar</a> 
                </p></li><li data-counter="48."><p id="ref-CR48">Loignon, M. et al. Stable high volumetric production of glycosylated human recombinant IFNα2b in HEK293 cells. <i>BMC Biotechnol.</i> <b>8</b>, 65 (2008).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18752669" aria-label="PubMed reference 48">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2538527" aria-label="PubMed Central reference 48">PubMed Central</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://doi.org/10.1186%2F1472-6750-8-65" aria-label="Article reference 48">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXhsV2msrvI" aria-label="CAS reference 48">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 48" href="http://scholar.google.com/scholar_lookup?&amp;title=Stable%20high%20volumetric%20production%20of%20glycosylated%20human%20recombinant%20IFN%CE%B12b%20in%20HEK293%20cells&amp;journal=BMC%20Biotechnol.&amp;volume=8&amp;publication_year=2008&amp;author=Loignon%2CM">
                    Google Scholar</a> 
                </p></li><li data-counter="49."><p id="ref-CR49">Sun, Y., Gallagher-Jones, M., Barker, C. &amp; Wright, G. J. A benchmarked protein microarray-based platform for the identification of novel low-affinity extracellular protein interactions. <i>Anal. Biochem.</i> <b>424</b>, 45–53 (2012).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XltVehurw%3D" aria-label="CAS reference 49">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22342946" aria-label="PubMed reference 49">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3325482" aria-label="PubMed Central reference 49">PubMed Central</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://doi.org/10.1016%2Fj.ab.2012.01.034" aria-label="Article reference 49">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 49" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20benchmarked%20protein%20microarray-based%20platform%20for%20the%20identification%20of%20novel%20low-affinity%20extracellular%20protein%20interactions&amp;journal=Anal.%20Biochem.&amp;volume=424&amp;pages=45-53&amp;publication_year=2012&amp;author=Sun%2CY&amp;author=Gallagher-Jones%2CM&amp;author=Barker%2CC&amp;author=Wright%2CGJ">
                    Google Scholar</a> 
                </p></li><li data-counter="50."><p id="ref-CR50">Müller-Sienerth, N. et al. A panel of recombinant proteins from human-infective <i>Plasmodium</i> species for serological surveillance. <i>Malar. J.</i> <b>19</b>, 31 (2020).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31952523" aria-label="PubMed reference 50">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6969409" aria-label="PubMed Central reference 50">PubMed Central</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://doi.org/10.1186%2Fs12936-020-3111-5" aria-label="Article reference 50">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXksVyitrY%3D" aria-label="CAS reference 50">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 50" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20panel%20of%20recombinant%20proteins%20from%20human-infective%20Plasmodium%20species%20for%20serological%20surveillance&amp;journal=Malar.%20J.&amp;volume=19&amp;publication_year=2020&amp;author=M%C3%BCller-Sienerth%2CN">
                    Google Scholar</a> 
                </p></li><li data-counter="51."><p id="ref-CR51">Staudt, N., Müller-Sienerth, N. &amp; Wright, G. J. Development of an antigen microarray for high throughput monoclonal antibody selection. <i>Biochem. Biophys. Res. Commun.</i> <b>445</b>, 785–790 (2014).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXitlensLg%3D" aria-label="CAS reference 51">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24472540" aria-label="PubMed reference 51">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3989025" aria-label="PubMed Central reference 51">PubMed Central</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://doi.org/10.1016%2Fj.bbrc.2013.12.033" aria-label="Article reference 51">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 51" href="http://scholar.google.com/scholar_lookup?&amp;title=Development%20of%20an%20antigen%20microarray%20for%20high%20throughput%20monoclonal%20antibody%20selection&amp;journal=Biochem.%20Biophys.%20Res.%20Commun.&amp;volume=445&amp;pages=785-790&amp;publication_year=2014&amp;author=Staudt%2CN&amp;author=M%C3%BCller-Sienerth%2CN&amp;author=Wright%2CGJ">
                    Google Scholar</a> 
                </p></li><li data-counter="52."><p id="ref-CR52">Taechalertpaisarn, T. et al. Biochemical and functional analysis of two <i>Plasmodium falciparum</i> blood-stage 6-cys proteins: P12 and P41. <i>PLoS One</i> <b>7</b>, e41937 (2012).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2012PLoSO...741937T" aria-label="ADS reference 52">ADS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XhtFCgur3O" aria-label="CAS reference 52">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22848665" aria-label="PubMed reference 52">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3407074" aria-label="PubMed Central reference 52">PubMed Central</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://doi.org/10.1371%2Fjournal.pone.0041937" aria-label="Article reference 52">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 52" href="http://scholar.google.com/scholar_lookup?&amp;title=Biochemical%20and%20functional%20analysis%20of%20two%20Plasmodium%20falciparum%20blood-stage%206-cys%20proteins%3A%20P12%20and%20P41&amp;journal=PLoS%20One&amp;volume=7&amp;publication_year=2012&amp;author=Taechalertpaisarn%2CT">
                    Google Scholar</a> 
                </p></li><li data-counter="53."><p id="ref-CR53">Galaway, F., Yu, R., Constantinou, A., Prugnolle, F. &amp; Wright, G. J. Resurrection of the ancestral RH5 invasion ligand provides a molecular explanation for the origin of <i>P. falciparum</i> malaria in humans. <i>PLoS Biol.</i> <b>17</b>, e3000490 (2019).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhsVWjtLg%3D" aria-label="CAS reference 53">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31613878" aria-label="PubMed reference 53">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6793842" aria-label="PubMed Central reference 53">PubMed Central</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://doi.org/10.1371%2Fjournal.pbio.3000490" aria-label="Article reference 53">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 53" href="http://scholar.google.com/scholar_lookup?&amp;title=Resurrection%20of%20the%20ancestral%20RH5%20invasion%20ligand%20provides%20a%20molecular%20explanation%20for%20the%20origin%20of%20P.%20falciparum%20malaria%20in%20humans&amp;journal=PLoS%20Biol.&amp;volume=17&amp;publication_year=2019&amp;author=Galaway%2CF&amp;author=Yu%2CR&amp;author=Constantinou%2CA&amp;author=Prugnolle%2CF&amp;author=Wright%2CGJ">
                    Google Scholar</a> 
                </p></li><li data-counter="54."><p id="ref-CR54">Efremova, M., Vento-Tormo, M., Teichmann, S. A. &amp; Vento-Tormo, R. CellPhoneDB: inferring cell–cell communication from combined expression of multi-subunit ligand-receptor complexes. <i>Nat. Protoc.</i> <b>15</b>, 1484–1506 (2020).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXjvFynsL0%3D" aria-label="CAS reference 54">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32103204" aria-label="PubMed reference 54">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://doi.org/10.1038%2Fs41596-020-0292-x" aria-label="Article reference 54">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 54" href="http://scholar.google.com/scholar_lookup?&amp;title=CellPhoneDB%3A%20inferring%20cell%E2%80%93cell%20communication%20from%20combined%20expression%20of%20multi-subunit%20ligand-receptor%20complexes&amp;journal=Nat.%20Protoc.&amp;volume=15&amp;pages=1484-1506&amp;publication_year=2020&amp;author=Efremova%2CM&amp;author=Vento-Tormo%2CM&amp;author=Teichmann%2CSA&amp;author=Vento-Tormo%2CR">
                    Google Scholar</a> 
                </p></li><li data-counter="55."><p id="ref-CR55">Wood, L. &amp; Wright, G. J. High-content imaging for large-scale detection of low-affinity extracellular protein interactions. <i>SLAS Discov.</i> <b>24</b>, 987–999 (2019).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXit1Sru7nF" aria-label="CAS reference 55">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31578119" aria-label="PubMed reference 55">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6873222" aria-label="PubMed Central reference 55">PubMed Central</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://doi.org/10.1177%2F2472555219879053" aria-label="Article reference 55">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 55" href="http://scholar.google.com/scholar_lookup?&amp;title=High-content%20imaging%20for%20large-scale%20detection%20of%20low-affinity%20extracellular%20protein%20interactions&amp;journal=SLAS%20Discov.&amp;volume=24&amp;pages=987-999&amp;publication_year=2019&amp;author=Wood%2CL&amp;author=Wright%2CGJ">
                    Google Scholar</a> 
                </p></li><li data-counter="56."><p id="ref-CR56">Engel, P. et al. CD nomenclature 2015: human leukocyte differentiation antigen workshops as a driving force in immunology. <i>J. Immunol.</i> <b>195</b>, 4555–4563 (2015).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXhslyit73N" aria-label="CAS reference 56">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26546687" aria-label="PubMed reference 56">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://doi.org/10.4049%2Fjimmunol.1502033" aria-label="Article reference 56">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 56" href="http://scholar.google.com/scholar_lookup?&amp;title=CD%20nomenclature%202015%3A%20human%20leukocyte%20differentiation%20antigen%20workshops%20as%20a%20driving%20force%20in%20immunology&amp;journal=J.%20Immunol.&amp;volume=195&amp;pages=4555-4563&amp;publication_year=2015&amp;author=Engel%2CP">
                    Google Scholar</a> 
                </p></li><li data-counter="57."><p id="ref-CR57">Omasits, U., Ahrens, C. H., Müller, S. &amp; Wollscheid, B. Protter: interactive protein feature visualization and integration with experimental proteomic data. <i>Bioinformatics</i> <b>30</b>, 884–886 (2014).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXks12jsLs%3D" aria-label="CAS reference 57">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24162465" aria-label="PubMed reference 57">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtt607" aria-label="Article reference 57">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 57" href="http://scholar.google.com/scholar_lookup?&amp;title=Protter%3A%20interactive%20protein%20feature%20visualization%20and%20integration%20with%20experimental%20proteomic%20data&amp;journal=Bioinformatics&amp;volume=30&amp;pages=884-886&amp;publication_year=2014&amp;author=Omasits%2CU&amp;author=Ahrens%2CCH&amp;author=M%C3%BCller%2CS&amp;author=Wollscheid%2CB">
                    Google Scholar</a> 
                </p></li><li data-counter="58."><p id="ref-CR58">Sun, Y. et al. A human platelet receptor protein microarray identifies the high affinity immunoglobulin E receptor subunit α (FcεR1α) as an activating platelet endothelium aggregation receptor 1 (PEAR1) ligand. <i>Mol. Cell. Proteomics</i> <b>14</b>, 1265–1274 (2015).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXotVWjtr4%3D" aria-label="CAS reference 58">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25713122" aria-label="PubMed reference 58">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4424398" aria-label="PubMed Central reference 58">PubMed Central</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://doi.org/10.1074%2Fmcp.M114.046946" aria-label="Article reference 58">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 58" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20human%20platelet%20receptor%20protein%20microarray%20identifies%20the%20high%20affinity%20immunoglobulin%20E%20receptor%20subunit%20%CE%B1%20%28Fc%CE%B5R1%CE%B1%29%20as%20an%20activating%20platelet%20endothelium%20aggregation%20receptor%201%20%28PEAR1%29%20ligand&amp;journal=Mol.%20Cell.%20Proteomics&amp;volume=14&amp;pages=1265-1274&amp;publication_year=2015&amp;author=Sun%2CY">
                    Google Scholar</a> 
                </p></li><li data-counter="59."><p id="ref-CR59">Voulgaraki, D. et al. Multivalent recombinant proteins for probing functions of leucocyte surface proteins such as the CD200 receptor. <i>Immunology</i> <b>115</b>, 337–346 (2005).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2MXlvFCms78%3D" aria-label="CAS reference 59">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15946251" aria-label="PubMed reference 59">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1782167" aria-label="PubMed Central reference 59">PubMed Central</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://doi.org/10.1111%2Fj.1365-2567.2005.02161.x" aria-label="Article reference 59">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 59" href="http://scholar.google.com/scholar_lookup?&amp;title=Multivalent%20recombinant%20proteins%20for%20probing%20functions%20of%20leucocyte%20surface%20proteins%20such%20as%20the%20CD200%20receptor&amp;journal=Immunology&amp;volume=115&amp;pages=337-346&amp;publication_year=2005&amp;author=Voulgaraki%2CD">
                    Google Scholar</a> 
                </p></li><li data-counter="60."><p id="ref-CR60">Dundas, K. et al. Alpha-v-containing integrins are host receptors for the <i>Plasmodium falciparum</i> sporozoite surface protein, TRAP. <i>Proc. Natl Acad. Sci. USA</i> <b>115</b>, 4477–4482 (2018).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhvVKjt7%2FI" aria-label="CAS reference 60">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29632205" aria-label="PubMed reference 60">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5924908" aria-label="PubMed Central reference 60">PubMed Central</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://doi.org/10.1073%2Fpnas.1719660115" aria-label="Article reference 60">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 60" href="http://scholar.google.com/scholar_lookup?&amp;title=Alpha-v-containing%20integrins%20are%20host%20receptors%20for%20the%20Plasmodium%20falciparum%20sporozoite%20surface%20protein%2C%20TRAP&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=115&amp;pages=4477-4482&amp;publication_year=2018&amp;author=Dundas%2CK">
                    Google Scholar</a> 
                </p></li><li data-counter="61."><p id="ref-CR61">Zola, H. et al. <i>Leukocyte and Stromal Cell Molecules: the CD Markers</i> (Wiley-Liss, 2007).</p></li><li data-counter="62."><p id="ref-CR62">Barclay, A. N. et al. <i>The Leucocyte Antigen Factsbook</i> 2nd edn (Academic Press, 1997).</p></li><li data-counter="63."><p id="ref-CR63">Ramilowski, J. A. et al. A draft network of ligand-receptor-mediated multicellular signalling in human. <i>Nat. Commun.</i> <b>6</b>, 7866 (2015).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2015NatCo...6.7866R" aria-label="ADS reference 63">ADS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXhtlWhu7%2FM" aria-label="CAS reference 63">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26198319" aria-label="PubMed reference 63">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://doi.org/10.1038%2Fncomms8866" aria-label="Article reference 63">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 63" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20draft%20network%20of%20ligand-receptor-mediated%20multicellular%20signalling%20in%20human&amp;journal=Nat.%20Commun.&amp;volume=6&amp;publication_year=2015&amp;author=Ramilowski%2CJA">
                    Google Scholar</a> 
                </p></li><li data-counter="64."><p id="ref-CR64">Teku, G. N., Ortutay, C. &amp; Vihinen, M. Identification of core T cell network based on immunome interactome. <i>BMC Syst. Biol.</i> <b>8</b>, 17 (2014).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24528953" aria-label="PubMed reference 64">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3937033" aria-label="PubMed Central reference 64">PubMed Central</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://doi.org/10.1186%2F1752-0509-8-17" aria-label="Article reference 64">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXptVOhsLo%3D" aria-label="CAS reference 64">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 64" href="http://scholar.google.com/scholar_lookup?&amp;title=Identification%20of%20core%20T%20cell%20network%20based%20on%20immunome%20interactome&amp;journal=BMC%20Syst.%20Biol.&amp;volume=8&amp;publication_year=2014&amp;author=Teku%2CGN&amp;author=Ortutay%2CC&amp;author=Vihinen%2CM">
                    Google Scholar</a> 
                </p></li><li data-counter="65."><p id="ref-CR65">Armingol, E., Officer, A., Harismendy, O. &amp; Lewis, N. E. Deciphering cell–cell interactions and communication from gene expression. <i>Nat. Rev. Genet.</i> <b>22</b>, 71–88 (2021).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXit1yht7vJ" aria-label="CAS reference 65">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33168968" aria-label="PubMed reference 65">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://doi.org/10.1038%2Fs41576-020-00292-x" aria-label="Article reference 65">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 65" href="http://scholar.google.com/scholar_lookup?&amp;title=Deciphering%20cell%E2%80%93cell%20interactions%20and%20communication%20from%20gene%20expression&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=22&amp;pages=71-88&amp;publication_year=2021&amp;author=Armingol%2CE&amp;author=Officer%2CA&amp;author=Harismendy%2CO&amp;author=Lewis%2CNE">
                    Google Scholar</a> 
                </p></li><li data-counter="66."><p id="ref-CR66">Luecken, M. D. &amp; Theis, F. J. Current best practices in single-cell RNA-seq analysis: a tutorial. <i>Mol. Syst. Biol.</i> <b>15</b>, e8746 (2019).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31217225" aria-label="PubMed reference 66">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6582955" aria-label="PubMed Central reference 66">PubMed Central</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://doi.org/10.15252%2Fmsb.20188746" aria-label="Article reference 66">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 66" href="http://scholar.google.com/scholar_lookup?&amp;title=Current%20best%20practices%20in%20single-cell%20RNA-seq%20analysis%3A%20a%20tutorial&amp;journal=Mol.%20Syst.%20Biol.&amp;volume=15&amp;publication_year=2019&amp;author=Luecken%2CMD&amp;author=Theis%2CFJ">
                    Google Scholar</a> 
                </p></li><li data-counter="67."><p id="ref-CR67">Zheng, G. X. Y. et al. Massively parallel digital transcriptional profiling of single cells. <i>Nat. Commun.</i> <b>8</b>, 14049 (2017).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2017NatCo...814049Z" aria-label="ADS reference 67">ADS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXht1WlsLo%3D" aria-label="CAS reference 67">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28091601" aria-label="PubMed reference 67">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5241818" aria-label="PubMed Central reference 67">PubMed Central</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://doi.org/10.1038%2Fncomms14049" aria-label="Article reference 67">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 67" href="http://scholar.google.com/scholar_lookup?&amp;title=Massively%20parallel%20digital%20transcriptional%20profiling%20of%20single%20cells&amp;journal=Nat.%20Commun.&amp;volume=8&amp;publication_year=2017&amp;author=Zheng%2CGXY">
                    Google Scholar</a> 
                </p></li><li data-counter="68."><p id="ref-CR68">Satija, R., Farrell, J. A., Gennert, D., Schier, A. F. &amp; Regev, A. Spatial reconstruction of single-cell gene expression data. <i>Nat. Biotechnol.</i> <b>33</b>, 495–502 (2015).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXmtlKktLo%3D" aria-label="CAS reference 68">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25867923" aria-label="PubMed reference 68">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4430369" aria-label="PubMed Central reference 68">PubMed Central</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://doi.org/10.1038%2Fnbt.3192" aria-label="Article reference 68">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 68" href="http://scholar.google.com/scholar_lookup?&amp;title=Spatial%20reconstruction%20of%20single-cell%20gene%20expression%20data&amp;journal=Nat.%20Biotechnol.&amp;volume=33&amp;pages=495-502&amp;publication_year=2015&amp;author=Satija%2CR&amp;author=Farrell%2CJA&amp;author=Gennert%2CD&amp;author=Schier%2CAF&amp;author=Regev%2CA">
                    Google Scholar</a> 
                </p></li><li data-counter="69."><p id="ref-CR69">Stuart, T. et al. Comprehensive integration of single-cell data. <i>Cell</i> <b>177</b>, 1888–1902 (2019).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhtFens77L" aria-label="CAS reference 69">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31178118" aria-label="PubMed reference 69">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6687398" aria-label="PubMed Central reference 69">PubMed Central</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://doi.org/10.1016%2Fj.cell.2019.05.031" aria-label="Article reference 69">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 69" href="http://scholar.google.com/scholar_lookup?&amp;title=Comprehensive%20integration%20of%20single-cell%20data&amp;journal=Cell&amp;volume=177&amp;pages=1888-1902&amp;publication_year=2019&amp;author=Stuart%2CT">
                    Google Scholar</a> 
                </p></li><li data-counter="70."><p id="ref-CR70">Love, M. I., Huber, W. &amp; Anders, S. Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. <i>Genome Biol.</i> <b>15</b>, 550 (2014).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25516281" aria-label="PubMed reference 70">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4302049" aria-label="PubMed Central reference 70">PubMed Central</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://doi.org/10.1186%2Fs13059-014-0550-8" aria-label="Article reference 70">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXjtVCrsL8%3D" aria-label="CAS reference 70">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 70" href="http://scholar.google.com/scholar_lookup?&amp;title=Moderated%20estimation%20of%20fold%20change%20and%20dispersion%20for%20RNA-seq%20data%20with%20DESeq2&amp;journal=Genome%20Biol.&amp;volume=15&amp;publication_year=2014&amp;author=Love%2CMI&amp;author=Huber%2CW&amp;author=Anders%2CS">
                    Google Scholar</a> 
                </p></li><li data-counter="71."><p id="ref-CR71">Bell, G. I. Models for the specific adhesion of cells to cells. <i>Science</i> <b>200</b>, 618–627 (1978).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=1978Sci...200..618B" aria-label="ADS reference 71">ADS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaE1cXktlyns7Y%3D" aria-label="CAS reference 71">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=347575" aria-label="PubMed reference 71">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://doi.org/10.1126%2Fscience.347575" aria-label="Article reference 71">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 71" href="http://scholar.google.com/scholar_lookup?&amp;title=Models%20for%20the%20specific%20adhesion%20of%20cells%20to%20cells&amp;journal=Science&amp;volume=200&amp;pages=618-627&amp;publication_year=1978&amp;author=Bell%2CGI">
                    Google Scholar</a> 
                </p></li><li data-counter="72."><p id="ref-CR72">Lopez, C. F., Muhlich, J. L., Bachman, J. A. &amp; Sorger, P. K. Programming biological models in Python using PySB. <i>Mol. Syst. Biol.</i> <b>9</b>, 646 (2013).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23423320" aria-label="PubMed reference 72">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3588907" aria-label="PubMed Central reference 72">PubMed Central</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://doi.org/10.1038%2Fmsb.2013.1" aria-label="Article reference 72">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 72" href="http://scholar.google.com/scholar_lookup?&amp;title=Programming%20biological%20models%20in%20Python%20using%20PySB&amp;journal=Mol.%20Syst.%20Biol.&amp;volume=9&amp;publication_year=2013&amp;author=Lopez%2CCF&amp;author=Muhlich%2CJL&amp;author=Bachman%2CJA&amp;author=Sorger%2CPK">
                    Google Scholar</a> 
                </p></li><li data-counter="73."><p id="ref-CR73">Carpenter, A. E. et al. CellProfiler: image analysis software for identifying and quantifying cell phenotypes. <i>Genome Biol.</i> <b>7</b>, R100 (2006).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17076895" aria-label="PubMed reference 73">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1794559" aria-label="PubMed Central reference 73">PubMed Central</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://doi.org/10.1186%2Fgb-2006-7-10-r100" aria-label="Article reference 73">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD28Xht1eitr7E" aria-label="CAS reference 73">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 73" href="http://scholar.google.com/scholar_lookup?&amp;title=CellProfiler%3A%20image%20analysis%20software%20for%20identifying%20and%20quantifying%20cell%20phenotypes&amp;journal=Genome%20Biol.&amp;volume=7&amp;publication_year=2006&amp;author=Carpenter%2CAE">
                    Google Scholar</a> 
                </p></li><li data-counter="74."><p id="ref-CR74">He, K., Zhang, X., Ren, S. &amp; Sun, J. Deep residual learning for image recognition. In <i>2016 IEEE Conference on Computer Vision and Pattern Recognition (CVPR)</i> (eds. Agapito, L. et al.) 770–778 (IEEE, 2016).</p></li><li data-counter="75."><p id="ref-CR75">Lee, N., Ishitani, A. &amp; Geraghty, D. E. HLA-F is a surface marker on activated lymphocytes. <i>Eur. J. Immunol.</i> <b>40</b>, 2308–2318 (2010).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXpsF2rsb0%3D" aria-label="CAS reference 75">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20865824" aria-label="PubMed reference 75">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3867582" aria-label="PubMed Central reference 75">PubMed Central</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://doi.org/10.1002%2Feji.201040348" aria-label="Article reference 75">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 75" href="http://scholar.google.com/scholar_lookup?&amp;title=HLA-F%20is%20a%20surface%20marker%20on%20activated%20lymphocytes&amp;journal=Eur.%20J.%20Immunol.&amp;volume=40&amp;pages=2308-2318&amp;publication_year=2010&amp;author=Lee%2CN&amp;author=Ishitani%2CA&amp;author=Geraghty%2CDE">
                    Google Scholar</a> 
                </p></li><li data-counter="76."><p id="ref-CR76">Wainwright, S. D., Biro, P. A. &amp; Holmes, C. H. HLA-F is a predominantly empty, intracellular, TAP-associated MHC class Ib protein with a restricted expression pattern. <i>J. Immunol.</i> <b>164</b>, 319–328 (2000).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3cXisFSg" aria-label="CAS reference 76">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=10605026" aria-label="PubMed reference 76">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://doi.org/10.4049%2Fjimmunol.164.1.319" aria-label="Article reference 76">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 76" href="http://scholar.google.com/scholar_lookup?&amp;title=HLA-F%20is%20a%20predominantly%20empty%2C%20intracellular%2C%20TAP-associated%20MHC%20class%20Ib%20protein%20with%20a%20restricted%20expression%20pattern&amp;journal=J.%20Immunol.&amp;volume=164&amp;pages=319-328&amp;publication_year=2000&amp;author=Wainwright%2CSD&amp;author=Biro%2CPA&amp;author=Holmes%2CCH">
                    Google Scholar</a> 
                </p></li><li data-counter="77."><p id="ref-CR77">Reinhard, C., Borgers, M., David, G. &amp; De Strooper, B. Soluble amyloid-β precursor protein binds its cell surface receptor in a cooperative fashion with glypican and syndecan proteoglycans. <i>J. Cell Sci.</i> <b>126</b>, 4856–4861 (2013).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXhvVygsLzN" aria-label="CAS reference 77">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23986479" aria-label="PubMed reference 77">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 77" href="http://scholar.google.com/scholar_lookup?&amp;title=Soluble%20amyloid-%CE%B2%20precursor%20protein%20binds%20its%20cell%20surface%20receptor%20in%20a%20cooperative%20fashion%20with%20glypican%20and%20syndecan%20proteoglycans&amp;journal=J.%20Cell%20Sci.&amp;volume=126&amp;pages=4856-4861&amp;publication_year=2013&amp;author=Reinhard%2CC&amp;author=Borgers%2CM&amp;author=David%2CG&amp;author=Strooper%2CB">
                    Google Scholar</a> 
                </p></li><li data-counter="78."><p id="ref-CR78">Rush, J. S. &amp; Hodgkin, P. D. B cells activated via CD40 and IL-4 undergo a division burst but require continued stimulation to maintain division, survival and differentiation. <i>Eur. J. Immunol.</i> <b>31</b>, 1150–1159 (2001).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3MXivFyjs74%3D" aria-label="CAS reference 78">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=11298340" aria-label="PubMed reference 78">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://doi.org/10.1002%2F1521-4141%28200104%2931%3A4%3C1150%3A%3AAID-IMMU1150%3E3.0.CO%3B2-V" aria-label="Article reference 78">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 78" href="http://scholar.google.com/scholar_lookup?&amp;title=B%20cells%20activated%20via%20CD40%20and%20IL-4%20undergo%20a%20division%20burst%20but%20require%20continued%20stimulation%20to%20maintain%20division%2C%20survival%20and%20differentiation&amp;journal=Eur.%20J.%20Immunol.&amp;volume=31&amp;pages=1150-1159&amp;publication_year=2001&amp;author=Rush%2CJS&amp;author=Hodgkin%2CPD">
                    Google Scholar</a> 
                </p></li><li data-counter="79."><p id="ref-CR79">Dienz, O. et al. The induction of antibody production by IL-6 is indirectly mediated by IL-21 produced by CD4<sup>+</sup> T cells. <i>J. Exp. Med.</i> <b>206</b>, 69–78 (2009).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXhtVOhtbk%3D" aria-label="CAS reference 79">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19139170" aria-label="PubMed reference 79">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2626667" aria-label="PubMed Central reference 79">PubMed Central</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://doi.org/10.1084%2Fjem.20081571" aria-label="Article reference 79">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 79" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20induction%20of%20antibody%20production%20by%20IL-6%20is%20indirectly%20mediated%20by%20IL-21%20produced%20by%20CD4%2B%20T%20cells&amp;journal=J.%20Exp.%20Med.&amp;volume=206&amp;pages=69-78&amp;publication_year=2009&amp;author=Dienz%2CO">
                    Google Scholar</a> 
                </p></li><li data-counter="80."><p id="ref-CR80">Armitage, R. J., Macduff, B. M., Eisenman, J., Paxton, R. &amp; Grabstein, K. H. IL-15 has stimulatory activity for the induction of B cell proliferation and differentiation. <i>J. Immunol.</i> <b>154</b>, 483–490 (1995).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK2MXivFyqs7s%3D" aria-label="CAS reference 80">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=7814861" aria-label="PubMed reference 80">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 80" href="http://scholar.google.com/scholar_lookup?&amp;title=IL-15%20has%20stimulatory%20activity%20for%20the%20induction%20of%20B%20cell%20proliferation%20and%20differentiation&amp;journal=J.%20Immunol.&amp;volume=154&amp;pages=483-490&amp;publication_year=1995&amp;author=Armitage%2CRJ&amp;author=Macduff%2CBM&amp;author=Eisenman%2CJ&amp;author=Paxton%2CR&amp;author=Grabstein%2CKH">
                    Google Scholar</a> 
                </p></li><li data-counter="81."><p id="ref-CR81">Santana Carrero, R. M. et al. IL-15 is a component of the inflammatory milieu in the tumor microenvironment promoting antitumor responses. <i>Proc. Natl Acad. Sci. USA</i> <b>116</b>, 599–608 (2019).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXlvVKrsw%3D%3D" aria-label="CAS reference 81">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30587590" aria-label="PubMed reference 81">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://doi.org/10.1073%2Fpnas.1814642116" aria-label="Article reference 81">Article</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 81" href="http://scholar.google.com/scholar_lookup?&amp;title=IL-15%20is%20a%20component%20of%20the%20inflammatory%20milieu%20in%20the%20tumor%20microenvironment%20promoting%20antitumor%20responses&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=116&amp;pages=599-608&amp;publication_year=2019&amp;author=Santana%20Carrero%2CRM">
                    Google Scholar</a> 
                </p></li><li data-counter="82."><p id="ref-CR82">Holbrook, N. J., Cox, W. I. &amp; Horner, H. C. Direct suppression of natural killer activity in human peripheral blood leukocyte cultures by glucocorticoids and its modulation by interferon. <i>Cancer Res.</i> <b>43</b>, 4019–4025 (1983).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaL3sXlsFant7k%3D" aria-label="CAS reference 82">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=6871844" aria-label="PubMed reference 82">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 82" href="http://scholar.google.com/scholar_lookup?&amp;title=Direct%20suppression%20of%20natural%20killer%20activity%20in%20human%20peripheral%20blood%20leukocyte%20cultures%20by%20glucocorticoids%20and%20its%20modulation%20by%20interferon&amp;journal=Cancer%20Res.&amp;volume=43&amp;pages=4019-4025&amp;publication_year=1983&amp;author=Holbrook%2CNJ&amp;author=Cox%2CWI&amp;author=Horner%2CHC">
                    Google Scholar</a> 
                </p></li><li data-counter="83."><p id="ref-CR83">Calderon, D. et al. Landscape of stimulation-responsive chromatin across diverse human immune cells. <i>Nat. Genet.</i> <b>51</b>, 1494–1505 (2019).</p></li></ol><p><a data-track="click" data-track-action="download citation references" data-track-label="link" href="https://citation-needed.springer.com/v2/references/10.1038/s41586-022-05028-x?format=refman&amp;flavour=references">Download references<svg width="16" height="16" focusable="false" role="img" aria-hidden="true"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#global-icon-download"></use></svg></a></p></div></div></div></section></div><section data-title="Acknowledgements"><div id="Ack1-section"><h2 id="Ack1">Acknowledgements</h2><p>This research was principally funded through the Wellcome Trust (grant 206194). Y.S. and B.S. were funded through the Swiss National Science Foundation (grant PP00P3_163961). We thank L. Parts for discussions on mathematical modelling; A. Wilbrey-Clark for coordinating in situ hybridization experiments; I. Bronner for assistance with robotics; and M. Quail and J. Parkhill for access to laboratory facilities.</p></div></section><section aria-labelledby="author-information" data-title="Author information"><div id="author-information-section"><h2 id="author-information">Author information</h2><div id="author-information-content"><h3 id="affiliations">Authors and Affiliations</h3><ol><li id="Aff1"><p>Cell Surface Signalling Laboratory, Wellcome Sanger Institute, Cambridge, UK</p><p>Jarrod Shilts, Francis Galaway, Nicole Müller-Sienerth, Zheng-Shan Chong &amp; Gavin J. Wright</p></li><li id="Aff2"><p>Institute of Molecular Systems Biology, ETH Zurich, Zurich, Switzerland</p><p>Yannik Severin &amp; Berend Snijder</p></li><li id="Aff3"><p>Cellular Genetics Programme, Wellcome Sanger Institute, Cambridge, UK</p><p>Sophie Pritchard, Sarah Teichmann &amp; Roser Vento-Tormo</p></li><li id="Aff4"><p>Department of Biology, Hull York Medical School, York Biomedical Research Institute, University of York, York, UK</p><p>Gavin J. Wright</p></li></ol><h3 id="contributions">Contributions</h3><p>J.S. and G.J.W. designed the study. J.S. developed the SAVEXIS technique. J.S. prepared all recombinant proteins, with assistance from N.M.-S. during screening. J.S., N.M.-S. and Z.-S.C. conducted the receptor interactome screen measurements. J.S. analysed screen data. J.S. performed cell-based binding assays. F.G. performed SPR. J.S. constructed the mathematical model of cell binding. S.P. performed in situ hybridization experiments. J.S. processed and analysed all transcriptome expression datasets. S.T. and R.V.-T. supervised expression analysis. Y.S. performed pharmacoscopy assays. Y.S. and J.S. analysed the imaging data. B.S. supervised the pharmacoscopy analysis. G.J.W. supervised all other experiments. J.S. wrote the manuscript with comments from all authors.</p><h3 id="corresponding-author">Corresponding authors</h3><p id="corresponding-author-list">Correspondence to
                <a id="corresp-c1" href="mailto:js44@sanger.ac.uk">Jarrod Shilts</a> or <a id="corresp-c2" href="mailto:gavin.wright@york.ac.uk">Gavin J. Wright</a>.</p></div></div></section><section data-title="Ethics declarations"><div id="ethics-section"><h2 id="ethics">Ethics declarations</h2><div id="ethics-content">
            
              <h3 id="FPar2">Competing interests</h3>
              <p>S.T. has received remuneration in the last three years for consulting and membership of scientific advisory boards from Foresite Labs, GlaxoSmithKline, Biogen, Qiagen and Transition Bio, and is an equity holder of Transition Bio. The remaining authors declare no competing interests.</p>
            
          </div></div></section><section data-title="Peer review"><div id="peer-review-section"><h2 id="peer-review">Peer review</h2><div id="peer-review-content">
            
            
              <h3 id="FPar1">Peer review information</h3>
              <p><i>Nature</i> thanks Igor Stagljar, Ulrich Stelzl and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41586-022-05028-x#MOESM5">Peer reviewer reports</a> are available.</p>
            
          </div></div></section><section data-title="Additional information"><div id="additional-information-section"><h2 id="additional-information">Additional information</h2><p><b>Publisher’s note</b> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div></section><section data-title="Extended data figures and tables"><div id="Sec40-section"><h2 id="Sec40">Extended data figures and tables</h2><div id="Sec40-content"><div data-test="supplementary-info"><div data-test="supp-item" id="Fig5"><h3><a data-track="click" data-track-action="view supplementary info" data-track-label="link" data-test="supp-info-link" href="https://www.nature.com/articles/s41586-022-05028-x/figures/5" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41586-022-05028-x/MediaObjects/41586_2022_5028_Fig5_ESM.jpg">Extended Data Fig. 1 Developing SAVEXIS.</a></h3><p><b>a</b>. Design of expression vectors and recombinant protein constructs for SAVEXIS screening of divergent architectural classes of cell-surface proteins. For heterodimers, the exact formulation of each chain will depend on the receptor subunit’s topology (e.g. using the Type I vector for integrins, and Type II for CD94/NKG2). <b>b</b>. Empirically gauging streptavidin multimerization stoichiometry by ELISA. Schematic of the procedure for measuring tetramerization around streptavidin by titrating soluble streptavidin (SAV) against a fixed concentration of biotinylated protein before transferring to a streptavidin-coated plate for an ELISA. The measured dilution at which signal ceases represents the optimal tetramerization stoichiometry. <b>c</b>. Calculated stoichiometric equivalence points of 6 example biotinylated proteins incubated with streptavidin. The 4:1 stoichiometric equivalence point inferred mathematically based on molecular mass calculations is indicated on the x axis as “1”. A dashed line indicated the empirically measured median equivalence point. A set of 6 <i>Danio rerio</i> Jam proteins were measured, with the average indicated by a dashed line. <b>d</b>. Titrating prey concentrations identifies a common prey activity with optimal sensitivity. For 4 known zebrafish receptor–ligand interactions of varying affinity (colour shades, ordered by known binding affinity expressed as dissociation half-life), the ratio of raw absorbance signal for the specific interaction against non-specific interactions (y axis) is measured across prey concentrations (y axis). Error bars represent the standard error of the mean. n = 6 technical replicates. <b>e</b>. Applying soluble desthiobiotin greatly enhances the assay signal to noise ratio by sealing unoccupied biotin-binding sites. Measurements of a set of 3 example interactions compared without applying any sealing step in between bait and prey incubations (“Original”), applying biotinylated rCD4 tag at either IC50 or plate-saturation concentrations, or applying a molar equivalent of saturation with either biotin or desthiobiotin. The ratio between mean signal to noise is indicated above each condition. Error bars represent the standard error of the mean. n ≥ 5 independent wells. <b>f</b>. Immobilizing lower quantities of bait protein can reduce off-target signals. Example interaction assays with baits saturating each well (top) or baits at half-maximal dilutions (bottom). A dilution series of prey is applied across each column. The top two rows are specific interactions (binding half-lives listed in green), and the bottom two are known non-interactors. <b>g</b>. Assay miniaturization into 384-well plates can retain and enhance performance. Different assay miniaturization strategies to adapt from 96-well format to 384-well format are indicated along the x axis, including reducing by a half or a third all volumes and protein amounts proportionally (left two), or only reducing volumes while concentrating the samples so the total protein quantities applied are conserved (right two). The ratio between mean signal to noise is indicated above each condition. Error bars represent the standard error of the mean. n ≥ 12 independent wells. <b>h</b>. Representative screening example for human receptors. The appearance of the raw screening plate (left) is shown alongside absorbance values following median polish normalization (right). Bold borders indicate interactions that are expected based on literature publications. One protein, corresponding to a construct later found to be incorrectly labelled, is omitted. <b>i</b>. SAVEXIS consumes small enough quantities of protein that thousands of assays are possible from small input sizes. For a sample set corresponding to all measured proteins from the leukocyte surface receptor library, the amount of interaction tests that could be performed for each protein based on its expression yield was calculated. As a typical case illustration in which protein is purified from a 30 mL cell culture, the median number of interaction tests possible is indicated by a dashed line.</p></div><div data-test="supp-item" id="Fig6"><h3><a data-track="click" data-track-action="view supplementary info" data-track-label="link" data-test="supp-info-link" href="https://www.nature.com/articles/s41586-022-05028-x/figures/6" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41586-022-05028-x/MediaObjects/41586_2022_5028_Fig6_ESM.jpg">Extended Data Fig. 2 Interaction screen quality controls testify to robust measurements.</a></h3><p><b>a</b>. Recombinant proteins produced at scale match their expected molecular masses. Observed molecular masses from denaturing protein gel electrophoresis (x axis) are compared against computationally predicted molecular mass. Predictions were made by taking the known masses of each amino acid in the protein after processing of its signal peptide, with 2.5 kDa added per predicted N-linked glycosylation site. Shading indicates the 95% compatibility interval for the least-squares linear regression fit. Full images for all gel electrophoresis samples are provided in Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41586-022-05028-x#MOESM3">1</a>. <b>b</b>. Quantitative protein concentration measurements by Bradford assay agree with qualitative estimates of protein concentration based on densitometry of Coomassie-stained protein gels. Measured concentration percentiles (y axis) are compared against discretized expression categories based on staining intensity (y axis and colour shade). <b>c</b>. Control wells included on every screening plate indicate high consistency across the primary interaction screen. Boxplots of plate measurements for negative control wells (blank baits and tag-only rCD4 baits), positive control wells (the known interaction between <i>P. falciparum</i> P12 and P41 at either a 1x or 3x dilution), and loading control wells (OX68 antibody) that capture prey proteins by recognizing their rCD4 tag. n = 1,262 wells per condition. <b>d</b>. Positive interactions from the primary screen were reproducible in a secondary screen with independently produced and measured proteins. For each protein–protein pair measured, the processed signal in the primary screen (x axis) is correlated against the signal in the secondary screen (y axis). Pairs previously described as being interactors are denoted by red colouration.</p></div><div data-test="supp-item" id="Fig7"><h3><a data-track="click" data-track-action="view supplementary info" data-track-label="link" data-test="supp-info-link" href="https://www.nature.com/articles/s41586-022-05028-x/figures/7" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41586-022-05028-x/MediaObjects/41586_2022_5028_Fig7_ESM.jpg">Extended Data Fig. 3 Interactome validation summaries.</a></h3><p><b>a</b>. Precision-recall curve corresponding to Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-022-05028-x#Fig1">1d</a>. Colour shading indicates the cut-off for the summed screen signal across both bait–prey orientations. The performance of a random classifier is shown by the dotted line, and grey shading indicates the valid range between perfect performance and a random classifier. This curve only considers proteins for which expression was detectable, and defines a positive set based on previously published interactions and a negative set based on randomized interaction pairs. <b>b</b>. ROC curves of screen performance are consistent across possible definitions of positive and negative sets. These curves consider all proteins regardless of whether they were detectably expressed. Columns provide different positive reference sets, and rows delineate possible negative reference sets. High-quality support refers to interactions with experimental support by SPR, isothermal titration calorimetry, analytical ultracentrifugation, or a co-crystal structure. <b>c</b>. Overview of evidence for newly identified interactions. Data from both measured bait–prey orientations in the primary and secondary screen are indicated in pink. Results from cell-binding experiments and SPR are categorized for each interaction along a simple qualitative scale of green to red for ease of comparison. This includes whether a binding response was detectable in SPR equilibrium experiments, if the binding response in SPR experiments was sufficiently quantifiable that 1:1 binding models could be fit, and whether gains in cell-surface binding were observed in cell-based assays when the counter-receptor was overexpressed. The full experimental results that are summarized here can be found in Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-022-05028-x#Fig8">4</a>.</p></div><div data-test="supp-item" id="Fig8"><h3><a data-track="click" data-track-action="view supplementary info" data-track-label="link" data-test="supp-info-link" href="https://www.nature.com/articles/s41586-022-05028-x/figures/8" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41586-022-05028-x/MediaObjects/41586_2022_5028_Fig8_ESM.jpg">Extended Data Fig. 4 Orthogonal binding assays to confirm each interaction.</a></h3><p><b>a</b>. Full set of cell-binding traces, extended from Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-022-05028-x#Fig2">2a</a>. For each interaction pair, the protein named on the right of the title was transfected into human cells and the protein named on the left was tested for binding as a fluorescently linked recombinant protein tetramer. Tetramer binding to cells (x axis) was measured by flow cytometry at different tetramer concentrations (y axis). The traces in blue are cells overexpressing the indicated receptor, and red shows binding to the mock-transfected control cells. Because cDNAs may express to widely varying levels or not at all, and some proteins may bind to endogenously expressed HEK293 surface proteins, some experiments give inconclusive binding data. For example, HLA-F is known to be predominantly sequestered intracellularly<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 75" title="Lee, N., Ishitani, A. &amp; Geraghty, D. E. HLA-F is a surface marker on activated lymphocytes. Eur. J. Immunol. 40, 2308–2318 (2010)." href="https://www.nature.com/articles/s41586-022-05028-x#ref-CR75" id="ref-link-section-d169881924e1973">75</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 76" title="Wainwright, S. D., Biro, P. A. &amp; Holmes, C. H. HLA-F is a predominantly empty, intracellular, TAP-associated MHC class Ib protein with a restricted expression pattern. J. Immunol. 164, 319–328 (2000)." href="https://www.nature.com/articles/s41586-022-05028-x#ref-CR76" id="ref-link-section-d169881924e1976">76</a></sup>, whereas soluble APP is known to already have strong baseline binding activity to cell lines<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 77" title="Reinhard, C., Borgers, M., David, G. &amp; De Strooper, B. Soluble amyloid-β precursor protein binds its cell surface receptor in a cooperative fashion with glypican and syndecan proteoglycans. J. Cell Sci. 126, 4856–4861 (2013)." href="https://www.nature.com/articles/s41586-022-05028-x#ref-CR77" id="ref-link-section-d169881924e1980">77</a></sup>. <b>b</b>. Full set of SPR sensorgrams, extended from Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-022-05028-x#Fig2">2b</a>. For each interacting pair, the sensorgram on the left side shows kinetic binding measurements, and the sensorgram on the right shows equilibrium binding measurements. The protein used as the analyte is named on the left and the immobilized ligand is on the right. Association and dissociation constants from a 1:1 binding model fit (red line) are displayed where applicable on the kinetic traces. All analytes were resolved by gel filtration immediately prior to use in binding experiments to reduce the influence of protein aggregates, which otherwise can dominate binding kinetics (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41586-022-05028-x#MOESM3">2</a>). Some analytes show clear evidence of two-phase binding kinetics such as PTPRF and MCAM.</p></div><div data-test="supp-item" id="Fig9"><h3><a data-track="click" data-track-action="view supplementary info" data-track-label="link" data-test="supp-info-link" href="https://www.nature.com/articles/s41586-022-05028-x/figures/9" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41586-022-05028-x/MediaObjects/41586_2022_5028_Fig9_ESM.jpg">Extended Data Fig. 5 Distribution of immune receptor binding affinities.</a></h3><p><b>a</b>. Antigen-presenting cells vary in their receptor contacts with circulating T lymphocytes. Each dot is a cell–cell pair comprising T cells (including all CD4 and CD8 subsets) with either a B cell (red distribution) or dendritic cell (blue distribution). Statistics are overlaid for a Welch’s t-test. <b>b</b>. The overall distribution of immune receptor binding affinities is centred in the range of micromolar dissociation constants. All quantified immune interactions in our network are plotted as a log-scale distribution of binding dissociation constants (<i>K</i><sub>D</sub>). <b>c</b>. Cell surface receptor–ligand binding affinities weakly correlate with protein expression level. For all immune cell types measured by proteomics and all protein interaction pairs measured in our study, the equilibrium binding constant (y axis) is correlated to the summed expression of the two proteins. Weaker interactions (corresponding to higher <i>K</i><sub>D</sub> values) are associated with higher expression. Owing to the number of points, a hex plot is shown with the number of unique combinations of cell type and protein pairs represented by colour shading. A least-squares linear regression line is overlaid in blue. <b>d</b>. Within individual cell types, a surface protein’s expression level weakly and variably correlates with its binding affinity. Instead of all pair combinations, each cell type measured by proteomics is individually presented. One point is placed for each detectable protein with an interaction in that cell type. In cases in which a single protein has multiple interactions of varying affinities, one point is drawn per binding interaction that protein participates in. Least-squares linear regression fit lines and 95% compatibility intervals are overlaid for each cell type. p-values for the Pearson correlation fit are accentuated for conditions where p &lt; 0.10.</p></div><div data-test="supp-item" id="Fig10"><h3><a data-track="click" data-track-action="view supplementary info" data-track-label="link" data-test="supp-info-link" href="https://www.nature.com/articles/s41586-022-05028-x/figures/10" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41586-022-05028-x/MediaObjects/41586_2022_5028_Fig10_ESM.jpg">Extended Data Fig. 6 Average receptor affinity increases after immune activation.</a></h3><p><b>a</b>. Extended version of Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-022-05028-x#Fig2">2e</a>, showing all cell subtypes identified in the original proteomics study<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 1" title="Rieckmann, J. C. et al. Social network architecture of human immune cells unveiled by quantitative proteomics. Nat. Immunol. 18, 583–593 (2017)." href="https://www.nature.com/articles/s41586-022-05028-x#ref-CR1" id="ref-link-section-d169881924e2068">1</a></sup> instead of higher-level cell-type categories. Each point represents the strength of a protein’s interaction for proteins that are either differentially downregulated upon activation (“Down”) or differentially upregulated upon activation (“Up”). To be classified as differentially expressed, the protein must have more than a 2-fold change upon activation. Data are shown as Tukey boxplots with Holm-corrected p-values calculated from a two-sided Welch’s t-test. <b>b</b>. Combined analysis of all cell types for which paired activated and resting expression data is available. As in panel a, differential expression was defined by more than 2-fold changes upon activation (n = 4 blood donors). Statistics are overlaid for a two-sided Welch’s t-test. <b>c</b>. Same as panel b except setting as the criterion for differential expression that the protein must have a corrected p-value below 0.05 across the 4 proteomics replicates available per condition. Statistics are overlaid for a two-sided Welch’s t-test. <b>d</b>. Same analysis as panel b performed on an independent dataset of RNA-seq measurements on sorted and stimulated immune cell populations<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 83" title="Calderon, D. et al. Landscape of stimulation-responsive chromatin across diverse human immune cells. Nat. Genet. 51, 1494–1505 (2019)." href="https://www.nature.com/articles/s41586-022-05028-x#ref-CR83" id="ref-link-section-d169881924e2082">83</a></sup>. Statistics are overlaid for a two-sided Welch’s t-test (n = 4 blood donors). <b>e</b>. Same analysis as panel a, performed on an independent dataset of RNA-seq measurements (Calderon et al. 2019). Every leukocyte subpopulation measured in activated and resting states is show as a separate box. Data are shown as Tukey boxplots with Holm-corrected p-values calculated from a two-sided Welch’s t-test.</p></div><div data-test="supp-item" id="Fig11"><h3><a data-track="click" data-track-action="view supplementary info" data-track-label="link" data-test="supp-info-link" href="https://www.nature.com/articles/s41586-022-05028-x/figures/11" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41586-022-05028-x/MediaObjects/41586_2022_5028_Fig11_ESM.jpg">Extended Data Fig. 7 Internal representations of the quantitative cellular interaction model and additional validations.</a></h3><p><b>a</b>. Interaction spectra showing how individual receptor interactions (x axis) contribute to the overall connection strength between cell pairs. The model output per pair, which corresponds to the calculated density of proteins in a bound configuration at a theoretical equilibrium, is shown on the y axis in log<sub>10</sub> scale. <b>b</b>. Relative contributions of different protein–protein interactions to a cell pair’s overall connection scores. The colour shading indicates the percent of the total calculated interactions between a given cell pair (x axis) that are attributable to each specific protein–protein interaction (y axis). <b>c</b>. Null distribution of correlations to published data when the model’s cell-surface protein interaction pairs are randomized. Histogram bins representing correlations equal to or greater than the fit of the true model are shaded in light blue after 1,000 random permutations were performed. <b>d</b>. Distribution of model correlation coefficients to previously published leukocyte binding data following a complete leave-one-out analysis of each surface protein. The histogram bin that includes the observed correlation in Fig.<a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-022-05028-x#Fig2"> 2g</a> is shaded in lighter blue. The two proteins leading to the greatest change in correlation (ICAM1 and PECAM1) are labelled. <b>e</b>. Model fits remain robust on independently measured datasets of leukocyte cellular contacts. Following an analogous approach to Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-022-05028-x#Fig2">2g</a>, the kinetic model’s predictions for baseline leukocyte interaction rates were compared to empirical data generated during our pharmacoscopy experiments. Only negative control conditions treated with PBS instead of recombinant protein were included, with interaction scores calculated following the same methodology as the previously published study<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 25" title="Vladimer, G. I. et al. Global survey of the immunomodulatory potential of common drugs. Nat. Chem. Biol. 13, 681–690 (2017)." href="https://www.nature.com/articles/s41586-022-05028-x#ref-CR25" id="ref-link-section-d169881924e2136">25</a></sup> described in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-022-05028-x#Fig2">2g</a>. Shading indicates the 95% compatibility interval of the least-squares linear regression fit. <b>f</b>. Differential equation model output simulating cell pairs reaching a binding equilibrium. Each colour is either a cell type in an unbound state or a cell pair. The absolute proportions in blood (y axis) were tracked over an arbitrary time scale (x axis) until equilibrium is reached.</p></div><div data-test="supp-item" id="Fig12"><h3><a data-track="click" data-track-action="view supplementary info" data-track-label="link" data-test="supp-info-link" href="https://www.nature.com/articles/s41586-022-05028-x/figures/12" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41586-022-05028-x/MediaObjects/41586_2022_5028_Fig12_ESM.jpg">Extended Data Fig. 8 Integrating single-cell expression atlases with cellular interactions.</a></h3><p><b>a</b>. Interactively searching immune interactions in human tissues through our web tool. This screenshot depicts the core features of the website, including a drop-down menu to select one of eight different tissue datasets, adjustable sliders for expression cut-off thresholds when determining interactions, and multiple tabs offering different kinds of views for both cellular and molecular interaction types. <b>b</b>. Myeloid-lineage cells do not express any greater quantity of interaction-capable surface proteins than non-myeloid cells. Each dot represents a specific measured cell type in the indicated dataset, which are grouped into categories as myeloid or non-myeloid (x axis). The absolute number of cell-surface proteins for which at least one interaction is annotated with another cell-surface protein (y axis) is compared between categories. p-values from a two-sided Welch’s t-test are shown alongside their corresponding false discovery rate-corrected q values. Tissues match those shown in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-022-05028-x#Fig3">3c</a>. <b>c</b>. Changes in cellular interaction frequencies among immune cells isolated from cancerous versus healthy kidney implicate cellular contacts with potential relevance to pathology. The total number of interactions detected in a single-cell sequencing dataset of paired healthy kidney and kidney tumours are compared across immune populations. <b>d</b>. Comparisons of interactions detected in paired samples of healthy and diseased tissue can suggest functional targets. Human lung tissue from healthy donors and patients with asthma were processed in identical ways through our web tool’s functions. At the indicated expression cut-off which requires the mRNA encoding an interacting surface protein be detected in at least 5% of cells from a given cell type, interactions between cells were categorized based on whether they were present in both healthy and asthmatic samples (purple), only healthy (green), or only asthma (red). Although more qualitative than differential expression based tests, this approach may have utility in conducting more sensitive exploratory analyses of interaction sets. <b>e</b>. Intercellular signalling pathways in tumour-infiltrating immune cells inferred by NicheNet analysis. Two analyses are shown on a single-cell RNA-seq dataset of immune cells isolated from kidney for cell-surface signals that differentially regulate gene expression in plasmacytoid dendritic cells (pDCs) and helper T cells within kidney tumours when compared to adjacent healthy kidney tissue. In the larger box, intercellular signals being received by pDCs are shown (left), matched to genes inferred to be regulated by those signals. In the smaller inset box, the finding that tumour pDCs upregulate JAML is expanded by analysing pDC communication specifically with helper T cells. Targets in our gene regulatory analysis were filtered to exclude those which recurred non-specifically in more than half of all cases.</p></div><div data-test="supp-item" id="Fig13"><h3><a data-track="click" data-track-action="view supplementary info" data-track-label="link" data-test="supp-info-link" href="https://www.nature.com/articles/s41586-022-05028-x/figures/13" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41586-022-05028-x/MediaObjects/41586_2022_5028_Fig13_ESM.jpg">Extended Data Fig. 9 Microscopy readouts identify that receptor binding partner transcripts are colocalized in human lymph nodes, and that polarization of B and NK cells serves as a marker of classical activation pathways.</a></h3><p><b>a</b>. Single-molecule fluorescent in situ hybridization in human lymph node demonstrates that cells transcribing genes encoding surface proteins found to physically interact in biochemical assays are also physically colocalized in the lymph node. A lymphoid follicle is shown for 4 different transcript pairs encoding proteins that directly interact. Each experiment was repeated on tissue from two donors. The scale bar is 100 μm and applies to all images. <b>b</b>. Polarization of B cells relative to PBS-treated controls after treatment with cytokines and other immunomodulatory molecules. As expected IL-4 and IL-6 are strong B-cell specific activators<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 78" title="Rush, J. S. &amp; Hodgkin, P. D. B cells activated via CD40 and IL-4 undergo a division burst but require continued stimulation to maintain division, survival and differentiation. Eur. J. Immunol. 31, 1150–1159 (2001)." href="https://www.nature.com/articles/s41586-022-05028-x#ref-CR78" id="ref-link-section-d169881924e2216">78</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 79" title="Dienz, O. et al. The induction of antibody production by IL-6 is indirectly mediated by IL-21 produced by CD4+ T cells. J. Exp. Med. 206, 69–78 (2009)." href="https://www.nature.com/articles/s41586-022-05028-x#ref-CR79" id="ref-link-section-d169881924e2219">79</a></sup>, whereas IL-15 and LPS activate both NK and B cells<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 80" title="Armitage, R. J., Macduff, B. M., Eisenman, J., Paxton, R. &amp; Grabstein, K. H. IL-15 has stimulatory activity for the induction of B cell proliferation and differentiation. J. Immunol. 154, 483–490 (1995)." href="https://www.nature.com/articles/s41586-022-05028-x#ref-CR80" id="ref-link-section-d169881924e2223">80</a></sup>. Data are shown as Tukey boxplots with Holm-corrected p-values calculated from a one-sided t-test. n = 7 blood donors. <b>c</b>. Polarization of NK cells relative to PBS-treated controls after treatment with cytokines and immunomodulatory molecules. As has been reported for NK cell activation, IL-15 invokes the strongest activation of NK cells<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 81" title="Santana Carrero, R. M. et al. IL-15 is a component of the inflammatory milieu in the tumor microenvironment promoting antitumor responses. Proc. Natl Acad. Sci. USA 116, 599–608 (2019)." href="https://www.nature.com/articles/s41586-022-05028-x#ref-CR81" id="ref-link-section-d169881924e2230">81</a></sup>, whereas there are no effects from IL-4 and IL-6. The NK cells are also inhibited by the steroid dexamethasone (Dex), consistent with known pharmacology<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 82" title="Holbrook, N. J., Cox, W. I. &amp; Horner, H. C. Direct suppression of natural killer activity in human peripheral blood leukocyte cultures by glucocorticoids and its modulation by interferon. Cancer Res. 43, 4019–4025 (1983)." href="https://www.nature.com/articles/s41586-022-05028-x#ref-CR82" id="ref-link-section-d169881924e2235">82</a></sup>. Data are shown as Tukey boxplots with Holm-corrected p-values calculated from a one-sided t-test. n = 7 blood donors.</p></div><div data-test="supp-item" id="Fig14"><h3><a data-track="click" data-track-action="view supplementary info" data-track-label="link" data-test="supp-info-link" href="https://www.nature.com/articles/s41586-022-05028-x/figures/14" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41586-022-05028-x/MediaObjects/41586_2022_5028_Fig14_ESM.jpg">Extended Data Fig. 10 High-content microscopy datasets show human leukocyte phenotypes after infusion of purified proteins.</a></h3><p><b>a</b>. Full dataset of changes to cellular state proportions, extended from Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-022-05028-x#Fig4">4b</a>. The proportions of measured cell types and cell states (x axis) are compared across different protein doses, timepoints, and with or without background LPS activation (y axis). Points are sized by their adjusted p-values, and shaded to show relative change compared to controls. n = 5 wells. <b>b</b>. Observed immunomodulatory phenotypes match the expression profiles of the cell types expressing the applied protein or its identified binding receptor. For every statistically significant phenotype upon protein application that is described in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-022-05028-x#Fig4">4b</a> (grey top bars), the protein itself and all its identified interaction partners (columns) had their expression in lymphocyte populations (x axis) compared according to quantitative proteomics measurements (y axis). The lymphocyte populations are coloured blue if they match the cell type for which the significant phenotype was found. <b>c</b>. Clustering on the full set of cell-to-cell connectivity changes identifies recurring modules of cellular shifts. All protein conditions and timepoints (y axis) were hierarchically clustered by the complete linkage method. As above, changes in each cell pair’s interactions (y axis) are indicated by the provided colour scale and sized based on their adjusted p-values. n = 10 wells.</p></div></div></div></div></section><section data-title="Supplementary information"><div id="Sec41-section"><h2 id="Sec41">Supplementary information</h2><div id="Sec41-content"><div data-test="supplementary-info">
                
                
                
                
                
                
              <div data-test="supp-item" id="MOESM1"><h3><a data-track="click" data-track-action="view supplementary info" data-track-label="link" data-test="supp-info-link" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41586-022-05028-x/MediaObjects/41586_2022_5028_MOESM1_ESM.pdf" data-supp-info-image="">Supplementary Information</a></h3><p>Supplementary Equations - a technical overview of the mathematical models implemented, along with derivations and definitions of equations. The text begins with a summary overview before defining modelling assumptions and calculations in detail.</p></div><div data-test="supp-item" id="MOESM3"><h3><a data-track="click" data-track-action="view supplementary info" data-track-label="link" data-test="supp-info-link" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41586-022-05028-x/MediaObjects/41586_2022_5028_MOESM3_ESM.pdf" data-supp-info-image="">Supplementary Figures</a></h3><p>Supplementary Figs. 1–4 show the complete uncropped images of Coomassie-stained protein gels and protein gel filtration traces. Example images illustrating the flow cytometry gating strategy are also included.</p></div><div data-test="supp-item" id="MOESM4"><h3><a data-track="click" data-track-action="view supplementary info" data-track-label="link" data-test="supp-info-link" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41586-022-05028-x/MediaObjects/41586_2022_5028_MOESM4_ESM.xlsx" data-supp-info-image="">Supplementary Tables</a></h3><p>Supplementary Tables 1–6, which include the pan-leukocyte surface protein library, the full matrix of interaction screen binding data, the curated interactions from the immunology literature, statistical testing of cell pair affinity distributions, the abbreviations used for describing immune cell populations and a list of differentially expressed genes in tumour-infiltrating kidney immune cells.</p></div></div></div></div></section><section data-title="Rights and permissions"><div id="rightslink-section"><h2 id="rightslink">Rights and permissions</h2><div id="rightslink-content">
              <p><b>Open Access</b>  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>.</p>
            <p><a data-track="click" data-track-action="view rights and permissions" data-track-label="link" href="https://s100.copyright.com/AppDispatchServlet?title=A%20physical%20wiring%20diagram%20for%20the%20human%20immune%20system&amp;author=Jarrod%20Shilts%20et%20al&amp;contentID=10.1038%2Fs41586-022-05028-x&amp;copyright=The%20Author%28s%29&amp;publication=0028-0836&amp;publicationDate=2022-08-03&amp;publisherName=SpringerNature&amp;orderBeanReset=true&amp;oa=CC%20BY">Reprints and Permissions</a></p></div></div></section><section aria-labelledby="article-info" data-title="About this article"><div id="article-info-section"><h2 id="article-info">About this article</h2><div id="article-info-content"><div><p><a data-crossmark="10.1038/s41586-022-05028-x" target="_blank" rel="noopener" href="https://crossmark.crossref.org/dialog/?doi=10.1038/s41586-022-05028-x" data-track="click" data-track-action="Click Crossmark" data-track-label="link" data-test="crossmark"><img width="57" height="81" alt="Verify currency and authenticity via CrossMark" src="data:image/svg+xml;base64,<svg height="81" width="57" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="m17.35 35.45 21.3-14.2v-17.03h-21.3" fill="#989898"/><path d="m38.65 35.45-21.3-14.2v-17.03h21.3" fill="#747474"/><path d="m28 .5c-12.98 0-23.5 10.52-23.5 23.5s10.52 23.5 23.5 23.5 23.5-10.52 23.5-23.5c0-6.23-2.48-12.21-6.88-16.62-4.41-4.4-10.39-6.88-16.62-6.88zm0 41.25c-9.8 0-17.75-7.95-17.75-17.75s7.95-17.75 17.75-17.75 17.75 7.95 17.75 17.75c0 4.71-1.87 9.22-5.2 12.55s-7.84 5.2-12.55 5.2z" fill="#535353"/><path d="m41 36c-5.81 6.23-15.23 7.45-22.43 2.9-7.21-4.55-10.16-13.57-7.03-21.5l-4.92-3.11c-4.95 10.7-1.19 23.42 8.78 29.71 9.97 6.3 23.07 4.22 30.6-4.86z" fill="#9c9c9c"/><path d="m.2 58.45c0-.75.11-1.42.33-2.01s.52-1.09.91-1.5c.38-.41.83-.73 1.34-.94.51-.22 1.06-.32 1.65-.32.56 0 1.06.11 1.51.35.44.23.81.5 1.1.81l-.91 1.01c-.24-.24-.49-.42-.75-.56-.27-.13-.58-.2-.93-.2-.39 0-.73.08-1.05.23-.31.16-.58.37-.81.66-.23.28-.41.63-.53 1.04-.13.41-.19.88-.19 1.39 0 1.04.23 1.86.68 2.46.45.59 1.06.88 1.84.88.41 0 .77-.07 1.07-.23s.59-.39.85-.68l.91 1c-.38.43-.8.76-1.28.99-.47.22-1 .34-1.58.34-.59 0-1.13-.1-1.64-.31-.5-.2-.94-.51-1.31-.91-.38-.4-.67-.9-.88-1.48-.22-.59-.33-1.26-.33-2.02zm8.4-5.33h1.61v2.54l-.05 1.33c.29-.27.61-.51.96-.72s.76-.31 1.24-.31c.73 0 1.27.23 1.61.71.33.47.5 1.14.5 2.02v4.31h-1.61v-4.1c0-.57-.08-.97-.25-1.21-.17-.23-.45-.35-.83-.35-.3 0-.56.08-.79.22-.23.15-.49.36-.78.64v4.8h-1.61zm7.37 6.45c0-.56.09-1.06.26-1.51.18-.45.42-.83.71-1.14.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.36c.07.62.29 1.1.65 1.44.36.33.82.5 1.38.5.29 0 .57-.04.83-.13s.51-.21.76-.37l.55 1.01c-.33.21-.69.39-1.09.53-.41.14-.83.21-1.26.21-.48 0-.92-.08-1.34-.25-.41-.16-.76-.4-1.07-.7-.31-.31-.55-.69-.72-1.13-.18-.44-.26-.95-.26-1.52zm4.6-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.07.45-.31.29-.5.73-.58 1.3zm2.5.62c0-.57.09-1.08.28-1.53.18-.44.43-.82.75-1.13s.69-.54 1.1-.71c.42-.16.85-.24 1.31-.24.45 0 .84.08 1.17.23s.61.34.85.57l-.77 1.02c-.19-.16-.38-.28-.56-.37-.19-.09-.39-.14-.61-.14-.56 0-1.01.21-1.35.63-.35.41-.52.97-.52 1.67 0 .69.17 1.24.51 1.66.34.41.78.62 1.32.62.28 0 .54-.06.78-.17.24-.12.45-.26.64-.42l.67 1.03c-.33.29-.69.51-1.08.65-.39.15-.78.23-1.18.23-.46 0-.9-.08-1.31-.24-.4-.16-.75-.39-1.05-.7s-.53-.69-.7-1.13c-.17-.45-.25-.96-.25-1.53zm6.91-6.45h1.58v6.17h.05l2.54-3.16h1.77l-2.35 2.8 2.59 4.07h-1.75l-1.77-2.98-1.08 1.23v1.75h-1.58zm13.69 1.27c-.25-.11-.5-.17-.75-.17-.58 0-.87.39-.87 1.16v.75h1.34v1.27h-1.34v5.6h-1.61v-5.6h-.92v-1.2l.92-.07v-.72c0-.35.04-.68.13-.98.08-.31.21-.57.4-.79s.42-.39.71-.51c.28-.12.63-.18 1.04-.18.24 0 .48.02.69.07.22.05.41.1.57.17zm.48 5.18c0-.57.09-1.08.27-1.53.17-.44.41-.82.72-1.13.3-.31.65-.54 1.04-.71.39-.16.8-.24 1.23-.24s.84.08 1.24.24c.4.17.74.4 1.04.71s.54.69.72 1.13c.19.45.28.96.28 1.53s-.09 1.08-.28 1.53c-.18.44-.42.82-.72 1.13s-.64.54-1.04.7-.81.24-1.24.24-.84-.08-1.23-.24-.74-.39-1.04-.7c-.31-.31-.55-.69-.72-1.13-.18-.45-.27-.96-.27-1.53zm1.65 0c0 .69.14 1.24.43 1.66.28.41.68.62 1.18.62.51 0 .9-.21 1.19-.62.29-.42.44-.97.44-1.66 0-.7-.15-1.26-.44-1.67-.29-.42-.68-.63-1.19-.63-.5 0-.9.21-1.18.63-.29.41-.43.97-.43 1.67zm6.48-3.44h1.33l.12 1.21h.05c.24-.44.54-.79.88-1.02.35-.24.7-.36 1.07-.36.32 0 .59.05.78.14l-.28 1.4-.33-.09c-.11-.01-.23-.02-.38-.02-.27 0-.56.1-.86.31s-.55.58-.77 1.1v4.2h-1.61zm-47.87 15h1.61v4.1c0 .57.08.97.25 1.2.17.24.44.35.81.35.3 0 .57-.07.8-.22.22-.15.47-.39.73-.73v-4.7h1.61v6.87h-1.32l-.12-1.01h-.04c-.3.36-.63.64-.98.86-.35.21-.76.32-1.24.32-.73 0-1.27-.24-1.61-.71-.33-.47-.5-1.14-.5-2.02zm9.46 7.43v2.16h-1.61v-9.59h1.33l.12.72h.05c.29-.24.61-.45.97-.63.35-.17.72-.26 1.1-.26.43 0 .81.08 1.15.24.33.17.61.4.84.71.24.31.41.68.53 1.11.13.42.19.91.19 1.44 0 .59-.09 1.11-.25 1.57-.16.47-.38.85-.65 1.16-.27.32-.58.56-.94.73-.35.16-.72.25-1.1.25-.3 0-.6-.07-.9-.2s-.59-.31-.87-.56zm0-2.3c.26.22.5.37.73.45.24.09.46.13.66.13.46 0 .84-.2 1.15-.6.31-.39.46-.98.46-1.77 0-.69-.12-1.22-.35-1.61-.23-.38-.61-.57-1.13-.57-.49 0-.99.26-1.52.77zm5.87-1.69c0-.56.08-1.06.25-1.51.16-.45.37-.83.65-1.14.27-.3.58-.54.93-.71s.71-.25 1.08-.25c.39 0 .73.07 1 .2.27.14.54.32.81.55l-.06-1.1v-2.49h1.61v9.88h-1.33l-.11-.74h-.06c-.25.25-.54.46-.88.64-.33.18-.69.27-1.06.27-.87 0-1.56-.32-2.07-.95s-.76-1.51-.76-2.65zm1.67-.01c0 .74.13 1.31.4 1.7.26.38.65.58 1.15.58.51 0 .99-.26 1.44-.77v-3.21c-.24-.21-.48-.36-.7-.45-.23-.08-.46-.12-.7-.12-.45 0-.82.19-1.13.59-.31.39-.46.95-.46 1.68zm6.35 1.59c0-.73.32-1.3.97-1.71.64-.4 1.67-.68 3.08-.84 0-.17-.02-.34-.07-.51-.05-.16-.12-.3-.22-.43s-.22-.22-.38-.3c-.15-.06-.34-.1-.58-.1-.34 0-.68.07-1 .2s-.63.29-.93.47l-.59-1.08c.39-.24.81-.45 1.28-.63.47-.17.99-.26 1.54-.26.86 0 1.51.25 1.93.76s.63 1.25.63 2.21v4.07h-1.32l-.12-.76h-.05c-.3.27-.63.48-.98.66s-.73.27-1.14.27c-.61 0-1.1-.19-1.48-.56-.38-.36-.57-.85-.57-1.46zm1.57-.12c0 .3.09.53.27.67.19.14.42.21.71.21.28 0 .54-.07.77-.2s.48-.31.73-.56v-1.54c-.47.06-.86.13-1.18.23-.31.09-.57.19-.76.31s-.33.25-.41.4c-.09.15-.13.31-.13.48zm6.29-3.63h-.98v-1.2l1.06-.07.2-1.88h1.34v1.88h1.75v1.27h-1.75v3.28c0 .8.32 1.2.97 1.2.12 0 .24-.01.37-.04.12-.03.24-.07.34-.11l.28 1.19c-.19.06-.4.12-.64.17-.23.05-.49.08-.76.08-.4 0-.74-.06-1.02-.18-.27-.13-.49-.3-.67-.52-.17-.21-.3-.48-.37-.78-.08-.3-.12-.64-.12-1.01zm4.36 2.17c0-.56.09-1.06.27-1.51s.41-.83.71-1.14c.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.37c.08.62.29 1.1.65 1.44.36.33.82.5 1.38.5.3 0 .58-.04.84-.13.25-.09.51-.21.76-.37l.54 1.01c-.32.21-.69.39-1.09.53s-.82.21-1.26.21c-.47 0-.92-.08-1.33-.25-.41-.16-.77-.4-1.08-.7-.3-.31-.54-.69-.72-1.13-.17-.44-.26-.95-.26-1.52zm4.61-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.08.45-.31.29-.5.73-.57 1.3zm3.01 2.23c.31.24.61.43.92.57.3.13.63.2.98.2.38 0 .65-.08.83-.23s.27-.35.27-.6c0-.14-.05-.26-.13-.37-.08-.1-.2-.2-.34-.28-.14-.09-.29-.16-.47-.23l-.53-.22c-.23-.09-.46-.18-.69-.3-.23-.11-.44-.24-.62-.4s-.33-.35-.45-.55c-.12-.21-.18-.46-.18-.75 0-.61.23-1.1.68-1.49.44-.38 1.06-.57 1.83-.57.48 0 .91.08 1.29.25s.71.36.99.57l-.74.98c-.24-.17-.49-.32-.73-.42-.25-.11-.51-.16-.78-.16-.35 0-.6.07-.76.21-.17.15-.25.33-.25.54 0 .14.04.26.12.36s.18.18.31.26c.14.07.29.14.46.21l.54.19c.23.09.47.18.7.29s.44.24.64.4c.19.16.34.35.46.58.11.23.17.5.17.82 0 .3-.06.58-.17.83-.12.26-.29.48-.51.68-.23.19-.51.34-.84.45-.34.11-.72.17-1.15.17-.48 0-.95-.09-1.41-.27-.46-.19-.86-.41-1.2-.68z" fill="#535353"/></g></svg>"/></a></p><div><h3 id="citeas">Cite this article</h3><p>Shilts, J., Severin, Y., Galaway, F. <i>et al.</i> A physical wiring diagram for the human immune system.
                    <i>Nature</i>  (2022). https://doi.org/10.1038/s41586-022-05028-x</p><p><a data-test="citation-link" data-track="click" data-track-action="download article citation" data-track-label="link" data-track-external="" href="https://citation-needed.springer.com/v2/references/10.1038/s41586-022-05028-x?format=refman&amp;flavour=citation">Download citation<svg width="16" height="16" focusable="false" role="img" aria-hidden="true"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#global-icon-download"></use></svg></a></p><ul data-test="publication-history"><li><p>Received<span>: </span><span><time datetime="2021-06-30">30 June 2021</time></span></p></li><li><p>Accepted<span>: </span><span><time datetime="2022-06-28">28 June 2022</time></span></p></li><li><p>Published<span>: </span><span><time datetime="2022-08-03">03 August 2022</time></span></p></li><li><p><abbr title="Digital Object Identifier">DOI</abbr><span>: </span><span>https://doi.org/10.1038/s41586-022-05028-x</span></p></li></ul></div></div></div></div></section>

            

            
                <section data-title="Comments"><div id="article-comments-section"><h2 id="article-comments">Comments</h2><p>By submitting a comment you agree to abide by our <a href="https://www.nature.com/info/tandc.html">Terms</a> and <a href="https://www.nature.com/info/community-guidelines.html">Community Guidelines</a>. If you find something abusive or that does not comply with our terms or guidelines please flag it as inappropriate.</p></div></section>
                
            

            </div></div>
  </body>
</html>
