<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <title>James Routley | Feed</title>
    <link
      rel="stylesheet"
      type="text/css"
      href="../styles.css"
      media="screen"
    />
  </head>
  <body>
    <a href="/index.html">Back</a>
    <a href="https://www.owlposting.com/p/a-primer-on-why-microbiome-research">Original</a>
    <h1>A primer on why microbiome research is hard</h1>
    
    <div id="readability-page-1" class="page"><div><div dir="auto"><div><figure><a target="_blank" href="https://substackcdn.com/image/fetch/f_auto,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F08375531-80c6-40c0-abf6-b2d83ae285ae_2040x1144.png" data-component-name="Image2ToDOM" rel=""><div><picture><source type="image/webp" srcset="https://substackcdn.com/image/fetch/w_424,c_limit,f_webp,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F08375531-80c6-40c0-abf6-b2d83ae285ae_2040x1144.png 424w, https://substackcdn.com/image/fetch/w_848,c_limit,f_webp,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F08375531-80c6-40c0-abf6-b2d83ae285ae_2040x1144.png 848w, https://substackcdn.com/image/fetch/w_1272,c_limit,f_webp,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F08375531-80c6-40c0-abf6-b2d83ae285ae_2040x1144.png 1272w, https://substackcdn.com/image/fetch/w_1456,c_limit,f_webp,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F08375531-80c6-40c0-abf6-b2d83ae285ae_2040x1144.png 1456w" sizes="100vw"/><img src="https://substackcdn.com/image/fetch/w_1456,c_limit,f_auto,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F08375531-80c6-40c0-abf6-b2d83ae285ae_2040x1144.png" width="1456" height="817" data-attrs="{&#34;src&#34;:&#34;https://substack-post-media.s3.amazonaws.com/public/images/08375531-80c6-40c0-abf6-b2d83ae285ae_2040x1144.png&#34;,&#34;srcNoWatermark&#34;:null,&#34;fullscreen&#34;:null,&#34;imageSize&#34;:null,&#34;height&#34;:817,&#34;width&#34;:1456,&#34;resizeWidth&#34;:null,&#34;bytes&#34;:3578744,&#34;alt&#34;:null,&#34;title&#34;:null,&#34;type&#34;:&#34;image/png&#34;,&#34;href&#34;:null,&#34;belowTheFold&#34;:false,&#34;topImage&#34;:true,&#34;internalRedirect&#34;:null}" alt="" srcset="https://substackcdn.com/image/fetch/w_424,c_limit,f_auto,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F08375531-80c6-40c0-abf6-b2d83ae285ae_2040x1144.png 424w, https://substackcdn.com/image/fetch/w_848,c_limit,f_auto,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F08375531-80c6-40c0-abf6-b2d83ae285ae_2040x1144.png 848w, https://substackcdn.com/image/fetch/w_1272,c_limit,f_auto,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F08375531-80c6-40c0-abf6-b2d83ae285ae_2040x1144.png 1272w, https://substackcdn.com/image/fetch/w_1456,c_limit,f_auto,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F08375531-80c6-40c0-abf6-b2d83ae285ae_2040x1144.png 1456w" sizes="100vw" fetchpriority="high"/></picture></div></a></figure></div><p>Audio:</p><div data-component-name="DigestPostEmbed"><div><a href="https://www.owlposting.com/p/a-primer-on-why-microbiome-research-53f" target="_blank" rel="noopener" class=""><div><picture><source type="image/webp" srcset="https://substackcdn.com/image/fetch/w_140,h_140,c_fill,f_webp,q_auto:good,fl_progressive:steep,g_auto/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F3d16848e-d81c-47c0-8583-3ba46657249f_1400x1400.png"/><img src="https://substackcdn.com/image/fetch/w_140,h_140,c_fill,f_auto,q_auto:good,fl_progressive:steep,g_auto/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F3d16848e-d81c-47c0-8583-3ba46657249f_1400x1400.png" sizes="100vw" alt="A primer on why microbiome research is hard (Audio)" width="140" height="140"/></picture></div></a></div></div><ol><li><p><a href="https://www.abhishaike.com/i/144562568/introduction" rel="">Introduction </a></p></li><li><p><a href="https://www.abhishaike.com/i/144562568/why-is-it-hard" rel="">Why is it hard? </a></p><ol><li><p><a href="https://www.abhishaike.com/i/144562568/difficulty-of-characterization" rel="">Difficulty of characterization </a></p></li><li><p><a href="https://www.abhishaike.com/i/144562568/causal-links-are-hard-to-establish" rel="">Causal links are hard to establish</a></p></li><li><p><a href="https://www.abhishaike.com/i/144562568/unknown-microbiota" rel="">Unknown microbiota</a></p></li><li><p><a href="https://www.abhishaike.com/i/144562568/therapies-are-complicated" rel="">Therapies are complicated</a></p></li></ol></li><li><p><a href="https://www.abhishaike.com/i/144562568/the-initial-promises" rel="">The initial promises</a></p><ol><li><p><a href="https://www.abhishaike.com/i/144562568/metabolic-conditions" rel="">Metabolic conditions</a></p></li><li><p><a href="https://www.abhishaike.com/i/144562568/tumor-microbiomes" rel="">Tumor microbiomes </a></p></li><li><p><a href="https://www.abhishaike.com/i/144562568/gut-brain-axis" rel="">Gut-brain axis</a></p></li></ol></li><li><p><a href="https://www.abhishaike.com/i/144562568/conclusion-tldr" rel="">Conclusion + TLDR</a></p></li></ol><p><span>Microbiomes are, as far as cutting-edge science goes, old news. Knowledge that the microbiome exists and is involved in digestive process has been known for over a half-century. But, with the discovery of </span><em>H. Pylori’s</em><span> role in causing peptic ulcers and the subsequently awarded 2005 Nobel Prize in medicine for it, the microbiome began to be viewed differently: something that is intertwined with the neurological, immunological, and metabolic health of its host. And perhaps something that represents a new therapeutic target entirely. Yet, over the course of the last 20 years, the microbiome field has little to show for it, despite billions in funding, thousands of published papers, and dozens of spun-up companies.</span></p><p><span>As far as I can tell, no scientific deceit was committed. No scandalous, Theranos-esque, or Alzheimers-esque fraud caused this lack of progress. Microbiome research is literally just </span><strong>hard</strong><span>. Why? This essay will attempt to explain that. </span></p><p><span>This all isn’t to say no utility in microbiome research has been found. There has been one extraordinary success in the field: </span><a href="https://www.health.harvard.edu/blog/stool-transplants-are-now-standard-of-care-for-recurrent-c-difficile-infections-2019050916576" rel="">fecal transplants for recurrent </a><em><a href="https://www.health.harvard.edu/blog/stool-transplants-are-now-standard-of-care-for-recurrent-c-difficile-infections-2019050916576" rel="">Clostridioides difficile (C. Diff)</a></em><a href="https://www.health.harvard.edu/blog/stool-transplants-are-now-standard-of-care-for-recurrent-c-difficile-infections-2019050916576" rel=""> infections</a><span>. Whereas this microbiome-aware treatment yields a near-perfect cure rate, </span><a href="https://academic.oup.com/cid/article/40/11/1586/444708" rel="">competing approaches had a dismal 20% cure rate</a><span>. Even more interestingly, knowledge of microbiomes have led us towards large-scale ecological engineering by </span><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4027352/" rel="">leveraging the sexual differentiation influence that the Wolbachia microbe has on mosquitos.</a><span> But these successes are largely isolated. </span></p><p><span>Studying the microbiome is incredibly challenging. You’re trying to observe a living, breathing ecosystem that will react to the most minute of perturbations, has an astonishing level of diversity, and can entirely change from day-to-day. And you’re trying to connect the total collective actions of all this to some functional impact on the human it lives in. And, ideally, you’d also like the ability to directly modify</span><strong> </strong><span>aspects of this ecosystem to change this functional impact! All hard things to do. </span><strong>We’ll discuss all these more in depth in this essay. </strong></p><p><span>But as with all difficult things, some very clever people did manage to make some headway. We mentioned </span><em>C. Diff </em><span>infections, which was one of the major achievements in the field and spurred further investigation into its clinical potential. </span><strong>We’ll also discuss three of the highest-potential areas in microbiome analysis, and where they ended up. </strong><span> </span></p><p><span>Lot of material to cover! Let’s start. </span><strong>One quick note: this essay will primarily focus on gut microbiomes, as that is where the field is most developed.</strong><span> Many of the points I’ll make will apply to studies that concern themselves with microbiomes in general, but the discussion will primarily focus around the gut.</span></p><p><span>One of the more fundamental challenges we face here is that extracting out the microbiome </span><strong>at all</strong><span> is hard! </span></p><p>We start off with a sample collected from the biome of interest. For the mouth, it&#39;d be a swab of your cheek, for the gut, it&#39;d be a stool sample, and so on. But how do we go about characterizing what bacterial, viral, fungal, and so on species are in this sample? </p><p><span>The earliest, scalable method of doing so was called ‘16S sequencing’, which attempted to read a set of genes commonly referred to as ‘16S rRNA genes’, or just 16S, which, when transcribed, </span><a href="https://en.wikipedia.org/wiki/Prokaryotic_small_ribosomal_subunit" rel="">yields a subunit of a ribosomal structure</a><span>. </span><strong>The set of genes encompassing 16S had been understood for decades as ‘taxonomic identifier’ genes; something that alone poked at the phylogeny, or evolutionary history, of a particular species.</strong><span> In turn, sequencing these genes in a microbe colony allowed researchers to learn all species contained in that particular colony! Unfortunately, as with any attempt to simplify biology, this turned out to have a variety of hidden flaws. </span><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6677816/" rel="">16S genes apparently could undergo horizontal gene transfer</a><span> (bacterial genes being transferred to neighboring bacteria), meaning microbes in your sample could very often </span><strong>be contaminated by its neighbors. </strong><span>This in turn meant that some species, </span><strong>even when not present in a sample</strong><span>, </span><a href="https://www.sciencedirect.com/science/article/abs/pii/S0168160517302842" rel="">could still be detected by bioinformatic pipelines using 16S sequencing as input</a><span>. Past the error rate of the test itself, 16S sequencing were more accurate for some taxonomies of microbes than others, some could be well characterized by it, others couldn’t be, which leads to massive selection bias in downstream analysis.  This was one of the earliest tests offered at consumer microbiome companies (</span><a href="https://en.wikipedia.org/wiki/UBiome" rel="">including the infamous uBiome</a><span>).</span></p><p><span>But, for the most part, 16S sequencing was replaced with two methods, both of which are the current dominant forms of microbiome characterization: </span><strong>shotgun metagenomic sequencing</strong><span> and </span><strong>metatranscriptomic sequencing</strong><span> and. As the names imply, the former is concerned with sequencing all DNA snippets in your sample, and the latter with all RNA snippets. This methods, enabled by the Next-Generation-Sequencing (NGS) revolution, allows us to characterize, in bulk, the exact evolutionary history (shotgun metagenomics) and ongoing behavior (metatranscriptomics) of our sample. Unlike 16S, these methods were generally far better at characterizing microbes across domains of life.</span></p><p>Unfortunately, these methods still have their own issues.</p><p><span>Let’s start with shotgun metagenomic sequencing — called ‘shotgun’ because it is untargeted sequencing of all DNA found in a sample. While going this route over purely sequencing 16S allows us to view the genetic landscape of our sample fairly well, practical interpretation of the results are difficult. The sheer diversity of the microbiome is absolutely enormous, and very little of it has been catalogued. </span><strong>As such, most DNA sequences derived from a metagenomic sequencing run will, in all likelihood, be unable to be perfectly matched to any catalogued species.</strong><span> This isn’t a problem in of itself, phylogenetic relationships can be derived from matching up unknown sequences with reference sequences in order to understand how unknown sequences ‘fit’ into the broader picture. </span><strong><a href="https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-021-01059-0" rel="">But this invariably opens the door to bias.</a></strong><span> Specifically, the the process used to perform this ‘fitting’ can dramatically alter what microbes are reported as present in the final result, with the results becoming far worse with poorly studied taxonomies of life, such as rare microbes. As such, multiple parallel metagenomic studies of the exact same sample may arrive to dramatically different conclusions of the species present within that sample,</span><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6716367/" rel=""> unless they rely on the exact same algorithms and reference databases.</a></p><p><span>Metatranscriptomics share basically all of the same issues as metagenomics has, since it also relies on references to characterized sequences. There is one more major problem in that </span><a href="https://www.frontiersin.org/journals/genetics/articles/10.3389/fgene.2019.00904/full" rel="">most of the harvested RNA is ribosomal RNA (90%~)</a><span>, which is largely useless for understanding the transcriptomic profile of microbes. So our problem with detecting low-population microbes becomes even </span><strong>more</strong><span> of an issue here!</span></p><p><span>One quick note: we may try to mitigate some of these issues by first culturing the microbes before sequencing them, encouraging low-population microbes to increase in number. This turns out to be an ill-fated approach in practice.</span><a href="https://www.nature.com/articles/nature17645" rel=""> Many microbes have hyper-specific living conditions</a><span> and will refuse to grow — or outright die — outside of their natural habitat (which happens to be you). People are working on fixing this, but it seems quite aways from actually being reliable. </span></p><p><strong>And we haven’t even mentioned that the long tail of everything before sequencing readout also is a source of bias!</strong><span> </span><a href="https://journals.asm.org/doi/10.1128/spectrum.00090-22" rel="">Everything from genetic material extraction to sequencing machine used</a><span> to even the </span><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8510183/" rel="">storage conditions</a><span> can yield systemic differences in the characterization of a microbiome sample.</span><a href="https://www.nationalgeographic.com/science/article/contaminomics-why-some-microbiome-studies-may-be-wrong" rel=""> Moreover, contamination of microbe samples is also a huge problem</a><span>, </span><strong>which can come from as innocuous of a place as the kit used for genetic material extraction</strong><span>. </span></p><p><span>A more skeptical reader may, at this point, claim that I’m overstating things here a bit. After all, we do DNA and RNA sequencing for human cells all the time, and seem to be happy with our results there. Aren’t many of these problems alleviated by simply following the same protocols for human samples? For the most part, no, microbes are genuinely </span><strong>much </strong><span>harder. Reference alignments are less of a deal for human cells, since we’ve largely characterized the full human genome, so we can trust readouts more. This ability to rely on references means that DNA/RNA contamination is less of an issue, since the reference sequences will save us if we do accidentally sequence non-human material. </span><a href="https://www.stjude.org/media-resources/news-releases/2023-medicine-science-news/humans-vs-bacteria-differences-in-ribosome-decoding-revealed.html" rel="">Human cells also produce proteins slower than bacteria</a><span>, meaning less ribosomal RNA, meaning less readout noise. Finally, the biomass of any given species of a microbe in a sample is astonishingly low — given the immense diversity of the microbiome — whereas human tissue is far more homogenous other than cell-type differentiation, leading to higher fidelity readouts.</span></p><p><span>All of this wraps up to a pretty worrying message: </span><strong>many microbiome studies may be fundamentally incorrect in their takeaway message due to nearly unavoidable problems in sample storage, sample preparation, sequencing, and sequence alignment</strong><span>. </span></p><p><span>But let’s not get ahead of ourselves. Let’s say we’re better than the literature and take extreme pains to ensure we’re doing things </span><strong>right</strong><span>. We take a sample of some microbiome (say, our gut), store it well, avoid contamination, sequence it correctly, align it correctly, and get to a fully accurate view of the genomic and transcriptomic landscape of our microbiome. </span></p><p>What do we do with the readout? Well, aren’t there good microbes and bad microbes? Shouldn’t our first check be to see if we have good ones, and none of the bad ones? Maybe check if the transcriptome matches up with people with gut microbiomes? </p><p>All decent ideas! But there is a minor issue to be aware of. </p><p>There are two sides of the microbiome literature. </p><p><span>One paints a relatively rosy picture of the connection between certain species of a microbiome and diseases. So-called ‘dysbiosis’, or irregularities/abnormalities in ones microbiome, became an increasingly popular view of how to view the microbiome. Associative studies showing the connection between </span><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7146472/" rel="">atherosclerosis</a><span>, </span><a href="https://www.nature.com/articles/s41579-020-0433-9" rel="">metabolic disorders</a><span>, and the like, to certain species of gut microbes have only further cemented this. There are dozens of papers in this realm, many of them gathering thousands of citations, all with effusively positive conclusions on the future of the field in clinical practice.</span></p><p><span>The other side points to two deeply unfortunate facts</span><strong>. One, there is no such thing as a ‘normal’ microbiome.</strong><span> </span><a href="https://www.frontiersin.org/journals/genetics/articles/10.3389/fgene.2022.917926/full" rel="">The microbiomes between otherwise healthy people can strongly differ</a><span> and </span><a href="https://www.cell.com/cell-host-microbe/fulltext/S1931-3128(14)00346-1" rel="">even change over time</a><span>; while there are some species whose overgrowth is clearly bad, there are very few hard rules.</span><strong> And two, it is very much unclear how much a microbiome is ‘chasing’ a certain phenotype (e.g. autism, T2D, etc) versus actually causing it.</strong><span> </span><a href="https://www.nature.com/articles/s41586-020-2881-9" rel="">After all, alterations in microbes don’t only correlate with disease, but also with ethnicity, age, medication intake, vegetable intake, whether they are vegan or not, and </a><strong><a href="https://www.nature.com/articles/s41586-020-2881-9" rel="">even whether they have a dog</a></strong><a href="https://www.nature.com/articles/s41586-020-2881-9" rel="">.</a><span> And most studies in the this space are purely associative with regards to disease, primarily comparing the microbiomes of diseased patients with healthy ones. </span></p><p>Given these two, how can you possibly assess causality of the microbiome doing anything at all? Well, application of a therapeutic agent to the microbiome may lead us somewhere. Enter human fecal microbiome transplants, or FMT. </p><p><strong>FMT represent the strongest piece of evidence with regards to the causal impact of the microbiome on physiological state.</strong><span> This treatment is exactly what it sounds like; transferring the fecal matter of one (healthy) human into another (diseased) human, usually either via pills or colonoscopy, allowing the microbes contained within to colonize a new gut. This way, we needn’t grapple with questions on good/bad microbes or what they are actually doing! Unfortunately, this is also rarely done outside of well-established use-cases — of which there is basically only one: </span><em>C. Diff </em><span>infections. </span><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8455422/" rel="">This is partially due to the fuzzy regulations regarding FMT</a><span>, but also the</span><a href="https://www.frontiersin.org/articles/10.3389/fmedt.2022.961569/full" rel=""> immense complexity in administering the therapy at all.</a><span> So while there are some studies using human FMT to study causality, which we’ll go over more deeply in the last few sections, it makes up a relatively small fraction of the literature. </span></p><p><span>There is another way to assess causality: </span><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7614635/" rel="">mendelian randomization</a><span>, or MR. In recent years, MR has leaked its way into the microbiome literature, allowing researchers to make strong claims about the </span><strong>causal</strong><span> impact of the microbiome on disease, while also still relying on non-FMT, observational studies in humans. The results of these MR studies, unfortunately, remain hazy in my opinion, even though there’s plenty of papers claiming to establish causality. </span><a href="https://www.nature.com/articles/s41598-023-31115-8.pdf" rel="">For example, while a human MR study between microbiome and longevity found that some microbes were causative of longer lifespans (with pretty small correlations),</a><span> </span><strong>the results didn’t transfer well from European to Chinese population.</strong><span>  How many other failure modes are like this in the microbiome literature? Relatively few studies attempt to steelman their own conclusions, so it’s hard to ascertain. Also, even when these studies are done, and singular species of interest are found as protective/harmful, one should recall the earlier discussion on difficulty of microbiome characterization! </span><a href="https://www.sciencedirect.com/science/article/abs/pii/S1075996422000324" rel="">For example, the gut microbe </a><em><a href="https://www.sciencedirect.com/science/article/abs/pii/S1075996422000324" rel="">Bacteroides dorei </a></em><a href="https://www.sciencedirect.com/science/article/abs/pii/S1075996422000324" rel="">has become increasingly recognized for its role in heart conditions and T1D.</a><span> But, this microbe </span><em>also</em><span> has a </span><a href="https://www.sciencedirect.com/science/article/abs/pii/S1075996422000324" rel="">close taxonomic relationship with </a><em><a href="https://www.sciencedirect.com/science/article/abs/pii/S1075996422000324" rel="">Bacteroides vulgatus</a><span>, </span></em><span>a far more innocuous gut microbe, making it hard to identify at all! While replication certainly helps assuage our concern about this issue, one-off studies should be looked at with a critical eye. </span></p><p><span>On a more anecdotal level, I know from personal experience that accurate causal inference is </span><strong>hard</strong><span>. The theory goes incredibly deep, and there are tons of failure modes. While MR is certainly a step up from purely associative studies, I’m immediately skeptical of the whole direction. </span><a href="https://www.nature.com/articles/nature25973#:~:text=Taken%20together%2C%20our%20results%20demonstrate,after%20accounting%20for%20host%20genetics." rel="">Given that the environment is the dominating factor — not host genes — in determining the microbiome</a><span>, using genetic instrumental variables just feels doomed.  </span></p><p>Well, okay. Assessing causality with human FMT is hard to do from a regulatory perspective, and MR is potentially leading us astray. How about relying on animal FMT to assess causality? It’s a decent idea, but still problematic. </p><p><a href="https://link.springer.com/article/10.1007/s00018-017-2693-8" rel="">The physical structure, digestion rate, pH, and glycan profiles of a stomach dramatically differ between human and animals.</a><span> This has functional impacts! </span><a href="https://pubmed.ncbi.nlm.nih.gov/34118462/" rel="">Entire species of microbes that can survive in humans cannot in mice, and vice-versa</a><span>. While there certainly is overlap in genus, the difference mounts up as we critically interrogate individual strains. </span><a href="https://chrismasterjohnphd.substack.com/p/the-greatest-error-in-microbiome" rel="">There is also an interesting essay I found about how mice naturally engage in coprophagy, or eating their own feces, and their microbiome has likely adapted for this!</a><span> While this is only a theory, and not backed up by a paper, it is interesting to consider that even behavioral differences between animals and humans could lead to downstream impacts on their microbiome.  </span><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9775123/" rel="">Using animals more closely related to humans, such as pigs or primates, will likely alleviate this problem</a><span>, but is challenging to scale in any meaningful way. </span></p><p>Okay, learning what’s actually in the microbiome is hard, and learning what the microbiome is actually doing is hard. What else is the issue?</p><p><a href="https://bmcbiol.biomedcentral.com/articles/10.1186/s12915-019-0667-z" rel="">Despite decades of study, a substantial fraction of the microbiome remains </a><strong><a href="https://bmcbiol.biomedcentral.com/articles/10.1186/s12915-019-0667-z" rel="">unable to sequenced in any meaningful way.</a><span> </span></strong></p><p><span>Low-abundance taxonomies of microbes are where our minds may jump to when we consider this, as high-throughout, sequencing-based approaches cannot easily discover them (given that they are often not included in reference databases). While directly cultivating such microbes is a possible solution, this is also not a great fix as we discussed earlier, as some microbes are resistant to being grown outside of their natural environment. As the fields collection of reference genomes grow, the problem of low-abundance microbes diminishes, </span><a href="https://bmcbiol.biomedcentral.com/articles/10.1186/s12915-019-0667-z" rel="">but, as it stands, the current range of ‘un-mappable genetic sequences’ is around 10% (optimistically).</a><span> While this is a step up from the previous numbers, which were closer to 30% to 50% a decade ago, we’re still quite far from having a complete picture of the microbiome. </span></p><p><strong>But this section actually has little to do with low-abundance microbes! </strong><span>Instead, there is a much deeper problem. While we’ve been politely using the words ‘microbes’ throughout this essay, realistically, what </span><em><span>most</span><strong> </strong></em><span>people are studying are the bacterial component of a microbiome. </span><strong><span>The fungal and viral components, or the ‘</span><a href="https://academic.oup.com/femsre/article/41/4/479/3738183?login=false" rel="">mycobiome</a><span>’ and ‘</span><a href="https://www.thelancet.com/journals/ebiom/article/PIIS2352-3964(22)00294-8/fulltext" rel="">virome</a><span>’, of the whole system are largely entirely left alone.</span></strong></p><p><span>That 10% of un-mappable sequences discussed above is an immense understatement of the true unknown set of species being missed, because fungal and viral genomes are quite small! This ties into why characterizing these domains of life is difficult, the smaller the genome, the more challenging it to accurately tie it back to a characterized species/genus/taxonomy. And the results from bacterial microbiome papers don’t seem to cleanly translate to understanding the behaviors of gut fungal/viral colonies either, </span><strong>different behaviors entirely are observed amongst them</strong><span>. For example, </span><a href="https://academic.oup.com/femsre/article/41/4/479/3738183?login=false" rel="">there seem to be a lack of ‘core’ species in the mycobiome</a><span> as time passes, very much unlike the bacterial microbiome, which does display a high degree of stability post-childhood. Moreover, there is a high degree of interaction between these domains, </span><a href="https://www.thelancet.com/journals/ebiom/article/PIIS2352-3964(22)00294-8/fulltext" rel="">with bacteriophages being able to dramatically alter the behaviors and compositions of nearby bacteria</a><span> (and likely fungi as well!). </span></p><p><strong>Realistically though, how important are understanding these territories of life?</strong><span> Most microbiome papers only attempt to sequence the bacterial components, so much of the historical literature cannot be trusted upon to gauge the individual impact of the virome or mycobiome. </span><a href="https://onlinelibrary.wiley.com/doi/pdfdirect/10.1002/advs.202301097" rel="">However, there is evidence to suggest at least the virome is a crucially important aspect of fecal microbiome transplants (FMT):</a></p><p>One angle of research supporting the role of the virome is the association between efficacy and overlapped viral communities. Studies[17,121,127] of recurrent CDI indicated that recipients who benefited from FMT had an increased relative abundance of Microviridae and a decreased abundance of Caudovirales. These results suggested that the Microviridae family and Caudovirales order potentially played a pivotal role in the efficacy of FMT. Moreover, Broecker et al.[128] observed that stable bacteriophages remained present for at least 4.5 years following FMT and were better correlated with successful FMT than bacterial communities. The negative correlation between recovery of the virome and CDI recurrence can further certify the vital role of the virome in FMT. For example, one study[121] reported that CDI patients who only had restored bacterial communities suffered from disease recurrence. In subjects with metabolic syndrome who did not respond to FMT, the differences between the viral communities shared by the recipient and the respective donor were greater.[124]</p><p><span>While the mycobiome is even more understudied than the virome, </span><a href="https://www.nature.com/articles/s41579-018-0116-y" rel="">as sequencing it is far more complicated</a><span>, there is</span><a href="https://genomemedicine.biomedcentral.com/articles/10.1186/gm467" rel=""> evidence suggesting a similar level of interplay between it and bacteria</a><span>, implying that efficacy of FMT is also partially tied to it. </span></p><p><span>Given the lack of insight into the bacterial, viral, and fungal elements of the microbiomes, </span><strong>performing FMT</strong><span> </span><strong>is akin to giving patients medications which contain (a small amount of) unknown active ingredients.</strong><span> Outside of the raw danger of performing FMT on patients, </span><a href="https://www.cidrap.umn.edu/fecal-transplant/fda-warns-about-infections-linked-fecal-microbiota-transplants" rel="">which has been an issue before</a><span>, this also means that interpreting the results of microbiome transplants is incredibly hazy, as the </span><strong><a href="https://gut.bmj.com/content/69/3/502" rel="">same underlying therapeutic could be composed of radically different elements based on the donor</a><span>. </span></strong><span>To be fair, there are efforts to fix this. </span><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8039740/" rel="">There is ongoing work in next-generation-FMT’s</a><span>, such as ones that rely on specifically-selected-for strains or metabolic byproducts of a microbiome, instead of the raw contents microbiome itself, increasing both reliability and safety. But as always, very much early days.</span></p><p>As we’ve stated previously, the thrust of a lot of microbiome research is not to simply understand the microbiome, but also to also alter it in clinically meaningful direction. How is this done in practice? </p><p><span>FMT is the typical method for doing this. However, when we previously discussed FMT in the context of establishing causality, we naively accepted that FMT </span><strong>does</strong><span> in fact change the microbiome and does so in ways that are predictable and sustainable. </span></p><p>Is that actually the case?</p><p><strong>Now, FMT does alter the microbiome, that much has been confirmed.</strong><span> However, the extent and durability of these changes can vary substantially depending on factors such as the donor’s microbiome, </span><a href="https://www.nature.com/articles/s41591-022-01913-0" rel="">the pre-existing state of the recipient&#39;s microbiome, </a><strong><a href="https://www.nature.com/articles/s41591-022-01913-0" rel="">and</a></strong><a href="https://www.nature.com/articles/s41591-022-01913-0" rel=""> the donor-recipient microbiome compatibility</a><span>. Moreover, even the way FMT is administered — whether that’s the antibiotic regimen used to prepare patients for FMT or how many times FMT is repeated — can modify the impact of it on the patient. </span><strong>And all these variables-to-take-into-account don’t have correct answers on how to deal with them, they are simply </strong><em><strong>known</strong></em><strong> to be a factor!</strong><span> I’m not giving many exact details here, and that’s because it’s complicated, the whole subject is just filled from top-to-bottom with nuances. There is a great Cell review paper on the whole topic </span><a href="https://www.cell.com/cell-host-microbe/pdf/S1931-3128(23)00125-7.pdf" rel="">here</a><span>, I highly recommend reading that paper if interested in this specific topic. The main takeaway here is that even if a researcher (</span><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7359198/" rel="">or even laymen doing DIY FMT</a><span>) performs FMT and finds positive/negative results, </span><strong>the minute details of exactly </strong><em><strong>how</strong></em><strong> they did it matters a lot.</strong><span> This is a big deal! While FMT may very well be transformative for a lot of conditions, we still don’t have a great grasp on how to administer it ‘correctly’. </span></p><p>Let’s take a big, big step back from something as complex as FMT for a second. How about just focusing on diet? Are those sufficient to alter a microbiome?</p><p><span>Well, diet does influence the microbiome, that’s been known for over a decade now. </span><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8321864/" rel="">Additions of things like fiber, polyphenols, probiotics, and so on do cause gut microbiome shifts in adult humans.</a><span> However, this is missing the full picture: </span><strong><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7266695/" rel="">dietary changes do not seem to reliably alter the microbiome permanently</a></strong><span>. In other words, while sudden changes to one’s diet do alter colony composition (</span><a href="https://www.nature.com/articles/nature12820" rel="">as fast as within 24 hours</a><span>), the microbiome quickly ‘snaps back’ to its usual state </span><strong>even if the diet is maintained for over a year! </strong><span>This implies that modifying microbial compositions cannot be done through passive changes of their energy sources (diet), but require necessitate forced recolonization efforts, as is done with FMT. Given the complexity we’ve seen with using FMT at all, this is an unfortunate result.</span></p><p>To sum it up, we don’t fully understand the therapeutic target (microbiome), the therapy (microbial modification), and the delivery of the therapy (FMT). </p><p>Microbiomes are hard!</p><p><span>Despite the difficulty of working with microbiomes, research on it has continued for decades, researchers desperately trying to find a clear clinical target. But after several decades of work, the primary established utility of microbiome transplants is still only for </span><em>C. Diff</em><span> infections, along with IBD-esque disorders, </span><a href="https://www.thelancet.com/journals/langas/article/PIIS2468-1253(23)00441-7/abstract#:~:text=your%20institutional-,Summary,small%2C%20phase%202%20clinical%20trials." rel="">though the success rate there is dramatically lower than it is for </a><em><a href="https://www.thelancet.com/journals/langas/article/PIIS2468-1253(23)00441-7/abstract#:~:text=your%20institutional-,Summary,small%2C%20phase%202%20clinical%20trials." rel="">C. Diff </a></em><a href="https://www.thelancet.com/journals/langas/article/PIIS2468-1253(23)00441-7/abstract#:~:text=your%20institutional-,Summary,small%2C%20phase%202%20clinical%20trials." rel="">specific conditions</a><span>. </span></p><p><strong>As of June 2024, there are only two FDA-approved microbiome therapeutic. And zero for non-C. Diff conditions. </strong></p><p>What happened to everything else that the microbiome field has tried to tackle? Let’s go over three. There’s a lot more than just these, but these are ones that will often pop up in the literature. </p><p><span>There was strong theoretical evidence, pretty early on, that there was interplay between metabolic conditions (T2D, obesity, cardiac conditions, etc) and the hosts microbiome. After all, </span><a href="https://gut.bmj.com/content/70/6/1174" rel="">if the microbiome is heavily involved in metabolic signals</a><span>, it isn’t a big stretch to imagine that directly altering the microbiome </span><strong>should</strong><span> yield improvements for metabolic conditions. The confirmation that there </span><a href="https://onlinelibrary.wiley.com/doi/full/10.1111/joim.12508" rel="">were microbiota differences between healthy patients and patients with metabolic diseases</a><span> lent some further credence to the theory. </span></p><p>Unfortunately, the theory hasn’t borne out well, the results are mixed-to-negative for human applications.</p><p><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6835402/" rel="">One paper that covered 3 human FMT studies had this to say:</a></p><div><p>Studies reported mixed results with regards to improvement in dysglycemia metabolic parameters. Vrieze, et al. [40] and Kootte, et al. [41] reported that peripheral insulin sensitivity (rate of glucose disappearance, RD) increased at 6 weeks in patients receiving donor FMT versus patients receiving the placebo control. Hepatic insulin sensitivity (endogenous glucose production, EGP) was further assessed in two studies but no statically differences were found. Kootte, et al. [41] observed a lower HbA1c level in patients who received donor FMT at 6 weeks than in patients receiving the placebo control. However, this study indicated the patients who received donor FMT did not show difference in HbA1c or insulin sensitivity (RD) after 18 weeks [41]. This finding suggests that the observed short-term benefit of FMT on dysglycemia was not maintained long-term. </p><p>…</p><p>Included studies demonstrated no differences between patients receiving donor FMT and patients receiving placebo with regards to cholesterol profile, including the levels of total cholesterol, HDL-C, LDL-C and TG. Vrieze, et al. [40] and Kootte, et al. [41] also reported no significant differences on BMI between patients receiving donor FMT and patients receiving placebo followed at 6 weeks.</p></div><p><strong>So, while there are improvements in parameters of interest in the short term (&lt;6 weeks), the effect disappeared over a longer time horizon.</strong><span> Another meta-study looked at nine human FMT studies and found similar results for BMI, cholesterol biomarkers, and a few others:  </span></p><div><p>This meta-analysis investigated studies using FMT for the treatment of obesity and metabolic syndrome and basically concluded that the treatment was effective in the short term. At 2 to 6 weeks after the intervention, mean HbA1c and mean fasting glucose were lower in the FMT group than in the placebo group, although this was a small mean difference. However, mean insulin levels were significantly lower in the FMT group, suggesting a significant improvement in insulin sensitivity.</p><p>…</p><p><span>We found no difference between the FMT and control groups through analysis of the mean differences in clinically significant parameters (</span><a href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0288718#pone-0288718-g004" rel="">Fig 4</a><span>) [past 6 weeks], except for a slight decrease in HbA1c at 12 weeks in the FMT group compared to the placebo group in the study by Yu et al. [</span><a href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0288718#pone.0288718.ref035" rel="">35</a><span>].</span></p></div><p><span>There certainly is </span><strong>something</strong><span> here! Improvements in baseline parameters in the short term still is something. </span><strong>But it does seem to be demonstrably difficult to permanently alter the metabolic aspects of the microbiome,</strong><span> </span><strong>even if you use FMT</strong><span>. This problem shows up a fair bit in the microbiome literature; while initial results may seem good, many FMT’s don’t seem to ‘take’ over the long term. It is still, as of yet, unclear why this may by the case, but I suspect it is strongly related to what we discussed in the ‘Unknown microbiota’ section. </span></p><p>Warning, we’re doing a minor deviation from gut microbiomes here!</p><p><span>It is well established that certain types of cancers can be triggered by certain microbiota interactions, </span><a href="https://journals.plos.org/plospathogens/article?id=10.1371/journal.ppat.1006573" rel="">such as </a><em><a href="https://journals.plos.org/plospathogens/article?id=10.1371/journal.ppat.1006573" rel="">Helicobacter pylori</a></em><a href="https://journals.plos.org/plospathogens/article?id=10.1371/journal.ppat.1006573" rel=""> for stomach cancer</a><span>, allowing a clinician to have a relatively easy diagnostic proxy of risk of cancer from microbiome analysis alone. However, it was generally thought that this, and a few others, were isolated cases. For almost every other type of cancer, a much more invasive biopsy was required to confirm cancer. This was up until a 2020 Nature paper from </span><em>Knight et al</em><span>. titled ‘</span><a href="https://www.nature.com/articles/s41586-020-2095-1" rel="">Microbiome analyses of blood and tissues suggest cancer diagnostic approach</a><span>’ that found, as the title implied, that there was a distinctive microbial signature found </span><strong>within</strong><span> </span><strong>tumors. </strong><span>Insanely, this microbial signature was so distinct, that it could even distinguish </span><strong>between different types of</strong><span> </span><strong>cancer with a near 100% accuracy rate. </strong></p><p><strong>If it was so distinct, why hadn’t we stumbled across it before?</strong><span> Well, microbe populations in general are extremely low biomass and thus hard to </span><strong>discover </strong><span>through sequencing alone, much less characterize. There are of course a few counterexamples to this: namely the gut and skin. But a microbiome anywhere else is hard to stumble across, you need to be intentionally looking for non-human DNA (microbes). This paper was simply the first that looked for it, and also took great pains to ensure the underlying sequence data was accurate — reminding readers that they removed the vast majority of microbial reads they found (&gt;92%) to ensure they only used high confidence readings. </span><strong>Why is this paper useful?</strong><span> Well, the DNA of these tumor microbiomes actually leaked into the bloodstream, allowing for cancer detection by looking for circulating microbial DNA! This would potentially be a whole new type of diagnostic test; allowing patients to avoid a painful, expensive, and invasive biopsy. This already exists and has a name, so-called ‘</span><a href="https://en.wikipedia.org/wiki/Liquid_biopsy" rel="">liquid biopsies</a><span>’. While other companies have been operating in this space for years, most of them are looking for shed-off tumor DNA circulating in the bloodstream as indication of cancer. This paper implied that we could instead rely on the distinctive microbes found alongside certain cancers. </span><strong>In other words,</strong><span> </span><strong>don’t focus on the tumor, focus on the microbiome that grows within the tumor.</strong><span> It was hard to tell what advantages this offered over pure tumor DNA, but it was still an interesting proposition with lots of room to explore. </span></p><p><span>It was a very clever paper with clear clinical implications and, obviously, immediately became famous. As of today, it’s racked up 800+ citations, and the dataset that it used was used for dozens of follow-up papers, </span><a href="https://www.cell.com/cell/fulltext/S0092-8674(22)01127-8" rel="">with one work even discovering a tumor mycobiome</a><span>! The paper authors eventually founded a company on this whole microbiome-sequence-for-liquid-biopsy approach, naming it </span><a href="https://micronoma.com/" rel="">Micronoma</a><span>, which continues to exist today.</span></p><p><strong>Around three years later, the validity of these results was called into question.</strong><span> </span></p><p><span>This skepticism came in the form of a pre-print paper titled ‘</span><a href="https://pubmed.ncbi.nlm.nih.gov/37555750/" rel="">Caution regarding the specificities of pan-cancer microbial structure</a><span>’ from </span><em>Gihawai et al.</em><span>, </span><strong>claiming that the analyses within the original paper were so deeply flawed that the  findings are entirely incorrect.</strong><span> The vast majority of the ‘microbial DNA’ that the paper had identified were, according to the authors, human in origin or the result of environmental contamination. No fraud had been done, merely incorrect analyses of the sequencing data (probably). </span></p><p><span>The authors of the original paper (</span><em>Knight et al)</em><span> quickly responded to it </span><a href="https://www.biorxiv.org/content/10.1101/2023.02.10.528049v1" rel="">with their own paper</a><span>, which had a funnily grandiose line in its abstract:</span></p><p><span>Therefore, despite numerous, high-impact, peer-reviewed research papers that either validated our conclusions or extended them using data we released we carefully considered criticism raised by Gihawi </span><em>et al</em><span>. about potential mishandling of contaminants, batch effects, and machine learning approaches…</span></p><p><span>The authors behind </span><em>Gihawi et al.</em><span> considered this response, and published another paper, this time peer-reviewed, with an even stronger title: ‘</span><a href="https://journals.asm.org/doi/full/10.1128/mbio.01607-23" rel="">Major data analysis errors invalidate cancer microbiome findings</a><span>’. </span></p><p><span>A rebuttal to this was, once again, mounted by </span><em>Knight et al.</em><span> within the same day, which </span><a href="https://github.com/gregpoore/tcga_rebuttal/tree/master" rel="">primarily lives on a Github repository</a><span>.</span></p><p>The rebuttal research trail here ends. The only response from the skeptics I could find was in this response by the PI to a request for commentary of the Github: </p><p><span>Salzberg did release a 2024 paper indirectly targeted at this whole concept, in which he states that cancer sequencing datasets (which the original cancer microbiome paper used) </span><strong><a href="https://www.biorxiv.org/content/10.1101/2024.05.24.595788v1.full.pdf" rel="">have no evidence of containing microbial colonies if you do the correct analysis</a></strong><a href="https://www.biorxiv.org/content/10.1101/2024.05.24.595788v1.full.pdf" rel="">.</a><span> Similarly, the tumor microbiome authors </span><strong>also</strong><span> released a general paper in 2024, claiming that there are, in fact,</span><strong><a href="https://www.nature.com/articles/s41388-024-02974-w" rel=""> such a thing as distinct microbial colonies if you do the correct analysis</a><span>.</span></strong><span> </span></p><p>Uh…so who is right?</p><p><span>I have no clue. Honestly, the fight becomes stranger the more you dig into it. Some points that each side raise feel entirely irrelevant; such as hyper-focusing on the correctness of the 2020 paper over whether there </span><strong>actually</strong><span> is a distinctive tumor microbiome. More confusingly, each side claims minutiae of the opponents rebuttals are actually a misunderstanding of the original point, making it extremely hard to parse who is in the right. Even Derek Lowe </span><a href="https://www.science.org/content/blog-post/arguing-about-cancer-microbiome" rel="">says the whole discussion doesn’t seem at all settled</a><span> circa mid-2023. </span></p><p>Maybe I’ll write a full blog post someday on a deeper dive of the cancer microbiome. But I guess this illustrates a consequence of the immense complexity of microbiomes; 3 rounds of research rebuttals over what should be a theoretically straightforward question (do tumors have a distinctive microbiome?) yielded no conclusive answers.  </p><p>If I was forced to make a call, who is right? The whole topic is, frankly, far out of my paygrade. I’ve read quite a few Twitter arguments over this between people who are actively publishing in the metagenomic field and were still unable to conclude/agree on anything substantive. My vote goes to the skeptics, but only because my prior is on microbiome research being fishy in general. </p><p><span>As is often the case with controversial biotech startups, the market is ultimately quite good at deciding on the validity of it. Time will tell whether the company formed around this, </span><a href="https://micronoma.com/" rel="">Micronoma</a><span>, comes up with anything useful in the end. </span></p><p>Let’s end on something a bit more positive!</p><p><span>The biochemical interplay between gut microbiota and the central nervous system was a minor revolution in the field, at least from the outside-looking-in. This had a pretty obvious founding basis: </span><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5414803/" rel="">with the amount of mood-altering hormones it secrete (more than 90% of all serotonin in your body!), gut microbiota are basically equivalent to an endocrine organ.</a><span> </span><strong>This had a similarly obvious therapeutic angle: modify the microbiome, modify neurological function.</strong><span> Similarly with metabolic conditions, </span><a href="https://onlinelibrary.wiley.com/doi/abs/10.1111/nmo.12378" rel="">correlational findings that individuals with depression display unique microbiota further led credence to this theory.</a><span> Similar discoveries were made for other neurological disorders, such as </span><a href="https://www.nature.com/articles/s41398-023-02325-5" rel="">anxiety</a><span> and even </span><a href="https://onlinelibrary.wiley.com/doi/abs/10.1111/nmo.12378" rel="">schizophrenia</a><span>. </span></p><p>How has this borne out in terms of actual clinical results? </p><p>Surprisingly positively! </p><p><a href="https://www.cambridge.org/core/journals/international-psychogeriatrics/article/fecal-microbiota-transplantation-in-an-elderly-patient-with-mental-depression/F60C1C353A935643E2328A4858928DEC" rel="">A case study of elderly patient with anxiety, depression, and loss-of-appetite showed massive improvement after FMT, and she continued to display the same improvement 12-months-post FMT.</a><span> But, of course, n-of-1 studies are always a little suspect. </span></p><p><a href="https://pubmed.ncbi.nlm.nih.gov/29684865/" rel="">A non-controlled retroactive study on the psychiatric impact of FMT — using patients treated with FMT for GI issues — found general improvement of anxiety and depressive symptoms.</a><span> The study only lasted for 4 weeks though.</span></p><p><a href="https://www.sciencedirect.com/science/article/pii/S2452231719300077#t0010" rel="">A controlled study of 42 patients with chronic fatigue syndrome (CFS) found improvement in CFS symptoms when using FMT versus oral probiotics over several months.</a><span>  Of note is that assessment of symptom improvement is qualitative assessment by physicians </span><strong>and not blinded.</strong><span> </span></p><p><span>Generally, the case for modifying the gut microbiome for neurological function is somewhat strong. Unlike with metabolic conditions or cancer microbiomes, the evidence seems quite consistent across papers, implies a degree of causation, and, </span><a href="https://journals.sagepub.com/doi/10.1177/07067437221150508" rel="">as a double-blinded randomized control study states</a><span>, is quite safe. </span></p><p><span>But why hasn’t it leaked into actual therapeutics? As of today, there is no officially approved drug that uses FMT for treatment of neurological conditions. I could hand-wave about how we need larger, multi-center, randomized controlled trials, which are expensive, and companies may not see enough value to invest, but I believe there’s a less scientific/clinical reason why FMT for neurological development has stalled: </span><strong><a href="https://pubmed.ncbi.nlm.nih.gov/29064350/" rel="">you technically cannot patent FMT-based therapeutics</a><span>, since it is a ‘living organism’, just like cell lines. This massively reduces the incentive for companies to invest here! </span></strong><span>This is a somewhat funny, if a little dark, example of how a therapy could be indefinitely stalled via a law that seems completely unrelated to it. To be clear, this clearly isn’t stopping microbiome therapies at large! </span><a href="https://www.vedantabio.com/" rel="">Vedanta Biosciences</a><span>, which focus on microbial therapies, did announce in late 2023 that they were successfully granted patents for their single-species microbe therapies, and there are two other FDA-approved microbiome drugs that are, presumbly, patented as well. So, these patent issues aren’t an absolute blocker, but does perhaps increase the time-to-market. </span></p><p>This essay covered a lot. Let’s do a quick TLDR: </p><ol><li><p>Characterizing the microbiome, both in terms of species present and their functional impact, remains extremely challenging due to technical limitations, unknown unknowns, and the sheer complexity of the system.</p></li><li><p>Establishing causality between the microbiome and disease is difficult. Most studies are purely associative, and even more rigorous approaches like MR and animal studies have significant limitations.</p></li><li><p>FMT, the main therapeutic approach, is still poorly understood in terms of durability, reproducibility, and long-term effects. We&#39;re essentially giving patients a complex, largely uncharacterized cocktail and hoping for the best.</p></li><li><p>Despite early hype, most clinical applications have failed to pan out so far, with the exception of C. diff infections. Results for metabolic conditions are mixed, and while microbial treatments targeting the gut-brain axis may have promise, it is likely that patent issues in the microbiome space more broadly will hurt its chances to reach the market. </p></li><li><p>Even supposedly groundbreaking findings, like the discovery of tumor microbiomes, remain mired in controversy and may not hold up to further scrutiny.</p></li></ol><p>What’s next? </p><p>I’m not sure. I generally feel extremely skeptical of this field for now. Everything here feels so deeply messy, hard to understand, and uncontrollable through the usual Western medicine approaches of throwing a therapeutic at it. Clearly, the microbiome is important! But modifying it is challenging, fraught with issues, and, where it does somewhat work, regulatory issues cripple it from moving quickly. </p><p><span>It’s possible that this may change soon! But, unlike most of my other pieces, I am quite skeptical that anything major here will happen soon, barring perhaps gut-brain-axis FMT therapeutics establishing efficacy in longitudinal double-blind trials and getting past the patent hurdle. The field has been working away for decades now with few actionable results besides curing </span><em>C. Diff, </em><span>and it doesn’t feel like anything has especially changed in the last few years to really change that trajectory. It is similarly unlikely that microbiome research really </span><strong>stops</strong><span> in any capacity, there is a fair bit of money still pushing it forwards. I suspect the next decade will involving shying away from the complexity of typical FMT’s, and instead focus on so-called ‘</span><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10724401/" rel="">next-generation FMT</a><span>’. These sets of methods — which isolate specific bacterial or viral components of a feces sample, rather than use all of it — will potentially allow microbiome therapeutics to reach previously impossible predictability, standardization, and efficacy. Perhaps this alone will address many of the issues we’ve raised here! Time will tell. </span></p><p><em>Thank you for reading this post! Every two weeks, I’ll be posting something about biology, ML, or the intersection of the two. If this interests you, please subscribe! If you enjoyed this post, here are some other ones you may like as well: </em></p><div data-component-name="DigestPostEmbed"><div><a href="https://www.abhishaike.com/p/a-primer-on-why-computational-predictive" target="_blank" rel="noopener" class=""><div><picture><source type="image/webp" srcset="https://substackcdn.com/image/fetch/w_140,h_140,c_fill,f_webp,q_auto:good,fl_progressive:steep,g_auto/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F5a4454bc-e4d7-4a9b-9272-dcf74ebf6636_850x634.png"/><img src="https://substackcdn.com/image/fetch/w_140,h_140,c_fill,f_auto,q_auto:good,fl_progressive:steep,g_auto/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F5a4454bc-e4d7-4a9b-9272-dcf74ebf6636_850x634.png" sizes="100vw" alt="A primer on why computational predictive toxicology is hard" width="140" height="140"/></picture></div></a></div></div><div data-component-name="DigestPostEmbed"><div><a href="https://www.abhishaike.com/p/a-primer-on-molecular-dynamics" target="_blank" rel="noopener" class=""><div><picture><source type="image/webp" srcset="https://substackcdn.com/image/fetch/w_140,h_140,c_fill,f_webp,q_auto:good,fl_progressive:steep,g_auto/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F69f5b5b7-cccd-40fb-8a45-0ea46ae78c0d_3350x966.png"/><img src="https://substackcdn.com/image/fetch/w_140,h_140,c_fill,f_auto,q_auto:good,fl_progressive:steep,g_auto/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F69f5b5b7-cccd-40fb-8a45-0ea46ae78c0d_3350x966.png" sizes="100vw" alt="A primer on molecular dynamics" width="140" height="140"/></picture></div></a></div></div><div data-component-name="DigestPostEmbed"><div><a href="https://www.abhishaike.com/p/a-primer-on-my-favorite-pessimistic" target="_blank" rel="noopener" class=""><div><picture><source type="image/webp" srcset="https://substackcdn.com/image/fetch/w_140,h_140,c_fill,f_webp,q_auto:good,fl_progressive:steep,g_auto/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F67e2677b-2539-4a09-9481-eb6bb2e3036a_1400x700.png"/><img src="https://substackcdn.com/image/fetch/w_140,h_140,c_fill,f_auto,q_auto:good,fl_progressive:steep,g_auto/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F67e2677b-2539-4a09-9481-eb6bb2e3036a_1400x700.png" sizes="100vw" alt="A primer on my favorite pessimistic scientific articles" width="140" height="140"/></picture></div></a></div></div></div></div></div>
  </body>
</html>
