<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <title>James Routley | Feed</title>
    <link
      rel="stylesheet"
      type="text/css"
      href="../styles.css"
      media="screen"
    />
  </head>
  <body>
    <a href="/index.html">Back</a>
    <a href="https://www.nature.com/articles/s41586-024-08165-7">Original</a>
    <h1>Adipose tissue retains an epigenetic memory of obesity after weight loss</h1>
    
    <div id="readability-page-1" class="page"><div>
                    
                        <section data-title="Main"><div id="Sec1-section"><h2 id="Sec1">Main</h2><div id="Sec1-content"><p>Obesity and its related comorbidities represent substantial health risks<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 1" title="Zimmet, P., Alberti, K. G. M. M. &amp; Shaw, J. Global and societal implications of the diabetes epidemic. Nature 414, 782–787 (2001)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR1" id="ref-link-section-d15230973e737">1</a></sup>. A primary clinical objective in managing obesity is to achieve appreciable weight loss (WL), typically through rigorous dietary and lifestyle interventions, pharmaceutical treatments or bariatric surgery (BaS)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 2" title="Wolfe, B. M., Kvach, E. &amp; Eckel, R. H. Treatment of obesity: weight loss and bariatric surgery. Circ. Res. 118, 1844 (2016)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR2" id="ref-link-section-d15230973e741">2</a></sup>. Strategies relying on behavioural and dietary changes frequently only result in short-term WL and are susceptible to the ‘yo-yo’ effect, in which individuals regain weight over time<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 3" title="Nordmo, M., Danielsen, Y. S. &amp; Nordmo, M. The challenge of keeping it off, a descriptive systematic review of high‐quality, follow‐up studies of obesity treatments. Obes. Rev. 21, e12949 (2020)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR3" id="ref-link-section-d15230973e745">3</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 5" title="Franz, M. J. et al. Weight-loss outcomes: a systematic review and meta-analysis of weight-loss clinical trials with a minimum 1-year follow-up. J. Am. Diet. Assoc. 107, 1755–1767 (2007)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR5" id="ref-link-section-d15230973e748">5</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 6" title="Wilding, J. P. H. et al. Weight regain and cardiometabolic effects after withdrawal of semaglutide: the STEP 1 trial extension. Diabetes Obes. Metab. 24, 1553–1564 (2022)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR6" id="ref-link-section-d15230973e751">6</a></sup>. This recurrent pattern may be partially attributable to an (obesogenic) metabolic memory that persists even after notable WL<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 4" title="Contreras, R. E., Schriever, S. C. &amp; Pfluger, P. T. Physiological and epigenetic features of yoyo dieting and weight control. Front. Genet. 10, 1015 (2019)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR4" id="ref-link-section-d15230973e755">4</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Izquierdo, A. G. &amp; Crujeiras, A. B. Obesity-related epigenetic changes after bariatric surgery. Front. Endocrinol. 10, 232 (2019)." href="#ref-CR7" id="ref-link-section-d15230973e758">7</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Mileti, E. et al. Human white adipose tissue displays selective insulin resistance in the obese state. Diabetes 70, 1486–1497 (2021)." href="#ref-CR8" id="ref-link-section-d15230973e758_1">8</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Petrus, P. et al. Transforming growth factor-β3 regulates adipocyte number in subcutaneous white adipose tissue. Cell Rep. 25, 551–560.e5 (2018)." href="#ref-CR9" id="ref-link-section-d15230973e758_2">9</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 10" title="Löfgren, P. et al. Long-term prospective and controlled studies demonstrate adipose tissue hypercellularity and relative leptin deficiency in the postobese state. J. Clin. Endocrinol. Metab. 90, 6207–6213 (2005)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR10" id="ref-link-section-d15230973e761">10</a></sup> or metabolic improvements<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Reddy, M. A., Zhang, E. &amp; Natarajan, R. Epigenetic mechanisms in diabetic complications and metabolic memory. Diabetologia 58, 443–455 (2015)." href="#ref-CR11" id="ref-link-section-d15230973e765">11</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Chalmers, J. &amp; Cooper, M. E. UKPDS and the legacy effect. N. Engl. J. Med. 359, 1618–1620 (2008)." href="#ref-CR12" id="ref-link-section-d15230973e765_1">12</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 13" title="Holman, R. R., Paul, S. K., Bethel, M. A., Matthews, D. R. &amp; Neil, H. A. W. 10-Year follow-up of intensive glucose control in type 2 diabetes. N. Engl. J. Med. 359, 1577–1589 (2008)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR13" id="ref-link-section-d15230973e768">13</a></sup>. Indeed, lasting phenotypic changes from previous metabolic states, that is, metabolic memory, have been reported in mouse adipose tissue (AT) or the stromal vascular fraction (SVF)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Guerra-Cantera, S. et al. Sex differences in metabolic recuperation after weight loss in high fat diet-induced obese mice. Front. Endocrinol. 12, 796661 (2021)." href="#ref-CR14" id="ref-link-section-d15230973e773">14</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Hahn, O. et al. A nutritional memory effect counteracts the benefits of dietary restriction in old mice. Nat. Metab. 1, 1059–1073 (2019)." href="#ref-CR15" id="ref-link-section-d15230973e773_1">15</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 16" title="Schmitz, J. et al. Obesogenic memory can confer long-term increases in adipose tissue but not liver inflammation and insulin resistance after weight loss. Mol. Metab. 5, 328–339 (2016)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR16" id="ref-link-section-d15230973e776">16</a></sup>, whereas in liver these were reversible<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Hahn, O. et al. A nutritional memory effect counteracts the benefits of dietary restriction in old mice. Nat. Metab. 1, 1059–1073 (2019)." href="#ref-CR15" id="ref-link-section-d15230973e780">15</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Schmitz, J. et al. Obesogenic memory can confer long-term increases in adipose tissue but not liver inflammation and insulin resistance after weight loss. Mol. Metab. 5, 328–339 (2016)." href="#ref-CR16" id="ref-link-section-d15230973e780_1">16</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 17" title="Siersbæk, M. et al. High fat diet-induced changes of mouse hepatic transcription and enhancer activity can be reversed by subsequent weight loss. Sci. Rep. 7, 40220 (2017)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR17" id="ref-link-section-d15230973e783">17</a></sup>. Persistent alterations after WL in the immune compartment<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 18" title="Cottam, M. A., Caslin, H. L., Winn, N. C. &amp; Hasty, A. H. Multiomics reveals persistence of obesity-associated immune cell phenotypes in adipose tissue during weight loss and weight regain in mice. Nat. Commun. 13, 2950 (2022)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR18" id="ref-link-section-d15230973e787">18</a></sup>, and transcriptional and functional memory of obesity in endothelial cells of many organs<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Bondareva, O. et al. Single-cell profiling of vascular endothelial cells reveals progressive organ-specific vulnerabilities during obesity. Nat. Metab. 4, 1591–1610 (2022)." href="#ref-CR19" id="ref-link-section-d15230973e791">19</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Kowluru, R. A. &amp; Mohammad, G. Epigenetics and mitochondrial stability in the metabolic memory phenomenon associated with continued progression of diabetic retinopathy. Sci. Rep. 10, 6655 (2020)." href="#ref-CR20" id="ref-link-section-d15230973e791_1">20</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Tallapragada, D. S. P., Karpe, P. A. &amp; Tikoo, K. Long-lasting partnership between insulin resistance and endothelial dysfunction: role of metabolic memory: high fat diet and metabolic memory. Br. J. Pharmacol. 172, 4012–4023 (2015)." href="#ref-CR21" id="ref-link-section-d15230973e791_2">21</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 22" title="El-Osta, A. et al. Transient high glucose causes persistent epigenetic changes and altered gene expression during subsequent normoglycemia. J. Exp. Med. 205, 2409–2417 (2008)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR22" id="ref-link-section-d15230973e794">22</a></sup>, have also been reported.</p><p>Epigenetic mechanisms and modifications are essential for development, differentiation and identity maintenance of adipocytes in vitro and in vivo<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Madsen, J. G. S. et al. Highly interconnected enhancer communities control lineage-determining genes in human mesenchymal stem cells. Nat. Genet. 52, 1227–1238 (2020)." href="#ref-CR23" id="ref-link-section-d15230973e801">23</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Matsumura, Y. et al. H3K4/H3K9me3 bivalent chromatin domains targeted by lineage-specific DNA methylation pauses adipocyte differentiation. Mol. Cell 60, 584–596 (2015)." href="#ref-CR24" id="ref-link-section-d15230973e801_1">24</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Roh, H. C. et al. Adipocytes fail to maintain cellular identity during obesity due to reduced PPARγ activity and elevated TGFβ-SMAD signaling. Mol. Metab. 42, 101086 (2020)." href="#ref-CR25" id="ref-link-section-d15230973e801_2">25</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Roh, H. C. et al. Warming induces significant reprogramming of beige, but not brown, adipocyte cellular identity. Cell Metab. 27, 1121–1137.e5 (2018)." href="#ref-CR26" id="ref-link-section-d15230973e801_3">26</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 27" title="Siersbæk, R. et al. Transcription factor cooperativity in early adipogenic hotspots and super-enhancers. Cell Rep. 7, 1443–1455 (2014)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR27" id="ref-link-section-d15230973e804">27</a></sup>, but are also expected to be crucial contributors to the cellular memory of obesity<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 4" title="Contreras, R. E., Schriever, S. C. &amp; Pfluger, P. T. Physiological and epigenetic features of yoyo dieting and weight control. Front. Genet. 10, 1015 (2019)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR4" id="ref-link-section-d15230973e808">4</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Izquierdo, A. G. &amp; Crujeiras, A. B. Obesity-related epigenetic changes after bariatric surgery. Front. Endocrinol. 10, 232 (2019)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR7" id="ref-link-section-d15230973e811">7</a></sup>. For example, lasting chromatin accessibility changes have been associated with pathological memory of obesity in mouse myeloid cells<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 28" title="Hata, M. et al. Past history of obesity triggers persistent epigenetic changes in innate immunity and exacerbates neuroinflammation. Science 379, 45–62 (2023)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR28" id="ref-link-section-d15230973e815">28</a></sup> and, also, cold exposure studies have indicated the existence of (epigenetic) cellular memory<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 26" title="Roh, H. C. et al. Warming induces significant reprogramming of beige, but not brown, adipocyte cellular identity. Cell Metab. 27, 1121–1137.e5 (2018)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR26" id="ref-link-section-d15230973e819">26</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 29" title="Rosenwald, M., Perdikari, A., Rülicke, T. &amp; Wolfrum, C. Bi-directional interconversion of brite and white adipocytes. Nat. Cell Biol. 15, 659–667 (2013)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR29" id="ref-link-section-d15230973e822">29</a></sup>. Hitherto, most human studies have focused on DNA methylation analysis in bulk tissues or whole blood to assess putative cellular memory<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Barres, R. et al. Weight loss after gastric bypass surgery in human obesity remodels promoter methylation. Cell Rep. 3, 1020–1027 (2013)." href="#ref-CR30" id="ref-link-section-d15230973e826">30</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Benton, M. C. et al. An analysis of DNA methylation in human adipose tissue reveals differential modification of obesity genes before and after gastric bypass and weight loss. Genome Biol. 16, 8 (2015)." href="#ref-CR31" id="ref-link-section-d15230973e826_1">31</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Talukdar, F. R. et al. Bariatric surgery-induced weight loss and associated genome-wide DNA-methylation alterations in obese individuals. Clin. Epigenetics 14, 176 (2022)." href="#ref-CR32" id="ref-link-section-d15230973e826_2">32</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 33" title="Chen, Z. et al. Epigenomic profiling reveals an association between persistence of DNA methylation and metabolic memory in the DCCT/EDIC type 1 diabetes cohort. Proc. Natl Acad. Sci. USA 113, E3002–E3011 (2016)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR33" id="ref-link-section-d15230973e829">33</a></sup>. These reports might be confounded by variations in cell type composition, which are poorly characterized in the AT during WL, and therefore serve foremost as indicators of cellular epigenetic memory.</p><p>In summary, it remains unresolved whether individual cells retain a metabolic memory and whether it is conferred through epigenetic mechanisms. Here, we set out to address this by first performing single-nucleus RNA sequencing (snRNA-seq) of AT from individuals living with obesity before and after significant WL, as well as lean, obese and formerly obese mice, confirming the presence of retained transcriptional changes, and, second, by characterizing the epigenome of mouse adipocytes, which revealed the long-term persistence of an epigenetic obesogenic memory.</p></div></div></section><section data-title="Transcriptional changes in human AT"><div id="Sec2-section"><h2 id="Sec2">Transcriptional changes in human AT</h2><div id="Sec2-content"><p>To explore whether signatures of previous obesogenic states persist in humans after appreciable WL, we obtained subcutaneous AT (scAT) and omental AT (omAT) biopsies from individuals with healthy weight who have never had obesity (called healthy weight here) and people living with obesity (but without diabetes) before (T0) and 2 yr after (T1) BaS from multiple independent studies (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig1">1a</a>). The omAT samples were from the multicentre two-step surgery (MTSS) study (<i>n</i> = 5 lean individuals, 1 male, 4 females; <i>n</i> = 8 individuals with obesity, 2 males, 6 females) and Leipzig two-step surgery (LTSS) study (<i>n</i> = 5 lean individuals, 2 males, 3 females; <i>n</i> = 5 individuals with obesity, 2 males, 3 females). Only patients exhibiting a minimum of 25% body mass index (BMI) reduction were included into our study (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig1">1a,b</a> and Extended Data Table <a data-track="click" data-track-label="link" data-track-action="table anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Tab1">1</a>). We performed snRNA-seq on pooled omAT per group and could annotate, on the basis of published data<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 34" title="Emont, M. P. et al. A single-cell atlas of human and mouse white adipose tissue. Nature 603, 926–933 (2022)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR34" id="ref-link-section-d15230973e866">34</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 35" title="Massier, L. et al. An integrated single cell and spatial transcriptomic map of human white adipose tissue. Nat. Commun. 14, 1438 (2023)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR35" id="ref-link-section-d15230973e869">35</a></sup>, 18 cell clusters in the omAT samples (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig1">1c</a> and Extended Data Figs. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig6">1a</a> and <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig7">2a–d</a>), including adipocytes, adipocyte progenitor cells (APCs), mesothelial cells, immune cells and endothelial cells. Although we did not observe consistent cellular composition differences between T0 and T1 in omAT, we observed inter-individual cellular composition variations after single nucleotide polymorphism (SNP)-based demultiplexing, possibly also affected by sampling during surgery (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig7">2e,f</a>). Notably, cell type-specific gene expression analysis revealed that many differentially expressed genes (DEGs) at T0 (obese versus healthy weight) were also deregulated at T1 in both studies (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig1">1d</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig6">1b,c</a>). We next performed the same analysis with scAT biopsies from the LTSS study (<i>n</i> = 5 lean individuals, 2 males, 3 females; <i>n</i> = 5 individuals with obesity, 2 males, 3 females) and NEFA trial (ClinicalTrials.gov registration no. NCT01727245; <i>n</i> = 8 lean individuals, all female; <i>n</i> = 7 individuals with obesity, all female), including only patients exhibiting a minimum of 25% BMI reduction (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig1">1a,b</a> and Extended Data Table <a data-track="click" data-track-label="link" data-track-action="table anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Tab1">1</a>). We annotated 13 cell clusters for scAT (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig1">1e</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig6">1d</a>), including APCs, adipocytes, endothelial cells and immune cells, on the basis of published markers<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Emont, M. P. et al. A single-cell atlas of human and mouse white adipose tissue. Nature 603, 926–933 (2022)." href="#ref-CR34" id="ref-link-section-d15230973e917">34</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Massier, L. et al. An integrated single cell and spatial transcriptomic map of human white adipose tissue. Nat. Commun. 14, 1438 (2023)." href="#ref-CR35" id="ref-link-section-d15230973e917_1">35</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Maniyadath, B., Zhang, Q., Gupta, R. K. &amp; Mandrup, S. Adipose tissue at single-cell resolution. Cell Metab. 35, 386–413 (2023)." href="#ref-CR36" id="ref-link-section-d15230973e917_2">36</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 37" title="Merrick, D. et al. Identification of a mesenchymal progenitor cell hierarchy in adipose tissue. Science 364, eaav2501 (2019)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR37" id="ref-link-section-d15230973e920">37</a></sup> (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig8">3a–d</a>). We did not observe consistent cellular composition differences between T0 and T1 in scAT (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig8">3e,f</a>). However, similar to omAT we found in both studies that many cell types retained transcriptional differences from T0 to T1 (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig1">1f</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig6">1e,f</a>). A further detailed analysis of cell type-specific gene expression changes in omAT and scAT showed that transcriptional deregulation during obesity was most pronounced in adipocytes, APCs and endothelial cells (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig6">1g–j</a>). In line with this observation, the absolute number of retained DEGs from T0 to T1 was highest in these cell types as well (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig6">1k</a>). Given that adipocytes showed strong retainment of transcriptional differences in each individual sample, we integrated the snRNA-seq data of all adipocytes from the omAT and scAT studies, respectively (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig6">1l,m</a>), and performed differential gene expression analysis. Pooled omAT adipocytes displayed a strong retention of downregulated DEGs (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig1">1g</a>), including relevant metabolic genes<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Boucher, J. et al. Impaired thermogenesis and adipose tissue development in mice with fat-specific disruption of insulin and IGF-1 signalling. Nat. Commun. 3, 902 (2012)." href="#ref-CR38" id="ref-link-section-d15230973e950">38</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Csaki, L. S. et al. Lipin-1 and lipin-3 together determine adiposity in vivo. Mol. Metab. 3, 145–154 (2014)." href="#ref-CR39" id="ref-link-section-d15230973e950_1">39</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Kang, H. S. et al. IDH1-dependent α-KG regulates brown fat differentiation and function by modulating histone methylation. Metabolism 105, 154173 (2020)." href="#ref-CR40" id="ref-link-section-d15230973e950_2">40</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 41" title="Seo, D. H. et al. Effects of a phosphodiesterase inhibitor on the browning of adipose tissue in mice. Biomedicines 10, 1852 (2022)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR41" id="ref-link-section-d15230973e953">41</a></sup> such as <i>IGF1</i>, <i>LPIN1</i>, <i>IDH1</i> or <i>PDE3A</i> (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig1">1h</a>). Similarly, the retention of downregulated DEGs in scAT adipocytes was pronounced (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig1">1i</a>) and included relevant metabolic genes<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 38" title="Boucher, J. et al. Impaired thermogenesis and adipose tissue development in mice with fat-specific disruption of insulin and IGF-1 signalling. Nat. Commun. 3, 902 (2012)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR38" id="ref-link-section-d15230973e976">38</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Petrus, P. et al. Glutamine links obesity to inflammation in human white adipose tissue. Cell Metab. 31, 375–390.e11 (2020)." href="#ref-CR42" id="ref-link-section-d15230973e979">42</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Ferguson, B. S., Nam, H., Stephens, J. M. &amp; Morrison, R. F. Mitogen-dependent regulation of DUSP1 governs ERK and p38 signaling during early 3T3-L1 adipocyte differentiation. J. Cell. Physiol. 231, 1562–1574 (2016)." href="#ref-CR43" id="ref-link-section-d15230973e979_1">43</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 44" title="Hauffe, R. et al. GPx3 dysregulation impacts adipose tissue insulin receptor expression and sensitivity. JCI Insight 5, e136283 (2020)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR44" id="ref-link-section-d15230973e982">44</a></sup> such as <i>IGF1</i>, <i>DUSP1</i>, <i>GPX3</i> and <i>GLUL</i> (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig1">1j</a>). Gene set enrichment analysis (GSEA) of retained DEGs in adipocytes of each study showed persistent downregulation of pathways linked to adipocyte metabolism and function (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig9">4a–d</a>) and persistent upregulation of pathways linked to fibrosis (related to <i>TGFβ</i> signalling) and apoptosis (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig9">4e–h</a>). These results indicate that obesity induces cellular and transcriptional (obesogenic) changes in the AT, which are not resolved following significant WL.</p><div data-test="figure" data-container-section="figure" id="figure-1" data-title="Human AT retains cellular transcriptional changes after BaS-induced WL."><figure><figcaption><b id="Fig1" data-test="figure-caption-text">Fig. 1: Human AT retains cellular transcriptional changes after BaS-induced WL.</b></figcaption><div><div><a data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="https://g-w1.github.io/articles/s41586-024-08165-7/figures/1" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41586-024-08165-7/MediaObjects/41586_2024_8165_Fig1_HTML.png?as=webp"/><img aria-describedby="Fig1" src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41586-024-08165-7/MediaObjects/41586_2024_8165_Fig1_HTML.png" alt="figure 1" loading="lazy" width="685" height="536"/></picture></a></div><div data-test="bottom-caption" id="figure-1-desc"><p><b>a</b>, omAT and scAT biopsies were collected from people living with obesity during BaS (T0) and 2 yr post-surgery (T1). Only individuals that had lost at least 25% of BMI compared with T0 were included. omAT and scAT biopsies were collected from healthy weight/lean individuals from the same studies (MTSS, LTSS and NEFA). <b>b</b>, Sex, age, starting BMI and BMI loss of lean donors and donors with obesity. <b>c</b>, Uniform manifold approximation and projection (UMAP) of 22,742 nuclei representing omAT pools from lean subjects (<i>n</i> = 5; 2 males, 3 females) and paired omAT from T0 and T1 (<i>n</i> = 5 each; 2 males, 3 females) from LTSS. <b>d</b>, Proportion of retained transcriptional changes in highly abundant cell types of LTSS omAT. <b>e</b>, UMAP of 15,347 nuclei representing scAT pools from lean subjects (<i>n</i> = 5; 2 males, 3 females) and paired scAT from T0 and T1 (<i>n</i> = 5 each; 2 males, 3 females) from LTSS. <b>f</b>, Proportion of retained transcriptional changes in highly abundant cell types of LTSS scAT. <b>g</b>, Proportion of retained transcriptional changes in integrated omAT adipocytes of LTSS and MTSS omAT. <b>h</b>, Normalized expression of selected memory DEGs in omAT adipocytes. <b>i</b>, Proportion of retained transcriptional changes in integrated omAT adipocytes of LTSS and NEFA scAT. <b>j</b>, Normalized expression of selected memory DEGs in scAT adipocytes. Wilcoxon rank-sum test with adjusted <i>P</i> &lt; 0.01 by the Bonferroni correction method, and log<sub>2</sub> fold change (log<sub>2</sub>FC) &gt; ±0.5 was used for DEG identification in <b>d</b>, <b>f</b>, <b>g</b>, <b>h</b>, <b>i</b> and <b>j</b>. DCs, dendritic cells; EndoCs, endothelial cells; EndoACs, arteriolar EndoCs; EndoSCs, stalk EndoCs; EndoVCs, venular EndoCs; LECs, lymphatic EndoCs; FAPs, fibro-adipogenic progenitors; Macro, macrophages; MastCs, mast cells; MesoCs, mesothelial cells; NeurCs, neuronal-like cells; SMCs, (vascular) smooth muscle cells; NA, not applicable; m/f, male/female. Credit: <b>a</b>, Copyright 2017—<a href="http://Simplemaps.com">Simplemaps.com</a> (<a href="https://simplemaps.com/resources/svg-maps">https://simplemaps.com/resources/svg-maps</a>).</p><p><a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#MOESM3">Source Data</a></p></div></div></figure></div></div></div></section><section data-title="Pathophysiology mostly resolves after WL"><div id="Sec3-section"><h2 id="Sec3">Pathophysiology mostly resolves after WL</h2><div id="Sec3-content"><p>To investigate the molecular mechanisms and pathophysiological importance of this putative metabolic memory of obesity, we assessed WL in an experimental animal model (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig2">2a</a>). The 6-week-old male mice were fed a high-fat diet (HFD; 60% kcal from fat) or low-fat chow diet (10% kcal from fat) for 12 (H and C) or 25 weeks (HH and CC_l). Subsequently, we switched the diet to a standard chow diet (HC, CC_s, HHC, CCC), leading to weight normalization in 4–8 weeks (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig2">2b,c</a>). Glucose tolerance was impaired in H but not in HH mice (compared with age-matched controls), whereas insulin sensitivity was lower in HH but not in H mice (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig10">5a,b</a>). Fasting blood glucose levels were greater in both groups (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig10">5c</a>). WL restored insulin sensitivity in HHC mice, whereas HC mice still showed impaired glucose tolerance (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig7">2d,e</a>). Fasting glucose levels were normalized by WL in both groups, matching those of control mice (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig7">2f</a>). After WL, hyperinsulinemia was resolved in HC mice, but only diminished in HHC mice (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig10">5g–i</a>). Leptin levels, which were elevated in obese mice, returned to control levels after WL (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig10">5j</a>). Energy expenditure and food intake showed no differences between HC and CC_s mice after WL (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig10">5k,l</a>). Liver triglyceride accumulation was normalized (to control levels) in HC, and most HHC, mice. (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig10">5m,n</a>). Similarly, C and H mice, and CC_s and HC mice, did not differ in the amount of lean mass nor did HC mice lose lean mass (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig10">5o</a>). Obese H mice had larger subcutaneous inguinal AT (ingAT), epididymal AT (epiAT) and brown AT (BAT) depots than corresponding control mice (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig10">5p,q</a>). ingAT and BAT depot sizes normalized after WL. In line with a recent report, epiAT of HC mice was smaller than that of controls after WL<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 18" title="Cottam, M. A., Caslin, H. L., Winn, N. C. &amp; Hasty, A. H. Multiomics reveals persistence of obesity-associated immune cell phenotypes in adipose tissue during weight loss and weight regain in mice. Nat. Commun. 13, 2950 (2022)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR18" id="ref-link-section-d15230973e1169">18</a></sup>. Interestingly, the phenomenon of epiAT shrinkage was already observed during obesity in 25-week HFD-fed (HH) mice, as previously reported<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 45" title="Strissel, K. J. et al. Adipocyte death, adipose tissue remodeling, and obesity complications. Diabetes 56, 2910–2918 (2007)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR45" id="ref-link-section-d15230973e1173">45</a></sup>, and maintained after WL in HHC mice (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig10">5r–v</a>). Adipocyte sizes varied between depots, and adipocytes were enlarged in ingAT of H and HH mice and normalized after WL in HC, but not in HHC, mice (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig10">5w,x</a>). epiAT adipocytes were also enlarged and shrunk to normal sizes in H and HC mice, respectively, whereas in HH and HHC mice adipocytes were of equal size, probably owing to the tissue shrinkage (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig10">5v,y</a>). The epiAT of obese mice (H and HH) showed immune cell infiltration and apical fibrosis, which partially improved after WL in HC, but not HHC, mice (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig10">5t–v</a>). Masson’s trichrome staining showed more collagen deposition in epiAT after WL (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig10">5z</a>). Overall, after WL, only a few mild metabolic impartments persisted, including glucose intolerance in HC mice, hyperinsulinemia and slight liver steatosis in HHC mice and a notable decrease in epiAT depot size after WL in both groups.</p><div data-test="figure" data-container-section="figure" id="figure-2" data-title="Transcriptional changes persist WL induced (partial) remodelling of epiAT."><figure><figcaption><b id="Fig2" data-test="figure-caption-text">Fig. 2: Transcriptional changes persist WL induced (partial) remodelling of epiAT.</b></figcaption><div><div><a data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="https://g-w1.github.io/articles/s41586-024-08165-7/figures/2" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41586-024-08165-7/MediaObjects/41586_2024_8165_Fig2_HTML.png?as=webp"/><img aria-describedby="Fig2" src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41586-024-08165-7/MediaObjects/41586_2024_8165_Fig2_HTML.png" alt="figure 2" loading="lazy" width="685" height="674"/></picture></a></div><div data-test="bottom-caption" id="figure-2-desc"><p><b>a</b>, Experimental setup of the WL study. <b>b</b>,<b>c</b>, 18-week-old (<b>b</b>) or 31-week-old (<b>c</b>) diet-induced obesity or age-matched control male mice were fed chow diet for 8 weeks. Body weight (<i>n</i> = 20 each; data from two experiments). <b>d</b>, UMAP of 48,046 nuclei representing integrated epiAT pools (<i>n</i> = 5 pooled mice each) from C, CC, CCC, H, HC, HH and HHC mice. <b>e</b>, Relative abundance of cell types/clusters per condition. <b>f</b>, Relative abundance of macrophage subclusters per condition as percentage of total macrophages. <b>g</b>,<b>h</b>, Proportion of retained upregulated (<b>g</b>) or downregulated (<b>h</b>) transcriptional changes in different cell types. <b>i</b>, Normalized expression of selected DEGs in adipocytes across all conditions that did not restore expression profile (left), restored only in HC adipocytes (middle) or restored expression profile (right) (Wilcoxon rank-sum test, adjusted <i>P</i> &lt; 0.05 by the Bonferroni correction method; FC &gt; ±0.5). Significance for <b>b</b> and <b>c</b> was calculated using unpaired, multiple <i>t</i>-tests with Benjamini, Krieger and Yekutieli post-hoc test for multiple comparisons. ***FDR &lt; 0.001, ****FDR &lt; 0.0001. Exact <i>P</i> values are in the <a data-track="click" data-track-label="link" data-track-action="section anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Sec52">Source Data</a>. EpiCs, epithelial cells; FDR, false discovery rate; FIPs, fibro-inflammatory progenitors; NPVMs, non-perivascular macrophages; PVMs, perivascular macrophages; P-LAMs, proliferating LAMs; W, week.</p><p><a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#MOESM3">Source Data</a></p></div></div></figure></div></div></div></section><section data-title="Transcriptional obesogenic memory in mice"><div id="Sec4-section"><h2 id="Sec4">Transcriptional obesogenic memory in mice</h2><div id="Sec4-content"><p>Considering our observations of persistent transcriptional changes in human AT, we examined mouse epiAT cellular changes throughout obesity and WL using snRNA-seq. We annotated 15 key cell populations using common marker genes<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 34" title="Emont, M. P. et al. A single-cell atlas of human and mouse white adipose tissue. Nature 603, 926–933 (2022)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR34" id="ref-link-section-d15230973e1290">34</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 36" title="Maniyadath, B., Zhang, Q., Gupta, R. K. &amp; Mandrup, S. Adipose tissue at single-cell resolution. Cell Metab. 35, 386–413 (2023)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR36" id="ref-link-section-d15230973e1293">36</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 46" title="Sárvári, A. K. et al. Plasticity of epididymal adipose tissue in response to diet-induced obesity at single-nucleus resolution. Cell Metab. 33, 437–453.e5 (2021)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR46" id="ref-link-section-d15230973e1296">46</a></sup>, including APCs, immune cells, adipocytes, mesothelial cells, endothelial cells and epithelial cells (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig2">2d</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig11">6a,b</a>). Consistent with previous findings<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 16" title="Schmitz, J. et al. Obesogenic memory can confer long-term increases in adipose tissue but not liver inflammation and insulin resistance after weight loss. Mol. Metab. 5, 328–339 (2016)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR16" id="ref-link-section-d15230973e1306">16</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 18" title="Cottam, M. A., Caslin, H. L., Winn, N. C. &amp; Hasty, A. H. Multiomics reveals persistence of obesity-associated immune cell phenotypes in adipose tissue during weight loss and weight regain in mice. Nat. Commun. 13, 2950 (2022)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR18" id="ref-link-section-d15230973e1309">18</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 46" title="Sárvári, A. K. et al. Plasticity of epididymal adipose tissue in response to diet-induced obesity at single-nucleus resolution. Cell Metab. 33, 437–453.e5 (2021)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR46" id="ref-link-section-d15230973e1312">46</a></sup>, macrophage cell number in epiAT was higher in obese conditions (H and HH), and was not fully normalized after WL, especially in HHC mice (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig2">2e</a>). Resident macrophages in control mice (C, CC and CCC) primarily consisted of perivascular macrophages and non-perivascular macrophages. Notably, during obesity mainly lipid-associated macrophage (LAM) and non-perivascular macrophage cell numbers increased in the epiAT, altering the macrophage population composition persistently (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig2">2f</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig11">6c,d</a>).</p><p>Motivated by our own observation of persistent transcriptional changes in human AT (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig1">1</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig6">1</a>) and corresponding recent reports in endothelial and immune cells<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 18" title="Cottam, M. A., Caslin, H. L., Winn, N. C. &amp; Hasty, A. H. Multiomics reveals persistence of obesity-associated immune cell phenotypes in adipose tissue during weight loss and weight regain in mice. Nat. Commun. 13, 2950 (2022)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR18" id="ref-link-section-d15230973e1335">18</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 19" title="Bondareva, O. et al. Single-cell profiling of vascular endothelial cells reveals progressive organ-specific vulnerabilities during obesity. Nat. Metab. 4, 1591–1610 (2022)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR19" id="ref-link-section-d15230973e1338">19</a></sup>, we next investigated transcriptional retention (‘memory’) in the mouse epiAT. On the basis of the number of DEGs in each cell type, we found stronger transcriptional deregulation in obesity and after WL in adipocytes, APCs, endothelial cells, epithelial cells and macrophages than in other cell types (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig12">7a</a>), corroborating the existence of persistent, cell-specific transcriptional changes in mouse epiAT. Indeed, across cell types many DEGs from the obesity time point remained deregulated after WL (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig2">2g,h</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig12">7b,c</a>). GSEA of retained DEGs in adipocytes, APCs, endothelial cells, LAMs, non-perivascular macrophages, perivascular macrophages and mesothelial cells showed persistent upregulation in HC and HHC mice of genes related to lysosome activity, apoptosis and other inflammatory pathways (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig12">7c,d</a>), indicating endoplasmic reticulum and cellular stress. Persistently downregulated retained DEGs in HC and HHC mice were mainly related to metabolic AT pathways, such as fatty acid omega oxidation, fatty acid biosynthesis, adipogenesis or peroxisome proliferator-activated receptor signalling (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig12">7e,f</a>), pointing to potential dysfunction in the AT after WL.</p><p>Focusing specifically on adipocytes, we identified three distinct patterns of DEGs (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig2">2i</a>): a group that failed to restore normal expression after WL in HC or HHC (for example, <i>Maob</i> or <i>Ctsd</i>); another group that restored expression in HC but not in HHC (for example, <i>Cyp2e1</i> or <i>Runx2</i>); and a third group that restored normal expression after WL in both HC and HHC mice (for example, <i>Gpam</i> or <i>Tyrobp</i>). Notably, we did not identify any DEGs that exclusively restored normal expression after WL in HHC mice but not in HC mice, suggesting that longer durations of obesity or relatively shorter WL periods exert a stronger influence on retainment of a transcriptional memory. In summary, after WL, adipocytes from mice maintained an upregulation of inflammatory- and extracellular matrix remodelling-related pathways, whereas adipocyte-specific metabolic pathways remained downregulated (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig12">7g,h</a>), mirroring our findings from human adipocytes (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig1">1h,j</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig9">4</a>).</p></div></div></section><section data-title="Epigenetic obesogenic memory in mice"><div id="Sec5-section"><h2 id="Sec5">Epigenetic obesogenic memory in mice</h2><div id="Sec5-content"><p>Having established the persistence of obesity-associated transcriptional changes after WL in human AT and mouse epiAT, our attention shifted towards exploring the underlying mechanisms conferring this putative memory. We decided to focus on adipocytes given their post-mitotic nature, immobility, long lifespan and central position in AT biology<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 47" title="Rosen, E. D. &amp; Spiegelman, B. M. What we talk about when we talk about fat. Cell 156, 20–44 (2014)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR47" id="ref-link-section-d15230973e1400">47</a></sup>. We conducted an epigenetic analysis of adipocytes derived from mouse epiAT. Considering the inherent difficulties in studying epigenetic signatures in heterogenous cell populations, we crossed tamoxifen-inducible AdipoERCre mice with NuTRAP reporter mice, and thereby labelled adipocyte nuclei with biotin and GFP-tagged ribosomes before HFD feeding (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig3">3a</a>). We then developed a protocol to assay multiple modalities from labelled adipocytes of the same epiAT depot (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig3">3b</a>) and performed paired analysis of the translatome using targeted purification of polysomal messenger RNA (translating ribosome affinity purification followed by RNA sequencing technology (TRAP–seq)), chromatin accessibility using assay for transposase-accessible chromatin (ATAC) with sequencing (ATAC–seq) and four histone post-translational modifications (hPTMs) using cleavage under targets and tagmentation (CUT&amp;Tag). In essence, we generated extensive epigenetic datasets from adipocytes of each epiAT sample (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig13">8a,b</a>) encompassing H3K27me3 (a polycomb-mediated repressive hPTM), H3K4me3 (which marks active transcription start sites (TSS)), H3K4me1 (indicative of active or poised enhancers) and H3K27ac (which marks active enhancers and other candidate <i>cis</i>-regulatory elements)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 48" title="Bannister, A. J. &amp; Kouzarides, T. Regulation of chromatin by histone modifications. Cell Res. 21, 381–395 (2011)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR48" id="ref-link-section-d15230973e1417">48</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 49" title="Lawrence, M., Daujat, S. &amp; Schneider, R. Lateral thinking: how histone modifications regulate gene expression. Trends Genet. 32, 42–56 (2016)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR49" id="ref-link-section-d15230973e1420">49</a></sup>. We observed strong correlation between the transcriptional profiles of labelled adipocytes and the adipocyte clusters identified by snRNA-seq (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig13">8c</a>). Consistent with our observation from the snRNA-seq, we also noted a restoration of the translational profile in adipocytes from HC and HHC mice (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig3">3c</a>).</p><div data-test="figure" data-container-section="figure" id="figure-3" data-title="Adipocyte promoters retain an epigenetic memory."><figure><figcaption><b id="Fig3" data-test="figure-caption-text">Fig. 3: Adipocyte promoters retain an epigenetic memory.</b></figcaption><div><div><a data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="https://g-w1.github.io/articles/s41586-024-08165-7/figures/3" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41586-024-08165-7/MediaObjects/41586_2024_8165_Fig3_HTML.png?as=webp"/><img aria-describedby="Fig3" src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41586-024-08165-7/MediaObjects/41586_2024_8165_Fig3_HTML.png" alt="figure 3" loading="lazy" width="685" height="647"/></picture></a></div><div data-test="bottom-caption" id="figure-3-desc"><p><b>a</b>, Experimental setup of the WL study in AdipoERCre x NuTRAP mice. <b>b</b>, Workflow of paired CUT&amp;Tag, ATAC–seq and TRAP–seq from one AT depot. Biotinylated nuclei and GFP-tagged ribosomes are isolated from frozen tissue, pulled down and subjected to CUT&amp;Tag, ATAC–seq (nuclei) and TRAP–seq (ribosomes). <b>c</b>, PCA of translatome (TRAP–seq) of labelled adipocytes from C, CC_s, CC_l, CCC, H, HH, HC and HHC. Each dot represents an individual biological replicate. <b>d</b>, MOFA plots showing the sample clustering along latent Factors 1 and 2 (left) and Factor 1 value distribution (right) across labelled adipocytes. Each dot corresponds to one biological replicate. For each replicate all six modalities are represented in one dot. <b>e</b>, Percentage of variance explained by each MOFA factor across one of six modalities. <b>f</b>, Dynamics of differentially H3K4me3-marked promoters (<i>y</i> axis) from H to HC. <b>g</b>, Dynamics of differentially H3K27me3-marked promoters (<i>y</i> axis) from H to HC. <b>h</b>, Scaled enrichment of H3K4me3 (left), H3K27me3 (middle) and H3K27ac (right) at selected promoters of genes and the log<sub>2</sub>FC of TRAP–seq from comparisons against controls for the same genes. <b>i</b>, Distribution of normalized reads of H3K4me3 and H3K27me3 at the <i>Cyp2e1</i> and <i>Icam1</i> loci across conditions. Scaling of reads was performed per hPTM. NS, not significant; v, versus.</p><p><a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#MOESM3">Source Data</a></p></div></div></figure></div><p>Next, to identify sources of biological variability (factors) in our datasets on the basis of all modalities across all conditions we used multi-omics factor analysis (MOFA)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 50" title="Argelaguet, R. et al. Multi-omics factor analysis—a framework for unsupervised integration of multi-omics data sets. Mol. Syst. Biol. 14, e8124 (2018)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR50" id="ref-link-section-d15230973e1499">50</a></sup>. This enables unsupervised integration and clustering of our paired multi-omic (epigenetic) datasets to overcome potential limitations of modality-specific analyses. HC and HHC samples clustered closer to H and HH samples than to controls along Factor 1, indicating that WL did not induce complete normalization of the adipocyte epigenome (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig3">3d</a>). MOFA inferred Factor 1 as the main source of data variability between the conditions, which was predominantly influenced by active hPTMs (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig3">3e</a>).</p><p>Motivated by our MOFA findings, we investigated promoters marked by H3K4me3 or H3K27me3 to identify differentially marked promoters for these hPTMs (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig13">8d</a>). We examined the dynamics of these modifications between adipocytes from obese and WL mice. More than 1,000 promoters showed differential enrichment of H3K4me3 in H and HC mice (H: 1,475; HC: 1,094), with a majority showing increased H3K4me3 levels (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig3">3f</a>). Similarly, 859 promoters were differentially marked in HH and HHC mice (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig13">8e</a>). Overall, many promoters remained activated after WL that were less actively marked in controls, and vice versa. In contrast to H3K4me3, overall, more promoters lost than gained H3K27me3 in obese mice, and a substantial number of these promoters remained repressed or did not regain trimethylation at K27 after WL compared with controls (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig3">3g</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig13">8e</a>).</p><p>We next performed a functional analysis of differentially marked promoters. The activity status of many promoters switched, transitioning from active (H3K4me3 and/or H3K27ac) to repressed (H3K27me3), or vice versa, in obese and WL conditions, compared with control samples. Many of these epigenetic changes were also reflected in the translatome (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig3">3h</a>) and nuclear transcriptome (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig2">2f</a>). Promoters that remained repressed (high H3K27me3 and low H3K4me3 and/or H3K27ac) were linked to adipocyte function-related genes (for example, <i>Gpam</i>, <i>Cyp2e1</i> or <i>Acacb</i>), whereas promoters that remained active (that is, high H3K4me3 and/or H3K27ac and low H3K27me3) were related to genes involved in extracellular matrix remodelling and inflammatory signalling (for example, <i>Icam1</i>, <i>Lyz2</i> or <i>Tyrobp</i>) (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig3">3h,i</a>). By GSEA, we confirmed that H3K4me3 persistence in adipocytes from H/HC and HH/HHC mice was associated with chemokine and inflammatory processes (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig13">8f,g</a>). Persistent H3K4me3 loss in H/HC-affected genes included those involved in adipocyte functions (for example, adipogenesis, triacylglyceride synthesis, peroxisome proliferator-activated receptor signalling, leptin and adiponectin signalling) (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig13">8f</a>), whereas adipogenesis-related genes were repressed by H3K27me3 gain and H3K4me3 loss in adipocytes from HHC/HH mice (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig13">8g</a>), suggesting a persistently impaired adipocyte function. Notably, the expression of relevant epigenetic modifiers was not deregulated in HC or HHC adipocytes (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig13">8h</a>).</p><p>Enhancers are key drivers of cellular identity and cell fate<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 51" title="Choi, J. et al. Evidence for additive and synergistic action of mammalian enhancers during cell fate determination. eLife 10, e65381 (2021)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR51" id="ref-link-section-d15230973e1575">51</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 52" title="Hnisz, D. et al. Super-enhancers in the control of cell identity and disease. Cell 155, 934–947 (2013)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR52" id="ref-link-section-d15230973e1578">52</a></sup>. MOFA (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig3">3d</a>) indicated that active (H3K27ac) and enhancer (H3K4me1) hPTMs, together with chromatin accessibility (ATAC), were the modalities mostly explaining data variability across all conditions. An analysis of the correlation coefficients of hPTM signatures in each condition against an aggregated control (composed of averaged healthy young controls) revealed large deviations between H3K27ac and H3K4me1 in obese or WL conditions from control mice (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig4">4a</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig14">9a</a>). We generated adipocyte-specific enhancer annotations for each condition on the basis of our data (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig14">9b–e</a>) and analysed enhancer dynamics in obese and WL mice. Next, we performed differential enrichment analysis of H3K4me1, H3K27ac and ATAC–seq in enhancers. By principal component analysis (PCA), we found that HC and HHC samples clustered closer to H and HH than to controls for H3K4me1, ATAC and H3K27ac (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig4">4b</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig14">9f–h</a>). H3K4me1 separated H/HH and HC/HHC from controls, indicating that not only active but also poised enhancers could drive persistent epigenetic alterations. We then analysed the dynamic behaviour of enhancers between the obese and WL adipocytes. Several thousand enhancers were differentially marked by H3K4me1 during obesity (H: 4,255, HH: 3,237) and/or after WL (HC: 3,439, HHC: 6,589), and remained altered from H to HC (<i>n</i> = 848) and from HH to HHC (<i>n</i> = 857) (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig4">4c</a>).</p><div data-test="figure" data-container-section="figure" id="figure-4" data-title="Adipocyte enhancers retain an epigenetic memory."><figure><figcaption><b id="Fig4" data-test="figure-caption-text">Fig. 4: Adipocyte enhancers retain an epigenetic memory.</b></figcaption><div><div><a data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="https://g-w1.github.io/articles/s41586-024-08165-7/figures/4" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41586-024-08165-7/MediaObjects/41586_2024_8165_Fig4_HTML.png?as=webp"/><img aria-describedby="Fig4" src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41586-024-08165-7/MediaObjects/41586_2024_8165_Fig4_HTML.png" alt="figure 4" loading="lazy" width="685" height="405"/></picture></a></div><div data-test="bottom-caption" id="figure-4-desc"><p><b>a</b>, Correlation coefficient <i>R</i> (Pearson) of quantified peaks of H3K4me1 and H3K4me3 against a hypothetical healthy control (<i>n</i> = 2–3 each) with s.d. Each dot represents an individual biological replicate. <b>b</b>, PCA plots of quantified adipocyte-specific enhancers as marked by H3K4me1. Each dot represents an individual biological replicate. <b>c</b>, Dynamics of differentially H3K4me1-marked enhancers (<i>y</i> axis) from H to HC (left) and HH to HHC (right). <b>d</b>, H3K27ac status of genes linked to newly emerged enhancers marked by H3K4me1 (from <b>c</b>) in different conditions identified by the presence of an H3K27ac peak associated to the gene. <i>n</i> = 218 left and <i>n</i> = 127 right. <b>e</b>, Top (significant) pathway terms for genes linked to newly emerged acetylated enhancers for H and HC (left) and HH and HHC (right) on the basis of WikiPathways database (Fisher’s exact test, adjusted <i>P</i> &lt; 0.05 by the Benjamini–Hochberg method for correction). <b>f</b>, Proportion of down- and upregulated memory DEGs from TRAP–seq that can be explained by one or more epigenetic modality in HC (<i>n</i> = 13; <i>n</i> = 72) and HHC (<i>n</i> = 7; <i>n</i> = 36).</p><p><a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#MOESM3">Source Data</a></p></div></div></figure></div><p>We termed enhancers that gained (and maintained) H3K4me1 in obesity and WL ‘new enhancers’. Most of these ‘new enhancers’ were also active (that is, marked by H3K27ac) during obesity and/or WL (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig4">4d</a>). We then annotated the enhancers to their closest gene and performed a GSEA. In agreement with the promoter GSEA above, we found that the ‘new active enhancers’ were related to inflammatory signalling, lysosome activity and extracellular matrix remodelling (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig4">4e</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig14">9i</a>), indicating a persistent shift of adipocytes towards a more inflammatory and less adipogenic identity. Corroborating these results, Roh et al. had analysed H3K27ac in adipocytes of obese mice and reported impaired identity maintenance during obesity<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 25" title="Roh, H. C. et al. Adipocytes fail to maintain cellular identity during obesity due to reduced PPARγ activity and elevated TGFβ-SMAD signaling. Mol. Metab. 42, 101086 (2020)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR25" id="ref-link-section-d15230973e1698">25</a></sup>.</p><p>To combine our findings regarding retained translational changes and epigenetic memory, we investigated whether epigenetic mechanisms, such as differentially marked promoters or enhancers, could explain the persistent translational obesity-associated changes after WL. Notably, 57–62% of downregulated and 68–75% of upregulated persistent translational DEGs after WL could be accounted for by one or more of the analysed epigenetic modalities (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig4">4f</a>). Overall, these results strongly suggest the presence of stable cellular, epigenetic and transcriptional memory in mouse adipocytes that persists after WL.</p></div></div></section><section data-title="Metabolic memory primes adipocytes"><div id="Sec6-section"><h2 id="Sec6">Metabolic memory primes adipocytes</h2><div id="Sec6-content"><p>We then asked whether this persistent memory primed mature adipocytes to respond differently to nutritional stimuli than non-primed controls. We collected mature epiAT and ingAT adipocytes from WL and control mice, cultured them for 48 h and then assessed glucose and palmitate uptake. Adipocytes from WL epiAT showed increased glucose and palmitate uptake compared with controls (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig5">5a,b</a>). ingAT adipocytes from HHC mice displayed significantly increased glucose uptake compared with controls, and for HC adipocytes we observed a trend towards an increased uptake (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig15">10a</a>). Assessing adipogenesis capacity, we found that the SVF from epiAT of HC and HHC mice accumulated lipids in response to insulin but failed to differentiate, unlike controls (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig15">10b</a>). Adipogenesis was slightly impaired in the SVF from ingAT of WL mice compared with controls (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig15">10c</a>). These findings indicate that persistent cellular memory confers phenotypic consequence ex vivo.</p><div data-test="figure" data-container-section="figure" id="figure-5" data-title="Memory primes adipocytes and mice for an accelerated response to obesogenic stimuli."><figure><figcaption><b id="Fig5" data-test="figure-caption-text">Fig. 5: Memory primes adipocytes and mice for an accelerated response to obesogenic stimuli.</b></figcaption><div><div><a data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="https://g-w1.github.io/articles/s41586-024-08165-7/figures/5" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41586-024-08165-7/MediaObjects/41586_2024_8165_Fig5_HTML.png?as=webp"/><img aria-describedby="Fig5" src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41586-024-08165-7/MediaObjects/41586_2024_8165_Fig5_HTML.png" alt="figure 5" loading="lazy" width="685" height="526"/></picture></a></div><div data-test="bottom-caption" id="figure-5-desc"><p><b>a</b>,<b>b</b>, Experiments with isolated, cultured primary epiAT adipocytes. Each dot represents an individual biological replicate of a pool of three mice. <b>a</b>, Glucose uptake. <b>b</b>, Palmitate uptake. <b>c</b>, HC and CC_s mice were put on HFD for 4 weeks. Body weight (<i>n</i> = 12 each). <b>d</b>–<b>g</b>, Experiments with HHC and CCH mice. Each dot indicates an individual biological replicate from two experiments. <b>d</b>, Fasting blood glucose (<i>n</i> = 10). <b>e</b>,<b>f</b>, Fed insulin (<b>e</b>) and leptin (<b>f</b>) (C, H, CC_s, HC: <i>n</i> = 6 each; CCH: <i>n</i> = 8; HCH: <i>n</i> = 5). <b>g</b>, Weights of ingAT, epiAT and BAT, normalized to body weight (<i>n</i> = 10). <b>h</b>, Representative images of epiAT. The ruler is in cm. <b>i</b>, Relative cell type abundance. <b>j</b>, Proportion of up- and downregulated DEGs in HCH adipocytes that can be explained by DEG status at HC time point or transcriptional memory. <b>k</b>, Normalized expression of selected DEGs in HCH adipocytes that were recovered in HC but were still differentially marked by one or more epigenetic modalities (Wilcoxon rank-sum test, adjusted <i>P</i> &lt; 0.05 by the Bonferroni correction method; FC &gt; ±0.5). <b>l</b>, Distribution of normalized reads of H3K4me3, H3K27me3, H3K27ac and H3K4me1 of HC and CC_s adipocytes at loci of <i>Tmsbx4</i> and <i>Gpam</i>. Scaling of reads was performed per hPTM. <b>m</b>, Proportion of up- and downregulated DEGs in HCH adipocytes that can be explained by an epigenetic memory. Significance was calculated between age-matched controls and experimental groups. Significance for <b>a</b>, <b>b</b>, <b>d</b>, <b>e</b>, <b>f</b> and <b>g</b> was calculated using two-tailed Mann–Whitney tests. Significance for <b>c</b> was calculated using unpaired, multiple <i>t</i>-tests with Benjamini, Krieger and Yekutieli post-hoc test for multiple comparisons. **FDR &lt; 0.01, ***FDR &lt; 0.001. Error bars represent s.d. Boxplots represent minimum, maximum and median.</p><p><a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#MOESM3">Source Data</a></p></div></div></figure></div><p>Next, we investigated the response of WL and control mice to 4 weeks of HFD feeding. HC mice gained weight faster than CC_s mice (called HCH and CCH, respectively, here) (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig5">5c</a>). Fasting blood glucose levels and postprandial insulin levels were elevated in HCH mice (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig5">5d,e</a>), but neither glucose tolerance nor insulin sensitivity was impaired when compared with CCH mice (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig15">10d–g</a>). Leptin levels in HCH mice returned to H mice levels, whereas CCH mice did not show a significant increase (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig5">5f</a>). Adipocytes in epiAT from HCH mice were larger on average, resembling the adipocyte size distribution of H mice, whereas epiAT adipocytes from CCH mice were similar to those in CC_s mice (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig15">10h</a>). HCH mice exhibited larger ingAT, BAT and epiAT depots compared with CCH mice (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig5">5g,h</a>) and showed increased triglyceride accumulation and hepatic steatosis (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig15">10i–k</a>).</p><p>We performed snRNA-seq of epiAT from HCH and CCH mice and observed higher macrophage infiltration in both HCH and CCH epiAT compared with the WL time point, with a greater infiltration in HCH epiAT (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig5">5i</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig15">10l</a>). The proportion of LAMs was greater in HCH epiAT, similar to that of H and HC mice, whereas CCH epiAT showed a greater proportion of LAMs compared with CC epiAT, indicating LAM infiltration occurred early during HFD feeding (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig15">10m</a>).</p><p>We assessed whether HCH and CCH adipocytes exhibited transcriptional differences. Neither the previous transcriptional status nor the transcriptional memory at the HC time point explained the transcriptional deregulation observed in adipocytes from HCH mice (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig5">5j</a>). Further analysis revealed that several DEGs in the HCH group were altered during obesity but recovered after WL in HC mice. Interestingly, these overlapped with promoters and enhancers carrying epigenetic memory (Figs. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig2">2f</a>, <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig3">3h</a> and <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig5">5k,l</a>). A more detailed analysis showed that epigenetic signatures could explain the 3–6 times more DEGs in the HCH group than the transcriptional memory or previous transcriptional status during HC (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig5">5m</a>). Specifically, the four hPTMs and ATAC–seq could predict or explain 31% of upregulated DEGs, which were related to inflammation, and 60% of downregulated DEGs, many of which were related to adipocyte function and identity, in the HCH group (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig15">10n,o</a>).</p><p>Together, these findings suggest that a persistent epigenetic memory, including local changes of hPTM deposition, contributes to the altered transcriptional response in adipocytes in the ‘yo-yo’ model of dieting and primes adipocytes for pathological responses to further HFD feeding, thus contributing to the pathophysiology of rebound obesity in mice. It is possible that other epigenetic modifications, such as other hPTMs, DNA methylation or non-coding RNAs, also contribute to the observed phenomena.</p><p>Although we performed well-controlled dietary intervention experiments in mice, the human AT samples were obtained from different BaS studies and AT depots, and reflect an overall heterogenous group of participants. Indeed, BaS is a successful but invasive method for achieving long-term WL<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 53" title="Tolvanen, L., Christenson, A., Surkan, P. J. &amp; Lagerros, Y. T. Patients’ experiences of weight regain after bariatric surgery. Obes. Surg. 32, 1498–1507 (2022)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR53" id="ref-link-section-d15230973e1931">53</a></sup>, yet sleeve gastrectomy and Roux-en-Y gastric bypass (RYGB) also affect the gut microbiome, micronutrient absorption, bile acid metabolism and incretin signalling<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Wang, W. et al. Role of bile acids in bariatric surgery. Front. Physiol. 10, 374 (2019)." href="#ref-CR54" id="ref-link-section-d15230973e1935">54</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Ramos-Molina, B. et al. Differential effects of restrictive and malabsorptive bariatric surgery procedures on the serum lipidome in obese subjects. J. Clin. Lipidol. 12, 1502–1512 (2018)." href="#ref-CR55" id="ref-link-section-d15230973e1935_1">55</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Inge, T. H. et al. Weight loss and health status 3 years after bariatric surgery in adolescents. N. Engl. J. Med. 374, 113–123 (2016)." href="#ref-CR56" id="ref-link-section-d15230973e1935_2">56</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 57" title="Ulker, İ. &amp; Yildiran, H. The effects of bariatric surgery on gut microbiota in patients with obesity: a review of the literature. Biosci. Microbiota Food Health 38, 3–9 (2019)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR57" id="ref-link-section-d15230973e1938">57</a></sup>. Nonetheless, we consistently observed retained transcriptional differences after significant WL in AT cells after sleeve gastrectomy (MTSS and LTSS studies), which induced significant WL, as well as after RYGB (NEFA study), which resulted in a complete return to a non-obese or lean state. The aforementioned alterations and the degree of WL achieved between individuals and studies are confounders that limit the direct comparability of our mouse and human data. The rapid WL achieved by BaS may even reduce or modify putative cellular memory in the human AT. Owing to the current lack of methods to isolate pure adipocyte nuclei from frozen human tissue, we could not perform the corresponding epigenetic analyses in human samples. Nonetheless, it stands to reason that obesity-induced transcriptional (and cellular) changes in humans are also mediated through epigenetic mechanisms that can persist after WL in the AT and contribute to human (patho)physiology.</p><p>Although our results do not provide final proof of a causal relationship between AT memory and the systemic yo-yo effect of accelerated weight gain, Hata et al. have shown that transplantation of WL epiAT into control mice enhances macular degeneration by impacting immune cells and angiogenesis and that epiAT macrophages retain an altered chromatin accessibility after WL<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 28" title="Hata, M. et al. Past history of obesity triggers persistent epigenetic changes in innate immunity and exacerbates neuroinflammation. Science 379, 45–62 (2023)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR28" id="ref-link-section-d15230973e1945">28</a></sup>. Our epigenetic analysis of adipocytes of the same tissue could serve as an explanation of how these alterations in chromatin accessibility can be retained. Further investigations are required to determine whether—in addition to adipocytes and macrophages<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 28" title="Hata, M. et al. Past history of obesity triggers persistent epigenetic changes in innate immunity and exacerbates neuroinflammation. Science 379, 45–62 (2023)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR28" id="ref-link-section-d15230973e1949">28</a></sup>—other post-mitotic or cycling cells, such as myofibers, neurons or APCs, also establish an epigenetic memory of obesity and contribute to the observed systemic weight regain effect.</p><p>Although our results are on the basis of BaS studies, the susceptibility to weight regain in human subjects undergoing WL using strict dietary regimens might be related to a transcriptional and/or epigenetic memory as well. At present, the use of incretin receptor agonists such as semaglutide or tirzepatide<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 6" title="Wilding, J. P. H. et al. Weight regain and cardiometabolic effects after withdrawal of semaglutide: the STEP 1 trial extension. Diabetes Obes. Metab. 24, 1553–1564 (2022)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR6" id="ref-link-section-d15230973e1956">6</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 58" title="Holst, J. J. GLP-1 physiology in obesity and development of incretin-based drugs for chronic weight management. Nat. Metab. 
                  https://doi.org/10.1038/s42255-024-01113-9
                  
                 (2024)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR58" id="ref-link-section-d15230973e1959">58</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 59" title="Aronne, L. J. et al. Continued treatment with tirzepatide for maintenance of weight reduction in adults with obesity: the SURMOUNT-4 randomized clinical trial. JAMA 331, 38–48 (2024)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR59" id="ref-link-section-d15230973e1962">59</a></sup> has emerged as a promising non-invasive strategy for significant WL. However, the extent to which these agonists induce long-lasting WL and physiological changes in humans beyond withdrawal remains poorly studied. Studies on semaglutide and on tirzepatide have shown that substantial weight regain occurs after their withdrawal<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 6" title="Wilding, J. P. H. et al. Weight regain and cardiometabolic effects after withdrawal of semaglutide: the STEP 1 trial extension. Diabetes Obes. Metab. 24, 1553–1564 (2022)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR6" id="ref-link-section-d15230973e1966">6</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 59" title="Aronne, L. J. et al. Continued treatment with tirzepatide for maintenance of weight reduction in adults with obesity: the SURMOUNT-4 randomized clinical trial. JAMA 331, 38–48 (2024)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR59" id="ref-link-section-d15230973e1969">59</a></sup>, indicating that at least these treatments do not induce stable, persistent changes. Whether this is also the case for other agonists remains to be investigated. Further studies are needed to elucidate whether these treatments could erase or diminish an obesogenic memory better than other non-surgery-based WL strategies.</p><p>The presence of a putative obesogenic epigenetic memory in adipocytes and potentially other cells suggests new potential therapeutic avenues to improve WL maintenance in humans. Although our experiments focused on obesity, it is plausible that epigenetic memory could also play a role in many other contexts, including addictive diseases. Recent advancements in targeted epigenetic editing<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 60" title="Cappelluti, M. A. et al. Durable and efficient gene silencing in vivo by hit-and-run epigenome editing. Nature 627, 416–423 (2024)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR60" id="ref-link-section-d15230973e1976">60</a></sup> and global remodelling of the epigenome<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 61" title="Browder, K. C. et al. In vivo partial reprogramming alters age-associated molecular changes during physiological aging in mice. Nat. Aging 2, 243–253 (2022)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR61" id="ref-link-section-d15230973e1980">61</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 62" title="Ocampo, A. et al. In vivo amelioration of age-associated hallmarks by partial reprogramming. Cell 167, 1719–1733.e12 (2016)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR62" id="ref-link-section-d15230973e1983">62</a></sup> provide promising new approaches.</p></div></div></section><section data-title="Methods"><div id="Sec7-section"><h2 id="Sec7">Methods</h2><div id="Sec7-content"><h3 id="Sec8">Data reporting</h3><p>No statistical methods were used to predetermine sample size. The experiments were not randomized, and the investigators were not blinded to allocation during experiments and outcome assessment.</p><h3 id="Sec9">Clinical sample acquisition</h3><p>Human AT biopsies were obtained from three independent studies: MTSS, LTSS and NEFA.</p><h4 id="Sec10">MTSS</h4><p>The MTSS samples comprised samples from omental visceral AT biopsies obtained in the context of a two-step BaS treatment, which included a sleeve gastrectomy as the first step (T0) and laparoscopic RYGB as the second step (T1)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 16" title="Schmitz, J. et al. Obesogenic memory can confer long-term increases in adipose tissue but not liver inflammation and insulin resistance after weight loss. Mol. Metab. 5, 328–339 (2016)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR16" id="ref-link-section-d15230973e2014">16</a></sup>. Individuals with syndromal, monogenic, early-onset obesity or individuals with other known concurrent diseases, including acute infections or malignant diseases, were not included in the study. Individuals were not required to adhere to any specific diet before or after surgery but received individual dietary recommendations during regular visits in the obesity management centre. Insulin resistance was determined using a hyperinsulinaemic–euglycaemic clamp technique or the homeostatic model assessment for insulin resistance (HOMA-IR). Only biopsies from individuals that (1) lost 25% or more of BMI between T0 and T1 (Extended Data Table <a data-track="click" data-track-label="link" data-track-action="table anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Tab1">1</a>), (2) had undergone surgery at the Municipal Hospital Karlsruhe or Municipal Hospital Dresden-Neustadt, (3) were not diagnosed with diabetes, and (4) did not receive any glucose-lowering medication were used for snRNA-seq in this study. AT samples were collected during elective laparoscopic abdominal surgery as previously described<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 63" title="Langhardt, J. et al. Effects of weight loss on glutathione peroxidase 3 serum concentrations and adipose tissue expression in human obesity. Obes. Facts 11, 475–490 (2018)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR63" id="ref-link-section-d15230973e2021">63</a></sup>, snap-frozen in liquid nitrogen and stored at −80 °C. Body composition and metabolic parameters were measured as previously described<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 64" title="Klöting, N. et al. Insulin-sensitive obesity. Am. J. Physiol. Endocrinol. Metab. 299, E506–E515 (2010)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR64" id="ref-link-section-d15230973e2025">64</a></sup>. Samples of healthy individuals who were not obese were collected during routine elective surgeries such as herniotomies, explorative laparoscopies and cholecystectomies at the same hospitals. The study was approved by the Ethics Committee of the University of Leipzig under approval number 159-12–21052012 and was performed in agreement with the Declaration of Helsinki.</p><h4 id="Sec11">LTSS</h4><p>The human study samples comprised samples from omental visceral and subcutaneous abdominal AT, collected in the context of a two-step BaS treatment. Following an initial sleeve gastrectomy (T0), a laparoscopic RYGB was made in the second step (T1)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 16" title="Schmitz, J. et al. Obesogenic memory can confer long-term increases in adipose tissue but not liver inflammation and insulin resistance after weight loss. Mol. Metab. 5, 328–339 (2016)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR16" id="ref-link-section-d15230973e2037">16</a></sup>. Individuals with syndromal, early-onset obesity or individuals with other known concurrent diseases, including acute infections or malignant diseases, were not included in the study. Individuals did not adhere to any specific diet before or after surgery but received individual healthy diet recommendations during regular visits in the obesity management centre. Insulin resistance was determined using HOMA-IR. Only individuals that (1) lost 25% or more of BMI between T0 and T1 (Extended Data Table <a data-track="click" data-track-label="link" data-track-action="table anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Tab1">1</a>), (2) had undergone surgery at the Leipzig University Hospital, (3) were not diagnosed with diabetes and (4) did not receive any glucose-lowering medication were included. AT samples were collected during elective laparoscopic abdominal surgery as previously described<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 63" title="Langhardt, J. et al. Effects of weight loss on glutathione peroxidase 3 serum concentrations and adipose tissue expression in human obesity. Obes. Facts 11, 475–490 (2018)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR63" id="ref-link-section-d15230973e2044">63</a></sup>, snap-frozen in liquid nitrogen and stored at −80 °C. Body composition and metabolic parameters were measured as previously described<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 64" title="Klöting, N. et al. Insulin-sensitive obesity. Am. J. Physiol. Endocrinol. Metab. 299, E506–E515 (2010)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR64" id="ref-link-section-d15230973e2048">64</a></sup>. Samples from healthy donors that were not obese were collected during routine elective surgeries (herniotomies, explorative laparoscopies, cholecystectomies) at the same hospital. The study was approved by the Ethics Committee of the University of Leipzig under approval number 159-12–21052012 and performed in agreement with the Declaration of Helsinki.</p><h4 id="Sec12">NEFA study</h4><p>The NEFA study (<a href="https://clinicaltrials.gov/ct2/show/NCT01727245">NCT01727245</a>) comprises samples from subcutaneous abdominal AT from individuals before and after RYGB surgery, as well as healthy controls who had never been obese<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 8" title="Mileti, E. et al. Human white adipose tissue displays selective insulin resistance in the obese state. Diabetes 70, 1486–1497 (2021)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR8" id="ref-link-section-d15230973e2067">8</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 65" title="Rydén, M. et al. The adipose transcriptional response to insulin is determined by obesity, not insulin sensitivity. Cell Rep. 16, 2317–2326 (2016)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR65" id="ref-link-section-d15230973e2070">65</a></sup>. For this, biopsies were obtained under local anaesthesia before (T0) and 2 yr post-surgery (T1). Only samples from individuals that (1) lost more than 25% BMI between T0 and T1, (2) were not diagnosed with diabetes at T0 and T1 and (3) did not take glucose-lowering medication were included in the present study (Extended Data Table <a data-track="click" data-track-label="link" data-track-action="table anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Tab1">1</a>). Samples from control subjects were obtained from individuals that were BMI- and age-matched to RYGB patients at T1 as reported previously<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 8" title="Mileti, E. et al. Human white adipose tissue displays selective insulin resistance in the obese state. Diabetes 70, 1486–1497 (2021)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR8" id="ref-link-section-d15230973e2077">8</a></sup>. AT samples were handled as reported before<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 65" title="Rydén, M. et al. The adipose transcriptional response to insulin is determined by obesity, not insulin sensitivity. Cell Rep. 16, 2317–2326 (2016)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR65" id="ref-link-section-d15230973e2081">65</a></sup>, snap-frozen in liquid nitrogen and stored at −80 °C. The study was conducted in accordance with the Declaration of Helsinki and approved by the Ethics Committee of the Karolinska Institute, Stockholm (approval number 2011/1002-31/1).</p><h3 id="Sec13">Mice</h3><p>All mice were kept on a 12-h/12-h light/dark cycle at 20–60% (23 °C) humidity in individually ventilated cages, in groups of between two and five mice, in a pathogen-free animal facility in the SLA building at ETH Zurich. The health of mice was monitored closely, and any mouse exhibiting persistent clinical signs of ill health or distress was excluded from this study. The 16- and 29-week-old male C57BL/6J diet-induced obesity mice (catalogue no. 380050) and diet-induced obesity control mice (catalogue no. 380056) were obtained from The Jackson Laboratory and were kept on the respective diets for another 2 weeks until tissue harvest or diet switch. Different mice were used for insulin tolerance tests and glucose tolerance tests. AdipoERCre<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 66" title="Moser, C. et al. Quantification of adipocyte numbers following adipose tissue remodeling. Cell Rep. 35, 109023 (2021)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR66" id="ref-link-section-d15230973e2094">66</a></sup> and NuTRAP<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 67" title="Roh, H. C. et al. Simultaneous transcriptional and epigenomic profiling from specific cell types within heterogeneous tissues in vivo. Cell Rep. 18, 1048–1061 (2017)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR67" id="ref-link-section-d15230973e2098">67</a></sup> mice were maintained on a C57BL/N background. Homozygous NuTRAP and AdipoERCre mice were bred to generate AdipoERCre x NuTRAP mice. AdipoERCre x NuTRAP mice were kept on HFD or chow diet for 12 or 25 weeks before tissue harvest or diet switch. The HFD used contained 60% (kcal%) fat (diet no. 2127, Provimi Kliba); the low-fat chow diet used contained 10% (kcal%) fat (diet no. 2125, Provimi Kliba). During the WL period both experimental groups received chow diet (diet no. 3437, Provimi Kliba). All animal experiments were approved by the Cantonal Veterinary Office, Zurich.</p><h4 id="Sec14">Tamoxifen application</h4><p>The 4–5-week-old AdipoERCre x NuTRAP mice were gavaged two times with 1 mg of tamoxifen dissolved in corn oil. Tamoxifen was washed out for 2 weeks before starting HFD.</p><h3 id="Sec15">Physiological measurements</h3><h4 id="Sec16">Glucose tolerance test</h4><p>Mice were fasted for 6 h during dark phase before administration of 1 g of glucose per kg body weight by intraperitoneal injection. Blood was collected from the tail vein at 0, 15, 30, 60, 90 and 120 min and blood glucose concentrations were measured using an Accu-Check Aviva glucometer.</p><h4 id="Sec17">Insulin tolerance test</h4><p>Mice were fasted for 6 h during dark phase before administration of 1 U per kg body weight of human insulin (insulin Actrapid HM, Novo Nordisk) by intraperitoneal injection. Blood was collected from the tail vein at 0, 15, 30, 60, 90 and 120 min and blood glucose concentrations were measured using a Accu-Check Aviva glucometer.</p><h4 id="Sec18">In vivo indirect calorimetry</h4><p>Measurements were obtained from one 8-cage and one 16-cage Promethion Core Behavioral System that were in the same room. Mice were habituated to the system for 36 h before measurements were started.</p><h4 id="Sec19">Live body composition</h4><p>Mice were fasted for 6 h during dark phase. Live mouse body composition was measured with a magnetic resonance imaging technique (EchoMRI 130, Echo Medical Systems). Fat and lean mass were analysed using EchoMRI 14 software.</p><h4 id="Sec20">Fasting insulin</h4><p>EDTA plasma was isolated from fasted blood samples (fasting 6 h). Insulin was measured with Ultra Sensitive Mouse Insulin ELISA Kit (Crystal Chem, catalogue no. 90080).</p><h4 id="Sec21">Postprandial insulin</h4><p>EDTA plasma (50 µl) was thawed on ice and used in a custom U-PLEX assay (Meso Scale Discovery) according to the manufacturer’s instructions. A Mesoscale SI 2400 was used to read the plate.</p><h4 id="Sec22">Postprandial leptin</h4><p>EDTA plasma (50 µl) was thawed on ice and used in a custom U-PLEX assay (Meso Scale Discovery) according to the manufacturer’s instructions. A Mesoscale SI 2400 was used to read the plate.</p><h4 id="Sec23">Liver triglycerides</h4><p>First, 50 mg of frozen liver was homogenized in 1 ml of isopropanol, lysed for 1 h at 4 °C and centrifuged for 10 min at 2,000<i>g</i> at 4 °C. The supernatant was transferred into a new tube and stored at −80 °C until use. Triglyceride levels were measured by mixing 200 µl of reagent R (Monlab, catalogue no. SR-41031) and 5 µl of sample or Cfas calibrator dilutions (Roche, catalogue no. 10759350; lot no. 41009301), then incubating for 10 min while shaking at room temperature and measuring optical density at 505 nm (OD<sub>505</sub>) with a plate reader (BioTek Gen5 Microplate Reader).</p><h3 id="Sec24">Cell culture experiments</h3><h4 id="Sec25">AT digestion</h4><p>AT was minced and digested at 37 °C while shaking in collagenase buffer (25 mM NaHCO<sub>3</sub>, 12 mM KH<sub>2</sub>PO<sub>4</sub>, 1.3 mM MgSO<sub>4</sub>, 4.8 mM KCl, 120 mM NaCl, 1.2 mM CaCl<sub>2</sub>, 5 mM glucose, 2.5% BSA; pH 7.4) using 2 mg of collagenase type II (Sigma-Aldrich, catalogue no. C6885-1G) per 0.25 g of tissue. After 30 min tissues were resuspended, and for ingAT digestion continued for 15 min whereas epiAT was processed immediately. An equal volume of growth medium (DMEM (Gibco, catalogue no. 31966021), 10% FBS (Gibco, catalogue no. 10500-064, Lot no. 2378399H), 1% penicillin-streptomycin (Gibco, catalogue no. 15140-122)) was added and digested tissue was centrifuged for 4 min at 300<i>g</i>, and the floating fraction was transferred into a new Falcon tube and kept at 37 °C. The SVF was resuspended in 5 ml of erythrocyte lysis buffer (154 mM NH<sub>4</sub>Cl, 10 mM NaHCO<sub>3</sub>, 0.1 mM EDTA, 1% penicillin-streptomycin), incubated at room temperature for 5 min, filtered through a 40 µM mesh filter and centrifuged for 5 min, 300<i>g</i>. The SVF was resuspended in growth medium and counted.</p><h4 id="Sec26">SVF differentiation</h4><p>A total of 10,000 cells were plated into one well of a collagen-coated (Sigma-Aldrich, catalogue no. C3867) 96-well plate and kept in culture until they reached confluency, with media change every 48 h. At 2 d post-confluence, medium was changed to induction medium (DMEM, 10% FBS, 1% penicillin-streptomycin, 10 nM insulin (Sigma-Aldrich, catalogue no. I9278), 0.5 mM 3-isobutyl-1-methylxanthin (Sigma-Aldrich, catalogue no. I7018-1G), 1 µM dexamethasone (Sigma-Aldrich, catalogue no. D4902), 1 µM rosiglitazone (Adipogen, catalogue no. AG-CR1-3570-M010)). After 48 h medium was changed to maintenance medium (DMEM, 10% FBS, 1% penicillin-streptomycin, 10 nM insulin). Medium was changed every 48 h for 8 d.</p><h4 id="Sec27">AdipoRed assay</h4><p>The SVF was cultured as described and controls were either kept in growth medium or only maintenance medium without induction. On day 8 after induction, cells were washed twice in PBS, and AdipoRed (Lonza, catalogue no. LZ-PT-7009) reagent was used according to the manufacturer’s instructions and read with a plate reader (BioTek Gen5 Microplate Reader).</p><h4 id="Sec28">Primary adipocyte culture</h4><p>Primary floating adipocytes were cultured under membranes according to Harms et al.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 68" title="Harms, M. J. et al. Mature human white adipocytes cultured under membranes maintain identity, function, and can transdifferentiate into brown-like adipocytes. Cell Rep. 27, 213–225.e5 (2019)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR68" id="ref-link-section-d15230973e2243">68</a></sup>. Packed adipocytes (30 µl) were seeded onto one membrane and kept in inverted culture for 48 h in maintenance medium (DMEM-F12 (Gibco, catalogue no. 31330095), 10% FBS, 1% penicillin-streptomycin, 10 nM insulin). After 48 h of maintenance, adipocytes were washed and serum and glucose starved overnight in KREBBS-Ringer buffer (120 mM NaCl, 4.7 mM KCl, 1.2 mM KH<sub>2</sub>PO<sub>4</sub>, 1.2 mM MgSO<sub>4</sub>, 2.5 mM CaCl<sub>2</sub>, 25 mM HEPES (Lonza, catalogue no. BEBP17-737E), pH 7.4) and 2.5% fat-free BSA (Sigma-Aldrich, catalogue no. A6003).</p><h4 id="Sec29">Glucose uptake</h4><p>Glucose uptake from primary adipocytes was measured using the Glucose Uptake-Glo Assay Kit (Promega, catalogue no. J1341) according to the manufacturer’s instructions. Adipocytes were preincubated with 5 nM insulin for 15 min before 2-deoxy-<span>d</span>-glucose was added at 1 mM final concentration. Protein concentration was measured using a Pierce 660 nm Protein Assay Kit (Thermo Fisher, catalogue no. 22662) and the Ionic Detergent Compatibility Reagent (Thermo Fisher, catalogue no. 22663). Both assays were read with a plate reader (BioTek Gen5 Microplate Reader).</p><h4 id="Sec30">C16 uptake</h4><p>Starved adipocytes were incubated with 5 nM BODIPY-palmitate (Thermo Fisher, catalogue no. D3821) in the presence of 10 nM insulin for 1 h. Subsequently, adipocytes were washed twice and lysed in 200 µl of RIPA buffer. Then, 100 µl of lysate was used to measure BODIPY signal. Diluted lysate was used to measure protein concentration using a DC Protein Assay Kit II (Bio-Rad Laboratories, catalogue no. 5000112) for normalization. Both assays were read with a plate reader (BioTek Gen5 Microplate Reader).</p><h4 id="Sec31">Histology</h4><p>Tissues were collected, fixed in 4% PBS-buffered formalin for 72 h at 4 °C and stored in PBS at 4 °C. Following paraffin embedding, tissues were sent to the pathology service centre at Instituto Murciano de Investigación Biosanitaria Virgen de la Arrixaca for sectioning, trichrome staining, haematoxylin and eosin staining, and imaging. Tissues from two independent experiments were sent for sectioning.</p><h4 id="Sec32">Adipocyte size quantification</h4><p>Images of ingAT and epiAT were taken with 3DHISTECH Slide Viewer 2 and then analysed with Adiposoft<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 69" title="Galarraga, M. et al. Adiposoft: automated software for the analysis of white adipose tissue cellularity in histological sections. J. Lipid Res. 53, 2791–2796 (2012)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR69" id="ref-link-section-d15230973e2291">69</a></sup> using Fiji ImageJ<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 70" title="Schindelin, J. et al. Fiji: an open-source platform for biological-image analysis. Nat. Methods 9, 676–682 (2012)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR70" id="ref-link-section-d15230973e2295">70</a></sup>. Five to ten images were taken of each section belonging to a biological replicate (<i>n</i> = 4).</p><h3 id="Sec33">Sample processing and library preparation</h3><h4 id="Sec34">Isolation of nuclei from mouse tissue</h4><p>Nuclei were isolated from snap-frozen epiAT in ice-cold Nuclei Extraction Buffer (Miltenyi, catalogue no. 130-128-024) supplemented with 0.2 U µl<sup>−1</sup> recombinant RNase Inhibitor (Takara, catalogue no. 2313) and 1× cOmplete EDTA-free Protease Inhibitor (Roche, catalogue no. 5056489001) using the gentleMACS Octo Dissociator (Miltenyi, catalogue no. 130-096-427), using C-tubes (Miltenyi, catalogue no. 130-093-237). Nuclei were subsequently filtered through a 50 µm cell strainer (Sysmex, catalogue no. 04-0042-2317) and washed two times in PBS-BSA (1% w/v) containing 0.2 U µl<sup>−1</sup> RNase inhibitor. For snRNA-seq, five mice were pooled per condition.</p><h4 id="Sec35">Isolation of nuclei from human tissue</h4><p>Nuclei were isolated from snap-frozen human AT (10–50 mg) in ice-cold Nuclei Extraction Buffer (Miltenyi, catalogue no. 130-128-024) supplemented with 1 U µl<sup>−1</sup> recombinant RNase Inhibitor (Takara, catalogue no. 2313), 1× cOmplete EDTA-free Protease Inhibitor (Roche, catalogue no. 5056489001) and 10 mM sodium butyrate using the gentleMACS Octo Dissociator (Miltenyi, catalogue no. 130-096-427), using C-tubes (Miltenyi, catalogue no. 130-093-237).</p><p>The nuclei suspension was filtered through a 50 µm strainer, supplemented with PBS-BSA (1% w/v) containing 1× protease inhibitor and RNase inhibitor and centrifuged at 4 °C, at 500<i>g</i> for 10 min. The nuclei pellet was resuspended in 1 ml of PBS-BSA (1%, w/v) supplemented with RNase inhibitor (0.5 U µl<sup>−1</sup>) and 1× protease inhibitor and was transferred into a new 1.5 ml tube.</p><h4 id="Sec36">snRNA-seq of AT</h4><p>Nuclei were counted using a haemocytometer and Trypan blue, concentration was adjusted to approximately 1,000 nuclei per µl and they were loaded onto a G-chip (10x Genomics, catalogue no. PN-1000127). Single-cell gene expression libraries were prepared using the Chromium Next GEM Single Cell 3′ v3.1 kit (10x Genomics) according to the manufacturer’s instructions. To accommodate for low RNA content, two cycles were added to the complementary DNA amplification PCR. Libraries were pooled equimolecularly and sequenced in PE150 (paired-end 150) mode on a NovaSeq 6000 with about 40,000 reads per nucleus at Novogene or using a NovaSeqX at the Functional Genomics Center, Zurich.</p><h4 id="Sec37">Paired TRAP–seq, CUT&amp;Tag and ATAC–seq</h4><p>Paired TRAP–seq, CUT&amp;Tag and ATAC–seq protocols were developed on the basis of published protocols<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 67" title="Roh, H. C. et al. Simultaneous transcriptional and epigenomic profiling from specific cell types within heterogeneous tissues in vivo. Cell Rep. 18, 1048–1061 (2017)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR67" id="ref-link-section-d15230973e2353">67</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Kaya-Okur, H. S. et al. CUT&amp;Tag for efficient epigenomic profiling of small samples and single cells. Nat. Commun. 10, 1–10 (2019)." href="#ref-CR71" id="ref-link-section-d15230973e2356">71</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Buenrostro, J. D., Giresi, P. G., Zaba, L. C., Chang, H. Y. &amp; Greenleaf, W. J. Transposition of native chromatin for fast and sensitive epigenomic profiling of open chromatin, DNA-binding proteins and nucleosome position. Nat. Methods 10, 1213–1218 (2013)." href="#ref-CR72" id="ref-link-section-d15230973e2356_1">72</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Heiman, M., Kulicke, R., Fenster, R. J., Greengard, P. &amp; Heintz, N. Cell-type-specific mRNA purification by translating ribosome affinity purification (TRAP). Nat. Protoc. 9, 1282–1291 (2014)." href="#ref-CR73" id="ref-link-section-d15230973e2356_2">73</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 74" title="Henikoff, S., Janssens, D., Kaya-Okur, H. S., Henikoff, J. &amp; Ahmad, K. CUT&amp;Tag@home V1. protools.io 
                  https://www.protocols.io/view/cut-amp-tag-home-bd26i8he
                  
                 (2020)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR74" id="ref-link-section-d15230973e2359">74</a></sup>.</p><h4 id="Sec38">Ribosome and nuclei isolation</h4><p>Nuclei and ribosomes were isolated from snap-frozen epiAT from AdipoERCre x NuTRAP mice in ice-cold Nuclei Extraction Buffer (Miltenyi, catalogue no. 130-128-024) supplemented with 0.2 U µl<sup>−1</sup> recombinant RNase Inhibitor (Takara, catalogue no. 2313), 1× cOmplete EDTA-free Protease Inhibitor (Roche, catalogue no. 5056489001) and 10 mM sodium butyrate using the gentleMACS Octo Dissociator (Miltenyi, catalogue no. 130-096-427), using C-tubes (Miltenyi, catalogue no. 130-093-237). The nuclei suspension was filtered through a 50 µm strainer and centrifuged at 4 °C, 500<i>g</i> for 5 min. The supernatant was transferred into a new tube and supplemented with 2 mM dithiothreitol, 100 µg ml<sup>−1</sup> cycloheximide (Sigma-Aldrich, catalogue no. 01810) and 1 mg ml<sup>−1</sup> sodium heparin (Sigma-Aldrich, catalogue no. H3149-10KU) and kept on ice. The nuclei pellet was resuspended in 1 ml of PBS-BSA (1%, w/v) supplemented with 0.2 U µl<sup>−1</sup> RNase inhibitor, 1× cOmplete EDTA-free Protease Inhibitor and 10 mM sodium butyrate and transferred into a new 1.5 ml tube. Nuclei were centrifuged and subsequently bound to Dynabeads MyOne Streptavidin C1 beads (Thermo Fisher, catalogue no. 65002) for 30 min at 4 °C followed by three washes with PBS-BSA (1% w/v).</p><h4 id="Sec39">TRAP–seq</h4><p>Per sample, 25 µl of GFP-Trap Magnetic Agarose Beads (ChromoTEK, catalogue no. gtma-20) were washed in 2 ml of polysome lysis buffer (50 mM TRIS-HCl pH 7.5, 100 mM NaCl, 12 mM MgCl<sub>2</sub>, 1% Igepal CA-630 (Sigma-Aldrich, catalogue no. I8896), 1× protease inhibitor). The supernatant was mixed with the beads and incubated at 4 °C on a rotator for 1–2 h. Subsequently, tubes were put on a magnetic stand and the supernatant was removed. The beads were washed three times with polysome lysis buffer supplemented with 2 mM dithiothreitol (Sigma-Aldrich, catalogue no. D0632-10G), 100 µg ml<sup>−1</sup> cycloheximide (Sigma, catalogue no. D0632-10G) and 1 mg ml<sup>−1</sup> sodium heparin (VWR, catalogue no. ACRO411210010) and resuspended in 1 ml Trizol (Thermo Fisher, catalogue no. 15596). Trizol preserved samples were kept at −80 °C until RNA isolation. RNA was isolated by adding 200 µl of chloroform (Sigma-Aldrich, catalogue no. 288306) to samples, followed by shaking and centrifugation at 4 °C, 12,000<i>g</i> for 15 min. The aqueous phase was transferred into a new tube and RNA was isolated and DNase treated with the RNA Clean and Concentrator-5 kit (Zymo Research, catalogue no. R1016), following the manufacturer’s instructions.</p><p>RNA libraries were prepared by performing reverse transcription and template switching using Maxima H Minus reverse transcriptase (Thermo Fisher, catalogue no. EP0753), a template switch oligo and an oligodT primer to generate full-length cDNA. cDNA was amplified using the KAPA Hotstart 2x ReadyMix (Roche Diagnostics, catalogue no. 7958935001). Then, 1–3 ng of cDNA was tagmentated using 1.3 µg of Tn5 and amplified using KAPA HiFi plus dNTPs (Roche Diagnostics, catalogue no. 07958846001) and the following PCR settings: 72 °C 5 min, 98 °C 30 s, 10 cycles of 98 °C for 10 s, 63 °C for 30 s, 72 °C for 1 min, hold at 4 °C. Libraries were quantified using the KAPA library quantification kit (Roche Diagnostics, catalogue no. 079602), and sequenced in PE150 mode on a NovaSeq 6000 at Novogene.</p><h4 id="Sec40">CUT&amp;Tag</h4><p>CUT&amp;Tag was performed as previously described with minor adjustments<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 74" title="Henikoff, S., Janssens, D., Kaya-Okur, H. S., Henikoff, J. &amp; Ahmad, K. CUT&amp;Tag@home V1. protools.io 
                  https://www.protocols.io/view/cut-amp-tag-home-bd26i8he
                  
                 (2020)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR74" id="ref-link-section-d15230973e2411">74</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 75" title="Galle, E. et al. H3K18 lactylation marks tissue-specific active enhancers. Genome Biol. 23, 207 (2022)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR75" id="ref-link-section-d15230973e2414">75</a></sup>. All buffers were supplemented with 1 x cOmplete EDTA-free Protease Inhibitor and 10 mM sodium butyrate. Briefly, nuclei bound to beads were aliquoted into 96-well LoBind plates (Eppendorf, catalogue no. 0030129547) and incubated with primary antibodies—anti-H3K4me3 (abcam, catalogue no. ab8580), anti-H3K27me3 (Cell Signaling Technology, catalogue no. C36B11), anti-H3K27ac (abcam, catalogue no. ab4729), anti-H3K4me1 (abcam, catalogue no. ab8895)—overnight at 4 °C. With the plate on a magnet, the primary antibody solution was removed, and the beads were resuspended in secondary antibody solution (guinea pig anti-rabbit IgG (antibodies-online, catalogue no. ABIN101961)) and incubated at room temperature. pA-Tn5 was bound to antibodies, and transposition was performed at 37 °C and stopped using TAPS-Wash solution. Nuclei were lysed and pA-Tn5 decrosslinked using SDS-release solution. PCR was performed using KAPA HiFi plus dNTPs (Roche Diagnostics, catalogue no. 07958846001) with the following PCR settings: 72 °C 5 min, 98 °C 30 s, 15 cycles of 98 °C 10 s, 63 °C 30 s, and 72 °C final extension for 1 min, hold at 4 °C.</p><h4 id="Sec41">ATAC–seq</h4><p>Beads with nuclei were resuspended in ATAC–seq solution (10 mM TAPS pH 8.5, 5 mM MgCl<sub>2</sub>, 10% DMF (Sigma-Aldrich, catalogue no. D4551), 0.2 µg µl<sup>−1</sup> transposase (Tn5)) and incubated at 37 °C for 30 min. Thereafter, 100 µl of DNA binding buffer (Zymo Research, catalogue no. D4003-1) was added and samples were stored at −20 °C. Then, DNA was extracted using Zymo DNA Clean and Concentrator-5 (Zymo Research, catalogue no. D4004). Library amplification was performed using KAPA HiFi plus dNTPs (Roche Diagnostics, catalogue no. 07958846001) and the following PCR settings: 72 °C 5 min, 98 °C 30 s, 10 cycles of 98 °C 10 s, 63 °C 30 s, 72 °C 1 min, hold at 4 °C.</p><p>Both ATAC–seq and CUT&amp;Tag libraries were cleaned using SPRI beads, eluted in nuclease-free water and pooled equimolecularly after library quantification using the KAPA library quantification kit (Roche Diagnostics, catalogue no. 079602). Libraries were sequenced in PE150 mode on a NovaSeq 6000 at Novogene.</p><h3 id="Sec42">Sequencing data processing</h3><h4 id="Sec43">snRNA-seq data processing and analysis</h4>
                    <h3 id="FPar1">Data integration and differential expression analysis for mouse snRNA-seq</h3>
                    <p>The 10x Genomics Cell Ranger v.6.1.2 pipeline was used for demultiplexing, read alignment to reference genome mm10-2020A (10x Genomics), barcode processing and unique molecular identifier (UMI) counting with Include introns argument set to ‘True’. The R package Seurat v.4.1.0 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 76" title="Hao, Y. et al. Integrated analysis of multimodal single-cell data. Cell 184, 3573–3587.e29 (2021)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR76" id="ref-link-section-d15230973e2450">76</a></sup>) was used to process, integrate and analyse datasets. scDblFinder<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 77" title="Germain, P.-L., Lun, A., Garcia Meixide, C., Macnair, W. &amp; Robinson, M. D. Doublet identification in single-cell sequencing data using scDblFinder. F1000Research 10, 979 (2022)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR77" id="ref-link-section-d15230973e2454">77</a></sup> was used to identify and remove doublets. Nuclei with unique feature counts less than 500 or greater than 3,000 and UMI counts greater than 40,000 were discarded during quality control (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig16">11a</a>). Highly expressed genes such as mitochondrial genes, pseudogenes and <i>Malat1</i> were excluded from the count matrix before normalization. SoupX<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 78" title="Young, M. D. &amp; Behjati, S. SoupX removes ambient RNA contamination from droplet-based single-cell RNA sequencing data. GigaScience 9, giaa151 (2020)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR78" id="ref-link-section-d15230973e2464">78</a></sup> was used to estimate potential ambient RNA contamination in all samples, but no sample required any correction. Samples were normalized using sctransform and integrated using the CCA (canonical correlation analysis) method built into Seurat. Filtered, normalized and integrated nuclei data were clustered by using the Louvain algorithm with a resolution of 0.4 using the first 30 principal components. Cluster markers were identified on the basis of differential gene expression analysis (Wilcoxon rank-sum test with |log<sub>2</sub>FC| &gt; 0.25 and adjusted <i>P</i> &lt; 0.05). Clusters were then annotated on the basis of known markers from literature<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 34" title="Emont, M. P. et al. A single-cell atlas of human and mouse white adipose tissue. Nature 603, 926–933 (2022)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR34" id="ref-link-section-d15230973e2474">34</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 36" title="Maniyadath, B., Zhang, Q., Gupta, R. K. &amp; Mandrup, S. Adipose tissue at single-cell resolution. Cell Metab. 35, 386–413 (2023)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR36" id="ref-link-section-d15230973e2477">36</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 37" title="Merrick, D. et al. Identification of a mesenchymal progenitor cell hierarchy in adipose tissue. Science 364, eaav2501 (2019)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR37" id="ref-link-section-d15230973e2480">37</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 46" title="Sárvári, A. K. et al. Plasticity of epididymal adipose tissue in response to diet-induced obesity at single-nucleus resolution. Cell Metab. 33, 437–453.e5 (2021)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR46" id="ref-link-section-d15230973e2483">46</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 79" title="Shan, B. et al. Perivascular mesenchymal cells control adipose-tissue macrophage accrual in obesity. Nat. Metab. 2, 1332–1349 (2020)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR79" id="ref-link-section-d15230973e2486">79</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 80" title="Schwalie, P. C. et al. A stromal cell population that inhibits adipogenesis in mammalian fat depots. Nature 559, 103–108 (2018)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR80" id="ref-link-section-d15230973e2489">80</a></sup>. Additionally, our manual cluster annotation was confirmed by reference mapping against a reference male mouse epiAT<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 34" title="Emont, M. P. et al. A single-cell atlas of human and mouse white adipose tissue. Nature 603, 926–933 (2022)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR34" id="ref-link-section-d15230973e2493">34</a></sup> dataset (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig16">11b,c</a>). Differential expression analysis (Wilcoxon rank-sum test with |log<sub>2</sub>FC| &gt; 0.5 and adjusted <i>P</i> &lt; 0.01) per cell type between different conditions was done using the FindMarkers function from Seurat. Differential expression analysis hits were intersected with a list of epigenetic modifier genes (see the <a data-track="click" data-track-label="link" data-track-action="section anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Sec52">Source Data</a> to Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig13">8</a>) to investigate their expression dynamics. For visualization of snRNA-seq data we used the R package SCpubr v.1 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 81" title="Blanco-Carmona, E. Generating publication ready visualizations for single cell transcriptomics using SCpubr. Preprint at bioRxiv 
                  https://doi.org/10.1101/2022.02.28.482303
                  
                 (2022)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR81" id="ref-link-section-d15230973e2512">81</a></sup>).</p>
                  
                    <h3 id="FPar2">Data integration and differential expression analysis for human snRNA-seq</h3>
                    <p>The 10x Genomics Cell Ranger v.7.2.0 pipeline was used for demultiplexing, read alignment to reference genome GRCh38-2020-A (10x Genomics), barcode processing and UMI counting, with force cells set to 10,000. The R package Seurat v.4.1.0 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 76" title="Hao, Y. et al. Integrated analysis of multimodal single-cell data. Cell 184, 3573–3587.e29 (2021)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR76" id="ref-link-section-d15230973e2524">76</a></sup>) was used to process, integrate and analyse datasets. scDblFinder<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 77" title="Germain, P.-L., Lun, A., Garcia Meixide, C., Macnair, W. &amp; Robinson, M. D. Doublet identification in single-cell sequencing data using scDblFinder. F1000Research 10, 979 (2022)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR77" id="ref-link-section-d15230973e2528">77</a></sup> was used to identify and remove doublets. Nuclei with unique feature counts &lt;300 or &gt;4,000 (LTSS) / 6,000 (NEFA), UMI counts &gt;15,000 (LTSS) / 25,000 (NEFA) and mitochondrial gene counts greater than 5% were discarded during quality control (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig17">12</a>). SoupX<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 78" title="Young, M. D. &amp; Behjati, S. SoupX removes ambient RNA contamination from droplet-based single-cell RNA sequencing data. GigaScience 9, giaa151 (2020)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR78" id="ref-link-section-d15230973e2535">78</a></sup> was used to estimate and correct for potential ambient RNA contamination in all samples. Samples were normalized using sctransform and integrated using the CCA method built into Seurat. Filtered, normalized and integrated nuclei data were clustered by using Louvain algorithm using the first 30 principal components. For each study, the cluster resolution was determined using the R package clustree<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 82" title="Zappia, L. &amp; Oshlack, A. Clustering trees: a visualization for evaluating clusterings at multiple resolutions. GigaScience 7, giy083 (2018)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR82" id="ref-link-section-d15230973e2539">82</a></sup>. Cluster markers were identified on the basis of differential gene expression analysis (Wilcoxon rank-sum test with |log<sub>2</sub>FC| &gt; 0.25 and adjusted <i>P</i> &lt; 0.01). Clusters were then annotated on the basis of known markers from literature<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Emont, M. P. et al. A single-cell atlas of human and mouse white adipose tissue. Nature 603, 926–933 (2022)." href="#ref-CR34" id="ref-link-section-d15230973e2549">34</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Massier, L. et al. An integrated single cell and spatial transcriptomic map of human white adipose tissue. Nat. Commun. 14, 1438 (2023)." href="#ref-CR35" id="ref-link-section-d15230973e2549_1">35</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Maniyadath, B., Zhang, Q., Gupta, R. K. &amp; Mandrup, S. Adipose tissue at single-cell resolution. Cell Metab. 35, 386–413 (2023)." href="#ref-CR36" id="ref-link-section-d15230973e2549_2">36</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 37" title="Merrick, D. et al. Identification of a mesenchymal progenitor cell hierarchy in adipose tissue. Science 364, eaav2501 (2019)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR37" id="ref-link-section-d15230973e2552">37</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 83" title="Sun, W. et al. snRNA-seq reveals a subpopulation of adipocytes that regulates thermogenesis. Nature 587, 98–102 (2020)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR83" id="ref-link-section-d15230973e2555">83</a></sup>. Additionally, our manual cluster annotation was confirmed by reference mapping against reference human white AT atlas<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 34" title="Emont, M. P. et al. A single-cell atlas of human and mouse white adipose tissue. Nature 603, 926–933 (2022)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR34" id="ref-link-section-d15230973e2559">34</a></sup> (Extended Data Figs. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig7">2</a> and <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig8">3</a>). For each AT depot, adipocytes from two studies were integrated together using the first 20 principal components following the steps as mentioned above. Differential expression analysis (Wilcoxon rank-sum test with |log<sub>2</sub>FC| &gt; 0.5 and adjusted <i>P</i> &lt; 0.01) per cell type between different conditions was done using the FindMarkers function from Seurat. Differential expression analysis hits were validated using MAST and likelihood-ratio tests using the FindMarkers function from Seurat. For visualization of snRNA-seq data, we used the R package SCpubr v.1 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 81" title="Blanco-Carmona, E. Generating publication ready visualizations for single cell transcriptomics using SCpubr. Preprint at bioRxiv 
                  https://doi.org/10.1101/2022.02.28.482303
                  
                 (2022)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR81" id="ref-link-section-d15230973e2575">81</a></sup>).</p>
                  
                    <h3 id="FPar3">SNP-based demultiplexing of human snRNA-seq datasets</h3>
                    <p>To perform SNP calling and demultiplexing on the pooled samples, cellsnp-lite<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 84" title="Huang, X. &amp; Huang, Y. Cellsnp-lite: an efficient tool for genotyping single cells. Bioinformatics 37, 4569–4571 (2021)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR84" id="ref-link-section-d15230973e2587">84</a></sup> was first used to call SNPs on a cell level using the 1000 Genomes-based reference variant call file for hg38 at a resolution of 7.4 million SNPs. SNPs with less than 20 counts and a minor allele frequency of less than 10% were filtered out, as per the developer recommendations. Finally, the tool vireo<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 85" title="Huang, Y., McCarthy, D. J. &amp; Stegle, O. Vireo: Bayesian demultiplexing of pooled single-cell RNA-seq data without genotype reference. Genome Biol. 20, 273 (2019)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR85" id="ref-link-section-d15230973e2591">85</a></sup> was used to demultiplex the pooled data using the cellsnp-lite-derived genotype information.</p>
                    <p>For each donor, we analysed tissue composition and removed nuclei belonging to donors in the case in which no nuclei were assigned as adipocytes (one case in NEFA) or more than 50% or nuclei were assigned as B cells (one case in MTSS; lean donor) after correspondence with surgeons.</p>
                  
                    <h3 id="FPar4">Transcriptional retention</h3>
                    <p>DEGs from obese and WL cells from mouse and human were overlayed, respectively. A DEG was considered restored if it was no longer deregulated in WL cells when compared with controls. If not restored, we considered a DEG part of a transcriptional memory. Clusters identified as similar cell types (for example, three clusters of endothelial cells) were merged for DEG quantification but not differential expression analysis itself. For human snRNA-seq, only cell types for which we obtained at least 30 cells per donor were considered for the retention analysis. T cells were not included in differential expression analysis or transcriptional retention analysis. For integrated human adipocyte differential expression analysis quantification, non-coding transcripts were excluded.</p>
                  
                    <h3 id="FPar5">TRAP–seq</h3>
                    <p>Quality control of the raw reads was performed using FastQC v.0.11.9. Raw reads were trimmed using TrimGalore v.0.6.6 (<a href="https://github.com/FelixKrueger/TrimGalore">https://github.com/FelixKrueger/TrimGalore</a>). Filtered reads were aligned against the reference mouse genome assembly mm10 using HISAT2 v.2.2.1. Raw gene counts were quantified using the featureCounts<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 86" title="Liao, Y., Smyth, G. K. &amp; Shi, W. featureCounts: an efficient general purpose program for assigning sequence reads to genomic features. Bioinformatics 30, 923–930 (2014)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR86" id="ref-link-section-d15230973e2622">86</a></sup> program of subread v.2.0.1. Differential expression analysis was performed using the R package EdgeR<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 87" title="Robinson, M. D., McCarthy, D. J. &amp; Smyth, G. K. edgeR: a Bioconductor package for differential expression analysis of digital gene expression data. Bioinformatics 26, 139–140 (2010)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR87" id="ref-link-section-d15230973e2626">87</a></sup>, with |log<sub>2</sub>FC| ≥ 1 and nominal <i>P</i> &lt; 0.01 as cut-offs.</p>
                  <h4 id="Sec44">CUT&amp;Tag and ATAC–seq data processing and analysis</h4><p>Quality control of CUT&amp;Tag and ATAC–seq data and generation of bedgraph files was performed as described previously<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 75" title="Galle, E. et al. H3K18 lactylation marks tissue-specific active enhancers. Genome Biol. 23, 207 (2022)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR75" id="ref-link-section-d15230973e2644">75</a></sup>. Peaks were called from CUT&amp;Tag sequencing and ATAC–seq libraries on individual bedgraph files using SEACR<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 88" title="Meers, M. P., Tenenbaum, D. &amp; Henikoff, S. Peak calling by Sparse Enrichment Analysis for CUT&amp;RUN chromatin profiling. Epigenetics Chromatin 12, 42 (2019)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR88" id="ref-link-section-d15230973e2648">88</a></sup> v.1.3 in stringent mode with a peak calling threshold of 0.01. Peaks overlapping with mouse blacklist regions<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 89" title="Amemiya, H. M., Kundaje, A. &amp; Boyle, A. P. The ENCODE blacklist: identification of problematic regions of the genome. Sci. Rep. 9, 9354 (2019)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR89" id="ref-link-section-d15230973e2652">89</a></sup> were filtered out. Called peaks were annotated using the R package ChIPSeeker<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 90" title="Yu, G., Wang, L.-G. &amp; He, Q.-Y. ChIPseeker: an R/Bioconductor package for ChIP peak annotation, comparison and visualization. Bioinformatics 31, 2382–2383 (2015)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR90" id="ref-link-section-d15230973e2656">90</a></sup>. Peak fold enrichment against genomic features was calculated using the formula: Σ(base pair (bp) overlap) × genome_size/[Σ(bp hPTM peak) × Σ(bp genomic feature)]. Genomic features tracks were downloaded from ENCODE using the R package annotatr<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 91" title="Cavalcante, R. G. &amp; Sartor, M. A. annotatr: genomic regions in context. Bioinformatics 33, 2381–2383 (2017)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR91" id="ref-link-section-d15230973e2660">91</a></sup>. Visual quality control of bam files was performed with Seqmonk<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 92" title="Andrews, S. SeqMonk mapped sequence analysis tool. Babraham Bioinformatics 
                  https://www.bioinformatics.babraham.ac.uk/projects/seqmonk/
                  
                 (2022)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR92" id="ref-link-section-d15230973e2665">92</a></sup>. Called peaks were combined to generate a union peak list and quantified using the R package chromVAR<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 93" title="Schep, A. N., Wu, B., Buenrostro, J. D. &amp; Greenleaf, W. J. chromVAR: inferring transcription-factor-associated accessibility from single-cell epigenomic data. Nat. Methods 14, 975–978 (2017)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR93" id="ref-link-section-d15230973e2669">93</a></sup> v.1.16, generating a raw peak count matrix.</p>
                    <h3 id="FPar7">MOFA</h3>
                    <p>MOFA<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 50" title="Argelaguet, R. et al. Multi-omics factor analysis—a framework for unsupervised integration of multi-omics data sets. Mol. Syst. Biol. 14, e8124 (2018)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR50" id="ref-link-section-d15230973e2680">50</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 94" title="Argelaguet, R. et al. MOFA+: a statistical framework for comprehensive integration of multi-modal single-cell data. Genome Biol. 21, 111 (2020)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR94" id="ref-link-section-d15230973e2683">94</a></sup> was run to identify the driving variation source across all conditions using all data modalities. For each modality, the top 3,000 variable features (genes or peaks) between all samples were selected using the R package DESeq2 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 95" title="Love, M. I., Huber, W. &amp; Anders, S. Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. Genome Biol. 15, 550 (2014)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR95" id="ref-link-section-d15230973e2687">95</a></sup>) and used as input to train the MOFA model. The trained MOFA model represented data variability in terms of five latent factors, which were further explored and visualized.</p>
                  
                    <h3 id="FPar8">Generation of enhancer tracks of adipocytes</h3>
                    <p>Adipocyte chromatin states were identified using ChromHMM v.1.22 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 96" title="Ernst, J. &amp; Kellis, M. ChromHMM: automating chromatin-state discovery and characterization. Nat. Methods 9, 215–216 (2012)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR96" id="ref-link-section-d15230973e2699">96</a></sup>) in concatenated mode with binned bam files (200-bp bins) from each condition combining all hPTMs and ATAC–seq. After final model selection<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 75" title="Galle, E. et al. H3K18 lactylation marks tissue-specific active enhancers. Genome Biol. 23, 207 (2022)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR75" id="ref-link-section-d15230973e2703">75</a></sup> with eight chromatin states and emission parameter calculation of hPTMs and ATAC–seq, chromatin state fold enrichment was performed against genomic features and ENCODE candidate <i>cis</i>-regulatory elements. Enhancer states were selected on the basis of genomic localization and hPTM enrichment. Subsequently, an enhancer track was generated per condition and merged for differential analysis.</p>
                  <h4 id="Sec45">Differential analysis of hPTMs and ATAC–seq</h4>
                    <h3 id="FPar9">Promoters</h3>
                    <p>Promoters were defined using the getPromoters function from ChIPSeeker with <i>TxDb.Mmusculus.UCSC.mm10.knownGene</i> as input and setting the TSSRegion to c(-2000, 2000). Peaks overlapping with promoters were extracted using the annotatePeak function from ChIPseeker<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 90" title="Yu, G., Wang, L.-G. &amp; He, Q.-Y. ChIPseeker: an R/Bioconductor package for ChIP peak annotation, comparison and visualization. Bioinformatics 31, 2382–2383 (2015)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR90" id="ref-link-section-d15230973e2726">90</a></sup> by selecting peaks annotated as promoters. For differential analysis, our raw peak count matrix was filtered for these promoter regions and counts were aggregated at gene level. Differential analysis of the same hPTM between two conditions was performed using the R package EdgeR<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 87" title="Robinson, M. D., McCarthy, D. J. &amp; Smyth, G. K. edgeR: a Bioconductor package for differential expression analysis of digital gene expression data. Bioinformatics 26, 139–140 (2010)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR87" id="ref-link-section-d15230973e2730">87</a></sup> with nominal <i>P</i> &lt; 0.01 and |log<sub>2</sub>FC| &gt; 1 as cut-offs.</p>
                  
                    <h3 id="FPar10">Enhancers</h3>
                    <p>ChromHMM was used to identify regions in the genome that were marked by H3K4me1, H3K27ac and open (ATAC–seq) but not enriched for H3K4me3 and that were not promoters (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig14">9b–e</a>). States 6 and 5 were selected as enhancer regions on the basis of their genomic locations (distal enhancer elements) (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig14">9b–e</a>).</p>
                    <p>Our raw peak count matrix was filtered for enhancer regions defined by chromHMM, and peaks around the TSS (±2,000 bp) were discarded. Linkage of putative enhancers to genes was done using the R package ChIPSeeker by selecting the closest gene (TSS or gene body) within 20,000 bp distance. Putative enhancers farther away than 20,000 from a TSS or gene body were not linked to any gene and were discarded from downstream GSEA.</p>
                    <p>For each hPTM, the raw filtered peak matrices were log-normalized using the R package EdgeR and Pearson’s correlation coefficient was computed using the cor function from the R package stats v.3.6.2.</p>
                    <p>Differential analysis of the same hPTM between two conditions was performed using the R package EdgeR with nominal FDR &lt; 0.05 and |log<sub>2</sub>FC| &gt; 1 as cut-offs.</p>
                  
                    <h3 id="FPar11">PCA</h3>
                    <p>Raw gene and promoter/enhancer-specific peak count matrices were log-normalized using the R package EdgeR. PCA of the normalized count matrices was performed using the prcomp function of R package stats v.3.6.2.</p>
                  
                    <h3 id="FPar12">GSEA</h3>
                    <p>GSEA was performed using the R package enrichR<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Chen, E. Y. et al. Enrichr: interactive and collaborative HTML5 gene list enrichment analysis tool. BMC Bioinformatics 14, 128 (2013)." href="#ref-CR97" id="ref-link-section-d15230973e2780">97</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Kuleshov, M. V. et al. Enrichr: a comprehensive gene set enrichment analysis web server 2016 update. Nucleic Acids Res. 44, W90–W97 (2016)." href="#ref-CR98" id="ref-link-section-d15230973e2780_1">98</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 99" title="Xie, Z. et al. Gene set knowledge discovery with Enrichr. Curr. Protoc. 1, e90 (2021)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR99" id="ref-link-section-d15230973e2783">99</a></sup>. For generation of heatmaps summarizing GSEA across cell types, significantly enriched terms were selected using the adjusted <i>P</i> value (&lt;0.01) and the combined.score (enrichment score) was scaled and visualized.</p>
                  <h3 id="Sec46">Visualization</h3><p>R v.4.2, GraphPad Prism v.9.5.1 and Seqmonk v.1.48.1 were used to generate plots and Affinity Designer and Publisher were used to adjust plots for clarity (for example, colour schemes).</p><h3 id="Sec47">Statistical analysis of physiological parameters from mice</h3><p>GraphPad Prism v.9.5.1 was used to analyse physiological data from mice. Each dataset of physiological parameters was tested for normality using the Shapiro–Wilk test. On the basis of the results, parametric or non-parametric tests were used to compare experimental with age-matched control groups. Tests are indicated in figure legends and the <a data-track="click" data-track-label="link" data-track-action="section anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Sec52">Source Data</a>.</p><h3 id="Sec48">Reporting summary</h3><p>Further information on research design is available in the <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#MOESM1">Nature Portfolio Reporting Summary</a> linked to this article.</p></div></div></section>
                    
                </div><div>
                <section data-title="Data availability"><div id="data-availability-section"><h2 id="data-availability">Data availability</h2><p>All mouse sequencing data that support the findings of this study have been deposited on GEO, with the accession code <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE236580">GSE236580</a>. Human snRNA-seq data from the MTSS and LTSS cohorts are available upon request from C.W. and M.B. Human snRNA-seq data from the NEFA cohort are available upon request from F.v.M., N.K. and M.R. Analysis code for human and mouse data is available on GitHub and Zenodo<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 100" title="Hinte, L. C. et al. Adipose tissue retains an epigenetic memory of obesity after weight loss. Zenodo 
                  https://doi.org/10.5281/zenodo.13870942
                  
                 (2024)." href="https://g-w1.github.io/articles/s41586-024-08165-7#ref-CR100" id="ref-link-section-d15230973e2936">100</a></sup>. An interactive snRNA-seq data browser link and links to interactive tables with results of differential gene expression and epigenetic analysis are available on GitHub (<a href="https://github.com/vonMeyennLab/AT_memory">https://github.com/vonMeyennLab/AT_memory</a>). <a data-track="click" data-track-label="link" data-track-action="section anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Sec52">Source data</a> are provided with this paper.</p></div></section><div id="MagazineFulltextArticleBodySuffix"><section aria-labelledby="Bib1" data-title="References"><div id="Bib1-section"><h2 id="Bib1">References</h2><div id="Bib1-content"><div data-container-section="references"><ol data-track-component="outbound reference" data-track-context="references section"><li data-counter="1."><p id="ref-CR1">Zimmet, P., Alberti, K. G. M. M. &amp; Shaw, J. Global and societal implications of the diabetes epidemic. <i>Nature</i> <b>414</b>, 782–787 (2001).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://g-w1.github.io/articles/cas-redirect/1:CAS:528:DC%2BD38Xhtlymtg%3D%3D" aria-label="CAS reference 1">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=11742409" aria-label="PubMed reference 1">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="ads reference" data-track-action="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2001Natur.414..782Z" aria-label="ADS reference 1">ADS</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 1" href="http://scholar.google.com/scholar_lookup?&amp;title=Global%20and%20societal%20implications%20of%20the%20diabetes%20epidemic&amp;journal=Nature&amp;volume=414&amp;pages=782-787&amp;publication_year=2001&amp;author=Zimmet%2CP&amp;author=Alberti%2CKGMM&amp;author=Shaw%2CJ">
                    Google Scholar</a> 
                </p></li><li data-counter="2."><p id="ref-CR2">Wolfe, B. M., Kvach, E. &amp; Eckel, R. H. Treatment of obesity: weight loss and bariatric surgery. <i>Circ. Res.</i> <b>118</b>, 1844 (2016).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://g-w1.github.io/articles/cas-redirect/1:CAS:528:DC%2BC28XosFyrsL4%3D" aria-label="CAS reference 2">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27230645" aria-label="PubMed reference 2">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4888907" aria-label="PubMed Central reference 2">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 2" href="http://scholar.google.com/scholar_lookup?&amp;title=Treatment%20of%20obesity%3A%20weight%20loss%20and%20bariatric%20surgery&amp;journal=Circ.%20Res.&amp;volume=118&amp;publication_year=2016&amp;author=Wolfe%2CBM&amp;author=Kvach%2CE&amp;author=Eckel%2CRH">
                    Google Scholar</a> 
                </p></li><li data-counter="3."><p id="ref-CR3">Nordmo, M., Danielsen, Y. S. &amp; Nordmo, M. The challenge of keeping it off, a descriptive systematic review of high‐quality, follow‐up studies of obesity treatments. <i>Obes. Rev.</i> <b>21</b>, e12949 (2020).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31675146" aria-label="PubMed reference 3">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 3" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20challenge%20of%20keeping%20it%20off%2C%20a%20descriptive%20systematic%20review%20of%20high%E2%80%90quality%2C%20follow%E2%80%90up%20studies%20of%20obesity%20treatments&amp;journal=Obes.%20Rev.&amp;volume=21&amp;publication_year=2020&amp;author=Nordmo%2CM&amp;author=Danielsen%2CYS&amp;author=Nordmo%2CM">
                    Google Scholar</a> 
                </p></li><li data-counter="4."><p id="ref-CR4">Contreras, R. E., Schriever, S. C. &amp; Pfluger, P. T. Physiological and epigenetic features of yoyo dieting and weight control. <i>Front. Genet.</i> <b>10</b>, 1015 (2019).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://g-w1.github.io/articles/cas-redirect/1:CAS:528:DC%2BB3cXhslenu77E" aria-label="CAS reference 4">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31921275" aria-label="PubMed reference 4">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6917653" aria-label="PubMed Central reference 4">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 4" href="http://scholar.google.com/scholar_lookup?&amp;title=Physiological%20and%20epigenetic%20features%20of%20yoyo%20dieting%20and%20weight%20control&amp;journal=Front.%20Genet.&amp;volume=10&amp;publication_year=2019&amp;author=Contreras%2CRE&amp;author=Schriever%2CSC&amp;author=Pfluger%2CPT">
                    Google Scholar</a> 
                </p></li><li data-counter="5."><p id="ref-CR5">Franz, M. J. et al. Weight-loss outcomes: a systematic review and meta-analysis of weight-loss clinical trials with a minimum 1-year follow-up. <i>J. Am. Diet. Assoc.</i> <b>107</b>, 1755–1767 (2007).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17904936" aria-label="PubMed reference 5">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 5" href="http://scholar.google.com/scholar_lookup?&amp;title=Weight-loss%20outcomes%3A%20a%20systematic%20review%20and%20meta-analysis%20of%20weight-loss%20clinical%20trials%20with%20a%20minimum%201-year%20follow-up&amp;journal=J.%20Am.%20Diet.%20Assoc.&amp;volume=107&amp;pages=1755-1767&amp;publication_year=2007&amp;author=Franz%2CMJ">
                    Google Scholar</a> 
                </p></li><li data-counter="6."><p id="ref-CR6">Wilding, J. P. H. et al. Weight regain and cardiometabolic effects after withdrawal of semaglutide: the STEP 1 trial extension. <i>Diabetes Obes. Metab.</i> <b>24</b>, 1553–1564 (2022).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://g-w1.github.io/articles/cas-redirect/1:CAS:528:DC%2BB38XhsVansLvJ" aria-label="CAS reference 6">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=35441470" aria-label="PubMed reference 6">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9542252" aria-label="PubMed Central reference 6">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 6" href="http://scholar.google.com/scholar_lookup?&amp;title=Weight%20regain%20and%20cardiometabolic%20effects%20after%20withdrawal%20of%20semaglutide%3A%20the%20STEP%201%20trial%20extension&amp;journal=Diabetes%20Obes.%20Metab.&amp;volume=24&amp;pages=1553-1564&amp;publication_year=2022&amp;author=Wilding%2CJPH">
                    Google Scholar</a> 
                </p></li><li data-counter="7."><p id="ref-CR7">Izquierdo, A. G. &amp; Crujeiras, A. B. Obesity-related epigenetic changes after bariatric surgery. <i>Front. Endocrinol.</i> <b>10</b>, 232 (2019).</p><p><a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 7" href="http://scholar.google.com/scholar_lookup?&amp;title=Obesity-related%20epigenetic%20changes%20after%20bariatric%20surgery&amp;journal=Front.%20Endocrinol.&amp;volume=10&amp;publication_year=2019&amp;author=Izquierdo%2CAG&amp;author=Crujeiras%2CAB">
                    Google Scholar</a> 
                </p></li><li data-counter="8."><p id="ref-CR8">Mileti, E. et al. Human white adipose tissue displays selective insulin resistance in the obese state. <i>Diabetes</i> <b>70</b>, 1486–1497 (2021).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://g-w1.github.io/articles/cas-redirect/1:CAS:528:DC%2BB3MXit12rsLfI" aria-label="CAS reference 8">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33863803" aria-label="PubMed reference 8">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 8" href="http://scholar.google.com/scholar_lookup?&amp;title=Human%20white%20adipose%20tissue%20displays%20selective%20insulin%20resistance%20in%20the%20obese%20state&amp;journal=Diabetes&amp;volume=70&amp;pages=1486-1497&amp;publication_year=2021&amp;author=Mileti%2CE">
                    Google Scholar</a> 
                </p></li><li data-counter="9."><p id="ref-CR9">Petrus, P. et al. Transforming growth factor-β3 regulates adipocyte number in subcutaneous white adipose tissue. <i>Cell Rep.</i> <b>25</b>, 551–560.e5 (2018).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://g-w1.github.io/articles/cas-redirect/1:CAS:528:DC%2BC1cXhvFeisrbO" aria-label="CAS reference 9">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30332637" aria-label="PubMed reference 9">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 9" href="http://scholar.google.com/scholar_lookup?&amp;title=Transforming%20growth%20factor-%CE%B23%20regulates%20adipocyte%20number%20in%20subcutaneous%20white%20adipose%20tissue&amp;journal=Cell%20Rep.&amp;volume=25&amp;pages=551-560.e5&amp;publication_year=2018&amp;author=Petrus%2CP">
                    Google Scholar</a> 
                </p></li><li data-counter="10."><p id="ref-CR10">Löfgren, P. et al. Long-term prospective and controlled studies demonstrate adipose tissue hypercellularity and relative leptin deficiency in the postobese state. <i>J. Clin. Endocrinol. Metab.</i> <b>90</b>, 6207–6213 (2005).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16131581" aria-label="PubMed reference 10">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 10" href="http://scholar.google.com/scholar_lookup?&amp;title=Long-term%20prospective%20and%20controlled%20studies%20demonstrate%20adipose%20tissue%20hypercellularity%20and%20relative%20leptin%20deficiency%20in%20the%20postobese%20state&amp;journal=J.%20Clin.%20Endocrinol.%20Metab.&amp;volume=90&amp;pages=6207-6213&amp;publication_year=2005&amp;author=L%C3%B6fgren%2CP">
                    Google Scholar</a> 
                </p></li><li data-counter="11."><p id="ref-CR11">Reddy, M. A., Zhang, E. &amp; Natarajan, R. Epigenetic mechanisms in diabetic complications and metabolic memory. <i>Diabetologia</i> <b>58</b>, 443–455 (2015).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://g-w1.github.io/articles/cas-redirect/1:CAS:528:DC%2BC2cXitVKqsbjF" aria-label="CAS reference 11">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25481708" aria-label="PubMed reference 11">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 11" href="http://scholar.google.com/scholar_lookup?&amp;title=Epigenetic%20mechanisms%20in%20diabetic%20complications%20and%20metabolic%20memory&amp;journal=Diabetologia&amp;volume=58&amp;pages=443-455&amp;publication_year=2015&amp;author=Reddy%2CMA&amp;author=Zhang%2CE&amp;author=Natarajan%2CR">
                    Google Scholar</a> 
                </p></li><li data-counter="12."><p id="ref-CR12">Chalmers, J. &amp; Cooper, M. E. UKPDS and the legacy effect. <i>N. Engl. J. Med.</i> <b>359</b>, 1618–1620 (2008).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://g-w1.github.io/articles/cas-redirect/1:CAS:528:DC%2BD1cXht1Wlt7nI" aria-label="CAS reference 12">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18843126" aria-label="PubMed reference 12">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 12" href="http://scholar.google.com/scholar_lookup?&amp;title=UKPDS%20and%20the%20legacy%20effect&amp;journal=N.%20Engl.%20J.%20Med.&amp;volume=359&amp;pages=1618-1620&amp;publication_year=2008&amp;author=Chalmers%2CJ&amp;author=Cooper%2CME">
                    Google Scholar</a> 
                </p></li><li data-counter="13."><p id="ref-CR13">Holman, R. R., Paul, S. K., Bethel, M. A., Matthews, D. R. &amp; Neil, H. A. W. 10-Year follow-up of intensive glucose control in type 2 diabetes. <i>N. Engl. J. Med.</i> <b>359</b>, 1577–1589 (2008).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://g-w1.github.io/articles/cas-redirect/1:CAS:528:DC%2BD1cXht1Wgt7nK" aria-label="CAS reference 13">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18784090" aria-label="PubMed reference 13">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 13" href="http://scholar.google.com/scholar_lookup?&amp;title=10-Year%20follow-up%20of%20intensive%20glucose%20control%20in%20type%202%20diabetes&amp;journal=N.%20Engl.%20J.%20Med.&amp;volume=359&amp;pages=1577-1589&amp;publication_year=2008&amp;author=Holman%2CRR&amp;author=Paul%2CSK&amp;author=Bethel%2CMA&amp;author=Matthews%2CDR&amp;author=Neil%2CHAW">
                    Google Scholar</a> 
                </p></li><li data-counter="14."><p id="ref-CR14">Guerra-Cantera, S. et al. Sex differences in metabolic recuperation after weight loss in high fat diet-induced obese mice. <i>Front. Endocrinol.</i> <b>12</b>, 796661 (2021).</p><p><a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 14" href="http://scholar.google.com/scholar_lookup?&amp;title=Sex%20differences%20in%20metabolic%20recuperation%20after%20weight%20loss%20in%20high%20fat%20diet-induced%20obese%20mice&amp;journal=Front.%20Endocrinol.&amp;volume=12&amp;publication_year=2021&amp;author=Guerra-Cantera%2CS">
                    Google Scholar</a> 
                </p></li><li data-counter="15."><p id="ref-CR15">Hahn, O. et al. A nutritional memory effect counteracts the benefits of dietary restriction in old mice. <i>Nat. Metab.</i> <b>1</b>, 1059–1073 (2019).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://g-w1.github.io/articles/cas-redirect/1:CAS:528:DC%2BB3cXks1KrtLY%3D" aria-label="CAS reference 15">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31742247" aria-label="PubMed reference 15">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6861129" aria-label="PubMed Central reference 15">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 15" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20nutritional%20memory%20effect%20counteracts%20the%20benefits%20of%20dietary%20restriction%20in%20old%20mice&amp;journal=Nat.%20Metab.&amp;volume=1&amp;pages=1059-1073&amp;publication_year=2019&amp;author=Hahn%2CO">
                    Google Scholar</a> 
                </p></li><li data-counter="16."><p id="ref-CR16">Schmitz, J. et al. Obesogenic memory can confer long-term increases in adipose tissue but not liver inflammation and insulin resistance after weight loss. <i>Mol. Metab.</i> <b>5</b>, 328–339 (2016).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://g-w1.github.io/articles/cas-redirect/1:CAS:528:DC%2BC28XnvFKnug%3D%3D" aria-label="CAS reference 16">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27110485" aria-label="PubMed reference 16">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4837291" aria-label="PubMed Central reference 16">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 16" href="http://scholar.google.com/scholar_lookup?&amp;title=Obesogenic%20memory%20can%20confer%20long-term%20increases%20in%20adipose%20tissue%20but%20not%20liver%20inflammation%20and%20insulin%20resistance%20after%20weight%20loss&amp;journal=Mol.%20Metab.&amp;volume=5&amp;pages=328-339&amp;publication_year=2016&amp;author=Schmitz%2CJ">
                    Google Scholar</a> 
                </p></li><li data-counter="17."><p id="ref-CR17">Siersbæk, M. et al. High fat diet-induced changes of mouse hepatic transcription and enhancer activity can be reversed by subsequent weight loss. <i>Sci. Rep.</i> <b>7</b>, 40220 (2017).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28071704" aria-label="PubMed reference 17">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5223143" aria-label="PubMed Central reference 17">PubMed Central</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="ads reference" data-track-action="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2017NatSR...740220S" aria-label="ADS reference 17">ADS</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 17" href="http://scholar.google.com/scholar_lookup?&amp;title=High%20fat%20diet-induced%20changes%20of%20mouse%20hepatic%20transcription%20and%20enhancer%20activity%20can%20be%20reversed%20by%20subsequent%20weight%20loss&amp;journal=Sci.%20Rep.&amp;volume=7&amp;publication_year=2017&amp;author=Siersb%C3%A6k%2CM">
                    Google Scholar</a> 
                </p></li><li data-counter="18."><p id="ref-CR18">Cottam, M. A., Caslin, H. L., Winn, N. C. &amp; Hasty, A. H. Multiomics reveals persistence of obesity-associated immune cell phenotypes in adipose tissue during weight loss and weight regain in mice. <i>Nat. Commun.</i> <b>13</b>, 2950 (2022).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://g-w1.github.io/articles/cas-redirect/1:CAS:528:DC%2BB38XhsVejsL7L" aria-label="CAS reference 18">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=35618862" aria-label="PubMed reference 18">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9135744" aria-label="PubMed Central reference 18">PubMed Central</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="ads reference" data-track-action="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2022NatCo..13.2950C" aria-label="ADS reference 18">ADS</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 18" href="http://scholar.google.com/scholar_lookup?&amp;title=Multiomics%20reveals%20persistence%20of%20obesity-associated%20immune%20cell%20phenotypes%20in%20adipose%20tissue%20during%20weight%20loss%20and%20weight%20regain%20in%20mice&amp;journal=Nat.%20Commun.&amp;volume=13&amp;publication_year=2022&amp;author=Cottam%2CMA&amp;author=Caslin%2CHL&amp;author=Winn%2CNC&amp;author=Hasty%2CAH">
                    Google Scholar</a> 
                </p></li><li data-counter="19."><p id="ref-CR19">Bondareva, O. et al. Single-cell profiling of vascular endothelial cells reveals progressive organ-specific vulnerabilities during obesity. <i>Nat. Metab.</i> <b>4</b>, 1591–1610 (2022).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://g-w1.github.io/articles/cas-redirect/1:CAS:528:DC%2BB38XjtVajsbrJ" aria-label="CAS reference 19">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=36400935" aria-label="PubMed reference 19">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9684070" aria-label="PubMed Central reference 19">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 19" href="http://scholar.google.com/scholar_lookup?&amp;title=Single-cell%20profiling%20of%20vascular%20endothelial%20cells%20reveals%20progressive%20organ-specific%20vulnerabilities%20during%20obesity&amp;journal=Nat.%20Metab.&amp;volume=4&amp;pages=1591-1610&amp;publication_year=2022&amp;author=Bondareva%2CO">
                    Google Scholar</a> 
                </p></li><li data-counter="20."><p id="ref-CR20">Kowluru, R. A. &amp; Mohammad, G. Epigenetics and mitochondrial stability in the metabolic memory phenomenon associated with continued progression of diabetic retinopathy. <i>Sci. Rep.</i> <b>10</b>, 6655 (2020).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://g-w1.github.io/articles/cas-redirect/1:CAS:528:DC%2BB3cXnsFSltL0%3D" aria-label="CAS reference 20">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32313015" aria-label="PubMed reference 20">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7171070" aria-label="PubMed Central reference 20">PubMed Central</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="ads reference" data-track-action="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2020NatSR..10.6655K" aria-label="ADS reference 20">ADS</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 20" href="http://scholar.google.com/scholar_lookup?&amp;title=Epigenetics%20and%20mitochondrial%20stability%20in%20the%20metabolic%20memory%20phenomenon%20associated%20with%20continued%20progression%20of%20diabetic%20retinopathy&amp;journal=Sci.%20Rep.&amp;volume=10&amp;publication_year=2020&amp;author=Kowluru%2CRA&amp;author=Mohammad%2CG">
                    Google Scholar</a> 
                </p></li><li data-counter="21."><p id="ref-CR21">Tallapragada, D. S. P., Karpe, P. A. &amp; Tikoo, K. Long-lasting partnership between insulin resistance and endothelial dysfunction: role of metabolic memory: high fat diet and metabolic memory. <i>Br. J. Pharmacol.</i> <b>172</b>, 4012–4023 (2015).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://g-w1.github.io/articles/cas-redirect/1:CAS:528:DC%2BC2MXhtFehtrzM" aria-label="CAS reference 21">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25825057" aria-label="PubMed reference 21">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4543609" aria-label="PubMed Central reference 21">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 21" href="http://scholar.google.com/scholar_lookup?&amp;title=Long-lasting%20partnership%20between%20insulin%20resistance%20and%20endothelial%20dysfunction%3A%20role%20of%20metabolic%20memory%3A%20high%20fat%20diet%20and%20metabolic%20memory&amp;journal=Br.%20J.%20Pharmacol.&amp;volume=172&amp;pages=4012-4023&amp;publication_year=2015&amp;author=Tallapragada%2CDSP&amp;author=Karpe%2CPA&amp;author=Tikoo%2CK">
                    Google Scholar</a> 
                </p></li><li data-counter="22."><p id="ref-CR22">El-Osta, A. et al. Transient high glucose causes persistent epigenetic changes and altered gene expression during subsequent normoglycemia. <i>J. Exp. Med.</i> <b>205</b>, 2409–2417 (2008).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://g-w1.github.io/articles/cas-redirect/1:CAS:528:DC%2BD1cXht1SrtLnN" aria-label="CAS reference 22">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18809715" aria-label="PubMed reference 22">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2556800" aria-label="PubMed Central reference 22">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 22" href="http://scholar.google.com/scholar_lookup?&amp;title=Transient%20high%20glucose%20causes%20persistent%20epigenetic%20changes%20and%20altered%20gene%20expression%20during%20subsequent%20normoglycemia&amp;journal=J.%20Exp.%20Med.&amp;volume=205&amp;pages=2409-2417&amp;publication_year=2008&amp;author=El-Osta%2CA">
                    Google Scholar</a> 
                </p></li><li data-counter="23."><p id="ref-CR23">Madsen, J. G. S. et al. Highly interconnected enhancer communities control lineage-determining genes in human mesenchymal stem cells. <i>Nat. Genet.</i> <b>52</b>, 1227–1238 (2020).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://g-w1.github.io/articles/cas-redirect/1:CAS:528:DC%2BB3cXhvF2jtLbK" aria-label="CAS reference 23">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33020665" aria-label="PubMed reference 23">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 23" href="http://scholar.google.com/scholar_lookup?&amp;title=Highly%20interconnected%20enhancer%20communities%20control%20lineage-determining%20genes%20in%20human%20mesenchymal%20stem%20cells&amp;journal=Nat.%20Genet.&amp;volume=52&amp;pages=1227-1238&amp;publication_year=2020&amp;author=Madsen%2CJGS">
                    Google Scholar</a> 
                </p></li><li data-counter="24."><p id="ref-CR24">Matsumura, Y. et al. H3K4/H3K9me3 bivalent chromatin domains targeted by lineage-specific DNA methylation pauses adipocyte differentiation. <i>Mol. Cell</i> <b>60</b>, 584–596 (2015).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://g-w1.github.io/articles/cas-redirect/1:CAS:528:DC%2BC2MXhvVyqtbrP" aria-label="CAS reference 24">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26590716" aria-label="PubMed reference 24">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 24" href="http://scholar.google.com/scholar_lookup?&amp;title=H3K4%2FH3K9me3%20bivalent%20chromatin%20domains%20targeted%20by%20lineage-specific%20DNA%20methylation%20pauses%20adipocyte%20differentiation&amp;journal=Mol.%20Cell&amp;volume=60&amp;pages=584-596&amp;publication_year=2015&amp;author=Matsumura%2CY">
                    Google Scholar</a> 
                </p></li><li data-counter="25."><p id="ref-CR25">Roh, H. C. et al. Adipocytes fail to maintain cellular identity during obesity due to reduced PPARγ activity and elevated TGFβ-SMAD signaling. <i>Mol. Metab.</i> <b>42</b>, 101086 (2020).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://g-w1.github.io/articles/cas-redirect/1:CAS:528:DC%2BB3cXitVKgsrzM" aria-label="CAS reference 25">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32992037" aria-label="PubMed reference 25">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7559520" aria-label="PubMed Central reference 25">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 25" href="http://scholar.google.com/scholar_lookup?&amp;title=Adipocytes%20fail%20to%20maintain%20cellular%20identity%20during%20obesity%20due%20to%20reduced%20PPAR%CE%B3%20activity%20and%20elevated%20TGF%CE%B2-SMAD%20signaling&amp;journal=Mol.%20Metab.&amp;volume=42&amp;publication_year=2020&amp;author=Roh%2CHC">
                    Google Scholar</a> 
                </p></li><li data-counter="26."><p id="ref-CR26">Roh, H. C. et al. Warming induces significant reprogramming of beige, but not brown, adipocyte cellular identity. <i>Cell Metab.</i> <b>27</b>, 1121–1137.e5 (2018).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://g-w1.github.io/articles/cas-redirect/1:CAS:528:DC%2BC1cXnsFCks7k%3D" aria-label="CAS reference 26">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29657031" aria-label="PubMed reference 26">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5932137" aria-label="PubMed Central reference 26">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 26" href="http://scholar.google.com/scholar_lookup?&amp;title=Warming%20induces%20significant%20reprogramming%20of%20beige%2C%20but%20not%20brown%2C%20adipocyte%20cellular%20identity&amp;journal=Cell%20Metab.&amp;volume=27&amp;pages=1121-1137.e5&amp;publication_year=2018&amp;author=Roh%2CHC">
                    Google Scholar</a> 
                </p></li><li data-counter="27."><p id="ref-CR27">Siersbæk, R. et al. Transcription factor cooperativity in early adipogenic hotspots and super-enhancers. <i>Cell Rep.</i> <b>7</b>, 1443–1455 (2014).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24857652" aria-label="PubMed reference 27">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 27" href="http://scholar.google.com/scholar_lookup?&amp;title=Transcription%20factor%20cooperativity%20in%20early%20adipogenic%20hotspots%20and%20super-enhancers&amp;journal=Cell%20Rep.&amp;volume=7&amp;pages=1443-1455&amp;publication_year=2014&amp;author=Siersb%C3%A6k%2CR">
                    Google Scholar</a> 
                </p></li><li data-counter="28."><p id="ref-CR28">Hata, M. et al. Past history of obesity triggers persistent epigenetic changes in innate immunity and exacerbates neuroinflammation. <i>Science</i> <b>379</b>, 45–62 (2023).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://g-w1.github.io/articles/cas-redirect/1:CAS:528:DC%2BB3sXmtlejsw%3D%3D" aria-label="CAS reference 28">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=36603072" aria-label="PubMed reference 28">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="ads reference" data-track-action="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2023Sci...379...45H" aria-label="ADS reference 28">ADS</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 28" href="http://scholar.google.com/scholar_lookup?&amp;title=Past%20history%20of%20obesity%20triggers%20persistent%20epigenetic%20changes%20in%20innate%20immunity%20and%20exacerbates%20neuroinflammation&amp;journal=Science&amp;volume=379&amp;pages=45-62&amp;publication_year=2023&amp;author=Hata%2CM">
                    Google Scholar</a> 
                </p></li><li data-counter="29."><p id="ref-CR29">Rosenwald, M., Perdikari, A., Rülicke, T. &amp; Wolfrum, C. Bi-directional interconversion of brite and white adipocytes. <i>Nat. Cell Biol.</i> <b>15</b>, 659–667 (2013).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://g-w1.github.io/articles/cas-redirect/1:CAS:528:DC%2BC3sXms1Wmu7k%3D" aria-label="CAS reference 29">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23624403" aria-label="PubMed reference 29">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 29" href="http://scholar.google.com/scholar_lookup?&amp;title=Bi-directional%20interconversion%20of%20brite%20and%20white%20adipocytes&amp;journal=Nat.%20Cell%20Biol.&amp;volume=15&amp;pages=659-667&amp;publication_year=2013&amp;author=Rosenwald%2CM&amp;author=Perdikari%2CA&amp;author=R%C3%BClicke%2CT&amp;author=Wolfrum%2CC">
                    Google Scholar</a> 
                </p></li><li data-counter="30."><p id="ref-CR30">Barres, R. et al. Weight loss after gastric bypass surgery in human obesity remodels promoter methylation. <i>Cell Rep.</i> <b>3</b>, 1020–1027 (2013).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://g-w1.github.io/articles/cas-redirect/1:CAS:528:DC%2BC3sXlvVCgt7g%3D" aria-label="CAS reference 30">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23583180" aria-label="PubMed reference 30">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 30" href="http://scholar.google.com/scholar_lookup?&amp;title=Weight%20loss%20after%20gastric%20bypass%20surgery%20in%20human%20obesity%20remodels%20promoter%20methylation&amp;journal=Cell%20Rep.&amp;volume=3&amp;pages=1020-1027&amp;publication_year=2013&amp;author=Barres%2CR">
                    Google Scholar</a> 
                </p></li><li data-counter="31."><p id="ref-CR31">Benton, M. C. et al. An analysis of DNA methylation in human adipose tissue reveals differential modification of obesity genes before and after gastric bypass and weight loss. <i>Genome Biol.</i> <b>16</b>, 8 (2015).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://g-w1.github.io/articles/cas-redirect/1:CAS:528:DC%2BC2MXjtlKns78%3D" aria-label="CAS reference 31">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25651499" aria-label="PubMed reference 31">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4301800" aria-label="PubMed Central reference 31">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 31" href="http://scholar.google.com/scholar_lookup?&amp;title=An%20analysis%20of%20DNA%20methylation%20in%20human%20adipose%20tissue%20reveals%20differential%20modification%20of%20obesity%20genes%20before%20and%20after%20gastric%20bypass%20and%20weight%20loss&amp;journal=Genome%20Biol.&amp;volume=16&amp;publication_year=2015&amp;author=Benton%2CMC">
                    Google Scholar</a> 
                </p></li><li data-counter="32."><p id="ref-CR32">Talukdar, F. R. et al. Bariatric surgery-induced weight loss and associated genome-wide DNA-methylation alterations in obese individuals. <i>Clin. Epigenetics</i> <b>14</b>, 176 (2022).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://g-w1.github.io/articles/cas-redirect/1:CAS:528:DC%2BB38XjtF2ktLfO" aria-label="CAS reference 32">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=36528638" aria-label="PubMed reference 32">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9759858" aria-label="PubMed Central reference 32">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 32" href="http://scholar.google.com/scholar_lookup?&amp;title=Bariatric%20surgery-induced%20weight%20loss%20and%20associated%20genome-wide%20DNA-methylation%20alterations%20in%20obese%20individuals&amp;journal=Clin.%20Epigenetics&amp;volume=14&amp;publication_year=2022&amp;author=Talukdar%2CFR">
                    Google Scholar</a> 
                </p></li><li data-counter="33."><p id="ref-CR33">Chen, Z. et al. Epigenomic profiling reveals an association between persistence of DNA methylation and metabolic memory in the DCCT/EDIC type 1 diabetes cohort. <i>Proc. Natl Acad. Sci. USA</i> <b>113</b>, E3002–E3011 (2016).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://g-w1.github.io/articles/cas-redirect/1:CAS:528:DC%2BC28XnsV2iu7Y%3D" aria-label="CAS reference 33">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27162351" aria-label="PubMed reference 33">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4890596" aria-label="PubMed Central reference 33">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 33" href="http://scholar.google.com/scholar_lookup?&amp;title=Epigenomic%20profiling%20reveals%20an%20association%20between%20persistence%20of%20DNA%20methylation%20and%20metabolic%20memory%20in%20the%20DCCT%2FEDIC%20type%201%20diabetes%20cohort&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=113&amp;pages=E3002-E3011&amp;publication_year=2016&amp;author=Chen%2CZ">
                    Google Scholar</a> 
                </p></li><li data-counter="34."><p id="ref-CR34">Emont, M. P. et al. A single-cell atlas of human and mouse white adipose tissue. <i>Nature</i> <b>603</b>, 926–933 (2022).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://g-w1.github.io/articles/cas-redirect/1:CAS:528:DC%2BB38XntlGgtbw%3D" aria-label="CAS reference 34">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=35296864" aria-label="PubMed reference 34">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9504827" aria-label="PubMed Central reference 34">PubMed Central</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="ads reference" data-track-action="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2022Natur.603..926E" aria-label="ADS reference 34">ADS</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 34" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20single-cell%20atlas%20of%20human%20and%20mouse%20white%20adipose%20tissue&amp;journal=Nature&amp;volume=603&amp;pages=926-933&amp;publication_year=2022&amp;author=Emont%2CMP">
                    Google Scholar</a> 
                </p></li><li data-counter="35."><p id="ref-CR35">Massier, L. et al. An integrated single cell and spatial transcriptomic map of human white adipose tissue. <i>Nat. Commun.</i> <b>14</b>, 1438 (2023).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://g-w1.github.io/articles/cas-redirect/1:CAS:528:DC%2BB3sXls1Oqsr8%3D" aria-label="CAS reference 35">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=36922516" aria-label="PubMed reference 35">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10017705" aria-label="PubMed Central reference 35">PubMed Central</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="ads reference" data-track-action="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2023NatCo..14.1438M" aria-label="ADS reference 35">ADS</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 35" href="http://scholar.google.com/scholar_lookup?&amp;title=An%20integrated%20single%20cell%20and%20spatial%20transcriptomic%20map%20of%20human%20white%20adipose%20tissue&amp;journal=Nat.%20Commun.&amp;volume=14&amp;publication_year=2023&amp;author=Massier%2CL">
                    Google Scholar</a> 
                </p></li><li data-counter="36."><p id="ref-CR36">Maniyadath, B., Zhang, Q., Gupta, R. K. &amp; Mandrup, S. Adipose tissue at single-cell resolution. <i>Cell Metab.</i> <b>35</b>, 386–413 (2023).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://g-w1.github.io/articles/cas-redirect/1:CAS:528:DC%2BB3sXkslyntb0%3D" aria-label="CAS reference 36">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=36889280" aria-label="PubMed reference 36">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10027403" aria-label="PubMed Central reference 36">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 36" href="http://scholar.google.com/scholar_lookup?&amp;title=Adipose%20tissue%20at%20single-cell%20resolution&amp;journal=Cell%20Metab.&amp;volume=35&amp;pages=386-413&amp;publication_year=2023&amp;author=Maniyadath%2CB&amp;author=Zhang%2CQ&amp;author=Gupta%2CRK&amp;author=Mandrup%2CS">
                    Google Scholar</a> 
                </p></li><li data-counter="37."><p id="ref-CR37">Merrick, D. et al. Identification of a mesenchymal progenitor cell hierarchy in adipose tissue. <i>Science</i> <b>364</b>, eaav2501 (2019).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://g-w1.github.io/articles/cas-redirect/1:CAS:528:DC%2BC1MXnvFOrurs%3D" aria-label="CAS reference 37">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31023895" aria-label="PubMed reference 37">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6816238" aria-label="PubMed Central reference 37">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 37" href="http://scholar.google.com/scholar_lookup?&amp;title=Identification%20of%20a%20mesenchymal%20progenitor%20cell%20hierarchy%20in%20adipose%20tissue&amp;journal=Science&amp;volume=364&amp;publication_year=2019&amp;author=Merrick%2CD">
                    Google Scholar</a> 
                </p></li><li data-counter="38."><p id="ref-CR38">Boucher, J. et al. Impaired thermogenesis and adipose tissue development in mice with fat-specific disruption of insulin and IGF-1 signalling. <i>Nat. Commun.</i> <b>3</b>, 902 (2012).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22692545" aria-label="PubMed reference 38">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="ads reference" data-track-action="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2012NatCo...3..902B" aria-label="ADS reference 38">ADS</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 38" href="http://scholar.google.com/scholar_lookup?&amp;title=Impaired%20thermogenesis%20and%20adipose%20tissue%20development%20in%20mice%20with%20fat-specific%20disruption%20of%20insulin%20and%20IGF-1%20signalling&amp;journal=Nat.%20Commun.&amp;volume=3&amp;publication_year=2012&amp;author=Boucher%2CJ">
                    Google Scholar</a> 
                </p></li><li data-counter="39."><p id="ref-CR39">Csaki, L. S. et al. Lipin-1 and lipin-3 together determine adiposity in vivo. <i>Mol. Metab.</i> <b>3</b>, 145–154 (2014).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://g-w1.github.io/articles/cas-redirect/1:CAS:528:DC%2BC3sXitVWnu7%2FE" aria-label="CAS reference 39">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24634820" aria-label="PubMed reference 39">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 39" href="http://scholar.google.com/scholar_lookup?&amp;title=Lipin-1%20and%20lipin-3%20together%20determine%20adiposity%20in%20vivo&amp;journal=Mol.%20Metab.&amp;volume=3&amp;pages=145-154&amp;publication_year=2014&amp;author=Csaki%2CLS">
                    Google Scholar</a> 
                </p></li><li data-counter="40."><p id="ref-CR40">Kang, H. S. et al. IDH1-dependent α-KG regulates brown fat differentiation and function by modulating histone methylation. <i>Metabolism</i> <b>105</b>, 154173 (2020).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://g-w1.github.io/articles/cas-redirect/1:CAS:528:DC%2BB3cXjtFequ74%3D" aria-label="CAS reference 40">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32035087" aria-label="PubMed reference 40">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 40" href="http://scholar.google.com/scholar_lookup?&amp;title=IDH1-dependent%20%CE%B1-KG%20regulates%20brown%20fat%20differentiation%20and%20function%20by%20modulating%20histone%20methylation&amp;journal=Metabolism&amp;volume=105&amp;publication_year=2020&amp;author=Kang%2CHS">
                    Google Scholar</a> 
                </p></li><li data-counter="41."><p id="ref-CR41">Seo, D. H. et al. Effects of a phosphodiesterase inhibitor on the browning of adipose tissue in mice. <i>Biomedicines</i> <b>10</b>, 1852 (2022).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://g-w1.github.io/articles/cas-redirect/1:CAS:528:DC%2BB38Xit1Knt7jI" aria-label="CAS reference 41">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=36009398" aria-label="PubMed reference 41">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9405663" aria-label="PubMed Central reference 41">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 41" href="http://scholar.google.com/scholar_lookup?&amp;title=Effects%20of%20a%20phosphodiesterase%20inhibitor%20on%20the%20browning%20of%20adipose%20tissue%20in%20mice&amp;journal=Biomedicines&amp;volume=10&amp;publication_year=2022&amp;author=Seo%2CDH">
                    Google Scholar</a> 
                </p></li><li data-counter="42."><p id="ref-CR42">Petrus, P. et al. Glutamine links obesity to inflammation in human white adipose tissue. <i>Cell Metab.</i> <b>31</b>, 375–390.e11 (2020).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://g-w1.github.io/articles/cas-redirect/1:CAS:528:DC%2BC1MXisVKrsrfO" aria-label="CAS reference 42">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31866443" aria-label="PubMed reference 42">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 42" href="http://scholar.google.com/scholar_lookup?&amp;title=Glutamine%20links%20obesity%20to%20inflammation%20in%20human%20white%20adipose%20tissue&amp;journal=Cell%20Metab.&amp;volume=31&amp;pages=375-390.e11&amp;publication_year=2020&amp;author=Petrus%2CP">
                    Google Scholar</a> 
                </p></li><li data-counter="43."><p id="ref-CR43">Ferguson, B. S., Nam, H., Stephens, J. M. &amp; Morrison, R. F. Mitogen-dependent regulation of DUSP1 governs ERK and p38 signaling during early 3T3-L1 adipocyte differentiation. <i>J. Cell. Physiol.</i> <b>231</b>, 1562–1574 (2016).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://g-w1.github.io/articles/cas-redirect/1:CAS:528:DC%2BC2MXhvFWqt7bI" aria-label="CAS reference 43">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26566083" aria-label="PubMed reference 43">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 43" href="http://scholar.google.com/scholar_lookup?&amp;title=Mitogen-dependent%20regulation%20of%20DUSP1%20governs%20ERK%20and%20p38%20signaling%20during%20early%203T3-L1%20adipocyte%20differentiation&amp;journal=J.%20Cell.%20Physiol.&amp;volume=231&amp;pages=1562-1574&amp;publication_year=2016&amp;author=Ferguson%2CBS&amp;author=Nam%2CH&amp;author=Stephens%2CJM&amp;author=Morrison%2CRF">
                    Google Scholar</a> 
                </p></li><li data-counter="44."><p id="ref-CR44">Hauffe, R. et al. GPx3 dysregulation impacts adipose tissue insulin receptor expression and sensitivity. <i>JCI Insight</i> <b>5</b>, e136283 (2020).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32369454" aria-label="PubMed reference 44">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7308064" aria-label="PubMed Central reference 44">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 44" href="http://scholar.google.com/scholar_lookup?&amp;title=GPx3%20dysregulation%20impacts%20adipose%20tissue%20insulin%20receptor%20expression%20and%20sensitivity&amp;journal=JCI%20Insight&amp;volume=5&amp;publication_year=2020&amp;author=Hauffe%2CR">
                    Google Scholar</a> 
                </p></li><li data-counter="45."><p id="ref-CR45">Strissel, K. J. et al. Adipocyte death, adipose tissue remodeling, and obesity complications. <i>Diabetes</i> <b>56</b>, 2910–2918 (2007).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://g-w1.github.io/articles/cas-redirect/1:CAS:528:DC%2BD2sXhsValurrP" aria-label="CAS reference 45">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17848624" aria-label="PubMed reference 45">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 45" href="http://scholar.google.com/scholar_lookup?&amp;title=Adipocyte%20death%2C%20adipose%20tissue%20remodeling%2C%20and%20obesity%20complications&amp;journal=Diabetes&amp;volume=56&amp;pages=2910-2918&amp;publication_year=2007&amp;author=Strissel%2CKJ">
                    Google Scholar</a> 
                </p></li><li data-counter="46."><p id="ref-CR46">Sárvári, A. K. et al. Plasticity of epididymal adipose tissue in response to diet-induced obesity at single-nucleus resolution. <i>Cell Metab.</i> <b>33</b>, 437–453.e5 (2021).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33378646" aria-label="PubMed reference 46">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 46" href="http://scholar.google.com/scholar_lookup?&amp;title=Plasticity%20of%20epididymal%20adipose%20tissue%20in%20response%20to%20diet-induced%20obesity%20at%20single-nucleus%20resolution&amp;journal=Cell%20Metab.&amp;volume=33&amp;pages=437-453.e5&amp;publication_year=2021&amp;author=S%C3%A1rv%C3%A1ri%2CAK">
                    Google Scholar</a> 
                </p></li><li data-counter="47."><p id="ref-CR47">Rosen, E. D. &amp; Spiegelman, B. M. What we talk about when we talk about fat. <i>Cell</i> <b>156</b>, 20–44 (2014).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://g-w1.github.io/articles/cas-redirect/1:CAS:528:DC%2BC2cXhtF2ht78%3D" aria-label="CAS reference 47">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24439368" aria-label="PubMed reference 47">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3934003" aria-label="PubMed Central reference 47">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 47" href="http://scholar.google.com/scholar_lookup?&amp;title=What%20we%20talk%20about%20when%20we%20talk%20about%20fat&amp;journal=Cell&amp;volume=156&amp;pages=20-44&amp;publication_year=2014&amp;author=Rosen%2CED&amp;author=Spiegelman%2CBM">
                    Google Scholar</a> 
                </p></li><li data-counter="48."><p id="ref-CR48">Bannister, A. J. &amp; Kouzarides, T. Regulation of chromatin by histone modifications. <i>Cell Res.</i> <b>21</b>, 381–395 (2011).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://g-w1.github.io/articles/cas-redirect/1:CAS:528:DC%2BC3MXivVCgurc%3D" aria-label="CAS reference 48">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21321607" aria-label="PubMed reference 48">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3193420" aria-label="PubMed Central reference 48">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 48" href="http://scholar.google.com/scholar_lookup?&amp;title=Regulation%20of%20chromatin%20by%20histone%20modifications&amp;journal=Cell%20Res.&amp;volume=21&amp;pages=381-395&amp;publication_year=2011&amp;author=Bannister%2CAJ&amp;author=Kouzarides%2CT">
                    Google Scholar</a> 
                </p></li><li data-counter="49."><p id="ref-CR49">Lawrence, M., Daujat, S. &amp; Schneider, R. Lateral thinking: how histone modifications regulate gene expression. <i>Trends Genet.</i> <b>32</b>, 42–56 (2016).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://g-w1.github.io/articles/cas-redirect/1:CAS:528:DC%2BC2MXhvFansbnE" aria-label="CAS reference 49">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26704082" aria-label="PubMed reference 49">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 49" href="http://scholar.google.com/scholar_lookup?&amp;title=Lateral%20thinking%3A%20how%20histone%20modifications%20regulate%20gene%20expression&amp;journal=Trends%20Genet.&amp;volume=32&amp;pages=42-56&amp;publication_year=2016&amp;author=Lawrence%2CM&amp;author=Daujat%2CS&amp;author=Schneider%2CR">
                    Google Scholar</a> 
                </p></li><li data-counter="50."><p id="ref-CR50">Argelaguet, R. et al. Multi-omics factor analysis—a framework for unsupervised integration of multi-omics data sets. <i>Mol. Syst. Biol.</i> <b>14</b>, e8124 (2018).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29925568" aria-label="PubMed reference 50">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6010767" aria-label="PubMed Central reference 50">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 50" href="http://scholar.google.com/scholar_lookup?&amp;title=Multi-omics%20factor%20analysis%E2%80%94a%20framework%20for%20unsupervised%20integration%20of%20multi-omics%20data%20sets&amp;journal=Mol.%20Syst.%20Biol.&amp;volume=14&amp;publication_year=2018&amp;author=Argelaguet%2CR">
                    Google Scholar</a> 
                </p></li><li data-counter="51."><p id="ref-CR51">Choi, J. et al. Evidence for additive and synergistic action of mammalian enhancers during cell fate determination. <i>eLife</i> <b>10</b>, e65381 (2021).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://g-w1.github.io/articles/cas-redirect/1:CAS:528:DC%2BB3MXitFeitLnJ" aria-label="CAS reference 51">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33770473" aria-label="PubMed reference 51">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8004103" aria-label="PubMed Central reference 51">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 51" href="http://scholar.google.com/scholar_lookup?&amp;title=Evidence%20for%20additive%20and%20synergistic%20action%20of%20mammalian%20enhancers%20during%20cell%20fate%20determination&amp;journal=eLife&amp;volume=10&amp;publication_year=2021&amp;author=Choi%2CJ">
                    Google Scholar</a> 
                </p></li><li data-counter="52."><p id="ref-CR52">Hnisz, D. et al. Super-enhancers in the control of cell identity and disease. <i>Cell</i> <b>155</b>, 934–947 (2013).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://g-w1.github.io/articles/cas-redirect/1:CAS:528:DC%2BC3sXhs1SrtLbL" aria-label="CAS reference 52">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24119843" aria-label="PubMed reference 52">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 52" href="http://scholar.google.com/scholar_lookup?&amp;title=Super-enhancers%20in%20the%20control%20of%20cell%20identity%20and%20disease&amp;journal=Cell&amp;volume=155&amp;pages=934-947&amp;publication_year=2013&amp;author=Hnisz%2CD">
                    Google Scholar</a> 
                </p></li><li data-counter="53."><p id="ref-CR53">Tolvanen, L., Christenson, A., Surkan, P. J. &amp; Lagerros, Y. T. Patients’ experiences of weight regain after bariatric surgery. <i>Obes. Surg.</i> <b>32</b>, 1498–1507 (2022).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=35061154" aria-label="PubMed reference 53">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8986695" aria-label="PubMed Central reference 53">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 53" href="http://scholar.google.com/scholar_lookup?&amp;title=Patients%E2%80%99%20experiences%20of%20weight%20regain%20after%20bariatric%20surgery&amp;journal=Obes.%20Surg.&amp;volume=32&amp;pages=1498-1507&amp;publication_year=2022&amp;author=Tolvanen%2CL&amp;author=Christenson%2CA&amp;author=Surkan%2CPJ&amp;author=Lagerros%2CYT">
                    Google Scholar</a> 
                </p></li><li data-counter="54."><p id="ref-CR54">Wang, W. et al. Role of bile acids in bariatric surgery. <i>Front. Physiol.</i> <b>10</b>, 374 (2019).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31001146" aria-label="PubMed reference 54">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6454391" aria-label="PubMed Central reference 54">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 54" href="http://scholar.google.com/scholar_lookup?&amp;title=Role%20of%20bile%20acids%20in%20bariatric%20surgery&amp;journal=Front.%20Physiol.&amp;volume=10&amp;publication_year=2019&amp;author=Wang%2CW">
                    Google Scholar</a> 
                </p></li><li data-counter="55."><p id="ref-CR55">Ramos-Molina, B. et al. Differential effects of restrictive and malabsorptive bariatric surgery procedures on the serum lipidome in obese subjects. <i>J. Clin. Lipidol.</i> <b>12</b>, 1502–1512 (2018).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30143432" aria-label="PubMed reference 55">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 55" href="http://scholar.google.com/scholar_lookup?&amp;title=Differential%20effects%20of%20restrictive%20and%20malabsorptive%20bariatric%20surgery%20procedures%20on%20the%20serum%20lipidome%20in%20obese%20subjects&amp;journal=J.%20Clin.%20Lipidol.&amp;volume=12&amp;pages=1502-1512&amp;publication_year=2018&amp;author=Ramos-Molina%2CB">
                    Google Scholar</a> 
                </p></li><li data-counter="56."><p id="ref-CR56">Inge, T. H. et al. Weight loss and health status 3 years after bariatric surgery in adolescents. <i>N. Engl. J. Med.</i> <b>374</b>, 113–123 (2016).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://g-w1.github.io/articles/cas-redirect/1:CAS:528:DC%2BC28XhtV2mur%2FM" aria-label="CAS reference 56">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26544725" aria-label="PubMed reference 56">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 56" href="http://scholar.google.com/scholar_lookup?&amp;title=Weight%20loss%20and%20health%20status%203%20years%20after%20bariatric%20surgery%20in%20adolescents&amp;journal=N.%20Engl.%20J.%20Med.&amp;volume=374&amp;pages=113-123&amp;publication_year=2016&amp;author=Inge%2CTH">
                    Google Scholar</a> 
                </p></li><li data-counter="57."><p id="ref-CR57">Ulker, İ. &amp; Yildiran, H. The effects of bariatric surgery on gut microbiota in patients with obesity: a review of the literature. <i>Biosci. Microbiota Food Health</i> <b>38</b>, 3–9 (2019).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://g-w1.github.io/articles/cas-redirect/1:CAS:528:DC%2BC1MXit1OlsrnP" aria-label="CAS reference 57">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30705797" aria-label="PubMed reference 57">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 57" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20effects%20of%20bariatric%20surgery%20on%20gut%20microbiota%20in%20patients%20with%20obesity%3A%20a%20review%20of%20the%20literature&amp;journal=Biosci.%20Microbiota%20Food%20Health&amp;volume=38&amp;pages=3-9&amp;publication_year=2019&amp;author=Ulker%2C%C4%B0&amp;author=Yildiran%2CH">
                    Google Scholar</a> 
                </p></li><li data-counter="58."><p id="ref-CR58">Holst, J. J. GLP-1 physiology in obesity and development of incretin-based drugs for chronic weight management. <i>Nat. Metab</i>. <a href="https://doi.org/10.1038/s42255-024-01113-9" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1038/s42255-024-01113-9">https://doi.org/10.1038/s42255-024-01113-9</a> (2024).</p></li><li data-counter="59."><p id="ref-CR59">Aronne, L. J. et al. Continued treatment with tirzepatide for maintenance of weight reduction in adults with obesity: the SURMOUNT-4 randomized clinical trial. <i>JAMA</i> <b>331</b>, 38–48 (2024).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://g-w1.github.io/articles/cas-redirect/1:CAS:528:DC%2BB2cXpvF2itg%3D%3D" aria-label="CAS reference 59">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=38078870" aria-label="PubMed reference 59">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 59" href="http://scholar.google.com/scholar_lookup?&amp;title=Continued%20treatment%20with%20tirzepatide%20for%20maintenance%20of%20weight%20reduction%20in%20adults%20with%20obesity%3A%20the%20SURMOUNT-4%20randomized%20clinical%20trial&amp;journal=JAMA&amp;volume=331&amp;pages=38-48&amp;publication_year=2024&amp;author=Aronne%2CLJ">
                    Google Scholar</a> 
                </p></li><li data-counter="60."><p id="ref-CR60">Cappelluti, M. A. et al. Durable and efficient gene silencing in vivo by hit-and-run epigenome editing. <i>Nature</i> <b>627</b>, 416–423 (2024).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://g-w1.github.io/articles/cas-redirect/1:CAS:528:DC%2BB2cXkslCltrY%3D" aria-label="CAS reference 60">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=38418872" aria-label="PubMed reference 60">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10937395" aria-label="PubMed Central reference 60">PubMed Central</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="ads reference" data-track-action="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2024Natur.627..416C" aria-label="ADS reference 60">ADS</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 60" href="http://scholar.google.com/scholar_lookup?&amp;title=Durable%20and%20efficient%20gene%20silencing%20in%20vivo%20by%20hit-and-run%20epigenome%20editing&amp;journal=Nature&amp;volume=627&amp;pages=416-423&amp;publication_year=2024&amp;author=Cappelluti%2CMA">
                    Google Scholar</a> 
                </p></li><li data-counter="61."><p id="ref-CR61">Browder, K. C. et al. In vivo partial reprogramming alters age-associated molecular changes during physiological aging in mice. <i>Nat. Aging</i> <b>2</b>, 243–253 (2022).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://g-w1.github.io/articles/cas-redirect/1:CAS:528:DC%2BB3sXmtFSksb0%3D" aria-label="CAS reference 61">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=37118377" aria-label="PubMed reference 61">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 61" href="http://scholar.google.com/scholar_lookup?&amp;title=In%20vivo%20partial%20reprogramming%20alters%20age-associated%20molecular%20changes%20during%20physiological%20aging%20in%20mice&amp;journal=Nat.%20Aging&amp;volume=2&amp;pages=243-253&amp;publication_year=2022&amp;author=Browder%2CKC">
                    Google Scholar</a> 
                </p></li><li data-counter="62."><p id="ref-CR62">Ocampo, A. et al. In vivo amelioration of age-associated hallmarks by partial reprogramming. <i>Cell</i> <b>167</b>, 1719–1733.e12 (2016).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://g-w1.github.io/articles/cas-redirect/1:CAS:528:DC%2BC28XitFWlsrrP" aria-label="CAS reference 62">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27984723" aria-label="PubMed reference 62">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5679279" aria-label="PubMed Central reference 62">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 62" href="http://scholar.google.com/scholar_lookup?&amp;title=In%20vivo%20amelioration%20of%20age-associated%20hallmarks%20by%20partial%20reprogramming&amp;journal=Cell&amp;volume=167&amp;pages=1719-1733.e12&amp;publication_year=2016&amp;author=Ocampo%2CA">
                    Google Scholar</a> 
                </p></li><li data-counter="63."><p id="ref-CR63">Langhardt, J. et al. Effects of weight loss on glutathione peroxidase 3 serum concentrations and adipose tissue expression in human obesity. <i>Obes. Facts</i> <b>11</b>, 475–490 (2018).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://g-w1.github.io/articles/cas-redirect/1:CAS:528:DC%2BC1MXjtlGksQ%3D%3D" aria-label="CAS reference 63">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30537708" aria-label="PubMed reference 63">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6341324" aria-label="PubMed Central reference 63">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 63" href="http://scholar.google.com/scholar_lookup?&amp;title=Effects%20of%20weight%20loss%20on%20glutathione%20peroxidase%203%20serum%20concentrations%20and%20adipose%20tissue%20expression%20in%20human%20obesity&amp;journal=Obes.%20Facts&amp;volume=11&amp;pages=475-490&amp;publication_year=2018&amp;author=Langhardt%2CJ">
                    Google Scholar</a> 
                </p></li><li data-counter="64."><p id="ref-CR64">Klöting, N. et al. Insulin-sensitive obesity. <i>Am. J. Physiol. Endocrinol. Metab.</i> <b>299</b>, E506–E515 (2010).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20570822" aria-label="PubMed reference 64">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 64" href="http://scholar.google.com/scholar_lookup?&amp;title=Insulin-sensitive%20obesity&amp;journal=Am.%20J.%20Physiol.%20Endocrinol.%20Metab.&amp;volume=299&amp;pages=E506-E515&amp;publication_year=2010&amp;author=Kl%C3%B6ting%2CN">
                    Google Scholar</a> 
                </p></li><li data-counter="65."><p id="ref-CR65">Rydén, M. et al. The adipose transcriptional response to insulin is determined by obesity, not insulin sensitivity. <i>Cell Rep.</i> <b>16</b>, 2317–2326 (2016).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27545890" aria-label="PubMed reference 65">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 65" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20adipose%20transcriptional%20response%20to%20insulin%20is%20determined%20by%20obesity%2C%20not%20insulin%20sensitivity&amp;journal=Cell%20Rep.&amp;volume=16&amp;pages=2317-2326&amp;publication_year=2016&amp;author=Ryd%C3%A9n%2CM">
                    Google Scholar</a> 
                </p></li><li data-counter="66."><p id="ref-CR66">Moser, C. et al. Quantification of adipocyte numbers following adipose tissue remodeling. <i>Cell Rep.</i> <b>35</b>, 109023 (2021).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://g-w1.github.io/articles/cas-redirect/1:CAS:528:DC%2BB3MXhtVShtrjE" aria-label="CAS reference 66">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33909996" aria-label="PubMed reference 66">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 66" href="http://scholar.google.com/scholar_lookup?&amp;title=Quantification%20of%20adipocyte%20numbers%20following%20adipose%20tissue%20remodeling&amp;journal=Cell%20Rep.&amp;volume=35&amp;publication_year=2021&amp;author=Moser%2CC">
                    Google Scholar</a> 
                </p></li><li data-counter="67."><p id="ref-CR67">Roh, H. C. et al. Simultaneous transcriptional and epigenomic profiling from specific cell types within heterogeneous tissues in vivo. <i>Cell Rep.</i> <b>18</b>, 1048–1061 (2017).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://g-w1.github.io/articles/cas-redirect/1:CAS:528:DC%2BC2sXhs1Gqt7o%3D" aria-label="CAS reference 67">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28122230" aria-label="PubMed reference 67">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5291126" aria-label="PubMed Central reference 67">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 67" href="http://scholar.google.com/scholar_lookup?&amp;title=Simultaneous%20transcriptional%20and%20epigenomic%20profiling%20from%20specific%20cell%20types%20within%20heterogeneous%20tissues%20in%20vivo&amp;journal=Cell%20Rep.&amp;volume=18&amp;pages=1048-1061&amp;publication_year=2017&amp;author=Roh%2CHC">
                    Google Scholar</a> 
                </p></li><li data-counter="68."><p id="ref-CR68">Harms, M. J. et al. Mature human white adipocytes cultured under membranes maintain identity, function, and can transdifferentiate into brown-like adipocytes. <i>Cell Rep.</i> <b>27</b>, 213–225.e5 (2019).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://g-w1.github.io/articles/cas-redirect/1:CAS:528:DC%2BC1MXms1Ogu7s%3D" aria-label="CAS reference 68">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30943403" aria-label="PubMed reference 68">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 68" href="http://scholar.google.com/scholar_lookup?&amp;title=Mature%20human%20white%20adipocytes%20cultured%20under%20membranes%20maintain%20identity%2C%20function%2C%20and%20can%20transdifferentiate%20into%20brown-like%20adipocytes&amp;journal=Cell%20Rep.&amp;volume=27&amp;pages=213-225.e5&amp;publication_year=2019&amp;author=Harms%2CMJ">
                    Google Scholar</a> 
                </p></li><li data-counter="69."><p id="ref-CR69">Galarraga, M. et al. Adiposoft: automated software for the analysis of white adipose tissue cellularity in histological sections. <i>J. Lipid Res.</i> <b>53</b>, 2791–2796 (2012).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://g-w1.github.io/articles/cas-redirect/1:CAS:528:DC%2BC38XhslOis7%2FO" aria-label="CAS reference 69">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22993232" aria-label="PubMed reference 69">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3494244" aria-label="PubMed Central reference 69">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 69" href="http://scholar.google.com/scholar_lookup?&amp;title=Adiposoft%3A%20automated%20software%20for%20the%20analysis%20of%20white%20adipose%20tissue%20cellularity%20in%20histological%20sections&amp;journal=J.%20Lipid%20Res.&amp;volume=53&amp;pages=2791-2796&amp;publication_year=2012&amp;author=Galarraga%2CM">
                    Google Scholar</a> 
                </p></li><li data-counter="70."><p id="ref-CR70">Schindelin, J. et al. Fiji: an open-source platform for biological-image analysis. <i>Nat. Methods</i> <b>9</b>, 676–682 (2012).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://g-w1.github.io/articles/cas-redirect/1:CAS:528:DC%2BC38XhtVKnurbJ" aria-label="CAS reference 70">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22743772" aria-label="PubMed reference 70">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 70" href="http://scholar.google.com/scholar_lookup?&amp;title=Fiji%3A%20an%20open-source%20platform%20for%20biological-image%20analysis&amp;journal=Nat.%20Methods&amp;volume=9&amp;pages=676-682&amp;publication_year=2012&amp;author=Schindelin%2CJ">
                    Google Scholar</a> 
                </p></li><li data-counter="71."><p id="ref-CR71">Kaya-Okur, H. S. et al. CUT&amp;Tag for efficient epigenomic profiling of small samples and single cells. <i>Nat. Commun.</i> <b>10</b>, 1–10 (2019).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://g-w1.github.io/articles/cas-redirect/1:CAS:528:DC%2BC1MXovVKrtrk%3D" aria-label="CAS reference 71">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="ads reference" data-track-action="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2019NatCo..10....1K" aria-label="ADS reference 71">ADS</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 71" href="http://scholar.google.com/scholar_lookup?&amp;title=CUT%26Tag%20for%20efficient%20epigenomic%20profiling%20of%20small%20samples%20and%20single%20cells&amp;journal=Nat.%20Commun.&amp;volume=10&amp;pages=1-10&amp;publication_year=2019&amp;author=Kaya-Okur%2CHS">
                    Google Scholar</a> 
                </p></li><li data-counter="72."><p id="ref-CR72">Buenrostro, J. D., Giresi, P. G., Zaba, L. C., Chang, H. Y. &amp; Greenleaf, W. J. Transposition of native chromatin for fast and sensitive epigenomic profiling of open chromatin, DNA-binding proteins and nucleosome position. <i>Nat. Methods</i> <b>10</b>, 1213–1218 (2013).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://g-w1.github.io/articles/cas-redirect/1:CAS:528:DC%2BC3sXhsFOiu7jO" aria-label="CAS reference 72">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24097267" aria-label="PubMed reference 72">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3959825" aria-label="PubMed Central reference 72">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 72" href="http://scholar.google.com/scholar_lookup?&amp;title=Transposition%20of%20native%20chromatin%20for%20fast%20and%20sensitive%20epigenomic%20profiling%20of%20open%20chromatin%2C%20DNA-binding%20proteins%20and%20nucleosome%20position&amp;journal=Nat.%20Methods&amp;volume=10&amp;pages=1213-1218&amp;publication_year=2013&amp;author=Buenrostro%2CJD&amp;author=Giresi%2CPG&amp;author=Zaba%2CLC&amp;author=Chang%2CHY&amp;author=Greenleaf%2CWJ">
                    Google Scholar</a> 
                </p></li><li data-counter="73."><p id="ref-CR73">Heiman, M., Kulicke, R., Fenster, R. J., Greengard, P. &amp; Heintz, N. Cell-type-specific mRNA purification by translating ribosome affinity purification (TRAP). <i>Nat. Protoc.</i> <b>9</b>, 1282–1291 (2014).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://g-w1.github.io/articles/cas-redirect/1:CAS:528:DC%2BC2cXns1Wqtr0%3D" aria-label="CAS reference 73">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24810037" aria-label="PubMed reference 73">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4102313" aria-label="PubMed Central reference 73">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 73" href="http://scholar.google.com/scholar_lookup?&amp;title=Cell-type-specific%20mRNA%20purification%20by%20translating%20ribosome%20affinity%20purification%20%28TRAP%29&amp;journal=Nat.%20Protoc.&amp;volume=9&amp;pages=1282-1291&amp;publication_year=2014&amp;author=Heiman%2CM&amp;author=Kulicke%2CR&amp;author=Fenster%2CRJ&amp;author=Greengard%2CP&amp;author=Heintz%2CN">
                    Google Scholar</a> 
                </p></li><li data-counter="74."><p id="ref-CR74">Henikoff, S., Janssens, D., Kaya-Okur, H. S., Henikoff, J. &amp; Ahmad, K. CUT&amp;Tag@home V1. <i>protools.io</i> <a href="https://www.protocols.io/view/cut-amp-tag-home-bd26i8he" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="https://www.protocols.io/view/cut-amp-tag-home-bd26i8he">https://www.protocols.io/view/cut-amp-tag-home-bd26i8he</a> (2020).</p></li><li data-counter="75."><p id="ref-CR75">Galle, E. et al. H3K18 lactylation marks tissue-specific active enhancers. <i>Genome Biol.</i> <b>23</b>, 207 (2022).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://g-w1.github.io/articles/cas-redirect/1:CAS:528:DC%2BB38XisFKhur3J" aria-label="CAS reference 75">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=36192798" aria-label="PubMed reference 75">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9531456" aria-label="PubMed Central reference 75">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 75" href="http://scholar.google.com/scholar_lookup?&amp;title=H3K18%20lactylation%20marks%20tissue-specific%20active%20enhancers&amp;journal=Genome%20Biol.&amp;volume=23&amp;publication_year=2022&amp;author=Galle%2CE">
                    Google Scholar</a> 
                </p></li><li data-counter="76."><p id="ref-CR76">Hao, Y. et al. Integrated analysis of multimodal single-cell data. <i>Cell</i> <b>184</b>, 3573–3587.e29 (2021).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://g-w1.github.io/articles/cas-redirect/1:CAS:528:DC%2BB3MXhtlSrtrvE" aria-label="CAS reference 76">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34062119" aria-label="PubMed reference 76">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8238499" aria-label="PubMed Central reference 76">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 76" href="http://scholar.google.com/scholar_lookup?&amp;title=Integrated%20analysis%20of%20multimodal%20single-cell%20data&amp;journal=Cell&amp;volume=184&amp;pages=3573-3587.e29&amp;publication_year=2021&amp;author=Hao%2CY">
                    Google Scholar</a> 
                </p></li><li data-counter="77."><p id="ref-CR77">Germain, P.-L., Lun, A., Garcia Meixide, C., Macnair, W. &amp; Robinson, M. D. Doublet identification in single-cell sequencing data using scDblFinder. <i>F1000Research</i> <b>10</b>, 979 (2022).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9204188" aria-label="PubMed Central reference 77">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 77" href="http://scholar.google.com/scholar_lookup?&amp;title=Doublet%20identification%20in%20single-cell%20sequencing%20data%20using%20scDblFinder&amp;journal=F1000Research&amp;volume=10&amp;publication_year=2022&amp;author=Germain%2CP-L&amp;author=Lun%2CA&amp;author=Garcia%20Meixide%2CC&amp;author=Macnair%2CW&amp;author=Robinson%2CMD">
                    Google Scholar</a> 
                </p></li><li data-counter="78."><p id="ref-CR78">Young, M. D. &amp; Behjati, S. SoupX removes ambient RNA contamination from droplet-based single-cell RNA sequencing data. <i>GigaScience</i> <b>9</b>, giaa151 (2020).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33367645" aria-label="PubMed reference 78">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7763177" aria-label="PubMed Central reference 78">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 78" href="http://scholar.google.com/scholar_lookup?&amp;title=SoupX%20removes%20ambient%20RNA%20contamination%20from%20droplet-based%20single-cell%20RNA%20sequencing%20data&amp;journal=GigaScience&amp;volume=9&amp;publication_year=2020&amp;author=Young%2CMD&amp;author=Behjati%2CS">
                    Google Scholar</a> 
                </p></li><li data-counter="79."><p id="ref-CR79">Shan, B. et al. Perivascular mesenchymal cells control adipose-tissue macrophage accrual in obesity. <i>Nat. Metab.</i> <b>2</b>, 1332–1349 (2020).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://g-w1.github.io/articles/cas-redirect/1:CAS:528:DC%2BB3cXitlCmtb3K" aria-label="CAS reference 79">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33139957" aria-label="PubMed reference 79">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7669663" aria-label="PubMed Central reference 79">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 79" href="http://scholar.google.com/scholar_lookup?&amp;title=Perivascular%20mesenchymal%20cells%20control%20adipose-tissue%20macrophage%20accrual%20in%20obesity&amp;journal=Nat.%20Metab.&amp;volume=2&amp;pages=1332-1349&amp;publication_year=2020&amp;author=Shan%2CB">
                    Google Scholar</a> 
                </p></li><li data-counter="80."><p id="ref-CR80">Schwalie, P. C. et al. A stromal cell population that inhibits adipogenesis in mammalian fat depots. <i>Nature</i> <b>559</b>, 103–108 (2018).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://g-w1.github.io/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtFOrsL%2FP" aria-label="CAS reference 80">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29925944" aria-label="PubMed reference 80">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="ads reference" data-track-action="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2018Natur.559..103S" aria-label="ADS reference 80">ADS</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 80" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20stromal%20cell%20population%20that%20inhibits%20adipogenesis%20in%20mammalian%20fat%20depots&amp;journal=Nature&amp;volume=559&amp;pages=103-108&amp;publication_year=2018&amp;author=Schwalie%2CPC">
                    Google Scholar</a> 
                </p></li><li data-counter="81."><p id="ref-CR81">Blanco-Carmona, E. Generating publication ready visualizations for single cell transcriptomics using SCpubr. Preprint at <i>bioRxiv</i> <a href="https://doi.org/10.1101/2022.02.28.482303" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1101/2022.02.28.482303">https://doi.org/10.1101/2022.02.28.482303</a> (2022).</p></li><li data-counter="82."><p id="ref-CR82">Zappia, L. &amp; Oshlack, A. Clustering trees: a visualization for evaluating clusterings at multiple resolutions. <i>GigaScience</i> <b>7</b>, giy083 (2018).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30010766" aria-label="PubMed reference 82">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6057528" aria-label="PubMed Central reference 82">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 82" href="http://scholar.google.com/scholar_lookup?&amp;title=Clustering%20trees%3A%20a%20visualization%20for%20evaluating%20clusterings%20at%20multiple%20resolutions&amp;journal=GigaScience&amp;volume=7&amp;publication_year=2018&amp;author=Zappia%2CL&amp;author=Oshlack%2CA">
                    Google Scholar</a> 
                </p></li><li data-counter="83."><p id="ref-CR83">Sun, W. et al. snRNA-seq reveals a subpopulation of adipocytes that regulates thermogenesis. <i>Nature</i> <b>587</b>, 98–102 (2020).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://g-w1.github.io/articles/cas-redirect/1:CAS:528:DC%2BB3cXit1GjsLnL" aria-label="CAS reference 83">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33116305" aria-label="PubMed reference 83">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="ads reference" data-track-action="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2020Natur.587...98S" aria-label="ADS reference 83">ADS</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 83" href="http://scholar.google.com/scholar_lookup?&amp;title=snRNA-seq%20reveals%20a%20subpopulation%20of%20adipocytes%20that%20regulates%20thermogenesis&amp;journal=Nature&amp;volume=587&amp;pages=98-102&amp;publication_year=2020&amp;author=Sun%2CW">
                    Google Scholar</a> 
                </p></li><li data-counter="84."><p id="ref-CR84">Huang, X. &amp; Huang, Y. Cellsnp-lite: an efficient tool for genotyping single cells. <i>Bioinformatics</i> <b>37</b>, 4569–4571 (2021).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://g-w1.github.io/articles/cas-redirect/1:CAS:528:DC%2BB38XhsFKjsrw%3D" aria-label="CAS reference 84">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33963851" aria-label="PubMed reference 84">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 84" href="http://scholar.google.com/scholar_lookup?&amp;title=Cellsnp-lite%3A%20an%20efficient%20tool%20for%20genotyping%20single%20cells&amp;journal=Bioinformatics&amp;volume=37&amp;pages=4569-4571&amp;publication_year=2021&amp;author=Huang%2CX&amp;author=Huang%2CY">
                    Google Scholar</a> 
                </p></li><li data-counter="85."><p id="ref-CR85">Huang, Y., McCarthy, D. J. &amp; Stegle, O. Vireo: Bayesian demultiplexing of pooled single-cell RNA-seq data without genotype reference. <i>Genome Biol.</i> <b>20</b>, 273 (2019).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31836005" aria-label="PubMed reference 85">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6909514" aria-label="PubMed Central reference 85">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 85" href="http://scholar.google.com/scholar_lookup?&amp;title=Vireo%3A%20Bayesian%20demultiplexing%20of%20pooled%20single-cell%20RNA-seq%20data%20without%20genotype%20reference&amp;journal=Genome%20Biol.&amp;volume=20&amp;publication_year=2019&amp;author=Huang%2CY&amp;author=McCarthy%2CDJ&amp;author=Stegle%2CO">
                    Google Scholar</a> 
                </p></li><li data-counter="86."><p id="ref-CR86">Liao, Y., Smyth, G. K. &amp; Shi, W. featureCounts: an efficient general purpose program for assigning sequence reads to genomic features. <i>Bioinformatics</i> <b>30</b>, 923–930 (2014).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://g-w1.github.io/articles/cas-redirect/1:CAS:528:DC%2BC2cXltFGqu7c%3D" aria-label="CAS reference 86">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24227677" aria-label="PubMed reference 86">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 86" href="http://scholar.google.com/scholar_lookup?&amp;title=featureCounts%3A%20an%20efficient%20general%20purpose%20program%20for%20assigning%20sequence%20reads%20to%20genomic%20features&amp;journal=Bioinformatics&amp;volume=30&amp;pages=923-930&amp;publication_year=2014&amp;author=Liao%2CY&amp;author=Smyth%2CGK&amp;author=Shi%2CW">
                    Google Scholar</a> 
                </p></li><li data-counter="87."><p id="ref-CR87">Robinson, M. D., McCarthy, D. J. &amp; Smyth, G. K. edgeR: a Bioconductor package for differential expression analysis of digital gene expression data. <i>Bioinformatics</i> <b>26</b>, 139–140 (2010).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://g-w1.github.io/articles/cas-redirect/1:CAS:528:DC%2BD1MXhs1WlurvO" aria-label="CAS reference 87">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19910308" aria-label="PubMed reference 87">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 87" href="http://scholar.google.com/scholar_lookup?&amp;title=edgeR%3A%20a%20Bioconductor%20package%20for%20differential%20expression%20analysis%20of%20digital%20gene%20expression%20data&amp;journal=Bioinformatics&amp;volume=26&amp;pages=139-140&amp;publication_year=2010&amp;author=Robinson%2CMD&amp;author=McCarthy%2CDJ&amp;author=Smyth%2CGK">
                    Google Scholar</a> 
                </p></li><li data-counter="88."><p id="ref-CR88">Meers, M. P., Tenenbaum, D. &amp; Henikoff, S. Peak calling by Sparse Enrichment Analysis for CUT&amp;RUN chromatin profiling. <i>Epigenetics Chromatin</i> <b>12</b>, 42 (2019).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31300027" aria-label="PubMed reference 88">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6624997" aria-label="PubMed Central reference 88">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 88" href="http://scholar.google.com/scholar_lookup?&amp;title=Peak%20calling%20by%20Sparse%20Enrichment%20Analysis%20for%20CUT%26RUN%20chromatin%20profiling&amp;journal=Epigenetics%20Chromatin&amp;volume=12&amp;publication_year=2019&amp;author=Meers%2CMP&amp;author=Tenenbaum%2CD&amp;author=Henikoff%2CS">
                    Google Scholar</a> 
                </p></li><li data-counter="89."><p id="ref-CR89">Amemiya, H. M., Kundaje, A. &amp; Boyle, A. P. The ENCODE blacklist: identification of problematic regions of the genome. <i>Sci. Rep.</i> <b>9</b>, 9354 (2019).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31249361" aria-label="PubMed reference 89">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6597582" aria-label="PubMed Central reference 89">PubMed Central</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="ads reference" data-track-action="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2019NatSR...9.9354A" aria-label="ADS reference 89">ADS</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 89" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20ENCODE%20blacklist%3A%20identification%20of%20problematic%20regions%20of%20the%20genome&amp;journal=Sci.%20Rep.&amp;volume=9&amp;publication_year=2019&amp;author=Amemiya%2CHM&amp;author=Kundaje%2CA&amp;author=Boyle%2CAP">
                    Google Scholar</a> 
                </p></li><li data-counter="90."><p id="ref-CR90">Yu, G., Wang, L.-G. &amp; He, Q.-Y. ChIPseeker: an R/Bioconductor package for ChIP peak annotation, comparison and visualization. <i>Bioinformatics</i> <b>31</b>, 2382–2383 (2015).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://g-w1.github.io/articles/cas-redirect/1:CAS:528:DC%2BC28Xht1CitrvM" aria-label="CAS reference 90">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25765347" aria-label="PubMed reference 90">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 90" href="http://scholar.google.com/scholar_lookup?&amp;title=ChIPseeker%3A%20an%20R%2FBioconductor%20package%20for%20ChIP%20peak%20annotation%2C%20comparison%20and%20visualization&amp;journal=Bioinformatics&amp;volume=31&amp;pages=2382-2383&amp;publication_year=2015&amp;author=Yu%2CG&amp;author=Wang%2CL-G&amp;author=He%2CQ-Y">
                    Google Scholar</a> 
                </p></li><li data-counter="91."><p id="ref-CR91">Cavalcante, R. G. &amp; Sartor, M. A. annotatr: genomic regions in context. <i>Bioinformatics</i> <b>33</b>, 2381–2383 (2017).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://g-w1.github.io/articles/cas-redirect/1:CAS:528:DC%2BC1cXitFOntb%2FN" aria-label="CAS reference 91">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28369316" aria-label="PubMed reference 91">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5860117" aria-label="PubMed Central reference 91">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 91" href="http://scholar.google.com/scholar_lookup?&amp;title=annotatr%3A%20genomic%20regions%20in%20context&amp;journal=Bioinformatics&amp;volume=33&amp;pages=2381-2383&amp;publication_year=2017&amp;author=Cavalcante%2CRG&amp;author=Sartor%2CMA">
                    Google Scholar</a> 
                </p></li><li data-counter="92."><p id="ref-CR92">Andrews, S. SeqMonk mapped sequence analysis tool. <i>Babraham Bioinformatics</i> <a href="https://www.bioinformatics.babraham.ac.uk/projects/seqmonk/" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="https://www.bioinformatics.babraham.ac.uk/projects/seqmonk/">https://www.bioinformatics.babraham.ac.uk/projects/seqmonk/</a> (2022).</p></li><li data-counter="93."><p id="ref-CR93">Schep, A. N., Wu, B., Buenrostro, J. D. &amp; Greenleaf, W. J. chromVAR: inferring transcription-factor-associated accessibility from single-cell epigenomic data. <i>Nat. Methods</i> <b>14</b>, 975–978 (2017).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://g-w1.github.io/articles/cas-redirect/1:CAS:528:DC%2BC2sXhtlKjtbbE" aria-label="CAS reference 93">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28825706" aria-label="PubMed reference 93">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5623146" aria-label="PubMed Central reference 93">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 93" href="http://scholar.google.com/scholar_lookup?&amp;title=chromVAR%3A%20inferring%20transcription-factor-associated%20accessibility%20from%20single-cell%20epigenomic%20data&amp;journal=Nat.%20Methods&amp;volume=14&amp;pages=975-978&amp;publication_year=2017&amp;author=Schep%2CAN&amp;author=Wu%2CB&amp;author=Buenrostro%2CJD&amp;author=Greenleaf%2CWJ">
                    Google Scholar</a> 
                </p></li><li data-counter="94."><p id="ref-CR94">Argelaguet, R. et al. MOFA+: a statistical framework for comprehensive integration of multi-modal single-cell data. <i>Genome Biol.</i> <b>21</b>, 111 (2020).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32393329" aria-label="PubMed reference 94">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7212577" aria-label="PubMed Central reference 94">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 94" href="http://scholar.google.com/scholar_lookup?&amp;title=MOFA%2B%3A%20a%20statistical%20framework%20for%20comprehensive%20integration%20of%20multi-modal%20single-cell%20data&amp;journal=Genome%20Biol.&amp;volume=21&amp;publication_year=2020&amp;author=Argelaguet%2CR">
                    Google Scholar</a> 
                </p></li><li data-counter="95."><p id="ref-CR95">Love, M. I., Huber, W. &amp; Anders, S. Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. <i>Genome Biol.</i> <b>15</b>, 550 (2014).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25516281" aria-label="PubMed reference 95">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4302049" aria-label="PubMed Central reference 95">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 95" href="http://scholar.google.com/scholar_lookup?&amp;title=Moderated%20estimation%20of%20fold%20change%20and%20dispersion%20for%20RNA-seq%20data%20with%20DESeq2&amp;journal=Genome%20Biol.&amp;volume=15&amp;publication_year=2014&amp;author=Love%2CMI&amp;author=Huber%2CW&amp;author=Anders%2CS">
                    Google Scholar</a> 
                </p></li><li data-counter="96."><p id="ref-CR96">Ernst, J. &amp; Kellis, M. ChromHMM: automating chromatin-state discovery and characterization. <i>Nat. Methods</i> <b>9</b>, 215–216 (2012).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://g-w1.github.io/articles/cas-redirect/1:CAS:528:DC%2BC38XivV2rtLk%3D" aria-label="CAS reference 96">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22373907" aria-label="PubMed reference 96">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3577932" aria-label="PubMed Central reference 96">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 96" href="http://scholar.google.com/scholar_lookup?&amp;title=ChromHMM%3A%20automating%20chromatin-state%20discovery%20and%20characterization&amp;journal=Nat.%20Methods&amp;volume=9&amp;pages=215-216&amp;publication_year=2012&amp;author=Ernst%2CJ&amp;author=Kellis%2CM">
                    Google Scholar</a> 
                </p></li><li data-counter="97."><p id="ref-CR97">Chen, E. Y. et al. Enrichr: interactive and collaborative HTML5 gene list enrichment analysis tool. <i>BMC Bioinformatics</i> <b>14</b>, 128 (2013).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23586463" aria-label="PubMed reference 97">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3637064" aria-label="PubMed Central reference 97">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 97" href="http://scholar.google.com/scholar_lookup?&amp;title=Enrichr%3A%20interactive%20and%20collaborative%20HTML5%20gene%20list%20enrichment%20analysis%20tool&amp;journal=BMC%20Bioinformatics&amp;volume=14&amp;publication_year=2013&amp;author=Chen%2CEY">
                    Google Scholar</a> 
                </p></li><li data-counter="98."><p id="ref-CR98">Kuleshov, M. V. et al. Enrichr: a comprehensive gene set enrichment analysis web server 2016 update. <i>Nucleic Acids Res.</i> <b>44</b>, W90–W97 (2016).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://g-w1.github.io/articles/cas-redirect/1:CAS:528:DC%2BC2sXhtV2itrfF" aria-label="CAS reference 98">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27141961" aria-label="PubMed reference 98">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4987924" aria-label="PubMed Central reference 98">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 98" href="http://scholar.google.com/scholar_lookup?&amp;title=Enrichr%3A%20a%20comprehensive%20gene%20set%20enrichment%20analysis%20web%20server%202016%20update&amp;journal=Nucleic%20Acids%20Res.&amp;volume=44&amp;pages=W90-W97&amp;publication_year=2016&amp;author=Kuleshov%2CMV">
                    Google Scholar</a> 
                </p></li><li data-counter="99."><p id="ref-CR99">Xie, Z. et al. Gene set knowledge discovery with Enrichr. <i>Curr. Protoc.</i> <b>1</b>, e90 (2021).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://g-w1.github.io/articles/cas-redirect/1:CAS:528:DC%2BB3MXntl2gtb0%3D" aria-label="CAS reference 99">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33780170" aria-label="PubMed reference 99">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8152575" aria-label="PubMed Central reference 99">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 99" href="http://scholar.google.com/scholar_lookup?&amp;title=Gene%20set%20knowledge%20discovery%20with%20Enrichr&amp;journal=Curr.%20Protoc.&amp;volume=1&amp;publication_year=2021&amp;author=Xie%2CZ">
                    Google Scholar</a> 
                </p></li><li data-counter="100."><p id="ref-CR100">Hinte, L. C. et al. Adipose tissue retains an epigenetic memory of obesity after weight loss. <i>Zenodo</i> <a href="https://doi.org/10.5281/zenodo.13870942" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.5281/zenodo.13870942">https://doi.org/10.5281/zenodo.13870942</a> (2024).</p></li></ol><p><a data-track="click" data-track-action="download citation references" data-track-label="link" rel="nofollow" href="https://citation-needed.springer.com/v2/references/10.1038/s41586-024-08165-7?format=refman&amp;flavour=references">Download references<svg width="16" height="16" focusable="false" role="img" aria-hidden="true"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-download-medium"></use></svg></a></p></div></div></div></section></div><section data-title="Acknowledgements"><div id="Ack1-section"><h2 id="Ack1">Acknowledgements</h2><p>We thank all members of the von Meyenn group and the Wolfrum group for helpful discussions and support. We also thank M. Sütö, C. Sert, M. Klug, C. Leuzinger, C. Kellenberger, L. Maak and R. von Wartburg for help with animal experiments and handling; J. P. A. de Sousa for help with computational pipelines; E. Masschelein and T. Dahlby for help with the metabolic cage measurements; E. Seelig for help with acquisition of human samples and scientific support; J. Bohacek and R. Waag for NuTRAP mice; and the Protein Production and Structure Core Facility at EPFL for the production and purification of pA-Tn5 and Tn5, especially K. Lau, F. Pojer and M. Francois. This work was supported by ETH Zurich core funding (F.v.M.), a European Research Council Starting Grant (no. 803491, BRITE to F.v.M.), the Basel Research Centre for Child Health (Multi-Investigator Project 2020 to F.v.M.), the Deutsche Forschungsgemeinschaft (Project no. 209933838–SFB 1052 (project B1) to M.B.), the Margareta af Ugglas foundation (M.R.), the Swedish Research Council (M.R., N.M., including an establishing grant to L.M.), a European Research Council Synergy Grant (no. 856404, SPHERES to M.R.), the Novo Nordisk Foundation (including the MeRIAD consortium grant no. 0064142 to M.R., and no. NNF20OC0061149 to N.M.), the Knut and Alice Wallenberg’s Foundation (Wallenberg Clinical Scholar to M.R.), the Center for Innovative Medicine (M.R.), the Swedish Diabetes Foundation (M.R.), the Stockholm County Council (M.R.), the Strategic Research Program in Diabetes at Karolinska Institutet (M.R.) and the European Foundation for the Study of Diabetes (Future Leaders award to N.M.). L.M. was funded by a postdoctoral grant from the Swedish Society for Medical Research.</p></div></section><section data-title="Funding"><div id="Fun-section"><h2 id="Fun">Funding</h2><p>Open access funding provided by Swiss Federal Institute of Technology Zurich.</p></div></section><section aria-labelledby="author-information" data-title="Author information"><div id="author-information-section"><h2 id="author-information">Author information</h2><div id="author-information-content"><p><span id="author-notes">Author notes</span></p><ol><li id="nAff8"><p>Daniel Castellano-Castillo</p><p>Present address: Medical Oncology Department, Virgen de la Victoria University Hospital, Málaga Biomedical Research Institute (IBIMA)-CIMES-UMA, Málaga, Spain</p></li><li id="nAff9"><p>Lucas Massier</p><p>Present address: Helmholtz Institute for Metabolic, Obesity and Vascular Research (HI-MAG), Helmholtz Zentrum München, University of Leipzig and University Hospital Leipzig, Leipzig, Germany</p></li><li id="nAff10"><p>Hua Dong</p><p>Present address: Stem Cell Bio Regenerative Med Institute, Stanford University, Stanford, CA, USA</p></li><li id="nAff11"><p>Wenfei Sun</p><p>Present address: Department of Bioengineering, Stanford University, Stanford, CA, USA</p></li><li id="na1"><p>These authors contributed equally: Laura C. Hinte, Daniel Castellano-Castillo</p></li></ol><h3 id="affiliations">Authors and Affiliations</h3><ol><li id="Aff1"><p>Laboratory of Nutrition and Metabolic Epigenetics, Institute of Food, Nutrition and Health, Department of Health Sciences and Technology, ETH Zurich, Zurich, Switzerland</p><p>Laura C. Hinte, Daniel Castellano-Castillo, Adhideb Ghosh, Kate Melrose, Emanuel Gasser, Falko Noé &amp; Ferdinand von Meyenn</p></li><li id="Aff2"><p>Biomedicine Programme, Life Science Zurich Graduate School, Zurich, Switzerland</p><p>Laura C. Hinte &amp; Kate Melrose</p></li><li id="Aff3"><p>Functional Genomics Center Zurich, ETH Zurich and University Zurich, Zurich, Switzerland</p><p>Adhideb Ghosh &amp; Falko Noé</p></li><li id="Aff4"><p>Laboratory of Translational Nutrition Biology, Institute of Food, Nutrition and Health, Department of Health Sciences and Technology, ETH Zurich, Zurich, Switzerland</p><p>Adhideb Ghosh, Falko Noé, Hua Dong, Wenfei Sun &amp; Christian Wolfrum</p></li><li id="Aff5"><p>Department of Medicine Huddinge, Karolinska Institutet, Karolinska University Hospital Huddinge, Stockholm, Sweden</p><p>Lucas Massier, Mikael Rydén &amp; Niklas Mejhert</p></li><li id="Aff6"><p>Helmholtz Institute for Metabolic, Obesity and Vascular Research (HI-MAG), Helmholtz Zentrum München, University of Leipzig and University Hospital Leipzig, Leipzig, Germany</p><p>Anne Hoffmann &amp; Matthias Blüher</p></li><li id="Aff7"><p>Medical Department III – Endocrinology, Nephrology, Rheumatology, University of Leipzig Medical Center, Leipzig, Germany</p><p>Matthias Blüher</p></li></ol><h3 id="contributions">Contributions</h3><p>D.C.-C. and L.C.H. performed mouse experiments and made libraries from mouse tissues. L.C.H. performed experiments with human samples and cell culture experiments. L.C.H., A.G. and D.C.-C. analysed data. K.M. assisted with dissection, cell culture experiments and artwork. E.G. assisted with dissection. M.B., C.W. and M.R. supervised the clinical studies and M.B., A.H., L.M., M.R. and N.M. provided human AT samples and clinical data. F.N. SNP-demultiplexed snRNA-seq data. H.D. and W.S. collected additional metadata of human samples. M.B., C.W., M.R. and N.M. supervised the human studies and human AT sample collections. L.C.H., D.C.-C. and F.v.M. conceptualized the study, interpreted the data and wrote the manuscript. All authors contributed to editing and reviewing the manuscript and approved its publication.</p><h3 id="corresponding-author">Corresponding author</h3><p id="corresponding-author-list">Correspondence to
                <a id="corresp-c1" href="mailto:ferdinand.vonmeyenn@hest.ethz.ch">Ferdinand von Meyenn</a>.</p></div></div></section><section data-title="Ethics declarations"><div id="ethics-section"><h2 id="ethics">Ethics declarations</h2><div id="ethics-content">
              
                <h3 id="FPar16">Competing interests</h3>
                <p>M.B. received honoraria as a consultant and speaker from Amgen, AstraZeneca, Bayer, Boehringer-Ingelhiem, Lilly, Novo Nordisk and Sanofi. M.R. received honoraria as a consultant and speaker from AstraZeneca, Boehringer-Ingelheim, Lilly, Novo Nordisk and Sanofi. The other authors declare no competing interests.</p>
              
            </div></div></section><section data-title="Peer review"><div id="peer-review-section"><h2 id="peer-review">Peer review</h2><div id="peer-review-content">
              
              
                <h3 id="FPar15">Peer review information</h3>
                <p><i>Nature</i> thanks Katherine Gallagher, Naomi Habib, Alyssa Hasty and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#MOESM2">Peer reviewer reports</a> are available.</p>
              
            </div></div></section><section data-title="Additional information"><div id="additional-information-section"><h2 id="additional-information">Additional information</h2><p><b>Publisher’s note</b> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div></section><section data-title="Extended data figures and tables"><div id="Sec50-section"><h2 id="Sec50">Extended data figures and tables</h2><div id="Sec50-content"><div data-test="supplementary-info"><div data-test="supp-item" id="Fig6"><h3><a data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 1 human at retains cellular tra" href="https://g-w1.github.io/articles/s41586-024-08165-7/figures/6" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41586-024-08165-7/MediaObjects/41586_2024_8165_Fig6_ESM.jpg">Extended Data Fig. 1 Human AT retains cellular transcriptional changes after bariatric surgery induced WL.</a></h3><div data-component="thumbnail-container"><p><b>a</b>, UMAP of 19,494 nuclei representing omAT pools from lean subjects (<i>n</i> = 5; 1 male, 4 females) and paired omAT from T0 and T1 (<i>n</i> = 8 each; 2 males, 6 females) from the MTSS study. <b>b</b>,<b>c</b> Proportion of retained transcriptional changes in highly abundant cell types of MTSS omAT. <b>d</b>, UMAP of 31,721 nuclei representing scAT pools from lean subjects (<i>n</i> = 8; 8 females) and paired scAT from T0 and T1 (<i>n</i> = 7 each; 7 females) from the NEFA study. <b>e</b>,<b>f</b> Proportion of retained transcriptional changes in highly abundant cell types of NEFA scAT. <b>g-j</b> Number of upregulated and downregulated DEGs per cell type obese donor scaled by column at T0 for omAT (left) and scAT (right) from MTSS, LTSS and NEFA studies. <b>k</b>, Number of persistently deregulated genes from T0 to T1 per cell type across AT pools from all studies. <b>l</b>, UMAP of 4,958 nuclei representing adipocytes from MTSS omAT and LTSS omAT (total lean <i>n</i> = 10; total T0/T1 <i>n</i> = 13). <b>m</b>, UMAP of 13,231 nuclei representing adipocytes from NEFA scAT and LTSS scAT (total lean <i>n</i> = 13; total T0/T1 <i>n</i> = 12). Wilcoxon Rank Sum test, with adjusted p-value &lt; 0.01 by the Bonferroni correction method and FC &gt; ±0.5 was used for DEG identification in b, c, e-k. APCs, adipocyte progenitor cells; ASDCs, AXL+ dendritic cells; DCs, dendritic cells; EndoCs, endothelial cells; EndoACs, arteriolar EndoCs; EndoSCs, stalk EndoCs; EndoVCs, venular EndoCs; LECs, lymphatic endothelial cells; FAPs, fibro-adipogenic progenitors; Macro, macrophages; MastCs, mast cells; MesoCs, mesothelial cells; NeurCs, neuronal like cells; SMCs, (vascular) smooth muscle cells.</p><p><a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#MOESM3">Source Data</a></p></div></div><div data-test="supp-item" id="Fig7"><h3><a data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 2 characterization of omat comp" href="https://g-w1.github.io/articles/s41586-024-08165-7/figures/7" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41586-024-08165-7/MediaObjects/41586_2024_8165_Fig7_ESM.jpg">Extended Data Fig. 2 Characterization of omAT composition.</a></h3><div data-component="thumbnail-container"><p><b>a</b>,<b>b</b>, Cluster markers used for annotating cell clusters in human omAT of the MTSS (left) and LTSS (right) study. <b>c</b>,<b>d</b>, UMAP visualization representing omAT pools from the MTSS study (c) and LTSS study (d) coloured by predicted cell subtypes from the Emont et al. visceral AT dataset from Caucasian individuals. Feature plots showing reference mapping scores illustrating how well omAT dataset maps to the Emont et al. dataset. <b>e</b>,<b>f</b>, Relative cell type abundance in omAT per condition and tissue donor of the LTSS (e) and MTSS (f) study. Lines connecting dots indicate paired samples. Significance between T0 and T1 for e-f was calculated using paired multiple Wilcoxon tests with Benjamini, Krieger and Yekutieli post hoc test for multiple comparisons. Error bars represent s.d.</p><p><a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#MOESM3">Source Data</a></p></div></div><div data-test="supp-item" id="Fig8"><h3><a data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 3 characterization of scat comp" href="https://g-w1.github.io/articles/s41586-024-08165-7/figures/8" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41586-024-08165-7/MediaObjects/41586_2024_8165_Fig8_ESM.jpg">Extended Data Fig. 3 Characterization of scAT composition.</a></h3><div data-component="thumbnail-container"><p><b>a</b>,<b>b</b>, Cluster markers used for annotating cell clusters in human scAT of the LTSS (left) and NEFA (right) study. <b>c</b>,<b>d</b>, UMAP visualization representing scAT pools from the LTSS study (c) and NEFA study (d) coloured by predicted cell subtypes from the Emont et al. subcutaneous AT dataset from Caucasian individuals. Feature plots showing reference mapping scores illustrating how well scAT dataset maps to the Emont et al. dataset. <b>e</b>,<b>f</b>, Relative cell type abundance in scAT per condition and tissue donor of the LTSS (e) and NEFA (f) study. Lines connecting dots indicate paired samples. Significance between T0 and T1 for e-f was calculated using paired multiple Wilcoxon tests with Benjamini, Krieger and Yekutieli post hoc test for multiple comparisons. Error bars represent s.d.</p><p><a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#MOESM3">Source Data</a></p></div></div><div data-test="supp-item" id="Fig9"><h3><a data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 4 gsea of retained degs in adip" href="https://g-w1.github.io/articles/s41586-024-08165-7/figures/9" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41586-024-08165-7/MediaObjects/41586_2024_8165_Fig9_ESM.jpg">Extended Data Fig. 4 GSEA of retained DEGs in adipocytes.</a></h3><div data-component="thumbnail-container"><p><b>a</b>,<b>b</b>, Top (significant) persistently downregulated (memory) pathway terms in omental adipocytes of the MTSS (a) and LTSS (b) study based on Wikipathways database. <b>c</b>,<b>d</b>, Top (significant) persistently downregulated (memory) pathway terms in subcutaneous adipocytes of the LTSS (c) and NEFA (d) study based on Wikipathways database. <b>e</b>,<b>f</b>, Top (significant) persistently upregulated (memory) pathway terms in omental adipocytes of the MTSS (e) and LTSS (f) study based on Wikipathways database. <b>g</b>,<b>h</b>, Top (significant) persistently downregulated (memory) pathway terms in subcutaneous adipocytes of the LTSS (g) and NEFA (d) study based on Wikipathways database. In g enrichment is not significant. Significance was calculated using Fisher’s exact test, with adjusted P-value &lt; 0.05 by the Benjamini-Hochberg method for correction.</p><p><a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#MOESM3">Source Data</a></p></div></div><div data-test="supp-item" id="Fig10"><h3><a data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 5 weight loss largely resolves " href="https://g-w1.github.io/articles/s41586-024-08165-7/figures/10" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41586-024-08165-7/MediaObjects/41586_2024_8165_Fig10_ESM.jpg">Extended Data Fig. 5 Weight loss largely resolves obesity induced physiological changes in mice.</a></h3><div data-component="thumbnail-container"><p>Data from mouse experiments. For <i>n</i>&lt;11, each dot represents an biological replicate. Data from 2-3 independent experiments. <b>a</b>, Glucose tolerance tests (GTTs) and area of the curve (AOC) for GTTs; (<i>n</i> = 10 each). <b>b</b>, Insulin tolerance tests (ITTs) and AOC for ITTs (<i>n</i> = 10 each). <b>c</b>, Fasting blood glucose (<i>n</i> = 10 each). <b>d</b>, GTTs and AOCs for GTTs; (<i>n</i> = 10 each from 2 independent experiments). <b>e</b>, ITTs and AOC for ITTs (<i>n</i> = 10 each from 2 independent experiments). <b>f</b>, Fasting blood glucose (CC_s&amp;HC: <i>n</i> = 10 each, CCC&amp;HHC: <i>n</i> = 20 each; from 2 independent experiments). <b>g,h</b>, Fasting insulin levels (<i>n</i> = 6). <b>i</b>,<b>j</b> Postprandial insulin and leptin levels. C, H, CC_s, HC, HH, HHC: <i>n</i> = 6 each; CC_l, CCC: <i>n</i> = 5 each. Boxplot represents minimum, maximum and median. <b>k</b>, Cumulative food intake from HC and CC_s mice in the last 3 days of WL chow diet feeding. (<i>n</i> = 10 mice each). <b>l</b>, Energy expenditure of HC and CC_s mice in the last 3 days of WL chow diet feeding. (<i>n</i> = 10 mice each). <b>m</b>, Liver triglycerides (tg) per μg liver tissue (C&amp;H: <i>n</i> = 6, CC_s: <i>n</i> = 10, HC, CC_l, HHC: <i>n</i> = 9, HH&amp;CCC: <i>n</i> = 8). Boxplot represents minimum, maximum and median. <b>n</b>, Haematoxylin and eosin (HE) staining liver sections, 20x magnification. Scale bar, 200 μm. <b>o</b>, Lean mass of HC and CC_s mice relative to lean mass measured at C and H timepoints of the same mice (right) (n = 19 each). <b>p</b>, weights of ingAT, epiAT and BAT, normalized to body weight (C&amp;H: <i>n</i> = 6, HC&amp;CC_s: <i>n</i> = 10, from 2 experiments). <b>q</b>, Representative photos of epiAT depots. Ruler is in cm. <b>r</b>, Weights of ingAT, epiAT and BAT, normalized to body weight (HH&amp;CC_l: <i>n</i> = 6, CCC&amp;HHC: <i>n</i> = 10). <b>s</b>, Representative photos of epiAT depots. Ruler is in cm. <b>t</b>, Representative photo of a HHC mouse. <b>u</b>, Representative image of a histological and HE stained section of a whole epiAT depot from a HHC mouse. Scale bar 2000 μm. <b>v</b>, Haematoxylin and eosin (HE) staining of epiAT, 20x magnification. Scale bar, 100 μm. Representative pictures. <b>w</b>, ingAT adipocyte area across conditions. (<i>n</i> = 4 mice each, 5-8 pictures each). <b>x</b>, HE staining of scAT, 20x magnification. Scale bar, 200 μm. <b>y</b>, epiAT adipocyte area across conditions. (<i>n</i> = 4 mice each, 5-8 pictures each). <b>z</b>, Quantification of collagen content from Maison’s Trichome staining. (<i>n</i> = 4 mice each, 20 pictures each). Significance was calculated between age matched controls and experimental groups. Significance a, b, d, e, i, j, z was calculated using two-tailed Mann-Whitney tests. Significance for c, f-h, m was calculated using unpaired, two-tailed t-tests with Welch’s correction. Error bars represent s.d. Significance for p and r was calculated using unpaired, multiple t-tests with Benjamini, Krieger and Yekutieli post hoc test for multiple comparisons. ns = <i>FDR</i> &gt; 0.01, **<i>FDR</i> &lt; 0.01, ***<i>FDR</i> &lt; 0.001, ****<i>FDR</i> &lt; 0.0001.</p><p><a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#MOESM3">Source Data</a></p></div></div><div data-test="supp-item" id="Fig11"><h3><a data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 6 annotation of mouse epiat." href="https://g-w1.github.io/articles/s41586-024-08165-7/figures/11" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41586-024-08165-7/MediaObjects/41586_2024_8165_Fig11_ESM.jpg">Extended Data Fig. 6 Annotation of mouse epiAT.</a></h3><div data-component="thumbnail-container"><p><b>a</b>, Cluster markers used to annotate cell clusters of mouse epiAT. <b>b</b>, UMAP visualization representing epiAT samples coloured by predicted cell subtypes from the Emont et al. mouse epididymal AT dataset. Feature plots showing reference mapping scores illustrating how well this dataset maps to the Emont et al. dataset. <b>c</b>, Macrophage subcluster markers. <b>d</b>, UMAP of 16,567 nuclei representing macrophage subclusters. APCs, adipocyte progenitor cells; DCs, dendritic cells; EpiCs, epithelial cells; EndoCs, endothelial cells; FIPs, fibro-inflammatory progenitors; LECs, lymphatic endothelial cells; MastCs, mast cells; MesoCs, mesothelial cells; SMCs, (vascular) smooth muscle cells; NPVMs, non-perivascular macrophages; LAMs, lipid-associated macrophages; PVMs, perivascular macrophages; P-LAMs, proliferating LAMs.</p><p><a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#MOESM3">Source Data</a></p></div></div><div data-test="supp-item" id="Fig12"><h3><a data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 7 transcriptional changes persi" href="https://g-w1.github.io/articles/s41586-024-08165-7/figures/12" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41586-024-08165-7/MediaObjects/41586_2024_8165_Fig12_ESM.jpg">Extended Data Fig. 7 Transcriptional changes persist weight loss in epiAT.</a></h3><div data-component="thumbnail-container"><p><b>a</b>, Number of upregulated (left) and downregulated (right) DEGs per cell type per comparison (H vs C, HC vs CC, HH vs CC, HHC vs CCC) scaled by column. <b>b</b>, Proportion of retained transcriptional changes in different cell types. (Wilcoxon Rank Sum test, adjusted p-value &lt; 0.05 by the Bonferroni correction method; FC &gt; ±0.5). <b>c</b>,<b>d</b>, Top (significant) persistently upregulated (memory) (c) and downregulated (d). pathway terms in HC adipocytes based on Wikipathways database. <b>e</b>,<b>f</b>, Significant Wikipathways term enrichment scores related to persistently upregulated genes in HHC (f) and HC (g) per cell type. <b>g</b>,<b>h</b>, Significant Wikipathways term enrichment scores related to persistently downregulated genes in HHC (f) and HC (g) per cell type. Significance for c-h was calculated using Fisher’s exact test, with adjusted P-value &lt; 0.05 by the Benjamini-Hochberg method for correction. APCs, adipocyte progenitor cells; DCs, dendritic cells; EpiCs, epithelial cells; EndoCs, endothelial cells; FIPs, fibro-inflammatory progenitors; LECs, lymphatic endothelial cells; MastCs, mast cells; MesoCs, mesothelial cells; SMCs, (vascular) smooth muscle cells; NPVMs, non-perivascular macrophages; LAMs, lipid-associated macrophages; PVMs, perivascular macrophages; P-LAMs, proliferating LAMs.</p><p><a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#MOESM3">Source Data</a></p></div></div><div data-test="supp-item" id="Fig13"><h3><a data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 8 epigenetic memory persists af" href="https://g-w1.github.io/articles/s41586-024-08165-7/figures/13" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41586-024-08165-7/MediaObjects/41586_2024_8165_Fig13_ESM.jpg">Extended Data Fig. 8 Epigenetic memory persists after weight loss.</a></h3><div data-component="thumbnail-container"><p><b>a</b>, Peak fold enrichment of called peaks from each CUT&amp;Tag library for genomic features, scaled from −2 to 2. <b>b</b>, Peak fold enrichment of called peaks from each CUT&amp;Tag library for ENCODE cCREs, scaled from −2 to 2. <b>c</b>, Scatterplots of pairwise correlation of average normalized expression of pseudo bulk adipocytes from snRNA-seq (log2 cpm) and average normalized expression of translating RNA (TRAPseq) from labelled adipocytes (log2 cpm) per condition. Spearman’s correlation coefficient R is indicated. <b>d</b>, PCA plots of H3K4me3, H3K27me3 and ATAC-seq across all conditions quantified over peaks overlapping promoters with reads summed up at gene level; each dot represents one biological replicate. <b>e</b>, Dynamics of differentially H3K4me3-marked (left) and H3K27me3-marked promoters (y-axis) from HH to HHC. <b>f</b>, Significant Wikipathways term enrichment scores related to genes associated with persistently differentially marked promoters by H3K27me3 (from Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig4">4f</a>) or H3K4me3 (from Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#Fig4">4g</a>) in HC adipocytes. <b>g</b>, Significant Wikipathways term enrichment scores related to genes associated with persistently differentially marked promoters by H3K27me3 (from e) or H3K4me3 (from e) in HHC adipocytes. <b>h</b>, Expression of genes encoding for epigenetic modifiers significantly deregulated either in H (*) or HH (#) adipocytes. (Wilcoxon Rank Sum test, adjusted p-value &lt; 0.05 by the Bonferroni correction method; fold change (FC) &gt; ±0.5). None of the epigenetic modifiers are deregulated in HC or HHC adipocytes. Significance for f-g was calculated using Fisher’s exact test, with adjusted P-value &lt; 0.05 by the Benjamini-Hochberg method for correction. cCREs, candidate cis-regulatory elements as defined by ENCODE. CTCF, not TSS-overlapping and with high DNase and CTCF signals only; DNase–H3K4me3, not TSS-overlapping and with high DNase and H3K4me3 signals only; dELS, TSS-distal with enhancer-like signatures; PLS, TSS-overlapping with promoter-like signatures; pELS, TSS-proximal with enhancer-like signatures.</p><p><a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#MOESM3">Source Data</a></p></div></div><div data-test="supp-item" id="Fig14"><h3><a data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 9 adipocyte specific enhancers " href="https://g-w1.github.io/articles/s41586-024-08165-7/figures/14" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41586-024-08165-7/MediaObjects/41586_2024_8165_Fig14_ESM.jpg">Extended Data Fig. 9 Adipocyte specific enhancers retain an epigenetic memory.</a></h3><div data-component="thumbnail-container"><p><b>a</b>, Correlation coefficient R (Pearson) of quantified peaks of H3K27ac against a hypothetical healthy control (<i>n</i> = 2-3 each) with s.d. Each dot represents an individual biological replicate. <b>b</b>,<b>c</b>, ChromHMM analysis of the adipocyte hPTM profiles for conditions C, CC_s, H and HC (b) and CC_l, CCC, HH and HHC (c). The colour scale corresponds to the emission parameter of each hPTM for each state. <b>d</b>,<b>e</b>, Fold enrichment of ChromHMM states from b and c for total genomic fraction coverage, ENCODE cCREs, and genomic features scaled from −2 to 2. State 5, 6 and 7 are identified as enhancers. <b>f</b>, PCA plot of quantified adipocyte specific enhancers from all conditions as marked by H3K4me1. Each dot represents an individual biological replicate. <b>g</b>, PCA plot of quantified adipocyte specific enhancers as marked by H3K27ac. Each dot represents an individual biological replicate. <b>h</b>, PCA plot of quantified adipocyte specific enhancers as marked by H3K27ac. Each dot represents an individual biological replicate. <b>i</b>, Top (significant) GO Cellular Component terms for genes linked to newly emerged and acetylated enhancers for H and HC (left) and HH and HHC (right). (Fisher’s exact test, adjusted p-value &lt; 0.05 by the Benjamini-Hochberg method for correction). cCREs, candidate cis-regulatory elements as defined by ENCODE. CTCF, not TSS-overlapping and with high DNase and CTCF signals only; DNase–H3K4me3, not TSS-overlapping and with high DNase and H3K4me3 signals only; dELS, TSS-distal with enhancer-like signatures; PLS, TSS-overlapping with promoter-like signatures; pELS, TSS-proximal with enhancer-like signatures.</p><p><a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#MOESM3">Source Data</a></p></div></div><div data-test="supp-item" id="Fig15"><h3><a data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 10 other responses of primed mi" href="https://g-w1.github.io/articles/s41586-024-08165-7/figures/15" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41586-024-08165-7/MediaObjects/41586_2024_8165_Fig15_ESM.jpg">Extended Data Fig. 10 Other responses of primed mice and cells to obesogenic stimuli.</a></h3><div data-component="thumbnail-container"><p><b>a</b>, Glucose uptake of isolated, cultured primary adipocytes from ingAT from CC_s and HC (left) and CCC and HHC (right) mice. Each dot represents an individual biological replicate of a pool of 3 mice. <b>b,c</b>, AdipoRed signal of dividing SVF (MM), SVF stimulated with 10 nm insulin only (MM + Ins) and induced SVF with 10 nm insulin (IMM + Ins) 10 days after induction/no induction of differentiation from epiAT (b) and ingAT (c) SVF from CC_s, HC, CCC and HHC mice. Each dot represents an individual biological replicate of a pool of 3 mice. Every SVF pool was tested in all three conditions. <b>d</b>, GTT of HCH and CCH mice; blood glucose levels (<i>n</i> = 5 each). <b>e</b>, AOC of GTTs from d. <b>f</b>, ITT of CCH and HCH mice; blood glucose levels (<i>n</i> = 5 each). <b>g</b>, AOC from ITTs from e. <b>h</b>, Distribution of epiAT adipocyte area. (<i>n</i> = 4 mice each, 10 pictures each). <b>i</b>, Representative images of liver HE stained sections from CCH and HCH, 20x magnification, scale bar 200 μm. <b>j</b>, Liver tg per μg liver tissue (C&amp;H:<i>n</i> = 6 each, CC_s: <i>n</i> = 10, HC: <i>n</i> = 9, CCH: <i>n</i> = 11, HCH: <i>n</i> = 12, from 2-3 experiments). Boxplot represents minimum, maximum and median. <b>k</b>, Pathological scoring of liver sections per group (<i>n</i> = 4 each). <b>l</b>, UMAP of 15,665 nuclei representing epiAT pools (<i>n</i> = 5 pooled mice each) from CCH and HCH split by condition. <b>m</b>, Relative abundance of macrophage subclusters. <b>n</b>,<b>o</b>, Top significant pathway terms from upregulated (n) and downregulated (o) HCH DEGs that are explained by the epigenetic state in HC adipocytes based on Reactome database. (Fisher’s exact test, adjusted p-value &lt; 0.05 by the Benjamini-Hochberg method for correction). Significance was calculated between age matched controls and experimental groups. Significance for a, e, g, was calculated using two-tailed Mann-Whitney tests. Significance for b, c was calculated using unpaired, two-tailed Student’s <i>t</i>-tests with Welch’s correction and Benjamini, Krieger, and Yekutieli correction for multiple testing. Significance for j was calculated using unpaired two-tailed Student’s <i>t</i>-tests with Welch’s correction. Error bars represent s.d. APCs, adipocyte progenitor cells; DCs, dendritic cells; EpiCs, epithelial cells; EndoCs, endothelial cells; FIPs, fibro-inflammatory progenitors; LECs, lymphatic endothelial cells; MastCs, mast cells; MesoCs, mesothelial cells; SMCs, (vascular) smooth muscle cells; NPVMs, non-perivascular macrophages; LAMs, lipid-associated macrophages; PVMs, perivascular macrophages; P-LAMs, proliferating LAMs.</p><p><a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://g-w1.github.io/articles/s41586-024-08165-7#MOESM3">Source Data</a></p></div></div><div data-test="supp-item" id="Fig16"><h3><a data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 11 quality metrics of mouse snr" href="https://g-w1.github.io/articles/s41586-024-08165-7/figures/16" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41586-024-08165-7/MediaObjects/41586_2024_8165_Fig16_ESM.jpg">Extended Data Fig. 11 Quality metrics of mouse snRNAseq data.</a></h3><p><b>a</b>, Gene counts and the number of unique molecular identifiers (UMIs) per condition of mouse epiAT samples. <b>b</b>, UMAP visualization representing integrated epiAT samples from the weight loss study (C, CC, CCC, H, HH, HC, HHC) and from the “yoyo” study (CCH, HCH) coloured by predicted cell subtypes from the Emont et al. mouse epididymal AT dataset. Feature plots showing reference mapping scores illustrating how well these datasets maps to the Emont et al. dataset. <b>c</b>, gene counts and the number of UMIs per cell type from mouse epiAT samples.</p></div><div data-test="supp-item" id="Fig17"><h3><a data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 12 quality metrics of human snr" href="https://g-w1.github.io/articles/s41586-024-08165-7/figures/17" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41586-024-08165-7/MediaObjects/41586_2024_8165_Fig17_ESM.jpg">Extended Data Fig. 12 Quality metrics of human snRNAseq data.</a></h3><p><b>a</b>, Gene counts and the number of UMIs per condition in the omAT samples from the MTSS (left), LTSS (second left) and in scAT samples from the LTSS (second from right) and NEFA (right) study. <b>b</b>,<b>c</b>, Gene counts and the number of UMIs per donor in the omAT samples from the MTSS (b) and LTSS (c) study. <b>d</b>,<b>e</b>, Gene counts and the number of UMIs per donor in scAT samples from the LTSS (d) and NEFA (e) study. <b>f</b>, Gene counts and the number of UMIs per assigned cell type in the omAT samples from the MTSS (left) and LTSS (right) study. <b>g</b>, Gene counts and the number of UMIs per assigned cell type in the scAT samples from the LTSS (left) and NEFA (right) study.</p></div><div data-test="supp-item"><div data-test="inline-table" data-container-section="table" id="table-1"><figure><figcaption><b id="Tab1" data-test="table-caption">Extended Data Table 1 Clinical parameters</b></figcaption></figure></div></div></div></div></div></section><section data-title="Supplementary information"><div id="Sec51-section"><h2 id="Sec51">Supplementary information</h2></div></section><section data-title="Source data"></section><section data-title="Rights and permissions"><div id="rightslink-section"><h2 id="rightslink">Rights and permissions</h2><div id="rightslink-content">
                <p><b>Open Access</b>  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>.</p>
              <p><a data-track="click" data-track-action="view rights and permissions" data-track-label="link" href="https://s100.copyright.com/AppDispatchServlet?title=Adipose%20tissue%20retains%20an%20epigenetic%20memory%20of%20obesity%20after%20weight%20loss&amp;author=Laura%20C.%20Hinte%20et%20al&amp;contentID=10.1038%2Fs41586-024-08165-7&amp;copyright=The%20Author%28s%29&amp;publication=0028-0836&amp;publicationDate=2024-11-18&amp;publisherName=SpringerNature&amp;orderBeanReset=true&amp;oa=CC%20BY">Reprints and permissions</a></p></div></div></section><section aria-labelledby="article-info" data-title="About this article"><div id="article-info-section"><h2 id="article-info">About this article</h2><div id="article-info-content"><div><p><a data-crossmark="10.1038/s41586-024-08165-7" target="_blank" rel="noopener" href="https://crossmark.crossref.org/dialog/?doi=10.1038/s41586-024-08165-7" data-track="click" data-track-action="Click Crossmark" data-track-label="link" data-test="crossmark"><img loading="lazy" width="57" height="81" alt="Check for updates. Verify currency and authenticity via CrossMark" src="data:image/svg+xml;base64,<svg height="81" width="57" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="m17.35 35.45 21.3-14.2v-17.03h-21.3" fill="#989898"/><path d="m38.65 35.45-21.3-14.2v-17.03h21.3" fill="#747474"/><path d="m28 .5c-12.98 0-23.5 10.52-23.5 23.5s10.52 23.5 23.5 23.5 23.5-10.52 23.5-23.5c0-6.23-2.48-12.21-6.88-16.62-4.41-4.4-10.39-6.88-16.62-6.88zm0 41.25c-9.8 0-17.75-7.95-17.75-17.75s7.95-17.75 17.75-17.75 17.75 7.95 17.75 17.75c0 4.71-1.87 9.22-5.2 12.55s-7.84 5.2-12.55 5.2z" fill="#535353"/><path d="m41 36c-5.81 6.23-15.23 7.45-22.43 2.9-7.21-4.55-10.16-13.57-7.03-21.5l-4.92-3.11c-4.95 10.7-1.19 23.42 8.78 29.71 9.97 6.3 23.07 4.22 30.6-4.86z" fill="#9c9c9c"/><path d="m.2 58.45c0-.75.11-1.42.33-2.01s.52-1.09.91-1.5c.38-.41.83-.73 1.34-.94.51-.22 1.06-.32 1.65-.32.56 0 1.06.11 1.51.35.44.23.81.5 1.1.81l-.91 1.01c-.24-.24-.49-.42-.75-.56-.27-.13-.58-.2-.93-.2-.39 0-.73.08-1.05.23-.31.16-.58.37-.81.66-.23.28-.41.63-.53 1.04-.13.41-.19.88-.19 1.39 0 1.04.23 1.86.68 2.46.45.59 1.06.88 1.84.88.41 0 .77-.07 1.07-.23s.59-.39.85-.68l.91 1c-.38.43-.8.76-1.28.99-.47.22-1 .34-1.58.34-.59 0-1.13-.1-1.64-.31-.5-.2-.94-.51-1.31-.91-.38-.4-.67-.9-.88-1.48-.22-.59-.33-1.26-.33-2.02zm8.4-5.33h1.61v2.54l-.05 1.33c.29-.27.61-.51.96-.72s.76-.31 1.24-.31c.73 0 1.27.23 1.61.71.33.47.5 1.14.5 2.02v4.31h-1.61v-4.1c0-.57-.08-.97-.25-1.21-.17-.23-.45-.35-.83-.35-.3 0-.56.08-.79.22-.23.15-.49.36-.78.64v4.8h-1.61zm7.37 6.45c0-.56.09-1.06.26-1.51.18-.45.42-.83.71-1.14.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.36c.07.62.29 1.1.65 1.44.36.33.82.5 1.38.5.29 0 .57-.04.83-.13s.51-.21.76-.37l.55 1.01c-.33.21-.69.39-1.09.53-.41.14-.83.21-1.26.21-.48 0-.92-.08-1.34-.25-.41-.16-.76-.4-1.07-.7-.31-.31-.55-.69-.72-1.13-.18-.44-.26-.95-.26-1.52zm4.6-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.07.45-.31.29-.5.73-.58 1.3zm2.5.62c0-.57.09-1.08.28-1.53.18-.44.43-.82.75-1.13s.69-.54 1.1-.71c.42-.16.85-.24 1.31-.24.45 0 .84.08 1.17.23s.61.34.85.57l-.77 1.02c-.19-.16-.38-.28-.56-.37-.19-.09-.39-.14-.61-.14-.56 0-1.01.21-1.35.63-.35.41-.52.97-.52 1.67 0 .69.17 1.24.51 1.66.34.41.78.62 1.32.62.28 0 .54-.06.78-.17.24-.12.45-.26.64-.42l.67 1.03c-.33.29-.69.51-1.08.65-.39.15-.78.23-1.18.23-.46 0-.9-.08-1.31-.24-.4-.16-.75-.39-1.05-.7s-.53-.69-.7-1.13c-.17-.45-.25-.96-.25-1.53zm6.91-6.45h1.58v6.17h.05l2.54-3.16h1.77l-2.35 2.8 2.59 4.07h-1.75l-1.77-2.98-1.08 1.23v1.75h-1.58zm13.69 1.27c-.25-.11-.5-.17-.75-.17-.58 0-.87.39-.87 1.16v.75h1.34v1.27h-1.34v5.6h-1.61v-5.6h-.92v-1.2l.92-.07v-.72c0-.35.04-.68.13-.98.08-.31.21-.57.4-.79s.42-.39.71-.51c.28-.12.63-.18 1.04-.18.24 0 .48.02.69.07.22.05.41.1.57.17zm.48 5.18c0-.57.09-1.08.27-1.53.17-.44.41-.82.72-1.13.3-.31.65-.54 1.04-.71.39-.16.8-.24 1.23-.24s.84.08 1.24.24c.4.17.74.4 1.04.71s.54.69.72 1.13c.19.45.28.96.28 1.53s-.09 1.08-.28 1.53c-.18.44-.42.82-.72 1.13s-.64.54-1.04.7-.81.24-1.24.24-.84-.08-1.23-.24-.74-.39-1.04-.7c-.31-.31-.55-.69-.72-1.13-.18-.45-.27-.96-.27-1.53zm1.65 0c0 .69.14 1.24.43 1.66.28.41.68.62 1.18.62.51 0 .9-.21 1.19-.62.29-.42.44-.97.44-1.66 0-.7-.15-1.26-.44-1.67-.29-.42-.68-.63-1.19-.63-.5 0-.9.21-1.18.63-.29.41-.43.97-.43 1.67zm6.48-3.44h1.33l.12 1.21h.05c.24-.44.54-.79.88-1.02.35-.24.7-.36 1.07-.36.32 0 .59.05.78.14l-.28 1.4-.33-.09c-.11-.01-.23-.02-.38-.02-.27 0-.56.1-.86.31s-.55.58-.77 1.1v4.2h-1.61zm-47.87 15h1.61v4.1c0 .57.08.97.25 1.2.17.24.44.35.81.35.3 0 .57-.07.8-.22.22-.15.47-.39.73-.73v-4.7h1.61v6.87h-1.32l-.12-1.01h-.04c-.3.36-.63.64-.98.86-.35.21-.76.32-1.24.32-.73 0-1.27-.24-1.61-.71-.33-.47-.5-1.14-.5-2.02zm9.46 7.43v2.16h-1.61v-9.59h1.33l.12.72h.05c.29-.24.61-.45.97-.63.35-.17.72-.26 1.1-.26.43 0 .81.08 1.15.24.33.17.61.4.84.71.24.31.41.68.53 1.11.13.42.19.91.19 1.44 0 .59-.09 1.11-.25 1.57-.16.47-.38.85-.65 1.16-.27.32-.58.56-.94.73-.35.16-.72.25-1.1.25-.3 0-.6-.07-.9-.2s-.59-.31-.87-.56zm0-2.3c.26.22.5.37.73.45.24.09.46.13.66.13.46 0 .84-.2 1.15-.6.31-.39.46-.98.46-1.77 0-.69-.12-1.22-.35-1.61-.23-.38-.61-.57-1.13-.57-.49 0-.99.26-1.52.77zm5.87-1.69c0-.56.08-1.06.25-1.51.16-.45.37-.83.65-1.14.27-.3.58-.54.93-.71s.71-.25 1.08-.25c.39 0 .73.07 1 .2.27.14.54.32.81.55l-.06-1.1v-2.49h1.61v9.88h-1.33l-.11-.74h-.06c-.25.25-.54.46-.88.64-.33.18-.69.27-1.06.27-.87 0-1.56-.32-2.07-.95s-.76-1.51-.76-2.65zm1.67-.01c0 .74.13 1.31.4 1.7.26.38.65.58 1.15.58.51 0 .99-.26 1.44-.77v-3.21c-.24-.21-.48-.36-.7-.45-.23-.08-.46-.12-.7-.12-.45 0-.82.19-1.13.59-.31.39-.46.95-.46 1.68zm6.35 1.59c0-.73.32-1.3.97-1.71.64-.4 1.67-.68 3.08-.84 0-.17-.02-.34-.07-.51-.05-.16-.12-.3-.22-.43s-.22-.22-.38-.3c-.15-.06-.34-.1-.58-.1-.34 0-.68.07-1 .2s-.63.29-.93.47l-.59-1.08c.39-.24.81-.45 1.28-.63.47-.17.99-.26 1.54-.26.86 0 1.51.25 1.93.76s.63 1.25.63 2.21v4.07h-1.32l-.12-.76h-.05c-.3.27-.63.48-.98.66s-.73.27-1.14.27c-.61 0-1.1-.19-1.48-.56-.38-.36-.57-.85-.57-1.46zm1.57-.12c0 .3.09.53.27.67.19.14.42.21.71.21.28 0 .54-.07.77-.2s.48-.31.73-.56v-1.54c-.47.06-.86.13-1.18.23-.31.09-.57.19-.76.31s-.33.25-.41.4c-.09.15-.13.31-.13.48zm6.29-3.63h-.98v-1.2l1.06-.07.2-1.88h1.34v1.88h1.75v1.27h-1.75v3.28c0 .8.32 1.2.97 1.2.12 0 .24-.01.37-.04.12-.03.24-.07.34-.11l.28 1.19c-.19.06-.4.12-.64.17-.23.05-.49.08-.76.08-.4 0-.74-.06-1.02-.18-.27-.13-.49-.3-.67-.52-.17-.21-.3-.48-.37-.78-.08-.3-.12-.64-.12-1.01zm4.36 2.17c0-.56.09-1.06.27-1.51s.41-.83.71-1.14c.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.37c.08.62.29 1.1.65 1.44.36.33.82.5 1.38.5.3 0 .58-.04.84-.13.25-.09.51-.21.76-.37l.54 1.01c-.32.21-.69.39-1.09.53s-.82.21-1.26.21c-.47 0-.92-.08-1.33-.25-.41-.16-.77-.4-1.08-.7-.3-.31-.54-.69-.72-1.13-.17-.44-.26-.95-.26-1.52zm4.61-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.08.45-.31.29-.5.73-.57 1.3zm3.01 2.23c.31.24.61.43.92.57.3.13.63.2.98.2.38 0 .65-.08.83-.23s.27-.35.27-.6c0-.14-.05-.26-.13-.37-.08-.1-.2-.2-.34-.28-.14-.09-.29-.16-.47-.23l-.53-.22c-.23-.09-.46-.18-.69-.3-.23-.11-.44-.24-.62-.4s-.33-.35-.45-.55c-.12-.21-.18-.46-.18-.75 0-.61.23-1.1.68-1.49.44-.38 1.06-.57 1.83-.57.48 0 .91.08 1.29.25s.71.36.99.57l-.74.98c-.24-.17-.49-.32-.73-.42-.25-.11-.51-.16-.78-.16-.35 0-.6.07-.76.21-.17.15-.25.33-.25.54 0 .14.04.26.12.36s.18.18.31.26c.14.07.29.14.46.21l.54.19c.23.09.47.18.7.29s.44.24.64.4c.19.16.34.35.46.58.11.23.17.5.17.82 0 .3-.06.58-.17.83-.12.26-.29.48-.51.68-.23.19-.51.34-.84.45-.34.11-.72.17-1.15.17-.48 0-.95-.09-1.41-.27-.46-.19-.86-.41-1.2-.68z" fill="#535353"/></g></svg>"/></a></p><div><h3 id="citeas">Cite this article</h3><p>Hinte, L.C., Castellano-Castillo, D., Ghosh, A. <i>et al.</i> Adipose tissue retains an epigenetic memory of obesity after weight loss.
                    <i>Nature</i> <b>636</b>, 457–465 (2024). https://doi.org/10.1038/s41586-024-08165-7</p><p><a data-test="citation-link" data-track="click" data-track-action="download article citation" data-track-label="link" data-track-external="" rel="nofollow" href="https://citation-needed.springer.com/v2/references/10.1038/s41586-024-08165-7?format=refman&amp;flavour=citation">Download citation<svg width="16" height="16" focusable="false" role="img" aria-hidden="true"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-download-medium"></use></svg></a></p><ul data-test="publication-history"><li><p>Received<span>: </span><span><time datetime="2023-01-20">20 January 2023</time></span></p></li><li><p>Accepted<span>: </span><span><time datetime="2024-10-07">07 October 2024</time></span></p></li><li><p>Published<span>: </span><span><time datetime="2024-11-18">18 November 2024</time></span></p></li><li><p>Issue Date<span>: </span><span><time datetime="2024-12-12">12 December 2024</time></span></p></li><li><p><abbr title="Digital Object Identifier">DOI</abbr><span>: </span><span>https://doi.org/10.1038/s41586-024-08165-7</span></p></li></ul></div></div></div></div></section>
            </div></div>
  </body>
</html>
