<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <title>James Routley | Feed</title>
    <link
      rel="stylesheet"
      type="text/css"
      href="../styles.css"
      media="screen"
    />
  </head>
  <body>
    <a href="/index.html">Back</a>
    <a href="https://github.com/google-deepmind/alphafold3">Original</a>
    <h1>AlphaFold 3 Code</h1>
    
    <div id="readability-page-1" class="page"><div data-hpc="true"><article itemprop="text"><p dir="auto"><a target="_blank" rel="noopener noreferrer" href="https://github.com/google-deepmind/alphafold3/blob/main/docs/header.jpg"><img src="https://github.com/google-deepmind/alphafold3/raw/main/docs/header.jpg" alt="header"/></a></p>

<p dir="auto">This package provides an implementation of the inference pipeline of AlphaFold
3. See below for how to access the model parameters. You may only use AlphaFold
3 model parameters if received directly from Google. Use is subject to these
<a href="https://github.com/google-deepmind/alphafold3/blob/main/WEIGHTS_TERMS_OF_USE.md">terms of use</a>.</p>
<p dir="auto">Any publication that discloses findings arising from using this source code, the
model parameters or outputs produced by those should <a href="#citing-this-work">cite</a>
the
<a href="https://doi.org/10.1038/s41586-024-07487-w" rel="nofollow">Accurate structure prediction of biomolecular interactions with AlphaFold 3</a>
paper.</p>
<p dir="auto">Please also refer to the Supplementary Information for a detailed description of
the method.</p>
<p dir="auto">AlphaFold 3 is also available at
<a href="https://alphafoldserver.com" rel="nofollow">alphafoldserver.com</a> for non-commercial use,
though with a more limited set of ligands and covalent modifications.</p>
<p dir="auto">If you have any questions, please contact the AlphaFold team at
<a href="mailto:alphafold@google.com">alphafold@google.com</a>.</p>
<div dir="auto"><h2 tabindex="-1" dir="auto">Obtaining Model Parameters</h2><a id="user-content-obtaining-model-parameters" aria-label="Permalink: Obtaining Model Parameters" href="#obtaining-model-parameters"><svg viewBox="0 0 16 16" version="1.1" width="16" height="16" aria-hidden="true"><path d="m7.775 3.275 1.25-1.25a3.5 3.5 0 1 1 4.95 4.95l-2.5 2.5a3.5 3.5 0 0 1-4.95 0 .751.751 0 0 1 .018-1.042.751.751 0 0 1 1.042-.018 1.998 1.998 0 0 0 2.83 0l2.5-2.5a2.002 2.002 0 0 0-2.83-2.83l-1.25 1.25a.751.751 0 0 1-1.042-.018.751.751 0 0 1-.018-1.042Zm-4.69 9.64a1.998 1.998 0 0 0 2.83 0l1.25-1.25a.751.751 0 0 1 1.042.018.751.751 0 0 1 .018 1.042l-1.25 1.25a3.5 3.5 0 1 1-4.95-4.95l2.5-2.5a3.5 3.5 0 0 1 4.95 0 .751.751 0 0 1-.018 1.042.751.751 0 0 1-1.042.018 1.998 1.998 0 0 0-2.83 0l-2.5 2.5a1.998 1.998 0 0 0 0 2.83Z"></path></svg></a></div>
<p dir="auto">This repository contains all necessary code for AlphaFold 3 inference. To
request access to the AlphaFold 3 model parameters, please complete
<a href="https://forms.gle/svvpY4u2jsHEwWYS6" rel="nofollow">this form</a>. Access will be granted at
Google DeepMind’s sole discretion. We will aim to respond to requests within 2–3
business days. You may only use AlphaFold 3 model parameters if received
directly from Google. Use is subject to these
<a href="https://github.com/google-deepmind/alphafold3/blob/main/WEIGHTS_TERMS_OF_USE.md">terms of use</a>.</p>
<div dir="auto"><h2 tabindex="-1" dir="auto">Installation and Running Your First Prediction</h2><a id="user-content-installation-and-running-your-first-prediction" aria-label="Permalink: Installation and Running Your First Prediction" href="#installation-and-running-your-first-prediction"><svg viewBox="0 0 16 16" version="1.1" width="16" height="16" aria-hidden="true"><path d="m7.775 3.275 1.25-1.25a3.5 3.5 0 1 1 4.95 4.95l-2.5 2.5a3.5 3.5 0 0 1-4.95 0 .751.751 0 0 1 .018-1.042.751.751 0 0 1 1.042-.018 1.998 1.998 0 0 0 2.83 0l2.5-2.5a2.002 2.002 0 0 0-2.83-2.83l-1.25 1.25a.751.751 0 0 1-1.042-.018.751.751 0 0 1-.018-1.042Zm-4.69 9.64a1.998 1.998 0 0 0 2.83 0l1.25-1.25a.751.751 0 0 1 1.042.018.751.751 0 0 1 .018 1.042l-1.25 1.25a3.5 3.5 0 1 1-4.95-4.95l2.5-2.5a3.5 3.5 0 0 1 4.95 0 .751.751 0 0 1-.018 1.042.751.751 0 0 1-1.042.018 1.998 1.998 0 0 0-2.83 0l-2.5 2.5a1.998 1.998 0 0 0 0 2.83Z"></path></svg></a></div>
<p dir="auto">See the <a href="https://github.com/google-deepmind/alphafold3/blob/main/docs/installation.md">installation documentation</a>.</p>
<p dir="auto">Once you have installed AlphaFold 3, you can test your setup using e.g. the
following input JSON file named <code>alphafold_input.json</code>:</p>
<div dir="auto" data-snippet-clipboard-copy-content="{
  &#34;name&#34;: &#34;2PV7&#34;,
  &#34;sequences&#34;: [
    {
      &#34;protein&#34;: {
        &#34;id&#34;: [&#34;A&#34;, &#34;B&#34;],
        &#34;sequence&#34;: &#34;GMRESYANENQFGFKTINSDIHKIVIVGGYGKLGGLFARYLRASGYPISILDREDWAVAESILANADVVIVSVPINLTLETIERLKPYLTENMLLADLTSVKREPLAKMLEVHTGAVLGLHPMFGADIASMAKQVVVRCDGRFPERYEWLLEQIQIWGAKIYQTNATEHDHNMTYIQALRHFSTFANGLHLSKQPINLANLLALSSPIYRLELAMIGRLFAQDAELYADIIMDKSENLAVIETLKQTYDEALTFFENNDRQGFIDAFHKVRDWFGDYSEQFLKESRQLLQQANDLKQG&#34;
      }
    }
  ],
  &#34;modelSeeds&#34;: [1],
  &#34;dialect&#34;: &#34;alphafold3&#34;,
  &#34;version&#34;: 1
}"><pre>{
  <span>&#34;name&#34;</span>: <span><span>&#34;</span>2PV7<span>&#34;</span></span>,
  <span>&#34;sequences&#34;</span>: [
    {
      <span>&#34;protein&#34;</span>: {
        <span>&#34;id&#34;</span>: [<span><span>&#34;</span>A<span>&#34;</span></span>, <span><span>&#34;</span>B<span>&#34;</span></span>],
        <span>&#34;sequence&#34;</span>: <span><span>&#34;</span>GMRESYANENQFGFKTINSDIHKIVIVGGYGKLGGLFARYLRASGYPISILDREDWAVAESILANADVVIVSVPINLTLETIERLKPYLTENMLLADLTSVKREPLAKMLEVHTGAVLGLHPMFGADIASMAKQVVVRCDGRFPERYEWLLEQIQIWGAKIYQTNATEHDHNMTYIQALRHFSTFANGLHLSKQPINLANLLALSSPIYRLELAMIGRLFAQDAELYADIIMDKSENLAVIETLKQTYDEALTFFENNDRQGFIDAFHKVRDWFGDYSEQFLKESRQLLQQANDLKQG<span>&#34;</span></span>
      }
    }
  ],
  <span>&#34;modelSeeds&#34;</span>: [<span>1</span>],
  <span>&#34;dialect&#34;</span>: <span><span>&#34;</span>alphafold3<span>&#34;</span></span>,
  <span>&#34;version&#34;</span>: <span>1</span>
}</pre></div>
<p dir="auto">You can then run AlphaFold 3 using the following command:</p>
<div data-snippet-clipboard-copy-content="docker run -it \
    --volume $HOME/af_input:/root/af_input \
    --volume $HOME/af_output:/root/af_output \
    --volume &lt;MODEL_PARAMETERS_DIR&gt;:/root/models \
    --volume &lt;DATABASES_DIR&gt;:/root/public_databases \
    --gpus all \
    alphafold3 \
python run_alphafold.py \
    --json_path=/root/af_input/fold_input.json \
    --model_dir=/root/models \
    --output_dir=/root/af_output"><pre><code>docker run -it \
    --volume $HOME/af_input:/root/af_input \
    --volume $HOME/af_output:/root/af_output \
    --volume &lt;MODEL_PARAMETERS_DIR&gt;:/root/models \
    --volume &lt;DATABASES_DIR&gt;:/root/public_databases \
    --gpus all \
    alphafold3 \
python run_alphafold.py \
    --json_path=/root/af_input/fold_input.json \
    --model_dir=/root/models \
    --output_dir=/root/af_output
</code></pre></div>

<p dir="auto">See the <a href="https://github.com/google-deepmind/alphafold3/blob/main/docs/input.md">input documentation</a>.</p>

<p dir="auto">See the <a href="https://github.com/google-deepmind/alphafold3/blob/main/docs/output.md">output documentation</a>.</p>

<p dir="auto">See the <a href="https://github.com/google-deepmind/alphafold3/blob/main/docs/performance.md">performance documentation</a>.</p>

<p dir="auto">Known issues are documented in the
<a href="https://github.com/google-deepmind/alphafold3/blob/main/docs/known_issues.md">known issues documentation</a>.</p>
<p dir="auto">Please
<a href="https://github.com/google-deepmind/alphafold3/issues/new/choose">create an issue</a>
if it is not already listed in <a href="https://github.com/google-deepmind/alphafold3/blob/main/docs/known_issues.md">Known Issues</a> or in the
<a href="https://github.com/google-deepmind/alphafold3/issues">issues tracker</a>.</p>

<p dir="auto">Any publication that discloses findings arising from using this source code, the
model parameters or outputs produced by those should cite:</p>
<div dir="auto" data-snippet-clipboard-copy-content="@article{Abramson2024,
  author  = {Abramson, Josh and Adler, Jonas and Dunger, Jack and Evans, Richard and Green, Tim and Pritzel, Alexander and Ronneberger, Olaf and Willmore, Lindsay and Ballard, Andrew J. and Bambrick, Joshua and Bodenstein, Sebastian W. and Evans, David A. and Hung, Chia-Chun and O’Neill, Michael and Reiman, David and Tunyasuvunakool, Kathryn and Wu, Zachary and Žemgulytė, Akvilė and Arvaniti, Eirini and Beattie, Charles and Bertolli, Ottavia and Bridgland, Alex and Cherepanov, Alexey and Congreve, Miles and Cowen-Rivers, Alexander I. and Cowie, Andrew and Figurnov, Michael and Fuchs, Fabian B. and Gladman, Hannah and Jain, Rishub and Khan, Yousuf A. and Low, Caroline M. R. and Perlin, Kuba and Potapenko, Anna and Savy, Pascal and Singh, Sukhdeep and Stecula, Adrian and Thillaisundaram, Ashok and Tong, Catherine and Yakneen, Sergei and Zhong, Ellen D. and Zielinski, Michal and Žídek, Augustin and Bapst, Victor and Kohli, Pushmeet and Jaderberg, Max and Hassabis, Demis and Jumper, John M.},
  journal = {Nature},
  title   = {Accurate structure prediction of biomolecular interactions with AlphaFold 3},
  year    = {2024},
  volume  = {630},
  number  = {8016},
  pages   = {493–-500},
  doi     = {10.1038/s41586-024-07487-w}
}"><pre><span>@article</span>{<span>Abramson2024</span>,
  <span>author</span>  = <span><span>{</span>Abramson, Josh and Adler, Jonas and Dunger, Jack and Evans, Richard and Green, Tim and Pritzel, Alexander and Ronneberger, Olaf and Willmore, Lindsay and Ballard, Andrew J. and Bambrick, Joshua and Bodenstein, Sebastian W. and Evans, David A. and Hung, Chia-Chun and O’Neill, Michael and Reiman, David and Tunyasuvunakool, Kathryn and Wu, Zachary and Žemgulytė, Akvilė and Arvaniti, Eirini and Beattie, Charles and Bertolli, Ottavia and Bridgland, Alex and Cherepanov, Alexey and Congreve, Miles and Cowen-Rivers, Alexander I. and Cowie, Andrew and Figurnov, Michael and Fuchs, Fabian B. and Gladman, Hannah and Jain, Rishub and Khan, Yousuf A. and Low, Caroline M. R. and Perlin, Kuba and Potapenko, Anna and Savy, Pascal and Singh, Sukhdeep and Stecula, Adrian and Thillaisundaram, Ashok and Tong, Catherine and Yakneen, Sergei and Zhong, Ellen D. and Zielinski, Michal and Žídek, Augustin and Bapst, Victor and Kohli, Pushmeet and Jaderberg, Max and Hassabis, Demis and Jumper, John M.<span>}</span></span>,
  <span>journal</span> = <span><span>{</span>Nature<span>}</span></span>,
  <span>title</span>   = <span><span>{</span>Accurate structure prediction of biomolecular interactions with AlphaFold 3<span>}</span></span>,
  <span>year</span>    = <span><span>{</span>2024<span>}</span></span>,
  <span>volume</span>  = <span><span>{</span>630<span>}</span></span>,
  <span>number</span>  = <span><span>{</span>8016<span>}</span></span>,
  <span>pages</span>   = <span><span>{</span>493–-500<span>}</span></span>,
  <span>doi</span>     = <span><span>{</span>10.1038/s41586-024-07487-w<span>}</span></span>
}</pre></div>

<p dir="auto">AlphaFold 3&#39;s release was made possible by the invaluable contributions of the
following people:</p>
<p dir="auto">Andrew Cowie, Bella Hansen, Charlie Beattie, Chris Jones, Grace Margand,
Jacob Kelly, James Spencer, Josh Abramson, Kathryn Tunyasuvunakool, Kuba Perlin,
Lindsay Willmore, Max Bileschi, Molly Beck, Oleg Kovalevskiy,
Sebastian Bodenstein, Sukhdeep Singh, Tim Green, Toby Sargeant, Uchechi Okereke,
Yotam Doron, and Augustin Žídek (engineering lead).</p>
<p dir="auto">We also extend our gratitude to our collaborators at Google and Isomorphic Labs.</p>
<p dir="auto">AlphaFold 3 uses the following separate libraries and packages:</p>
<ul dir="auto">
<li><a href="https://github.com/abseil/abseil-cpp">abseil-cpp</a> and
<a href="https://github.com/abseil/abseil-py">abseil-py</a></li>
<li><a href="https://github.com/deepmind/chex">Chex</a></li>
<li><a href="https://www.docker.com" rel="nofollow">Docker</a></li>
<li><a href="https://github.com/PDB-REDO/dssp">DSSP</a></li>
<li><a href="https://github.com/EddyRivasLab/hmmer">HMMER Suite</a></li>
<li><a href="https://github.com/deepmind/dm-haiku">Haiku</a></li>
<li><a href="https://github.com/google/jax/">JAX</a></li>
<li><a href="https://github.com/jax-ml/jax-triton">jax-triton</a></li>
<li><a href="https://github.com/patrick-kidger/jaxtyping">jaxtyping</a></li>
<li><a href="https://github.com/pdb-redo/libcifpp">libcifpp</a></li>
<li><a href="https://github.com/numpy/numpy">NumPy</a></li>
<li><a href="https://github.com/pybind/pybind11">pybind11</a> and
<a href="https://github.com/pybind/pybind11_abseil">pybind11_abseil</a></li>
<li><a href="https://github.com/rdkit/rdkit">RDKit</a></li>
<li><a href="https://github.com/deepmind/tree">Tree</a></li>
<li><a href="https://github.com/triton-lang/triton">Triton</a></li>
<li><a href="https://github.com/tqdm/tqdm">tqdm</a></li>
</ul>
<p dir="auto">We thank all their contributors and maintainers!</p>

<p dir="auto">If you have any questions not covered in this overview, please contact the
AlphaFold team at <a href="mailto:alphafold@google.com">alphafold@google.com</a>.</p>
<p dir="auto">We would love to hear your feedback and understand how AlphaFold 3 has been
useful in your research. Share your stories with us at
<a href="mailto:alphafold@google.com">alphafold@google.com</a>.</p>

<p dir="auto">This is not an officially supported Google product.</p>
<p dir="auto">Copyright 2024 DeepMind Technologies Limited.</p>
<div dir="auto"><h3 tabindex="-1" dir="auto">AlphaFold 3 Source Code and Model Parameters</h3><a id="user-content-alphafold-3-source-code-and-model-parameters" aria-label="Permalink: AlphaFold 3 Source Code and Model Parameters" href="#alphafold-3-source-code-and-model-parameters"><svg viewBox="0 0 16 16" version="1.1" width="16" height="16" aria-hidden="true"><path d="m7.775 3.275 1.25-1.25a3.5 3.5 0 1 1 4.95 4.95l-2.5 2.5a3.5 3.5 0 0 1-4.95 0 .751.751 0 0 1 .018-1.042.751.751 0 0 1 1.042-.018 1.998 1.998 0 0 0 2.83 0l2.5-2.5a2.002 2.002 0 0 0-2.83-2.83l-1.25 1.25a.751.751 0 0 1-1.042-.018.751.751 0 0 1-.018-1.042Zm-4.69 9.64a1.998 1.998 0 0 0 2.83 0l1.25-1.25a.751.751 0 0 1 1.042.018.751.751 0 0 1 .018 1.042l-1.25 1.25a3.5 3.5 0 1 1-4.95-4.95l2.5-2.5a3.5 3.5 0 0 1 4.95 0 .751.751 0 0 1-.018 1.042.751.751 0 0 1-1.042.018 1.998 1.998 0 0 0-2.83 0l-2.5 2.5a1.998 1.998 0 0 0 0 2.83Z"></path></svg></a></div>
<p dir="auto">The AlphaFold 3 source code is licensed under the Creative Commons
Attribution-Non-Commercial ShareAlike International License, Version 4.0
(CC-BY-NC-SA 4.0) (the &#34;License&#34;); you may not use this file except in
compliance with the License. You may obtain a copy of the License at
<a href="https://github.com/google-deepmind/alphafold3/blob/main/LICENSE">https://github.com/google-deepmind/alphafold3/blob/main/LICENSE</a>.</p>
<p dir="auto">The AlphaFold 3 model parameters are made available under the
<a href="https://github.com/google-deepmind/alphafold3/blob/main/WEIGHTS_TERMS_OF_USE.md">AlphaFold 3 Model Parameters Terms of Use</a>
(the &#34;Terms&#34;); you may not use these except in compliance with the Terms. You
may obtain a copy of the Terms at
<a href="https://github.com/google-deepmind/alphafold3/blob/main/WEIGHTS_TERMS_OF_USE.md">https://github.com/google-deepmind/alphafold3/blob/main/WEIGHTS_TERMS_OF_USE.md</a>.</p>
<p dir="auto">Unless required by applicable law, AlphaFold 3 and its output are distributed on
an &#34;AS IS&#34; BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
or implied. You are solely responsible for determining the appropriateness of
using AlphaFold 3, or using or distributing its source code or output, and
assume any and all risks associated with such use or distribution and your
exercise of rights and obligations under the relevant terms. Output are
predictions with varying levels of confidence and should be interpreted
carefully. Use discretion before relying on, publishing, downloading or
otherwise using the AlphaFold 3 Assets.</p>
<p dir="auto">AlphaFold 3 and its output are for theoretical modeling only. They are not
intended, validated, or approved for clinical use. You should not use the
AlphaFold 3 or its output for clinical purposes or rely on them for medical or
other professional advice. Any content regarding those topics is provided for
informational purposes only and is not a substitute for advice from a qualified
professional. See the relevant terms for the specific language governing
permissions and limitations under the terms.</p>

<p dir="auto">Use of the third-party software, libraries or code referred to in the
<a href="#acknowledgements">Acknowledgements</a> section above may be governed by separate
terms and conditions or license provisions. Your use of the third-party
software, libraries or code is subject to any such terms and you should check
that you can comply with any applicable restrictions or terms and conditions
before use.</p>
<div dir="auto"><h3 tabindex="-1" dir="auto">Mirrored and Reference Databases</h3><a id="user-content-mirrored-and-reference-databases" aria-label="Permalink: Mirrored and Reference Databases" href="#mirrored-and-reference-databases"><svg viewBox="0 0 16 16" version="1.1" width="16" height="16" aria-hidden="true"><path d="m7.775 3.275 1.25-1.25a3.5 3.5 0 1 1 4.95 4.95l-2.5 2.5a3.5 3.5 0 0 1-4.95 0 .751.751 0 0 1 .018-1.042.751.751 0 0 1 1.042-.018 1.998 1.998 0 0 0 2.83 0l2.5-2.5a2.002 2.002 0 0 0-2.83-2.83l-1.25 1.25a.751.751 0 0 1-1.042-.018.751.751 0 0 1-.018-1.042Zm-4.69 9.64a1.998 1.998 0 0 0 2.83 0l1.25-1.25a.751.751 0 0 1 1.042.018.751.751 0 0 1 .018 1.042l-1.25 1.25a3.5 3.5 0 1 1-4.95-4.95l2.5-2.5a3.5 3.5 0 0 1 4.95 0 .751.751 0 0 1-.018 1.042.751.751 0 0 1-1.042.018 1.998 1.998 0 0 0-2.83 0l-2.5 2.5a1.998 1.998 0 0 0 0 2.83Z"></path></svg></a></div>
<p dir="auto">The following databases have been: (1) mirrored by Google DeepMind; and (2) in
part, included with the inference code package for testing purposes, and are
available with reference to the following:</p>
<ul dir="auto">
<li><a href="https://bfd.mmseqs.com/" rel="nofollow">BFD</a> (modified), by Steinegger M. and Söding J.,
modified by Google DeepMind, available under a
<a href="https://creativecommons.org/licenses/by/4.0/deed.en" rel="nofollow">Creative Commons Attribution 4.0 International License</a>.
See the Methods section of the
<a href="https://www.nature.com/articles/s41586-021-03828-1" rel="nofollow">AlphaFold proteome paper</a>
for details.</li>
<li><a href="https://wwpdb.org" rel="nofollow">PDB</a> (unmodified), by H.M. Berman et al., available free
of all copyright restrictions and made fully and freely available for both
non-commercial and commercial use under
<a href="https://creativecommons.org/publicdomain/zero/1.0/" rel="nofollow">CC0 1.0 Universal (CC0 1.0) Public Domain Dedication</a>.</li>
<li><a href="https://ftp.ebi.ac.uk/pub/databases/metagenomics/peptide_database/2022_05/README.txt" rel="nofollow">MGnify: v2022_05</a>
(unmodified), by Mitchell AL et al., available free of all copyright
restrictions and made fully and freely available for both non-commercial and
commercial use under
<a href="https://creativecommons.org/publicdomain/zero/1.0/" rel="nofollow">CC0 1.0 Universal (CC0 1.0) Public Domain Dedication</a>.</li>
<li><a href="https://www.uniprot.org/" rel="nofollow">UniProt: 2021_04</a> (unmodified), by The UniProt
Consortium, available under a
<a href="https://creativecommons.org/licenses/by/4.0/deed.en" rel="nofollow">Creative Commons Attribution 4.0 International License</a>.</li>
<li><a href="https://www.uniprot.org/" rel="nofollow">UniRef90: 2022_05</a> (unmodified) by The UniProt
Consortium, available under a
<a href="https://creativecommons.org/licenses/by/4.0/deed.en" rel="nofollow">Creative Commons Attribution 4.0 International License</a>.</li>
<li><a href="https://www.ncbi.nlm.nih.gov/nucleotide/" rel="nofollow">NT: 2023_02_23</a> (modified) See
the Supplementary Information of the
<a href="https://nature.com/articles/s41586-024-07487-w" rel="nofollow">AlphaFold 3 paper</a> for
details.</li>
<li><a href="https://rfam.org/" rel="nofollow">RFam: 14_4</a> (modified), by I. Kalvari et al., available
free of all copyright restrictions and made fully and freely available for
both non-commercial and commercial use under
<a href="https://creativecommons.org/publicdomain/zero/1.0/" rel="nofollow">CC0 1.0 Universal (CC0 1.0) Public Domain Dedication</a>.
See the Supplementary Information of the
<a href="https://nature.com/articles/s41586-024-07487-w" rel="nofollow">AlphaFold 3 paper</a> for
details.</li>
<li><a href="https://rnacentral.org/" rel="nofollow">RNACentral: 21_0</a> (modified), by The RNAcentral
Consortium available free of all copyright restrictions and made fully and
freely available for both non-commercial and commercial use under
<a href="https://creativecommons.org/publicdomain/zero/1.0/" rel="nofollow">CC0 1.0 Universal (CC0 1.0) Public Domain Dedication</a>.
See the Supplementary Information of the
<a href="https://nature.com/articles/s41586-024-07487-w" rel="nofollow">AlphaFold 3 paper</a> for
details.</li>
</ul>
</article></div></div>
  </body>
</html>
