<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <title>James Routley | Feed</title>
    <link
      rel="stylesheet"
      type="text/css"
      href="../styles.css"
      media="screen"
    />
  </head>
  <body>
    <a href="/index.html">Back</a>
    <a href="https://www.science.org/content/blog-post/als-protein-revealed">Original</a>
    <h1>An ALS Protein, Revealed</h1>
    
    <div id="readability-page-1" class="page"><article>
								<section>
									<p><a href="https://en.wikipedia.org/wiki/Amyotrophic_lateral_sclerosis">Amyotrophic lateral sclerosis</a> (ALS) is a terrible disease. No controversy in saying that - there is no treatment available, and as a motor neuron syndrome its end result is an early death as movement, speaking, swallowing, and finally breathing are inexorably affected. Less than ten per cent of the cases can be clearly traced to genetic background; the rest have no known cause (although genetics are highly likely to be involved in ways that we don&#39;t yet understand). There are environmental factors as well, but they&#39;re less clear than you&#39;d think. Head injury (such as the repeated head injuries of American football, boxing, and soccer) seems raise the chances of developing ALS and other neurodegenerative diseases, but the statistics on that are not as robust as you would have thought. Other studies, for example, have found ALS to be more associated with &#34;white-collar&#34; occupations not typically associated with repeated cranial trauma. </p>
<p>But what<em> is</em> that problem? Here&#39;s where it gets difficult. There are about twenty genes associated with the familial cases (and a few of the sporadic ones, when a mutation occurs), so there is not just one smoking gun genetic factor as with some other disorders. When you start looking at the neurons themselves, well, you&#39;re in the same space that you are with other neurodegenerative disorders, such as Alzheimers (or with frontotemporal dementia, FTD, which is related to ALS on a molecular level, as we&#39;ll see below. That is, you can see neurons dying, but it is extremely difficult to figure out what they&#39;re dying <em>from</em>. <a href="https://en.wikipedia.org/wiki/Amyotrophic_lateral_sclerosis#Biochemistry">Here you will find</a> a diagram with ten hypotheses about the disease mechanism, which gives you a good idea of where the field is.</p>
<p>The most obvious things you can determine, in fact, are probably mostly effects of neuronal death rather than the causes, and the causes themselves might start out in very subtle ways. The fact that ALS often manifests itself later in life speaks to that as well: whatever the problem is, it may well come on slowly in a mass of mixed genetic and environmental determinants. The end result is not subtle at all, of course, but that also means that the signs and clues that you&#39;re picking up at that point may well be from that late-stage environment. Plenty of metaphors recur when describing this problem, among them the idea that you should not attempt to put out a fire by removing the smoke and that crunched-in car fenders and broken glass are generally not key reasons for road accidents, even if you see them associated almost every time.</p>
<p>Among these obvious molecular signs of ALS (and of FTD) is the presence of aggregated clumps of a protein called <a href="https://en.wikipedia.org/wiki/TAR_DNA-binding_protein_43">TDP-43</a>. Over the years, it&#39;s become more apparent that these aggregates are seen first in the brain&#39;s motor cortex and in the brainstem and spinal cord, and then spread with time into other regions. It&#39;s quite possible that this spread takes place by <a href="https://en.wikipedia.org/wiki/Prion">prion-type</a> mechanisms, the sort of &#34;contagious protein misfolding&#34; that&#39;s seen in Creutzfel-Jacob and in mad cow disease. That argues for causation, but such TDP-43 aggregates are also found (to a lesser degree) in Alzheimer&#39;s and Parkinson&#39;s patients, which suggests on the other hand that neurons and glial cells under stress might tend to misfold proteins in general (or be unable to police the process as well as they need to).</p>
<p>TDP43 itself is an interesting beast: it has some regions that are optimized to bind to DNA and RNA species, some sequences that can localize it to mitrochondria or send it into the cell nucleus, an N-terminal domain that&#39;s involved in making dimers (and higher-order oligomers) with other TDP-43 molecules, a process that surely has effects on its other functions. It also has a disordered region that interacts with a number of protein partners as it adopts different conformations. Many of the mutations that are associated with ALS in in that last low-structure low-complexity part, which surely tells us something (although it&#39;s not quite clear what).</p>
<p><img src="https://blog.plover.com/do/10.1126/9e09da02-bee2-4154-9168-6a67c2115d0b/full/tdp43.png" alt="" width="400"/></p>
<p>Now for the first time we have <a href="https://www.nature.com/articles/s41586-021-04199-3">an experimental structure</a> (via cryo-EM) of what the aggregate looks like, and it has a lot of intriguing features. This is useful for ALS researchers, for people working on other protein folding diseases, and for people working on protein folding in general (recall the <a href="https://blog.plover.com/content/blog-post/more-protein-folding-progress---what-s-it-mean">recent successes</a> in predicting these things computationally). Shown is a slice through the aggregate from the cryo-EM data; the aggregates themselves have a broadly helical structure that extend up and down out of the plane shown. You can see the protein in rather startling detail - that particular slice shows you residues 282 through 360, and for example, can you pick out that short flat bright part, left of center, sticking up towards about ten o&#39;clock? That&#39;s the indole side chain of tryptophan 334. The blurrier part across the bottom and left of the sequences is a glycine-rich region, while the middle (with that tryptophan) is a hydrophobic stretch, and the zig-zag part across the top is a Q/N rich region.</p>
<p>These motifs repeat, giving you a gently twisting helical stack of TDP-43 molecules about 4.8 Å apart from each other. And that arrangement you see is an odd &#34;double spiral fold&#34; which has not been seen before in protein structures, from what I can tell. The low-structure part is the one with a lot of glycines in it, and these (which are like little universal joints for a protein) allow for the particular twists and turns that let this aggregate form. The very flexibility that TDP-43 needs to do its job comes back in an evil form, because once this double-spiral arrangement starts up, all flexibility is lost and the aggregated protein just starts piling up in an orderly, insoluble, useless mass.</p>
<p>The same structure was seen in samples from different brain regions and in different patients, so this is very likely the universal TDP-43 aggregate we&#39;re looking at (which goes for frontotemporal dementia as well). What it <em>doesn&#39;t</em> look like are the TDP-43 fibers that you can form in vitro from the isolated protein! Any hypotheses based on those lab-generated fibrils are going to have to go out the window, because the real one differs in its folds, its individual amino acid interactions, and in its whole secondary structure. One thing this paper does is throw down the challenge of reproducing this real disease aggregate structure <em>in vitro</em>, and you can bet that there are people working on that right now.</p>
<p>What about those known mutations in the low-structure region? There are about 24 of them known, and according to the paper, 18 of them are compatible with this structure (and, in fact, might well have a role it making it somewhat more likely to form?) The A315E mutation, for example, can already be rationalized in that way, because it seems to lead to a salt-bridge possibility that would make the whole structure more stable and quite possibly more easy to assemble in the first place. But that leaves six that don&#39;t quite fit. Most of these could be accommodated by some local rearrangements in the structure, but one of the (S332N) just can&#39;t be worked in at all. It will be very useful to see what the TDP-43 fibrils with these mutations look like, particularly that last one.</p>
<p>This new structure immediately sets off a lot of good experiments around the prion-propagation hypothesis as well. That stacked-helix structure certainly fits into the idea, and further work will need to characterize more samples from ALS and FTD patients to see what different varieties of this double-spiral fold might exist and how likely they are to induce wild-type proteins to fall in with them. Then you have to start thinking about whether this structural knowledge gives you any hopes of interrupting the whole process: is there a small molecule that could be built to stop it all? People have certainly tried phenotypic screens to find such agents, without much success. But it might be time to do some designing from scratch, as difficult as that is.</p>
<p>Finally, let&#39;s stop to think about the computational aspect. Neither AlphaFold nor RosettaFold would have given you the structure above. And that&#39;s largely because these programs did not know that the double-spiral-fold was even possible. Remember, the protein-folding software works by analogy to known structures, which for the bulk of proteins can take you quite far (with ingenious software and lots of processing power). But they will not create new protein folds <em>ex nihilo</em>. Now that we know that this structure exists, the software can take this possibility into account, and it&#39;ll be interesting to see what it turns up now that the lights have come on in this direction. But experimental data, once again, have to come through - and they just have.</p>


								</section>
							</article></div>
  </body>
</html>
