<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <title>James Routley | Feed</title>
    <link
      rel="stylesheet"
      type="text/css"
      href="../styles.css"
      media="screen"
    />
  </head>
  <body>
    <a href="/index.html">Back</a>
    <a href="https://www.nature.com/articles/s41467-023-43885-w">Original</a>
    <h1>Ants treat injured nestmates with antimicrobial compound in ant society</h1>
    
    <div id="readability-page-1" class="page"><div id="Sec7-content"><p>The research conducted in this study complies with all relevant ethical regulations and was approved by the park management of Office Ivoirien des Parcs et Réserves (OIPR) in Côte d’Ivoire as part of the bilateral research agreement between Germany (represented by the University of Würzburg) and Côte d’Ivoire (represented by OIPR). The ants collected for this study are part of the bilateral research agreement under research permit number N°018 / MINEDD / OIPR / DZ.</p><h3 id="Sec8">Experimental design</h3><p>The study was conducted in a humid savannah woodland located in the Comoé National Park, northern Côte d’Ivoire (Ivory Coast), at the Comoé National Park Research Station (8°46’N, 3°47’W)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 31" title="Konaté, S. &amp; Kampmann, D. Biodiversity atlas of West Africa, 3: Côte d’Ivoire. (Abidjan &amp; Frankfurt am Main, 2010)." href="https://www.nature.com/articles/s41467-023-43885-w#ref-CR31" id="ref-link-section-d111212994e1653">31</a></sup>. Experiments, observations, and sample collections in the Comoé National Park were carried out from April to June 2018, February, April to June and September to October 2019, April 2020 and October 2022. <i>Megaponera analis</i> is found throughout sub-Saharan Africa from 25°S to 12°N<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 32" title="Schmidt, C. A. &amp; Shattuck, S. O. The higher classification of the ant subfamily ponerinae (hymenoptera: formicidae), with a review of ponerine ecology and behavior. Zootaxa 3817, 1–+ (2014)." href="https://www.nature.com/articles/s41467-023-43885-w#ref-CR32" id="ref-link-section-d111212994e1660">32</a></sup> and known to show monophasic allometry within its worker sizes<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 33" title="Villet, M. H. Division of labor in the matabele ant Megaponera foetens (Fabr) (Hymenoptera-Formicidae). Ethol. Ecol. Evol. 2, 397–417 (1990)." href="https://www.nature.com/articles/s41467-023-43885-w#ref-CR33" id="ref-link-section-d111212994e1664">33</a></sup>. We thus divided the workers into majors (head width &gt; than 2.40 mm), minors (head width &lt;1.99 mm), and intermediates (head width 2.40 − 1.99 mm). All experiments were carried out on the minor caste, the individuals most frequently injured<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 2" title="Frank, E. T. et al. Saving the injured: rescue behavior in the termite hunting ant Megaponera analis. Sci. Adv. 3, e1602187 (2017)." href="https://www.nature.com/articles/s41467-023-43885-w#ref-CR2" id="ref-link-section-d111212994e1668">2</a></sup>. All field studies were conducted in accordance with local legislation and permission by the Office Ivoirien des Parcs et Réserves (OIPR).</p><h3 id="Sec9">Laboratory colonies</h3><p>Eleven colonies, including queen and brood (colony size 1083 ± 258 ants), were excavated and placed in artificial nests in the field stations laboratory. PVC nests (30x20x10 cm) were connected to a 1x1m feeding arena. The ground surface was covered with soil from the surrounding area (up to a height of 2 cm). Colonies were fed by placing in the feeding arena <i>Macrotermes bellicosus</i> termites collected from the surrounding area. These termites were found by scouts and triggered raiding behavior. The laboratory windows were kept open to maintain a natural humidity, temperature, and day-night cycle (light regime).</p><h3 id="Sec10">Survival of injured ants</h3><p>To quantify the lethality of various types of pathogens, workers were injured by a sterile cut in the middle of the femur on the hind leg and had the fresh wound submerged for 2 s in a 10 μL phosphate-buffered saline (PBS) solution with a known pathogen concentration (~10<sup>6</sup> bacteria of either the mix of soil pathogens or isolated pathogen). Afterwards the injured ants were placed inside a cylindrical glass container with a diameter of 3 cm and a height of 5 cm. Before the experiments, the glass containers were filled with 1 cm of surface soil and placed for 3 h at 220 °C in an oven together with the forceps and scissors for sterilization. Nest-like humidity was created by moistening the soil with 1 mL of sterilized water (boiled for 10 min) and covered with aluminum foil. The isolation experiments were conducted at 24 °C in a sterile room. To test for mortality, the injured ant was checked upon once per hour for the next 36 h, if no reaction was observed even after shaking the container the ant was classified as dead. To ensure replicability of the survival experiments a negative (sterile PBS solution) and positive control (using a sterile PBS solution with 0.1 optical density (OD) of <i>Pseudomonas aeruginosa</i> (PSE)) were always included during each survival experiment. The sample sizes were PBS: <i>n</i> = 104; PSE 0.1: <i>n</i> = 61; PSE 0.05: <i>n</i> = 15 (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41467-023-43885-w#Fig2">2a</a> and Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41467-023-43885-w#MOESM1">3</a>). For all sample sizes and statistics see Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41467-023-43885-w#MOESM1">2</a>.</p><p>To test if wound care by nestmates reduces the mortality of infected ants, we first marked 10 ants per colony during a raid. 24 h later, all marked ants were injured in the same way as in the isolation trial. Afterwards, the wound of the injured ant was either exposed to a sterile PBS solution (sterile), a sterile PBS solution containing 0.05 OD of <i>P. aeruginosa</i> or a sterile PBS solution containing 0.1 OD of surface soil pathogens (grown on agar plates). Isolation trials with the same treatments were always conducted in parallel. Sample size of in nest survival experiments: PBS: <i>n</i> = 12; PSE 0.05: <i>n</i> = 12; Soil <i>n</i> = 18 (Figs. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41467-023-43885-w#Fig1">1</a> and <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41467-023-43885-w#Fig2">2a</a>). For all sample sizes and statistics see Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41467-023-43885-w#MOESM1">2</a>.</p><p>To test the importance of the Metapleural gland (MG) secretions we divided three colonies (~1000 ants) into two equally sized sub-colonies including brood. The queens were left with a small retinue of 20 workers in a separate container. For each colony we plugged the MG opening of all workers of one of the sub-colonies with acrylic color, while for the other sub-colony acrylic color was placed on the thorax of the workers. Afterwards the experimental procedure was identical to the wound care experiment described above. Eight individuals were removed from each sub-colony, marked, and wounded and the wound exposed to either a sterile PBS solution (<i>n</i> = 4 per sub-colony) or a sterile PBS solution containing 0.05 OD of <i>P. aeruginosa</i> (<i>n</i> = 4 per sub-colony). Sample sizes were thus in total: Sterile: <i>n</i> = 12; Infected: <i>n</i> = 12 for both treatments (with and without plugged MG opening; Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41467-023-43885-w#Fig4">4</a>). In addition, to quantify any effects of the plugging of the MG opening on individuals, a parallel experiment with <i>n</i> = 12 Sterile and <i>n</i> = 12 Infected ants with or without plugged MG opening was run in isolation (Supplementary fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41467-023-43885-w#MOESM1">5</a> and Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41467-023-43885-w#MOESM1">6</a>).</p><p>The pathogen concentration was measured by optical density (OD) using a portable Ultrospec 10 cell density meter (Biochrom) with sterile PBS as solvent. For the soil pathogens, we collected surface soil in the surrounding area of the nest and grew it over 36 h on agar plates (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41467-023-43885-w#MOESM1">1c</a>). For <i>P. aeruginosa</i> we created cultures of the isolated strains (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41467-023-43885-w#MOESM1">1d</a>) in the field lab from frozen samples kept in Tryptic Soy Broth (TSB) medium with 25% glycerol (stored and transported at −23 °C). After replating the bacterial culture once on a fresh plate, we waited 16 h before applying the pathogen on fresh wounds. For all experiments, we used trypticase soy agar (TSA) plates to culture the bacteria.</p><h3 id="Sec11">Treatment of wounds by nestmates</h3><p>To quantify the wound care behaviors inside the nest, we filmed the ants using a Panasonic HC-X1000 and analyzed the videos using VLC media player v.3.0.16 Vetinari (intel 64 bit). The wound of the injured ant was either exposed to a sterile PBS solution (Sterile) or a sterile PBS solution containing 0.05 OD of <i>P. aeruginosa</i> (Infected).</p><p>All manipulated ants were placed in front of the nest entrance directly after a raid and the nest was filmed for the subsequent 24 h. Only one trial was conducted per colony with a total of four injured ants per colony (two sterile and two infected), in a total of three colonies (i.e., <i>n</i> = 6 infected ants and <i>n</i> = 6 sterile ants, Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41467-023-43885-w#Fig3">3b, c</a>). The observed wound care behaviors were classified into two categories: (1) wound care: a nestmate cleans the open wound with its mouthparts; (2) metapleural gland (MG) care: a nestmate collects MG secretions either from its own gland (Supplementary Movie <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41467-023-43885-w#MOESM4">1</a>) or from the injured ant (Supplementary Movie <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41467-023-43885-w#MOESM5">2</a>) in its mouth before caring for the wound. These behaviors were quantified for the first 24 h and summarized in 10 min intervals.</p><h3 id="Sec12">Sample collection protocol for chemical, genetic, and microbial analyses</h3><p>To quantify the cuticular hydrocarbon (CHC) profile (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41467-023-43885-w#MOESM1">6</a>), differential gene expression (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41467-023-43885-w#Fig5">5</a>), and pathogen load in the thorax content (Figs. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41467-023-43885-w#Fig1">1</a>a and <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41467-023-43885-w#Fig2">2b</a>), we used the same experimental design as used to quantify the survival of injured ants (see above). In total, 30 sterile and 30 infected ants (<i>P. aeruginosa</i> OD = 0.05) were prepared, 12 sterile and 12 infected ants were kept in isolation, another 12 sterile and infected ants were placed inside the nest and 6 sterile and 6 infected ants were collected immediately after injury. Sterile or infected ants were then collected from their enclosures at either 2 or 11 h after manipulation (<i>n</i> = 6 per treatment). Only ants that did not die until these time points were used for further analyses. The collected ants were then first placed in hexane for 10 min to extract the CHC profile. The gaster was then cut off and placed in RNAlater for genetic analyses and the thorax was placed in 100% ethanol for the microbial analyses. The samples for genetic and microbial analyses were brought to the University of Lausanne and the samples for chemical analyses to the University of Würzburg. Another 10 sterile and 10 infected ants kept in isolation were further observed for a total of 36 h to ensure that the survival curves in this experiment resembled those in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41467-023-43885-w#Fig2">2a</a>.</p><h3 id="Sec13">Pathogen identification &amp; isolation</h3><p>To isolate potential pathogens, we collected surface soil in the surrounding area of the nest and grew its water extract on Sarborough Dextrose Agar (SDA) plates for 36 h at local temperature to get the ‘soil pathogen mix’ (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41467-023-43885-w#MOESM1">1c</a>). Most of the plate was at first overgrown by black mycelia identified as <i>Rhizopus microsporus</i>. This fungus is known to contain symbiotic bacteria <i>Burkholderia</i> sp. (species not identified), as we confirmed by Sanger sequencing and bacterial community analysis (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41467-023-43885-w#Fig1">1</a>c). After several days, the culture started to show several large colonies of slimy microorganisms that were isolated and identified as <i>Pseudomonas aeruginosa</i> (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41467-023-43885-w#MOESM1">1d</a>). For infection assays (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41467-023-43885-w#MOESM1">3</a>), we isolated <i>Burkholderia</i> and its fungal host <i>Rhizopus</i> from each other by repeated passaging with antifungal nystatin (0.25 μl/ml in M9 agar, 30 °C) or antibacterial ciprofloxamin (0.02 mg/ml in SDA). No representatives of <i>Klebsiella</i> were isolated.</p><p>The identification of all microorganisms was done by preparation of a PCR-ready DNA from a piece of biomass as described in Lõoke et al.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 34" title="Looke, M., Kristjuhan, K. &amp; Kristjuhan, A. Extraction of genomic DNA from yeasts for PCR-based applications. Biotechniques 50, 325–328 (2017)." href="https://www.nature.com/articles/s41467-023-43885-w#ref-CR34" id="ref-link-section-d111212994e1888">34</a></sup>. PCR reactions were done with universal fungal primers ITS5 5’-TCCTCCGCTTATTGATATGC-3′ and ITS4 5′-GGAAGTAAAAGTCGTAACAAGG-3′<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 35" title="Schoch, C. L. et al. Nuclear ribosomal internal transcribed spacer (ITS) region as a universal DNA barcode marker for Fungi. Proc. Natl Acad. Sci. USA 109, 6241–6246 (2012)." href="https://www.nature.com/articles/s41467-023-43885-w#ref-CR35" id="ref-link-section-d111212994e1892">35</a></sup> (GoTaq polymerase, Tm = 55 °C, elongation for 40 s) and commonly used universal bacterial primers 27F 5′-AGRGTTYGATYMTGGCTCAG-3′ and 1492 R 5′-GGTTACCTTGTTACGACTT-3’ (same protocol but elongation for 1 min 20 s). The presence of PCR amplicons was verified on electrophoresis gel and the fragments were sent for Sanger sequencing. The resulting sequences were then screened against NCBI database with nucleotide BLAST<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 36" title="Boratyn, G. M., Thierry-Mieg, J., Thierry-Mieg, D., Busby, B. &amp; Madden, T. L. Magic-BLAST, an accurate RNA-seq aligner for long and short reads. BMC Bioinform. 20, 405 (2019)." href="https://www.nature.com/articles/s41467-023-43885-w#ref-CR36" id="ref-link-section-d111212994e1896">36</a></sup>.</p><h3 id="Sec14">Microbiome analysis and bacterial load quantification</h3><p>To quantify the composition of the bacterial community in workers, we used the Powersoil DNA isolation kit (MO Bio) to extract DNA from <i>M. analis</i> thoraxes kept in RNAlater. Samples were homogenized by vortexing for 10 min, followed by two times 45 sec bead beating at 6 m/s using a FastPrep24<sup>TM</sup> 5 G homogenizer. Then we continued according to the kit’s manual and eluted DNA in 100 µL of nuclease-free water.</p><p>Bacterial loads were quantified with a QuantStudio5 qPCR instrument (Applied Biosystems) using the reaction set-up described in Kešnerová et al.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 37" title="Kesnerova, L. et al. Disentangling metabolic functions of bacteria in the honey bee gut. PLoS Biol. 15, e2003467 (2017)." href="https://www.nature.com/articles/s41467-023-43885-w#ref-CR37" id="ref-link-section-d111212994e1916">37</a></sup> and the thermal cycling conditions recommended for SYBR® Select Master Mix. For quantifying total bacterial loads, we used our designed primers #1047 5’-AGGATTAGATACCCTRGTAGTC-3’ and #1049 5’-CATSMTCCACCRCTTGTGC-3’ (at doubled 0.4 µM concentration). For specific targeting we used <i>P. aeruginosa</i> primers #1209 5’-GTAGATATAGGAAGGAACACCAG-3’ and #1210 5’-GGTATCTAATCCTGTTTGCTCC-3’ and for normalization to the host’s housekeeping gene we used <i>M. analis</i> 28S rRNA gene primers #1207 5’-CTGCCCGGCGGTACTCG-3’ and #1208 5’-ACCGGGGACGGCGCTAG-3’. Serial dilutions (10x) showed that these primers performed with an amplification efficiency (E) of 1.86 (<i>R</i><sup>2</sup> = 0.99), 2.00 (<i>R</i><sup>2</sup> = 0.99), and 1.87 (<i>R</i><sup>2</sup> = 0.99), respectively.</p><p>Total bacterial (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41467-023-43885-w#Fig1">1a</a>) and <i>P.</i> <i>aeruginosa</i> (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41467-023-43885-w#Fig2">2b</a>) 16 S rRNA gene (target) copy numbers were expressed relatively to <i>M. analis</i> 28 S rRNA gene copy numbers (host) based on the following equation: ΔCq= 2^(Cq<sub>host</sub> - Cq<sub>target</sub>), where Cq was the measured ‘quantification cycle’ value. To calculate the total 16 S rRNA gene copy number in 1 μl of each DNA sample that was used for absolute bacterial abundance based on ASV counts (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41467-023-43885-w#Fig1">1b</a>) we used the equation n  =  E^(intercept – Cq)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 38" title="Gallup, J. M. PCR Troubleshooting And Optimization: The Essential Guide. (eds. S. Kennedy, N. Oswald) Ch. 2 (Caister Academic Press, 2011)." href="https://www.nature.com/articles/s41467-023-43885-w#ref-CR38" id="ref-link-section-d111212994e1965">38</a></sup>, where standard curve’s <i>intercept</i> = 38.23<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 37" title="Kesnerova, L. et al. Disentangling metabolic functions of bacteria in the honey bee gut. PLoS Biol. 15, e2003467 (2017)." href="https://www.nature.com/articles/s41467-023-43885-w#ref-CR37" id="ref-link-section-d111212994e1972">37</a></sup>.</p><h3 id="Sec15">16 S rRNA gene amplicon-sequencing</h3><p>16 S rRNA gene amplicon sequencing data were obtained from 40 experimental samples, a mock sample (to verify consistency of the MiSeq run compared to previous studies in our group), two blank DNA extractions and a negative PCR control with only H<sub>2</sub>O. We followed the Illumina 16 S metagenomic sequencing preparation guide (<a href="https://support.illumina.com/documents/documentation/chemistry_documentation/16s/16s-metagenomic-library-prep-guide-15044223-b.pdf">https://support.illumina.com/documents/documentation/chemistry_documentation/16s/16s-metagenomic-library-prep-guide-15044223-b.pdf</a>) to amplify and sequence the V4 region of the 16 S rRNA gene. Primers for the first PCR step were 515F-Nex (TCGTCGGCAGCGTCAGATGTGTATAAGAGACAGGTGCCAGCMGCCGCGGTAA) and 806R-Nex (GTCTCGTGGGCTCGGAGATGTGTATAAGAGACAGGGACTACHVGGGTWTCTAAT). PCR amplifications were performed in a mix of 12.5 μL of Invitrogen Platinum SuperFi DNA Polymerase Master Mix, 5 μL of MilliQ water, 2.5 μL of each primer (5 μM), and 2.5 μL of template DNA. PCR conditions were 98 °C for 30 s, 25 cycles of 98 °C for 10 s, 55 °C for 20 s, and 72 °C for 20 s, and a final extension step at 72 °C for 5 min. Amplifications were confirmed by 2% agarose gel electrophoresis. The PCR products were then purified with Clean NGS purification beads (CleanNA) in a 1:0.8 ratio of PCR product to beads, and eluted in 27.5 μL of 10 mM Tris, pH 8.5. We then performed a second PCR step in which unique dual-index combinations were appended to the amplicons using the Nextera XT index kit (Illumina). Second-step PCRs were performed in a 25 μL, using 2.5 μL of the PCR products, 12.5 μL of Invitrogen Platinum SuperFi DNA Polymerase Master Mix, 5 μL of MilliQ water, and 2.5 μL of each of the Nextera XT indexing primers 1 and 2. PCR conditions were 95 °C for 3 min followed by eight cycles of 30 s at 95 °C, 30 s at 55 °C, and 30 s at 72 °C, and a final extension step at 72 °C for 5 min. The libraries were again purified using Clean NGS purification beads in a 1:1.12 ratio of PCR product to beads, and eluted in 27.5 μL of 10 mM Tris, pH 8.5. The amplicon concentrations, including the negative PCR control, mock, and blanks, were quantified by PicoGreen and pooled in equimolar concentrations (except for the water and blank samples that were kept at lower concentrations). We verified that the final pool was of the right size using a Fragment Analyzer (Advanced Analytical). MiSeq (Illumina) sequencing was then performed at the Genomic Technology Facility of the University of Lausanne, producing (2 × 250 bp) reads. We obtained a total of 979’126 paired-end reads across the 40 experimental samples.</p><h3 id="Sec16">Analyses of 16 rRNA gene amplicon-sequencing data</h3><p>Raw sequencing data (deposited at the Sequence Read Archive (SRA) under PRJNA826317) were analyzed with the Divisive Amplicon Denoising Algorithm 2 (DADA2) package v.1.20.0 in R. All functions were run using the recommended parameters (<a href="https://benjjneb.github.io/dada2/tutorial.html">https://benjjneb.github.io/dada2/tutorial.html</a>) except for the filtering step in which we truncated the forward and reverse reads after 232 and 231 bp, respectively. At the learnErrors step, we then set randomize=TRUE and nbases=3e8. Amplicon-sequence variants (ASVs) were classified with the SILVA database (version 138). Unclassified ASVs and any ASV classified as chloroplast, mitochondria or Eukaryota were removed with the “phyloseq” package version 1.36.0, using the “subset taxa” function. We then used the “prevalence” method in the R package “decontam” v.1.12.0 to identify and remove contaminants introduced during laboratory procedures, using the negative PCR control and the blank samples as reference. This procedure filtered out three contaminant ASVs. Four additional ASVs were removed as they clearly represented (low abundance) contaminants due to index swapping from samples of a different project that were sequenced in the same sequencing run. We then calculated absolute bacterial abundances of each ASV by multiplying the proportion of each ASV by the total 16S rRNA gene copy number of each sample as measured by qPCR. To assess differences in community structure between treatments we ran ADONIS tests after calculating Bray-Curtis dissimilarities with the absolute ASV abundance matrix and plotted ordinations based on these Bray-Curtis dissimilarities (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41467-023-43885-w#MOESM1">1a</a>).</p><p>To test for differences in absolute abundance of individual bacterial genera between sterile and infected ants (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41467-023-43885-w#MOESM1">1b</a>), we used a permutation approach (permutation t-test) as done in Kešnerová et al.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 39" title="Kesnerova, L. et al. Gut microbiota structure differs between honeybees in winter and summer. ISME J. 14, 801–814 (2020)." href="https://www.nature.com/articles/s41467-023-43885-w#ref-CR39" id="ref-link-section-d111212994e2017">39</a></sup>. To do this, we selected the ASVs that had at least 1% relative abundance across five samples (18 ASVs belonging to 11 genera). We then calculated copy numbers at the genus-level for each of the 40 samples. At each time-point (2 h and 11 h), we randomized the values of the calculated copy numbers for each genus 10,000 times and computed the <i>t</i> values for the tested effect for each randomized dataset. The <i>P</i> values corresponding to the effects were calculated as the proportion of 10,000 <i>t</i> values that were equal or higher than the observed one.</p><h3 id="Sec17">Chemical analysis of cuticular hydrocarbons</h3><p>To quantify differences in CHC profiles between infected and sterile ant workers, cuticular hydrocarbon extracts were evaporated to a volume of approximately 100 μL and 1 μL was analyzed by using a 6890 gas chromatograph (GC) coupled to a 5975 mass selective detector (MS) by Agilent Technologies (Waldbronn, Germany). The GC was equipped with a DB-5 capillary column (0.25 mm ID × 30 m; film thickness 0.25 μm, J &amp; W Scientific, Folsom, Ca, USA). Helium was used as a carrier gas with a constant flow of 1 mL/min. A temperature program from 60 °C to 300 °C with 5 °C/min and finally 10 min at 300 °C was employed. Mass spectra were recorded in the EI mode with an ionization voltage of 70 eV and a source temperature of 230 °C. The software ChemStation v. F.01.03.2357 (Agilent Technologies, Waldbronn, Germany) for windows was used for data acquisition. Identification of the components was accomplished by comparison of library data (NIST 17) with mass spectral data of commercially purchased standards and diagnostic ions.</p><p>To compare the relative abundances of the different compound groups (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41467-023-43885-w#MOESM1">6</a>), all compounds were identified (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41467-023-43885-w#MOESM1">8</a>) and grouped either into Alkanes, Alkenes, Alkadienes or Methyl-branched alkanes for each individual.</p><h3 id="Sec18">Antimicrobial assay</h3><p>To assess the antimicrobial efficacy of MG extracts (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41467-023-43885-w#Fig3">3d</a>), we quantified the increase over time of the OD in a 96-well plate box, with the outer row filled with 70 μL of PBS. In total we did 3 replicates per sample with a total of 70 μL per sample. Negative control: 70 μL Luria-Bertani (LB) broth (<i>n</i> = 6). Positive control: 68 μL LB broth + 2 μL <i>P. aeruginosa</i> (<i>n</i> = 6). MG sample: 66 μL LB broth + 2 μL <i>P. aeruginosa</i> + 2 μL MG sample (<i>n</i> = 9).</p><p>For the preparation of the pathogen sample, we plated <i>P. aeruginosa</i> from a frozen stock on TSA plates. After 24 h the bacterial culture was replated. After 12 h aliquots were created using the new bacterial culture in a flask containing 15 mL of freshly sterilized LB broth for an initial OD reading. Afterwards the flask was placed in an incubator shaker (Amerex Steadyshake 757) at 180 rpm, 30 °C for the bacteria to grow. Once the OD readings were between 02–0.5 OD (the exponential growth phase) the flask was put on ice until the experiments started.</p><p>For the preparation of the MG sample, we pooled 10 MGs in 1.5 mL Eppendorf-Cups. We then froze the samples in liquid nitrogen and crushed the sample material using sterile pellets. We then added 50 μL of PBS-Buffer, vortexed shortly before centrifuging the sample for 5 min at 3000 x g at 4 °C. We then extracted 30 μL of the supernatant into a new cup and repeated the centrifugation process. Afterwards 20 μL of the supernatant was placed again in a new cup and stored at −20 °C. Preparation of the wells was conducted on ice at 4 °C to prevent bacterial growth and degradation of the MG samples. The antimicrobial assays were done in a microplate reader (Synergy H1 BioTek) at 30 °C with a 600 nm wavelength for 8 h with a double orbital shaking step after each OD reading cycle (every 10 min) at the University of Lausanne.</p><p>To calculate the intrinsic growth rate of the microbial population (r) in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41467-023-43885-w#Fig3">3d</a> we used the package growthcurver (v. 0.3.1) with the statistical software R v4.1.0. “r” represents the growth rate that would occur if there were no restrictions imposed on total population size.</p><h3 id="Sec19">Sample collection for metapleural gland extracts</h3><p>Due to the difficulty of collecting adequate amounts of metapleural gland secretions from the gland’s atrium (it is a very sticky substance that adheres to the cuticle), we decided to remove the atrium of the gland together with the secretory cells entirely, using a microscalpel and microscissors. To avoid using a solvent we used a Thermodesorber unit coupled to a GC-MS (TD-GC-MS). For this we transported frozen ants to the University of Würzburg and did the extractions directly in the laboratory next to the TD-GC-MS. One metapleural gland from each of six worker was pooled per sample. As a control we further collected pieces of cuticle of similar size from the side of the thorax to identify any potential contaminations which might have occurred during dissections. In total 3 samples (of 6 individuals each) were analyzed (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41467-023-43885-w#MOESM1">8</a>, Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41467-023-43885-w#MOESM1">13</a>).</p><h3 id="Sec20">Chemical analysis of the metapleural gland</h3><p>The MG samples were placed in a glass-wool-packed thermodesorption tube and placed in the thermodesorber unit (TDU; TD100-xr, Markes, Offenbach am Main, Germany). The thermodesorption tube was heated up to 260 °C for 10 min. The desorbed components were transferred to the cold trap (5 °C) to focus the analytes using N<sub>2</sub> flow in splitless mode. The cold trap was rapidly heated up to 310 °C at a rate of 60 °C per minute, held for 5 min and connected to the GC-MS (Agilent 7890B GC and 5977 MS, Agilent Technologies, Palo Alto, USA) via a heated transfer line (300 °C). The GC was equipped with an HP-5MS UI capillary column (0.25 mm ID × 30 m; film thickness 0.25 μm, J &amp; W Scientific, Folsom, Ca, USA). Helium was the carrier gas using 1.2874 ml/min flow. The initial GC oven temperature was 40 °C for 1 min, then raised at a rate of 5 °C per min until reaching 300 °C, where it was held for 3 min. The transfer line temperature between GC and MS was 300 °C. The mass spectrometer was operated in electron impact (EI) ionization mode, scanning m/z from 40 to 650, at 2.4 scans per second. Chemical compounds were identified using the same protocol as for the CHCs.</p><h3 id="Sec21">Proteomic analysis and sample preparation of the metapleural gland</h3><p>To characterize the proteins secreted by the metapleural gland, we analyzed the proteome of the atrium of the gland, the metapleural gland secretory cells and the hemolymph (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41467-023-43885-w#MOESM1">7</a>). To be considered a metapleural gland protein that could mediate antimicrobial activity, proteins had to be found in the gland’s atrium, and must have a higher abundance in the gland’s atrium than in the hemolymph. The samples for proteomic analysis were collected in April 2020 from workers of field colonies collected in the Comoé National Park. Three types of samples were collected: Secretory: dissected secretory cells without the atrium, (<i>n</i> = 6 samples, each pooled from five dissections); Hemolymph: hemolymph was collected from the thorax by a glass microcapillary through a small wound (<i>n</i> = 6 samples, each pooled over five individuals, yielding 4–5 μL); Atrium: due to the difficulty of collecting pure MG content we chose to first widen the opening of the atrium and add 1 μl of PBS before extracting the content together with the PBS (<i>n</i> = 6 samples, each pooled from five individuals). The samples were kept at −23 °C in Lo-bind Eppendorf tubes with 5 μl of PBS together with a 1x Protease inhibitor Cocktail (Sigmafast) during transportation. Once in Lausanne, Switzerland, the samples were kept at −80 °C and processed swiftly.</p><p>Proteins were digested according to a modified version of the iST protocol<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 40" title="Kulak, N. A., Pichler, G., Paron, I., Nagaraj, N. &amp; Mann, M. Minimal, encapsulated proteomic-sample processing applied to copy-number estimation in eukaryotic cells. Nat. Methods 11, 300–319 (2014)." href="https://www.nature.com/articles/s41467-023-43885-w#ref-CR40" id="ref-link-section-d111212994e2137">40</a></sup>. Samples were resuspended in 20 μL of modified iST buffer (1% sodium deoxycholate, 10 mM DTT, 100 mM Tris pH 8.6) and heated at 95 °C for 5 min. They were then diluted with 24 μL of 4 mM MgCl<sub>2</sub> and 1:100 of benzonase in H<sub>2</sub>O and incubated 15 min at ambient temperature. 14 μL of 160 mM chloroacetamide (in 10 mM Tris pH 8.6) were then added and cysteines were alkylated for 45 min at 25 °C in the dark. After addition of 0.5 M EDTA (3 mM final concentration), samples were digested with 0.1 μg of trypsin/Lys-C mix (Promega) at 37 °C for 1 h, followed by a second enzyme addition (0.1 μg trypsin/LysC) and 1 h incubation. Two volumes of isopropanol + 1% trifluoroacetic acid (TFA) were added to one volume of sample and loaded onto an equilibrated OASIS MCX uElution plate (Waters) prefilled with SCX0 buffer (20% MeCN, 0.5% formic acid) and centrifuged. The columns were washed three times with 200 μL isopropanol + 1% TFA and once with 200 μL HPLC solvent A (2% MeCN, 0.1% formic acid). The peptide mixture was then sequentially eluted with 150 μL SCX125 buffer (20% MeCN, 0.5% formic acid, 125 mM ammonium acetate), 150 μL SCX500 buffer (20% MeCN, 0.5% formic acid, 500 mM ammonium acetate), and lastly with 150 μL basic elution buffer (80% MeCN, 19% water, 1% NH3).</p><p>Tryptic peptides fractions were dried and resuspended in 0.05% trifluoroacetic acid, 2% (v/v) acetonitrile, for mass spectrometry analyses. Tryptic peptide mixtures were injected on an Ultimate RSLC 3000 nanoHPLC system (Dionex, Sunnyvale, CA, USA) interfaced to an Orbitrap Fusion Tribrid mass spectrometer (Thermo Scientific, Bremen, Germany). Peptides were loaded onto a trapping microcolumn Acclaim PepMap100 C18 (20 mm × 100 μm ID, 5 μm, 100 Å, Thermo Scientific) before separation on a reversed-phase custom-packed nanocolumn (75 μm ID × 40 cm, 1.8 μm particles, Reprosil Pur, Dr. Maisch). A flow rate of 0.25 μL/min was used with a gradient from 4 to 76% acetonitrile in 0.1% formic acid (total time: 65 min). Full survey scans were performed at a 120’000 resolution, and a top-speed precursor selection strategy was applied to maximize acquisition of peptide tandem MS spectra with a maximum cycle time of 0.6 s. HCD fragmentation mode was used at a normalized collision energy of 32%, with a precursor isolation window of 1.6 m/z, and MS/MS spectra were acquired in the ion trap. Peptides selected for MS/MS were excluded from further fragmentation during 60 s.</p><p>Tandem MS data were processed by the MaxQuant software (version 1.6.14.0)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 41" title="Cox, J. &amp; Mann, M. MaxQuant enables high peptide identification rates, individualized p.p.b.-range mass accuracies and proteome-wide protein quantification. Nat. Biotechnol. 26, 1367–1372 (2008)." href="https://www.nature.com/articles/s41467-023-43885-w#ref-CR41" id="ref-link-section-d111212994e2151">41</a></sup> using the Andromeda search engine<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 42" title="Cox, J. et al. Andromeda: a peptide search engine integrated into the maxquant environment. J. Proteome Res. 10, 1794–1805 (2011)." href="https://www.nature.com/articles/s41467-023-43885-w#ref-CR42" id="ref-link-section-d111212994e2155">42</a></sup> matching to a custom-made protein database containing 19’618 sequences of <i>M. analis</i> (July 2019 version, see section “Genome annotation” for details), supplemented with sequences of common contaminants. Trypsin (cleavage at K,R) was used as the enzyme definition, allowing 2 missed cleavages. Mass tolerance was 4.5 ppm on precursors (after recalibration) and 0.5 Da on MS/MS fragments. Only peptides with a minimal length of 7 were considered for protein identifications. Carbamidomethylation of cysteine was specified as a fixed modification. N-terminal acetylation of protein and oxidation of methionine were specified as variable modifications. All identifications were filtered at 1% FDR at both the peptide and protein levels with default MaxQuant parameters. For protein quantitation the iBAQ values<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 43" title="Schwanhausser, B. et al. Global quantification of mammalian gene expression control. Nature 473, 337–342 (2011)." href="https://www.nature.com/articles/s41467-023-43885-w#ref-CR43" id="ref-link-section-d111212994e2162">43</a></sup> were used. MaxQuant data were further processed with Perseus software (version 1.6.14.0)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 44" title="Tyanova, S. et al. The Perseus computational platform for comprehensive analysis of (prote)omics data. Nat. Methods 13, 731–740 (2016)." href="https://www.nature.com/articles/s41467-023-43885-w#ref-CR44" id="ref-link-section-d111212994e2166">44</a></sup> for filtering, removing contaminants and proteins identified with only 1 peptide, log2-transformation, normalization of values, and ortholog annotations.</p><p>To determine which proteins are secreted by the MG, we proceeded through a series of filters. Proteins not found in the atrium samples were eliminated and those found only in the MG atrium were considered hits. The log2 fold change between the atrium and the hemolymph were determined for all remaining proteins. Only proteins that were on average 1.5-fold more abundant in atrium samples than hemolymph were retained as hits. Percent of total iBAQ per sample was assigned to each protein to indicate abundance. Visualization of heatmaps was performed in Matlab 2020b (clustergram function). To determine whether our selected proteins were toxin-like, we ran clantox<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 45" title="Naamati, G., Askenazi, M. &amp; Linial, M. ClanTox: a classifier of short animal toxins. Nucleic Acids Res. 37, W363–W368 (2009)." href="https://www.nature.com/articles/s41467-023-43885-w#ref-CR45" id="ref-link-section-d111212994e2174">45</a></sup> (<a href="http://www.clantox.cs.huji.ac.il/">http://www.clantox.cs.huji.ac.il/</a>). Annotations (function, orthology depth, and implications for wound healing) found in Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41467-023-43885-w#MOESM1">12</a> were determined using protein BLAST with the experimental clustered nr database<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 46" title="Steinegger, M. &amp; Soding, J. MMseqs2 enables sensitive protein sequence searching for the analysis of massive data sets. Nat. Biotechnol. 35, 1026–1028 (2017)." href="https://www.nature.com/articles/s41467-023-43885-w#ref-CR46" id="ref-link-section-d111212994e2188">46</a></sup>.</p><p>The mass spectrometry proteomics data are deposited at the ProteomeXchange Consortium via the PRIDE partner repository with the dataset identifier PXD033003.</p><h3 id="Sec22">Genome sequencing and assembly</h3><p>The genome of <i>M. analis</i> was sequenced and assembled by The Global Ant Genomics Alliance (GAGA, antgenomics.dk)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 47" title="Boomsma, J. J. et al. The global ant genomics alliance (GAGA). Myrmecol. News 25, 61–66 (2017)." href="https://www.nature.com/articles/s41467-023-43885-w#ref-CR47" id="ref-link-section-d111212994e2206">47</a></sup>. To identify repeats in the genome assembly, we used the package RepeatModeler v2.0, which combines three <i>de-novo</i> repeat finding programs (RECON, RepeatScout and LtrHarvest/Ltr_retriever)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 48" title="Flynn, J. M. et al. RepeatModeler2 for automated genomic discovery of transposable element families. Proc. Natl Acad. Sci. USA 117, 9451–9457 (2020)." href="https://www.nature.com/articles/s41467-023-43885-w#ref-CR48" id="ref-link-section-d111212994e2213">48</a></sup>. RepeatModeler first probes chunks of the genome assembly to find repeats, then clusters and classifies identified repeats, producing a high-quality library of consensus sequences of repeated sequence families. The consensus sequences were used with RepeatMasker to softmask (in lower-case) all repeats in the genome assembly.</p><h3 id="Sec23">Genome annotation</h3><p>To annotate the <i>M. analis</i> genome assembly we used the ab-initio gene predictors Augustus v3.3.3<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 49" title="Stanke, M., Diekhans, M., Baertsch, R. &amp; Haussler, D. Using native and syntenically mapped cDNA alignments to improve de novo gene finding. J. Bioinform. 24, 637–644 (2008)." href="https://www.nature.com/articles/s41467-023-43885-w#ref-CR49" id="ref-link-section-d111212994e2229">49</a></sup> and genemark<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 50" title="Lomsadze, A., Ter-Hovhannisyan, V., Chernoff, Y. O. &amp; Borodovsky, M. Gene identification in novel eukaryotic genomes by self-training algorithm. Nucleic Acids Res. 33, 6494–6506 (2005)." href="https://www.nature.com/articles/s41467-023-43885-w#ref-CR50" id="ref-link-section-d111212994e2233">50</a></sup> within the BRAKER2 pipeline v2.1.4<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 51" title="Hoff, K. J., Lange, S., Lomsadze, A., Borodovsky, M. &amp; Stanke, M. BRAKER1: unsupervised RNA-Seq-Based genome annotation with geneMark-ET and AUGUSTUS. J. Bioinform. 32, 767–769 (2016)." href="https://www.nature.com/articles/s41467-023-43885-w#ref-CR51" id="ref-link-section-d111212994e2237">51</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 52" title="Bruna, T., Hoff, K. J., Lomsadze, A., Stanke, M. &amp; Borodovsky, M. BRAKER2: automatic eukaryotic genome annotation with GeneMark-EP+ and AUGUSTUS supported by a protein database. NAR Genom. J. Bioinform 3, lqaa108 (2021)." href="https://www.nature.com/articles/s41467-023-43885-w#ref-CR52" id="ref-link-section-d111212994e2240">52</a></sup>. Both tools were trained using RNA-seq reads (see below) aligned to the softmasked version of the genome with STAR v2.7<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 53" title="Dobin, A. et al. STAR: ultrafast universal RNA-seq aligner. Bioinformatics 29, 15–21 (2013)." href="https://www.nature.com/articles/s41467-023-43885-w#ref-CR53" id="ref-link-section-d111212994e2244">53</a></sup> and protein evidence from NCBI refseq predictions for <i>Acromyrmex echinatior, Atta colombica, Camponotus floridanus, Dinoponera quadriceps, Linepithema humile, Monomorium pharaonis, Ooceraea biroi, Pogonomyrmex barbatus, Solenopsis invicta, Temnothorax curvispinosus, Vollenhovia emeryi</i> and <i>Wasmannia auropunctata</i>. To enable functional interpretation of experimental data, we annotated gene products. First, we identified gene orthologs in <i>Drosophila melanogaster</i> to leverage the excellent quality of the functional annotation in this species. To identify the orthologs, we applied a reciprocal best blast approach using Orthologr<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 54" title="Drost, H. G., Gabel, A., Grosse, I. &amp; Quint, M. Evidence for active maintenance of phylotranscriptomic hourglass patterns in animal and plant embryogenesis. Mol. Biol. Evol. 32, 1221–1231 (2015)." href="https://www.nature.com/articles/s41467-023-43885-w#ref-CR54" id="ref-link-section-d111212994e2258">54</a></sup> between the longest protein isoform for each gene from both species<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 55" title="Altschul, S. F., Gish, W., Miller, W., Myers, E. W. &amp; Lipman, D. J. Basic local alignment search tool. J. Mol. Biol. 215, 403–410 (1990)." href="https://www.nature.com/articles/s41467-023-43885-w#ref-CR55" id="ref-link-section-d111212994e2262">55</a></sup>. Additionally, we identified orthologs with <i>Apis mellifera</i> and <i>Camponotus floridanus</i> NCBI RefSeq gene annotations. Finally, we ran InterProScan (version 5.30–69.0 and panther-data-14.1) on all proteins using default settings. To assign functional information to <i>M. analis</i> annotated genes, we primarily used the gene ontology of <i>D. melanogaster</i> orthologs, but also results from sequence homology search on Interproscan and the Uniprot database.</p><p>To study the effect of infections on gene expression (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41467-023-43885-w#Fig5">5</a>), we conducted a transcriptomic analysis. We dissected the gaster from the same infected (soil OD = 0.1, <i>n</i> = 20) and sterile ants (<i>n</i> = 20), whose thorax were used for the microbiome analyses in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41467-023-43885-w#Fig1">1a, b</a>. Samples were collected 2 and 11 h after manipulation (<i>n</i> = 10 per timepoint) in the Comoé field research station and had the sting together with the venom reservoir removed. The gaster was dissected without solvent and stored in a RNAlater stabilisation solution at −23 °C (ThermoFisher Scientific) for later analyses at the University of Lausanne. Afterwards, the samples were homogenized with ceramic beads in 1 ml of Trizol reagent. Homogenized samples were incubated for 5 min at room temperature (RT) in Trizol, before adding Chloroform (200 μL). Samples were incubated for 5 min at RT then centrifuged (25 s at 12,000 rpm and 4 °C) and the upper aqueous layer (~500 μL) transferred to a new tube. We added Isopropanol (650 μL) and Glycogen blue (1 μL; RNAse-free, Invitrogen, 15 mg/mL, #AM9516) then vortexed and incubated overnight at −20 °C. To purify the RNA, we used a EtoH precipitation method: samples were centrifuged (30 s at full speed at 4 °C), the supernatant was discarded and EtOH (1 mL at 80%) added. We repeated this step a second time with EtOH (1 mL at 70%). Finally, the supernatant was removed and the pellet, after a brief air dry (15–20 s) at RT, was resuspended in nuclease-free water. Libraries were prepared with the KAPA Stranded mRNASeq Library Preparation Kit (#KK8421) according to the manufacturer’s protocol. Paired end sequencing was performed on an Illumina Hiseq4000 sequencer at the Genomic Technology Facility of the University of Lausanne. We obtained ~15–25 million PE reads per individual. Sequence reads have been deposited in NCBI Sequence Read Archive (SRA) under the accession number PRJNA823913.</p><p>We mapped RNA-seq reads on the softmask genome assembly using STAR v2.7 with the 2-pass mapping option and using the gene annotations to properly find exonic junctions<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 53" title="Dobin, A. et al. STAR: ultrafast universal RNA-seq aligner. Bioinformatics 29, 15–21 (2013)." href="https://www.nature.com/articles/s41467-023-43885-w#ref-CR53" id="ref-link-section-d111212994e2300">53</a></sup>. Then, we used the program featureCounts from the Subread package to count the number of reads mapped on each exon for each gene<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 56" title="Liao, Y., Smyth, G. K. &amp; Shi, W. featureCounts: an efficient general purpose program for assigning sequence reads to genomic features. J. Bioinform. 30, 923–930 (2014)." href="https://www.nature.com/articles/s41467-023-43885-w#ref-CR56" id="ref-link-section-d111212994e2304">56</a></sup>. Then using the count’s files, we performed differential gene expression analyses using DESeq2<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 57" title="Love, M. I., Huber, W. &amp; Anders, S. Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. Genome Biol. 15, 550 (2014)." href="https://www.nature.com/articles/s41467-023-43885-w#ref-CR57" id="ref-link-section-d111212994e2308">57</a></sup> of sterile and infected workers collected at 2 and 11 h after treatment. <i>P</i> values of differential expression analyses were corrected for multiple testing with a false discover rate (FDR) of 5%.</p><h3 id="Sec24">X-Ray Micro-CT imaging</h3><p>To examine the internal morphology of the MG (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41467-023-43885-w#Fig3">3a</a>), we scanned and visualized an intermediate-sized worker (CASENT0744096), although multiple other workers of different sizes were scanned and visually inspected to confirm the morphology of the exemplar specimen was representative. The specimens were fixed in 90% ethanol, stained in 2 M iodine solution for a minimum of 7 days, then washed and sealed in a pipette tip with 99% ethanol for scanning. The scans were performed with the ZEISS Xradia 510 Versa 3D X-ray microscope at the Okinawa Institute of Science and Technology Graduate University, Japan. We scanned the thorax (mesosoma) and whole body to visualize both the structure of the gland, and its location in the whole ant. The scan parameters for the thorax scan were: Voxel size, 4.404 μm; Exposure time, 6 s; Voltage, 40 kV; Power, 3 W; Source distance, 23.147 mm; Objective, 4x; Projections, 1601. For the full body scan, we used: Voxel size, 12.5 μm; Exposure time, 1 s; Voltage, 40 kV; Power, 3 W; Source distance, 25.03 mm; Objective, 0.4x; Projections, 1601. The 3D reconstruction was performed using the ZEISS Scout-and-Scan Control System Reconstructor software (ZEISS Microscopy, Jena, Germany). The MG structures were manually segmented using Amira (version 2019.2; Thermo Fisher Scientific, Berlin, Germany) and visualized using VGStudio3.4 (Volume Graphics GmbH, Heidelberg, Germany). The gland atrium was rendered using isosurface and a clipping plane was created to visualize the cross-section. All other materials were visualized using Phong volume rendering.</p><h3 id="Sec25">Statistical analysis</h3><p>For statistical analyses and graphical illustration, we used the statistical software R v4.1.0<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 58" title="R: A Language And Environment For Statistical Computing (R Foundation for Statistical Computing, Vienna, Austria, 2013)." href="https://www.nature.com/articles/s41467-023-43885-w#ref-CR58" id="ref-link-section-d111212994e2334">58</a></sup> with the user interface RStudio v1.4.1717 and the R package ggplot2 v3.3.5<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 59" title="Wickham, H. ggplot2: Elegant Graphics for Data Analysis (Springer-Verlag New York, 2009)." href="https://www.nature.com/articles/s41467-023-43885-w#ref-CR59" id="ref-link-section-d111212994e2338">59</a></sup>. To select the appropriate statistical tests, we tested for deviations from the normal distribution with the Shapiro–Wilks test (<i>P</i> &gt; 0.05). A Bartlett test was used to verify homoscedasticity (<i>P</i> &gt; 0.05). All our models included colony as a random factor and an overall likelihood ratio test against an intercept only model. In case of multiple testing, a Holm-Bonferroni correction was performed with the adjusted P-values given throughout the text. To test for significant differences in the survival curves, we conducted mixed effect cox proportional hazards regression models (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41467-023-43885-w#MOESM1">2</a>) using the R package survminer (v0.4.9) followed by post-hoc analyses using least square means with the R package lsmeans (v.2.30; Supplementary Tables <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41467-023-43885-w#MOESM1">3</a>, 4 and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41467-023-43885-w#MOESM1">6</a>). The survival curves were illustrated using Kaplan-Meier cumulative survival curves (Figs. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41467-023-43885-w#Fig1">1</a>c, <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41467-023-43885-w#Fig2">2</a>a, and <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41467-023-43885-w#Fig4">4</a>, and Supplementary Figs. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41467-023-43885-w#MOESM1">3</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41467-023-43885-w#MOESM1">5</a>). For the bacterial load (ΔCq) within the thorax content between treated and untreated infected individuals (Figs. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41467-023-43885-w#Fig1">1</a>a and <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41467-023-43885-w#Fig2">2b</a>) we conducted linear mixed effect models with a least square means (lsmeans) post-hoc analysis. For the bacterial growth inhibition (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41467-023-43885-w#Fig3">3d</a>) we conducted a Mann–Whitney <i>U</i> test. Differences in the CHC-profile composition were calculated using a permutational multivariate analysis of variance (ADONIS) on a Bray-Curtis dissimilarity matrix using the package vegan (v.2.5–7; Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41467-023-43885-w#MOESM1">7</a>). For the different durations of MG care between sterile and infected individuals (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41467-023-43885-w#MOESM1">4</a>), we conducted a linear mixed effect model with individual as random factor followed by a Satterthwaite’s t-test. For differences between chemical compound groups of the CHC profiles (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41467-023-43885-w#MOESM1">6</a>), we conducted an analysis of variance (AOV) followed by a Tukey Honest Significant differences test (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41467-023-43885-w#MOESM1">9</a>). For behavioral differences in wound care between sterile and infected individuals (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41467-023-43885-w#Fig3">3b, c</a>), we modeled wound care as a binary event using binomial generalized additive models with posthoc contrasts to identify intervals of time during which the probability of receiving wound care differed between sterile and infected individuals. See Supplementary Information for a reproducible modeling workflow.</p><h3 id="Sec26">Ethics and inclusion statement</h3><p>Our study brings together authors from a number of different countries but failed to include scientists based in the country where the study was carried out. We recognize that more could have been done to engage local scientists with our research as our project developed, and to embed our research within the national context and research priorities. Whenever possible, our research was discussed with local stakeholders to seek feedback on the questions to be tackled and the approach to be considered. Whenever relevant, literature published by scientists from the region was also cited; efforts were made to consider relevant work published in the local language.</p><h3 id="Sec27">Reporting summary</h3><p>Further information on research design is available in the <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41467-023-43885-w#MOESM8">Nature Portfolio Reporting Summary</a> linked to this article.</p></div></div>
  </body>
</html>
