<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <title>James Routley | Feed</title>
    <link
      rel="stylesheet"
      type="text/css"
      href="../styles.css"
      media="screen"
    />
  </head>
  <body>
    <a href="/index.html">Back</a>
    <a href="https://arcinstitute.org/">Original</a>
    <h1>Arc Institute – for curiosity-driven biomedical science and technology</h1>
    
    <div id="readability-page-1" class="page"><div><section><div><div><p><a href="http://harihareswara.net/blog/introducing-arc-institute"><span>Blog</span><span><span>Introducing Arc Institute</span><svg width="11" height="9" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 560 462"><path d="m546.786 198.393-184.35-184.35A45.15 45.15 0 0 0 330.299 0a45.146 45.146 0 0 0-45.714 45.68 45.137 45.137 0 0 0 14.012 32.145l108.97 108.81H45.137a45.142 45.142 0 0 0-39.09 22.57 45.132 45.132 0 0 0 0 45.133 45.14 45.14 0 0 0 39.09 22.57h361.42l-107.96 108.14a45.137 45.137 0 0 0 63.84 63.781l184.35-184.35a44.798 44.798 0 0 0 13.215-31.918c.031-.375.031-.75 0-1.121a8.946 8.946 0 0 0 0-1.121 45.194 45.194 0 0 0-13.215-31.918l-.001-.008Z" fill="currentColor" fill-rule="nonzero"></path></svg></span></a></p></div><div><p>Headquartered in Palo Alto, California, Arc is a nonprofit research organization founded on the belief that many important scientific programs can be enabled by new organizational models. Arc operates in partnership with Stanford University, UCSF, and UC Berkeley.</p><p>Arc gives scientists no-strings-attached, multi-year funding, so that they don’t have to apply for external grants, and invests in the rapid development of experimental and computational technological tools.</p><p>As individuals, Arc researchers collaborate across diverse disciplines to study complex diseases, including cancer, neurodegeneration, and immune dysfunction. As an organization, Arc strives to enable ambitious, long-term research agendas.</p><p>Arc’s mission is to accelerate scientific progress, understand the root causes of disease, and narrow the gap between discoveries and impact on patients.</p></div></div></section><section><div><p><svg width="181" height="13" viewBox="0 0 181 13" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path stroke="var(--title-decoration-color, currentColor)" d="m1 11 62-9"></path><path stroke="var(--title-decoration-color, currentColor)" d="m50 11 64-5"></path><path stroke="var(--title-decoration-color, currentColor)" d="m114 6 54 5"></path><path stroke="var(--title-decoration-color, currentColor)" d="m62 2 18 9"></path><circle fill="var(--title-decoration-color, currentColor)" fill-rule="nonzero" cx="49.5" cy="10.5" r="1.5"></circle><circle fill="var(--title-decoration-color, currentColor)" fill-rule="nonzero" cx="1.5" cy="11.5" r="1.5"></circle><circle fill="var(--title-decoration-color, currentColor)" fill-rule="nonzero" cx="62.5" cy="1.5" r="1.5"></circle><circle fill="var(--title-decoration-color, currentColor)" fill-rule="nonzero" cx="102.5" cy="6.5" r="1.5"></circle><circle fill="var(--title-decoration-color, currentColor)" fill-rule="nonzero" cx="61" cy="6" r="1"></circle><circle fill="var(--title-decoration-color, currentColor)" fill-rule="nonzero" cx="57.5" cy="4.5" r="1"></circle><circle fill="var(--title-decoration-color, currentColor)" fill-rule="nonzero" cx="80.5" cy="11.5" r="1.5"></circle><circle fill="var(--title-decoration-color, currentColor)" fill-rule="nonzero" cx="114.5" cy="5.5" r="1.5"></circle><circle fill="var(--title-decoration-color, currentColor)" fill-rule="nonzero" cx="167.5" cy="10.5" r="1.5"></circle><circle fill="var(--title-decoration-color, currentColor)" fill-rule="nonzero" cx="179.5" cy="10.5" r="1.5"></circle></g></svg><h2>The Arc Model</h2></p><p>Arc is organized around three core concepts, each consisting of an institutional experiment in how research can be enabled.</p><p><a href="http://harihareswara.net/model">Learn more</a></p><div><div><p><h4>Core investigators</h4></p><p>We fully fund investigators and their labs with complete freedom to pursue curiosity-driven research agendas. Arc investigators may hold faculty positions at our partner institutions, and graduate students at Stanford, UC Berkeley, and UCSF can pursue their PhD studies at Arc labs.</p></div><div><p><h4>Technology Development</h4></p><p>Our Technology Centers provide long-term career options for Arc scientists beyond their training period, enabling them to develop, optimize and deploy rapidly advancing experimental and computational technologies. Initial Centers will focus on machine learning, genome engineering, cellular and animal models of human disease, and multi-omics.</p></div><div><p><h4>Translational Programs</h4></p><p>In addition to curiosity-driven knowledge building, producing medically useful discoveries is a core mission of the Institute. We believe the current system for real-world impact of the life sciences can be greatly optimized. Arc will build translational infrastructure to accelerate the advancement of new biological insights or biotechnologies into impact on patients.</p></div></div><p><span><span><img alt="" aria-hidden="true" src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMjE2MCIgaGVpZ2h0PSIxMTc0IiB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHZlcnNpb249IjEuMSIvPg=="/></span><img alt="Arc Institute headquarters in Palo Alto, California" srcset="/_next/image?url=%2F_next%2Fstatic%2Fmedia%2Farc-institute-hq.dd0fcaef.jpg&amp;w=3840&amp;q=75 1x" src="http://harihareswara.net/_next/image?url=%2F_next%2Fstatic%2Fmedia%2Farc-institute-hq.dd0fcaef.jpg&amp;w=3840&amp;q=75" decoding="async" data-nimg="intrinsic"/></span><span>Arc Institute headquarters in Palo Alto, California</span></p></div></section><section><div><div><p><h2>Open positions</h2></p><p>We have open positions for Technology Center group leaders, research scientists, and operational staff. We will open a call for core and affiliate investigators as well as Institute fellows in early 2022. In the initial phase of the institute, we will scale to a total headcount of approximately 150 scientific personnel. We expect to get there within 3 years.</p><div><p><a href="http://harihareswara.net/jobs">See open positions</a></p></div></div></div></section><section><div><div><div><p><span><span><img alt="" aria-hidden="true" src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iNjQwIiBoZWlnaHQ9IjY0MCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiB2ZXJzaW9uPSIxLjEiLz4="/></span><img alt="Silvana Konermann" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" decoding="async" data-nimg="intrinsic"/></span></p><div><h3>Silvana Konermann</h3><p>Silvana is an Assistant Professor of Biochemistry at Stanford. Her research laboratory aims to understand the molecular pathways that drive the development of Alzheimer’s disease using next-generation functional genomics, with the long-term goal of developing rationally targeted therapeutics for neurodegenerative disorders. She received her Ph.D. in Neuroscience from MIT. Silvana’s pioneering work on tools to directly perturb the transcriptomic landscape of the cell using CRISPR has been recognized by her faculty appointment as a Chan Zuckerberg Biohub Investigator and Hanna Gray Fellow of the Howard Hughes Medical Institute.</p></div></div><div><p><span><span><img alt="" aria-hidden="true" src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iNjQwIiBoZWlnaHQ9IjY0MCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiB2ZXJzaW9uPSIxLjEiLz4="/></span><img alt="Patrick Hsu" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" decoding="async" data-nimg="intrinsic"/></span></p><div><h3>Patrick Hsu</h3><p>Patrick Hsu is an Assistant Professor of Bioengineering and Deb Faculty Fellow at the University of California, Berkeley. His lab develops molecular technologies to improve human health through next-generation diagnostics and therapeutics. Patrick received A.M. and Ph.D. degrees from Harvard University and the Broad Institute, where he helped to pioneer CRISPR-Cas9 technologies for genome editing. His research has been recognized in Forbes&#39; 30 Under 30, the NIH Early Independence Award, the MIT Technology Review&#39;s Innovators Under 35, the Rainwater Prize for Innovative Early Career Scientist, and the Amgen Young Investigator Award.</p></div></div><div><p><span><span><img alt="" aria-hidden="true" src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iNjQwIiBoZWlnaHQ9IjY0MCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiB2ZXJzaW9uPSIxLjEiLz4="/></span><img alt="Lingyin Li" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" decoding="async" data-nimg="intrinsic"/></span></p><div><h3>Lingyin Li</h3><p>Lingyin Li is an Associate Professor in the Biochemistry Department and the ChEM-H institute at Stanford. She received her Ph.D at the University of Wisconsin-Madison with Dr. Laura Kiessling and performed postdoctoral studies with Dr. Timothy Mitchison at Harvard Medical School. The Li Laboratory has pioneered the development of chemical tools to define and manipulate new modulators of the innate immune STING pathway. Dr. Li is the recipient of the NIH New Innovator Award in 2017, the Ono Pharma Foundation Breakthrough Science Initiative Award in 2017, and the 2022 Eli Lilly Award in Biological Chemistry.</p></div></div></div></div></section><section><div><p><svg width="181" height="13" viewBox="0 0 181 13" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path stroke="var(--title-decoration-color, currentColor)" d="m1 11 62-9"></path><path stroke="var(--title-decoration-color, currentColor)" d="m50 11 64-5"></path><path stroke="var(--title-decoration-color, currentColor)" d="m114 6 54 5"></path><path stroke="var(--title-decoration-color, currentColor)" d="m62 2 18 9"></path><circle fill="var(--title-decoration-color, currentColor)" fill-rule="nonzero" cx="49.5" cy="10.5" r="1.5"></circle><circle fill="var(--title-decoration-color, currentColor)" fill-rule="nonzero" cx="1.5" cy="11.5" r="1.5"></circle><circle fill="var(--title-decoration-color, currentColor)" fill-rule="nonzero" cx="62.5" cy="1.5" r="1.5"></circle><circle fill="var(--title-decoration-color, currentColor)" fill-rule="nonzero" cx="102.5" cy="6.5" r="1.5"></circle><circle fill="var(--title-decoration-color, currentColor)" fill-rule="nonzero" cx="61" cy="6" r="1"></circle><circle fill="var(--title-decoration-color, currentColor)" fill-rule="nonzero" cx="57.5" cy="4.5" r="1"></circle><circle fill="var(--title-decoration-color, currentColor)" fill-rule="nonzero" cx="80.5" cy="11.5" r="1.5"></circle><circle fill="var(--title-decoration-color, currentColor)" fill-rule="nonzero" cx="114.5" cy="5.5" r="1.5"></circle><circle fill="var(--title-decoration-color, currentColor)" fill-rule="nonzero" cx="167.5" cy="10.5" r="1.5"></circle><circle fill="var(--title-decoration-color, currentColor)" fill-rule="nonzero" cx="179.5" cy="10.5" r="1.5"></circle></g></svg><h2>Recent work</h2></p><p>Arc seeks to get important discoveries into the public domain as quickly as possible. Below is some recent work from labs led by Arc’s Core Investigators.</p><article><div><h3>Large-scale discovery of recombinases for integrating DNA into the human genome</h3><h5>Preprint</h5><p>Matthew G. Durrant, Alison Fanton, Josh Tycko, Michaela Hinks, Sita S. Chandrasekaran, Nicholas T. Perry, Julia Schaepe, Peter P. Du, Peter Lotfy, Michael C. Bassik, Lacramioara Bintu, Ami S. Bhatt, Patrick D. Hsu</p><p><a href="https://www.biorxiv.org/content/biorxiv/early/2021/11/06/2021.11.05.467528.full.pdf">Download PDF</a></p></div><div><h5><svg width="19" height="24" fill="none" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" clip-rule="evenodd" d="M15.53 0a2.824 2.824 0 0 1 2.823 2.824v18.352A2.823 2.823 0 0 1 15.529 24H2.824A2.824 2.824 0 0 1 0 21.177V2.823A2.824 2.824 0 0 1 2.824 0h12.705Zm0 19.765H2.823v1.412h12.705v-1.412Zm0-3.53h-2.824v1.412h2.823v-1.412Zm-4.236 0h-8.47v1.412h8.47v-1.412Zm4.235-3.53H2.824v1.413h12.705v-1.412Zm0-3.529H2.824v1.412h12.705V9.176Zm0-6.352h-4.235v1.411h4.235V2.824Zm-7.058 0H2.824v1.411H8.47V2.824Z" fill="currentColor"></path></svg><span>Abstract</span></h5><p>Recent microbial genome sequencing efforts have revealed a vast reservoir of mobile genetic elements containing integrases that could be useful genome engineering tools. Large serine recombinases (LSRs), such as Bxb1 and PhiC31, are bacteriophage-encoded integrases that can facilitate the insertion of phage DNA into bacterial genomes. However, only a few LSRs have been previously characterized and they have limited efficiency in human cells. Here, we developed a systematic computational discovery workflow that identifies thousands of new LSRs and their cognate DNA attachment sites by. We validate this approach via experimental characterization of LSRs in human cells, leading to three classes of LSRs distinguished from one another by their efficiency and specificity. We identify landing pad LSRs that efficiently integrate into synthetically installed attachment sites orthogonal to the human genome, human genome-targeting LSRs with computationally predictable pseudosites, and multi-targeting LSRs that can unidirectionally integrate cargos at with similar efficiency and superior specificity to commonly used transposases. LSRs from each category were functionally characterized in human cells, overall achieving up to 7-fold higher plasmid recombination than Bxb1 and genome insertion efficiencies of 40-70% with cargo sizes over 7 kb. Overall, we establish a paradigm for large-scale discovery of microbial recombinases and reconstruction of their target sites directly from microbial sequencing data. This strategy provides a rich resource of over 60 experimentally characterized LSRs that can function in human cells and thousands of additional candidates for large-payload genome editing without exposed DNA double-stranded breaks.</p></div></article><article><div><h3>Deep learning of Cas13 guide activity from high-throughput gene essentiality screening</h3><h5>Preprint</h5><p>Jingyi Wei, Peter Lotfy, Kian Faizi, Hugo Kitano, Patrick D. Hsu, Silvana Konermann</p><p><a href="https://www.biorxiv.org/content/10.1101/2021.09.14.460134v1.full.pdf">Download PDF</a></p></div><div><h5><svg width="19" height="24" fill="none" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" clip-rule="evenodd" d="M15.53 0a2.824 2.824 0 0 1 2.823 2.824v18.352A2.823 2.823 0 0 1 15.529 24H2.824A2.824 2.824 0 0 1 0 21.177V2.823A2.824 2.824 0 0 1 2.824 0h12.705Zm0 19.765H2.823v1.412h12.705v-1.412Zm0-3.53h-2.824v1.412h2.823v-1.412Zm-4.236 0h-8.47v1.412h8.47v-1.412Zm4.235-3.53H2.824v1.413h12.705v-1.412Zm0-3.529H2.824v1.412h12.705V9.176Zm0-6.352h-4.235v1.411h4.235V2.824Zm-7.058 0H2.824v1.411H8.47V2.824Z" fill="currentColor"></path></svg><span>Abstract</span></h5><p>Transcriptome engineering requires flexible RNA-targeting technologies that can perturb mammalian transcripts in a robust and scalable manner. CRISPR systems that natively target RNA molecules, such as Cas13 enzymes, are enabling rapid progress in the investigation of RNA biology and advancement of RNA therapeutics. Here, we sought to develop a Cas13 platform for high-throughput phenotypic screening and elucidate the design principles underpinning its RNA targeting efficiency. We employed the RfxCas13d (CasRx) system in a positive selection screen by tiling 55 known essential genes with single nucleotide resolution. Leveraging this dataset of over 127,000 guide RNAs, we systematically compared a series of linear regression and machine learning algorithms to train a convolutional neural network (CNN) model that is able to robustly predict guide RNA performance based on guide sequence alone. We further incorporated secondary features including secondary structure, free energy, target site position, and target isoform percent. To evaluate model performance, we conducted orthogonal screens via cell surface protein knockdown. The final CNN model is able to predict highly effective guide RNAs (gRNAs) within each transcript with &gt;90% accuracy in this independent test set. To provide user interpretability, we evaluate feature contributions using both integrated gradients and SHapley Additive exPlanations (SHAP). We identify a specific sequence motif at guide position 15-24 along with selected secondary features to be predictive of highly efficient guides. Taken together, we derive Cas13d guide design rules from large-scale screen data, release a guide design tool (http://RNAtargeting.org) to advance the RNA targeting toolbox, and describe a path for systematic development of deep learning models to predict CRISPR activity.</p></div></article><article><div><h3>STING Polymer Structure Reveals Mechanisms for Activation, Hyperactivation, and Inhibition</h3><h5>Cell (2019)</h5><p>Sabrina L Ergun, Daniel Fernandez, Thomas M Weiss, Lingyin Li</p><p><a href="https://www.cell.com/cell/pdf/S0092-8674(19)30564-1.pdf">Download PDF</a></p></div><div><h5><svg width="19" height="24" fill="none" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" clip-rule="evenodd" d="M15.53 0a2.824 2.824 0 0 1 2.823 2.824v18.352A2.823 2.823 0 0 1 15.529 24H2.824A2.824 2.824 0 0 1 0 21.177V2.823A2.824 2.824 0 0 1 2.824 0h12.705Zm0 19.765H2.823v1.412h12.705v-1.412Zm0-3.53h-2.824v1.412h2.823v-1.412Zm-4.236 0h-8.47v1.412h8.47v-1.412Zm4.235-3.53H2.824v1.413h12.705v-1.412Zm0-3.529H2.824v1.412h12.705V9.176Zm0-6.352h-4.235v1.411h4.235V2.824Zm-7.058 0H2.824v1.411H8.47V2.824Z" fill="currentColor"></path></svg><span>Abstract</span></h5><p>How the central innate immune protein, STING, is activated by its ligands remains unknown. Here, using structural biology and biochemistry, we report that the metazoan second messenger 2&#39;3&#39;-cGAMP induces closing of the human STING homodimer and release of the STING C-terminal tail, which exposes a polymerization interface on the STING dimer and leads to the formation of disulfide-linked polymers via cysteine residue 148. Disease-causing hyperactive STING mutations either flank C148 and depend on disulfide formation or reside in the C-terminal tail binding site and cause constitutive C-terminal tail release and polymerization. Finally, bacterial cyclic-di-GMP induces an alternative active STING conformation, activates STING in a cooperative manner, and acts as a partial antagonist of 2&#39;3&#39;-cGAMP signaling. Our insights explain the tight control of STING signaling given varying background activation signals and provide a therapeutic hypothesis for autoimmune syndrome treatment.</p></div></article></div></section></div></div>
  </body>
</html>
