<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <title>James Routley | Feed</title>
    <link
      rel="stylesheet"
      type="text/css"
      href="../styles.css"
      media="screen"
    />
  </head>
  <body>
    <a href="/index.html">Back</a>
    <a href="https://academic.oup.com/mbe/article/34/4/1021/3065792">Original</a>
    <h1>Arctic Inuit, Native American adaptations to cold and body fat distribution</h1>
    
    <div id="readability-page-1" class="page"><div>
        <div>
    <div data-widgetname="ArticleFulltext">





<p>In the Arctic, the Inuits have adapted to severe cold and a predominantly seafood diet. After the first population genomic analysis of the Greenland Inuits (Fumagalli, Moltke et al. 2015, Science doi:10.1126/science.aab2319), a region in the genome containing two genes has now been scrutinized by scientists: <em>TBX15</em> and <em>WARS2</em>. This region is thought to be central to cold adaptation by generating heat from a specific type of body fat, and was earlier found to be a candidate for adaptation in the Inuits. </p><p>Now, a team of scientists led by Fernando Racimo, Rasmus Nielsen et al. have followed up on the first natural selection study in Inuits to trace back the origins of these adaptations.</p><p>To perform the study, they used the genomic data from nearly 200 Greenlandic Inuits and compared this with the 1000 Genomes Project and ancient hominid DNA from Neanderthals and Denisovans. The results provide convincing evidence that the Inuit variant of the <em>TBX15/WARS2</em> region first came into modern humans from an archaic hominid population, likely related to the Denisovans.</p><p>“The Inuit DNA sequence in this region matches very well with the Denisovan genome, and it is highly differentiated from other present-day human sequences, though we can’t discard the possibility that the variant was introduced from another archaic group whose genomes we haven’t sampled yet”, said Fernando <span id="jumplink-msw300-B1"></span>Racimo (2016), lead author of the study.</p><p>The authors found that the variant is present at low-to-intermediate frequencies throughout Eurasia, and at especially high frequencies in the Inuits and Native American populations, but almost absent in Africa. <em>TBX15</em> is a gene known to affect the human body’s response to cold, and is associated with a number of traits related to body fat distribution. The authors speculate that the archaic variant may have been beneficial to modern humans during their expansion throughout Siberia and across Beringia, into the Americas.</p><p>The research team also worked to understand the physiological role of the region, which may be of interest to scientists concerned with factors that help determine BMI index and fat metabolism. They found an association between the archaic region and the gene expression of <em>TBX15</em> and <em>WARS2</em> in various tissues, like fibroblasts and adipose tissue. They also observed that the methylation patterns in this region in the Denisovan genome are very different from those of Neanderthals and present-day humans. “All this suggests that the introduced variant may have altered the regulation of these genes, thought the exact mechanism by which this occurred remains elusive”, said Racimo, who was a graduate student in UC Berkeley at the time of the study, and now works at the New York Genome Center.</p><p>The evidence adds to the remarkable number of recent examples of ancient interbreeding that may have conferred unique adaptive traits to modern humans, either from Neanderthals or Denisovans. And it is the second major example—the other being the <em>EPAS1</em> genomic locus (found in the high altitude adaptation of Tibetans) to be passed on from archaic humans into the modern human gene pool.</p>                    <h2 scrollto-destination="63711767" id="63711767">Reference</h2>
<div><div content-id="msw300-B1" data-legacy-id="msw300-B1"><div><div id="ref-auto-msw300-B1" data-id="msw300-B1"><div><p>Racimo</p> <p>F</p><p>, et al. . </p><p>2016</p><p>. 
</p><p>Archaic Adaptive Introgression in TBX15/WARS2</p><p>. </p><p>Mol Biol Evol.</p> <p>34</p><p>(</p><p>3</p><p>):</p><p>509</p><p>–</p><p>524</p><p>.</p><!--citationLinks: case 2--></div></div></div></div></div>    <!-- /foreach in Model.Sections -->



        

        
<p>© The Author 2017. Published by Oxford University Press on behalf of the Society for Molecular Biology and Evolution. All rights reserved. For permissions, please e-mail: journals.permissions@oup.com</p><!-- /foreach -->

        </div><!-- /.widget-items -->
</div><!-- /.module-widget -->





 
    </div></div>
  </body>
</html>
