<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <title>James Routley | Feed</title>
    <link
      rel="stylesheet"
      type="text/css"
      href="../styles.css"
      media="screen"
    />
  </head>
  <body>
    <a href="/index.html">Back</a>
    <a href="https://www.nature.com/articles/s41586-023-06600-9">Original</a>
    <h1>Assembly theory explains and quantifies selection and evolution</h1>
    
    <div id="readability-page-1" class="page"><div>
                <section data-title="Main"><div id="Sec1-section"><h2 id="Sec1">Main</h2><div id="Sec1-content"><p>In evolutionary theory, natural selection<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 1" title="Kauffman, S. A. The Origins of Order: Self-organization and Selection in Evolution (Oxford Univ. Press, 1993)." href="https://www.nature.com/articles/s41586-023-06600-9#ref-CR1" id="ref-link-section-d35886278e434">1</a></sup> describes why some things exist and others do not<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 2" title="Gregory, T. R. Understanding natural selection: essential concepts and common misconceptions. Evol. Educ. Outreach 2, 156–175 (2009)." href="https://www.nature.com/articles/s41586-023-06600-9#ref-CR2" id="ref-link-section-d35886278e438">2</a></sup>. Darwin’s theory of evolution and its modern synthesis point out how selection among variants in the past generates current functionality<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 3" title="Darwin, C. On the Origin of Species by Means of Natural Selection, or, The Preservation of Favoured Races in the Struggle for Life (Natural History Museum, 2019)." href="https://www.nature.com/articles/s41586-023-06600-9#ref-CR3" id="ref-link-section-d35886278e442">3</a></sup>, as well as a forward-looking process<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 4" title="Frank, S. A. &amp; Fox, G. A. in The Theory of Evolution (eds Scheiner, S. M. &amp; Mindell D. P.) 171–193 (Univ. of Chicago Press, 2020)." href="https://www.nature.com/articles/s41586-023-06600-9#ref-CR4" id="ref-link-section-d35886278e446">4</a></sup>. Neither addresses the space in which new phenotypic variants are generated. Physics can, in theory, take us from past initial conditions to current and future states. However, because physics has no functional view of the Universe, it cannot distinguish novel functional features from random fluctuations, which means that talking about true novelty is impossible in physical reductionism. Thus, the open-ended generation of novelty<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 5" title="Carroll, S. B. Chance and necessity: the evolution of morphological complexity and diversity. Nature 409, 1102–1109 (2001)." href="https://www.nature.com/articles/s41586-023-06600-9#ref-CR5" id="ref-link-section-d35886278e450">5</a></sup> does not fit cleanly in the paradigmatic frameworks of either biology<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 6" title="Chesson, P. Mechanisms of maintenance of species diversity. Annu. Rev. Ecol. Syst. 31, 343–366 (2000)." href="https://www.nature.com/articles/s41586-023-06600-9#ref-CR6" id="ref-link-section-d35886278e455">6</a></sup> or physics<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Newton, I. Newton’s Principia. The Mathematical Principles of Natural Philosophy (Daniel Adee, 1846)." href="https://www.nature.com/articles/s41586-023-06600-9#ref-CR7" id="ref-link-section-d35886278e459">7</a></sup>, and so must resort ultimately to randomness<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 8" title="Cross, M. C. &amp; Hohenberg, P. C. Pattern formation outside of equilibrium. Rev. Mod. Phys. 65, 851–1112 (1993)." href="https://www.nature.com/articles/s41586-023-06600-9#ref-CR8" id="ref-link-section-d35886278e463">8</a></sup>. There have been several efforts to explore the gap between physics and evolution<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 9" title="Tilman, D. Resource Competition and Community Structure. (MPB-17) Vol. 17 (Princeton Univ. Press, 2020)." href="https://www.nature.com/articles/s41586-023-06600-9#ref-CR9" id="ref-link-section-d35886278e467">9</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 10" title="Elena, S. F., Cooper, V. S. &amp; Lenski, R. E. Punctuated evolution caused by selection of rare beneficial mutations. Science 272, 1802–1804 (1996)." href="https://www.nature.com/articles/s41586-023-06600-9#ref-CR10" id="ref-link-section-d35886278e470">10</a></sup>. This is because a growing state space over time requires the exploration of a large combinatorial set of possibilities<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 11" title="Lutz, E. Power-law tail distributions and nonergodicity. Phys. Rev. Lett. 93, 190602 (2004)." href="https://www.nature.com/articles/s41586-023-06600-9#ref-CR11" id="ref-link-section-d35886278e474">11</a></sup>, such as in the theory of the adjacent possible<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 12" title="Cortês, M., Kauffman, S. A., Liddle, A. R. &amp; Smolin, L. The TAP equation: evaluating combinatorial innovation in biocosmology. Preprint at 
                http://arxiv.org/abs/2204.14115
                
               (2023)." href="https://www.nature.com/articles/s41586-023-06600-9#ref-CR12" id="ref-link-section-d35886278e478">12</a></sup>. However, the search generates an unsustainable expansion in the number of configurations possible in a finite universe in finite time, and does not include selection. In addition, this approach has limited predictive power with respect to why only some evolutionary innovations happen and not others. Other efforts have studied the evolution of rules acting on other rules<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 13" title="Fontana, W. &amp; Buss, L. W. in Boundaries and Barriers (eds Casti, J. &amp; Karlqvist, A.) 56–116 (Addison-Wesley, 1996)." href="https://www.nature.com/articles/s41586-023-06600-9#ref-CR13" id="ref-link-section-d35886278e483">13</a></sup>; however, these models are abstract so it is difficult to see how they can describe—and predict—the evolution of physical objects.</p><p>Here, we introduce AT, which addresses these challenges by describing how novelty generation and selection can operate in forward-evolving processes. The framework of AT allows us to predict features of new discoveries during selection, and to quantify how much selection was necessary to produce observed objects<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 14" title="Marshall, S. M., Murray, A. R. G. &amp; Cronin, L. A probabilistic framework for identifying biosignatures using Pathway Complexity. Philos. Trans. R. Soc. Math. Phys. Eng. Sci. 375, 20160342 (2017)." href="https://www.nature.com/articles/s41586-023-06600-9#ref-CR14" id="ref-link-section-d35886278e490">14</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 15" title="Marshall, S. M., Moore, D. G., Murray, A. R. G., Walker, S. I. &amp; Cronin, L. Formalising the pathways to life using assembly spaces. Entropy 24, 884 (2022)." href="https://www.nature.com/articles/s41586-023-06600-9#ref-CR15" id="ref-link-section-d35886278e493">15</a></sup> without having to prespecify individuals or units of selection. In AT, objects are not considered as point particles (as in most physics), but are defined by the histories of their formation as an intrinsic property, mapped as an assembly space. The assembly space is defined as the pathway by which a given object can be built from elementary building blocks, using only recursive operations. For the shortest path, the assembly space captures the minimal memory, in terms of the minimal number of operations necessary to construct an observed object based on objects that could have existed in its past<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 16" title="Liu, Y. et al. Exploring and mapping chemical space with molecular assembly trees. Sci. Adv. 7, eabj2465 (2021)." href="https://www.nature.com/articles/s41586-023-06600-9#ref-CR16" id="ref-link-section-d35886278e497">16</a></sup>. One feature of biological assemblies of objects is multiple realizability wherein biological evolution can produce functionally equivalent classes of objects with modular use of units in many different contexts. For each unit, the minimal assembly is unique and independent of its formation, and therefore accounts for multiple realizability in how it could be constructed<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 17" title="Ellis, G. F. R. Top-down causation and emergence: some comments on mechanisms. Interface Focus 2, 126–140 (2012)." href="https://www.nature.com/articles/s41586-023-06600-9#ref-CR17" id="ref-link-section-d35886278e501">17</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 18" title="Koskinen, R. Multiple realizability as a design heuristic in biological engineering. Eur. J. Philos. Sci. 9, 15 (2019)." href="https://www.nature.com/articles/s41586-023-06600-9#ref-CR18" id="ref-link-section-d35886278e504">18</a></sup>.</p><p>We introduce the foundations of AT and its implementation to quantify the degree of selection and evolution found in a collection of objects. Assembly is a function of two quantities: the number of copies of the observed objects and the objects’ assembly indices (an assembly index is the number of steps on a minimal path producing the object). Assembly captures the amount of memory necessary to produce a selected configuration of historically contingent objects in a manner similar to how entropy quantifies the information (or lack thereof) necessary to specify the configuration of an ensemble of point particles, but assembly differs from entropy because of its explicit dependence on the contingency in construction paths intrinsic to complex objects. We demonstrate how AT leads to a unified language for describing selection and the generation of novelty, and thereby produce a framework to unify descriptions of selection across physics and biology.</p></div></div></section><section data-title="Assembly theory"><div id="Sec2-section"><h2 id="Sec2">Assembly theory</h2><p>The concept of an object in AT is simple and rigorously defined. An object is finite, is distinguishable, persists over time and is breakable such that the set of constraints to construct it from elementary building blocks is quantifiable. This definition is, in some sense, opposite to standard physics, which treats objects of interest as fundamental and unbreakable (for example, the concept of ‘atoms’ as indivisible, which now applies to elementary particles). In AT, we recognize that the smallest unit of matter is typically defined by the limits of observational measurements and may not itself be fundamental. A more universal concept is to treat objects as anything that can be broken and built. This allows us to naturally account for the emergent objects produced by evolution and selection as fundamental to the theory. The concept of copy number is of foundational importance in defining a theory that accounts for selection. The more complex a given object, the less likely an identical copy can exist without selection of some information-driven mechanism that generates that object. An object that exists in multiple copies allows the signatures describing the set of constraints that built it to be measured experimentally. For example, mass spectrometry can be used to measure assembly for molecules, because it can measure how molecules are built by making bonds<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 19" title="Marshall, S. M. et al. Identifying molecules as biosignatures with assembly theory and mass spectrometry. Nat. Commun. 12, 3033 (2021)." href="https://www.nature.com/articles/s41586-023-06600-9#ref-CR19" id="ref-link-section-d35886278e519">19</a></sup>.</p></div></section><section data-title="Assembly index and copy number"><div id="Sec3-section"><h2 id="Sec3">Assembly index and copy number</h2><div id="Sec3-content"><p>To construct an assembly space for an object, one starts from elementary building blocks comprising that object and recursively joins these to form new structures, whereby, at each recursive step, the objects formed are added back to the assembly pool and are available for subsequent steps (Supplementary Information Sections <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41586-023-06600-9#MOESM1">1</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41586-023-06600-9#MOESM1">2</a>). AT captures symmetry breaking arising along construction paths due to recursive use of past objects that can be combined in different ways to make new things. For any given object <i>i</i>, we can define its assembly space as all recursively assembled pathways that produce it. For each object, the most important feature is the assembly index <span>\({a}_{i}\)</span>, which corresponds to the shortest number of steps required to generate the object from basic building blocks. This can be quantified as the length of the shortest assembly pathway that can generate the object (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06600-9#Fig1">1)</a>.</p><div data-test="figure" data-container-section="figure" id="figure-1" data-title="Assembly index and shortest path(s)."><figure><figcaption><b id="Fig1" data-test="figure-caption-text">Fig. 1: Assembly index and shortest path(s).</b></figcaption><div><div><a data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="https://www.nature.com/articles/s41586-023-06600-9/figures/1" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41586-023-06600-9/MediaObjects/41586_2023_6600_Fig1_HTML.png?as=webp"/><img aria-describedby="Fig1" src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41586-023-06600-9/MediaObjects/41586_2023_6600_Fig1_HTML.png" alt="figure 1" loading="lazy" width="685" height="576"/></picture></a></div><p><b>a</b>–<b>c</b>, AT is generalizable to different classes of objects, illustrated here for three different general types. <b>a</b>, Assembly pathway to construct diethyl phthalate molecule considering molecular bonds as the building blocks. The figure shows the pathway starting with the irreducible constructs to create the molecule with assembly index 8. <b>b</b>, Assembly pathway of a peptide chain by considering building blocks as strings. Left, four amino acids as building blocks. Middle, the actual object and its representation as a string. Right, assembly pathway to construct the string. <b>c</b>, Generalized assembly pathway of an object comprising discrete components.</p></div></figure></div><p>In chemical systems, molecular assembly theory treats bonds as the elementary operations from which molecules are constructed. The shortest path to build a given molecule can be found by breaking its bonds and then ordering its motifs in order of size, starting from atoms and moving to larger motifs by adding bonds in sequence. Given a motif generated on the path, the motif remains available for reuse. The recursivity allows identifying the shortest construction path with parts already built on that path, allowing us to quantify the minimum number of constraints, or memory size, to construct the molecule. The assembly index can be estimated from any complex discrete object with well-defined building blocks, which can be broken apart, as shown in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06600-9#Fig1">1</a>. At every step, the size of the object increases by at least one. The number of total possible steps, although potentially large, is always finite for any finite object and thus the assembly index is computable in finite time. For molecules, the assembly index can be determined experimentally.</p><p>A hallmark feature of life is how complex objects are generated by evolution, of which many are functional. For example, a DNA molecule holds genetic information reliably and can be copied easily. By contrast, a random string of letters requires much information to describe it, but is not normally seen as very complex or useful. Thus far, science has not been able to find a measure that quantifies the complexity of functionality to distinguish these two cases. Here we overcome this inherent problem by pointing out another feature of the evolutionary process: the complex and functional objects it generates take many steps to make, and selection allows many identical copies of these objects. Therefore, an evolutionary process can be identified by the production of many identical, or near-identical, multistep objects. The assembly index on its own cannot detect selection, but copy number combined with the assembly index can. This approach defines a new way to measure complexity in terms of the hierarchy of causation stemming from selection at different levels.</p><p>Because we do not typically know the full assembly trajectory of an object, we instead adopt a conservative alternative. AT finds the minimal number of steps to produce the object. We assume that every subobject, once available, can be used as often as needed to generate the object. A different approach would be to use Kolmogorov complexity<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 20" title="Arora, S. &amp; Barak, B. Computational Complexity: A Modern Approach (Cambridge Univ. Press, 2009)." href="https://www.nature.com/articles/s41586-023-06600-9#ref-CR20" id="ref-link-section-d35886278e618">20</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 21" title="Wallace, C. S. Minimum message length and Kolmogorov complexity. Comput. J. 42, 270–283 (1999)." href="https://www.nature.com/articles/s41586-023-06600-9#ref-CR21" id="ref-link-section-d35886278e621">21</a></sup> applied to a given molecule, but this requires starting with a graphical representation, and a program to compute the graph of that molecule. The Kolmogorov complexity of a string is the shortest program that will output that string for a programming language capable of universal computation. This measure cannot be easily computed, because checking whether any single program will output the string is uncomputable, as it involves, at least, deciding whether the program stops. Running this program reflects nothing of the underlying process of how the molecule was constructed. Only late in the evolutionary process will molecules be produced by anything starting to resemble Turing machines, loops, stacks, tapes and so on<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 22" title="Bennett, C. H. in The Universal Turing Machine: A Half Century Survey (ed. Herken, R.) 227–257 (Oxford Univ. Press, 1988)." href="https://www.nature.com/articles/s41586-023-06600-9#ref-CR22" id="ref-link-section-d35886278e625">22</a></sup>. Thus, using universal computation to assess molecules adds unrealistic dynamics, making the answer uncomputable. The assembly measure that we have presented here both uses realistic dynamics for molecules, using bonds as building blocks, and is computable for any molecule. The main work for detecting evolution and memory is done here by combining the assembly index and copy number of the objects.</p><p>The aim of AT is to develop a new understanding of the evolution of complex matter that naturally accounts for selection and history in terms of what operations are physically possible in constructing an object<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 23" title="Deutsch, D. &amp; Marletto, C. Constructor theory of information. Proc. R. Soc. Math. Phys. Eng. Sci. 471, 20140540 (2015)." href="https://www.nature.com/articles/s41586-023-06600-9#ref-CR23" id="ref-link-section-d35886278e633">23</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 24" title="Marletto, C. Constructor theory of life. J. R. Soc. Interface 12, 20141226 (2015)." href="https://www.nature.com/articles/s41586-023-06600-9#ref-CR24" id="ref-link-section-d35886278e636">24</a></sup>. We will discuss AT as applied to chemical systems as the main application in this manuscript because their assembly index has been experimentally measured. For molecules, assembly index has a clear physical interpretation and has been validated as quantifying evidence of selection in its application to the detection of molecular signatures of life. However, we anticipate the theory to be sufficiently general to apply to a wide variety of other systems including polymers, cell morphology, graphs, images, computer programs, human languages and memes, as well as many others. The challenge in each case will be to construct an assembly space that has a clear physical meaning in terms of what operations can be caused to occur to make the object<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 23" title="Deutsch, D. &amp; Marletto, C. Constructor theory of information. Proc. R. Soc. Math. Phys. Eng. Sci. 471, 20140540 (2015)." href="https://www.nature.com/articles/s41586-023-06600-9#ref-CR23" id="ref-link-section-d35886278e640">23</a></sup> (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06600-9#Fig1">1</a>).</p><p>In AT there are two important features of the context the object is found in. First, there must be objects in its environment that can constrain the steps to assemble the object and second these objects themselves have been selected because they must be retained over subsequent steps to physically instantiate the memory needed to build the target object. Among the most relatable examples are enzyme catalysts in biochemistry, which permit the formation of very unlikely molecules in large numbers because the enzymes themselves are also selected to exist with many copies. We make no distinction between the traditional notion of biological ‘individual’ and objects that are selected in the environment to quantify the selection necessary to produce a given configuration. Thus, our approach naturally accounts for well-known phenomena, such as niche construction, whereby organisms and environment are co-constructed and co-selected.</p><p>Copy number is important because a single example of a highly complex molecule (with a very high assembly index) could potentially be generated in a series of random events that become increasingly less likely with increasing assembly index. If we consider a forward-building assembly process (see Supplementary Information Sections <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41586-023-06600-9#MOESM1">1</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41586-023-06600-9#MOESM1">2</a> for details), without a specific target in mind, the number of possible objects that could be built at each recursive step grows super-exponentially in the absence of any constraints. The likelihood of finding and measuring more than one copy of an object therefore decreases super-exponentially with increasing assembly index in the absence of selection for a specified target. Objects with high assembly index, found in abundance, provide evidence of selection because of the combinatorially growing space of possible objects at each recursive assembly step (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06600-9#Fig2">2)</a>. Finding more than one identical copy indicates the presence of a non-random process generating the object.</p><div data-test="figure" data-container-section="figure" id="figure-2" data-title="Selection in assembly space."><figure><figcaption><b id="Fig2" data-test="figure-caption-text">Fig. 2: Selection in assembly space.</b></figcaption><div><div><a data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="https://www.nature.com/articles/s41586-023-06600-9/figures/2" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41586-023-06600-9/MediaObjects/41586_2023_6600_Fig2_HTML.png?as=webp"/><img aria-describedby="Fig2" src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41586-023-06600-9/MediaObjects/41586_2023_6600_Fig2_HTML.png" alt="figure 2" loading="lazy" width="685" height="775"/></picture></a></div><p><b>a</b>, Pictorial representation of the assembly space representing the formation of combinatorial object space from building blocks and physical constraints. <b>b</b>, Observed copy number distributions of objects at different assembly indices as an outcome of selection or no selection. <b>c</b>, Representation of physical pathways to construct objects with undirected and directed pathways (selected) leading to the low and high copy numbers of the observed object.</p></div></figure></div></div></div></section><section data-title="The assembly equation"><div id="Sec4-section"><h2 id="Sec4">The assembly equation</h2><div id="Sec4-content"><p>We define assembly as the total amount of selection necessary to produce an ensemble of observed objects, quantified using equation (<a data-track="click" data-track-label="link" data-track-action="equation anchor" href="https://www.nature.com/articles/s41586-023-06600-9#Equ1">1</a>):</p><div id="Equ1"><p><span>$$A=\mathop{\sum }\limits_{i=1}^{N}{e}^{{a}_{i}}\left(\frac{{n}_{i}-1}{{N}_{{\rm{T}}}}\right)$$</span></p><p>
                    (1)
                </p></div><p>where <span>\(A\)</span> is the assembly of the ensemble, <span>\({a}_{i}\)</span> is the assembly index of object <span>\(i\)</span>, <span>\({n}_{i}\)</span> is its copy number, <i>N</i> is the total number of unique objects, <i>e</i> is Euler’s number and <span>\({N}_{{\rm{T}}}\)</span> is the total number of objects in the ensemble. Normalizing by the number of objects in the ensemble allows assembly to be compared between ensembles with different numbers of objects.</p><p>Assembly quantifies two competing effects, the difficulty of discovering new objects, but, once discovered, some objects become easier to make; this is indicative of how selection was required to discover and make them. The exponential growth of assembly with depth in assembly space, as quantified by assembly index, is derived by considering a linearly expanding assembly pool that has objects that combine at step <span>\(a\to a+1\)</span>, whereby an object at the assembly index <span>\(a\)</span> combines with another object from the assembly pool. Discovering new objects at increasing depth in an assembly space gets increasingly harder with depth because the space of possibilities expands exponentially. Once the pathway for a new object has been discovered, the production of an object (copy number greater than 1) gets easier as the copy number increases because a high copy number implies that an object can be produced readily in a given context. Thus, the hardest innovation is making an object for the first time, which is equivalent to its discovery, followed by making the first copy of that object, but once an object exists in very high abundance it must already be relatively easy to make. Hence, assembly (<i>A</i>) scales linearly with copy number for more than one object for a fixed cost per object once a process has been discovered (see Supplementary Information Section <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41586-023-06600-9#MOESM1">3</a> for additional details).</p><p>Increasing assembly (<span>\(A\)</span>) results from increasing copy numbers <span>\(n\)</span> and increasing assembly indices <span>\(a\)</span>. If high values of assembly can be shown to capture cases in which selection has occurred, it implies that finding high assembly index objects in high abundance is a signature of selection. In AT, the information required at each step to construct the object is ‘stored’ within the object (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06600-9#Fig2">2</a>). Each time two objects are combined from an assembly pool, the specificity of the combination process constitutes selection. As we will show, randomly combining objects within the assembly pool at each step does not constitute selection because no combinations exist in memory to be used again for building the same object. If, instead, certain combinations are preferentially used, it implies that a mechanism exists that selects the specific operations and, by extension, specific target objects to be generated. Later we will quantify the degree of selectivity by parameter <span>\(\alpha \)</span> in the growth dynamics, which allows parameterizing selection in an empirically observable manner by parameterizing reuses of specific sets of operations (see Supplementary Information Section <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41586-023-06600-9#MOESM1">3</a> for example).</p><p>Assembly as given in equation (<a data-track="click" data-track-label="link" data-track-action="equation anchor" href="https://www.nature.com/articles/s41586-023-06600-9#Equ1">1</a>) is determined for identified finite and distinguishable objects (with copy number greater than 1) and their distinct assembly spaces. However, in real samples, there are almost always several different coexisting objects, which will include a common history for their formation. Transistors, for example, are used across several different technologies, suggesting a common subspace in the assembly spaces of many modern technologies that includes transistor-like objects. This common subspace, constituting the overlap in the assembly paths of distinct structures, is called a co-assembly space. By contrast, a joint assembly space of several objects is the combined assembly space required to generate those objects. As a potential extension of the assembly equation, to account for the joint assembly of objects, we expand the formulation of the assembly equation that includes the quantification of shared pathways to construct objects to determine the assembly (<span>\(A\)</span>) of an ensemble with different objects that share common history (Supplementary Information Section <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41586-023-06600-9#MOESM1">3</a>).</p></div></div></section><section data-title="Selection within assembly spaces"><div id="Sec5-section"><h2 id="Sec5">Selection within assembly spaces</h2><div id="Sec5-content"><p>The concept of the assembly space allows us to understand how selection and historical contingency impose constraints on what can be made in the future. By aiming to detect ‘selection’, we mean a process similar to selection in Darwinian evolution. We do not, however, model functional differences that selection might act on. Instead, we account only for the specificity of selection—that some objects are more likely to be used to make new things and some are less likely. The only functionality we want to detect or describe is in the memory of the process to generate the object, with examples including a metabolic reaction network or a genome. This allows the three Lewontin conditions for evolution to hold<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 25" title="Lewontin, R. C. The units of selection. Annu. Rev. Ecol. Syst. 1, 1–18 (1970)." href="https://www.nature.com/articles/s41586-023-06600-9#ref-CR25" id="ref-link-section-d35886278e1115">25</a></sup>. A key feature of assembly spaces is that they are combinatorial, with objects combined at every step. Combinatorial spaces do not play a prominent role in current physics, because their objects are modelled as point particles and not as combinatorial objects (with limited exceptions). However, combinatorial objects are important in chemistry, biology and technology, in which most objects of interest (if not all) are hierarchical modular structures. More objects exist in assembly space than can be built in finite time with finite resources because the space of possibilities grows super-exponentially with the assembly index. To tame this explosive growth, in AT historical contingency is intrinsic with the space built compositionally, where items are combined recursively (accounting for hierarchical modularity) and this substantially constrains the number of possible objects. It is the combination of this compositionality with combinatorics that allows us to describe selection (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06600-9#Fig3">3</a>).</p><div data-test="figure" data-container-section="figure" id="figure-3" data-title="Assembly spaces."><figure><figcaption><b id="Fig3" data-test="figure-caption-text">Fig. 3: Assembly spaces.</b></figcaption><div><div><a data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="https://www.nature.com/articles/s41586-023-06600-9/figures/3" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41586-023-06600-9/MediaObjects/41586_2023_6600_Fig3_HTML.png?as=webp"/><img aria-describedby="Fig3" src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41586-023-06600-9/MediaObjects/41586_2023_6600_Fig3_HTML.png" alt="figure 3" loading="lazy" width="685" height="329"/></picture></a></div><p><b>a</b>, Assembly observed of the three objects shown as graphs (P<sub>1</sub>, P<sub>2</sub> and P<sub>3</sub>) with their shared minimal construction process called their ‘joint assembly space’. <b>b</b>, Illustration of the expansion of the assembly universe, assembly possible, assembly contingent and assembly observed (see text for details). Assembly universe has no dynamics and is displayed with assembly steps as the time axis. Note that the figure illustrates their nested structure only, not the relative size of the spaces where each set is typically exponentially larger than the subset.</p></div></figure></div><p>To produce an assembly space, an observed object is broken down recursively to generate a set of elementary building units. These units can be used to then recursively construct the assembly pathways of the original object(s) to build what we call assembly observed, <i>A</i><sub>O</sub>. <i>A</i><sub>O</sub> captures all histories for the construction of the observed object(s) from elementary building blocks, consistent with what physical operations are possible. Because objects in AT are compositional, they contain information about the larger space of possible objects from which they were selected. To see how, we first build an assembly space from the same building blocks in <i>A</i><sub>O</sub>, which include all possible pathways for assembling any object composed of the same set elementary building blocks as our target object. The space so constructed is the assembly universe (<i>A</i><sub>U</sub>).</p><p>In the assembly universe, all objects are possible with no rules, yielding a combinatorial explosion and with double exponential growth in the number of objects, as is characteristic of exploding state spaces and the adjacent possible (see Supplementary Information Section <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41586-023-06600-9#MOESM1">4</a> for details). Although mathematically well defined, this double exponential growth is unphysical because the physical processes place restrictions on what is possible (in the case of molecules, an example is how quantum mechanics constrains the numbers of bonds per atom). The assembly universe also has no concept of directionality in time, as there is no ordering to construction processes. Because everything can exist, there is an implication that objects can be constructed independently of what has existed in the past and of resource or time constraints, which is not what we observe in the real universe. For most systems of interest, including in molecular assembly spaces, the number of molecules in the assembly universe is orders of magnitude larger than the amount of matter available in the cosmologically observable universe. There is no way to computationally build and exhaust the entire space, even for objects with relatively low assembly indices. For larger objects, such as proteins, this can be truly gigantic<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 26" title="Beasley, J. R. &amp; Hecht, M. H. Protein design: the choice of de novo sequences. J. Biol. Chem. 272, 2031–2034 (1997)." href="https://www.nature.com/articles/s41586-023-06600-9#ref-CR26" id="ref-link-section-d35886278e1179">26</a></sup>. In AT, we do not observe all possible objects at a given depth in the assembly space because of selection, more reflective of what we see in the real universe. We next show how taking account of memory and resource limitation severely restricts the size of the space of what can be built, but also allows higher-assembly objects to be built before exhausting resources constructing all the possible lower-assembly objects. AT can account for selection precisely because of the historical contingency in the recursive construction of objects along assembly paths.</p><p>Assembly possible (<i>A</i><sub>P</sub>) is the space of physically possible objects, which can be generated by means of the combinatorial expansion of all the known physical rules of object construction and allowing all rules to be available at every step to every object. This can be described by a dynamical model representing undirected forward dynamics in AT. When an object with assembly index <span>\(a\)</span> combines with its own history, its assembly index increases by one, <span>\(a\to a+1\)</span>. If the resulting object can be made by means of other, shorter path(s), its assembly index will be smaller than <span>\(a+1\)</span> or even <span>\(a\)</span>. Another assumption behind the dynamical model of undirected dynamics is a microscopically driven stochastic rule that uses existing objects uniformly: the probability of choosing an object with assembly index <span>\(a\)</span> to be combined with any other object is proportional to <span>\({N}_{a}\)</span>, the number of objects with assembly index <span>\(a\)</span> (see Supplementary Information Section <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41586-023-06600-9#MOESM1">5</a> for further details).</p><p>Within assembly possible, assembly contingent (<i>A</i><sub>C</sub>) describes the possible space of objects where history, and selection on that history, matter. Historical contingency is introduced by assuming that only the knowledge or constraints built on a given path can be used in the future, or with different paths interacting in cases in which selected objects that had not interacted previously now interact. We define the probability <span>\({P}_{a}\)</span> of an object being selected with assembly index (<span>\(a\)</span>) as <span>\({P}_{a}\propto {({N}_{a})}^{\alpha }\)</span>, where <span>\({N}_{a}\)</span> is the number of objects with assembly index <span>\(a\)</span>. Here, <span>\(\alpha \)</span> parameterizes the degree of selection: for <span>\(\alpha =1\)</span> all objects that have been assembled in the past are available for reuse, and for <span>\(0\le \alpha &lt; 1\)</span>, only a subset (that grows non-linearly with assembly index) are available for reuse, indicating that selection has occurred. This leads to the growth dynamics:</p><div id="Equ2"><p><span>$$\frac{{\rm{d}}{N}_{a+1}}{{\rm{d}}t}={k}_{{\rm{d}}}{({N}_{a})}^{\alpha }$$</span></p><p>
                    (2)
                </p></div><p>where <span>\({k}_{{\rm{d}}}\)</span> represents the rate of discovery (expansion rate) of new objects. For <span>\(\alpha =1\)</span>, there is historical dependence without selection. We build assembly paths by taking two randomly chosen objects from the assembly pool and combining them; if a new object is formed, it is added back into the pool. Here we are building random objects, but these are fundamentally different from random combinatorial objects because the randomness we implement is distributed across the recursive construction steps leading to an object (see Supplementary Information Section <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41586-023-06600-9#MOESM1">5</a> for solutions). The case of <span>\(\alpha =1\)</span>, in which there is historical dependence but no selection, defines the boundary of assembly possible.</p><p>Within assembly possible, the assembly contingent (<i>A</i><sub>C</sub>) is the space of possible configurations of objects where <span>\(0\le \alpha &lt; 1\)</span>, that is, where selection is possible, and the objects found in the space are controlled by a path-dependency contingent on each object that has already been built. The growth of the assembly contingent is much slower than exponential; indeed, not all possible paths are explored equally. Instead, the dynamics are channelled by constraints imposed by the selectivity emerging along specific paths. Indeed, a signature of selection in assembly spaces is a slower-than-exponential growth of the number of unique objects. To show this, we use a simple phenomenological model of linear polymers to demonstrate how assembly differentiates cases when selection happens. Starting with a single monomer in the assembly pool, the undirected exploration process combines two randomly selected polymers and adds them back to the assembly pool. In the case of directed exploration with selection, the polymer that has been created most recently is selected to join a randomly selected polymer from the assembly pool. For both directed and undirected exploration, this process was iterated up to 10<sup>4</sup> steps and repeated 25 times. For each observed polymer in the assembly pool, the shortest pathway was generated. For each run, the assembly space of multiple coexisting polymers, their joint assembly space, was approximated by the union of the shortest pathways of all observed polymers. An example of joint assembly space in an undirected exploration up to 30 steps is shown in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06600-9#Fig4">4a</a>.</p><div data-test="figure" data-container-section="figure" id="figure-4" data-title="Undirected and directed exploration in a forward assembly process."><figure><figcaption><b id="Fig4" data-test="figure-caption-text">Fig. 4: Undirected and directed exploration in a forward assembly process.</b></figcaption><div><div><a data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="https://www.nature.com/articles/s41586-023-06600-9/figures/4" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41586-023-06600-9/MediaObjects/41586_2023_6600_Fig4_HTML.png?as=webp"/><img aria-describedby="Fig4" src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41586-023-06600-9/MediaObjects/41586_2023_6600_Fig4_HTML.png" alt="figure 4" loading="lazy" width="685" height="767"/></picture></a></div><p><b>a</b>, The joint assembly space of polymeric chains (with their lengths indicated) after 30 steps created by combining randomly selected polymers from the assembly pool. The length of the realized polymers is shown in blue (observed nodes), whereas nodes shown in black represent polymers that have not been realized but are part of the joint assembly space of all realized objects (contingent nodes). For simplicity of representing the joint assembly space, the edge nodes (shown in red) represent the combined node along the directed graph. <b>b</b>, The comparison between undirected and directed exploration after 100 assembly steps using a graph with radial embedding (observed and contingent nodes shown in red and grey, respectively). <b>c</b>, The mean and standard deviation of the exploration ratio (defined by the ratio of the number of observed nodes and the number of total nodes, which includes observed and contingent nodes) and mean maximum assembly index. <i>n</i> is 25 runs all averaged up to 10<sup>4</sup> assembly steps.</p></div></figure></div><p>Comparison between the explored joint assembly space in undirected and directed exploration up to 100 steps is shown in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06600-9#Fig4">4b</a> (see Supplementary Information Section <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41586-023-06600-9#MOESM1">6</a> for details). To quantify the degree of exploration at a given assembly step, we calculated the exploration ratio, defined by the ratio of observed nodes to total number of nodes present in the joint assembly space. Figure <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06600-9#Fig4">4c</a> shows the exploration ratio and the mean maximum assembly index observed, approximated by <span>\({\log }_{2}\left(n\right)\)</span>,where <span>\(n\)</span> is the length of the polymer for the undirected and directed exploration processes (both upper and lower bounds scale as <span>\({\log }_{2}\left(n\right)\)</span> in leading order). Here, the mean maximum assembly index was estimated by calculating the assembly index of the mean value of the longest observed polymeric chains over 25 runs. Comparing the directed process to the undirected exploration illustrates a central principle: the signal of selection is simply a lower exploration ratio and higher complexity (as defined by the maximum assembly index). The observation of a lower exploration ratio in the directed process than in the undirected process is the evidence of the presence of selectivity in the combination process between the polymers existing in the assembly pool. The process representing sorting and selecting chains within the assembly pool represents an outcome of a physical process leading to selection (see Supplementary Information Section <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41586-023-06600-9#MOESM1">7</a> for an additional model).</p><p>We conjecture that, the ‘more assembled’ an ensemble of objects, the more selection is required for it to come into existence. The historical contingency in AT means that assembly dynamics explores higher-assembly objects before exhausting all lower-assembly objects, leading to a vast separation in scales separating the number of objects that could have been explored versus those that are actually constructed following a particular path. For example, proteins built both from <span>d</span> and <span>l</span> amino acids and their pathways are part of assembly possible, but, within an assembly contingent trajectory, only proteins constructed out of <span>l</span> amino acids might be present, because of early selection events. This early symmetry breaking along historically contingent paths is a fundamental property of all assembly processes. It introduces an ‘assembly time’ that ticks at each object being made: assembly physics includes an explicit arrow of time intrinsic to the structure of objects.</p></div></div></section><section data-title="Assembly unifies selection with physics"><div id="Sec6-section"><h2 id="Sec6">Assembly unifies selection with physics</h2><div id="Sec6-content"><p>In the real universe, objects can be built only from parts that already exist. The discovery of new objects is therefore historically contingent. The rate of discovery of new objects can be defined by the expansion rate (<span>\({k}_{{\rm{d}}}\)</span>) from equation <a data-track="click" data-track-label="link" data-track-action="equation anchor" href="https://www.nature.com/articles/s41586-023-06600-9#Equ2">(2)</a>, introducing a characteristic timescale <span>\({\tau }_{{\rm{d}}}\approx \frac{1}{{k}_{{\rm{d}}}}\)</span>, defined as the discovery time. In addition, once a pathway to build an object is discovered, the object can be reproduced if the mechanism in its environment is selected to build it again. Thus far, we have considered discovery dynamics within the assembly spaces and did not account for the abundance or copy number of the observed objects when discovered. To include copy number in the dynamics of AT, we must introduce a second timescale, the rate of production (<span>\({k}_{{\rm{p}}}\)</span>) of a specific object, with a characteristic production timescale <span>\({\tau }_{{\rm{p}}}\approx \frac{1}{{k}_{{\rm{p}}}}\)</span> (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06600-9#Fig5">5</a>). For simplicity, we assume that selectivity and interaction among emerging objects are similar across assembled objects. Defining these two distinct timescales for initial discovery of an object and making copies of existing objects allows us to determine the regimes in which selection is possible (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06600-9#Fig5">5</a>).</p><div data-test="figure" data-container-section="figure" id="figure-5" data-title="Selection and evolution in assembly space."><figure><figcaption><b id="Fig5" data-test="figure-caption-text">Fig. 5: Selection and evolution in assembly space.</b></figcaption><div><div><a data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="https://www.nature.com/articles/s41586-023-06600-9/figures/5" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41586-023-06600-9/MediaObjects/41586_2023_6600_Fig5_HTML.png?as=webp"/><img aria-describedby="Fig5" src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41586-023-06600-9/MediaObjects/41586_2023_6600_Fig5_HTML.png" alt="figure 5" loading="lazy" width="685" height="879"/></picture></a></div><p><b>a</b>, Assembly processes with and without selection. The selection process is defined by a transition from undirected to directed exploration. The parameter <span>\(\alpha \)</span> represents the selectivity of the assembly process (<span>\(\alpha =1\)</span>: undirected/random expansion, <span>\(\alpha &lt; 1\)</span>: directed expansion). Undirected exploration leads to the fast homogeneous expansion of discovered objects in the assembly space, whereas directed exploration leads to a process that is more like a depth-first search. Here, <span>\({\tau }_{{\rm{d}}}\)</span> is the characteristic timescale of discovery, determining the growth of the expansion front, and <span>\({\tau }_{{\rm{p}}}\)</span> is the characteristic timescale of production that determines the rate of formation of objects (increasing copy number). <b>b</b>, Rate of discovery of unique objects at assembly <span>\(a+1\)</span> versus number of objects at assembly <span>\(a\)</span>. The transition of <span>\(\alpha =1\)</span> to <span>\(\alpha &lt; 1\)</span> represents the emergence of selectivity limiting the discovery of new objects. <b>c</b>, Phase space defined by the production (<span>\({\tau }_{{\rm{p}}}\)</span>) and discovery (<span>\({\tau }_{{\rm{d}}}\)</span>) timescales. The figure shows three different regimes: (1) <span>\({\tau }_{{\rm{d}}}\ll {\tau }_{{\rm{p}}}\)</span>, (2) <span>\({\tau }_{{\rm{d}}}\gg {\tau }_{{\rm{p}}}\)</span>, and (3) <span>\({\tau }_{{\rm{d}}}\approx {\tau }_{{\rm{p}}}\)</span>. Selection is unlikely to emerge in regimes 1 and 2, and is possible in regime 3.</p></div></figure></div><p>For <span>\(\frac{{\tau }_{{\rm{p}}}}{{\tau }_{{\rm{d}}}}\gg 1\)</span>, whereby objects are discovered quickly but reproduced slowly, the expansion of assembly space is too fast under mass constraints to accumulate a high abundance of any distinguishable objects, leading to a combinatorial explosion of unique objects with low copy numbers. This is consistent with how some unconstrained prebiotic synthesis reactions, such as the formose reaction, end up producing tar, which is composed of a large number of molecules with too low a copy number to be individually identifiable<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 27" title="Kim, H.-J. et al. Synthesis of carbohydrates in mineral-guided prebiotic cycles. J. Am. Chem. Soc. 133, 9457–9468 (2011)." href="https://www.nature.com/articles/s41586-023-06600-9#ref-CR27" id="ref-link-section-d35886278e2671">27</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 28" title="Asche, S., Cooper, G. J. T., Mathis, C. &amp; Cronin, L. A robotic prebiotic chemist probes long term reactions of complexifying mixtures. Nat. Commun. 12, 3547 (2021)." href="https://www.nature.com/articles/s41586-023-06600-9#ref-CR28" id="ref-link-section-d35886278e2674">28</a></sup>. Selection and evolution cannot emerge if new objects are generated on timescales so fast that resources are not available for making more copies of those objects that already exist. For <span>\(\frac{{\tau }_{{\rm{p}}}}{{\tau }_{{\rm{d}}}}\ll 1\)</span>, objects are reproduced quickly but new ones are discovered slowly. Here resources are primarily consumed in producing additional copies of objects that already exist. Typically, new objects are discovered infrequently. This leads to a high abundance of objects produced by extreme constraints, which could limit the further growth of assembly space. This illustrates how exploration versus exploitation can play out in AT. Significant separation of the two timescales of discovery of new objects and (re)production of selected objects results in either a combinatorial explosion of objects with low copy numbers or, conversely, high copy numbers of low assembly objects. In both cases, we will not observe trajectories that grow more complex structures.</p><p>The emergence of selection and open-ended evolution in a physical system should occur in the transition regime where there is only a small separation in the timescales between discovering new objects and reproducing ones that are selected, for example the region located between <span>\({\tau }_{{\rm{d}}}\ll {\tau }_{{\rm{p}}}\)</span> and <span>\({\tau }_{{\rm{d}}}\gg {\tau }_{{\rm{p}}}\)</span> (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41586-023-06600-9#Fig5">5)</a>. To investigate discovery and production dynamics simultaneously, we introduce mass action kinetics in the framework of AT. Our aim is to demonstrate how the generation of novelty can be described alongside selection in a forward process (thus unifying key features of life with physics) and how measuring assembly identifies how much selection occurred. We do so by studying phenomenological models, with the understanding that we are putting selection in by hand in our examples to demonstrate foundational principles of how assembly quantifies selection. To explore this, we consider a forward assembly process whereby the copy numbers of emerging objects follow homogeneous kinetics, together with the discovery dynamics as given by equation (<a data-track="click" data-track-label="link" data-track-action="equation anchor" href="https://www.nature.com/articles/s41586-023-06600-9#Equ2">2</a>). With the discovery of new unique objects over time, symmetry breaking in the construction of contingent assembly paths will create a network of growing branches within the assembly possible. In principle, interactions among existing objects and external factors lead to discovery of new objects, expanding the space of possible future objects. Such events can drastically change the copy number distribution of objects at various assembly indices, depending on the emerging kinetics in the formation of new objects. By combining discovery and production kinetics in a simplified formulation, we estimate copy numbers of objects at different assembly indices and show assembly of the ensemble over time in the forward process at different degrees of selection (see Supplementary Information Section <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41586-023-06600-9#MOESM1">8</a> for an example).</p><p>The interplay between the two characteristic timescales describes how discovery dynamics (<span>\({{\tau }_{{\rm{d}}}\approx 1/k}_{{\rm{d}}}\)</span>) and forward kinetics (<span>\({\tau }_{{\rm{p}}}\approx 1/{k}_{{\rm{p}}}\)</span>), together with selection (characterized by the selection parameter <span>\(\alpha \)</span>), are essential for driving processes towards creating higher-assembly objects. This is characteristic of trajectories within assembly contingent. Assembly captures key features of how the open-ended growth of complexity can occur within a restricted space only by generating new objects with increasing assembly indices, while also producing them with a high copy number. Selectivity (<span>\(\alpha &lt; 1\)</span>) together with comparable production timescales (<span>\({\tau }_{{\rm{d}}}\approx {\tau }_{{\rm{p}}}\)</span>) is essential for the production of high assembly ensembles. This suggests that selectivity in an unknown physical process can be explained by experimentally detecting the number of objects, their assembly index and copy number as a function of time. Considering molecules as objects and assuming that molecules observed using analytical techniques such as mass spectrometry implies a high copy number, the discovery rate and the selection index (<span>\(\alpha \)</span>) can be computed from the temporal data of observed molecules at all assembly indices.</p></div></div></section><section data-title="Conclusions"><div id="Sec7-section"><h2 id="Sec7">Conclusions</h2><p>We have introduced the foundations of AT and how it can be implemented to quantify the degree of selection found in an ensemble of evolved objects, agnostic to the detailed formation mechanisms of the objects or knowing a priori which objects are products of units of selection. To do so, we introduced a quantity, assembly, built from two quantities: the number of copies of an object and its assembly index, where the assembly index is the minimal number of recursive steps necessary to build the object (its size). We demonstrated how AT allows a unified language for describing selection and the generation of novelty by showing how it quantifies the discovery and production of selected objects in a forward process described by mass action kinetics. AT provides a framework to unify descriptions of selection across physics and biology, with the potential to build a new physics that emerges in chemistry in which history and causal contingency through selection must start to play a prominent role in our descriptions of matter. For molecules, computing the assembly index is not explicitly necessary, because the assembly index can be probed directly experimentally with high accuracy with spectroscopy techniques including mass spectroscopy, infrared and nuclear magnetic resonance spectroscopy<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 29" title="Jirasek, M. et al. Multimodal techniques for detecting alien life using assembly theory and spectroscopy. Preprint at 
                https://doi.org/10.48550/ARXIV.2302.13753
                
               (2023)." href="https://www.nature.com/articles/s41586-023-06600-9#ref-CR29" id="ref-link-section-d35886278e3064">29</a></sup>.</p></div></section><section data-title="Methods"><div id="Sec8-section"><h2 id="Sec8">Methods</h2><div id="Sec8-content"><p>All the calculations were performed using Mathematica 13 (Wolfram Ltd). In addition, assembly index calculations on polymeric strings in the <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41586-023-06600-9#MOESM1">Supplementary Information</a> were performed using a string assembly calculator previously developed using Python and C++.</p><h3 id="Sec9">Reporting summary</h3><p>Further information on research design is available in the <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41586-023-06600-9#MOESM2">Nature Portfolio Reporting Summary</a> linked to this article.</p></div></div></section>
                </div><div>
            <section data-title="Data availability"><div id="data-availability-section"><h2 id="data-availability">Data availability</h2><div id="data-availability-content">
            
            <p>All Mathematica Notebooks used to perform the calculations are available at <a href="https://github.com/croningp/assemblyphysics">https://github.com/croningp/assemblyphysics</a>. The string assembly calculator and the dataset of assembly index calculations is available from the Zenodo repository <a href="https://doi.org/10.5281/zenodo.8017327">https://doi.org/10.5281/zenodo.8017327</a>.</p>
          </div></div></section><div id="MagazineFulltextArticleBodySuffix"><section aria-labelledby="Bib1" data-title="References"><div id="Bib1-section"><h2 id="Bib1">References</h2><div id="Bib1-content"><div data-container-section="references"><ol data-track-component="outbound reference"><li data-counter="1."><p id="ref-CR1">Kauffman, S. A. <i>The Origins of Order: Self-organization and Selection in Evolution</i> (Oxford Univ. Press, 1993).</p></li><li data-counter="2."><p id="ref-CR2">Gregory, T. R. Understanding natural selection: essential concepts and common misconceptions. <i>Evol. Educ. Outreach</i> <b>2</b>, 156–175 (2009).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1007/s12052-009-0128-1" data-track-action="article reference" href="https://doi.org/10.1007%2Fs12052-009-0128-1" aria-label="Article reference 2" data-doi="10.1007/s12052-009-0128-1">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 2" href="http://scholar.google.com/scholar_lookup?&amp;title=Understanding%20natural%20selection%3A%20essential%20concepts%20and%20common%20misconceptions&amp;journal=Evol.%20Educ.%20Outreach&amp;doi=10.1007%2Fs12052-009-0128-1&amp;volume=2&amp;pages=156-175&amp;publication_year=2009&amp;author=Gregory%2CTR">
                    Google Scholar</a> 
                </p></li><li data-counter="3."><p id="ref-CR3">Darwin, C. <i>On the Origin of Species by Means of Natural Selection, or, The Preservation of Favoured Races in the Struggle for Life</i> (Natural History Museum, 2019).</p></li><li data-counter="4."><p id="ref-CR4">Frank, S. A. &amp; Fox, G. A. in <i>The Theory of Evolution</i> (eds Scheiner, S. M. &amp; Mindell D. P.) 171–193 (Univ. of Chicago Press, 2020).</p></li><li data-counter="5."><p id="ref-CR5">Carroll, S. B. Chance and necessity: the evolution of morphological complexity and diversity. <i>Nature</i> <b>409</b>, 1102–1109 (2001).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/35059227" data-track-action="article reference" href="https://doi.org/10.1038%2F35059227" aria-label="Article reference 5" data-doi="10.1038/35059227">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2001Natur.409.1102C" aria-label="ADS reference 5">ADS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3MXhs1Gjur4%3D" aria-label="CAS reference 5">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=11234024" aria-label="PubMed reference 5">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 5" href="http://scholar.google.com/scholar_lookup?&amp;title=Chance%20and%20necessity%3A%20the%20evolution%20of%20morphological%20complexity%20and%20diversity&amp;journal=Nature&amp;doi=10.1038%2F35059227&amp;volume=409&amp;pages=1102-1109&amp;publication_year=2001&amp;author=Carroll%2CSB">
                    Google Scholar</a> 
                </p></li><li data-counter="6."><p id="ref-CR6">Chesson, P. Mechanisms of maintenance of species diversity. <i>Annu. Rev. Ecol. Syst.</i> <b>31</b>, 343–366 (2000).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1146/annurev.ecolsys.31.1.343" data-track-action="article reference" href="https://doi.org/10.1146%2Fannurev.ecolsys.31.1.343" aria-label="Article reference 6" data-doi="10.1146/annurev.ecolsys.31.1.343">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 6" href="http://scholar.google.com/scholar_lookup?&amp;title=Mechanisms%20of%20maintenance%20of%20species%20diversity&amp;journal=Annu.%20Rev.%20Ecol.%20Syst.&amp;doi=10.1146%2Fannurev.ecolsys.31.1.343&amp;volume=31&amp;pages=343-366&amp;publication_year=2000&amp;author=Chesson%2CP">
                    Google Scholar</a> 
                </p></li><li data-counter="7."><p id="ref-CR7">Newton, I. <i>Newton’s Principia. The Mathematical Principles of Natural Philosophy</i> (Daniel Adee, 1846).</p></li><li data-counter="8."><p id="ref-CR8">Cross, M. C. &amp; Hohenberg, P. C. Pattern formation outside of equilibrium. <i>Rev. Mod. Phys.</i> <b>65</b>, 851–1112 (1993).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1103/RevModPhys.65.851" data-track-action="article reference" href="https://doi.org/10.1103%2FRevModPhys.65.851" aria-label="Article reference 8" data-doi="10.1103/RevModPhys.65.851">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=1993RvMP...65..851C" aria-label="ADS reference 8">ADS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK2cXhtFShs7c%3D" aria-label="CAS reference 8">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="math reference" href="http://www.emis.de/MATH-item?1371.37001" aria-label="MATH reference 8">MATH</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 8" href="http://scholar.google.com/scholar_lookup?&amp;title=Pattern%20formation%20outside%20of%20equilibrium&amp;journal=Rev.%20Mod.%20Phys.&amp;doi=10.1103%2FRevModPhys.65.851&amp;volume=65&amp;pages=851-1112&amp;publication_year=1993&amp;author=Cross%2CMC&amp;author=Hohenberg%2CPC">
                    Google Scholar</a> 
                </p></li><li data-counter="9."><p id="ref-CR9">Tilman, D. <i>Resource Competition and Community Structure. (MPB-17)</i> Vol. 17 (Princeton Univ. Press, 2020).</p></li><li data-counter="10."><p id="ref-CR10">Elena, S. F., Cooper, V. S. &amp; Lenski, R. E. Punctuated evolution caused by selection of rare beneficial mutations. <i>Science</i> <b>272</b>, 1802–1804 (1996).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1126/science.272.5269.1802" data-track-action="article reference" href="https://doi.org/10.1126%2Fscience.272.5269.1802" aria-label="Article reference 10" data-doi="10.1126/science.272.5269.1802">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=1996Sci...272.1802E" aria-label="ADS reference 10">ADS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK28XjslWnu7s%3D" aria-label="CAS reference 10">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=8650581" aria-label="PubMed reference 10">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 10" href="http://scholar.google.com/scholar_lookup?&amp;title=Punctuated%20evolution%20caused%20by%20selection%20of%20rare%20beneficial%20mutations&amp;journal=Science&amp;doi=10.1126%2Fscience.272.5269.1802&amp;volume=272&amp;pages=1802-1804&amp;publication_year=1996&amp;author=Elena%2CSF&amp;author=Cooper%2CVS&amp;author=Lenski%2CRE">
                    Google Scholar</a> 
                </p></li><li data-counter="11."><p id="ref-CR11">Lutz, E. Power-law tail distributions and nonergodicity. <i>Phys. Rev. Lett.</i> <b>93</b>, 190602 (2004).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1103/PhysRevLett.93.190602" data-track-action="article reference" href="https://doi.org/10.1103%2FPhysRevLett.93.190602" aria-label="Article reference 11" data-doi="10.1103/PhysRevLett.93.190602">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2004PhRvL..93s0602L" aria-label="ADS reference 11">ADS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15600821" aria-label="PubMed reference 11">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 11" href="http://scholar.google.com/scholar_lookup?&amp;title=Power-law%20tail%20distributions%20and%20nonergodicity&amp;journal=Phys.%20Rev.%20Lett.&amp;doi=10.1103%2FPhysRevLett.93.190602&amp;volume=93&amp;publication_year=2004&amp;author=Lutz%2CE">
                    Google Scholar</a> 
                </p></li><li data-counter="12."><p id="ref-CR12">Cortês, M., Kauffman, S. A., Liddle, A. R. &amp; Smolin, L. The TAP equation: evaluating combinatorial innovation in biocosmology. Preprint at <a href="http://arxiv.org/abs/2204.14115" data-track="click" data-track-action="external reference" data-track-label="http://arxiv.org/abs/2204.14115">http://arxiv.org/abs/2204.14115</a> (2023).</p></li><li data-counter="13."><p id="ref-CR13">Fontana, W. &amp; Buss, L. W. in <i>Boundaries and Barriers</i> (eds Casti, J. &amp; Karlqvist, A.) 56–116 (Addison-Wesley, 1996).</p></li><li data-counter="14."><p id="ref-CR14">Marshall, S. M., Murray, A. R. G. &amp; Cronin, L. A probabilistic framework for identifying biosignatures using Pathway Complexity. <i>Philos. Trans. R. Soc. Math. Phys. Eng. Sci.</i> <b>375</b>, 20160342 (2017).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2017RSPTA.37560342M" aria-label="ADS reference 14">ADS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 14" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20probabilistic%20framework%20for%20identifying%20biosignatures%20using%20Pathway%20Complexity&amp;journal=Philos.%20Trans.%20R.%20Soc.%20Math.%20Phys.%20Eng.%20Sci.&amp;volume=375&amp;publication_year=2017&amp;author=Marshall%2CSM&amp;author=Murray%2CARG&amp;author=Cronin%2CL">
                    Google Scholar</a> 
                </p></li><li data-counter="15."><p id="ref-CR15">Marshall, S. M., Moore, D. G., Murray, A. R. G., Walker, S. I. &amp; Cronin, L. Formalising the pathways to life using assembly spaces. <i>Entropy</i> <b>24</b>, 884 (2022).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.3390/e24070884" data-track-action="article reference" href="https://doi.org/10.3390%2Fe24070884" aria-label="Article reference 15" data-doi="10.3390/e24070884">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2022Entrp..24..884M" aria-label="ADS reference 15">ADS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="mathscinet reference" href="http://www.ams.org/mathscinet-getitem?mr=4467768" aria-label="MathSciNet reference 15">MathSciNet</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=35885107" aria-label="PubMed reference 15">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9323097" aria-label="PubMed Central reference 15">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 15" href="http://scholar.google.com/scholar_lookup?&amp;title=Formalising%20the%20pathways%20to%20life%20using%20assembly%20spaces&amp;journal=Entropy&amp;doi=10.3390%2Fe24070884&amp;volume=24&amp;publication_year=2022&amp;author=Marshall%2CSM&amp;author=Moore%2CDG&amp;author=Murray%2CARG&amp;author=Walker%2CSI&amp;author=Cronin%2CL">
                    Google Scholar</a> 
                </p></li><li data-counter="16."><p id="ref-CR16">Liu, Y. et al. Exploring and mapping chemical space with molecular assembly trees. <i>Sci. Adv.</i> <b>7</b>, eabj2465 (2021).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1126/sciadv.abj2465" data-track-action="article reference" href="https://doi.org/10.1126%2Fsciadv.abj2465" aria-label="Article reference 16" data-doi="10.1126/sciadv.abj2465">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2021SciA....7.2465L" aria-label="ADS reference 16">ADS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXisFalsLfI" aria-label="CAS reference 16">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34559562" aria-label="PubMed reference 16">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8462901" aria-label="PubMed Central reference 16">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 16" href="http://scholar.google.com/scholar_lookup?&amp;title=Exploring%20and%20mapping%20chemical%20space%20with%20molecular%20assembly%20trees&amp;journal=Sci.%20Adv.&amp;doi=10.1126%2Fsciadv.abj2465&amp;volume=7&amp;publication_year=2021&amp;author=Liu%2CY">
                    Google Scholar</a> 
                </p></li><li data-counter="17."><p id="ref-CR17">Ellis, G. F. R. Top-down causation and emergence: some comments on mechanisms. <i>Interface Focus</i> <b>2</b>, 126–140 (2012).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1098/rsfs.2011.0062" data-track-action="article reference" href="https://doi.org/10.1098%2Frsfs.2011.0062" aria-label="Article reference 17" data-doi="10.1098/rsfs.2011.0062">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23386967" aria-label="PubMed reference 17">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 17" href="http://scholar.google.com/scholar_lookup?&amp;title=Top-down%20causation%20and%20emergence%3A%20some%20comments%20on%20mechanisms&amp;journal=Interface%20Focus&amp;doi=10.1098%2Frsfs.2011.0062&amp;volume=2&amp;pages=126-140&amp;publication_year=2012&amp;author=Ellis%2CGFR">
                    Google Scholar</a> 
                </p></li><li data-counter="18."><p id="ref-CR18">Koskinen, R. Multiple realizability as a design heuristic in biological engineering. <i>Eur. J. Philos. Sci.</i> <b>9</b>, 15 (2019).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1007/s13194-018-0243-3" data-track-action="article reference" href="https://doi.org/10.1007%2Fs13194-018-0243-3" aria-label="Article reference 18" data-doi="10.1007/s13194-018-0243-3">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 18" href="http://scholar.google.com/scholar_lookup?&amp;title=Multiple%20realizability%20as%20a%20design%20heuristic%20in%20biological%20engineering&amp;journal=Eur.%20J.%20Philos.%20Sci.&amp;doi=10.1007%2Fs13194-018-0243-3&amp;volume=9&amp;publication_year=2019&amp;author=Koskinen%2CR">
                    Google Scholar</a> 
                </p></li><li data-counter="19."><p id="ref-CR19">Marshall, S. M. et al. Identifying molecules as biosignatures with assembly theory and mass spectrometry. <i>Nat. Commun.</i> <b>12</b>, 3033 (2021).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/s41467-021-23258-x" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41467-021-23258-x" aria-label="Article reference 19" data-doi="10.1038/s41467-021-23258-x">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2021NatCo..12.3033M" aria-label="ADS reference 19">ADS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhtFyntbvL" aria-label="CAS reference 19">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34031398" aria-label="PubMed reference 19">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8144626" aria-label="PubMed Central reference 19">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 19" href="http://scholar.google.com/scholar_lookup?&amp;title=Identifying%20molecules%20as%20biosignatures%20with%20assembly%20theory%20and%20mass%20spectrometry&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-021-23258-x&amp;volume=12&amp;publication_year=2021&amp;author=Marshall%2CSM">
                    Google Scholar</a> 
                </p></li><li data-counter="20."><p id="ref-CR20">Arora, S. &amp; Barak, B. <i>Computational Complexity: A Modern Approach</i> (Cambridge Univ. Press, 2009).</p></li><li data-counter="21."><p id="ref-CR21">Wallace, C. S. Minimum message length and Kolmogorov complexity. <i>Comput. J.</i> <b>42</b>, 270–283 (1999).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1093/comjnl/42.4.270" data-track-action="article reference" href="https://doi.org/10.1093%2Fcomjnl%2F42.4.270" aria-label="Article reference 21" data-doi="10.1093/comjnl/42.4.270">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="math reference" href="http://www.emis.de/MATH-item?0946.68062" aria-label="MATH reference 21">MATH</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 21" href="http://scholar.google.com/scholar_lookup?&amp;title=Minimum%20message%20length%20and%20Kolmogorov%20complexity&amp;journal=Comput.%20J.&amp;doi=10.1093%2Fcomjnl%2F42.4.270&amp;volume=42&amp;pages=270-283&amp;publication_year=1999&amp;author=Wallace%2CCS">
                    Google Scholar</a> 
                </p></li><li data-counter="22."><p id="ref-CR22">Bennett, C. H. in <i>The Universal Turing Machine: A Half Century Survey</i> (ed. Herken, R.) 227–257 (Oxford Univ. Press, 1988).</p></li><li data-counter="23."><p id="ref-CR23">Deutsch, D. &amp; Marletto, C. Constructor theory of information. <i>Proc. R. Soc. Math. Phys. Eng. Sci.</i> <b>471</b>, 20140540 (2015).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2014RSPSA.47140540D" aria-label="ADS reference 23">ADS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="mathscinet reference" href="http://www.ams.org/mathscinet-getitem?mr=3303400" aria-label="MathSciNet reference 23">MathSciNet</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="math reference" href="http://www.emis.de/MATH-item?1371.81063" aria-label="MATH reference 23">MATH</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 23" href="http://scholar.google.com/scholar_lookup?&amp;title=Constructor%20theory%20of%20information&amp;journal=Proc.%20R.%20Soc.%20Math.%20Phys.%20Eng.%20Sci.&amp;volume=471&amp;publication_year=2015&amp;author=Deutsch%2CD&amp;author=Marletto%2CC">
                    Google Scholar</a> 
                </p></li><li data-counter="24."><p id="ref-CR24">Marletto, C. Constructor theory of life. <i>J. R. Soc. Interface</i> <b>12</b>, 20141226 (2015).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1098/rsif.2014.1226" data-track-action="article reference" href="https://doi.org/10.1098%2Frsif.2014.1226" aria-label="Article reference 24" data-doi="10.1098/rsif.2014.1226">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25589566" aria-label="PubMed reference 24">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4345487" aria-label="PubMed Central reference 24">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 24" href="http://scholar.google.com/scholar_lookup?&amp;title=Constructor%20theory%20of%20life&amp;journal=J.%20R.%20Soc.%20Interface&amp;doi=10.1098%2Frsif.2014.1226&amp;volume=12&amp;publication_year=2015&amp;author=Marletto%2CC">
                    Google Scholar</a> 
                </p></li><li data-counter="25."><p id="ref-CR25">Lewontin, R. C. The units of selection. <i>Annu. Rev. Ecol. Syst.</i> <b>1</b>, 1–18 (1970).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1146/annurev.es.01.110170.000245" data-track-action="article reference" href="https://doi.org/10.1146%2Fannurev.es.01.110170.000245" aria-label="Article reference 25" data-doi="10.1146/annurev.es.01.110170.000245">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 25" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20units%20of%20selection&amp;journal=Annu.%20Rev.%20Ecol.%20Syst.&amp;doi=10.1146%2Fannurev.es.01.110170.000245&amp;volume=1&amp;pages=1-18&amp;publication_year=1970&amp;author=Lewontin%2CRC">
                    Google Scholar</a> 
                </p></li><li data-counter="26."><p id="ref-CR26">Beasley, J. R. &amp; Hecht, M. H. Protein design: the choice of de novo sequences. <i>J. Biol. Chem.</i> <b>272</b>, 2031–2034 (1997).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1074/jbc.272.4.2031" data-track-action="article reference" href="https://doi.org/10.1074%2Fjbc.272.4.2031" aria-label="Article reference 26" data-doi="10.1074/jbc.272.4.2031">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK2sXotVOjsg%3D%3D" aria-label="CAS reference 26">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=9036150" aria-label="PubMed reference 26">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 26" href="http://scholar.google.com/scholar_lookup?&amp;title=Protein%20design%3A%20the%20choice%20of%20de%20novo%20sequences&amp;journal=J.%20Biol.%20Chem.&amp;doi=10.1074%2Fjbc.272.4.2031&amp;volume=272&amp;pages=2031-2034&amp;publication_year=1997&amp;author=Beasley%2CJR&amp;author=Hecht%2CMH">
                    Google Scholar</a> 
                </p></li><li data-counter="27."><p id="ref-CR27">Kim, H.-J. et al. Synthesis of carbohydrates in mineral-guided prebiotic cycles. <i>J. Am. Chem. Soc.</i> <b>133</b>, 9457–9468 (2011).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1021/ja201769f" data-track-action="article reference" href="https://doi.org/10.1021%2Fja201769f" aria-label="Article reference 27" data-doi="10.1021/ja201769f">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXmslGmt74%3D" aria-label="CAS reference 27">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21553892" aria-label="PubMed reference 27">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 27" href="http://scholar.google.com/scholar_lookup?&amp;title=Synthesis%20of%20carbohydrates%20in%20mineral-guided%20prebiotic%20cycles&amp;journal=J.%20Am.%20Chem.%20Soc.&amp;doi=10.1021%2Fja201769f&amp;volume=133&amp;pages=9457-9468&amp;publication_year=2011&amp;author=Kim%2CH-J">
                    Google Scholar</a> 
                </p></li><li data-counter="28."><p id="ref-CR28">Asche, S., Cooper, G. J. T., Mathis, C. &amp; Cronin, L. A robotic prebiotic chemist probes long term reactions of complexifying mixtures. <i>Nat. Commun.</i> <b>12</b>, 3547 (2021).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/s41467-021-23828-z" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41467-021-23828-z" aria-label="Article reference 28" data-doi="10.1038/s41467-021-23828-z">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2021NatCo..12.3547A" aria-label="ADS reference 28">ADS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhsVOqtL%2FN" aria-label="CAS reference 28">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34112788" aria-label="PubMed reference 28">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8192940" aria-label="PubMed Central reference 28">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 28" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20robotic%20prebiotic%20chemist%20probes%20long%20term%20reactions%20of%20complexifying%20mixtures&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-021-23828-z&amp;volume=12&amp;publication_year=2021&amp;author=Asche%2CS&amp;author=Cooper%2CGJT&amp;author=Mathis%2CC&amp;author=Cronin%2CL">
                    Google Scholar</a> 
                </p></li><li data-counter="29."><p id="ref-CR29">Jirasek, M. et al. Multimodal techniques for detecting alien life using assembly theory and spectroscopy. Preprint at <a href="https://doi.org/10.48550/ARXIV.2302.13753" data-track="click" data-track-action="external reference" data-track-label="10.48550/ARXIV.2302.13753">https://doi.org/10.48550/ARXIV.2302.13753</a> (2023).</p></li></ol><p><a data-track="click" data-track-action="download citation references" data-track-label="link" rel="nofollow" href="https://citation-needed.springer.com/v2/references/10.1038/s41586-023-06600-9?format=refman&amp;flavour=references">Download references<svg width="16" height="16" focusable="false" role="img" aria-hidden="true"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-download"></use></svg></a></p></div></div></div></section></div><section data-title="Acknowledgements"><div id="Ack1-section"><h2 id="Ack1">Acknowledgements</h2><p>L.C., S.I.W., D.C. and A.S. would like to acknowledge our teams and colleagues at the University of Glasgow and Arizona State University for discussions, including C. Mathis, D. Moore, S. Marshall and P. Davies. We acknowledge financial support from the John Templeton Foundation (grant nos. 61184 and 62231), the Engineering and Physical Sciences Research Council (EPSRC) (grant nos. EP/L023652/1, EP/R01308X/1, EP/S019472/1 and EP/P00153X/1), the Breakthrough Prize Foundation and NASA (Agnostic Biosignatures award no. 80NSSC18K1140), MINECO (project CTQ2017-87392-P) and the European Research Council (ERC) (project 670467 SMART-POM).</p></div></section><section aria-labelledby="author-information" data-title="Author information"><div id="author-information-section"><h2 id="author-information">Author information</h2><div id="author-information-content"><p><span id="author-notes">Author notes</span></p><ol><li id="na1"><p>These authors contributed equally: Abhishek Sharma, Dániel Czégel</p></li></ol><h3 id="affiliations">Authors and Affiliations</h3><ol><li id="Aff1"><p>School of Chemistry, University of Glasgow, Glasgow, UK</p><p>Abhishek Sharma &amp; Leroy Cronin</p></li><li id="Aff2"><p>BEYOND Center for Fundamental Concepts in Science, Arizona State University, Tempe, AZ, USA</p><p>Dániel Czégel &amp; Sara I. Walker</p></li><li id="Aff3"><p>Institute of Evolution, Centre for Ecological Research, Budapest, Hungary</p><p>Dániel Czégel</p></li><li id="Aff4"><p>The Santa Fe Institute, Santa Fe, NM, USA</p><p>Michael Lachmann &amp; Christopher P. Kempes</p></li><li id="Aff5"><p>School of Earth and Space Exploration, Arizona State University, Tempe, AZ, USA</p><p>Sara I. Walker</p></li></ol><h3 id="contributions">Contributions</h3><p>L.C. and S.I.W. conceived the theoretical framework building on the concept of the theory. A.S. and D.C. developed the mathematical basis for the framework and explored the assembly equation, and A.S. did the simulations with input from D.C. M.L. and C.P.K. helped with the development of the fundamentals of assembly theory. L.C. and S.I.W. wrote the manuscript with input from all the authors.</p><h3 id="corresponding-author">Corresponding authors</h3><p id="corresponding-author-list">Correspondence to
                <a id="corresp-c1" href="mailto:sara.i.walker@asu.edu">Sara I. Walker</a> or <a id="corresp-c2" href="mailto:Lee.Cronin@glasgow.ac.uk">Leroy Cronin</a>.</p></div></div></section><section data-title="Ethics declarations"><div id="ethics-section"><h2 id="ethics">Ethics declarations</h2><div id="ethics-content">
            
              <h3 id="FPar2">Competing interests</h3>
              <p>The authors declare no competing interests.</p>
            
          </div></div></section><section data-title="Peer review"><div id="peer-review-section"><h2 id="peer-review">Peer review</h2><div id="peer-review-content">
            
            
              <h3 id="FPar1">Peer review information</h3>
              <p><i>Nature</i> thanks Pierrick Bourrat, George Ellis and the other, anonymous, reviewer(s) for their contribution to the peer review of this work.</p>
            
          </div></div></section><section data-title="Additional information"><div id="additional-information-section"><h2 id="additional-information">Additional information</h2><p><b>Publisher’s note</b> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div></section><section data-title="Supplementary information"><div id="Sec11-section"><h2 id="Sec11">Supplementary information</h2><div id="Sec11-content"><div data-test="supplementary-info"><div data-test="supp-item" id="MOESM1"><h3><a data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="supplementary information" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41586-023-06600-9/MediaObjects/41586_2023_6600_MOESM1_ESM.pdf" data-supp-info-image="">Supplementary Information</a></h3><p>Details of the mathematical models, simulations and examples used in the manuscript, Supplementary Sections 1–10, Figs. 1–18 and one additional reference.</p></div></div></div></div></section><section data-title="Rights and permissions"><div id="rightslink-section"><h2 id="rightslink">Rights and permissions</h2><div id="rightslink-content">
              <p><b>Open Access</b>  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>.</p>
            <p><a data-track="click" data-track-action="view rights and permissions" data-track-label="link" href="https://s100.copyright.com/AppDispatchServlet?title=Assembly%20theory%20explains%20and%20quantifies%20selection%20and%20evolution&amp;author=Abhishek%20Sharma%20et%20al&amp;contentID=10.1038%2Fs41586-023-06600-9&amp;copyright=The%20Author%28s%29&amp;publication=0028-0836&amp;publicationDate=2023-10-04&amp;publisherName=SpringerNature&amp;orderBeanReset=true&amp;oa=CC%20BY">Reprints and Permissions</a></p></div></div></section><section aria-labelledby="article-info" data-title="About this article"><div id="article-info-section"><h2 id="article-info">About this article</h2><div id="article-info-content"><div><p><a data-crossmark="10.1038/s41586-023-06600-9" target="_blank" rel="noopener" href="https://crossmark.crossref.org/dialog/?doi=10.1038/s41586-023-06600-9" data-track="click" data-track-action="Click Crossmark" data-track-label="link" data-test="crossmark"><img width="57" height="81" alt="Check for updates. Verify currency and authenticity via CrossMark" src="data:image/svg+xml;base64,<svg height="81" width="57" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="m17.35 35.45 21.3-14.2v-17.03h-21.3" fill="#989898"/><path d="m38.65 35.45-21.3-14.2v-17.03h21.3" fill="#747474"/><path d="m28 .5c-12.98 0-23.5 10.52-23.5 23.5s10.52 23.5 23.5 23.5 23.5-10.52 23.5-23.5c0-6.23-2.48-12.21-6.88-16.62-4.41-4.4-10.39-6.88-16.62-6.88zm0 41.25c-9.8 0-17.75-7.95-17.75-17.75s7.95-17.75 17.75-17.75 17.75 7.95 17.75 17.75c0 4.71-1.87 9.22-5.2 12.55s-7.84 5.2-12.55 5.2z" fill="#535353"/><path d="m41 36c-5.81 6.23-15.23 7.45-22.43 2.9-7.21-4.55-10.16-13.57-7.03-21.5l-4.92-3.11c-4.95 10.7-1.19 23.42 8.78 29.71 9.97 6.3 23.07 4.22 30.6-4.86z" fill="#9c9c9c"/><path d="m.2 58.45c0-.75.11-1.42.33-2.01s.52-1.09.91-1.5c.38-.41.83-.73 1.34-.94.51-.22 1.06-.32 1.65-.32.56 0 1.06.11 1.51.35.44.23.81.5 1.1.81l-.91 1.01c-.24-.24-.49-.42-.75-.56-.27-.13-.58-.2-.93-.2-.39 0-.73.08-1.05.23-.31.16-.58.37-.81.66-.23.28-.41.63-.53 1.04-.13.41-.19.88-.19 1.39 0 1.04.23 1.86.68 2.46.45.59 1.06.88 1.84.88.41 0 .77-.07 1.07-.23s.59-.39.85-.68l.91 1c-.38.43-.8.76-1.28.99-.47.22-1 .34-1.58.34-.59 0-1.13-.1-1.64-.31-.5-.2-.94-.51-1.31-.91-.38-.4-.67-.9-.88-1.48-.22-.59-.33-1.26-.33-2.02zm8.4-5.33h1.61v2.54l-.05 1.33c.29-.27.61-.51.96-.72s.76-.31 1.24-.31c.73 0 1.27.23 1.61.71.33.47.5 1.14.5 2.02v4.31h-1.61v-4.1c0-.57-.08-.97-.25-1.21-.17-.23-.45-.35-.83-.35-.3 0-.56.08-.79.22-.23.15-.49.36-.78.64v4.8h-1.61zm7.37 6.45c0-.56.09-1.06.26-1.51.18-.45.42-.83.71-1.14.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.36c.07.62.29 1.1.65 1.44.36.33.82.5 1.38.5.29 0 .57-.04.83-.13s.51-.21.76-.37l.55 1.01c-.33.21-.69.39-1.09.53-.41.14-.83.21-1.26.21-.48 0-.92-.08-1.34-.25-.41-.16-.76-.4-1.07-.7-.31-.31-.55-.69-.72-1.13-.18-.44-.26-.95-.26-1.52zm4.6-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.07.45-.31.29-.5.73-.58 1.3zm2.5.62c0-.57.09-1.08.28-1.53.18-.44.43-.82.75-1.13s.69-.54 1.1-.71c.42-.16.85-.24 1.31-.24.45 0 .84.08 1.17.23s.61.34.85.57l-.77 1.02c-.19-.16-.38-.28-.56-.37-.19-.09-.39-.14-.61-.14-.56 0-1.01.21-1.35.63-.35.41-.52.97-.52 1.67 0 .69.17 1.24.51 1.66.34.41.78.62 1.32.62.28 0 .54-.06.78-.17.24-.12.45-.26.64-.42l.67 1.03c-.33.29-.69.51-1.08.65-.39.15-.78.23-1.18.23-.46 0-.9-.08-1.31-.24-.4-.16-.75-.39-1.05-.7s-.53-.69-.7-1.13c-.17-.45-.25-.96-.25-1.53zm6.91-6.45h1.58v6.17h.05l2.54-3.16h1.77l-2.35 2.8 2.59 4.07h-1.75l-1.77-2.98-1.08 1.23v1.75h-1.58zm13.69 1.27c-.25-.11-.5-.17-.75-.17-.58 0-.87.39-.87 1.16v.75h1.34v1.27h-1.34v5.6h-1.61v-5.6h-.92v-1.2l.92-.07v-.72c0-.35.04-.68.13-.98.08-.31.21-.57.4-.79s.42-.39.71-.51c.28-.12.63-.18 1.04-.18.24 0 .48.02.69.07.22.05.41.1.57.17zm.48 5.18c0-.57.09-1.08.27-1.53.17-.44.41-.82.72-1.13.3-.31.65-.54 1.04-.71.39-.16.8-.24 1.23-.24s.84.08 1.24.24c.4.17.74.4 1.04.71s.54.69.72 1.13c.19.45.28.96.28 1.53s-.09 1.08-.28 1.53c-.18.44-.42.82-.72 1.13s-.64.54-1.04.7-.81.24-1.24.24-.84-.08-1.23-.24-.74-.39-1.04-.7c-.31-.31-.55-.69-.72-1.13-.18-.45-.27-.96-.27-1.53zm1.65 0c0 .69.14 1.24.43 1.66.28.41.68.62 1.18.62.51 0 .9-.21 1.19-.62.29-.42.44-.97.44-1.66 0-.7-.15-1.26-.44-1.67-.29-.42-.68-.63-1.19-.63-.5 0-.9.21-1.18.63-.29.41-.43.97-.43 1.67zm6.48-3.44h1.33l.12 1.21h.05c.24-.44.54-.79.88-1.02.35-.24.7-.36 1.07-.36.32 0 .59.05.78.14l-.28 1.4-.33-.09c-.11-.01-.23-.02-.38-.02-.27 0-.56.1-.86.31s-.55.58-.77 1.1v4.2h-1.61zm-47.87 15h1.61v4.1c0 .57.08.97.25 1.2.17.24.44.35.81.35.3 0 .57-.07.8-.22.22-.15.47-.39.73-.73v-4.7h1.61v6.87h-1.32l-.12-1.01h-.04c-.3.36-.63.64-.98.86-.35.21-.76.32-1.24.32-.73 0-1.27-.24-1.61-.71-.33-.47-.5-1.14-.5-2.02zm9.46 7.43v2.16h-1.61v-9.59h1.33l.12.72h.05c.29-.24.61-.45.97-.63.35-.17.72-.26 1.1-.26.43 0 .81.08 1.15.24.33.17.61.4.84.71.24.31.41.68.53 1.11.13.42.19.91.19 1.44 0 .59-.09 1.11-.25 1.57-.16.47-.38.85-.65 1.16-.27.32-.58.56-.94.73-.35.16-.72.25-1.1.25-.3 0-.6-.07-.9-.2s-.59-.31-.87-.56zm0-2.3c.26.22.5.37.73.45.24.09.46.13.66.13.46 0 .84-.2 1.15-.6.31-.39.46-.98.46-1.77 0-.69-.12-1.22-.35-1.61-.23-.38-.61-.57-1.13-.57-.49 0-.99.26-1.52.77zm5.87-1.69c0-.56.08-1.06.25-1.51.16-.45.37-.83.65-1.14.27-.3.58-.54.93-.71s.71-.25 1.08-.25c.39 0 .73.07 1 .2.27.14.54.32.81.55l-.06-1.1v-2.49h1.61v9.88h-1.33l-.11-.74h-.06c-.25.25-.54.46-.88.64-.33.18-.69.27-1.06.27-.87 0-1.56-.32-2.07-.95s-.76-1.51-.76-2.65zm1.67-.01c0 .74.13 1.31.4 1.7.26.38.65.58 1.15.58.51 0 .99-.26 1.44-.77v-3.21c-.24-.21-.48-.36-.7-.45-.23-.08-.46-.12-.7-.12-.45 0-.82.19-1.13.59-.31.39-.46.95-.46 1.68zm6.35 1.59c0-.73.32-1.3.97-1.71.64-.4 1.67-.68 3.08-.84 0-.17-.02-.34-.07-.51-.05-.16-.12-.3-.22-.43s-.22-.22-.38-.3c-.15-.06-.34-.1-.58-.1-.34 0-.68.07-1 .2s-.63.29-.93.47l-.59-1.08c.39-.24.81-.45 1.28-.63.47-.17.99-.26 1.54-.26.86 0 1.51.25 1.93.76s.63 1.25.63 2.21v4.07h-1.32l-.12-.76h-.05c-.3.27-.63.48-.98.66s-.73.27-1.14.27c-.61 0-1.1-.19-1.48-.56-.38-.36-.57-.85-.57-1.46zm1.57-.12c0 .3.09.53.27.67.19.14.42.21.71.21.28 0 .54-.07.77-.2s.48-.31.73-.56v-1.54c-.47.06-.86.13-1.18.23-.31.09-.57.19-.76.31s-.33.25-.41.4c-.09.15-.13.31-.13.48zm6.29-3.63h-.98v-1.2l1.06-.07.2-1.88h1.34v1.88h1.75v1.27h-1.75v3.28c0 .8.32 1.2.97 1.2.12 0 .24-.01.37-.04.12-.03.24-.07.34-.11l.28 1.19c-.19.06-.4.12-.64.17-.23.05-.49.08-.76.08-.4 0-.74-.06-1.02-.18-.27-.13-.49-.3-.67-.52-.17-.21-.3-.48-.37-.78-.08-.3-.12-.64-.12-1.01zm4.36 2.17c0-.56.09-1.06.27-1.51s.41-.83.71-1.14c.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.37c.08.62.29 1.1.65 1.44.36.33.82.5 1.38.5.3 0 .58-.04.84-.13.25-.09.51-.21.76-.37l.54 1.01c-.32.21-.69.39-1.09.53s-.82.21-1.26.21c-.47 0-.92-.08-1.33-.25-.41-.16-.77-.4-1.08-.7-.3-.31-.54-.69-.72-1.13-.17-.44-.26-.95-.26-1.52zm4.61-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.08.45-.31.29-.5.73-.57 1.3zm3.01 2.23c.31.24.61.43.92.57.3.13.63.2.98.2.38 0 .65-.08.83-.23s.27-.35.27-.6c0-.14-.05-.26-.13-.37-.08-.1-.2-.2-.34-.28-.14-.09-.29-.16-.47-.23l-.53-.22c-.23-.09-.46-.18-.69-.3-.23-.11-.44-.24-.62-.4s-.33-.35-.45-.55c-.12-.21-.18-.46-.18-.75 0-.61.23-1.1.68-1.49.44-.38 1.06-.57 1.83-.57.48 0 .91.08 1.29.25s.71.36.99.57l-.74.98c-.24-.17-.49-.32-.73-.42-.25-.11-.51-.16-.78-.16-.35 0-.6.07-.76.21-.17.15-.25.33-.25.54 0 .14.04.26.12.36s.18.18.31.26c.14.07.29.14.46.21l.54.19c.23.09.47.18.7.29s.44.24.64.4c.19.16.34.35.46.58.11.23.17.5.17.82 0 .3-.06.58-.17.83-.12.26-.29.48-.51.68-.23.19-.51.34-.84.45-.34.11-.72.17-1.15.17-.48 0-.95-.09-1.41-.27-.46-.19-.86-.41-1.2-.68z" fill="#535353"/></g></svg>"/></a></p><div><h3 id="citeas">Cite this article</h3><p>Sharma, A., Czégel, D., Lachmann, M. <i>et al.</i> Assembly theory explains and quantifies selection and evolution.
                    <i>Nature</i>  (2023). https://doi.org/10.1038/s41586-023-06600-9</p><p><a data-test="citation-link" data-track="click" data-track-action="download article citation" data-track-label="link" data-track-external="" rel="nofollow" href="https://citation-needed.springer.com/v2/references/10.1038/s41586-023-06600-9?format=refman&amp;flavour=citation">Download citation<svg width="16" height="16" focusable="false" role="img" aria-hidden="true"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-download"></use></svg></a></p><ul data-test="publication-history"><li><p>Received<span>: </span><span><time datetime="2023-04-01">01 April 2023</time></span></p></li><li><p>Accepted<span>: </span><span><time datetime="2023-08-31">31 August 2023</time></span></p></li><li><p>Published<span>: </span><span><time datetime="2023-10-04">04 October 2023</time></span></p></li><li><p><abbr title="Digital Object Identifier">DOI</abbr><span>: </span><span>https://doi.org/10.1038/s41586-023-06600-9</span></p></li></ul></div></div></div></div></section>
            </div></div>
  </body>
</html>
