<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <title>James Routley | Feed</title>
    <link
      rel="stylesheet"
      type="text/css"
      href="../styles.css"
      media="screen"
    />
  </head>
  <body>
    <a href="/index.html">Back</a>
    <a href="https://news.cornell.edu/stories/2022/10/butterfly-wing-patterns-emerge-ancient-junk-dna">Original</a>
    <h1>Butterfly wing patterns emerge from ancient ‘junk’ DNA</h1>
    
    <div id="readability-page-1" class="page"><div>
      <div><p><span><span><span><span><span><span><span><span><span>Butterfly wing patterns have a basic plan to them, which is manipulated by non-coding regulatory DNA to create the diversity of wings seen in different species, according to new research. </span></span></span></span></span></span></span></span></span></p>

<p><span><span><span><span><span><span><span><span><span>The study, “<a href="https://www.science.org/doi/10.1126/science.abi9407">Deep cis-regulatory homology of the butterfly wing pattern ground plan</a>,” published as the cover story in the Oct. 21 issue of Science, explains how DNA that sits between genes – called ‘junk’ DNA or non-coding regulatory DNA – accommodates a basic plan conserved over tens to hundreds of millions of years while at the same time allowing wing patterns to evolve extremely quickly.</span></span></span></span></span></span></span></span></span></p>

<div data-embed-button="image_embed" data-entity-embed-display="view_mode:media_embed.image_caption" data-entity-type="media_embed" data-entity-uuid="b9289554-0f28-4870-b603-d958be8eee18" data-langcode="en"><figure>


            <figcaption><p>Gulf fritillary butterfly – Agraulis vanillae.</p>
</figcaption></figure></div>
<p><span><span><span><span><span><span><span><span><span>The research supports the idea that an ancient color pattern ground plan</span><span><span><span> is already encoded in the genome and that non-coding regulatory DNA works like switches to turn up some patterns and turn down others. </span></span></span></span></span></span></span></span></span></span></span></p>

<p><span><span><span><span><span><span><span><span><span><span><span>“We are interested to know how the same gene can build these very different looking butterflies,” said Anyi Mazo-Vargas, Ph.D. ’20, the study’s first author and a former graduate student in the lab of senior author, Robert Reed, professor of ecology and evolutionary biology in the College of Agriculture and Life Sciences. Mazo-Vargas is currently a postdoctoral researcher at George Washington University. </span></span></span></span></span></span></span></span></span></span></span></p>

<p><span><span><span><span><span><span><span><span><span><span><span>“We see that there’s a very conserved group of switches [non-coding DNA] that are working in different positions and are activated and driving the gene,” Mazo-Vargas said.</span></span></span></span></span></span></span></span></span></span></span></p>

<p><span><span><span><span><span><span><span><span><span><a href="https://news.cornell.edu/stories/2017/09/black-and-white-proof-single-genes-control-color-stripes-butterfly-wings"><span>Previous work</span></a><span><span> in Reed’s lab has uncovered key color pattern genes: one (WntA) that controls stripes and another (Optix) that controls color and iridescence in butterfly wings. When the researchers disabled the Optix gene, the wings appeared black, and when the WntA gene was deleted, stripe patterns disappeared.</span></span></span></span></span></span></span></span></span></span></span></p>

<p><span><span><span><span><span><span><span><span><span><span><span>This study focused on the effect of non-coding DNA on the WntA gene. Specifically, the researchers ran experiments on 46 of these non-coding elements in five species of nymphalid butterflies, which is the largest family of butterflies. </span></span></span></span></span></span></span></span></span></span></span></p>

<p><span><span><span><span><span><span><span><span><span><span><span>In order for these non-coding regulatory elements to control genes, tightly wound coils of DNA become unspooled, a sign that a regulatory element is interacting with a gene to activate it, or in some cases, turn it off. </span></span></span></span></span></span></span></span></span></span></span></p>

<p><span><span><span><span><span><span><span><span><span><span><span>In the study, the researchers used a technology called ATAC-seq to identify regions in the genome where this unraveling is occurring. Mazo-Vargas compared ATAC-seq profiles from the wings of five butterfly species, in order to identify genetic regions involved in wing pattern development. They were surprised to find that a large number of regulatory regions were shared across very different butterfly species.</span></span></span></span></span></span></span></span></span></span></span></p>

<p><span><span><span><span><span><span><span><span><span><span><span>Mazo-Vargas and colleagues then employed CRISPR-Cas gene editing technology to disable 46 regulatory elements one at a time, in order to see the effects on wing patterns when each of these non-coding DNA sequences were broken. When deleted, each non-coding element changed an aspect of the wing patterns of the butterflies.</span></span></span></span></span></span></span></span></span></span></span></p>

<p><span><span><span><span><span><span><span><span><span><span><span>The researchers found that across four of the </span></span></span><span><span><span>species – Junonia coenia (buckeye), Vanessa cardui (painted lady), Heliconius himera and Agraulis vanillae (gulf fritillary)</span></span></span><span><span><span> – each of these non-coding elements had similar functions with respect to the WntA gene, proving they were ancient and conserved, likely originating in a distant common ancestor.</span></span></span></span></span></span></span></span></span></span></span></p>

<p><span><span><span><span><span><span><span><span><span><span><span>They also found that D. plexippus (monarch) used different regulatory elements from the other four species to control its WntA gene, perhaps because it lost some of its genetic information over its history and had to reinvent its own regulatory system to develop its unique color patterns.</span></span></span></span></span></span></span></span></span></span></span></p>

<p><span><span><span><span><span><span><span><span><span><span><span>“We have progressively come to understand that most evolution occurs because of mutations in these non-coding regions,” Reed said. “What I hope is that this paper will be a case study that shows how people can use this combination of ATAC-seq and CRISPR to begin to interrogate these interesting regions in their own study systems, whether they work on birds or flies or worms.”</span></span></span></span></span></span></span></span></span></span></span></p>

<p><span><span><span><span><span><span><span><span><span><span><span>The study was funded by the National Science Foundation (NSF). </span></span></span></span></span></span></span></span></span></span></span></p>

<p><span><span><span><span><span><span><span><span><span><span>“This research is a breakthrough for our understanding of the genetic control of complex traits, and not only in butterflies,” said Theodore Morgan, a program director at the NSF. “Not only did the study show how the instructions for butterfly color patterns are deeply conserved across evolutionary history, but it also revealed new evidence for how regulatory DNA segments positively and negatively influence traits such as color and shape.”</span></span></span></span></span></span></span></span></span></span></p></div>
</div></div>
  </body>
</html>
