<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <title>James Routley | Feed</title>
    <link
      rel="stylesheet"
      type="text/css"
      href="../styles.css"
      media="screen"
    />
  </head>
  <body>
    <a href="/index.html">Back</a>
    <a href="https://www.nature.com/articles/d41586-023-00347-z">Original</a>
    <h1>Covid drug drives viral mutations – and now some want to halt its use</h1>
    
    <div id="readability-page-1" class="page"><div>
                    <figure>
 <picture>
  <source type="image/webp" srcset="//media.nature.com/lw767/magazine-assets/d41586-023-00347-z/d41586-023-00347-z_23987520.jpg?as=webp 767w, //media.nature.com/lw319/magazine-assets/d41586-023-00347-z/d41586-023-00347-z_23987520.jpg?as=webp 319w" sizes="(max-width: 319px) 319px, (min-width: 1023px) 100vw,  767px"/>
  <img alt="Close-up view of hands and boxes of molnupiravir." loading="lazy" src="https://media.nature.com/lw767/magazine-assets/d41586-023-00347-z/d41586-023-00347-z_23987520.jpg"/>
  <figcaption>
   <p><span>Genomic analysis has linked the antiviral drug molnupiravir to highly mutated lineages of SARS-CoV-2.</span><span>Credit: Ziad Ahmed/NurPhoto/Getty</span></p>
  </figcaption>
 </picture>
</figure><p>A drug widely used to treat COVID-19 might be spurring the evolution of new SARS-CoV-2 variants.</p><p><a href="https://www.nature.com/articles/d41586-021-03667-0" data-track="click" data-label="https://www.nature.com/articles/d41586-021-03667-0" data-track-category="body text link">The drug, molnupiravir</a>, works by introducing a flurry of mutations to the viral genome; this helps to clear infections. But a study<sup><a href="#ref-CR1" data-track="click" data-action="anchor-link" data-track-label="go to reference" data-track-category="references">1</a></sup> of more than 13 million SARS-CoV-2 sequences has uncovered sequences that bear molnupiravir’s fingerprints. The study’s authors say the results suggest that molnupiravir treatment has sparked the evolution of viral lineages carrying numerous mutations that, in at least some cases, have the capacity to spread to other individuals.</p><p>The study was posted on the medRxiv preprint server in January. It has not yet been peer reviewed.</p><p>“Whether this should be of concern — that is an open question,” says Jesse Bloom, an evolutionary virologist at the Fred Hutchinson Cancer Research Center in Seattle, Washington. <a href="https://www.nature.com/articles/d41586-022-00510-y" data-track="click" data-label="https://www.nature.com/articles/d41586-022-00510-y" data-track-category="body text link">Mutations to the SARS-CoV-2 genome can help the virus to evade immunity and become more transmissible</a>, but most mutations are likely to harm it.</p><h2>Molecular mimic</h2><p>Molnupiravir was developed by the pharmaceutical giant Merck, based in Rahway, New Jersey, and was authorized by regulators in the United States and United Kingdom in late 2021, and in Australia in early 2022. A company-sponsored clinical trial found that the drug — a pill taken for five days — reduced hospitalizations and deaths in people at risk of severe COVID-19.</p><p>Molnupiravir mimics some of the building blocks of RNA, the genetic material in SARS-CoV-2. The drug peppers the viral genome with mutations, which add up to make SARS-CoV-2 worse at replicating. This results in a marked reduction in SARS-CoV-2 levels in infected human cells and hamsters<sup><a href="#ref-CR2" data-track="click" data-action="anchor-link" data-track-label="go to reference" data-track-category="references">2</a></sup>.</p><p>But scientists have raised the possibility that, in rare cases, molnupiravir treatment might not entirely eliminate SARS-CoV-2, allowing some individuals who have taken the drug to continue to transmit the virus.</p><article data-label="Related">
  <a href="https://www.nature.com/articles/d41586-023-00086-1" data-track="click" data-track-label="recommended article"><img alt="" src="https://media.nature.com/w400/magazine-assets/d41586-023-00347-z/d41586-023-00347-z_23944226.jpg"/><p>How your first brush with COVID warps your immunity</p></a>
 </article><p>Those concerns were largely theoretical until Ryan Hisner, a US schoolteacher who is part of a loose-knit community of SARS-CoV-2 variant spotters, started noticing peculiar patterns in sequencing data from around the world: a number of lineages carried many more mutations than their nearest relatives. Most of the genetic changes were of the type induced by molnupiravir.</p><p>The researchers’ analysis of global sequences showed that the prevalence of the suspect lineages rose substantially in 2022, the first year molnupiravir was widely used. The lineages were also much more likely to show up in sequencing data from countries where the drug was used — the United States, the United Kingdom and Australia — than in data from countries that hadn’t approved it, such as France and Canada.</p><p>Where SARS-CoV-2 sequence data include the age of the individual sampled, the telltale lineages tended to have come from older people, who are more likely to take molnupiravir. And in Australia, where molnupiravir was pre-placed in care homes, the researchers identified a viral lineage carrying 25 mutations that had infected at least 20 people, mostly in their eighties and nineties.</p><p>Among the lineages that showed molnupiravir’s influence, a few — including the one from Australia — were represented by multiple sequences, indicating that they were capable of spreading. “I would say that our work closes down the possibility that these viruses can never be transmitted,” says team member Theo Sanderson, a computational biologist at the Francis Crick Institute in London.</p><h2>Evolutionary dead ends?</h2><p>Evidence linking molnupiravir to the mutation-heavy sequences is “circumstantial”, Merck said in a statement. “The authors assume these mutations were associated with molnupiravir treatment without evidence the viral sequences were isolated from treated patients.”</p><p>Bloom thinks the researchers make a good case that molnupiravir treatment is yielding some highly mutated viruses with the capacity to spread. But it’s not clear whether this could contribute to new coronavirus variants, or whether it is simply creating weakling viruses that are unlikely to spread very far. “SARS-CoV-2 is already generating plenty of mutations even in the absence of drug treatment,” he adds.</p><p>Rustem Ismagilov, a quantitative bioscientist at the California Institute of Technology in Pasadena, says the study underscores the need to quickly measure any risk that molnupiravir poses in terms of sparking new variants, and to weigh them against the drug’s benefits. “If we are playing Russian roulette, we’d better know our odds.”</p><p>Sarah Otto, an evolutionary biologist at the University of British Columbia in Vancouver, Canada, says the paper is another blow to the continued use of molnupiravir. She notes that a large-scale UK study found that the drug had no effect on hospitalizations or deaths<sup><a href="#ref-CR3" data-track="click" data-action="anchor-link" data-track-label="go to reference" data-track-category="references">3</a></sup>. “Given the large-scale risks of this mutagen producing new variants faster, including variants that are immune evasive, I encourage public-health leaders to call for a global halt to its use,” Otto says.</p>
                </div><div id="references">
                <section aria-labelledby="Bib1"><div id="Bib1-section"><h2 id="Bib1">References</h2><div id="Bib1-content"><div data-container-section="references"><ol data-track-component="outbound reference"><li data-counter="1."><p id="ref-CR1">Sanderson, T., Hisner, R., Donovan-Banfield, I., Peacock, T. &amp; Ruis, C. Preprint at medRxiv https://doi.org/10.1101/2023.01.26.23284998 (2023).</p></li><li data-counter="2."><p id="ref-CR2">Rosenke, K. <i>et al.</i> <i>Nature Commun.</i> <b>12</b>, 2295 (2021).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/s41467-021-22580-8" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41467-021-22580-8" aria-label="Article reference 2" data-doi="10.1038/s41467-021-22580-8">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 2" href="http://scholar.google.com/scholar_lookup?&amp;title=&amp;journal=Nature%20Commun.&amp;doi=10.1038%2Fs41467-021-22580-8&amp;volume=12&amp;publication_year=2021&amp;author=Rosenke%2CK.">
                    Google Scholar</a> 
                </p></li><li data-counter="3."><p id="ref-CR3">Butler, C. C. <i>et al.</i> <i>Lancet</i> <b>401</b>, 281–293 (2023).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/S0140-6736(22)02597-1" data-track-action="article reference" href="https://doi.org/10.1016%2FS0140-6736%2822%2902597-1" aria-label="Article reference 3" data-doi="10.1016/S0140-6736(22)02597-1">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 3" href="http://scholar.google.com/scholar_lookup?&amp;title=&amp;journal=Lancet&amp;doi=10.1016%2FS0140-6736%2822%2902597-1&amp;volume=401&amp;pages=281-293&amp;publication_year=2023&amp;author=Butler%2CC.%20C.">
                    Google Scholar</a> 
                </p></li></ol><p><a data-track="click" data-track-action="download citation references" data-track-label="link" rel="nofollow" href="https://citation-needed.springer.com/v2/references/10.1038/d41586-023-00347-z?format=refman&amp;flavour=references">Download references<svg width="16" height="16" focusable="false" role="img" aria-hidden="true"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#global-icon-download"></use></svg></a></p></div></div></div></section>
            </div></div>
  </body>
</html>
