<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <title>James Routley | Feed</title>
    <link
      rel="stylesheet"
      type="text/css"
      href="../styles.css"
      media="screen"
    />
  </head>
  <body>
    <a href="/index.html">Back</a>
    <a href="https://www.owlposting.com/p/creating-the-largest-protein-protein">Original</a>
    <h1>Creating the largest protein-protein interaction dataset in the world</h1>
    
    <div id="readability-page-1" class="page"><div><div><article><div><div><div dir="auto"><div><figure><a target="_blank" href="https://substackcdn.com/image/fetch/f_auto,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F8eb3ff4a-9c8c-47fd-a2f2-4f8a3300cefe_1024x576.png" data-component-name="Image2ToDOM" rel=""><div><picture><source type="image/webp" srcset="https://substackcdn.com/image/fetch/w_424,c_limit,f_webp,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F8eb3ff4a-9c8c-47fd-a2f2-4f8a3300cefe_1024x576.png 424w, https://substackcdn.com/image/fetch/w_848,c_limit,f_webp,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F8eb3ff4a-9c8c-47fd-a2f2-4f8a3300cefe_1024x576.png 848w, https://substackcdn.com/image/fetch/w_1272,c_limit,f_webp,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F8eb3ff4a-9c8c-47fd-a2f2-4f8a3300cefe_1024x576.png 1272w, https://substackcdn.com/image/fetch/w_1456,c_limit,f_webp,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F8eb3ff4a-9c8c-47fd-a2f2-4f8a3300cefe_1024x576.png 1456w" sizes="100vw"/><img src="https://substackcdn.com/image/fetch/w_1456,c_limit,f_auto,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F8eb3ff4a-9c8c-47fd-a2f2-4f8a3300cefe_1024x576.png" width="1024" height="576" data-attrs="{&#34;src&#34;:&#34;https://substack-post-media.s3.amazonaws.com/public/images/8eb3ff4a-9c8c-47fd-a2f2-4f8a3300cefe_1024x576.png&#34;,&#34;srcNoWatermark&#34;:null,&#34;fullscreen&#34;:null,&#34;imageSize&#34;:null,&#34;height&#34;:576,&#34;width&#34;:1024,&#34;resizeWidth&#34;:null,&#34;bytes&#34;:null,&#34;alt&#34;:&#34;Preview&#34;,&#34;title&#34;:null,&#34;type&#34;:null,&#34;href&#34;:null,&#34;belowTheFold&#34;:false,&#34;topImage&#34;:true,&#34;internalRedirect&#34;:null}" alt="Preview" title="Preview" srcset="https://substackcdn.com/image/fetch/w_424,c_limit,f_auto,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F8eb3ff4a-9c8c-47fd-a2f2-4f8a3300cefe_1024x576.png 424w, https://substackcdn.com/image/fetch/w_848,c_limit,f_auto,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F8eb3ff4a-9c8c-47fd-a2f2-4f8a3300cefe_1024x576.png 848w, https://substackcdn.com/image/fetch/w_1272,c_limit,f_auto,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F8eb3ff4a-9c8c-47fd-a2f2-4f8a3300cefe_1024x576.png 1272w, https://substackcdn.com/image/fetch/w_1456,c_limit,f_auto,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F8eb3ff4a-9c8c-47fd-a2f2-4f8a3300cefe_1024x576.png 1456w" sizes="100vw" fetchpriority="high"/></picture></div></a></figure></div><p><em><span>I think this startup is promising and I hope they succeed! This is not a sponsored post, not meant to be anyone’s opinion other than my own, and is not investment advice.</span><a href="https://www.abhishaike.com/p/why-write-about-biotech-startups" rel=""> Here is some more information about why I write about startups at all.</a></em></p><p><em><span>Also, I wrote this post and then accidentally stumbled into a conversation with the CTO (and co-founder), </span><a href="https://www.linkedin.com/in/randolph-lopez-807b3b34/" rel="">Randolph Lopez</a><span>! Consider almost everything in this article a view into what’s publicly available about A-Alpha Bio, and the ‘</span><strong>Addendum:</strong></em><strong> </strong><em><strong>Some notes by the CTO</strong><span>’ section as a view into some of the yet-to-be-released stuff. </span></em></p><p><em>That last section will clear up some questions/misunderstandings I got from purely the publicly available info, so make sure to read it!</em></p><ol><li><p><a href="https://www.owlposting.com/i/147105147/introduction" rel="">Introduction </a></p></li><li><p><a href="https://www.owlposting.com/i/147105147/the-product" rel="">The product </a></p><ol><li><p><a href="https://www.owlposting.com/i/147105147/alphaseq" rel="">AlphaSeq</a></p><ol><li><p><a href="https://www.owlposting.com/i/147105147/how-does-it-work" rel="">How does it work?</a></p></li><li><p><a href="https://www.owlposting.com/i/147105147/what-are-the-alternatives" rel="">What are the alternatives?</a></p></li><li><p><a href="https://www.owlposting.com/i/147105147/outstanding-concerns" rel="">Outstanding concerns</a></p></li></ol></li><li><p><a href="https://www.owlposting.com/i/147105147/alphabind" rel="">AlphaBind</a></p></li></ol></li><li><p><a href="https://www.owlposting.com/i/147105147/how-do-they-make-money" rel="">How do they make money? </a></p><ol><li><p><a href="https://www.owlposting.com/i/147105147/immunocytokines" rel="">Immunocytokines</a></p></li><li><p><a href="https://www.owlposting.com/i/147105147/molecular-glues" rel="">Molecular glues</a></p></li></ol></li><li><p><a href="https://www.owlposting.com/i/147105147/what-bets-is-the-company-making" rel="">What bets is the company making?</a></p></li><li><p><a href="https://www.owlposting.com/i/147105147/addendum-some-notes-by-the-cto" rel="">Addendum: some notes by the CTO</a><span> (read this!) </span></p></li></ol><p><span>Understanding how proteins interact with one another is </span><strong>hard</strong><span>. Genuinely, like stupidly hard. Way, way harder than you’d naively think it is. </span></p><p>One might naturally assume that because Alphafold2 largely solved the monomeric protein structure prediction problem, it let us make headway on the protein interaction problem. </p><p>After all, it’s the same underlying physics: ionic bonds, hydrogen bonds, van der Walls forces, pi-pi interactions, and so on. And it did, but the multimeric version of Alphafold2, and even Alphafold3, is still quite bad at the whole problem. </p><p><strong>Want to know the structure of the human insulin receptor?</strong><span> Alphafold2 is decent at predicting it, no need to crystallize anything. </span></p><p><strong>Want to know how well a 13 amino acid peptide binds to the receptor?</strong><span> You’ll probably need to re-run Alphafold2 with dozens of different seeds, maybe do MSA subsampling, and a laundry list of other ‘hacks’ to get these models to give a somewhat correct structure and, thus, a proxy for binding affinity. And, after all that, you’ll probably still need to do a wet-lab binding assay to confirm the inevitability noisy prediction. </span></p><p>A lot of this has to do with the complexity of so-called protein-protein interactions, or PPI’s.  Even though the space of forces is the same, the distribution of ‘possible’ structures grows exponentially. Atom-sized deviations can cause catastrophic failures in predicted final structures, conformational flexibility can lead to unpredictable binding modes, and the sheer number of potential interaction surfaces explodes combinatorially.</p><p><span>Given enough data though, models should still be able to grasp the complexity. But PPI datasets are notoriously tiny. </span><strong><a href="https://www.pdbbind-plus.org.cn/" rel="">The latest collation of binding affinity dataset</a><span> (PDBbind+) amounts to just 3,176 proteins in total.</span></strong><span> While there are ML models built off such datasets, they are, at best, academic curiosities, and not something essential to have in one’s protein design toolbox. </span></p><p><a href="https://www.aalphabio.com/" rel="">A-Alpha Bio</a><span> is a biotech startup trying to solve the data problem. The company was created in 2017 by two graduate students at the University of Washington, </span><a href="https://www.linkedin.com/in/david-a-younger" rel="">David Younger</a><span> and </span><a href="https://www.linkedin.com/in/randolph-lopez-807b3b34/" rel="">Randolph Lopez</a><span>, the former of whom is a Baker Lab alumni. Their pitch is fundamentally a wet-lab innovation, </span><a href="https://www.owlposting.com/p/wet-lab-innovations-will-lead-the" rel="">which I’ve written about before</a><span> as being a common characteristic among ML-bio companies I’m excited about. </span></p><p>In this essay, we’ll discuss what this innovation is, the computational angle they have, what they are using the innovation for, and the risks the company is taking. </p><p><span>The beginnings of A-Alpha Bio stem from a 2017 paper titled ‘</span><a href="https://www.pnas.org/doi/10.1073/pnas.1705867114" rel="">High-throughput characterization of protein–protein interactions by reprogramming yeast mating’</a><span>. The paper introduces the first iteration of A-Alpha Bio’s method for gathering protein-protein interaction data at scale, which is also called ‘AlphaSeq’. </span></p><p>Let’s go over it!</p><p>AlphaSeq functions via exploiting yeast cell mating. </p><p>Yeast cells have two mating types: MATa and MATα. In nature, these cells find each other and fuse to form diploid cells through a process called sexual agglutination. This process is mediated by two proteins: Aga2 on MATa cells and Sag1 on MATα cells. When these proteins interact, they cause the cells to stick together, facilitating mating + the creation of a diploid cell.</p><p><strong>AlphaSeq hijacks this natural process.</strong><span> Instead of using the native Aga2 and Sag1 proteins, the researchers genetically engineered yeast cells to display proteins of interest on their surface. This alone isn’t particularly novel, this has been done for decades underneath the name ‘</span><a href="https://en.wikipedia.org/wiki/Yeast_display" rel="">yeast surface display</a><span>’. The main novelty here is in exploiting the mating aspect of it: MATa cells display one set of proteins, while MATα cells display another set. When you mix these engineered cells, </span><strong>the likelihood of them mating becomes a function of how strongly the displayed proteins interact. </strong></p><p>But how do you measure these interactions across thousands or millions of protein pairs? The answer is, of course, DNA-encoded libraries. Each protein displayed on the yeast surface is associated with a unique DNA barcode. When two yeast cells mate, these barcodes are brought together in the resulting diploid cell. Through some genetic engineering, these barcodes end up next to each other on the same chromosome. </p><p><span>From there on, you can simply extract the DNA from all these mated cells and sequence it. The frequency with which you see two barcodes together directly correlates with the strength of interaction between the proteins those barcodes represent. </span><strong>More frequent pairings indicate stronger interactions.</strong></p><p><span>Thus, we can get an</span><strong> </strong><span>N by M protein interaction screen, where N MATa cells are each displaying 1 of N proteins, and so on for the M MATα cells. </span></p><p>When I first read about this, I had a list of questions. I’ll go through them, one by one. </p><ol><li><p><strong>That&#39;s a weird mating pattern. Why do yeast cells do that?</strong><span> It’s not </span><strong>that</strong><span> weird, it’s not too dissimilar to male and female sexes in mammals, sex recognition here just occurs via cell surface proteins. And why ‘types’ exist for yeast cells is for the reason sexes exist for animals: a way to nudge life towards greater genetic diversity. One note: this isn’t </span><strong>actually</strong><span> equivalent to sex, I used the phrase ‘mating types’ for a reason. Different sexes imply differently sized gametes (large eggs, small sperm), but the gametes between yeast mating cell types are the same, only surface proteins are different. </span></p></li><li><p><strong>Does each given MATa and MATα cell only express one protein at a time?</strong><span> Yup! The paper doesn’t explore multiple surface protein expression, but, just thinking about it naively, it sounds like it’d be annoying to make something out of the resulting dataset. </span></p></li><li><p><strong>Can this method account for post-translational modifications?</strong><span> The original paper is pessimistic on this front, writing ‘</span><em>the detection of interactions requiring specific post-translational modifications may not be possible</em><span>’. On a more theoretical level, it is known that yeast cells are incapable of recapitulating certain post-translational modifications that mammalian cells can do (e.g </span><a href="https://www.sciencedirect.com/science/article/abs/pii/S0958166907001139" rel="">high-mannose glycans</a><span>), so I’d expect AlphaSeq to have a similar failure mode. </span></p></li><li><p><strong>What is the actual output of an AlphaSeq run? </strong><span> It’s a value referred to as the ‘</span><em>mating efficiency</em><span>’ of the yeast cells. More specifically, it is (number of diploid cells) divided by (number of MATa cells + number of MATα cells + Number of Diploid Cells), for a given protein-protein pair. So, you’d end up with one of these values for every protein-protein pair. </span></p></li><li><p><strong>Is it possible to connect mating efficiency to a traditional binding affinity value?</strong><span> The researchers established a calibration curve using protein pairs with known binding affinities. They found a log-linear relationship between mating efficiency and experimentally-determined binding affinity (Kd) with a pretty high correlation (R</span><sup>2</sup><span> of .87). From the paper’s analyses, there seems to be a wide dynamic range here; the relationship held over multiple orders of magnitude, from 500 pM (strong) to 300 μM (weak). </span><strong>This means that the output of AlphaSeq</strong><span> </span><strong>the mating efficiency values can directly represent a true affinity value, ranging from very strong to very weak.</strong></p></li><li><p><strong>What scale can this operate at? </strong><span>The original paper could generate 7,000 distinct PPI affinity values in a single experimental run. Which is quite large, it immediately outstrips the existing set of publicly available data. But it’s also relatively small given the dimensionality at which biology operates at. </span><a href="https://www.nature.com/articles/s41597-022-01779-4" rel="">A 2023 paper by A-Alpha Bio</a><span> pushed this further, generating binding affinity data for 104k~ antibodies to a single SARS-CoV-2 peptide in a (seemingly?) single run! </span><strong><a href="https://www.aalphabio.com/static/7a44b7b5f69a3e05eb02829a9278c470/ICIS_2023_poster_final.pdf" rel="">Finally, A-Alpha Bio’s most recent work claimed to run 15,000 antibodies by 200 target runs, so 3 million interactions in total.</a><span> </span></strong><span>Curiously, this was a throwaway example in the attached poster and not the actual discussed result. Which is…weird. The actual experiment they ran was far smaller in scale, with less than 500~ total interactions. </span></p></li></ol><p>Overall, cool method! We’ll discuss some outstanding concerns in a bit, but let’s first ponder on what this method is trying to replace. </p><p>There are a few competitors in the ‘high-throughput in-vivo PPI screening’ world. </p><p>It’s a surprisingly diverse area, but two main names seem to consistently pop up, both of which also rely on yeast: yeast two-hybrid (Y2H) and yeast display. We could mention phage display as well, but it has the same practical constraints as yeast display, so I’ll leave out explicitly discussing it for now. </p><p><strong>First, Y2H.</strong><span> In this method, there’s a ‘bait’ protein and a ‘prey’ protein, both of which are proteins you’re interested in assessing interactions between. The &#34;bait&#34; protein is fused to a DNA-binding domain (BD) of a transcription factor, and the &#34;prey&#34; protein is fused to an activation domain (AD) of the same transcription factor. </span></p><p>The DB protein, as the name implies, immediately grabs onto DNA. If the bait and prey proteins interact, the BD and AD are brought together, activating the transcription of a reporter gene, which can be detected with other methods. If they don’t interact, the AD will be (theoretically) incapable of ‘finding’ the reporter gene. As the DB and AD fusion proteins are expressed intracellularly, Y2H occurs entirely in the nucleus. </p><div><figure><a target="_blank" href="https://substackcdn.com/image/fetch/f_auto,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F297272c3-7ce0-4b56-98d3-c54c57e3e2ff_1142x612.png" data-component-name="Image2ToDOM" rel=""><div><picture><source type="image/webp" srcset="https://substackcdn.com/image/fetch/w_424,c_limit,f_webp,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F297272c3-7ce0-4b56-98d3-c54c57e3e2ff_1142x612.png 424w, https://substackcdn.com/image/fetch/w_848,c_limit,f_webp,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F297272c3-7ce0-4b56-98d3-c54c57e3e2ff_1142x612.png 848w, https://substackcdn.com/image/fetch/w_1272,c_limit,f_webp,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F297272c3-7ce0-4b56-98d3-c54c57e3e2ff_1142x612.png 1272w, https://substackcdn.com/image/fetch/w_1456,c_limit,f_webp,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F297272c3-7ce0-4b56-98d3-c54c57e3e2ff_1142x612.png 1456w" sizes="100vw"/><img src="https://substackcdn.com/image/fetch/w_1456,c_limit,f_auto,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F297272c3-7ce0-4b56-98d3-c54c57e3e2ff_1142x612.png" width="1142" height="612" data-attrs="{&#34;src&#34;:&#34;https://substack-post-media.s3.amazonaws.com/public/images/297272c3-7ce0-4b56-98d3-c54c57e3e2ff_1142x612.png&#34;,&#34;srcNoWatermark&#34;:null,&#34;fullscreen&#34;:null,&#34;imageSize&#34;:null,&#34;height&#34;:612,&#34;width&#34;:1142,&#34;resizeWidth&#34;:null,&#34;bytes&#34;:222988,&#34;alt&#34;:null,&#34;title&#34;:null,&#34;type&#34;:&#34;image/png&#34;,&#34;href&#34;:null,&#34;belowTheFold&#34;:true,&#34;topImage&#34;:false,&#34;internalRedirect&#34;:null}" alt="" srcset="https://substackcdn.com/image/fetch/w_424,c_limit,f_auto,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F297272c3-7ce0-4b56-98d3-c54c57e3e2ff_1142x612.png 424w, https://substackcdn.com/image/fetch/w_848,c_limit,f_auto,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F297272c3-7ce0-4b56-98d3-c54c57e3e2ff_1142x612.png 848w, https://substackcdn.com/image/fetch/w_1272,c_limit,f_auto,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F297272c3-7ce0-4b56-98d3-c54c57e3e2ff_1142x612.png 1272w, https://substackcdn.com/image/fetch/w_1456,c_limit,f_auto,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F297272c3-7ce0-4b56-98d3-c54c57e3e2ff_1142x612.png 1456w" sizes="100vw" loading="lazy"/></picture></div></a><figcaption><span>From </span><a href="https://www.singerinstruments.com/resource/yeast-2-hybrid/" rel="">here</a><span>. B is no binding, so no reporter gene activation. C is binding, so there is reporter gene activation. </span></figcaption></figure></div><p><span>You may immediately notice that Y2H doesn’t incorporate barcodes, so the scalability of the whole method is quite low; the possible transcription of the reporter gene must be found one at a time. There is a </span><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4848762/" rel="">method called BFG‐Y2H that fixes this</a><span>, incorporating barcodes into the whole system and allowing us to rely on high-throughput sequencing. This method claims a scale of 2.5M protein-protein pairs, which is quite high! </span></p><p><strong><span>However, Y2H as a method suffers from relatively low accuracy, </span><a href="https://web.mit.edu/endy/www/scraps/signal/JMB(327)919.pdf" rel="">something along the lines of a 50% true-positive rate</a><span>, an accuracy that BFG‐Y2H most likely shares.</span></strong><span> This is primarily due to intracellular environments being a bit uncontrollable, </span><a href="https://www.asimov.press/p/burrito-biology" rel="">a cell is a crowded place after all</a><span>, and spurious interactions are the norm. AlphaSeq likely doesn’t suffer from this problem, given that binding occurs in a less dense + more controllable extracellular region. </span></p><p><strong>Moving on, yeast display is our next contender.</strong><span> The general idea here is simpler than Y2H. Transfect a cell with your desired gene, the protein displays on the surface of the cell, wash the cell with a purified protein you’re interested in, and then use flow cytometry (or something else) to detect binding. The cell surface has one protein of interest, the wash has another protein of interest, and flow cytometry reveals whether binding occurred (via an attached chromophore on the washed purified protein). </span></p><p><span>Yeast display can </span><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4546347/" rel="">operate at the 10M+ scale</a><span> of interactions and, as the interactions are occurring extracellularly, won’t have the same accuracy drawbacks as Y2H has. </span><strong>The disadvantage with this method seems to primarily boil down to two things:</strong><span> </span><strong>dependence on purified proteins and spectral resolution constraints.</strong></p><p>For the former, purified proteins are expensive, may be structurally unstable, and some classes of proteins cannot be purified at all (e.g. membrane proteins). It’d be ideal if everything could simply be expressed within the cell itself, as is done with AlphaSeq. </p><p><span>For the latter, flow cytometry has a finite number of distinct fluorescent channels that can be used simultaneously. Use too many channels and you’re drowned in noise! This means that only a limited number of targets can be used per round</span><a href="https://www.nature.com/articles/nri1416" rel="">, typically around 15-20.</a><span> While more powerful versions of flow cytometry can push that number up more, it still isn’t high! Yeast display is fundamentally incompatible with doing something like a 1000 nanobody to 1000 receptors screen, more like a 1000 nanobody to 10 receptor screen. Comparatively, as AlphaSeq relies on DNA encoding to represent binding, the number of targets can be far higher. </span></p><p>So, AlphaSeq does seem better in some respects. But what about where it’s worse than other methods? </p><p>Obviously, there are problems with yeast as a platform at all. Specifically, in that it cannot recapitulate the same post-translational modifications as done in human cells, which may affect the folding of an expressed protein, which may affect binding. </p><p>But, given that the main competitors are also based on yeast and will suffer from the same problems, let’s ignore those. </p><p>What’s specifically uniquely bad about AlphaSeq? </p><p>Truthfully, I struggle to come up with anything concrete. </p><p><strong>One complaint is potentially that AlphaSeq is worse at scaling measurements — if we are to take existing papers as proof of how far it could be pushed — than other yeast-based PPI detection methods.</strong><span> I imagine that are some inherent physical limitations based on a yeast cell’s probability of being able to interact with every other yeast cell of the opposing mating type. </span></p><p><span>To be clear, maybe it is on par, capable of reaching 10</span><sup>8</sup><span> measurements and beyond. The papers thus far haven’t revealed much…we know 10</span><sup>6</sup><span> scale is absolutely possible, and </span><strong>maybe</strong><span> 10</span><sup>7</sup><span>. The  10</span><sup>7</sup><span> case was a throwaway example and not something they expanded upon heavily, so I’m currently taking it with a grain of salt. Perhaps the error rate of that level of scale is extremely high? </span></p><p><strong>There’s also this vague worry I have about the mating efficiency → binding affinity calculation.</strong><span> It’s surprising to me that it works at all! Variations in how well different proteins are expressed on the yeast surface and the experimental medium of the cell could change folding patterns of proteins, so I wonder how widely applicable the method is in practice amongst diverse proteins. </span></p><p>AlphaBind doesn’t seem to be a single thing, but seems to broadly refer to a suite of ML models that A-Alpha Bio has built on top of their binding affinity data. </p><p><span>There is…basically zero information online about this. They claim to have ‘</span><a href="https://www.aalphabio.com/technology/" rel="">750 million affinity measurements</a><span>’ upon which these models are trained, which is immense, but there aren’t any concrete results showing how useful the resulting model is. </span></p><p>Hoping for a paper in this area to be released soon! </p><p>They seem to be placing bets on both the therapeutic angle and the platform angle.</p><p>For therapeutics, it’s immunocytokines. </p><p>For platforms (in partnerships with Big Pharma), it’s molecular glues. </p><p>At least, that seems to be their focus based on the pipeline on their website. </p><p>Why those two? Likely because both of them (seem to) benefit from high N by M interaction screens, where both N and M are quite high. This is opposed to, say, few-target optimization, where you might only need to screen a large library against one or a handful of targets. </p><p>Let’s go through a quick explanation of both modalities + their work in it. </p><p><span>Immunocytokines are fusion proteins that combine an antibody with a </span><a href="https://en.wikipedia.org/wiki/Cytokine" rel="">cytokine</a><span> (a protein that interacts with the immune system). The antibody component provides targeted delivery to specific cells or tissues, while the cytokine portion delivers an immunomodulatory signal to nearby immune cells. This combination aims to concentrate the cytokine&#39;s effects at the desired site of action (such as a tumor), potentially enhancing efficacy while reducing systemic side effects. </span></p><p><strong>For example, consider an antibody tuned to bind to antigens on a tumor.</strong><span> </span></p><p><span>If we fuse this antibody with </span><a href="https://en.wikipedia.org/wiki/Interleukin_2" rel="">IL-2,</a><span> a cytokine, the antibody will grab onto the tumor and any passing T or NK cells will bind to the free-floating IL-2. </span></p><div><figure><a target="_blank" href="https://substackcdn.com/image/fetch/f_auto,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F8adbc065-18f5-44d0-80b1-5f1140517ace_800x492.jpeg" data-component-name="Image2ToDOM" rel=""><div><picture><source type="image/webp" srcset="https://substackcdn.com/image/fetch/w_424,c_limit,f_webp,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F8adbc065-18f5-44d0-80b1-5f1140517ace_800x492.jpeg 424w, https://substackcdn.com/image/fetch/w_848,c_limit,f_webp,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F8adbc065-18f5-44d0-80b1-5f1140517ace_800x492.jpeg 848w, https://substackcdn.com/image/fetch/w_1272,c_limit,f_webp,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F8adbc065-18f5-44d0-80b1-5f1140517ace_800x492.jpeg 1272w, https://substackcdn.com/image/fetch/w_1456,c_limit,f_webp,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F8adbc065-18f5-44d0-80b1-5f1140517ace_800x492.jpeg 1456w" sizes="100vw"/><img src="https://substackcdn.com/image/fetch/w_1456,c_limit,f_auto,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F8adbc065-18f5-44d0-80b1-5f1140517ace_800x492.jpeg" width="599" height="368.385" data-attrs="{&#34;src&#34;:&#34;https://substack-post-media.s3.amazonaws.com/public/images/8adbc065-18f5-44d0-80b1-5f1140517ace_800x492.jpeg&#34;,&#34;srcNoWatermark&#34;:null,&#34;fullscreen&#34;:null,&#34;imageSize&#34;:null,&#34;height&#34;:492,&#34;width&#34;:800,&#34;resizeWidth&#34;:599,&#34;bytes&#34;:null,&#34;alt&#34;:&#34;An external file that holds a picture, illustration, etc.\nObject name is emss-68179-f002.jpg&#34;,&#34;title&#34;:null,&#34;type&#34;:null,&#34;href&#34;:null,&#34;belowTheFold&#34;:true,&#34;topImage&#34;:false,&#34;internalRedirect&#34;:null}" alt="An external file that holds a picture, illustration, etc.
Object name is emss-68179-f002.jpg" title="An external file that holds a picture, illustration, etc.
Object name is emss-68179-f002.jpg" srcset="https://substackcdn.com/image/fetch/w_424,c_limit,f_auto,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F8adbc065-18f5-44d0-80b1-5f1140517ace_800x492.jpeg 424w, https://substackcdn.com/image/fetch/w_848,c_limit,f_auto,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F8adbc065-18f5-44d0-80b1-5f1140517ace_800x492.jpeg 848w, https://substackcdn.com/image/fetch/w_1272,c_limit,f_auto,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F8adbc065-18f5-44d0-80b1-5f1140517ace_800x492.jpeg 1272w, https://substackcdn.com/image/fetch/w_1456,c_limit,f_auto,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F8adbc065-18f5-44d0-80b1-5f1140517ace_800x492.jpeg 1456w" sizes="100vw" loading="lazy"/></picture></div></a><figcaption><span>From </span><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5215124/" rel="">here</a><span>. Focus on the stuff happening on the right. </span></figcaption></figure></div><p><strong>What happens when IL-2 binds to these passing-by immune cells?</strong><span> A lot of things. </span><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8006145/" rel="">The immune cells ramp up its ability to be cytotoxic, it replicates, and produce even more cytokines to alert other immune cells.</a><span> The hope is through this </span><strong>local</strong><span> activation of the immune cells, the brunt of their ‘response’ is directed towards the tumor. </span></p><p><span>Unfortunately, the efficacy of immunocytokines has been limited by systemic toxicity. This is because the attached </span><strong>cytokine can bind to receptors throughout the body, not just the immune cells</strong><span>. In other words, cross-reactivity is a big problem. </span></p><p><span>The utility of the AlphaSeq platform comes into play here! If you want to build cytokines that can activate a specific target of interest (e.g. immune cell receptors) while avoiding </span><strong>everything else in the body</strong><span>, a scalable protein-protein interaction screen would come in handy.  Engineering thousands of cytokines and assessing their affinity across thousands of receptors, trying to increase affinity to one of these receptors and diminish it on everything else. </span></p><p><span>A-Alpha Bio has poked at one part of this problem in a recent poster titled ‘</span><a href="https://www.aalphabio.com/static/7a44b7b5f69a3e05eb02829a9278c470/ICIS_2023_poster_final.pdf" rel="">Cytokine affinity tuning using the AlphaSeq platform to generate targeted immuno-oncology therapeutics</a><span>’, performing single-site mutagenesis on a wild-type cytokine to assess each of their affinity to two receptors at once. </span></p><p>It works according to their results, but…it’s also somewhat unimpressive. </p><p><span>This was essentially a 500~ cytokine (at the upper end) by 2 receptors PPI screen. </span><strong>Which is a fine problem set-up, but also a bit disappointing if scale is really the selling factor here.</strong><span> I intuitively get the sense that AlphaSeq is a fair bit more accurate than Y2H, given how terrible Y2H is, and easier to set up than Yeast Display, since purified protein isn’t needed, so that’s a win in of itself! </span></p><p>But if scale is what is being advertised, the existing paper on the topic doesn’t quite prove it. My guess is that the paper is more of a proof of concept than anything else, given that this one came after their much larger, 100k~ antibody AlphaSeq screen. Looking forwards to more work in the future!</p><p>Molecular glues are a class of small molecules that facilitate protein-protein interactions (PPIs) that wouldn&#39;t occur naturally or would occur only weakly. Unlike traditional drugs that inhibit protein function, molecular glues work by bringing two proteins together, leading to the degradation of a target protein.</p><p><span>The classic example of a molecular glue is thalidomide. Yes, the drug that causes horrific birth defects </span><a href="https://www.cellgs.com/blog/thalidomide-the-trailblazing-molecular-glue.html" rel="">was one of the first molecular glues ever used in clinical practice</a><span>, though its mechanism of action wasn’t known at the time. </span></p><p><strong>Quick explanation on how it works:</strong><span> Thalidomide allows the cereblon protein (CRBN), an </span><a href="https://en.wikipedia.org/wiki/Ubiquitin_ligase" rel="">E3 ligase</a><span>, to bind to IKAROS proteins, a set of transcription factors. Under ordinary circumstances, these two proteins do not interact. But thalidomide allows it occur by acting, as its name implies, a glue! </span></p><p><strong>Why is the interaction useful?</strong><span> Through some complex chemical magic facilitated by the attached E3 ligase, multiple </span><a href="https://en.wikipedia.org/wiki/Ubiquitin" rel="">ubiquitin</a><span> proteins are attached to the IKAROSE protein, also known as </span><strong>ubiquitination</strong><span>. This particular protein tag is recognized by the </span><a href="https://en.wikipedia.org/wiki/Proteasome" rel="">proteasome</a><span>, which are protein complexes that degrade other proteins into individual amino acids or peptides. Thus, thalidomide reduces levels of intracellular IKAROS proteins by tagging it for degradation. </span></p><p><span>To note, this is why CRBN being an E3 ligase is important, since E3 ligases — </span><a href="https://en.wikipedia.org/wiki/Ubiquitin_ligase" rel="">of which there are 600~</a><span> — are capable of ubiquitination of attached proteins. </span></p><p><strong>Why reduce IKAROS protein levels?</strong><span> Well, cancer often have overactive transcription protein levels due to their rapid growth, so </span><a href="https://www.dana-farber.org/newsroom/news-releases/2013/scientists-discover-how-thalidomide-like-drugs-fight-cancer" rel="">reducing IKAROS levels helps stem their proliferation</a><span>. But the initial prescription of thalidomide was far a more minor and widespread condition: pregnancy morning sickness. Of course, as it turned out, IKAROS proteins are also responsible for increased levels of the FGF8 protein,</span><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4737249/" rel=""> which is essential for correct limb and brain organization during embryonic development</a><span>. Thus, the tragedy of </span><a href="https://en.wikipedia.org/wiki/Thalidomide_scandal" rel="">thalidomide babies.</a></p><p><span>Despite the past of thalidomide, the concept of a molecular glue remained curious, </span><a href="https://cen.acs.org/pharmaceuticals/drug-discovery/Molecular-glues-startups-search-beyond-PROTACS/100/i29" rel="">given its advantages over typical inhibition-based drug</a><span>s, and research continues to this day, still primarily focused on gluing stuff to E3 ligases.</span></p><p><span>On face value, the role of A-Alpha Bio’s AlphaSeq here is a bit uncertain. Molecular glues are small molecules, why is a PPI company interested in them? Curiously, they are placing themselves as not the people discovering glues, but rather the people discovering </span><strong>pairs of proteins where glues can be applied in the first place.</strong><span> </span></p><p><a href="https://www.aalphabio.com/technology/" rel="">From their website:</a></p><div><p>The throughput and sensitivity of AlphaSeq makes the platform well suited to uncover novel targets for molecular glues by discovering and characterizing weak interactions between E3 ubiquitin ligases (or other effector proteins) and target proteins that may be enhanced by a small molecule binding at the interface. We measure interactions between our proprietary E3 ligase library and undruggable target proteins to provide a starting point for the rational discovery of molecular glues. We have discovered and validated many pairs internally and look to partner with industry leaders to progress protein-protein binding insights into small molecule drugs.</p><div><figure><a target="_blank" href="https://substackcdn.com/image/fetch/f_auto,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F992ad562-8ba6-4ff0-888b-e74cead77d5f_900x638.jpeg" data-component-name="Image2ToDOM" rel=""><div><picture><source type="image/webp" srcset="https://substackcdn.com/image/fetch/w_424,c_limit,f_webp,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F992ad562-8ba6-4ff0-888b-e74cead77d5f_900x638.jpeg 424w, https://substackcdn.com/image/fetch/w_848,c_limit,f_webp,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F992ad562-8ba6-4ff0-888b-e74cead77d5f_900x638.jpeg 848w, https://substackcdn.com/image/fetch/w_1272,c_limit,f_webp,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F992ad562-8ba6-4ff0-888b-e74cead77d5f_900x638.jpeg 1272w, https://substackcdn.com/image/fetch/w_1456,c_limit,f_webp,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F992ad562-8ba6-4ff0-888b-e74cead77d5f_900x638.jpeg 1456w" sizes="100vw"/><img src="https://substackcdn.com/image/fetch/w_1456,c_limit,f_auto,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F992ad562-8ba6-4ff0-888b-e74cead77d5f_900x638.jpeg" width="517" height="366.49555555555554" data-attrs="{&#34;src&#34;:&#34;https://substack-post-media.s3.amazonaws.com/public/images/992ad562-8ba6-4ff0-888b-e74cead77d5f_900x638.jpeg&#34;,&#34;srcNoWatermark&#34;:null,&#34;fullscreen&#34;:null,&#34;imageSize&#34;:null,&#34;height&#34;:638,&#34;width&#34;:900,&#34;resizeWidth&#34;:517,&#34;bytes&#34;:null,&#34;alt&#34;:null,&#34;title&#34;:null,&#34;type&#34;:null,&#34;href&#34;:null,&#34;belowTheFold&#34;:true,&#34;topImage&#34;:false,&#34;internalRedirect&#34;:null}" alt="" srcset="https://substackcdn.com/image/fetch/w_424,c_limit,f_auto,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F992ad562-8ba6-4ff0-888b-e74cead77d5f_900x638.jpeg 424w, https://substackcdn.com/image/fetch/w_848,c_limit,f_auto,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F992ad562-8ba6-4ff0-888b-e74cead77d5f_900x638.jpeg 848w, https://substackcdn.com/image/fetch/w_1272,c_limit,f_auto,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F992ad562-8ba6-4ff0-888b-e74cead77d5f_900x638.jpeg 1272w, https://substackcdn.com/image/fetch/w_1456,c_limit,f_auto,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F992ad562-8ba6-4ff0-888b-e74cead77d5f_900x638.jpeg 1456w" sizes="100vw" loading="lazy"/></picture></div></a></figure></div></div><p>I was initially a little skeptical of the value of this. </p><p><span>Intuitively, my understanding of molecular glues is that that weak interactions aren’t strictly necessary; they can cause interactions between </span><strong>anything</strong><span>. What’s the purpose of learning about weak interactions?</span></p><p><span>As it turns out, while molecular glues can induce interactions between non-interacting proteins, </span><strong>the primary way they function is by stabilizing weak interactions!</strong><span> </span></p><p><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9994104/" rel="">One review paper over molecular glues explicitly states</a><span>:</span></p><ul><li><p><em>Many molecular glues take advantage of weak, fortuitous, pre-existing protein–protein interfaces (PPIs) that can be further strengthened by their binding</em></p></li><li><p><em>Discovering weak protein–protein interactions that can be further stabilized is key to develop molecular glues. Rational design of molecular glues has been difficult mainly due to a lack of understanding and predictability of weak interactions</em><span>.</span></p></li></ul><p><a href="https://pubmed.ncbi.nlm.nih.gov/38104659/" rel="">Another review paper discussing the challenges of molecular glue development says something similar</a><span>:</span></p><ul><li><p><em>We believe that utilizing methods such as high-throughput global proteomics to understand the target interactome ahead of establishing screening will be more successful. This will undoubtedly require assays that are able to measure weak and transient interactions between target and effector</em><span>.</span></p></li></ul><p><span>This also answers a side piece of skepticism I had about whether pairs are actually the important piece (versus the molecular glue development itself)! </span><strong>It seems that the molecular glue field is so early that good targets to develop glues on top of are, for the most part, still unknown.</strong><span> </span></p><p><span>This is where A-Alpha Bio comes in, as they are the only group who can screen many E3 ligases against many target proteins. </span><strong>They aren’t developing a therapeutic themselves, but helping form the basis for a therapeutic at all!</strong><span> This is likely why many of their existing partnerships — </span><a href="https://www.aalphabio.com/news/" rel="">Amgen, Squibb, and Kymera Therapeutics </a><span>— all focus on molecular glues. </span></p><p><span>It’s hard to assess how well this is going, since there are no publications focused on it, but partnerships usually mean there is </span><strong>something</strong><span> here!</span></p><p><span>A-Alpha Bio feels deeply interesting to me. Again, it’s built off a fundamental advance in wet lab innovation, </span><a href="https://www.owlposting.com/p/wet-lab-innovations-will-lead-the" rel="">which I’ve discussed previously as being very important</a><span>, and has lots of ongoing partnerships. Their future feels promising!</span></p><p>But all companies, by virtue of existing at all, are implicitly making bets on where the future is heading. A-Alpha Bio is no different. Let’s end this essay by going through them!</p><p>Here are the bets I’m seeing. These are in no particular order! Generally, there aren’t any huge concerns I have. </p><ol><li><p><strong>AlphaSeq</strong><span> </span><strong>will be able to scale further. </strong><span>There’s a deep disconnect between the claimed capabilities of AlphaSeq (millions of interactions per experiment) and the </span><a href="https://www.nature.com/articles/s41597-022-01779-4" rel="">published capabilities</a><span> (100k~ in one run). While there is this constantly </span><a href="https://www.aalphabio.com/static/7a44b7b5f69a3e05eb02829a9278c470/ICIS_2023_poster_final.pdf" rel="">mentioned line about being able to scale up to 3M interactions</a><span>, there is no published paper about that experiment specifically! Very curious to know why! It may be the case that such experiments are simply too expensive to run for an academic paper and that AlphaSeq is perfectly capable of scaling as much as desired. And even if 3M can be reached, could it be pushed even further? </span></p></li><li><p><strong>N by M interactions will continue to be valuable, compared to N by &lt;20 interactions.</strong><span> AlphaSeq’s primary alpha lies in its ability to do many-by-many screening. This isn’t naively </span><strong>always</strong><span> valuable; many drug programs focus explicitly on a few targets and optimize molecules for those specifically. For those types of screens, you could do typical yeast/phage display. AlphaSeq’s sort of exploratory screens are well suited for A-Alpha Bio’s programs (immunocytokines and molecular glues), so this is a bet that those programs either yield results or that there are other areas where N by M screens are useful. </span></p></li><li><p><strong>There is high translatability between yeast-cell expressed proteins and in-vivo human proteins.</strong><span> It&#39;s still unclear how well the data collected by A-Alpha Bio translates to human biology, given the differences in post-translational modifications between yeast and human cells. While this limitation will be present in any yeast screening methods, it becomes especially pertinent if your entire company is built off a yeast-based method! Still, low priority in general given that they have done some </span><a href="https://www.aalphabio.com/static/7a44b7b5f69a3e05eb02829a9278c470/ICIS_2023_poster_final.pdf" rel="">validation of translatability for a few proteins.</a></p></li><li><p><strong>AlphaBind models works better than other people’s models. </strong><span>The lack of public information or academic publications about the suite of AlphaBind models is somewhat concerning. It may be the case that more foundation-model-esque things like </span><a href="https://www.nature.com/articles/s41586-024-07487-w" rel="">AlphaFold3</a><span> are strongly outperforming internal AlphaBind results, or even simple dataset cleanups of existing open-sourced data, like </span><a href="https://www.biorxiv.org/content/10.1101/2024.07.17.603980v2.full.pdf" rel="">PINDER</a><span>, get you as far as AlphaBind. Alternatively, A-Alpha Bio may just be keeping things under wraps! Hard to tell. </span></p></li></ol><p>That’s about it! Overall, this is a company I’m enormously bullish on, and I have high hopes for their future success. </p><p>Thank you for reading!</p><p>This is a section I made last minute! </p><p><a href="https://x.com/owl_poster/status/1822375485697147144" rel="">After teasing the post on Twitter the other day,</a><span> the CTO/co-founder of A-Alpha Bio,</span><em> </em><a href="https://www.linkedin.com/in/randolph-lopez-807b3b34/" rel="">Randolph Lopez</a><em>, </em><span>followed me, and I took a chance to ask a few questions that were on my mind after writing the post. </span></p><p>I’ll organize this as an FAQ, with a ‘C’ for comments. </p><p><strong>Q: What is the scale that AlphaSeq is capable of?</strong><span> </span></p><p><em>A: We currently execute about 30 AlphaSeq assays per month ranging for 100k intersections to about 5M per assay. The larger the network size, the more difficult it becomes to detect weak interactions so different applications require different network sizes.</em></p><p>C: This answers the 3M question; it’s not just a throwaway example, it’s real and possible! The ‘30 AlphaSeq assays per month’ comment also feels insane given the scale of each output. That’s 30M-150M datapoints each month!</p><p><strong>Q: What’s the limiting factor to pushing AlphaSeq further?</strong><span> </span></p><p><em>A: Fundamentally, the two constraints to network size are assay volume, which determines yeast collision sampling and sequencing coverage.</em><span> </span></p><p><span>C: The yeast collision sampling part sounds similar to what I mentioned earlier: ‘</span><em>inherent physical limitations based on a yeast cell’s probability of being able to interact with every other yeast cell of the opposing mating type.</em><span>’. But the sequencing coverage part wasn’t something I would’ve guessed is a bottleneck! It makes sense though, number of reads per unique pair will dramatically fall as the matrix grows and the confidence in any given result likely plummets. </span></p><p><strong>Q: Is there a limitation on the diversity of proteins used in the AlphaSeq screen? Like, is 1000 random proteins by 1000 random proteins viable?</strong></p><p>A: It is viable, main challenge is DNA synthesis cost. We’ve been experiment with oligo batch assembly approaches to build larger protein libraries from oligo pools to circumvent this.</p><p><span>C: No real comments, cool! My original concern about diversity (‘</span><em>I wonder how widely applicable the method is in practice amongst diverse proteins.‘</em><span>) seems to be a non-issue.</span></p><p>Q: Will anything about AlphaBind will be published soon?</p><p><span>A: </span><em>That’s the plan! Aiming to have something out before EOY.</em></p><p>C: Hype!</p></div></div></div></article></div></div></div>
  </body>
</html>
