<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <title>James Routley | Feed</title>
    <link
      rel="stylesheet"
      type="text/css"
      href="../styles.css"
      media="screen"
    />
  </head>
  <body>
    <a href="/index.html">Back</a>
    <a href="https://www.nature.com/articles/s41565-023-01527-8">Original</a>
    <h1>DNA turbine powered by a transmembrane potential across a nanopore</h1>
    
    <div id="readability-page-1" class="page"><div>
                <section data-title="Main"><div id="Sec1-section"><h2 id="Sec1">Main</h2><div id="Sec1-content"><p>At the heart of any active mechanical system is an engine, which converts one type of energy, typically chemical or electrical, into mechanical work. In biological systems, such work is done by motor proteins such as kinesin<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 1" title="Carter, N. J. &amp; Cross, R. Mechanics of the kinesin step. Nature 435, 308–312 (2005)." href="https://www.nature.com/articles/s41565-023-01527-8#ref-CR1" id="ref-link-section-d6783710e592">1</a></sup>, the bacterial flagella motor<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 2" title="Berg, H. C. The rotary motor of bacterial flagella. Annu. Rev. Biochem. 72, 19–54 (2003)." href="https://www.nature.com/articles/s41565-023-01527-8#ref-CR2" id="ref-link-section-d6783710e596">2</a></sup> and F<sub>o</sub>F<sub>1</sub>-ATP synthase<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 3" title="Hahn, A., Vonck, J., Mills, D. J., Meier, T. &amp; Kühlbrandt, W. Structure, mechanism, and regulation of the chloroplast ATP synthase. Science 360, eaat4318 (2018)." href="https://www.nature.com/articles/s41565-023-01527-8#ref-CR3" id="ref-link-section-d6783710e604">3</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 4" title="Srivastava, A. P. et al. High-resolution cryo-EM analysis of the yeast ATP synthase in a lipid membrane. Science 360, eaas9699 (2018)." href="https://www.nature.com/articles/s41565-023-01527-8#ref-CR4" id="ref-link-section-d6783710e607">4</a></sup>. In the latter, electrochemical potential energy from a concentration gradient of ions is converted into mechanical rotary motion of the F<sub>o</sub> motor, which drives the F<sub>1</sub> rotary complex to catalyse the synthesis of ATP, the molecule that provides free energy for many cellular processes. Despite the extensive knowledge and success of building rotary engines of sizes spanning many orders of magnitude on the macroscale, designing, building and demonstrating functioning artificial nanoscale counterparts of these sophisticated biological motors has proven challenging.</p><p>The critical step of building such nanoscale rotary engines is to demonstrate their ability to transduce local free energy continuously and autonomously into designed mechanical motion and useful work. Previous work led to multiple designs of rotary assemblies<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Ketterer, P., Willner, E. M. &amp; Dietz, H. Nanoscale rotary apparatus formed from tight-fitting 3D DNA components. Sci. Adv. 2, e1501209 (2016)." href="#ref-CR5" id="ref-link-section-d6783710e619">5</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Bertosin, E. et al. A nanoscale reciprocating rotary mechanism with coordinated mobility control. Nat. Commun. 12, 7138 (2021)." href="#ref-CR6" id="ref-link-section-d6783710e619_1">6</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Courbet, A. et al. Computational design of mechanically coupled axle–rotor protein assemblies. Science 376, 383–390 (2022)." href="#ref-CR7" id="ref-link-section-d6783710e619_2">7</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 8" title="Ahmadi, Y. et al. The Brownian and flow‐driven rotational dynamics of a multicomponent DNA origami‐based rotor. Small 16, 2001855 (2020)." href="https://www.nature.com/articles/s41565-023-01527-8#ref-CR8" id="ref-link-section-d6783710e622">8</a></sup>, and established a certain level of directed motion as an external operator manually cycled environmental conditions<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 9" title="Erbas-Cakmak, S., Leigh, D. A., McTernan, C. T. &amp; Nussbaumer, A. L. Artificial molecular machines. Chem. Rev. 115, 10081–10206 (2015)." href="https://www.nature.com/articles/s41565-023-01527-8#ref-CR9" id="ref-link-section-d6783710e626">9</a></sup> such as light and temperature<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 10" title="Van Delden, R. A. et al. Unidirectional molecular motor on a gold surface. Nature 437, 1337–1340 (2005)." href="https://www.nature.com/articles/s41565-023-01527-8#ref-CR10" id="ref-link-section-d6783710e630">10</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 11" title="Roke, D., Wezenberg, S. J. &amp; Feringa, B. L. Molecular rotary motors: unidirectional motion around double bonds. Proc. Natl Acad. Sci. USA 115, 9423–9431 (2018)." href="https://www.nature.com/articles/s41565-023-01527-8#ref-CR11" id="ref-link-section-d6783710e633">11</a></sup>, chemical compounds<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 12" title="Qiu, Y., Feng, Y., Guo, Q.-H., Astumian, R. D. &amp; Stoddart, J. F. Pumps through the ages. Chem 6, 1952–1977 (2020)." href="https://www.nature.com/articles/s41565-023-01527-8#ref-CR12" id="ref-link-section-d6783710e637">12</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 13" title="Borsley, S., Kreidt, E., Leigh, D. A. &amp; Roberts, B. M. Autonomous fuelled directional rotation about a covalent single bond. Nature 604, 80–85 (2022)." href="https://www.nature.com/articles/s41565-023-01527-8#ref-CR13" id="ref-link-section-d6783710e640">13</a></sup> or alternate macroscale electric fields<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 14" title="Kopperger, E. et al. A self-assembled nanoscale robotic arm controlled by electric fields. Science 359, 296–301 (2018)." href="https://www.nature.com/articles/s41565-023-01527-8#ref-CR14" id="ref-link-section-d6783710e644">14</a></sup>. Molecular dynamics (MD) simulations have shown the conceptual feasibility of using a DNA helix to convert electric field into torque<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 15" title="Maffeo, C., Quednau, L., Wilson, J. &amp; Aksimentiev, A. DNA double helix, a tiny electromotor. Nat. Nanotechnol. 18, 238–242 (2023)." href="https://www.nature.com/articles/s41565-023-01527-8#ref-CR15" id="ref-link-section-d6783710e649">15</a></sup>; however, experimental demonstration of a rotary mechanism programmed for sustained conversion of a transmembrane electric potential into the mechanical rotation had not been achieved.</p><p>Here, we demonstrate a bottom-up designed DNA nanoturbine that is powered by a nanoscale hydrodynamic flow inside a nanopore. It contains a central axle decorated with three blades arranged in a chiral configuration, either left or right handed. The turbine has a height of 24 or 27 nm, comparable to the 20-nm-tall ATP synthase. The turbine’s stator is provided by a solid-state nanopore in a 20-nm-thickness silicon nitride membrane. Using single-molecule fluorescence, we monitor the rotation of the nanoturbine driven by either a direct current (DC) voltage or a transmembrane ion gradient, which mimics the working environment of rotary motors in biological cells. As we demonstrate below, the nanoturbine can drive a long DNA bundle as a hydrodynamic load into sustained rotary motion of up to 10 revolutions s<sup>−1</sup>, equivalent to delivering tens of piconewton nanometres of torque, which compares well with the ~50 pN nm torque that can be generated by natural ATP synthase<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 16" title="Klusch, N., Murphy, B. J., Mills, D. J., Yildiz, Ö. &amp; Kühlbrandt, W. Structural basis of proton translocation and force generation in mitochondrial ATP synthase. eLife 6, e33274 (2017)." href="https://www.nature.com/articles/s41565-023-01527-8#ref-CR16" id="ref-link-section-d6783710e658">16</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 17" title="Junge, W., Sielaff, H. &amp; Engelbrecht, S. Torque generation and elastic power transmission in the rotary FOF1-ATPase. Nature 459, 364–370 (2009)." href="https://www.nature.com/articles/s41565-023-01527-8#ref-CR17" id="ref-link-section-d6783710e661">17</a></sup>.</p></div></div></section><section data-title="Design of DNA nanoturbines"><div id="Sec2-section"><h2 id="Sec2">Design of DNA nanoturbines</h2><div id="Sec2-content"><p>Our DNA nanoturbine is a multilayer DNA origami structure containing an intentionally designed chiral twist (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41565-023-01527-8#Fig1">1a,b</a>). The structure consists of 30 double-stranded DNA helices, each 72 base pairs (bp) in length on average, where the six parallel central helices form an axle, and the three eight-helix blades are obliquely attached to the axle and symmetrically spaced at 120° angles across the circumference. The chiral twist in the blades of the turbine is induced by adjusting the number of base pairs between each staple crossover away from the one-per-7 bp value required for an achiral structure<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 18" title="Dietz, H., Douglas, S. M. &amp; Shih, W. M. Folding DNA into twisted and curved nanoscale shapes. Science 325, 725–730 (2009)." href="https://www.nature.com/articles/s41565-023-01527-8#ref-CR18" id="ref-link-section-d6783710e676">18</a></sup>, yielding a strongly right-handed twisted structure for an 8 bp crossover density and a left-handed twisted turbine for 6.5 bp spacings on average, while maintaining a good folding yield of the structures (Supplementary Figs. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41565-023-01527-8#MOESM1">1–5</a>). The objects were self-assembled as described previously<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 19" title="Castro, C. E. et al. A primer to scaffolded DNA origami. Nat. Methods 8, 221–229 (2011)." href="https://www.nature.com/articles/s41565-023-01527-8#ref-CR19" id="ref-link-section-d6783710e683">19</a></sup> (for details, see <a data-track="click" data-track-label="link" data-track-action="section anchor" href="https://www.nature.com/articles/s41565-023-01527-8#Sec6">Methods</a>). We used single-particle cryogenic electron microscopy (cryo-EM) to determine three-dimensional (3D) electron density maps of the right-handed and left-handed turbine structures (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41565-023-01527-8#Fig1">1c,e</a> and Supplementary Figs. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41565-023-01527-8#MOESM1">6–9</a>). The cryo-EM reconstructions showed the desired structural features such as the three blades and the axle, and revealed the twisted orientation of the blades. The twist and the blade angles were measured from the cryo-EM data, yielding a −1.1° bp<sup>−1</sup> twist density and a blade angle with respect to the turbine axis of −36° for the right-handed structure, and +0.69° bp<sup>−1</sup> and +24° for the left-handed structure (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41565-023-01527-8#Fig1">1d,f</a>; for details see <a data-track="click" data-track-label="link" data-track-action="section anchor" href="https://www.nature.com/articles/s41565-023-01527-8#Sec6">Methods</a> and Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41565-023-01527-8#MOESM1">10</a>). The turbine structures were 27 nm and 24 nm tall (right and left handed, respectively) and had diameters of 27 nm and 25 nm, respectively.</p><div data-test="figure" data-container-section="figure" id="figure-1" data-title="Design of a nanopore-powered DNA origami turbine."><figure><figcaption><b id="Fig1" data-test="figure-caption-text">Fig. 1: Design of a nanopore-powered DNA origami turbine.</b></figcaption><div><div><a data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="https://www.nature.com/articles/s41565-023-01527-8/figures/1" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41565-023-01527-8/MediaObjects/41565_2023_1527_Fig1_HTML.png?as=webp"/><img aria-describedby="Fig1" src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41565-023-01527-8/MediaObjects/41565_2023_1527_Fig1_HTML.png" alt="figure 1" loading="lazy" width="685" height="692"/></picture></a></div><p><b>a</b>, Schematic of a right-handed DNA turbine docked into a nanopore (side view at top; axial view at bottom). <b>b</b>, Two cross-sections of the DNA turbine highlighting the designed twist. <b>c</b>, 3D electron density map of the right-handed DNA turbine determined via single-particle cryo-EM (side and bottom views; see also Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41565-023-01527-8#MOESM1">2</a>). <b>d</b>, Cross-sections at the top and the bottom of the 3D cryo-EM reconstruction right-handed turbine, highlighting the right-handed twist density of −1.1° bp<sup>−1</sup>. <b>e</b>,<b>f</b>, The same as <b>c</b>,<b>d</b> but for the left-handed DNA turbine, highlighting the twist density of +0.69° bp<sup>−1</sup>. <b>g</b>, Schematic of a right-handed DNA turbine with its load, a 300-nm-long DNA bundle with the middle 220 nm reinforced with 16 DNA helices instead of 6 helices, and a 900 nm looped leash docked onto a solid-state silicon nitride nanopore. <b>h</b>, SNUPI-simulated structure of the right-handed DNA turbine with a DNA bundle attached as a load (leash excluded). <b>i</b>, Negatively stained transmission electron micrograph of a typical right-handed DNA turbine with the load.</p></div></figure></div></div></div></section><section data-title="Unidirectional rotation of DNA turbines driven by a salinity gradient or transmembrane voltage"><div id="Sec3-section"><h2 id="Sec3">Unidirectional rotation of DNA turbines driven by a salinity gradient or transmembrane voltage</h2><div id="Sec3-content"><p>To demonstrate that the turbines can generate torque and work, we docked the structure into a nanopore, and optically monitored rotations at the single-molecule level. To create a hydrodynamic load as well as to hold the turbine in the nanopore and to facilitate optical tracking using super-resolution microscopy, we attached a 300-nm-long DNA six-helix bundle as a crossbar featuring a reinforced 220-nm-long central 16-helix bundle segment (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41565-023-01527-8#MOESM1">12</a>) to the top part of the turbine axle as one continuous rigid body (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41565-023-01527-8#Fig1">1g</a>). One end of the DNA bundle was labelled with ten Cy3 fluorophores to allow continuous monitoring of its motion by fluorescence microscopy at 5 ms temporal resolution and sub-diffraction-limit localization precision (<a data-track="click" data-track-label="link" data-track-action="section anchor" href="https://www.nature.com/articles/s41565-023-01527-8#Sec6">Methods</a>). A 900-nm-long loop of nicked double-stranded DNA was engineered to extend from the bottom of the axle and act as a leash guiding the insertion of the turbine into the nanopore during the docking process (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41565-023-01527-8#Fig1">1g</a>). Coarse-grained simulations (SNUPI<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 20" title="Lee, J. Y. et al. Rapid computational analysis of DNA origami assemblies at near-atomic resolution. ACS Nano 15, 1002–1015 (2021)." href="https://www.nature.com/articles/s41565-023-01527-8#ref-CR20" id="ref-link-section-d6783710e792">20</a></sup>) were used to analyse the structural rigidity of the integrated design (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41565-023-01527-8#Fig1">1h</a>). Proper folding of the entire assembly was verified by negative-stained transmission electron microscopy (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41565-023-01527-8#Fig1">1i</a>; for details see <a data-track="click" data-track-label="link" data-track-action="section anchor" href="https://www.nature.com/articles/s41565-023-01527-8#Sec6">Methods</a>). An array of 50-nm-diameter nanopores was fabricated in 20-nm-thickness silicon nitride membranes using electron-beam lithography and reactive ion etching (<a data-track="click" data-track-label="link" data-track-action="section anchor" href="https://www.nature.com/articles/s41565-023-01527-8#Sec6">Methods</a>) and characterized with transmission electron microscopy (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41565-023-01527-8#MOESM1">13</a>).</p><p>Our single-molecule observations show that the DNA turbine can drive a unidirectional rotation of the load under a transmembrane gradient of ion concentration. Initially, both compartments of the flow cell were filled with 50 mM NaCl buffer, and the DNA turbines were added to the <i>cis</i> compartment. Subsequently, a higher concentration of NaCl (0.5–3 M) was flushed into the <i>trans</i> compartment (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41565-023-01527-8#Fig2">2a</a>), causing the DNA turbines to move towards the nanopores by diffusiophoresis, which led to the insertion of the turbines into the nanopores. As the leash guided the initial insertion of the large structure<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 21" title="Ermann, N. et al. Promoting single-file DNA translocations through nanopores using electro-osmotic flow. J. Chem. Phys. 149, 163311 (2018)." href="https://www.nature.com/articles/s41565-023-01527-8#ref-CR21" id="ref-link-section-d6783710e824">21</a></sup>, the turbine oriented itself upon docking in the designed orientation, where the extended crossbar bundle prevented the turbine from translocating through the nanopore. After docking, we tracked the rotary motion of the DNA bundle by monitoring the position of the fluorophores, which were located at one end of the bundle (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41565-023-01527-8#MOESM1">14</a>).</p><div data-test="figure" data-container-section="figure" id="figure-2" data-title="Sustained unidirectional rotation of DNA origami turbines in a salt gradient."><figure><figcaption><b id="Fig2" data-test="figure-caption-text">Fig. 2: Sustained unidirectional rotation of DNA origami turbines in a salt gradient.</b></figcaption><div><div><a data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="https://www.nature.com/articles/s41565-023-01527-8/figures/2" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41565-023-01527-8/MediaObjects/41565_2023_1527_Fig2_HTML.png?as=webp"/><img aria-describedby="Fig2" src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41565-023-01527-8/MediaObjects/41565_2023_1527_Fig2_HTML.png" alt="figure 2" loading="lazy" width="685" height="547"/></picture></a></div><div data-test="bottom-caption" id="figure-2-desc"><p><b>a</b>, Schematic of a DNA turbine docking onto a nanopore by diffusiophoresis. <b>b</b>, Typical heatmap (blue pixels) of obtained centres of fluorophores at the tip of DNA bundle from single-particle localizations from 8,000 frames (<a data-track="click" data-track-label="link" data-track-action="section anchor" href="https://www.nature.com/articles/s41565-023-01527-8#Sec6">Methods</a>) and example trajectory of six subsequent positions of the labelled tip (red), which shows clear directional rotation. <b>c</b>, Typical cumulative angle versus time for a right-handed turbine in 50 mM:1 M NaCl, showing a sustained rotation of hundreds of turns. <b>d</b>,<b>e</b>, Cumulative angle versus time of left-handed (<b>d</b>) and right-handed (<b>e</b>) turbines for a NaCl concentration gradient of 50 mM:1 M (<i>n</i> = 77 and 151, respectively). <b>f</b>, Average rotation speed of left- and right-handed turbines in transmembrane NaCl concentration gradients of 50 mM:500 mM and 50 mM:1 M (<i>n</i> = 98, 141, 124 and 74, respectively). In all box plots: centre line, median; box limits, upper and lower quartiles; whiskers, 1.5 × interquartile range.</p><p><a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41565-023-01527-8#MOESM5">Source data</a></p></div></div></figure></div><p>As shown in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41565-023-01527-8#Fig2">2</a>, we observed clear, directed rotation of the turbine particles. Figure <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41565-023-01527-8#Fig2">2b</a> shows the rotary motion of the bundle end, which can be traced to follow a circular path over time. Figure <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41565-023-01527-8#Fig2">2c</a> shows the cumulative angular displacement corresponding to the example in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41565-023-01527-8#Fig2">2b</a>, which continues over 200 clockwise rotations over the full period of observation (40 s in this case). Figure <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41565-023-01527-8#Fig2">2d,e</a> shows typical data for 228 turbines, where right- and left-handed turbines led to, respectively, upwards and downwards linear angular rotation curves. The linearity of these curves (and the corresponding superlinear mean-square angular rotation curves; Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41565-023-01527-8#MOESM1">15</a>) is direct evidence of a driven motion. The data show that the DNA turbines can exert a substantial torque on the DNA load bundle and drive it into sustained unidirectional rotary motion.</p><p>Importantly, we find that the designed chirality sets the rotation direction. The turbine variant with left-handed turbine blades displayed preferentially anticlockwise rotations (as viewed from the <i>cis</i> side). In contrast, the turbines with right-handed turbine blades showed almost exclusively clockwise rotations. These data indicate that the designed chirality controls the rotation direction. We determined the average angular velocity in different salt-gradient conditions (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41565-023-01527-8#Fig2">2f</a> and Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41565-023-01527-8#MOESM1">16</a>). The velocity directions corresponded well to the designed chiralities of the structures, and the angular velocities were distributed with noticeable spread, with maximum values as high as ~10 revolutions s<sup>−1</sup>. We attribute the spread in the velocity distribution to heterogeneity in the local interactions between the DNA structure and the silicon nitride surface and to potential deformations in the DNA turbine crossbar that can modulate the rotation speed<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 22" title="Shi, X. et al. Sustained unidirectional rotation of a self-organized DNA rotor on a nanopore. Nat. Phys. 18, 1105–1111 (2022)." href="https://www.nature.com/articles/s41565-023-01527-8#ref-CR22" id="ref-link-section-d6783710e925">22</a></sup>.</p><p>Subsequently, we operated the DNA turbines under a transmembrane voltage that was applied across the compartments at an equal 50 mM NaCl concentration. Immediately after applying the transmembrane voltage (100 mV, Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41565-023-01527-8#Fig3">3a,b</a>), docking and rotary motion of DNA turbines was observed—see Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41565-023-01527-8#Fig3">3b,h</a> for typical traces of a left-handed and a right-handed DNA turbine, respectively. Clear circular trajectories were obtained, indicating a sustained and constant rotary motion over time, very similar to the trajectories observed in the ion-gradient-driven experiments. The extracted rotational velocities of the DNA load bundle showed driven rotary motion, with again predominantly the same rotation direction depending on the chirality of the turbine variant under study, as can be seen in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41565-023-01527-8#Fig3">3c,i</a>. From the rotational speed of the DNA beam, we estimated the torque of our DNA turbine (Supplementary Section <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41565-023-01527-8#MOESM1">1</a> and Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41565-023-01527-8#MOESM1">20</a>) as tens of piconewton nanometres. As a control, we also tested an approximately non-chiral, straight version of the turbine with the same crossbar load (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41565-023-01527-8#MOESM1">18</a>), which was assembled by removing the residual twist in the blades<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 18" title="Dietz, H., Douglas, S. M. &amp; Shih, W. M. Folding DNA into twisted and curved nanoscale shapes. Science 325, 725–730 (2009)." href="https://www.nature.com/articles/s41565-023-01527-8#ref-CR18" id="ref-link-section-d6783710e952">18</a></sup>. For this variant, no preferred rotational directionality was observed, while some residual rotation without preferred directionality was observed due to the self-organization of the DNA crossbar<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 21" title="Ermann, N. et al. Promoting single-file DNA translocations through nanopores using electro-osmotic flow. J. Chem. Phys. 149, 163311 (2018)." href="https://www.nature.com/articles/s41565-023-01527-8#ref-CR21" id="ref-link-section-d6783710e956">21</a></sup> (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41565-023-01527-8#MOESM1">18</a>).</p><div data-test="figure" data-container-section="figure" id="figure-3" data-title="Sustained unidirectional rotation of DNA origami turbines in an applied field."><figure><figcaption><b id="Fig3" data-test="figure-caption-text">Fig. 3: Sustained unidirectional rotation of DNA origami turbines in an applied field.</b></figcaption><div><div><a data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="https://www.nature.com/articles/s41565-023-01527-8/figures/3" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41565-023-01527-8/MediaObjects/41565_2023_1527_Fig3_HTML.png?as=webp"/><img aria-describedby="Fig3" src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41565-023-01527-8/MediaObjects/41565_2023_1527_Fig3_HTML.png" alt="figure 3" loading="lazy" width="685" height="637"/></picture></a></div><div data-test="bottom-caption" id="figure-3-desc"><p><b>a</b>, Schematic of a DNA turbine docking into and undocking from a nanopore on applying a transmembrane voltage. <b>b</b>, Typical cumulative angle versus time for a left-handed turbine for a 100 mV bias voltage in 50 mM NaCl, showing a sustained rotation over hundreds of turns. Inset: corresponding heatmap (blue pixels) of single-particle localizations for the tip of the DNA bundle with an example trajectory of the labelled tip overlaid. <b>c</b>, Cumulative angular-displacement curves for left-handed turbine-driven DNA bundles as in <b>b</b> but for <i>n</i> = 210 turbines. <b>d</b>,<b>e</b>, The same as <b>b</b>,<b>c</b> but in 3 M NaCl electrolyte (<i>n</i> = 159). <b>f</b>, Average rotation speed for left-handed turbines in NaCl concentrations of 50 mM, 500 mM, 1 M, 2 M and 3 M (<i>n</i> = 198, 77, 86, 252 and 150 respectively). <b>g</b>, Mean rotary speed of left-handed turbines for various buffer salt concentrations (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41565-023-01527-8#MOESM1">8</a>). Error bars are s.e.m. <b>h</b>–<b>m</b>, The same as <b>b</b>–<b>g</b> but for right-handed DNA turbines (<i>n</i><sub><i>i</i></sub> = 174, <i>n</i><sub><i>k</i></sub> = 298, <i>n</i><sub><i>l</i></sub> = 116, 252, 164, 200 and 260. respectively). In all box plots: centre line, median; box limits, upper and lower quartiles; whiskers, 1.5 × interquartile range. All scale bars are 100 nm.</p><p><a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41565-023-01527-8#MOESM6">Source data</a></p></div></div></figure></div></div></div></section><section data-title="Direction reversal under a transmembrane voltage"><div id="Sec4-section"><h2 id="Sec4">Direction reversal under a transmembrane voltage</h2><div id="Sec4-content"><p>To our surprise, we found that we can control the rotational direction of the DNA turbines by the ionic strength of the buffer. When the voltage-driven experiments were performed in a high-salt buffer containing 3 M NaCl instead of dilute 50 mM, the directionality of the rotary motion reversed for the same turbine. This phenomenon occurred for both turbine chiralities, as shown in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41565-023-01527-8#Fig3">3d,e,j,k</a> (and corresponding mean square displacement, MSD, plots in Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41565-023-01527-8#MOESM1">19</a>). By titrating the salt concentration from 50 mM to 3 M, we observed that the average rotation speed changed from positive to negative values for the left-handed turbines, that is, rotations changed from anticlockwise to clockwise (and a reverse crossover occurred for right-handed turbines), with a crossover at ~0.5–1 M for both chiralities—see Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41565-023-01527-8#Fig3">3g,m</a> (and corresponding histograms for the speed distribution in Supplementary Figs. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41565-023-01527-8#MOESM1">21</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41565-023-01527-8#MOESM1">22</a>). These observations point to the appearance of strong ionic effects on the water flow when the turbine operates in a high-ionic-strength environment.</p><p>To understand this rotational reversal induced by different ionic strengths, we first sought insights from continuum theory. As a model for the turbine blade, we consider a rigid cylindrical DNA rod that is held in a fixed vertical position in a wide nanopore, oriented at an angle of <i>θ</i> with respect to the <i>z</i> axis (cf. Supplementary Section <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41565-023-01527-8#MOESM1">1</a>). This rod has a hydrodynamic mobility <b>M</b><sub>h</sub> that is anisotropic, as a rod moves twice as fast through a liquid along its length as perpendicular to it<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 23" title="Van den Heuvel, M., De Graaff, M., Lemay, S. &amp; Dekker, C. Electrophoresis of individual microtubules in microchannels. Proc. Natl Acad. Sci. USA 104, 7770–7775 (2007)." href="https://www.nature.com/articles/s41565-023-01527-8#ref-CR23" id="ref-link-section-d6783710e1100">23</a></sup>. Similarly, the electrophoretic mobility <b>M</b><sub>el</sub> of the rod, which describes its freely suspended motion in an applied electric field <b>E</b>, is anisotropic<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 23" title="Van den Heuvel, M., De Graaff, M., Lemay, S. &amp; Dekker, C. Electrophoresis of individual microtubules in microchannels. Proc. Natl Acad. Sci. USA 104, 7770–7775 (2007)." href="https://www.nature.com/articles/s41565-023-01527-8#ref-CR23" id="ref-link-section-d6783710e1112">23</a></sup>. Notably, <b>M</b><sub>el</sub> is controlled by the electrical double layer, which changes with the ionic strength of the solution<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 24" title="Van Der Drift, W., De Keizer, A. &amp; Overbeek, J. T. G. Electrophoretic mobility of a cylinder with high surface charge density. J. Colloid Interface Sci. 71, 67–78 (1979)." href="https://www.nature.com/articles/s41565-023-01527-8#ref-CR24" id="ref-link-section-d6783710e1120">24</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 25" title="Yariv, E. &amp; Schnitzer, O. The electrophoretic mobility of rod-like particles. J. Fluid Mech. 719, R3 (2013)." href="https://www.nature.com/articles/s41565-023-01527-8#ref-CR25" id="ref-link-section-d6783710e1123">25</a></sup>. The combination of <b>M</b><sub>h</sub> and <b>M</b><sub>el</sub> determines the in-plane force on a turbine blade in a nanopore, which will drive the rotation. We can derive an expression for the in-plane velocity component <i>v</i><sub><i>x</i></sub> (Supplementary Section <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41565-023-01527-8#MOESM1">2</a>) as follows:</p><div id="Equa"><p><span>$${v}_{x}=\frac{\frac{1}{2}{M}_{\mathrm{el},\parallel }{E}_{z}\sin 2\theta }{{\rm{co}}{{\rm{s}}}^{2}\theta +\left(\frac{{M}_{{\mathrm{h}},\perp }}{{M}_{{\mathrm{h}},\parallel }}\right){\rm{si}}{{\rm{n}}}^{2}\theta }\Bigg(\frac{{M}_{{\mathrm{h}},\perp }}{{M}_{{\mathrm{h}},\parallel }}-\frac{{M}_{{\mathrm{el}},\perp }}{{M}_{{\mathrm{el}},\parallel }}\Bigg).$$</span></p></div><p>This equation indicates that the sign of <i>v</i><sub><i>x</i></sub>, that is, the rotation direction, is determined by the difference between the hydrodynamic and electrophoretic anisotropy ratios. While the hydrodynamic anisotropy ratio <i>M</i><sub>h,<span>⟂</span></sub>/<i>M</i><sub>h,<span>∥</span></sub> is constant at a value of 0.5, the electrophoretic anisotropy ratio <i>M</i><sub>el,<span>⟂</span></sub>/<i>M</i><sub>el,<span>∥</span></sub> can adopt values between 0 and 1, depending on ion concentration and DNA surface charge<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 25" title="Yariv, E. &amp; Schnitzer, O. The electrophoretic mobility of rod-like particles. J. Fluid Mech. 719, R3 (2013)." href="https://www.nature.com/articles/s41565-023-01527-8#ref-CR25" id="ref-link-section-d6783710e1465">25</a></sup>; for example, <i>M</i><sub>el,<span>⟂</span></sub>/<i>M</i><sub>el,<span>∥</span></sub> decreases from 1 to 0.5 with decreasing ion concentration for moderate surface charges, and adopts even smaller values for high surface charges<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 25" title="Yariv, E. &amp; Schnitzer, O. The electrophoretic mobility of rod-like particles. J. Fluid Mech. 719, R3 (2013)." href="https://www.nature.com/articles/s41565-023-01527-8#ref-CR25" id="ref-link-section-d6783710e1478">25</a></sup>. Continuum theory thus indicates that a sign reversal of the rotations may occur due to a change in the electrophoretic anisotropy ratio with salt concentration.</p><p>To elucidate the microscopic mechanism of the torque generation, we performed all-atom MD simulations of the DNA origami turbine submerged in a low-salt electrolyte solution (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41565-023-01527-8#Fig4">4a</a>). The application of a 100 mV nm<sup>−1</sup> axial electric field was observed to rotate the turbine by ~120° in 56 ns in the expected direction (left handed about the applied field axis, Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41565-023-01527-8#Fig4">4b</a> and Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41565-023-01527-8#MOESM1">24</a>, see <a data-track="click" data-track-label="link" data-track-action="section anchor" href="https://www.nature.com/articles/s41565-023-01527-8#Sec6">Methods</a> for details). A water flow induced by a pressure gradient was observed to rotate the turbine in the expected direction, whereas reversing the direction of the electric field or the pressure gradient reversed the rotation direction. The turbine’s rotation speed was found to be determined by the velocity of water molecules moving past the blades of the turbine (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41565-023-01527-8#MOESM1">25b</a>)—all indicating a rotation mechanism similar to that of a macroscopic turbine. However, in equivalent simulations of the same DNA turbine carried out in 3 M NaCl, the direction of rotation reversed, while the overall rotation speed decreased (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41565-023-01527-8#Fig4">4d,e</a>). Furthermore, the average effective torque produced by the turbine under high salt was seen to change sign when compared with the torque under low salt (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41565-023-01527-8#Fig4">4f</a>). These MD simulation results thus present a striking qualitative resemblance to the experimental results.</p><div data-test="figure" data-container-section="figure" id="figure-4" data-title="All-atom MD simulation of a DNA turbine rotation."><figure><figcaption><b id="Fig4" data-test="figure-caption-text">Fig. 4: All-atom MD simulation of a DNA turbine rotation.</b></figcaption><div><div><a data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="https://www.nature.com/articles/s41565-023-01527-8/figures/4" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41565-023-01527-8/MediaObjects/41565_2023_1527_Fig4_HTML.png?as=webp"/><img aria-describedby="Fig4" src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41565-023-01527-8/MediaObjects/41565_2023_1527_Fig4_HTML.png" alt="figure 4" loading="lazy" width="685" height="810"/></picture></a></div><div data-test="bottom-caption" id="figure-4-desc"><p><b>a</b>, 4,322,088-atom model of a DNA origami turbine that is depicted using a molecular surface representation (white shaft, multicoloured blades), solvent shown as a semi-transparent surface, and ions (50 mM NaCl, 10 mM Mg<sup>2+</sup>) shown explicitly. <b>b</b>, Rotation of the turbine driven by electric field (50 mM NaCl). A 100 mV nm<sup>−1</sup> field was applied out of the page while restraints prevented the drift and tilting of the turbine. <b>c</b>, Rotation angle of the turbine due to an applied field or pressure gradient. <b>d</b>,<b>e</b>, The same as <b>b</b>,<b>c</b>, but for simulations at 3 M NaCl. A reversal of the rotation direction is observed. <b>f</b>, Torque on the turbine measured by restraining its spin angle. <b>g</b>, Single DNA helix as a minimal model for a turbine blade. A field or pressure gradient is applied along the +<i>z</i> direction. <b>h</b>, Ratio of the electrophoretic and hydrodynamic mobilities for motion perpendicular and parallel to the DNA helical axis observed in simulations of a single DNA helix. Ionic strength is calculated from the average molality observed at distances beyond 6 nm from the helical axis. <b>i</b>, Force on DNA orthogonal to the electric field measured from the single-helix simulations. <b>j</b>, In-plane solvent flow along the <i>x</i> axis in the simulations under an applied electric field. Heat maps depict the average along the helical axis of the DNA. The coordinate system is defined in <b>g</b>. <b>k</b>, Solvent forces extracted from MD simulation under an applied electric field. <b>l</b>, Force on DNA orthogonal to the applied force axis when, as a control, the solvent forces extracted from the low- and high-salt systems are applied to high- and low-salt systems, respectively. The forces were seen to reverse when compared with those in <b>i</b>, proving the causal role of the ion distribution.</p><p><a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41565-023-01527-8#MOESM7">Source data</a></p></div></div></figure></div><p>To understand the ion-concentration-dependent reversal of the effective torque, we simulated a DNA duplex that was oriented parallel or perpendicular (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41565-023-01527-8#MOESM1">26a</a>) to an applied electric field. The observed <i>M</i><sub>el,<span>⟂</span></sub>/<i>M</i><sub>el,<span>∥</span></sub> did indeed change with the measured salt concentration, from about 1 at 2.6 M NaCl to 0.38 at 4 mM NaCl (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41565-023-01527-8#Fig4">4h</a> and Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41565-023-01527-8#MOESM1">26b</a>)—a very notable change that would reverse the direction of rotation according to the continuum model. To determine the microscopic mechanism of the rotation reversal, we simulated a DNA duplex that was tilted by 35° relative to the vertically applied field (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41565-023-01527-8#Fig4">4g</a>), which approximates the inclination of the blades in our turbines. While the duplex’s centre of mass and the spinning angle were harmonically restrained, we measured the effective in-plane force <i>F</i><sub><i>x</i></sub> acting on the duplex (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41565-023-01527-8#Fig4">4i</a>). At low salt a negative force was measured, as expected, whereas at high salt the average in-plane force was reversed to a positive value. As the flow profile around the DNA (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41565-023-01527-8#Fig4">4j</a>) is driven by the distribution of charges in the solvent, we calculated a 3D map of the solvent forces—the position-dependent average force on solvent voxels applied by the electric field (<a data-track="click" data-track-label="link" data-track-action="section anchor" href="https://www.nature.com/articles/s41565-023-01527-8#Sec6">Methods</a>). The map revealed a slight overcharging of the DNA in the 3 M condition (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41565-023-01527-8#Fig4">4k</a>), which was nevertheless sufficient to substantially alter the flow of solvent near the DNA (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41565-023-01527-8#Fig4">4j</a>). To prove the causal role of the ion distribution, we applied the high-/low-salt solvent force on the low-/high-salt systems which, gratifyingly, swapped the effective force distributions (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41565-023-01527-8#Fig4">4l</a>).</p></div></div></section><section data-title="Conclusions"><div id="Sec5-section"><h2 id="Sec5">Conclusions</h2><div id="Sec5-content"><p>In summary, we have demonstrated a new type of autonomous active nanomachine, a DNA nanoturbine. We have shown its functionality from the observation of a sustained rotation of a DNA load bundle when the turbines were driven by an electrophoretic or hydrodynamic flow of solvent through a nanoscale pore. Building from our previous work<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 22" title="Shi, X. et al. Sustained unidirectional rotation of a self-organized DNA rotor on a nanopore. Nat. Phys. 18, 1105–1111 (2022)." href="https://www.nature.com/articles/s41565-023-01527-8#ref-CR22" id="ref-link-section-d6783710e1650">22</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 26" title="Pumm, A.-K. et al. A DNA origami rotary ratchet motor. Nature 607, 492–498 (2022)." href="https://www.nature.com/articles/s41565-023-01527-8#ref-CR26" id="ref-link-section-d6783710e1653">26</a></sup>, the direction of rotation is now controlled by the designed chirality of the turbine blades as well as by the ionic strength of the buffer. The latter revealed, strikingly, that ionic interactions can even reverse the rotation direction, as the electrophoretic mobility anisotropy changes with buffer salt concentration. Up to tens of piconewton nanometres of torque can be generated by these merely ~25-nm-tall turbines (an order of magnitude smaller than previously reported DNA origami rotors)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 22" title="Shi, X. et al. Sustained unidirectional rotation of a self-organized DNA rotor on a nanopore. Nat. Phys. 18, 1105–1111 (2022)." href="https://www.nature.com/articles/s41565-023-01527-8#ref-CR22" id="ref-link-section-d6783710e1657">22</a></sup>, numbers that are comparable to those of the F<sub>o</sub> motor of ATP synthase<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 17" title="Junge, W., Sielaff, H. &amp; Engelbrecht, S. Torque generation and elastic power transmission in the rotary FOF1-ATPase. Nature 459, 364–370 (2009)." href="https://www.nature.com/articles/s41565-023-01527-8#ref-CR17" id="ref-link-section-d6783710e1663">17</a></sup>. The DNA turbines could operate autonomously on physiologically and biologically compatible energy sources (electrochemical potentials) and work at a scale comparable to nature’s own motor proteins, without requiring manual cyclic intervention.</p><p>Our work demonstrates a practical approach to designing nanoscale active engines, that is, by using chiral nanoscale structures to leverage transmembrane potential differences through nanoscale hydrodynamic interactions<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 15" title="Maffeo, C., Quednau, L., Wilson, J. &amp; Aksimentiev, A. DNA double helix, a tiny electromotor. Nat. Nanotechnol. 18, 238–242 (2023)." href="https://www.nature.com/articles/s41565-023-01527-8#ref-CR15" id="ref-link-section-d6783710e1670">15</a></sup>. We believe this to be a powerful new approach to building active nanoscale systems, in particular because of the extensible design with DNA nanotechnology, its flexibility and potential for integration with various biocompatible membrane systems, and its potential compatibility with physiological environments. As in ATP synthase, microscopic reversibility may be exploited in future variants of such turbines to couple this mechanical rotation to uphill chemical synthesis. Nanoscale hydrodynamic turbines constitute biocompatible engines that may serve as a step towards building self-powered nanorobotic systems in environments relevant to molecular biology. Current limitations of our approach include the non-specific interaction between the turbine and the solid surface, the wide variance of the rotational speed and the finite power efficiency. Further work can be done to demonstrate nanorobotic systems based on the turbine concept, specifically the integration of nanoscale motors into biocompatible membranes, improving their efficiency in utilizing ion gradients and integrating them with other passive functional nanomachines to perform more complex tasks.</p></div></div></section><section data-title="Methods"><div id="Sec6-section"><h2 id="Sec6">Methods</h2><div id="Sec6-content"><h3 id="Sec7">Nanopore array fabrication</h3><p>Nanopore arrays are fabricated as reported before<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 27" title="Verschueren, D. V., Yang, W. &amp; Dekker, C. Lithography-based fabrication of nanopore arrays in freestanding SiN and graphene membranes. Nanotechnology 29, 145302 (2018)." href="https://www.nature.com/articles/s41565-023-01527-8#ref-CR27" id="ref-link-section-d6783710e1687">27</a></sup>. In brief, a 100-nm-thick layer of poly(methyl methacrylate) electron-sensitive resist (molecular weight 950,000, 3% dissolved in anisole, MicroChem Corp) was spin-coated on 20 nm free-standing silicon nitride membranes supported by silicon. Subsequently, the resist was exposed and patterned by an electron-beam pattern generator (EBPG5200, Raith) with 100 keV electron beams. The pattern is developed in a mixture of methyl-isobutyl-ketone and isopropanol with a ratio of 1:3 for 1 min, then stopped in isopropanol for 30 s. The exposed substrates were then etched using reactive-ion etching with fluoroform and argon (200 s, 50 W, 50 sccm of CHF<sub>3</sub>, 25 sccm of Ar, 10 μbar, SENTECH SI 200 plasma system). Finally, the resist was removed in oxygen plasma for 1 min (200 cm<sup>3</sup> min<sup>−1</sup> O<sub>2</sub>, 100 W, PVA TePla 300) followed by an acetone bath for 5 min.</p><h3 id="Sec8">Design, folding and purification of DNA origami structures</h3><p>All structures were designed using caDNAno v.0.2<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 28" title="Douglas, S. M. et al. Rapid prototyping of 3D DNA-origami shapes with caDNAno. Nucleic Acids Res. 37, 5001–5006 (2009)." href="https://www.nature.com/articles/s41565-023-01527-8#ref-CR28" id="ref-link-section-d6783710e1707">28</a></sup>. For the cryo-EM reconstruction of the turbine part, all structures were designed with a compact beam on top of each turbine structure (Supplementary Figs. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41565-023-01527-8#MOESM1">6</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41565-023-01527-8#MOESM1">7</a>) and designed only using a 7,560-base-long scaffold. The folding reaction mixtures contained a final scaffold concentration of 50 nM and oligonucleotide strands (IDT) of 500 nM. The folding reaction buffer contained 5 mM Tris, 1 mM EDTA, 5 mM NaCl and 20 mM MgCl<sub>2</sub>. The folding solutions were thermally annealed using TETRAD (MJ Research, now Bio-Rad) thermal cycling devices. The reactions were left at 65 °C for 15 min and then subsequently subjected to a thermal annealing ramp from 60 °C to 20 °C (1 °C h<sup>−1</sup>). The folded structures were purified from excess oligonucleotides by physical extraction from agarose gels and stored at room temperature until further usage. The list of oligonucleotides can be found in <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41565-023-01527-8#MOESM1">Supplementary Information</a>.</p><p>The turbine structure with a long DNA bundle as load was designed using a scaffold of 8,064 bases and a scaffold of 9,072 bases. The folding reaction mixtures contained a final scaffold concentration of 10 nM plus oligonucleotide strands (IDT) of 100 nM each. The folding reaction buffer contained 5 mM Tris, 1 mM EDTA, 5 mM NaCl and 15 mM MgCl<sub>2</sub> for the left-handed and right-handed versions or 20 mM MgCl<sub>2</sub> for the achiral version of the turbine. The folding reaction mixtures were thermally annealed using TETRAD (MJ Research) thermal cycling devices. The reactions were left at 65 °C for 15 min and then subjected to a thermal annealing ramp from 60 °C to 20 °C (1 °C h<sup>−1</sup>). The folded structures were purified from excess oligonucleotides by polyethylene glycol precipitation and stored at room temperature until further usage. Details of all the procedures can be found in ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 29" title="Wagenbauer, K. F. et al. How we make DNA origami. ChemBioChem 18, 1873–1885 (2017)." href="https://www.nature.com/articles/s41565-023-01527-8#ref-CR29" id="ref-link-section-d6783710e1734">29</a></sup>.</p><h3 id="Sec9">Cryo-EM sample preparation, image acquisition and processing</h3><p>Grid preparation, image acquisition and data processing were largely performed as reported previously<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 30" title="Kube, M. et al. Revealing the structures of megadalton-scale DNA complexes with nucleotide resolution. Nat. Commun. 11, 6229 (2020)." href="https://www.nature.com/articles/s41565-023-01527-8#ref-CR30" id="ref-link-section-d6783710e1746">30</a></sup>. The sample was applied to a glow-discharged C-Flat 1.2/1.3 4C thick grid (Protochips) and vitrified using a Vitrobot mark IV (FEI, now Thermo Scientific) at a temperature of 22 °C, a humidity of 100%, 0 s wait time, 2 s blot time, −1 blot force (arbitrary device units) and 0 s drain time. Micrograph videos with 10 frames were collected for the right-handed and left-handed versions (3,427 and 5,997 respectively) at a magnified pixel size of 2.28 Å and an accumulated dose of ~60 e Å<sup>−</sup><sup>2</sup> using the EPU software and a Falcon 3 detector (FEI) on a Cs-corrected (CEOS) 300 kV Titan Krios electron microscope (FEI). For the left-handed version, acquisition with a stage tilt of 20° was used to reduce the orientation bias of the particles.</p><p>Motion correction and contrast transfer function estimation of the micrographs were performed using the implementation in RELION 4.0 beta<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 31" title="Scheres, S. H. A Bayesian view on cryo-EM structure determination. J. Mol. Biol. 415, 406–418 (2012)." href="https://www.nature.com/articles/s41565-023-01527-8#ref-CR31" id="ref-link-section-d6783710e1756">31</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 32" title="Kimanius, D., Dong, L., Sharov, G., Nakane, T. &amp; Scheres, S. H. New tools for automated cryo-EM single-particle analysis in RELION-4.0. Biochem. J. 478, 4169–4185 (2021)." href="https://www.nature.com/articles/s41565-023-01527-8#ref-CR32" id="ref-link-section-d6783710e1759">32</a></sup> and CTFFIND4, respectively<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 33" title="Rohou, A. &amp; Grigorieff, N. CTFFIND4: fast and accurate defocus estimation from electron micrographs. J. Struct. Biol. 192, 216–221 (2015)." href="https://www.nature.com/articles/s41565-023-01527-8#ref-CR33" id="ref-link-section-d6783710e1763">33</a></sup>. Particles were autopicked using TOPAZ<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 34" title="Bepler, T. et al. Positive-unlabeled convolutional neural networks for particle picking in cryo-electron micrographs. Nat. Methods 16, 1153–1160 (2019)." href="https://www.nature.com/articles/s41565-023-01527-8#ref-CR34" id="ref-link-section-d6783710e1767">34</a></sup> and subjected to a selection process consisting of multiple rounds of 2D and 3D classification in RELION to remove falsely picked particles and damaged particles. Using an ab initio initial model, a refined 3D map was reconstructed from 97,054 and 71,992 particles for the right-handed and left-handed versions, respectively, followed by per-particle motion correction and dose weighting and 3D refinement (Supplementary Figs. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41565-023-01527-8#MOESM1">6</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41565-023-01527-8#MOESM1">7</a>). For a focused reconstruction of the turbine, multibody refinement<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 35" title="Nakane, T., Kimanius, D., Lindahl, E. &amp; Scheres, S. H. Characterisation of molecular motions in cryo-EM single-particle data by multi-body refinement in RELION. eLife 7, e36861 (2018)." href="https://www.nature.com/articles/s41565-023-01527-8#ref-CR35" id="ref-link-section-d6783710e1778">35</a></sup> was performed. The consensus map was divided into two parts containing the lever and the turbine using the eraser tool in UCSF Chimera<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 36" title="Pettersen, E. F. et al. UCSF Chimera—a visualization system for exploratory research and analysis. J. Comput. Chem. 25, 1605–1612 (2004)." href="https://www.nature.com/articles/s41565-023-01527-8#ref-CR36" id="ref-link-section-d6783710e1782">36</a></sup>, and low-pass-filtered soft masks of the respective regions were created in RELION (Supplementary Figs. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41565-023-01527-8#MOESM1">8</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41565-023-01527-8#MOESM1">9</a>). After multibody refinement, a set of particles with the subtracted signal of the lever arm was calculated and subjected to another round of 3D refinement. The final maps were masked, sharpened and low-pass filtered using the estimated resolution based on the 0.143 Fourier shell correlation criterion. Atomic models were constructed using a cascaded relaxation protocol as described previously<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 30" title="Kube, M. et al. Revealing the structures of megadalton-scale DNA complexes with nucleotide resolution. Nat. Commun. 11, 6229 (2020)." href="https://www.nature.com/articles/s41565-023-01527-8#ref-CR30" id="ref-link-section-d6783710e1792">30</a></sup> (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41565-023-01527-8#MOESM1">11</a>).</p><p>The dimensions of the turbines were measured in Fiji<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 37" title="Schindelin, J. et al. Fiji: an open-source platform for biological-image analysis. Nat. Methods 9, 676–682 (2012)." href="https://www.nature.com/articles/s41565-023-01527-8#ref-CR37" id="ref-link-section-d6783710e1802">37</a></sup> using orthographic projections of the maps created with ChimeraX<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 38" title="Pettersen, E. F. et al. UCSF ChimeraX: structure visualization for researchers, educators, and developers. Protein Sci. 30, 70–82 (2021)." href="https://www.nature.com/articles/s41565-023-01527-8#ref-CR38" id="ref-link-section-d6783710e1806">38</a></sup>. For the twist measurement of the turbine versions, slices from the well resolved central parts were extracted from the cryo-EM density maps using atomic model fits at base-pair positions that are on the same plane in the design, with a spacing of 33 bp and 34 bp for the right- and the left-handed version, respectively (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41565-023-01527-8#MOESM1">10</a>). For each version, the slices were fitted into each other on the basis of maximum overlay using ChimeraX<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 38" title="Pettersen, E. F. et al. UCSF ChimeraX: structure visualization for researchers, educators, and developers. Protein Sci. 30, 70–82 (2021)." href="https://www.nature.com/articles/s41565-023-01527-8#ref-CR38" id="ref-link-section-d6783710e1813">38</a></sup> to determine the rotation angle. From the twist density, the diameter and the length of the helices, the outer blade angle with respect to the helical axes was calculated.</p><h3 id="Sec10">Single-particle fluorescence imaging</h3><p>Solid-state nanopore chips were oxygen-plasma cleaned before all the fluorescence experiments (100 W for 1 min, Plasma Prep III, SPI Supplies). Coverslips (VWR, no. 1.5) were cleaned by ultrasonication sequentially in acetone, isopropanol, water, 1 M KOH solution and deionized water (Milli-Q) for 30 min each. The cleaned coverslips were then blow-dried thoroughly with compressed nitrogen. The nanopore chip was glued into the PDMS (SYLGARD 184 silicone elastomer) flow cell using a two-component silicone rubber (Ecoflex 5, Smooth-ON), then the PDMS flow cell was bonded to the cleaned coverslip after oxygen-plasma treatment (50 W, 50 mbar for 30 s) and post-bake at 120 °C for 30 min. After assembly, the whole device was again treated with oxygen plasma (50 W, 50 mbar) for 4 min before embedding a pair of Ag/AgCl electrodes, one in each side of the reservoir, and flushing in deionized water to wet the channels. This is essential for increasing the hydrophilicity of the membrane and ensuring a negatively charged silicon nitride surface. The PDMS nanopore devices were always assembled shortly before each experiment and never reused.</p><p>The nanopore chip was then imaged using an epifluorescence microscope with a ×60 water immersion objective (Olympus UPlanSApo, numerical aperture 1.20) and a fast scientific complementary metal–oxide–semiconductor camera (Prime BSI, Teledmy Photometrics). The camera field of view was reduced as needed to achieve high frame rates (typically around 200 pixels × 200 pixels). To image Cy3-labelled DNA turbines, a 561 nm laser (Stradus, Vortran Laser Technology) was used to excite the fluorophores. The typical exposure time of the experiments was 5 ms, which led to a frame rate of around 190–200 fps. To simplify the data analysis, a fixed frame rate value (200 fps) is used. Before imaging, the imaging buffer (50 mM Tris-HCl pH 7.5; 50 mM NaCl unless otherwise stated, 5 mM MgCl<sub>2</sub>; 1 mM dithiothreitol, 5% (w/v) <span>d</span>-dextrose, 2 mM Trolox, 40 μg ml<sup>−1</sup> glucose oxidase, 17 μg ml<sup>−1</sup> catalase; 0.05% TWEEN 20) was placed into the reservoirs on either side of the silicon nitride membrane.</p><h3 id="Sec11">Driving DNA turbines using transmembrane ion gradients</h3><p>An imaging buffer with the same salt concentration (50 mM NaCl) was flushed into the flow cell first, with DNA turbines on the <i>cis</i> side of the membrane. Subsequently, an imaging buffer containing a higher NaCl concentration was flushed into the <i>trans</i> side of the membrane. With single-particle fluorescence microscopy, the docking and rotation of the DNA turbines could be observed and recorded. To release the turbines from the nanopore, we either inserted a pair of temporary electrodes into the inlet and outlet of the flow channels and released the turbines electrically, or we flushed in the same (lower-concentration) buffer as the <i>cis</i> side. Because of the photobleaching and accumulation of the DNA turbines near nanopore arrays, we chose 40 s as a typical observation duration. Examples of longer recordings are shown in Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41565-023-01527-8#MOESM1">23</a>.</p><h3 id="Sec12">Driving DNA turbines using transmembrane voltages</h3><p>In contrast to the salt-gradient-driven mode, a pair of electrodes was embedded into the flow cells. We used a custom-built circuit to apply voltages<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 39" title="Pud, S. et al. Self-aligned plasmonic nanopores by optically controlled dielectric breakdown. Nano Lett. 15, 7112–7117 (2015)." href="https://www.nature.com/articles/s41565-023-01527-8#ref-CR39" id="ref-link-section-d6783710e1866">39</a></sup>. The output voltage was controlled by a custom LabVIEW program. The electrodes embedded in the two reservoirs were connected to the circuit. The DNA origami turbines were placed into the electrically grounded side (<i>cis</i> side) of the flow cell with a typical concentration of 1 pM. After applying the voltage, the DNA turbines were docked onto the nanopores under a 100 mV bias voltage (unless otherwise stated) across the membrane. The turbines could be easily released from the nanopore array by flipping the voltage polarity and then setting it to 0 mV for several seconds to allow the imaged turbines to diffuse away from the capture region. To avoid overcrowding of DNA turbines near the nanopore array, which increased the fluorescence background fluctuation, we typically imaged the turbines before the array was fully filled, and subsequently released them from the nanopore. Then a new group of turbines could be captured and docked again by applying a positive bias.</p><h3 id="Sec13">Fluorescence microscopy data analysis</h3><p>For image processing, first a single-molecule localization was carried out using Fiji (ImageJ<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 37" title="Schindelin, J. et al. Fiji: an open-source platform for biological-image analysis. Nat. Methods 9, 676–682 (2012)." href="https://www.nature.com/articles/s41565-023-01527-8#ref-CR37" id="ref-link-section-d6783710e1881">37</a></sup>) with the ThunderSTORM plugin<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 40" title="Ovesný, M., Křížek, P., Borkovec, J., Švindrych, Z. &amp; Hagen, G. M. ThunderSTORM: a comprehensive ImageJ plug-in for PALM and STORM data analysis and super-resolution imaging. Bioinformatics 30, 2389–2390 (2014)." href="https://www.nature.com/articles/s41565-023-01527-8#ref-CR40" id="ref-link-section-d6783710e1885">40</a></sup> for all frames in the acquired image sequences. A wavelet filter (B-spline) and an integrated Gaussian method were used for the localizations. Then the results were filtered on the basis of their quality (uncertainty &lt; 50 nm) and the local density (filter of 15 particles in every 50 nm among all localized data points in the sequence) to rule out free-diffusing (non-captured) turbines. Next, the single-molecule localization results were analysed using a custom MATLAB script (Code availability). In brief, all coordinates of localized particle positions were clustered on the basis of their Euclidean distance for each turbine. When localized particle positions were deduced in a video, a circle was fitted to the data to obtain the centre and a radius, which subsequently was used for calculating the angular position of the fluorophores in each frame. Next, we determined if the fluorophores occupy spatial states that can be fitted to a circular path. We did this by comparing the point density of the coordinates within an annulus around the fitted circular perimeter (±1 nm) with the density of points around the centre (with a radius <i>r</i>, so that the area of this central circle is equal to that of the ring region). If the point density within the annulus was higher, then this data group would be kept in the statistics, else it would be considered an invalid trajectory and discarded. Finally, the script calculated all necessary motion properties of the turbine, including its cumulative angular displacements, MSD, angular velocity and torque. The angular velocity <i>ω</i><sub>d</sub> was determined by fitting MSD = <i>ω</i><sub>d</sub><sup>2</sup><i>t</i><sup>2</sup> + 2<i>D</i><sub>r</sub><i>t</i> to the MSD curve of each turbine, where <i>t</i> is the lag time and <i>D</i><sub>r</sub> is the rotational diffusion coefficient (also as a fitting parameter). The estimation of the torque is discussed in Supplementary Section <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41565-023-01527-8#MOESM1">1</a>.</p><h3 id="Sec14">MD simulations</h3><p>All MD simulations were performed using the NAMD program<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 41" title="Phillips, J. C. et al. Scalable molecular dynamics on CPU and GPU architectures with NAMD. J. Chem. Phys. 153, 044130 (2020)." href="https://www.nature.com/articles/s41565-023-01527-8#ref-CR41" id="ref-link-section-d6783710e1930">41</a></sup>, CHARMM36 parameters for DNA, water and ions<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 42" title="Hart, K. et al. Optimization of the CHARMM additive force field for DNA: improved treatment of the BI/BII conformational equilibrium. J. Chem. Theory Comput. 8, 348–362 (2012)." href="https://www.nature.com/articles/s41565-023-01527-8#ref-CR42" id="ref-link-section-d6783710e1934">42</a></sup> with CUFIX<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 43" title="Yoo, J. &amp; Aksimentiev, A. Improved parametrization of Li+, Na+, K+, and Mg2+ ions for all-atom molecular dynamics simulations of nucleic acid systems. J. Phys. Chem. Lett. 3, 45–50 (2012)." href="https://www.nature.com/articles/s41565-023-01527-8#ref-CR43" id="ref-link-section-d6783710e1938">43</a></sup> corrections, periodic boundary conditions and the TIP3P model of water<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 44" title="Jorgensen, W. L., Chandrasekhar, J., Madura, J. D., Impey, R. W. &amp; Klein, M. L. Comparison of simple potential functions for simulating liquid water. J. Chem. Phys. 79, 926–935 (1983)." href="https://www.nature.com/articles/s41565-023-01527-8#ref-CR44" id="ref-link-section-d6783710e1942">44</a></sup>. The long-range electrostatic interactions were computed using the particle-mesh Ewald scheme over a grid with 1 Å spacing<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 45" title="Darden, T., York, D. &amp; Pedersen, L. Particle mesh Ewald: an N⋅log(N) method for Ewald sums in large systems. J. Chem. Phys. 98, 10089–10092 (1993)." href="https://www.nature.com/articles/s41565-023-01527-8#ref-CR45" id="ref-link-section-d6783710e1946">45</a></sup>. Van der Waals and short-range electrostatic forces were evaluated using the 10–12 Å smooth cutoff scheme. Hydrogen mass repartitioning<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 46" title="Balusek, C. et al. Accelerating membrane simulations with hydrogen mass repartitioning. J. Chem. Theory Comput. 15, 4673–4686 (2019)." href="https://www.nature.com/articles/s41565-023-01527-8#ref-CR46" id="ref-link-section-d6783710e1951">46</a></sup> and the SHAKE<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 47" title="Ryckaert, J.-P., Ciccotti, G. &amp; Berendsen, H. J. Numerical integration of the Cartesian equations of motion of a system with constraints: molecular dynamics of n-alkanes. J. Comput. Phys. 23, 327–341 (1977)." href="https://www.nature.com/articles/s41565-023-01527-8#ref-CR47" id="ref-link-section-d6783710e1955">47</a></sup> and SETTLE<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 48" title="Miyamoto, S. &amp; Kollman, P. A. SETTLE: an analytical version of the SHAKE and RATTLE algorithm for rigid water models. J. Comput. Chem. 13, 952–962 (1992)." href="https://www.nature.com/articles/s41565-023-01527-8#ref-CR48" id="ref-link-section-d6783710e1959">48</a></sup> algorithms were used, enabling a 4 fs integration time step. The full electrostatics were calculated every two-time step. Except where specified, a Langevin thermostat with a 0.1 ps<sup>−1</sup> damping coefficient maintained a temperature of 295 K in all simulations. Coordinates were recorded every 2,500 steps.</p><p>Atomistic models of the entire turbine were assembled from the caDNAno<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 28" title="Douglas, S. M. et al. Rapid prototyping of 3D DNA-origami shapes with caDNAno. Nucleic Acids Res. 37, 5001–5006 (2009)." href="https://www.nature.com/articles/s41565-023-01527-8#ref-CR28" id="ref-link-section-d6783710e1968">28</a></sup> design file using a custom mrdna script<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 49" title="Maffeo, C. &amp; Aksimentiev, A. MrDNA: a multi-resolution model for predicting the structure and dynamics of DNA systems. Nucleic Acids Res. 48, 5135–5146 (2020)." href="https://www.nature.com/articles/s41565-023-01527-8#ref-CR49" id="ref-link-section-d6783710e1972">49</a></sup>. In addition to neutralizing Mg<sup>2+</sup>, 10 mM Mg<sup>2+</sup> hexahydrate was placed adjacent to the DNA according to a previously described protocol<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 43" title="Yoo, J. &amp; Aksimentiev, A. Improved parametrization of Li+, Na+, K+, and Mg2+ ions for all-atom molecular dynamics simulations of nucleic acid systems. J. Phys. Chem. Lett. 3, 45–50 (2012)." href="https://www.nature.com/articles/s41565-023-01527-8#ref-CR43" id="ref-link-section-d6783710e1980">43</a></sup>. Water and monovalent ions were added to the system using the solvate and autoionize plugins for VMD<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 50" title="Humphrey, W., Dalke, A. &amp; Schulten, K. VMD: visual molecular dynamics. J. Mol. Graph. 14, 33–38 (1996)." href="https://www.nature.com/articles/s41565-023-01527-8#ref-CR50" id="ref-link-section-d6783710e1985">50</a></sup>, with the solvent box cut to form a hexagonal prism. For each salt condition, a 4 ns simulation was performed with a Nosé–Hoover Langevin piston barostat<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 51" title="Martyna, G. J., Tobias, D. J. &amp; Klein, M. L. Constant pressure molecular dynamics algorithms. J. Chem. Phys. 101, 4177–4189 (1994)." href="https://www.nature.com/articles/s41565-023-01527-8#ref-CR51" id="ref-link-section-d6783710e1989">51</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 52" title="Feller, S. E., Zhang, Y., Pastor, R. W. &amp; Brooks, B. R. Constant pressure molecular dynamics simulation: the Langevin piston method. J. Chem. Phys. 103, 4613–4621 (1995)." href="https://www.nature.com/articles/s41565-023-01527-8#ref-CR52" id="ref-link-section-d6783710e1992">52</a></sup> set to maintain a target pressure of 1 bar, allowing the equilibrium volume of the system to be determined. The resulting system dimensions were used in constant-volume simulations to equilibrate the turbine with harmonic position restraints holding the phosphorus atoms to their initial coordinates during the first 7 ns of the simulation (<i>k</i><sub>spring</sub> = 1 kcal mol<sup>–1</sup> Å<sup>–2</sup> for <i>t</i> &lt; 5 ns; 0.1 for 5 ns &lt; <i>t</i> &lt; 7 ns). After 30 ns of equilibration for the 50 mM and 75 ns for the 3 M system, a snapshot of the configuration was used to initialize subsequent simulations with either an electric field or pressure gradient applied to drive the turbine. Additional equilibration was performed for the 50 mM NaCl system for another 128 ns to initialize the ‘Alternate conf.’ system (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41565-023-01527-8#MOESM1">24</a>).</p><p>The conformation of the turbine at the end of equilibration was used to determine the rest positions of several harmonic collective variable (colvar)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 53" title="Fiorin, G., Klein, M. L. &amp; Hénin, J. Using collective variables to drive molecular dynamics simulations. Mol. Phys. 111, 3345–3362 (2013)." href="https://www.nature.com/articles/s41565-023-01527-8#ref-CR53" id="ref-link-section-d6783710e2017">53</a></sup> potentials, including a spring restraining the centre of mass of every third phosphorus atom (<i>k</i><sub>spring</sub> = 500 kcal mol<sup>–1</sup> Å<sup>–2</sup>); a spring restraining the root-mean-square deviation (RMSD) of these phosphorus atoms with respect to the post-equilibration configuration (<i>k</i><sub>spring</sub> = 1,000 kcal mol<sup>–1</sup> Å<sup>–2</sup>; resting RMSD = 0), after optimal rigid body transformations so that the potential does not apply a net torque or force; and a pair of centre of mass harmonic restraints applied to 16-bp-long sections of the central six-helix bundle (<i>k</i><sub>spring</sub> = 50 kcal mol<sup>–1</sup> Å<sup>–2</sup>), placed near either the end of the shaft to prevent the turbine from tilting. With these colvars preventing translation, conformational fluctuations, or tilting of the turbine, an electric field was applied by placing a constant force on each atom with a magnitude proportional to the charge of the atom. Similarly, a pressure gradient was achieved by placing a small force on every water molecule of the system. Finally, in simulations where the torque was measured, an additional spin angle colvar (<i>k</i><sub>spring</sub> = 100 kcal mol<sup>–1</sup> °<sup>–2</sup>) prevented rotation of the turbine and reported the torque.</p><p>Simulation systems were prepared to study the forces on and flows around a DNA helix mimicking the DNA in the turbine blade. The 21 bp helix was made effectively infinite by connecting the ends of each strand across the periodic boundary. Solvent (neutralizing Na<sup>+</sup>, 100 mM and 3 M NaCl; no Mg<sup>2+</sup>) was added around the helix. Systems were equilibrated for 5–50 ns with the DNA phosphorus atoms harmonically restrained (<i>k</i><sub>spring</sub> = 0.2 kcal mol<sup>–1</sup> Å<sup>–2</sup>). Mobility measurements were performed using a field of 5 mV nm<sup>−1</sup> or a hydrostatic pressure of ~1.3 bar nm<sup>−1</sup> parallel or transverse to the helical axis, with each condition employing four replicate simulations lasting a total of 400 (neutralizing Na<sup>+</sup>) to 4,000 ns (3 M). Except where otherwise specified, a 100 mV nm<sup>−1</sup> electric field was applied to the system at a 35° angle with respect to the DNA while a centre of mass colvar restrained the DNA (<i>k</i><sub>spring</sub> = 500 kcal mol<sup>–1</sup> Å<sup>–2</sup>), an RMSD colvar retained an idealized DNA configuration (<i>k</i><sub>spring</sub> = 100 kcal mol<sup>–1</sup> Å<sup>–2</sup>) and a spin angle colvar (<i>k</i><sub>spring</sub> = 100 kcal mol<sup>–1</sup> °<sup>–2</sup>) prevented rotation of the DNA and reported on the torque. Eight replicate systems were employed during simulations lasting a total of 1,040 ns (100 mM) or 2,055 ns (3 M). The flow and concentration of ions and water oxygen atoms were analysed by binning the system into ~1 Å voxels, counting the flux through and concentration in each voxel using a centred finite-difference approximation for the flux. The difference in concentration between sodium and chloride ions provided the net local charge density of the fluid around the DNA in each case. Multiplying this charge by the electric field provided the solvent force. In subsequent simulations, the 3D map of the solvent force was used to apply a position-dependent force to each water oxygen atom using the TclBC feature of NAMD and employing the approximation that the density of water oxygen atoms is uniformly 33 nm<sup>−</sup><sup>3</sup>. Again, eight replicate systems were employed for simulations lasting a total of 480 and 570 ns for 100 mM and 3 M conditions, respectively.</p><h3 id="Sec15">Statistics and reproducibility</h3><p>No statistical method was used to predetermine the sample size.</p></div></div></section>
                </div><div>
            <section data-title="Data availability"><div id="data-availability-section"><h2 id="data-availability">Data availability</h2><p>The electron density maps of the left- and right-handed turbines are available in the Electron Microscopy Data Bank (EMDB) as entries EMD-<a href="http://www.ebi.ac.uk/pdbe/entry/emdb/EMD-17600">17600</a> and EMD-<a href="http://www.ebi.ac.uk/pdbe/entry/emdb/EMD-17606">17606</a>, respectively. Fluorescence microscopy and nanopore experimental data are available at <a href="https://doi.org/10.5281/zenodo.8091178">https://doi.org/10.5281/zenodo.8091178</a>. Simulation trajectory data are available at <a href="https://doi.org/10.13012/B2IDB-3458097_V1">https://doi.org/10.13012/B2IDB-3458097_V1</a>. <a data-track="click" data-track-label="link" data-track-action="section anchor" href="https://www.nature.com/articles/s41565-023-01527-8#Sec18">Source data</a> are provided with this paper.</p></div></section><section data-title="Code availability"><div id="code-availability-section"><h2 id="code-availability">Code availability</h2><div id="code-availability-content">
            
            <p>The corresponding MATLAB scripts for data processing and producing the final figures are available at <a href="https://doi.org/10.5281/zenodo.8091178">https://doi.org/10.5281/zenodo.8091178</a>. Scripts related to simulation set-up and analysis are available at <a href="https://doi.org/10.13012/B2IDB-3458097_V1">https://doi.org/10.13012/B2IDB-3458097_V1</a>.</p>
          </div></div></section><div id="MagazineFulltextArticleBodySuffix"><section aria-labelledby="Bib1" data-title="References"><div id="Bib1-section"><h2 id="Bib1">References</h2><div id="Bib1-content"><div data-container-section="references"><ol data-track-component="outbound reference"><li data-counter="1."><p id="ref-CR1">Carter, N. J. &amp; Cross, R. Mechanics of the kinesin step. <i>Nature</i> <b>435</b>, 308–312 (2005).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2MXkt1Wkt78%3D" aria-label="CAS reference 1">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 1" href="http://scholar.google.com/scholar_lookup?&amp;title=Mechanics%20of%20the%20kinesin%20step&amp;journal=Nature&amp;volume=435&amp;pages=308-312&amp;publication_year=2005&amp;author=Carter%2CNJ&amp;author=Cross%2CR">
                    Google Scholar</a> 
                </p></li><li data-counter="2."><p id="ref-CR2">Berg, H. C. The rotary motor of bacterial flagella. <i>Annu. Rev. Biochem.</i> <b>72</b>, 19–54 (2003).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3sXntFSgt7Y%3D" aria-label="CAS reference 2">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 2" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20rotary%20motor%20of%20bacterial%20flagella&amp;journal=Annu.%20Rev.%20Biochem.&amp;volume=72&amp;pages=19-54&amp;publication_year=2003&amp;author=Berg%2CHC">
                    Google Scholar</a> 
                </p></li><li data-counter="3."><p id="ref-CR3">Hahn, A., Vonck, J., Mills, D. J., Meier, T. &amp; Kühlbrandt, W. Structure, mechanism, and regulation of the chloroplast ATP synthase. <i>Science</i> <b>360</b>, eaat4318 (2018).</p><p><a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 3" href="http://scholar.google.com/scholar_lookup?&amp;title=Structure%2C%20mechanism%2C%20and%20regulation%20of%20the%20chloroplast%20ATP%20synthase&amp;journal=Science&amp;volume=360&amp;publication_year=2018&amp;author=Hahn%2CA&amp;author=Vonck%2CJ&amp;author=Mills%2CDJ&amp;author=Meier%2CT&amp;author=K%C3%BChlbrandt%2CW">
                    Google Scholar</a> 
                </p></li><li data-counter="4."><p id="ref-CR4">Srivastava, A. P. et al. High-resolution cryo-EM analysis of the yeast ATP synthase in a lipid membrane. <i>Science</i> <b>360</b>, eaas9699 (2018).</p><p><a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 4" href="http://scholar.google.com/scholar_lookup?&amp;title=High-resolution%20cryo-EM%20analysis%20of%20the%20yeast%20ATP%20synthase%20in%20a%20lipid%20membrane&amp;journal=Science&amp;volume=360&amp;publication_year=2018&amp;author=Srivastava%2CAP">
                    Google Scholar</a> 
                </p></li><li data-counter="5."><p id="ref-CR5">Ketterer, P., Willner, E. M. &amp; Dietz, H. Nanoscale rotary apparatus formed from tight-fitting 3D DNA components. <i>Sci. Adv.</i> <b>2</b>, e1501209 (2016).</p><p><a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 5" href="http://scholar.google.com/scholar_lookup?&amp;title=Nanoscale%20rotary%20apparatus%20formed%20from%20tight-fitting%203D%20DNA%20components&amp;journal=Sci.%20Adv.&amp;volume=2&amp;publication_year=2016&amp;author=Ketterer%2CP&amp;author=Willner%2CEM&amp;author=Dietz%2CH">
                    Google Scholar</a> 
                </p></li><li data-counter="6."><p id="ref-CR6">Bertosin, E. et al. A nanoscale reciprocating rotary mechanism with coordinated mobility control. <i>Nat. Commun.</i> <b>12</b>, 7138 (2021).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXislWktrvK" aria-label="CAS reference 6">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 6" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20nanoscale%20reciprocating%20rotary%20mechanism%20with%20coordinated%20mobility%20control&amp;journal=Nat.%20Commun.&amp;volume=12&amp;publication_year=2021&amp;author=Bertosin%2CE">
                    Google Scholar</a> 
                </p></li><li data-counter="7."><p id="ref-CR7">Courbet, A. et al. Computational design of mechanically coupled axle–rotor protein assemblies. <i>Science</i> <b>376</b>, 383–390 (2022).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XhtFynsL7I" aria-label="CAS reference 7">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 7" href="http://scholar.google.com/scholar_lookup?&amp;title=Computational%20design%20of%20mechanically%20coupled%20axle%E2%80%93rotor%20protein%20assemblies&amp;journal=Science&amp;volume=376&amp;pages=383-390&amp;publication_year=2022&amp;author=Courbet%2CA">
                    Google Scholar</a> 
                </p></li><li data-counter="8."><p id="ref-CR8">Ahmadi, Y. et al. The Brownian and flow‐driven rotational dynamics of a multicomponent DNA origami‐based rotor. <i>Small</i> <b>16</b>, 2001855 (2020).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXos1Sjtr0%3D" aria-label="CAS reference 8">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 8" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20Brownian%20and%20flow%E2%80%90driven%20rotational%20dynamics%20of%20a%20multicomponent%20DNA%20origami%E2%80%90based%20rotor&amp;journal=Small&amp;volume=16&amp;publication_year=2020&amp;author=Ahmadi%2CY">
                    Google Scholar</a> 
                </p></li><li data-counter="9."><p id="ref-CR9">Erbas-Cakmak, S., Leigh, D. A., McTernan, C. T. &amp; Nussbaumer, A. L. Artificial molecular machines. <i>Chem. Rev.</i> <b>115</b>, 10081–10206 (2015).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXhsVKitL7E" aria-label="CAS reference 9">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 9" href="http://scholar.google.com/scholar_lookup?&amp;title=Artificial%20molecular%20machines&amp;journal=Chem.%20Rev.&amp;volume=115&amp;pages=10081-10206&amp;publication_year=2015&amp;author=Erbas-Cakmak%2CS&amp;author=Leigh%2CDA&amp;author=McTernan%2CCT&amp;author=Nussbaumer%2CAL">
                    Google Scholar</a> 
                </p></li><li data-counter="10."><p id="ref-CR10">Van Delden, R. A. et al. Unidirectional molecular motor on a gold surface. <i>Nature</i> <b>437</b>, 1337–1340 (2005).</p><p><a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 10" href="http://scholar.google.com/scholar_lookup?&amp;title=Unidirectional%20molecular%20motor%20on%20a%20gold%20surface&amp;journal=Nature&amp;volume=437&amp;pages=1337-1340&amp;publication_year=2005&amp;author=Delden%2CRA">
                    Google Scholar</a> 
                </p></li><li data-counter="11."><p id="ref-CR11">Roke, D., Wezenberg, S. J. &amp; Feringa, B. L. Molecular rotary motors: unidirectional motion around double bonds. <i>Proc. Natl Acad. Sci. USA</i> <b>115</b>, 9423–9431 (2018).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXitVKhurbJ" aria-label="CAS reference 11">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 11" href="http://scholar.google.com/scholar_lookup?&amp;title=Molecular%20rotary%20motors%3A%20unidirectional%20motion%20around%20double%20bonds&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=115&amp;pages=9423-9431&amp;publication_year=2018&amp;author=Roke%2CD&amp;author=Wezenberg%2CSJ&amp;author=Feringa%2CBL">
                    Google Scholar</a> 
                </p></li><li data-counter="12."><p id="ref-CR12">Qiu, Y., Feng, Y., Guo, Q.-H., Astumian, R. D. &amp; Stoddart, J. F. Pumps through the ages. <i>Chem</i> <b>6</b>, 1952–1977 (2020).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhsFKgurjO" aria-label="CAS reference 12">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 12" href="http://scholar.google.com/scholar_lookup?&amp;title=Pumps%20through%20the%20ages&amp;journal=Chem&amp;volume=6&amp;pages=1952-1977&amp;publication_year=2020&amp;author=Qiu%2CY&amp;author=Feng%2CY&amp;author=Guo%2CQ-H&amp;author=Astumian%2CRD&amp;author=Stoddart%2CJF">
                    Google Scholar</a> 
                </p></li><li data-counter="13."><p id="ref-CR13">Borsley, S., Kreidt, E., Leigh, D. A. &amp; Roberts, B. M. Autonomous fuelled directional rotation about a covalent single bond. <i>Nature</i> <b>604</b>, 80–85 (2022).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XptF2hu7w%3D" aria-label="CAS reference 13">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 13" href="http://scholar.google.com/scholar_lookup?&amp;title=Autonomous%20fuelled%20directional%20rotation%20about%20a%20covalent%20single%20bond&amp;journal=Nature&amp;volume=604&amp;pages=80-85&amp;publication_year=2022&amp;author=Borsley%2CS&amp;author=Kreidt%2CE&amp;author=Leigh%2CDA&amp;author=Roberts%2CBM">
                    Google Scholar</a> 
                </p></li><li data-counter="14."><p id="ref-CR14">Kopperger, E. et al. A self-assembled nanoscale robotic arm controlled by electric fields. <i>Science</i> <b>359</b>, 296–301 (2018).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtFKntrk%3D" aria-label="CAS reference 14">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 14" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20self-assembled%20nanoscale%20robotic%20arm%20controlled%20by%20electric%20fields&amp;journal=Science&amp;volume=359&amp;pages=296-301&amp;publication_year=2018&amp;author=Kopperger%2CE">
                    Google Scholar</a> 
                </p></li><li data-counter="15."><p id="ref-CR15">Maffeo, C., Quednau, L., Wilson, J. &amp; Aksimentiev, A. DNA double helix, a tiny electromotor. <i>Nat. Nanotechnol.</i> <b>18</b>, 238–242 (2023).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XjtFymurvE" aria-label="CAS reference 15">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 15" href="http://scholar.google.com/scholar_lookup?&amp;title=DNA%20double%20helix%2C%20a%20tiny%20electromotor&amp;journal=Nat.%20Nanotechnol.&amp;volume=18&amp;pages=238-242&amp;publication_year=2023&amp;author=Maffeo%2CC&amp;author=Quednau%2CL&amp;author=Wilson%2CJ&amp;author=Aksimentiev%2CA">
                    Google Scholar</a> 
                </p></li><li data-counter="16."><p id="ref-CR16">Klusch, N., Murphy, B. J., Mills, D. J., Yildiz, Ö. &amp; Kühlbrandt, W. Structural basis of proton translocation and force generation in mitochondrial ATP synthase. <i>eLife</i> <b>6</b>, e33274 (2017).</p><p><a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 16" href="http://scholar.google.com/scholar_lookup?&amp;title=Structural%20basis%20of%20proton%20translocation%20and%20force%20generation%20in%20mitochondrial%20ATP%20synthase&amp;journal=eLife&amp;volume=6&amp;publication_year=2017&amp;author=Klusch%2CN&amp;author=Murphy%2CBJ&amp;author=Mills%2CDJ&amp;author=Yildiz%2C%C3%96&amp;author=K%C3%BChlbrandt%2CW">
                    Google Scholar</a> 
                </p></li><li data-counter="17."><p id="ref-CR17">Junge, W., Sielaff, H. &amp; Engelbrecht, S. Torque generation and elastic power transmission in the rotary FOF1-ATPase. <i>Nature</i> <b>459</b>, 364–370 (2009).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXmtFKmt7k%3D" aria-label="CAS reference 17">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 17" href="http://scholar.google.com/scholar_lookup?&amp;title=Torque%20generation%20and%20elastic%20power%20transmission%20in%20the%20rotary%20FOF1-ATPase&amp;journal=Nature&amp;volume=459&amp;pages=364-370&amp;publication_year=2009&amp;author=Junge%2CW&amp;author=Sielaff%2CH&amp;author=Engelbrecht%2CS">
                    Google Scholar</a> 
                </p></li><li data-counter="18."><p id="ref-CR18">Dietz, H., Douglas, S. M. &amp; Shih, W. M. Folding DNA into twisted and curved nanoscale shapes. <i>Science</i> <b>325</b>, 725–730 (2009).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXptl2guro%3D" aria-label="CAS reference 18">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 18" href="http://scholar.google.com/scholar_lookup?&amp;title=Folding%20DNA%20into%20twisted%20and%20curved%20nanoscale%20shapes&amp;journal=Science&amp;volume=325&amp;pages=725-730&amp;publication_year=2009&amp;author=Dietz%2CH&amp;author=Douglas%2CSM&amp;author=Shih%2CWM">
                    Google Scholar</a> 
                </p></li><li data-counter="19."><p id="ref-CR19">Castro, C. E. et al. A primer to scaffolded DNA origami. <i>Nat. Methods</i> <b>8</b>, 221–229 (2011).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXisFentb0%3D" aria-label="CAS reference 19">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 19" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20primer%20to%20scaffolded%20DNA%20origami&amp;journal=Nat.%20Methods&amp;volume=8&amp;pages=221-229&amp;publication_year=2011&amp;author=Castro%2CCE">
                    Google Scholar</a> 
                </p></li><li data-counter="20."><p id="ref-CR20">Lee, J. Y. et al. Rapid computational analysis of DNA origami assemblies at near-atomic resolution. <i>ACS Nano</i> <b>15</b>, 1002–1015 (2021).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXlt1Cqsw%3D%3D" aria-label="CAS reference 20">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 20" href="http://scholar.google.com/scholar_lookup?&amp;title=Rapid%20computational%20analysis%20of%20DNA%20origami%20assemblies%20at%20near-atomic%20resolution&amp;journal=ACS%20Nano&amp;volume=15&amp;pages=1002-1015&amp;publication_year=2021&amp;author=Lee%2CJY">
                    Google Scholar</a> 
                </p></li><li data-counter="21."><p id="ref-CR21">Ermann, N. et al. Promoting single-file DNA translocations through nanopores using electro-osmotic flow. <i>J. Chem. Phys.</i> <b>149</b>, 163311 (2018).</p><p><a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 21" href="http://scholar.google.com/scholar_lookup?&amp;title=Promoting%20single-file%20DNA%20translocations%20through%20nanopores%20using%20electro-osmotic%20flow&amp;journal=J.%20Chem.%20Phys.&amp;volume=149&amp;publication_year=2018&amp;author=Ermann%2CN">
                    Google Scholar</a> 
                </p></li><li data-counter="22."><p id="ref-CR22">Shi, X. et al. Sustained unidirectional rotation of a self-organized DNA rotor on a nanopore. <i>Nat. Phys.</i> <b>18</b>, 1105–1111 (2022).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XitVOhtr3P" aria-label="CAS reference 22">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 22" href="http://scholar.google.com/scholar_lookup?&amp;title=Sustained%20unidirectional%20rotation%20of%20a%20self-organized%20DNA%20rotor%20on%20a%20nanopore&amp;journal=Nat.%20Phys.&amp;volume=18&amp;pages=1105-1111&amp;publication_year=2022&amp;author=Shi%2CX">
                    Google Scholar</a> 
                </p></li><li data-counter="23."><p id="ref-CR23">Van den Heuvel, M., De Graaff, M., Lemay, S. &amp; Dekker, C. Electrophoresis of individual microtubules in microchannels. <i>Proc. Natl Acad. Sci. USA</i> <b>104</b>, 7770–7775 (2007).</p><p><a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 23" href="http://scholar.google.com/scholar_lookup?&amp;title=Electrophoresis%20of%20individual%20microtubules%20in%20microchannels&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=104&amp;pages=7770-7775&amp;publication_year=2007&amp;author=Heuvel%2CM&amp;author=Graaff%2CM&amp;author=Lemay%2CS&amp;author=Dekker%2CC">
                    Google Scholar</a> 
                </p></li><li data-counter="24."><p id="ref-CR24">Van Der Drift, W., De Keizer, A. &amp; Overbeek, J. T. G. Electrophoretic mobility of a cylinder with high surface charge density. <i>J. Colloid Interface Sci.</i> <b>71</b>, 67–78 (1979).</p><p><a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 24" href="http://scholar.google.com/scholar_lookup?&amp;title=Electrophoretic%20mobility%20of%20a%20cylinder%20with%20high%20surface%20charge%20density&amp;journal=J.%20Colloid%20Interface%20Sci.&amp;volume=71&amp;pages=67-78&amp;publication_year=1979&amp;author=Drift%2CW&amp;author=Keizer%2CA&amp;author=Overbeek%2CJTG">
                    Google Scholar</a> 
                </p></li><li data-counter="25."><p id="ref-CR25">Yariv, E. &amp; Schnitzer, O. The electrophoretic mobility of rod-like particles. <i>J. Fluid Mech.</i> <b>719</b>, R3 (2013).</p><p><a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 25" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20electrophoretic%20mobility%20of%20rod-like%20particles&amp;journal=J.%20Fluid%20Mech.&amp;volume=719&amp;publication_year=2013&amp;author=Yariv%2CE&amp;author=Schnitzer%2CO">
                    Google Scholar</a> 
                </p></li><li data-counter="26."><p id="ref-CR26">Pumm, A.-K. et al. A DNA origami rotary ratchet motor. <i>Nature</i> <b>607</b>, 492–498 (2022).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XhvVOhsr3I" aria-label="CAS reference 26">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 26" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20DNA%20origami%20rotary%20ratchet%20motor&amp;journal=Nature&amp;volume=607&amp;pages=492-498&amp;publication_year=2022&amp;author=Pumm%2CA-K">
                    Google Scholar</a> 
                </p></li><li data-counter="27."><p id="ref-CR27">Verschueren, D. V., Yang, W. &amp; Dekker, C. Lithography-based fabrication of nanopore arrays in freestanding SiN and graphene membranes. <i>Nanotechnology</i> <b>29</b>, 145302 (2018).</p><p><a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 27" href="http://scholar.google.com/scholar_lookup?&amp;title=Lithography-based%20fabrication%20of%20nanopore%20arrays%20in%20freestanding%20SiN%20and%20graphene%20membranes&amp;journal=Nanotechnology&amp;volume=29&amp;publication_year=2018&amp;author=Verschueren%2CDV&amp;author=Yang%2CW&amp;author=Dekker%2CC">
                    Google Scholar</a> 
                </p></li><li data-counter="28."><p id="ref-CR28">Douglas, S. M. et al. Rapid prototyping of 3D DNA-origami shapes with caDNAno. <i>Nucleic Acids Res.</i> <b>37</b>, 5001–5006 (2009).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXhtVKntbzE" aria-label="CAS reference 28">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 28" href="http://scholar.google.com/scholar_lookup?&amp;title=Rapid%20prototyping%20of%203D%20DNA-origami%20shapes%20with%20caDNAno&amp;journal=Nucleic%20Acids%20Res.&amp;volume=37&amp;pages=5001-5006&amp;publication_year=2009&amp;author=Douglas%2CSM">
                    Google Scholar</a> 
                </p></li><li data-counter="29."><p id="ref-CR29">Wagenbauer, K. F. et al. How we make DNA origami. <i>ChemBioChem</i> <b>18</b>, 1873–1885 (2017).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXht12ltbzE" aria-label="CAS reference 29">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 29" href="http://scholar.google.com/scholar_lookup?&amp;title=How%20we%20make%20DNA%20origami&amp;journal=ChemBioChem&amp;volume=18&amp;pages=1873-1885&amp;publication_year=2017&amp;author=Wagenbauer%2CKF">
                    Google Scholar</a> 
                </p></li><li data-counter="30."><p id="ref-CR30">Kube, M. et al. Revealing the structures of megadalton-scale DNA complexes with nucleotide resolution. <i>Nat. Commun.</i> <b>11</b>, 6229 (2020).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXisFems7zM" aria-label="CAS reference 30">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 30" href="http://scholar.google.com/scholar_lookup?&amp;title=Revealing%20the%20structures%20of%20megadalton-scale%20DNA%20complexes%20with%20nucleotide%20resolution&amp;journal=Nat.%20Commun.&amp;volume=11&amp;publication_year=2020&amp;author=Kube%2CM">
                    Google Scholar</a> 
                </p></li><li data-counter="31."><p id="ref-CR31">Scheres, S. H. A Bayesian view on cryo-EM structure determination. <i>J. Mol. Biol.</i> <b>415</b>, 406–418 (2012).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XnvF2jtg%3D%3D" aria-label="CAS reference 31">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 31" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20Bayesian%20view%20on%20cryo-EM%20structure%20determination&amp;journal=J.%20Mol.%20Biol.&amp;volume=415&amp;pages=406-418&amp;publication_year=2012&amp;author=Scheres%2CSH">
                    Google Scholar</a> 
                </p></li><li data-counter="32."><p id="ref-CR32">Kimanius, D., Dong, L., Sharov, G., Nakane, T. &amp; Scheres, S. H. New tools for automated cryo-EM single-particle analysis in RELION-4.0. <i>Biochem. J.</i> <b>478</b>, 4169–4185 (2021).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XhsVaru7k%3D" aria-label="CAS reference 32">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 32" href="http://scholar.google.com/scholar_lookup?&amp;title=New%20tools%20for%20automated%20cryo-EM%20single-particle%20analysis%20in%20RELION-4.0&amp;journal=Biochem.%20J.&amp;volume=478&amp;pages=4169-4185&amp;publication_year=2021&amp;author=Kimanius%2CD&amp;author=Dong%2CL&amp;author=Sharov%2CG&amp;author=Nakane%2CT&amp;author=Scheres%2CSH">
                    Google Scholar</a> 
                </p></li><li data-counter="33."><p id="ref-CR33">Rohou, A. &amp; Grigorieff, N. CTFFIND4: fast and accurate defocus estimation from electron micrographs. <i>J. Struct. Biol.</i> <b>192</b>, 216–221 (2015).</p><p><a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 33" href="http://scholar.google.com/scholar_lookup?&amp;title=CTFFIND4%3A%20fast%20and%20accurate%20defocus%20estimation%20from%20electron%20micrographs&amp;journal=J.%20Struct.%20Biol.&amp;volume=192&amp;pages=216-221&amp;publication_year=2015&amp;author=Rohou%2CA&amp;author=Grigorieff%2CN">
                    Google Scholar</a> 
                </p></li><li data-counter="34."><p id="ref-CR34">Bepler, T. et al. Positive-unlabeled convolutional neural networks for particle picking in cryo-electron micrographs. <i>Nat. Methods</i> <b>16</b>, 1153–1160 (2019).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhvFejs7%2FM" aria-label="CAS reference 34">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 34" href="http://scholar.google.com/scholar_lookup?&amp;title=Positive-unlabeled%20convolutional%20neural%20networks%20for%20particle%20picking%20in%20cryo-electron%20micrographs&amp;journal=Nat.%20Methods&amp;volume=16&amp;pages=1153-1160&amp;publication_year=2019&amp;author=Bepler%2CT">
                    Google Scholar</a> 
                </p></li><li data-counter="35."><p id="ref-CR35">Nakane, T., Kimanius, D., Lindahl, E. &amp; Scheres, S. H. Characterisation of molecular motions in cryo-EM single-particle data by multi-body refinement in RELION. <i>eLife</i> <b>7</b>, e36861 (2018).</p><p><a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 35" href="http://scholar.google.com/scholar_lookup?&amp;title=Characterisation%20of%20molecular%20motions%20in%20cryo-EM%20single-particle%20data%20by%20multi-body%20refinement%20in%20RELION&amp;journal=eLife&amp;volume=7&amp;publication_year=2018&amp;author=Nakane%2CT&amp;author=Kimanius%2CD&amp;author=Lindahl%2CE&amp;author=Scheres%2CSH">
                    Google Scholar</a> 
                </p></li><li data-counter="36."><p id="ref-CR36">Pettersen, E. F. et al. UCSF Chimera—a visualization system for exploratory research and analysis. <i>J. Comput. Chem.</i> <b>25</b>, 1605–1612 (2004).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2cXmvVOhsbs%3D" aria-label="CAS reference 36">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 36" href="http://scholar.google.com/scholar_lookup?&amp;title=UCSF%20Chimera%E2%80%94a%20visualization%20system%20for%20exploratory%20research%20and%20analysis&amp;journal=J.%20Comput.%20Chem.&amp;volume=25&amp;pages=1605-1612&amp;publication_year=2004&amp;author=Pettersen%2CEF">
                    Google Scholar</a> 
                </p></li><li data-counter="37."><p id="ref-CR37">Schindelin, J. et al. Fiji: an open-source platform for biological-image analysis. <i>Nat. Methods</i> <b>9</b>, 676–682 (2012).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XhtVKnurbJ" aria-label="CAS reference 37">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 37" href="http://scholar.google.com/scholar_lookup?&amp;title=Fiji%3A%20an%20open-source%20platform%20for%20biological-image%20analysis&amp;journal=Nat.%20Methods&amp;volume=9&amp;pages=676-682&amp;publication_year=2012&amp;author=Schindelin%2CJ">
                    Google Scholar</a> 
                </p></li><li data-counter="38."><p id="ref-CR38">Pettersen, E. F. et al. UCSF ChimeraX: structure visualization for researchers, educators, and developers. <i>Protein Sci.</i> <b>30</b>, 70–82 (2021).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXitFamtrzJ" aria-label="CAS reference 38">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 38" href="http://scholar.google.com/scholar_lookup?&amp;title=UCSF%20ChimeraX%3A%20structure%20visualization%20for%20researchers%2C%20educators%2C%20and%20developers&amp;journal=Protein%20Sci.&amp;volume=30&amp;pages=70-82&amp;publication_year=2021&amp;author=Pettersen%2CEF">
                    Google Scholar</a> 
                </p></li><li data-counter="39."><p id="ref-CR39">Pud, S. et al. Self-aligned plasmonic nanopores by optically controlled dielectric breakdown. <i>Nano Lett.</i> <b>15</b>, 7112–7117 (2015).</p><p><a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 39" href="http://scholar.google.com/scholar_lookup?&amp;title=Self-aligned%20plasmonic%20nanopores%20by%20optically%20controlled%20dielectric%20breakdown&amp;journal=Nano%20Lett.&amp;volume=15&amp;pages=7112-7117&amp;publication_year=2015&amp;author=Pud%2CS">
                    Google Scholar</a> 
                </p></li><li data-counter="40."><p id="ref-CR40">Ovesný, M., Křížek, P., Borkovec, J., Švindrych, Z. &amp; Hagen, G. M. ThunderSTORM: a comprehensive ImageJ plug-in for PALM and STORM data analysis and super-resolution imaging. <i>Bioinformatics</i> <b>30</b>, 2389–2390 (2014).</p><p><a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 40" href="http://scholar.google.com/scholar_lookup?&amp;title=ThunderSTORM%3A%20a%20comprehensive%20ImageJ%20plug-in%20for%20PALM%20and%20STORM%20data%20analysis%20and%20super-resolution%20imaging&amp;journal=Bioinformatics&amp;volume=30&amp;pages=2389-2390&amp;publication_year=2014&amp;author=Ovesn%C3%BD%2CM&amp;author=K%C5%99%C3%AD%C5%BEek%2CP&amp;author=Borkovec%2CJ&amp;author=%C5%A0vindrych%2CZ&amp;author=Hagen%2CGM">
                    Google Scholar</a> 
                </p></li><li data-counter="41."><p id="ref-CR41">Phillips, J. C. et al. Scalable molecular dynamics on CPU and GPU architectures with NAMD. <i>J. Chem. Phys.</i> <b>153</b>, 044130 (2020).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhsFajsL3J" aria-label="CAS reference 41">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 41" href="http://scholar.google.com/scholar_lookup?&amp;title=Scalable%20molecular%20dynamics%20on%20CPU%20and%20GPU%20architectures%20with%20NAMD&amp;journal=J.%20Chem.%20Phys.&amp;volume=153&amp;publication_year=2020&amp;author=Phillips%2CJC">
                    Google Scholar</a> 
                </p></li><li data-counter="42."><p id="ref-CR42">Hart, K. et al. Optimization of the CHARMM additive force field for DNA: improved treatment of the BI/BII conformational equilibrium. <i>J. Chem. Theory Comput.</i> <b>8</b>, 348–362 (2012).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXhsF2lsbbO" aria-label="CAS reference 42">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 42" href="http://scholar.google.com/scholar_lookup?&amp;title=Optimization%20of%20the%20CHARMM%20additive%20force%20field%20for%20DNA%3A%20improved%20treatment%20of%20the%20BI%2FBII%20conformational%20equilibrium&amp;journal=J.%20Chem.%20Theory%20Comput.&amp;volume=8&amp;pages=348-362&amp;publication_year=2012&amp;author=Hart%2CK">
                    Google Scholar</a> 
                </p></li><li data-counter="43."><p id="ref-CR43">Yoo, J. &amp; Aksimentiev, A. Improved parametrization of Li<sup>+</sup>, Na<sup>+</sup>, K<sup>+</sup>, and Mg<sup>2+</sup> ions for all-atom molecular dynamics simulations of nucleic acid systems. <i>J. Phys. Chem. Lett.</i> <b>3</b>, 45–50 (2012).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXhsF2ltr3K" aria-label="CAS reference 43">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 43" href="http://scholar.google.com/scholar_lookup?&amp;title=Improved%20parametrization%20of%20Li%2B%2C%20Na%2B%2C%20K%2B%2C%20and%20Mg2%2B%20ions%20for%20all-atom%20molecular%20dynamics%20simulations%20of%20nucleic%20acid%20systems&amp;journal=J.%20Phys.%20Chem.%20Lett.&amp;volume=3&amp;pages=45-50&amp;publication_year=2012&amp;author=Yoo%2CJ&amp;author=Aksimentiev%2CA">
                    Google Scholar</a> 
                </p></li><li data-counter="44."><p id="ref-CR44">Jorgensen, W. L., Chandrasekhar, J., Madura, J. D., Impey, R. W. &amp; Klein, M. L. Comparison of simple potential functions for simulating liquid water. <i>J. Chem. Phys.</i> <b>79</b>, 926–935 (1983).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaL3sXksF2htL4%3D" aria-label="CAS reference 44">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 44" href="http://scholar.google.com/scholar_lookup?&amp;title=Comparison%20of%20simple%20potential%20functions%20for%20simulating%20liquid%20water&amp;journal=J.%20Chem.%20Phys.&amp;volume=79&amp;pages=926-935&amp;publication_year=1983&amp;author=Jorgensen%2CWL&amp;author=Chandrasekhar%2CJ&amp;author=Madura%2CJD&amp;author=Impey%2CRW&amp;author=Klein%2CML">
                    Google Scholar</a> 
                </p></li><li data-counter="45."><p id="ref-CR45">Darden, T., York, D. &amp; Pedersen, L. Particle mesh Ewald: an <i>N</i><span>⋅</span>log(<i>N</i>) method for Ewald sums in large systems. <i>J. Chem. Phys.</i> <b>98</b>, 10089–10092 (1993).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK3sXks1Ohsr0%3D" aria-label="CAS reference 45">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 45" href="http://scholar.google.com/scholar_lookup?&amp;title=Particle%20mesh%20Ewald%3A%20an%20N%E2%8B%85log%28N%29%20method%20for%20Ewald%20sums%20in%20large%20systems&amp;journal=J.%20Chem.%20Phys.&amp;volume=98&amp;pages=10089-10092&amp;publication_year=1993&amp;author=Darden%2CT&amp;author=York%2CD&amp;author=Pedersen%2CL">
                    Google Scholar</a> 
                </p></li><li data-counter="46."><p id="ref-CR46">Balusek, C. et al. Accelerating membrane simulations with hydrogen mass repartitioning. <i>J. Chem. Theory Comput.</i> <b>15</b>, 4673–4686 (2019).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXht1SqtL7E" aria-label="CAS reference 46">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 46" href="http://scholar.google.com/scholar_lookup?&amp;title=Accelerating%20membrane%20simulations%20with%20hydrogen%20mass%20repartitioning&amp;journal=J.%20Chem.%20Theory%20Comput.&amp;volume=15&amp;pages=4673-4686&amp;publication_year=2019&amp;author=Balusek%2CC">
                    Google Scholar</a> 
                </p></li><li data-counter="47."><p id="ref-CR47">Ryckaert, J.-P., Ciccotti, G. &amp; Berendsen, H. J. Numerical integration of the Cartesian equations of motion of a system with constraints: molecular dynamics of n-alkanes. <i>J. Comput. Phys.</i> <b>23</b>, 327–341 (1977).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaE2sXktVGhsL4%3D" aria-label="CAS reference 47">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 47" href="http://scholar.google.com/scholar_lookup?&amp;title=Numerical%20integration%20of%20the%20Cartesian%20equations%20of%20motion%20of%20a%20system%20with%20constraints%3A%20molecular%20dynamics%20of%20n-alkanes&amp;journal=J.%20Comput.%20Phys.&amp;volume=23&amp;pages=327-341&amp;publication_year=1977&amp;author=Ryckaert%2CJ-P&amp;author=Ciccotti%2CG&amp;author=Berendsen%2CHJ">
                    Google Scholar</a> 
                </p></li><li data-counter="48."><p id="ref-CR48">Miyamoto, S. &amp; Kollman, P. A. SETTLE: an analytical version of the SHAKE and RATTLE algorithm for rigid water models. <i>J. Comput. Chem.</i> <b>13</b>, 952–962 (1992).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK38Xlslykt7o%3D" aria-label="CAS reference 48">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 48" href="http://scholar.google.com/scholar_lookup?&amp;title=SETTLE%3A%20an%20analytical%20version%20of%20the%20SHAKE%20and%20RATTLE%20algorithm%20for%20rigid%20water%20models&amp;journal=J.%20Comput.%20Chem.&amp;volume=13&amp;pages=952-962&amp;publication_year=1992&amp;author=Miyamoto%2CS&amp;author=Kollman%2CPA">
                    Google Scholar</a> 
                </p></li><li data-counter="49."><p id="ref-CR49">Maffeo, C. &amp; Aksimentiev, A. MrDNA: a multi-resolution model for predicting the structure and dynamics of DNA systems. <i>Nucleic Acids Res.</i> <b>48</b>, 5135–5146 (2020).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXisVKlurfJ" aria-label="CAS reference 49">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 49" href="http://scholar.google.com/scholar_lookup?&amp;title=MrDNA%3A%20a%20multi-resolution%20model%20for%20predicting%20the%20structure%20and%20dynamics%20of%20DNA%20systems&amp;journal=Nucleic%20Acids%20Res.&amp;volume=48&amp;pages=5135-5146&amp;publication_year=2020&amp;author=Maffeo%2CC&amp;author=Aksimentiev%2CA">
                    Google Scholar</a> 
                </p></li><li data-counter="50."><p id="ref-CR50">Humphrey, W., Dalke, A. &amp; Schulten, K. VMD: visual molecular dynamics. <i>J. Mol. Graph.</i> <b>14</b>, 33–38 (1996).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK28Xis12nsrg%3D" aria-label="CAS reference 50">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 50" href="http://scholar.google.com/scholar_lookup?&amp;title=VMD%3A%20visual%20molecular%20dynamics&amp;journal=J.%20Mol.%20Graph.&amp;volume=14&amp;pages=33-38&amp;publication_year=1996&amp;author=Humphrey%2CW&amp;author=Dalke%2CA&amp;author=Schulten%2CK">
                    Google Scholar</a> 
                </p></li><li data-counter="51."><p id="ref-CR51">Martyna, G. J., Tobias, D. J. &amp; Klein, M. L. Constant pressure molecular dynamics algorithms. <i>J. Chem. Phys.</i> <b>101</b>, 4177–4189 (1994).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK2cXmtFeht7o%3D" aria-label="CAS reference 51">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 51" href="http://scholar.google.com/scholar_lookup?&amp;title=Constant%20pressure%20molecular%20dynamics%20algorithms&amp;journal=J.%20Chem.%20Phys.&amp;volume=101&amp;pages=4177-4189&amp;publication_year=1994&amp;author=Martyna%2CGJ&amp;author=Tobias%2CDJ&amp;author=Klein%2CML">
                    Google Scholar</a> 
                </p></li><li data-counter="52."><p id="ref-CR52">Feller, S. E., Zhang, Y., Pastor, R. W. &amp; Brooks, B. R. Constant pressure molecular dynamics simulation: the Langevin piston method. <i>J. Chem. Phys.</i> <b>103</b>, 4613–4621 (1995).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK2MXotVentLo%3D" aria-label="CAS reference 52">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 52" href="http://scholar.google.com/scholar_lookup?&amp;title=Constant%20pressure%20molecular%20dynamics%20simulation%3A%20the%20Langevin%20piston%20method&amp;journal=J.%20Chem.%20Phys.&amp;volume=103&amp;pages=4613-4621&amp;publication_year=1995&amp;author=Feller%2CSE&amp;author=Zhang%2CY&amp;author=Pastor%2CRW&amp;author=Brooks%2CBR">
                    Google Scholar</a> 
                </p></li><li data-counter="53."><p id="ref-CR53">Fiorin, G., Klein, M. L. &amp; Hénin, J. Using collective variables to drive molecular dynamics simulations. <i>Mol. Phys.</i> <b>111</b>, 3345–3362 (2013).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXpslGit7s%3D" aria-label="CAS reference 53">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 53" href="http://scholar.google.com/scholar_lookup?&amp;title=Using%20collective%20variables%20to%20drive%20molecular%20dynamics%20simulations&amp;journal=Mol.%20Phys.&amp;volume=111&amp;pages=3345-3362&amp;publication_year=2013&amp;author=Fiorin%2CG&amp;author=Klein%2CML&amp;author=H%C3%A9nin%2CJ">
                    Google Scholar</a> 
                </p></li></ol><p><a data-track="click" data-track-action="download citation references" data-track-label="link" rel="nofollow" href="https://citation-needed.springer.com/v2/references/10.1038/s41565-023-01527-8?format=refman&amp;flavour=references">Download references<svg width="16" height="16" focusable="false" role="img" aria-hidden="true"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-download"></use></svg></a></p></div></div></div></section></div><section data-title="Acknowledgements"><div id="Ack1-section"><h2 id="Ack1">Acknowledgements</h2><p>We thank M. Tišma for help with fluorescence microscopy imaging and P. Ketterer for initial turbine designs. We acknowledge funding support by the ERC Advanced Grant no. 883684 and the NanoFront and BaSyC programmes (to C.D.). This work was further supported by an ERC Consolidator Grant to H.D. (GA no. 724261), the Deutsche Forschungsgemeinschaft via the Gottfried-Wilhelm-Leibniz Programme (to H.D.) and the SFB863 Project ID 111166240 TPA9 (to H.D.). A.A. and C.M. acknowledge support through the National Science Foundation (USA) under grant DMR-1827346 (to A.A.). This work has received support from the Max Planck School Matter to Life and the MaxSynBio Consortium, which are jointly funded by the Federal Ministry of Education and Research (BMBF) of Germany, and the Max Planck Society (to R.G.). Supercomputer time was provided through Leadership Resource Allocation MCB20012 on Frontera and through ACCESS allocation MCA05S028.</p></div></section><section aria-labelledby="author-information" data-title="Author information"><div id="author-information-section"><h2 id="author-information">Author information</h2><div id="author-information-content"><p><span id="author-notes">Author notes</span></p><ol><li id="nAff7"><p>Xin Shi</p><p>Present address: Department of Chemistry, KU Leuven, Leuven, Belgium</p></li><li id="nAff8"><p>Daniel Verschueren</p><p>Present address: The SW7 Group, London, UK</p></li></ol><h3 id="affiliations">Authors and Affiliations</h3><ol><li id="Aff1"><p>Department of Bionanoscience, Kavli Institute of Nanoscience Delft, Delft University of Technology, Delft, The Netherlands</p><p>Xin Shi, Wenxuan Zhao, Daniel Verschueren &amp; Cees Dekker</p></li><li id="Aff2"><p>Department of Bioscience, School of Natural Sciences, Technical University of Munich, Garching, Germany</p><p>Anna-Katharina Pumm, Fabian Kohler, Elija Feigl &amp; Hendrik Dietz</p></li><li id="Aff3"><p>Munich Institute of Biomedical Engineering, Technical University of Munich, Garching, Germany</p><p>Anna-Katharina Pumm</p></li><li id="Aff4"><p>Department of Physics, University of Illinois at Urbana-Champaign, Urbana, IL, USA</p><p>Christopher Maffeo &amp; Aleksei Aksimentiev</p></li><li id="Aff5"><p>Max Planck Institute for Dynamics and Self-Organization, Göttingen, Germany</p><p>Ramin Golestanian</p></li><li id="Aff6"><p>Rudolf Peierls Centre for Theoretical Physics, University of Oxford, Oxford, UK</p><p>Ramin Golestanian</p></li></ol><h3 id="contributions">Contributions</h3><p>H.D. and C.D. conceived the concept of DNA turbines in nanopores and nanopore arrays. A.-K.P. and X.S. co-designed the geometries of the turbines. A.-K.P. designed and prepared the DNA origami structures. X.S. designed the nanopore experiment and fabricated nanopore devices, supported by D.V. X.S. and W.Z. conducted nanopore experiments. F.K. performed cryo-EM measurements; E.F. performed EM data analysis and the atomic model construction. X.S. and D.V. wrote the data analysis scripts and analysed data. R.G. constructed the continuum model. C.M. and A.A. designed and conducted MD simulations. All authors discussed the findings and co-wrote the manuscript.</p><h3 id="corresponding-author">Corresponding authors</h3><p id="corresponding-author-list">Correspondence to
                <a id="corresp-c1" href="mailto:aksiment@illinois.edu">Aleksei Aksimentiev</a>, <a id="corresp-c2" href="mailto:dietz@tum.de">Hendrik Dietz</a> or <a id="corresp-c3" href="mailto:c.dekker@tudelft.nl">Cees Dekker</a>.</p></div></div></section><section data-title="Ethics declarations"><div id="ethics-section"><h2 id="ethics">Ethics declarations</h2><div id="ethics-content">
            
              <h3 id="FPar2">Competing interests</h3>
              <p>The authors declare no competing interests.</p>
            
          </div></div></section><section data-title="Peer review"><div id="peer-review-section"><h2 id="peer-review">Peer review</h2><div id="peer-review-content">
            
            
              <h3 id="FPar1">Peer review information</h3>
              <p><i>Nature Nanotechnology</i> thanks Luca Piantanida and the other, anonymous, reviewer(s) for their contribution to the peer review of this work.</p>
            
          </div></div></section><section data-title="Additional information"><div id="additional-information-section"><h2 id="additional-information">Additional information</h2><p><b>Publisher’s note</b> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div></section><section data-title="Supplementary information"><div id="Sec17-section"><h2 id="Sec17">Supplementary information</h2></div></section><section data-title="Source data"></section><section data-title="Rights and permissions"><div id="rightslink-section"><h2 id="rightslink">Rights and permissions</h2><div id="rightslink-content">
              <p><b>Open Access</b>  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this license, visit <a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>.</p>
            <p><a data-track="click" data-track-action="view rights and permissions" data-track-label="link" href="https://s100.copyright.com/AppDispatchServlet?title=A%20DNA%20turbine%20powered%20by%20a%20transmembrane%20potential%20across%20a%20nanopore&amp;author=Xin%20Shi%20et%20al&amp;contentID=10.1038%2Fs41565-023-01527-8&amp;copyright=The%20Author%28s%29&amp;publication=1748-3387&amp;publicationDate=2023-10-26&amp;publisherName=SpringerNature&amp;orderBeanReset=true&amp;oa=CC%20BY">Reprints and Permissions</a></p></div></div></section><section aria-labelledby="article-info" data-title="About this article"><div id="article-info-section"><h2 id="article-info">About this article</h2><div id="article-info-content"><div><p><a data-crossmark="10.1038/s41565-023-01527-8" target="_blank" rel="noopener" href="https://crossmark.crossref.org/dialog/?doi=10.1038/s41565-023-01527-8" data-track="click" data-track-action="Click Crossmark" data-track-label="link" data-test="crossmark"><img width="57" height="81" alt="Check for updates. Verify currency and authenticity via CrossMark" src="data:image/svg+xml;base64,<svg height="81" width="57" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="m17.35 35.45 21.3-14.2v-17.03h-21.3" fill="#989898"/><path d="m38.65 35.45-21.3-14.2v-17.03h21.3" fill="#747474"/><path d="m28 .5c-12.98 0-23.5 10.52-23.5 23.5s10.52 23.5 23.5 23.5 23.5-10.52 23.5-23.5c0-6.23-2.48-12.21-6.88-16.62-4.41-4.4-10.39-6.88-16.62-6.88zm0 41.25c-9.8 0-17.75-7.95-17.75-17.75s7.95-17.75 17.75-17.75 17.75 7.95 17.75 17.75c0 4.71-1.87 9.22-5.2 12.55s-7.84 5.2-12.55 5.2z" fill="#535353"/><path d="m41 36c-5.81 6.23-15.23 7.45-22.43 2.9-7.21-4.55-10.16-13.57-7.03-21.5l-4.92-3.11c-4.95 10.7-1.19 23.42 8.78 29.71 9.97 6.3 23.07 4.22 30.6-4.86z" fill="#9c9c9c"/><path d="m.2 58.45c0-.75.11-1.42.33-2.01s.52-1.09.91-1.5c.38-.41.83-.73 1.34-.94.51-.22 1.06-.32 1.65-.32.56 0 1.06.11 1.51.35.44.23.81.5 1.1.81l-.91 1.01c-.24-.24-.49-.42-.75-.56-.27-.13-.58-.2-.93-.2-.39 0-.73.08-1.05.23-.31.16-.58.37-.81.66-.23.28-.41.63-.53 1.04-.13.41-.19.88-.19 1.39 0 1.04.23 1.86.68 2.46.45.59 1.06.88 1.84.88.41 0 .77-.07 1.07-.23s.59-.39.85-.68l.91 1c-.38.43-.8.76-1.28.99-.47.22-1 .34-1.58.34-.59 0-1.13-.1-1.64-.31-.5-.2-.94-.51-1.31-.91-.38-.4-.67-.9-.88-1.48-.22-.59-.33-1.26-.33-2.02zm8.4-5.33h1.61v2.54l-.05 1.33c.29-.27.61-.51.96-.72s.76-.31 1.24-.31c.73 0 1.27.23 1.61.71.33.47.5 1.14.5 2.02v4.31h-1.61v-4.1c0-.57-.08-.97-.25-1.21-.17-.23-.45-.35-.83-.35-.3 0-.56.08-.79.22-.23.15-.49.36-.78.64v4.8h-1.61zm7.37 6.45c0-.56.09-1.06.26-1.51.18-.45.42-.83.71-1.14.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.36c.07.62.29 1.1.65 1.44.36.33.82.5 1.38.5.29 0 .57-.04.83-.13s.51-.21.76-.37l.55 1.01c-.33.21-.69.39-1.09.53-.41.14-.83.21-1.26.21-.48 0-.92-.08-1.34-.25-.41-.16-.76-.4-1.07-.7-.31-.31-.55-.69-.72-1.13-.18-.44-.26-.95-.26-1.52zm4.6-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.07.45-.31.29-.5.73-.58 1.3zm2.5.62c0-.57.09-1.08.28-1.53.18-.44.43-.82.75-1.13s.69-.54 1.1-.71c.42-.16.85-.24 1.31-.24.45 0 .84.08 1.17.23s.61.34.85.57l-.77 1.02c-.19-.16-.38-.28-.56-.37-.19-.09-.39-.14-.61-.14-.56 0-1.01.21-1.35.63-.35.41-.52.97-.52 1.67 0 .69.17 1.24.51 1.66.34.41.78.62 1.32.62.28 0 .54-.06.78-.17.24-.12.45-.26.64-.42l.67 1.03c-.33.29-.69.51-1.08.65-.39.15-.78.23-1.18.23-.46 0-.9-.08-1.31-.24-.4-.16-.75-.39-1.05-.7s-.53-.69-.7-1.13c-.17-.45-.25-.96-.25-1.53zm6.91-6.45h1.58v6.17h.05l2.54-3.16h1.77l-2.35 2.8 2.59 4.07h-1.75l-1.77-2.98-1.08 1.23v1.75h-1.58zm13.69 1.27c-.25-.11-.5-.17-.75-.17-.58 0-.87.39-.87 1.16v.75h1.34v1.27h-1.34v5.6h-1.61v-5.6h-.92v-1.2l.92-.07v-.72c0-.35.04-.68.13-.98.08-.31.21-.57.4-.79s.42-.39.71-.51c.28-.12.63-.18 1.04-.18.24 0 .48.02.69.07.22.05.41.1.57.17zm.48 5.18c0-.57.09-1.08.27-1.53.17-.44.41-.82.72-1.13.3-.31.65-.54 1.04-.71.39-.16.8-.24 1.23-.24s.84.08 1.24.24c.4.17.74.4 1.04.71s.54.69.72 1.13c.19.45.28.96.28 1.53s-.09 1.08-.28 1.53c-.18.44-.42.82-.72 1.13s-.64.54-1.04.7-.81.24-1.24.24-.84-.08-1.23-.24-.74-.39-1.04-.7c-.31-.31-.55-.69-.72-1.13-.18-.45-.27-.96-.27-1.53zm1.65 0c0 .69.14 1.24.43 1.66.28.41.68.62 1.18.62.51 0 .9-.21 1.19-.62.29-.42.44-.97.44-1.66 0-.7-.15-1.26-.44-1.67-.29-.42-.68-.63-1.19-.63-.5 0-.9.21-1.18.63-.29.41-.43.97-.43 1.67zm6.48-3.44h1.33l.12 1.21h.05c.24-.44.54-.79.88-1.02.35-.24.7-.36 1.07-.36.32 0 .59.05.78.14l-.28 1.4-.33-.09c-.11-.01-.23-.02-.38-.02-.27 0-.56.1-.86.31s-.55.58-.77 1.1v4.2h-1.61zm-47.87 15h1.61v4.1c0 .57.08.97.25 1.2.17.24.44.35.81.35.3 0 .57-.07.8-.22.22-.15.47-.39.73-.73v-4.7h1.61v6.87h-1.32l-.12-1.01h-.04c-.3.36-.63.64-.98.86-.35.21-.76.32-1.24.32-.73 0-1.27-.24-1.61-.71-.33-.47-.5-1.14-.5-2.02zm9.46 7.43v2.16h-1.61v-9.59h1.33l.12.72h.05c.29-.24.61-.45.97-.63.35-.17.72-.26 1.1-.26.43 0 .81.08 1.15.24.33.17.61.4.84.71.24.31.41.68.53 1.11.13.42.19.91.19 1.44 0 .59-.09 1.11-.25 1.57-.16.47-.38.85-.65 1.16-.27.32-.58.56-.94.73-.35.16-.72.25-1.1.25-.3 0-.6-.07-.9-.2s-.59-.31-.87-.56zm0-2.3c.26.22.5.37.73.45.24.09.46.13.66.13.46 0 .84-.2 1.15-.6.31-.39.46-.98.46-1.77 0-.69-.12-1.22-.35-1.61-.23-.38-.61-.57-1.13-.57-.49 0-.99.26-1.52.77zm5.87-1.69c0-.56.08-1.06.25-1.51.16-.45.37-.83.65-1.14.27-.3.58-.54.93-.71s.71-.25 1.08-.25c.39 0 .73.07 1 .2.27.14.54.32.81.55l-.06-1.1v-2.49h1.61v9.88h-1.33l-.11-.74h-.06c-.25.25-.54.46-.88.64-.33.18-.69.27-1.06.27-.87 0-1.56-.32-2.07-.95s-.76-1.51-.76-2.65zm1.67-.01c0 .74.13 1.31.4 1.7.26.38.65.58 1.15.58.51 0 .99-.26 1.44-.77v-3.21c-.24-.21-.48-.36-.7-.45-.23-.08-.46-.12-.7-.12-.45 0-.82.19-1.13.59-.31.39-.46.95-.46 1.68zm6.35 1.59c0-.73.32-1.3.97-1.71.64-.4 1.67-.68 3.08-.84 0-.17-.02-.34-.07-.51-.05-.16-.12-.3-.22-.43s-.22-.22-.38-.3c-.15-.06-.34-.1-.58-.1-.34 0-.68.07-1 .2s-.63.29-.93.47l-.59-1.08c.39-.24.81-.45 1.28-.63.47-.17.99-.26 1.54-.26.86 0 1.51.25 1.93.76s.63 1.25.63 2.21v4.07h-1.32l-.12-.76h-.05c-.3.27-.63.48-.98.66s-.73.27-1.14.27c-.61 0-1.1-.19-1.48-.56-.38-.36-.57-.85-.57-1.46zm1.57-.12c0 .3.09.53.27.67.19.14.42.21.71.21.28 0 .54-.07.77-.2s.48-.31.73-.56v-1.54c-.47.06-.86.13-1.18.23-.31.09-.57.19-.76.31s-.33.25-.41.4c-.09.15-.13.31-.13.48zm6.29-3.63h-.98v-1.2l1.06-.07.2-1.88h1.34v1.88h1.75v1.27h-1.75v3.28c0 .8.32 1.2.97 1.2.12 0 .24-.01.37-.04.12-.03.24-.07.34-.11l.28 1.19c-.19.06-.4.12-.64.17-.23.05-.49.08-.76.08-.4 0-.74-.06-1.02-.18-.27-.13-.49-.3-.67-.52-.17-.21-.3-.48-.37-.78-.08-.3-.12-.64-.12-1.01zm4.36 2.17c0-.56.09-1.06.27-1.51s.41-.83.71-1.14c.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.37c.08.62.29 1.1.65 1.44.36.33.82.5 1.38.5.3 0 .58-.04.84-.13.25-.09.51-.21.76-.37l.54 1.01c-.32.21-.69.39-1.09.53s-.82.21-1.26.21c-.47 0-.92-.08-1.33-.25-.41-.16-.77-.4-1.08-.7-.3-.31-.54-.69-.72-1.13-.17-.44-.26-.95-.26-1.52zm4.61-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.08.45-.31.29-.5.73-.57 1.3zm3.01 2.23c.31.24.61.43.92.57.3.13.63.2.98.2.38 0 .65-.08.83-.23s.27-.35.27-.6c0-.14-.05-.26-.13-.37-.08-.1-.2-.2-.34-.28-.14-.09-.29-.16-.47-.23l-.53-.22c-.23-.09-.46-.18-.69-.3-.23-.11-.44-.24-.62-.4s-.33-.35-.45-.55c-.12-.21-.18-.46-.18-.75 0-.61.23-1.1.68-1.49.44-.38 1.06-.57 1.83-.57.48 0 .91.08 1.29.25s.71.36.99.57l-.74.98c-.24-.17-.49-.32-.73-.42-.25-.11-.51-.16-.78-.16-.35 0-.6.07-.76.21-.17.15-.25.33-.25.54 0 .14.04.26.12.36s.18.18.31.26c.14.07.29.14.46.21l.54.19c.23.09.47.18.7.29s.44.24.64.4c.19.16.34.35.46.58.11.23.17.5.17.82 0 .3-.06.58-.17.83-.12.26-.29.48-.51.68-.23.19-.51.34-.84.45-.34.11-.72.17-1.15.17-.48 0-.95-.09-1.41-.27-.46-.19-.86-.41-1.2-.68z" fill="#535353"/></g></svg>"/></a></p><div><h3 id="citeas">Cite this article</h3><p>Shi, X., Pumm, AK., Maffeo, C. <i>et al.</i> A DNA turbine powered by a transmembrane potential across a nanopore.
                    <i>Nat. Nanotechnol.</i>  (2023). https://doi.org/10.1038/s41565-023-01527-8</p><p><a data-test="citation-link" data-track="click" data-track-action="download article citation" data-track-label="link" data-track-external="" rel="nofollow" href="https://citation-needed.springer.com/v2/references/10.1038/s41565-023-01527-8?format=refman&amp;flavour=citation">Download citation<svg width="16" height="16" focusable="false" role="img" aria-hidden="true"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-download"></use></svg></a></p><ul data-test="publication-history"><li><p>Received<span>: </span><span><time datetime="2023-03-30">30 March 2023</time></span></p></li><li><p>Accepted<span>: </span><span><time datetime="2023-09-12">12 September 2023</time></span></p></li><li><p>Published<span>: </span><span><time datetime="2023-10-26">26 October 2023</time></span></p></li><li><p><abbr title="Digital Object Identifier">DOI</abbr><span>: </span><span>https://doi.org/10.1038/s41565-023-01527-8</span></p></li></ul></div></div></div></div></section>
            </div></div>
  </body>
</html>
