<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <title>James Routley | Feed</title>
    <link
      rel="stylesheet"
      type="text/css"
      href="../styles.css"
      media="screen"
    />
  </head>
  <body>
    <a href="/index.html">Back</a>
    <a href="https://berthub.eu/articles/posts/amazing-dna/">Original</a>
    <h1>DNA seen through the eyes of a coder</h1>
    
    <div id="readability-page-1" class="page"><article lang="en">
  

  
  

  <div>
  <blockquote>
<p><strong>Updates:</strong>
12th of September 2021: I’m writing a book on DNA! If you want to become a
beta reader, or have suggestions, <a href="https://berthub.eu/dna-book">I’d love to hear from you</a>!</p>
<p>8th of January 2021:
This article has been revised and updated, scientifically and in terms of dead links. Revision made by Tomás Simões (<a href="https://twitter.com/putadagravidade">@putadagravidade</a> / <a href="mailto:tomasprsimoes@gmail.com">tomasprsimoes@gmail.com</a>). Feel free to contact me if I made a mistake.</p>
<p>25th of August 2017:
This page has led to a two-hour presentation called <a href="https://berthub.eu/dna/">DNA: The code of Life</a> as presented at <a href="https://sha2017.org/">SHA 2017</a>. Includes slides and video and a <a href="https://berthub.eu/articles/posts/dna-the-code-of-life/">summarizing blogpost</a>. If you like this page, you’ll love the presentation.</p>
</blockquote>
<p>This is some rambling by a computer programmer about DNA. I’m not a molecular geneticist
(Update: 20 years after starting this post, I can fake it reasonably well.
This page was started somewhere in 2001, and it may need some more updating
here and there. Since 2001 I’ve learned a few things and I think I need to
revisit some parts of this page.)</p>
<p>If you spot mistakes, please contact me (<a href="https://twitter.com/bert_hu_bert">@bert_hu_bert</a> / <a href="mailto:bert@hubertnet.nl">bert@hubertnet.nl</a>).</p>
<p>I’m not trying to force my view unto the DNA - each observation here is quite ‘uncramped’. To see where I got all this from, head to the Bibliography (end of the page).</p>
<h2 id="the-source-code">The source code</h2>
<p>Is <a href="ftp://ftp.ensembl.org/pub/current_genbank/homo_sapiens/">here</a>. This not a joke. We can wonder about the license though. Maybe we should ask the walking product of this source: <a href="https://www.theguardian.com/world/2002/apr/28/medicalscience.research">Craig Venter</a> (update: not quite true, it is mostly someone else). The source can be viewed via a wonderful set of perl scripts called <a href="http://www.ensembl.org/index.html">‘Ensembl’</a>. The human genome is about 3 gigabases long, which boils down to 750 megabytes. Depressingly enough, this is only 3.6 (update: used to be 2.8, apparently Firefox decreased in size, huh.) Mozilla browsers.</p>
<p>DNA is not like C source but more like byte-compiled code for a virtual machine called ‘the nucleus’. It is very doubtful that there is a source to this byte compilation - what you see is all you get.</p>
<center>
<figure>
    <img src="https://berthub.eu/articles/amazing-dna/dna.jpg" alt="Illustration of a DNA molecule."/> <figcaption>
            <p>Illustration of a DNA molecule.</p>
        </figcaption>
</figure>

</center>
<p>The language of DNA is digital, but not binary. Where binary encoding has 0 and 1 to work with (2 - hence the ‘bi’nary), DNA has 4 positions, T, C, G and A.</p>
<p>Whereas a digital byte is mostly 8 binary digits, a DNA ‘byte’ (called a ‘codon’) has three digits. Because each digit can have 4 values instead of 2, a DNA codon has 64 possible values, compared to a binary byte which has 256.</p>
<p>A typical example of a DNA codon is ‘GCC’, which encodes the amino acid Alanine. A larger number of these amino acids combined are called a ‘polypeptide’ or ‘protein’, and these are chemically active in making a living being.</p>
<p>See also <a href="https://www.nature.com/scitable/definition/codon-155/">https://www.nature.com/scitable/definition/codon-155/</a></p>
<h2 id="position-independent-code">Position Independent Code</h2>
<p><em>Dynamically linked libraries (.so under Unix, .dll on Windows) code cannot use static addresses internally because the code may appear in different places in memory in different situations. DNA has this too, where it is called ‘transposing code’:</em></p>
<p>Nearly half of the human genome is composed of transposable elements or jumping DNA. First recognized in the 1940s by Dr. Barbara McClintock in studies of peculiar inheritance patterns found in the colors of Indian corn, jumping DNA refers to the idea that some stretches of DNA are unstable and “transposable,” ie., they can move around – on and between chromosomes.</p>
<p><a href="https://www.nature.com/scitable/topicpage/transposons-the-jumping-genes-518/">https://www.nature.com/scitable/topicpage/transposons-the-jumping-genes-518/</a></p>
<h2 id="conditional-compilation">Conditional compilation</h2>
<center>
<figure>
    <img src="https://berthub.eu/articles/amazing-dna/human_chrom.gif" alt="Illustration of human chromosomes."/> <figcaption>
            <p>Illustration of human chromosomes.</p>
        </figcaption>
</figure>

</center>
<p>Of the 20,000 to 30,000 genes now thought to be in the human genome (update: <strong>quite</strong> <a href="https://bmcbiol.biomedcentral.com/articles/10.1186/s12915-018-0564-x#Sec5">debatable</a>), most cells express only a very small part - which makes sense, a liver cell has little need for the DNA code that makes neurons.</p>
<p>But as almost all cells carry around a full copy (‘distribution’) of the genome, a system is needed to #ifdef out stuff not needed. And that is just how it works. The genetic code is full of #if/#endif statements.</p>
<p>This is why ‘<a href="https://www.bhf.org.uk/informationsupport/heart-matters-magazine/research/breakthroughs-in-stem-cell-research">stem cells</a>’ are so hot right now - these cells have the ability to differentiate into everything. The code hasn’t been #ifdeffed out yet, so to speak.</p>
<p>Stated more exactly, stem cells do not have everything turned on - they are not at once liver cells and neurons. Cells can be likened to state machines, starting out as a stem cell. Over the lifetime of the cell, during which time it may clone (‘fork()&#39;) many times, it specializes. Each specialization can be regarded as choosing a branch in a tree.</p>
<p>Each cell can make (or be induced to make) decisions about its future, which each make it more specialized. These decisions are persistent over cloning using transcription factors and by modifying the way DNA is stored spatially (‘steric effects’).</p>
<p>A liver cell, although it carries the genes to do so, will generally not be able to function as a skin cell. There are some indications out there that it is possible to ‘breed’ cells ‘upwards’ into the hierarchy, making them pluripotent. See also <a href="https://www.sciencedaily.com/releases/2002/01/020131074645.htm">this article</a>.</p>
<h2 id="epigenetics--imprinting-runtime-binary-patching">Epigenetics &amp; imprinting: runtime binary patching</h2>
<p>Although the actual relevant changes in the DNA of an organism rarely occur within a generation, substantial tinkering goes on by activating or deactivating parts of our genome, without altering the actual code.</p>
<p>This can be compared to the Linux kernel, which at boot time discovers what CPU it is running on, and actually disables parts of its binary code in case (for example) it is running on a single CPU system. This goes beyond something like if(numcpus &gt; 1), it is the actual nopping out of locking. Crucially, this nopping occurs in memory and not on the disk based image.</p>
<p>Similarly, as an embryo develops in the mother’s womb, its DNA is edited substantially to reduce its growth rate, and the size of the placenta. In such a way, the competing interests of the father (‘large strong children’) and the mother (‘survive pregnancy’) are balanced. Such ‘imprinting’ can only happen within the mother, since the father’s genome doesn’t know anything about the size of the mother.</p>
<p>Recently, it is also becoming clear that the metabolic status of the parents influences the <a href="https://en.wikipedia.org/wiki/Transgenerational_epigenetic_inheritance">chances of long life, cancer and diabetes in their grandchildren</a>. This also makes sense, as surviving in a food poor climate may require a different metabolic strategy than in one where food is abundantly available.</p>
<p>Mechanisms behind epigenetics and imprinting are ‘methylation’, which attaches methyl groups to DNA to ‘flip’ their activation status, but also histone modification, which can curl up DNA so it is not activated.</p>
<p>Some of these DNA edits are heritable and passed on to children, other forms may only impact one animal.</p>
<p>This field is still developing rapidly, and it may be that our DNA is much more dynamic than originally thought.</p>

<center>
<figure>
    <img src="https://berthub.eu/articles/amazing-dna/intron.gif" alt="From genes to proteins."/> <figcaption>
            <p>From genes to proteins.</p>
        </figcaption>
</figure>

</center>
<p>The genome is littered with old copies of genes and experiments that went wrong somewhere in the recent past - say, the last half a million years. This code is there but inactive. These are called the ‘pseudo genes’.</p>
<p>Furthermore, 97% of your DNA is commented out. DNA is linear and read from start to end. The parts that should not be decoded are marked very clearly, much like C comments. The 3% that is used directly form the so called ‘exons’. The comments, that come ‘in-between’ are called ‘introns’.</p>
<p>These comments are fascinating in their own right. Like C comments they have a start marker, like /*, and a stop marker, like */. But they have some more structure. Remember that DNA is like a tape - the comments need to be snipped out physically! The start of a comment is almost always indicated by the letters ‘GT’, which thus corresponds to /*, the end is signaled by ‘AG’, which is then like */.</p>
<p>However because of the snipping, some glue is needed to connect the code before the comment to the code after, which makes the comments more like html comments, which are longer: ‘&lt;!–’ signifies the start, ‘–&gt;’ the end.</p>
<p>So an actual stretch of DNA with exons and introns might look like this:</p>
<pre><code>ACTUAL CODE&lt;!-- blah blah blah blah ---- blah --&gt;ACTUAL CODE
  |         |            |           |        |        |
exon 1    donor       intron 1     branch  acceptor  exon 2
     (start of comment)                 (end of comment)
</code></pre><p>The start of the comment is clear, which is then followed by a lot of non-coding DNA. Somewhere very near the end of the comment there is a ‘branch site’, which indicates that the comment will end soon. Then some more comment follows, and then the actual terminator.</p>
<p>The actual cutting of the comments happens after the DNA has been transcribed into RNA and is performed by looping the comment and bringing the pieces of actual code close together. Then the RNA is cut at the ‘branch site’ near the end of the comment, after which the ‘donor’ (comment start) and ‘acceptor’ (comment end) are connected to each other.</p>
<p>Now, what are these comments good for? That discussion is part of a holy war that can rival the <a href="https://xkcd.com/378/">vi/emacs one</a>. When comparing different species, we know that some introns show fewer code changes than the neighboring exons. This suggests that the comments are doing something important.</p>
<p>There are lots of possible explanations for the massive amount of non-coding DNA - one of the most appealing (to a coder) has to do with ‘folding propensity’. DNA needs to be stored in a highly coiled form, but not all DNA codes lend themselves well to this.</p>
<p>This may remind you of <a href="https://www.redhill.net.au/d/10.php">RLL or MFM</a> coding. On a hard disk, a bit is encoded by a polarity transition or the lack thereof. A naive encoding would encode a 0 as ‘no transition’ and 1 as ‘a transition’.</p>
<p>Encoding 000000 is easy - just keep the magnetic phase unchanged for a few micrometers. However, when decoding, uncertainty creeps in - how many micrometers did we read? Does this correspond to 6 zeroes or 5? To prevent this problem, data is treated such that these long stretches of no transitions do not occur.</p>
<p>If we see ‘no transition,no transition,transition,transition’ on disk, we can be sure that this corresponds to ‘0011’ - it is exceedingly unlikely that our reading process is so imprecise that this might correspond to ‘00011’ or ‘00111’. So we need to insert spacers so as to prevent too little transitions. This is called ‘Run Length Limiting’ on magnetic media.</p>
<p>The thing to note is that sometimes, transitions need to be inserted to make sure that the data can be stored reliably. Introns may do much the same thing by making sure that the resulting code can be coiled properly.</p>
<p>However, this area of molecular biology is a minefield! Huge diatribes rage about variants with exciting names like ‘introns early’ or ‘introns late’, and massive words like ‘folding propensity’ and ‘stem-loop potential’. I think it best to let this discussion rage on a bit.</p>
<blockquote>
<p>2013 Update: ten years on, the debate still hasn’t settled! It is very clear that ‘Junk DNA’ is a misnomer, but as to its immediate function, there is no consensus. Check out Fighting about ENCODE and junk for a discussion of where we stand.</p>
<p>2021 Update: eighteen years on, the debate is nowhere close to being settled. It is now somewhat consensual that ‘Junk DNA’ has important and diverse functions, but new discoveries are being made on a daily basis. <a href="https://www.advancedsciencenews.com/that-junk-dna-is-full-of-information/">https://www.advancedsciencenews.com/that-junk-dna-is-full-of-information/</a></p>
</blockquote>
<h2 id="fork-and-fork-bombs-tumors">fork() and fork bombs (‘tumors’)</h2>
<p>Like with Unix, cells are not ‘spawned’ - they are forked. All cells started out from your ovum which has forked itself many times since. Like processes, both halves of the fork() are (mostly) identical to begin with, but they may from then on decide to do different things.</p>
<p>As with Unix, great problems arise when cells keep on forking. They quickly exhaust resources, sometimes leading to death. This is called a tumor. The cell is riddled with ‘ulimits’ and ‘watchdogs’ to prevent this sort of thing from happening. The number of divisions is limited by <a href="https://en.wikipedia.org/wiki/Telomere#Shortening">Telomere shortening</a>, for example.</p>
<p>A cell cannot clone unless very stringent conditions are met - a ‘<a href="https://www.openbsd.org/">secure by default</a>’ configuration. It is only when these safeguards fail that tumors can grow. Like with computer security, it is hard to strike a balance between security (‘no cells can divide’) and usability.</p>
<p>Compare this to the well known <a href="https://www.youtube.com/watch?v=eqvBaj8UYz4">Halting Problem</a>, first described by the founder of Computer Science, <a href="https://www.turing.org.uk/index.html">Alan Turing</a>. Perhaps it is as impossible to predict if a program will ever finish as it is to create a functional genome that cannot get cancer?</p>
<h2 id="mirroring-failover">Mirroring, failover</h2>
<center>
<figure>
    <img src="https://berthub.eu/articles/amazing-dna/helix.gif" alt="Stretch of DNA."/> <figcaption>
            <p>Stretch of DNA.</p>
        </figcaption>
</figure>

</center>
<p>Each DNA Helix is redundant in itself - you can see the genome as a twisted ladder whereby each spoke contains two bases - hence the word ‘base-pair’. If one of these bases is missing, it can be derived from the one on the other side. T always binds to A, C always to G. So, we can state that the genome is mirrored within the helix. ‘RAID-1’ so to speak.</p>
<p>Furthermore, there are two copies of each chromosome present - one from each parent, with the notable exception of the Y chromosome, which is only present in males. The actual details are complicated - but most genes are thus present twice. In case one is broken or unsuccessfully mutated, the other independent copy is still there. This is what we would normally call ‘failover’.</p>
<h2 id="cluttered-apis-dependency-hell">Cluttered APIs, dependency hell</h2>
<p>As proteins interact in the cell, they rely on each others’ characteristics. It has just been shown that <a href="https://www.nature.com/articles/srep44948">proteins that interact with a lot of other proteins cannot evolve</a>, or at least, only do so at a very slow rate.</p>
<p>They propose that this is because of great internal dependencies which inhibit the changing of the ‘contract’ of the protein. It is also noted that evolution does take place, but very slowly as both parts of the dependency need to evolve in a compatible way at the same time.</p>
<h2 id="viruses-worms">Viruses, worms</h2>
<p>Somebody recently proposed in a discussion that it would be really cool to hack the genome and compromise it so as to insert code that would copy itself to other genomes, using the host-body as its vehicle. ‘Just like the <a href="https://en.wikipedia.org/wiki/Nimda">nimda worm</a>!’</p>
<p><em>He shortly thereafter realized that this is exactly what biological viruses have been doing for millions of years. And they are exceedingly good at it.
A lot of these viruses have become a fixed part of our genome and hitch a ride with all of us. To do so, they have to hide from the virus scanner which tries to detect foreign code and prevent it from getting into the DNA.</em></p>
<h2 id="the-central-dogma-c---o---aoutexe">The Central Dogma: .c -&gt; .o -&gt; a.out/.exe</h2>
<p>When scientists were still discovering the basics of genetics they were faced with lots of different chemicals but the correlation was unclear. When it became clear what comes from what it was hailed as a great triumph and called ‘The Central Dogma’.</p>
<p>This dogma tells us that DNA is used to make RNA and that RNA is used to make proteins, which is like saying that from a .c file comes a .o object file, which can be compiled into an executable (a.out/exe). It also tells us that this is the only order in which information flows.</p>
<p>Now, the Central Dogma has <a href="https://en.wikipedia.org/wiki/Central_dogma_of_molecular_biology#Transfers_of_information_not_explicitly_covered_in_the_theory">recently been tarnished</a> somewhat. Like any billion year old coding project, a lot of hacking has been going on, and sometimes information flows the other way. Sometimes RNA patches the DNA and at other times, the DNA is modified by proteines created earlier.</p>
<p>But generally, the dependencies are clear, so the Central Dogma remains important.</p>
<h2 id="binary-patching-aka-gene-therapy">Binary patching aka ‘Gene therapy’</h2>
<center>
<figure>
    <img src="https://berthub.eu/articles/amazing-dna/genetherapy1big.gif"/> 
</figure>

</center><p>
We can fiddle easily enough with DNA. There are companies to which you can send an ASCII file with DNA characters, and they will synthesize the corresponding &#39;output&#39; for you. We can also splice DNA into developing animals and plants.
</p><p>It is far harder to ‘patch the running executable’, as any programmer can attest. It is just like that with the genome. To change a running copy (‘a human’), you need to edit each and every relevant copy of the gene you want to patch.</p>
<p>For many years, medical science has tried to patch people with SCID, or ‘Severe Combined Immunodeficiency’, which is a very nasty disease which in effect disables the immune system - leading to very ill patients. It has been clear for quite a while now which letters in the DNA need to be fixed in order to cure these people.</p>
<p>Many attempts where made to patch running people, using viruses that insert new DNA into living organisms, but this proved to be very hard. The genome is guarded far too well for such a simple approach to work - cells guard their code <a href="https://www.zdnet.com/article/windows-xp-source-code-leaked-online-on-4chan-out-of-all-places/">better than Microsoft</a>!</p>
<p>However, recently the right virus was found which was able to breach the protection of the genome and fix the broken characters, leading to <a href="https://www.medicalnewstoday.com/articles/320018">apparently healthy people</a>.</p>
<h2 id="bug-regression">Bug Regression</h2>
<p>When fixing a bug in a computer program, we often introduce new bugs in the course of doing so. The genome is rife with this thing. A lot of African Americans are immune to Malaria but instead suffer from <a href="https://en.wikipedia.org/wiki/Sickle_cell_disease">sickle cell anemia</a>:</p>
<blockquote>
<p>In tropical regions of the world where the parasite-borne disease malaria is prevalent, people with a single copy of a particular genetic mutation have a survival advantage.
(…)
While inheriting one copy of the mutation confers a benefit, inheriting two copies is a tragedy. Children born with two copies of the genetic mutation have sickle cell anemia, a painful disease that affects the red blood cells.</p>
<p><a href="https://books.google.pt/books?id=xBFNZCI3kd4C&amp;pg=RA3-PA17&amp;lpg=RA3-PA17&amp;dq=While+inheriting+one+copy+of+the+mutation+confers+a+benefit,+inheriting+two+copies+is+a+tragedy.+Children+born+with+two+copies+of+the+genetic+mutation&amp;source=bl&amp;ots=9LdQL5OKSx&amp;sig=ACfU3U13NQpobFuPP7fVuSf1E8ZO1eLhTA&amp;hl=pt-PT&amp;sa=X&amp;ved=2ahUKEwi1hZ6RlYvuAhVDxIUKHVvOCyoQ6AEwAHoECAEQAg#v=onepage&amp;q=While%20inheriting%20one%20copy%20of%20the%20mutation%20confers%20a%20benefit%2C%20inheriting%20two%20copies%20is%20a%20tragedy.%20Children%20born%20with%20two%20copies%20of%20the%20genetic%20mutation&amp;f=false">Link to quote</a></p>
</blockquote>
<p>There are quite a few examples of this happening. See also the wonderful book <a href="https://www.amazon.com/exec/obidos/ASIN/0060932902">‘Genome’ by Matt Ridley.</a></p>
<h2 id="reed-solomon-codeshttpsbooksgoogleptbookshlpt-ptlridyws55rx1orec-forward-error-correction"><a href="https://books.google.pt/books?hl=pt-PT&amp;lr=&amp;id=yws55Rx1orEC">Reed-Solomon codes</a>: ‘Forward Error Correction’</h2>
<p>Like computer storage, DNA (and its intermediate ‘RNA’) can get corrupted. To protect against common ‘single bit errors’, the encoding from individual DNA letters to proteins is degenerate. There are 4 RNA characters, U, C, G and A - in other words, a ‘byte’ is 2 bits long. Three characters correspond to an amino acid.</p>
<center>
<figure>
    <img src="https://berthub.eu/articles/amazing-dna/nucleotides.jpg"/> 
</figure>

</center>
<p>6 bits could conceivably map to 64 amino acids, yet there are only 20 in use. For example, UCU, UCC, UCA and UCG all encode for ‘Serine’, whereas only UGG maps to ‘Tryptophan’.
Now, it turns out that some likely ‘typos’ (UCU -&gt; UCC) in the encoding lead to an identical amino acid being expressed. For more about this fascinating phenomenon, read <a href="https://www.amazon.com/exec/obidos/ASIN/0465045669">‘Metamagical Themas’ by Douglas Hofstadter</a>.</p>
<h2 id="holy-code--you-are-not-expected-to-understand-thishttpswebarchiveorgweb20080101101354httpcmbell-labscomcmcswhodmroddhtml-">Holy Code: /* <a href="https://web.archive.org/web/20080101101354/http://cm.bell-labs.com/cm/cs/who/dmr/odd.html">You are not expected to understand this</a>. */</h2>
<p><a href="https://www.reddit.com/r/ProgrammerHumor/comments/a482un/the_base_code_is_sacred_and_cannot_be_questioned/">Some code is sacred</a>. We may not remember who wrote it, or why - we just know that it works. The guy who thought it up may have left the company already. Such code is not to be tinkered with.</p>
<p>DNA knows the concept of the ‘molecular clock’. Some parts of the genome are actively changing and some parts are sacrosanct. A good example of the latter are the Histone genes H3 and H4.</p>
<p>These genes are fundamental to the actual storage of the genome and are thus of paramount importance. Any failure in this code rapidly leads to a non-functioning organism.</p>
<p>So it is to be expected that this code isn’t tinkered with and that turns out the case. The H3 an H4 genes have a <em>zero</em> effective mutation rate in humans. But it goes far beyond that. You share almost the exact same code with anything from chickens to grass or molds.</p>
<table><tbody><tr><td></td><td>
RATES OF NUCLEOTIDE SUBSTITUTION PER SITE PER <b>1000 MILLION YEARS</b> BETWEEN
VARIOUS HUMAN AND RODENT PROTEINS-CODING GENES WITH DIVERGENCE SET AT
80 MILLION YEARS BASED ON FOSSIL EVIDENCE:<table><tbody><tr><td>gene</td><td>Number of codons</td><td>Effective rate</td>
</tr><tr><td>histone 3</td><td>135</td><td>0.00</td>
</tr><tr><td>histone 4</td><td>101</td><td>0.00</td>
</tr><tr><td>insulin</td><td>51</td><td>0.13</td>
</tr><tr><td>gamma interferon</td><td>136</td><td>2.79</td>
</tr></tbody></table>
</td><td></td></tr><tr><td></td>
<td>
<p>
<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC117386/">Source</a>
</p></td></tr></tbody></table>
<p>Now, it does appear that there are two ways the genome can make sure that code does not mutate. The first way is described above: use amino acids that are highly degenerate and making sure that those typos that DO occur result in the same output.</p>
<p>Furthermore, genes can be copied earlier or later in the cell’s reproductive process, leading to more or less favorable copying conditions. Many more of such conditions apply.</p>
<p>It appears as if H3 and H4 were authored very carefully as they do have a lot of ‘synonymous changes’, which through the clever techniques described above do not lead to changes in the output.</p>
<h2 id="framing-errors-start-and-stop-bits">Framing errors: start and stop bits</h2>
<p><code>...0 0000 0001   0000 0010  0000 0011 0...</code></p>
<p>This clearly describes the 8 bit values 1, 2 and 3. The spaces I added make it clear where a byte starts and stops. Many serial devices employ stop and start bits to encode where you start reading. If we shift this sequence slightly:</p>
<p><code>...00 0000 0010   000 00100  000 00110 ...</code></p>
<p>It suddenly reads 2, 4, 6! To prevent this from happening in DNA there are elaborate signals that tell the cell where to start reading. Interestingly, there are pieces of genome that can be read from multiple starting points, and produce useful (but different) results either way. That is what I call a cool hack!</p>
<p>Each way a strand of DNA can be read is called an <a href="http://bioweb.uwlax.edu/GenWeb/Molecular/Seq_Anal/Translation/translation.html">Open Reading Frame</a> and there are generally 6, 3 each way.</p>
<h2 id="massive-multiprocessing-each-cell-is-a-universe">Massive multiprocessing: each cell is a universe</h2>
<p>Now, DNA is not like a computer programming language. It really isn’t. But there are some whopping analogies. We can view each cell as a CPU, running its own kernel. Each cell has a copy of the entire kernel, but choses to activate only the relevant parts. Which modules or drivers it loads, so to speak.</p>
<center>
<figure>
    <img src="https://berthub.eu/articles/amazing-dna/cell.jpg" alt="A cell"/> <figcaption>
            <p>A cell</p>
        </figcaption>
</figure>

</center>
<p>If a cell needs to do something (‘call a function’), it whips up the right piece of the genome and transcribes it into RNA. The RNA is then translated into a sequence of amino acids, which together make up a protein the DNA coded for. Now for the really cool bit :-)</p>
<p>This protein is tagged with a shipping address. This is a marker consisting of several amino acids which tell the rest of the cell where this protein needs to go. There is machinery which acts on these instructions, and delivers the protein, which is potentially on the outside of the cell.</p>
<p>The delivery instruction is then stripped off and several post processing steps may be performed, possibly activating the protein - which is good, because you may not want to transport an active protein through places where it should not do work.</p>
<h2 id="self-hosting--bootstrapping">Self hosting &amp; bootstrapping</h2>
<p>If we were to destroy all existing C compilers on the planet and leave only the code for one, we would be in great trouble. Yes, we have the C code to a C compiler, but we need a C compiler to compile it!</p>
<p>In actual fact, this was solved by not writing the first C compiler in C (duh), but in a language that was available already: B. See <a href="https://web.archive.org/web/20080113133738/cm.bell-labs.com/cm/cs/who/dmr/chist.html">here</a> for details about ‘bootstrapping’.</p>
<p>The same holds for the genome. To create a new ‘binary’ of a specimen, a <em>living</em> copy is required. The genome needs an elaborate toolchain in order to deliver a living thing. The code itself is impotent. This toolchain is commonly called ‘your parents’.</p>
<p>Update: Recently, it has become possible to ‘bootstrap’ life with very little actually living source material. The dictum “every cell comes from a cell” is becoming less true. See for example Mycoplasma laboratory.</p>
<p>It appears that RNA, which is an intermediate code between DNA and a protein, may have been the ‘B’ for DNA. Which begs the question where RNA came from. It is very interesting to note that extra-terrestrial objects often contain amino acids! See <a href="http://www.google.com/search?hl=en&amp;q=amino+acids+meteorites">http://www.google.com/search?hl=en&amp;q=amino+acids+meteorites</a></p>
<h2 id="the-makefile">The Makefile</h2>
<center>
<figure>
    <img src="https://berthub.eu/articles/amazing-dna/homeobox.gif"/> 
</figure>

</center><p>
Organisms typically start out as a single cell, which as said before contains two entire copies of the genome. The big tarfile so to speak, with all files extracted, ready to go. Now what?
</p><p>Enter the Homeobox genes. Cells must be copied and assigned a purpose. The Homeobox genes start out by laying a ‘top to bottom’ dependency which reads ‘start with the head’. In order to make this happen, a chemical gradient is created by which cells can sense where they are, and decide if they need to do things useful for building a head, or for building a primordial notochord.</p>
<p>Only discovered in 1983, the <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2561923/">Homeobox genes</a> are a very exciting area of research right now. It is interesting to note that like a Makefile, ‘HOX’ genes only trigger things in other genes and don’t materially build things themselves.</p>
<p>The homeobox ‘syntax’ appears to be very ‘holy’ in the sense described above. What happens if you copy paste the ‘legs selector’ part of a mouse HOX gene into the fruit fly Homeobox:</p>
<blockquote>
<p>“In fact, when the mouse Hox-B6 gene is inserted in Drosophila, it can substitute for Antennapedia and produce legs in place of antennae”</p>
<p><a href="https://biology-pages.info/H/HomeoboxGenes.html">https://biology-pages.info/H/HomeoboxGenes.html</a></p>
</blockquote>
<p>The fruitfly and human genomes did not branch just millions of years ago but <strong>hundreds of millions of years ago</strong>. And you can copy paste parts (‘Selectors’ in the genetic language) of the Makefile and it still clicks. Please note that the ‘build a leg’ routine in a fruit fly is of course radically different from that in a mouse, but the ‘selector’ correctly triggers the right instructions.</p>
<h2 id="plugins-plasmids">Plugins: Plasmids</h2>
<p>All living organisms have DNA, sometimes organized into multiple chromosomes (‘libraries’), sometimes in only one, typically circular in that case. This goes for most bacteria. Next to this large main genome, such bacteria frequently also host ‘plasmids’: tiny circles of DNA with specific functions.</p>
<p>Such plasmids are somewhat portable between species, and through a variety of mechanisms they do indeed get transferred horizontally. By this way even non-identical bacteria can ‘learn’ antibiotic resistance from each other, for example.</p>
<p>Compared to the programming world, a plasmid is not voluntary, and is like LD_PRELOADing a .so or the equivalent on other platforms. And in fact, plasmids are frequently injected for research purposes. They can easily be injected in all kinds of bacteria, and immediately get to work.</p>
<p>Plasmids copy themselves independently from the main chromosome, and are thus a permanent fixture of bacteria. To make this happen, a plasmid features the magnificently named <strong>&#39;<a href="https://en.wikipedia.org/wiki/Origin_of_replication">Origin of Replication</a>&#39;</strong> gene which gets triggered when the cell wants to divide.</p>
<h2 id="further-reading">Further reading</h2>
<p><strong><a href="https://www.amazon.com/exec/obidos/ASIN/0060932902">Genome</a> by Matt Ridley</strong></p>
<p>An amazing account of an effect each chromosome has on our lives. Very readable yet strict in not ‘dumbing down’ the theory. Contains an impressive set of references.
Source of many of the more impressive examples found on this page.
And to help Matt along in the quest he clearly sets out in his book, I would like to state quite clearly:
<strong>Genes are not there to cause diseases</strong></p>
<p><strong><a href="https://www.amazon.com/exec/obidos/ASIN/0471330612">Human Molecular Genetics, second edition</a> by Tom Strachan and Andrew P. Read</strong></p>
<p>Neatly fills the gap between ‘primary literature’ (ie, peer reviewed academic magazines and papers) and introductory textbooks. I’m literally dragging myself through this book, constantly looking things up in order to understand everything. If you really want to know the details about introns, exons, RNA in all its variants, how genes cause and prevent diseases, this is the book.</p>
<p><strong><a href="https://www.amazon.com/exec/obidos/ASIN/0192860925">The Selfish Gene</a> by Richard Dawkins</strong></p>
<p><a href="https://en.wikipedia.org/wiki/Richard_Dawkins">Richard Dawkins</a> is the <a href="https://en.wikipedia.org/wiki/W._Richard_Stevens">Richard Stevens</a> of evolution theory. Both have contributed practical work but are most famous for their crystal clear expositions of existing theory, opening up the world they describe to an audience of millions.</p>
<p><strong><a href="https://www.amazon.com/exec/obidos/ASIN/0393315703">The Blind Watchmaker : Why the Evidence of Evolution Reveals a Universe Without Design</a> by Richard Dawkins</strong></p>
<p>Again a book by Dawkins. More about evolution than about genes but clearly explains how evolution can be responsible for the intricate design found in many living things.
Again very readable and fascinating on every level.</p>
<p><strong><a href="https://www.amazon.com/exec/obidos/ASIN/0465045669">Metamagical Themas</a> by Douglas Hofstadter</strong></p>
<p>This is an ‘idea’ book. It is filled to the brim with ideas, they simply ooze out of the pages. Many of these ideas are about information theory, genetics, life, intelligence, music, mathematics and people.
Clearly not a genetic textbook but has been influential in imbuing enthusiasm for all things genetic in many people. Can often be found dirt cheap in second hand bookstores.
Recommended.</p>

</div>

  



</article></div>
  </body>
</html>
