<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <title>James Routley | Feed</title>
    <link
      rel="stylesheet"
      type="text/css"
      href="../styles.css"
      media="screen"
    />
  </head>
  <body>
    <a href="/index.html">Back</a>
    <a href="https://www.nature.com/articles/s41467-025-60001-2">Original</a>
    <h1>Efficient mRNA delivery to resting T cells to reverse HIV latency</h1>
    
    <div id="readability-page-1" class="page"><div>
                    
                        <section data-title="Introduction"><div id="Sec1-section"><h2 id="Sec1">Introduction</h2><div id="Sec1-content"><p>Recent advances in mRNA and lipid nanoparticle (LNP) technology have allowed for the development of new vaccines and therapeutics, holding great promise for gene therapy. In 2018, patisiran (Onpattro) was approved as the first LNP-based therapeutic for the delivery of silencing (si)RNA for transthyretin-mediated amyloidosis<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 1" title="Luigetti, M. &amp; Servidei, S. Patisiran in hereditary transthyretin-mediated amyloidosis. Lancet Neurol. 20, 21–23 (2021)." href="https://www.nature.com/articles/s41467-025-60001-2#ref-CR1" id="ref-link-section-d326275717e1000">1</a></sup>. The same platform has since been used to deliver mRNA to generate two of the most effective vaccines against COVID-19 (Comirnaty (Pfizer/BioNTech) and Spikevax (Moderna)). More recently, LNPs have been used to deliver mRNA for CRISPR-Cas9 gene editing in vivo, which was found to be safe in human clinical trials<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 2" title="Gillmore, J. D. et al. CRISPR-Cas9 in vivo gene editing for transthyretin amyloidosis. N. Engl. J. Med. 385, 493–502 (2021)." href="https://www.nature.com/articles/s41467-025-60001-2#ref-CR2" id="ref-link-section-d326275717e1004">2</a></sup>. These advances have triggered enormous interest in the use of mRNA-LNPs as versatile therapeutics, including for infectious diseases such as HIV.</p><p>Whilst antiretroviral therapy (ART) effectively inhibits active HIV replication and reduces morbidity and mortality, it is not curative and treatment is life-long<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 3" title="Deeks, S. G. et al. Research priorities for an HIV cure: International AIDS Society Global Scientific Strategy 2021. Nat. Med. 27, 2085–2098 (2021)." href="https://www.nature.com/articles/s41467-025-60001-2#ref-CR3" id="ref-link-section-d326275717e1011">3</a></sup>. The major barrier to HIV cure is the persistence of latently infected, resting CD4<sup>+</sup> T cells harbouring replication-competent virus<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Chun, T.-W. et al. Presence of an inducible HIV-1 latent reservoir during highly active antiretroviral therapy. Proc. Natl Acad. Sci. USA94, 13193–13197 (1997)." href="#ref-CR4" id="ref-link-section-d326275717e1017">4</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Finzi, D. et al. Identification of a reservoir for HIV-1 in patients on highly active antiretroviral therapy. Science 278, 1295–1300 (1997)." href="#ref-CR5" id="ref-link-section-d326275717e1017_1">5</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 6" title="Wong, J. K. et al. Recovery of replication-competent HIV despite prolonged suppression of plasma viremia. Science 278, 1291–1295 (1997)." href="https://www.nature.com/articles/s41467-025-60001-2#ref-CR6" id="ref-link-section-d326275717e1020">6</a></sup>, which can rebound following T cell activation and re-establish viremia in the absence of ART. One approach towards an HIV cure is to reactivate HIV transcription using latency-reversing agents (LRAs) while on ART, with the goal of subsequently inducing death of the infected cells and reduction of the HIV reservoir through viral cytopathic effects or immune-mediated clearance of the infected cell<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Archin, N. M. et al. Administration of vorinostat disrupts HIV-1 latency in patients on antiretroviral therapy. Nature 487, 482–485 (2012)." href="https://www.nature.com/articles/s41467-025-60001-2#ref-CR7" id="ref-link-section-d326275717e1024">7</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 8" title="Elliott, J. H. et al. Activation of HIV transcription with short-course vorinostat in HIV-infected patients on suppressive antiretroviral therapy. PLoS Pathog. 10, 1–19 (2014)." href="https://www.nature.com/articles/s41467-025-60001-2#ref-CR8" id="ref-link-section-d326275717e1027">8</a></sup>. Traditionally, this has been achieved using small, hydrophobic compounds, that target different cellular pathways to activate transcription<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 9" title="Kim, Y., Anderson, J. L. &amp; Lewin, S. R. Getting the “kill” into “shock and kill”: strategies to eliminate latent HIV. Cell Host Microbe 23, 14–26 (2018)." href="https://www.nature.com/articles/s41467-025-60001-2#ref-CR9" id="ref-link-section-d326275717e1031">9</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 10" title="Zerbato, J. M., Purves, H. V., Lewin, S. R. &amp; Rasmussen, T. A. Between a shock and a hard place: challenges and developments in HIV latency reversal. Curr. Opin. Virol. 38, 1–9 (2019)." href="https://www.nature.com/articles/s41467-025-60001-2#ref-CR10" id="ref-link-section-d326275717e1034">10</a></sup>. These first-generation LRAs, such as histone deacetylase inhibitors, can be administered systemically and passively cross the plasma membrane to exert their effect intracellularly. Several LRAs have demonstrated induction of HIV RNA in vitro, ex vivo and in clinical trials; yet, to date, no clinical trial of an LRA alone has shown a reduction in the size of the HIV reservoir<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Archin, N. M. et al. Administration of vorinostat disrupts HIV-1 latency in patients on antiretroviral therapy. Nature 487, 482–485 (2012)." href="https://www.nature.com/articles/s41467-025-60001-2#ref-CR7" id="ref-link-section-d326275717e1039">7</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 8" title="Elliott, J. H. et al. Activation of HIV transcription with short-course vorinostat in HIV-infected patients on suppressive antiretroviral therapy. PLoS Pathog. 10, 1–19 (2014)." href="https://www.nature.com/articles/s41467-025-60001-2#ref-CR8" id="ref-link-section-d326275717e1042">8</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Fidler, S. et al. Antiretroviral therapy alone versus antiretroviral therapy with a kick and kill approach, on measures of the HIV reservoir in participants with recent HIV infection (the RIVER trial): a phase 2, randomised trial. Lancet 395, 888–898 (2020)." href="#ref-CR11" id="ref-link-section-d326275717e1045">11</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Elliott, J. H. et al. Short-term administration of disulfiram for reversal of latent HIV infection: a phase 2 dose-escalation study. Lancet HIV 2, e520–e529 (2015)." href="#ref-CR12" id="ref-link-section-d326275717e1045_1">12</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Rasmussen, T. A. et al. Panobinostat, a histone deacetylase inhibitor, for latent-virus reactivation in HIV-infected patients on suppressive antiretroviral therapy: a phase 1/2, single group, clinical trial. Lancet HIV 1, e13–21 (2014)." href="#ref-CR13" id="ref-link-section-d326275717e1045_2">13</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Gutierrez, C. et al. Bryostatin-1 for latent virus reactivation in HIV-infected patients on antiretroviral therapy. AIDS 30, 1385–1392 (2016)." href="#ref-CR14" id="ref-link-section-d326275717e1045_3">14</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 15" title="Sogaard, O. S. et al. The depsipeptide romidepsin reverses HIV-1 latency in vivo. PLoS Pathog. 11, e1005142 (2015)." href="https://www.nature.com/articles/s41467-025-60001-2#ref-CR15" id="ref-link-section-d326275717e1048">15</a></sup>.</p><p>There are several hypotheses as to why LRAs alone have not been able to induce clearance of infected cells. First, most first-generation LRAs only increase the initiation of HIV transcription (as measured by unspliced HIV RNA), but fail to overcome subsequent blocks in transcription elongation, completion and splicing that persist in a resting CD4<sup>+</sup> T cell<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 10" title="Zerbato, J. M., Purves, H. V., Lewin, S. R. &amp; Rasmussen, T. A. Between a shock and a hard place: challenges and developments in HIV latency reversal. Curr. Opin. Virol. 38, 1–9 (2019)." href="https://www.nature.com/articles/s41467-025-60001-2#ref-CR10" id="ref-link-section-d326275717e1057">10</a></sup>. Second, first-generation LRAs are not HIV-specific, and therefore their potency to reactivate HIV is unavoidably coupled to off-target and adverse effects<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 8" title="Elliott, J. H. et al. Activation of HIV transcription with short-course vorinostat in HIV-infected patients on suppressive antiretroviral therapy. PLoS Pathog. 10, 1–19 (2014)." href="https://www.nature.com/articles/s41467-025-60001-2#ref-CR8" id="ref-link-section-d326275717e1061">8</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 16" title="Uldrick, T. S. et al. Assessment of the safety of pembrolizumab in patients with HIV and advanced cancer-a phase 1 study. JAMA Oncol. 5, 1332–1339 (2019)." href="https://www.nature.com/articles/s41467-025-60001-2#ref-CR16" id="ref-link-section-d326275717e1064">16</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 17" title="McMahon, J. H. et al. Neurotoxicity with high-dose disulfiram and vorinostat used for HIV latency reversal. AIDS 36, 75–82 (2022)." href="https://www.nature.com/articles/s41467-025-60001-2#ref-CR17" id="ref-link-section-d326275717e1067">17</a></sup>. Indeed, certain LRAs have been shown to directly inhibit CD8<sup>+</sup> T cell and natural killer cell function in vitro, potentially hampering immune-mediated clearance of reactivated cells<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 18" title="Pace, M. et al. Histone deacetylase inhibitors enhance CD4 T cell susceptibility to NK cell killing but reduce NK cell function. PLoS Pathog. 12, e1005782 (2016)." href="https://www.nature.com/articles/s41467-025-60001-2#ref-CR18" id="ref-link-section-d326275717e1073">18</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 19" title="Jones, R. B. et al. Histone deacetylase inhibitors impair the elimination of HIV-infected cells by cytotoxic T-lymphocytes. PLoS Pathog. 10, e1004287 (2014)." href="https://www.nature.com/articles/s41467-025-60001-2#ref-CR19" id="ref-link-section-d326275717e1076">19</a></sup>. Therefore, there is a need for new LRAs that have greater potency, lower toxicity, and greater specificity for the HIV provirus.</p><p>Recently, two HIV-specific nucleic acid-based LRAs have been reported. The first is an LNP encapsulating mRNA encoding the HIV protein Trans-activator of Transcription (Tat), which binds to the trans-activation response element (TAR) in nascent HIV transcripts produced from the HIV long terminal repeat (LTR) promotor region and potently enhances transcriptional processivity<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Das, A. T., Harwig, A. &amp; Berkhout, B. The HIV-1 tat protein has a versatile role in activating viral transcription. J. Virol. 85, 9506–9516 (2011)." href="#ref-CR20" id="ref-link-section-d326275717e1083">20</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Raines, S. L. M. et al. Nanoparticle delivery of Tat synergizes with classical latency reversal agents to express HIV antigen targets. Antimicrob. Agents Chemother. 68, e00201-24 (2024)." href="#ref-CR21" id="ref-link-section-d326275717e1083_1">21</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 22" title="Van Gulck, E. et al. A truncated HIV Tat demonstrates potent and specific latency reversal activity. Antimicrob. Agents Chemother. 67, e0041723 (2023)." href="https://www.nature.com/articles/s41467-025-60001-2#ref-CR22" id="ref-link-section-d326275717e1086">22</a></sup>. Although this Tat-LNP could reverse latency both in vitro and ex vivo<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 21" title="Raines, S. L. M. et al. Nanoparticle delivery of Tat synergizes with classical latency reversal agents to express HIV antigen targets. Antimicrob. Agents Chemother. 68, e00201-24 (2024)." href="https://www.nature.com/articles/s41467-025-60001-2#ref-CR21" id="ref-link-section-d326275717e1090">21</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 22" title="Van Gulck, E. et al. A truncated HIV Tat demonstrates potent and specific latency reversal activity. Antimicrob. Agents Chemother. 67, e0041723 (2023)." href="https://www.nature.com/articles/s41467-025-60001-2#ref-CR22" id="ref-link-section-d326275717e1093">22</a></sup>, a small molecule non-specific LRA was required to achieve high potency—an observation consistent with suboptimal delivery of the Tat mRNA<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 21" title="Raines, S. L. M. et al. Nanoparticle delivery of Tat synergizes with classical latency reversal agents to express HIV antigen targets. Antimicrob. Agents Chemother. 68, e00201-24 (2024)." href="https://www.nature.com/articles/s41467-025-60001-2#ref-CR21" id="ref-link-section-d326275717e1097">21</a></sup>. The second is HIV LTR-targeted CRISPR activation (CRISPRa), comprising of a catalytically inactive Cas9 protein combined with transcriptional activator domains, which provides a promising strategy for highly HIV-specific activation of transcription without affecting host-cell transcription<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Zhang, Y. et al. CRISPR/gRNA-directed synergistic activation mediator (SAM) induces specific, persistent and robust reactivation of the HIV-1 latent reservoirs. Sci. Rep. 5, 16277–16277 (2015)." href="#ref-CR23" id="ref-link-section-d326275717e1101">23</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Bialek, J. K. et al. Targeted HIV-1 latency reversal using CRISPR/Cas9-derived transcriptional activator systems. PLoS One 11, e0158294–e0158294 (2016)." href="#ref-CR24" id="ref-link-section-d326275717e1101_1">24</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Ji, H. et al. Specific reactivation of latent HIV-1 by dCas9-SunTag-VP64-mediated guide RNA targeting the HIV-1 promoter. Mol. Ther. 24, 508–521 (2016)." href="#ref-CR25" id="ref-link-section-d326275717e1101_2">25</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Limsirichai, P., Gaj, T. &amp; Schaffer, D. V. CRISPR-mediated activation of latent HIV-1 expression. Mol. Ther. 24, 499–507 (2016)." href="#ref-CR26" id="ref-link-section-d326275717e1101_3">26</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Saayman, S. M. et al. Potent and targeted activation of latent HIV-1 using the CRISPR/dCas9 activator vomplex. Mol. Ther. 24, 488–498 (2016)." href="#ref-CR27" id="ref-link-section-d326275717e1101_4">27</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 28" title="Klinnert, S., Chemnitzer, A., Rusert, P. &amp; Metzner, K. J. Systematic HIV-1 promoter targeting with CRISPR/dCas9-VPR reveals optimal region for activation of the latent provirus. J. Gen. Virol. 103, 1–10 (2022)." href="https://www.nature.com/articles/s41467-025-60001-2#ref-CR28" id="ref-link-section-d326275717e1104">28</a></sup>. However, both these novel LRAs have not progressed to primary cells or the clinic due to the lack of an efficient delivery vehicle to resting CD4<sup>+</sup> T cells, which are known to be recalcitrant to traditional gene delivery systems<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 29" title="Cevaal, P. M. et al. In Vivo T cell-targeting nanoparticle drug delivery systems: considerations for rational design. ACS Nano 15, 3736–3753 (2021)." href="https://www.nature.com/articles/s41467-025-60001-2#ref-CR29" id="ref-link-section-d326275717e1111">29</a></sup>.</p><p>We therefore aimed to develop an LNP capable of delivering mRNA to resting T cells for potent, HIV-specific latency reversal. We identified an LNP formulation with the unique capability to transfect CD4<sup>+</sup> T cells in the absence of pre-stimulation. We then used this LNP formulation, LNP X, to encapsulate two mRNA-based LRAs encoding for HIV Tat and CRISPRa, which demonstrated efficient activation of HIV transcription in CD4<sup>+</sup> T cells from people living with HIV on ART ex vivo. Together, these findings provide an exciting new approach to mRNA-based therapeutics for T cells.</p></div></div></section><section data-title="Results"><div id="Sec2-section"><h2 id="Sec2">Results</h2><div id="Sec2-content"><h3 id="Sec3">LNP X enables delivery of mRNA to resting T cells</h3><p>We first assessed whether an LNP similar to an existing, FDA-approved LNP formulation (patisiran) was able to transfect non-stimulated primary CD4<sup>+</sup> T cells by delivering a reporter mCherry mRNA. Our patisiran-like LNP (hereafter referred to as patisiran LNP) is formulated with DMG-PEG2000 instead of PEG2000-C-DMG. At the highest dose tested of 500 ng per 10<sup>5</sup> cells, mCherry expression was only detected in 2.1 ± 0.4% (mean ± SEM) of live cells (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41467-025-60001-2#Fig1">1a</a> and Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41467-025-60001-2#MOESM1">S1a</a>) after 72 h incubation. Pre-stimulation of the CD4<sup>+</sup> T cells with anti-CD3/anti-CD28 led to substantially higher transfection efficiencies of up to 51 ± 5.1% compared to non-stimulated T cells (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41467-025-60001-2#Fig1">1a</a>), though the toxicity associated with patisiran LNP treatment was greater in pre-stimulated compared to non-stimulated CD4<sup>+</sup> T cells at higher LNP doses (Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41467-025-60001-2#MOESM1">S1b</a>). These data show that successful mRNA delivery to primary CD4<sup>+</sup> T cells using patisiran LNPs is dependent on the activation state of the T cell.</p><div data-test="figure" data-container-section="figure" id="figure-1" data-title="LNP X formulation potently transfects primary CD4+ T cells in the absence of pre-stimulation."><figure><figcaption><b id="Fig1" data-test="figure-caption-text">Fig. 1: LNP X formulation potently transfects primary CD4<sup>+</sup> T cells in the absence of pre-stimulation.</b></figcaption><div><div><a data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="https://www.nature.com/articles/s41467-025-60001-2/figures/1" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41467-025-60001-2/MediaObjects/41467_2025_60001_Fig1_HTML.png?as=webp"/><img aria-describedby="Fig1" src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41467-025-60001-2/MediaObjects/41467_2025_60001_Fig1_HTML.png" alt="figure 1" loading="lazy" width="685" height="456"/></picture></a></div><p><b>a</b> CD4<sup>+</sup> T cells from HIV-negative donors were rested or pre-stimulated with anti-CD3/anti-CD28, then treated with LNPs (similar to the patisiran lipid composition) encapsulating mCherry mRNA for 72 h at the indicated doses. Highest dose of mCherry-LNP X corresponds to 2.5 µg/mL. Transfection efficiency was determined by measuring mCherry expression using flow cytometry. Mean ± SEM, <i>n</i> = 6 donors. <b>b</b> Schematic representation of patisiran-like LNP formulation <i>versus</i> novel LNP formulation X. Mol %, relative molar percentage of indicated lipid in lipid mixture. Created in BioRender: <i>Cevaal, P. (2025)</i> <a href="https://BioRender.com/ypaicaz">https://BioRender.com/ypaicaz</a>. <b>c</b> Size distribution of patisiran LNP (yellow) <i>versus</i> LNP X (green) encapsulating mCherry mRNA. Lines represent individual LNP batches. <b>d</b> Relative mCherry expression levels in Jurkat T cells treated for 24 h with indicated doses of patisiran LNP (yellow) or LNP X (green) encapsulating mCherry mRNA. MFI, median fluorescent intensity. Mean ± SEM, n = 2. <b>e</b> CD4<sup>+</sup> T cells from HIV-negative donors were rested or pre-stimulated with anti-CD3/anti-CD28, then treated with LNP X encapsulating mCherry mRNA for 72 h at the indicated doses. Highest dose of mCherry-LNP X corresponds to 2.5 µg/mL. Transfection efficiency was determined using flow cytometry. Mean ± SEM, <i>n</i> = 6 donors.</p></div></figure></div><p>We therefore modified the lipid composition of the LNP to enhance potency. First, the ionisable lipid DLin-MC3-DMA (MC3) was replaced with SM-102, an ionisable lipid previously shown to lead to greater cytosolic mRNA delivery through enhanced endosomal escape<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 30" title="Sabnis, S. et al. A novel amino lipid series for mRNA delivery: improved endosomal escape and sustained pharmacology and safety in non-human primates. Mol. Ther. 26, 1509–1519 (2018)." href="https://www.nature.com/articles/s41467-025-60001-2#ref-CR30" id="ref-link-section-d326275717e1226">30</a></sup>. Second, the SM-102-LNPs were further modified using ß-sitosterol, a naturally-occurring cholesterol analogue associated with enhanced mRNA delivery<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 31" title="Patel, S. et al. Naturally-occurring cholesterol analogues in lipid nanoparticles induce polymorphic shape and enhance intracellular delivery of mRNA. Nat. Commun. 11, 983 (2020)." href="https://www.nature.com/articles/s41467-025-60001-2#ref-CR31" id="ref-link-section-d326275717e1230">31</a></sup>, to create a formulation referred to as LNP X (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41467-025-60001-2#Fig1">1b</a>). LNP X encapsulating reporter mCherry mRNA was formulated reproducibly and did not significantly differ in size, polydispersity or mRNA encapsulation efficiency compared to patisiran LNP (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41467-025-60001-2#Fig1">1c</a> and Table <a data-track="click" data-track-label="link" data-track-action="table anchor" href="https://www.nature.com/articles/s41467-025-60001-2#Tab1">1</a>).</p><div data-test="inline-table" data-container-section="table" id="table-1"><figure><figcaption><b id="Tab1" data-test="table-caption">Table. 1 Characterisation of patisiran LNP and LNP X</b></figcaption></figure></div><p>We assessed the potency of LNP X by treating Jurkat T cells, an immortalised T cell line, with increasing doses of reporter mRNA encapsulated in either patisiran LNP or LNP X. Treatment with LNP X compared to patisiran LNP resulted in up to 6-fold higher mCherry expression (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41467-025-60001-2#Fig1">1d</a>), confirming its superior potency. To assess the potency of our LNP X formulation further, we treated pre-stimulated CD4<sup>+</sup> T cells with LNP X and observed mCherry expression in &gt;75% of cells, in all doses tested, including a low dose of 6.25 ng mCherry-LNP X per 10<sup>5</sup> cells (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41467-025-60001-2#Fig1">1e</a> and Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41467-025-60001-2#MOESM1">S1a</a>). Impressively, and in stark contrast to patisiran LNP, LNP X was able to achieve transfection of up to 76 ± 3.8% CD4<sup>+</sup> T cells in the absence of pre-stimulation (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41467-025-60001-2#Fig1">1a–e</a>). Moreover, toxicity was negligible compared to baseline in both pre-stimulated and non-stimulated CD4<sup>+</sup> T cells (Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41467-025-60001-2#MOESM1">S1c</a>). To our knowledge, this is the first demonstration of potent transfection of primary, resting T cells in vitro in the absence of cellular toxicity.</p><p>LNP X was able to transfect both naïve and all memory T cell subsets (Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41467-025-60001-2#MOESM1">S2a–d</a>), with effector memory T cells expressing higher levels of mCherry than naïve T cells (<i>p</i> = 0.007), potentially reflecting the higher basal translational activity in this memory T cell subset. Even in the context of peripheral mononuclear cells (PBMCs), LNP X was able to deliver mRNA to CD4<sup>+</sup> T cells (Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41467-025-60001-2#MOESM1">S2e, f</a>). Expression of mRNA was also detected in most other PBMC subsets, specifically monocytes, demonstrating that LNP X is T cell tropic, but not T cell specific.</p><h3 id="Sec4">Superior potency of LNP X is not related to endosomal escape</h3><p>To understand the mechanism behind the superior potency of LNP X, we employed a SNAP<sub>switch</sub> reporter system to quantify sub-cellular localisation of therapeutics in Jurkat T cells<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 32" title="FitzGerald, L. I. et al. A molecular sensor to quantify the localization of proteins, DNA and nanoparticles in cells. Nat. Commun. 11 4482 (2020)." href="https://www.nature.com/articles/s41467-025-60001-2#ref-CR32" id="ref-link-section-d326275717e1449">32</a></sup> (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41467-025-60001-2#Fig2">2a</a>). SNAP<sub>switch</sub> was recently used to probe the role that ionisable lipids play in delivering mRNA to the cytosol. This work showed that SM-102 had similar cytosolic delivery compared to DLin-MC3-DMA, the ionisable lipid found in patisiran LNP, however the protein expression induced by SM-102 was significantly higher<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 33" title="Liu, H. et al. Beyond the endosomal bottleneck: understanding the efficiency of mRNA/LNP delivery. Adv. Funct. Mater. 34, 2404510 (2024)." href="https://www.nature.com/articles/s41467-025-60001-2#ref-CR33" id="ref-link-section-d326275717e1458">33</a></sup>. To probe LNP X’s superior potency, we formulated SNAP<sub>switch</sub> LNP X and compared these to LNPs formulated with SM-102, DSPC, DMG-PEG2000 and cholesterol. SNAP<sub>switch</sub> LNP X did not exhibit different physicochemical characteristics compared to LNP X encapsulating a single reporter mRNA (Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41467-025-60001-2#MOESM1">S3</a>). The LNP X formulation resulted in 10.5-fold enhanced mRNA expression in SNAP<sub>switch</sub> Jurkat T cells (<i>p</i> = 0.0029, Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41467-025-60001-2#Fig2">2b</a>). This enhanced transfection was in part explained by a 2.5-fold increase in LNP association, representing LNPs bound to the cell surface or internalised (<i>p</i> = 0.0025, Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41467-025-60001-2#Fig2">2c</a>), yet LNP X still exhibited 4.1-fold higher protein expression relative to the amount of LNP association (<i>p</i> = 0.0032, Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41467-025-60001-2#Fig2">2d</a>).</p><div data-test="figure" data-container-section="figure" id="figure-2" data-title="Superior potency of LNP X is not explained by differences in endosomal escape."><figure><figcaption><b id="Fig2" data-test="figure-caption-text">Fig. 2: Superior potency of LNP X is not explained by differences in endosomal escape.</b></figcaption><div><div><a data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="https://www.nature.com/articles/s41467-025-60001-2/figures/2" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41467-025-60001-2/MediaObjects/41467_2025_60001_Fig2_HTML.png?as=webp"/><img aria-describedby="Fig2" src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41467-025-60001-2/MediaObjects/41467_2025_60001_Fig2_HTML.png" alt="figure 2" loading="lazy" width="685" height="465"/></picture></a></div><p><b>a</b> The SNAP<sub>switch</sub> assay simultaneously tracks the degree of nanoparticle association, endosomal escape and mRNA expression of LNPs co-encapsulating an AF488 oligo, Cy5-SNAP<sub>switch</sub> oligo and mRNA encoding mScarlet. Figure adapted from Liu et al.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 33" title="Liu, H. et al. Beyond the endosomal bottleneck: understanding the efficiency of mRNA/LNP delivery. Adv. Funct. Mater. 34, 2404510 (2024)." href="https://www.nature.com/articles/s41467-025-60001-2#ref-CR33" id="ref-link-section-d326275717e1509">33</a></sup> and created in BioRender: <i>Cevaal, P. (2025)</i> <a href="https://BioRender.com/3ajxnam">https://BioRender.com/3ajxnam</a>. <b>b</b>–<b>g</b> SNAP<sub>switch</sub>-reporter Jurkat T cells were incubated with LNPs containing SM-102, DSPC, DMG-PEG2000 and either Cholesterol (Chol) or ß-sitosterol (Sito; LNP X) for 4 h. LNPs encapsulated a reporter mScarlet mRNA, AF488-tagged oligo and a Cy5-SNAP<sub>switch</sub> oligo or constitutively active Cy5-oligo. Fluorescence was determined using flow cytometry. <b>b</b> Protein expression of the mScarlet reporter mRNA. <b>c</b> LNP association as determined by the fluorescence of the AF488-tagged oligo. <b>d</b> mScarlet protein expression (as in <b>b</b>) relative to LNP association (as in <b>c</b>). <b>e</b> Amount of cytosolic delivery of nucleic acid cargo as quantified by the Cy5-SNAP<sub>switch</sub> oligo fluorescence. <b>f</b> Efficiency of endosomal escape based on the Cy5-SNAP<sub>switch</sub> oligo fluorescence (as in <b>e</b>) normalised to a constitutively fluorescent Cy5-tagged oligo. <b>g</b> mScarlet protein expression (as in <b>b</b>) relative to the amount of cytosolic cargo delivery (as in <b>e</b>). <b>b</b>–<b>g</b> Bars represent aggregate mean of <i>n</i> = 3 independent experiments, each symbol representing the average of triplicate technical replicates. Significance was determined using a one-tailed paired ratio t test in (<b>b</b>–<b>e</b>,<b>g</b>) to allow comparisons of MFI values between experiments, or one-tailed unpaired t test in (<b>f</b>), ns non-significant.</p></div></figure></div><p>While LNP X yielded an overall higher level of cytosolic cargo delivery (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41467-025-60001-2#Fig2">2e</a>), this effect correlated with the higher levels of LNP association to cells observed with LNP X. Indeed, the efficiency of endosomal escape was not improved by replacing cholesterol with ß-sitosterol (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41467-025-60001-2#Fig2">2f</a>). Importantly, the protein expression relative to the amount of mRNA delivered to the cytosol was enhanced by 5.3-fold when using LNP X (<i>p</i> = 0.0012, Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41467-025-60001-2#Fig2">2g</a>), suggesting that the superior potency of LNP X involved processes downstream of endosomal escape and cytosolic delivery of the mRNA cargo. Our findings regarding the role of ß-sitosterol are consistent with previous studies comparing DLin-MC3-DMA/cholesterol to DLin-MC3-DMA/ß-sitosterol LNPs<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 33" title="Liu, H. et al. Beyond the endosomal bottleneck: understanding the efficiency of mRNA/LNP delivery. Adv. Funct. Mater. 34, 2404510 (2024)." href="https://www.nature.com/articles/s41467-025-60001-2#ref-CR33" id="ref-link-section-d326275717e1618">33</a></sup>, indicating the observed effects of ß-sitosterol are independent of the ionisable lipid.</p><h3 id="Sec5">LNP X delivers mRNA encoding HIV Tat to reverse HIV latency</h3><p>We next aimed to assess whether LNP X was able to deliver a small, therapeutic mRNA expressing the 72 amino acids of the first coding exon of HIV Tat (Tat-LNP X, Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41467-025-60001-2#Fig3">3a</a> and Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41467-025-60001-2#MOESM1">S1</a>). This form of HIV Tat is normally expressed from the late-phase, Rev-dependent 4 kb viral Tat mRNA<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 34" title="Purcell, D. F. &amp; Martin, M. A. Alternative splicing of human immunodeficiency virus type 1 mRNA modulates viral protein expression, replication, and infectivity. J. Virol. 67, 6365–6378 (1993)." href="https://www.nature.com/articles/s41467-025-60001-2#ref-CR34" id="ref-link-section-d326275717e1636">34</a></sup>. We first validated the potency of Tat-LNP X in J-Lat 10.6 cells, a T cell line model of HIV latency that expresses green fluorescent protein (GFP) under the control of the HIV LTR promoter. Treatment with Tat-LNP X, but not control mCherry-LNP X, resulted in an increase in GFP expression consistent with potent reactivation of LTR-mediated transcription (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41467-025-60001-2#Fig3">3b</a>). No impact on cellular viability was observed (Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41467-025-60001-2#MOESM1">S4</a>). Similarly, Tat-LNP X was able to induce productive infection in primary CD4<sup>+</sup> T cells infected with an HIV reporter virus (pMorpheus-V5<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 35" title="Kim, E. H. et al. Development of an HIV reporter virus that identifies latently infected CD4(+) T cells. Cell Rep. Methods 2, 100238 (2022)." href="https://www.nature.com/articles/s41467-025-60001-2#ref-CR35" id="ref-link-section-d326275717e1649">35</a></sup>; <i>p</i> = 0.0025) similar to levels seen after T cell activation with the mitogens phorbol 12-myristate 13-acetate (PMA) and ionomycin (Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41467-025-60001-2#MOESM1">S5</a>).</p><div data-test="figure" data-container-section="figure" id="figure-3" data-title="mRNA encoding HIV Tat exon 1 delivered by LNP X is a potent activator of HIV transcription."><figure><figcaption><b id="Fig3" data-test="figure-caption-text">Fig. 3: mRNA encoding HIV Tat exon 1 delivered by LNP X is a potent activator of HIV transcription.</b></figcaption><div><div><a data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="https://www.nature.com/articles/s41467-025-60001-2/figures/3" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41467-025-60001-2/MediaObjects/41467_2025_60001_Fig3_HTML.png?as=webp"/><img aria-describedby="Fig3" src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41467-025-60001-2/MediaObjects/41467_2025_60001_Fig3_HTML.png" alt="figure 3" loading="lazy" width="685" height="919"/></picture></a></div><p><b>a</b> Schematic overview of the mechanism of action of HIV Tat on promoting transcription elongation. TAR; trans-activation response element. Created in BioRender: <i>Cevaal, P. (2025)</i> <a href="https://BioRender.com/cz40qwv">https://BioRender.com/cz40qwv</a>. <b>b</b> J-Lat 10.6 cells were treated for 24 h with indicated doses of LNP X encapsulating codon-optimised mRNA expressing the 72 amino acids of the first coding exon of HIV Tat (Tat-LNP X) or mCherry (mCherry-LNP X) as control. Reactivation of HIV LTR-mediated transcription was determined after 24 h by measuring GFP expression. Highest dose of Tat-LNP X corresponds to 1 µg/mL. Treatment with PMA/ionomycin was included as a positive control. Mean ± SEM, <i>n</i> = 3 independent experiments. <b>c</b>–<b>i</b> CD4<sup>+</sup> T cells from people living with HIV on suppressive ART were treated with 200 ng Tat-LNP X or mCherry-LNP X per 10<sup>5</sup> cells (4 µg/mL) or PMA/PHA as a positive control. After 48 h (squares) or 72 h (circles), expression of HIV transcripts TAR (<b>c</b>) LongLTR (<b>d</b>), Pol (<b>e</b>), PolyA (<b>f</b>) and Tat-Rev (<b>g</b>) representing transcription initiation, proximal elongation, distal elongation, completion and splicing, respectively, was determined using digital RT-PCR. Data were normalised to RNA input, then presented as fold-change induction compared to the corresponding non-treated (NT) control. Horizontal dashed line (<b>c</b>–<b>g</b>) represents no change relative to untreated cells. <b>h</b> Cellular toxicity was determined using flow cytometry. Where datapoints are missing in (<b>h</b>), cell input was insufficient to perform an accurate measurement. <b>i</b> After 72 h, the number of copies of HIV RNA per mL of supernatant was quantified using RT-PCR and normalised to the non-treated control. <b>c</b>–<b>i</b> Short horizontal line represents the median of <i>n</i> = 7–8 donors. Significance in (<b>c</b>–<b>g</b> and <b>i</b>) was determined using a one-tailed Wilcoxon signed-rank test, ns non-significant. After 120 h, the number of intact (<b>j</b>), 5’ defective (<b>k</b>) or 3’ defective (<b>l)</b> HIV DNA copies were determined using digital PCR. Copy numbers were normalised to cell input. Datapoints represent individual donors (<i>n</i> = 7). Note different donors were used in (<b>j</b>–<b>l</b> and <b>c</b>–<b>i)</b>. Significance was determined using a two-tailed Wilcoxon signed-rank test, ns non-significant.</p></div></figure></div><p>To assess latency reversal ex vivo, we treated CD4<sup>+</sup> T cells from people living with HIV on suppressive ART with an equivalent dose of 200 ng per 10<sup>5</sup> cells Tat-LNP X or control mCherry-LNP X for 48–72 h. The expression of cell-associated HIV RNA was measured as described previously<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 36" title="Tumpach, C. et al. Adaptation of droplet digital PCR-based HIV transcription profiling to digital PCR and association of HIV transcription and total or intact HIV DNA. Viruses 15, 1606 (2023)." href="https://www.nature.com/articles/s41467-025-60001-2#ref-CR36" id="ref-link-section-d326275717e1791">36</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 37" title="Yukl, S. A. et al. HIV latency in isolated patient CD4(+) T cells may be due to blocks in HIV transcriptional elongation, completion, and splicing. Sci. Transl. Med. 10 (2018)." href="https://www.nature.com/articles/s41467-025-60001-2#ref-CR37" id="ref-link-section-d326275717e1794">37</a></sup>, whereby we simultaneously quantified transcription initiation, elongation, completion and splicing<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 37" title="Yukl, S. A. et al. HIV latency in isolated patient CD4(+) T cells may be due to blocks in HIV transcriptional elongation, completion, and splicing. Sci. Transl. Med. 10 (2018)." href="https://www.nature.com/articles/s41467-025-60001-2#ref-CR37" id="ref-link-section-d326275717e1798">37</a></sup>. After a single dose of Tat-LNP X, all measured HIV transcripts were significantly upregulated compared to non-treated control cells suggesting Tat-LNP X induces, and overcomes blocks in, transcription initiation (TAR transcript), proximal and distal elongation (Long-LTR and Pol transcripts, respectively), completion of transcription (Poly(A) transcripts) and splicing (Tat-Rev transcripts; detected by a primer/probe set that spans the junction between the first and second coding exon of <i>tat</i> and <i>rev</i>) (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41467-025-60001-2#Fig3">3c–g</a> and Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41467-025-60001-2#MOESM1">S6a–e</a>). The induction of multiply-spliced Tat-Rev transcripts, an important predictor of virion production ex vivo<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 38" title="Zerbato, J. M. et al. Multiply spliced HIV RNA is a predictive measure of virus production ex vivo and in vivo following reversal of HIV latency. EBioMedicine 65, 103241 (2021)." href="https://www.nature.com/articles/s41467-025-60001-2#ref-CR38" id="ref-link-section-d326275717e1815">38</a></sup>, was induced by 112-fold (<i>p</i> &lt; 0.01 compared to untreated control). Treatment with Tat-LNP X or control mCherry-LNP X resulted in minimal overall toxicity (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41467-025-60001-2#Fig3">3h</a>). Importantly, the induction of HIV RNA transcription following Tat-LNP X significantly exceeded that observed in fully activated T cells stimulated using two mitogens, PMA and phytohaemagglutinin (PHA) (considered the gold-standard to reverse HIV latency in vitro and ex vivo<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 37" title="Yukl, S. A. et al. HIV latency in isolated patient CD4(+) T cells may be due to blocks in HIV transcriptional elongation, completion, and splicing. Sci. Transl. Med. 10 (2018)." href="https://www.nature.com/articles/s41467-025-60001-2#ref-CR37" id="ref-link-section-d326275717e1826">37</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 38" title="Zerbato, J. M. et al. Multiply spliced HIV RNA is a predictive measure of virus production ex vivo and in vivo following reversal of HIV latency. EBioMedicine 65, 103241 (2021)." href="https://www.nature.com/articles/s41467-025-60001-2#ref-CR38" id="ref-link-section-d326275717e1829">38</a></sup>) (<i>p</i> &lt; 0.01 for all transcripts). However, in contrast to PMA/PHA treatment, Tat-LNP X did not induce any change in cellular activation as observed by minimal changes to the expression of activation markers CD25, CD69 and HLA-DR compared to non-treated control cells (Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41467-025-60001-2#MOESM1">S6f–h</a>). Furthermore, treatment with Tat-LNP X, but not mCherry-LNP X, resulted in a 17.2-fold increase in supernatant HIV RNA, indicating virion production following latency reversal (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41467-025-60001-2#Fig3">3i</a>). However, the reversal of latency by Tat-LNP X did not result in a decline in intact proviral DNA after 5-day culture (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41467-025-60001-2#Fig3">3j–l</a>). Combined, these findings demonstrate that Tat-LNP X is a highly potent latency-reversing agent that can overcome HIV RNA transcription and processing blocks to elongation, splicing and completion, and induce viral protein expression, all in the absence of T cell activation.</p><h3 id="Sec6">CRISPR activation machinery can be co-encapsulated by LNP X</h3><p>We next assessed whether LNP X was able to encapsulate and deliver CRISPR activation (CRISPRa) machinery as a more complex and larger RNA-based, highly HIV-specific therapeutic. The dCas9-synergistic activation mediator (dCas9-SAM) CRISPRa system consists of a catalytically inactive (dead) Cas9 (dCas9) fused to a multimer of C-terminal herpes virus transcriptional activation domain 16 (VP64), which is guided to the genomic target site by a guide RNA (gRNA). The gRNA is modified to contain minimal hairpin aptamers that allow the recruitment of introduced p65 and heat shock factor 1 (HSF1) transcription activators through binding of the bacteriophage protein MS2<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 39" title="Konermann, S. et al. Genome-scale transcriptional activation by an engineered CRISPR-Cas9 complex. Nature 517, 583–588 (2015)." href="https://www.nature.com/articles/s41467-025-60001-2#ref-CR39" id="ref-link-section-d326275717e1853">39</a></sup> (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41467-025-60001-2#Fig4">4a</a>). The recruitment of multiple copies of the MS2-p65-HSF1 fusion protein, which synergise with dCas9-VP64, greatly enhances the potency of CRISPRa<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 39" title="Konermann, S. et al. Genome-scale transcriptional activation by an engineered CRISPR-Cas9 complex. Nature 517, 583–588 (2015)." href="https://www.nature.com/articles/s41467-025-60001-2#ref-CR39" id="ref-link-section-d326275717e1860">39</a></sup>, but can result in non-specific promoter activation if present in excess. We therefore first optimised the relative dose of the three RNAs that comprise the dCas9-SAM CRISPRa machinery. We found that a relative mass ratio of 0.8: 0.00625: 1 (dCas9-VP64 mRNA: MS2-p65-HSF1 mRNA: gRNA) yielded the optimal balance between on-target potency and non-specific background reactivation in cell lines (Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41467-025-60001-2#MOESM1">S7</a>).</p><div data-test="figure" data-container-section="figure" id="figure-4" data-title="LNP X co-encapsulating CRISPR activation machinery can be used to induce expression of endogenous genes in T cells."><figure><figcaption><b id="Fig4" data-test="figure-caption-text">Fig. 4: LNP X co-encapsulating CRISPR activation machinery can be used to induce expression of endogenous genes in T cells.</b></figcaption><div><div><a data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="https://www.nature.com/articles/s41467-025-60001-2/figures/4" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41467-025-60001-2/MediaObjects/41467_2025_60001_Fig4_HTML.png?as=webp"/><img aria-describedby="Fig4" src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41467-025-60001-2/MediaObjects/41467_2025_60001_Fig4_HTML.png" alt="figure 4" loading="lazy" width="685" height="502"/></picture></a></div><p><b>a</b> Schematic overview of the dCas9-synergistic activation mediator (SAM) CRISPR activation system, consisting of a catalytically inactive (dead, d)Cas9 fused to transcriptional activator domain VP64, a gRNA and a further transcriptional activation MS2-p65-HSF1 fusion protein that is recruited to the Cas9-gRNA complex <i>via</i> MS2-binding to stem-loop structures in the gRNA scaffold. Created in BioRender: <i>Cevaal, P. (2025)</i> <a href="https://BioRender.com/p8j24y6">https://BioRender.com/p8j24y6</a>. <b>b</b>–<b>d</b> LNP formulation X was used to encapsulate the CRISPR activation machinery (CRISPRa-LNP X) including a gRNA targeting the endogenous gene encoding CD25 (CD25 CRISPRa-LNP X) or a scrambled control gRNA (scr CRISPRa-LNP X). <b>b</b> Jurkat T cells were treated for 24 h with CRISPRa-LNP X, control mCherry-LNP X or PMA/ionomycin as positive control. Induction of CD25 expression was measured by surface stain using flow cytometry. CD4<sup>+</sup> T cells from HIV-negative donors were treated with indicated doses of CRISPRa-LNP X per 10<sup>5</sup> cells or mCherry-LNP X as a control for 72 h (<b>c</b>) or 6 days (<b>d</b>), after which CD25 expression was determined using flow cytometry. Highest dose of CD25 CRISPRa-LNP X corresponds to 4 µg/mL. Treatment with PMA/ionomycin was included as a positive control. Dotted line represented average baseline CD25 expression in the absence of treatment. Mean ± SEM, <i>n</i> = 4 independent experiments (<b>b</b>) or <i>n</i> = 4–6 donors (<b>c</b>, <b>d</b>). Significance was determined using a two-tailed Student’s t-test.</p></div></figure></div><p>We next developed a model system in which CRISPRa was targeted to the promoter of an endogenous gene, <i>IL2RA (cd25)</i>, to induce overexpression of the CD25 receptor on the cell surface. CRISPRa-LNP X was formulated by co-encapsulating the two mRNAs and a CD25-targeting or a scrambled control gRNA. The CD25-targeting gRNA was validated in Jurkat T cells, which express negligible CD25 at baseline. We found that 24 h following treatment with CD25 CRISPRa-LNP X, there was an increase in surface CD25 expression in up to 29.6 ± 8.6% of Jurkat T cells, while minimal to no background expression of CD25 was observed with scrambled gRNA (scr) CRISPRa-LNP X (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41467-025-60001-2#Fig4">4b</a> and Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41467-025-60001-2#MOESM1">S8a</a>). We demonstrated a similar level of potency in non-stimulated primary CD4<sup>+</sup> T cells from HIV-negative donors, yielding a significant increase in cells expressing CD25 from 8.7 ± 0.7% with scr CRISPRa-LNP X to 14.6 ± 0.9% after 72 h treatment with 800 ng CD25 CRISPRa-LNP X per 10<sup>5</sup> cells (<i>p</i> &lt; 0.01) (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41467-025-60001-2#Fig4">4c</a> and Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41467-025-60001-2#MOESM1">S8b</a>).</p><p>We then extended the incubation time after a single dose of CD25 CRISPRa-LNP X, scr CRISPRa-LNP X or control mCherry-LNP X from 72 h to 6 days. The percentage of cells expressing CD25 after 6 days was increased to 24.6 ± 5.1% of live cells (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41467-025-60001-2#Fig4">4d</a> and Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41467-025-60001-2#MOESM1">S8c</a>), suggesting the peak response of CRISPRa-induced protein expression occurs later than 72 h post-treatment. In contrast, the expression of mCherry was reduced at 6 days compared to 72 h post-treatment (Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41467-025-60001-2#MOESM1">S8d, e</a>), consistent with transient expression of LNP-mediated delivery of exogenous mRNA. The enhanced CD25 expression at day 6 was therefore likely not the result of increased expression of the CRISPRa machinery (dCas9-VP64 and MS2-p65-HSF1) at day 6, rather the extended incubation is required to capture the translation of the induced <i>CD25</i> mRNA and its trafficking to the cell surface. Importantly, these findings confirm, the first to our knowledge, successful delivery of the dCas9-SAM CRISPRa machinery to non-stimulated T cells in vitro.</p><h3 id="Sec7">HIV CRISPRa-LNP X activates HIV transcription ex vivo</h3><p>Finally, to generate HIV LTR-targeted CRISPRa-LNP X, we designed four MS2-p65-HSF1 recruiting hairpin-modified gRNAs (B, C, L and O) targeting different HIV LTR regions upstream of the transcription start site based on previous work<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 23" title="Zhang, Y. et al. CRISPR/gRNA-directed synergistic activation mediator (SAM) induces specific, persistent and robust reactivation of the HIV-1 latent reservoirs. Sci. Rep. 5, 16277–16277 (2015)." href="https://www.nature.com/articles/s41467-025-60001-2#ref-CR23" id="ref-link-section-d326275717e1988">23</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 40" title="Tantale, K. et al. Stochastic pausing at latent HIV-1 promoters generates transcriptional bursting. Nat. Commun. 12, 4503 (2021)." href="https://www.nature.com/articles/s41467-025-60001-2#ref-CR40" id="ref-link-section-d326275717e1991">40</a></sup> (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41467-025-60001-2#Fig5">5a</a>, Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41467-025-60001-2#MOESM1">S1</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41467-025-60001-2#MOESM1">S2</a>). We found that all four HIV LTR-targeting gRNAs, compared to a scrambled gRNA, could induce GFP expression in J-Lat 10.6 cells (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41467-025-60001-2#Fig5">5b</a>). To assess the potency of HIV LTR-targeting CRISPRa-LNP X ex vivo, we treated CD4<sup>+</sup> T cells from ART-suppressed people living with HIV for 72 h with an equivalent dose of 200 ng CRISPRa-LNP X per 10<sup>5</sup> cells, analogous to what we used to deliver Tat mRNA. We used CRISPRa-LNP X co-encapsulating a 1:1 mass ratio of gRNA L and gRNA O (L + O CRISPRa-LNP X), which was found to exceed the potency of either gRNA alone (Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41467-025-60001-2#MOESM1">S9</a>) and was hypothesised to increase the sequence coverage of HIV subtype B LTR sequences, the most common subtype in our cohort. L + O CRISPRa-LNP X treatment significantly increased all measured HIV transcripts except multiply-spliced Tat-Rev (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41467-025-60001-2#Fig5">5c-g</a> and Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41467-025-60001-2#MOESM1">S10a–e</a>). The potency of latency-reversal was lower compared to Tat-LNP X treatment, with a maximum fold-increase over non-treated of 2.0 (IQR 0.87-3.39) for Pol transcripts, and did not exceed that of PMA/PHA treatment. No HIV transcripts were upregulated after treatment with a scrambled gRNA CRISPRa-LNP X, indicating the activation of transcription was gRNA-mediated. Treatment with HIV LTR-targeted CRISPRa-LNP X resulted in minimal toxicity, similar to treatment with Tat- or mCherry-LNP X, and did not result in generalised cellular activation (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41467-025-60001-2#Fig5">5h</a> and Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41467-025-60001-2#MOESM1">S10f–h</a>), demonstrating the specificity of CRISPRa-LNP X as a next-generation LRA. However, treatment with L + O CRISPRa-LNP X did not lead to an induction of supernatant HIV RNA, as a measure of virion release (Figs. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41467-025-60001-2#Fig5">5i</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41467-025-60001-2#MOESM1">S10i</a>).</p><div data-test="figure" data-container-section="figure" id="figure-5" data-title="CRISPRa-LNP X activates HIV transcription in CD4+ T cells from people living with HIV."><figure><figcaption><b id="Fig5" data-test="figure-caption-text">Fig. 5: CRISPRa-LNP X activates HIV transcription in CD4<sup>+</sup> T cells from people living with HIV.</b></figcaption><div><div><a data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="https://www.nature.com/articles/s41467-025-60001-2/figures/5" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41467-025-60001-2/MediaObjects/41467_2025_60001_Fig5_HTML.png?as=webp"/><img aria-describedby="Fig5" src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41467-025-60001-2/MediaObjects/41467_2025_60001_Fig5_HTML.png" alt="figure 5" loading="lazy" width="685" height="919"/></picture></a></div><p><b>a</b> Schematic overview of the CRISPRa system targeting the HIV LTR. Created in BioRender: <i>Cevaal, P. (2025)</i> <a href="https://BioRender.com/p8j24y6">https://BioRender.com/p8j24y6</a>. <b>b</b> J-Lat 10.6 cells were treated for 24 h with indicated doses of CRISPRa-LNP X containing one of four HIV LTR-targeting gRNAs (L, O, B, C) or scrambled control gRNA. Reactivation of HIV LTR-mediated transcription was determined after 24 h by measuring GFP expression. Highest dose of CRISPRa-LNP X corresponds to 2.5 µg/mL. Treatment with PMA/ionomycin was included as a positive control. Mean ± SEM, <i>n</i> = 3 independent experiments. <b>c</b>–<b>i</b> CD4<sup>+</sup> T cells from people living with HIV on suppressive ART were treated with 200 ng CRISPRa-LNP X containing gRNAs L and O (L + O CRISPRa-LNP X) or a scrambled gRNA control (scr CRISPRa-LNP X) per 10<sup>5</sup> cells (4 µg/mL) or PMA/PHA as a positive control. After 48 h (squares) or 72 h (circles), expression of HIV transcripts TAR (<b>c</b>) LongLTR (<b>d</b>), Pol (<b>e</b>), PolyA (<b>f</b>) and Tat-Rev (<b>g</b>) representing transcription initiation, proximal elongation, distal elongation, completion and splicing, respectively, was determined using digital RT-PCR. Data were normalised to RNA input, then presented as fold-change induction compared to the corresponding non-treated (NT) control. Horizontal dashed line (<b>c</b>–<b>g</b>) represents no change relative to untreated cells. <b>h</b> Concurrently, cellular toxicity was determined using flow cytometry. Where datapoints are missing in (<b>h</b>), cell input was insufficient to perform an accurate measurement. <b>i</b> After 72 h, the number of copies of HIV RNA per mL of supernatant was quantified using RT-PCR and normalised to the non-treated control. <b>c</b>–<b>i</b> Short horizontal line represents the median of <i>n</i> = 7–8 donors. Significance in (<b>c</b>–<b>g</b>) and (<b>i</b>) was determined using a one-tailed Wilcoxon signed-rank test, ns non-significant.</p></div></figure></div></div></div></section><section data-title="Discussion"><div id="Sec8-section"><h2 id="Sec8">Discussion</h2><div id="Sec8-content"><p>Advances in mRNA technology provide an opportunity for a new generation of therapeutics with enhanced potency and reduced toxicity. Here, we explored the use of LNPs to enable therapeutic mRNA delivery to primary T cells, including CD4<sup>+</sup> T cells latently infected with HIV. We reported an LNP formulation that is based on a combination of SM-102 and β-sitosterol. This LNP X reached transfection efficiencies of &gt;75% in primary CD4<sup>+</sup> T cells. To our knowledge, this is the first demonstration of successful, non-toxic in vitro transfection of primary T cells in the absence of T cell pre-stimulation, which was previously thought a pre-requisite for efficient LNP transfection<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 41" title="Patel, S. K. et al. Hydroxycholesterol substitution in ionizable lipid nanoparticles for mRNA delivery to T cells. J. Control. Release 347, 521–532 (2022)." href="https://www.nature.com/articles/s41467-025-60001-2#ref-CR41" id="ref-link-section-d326275717e2150">41</a></sup>. Our data suggest that the superior potency of LNP X is due to increased cellular association as well as an increased efficiency of mRNA translation after delivery to the cytosol, contrasting with earlier hypotheses that β-sitosterol enhanced the efficiency of endosomal escape<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 31" title="Patel, S. et al. Naturally-occurring cholesterol analogues in lipid nanoparticles induce polymorphic shape and enhance intracellular delivery of mRNA. Nat. Commun. 11, 983 (2020)." href="https://www.nature.com/articles/s41467-025-60001-2#ref-CR31" id="ref-link-section-d326275717e2154">31</a></sup>. Together, these findings identify LNP X as a promising in vitro transfection tool that has the potential to replace nucleofection or viral transduction in the development of therapeutics for a range of T cell-implicated diseases or the generation of T cell-based immunotherapies. Furthermore, our data justify exploring LNP X for delivering nucleic acid cargo to other hard-to-transfect cell types.</p><p>We here used LNP X to address the need for more potent, HIV-specific latency-reversing agents, in order to deplete the latent HIV reservoir and contribute to virologic control off ART<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 3" title="Deeks, S. G. et al. Research priorities for an HIV cure: International AIDS Society Global Scientific Strategy 2021. Nat. Med. 27, 2085–2098 (2021)." href="https://www.nature.com/articles/s41467-025-60001-2#ref-CR3" id="ref-link-section-d326275717e2161">3</a></sup>. Combined with an mRNA expressing the 72 amino acids of the first coding exon of HIV Tat, LNP X was able to overcome all blocks in HIV transcription initiation and RNA processing with striking potency, exceeding the viral reactivation achieved by stimulation with a general T cell mitogen. Tat-LNP X showed a greater ability to reverse HIV latency than previously reported Tat-LNPs<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 21" title="Raines, S. L. M. et al. Nanoparticle delivery of Tat synergizes with classical latency reversal agents to express HIV antigen targets. Antimicrob. Agents Chemother. 68, e00201-24 (2024)." href="https://www.nature.com/articles/s41467-025-60001-2#ref-CR21" id="ref-link-section-d326275717e2165">21</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 22" title="Van Gulck, E. et al. A truncated HIV Tat demonstrates potent and specific latency reversal activity. Antimicrob. Agents Chemother. 67, e0041723 (2023)." href="https://www.nature.com/articles/s41467-025-60001-2#ref-CR22" id="ref-link-section-d326275717e2168">22</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 42" title="Pardons, M. et al. Potent latency reversal by Tat RNA-containing nanoparticle enables multi-omic analysis of the HIV-1 reservoir. Nat. Commun. 14, 8397 (2023)." href="https://www.nature.com/articles/s41467-025-60001-2#ref-CR42" id="ref-link-section-d326275717e2171">42</a></sup>. The lack of disclosure of the LNP formulation and uncertainty of the full biological properties of the Tat<sub>66</sub> isoform used in the previously mentioned work means that direct comparisons of both Tat-LNPs were not possible. However, we hypothesise the superiority of our Tat-LNP X is primarily based on the high efficiency of mRNA delivery by LNP X, consistent with sub-optimal transfection efficiency of previously reported Tat-LNP<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 21" title="Raines, S. L. M. et al. Nanoparticle delivery of Tat synergizes with classical latency reversal agents to express HIV antigen targets. Antimicrob. Agents Chemother. 68, e00201-24 (2024)." href="https://www.nature.com/articles/s41467-025-60001-2#ref-CR21" id="ref-link-section-d326275717e2177">21</a></sup>. Interestingly, the high level of latency reversal by Tat-LNP X appeared insufficient to drive virus-mediated cytotoxicity ex vivo, consistent with latently infected cells overexpressing pro-survival proteins such as B-Cell Lymphoma (BCL)-2<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 43" title="Arandjelovic, P. et al. Venetoclax, alone and in combination with the BH3 mimetic S63845, depletes HIV-1 latently infected cells and delays rebound in humanized mice. Cell Rep. Med. 4, 101178 (2023)." href="https://www.nature.com/articles/s41467-025-60001-2#ref-CR43" id="ref-link-section-d326275717e2181">43</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 44" title="Ren, Y. et al. BCL-2 antagonism sensitizes cytotoxic T cell-resistant HIV reservoirs to elimination ex vivo. J. Clin. Investig. 130, 2542–2559 (2020)." href="https://www.nature.com/articles/s41467-025-60001-2#ref-CR44" id="ref-link-section-d326275717e2184">44</a></sup> and BIRC-5<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 45" title="Kuo, H. H. et al. Anti-apoptotic protein BIRC5 maintains survival of HIV-1-infected CD4(+) T cells. Immunity 48, 1183–1194 e1185 (2018)." href="https://www.nature.com/articles/s41467-025-60001-2#ref-CR45" id="ref-link-section-d326275717e2189">45</a></sup>. These findings highlight the need to combine latency reversal with additional interventions to sensitise infected cells to death or enhance immune-mediated clearance<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 3" title="Deeks, S. G. et al. Research priorities for an HIV cure: International AIDS Society Global Scientific Strategy 2021. Nat. Med. 27, 2085–2098 (2021)." href="https://www.nature.com/articles/s41467-025-60001-2#ref-CR3" id="ref-link-section-d326275717e2193">3</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 9" title="Kim, Y., Anderson, J. L. &amp; Lewin, S. R. Getting the “kill” into “shock and kill”: strategies to eliminate latent HIV. Cell Host Microbe 23, 14–26 (2018)." href="https://www.nature.com/articles/s41467-025-60001-2#ref-CR9" id="ref-link-section-d326275717e2196">9</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 46" title="Dashti, A. et al. AZD5582 plus SIV-specific antibodies reduce lymph node viral reservoirs in antiretroviral therapy-suppressed macaques. Nat. Med. 29, 2535–2546 (2023)." href="https://www.nature.com/articles/s41467-025-60001-2#ref-CR46" id="ref-link-section-d326275717e2199">46</a></sup>. Furthermore, single-cell or limiting-dilution assays will be required to assess the ability of Tat-LNP X to uniformly induce HIV transcription across the breadth of the HIV reservoir, including cells harbouring proviruses in a deep quiescent state<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 47" title="Dufour, C., Gantner, P., Fromentin, R. &amp; Chomont, N. The multifaceted nature of HIV latency. J. Clin. Investig. 130, 3381–3390 (2021)." href="https://www.nature.com/articles/s41467-025-60001-2#ref-CR47" id="ref-link-section-d326275717e2203">47</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 48" title="Fisher, B. M., Cevaal, P. M., Roche, M. &amp; Lewin, S. R. HIV Tat as a latency reversing agent: Turning the tables on viral persistence. Front. Immunol. 16, 1571151 (2025)." href="https://www.nature.com/articles/s41467-025-60001-2#ref-CR48" id="ref-link-section-d326275717e2206">48</a></sup>. We showed that LNP X could co-encapsulate and deliver CRISPRa machinery, including a large dCas9-VP64 mRNA. One limitation of our study was that the delivery efficiency of the three CRISPRa RNA components was not individually assessed. Direct analysis of the efficiency and kinetics of delivery of various RNA cargo, as well as their respective half-lives once delivered, are likely to be key in further optimisation of the CRISPR activation potency in primary cells<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 49" title="Beyersdorf, J. P. et al. Robust, Durable Gene Activation In Vivo via mRNA-Encoded Activators. ACS Nano 16, 5671–5671 (2022)." href="https://www.nature.com/articles/s41467-025-60001-2#ref-CR49" id="ref-link-section-d326275717e2210">49</a></sup>. Nonetheless, the potency of CRISPRa-LNP X is a promising indication that LNP X could be used to deliver other CRISPR variants to T cells, such as CRISPR-Cas9 to excise the integrated HIV provirus<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 50" title="Dash, P. K. et al. CRISPR editing of CCR5 and HIV-1 facilitates viral elimination in antiretroviral drug-suppressed virus-infected humanized mice. Proc. Natl. Acad. Sci. USA 120, e2217887120 (2023)." href="https://www.nature.com/articles/s41467-025-60001-2#ref-CR50" id="ref-link-section-d326275717e2214">50</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 51" title="Dash, P. K. et al. Sequential lasER ART and CRISPR treatments eliminate HIV-1 in a subset of infected humanized mice. Nat. Commun. 10, 2753–2753 (2019)." href="https://www.nature.com/articles/s41467-025-60001-2#ref-CR51" id="ref-link-section-d326275717e2217">51</a></sup>, or knock out CCR5 as the HIV entry receptor<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 52" title="Xu, L. et al. CRISPR-edited stem cells in a patient with HIV and acute lymphocytic leukemia. N. Engl. J. Med. 381, 1240–1247 (2019)." href="https://www.nature.com/articles/s41467-025-60001-2#ref-CR52" id="ref-link-section-d326275717e2221">52</a></sup>.</p><p>Our current results did not indicate any in vitro toxicity induced by treatment with LRA-LNP X, nor any signs of cellular activation. However, an in-depth analysis of changes to the cellular transcriptome or metabolome is warranted in further investigations into Tat or CRISPRa as an HIV-specific latency-reversing agent. To explore the potential of LNP X for in vivo therapeutics, studies assessing the immunogenicity, biodistribution and half-life in circulation will be required, as well as dose-finding and safety studies to determine the optimal therapeutic dose. Prior studies that administered LNP intravenously in people have shown that 0.1 – 0.6 mg/kg of mRNA was well-tolerated in humans<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 2" title="Gillmore, J. D. et al. CRISPR-Cas9 in vivo gene editing for transthyretin amyloidosis. N. Engl. J. Med. 385, 493–502 (2021)." href="https://www.nature.com/articles/s41467-025-60001-2#ref-CR2" id="ref-link-section-d326275717e2228">2</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 53" title="August, A. et al. A phase 1 trial of lipid-encapsulated mRNA encoding a monoclonal antibody with neutralizing activity against Chikungunya virus. Nat. Med. 27, 2224–2233 (2021)." href="https://www.nature.com/articles/s41467-025-60001-2#ref-CR53" id="ref-link-section-d326275717e2231">53</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 54" title="Koeberl, D. et al. Interim analyses of a first-in-human phase 1/2 mRNA trial for propionic acidaemia. Nature 628, 872–877 (2024)." href="https://www.nature.com/articles/s41467-025-60001-2#ref-CR54" id="ref-link-section-d326275717e2234">54</a></sup>. Future work exploring the potential to specifically target LNP X to T cells through ligand-mediated targeting could further contribute to the therapeutic potential of LNP X in vivo.</p></div></div></section><section data-title="Methods"><div id="Sec9-section"><h2 id="Sec9">Methods</h2><div id="Sec9-content"><h3 id="Sec10">Ethics statement</h3><p>Blood samples from people living with HIV were collected through leukapheresis at the Alfred Hospital (Melbourne, Australia) and the University of California San Francisco (UCSF) (USA) with informed consent and under institutional guidelines. People living with HIV included in these studies were aged 18 years or older and on suppressive antiretroviral treatment (ART) (viral load &lt;50 copies/mL) for ≥3 years. The use of these samples was approved by the Human Research Ethics Committees at the Alfred Hospital, the University of Melbourne, and the Institutional Review Board at UCSF.</p><h3 id="Sec11">CD4<sup>+</sup> T cell isolation and cell culture</h3><p>Peripheral blood mononuclear cells (PBMCs) were obtained using Ficoll-Paque density gradient centrifugation of buffy coats (Australian Red Cross Lifeblood) or whole blood collected through leukapheresis. CD4<sup>+</sup> T-cells were isolated from PBMCs using the EasyStep<sup>TM</sup> Human CD4<sup>+</sup> T cell Isolation Kit (Stemcell Technologies), following manufacturer’s instructions, and stored in liquid nitrogen until use. After thawing, CD4<sup>+</sup> T cells were rested overnight prior to treatment and cultured in RPMI 1640 (Thermo Fisher Scientific) supplemented with 10% heat-inactivated foetal bovine serum (FBS; Cellsera Australia), 100 U/mL penicillin, 100 µg/mL streptomycin, 2 mM L-glutamine (all Thermo Fisher Scientific) (RF10) and 10 U/mL interleukin-2 (IL-2; Sigma-Aldrich). J-Lat A2 (RRID:CVCL_1G43) and J-Lat 10.6 cells (RRID:CVCL_8281)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 55" title="Jordan, A., Bisgrove, D. &amp; Verdin, E. HIV reproducibly establishes a latent infection after acute infection of T cells in vitro.  EMBO J. 22, 1868–1877 (2003)." href="https://www.nature.com/articles/s41467-025-60001-2#ref-CR55" id="ref-link-section-d326275717e2269">55</a></sup> were cultured in RF10 media. Cell lines and primary cells were maintained in a 37 <sup>o</sup>C humified incubator with 5% CO<sub>2</sub>.</p><h3 id="Sec12">RNA reagents for LNP synthesis</h3><p>For studies comparing LNP X potency to benchmark LNP formulation (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41467-025-60001-2#Fig1">1</a>, Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41467-025-60001-2#MOESM1">S1</a>), CleanCap® mCherry mRNA containing 5-methoxyuridine base modifications was obtained from TriLink Biotechnologies (L-7203). All other studies were performed with codon-optimised, capped, N1-methylpseudouridine base-modified mCherry mRNA purchased from Messenger Bio (Melbourne, Australia). Reporter mRNA encoding mScarlet was synthesised using in vitro transcription from codon-optimised PCR template using HiScribe T7 High Yield RNA Synthesis Kit (New England Biolabs), replacing all uridine with N1-methylpseudouridine (TriLink Biotechnologies). mRNA was co-transcriptional capped using CleanCap Reagent AG (TriLink Biotechnologies) and cleaned up by cellulose to remove dsRNA. AF488-oligo (5’-AF488AGA GTT CCC AAG ACC AGG CGG-3’) and Cy5-oligo (5’-Cy5AGA GTT CCC AAG ACC AGG CGG-3’) were purchased from Integrated DNA technologies. SNAP<sub>switch</sub>-labelled oligo was synthesised by conjugating SNAP<sub>switch</sub>-azide (ADKL Labs) to DBCO-oligo (5’-TCA GTT CAG GAC CCT CGG CTDBCO-3’) (TriLink Biotechnologies) as described previously<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 33" title="Liu, H. et al. Beyond the endosomal bottleneck: understanding the efficiency of mRNA/LNP delivery. Adv. Funct. Mater. 34, 2404510 (2024)." href="https://www.nature.com/articles/s41467-025-60001-2#ref-CR33" id="ref-link-section-d326275717e2296">33</a></sup>. Codon-optimised mRNAs encoding dCas9-VP64 (MW: 1528 kDa) and MS2-p65-HSF1 (MW: 562.6 kDa) were produced by Messenger Bio using previously published sequences<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 39" title="Konermann, S. et al. Genome-scale transcriptional activation by an engineered CRISPR-Cas9 complex. Nature 517, 583–588 (2015)." href="https://www.nature.com/articles/s41467-025-60001-2#ref-CR39" id="ref-link-section-d326275717e2301">39</a></sup>. mRNA expressing the 72 amino acids of the first coding exon of HIV Tat was designed following the NL4-3 reference sequence and produced by Messenger Bio (MW: 177.7 kDa). All custom-designed mRNAs purchased from Messenger Bio included N1-methylpseudouridine base modifications. Guide RNAs (gRNAs) were designed using previously described spacer sequences targeting the HIV LTR<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 23" title="Zhang, Y. et al. CRISPR/gRNA-directed synergistic activation mediator (SAM) induces specific, persistent and robust reactivation of the HIV-1 latent reservoirs. Sci. Rep. 5, 16277–16277 (2015)." href="https://www.nature.com/articles/s41467-025-60001-2#ref-CR23" id="ref-link-section-d326275717e2305">23</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 40" title="Tantale, K. et al. Stochastic pausing at latent HIV-1 promoters generates transcriptional bursting. Nat. Commun. 12, 4503 (2021)." href="https://www.nature.com/articles/s41467-025-60001-2#ref-CR40" id="ref-link-section-d326275717e2308">40</a></sup> (Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41467-025-60001-2#MOESM1">S1</a>) or <i>CD25</i> (<i>IL2RA</i>) (5’-TTATGGGCGTAGCTGAAGAA-3’)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 56" title="Simeonov, D. R. et al. Discovery of stimulation-responsive immune enhancers with CRISPR activation. Nature 549, 111–115 (2017)." href="https://www.nature.com/articles/s41467-025-60001-2#ref-CR56" id="ref-link-section-d326275717e2321">56</a></sup> and a CRISPRa gRNA scaffold containing two MS2 hairpin aptamers, as described previously<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 39" title="Konermann, S. et al. Genome-scale transcriptional activation by an engineered CRISPR-Cas9 complex. Nature 517, 583–588 (2015)." href="https://www.nature.com/articles/s41467-025-60001-2#ref-CR39" id="ref-link-section-d326275717e2326">39</a></sup> (Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41467-025-60001-2#MOESM1">S1</a>). gRNAs were obtained from Integrated DNA technologies using custom IDT Alt-R<sup>TM</sup> gRNA synthesis, containing two 2’O-methyl base and phosphorothioate bond modifications at the 5’ and 3’ end of the gRNA (MW: 52.0 kDa).</p><h3 id="Sec13">Lipid nanoparticle formulation</h3><p>Lipid nanoparticles were assembled using the ionisable lipid DLin-MC3-DMA (HY-112251, MedChemExpress) or SM-102 (33474, Cayman Chemical Company), DSPC (850365P, Avanti Polar Lipids), Cholesterol (C3045, Sigma-Aldrich) or β-sitosterol (700095P, Sigma-Aldrich), and DMG-PEG2000 (880151P, Avanti Polar Lipids). Lipids were reconstituted at 10 mM in ethanol and mixed at a molar ratio of 50:10:38.5:1.5 of DLin-MC3-DMA:DSPC:Cholesterol:DMG-PEG2000 or SM-102:DSPC:β-sitosterol:DMG-PEG2000 for patisiran LNPs and LNP X, respectively. Immediately prior to mixing, RNA was diluted to 150 ng/μL in 30 mM RNAse-free sodium acetate buffer, pH4.0. For CRISPRa-LNPs, RNA components were mixed at a mass ratio of 0.8:0.00625:1 of dCas9-VP64 mRNA: MS2-p65-HSF1 mRNA: gRNA, corresponding to a molar ratio of 0.027:0.00058:1. LNPs were synthesised using a NanoAssemblr Spark (Precision NanoSystems) whereby the aqueous (RNA) and organic (lipid) phases were mixed at a flow rate ratio of 1.8 (aqueous): 1 (organic), with an N/P ratio of 6:1. These formulation parameters are summarised in Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41467-025-60001-2#MOESM1">S3</a>. The LNP size and polydispersity were analysed through dynamic light scattering using a Zetasizer Ultra (Malvern Panalytical). RNA encapsulation efficiency and total RNA concentration of the LNP formulations were determined using a Quant-it<sup>TM</sup> RiboGreen RNA Assay as per the manufacturer’s low-range assay protocol (Invitrogen). Total RNA concentrations were used to dose LNPs throughout. For endosomal escape studies (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41467-025-60001-2#Fig2">2</a>), LNPs encapsulating a mixture of 95:5 (mass ratio) mScarlet mRNA and either SNAP<sub>switch</sub>/AF488-labelled or Cy5/AF488-labelled oligos were synthesised using a NanoAssemblr Benchtop as described previously<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 33" title="Liu, H. et al. Beyond the endosomal bottleneck: understanding the efficiency of mRNA/LNP delivery. Adv. Funct. Mater. 34, 2404510 (2024)." href="https://www.nature.com/articles/s41467-025-60001-2#ref-CR33" id="ref-link-section-d326275717e2353">33</a></sup>.</p><h3 id="Sec14">Transfection of mRNA in CD4<sup>+</sup> T cells in the presence and absence of other cell types</h3><p>To activate CD4<sup>+</sup> T cells prior to transfection, cells were stimulated with plate-bound anti-CD3 (clone OKT3; BioLegend) and 2 µg/mL of soluble anti-CD28 (clone CD28.2, Biolegend) for 72 h. In all studies using HIV-negative CD4<sup>+</sup> T cells, 100,000 cells were treated with indicated doses of patisiran LNPs or LNP X in a total culture volume of 200 µL of RF10 with 10 U/ml IL-2. After 72 h, cells were stained using the LIVE/DEAD<sup>TM</sup> Fixable Violet Dead Cell Stain Kit (Thermo Fisher Scientific) and fixed prior to read-out. To identify CD4<sup>+</sup> T cell subsets, the following antibodies were used: FITC anti-human CD3 (Clone UCHT1, BD Biosciences), BUV805 anti-human CD4 (Clone SK3, BD Biosciences), PerCP/Cy5.5 anti-human CD45RA (Clone HI100, BioLegend), APC/Cy7 anti-human CCR7 (Clone C043H7, BioLegend) and BV711 anti-human CD27 (Clone L128, BD Biosciences). For CD25 activation experiments, non-stimulated CD4<sup>+</sup> T cells were incubated with CRISPRa-LNPs for either 72 h or 6 days and additionally stained with PE/Cyanine7 anti-human CD25 (Clone BC96; BioLegend). mCherry expression, CD25 expression and cell viability were assessed by flow cytometry using an LSRFortessa<sup>TM</sup> (BD Biosciences). To assess the tropism and specificity of LNP X, PBMCs from HIV-negative donors were plated at 300,000 cells in 100 µL of RF10 and treated with 200 ng mScarlet-LNP X for 24 h. PBMC cell subsets and the expression of mScarlet was determined by flow cytometry using the following antibodies: FITC anti-human CD14 (Clone M5E2, BD Biosciences), APC/Cy7 anti-human CD56 (Clone HCD56, BioLegend), BUV395 anti-human CD3 (Clone UCHT1, BD Biosciences), BUV805 anti-human CD8 (Clone SK1, BD Biosciences), BV510 anti-human CD19 (Clone HIB19, BioLegend), BV650 anti-human HLA-DR (Clone G46-6, BD Biosciences), BV785 anti-human CD16 (Clone 3G8, BioLegend), PE/Cy7 anti-human CD4 (Clone OKT4, BioLegend).</p><h3 id="Sec15">Quantifying LNP endosomal escape</h3><p>LNP association, cytosolic delivery and expression of mRNA and endosomal escape efficiency of LNPs were determined in Jurkat T cells as described previously<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 33" title="Liu, H. et al. Beyond the endosomal bottleneck: understanding the efficiency of mRNA/LNP delivery. Adv. Funct. Mater. 34, 2404510 (2024)." href="https://www.nature.com/articles/s41467-025-60001-2#ref-CR33" id="ref-link-section-d326275717e2390">33</a></sup>. In brief, SNAP-actin expressing Jurkat T cells were treated with 100 ng mScarlet/SNAP<sub>switch</sub> oligo-LNPs per 100,000 cells for 4 h. Background-corrected geometric mean fluorescence was measured using an Aurora flow cytometer (Cytek) to assess LNP association (AF488), cytosolic delivery (SNAP<sub>switch</sub>) and mRNA expression (mScarlet). To determine endosomal escape efficiency, the cytosolic delivery observed with the SNAP<sub>switch</sub>-LNPs was compared to a reference LNP containing an unquenched Cy5-labelled oligo, representing an endosomal escape efficiency of 100%.</p><h3 id="Sec16">J-Lat cell line reactivation</h3><p>J-Lat A2<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 55" title="Jordan, A., Bisgrove, D. &amp; Verdin, E. HIV reproducibly establishes a latent infection after acute infection of T cells in vitro.  EMBO J. 22, 1868–1877 (2003)." href="https://www.nature.com/articles/s41467-025-60001-2#ref-CR55" id="ref-link-section-d326275717e2408">55</a></sup> (ARP-9854) and J-Lat 10.6<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 55" title="Jordan, A., Bisgrove, D. &amp; Verdin, E. HIV reproducibly establishes a latent infection after acute infection of T cells in vitro.  EMBO J. 22, 1868–1877 (2003)." href="https://www.nature.com/articles/s41467-025-60001-2#ref-CR55" id="ref-link-section-d326275717e2412">55</a></sup> (ARP-9849) cells were obtained through the AIDS Research and Reference Reagent Programme, Division of AIDS, NIAID, NIH. Cells were plated at 100,000 cells per well and treated with Tat-, mCherry- or CRISPRa-LNP X in a total culture volume of 100 μL RF10. For assessment of the optimal ratio of gRNA and CRISPRa mRNA components, LNPs containing either gRNA L, gRNA scr, dCas9-VP64 mRNA, or MS2-p65-HSF1 mRNA alone were used. For assessment of multiplexed gRNA delivery, CRISPRa-LNP X containing either gRNA L, gRNA O, or a combination of gRNA L and gRNA O (co-encapsulated at a 1:1 mass ratio) and CRISPRa mRNA components were used. J-Lat cells treated with phorbol 12-myristate 13-acetate (PMA, 16 nM, Sigma-Aldrich) and ionomycin (500 nM, Sigma-Aldrich) were used as positive controls. After 24 h, cells were stained with LIVE/DEAD<sup>TM</sup> Fixable Violet Dead Cell Stain Kit (Thermo Fisher Scientific). Reactivation of LTR-mediated transcription was measured by quantifying GFP expression using flow cytometry.</p><h3 id="Sec17">Reactivation of HIV reporter virus</h3><p>CD4<sup>+</sup> T cells from HIV-negative donors were spinoculated with pMorheus-V5<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 35" title="Kim, E. H. et al. Development of an HIV reporter virus that identifies latently infected CD4(+) T cells. Cell Rep. Methods 2, 100238 (2022)." href="https://www.nature.com/articles/s41467-025-60001-2#ref-CR35" id="ref-link-section-d326275717e2428">35</a></sup> at a TCID<sub>50</sub> unit per cell of 0.005 in RF10 supplemented with 10 U/mL IL-2. pMorpheus-V5 is a single-round virus that harbours an mCherry reporter construct under the control of the HIV LTR promotor, allowing for the identification of productively infected cells undergoing LTR-mediated transcription. After 3 days, virus was aspirated and cells were seeded in fresh RF10 supplemented with 10 U/mL IL-2, then treated with 16 nM of PMA plus 500 nM ionomycin, 2 μg Tat-LNP X per million cells or 0.1% DMSO in a final culture volume of 1 mL per million cells. After a further 48 h, the levels of productive HIV infection were determined by quantifying mCherry expression within live cells.</p><h3 id="Sec18">Ex vivo HIV reactivation</h3><p>PBMCs from people living with HIV on suppressive ART were thawed, after which CD4<sup>+</sup> T cells were isolated using using the EasyStep<sup>TM</sup> Human CD4<sup>+</sup> T cell Isolation Kit (Stemcell Technologies), following manufacturer’s instructions. Isolated CD4<sup>+</sup> T cells were rested overnight in RF10 containing 10 U/mL of IL-2 and 1 µM of Raltegravir. 2 × 10<sup>6</sup> cells were then treated with 4 µg of LNPs at a concentration of 4 µg/mL, or 10 nM of PMA and 10 µg/mL of phytohaemagglutinin (PHA) as a positive control. After 72 h, cells were separated from supernatant using centrifugation. Supernatant HIV RNA was quantified by the Victorian Infectious Diseases Reference Laboratory (VIDRL) using the Alinity m HIV-1 assay (Abbott) as previously described<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 57" title="Braun, P. et al. Multicenter clinical comparative evaluation of alinity m HIV-1 assay performance. J. Clin. Virol. 129, 104530 (2020)." href="https://www.nature.com/articles/s41467-025-60001-2#ref-CR57" id="ref-link-section-d326275717e2453">57</a></sup>. Cells were stained with LIVE/DEAD<sup>TM</sup> Fixable Violet Dead Cell Stain Kit and anti-CD25 (Clone BC96, BioLegend), Brilliant Ultra Violet<sup>TM</sup> 395 anti-CD69 (Clone FN50, Invitrogen) and FITC anti-HLA-DR (Clone L243, Invitrogen). Cell viability and the expression of cellular activation markers were assessed using flow cytometry. To quantify the induction of HIV transcripts, RNA was extracted using polyacryl carrier enriched TRI Reagent (Molecular Research Center) according to the manufacturer’s protocol. TAR levels were quantified by 3-step polyadenylation-RT-dPCR, whilst Long-LTR, Pol, PolyA and Tat-Rev transcripts were quantified by 2-step RT-dPCR using the QIAcuity Four 5-plex digital PCR system (Qiagen), as previously described<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 36" title="Tumpach, C. et al. Adaptation of droplet digital PCR-based HIV transcription profiling to digital PCR and association of HIV transcription and total or intact HIV DNA. Viruses 15, 1606 (2023)." href="https://www.nature.com/articles/s41467-025-60001-2#ref-CR36" id="ref-link-section-d326275717e2461">36</a></sup>. HIV RNA copies were normalised to RNA input. To quantify changes to the size of the proviral reservoir, cells were treated with Tat-LNP X for 120 h. DNA was extracted using AllPrep DNA/RNA Mini Kit (Qiagen) as per the manufacturer’s protocol. Intact, 5’ defective and 3’ defective HIV DNA was quantified using Qiagen’s QIAcuity Four 5-plex digital PCR targeting parts of the <i>psi</i> and <i>env</i> region, as previously described<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 58" title="Tumpach, C. et al. Adaptation of the intact proviral DNA assay to a nanowell-based digital PCR platform. J. Virus Erad. 9, 100335 (2023)." href="https://www.nature.com/articles/s41467-025-60001-2#ref-CR58" id="ref-link-section-d326275717e2472">58</a></sup>. The RPP30 host gene was used to measure the analysed number of cells and the DNA shearing index (DSI) as an indication of DNA fragmentation. The number of HIV copies was corrected to DSI and normalised to cell input.</p><h3 id="Sec19">Statistics</h3><p>Statistical analyses were performed in Graphpad Prism 10. A Student’s t test or ratio paired t test was used to compare two groups for primary cell transfection experiments involving uninfected cells or cells infected in vitro with pMorpheus-V5. For ex vivo experiments using CD4<sup>+</sup> T cells from people living with HIV, a one-sided (for induction of HIV RNA) or two-sided (for changes to intact proviral DNA levels) Wilcoxon signed-rank test was used to compare treatment conditions with untreated or positive controls.</p><h3 id="Sec20">Reporting summary</h3><p>Further information on research design is available in the <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41467-025-60001-2#MOESM3">Nature Portfolio Reporting Summary</a> linked to this article.</p></div></div></section>
                    
                </div><div>
                <section data-title="Data availability"><div id="data-availability-section"><h2 id="data-availability">Data availability</h2><p>Source data are provided for all experimental results presented in the main manuscript and supplementary information. <a data-track="click" data-track-label="link" data-track-action="section anchor" href="https://www.nature.com/articles/s41467-025-60001-2#Sec22">Source data</a> are provided with this paper.</p></div></section><div id="MagazineFulltextArticleBodySuffix"><section aria-labelledby="Bib1" data-title="References"><div id="Bib1-section"><h2 id="Bib1">References</h2><div id="Bib1-content"><div data-container-section="references"><ol data-track-component="outbound reference" data-track-context="references section"><li data-counter="1."><p id="ref-CR1">Luigetti, M. &amp; Servidei, S. Patisiran in hereditary transthyretin-mediated amyloidosis. <i>Lancet Neurol.</i> <b>20</b>, 21–23 (2021).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/S1474-4422(20)30397-5" data-track-item_id="10.1016/S1474-4422(20)30397-5" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2FS1474-4422%2820%2930397-5" aria-label="Article reference 1" data-doi="10.1016/S1474-4422(20)30397-5">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXitlKkur3F" aria-label="CAS reference 1">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33212064" aria-label="PubMed reference 1">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 1" href="http://scholar.google.com/scholar_lookup?&amp;title=Patisiran%20in%20hereditary%20transthyretin-mediated%20amyloidosis&amp;journal=Lancet%20Neurol.&amp;doi=10.1016%2FS1474-4422%2820%2930397-5&amp;volume=20&amp;pages=21-23&amp;publication_year=2021&amp;author=Luigetti%2CM&amp;author=Servidei%2CS">
                    Google Scholar</a> 
                </p></li><li data-counter="2."><p id="ref-CR2">Gillmore, J. D. et al. CRISPR-Cas9 in vivo gene editing for transthyretin amyloidosis. <i>N. Engl. J. Med.</i> <b>385</b>, 493–502 (2021).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1056/NEJMoa2107454" data-track-item_id="10.1056/NEJMoa2107454" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1056%2FNEJMoa2107454" aria-label="Article reference 2" data-doi="10.1056/NEJMoa2107454">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhslyisLbE" aria-label="CAS reference 2">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34215024" aria-label="PubMed reference 2">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 2" href="http://scholar.google.com/scholar_lookup?&amp;title=CRISPR-Cas9%20in%20vivo%20gene%20editing%20for%20transthyretin%20amyloidosis&amp;journal=N.%20Engl.%20J.%20Med.&amp;doi=10.1056%2FNEJMoa2107454&amp;volume=385&amp;pages=493-502&amp;publication_year=2021&amp;author=Gillmore%2CJD">
                    Google Scholar</a> 
                </p></li><li data-counter="3."><p id="ref-CR3">Deeks, S. G. et al. Research priorities for an HIV cure: International AIDS Society Global Scientific Strategy 2021. <i>Nat. Med.</i> <b>27</b>, 2085–2098 (2021).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41591-021-01590-5" data-track-item_id="10.1038/s41591-021-01590-5" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41591-021-01590-5" aria-label="Article reference 3" data-doi="10.1038/s41591-021-01590-5">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXis1arurrF" aria-label="CAS reference 3">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34848888" aria-label="PubMed reference 3">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 3" href="http://scholar.google.com/scholar_lookup?&amp;title=Research%20priorities%20for%20an%20HIV%20cure%3A%20International%20AIDS%20Society%20Global%20Scientific%20Strategy%202021&amp;journal=Nat.%20Med.&amp;doi=10.1038%2Fs41591-021-01590-5&amp;volume=27&amp;pages=2085-2098&amp;publication_year=2021&amp;author=Deeks%2CSG">
                    Google Scholar</a> 
                </p></li><li data-counter="4."><p id="ref-CR4">Chun, T.-W. et al. Presence of an inducible HIV-1 latent reservoir during highly active antiretroviral therapy. <i>Proc. Natl Acad. Sci. USA</i><b>94</b>, 13193–13197 (1997).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1073/pnas.94.24.13193" data-track-item_id="10.1073/pnas.94.24.13193" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1073%2Fpnas.94.24.13193" aria-label="Article reference 4" data-doi="10.1073/pnas.94.24.13193">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="ads reference" data-track-action="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=1997PNAS...9413193C" aria-label="ADS reference 4">ADS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK2sXnvFamtrc%3D" aria-label="CAS reference 4">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=9371822" aria-label="PubMed reference 4">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC24285" aria-label="PubMed Central reference 4">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 4" href="http://scholar.google.com/scholar_lookup?&amp;title=Presence%20of%20an%20inducible%20HIV-1%20latent%20reservoir%20during%20highly%20active%20antiretroviral%20therapy&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;doi=10.1073%2Fpnas.94.24.13193&amp;volume=94&amp;pages=13193-13197&amp;publication_year=1997&amp;author=Chun%2CT-W">
                    Google Scholar</a> 
                </p></li><li data-counter="5."><p id="ref-CR5">Finzi, D. et al. Identification of a reservoir for HIV-1 in patients on highly active antiretroviral therapy. <i>Science</i> <b>278</b>, 1295–1300 (1997).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1126/science.278.5341.1295" data-track-item_id="10.1126/science.278.5341.1295" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1126%2Fscience.278.5341.1295" aria-label="Article reference 5" data-doi="10.1126/science.278.5341.1295">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="ads reference" data-track-action="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=1997Sci...278.1295F" aria-label="ADS reference 5">ADS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK2sXntlOmsro%3D" aria-label="CAS reference 5">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=9360927" aria-label="PubMed reference 5">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 5" href="http://scholar.google.com/scholar_lookup?&amp;title=Identification%20of%20a%20reservoir%20for%20HIV-1%20in%20patients%20on%20highly%20active%20antiretroviral%20therapy&amp;journal=Science&amp;doi=10.1126%2Fscience.278.5341.1295&amp;volume=278&amp;pages=1295-1300&amp;publication_year=1997&amp;author=Finzi%2CD">
                    Google Scholar</a> 
                </p></li><li data-counter="6."><p id="ref-CR6">Wong, J. K. et al. Recovery of replication-competent HIV despite prolonged suppression of plasma viremia. <i>Science</i> <b>278</b>, 1291–1295 (1997).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1126/science.278.5341.1291" data-track-item_id="10.1126/science.278.5341.1291" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1126%2Fscience.278.5341.1291" aria-label="Article reference 6" data-doi="10.1126/science.278.5341.1291">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="ads reference" data-track-action="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=1997Sci...278.1291W" aria-label="ADS reference 6">ADS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK2sXntlOnu7k%3D" aria-label="CAS reference 6">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=9360926" aria-label="PubMed reference 6">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 6" href="http://scholar.google.com/scholar_lookup?&amp;title=Recovery%20of%20replication-competent%20HIV%20despite%20prolonged%20suppression%20of%20plasma%20viremia&amp;journal=Science&amp;doi=10.1126%2Fscience.278.5341.1291&amp;volume=278&amp;pages=1291-1295&amp;publication_year=1997&amp;author=Wong%2CJK">
                    Google Scholar</a> 
                </p></li><li data-counter="7."><p id="ref-CR7">Archin, N. M. et al. Administration of vorinostat disrupts HIV-1 latency in patients on antiretroviral therapy. <i>Nature</i> <b>487</b>, 482–485 (2012).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/nature11286" data-track-item_id="10.1038/nature11286" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fnature11286" aria-label="Article reference 7" data-doi="10.1038/nature11286">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="ads reference" data-track-action="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2012Natur.487..482A" aria-label="ADS reference 7">ADS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XhtFWjtrfP" aria-label="CAS reference 7">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22837004" aria-label="PubMed reference 7">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3704185" aria-label="PubMed Central reference 7">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 7" href="http://scholar.google.com/scholar_lookup?&amp;title=Administration%20of%20vorinostat%20disrupts%20HIV-1%20latency%20in%20patients%20on%20antiretroviral%20therapy&amp;journal=Nature&amp;doi=10.1038%2Fnature11286&amp;volume=487&amp;pages=482-485&amp;publication_year=2012&amp;author=Archin%2CNM">
                    Google Scholar</a> 
                </p></li><li data-counter="8."><p id="ref-CR8">Elliott, J. H. et al. Activation of HIV transcription with short-course vorinostat in HIV-infected patients on suppressive antiretroviral therapy. <i>PLoS Pathog.</i> <b>10</b>, 1–19 (2014).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1371/journal.ppat.1004473" data-track-item_id="10.1371/journal.ppat.1004473" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1371%2Fjournal.ppat.1004473" aria-label="Article reference 8" data-doi="10.1371/journal.ppat.1004473">Article</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 8" href="http://scholar.google.com/scholar_lookup?&amp;title=Activation%20of%20HIV%20transcription%20with%20short-course%20vorinostat%20in%20HIV-infected%20patients%20on%20suppressive%20antiretroviral%20therapy&amp;journal=PLoS%20Pathog.&amp;doi=10.1371%2Fjournal.ppat.1004473&amp;volume=10&amp;pages=1-19&amp;publication_year=2014&amp;author=Elliott%2CJH">
                    Google Scholar</a> 
                </p></li><li data-counter="9."><p id="ref-CR9">Kim, Y., Anderson, J. L. &amp; Lewin, S. R. Getting the “kill” into “shock and kill”: strategies to eliminate latent HIV. <i>Cell Host Microbe</i> <b>23</b>, 14–26 (2018).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.chom.2017.12.004" data-track-item_id="10.1016/j.chom.2017.12.004" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.chom.2017.12.004" aria-label="Article reference 9" data-doi="10.1016/j.chom.2017.12.004">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXjs1aksrw%3D" aria-label="CAS reference 9">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29324227" aria-label="PubMed reference 9">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5990418" aria-label="PubMed Central reference 9">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 9" href="http://scholar.google.com/scholar_lookup?&amp;title=Getting%20the%20%E2%80%9Ckill%E2%80%9D%20into%20%E2%80%9Cshock%20and%20kill%E2%80%9D%3A%20strategies%20to%20eliminate%20latent%20HIV&amp;journal=Cell%20Host%20Microbe&amp;doi=10.1016%2Fj.chom.2017.12.004&amp;volume=23&amp;pages=14-26&amp;publication_year=2018&amp;author=Kim%2CY&amp;author=Anderson%2CJL&amp;author=Lewin%2CSR">
                    Google Scholar</a> 
                </p></li><li data-counter="10."><p id="ref-CR10">Zerbato, J. M., Purves, H. V., Lewin, S. R. &amp; Rasmussen, T. A. Between a shock and a hard place: challenges and developments in HIV latency reversal. <i>Curr. Opin. Virol.</i> <b>38</b>, 1–9 (2019).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.coviro.2019.03.004" data-track-item_id="10.1016/j.coviro.2019.03.004" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.coviro.2019.03.004" aria-label="Article reference 10" data-doi="10.1016/j.coviro.2019.03.004">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31048093" aria-label="PubMed reference 10">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6819240" aria-label="PubMed Central reference 10">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 10" href="http://scholar.google.com/scholar_lookup?&amp;title=Between%20a%20shock%20and%20a%20hard%20place%3A%20challenges%20and%20developments%20in%20HIV%20latency%20reversal&amp;journal=Curr.%20Opin.%20Virol.&amp;doi=10.1016%2Fj.coviro.2019.03.004&amp;volume=38&amp;pages=1-9&amp;publication_year=2019&amp;author=Zerbato%2CJM&amp;author=Purves%2CHV&amp;author=Lewin%2CSR&amp;author=Rasmussen%2CTA">
                    Google Scholar</a> 
                </p></li><li data-counter="11."><p id="ref-CR11">Fidler, S. et al. Antiretroviral therapy alone versus antiretroviral therapy with a kick and kill approach, on measures of the HIV reservoir in participants with recent HIV infection (the RIVER trial): a phase 2, randomised trial. <i>Lancet</i> <b>395</b>, 888–898 (2020).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/S0140-6736(19)32990-3" data-track-item_id="10.1016/S0140-6736(19)32990-3" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2FS0140-6736%2819%2932990-3" aria-label="Article reference 11" data-doi="10.1016/S0140-6736(19)32990-3">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXjsValt7s%3D" aria-label="CAS reference 11">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32085823" aria-label="PubMed reference 11">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 11" href="http://scholar.google.com/scholar_lookup?&amp;title=Antiretroviral%20therapy%20alone%20versus%20antiretroviral%20therapy%20with%20a%20kick%20and%20kill%20approach%2C%20on%20measures%20of%20the%20HIV%20reservoir%20in%20participants%20with%20recent%20HIV%20infection%20%28the%20RIVER%20trial%29%3A%20a%20phase%202%2C%20randomised%20trial&amp;journal=Lancet&amp;doi=10.1016%2FS0140-6736%2819%2932990-3&amp;volume=395&amp;pages=888-898&amp;publication_year=2020&amp;author=Fidler%2CS">
                    Google Scholar</a> 
                </p></li><li data-counter="12."><p id="ref-CR12">Elliott, J. H. et al. Short-term administration of disulfiram for reversal of latent HIV infection: a phase 2 dose-escalation study. <i>Lancet HIV</i> <b>2</b>, e520–e529 (2015).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/S2352-3018(15)00226-X" data-track-item_id="10.1016/S2352-3018(15)00226-X" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2FS2352-3018%2815%2900226-X" aria-label="Article reference 12" data-doi="10.1016/S2352-3018(15)00226-X">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26614966" aria-label="PubMed reference 12">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5108570" aria-label="PubMed Central reference 12">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 12" href="http://scholar.google.com/scholar_lookup?&amp;title=Short-term%20administration%20of%20disulfiram%20for%20reversal%20of%20latent%20HIV%20infection%3A%20a%20phase%202%20dose-escalation%20study&amp;journal=Lancet%20HIV&amp;doi=10.1016%2FS2352-3018%2815%2900226-X&amp;volume=2&amp;pages=e520-e529&amp;publication_year=2015&amp;author=Elliott%2CJH">
                    Google Scholar</a> 
                </p></li><li data-counter="13."><p id="ref-CR13">Rasmussen, T. A. et al. Panobinostat, a histone deacetylase inhibitor, for latent-virus reactivation in HIV-infected patients on suppressive antiretroviral therapy: a phase 1/2, single group, clinical trial. <i>Lancet HIV</i> <b>1</b>, e13–21 (2014).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/S2352-3018(14)70014-1" data-track-item_id="10.1016/S2352-3018(14)70014-1" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2FS2352-3018%2814%2970014-1" aria-label="Article reference 13" data-doi="10.1016/S2352-3018(14)70014-1">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26423811" aria-label="PubMed reference 13">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 13" href="http://scholar.google.com/scholar_lookup?&amp;title=Panobinostat%2C%20a%20histone%20deacetylase%20inhibitor%2C%20for%20latent-virus%20reactivation%20in%20HIV-infected%20patients%20on%20suppressive%20antiretroviral%20therapy%3A%20a%20phase%201%2F2%2C%20single%20group%2C%20clinical%20trial&amp;journal=Lancet%20HIV&amp;doi=10.1016%2FS2352-3018%2814%2970014-1&amp;volume=1&amp;pages=e13-21&amp;publication_year=2014&amp;author=Rasmussen%2CTA">
                    Google Scholar</a> 
                </p></li><li data-counter="14."><p id="ref-CR14">Gutierrez, C. et al. Bryostatin-1 for latent virus reactivation in HIV-infected patients on antiretroviral therapy. <i>AIDS</i> <b>30</b>, 1385–1392 (2016).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1097/QAD.0000000000001064" data-track-item_id="10.1097/QAD.0000000000001064" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1097%2FQAD.0000000000001064" aria-label="Article reference 14" data-doi="10.1097/QAD.0000000000001064">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XnvVGht74%3D" aria-label="CAS reference 14">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26891037" aria-label="PubMed reference 14">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 14" href="http://scholar.google.com/scholar_lookup?&amp;title=Bryostatin-1%20for%20latent%20virus%20reactivation%20in%20HIV-infected%20patients%20on%20antiretroviral%20therapy&amp;journal=AIDS&amp;doi=10.1097%2FQAD.0000000000001064&amp;volume=30&amp;pages=1385-1392&amp;publication_year=2016&amp;author=Gutierrez%2CC">
                    Google Scholar</a> 
                </p></li><li data-counter="15."><p id="ref-CR15">Sogaard, O. S. et al. The depsipeptide romidepsin reverses HIV-1 latency in vivo. <i>PLoS Pathog.</i> <b>11</b>, e1005142 (2015).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1371/journal.ppat.1005142" data-track-item_id="10.1371/journal.ppat.1005142" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1371%2Fjournal.ppat.1005142" aria-label="Article reference 15" data-doi="10.1371/journal.ppat.1005142">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26379282" aria-label="PubMed reference 15">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4575032" aria-label="PubMed Central reference 15">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 15" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20depsipeptide%20romidepsin%20reverses%20HIV-1%20latency%20in%20vivo&amp;journal=PLoS%20Pathog.&amp;doi=10.1371%2Fjournal.ppat.1005142&amp;volume=11&amp;publication_year=2015&amp;author=Sogaard%2COS">
                    Google Scholar</a> 
                </p></li><li data-counter="16."><p id="ref-CR16">Uldrick, T. S. et al. Assessment of the safety of pembrolizumab in patients with HIV and advanced cancer-a phase 1 study. <i>JAMA Oncol.</i> <b>5</b>, 1332–1339 (2019).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1001/jamaoncol.2019.2244" data-track-item_id="10.1001/jamaoncol.2019.2244" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1001%2Fjamaoncol.2019.2244" aria-label="Article reference 16" data-doi="10.1001/jamaoncol.2019.2244">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31154457" aria-label="PubMed reference 16">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6547135" aria-label="PubMed Central reference 16">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 16" href="http://scholar.google.com/scholar_lookup?&amp;title=Assessment%20of%20the%20safety%20of%20pembrolizumab%20in%20patients%20with%20HIV%20and%20advanced%20cancer-a%20phase%201%20study&amp;journal=JAMA%20Oncol.&amp;doi=10.1001%2Fjamaoncol.2019.2244&amp;volume=5&amp;pages=1332-1339&amp;publication_year=2019&amp;author=Uldrick%2CTS">
                    Google Scholar</a> 
                </p></li><li data-counter="17."><p id="ref-CR17">McMahon, J. H. et al. Neurotoxicity with high-dose disulfiram and vorinostat used for HIV latency reversal. <i>AIDS</i> <b>36</b>, 75–82 (2022).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1097/QAD.0000000000003091" data-track-item_id="10.1097/QAD.0000000000003091" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1097%2FQAD.0000000000003091" aria-label="Article reference 17" data-doi="10.1097/QAD.0000000000003091">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXis1KmsrbM" aria-label="CAS reference 17">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34586085" aria-label="PubMed reference 17">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 17" href="http://scholar.google.com/scholar_lookup?&amp;title=Neurotoxicity%20with%20high-dose%20disulfiram%20and%20vorinostat%20used%20for%20HIV%20latency%20reversal&amp;journal=AIDS&amp;doi=10.1097%2FQAD.0000000000003091&amp;volume=36&amp;pages=75-82&amp;publication_year=2022&amp;author=McMahon%2CJH">
                    Google Scholar</a> 
                </p></li><li data-counter="18."><p id="ref-CR18">Pace, M. et al. Histone deacetylase inhibitors enhance CD4 T cell susceptibility to NK cell killing but reduce NK cell function. <i>PLoS Pathog.</i> <b>12</b>, e1005782 (2016).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1371/journal.ppat.1005782" data-track-item_id="10.1371/journal.ppat.1005782" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1371%2Fjournal.ppat.1005782" aria-label="Article reference 18" data-doi="10.1371/journal.ppat.1005782">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27529554" aria-label="PubMed reference 18">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4986965" aria-label="PubMed Central reference 18">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 18" href="http://scholar.google.com/scholar_lookup?&amp;title=Histone%20deacetylase%20inhibitors%20enhance%20CD4%20T%20cell%20susceptibility%20to%20NK%20cell%20killing%20but%20reduce%20NK%20cell%20function&amp;journal=PLoS%20Pathog.&amp;doi=10.1371%2Fjournal.ppat.1005782&amp;volume=12&amp;publication_year=2016&amp;author=Pace%2CM">
                    Google Scholar</a> 
                </p></li><li data-counter="19."><p id="ref-CR19">Jones, R. B. et al. Histone deacetylase inhibitors impair the elimination of HIV-infected cells by cytotoxic T-lymphocytes. <i>PLoS Pathog.</i> <b>10</b>, e1004287 (2014).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1371/journal.ppat.1004287" data-track-item_id="10.1371/journal.ppat.1004287" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1371%2Fjournal.ppat.1004287" aria-label="Article reference 19" data-doi="10.1371/journal.ppat.1004287">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25122219" aria-label="PubMed reference 19">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4133386" aria-label="PubMed Central reference 19">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 19" href="http://scholar.google.com/scholar_lookup?&amp;title=Histone%20deacetylase%20inhibitors%20impair%20the%20elimination%20of%20HIV-infected%20cells%20by%20cytotoxic%20T-lymphocytes&amp;journal=PLoS%20Pathog.&amp;doi=10.1371%2Fjournal.ppat.1004287&amp;volume=10&amp;publication_year=2014&amp;author=Jones%2CRB">
                    Google Scholar</a> 
                </p></li><li data-counter="20."><p id="ref-CR20">Das, A. T., Harwig, A. &amp; Berkhout, B. The HIV-1 tat protein has a versatile role in activating viral transcription. <i>J. Virol.</i> <b>85</b>, 9506–9516 (2011).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1128/JVI.00650-11" data-track-item_id="10.1128/JVI.00650-11" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1128%2FJVI.00650-11" aria-label="Article reference 20" data-doi="10.1128/JVI.00650-11">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXhtFaqsLbN" aria-label="CAS reference 20">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21752913" aria-label="PubMed reference 20">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3165771" aria-label="PubMed Central reference 20">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 20" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20HIV-1%20tat%20protein%20has%20a%20versatile%20role%20in%20activating%20viral%20transcription&amp;journal=J.%20Virol.&amp;doi=10.1128%2FJVI.00650-11&amp;volume=85&amp;pages=9506-9516&amp;publication_year=2011&amp;author=Das%2CAT&amp;author=Harwig%2CA&amp;author=Berkhout%2CB">
                    Google Scholar</a> 
                </p></li><li data-counter="21."><p id="ref-CR21">Raines, S. L. M. et al. Nanoparticle delivery of Tat synergizes with classical latency reversal agents to express HIV antigen targets. <i>Antimicrob. Agents Chemother.</i> <b>68</b>, e00201-24 (2024).</p></li><li data-counter="22."><p id="ref-CR22">Van Gulck, E. et al. A truncated HIV Tat demonstrates potent and specific latency reversal activity. <i>Antimicrob. Agents Chemother.</i> <b>67</b>, e0041723 (2023).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1128/aac.00417-23" data-track-item_id="10.1128/aac.00417-23" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1128%2Faac.00417-23" aria-label="Article reference 22" data-doi="10.1128/aac.00417-23">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=37874295" aria-label="PubMed reference 22">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 22" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20truncated%20HIV%20Tat%20demonstrates%20potent%20and%20specific%20latency%20reversal%20activity&amp;journal=Antimicrob.%20Agents%20Chemother.&amp;doi=10.1128%2Faac.00417-23&amp;volume=67&amp;publication_year=2023&amp;author=Gulck%2CE">
                    Google Scholar</a> 
                </p></li><li data-counter="23."><p id="ref-CR23">Zhang, Y. et al. CRISPR/gRNA-directed synergistic activation mediator (SAM) induces specific, persistent and robust reactivation of the HIV-1 latent reservoirs. <i>Sci. Rep.</i> <b>5</b>, 16277–16277 (2015).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/srep16277" data-track-item_id="10.1038/srep16277" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fsrep16277" aria-label="Article reference 23" data-doi="10.1038/srep16277">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="ads reference" data-track-action="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2015NatSR...516277Z" aria-label="ADS reference 23">ADS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXhslyhurfN" aria-label="CAS reference 23">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26538064" aria-label="PubMed reference 23">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4633726" aria-label="PubMed Central reference 23">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 23" href="http://scholar.google.com/scholar_lookup?&amp;title=CRISPR%2FgRNA-directed%20synergistic%20activation%20mediator%20%28SAM%29%20induces%20specific%2C%20persistent%20and%20robust%20reactivation%20of%20the%20HIV-1%20latent%20reservoirs&amp;journal=Sci.%20Rep.&amp;doi=10.1038%2Fsrep16277&amp;volume=5&amp;pages=16277-16277&amp;publication_year=2015&amp;author=Zhang%2CY">
                    Google Scholar</a> 
                </p></li><li data-counter="24."><p id="ref-CR24">Bialek, J. K. et al. Targeted HIV-1 latency reversal using CRISPR/Cas9-derived transcriptional activator systems. <i>PLoS One</i> <b>11</b>, e0158294–e0158294 (2016).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1371/journal.pone.0158294" data-track-item_id="10.1371/journal.pone.0158294" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1371%2Fjournal.pone.0158294" aria-label="Article reference 24" data-doi="10.1371/journal.pone.0158294">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27341108" aria-label="PubMed reference 24">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4920395" aria-label="PubMed Central reference 24">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 24" href="http://scholar.google.com/scholar_lookup?&amp;title=Targeted%20HIV-1%20latency%20reversal%20using%20CRISPR%2FCas9-derived%20transcriptional%20activator%20systems&amp;journal=PLoS%20One&amp;doi=10.1371%2Fjournal.pone.0158294&amp;volume=11&amp;pages=e0158294-e0158294&amp;publication_year=2016&amp;author=Bialek%2CJK">
                    Google Scholar</a> 
                </p></li><li data-counter="25."><p id="ref-CR25">Ji, H. et al. Specific reactivation of latent HIV-1 by dCas9-SunTag-VP64-mediated guide RNA targeting the HIV-1 promoter. <i>Mol. Ther.</i> <b>24</b>, 508–521 (2016).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/mt.2016.7" data-track-item_id="10.1038/mt.2016.7" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fmt.2016.7" aria-label="Article reference 25" data-doi="10.1038/mt.2016.7">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26775808" aria-label="PubMed reference 25">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4786936" aria-label="PubMed Central reference 25">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 25" href="http://scholar.google.com/scholar_lookup?&amp;title=Specific%20reactivation%20of%20latent%20HIV-1%20by%20dCas9-SunTag-VP64-mediated%20guide%20RNA%20targeting%20the%20HIV-1%20promoter&amp;journal=Mol.%20Ther.&amp;doi=10.1038%2Fmt.2016.7&amp;volume=24&amp;pages=508-521&amp;publication_year=2016&amp;author=Ji%2CH">
                    Google Scholar</a> 
                </p></li><li data-counter="26."><p id="ref-CR26">Limsirichai, P., Gaj, T. &amp; Schaffer, D. V. CRISPR-mediated activation of latent HIV-1 expression. <i>Mol. Ther.</i> <b>24</b>, 499–507 (2016).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/mt.2015.213" data-track-item_id="10.1038/mt.2015.213" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fmt.2015.213" aria-label="Article reference 26" data-doi="10.1038/mt.2015.213">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XjslCmtg%3D%3D" aria-label="CAS reference 26">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26607397" aria-label="PubMed reference 26">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4786916" aria-label="PubMed Central reference 26">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 26" href="http://scholar.google.com/scholar_lookup?&amp;title=CRISPR-mediated%20activation%20of%20latent%20HIV-1%20expression&amp;journal=Mol.%20Ther.&amp;doi=10.1038%2Fmt.2015.213&amp;volume=24&amp;pages=499-507&amp;publication_year=2016&amp;author=Limsirichai%2CP&amp;author=Gaj%2CT&amp;author=Schaffer%2CDV">
                    Google Scholar</a> 
                </p></li><li data-counter="27."><p id="ref-CR27">Saayman, S. M. et al. Potent and targeted activation of latent HIV-1 using the CRISPR/dCas9 activator vomplex. <i>Mol. Ther.</i> <b>24</b>, 488–498 (2016).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/mt.2015.202" data-track-item_id="10.1038/mt.2015.202" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fmt.2015.202" aria-label="Article reference 27" data-doi="10.1038/mt.2015.202">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XjslCnuw%3D%3D" aria-label="CAS reference 27">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26581162" aria-label="PubMed reference 27">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4786915" aria-label="PubMed Central reference 27">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 27" href="http://scholar.google.com/scholar_lookup?&amp;title=Potent%20and%20targeted%20activation%20of%20latent%20HIV-1%20using%20the%20CRISPR%2FdCas9%20activator%20vomplex&amp;journal=Mol.%20Ther.&amp;doi=10.1038%2Fmt.2015.202&amp;volume=24&amp;pages=488-498&amp;publication_year=2016&amp;author=Saayman%2CSM">
                    Google Scholar</a> 
                </p></li><li data-counter="28."><p id="ref-CR28">Klinnert, S., Chemnitzer, A., Rusert, P. &amp; Metzner, K. J. Systematic HIV-1 promoter targeting with CRISPR/dCas9-VPR reveals optimal region for activation of the latent provirus. <i>J. Gen. Virol.</i> <b>103</b>, 1–10 (2022).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1099/jgv.0.001754" data-track-item_id="10.1099/jgv.0.001754" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1099%2Fjgv.0.001754" aria-label="Article reference 28" data-doi="10.1099/jgv.0.001754">Article</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 28" href="http://scholar.google.com/scholar_lookup?&amp;title=Systematic%20HIV-1%20promoter%20targeting%20with%20CRISPR%2FdCas9-VPR%20reveals%20optimal%20region%20for%20activation%20of%20the%20latent%20provirus&amp;journal=J.%20Gen.%20Virol.&amp;doi=10.1099%2Fjgv.0.001754&amp;volume=103&amp;pages=1-10&amp;publication_year=2022&amp;author=Klinnert%2CS&amp;author=Chemnitzer%2CA&amp;author=Rusert%2CP&amp;author=Metzner%2CKJ">
                    Google Scholar</a> 
                </p></li><li data-counter="29."><p id="ref-CR29">Cevaal, P. M. et al. In Vivo T cell-targeting nanoparticle drug delivery systems: considerations for rational design. <i>ACS Nano</i> <b>15</b>, 3736–3753 (2021).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1021/acsnano.0c09514" data-track-item_id="10.1021/acsnano.0c09514" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1021%2Facsnano.0c09514" aria-label="Article reference 29" data-doi="10.1021/acsnano.0c09514">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXkt1Orsr0%3D" aria-label="CAS reference 29">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33600163" aria-label="PubMed reference 29">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 29" href="http://scholar.google.com/scholar_lookup?&amp;title=In%20Vivo%20T%20cell-targeting%20nanoparticle%20drug%20delivery%20systems%3A%20considerations%20for%20rational%20design&amp;journal=ACS%20Nano&amp;doi=10.1021%2Facsnano.0c09514&amp;volume=15&amp;pages=3736-3753&amp;publication_year=2021&amp;author=Cevaal%2CPM">
                    Google Scholar</a> 
                </p></li><li data-counter="30."><p id="ref-CR30">Sabnis, S. et al. A novel amino lipid series for mRNA delivery: improved endosomal escape and sustained pharmacology and safety in non-human primates. <i>Mol. Ther.</i> <b>26</b>, 1509–1519 (2018).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.ymthe.2018.03.010" data-track-item_id="10.1016/j.ymthe.2018.03.010" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.ymthe.2018.03.010" aria-label="Article reference 30" data-doi="10.1016/j.ymthe.2018.03.010">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtV2lsrbM" aria-label="CAS reference 30">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29653760" aria-label="PubMed reference 30">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5986714" aria-label="PubMed Central reference 30">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 30" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20novel%20amino%20lipid%20series%20for%20mRNA%20delivery%3A%20improved%20endosomal%20escape%20and%20sustained%20pharmacology%20and%20safety%20in%20non-human%20primates&amp;journal=Mol.%20Ther.&amp;doi=10.1016%2Fj.ymthe.2018.03.010&amp;volume=26&amp;pages=1509-1519&amp;publication_year=2018&amp;author=Sabnis%2CS">
                    Google Scholar</a> 
                </p></li><li data-counter="31."><p id="ref-CR31">Patel, S. et al. Naturally-occurring cholesterol analogues in lipid nanoparticles induce polymorphic shape and enhance intracellular delivery of mRNA. <i>Nat. Commun.</i> <b>11</b>, 983 (2020).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41467-020-14527-2" data-track-item_id="10.1038/s41467-020-14527-2" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41467-020-14527-2" aria-label="Article reference 31" data-doi="10.1038/s41467-020-14527-2">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="ads reference" data-track-action="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2020NatCo..11..983P" aria-label="ADS reference 31">ADS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXlt1Shsbs%3D" aria-label="CAS reference 31">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32080183" aria-label="PubMed reference 31">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7033178" aria-label="PubMed Central reference 31">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 31" href="http://scholar.google.com/scholar_lookup?&amp;title=Naturally-occurring%20cholesterol%20analogues%20in%20lipid%20nanoparticles%20induce%20polymorphic%20shape%20and%20enhance%20intracellular%20delivery%20of%20mRNA&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-020-14527-2&amp;volume=11&amp;publication_year=2020&amp;author=Patel%2CS">
                    Google Scholar</a> 
                </p></li><li data-counter="32."><p id="ref-CR32">FitzGerald, L. I. <i>et al</i>. A molecular sensor to quantify the localization of proteins, DNA and nanoparticles in cells. <i>Nat. Commun</i>. <b>11</b> 4482 (2020).</p></li><li data-counter="33."><p id="ref-CR33">Liu, H. et al. Beyond the endosomal bottleneck: understanding the efficiency of mRNA/LNP delivery. <i>Adv. Funct. Mater</i>. <b>34</b>, 2404510 (2024).</p></li><li data-counter="34."><p id="ref-CR34">Purcell, D. F. &amp; Martin, M. A. Alternative splicing of human immunodeficiency virus type 1 mRNA modulates viral protein expression, replication, and infectivity. <i>J. Virol.</i> <b>67</b>, 6365–6378 (1993).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1128/jvi.67.11.6365-6378.1993" data-track-item_id="10.1128/jvi.67.11.6365-6378.1993" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1128%2Fjvi.67.11.6365-6378.1993" aria-label="Article reference 34" data-doi="10.1128/jvi.67.11.6365-6378.1993">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK2cXkt1yl" aria-label="CAS reference 34">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=8411338" aria-label="PubMed reference 34">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC238071" aria-label="PubMed Central reference 34">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 34" href="http://scholar.google.com/scholar_lookup?&amp;title=Alternative%20splicing%20of%20human%20immunodeficiency%20virus%20type%201%20mRNA%20modulates%20viral%20protein%20expression%2C%20replication%2C%20and%20infectivity&amp;journal=J.%20Virol.&amp;doi=10.1128%2Fjvi.67.11.6365-6378.1993&amp;volume=67&amp;pages=6365-6378&amp;publication_year=1993&amp;author=Purcell%2CDF&amp;author=Martin%2CMA">
                    Google Scholar</a> 
                </p></li><li data-counter="35."><p id="ref-CR35">Kim, E. H. et al. Development of an HIV reporter virus that identifies latently infected CD4(+) T cells. <i>Cell Rep. Methods</i> <b>2</b>, 100238 (2022).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.crmeth.2022.100238" data-track-item_id="10.1016/j.crmeth.2022.100238" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.crmeth.2022.100238" aria-label="Article reference 35" data-doi="10.1016/j.crmeth.2022.100238">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XitVWgsbnF" aria-label="CAS reference 35">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=35784650" aria-label="PubMed reference 35">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9243624" aria-label="PubMed Central reference 35">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 35" href="http://scholar.google.com/scholar_lookup?&amp;title=Development%20of%20an%20HIV%20reporter%20virus%20that%20identifies%20latently%20infected%20CD4%28%2B%29%20T%20cells&amp;journal=Cell%20Rep.%20Methods&amp;doi=10.1016%2Fj.crmeth.2022.100238&amp;volume=2&amp;publication_year=2022&amp;author=Kim%2CEH">
                    Google Scholar</a> 
                </p></li><li data-counter="36."><p id="ref-CR36">Tumpach, C. et al. Adaptation of droplet digital PCR-based HIV transcription profiling to digital PCR and association of HIV transcription and total or intact HIV DNA. <i>Viruses</i> <b>15</b>, 1606 (2023).</p></li><li data-counter="37."><p id="ref-CR37">Yukl, S. A. et al. HIV latency in isolated patient CD4(+) T cells may be due to blocks in HIV transcriptional elongation, completion, and splicing. <i>Sci. Transl. Med</i>. <b>10</b> (2018).</p></li><li data-counter="38."><p id="ref-CR38">Zerbato, J. M. et al. Multiply spliced HIV RNA is a predictive measure of virus production ex vivo and in vivo following reversal of HIV latency. <i>EBioMedicine</i> <b>65</b>, 103241 (2021).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.ebiom.2021.103241" data-track-item_id="10.1016/j.ebiom.2021.103241" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.ebiom.2021.103241" aria-label="Article reference 38" data-doi="10.1016/j.ebiom.2021.103241">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXotlOku7o%3D" aria-label="CAS reference 38">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33647768" aria-label="PubMed reference 38">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7920823" aria-label="PubMed Central reference 38">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 38" href="http://scholar.google.com/scholar_lookup?&amp;title=Multiply%20spliced%20HIV%20RNA%20is%20a%20predictive%20measure%20of%20virus%20production%20ex%20vivo%20and%20in%20vivo%20following%20reversal%20of%20HIV%20latency&amp;journal=EBioMedicine&amp;doi=10.1016%2Fj.ebiom.2021.103241&amp;volume=65&amp;publication_year=2021&amp;author=Zerbato%2CJM">
                    Google Scholar</a> 
                </p></li><li data-counter="39."><p id="ref-CR39">Konermann, S. et al. Genome-scale transcriptional activation by an engineered CRISPR-Cas9 complex. <i>Nature</i> <b>517</b>, 583–588 (2015).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/nature14136" data-track-item_id="10.1038/nature14136" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fnature14136" aria-label="Article reference 39" data-doi="10.1038/nature14136">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="ads reference" data-track-action="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2015Natur.517..583K" aria-label="ADS reference 39">ADS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXhvVWhtbo%3D" aria-label="CAS reference 39">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25494202" aria-label="PubMed reference 39">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 39" href="http://scholar.google.com/scholar_lookup?&amp;title=Genome-scale%20transcriptional%20activation%20by%20an%20engineered%20CRISPR-Cas9%20complex&amp;journal=Nature&amp;doi=10.1038%2Fnature14136&amp;volume=517&amp;pages=583-588&amp;publication_year=2015&amp;author=Konermann%2CS">
                    Google Scholar</a> 
                </p></li><li data-counter="40."><p id="ref-CR40">Tantale, K. et al. Stochastic pausing at latent HIV-1 promoters generates transcriptional bursting. <i>Nat. Commun.</i> <b>12</b>, 4503 (2021).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41467-021-24462-5" data-track-item_id="10.1038/s41467-021-24462-5" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41467-021-24462-5" aria-label="Article reference 40" data-doi="10.1038/s41467-021-24462-5">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="ads reference" data-track-action="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2021NatCo..12.4503T" aria-label="ADS reference 40">ADS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhsleqsrrL" aria-label="CAS reference 40">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34301927" aria-label="PubMed reference 40">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8302722" aria-label="PubMed Central reference 40">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 40" href="http://scholar.google.com/scholar_lookup?&amp;title=Stochastic%20pausing%20at%20latent%20HIV-1%20promoters%20generates%20transcriptional%20bursting&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-021-24462-5&amp;volume=12&amp;publication_year=2021&amp;author=Tantale%2CK">
                    Google Scholar</a> 
                </p></li><li data-counter="41."><p id="ref-CR41">Patel, S. K. et al. Hydroxycholesterol substitution in ionizable lipid nanoparticles for mRNA delivery to T cells. <i>J. Control. Release</i> <b>347</b>, 521–532 (2022).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.jconrel.2022.05.020" data-track-item_id="10.1016/j.jconrel.2022.05.020" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.jconrel.2022.05.020" aria-label="Article reference 41" data-doi="10.1016/j.jconrel.2022.05.020">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XhtlGqsb%2FN" aria-label="CAS reference 41">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=35569584" aria-label="PubMed reference 41">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9376797" aria-label="PubMed Central reference 41">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 41" href="http://scholar.google.com/scholar_lookup?&amp;title=Hydroxycholesterol%20substitution%20in%20ionizable%20lipid%20nanoparticles%20for%20mRNA%20delivery%20to%20T%20cells&amp;journal=J.%20Control.%20Release&amp;doi=10.1016%2Fj.jconrel.2022.05.020&amp;volume=347&amp;pages=521-532&amp;publication_year=2022&amp;author=Patel%2CSK">
                    Google Scholar</a> 
                </p></li><li data-counter="42."><p id="ref-CR42">Pardons, M. et al. Potent latency reversal by Tat RNA-containing nanoparticle enables multi-omic analysis of the HIV-1 reservoir. <i>Nat. Commun.</i> <b>14</b>, 8397 (2023).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41467-023-44020-5" data-track-item_id="10.1038/s41467-023-44020-5" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41467-023-44020-5" aria-label="Article reference 42" data-doi="10.1038/s41467-023-44020-5">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="ads reference" data-track-action="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2023NatCo..14.8397P" aria-label="ADS reference 42">ADS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXis1KgtrnL" aria-label="CAS reference 42">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=38110433" aria-label="PubMed reference 42">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10728105" aria-label="PubMed Central reference 42">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 42" href="http://scholar.google.com/scholar_lookup?&amp;title=Potent%20latency%20reversal%20by%20Tat%20RNA-containing%20nanoparticle%20enables%20multi-omic%20analysis%20of%20the%20HIV-1%20reservoir&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-023-44020-5&amp;volume=14&amp;publication_year=2023&amp;author=Pardons%2CM">
                    Google Scholar</a> 
                </p></li><li data-counter="43."><p id="ref-CR43">Arandjelovic, P. et al. Venetoclax, alone and in combination with the BH3 mimetic S63845, depletes HIV-1 latently infected cells and delays rebound in humanized mice. <i>Cell Rep. Med.</i> <b>4</b>, 101178 (2023).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.xcrm.2023.101178" data-track-item_id="10.1016/j.xcrm.2023.101178" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.xcrm.2023.101178" aria-label="Article reference 43" data-doi="10.1016/j.xcrm.2023.101178">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXhvVKns7%2FI" aria-label="CAS reference 43">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=37652018" aria-label="PubMed reference 43">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10518630" aria-label="PubMed Central reference 43">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 43" href="http://scholar.google.com/scholar_lookup?&amp;title=Venetoclax%2C%20alone%20and%20in%20combination%20with%20the%20BH3%20mimetic%20S63845%2C%20depletes%20HIV-1%20latently%20infected%20cells%20and%20delays%20rebound%20in%20humanized%20mice&amp;journal=Cell%20Rep.%20Med.&amp;doi=10.1016%2Fj.xcrm.2023.101178&amp;volume=4&amp;publication_year=2023&amp;author=Arandjelovic%2CP">
                    Google Scholar</a> 
                </p></li><li data-counter="44."><p id="ref-CR44">Ren, Y. et al. BCL-2 antagonism sensitizes cytotoxic T cell-resistant HIV reservoirs to elimination ex vivo. <i>J. Clin. Investig.</i> <b>130</b>, 2542–2559 (2020).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1172/JCI132374" data-track-item_id="10.1172/JCI132374" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1172%2FJCI132374" aria-label="Article reference 44" data-doi="10.1172/JCI132374">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXpt12ktrs%3D" aria-label="CAS reference 44">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32027622" aria-label="PubMed reference 44">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7191002" aria-label="PubMed Central reference 44">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 44" href="http://scholar.google.com/scholar_lookup?&amp;title=BCL-2%20antagonism%20sensitizes%20cytotoxic%20T%20cell-resistant%20HIV%20reservoirs%20to%20elimination%20ex%20vivo&amp;journal=J.%20Clin.%20Investig.&amp;doi=10.1172%2FJCI132374&amp;volume=130&amp;pages=2542-2559&amp;publication_year=2020&amp;author=Ren%2CY">
                    Google Scholar</a> 
                </p></li><li data-counter="45."><p id="ref-CR45">Kuo, H. H. et al. Anti-apoptotic protein BIRC5 maintains survival of HIV-1-infected CD4(+) T cells. <i>Immunity</i> <b>48</b>, 1183–1194 e1185 (2018).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.immuni.2018.04.004" data-track-item_id="10.1016/j.immuni.2018.04.004" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.immuni.2018.04.004" aria-label="Article reference 45" data-doi="10.1016/j.immuni.2018.04.004">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXpvFWktbs%3D" aria-label="CAS reference 45">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29802019" aria-label="PubMed reference 45">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6013384" aria-label="PubMed Central reference 45">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 45" href="http://scholar.google.com/scholar_lookup?&amp;title=Anti-apoptotic%20protein%20BIRC5%20maintains%20survival%20of%20HIV-1-infected%20CD4%28%2B%29%20T%20cells&amp;journal=Immunity&amp;doi=10.1016%2Fj.immuni.2018.04.004&amp;volume=48&amp;pages=1183-1194%20e1185&amp;publication_year=2018&amp;author=Kuo%2CHH">
                    Google Scholar</a> 
                </p></li><li data-counter="46."><p id="ref-CR46">Dashti, A. et al. AZD5582 plus SIV-specific antibodies reduce lymph node viral reservoirs in antiretroviral therapy-suppressed macaques. <i>Nat. Med.</i> <b>29</b>, 2535–2546 (2023).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41591-023-02570-7" data-track-item_id="10.1038/s41591-023-02570-7" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41591-023-02570-7" aria-label="Article reference 46" data-doi="10.1038/s41591-023-02570-7">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXitVeqsrvN" aria-label="CAS reference 46">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=37783968" aria-label="PubMed reference 46">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10579098" aria-label="PubMed Central reference 46">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 46" href="http://scholar.google.com/scholar_lookup?&amp;title=AZD5582%20plus%20SIV-specific%20antibodies%20reduce%20lymph%20node%20viral%20reservoirs%20in%20antiretroviral%20therapy-suppressed%20macaques&amp;journal=Nat.%20Med.&amp;doi=10.1038%2Fs41591-023-02570-7&amp;volume=29&amp;pages=2535-2546&amp;publication_year=2023&amp;author=Dashti%2CA">
                    Google Scholar</a> 
                </p></li><li data-counter="47."><p id="ref-CR47">Dufour, C., Gantner, P., Fromentin, R. &amp; Chomont, N. The multifaceted nature of HIV latency. <i>J. Clin. Investig</i>. <b>130</b>, 3381–3390 (2021).</p></li><li data-counter="48."><p id="ref-CR48">Fisher, B. M., Cevaal, P. M., Roche, M. &amp; Lewin, S. R. HIV Tat as a latency reversing agent: Turning the tables on viral persistence. <i>Front. Immunol</i>. <b>16</b>, 1571151 (2025).</p></li><li data-counter="49."><p id="ref-CR49">Beyersdorf, J. P. et al. Robust, Durable Gene Activation In Vivo via mRNA-Encoded Activators. <i>ACS Nano</i> <b>16</b>, 5671–5671 (2022).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1021/acsnano.1c10631" data-track-item_id="10.1021/acsnano.1c10631" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1021%2Facsnano.1c10631" aria-label="Article reference 49" data-doi="10.1021/acsnano.1c10631">Article</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 49" href="http://scholar.google.com/scholar_lookup?&amp;title=Robust%2C%20Durable%20Gene%20Activation%20In%20Vivo%20via%20mRNA-Encoded%20Activators&amp;journal=ACS%20Nano&amp;doi=10.1021%2Facsnano.1c10631&amp;volume=16&amp;pages=5671-5671&amp;publication_year=2022&amp;author=Beyersdorf%2CJP">
                    Google Scholar</a> 
                </p></li><li data-counter="50."><p id="ref-CR50">Dash, P. K. et al. CRISPR editing of CCR5 and HIV-1 facilitates viral elimination in antiretroviral drug-suppressed virus-infected humanized mice. <i>Proc. Natl. Acad. Sci. USA</i> <b>120</b>, e2217887120 (2023).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1073/pnas.2217887120" data-track-item_id="10.1073/pnas.2217887120" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1073%2Fpnas.2217887120" aria-label="Article reference 50" data-doi="10.1073/pnas.2217887120">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXhtFelu7zP" aria-label="CAS reference 50">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=37126704" aria-label="PubMed reference 50">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10175831" aria-label="PubMed Central reference 50">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 50" href="http://scholar.google.com/scholar_lookup?&amp;title=CRISPR%20editing%20of%20CCR5%20and%20HIV-1%20facilitates%20viral%20elimination%20in%20antiretroviral%20drug-suppressed%20virus-infected%20humanized%20mice&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;doi=10.1073%2Fpnas.2217887120&amp;volume=120&amp;publication_year=2023&amp;author=Dash%2CPK">
                    Google Scholar</a> 
                </p></li><li data-counter="51."><p id="ref-CR51">Dash, P. K. et al. Sequential lasER ART and CRISPR treatments eliminate HIV-1 in a subset of infected humanized mice. <i>Nat. Commun.</i> <b>10</b>, 2753–2753 (2019).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41467-019-10366-y" data-track-item_id="10.1038/s41467-019-10366-y" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41467-019-10366-y" aria-label="Article reference 51" data-doi="10.1038/s41467-019-10366-y">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="ads reference" data-track-action="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2019NatCo..10.2753D" aria-label="ADS reference 51">ADS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31266936" aria-label="PubMed reference 51">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6606613" aria-label="PubMed Central reference 51">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 51" href="http://scholar.google.com/scholar_lookup?&amp;title=Sequential%20lasER%20ART%20and%20CRISPR%20treatments%20eliminate%20HIV-1%20in%20a%20subset%20of%20infected%20humanized%20mice&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-019-10366-y&amp;volume=10&amp;pages=2753-2753&amp;publication_year=2019&amp;author=Dash%2CPK">
                    Google Scholar</a> 
                </p></li><li data-counter="52."><p id="ref-CR52">Xu, L. et al. CRISPR-edited stem cells in a patient with HIV and acute lymphocytic leukemia. <i>N. Engl. J. Med.</i> <b>381</b>, 1240–1247 (2019).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1056/NEJMoa1817426" data-track-item_id="10.1056/NEJMoa1817426" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1056%2FNEJMoa1817426" aria-label="Article reference 52" data-doi="10.1056/NEJMoa1817426">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhvFWnsbbN" aria-label="CAS reference 52">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31509667" aria-label="PubMed reference 52">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 52" href="http://scholar.google.com/scholar_lookup?&amp;title=CRISPR-edited%20stem%20cells%20in%20a%20patient%20with%20HIV%20and%20acute%20lymphocytic%20leukemia&amp;journal=N.%20Engl.%20J.%20Med.&amp;doi=10.1056%2FNEJMoa1817426&amp;volume=381&amp;pages=1240-1247&amp;publication_year=2019&amp;author=Xu%2CL">
                    Google Scholar</a> 
                </p></li><li data-counter="53."><p id="ref-CR53">August, A. et al. A phase 1 trial of lipid-encapsulated mRNA encoding a monoclonal antibody with neutralizing activity against Chikungunya virus. <i>Nat. Med.</i> <b>27</b>, 2224–2233 (2021).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41591-021-01573-6" data-track-item_id="10.1038/s41591-021-01573-6" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41591-021-01573-6" aria-label="Article reference 53" data-doi="10.1038/s41591-021-01573-6">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXislWkt7rI" aria-label="CAS reference 53">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34887572" aria-label="PubMed reference 53">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8674127" aria-label="PubMed Central reference 53">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 53" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20phase%201%20trial%20of%20lipid-encapsulated%20mRNA%20encoding%20a%20monoclonal%20antibody%20with%20neutralizing%20activity%20against%20Chikungunya%20virus&amp;journal=Nat.%20Med.&amp;doi=10.1038%2Fs41591-021-01573-6&amp;volume=27&amp;pages=2224-2233&amp;publication_year=2021&amp;author=August%2CA">
                    Google Scholar</a> 
                </p></li><li data-counter="54."><p id="ref-CR54">Koeberl, D. et al. Interim analyses of a first-in-human phase 1/2 mRNA trial for propionic acidaemia. <i>Nature</i> <b>628</b>, 872–877 (2024).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41586-024-07266-7" data-track-item_id="10.1038/s41586-024-07266-7" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41586-024-07266-7" aria-label="Article reference 54" data-doi="10.1038/s41586-024-07266-7">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="ads reference" data-track-action="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2024Natur.628..872K" aria-label="ADS reference 54">ADS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB2cXnsVKksrk%3D" aria-label="CAS reference 54">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=38570682" aria-label="PubMed reference 54">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC11156579" aria-label="PubMed Central reference 54">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 54" href="http://scholar.google.com/scholar_lookup?&amp;title=Interim%20analyses%20of%20a%20first-in-human%20phase%201%2F2%20mRNA%20trial%20for%20propionic%20acidaemia&amp;journal=Nature&amp;doi=10.1038%2Fs41586-024-07266-7&amp;volume=628&amp;pages=872-877&amp;publication_year=2024&amp;author=Koeberl%2CD">
                    Google Scholar</a> 
                </p></li><li data-counter="55."><p id="ref-CR55">Jordan, A., Bisgrove, D. &amp; Verdin, E. HIV reproducibly establishes a latent infection after acute infection of T cells in vitro. <i> EMBO J.</i> <b>22</b>, 1868–1877 (2003).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1093/emboj/cdg188" data-track-item_id="10.1093/emboj/cdg188" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1093%2Femboj%2Fcdg188" aria-label="Article reference 55" data-doi="10.1093/emboj/cdg188">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3sXivFCjur4%3D" aria-label="CAS reference 55">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=12682019" aria-label="PubMed reference 55">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC154479" aria-label="PubMed Central reference 55">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 55" href="http://scholar.google.com/scholar_lookup?&amp;title=HIV%20reproducibly%20establishes%20a%20latent%20infection%20after%20acute%20infection%20of%20T%20cells%20in%20vitro&amp;journal=EMBO%20J.&amp;doi=10.1093%2Femboj%2Fcdg188&amp;volume=22&amp;pages=1868-1877&amp;publication_year=2003&amp;author=Jordan%2CA&amp;author=Bisgrove%2CD&amp;author=Verdin%2CE">
                    Google Scholar</a> 
                </p></li><li data-counter="56."><p id="ref-CR56">Simeonov, D. R. et al. Discovery of stimulation-responsive immune enhancers with CRISPR activation. <i>Nature</i> <b>549</b>, 111–115 (2017).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/nature23875" data-track-item_id="10.1038/nature23875" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fnature23875" aria-label="Article reference 56" data-doi="10.1038/nature23875">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="ads reference" data-track-action="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2017Natur.549..111S" aria-label="ADS reference 56">ADS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhsVWhu7jO" aria-label="CAS reference 56">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28854172" aria-label="PubMed reference 56">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5675716" aria-label="PubMed Central reference 56">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 56" href="http://scholar.google.com/scholar_lookup?&amp;title=Discovery%20of%20stimulation-responsive%20immune%20enhancers%20with%20CRISPR%20activation&amp;journal=Nature&amp;doi=10.1038%2Fnature23875&amp;volume=549&amp;pages=111-115&amp;publication_year=2017&amp;author=Simeonov%2CDR">
                    Google Scholar</a> 
                </p></li><li data-counter="57."><p id="ref-CR57">Braun, P. et al. Multicenter clinical comparative evaluation of alinity m HIV-1 assay performance. <i>J. Clin. Virol.</i> <b>129</b>, 104530 (2020).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.jcv.2020.104530" data-track-item_id="10.1016/j.jcv.2020.104530" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.jcv.2020.104530" aria-label="Article reference 57" data-doi="10.1016/j.jcv.2020.104530">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhsVWgsL3J" aria-label="CAS reference 57">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32688329" aria-label="PubMed reference 57">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 57" href="http://scholar.google.com/scholar_lookup?&amp;title=Multicenter%20clinical%20comparative%20evaluation%20of%20alinity%20m%20HIV-1%20assay%20performance&amp;journal=J.%20Clin.%20Virol.&amp;doi=10.1016%2Fj.jcv.2020.104530&amp;volume=129&amp;publication_year=2020&amp;author=Braun%2CP">
                    Google Scholar</a> 
                </p></li><li data-counter="58."><p id="ref-CR58">Tumpach, C. et al. Adaptation of the intact proviral DNA assay to a nanowell-based digital PCR platform. <i>J. Virus Erad.</i> <b>9</b>, 100335 (2023).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.jve.2023.100335" data-track-item_id="10.1016/j.jve.2023.100335" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.jve.2023.100335" aria-label="Article reference 58" data-doi="10.1016/j.jve.2023.100335">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXhtlGnu7jI" aria-label="CAS reference 58">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=37440871" aria-label="PubMed reference 58">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10334350" aria-label="PubMed Central reference 58">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 58" href="http://scholar.google.com/scholar_lookup?&amp;title=Adaptation%20of%20the%20intact%20proviral%20DNA%20assay%20to%20a%20nanowell-based%20digital%20PCR%20platform&amp;journal=J.%20Virus%20Erad.&amp;doi=10.1016%2Fj.jve.2023.100335&amp;volume=9&amp;publication_year=2023&amp;author=Tumpach%2CC">
                    Google Scholar</a> 
                </p></li></ol><p><a data-track="click" data-track-action="download citation references" data-track-label="link" rel="nofollow" href="https://citation-needed.springer.com/v2/references/10.1038/s41467-025-60001-2?format=refman&amp;flavour=references">Download references<svg width="16" height="16" focusable="false" role="img" aria-hidden="true"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-download-medium"></use></svg></a></p></div></div></div></section></div><section data-title="Acknowledgements"><div id="Ack1-section"><h2 id="Ack1">Acknowledgements</h2><p>We would like to thank the people living with HIV for generously donating blood to this study. We would like to thank the Infectious Diseases Unit at the Alfred Hospital and the Lewin Clinical Research Group at the Peter Doherty Institute for providing the leukapheresis samples. Specific thanks to Ajantha Rhodes, Judy Chang, Ashanti Dantanarayana, Sureka Tennakoon, Socheata Chea and Lauren Wallace for processing, storing and managing the leukapheresis samples. We acknowledge Professor Viviana Simon and Assistant Professor Lubbertus Mulder from the Icahn School of Medicine at Mount Sinai for supplying plasmids required for the production of the pMorpheus-V5 reporter virus. This research was supported by the Australian National Health and Medical Research Council (NHMRC; Programme Grant No. GNT1149990, Practitioner Fellowship No. 1135851, Investigator Grant No. 2026490 (SRL), Investigator Grant No. GNT2016732 (FC) and The Medical Research Future Fund Grant No. MRF2016144), the National Institute of Allergy and Infectious Diseases Delaney AIDS Research Enterprise to find a cure (DARE) (1UM1AI164560-01), the Australian Centre for HIV and Hepatitis Virology Research (ACH4), the Harold &amp; Cora Brennen Benevolent Trust (Equipment grant No. BREN20122), The Foundation for AIDS Research (amfAR; TARGET Grant No. 110406-73-RGRL_Lewin), The Gandel Foundation and The mRNA Victoria Research Acceleration Fund (mVRAF, GA-F3673757-3598). S.T. is supported by the Doherty Institute for Infection and Immunity Locarnini Fellowship in Virology and University of Melbourne Department of Infectious Diseases Support Package. The authors acknowledge the Melbourne Cytometry Platform (Peter Doherty Institute node) for the provision of flow cytometry services. Schematic figures were generated using Biorender.com.</p></div></section><section aria-labelledby="author-information" data-title="Author information"><div id="author-information-section"><h2 id="author-information">Author information</h2><div id="author-information-content"><p><span id="author-notes">Author notes</span></p><ol><li id="na1"><p>These authors contributed equally: Paula M. Cevaal, Stanislav Kan, Bridget M. Fisher, Michael A. Moso.</p></li><li id="na2"><p>These authors jointly supervised this work: Sharon R. Lewin, Michael Roche.</p></li></ol><h3 id="affiliations">Authors and Affiliations</h3><ol><li id="Aff1"><p>Department of Infectious Diseases, The University of Melbourne at The Peter Doherty Institute for Infection and Immunity, Melbourne, VIC, Australia</p><p>Paula M. Cevaal, Stanislav Kan, Bridget M. Fisher, Michael A. Moso, Abigail Tan, Abdalla Ali, Kiho Tanaka, Rory A. Shepherd, Youry Kim, Jesslyn Ong, Wei Zhao, Sushama Telwatte, Sharon R. Lewin &amp; Michael Roche</p></li><li id="Aff2"><p>Victorian Infectious Diseases Service, The Royal Melbourne Hospital at the Peter Doherty Institute for Infection and Immunity, Melbourne, VIC, Australia</p><p>Michael A. Moso &amp; Sharon R. Lewin</p></li><li id="Aff3"><p>Monash Institute of Pharmaceutical Sciences, Monash University, Parkville, VIC, Australia</p><p>Haiyin Liu, Thomas Payne, Colin W. Pouton &amp; Angus P. R. Johnston</p></li><li id="Aff4"><p>Department of Chemical Engineering, The University of Melbourne, Parkville, VIC, Australia</p><p>Denzil L. Furtado &amp; Frank Caruso</p></li><li id="Aff5"><p>Department of Microbiology and Immunology, The University of Melbourne at The Peter Doherty Institute for Infection and Immunity, Melbourne, VIC, Australia</p><p>Marvin Holz &amp; Damian F. J. Purcell</p></li><li id="Aff6"><p>Cancer Immunology Program, Peter MacCallum Cancer Centre, Melbourne, VIC, Australia</p><p>Joshua M. L. Casan &amp; Mohamed Fareh</p></li><li id="Aff7"><p>Sir Peter MacCallum Department of Oncology, The University of Melbourne, Parkville, VIC, Australia</p><p>Joshua M. L. Casan &amp; Mohamed Fareh</p></li><li id="Aff8"><p>Department of Infectious Diseases, Alfred Hospital and Monash University, Melbourne, VIC, Australia</p><p>James H. McMahon &amp; Sharon R. Lewin</p></li><li id="Aff9"><p>Department of Medicine, University of California, San Francisco, San Francisco, CA, USA</p><p>Steven G. Deeks &amp; Rebecca Hoh</p></li><li id="Aff10"><p>Translational Virology, Department of Medical Microbiology, University Medical Center, Utrecht, the Netherlands</p><p>Jori Symons</p></li></ol><h3 id="contributions">Contributions</h3><p>P.M.C., F.C., J.S., S.R.L. and M.R. conceived the project. P.M.C., B.M.F., M.A.M., A.T., K.T., R.A.S., H.L., A.P.R.J., S.R.L. and M.R. designed the experiments. P.M.C., B.M.F., M.A.M., A.T., K.T., R.A.S., Y.K., J.O. and H.L. executed experiments. S.K., A.A., D.L.F., M.H., D.F.J.P., J.M.L.C., M.F., T.P., W.Z., S.T., C.P. and F.C. provided intellectual input in the experimental design and the design and synthesis of mRNA and lipid nanoparticles. S.K. conceived the LNP X formulation. P.M.C., B.M.F., M.A.M., A.T., K.T., R.A.S. and H.L. performed the experimental analysis. P.M.C., B.M.F., M.A.M., K.T., R.A.S., H.L., A.P.R.J., S.R.L. and M.R. interpreted the data. J.H.M., S.G.D., R.H. and S.R.L. provided access to leukapheresis samples of people living with HIV on ART. P.M.C., B.M.F., M.A.M., S.R.L. and M.R. wrote and edited the manuscript. P.M.C. prepared the figures. All authors read and approved the final manuscript.</p><h3 id="corresponding-author">Corresponding author</h3><p id="corresponding-author-list">Correspondence to
                <a id="corresp-c1" href="mailto:sharon.lewin@unimelb.edu.au">Sharon R. Lewin</a>.</p></div></div></section><section data-title="Ethics declarations"><div id="ethics-section"><h2 id="ethics">Ethics declarations</h2><div id="ethics-content">
              
                <h3 id="FPar2">Competing interests</h3>
                <p>S.R.L. has received honoraria unrelated to the content of this manuscript for participation in advisory boards for Gilead, Viiv Healthcare, Merck, Abbvie, Esfam, Immunocore and First Health. She has received funding from Gilead and Merck for investigator-initiated research projects unrelated to the content of this manuscript. DLF and FC are scientific founders of and hold equity in Messenger Bio. Messenger Bio was paid to synthesise mRNA for 60% of the work presented in the manuscript. P.M.C. was supported by a donation from the J and M Wright Foundation. The donation supported salary to complete some of this work. B.M.F., M.A.M., R.A.S. were supported by an Australian Government Research Training Programme Scholarship. M.A.M. was further supported by an NHMRC Postgraduate Scholarship, Rowden White Scholarship and Elizabeth Mary Sweet Scholarship. K.T. was supported by a Melbourne Research Scholarship. P.M.C., S.K., B.M.F., M.A.M., M.F., M.R., J.S. and S.R.L. are named investigators on a patent related to this work (PCT/AU2024/050506). No other authors declare any competing interests.</p>
              
            </div></div></section><section data-title="Peer review"><div id="peer-review-section"><h2 id="peer-review">Peer review</h2><div id="peer-review-content">
              
              
                <h3 id="FPar1">Peer review information</h3>
                <p><i>Nature Communications</i> thanks the anonymous reviewers for their contribution to the peer review of this work. A peer review file is available.</p>
              
            </div></div></section><section data-title="Additional information"><div id="additional-information-section"><h2 id="additional-information">Additional information</h2><p><b>Publisher’s note</b> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div></section><section data-title="Supplementary information"><div id="Sec21-section"><h2 id="Sec21">Supplementary information</h2></div></section><section data-title="Source data"></section><section data-title="Rights and permissions"><div id="rightslink-section"><h2 id="rightslink">Rights and permissions</h2><div id="rightslink-content">
                <p><b>Open Access</b>  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>.</p>
              <p><a data-track="click" data-track-action="view rights and permissions" data-track-label="link" href="https://s100.copyright.com/AppDispatchServlet?title=Efficient%20mRNA%20delivery%20to%20resting%20T%20cells%20to%20reverse%20HIV%20latency&amp;author=Paula%20M.%20Cevaal%20et%20al&amp;contentID=10.1038%2Fs41467-025-60001-2&amp;copyright=The%20Author%28s%29&amp;publication=2041-1723&amp;publicationDate=2025-05-29&amp;publisherName=SpringerNature&amp;orderBeanReset=true&amp;oa=CC%20BY">Reprints and permissions</a></p></div></div></section><section aria-labelledby="article-info" data-title="About this article"><div id="article-info-section"><h2 id="article-info">About this article</h2><div id="article-info-content"><div><p><a data-crossmark="10.1038/s41467-025-60001-2" target="_blank" rel="noopener" href="https://crossmark.crossref.org/dialog/?doi=10.1038/s41467-025-60001-2" data-track="click" data-track-action="Click Crossmark" data-track-label="link" data-test="crossmark"><img loading="lazy" width="57" height="81" alt="Check for updates. Verify currency and authenticity via CrossMark" src="data:image/svg+xml;base64,<svg height="81" width="57" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="m17.35 35.45 21.3-14.2v-17.03h-21.3" fill="#989898"/><path d="m38.65 35.45-21.3-14.2v-17.03h21.3" fill="#747474"/><path d="m28 .5c-12.98 0-23.5 10.52-23.5 23.5s10.52 23.5 23.5 23.5 23.5-10.52 23.5-23.5c0-6.23-2.48-12.21-6.88-16.62-4.41-4.4-10.39-6.88-16.62-6.88zm0 41.25c-9.8 0-17.75-7.95-17.75-17.75s7.95-17.75 17.75-17.75 17.75 7.95 17.75 17.75c0 4.71-1.87 9.22-5.2 12.55s-7.84 5.2-12.55 5.2z" fill="#535353"/><path d="m41 36c-5.81 6.23-15.23 7.45-22.43 2.9-7.21-4.55-10.16-13.57-7.03-21.5l-4.92-3.11c-4.95 10.7-1.19 23.42 8.78 29.71 9.97 6.3 23.07 4.22 30.6-4.86z" fill="#9c9c9c"/><path d="m.2 58.45c0-.75.11-1.42.33-2.01s.52-1.09.91-1.5c.38-.41.83-.73 1.34-.94.51-.22 1.06-.32 1.65-.32.56 0 1.06.11 1.51.35.44.23.81.5 1.1.81l-.91 1.01c-.24-.24-.49-.42-.75-.56-.27-.13-.58-.2-.93-.2-.39 0-.73.08-1.05.23-.31.16-.58.37-.81.66-.23.28-.41.63-.53 1.04-.13.41-.19.88-.19 1.39 0 1.04.23 1.86.68 2.46.45.59 1.06.88 1.84.88.41 0 .77-.07 1.07-.23s.59-.39.85-.68l.91 1c-.38.43-.8.76-1.28.99-.47.22-1 .34-1.58.34-.59 0-1.13-.1-1.64-.31-.5-.2-.94-.51-1.31-.91-.38-.4-.67-.9-.88-1.48-.22-.59-.33-1.26-.33-2.02zm8.4-5.33h1.61v2.54l-.05 1.33c.29-.27.61-.51.96-.72s.76-.31 1.24-.31c.73 0 1.27.23 1.61.71.33.47.5 1.14.5 2.02v4.31h-1.61v-4.1c0-.57-.08-.97-.25-1.21-.17-.23-.45-.35-.83-.35-.3 0-.56.08-.79.22-.23.15-.49.36-.78.64v4.8h-1.61zm7.37 6.45c0-.56.09-1.06.26-1.51.18-.45.42-.83.71-1.14.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.36c.07.62.29 1.1.65 1.44.36.33.82.5 1.38.5.29 0 .57-.04.83-.13s.51-.21.76-.37l.55 1.01c-.33.21-.69.39-1.09.53-.41.14-.83.21-1.26.21-.48 0-.92-.08-1.34-.25-.41-.16-.76-.4-1.07-.7-.31-.31-.55-.69-.72-1.13-.18-.44-.26-.95-.26-1.52zm4.6-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.07.45-.31.29-.5.73-.58 1.3zm2.5.62c0-.57.09-1.08.28-1.53.18-.44.43-.82.75-1.13s.69-.54 1.1-.71c.42-.16.85-.24 1.31-.24.45 0 .84.08 1.17.23s.61.34.85.57l-.77 1.02c-.19-.16-.38-.28-.56-.37-.19-.09-.39-.14-.61-.14-.56 0-1.01.21-1.35.63-.35.41-.52.97-.52 1.67 0 .69.17 1.24.51 1.66.34.41.78.62 1.32.62.28 0 .54-.06.78-.17.24-.12.45-.26.64-.42l.67 1.03c-.33.29-.69.51-1.08.65-.39.15-.78.23-1.18.23-.46 0-.9-.08-1.31-.24-.4-.16-.75-.39-1.05-.7s-.53-.69-.7-1.13c-.17-.45-.25-.96-.25-1.53zm6.91-6.45h1.58v6.17h.05l2.54-3.16h1.77l-2.35 2.8 2.59 4.07h-1.75l-1.77-2.98-1.08 1.23v1.75h-1.58zm13.69 1.27c-.25-.11-.5-.17-.75-.17-.58 0-.87.39-.87 1.16v.75h1.34v1.27h-1.34v5.6h-1.61v-5.6h-.92v-1.2l.92-.07v-.72c0-.35.04-.68.13-.98.08-.31.21-.57.4-.79s.42-.39.71-.51c.28-.12.63-.18 1.04-.18.24 0 .48.02.69.07.22.05.41.1.57.17zm.48 5.18c0-.57.09-1.08.27-1.53.17-.44.41-.82.72-1.13.3-.31.65-.54 1.04-.71.39-.16.8-.24 1.23-.24s.84.08 1.24.24c.4.17.74.4 1.04.71s.54.69.72 1.13c.19.45.28.96.28 1.53s-.09 1.08-.28 1.53c-.18.44-.42.82-.72 1.13s-.64.54-1.04.7-.81.24-1.24.24-.84-.08-1.23-.24-.74-.39-1.04-.7c-.31-.31-.55-.69-.72-1.13-.18-.45-.27-.96-.27-1.53zm1.65 0c0 .69.14 1.24.43 1.66.28.41.68.62 1.18.62.51 0 .9-.21 1.19-.62.29-.42.44-.97.44-1.66 0-.7-.15-1.26-.44-1.67-.29-.42-.68-.63-1.19-.63-.5 0-.9.21-1.18.63-.29.41-.43.97-.43 1.67zm6.48-3.44h1.33l.12 1.21h.05c.24-.44.54-.79.88-1.02.35-.24.7-.36 1.07-.36.32 0 .59.05.78.14l-.28 1.4-.33-.09c-.11-.01-.23-.02-.38-.02-.27 0-.56.1-.86.31s-.55.58-.77 1.1v4.2h-1.61zm-47.87 15h1.61v4.1c0 .57.08.97.25 1.2.17.24.44.35.81.35.3 0 .57-.07.8-.22.22-.15.47-.39.73-.73v-4.7h1.61v6.87h-1.32l-.12-1.01h-.04c-.3.36-.63.64-.98.86-.35.21-.76.32-1.24.32-.73 0-1.27-.24-1.61-.71-.33-.47-.5-1.14-.5-2.02zm9.46 7.43v2.16h-1.61v-9.59h1.33l.12.72h.05c.29-.24.61-.45.97-.63.35-.17.72-.26 1.1-.26.43 0 .81.08 1.15.24.33.17.61.4.84.71.24.31.41.68.53 1.11.13.42.19.91.19 1.44 0 .59-.09 1.11-.25 1.57-.16.47-.38.85-.65 1.16-.27.32-.58.56-.94.73-.35.16-.72.25-1.1.25-.3 0-.6-.07-.9-.2s-.59-.31-.87-.56zm0-2.3c.26.22.5.37.73.45.24.09.46.13.66.13.46 0 .84-.2 1.15-.6.31-.39.46-.98.46-1.77 0-.69-.12-1.22-.35-1.61-.23-.38-.61-.57-1.13-.57-.49 0-.99.26-1.52.77zm5.87-1.69c0-.56.08-1.06.25-1.51.16-.45.37-.83.65-1.14.27-.3.58-.54.93-.71s.71-.25 1.08-.25c.39 0 .73.07 1 .2.27.14.54.32.81.55l-.06-1.1v-2.49h1.61v9.88h-1.33l-.11-.74h-.06c-.25.25-.54.46-.88.64-.33.18-.69.27-1.06.27-.87 0-1.56-.32-2.07-.95s-.76-1.51-.76-2.65zm1.67-.01c0 .74.13 1.31.4 1.7.26.38.65.58 1.15.58.51 0 .99-.26 1.44-.77v-3.21c-.24-.21-.48-.36-.7-.45-.23-.08-.46-.12-.7-.12-.45 0-.82.19-1.13.59-.31.39-.46.95-.46 1.68zm6.35 1.59c0-.73.32-1.3.97-1.71.64-.4 1.67-.68 3.08-.84 0-.17-.02-.34-.07-.51-.05-.16-.12-.3-.22-.43s-.22-.22-.38-.3c-.15-.06-.34-.1-.58-.1-.34 0-.68.07-1 .2s-.63.29-.93.47l-.59-1.08c.39-.24.81-.45 1.28-.63.47-.17.99-.26 1.54-.26.86 0 1.51.25 1.93.76s.63 1.25.63 2.21v4.07h-1.32l-.12-.76h-.05c-.3.27-.63.48-.98.66s-.73.27-1.14.27c-.61 0-1.1-.19-1.48-.56-.38-.36-.57-.85-.57-1.46zm1.57-.12c0 .3.09.53.27.67.19.14.42.21.71.21.28 0 .54-.07.77-.2s.48-.31.73-.56v-1.54c-.47.06-.86.13-1.18.23-.31.09-.57.19-.76.31s-.33.25-.41.4c-.09.15-.13.31-.13.48zm6.29-3.63h-.98v-1.2l1.06-.07.2-1.88h1.34v1.88h1.75v1.27h-1.75v3.28c0 .8.32 1.2.97 1.2.12 0 .24-.01.37-.04.12-.03.24-.07.34-.11l.28 1.19c-.19.06-.4.12-.64.17-.23.05-.49.08-.76.08-.4 0-.74-.06-1.02-.18-.27-.13-.49-.3-.67-.52-.17-.21-.3-.48-.37-.78-.08-.3-.12-.64-.12-1.01zm4.36 2.17c0-.56.09-1.06.27-1.51s.41-.83.71-1.14c.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.37c.08.62.29 1.1.65 1.44.36.33.82.5 1.38.5.3 0 .58-.04.84-.13.25-.09.51-.21.76-.37l.54 1.01c-.32.21-.69.39-1.09.53s-.82.21-1.26.21c-.47 0-.92-.08-1.33-.25-.41-.16-.77-.4-1.08-.7-.3-.31-.54-.69-.72-1.13-.17-.44-.26-.95-.26-1.52zm4.61-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.08.45-.31.29-.5.73-.57 1.3zm3.01 2.23c.31.24.61.43.92.57.3.13.63.2.98.2.38 0 .65-.08.83-.23s.27-.35.27-.6c0-.14-.05-.26-.13-.37-.08-.1-.2-.2-.34-.28-.14-.09-.29-.16-.47-.23l-.53-.22c-.23-.09-.46-.18-.69-.3-.23-.11-.44-.24-.62-.4s-.33-.35-.45-.55c-.12-.21-.18-.46-.18-.75 0-.61.23-1.1.68-1.49.44-.38 1.06-.57 1.83-.57.48 0 .91.08 1.29.25s.71.36.99.57l-.74.98c-.24-.17-.49-.32-.73-.42-.25-.11-.51-.16-.78-.16-.35 0-.6.07-.76.21-.17.15-.25.33-.25.54 0 .14.04.26.12.36s.18.18.31.26c.14.07.29.14.46.21l.54.19c.23.09.47.18.7.29s.44.24.64.4c.19.16.34.35.46.58.11.23.17.5.17.82 0 .3-.06.58-.17.83-.12.26-.29.48-.51.68-.23.19-.51.34-.84.45-.34.11-.72.17-1.15.17-.48 0-.95-.09-1.41-.27-.46-.19-.86-.41-1.2-.68z" fill="#535353"/></g></svg>"/></a></p><div><h3 id="citeas">Cite this article</h3><p>Cevaal, P.M., Kan, S., Fisher, B.M. <i>et al.</i> Efficient mRNA delivery to resting T cells to reverse HIV latency.
                    <i>Nat Commun</i> <b>16</b>, 4979 (2025). https://doi.org/10.1038/s41467-025-60001-2</p><p><a data-test="citation-link" data-track="click" data-track-action="download article citation" data-track-label="link" data-track-external="" rel="nofollow" href="https://citation-needed.springer.com/v2/references/10.1038/s41467-025-60001-2?format=refman&amp;flavour=citation">Download citation<svg width="16" height="16" focusable="false" role="img" aria-hidden="true"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-download-medium"></use></svg></a></p><ul data-test="publication-history"><li><p>Received<span>: </span><span><time datetime="2025-01-11">11 January 2025</time></span></p></li><li><p>Accepted<span>: </span><span><time datetime="2025-05-12">12 May 2025</time></span></p></li><li><p>Published<span>: </span><span><time datetime="2025-05-29">29 May 2025</time></span></p></li><li><p><abbr title="Digital Object Identifier">DOI</abbr><span>: </span><span>https://doi.org/10.1038/s41467-025-60001-2</span></p></li></ul></div></div></div></div></section>
            </div></div>
  </body>
</html>
