<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <title>James Routley | Feed</title>
    <link
      rel="stylesheet"
      type="text/css"
      href="../styles.css"
      media="screen"
    />
  </head>
  <body>
    <a href="/index.html">Back</a>
    <a href="https://www.nature.com/articles/s41586-023-05754-w">Original</a>
    <h1>Entwined African and Asian genetic roots of medieval people of the Swahili coast</h1>
    
    <div id="readability-page-1" class="page"><div>
                <section data-title="Main"><div id="Sec1-section"><h2 id="Sec1">Main</h2><div id="Sec1-content"><p>The medieval and early modern Swahili culture of eastern Africa from the seventh century <span>ad</span> was defined by a set of shared features: a common language of African origin (Kiswahili), a shared predominant religion (Islam) and a geographic distribution in coastal towns and villages. People of the Swahili culture lived across a vast coastal region that included northern Mozambique, southern Somalia, Madagascar and the archipelagos of Comoros, Kilwa, Mafia, Zanzibar and Lamu<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 1" title="Wynne-Jones, S. &amp; LaViolette, A. J. The Swahili World (Routledge, 2017)." href="https://medium.com/articles/s41586-023-05754-w#ref-CR1" id="ref-link-section-d194324060e1455">1</a></sup> (yellow outlines in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://medium.com/articles/s41586-023-05754-w#Fig1">1a</a>). Millions of present-day coastal people identify as Swahili, although for many this is a secondary identity, with primary identities often being more based on town of origin, family history or traditional social status<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 5" title="Spear, T. Early Swahili history reconsidered. Int. J. Afr. Hist. Stud. 33, 257–290 (2000)." href="https://medium.com/articles/s41586-023-05754-w#ref-CR5" id="ref-link-section-d194324060e1462">5</a></sup>. How people who identify as Swahili in the present day relate to people of the medieval and early modern Swahili culture has been difficult to elucidate in the absence of ancient DNA.</p><div data-test="figure" data-container-section="figure" id="figure-1" data-title="Dataset overview."><figure><figcaption><b id="Fig1" data-test="figure-caption-text">Fig. 1: Dataset overview.</b></figcaption><div><div><a data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="https://medium.com/articles/s41586-023-05754-w/figures/1" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41586-023-05754-w/MediaObjects/41586_2023_5754_Fig1_HTML.png?as=webp"/><img aria-describedby="Fig1" src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41586-023-05754-w/MediaObjects/41586_2023_5754_Fig1_HTML.png" alt="figure 1" loading="lazy" width="685" height="503"/></picture></a></div><p><b>a</b>, Coastal areas associated with the medieval Swahili culture are shown in yellow. Sites represented in the ancient DNA samples are marked with black shapes. Numbers in parentheses are formatted <i>X</i>|<i>Y</i>, where <i>X</i> is the number of individuals for whom there are data, and <i>Y</i> is the number of individuals for whom we report high-resolution analyses. The chronology is given as the union of 95% confidence intervals for direct radiocarbon dates on the skeletons rounding to the nearest 50 years and is shown as calibrated (cal.) yr <span>ad</span>, or as <span>ad</span> for sites with only archaeological context (<a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://medium.com/articles/s41586-023-05754-w#MOESM1">Supplementary Information</a>; Extended Data Table <a data-track="click" data-track-label="link" data-track-action="table anchor" href="https://medium.com/articles/s41586-023-05754-w#Tab3">1</a>). The base map was made with the Natural Earth R package using a CC BY license. Bodies of water were added from the RCMRD Geoportal with a CC BY license in R. All points were added and modifications were performed in R and Adobe Illustrator. The map is published for the first time in this Article. <b>b</b>, In a principal component analysis, eigenvector 1 correlates to variation maximized in sub-Saharan Africa, and eigenvector 2 correlates to Eurasian variation. <b>c</b>, Ancestry component assignment using ADMIXTURE with <i>K</i> = 9 clusters (selected on the basis of low cross-validation errors, high log-likelihood scores, and a low number of reference populations to not overfit; groups that maximize each components are shown on the right). Individuals with sufficient data for high-resolution analysis are plotted in approximate chronological order from left to right. Ancient individuals are labelled and plotted at four times the width of present-day individuals.</p></div></figure></div><p>The medieval largely autonomous towns and polities known as the Swahili states arose out of fishing and agropastoral settlements on the eastern African coast during the late first millennium <span>ad</span><sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 6" title="Kusimba, C. M. in State Formations: Global Histories and Cultures of Statehood (eds Anderson, G., Brooke, J. L. &amp; Strauss, J. C.) 90–107 (Cambridge Univ. Press, 2018)." href="https://medium.com/articles/s41586-023-05754-w#ref-CR6" id="ref-link-section-d194324060e1527">6</a></sup>. First millennium <span>ad</span> sites on the littoral, beginning in the seventh century, were part of a shared material culture and practice network across the eastern African region<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Fleisher, J. &amp; Wynne-Jones, S. Ceramics and the early Swahili: deconstructing the early Tana tradition. Afr. Archaeol. Rev. 28, 245–278 (2011)." href="https://medium.com/articles/s41586-023-05754-w#ref-CR7" id="ref-link-section-d194324060e1534">7</a></sup>. These sites were engaged in the Indian Ocean trading system, facilitated by southwest monsoons from May to October that enabled merchant vessels to travel from India or the Arabian Peninsula to the eastern African coast, and northeast monsoons from November to March enabling their return in the same year<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 8" title="Pollard, E. &amp; Kinyera, O. C. The Swahili Coast and the Indian Ocean trade patterns in the 7th–10th centuries ce. J. S. Afr. Stud. 43, 927–947 (2017)." href="https://medium.com/articles/s41586-023-05754-w#ref-CR8" id="ref-link-section-d194324060e1538">8</a></sup>.</p><p>Muslims were present from the eighth century <span>ad</span>, probably as a minority<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 9" title="Horton, M. C. Shanga: An Early Muslim Trading Community on the Coast of Eastern Africa, Vol. 14. (British Institute in Eastern Africa, 1996)." href="https://medium.com/articles/s41586-023-05754-w#ref-CR9" id="ref-link-section-d194324060e1548">9</a></sup>. A major archaeological transition is evident during the eleventh century, with the establishment of new settlements and the elaboration of older ones with coral-built mosques and tombs, a set of changes generally understood as coinciding with the widespread adoption of Islam<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 10" title="Fleisher, J. et al. When did the Swahili become maritime? Am. Anthropol. 117, 100–115 (2015)." href="https://medium.com/articles/s41586-023-05754-w#ref-CR10" id="ref-link-section-d194324060e1552">10</a></sup>. At this time, clearer distinctions also emerged between coastal ceramics and material traditions and those of inland assemblages<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 2" title="Kusimba, C. M. The Rise and Fall of Swahili States (Altamira Press, 1999)." href="https://medium.com/articles/s41586-023-05754-w#ref-CR2" id="ref-link-section-d194324060e1556">2</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 11" title="Wynne-Jones, S. A Material Culture: Consumption and Materiality on the Coast of Precolonial East Africa (Oxford Univ. Press, 2016)." href="https://medium.com/articles/s41586-023-05754-w#ref-CR11" id="ref-link-section-d194324060e1559">11</a></sup>, even as many aspects of material culture remained deeply linked with inland African groups.</p><p>The political and administrative independence of Swahili towns diminished in the sixteenth century as the Portuguese naval and economic dominance in the Indian Ocean spread<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 12" title="Vernet, T. Le territoire hors les murs des cités-états Swahili de l’Archipel de Lamu, 1600–1800. J. Afr. 74, 381–411 (2004)." href="https://medium.com/articles/s41586-023-05754-w#ref-CR12" id="ref-link-section-d194324060e1566">12</a></sup>. In the early eighteenth century, the Portuguese influence waned and the Sultanates of Oman and later Zanzibar became dominant<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 4" title="Sheriff, A. Slaves, Spices and Ivory in Zanzibar. Integration of an East African Commercial Empire into the World Economy, 1770–1873 (James Currey, 1987)." href="https://medium.com/articles/s41586-023-05754-w#ref-CR4" id="ref-link-section-d194324060e1570">4</a></sup>. In the nineteenth century, the growth of overseas trade, including in enslaved people, led to large-scale population movements from central regions of Africa and settlers from the Yemeni region of Hadramawt<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 4" title="Sheriff, A. Slaves, Spices and Ivory in Zanzibar. Integration of an East African Commercial Empire into the World Economy, 1770–1873 (James Currey, 1987)." href="https://medium.com/articles/s41586-023-05754-w#ref-CR4" id="ref-link-section-d194324060e1574">4</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 13" title="Glassman, J. Feasts and Riot: Revelry, Rebellion, and Popular Consciousness on the Swahili Coast 1856–1888 (James Currey, 1995)." href="https://medium.com/articles/s41586-023-05754-w#ref-CR13" id="ref-link-section-d194324060e1577">13</a></sup>. In the mid-nineteenth century, Britain and other European powers became dominant, leading to the settlement of Europeans, the arrival of labourers from South Asia, and further interactions with non-coastal eastern Africans.</p><p>In light of this multi-layered history, the extent to which people who identify as Swahili in the present day are genetically linked with people who built the medieval trading towns is unclear, as is the relationship of the medieval groups to earlier groups. Although the intercontinental connections maintained as part of the Indian Ocean trading network meant that foreigners were consistently present along the coast, the extent to which they had families with African residents has long been debated<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 14" title="Caplan, P. ‘But the coast, of course, is quite different’: academic and local Ideas about the East African littoral. J. East. Afr. Stud. 1, 305–320 (2007)." href="https://medium.com/articles/s41586-023-05754-w#ref-CR14" id="ref-link-section-d194324060e1585">14</a></sup>.</p><p>Swahili traditions suggest that foreigners had an important impact; a set of common oral histories relates the founding of coastal towns to the arrival of a group known as the Shirazi, referring to a region in Persia<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 3" title="Spear, T. T. The Shirazi in Swahili traditions, culture and history. Hist. Afr. 11, 291–305 (1984)." href="https://medium.com/articles/s41586-023-05754-w#ref-CR3" id="ref-link-section-d194324060e1592">3</a></sup>. This Shirazi tradition was put into writing in the Kilwa Chronicle in the sixteenth century<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 15" title="Freeman-Grenville, G. S. P. The East African Coast. Select Documents from the First to the Earlier Nineteenth Centuries (Clarendon Press, 1962)." href="https://medium.com/articles/s41586-023-05754-w#ref-CR15" id="ref-link-section-d194324060e1596">15</a></sup>. These accounts of Shirazi roots were central to the narrative constructed by mid-twentieth century colonialist archaeologists, who interpreted second millennium coastal eastern African sites as having been built by Persian and Arab settlers, and focused on connections with the broader Indian Ocean world<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 16" title="Chittick, H. N. in East Africa and the Orient: Cultural Syntheses in Pre-Colonial Times eds Chittick, H. N. &amp; Rotberg, R. I.) 16–43 (Africana, 1975)." href="https://medium.com/articles/s41586-023-05754-w#ref-CR16" id="ref-link-section-d194324060e1600">16</a></sup>.</p><p>However, narratives of foreign origin have the potential to be misleading, as Swahili social ‘elites’ used claims of foreign origin and rejection of cultural connections within Africa to establish their social status and to signal their religious and cultural affinities<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 17" title="Pouwels, R. L. Horn and Crescent. Cultural change and traditional Islam on the East African coast, 800–1900. African Studies Series, Vol. 53. (Cambridge Univ. Press, 1987)." href="https://medium.com/articles/s41586-023-05754-w#ref-CR17" id="ref-link-section-d194324060e1607">17</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 18" title="Saad, E. Kilwa dynastic historiography: a critical study. Hist. Afr. 6, 177–209 (1979)." href="https://medium.com/articles/s41586-023-05754-w#ref-CR18" id="ref-link-section-d194324060e1610">18</a></sup>.</p><p>Recent research has shown that archaeology during colonial times tended to ignore the evidence of deep African roots, emphasizing foreign objects at medieval Swahili sites rather than providing a balanced picture of the archaeological record<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 2" title="Kusimba, C. M. The Rise and Fall of Swahili States (Altamira Press, 1999)." href="https://medium.com/articles/s41586-023-05754-w#ref-CR2" id="ref-link-section-d194324060e1617">2</a></sup>. Imports at most coastal sites typically comprise less than 5% of total assemblages<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 2" title="Kusimba, C. M. The Rise and Fall of Swahili States (Altamira Press, 1999)." href="https://medium.com/articles/s41586-023-05754-w#ref-CR2" id="ref-link-section-d194324060e1621">2</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 9" title="Horton, M. C. Shanga: An Early Muslim Trading Community on the Coast of Eastern Africa, Vol. 14. (British Institute in Eastern Africa, 1996)." href="https://medium.com/articles/s41586-023-05754-w#ref-CR9" id="ref-link-section-d194324060e1624">9</a></sup>. Other aspects of the material culture also show continuity with earlier settlements, including the persistence of crops, domesticated animals, craft styles and ceramics<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 9" title="Horton, M. C. Shanga: An Early Muslim Trading Community on the Coast of Eastern Africa, Vol. 14. (British Institute in Eastern Africa, 1996)." href="https://medium.com/articles/s41586-023-05754-w#ref-CR9" id="ref-link-section-d194324060e1628">9</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 19" title="Crowther, A. et al. Subsistence mosaics, forager-farmer interactions, and the transition to food production in eastern Africa. Quat. Int. 489, 101–120 (2018)." href="https://medium.com/articles/s41586-023-05754-w#ref-CR19" id="ref-link-section-d194324060e1631">19</a></sup>. Linguistic evidence provides additional evidence of African roots: Kiswahili is an African Bantu language with Asian loanwords<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 20" title="Nurse, D. &amp; Spear, T. The Swahili: Reconstructing the History and Language of an African Society, 800–1500 (Univ. of Pennsylvania Press, 1985)." href="https://medium.com/articles/s41586-023-05754-w#ref-CR20" id="ref-link-section-d194324060e1635">20</a></sup>. However, without ancient DNA evidence, it is not possible to directly address questions of how genetic ancestry changed over time.</p><p>We generated ancient DNA data from the skeletal remains of individuals found at six coastal or island towns: Mtwapa, Manda, Faza, Kilwa, Songo Mnara and Lindi. These individuals date to <span>ad</span> 1250–1800 but provide insight into genetic events from the tenth century <span>ad</span> onwards. We also generated ancient DNA from the remains of individuals found at the site of Makwasinyi (postdating around <span>ad</span> 1650), about 100 km inland from the southern Kenyan coast, which was inhabited by people who were in cultural contact with coastal groups. We compare the newly reported data from the ancient individuals with that of present-day coastal Swahili speakers and with previously published data from diverse ancient and present-day eastern African and Eurasian groups.</p></div></div></section><section data-title="Dataset overview"><div id="Sec2-section"><h2 id="Sec2">Dataset overview</h2><div id="Sec2-content"><p>We generated 179 ancient DNA libraries from 156 distinct skeletal samples (Supplementary Data File <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://medium.com/articles/s41586-023-05754-w#MOESM4">1</a> and Methods). We applied in-solution enrichment for a targeted set of about 1.2 million single nucleotide polymorphisms (SNPs) to obtain genome-wide data passing standard measures of ancient DNA authenticity from 80 distinct individuals. The individuals were buried at seven second millennium <span>ad</span> sites in eastern Africa (black shapes in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://medium.com/articles/s41586-023-05754-w#Fig1">1a</a>; individuals are listed in Extended Data Table <a data-track="click" data-track-label="link" data-track-action="table anchor" href="https://medium.com/articles/s41586-023-05754-w#Tab3">1</a> and Supplementary Data File <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://medium.com/articles/s41586-023-05754-w#MOESM4">2</a>; see <a data-track="click" data-track-label="link" data-track-action="section anchor" href="https://medium.com/articles/s41586-023-05754-w#Sec11">Methods</a>, ‘Inclusion and ethics’ and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://medium.com/articles/s41586-023-05754-w#MOESM1">Supplementary Information</a> for details of archaeological and genetic permissions and sampling). We obtained direct radiocarbon dates (Supplementary Data File <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://medium.com/articles/s41586-023-05754-w#MOESM4">3</a>) for 33 of the skeletons, and estimated date ranges for the other individuals on the basis of archaeological context or genetic evidence of relatedness to individuals for whom we had direct dates (Extended Data Table <a data-track="click" data-track-label="link" data-track-action="table anchor" href="https://medium.com/articles/s41586-023-05754-w#Tab3">1</a> and Supplementary Data File <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://medium.com/articles/s41586-023-05754-w#MOESM4">2</a>). Because of the reliance on seafood, old carbon entering the food chain (marine reservoir effects) could mean that the radiocarbon dates of some individuals are older than their true dates. Moreover, differences in the dependence on marine food across the archaeological sites could mean that the relative chronology of the coastal individuals and sites may not be possible to determine with full confidence. We also generated new genome-wide data on the Affymetrix Human Origins SNP array from 93 present-day individuals who identified as Swahili and indicated that their ancestors lived for many generations in coastal towns<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 21" title="Raaum, R. L. et al. in The Swahili World (eds Wynne-Jones, S. &amp; LaViolette, A. J.) 81–102 (Routledge, 2017)." href="https://medium.com/articles/s41586-023-05754-w#ref-CR21" id="ref-link-section-d194324060e1690">21</a></sup> (Supplementary Data File <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://medium.com/articles/s41586-023-05754-w#MOESM4">4</a>). Finally, we generated new genome-wide data from 19 individuals from Madagascar, and 10 from the United Arab Emirates.</p><p>Three sites were northern coastal towns: Mtwapa (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://medium.com/articles/s41586-023-05754-w#MOESM1">1</a>, 48 individuals spanning <span>ad</span> 1250 to <span>ad</span> 1650), Faza on Pate Island (1 individual), and Manda Island (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://medium.com/articles/s41586-023-05754-w#MOESM1">2</a>, 8 individuals spanning <span>ad</span> 1450 to <span>ad</span> 1650). Three additional sites were southern coastal towns: Songo Mnara (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://medium.com/articles/s41586-023-05754-w#MOESM1">3</a> and Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://medium.com/articles/s41586-023-05754-w#MOESM1">1</a>, 7 individuals spanning <span>ad</span> 1300 to <span>ad</span> 1800), Lindi (1 individual at <span>ad</span> 1500 to <span>ad</span> 1650), and Kilwa Kisiwani (2 individuals spanning <span>ad</span> 1300 to <span>ad</span> 1600). The remains at Mtwapa, Manda and Songo Mnara were mainly from Muslim burials of elites, often located near mosques. We do not have enough context for the Faza, Kilwa and Lindi burials to know if they followed the same pattern. We also analysed 13 individuals from Makwasinyi (<span>ad</span> 1650–1950), approximately 100 km inland from the coast of present-day Kenya. Although these burials post-date the coastal sites, the Makwasinyi community traded with coastal peoples while remaining isolated in most respects. We hypothesized that the ancestry of Makwasinyi people might be a good proxy to represent inland African groups that may have been in contact with people from medieval towns on the northern Swahili coast in previous centuries<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 22" title="Kusimba, C. M., Kusimba, S. B. &amp; Dussubieux, L. Beyond the coastalscapes: preindustrial social and political networks in East Africa. Afr. Archaeol. Rev. 30, 399–426 (2013)." href="https://medium.com/articles/s41586-023-05754-w#ref-CR22" id="ref-link-section-d194324060e1748">22</a></sup>.</p><p>Of the 80 individuals for whom we report data, we exclude 26 in genome-wide analyses, although their data remain valuable. Of these, 18 had too few SNPs for high-resolution whole-genome analyses although they yielded useful data such as reliably determined mitochondrial sequences; 5 were first- or possibly second-degree relatives of other individuals in the dataset with higher-quality data; 2 showed evidence of contamination; and 1 was a population genetic outlier with limited data, raising the possibility of contamination (Supplementary Data File <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://medium.com/articles/s41586-023-05754-w#MOESM4">2</a>).</p><p>Ancient DNA data from four individuals from the eastern African coast have previously been published (Supplementary Data File <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://medium.com/articles/s41586-023-05754-w#MOESM4">5</a>), but none have been published from a Swahili town<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 23" title="Skoglund, P. et al. Reconstructing prehistoric African population structure. Cell 171, 59–71.e21 (2017)." href="https://medium.com/articles/s41586-023-05754-w#ref-CR23" id="ref-link-section-d194324060e1764">23</a></sup>. An individual from around <span>ad</span> 1400 whose remains were recovered from Makangale Cave on Pemba Island had ancestry predominantly related to western African groups<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 23" title="Skoglund, P. et al. Reconstructing prehistoric African population structure. Cell 171, 59–71.e21 (2017)." href="https://medium.com/articles/s41586-023-05754-w#ref-CR23" id="ref-link-section-d194324060e1771">23</a></sup> (an ancestry common today in speakers of Bantu languages and prevalent in eastern Africa—hereafter referred to as ‘Bantu-associated’). Another individual from Makangale Cave on Pemba Island dated to around <span>ad</span> 600, an individual from around <span>ad</span> 600 from Kuumbi Cave on Zanzibar Island, and an individual from around <span>ad</span> 1500 from Panga ya Saidi in Kenya all had predominantly sub-Saharan African forager-associated ancestry<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 23" title="Skoglund, P. et al. Reconstructing prehistoric African population structure. Cell 171, 59–71.e21 (2017)." href="https://medium.com/articles/s41586-023-05754-w#ref-CR23" id="ref-link-section-d194324060e1785">23</a></sup>. There is no indication of Eurasian ancestry deriving from migrations in the last 2,000 years in any of these individuals, which differs from nearly all of the individuals from medieval coastal towns newly reported here.</p><p>In this Article, we use ‘African ancestry’ to refer to DNA deriving from people who can be genetically well-proxied by sub-Saharan Africans for whom there is published ancient DNA data dating to between 2000 <span>bc</span> and <span>ad</span> 1000. We use the terms ‘Eurasian’, ‘Persian’, ‘Arabian’ and ‘Indian’ to refer to ancestry that can be proxied by modern populations from these regions and that are not known to be similar to ancestry in sub-Saharan Africans between 2000 <span>bc</span> and <span>ad</span> 1000. The evidence that a proportion of the ancestors of Africans from between 2000 <span>bc</span> and <span>ad</span> 1000 may have come from Eurasia—for example, approximately 40% of the ancestry of people of the eastern African Pastoral Neolithic culture<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 24" title="Prendergast Mary, E. et al. Ancient DNA reveals a multistep spread of the first herders into sub-Saharan Africa. Science 365, eaaw6275 (2019)." href="https://medium.com/articles/s41586-023-05754-w#ref-CR24" id="ref-link-section-d194324060e1812">24</a></sup>—does not contradict these definitions, as all humans are mixed at multiple time depths of history. As long as we specify both the time and the geography for the source populations, we can be precise in the use of the term ‘African ancestry’<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 25" title="Mathieson, I. &amp; Scally, A. What is ancestry? PLoS Genet. 16, e1008624 (2020)." href="https://medium.com/articles/s41586-023-05754-w#ref-CR25" id="ref-link-section-d194324060e1816">25</a></sup>.</p></div></div></section><section data-title="Genetic affinities"><div id="Sec3-section"><h2 id="Sec3">Genetic affinities</h2><p>To obtain a qualitative picture of the sources of the ancestry in the ancient individuals, we performed principal component analysis (PCA) (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://medium.com/articles/s41586-023-05754-w#Fig1">1b</a>). We used 1,286 present-day Eurasian and African individuals to compute the axes (Supplementary Data File <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://medium.com/articles/s41586-023-05754-w#MOESM4">6</a>). We projected the newly reported ancient individuals onto this PCA and found that they form a cline, with one end overlapping with ancient and present-day African groups and the other falling between present-day Persians and Indians. This suggests mixtures of different proportions of source populations at either end of the cline, with these sources potentially having multiple deeper ancestry components. Some coastal individuals—particularly from Songo Mnara and Lindi, do not fall on this cline—suggesting additional complexity, although our power to understand this variability is limited by the small sample sizes. Similar patterns are evident with unsupervised clustering using ADMIXTURE, which further suggests sub-Saharan African-associated components, southwest Asian-associated components, and East Asian or Indian-associated components (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://medium.com/articles/s41586-023-05754-w#Fig1">1c</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://medium.com/articles/s41586-023-05754-w#Fig4">1</a>).</p></div></section><section data-title="Proportions of African, Persian and Indian DNA"><div id="Sec4-section"><h2 id="Sec4">Proportions of African, Persian and Indian DNA</h2><div id="Sec4-content"><p>Using qpAdm<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 26" title="Patterson, N. et al. Ancient admixture in human history. Genetics 192, 1065–1093 (2012)." href="https://medium.com/articles/s41586-023-05754-w#ref-CR26" id="ref-link-section-d194324060e1848">26</a></sup>, we find that most medieval and early modern individuals can only be fit by a model with at least three ancestry components that can be proxied with ancient African, present-day Iranian and present-day Indian populations (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://medium.com/articles/s41586-023-05754-w#Fig2">2a</a>, Extended Data Table <a data-track="click" data-track-label="link" data-track-action="table anchor" href="https://medium.com/articles/s41586-023-05754-w#Tab4">2</a>, Supplementary Tables <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://medium.com/articles/s41586-023-05754-w#MOESM1">2–10</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://medium.com/articles/s41586-023-05754-w#MOESM1">Supplementary Information</a>). Such a three-source model fits the pool of 48 Mtwapa individuals and the Faza individual (<i>P</i> value for fit = 0.23); the pool of Manda (<i>P</i> = 0.28) individuals, and at least one Songo Mnara individual (I19550) (<i>P</i> = 0.38). One Kilwa individual (I8816) had a relatively high proportion of ancestry related to inland sub-Saharan Africans, and so the Indian proportion, which is the smallest contributor, falls below the threshold of definitive detection (a two-source model fits; <i>P</i> = 0.27).</p><div data-test="figure" data-container-section="figure" id="figure-2" data-title="Individual ancestry proportions."><figure><figcaption><b id="Fig2" data-test="figure-caption-text">Fig. 2: Individual ancestry proportions.</b></figcaption><div><div><a data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="https://medium.com/articles/s41586-023-05754-w/figures/2" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41586-023-05754-w/MediaObjects/41586_2023_5754_Fig2_HTML.png?as=webp"/><img aria-describedby="Fig2" src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41586-023-05754-w/MediaObjects/41586_2023_5754_Fig2_HTML.png" alt="figure 2" loading="lazy" width="685" height="587"/></picture></a></div><p><b>a</b>, Inferences from qpAdm (see Extended Data Table <a data-track="click" data-track-label="link" data-track-action="table anchor" href="https://medium.com/articles/s41586-023-05754-w#Tab4">2</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://medium.com/articles/s41586-023-05754-w#MOESM1">Supplementary Information</a> for model details and statistical fit). Blue represents African ancestry: the most common are Bantu-associated (common at southern sites) and Makwasinyi associated (northern sites), which itself is approximately 80% Bantu-related and 20% pastoralist-related. Yellow represents Southwest Asian ancestry: Persian or Arabian. Grey represents Indian ancestry. Bars represent s.e.m., computed using a block jackknife across all 5-centimorgan (cM) segments of the autosomes, and are meaningful even for single individuals as the genome contains information from a large sample size of ancestors. <b>b</b>, Ternary plot of Makwasinyi, Persian and Indian ancestry components in Mtwapa and Faza (red (high coverage) and yellow (low coverage)) and Manda (blue (high coverage) and green (low coverage)). Individuals with higher coverage (&gt;100,000 SNPs overlapping positions on the Human Origins SNP array) are used to fit a linear regression (dashed line), which intersects at nearly 100% Makwasinyi and 0% Persian and Indian, consistent with a Makwasinyi-related population with little or no recent Asian ancestry mixing with an already-mixed Persian–Indian population. <b>c</b>, Bar graph showing <i>P</i> values from Hotelling <i>T</i>-squared tests for a qpAdm model with a mixed Persian–Indian source. The <i>x</i>-axis specifies the proportion of Persian ancestry in the source.</p></div></figure></div><p>The type of African ancestry needed to make the models fit differed between individuals from the north (Mtwapa, Faza and Manda) and south (Kilwa and Songo Mnara) of the studied region. In Kenya, the best-fitting proxy African source is the inland Makwasinyi individuals (Extended Data Table <a data-track="click" data-track-label="link" data-track-action="table anchor" href="https://medium.com/articles/s41586-023-05754-w#Tab4">2</a>), who are themselves well-modelled as mixtures of about 80% Bantu-associated and 20% ancient eastern African Pastoral Neolithic ancestry<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 24" title="Prendergast Mary, E. et al. Ancient DNA reveals a multistep spread of the first herders into sub-Saharan Africa. Science 365, eaaw6275 (2019)." href="https://medium.com/articles/s41586-023-05754-w#ref-CR24" id="ref-link-section-d194324060e1927">24</a></sup> (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://medium.com/articles/s41586-023-05754-w#Fig3">3a</a>, Extended Data Table <a data-track="click" data-track-label="link" data-track-action="table anchor" href="https://medium.com/articles/s41586-023-05754-w#Tab5">3</a>, Supplementary Tables <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://medium.com/articles/s41586-023-05754-w#MOESM1">2</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://medium.com/articles/s41586-023-05754-w#MOESM1">3</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://medium.com/articles/s41586-023-05754-w#MOESM1">Supplementary Information</a>). In Tanzania, the best-fitting African proxy source is Bantu-associated without evidence of a Pastoral Neolithic contribution. We use the individual buried at around <span>ad</span> 1600 in Lindi as a proxy Bantu-associated source for the Kilwa individual and individual I19550 from Songo Mnara (Extended Data Table <a data-track="click" data-track-label="link" data-track-action="table anchor" href="https://medium.com/articles/s41586-023-05754-w#Tab4">2</a>).</p><div data-test="figure" data-container-section="figure" id="figure-3" data-title="Inferred admixture events along the eastern African coast."><figure><figcaption><b id="Fig3" data-test="figure-caption-text">Fig. 3: Inferred admixture events along the eastern African coast.</b></figcaption><div><div><a data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="https://medium.com/articles/s41586-023-05754-w/figures/3" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41586-023-05754-w/MediaObjects/41586_2023_5754_Fig3_HTML.png?as=webp"/><img aria-describedby="Fig3" src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41586-023-05754-w/MediaObjects/41586_2023_5754_Fig3_HTML.png" alt="figure 3" loading="lazy" width="685" height="445"/></picture></a></div><p>African populations are represented in shades of blue; Southwest Asian and Indian populations are represented in shades of yellow. Populations with both colours represent those that are admixed between the corresponding proxy source populations.</p></div></figure></div><p>Although three continental sources are required to fit the data, the individuals from Manda, Faza and Mtwapa form a cline in PCA, suggesting two proximal source populations (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://medium.com/articles/s41586-023-05754-w#Fig2">2b</a>). Using linear regression, we extrapolate the ancestry of these two sources and infer that one was consistent with a 100% African origin (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://medium.com/articles/s41586-023-05754-w#MOESM1">5</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://medium.com/articles/s41586-023-05754-w#MOESM1">Supplementary Information</a>). The same analysis concludes that the other source had both Persian and Indian ancestry. This is consistent with sub-Saharan Africans mixing with a group that already had a mixture of Persian and Indian ancestry components. Given the two different African sources for the northern and southern individuals, there must have been at least two, but possibly more, admixture events. This would be expected if people of mixed Persian and Indian ancestry had children with people from different local African populations at different locations along the coast.</p><p>When we analyse the Mtwapa and Faza, Manda, Kilwa and Songo Mnara individuals separately, the estimated proportions of Eurasian ancestry from India overlap, which could be consistent with a homogeneous source population of mixed Indian–Persian ancestry for all sites (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://medium.com/articles/s41586-023-05754-w#Fig2">2c</a>). However, a variable proportion among the early immigrants cannot be ruled out, and thus we cannot distinguish between scenarios of two or more streams of Persian–Indian migrants. Our statistical power to detect Indian ancestry relies on pooling of data from multiple individuals. There are a number of individuals with limited data or low Eurasian ancestry, for example individual I8816 from Kilwa, for which we have little power to detect Indian ancestry and cannot definitely document it (<a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://medium.com/articles/s41586-023-05754-w#MOESM1">Supplementary Information</a>).</p></div></div></section><section data-title="Males from Persia and females from Africa"><div id="Sec5-section"><h2 id="Sec5">Males from Persia and females from Africa</h2><div id="Sec5-content"><p>We tested whether male and female ancestors contributed the same proportions of African-like, Persian-like and Indian-like ancestry to ancient individuals in the northern coastal sites and Kilwa (Table <a data-track="click" data-track-label="link" data-track-action="table anchor" href="https://medium.com/articles/s41586-023-05754-w#Tab1">1</a>). To carry out this analysis, we used the fact that chromosomes 1–22, the X chromosome, mitochondrial DNA (mtDNA) and the Y chromosome are passed down to subsequent generations in different ways by males and females. We could not perform the same analysis at Songo Mnara because no individuals with high quality data fit the three-way model.</p><div data-test="inline-table" data-container-section="table" id="table-1"><figure><figcaption><b id="Tab1" data-test="table-caption">Table 1 Population mixture and evidence of sex bias in three groups from the Swahili coast</b></figcaption></figure></div><p>We first analysed mtDNA (Extended Data Table <a data-track="click" data-track-label="link" data-track-action="table anchor" href="https://medium.com/articles/s41586-023-05754-w#Tab3">1</a> and Table <a data-track="click" data-track-label="link" data-track-action="table anchor" href="https://medium.com/articles/s41586-023-05754-w#Tab1">1</a>). Analysing 62 individuals with confidently determined mtDNA haplogroups (including relatives and individuals with low coverage genome-wide data), we find that 59 carry an L* haplogroup, which in the present day is almost entirely restricted to sub-Saharan Africans<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 27" title="Quintana-Murci, L. et al. Where West meets East: the complex mtDNA landscape of the Southwest and Central Asian Corridor. Am. J. Hum. Genet. 74, 827–845 (2004)." href="https://medium.com/articles/s41586-023-05754-w#ref-CR27" id="ref-link-section-d194324060e2523">27</a></sup>. The exceptions are a pair of first- or second-degree relatives from Mtwapa carrying M30d1, which in the present day is largely restricted to South Asia<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 27" title="Quintana-Murci, L. et al. Where West meets East: the complex mtDNA landscape of the Southwest and Central Asian Corridor. Am. J. Hum. Genet. 74, 827–845 (2004)." href="https://medium.com/articles/s41586-023-05754-w#ref-CR27" id="ref-link-section-d194324060e2527">27</a></sup>, and an individual with haplogroup R0+16189, which today is characteristic of Saudi Arabia and the Horn of Africa<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 28" title="Gandini, F. et al. Mapping human dispersals into the Horn of Africa from Arabian Ice Age refugia using mitogenomes. Sci. Rep. 6, 25472 (2016)." href="https://medium.com/articles/s41586-023-05754-w#ref-CR28" id="ref-link-section-d194324060e2531">28</a></sup>. These results are consistent with female ancestry deriving overwhelmingly from African sources.</p><p>Analysing male-transmitted Y chromosome DNA, we find that two out of three non-first-degree related males from Manda carry haplogroup J2, and the third carries G2. Both haplogroups are characteristic of Southwest Asia (plausibly Persia) and are largely absent in sub-Saharan Africans<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 29" title="Cadenas, A. M. et al. Y-chromosome diversity characterizes the Gulf of Oman. Eur. J. Hum. Genet. 16, 374–386 (2008)." href="https://medium.com/articles/s41586-023-05754-w#ref-CR29" id="ref-link-section-d194324060e2538">29</a></sup>. The Kilwa individual also carries J2. Fourteen out of 19 males from Mtwapa have Y chromosome haplogroups in the J family, and two are of the R1a haplogroup, all considered typically non-African. Only 3 out of 19 Mtwapa males, along with the Faza male, are in the E1 family characteristic of sub-Saharan Africa.</p><p>We next compared chromosome X, which occurs as two copies in females and one in males and so reflects mostly female history, to the autosomes (chromosomes 1–22), which equally reflect female and male history (Methods and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://medium.com/articles/s41586-023-05754-w#MOESM1">Supplementary Information</a>). Chromosome X estimates of African ancestry are higher than on the autosomes at all sites, providing an independent line of evidence that African ancestry is primarily from females and Persian ancestry is primarily from males (Table <a data-track="click" data-track-label="link" data-track-action="table anchor" href="https://medium.com/articles/s41586-023-05754-w#Tab1">1</a>). Assuming that the mixture occurred over just a few generations, we obtain quantitative estimates of the proportion of African ancestry from females as 100% at Manda, 69–97% at Mtwapa-Faza, and 69–100% at Kilwa (Methods and Table <a data-track="click" data-track-label="link" data-track-action="table anchor" href="https://medium.com/articles/s41586-023-05754-w#Tab1">1</a>). We estimate Persian ancestry at Mtwapa-Faza and Kilwa to be 100%, and at Manda as 90–100%. If the mixture occurred over more generations, we cannot obtain a point estimate, but can nevertheless infer primarily African female and Persian male sources.</p><p>Together, these multiple lines of evidence show that the Southwest Asian ancestors of the Mtwapa and Faza, Manda and Kilwa individuals were almost entirely male, whereas the African ancestors were almost entirely female.</p></div></div></section><section data-title="Mixing began by ad 1000"><div id="Sec6-section"><h2 id="Sec6">Mixing began by <span>ad</span> 1000</h2><p>We estimated when mixture occurred on the basis of the sizes of stretches of ancestry inherited from the ancestral populations, which break up at a known rate every generation<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 30" title="Chintalapati, M., Patterson, N. &amp; Moorjani, P. The spatiotemporal patterns of major human admixture events during the European Holocene. eLife 11, e77625 (2022)." href="https://medium.com/articles/s41586-023-05754-w#ref-CR30" id="ref-link-section-d194324060e2571">30</a></sup>. We calculated 95% confidence intervals for the inferred dates of <span>ad</span> 795–1085 for a pool of the northern Mtwapa, Faza and Manda individuals, and <span>ad</span> 708–1219 for a pool of the southern Kilwa and Songo Mnara I19550 individuals (Table <a data-track="click" data-track-label="link" data-track-action="table anchor" href="https://medium.com/articles/s41586-023-05754-w#Tab1">1</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://medium.com/articles/s41586-023-05754-w#Fig6">3</a>). The uncertainty intervals overlap from <span>ad</span> 795 to <span>ad</span> 1085. These estimates would be biased too old if there was a marine reservoir effect. The inferred dates also reflect an assumption that the mixture occurred all at once; however, mixture of Eurasians and Africans was certainly drawn out over a number of generations, and indeed the historical evidence and our genetic analysis that follows document continued incorporation of migrants from both inland Africa and Eurasia until the present. However, simulations show that mixture must have begun by the inferred date<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 31" title="Narasimhan Vagheesh, M. et al. The formation of human populations in South and Central Asia. Science 365, eaat7487 (2019)." href="https://medium.com/articles/s41586-023-05754-w#ref-CR31" id="ref-link-section-d194324060e2594">31</a></sup>, and thus we can be confident that already-mixed males with both Indian and Persian ancestry were present along the coast by around <span>ad</span> 1000, and began mixing with primarily female sub-Saharan Africans by that time.</p></div></section><section data-title="Arabians and other migratory influences"><div id="Sec7-section"><h2 id="Sec7">Arabians and other migratory influences</h2><div id="Sec7-content"><p>Although almost all the coastal individuals we analysed had Asian ancestry, there were exceptions. Some early modern individuals at Lindi and Songo Mnara showed no evidence of recent Asian ancestry (I14001 and I7944) (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://medium.com/articles/s41586-023-05754-w#Fig5">2a</a>, Extended Data Table <a data-track="click" data-track-label="link" data-track-action="table anchor" href="https://medium.com/articles/s41586-023-05754-w#Tab5">3</a>, Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://medium.com/articles/s41586-023-05754-w#MOESM1">11</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://medium.com/articles/s41586-023-05754-w#MOESM1">Supplementary Information</a>). We find possible Malagasy-associated ancestry in Songo Mnara (I19547) (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://medium.com/articles/s41586-023-05754-w#Fig5">2a</a>, Extended Data Table <a data-track="click" data-track-label="link" data-track-action="table anchor" href="https://medium.com/articles/s41586-023-05754-w#Tab5">3</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://medium.com/articles/s41586-023-05754-w#MOESM1">Supplementary Information</a>). Our finding of coastal individuals who differ from others from similar times or regions attest to continued exchange with people in the Indian Ocean trading network, although our sample size is too small to identify general patterns.</p><p>For some of the individuals in our study with Asian ancestry (from Manda, Kilwa and I19550 from Songo Mnara), there is only evidence of Persian or Persian–Indian ancestry. However, other individuals, particularly from Mtwapa, can only be modelled by sources with some Arabian-associated ancestry when using other Mtwapa or Manda individuals as a source (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://medium.com/articles/s41586-023-05754-w#MOESM1">6</a>, Supplementary Tables <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://medium.com/articles/s41586-023-05754-w#MOESM1">12–15</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://medium.com/articles/s41586-023-05754-w#MOESM1">Supplementary Information</a>). We were unable to determine the exact source of the Arabian-associated ancestry. However, we know that it is somewhere on the genetic gradient between Arabians and Persians. A proxy source that provides a fit for this Arabian-related ancestry at Mtwapa is present-day people who live on the shores of the Strait of Hormuz, which separates the Arabian Peninsula from Iran. The Strait of Hormuz and the Swahili coast were under Omani control by the end of the seventeenth century.</p><p>Direct genetic evidence for Arabian-associated migration comes from two individuals from Songo Mnara. Both date to the early modern period when contacts with Arabia are well documented, and they can only be modelled with Arabian-related ancestry in qpAdm (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://medium.com/articles/s41586-023-05754-w#Fig2">2a</a>). Analyses of present-day coastal populations also point to Arabian genetic influences. Whereas the Asian ancestry of some individuals can be modelled entirely as Persian–Indian as in medieval Manda, for other individuals, ancestry from the Strait of Hormuz is a better fit (coloured salmon in Supplementary Data File <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://medium.com/articles/s41586-023-05754-w#MOESM4">7</a>).</p></div></div></section><section data-title="Relating medieval to modern Swahili"><div id="Sec8-section"><h2 id="Sec8">Relating medieval to modern Swahili</h2><div id="Sec8-content"><p>We assembled genome-wide data for two groups of present-day people who identify as Swahili: 89 with previously reported data<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 32" title="Brucato, N. et al. The Comoros show the earliest Austronesian gene flow into the Swahili Corridor. Am. J. Hum. Genet. 102, 58–68 (2018)." href="https://medium.com/articles/s41586-023-05754-w#ref-CR32" id="ref-link-section-d194324060e2660">32</a></sup> and 93 for which we generated new data.</p><p>Most of the individuals in the previously published dataset (87 out of 89) have only a modest inheritance from people with ancestry resembling the medieval people that we sampled from nearby coastal areas (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://medium.com/articles/s41586-023-05754-w#Fig5">2a</a>, Extended Data Table <a data-track="click" data-track-label="link" data-track-action="table anchor" href="https://medium.com/articles/s41586-023-05754-w#Tab5">3</a>, Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://medium.com/articles/s41586-023-05754-w#MOESM1">16</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://medium.com/articles/s41586-023-05754-w#MOESM1">Supplementary Information</a>). We estimate that 11 ± 4% have medieval Swahili-associated ancestry; 84 ± 3% have Bantu-associated ancestry; and 6 ± 3% have Pastoral Iron Age-associated ancestry<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 24" title="Prendergast Mary, E. et al. Ancient DNA reveals a multistep spread of the first herders into sub-Saharan Africa. Science 365, eaaw6275 (2019)." href="https://medium.com/articles/s41586-023-05754-w#ref-CR24" id="ref-link-section-d194324060e2679">24</a></sup> (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://medium.com/articles/s41586-023-05754-w#Fig5">2a</a> and Extended Data Table <a data-track="click" data-track-label="link" data-track-action="table anchor" href="https://medium.com/articles/s41586-023-05754-w#Tab5">3</a>). These patterns are mirrored in the Y chromosome haplogroups, with 95% having typically African haplogroups—in contrast to the medieval coastal individuals, for whom almost all Y haplogroups are associated with Near Eastern people (Tables <a data-track="click" data-track-label="link" data-track-action="table anchor" href="https://medium.com/articles/s41586-023-05754-w#Tab1">1</a> and <a data-track="click" data-track-label="link" data-track-action="table anchor" href="https://medium.com/articles/s41586-023-05754-w#Tab2">2</a>). However, the newly generated data show a much higher inferred proportion of ancestry from groups similar to the medieval ones, ranging from 46 to 77% medieval Swahili-associated ancestry (depending on whether we use a pool of Manda and Mtwapa individuals as the sources) (Extended Data Table <a data-track="click" data-track-label="link" data-track-action="table anchor" href="https://medium.com/articles/s41586-023-05754-w#Tab6">4</a> and Supplementary Data File <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://medium.com/articles/s41586-023-05754-w#MOESM4">7</a>). Y chromosome haplogroups in the new data are also consistent with a greater contribution from medieval peoples: the African-associated Y-haplogroup frequency was 45%, much larger than the 17% in the medieval individuals, but smaller than the 95% estimated in the previously published study (Table <a data-track="click" data-track-label="link" data-track-action="table anchor" href="https://medium.com/articles/s41586-023-05754-w#Tab2">2</a>).</p><div data-test="inline-table" data-container-section="table" id="table-2"><figure><figcaption><b id="Tab2" data-test="table-caption">Table 2 Comparison of haplogroup distributions in medieval Swahili individuals with those from two studies of present-day Swahili people</b></figcaption></figure></div><p>The differences in the proportions of ancestry in the present-day individuals identifying as Swahili may reflect differences in how they were estimated. The previously published dataset included people from the coastal towns of Kilifi, Lamu and Mombasa in Kenya who indicated that their family had been Swahili-speaking for the past three generations. The newly published dataset included people from 13 locations along the Kenyan coast who indicated that their ancestors had lived in coastal towns and had a Swahili identity for many generations, thus enriching for more traditional upper-class Swahili people who plausibly retained more ancestry from medieval coastal individuals<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 30" title="Chintalapati, M., Patterson, N. &amp; Moorjani, P. The spatiotemporal patterns of major human admixture events during the European Holocene. eLife 11, e77625 (2022)." href="https://medium.com/articles/s41586-023-05754-w#ref-CR30" id="ref-link-section-d194324060e2920">30</a></sup>. The greater medieval coastal ancestry may also reflect isolation. In the newly published dataset from the mainland, individuals from the site of Jomvu Kuu had significantly less medieval coastal ancestry than the individuals from the other sites, all of which were from islands that were plausibly more isolated from admixture with inland groups (<i>P</i> = 3.3 × 10<sup>−6</sup>, one-sided Wilcoxon rank sum test).</p><p>We used the lengths of African and Persian genetic segments to estimate the age of admixture in the newly published modern individuals<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 30" title="Chintalapati, M., Patterson, N. &amp; Moorjani, P. The spatiotemporal patterns of major human admixture events during the European Holocene. eLife 11, e77625 (2022)." href="https://medium.com/articles/s41586-023-05754-w#ref-CR30" id="ref-link-section-d194324060e2932">30</a></sup> at <span>ad</span> 1096–1410. These data are more recent than the medieval coastal samples, consistent with ongoing mixture with African or Asian populations since medieval times.</p></div></div></section><section data-title="No recent Asian ancestry in inland people"><div id="Sec9-section"><h2 id="Sec9">No recent Asian ancestry in inland people</h2><p>The Makwasinyi individuals date to the past three centuries, and are from deep in the Tsavo region, nearly 50 km from the nearest population centre. The Makwasinyi group fits as a proxy source for African ancestry in qpAdm modelling of the Mtwapa, Faza and Manda individuals, but unlike these individuals, qpAdm finds no evidence of recent Asian ancestry in the Makwasinyi group (similar to present-day non-coastal populations; Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://medium.com/articles/s41586-023-05754-w#MOESM1">17</a>). Instead, Makwasinyi individuals are similar in ancestry to the modern individuals identified as Swahili in the previously reported dataset<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 32" title="Brucato, N. et al. The Comoros show the earliest Austronesian gene flow into the Swahili Corridor. Am. J. Hum. Genet. 102, 58–68 (2018)." href="https://medium.com/articles/s41586-023-05754-w#ref-CR32" id="ref-link-section-d194324060e2950">32</a></sup>. They are well-modelled as 21.3 ± 1.2% Pastoral Neolithic-associated ancestry (from herders present in eastern African after 3000 <span>bc</span>) and 78.7 ± 1.2% Bantu-associated ancestry (from farmers present after 1000 <span>bc</span>) (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://medium.com/articles/s41586-023-05754-w#Fig5">2a</a>). We did not detect sex bias in the history of the formation of this population (Methods, <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://medium.com/articles/s41586-023-05754-w#MOESM1">Supplementary Information</a>). The average date for the Bantu-Pastoral Neolithic-associated mixture is around <span>ad</span> 300–1200 (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://medium.com/articles/s41586-023-05754-w#Fig6">3</a>), with most of this range consistent with the archaeological evidence for the impact of the Bantu expansion on this region.</p></div></section><section data-title="Discussion"><div id="Sec10-section"><h2 id="Sec10">Discussion</h2><div id="Sec10-content"><p>A key finding of this study is evidence of mixture at roughly <span>ad</span> 1000 between peoples of African and Persian ancestries (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://medium.com/articles/s41586-023-05754-w#Fig3">3</a> and Table <a data-track="click" data-track-label="link" data-track-action="table anchor" href="https://medium.com/articles/s41586-023-05754-w#Tab1">1</a>). This is consistent with the Kilwa Chronicle, which describes the arrival of Persians on the Swahili coast and interactions between them and coastal residents. Whether or not this history has a basis in an actual voyage, the ancient DNA provides direct evidence for Persian-associated ancestry being derived overwhelmingly from males and arriving on the eastern African coast by about <span>ad</span> 1000. This timing coincides with archaeological evidence for a substantial cultural transformation on the coast, including the widespread adoption of Islam<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 10" title="Fleisher, J. et al. When did the Swahili become maritime? Am. Anthropol. 117, 100–115 (2015)." href="https://medium.com/articles/s41586-023-05754-w#ref-CR10" id="ref-link-section-d194324060e2993">10</a></sup>. At Kilwa, coin evidence has dated a ruler linked to a Shirazi (Persian) dynasty, Ali bin al-Hasan, to the mid-eleventh century<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 33" title="Horton, M., Olsen, J., Fleisher, J. &amp; Wynne-Jones, S. The chronology of Kilwa Kisiwani, ad 800–1500. Afr. Archaeol. Rev. 39, 143–166 (2022)." href="https://medium.com/articles/s41586-023-05754-w#ref-CR33" id="ref-link-section-d194324060e2998">33</a></sup>. The genetic evidence suggests that this arrival was accompanied by mixture, which began by <span>ad</span> 1000, and continued later. People of both African and Asian ancestry made major contributions, with African proportions of approximately 57% on average at Mtwapa and Faza, 32% at Manda, 67% at Songo Mnara and 74% at Kilwa (Table <a data-track="click" data-track-label="link" data-track-action="table anchor" href="https://medium.com/articles/s41586-023-05754-w#Tab1">1</a>, Extended Data Table <a data-track="click" data-track-label="link" data-track-action="table anchor" href="https://medium.com/articles/s41586-023-05754-w#Tab4">2</a> and Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://medium.com/articles/s41586-023-05754-w#MOESM1">8</a>).</p><p>Archaeological evidence provides important context for our genetic findings. The individuals that we analysed lived in the thirteenth to eighteenth centuries, and were excavated mostly from elite contexts. However, coastal sites from around <span>ad</span> 1000, when the mixture occurred, showed little evidence for distinct societal elites. Three of the sites sampled here (Mtwapa, Songo Mnara and Faza) did not exist as towns in <span>ad</span> 1000, and so these admixed populations moved to those towns later. Thus, the elite inhabitants of Mtwapa and other sites developed from admixed populations and were not foreign migrants or colonists.</p><p>Linguistic evidence provides further context. Kiswahili is a Bantu language, and since most ancestry in medieval Swahili people derives from African people, our results suggest that the children of immigrant men of Asian origin adopted the languages of their mothers, a common pattern in matrilocal cultures<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 34" title="Lansing, J. S. et al. Kinship structures create persistent channels for language transmission. Proc. Natl Acad. Sci. USA 114, 12910–12915 (2017)." href="https://medium.com/articles/s41586-023-05754-w#ref-CR34" id="ref-link-section-d194324060e3026">34</a></sup>. However, Kiswahili also has non-African influences, reflecting a millennium and a half of intense interaction with societies around the Indian Ocean rim. Persian loanwords contribute up to 3% of Kiswahili, but it is unclear whether they are derived directly from Persian or through adoption into other Indian Ocean languages<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 35" title="Haspelmath, M. &amp; Tadmor, U. Loanwords in the World’s Languages : A Comparative Handbook (De Gruyter Mouton, 2009)." href="https://medium.com/articles/s41586-023-05754-w#ref-CR35" id="ref-link-section-d194324060e3030">35</a></sup>. Arabic loanwords are the single largest non-Bantu element in Kiswahili<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 35" title="Haspelmath, M. &amp; Tadmor, U. Loanwords in the World’s Languages : A Comparative Handbook (De Gruyter Mouton, 2009)." href="https://medium.com/articles/s41586-023-05754-w#ref-CR35" id="ref-link-section-d194324060e3034">35</a></sup> (16–20% of words), and may be primarily due<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 20" title="Nurse, D. &amp; Spear, T. The Swahili: Reconstructing the History and Language of an African Society, 800–1500 (Univ. of Pennsylvania Press, 1985)." href="https://medium.com/articles/s41586-023-05754-w#ref-CR20" id="ref-link-section-d194324060e3038">20</a></sup> to incorporations after <span>ad</span> 1500.</p><p>A recurrent theme of our findings is the different participation of males and females in population mixture events. We find evidence of predominantly male Southwest Asian ancestors mixing with predominantly female African and, to a much lesser extent, female Indian ancestors in the lineages of medieval people on the Swahili coast. This provides evidence for asymmetric social interactions between groups as cultural contact occurred, although such genetic data cannot reveal the processes contributing to these patterns.</p><p>This study provides information from only a subset of times and places relevant to the medieval coastal civilization, and it is important to recognize these biases. The geographical coverage is skewed towards Kenya, with the individuals from Tanzanian sites such as Songo Mnara, Kilwa and Lindi being sufficient to identify similarities and differences in ancestry profiles from Kenya but not sufficient to define a general pattern. In addition, the individuals that we analysed were not fully representative of all social and economic groups in Swahili society. Nearly all of the coastal graves and tombs in this study occupied prominent positions in medieval and early modern townscapes (see <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://medium.com/articles/s41586-023-05754-w#MOESM1">Supplementary Information</a>). With the possible exceptions of Kilwa, Lindi and Faza, we analysed elite individuals from high-profile coastal sites. However, the Swahili cultural world included many non-elite settlements, where ancestry might be systematically different<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 36" title="LaViolette, A. &amp; Fleisher, J. B. Developments in rural life on the Eastern African coast, a.d. 700–1500. J. Field Archaeol. 43, 380–398 (2018)." href="https://medium.com/articles/s41586-023-05754-w#ref-CR36" id="ref-link-section-d194324060e3055">36</a></sup>. Our analysis of data from two samplings of present-day people who identify as Swahili with different strategies for determining this identity also reveals qualitative differences in ancestry patterns, revealing how groups identified as Swahili retain high substructure and variation today.</p><p>These findings highlight multiple directions for future work on ancient DNA. One approach is to study individuals pre-dating the twelfth century, including before and after the major population mixtures that we show occurred around <span>ad</span> 1000. Another approach is to study individuals from unsampled parts of the Swahili world, including the present-day countries of Somalia, Mozambique, the Comoros Islands and Madagascar. However, the results presented here provide unambiguous evidence of ongoing cultural mixing on the East African coast for more than a millennium, in which African people interacted and had families with immigrants from other parts of Africa and the Indian Ocean world. Narratives of ancestry on the eastern African coast have a complex history, and the genetic findings of long-standing, sex-biased mixtures add to this complexity.</p></div></div></section><section data-title="Methods"><div id="Sec11-section"><h2 id="Sec11">Methods</h2><div id="Sec11-content"><h3 id="Sec12">Inclusion and ethics</h3><p>The present-day communities of the Swahili coast have strong traditions of connection to the people of the medieval coastal towns—including in some cases a tradition of descent from the people who lived in these communities, as well as shared language and religion—and thus community consultation is an important part of this work. Many medieval and present-day coastal people are also Muslims, and thus it is important to carry out any analysis in ways that are sensitive to Muslim proscriptions against disturbing the dead. The sampling for this study emerged from decades-long community-based archaeology projects—including by S.K. and C.M.K. in the Kenya region, and by S.W.-J., J.F., Tanzanian Antiquities and the Songo Mnara Ruins Committee in the Tanzania region—that involved participation in and return of results to local communities, following rules for handling and reburying of remains agreed by the communities. Prior to submission of this study for publication, the corresponding authors held return-of-results consultation meetings in Lamu, Songo Mnara and Kilwa Kisiwani, and feedback from these engagements was incorporated into the final manuscript. The community meeting held in Lamu was both in person and on zoom. Mtwapa no longer has a viable Swahili community. The descendants of the residents of Manda are no longer visible, but given Manda’s proximity to Lamu, we chose Lamu as the most viable community to host the forum. Makwasinyi elders were consulted in 2001 and 2002 during fieldwork and again to discuss the results of the present analysis.</p><h3 id="Sec13">Ancient DNA data generation</h3><p>To generate the genetic data for the samples in Extended Data Table <a data-track="click" data-track-label="link" data-track-action="table anchor" href="https://medium.com/articles/s41586-023-05754-w#Tab3">1</a> and Supplementary Data File <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://medium.com/articles/s41586-023-05754-w#MOESM4">2</a> (by-library metrics in Supplementary Data File <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://medium.com/articles/s41586-023-05754-w#MOESM4">1</a>), we used established protocols in dedicated clean rooms, involving first sampling typically 40 mg of powder from skeletal remains, then extracting DNA using methods designed to retain short, damaged fragments<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 37" title="Dabney, J. et al. Complete mitochondrial genome sequence of a Middle Pleistocene cave bear reconstructed from ultrashort DNA fragments. Proc. Natl Acad. Sci. USA 110, 15758–15763 (2013)." href="https://medium.com/articles/s41586-023-05754-w#ref-CR37" id="ref-link-section-d194324060e3094">37</a></sup>, then building individually barcoded libraries<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 38" title="Rohland, N. et al. Partial uracil-DNA-glycosylase treatment for screening of ancient DNA. Philos. Trans. R. Soc. Lond. B 370, 20130624–20130624 (2015)." href="https://medium.com/articles/s41586-023-05754-w#ref-CR38" id="ref-link-section-d194324060e3098">38</a></sup> after incubation with uracil DNA glycosylase (UDG) with the goal of greatly reducing the characteristic errors typical of ancient DNA. We amplified libraries and carried out in-solution enrichment of them for about 1.2 million SNPs<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 39" title="Fu, Q. et al. An early modern human from Romania with a recent Neanderthal ancestor. Nature 524, 216–219 (2015)." href="https://medium.com/articles/s41586-023-05754-w#ref-CR39" id="ref-link-section-d194324060e3103">39</a></sup> as well as baits targeting the mitochondrial genome<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 40" title="Maricic, T., Whitten, M. &amp; Pääbo, S. Multiplexed DNA sequence capture of mitochondrial genomes using PCR products. PLoS ONE 5, e14004–e14004 (2010)." href="https://medium.com/articles/s41586-023-05754-w#ref-CR40" id="ref-link-section-d194324060e3107">40</a></sup>. We sequenced the enriched products along with a small amount of unenriched library on Illumina NextSeq 500 or HiSeqX10 instruments. The resulting sequenced paired-end reads were separated into respective libraries and stripped of identification tags and adapter related sequences. Read pairs were merged prior to alignment, requiring a minimum of 15 base pair overlap, allowing one mismatch if base quality was at least 20, or up to three mismatches at lower base quality. We restricted to sequences that were at least 30 base pairs long. The resulting sequences were aligned using samse from bwa-v0.6.1<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 41" title="Li, H. &amp; Durbin, R. Fast and accurate long-read alignment with Burrows–Wheeler transform. Bioinformatics 26, 589–595 (2010)." href="https://medium.com/articles/s41586-023-05754-w#ref-CR41" id="ref-link-section-d194324060e3111">41</a></sup> separately to both the human genome reference sequence (hg19) and the mitochondrial RSRS genome<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 42" title="Behar, DoronM. et al. A “Copernican” reassessment of the human mitochondrial DNA tree from its root. Am. J. Hum. Genet. 90, 675–684 (2012)." href="https://medium.com/articles/s41586-023-05754-w#ref-CR42" id="ref-link-section-d194324060e3115">42</a></sup> to allow the targeted nuclear and mitochondrial specific sequences to be assessed.</p><p>We performed a series of quality control measures. We inferred the rate of mismatch of the sequences mapping to the mitochondrial consensus sequence (of libraries with at least twofold coverage) and flagged libraries where the upper bound of the inferred match rate<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 43" title="Fu, Q. et al. DNA analysis of an early modern human from Tianyuan Cave, China. Proc. Natl Acad. Sci. USA 110, 2223–2227 (2013)." href="https://medium.com/articles/s41586-023-05754-w#ref-CR43" id="ref-link-section-d194324060e3122">43</a></sup> was at least 95%; we also examined the rate of polymorphism on the X chromosome in males and flagged individuals as appreciably contaminated if the lower bound of the 95% confidence interval<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 44" title="Korneliussen, T. S., Albrechtsen, A. &amp; Nielsen, R. ANGSD: analysis of next generation sequencing data. BMC Bioinformatics 15, 356 (2014)." href="https://medium.com/articles/s41586-023-05754-w#ref-CR44" id="ref-link-section-d194324060e3126">44</a></sup> was at least 1%. We tested if the ratio of Y chromosome to the sum of X and Y chromosome sequences was in the range expected for DNA entirely from a female (&lt;3%) or male (&gt;35%), and we tested for an elevated rate of cytosine to thymine substitution in the final nucleotide using a threshold of at least 1.8% (this was a lower threshold than is typical in ancient DNA studies due to the young age of the samples; the median damage rate of libraries is 9.5%). If a sample had sufficient evidence of contamination, we restricted to only damaged molecules of sequenced DNA, and these samples are denoted with ‘_d’ in their Genetic ID in Supplementary Data File <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://medium.com/articles/s41586-023-05754-w#MOESM4">2</a>. Several samples did not pass quality control; these are any samples where all libraries failed in Supplementary Data File <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://medium.com/articles/s41586-023-05754-w#MOESM4">1</a>. No individual from two sampled sites—Bungule and Shanga—passed quality control. For each individual, we represented each position in the genome based on a single randomly selected sequence</p><p>A total of 80 individuals passed quality control screening. However, the analyses in this study focus on the 54 non-outlier individuals that are not first- or second-degree relatives of another individual in the study for which we have better quality data and for which we have data on a minimum of 15,000 SNPs. Relative relationships are determined by a method similar to READ<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 45" title="Monroy Kuhn, J. M., Jakobsson, M. &amp; Günther, T. Estimating genetic kin relationships in prehistoric populations. PLoS ONE 13, e0195491 (2018)." href="https://medium.com/articles/s41586-023-05754-w#ref-CR45" id="ref-link-section-d194324060e3139">45</a></sup>.</p><p>Supplementary Data File <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://medium.com/articles/s41586-023-05754-w#MOESM4">1</a> provides a full report of sequencing results on both libraries that yielded data passing standard measures of ancient DNA authenticity and libraries that failed screening; we also report negative results as a guide for future sampling efforts.</p><h3 id="Sec14">Genotyping of modern Swahili individuals</h3><p>We genotyped 93 present-day Swahili individuals, originally collected in ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 21" title="Raaum, R. L. et al. in The Swahili World (eds Wynne-Jones, S. &amp; LaViolette, A. J.) 81–102 (Routledge, 2017)." href="https://medium.com/articles/s41586-023-05754-w#ref-CR21" id="ref-link-section-d194324060e3157">21</a></sup>, on the Affymetrix Human Origins SNP array<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 26" title="Patterson, N. et al. Ancient admixture in human history. Genetics 192, 1065–1093 (2012)." href="https://medium.com/articles/s41586-023-05754-w#ref-CR26" id="ref-link-section-d194324060e3161">26</a></sup>. As described in the study where mtDNA and Y chromosome haplogroup information for these individuals was initially reported<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 21" title="Raaum, R. L. et al. in The Swahili World (eds Wynne-Jones, S. &amp; LaViolette, A. J.) 81–102 (Routledge, 2017)." href="https://medium.com/articles/s41586-023-05754-w#ref-CR21" id="ref-link-section-d194324060e3165">21</a></sup>, individuals known to the research team to have origins in or near the targeted recruitment communities assisted in participant recruitment in a form of ‘snowball sampling’. On sampling days, potential participants belonging to families known to have long residency in the community were approached and, if they consented to participation, would be asked afterwards if they knew of anyone else not closely related to them who might be available that day who met the inclusion criteria of being a healthy individual of at least 18 years of age who is a resident of that town and whose grandparents all identify as belonging to a group typically included in the broader Swahili identity. Each participant was provided information about the project in either English or Swahili from Institutional Review Board (IRB)-approved forms and those who elected to continue provided written consent. Protocols for collecting saliva and genealogical information were reviewed and approved by the Lehman College (no. 141-09-070) and City University of New York Integrated IRB (no. 323935). The saliva samples and genealogical information were collected in December 2009 and January 2010 in the Kenyan towns of Faza, Jomvu Kuu, Kizingitini, Lamu, Mikindani, Ndau, Pate, Siyu, Tchundwa and Wasini. A total of 96 samples were chosen for whole-genome analysis to maximize geographic spread, with 16 individuals each from Faza, Kizingitini, Ndau, Pate, Wasini and Jomvu Kuu. We limited to individuals who self-reported that the origin of both their parents and grandparents are local and Swahili, with a priority for males. Data from a total of 93 individuals passed quality control and their data are reported and analysed.</p><p>We genotyped on the Human Origins SNP array 19 newly reported individuals from Madagascar, for whom samples were collected between 2007–2014 with approval by the Human Subjects’ Ethics Committees of the Health Ministry of Madagascar and by committees in France (Ministry of Research, National Commission for Data Protection and Liberties and Persons Protection Committee). Individuals all gave written consent before the study. Samples came from two locations in the north and south of the country. Sampled villages were founded before 1900, and individuals were 61 ± 15 years old, with the maternal grandmother and paternal grandfather born within 50 km of the sampling location.</p><p>We genotyped on the Human Origins SNP array 10 newly reported Emirati samples that were collected from Emirati nationals from the city of Al Ain, United Arab Emirates. Ethical approval was obtained from the Al Ain District Human Research Ethical Committee. The samples were collected from healthy adults, and first genotyped on another SNP array as controls for a rare disease study<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 46" title="Reiff, R. E. et al. METTL23, a transcriptional partner of GABPA, is essential for human cognition. Hum. Mol. Genet. 23, 3456–3466 (2014)." href="https://medium.com/articles/s41586-023-05754-w#ref-CR46" id="ref-link-section-d194324060e3175">46</a></sup>.</p><p>A list of the newly reported individuals with data on the Human Origins SNP array and some relevant information can be found in Supplementary Data File <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://medium.com/articles/s41586-023-05754-w#MOESM4">4</a>.</p><h3 id="Sec15">Principal component analysis</h3><p>We used smartpca version 18180 from EIGENSOFT version 8.0.0<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 47" title="Patterson, N., Price, A. L. &amp; Reich, D. Population structure and eigenanalysis. PLoS Genet. 2, e190 (2006)." href="https://medium.com/articles/s41586-023-05754-w#ref-CR47" id="ref-link-section-d194324060e3193">47</a></sup> with optional parameters (numoutlieriter: 2, numoutevec: 3, lsqproject: YES, newshrink: YES, and hiprecision: YES). We computed eigenvectors of the covariance matrix of SNPs from present-day individuals from Africa and Eurasia genotyped on the Affymetrix Human Origins (HO) SNP array, which targets approximately 600,000 SNPs that are a subset of those targeted by in-solution enrichment (see <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://medium.com/articles/s41586-023-05754-w#MOESM1">Supplementary Information</a> for a list of populations used). Ancient individuals and present-day individuals (most genotyped on the HO SNP array, and some also genotyped on the Illumina Human Omni5 Bead Chip<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 32" title="Brucato, N. et al. The Comoros show the earliest Austronesian gene flow into the Swahili Corridor. Am. J. Hum. Genet. 102, 58–68 (2018)." href="https://medium.com/articles/s41586-023-05754-w#ref-CR32" id="ref-link-section-d194324060e3200">32</a></sup>) were projected into the three-dimensional (3D) eigenspace determined by the first three principal components.</p><h3 id="Sec16">ADMIXTURE clustering analysis</h3><p>We prepared data for the ADMIXTURE<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 48" title="Alexander, D. H., Novembre, J. &amp; Lange, K. Fast model-based estimation of ancestry in unrelated individuals. Genome Res. 19, 1655–1664 (2009)." href="https://medium.com/articles/s41586-023-05754-w#ref-CR48" id="ref-link-section-d194324060e3213">48</a></sup> plots in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://medium.com/articles/s41586-023-05754-w#Fig1">1c</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://medium.com/articles/s41586-023-05754-w#Fig4">1</a> using PLINK2<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 49" title="Chang, C. C. et al. Second-generation PLINK: rising to the challenge of larger and richer datasets. GigaScience 4, 7 (2015)." href="https://medium.com/articles/s41586-023-05754-w#ref-CR49" id="ref-link-section-d194324060e3223">49</a></sup>. We applied the maf 0.01 option, which only includes SNPs with minor allele frequency of at least 0.01. We pruned SNPs based on LD, using the indep-pairwise option with a window size of 200 variants, a step size of 25 variants, and a pairwise <i>r</i><sup>2</sup> threshold of 0.4. We ran 4 replicates of ADMIXTURE with random seeds with <i>K</i> = 4 to <i>K</i> = 12 ancestral reference populations.</p><h3 id="Sec17">Estimates of mixture proportions</h3><p>We used qpAdm (ADMIXTOOLS version 7.0.2)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 26" title="Patterson, N. et al. Ancient admixture in human history. Genetics 192, 1065–1093 (2012)." href="https://medium.com/articles/s41586-023-05754-w#ref-CR26" id="ref-link-section-d194324060e3246">26</a></sup> to test if the Makwasinyi, Manda, Mtwapa and Faza, Kilwa, and Songo Mnara target populations can be formally modelled as having derived from a set of source populations (termed ‘left’ groups) relative to a set of reference populations (termed ‘right’ groups). See <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://medium.com/articles/s41586-023-05754-w#MOESM1">Supplementary Information</a> for further details. qpAdm also provides a <i>P</i> value for fit to the model based on a block jackknife across all 5 cM segments of the autosomes.</p><p>In our application of qpAdm, we use a cycling approach, treating the target as a linear combination of all possible subsets of the candidate source populations, and moving the other candidate source population to the right. Cycling populations to the right allows us to test if a proposed set of left source populations is consistent with being more closely related to the target than other populations. Thus, we can build the closest admixture model within the constraints of our dataset and test its fit to the data.</p><h3 id="Sec18">Y and mitochondrial haplogroups</h3><p>Mitochondrial haplogroups were determined using HaploGrep 2<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 50" title="Weissensteiner, H. et al. HaploGrep 2: mitochondrial haplogroup classification in the era of high-throughput sequencing. Nucleic Acids Res. 44, W58–W63 (2016)." href="https://medium.com/articles/s41586-023-05754-w#ref-CR50" id="ref-link-section-d194324060e3267">50</a></sup>. Y chromosome haplogroups were determined according to the Yfull tree version 8.09 (<a href="https://github.com/YFullTeam/YTree/blob/master/ytree/tree_8.09.0.json">https://github.com/YFullTeam/YTree/blob/master/ytree/tree_8.09.0.json</a>)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 51" title="Lazaridis, I. et al. The genetic history of the Southern Arc: A bridge between West Asia and Europe. Science 377, eabm4247 (2022)." href="https://medium.com/articles/s41586-023-05754-w#ref-CR51" id="ref-link-section-d194324060e3278">51</a></sup>.</p><h3 id="Sec19">Sex bias from X-autosome comparisons</h3><p>We estimated the extent to which the ancestry from each source population was contributed by female and male ancestors. To do this, we compared the inferences of proportions of ancestry on chromosomes 1–22 (the autosomes) which reflect 50% female and 50% male inheritance (the autosomal coefficient for proportion of a specific ancestry, <i>C</i><sub>A</sub> = (<i>m</i> + <i>f</i>)/2), and on chromosome X, which reflects 67% female and 33% male ancestry (the X chromosome coefficient for proportion of a specific ancestry, <i>C</i><sub>X</sub> = (2<i>f</i> + <i>m</i>)/3). We determined <i>Z</i>-scores for differences between these two estimates as in ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 52" title="Mathieson, I. et al. The genomic history of southeastern Europe. Nature 555, 197–203 (2018)." href="https://medium.com/articles/s41586-023-05754-w#ref-CR52" id="ref-link-section-d194324060e3314">52</a></sup>. For each of Mtwapa (including the one Faza individual), Manda, Kilwa and Makwasinyi, we quantified sex bias in the following manner. (1) We sampled 10<sup>6</sup> sets of coefficients by generating random numbers from a multivariate normal distribution based on the qpAdm-determined jackknife mean coefficients and the error covariance matrix for both the autosomes and chromosome X, separately. We ensured that the eigenvalues of the matrices are all greater than or equal to zero by adding a small offset to the matrices determined as the absolute value of its respective minimum eigenvalue. (2) We removed from consideration all infeasible sets of coefficients, namely those that include a coefficient below 0. (3) We next calculated the female and male proportions of ancestry for each source population. Given the coefficient <i>C</i><sub>A</sub> on the autosomes and the coefficient <i>C</i><sub>X</sub> on the X chromosome, we can determine the likely proportion of female and male ancestry from any given source by solving the system of two above equations, giving <i>f</i> = 3<i>C</i><sub>X</sub> − 2<i>C</i><sub>A</sub> and <i>m</i> = 4<i>C</i><sub>A</sub> − 3<i>C</i><sub>X</sub>. We calculate these <i>f</i> and <i>m</i> proportions for each source population for the sampled set of autosomal and X chromosome coefficients. (4) We normalize the proportions of female and male ancestry as <span>\(\hat{f}=f/(f+m)\)</span> and <span>\(\hat{m}=m/\left(f+m\right)\)</span> for each source for the sampled sets. (5) We found the mean and standard deviation of all the <span>\(\hat{f}\)</span> and <span>\(\hat{m}\)</span> values from the sampled set for each of the source populations. (3) We took 95% confidence intervals of all the sampled sets of ancestry proportions and record the values as ranges in Table <a data-track="click" data-track-label="link" data-track-action="table anchor" href="https://medium.com/articles/s41586-023-05754-w#Tab1">1</a>. Under the simple model of all the mixture occurring at once, these formulae can be interpreted as estimating the fraction of ancestors at the time of mixing that are on the male or female side. However, if the mixture was more gradual, the interpretation is more complicated albeit still informative about sex bias.</p><h3 id="Sec20">Proximate sources of the coastal cline</h3><p>We plot all the Mtwapa, Faza and Manda individuals on a ternary plot with their respective proportions of Makwasinyi, Iranian and Indian ancestry. We apply a linear regression to the high coverage individuals from Mtwapa (<a href="https://www.ncbi.nlm.nih.gov/nuccore/I19381">I19381</a>, <a href="https://www.ncbi.nlm.nih.gov/nuccore/I19394">I19394</a>, <a href="https://www.ncbi.nlm.nih.gov/nuccore/I19384">I19384</a>, <a href="https://www.ncbi.nlm.nih.gov/nuccore/I19414">I19414</a>, <a href="https://www.ncbi.nlm.nih.gov/nuccore/I19420">I19420</a>, <a href="https://www.ncbi.nlm.nih.gov/nuccore/I19413">I19413</a>, <a href="https://www.ncbi.nlm.nih.gov/nuccore/I19417">I19417</a>, <a href="https://www.ncbi.nlm.nih.gov/nuccore/I19401">I19401</a>, <a href="https://www.ncbi.nlm.nih.gov/nuccore/I23662">I23662</a>, <a href="https://www.ncbi.nlm.nih.gov/nuccore/I17409">I17409</a>, <a href="https://www.ncbi.nlm.nih.gov/nuccore/I19391">I19391</a>, <a href="https://www.ncbi.nlm.nih.gov/nuccore/I13611">I13611</a>, <a href="https://www.ncbi.nlm.nih.gov/nuccore/I17412">I17412</a>, <a href="https://www.ncbi.nlm.nih.gov/nuccore/I17413">I17413</a>, <a href="https://www.ncbi.nlm.nih.gov/nuccore/I23558">I23558</a>, <a href="https://www.ncbi.nlm.nih.gov/nuccore/I19415">I19415</a>, <a href="https://www.ncbi.nlm.nih.gov/nuccore/I23561">I23561</a>, and <a href="https://www.ncbi.nlm.nih.gov/nuccore/I21475">I21475</a>) and Manda (I7934) as seen in the ternary plot (see Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://medium.com/articles/s41586-023-05754-w#Fig2">2b</a> and Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://medium.com/articles/s41586-023-05754-w#MOESM1">3</a>). The associated Cartesian equation is <i>f</i>(<i>x</i>) = 0.09856<i>x</i> + 0.01301. The line intersects the Makwasinyi axis at 98 ± 6% (left circle in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://medium.com/articles/s41586-023-05754-w#Fig2">2b</a> and Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://medium.com/articles/s41586-023-05754-w#MOESM1">3</a>), allowing for 2 ± 6% Indian ancestry. This is statistically consistent with 100% Makwasinyi and 0% Iranian and 0% Indian ancestries. The other end of the regression line intersects the Indian axis (right circle in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://medium.com/articles/s41586-023-05754-w#Fig2">2b</a> and Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://medium.com/articles/s41586-023-05754-w#MOESM1">3</a>) at 12 ± 5% Indian ancestry and 88 ± 5% Iranian ancestry. We further refined these proportions by jackknifing the coefficient estimates. We estimate that one admixing source population is 98.4 ± 2.5% Makwasinyi-related, consistent with a parsimonious model of 100% Makwasinyi ancestry at that end of the cline. The other source population is 12.2 ± 2.8% Indian and 87.8 ± 2.8% Iranian (see <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://medium.com/articles/s41586-023-05754-w#MOESM1">Supplementary Information</a>).</p><h3 id="Sec21">Determining the date of mixture</h3><p>DATES measures the covariance between pairs of positions in the genome separated by a specified genetic distance, for an admixture model with two source populations<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 30" title="Chintalapati, M., Patterson, N. &amp; Moorjani, P. The spatiotemporal patterns of major human admixture events during the European Holocene. eLife 11, e77625 (2022)." href="https://medium.com/articles/s41586-023-05754-w#ref-CR30" id="ref-link-section-d194324060e3694">30</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 31" title="Narasimhan Vagheesh, M. et al. The formation of human populations in South and Central Asia. Science 365, eaat7487 (2019)." href="https://medium.com/articles/s41586-023-05754-w#ref-CR31" id="ref-link-section-d194324060e3697">31</a></sup>. This analysis is applied separately to all individuals, and the inferences are pooled across individuals to increase resolution. We pool the individuals from Mtwapa, Faza, and Manda into a northern group, and the individuals from Kilwa and Songo Mnara <a href="https://www.ncbi.nlm.nih.gov/nuccore/I19550">I19550</a> into a southern group. We use a 28 ± 2 year-to-generation conversion estimate<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 53" title="Moorjani, P. et al. A genetic method for dating ancient genomes provides a direct estimate of human generation interval in the last 45,000 years. Proc. Natl Acad. Sci. USA 113, 5652 (2016)." href="https://medium.com/articles/s41586-023-05754-w#ref-CR53" id="ref-link-section-d194324060e3708">53</a></sup> to calculate the average date of the admixture.</p><h3 id="Sec22">Reporting summary</h3><p>Further information on research design is available in the <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://medium.com/articles/s41586-023-05754-w#MOESM2">Nature Portfolio Reporting Summary</a> linked to this article.</p></div></div></section>
                </div><div>
            <section data-title="Data availability"><div id="data-availability-section"><h2 id="data-availability">Data availability</h2><p>Bam files of aligned reads for the 80 newly published ancient individuals can be obtained from the European Nucleotide Archive (accession no. <a href="https://www.ebi.ac.uk/ena/data/view/PRJEB58698">PRJEB58698</a>). SNP array genotype data for 122 newly reported modern individuals can be obtained from a permanent link at the Dataverse repository at (<a href="https://doi.org/10.7910/DVN/NC28XW">https://doi.org/10.7910/DVN/NC28XW</a>). The data for 103 of these individuals can be downloaded without registering, whereas the informed consent for 19 individuals from Madagascar is not consistent with unmediated public posting of data, and may be downloaded after filling out a form with an email address and institutional or professional affiliation, and including an affirmation of the following statements: (a) I will not distribute the data outside my collaboration; (b) I will not post the data publicly; (c) I will make no attempt to connect the genetic data to personal identifiers for the samples; (d) I will use the data only for studies of population history; and (e) I will not use the data for commercial purposes.</p></div></section><div id="MagazineFulltextArticleBodySuffix"><section aria-labelledby="Bib1" data-title="References"><div id="Bib1-section"><h2 id="Bib1">References</h2><div id="Bib1-content"><div data-container-section="references"><ol data-track-component="outbound reference"><li data-counter="1."><p id="ref-CR1">Wynne-Jones, S. &amp; LaViolette, A. J. <i>The Swahili World</i> (Routledge, 2017).</p></li><li data-counter="2."><p id="ref-CR2">Kusimba, C. M. <i>The Rise and Fall of Swahili States</i> (Altamira Press, 1999).</p></li><li data-counter="3."><p id="ref-CR3">Spear, T. T. The Shirazi in Swahili traditions, culture and history. <i>Hist. Afr.</i> <b>11</b>, 291–305 (1984).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.2307/3171638" data-track-action="article reference" href="https://doi.org/10.2307%2F3171638" aria-label="Article reference 3" data-doi="10.2307/3171638">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 3" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20Shirazi%20in%20Swahili%20traditions%2C%20culture%20and%20history&amp;journal=Hist.%20Afr.&amp;doi=10.2307%2F3171638&amp;volume=11&amp;pages=291-305&amp;publication_year=1984&amp;author=Spear%2CTT">
                    Google Scholar</a> 
                </p></li><li data-counter="4."><p id="ref-CR4">Sheriff, A. <i>Slaves, Spices and Ivory in Zanzibar. Integration of an East African Commercial Empire into the World Economy, 1770–1873</i> (James Currey, 1987).</p></li><li data-counter="5."><p id="ref-CR5">Spear, T. Early Swahili history reconsidered. <i>Int. J. Afr. Hist. Stud.</i> <b>33</b>, 257–290 (2000).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.2307/220649" data-track-action="article reference" href="https://doi.org/10.2307%2F220649" aria-label="Article reference 5" data-doi="10.2307/220649">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 5" href="http://scholar.google.com/scholar_lookup?&amp;title=Early%20Swahili%20history%20reconsidered&amp;journal=Int.%20J.%20Afr.%20Hist.%20Stud.&amp;doi=10.2307%2F220649&amp;volume=33&amp;pages=257-290&amp;publication_year=2000&amp;author=Spear%2CT">
                    Google Scholar</a> 
                </p></li><li data-counter="6."><p id="ref-CR6">Kusimba, C. M. in <i>State Formations: Global Histories and Cultures of Statehood</i> (eds Anderson, G., Brooke, J. L. &amp; Strauss, J. C.) 90–107 (Cambridge Univ. Press, 2018).</p></li><li data-counter="7."><p id="ref-CR7">Fleisher, J. &amp; Wynne-Jones, S. Ceramics and the early Swahili: deconstructing the early Tana tradition. <i>Afr. Archaeol. Rev.</i> <b>28</b>, 245–278 (2011).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1007/s10437-011-9104-6" data-track-action="article reference" href="https://doi.org/10.1007%2Fs10437-011-9104-6" aria-label="Article reference 7" data-doi="10.1007/s10437-011-9104-6">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 7" href="http://scholar.google.com/scholar_lookup?&amp;title=Ceramics%20and%20the%20early%20Swahili%3A%20deconstructing%20the%20early%20Tana%20tradition&amp;journal=Afr.%20Archaeol.%20Rev.&amp;doi=10.1007%2Fs10437-011-9104-6&amp;volume=28&amp;pages=245-278&amp;publication_year=2011&amp;author=Fleisher%2CJ&amp;author=Wynne-Jones%2CS">
                    Google Scholar</a> 
                </p></li><li data-counter="8."><p id="ref-CR8">Pollard, E. &amp; Kinyera, O. C. The Swahili Coast and the Indian Ocean trade patterns in the 7th–10th centuries <span>ce</span>. <i>J. S. Afr. Stud.</i> <b>43</b>, 927–947 (2017).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1080/03057070.2017.1345266" data-track-action="article reference" href="https://doi.org/10.1080%2F03057070.2017.1345266" aria-label="Article reference 8" data-doi="10.1080/03057070.2017.1345266">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 8" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20Swahili%20Coast%20and%20the%20Indian%20Ocean%20trade%20patterns%20in%20the%207th%E2%80%9310th%20centuries%20ce&amp;journal=J.%20S.%20Afr.%20Stud.&amp;doi=10.1080%2F03057070.2017.1345266&amp;volume=43&amp;pages=927-947&amp;publication_year=2017&amp;author=Pollard%2CE&amp;author=Kinyera%2COC">
                    Google Scholar</a> 
                </p></li><li data-counter="9."><p id="ref-CR9">Horton, M. C. <i>Shanga: An Early Muslim Trading Community on the Coast of Eastern Africa</i>, Vol. 14. (British Institute in Eastern Africa, 1996).</p></li><li data-counter="10."><p id="ref-CR10">Fleisher, J. et al. When did the Swahili become maritime? <i>Am. Anthropol.</i> <b>117</b>, 100–115 (2015).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1111/aman.12171" data-track-action="article reference" href="https://doi.org/10.1111%2Faman.12171" aria-label="Article reference 10" data-doi="10.1111/aman.12171">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25821235" aria-label="PubMed reference 10">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4368416" aria-label="PubMed Central reference 10">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 10" href="http://scholar.google.com/scholar_lookup?&amp;title=When%20did%20the%20Swahili%20become%20maritime%3F&amp;journal=Am.%20Anthropol.&amp;doi=10.1111%2Faman.12171&amp;volume=117&amp;pages=100-115&amp;publication_year=2015&amp;author=Fleisher%2CJ">
                    Google Scholar</a> 
                </p></li><li data-counter="11."><p id="ref-CR11">Wynne-Jones, S. <i>A Material Culture: Consumption and Materiality on the Coast of Precolonial East Africa</i> (Oxford Univ. Press, 2016).</p></li><li data-counter="12."><p id="ref-CR12">Vernet, T. Le territoire hors les murs des cités-états Swahili de l’Archipel de Lamu, 1600–1800. <i>J. Afr.</i> <b>74</b>, 381–411 (2004).</p><p><a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 12" href="http://scholar.google.com/scholar_lookup?&amp;title=Le%20territoire%20hors%20les%20murs%20des%20cit%C3%A9s-%C3%A9tats%20Swahili%20de%20l%E2%80%99Archipel%20de%20Lamu%2C%201600%E2%80%931800&amp;journal=J.%20Afr.&amp;volume=74&amp;pages=381-411&amp;publication_year=2004&amp;author=Vernet%2CT">
                    Google Scholar</a> 
                </p></li><li data-counter="13."><p id="ref-CR13">Glassman, J. <i>Feasts and Riot: Revelry, Rebellion, and Popular Consciousness on the Swahili Coast</i> 1856–1888 (James Currey, 1995).</p></li><li data-counter="14."><p id="ref-CR14">Caplan, P. ‘But the coast, of course, is quite different’: academic and local Ideas about the East African littoral. <i>J. East. Afr. Stud.</i> <b>1</b>, 305–320 (2007).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1080/17531050701452663" data-track-action="article reference" href="https://doi.org/10.1080%2F17531050701452663" aria-label="Article reference 14" data-doi="10.1080/17531050701452663">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2007gmee.book.....C" aria-label="ADS reference 14">ADS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 14" href="http://scholar.google.com/scholar_lookup?&amp;title=%E2%80%98But%20the%20coast%2C%20of%20course%2C%20is%20quite%20different%E2%80%99%3A%20academic%20and%20local%20Ideas%20about%20the%20East%20African%20littoral&amp;journal=J.%20East.%20Afr.%20Stud.&amp;doi=10.1080%2F17531050701452663&amp;volume=1&amp;pages=305-320&amp;publication_year=2007&amp;author=Caplan%2CP">
                    Google Scholar</a> 
                </p></li><li data-counter="15."><p id="ref-CR15">Freeman-Grenville, G. S. P. <i>The East African Coast. Select Documents from the First to the Earlier Nineteenth Centuries</i> (Clarendon Press, 1962).</p></li><li data-counter="16."><p id="ref-CR16">Chittick, H. N. in <i>East Africa and the Orient: Cultural Syntheses in Pre-Colonial Times</i> eds Chittick, H. N. &amp; Rotberg, R. I.) 16–43 (Africana, 1975).</p></li><li data-counter="17."><p id="ref-CR17">Pouwels, R. L.<i> Horn and Crescent. Cultural change and traditional Islam on the East African coast,</i> 800–1900. <i>African Studies Series</i>, Vol. 53. (Cambridge Univ. Press, 1987).</p></li><li data-counter="18."><p id="ref-CR18">Saad, E. Kilwa dynastic historiography: a critical study. <i>Hist. Afr.</i> <b>6</b>, 177–209 (1979).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.2307/3171745" data-track-action="article reference" href="https://doi.org/10.2307%2F3171745" aria-label="Article reference 18" data-doi="10.2307/3171745">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 18" href="http://scholar.google.com/scholar_lookup?&amp;title=Kilwa%20dynastic%20historiography%3A%20a%20critical%20study&amp;journal=Hist.%20Afr.&amp;doi=10.2307%2F3171745&amp;volume=6&amp;pages=177-209&amp;publication_year=1979&amp;author=Saad%2CE">
                    Google Scholar</a> 
                </p></li><li data-counter="19."><p id="ref-CR19">Crowther, A. et al. Subsistence mosaics, forager-farmer interactions, and the transition to food production in eastern Africa. <i>Quat. Int.</i> <b>489</b>, 101–120 (2018).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.quaint.2017.01.014" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.quaint.2017.01.014" aria-label="Article reference 19" data-doi="10.1016/j.quaint.2017.01.014">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 19" href="http://scholar.google.com/scholar_lookup?&amp;title=Subsistence%20mosaics%2C%20forager-farmer%20interactions%2C%20and%20the%20transition%20to%20food%20production%20in%20eastern%20Africa&amp;journal=Quat.%20Int.&amp;doi=10.1016%2Fj.quaint.2017.01.014&amp;volume=489&amp;pages=101-120&amp;publication_year=2018&amp;author=Crowther%2CA">
                    Google Scholar</a> 
                </p></li><li data-counter="20."><p id="ref-CR20">Nurse, D. &amp; Spear, T. <i>The Swahili: Reconstructing the History and Language of an African Society,</i> 800–1500 (Univ. of Pennsylvania Press, 1985).</p></li><li data-counter="21."><p id="ref-CR21">Raaum, R. L. et al. in <i>The Swahili World</i> (eds Wynne-Jones, S. &amp; LaViolette, A. J.) 81–102 (Routledge, 2017).</p></li><li data-counter="22."><p id="ref-CR22">Kusimba, C. M., Kusimba, S. B. &amp; Dussubieux, L. Beyond the coastalscapes: preindustrial social and political networks in East Africa. <i>Afr. Archaeol. Rev.</i> <b>30</b>, 399–426 (2013).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1007/s10437-013-9133-4" data-track-action="article reference" href="https://doi.org/10.1007%2Fs10437-013-9133-4" aria-label="Article reference 22" data-doi="10.1007/s10437-013-9133-4">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 22" href="http://scholar.google.com/scholar_lookup?&amp;title=Beyond%20the%20coastalscapes%3A%20preindustrial%20social%20and%20political%20networks%20in%20East%20Africa&amp;journal=Afr.%20Archaeol.%20Rev.&amp;doi=10.1007%2Fs10437-013-9133-4&amp;volume=30&amp;pages=399-426&amp;publication_year=2013&amp;author=Kusimba%2CCM&amp;author=Kusimba%2CSB&amp;author=Dussubieux%2CL">
                    Google Scholar</a> 
                </p></li><li data-counter="23."><p id="ref-CR23">Skoglund, P. et al. Reconstructing prehistoric African population structure. <i>Cell</i> <b>171</b>, 59–71.e21 (2017).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.cell.2017.08.049" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.cell.2017.08.049" aria-label="Article reference 23" data-doi="10.1016/j.cell.2017.08.049">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://medium.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhsFGgurnI" aria-label="CAS reference 23">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28938123" aria-label="PubMed reference 23">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5679310" aria-label="PubMed Central reference 23">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 23" href="http://scholar.google.com/scholar_lookup?&amp;title=Reconstructing%20prehistoric%20African%20population%20structure&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2017.08.049&amp;volume=171&amp;pages=59-71.e21&amp;publication_year=2017&amp;author=Skoglund%2CP">
                    Google Scholar</a> 
                </p></li><li data-counter="24."><p id="ref-CR24">Prendergast Mary, E. et al. Ancient DNA reveals a multistep spread of the first herders into sub-Saharan Africa. <i>Science</i> <b>365</b>, eaaw6275 (2019).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1126/science.aaw6275" data-track-action="article reference" href="https://doi.org/10.1126%2Fscience.aaw6275" aria-label="Article reference 24" data-doi="10.1126/science.aaw6275">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2019Sci...365.6275P" aria-label="ADS reference 24">ADS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31147405" aria-label="PubMed reference 24">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6827346" aria-label="PubMed Central reference 24">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 24" href="http://scholar.google.com/scholar_lookup?&amp;title=Ancient%20DNA%20reveals%20a%20multistep%20spread%20of%20the%20first%20herders%20into%20sub-Saharan%20Africa&amp;journal=Science&amp;doi=10.1126%2Fscience.aaw6275&amp;volume=365&amp;publication_year=2019&amp;author=Prendergast%20Mary%2CE">
                    Google Scholar</a> 
                </p></li><li data-counter="25."><p id="ref-CR25">Mathieson, I. &amp; Scally, A. What is ancestry? <i>PLoS Genet.</i> <b>16</b>, e1008624 (2020).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1371/journal.pgen.1008624" data-track-action="article reference" href="https://doi.org/10.1371%2Fjournal.pgen.1008624" aria-label="Article reference 25" data-doi="10.1371/journal.pgen.1008624">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://medium.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXpsFSht7g%3D" aria-label="CAS reference 25">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32150538" aria-label="PubMed reference 25">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7082057" aria-label="PubMed Central reference 25">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 25" href="http://scholar.google.com/scholar_lookup?&amp;title=What%20is%20ancestry%3F&amp;journal=PLoS%20Genet.&amp;doi=10.1371%2Fjournal.pgen.1008624&amp;volume=16&amp;publication_year=2020&amp;author=Mathieson%2CI&amp;author=Scally%2CA">
                    Google Scholar</a> 
                </p></li><li data-counter="26."><p id="ref-CR26">Patterson, N. et al. Ancient admixture in human history. <i>Genetics</i> <b>192</b>, 1065–1093 (2012).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1534/genetics.112.145037" data-track-action="article reference" href="https://doi.org/10.1534%2Fgenetics.112.145037" aria-label="Article reference 26" data-doi="10.1534/genetics.112.145037">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22960212" aria-label="PubMed reference 26">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3522152" aria-label="PubMed Central reference 26">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 26" href="http://scholar.google.com/scholar_lookup?&amp;title=Ancient%20admixture%20in%20human%20history&amp;journal=Genetics&amp;doi=10.1534%2Fgenetics.112.145037&amp;volume=192&amp;pages=1065-1093&amp;publication_year=2012&amp;author=Patterson%2CN">
                    Google Scholar</a> 
                </p></li><li data-counter="27."><p id="ref-CR27">Quintana-Murci, L. et al. Where West meets East: the complex mtDNA landscape of the Southwest and Central Asian Corridor. <i>Am. J. Hum. Genet.</i> <b>74</b>, 827–845 (2004).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1086/383236" data-track-action="article reference" href="https://doi.org/10.1086%2F383236" aria-label="Article reference 27" data-doi="10.1086/383236">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://medium.com/articles/cas-redirect/1:CAS:528:DC%2BD2cXjvVeis7o%3D" aria-label="CAS reference 27">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15077202" aria-label="PubMed reference 27">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1181978" aria-label="PubMed Central reference 27">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 27" href="http://scholar.google.com/scholar_lookup?&amp;title=Where%20West%20meets%20East%3A%20the%20complex%20mtDNA%20landscape%20of%20the%20Southwest%20and%20Central%20Asian%20Corridor&amp;journal=Am.%20J.%20Hum.%20Genet.&amp;doi=10.1086%2F383236&amp;volume=74&amp;pages=827-845&amp;publication_year=2004&amp;author=Quintana-Murci%2CL">
                    Google Scholar</a> 
                </p></li><li data-counter="28."><p id="ref-CR28">Gandini, F. et al. Mapping human dispersals into the Horn of Africa from Arabian Ice Age refugia using mitogenomes. <i>Sci. Rep.</i> <b>6</b>, 25472 (2016).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/srep25472" data-track-action="article reference" href="https://doi.org/10.1038%2Fsrep25472" aria-label="Article reference 28" data-doi="10.1038/srep25472">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2016NatSR...625472G" aria-label="ADS reference 28">ADS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://medium.com/articles/cas-redirect/1:CAS:528:DC%2BC28XnsVeltLw%3D" aria-label="CAS reference 28">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27146119" aria-label="PubMed reference 28">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4857117" aria-label="PubMed Central reference 28">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 28" href="http://scholar.google.com/scholar_lookup?&amp;title=Mapping%20human%20dispersals%20into%20the%20Horn%20of%20Africa%20from%20Arabian%20Ice%20Age%20refugia%20using%20mitogenomes&amp;journal=Sci.%20Rep.&amp;doi=10.1038%2Fsrep25472&amp;volume=6&amp;publication_year=2016&amp;author=Gandini%2CF">
                    Google Scholar</a> 
                </p></li><li data-counter="29."><p id="ref-CR29">Cadenas, A. M. et al. Y-chromosome diversity characterizes the Gulf of Oman. <i>Eur. J. Hum. Genet.</i> <b>16</b>, 374–386 (2008).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/sj.ejhg.5201934" data-track-action="article reference" href="https://doi.org/10.1038%2Fsj.ejhg.5201934" aria-label="Article reference 29" data-doi="10.1038/sj.ejhg.5201934">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://medium.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXisVSrtb4%3D" aria-label="CAS reference 29">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17928816" aria-label="PubMed reference 29">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 29" href="http://scholar.google.com/scholar_lookup?&amp;title=Y-chromosome%20diversity%20characterizes%20the%20Gulf%20of%20Oman&amp;journal=Eur.%20J.%20Hum.%20Genet.&amp;doi=10.1038%2Fsj.ejhg.5201934&amp;volume=16&amp;pages=374-386&amp;publication_year=2008&amp;author=Cadenas%2CAM">
                    Google Scholar</a> 
                </p></li><li data-counter="30."><p id="ref-CR30">Chintalapati, M., Patterson, N. &amp; Moorjani, P. The spatiotemporal patterns of major human admixture events during the European Holocene. <i>eLife</i> <b>11</b>, e77625 (2022).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.7554/eLife.77625" data-track-action="article reference" href="https://doi.org/10.7554%2FeLife.77625" aria-label="Article reference 30" data-doi="10.7554/eLife.77625">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=35635751" aria-label="PubMed reference 30">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9293011" aria-label="PubMed Central reference 30">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 30" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20spatiotemporal%20patterns%20of%20major%20human%20admixture%20events%20during%20the%20European%20Holocene&amp;journal=eLife&amp;doi=10.7554%2FeLife.77625&amp;volume=11&amp;publication_year=2022&amp;author=Chintalapati%2CM&amp;author=Patterson%2CN&amp;author=Moorjani%2CP">
                    Google Scholar</a> 
                </p></li><li data-counter="31."><p id="ref-CR31">Narasimhan Vagheesh, M. et al. The formation of human populations in South and Central Asia. <i>Science</i> <b>365</b>, eaat7487 (2019).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1126/science.aat7487" data-track-action="article reference" href="https://doi.org/10.1126%2Fscience.aat7487" aria-label="Article reference 31" data-doi="10.1126/science.aat7487">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31488661" aria-label="PubMed reference 31">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6822619" aria-label="PubMed Central reference 31">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 31" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20formation%20of%20human%20populations%20in%20South%20and%20Central%20Asia&amp;journal=Science&amp;doi=10.1126%2Fscience.aat7487&amp;volume=365&amp;publication_year=2019&amp;author=Narasimhan%20Vagheesh%2CM">
                    Google Scholar</a> 
                </p></li><li data-counter="32."><p id="ref-CR32">Brucato, N. et al. The Comoros show the earliest Austronesian gene flow into the Swahili Corridor. <i>Am. J. Hum. Genet.</i> <b>102</b>, 58–68 (2018).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.ajhg.2017.11.011" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.ajhg.2017.11.011" aria-label="Article reference 32" data-doi="10.1016/j.ajhg.2017.11.011">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://medium.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXkslGluw%3D%3D" aria-label="CAS reference 32">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29304377" aria-label="PubMed reference 32">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5777450" aria-label="PubMed Central reference 32">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 32" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20Comoros%20show%20the%20earliest%20Austronesian%20gene%20flow%20into%20the%20Swahili%20Corridor&amp;journal=Am.%20J.%20Hum.%20Genet.&amp;doi=10.1016%2Fj.ajhg.2017.11.011&amp;volume=102&amp;pages=58-68&amp;publication_year=2018&amp;author=Brucato%2CN">
                    Google Scholar</a> 
                </p></li><li data-counter="33."><p id="ref-CR33">Horton, M., Olsen, J., Fleisher, J. &amp; Wynne-Jones, S. The chronology of Kilwa Kisiwani, <span>ad</span> 800–1500. <i>Afr. Archaeol. Rev.</i> <b>39</b>, 143–166 (2022).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1007/s10437-022-09476-8" data-track-action="article reference" href="https://doi.org/10.1007%2Fs10437-022-09476-8" aria-label="Article reference 33" data-doi="10.1007/s10437-022-09476-8">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 33" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20chronology%20of%20Kilwa%20Kisiwani%2C%20ad%20800%E2%80%931500&amp;journal=Afr.%20Archaeol.%20Rev.&amp;doi=10.1007%2Fs10437-022-09476-8&amp;volume=39&amp;pages=143-166&amp;publication_year=2022&amp;author=Horton%2CM&amp;author=Olsen%2CJ&amp;author=Fleisher%2CJ&amp;author=Wynne-Jones%2CS">
                    Google Scholar</a> 
                </p></li><li data-counter="34."><p id="ref-CR34">Lansing, J. S. et al. Kinship structures create persistent channels for language transmission. <i>Proc. Natl Acad. Sci. USA</i> <b>114</b>, 12910–12915 (2017).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1073/pnas.1706416114" data-track-action="article reference" href="https://doi.org/10.1073%2Fpnas.1706416114" aria-label="Article reference 34" data-doi="10.1073/pnas.1706416114">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2017PNAS..11412910L" aria-label="ADS reference 34">ADS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://medium.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhvVeitLjP" aria-label="CAS reference 34">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29158378" aria-label="PubMed reference 34">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5724253" aria-label="PubMed Central reference 34">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 34" href="http://scholar.google.com/scholar_lookup?&amp;title=Kinship%20structures%20create%20persistent%20channels%20for%20language%20transmission&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;doi=10.1073%2Fpnas.1706416114&amp;volume=114&amp;pages=12910-12915&amp;publication_year=2017&amp;author=Lansing%2CJS">
                    Google Scholar</a> 
                </p></li><li data-counter="35."><p id="ref-CR35">Haspelmath, M. &amp; Tadmor, U. <i>Loanwords in the World’s Languages : A Comparative Handbook</i> (De Gruyter Mouton, 2009).</p></li><li data-counter="36."><p id="ref-CR36">LaViolette, A. &amp; Fleisher, J. B. Developments in rural life on the Eastern African coast, <span>a.d</span>. 700–1500. <i>J. Field Archaeol.</i> <b>43</b>, 380–398 (2018).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1080/00934690.2018.1489661" data-track-action="article reference" href="https://doi.org/10.1080%2F00934690.2018.1489661" aria-label="Article reference 36" data-doi="10.1080/00934690.2018.1489661">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 36" href="http://scholar.google.com/scholar_lookup?&amp;title=Developments%20in%20rural%20life%20on%20the%20Eastern%20African%20coast%2C%20a.d.%20700%E2%80%931500&amp;journal=J.%20Field%20Archaeol.&amp;doi=10.1080%2F00934690.2018.1489661&amp;volume=43&amp;pages=380-398&amp;publication_year=2018&amp;author=LaViolette%2CA&amp;author=Fleisher%2CJB">
                    Google Scholar</a> 
                </p></li><li data-counter="37."><p id="ref-CR37">Dabney, J. et al. Complete mitochondrial genome sequence of a Middle Pleistocene cave bear reconstructed from ultrashort DNA fragments. <i>Proc. Natl Acad. Sci. USA</i> <b>110</b>, 15758–15763 (2013).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1073/pnas.1314445110" data-track-action="article reference" href="https://doi.org/10.1073%2Fpnas.1314445110" aria-label="Article reference 37" data-doi="10.1073/pnas.1314445110">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2013PNAS..11015758D" aria-label="ADS reference 37">ADS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://medium.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXhs1WlurnO" aria-label="CAS reference 37">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24019490" aria-label="PubMed reference 37">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3785785" aria-label="PubMed Central reference 37">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 37" href="http://scholar.google.com/scholar_lookup?&amp;title=Complete%20mitochondrial%20genome%20sequence%20of%20a%20Middle%20Pleistocene%20cave%20bear%20reconstructed%20from%20ultrashort%20DNA%20fragments&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;doi=10.1073%2Fpnas.1314445110&amp;volume=110&amp;pages=15758-15763&amp;publication_year=2013&amp;author=Dabney%2CJ">
                    Google Scholar</a> 
                </p></li><li data-counter="38."><p id="ref-CR38">Rohland, N. et al. Partial uracil-DNA-glycosylase treatment for screening of ancient DNA. <i>Philos. Trans. R. Soc. Lond. B</i> <b>370</b>, 20130624–20130624 (2015).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1098/rstb.2013.0624" data-track-action="article reference" href="https://doi.org/10.1098%2Frstb.2013.0624" aria-label="Article reference 38" data-doi="10.1098/rstb.2013.0624">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 38" href="http://scholar.google.com/scholar_lookup?&amp;title=Partial%20uracil-DNA-glycosylase%20treatment%20for%20screening%20of%20ancient%20DNA&amp;journal=Philos.%20Trans.%20R.%20Soc.%20Lond.%20B&amp;doi=10.1098%2Frstb.2013.0624&amp;volume=370&amp;pages=20130624-20130624&amp;publication_year=2015&amp;author=Rohland%2CN">
                    Google Scholar</a> 
                </p></li><li data-counter="39."><p id="ref-CR39">Fu, Q. et al. An early modern human from Romania with a recent Neanderthal ancestor. <i>Nature</i> <b>524</b>, 216–219 (2015).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/nature14558" data-track-action="article reference" href="https://doi.org/10.1038%2Fnature14558" aria-label="Article reference 39" data-doi="10.1038/nature14558">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2015Natur.524..216F" aria-label="ADS reference 39">ADS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://medium.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXhtFyltL7J" aria-label="CAS reference 39">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26098372" aria-label="PubMed reference 39">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4537386" aria-label="PubMed Central reference 39">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 39" href="http://scholar.google.com/scholar_lookup?&amp;title=An%20early%20modern%20human%20from%20Romania%20with%20a%20recent%20Neanderthal%20ancestor&amp;journal=Nature&amp;doi=10.1038%2Fnature14558&amp;volume=524&amp;pages=216-219&amp;publication_year=2015&amp;author=Fu%2CQ">
                    Google Scholar</a> 
                </p></li><li data-counter="40."><p id="ref-CR40">Maricic, T., Whitten, M. &amp; Pääbo, S. Multiplexed DNA sequence capture of mitochondrial genomes using PCR products. <i>PLoS ONE</i> <b>5</b>, e14004–e14004 (2010).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1371/journal.pone.0014004" data-track-action="article reference" href="https://doi.org/10.1371%2Fjournal.pone.0014004" aria-label="Article reference 40" data-doi="10.1371/journal.pone.0014004">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2010PLoSO...514004M" aria-label="ADS reference 40">ADS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21103372" aria-label="PubMed reference 40">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2982832" aria-label="PubMed Central reference 40">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 40" href="http://scholar.google.com/scholar_lookup?&amp;title=Multiplexed%20DNA%20sequence%20capture%20of%20mitochondrial%20genomes%20using%20PCR%20products&amp;journal=PLoS%20ONE&amp;doi=10.1371%2Fjournal.pone.0014004&amp;volume=5&amp;pages=e14004-e14004&amp;publication_year=2010&amp;author=Maricic%2CT&amp;author=Whitten%2CM&amp;author=P%C3%A4%C3%A4bo%2CS">
                    Google Scholar</a> 
                </p></li><li data-counter="41."><p id="ref-CR41">Li, H. &amp; Durbin, R. Fast and accurate long-read alignment with Burrows–Wheeler transform. <i>Bioinformatics</i> <b>26</b>, 589–595 (2010).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1093/bioinformatics/btp698" data-track-action="article reference" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtp698" aria-label="Article reference 41" data-doi="10.1093/bioinformatics/btp698">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20080505" aria-label="PubMed reference 41">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2828108" aria-label="PubMed Central reference 41">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 41" href="http://scholar.google.com/scholar_lookup?&amp;title=Fast%20and%20accurate%20long-read%20alignment%20with%20Burrows%E2%80%93Wheeler%20transform&amp;journal=Bioinformatics&amp;doi=10.1093%2Fbioinformatics%2Fbtp698&amp;volume=26&amp;pages=589-595&amp;publication_year=2010&amp;author=Li%2CH&amp;author=Durbin%2CR">
                    Google Scholar</a> 
                </p></li><li data-counter="42."><p id="ref-CR42">Behar, DoronM. et al. A “Copernican” reassessment of the human mitochondrial DNA tree from its root. <i>Am. J. Hum. Genet.</i> <b>90</b>, 675–684 (2012).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.ajhg.2012.03.002" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.ajhg.2012.03.002" aria-label="Article reference 42" data-doi="10.1016/j.ajhg.2012.03.002">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://medium.com/articles/cas-redirect/1:CAS:528:DC%2BC38XlsFGjsrw%3D" aria-label="CAS reference 42">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22482806" aria-label="PubMed reference 42">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3322232" aria-label="PubMed Central reference 42">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 42" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20%E2%80%9CCopernican%E2%80%9D%20reassessment%20of%20the%20human%20mitochondrial%20DNA%20tree%20from%20its%20root&amp;journal=Am.%20J.%20Hum.%20Genet.&amp;doi=10.1016%2Fj.ajhg.2012.03.002&amp;volume=90&amp;pages=675-684&amp;publication_year=2012&amp;author=Behar%2CDoronM">
                    Google Scholar</a> 
                </p></li><li data-counter="43."><p id="ref-CR43">Fu, Q. et al. DNA analysis of an early modern human from Tianyuan Cave, China. <i>Proc. Natl Acad. Sci. USA</i> <b>110</b>, 2223–2227 (2013).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1073/pnas.1221359110" data-track-action="article reference" href="https://doi.org/10.1073%2Fpnas.1221359110" aria-label="Article reference 43" data-doi="10.1073/pnas.1221359110">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2013PNAS..110.2223F" aria-label="ADS reference 43">ADS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://medium.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXjtVersLs%3D" aria-label="CAS reference 43">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23341637" aria-label="PubMed reference 43">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3568306" aria-label="PubMed Central reference 43">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 43" href="http://scholar.google.com/scholar_lookup?&amp;title=DNA%20analysis%20of%20an%20early%20modern%20human%20from%20Tianyuan%20Cave%2C%20China&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;doi=10.1073%2Fpnas.1221359110&amp;volume=110&amp;pages=2223-2227&amp;publication_year=2013&amp;author=Fu%2CQ">
                    Google Scholar</a> 
                </p></li><li data-counter="44."><p id="ref-CR44">Korneliussen, T. S., Albrechtsen, A. &amp; Nielsen, R. ANGSD: analysis of next generation sequencing data. <i>BMC Bioinformatics</i> <b>15</b>, 356 (2014).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1186/s12859-014-0356-4" data-track-action="article reference" href="https://doi.org/10.1186%2Fs12859-014-0356-4" aria-label="Article reference 44" data-doi="10.1186/s12859-014-0356-4">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25420514" aria-label="PubMed reference 44">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4248462" aria-label="PubMed Central reference 44">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 44" href="http://scholar.google.com/scholar_lookup?&amp;title=ANGSD%3A%20analysis%20of%20next%20generation%20sequencing%20data&amp;journal=BMC%20Bioinformatics&amp;doi=10.1186%2Fs12859-014-0356-4&amp;volume=15&amp;publication_year=2014&amp;author=Korneliussen%2CTS&amp;author=Albrechtsen%2CA&amp;author=Nielsen%2CR">
                    Google Scholar</a> 
                </p></li><li data-counter="45."><p id="ref-CR45">Monroy Kuhn, J. M., Jakobsson, M. &amp; Günther, T. Estimating genetic kin relationships in prehistoric populations. <i>PLoS ONE</i> <b>13</b>, e0195491 (2018).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1371/journal.pone.0195491" data-track-action="article reference" href="https://doi.org/10.1371%2Fjournal.pone.0195491" aria-label="Article reference 45" data-doi="10.1371/journal.pone.0195491">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29684051" aria-label="PubMed reference 45">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5912749" aria-label="PubMed Central reference 45">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 45" href="http://scholar.google.com/scholar_lookup?&amp;title=Estimating%20genetic%20kin%20relationships%20in%20prehistoric%20populations&amp;journal=PLoS%20ONE&amp;doi=10.1371%2Fjournal.pone.0195491&amp;volume=13&amp;publication_year=2018&amp;author=Monroy%20Kuhn%2CJM&amp;author=Jakobsson%2CM&amp;author=G%C3%BCnther%2CT">
                    Google Scholar</a> 
                </p></li><li data-counter="46."><p id="ref-CR46">Reiff, R. E. et al. METTL23, a transcriptional partner of GABPA, is essential for human cognition. <i>Hum. Mol. Genet.</i> <b>23</b>, 3456–3466 (2014).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1093/hmg/ddu054" data-track-action="article reference" href="https://doi.org/10.1093%2Fhmg%2Fddu054" aria-label="Article reference 46" data-doi="10.1093/hmg/ddu054">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://medium.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXpvV2rsLw%3D" aria-label="CAS reference 46">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24501276" aria-label="PubMed reference 46">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4049305" aria-label="PubMed Central reference 46">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 46" href="http://scholar.google.com/scholar_lookup?&amp;title=METTL23%2C%20a%20transcriptional%20partner%20of%20GABPA%2C%20is%20essential%20for%20human%20cognition&amp;journal=Hum.%20Mol.%20Genet.&amp;doi=10.1093%2Fhmg%2Fddu054&amp;volume=23&amp;pages=3456-3466&amp;publication_year=2014&amp;author=Reiff%2CRE">
                    Google Scholar</a> 
                </p></li><li data-counter="47."><p id="ref-CR47">Patterson, N., Price, A. L. &amp; Reich, D. Population structure and eigenanalysis. <i>PLoS Genet.</i> <b>2</b>, e190 (2006).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1371/journal.pgen.0020190" data-track-action="article reference" href="https://doi.org/10.1371%2Fjournal.pgen.0020190" aria-label="Article reference 47" data-doi="10.1371/journal.pgen.0020190">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17194218" aria-label="PubMed reference 47">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1713260" aria-label="PubMed Central reference 47">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 47" href="http://scholar.google.com/scholar_lookup?&amp;title=Population%20structure%20and%20eigenanalysis&amp;journal=PLoS%20Genet.&amp;doi=10.1371%2Fjournal.pgen.0020190&amp;volume=2&amp;publication_year=2006&amp;author=Patterson%2CN&amp;author=Price%2CAL&amp;author=Reich%2CD">
                    Google Scholar</a> 
                </p></li><li data-counter="48."><p id="ref-CR48">Alexander, D. H., Novembre, J. &amp; Lange, K. Fast model-based estimation of ancestry in unrelated individuals. <i>Genome Res.</i> <b>19</b>, 1655–1664 (2009).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1101/gr.094052.109" data-track-action="article reference" href="https://doi.org/10.1101%2Fgr.094052.109" aria-label="Article reference 48" data-doi="10.1101/gr.094052.109">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://medium.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXhtFCjsLvL" aria-label="CAS reference 48">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19648217" aria-label="PubMed reference 48">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2752134" aria-label="PubMed Central reference 48">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 48" href="http://scholar.google.com/scholar_lookup?&amp;title=Fast%20model-based%20estimation%20of%20ancestry%20in%20unrelated%20individuals&amp;journal=Genome%20Res.&amp;doi=10.1101%2Fgr.094052.109&amp;volume=19&amp;pages=1655-1664&amp;publication_year=2009&amp;author=Alexander%2CDH&amp;author=Novembre%2CJ&amp;author=Lange%2CK">
                    Google Scholar</a> 
                </p></li><li data-counter="49."><p id="ref-CR49">Chang, C. C. et al. Second-generation PLINK: rising to the challenge of larger and richer datasets. <i>GigaScience</i> <b>4</b>, 7 (2015).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1186/s13742-015-0047-8" data-track-action="article reference" href="https://doi.org/10.1186%2Fs13742-015-0047-8" aria-label="Article reference 49" data-doi="10.1186/s13742-015-0047-8">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25722852" aria-label="PubMed reference 49">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4342193" aria-label="PubMed Central reference 49">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 49" href="http://scholar.google.com/scholar_lookup?&amp;title=Second-generation%20PLINK%3A%20rising%20to%20the%20challenge%20of%20larger%20and%20richer%20datasets&amp;journal=GigaScience&amp;doi=10.1186%2Fs13742-015-0047-8&amp;volume=4&amp;publication_year=2015&amp;author=Chang%2CCC">
                    Google Scholar</a> 
                </p></li><li data-counter="50."><p id="ref-CR50">Weissensteiner, H. et al. HaploGrep 2: mitochondrial haplogroup classification in the era of high-throughput sequencing. <i>Nucleic Acids Res.</i> <b>44</b>, W58–W63 (2016).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1093/nar/gkw233" data-track-action="article reference" href="https://doi.org/10.1093%2Fnar%2Fgkw233" aria-label="Article reference 50" data-doi="10.1093/nar/gkw233">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://medium.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhtV2itr%2FL" aria-label="CAS reference 50">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27084951" aria-label="PubMed reference 50">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4987869" aria-label="PubMed Central reference 50">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 50" href="http://scholar.google.com/scholar_lookup?&amp;title=HaploGrep%202%3A%20mitochondrial%20haplogroup%20classification%20in%20the%20era%20of%20high-throughput%20sequencing&amp;journal=Nucleic%20Acids%20Res.&amp;doi=10.1093%2Fnar%2Fgkw233&amp;volume=44&amp;pages=W58-W63&amp;publication_year=2016&amp;author=Weissensteiner%2CH">
                    Google Scholar</a> 
                </p></li><li data-counter="51."><p id="ref-CR51">Lazaridis, I. et al. The genetic history of the Southern Arc: A bridge between West Asia and Europe. <i>Science</i> <b>377</b>, eabm4247 (2022).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1126/science.abm4247" data-track-action="article reference" href="https://doi.org/10.1126%2Fscience.abm4247" aria-label="Article reference 51" data-doi="10.1126/science.abm4247">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://medium.com/articles/cas-redirect/1:CAS:528:DC%2BB38Xit1yrtLjN" aria-label="CAS reference 51">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=36007055" aria-label="PubMed reference 51">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 51" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20genetic%20history%20of%20the%20Southern%20Arc%3A%20A%20bridge%20between%20West%20Asia%20and%20Europe&amp;journal=Science&amp;doi=10.1126%2Fscience.abm4247&amp;volume=377&amp;publication_year=2022&amp;author=Lazaridis%2CI">
                    Google Scholar</a> 
                </p></li><li data-counter="52."><p id="ref-CR52">Mathieson, I. et al. The genomic history of southeastern Europe. <i>Nature</i> <b>555</b>, 197–203 (2018).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/nature25778" data-track-action="article reference" href="https://doi.org/10.1038%2Fnature25778" aria-label="Article reference 52" data-doi="10.1038/nature25778">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2018Natur.555..197M" aria-label="ADS reference 52">ADS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://medium.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXjtFCksbc%3D" aria-label="CAS reference 52">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29466330" aria-label="PubMed reference 52">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6091220" aria-label="PubMed Central reference 52">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 52" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20genomic%20history%20of%20southeastern%20Europe&amp;journal=Nature&amp;doi=10.1038%2Fnature25778&amp;volume=555&amp;pages=197-203&amp;publication_year=2018&amp;author=Mathieson%2CI">
                    Google Scholar</a> 
                </p></li><li data-counter="53."><p id="ref-CR53">Moorjani, P. et al. A genetic method for dating ancient genomes provides a direct estimate of human generation interval in the last 45,000 years. <i>Proc. Natl Acad. Sci. USA</i> <b>113</b>, 5652 (2016).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1073/pnas.1514696113" data-track-action="article reference" href="https://doi.org/10.1073%2Fpnas.1514696113" aria-label="Article reference 53" data-doi="10.1073/pnas.1514696113">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2016PNAS..113.5652M" aria-label="ADS reference 53">ADS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://medium.com/articles/cas-redirect/1:CAS:528:DC%2BC28XntVChurc%3D" aria-label="CAS reference 53">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27140627" aria-label="PubMed reference 53">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4878468" aria-label="PubMed Central reference 53">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 53" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20genetic%20method%20for%20dating%20ancient%20genomes%20provides%20a%20direct%20estimate%20of%20human%20generation%20interval%20in%20the%20last%2045%2C000%20years&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;doi=10.1073%2Fpnas.1514696113&amp;volume=113&amp;publication_year=2016&amp;author=Moorjani%2CP">
                    Google Scholar</a> 
                </p></li><li data-counter="54."><p id="ref-CR54">Lazaridis, I. et al. Ancient human genomes suggest three ancestral populations for present-day Europeans. <i>Nature</i> <b>513</b>, 409–413 (2014).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/nature13673" data-track-action="article reference" href="https://doi.org/10.1038%2Fnature13673" aria-label="Article reference 54" data-doi="10.1038/nature13673">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2014Natur.513..409L" aria-label="ADS reference 54">ADS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://medium.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXhsFOlu7nL" aria-label="CAS reference 54">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25230663" aria-label="PubMed reference 54">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4170574" aria-label="PubMed Central reference 54">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 54" href="http://scholar.google.com/scholar_lookup?&amp;title=Ancient%20human%20genomes%20suggest%20three%20ancestral%20populations%20for%20present-day%20Europeans&amp;journal=Nature&amp;doi=10.1038%2Fnature13673&amp;volume=513&amp;pages=409-413&amp;publication_year=2014&amp;author=Lazaridis%2CI">
                    Google Scholar</a> 
                </p></li><li data-counter="55."><p id="ref-CR55">Lazaridis, I. et al. Genomic insights into the origin of farming in the ancient Near East. <i>Nature</i> <b>536</b>, 419–424 (2016).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/nature19310" data-track-action="article reference" href="https://doi.org/10.1038%2Fnature19310" aria-label="Article reference 55" data-doi="10.1038/nature19310">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2016Natur.536..419L" aria-label="ADS reference 55">ADS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://medium.com/articles/cas-redirect/1:CAS:528:DC%2BC28XhsVSrtbjF" aria-label="CAS reference 55">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27459054" aria-label="PubMed reference 55">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5003663" aria-label="PubMed Central reference 55">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 55" href="http://scholar.google.com/scholar_lookup?&amp;title=Genomic%20insights%20into%20the%20origin%20of%20farming%20in%20the%20ancient%20Near%20East&amp;journal=Nature&amp;doi=10.1038%2Fnature19310&amp;volume=536&amp;pages=419-424&amp;publication_year=2016&amp;author=Lazaridis%2CI">
                    Google Scholar</a> 
                </p></li><li data-counter="56."><p id="ref-CR56">Nakatsuka, N. et al. The promise of discovering population-specific disease-associated genes in South Asia. <i>Nat. Genet.</i> <b>49</b>, 1403–1407 (2017).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/ng.3917" data-track-action="article reference" href="https://doi.org/10.1038%2Fng.3917" aria-label="Article reference 56" data-doi="10.1038/ng.3917">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://medium.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhtFOru7vJ" aria-label="CAS reference 56">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28714977" aria-label="PubMed reference 56">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5675555" aria-label="PubMed Central reference 56">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 56" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20promise%20of%20discovering%20population-specific%20disease-associated%20genes%20in%20South%20Asia&amp;journal=Nat.%20Genet.&amp;doi=10.1038%2Fng.3917&amp;volume=49&amp;pages=1403-1407&amp;publication_year=2017&amp;author=Nakatsuka%2CN">
                    Google Scholar</a> 
                </p></li></ol><p><a data-track="click" data-track-action="download citation references" data-track-label="link" rel="nofollow" href="https://citation-needed.springer.com/v2/references/10.1038/s41586-023-05754-w?format=refman&amp;flavour=references">Download references<svg width="16" height="16" focusable="false" role="img" aria-hidden="true"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#global-icon-download"></use></svg></a></p></div></div></div></section></div><section data-title="Acknowledgements"><div id="Ack1-section"><h2 id="Ack1">Acknowledgements</h2><p>We thank N. Adamski, R. Bernardos, B. Culleton, I. Greenslade, D. Kennett, M. Mah, A. Micco and Z. Zhang for contributions to data generation, processing and curation. E.S.B. was supported by EMBO Postdoctoral fellowship ALTF 242-2021. The work of C.M.K., S.B.K., S.R.W. and J.M. in Kenya was supported by the National Museums of Kenya and the Republic of Kenya. The bulk of the research at Manda, Mtwapa, Faza and Makwasinyi was carried out when C.M.K. was at the Field Museum of Natural History. We acknowledge financial support from the US National Science Foundation SBR 9024683 (1991–1993); BCS 9615291 (1996–1998); BCS 0106664 (2002–2004); BCS 0352681 (2003–2004); BCS 0648762 (2007–2009; BCS 1030081 (2010–2012), the US National Endowment for the Humanities (2012–2014), the US IIE J. W. Fulbright Sr Scholars Program 2002–2003; 2012), and the National Geographic Society (1996–1997). The research of S.W.-J. and J.F. at Songo Mnara was supported by the Antiquities Division of the Ministry of Natural Resources and Tourism, Tanzania, and also by the National Science Foundation (BCS 1123091) and the Arts and Humanities Research Council (AH/J502716/1). The Madagascar modern sample collection was supported by the MAGE consortium. D.R. is an Investigator of the Howard Hughes Medical Institute, and the ancient DNA laboratory work and analysis were also supported by National Institutes of Health grant HG012287, by John Templeton Foundation grant 61220, by the Allen Discovery Center program, which is a Paul G. Allen Frontiers Group advised program of the Paul G. Allen Family Foundation, and by a gift from J.-F. Clin. S.W.-J. and all co-authors are grateful to UK Research and Innovation for enabling fully Open Access publication of this study via financial support to the York Open Access Fund.</p></div></section><section aria-labelledby="author-information" data-title="Author information"><div id="author-information-section"><h2 id="author-information">Author information</h2><div id="author-information-content"><p><span id="author-notes">Author notes</span></p><ol><li id="na1"><p>These authors contributed equally: Jeffrey Fleisher and Stephanie Wynne-Jones</p></li><li id="na2"><p>These authors jointly supervised this work: David Reich and Chapurukha M. Kusimba</p></li></ol><h3 id="affiliations">Authors and Affiliations</h3><ol><li id="Aff1"><p>Department of Human Evolutionary Biology, Harvard University, Cambridge, MA, USA</p><p>Esther S. Brielle, Kendra Sirak, Nasreen Broomandkhoshbacht, Kim Callan, Elizabeth Curtis, Lora Iliev, Ann Marie Lawson, Jonas Oppenheimer, Kristin Stewardson, J. Noah Workman, Fatma Zalzala, Nick Patterson &amp; David Reich</p></li><li id="Aff2"><p>Department of Anthropology, Rice University, Houston, TX, USA</p><p>Jeffrey Fleisher &amp; Mary E. Prendergast</p></li><li id="Aff3"><p>Department of Archaeology, University of York, York, UK</p><p>Stephanie Wynne-Jones</p></li><li id="Aff4"><p>University of South Africa, Pretoria, South Africa</p><p>Stephanie Wynne-Jones</p></li><li id="Aff5"><p>Department of Genetics, Harvard Medical School, Boston, MA, USA</p><p>Kendra Sirak, Nasreen Broomandkhoshbacht, Kim Callan, Elizabeth Curtis, Lora Iliev, Ann Marie Lawson, Jonas Oppenheimer, Lijun Qiu, Kristin Stewardson, J. Noah Workman, Fatma Zalzala, Swapan Mallick, Nadin Rohland, Mary E. Prendergast &amp; David Reich</p></li><li id="Aff6"><p>Howard Hughes Medical Institute, Harvard Medical School, Boston, MA, USA</p><p>Nasreen Broomandkhoshbacht, Kim Callan, Elizabeth Curtis, Lora Iliev, Ann Marie Lawson, Lijun Qiu, Kristin Stewardson, Fatma Zalzala, Swapan Mallick &amp; David Reich</p></li><li id="Aff7"><p>Jaramogi Oginga Odinga University of Science and Technology, Bondo, Kenya</p><p>George Ayodo</p></li><li id="Aff8"><p>National Museums of Tanzania, Dar es Salaam, Tanzania</p><p>Agness O. Gidna &amp; Amandus Kwekason</p></li><li id="Aff9"><p>Department of Archaeology, National Museums of Kenya, Nairobi, Kenya</p><p>Angela Kabiru &amp; Chapurukha M. Kusimba</p></li><li id="Aff10"><p>British Institute of Eastern Africa, Nairobi, Kenya</p><p>Angela Kabiru</p></li><li id="Aff11"><p>Department of Archaeology and Heritage Studies, University of Dar es Salaam, Dar es Salaam, Tanzania</p><p>Audax Z. P. Mabulla</p></li><li id="Aff12"><p>Department of Earth Sciences, National Museums of Kenya, Nairobi, Kenya</p><p>Fredrick K. Manthi, Emmanuel Ndiema &amp; Christine Ogola</p></li><li id="Aff13"><p>Cleveland Museum of Natural History, Cleveland, OH, USA</p><p>Elizabeth Sawchuk</p></li><li id="Aff14"><p>Department of Anthropology, University of Alberta, Edmonton, Alberta, Canada</p><p>Elizabeth Sawchuk</p></li><li id="Aff15"><p>Department of Anthropology, Stony Brook University, Stony Brook, NY, USA</p><p>Elizabeth Sawchuk</p></li><li id="Aff16"><p>Department of Genetics and Genomics, College of Medicine and Health Sciences, United Arab Emirates University, Al-Ain, United Arab Emirates</p><p>Lihadh Al-Gazali, Bassam R. Ali &amp; Salma Ben-Salem</p></li><li id="Aff17"><p>Laboratoire Evolution et Santé Orale, Faculté de Chirurgie Dentaire, Université Toulouse III—Paul Sabatier, Toulouse, France</p><p>Thierry Letellier &amp; Denis Pierron</p></li><li id="Aff18"><p>Institut de Civilisations/Musée d’Art et d’Archéologie, Université d’Antananarivo, Antananarivo, Madagascar</p><p>Chantal Radimilahy &amp; Jean-Aimé Rakotoarisoa</p></li><li id="Aff19"><p>Department of Anthropology, Lehman College and The Graduate Center, The City University of New York, New York, NY, USA</p><p>Ryan L. Raaum</p></li><li id="Aff20"><p>The New York Consortium in Evolutionary Primatology, New York, NY, USA</p><p>Ryan L. Raaum</p></li><li id="Aff21"><p>Institutes of Energy and the Environment, The Pennsylvania State University, University Park, PA, USA</p><p>Brendan J. Culleton</p></li><li id="Aff22"><p>Broad Institute of Harvard and MIT, Cambridge, MA, USA</p><p>Swapan Mallick, Nick Patterson &amp; David Reich</p></li><li id="Aff23"><p>National Museums of Kenya, Lamu Museums, Lamu, Kenya</p><p>Mohammed Ali Mwenje</p></li><li id="Aff24"><p>Coastal Archaeology, Gede National Monument, Gede, Kenya</p><p>Khalfan Bini Ahmed</p></li><li id="Aff25"><p>Coastal Archaeology, Fort Jesus Museum, Mombasa, Kenya</p><p>Mohamed Mchulla Mohamed</p></li><li id="Aff26"><p>Department of Anthropology, University of Illinois at Chicago, Chicago, IL, USA</p><p>Sloan R. Williams</p></li><li id="Aff27"><p>University of Pennsylvania Museum of Archaeology and Anthropology, Philadelphia, PA, USA</p><p>Janet Monge</p></li><li id="Aff28"><p>Department of Anthropology, University of South Florida, Tampa, FL, USA</p><p>Sibel Kusimba &amp; Chapurukha M. Kusimba</p></li><li id="Aff29"><p>Institute of African Studies, University of Nairobi, Museum Hill, Nairobi, Kenya</p><p>Chapurukha M. Kusimba</p></li></ol><h3 id="contributions">Contributions</h3><p>E.S.B., J.F., S.W.-J., S.R.W., J.M., M.E.P., D.R. and C.M.K. conceptualized the study. E.S.B., K.S. and S.M. formally analysed the genetic data. E.S.B., N.B., K.C., E.C., L.I., A.M.L., J.O., L.Q., K.S., J.N.W., F.Z., B.J.C., S.M., N.R., N.P., K.B.A., M.M.M., S.K. and D.R. were involved in the genetic data investigation. J.F., S.W.-J., J.O., G.A., A.O.G., A. Kabiru, A. Kwekason, A.Z.P.M., F.K.M., E.N., C.O., E.S., L.A.-G., B.R.A., S.B.-S., T.L., D.P., C.R., J.-A.R., R.L.R., B.J.C., M.A.M., K.B.A., M.M.M., S.R.W., J.M., S.K., M.E.P., D.R. and C.M.K. provided resources. E.S.B., J.F., S.W.-J., M.E.P., D.R. and C.M.K. wrote the paper.</p><h3 id="corresponding-author">Corresponding authors</h3><p id="corresponding-author-list">Correspondence to
                <a id="corresp-c1" href="mailto:ebrielle@fas.harvard.edu">Esther S. Brielle</a>, <a id="corresp-c2" href="mailto:jfleisher@rice.edu">Jeffrey Fleisher</a>, <a id="corresp-c3" href="mailto:stephanie.wynne-jones@york.ac.uk">Stephanie Wynne-Jones</a>, <a id="corresp-c4" href="mailto:reich@genetics.med.harvard.edu">David Reich</a> or <a id="corresp-c5" href="mailto:ckusimba@usf.edu">Chapurukha M. Kusimba</a>.</p></div></div></section><section data-title="Ethics declarations"><div id="ethics-section"><h2 id="ethics">Ethics declarations</h2><div id="ethics-content">
              
                <h3 id="FPar2">Competing interests</h3>
                <p>The authors declare no competing interests.</p>
              
            </div></div></section><section data-title="Peer review"><div id="peer-review-section"><h2 id="peer-review">Peer review</h2><div id="peer-review-content">
              
              
                <h3 id="FPar1">Peer review information</h3>
                <p><i>Nature</i> thanks Annalisa Christie, Diyendo Massilani and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Peer review reports are available.</p>
              
            </div></div></section><section data-title="Additional information"><div id="additional-information-section"><h2 id="additional-information">Additional information</h2><p><b>Publisher’s note</b> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div></section><section data-title="Extended data figures and tables"><div id="Sec24-section"><h2 id="Sec24">Extended data figures and tables</h2><div id="Sec24-content"><div data-test="supplementary-info"><div data-test="supp-item" id="Fig4"><h3><a data-track="click" data-track-action="view supplementary info" data-track-label="link" data-test="supp-info-link" href="https://medium.com/articles/s41586-023-05754-w/figures/4" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41586-023-05754-w/MediaObjects/41586_2023_5754_Fig4_ESM.jpg">Extended Data Fig. 1 ADMIXTURE analysis.</a></h3><p>We analyzed 6422 individuals including African populations genotyped with the Human Origins SNP array, along with 1240K capture ancient individuals from present-day Kenya, present-day Tanzania, and surrounding areas; Swahili coast present-day samples;<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 21" title="Raaum, R. L. et al. in The Swahili World (eds Wynne-Jones, S. &amp; LaViolette, A. J.) 81–102 (Routledge, 2017)." href="https://medium.com/articles/s41586-023-05754-w#ref-CR21" id="ref-link-section-d194324060e3869">21</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 32" title="Brucato, N. et al. The Comoros show the earliest Austronesian gene flow into the Swahili Corridor. Am. J. Hum. Genet. 102, 58–68 (2018)." href="https://medium.com/articles/s41586-023-05754-w#ref-CR32" id="ref-link-section-d194324060e3872">32</a></sup> ancient Israel_Natufian; and Pulliyar samples from present-day India. We chose K = 10 ancestral reference populations (chosen based on low cross-validation errors, high log-likelihood scores, and a low number of reference populations so as to not overfit) to highlight various ancestries found throughout Eastern Africa). We were able to discern a variety of African pastoral, farming, and forager ancestries within coastal and inland Swahili-speaking populations of the Human Origins dataset. The Makwasinyi reference ancestry composition resemble those of the present-day coastal populations sampled in a previously published dataset<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 32" title="Brucato, N. et al. The Comoros show the earliest Austronesian gene flow into the Swahili Corridor. Am. J. Hum. Genet. 102, 58–68 (2018)." href="https://medium.com/articles/s41586-023-05754-w#ref-CR32" id="ref-link-section-d194324060e3876">32</a></sup>, albeit with different proportions. The x’s identify two outlier individuals from the dataset with more Asian admixture; we removed them for population-level modeling. The genome-wide ancestry composition of the ancient coastal samples more closely resembles that of the newly published present-day individuals for which haplogroup data was published in<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 21" title="Raaum, R. L. et al. in The Swahili World (eds Wynne-Jones, S. &amp; LaViolette, A. J.) 81–102 (Routledge, 2017)." href="https://medium.com/articles/s41586-023-05754-w#ref-CR21" id="ref-link-section-d194324060e3880">21</a></sup>.</p></div><div data-test="supp-item" id="Fig5"><h3><a data-track="click" data-track-action="view supplementary info" data-track-label="link" data-test="supp-info-link" href="https://medium.com/articles/s41586-023-05754-w/figures/5" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41586-023-05754-w/MediaObjects/41586_2023_5754_Fig5_ESM.jpg">Extended Data Fig. 2 Relationship of medieval to recent East Africans.</a></h3><p>A) Graphical representation of individual and population level <i>qpAdm</i> models for Makwasinyi individuals, 89 present-day people who identify as and speak Swahili<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 32" title="Brucato, N. et al. The Comoros show the earliest Austronesian gene flow into the Swahili Corridor. Am. J. Hum. Genet. 102, 58–68 (2018)." href="https://medium.com/articles/s41586-023-05754-w#ref-CR32" id="ref-link-section-d194324060e3911">32</a></sup>, and 93 present-day people who identify as Swahili and have long familial residency in the town, sampled from 6 coastal regions along Kenya<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 21" title="Raaum, R. L. et al. in The Swahili World (eds Wynne-Jones, S. &amp; LaViolette, A. J.) 81–102 (Routledge, 2017)." href="https://medium.com/articles/s41586-023-05754-w#ref-CR21" id="ref-link-section-d194324060e3915">21</a></sup>. The green ancestry component represents the modeled contribution from medieval Swahili coast people, as proxied by individuals buried in Kilwa or Manda. One standard error bars around the mean are computed using a Block Jackknife across all 5 centimorgan segments of the autosomes, and are meaningful even for single individuals as the genome reflects ancestral contributions of large numbers of ancestors. B) PCA with the same present-day Swahili-identifying people (asterisks) and other present and ancient people (filled circles)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 23" title="Skoglund, P. et al. Reconstructing prehistoric African population structure. Cell 171, 59–71.e21 (2017)." href="https://medium.com/articles/s41586-023-05754-w#ref-CR23" id="ref-link-section-d194324060e3919">23</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 54" title="Lazaridis, I. et al. Ancient human genomes suggest three ancestral populations for present-day Europeans. Nature 513, 409–413 (2014)." href="https://medium.com/articles/s41586-023-05754-w#ref-CR54" id="ref-link-section-d194324060e3922">54</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 55" title="Lazaridis, I. et al. Genomic insights into the origin of farming in the ancient Near East. Nature 536, 419–424 (2016)." href="https://medium.com/articles/s41586-023-05754-w#ref-CR55" id="ref-link-section-d194324060e3925">55</a></sup>, and ancient individuals from eastern Africa published in this study (open shapes)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 23" title="Skoglund, P. et al. Reconstructing prehistoric African population structure. Cell 171, 59–71.e21 (2017)." href="https://medium.com/articles/s41586-023-05754-w#ref-CR23" id="ref-link-section-d194324060e3929">23</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 24" title="Prendergast Mary, E. et al. Ancient DNA reveals a multistep spread of the first herders into sub-Saharan Africa. Science 365, eaaw6275 (2019)." href="https://medium.com/articles/s41586-023-05754-w#ref-CR24" id="ref-link-section-d194324060e3932">24</a></sup> projected onto the first two eigenvectors.</p></div><div data-test="supp-item" id="Fig6"><h3><a data-track="click" data-track-action="view supplementary info" data-track-label="link" data-test="supp-info-link" href="https://medium.com/articles/s41586-023-05754-w/figures/6" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41586-023-05754-w/MediaObjects/41586_2023_5754_Fig6_ESM.jpg">Extended Data Fig. 3 Estimates of dates of population mixture.</a></h3><p>Curves that show the exponential decay of linkage disequilibrium generated by mixture between two populations related differentially to the two sources as a function of the number of elapsed generations since the mixture event. Multiplying by a 28 ± 2 year-to-generation conversion factor<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 53" title="Moorjani, P. et al. A genetic method for dating ancient genomes provides a direct estimate of human generation interval in the last 45,000 years. Proc. Natl Acad. Sci. USA 113, 5652 (2016)." href="https://medium.com/articles/s41586-023-05754-w#ref-CR53" id="ref-link-section-d194324060e3960">53</a></sup> and subtracting from the average calibrated radiocarbon dates or the archaeologically estimated date of the ancient individuals gives estimates in calendar years. For present-day Swahili individuals, we subtract from the year 2015, when sampling occured.</p></div><div data-test="supp-item"><div data-test="inline-table" data-container-section="table" id="table-3"><figure><figcaption><b id="Tab3" data-test="table-caption">Extended Data Table 1 Ancient individuals with high coverage genome-wide data newly reported in this study</b></figcaption></figure></div></div><div data-test="supp-item"><div data-test="inline-table" data-container-section="table" id="table-4"><figure><figcaption><b id="Tab4" data-test="table-caption">Extended Data Table 2 Ancient individuals with high coverage genome-wide data newly reported in this study that have significant evidence of Persian and often Indian ancestry</b></figcaption></figure></div></div><div data-test="supp-item"><div data-test="inline-table" data-container-section="table" id="table-5"><figure><figcaption><b id="Tab5" data-test="table-caption">Extended Data Table 3 Individuals with high coverage genome-wide data that do not have evidence of Persian ancestry</b></figcaption></figure></div></div><div data-test="supp-item"><div data-test="inline-table" data-container-section="table" id="table-6"><figure><figcaption><b id="Tab6" data-test="table-caption">Extended Data Table 4 Proportions of ancestry for present-day Swahili-identified individuals newly genotyped for this study</b></figcaption></figure></div></div></div></div></div></section><section data-title="Supplementary information"><div id="Sec25-section"><h2 id="Sec25">Supplementary information</h2></div></section><section data-title="Rights and permissions"><div id="rightslink-section"><h2 id="rightslink">Rights and permissions</h2><div id="rightslink-content">
                <p><b>Open Access</b>  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>.</p>
              <p><a data-track="click" data-track-action="view rights and permissions" data-track-label="link" href="https://s100.copyright.com/AppDispatchServlet?title=Entwined%20African%20and%20Asian%20genetic%20roots%20of%20medieval%20peoples%20of%20the%20Swahili%20coast&amp;author=Esther%20S.%20Brielle%20et%20al&amp;contentID=10.1038%2Fs41586-023-05754-w&amp;copyright=The%20Author%28s%29&amp;publication=0028-0836&amp;publicationDate=2023-03-29&amp;publisherName=SpringerNature&amp;orderBeanReset=true&amp;oa=CC%20BY">Reprints and Permissions</a></p></div></div></section><section aria-labelledby="article-info" data-title="About this article"><div id="article-info-section"><h2 id="article-info">About this article</h2><div id="article-info-content"><div><p><a data-crossmark="10.1038/s41586-023-05754-w" target="_blank" rel="noopener" href="https://crossmark.crossref.org/dialog/?doi=10.1038/s41586-023-05754-w" data-track="click" data-track-action="Click Crossmark" data-track-label="link" data-test="crossmark"><img width="57" height="81" alt="Verify currency and authenticity via CrossMark" src="data:image/svg+xml;base64,<svg height="81" width="57" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="m17.35 35.45 21.3-14.2v-17.03h-21.3" fill="#989898"/><path d="m38.65 35.45-21.3-14.2v-17.03h21.3" fill="#747474"/><path d="m28 .5c-12.98 0-23.5 10.52-23.5 23.5s10.52 23.5 23.5 23.5 23.5-10.52 23.5-23.5c0-6.23-2.48-12.21-6.88-16.62-4.41-4.4-10.39-6.88-16.62-6.88zm0 41.25c-9.8 0-17.75-7.95-17.75-17.75s7.95-17.75 17.75-17.75 17.75 7.95 17.75 17.75c0 4.71-1.87 9.22-5.2 12.55s-7.84 5.2-12.55 5.2z" fill="#535353"/><path d="m41 36c-5.81 6.23-15.23 7.45-22.43 2.9-7.21-4.55-10.16-13.57-7.03-21.5l-4.92-3.11c-4.95 10.7-1.19 23.42 8.78 29.71 9.97 6.3 23.07 4.22 30.6-4.86z" fill="#9c9c9c"/><path d="m.2 58.45c0-.75.11-1.42.33-2.01s.52-1.09.91-1.5c.38-.41.83-.73 1.34-.94.51-.22 1.06-.32 1.65-.32.56 0 1.06.11 1.51.35.44.23.81.5 1.1.81l-.91 1.01c-.24-.24-.49-.42-.75-.56-.27-.13-.58-.2-.93-.2-.39 0-.73.08-1.05.23-.31.16-.58.37-.81.66-.23.28-.41.63-.53 1.04-.13.41-.19.88-.19 1.39 0 1.04.23 1.86.68 2.46.45.59 1.06.88 1.84.88.41 0 .77-.07 1.07-.23s.59-.39.85-.68l.91 1c-.38.43-.8.76-1.28.99-.47.22-1 .34-1.58.34-.59 0-1.13-.1-1.64-.31-.5-.2-.94-.51-1.31-.91-.38-.4-.67-.9-.88-1.48-.22-.59-.33-1.26-.33-2.02zm8.4-5.33h1.61v2.54l-.05 1.33c.29-.27.61-.51.96-.72s.76-.31 1.24-.31c.73 0 1.27.23 1.61.71.33.47.5 1.14.5 2.02v4.31h-1.61v-4.1c0-.57-.08-.97-.25-1.21-.17-.23-.45-.35-.83-.35-.3 0-.56.08-.79.22-.23.15-.49.36-.78.64v4.8h-1.61zm7.37 6.45c0-.56.09-1.06.26-1.51.18-.45.42-.83.71-1.14.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.36c.07.62.29 1.1.65 1.44.36.33.82.5 1.38.5.29 0 .57-.04.83-.13s.51-.21.76-.37l.55 1.01c-.33.21-.69.39-1.09.53-.41.14-.83.21-1.26.21-.48 0-.92-.08-1.34-.25-.41-.16-.76-.4-1.07-.7-.31-.31-.55-.69-.72-1.13-.18-.44-.26-.95-.26-1.52zm4.6-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.07.45-.31.29-.5.73-.58 1.3zm2.5.62c0-.57.09-1.08.28-1.53.18-.44.43-.82.75-1.13s.69-.54 1.1-.71c.42-.16.85-.24 1.31-.24.45 0 .84.08 1.17.23s.61.34.85.57l-.77 1.02c-.19-.16-.38-.28-.56-.37-.19-.09-.39-.14-.61-.14-.56 0-1.01.21-1.35.63-.35.41-.52.97-.52 1.67 0 .69.17 1.24.51 1.66.34.41.78.62 1.32.62.28 0 .54-.06.78-.17.24-.12.45-.26.64-.42l.67 1.03c-.33.29-.69.51-1.08.65-.39.15-.78.23-1.18.23-.46 0-.9-.08-1.31-.24-.4-.16-.75-.39-1.05-.7s-.53-.69-.7-1.13c-.17-.45-.25-.96-.25-1.53zm6.91-6.45h1.58v6.17h.05l2.54-3.16h1.77l-2.35 2.8 2.59 4.07h-1.75l-1.77-2.98-1.08 1.23v1.75h-1.58zm13.69 1.27c-.25-.11-.5-.17-.75-.17-.58 0-.87.39-.87 1.16v.75h1.34v1.27h-1.34v5.6h-1.61v-5.6h-.92v-1.2l.92-.07v-.72c0-.35.04-.68.13-.98.08-.31.21-.57.4-.79s.42-.39.71-.51c.28-.12.63-.18 1.04-.18.24 0 .48.02.69.07.22.05.41.1.57.17zm.48 5.18c0-.57.09-1.08.27-1.53.17-.44.41-.82.72-1.13.3-.31.65-.54 1.04-.71.39-.16.8-.24 1.23-.24s.84.08 1.24.24c.4.17.74.4 1.04.71s.54.69.72 1.13c.19.45.28.96.28 1.53s-.09 1.08-.28 1.53c-.18.44-.42.82-.72 1.13s-.64.54-1.04.7-.81.24-1.24.24-.84-.08-1.23-.24-.74-.39-1.04-.7c-.31-.31-.55-.69-.72-1.13-.18-.45-.27-.96-.27-1.53zm1.65 0c0 .69.14 1.24.43 1.66.28.41.68.62 1.18.62.51 0 .9-.21 1.19-.62.29-.42.44-.97.44-1.66 0-.7-.15-1.26-.44-1.67-.29-.42-.68-.63-1.19-.63-.5 0-.9.21-1.18.63-.29.41-.43.97-.43 1.67zm6.48-3.44h1.33l.12 1.21h.05c.24-.44.54-.79.88-1.02.35-.24.7-.36 1.07-.36.32 0 .59.05.78.14l-.28 1.4-.33-.09c-.11-.01-.23-.02-.38-.02-.27 0-.56.1-.86.31s-.55.58-.77 1.1v4.2h-1.61zm-47.87 15h1.61v4.1c0 .57.08.97.25 1.2.17.24.44.35.81.35.3 0 .57-.07.8-.22.22-.15.47-.39.73-.73v-4.7h1.61v6.87h-1.32l-.12-1.01h-.04c-.3.36-.63.64-.98.86-.35.21-.76.32-1.24.32-.73 0-1.27-.24-1.61-.71-.33-.47-.5-1.14-.5-2.02zm9.46 7.43v2.16h-1.61v-9.59h1.33l.12.72h.05c.29-.24.61-.45.97-.63.35-.17.72-.26 1.1-.26.43 0 .81.08 1.15.24.33.17.61.4.84.71.24.31.41.68.53 1.11.13.42.19.91.19 1.44 0 .59-.09 1.11-.25 1.57-.16.47-.38.85-.65 1.16-.27.32-.58.56-.94.73-.35.16-.72.25-1.1.25-.3 0-.6-.07-.9-.2s-.59-.31-.87-.56zm0-2.3c.26.22.5.37.73.45.24.09.46.13.66.13.46 0 .84-.2 1.15-.6.31-.39.46-.98.46-1.77 0-.69-.12-1.22-.35-1.61-.23-.38-.61-.57-1.13-.57-.49 0-.99.26-1.52.77zm5.87-1.69c0-.56.08-1.06.25-1.51.16-.45.37-.83.65-1.14.27-.3.58-.54.93-.71s.71-.25 1.08-.25c.39 0 .73.07 1 .2.27.14.54.32.81.55l-.06-1.1v-2.49h1.61v9.88h-1.33l-.11-.74h-.06c-.25.25-.54.46-.88.64-.33.18-.69.27-1.06.27-.87 0-1.56-.32-2.07-.95s-.76-1.51-.76-2.65zm1.67-.01c0 .74.13 1.31.4 1.7.26.38.65.58 1.15.58.51 0 .99-.26 1.44-.77v-3.21c-.24-.21-.48-.36-.7-.45-.23-.08-.46-.12-.7-.12-.45 0-.82.19-1.13.59-.31.39-.46.95-.46 1.68zm6.35 1.59c0-.73.32-1.3.97-1.71.64-.4 1.67-.68 3.08-.84 0-.17-.02-.34-.07-.51-.05-.16-.12-.3-.22-.43s-.22-.22-.38-.3c-.15-.06-.34-.1-.58-.1-.34 0-.68.07-1 .2s-.63.29-.93.47l-.59-1.08c.39-.24.81-.45 1.28-.63.47-.17.99-.26 1.54-.26.86 0 1.51.25 1.93.76s.63 1.25.63 2.21v4.07h-1.32l-.12-.76h-.05c-.3.27-.63.48-.98.66s-.73.27-1.14.27c-.61 0-1.1-.19-1.48-.56-.38-.36-.57-.85-.57-1.46zm1.57-.12c0 .3.09.53.27.67.19.14.42.21.71.21.28 0 .54-.07.77-.2s.48-.31.73-.56v-1.54c-.47.06-.86.13-1.18.23-.31.09-.57.19-.76.31s-.33.25-.41.4c-.09.15-.13.31-.13.48zm6.29-3.63h-.98v-1.2l1.06-.07.2-1.88h1.34v1.88h1.75v1.27h-1.75v3.28c0 .8.32 1.2.97 1.2.12 0 .24-.01.37-.04.12-.03.24-.07.34-.11l.28 1.19c-.19.06-.4.12-.64.17-.23.05-.49.08-.76.08-.4 0-.74-.06-1.02-.18-.27-.13-.49-.3-.67-.52-.17-.21-.3-.48-.37-.78-.08-.3-.12-.64-.12-1.01zm4.36 2.17c0-.56.09-1.06.27-1.51s.41-.83.71-1.14c.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.37c.08.62.29 1.1.65 1.44.36.33.82.5 1.38.5.3 0 .58-.04.84-.13.25-.09.51-.21.76-.37l.54 1.01c-.32.21-.69.39-1.09.53s-.82.21-1.26.21c-.47 0-.92-.08-1.33-.25-.41-.16-.77-.4-1.08-.7-.3-.31-.54-.69-.72-1.13-.17-.44-.26-.95-.26-1.52zm4.61-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.08.45-.31.29-.5.73-.57 1.3zm3.01 2.23c.31.24.61.43.92.57.3.13.63.2.98.2.38 0 .65-.08.83-.23s.27-.35.27-.6c0-.14-.05-.26-.13-.37-.08-.1-.2-.2-.34-.28-.14-.09-.29-.16-.47-.23l-.53-.22c-.23-.09-.46-.18-.69-.3-.23-.11-.44-.24-.62-.4s-.33-.35-.45-.55c-.12-.21-.18-.46-.18-.75 0-.61.23-1.1.68-1.49.44-.38 1.06-.57 1.83-.57.48 0 .91.08 1.29.25s.71.36.99.57l-.74.98c-.24-.17-.49-.32-.73-.42-.25-.11-.51-.16-.78-.16-.35 0-.6.07-.76.21-.17.15-.25.33-.25.54 0 .14.04.26.12.36s.18.18.31.26c.14.07.29.14.46.21l.54.19c.23.09.47.18.7.29s.44.24.64.4c.19.16.34.35.46.58.11.23.17.5.17.82 0 .3-.06.58-.17.83-.12.26-.29.48-.51.68-.23.19-.51.34-.84.45-.34.11-.72.17-1.15.17-.48 0-.95-.09-1.41-.27-.46-.19-.86-.41-1.2-.68z" fill="#535353"/></g></svg>"/></a></p><div><h3 id="citeas">Cite this article</h3><p>Brielle, E.S., Fleisher, J., Wynne-Jones, S. <i>et al.</i> Entwined African and Asian genetic roots of medieval peoples of the Swahili coast.
                    <i>Nature</i> <b>615</b>, 866–873 (2023). https://doi.org/10.1038/s41586-023-05754-w</p><p><a data-test="citation-link" data-track="click" data-track-action="download article citation" data-track-label="link" data-track-external="" rel="nofollow" href="https://citation-needed.springer.com/v2/references/10.1038/s41586-023-05754-w?format=refman&amp;flavour=citation">Download citation<svg width="16" height="16" focusable="false" role="img" aria-hidden="true"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#global-icon-download"></use></svg></a></p><ul data-test="publication-history"><li><p>Received<span>: </span><span><time datetime="2022-07-07">07 July 2022</time></span></p></li><li><p>Accepted<span>: </span><span><time datetime="2023-01-24">24 January 2023</time></span></p></li><li><p>Published<span>: </span><span><time datetime="2023-03-29">29 March 2023</time></span></p></li><li><p>Issue Date<span>: </span><span><time datetime="2023-03-30">30 March 2023</time></span></p></li><li><p><abbr title="Digital Object Identifier">DOI</abbr><span>: </span><span>https://doi.org/10.1038/s41586-023-05754-w</span></p></li></ul></div></div></div></div></section>
            </div></div>
  </body>
</html>
