<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <title>James Routley | Feed</title>
    <link
      rel="stylesheet"
      type="text/css"
      href="../styles.css"
      media="screen"
    />
  </head>
  <body>
    <a href="/index.html">Back</a>
    <a href="https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC8702434/">Original</a>
    <h1>Evidence for a mouse origin of the SARS-CoV-2 Omicron variant</h1>
    
    <div id="readability-page-1" class="page"><div id="main-content">
    <article>
        <section>
         
    <ul>
            
                <li>
                    <a href="https://www.ncbi.nlm.nih.gov/labs/pmc/journals/">Journal List</a>
                </li>
            
                <li>
                    <a href="https://www.ncbi.nlm.nih.gov/pmc/?term=Elsevier%20Public%20Health%20Emergency%20Collection[filter]">Elsevier Public Health Emergency Collection</a>
                </li>
            
                <li aria-current="page">
                    PMC8702434
                </li>
            
    </ul>
 

        </section>
        
        <section role="document">
             
                <div id="mc"><!--main-content--><div data-jigconfig="smoothScroll: false, allHeadingLevels: [&#39;h2&#39;], headingExclude: &#39;:hidden,.nomenu&#39;"><div><div><div><div><p><span id="pmcmata">J Genet Genomics.</span> 2021 Dec 24 </p></div></div></div><div><div><p><a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Wei%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=34954396" id="au1" co-rid="_co_idm140398052352224" co-class="co-affbox">Changshuo Wei</a>,<sup>a,</sup><sup>b,</sup><sup>1</sup> <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Shan%20KJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=34954396" id="au2" co-rid="_co_idm140398025680208" co-class="co-affbox">Ke-Jia Shan</a>,<sup>a,</sup><sup>b,</sup><sup>1</sup> <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Wang%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=34954396" id="au3" co-rid="_co_idm140398025676736" co-class="co-affbox">Weiguang Wang</a>,<sup>a,</sup><sup>b,</sup><sup>1</sup> <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Zhang%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=34954396" id="au4" co-rid="_co_idm140398097299616" co-class="co-affbox">Shuya Zhang</a>,<sup>a,</sup><sup>b</sup> <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Huan%20Q%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=34954396" id="au5" co-rid="_co_idm140398019280320" co-class="co-affbox">Qing Huan</a>,<sup>a,</sup><sup>∗∗</sup> and  <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Qian%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=34954396" id="au6" co-rid="_co_idm140398019277360" co-class="co-affbox">Wenfeng Qian</a><sup>a,</sup><sup>b,</sup><sup>∗</sup></p></div></div></div><div id="abs0010" lang="en"><h2 id="abs0010title">Abstract</h2><!--article-meta--><p id="__p1">The rapid accumulation of mutations in the SARS-CoV-2 Omicron variant that enabled its outbreak raises questions as to whether its proximal origin occurred in humans or another mammalian host. Here, we identified 45 point mutations that Omicron acquired since divergence from the B.1.1 lineage. We found that the Omicron spike protein sequence was subjected to stronger positive selection than that of any reported SARS-CoV-2 variants known to evolve persistently in human hosts, suggesting a possibility of host-jumping. The molecular spectrum of mutations (<em>i.e.</em>, the relative frequency of the 12 types of base substitutions) acquired by the progenitor of Omicron was significantly different from the spectrum for viruses that evolved in human patients, but resembled the spectra associated with virus evolution in a mouse cellular environment. Furthermore, mutations in the Omicron spike protein significantly overlapped with SARS-CoV-2 mutations known to promote adaptation to mouse hosts, particularly through enhanced spike protein binding affinity for the mouse cell entry receptor. Collectively, our results suggest that the progenitor of Omicron jumped from humans to mice, rapidly accumulated mutations conducive to infecting that host, then jumped back into humans, indicating an inter-species evolutionary trajectory for the Omicron outbreak.</p><p><strong>Keywords: </strong><span>SARS-CoV-2, Omicron, Evolutionary origins, Molecular spectrum of mutations, Spike-ACE2 interaction, Receptor-binding domain</span></p></div><div id="sec1"><h2 id="sec1title">Introduction</h2><p id="p0010">The coronavirus disease 2019 (COVID-19) pandemic, caused by the SARS-CoV-2 RNA virus, has led to significant illness and death worldwide. The SARS-CoV-2 Omicron variant was first reported in South Africa on November 24<sup>th</sup>, 2021, and was designated as a variant of concern (VOC) within two days by the World Health Organization (WHO) based on the increase in infections attributable to this variant in South Africa (<em>i.e.</em>, Omicron outbreak). In addition, the open reading frame encoding the spike protein (ORF <em>S</em>) of Omicron harbors an exceptionally high number of mutations. These mutations are particularly relevant to infection characteristics because the SARS-CoV-2 spike protein is well-known to mediate viral entry into the host cell by interacting with angiotensin-converting enzyme 2 (ACE2) on the cell surface (<a href="#bib46" rid="bib46">Zhou et al., 2020</a>). In addition, the spike protein is also a target for vaccine development and antibody-blocking therapy (<a href="#bib13" rid="bib13">Huang et al., 2020</a>; <a href="#bib25" rid="bib25">Martinez-Flores et al., 2021</a>).</p><p id="p0015">The proximal origins of Omicron have quickly become a controversial topic of heated debate in the scientific and public health communities (<a href="#bib3" rid="bib3">Callaway, 2021</a>; <a href="#bib19" rid="bib19">Kupferschmidt, 2021</a>). Many mutations detected in Omicron were rarely reported among previously sequenced SARS-CoV-2 variants (<a href="#bib33" rid="bib33">Shu and McCauley, 2017</a>; <a href="#bib9" rid="bib9">Hadfield et al., 2018</a>), leading to three prevalent hypotheses regarding its evolutionary history. The first hypothesis is that Omicron could have “cryptically spread” and circulated in a population with insufficient viral surveillance and sequencing. Second, Omicron could have evolved in a chronically infected COVID-19 patient, such as an immunocompromised individual who provided a suitable host environment conducive to long-term intra-host virus adaptation. The third possibility is that Omicron could have accumulated mutations in a nonhuman host and then jumped into humans. Currently, the second scenario represents the most popular hypothesis regarding the proximal origins of Omicron (<a href="#bib3" rid="bib3">Callaway, 2021</a>; <a href="#bib19" rid="bib19">Kupferschmidt, 2021</a>).</p><p id="p0020">The first two hypotheses assume that Omicron acquired these mutations in humans (collectively referred to as “human origin hypothesis” hereafter), while the third assumes that Omicron acquired mutations in a nonhuman species. Based on our previous work in viral evolution, we hypothesized that the host species in which Omicron acquired its particular set of mutations could be determined by analyzing the molecular spectra of mutations (<em>i.e.</em>, the relative frequency of the 12 types of base substitutions). In previous work, we showed that <em>de novo</em> mutations in RNA virus genomes are generated in a replication-independent manner and are highly dependent on mutagenic mechanisms specific to the host cellular environment, resulting in overrepresentation with specific mutation types. For example, reactive oxygen species (ROS) can oxidize guanine to 8-oxoguanine and thereby induce the G&gt;U transversion (<a href="#bib24" rid="bib24">Li et al., 2006</a>; <a href="#bib17" rid="bib17">Kong and Lin, 2010</a>), while cytidine deaminases can induce RNA editing such as C&gt;U transitions (<a href="#bib2" rid="bib2">Blanc and Davidson, 2010</a>; <a href="#bib10" rid="bib10">Harris and Dudley, 2015</a>). Consistent with this phenomenon, viruses belonging to different orders (<em>e.g.</em>, poliovirus, Ebola virus, and SARS-CoV-2) were found to exhibit similar molecular spectra of mutations when evolving in the same host species, while members of the same virus species exhibit divergent molecular spectra when evolving in different host species (<a href="#bib32" rid="bib32">Shan et al., 2021</a>). Since <em>de novo</em> mutations can thus strongly influence the molecular spectrum of mutations that accumulate during virus evolution in a host-specific manner, the host species in which Omicron acquired its mutations could be determined by analyzing information carried by the mutations themselves.</p><p id="p0025">In this study, we identified mutations acquired by Omicron before its outbreak, and tested whether the molecular spectrum of these mutations was consistent with the cellular environment of human hosts. Prominent dissimilarities were observed between the molecular spectrum of Omicron and a relatively comprehensive set of molecular spectra from variants known to have evolved in humans, including those of three isolates from chronic COVID-19 patients. Therefore, we next examined the molecular spectra of mutations obtained from a wide range of host mammals for comparison with that of Omicron. Finally, we used molecular docking-based analyses to investigate whether the mutations in the Omicron spike protein could be associated with adaptation to the host species inferred from molecular spectrum analysis. Our study provides insight into the evolutionary trajectory and proximal origins of Omicron through careful scrutiny of its mutations, and suggests strategies for avoiding future outbreaks caused by SARS-CoV-2 variants proliferating in wild animals.</p></div><div id="sec2"><h2 id="sec2title">Results</h2><div id="sec2.1"><h3 id="sec2.1title">Over-representation of nonsynonymous mutations in Omicron ORF <em>S</em> suggests strong positive selection</h3><p id="p0030">To first identify mutations that accumulated in the SARS-CoV-2 genome prior to the Omicron outbreak, we constructed a phylogenetic tree that included the genomic sequences of the reference SARS-CoV-2 (<a href="#bib43" rid="bib43">Wu et al., 2020a</a>), two variants in the B.1.1 lineage which were genetically close to Omicron (based on the results of BLASTn), and 48 Omicron variants sampled before November 15<sup>th</sup>, 2021 (<a href="https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC8702434/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Fig. 1</span></a>
A). These two B.1.1 variants were sampled during April 22<sup>nd</sup>–May 5<sup>th</sup>, 2020, which suggested that the progenitor of Omicron diverged from the B.1.1 lineage roughly in mid-2020. Intermediate versions have gone largely undetected, thus resulting in an exceptionally long branch leading to the most recent common ancestor (MRCA) of Omicron in the phylogenetic tree (<a href="https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC8702434/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Fig. 1</span></a>A). We hereafter refer to this long branch as Branch O.</p><!--fig ft0--><!--fig mode=article f1--><div id="fig1" co-legend-rid="lgnd_fig1"><a href="https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC8702434/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140398018912080"><a href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=8702434_gr1_lrg.jpg" target="tileshopwindow" rel="noopener"><img loading="lazy" alt="Fig. 1" title="Click on image to zoom" src="https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC8702434/bin/gr1_lrg.jpg"/></a></div><div id="lgnd_fig1"><!--caption a7--><p id="__p2">The characterization of pre-outbreak Omicron mutations. <strong>A</strong>: The phylogenetic tree of Omicron variants, including the reference genome of SARS-CoV-2, two B.1.1 variants, and 48 Omicron variants. A total of 45 pre-outbreak Omicron point mutations in the long branch (Branch O, labeled in purple) leading to the MRCA of Omicron (dot in red) in the phylogenetic tree are shown. <strong>B</strong>: The genomic distribution of the 45 pre-outbreak Omicron mutations, the mutations detected in each progenitor of the other four VOCs (<em>i.e.</em>, Alpha, Beta, Gamma, and Delta), and the mutations identified from the SARS-CoV-2 isolates of three chronically infected patients. The density curves for mutations were generated by the geom_freqpoly function in <em>R</em>. <strong>C</strong>: Number of mutations that accumulated in ORF <em>S</em> of the MRCA of Omicron (red), the other four VOCs (cyan), and three SARS-CoV-2 isolates from chronically infected patients (blue), against the date of sample collection. SARS-CoV-2 variants randomly sampled (one variant per day) are shown in grey, and the grey line represents their linear regression. <strong>D</strong>: Similar to (<strong>C</strong>), for the whole genome. <strong>E</strong>: A scatterplot shows the numbers of synonymous and nonsynonymous mutations in ORF <em>S</em> (jittered in order to reduce overplotting). UTR, untranslated region.</p></div></div><p id="p0035">We identified 45 point mutations that were introduced in Branch O (hereafter referred to as “pre-outbreak Omicron mutations”; <a href="#appsec1" rid="appsec1">Fig. S1</a>). We observed that the pre-outbreak Omicron mutations were over-represented in ORF <em>S</em> (<em>P</em> = 1.2 × 10<sup>−13</sup>, binomial test with the expected probability equal to the length of ORF <em>S</em> relative to the SARS-CoV-2 genome; <a href="https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC8702434/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Fig. 1</span></a>B), especially in the coding region of the receptor-binding domain (RBD) (<em>P</em> = 1.1 × 10<sup>−13</sup>, <a href="https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC8702434/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Fig. 1</span></a>B). We further identified mutations in the other four SARS-CoV-2 VOCs (<em>i.e.</em>, Alpha, Beta, Gamma, and Delta) as well as those in the SARS-CoV-2 variants isolated from three chronically infected patients (<a href="#bib15" rid="bib15">Kemp et al., 2021</a>; <a href="#bib37" rid="bib37">Truong et al., 2021</a>), but did not observe such a level of over-representation of mutations in ORF <em>S</em> or RBD region as in the pre-outbreak Omicron mutations (<a href="https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC8702434/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Fig. 1</span></a>B).</p><p id="p0040">To test if the rate at which mutations accumulated in ORF <em>S</em> was accelerated in Branch O, we randomly sampled one SARS-CoV-2 variant per day since December 24<sup>th</sup>, 2019 from the Global Initiative on Sharing All Influenza Data (GISAID) (<a href="#bib33" rid="bib33">Shu and McCauley, 2017</a>) to compare mutation accumulation rates among different variants. We found that mutations accumulated in ORF <em>S</em> at a rate of ∼0.45 mutations per month on average. In sharp contrast, 27 mutations accumulated in ORF <em>S</em> in Branch O during the 18 months spanning May 2020–November 2021, equivalent to ∼1.5 mutations per month, or ∼3.3 times faster than the average rate of other variants (<a href="https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC8702434/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Fig. 1</span></a>C).</p><p id="p0045">Counting mutations across the whole SARS-CoV-2 genome indicated that Omicron acquired mutations in the genome at a similar rate to other variants (<a href="https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC8702434/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Fig. 1</span></a>D), suggesting that the accelerated evolutionary rate of ORF <em>S</em> could not be explained by an overall elevated mutation rate in Omicron progenitors. In light of these findings, we hypothesized that positive selection could have helped accelerate the evolutionary rate of ORF <em>S</em>. To test this hypothesis, we sought to infer the strength of positive selection by estimating the ratio of nonsynonymous to synonymous mutations. Twenty-six of the 27 pre-outbreak mutations in the ORF <em>S</em> of Omicron were nonsynonymous (<a href="https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC8702434/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Fig. 1</span></a>E), resulting in a <em>d</em>
<sub>N</sub>/<em>d</em>
<sub>S</sub> ratio of 6.64, significantly greater than a <em>d</em>
<sub>N</sub>/<em>d</em>
<sub>S</sub> of 1.00 (<em>P</em> = 0.03, Fisher’s exact test). These results indicated that positive selection contributed to increasing the mutation rate in ORF <em>S</em> in Branch O.</p><p id="p0050">To test if such a level of positive selection is common among SARS-CoV-2 variants, we counted the number of nonsynonymous and synonymous mutations in ORF <em>S</em> in the other four VOCs as well as in the variants isolated from three chronically infected patients (<a href="#bib15" rid="bib15">Kemp et al., 2021</a>; <a href="#bib37" rid="bib37">Truong et al., 2021</a>). None of these other VOCs or isolates exhibited comparable numbers of nonsynonymous mutations as that of mutations in Branch O (<a href="https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC8702434/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Fig. 1</span></a>E). These observations strongly suggested that the Omicron variant had undergone a strong positive selection for the spike protein that no other known SARS-CoV-2 variants evolved in humans had been subjected to. Considering that the spike protein determines the host range of a coronavirus (<em>i.e.</em>, which organisms it can infect), we therefore hypothesized that the progenitor of Omicron might host-jump from humans to a nonhuman species because this process would require substantial mutations in the spike protein for rapid adaptation to a new host.</p></div><div id="sec2.2"><h3 id="sec2.2title">The molecular spectrum of pre-outbreak Omicron mutations is inconsistent with an evolutionary history in humans</h3><p id="p0055">Previous studies showed that the molecular spectrum of mutations that accumulate in a viral genome reflects a host-specific cellular environment (<a href="#bib6" rid="bib6">Deng et al., 2021</a>; <a href="#bib32" rid="bib32">Shan et al., 2021</a>). To test the human origin hypothesis of Omicron, we compared the molecular spectrum of the 45 pre-outbreak Omicron mutations with the “standard” molecular spectrum for SARS-CoV-2 variants known to have evolved strictly in humans (hereafter referred to as “the hSCV2 spectrum”; <a href="https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC8702434/figure/fig2/" target="figure" rid-figpopup="fig2" rid-ob="ob-fig2" co-legend-rid="lgnd_fig2"><span>Fig. 2</span></a>
A). The hSCV2 spectrum included 6986 point mutations that were compiled from 34,853 high-quality sequences of SARS-CoV-2 variants isolated from patients worldwide (<a href="#bib32" rid="bib32">Shan et al., 2021</a>). We found that the molecular spectrum of the pre-outbreak Omicron mutations was significantly different from the hSCV2 spectrum (<em>P</em> = 0.004, <em>G</em>-test; <a href="https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC8702434/figure/fig2/" target="figure" rid-figpopup="fig2" rid-ob="ob-fig2" co-legend-rid="lgnd_fig2"><span>Fig. 2</span></a>B). In particular, as in the hSCV2 spectrum, transitions were more abundant than transversions, and C&gt;U mutation was more abundant than its complementary mutation G&gt;A. However, a hallmark of RNA virus mutations when evolving in humans—a higher rate of G&gt;U mutation than its complementary mutation C&gt;A (<a href="#bib29" rid="bib29">Panchin and Panchin, 2020</a>; <a href="#bib5" rid="bib5">De Maio et al., 2021</a>; <a href="#bib6" rid="bib6">Deng et al., 2021</a>; <a href="#bib32" rid="bib32">Shan et al., 2021</a>) that is likely caused by cellular ROS—was absent in the pre-outbreak Omicron mutations.</p><!--fig ft0--><!--fig mode=article f1--><div id="fig2" co-legend-rid="lgnd_fig2"><a href="https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC8702434/figure/fig2/" target="figure" rid-figpopup="fig2" rid-ob="ob-fig2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140398013056256"><a href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=8702434_gr2_lrg.jpg" target="tileshopwindow" rel="noopener"><img loading="lazy" alt="Fig. 2" title="Click on image to zoom" src="https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC8702434/bin/gr2_lrg.jpg"/></a></div><div id="lgnd_fig2"><!--caption a7--><p id="__p3">Comparison of the molecular spectrum of pre-outbreak Omicron mutations and spectra of mutations known to accumulate in humans. <strong>A</strong>: The molecular spectrum of viral mutations that accumulated in humans (the hSCV2 spectrum). <strong>B</strong>: The molecular spectra of pre- and post-outbreak Omicron mutations. <em>P</em> values were given by <em>G-</em>tests against the hSCV2 spectrum. <strong>C</strong>: The distribution of <em>P</em> values (given by <em>G</em>-tests) of 100 pseudo samples that were down sampled from the hSCV2 spectrum. The number of mutations (<em>n</em>) of each pseudo sample was equal to 45. SARS-CoV-2 data of Patient 1 were retrieved from <a href="#bib15" rid="bib15">Kemp et al. (2021)</a> and those of Patients 2 and 3 were retrieved from <a href="#bib37" rid="bib37">Truong et al. (2021)</a>.</p></div></div><p id="p0060">To exclude the possibility that this apparent difference in the molecular spectrum was caused by the relatively small number of pre-outbreak Omicron mutations, we generated 100 “pseudo” variants <em>in silico</em> by randomly down sampling 45 mutations from the hSCV2 spectrum. None of the pseudo variants showed smaller <em>P</em> values (based on <em>G</em>-tests) than that obtained using the pre-outbreak Omicron mutations (<a href="https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC8702434/figure/fig2/" target="figure" rid-figpopup="fig2" rid-ob="ob-fig2" co-legend-rid="lgnd_fig2"><span>Fig. 2</span></a>C), nor did the SARS-CoV-2 isolates with mutations known to be acquired in the three chronically infected patients (# of mutations are 30, 47, and 81; <a href="https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC8702434/figure/fig2/" target="figure" rid-figpopup="fig2" rid-ob="ob-fig2" co-legend-rid="lgnd_fig2"><span>Fig. 2</span></a>C). These observations indicated that the difference between the molecular spectrum of pre-outbreak Omicron mutations and the hSCV2 spectrum could not be strictly attributed to statistical randomness.</p><p id="p0065">To exclude the possibility that some mutations which occurred early in the evolution of Omicron (<em>e.g.</em>, mutations in the RNA-dependent RNA polymerase) distorted the molecular spectrum of mutations that accumulated afterward, we identified 120 point mutations that occurred after the Omicron outbreak, by screening 695 Omicron variants collected spanning November 8<sup>th</sup>–December 7<sup>th</sup>, 2021 (hereafter referred to as “post-outbreak Omicron mutations”). The molecular spectrum of these post-outbreak Omicron mutations was not significantly different from the hSCV2 spectrum (<em>P</em> = 0.64, <em>G</em>-test; <a href="https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC8702434/figure/fig2/" target="figure" rid-figpopup="fig2" rid-ob="ob-fig2" co-legend-rid="lgnd_fig2"><span>Fig. 2</span></a>B–2C). This finding indicated that Omicron would acquire mutations following the same molecular spectrum as other SARS-CoV-2 variants during its evolution in human hosts. Collectively, these molecular spectrum analyses revealed that pre-outbreak Omicron mutations were unlikely to have been acquired in humans.</p></div><div id="sec2.3"><h3 id="sec2.3title">The molecular spectrum of pre-outbreak Omicron mutations is consistent with an evolutionary history in mice</h3><p id="p0070">In light of our findings that Omicron may have evolved in another host before its outbreak, we next sought to determine the nonhuman host species in which these mutations accumulated. To this end, we first characterized the molecular spectra of coronaviruses that evolved in different host species for comparison with that of Omicron. Specifically, we retrieved 17 sequences of murine hepatitis viruses, 13 canine coronaviruses, 54 feline coronaviruses, 23 bovine coronaviruses, and 110 porcine deltacoronaviruses (<a href="#appsec1" rid="appsec1">Table S1</a>), constructed the phylogenetic tree for the coronaviruses isolated from each host species (canine coronavirus as an example shown in <a href="https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC8702434/figure/fig3/" target="figure" rid-figpopup="fig3" rid-ob="ob-fig3" co-legend-rid="lgnd_fig3"><span>Fig. 3</span></a>
A and the rest are shown in <a href="#appsec1" rid="appsec1">Fig. S2</a>), and identified the mutations that accumulated in each branch (<a href="https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC8702434/figure/fig3/" target="figure" rid-figpopup="fig3" rid-ob="ob-fig3" co-legend-rid="lgnd_fig3"><span>Fig. 3</span></a>A). The longest five external branches of each host species were used for the subsequent analysis (see Materials and methods). We also included some previously reported molecular spectra (<a href="#bib32" rid="bib32">Shan et al., 2021</a>), including 17 spectra of mutations acquired by SARS-CoV-, SARS-CoV-2-, and MERS-CoV-related coronaviruses during their evolution in bats, two spectra of camel MERS-CoV, one spectrum estimated from 807 MERS-CoV mutations accumulated in human (the hMERS spectrum), as well as the hSCV2 spectrum. Furthermore, we also included the molecular spectrum of mutations identified in an early variant of each of the other four VOCs.</p><!--fig ft0--><!--fig mode=article f1--><div id="fig3" co-legend-rid="lgnd_fig3"><a href="https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC8702434/figure/fig3/" target="figure" rid-figpopup="fig3" rid-ob="ob-fig3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140398022336112"><a href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=8702434_gr3_lrg.jpg" target="tileshopwindow" rel="noopener"><img loading="lazy" alt="Fig. 3" title="Click on image to zoom" src="https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC8702434/bin/gr3_lrg.jpg"/></a></div><div id="lgnd_fig3"><!--caption a7--><p id="__p4">The similarity in molecular spectra between Omicron and coronaviruses isolated from various mammalian species. <strong>A</strong>: A schematic shows the workflow for analyzing the similarity in molecular spectra across various hosts. <strong>B</strong>: The principal component analysis plot depicts the molecular spectra of virus mutations that accumulated in humans and various host species. Dots were colored according to the corresponding host species. The 95% confidence ellipses are shown for each host species. VOC, variant of concern.</p></div></div><p id="p0075">We performed principal component analysis to reduce the dimensionality of the molecular spectrum of mutations, and subsequently visualized the data using the first two principal components (<a href="https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC8702434/figure/fig3/" target="figure" rid-figpopup="fig3" rid-ob="ob-fig3" co-legend-rid="lgnd_fig3"><span>Fig. 3</span></a>B). Consistent with the results of our previous study (<a href="#bib32" rid="bib32">Shan et al., 2021</a>), drawing 95% confidence ellipses for each host species showed that the molecular spectra clustered according to their respective hosts (<a href="https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC8702434/figure/fig3/" target="figure" rid-figpopup="fig3" rid-ob="ob-fig3" co-legend-rid="lgnd_fig3"><span>Fig. 3</span></a>B), likely because viruses evolving in the same host species share the mutagens specific to that host’s cellular environment. Supporting this point, the molecular spectrum of post-outbreak Omicron mutations (which are known to have accumulated in humans) was located within the human 95% confidence ellipse. In contrast, the molecular spectrum of pre-outbreak Omicron mutations was within the mouse ellipse, suggesting that the pre-outbreak mutations accumulated in a rodent (in particular a mouse) host.</p></div><div id="sec2.4"><h3 id="sec2.4title">Pre-outbreak Omicron mutations in the spike protein significantly overlap with mutations in mouse-adapted SARS-CoV-2</h3><p id="p0080">Mice were previously reported to serve as poor hosts for SARS-CoV-2 because the spike protein of early SARS-CoV-2 variants exhibited low-affinity interactions with mouse ACE2 (<a href="#bib20" rid="bib20">Lam et al., 2020</a>; <a href="#bib46" rid="bib46">Zhou et al., 2020</a>; <a href="#bib30" rid="bib30">Ren et al., 2021</a>; <a href="#bib42" rid="bib42">Wong et al., 2021</a>). However, over the course of the pandemic, SARS-CoV-2 variants emerged that could infect mice. For example, variants harboring the spike mutation N501Y, which are relatively common in human patients (24.7%, CoV-GLUE-Viz, accessed on November 23<sup>th</sup>, 2021), could infect mice (<a href="#bib8" rid="bib8">Gu et al., 2020</a>; <a href="#bib22" rid="bib22">Leist et al., 2020</a>; <a href="#bib35" rid="bib35">Sun et al., 2021</a>). If the progenitor of Omicron indeed evolved in a mouse species before the Omicron outbreak, we postulated that its spike protein likely adapted through increased binding affinity for mouse ACE2. To test this possibility, we projected the pre-outbreak Omicron mutations in the spike protein onto a three-dimensional structure of the spike:ACE2 complex (<a href="#bib21" rid="bib21">Lan et al., 2020</a>). Seven mutations (<em>i.e.</em>, K417N, G446S, E484A, Q493R, G496S, Q498R, and N501Y) were located at the interface of ACE2 and the spike protein RBD, and could potentially affect their interactions (<a href="https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC8702434/figure/fig4/" target="figure" rid-figpopup="fig4" rid-ob="ob-fig4" co-legend-rid="lgnd_fig4"><span>Fig. 4</span></a>
A).</p><!--fig ft0--><!--fig mode=article f1--><div id="fig4" co-legend-rid="lgnd_fig4"><a href="https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC8702434/figure/fig4/" target="figure" rid-figpopup="fig4" rid-ob="ob-fig4"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140398015659168"><a href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=8702434_gr4_lrg.jpg" target="tileshopwindow" rel="noopener"><img loading="lazy" alt="Fig. 4" title="Click on image to zoom" src="https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC8702434/bin/gr4_lrg.jpg"/></a></div><div id="lgnd_fig4"><!--caption a7--><p id="__p5">The similarity in the spike protein sequence between Omicron and SARS-CoV-2 variants isolated from various hosts. <strong>A</strong>: The interface structure between SARS-CoV-2 spike protein and human ACE2 (PDB: 6M0J). RBD residues on the interface (with a distance cut-off of 5 Å) were labeled. <strong>B</strong>: The pre-outbreak Omicron mutations in the spike protein. <strong>C</strong>: The statistical assessment on the overlapping in mutated positions of the spike protein between Omicron and SARS-CoV-2 variants isolated from chronically infected patients or nonhuman mammals using Fisher’s exact tests. The 2 × 2 contingency table for mice is shown. <strong>D</strong>: Comparison between pre-outbreak Omicron mutations and mutations detected in seven mouse-adapted SARS-CoV-2 variants in the spike protein. OR, odds ratio.</p></div></div><p id="p0085">Previous studies reported specific amino acid mutations that allow SARS-CoV-2 variants (mouse-adapted SARS-CoV-2) to use mouse ACE2 for entry into cells (<a href="#bib22" rid="bib22">Leist et al., 2020</a>; <a href="#bib44" rid="bib44">Wu et al., 2020b</a>; <a href="#bib12" rid="bib12">Huang et al., 2021</a>; <a href="#bib26" rid="bib26">Montagutelli et al., 2021</a>; <a href="#bib35" rid="bib35">Sun et al., 2021</a>; <a href="#bib42" rid="bib42">Wong et al., 2021</a>; <a href="#bib45" rid="bib45">Zhang et al., 2021</a>). In addition, previous studies have described some reverse zoonotic events (<em>i.e.</em>, host-jumping from humans to other mammals such as mink and white-tailed deer) for SARS-CoV-2 (<a href="#bib4" rid="bib4">Chandler et al., 2021</a>; <a href="#bib28" rid="bib28">Oude Munnink et al., 2021</a>), and the variants isolated from these mammalian hosts presumably harbored amino acid mutations that could potentially participate in their adaptation to these hosts (<a href="#bib36" rid="bib36">Telenti et al., 2021</a>). Thus, if the progenitor of Omicron evolved in mice and adapted to mouse ACE2, we predicted that the pre-outbreak Omicron mutations should share considerable overlap with the mutations identified in these mouse-adapted SARS-CoV-2 variants, but not those of other mammalian species.</p><p id="p0090">To test this prediction, we identified the mutations in ORF <em>S</em> of SARS-CoV-2 variants isolated from 18 mammalian species (<em>e.g.</em>, mice, cats, dogs, minks, and deer; <a href="#appsec1" rid="appsec1">Tables S2 and S3</a>) and found that pre-outbreak Omicron mutations tended to share the same positions as the ORF <em>S</em> mutations identified in mice (odds ratio = 231.4, <em>P</em> = 1.6 × 10<sup>−11</sup>, Fisher’s exact test; <a href="https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC8702434/figure/fig4/" target="figure" rid-figpopup="fig4" rid-ob="ob-fig4" co-legend-rid="lgnd_fig4"><span>Fig. 4</span></a>B–4C). In contrast, same statistical test showed much lower odds ratios and significance levels for overlap in these mutations with other species (<a href="https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC8702434/figure/fig4/" target="figure" rid-figpopup="fig4" rid-ob="ob-fig4" co-legend-rid="lgnd_fig4"><span>Fig. 4</span></a>C<strong>)</strong>. Pre-outbreak Omicron mutations also overlapped with some mutations detected in isolates from chronically infected patients (<a href="#bib15" rid="bib15">Kemp et al., 2021</a>; <a href="#bib37" rid="bib37">Truong et al., 2021</a>); however, they too showed substantially lower odds ratios and significance levels than those isolated from mice (<a href="https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC8702434/figure/fig4/" target="figure" rid-figpopup="fig4" rid-ob="ob-fig4" co-legend-rid="lgnd_fig4"><span>Fig. 4</span></a>C). These observations implied that the pre-outbreak Omicron mutations in ORF <em>S</em> promoted its adaptation to a mouse host.</p><p id="p0095">We then conducted enrichment analysis for each of the seven mouse-adapted SARS-CoV-2 variants and observed statistical significance for all these variants (<a href="https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC8702434/figure/fig4/" target="figure" rid-figpopup="fig4" rid-ob="ob-fig4" co-legend-rid="lgnd_fig4"><span>Fig. 4</span></a>D). In particular, we observed amino acid mutations at residues 493 and 498 in five and six of the seven mouse-adapted SARS-CoV-2 variants, respectively (<a href="https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC8702434/figure/fig4/" target="figure" rid-figpopup="fig4" rid-ob="ob-fig4" co-legend-rid="lgnd_fig4"><span>Fig. 4</span></a>D). Identical amino acid mutations (<em>i.e.</em>, Q493R and Q498R) were both observed in two variants (<a href="#bib26" rid="bib26">Montagutelli et al., 2021</a>; <a href="#bib42" rid="bib42">Wong et al., 2021</a>). Considering that these two amino acid mutations are uncommon in human patients infected by non-Omicron SARS-CoV-2 variants (0.005% and 0.002%, respectively, CoV-GLUE-Viz, accessed on November 23<sup>th</sup>, 2021) we proposed the hypothesis that the progenitor of Omicron evolved in mice.</p></div><div id="sec2.5"><h3 id="sec2.5title">Pre-outbreak Omicron mutations in the RBD significantly enhance binding affinity with mouse ACE2</h3><p id="p0100">To investigate the mechanisms by which the pre-outbreak Omicron mutations in the spike protein could have contributed to its adaptation to a mouse host, we examined their interaction through molecular docking analysis of the spike protein RBD and mouse ACE2 (<a href="https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC8702434/figure/fig5/" target="figure" rid-figpopup="fig5" rid-ob="ob-fig5" co-legend-rid="lgnd_fig5"><span>Fig. 5</span></a>
A). Following previous studies (<a href="#bib20" rid="bib20">Lam et al., 2020</a>; <a href="#bib31" rid="bib31">Rodrigues et al., 2020</a>), we estimated the HADDOCK score (<a href="#bib38" rid="bib38">van Zundert et al., 2016</a>), which is positively associated with the dissociation constant (<em>K</em>
<sub>D</sub>, with smaller <em>K</em>
<sub>D</sub> indicating stronger binding) of protein interactions (<a href="#bib14" rid="bib14">Kastritis and Bonvin, 2010</a>), and can be used to predict the susceptibility of a mammalian species to infection with SARS-CoV-2 (<a href="#bib31" rid="bib31">Rodrigues et al., 2020</a>).</p><!--fig ft0--><!--fig mode=article f1--><div id="fig5" co-legend-rid="lgnd_fig5"><a href="https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC8702434/figure/fig5/" target="figure" rid-figpopup="fig5" rid-ob="ob-fig5"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140398009883776"><a href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=8702434_gr5_lrg.jpg" target="tileshopwindow" rel="noopener"><img loading="lazy" alt="Fig. 5" title="Click on image to zoom" src="https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC8702434/bin/gr5_lrg.jpg"/></a></div><div id="lgnd_fig5"><!--caption a7--><p id="__p6">Predicted binding affinities between RBD variants and mouse ACE2. <strong>A</strong>: A schematic shows the workflow to estimate the HADDOCK scores between RBD variants and mouse ACE2. <strong>B</strong>: The HADDOCK scores for the interaction of various RBD variants with the mouse ACE2. The error bars represent standard errors. Penta-mutant of RBD harbored five mutations (K417N, E484A, Q493R, Q498R, and N501Y). The result of Patient 2, who did not harbor any amino acid mutations in RBD, was not shown. RBD, receptor-binding domain.</p></div></div><p id="p0105">To confirm the accuracy of molecular docking-based inferences regarding the binding affinity between spike protein RBD and ACE2, we estimated HADDOCK scores for the interaction between the reference RBD and ACE2 of various mammalian species that have experimental evidence about the susceptibility to infection with the reference SARS-CoV-2. The susceptible mammalian species indeed exhibited lower HADDOCK scores (<em>P</em> = 0.001, <em>t</em>-test; <a href="#appsec1" rid="appsec1">Fig. S3</a>). Furthermore, we calculated the HADDOCK score for eight experimentally determined <em>K</em>
<sub>D</sub> values between four RBD variants and human (or mouse) ACE2 (<a href="#bib35" rid="bib35">Sun et al., 2021</a>). The HADDOCK scores were positively correlated with the <em>K</em>
<sub>D</sub> values in the analysis (Pearson’s correlation coefficient <em>r</em> = 0.93, <em>P</em> = 0.002; <a href="#appsec1" rid="appsec1">Fig. S4A–S4C</a>). In addition, the binding affinity with mouse ACE2 was elevated in all seven mouse-adapted SARS-CoV-2 variants (five of them were statistically significant; <a href="#appsec1" rid="appsec1">Fig. S4D</a>). All these observations supported the validity of molecular docking-based predictions of ACE2-binding affinity for other RBD variants.</p><p id="p0110">The molecular docking-based predictions suggested that the RBD of Omicron exhibited higher binding affinity for mouse ACE2 than that of RBD encoded in the reference SARS-CoV-2 genome, further suggesting an evolutionary history in mice (<a href="https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC8702434/figure/fig5/" target="figure" rid-figpopup="fig5" rid-ob="ob-fig5" co-legend-rid="lgnd_fig5"><span>Fig. 5</span></a>B). And as expected, the mutations detected in the RBD of the other four VOCs of SARS-CoV-2 as well as those of variants isolated from chronically infected human patients showed no apparent changes in their binding affinity for mouse ACE2 compared with the reference RBD (<a href="https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC8702434/figure/fig5/" target="figure" rid-figpopup="fig5" rid-ob="ob-fig5" co-legend-rid="lgnd_fig5"><span>Fig. 5</span></a>B).</p><p id="p0115">Since five amino acid mutations were shared between Omicron and mouse-adapted SARS-CoV-2 variants in RBD (<em>i.e.</em>, K417N, E484A, Q493R, Q498R, and N501Y; <a href="https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC8702434/figure/fig4/" target="figure" rid-figpopup="fig4" rid-ob="ob-fig4" co-legend-rid="lgnd_fig4"><span>Fig. 4</span></a>B), and that they together enhanced RBD binding affinity for mouse ACE2 (<a href="https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC8702434/figure/fig5/" target="figure" rid-figpopup="fig5" rid-ob="ob-fig5" co-legend-rid="lgnd_fig5"><span>Fig. 5</span></a>B), we next determined the individual effects of each of these five mutations. Notably, only Q493R and Q498R significantly increased the binding affinity with mouse ACE2, which was consistent with their repeated detection in mouse-adapted SARS-CoV-2 variants (<a href="#bib26" rid="bib26">Montagutelli et al., 2021</a>; <a href="#bib42" rid="bib42">Wong et al., 2021</a>). Indeed, docking analysis showed that Q493R/Q498R double mutation could further increase the RBD binding affinity for mouse ACE2 (<a href="https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC8702434/figure/fig5/" target="figure" rid-figpopup="fig5" rid-ob="ob-fig5" co-legend-rid="lgnd_fig5"><span>Fig. 5</span></a>B). By contrast, the other three mutations showed no significant effects on the binding affinity between RBD and mouse ACE2, neither in the reference RBD nor in the Q493R/Q498R double mutant (<a href="https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC8702434/figure/fig5/" target="figure" rid-figpopup="fig5" rid-ob="ob-fig5" co-legend-rid="lgnd_fig5"><span>Fig. 5</span></a>B), suggesting that they did not contribute to the enhanced interaction between Omicron RBD and mouse ACE2. We speculated that these mutations (K417N, E484K, and N501Y) were acquired in Omicron because they were related to escape from neutralizing antibodies, as indicated by previous studies (<a href="#bib23" rid="bib23">Li et al., 2021</a>; <a href="#bib27" rid="bib27">Nelson et al., 2021</a>).</p></div><div id="sec2.6"><h3 id="sec2.6title">The pre-outbreak Omicron mutations in the RBD showed the greatest enhanced binding affinity for mouse ACE2 among 32 mammals</h3><p id="p0120">Our characterization of the molecular spectrum of mutations and observations of RBD-ACE2 interactions both suggested that mice were the most likely host species in which the progenitor of Omicron evolved. However, it remained plausible that Omicron could have evolved in another species with a similar cellular mutagen environment and ACE2 structure to that of mice. We therefore postulated that if Omicron evolved in another species, the pre-outbreak Omicron mutations in the RBD should enhance its interactions with the ACE2 of that host. To test this prediction, we applied molecular docking analysis to ACE2 from 31 other species, representing markedly different mammalian lineages (<a href="#bib18" rid="bib18">Kumar et al., 2017</a>). We found that, compared with the RBD encoded in the reference genome, the Omicron RBD showed the highest ACE2-interaction enhancement with mice among all these mammals (<a href="https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC8702434/figure/fig6/" target="figure" rid-figpopup="fig6" rid-ob="ob-fig6" co-legend-rid="lgnd_fig6"><span>Fig. 6</span></a>
), suggesting that mice were the most likely host species to influence the evolution of the progenitor of Omicron.</p><!--fig ft0--><!--fig mode=article f1--><div id="fig6" co-legend-rid="lgnd_fig6"><a href="https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC8702434/figure/fig6/" target="figure" rid-figpopup="fig6" rid-ob="ob-fig6"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140398026668016"><a href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=8702434_gr6_lrg.jpg" target="tileshopwindow" rel="noopener"><img loading="lazy" alt="Fig. 6" title="Click on image to zoom" src="https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC8702434/bin/gr6_lrg.jpg"/></a></div><div id="lgnd_fig6"><!--caption a7--><p id="__p7">Predicted interaction enhancement with ACE2 of various mammalian species caused by pre-outbreak Omicron mutations in the spike protein. <em>P</em> values were given by two-tailed <em>t</em>-tests. The phylogenetic tree was constructed using TimeTree, in the unit of MY. MY, million years.</p></div></div></div></div><div id="sec3"><h2 id="sec3title">Discussion</h2><p id="p0125">In this study, we used the molecular spectrum of mutations of the SARS-CoV-2 Omicron variant to trace its proximal host origins. We found that the molecular spectrum of pre-outbreak Omicron mutations was inconsistent with the rapid accumulation of mutations in humans, but rather suggested a trajectory in which the progenitor of Omicron experienced a reverse zoonotic event from humans to mice sometime during the pandemic (most likely in mid-2020) and accumulated mutations in a mouse host for more than one year before jumping back to humans in late-2021. While evolving in mice, the progenitor of Omicron adapted to the mouse host by acquiring amino acid mutations in the spike protein that increased its binding affinity with mouse ACE2. In addition, mutations associated with immune escape also accumulated, which may also be a contributing factor in its rapid spread in humans.</p><p id="p0130">The B.1.1 variants showed the highest sequence similarities to Omicron in the GISAID database (where SARS-CoV-2-related viruses such as those isolated from bats were also deposited), strongly suggesting that the progenitor of Omicron jumped from humans, instead of another animal (such as bats), to mice. Nevertheless, it remains plausible that the MRCA of Omicron was an evolutionary product of recombination between a human variant (that provided the genomic sequence for the non-RBD region, or the “backbone”) and a variant from another species (that provided the RBD region). Although not highlighted in our results, note that we did test this possibility by BLAST searching against the GISAID database using Omicron’s backbone sequence. The top hits were again from the B.1.1 lineage, which differed from Omicron by 31 mutations, indicating that human SARS-CoV-2 variants reported to date could not provide a backbone for Omicron. Furthermore, the molecular spectrum of these 31 mutations in Omicron was also significantly different from the hSCV2 spectrum (<em>P</em> = 0.008, <em>G</em>-test; <a href="#appsec1" rid="appsec1">Fig. S5</a>), suggesting that these backbone mutations were not acquired in humans.</p><p id="p0135">While we show a phylogenetically long branch leading to the MRCA of current Omicron variants (<em>i.e.</em>, Branch O), it is worth noting that intermediate versions of Omicron were occasionally reported. For example, a SARS-CoV-2 variant (EPI_ISL_7136300) was collected by the Utah Public Health Laboratory on December 1<sup>st</sup>, 2021 which harbored 32 of the 45 pre-outbreak Omicron mutations. However, the 13 mutations absent in this variant clustered within residues 371–501 of the spike protein (<a href="#appsec1" rid="appsec1">Fig. S6</a>). The absence of these spike protein mutations thus suggested that this variant was a product of recombination between an Omicron variant and another SARS-CoV-2 variant, rather than a direct progenitor of Omicron. Considering the large number of pre-outbreak Omicron mutations (45) combined with the sparsity of intermediate versions identified to date, this long branch leading to Omicron in our phylogenetic reconstruction remains valid.</p><p id="p0140">Although we primarily focused on point mutations because the molecular spectrum of these mutations can reflect the host cellular environment (<a href="#bib6" rid="bib6">Deng et al., 2021</a>; <a href="#bib32" rid="bib32">Shan et al., 2021</a>), we also realized that the information of deletions and insertions could be used to infer the evolutionary trajectory of Omicron. For example, it was noted that Omicron harbored a nine-nucleotide insertion (GAGCCAGAA, encoding the peptide EPE) after residue 214 in the spike protein. This insertion is identical to the sequence of <em>TMEM245</em> in the human genome or that of ORF <em>S</em> in the human coronavirus hCoV-229E, which was used as evidence to support a human origin for Omicron (<a href="#bib39" rid="bib39">Venkatakrishnan et al., 2021</a>). However, we provide a simpler explanation for this insertion, namely that it was derived from an RNA fragment of ORF <em>N</em> in the SARS-CoV-2 genome (<a href="#appsec1" rid="appsec1">Fig. S7</a>), because the RNA abundance of ORF <em>N</em> is much higher than that of mRNA encoded by the human genome (<a href="#bib41" rid="bib41">Wei et al., 2021</a>). And this is especially so for ORF <em>N</em> due to the nested nature of the coronavirus genome and subgenomes (<a href="#bib16" rid="bib16">Kim et al., 2020</a>).</p><p id="p0145">The molecular docking-based predictions showed that the adaptation of Omicron to mice also promoted its adaptation to other species, such as humans, camels, and goats, via stronger RBD-ACE2 interaction (<a href="https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC8702434/figure/fig6/" target="figure" rid-figpopup="fig6" rid-ob="ob-fig6" co-legend-rid="lgnd_fig6"><span>Fig. 6</span></a>). Such a “pleiotropic effect” of mutations was likely caused by structural similarity of ACE2 across species, and indicates that once a SARS-CoV-2 variant acquires the capacity to infect a new host, it can accumulate mutations in this new animal reservoir and becomes transmittable to another host. This “chain reaction” of host jumping could potentially lead to remarkably high diversity in the adaptation to ACE2 from various host species. Consistent with this possibility, numerous mutations were identified in the spike protein of SARS-CoV-2 RNA fragment amplified from wastewater samples (<a href="#bib34" rid="bib34">Smyth et al., 2021</a>).</p><p id="p0150">Humans represent the largest known reservoir of SARS-CoV-2, and frequently come in contact with other animals, including livestock animals, pets, or wild animals that invade homes searching for food and shelter. Given the ability of SARS-CoV-2 to jump across various species, it appears likely that global populations will face additional animal-derived variants until the pandemic is well under control. Our study thus emphasizes the need for viral surveillance and sequencing in animals, especially those in close contact with humans. Furthermore, computational characterization of the spike RBD in animals and identification of their potentials to interact with human ACE2 will likely help to prevent future outbreaks of dangerous SARS-CoV-2 variants.</p></div><div id="sec4"><h2 id="sec4title">Materials and methods</h2><div id="sec4.1"><h3 id="sec4.1title">Identification of pre-outbreak and post-outbreak Omicron mutations</h3><p id="p0155">Genomic sequences of 695 SARS-CoV-2 Omicron variants were downloaded from GISAID (<a href="https://www.gisaid.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=8702434&amp;issue-id=396485&amp;journal-id=3815&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://www.gisaid.org/</a>) on December 7th, 2021. The reference genome of SARS-CoV-2 (EPI_ISL_402125) and two variants in the B.1.1 lineage (EPI_ISL_698296 and EPI_ISL_493480) were also downloaded from GISAID. The variants from the B.1.1 lineage were chosen because they showed the highest sequence similarities to the early Omicron samples in a BLASTn search.</p><p id="p0160">The genomes of SARS-CoV-2 variants were aligned by MUSCLE v3.8.1551 (<a href="#bib7" rid="bib7">Edgar, 2004</a>). The phylogenetic tree and ancestral sequences were reconstructed using FastML v3.11 (<a href="#bib1" rid="bib1">Ashkenazy et al., 2012</a>) with default parameters. The single-nucleotide substitutions obtained by the most recent common ancestor (MRCA) of Omicron variants after its divergence from the B.1.1 lineage were defined as pre-outbreak Omicron mutations. To detect the post-outbreak Omicron mutations, the sequences of 695 Omicron variants were aligned to the Omicron’s MRCA sequence, and sequences with &gt;10 single-nucleotide substitutions were discarded. The single-nucleotide substitutions detected in at least two variants were defined as the post-outbreak Omicron mutations.</p><p id="p0165">The numbers of synonymous and nonsynonymous sites in ORF <em>S</em> of SARS-CoV-2 were estimated by PAML in a previous study (<a href="#bib41" rid="bib41">Wei et al., 2021</a>). Briefly, <em>d</em>
<sub>N</sub> was calculated as the ratio between the number of nonsynonymous mutations and the number of nonsynonymous sites, while <em>d</em>
<sub>S</sub> was calculated as the ratio between the number of synonymous mutations and the number of synonymous sites.</p></div><div id="sec4.2"><h3 id="sec4.2title">Comparison between the sequence evolutionary rate of Omicron and other SARS-CoV-2 variants</h3><p id="p0170">A total of 764 variant sequences were randomly sampled from patient-related SARS-CoV-2 genomic sequences deposited at GISAID, one variant each day since COVID-19 outbreak. The progenitors of other four VOCs (Alpha, Beta, Gamma, and Delta) were retrieved from Nextstrain (<a href="https://nextstrain.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=8702434&amp;issue-id=396485&amp;journal-id=3815&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://nextstrain.org/</a>) (<a href="#bib9" rid="bib9">Hadfield et al., 2018</a>). Single-nucleotide substitutions (relative to the reference genome) of each variant were defined as the mutations acquired by the SARS-CoV-2 variant. The single-nucleotide base substitutions of three chronically infected patients were retrieved from two previous studies (<a href="#bib15" rid="bib15">Kemp et al., 2021</a>; <a href="#bib37" rid="bib37">Truong et al., 2021</a>). The mutations with allele frequency &gt; 50% on the final monitored day were used to count mutations that accumulated in a chronically infected patient.</p><p id="p0175">We performed a resampling test to estimate the statistical significance. Specifically, we randomly sampled 45 mutations from the 6986 point mutations identified in a previous study from the 34,853 high-quality sequences of SARS-CoV-2 variants isolated from patients worldwide (<a href="#bib32" rid="bib32">Shan et al., 2021</a>). This operation was repeated 100 times <em>in silico</em>.</p></div><div id="sec4.3"><h3 id="sec4.3title">Characterization of molecular spectra of mutations</h3><p id="p0180">Complete genomic sequences of 23 bovine coronavirus (<em>Betacoronavirus 1</em>), 13 canine coronavirus (<em>Alphacoronavirus 1</em>), 54 feline coronavirus (<em>Alphacoronavirus 1</em>), 17 murine hepatitis virus (<em>Murine coronavirus</em>), and 110 porcine deltacoronavirus (<em>Coronavirus HKU15</em>) were downloaded from National Center for Biotechnology Information (NCBI) Virus database (<a href="https://www.ncbi.nlm.nih.gov/labs/virus/vssi/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=8702434&amp;issue-id=396485&amp;journal-id=3815&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://www.ncbi.nlm.nih.gov/labs/virus/vssi/</a>) (<a href="#bib11" rid="bib11">Hatcher et al., 2017</a>), querying the hosts as <em>Bos taurus</em> (cattle), <em>Canis lupus familiaris</em> (dogs), <em>Felis catus</em> (cats)<em>, Mus musculus</em> (mice)<em>,</em> and <em>Sus scrofa</em> (pigs), respectively (<a href="#appsec1" rid="appsec1">Table S1</a>). These coronaviruses were chosen from coronaviruses recorded in the International Committee on Taxonomy of Viruses (<a href="https://talk.ictvonline.org/taxonomy/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=8702434&amp;issue-id=396485&amp;journal-id=3815&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://talk.ictvonline.org/taxonomy/</a>). All mammalian coronaviruses were retrieved, and only those with at least ten reported sequences were used, to ensure accurate estimation of the molecular spectrum. If there were multiple coronaviruses infecting the same host species, we randomly chose one for the subsequent analyses. The molecular spectra of accumulated mutations in the coronaviruses that infected bats, camels, or humans were retrieved from a previous study (<a href="#bib32" rid="bib32">Shan, et al., 2021</a>).</p><p id="p0185">The virus genome sequences were aligned by MUSCLE, and the phylogenetic trees and ancestral sequences were reconstructed using FastML. Since the roots of these phylogenetic trees were not readily identified, we kept only external branches to ensure the correction direction of base substitutions (<em>e.g.</em>, C&gt;U vs. U&gt;C). For the sake of clarity, we showed the molecular spectra for five branches with the largest number of mutations for each coronavirus species in the main text. The full data set is available in <a href="#appsec1" rid="appsec1">Table S1</a>.</p><p id="p0190">We characterized the molecular spectra of mutations accumulated in chronically infected patients, in which single-nucleotide base substitutions that ever occurred during the monitored period were counted. We downloaded the genomic sequences of four variants (EPI_ISL_5803018, EPI_ISL_3730369, EPI_ISL_4003132, and EPI_ISL_6260720), each from one of the other four VOCs (Alpha, Beta, Gamma, and Delta, respectively), to estimate the molecular spectra of mutations accumulated in VOCs.</p></div><div id="sec4.4"><h3 id="sec4.4title">Principal component analyses</h3><p id="p0195">We performed principal component analysis (prcomp function in <em>R</em>) with the proportions of the 12 base-substitution types as the input, and then projected molecular spectra into a two-dimensional space according to the first two principal components. To define the borderlines of molecular spectra for each host species (<em>i.e.</em>, cattle, bats, dogs, cats, mice, pigs, or humans), we estimated the 95% confidence ellipses (stat_ellipse function in <em>R</em>) from the molecular spectra of these host species. The spectra of pre- and post-outbreak Omicron mutations were further projected into the same two-dimensional space.</p></div><div id="sec4.5"><h3 id="sec4.5title">Comparison of pre-outbreak Omicron mutations with mutations detected in SARS-COV-2 variants isolated from various mammalian hosts</h3><p id="p0200">We downloaded from GISIAD the genomic sequences of SARS-CoV-2 variants isolated from 21 mammalian hosts (<a href="#appsec1" rid="appsec1">Tables S2 and S3</a>): <em>Aonyx cinereus</em> (Asian small-clawed otter), <em>Arctictis binturong</em> (binturong), <em>Canis lupus familiaris</em> (dog), <em>Crocuta crocuta</em> (spotted hyena), <em>Felis catus</em> (cat), <em>Gorilla gorilla</em> (western gorilla), <em>Mus musculus</em> (mouse), <em>Mustela furo</em> (ferret), <em>Neovison vison</em> (American mink), <em>Odocoileus virginianus</em> (white-tailed deer), <em>Panthera leo</em> (lion), <em>Panthera tigris</em> (tiger), <em>Panthera uncia</em> (snow leopard), <em>Prionailurus bengalensis</em> (leopard cat), <em>Prionailurus viverrinus</em> (fishing cat), <em>Hippopotamus amphibius</em> (hippopotamus), <em>Manis javanica</em> (pangolin), <em>Mesocricetus auratus</em> (golden hamster), <em>Chlorocebus sabaeus</em> (green monkey), <em>Puma concolor</em> (puma), and the bats from genus <em>Rhinolophus</em>. BLASTx was performed to identify ORF <em>S</em> in each variant, and mutations relative to the reference SARS-CoV-2 genome were identified at the same time. Three species (<em>Mesocricetus auratus</em>, <em>Chlorocebus sabaeus</em>, and <em>Puma concolor</em>) were discarded because they harbored less than three single amino acid mutations. Amino acid mutation data from three additional viruses isolated from mice were retrieved from three studies (<a href="#bib22" rid="bib22">Leist et al., 2020</a>; <a href="#bib26" rid="bib26">Montagutelli et al., 2021</a>; <a href="#bib35" rid="bib35">Sun et al., 2021</a>).</p></div><div id="sec4.6"><h3 id="sec4.6title">Estimation of the binding affinity of RBD-ACE2 interaction by molecular docking</h3><p id="p0205">We extracted three-dimensional structures of the spike RBD and human ACE2 from the crystal structure (PDB: 6M0J) reported in a previous study (<a href="#bib21" rid="bib21">Lan et al., 2020</a>), and those of other representative mammalian ACE2 from the predicted models reported in a previous study (<a href="#bib20" rid="bib20">Lam et al., 2020</a>). The structure models of the Omicron RBD were generated using SWISS-MODEL (<a href="#bib40" rid="bib40">Waterhouse et al., 2018</a>), and those of other RBD variants were generated using PyMOL “mutagenesis” (<a href="https://pymol.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=8702434&amp;issue-id=396485&amp;journal-id=3815&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://pymol.org/</a>). The structure models of the RBD:ACE2 complex were generated by aligning against the reported complex structure of the corresponding species using PyMOL (<a href="#bib20" rid="bib20">Lam et al., 2020</a>; <a href="#bib21" rid="bib21">Lan et al., 2020</a>).</p><p id="p0210">We performed molecular docking following previous studies (<a href="#bib20" rid="bib20">Lam et al., 2020</a>; <a href="#bib31" rid="bib31">Rodrigues et al., 2020</a>). Briefly, we refined the three-dimensional models using default refinement protocols, and then estimated the HADDOCK scores for each RBD:ACE2 complex using HADDOCKv2.4 web server (<a href="#bib38" rid="bib38">van Zundert et al., 2016</a>). Docking results of each RBD-ACE2 variant pair were clustered, and the average HADDOCK score of the top cluster was reported for the RBD:ACE2 complex.</p></div></div><div id="sec6"><h2 id="sec6title">CRediT authorship contribution statement</h2><p id="p0220"><strong>Changshuo Wei</strong>: Data curation, Investigation, Writing - Original draft, Writing - Review &amp; Editing. <strong>Ke-Jia Shan</strong>: Data curation, Investigation, Writing - Original draft, Writing - Review &amp; Editing. <strong>Weiguang Wang</strong>: Data curation, Investigation, Writing - Review &amp; Editing. <strong>Shuya Zhang</strong>: Investigation. <strong>Qing Huan</strong>: Writing - Original draft, Writing - Review &amp; Editing, Supervision. <strong>Wenfeng Qian</strong>: Conceptualization, Writing - Original draft, Writing - Review &amp; Editing, Supervision, Funding acquisition.</p></div><div id="__sec1"><h2 id="__sec1title">Declaration of Competing Interest</h2><p id="p0225">The authors declare that they have no competing interests.</p></div><div id="ack0010"><h2 id="ack0010title">Acknowledgments</h2><p id="p0230">We thank Dr. Xionglei He from Sun Yat-sen University and Dr. Mingkun Li from Beijing Institute of Genomics CAS for discussion. We acknowledge the authors and laboratories for generating and submitting the sequences to GISAID Database on which this research is based. The list is detailed in <a href="#appsec1" rid="appsec1">Table S3</a>. This work was supported by grants from the National Natural Science Foundation of China (31922014).</p></div><div id="appsec1"><h2 id="appsec1title">Appendix A. Supplementary data</h2><!--/article/back/sec/--><p id="p0235">The following are the Supplementary data to this article:</p></div><div id="cebib0010"><h2 id="cebib0010title">References</h2><div id="reference-list"><ul><li id="bib1"><span id="sref1">Ashkenazy H., Penn O., Doron-Faigenboim A., Cohen O., Cannarozzi G., Zomer O., Pupko T. FastML: a web server for probabilistic reconstruction of ancestral sequences. <span><span>Nucleic Acids Res. </span>2012;<span>40</span>:W580–584.</span> <span>[<a href="https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC3394241/">PMC free article</a>]</span> [<a href="https://www.ncbi.nlm.nih.gov/pubmed/22661579" ref="reftype=pubmed&amp;article-id=8702434&amp;issue-id=396485&amp;journal-id=3815&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span>[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=FastML:+a+web+server+for+probabilistic+reconstruction+of+ancestral+sequences&amp;author=H.+Ashkenazy&amp;author=O.+Penn&amp;author=A.+Doron-Faigenboim&amp;author=O.+Cohen&amp;author=G.+Cannarozzi&amp;volume=40&amp;publication_year=2012&amp;pages=W580-584&amp;pmid=22661579&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8702434&amp;issue-id=396485&amp;journal-id=3815&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="bib2"><span id="sref2">Blanc V., Davidson N.O. APOBEC-1-mediated RNA editing. <span><span>Wiley Interdiscip. Rev. Syst. Biol. Med. </span>2010;<span>2</span>:594–602.</span> <span>[<a href="https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC3086428/">PMC free article</a>]</span> [<a href="https://www.ncbi.nlm.nih.gov/pubmed/20836050" ref="reftype=pubmed&amp;article-id=8702434&amp;issue-id=396485&amp;journal-id=3815&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span>[<a href="https://scholar.google.com/scholar_lookup?journal=Wiley+Interdiscip.+Rev.+Syst.+Biol.+Med.&amp;title=APOBEC-1-mediated+RNA+editing&amp;author=V.+Blanc&amp;author=N.O.+Davidson&amp;volume=2&amp;publication_year=2010&amp;pages=594-602&amp;pmid=20836050&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8702434&amp;issue-id=396485&amp;journal-id=3815&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="bib3"><span id="sref3">Callaway E. Heavily mutated Omicron variant puts scientists on alert. <span><span>Nature. </span>2021 doi: 10.1038/d41586-021-03552-w.</span> [<a href="https://www.ncbi.nlm.nih.gov/pubmed/34824381" ref="reftype=pubmed&amp;article-id=8702434&amp;issue-id=396485&amp;journal-id=3815&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="https://dx.doi.org/10.1038%2Fd41586-021-03552-w" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8702434&amp;issue-id=396485&amp;journal-id=3815&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span>[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Heavily+mutated+Omicron+variant+puts+scientists+on+alert&amp;author=E.+Callaway&amp;publication_year=2021&amp;doi=10.1038/d41586-021-03552-w&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8702434&amp;issue-id=396485&amp;journal-id=3815&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="bib4"><span id="sref4">Chandler J.C., Bevins S.N., Ellis J.W., Linder T.J., Tell R.M., Jenkins-Moore M., Root J.J., Lenoch J.B., Robbe-Austerman S., DeLiberto T.J., et al.  SARS-CoV-2 exposure in wild white-tailed deer (<em>Odocoileus virginianus</em>) <span><span>Proc. Natl. Acad. Sci. U. S. A. </span>2021;<span>118</span></span> <span>[<a href="https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC8617405/">PMC free article</a>]</span> [<a href="https://www.ncbi.nlm.nih.gov/pubmed/34732584" ref="reftype=pubmed&amp;article-id=8702434&amp;issue-id=396485&amp;journal-id=3815&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span>[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+U.+S.+A.&amp;title=SARS-CoV-2+exposure+in+wild+white-tailed+deer+(Odocoileus+virginianus)&amp;author=J.C.+Chandler&amp;author=S.N.+Bevins&amp;author=J.W.+Ellis&amp;author=T.J.+Linder&amp;author=R.M.+Tell&amp;volume=118&amp;publication_year=2021&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8702434&amp;issue-id=396485&amp;journal-id=3815&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="bib5"><span id="sref5">De Maio N., Walker C.R., Turakhia Y., Lanfear R., Corbett-Detig R., Goldman N. Mutation rates and selection on synonymous mutations in SARS-CoV-2. <span><span>Genome Biol. Evol. </span>2021;<span>13</span></span> <span>[<a href="https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC8135539/">PMC free article</a>]</span> [<a href="https://www.ncbi.nlm.nih.gov/pubmed/33895815" ref="reftype=pubmed&amp;article-id=8702434&amp;issue-id=396485&amp;journal-id=3815&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span>[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Biol.+Evol.&amp;title=Mutation+rates+and+selection+on+synonymous+mutations+in+SARS-CoV-2&amp;author=N.+De+Maio&amp;author=C.R.+Walker&amp;author=Y.+Turakhia&amp;author=R.+Lanfear&amp;author=R.+Corbett-Detig&amp;volume=13&amp;publication_year=2021&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8702434&amp;issue-id=396485&amp;journal-id=3815&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="bib6"><span id="sref6">Deng S., Xing K., He X. Mutation signatures inform the natural host of SARS-CoV-2. <span><span>Natl. Sci. Rev. </span>2021 doi: 10.1093/nsr/nwab220.</span> [<a href="https://dx.doi.org/10.1093%2Fnsr%2Fnwab220" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8702434&amp;issue-id=396485&amp;journal-id=3815&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span>[<a href="https://scholar.google.com/scholar_lookup?journal=Natl.+Sci.+Rev.&amp;title=Mutation+signatures+inform+the+natural+host+of+SARS-CoV-2&amp;author=S.+Deng&amp;author=K.+Xing&amp;author=X.+He&amp;publication_year=2021&amp;doi=10.1093/nsr/nwab220&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8702434&amp;issue-id=396485&amp;journal-id=3815&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="bib7"><span id="sref7">Edgar R.C. MUSCLE: A multiple sequence alignment method with reduced time and space complexity. <span><span>BMC Bioinformatics. </span>2004;<span>5</span>:113.</span> <span>[<a href="https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC517706/">PMC free article</a>]</span> [<a href="https://www.ncbi.nlm.nih.gov/pubmed/15318951" ref="reftype=pubmed&amp;article-id=8702434&amp;issue-id=396485&amp;journal-id=3815&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span>[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Bioinformatics&amp;title=MUSCLE:+A+multiple+sequence+alignment+method+with+reduced+time+and+space+complexity&amp;author=R.C.+Edgar&amp;volume=5&amp;publication_year=2004&amp;pages=113&amp;pmid=15318951&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8702434&amp;issue-id=396485&amp;journal-id=3815&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="bib8"><span id="sref8">Gu H., Chen Q., Yang G., He L., Fan H., Deng Y.Q., Wang Y., Teng Y., Zhao Z., Cui Y., et al.  Adaptation of SARS-CoV-2 in BALB/c mice for testing vaccine efficacy. <span><span>Science. </span>2020;<span>369</span>:1603–1607.</span> <span>[<a href="https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC7574913/">PMC free article</a>]</span> [<a href="https://www.ncbi.nlm.nih.gov/pubmed/32732280" ref="reftype=pubmed&amp;article-id=8702434&amp;issue-id=396485&amp;journal-id=3815&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span>[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Adaptation+of+SARS-CoV-2+in+BALB/c+mice+for+testing+vaccine+efficacy&amp;author=H.+Gu&amp;author=Q.+Chen&amp;author=G.+Yang&amp;author=L.+He&amp;author=H.+Fan&amp;volume=369&amp;publication_year=2020&amp;pages=1603-1607&amp;pmid=32732280&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8702434&amp;issue-id=396485&amp;journal-id=3815&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="bib9"><span id="sref9">Hadfield J., Megill C., Bell S.M., Huddleston J., Potter B., Callender C., Sagulenko P., Bedford T., Neher R.A. Nextstrain: real-time tracking of pathogen evolution. <span><span>Bioinformatics. </span>2018;<span>34</span>:4121–4123.</span> <span>[<a href="https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC6247931/">PMC free article</a>]</span> [<a href="https://www.ncbi.nlm.nih.gov/pubmed/29790939" ref="reftype=pubmed&amp;article-id=8702434&amp;issue-id=396485&amp;journal-id=3815&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span>[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Nextstrain:+real-time+tracking+of+pathogen+evolution&amp;author=J.+Hadfield&amp;author=C.+Megill&amp;author=S.M.+Bell&amp;author=J.+Huddleston&amp;author=B.+Potter&amp;volume=34&amp;publication_year=2018&amp;pages=4121-4123&amp;pmid=29790939&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8702434&amp;issue-id=396485&amp;journal-id=3815&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="bib10"><span id="sref10">Harris R.S., Dudley J.P. APOBECs and virus restriction. <span><span>Virology. </span>2015;<span>479-480</span>:131–145.</span> <span>[<a href="https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC4424171/">PMC free article</a>]</span> [<a href="https://www.ncbi.nlm.nih.gov/pubmed/25818029" ref="reftype=pubmed&amp;article-id=8702434&amp;issue-id=396485&amp;journal-id=3815&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span>[<a href="https://scholar.google.com/scholar_lookup?journal=Virology&amp;title=APOBECs+and+virus+restriction&amp;author=R.S.+Harris&amp;author=J.P.+Dudley&amp;volume=479-480&amp;publication_year=2015&amp;pages=131-145&amp;pmid=25818029&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8702434&amp;issue-id=396485&amp;journal-id=3815&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="bib11"><span id="sref11">Hatcher E.L., Zhdanov S.A., Bao Y., Blinkova O., Nawrocki E.P., Ostapchuck Y., Schaffer A.A., Brister J.R. Virus variation resource – improved response to emergent viral outbreaks. <span><span>Nucleic Acids Res. </span>2017;<span>45</span>:D482–D490.</span> <span>[<a href="https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC5210549/">PMC free article</a>]</span> [<a href="https://www.ncbi.nlm.nih.gov/pubmed/27899678" ref="reftype=pubmed&amp;article-id=8702434&amp;issue-id=396485&amp;journal-id=3815&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span>[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=Virus+variation+resource+–+improved+response+to+emergent+viral+outbreaks&amp;author=E.L.+Hatcher&amp;author=S.A.+Zhdanov&amp;author=Y.+Bao&amp;author=O.+Blinkova&amp;author=E.P.+Nawrocki&amp;volume=45&amp;publication_year=2017&amp;pages=D482-D490&amp;pmid=27899678&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8702434&amp;issue-id=396485&amp;journal-id=3815&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="bib12"><span id="sref12">Huang K., Zhang Y., Hui X., Zhao Y., Gong W., Wang T., Zhang S., Yang Y., Deng F., Zhang Q., et al.  Q493K and Q498H substitutions in Spike promote adaptation of SARS-CoV-2 in mice. <span><span>EBioMedicine. </span>2021;<span>67</span>:103381.</span> <span>[<a href="https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC8118724/">PMC free article</a>]</span> [<a href="https://www.ncbi.nlm.nih.gov/pubmed/33993052" ref="reftype=pubmed&amp;article-id=8702434&amp;issue-id=396485&amp;journal-id=3815&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span>[<a href="https://scholar.google.com/scholar_lookup?journal=EBioMedicine&amp;title=Q493K+and+Q498H+substitutions+in+Spike+promote+adaptation+of+SARS-CoV-2+in+mice&amp;author=K.+Huang&amp;author=Y.+Zhang&amp;author=X.+Hui&amp;author=Y.+Zhao&amp;author=W.+Gong&amp;volume=67&amp;publication_year=2021&amp;pages=103381&amp;pmid=33993052&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8702434&amp;issue-id=396485&amp;journal-id=3815&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="bib13"><span id="sref13">Huang Y., Yang C., Xu X.F., Xu W., Liu S.W. Structural and functional properties of SARS-CoV-2 spike protein: potential antivirus drug development for COVID-19. <span><span>Acta. Pharmacol. Sin. </span>2020;<span>41</span>:1141–1149.</span> <span>[<a href="https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC7396720/">PMC free article</a>]</span> [<a href="https://www.ncbi.nlm.nih.gov/pubmed/32747721" ref="reftype=pubmed&amp;article-id=8702434&amp;issue-id=396485&amp;journal-id=3815&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span>[<a href="https://scholar.google.com/scholar_lookup?journal=Acta.+Pharmacol.+Sin.&amp;title=Structural+and+functional+properties+of+SARS-CoV-2+spike+protein:+potential+antivirus+drug+development+for+COVID-19&amp;author=Y.+Huang&amp;author=C.+Yang&amp;author=X.F.+Xu&amp;author=W.+Xu&amp;author=S.W.+Liu&amp;volume=41&amp;publication_year=2020&amp;pages=1141-1149&amp;pmid=32747721&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8702434&amp;issue-id=396485&amp;journal-id=3815&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="bib14"><span id="sref14">Kastritis P.L., Bonvin A.M. Are scoring functions in protein-protein docking ready to predict interactomes? Clues from a novel binding affinity benchmark. <span><span>J. Proteome Res. </span>2010;<span>9</span>:2216–2225.</span> [<a href="https://www.ncbi.nlm.nih.gov/pubmed/20329755" ref="reftype=pubmed&amp;article-id=8702434&amp;issue-id=396485&amp;journal-id=3815&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span>[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Proteome+Res.&amp;title=Are+scoring+functions+in+protein-protein+docking+ready+to+predict+interactomes?+Clues+from+a+novel+binding+affinity+benchmark&amp;author=P.L.+Kastritis&amp;author=A.M.+Bonvin&amp;volume=9&amp;publication_year=2010&amp;pages=2216-2225&amp;pmid=20329755&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8702434&amp;issue-id=396485&amp;journal-id=3815&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="bib15"><span id="sref15">Kemp S.A., Collier D.A., Datir R.P., Ferreira I., Gayed S., Jahun A., Hosmillo M., Rees-Spear C., Mlcochova P., Lumb I.U., et al.  SARS-CoV-2 evolution during treatment of chronic infection. <span><span>Nature. </span>2021;<span>592</span>:277–282.</span> <span>[<a href="https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC7610568/">PMC free article</a>]</span> [<a href="https://www.ncbi.nlm.nih.gov/pubmed/33545711" ref="reftype=pubmed&amp;article-id=8702434&amp;issue-id=396485&amp;journal-id=3815&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span>[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=SARS-CoV-2+evolution+during+treatment+of+chronic+infection&amp;author=S.A.+Kemp&amp;author=D.A.+Collier&amp;author=R.P.+Datir&amp;author=I.+Ferreira&amp;author=S.+Gayed&amp;volume=592&amp;publication_year=2021&amp;pages=277-282&amp;pmid=33545711&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8702434&amp;issue-id=396485&amp;journal-id=3815&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="bib16"><span id="sref16">Kim D., Lee J.Y., Yang J.S., Kim J.W., Kim V.N., Chang H. The architecture of SARS-CoV-2 transcriptome. <span><span>Cell. </span>2020;<span>181</span>:914–921.</span> <span>[<a href="https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC7179501/">PMC free article</a>]</span> [<a href="https://www.ncbi.nlm.nih.gov/pubmed/32330414" ref="reftype=pubmed&amp;article-id=8702434&amp;issue-id=396485&amp;journal-id=3815&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span>[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=The+architecture+of+SARS-CoV-2+transcriptome&amp;author=D.+Kim&amp;author=J.Y.+Lee&amp;author=J.S.+Yang&amp;author=J.W.+Kim&amp;author=V.N.+Kim&amp;volume=181&amp;publication_year=2020&amp;pages=914-921&amp;pmid=32330414&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8702434&amp;issue-id=396485&amp;journal-id=3815&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="bib17"><span id="sref17">Kong Q., Lin C.L. Oxidative damage to RNA: Mechanisms, consequences, and diseases. <span><span>Cell Mol. Life Sci. </span>2010;<span>67</span>:1817–1829.</span> <span>[<a href="https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC3010397/">PMC free article</a>]</span> [<a href="https://www.ncbi.nlm.nih.gov/pubmed/20148281" ref="reftype=pubmed&amp;article-id=8702434&amp;issue-id=396485&amp;journal-id=3815&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span>[<a href="https://scholar.google.com/scholar_lookup?journal=Cell+Mol.+Life+Sci.&amp;title=Oxidative+damage+to+RNA:+Mechanisms,+consequences,+and+diseases&amp;author=Q.+Kong&amp;author=C.L.+Lin&amp;volume=67&amp;publication_year=2010&amp;pages=1817-1829&amp;pmid=20148281&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8702434&amp;issue-id=396485&amp;journal-id=3815&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="bib18"><span id="sref18">Kumar S., Stecher G., Suleski M., Hedges S.B. TimeTree: A resource for timelines, timetrees, and divergence times. <span><span>Mol. Biol. Evol. </span>2017;<span>34</span>:1812–1819.</span> [<a href="https://www.ncbi.nlm.nih.gov/pubmed/28387841" ref="reftype=pubmed&amp;article-id=8702434&amp;issue-id=396485&amp;journal-id=3815&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span>[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Biol.+Evol.&amp;title=TimeTree:+A+resource+for+timelines,+timetrees,+and+divergence+times&amp;author=S.+Kumar&amp;author=G.+Stecher&amp;author=M.+Suleski&amp;author=S.B.+Hedges&amp;volume=34&amp;publication_year=2017&amp;pages=1812-1819&amp;pmid=28387841&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8702434&amp;issue-id=396485&amp;journal-id=3815&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="bib19"><span id="sref19">Kupferschmidt K. Where did ‘weird’ Omicron come from? <span><span>Science. </span>2021;<span>374</span>:1179.</span> [<a href="https://www.ncbi.nlm.nih.gov/pubmed/34855502" ref="reftype=pubmed&amp;article-id=8702434&amp;issue-id=396485&amp;journal-id=3815&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span>[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Where+did+‘weird’+Omicron+come+from?&amp;author=K.+Kupferschmidt&amp;volume=374&amp;publication_year=2021&amp;pages=1179&amp;pmid=34855502&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8702434&amp;issue-id=396485&amp;journal-id=3815&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="bib20"><span id="sref20">Lam S.D., Bordin N., Waman V.P., Scholes H.M., Ashford P., Sen N., van Dorp L., Rauer C., Dawson N.L., Pang C.S.M., et al.  SARS-CoV-2 spike protein predicted to form complexes with host receptor protein orthologues from a broad range of mammals. <span><span>Sci. Rep. </span>2020;<span>10</span>:16471.</span> <span>[<a href="https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC7536205/">PMC free article</a>]</span> [<a href="https://www.ncbi.nlm.nih.gov/pubmed/33020502" ref="reftype=pubmed&amp;article-id=8702434&amp;issue-id=396485&amp;journal-id=3815&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span>[<a href="https://scholar.google.com/scholar_lookup?journal=Sci.+Rep.&amp;title=SARS-CoV-2+spike+protein+predicted+to+form+complexes+with+host+receptor+protein+orthologues+from+a+broad+range+of+mammals&amp;author=S.D.+Lam&amp;author=N.+Bordin&amp;author=V.P.+Waman&amp;author=H.M.+Scholes&amp;author=P.+Ashford&amp;volume=10&amp;publication_year=2020&amp;pages=16471&amp;pmid=33020502&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8702434&amp;issue-id=396485&amp;journal-id=3815&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="bib21"><span id="sref21">Lan J., Ge J., Yu J., Shan S., Zhou H., Fan S., Zhang Q., Shi X., Wang Q., Zhang L., et al.  Structure of the SARS-CoV-2 spike receptor-binding domain bound to the ACE2 receptor. <span><span>Nature. </span>2020;<span>581</span>:215–220.</span> [<a href="https://www.ncbi.nlm.nih.gov/pubmed/32225176" ref="reftype=pubmed&amp;article-id=8702434&amp;issue-id=396485&amp;journal-id=3815&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span>[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Structure+of+the+SARS-CoV-2+spike+receptor-binding+domain+bound+to+the+ACE2+receptor&amp;author=J.+Lan&amp;author=J.+Ge&amp;author=J.+Yu&amp;author=S.+Shan&amp;author=H.+Zhou&amp;volume=581&amp;publication_year=2020&amp;pages=215-220&amp;pmid=32225176&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8702434&amp;issue-id=396485&amp;journal-id=3815&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="bib22"><span id="sref22">Leist S.R., Dinnon K.H., 3rd, Schafer A., Tse L.V., Okuda K., Hou Y.J., West A., Edwards C.E., Sanders W., Fritch E.J., et al.  A mouse-adapted SARS-CoV-2 induces acute lung injury and mortality in standard laboratory mice. <span><span>Cell. </span>2020;<span>183</span>:1070–1085.</span> <span>[<a href="https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC7510428/">PMC free article</a>]</span> [<a href="https://www.ncbi.nlm.nih.gov/pubmed/33031744" ref="reftype=pubmed&amp;article-id=8702434&amp;issue-id=396485&amp;journal-id=3815&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span>[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=A+mouse-adapted+SARS-CoV-2+induces+acute+lung+injury+and+mortality+in+standard+laboratory+mice&amp;author=S.R.+Leist&amp;author=K.H.+Dinnon&amp;author=A.+Schafer&amp;author=L.V.+Tse&amp;author=K.+Okuda&amp;volume=183&amp;publication_year=2020&amp;pages=1070-1085&amp;pmid=33031744&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8702434&amp;issue-id=396485&amp;journal-id=3815&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="bib23"><span id="sref23">Li Q., Nie J., Wu J., Zhang L., Ding R., Wang H., Zhang Y., Li T., Liu S., Zhang M., et al.  SARS-CoV-2 501Y.V2 variants lack higher infectivity but do have immune escape. <span><span>Cell. </span>2021;<span>184</span>:2362–2371.</span> <span>[<a href="https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC7901273/">PMC free article</a>]</span> [<a href="https://www.ncbi.nlm.nih.gov/pubmed/33735608" ref="reftype=pubmed&amp;article-id=8702434&amp;issue-id=396485&amp;journal-id=3815&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span>[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=SARS-CoV-2+501Y.V2+variants+lack+higher+infectivity+but+do+have+immune+escape&amp;author=Q.+Li&amp;author=J.+Nie&amp;author=J.+Wu&amp;author=L.+Zhang&amp;author=R.+Ding&amp;volume=184&amp;publication_year=2021&amp;pages=2362-2371&amp;pmid=33735608&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8702434&amp;issue-id=396485&amp;journal-id=3815&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="bib24"><span id="sref24">Li Z., Wu J., Deleo C.J. RNA damage and surveillance under oxidative stress. <span><span>IUBMB Life. </span>2006;<span>58</span>:581–588.</span> [<a href="https://www.ncbi.nlm.nih.gov/pubmed/17050375" ref="reftype=pubmed&amp;article-id=8702434&amp;issue-id=396485&amp;journal-id=3815&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span>[<a href="https://scholar.google.com/scholar_lookup?journal=IUBMB+Life&amp;title=RNA+damage+and+surveillance+under+oxidative+stress&amp;author=Z.+Li&amp;author=J.+Wu&amp;author=C.J.+Deleo&amp;volume=58&amp;publication_year=2006&amp;pages=581-588&amp;pmid=17050375&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8702434&amp;issue-id=396485&amp;journal-id=3815&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="bib25"><span id="sref25">Martinez-Flores D., Zepeda-Cervantes J., Cruz-Resendiz A., Aguirre-Sampieri S., Sampieri A., Vaca L. SARS-CoV-2 vaccines based on the spike glycoprotein and implications of new viral variants. <span><span>Front Immunol. </span>2021;<span>12</span>:701501.</span> <span>[<a href="https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC8311925/">PMC free article</a>]</span> [<a href="https://www.ncbi.nlm.nih.gov/pubmed/34322129" ref="reftype=pubmed&amp;article-id=8702434&amp;issue-id=396485&amp;journal-id=3815&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span>[<a href="https://scholar.google.com/scholar_lookup?journal=Front+Immunol&amp;title=SARS-CoV-2+vaccines+based+on+the+spike+glycoprotein+and+implications+of+new+viral+variants&amp;author=D.+Martinez-Flores&amp;author=J.+Zepeda-Cervantes&amp;author=A.+Cruz-Resendiz&amp;author=S.+Aguirre-Sampieri&amp;author=A.+Sampieri&amp;volume=12&amp;publication_year=2021&amp;pages=701501&amp;pmid=34322129&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8702434&amp;issue-id=396485&amp;journal-id=3815&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="bib26"><span id="sref26">Montagutelli X., Prot M., Jouvion G., Levillayer L., Conquet L., Reyes-Gomez E., Donati F., Albert M., van der Werf S., Jaubert J., et al.  A mouse-adapted SARS-CoV-2 strain replicating in standard laboratory mice. <span><span>bioRxiv. </span>2021 doi: 10.1101/2021.07.10.451880.</span> [<a href="https://dx.doi.org/10.1101%2F2021.07.10.451880" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8702434&amp;issue-id=396485&amp;journal-id=3815&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span>[<a href="https://scholar.google.com/scholar_lookup?journal=bioRxiv&amp;title=A+mouse-adapted+SARS-CoV-2+strain+replicating+in+standard+laboratory+mice&amp;author=X.+Montagutelli&amp;author=M.+Prot&amp;author=G.+Jouvion&amp;author=L.+Levillayer&amp;author=L.+Conquet&amp;publication_year=2021&amp;doi=10.1101/2021.07.10.451880&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8702434&amp;issue-id=396485&amp;journal-id=3815&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="bib27"><span id="sref27">Nelson G., Buzko O., Spilman P., Niazi K., Rabizadeh S., Soon-Shiong P. Molecular dynamic simulation reveals E484K mutation enhances spike RBD-ACE2 affinity and the combination of E484K, K417N and N501Y mutations (501Y.V2 variant) induces conformational change greater than N501Y mutant alone, potentially resulting in an escape mutant. <span><span>bioRxiv. </span>2021 doi: 10.1101/2021.01.13.426558.</span> [<a href="https://dx.doi.org/10.1101%2F2021.01.13.426558" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8702434&amp;issue-id=396485&amp;journal-id=3815&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span>[<a href="https://scholar.google.com/scholar_lookup?journal=bioRxiv&amp;title=Molecular+dynamic+simulation+reveals+E484K+mutation+enhances+spike+RBD-ACE2+affinity+and+the+combination+of+E484K,+K417N+and+N501Y+mutations+(501Y.V2+variant)+induces+conformational+change+greater+than+N501Y+mutant+alone,+potentially+resulting+in+an+escape+mutant&amp;author=G.+Nelson&amp;author=O.+Buzko&amp;author=P.+Spilman&amp;author=K.+Niazi&amp;author=S.+Rabizadeh&amp;publication_year=2021&amp;doi=10.1101/2021.01.13.426558&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8702434&amp;issue-id=396485&amp;journal-id=3815&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="bib28"><span id="sref28">Oude Munnink B.B., Sikkema R.S., Nieuwenhuijse D.F., Molenaar R.J., Munger E., Molenkamp R., van der Spek A., Tolsma P., Rietveld A., Brouwer M., et al.  Transmission of SARS-CoV-2 on mink farms between humans and mink and back to humans. <span><span>Science. </span>2021;<span>371</span>:172–177.</span> <span>[<a href="https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC7857398/">PMC free article</a>]</span> [<a href="https://www.ncbi.nlm.nih.gov/pubmed/33172935" ref="reftype=pubmed&amp;article-id=8702434&amp;issue-id=396485&amp;journal-id=3815&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span>[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Transmission+of+SARS-CoV-2+on+mink+farms+between+humans+and+mink+and+back+to+humans&amp;author=B.B.+Oude+Munnink&amp;author=R.S.+Sikkema&amp;author=D.F.+Nieuwenhuijse&amp;author=R.J.+Molenaar&amp;author=E.+Munger&amp;volume=371&amp;publication_year=2021&amp;pages=172-177&amp;pmid=33172935&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8702434&amp;issue-id=396485&amp;journal-id=3815&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="bib29"><span id="sref29">Panchin A.Y., Panchin Y.V. Excessive G-U transversions in novel allele variants in SARS-CoV-2 genomes. <span><span>PeerJ. </span>2020;<span>8</span></span> <span>[<a href="https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC7394058/">PMC free article</a>]</span> [<a href="https://www.ncbi.nlm.nih.gov/pubmed/33194341" ref="reftype=pubmed&amp;article-id=8702434&amp;issue-id=396485&amp;journal-id=3815&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span>[<a href="https://scholar.google.com/scholar_lookup?journal=PeerJ&amp;title=Excessive+G-U+transversions+in+novel+allele+variants+in+SARS-CoV-2+genomes&amp;author=A.Y.+Panchin&amp;author=Y.V.+Panchin&amp;volume=8&amp;publication_year=2020&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8702434&amp;issue-id=396485&amp;journal-id=3815&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="bib30"><span id="sref30">Ren W., Zhu Y., Wang Y., Shi H., Yu Y., Hu G., Feng F., Zhao X., Lan J., Wu J., et al.  Comparative analysis reveals the species-specific genetic determinants of ACE2 required for SARS-CoV-2 entry. <span><span>PLoS Pathog. </span>2021;<span>17</span></span> <span>[<a href="https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC7990223/">PMC free article</a>]</span> [<a href="https://www.ncbi.nlm.nih.gov/pubmed/33760889" ref="reftype=pubmed&amp;article-id=8702434&amp;issue-id=396485&amp;journal-id=3815&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span>[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Pathog&amp;title=Comparative+analysis+reveals+the+species-specific+genetic+determinants+of+ACE2+required+for+SARS-CoV-2+entry&amp;author=W.+Ren&amp;author=Y.+Zhu&amp;author=Y.+Wang&amp;author=H.+Shi&amp;author=Y.+Yu&amp;volume=17&amp;publication_year=2021&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8702434&amp;issue-id=396485&amp;journal-id=3815&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="bib31"><span id="sref31">Rodrigues J., Barrera-Vilarmau S., J M.C.T., Sorokina M., Seckel E., Kastritis P.L., Levitt M. Insights on cross-species transmission of SARS-CoV-2 from structural modeling. <span><span>PLoS Comput. Biol. </span>2020;<span>16</span></span> <span>[<a href="https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC7714162/">PMC free article</a>]</span> [<a href="https://www.ncbi.nlm.nih.gov/pubmed/33270653" ref="reftype=pubmed&amp;article-id=8702434&amp;issue-id=396485&amp;journal-id=3815&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span>[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Comput.+Biol.&amp;title=Insights+on+cross-species+transmission+of+SARS-CoV-2+from+structural+modeling&amp;author=J.+Rodrigues&amp;author=S.+Barrera-Vilarmau&amp;author=M.C.T.+J&amp;author=M.+Sorokina&amp;author=E.+Seckel&amp;volume=16&amp;publication_year=2020&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8702434&amp;issue-id=396485&amp;journal-id=3815&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="bib32"><span id="sref32">Shan K.J., Wei C., Wang Y., Huan Q., Qian W. Host-specific asymmetric accumulation of mutation types reveals that the origin of SARS-CoV-2 is consistent with a natural process. <span><span>The Innovation. </span>2021;<span>2</span>:100159.</span> <span>[<a href="https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC8405235/">PMC free article</a>]</span> [<a href="https://www.ncbi.nlm.nih.gov/pubmed/34485968" ref="reftype=pubmed&amp;article-id=8702434&amp;issue-id=396485&amp;journal-id=3815&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span>[<a href="https://scholar.google.com/scholar_lookup?journal=The+Innovation&amp;title=Host-specific+asymmetric+accumulation+of+mutation+types+reveals+that+the+origin+of+SARS-CoV-2+is+consistent+with+a+natural+process&amp;author=K.J.+Shan&amp;author=C.+Wei&amp;author=Y.+Wang&amp;author=Q.+Huan&amp;author=W.+Qian&amp;volume=2&amp;publication_year=2021&amp;pages=100159&amp;pmid=34485968&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8702434&amp;issue-id=396485&amp;journal-id=3815&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="bib33"><span id="sref33">Shu Y., McCauley J. GISAID: Global initiative on sharing all influenza data - from vision to reality. <span><span>Euro. Surveill. </span>2017;<span>22</span>:30494.</span> <span>[<a href="https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC5388101/">PMC free article</a>]</span> [<a href="https://www.ncbi.nlm.nih.gov/pubmed/28382917" ref="reftype=pubmed&amp;article-id=8702434&amp;issue-id=396485&amp;journal-id=3815&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span>[<a href="https://scholar.google.com/scholar_lookup?journal=Euro.+Surveill.&amp;title=GISAID:+Global+initiative+on+sharing+all+influenza+data+-+from+vision+to+reality&amp;author=Y.+Shu&amp;author=J.+McCauley&amp;volume=22&amp;publication_year=2017&amp;pages=30494&amp;pmid=28382917&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8702434&amp;issue-id=396485&amp;journal-id=3815&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="bib34"><span id="sref34">Smyth D.S., Trujillo M., Gregory D.A., Cheung K., Gao A., Graham M., Guan Y., Guldenpfennig C., Hoxie I., Kannoly S., et al.  Tracking cryptic SARS-CoV-2 lineages detected in NYC wastewater. <span><span>medRxiv. </span>2021 doi: 10.1101/2021.07.26.21261142.</span> [<a href="https://dx.doi.org/10.1101%2F2021.07.26.21261142" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8702434&amp;issue-id=396485&amp;journal-id=3815&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span>[<a href="https://scholar.google.com/scholar_lookup?journal=medRxiv&amp;title=Tracking+cryptic+SARS-CoV-2+lineages+detected+in+NYC+wastewater&amp;author=D.S.+Smyth&amp;author=M.+Trujillo&amp;author=D.A.+Gregory&amp;author=K.+Cheung&amp;author=A.+Gao&amp;publication_year=2021&amp;doi=10.1101/2021.07.26.21261142&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8702434&amp;issue-id=396485&amp;journal-id=3815&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="bib35"><span id="sref35">Sun S., Gu H., Cao L., Chen Q., Ye Q., Yang G., Li R.T., Fan H., Deng Y.Q., Song X., et al.  Characterization and structural basis of a lethal mouse-adapted SARS-CoV-2. <span><span>Nat. Commun. </span>2021;<span>12</span>:5654.</span> <span>[<a href="https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC8476561/">PMC free article</a>]</span> [<a href="https://www.ncbi.nlm.nih.gov/pubmed/34580297" ref="reftype=pubmed&amp;article-id=8702434&amp;issue-id=396485&amp;journal-id=3815&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span>[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Commun.&amp;title=Characterization+and+structural+basis+of+a+lethal+mouse-adapted+SARS-CoV-2&amp;author=S.+Sun&amp;author=H.+Gu&amp;author=L.+Cao&amp;author=Q.+Chen&amp;author=Q.+Ye&amp;volume=12&amp;publication_year=2021&amp;pages=5654&amp;pmid=34580297&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8702434&amp;issue-id=396485&amp;journal-id=3815&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="bib36"><span id="sref36">Telenti A., Arvin A., Corey L., Corti D., Diamond M.S., Garcia-Sastre A., Garry R.F., Holmes E.C., Pang P.S., Virgin H.W. After the pandemic: perspectives on the future trajectory of COVID-19. <span><span>Nature. </span>2021;<span>596</span>:495–504.</span> [<a href="https://www.ncbi.nlm.nih.gov/pubmed/34237771" ref="reftype=pubmed&amp;article-id=8702434&amp;issue-id=396485&amp;journal-id=3815&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span>[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=After+the+pandemic:+perspectives+on+the+future+trajectory+of+COVID-19&amp;author=A.+Telenti&amp;author=A.+Arvin&amp;author=L.+Corey&amp;author=D.+Corti&amp;author=M.S.+Diamond&amp;volume=596&amp;publication_year=2021&amp;pages=495-504&amp;pmid=34237771&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8702434&amp;issue-id=396485&amp;journal-id=3815&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="bib37"><span id="sref37">Truong T.T., Ryutov A., Pandey U., Yee R., Goldberg L., Bhojwani D., Aguayo-Hiraldo P., Pinsky B.A., Pekosz A., Shen L., et al.  Increased viral variants in children and young adults with impaired humoral immunity and persistent SARS-CoV-2 infection: A consecutive case series. <span><span>EBioMedicine. </span>2021;<span>67</span>:103355.</span> <span>[<a href="https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC8072072/">PMC free article</a>]</span> [<a href="https://www.ncbi.nlm.nih.gov/pubmed/33915337" ref="reftype=pubmed&amp;article-id=8702434&amp;issue-id=396485&amp;journal-id=3815&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span>[<a href="https://scholar.google.com/scholar_lookup?journal=EBioMedicine&amp;title=Increased+viral+variants+in+children+and+young+adults+with+impaired+humoral+immunity+and+persistent+SARS-CoV-2+infection:+A+consecutive+case+series&amp;author=T.T.+Truong&amp;author=A.+Ryutov&amp;author=U.+Pandey&amp;author=R.+Yee&amp;author=L.+Goldberg&amp;volume=67&amp;publication_year=2021&amp;pages=103355&amp;pmid=33915337&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8702434&amp;issue-id=396485&amp;journal-id=3815&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="bib38"><span id="sref38">van Zundert G.C.P., Rodrigues J., Trellet M., Schmitz C., Kastritis P.L., Karaca E., Melquiond A.S.J., van Dijk M., de Vries S.J., Bonvin A. The HADDOCK2.2 web server: User-friendly integrative modeling of biomolecular complexes. <span><span>J. Mol. Biol. </span>2016;<span>428</span>:720–725.</span> [<a href="https://www.ncbi.nlm.nih.gov/pubmed/26410586" ref="reftype=pubmed&amp;article-id=8702434&amp;issue-id=396485&amp;journal-id=3815&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span>[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Mol.+Biol.&amp;title=The+HADDOCK2.2+web+server:+User-friendly+integrative+modeling+of+biomolecular+complexes&amp;author=G.C.P.+van+Zundert&amp;author=J.+Rodrigues&amp;author=M.+Trellet&amp;author=C.+Schmitz&amp;author=P.L.+Kastritis&amp;volume=428&amp;publication_year=2016&amp;pages=720-725&amp;pmid=26410586&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8702434&amp;issue-id=396485&amp;journal-id=3815&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="bib39"><span id="sref39">Venkatakrishnan A., Anand P., Lenehan P., Suratekar R., Raghunathan B., Niesen M.J., Soundararajan V. Omicron variant of SARS-CoV-2 harbors a unique insertion mutation of putative viral or human genomic origin. <span><span>OSF Preprints. </span>2021 doi: 10.31219/osf.io/f7txy.</span> [<a href="https://dx.doi.org/10.31219%2Fosf.io%2Ff7txy" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8702434&amp;issue-id=396485&amp;journal-id=3815&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span>[<a href="https://scholar.google.com/scholar_lookup?journal=OSF+Preprints&amp;title=Omicron+variant+of+SARS-CoV-2+harbors+a+unique+insertion+mutation+of+putative+viral+or+human+genomic+origin&amp;author=A.+Venkatakrishnan&amp;author=P.+Anand&amp;author=P.+Lenehan&amp;author=R.+Suratekar&amp;author=B.+Raghunathan&amp;publication_year=2021&amp;doi=10.31219/osf.io/f7txy&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8702434&amp;issue-id=396485&amp;journal-id=3815&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="bib40"><span id="sref40">Waterhouse A., Bertoni M., Bienert S., Studer G., Tauriello G., Gumienny R., Heer F.T., de Beer T.A.P., Rempfer C., Bordoli L., et al.  SWISS-MODEL: homology modelling of protein structures and complexes. <span><span>Nucleic Acids Res. </span>2018;<span>46</span>:W296–W303.</span> <span>[<a href="https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC6030848/">PMC free article</a>]</span> [<a href="https://www.ncbi.nlm.nih.gov/pubmed/29788355" ref="reftype=pubmed&amp;article-id=8702434&amp;issue-id=396485&amp;journal-id=3815&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span>[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=SWISS-MODEL:+homology+modelling+of+protein+structures+and+complexes&amp;author=A.+Waterhouse&amp;author=M.+Bertoni&amp;author=S.+Bienert&amp;author=G.+Studer&amp;author=G.+Tauriello&amp;volume=46&amp;publication_year=2018&amp;pages=W296-W303&amp;pmid=29788355&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8702434&amp;issue-id=396485&amp;journal-id=3815&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="bib41"><span id="sref41">Wei C., Chen Y.M., Chen Y., Qian W. The missing expression level-evolutionary rate anticorrelation in viruses does not support protein function as a main constraint on sequence evolution. <span><span>Genome Biol. Evol. </span>2021;<span>13</span></span> <span>[<a href="https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC7989579/">PMC free article</a>]</span> [<a href="https://www.ncbi.nlm.nih.gov/pubmed/33713114" ref="reftype=pubmed&amp;article-id=8702434&amp;issue-id=396485&amp;journal-id=3815&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span>[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Biol.+Evol.&amp;title=The+missing+expression+level-evolutionary+rate+anticorrelation+in+viruses+does+not+support+protein+function+as+a+main+constraint+on+sequence+evolution&amp;author=C.+Wei&amp;author=Y.M.+Chen&amp;author=Y.+Chen&amp;author=W.+Qian&amp;volume=13&amp;publication_year=2021&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8702434&amp;issue-id=396485&amp;journal-id=3815&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="bib42"><span id="sref42">Wong L.Y.R., Zheng J., Wilhelmsen K., Li K., Ortiz M.E., Schnicker N.J., Pezzulo A.A., Szachowicz P.J., Klumpp K., Aswad F., et al.  Eicosanoid signaling as a therapeutic target in middle-aged mice with severe COVID-19. <span><span>bioRxiv. </span>2021 doi: 10.1101/2021.04.20.440676.</span> [<a href="https://dx.doi.org/10.1101%2F2021.04.20.440676" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8702434&amp;issue-id=396485&amp;journal-id=3815&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span>[<a href="https://scholar.google.com/scholar_lookup?journal=bioRxiv&amp;title=Eicosanoid+signaling+as+a+therapeutic+target+in+middle-aged+mice+with+severe+COVID-19&amp;author=L.Y.R.+Wong&amp;author=J.+Zheng&amp;author=K.+Wilhelmsen&amp;author=K.+Li&amp;author=M.E.+Ortiz&amp;publication_year=2021&amp;doi=10.1101/2021.04.20.440676&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8702434&amp;issue-id=396485&amp;journal-id=3815&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="bib43"><span id="sref43">Wu F., Zhao S., Yu B., Chen Y.M., Wang W., Song Z.G., Hu Y., Tao Z.W., Tian J.H., Pei Y.Y., et al.  A new coronavirus associated with human respiratory disease in China. <span><span>Nature. </span>2020;<span>579</span>:265–269.</span> <span>[<a href="https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC7094943/">PMC free article</a>]</span> [<a href="https://www.ncbi.nlm.nih.gov/pubmed/32015508" ref="reftype=pubmed&amp;article-id=8702434&amp;issue-id=396485&amp;journal-id=3815&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span>[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A+new+coronavirus+associated+with+human+respiratory+disease+in+China&amp;author=F.+Wu&amp;author=S.+Zhao&amp;author=B.+Yu&amp;author=Y.M.+Chen&amp;author=W.+Wang&amp;volume=579&amp;publication_year=2020&amp;pages=265-269&amp;pmid=32015508&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8702434&amp;issue-id=396485&amp;journal-id=3815&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="bib44"><span id="sref44">Wu S., Zhong G., Zhang J., Shuai L., Zhang Z., Wen Z., Wang B., Zhao Z., Song X., Chen Y., et al.  A single dose of an adenovirus-vectored vaccine provides protection against SARS-CoV-2 challenge. <span><span>Nat. Commun. </span>2020;<span>11</span>:4081.</span> <span>[<a href="https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC7427994/">PMC free article</a>]</span> [<a href="https://www.ncbi.nlm.nih.gov/pubmed/32796842" ref="reftype=pubmed&amp;article-id=8702434&amp;issue-id=396485&amp;journal-id=3815&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span>[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Commun.&amp;title=A+single+dose+of+an+adenovirus-vectored+vaccine+provides+protection+against+SARS-CoV-2+challenge&amp;author=S.+Wu&amp;author=G.+Zhong&amp;author=J.+Zhang&amp;author=L.+Shuai&amp;author=Z.+Zhang&amp;volume=11&amp;publication_year=2020&amp;pages=4081&amp;pmid=32796842&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8702434&amp;issue-id=396485&amp;journal-id=3815&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="bib45"><span id="sref45">Zhang Y., Huang K., Wang T., Deng F., Gong W., Hui X., Zhao Y., He X., Li C., Zhang Q., et al.  SARS-CoV-2 rapidly adapts in aged BALB/c mice and induces typical pneumonia. <span><span>J. Virol. </span>2021;<span>95</span></span> e02477-20. <span>[<a href="https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC8139693/">PMC free article</a>]</span> [<a href="https://www.ncbi.nlm.nih.gov/pubmed/33692211" ref="reftype=pubmed&amp;article-id=8702434&amp;issue-id=396485&amp;journal-id=3815&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span>[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Virol.&amp;title=SARS-CoV-2+rapidly+adapts+in+aged+BALB/c+mice+and+induces+typical+pneumonia&amp;author=Y.+Zhang&amp;author=K.+Huang&amp;author=T.+Wang&amp;author=F.+Deng&amp;author=W.+Gong&amp;volume=95&amp;publication_year=2021&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8702434&amp;issue-id=396485&amp;journal-id=3815&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="bib46"><span id="sref46">Zhou P., Yang X.L., Wang X.G., Hu B., Zhang L., Zhang W., Si H.R., Zhu Y., Li B., Huang C.L., et al.  A pneumonia outbreak associated with a new coronavirus of probable bat origin. <span><span>Nature. </span>2020;<span>579</span>:270–273.</span> <span>[<a href="https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC7095418/">PMC free article</a>]</span> [<a href="https://www.ncbi.nlm.nih.gov/pubmed/32015507" ref="reftype=pubmed&amp;article-id=8702434&amp;issue-id=396485&amp;journal-id=3815&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span>[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A+pneumonia+outbreak+associated+with+a+new+coronavirus+of+probable+bat+origin&amp;author=P.+Zhou&amp;author=X.L.+Yang&amp;author=X.G.+Wang&amp;author=B.+Hu&amp;author=L.+Zhang&amp;volume=579&amp;publication_year=2020&amp;pages=270-273&amp;pmid=32015507&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8702434&amp;issue-id=396485&amp;journal-id=3815&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li></ul></div></div></div><!--post-content--></div>
            
        </section>
    </article>
    
</div></div>
  </body>
</html>
