<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <title>James Routley | Feed</title>
    <link
      rel="stylesheet"
      type="text/css"
      href="../styles.css"
      media="screen"
    />
  </head>
  <body>
    <a href="/index.html">Back</a>
    <a href="https://www.nature.com/articles/s44323-024-00005-1">Original</a>
    <h1>Evidence for ~12-h ultradian gene programs in humans</h1>
    
    <div id="readability-page-1" class="page"><div>
                    <p>Our understanding of biological rhythms is expanded in part by alternative, non-circadian rhythms discovered in lower organisms. Coastal marine organisms, such as the sea anemone <i>A. diaphana</i>, exhibit ~12-h ultradian rhythms, entrained by the twice daily movements of the tides<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 1" title="Andreatta, G. &amp; Tessmar-Raible, K. The still dark side of the moon: molecular mechanisms of lunar-controlled rhythms and clocks. J. Mol. Biol. 432, 3525–3546 (2020)." href="https://www.nature.com/articles/s44323-024-00005-1#ref-CR1" id="ref-link-section-d102698130e682">1</a></sup>. In mice, ~12-h rhythms of gene expression involved in mRNA, protein and lipid metabolism are established by an XBP1-dependent oscillator independent of the circadian clock or cell cycle<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 2" title="Ballance, H. &amp; Zhu, B. Revealing the hidden reality of the mammalian 12-h ultradian rhythms. Cell. Mol. Life Sci. 
                  https://doi.org/10.1007/s00018-020-03730-5
                  
                 (2021)." href="https://www.nature.com/articles/s44323-024-00005-1#ref-CR2" id="ref-link-section-d102698130e686">2</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 3" title="Asher, G. &amp; Zhu, B. Beyond circadian rhythms: emerging roles of ultradian rhythms in control of liver functions. Hepatology 
                  https://doi.org/10.1002/hep.32580
                  
                 (2022)." href="https://www.nature.com/articles/s44323-024-00005-1#ref-CR3" id="ref-link-section-d102698130e689">3</a></sup>. Oscillation of some physiological metrics at an ~12-h interval in humans suggests the possible existence of 12-h rhythms in humans<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Broughton, R. &amp; Mullington, J. Circasemidian sleep propensity and the phase-amplitude maintenance model of human sleep/wake regulation. J. Sleep Res. 1, 93–98 (1992)." href="#ref-CR4" id="ref-link-section-d102698130e693">4</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Hayashi, M., Morikawa, T. &amp; Hori, T. Circasemidian 12 h cycle of slow wave sleep under constant darkness. Clin. Neurophysiol. 113, 1505–1516 (2002)." href="#ref-CR5" id="ref-link-section-d102698130e693_1">5</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Wan, C., Wang, Z., Cornelissen, G. &amp; Halberg, F. Age, gender and circadian or circasemidian blood pressure and heart rate variation of children. Chronobiologia 19, 121–129 (1992)." href="#ref-CR6" id="ref-link-section-d102698130e693_2">6</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Otsuka, K. et al. Long-term exposure to space’s microgravity alters the time structure of heart rate variability of astronauts. Heliyon 2, e00211 (2016)." href="#ref-CR7" id="ref-link-section-d102698130e693_3">7</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Otsuka, K. et al. Methods for assessing change in brain plasticity at night and psychological resilience during daytime between repeated long-duration space missions. Sci. Rep. 13, 10909 (2023)." href="#ref-CR8" id="ref-link-section-d102698130e693_4">8</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 9" title="Otsuka, K. et al. Appropriate circadian-circasemidian coupling protects blood pressure from morning surge and promotes human resilience and wellbeing. Clin. Interv. Aging 18, 755–769 (2023)." href="https://www.nature.com/articles/s44323-024-00005-1#ref-CR9" id="ref-link-section-d102698130e696">9</a></sup>. However, evidence for ~12-h rhythms at the molecular level in humans is lacking.</p><div><p>We studied three healthy males (Table <a data-track="click" data-track-label="link" data-track-action="table anchor" href="https://www.nature.com/articles/s44323-024-00005-1#Tab1">1</a>) who self-reported a regular nighttime sleep schedule and did not engage in nighttime shift work or other sleep-disrupting activities and who had a body mass index between 23.3 and 24.4 kg/m<sup>2</sup>. Participants were admitted for a 1-day acclimatization period prior to 48 h of blood sampling at a 2-h sampling interval (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s44323-024-00005-1#Fig1">1A</a>), a frequency that allowed detection of oscillations with periods of 6 h or longer with high confidence<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 10" title="Zhu, B., Dacso, C. C. &amp; O’Malley, B. W. Unveiling “musica universalis” of the cell: a brief history of biological 12-hour rhythms. J. Endocr. Soc. 2, 727–752 (2018)." href="https://www.nature.com/articles/s44323-024-00005-1#ref-CR10" id="ref-link-section-d102698130e711">10</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 11" title="Antoulas, A. C. et al. A novel mathematical method for disclosing oscillations in gene transcription: a comparative study. PLoS One 13, e0198503 (2018)." href="https://www.nature.com/articles/s44323-024-00005-1#ref-CR11" id="ref-link-section-d102698130e714">11</a></sup>. The protocol reflected our aim to sample with sufficient frequency and duration to detect ~12-h rhythms within individuals. We also sought to mimic free-living conditions by providing a diet similar in caloric content to their typical intake and encouraging maintenance of their routine sleep/wake cycles. Overnight, blood was collected via a long intravenous line from outside the room to avoid exposure to light or sleep disruption.</p><div data-test="inline-table" data-container-section="table" id="table-1"><figure><figcaption><b id="Tab1" data-test="table-caption">Table 1 Participant characteristics.</b></figcaption></figure></div><div data-test="figure" data-container-section="figure" id="figure-1" data-title="Identification of inter-individual variability of ~24-h and ultradian rhythms of gene expression in peripheral white blood cells of humans."><figure><figcaption><b id="Fig1" data-test="figure-caption-text">Fig. 1: Identification of inter-individual variability of ~24-h and ultradian rhythms of gene expression in peripheral white blood cells of humans.</b></figcaption><div><div><a data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="https://www.nature.com/articles/s44323-024-00005-1/figures/1" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs44323-024-00005-1/MediaObjects/44323_2024_5_Fig1_HTML.png?as=webp"/><img aria-describedby="Fig1" src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs44323-024-00005-1/MediaObjects/44323_2024_5_Fig1_HTML.png" alt="figure 1" loading="lazy" width="685" height="785"/></picture></a></div><p><b>A</b> Human protocol and study schematic. <b>B</b>–<b>D</b> Histograms showing the period distributions of all rhythmic genes uncovered from the three participants (with <i>p</i> value &lt; 0.05) as well as the period distribution of rhythmic genes calculated by the meta <i>p</i> values (meta <i>p</i> &lt; 0.05). For the first individual, periods calculated from all 48 h of data, or the first 24 h of the dataset were both shown. <b>C</b> Table summarizing the number of ~24-h and ~12-h genes with different statistical cut-offs. <b>D</b> Table listing the meta P and meta adj-P (FDR) values of thirteen ~24-h genes previously used as training datasets for predicting ~24-h phase from human blood samples<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 15" title="Wittenbrink, N. et al. High-accuracy determination of internal circadian time from a single blood sample. J. Clin. Investig. 128, 3826–3839 (2018)." href="https://www.nature.com/articles/s44323-024-00005-1#ref-CR15" id="ref-link-section-d102698130e1015">15</a></sup>. <b>E</b> Raw temporal expression profile (dot) and spline fit (solid line) of five of the thirteen ~24-h genes. <b>F</b> Gene set enrichment analysis (GSEA) showing enrichment of a previously identified set of top forty robustly-cycling ~24-h genes in human white blood cells<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 15" title="Wittenbrink, N. et al. High-accuracy determination of internal circadian time from a single blood sample. J. Clin. Investig. 128, 3826–3839 (2018)." href="https://www.nature.com/articles/s44323-024-00005-1#ref-CR15" id="ref-link-section-d102698130e1025">15</a></sup> with robust ~24-h rhythmicity in our study. <b>G</b> GO analyses of top KEGG pathways for ~24-h genes with meta adj-P (FDR) less than 0.01.</p></div></figure></div></div><p>We performed bulk mRNA-Seq in buffy coat fractions prospectively collected at 2-h intervals for 48 h (24 samples/participant) (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s44323-024-00005-1#Fig1">1A</a>) (Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s44323-024-00005-1#MOESM2">S1</a>). To identify genes cycling with a period ranging from 6 to 32 h, we applied the RAIN algorithm to each participant’s temporal transcriptome<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 12" title="Thaben, P. F. &amp; Westermark, P. O. Detecting rhythms in time series with RAIN. J. Biol. Rhythms 29, 391–400 (2014)." href="https://www.nature.com/articles/s44323-024-00005-1#ref-CR12" id="ref-link-section-d102698130e1050">12</a></sup>. Compared to alternative methods like JTK_CYCLE, RAIN detects rhythms with arbitrary waveforms and therefore more robustly uncovers ultradian rhythms<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 11" title="Antoulas, A. C. et al. A novel mathematical method for disclosing oscillations in gene transcription: a comparative study. PLoS One 13, e0198503 (2018)." href="https://www.nature.com/articles/s44323-024-00005-1#ref-CR11" id="ref-link-section-d102698130e1054">11</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 13" title="Zhu, B. et al. A cell-autonomous mammalian 12 h clock coordinates metabolic and stress rhythms. Cell Metab. 25, 1305–1319.e1309 (2017)." href="https://www.nature.com/articles/s44323-024-00005-1#ref-CR13" id="ref-link-section-d102698130e1057">13</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 14" title="Pan, Y. et al. 12-h clock regulation of genetic information flow by XBP1s. PLOS Biol. 18, e3000580 (2020)." href="https://www.nature.com/articles/s44323-024-00005-1#ref-CR14" id="ref-link-section-d102698130e1060">14</a></sup>. As shown in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s44323-024-00005-1#Fig1">1B</a>, we observed inter-individual variability in the number of genes cycling at different periods. Specifically, the first individual exhibited dominant oscillations cycling at the period between 20 and 24-h, with a secondary population detected at the ultradian periods of 10–12 h. These 10–12-h oscillations were more evident in the first 24 h, with dampening of ultradian rhythms on the second day, despite there being no obvious physiological perturbation or difference between the first and second day of sampling (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s44323-024-00005-1#Fig1">1B</a>). Aside from oscillations between 18 and 24 h, the second participant exhibited ultradian rhythms cycling at periods of 6 and 12-h (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s44323-024-00005-1#Fig1">1B</a>). The third individual also exhibited oscillations cycling at 6-h, 12-h, and at ~20- h with fewer genes observed cycling at 24-h (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s44323-024-00005-1#Fig1">1B</a>). The observed inter-individual variability of transcriptomes cycling at ~24-h is not surprising, as inter-individual variations in the number, period, phase and amplitude of circadian genes were also observed in a prior study where participants were maintained in a semi-recumbent position under dim light and constant temperature and humidity during 40 h of sleep deprivation (Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s44323-024-00005-1#MOESM1">S1A, B</a>)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 15" title="Wittenbrink, N. et al. High-accuracy determination of internal circadian time from a single blood sample. J. Clin. Investig. 128, 3826–3839 (2018)." href="https://www.nature.com/articles/s44323-024-00005-1#ref-CR15" id="ref-link-section-d102698130e1080">15</a></sup>.</p><p>To increase statistical power to detect oscillations common to all three individuals, we performed a meta-analysis and generated a combined <i>p</i>-value for each gene at periods from 6–32-h using Fisher’s method. This method has been extensively used in medical and genetic research to merge results from independent tests, each with the same null hypothesis<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 16" title="Yoon, S., Baik, B., Park, T. &amp; Nam, D. Powerful p-value combination methods to detect incomplete association. Sci. Rep. 11, 6980 (2021)." href="https://www.nature.com/articles/s44323-024-00005-1#ref-CR16" id="ref-link-section-d102698130e1090">16</a></sup>. In our study, each participant was distinct and studied at a different time, however the null hypothesis was the same: the absence of rhythms. Using combined p-values (herein referred to as meta <i>p</i> values) with an alpha of 0.05, we observed two major populations of oscillations cycling at the periods of 12 and 18–26-h, with the largest number of genes observed cycling at the period of 20-h (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s44323-024-00005-1#Fig1">1B</a>). Using the Benjamini-Hochberg procedure to further adjust for the false discovery rate on meta <i>p</i> values (meta adj-<i>P</i> value or FDR), we uncovered 653 genes with periods between 10–14-h and 5453 genes with periods between 22–26-h genes with an FDR less than 0.05 (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s44323-024-00005-1#Fig1">1C</a>) (Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s44323-024-00005-1#MOESM3">S2</a>).</p><p>To determine the soundness of our RNA-seq dataset, we compared 22–26-h cycling genes identified in our study with those uncovered by Wittenbrink and colleagues<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 15" title="Wittenbrink, N. et al. High-accuracy determination of internal circadian time from a single blood sample. J. Clin. Investig. 128, 3826–3839 (2018)." href="https://www.nature.com/articles/s44323-024-00005-1#ref-CR15" id="ref-link-section-d102698130e1116">15</a></sup>. In that study, thirteen genes exhibiting robust circadian expression in peripheral white blood cells were utilized as a training gene set to predict the circadian phases from single time point blood samples<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 15" title="Wittenbrink, N. et al. High-accuracy determination of internal circadian time from a single blood sample. J. Clin. Investig. 128, 3826–3839 (2018)." href="https://www.nature.com/articles/s44323-024-00005-1#ref-CR15" id="ref-link-section-d102698130e1120">15</a></sup>. In our study, seven and nine (of the core set of 13 genes) exhibited meta adj-<i>P</i> values less than 0.05 and 0.1, respectively, including canonical circadian clock genes <i>PER1</i>, <i>PER2</i>, <i>PER3</i>, <i>NR1D1</i>, <i>NR1D2</i> and <i>CRY1</i> (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s44323-024-00005-1#Fig1">1D, E</a>). Gene set enrichment analysis (GSEA) further indicated that the forty circadian genes that cycled in at least 50% of the human participants in the study by Wittenbrink et al.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 15" title="Wittenbrink, N. et al. High-accuracy determination of internal circadian time from a single blood sample. J. Clin. Investig. 128, 3826–3839 (2018)." href="https://www.nature.com/articles/s44323-024-00005-1#ref-CR15" id="ref-link-section-d102698130e1149">15</a></sup>, also exhibited oscillations cycling between 22–26 h with low meta <i>p</i> values in our study (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s44323-024-00005-1#Fig1">1F</a>). These genes were linked to pathways of platelet activation, regulation of actin cytoskeleton and adhesions (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s44323-024-00005-1#Fig1">1G</a>). Collectively, these results demonstrate the possibility of detecting biologically relevant rhythms in human gene expression under conditions designed to mimic free-living.</p><div><p>Oscillations cycling close to a 12-h period were the second most abundant in all three individuals after the 22–26-h population (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s44323-024-00005-1#Fig1">1B</a>). Using meta adj-<i>P</i> values (FDR) as cut-off, we identified 653 and 2768 ~ 12-h genes (with period between 10–14-h) with meta adj-P (FDR) less than 0.05 and 0.1, respectively (Figs. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s44323-024-00005-1#Fig1">1</a>C, <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s44323-024-00005-1#Fig2">2A</a>). As observed with 22–26-h genes, inter-individual variability was also observed for ~12-h ultradian rhythms. In the first and second participants, the acrophases (time of gene expression peak) were at 8 am and 8 pm, while ~12-h rhythms in the third individual peaked around 4 am and 4 pm (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s44323-024-00005-1#Fig2">2A</a>). Examples of genes that robustly cycled at an ~12-h period in all three individuals include <i>MYDGF</i>, <i>ICMT</i>, <i>RNF7</i>, <i>TCEB1</i>, <i>MFF,</i> and <i>OXA1L</i> (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s44323-024-00005-1#Fig2">2B</a>). <i>MYDGF</i> encodes an endoplasmic reticulum (ER)-localized protein with secreted forms from monocytes/macrophages that promote tissue repair in a murine model of myocardial infarction<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 17" title="Korf-Klingebiel, M. et al. Myeloid-derived growth factor (C19orf10) mediates cardiac repair following myocardial infarction. Nat. Med. 21, 140–149 (2015)." href="https://www.nature.com/articles/s44323-024-00005-1#ref-CR17" id="ref-link-section-d102698130e1208">17</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 18" title="Bortnov, V. et al. Solution structure of human myeloid-derived growth factor suggests a conserved function in the endoplasmic reticulum. Nat. Commun. 10, 5612 (2019)." href="https://www.nature.com/articles/s44323-024-00005-1#ref-CR18" id="ref-link-section-d102698130e1211">18</a></sup>. <i>ICMT</i> encodes a protein-<i>S</i>-isoprenylcysteine <i>O</i>-methyltransferase that also localizes to the ER and is responsible for cell membrane-targeting of selective proteins<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 19" title="Borini Etichetti, C. et al. Isoprenylcysteine carboxy methyltransferase (ICMT) is associated with tumor aggressiveness and its expression is controlled by the p53 tumor suppressor. J. Biol. Chem. 294, 5060–5073 (2019)." href="https://www.nature.com/articles/s44323-024-00005-1#ref-CR19" id="ref-link-section-d102698130e1225">19</a></sup>. <i>RNF7</i> encodes an essential subunit of SKP1-cullin/CDC53-F box protein ubiquitin ligases<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 20" title="Telesio, G. et al. The E3 ubiquitin ligase RNF7 negatively regulates CARD14/CARMA2sh signaling. Int. J. Mol. Sci. 18. 
                  https://doi.org/10.3390/ijms18122581
                  
                 (2017)." href="https://www.nature.com/articles/s44323-024-00005-1#ref-CR20" id="ref-link-section-d102698130e1232">20</a></sup> and TCEB1 protein is a subunit of the transcription factor B responsible for transcription elongation<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 21" title="Hakimi, A. A. et al. TCEB1-mutated renal cell carcinoma: a distinct genomic and morphological subtype. Mod. Pathol. 28, 845–853 (2015)." href="https://www.nature.com/articles/s44323-024-00005-1#ref-CR21" id="ref-link-section-d102698130e1236">21</a></sup>. Both MFF and OXA1L are mitochondrial proteins localized to outer and inner mitochondria membranes, respectively, and are essential for mitochondria fission and assembly of respiratory complex I<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 22" title="Bonnefoy, N. et al. Cloning of a human gene involved in cytochrome oxidase assembly by functional complementation of an oxa1- mutation in Saccharomyces cerevisiae. Proc. Natl Acad. Sci. USA 91, 11978–11982 (1994)." href="https://www.nature.com/articles/s44323-024-00005-1#ref-CR22" id="ref-link-section-d102698130e1240">22</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 23" title="Otera, H. et al. Mff is an essential factor for mitochondrial recruitment of Drp1 during mitochondrial fission in mammalian cells. J. Cell Biol. 191, 1141–1158 (2010)." href="https://www.nature.com/articles/s44323-024-00005-1#ref-CR23" id="ref-link-section-d102698130e1243">23</a></sup>.</p><div data-test="figure" data-container-section="figure" id="figure-2" data-title="~12-h gene expression rhythms are enriched in mRNA and protein homeostasis pathways."><figure><figcaption><b id="Fig2" data-test="figure-caption-text">Fig. 2: ~12-h gene expression rhythms are enriched in mRNA and protein homeostasis pathways.</b></figcaption><div><div><a data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="https://www.nature.com/articles/s44323-024-00005-1/figures/2" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs44323-024-00005-1/MediaObjects/44323_2024_5_Fig2_HTML.png?as=webp"/><img aria-describedby="Fig2" src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs44323-024-00005-1/MediaObjects/44323_2024_5_Fig2_HTML.png" alt="figure 2" loading="lazy" width="685" height="807"/></picture></a></div><p><b>A</b> Heatmap and quantification of ~12-h genes from all three participants with meta adj-P (FDR) values less than 0.05 or between 0.05 and 0.1. <b>B</b> Raw temporal expression (dot) profile and spline fit (solid line) of six ~12-h genes with meta <i>p</i> values also shown. <b>C</b> Cumulative distribution of the number of ~24-h and ~12-h genes with different meta p and meta adj-<i>P</i> cut-offs. The ~24-h data set consisted of a 4 h sampling interval so that the period to sampling interval was matched to the ~12-h data set. <b>D</b> Scatter plot comparing log-transformed meta <i>p</i> values for each gene exhibiting ~24-h versus ~12-h rhythms. Both observed and predicted percentage of genes (under the null hypothesis that ~12-h and ~24-h genes are independently regulated) with meta <i>p</i> value smaller or larger than 0.05 are further shown. <i>P</i> value of 0.73 is calculated by Chi-square test. <b>E</b> GO analysis of ~24-h and ~12-h genes with different statistical cut-offs. <b>F</b> GSEA showing enrichment scores of different gene sets for ~12-h (top) and ~24-h (bottom) genes.</p></div></figure></div></div><p>One important question regarding ultradian rhythmicity is whether it reflects “real’ rhythms with biological significance, or simply mathematical artefact. This is particularly relevant for rhythms that cycle at harmonic frequencies of the 24-h period, as non-sinusoidal circadian waveforms (seesaw-like or square-shaped waveforms) often have superimposed ~12- and ~8-h harmonic frequencies detected by spectral analytical methods such as a Fourier transformation, eigenvalue/pencil, or wavelet analyses<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 13" title="Zhu, B. et al. A cell-autonomous mammalian 12 h clock coordinates metabolic and stress rhythms. Cell Metab. 25, 1305–1319.e1309 (2017)." href="https://www.nature.com/articles/s44323-024-00005-1#ref-CR13" id="ref-link-section-d102698130e1303">13</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="van der Veen, D. R. &amp; Gerkema, M. P. Unmasking ultradian rhythms in gene expression. FASEB J. 31, 743–750 (2017)." href="#ref-CR24" id="ref-link-section-d102698130e1306">24</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Kembro, J. M. et al. A dynamically coherent pattern of rhythms that matches between distant species across the evolutionary scale. Sci. Rep. 13, 5326 (2023)." href="#ref-CR25" id="ref-link-section-d102698130e1306_1">25</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 26" title="Hughes, M. E. et al. Harmonics of circadian gene transcription in mammals. PLoS Genet. 5, e1000442 (2009)." href="https://www.nature.com/articles/s44323-024-00005-1#ref-CR26" id="ref-link-section-d102698130e1309">26</a></sup>. We took several approaches to address this question. First, 12-h transcriptional oscillations did not cycle at the exact second harmonic of the dominant 22–26-h period in any of the three participants: the first harmonic periods were 22, 18 and 20-h in the three individuals, and 20-h for the meta-analysis (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s44323-024-00005-1#Fig1">1B</a>). Second, if ~12-h oscillations are indeed harmonics of the 22–26-h rhythm, then genes exhibiting both 22–26-h and 10–14-h ultradian rhythmicity should be detected at a higher frequency than expected by chance. To test this, we generated a second set of 24-h gene set using a 4-h interval subset of the original dataset (8am, 12am, 4 pm, etc.) to achieve an equivalent number of data points per period—i.e. equivalent sensitivity—and a meta-p cut-off of 0.05. Importantly, the new 24-h subset showed a strong positive correlation with meta-<i>p</i> values obtained with the original full 2-h interval dataset (Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s44323-024-00005-1#MOESM1">S2A</a>). After this adjustment to match sampling interval to period ratios, 10–14-h genes were more prevalent than 24-h genes (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s44323-024-00005-1#Fig2">2C</a>). Moreover, the expected and observed percentages of genes having 24-h and/or 10–14-h rhythms were nearly identical (<i>p</i> = 0.73 by Chi-square test), indicating independent detection of these two different rhythms (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s44323-024-00005-1#Fig2">2D</a>). Third, we performed Gene Ontology (GO) and GSEA to compare biological pathways enriched in 22–26-h and 10–14-h genes. We reasoned that if 10–14-h oscillations are mathematical harmonics of the 22–26-h rhythm, then these two groups of genes should share enriched biological pathways. Both analyses revealed largely distinct pathways associated with genes with 22–26-h and 10–14-h rhythms (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s44323-024-00005-1#Fig2">2E</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s44323-024-00005-1#MOESM1">S2B</a>). While the former were enriched in platelet activation, blood coagulation and cell adhesion as previously demonstrated (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s44323-024-00005-1#Fig2">2E</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s44323-024-00005-1#MOESM1">S2B</a>), the latter was associated with fundamental biological processes of mRNA (such as spliceosome and RNA splicing), protein metabolism (including ubiquitin-mediated proteolysis, response to ER stress, protein folding and protein transport in the ER and Golgi), and mitochondria complex chain assembly (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s44323-024-00005-1#Fig2">2E</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s44323-024-00005-1#MOESM1">S2B</a>). To study the potential disease relevance of human 10–14-h genes, we further performed GSEA with disease gene sets. Consistent with the known causal roles of dysregulated proteostasis in proteinopathies<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 27" title="Golde, T. E., Borchelt, D. R., Giasson, B. I. &amp; Lewis, J. Thinking laterally about neurodegenerative proteinopathies. J. Clin. Investig. 123, 1847–1855 (2013)." href="https://www.nature.com/articles/s44323-024-00005-1#ref-CR27" id="ref-link-section-d102698130e1351">27</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 28" title="Bayer, T. A. Proteinopathies, a core concept for understanding and ultimately treating degenerative disorders? Eur. Neuropsychopharmacol. 25, 713–724 (2015)." href="https://www.nature.com/articles/s44323-024-00005-1#ref-CR28" id="ref-link-section-d102698130e1354">28</a></sup>, and impaired mitochondrial metabolism in metabolic syndromes<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 29" title="Prasun, P. Mitochondrial dysfunction in metabolic syndrome. Biochim. Biophys. Acta Mol. Basis Dis. 1866, 165838 (2020)." href="https://www.nature.com/articles/s44323-024-00005-1#ref-CR29" id="ref-link-section-d102698130e1358">29</a></sup>, 10–14-h genes were strongly enriched in gene sets of neurogenerative disease and nonalcoholic fatty liver disease (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s44323-024-00005-1#Fig2">2F</a>). Enrichment of protein metabolism and mitochondrial respiration genes is also in alignment with ~12-h gene signatures in human dorsolateral prefrontal cortex<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 30" title="Scott, M. R. et al. Twelve-hour rhythms in transcript expression within the human dorsolateral prefrontal cortex are altered in schizophrenia. PLOS Biol. 21, e3001688 (2023)." href="https://www.nature.com/articles/s44323-024-00005-1#ref-CR30" id="ref-link-section-d102698130e1365">30</a></sup>. Collectively, these analyses support the identification of ~12-h biological rhythms; however, these data do not establish the regulatory mechanism(s) of ~12-h rhythms, which could include cell-autonomous oscillators, homeostatic responses to environmental stimuli, or an “hour-glass“ timing mechanism<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 31" title="Rensing, L., Meyer-Grahle, U. &amp; Ruoff, P. Biological timing and the clock metaphor: oscillatory and hourglass mechanisms. Chronobiol. Int. 18, 329–369 (2001)." href="https://www.nature.com/articles/s44323-024-00005-1#ref-CR31" id="ref-link-section-d102698130e1370">31</a></sup>.</p><p>We next performed pathway analyses of the 10–14-h gene sets for each individual participant. We either analyzed the data as a continuous 48-h time series with a single data point at each time point (RAIN conti) or as a 24-h time series where data points collected at the same time on two consecutive days were treated as biological replicates (RAIN dupli) (Figs. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s44323-024-00005-1#MOESM1">S3</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s44323-024-00005-1#MOESM1">4</a>, and Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s44323-024-00005-1#MOESM3">S2</a>). The RAIN duplicate approach revealed larger 10–14-h gene programs: 3462 genes (<i>p</i> value &lt; 0.05, with FDR &lt; 0.224), 7060 genes (<i>p</i> value &lt; 0.05, with FDR &lt; 0.119) and 4807 genes (p value &lt; 0.05, with FDR &lt; 0.166) for the three participants (Figs. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s44323-024-00005-1#MOESM1">S4A</a>). As expected, 10–14-h genes common to all three individuals also tended to have small meta-<i>p</i> values (Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s44323-024-00005-1#MOESM1">S3</a>E and Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s44323-024-00005-1#MOESM1">S4E</a>). Pathways related to mRNA and protein metabolism emerged as significantly enriched for 10–14-h rhythm genes in each participant regardless of the inputs or thresholds for RAIN analysis (Figs. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s44323-024-00005-1#MOESM1">S3</a>G and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s44323-024-00005-1#MOESM1">S4C, E</a>).</p><p>To determine robustness to different analytic methods, we also performed spectrum analysis with the eigenvalue/pencil method<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 11" title="Antoulas, A. C. et al. A novel mathematical method for disclosing oscillations in gene transcription: a comparative study. PLoS One 13, e0198503 (2018)." href="https://www.nature.com/articles/s44323-024-00005-1#ref-CR11" id="ref-link-section-d102698130e1414">11</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 13" title="Zhu, B. et al. A cell-autonomous mammalian 12 h clock coordinates metabolic and stress rhythms. Cell Metab. 25, 1305–1319.e1309 (2017)." href="https://www.nature.com/articles/s44323-024-00005-1#ref-CR13" id="ref-link-section-d102698130e1417">13</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 14" title="Pan, Y. et al. 12-h clock regulation of genetic information flow by XBP1s. PLOS Biol. 18, e3000580 (2020)." href="https://www.nature.com/articles/s44323-024-00005-1#ref-CR14" id="ref-link-section-d102698130e1420">14</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Meng, H. et al. XBP1 links the 12-hour clock to NAFLD and regulation of membrane fluidity and lipid homeostasis. Nat. Commun. 11, 6215 (2020)." href="#ref-CR32" id="ref-link-section-d102698130e1423">32</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Dion, W. et al. Four-dimensional nuclear speckle phase separation dynamics regulate proteostasis. Sci. Adv. 8, eabl4150 (2022)." href="#ref-CR33" id="ref-link-section-d102698130e1423_1">33</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 34" title="Meng, H. et al. Defining the mammalian coactivation of hepatic 12-h clock and lipid metabolism. Cell Rep. 38, 110491 (2022)." href="https://www.nature.com/articles/s44323-024-00005-1#ref-CR34" id="ref-link-section-d102698130e1426">34</a></sup>, which unlike statistical methods such as JTK_CYCLE and RAIN does not require pre-assignment of period range, enabling unbiased identification of multiple oscillations for any given gene<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 11" title="Antoulas, A. C. et al. A novel mathematical method for disclosing oscillations in gene transcription: a comparative study. PLoS One 13, e0198503 (2018)." href="https://www.nature.com/articles/s44323-024-00005-1#ref-CR11" id="ref-link-section-d102698130e1430">11</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 13" title="Zhu, B. et al. A cell-autonomous mammalian 12 h clock coordinates metabolic and stress rhythms. Cell Metab. 25, 1305–1319.e1309 (2017)." href="https://www.nature.com/articles/s44323-024-00005-1#ref-CR13" id="ref-link-section-d102698130e1433">13</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 14" title="Pan, Y. et al. 12-h clock regulation of genetic information flow by XBP1s. PLOS Biol. 18, e3000580 (2020)." href="https://www.nature.com/articles/s44323-024-00005-1#ref-CR14" id="ref-link-section-d102698130e1436">14</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Meng, H. et al. XBP1 links the 12-hour clock to NAFLD and regulation of membrane fluidity and lipid homeostasis. Nat. Commun. 11, 6215 (2020)." href="#ref-CR32" id="ref-link-section-d102698130e1439">32</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Dion, W. et al. Four-dimensional nuclear speckle phase separation dynamics regulate proteostasis. Sci. Adv. 8, eabl4150 (2022)." href="#ref-CR33" id="ref-link-section-d102698130e1439_1">33</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 34" title="Meng, H. et al. Defining the mammalian coactivation of hepatic 12-h clock and lipid metabolism. Cell Rep. 38, 110491 (2022)." href="https://www.nature.com/articles/s44323-024-00005-1#ref-CR34" id="ref-link-section-d102698130e1442">34</a></sup>. Eigenvalue/pencil analyses also revealed ~24-h and ~12-h oscillations in all three individuals (Figs. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s44323-024-00005-1#MOESM1">S5–7</a>, and Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s44323-024-00005-1#MOESM4">S3</a>). ~12-h gene programs were distinct from ~24-h gene sets in all three individuals and enriched in mRNA and protein metabolism pathways (Figs. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s44323-024-00005-1#MOESM1">S5</a>J, <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s44323-024-00005-1#MOESM1">S6</a>J and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s44323-024-00005-1#MOESM1">S7J</a>). Taken together, orthogonal analytical methods and statistical thresholds suggested ~12-h rhythms of gene expression implicated in mRNA and protein metabolism in human white blood cells.</p><div><p>To infer gene regulatory networks governing ~12-h rhythms, we performed LISA<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 35" title="Qin, Q. et al. Lisa: inferring transcriptional regulators through integrative modeling of public chromatin accessibility and ChIP-seq data. Genome Biol. 21, 32 (2020)." href="https://www.nature.com/articles/s44323-024-00005-1#ref-CR35" id="ref-link-section-d102698130e1465">35</a></sup> and motif analysis on the top 500 10–14-h genes uncovered by the RAIN method with the lowest meta-<i>p</i> values. We cross-referenced enriched motifs with transcription factor genes that exhibited 10–14-h rhythms (with meta-adjP&lt;0.1), identifying XBP1, CREBZF, GABPB1/2, NFYB, GMEB1, KLF16 as top candidates that were preferentially enriched with 10–14-h genes compared to 22–26-h genes (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s44323-024-00005-1#Fig3">3A–D</a>). XBP1, CREBZF and CREB1 belong to the Basic Leucine Zipper (bZIP) transcription factor family and are established transcription regulators of the unfolded protein response (UPR) and adaptive stress response<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Zhang, R. &amp; Misra, V. Effects of cyclic AMP response element binding protein-Zhangfei (CREBZF) on the unfolded protein response and cell growth are exerted through the tumor suppressor p53. Cell Cycle 13, 279–292 (2014)." href="#ref-CR36" id="ref-link-section-d102698130e1475">36</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Zhang, R. et al. Zhangfei/CREB-ZF—a potential regulator of the unfolded protein response. PLOS ONE 8, e77256 (2013)." href="#ref-CR37" id="ref-link-section-d102698130e1475_1">37</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Kikuchi, D., Tanimoto, K. &amp; Nakayama, K. CREB is activated by ER stress and modulates the unfolded protein response by regulating the expression of IRE1α and PERK. Biochem. Biophys. Res. Commun. 469, 243–250 (2016)." href="#ref-CR38" id="ref-link-section-d102698130e1475_2">38</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Sriburi, R., Jackowski, S., Mori, K. &amp; Brewer, J. W. XBP1: a link between the unfolded protein response, lipid biosynthesis, and biogenesis of the endoplasmic reticulum. J. Cell Biol. 167, 35–41 (2004)." href="#ref-CR39" id="ref-link-section-d102698130e1475_3">39</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 40" title="Li, Y. et al. A CREBZF/Zhangfei isoform activates CHOP during prolonged cellular stress. bioRxiv, 2022.2010.2010.511668. 
                  https://doi.org/10.1101/2022.10.10.511668
                  
                 (2022)." href="https://www.nature.com/articles/s44323-024-00005-1#ref-CR40" id="ref-link-section-d102698130e1478">40</a></sup>. GABPB1/B2 are ETS family transcription factors involved in mitochondria biogenesis and oxidative phosphorylation<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 41" title="Yang, Z. F., Drumea, K., Mott, S., Wang, J. &amp; Rosmarin, A. G. GABP transcription factor (nuclear respiratory factor 2) is required for mitochondrial biogenesis. Mol. Cell Biol. 34, 3194–3201 (2014)." href="https://www.nature.com/articles/s44323-024-00005-1#ref-CR41" id="ref-link-section-d102698130e1482">41</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 42" title="Mootha, V. K. et al. Erralpha and Gabpa/b specify PGC-1alpha-dependent oxidative phosphorylation gene expression that is altered in diabetic muscle. Proc. Natl Acad. Sci. USA 101, 6570–6575 (2004)." href="https://www.nature.com/articles/s44323-024-00005-1#ref-CR42" id="ref-link-section-d102698130e1485">42</a></sup>. NFYB encodes one of the three subunits of nuclear transcription factor Y (the other being NFYA and NFYC) and its DNA binding motif was recently shown to co-occur with XBP1 in mouse type-2 T helper cells<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 43" title="Pramanik, J. et al. Genome-wide analyses reveal the IRE1a-XBP1 pathway promotes T helper cell differentiation by resolving secretory stress and accelerating proliferation. Genome Med. 10, 76 (2018)." href="https://www.nature.com/articles/s44323-024-00005-1#ref-CR43" id="ref-link-section-d102698130e1490">43</a></sup>. Several of these transcription factors have previously been implicated in regulation of ~12-h rhythms in mice. For instance, it was previously proposed that ~12-h rhythms are generally established by a tripartite network comprising ETS, bZIP and NFY transcription factors, whereas KLF transcription factors may control ~12-h rhythms in a more tissue specific manner<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 44" title="Zhu, B. Decoding the function and regulation of the mammalian 12h-clock. J. Mol. Cell Biol. 
                  https://doi.org/10.1093/jmcb/mjaa021
                  
                 (2020)." href="https://www.nature.com/articles/s44323-024-00005-1#ref-CR44" id="ref-link-section-d102698130e1494">44</a></sup>. Enriched GMEB DNA binding motifs were also observed in the promoters of hepatic ~12-h rhythms in mice<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 32" title="Meng, H. et al. XBP1 links the 12-hour clock to NAFLD and regulation of membrane fluidity and lipid homeostasis. Nat. Commun. 11, 6215 (2020)." href="https://www.nature.com/articles/s44323-024-00005-1#ref-CR32" id="ref-link-section-d102698130e1498">32</a></sup>. We next incorporated a recently published XBP1 ChIPmentation dataset from mouse T helper cells into our analysis<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 43" title="Pramanik, J. et al. Genome-wide analyses reveal the IRE1a-XBP1 pathway promotes T helper cell differentiation by resolving secretory stress and accelerating proliferation. Genome Med. 10, 76 (2018)." href="https://www.nature.com/articles/s44323-024-00005-1#ref-CR43" id="ref-link-section-d102698130e1502">43</a></sup>. We found that XBP1 target genes—those genes directly bound by XBP1 and whose expression is reduced in the absence of XBP1—were significantly enriched in the putative 10–14-h human gene set, but not in the 22–26-h gene set (Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s44323-024-00005-1#MOESM1">S8A–C</a>). These results suggest XBP1, GABP and KLF transcription factor family members as candidate transcriptional regulators of ~12-h rhythms, with XBP1 a strong candidate given its previously identified role as a major regulator of ~12-h transcriptional oscillations in the murine liver<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 14" title="Pan, Y. et al. 12-h clock regulation of genetic information flow by XBP1s. PLOS Biol. 18, e3000580 (2020)." href="https://www.nature.com/articles/s44323-024-00005-1#ref-CR14" id="ref-link-section-d102698130e1509">14</a></sup>.</p><div data-test="figure" data-container-section="figure" id="figure-3" data-title="Regulatory and functional dissection of human ~ 12-h rhythms."><figure><figcaption><b id="Fig3" data-test="figure-caption-text">Fig. 3: Regulatory and functional dissection of human ~ 12-h rhythms.</b></figcaption><div><div><a data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="https://www.nature.com/articles/s44323-024-00005-1/figures/3" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs44323-024-00005-1/MediaObjects/44323_2024_5_Fig3_HTML.png?as=webp"/><img aria-describedby="Fig3" src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs44323-024-00005-1/MediaObjects/44323_2024_5_Fig3_HTML.png" alt="figure 3" loading="lazy" width="685" height="672"/></picture></a></div><p><b>A</b> Scatter plot demonstrating the log normalized <i>p</i> values of motifs identified for top 500 (ranked by meta <i>p</i> values) ~12-h (x axis) versus the ~24-h genes using the LISA program, with selective TFs highlighted. <b>B</b> Top motifs enriched at the promoters of ~top 500 12-h genes using the SeqPos motif tool in Galaxy/Cistrome. <b>C</b> A table listing the TFs whose motifs are enriched in the promoters of ~12-h genes and whose gene expression also exhibit ~12-h rhythms with meta adj-P less than 0.1. <b>D</b> Temporal expression of <i>XBP1</i> in all three individuals. Criterions for IR are set as <i>T</i> &gt; = 20, <i>J</i> &gt; = 1, FPKM &gt; = 2 and NE score &gt; =0.9. Heatmap (left) and quantification (right) of temporal intron retention events, superimposed with the Z score normalized temporal expression of splicing genes exhibiting ~12-h rhythms (<b>E</b>). Statistics for IR and spicing gene expression ~12-h rhythms detection by RAIN (<b>F</b>).</p></div></figure></div></div><p>Given representation of RNA metabolism and mRNA splicing pathways in ~12-h transcriptional rhythms, we tested whether such oscillations would translate into a downstream functional effect in the form of alterations in mRNA splicing. We interrogated the RNA-seq data for evidence of rhythmicity in intron retention (IR) events, predicting that if rhythmicity of RNA splicing genes was functionally relevant, it would temporally correlate with global IR. We applied a recently published algorithm iREAD<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 45" title="Li, H.-D., Funk, C. C. &amp; Price, N. D. iREAD: a tool for intron retention detection from RNA-seq data. BMC Genom. 21, 128 (2020)." href="https://www.nature.com/articles/s44323-024-00005-1#ref-CR45" id="ref-link-section-d102698130e1570">45</a></sup>. Using two different criteria to define retained introns, we identified 10–12-h rhythms of global IR events in all three participants (with meta-<i>p</i> &lt; 0.003) (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s44323-024-00005-1#Fig3">3E, F</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s44323-024-00005-1#MOESM1">S9A–D</a>). IR rhythms were synchronized to the expression of mRNA splicing genes (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s44323-024-00005-1#Fig3">3E, F</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s44323-024-00005-1#MOESM1">S9A–D</a>). We next performed GO analysis on the set of transcripts in which we detected retained introns. While GO terms associated with intron retention genes were largely consistent at the morning and evening peaks, we also observed differential enrichment in a subset. The morning intron retention gene sets were enriched in immune functions, especially those involving the display of intracellular peptide fragments to cytotoxic T cells via MHC class 1 complexes (e.g., HLA-A/C/E and B2M), whereas the evening intron retention gene sets exhibited greater heterogeneity across the three participants (Figs. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s44323-024-00005-1#MOESM1">S10</a>, <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s44323-024-00005-1#MOESM1">11</a>). This data suggests synchronization of mRNA splicing gene programs to splicing functionality, consistent with a potential role for ~12-h rhythms in RNA metabolism.</p><div><p>~12-h rhythms in coastal and estuarine animals suggest an ancient evolutionary origin<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 2" title="Ballance, H. &amp; Zhu, B. Revealing the hidden reality of the mammalian 12-h ultradian rhythms. Cell. Mol. Life Sci. 
                  https://doi.org/10.1007/s00018-020-03730-5
                  
                 (2021)." href="https://www.nature.com/articles/s44323-024-00005-1#ref-CR2" id="ref-link-section-d102698130e1600">2</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 44" title="Zhu, B. Decoding the function and regulation of the mammalian 12h-clock. J. Mol. Cell Biol. 
                  https://doi.org/10.1093/jmcb/mjaa021
                  
                 (2020)." href="https://www.nature.com/articles/s44323-024-00005-1#ref-CR44" id="ref-link-section-d102698130e1603">44</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Castellana, S. et al. Systematic analysis of mouse genome reveals distinct evolutionary and functional properties among circadian and ultradian genes. Front. Physiol. 9, 1178 (2018)." href="#ref-CR46" id="ref-link-section-d102698130e1606">46</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Schnytzer, Y. et al. Tidal and diel orchestration of behaviour and gene expression in an intertidal mollusc. Sci. Rep. 8, 4917 (2018)." href="#ref-CR47" id="ref-link-section-d102698130e1606_1">47</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Sorek, M. et al. Setting the pace: host rhythmic behaviour and gene expression patterns in the facultatively symbiotic cnidarian Aiptasia are determined largely by symbiodinium. Microbiome 6, 83 (2018)." href="#ref-CR48" id="ref-link-section-d102698130e1606_2">48</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Rinsky, M. et al. Temporal gene expression patterns in the coral Euphyllia paradivisa reveal the complexity of biological clocks in the cnidarian-algal symbiosis. Sci. Adv. 8, eabo6467 (2022)." href="#ref-CR49" id="ref-link-section-d102698130e1606_3">49</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 50" title="Yokomizo, T. &amp; Takahashi, Y. Plasticity of circadian and circatidal rhythms in activity and transcriptomic dynamics in a freshwater snail. Heredity (Edinb). 132, 267–274 (2024)." href="https://www.nature.com/articles/s44323-024-00005-1#ref-CR50" id="ref-link-section-d102698130e1609">50</a></sup> (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s44323-024-00005-1#Fig4">4A</a>). To test for potential homology between human ~12-h rhythms and circatidal genes in marine animals, we compared the most robust 10–14-h human genes (meta-adjP less than 0.10) with the circatidal gene program uncovered in <i>Aiptasia diaphana</i><sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 48" title="Sorek, M. et al. Setting the pace: host rhythmic behaviour and gene expression patterns in the facultatively symbiotic cnidarian Aiptasia are determined largely by symbiodinium. Microbiome 6, 83 (2018)." href="https://www.nature.com/articles/s44323-024-00005-1#ref-CR48" id="ref-link-section-d102698130e1618">48</a></sup>, a sea anemone species which shares a eumetazoan ancestor with <i>Homo sapiens</i> ~ 700 million years ago in the Cryogenian period<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 51" title="Putnam, N. H. et al. Sea anemone genome reveals ancestral eumetazoan gene repertoire and genomic organization. Science 317, 86–94 (2007)." href="https://www.nature.com/articles/s44323-024-00005-1#ref-CR51" id="ref-link-section-d102698130e1625">51</a></sup>. In line with our human data, the most enriched biological pathways amongst <i>A. diaphana</i> circatidal genes are related to mRNA processing and protein metabolism, distinct from those enriched in the circadian genes, which included pathways of carbohydrate metabolism and detoxification<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 48" title="Sorek, M. et al. Setting the pace: host rhythmic behaviour and gene expression patterns in the facultatively symbiotic cnidarian Aiptasia are determined largely by symbiodinium. Microbiome 6, 83 (2018)." href="https://www.nature.com/articles/s44323-024-00005-1#ref-CR48" id="ref-link-section-d102698130e1633">48</a></sup> (Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s44323-024-00005-1#MOESM1">S12A</a>). We found 404 ~ 12-h genes common to both species, a number significantly higher than the expected value of 263 if there was no evolutionary conservation (<i>p</i> = 4.6e-23 by Chi-square test) (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s44323-024-00005-1#Fig4">4B</a>). This subset was also enriched in mRNA splicing and proteostasis pathways (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s44323-024-00005-1#Fig4">4C, D</a>). We further examined potential conservation with two additional human 10–14-h gene sets: 653 gene set with meta-adjP values less than 0.05 (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s44323-024-00005-1#Fig1">1C</a>), or the 851 gene set shared in all three individuals depicted in Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s44323-024-00005-1#MOESM1">S4B</a> (Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s44323-024-00005-1#MOESM1">S12B, C</a>). Protein processing in the ER was the most consistently enriched GO term using the different selection criteria for ~12-h genes (Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s44323-024-00005-1#MOESM1">S12B, C</a>). We also found evidence of conserved ~24-h gene expression between human (with meta-adjP value less than 0.10) and <i>A. diaphana</i> (<i>p</i> = 0.049 by Chi-square test), with the 102 common genes enriched in carbohydrate metabolism and detoxification pathways (Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s44323-024-00005-1#MOESM1">S12D–F</a>).</p><div data-test="figure" data-container-section="figure" id="figure-4" data-title="Evolutionary conservation of ~ 12-h gene programs."><figure><figcaption><b id="Fig4" data-test="figure-caption-text">Fig. 4: Evolutionary conservation of ~ 12-h gene programs.</b></figcaption><div><div><a data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="https://www.nature.com/articles/s44323-024-00005-1/figures/4" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs44323-024-00005-1/MediaObjects/44323_2024_5_Fig4_HTML.png?as=webp"/><img aria-describedby="Fig4" src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs44323-024-00005-1/MediaObjects/44323_2024_5_Fig4_HTML.png" alt="figure 4" loading="lazy" width="685" height="902"/></picture></a></div><p><b>A</b> Phylogenetic tree of select species for which ~12-h rhythms of gene expression have been demonstrated. <i>A. diaphana</i> is the most distant from <i>H. sapiens</i>. <b>B</b> Venn diagram comparing distinct and shared ~12-h genes in human (meta adj-<i>P</i> &lt; 0.1) and <i>A. diaphana</i> (reported in<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 48" title="Sorek, M. et al. Setting the pace: host rhythmic behaviour and gene expression patterns in the facultatively symbiotic cnidarian Aiptasia are determined largely by symbiodinium. Microbiome 6, 83 (2018)." href="https://www.nature.com/articles/s44323-024-00005-1#ref-CR48" id="ref-link-section-d102698130e1702">48</a></sup>). Only genes expressed in human white blood cells (denoted by the grey circle) are included in the analysis. Both observed and predicted number of genes (under the null hypothesis that ~12-h genes are not evolutionarily conserved and thus independently detected in these two species) are further shown. <i>P</i> value of 4.6e-23 is calculated by Chi-square test. Heatmap of temporal expression (Z score normalized) of 404 circatidal genes in <i>A. diaphana</i> (<b>C</b>) and GO analysis of the top enriched pathways (<b>D</b>). <b>E</b> Venn diagram comparing distinct and shared ~12-h genes in human (meta adj-P &lt; 0.1) and mouse liver (reported in ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 14" title="Pan, Y. et al. 12-h clock regulation of genetic information flow by XBP1s. PLOS Biol. 18, e3000580 (2020)." href="https://www.nature.com/articles/s44323-024-00005-1#ref-CR14" id="ref-link-section-d102698130e1722">14</a></sup>). Only genes that are expressed in human white blood cells (denoted by the grey circle) are included in the analysis. Both observed and predicted number of genes (under the null hypothesis that ~12-h genes are not evolutionarily conserved and thus independently detected in these two species) are further shown. <i>P</i> value of 2.0e-21 is calculated by Chi-square test. Heatmap of temporal expression (log 2 normalized) of 666 ~ 12-h genes in mouse liver (<b>F</b>) and GO analysis of the top enriched pathways (<b>G</b>). <b>H</b> Venn diagram further comparing the overlap of commonly identified 404 ~ 12-h genes in human and A. diaphana, and 666 ~ 12-h genes in human and mouse. Both observed and predicted number of genes (under the null hypothesis that ~12-h genes are not evolutionarily conserved and thus independently detected in all three species) are further shown. <i>P</i> value of 1.4e-7 is calculated by Chi-square test. <b>I</b> GO analysis of the top enriched pathways of 139 ~ 12-h genes commonly identified in all three species. <b>J</b> Representative temporal expression of evolutionarily conserved ~12-h genes in human, mouse, and <i>A. diaphana</i>. <b>K</b> ~ 12-h rhythms of intron retention events (top) and RNA splicing genes expression (bottom) are attenuated by liver-specific loss of function of XBP1 (<i>Xbp1</i><sup><i>LKO</i></sup>: red), relative to control (<i>Xbp1</i><sup><i>Flox</i></sup>: black) in mice. <i>P</i> values calculated by RAIN for ~12-h rhythms.</p></div></figure></div></div><p>If the ~12-h gene program is conserved from coastal invertebrates to humans, we reasoned that lower mammals like mice should also exhibit oscillations in similar gene programs. We re-examined a previously published temporal hepatic RNA-seq data set in mouse liver and found 666 ~ 12-h genes common to both human and mouse (<i>p</i> = 2.02e-21 by Chi-square test), again highly enriched in mRNA splicing and protein processing pathways (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s44323-024-00005-1#Fig4">4E–G</a>). We identified 139 ~ 12-h genes common to all three species: human, mouse, and <i>A. diaphana</i>, which exceeded the 97 expected by chance (<i>p</i> = 1.41e-7 by Chi-square test) (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s44323-024-00005-1#Fig4">4H</a>). These 139 genes are enriched in protein processing in ER, spliceosome, and nucleocytoplasmic transport, the three pivotal steps in central dogma information flow. Specific examples include DNAJA1, a heat shock protein 70 cochaperones<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 52" title="Rodríguez-González, C., Lin, S., Arkan, S. &amp; Hansen, C. Co-chaperones DNAJA1 and DNAJB6 are critical for regulation of polyglutamine aggregation. Sci. Rep. 10, 8130 (2020)." href="https://www.nature.com/articles/s44323-024-00005-1#ref-CR52" id="ref-link-section-d102698130e1797">52</a></sup>, UGGT1 that serves as the predominant ER glycoprotein quality control sensor<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 53" title="Adams, B. M., Canniff, N. P., Guay, K. P., Larsen, I. S. B. &amp; Hebert, D. N. Quantitative glycoproteomics reveals cellular substrate selectivity of the ER protein quality control sensors UGGT1 and UGGT2. eLife 9, e63997 (2020)." href="https://www.nature.com/articles/s44323-024-00005-1#ref-CR53" id="ref-link-section-d102698130e1801">53</a></sup> and <i>EIF2S1</i>, encoding the alpha subunit of the translation initiation factor eIF2 protein complex<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 54" title="Boye, E. &amp; Grallert, B. eIF2α phosphorylation and the regulation of translation. Curr. Genet. 66, 293–297 (2020)." href="https://www.nature.com/articles/s44323-024-00005-1#ref-CR54" id="ref-link-section-d102698130e1808">54</a></sup> (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s44323-024-00005-1#Fig4">4I, J</a>). These results are congruent with our previous work in murine liver in identifying a regulatory function for ~12-h oscillator in central dogma information flow<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 14" title="Pan, Y. et al. 12-h clock regulation of genetic information flow by XBP1s. PLOS Biol. 18, e3000580 (2020)." href="https://www.nature.com/articles/s44323-024-00005-1#ref-CR14" id="ref-link-section-d102698130e1815">14</a></sup>.</p><p>Finally, we tested for synchronization between the ~12-h rhythms and RNA splicing in mice by performing intron retention analysis of our previously published temporal RNA-seq data from mouse liver<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 14" title="Pan, Y. et al. 12-h clock regulation of genetic information flow by XBP1s. PLOS Biol. 18, e3000580 (2020)." href="https://www.nature.com/articles/s44323-024-00005-1#ref-CR14" id="ref-link-section-d102698130e1822">14</a></sup>. Like the human data, we observed alignment between ~12-h rhythms in splicing gene expression and global intron retention events peaking at CT2 and CT14 (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s44323-024-00005-1#Fig4">4K</a>). Liver-specific genetic ablation of XBP1 (a transcriptional regulator of murine hepatic 12-h oscillations) weakened ~12-h rhythms of splicing gene expression and intron retention rhythms, leading to constant intron retention across the day (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s44323-024-00005-1#Fig4">4K</a>). These collective data support evolutionary conservation of a ~ 12-h gene program in humans related to RNA and protein metabolism.</p><p>In this study, we discovered ~12-h gene programs related to fundamental processes of mRNA and protein metabolism in humans. Remarkably, pathways and specific gene orthologs exhibiting ~12-h circatidal rhythms in marine animals<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 55" title="Zhang, H. et al. Circatidal control of gene expression in the deep-sea hot vent shrimp &lt;em&gt;Rimicaris leurokolos&lt;/em&gt;. bioRxiv, 2024.2001.2012.575359. 
                  https://doi.org/10.1101/2024.01.12.575359
                  
                 (2024)." href="https://www.nature.com/articles/s44323-024-00005-1#ref-CR55" id="ref-link-section-d102698130e1835">55</a></sup> were also detected in humans. Our discovery of one such ~12-h pathway, ‘mRNA splicing,’ provided an opportunity to establish functional significance through identification of corresponding ~12-h rhythms in global intron retention events. While our study provides evidence of ~12-h rhythms of gene expression in humans, it does not establish causality, nor does it fully address their relationship with the circadian clock. At least three mutually non-exclusive mechanisms have been proposed to explain the origin and regulatory mechanisms of ~12-h rhythms in mice, namely that they are not cell-autonomous and controlled by a combination of the circadian clock and environmental cues<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 26" title="Hughes, M. E. et al. Harmonics of circadian gene transcription in mammals. PLoS Genet. 5, e1000442 (2009)." href="https://www.nature.com/articles/s44323-024-00005-1#ref-CR26" id="ref-link-section-d102698130e1839">26</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 56" title="Cretenet, G., Le Clech, M. &amp; Gachon, F. Circadian clock-coordinated 12 h period rhythmic activation of the IRE1alpha pathway controls lipid metabolism in mouse liver. Cell Metab. 11, 47–57 (2010)." href="https://www.nature.com/articles/s44323-024-00005-1#ref-CR56" id="ref-link-section-d102698130e1842">56</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 57" title="Hughes, M. E. et al. Brain-specific rescue of Clock reveals system-driven transcriptional rhythms in peripheral tissue. PLoS Genet. 8, e1002835 (2012)." href="https://www.nature.com/articles/s44323-024-00005-1#ref-CR57" id="ref-link-section-d102698130e1845">57</a></sup>, that they are regulated by two anti-phase circadian transcriptional factors in a cell-autonomous manner<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 58" title="Westermark, P. O. &amp; Herzel, H. Mechanism for 12 h rhythm generation by the circadian clock. Cell Rep. 3, 1228–1238 (2013)." href="https://www.nature.com/articles/s44323-024-00005-1#ref-CR58" id="ref-link-section-d102698130e1849">58</a></sup>, or that they are established by a cell-autonomous ~12-h oscillator<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 13" title="Zhu, B. et al. A cell-autonomous mammalian 12 h clock coordinates metabolic and stress rhythms. Cell Metab. 25, 1305–1319.e1309 (2017)." href="https://www.nature.com/articles/s44323-024-00005-1#ref-CR13" id="ref-link-section-d102698130e1853">13</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 14" title="Pan, Y. et al. 12-h clock regulation of genetic information flow by XBP1s. PLOS Biol. 18, e3000580 (2020)." href="https://www.nature.com/articles/s44323-024-00005-1#ref-CR14" id="ref-link-section-d102698130e1856">14</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Meng, H. et al. XBP1 links the 12-hour clock to NAFLD and regulation of membrane fluidity and lipid homeostasis. Nat. Commun. 11, 6215 (2020)." href="#ref-CR32" id="ref-link-section-d102698130e1859">32</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Dion, W. et al. Four-dimensional nuclear speckle phase separation dynamics regulate proteostasis. Sci. Adv. 8, eabl4150 (2022)." href="#ref-CR33" id="ref-link-section-d102698130e1859_1">33</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 34" title="Meng, H. et al. Defining the mammalian coactivation of hepatic 12-h clock and lipid metabolism. Cell Rep. 38, 110491 (2022)." href="https://www.nature.com/articles/s44323-024-00005-1#ref-CR34" id="ref-link-section-d102698130e1862">34</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 59" title="Zhu, B. &amp; Liu, S. Preservation of ∼12-h ultradian rhythms of gene expression of mRNA and protein metabolism in the absence of canonical circadian clock. Front. Physiol. 14. 
                  https://doi.org/10.3389/fphys.2023.1195001
                  
                 (2023)." href="https://www.nature.com/articles/s44323-024-00005-1#ref-CR59" id="ref-link-section-d102698130e1865">59</a></sup>. Future studies will be required to test whether candidate 12-h regulatory factors are causal drivers of ~12-h gene expression rhythms and any direct relationship with the circadian clock in humans.</p><p>How ~12-h rhythms in mRNA and protein metabolism confer an evolutionary advantage to non-coastal terrestrial mammals is an open question. One possibility is that ~12-h rhythms have been co-opted by terrestrial mammals as an adaptive response to ~12-h metabolic stress cycles that peak at critical transitions<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 10" title="Zhu, B., Dacso, C. C. &amp; O’Malley, B. W. Unveiling “musica universalis” of the cell: a brief history of biological 12-hour rhythms. J. Endocr. Soc. 2, 727–752 (2018)." href="https://www.nature.com/articles/s44323-024-00005-1#ref-CR10" id="ref-link-section-d102698130e1872">10</a></sup>. The absence of energy intake during sleep crescendos to a peak of energy shortage at dawn. Conversely, at dusk, a reduction in active energy expenditure coupled with cumulative energy intake throughout the day, drives peak positive energy balance. Both ends of the energy balance spectrum are metabolically stressful and activate XBP1s, thereby triggering the UPR<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 44" title="Zhu, B. Decoding the function and regulation of the mammalian 12h-clock. J. Mol. Cell Biol. 
                  https://doi.org/10.1093/jmcb/mjaa021
                  
                 (2020)." href="https://www.nature.com/articles/s44323-024-00005-1#ref-CR44" id="ref-link-section-d102698130e1876">44</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 60" title="Deng, Y. et al. The Xbp1s/GalE axis links ER stress to postprandial hepatic metabolism. J. Clin. Investig. 123, 455–468 (2013)." href="https://www.nature.com/articles/s44323-024-00005-1#ref-CR60" id="ref-link-section-d102698130e1879">60</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 61" title="Shao, M. et al. Hepatic IRE1alpha regulates fasting-induced metabolic adaptive programs through the XBP1s-PPARalpha axis signalling. Nat. Commun. 5, 3528 (2014)." href="https://www.nature.com/articles/s44323-024-00005-1#ref-CR61" id="ref-link-section-d102698130e1882">61</a></sup>. Therefore, we speculate that mammalian ~12-h rhythms provide an anticipatory advantage to address twice daily metabolic stress.</p><section data-title="Methods"><div id="Sec1-section"><h2 id="Sec1">Methods</h2><div id="Sec1-content"><h3 id="Sec2">Human participants and study protocol</h3><p>The study protocol was approved by the University of Pittsburgh Institutional Review Board (Study 20020034; approval date: 6/4/2020) and written consent was obtained from all study participants. We studied 3 healthy male participants, who were recruited through online advertisements. Inclusion criteria consisted of individuals 18–35 years of age with a self-reported regular nighttime sleep schedule and a body mass index between 18.5–24.9 kg/m<sup>2</sup>. Volunteers were excluded if they admitted to nighttime shift work or other regular nighttime sleep-disrupting activities, if they had any chronic medical conditions, took any medications or recreational drugs, or used tobacco products. Potential volunteers presented for a screening visit, inclusive of measurement of body weight, height, BMI, and laboratory studies, including a comprehensive metabolic panel (electrolytes, kidney function and liver function tests), complete blood count, 25-OH vitamin D, and thyroid stimulating hormone level, to screen for potential subclinical chronic diseases. We excluded participants with low hemoglobin/hematocrit, abnormal thyroid function and individuals with 25-OH vitamin D &lt; 20 ng/mL.</p><p>Qualifying study participants maintained a food diary, which was used to estimate their daily caloric intake and subsequently presented to the University of Pittsburgh Medical Center (UPMC) Clinical Translational Research Center (CTRC) for a 3-day inpatient visit. On the morning of admission, participants selected items from a food menu designed to match their standard daily caloric intake with a uniform macronutrient composition of 55% carbohydrates, 25% fat, 20% protein per day. No interventions were performed during the first 24-h period of acclimatization to the hospital. Each study participant was housed in the same room in the CTRC, which contained a window to the outside. The participants were not entrained to any specific environmental cues during the study. They were encouraged to maintain their usual sleep-wake cycles; however, they maintained control over the lighting in the room. They were free to ambulate in the hallway of the CTRC; however, they did not leave the unit during the 3-day stay. The three meals were delivered at roughly the same time each day based on the standard CTRC schedule (8 A.M., 12 P.M., 6 P.M.); however, there were no rules dictating the duration of their meals. These measures were implemented to mimic free-living conditions. On the morning of the second day, an intravenous (IV) line was placed to commence blood collections at 8 A.M. and then every 2 h for 48 h (total 24 samples). Nighttime blood collection was performed through a long IV line from outside the room so that the participant would not be exposed to light or disturbed during blood collection. Blood was immediately processed in the Center for Human Integrative Physiology, two floors above the UPMC CTRC by a rotating study team, all of whom were trained in the processing procedures for this study. Blood was centrifuged (1900 RCF x 10 min) and the buffy coats were collected and immediately snapped frozen in liquid nitrogen for storage at −80C.</p><h3 id="Sec3">RNA-Seq and data analysis</h3><p>RNA was isolated from peripheral blood buffy coat samples on the automated Chemagic 360 (Perkin Elmer) instrument according to the manufacturer’s instructions. Extracted RNA was quantitated by Qubit™ RNA BR Assay Kit (Thermo Fisher Scientific) followed by RNA quality check using Fragment Analyzer (Agilent). For each sample, RNA libraries were prepared from 100 ng RNA using the KAPA RNA HyperPrep Kit with RiboErase (Kapa Biosystems) according to manufacturer’s protocol, followed by quality check using Fragment Analyzer (Agilent) and quantification by qPCR (Kapa qPCR quantification kit, Kapa biosystem) on the LightCycler 480 (Roche). The libraries were normalized and pooled, and then sequenced using NovaSeq6000 platform (Illumina) to an average of 40 M 101 bp paired end reads per sample. Low-quality reads and adapter sequences were trimmed from the raw sequencing data with Trimmomatic<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 62" title="Bolger, A. M., Lohse, M. &amp; Usadel, B. Trimmomatic: a flexible trimmer for Illumina sequence data. Bioinformatics 30, 2114–2120 (2014)." href="https://www.nature.com/articles/s44323-024-00005-1#ref-CR62" id="ref-link-section-d102698130e1910">62</a></sup>. The remaining reads were then aligned to human reference genome hg38 with STAR aligner<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 63" title="Dobin, A. et al. STAR: ultrafast universal RNA-seq aligner. Bioinformatics 29, 15–21 (2013)." href="https://www.nature.com/articles/s44323-024-00005-1#ref-CR63" id="ref-link-section-d102698130e1914">63</a></sup>. Gene counts were quantified with the STAR-quantMode GeneCounts function. Fragments per kilobase of transcript per million mapped fragments (FPKM) were quantified with Cufflinks<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 64" title="Trapnell, C. et al. Differential gene and transcript expression analysis of RNA-seq experiments with TopHat and Cufflinks. Nat. Protoc. 7, 562–578 (2012)." href="https://www.nature.com/articles/s44323-024-00005-1#ref-CR64" id="ref-link-section-d102698130e1918">64</a></sup>.</p><h3 id="Sec4">Identification of the oscillating transcriptome</h3><p>Averaged FPKM values at each time were used for cycling transcripts identification. Lowly expressed transcripts were removed by calculating the background expression in each participant using the average expression of a panel of 62 genes known not to be expressed in peripheral blood cells (Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s44323-024-00005-1#MOESM3">S2</a>). Temporal transcriptomes were subjected to linear detrend prior to identification of oscillations by either the eigenvalue/pencil or RAIN methods. For the eigenvalue/pencil method<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 11" title="Antoulas, A. C. et al. A novel mathematical method for disclosing oscillations in gene transcription: a comparative study. PLoS One 13, e0198503 (2018)." href="https://www.nature.com/articles/s44323-024-00005-1#ref-CR11" id="ref-link-section-d102698130e1933">11</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 13" title="Zhu, B. et al. A cell-autonomous mammalian 12 h clock coordinates metabolic and stress rhythms. Cell Metab. 25, 1305–1319.e1309 (2017)." href="https://www.nature.com/articles/s44323-024-00005-1#ref-CR13" id="ref-link-section-d102698130e1936">13</a></sup>, a maximum of four oscillations were identified for each gene. Criterion for ~24-h genes were: period between 20 h to 25 h for first and second participants and 24 h to 30 h for the third participant, decay rate between 0.8 and 1.2; for ~12-h genes: period between 9.6 h to 13.6 h for the second and third participants and 10 h to 14 h for the first participant, decay rate between 0.8 and 1.2; for ~8 h genes: period between 6 h to 8 h for the first participant and 7 h to 9 h for the second participant, decay rate between 0.8 and 1.2; for ~16 h genes; period between 14 h to 18 h for the third participant. The relative amplitude was calculated by dividing the amplitude by the mean expression value for each gene. To determine FDR, we used a permutation-based method that randomly shuffles the time label of gene expression data and subjected each permutation dataset to the eigenvalue/pencil method applied with the same criterion<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 65" title="Rey, G. et al. Metabolic oscillations on the circadian time scale in Drosophila cells lacking clock genes. Mol. Syst. Biol. 14, e8376 (2018)." href="https://www.nature.com/articles/s44323-024-00005-1#ref-CR65" id="ref-link-section-d102698130e1940">65</a></sup>. These permutation tests were run 5000 times, and FDR was estimated by taking the ratio between the mean number of rhythmic profiles identified in the permutated samples (false positives) and the number of rhythmic profiles identified in the original data. Analyses were performed in MatlabR2017A. RAIN analysis was performed as previously described in Bioconductor (3.4) (<a href="http://www.bioconductor.org/packages/release/bioc/html/rain.html">http://www.bioconductor.org/packages/release/bioc/html/rain.html</a>) with either 48-h continuous data or 24-h data with biological duplicates as input<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 12" title="Thaben, P. F. &amp; Westermark, P. O. Detecting rhythms in time series with RAIN. J. Biol. Rhythms 29, 391–400 (2014)." href="https://www.nature.com/articles/s44323-024-00005-1#ref-CR12" id="ref-link-section-d102698130e1951">12</a></sup>. Genes exhibiting a period range between 10-h and 14-h and a period range between 22-h and 26-h are considered as ~12-h and ~24-h genes, respectively. FDR was calculated using the Benjamini-Hochberg procedure. Heat maps were generated with Gene Cluster 3.0 and TreeView 3.0 alpha 3.0 using Z score normalized values.</p><p>For meta-analysis, we used Fisher’s method, which combines extreme value probabilities from each test, commonly known as “<i>p</i> values”, into one test statistic (X2) using the formula</p><div id="Equ1"><p><span>$${X}_{2k}^{2}=-2{\sum \limits_{i=1}^{k}}{\mathrm{ln}}\,{p}_{i}$$</span></p><p>
                    (1)
                </p></div><p>where pi is the <i>p</i> value for the ith hypothesis test. For meta-analysis of ~12-h (10–14-h) and ~24-h genes (22–26-h), the smallest <i>p</i> value for each gene within these period ranges was used for each individual. For example, <i>XBP1</i> was found to oscillate with periods of 12-h (<i>p</i> = 0.068), 12-h (<i>p</i> = 0.014) and 10-h (<i>p</i> = 0.049) in participant 1, 2, and 3, respectively, and the meta <i>p</i> values for being a common ~12-h gene (cycling with a period between 10-h and 14-h) were calculated by “merging” the three respective <i>p</i> values (meta <i>p</i> = 0.0029). The same procedure was also performed for genes cycling with a period between 22-h and 26-h. This meta-analysis is feasible because the statistical test for each gene in each individual shares the same null hypothesis: absence of rhythms.</p><p>For RAIN analysis on temporal IR events and splicing gene expression, raw data was subjected to polynomial detrend (<i>n</i> = 2) before RAIN analysis.</p><h3 id="Sec5">Defining oscillating genes</h3><p>The eigenvalue method can detect multiple superimposed oscillations. Therefore, we defined a ~ 24-h gene as one that exhibited a ~ 24-h rhythm, regardless of its amplitude relative to other superimposed oscillations. Similar criteria were applied to other oscillations. As such, a gene can meet criteria for both a ~ 24-h and ~12-h gene. By comparison, we define a dominant ~24-h gene as one in which the superimposed ~24-h rhythm has the largest amplitude among all oscillations. With this definition, dominant ~24-h and dominant ~12-h genes are mutually exclusive.</p><h3 id="Sec6">Intron retention detection</h3><p>Intron retention events were detected by tool iREAD<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 45" title="Li, H.-D., Funk, C. C. &amp; Price, N. D. iREAD: a tool for intron retention detection from RNA-seq data. BMC Genom. 21, 128 (2020)." href="https://www.nature.com/articles/s44323-024-00005-1#ref-CR45" id="ref-link-section-d102698130e2100">45</a></sup>. Intron retention events are selected either with default settings <i>T</i> &gt; = 20, <i>J</i> &gt; = 1, FPKM &gt; = 2 and NE score &gt; =0.9 or more stringent settings where <i>T</i> &gt; = 20, <i>J</i> &gt; = 1, FPKM &gt; = 3 and NE score &gt; =0.9.</p><h3 id="Sec7">Gene ontology analysis</h3><p>DAVID (Version 2021)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 66" title="Huang da, W., Sherman, B. T. &amp; Lempicki, R. A. Systematic and integrative analysis of large gene lists using DAVID bioinformatics resources. Nat. Protoc. 4, 44–57 (2009)." href="https://www.nature.com/articles/s44323-024-00005-1#ref-CR66" id="ref-link-section-d102698130e2124">66</a></sup> (<a href="https://david.ncifcrf.gov">https://david.ncifcrf.gov</a>) was used to perform Gene Ontology analyses. Briefly, gene names were first converted to DAVID-recognizable IDs using Gene Accession Conversion Tool. The updated gene list was then subject to GO analysis using all Homo Sapiens as background and with Functional Annotation Chart function. GO_BP_DIRECT, KEGG_PATHWAY or UP_KW_BIOLOGICAL_PROCESS were used as GO categories. Only GO terms with a <i>p</i> value less than 0.05 were included for further analysis.</p><h3 id="Sec8">Motif analysis</h3><p>Motif analysis was performed with the SeqPos motif tool (version 0.590) embedded in Galaxy Cistrome using all motifs within the homo sapiens reference genome hg19 as background. LISA analysis was performed using web tool (<a href="http://lisa.cistrome.org/">http://lisa.cistrome.org/</a>).</p></div></div></section>
                </div><div>
                <section data-title="Data availability"><div id="data-availability-section"><h2 id="data-availability">Data availability</h2><p>All raw and processed sequencing data generated in this study have been submitted to the NCBI Gene Expression Omnibus (GEO; <a href="http://www.ncbi.nlm.nih.gov/geo/">http://www.ncbi.nlm.nih.gov/geo/</a>) under accession numbers GSE220120.</p></div></section><div id="MagazineFulltextArticleBodySuffix"><section aria-labelledby="Bib1" data-title="References"><div id="Bib1-section"><h2 id="Bib1">References</h2><div id="Bib1-content"><div data-container-section="references"><ol data-track-component="outbound reference" data-track-context="references section"><li data-counter="1."><p id="ref-CR1">Andreatta, G. &amp; Tessmar-Raible, K. The still dark side of the moon: molecular mechanisms of lunar-controlled rhythms and clocks. <i>J. Mol. Biol.</i> <b>432</b>, 3525–3546 (2020).</p><p><a data-track="click||click_references" rel="nofollow noopener" data-track-label="10.1016/j.jmb.2020.03.009" data-track-item_id="10.1016/j.jmb.2020.03.009" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.jmb.2020.03.009" aria-label="Article reference 1" data-doi="10.1016/j.jmb.2020.03.009">Article</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXltlCktbc%3D" aria-label="CAS reference 1">CAS</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32198116" aria-label="PubMed reference 1">PubMed</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7322537" aria-label="PubMed Central reference 1">PubMed Central</a> 
    <a data-track="click||click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 1" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20still%20dark%20side%20of%20the%20moon%3A%20molecular%20mechanisms%20of%20lunar-controlled%20rhythms%20and%20clocks&amp;journal=J.%20Mol.%20Biol.&amp;doi=10.1016%2Fj.jmb.2020.03.009&amp;volume=432&amp;pages=3525-3546&amp;publication_year=2020&amp;author=Andreatta%2CG&amp;author=Tessmar-Raible%2CK">
                    Google Scholar</a> 
                </p></li><li data-counter="2."><p id="ref-CR2">Ballance, H. &amp; Zhu, B. Revealing the hidden reality of the mammalian 12-h ultradian rhythms. <i>Cell. Mol. Life Sci.</i> <a href="https://doi.org/10.1007/s00018-020-03730-5" data-track="click||click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1007/s00018-020-03730-5">https://doi.org/10.1007/s00018-020-03730-5</a> (2021).</p></li><li data-counter="3."><p id="ref-CR3">Asher, G. &amp; Zhu, B. Beyond circadian rhythms: emerging roles of ultradian rhythms in control of liver functions. <i>Hepatology</i> <a href="https://doi.org/10.1002/hep.32580" data-track="click||click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1002/hep.32580">https://doi.org/10.1002/hep.32580</a> (2022).</p></li><li data-counter="4."><p id="ref-CR4">Broughton, R. &amp; Mullington, J. Circasemidian sleep propensity and the phase-amplitude maintenance model of human sleep/wake regulation. <i>J. Sleep Res.</i> <b>1</b>, 93–98 (1992).</p><p><a data-track="click||click_references" rel="nofollow noopener" data-track-label="10.1111/j.1365-2869.1992.tb00017.x" data-track-item_id="10.1111/j.1365-2869.1992.tb00017.x" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1111%2Fj.1365-2869.1992.tb00017.x" aria-label="Article reference 4" data-doi="10.1111/j.1365-2869.1992.tb00017.x">Article</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:STN:280:DC%2BC2sbht1Crsg%3D%3D" aria-label="CAS reference 4">CAS</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=10607032" aria-label="PubMed reference 4">PubMed</a> 
    <a data-track="click||click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 4" href="http://scholar.google.com/scholar_lookup?&amp;title=Circasemidian%20sleep%20propensity%20and%20the%20phase-amplitude%20maintenance%20model%20of%20human%20sleep%2Fwake%20regulation&amp;journal=J.%20Sleep%20Res.&amp;doi=10.1111%2Fj.1365-2869.1992.tb00017.x&amp;volume=1&amp;pages=93-98&amp;publication_year=1992&amp;author=Broughton%2CR&amp;author=Mullington%2CJ">
                    Google Scholar</a> 
                </p></li><li data-counter="5."><p id="ref-CR5">Hayashi, M., Morikawa, T. &amp; Hori, T. Circasemidian 12 h cycle of slow wave sleep under constant darkness. <i>Clin. Neurophysiol.</i> <b>113</b>, 1505–1516 (2002).</p><p><a data-track="click||click_references" rel="nofollow noopener" data-track-label="10.1016/S1388-2457(02)00168-2" data-track-item_id="10.1016/S1388-2457(02)00168-2" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2FS1388-2457%2802%2900168-2" aria-label="Article reference 5" data-doi="10.1016/S1388-2457(02)00168-2">Article</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=12169334" aria-label="PubMed reference 5">PubMed</a> 
    <a data-track="click||click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 5" href="http://scholar.google.com/scholar_lookup?&amp;title=Circasemidian%2012%20h%20cycle%20of%20slow%20wave%20sleep%20under%20constant%20darkness&amp;journal=Clin.%20Neurophysiol.&amp;doi=10.1016%2FS1388-2457%2802%2900168-2&amp;volume=113&amp;pages=1505-1516&amp;publication_year=2002&amp;author=Hayashi%2CM&amp;author=Morikawa%2CT&amp;author=Hori%2CT">
                    Google Scholar</a> 
                </p></li><li data-counter="6."><p id="ref-CR6">Wan, C., Wang, Z., Cornelissen, G. &amp; Halberg, F. Age, gender and circadian or circasemidian blood pressure and heart rate variation of children. <i>Chronobiologia</i> <b>19</b>, 121–129 (1992).</p><p><a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:STN:280:DyaK3s7itFyhsg%3D%3D" aria-label="CAS reference 6">CAS</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=1478113" aria-label="PubMed reference 6">PubMed</a> 
    <a data-track="click||click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 6" href="http://scholar.google.com/scholar_lookup?&amp;title=Age%2C%20gender%20and%20circadian%20or%20circasemidian%20blood%20pressure%20and%20heart%20rate%20variation%20of%20children&amp;journal=Chronobiologia&amp;volume=19&amp;pages=121-129&amp;publication_year=1992&amp;author=Wan%2CC&amp;author=Wang%2CZ&amp;author=Cornelissen%2CG&amp;author=Halberg%2CF">
                    Google Scholar</a> 
                </p></li><li data-counter="7."><p id="ref-CR7">Otsuka, K. et al. Long-term exposure to space’s microgravity alters the time structure of heart rate variability of astronauts. <i>Heliyon</i> <b>2</b>, e00211 (2016).</p><p><a data-track="click||click_references" rel="nofollow noopener" data-track-label="10.1016/j.heliyon.2016.e00211" data-track-item_id="10.1016/j.heliyon.2016.e00211" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.heliyon.2016.e00211" aria-label="Article reference 7" data-doi="10.1016/j.heliyon.2016.e00211">Article</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28050606" aria-label="PubMed reference 7">PubMed</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5192238" aria-label="PubMed Central reference 7">PubMed Central</a> 
    <a data-track="click||click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 7" href="http://scholar.google.com/scholar_lookup?&amp;title=Long-term%20exposure%20to%20space%E2%80%99s%20microgravity%20alters%20the%20time%20structure%20of%20heart%20rate%20variability%20of%20astronauts&amp;journal=Heliyon&amp;doi=10.1016%2Fj.heliyon.2016.e00211&amp;volume=2&amp;publication_year=2016&amp;author=Otsuka%2CK">
                    Google Scholar</a> 
                </p></li><li data-counter="8."><p id="ref-CR8">Otsuka, K. et al. Methods for assessing change in brain plasticity at night and psychological resilience during daytime between repeated long-duration space missions. <i>Sci. Rep.</i> <b>13</b>, 10909 (2023).</p><p><a data-track="click||click_references" rel="nofollow noopener" data-track-label="10.1038/s41598-023-36389-6" data-track-item_id="10.1038/s41598-023-36389-6" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41598-023-36389-6" aria-label="Article reference 8" data-doi="10.1038/s41598-023-36389-6">Article</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXhsVejtrrO" aria-label="CAS reference 8">CAS</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=37407662" aria-label="PubMed reference 8">PubMed</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10322820" aria-label="PubMed Central reference 8">PubMed Central</a> 
    <a data-track="click||click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 8" href="http://scholar.google.com/scholar_lookup?&amp;title=Methods%20for%20assessing%20change%20in%20brain%20plasticity%20at%20night%20and%20psychological%20resilience%20during%20daytime%20between%20repeated%20long-duration%20space%20missions&amp;journal=Sci.%20Rep.&amp;doi=10.1038%2Fs41598-023-36389-6&amp;volume=13&amp;publication_year=2023&amp;author=Otsuka%2CK">
                    Google Scholar</a> 
                </p></li><li data-counter="9."><p id="ref-CR9">Otsuka, K. et al. Appropriate circadian-circasemidian coupling protects blood pressure from morning surge and promotes human resilience and wellbeing. <i>Clin. Interv. Aging</i> <b>18</b>, 755–769 (2023).</p><p><a data-track="click||click_references" rel="nofollow noopener" data-track-label="10.2147/CIA.S398957" data-track-item_id="10.2147/CIA.S398957" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.2147%2FCIA.S398957" aria-label="Article reference 9" data-doi="10.2147/CIA.S398957">Article</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=37193339" aria-label="PubMed reference 9">PubMed</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10183193" aria-label="PubMed Central reference 9">PubMed Central</a> 
    <a data-track="click||click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 9" href="http://scholar.google.com/scholar_lookup?&amp;title=Appropriate%20circadian-circasemidian%20coupling%20protects%20blood%20pressure%20from%20morning%20surge%20and%20promotes%20human%20resilience%20and%20wellbeing&amp;journal=Clin.%20Interv.%20Aging&amp;doi=10.2147%2FCIA.S398957&amp;volume=18&amp;pages=755-769&amp;publication_year=2023&amp;author=Otsuka%2CK">
                    Google Scholar</a> 
                </p></li><li data-counter="10."><p id="ref-CR10">Zhu, B., Dacso, C. C. &amp; O’Malley, B. W. Unveiling “musica universalis” of the cell: a brief history of biological 12-hour rhythms. <i>J. Endocr. Soc.</i> <b>2</b>, 727–752 (2018).</p><p><a data-track="click||click_references" rel="nofollow noopener" data-track-label="10.1210/js.2018-00113" data-track-item_id="10.1210/js.2018-00113" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1210%2Fjs.2018-00113" aria-label="Article reference 10" data-doi="10.1210/js.2018-00113">Article</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhvFyltLo%3D" aria-label="CAS reference 10">CAS</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29978151" aria-label="PubMed reference 10">PubMed</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6025213" aria-label="PubMed Central reference 10">PubMed Central</a> 
    <a data-track="click||click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 10" href="http://scholar.google.com/scholar_lookup?&amp;title=Unveiling%20%E2%80%9Cmusica%20universalis%E2%80%9D%20of%20the%20cell%3A%20a%20brief%20history%20of%20biological%2012-hour%20rhythms&amp;journal=J.%20Endocr.%20Soc.&amp;doi=10.1210%2Fjs.2018-00113&amp;volume=2&amp;pages=727-752&amp;publication_year=2018&amp;author=Zhu%2CB&amp;author=Dacso%2CCC&amp;author=O%E2%80%99Malley%2CBW">
                    Google Scholar</a> 
                </p></li><li data-counter="11."><p id="ref-CR11">Antoulas, A. C. et al. A novel mathematical method for disclosing oscillations in gene transcription: a comparative study. <i>PLoS One</i> <b>13</b>, e0198503 (2018).</p><p><a data-track="click||click_references" rel="nofollow noopener" data-track-label="10.1371/journal.pone.0198503" data-track-item_id="10.1371/journal.pone.0198503" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1371%2Fjournal.pone.0198503" aria-label="Article reference 11" data-doi="10.1371/journal.pone.0198503">Article</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30231032" aria-label="PubMed reference 11">PubMed</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6145530" aria-label="PubMed Central reference 11">PubMed Central</a> 
    <a data-track="click||click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 11" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20novel%20mathematical%20method%20for%20disclosing%20oscillations%20in%20gene%20transcription%3A%20a%20comparative%20study&amp;journal=PLoS%20One&amp;doi=10.1371%2Fjournal.pone.0198503&amp;volume=13&amp;publication_year=2018&amp;author=Antoulas%2CAC">
                    Google Scholar</a> 
                </p></li><li data-counter="12."><p id="ref-CR12">Thaben, P. F. &amp; Westermark, P. O. Detecting rhythms in time series with RAIN. <i>J. Biol. Rhythms</i> <b>29</b>, 391–400 (2014).</p><p><a data-track="click||click_references" rel="nofollow noopener" data-track-label="10.1177/0748730414553029" data-track-item_id="10.1177/0748730414553029" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1177%2F0748730414553029" aria-label="Article reference 12" data-doi="10.1177/0748730414553029">Article</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25326247" aria-label="PubMed reference 12">PubMed</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4266694" aria-label="PubMed Central reference 12">PubMed Central</a> 
    <a data-track="click||click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 12" href="http://scholar.google.com/scholar_lookup?&amp;title=Detecting%20rhythms%20in%20time%20series%20with%20RAIN&amp;journal=J.%20Biol.%20Rhythms&amp;doi=10.1177%2F0748730414553029&amp;volume=29&amp;pages=391-400&amp;publication_year=2014&amp;author=Thaben%2CPF&amp;author=Westermark%2CPO">
                    Google Scholar</a> 
                </p></li><li data-counter="13."><p id="ref-CR13">Zhu, B. et al. A cell-autonomous mammalian 12 h clock coordinates metabolic and stress rhythms. <i>Cell Metab.</i> <b>25</b>, 1305–1319.e1309 (2017).</p><p><a data-track="click||click_references" rel="nofollow noopener" data-track-label="10.1016/j.cmet.2017.05.004" data-track-item_id="10.1016/j.cmet.2017.05.004" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.cmet.2017.05.004" aria-label="Article reference 13" data-doi="10.1016/j.cmet.2017.05.004">Article</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXpslWgtrw%3D" aria-label="CAS reference 13">CAS</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28591634" aria-label="PubMed reference 13">PubMed</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5526350" aria-label="PubMed Central reference 13">PubMed Central</a> 
    <a data-track="click||click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 13" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20cell-autonomous%20mammalian%2012%E2%80%89h%20clock%20coordinates%20metabolic%20and%20stress%20rhythms&amp;journal=Cell%20Metab.&amp;doi=10.1016%2Fj.cmet.2017.05.004&amp;volume=25&amp;pages=1305-1319.e1309&amp;publication_year=2017&amp;author=Zhu%2CB">
                    Google Scholar</a> 
                </p></li><li data-counter="14."><p id="ref-CR14">Pan, Y. et al. 12-h clock regulation of genetic information flow by XBP1s. <i>PLOS Biol.</i> <b>18</b>, e3000580 (2020).</p><p><a data-track="click||click_references" rel="nofollow noopener" data-track-label="10.1371/journal.pbio.3000580" data-track-item_id="10.1371/journal.pbio.3000580" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1371%2Fjournal.pbio.3000580" aria-label="Article reference 14" data-doi="10.1371/journal.pbio.3000580">Article</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31935211" aria-label="PubMed reference 14">PubMed</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6959563" aria-label="PubMed Central reference 14">PubMed Central</a> 
    <a data-track="click||click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 14" href="http://scholar.google.com/scholar_lookup?&amp;title=12-h%20clock%20regulation%20of%20genetic%20information%20flow%20by%20XBP1s&amp;journal=PLOS%20Biol.&amp;doi=10.1371%2Fjournal.pbio.3000580&amp;volume=18&amp;publication_year=2020&amp;author=Pan%2CY">
                    Google Scholar</a> 
                </p></li><li data-counter="15."><p id="ref-CR15">Wittenbrink, N. et al. High-accuracy determination of internal circadian time from a single blood sample. <i>J. Clin. Investig.</i> <b>128</b>, 3826–3839 (2018).</p><p><a data-track="click||click_references" rel="nofollow noopener" data-track-label="10.1172/JCI120874" data-track-item_id="10.1172/JCI120874" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1172%2FJCI120874" aria-label="Article reference 15" data-doi="10.1172/JCI120874">Article</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29953415" aria-label="PubMed reference 15">PubMed</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6118629" aria-label="PubMed Central reference 15">PubMed Central</a> 
    <a data-track="click||click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 15" href="http://scholar.google.com/scholar_lookup?&amp;title=High-accuracy%20determination%20of%20internal%20circadian%20time%20from%20a%20single%20blood%20sample&amp;journal=J.%20Clin.%20Investig.&amp;doi=10.1172%2FJCI120874&amp;volume=128&amp;pages=3826-3839&amp;publication_year=2018&amp;author=Wittenbrink%2CN">
                    Google Scholar</a> 
                </p></li><li data-counter="16."><p id="ref-CR16">Yoon, S., Baik, B., Park, T. &amp; Nam, D. Powerful <i>p</i>-value combination methods to detect incomplete association. <i>Sci. Rep.</i> <b>11</b>, 6980 (2021).</p><p><a data-track="click||click_references" rel="nofollow noopener" data-track-label="10.1038/s41598-021-86465-y" data-track-item_id="10.1038/s41598-021-86465-y" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41598-021-86465-y" aria-label="Article reference 16" data-doi="10.1038/s41598-021-86465-y">Article</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXnslCluro%3D" aria-label="CAS reference 16">CAS</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33772054" aria-label="PubMed reference 16">PubMed</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7997958" aria-label="PubMed Central reference 16">PubMed Central</a> 
    <a data-track="click||click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 16" href="http://scholar.google.com/scholar_lookup?&amp;title=Powerful%20p-value%20combination%20methods%20to%20detect%20incomplete%20association&amp;journal=Sci.%20Rep.&amp;doi=10.1038%2Fs41598-021-86465-y&amp;volume=11&amp;publication_year=2021&amp;author=Yoon%2CS&amp;author=Baik%2CB&amp;author=Park%2CT&amp;author=Nam%2CD">
                    Google Scholar</a> 
                </p></li><li data-counter="17."><p id="ref-CR17">Korf-Klingebiel, M. et al. Myeloid-derived growth factor (C19orf10) mediates cardiac repair following myocardial infarction. <i>Nat. Med.</i> <b>21</b>, 140–149 (2015).</p><p><a data-track="click||click_references" rel="nofollow noopener" data-track-label="10.1038/nm.3778" data-track-item_id="10.1038/nm.3778" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fnm.3778" aria-label="Article reference 17" data-doi="10.1038/nm.3778">Article</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXmslKrsg%3D%3D" aria-label="CAS reference 17">CAS</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25581518" aria-label="PubMed reference 17">PubMed</a> 
    <a data-track="click||click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 17" href="http://scholar.google.com/scholar_lookup?&amp;title=Myeloid-derived%20growth%20factor%20%28C19orf10%29%20mediates%20cardiac%20repair%20following%20myocardial%20infarction&amp;journal=Nat.%20Med.&amp;doi=10.1038%2Fnm.3778&amp;volume=21&amp;pages=140-149&amp;publication_year=2015&amp;author=Korf-Klingebiel%2CM">
                    Google Scholar</a> 
                </p></li><li data-counter="18."><p id="ref-CR18">Bortnov, V. et al. Solution structure of human myeloid-derived growth factor suggests a conserved function in the endoplasmic reticulum. <i>Nat. Commun.</i> <b>10</b>, 5612 (2019).</p><p><a data-track="click||click_references" rel="nofollow noopener" data-track-label="10.1038/s41467-019-13577-5" data-track-item_id="10.1038/s41467-019-13577-5" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41467-019-13577-5" aria-label="Article reference 18" data-doi="10.1038/s41467-019-13577-5">Article</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXitlynt7rE" aria-label="CAS reference 18">CAS</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31819058" aria-label="PubMed reference 18">PubMed</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6901522" aria-label="PubMed Central reference 18">PubMed Central</a> 
    <a data-track="click||click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 18" href="http://scholar.google.com/scholar_lookup?&amp;title=Solution%20structure%20of%20human%20myeloid-derived%20growth%20factor%20suggests%20a%20conserved%20function%20in%20the%20endoplasmic%20reticulum&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-019-13577-5&amp;volume=10&amp;publication_year=2019&amp;author=Bortnov%2CV">
                    Google Scholar</a> 
                </p></li><li data-counter="19."><p id="ref-CR19">Borini Etichetti, C. et al. Isoprenylcysteine carboxy methyltransferase (ICMT) is associated with tumor aggressiveness and its expression is controlled by the p53 tumor suppressor. <i>J. Biol. Chem.</i> <b>294</b>, 5060–5073 (2019).</p><p><a data-track="click||click_references" rel="nofollow noopener" data-track-label="10.1074/jbc.RA118.006037" data-track-item_id="10.1074/jbc.RA118.006037" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1074%2Fjbc.RA118.006037" aria-label="Article reference 19" data-doi="10.1074/jbc.RA118.006037">Article</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30655292" aria-label="PubMed reference 19">PubMed</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6442035" aria-label="PubMed Central reference 19">PubMed Central</a> 
    <a data-track="click||click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 19" href="http://scholar.google.com/scholar_lookup?&amp;title=Isoprenylcysteine%20carboxy%20methyltransferase%20%28ICMT%29%20is%20associated%20with%20tumor%20aggressiveness%20and%20its%20expression%20is%20controlled%20by%20the%20p53%20tumor%20suppressor&amp;journal=J.%20Biol.%20Chem.&amp;doi=10.1074%2Fjbc.RA118.006037&amp;volume=294&amp;pages=5060-5073&amp;publication_year=2019&amp;author=Borini%20Etichetti%2CC">
                    Google Scholar</a> 
                </p></li><li data-counter="20."><p id="ref-CR20">Telesio, G. et al. The E3 ubiquitin ligase RNF7 negatively regulates CARD14/CARMA2sh signaling. <i>Int. J. Mol. Sci.</i> <b>18</b>. <a href="https://doi.org/10.3390/ijms18122581" data-track="click||click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.3390/ijms18122581">https://doi.org/10.3390/ijms18122581</a> (2017).</p></li><li data-counter="21."><p id="ref-CR21">Hakimi, A. A. et al. TCEB1-mutated renal cell carcinoma: a distinct genomic and morphological subtype. <i>Mod. Pathol.</i> <b>28</b>, 845–853 (2015).</p><p><a data-track="click||click_references" rel="nofollow noopener" data-track-label="10.1038/modpathol.2015.6" data-track-item_id="10.1038/modpathol.2015.6" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fmodpathol.2015.6" aria-label="Article reference 21" data-doi="10.1038/modpathol.2015.6">Article</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXivFWmu7w%3D" aria-label="CAS reference 21">CAS</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25676555" aria-label="PubMed reference 21">PubMed</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4449825" aria-label="PubMed Central reference 21">PubMed Central</a> 
    <a data-track="click||click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 21" href="http://scholar.google.com/scholar_lookup?&amp;title=TCEB1-mutated%20renal%20cell%20carcinoma%3A%20a%20distinct%20genomic%20and%20morphological%20subtype&amp;journal=Mod.%20Pathol.&amp;doi=10.1038%2Fmodpathol.2015.6&amp;volume=28&amp;pages=845-853&amp;publication_year=2015&amp;author=Hakimi%2CAA">
                    Google Scholar</a> 
                </p></li><li data-counter="22."><p id="ref-CR22">Bonnefoy, N. et al. Cloning of a human gene involved in cytochrome oxidase assembly by functional complementation of an oxa1- mutation in Saccharomyces cerevisiae. <i>Proc. Natl Acad. Sci. USA</i> <b>91</b>, 11978–11982 (1994).</p><p><a data-track="click||click_references" rel="nofollow noopener" data-track-label="10.1073/pnas.91.25.11978" data-track-item_id="10.1073/pnas.91.25.11978" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1073%2Fpnas.91.25.11978" aria-label="Article reference 22" data-doi="10.1073/pnas.91.25.11978">Article</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK2MXisFyqtLg%3D" aria-label="CAS reference 22">CAS</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=7991568" aria-label="PubMed reference 22">PubMed</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC45359" aria-label="PubMed Central reference 22">PubMed Central</a> 
    <a data-track="click||click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 22" href="http://scholar.google.com/scholar_lookup?&amp;title=Cloning%20of%20a%20human%20gene%20involved%20in%20cytochrome%20oxidase%20assembly%20by%20functional%20complementation%20of%20an%20oxa1-%20mutation%20in%20Saccharomyces%20cerevisiae&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;doi=10.1073%2Fpnas.91.25.11978&amp;volume=91&amp;pages=11978-11982&amp;publication_year=1994&amp;author=Bonnefoy%2CN">
                    Google Scholar</a> 
                </p></li><li data-counter="23."><p id="ref-CR23">Otera, H. et al. Mff is an essential factor for mitochondrial recruitment of Drp1 during mitochondrial fission in mammalian cells. <i>J. Cell Biol.</i> <b>191</b>, 1141–1158 (2010).</p><p><a data-track="click||click_references" rel="nofollow noopener" data-track-label="10.1083/jcb.201007152" data-track-item_id="10.1083/jcb.201007152" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1083%2Fjcb.201007152" aria-label="Article reference 23" data-doi="10.1083/jcb.201007152">Article</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXhs1SmurfL" aria-label="CAS reference 23">CAS</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21149567" aria-label="PubMed reference 23">PubMed</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3002033" aria-label="PubMed Central reference 23">PubMed Central</a> 
    <a data-track="click||click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 23" href="http://scholar.google.com/scholar_lookup?&amp;title=Mff%20is%20an%20essential%20factor%20for%20mitochondrial%20recruitment%20of%20Drp1%20during%20mitochondrial%20fission%20in%20mammalian%20cells&amp;journal=J.%20Cell%20Biol.&amp;doi=10.1083%2Fjcb.201007152&amp;volume=191&amp;pages=1141-1158&amp;publication_year=2010&amp;author=Otera%2CH">
                    Google Scholar</a> 
                </p></li><li data-counter="24."><p id="ref-CR24">van der Veen, D. R. &amp; Gerkema, M. P. Unmasking ultradian rhythms in gene expression. <i>FASEB J.</i> <b>31</b>, 743–750 (2017).</p><p><a data-track="click||click_references" rel="nofollow noopener" data-track-label="10.1096/fj.201600872R" data-track-item_id="10.1096/fj.201600872R" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1096%2Ffj.201600872R" aria-label="Article reference 24" data-doi="10.1096/fj.201600872R">Article</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27871062" aria-label="PubMed reference 24">PubMed</a> 
    <a data-track="click||click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 24" href="http://scholar.google.com/scholar_lookup?&amp;title=Unmasking%20ultradian%20rhythms%20in%20gene%20expression&amp;journal=FASEB%20J.&amp;doi=10.1096%2Ffj.201600872R&amp;volume=31&amp;pages=743-750&amp;publication_year=2017&amp;author=Veen%2CDR&amp;author=Gerkema%2CMP">
                    Google Scholar</a> 
                </p></li><li data-counter="25."><p id="ref-CR25">Kembro, J. M. et al. A dynamically coherent pattern of rhythms that matches between distant species across the evolutionary scale. <i>Sci. Rep.</i> <b>13</b>, 5326 (2023).</p><p><a data-track="click||click_references" rel="nofollow noopener" data-track-label="10.1038/s41598-023-32286-0" data-track-item_id="10.1038/s41598-023-32286-0" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41598-023-32286-0" aria-label="Article reference 25" data-doi="10.1038/s41598-023-32286-0">Article</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXmvFOgsL8%3D" aria-label="CAS reference 25">CAS</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=37005423" aria-label="PubMed reference 25">PubMed</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10067965" aria-label="PubMed Central reference 25">PubMed Central</a> 
    <a data-track="click||click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 25" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20dynamically%20coherent%20pattern%20of%20rhythms%20that%20matches%20between%20distant%20species%20across%20the%20evolutionary%20scale&amp;journal=Sci.%20Rep.&amp;doi=10.1038%2Fs41598-023-32286-0&amp;volume=13&amp;publication_year=2023&amp;author=Kembro%2CJM">
                    Google Scholar</a> 
                </p></li><li data-counter="26."><p id="ref-CR26">Hughes, M. E. et al. Harmonics of circadian gene transcription in mammals. <i>PLoS Genet.</i> <b>5</b>, e1000442 (2009).</p><p><a data-track="click||click_references" rel="nofollow noopener" data-track-label="10.1371/journal.pgen.1000442" data-track-item_id="10.1371/journal.pgen.1000442" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1371%2Fjournal.pgen.1000442" aria-label="Article reference 26" data-doi="10.1371/journal.pgen.1000442">Article</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19343201" aria-label="PubMed reference 26">PubMed</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2654964" aria-label="PubMed Central reference 26">PubMed Central</a> 
    <a data-track="click||click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 26" href="http://scholar.google.com/scholar_lookup?&amp;title=Harmonics%20of%20circadian%20gene%20transcription%20in%20mammals&amp;journal=PLoS%20Genet.&amp;doi=10.1371%2Fjournal.pgen.1000442&amp;volume=5&amp;publication_year=2009&amp;author=Hughes%2CME">
                    Google Scholar</a> 
                </p></li><li data-counter="27."><p id="ref-CR27">Golde, T. E., Borchelt, D. R., Giasson, B. I. &amp; Lewis, J. Thinking laterally about neurodegenerative proteinopathies. <i>J. Clin. Investig.</i> <b>123</b>, 1847–1855 (2013).</p><p><a data-track="click||click_references" rel="nofollow noopener" data-track-label="10.1172/JCI66029" data-track-item_id="10.1172/JCI66029" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1172%2FJCI66029" aria-label="Article reference 27" data-doi="10.1172/JCI66029">Article</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXns1GhsL8%3D" aria-label="CAS reference 27">CAS</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23635781" aria-label="PubMed reference 27">PubMed</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3635732" aria-label="PubMed Central reference 27">PubMed Central</a> 
    <a data-track="click||click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 27" href="http://scholar.google.com/scholar_lookup?&amp;title=Thinking%20laterally%20about%20neurodegenerative%20proteinopathies&amp;journal=J.%20Clin.%20Investig.&amp;doi=10.1172%2FJCI66029&amp;volume=123&amp;pages=1847-1855&amp;publication_year=2013&amp;author=Golde%2CTE&amp;author=Borchelt%2CDR&amp;author=Giasson%2CBI&amp;author=Lewis%2CJ">
                    Google Scholar</a> 
                </p></li><li data-counter="28."><p id="ref-CR28">Bayer, T. A. Proteinopathies, a core concept for understanding and ultimately treating degenerative disorders? <i>Eur. Neuropsychopharmacol.</i> <b>25</b>, 713–724 (2015).</p><p><a data-track="click||click_references" rel="nofollow noopener" data-track-label="10.1016/j.euroneuro.2013.03.007" data-track-item_id="10.1016/j.euroneuro.2013.03.007" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.euroneuro.2013.03.007" aria-label="Article reference 28" data-doi="10.1016/j.euroneuro.2013.03.007">Article</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXmvVeksbc%3D" aria-label="CAS reference 28">CAS</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23642796" aria-label="PubMed reference 28">PubMed</a> 
    <a data-track="click||click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 28" href="http://scholar.google.com/scholar_lookup?&amp;title=Proteinopathies%2C%20a%20core%20concept%20for%20understanding%20and%20ultimately%20treating%20degenerative%20disorders%3F&amp;journal=Eur.%20Neuropsychopharmacol.&amp;doi=10.1016%2Fj.euroneuro.2013.03.007&amp;volume=25&amp;pages=713-724&amp;publication_year=2015&amp;author=Bayer%2CTA">
                    Google Scholar</a> 
                </p></li><li data-counter="29."><p id="ref-CR29">Prasun, P. Mitochondrial dysfunction in metabolic syndrome. <i>Biochim. Biophys. Acta Mol. Basis Dis.</i> <b>1866</b>, 165838 (2020).</p><p><a data-track="click||click_references" rel="nofollow noopener" data-track-label="10.1016/j.bbadis.2020.165838" data-track-item_id="10.1016/j.bbadis.2020.165838" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.bbadis.2020.165838" aria-label="Article reference 29" data-doi="10.1016/j.bbadis.2020.165838">Article</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhtVOlsrbO" aria-label="CAS reference 29">CAS</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32428560" aria-label="PubMed reference 29">PubMed</a> 
    <a data-track="click||click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 29" href="http://scholar.google.com/scholar_lookup?&amp;title=Mitochondrial%20dysfunction%20in%20metabolic%20syndrome&amp;journal=Biochim.%20Biophys.%20Acta%20Mol.%20Basis%20Dis.&amp;doi=10.1016%2Fj.bbadis.2020.165838&amp;volume=1866&amp;publication_year=2020&amp;author=Prasun%2CP">
                    Google Scholar</a> 
                </p></li><li data-counter="30."><p id="ref-CR30">Scott, M. R. et al. Twelve-hour rhythms in transcript expression within the human dorsolateral prefrontal cortex are altered in schizophrenia. <i>PLOS Biol.</i> <b>21</b>, e3001688 (2023).</p><p><a data-track="click||click_references" rel="nofollow noopener" data-track-label="10.1371/journal.pbio.3001688" data-track-item_id="10.1371/journal.pbio.3001688" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1371%2Fjournal.pbio.3001688" aria-label="Article reference 30" data-doi="10.1371/journal.pbio.3001688">Article</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXit1agsLg%3D" aria-label="CAS reference 30">CAS</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=36693045" aria-label="PubMed reference 30">PubMed</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9873190" aria-label="PubMed Central reference 30">PubMed Central</a> 
    <a data-track="click||click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 30" href="http://scholar.google.com/scholar_lookup?&amp;title=Twelve-hour%20rhythms%20in%20transcript%20expression%20within%20the%20human%20dorsolateral%20prefrontal%20cortex%20are%20altered%20in%20schizophrenia&amp;journal=PLOS%20Biol.&amp;doi=10.1371%2Fjournal.pbio.3001688&amp;volume=21&amp;publication_year=2023&amp;author=Scott%2CMR">
                    Google Scholar</a> 
                </p></li><li data-counter="31."><p id="ref-CR31">Rensing, L., Meyer-Grahle, U. &amp; Ruoff, P. Biological timing and the clock metaphor: oscillatory and hourglass mechanisms. <i>Chronobiol. Int.</i> <b>18</b>, 329–369 (2001).</p><p><a data-track="click||click_references" rel="nofollow noopener" data-track-label="10.1081/CBI-100103961" data-track-item_id="10.1081/CBI-100103961" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1081%2FCBI-100103961" aria-label="Article reference 31" data-doi="10.1081/CBI-100103961">Article</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3MXlvVKhsLg%3D" aria-label="CAS reference 31">CAS</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=11475408" aria-label="PubMed reference 31">PubMed</a> 
    <a data-track="click||click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 31" href="http://scholar.google.com/scholar_lookup?&amp;title=Biological%20timing%20and%20the%20clock%20metaphor%3A%20oscillatory%20and%20hourglass%20mechanisms&amp;journal=Chronobiol.%20Int.&amp;doi=10.1081%2FCBI-100103961&amp;volume=18&amp;pages=329-369&amp;publication_year=2001&amp;author=Rensing%2CL&amp;author=Meyer-Grahle%2CU&amp;author=Ruoff%2CP">
                    Google Scholar</a> 
                </p></li><li data-counter="32."><p id="ref-CR32">Meng, H. et al. XBP1 links the 12-hour clock to NAFLD and regulation of membrane fluidity and lipid homeostasis. <i>Nat. Commun.</i> <b>11</b>, 6215 (2020).</p><p><a data-track="click||click_references" rel="nofollow noopener" data-track-label="10.1038/s41467-020-20028-z" data-track-item_id="10.1038/s41467-020-20028-z" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41467-020-20028-z" aria-label="Article reference 32" data-doi="10.1038/s41467-020-20028-z">Article</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXisFems7%2FP" aria-label="CAS reference 32">CAS</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33277471" aria-label="PubMed reference 32">PubMed</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7718229" aria-label="PubMed Central reference 32">PubMed Central</a> 
    <a data-track="click||click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 32" href="http://scholar.google.com/scholar_lookup?&amp;title=XBP1%20links%20the%2012-hour%20clock%20to%20NAFLD%20and%20regulation%20of%20membrane%20fluidity%20and%20lipid%20homeostasis&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-020-20028-z&amp;volume=11&amp;publication_year=2020&amp;author=Meng%2CH">
                    Google Scholar</a> 
                </p></li><li data-counter="33."><p id="ref-CR33">Dion, W. et al. Four-dimensional nuclear speckle phase separation dynamics regulate proteostasis. <i>Sci. Adv.</i> <b>8</b>, eabl4150 (2022).</p><p><a data-track="click||click_references" rel="nofollow noopener" data-track-label="10.1126/sciadv.abl4150" data-track-item_id="10.1126/sciadv.abl4150" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1126%2Fsciadv.abl4150" aria-label="Article reference 33" data-doi="10.1126/sciadv.abl4150">Article</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38Xis1Cgu7k%3D" aria-label="CAS reference 33">CAS</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34985945" aria-label="PubMed reference 33">PubMed</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8730402" aria-label="PubMed Central reference 33">PubMed Central</a> 
    <a data-track="click||click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 33" href="http://scholar.google.com/scholar_lookup?&amp;title=Four-dimensional%20nuclear%20speckle%20phase%20separation%20dynamics%20regulate%20proteostasis&amp;journal=Sci.%20Adv.&amp;doi=10.1126%2Fsciadv.abl4150&amp;volume=8&amp;publication_year=2022&amp;author=Dion%2CW">
                    Google Scholar</a> 
                </p></li><li data-counter="34."><p id="ref-CR34">Meng, H. et al. Defining the mammalian coactivation of hepatic 12-h clock and lipid metabolism. <i>Cell Rep.</i> <b>38</b>, 110491 (2022).</p><p><a data-track="click||click_references" rel="nofollow noopener" data-track-label="10.1016/j.celrep.2022.110491" data-track-item_id="10.1016/j.celrep.2022.110491" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.celrep.2022.110491" aria-label="Article reference 34" data-doi="10.1016/j.celrep.2022.110491">Article</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XmsFaguro%3D" aria-label="CAS reference 34">CAS</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=35263593" aria-label="PubMed reference 34">PubMed</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8958721" aria-label="PubMed Central reference 34">PubMed Central</a> 
    <a data-track="click||click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 34" href="http://scholar.google.com/scholar_lookup?&amp;title=Defining%20the%20mammalian%20coactivation%20of%20hepatic%2012-h%20clock%20and%20lipid%20metabolism&amp;journal=Cell%20Rep.&amp;doi=10.1016%2Fj.celrep.2022.110491&amp;volume=38&amp;publication_year=2022&amp;author=Meng%2CH">
                    Google Scholar</a> 
                </p></li><li data-counter="35."><p id="ref-CR35">Qin, Q. et al. Lisa: inferring transcriptional regulators through integrative modeling of public chromatin accessibility and ChIP-seq data. <i>Genome Biol.</i> <b>21</b>, 32 (2020).</p><p><a data-track="click||click_references" rel="noopener" data-track-label="10.1186/s13059-020-1934-6" data-track-item_id="10.1186/s13059-020-1934-6" data-track-value="article reference" data-track-action="article reference" href="https://link.springer.com/doi/10.1186/s13059-020-1934-6" aria-label="Article reference 35" data-doi="10.1186/s13059-020-1934-6">Article</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32033573" aria-label="PubMed reference 35">PubMed</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7007693" aria-label="PubMed Central reference 35">PubMed Central</a> 
    <a data-track="click||click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 35" href="http://scholar.google.com/scholar_lookup?&amp;title=Lisa%3A%20inferring%20transcriptional%20regulators%20through%20integrative%20modeling%20of%20public%20chromatin%20accessibility%20and%20ChIP-seq%20data&amp;journal=Genome%20Biol.&amp;doi=10.1186%2Fs13059-020-1934-6&amp;volume=21&amp;publication_year=2020&amp;author=Qin%2CQ">
                    Google Scholar</a> 
                </p></li><li data-counter="36."><p id="ref-CR36">Zhang, R. &amp; Misra, V. Effects of cyclic AMP response element binding protein-Zhangfei (CREBZF) on the unfolded protein response and cell growth are exerted through the tumor suppressor p53. <i>Cell Cycle</i> <b>13</b>, 279–292 (2014).</p><p><a data-track="click||click_references" rel="nofollow noopener" data-track-label="10.4161/cc.27053" data-track-item_id="10.4161/cc.27053" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.4161%2Fcc.27053" aria-label="Article reference 36" data-doi="10.4161/cc.27053">Article</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXhtFWgs77F" aria-label="CAS reference 36">CAS</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24200963" aria-label="PubMed reference 36">PubMed</a> 
    <a data-track="click||click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 36" href="http://scholar.google.com/scholar_lookup?&amp;title=Effects%20of%20cyclic%20AMP%20response%20element%20binding%20protein-Zhangfei%20%28CREBZF%29%20on%20the%20unfolded%20protein%20response%20and%20cell%20growth%20are%20exerted%20through%20the%20tumor%20suppressor%20p53&amp;journal=Cell%20Cycle&amp;doi=10.4161%2Fcc.27053&amp;volume=13&amp;pages=279-292&amp;publication_year=2014&amp;author=Zhang%2CR&amp;author=Misra%2CV">
                    Google Scholar</a> 
                </p></li><li data-counter="37."><p id="ref-CR37">Zhang, R. et al. Zhangfei/CREB-ZF—a potential regulator of the unfolded protein response. <i>PLOS ONE</i> <b>8</b>, e77256 (2013).</p><p><a data-track="click||click_references" rel="nofollow noopener" data-track-label="10.1371/journal.pone.0077256" data-track-item_id="10.1371/journal.pone.0077256" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1371%2Fjournal.pone.0077256" aria-label="Article reference 37" data-doi="10.1371/journal.pone.0077256">Article</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXhs1KiurrI" aria-label="CAS reference 37">CAS</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24155933" aria-label="PubMed reference 37">PubMed</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3796484" aria-label="PubMed Central reference 37">PubMed Central</a> 
    <a data-track="click||click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 37" href="http://scholar.google.com/scholar_lookup?&amp;title=Zhangfei%2FCREB-ZF%E2%80%94a%20potential%20regulator%20of%20the%20unfolded%20protein%20response&amp;journal=PLOS%20ONE&amp;doi=10.1371%2Fjournal.pone.0077256&amp;volume=8&amp;publication_year=2013&amp;author=Zhang%2CR">
                    Google Scholar</a> 
                </p></li><li data-counter="38."><p id="ref-CR38">Kikuchi, D., Tanimoto, K. &amp; Nakayama, K. CREB is activated by ER stress and modulates the unfolded protein response by regulating the expression of IRE1α and PERK. <i>Biochem. Biophys. Res. Commun.</i> <b>469</b>, 243–250 (2016).</p><p><a data-track="click||click_references" rel="nofollow noopener" data-track-label="10.1016/j.bbrc.2015.11.113" data-track-item_id="10.1016/j.bbrc.2015.11.113" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.bbrc.2015.11.113" aria-label="Article reference 38" data-doi="10.1016/j.bbrc.2015.11.113">Article</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXitVyqurjL" aria-label="CAS reference 38">CAS</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26642955" aria-label="PubMed reference 38">PubMed</a> 
    <a data-track="click||click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 38" href="http://scholar.google.com/scholar_lookup?&amp;title=CREB%20is%20activated%20by%20ER%20stress%20and%20modulates%20the%20unfolded%20protein%20response%20by%20regulating%20the%20expression%20of%20IRE1%CE%B1%20and%20PERK&amp;journal=Biochem.%20Biophys.%20Res.%20Commun.&amp;doi=10.1016%2Fj.bbrc.2015.11.113&amp;volume=469&amp;pages=243-250&amp;publication_year=2016&amp;author=Kikuchi%2CD&amp;author=Tanimoto%2CK&amp;author=Nakayama%2CK">
                    Google Scholar</a> 
                </p></li><li data-counter="39."><p id="ref-CR39">Sriburi, R., Jackowski, S., Mori, K. &amp; Brewer, J. W. XBP1: a link between the unfolded protein response, lipid biosynthesis, and biogenesis of the endoplasmic reticulum. <i>J. Cell Biol.</i> <b>167</b>, 35–41 (2004).</p><p><a data-track="click||click_references" rel="nofollow noopener" data-track-label="10.1083/jcb.200406136" data-track-item_id="10.1083/jcb.200406136" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1083%2Fjcb.200406136" aria-label="Article reference 39" data-doi="10.1083/jcb.200406136">Article</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2cXosVOiu74%3D" aria-label="CAS reference 39">CAS</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15466483" aria-label="PubMed reference 39">PubMed</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2172532" aria-label="PubMed Central reference 39">PubMed Central</a> 
    <a data-track="click||click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 39" href="http://scholar.google.com/scholar_lookup?&amp;title=XBP1%3A%20a%20link%20between%20the%20unfolded%20protein%20response%2C%20lipid%20biosynthesis%2C%20and%20biogenesis%20of%20the%20endoplasmic%20reticulum&amp;journal=J.%20Cell%20Biol.&amp;doi=10.1083%2Fjcb.200406136&amp;volume=167&amp;pages=35-41&amp;publication_year=2004&amp;author=Sriburi%2CR&amp;author=Jackowski%2CS&amp;author=Mori%2CK&amp;author=Brewer%2CJW">
                    Google Scholar</a> 
                </p></li><li data-counter="40."><p id="ref-CR40">Li, Y. et al. A CREBZF/Zhangfei isoform activates CHOP during prolonged cellular stress. <i>bioRxiv</i>, 2022.2010.2010.511668. <a href="https://doi.org/10.1101/2022.10.10.511668" data-track="click||click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1101/2022.10.10.511668">https://doi.org/10.1101/2022.10.10.511668</a> (2022).</p></li><li data-counter="41."><p id="ref-CR41">Yang, Z. F., Drumea, K., Mott, S., Wang, J. &amp; Rosmarin, A. G. GABP transcription factor (nuclear respiratory factor 2) is required for mitochondrial biogenesis. <i>Mol. Cell Biol.</i> <b>34</b>, 3194–3201 (2014).</p><p><a data-track="click||click_references" rel="nofollow noopener" data-track-label="10.1128/MCB.00492-12" data-track-item_id="10.1128/MCB.00492-12" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1128%2FMCB.00492-12" aria-label="Article reference 41" data-doi="10.1128/MCB.00492-12">Article</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24958105" aria-label="PubMed reference 41">PubMed</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4135556" aria-label="PubMed Central reference 41">PubMed Central</a> 
    <a data-track="click||click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 41" href="http://scholar.google.com/scholar_lookup?&amp;title=GABP%20transcription%20factor%20%28nuclear%20respiratory%20factor%202%29%20is%20required%20for%20mitochondrial%20biogenesis&amp;journal=Mol.%20Cell%20Biol.&amp;doi=10.1128%2FMCB.00492-12&amp;volume=34&amp;pages=3194-3201&amp;publication_year=2014&amp;author=Yang%2CZF&amp;author=Drumea%2CK&amp;author=Mott%2CS&amp;author=Wang%2CJ&amp;author=Rosmarin%2CAG">
                    Google Scholar</a> 
                </p></li><li data-counter="42."><p id="ref-CR42">Mootha, V. K. et al. Erralpha and Gabpa/b specify PGC-1alpha-dependent oxidative phosphorylation gene expression that is altered in diabetic muscle. <i>Proc. Natl Acad. Sci. USA</i> <b>101</b>, 6570–6575 (2004).</p><p><a data-track="click||click_references" rel="nofollow noopener" data-track-label="10.1073/pnas.0401401101" data-track-item_id="10.1073/pnas.0401401101" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1073%2Fpnas.0401401101" aria-label="Article reference 42" data-doi="10.1073/pnas.0401401101">Article</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2cXjvVyiuro%3D" aria-label="CAS reference 42">CAS</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15100410" aria-label="PubMed reference 42">PubMed</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC404086" aria-label="PubMed Central reference 42">PubMed Central</a> 
    <a data-track="click||click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 42" href="http://scholar.google.com/scholar_lookup?&amp;title=Erralpha%20and%20Gabpa%2Fb%20specify%20PGC-1alpha-dependent%20oxidative%20phosphorylation%20gene%20expression%20that%20is%20altered%20in%20diabetic%20muscle&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;doi=10.1073%2Fpnas.0401401101&amp;volume=101&amp;pages=6570-6575&amp;publication_year=2004&amp;author=Mootha%2CVK">
                    Google Scholar</a> 
                </p></li><li data-counter="43."><p id="ref-CR43">Pramanik, J. et al. Genome-wide analyses reveal the IRE1a-XBP1 pathway promotes T helper cell differentiation by resolving secretory stress and accelerating proliferation. <i>Genome Med.</i> <b>10</b>, 76 (2018).</p><p><a data-track="click||click_references" rel="noopener" data-track-label="10.1186/s13073-018-0589-3" data-track-item_id="10.1186/s13073-018-0589-3" data-track-value="article reference" data-track-action="article reference" href="https://link.springer.com/doi/10.1186/s13073-018-0589-3" aria-label="Article reference 43" data-doi="10.1186/s13073-018-0589-3">Article</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXpslKktbs%3D" aria-label="CAS reference 43">CAS</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30355343" aria-label="PubMed reference 43">PubMed</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6199730" aria-label="PubMed Central reference 43">PubMed Central</a> 
    <a data-track="click||click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 43" href="http://scholar.google.com/scholar_lookup?&amp;title=Genome-wide%20analyses%20reveal%20the%20IRE1a-XBP1%20pathway%20promotes%20T%20helper%20cell%20differentiation%20by%20resolving%20secretory%20stress%20and%20accelerating%20proliferation&amp;journal=Genome%20Med.&amp;doi=10.1186%2Fs13073-018-0589-3&amp;volume=10&amp;publication_year=2018&amp;author=Pramanik%2CJ">
                    Google Scholar</a> 
                </p></li><li data-counter="44."><p id="ref-CR44">Zhu, B. Decoding the function and regulation of the mammalian 12h-clock. <i>J. Mol. Cell Biol.</i> <a href="https://doi.org/10.1093/jmcb/mjaa021" data-track="click||click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1093/jmcb/mjaa021">https://doi.org/10.1093/jmcb/mjaa021</a> (2020).</p></li><li data-counter="45."><p id="ref-CR45">Li, H.-D., Funk, C. C. &amp; Price, N. D. iREAD: a tool for intron retention detection from RNA-seq data. <i>BMC Genom.</i> <b>21</b>, 128 (2020).</p><p><a data-track="click||click_references" rel="noopener" data-track-label="10.1186/s12864-020-6541-0" data-track-item_id="10.1186/s12864-020-6541-0" data-track-value="article reference" data-track-action="article reference" href="https://link.springer.com/doi/10.1186/s12864-020-6541-0" aria-label="Article reference 45" data-doi="10.1186/s12864-020-6541-0">Article</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXksFaltbc%3D" aria-label="CAS reference 45">CAS</a> 
    <a data-track="click||click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 45" href="http://scholar.google.com/scholar_lookup?&amp;title=iREAD%3A%20a%20tool%20for%20intron%20retention%20detection%20from%20RNA-seq%20data&amp;journal=BMC%20Genom.&amp;doi=10.1186%2Fs12864-020-6541-0&amp;volume=21&amp;publication_year=2020&amp;author=Li%2CH-D&amp;author=Funk%2CCC&amp;author=Price%2CND">
                    Google Scholar</a> 
                </p></li><li data-counter="46."><p id="ref-CR46">Castellana, S. et al. Systematic analysis of mouse genome reveals distinct evolutionary and functional properties among circadian and ultradian genes. <i>Front. Physiol.</i> <b>9</b>, 1178 (2018).</p><p><a data-track="click||click_references" rel="nofollow noopener" data-track-label="10.3389/fphys.2018.01178" data-track-item_id="10.3389/fphys.2018.01178" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.3389%2Ffphys.2018.01178" aria-label="Article reference 46" data-doi="10.3389/fphys.2018.01178">Article</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30190679" aria-label="PubMed reference 46">PubMed</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6115496" aria-label="PubMed Central reference 46">PubMed Central</a> 
    <a data-track="click||click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 46" href="http://scholar.google.com/scholar_lookup?&amp;title=Systematic%20analysis%20of%20mouse%20genome%20reveals%20distinct%20evolutionary%20and%20functional%20properties%20among%20circadian%20and%20ultradian%20genes&amp;journal=Front.%20Physiol.&amp;doi=10.3389%2Ffphys.2018.01178&amp;volume=9&amp;publication_year=2018&amp;author=Castellana%2CS">
                    Google Scholar</a> 
                </p></li><li data-counter="47."><p id="ref-CR47">Schnytzer, Y. et al. Tidal and diel orchestration of behaviour and gene expression in an intertidal mollusc. <i>Sci. Rep.</i> <b>8</b>, 4917 (2018).</p><p><a data-track="click||click_references" rel="nofollow noopener" data-track-label="10.1038/s41598-018-23167-y" data-track-item_id="10.1038/s41598-018-23167-y" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41598-018-23167-y" aria-label="Article reference 47" data-doi="10.1038/s41598-018-23167-y">Article</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:STN:280:DC%2BC1MnjtFKhsQ%3D%3D" aria-label="CAS reference 47">CAS</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29559663" aria-label="PubMed reference 47">PubMed</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5861051" aria-label="PubMed Central reference 47">PubMed Central</a> 
    <a data-track="click||click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 47" href="http://scholar.google.com/scholar_lookup?&amp;title=Tidal%20and%20diel%20orchestration%20of%20behaviour%20and%20gene%20expression%20in%20an%20intertidal%20mollusc&amp;journal=Sci.%20Rep.&amp;doi=10.1038%2Fs41598-018-23167-y&amp;volume=8&amp;publication_year=2018&amp;author=Schnytzer%2CY">
                    Google Scholar</a> 
                </p></li><li data-counter="48."><p id="ref-CR48">Sorek, M. et al. Setting the pace: host rhythmic behaviour and gene expression patterns in the facultatively symbiotic cnidarian Aiptasia are determined largely by symbiodinium. <i>Microbiome</i> <b>6</b>, 83 (2018).</p><p><a data-track="click||click_references" rel="noopener" data-track-label="10.1186/s40168-018-0465-9" data-track-item_id="10.1186/s40168-018-0465-9" data-track-value="article reference" data-track-action="article reference" href="https://link.springer.com/doi/10.1186/s40168-018-0465-9" aria-label="Article reference 48" data-doi="10.1186/s40168-018-0465-9">Article</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29739445" aria-label="PubMed reference 48">PubMed</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5941691" aria-label="PubMed Central reference 48">PubMed Central</a> 
    <a data-track="click||click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 48" href="http://scholar.google.com/scholar_lookup?&amp;title=Setting%20the%20pace%3A%20host%20rhythmic%20behaviour%20and%20gene%20expression%20patterns%20in%20the%20facultatively%20symbiotic%20cnidarian%20Aiptasia%20are%20determined%20largely%20by%20symbiodinium&amp;journal=Microbiome&amp;doi=10.1186%2Fs40168-018-0465-9&amp;volume=6&amp;publication_year=2018&amp;author=Sorek%2CM">
                    Google Scholar</a> 
                </p></li><li data-counter="49."><p id="ref-CR49">Rinsky, M. et al. Temporal gene expression patterns in the coral Euphyllia paradivisa reveal the complexity of biological clocks in the cnidarian-algal symbiosis. <i>Sci. Adv.</i> <b>8</b>, eabo6467 (2022).</p><p><a data-track="click||click_references" rel="nofollow noopener" data-track-label="10.1126/sciadv.abo6467" data-track-item_id="10.1126/sciadv.abo6467" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1126%2Fsciadv.abo6467" aria-label="Article reference 49" data-doi="10.1126/sciadv.abo6467">Article</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XisFOrurrF" aria-label="CAS reference 49">CAS</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=36112690" aria-label="PubMed reference 49">PubMed</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9481131" aria-label="PubMed Central reference 49">PubMed Central</a> 
    <a data-track="click||click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 49" href="http://scholar.google.com/scholar_lookup?&amp;title=Temporal%20gene%20expression%20patterns%20in%20the%20coral%20Euphyllia%20paradivisa%20reveal%20the%20complexity%20of%20biological%20clocks%20in%20the%20cnidarian-algal%20symbiosis&amp;journal=Sci.%20Adv.&amp;doi=10.1126%2Fsciadv.abo6467&amp;volume=8&amp;publication_year=2022&amp;author=Rinsky%2CM">
                    Google Scholar</a> 
                </p></li><li data-counter="50."><p id="ref-CR50">Yokomizo, T. &amp; Takahashi, Y. Plasticity of circadian and circatidal rhythms in activity and transcriptomic dynamics in a freshwater snail. <i>Heredity (Edinb).</i> <b>132</b>, 267–274 (2024).</p><p><a data-track="click||click_references" rel="nofollow noopener" data-track-label="10.1038/s41437-024-00680-7" data-track-item_id="10.1038/s41437-024-00680-7" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41437-024-00680-7" aria-label="Article reference 50" data-doi="10.1038/s41437-024-00680-7">Article</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=38538720" aria-label="PubMed reference 50">PubMed</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC11074255" aria-label="PubMed Central reference 50">PubMed Central</a> 
    <a data-track="click||click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 50" href="http://scholar.google.com/scholar_lookup?&amp;title=Plasticity%20of%20circadian%20and%20circatidal%20rhythms%20in%20activity%20and%20transcriptomic%20dynamics%20in%20a%20freshwater%20snail&amp;journal=Heredity%20%28Edinb%29.&amp;doi=10.1038%2Fs41437-024-00680-7&amp;volume=132&amp;pages=267-274&amp;publication_year=2024&amp;author=Yokomizo%2CT&amp;author=Takahashi%2CY">
                    Google Scholar</a> 
                </p></li><li data-counter="51."><p id="ref-CR51">Putnam, N. H. et al. Sea anemone genome reveals ancestral eumetazoan gene repertoire and genomic organization. <i>Science</i> <b>317</b>, 86–94 (2007).</p><p><a data-track="click||click_references" rel="nofollow noopener" data-track-label="10.1126/science.1139158" data-track-item_id="10.1126/science.1139158" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1126%2Fscience.1139158" aria-label="Article reference 51" data-doi="10.1126/science.1139158">Article</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2sXnt1Sjtbo%3D" aria-label="CAS reference 51">CAS</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17615350" aria-label="PubMed reference 51">PubMed</a> 
    <a data-track="click||click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 51" href="http://scholar.google.com/scholar_lookup?&amp;title=Sea%20anemone%20genome%20reveals%20ancestral%20eumetazoan%20gene%20repertoire%20and%20genomic%20organization&amp;journal=Science&amp;doi=10.1126%2Fscience.1139158&amp;volume=317&amp;pages=86-94&amp;publication_year=2007&amp;author=Putnam%2CNH">
                    Google Scholar</a> 
                </p></li><li data-counter="52."><p id="ref-CR52">Rodríguez-González, C., Lin, S., Arkan, S. &amp; Hansen, C. Co-chaperones DNAJA1 and DNAJB6 are critical for regulation of polyglutamine aggregation. <i>Sci. Rep.</i> <b>10</b>, 8130 (2020).</p><p><a data-track="click||click_references" rel="nofollow noopener" data-track-label="10.1038/s41598-020-65046-5" data-track-item_id="10.1038/s41598-020-65046-5" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41598-020-65046-5" aria-label="Article reference 52" data-doi="10.1038/s41598-020-65046-5">Article</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32424160" aria-label="PubMed reference 52">PubMed</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7235262" aria-label="PubMed Central reference 52">PubMed Central</a> 
    <a data-track="click||click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 52" href="http://scholar.google.com/scholar_lookup?&amp;title=Co-chaperones%20DNAJA1%20and%20DNAJB6%20are%20critical%20for%20regulation%20of%20polyglutamine%20aggregation&amp;journal=Sci.%20Rep.&amp;doi=10.1038%2Fs41598-020-65046-5&amp;volume=10&amp;publication_year=2020&amp;author=Rodr%C3%ADguez-Gonz%C3%A1lez%2CC&amp;author=Lin%2CS&amp;author=Arkan%2CS&amp;author=Hansen%2CC">
                    Google Scholar</a> 
                </p></li><li data-counter="53."><p id="ref-CR53">Adams, B. M., Canniff, N. P., Guay, K. P., Larsen, I. S. B. &amp; Hebert, D. N. Quantitative glycoproteomics reveals cellular substrate selectivity of the ER protein quality control sensors UGGT1 and UGGT2. <i>eLife</i> <b>9</b>, e63997 (2020).</p><p><a data-track="click||click_references" rel="nofollow noopener" data-track-label="10.7554/eLife.63997" data-track-item_id="10.7554/eLife.63997" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.7554%2FeLife.63997" aria-label="Article reference 53" data-doi="10.7554/eLife.63997">Article</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXovVersb8%3D" aria-label="CAS reference 53">CAS</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33320095" aria-label="PubMed reference 53">PubMed</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7771966" aria-label="PubMed Central reference 53">PubMed Central</a> 
    <a data-track="click||click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 53" href="http://scholar.google.com/scholar_lookup?&amp;title=Quantitative%20glycoproteomics%20reveals%20cellular%20substrate%20selectivity%20of%20the%20ER%20protein%20quality%20control%20sensors%20UGGT1%20and%20UGGT2&amp;journal=eLife&amp;doi=10.7554%2FeLife.63997&amp;volume=9&amp;publication_year=2020&amp;author=Adams%2CBM&amp;author=Canniff%2CNP&amp;author=Guay%2CKP&amp;author=Larsen%2CISB&amp;author=Hebert%2CDN">
                    Google Scholar</a> 
                </p></li><li data-counter="54."><p id="ref-CR54">Boye, E. &amp; Grallert, B. eIF2α phosphorylation and the regulation of translation. <i>Curr. Genet.</i> <b>66</b>, 293–297 (2020).</p><p><a data-track="click||click_references" rel="noopener" data-track-label="10.1007/s00294-019-01026-1" data-track-item_id="10.1007/s00294-019-01026-1" data-track-value="article reference" data-track-action="article reference" href="https://link.springer.com/doi/10.1007/s00294-019-01026-1" aria-label="Article reference 54" data-doi="10.1007/s00294-019-01026-1">Article</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhslWisrrM" aria-label="CAS reference 54">CAS</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31485739" aria-label="PubMed reference 54">PubMed</a> 
    <a data-track="click||click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 54" href="http://scholar.google.com/scholar_lookup?&amp;title=eIF2%CE%B1%20phosphorylation%20and%20the%20regulation%20of%20translation&amp;journal=Curr.%20Genet.&amp;doi=10.1007%2Fs00294-019-01026-1&amp;volume=66&amp;pages=293-297&amp;publication_year=2020&amp;author=Boye%2CE&amp;author=Grallert%2CB">
                    Google Scholar</a> 
                </p></li><li data-counter="55."><p id="ref-CR55">Zhang, H. et al. Circatidal control of gene expression in the deep-sea hot vent shrimp &lt;em&gt;Rimicaris leurokolos&lt;/em&gt;. bioRxiv, 2024.2001.2012.575359. <a href="https://doi.org/10.1101/2024.01.12.575359" data-track="click||click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1101/2024.01.12.575359">https://doi.org/10.1101/2024.01.12.575359</a> (2024).</p></li><li data-counter="56."><p id="ref-CR56">Cretenet, G., Le Clech, M. &amp; Gachon, F. Circadian clock-coordinated 12 h period rhythmic activation of the IRE1alpha pathway controls lipid metabolism in mouse liver. <i>Cell Metab.</i> <b>11</b>, 47–57 (2010).</p><p><a data-track="click||click_references" rel="nofollow noopener" data-track-label="10.1016/j.cmet.2009.11.002" data-track-item_id="10.1016/j.cmet.2009.11.002" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.cmet.2009.11.002" aria-label="Article reference 56" data-doi="10.1016/j.cmet.2009.11.002">Article</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXlt1Klu7c%3D" aria-label="CAS reference 56">CAS</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20074527" aria-label="PubMed reference 56">PubMed</a> 
    <a data-track="click||click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 56" href="http://scholar.google.com/scholar_lookup?&amp;title=Circadian%20clock-coordinated%2012%E2%80%89h%20period%20rhythmic%20activation%20of%20the%20IRE1alpha%20pathway%20controls%20lipid%20metabolism%20in%20mouse%20liver&amp;journal=Cell%20Metab.&amp;doi=10.1016%2Fj.cmet.2009.11.002&amp;volume=11&amp;pages=47-57&amp;publication_year=2010&amp;author=Cretenet%2CG&amp;author=Clech%2CM&amp;author=Gachon%2CF">
                    Google Scholar</a> 
                </p></li><li data-counter="57."><p id="ref-CR57">Hughes, M. E. et al. Brain-specific rescue of Clock reveals system-driven transcriptional rhythms in peripheral tissue. <i>PLoS Genet.</i> <b>8</b>, e1002835 (2012).</p><p><a data-track="click||click_references" rel="nofollow noopener" data-track-label="10.1371/journal.pgen.1002835" data-track-item_id="10.1371/journal.pgen.1002835" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1371%2Fjournal.pgen.1002835" aria-label="Article reference 57" data-doi="10.1371/journal.pgen.1002835">Article</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XhtFeisrnJ" aria-label="CAS reference 57">CAS</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22844252" aria-label="PubMed reference 57">PubMed</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3405989" aria-label="PubMed Central reference 57">PubMed Central</a> 
    <a data-track="click||click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 57" href="http://scholar.google.com/scholar_lookup?&amp;title=Brain-specific%20rescue%20of%20Clock%20reveals%20system-driven%20transcriptional%20rhythms%20in%20peripheral%20tissue&amp;journal=PLoS%20Genet.&amp;doi=10.1371%2Fjournal.pgen.1002835&amp;volume=8&amp;publication_year=2012&amp;author=Hughes%2CME">
                    Google Scholar</a> 
                </p></li><li data-counter="58."><p id="ref-CR58">Westermark, P. O. &amp; Herzel, H. Mechanism for 12 h rhythm generation by the circadian clock. <i>Cell Rep.</i> <b>3</b>, 1228–1238 (2013).</p><p><a data-track="click||click_references" rel="nofollow noopener" data-track-label="10.1016/j.celrep.2013.03.013" data-track-item_id="10.1016/j.celrep.2013.03.013" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.celrep.2013.03.013" aria-label="Article reference 58" data-doi="10.1016/j.celrep.2013.03.013">Article</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXlvVCqsbo%3D" aria-label="CAS reference 58">CAS</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23583178" aria-label="PubMed reference 58">PubMed</a> 
    <a data-track="click||click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 58" href="http://scholar.google.com/scholar_lookup?&amp;title=Mechanism%20for%2012%E2%80%89h%20rhythm%20generation%20by%20the%20circadian%20clock&amp;journal=Cell%20Rep.&amp;doi=10.1016%2Fj.celrep.2013.03.013&amp;volume=3&amp;pages=1228-1238&amp;publication_year=2013&amp;author=Westermark%2CPO&amp;author=Herzel%2CH">
                    Google Scholar</a> 
                </p></li><li data-counter="59."><p id="ref-CR59">Zhu, B. &amp; Liu, S. Preservation of <span>∼</span>12-h ultradian rhythms of gene expression of mRNA and protein metabolism in the absence of canonical circadian clock. <i>Front. Physiol.</i> <b>14</b>. <a href="https://doi.org/10.3389/fphys.2023.1195001" data-track="click||click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.3389/fphys.2023.1195001">https://doi.org/10.3389/fphys.2023.1195001</a> (2023).</p></li><li data-counter="60."><p id="ref-CR60">Deng, Y. et al. The Xbp1s/GalE axis links ER stress to postprandial hepatic metabolism. <i>J. Clin. Investig.</i> <b>123</b>, 455–468 (2013).</p><p><a data-track="click||click_references" rel="nofollow noopener" data-track-label="10.1172/JCI62819" data-track-item_id="10.1172/JCI62819" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1172%2FJCI62819" aria-label="Article reference 60" data-doi="10.1172/JCI62819">Article</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXnsFGltg%3D%3D" aria-label="CAS reference 60">CAS</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23257357" aria-label="PubMed reference 60">PubMed</a> 
    <a data-track="click||click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 60" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20Xbp1s%2FGalE%20axis%20links%20ER%20stress%20to%20postprandial%20hepatic%20metabolism&amp;journal=J.%20Clin.%20Investig.&amp;doi=10.1172%2FJCI62819&amp;volume=123&amp;pages=455-468&amp;publication_year=2013&amp;author=Deng%2CY">
                    Google Scholar</a> 
                </p></li><li data-counter="61."><p id="ref-CR61">Shao, M. et al. Hepatic IRE1alpha regulates fasting-induced metabolic adaptive programs through the XBP1s-PPARalpha axis signalling. <i>Nat. Commun.</i> <b>5</b>, 3528 (2014).</p><p><a data-track="click||click_references" rel="nofollow noopener" data-track-label="10.1038/ncomms4528" data-track-item_id="10.1038/ncomms4528" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fncomms4528" aria-label="Article reference 61" data-doi="10.1038/ncomms4528">Article</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24670948" aria-label="PubMed reference 61">PubMed</a> 
    <a data-track="click||click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 61" href="http://scholar.google.com/scholar_lookup?&amp;title=Hepatic%20IRE1alpha%20regulates%20fasting-induced%20metabolic%20adaptive%20programs%20through%20the%20XBP1s-PPARalpha%20axis%20signalling&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fncomms4528&amp;volume=5&amp;publication_year=2014&amp;author=Shao%2CM">
                    Google Scholar</a> 
                </p></li><li data-counter="62."><p id="ref-CR62">Bolger, A. M., Lohse, M. &amp; Usadel, B. Trimmomatic: a flexible trimmer for Illumina sequence data. <i>Bioinformatics</i> <b>30</b>, 2114–2120 (2014).</p><p><a data-track="click||click_references" rel="nofollow noopener" data-track-label="10.1093/bioinformatics/btu170" data-track-item_id="10.1093/bioinformatics/btu170" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtu170" aria-label="Article reference 62" data-doi="10.1093/bioinformatics/btu170">Article</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXht1Sqt7nP" aria-label="CAS reference 62">CAS</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24695404" aria-label="PubMed reference 62">PubMed</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4103590" aria-label="PubMed Central reference 62">PubMed Central</a> 
    <a data-track="click||click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 62" href="http://scholar.google.com/scholar_lookup?&amp;title=Trimmomatic%3A%20a%20flexible%20trimmer%20for%20Illumina%20sequence%20data&amp;journal=Bioinformatics&amp;doi=10.1093%2Fbioinformatics%2Fbtu170&amp;volume=30&amp;pages=2114-2120&amp;publication_year=2014&amp;author=Bolger%2CAM&amp;author=Lohse%2CM&amp;author=Usadel%2CB">
                    Google Scholar</a> 
                </p></li><li data-counter="63."><p id="ref-CR63">Dobin, A. et al. STAR: ultrafast universal RNA-seq aligner. <i>Bioinformatics</i> <b>29</b>, 15–21 (2013).</p><p><a data-track="click||click_references" rel="nofollow noopener" data-track-label="10.1093/bioinformatics/bts635" data-track-item_id="10.1093/bioinformatics/bts635" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1093%2Fbioinformatics%2Fbts635" aria-label="Article reference 63" data-doi="10.1093/bioinformatics/bts635">Article</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XhvV2gsbnF" aria-label="CAS reference 63">CAS</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23104886" aria-label="PubMed reference 63">PubMed</a> 
    <a data-track="click||click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 63" href="http://scholar.google.com/scholar_lookup?&amp;title=STAR%3A%20ultrafast%20universal%20RNA-seq%20aligner&amp;journal=Bioinformatics&amp;doi=10.1093%2Fbioinformatics%2Fbts635&amp;volume=29&amp;pages=15-21&amp;publication_year=2013&amp;author=Dobin%2CA">
                    Google Scholar</a> 
                </p></li><li data-counter="64."><p id="ref-CR64">Trapnell, C. et al. Differential gene and transcript expression analysis of RNA-seq experiments with TopHat and Cufflinks. <i>Nat. Protoc.</i> <b>7</b>, 562–578 (2012).</p><p><a data-track="click||click_references" rel="nofollow noopener" data-track-label="10.1038/nprot.2012.016" data-track-item_id="10.1038/nprot.2012.016" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fnprot.2012.016" aria-label="Article reference 64" data-doi="10.1038/nprot.2012.016">Article</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38Xjt1Cjsrc%3D" aria-label="CAS reference 64">CAS</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22383036" aria-label="PubMed reference 64">PubMed</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3334321" aria-label="PubMed Central reference 64">PubMed Central</a> 
    <a data-track="click||click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 64" href="http://scholar.google.com/scholar_lookup?&amp;title=Differential%20gene%20and%20transcript%20expression%20analysis%20of%20RNA-seq%20experiments%20with%20TopHat%20and%20Cufflinks&amp;journal=Nat.%20Protoc.&amp;doi=10.1038%2Fnprot.2012.016&amp;volume=7&amp;pages=562-578&amp;publication_year=2012&amp;author=Trapnell%2CC">
                    Google Scholar</a> 
                </p></li><li data-counter="65."><p id="ref-CR65">Rey, G. et al. Metabolic oscillations on the circadian time scale in Drosophila cells lacking clock genes. <i>Mol. Syst. Biol.</i> <b>14</b>, e8376 (2018).</p><p><a data-track="click||click_references" rel="nofollow noopener" data-track-label="10.15252/msb.20188376" data-track-item_id="10.15252/msb.20188376" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.15252%2Fmsb.20188376" aria-label="Article reference 65" data-doi="10.15252/msb.20188376">Article</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30072421" aria-label="PubMed reference 65">PubMed</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6078164" aria-label="PubMed Central reference 65">PubMed Central</a> 
    <a data-track="click||click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 65" href="http://scholar.google.com/scholar_lookup?&amp;title=Metabolic%20oscillations%20on%20the%20circadian%20time%20scale%20in%20Drosophila%20cells%20lacking%20clock%20genes&amp;journal=Mol.%20Syst.%20Biol.&amp;doi=10.15252%2Fmsb.20188376&amp;volume=14&amp;publication_year=2018&amp;author=Rey%2CG">
                    Google Scholar</a> 
                </p></li><li data-counter="66."><p id="ref-CR66">Huang da, W., Sherman, B. T. &amp; Lempicki, R. A. Systematic and integrative analysis of large gene lists using DAVID bioinformatics resources. <i>Nat. Protoc.</i> <b>4</b>, 44–57 (2009).</p><p><a data-track="click||click_references" rel="nofollow noopener" data-track-label="10.1038/nprot.2008.211" data-track-item_id="10.1038/nprot.2008.211" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fnprot.2008.211" aria-label="Article reference 66" data-doi="10.1038/nprot.2008.211">Article</a> 
    <a data-track="click||click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19131956" aria-label="PubMed reference 66">PubMed</a> 
    <a data-track="click||click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 66" href="http://scholar.google.com/scholar_lookup?&amp;title=Systematic%20and%20integrative%20analysis%20of%20large%20gene%20lists%20using%20DAVID%20bioinformatics%20resources&amp;journal=Nat.%20Protoc.&amp;doi=10.1038%2Fnprot.2008.211&amp;volume=4&amp;pages=44-57&amp;publication_year=2009&amp;author=Huang%20da%2CW&amp;author=Sherman%2CBT&amp;author=Lempicki%2CRA">
                    Google Scholar</a> 
                </p></li></ol><p><a data-track="click" data-track-action="download citation references" data-track-label="link" rel="nofollow" href="https://citation-needed.springer.com/v2/references/10.1038/s44323-024-00005-1?format=refman&amp;flavour=references">Download references<svg width="16" height="16" focusable="false" role="img" aria-hidden="true"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-download-medium"></use></svg></a></p></div></div></div></section></div><section data-title="Acknowledgements"><div id="Ack1-section"><h2 id="Ack1">Acknowledgements</h2><p>We thank Toren Finkel for critical discussion and insightful advice on the manuscript, and Soukaina Eljamri for help with the human study protocols. This research was supported in part by the University of Pittsburgh Center for Research Computing through the resources provided. Specifically, this work used the HTC cluster, which is supported by NIH award number S10OD028483. This work was further supported by internal funds from the Center for Human Integrative Physiology, Aging Institute, University of Pittsburgh School of Medicine to PKF and MLS, National Institute of Health grants DP2GM140924 and R21AG071893 to BZ, and National Institute of Health grant P30DK120531to SL.</p></div></section><section aria-labelledby="author-information" data-title="Author information"><div id="author-information-section"><h2 id="author-information">Author information</h2><div id="author-information-content"><h3 id="affiliations">Authors and Affiliations</h3><ol><li id="Aff1"><p>Aging Institute of UPMC, University of Pittsburgh School of Medicine, Pittsburgh, PA, USA</p><p>Bokai Zhu, Natalie L. David, William Dion, Nandini K. Doshi, Tânia Amorim, Rosemary E. Andrews, G. V. Naveen Kumar, Saad Irfan, Tristan Pesaresi, Ankit X. Sharma, Michelle Sun &amp; Matthew L. Steinhauser</p></li><li id="Aff2"><p>Pittsburgh Liver Research Center, University of Pittsburgh, Pittsburgh, PA, USA</p><p>Bokai Zhu &amp; Silvia Liu</p></li><li id="Aff3"><p>Division of Endocrinology and Metabolism, Department of Medicine, University of Pittsburgh School of Medicine, Pittsburgh, PA, USA</p><p>Bokai Zhu</p></li><li id="Aff4"><p>Department of Pathology, University of Pittsburgh School of Medicine, Pittsburgh, PA, USA</p><p>Silvia Liu</p></li><li id="Aff5"><p>Neuroendocrinology Unit, Division of Endocrinology and Metabolism, Department of Medicine, University of Pittsburgh School of Medicine, Pittsburgh, PA, USA</p><p>Natalie L. David, Lauren B. Siegel, Tânia Amorim &amp; Pouneh K. Fazeli</p></li><li id="Aff6"><p>Center for Human Integrative Physiology, University of Pittsburgh School of Medicine, Pittsburgh, PA, USA</p><p>Natalie L. David, Nandini K. Doshi, Tânia Amorim, Rosemary E. Andrews, Tristan Pesaresi, Pouneh K. Fazeli &amp; Matthew L. Steinhauser</p></li><li id="Aff7"><p>Department of Statistics, Kenneth P. Dietrich School of Arts and Sciences, University of Pittsburgh, Pittsburgh, PA, USA</p><p>Hanwen Li</p></li><li id="Aff8"><p>Division of Cardiology, Department of Medicine, University of Pittsburgh School of Medicine, Pittsburgh, PA, USA</p><p>Matthew L. Steinhauser</p></li></ol><h3 id="contributions">Contributions</h3><p>Conceptualization: B.Z., P.K.F., M.L.S. Methodology: B.Z., S.L., P.K.F., M.L.S. Investigation: B.Z., S.L., P.K.F., M.L.S., N.L.D., N.K.D., W.D., L.B.S., T.A., R.E.A., G.V.N.K., S.I., T.P., A.X.S., M.S. Visualization: B.Z., S.L. Formal Analysis: B.Z., S.L., H.L. Funding acquisition: B.Z., P.K.F., M.L.S. Resources: B.Z., P.K.F., M.L.S. Software: B.Z., S.L. Validation: B.Z., S.L. Supervision: B.Z., P.K.F., M.L.S. Writing—original draft: B.Z., S.L., P.K.F, M.L.S. Writing—review &amp; editing: all authors.</p><h3 id="corresponding-author">Corresponding authors</h3><p id="corresponding-author-list">Correspondence to
                <a id="corresp-c1" href="mailto:bzhu@pitt.edu">Bokai Zhu</a> or <a id="corresp-c2" href="mailto:msteinhauser@pitt.edu">Matthew L. Steinhauser</a>.</p></div></div></section><section data-title="Ethics declarations"><div id="ethics-section"><h2 id="ethics">Ethics declarations</h2><div id="ethics-content">
              
                <h3 id="FPar1">Competing interests</h3>
                <p>The authors declare no competing interests.</p>
              
            </div></div></section><section data-title="Additional information"><div id="additional-information-section"><h2 id="additional-information">Additional information</h2><p><b>Publisher’s note</b> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div></section><section data-title="Supplementary information"><div id="Sec9-section"><h2 id="Sec9">Supplementary information</h2></div></section><section data-title="Rights and permissions"><div id="rightslink-section"><h2 id="rightslink">Rights and permissions</h2><div id="rightslink-content">
                <p><b>Open Access</b>  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>.</p>
              <p><a data-track="click" data-track-action="view rights and permissions" data-track-label="link" href="https://s100.copyright.com/AppDispatchServlet?title=Evidence%20for%20~12-h%20ultradian%20gene%20programs%20in%20humans&amp;author=Bokai%20Zhu%20et%20al&amp;contentID=10.1038%2Fs44323-024-00005-1&amp;copyright=The%20Author%28s%29&amp;publication=2948-281X&amp;publicationDate=2024-08-09&amp;publisherName=SpringerNature&amp;orderBeanReset=true&amp;oa=CC%20BY">Reprints and permissions</a></p></div></div></section><section aria-labelledby="article-info" data-title="About this article"><div id="article-info-section"><h2 id="article-info">About this article</h2><div id="article-info-content"><div><p><a data-crossmark="10.1038/s44323-024-00005-1" target="_blank" rel="noopener" href="https://crossmark.crossref.org/dialog/?doi=10.1038/s44323-024-00005-1" data-track="click" data-track-action="Click Crossmark" data-track-label="link" data-test="crossmark"><img loading="lazy" width="57" height="81" alt="Check for updates. Verify currency and authenticity via CrossMark" src="data:image/svg+xml;base64,<svg height="81" width="57" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="m17.35 35.45 21.3-14.2v-17.03h-21.3" fill="#989898"/><path d="m38.65 35.45-21.3-14.2v-17.03h21.3" fill="#747474"/><path d="m28 .5c-12.98 0-23.5 10.52-23.5 23.5s10.52 23.5 23.5 23.5 23.5-10.52 23.5-23.5c0-6.23-2.48-12.21-6.88-16.62-4.41-4.4-10.39-6.88-16.62-6.88zm0 41.25c-9.8 0-17.75-7.95-17.75-17.75s7.95-17.75 17.75-17.75 17.75 7.95 17.75 17.75c0 4.71-1.87 9.22-5.2 12.55s-7.84 5.2-12.55 5.2z" fill="#535353"/><path d="m41 36c-5.81 6.23-15.23 7.45-22.43 2.9-7.21-4.55-10.16-13.57-7.03-21.5l-4.92-3.11c-4.95 10.7-1.19 23.42 8.78 29.71 9.97 6.3 23.07 4.22 30.6-4.86z" fill="#9c9c9c"/><path d="m.2 58.45c0-.75.11-1.42.33-2.01s.52-1.09.91-1.5c.38-.41.83-.73 1.34-.94.51-.22 1.06-.32 1.65-.32.56 0 1.06.11 1.51.35.44.23.81.5 1.1.81l-.91 1.01c-.24-.24-.49-.42-.75-.56-.27-.13-.58-.2-.93-.2-.39 0-.73.08-1.05.23-.31.16-.58.37-.81.66-.23.28-.41.63-.53 1.04-.13.41-.19.88-.19 1.39 0 1.04.23 1.86.68 2.46.45.59 1.06.88 1.84.88.41 0 .77-.07 1.07-.23s.59-.39.85-.68l.91 1c-.38.43-.8.76-1.28.99-.47.22-1 .34-1.58.34-.59 0-1.13-.1-1.64-.31-.5-.2-.94-.51-1.31-.91-.38-.4-.67-.9-.88-1.48-.22-.59-.33-1.26-.33-2.02zm8.4-5.33h1.61v2.54l-.05 1.33c.29-.27.61-.51.96-.72s.76-.31 1.24-.31c.73 0 1.27.23 1.61.71.33.47.5 1.14.5 2.02v4.31h-1.61v-4.1c0-.57-.08-.97-.25-1.21-.17-.23-.45-.35-.83-.35-.3 0-.56.08-.79.22-.23.15-.49.36-.78.64v4.8h-1.61zm7.37 6.45c0-.56.09-1.06.26-1.51.18-.45.42-.83.71-1.14.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.36c.07.62.29 1.1.65 1.44.36.33.82.5 1.38.5.29 0 .57-.04.83-.13s.51-.21.76-.37l.55 1.01c-.33.21-.69.39-1.09.53-.41.14-.83.21-1.26.21-.48 0-.92-.08-1.34-.25-.41-.16-.76-.4-1.07-.7-.31-.31-.55-.69-.72-1.13-.18-.44-.26-.95-.26-1.52zm4.6-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.07.45-.31.29-.5.73-.58 1.3zm2.5.62c0-.57.09-1.08.28-1.53.18-.44.43-.82.75-1.13s.69-.54 1.1-.71c.42-.16.85-.24 1.31-.24.45 0 .84.08 1.17.23s.61.34.85.57l-.77 1.02c-.19-.16-.38-.28-.56-.37-.19-.09-.39-.14-.61-.14-.56 0-1.01.21-1.35.63-.35.41-.52.97-.52 1.67 0 .69.17 1.24.51 1.66.34.41.78.62 1.32.62.28 0 .54-.06.78-.17.24-.12.45-.26.64-.42l.67 1.03c-.33.29-.69.51-1.08.65-.39.15-.78.23-1.18.23-.46 0-.9-.08-1.31-.24-.4-.16-.75-.39-1.05-.7s-.53-.69-.7-1.13c-.17-.45-.25-.96-.25-1.53zm6.91-6.45h1.58v6.17h.05l2.54-3.16h1.77l-2.35 2.8 2.59 4.07h-1.75l-1.77-2.98-1.08 1.23v1.75h-1.58zm13.69 1.27c-.25-.11-.5-.17-.75-.17-.58 0-.87.39-.87 1.16v.75h1.34v1.27h-1.34v5.6h-1.61v-5.6h-.92v-1.2l.92-.07v-.72c0-.35.04-.68.13-.98.08-.31.21-.57.4-.79s.42-.39.71-.51c.28-.12.63-.18 1.04-.18.24 0 .48.02.69.07.22.05.41.1.57.17zm.48 5.18c0-.57.09-1.08.27-1.53.17-.44.41-.82.72-1.13.3-.31.65-.54 1.04-.71.39-.16.8-.24 1.23-.24s.84.08 1.24.24c.4.17.74.4 1.04.71s.54.69.72 1.13c.19.45.28.96.28 1.53s-.09 1.08-.28 1.53c-.18.44-.42.82-.72 1.13s-.64.54-1.04.7-.81.24-1.24.24-.84-.08-1.23-.24-.74-.39-1.04-.7c-.31-.31-.55-.69-.72-1.13-.18-.45-.27-.96-.27-1.53zm1.65 0c0 .69.14 1.24.43 1.66.28.41.68.62 1.18.62.51 0 .9-.21 1.19-.62.29-.42.44-.97.44-1.66 0-.7-.15-1.26-.44-1.67-.29-.42-.68-.63-1.19-.63-.5 0-.9.21-1.18.63-.29.41-.43.97-.43 1.67zm6.48-3.44h1.33l.12 1.21h.05c.24-.44.54-.79.88-1.02.35-.24.7-.36 1.07-.36.32 0 .59.05.78.14l-.28 1.4-.33-.09c-.11-.01-.23-.02-.38-.02-.27 0-.56.1-.86.31s-.55.58-.77 1.1v4.2h-1.61zm-47.87 15h1.61v4.1c0 .57.08.97.25 1.2.17.24.44.35.81.35.3 0 .57-.07.8-.22.22-.15.47-.39.73-.73v-4.7h1.61v6.87h-1.32l-.12-1.01h-.04c-.3.36-.63.64-.98.86-.35.21-.76.32-1.24.32-.73 0-1.27-.24-1.61-.71-.33-.47-.5-1.14-.5-2.02zm9.46 7.43v2.16h-1.61v-9.59h1.33l.12.72h.05c.29-.24.61-.45.97-.63.35-.17.72-.26 1.1-.26.43 0 .81.08 1.15.24.33.17.61.4.84.71.24.31.41.68.53 1.11.13.42.19.91.19 1.44 0 .59-.09 1.11-.25 1.57-.16.47-.38.85-.65 1.16-.27.32-.58.56-.94.73-.35.16-.72.25-1.1.25-.3 0-.6-.07-.9-.2s-.59-.31-.87-.56zm0-2.3c.26.22.5.37.73.45.24.09.46.13.66.13.46 0 .84-.2 1.15-.6.31-.39.46-.98.46-1.77 0-.69-.12-1.22-.35-1.61-.23-.38-.61-.57-1.13-.57-.49 0-.99.26-1.52.77zm5.87-1.69c0-.56.08-1.06.25-1.51.16-.45.37-.83.65-1.14.27-.3.58-.54.93-.71s.71-.25 1.08-.25c.39 0 .73.07 1 .2.27.14.54.32.81.55l-.06-1.1v-2.49h1.61v9.88h-1.33l-.11-.74h-.06c-.25.25-.54.46-.88.64-.33.18-.69.27-1.06.27-.87 0-1.56-.32-2.07-.95s-.76-1.51-.76-2.65zm1.67-.01c0 .74.13 1.31.4 1.7.26.38.65.58 1.15.58.51 0 .99-.26 1.44-.77v-3.21c-.24-.21-.48-.36-.7-.45-.23-.08-.46-.12-.7-.12-.45 0-.82.19-1.13.59-.31.39-.46.95-.46 1.68zm6.35 1.59c0-.73.32-1.3.97-1.71.64-.4 1.67-.68 3.08-.84 0-.17-.02-.34-.07-.51-.05-.16-.12-.3-.22-.43s-.22-.22-.38-.3c-.15-.06-.34-.1-.58-.1-.34 0-.68.07-1 .2s-.63.29-.93.47l-.59-1.08c.39-.24.81-.45 1.28-.63.47-.17.99-.26 1.54-.26.86 0 1.51.25 1.93.76s.63 1.25.63 2.21v4.07h-1.32l-.12-.76h-.05c-.3.27-.63.48-.98.66s-.73.27-1.14.27c-.61 0-1.1-.19-1.48-.56-.38-.36-.57-.85-.57-1.46zm1.57-.12c0 .3.09.53.27.67.19.14.42.21.71.21.28 0 .54-.07.77-.2s.48-.31.73-.56v-1.54c-.47.06-.86.13-1.18.23-.31.09-.57.19-.76.31s-.33.25-.41.4c-.09.15-.13.31-.13.48zm6.29-3.63h-.98v-1.2l1.06-.07.2-1.88h1.34v1.88h1.75v1.27h-1.75v3.28c0 .8.32 1.2.97 1.2.12 0 .24-.01.37-.04.12-.03.24-.07.34-.11l.28 1.19c-.19.06-.4.12-.64.17-.23.05-.49.08-.76.08-.4 0-.74-.06-1.02-.18-.27-.13-.49-.3-.67-.52-.17-.21-.3-.48-.37-.78-.08-.3-.12-.64-.12-1.01zm4.36 2.17c0-.56.09-1.06.27-1.51s.41-.83.71-1.14c.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.37c.08.62.29 1.1.65 1.44.36.33.82.5 1.38.5.3 0 .58-.04.84-.13.25-.09.51-.21.76-.37l.54 1.01c-.32.21-.69.39-1.09.53s-.82.21-1.26.21c-.47 0-.92-.08-1.33-.25-.41-.16-.77-.4-1.08-.7-.3-.31-.54-.69-.72-1.13-.17-.44-.26-.95-.26-1.52zm4.61-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.08.45-.31.29-.5.73-.57 1.3zm3.01 2.23c.31.24.61.43.92.57.3.13.63.2.98.2.38 0 .65-.08.83-.23s.27-.35.27-.6c0-.14-.05-.26-.13-.37-.08-.1-.2-.2-.34-.28-.14-.09-.29-.16-.47-.23l-.53-.22c-.23-.09-.46-.18-.69-.3-.23-.11-.44-.24-.62-.4s-.33-.35-.45-.55c-.12-.21-.18-.46-.18-.75 0-.61.23-1.1.68-1.49.44-.38 1.06-.57 1.83-.57.48 0 .91.08 1.29.25s.71.36.99.57l-.74.98c-.24-.17-.49-.32-.73-.42-.25-.11-.51-.16-.78-.16-.35 0-.6.07-.76.21-.17.15-.25.33-.25.54 0 .14.04.26.12.36s.18.18.31.26c.14.07.29.14.46.21l.54.19c.23.09.47.18.7.29s.44.24.64.4c.19.16.34.35.46.58.11.23.17.5.17.82 0 .3-.06.58-.17.83-.12.26-.29.48-.51.68-.23.19-.51.34-.84.45-.34.11-.72.17-1.15.17-.48 0-.95-.09-1.41-.27-.46-.19-.86-.41-1.2-.68z" fill="#535353"/></g></svg>"/></a></p><div><h3 id="citeas">Cite this article</h3><p>Zhu, B., Liu, S., David, N.L. <i>et al.</i> Evidence for ~12-h ultradian gene programs in humans.
                    <i>npj Biol Timing Sleep</i> <b>1</b>, 4 (2024). https://doi.org/10.1038/s44323-024-00005-1</p><p><a data-test="citation-link" data-track="click" data-track-action="download article citation" data-track-label="link" data-track-external="" rel="nofollow" href="https://citation-needed.springer.com/v2/references/10.1038/s44323-024-00005-1?format=refman&amp;flavour=citation">Download citation<svg width="16" height="16" focusable="false" role="img" aria-hidden="true"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-download-medium"></use></svg></a></p><ul data-test="publication-history"><li><p>Received<span>: </span><span><time datetime="2024-02-27">27 February 2024</time></span></p></li><li><p>Accepted<span>: </span><span><time datetime="2024-07-11">11 July 2024</time></span></p></li><li><p>Published<span>: </span><span><time datetime="2024-08-09">09 August 2024</time></span></p></li><li><p><abbr title="Digital Object Identifier">DOI</abbr><span>: </span><span>https://doi.org/10.1038/s44323-024-00005-1</span></p></li></ul></div></div></div></div></section>
            </div></div>
  </body>
</html>
