<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <title>James Routley | Feed</title>
    <link
      rel="stylesheet"
      type="text/css"
      href="../styles.css"
      media="screen"
    />
  </head>
  <body>
    <a href="/index.html">Back</a>
    <a href="https://www.nature.com/articles/d41586-024-02214-x">Original</a>
    <h1>Ex-Meta scientists debut gigantic AI protein design model</h1>
    
    <div id="readability-page-1" class="page"><div>
                    <figure>
 <picture>
  <source type="image/webp" srcset="//media.nature.com/lw767/magazine-assets/d41586-024-02214-x/d41586-024-02214-x_27319756.jpg?as=webp 767w, //media.nature.com/lw319/magazine-assets/d41586-024-02214-x/d41586-024-02214-x_27319756.jpg?as=webp 319w" sizes="(max-width: 319px) 319px, (min-width: 1023px) 100vw,  767px"/>
  <img alt="Molecular model of the bright green fluorescent protein StayGold from Cytaeis uchidae." loading="lazy" src="https://media.nature.com/lw767/magazine-assets/d41586-024-02214-x/d41586-024-02214-x_27319756.jpg"/>
  <figcaption>
   <p><span>A structural model of green fluorescent protein, a workhorse of biotechnology.</span><span>Credit: Laguna Design/Science Photo Library</span></p>
  </figcaption>
 </picture>
</figure><p>An <a href="https://www.nature.com/subjects/machine-learning" data-track="click" data-label="https://www.nature.com/subjects/machine-learning" data-track-category="body text link">artificial intelligence</a> (AI) model that speaks the <a href="https://www.nature.com/subjects/protein-design" data-track="click" data-label="https://www.nature.com/subjects/protein-design" data-track-category="body text link">language of proteins</a> — one of the largest yet developed for biology — has been used to create new fluorescent molecules.</p><p>The proof-of-principle demonstration was announced this month by EvolutionaryScale in New York City, alongside US$142 million in new funding to apply its model to <a href="https://www.nature.com/articles/d41586-023-02227-y" data-track="click" data-label="https://www.nature.com/articles/d41586-023-02227-y" data-track-category="body text link">drug development</a>, sustainability and other pursuits. The company, launched by scientists who previously worked at tech giant Meta, is the latest entrant in an increasingly crowded field that is applying cutting-edge machine-learning models trained on language and images to biological data.</p><article data-label="Related">
  <a href="https://www.nature.com/articles/d41586-023-02227-y" data-track="click" data-track-label="recommended article"><img alt="" src="https://media.nature.com/w400/magazine-assets/d41586-024-02214-x/d41586-024-02214-x_26069432.jpg"/><p>AI tools are designing entirely new proteins that could transform medicine</p></a>
 </article><p>“We want to build tools that can make biology programmable,” says Alex Rives, the company’s chief scientist, who was part of Meta’s efforts to apply AI to biological data.</p><p>EvolutionaryScale’s AI tool, called ESM3, is what’s known as a protein language model. It was trained on more than 2.7 billion protein sequences and structures, as well as information about these proteins’ functions. The model can be used to <a href="https://www.nature.com/articles/d41586-023-02227-y" data-track="click" data-label="https://www.nature.com/articles/d41586-023-02227-y" data-track-category="body text link">create proteins</a> to specifications provided by users, akin to the text spit out by chatbots such as ChatGPT.</p><p>“It’s going to be one of the AI models in biology that everybody’s paying attention to,” says Anthony Gitter, a computational biologist at the University of Wisconsin–Madison.</p><h2>Glowing up</h2><p>Rives and his colleagues worked on earlier iterations of the ESM model at Meta, but struck out on their own last year, after Meta ended its work in this area. They had previously used the model ESM-2 to create a <a href="https://www.nature.com/articles/d41586-022-03539-1" data-track="click" data-label="https://www.nature.com/articles/d41586-022-03539-1" data-track-category="body text link">freely available database of 600 million predicted protein structures</a><sup><a href="#ref-CR1" data-track="click" data-action="anchor-link" data-track-label="go to reference" data-track-category="references">1</a></sup>. Other teams have since used versions of ESM-1 to <a href="https://www.nature.com/articles/d41586-023-01516-w" data-track="click" data-label="https://www.nature.com/articles/d41586-023-01516-w" data-track-category="body text link">design antibodies with improved activity against pathogens</a> including SARS-CoV-2<sup><a href="#ref-CR2" data-track="click" data-action="anchor-link" data-track-label="go to reference" data-track-category="references">2</a></sup> and to re-engineer ‘anti-CRISPR’ proteins to improve the efficiency of gene-editing tools<sup><a href="#ref-CR3" data-track="click" data-action="anchor-link" data-track-label="go to reference" data-track-category="references">3</a></sup>.</p><p>This year, another biology AI company, Profluent in Berkeley, California, used its own protein language model to <a href="https://www.nature.com/articles/d41586-024-01243-w" data-track="click" data-label="https://www.nature.com/articles/d41586-024-01243-w" data-track-category="body text link">create new CRISPR-inspired gene-editing proteins</a>, and made one such molecule freely available for use.</p><p>To demonstrate its latest model, Rives’ team set out to overhaul another biotechnology workhorse: the green fluorescent protein (GFP), which absorbs blue light and glows green. Researchers isolated GFP in the 1960s, from the bioluminescent jellyfish <i>Aequorea victoria</i>. Later work — which, with the discovery, was recognized with a Nobel prize — showed how GFP could label other proteins viewed under a microscope, explained the molecular basis for its fluorescence and developed synthetic versions of the protein that glowed much more brightly and in different colours.</p><p>Researchers have since identified other similarly shaped fluorescent proteins, all sharing a light-absorbing and -emitting ‘chromophore’ core surrounded by a barrel-shaped scaffold. Rives’ team asked ESM3 to create examples of GFP-like proteins that contained a set of key amino acids found in GFP’s chromophore.</p><p>The researchers synthesized 88 of the most promising designs and measured their ability to fluoresce. Most were duds, but one design, dissimilar to known fluorescent proteins, glowed faintly — about 50 times weaker than natural forms of GFP. Using this molecule’s sequence as a starting point, the researchers tasked ESM3 with improving on its work. When the researchers made around 100 of the resulting designs, several were as bright as natural GFPs, which are still vastly dimmer than lab-engineered variants.</p><p>One of the brightest ESM3-designed proteins, dubbed esmGFP, is predicted to have a structure resembling those of natural fluorescent proteins. However, its amino-acid sequence is vastly different, matching less than 60% of the sequence of the most closely related fluorescent protein in its training data set. In a preprint posted on the server bioRxiv<sup><a href="#ref-CR4" data-track="click" data-action="anchor-link" data-track-label="go to reference" data-track-category="references">4</a></sup>, Rives and his colleagues say that on the basis of natural mutation rates, this level of sequence difference equates to “over 500 million years of evolution”.</p><p>But Gitter worries that this comparison is an unhelpful, and potentially misleading, way of describing the product of a cutting-edge AI model. “It sounds scary when you think about AI and accelerating evolution,” he says. “I feel like overhyping what a model does can hurt the field and it can be dangerous for the public.”</p><p>Rives sees ESM3’s generation of new proteins by iterating through various sequences as analogous to evolution. “We think the perspective of what it would take for nature to generate something like this is an interesting one,” he adds.</p><h2>Risk threshold</h2><p>ESM3 is among the first biological AI models to use enough computing power during its training to require developers to notify the US government and report <a href="https://www.nature.com/articles/d41586-024-00699-0" data-track="click" data-label="https://www.nature.com/articles/d41586-024-00699-0" data-track-category="body text link">risk-mitigation measures</a>, under a 2023 presidential executive order. EvolutionaryScale says it has already been in touch with the US Office of Science and Technology Policy.</p><article data-label="Related">
  <a href="https://www.nature.com/articles/d41586-024-00699-0" data-track="click" data-track-label="recommended article"><img alt="" src="https://media.nature.com/w400/magazine-assets/d41586-024-02214-x/d41586-024-02214-x_26828914.jpg"/><p>Could AI-designed proteins be weaponized? Scientists lay out safety guidelines</p></a>
 </article><p>The version of ESM3 that eclipsed that threshold — comprising nearly 100 billion parameters, or variables the model uses to represent relationships between sequences — is not publicly available. For a smaller open-source version, certain sequences, such as those from viruses and a US government list of worrying pathogens and toxins, were excluded from training. Neither can ESM3-open — which scientists anywhere can download and run independently — be prompted to generate such proteins.</p><p>Martin Pacesa, a structural biologist at the Swiss Federal Institute of Technology in Lausanne, is excited to begin working with ESM3. It is one of the first biological models to allow researchers to specify designs using natural-language descriptions of its properties and functions, he notes, and he is eager to see how this and other features perform experimentally.</p><p>Pacesa is impressed that EvolutionaryScale released an open-source version of ESM3, and a clear description of how the largest version was trained. But the largest model would take immense computing resources to develop independently, he says. “No academic lab will be able to replicate it.”</p><p>Rives is eager to apply ESM3 to other designs. Pacesa, who was part of the team that used a different protein language model to make new CRISPR proteins, says it will be interesting to see how ESM3 does at this. Rives envisions applications in sustainability — a video on the company’s website shows the design of plastic-eating enzymes — and in the development of antibodies and other protein-based drugs. “It’s really a model at the frontier,” he says.</p>
                </div></div>
  </body>
</html>
