<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <title>James Routley | Feed</title>
    <link
      rel="stylesheet"
      type="text/css"
      href="../styles.css"
      media="screen"
    />
  </head>
  <body>
    <a href="/index.html">Back</a>
    <a href="https://www.nature.com/articles/s41598-023-46151-7">Original</a>
    <h1>Extracting DNA from the air – DNA evidence of human occupancy in indoor premises</h1>
    
    <div id="readability-page-1" class="page"><div>
                    
                        <section data-title="Introduction"><div id="Sec1-section"><h2 id="Sec1">Introduction</h2><div id="Sec1-content"><p>A human adult sheds about 1000 skin cells per cm<sup>2</sup> per hour, causing billions of cells to be released from the body every day<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 1" title="Roberts, D. &amp; Marks, R. The determination of regional and age variations in the rate of desquamation: A comparison of four techniques. J. Invest. Dermatol. 74, 13–16. 
                  https://doi.org/10.1111/1523-1747.ep12514568
                  
                 (1980)." href="https://interjectedfuture.com/articles/s41598-023-46151-7#ref-CR1" id="ref-link-section-d43177717e385">1</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 2" title="Milstone, L. M. Epidermal desquamation. J. Dermatol. Sci. 36, 131–140. 
                  https://doi.org/10.1016/j.jdermsci.2004.05.004
                  
                 (2004)." href="https://interjectedfuture.com/articles/s41598-023-46151-7#ref-CR2" id="ref-link-section-d43177717e388">2</a></sup>. This happens through epidermal desquamation, which is a continuous process that constitutes the final active step in the keratinocytes’ differentiation program. Desquamation causes a spontaneous detachment of dead skin cells known as corneocytes<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Milstone, L. M. Epidermal desquamation. J. Dermatol. Sci. 36, 131–140. 
                  https://doi.org/10.1016/j.jdermsci.2004.05.004
                  
                 (2004)." href="#ref-CR2" id="ref-link-section-d43177717e392">2</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Candi, E., Schmidt, R. &amp; Melino, G. The cornified envelope: A model of cell death in the skin. Nat. Rev. Mol. Cell Biol. 6, 328–340. 
                  https://doi.org/10.1038/nrm1619
                  
                 (2005)." href="#ref-CR3" id="ref-link-section-d43177717e392_1">3</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 4" title="Sun, T. T. &amp; Green, H. Differentiation of the epidermal keratinocyte in cell culture: Formation of the cornified envelope. Cell 9, 511–521. 
                  https://doi.org/10.1016/0092-8674(76)90033-7
                  
                 (1976)." href="https://interjectedfuture.com/articles/s41598-023-46151-7#ref-CR4" id="ref-link-section-d43177717e395">4</a></sup>. The average size of these skin particles is smaller than the pores of typical clothing fabrics, allowing them to pass through and become aerosolized<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 5" title="Lidwell, O. M., Mackintosh, C. A. &amp; Towers, A. G. The evaluation of fabrics in relation to their use as protective garments in nursing and surgery. II. Dispersal of skin organisms in a test chamber. J. Hyg. (Lond.) 81, 453–469. 
                  https://doi.org/10.1017/s002217240002533x
                  
                 (1978)." href="https://interjectedfuture.com/articles/s41598-023-46151-7#ref-CR5" id="ref-link-section-d43177717e399">5</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 6" title="Mackintosh, C. A., Lidwell, O. M., Towers, A. G. &amp; Marples, R. R. The dimensions of skin fragments dispersed into the air during activity. J. Hyg. (Lond.) 81, 471–479. 
                  https://doi.org/10.1017/s0022172400025341
                  
                 (1978)." href="https://interjectedfuture.com/articles/s41598-023-46151-7#ref-CR6" id="ref-link-section-d43177717e402">6</a></sup>. Varying levels of DNA may be retained within corneocytes; enough to yield detectable profiles<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Burrill, J., Daniel, B. &amp; Frascione, N. Illuminating touch deposits through cellular characterization of hand rinses and body fluids with nucleic acid fluorescence. Forensic Sci. Int. Genet. 46, 102269. 
                  https://doi.org/10.1016/j.fsigen.2020.102269
                  
                 (2020)." href="#ref-CR7" id="ref-link-section-d43177717e406">7</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Kanokwongnuwut, P., Kirkbride, K. P. &amp; Linacre, A. Detection of latent DNA. Forensic Sci. Int. Genet. 37, 95–101. 
                  https://doi.org/10.1016/j.fsigen.2018.08.004
                  
                 (2018)." href="#ref-CR8" id="ref-link-section-d43177717e406_1">8</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Burrill, J., Rammenou, E., Alawar, F., Daniel, B. &amp; Frascione, N. Corneocyte lysis and fragmented DNA considerations for the cellular component of forensic touch DNA. Forensic Sci. Int. Genet. 51, 102428. 
                  https://doi.org/10.1016/j.fsigen.2020.102428
                  
                 (2021)." href="#ref-CR9" id="ref-link-section-d43177717e406_2">9</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 10" title="Hammer, U., Bulnheim, U., Karstädt, G., Meissner, D. &amp; Wegener, R. Zur DNA-Typisierung übertragener Hautzellen nach körperlicher Gewalt. Rechtsmedizin 7, 180–183. 
                  https://doi.org/10.1007/s001940050010
                  
                 (1997)." href="https://interjectedfuture.com/articles/s41598-023-46151-7#ref-CR10" id="ref-link-section-d43177717e409">10</a></sup>. DNA can be released in the air not only through skin cells but also in other forms. Dandruff constitutes part of bioaerosol material shed by humans, with a single particle containing between 0.8 and 16.6 ng of DNA<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 11" title="Herber, B. &amp; Herold, K. DNA typing of human dandruff. J. Forensic Sci. 43, 648–656 (1998)." href="https://interjectedfuture.com/articles/s41598-023-46151-7#ref-CR11" id="ref-link-section-d43177717e414">11</a></sup>. Detectable amounts of DNA can also be transferred to the surrounding environment by an individual speaking, coughing or sneezing<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Finnebraaten, M., Granér, T. &amp; Hoff-Olsen, P. May a speaking individual contaminate the routine DNA laboratory?. Forensic Sci. Int. Genet. Suppl. Ser. 1, 421–422. 
                  https://doi.org/10.1016/j.fsigss.2007.10.030
                  
                 (2008)." href="#ref-CR12" id="ref-link-section-d43177717e418">12</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Port, N. J., Bowyer, V. L., Graham, E. A., Batuwangala, M. S. &amp; Rutty, G. N. How long does it take a static speaking individual to contaminate the immediate environment?. Forensic Sci. Med. Pathol. 2, 157–163. 
                  https://doi.org/10.1007/s12024-006-0004-z
                  
                 (2006)." href="#ref-CR13" id="ref-link-section-d43177717e418_1">13</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 14" title="Rutty, G. N., Hopwood, A. &amp; Tucker, V. The effectiveness of protective clothing in the reduction of potential DNA contamination of the scene of crime. Int. J. Legal Med. 117, 170–174. 
                  https://doi.org/10.1007/s00414-002-0348-1
                  
                 (2003)." href="https://interjectedfuture.com/articles/s41598-023-46151-7#ref-CR14" id="ref-link-section-d43177717e421">14</a></sup>. The potential for indirect DNA transfer through agitation of dry biological materials, including DNA on touched surfaces, and personal items like clothing, has also been demonstrated in the absence of direct contact<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 15" title="Thornbury, D., Goray, M. &amp; van Oorschot, R. A. H. Indirect DNA transfer without contact from dried biological materials on various surfaces. Forensic Sci. Int. Genet. 51, 102457. 
                  https://doi.org/10.1016/j.fsigen.2020.102457
                  
                 (2021)." href="https://interjectedfuture.com/articles/s41598-023-46151-7#ref-CR15" id="ref-link-section-d43177717e425">15</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 16" title="Thornbury, D., Goray, M. &amp; van Oorschot, R. A. H. Transfer of DNA without contact from used clothing, pillowcases and towels by shaking agitation. Sci. Justice 61, 797–805. 
                  https://doi.org/10.1016/j.scijus.2021.10.005
                  
                 (2021)." href="https://interjectedfuture.com/articles/s41598-023-46151-7#ref-CR16" id="ref-link-section-d43177717e428">16</a></sup>. Individuals can deposit DNA on untouched surfaces in areas they occupy; the probability of detecting DNA increases with the duration of their presence<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 17" title="Puliatti, L., Handt, O. &amp; Taylor, D. The level of DNA an individual transfers to untouched items in their immediate surroundings. Forensic Sci. Int. Genet. 54, 102561. 
                  https://doi.org/10.1016/j.fsigen.2021.102561
                  
                 (2021)." href="https://interjectedfuture.com/articles/s41598-023-46151-7#ref-CR17" id="ref-link-section-d43177717e432">17</a></sup>. Furthermore, significant differences were observed in the propensity of individuals to shed DNA into their immediate surroundings, without coming into direct contact with surfaces; an “environmental shedder status”<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 17" title="Puliatti, L., Handt, O. &amp; Taylor, D. The level of DNA an individual transfers to untouched items in their immediate surroundings. Forensic Sci. Int. Genet. 54, 102561. 
                  https://doi.org/10.1016/j.fsigen.2021.102561
                  
                 (2021)." href="https://interjectedfuture.com/articles/s41598-023-46151-7#ref-CR17" id="ref-link-section-d43177717e436">17</a></sup> can be assigned to individuals to quantify this variability. Skin cells and DNA fragments floating in the air make up a significant proportion of indoor dust<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 18" title="Locard, E. The analysis of dust traces. Part I. Am. J. Police Sci. 1, 276–298. 
                  https://doi.org/10.2307/1147154
                  
                 (1930)." href="https://interjectedfuture.com/articles/s41598-023-46151-7#ref-CR18" id="ref-link-section-d43177717e440">18</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 19" title="Toothman, M. H. et al. Characterization of human DNA in environmental samples. Forensic Sci. Int. 178, 7–15. 
                  https://doi.org/10.1016/j.forsciint.2008.01.016
                  
                 (2008)." href="https://interjectedfuture.com/articles/s41598-023-46151-7#ref-CR19" id="ref-link-section-d43177717e443">19</a></sup>. It was previously demonstrated by Toothman et al.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 19" title="Toothman, M. H. et al. Characterization of human DNA in environmental samples. Forensic Sci. Int. 178, 7–15. 
                  https://doi.org/10.1016/j.forsciint.2008.01.016
                  
                 (2008)." href="https://interjectedfuture.com/articles/s41598-023-46151-7#ref-CR19" id="ref-link-section-d43177717e448">19</a></sup> that human DNA is present in indoor dust in sufficient quantity and quality to produce allele calls in STR analysis. However, the work described in this paper is the first study that explores the potential use of air and dust to investigate serious crimes, along with a description of limitations and potential pit-falls.</p><p>There are many cases in police investigations that require identification of individuals from crime scenes<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 20" title="Amankwaa, A. O. &amp; McCartney, C. The effectiveness of the current use of forensic DNA in criminal investigations in England and Wales. Wiley Interdiscip. Rev. Forensic Sci. 3, e1414. 
                  https://doi.org/10.1002/wfs2.1414
                  
                 (2021)." href="https://interjectedfuture.com/articles/s41598-023-46151-7#ref-CR20" id="ref-link-section-d43177717e455">20</a></sup>. Some examples are cases of burglaries, homicides, domestic violence and identification of members of illegal organizations. Often, the most difficult cases involve organized crime and terrorism, where the participants may be forensically aware, hence the detection of conventional DNA and fingerprints may be difficult to achieve. This happens when (a) there is no body fluid to identify, (b) surfaces such as tables, door handles, light switches may have been cleaned or (c) criminals may have worn gloves, which minimizes their direct skin contact with surfaces. </p><p>In addition, criminals typically employ “safe-houses”, which may change over time to evade detection. These “safe-houses” serve as temporary locations for various illegal activities, ranging from the production of illicit drugs and explosives to housing illegal organizations such as cults, scams syndicates, and unlicensed clinics, often under clever disguises<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 21" title="Secretary of State for the Home Department (2013). Serious and Organised Crime Strategy.UK: The Stationery Office. 
                  https://www.gov.uk/government/publications/serious-organised-crime-strategy
                  
                ." href="https://interjectedfuture.com/articles/s41598-023-46151-7#ref-CR21" id="ref-link-section-d43177717e462">21</a></sup>. These activities, regardless of their nature, could potentially lead to police raids based on intelligence reports. During such operations, while some of the current occupants may be apprehended, it becomes essential to gather evidence not only from those currently present but also from individuals who may have been associated with these locations at some point in the past. To achieve this, DNA isolated from dust provides an historical record and may be useful to identify occupants over a long time-frame, from a few months up to a few years. Conversely, DNA extracted from air samples could provide a more recent record: collection of air samples may be useful in specific cases where the offender has been present in a confined area for a short period of time, insufficient for skin cells and DNA to accumulate in dust. For example, air could be a valuable source of information in the case of car theft, where the forensically aware criminal may take care to wear gloves, which reduces the possibility of DNA detection from surfaces such as steering wheels. </p><p>Whereas, the investigation of crime is a strong motivation, it is important to consider the practical implications of collection and analysis of air and dust samples in the laboratory environment. The high sensitivity of DNA technology makes it possible to obtain results from very low levels of biological materials (just a few cells)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 22" title="Watkins, D. R. L., Myers, D., Xavier, H. E. &amp; Marciano, M. A. Revisiting single cell analysis in forensic science. Sci. Rep. 11, 7054. 
                  https://doi.org/10.1038/s41598-021-86271-6
                  
                 (2021)." href="https://interjectedfuture.com/articles/s41598-023-46151-7#ref-CR22" id="ref-link-section-d43177717e469">22</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 23" title="van Oorschot, R. A., Ballantyne, K. N. &amp; Mitchell, R. J. Forensic trace DNA: A review. Investig. Genet. 1, 14. 
                  https://doi.org/10.1186/2041-2223-1-14
                  
                 (2010)." href="https://interjectedfuture.com/articles/s41598-023-46151-7#ref-CR23" id="ref-link-section-d43177717e472">23</a></sup>. Consequently, there is an increased possibility that environmental background DNA, defined as originating from unknown individuals<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 24" title="van Oorschot, R. A. H., Szkuta, B., Meakin, G. E., Kokshoorn, B. &amp; Goray, M. DNA transfer in forensic science: A review. Forensic Sci. Int. Genet. 38, 140–166. 
                  https://doi.org/10.1016/j.fsigen.2018.10.014
                  
                 (2019)." href="https://interjectedfuture.com/articles/s41598-023-46151-7#ref-CR24" id="ref-link-section-d43177717e476">24</a></sup>, may be detected<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="van Oorschot, R. A. H., Szkuta, B., Meakin, G. E., Kokshoorn, B. &amp; Goray, M. DNA transfer in forensic science: A review. Forensic Sci. Int. Genet. 38, 140–166. 
                  https://doi.org/10.1016/j.fsigen.2018.10.014
                  
                 (2019)." href="#ref-CR24" id="ref-link-section-d43177717e480">24</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Gill, P. DNA evidence and miscarriages of justice. Forensic Sci. Int. 294, e1–e3. 
                  https://doi.org/10.1016/j.forsciint.2018.12.003
                  
                 (2019)." href="#ref-CR25" id="ref-link-section-d43177717e480_1">25</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Fonneløp, A. E., Johannessen, H., Egeland, T. &amp; Gill, P. Contamination during criminal investigation: Detecting police contamination and secondary DNA transfer from evidence bags. Forensic Sci. Int. Genet. 23, 121–129. 
                  https://doi.org/10.1016/j.fsigen.2016.04.003
                  
                 (2016)." href="#ref-CR26" id="ref-link-section-d43177717e480_2">26</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Reither, J. B. et al. Investigation into the prevalence of background DNA on flooring within houses and its transfer to a contacting surface. Forensic Sci. Int. 318, 110563. 
                  https://doi.org/10.1016/j.forsciint.2020.110563
                  
                 (2021)." href="#ref-CR27" id="ref-link-section-d43177717e480_3">27</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Proff, C., Schmitt, C., Schneider, P. M., Foerster, G. &amp; Rothschild, M. A. Experiments on the DNA contamination risk via latent fingerprint brushes. Int. Congress Ser. 1288, 601–603. 
                  https://doi.org/10.1016/j.ics.2005.10.053
                  
                 (2006)." href="#ref-CR28" id="ref-link-section-d43177717e480_4">28</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="van den Berge, M., Ozcanhan, G., Zijlstra, S., Lindenbergh, A. &amp; Sijen, T. Prevalence of human cell material: DNA and RNA profiling of public and private objects and after activity scenarios. Forensic Sci. Int. Genet. 21, 81–89. 
                  https://doi.org/10.1016/j.fsigen.2015.12.012
                  
                 (2016)." href="#ref-CR29" id="ref-link-section-d43177717e480_5">29</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 30" title="Vandewoestyne, M. et al. Sources of DNA contamination and decontamination procedures in the forensic laboratory. Int. J. Forensic Pract. Res. 
                  https://doi.org/10.4172/2157-7145.S2-001
                  
                 (2011)." href="https://interjectedfuture.com/articles/s41598-023-46151-7#ref-CR30" id="ref-link-section-d43177717e483">30</a></sup>. In a laboratory where cases are analyzed, there is a risk that aerosol DNA may contaminate the evidence. Two different studies<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 30" title="Vandewoestyne, M. et al. Sources of DNA contamination and decontamination procedures in the forensic laboratory. Int. J. Forensic Pract. Res. 
                  https://doi.org/10.4172/2157-7145.S2-001
                  
                 (2011)." href="https://interjectedfuture.com/articles/s41598-023-46151-7#ref-CR30" id="ref-link-section-d43177717e487">30</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 31" title="Witt, N. et al. An assessment of air as a source of DNA contamination encountered when performing PCR. J. Biomol. Tech. 20, 236–240 (2009)." href="https://interjectedfuture.com/articles/s41598-023-46151-7#ref-CR31" id="ref-link-section-d43177717e490">31</a></sup> showed that air is unlikely to be a source of contamination in a forensic laboratory; however, the experiments were conducted more than 10 years ago, using methods less sensitive than those available today.</p><p>Dust and air may result in challenging forensic samples, typically consisting of DNA mixtures from multiple contributors, and prone to allelic dropout due to degraded DNA<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 32" title="Gill, P., Benschop, C., Buckleton, J., Bleka, Ø. &amp; Taylor, D. A review of probabilistic genotyping systems: EuroForMix, DNAStatistX and STRmix™. Genes (Basel) 12, 1559. 
                  https://doi.org/10.3390/genes12101559
                  
                 (2021)." href="https://interjectedfuture.com/articles/s41598-023-46151-7#ref-CR32" id="ref-link-section-d43177717e498">32</a></sup>. Moreover, the police may have an incomplete list of suspects with which to compare evidential samples. They may wish to extend the search to encompass national DNA databases of reference samples. This may entail many thousands or even millions of comparisons in order to properly search databases to compile lists of suspects, and to identify which evidential material they may have contributed to. Traditional database search engines such as CODIS (Combined DNA Index System)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 33" title="Budowle B., M. T. R., Niezgoda S. J., Brown B. L. CODIS and PCR-based short tandem repeat loci: Law enforcement tools. In Second European symposium on human identification. 73–88 (1998)." href="https://interjectedfuture.com/articles/s41598-023-46151-7#ref-CR33" id="ref-link-section-d43177717e502">33</a></sup> are inefficient, requiring user input, and limited to profiles that are well represented. An alternative strategy is to use probabilistic genotyping to calculate likelihood ratio statistics, where the value of the evidence is quantified for any given candidate donor to a questioned DNA profile. The open-source software EuroForMix<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 34" title="Bleka, Ø., Storvik, G. &amp; Gill, P. EuroForMix: An open source software based on a continuous model to evaluate STR DNA profiles from a mixture of contributors with artefacts. Forensic Sci. Int. Genet. 21, 35–44. 
                  https://doi.org/10.1016/j.fsigen.2015.11.008
                  
                 (2016)." href="https://interjectedfuture.com/articles/s41598-023-46151-7#ref-CR34" id="ref-link-section-d43177717e506">34</a></sup> is utilized for this purpose. Because of the considerable automation inherent with this program, it is highly efficient; here we provide a demonstration with a small reference database.</p><p>We have used the building of our institute as the test-bed for the investigation; the experiments were conducted in 14 offices, two meeting rooms and five laboratory areas which are accessed by corridors. Following ethical approval, a reference database was built from employees that gave written consent for their profiles to be included (n = 55 of 64 employees). This was used to compare findings from air and dust samples taken from all areas of the building. Laboratory areas were of particular interest, since it is here that risks of contamination of evidence samples must be carefully managed. Novel methods to collect, isolate, and analyze human DNA from the air and dust were developed for the study.</p></div></div></section><section data-title="Results"><div id="Sec2-section"><h2 id="Sec2">Results</h2><div id="Sec2-content"><h3 id="Sec3">Characterization of DNA profiles in air and dust samples</h3><p>A total of 40 air samples and 144 dust samples were collected and analyzed from 14 offices, two meeting rooms and five laboratory areas. Air samples displayed significantly lower DNA quantities (median = 0.1 ng) compared to dust samples (median = 2.3 ng), (Wilcoxon, p &lt; 2.2 × 10<sup>–16</sup>), (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://interjectedfuture.com/articles/s41598-023-46151-7#Fig1">1</a>). For detailed information on each sample, refer to Supplementary Data <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://interjectedfuture.com/articles/s41598-023-46151-7#MOESM1">1</a>.</p><div data-test="figure" data-container-section="figure" id="figure-1" data-title="Figure 1"><figure><figcaption><b id="Fig1" data-test="figure-caption-text">Figure 1</b></figcaption><div><div><a data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="https://interjectedfuture.com/articles/s41598-023-46151-7/figures/1" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41598-023-46151-7/MediaObjects/41598_2023_46151_Fig1_HTML.png?as=webp"/><img aria-describedby="Fig1" src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41598-023-46151-7/MediaObjects/41598_2023_46151_Fig1_HTML.png" alt="figure 1" loading="lazy" width="685" height="583"/></picture></a></div><p>Log<sub>10</sub> of DNA quantity (ng) recovered for air and dust samples.</p></div></figure></div><p>Among the air samples, 7.5% (N = 3) did not provide any DNA profile, 27.5% (N = 11) resulted in partial single-source profiles, whilst 65% (N = 26) produced DNA mixtures with varying numbers of alleles detected. Two-, three-, and four-person mixtures were observed in approximately equal proportions (20%-22.5%). The dust samples generally exhibited more complex mixtures than air samples, with 91% (N = 131) of the dust samples indicating four or more contributors. Four was the maximum number of contributors (NOC) considered in the analysis; the threshold was applied to the settings in the dnamatch2 software, which was used to compare resultant DNA profiles to reference samples of the participants of this study (see Methods): more could be considered but the analysis time increases exponentially. Only 3% (N = 5) of the dust samples resulted in two-person mixtures, whilst 5% (N = 7) resulted in three-person mixtures (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://interjectedfuture.com/articles/s41598-023-46151-7#Fig2">2</a>).</p><div data-test="figure" data-container-section="figure" id="figure-2" data-title="Figure 2"><figure><figcaption><b id="Fig2" data-test="figure-caption-text">Figure 2</b></figcaption><div><div><a data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="https://interjectedfuture.com/articles/s41598-023-46151-7/figures/2" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41598-023-46151-7/MediaObjects/41598_2023_46151_Fig2_HTML.png?as=webp"/><img aria-describedby="Fig2" src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41598-023-46151-7/MediaObjects/41598_2023_46151_Fig2_HTML.png" alt="figure 2" loading="lazy" width="685" height="592"/></picture></a></div><p>Distribution of the number of contributors (NOC) detected in air and dust samples.</p></div></figure></div><p>A reference database <span>\(R\)</span> was prepared of 55 out of 64 employees at our institute. To detect ‘matches’ with any given air and dust sample, a likelihood ratio (LR) was calculated (Eq. (<a data-track="click" data-track-label="link" data-track-action="equation anchor" href="https://interjectedfuture.com/articles/s41598-023-46151-7#Equ1">1</a>)) for each reference sample (<i>PoI</i>) in the database, where for a given evidence sample (<i>E</i>), the following hypotheses were compared:</p><div id="Equ1"><p><span>$$LR=\frac{\mathrm{Pr}\left(E|{H}_{1}\right)}{\mathrm{Pr}\left(E|{H}_{2}\right)}$$</span></p><p>
                    (1)
                </p></div><p>Likelihood ratios (Eq. (<a data-track="click" data-track-label="link" data-track-action="equation anchor" href="https://interjectedfuture.com/articles/s41598-023-46151-7#Equ1">1</a>)) were calculated by comparison of two alternative propositions (<i>H</i><sub><i>1</i></sub>) and (<i>H</i><sub><i>2</i></sub>). Under <span>\({H}_{1}\)</span> the reference profile of a given person of interest (<i>PoI)</i> is conditioned, which is an individual from the reference database <span>\(R\)</span> (see Methods for further details) and there are <i>L-1</i> unrelated unknown contributors. Under<i> H</i><sub><i>2</i></sub> the <i>PoI</i> is replaced by an unknown contributor. The <i>LR</i> is summarized: <span>\({H}_{1}\)</span>: “<span>\(PoI\)</span> and <span>\(L-1\)</span> unrelated unknowns are contributors” versus <span>\({H}_{2}\)</span>: “<span>\(L\)</span> unrelated unknowns are contributors”.</p><p>The expected number of false positives when searching a database of random individuals, based on a “positive” criterion <span>\(LR\ge x\)</span> where <span>\(x\)</span> is a specified threshold, can be derived from the Turing expectation <span>\(p\left(LR &gt; x|Hd\right)&lt;\frac{1}{x}\)</span><sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Gill, P., Bleka, Ø. &amp; Egeland, T. Does an English appeal court ruling increase the risks of miscarriages of justice when complex DNA profiles are searched against the national DNA database?. Forensic Sci. Int. Genet. 13, 167–175. 
                  https://doi.org/10.1016/j.fsigen.2014.07.015
                  
                 (2014)." href="#ref-CR35" id="ref-link-section-d43177717e958">35</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Taylor, D., Curran, J. M. &amp; Buckleton, J. Importance sampling allows H(d) true tests of highly discriminating DNA profiles. Forensic Sci. Int. Genet. 27, 74–81. 
                  https://doi.org/10.1016/j.fsigen.2016.12.004
                  
                 (2017)." href="#ref-CR36" id="ref-link-section-d43177717e958_1">36</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 37" title="Good, I. J. Probability and the Weighing of Evidence. Vol. 26 (C. Griffin London, 1950)." href="https://interjectedfuture.com/articles/s41598-023-46151-7#ref-CR37" id="ref-link-section-d43177717e961">37</a></sup>. If a database is of size <i>N</i> we never expect more than <span>\(N/x\)</span> adventitious matches. It follows that if <span>\(N\)</span> can be reduced by searching a subset of a database, e.g., by searching for individuals of a certain age-range or location, this will usefully reduce the chance of non-contributor matches. This does not mean that all positives are true positives, neither does it mean that all negatives are true negatives. All database search strategies are investigative rather than evaluative. The aim is to provide the authorities with a list of potential suspects that may be further investigated. If the profile is poorly represented then a <i>PoI</i> may be present, but undetected because the LR is below threshold. Where LRs are above threshold, the <i>PoI</i> may be further investigated, but the aim is to minimize the number of adventitious matches rather than to avoid them completely. </p><p>For this study we used an arbitrary LR = 100 as a reasonable threshold to distinguish a positive (a match) from a negative result. If <span>\(N\)</span>=1mill, then the LR threshold <span>\(x\)</span> would need to be equivalent in size, to avoid too many adventitious matches. The analysis returns a list of candidates for further investigation. The resulting LR values ranged from 1 to 3.78 × 10<sup>32</sup>, for 55 <i>PoIs</i> from each of the evidence samples that gave a DNA profile.</p><p>Comparing the log<sub>10</sub>LR distributions of those samples that provided positive matches (LR ≥ 100), from the different types of sampling locations (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://interjectedfuture.com/articles/s41598-023-46151-7#Fig3">3</a>), showed that offices generally gave higher log<sub>10</sub>LR values than meeting rooms and laboratories, both for air and dust samples. Higher LRs are associated with higher quality of the DNA profiles obtained. For the air samples, the log<sub>10</sub>LR values were significantly different between offices and meeting rooms (Wilcoxon, <i>p</i> = 0.03). Only one air sample from laboratories gave LR &gt; 100 (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://interjectedfuture.com/articles/s41598-023-46151-7#Fig3">3</a>). Similarly, for dust samples, log<sub>10</sub>LR values from offices were higher than for meeting rooms, at a level approaching statistical significance (Wilcoxon, <i>p</i> = 0.06) and significant for laboratories (Wilcoxon, <i>p</i> = 9 × 10<sup>–4</sup>). No significant differences were observed between meeting rooms and laboratories (Wilcoxon, <i>p</i> = 0.5). The log<sub>10</sub>LR values for each sample are given in Supplementary Data <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://interjectedfuture.com/articles/s41598-023-46151-7#MOESM2">2</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://interjectedfuture.com/articles/s41598-023-46151-7#MOESM3">3</a>.</p><div data-test="figure" data-container-section="figure" id="figure-3" data-title="Figure 3"><figure><figcaption><b id="Fig3" data-test="figure-caption-text">Figure 3</b></figcaption><div><div><a data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="https://interjectedfuture.com/articles/s41598-023-46151-7/figures/3" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41598-023-46151-7/MediaObjects/41598_2023_46151_Fig3_HTML.png?as=webp"/><img aria-describedby="Fig3" src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41598-023-46151-7/MediaObjects/41598_2023_46151_Fig3_HTML.png" alt="figure 3" loading="lazy" width="685" height="580"/></picture></a></div><p>Log<sub>10</sub>LR values (LR ≥ 100) for air and dust samples collected from three different locations (laboratories, meeting rooms and offices).</p></div></figure></div></div></div></section><section data-title="Success rate of identifying known occupants and non-occupants"><div id="Sec4-section"><h2 id="Sec4">Success rate of identifying known occupants and non-occupants</h2><div id="Sec4-content"><p>A given evidence sample may contain DNA from both known and unknown contributors, else the result may be negative. All office occupants, except two (from offices R4 and R12), participated in the study and had their reference profile included in the reference database used for comparison in the dnamatch2 software<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 38" title="Bleka, Ø., Bouzga, M., Fonneløp, A. &amp; Gill, P. dnamatch2: An open source software to carry out large scale database searches of mixtures using qualitative and quantitative models. Forensic Sci. Int. Genet. Suppl. Ser. 6, e404–e406. 
                  https://doi.org/10.1016/j.fsigss.2017.09.160
                  
                 (2017)." href="https://interjectedfuture.com/articles/s41598-023-46151-7#ref-CR38" id="ref-link-section-d43177717e1129">38</a></sup>. Success rates were defined as the proportion of samples where one or more known occupants (<i>φ</i><sub><i>1</i></sub>) and non-occupants (<i>φ</i><sub><i>2</i></sub>) were recovered, regardless of whether each individual was detected only in one sample (unique observation) or more; calculated using Eq. (<a data-track="click" data-track-label="link" data-track-action="equation anchor" href="https://interjectedfuture.com/articles/s41598-023-46151-7#Equ4">4</a>) and Eq. (<a data-track="click" data-track-label="link" data-track-action="equation anchor" href="https://interjectedfuture.com/articles/s41598-023-46151-7#Equ5">5</a>) in “<a data-track="click" data-track-label="link" data-track-action="section anchor" href="https://interjectedfuture.com/articles/s41598-023-46151-7#Sec11">Methods</a>”. These parameters were calculated for dust and air samples collected from the three different locations: offices, meeting rooms and laboratories (divided into washing room, DNA casework laboratories and investigation room). The success rate is a convenient way to compare the results from samples collected from different environments. The results from all experiments are summarized in Table <a data-track="click" data-track-label="link" data-track-action="table anchor" href="https://interjectedfuture.com/articles/s41598-023-46151-7#Tab1">1</a>.</p><div data-test="inline-table" data-container-section="table" id="table-1"><figure><figcaption><b id="Tab1" data-test="table-caption">Table 1 Success rates for samples collected from the three different locations (offices, meeting rooms and laboratories).</b></figcaption></figure></div><h3 id="Sec5">DNA recovery from dust</h3><p>A total of 96 dust samples were collected across 14 different offices (1–7 occupants). All samples gave DNA results, however three were negative as they did not give any match with a known <i>PoI</i> with LR &gt; 100, (Table <a data-track="click" data-track-label="link" data-track-action="table anchor" href="https://interjectedfuture.com/articles/s41598-023-46151-7#Tab2">2</a>). Positive results were obtained from 93 out of 96 dust samples for one or more office occupants, resulting in a success rate of <i>φ</i><sub><i>1</i></sub> = 0.97. Only 22 of the same samples matched with a known non-occupant, giving a non-occupant success rate of <i>φ</i><sub><i>2</i></sub> = 22/96 = 0.23.</p><div data-test="inline-table" data-container-section="table" id="table-2"><figure><figcaption><b id="Tab2" data-test="table-caption">Table 2 Data from dust samples collected from offices.</b></figcaption></figure></div><p>The results for meeting rooms and laboratories are detailed in Supplementary Tables <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://interjectedfuture.com/articles/s41598-023-46151-7#MOESM5">1</a>, <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://interjectedfuture.com/articles/s41598-023-46151-7#MOESM5">2</a>. For meeting rooms, the success rate for occupants was <i>φ</i><sub><i>1</i></sub> = 0.29, significantly lower than for offices <i>φ</i><sub><i>1</i></sub> = 0.97, (Fisher exact two tail test,<i> p</i> = 1.7 × 10<sup>−12</sup>). Meeting rooms are used by all 64 members of staff (albeit more sporadically than offices), hence, assuming each has equal probability of detection, the detection rate per member of staff is much lower than for offices (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://interjectedfuture.com/articles/s41598-023-46151-7#MOESM5">1</a>).</p><p>Laboratories have restricted access to individuals who wear protective clothing in order to prevent cross-contamination of live casework material, so there is an expectation that recovery of DNA in dust and air should be lower than for other areas. In the washing room an individual (participant P1) was identified in seven out of eight dust samples collected from different surfaces; the success rate of detecting one or more occupants was surprisingly high <i>φ</i><sub><i>1</i></sub> = 0.88 but was influenced by presence of just one high-shedder-status individual found in multiple samples. However, two participants that had no regular access to the washing room were also detected; the success rate of detecting non-occupants in the washing room was calculated as <i>φ</i><sub><i>2</i></sub> = 0.25 (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://interjectedfuture.com/articles/s41598-023-46151-7#MOESM5">2</a>).</p><p>A total of 12 dust samples were collected from DNA casework laboratories, 11 of these provided DNA results. The occupant success rate was <i>φ</i><sub><i>1</i></sub> = 0.58, however, this was also driven by participant P1 who was identified in five out of 12 samples collected. Two individuals who did not regularly work in the DNA laboratories, but declared that they had entered the rooms at least once, were also identified, giving a success rate for non-occupants of <i>φ</i><sub><i>2</i></sub> = 0.17 (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://interjectedfuture.com/articles/s41598-023-46151-7#MOESM5">2</a>).</p><p>Two out of four samples collected from the investigation room provided a positive result for one or more occupants <i>φ</i><sub><i>1</i></sub> = 0.5. No known non-occupants were detected (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://interjectedfuture.com/articles/s41598-023-46151-7#MOESM5">2</a>).</p><h3 id="Sec6">DNA recovery from the air</h3><p>The analysis of samples from the air required specialized equipment (see Methods) and collections took several hours, hence compared with analysis of dust samples, it was not as convenient. Consequently, the number of samples was restricted by the availability of sampling devices and data collections were not as extensive. </p><p>For offices, the sampling regime was (a) 0 h after occupancy, where collection was carried out immediately following vacation of the room and (b) 16 h after occupancy. Two samples were collected at three days, and one sample was taken at 15 days after occupancy; these three samples were however excluded from the following analysis as the sample size was not representative of the two time-points. There was to our knowledge no movement into the room during sampling and doors were kept closed. All 17 samples collected from offices immediately after occupancy provided a DNA result. However, five samples did not provide any match with an LR &gt; 100 (Table <a data-track="click" data-track-label="link" data-track-action="table anchor" href="https://interjectedfuture.com/articles/s41598-023-46151-7#Tab3">3</a>).</p><div data-test="inline-table" data-container-section="table" id="table-3"><figure><figcaption><b id="Tab3" data-test="table-caption">Table 3 Data from air samples collected from offices at 0 h after occupancy.</b></figcaption></figure></div><p>In office samples, occupant success rate for the 0 h samples (<i>φ</i><sub><i>1</i></sub> = 0.71) was comparable to dust, while the non-occupant success rate was much lower (<i>φ</i><sub><i>2</i></sub> = 0.06), (Table <a data-track="click" data-track-label="link" data-track-action="table anchor" href="https://interjectedfuture.com/articles/s41598-023-46151-7#Tab3">3</a>). For 16 h samples, occupants’ <i>φ</i><sub><i>1</i></sub> = 0.25 was lower than for 0 h samples (Fisher exact test p = 0.08) and was exceeded by non-occupants’ <i>φ</i><sub><i>2</i></sub> = 0.50 (Fisher exact test, <i>p</i> = 0.02), (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://interjectedfuture.com/articles/s41598-023-46151-7#MOESM5">3</a>). In addition, LR values were found to be lower for samples collected 16 h after occupancy in comparison to samples collected immediately after occupancy, and this is a reflection of reduced quality of DNA recovery (Supplementary Data <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://interjectedfuture.com/articles/s41598-023-46151-7#MOESM2">2</a>).</p><p> Three individuals were identified from air samples collected from meeting rooms, giving a success rate of <i>φ</i><sub><i>1</i></sub> = 0.5. Participant P1 was one of the three individuals identified (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://interjectedfuture.com/articles/s41598-023-46151-7#MOESM5">4</a>).</p><p>In laboratories, only one positive recovery was recorded. The air sample was collected from the washing room L1, it consisted of very low quality DNA, with an LR = 100 (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://interjectedfuture.com/articles/s41598-023-46151-7#MOESM5">5</a>).</p></div></div></section><section data-title="Probability of detecting an occupant in a sample relative to the number of occupants in a location"><div id="Sec7-section"><h2 id="Sec7">Probability of detecting an occupant in a sample relative to the number of occupants in a location</h2><div id="Sec7-content"><p>We estimated the probability of detecting DNA of an occupant in a location with <i>K</i> number of occupants. For this we used logistic regression based on the observed number of successes per occupant, per office, for dust samples presented in Table <a data-track="click" data-track-label="link" data-track-action="table anchor" href="https://interjectedfuture.com/articles/s41598-023-46151-7#Tab2">2</a>. The logistic regression (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://interjectedfuture.com/articles/s41598-023-46151-7#Fig4">4</a>) assumes <span>\({X}_{k,z}\)</span>~Binom(<span>\({S}_{z},{p}_{K})\)</span>, where <span>\({X}_{k,z}\)</span> is the number of positive samples where an occupant <span>\(k\)</span> in a location <i>z</i> with <i>K</i> occupants was detected such that the estimated probability becomes:</p><div id="Equ2"><p><span>$${p}_{K}=\frac{1}{1+{e}^{-(b0+b1K)}}$$</span></p><p>
                    (2)
                </p></div><p>with estimated coefficients as <span>\({b}_{0}=1.06\)</span> (intercept) and  <span>\({b}_{1}=-0.27\)</span> (slope).</p><div data-test="figure" data-container-section="figure" id="figure-4" data-title="Figure 4"><figure><figcaption><b id="Fig4" data-test="figure-caption-text">Figure 4</b></figcaption><div><div><a data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="https://interjectedfuture.com/articles/s41598-023-46151-7/figures/4" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41598-023-46151-7/MediaObjects/41598_2023_46151_Fig4_HTML.png?as=webp"/><img aria-describedby="Fig4" src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41598-023-46151-7/MediaObjects/41598_2023_46151_Fig4_HTML.png" alt="figure 4" loading="lazy" width="685" height="684"/></picture></a></div><p>Logistic regression of dust samples from offices, comparing number of occupants in a given office vs probability of detection (success) of any given occupant in any given sample analyzed from that office. The blue line represents the logistic regression model fitted to the data and the grey area is the 95% confidence interval. Hosmer Lemeshow goodness of fit test gave p-value = 0.43, indicating a good fit to the data.</p></div></figure></div><p>As displayed in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://interjectedfuture.com/articles/s41598-023-46151-7#Fig4">4</a>, there is an inverse relationship between the number of occupants (<i>K</i>) and the probability of detecting DNA from a given occupant <i>k</i><sub><i>z</i></sub>; as <i>K</i> increases the probability trends towards zero.</p><p>There were insufficient data to carry out a meaningful logistic regression analysis of dust samples from meeting rooms and laboratories.</p></div></div></section><section data-title="Estimating the success rate of identifying all occupants of a location"><div id="Sec8-section"><h2 id="Sec8">Estimating the success rate of identifying all occupants of a location</h2><div id="Sec8-content"><p>We consider here the probability <span>\({\upphi }_{\mathrm{z}}\)</span> of observing all <i>K</i><sub><i>z</i></sub> occupants at least once, in a given location (<i>z</i>), amongst <i>S</i><sub><i>z</i></sub> samples that were collected. The formula is given below (Eq. (<a data-track="click" data-track-label="link" data-track-action="equation anchor" href="https://interjectedfuture.com/articles/s41598-023-46151-7#Equ3">3</a>)):</p><div id="Equ3"><p><span>$${\upphi }_{z}= \prod\nolimits_{{k = 1..K_{z} }} {1 - \left( {1 - p_{{k,z}} } \right)^{{S_{z} }} }$$</span></p><p>
                    (3)
                </p></div><p>Here <span>\({\mathrm{p}}_{k,z}\)</span> is the probability of detection of occupant <i>k</i> out of <i>K</i><sub><i>z</i></sub> occupants in a given location<i> z</i>, and is inferred using data from Table <a data-track="click" data-track-label="link" data-track-action="table anchor" href="https://interjectedfuture.com/articles/s41598-023-46151-7#Tab2">2</a>. With a frequentist approach, as an example, occupant <i>k</i> = P4 from office <i>z</i> = R1, there are <span>\({X}_{k,z}=5\)</span> out of <i>S</i><sub><i>z</i></sub> = 12 samples, which gives <span>\({p}_{P4,R1}=\frac{{X}_{\mathrm{P}4,z}}{Sz}=5/12=0.42\)</span></p><p>Because of limited number of observations, we continued with a Bayesian approach, which requires the definition of a prior distribution for the <span>\({\mathrm{p}}_{k,z}\)</span> probabilities; a prior beta distribution <span>\(B\)</span>(<span>\(\alpha\)</span>=3, <span>\(\beta\)</span>=3) was used for all probabilities. From this we generated posterior samples of <span>\({\mathrm{p}}_{k,z}\)</span> for each occupant <span>\(k\)</span> in location <span>\(z\)</span>, to obtain the posterior distribution of <span>\({\upphi }_{\mathrm{z}}\)</span> (for given samples size <span>\({S}_{z}).\)</span>  The assumed prior can be justified from the logistic regression model in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://interjectedfuture.com/articles/s41598-023-46151-7#Fig4">4</a>, which indicates the prior belief that the probability of success is a mean of 0.5 for the range of 1–7 occupants. This prior can be interpreted as being moderately informative but allows for a wide range of possible values to take account of shedder and occupancy status. Continuing with the example above, we obtain the posterior expectation as:</p><div id="Equa"><p><span>$$E({p}_{k,z})=\frac{{X}_{k,z}+\alpha }{{S}_{z}+\alpha +\beta }=\frac{5+3}{12+3+3}=0.44$$</span></p></div><p>Hence, we see that the prior has large influence on the expectation when the number of samples <span>\({S}_{z}\)</span> is low, and only small influence when the number of samples is high.</p><p>From the perspective of the investigator, it is important to estimate the number of dust samples (<i>S</i><sub><i>z</i></sub>) that should be retrieved from a crime scene in order to have a reasonable probability to detect each of the <i>K</i> occupants of a location. To obtain different measures of central tendency to express the results, three different estimators of <span>\({\upphi }_{\mathrm{z}}\)</span> were employed across a range of values for <i>S</i><sub><i>z</i></sub>: posterior expectation (EXP), posterior mode (MODE), and posterior medians (MEDIAN) were calculated based on the Bayesian approach.</p><p>The horizontal dashed line in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://interjectedfuture.com/articles/s41598-023-46151-7#Fig5">5</a> represents the 95% probability of obtaining all <i>K</i> occupants from <i>S</i><sub><i>z</i></sub> sample trials. Using the median estimate for example, this limit is achieved at <i>S</i><sub><i>z</i></sub> = 20. The wide coverage intervals indicates that the uncertainty of this estimate is very high. Other point estimates obtained a smaller number of trials. A higher confidence in the result would require a considerable higher number of trials.</p><div data-test="figure" data-container-section="figure" id="figure-5" data-title="Figure 5"><figure><figcaption><b id="Fig5" data-test="figure-caption-text">Figure 5</b></figcaption><div><div><a data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="https://interjectedfuture.com/articles/s41598-023-46151-7/figures/5" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41598-023-46151-7/MediaObjects/41598_2023_46151_Fig5_HTML.png?as=webp"/><img aria-describedby="Fig5" src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41598-023-46151-7/MediaObjects/41598_2023_46151_Fig5_HTML.png" alt="figure 5" loading="lazy" width="685" height="486"/></picture></a></div><p>Office R10, <i>K</i> = 7: Plot of probability of detecting all <i>k</i><sub><i>z</i></sub><i> (</i><span>\({\upphi }_{\mathrm{z}}\)</span>) at least once out of <i>K</i> occupants, in a set of samples, varying size (<i>S</i><sub><i>z</i></sub>) using three different estimators.</p></div></figure></div><p>The results for all offices are compiled in Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://interjectedfuture.com/articles/s41598-023-46151-7#MOESM5">6</a> and Supplementary Figs. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://interjectedfuture.com/articles/s41598-023-46151-7#MOESM5">1</a>–<a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://interjectedfuture.com/articles/s41598-023-46151-7#MOESM5">13</a>. The resulting figures show that the probability of identifying all occupants at least once is dependent on the total number of occupants in the location.</p></div></div></section><section data-title="Representing results by a match network"><div id="Sec9-section"><h2 id="Sec9">Representing results by a match network</h2><div id="Sec9-content"><p>Results of pairwise comparisons of dust samples are facilitated by the dnamatch2 program<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 38" title="Bleka, Ø., Bouzga, M., Fonneløp, A. &amp; Gill, P. dnamatch2: An open source software to carry out large scale database searches of mixtures using qualitative and quantitative models. Forensic Sci. Int. Genet. Suppl. Ser. 6, e404–e406. 
                  https://doi.org/10.1016/j.fsigss.2017.09.160
                  
                 (2017)." href="https://interjectedfuture.com/articles/s41598-023-46151-7#ref-CR38" id="ref-link-section-d43177717e5816">38</a></sup>. The results are represented by a match network (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://interjectedfuture.com/articles/s41598-023-46151-7#Fig6">6</a>); the corresponding log<sub>10</sub>LR values for each sample represented in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://interjectedfuture.com/articles/s41598-023-46151-7#Fig6">6</a> are reported in Supplementary Data <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://interjectedfuture.com/articles/s41598-023-46151-7#MOESM4">4</a>. The network is useful for investigative purposes because it can show which occupants contributed to which samples, thereby giving a valuable overview of potential movements of individuals. For example, it is notable that individual P1 (identified as high shedder from a previous internal study) appears in multiple locations, whereas others, such as P34 are observed once only. Participant P1 was detected from two air samples and from 11 of 12 dust samples recovered from the participant’s own office (R2). This individual was also identified in one air and one dust sample collected from a meeting room (M1), in seven dust samples collected from the washing room (L1) and in five samples recovered from DNA casework laboratories (L2-L4). In addition, P1 was detected as a non-occupant in five samples collected from different offices (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://interjectedfuture.com/articles/s41598-023-46151-7#Fig6">6</a>). Three of these were from one office that the participant had been occupying until four months previous to sample collection.</p><div data-test="figure" data-container-section="figure" id="figure-6" data-title="Figure 6"><figure><figcaption><b id="Fig6" data-test="figure-caption-text">Figure 6</b></figcaption><div><div><a data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="https://interjectedfuture.com/articles/s41598-023-46151-7/figures/6" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41598-023-46151-7/MediaObjects/41598_2023_46151_Fig6_HTML.png?as=webp"/><img aria-describedby="Fig6" src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41598-023-46151-7/MediaObjects/41598_2023_46151_Fig6_HTML.png" alt="figure 6" loading="lazy" width="685" height="685"/></picture></a></div><p>Match network from dust samples. Location and sample name: orange R<sub>n</sub> = office, blue M<sub>n</sub> = meeting room, red L<sub>n</sub> = laboratory; the value after the decimal indicates the sample name from a specific location. The green nodes report the participants identified (P<sub>n</sub>). The network includes only matches that provided an LR &gt; 100. The distance between nodes is inversely proportional to log<sub>10</sub>LR, the thickness of a node is directly proportional to log<sub>10</sub>LR: i.e. the shorter and thicker the edge, the higher log<sub>10</sub>LR value. The corresponding log<sub>10</sub>LR values for each sample are displayed in Supplementary Data 4.</p></div></figure></div><p>Three other dust samples matched with non-occupants who declared that they had been working in an office in the past, ranging from one month to up to two years prior to sampling. Two of these samples were collected from the top of mirror frames, a hollow surface not typically cleaned, hence a perfect undisturbed place for indoor dust to accumulate and persist for long period of time.</p><p>DNA from another participant (P25), who had also been identified as high shedder from a previous study, was detected in DNA mixtures obtained from two air samples collected from this participant’s own office (R7) with log<sub>10</sub>LR values of 12 and 10, respectively; at the time of sampling, the individual had been absent from the office for 20–30 days. </p><p>Overall, in offices with multiple occupants, it was observed that certain individuals were detected more frequently than others.</p></div></div></section><section data-title="Discussion"><div id="Sec10-section"><h2 id="Sec10">Discussion</h2><div id="Sec10-content"><p>Previous research has shown that human DNA can be successfully collected and isolated from dust samples<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 19" title="Toothman, M. H. et al. Characterization of human DNA in environmental samples. Forensic Sci. Int. 178, 7–15. 
                  https://doi.org/10.1016/j.forsciint.2008.01.016
                  
                 (2008)." href="https://interjectedfuture.com/articles/s41598-023-46151-7#ref-CR19" id="ref-link-section-d43177717e5889">19</a></sup>. The current study, utilising improved DNA technology, reaffirms these findings. However, dust samples consistently yielded complex DNA mixtures, while air samples produced both partial single profiles and DNA mixtures. Dust is a reservoir of human skin cells and biological material, which can originate from current and past occupants of a specific location. DNA accumulated in dust can persist for a long time, from a few months up to a few years, depending on environmental factors such as light, heat, moisture, and cleaning agents, which may cause degradation<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 19" title="Toothman, M. H. et al. Characterization of human DNA in environmental samples. Forensic Sci. Int. 178, 7–15. 
                  https://doi.org/10.1016/j.forsciint.2008.01.016
                  
                 (2008)." href="https://interjectedfuture.com/articles/s41598-023-46151-7#ref-CR19" id="ref-link-section-d43177717e5893">19</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Abdel Hady, R. H., Thabet, H. Z., Ebrahem, N. E. &amp; Yassa, H. A. Thermal effects on DNA degradation in blood and seminal stains: Forensic view. Acad. Forensic Pathol. 11, 7–23. 
                  https://doi.org/10.1177/1925362121998547
                  
                 (2021)." href="#ref-CR39" id="ref-link-section-d43177717e5896">39</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Hall, A., Sims, L. M. &amp; Ballantyne, J. Assessment of DNA damage induced by terrestrial UV irradiation of dried bloodstains: Forensic implications. Forensic Sci. Int. Genet. 8, 24–32. 
                  https://doi.org/10.1016/j.fsigen.2013.06.010
                  
                 (2014)." href="#ref-CR40" id="ref-link-section-d43177717e5896_1">40</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 41" title="Asari, M. et al. Assessment of DNA degradation of buccal cells under humid conditions and DNA repair by DOP-PCR using locked nucleic acids. Leg. Med. (Tokyo) 35, 29–33. 
                  https://doi.org/10.1016/j.legalmed.2018.09.005
                  
                 (2018)." href="https://interjectedfuture.com/articles/s41598-023-46151-7#ref-CR41" id="ref-link-section-d43177717e5899">41</a></sup>. On the other hand, DNA present in the air is more likely to represent individuals who recently occupied a room. The distribution of likelihood ratio (LR) values for different references varied among air and dust samples collected from different locations. Offices yielded the highest LR values, followed by meeting rooms and laboratories, influenced by the usage patterns. Individuals spending more time in an area, such as offices, are likely to disperse higher amounts of cells and DNA compared to those spending less time, as in meeting rooms. Areas where individuals spend more time during the day (i.e. offices) tend to have higher DNA transfer rates. Reither et al.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 27" title="Reither, J. B. et al. Investigation into the prevalence of background DNA on flooring within houses and its transfer to a contacting surface. Forensic Sci. Int. 318, 110563. 
                  https://doi.org/10.1016/j.forsciint.2020.110563
                  
                 (2021)." href="https://interjectedfuture.com/articles/s41598-023-46151-7#ref-CR27" id="ref-link-section-d43177717e5903">27</a></sup> found that the total amount of DNA recovered from household floors increased as residents spent more time at home. Puliatti et al.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 17" title="Puliatti, L., Handt, O. &amp; Taylor, D. The level of DNA an individual transfers to untouched items in their immediate surroundings. Forensic Sci. Int. Genet. 54, 102561. 
                  https://doi.org/10.1016/j.fsigen.2021.102561
                  
                 (2021)." href="https://interjectedfuture.com/articles/s41598-023-46151-7#ref-CR17" id="ref-link-section-d43177717e5907">17</a></sup> found that the amount of DNA transferred by individuals to non-touched surfaces near their workstation was time dependent. Cells and DNA can indeed be shed in the air directly by an individual from speaking, coughing or sneezing<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 12" title="Finnebraaten, M., Granér, T. &amp; Hoff-Olsen, P. May a speaking individual contaminate the routine DNA laboratory?. Forensic Sci. Int. Genet. Suppl. Ser. 1, 421–422. 
                  https://doi.org/10.1016/j.fsigss.2007.10.030
                  
                 (2008)." href="https://interjectedfuture.com/articles/s41598-023-46151-7#ref-CR12" id="ref-link-section-d43177717e5911">12</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 13" title="Port, N. J., Bowyer, V. L., Graham, E. A., Batuwangala, M. S. &amp; Rutty, G. N. How long does it take a static speaking individual to contaminate the immediate environment?. Forensic Sci. Med. Pathol. 2, 157–163. 
                  https://doi.org/10.1007/s12024-006-0004-z
                  
                 (2006)." href="https://interjectedfuture.com/articles/s41598-023-46151-7#ref-CR13" id="ref-link-section-d43177717e5914">13</a></sup>, but can also be transferred indirectly from skin, clothes and personal items<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 6" title="Mackintosh, C. A., Lidwell, O. M., Towers, A. G. &amp; Marples, R. R. The dimensions of skin fragments dispersed into the air during activity. J. Hyg. (Lond.) 81, 471–479. 
                  https://doi.org/10.1017/s0022172400025341
                  
                 (1978)." href="https://interjectedfuture.com/articles/s41598-023-46151-7#ref-CR6" id="ref-link-section-d43177717e5919">6</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 9" title="Burrill, J., Rammenou, E., Alawar, F., Daniel, B. &amp; Frascione, N. Corneocyte lysis and fragmented DNA considerations for the cellular component of forensic touch DNA. Forensic Sci. Int. Genet. 51, 102428. 
                  https://doi.org/10.1016/j.fsigen.2020.102428
                  
                 (2021)." href="https://interjectedfuture.com/articles/s41598-023-46151-7#ref-CR9" id="ref-link-section-d43177717e5922">9</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 15" title="Thornbury, D., Goray, M. &amp; van Oorschot, R. A. H. Indirect DNA transfer without contact from dried biological materials on various surfaces. Forensic Sci. Int. Genet. 51, 102457. 
                  https://doi.org/10.1016/j.fsigen.2020.102457
                  
                 (2021)." href="https://interjectedfuture.com/articles/s41598-023-46151-7#ref-CR15" id="ref-link-section-d43177717e5925">15</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 16" title="Thornbury, D., Goray, M. &amp; van Oorschot, R. A. H. Transfer of DNA without contact from used clothing, pillowcases and towels by shaking agitation. Sci. Justice 61, 797–805. 
                  https://doi.org/10.1016/j.scijus.2021.10.005
                  
                 (2021)." href="https://interjectedfuture.com/articles/s41598-023-46151-7#ref-CR16" id="ref-link-section-d43177717e5928">16</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 42" title="Goray, M. &amp; van Oorschot, R. A. The complexities of DNA transfer during a social setting. Leg. Med. (Tokyo) 17, 82–91. 
                  https://doi.org/10.1016/j.legalmed.2014.10.003
                  
                 (2015)." href="https://interjectedfuture.com/articles/s41598-023-46151-7#ref-CR42" id="ref-link-section-d43177717e5931">42</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 43" title="Szkuta, B. et al. Assessment of the transfer, persistence, prevalence and recovery of DNA traces from clothing: An inter-laboratory study on worn upper garments. Forensic Sci. Int. Genet. 42, 56–68. 
                  https://doi.org/10.1016/j.fsigen.2019.06.011
                  
                 (2019)." href="https://interjectedfuture.com/articles/s41598-023-46151-7#ref-CR43" id="ref-link-section-d43177717e5934">43</a></sup>. Cleaning habits also affect levels of DNA found on surfaces and in the air. Taylor et al.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 44" title="Taylor, D., Abarno, D., Rowe, E. &amp; Rask-Nielsen, L. Observations of DNA transfer within an operational Forensic Biology Laboratory. Forensic Sci. Int. Genet. 23, 33–49. 
                  https://doi.org/10.1016/j.fsigen.2016.02.011
                  
                 (2016)." href="https://interjectedfuture.com/articles/s41598-023-46151-7#ref-CR44" id="ref-link-section-d43177717e5938">44</a></sup> showed that sensitive areas, such as laboratories, had fewer inclusionary LR values compared to non-sensitive areas such as offices. In this study, the cleaning regime in sensitive areas helped to reduce detection of employees’ DNA; however some high LR values matching laboratory’s staff members were still found. We showed that detectable levels of human DNA are present in the dust analyzed from isolated areas of laboratories such as door ledges, which if disturbed, could be a potential source of contamination of casework material.</p><p>Two air samples collected from laboratories (washing room and DNA extraction laboratory), resulted in DNA mixtures with alleles at all 23 loci, but when analyzed using dnamatch2, did not provide matches with any of the known participants i.e. these profiles were of unknown origin. Remaining air samples collected from laboratories resulted in DNA profiles of poor quality (i.e. few or no alleles detected).</p><p>Success rates varied between the three environments. Offices had higher success rates compared to meeting rooms and laboratories. Non-occupants’ success rates were lower in dust samples from offices and laboratories, while higher rates were found for air samples collected 16 h after occupancy. The frequent air changes in the locations selected for the experiments of this study contributed to declining DNA recovery over time and may have limited detection to recent occupants. Ventilation can also transfer dust containing non-occupants DNA between rooms<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 19" title="Toothman, M. H. et al. Characterization of human DNA in environmental samples. Forensic Sci. Int. 178, 7–15. 
                  https://doi.org/10.1016/j.forsciint.2008.01.016
                  
                 (2008)." href="https://interjectedfuture.com/articles/s41598-023-46151-7#ref-CR19" id="ref-link-section-d43177717e5948">19</a></sup>. Opening doors can introduce external DNA from a corridor into a room; non-occupants DNA on surfaces may also disperse into the air through normal activity or mediated by investigator involvement. Moreover, the different air exchange rates in different rooms may have influenced the levels of DNA that could be recovered from the air. Future studies should consider ventilation levels and their types. The experiments described in this study were performed in a “clean” office environment for practical reasons: it was important to have the availability of a reference database from occupants of the premises. Further investigations into air and dust sampling should be performed in different indoor spaces (e.g. common households), since these are more likely to resemble a potential crime scene. DNA was found in restricted access areas of laboratories, suggesting the presence of individuals who were not regular occupants. This raises concerns in trace evidence examinations, as DNA can move between rooms in controlled environments. Undisturbed and inaccessible surfaces pose a lower contamination risk when collecting dust samples. However, DNA accumulates on undisturbed surfaces, not only from occupants but also from adjacent areas, despite occupants following strict cleaning protocols and wearing protective clothing. Disturbed dust becomes an aerosolized contamination risk, regardless of protection levels. Maintaining comprehensive staff elimination databases is vital, to include individuals working in offices and visitors, to detect the potential transfer of aerosol DNA within a facility<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 45" title="Lapointe, M., Rogic, A., Bourgoin, S., Jolicoeur, C. &amp; Séguin, D. Leading-edge forensic DNA analyses and the necessity of including crime scene investigators, police officers and technicians in a DNA elimination database. Forensic Sci. Int. Genet. 19, 50–55. 
                  https://doi.org/10.1016/j.fsigen.2015.06.002
                  
                 (2015)." href="https://interjectedfuture.com/articles/s41598-023-46151-7#ref-CR45" id="ref-link-section-d43177717e5952">45</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 46" title="Mercer, C., Taylor, D., Henry, J. &amp; Linacre, A. DNA accumulation and transfer within an operational forensic exhibit storeroom. Forensic Sci. Int. Genet. 62, 102799. 
                  https://doi.org/10.1016/j.fsigen.2022.102799
                  
                 (2023)." href="https://interjectedfuture.com/articles/s41598-023-46151-7#ref-CR46" id="ref-link-section-d43177717e5955">46</a></sup>. </p><p>The number of occupants in a location inversely affected the likelihood of detecting DNA from individual occupants. Additionally, the probability of identifying all occupants at least once depended on the total number of occupants in the location. To aid in determining the optimal number of dust samples needed to detect all inhabitants with a 95% probability, where sample size was taken into account, we developed a Bayesian statistical model. Our observations revealed that as the number of occupants increased, a greater number of samples was required to capture the majority of individuals. Although the model was solely tested on dust samples from offices, due to limited data for meeting rooms and laboratories, it serves as a suitable framework for future studies with more extensive datasets. Since occupants may traverse multiple locations (i.e. offices, meeting rooms and laboratories), a graphical model was used to map individuals to respective locations: this provides a demonstration of the utility of air/dust examination to aid criminal investigations where potential movement and past location of individuals is an important motivation. One individual (P1) could be detected in 32 air and dust samples from different offices, meeting rooms and laboratories. This individual was known from previous studies to be a high shedder. Shedder status may have an influence not only on the DNA deposited upon contact with items or surfaces<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="van Oorschot, R. A. &amp; Jones, M. K. DNA fingerprints from fingerprints. Nature 387, 767. 
                  https://doi.org/10.1038/42838
                  
                 (1997)." href="#ref-CR47" id="ref-link-section-d43177717e5962">47</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Lowe, A., Murray, C., Whitaker, J., Tully, G. &amp; Gill, P. The propensity of individuals to deposit DNA and secondary transfer of low level DNA from individuals to inert surfaces. Forensic Sci. Int. 129, 25–34. 
                  https://doi.org/10.1016/s0379-0738(02)00207-4
                  
                 (2002)." href="#ref-CR48" id="ref-link-section-d43177717e5962_1">48</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Phipps, M. &amp; Petricevic, S. The tendency of individuals to transfer DNA to handled items. Forensic Sci. Int. 168, 162–168. 
                  https://doi.org/10.1016/j.forsciint.2006.07.010
                  
                 (2007)." href="#ref-CR49" id="ref-link-section-d43177717e5962_2">49</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Fonneløp, A. E., Ramse, M., Egeland, T. &amp; Gill, P. The implications of shedder status and background DNA on direct and secondary transfer in an attack scenario. Forensic Sci. Int. Genet. 29, 48–60. 
                  https://doi.org/10.1016/j.fsigen.2017.03.019
                  
                 (2017)." href="#ref-CR50" id="ref-link-section-d43177717e5962_3">50</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Goray, M. &amp; van Oorschot, R. A. H. Shedder status: Exploring means of determination. Sci. Justice 61, 391–400. 
                  https://doi.org/10.1016/j.scijus.2021.03.004
                  
                 (2021)." href="#ref-CR51" id="ref-link-section-d43177717e5962_4">51</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Johannessen, H., Gill, P., Roseth, A. &amp; Fonneløp, A. E. Determination of shedder status: A comparison of two methods involving cell counting in fingerprints and the DNA analysis of handheld tubes. Forensic Sci. Int. Genet. 53, 102541. 
                  https://doi.org/10.1016/j.fsigen.2021.102541
                  
                 (2021)." href="#ref-CR52" id="ref-link-section-d43177717e5962_5">52</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 53" title="Kanokwongnuwut, P., Martin, B., Kirkbride, K. P. &amp; Linacre, A. Shedding light on shedders. Forensic Sci. Int. Genet. 36, 20–25. 
                  https://doi.org/10.1016/j.fsigen.2018.06.004
                  
                 (2018)." href="https://interjectedfuture.com/articles/s41598-023-46151-7#ref-CR53" id="ref-link-section-d43177717e5965">53</a></sup>, but also on the DNA shed into the environment. The concept of shedder status is related to an individual’s natural propensity to shed DNA into the surrounding environment and may be influenced by factors such as personal and hygiene habits, behaviors and activities performed<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 17" title="Puliatti, L., Handt, O. &amp; Taylor, D. The level of DNA an individual transfers to untouched items in their immediate surroundings. Forensic Sci. Int. Genet. 54, 102561. 
                  https://doi.org/10.1016/j.fsigen.2021.102561
                  
                 (2021)." href="https://interjectedfuture.com/articles/s41598-023-46151-7#ref-CR17" id="ref-link-section-d43177717e5969">17</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 54" title="Lacerenza, D. et al. A molecular exploration of human DNA/RNA co-extracted from the palmar surface of the hands and fingers. Forensic Sci. Int. Genet. 22, 44–53. 
                  https://doi.org/10.1016/j.fsigen.2016.01.012
                  
                 (2016)." href="https://interjectedfuture.com/articles/s41598-023-46151-7#ref-CR54" id="ref-link-section-d43177717e5972">54</a></sup>.</p><p>DNA present in dust samples can serve as an historical record of individuals who were present many years ago, as DNA degrades slowly in dry conditions. Touch DNA left on surfaces, such as fingerprints, can be easily removed, indicating the potential for aerosolization<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 55" title="Van Hoofstat, D. E., Deforce, D. L., Hubert De Pauw, I. P. &amp; Van den Eeckhout, E. G. DNA typing of fingerprints using capillary electrophoresis: Effect of dactyloscopic powders. Electrophoresis 20, 2870–2876. 
                  https://doi.org/10.1002/(sici)1522-2683(19991001)20:14
                  
                 (1999)." href="https://interjectedfuture.com/articles/s41598-023-46151-7#ref-CR55" id="ref-link-section-d43177717e5980">55</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 56" title="Castillo, J. A., Staton, S. J., Taylor, T. J., Herckes, P. &amp; Hayes, M. A. Exploring the feasibility of bioaerosol analysis as a novel fingerprinting technique. Anal. Bioanal. Chem. 403, 15–26. 
                  https://doi.org/10.1007/s00216-012-5725-0
                  
                 (2012)." href="https://interjectedfuture.com/articles/s41598-023-46151-7#ref-CR56" id="ref-link-section-d43177717e5983">56</a></sup>. Indoor dust undergoes constant re-aerosolization, allowing skin flakes and attached DNA to circulate in the air<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 57" title="Tovey, E. R., Kemp, A. S., Almqvist, C., Sharland, A. &amp; Marks, G. B. Do immune responses to inhaled skin flakes modulate the expression of allergic disease?. Clin. Exp. Allergy 37, 1199–1203. 
                  https://doi.org/10.1111/j.1365-2222.2007.02770.x
                  
                 (2007)." href="https://interjectedfuture.com/articles/s41598-023-46151-7#ref-CR57" id="ref-link-section-d43177717e5987">57</a></sup>. In a specific example, an individual (P25) who was absent from office R7 for 20–30 days appeared as a contributor in two samples collected from that office immediately after it was occupied by two colleagues. This individual, known to be a high shedder, may have had some of his DNA present on his desk or personal items, which could have been aerosolized by his colleagues during their regular activities.</p><p>The forensic laboratory has no a priori knowledge about the origin of DNA that may become evidence in a case. A contribution to a crime-sample may come from the result of a crime event itself or it may be present beforehand as environmental background DNA (from unknown persons) or as prevalent DNA (from known persons). To evaluate evidence, interpretation is carried out using the “hierarchy of propositions” framework<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 58" title="Evett, I. W., Gill, P. D., Jackson, G., Whitaker, J. &amp; Champod, C. Interpreting small quantities of DNA: The hierarchy of propositions and the use of Bayesian networks. J. Forensic Sci. 47, 520–530 (2002)." href="https://interjectedfuture.com/articles/s41598-023-46151-7#ref-CR58" id="ref-link-section-d43177717e5994">58</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 59" title="Cook, R., Evett, I. W., Jackson, G., Jones, P. J. &amp; Lambert, J. A. A model for case assessment and interpretation. Sci. Justice 38, 151–156. 
                  https://doi.org/10.1016/s1355-0306(98)72099-4
                  
                 (1998)." href="https://interjectedfuture.com/articles/s41598-023-46151-7#ref-CR59" id="ref-link-section-d43177717e5997">59</a></sup> which separates propositions of identity (sub-source) from the activity which led to the deposition of evidential material (activity level propositions). To evaluate DNA profiling evidence at the activity level, forensic scientists use a probabilistic framework that already takes account of risks of environmental background and prevalent DNA transfer, including secondary transfer, using Bayesian networks<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 60" title="Gill, P. et al. DNA commission of the International society for forensic genetics: Assessing the value of forensic biological evidence—Guidelines highlighting the importance of propositions. Part II: Evaluation of biological traces considering activity level propositions. Forensic Sci. Int. Genet. 44, 102186. 
                  https://doi.org/10.1016/j.fsigen.2019.102186
                  
                 (2020)." href="https://interjectedfuture.com/articles/s41598-023-46151-7#ref-CR60" id="ref-link-section-d43177717e6001">60</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 61" title="Biedermann, A. &amp; Taroni, F. Bayesian networks for evaluating forensic DNA profiling evidence: A review and guide to literature. Forensic Sci. Int. Genet. 6, 147–157. 
                  https://doi.org/10.1016/j.fsigen.2011.06.009
                  
                 (2012)." href="https://interjectedfuture.com/articles/s41598-023-46151-7#ref-CR61" id="ref-link-section-d43177717e6004">61</a></sup> and this minimises risks of potential misreporting<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 62" title="Making sense of Forensic Genetics, 
                  https://senseaboutscience.org/activities/making-sense-of-forensic-genetics/
                  
                 (2017)." href="https://interjectedfuture.com/articles/s41598-023-46151-7#ref-CR62" id="ref-link-section-d43177717e6008">62</a></sup>.</p><p>Recently, Whitmore et al.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 63" title="Whitmore, L. et al. Inadvertent human genomic bycatch and intentional capture raise beneficial applications and ethical concerns with environmental DNA. Nat. Ecol. Evol. 
                  https://doi.org/10.1038/s41559-023-02056-2
                  
                 (2023)." href="https://interjectedfuture.com/articles/s41598-023-46151-7#ref-CR63" id="ref-link-section-d43177717e6015">63</a></sup> described recovery of human DNA from water, air and sand. They used untargeted shotgun-sequencing and were able to detect genetic information. They observed that the amount of DNA detected was dependent on the distance from nearest human residence. Humans leave behind traces of their DNA at any location they occupy and this DNA can transfer to the nearby surroundings. With the rapid advances in DNA technologies, it is now possible to detect single fragments of human DNA in environmental samples. From a forensic perspective this gives new opportunities to detect DNA from occupants of a crime scene, but before adapting any new method it is important to understand the limitations, especially in the context of transfer, prevalence, persistence and recovery. Hence, more research is needed before this method can be adopted to criminal investigations.</p><p>In forensic DNA sampling there is always the possibility to collect DNA from persons not involved in a crime. In contrast to the whole genome sequencing approach utilized by environmental scientists, the traditional forensic DNA profiling does not obtain information about genes or genetic variants, it will only provide information about the identity of the donor if compared with his/her reference profile. The method proposed here does not offer any genetic information that exceeds that already available from traditional forensic samples, and furthermore is compatible with national DNA databases. The method is “investigative” rather than “evaluative”. Consistent with current practice, this means that the purpose is to provide a candidate list of references to investigators, where this list may contain both false positives and false negative results. Much more information is needed before an evaluative report can be prepared and the findings can be considered fit for testimony. Regardless, when new methods or genetic markers are applied in forensic genetics, it is always necessary to properly consider the ethical implications that may arise. </p><p>In conclusion, although environmental DNA in forensic genetics has been known about for more than 20 years, the focus has been on contamination issues and associated risk management that includes use of restricted laboratories and protective clothing, along with well-established probabilistic Bayesian frameworks to interpret evidence. Here the focus has shifted in order to show how environmental DNA can be beneficially used in criminal investigations.</p><p>Novel methods were developed to collect and isolate human DNA from the air and dust. It was shown that air samples can provide STR genotypes of recent room occupants, while DNA from dust samples can identify individuals who occupied the room over longer periods. The study also revealed detectable levels of DNA in air and dust samples from forensic laboratories, improving our understanding of potential contamination sources.</p><p>The Bayesian models introduced can characterize DNA transfer in indoor dust and be used to estimate the number of samples needed to detect all occupants. The information obtained from air and dust samples may be used to search national DNA databases for intelligence purposes, identifying individuals who may have been present in the premises at recent and more distant times.</p></div></div></section><section data-title="Methods"><div id="Sec11-section"><h2 id="Sec11">Methods</h2><div id="Sec11-content"><h3 id="Sec12">Ethical declaration</h3><p>This study was approved by the data protection officer (DPO) at Oslo University Hospital (reference: 20/20155). Written informed consent was obtained from all participants. All experiments were performed in accordance with the relevant guidelines and regulations.</p><h3 id="Sec13">Sample collection</h3><p>A total of 184 samples were collected and analyzed for this study, consisting of 40 air samples and 144 dust samples. The study was conducted in the premises of the Forensic Genetics Unit at Oslo University Hospital. Out of 64 regular employees, 55 consented to take part in the study. Fourteen offices (with 1–7 occupants), two meeting rooms and five laboratories, connected by corridors, were used as the test site.</p><p>The first experiment examined dust and air DNA from the 14 different offices. The office environment is well defined, since the occupants are known, along with duration of occupancy. In the second experiment, similar studies were carried out with respect to two communal meeting rooms; here conditions were less well defined: the potential number of occupants is much higher than per office, but the frequency and duration of occupancy was much lower per individual. The third experiment was an examination of laboratories where access is restricted and protective clothing is worn to minimize risks of cross contamination.</p><h3 id="Sec14">Air samples</h3><p>This study was conducted in hospital premises which is provided with an efficient ventilation system that enables high air exchange rates. A double filtering system (filter classes F7 and F9) is installed to introduce outdoor air into the premises. The total air volume of offices is completely changed around 5 times per hour, depending on the room dimensions; in meeting rooms, air is completely changed 8 times per hour while air in laboratories is changed 10 times per hours.</p><p>A total of 41 air samples were collected. Seventeen air samples were collected from offices directly after occupancy (0 h), eight samples were collected c.16 h after occupancy (not consecutive to the 0 h samples), two samples were collected three days after occupancy and one 15 days after occupancy. Seven air samples were collected from the two meeting rooms and six samples were collected from five laboratories directly after occupancy. In the intervening period, rooms were left undisturbed. One air sample collected from a meeting room was excluded from the dataset, as there was evidence of contamination that occurred from the researcher that collected the sample (Supplementary Data <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://interjectedfuture.com/articles/s41598-023-46151-7#MOESM1">1</a>). Participants were asked complete a questionnaire to gather information about their activity in their office during the sampling day; specifically, each participant declared how many hours they spent in the office, who was the last person to leave the office and if the office had been visited by other individuals. Samples were collected using the AirPrep™ ACD220 electret filter air sampler (Innovaprep<sup>®</sup>), a device provided with a 360° aerosol inlet, working with a 52 mm electret filter<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 64" title="InnovaPrep. AirPrep™ Cub Sampler User Guide models ACD210 and ACD200. (2021)." href="https://interjectedfuture.com/articles/s41598-023-46151-7#ref-CR64" id="ref-link-section-d43177717e6071">64</a></sup>. All rooms were unoccupied during sample collection. To set up the equipment, a single researcher entered the room with full personal protective equipment (lab coat, hairnet, face mask and gloves) and remained in the room just for the time necessary to set up the air sampler. The reference profile of the researcher that collected and processed the samples was compared to the DNA profiles obtained from all air samples to detect adventitious contaminations. The air sampler was run continuously for 2 h at a maximum flow rate of 200 LPM, positioned on a standard off-camera tripod at a distance of 110 cm from the floor. The sampler was located in the middle of the room, as far away from any furniture (e.g. chairs, desks) as possible, to avoid the collection of particles from nearby sources.</p><p>After collecting the sample, the air filter was secured in a sterile, pre-labeled, zip-lock bag and stored at room temperature until further processing. The whole sampler surface was cleaned before each sample collection using 70% ethanol and subsequently UV irradiated for 15 min. The external sampler surface and the tripod were further cleaned with a 0.1% hypochlorite solution to avoid cross-contamination. Two control samples were collected from the sampler’s internal surface that comes in contact with the air filter and two from the sampler’s lid, using moistened cotton swabs, after two different cleaning procedures, to evaluate their efficacy. In each test carried out, all four samples provided negative results (no CT) when quantified with PowerQuant<sup>®</sup> System (Promega) on the 7500 RealTime PCR System (Applied Biosystems), showing that no detectable biological material was present on the sampler surfaces after the cleaning process.</p><h3 id="Sec15">Dust samples</h3><p>A total of 144 dust samples were collected in parallel to air samples (Supplementary Data <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://interjectedfuture.com/articles/s41598-023-46151-7#MOESM1">1</a>). Dust samples were collected on the same day as the corresponding air samples, immediately after the air sample collection was concluded. Samples were taken using moistened cotton swabs from different dusty undisturbed surfaces that were not subject to regular cleaning, including door ledges, radiators, electric fittings and window sills. Samples were collected by swabbing a small area with light pressure, for about 10 s. Each cotton swab was placed into a labeled evidence bag and stored at room temperature until further processing.</p><h3 id="Sec16">Sample processing</h3><p>Samples were processed in a “DNA-free” investigation room. To avoid contamination the researcher handling the samples wore personal protective equipment during sample processing. Gloves were changed between each sample. Air samples were processed separately from dust, and samples from different rooms were processed on clean bench surfaces wiped with 0.1% hypochlorite solution, and covered with new bench paper. Each filter (air sample) was extracted from its zip-lock bag and the filter membrane was removed from its plastic shell using sterile scissors and tweezers. The filter membrane was cut in half and each half was cut into small pieces so that they could fit into two 2 ml extraction tubes. Each cotton swab (dust sample) was extracted from its sampling bag; the cotton swab tip was cut and placed into a 2 ml extraction tube. Samples were stored at room temperature until DNA extraction.</p><h3 id="Sec17">DNA extraction</h3><p>The DNA extraction from the air filters was performed following a modified QIAamp<sup>®</sup> DNA mini kit (Qiagen) procedure. The protocol was adapted from Clare et al.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 65" title="Clare, E. L. et al. eDNAir: Proof of concept that animal DNA can be collected from air sampling. PeerJ 9, e11030. 
                  https://doi.org/10.7717/peerj.11030
                  
                 (2021)." href="https://interjectedfuture.com/articles/s41598-023-46151-7#ref-CR65" id="ref-link-section-d43177717e6110">65</a></sup> and Duhaime and Cable<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 66" title="Duhaime M. B., Cable R. (2019). DNA Extraction from Filters using QIAgen DNeasy and QIAshredder V.1. 
                  https://doi.org/10.17504/protocols.io.jx5cpq6
                  
                ." href="https://interjectedfuture.com/articles/s41598-023-46151-7#ref-CR66" id="ref-link-section-d43177717e6114">66</a></sup>. The extraction of one air sample was carried out using two 2 ml extraction tubes, each containing half-filter cut into small pieces. Cell lysis was performed by submerging all filter pieces into 450 µl of ATL and 500 µl of AL lysis buffers and 50 µl of Proteinase K and incubating the samples at 56 °C, shaking them at 650 rpm for 1 h. The samples were further shaken at maximum speed for 10 min, before centrifugation at 17.000 <i>g</i> for 30 s. The filter pieces were subsequently transferred using sterile tweezers, along with the sample lysate, into QIAshredder columns and centrifuged at 17.000 <i>g</i> for 15 s. The transfer was performed in multiple steps, until all filter pieces and lysate had been moved to the columns. After discarding the QIAshredder columns with the retained dry filter pieces, 500 µl of 100% ethanol was added to the solution and mixed by vortexing. Following the manufacturer’s protocol, the mixture of lysate and ethanol was transferred onto the QIAamp DNA mini spin columns and centrifuged at 17.000 <i>g</i> for 15 s. The columns were then washed with 500 µl of AW1 wash buffer and centrifuged for 15 s at 17.000 <i>g</i>, and further washed with AW2 wash buffer and centrifuged at 12,000 <i>g</i> for 3 min. A second centrifugation step was done at full speed for 1 min. The sample was eluted using 30 µl of AE elution buffer, preheated to 70 °C after an incubation of 5 min and was centrifuged for 1 min at ≥ 8000 <i>g</i>. The elution step was repeated by passing the first eluate through the column a second time. Finally, since one filter was processed in two tubes, the eluates from the two tubes were pooled, resulting in 60 µl of extract. Negative extraction controls were extracted along with the samples. Dust samples were extracted following the same protocol as described for air samples, but processing the cotton swab tip in a single tube, resulting in 30 µl of extract.</p><h3 id="Sec18">DNA analysis</h3><p>Samples were quantified using the PowerQuant<sup>®</sup> System (Promega) on the 7500 RealTime PCR System (Applied Biosystems) and amplified using PowerPlex<sup>®</sup> Fusion 6C (Promega) kit with 1.0 ng template DNA input in a final reaction volume of 25 µl. Applied Biosystem<sup>®</sup> Veriti 96-Well Thermal Cycler (ThermoFisher) with a 29 PCR cycles program was used for the amplification. Negative and positive controls were run with the samples. Capillary electrophoresis was performed on the Applied Biosysetm<sup>®</sup> 3500xL Genetic Analyzer (ThermoFisher) with an injection time of 24 s and a voltage of 1.2 kV.</p><h3 id="Sec19">Data analysis</h3><p>The data were analyzed using GeneMapper ID-X v.1.6 (Life Technologies) with an analytical threshold (AT) set to 50 RFU for air samples and to 100 RFU for dust samples. Technical artefacts were filtered from the DNA profiles, according to internal guidelines. DNA profiles obtained from air samples were evaluated in relation to the use and length of time since occupancy of the room from where samples were collected. DNA profiles obtained from dust samples were evaluated in relation to the use of the room and the number of room occupants. A comparative analysis of the results obtained from air and dust samples was also conducted.</p><h3 id="Sec20">Calculation of the likelihood ratio</h3><p>A likelihood ratio framework was applied to assign the value of the evidence conditioned on a given person of interest (<i>PoI</i>), e.g., the location occupant. An LR was calculated for each matching reference sample from the database as the <i>PoI</i> in turn (Eq. (<a data-track="click" data-track-label="link" data-track-action="equation anchor" href="https://interjectedfuture.com/articles/s41598-023-46151-7#Equ1">1</a>)).</p><p>Likelihood ratio (LR) values given by Eq. (<a data-track="click" data-track-label="link" data-track-action="equation anchor" href="https://interjectedfuture.com/articles/s41598-023-46151-7#Equ1">1</a>) were calculated from evidence profile <i>E</i>, given two alternative propositions (<i>H</i><sub><i>1</i></sub>) and (<i>H</i><sub><i>2</i></sub>), where <span>\({H}_{1}\)</span> is: “<span>\(PoI\)</span> and <span>\(L-1\)</span> unrelated unknowns are contributors”, versus <span>\({H}_{2}\)</span>: “<span>\(L\)</span> unrelated unknowns are contributors”. A reference genotype database <i>R</i> of the consenting 55 occupants of the building participating in the study was compiled. Under <span>\({H}_{1}\)</span> the reference profile <i>PoI</i> is conditioned, which is an individual from the reference database <span>\(R\)</span>. Every DNA profile retrieved was compared against each reference in the database (POI<sub>1:55</sub>) using dnamatch2<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 38" title="Bleka, Ø., Bouzga, M., Fonneløp, A. &amp; Gill, P. dnamatch2: An open source software to carry out large scale database searches of mixtures using qualitative and quantitative models. Forensic Sci. Int. Genet. Suppl. Ser. 6, e404–e406. 
                  https://doi.org/10.1016/j.fsigss.2017.09.160
                  
                 (2017)." href="https://interjectedfuture.com/articles/s41598-023-46151-7#ref-CR38" id="ref-link-section-d43177717e6354">38</a></sup> (<a href="http://www.euroformix.com/dnamatch2">http://www.euroformix.com/dnamatch2</a>). This open-source software is a fast, large-scale DNA database search algorithm based on EuroForMix (<a href="http://www.euroformix.com/">http://www.euroformix.com/</a>, version 4.0.3 applied).</p><p>The analysis enabled identification of “matches” with genotypes of the 55 known occupants of the building and to calculate the corresponding LR values, the number of contributors (NOC) and mixture proportions (<i>M</i><sub><i>x</i></sub>). The following settings for dnamatch2 were used for the analyses: score thresholds: matching allele counting (MAC) = 0.3, qualitative and quantitative LR = 1; model set-up: drop-in probability = 0.05, Lambda parameter = 0.01, min freq = 0.001; prefilter thresholds: analytical threshold (AT) = 50 RFU, stutter rate threshold = 0.1, major extraction rate threshold = 0.6, minimum loci requirement (Evid and Maj) = 3; the maximum number of contributor was set to 4. Stutter models not applied.</p><p>To minimize adventitious or false hits with the database, a match of the evidence with a reference sample was defined as a likelihood ratio (LR) greater than 100. The reference database used in our study was relatively small, however there is no restriction placed upon its size: indeed, massive intelligence databases of several million individuals are amenable. In practice, comparisons with large intelligence databases size N, will require high LRs in order to reduce the number of false positive matches. For example, if the database size is 1 m, and the LR recovered is 10,000, from the Turing expectation c. 100 adventitious matches may be observed in such a database (Pm = N/LR)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Gill, P., Bleka, Ø. &amp; Egeland, T. Does an English appeal court ruling increase the risks of miscarriages of justice when complex DNA profiles are searched against the national DNA database?. Forensic Sci. Int. Genet. 13, 167–175. 
                  https://doi.org/10.1016/j.fsigen.2014.07.015
                  
                 (2014)." href="#ref-CR35" id="ref-link-section-d43177717e6385">35</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Taylor, D., Curran, J. M. &amp; Buckleton, J. Importance sampling allows H(d) true tests of highly discriminating DNA profiles. Forensic Sci. Int. Genet. 27, 74–81. 
                  https://doi.org/10.1016/j.fsigen.2016.12.004
                  
                 (2017)." href="#ref-CR36" id="ref-link-section-d43177717e6385_1">36</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 37" title="Good, I. J. Probability and the Weighing of Evidence. Vol. 26 (C. Griffin London, 1950)." href="https://interjectedfuture.com/articles/s41598-023-46151-7#ref-CR37" id="ref-link-section-d43177717e6388">37</a></sup>. Smaller databases (N = 55), such as that used here will result in much lower adventitious match rates. Therefore, for this experiment we used an arbitrary LR = 100 as a reasonable threshold to distinguish true positives from false negatives; but the optimum threshold used depends upon the size of the intelligence database used.</p><p>The final list of candidate matches and relative LR values was further evaluated with respect to the information available regarding each sample, such as the identity of the individual(s) occupying a specific room and the information gathered through the activity questionnaire (for air samples). This enabled us to establish the number of known occupants, and known non-occupants detected for each sample collected. This study focused on known individuals (i.e. the participants of the study); unknown contributors (i.e. individuals not present in the reference database) were not considered. LR values were compared between air and dust samples collected from the same locations.</p><h3 id="Sec21">Outline of the model</h3><p>The following factors and variables were recorded:</p><ol>
                  <li>
                    <span>1)</span>
                    
                      <p>Location: <span>\(z=1,\dots Z\)</span>. Where locations are offices (R), meeting rooms (M) and laboratories (L).</p>
                    
                  </li>
                  <li>
                    <span>2)</span>
                    
                      <p><span>\({K}_{z}\)</span>: Number of unique, known occupants in the location.</p>
                    
                  </li>
                  <li>
                    <span>3)</span>
                    
                      <p><i>k</i><sub><i>z</i></sub>: A specific occupant from location <i>z</i>.</p>
                    
                  </li>
                  <li>
                    <span>4)</span>
                    
                      <p><span>\({S}_{z}:\)</span> Number of samples taken from location <i>z</i> (over a period of days).</p>
                    
                  </li>
                  <li>
                    <span>5)</span>
                    
                      <p><span>\({N}_{z}\)</span>: Number of samples from location <i>z</i> where no known occupants were positively identified.</p>
                    
                  </li>
                  <li>
                    <span>6)</span>
                    
                      <p><span>\({NC}_{z}\)</span>: Number of samples from location <i>z</i> where no known non-occupants were positively identified.</p>
                    
                  </li>
                  <li>
                    <span>7)</span>
                    
                      <p><i>X</i><sub><i>k,z</i></sub>: Number of samples from location <i>z</i> with positive identification of occupant <i>k</i><sub><i>z</i></sub>.</p>
                    
                  </li>
                  <li>
                    <span>8)</span>
                    
                      <p><i>φ</i><sub><i>1</i></sub>: Success rate for occupants, defined as the proportion of samples where one or more occupants were detected (Eq. (<a data-track="click" data-track-label="link" data-track-action="equation anchor" href="https://interjectedfuture.com/articles/s41598-023-46151-7#Equ4">4</a>)).</p>
                    
                  </li>
                  <li>
                    <span>9)</span>
                    
                      <p><i>φ</i><sub><i>2</i></sub>: Success rate for non-occupants, defined as the proportion of samples where one or more known non-occupants were detected (Eq. (<a data-track="click" data-track-label="link" data-track-action="equation anchor" href="https://interjectedfuture.com/articles/s41598-023-46151-7#Equ5">5</a>)).</p><div id="Equ4"><p><span>$${\it {\varphi }_{1}}=1-\frac{\sum_{z}{N}_{z}}{\sum_{z}{S}_{z}}$$</span></p><p>
                    (4)
                </p></div><div id="Equ5"><p><span>$${\it {\varphi }_{2}}=1-\frac{\sum_{z}{NC}_{z}}{\sum_{z}{S}_{z}}$$</span></p><p>
                    (5)
                </p></div>
                    
                  </li>
                </ol><h3 id="Sec22">Statistical analysis</h3><p>Wilcoxon and Fisher exact two-tailed tests were used to determine if the expectations were the same in different groups with a 5% significance level. RStudio 2022.07.2 was used to perform the statistical analyses and the packages ggplot2 v. 3.4.0 and igraph 1.3.5 were used to generate the figures presented in this manuscript.</p></div></div></section>
                    
                </div><div>
                <section data-title="Data availability"><div id="data-availability-section"><h2 id="data-availability">Data availability</h2><div id="data-availability-content">
              
              <p>The data and programs used to carry out the logistic regression and Bayesian analysis in this study are available at: <a href="https://github.com/peterdgill/DustandAir">https://github.com/peterdgill/DustandAir</a>.</p>
            </div></div></section><div id="MagazineFulltextArticleBodySuffix"><section aria-labelledby="Bib1" data-title="References"><div id="Bib1-section"><h2 id="Bib1">References</h2><div id="Bib1-content"><div data-container-section="references"><ol data-track-component="outbound reference" data-track-context="references section"><li data-counter="1."><p id="ref-CR1">Roberts, D. &amp; Marks, R. The determination of regional and age variations in the rate of desquamation: A comparison of four techniques. <i>J. Invest. Dermatol.</i> <b>74</b>, 13–16. <a href="https://doi.org/10.1111/1523-1747.ep12514568" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1111/1523-1747.ep12514568">https://doi.org/10.1111/1523-1747.ep12514568</a> (1980).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1111/1523-1747.ep12514568" data-track-item_id="10.1111/1523-1747.ep12514568" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1111%2F1523-1747.ep12514568" aria-label="Article reference 1" data-doi="10.1111/1523-1747.ep12514568">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://interjectedfuture.com/articles/cas-redirect/1:STN:280:DyaL3c%2FpsFKhsQ%3D%3D" aria-label="CAS reference 1">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=6985944" aria-label="PubMed reference 1">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 1" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20determination%20of%20regional%20and%20age%20variations%20in%20the%20rate%20of%20desquamation%3A%20A%20comparison%20of%20four%20techniques&amp;journal=J.%20Invest.%20Dermatol.&amp;doi=10.1111%2F1523-1747.ep12514568&amp;volume=74&amp;pages=13-16&amp;publication_year=1980&amp;author=Roberts%2CD&amp;author=Marks%2CR">
                    Google Scholar</a> 
                </p></li><li data-counter="2."><p id="ref-CR2">Milstone, L. M. Epidermal desquamation. <i>J. Dermatol. Sci.</i> <b>36</b>, 131–140. <a href="https://doi.org/10.1016/j.jdermsci.2004.05.004" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1016/j.jdermsci.2004.05.004">https://doi.org/10.1016/j.jdermsci.2004.05.004</a> (2004).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.jdermsci.2004.05.004" data-track-item_id="10.1016/j.jdermsci.2004.05.004" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.jdermsci.2004.05.004" aria-label="Article reference 2" data-doi="10.1016/j.jdermsci.2004.05.004">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15541634" aria-label="PubMed reference 2">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 2" href="http://scholar.google.com/scholar_lookup?&amp;title=Epidermal%20desquamation&amp;journal=J.%20Dermatol.%20Sci.&amp;doi=10.1016%2Fj.jdermsci.2004.05.004&amp;volume=36&amp;pages=131-140&amp;publication_year=2004&amp;author=Milstone%2CLM">
                    Google Scholar</a> 
                </p></li><li data-counter="3."><p id="ref-CR3">Candi, E., Schmidt, R. &amp; Melino, G. The cornified envelope: A model of cell death in the skin. <i>Nat. Rev. Mol. Cell Biol.</i> <b>6</b>, 328–340. <a href="https://doi.org/10.1038/nrm1619" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1038/nrm1619">https://doi.org/10.1038/nrm1619</a> (2005).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/nrm1619" data-track-item_id="10.1038/nrm1619" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fnrm1619" aria-label="Article reference 3" data-doi="10.1038/nrm1619">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://interjectedfuture.com/articles/cas-redirect/1:CAS:528:DC%2BD2MXivVaktLc%3D" aria-label="CAS reference 3">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15803139" aria-label="PubMed reference 3">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 3" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20cornified%20envelope%3A%20A%20model%20of%20cell%20death%20in%20the%20skin&amp;journal=Nat.%20Rev.%20Mol.%20Cell%20Biol.&amp;doi=10.1038%2Fnrm1619&amp;volume=6&amp;pages=328-340&amp;publication_year=2005&amp;author=Candi%2CE&amp;author=Schmidt%2CR&amp;author=Melino%2CG">
                    Google Scholar</a> 
                </p></li><li data-counter="4."><p id="ref-CR4">Sun, T. T. &amp; Green, H. Differentiation of the epidermal keratinocyte in cell culture: Formation of the cornified envelope. <i>Cell</i> <b>9</b>, 511–521. <a href="https://doi.org/10.1016/0092-8674(76)90033-7" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1016/0092-8674(76)90033-7">https://doi.org/10.1016/0092-8674(76)90033-7</a> (1976).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/0092-8674(76)90033-7" data-track-item_id="10.1016/0092-8674(76)90033-7" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2F0092-8674%2876%2990033-7" aria-label="Article reference 4" data-doi="10.1016/0092-8674(76)90033-7">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://interjectedfuture.com/articles/cas-redirect/1:CAS:528:DyaE2sXmvFaluw%3D%3D" aria-label="CAS reference 4">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=1009573" aria-label="PubMed reference 4">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 4" href="http://scholar.google.com/scholar_lookup?&amp;title=Differentiation%20of%20the%20epidermal%20keratinocyte%20in%20cell%20culture%3A%20Formation%20of%20the%20cornified%20envelope&amp;journal=Cell&amp;doi=10.1016%2F0092-8674%2876%2990033-7&amp;volume=9&amp;pages=511-521&amp;publication_year=1976&amp;author=Sun%2CTT&amp;author=Green%2CH">
                    Google Scholar</a> 
                </p></li><li data-counter="5."><p id="ref-CR5">Lidwell, O. M., Mackintosh, C. A. &amp; Towers, A. G. The evaluation of fabrics in relation to their use as protective garments in nursing and surgery. II. Dispersal of skin organisms in a test chamber. <i>J. Hyg. (Lond.)</i> <b>81</b>, 453–469. <a href="https://doi.org/10.1017/s002217240002533x" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1017/s002217240002533x">https://doi.org/10.1017/s002217240002533x</a> (1978).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1017/s002217240002533x" data-track-item_id="10.1017/s002217240002533x" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1017%2Fs002217240002533x" aria-label="Article reference 5" data-doi="10.1017/s002217240002533x">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://interjectedfuture.com/articles/cas-redirect/1:STN:280:DyaE1M%2FovFKmsA%3D%3D" aria-label="CAS reference 5">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=731025" aria-label="PubMed reference 5">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 5" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20evaluation%20of%20fabrics%20in%20relation%20to%20their%20use%20as%20protective%20garments%20in%20nursing%20and%20surgery.%20II.%20Dispersal%20of%20skin%20organisms%20in%20a%20test%20chamber&amp;journal=J.%20Hyg.%20%28Lond.%29&amp;doi=10.1017%2Fs002217240002533x&amp;volume=81&amp;pages=453-469&amp;publication_year=1978&amp;author=Lidwell%2COM&amp;author=Mackintosh%2CCA&amp;author=Towers%2CAG">
                    Google Scholar</a> 
                </p></li><li data-counter="6."><p id="ref-CR6">Mackintosh, C. A., Lidwell, O. M., Towers, A. G. &amp; Marples, R. R. The dimensions of skin fragments dispersed into the air during activity. <i>J. Hyg. (Lond.)</i> <b>81</b>, 471–479. <a href="https://doi.org/10.1017/s0022172400025341" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1017/s0022172400025341">https://doi.org/10.1017/s0022172400025341</a> (1978).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1017/s0022172400025341" data-track-item_id="10.1017/s0022172400025341" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1017%2Fs0022172400025341" aria-label="Article reference 6" data-doi="10.1017/s0022172400025341">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://interjectedfuture.com/articles/cas-redirect/1:STN:280:DyaE1M%2FovFKmsQ%3D%3D" aria-label="CAS reference 6">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=731026" aria-label="PubMed reference 6">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 6" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20dimensions%20of%20skin%20fragments%20dispersed%20into%20the%20air%20during%20activity&amp;journal=J.%20Hyg.%20%28Lond.%29&amp;doi=10.1017%2Fs0022172400025341&amp;volume=81&amp;pages=471-479&amp;publication_year=1978&amp;author=Mackintosh%2CCA&amp;author=Lidwell%2COM&amp;author=Towers%2CAG&amp;author=Marples%2CRR">
                    Google Scholar</a> 
                </p></li><li data-counter="7."><p id="ref-CR7">Burrill, J., Daniel, B. &amp; Frascione, N. Illuminating touch deposits through cellular characterization of hand rinses and body fluids with nucleic acid fluorescence. <i>Forensic Sci. Int. Genet.</i> <b>46</b>, 102269. <a href="https://doi.org/10.1016/j.fsigen.2020.102269" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1016/j.fsigen.2020.102269">https://doi.org/10.1016/j.fsigen.2020.102269</a> (2020).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.fsigen.2020.102269" data-track-item_id="10.1016/j.fsigen.2020.102269" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.fsigen.2020.102269" aria-label="Article reference 7" data-doi="10.1016/j.fsigen.2020.102269">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://interjectedfuture.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXksVOisrw%3D" aria-label="CAS reference 7">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32155589" aria-label="PubMed reference 7">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 7" href="http://scholar.google.com/scholar_lookup?&amp;title=Illuminating%20touch%20deposits%20through%20cellular%20characterization%20of%20hand%20rinses%20and%20body%20fluids%20with%20nucleic%20acid%20fluorescence&amp;journal=Forensic%20Sci.%20Int.%20Genet.&amp;doi=10.1016%2Fj.fsigen.2020.102269&amp;volume=46&amp;publication_year=2020&amp;author=Burrill%2CJ&amp;author=Daniel%2CB&amp;author=Frascione%2CN">
                    Google Scholar</a> 
                </p></li><li data-counter="8."><p id="ref-CR8">Kanokwongnuwut, P., Kirkbride, K. P. &amp; Linacre, A. Detection of latent DNA. <i>Forensic Sci. Int. Genet.</i> <b>37</b>, 95–101. <a href="https://doi.org/10.1016/j.fsigen.2018.08.004" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1016/j.fsigen.2018.08.004">https://doi.org/10.1016/j.fsigen.2018.08.004</a> (2018).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.fsigen.2018.08.004" data-track-item_id="10.1016/j.fsigen.2018.08.004" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.fsigen.2018.08.004" aria-label="Article reference 8" data-doi="10.1016/j.fsigen.2018.08.004">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://interjectedfuture.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhsVyntbjP" aria-label="CAS reference 8">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30103147" aria-label="PubMed reference 8">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 8" href="http://scholar.google.com/scholar_lookup?&amp;title=Detection%20of%20latent%20DNA&amp;journal=Forensic%20Sci.%20Int.%20Genet.&amp;doi=10.1016%2Fj.fsigen.2018.08.004&amp;volume=37&amp;pages=95-101&amp;publication_year=2018&amp;author=Kanokwongnuwut%2CP&amp;author=Kirkbride%2CKP&amp;author=Linacre%2CA">
                    Google Scholar</a> 
                </p></li><li data-counter="9."><p id="ref-CR9">Burrill, J., Rammenou, E., Alawar, F., Daniel, B. &amp; Frascione, N. Corneocyte lysis and fragmented DNA considerations for the cellular component of forensic touch DNA. <i>Forensic Sci. Int. Genet.</i> <b>51</b>, 102428. <a href="https://doi.org/10.1016/j.fsigen.2020.102428" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1016/j.fsigen.2020.102428">https://doi.org/10.1016/j.fsigen.2020.102428</a> (2021).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.fsigen.2020.102428" data-track-item_id="10.1016/j.fsigen.2020.102428" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.fsigen.2020.102428" aria-label="Article reference 9" data-doi="10.1016/j.fsigen.2020.102428">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://interjectedfuture.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXis1Sgt7rF" aria-label="CAS reference 9">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33338861" aria-label="PubMed reference 9">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 9" href="http://scholar.google.com/scholar_lookup?&amp;title=Corneocyte%20lysis%20and%20fragmented%20DNA%20considerations%20for%20the%20cellular%20component%20of%20forensic%20touch%20DNA&amp;journal=Forensic%20Sci.%20Int.%20Genet.&amp;doi=10.1016%2Fj.fsigen.2020.102428&amp;volume=51&amp;publication_year=2021&amp;author=Burrill%2CJ&amp;author=Rammenou%2CE&amp;author=Alawar%2CF&amp;author=Daniel%2CB&amp;author=Frascione%2CN">
                    Google Scholar</a> 
                </p></li><li data-counter="10."><p id="ref-CR10">Hammer, U., Bulnheim, U., Karstädt, G., Meissner, D. &amp; Wegener, R. Zur DNA-Typisierung übertragener Hautzellen nach körperlicher Gewalt. <i>Rechtsmedizin</i> <b>7</b>, 180–183. <a href="https://doi.org/10.1007/s001940050010" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1007/s001940050010">https://doi.org/10.1007/s001940050010</a> (1997).</p><p><a data-track="click_references" rel="noopener" data-track-label="10.1007/s001940050010" data-track-item_id="10.1007/s001940050010" data-track-value="article reference" data-track-action="article reference" href="https://link.springer.com/doi/10.1007/s001940050010" aria-label="Article reference 10" data-doi="10.1007/s001940050010">Article</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 10" href="http://scholar.google.com/scholar_lookup?&amp;title=Zur%20DNA-Typisierung%20%C3%BCbertragener%20Hautzellen%20nach%20k%C3%B6rperlicher%20Gewalt&amp;journal=Rechtsmedizin&amp;doi=10.1007%2Fs001940050010&amp;volume=7&amp;pages=180-183&amp;publication_year=1997&amp;author=Hammer%2CU&amp;author=Bulnheim%2CU&amp;author=Karst%C3%A4dt%2CG&amp;author=Meissner%2CD&amp;author=Wegener%2CR">
                    Google Scholar</a> 
                </p></li><li data-counter="11."><p id="ref-CR11">Herber, B. &amp; Herold, K. DNA typing of human dandruff. <i>J. Forensic Sci.</i> <b>43</b>, 648–656 (1998).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1520/JFS16197J" data-track-item_id="10.1520/JFS16197J" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1520%2FJFS16197J" aria-label="Article reference 11" data-doi="10.1520/JFS16197J">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://interjectedfuture.com/articles/cas-redirect/1:STN:280:DyaK1c3ntFKgsg%3D%3D" aria-label="CAS reference 11">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=9608703" aria-label="PubMed reference 11">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 11" href="http://scholar.google.com/scholar_lookup?&amp;title=DNA%20typing%20of%20human%20dandruff&amp;journal=J.%20Forensic%20Sci.&amp;doi=10.1520%2FJFS16197J&amp;volume=43&amp;pages=648-656&amp;publication_year=1998&amp;author=Herber%2CB&amp;author=Herold%2CK">
                    Google Scholar</a> 
                </p></li><li data-counter="12."><p id="ref-CR12">Finnebraaten, M., Granér, T. &amp; Hoff-Olsen, P. May a speaking individual contaminate the routine DNA laboratory?. <i>Forensic Sci. Int. Genet. Suppl. Ser.</i> <b>1</b>, 421–422. <a href="https://doi.org/10.1016/j.fsigss.2007.10.030" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1016/j.fsigss.2007.10.030">https://doi.org/10.1016/j.fsigss.2007.10.030</a> (2008).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.fsigss.2007.10.030" data-track-item_id="10.1016/j.fsigss.2007.10.030" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.fsigss.2007.10.030" aria-label="Article reference 12" data-doi="10.1016/j.fsigss.2007.10.030">Article</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 12" href="http://scholar.google.com/scholar_lookup?&amp;title=May%20a%20speaking%20individual%20contaminate%20the%20routine%20DNA%20laboratory%3F&amp;journal=Forensic%20Sci.%20Int.%20Genet.%20Suppl.%20Ser.&amp;doi=10.1016%2Fj.fsigss.2007.10.030&amp;volume=1&amp;pages=421-422&amp;publication_year=2008&amp;author=Finnebraaten%2CM&amp;author=Gran%C3%A9r%2CT&amp;author=Hoff-Olsen%2CP">
                    Google Scholar</a> 
                </p></li><li data-counter="13."><p id="ref-CR13">Port, N. J., Bowyer, V. L., Graham, E. A., Batuwangala, M. S. &amp; Rutty, G. N. How long does it take a static speaking individual to contaminate the immediate environment?. <i>Forensic Sci. Med. Pathol.</i> <b>2</b>, 157–163. <a href="https://doi.org/10.1007/s12024-006-0004-z" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1007/s12024-006-0004-z">https://doi.org/10.1007/s12024-006-0004-z</a> (2006).</p><p><a data-track="click_references" rel="noopener" data-track-label="10.1007/s12024-006-0004-z" data-track-item_id="10.1007/s12024-006-0004-z" data-track-value="article reference" data-track-action="article reference" href="https://link.springer.com/doi/10.1007/s12024-006-0004-z" aria-label="Article reference 13" data-doi="10.1007/s12024-006-0004-z">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25868693" aria-label="PubMed reference 13">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 13" href="http://scholar.google.com/scholar_lookup?&amp;title=How%20long%20does%20it%20take%20a%20static%20speaking%20individual%20to%20contaminate%20the%20immediate%20environment%3F&amp;journal=Forensic%20Sci.%20Med.%20Pathol.&amp;doi=10.1007%2Fs12024-006-0004-z&amp;volume=2&amp;pages=157-163&amp;publication_year=2006&amp;author=Port%2CNJ&amp;author=Bowyer%2CVL&amp;author=Graham%2CEA&amp;author=Batuwangala%2CMS&amp;author=Rutty%2CGN">
                    Google Scholar</a> 
                </p></li><li data-counter="14."><p id="ref-CR14">Rutty, G. N., Hopwood, A. &amp; Tucker, V. The effectiveness of protective clothing in the reduction of potential DNA contamination of the scene of crime. <i>Int. J. Legal Med.</i> <b>117</b>, 170–174. <a href="https://doi.org/10.1007/s00414-002-0348-1" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1007/s00414-002-0348-1">https://doi.org/10.1007/s00414-002-0348-1</a> (2003).</p><p><a data-track="click_references" rel="noopener" data-track-label="10.1007/s00414-002-0348-1" data-track-item_id="10.1007/s00414-002-0348-1" data-track-value="article reference" data-track-action="article reference" href="https://link.springer.com/doi/10.1007/s00414-002-0348-1" aria-label="Article reference 14" data-doi="10.1007/s00414-002-0348-1">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://interjectedfuture.com/articles/cas-redirect/1:STN:280:DC%2BD3s3nt1artA%3D%3D" aria-label="CAS reference 14">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=12684818" aria-label="PubMed reference 14">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 14" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20effectiveness%20of%20protective%20clothing%20in%20the%20reduction%20of%20potential%20DNA%20contamination%20of%20the%20scene%20of%20crime&amp;journal=Int.%20J.%20Legal%20Med.&amp;doi=10.1007%2Fs00414-002-0348-1&amp;volume=117&amp;pages=170-174&amp;publication_year=2003&amp;author=Rutty%2CGN&amp;author=Hopwood%2CA&amp;author=Tucker%2CV">
                    Google Scholar</a> 
                </p></li><li data-counter="15."><p id="ref-CR15">Thornbury, D., Goray, M. &amp; van Oorschot, R. A. H. Indirect DNA transfer without contact from dried biological materials on various surfaces. <i>Forensic Sci. Int. Genet.</i> <b>51</b>, 102457. <a href="https://doi.org/10.1016/j.fsigen.2020.102457" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1016/j.fsigen.2020.102457">https://doi.org/10.1016/j.fsigen.2020.102457</a> (2021).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.fsigen.2020.102457" data-track-item_id="10.1016/j.fsigen.2020.102457" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.fsigen.2020.102457" aria-label="Article reference 15" data-doi="10.1016/j.fsigen.2020.102457">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://interjectedfuture.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhtlOltLg%3D" aria-label="CAS reference 15">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33465559" aria-label="PubMed reference 15">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 15" href="http://scholar.google.com/scholar_lookup?&amp;title=Indirect%20DNA%20transfer%20without%20contact%20from%20dried%20biological%20materials%20on%20various%20surfaces&amp;journal=Forensic%20Sci.%20Int.%20Genet.&amp;doi=10.1016%2Fj.fsigen.2020.102457&amp;volume=51&amp;publication_year=2021&amp;author=Thornbury%2CD&amp;author=Goray%2CM&amp;author=Oorschot%2CRAH">
                    Google Scholar</a> 
                </p></li><li data-counter="16."><p id="ref-CR16">Thornbury, D., Goray, M. &amp; van Oorschot, R. A. H. Transfer of DNA without contact from used clothing, pillowcases and towels by shaking agitation. <i>Sci. Justice</i> <b>61</b>, 797–805. <a href="https://doi.org/10.1016/j.scijus.2021.10.005" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1016/j.scijus.2021.10.005">https://doi.org/10.1016/j.scijus.2021.10.005</a> (2021).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.scijus.2021.10.005" data-track-item_id="10.1016/j.scijus.2021.10.005" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.scijus.2021.10.005" aria-label="Article reference 16" data-doi="10.1016/j.scijus.2021.10.005">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34802654" aria-label="PubMed reference 16">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 16" href="http://scholar.google.com/scholar_lookup?&amp;title=Transfer%20of%20DNA%20without%20contact%20from%20used%20clothing%2C%20pillowcases%20and%20towels%20by%20shaking%20agitation&amp;journal=Sci.%20Justice&amp;doi=10.1016%2Fj.scijus.2021.10.005&amp;volume=61&amp;pages=797-805&amp;publication_year=2021&amp;author=Thornbury%2CD&amp;author=Goray%2CM&amp;author=Oorschot%2CRAH">
                    Google Scholar</a> 
                </p></li><li data-counter="17."><p id="ref-CR17">Puliatti, L., Handt, O. &amp; Taylor, D. The level of DNA an individual transfers to untouched items in their immediate surroundings. <i>Forensic Sci. Int. Genet.</i> <b>54</b>, 102561. <a href="https://doi.org/10.1016/j.fsigen.2021.102561" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1016/j.fsigen.2021.102561">https://doi.org/10.1016/j.fsigen.2021.102561</a> (2021).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.fsigen.2021.102561" data-track-item_id="10.1016/j.fsigen.2021.102561" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.fsigen.2021.102561" aria-label="Article reference 17" data-doi="10.1016/j.fsigen.2021.102561">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://interjectedfuture.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhs1WntbvM" aria-label="CAS reference 17">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34303991" aria-label="PubMed reference 17">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 17" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20level%20of%20DNA%20an%20individual%20transfers%20to%20untouched%20items%20in%20their%20immediate%20surroundings&amp;journal=Forensic%20Sci.%20Int.%20Genet.&amp;doi=10.1016%2Fj.fsigen.2021.102561&amp;volume=54&amp;publication_year=2021&amp;author=Puliatti%2CL&amp;author=Handt%2CO&amp;author=Taylor%2CD">
                    Google Scholar</a> 
                </p></li><li data-counter="18."><p id="ref-CR18">Locard, E. The analysis of dust traces. <i>Part I. Am. J. Police Sci.</i> <b>1</b>, 276–298. <a href="https://doi.org/10.2307/1147154" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.2307/1147154">https://doi.org/10.2307/1147154</a> (1930).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.2307/1147154" data-track-item_id="10.2307/1147154" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.2307%2F1147154" aria-label="Article reference 18" data-doi="10.2307/1147154">Article</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 18" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20analysis%20of%20dust%20traces&amp;journal=Part%20I.%20Am.%20J.%20Police%20Sci.&amp;doi=10.2307%2F1147154&amp;volume=1&amp;pages=276-298&amp;publication_year=1930&amp;author=Locard%2CE">
                    Google Scholar</a> 
                </p></li><li data-counter="19."><p id="ref-CR19">Toothman, M. H. <i>et al.</i> Characterization of human DNA in environmental samples. <i>Forensic Sci. Int.</i> <b>178</b>, 7–15. <a href="https://doi.org/10.1016/j.forsciint.2008.01.016" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1016/j.forsciint.2008.01.016">https://doi.org/10.1016/j.forsciint.2008.01.016</a> (2008).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.forsciint.2008.01.016" data-track-item_id="10.1016/j.forsciint.2008.01.016" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.forsciint.2008.01.016" aria-label="Article reference 19" data-doi="10.1016/j.forsciint.2008.01.016">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://interjectedfuture.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXlslOqt7s%3D" aria-label="CAS reference 19">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18420364" aria-label="PubMed reference 19">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 19" href="http://scholar.google.com/scholar_lookup?&amp;title=Characterization%20of%20human%20DNA%20in%20environmental%20samples&amp;journal=Forensic%20Sci.%20Int.&amp;doi=10.1016%2Fj.forsciint.2008.01.016&amp;volume=178&amp;pages=7-15&amp;publication_year=2008&amp;author=Toothman%2CMH">
                    Google Scholar</a> 
                </p></li><li data-counter="20."><p id="ref-CR20">Amankwaa, A. O. &amp; McCartney, C. The effectiveness of the current use of forensic DNA in criminal investigations in England and Wales. <i>Wiley Interdiscip. Rev. Forensic Sci.</i> <b>3</b>, e1414. <a href="https://doi.org/10.1002/wfs2.1414" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1002/wfs2.1414">https://doi.org/10.1002/wfs2.1414</a> (2021).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1002/wfs2.1414" data-track-item_id="10.1002/wfs2.1414" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1002%2Fwfs2.1414" aria-label="Article reference 20" data-doi="10.1002/wfs2.1414">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://interjectedfuture.com/articles/cas-redirect/1:CAS:528:DC%2BB38Xhslaqs7c%3D" aria-label="CAS reference 20">CAS</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 20" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20effectiveness%20of%20the%20current%20use%20of%20forensic%20DNA%20in%20criminal%20investigations%20in%20England%20and%20Wales&amp;journal=Wiley%20Interdiscip.%20Rev.%20Forensic%20Sci.&amp;doi=10.1002%2Fwfs2.1414&amp;volume=3&amp;publication_year=2021&amp;author=Amankwaa%2CAO&amp;author=McCartney%2CC">
                    Google Scholar</a> 
                </p></li><li data-counter="21."><p id="ref-CR21">Secretary of State for the Home Department (2013). <i>Serious and Organised Crime Strategy.</i>UK<i>:</i> The Stationery Office. <a href="https://www.gov.uk/government/publications/serious-organised-crime-strategy" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="https://www.gov.uk/government/publications/serious-organised-crime-strategy">https://www.gov.uk/government/publications/serious-organised-crime-strategy</a>.</p></li><li data-counter="22."><p id="ref-CR22">Watkins, D. R. L., Myers, D., Xavier, H. E. &amp; Marciano, M. A. Revisiting single cell analysis in forensic science. <i>Sci. Rep.</i> <b>11</b>, 7054. <a href="https://doi.org/10.1038/s41598-021-86271-6" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1038/s41598-021-86271-6">https://doi.org/10.1038/s41598-021-86271-6</a> (2021).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41598-021-86271-6" data-track-item_id="10.1038/s41598-021-86271-6" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41598-021-86271-6" aria-label="Article reference 22" data-doi="10.1038/s41598-021-86271-6">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="ads reference" data-track-action="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2021NatSR..11.7054W" aria-label="ADS reference 22">ADS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://interjectedfuture.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXnvVagu7s%3D" aria-label="CAS reference 22">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33782417" aria-label="PubMed reference 22">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8007698" aria-label="PubMed Central reference 22">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 22" href="http://scholar.google.com/scholar_lookup?&amp;title=Revisiting%20single%20cell%20analysis%20in%20forensic%20science&amp;journal=Sci.%20Rep.&amp;doi=10.1038%2Fs41598-021-86271-6&amp;volume=11&amp;publication_year=2021&amp;author=Watkins%2CDRL&amp;author=Myers%2CD&amp;author=Xavier%2CHE&amp;author=Marciano%2CMA">
                    Google Scholar</a> 
                </p></li><li data-counter="23."><p id="ref-CR23">van Oorschot, R. A., Ballantyne, K. N. &amp; Mitchell, R. J. Forensic trace DNA: A review. <i>Investig. Genet.</i> <b>1</b>, 14. <a href="https://doi.org/10.1186/2041-2223-1-14" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1186/2041-2223-1-14">https://doi.org/10.1186/2041-2223-1-14</a> (2010).</p><p><a data-track="click_references" rel="noopener" data-track-label="10.1186/2041-2223-1-14" data-track-item_id="10.1186/2041-2223-1-14" data-track-value="article reference" data-track-action="article reference" href="https://link.springer.com/doi/10.1186/2041-2223-1-14" aria-label="Article reference 23" data-doi="10.1186/2041-2223-1-14">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://interjectedfuture.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXhs1WiurjI" aria-label="CAS reference 23">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21122102" aria-label="PubMed reference 23">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3012025" aria-label="PubMed Central reference 23">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 23" href="http://scholar.google.com/scholar_lookup?&amp;title=Forensic%20trace%20DNA%3A%20A%20review&amp;journal=Investig.%20Genet.&amp;doi=10.1186%2F2041-2223-1-14&amp;volume=1&amp;publication_year=2010&amp;author=Oorschot%2CRA&amp;author=Ballantyne%2CKN&amp;author=Mitchell%2CRJ">
                    Google Scholar</a> 
                </p></li><li data-counter="24."><p id="ref-CR24">van Oorschot, R. A. H., Szkuta, B., Meakin, G. E., Kokshoorn, B. &amp; Goray, M. DNA transfer in forensic science: A review. <i>Forensic Sci. Int. Genet.</i> <b>38</b>, 140–166. <a href="https://doi.org/10.1016/j.fsigen.2018.10.014" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1016/j.fsigen.2018.10.014">https://doi.org/10.1016/j.fsigen.2018.10.014</a> (2019).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.fsigen.2018.10.014" data-track-item_id="10.1016/j.fsigen.2018.10.014" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.fsigen.2018.10.014" aria-label="Article reference 24" data-doi="10.1016/j.fsigen.2018.10.014">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://interjectedfuture.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXitVygs7rJ" aria-label="CAS reference 24">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30399535" aria-label="PubMed reference 24">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 24" href="http://scholar.google.com/scholar_lookup?&amp;title=DNA%20transfer%20in%20forensic%20science%3A%20A%20review&amp;journal=Forensic%20Sci.%20Int.%20Genet.&amp;doi=10.1016%2Fj.fsigen.2018.10.014&amp;volume=38&amp;pages=140-166&amp;publication_year=2019&amp;author=Oorschot%2CRAH&amp;author=Szkuta%2CB&amp;author=Meakin%2CGE&amp;author=Kokshoorn%2CB&amp;author=Goray%2CM">
                    Google Scholar</a> 
                </p></li><li data-counter="25."><p id="ref-CR25">Gill, P. DNA evidence and miscarriages of justice. <i>Forensic Sci. Int.</i> <b>294</b>, e1–e3. <a href="https://doi.org/10.1016/j.forsciint.2018.12.003" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1016/j.forsciint.2018.12.003">https://doi.org/10.1016/j.forsciint.2018.12.003</a> (2019).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.forsciint.2018.12.003" data-track-item_id="10.1016/j.forsciint.2018.12.003" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.forsciint.2018.12.003" aria-label="Article reference 25" data-doi="10.1016/j.forsciint.2018.12.003">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://interjectedfuture.com/articles/cas-redirect/1:STN:280:DC%2BB3cnltVajtQ%3D%3D" aria-label="CAS reference 25">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30579629" aria-label="PubMed reference 25">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 25" href="http://scholar.google.com/scholar_lookup?&amp;title=DNA%20evidence%20and%20miscarriages%20of%20justice&amp;journal=Forensic%20Sci.%20Int.&amp;doi=10.1016%2Fj.forsciint.2018.12.003&amp;volume=294&amp;pages=e1-e3&amp;publication_year=2019&amp;author=Gill%2CP">
                    Google Scholar</a> 
                </p></li><li data-counter="26."><p id="ref-CR26">Fonneløp, A. E., Johannessen, H., Egeland, T. &amp; Gill, P. Contamination during criminal investigation: Detecting police contamination and secondary DNA transfer from evidence bags. <i>Forensic Sci. Int. Genet.</i> <b>23</b>, 121–129. <a href="https://doi.org/10.1016/j.fsigen.2016.04.003" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1016/j.fsigen.2016.04.003">https://doi.org/10.1016/j.fsigen.2016.04.003</a> (2016).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.fsigen.2016.04.003" data-track-item_id="10.1016/j.fsigen.2016.04.003" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.fsigen.2016.04.003" aria-label="Article reference 26" data-doi="10.1016/j.fsigen.2016.04.003">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://interjectedfuture.com/articles/cas-redirect/1:CAS:528:DC%2BC28Xmtlektrw%3D" aria-label="CAS reference 26">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27100680" aria-label="PubMed reference 26">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 26" href="http://scholar.google.com/scholar_lookup?&amp;title=Contamination%20during%20criminal%20investigation%3A%20Detecting%20police%20contamination%20and%20secondary%20DNA%20transfer%20from%20evidence%20bags&amp;journal=Forensic%20Sci.%20Int.%20Genet.&amp;doi=10.1016%2Fj.fsigen.2016.04.003&amp;volume=23&amp;pages=121-129&amp;publication_year=2016&amp;author=Fonnel%C3%B8p%2CAE&amp;author=Johannessen%2CH&amp;author=Egeland%2CT&amp;author=Gill%2CP">
                    Google Scholar</a> 
                </p></li><li data-counter="27."><p id="ref-CR27">Reither, J. B. <i>et al.</i> Investigation into the prevalence of background DNA on flooring within houses and its transfer to a contacting surface. <i>Forensic Sci. Int.</i> <b>318</b>, 110563. <a href="https://doi.org/10.1016/j.forsciint.2020.110563" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1016/j.forsciint.2020.110563">https://doi.org/10.1016/j.forsciint.2020.110563</a> (2021).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.forsciint.2020.110563" data-track-item_id="10.1016/j.forsciint.2020.110563" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.forsciint.2020.110563" aria-label="Article reference 27" data-doi="10.1016/j.forsciint.2020.110563">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://interjectedfuture.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXit1OnurnE" aria-label="CAS reference 27">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33168422" aria-label="PubMed reference 27">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 27" href="http://scholar.google.com/scholar_lookup?&amp;title=Investigation%20into%20the%20prevalence%20of%20background%20DNA%20on%20flooring%20within%20houses%20and%20its%20transfer%20to%20a%20contacting%20surface&amp;journal=Forensic%20Sci.%20Int.&amp;doi=10.1016%2Fj.forsciint.2020.110563&amp;volume=318&amp;publication_year=2021&amp;author=Reither%2CJB">
                    Google Scholar</a> 
                </p></li><li data-counter="28."><p id="ref-CR28">Proff, C., Schmitt, C., Schneider, P. M., Foerster, G. &amp; Rothschild, M. A. Experiments on the DNA contamination risk via latent fingerprint brushes. <i>Int. Congress Ser.</i> <b>1288</b>, 601–603. <a href="https://doi.org/10.1016/j.ics.2005.10.053" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1016/j.ics.2005.10.053">https://doi.org/10.1016/j.ics.2005.10.053</a> (2006).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.ics.2005.10.053" data-track-item_id="10.1016/j.ics.2005.10.053" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.ics.2005.10.053" aria-label="Article reference 28" data-doi="10.1016/j.ics.2005.10.053">Article</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 28" href="http://scholar.google.com/scholar_lookup?&amp;title=Experiments%20on%20the%20DNA%20contamination%20risk%20via%20latent%20fingerprint%20brushes&amp;journal=Int.%20Congress%20Ser.&amp;doi=10.1016%2Fj.ics.2005.10.053&amp;volume=1288&amp;pages=601-603&amp;publication_year=2006&amp;author=Proff%2CC&amp;author=Schmitt%2CC&amp;author=Schneider%2CPM&amp;author=Foerster%2CG&amp;author=Rothschild%2CMA">
                    Google Scholar</a> 
                </p></li><li data-counter="29."><p id="ref-CR29">van den Berge, M., Ozcanhan, G., Zijlstra, S., Lindenbergh, A. &amp; Sijen, T. Prevalence of human cell material: DNA and RNA profiling of public and private objects and after activity scenarios. <i>Forensic Sci. Int. Genet.</i> <b>21</b>, 81–89. <a href="https://doi.org/10.1016/j.fsigen.2015.12.012" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1016/j.fsigen.2015.12.012">https://doi.org/10.1016/j.fsigen.2015.12.012</a> (2016).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.fsigen.2015.12.012" data-track-item_id="10.1016/j.fsigen.2015.12.012" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.fsigen.2015.12.012" aria-label="Article reference 29" data-doi="10.1016/j.fsigen.2015.12.012">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://interjectedfuture.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXitV2mt7rF" aria-label="CAS reference 29">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26736139" aria-label="PubMed reference 29">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 29" href="http://scholar.google.com/scholar_lookup?&amp;title=Prevalence%20of%20human%20cell%20material%3A%20DNA%20and%20RNA%20profiling%20of%20public%20and%20private%20objects%20and%20after%20activity%20scenarios&amp;journal=Forensic%20Sci.%20Int.%20Genet.&amp;doi=10.1016%2Fj.fsigen.2015.12.012&amp;volume=21&amp;pages=81-89&amp;publication_year=2016&amp;author=Berge%2CM&amp;author=Ozcanhan%2CG&amp;author=Zijlstra%2CS&amp;author=Lindenbergh%2CA&amp;author=Sijen%2CT">
                    Google Scholar</a> 
                </p></li><li data-counter="30."><p id="ref-CR30">Vandewoestyne, M. <i>et al.</i> Sources of DNA contamination and decontamination procedures in the forensic laboratory. <i>Int. J. Forensic Pract. Res.</i> <a href="https://doi.org/10.4172/2157-7145.S2-001" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.4172/2157-7145.S2-001">https://doi.org/10.4172/2157-7145.S2-001</a> (2011).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.4172/2157-7145.S2-001" data-track-item_id="10.4172/2157-7145.S2-001" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.4172%2F2157-7145.S2-001" aria-label="Article reference 30" data-doi="10.4172/2157-7145.S2-001">Article</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 30" href="http://scholar.google.com/scholar_lookup?&amp;title=Sources%20of%20DNA%20contamination%20and%20decontamination%20procedures%20in%20the%20forensic%20laboratory&amp;journal=Int.%20J.%20Forensic%20Pract.%20Res.&amp;doi=10.4172%2F2157-7145.S2-001&amp;publication_year=2011&amp;author=Vandewoestyne%2CM">
                    Google Scholar</a> 
                </p></li><li data-counter="31."><p id="ref-CR31">Witt, N. <i>et al.</i> An assessment of air as a source of DNA contamination encountered when performing PCR. <i>J. Biomol. Tech.</i> <b>20</b>, 236–240 (2009).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19949694" aria-label="PubMed reference 31">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2777341" aria-label="PubMed Central reference 31">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 31" href="http://scholar.google.com/scholar_lookup?&amp;title=An%20assessment%20of%20air%20as%20a%20source%20of%20DNA%20contamination%20encountered%20when%20performing%20PCR&amp;journal=J.%20Biomol.%20Tech.&amp;volume=20&amp;pages=236-240&amp;publication_year=2009&amp;author=Witt%2CN">
                    Google Scholar</a> 
                </p></li><li data-counter="32."><p id="ref-CR32">Gill, P., Benschop, C., Buckleton, J., Bleka, Ø. &amp; Taylor, D. A review of probabilistic genotyping systems: EuroForMix, DNAStatistX and STRmix™. <i>Genes (Basel)</i> <b>12</b>, 1559. <a href="https://doi.org/10.3390/genes12101559" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.3390/genes12101559">https://doi.org/10.3390/genes12101559</a> (2021).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.3390/genes12101559" data-track-item_id="10.3390/genes12101559" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.3390%2Fgenes12101559" aria-label="Article reference 32" data-doi="10.3390/genes12101559">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://interjectedfuture.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXisVart7rP" aria-label="CAS reference 32">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34680954" aria-label="PubMed reference 32">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 32" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20review%20of%20probabilistic%20genotyping%20systems%3A%20EuroForMix%2C%20DNAStatistX%20and%20STRmix%E2%84%A2&amp;journal=Genes%20%28Basel%29&amp;doi=10.3390%2Fgenes12101559&amp;volume=12&amp;publication_year=2021&amp;author=Gill%2CP&amp;author=Benschop%2CC&amp;author=Buckleton%2CJ&amp;author=Bleka%2C%C3%98&amp;author=Taylor%2CD">
                    Google Scholar</a> 
                </p></li><li data-counter="33."><p id="ref-CR33">Budowle B., M. T. R., Niezgoda S. J., Brown B. L. CODIS and PCR-based short tandem repeat loci: Law enforcement tools. In Second European symposium on human identification. 73–88 (1998).</p></li><li data-counter="34."><p id="ref-CR34">Bleka, Ø., Storvik, G. &amp; Gill, P. EuroForMix: An open source software based on a continuous model to evaluate STR DNA profiles from a mixture of contributors with artefacts. <i>Forensic Sci. Int. Genet.</i> <b>21</b>, 35–44. <a href="https://doi.org/10.1016/j.fsigen.2015.11.008" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1016/j.fsigen.2015.11.008">https://doi.org/10.1016/j.fsigen.2015.11.008</a> (2016).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.fsigen.2015.11.008" data-track-item_id="10.1016/j.fsigen.2015.11.008" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.fsigen.2015.11.008" aria-label="Article reference 34" data-doi="10.1016/j.fsigen.2015.11.008">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://interjectedfuture.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXitVyksr%2FI" aria-label="CAS reference 34">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26720812" aria-label="PubMed reference 34">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 34" href="http://scholar.google.com/scholar_lookup?&amp;title=EuroForMix%3A%20An%20open%20source%20software%20based%20on%20a%20continuous%20model%20to%20evaluate%20STR%20DNA%20profiles%20from%20a%20mixture%20of%20contributors%20with%20artefacts&amp;journal=Forensic%20Sci.%20Int.%20Genet.&amp;doi=10.1016%2Fj.fsigen.2015.11.008&amp;volume=21&amp;pages=35-44&amp;publication_year=2016&amp;author=Bleka%2C%C3%98&amp;author=Storvik%2CG&amp;author=Gill%2CP">
                    Google Scholar</a> 
                </p></li><li data-counter="35."><p id="ref-CR35">Gill, P., Bleka, Ø. &amp; Egeland, T. Does an English appeal court ruling increase the risks of miscarriages of justice when complex DNA profiles are searched against the national DNA database?. <i>Forensic Sci. Int. Genet.</i> <b>13</b>, 167–175. <a href="https://doi.org/10.1016/j.fsigen.2014.07.015" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1016/j.fsigen.2014.07.015">https://doi.org/10.1016/j.fsigen.2014.07.015</a> (2014).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.fsigen.2014.07.015" data-track-item_id="10.1016/j.fsigen.2014.07.015" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.fsigen.2014.07.015" aria-label="Article reference 35" data-doi="10.1016/j.fsigen.2014.07.015">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://interjectedfuture.com/articles/cas-redirect/1:STN:280:DC%2BC2M%2FksFKgtQ%3D%3D" aria-label="CAS reference 35">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25151459" aria-label="PubMed reference 35">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 35" href="http://scholar.google.com/scholar_lookup?&amp;title=Does%20an%20English%20appeal%20court%20ruling%20increase%20the%20risks%20of%20miscarriages%20of%20justice%20when%20complex%20DNA%20profiles%20are%20searched%20against%20the%20national%20DNA%20database%3F&amp;journal=Forensic%20Sci.%20Int.%20Genet.&amp;doi=10.1016%2Fj.fsigen.2014.07.015&amp;volume=13&amp;pages=167-175&amp;publication_year=2014&amp;author=Gill%2CP&amp;author=Bleka%2C%C3%98&amp;author=Egeland%2CT">
                    Google Scholar</a> 
                </p></li><li data-counter="36."><p id="ref-CR36">Taylor, D., Curran, J. M. &amp; Buckleton, J. Importance sampling allows H(d) true tests of highly discriminating DNA profiles. <i>Forensic Sci. Int. Genet.</i> <b>27</b>, 74–81. <a href="https://doi.org/10.1016/j.fsigen.2016.12.004" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1016/j.fsigen.2016.12.004">https://doi.org/10.1016/j.fsigen.2016.12.004</a> (2017).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.fsigen.2016.12.004" data-track-item_id="10.1016/j.fsigen.2016.12.004" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.fsigen.2016.12.004" aria-label="Article reference 36" data-doi="10.1016/j.fsigen.2016.12.004">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://interjectedfuture.com/articles/cas-redirect/1:CAS:528:DC%2BC28XitFCgtbbE" aria-label="CAS reference 36">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28012376" aria-label="PubMed reference 36">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 36" href="http://scholar.google.com/scholar_lookup?&amp;title=Importance%20sampling%20allows%20H%28d%29%20true%20tests%20of%20highly%20discriminating%20DNA%20profiles&amp;journal=Forensic%20Sci.%20Int.%20Genet.&amp;doi=10.1016%2Fj.fsigen.2016.12.004&amp;volume=27&amp;pages=74-81&amp;publication_year=2017&amp;author=Taylor%2CD&amp;author=Curran%2CJM&amp;author=Buckleton%2CJ">
                    Google Scholar</a> 
                </p></li><li data-counter="37."><p id="ref-CR37">Good, I. J. <i>Probability and the Weighing of Evidence</i>. Vol. 26 (C. Griffin London, 1950).</p></li><li data-counter="38."><p id="ref-CR38">Bleka, Ø., Bouzga, M., Fonneløp, A. &amp; Gill, P. dnamatch2: An open source software to carry out large scale database searches of mixtures using qualitative and quantitative models. <i>Forensic Sci. Int. Genet. Suppl. Ser.</i> <b>6</b>, e404–e406. <a href="https://doi.org/10.1016/j.fsigss.2017.09.160" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1016/j.fsigss.2017.09.160">https://doi.org/10.1016/j.fsigss.2017.09.160</a> (2017).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.fsigss.2017.09.160" data-track-item_id="10.1016/j.fsigss.2017.09.160" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.fsigss.2017.09.160" aria-label="Article reference 38" data-doi="10.1016/j.fsigss.2017.09.160">Article</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 38" href="http://scholar.google.com/scholar_lookup?&amp;title=dnamatch2%3A%20An%20open%20source%20software%20to%20carry%20out%20large%20scale%20database%20searches%20of%20mixtures%20using%20qualitative%20and%20quantitative%20models&amp;journal=Forensic%20Sci.%20Int.%20Genet.%20Suppl.%20Ser.&amp;doi=10.1016%2Fj.fsigss.2017.09.160&amp;volume=6&amp;pages=e404-e406&amp;publication_year=2017&amp;author=Bleka%2C%C3%98&amp;author=Bouzga%2CM&amp;author=Fonnel%C3%B8p%2CA&amp;author=Gill%2CP">
                    Google Scholar</a> 
                </p></li><li data-counter="39."><p id="ref-CR39">Abdel Hady, R. H., Thabet, H. Z., Ebrahem, N. E. &amp; Yassa, H. A. Thermal effects on DNA degradation in blood and seminal stains: Forensic view. <i>Acad. Forensic Pathol.</i> <b>11</b>, 7–23. <a href="https://doi.org/10.1177/1925362121998547" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1177/1925362121998547">https://doi.org/10.1177/1925362121998547</a> (2021).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1177/1925362121998547" data-track-item_id="10.1177/1925362121998547" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1177%2F1925362121998547" aria-label="Article reference 39" data-doi="10.1177/1925362121998547">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34040682" aria-label="PubMed reference 39">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8129487" aria-label="PubMed Central reference 39">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 39" href="http://scholar.google.com/scholar_lookup?&amp;title=Thermal%20effects%20on%20DNA%20degradation%20in%20blood%20and%20seminal%20stains%3A%20Forensic%20view&amp;journal=Acad.%20Forensic%20Pathol.&amp;doi=10.1177%2F1925362121998547&amp;volume=11&amp;pages=7-23&amp;publication_year=2021&amp;author=Abdel%20Hady%2CRH&amp;author=Thabet%2CHZ&amp;author=Ebrahem%2CNE&amp;author=Yassa%2CHA">
                    Google Scholar</a> 
                </p></li><li data-counter="40."><p id="ref-CR40">Hall, A., Sims, L. M. &amp; Ballantyne, J. Assessment of DNA damage induced by terrestrial UV irradiation of dried bloodstains: Forensic implications. <i>Forensic Sci. Int. Genet.</i> <b>8</b>, 24–32. <a href="https://doi.org/10.1016/j.fsigen.2013.06.010" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1016/j.fsigen.2013.06.010">https://doi.org/10.1016/j.fsigen.2013.06.010</a> (2014).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.fsigen.2013.06.010" data-track-item_id="10.1016/j.fsigen.2013.06.010" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.fsigen.2013.06.010" aria-label="Article reference 40" data-doi="10.1016/j.fsigen.2013.06.010">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://interjectedfuture.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXitVWntrjM" aria-label="CAS reference 40">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24315585" aria-label="PubMed reference 40">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 40" href="http://scholar.google.com/scholar_lookup?&amp;title=Assessment%20of%20DNA%20damage%20induced%20by%20terrestrial%20UV%20irradiation%20of%20dried%20bloodstains%3A%20Forensic%20implications&amp;journal=Forensic%20Sci.%20Int.%20Genet.&amp;doi=10.1016%2Fj.fsigen.2013.06.010&amp;volume=8&amp;pages=24-32&amp;publication_year=2014&amp;author=Hall%2CA&amp;author=Sims%2CLM&amp;author=Ballantyne%2CJ">
                    Google Scholar</a> 
                </p></li><li data-counter="41."><p id="ref-CR41">Asari, M. <i>et al.</i> Assessment of DNA degradation of buccal cells under humid conditions and DNA repair by DOP-PCR using locked nucleic acids. <i>Leg. Med. (Tokyo)</i> <b>35</b>, 29–33. <a href="https://doi.org/10.1016/j.legalmed.2018.09.005" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1016/j.legalmed.2018.09.005">https://doi.org/10.1016/j.legalmed.2018.09.005</a> (2018).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.legalmed.2018.09.005" data-track-item_id="10.1016/j.legalmed.2018.09.005" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.legalmed.2018.09.005" aria-label="Article reference 41" data-doi="10.1016/j.legalmed.2018.09.005">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://interjectedfuture.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhslOgsrrE" aria-label="CAS reference 41">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30248610" aria-label="PubMed reference 41">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 41" href="http://scholar.google.com/scholar_lookup?&amp;title=Assessment%20of%20DNA%20degradation%20of%20buccal%20cells%20under%20humid%20conditions%20and%20DNA%20repair%20by%20DOP-PCR%20using%20locked%20nucleic%20acids&amp;journal=Leg.%20Med.%20%28Tokyo%29&amp;doi=10.1016%2Fj.legalmed.2018.09.005&amp;volume=35&amp;pages=29-33&amp;publication_year=2018&amp;author=Asari%2CM">
                    Google Scholar</a> 
                </p></li><li data-counter="42."><p id="ref-CR42">Goray, M. &amp; van Oorschot, R. A. The complexities of DNA transfer during a social setting. <i>Leg. Med. (Tokyo)</i> <b>17</b>, 82–91. <a href="https://doi.org/10.1016/j.legalmed.2014.10.003" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1016/j.legalmed.2014.10.003">https://doi.org/10.1016/j.legalmed.2014.10.003</a> (2015).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.legalmed.2014.10.003" data-track-item_id="10.1016/j.legalmed.2014.10.003" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.legalmed.2014.10.003" aria-label="Article reference 42" data-doi="10.1016/j.legalmed.2014.10.003">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://interjectedfuture.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXhslGrsL%2FM" aria-label="CAS reference 42">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25454534" aria-label="PubMed reference 42">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 42" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20complexities%20of%20DNA%20transfer%20during%20a%20social%20setting&amp;journal=Leg.%20Med.%20%28Tokyo%29&amp;doi=10.1016%2Fj.legalmed.2014.10.003&amp;volume=17&amp;pages=82-91&amp;publication_year=2015&amp;author=Goray%2CM&amp;author=Oorschot%2CRA">
                    Google Scholar</a> 
                </p></li><li data-counter="43."><p id="ref-CR43">Szkuta, B. <i>et al.</i> Assessment of the transfer, persistence, prevalence and recovery of DNA traces from clothing: An inter-laboratory study on worn upper garments. <i>Forensic Sci. Int. Genet.</i> <b>42</b>, 56–68. <a href="https://doi.org/10.1016/j.fsigen.2019.06.011" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1016/j.fsigen.2019.06.011">https://doi.org/10.1016/j.fsigen.2019.06.011</a> (2019).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.fsigen.2019.06.011" data-track-item_id="10.1016/j.fsigen.2019.06.011" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.fsigen.2019.06.011" aria-label="Article reference 43" data-doi="10.1016/j.fsigen.2019.06.011">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://interjectedfuture.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXht1WlsL%2FE" aria-label="CAS reference 43">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31229887" aria-label="PubMed reference 43">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 43" href="http://scholar.google.com/scholar_lookup?&amp;title=Assessment%20of%20the%20transfer%2C%20persistence%2C%20prevalence%20and%20recovery%20of%20DNA%20traces%20from%20clothing%3A%20An%20inter-laboratory%20study%20on%20worn%20upper%20garments&amp;journal=Forensic%20Sci.%20Int.%20Genet.&amp;doi=10.1016%2Fj.fsigen.2019.06.011&amp;volume=42&amp;pages=56-68&amp;publication_year=2019&amp;author=Szkuta%2CB">
                    Google Scholar</a> 
                </p></li><li data-counter="44."><p id="ref-CR44">Taylor, D., Abarno, D., Rowe, E. &amp; Rask-Nielsen, L. Observations of DNA transfer within an operational Forensic Biology Laboratory. <i>Forensic Sci. Int. Genet.</i> <b>23</b>, 33–49. <a href="https://doi.org/10.1016/j.fsigen.2016.02.011" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1016/j.fsigen.2016.02.011">https://doi.org/10.1016/j.fsigen.2016.02.011</a> (2016).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.fsigen.2016.02.011" data-track-item_id="10.1016/j.fsigen.2016.02.011" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.fsigen.2016.02.011" aria-label="Article reference 44" data-doi="10.1016/j.fsigen.2016.02.011">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://interjectedfuture.com/articles/cas-redirect/1:CAS:528:DC%2BC28XktV2rsbk%3D" aria-label="CAS reference 44">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26977932" aria-label="PubMed reference 44">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 44" href="http://scholar.google.com/scholar_lookup?&amp;title=Observations%20of%20DNA%20transfer%20within%20an%20operational%20Forensic%20Biology%20Laboratory&amp;journal=Forensic%20Sci.%20Int.%20Genet.&amp;doi=10.1016%2Fj.fsigen.2016.02.011&amp;volume=23&amp;pages=33-49&amp;publication_year=2016&amp;author=Taylor%2CD&amp;author=Abarno%2CD&amp;author=Rowe%2CE&amp;author=Rask-Nielsen%2CL">
                    Google Scholar</a> 
                </p></li><li data-counter="45."><p id="ref-CR45">Lapointe, M., Rogic, A., Bourgoin, S., Jolicoeur, C. &amp; Séguin, D. Leading-edge forensic DNA analyses and the necessity of including crime scene investigators, police officers and technicians in a DNA elimination database. <i>Forensic Sci. Int. Genet.</i> <b>19</b>, 50–55. <a href="https://doi.org/10.1016/j.fsigen.2015.06.002" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1016/j.fsigen.2015.06.002">https://doi.org/10.1016/j.fsigen.2015.06.002</a> (2015).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.fsigen.2015.06.002" data-track-item_id="10.1016/j.fsigen.2015.06.002" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.fsigen.2015.06.002" aria-label="Article reference 45" data-doi="10.1016/j.fsigen.2015.06.002">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://interjectedfuture.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXhsFSiur3K" aria-label="CAS reference 45">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26117338" aria-label="PubMed reference 45">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 45" href="http://scholar.google.com/scholar_lookup?&amp;title=Leading-edge%20forensic%20DNA%20analyses%20and%20the%20necessity%20of%20including%20crime%20scene%20investigators%2C%20police%20officers%20and%20technicians%20in%20a%20DNA%20elimination%20database&amp;journal=Forensic%20Sci.%20Int.%20Genet.&amp;doi=10.1016%2Fj.fsigen.2015.06.002&amp;volume=19&amp;pages=50-55&amp;publication_year=2015&amp;author=Lapointe%2CM&amp;author=Rogic%2CA&amp;author=Bourgoin%2CS&amp;author=Jolicoeur%2CC&amp;author=S%C3%A9guin%2CD">
                    Google Scholar</a> 
                </p></li><li data-counter="46."><p id="ref-CR46">Mercer, C., Taylor, D., Henry, J. &amp; Linacre, A. DNA accumulation and transfer within an operational forensic exhibit storeroom. <i>Forensic Sci. Int. Genet.</i> <b>62</b>, 102799. <a href="https://doi.org/10.1016/j.fsigen.2022.102799" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1016/j.fsigen.2022.102799">https://doi.org/10.1016/j.fsigen.2022.102799</a> (2023).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.fsigen.2022.102799" data-track-item_id="10.1016/j.fsigen.2022.102799" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.fsigen.2022.102799" aria-label="Article reference 46" data-doi="10.1016/j.fsigen.2022.102799">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://interjectedfuture.com/articles/cas-redirect/1:CAS:528:DC%2BB38Xis1Oqtb7N" aria-label="CAS reference 46">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=36274409" aria-label="PubMed reference 46">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 46" href="http://scholar.google.com/scholar_lookup?&amp;title=DNA%20accumulation%20and%20transfer%20within%20an%20operational%20forensic%20exhibit%20storeroom&amp;journal=Forensic%20Sci.%20Int.%20Genet.&amp;doi=10.1016%2Fj.fsigen.2022.102799&amp;volume=62&amp;publication_year=2023&amp;author=Mercer%2CC&amp;author=Taylor%2CD&amp;author=Henry%2CJ&amp;author=Linacre%2CA">
                    Google Scholar</a> 
                </p></li><li data-counter="47."><p id="ref-CR47">van Oorschot, R. A. &amp; Jones, M. K. DNA fingerprints from fingerprints. <i>Nature</i> <b>387</b>, 767. <a href="https://doi.org/10.1038/42838" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1038/42838">https://doi.org/10.1038/42838</a> (1997).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/42838" data-track-item_id="10.1038/42838" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2F42838" aria-label="Article reference 47" data-doi="10.1038/42838">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="ads reference" data-track-action="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=1997Natur.387..767V" aria-label="ADS reference 47">ADS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://interjectedfuture.com/articles/cas-redirect/1:CAS:528:DyaK2sXktVOnsrs%3D" aria-label="CAS reference 47">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=9194555" aria-label="PubMed reference 47">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 47" href="http://scholar.google.com/scholar_lookup?&amp;title=DNA%20fingerprints%20from%20fingerprints&amp;journal=Nature&amp;doi=10.1038%2F42838&amp;volume=387&amp;publication_year=1997&amp;author=Oorschot%2CRA&amp;author=Jones%2CMK">
                    Google Scholar</a> 
                </p></li><li data-counter="48."><p id="ref-CR48">Lowe, A., Murray, C., Whitaker, J., Tully, G. &amp; Gill, P. The propensity of individuals to deposit DNA and secondary transfer of low level DNA from individuals to inert surfaces. <i>Forensic Sci. Int.</i> <b>129</b>, 25–34. <a href="https://doi.org/10.1016/s0379-0738(02)00207-4" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1016/s0379-0738(02)00207-4">https://doi.org/10.1016/s0379-0738(02)00207-4</a> (2002).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/s0379-0738(02)00207-4" data-track-item_id="10.1016/s0379-0738(02)00207-4" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fs0379-0738%2802%2900207-4" aria-label="Article reference 48" data-doi="10.1016/s0379-0738(02)00207-4">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://interjectedfuture.com/articles/cas-redirect/1:CAS:528:DC%2BD38XmvVCnt70%3D" aria-label="CAS reference 48">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=12230994" aria-label="PubMed reference 48">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 48" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20propensity%20of%20individuals%20to%20deposit%20DNA%20and%20secondary%20transfer%20of%20low%20level%20DNA%20from%20individuals%20to%20inert%20surfaces&amp;journal=Forensic%20Sci.%20Int.&amp;doi=10.1016%2Fs0379-0738%2802%2900207-4&amp;volume=129&amp;pages=25-34&amp;publication_year=2002&amp;author=Lowe%2CA&amp;author=Murray%2CC&amp;author=Whitaker%2CJ&amp;author=Tully%2CG&amp;author=Gill%2CP">
                    Google Scholar</a> 
                </p></li><li data-counter="49."><p id="ref-CR49">Phipps, M. &amp; Petricevic, S. The tendency of individuals to transfer DNA to handled items. <i>Forensic Sci. Int.</i> <b>168</b>, 162–168. <a href="https://doi.org/10.1016/j.forsciint.2006.07.010" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1016/j.forsciint.2006.07.010">https://doi.org/10.1016/j.forsciint.2006.07.010</a> (2007).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.forsciint.2006.07.010" data-track-item_id="10.1016/j.forsciint.2006.07.010" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.forsciint.2006.07.010" aria-label="Article reference 49" data-doi="10.1016/j.forsciint.2006.07.010">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://interjectedfuture.com/articles/cas-redirect/1:CAS:528:DC%2BD2sXkslCgtb8%3D" aria-label="CAS reference 49">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16930898" aria-label="PubMed reference 49">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 49" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20tendency%20of%20individuals%20to%20transfer%20DNA%20to%20handled%20items&amp;journal=Forensic%20Sci.%20Int.&amp;doi=10.1016%2Fj.forsciint.2006.07.010&amp;volume=168&amp;pages=162-168&amp;publication_year=2007&amp;author=Phipps%2CM&amp;author=Petricevic%2CS">
                    Google Scholar</a> 
                </p></li><li data-counter="50."><p id="ref-CR50">Fonneløp, A. E., Ramse, M., Egeland, T. &amp; Gill, P. The implications of shedder status and background DNA on direct and secondary transfer in an attack scenario. <i>Forensic Sci. Int. Genet.</i> <b>29</b>, 48–60. <a href="https://doi.org/10.1016/j.fsigen.2017.03.019" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1016/j.fsigen.2017.03.019">https://doi.org/10.1016/j.fsigen.2017.03.019</a> (2017).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.fsigen.2017.03.019" data-track-item_id="10.1016/j.fsigen.2017.03.019" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.fsigen.2017.03.019" aria-label="Article reference 50" data-doi="10.1016/j.fsigen.2017.03.019">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://interjectedfuture.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXlt1Chs7o%3D" aria-label="CAS reference 50">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28363123" aria-label="PubMed reference 50">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 50" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20implications%20of%20shedder%20status%20and%20background%20DNA%20on%20direct%20and%20secondary%20transfer%20in%20an%20attack%20scenario&amp;journal=Forensic%20Sci.%20Int.%20Genet.&amp;doi=10.1016%2Fj.fsigen.2017.03.019&amp;volume=29&amp;pages=48-60&amp;publication_year=2017&amp;author=Fonnel%C3%B8p%2CAE&amp;author=Ramse%2CM&amp;author=Egeland%2CT&amp;author=Gill%2CP">
                    Google Scholar</a> 
                </p></li><li data-counter="51."><p id="ref-CR51">Goray, M. &amp; van Oorschot, R. A. H. Shedder status: Exploring means of determination. <i>Sci. Justice</i> <b>61</b>, 391–400. <a href="https://doi.org/10.1016/j.scijus.2021.03.004" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1016/j.scijus.2021.03.004">https://doi.org/10.1016/j.scijus.2021.03.004</a> (2021).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.scijus.2021.03.004" data-track-item_id="10.1016/j.scijus.2021.03.004" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.scijus.2021.03.004" aria-label="Article reference 51" data-doi="10.1016/j.scijus.2021.03.004">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://interjectedfuture.com/articles/cas-redirect/1:STN:280:DC%2BB2c3is1GjtQ%3D%3D" aria-label="CAS reference 51">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34172128" aria-label="PubMed reference 51">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 51" href="http://scholar.google.com/scholar_lookup?&amp;title=Shedder%20status%3A%20Exploring%20means%20of%20determination&amp;journal=Sci.%20Justice&amp;doi=10.1016%2Fj.scijus.2021.03.004&amp;volume=61&amp;pages=391-400&amp;publication_year=2021&amp;author=Goray%2CM&amp;author=Oorschot%2CRAH">
                    Google Scholar</a> 
                </p></li><li data-counter="52."><p id="ref-CR52">Johannessen, H., Gill, P., Roseth, A. &amp; Fonneløp, A. E. Determination of shedder status: A comparison of two methods involving cell counting in fingerprints and the DNA analysis of handheld tubes. <i>Forensic Sci. Int. Genet.</i> <b>53</b>, 102541. <a href="https://doi.org/10.1016/j.fsigen.2021.102541" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1016/j.fsigen.2021.102541">https://doi.org/10.1016/j.fsigen.2021.102541</a> (2021).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.fsigen.2021.102541" data-track-item_id="10.1016/j.fsigen.2021.102541" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.fsigen.2021.102541" aria-label="Article reference 52" data-doi="10.1016/j.fsigen.2021.102541">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://interjectedfuture.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXht1ajs7fN" aria-label="CAS reference 52">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34090062" aria-label="PubMed reference 52">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 52" href="http://scholar.google.com/scholar_lookup?&amp;title=Determination%20of%20shedder%20status%3A%20A%20comparison%20of%20two%20methods%20involving%20cell%20counting%20in%20fingerprints%20and%20the%20DNA%20analysis%20of%20handheld%20tubes&amp;journal=Forensic%20Sci.%20Int.%20Genet.&amp;doi=10.1016%2Fj.fsigen.2021.102541&amp;volume=53&amp;publication_year=2021&amp;author=Johannessen%2CH&amp;author=Gill%2CP&amp;author=Roseth%2CA&amp;author=Fonnel%C3%B8p%2CAE">
                    Google Scholar</a> 
                </p></li><li data-counter="53."><p id="ref-CR53">Kanokwongnuwut, P., Martin, B., Kirkbride, K. P. &amp; Linacre, A. Shedding light on shedders. <i>Forensic Sci. Int. Genet.</i> <b>36</b>, 20–25. <a href="https://doi.org/10.1016/j.fsigen.2018.06.004" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1016/j.fsigen.2018.06.004">https://doi.org/10.1016/j.fsigen.2018.06.004</a> (2018).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.fsigen.2018.06.004" data-track-item_id="10.1016/j.fsigen.2018.06.004" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.fsigen.2018.06.004" aria-label="Article reference 53" data-doi="10.1016/j.fsigen.2018.06.004">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://interjectedfuture.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtFSqtLjI" aria-label="CAS reference 53">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29902671" aria-label="PubMed reference 53">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 53" href="http://scholar.google.com/scholar_lookup?&amp;title=Shedding%20light%20on%20shedders&amp;journal=Forensic%20Sci.%20Int.%20Genet.&amp;doi=10.1016%2Fj.fsigen.2018.06.004&amp;volume=36&amp;pages=20-25&amp;publication_year=2018&amp;author=Kanokwongnuwut%2CP&amp;author=Martin%2CB&amp;author=Kirkbride%2CKP&amp;author=Linacre%2CA">
                    Google Scholar</a> 
                </p></li><li data-counter="54."><p id="ref-CR54">Lacerenza, D. <i>et al.</i> A molecular exploration of human DNA/RNA co-extracted from the palmar surface of the hands and fingers. <i>Forensic Sci. Int. Genet.</i> <b>22</b>, 44–53. <a href="https://doi.org/10.1016/j.fsigen.2016.01.012" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1016/j.fsigen.2016.01.012">https://doi.org/10.1016/j.fsigen.2016.01.012</a> (2016).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.fsigen.2016.01.012" data-track-item_id="10.1016/j.fsigen.2016.01.012" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.fsigen.2016.01.012" aria-label="Article reference 54" data-doi="10.1016/j.fsigen.2016.01.012">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://interjectedfuture.com/articles/cas-redirect/1:CAS:528:DC%2BC28XhvVSluro%3D" aria-label="CAS reference 54">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26844918" aria-label="PubMed reference 54">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 54" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20molecular%20exploration%20of%20human%20DNA%2FRNA%20co-extracted%20from%20the%20palmar%20surface%20of%20the%20hands%20and%20fingers&amp;journal=Forensic%20Sci.%20Int.%20Genet.&amp;doi=10.1016%2Fj.fsigen.2016.01.012&amp;volume=22&amp;pages=44-53&amp;publication_year=2016&amp;author=Lacerenza%2CD">
                    Google Scholar</a> 
                </p></li><li data-counter="55."><p id="ref-CR55">Van Hoofstat, D. E., Deforce, D. L., Hubert De Pauw, I. P. &amp; Van den Eeckhout, E. G. DNA typing of fingerprints using capillary electrophoresis: Effect of dactyloscopic powders. <i>Electrophoresis</i> <b>20</b>, 2870–2876. <a href="https://doi.org/10.1002/(sici)1522-2683(19991001)20:14" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1002/(sici)1522-2683(19991001)20:14">https://doi.org/10.1002/(sici)1522-2683(19991001)20:14</a> (1999).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1002/(sici)1522-2683(19991001)20:14" data-track-item_id="10.1002/(sici)1522-2683(19991001)20:14" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1002%2F%28sici%291522-2683%2819991001%2920%3A14" aria-label="Article reference 55" data-doi="10.1002/(sici)1522-2683(19991001)20:14">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=10546822" aria-label="PubMed reference 55">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 55" href="http://scholar.google.com/scholar_lookup?&amp;title=DNA%20typing%20of%20fingerprints%20using%20capillary%20electrophoresis%3A%20Effect%20of%20dactyloscopic%20powders&amp;journal=Electrophoresis&amp;doi=10.1002%2F%28sici%291522-2683%2819991001%2920%3A14&amp;volume=20&amp;pages=2870-2876&amp;publication_year=1999&amp;author=Hoofstat%2CDE&amp;author=Deforce%2CDL&amp;author=Hubert%20De%20Pauw%2CIP&amp;author=Eeckhout%2CEG">
                    Google Scholar</a> 
                </p></li><li data-counter="56."><p id="ref-CR56">Castillo, J. A., Staton, S. J., Taylor, T. J., Herckes, P. &amp; Hayes, M. A. Exploring the feasibility of bioaerosol analysis as a novel fingerprinting technique. <i>Anal. Bioanal. Chem.</i> <b>403</b>, 15–26. <a href="https://doi.org/10.1007/s00216-012-5725-0" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1007/s00216-012-5725-0">https://doi.org/10.1007/s00216-012-5725-0</a> (2012).</p><p><a data-track="click_references" rel="noopener" data-track-label="10.1007/s00216-012-5725-0" data-track-item_id="10.1007/s00216-012-5725-0" data-track-value="article reference" data-track-action="article reference" href="https://link.springer.com/doi/10.1007/s00216-012-5725-0" aria-label="Article reference 56" data-doi="10.1007/s00216-012-5725-0">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://interjectedfuture.com/articles/cas-redirect/1:CAS:528:DC%2BC38XhvVais7c%3D" aria-label="CAS reference 56">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22311424" aria-label="PubMed reference 56">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3319984" aria-label="PubMed Central reference 56">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 56" href="http://scholar.google.com/scholar_lookup?&amp;title=Exploring%20the%20feasibility%20of%20bioaerosol%20analysis%20as%20a%20novel%20fingerprinting%20technique&amp;journal=Anal.%20Bioanal.%20Chem.&amp;doi=10.1007%2Fs00216-012-5725-0&amp;volume=403&amp;pages=15-26&amp;publication_year=2012&amp;author=Castillo%2CJA&amp;author=Staton%2CSJ&amp;author=Taylor%2CTJ&amp;author=Herckes%2CP&amp;author=Hayes%2CMA">
                    Google Scholar</a> 
                </p></li><li data-counter="57."><p id="ref-CR57">Tovey, E. R., Kemp, A. S., Almqvist, C., Sharland, A. &amp; Marks, G. B. Do immune responses to inhaled skin flakes modulate the expression of allergic disease?. <i>Clin. Exp. Allergy</i> <b>37</b>, 1199–1203. <a href="https://doi.org/10.1111/j.1365-2222.2007.02770.x" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1111/j.1365-2222.2007.02770.x">https://doi.org/10.1111/j.1365-2222.2007.02770.x</a> (2007).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1111/j.1365-2222.2007.02770.x" data-track-item_id="10.1111/j.1365-2222.2007.02770.x" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1111%2Fj.1365-2222.2007.02770.x" aria-label="Article reference 57" data-doi="10.1111/j.1365-2222.2007.02770.x">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://interjectedfuture.com/articles/cas-redirect/1:STN:280:DC%2BD2svjtFSkuw%3D%3D" aria-label="CAS reference 57">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17651150" aria-label="PubMed reference 57">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 57" href="http://scholar.google.com/scholar_lookup?&amp;title=Do%20immune%20responses%20to%20inhaled%20skin%20flakes%20modulate%20the%20expression%20of%20allergic%20disease%3F&amp;journal=Clin.%20Exp.%20Allergy&amp;doi=10.1111%2Fj.1365-2222.2007.02770.x&amp;volume=37&amp;pages=1199-1203&amp;publication_year=2007&amp;author=Tovey%2CER&amp;author=Kemp%2CAS&amp;author=Almqvist%2CC&amp;author=Sharland%2CA&amp;author=Marks%2CGB">
                    Google Scholar</a> 
                </p></li><li data-counter="58."><p id="ref-CR58">Evett, I. W., Gill, P. D., Jackson, G., Whitaker, J. &amp; Champod, C. Interpreting small quantities of DNA: The hierarchy of propositions and the use of Bayesian networks. <i>J. Forensic Sci.</i> <b>47</b>, 520–530 (2002).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1520/JFS15291J" data-track-item_id="10.1520/JFS15291J" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1520%2FJFS15291J" aria-label="Article reference 58" data-doi="10.1520/JFS15291J">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=12051330" aria-label="PubMed reference 58">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 58" href="http://scholar.google.com/scholar_lookup?&amp;title=Interpreting%20small%20quantities%20of%20DNA%3A%20The%20hierarchy%20of%20propositions%20and%20the%20use%20of%20Bayesian%20networks&amp;journal=J.%20Forensic%20Sci.&amp;doi=10.1520%2FJFS15291J&amp;volume=47&amp;pages=520-530&amp;publication_year=2002&amp;author=Evett%2CIW&amp;author=Gill%2CPD&amp;author=Jackson%2CG&amp;author=Whitaker%2CJ&amp;author=Champod%2CC">
                    Google Scholar</a> 
                </p></li><li data-counter="59."><p id="ref-CR59">Cook, R., Evett, I. W., Jackson, G., Jones, P. J. &amp; Lambert, J. A. A model for case assessment and interpretation. <i>Sci. Justice</i> <b>38</b>, 151–156. <a href="https://doi.org/10.1016/s1355-0306(98)72099-4" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1016/s1355-0306(98)72099-4">https://doi.org/10.1016/s1355-0306(98)72099-4</a> (1998).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/s1355-0306(98)72099-4" data-track-item_id="10.1016/s1355-0306(98)72099-4" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fs1355-0306%2898%2972099-4" aria-label="Article reference 59" data-doi="10.1016/s1355-0306(98)72099-4">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://interjectedfuture.com/articles/cas-redirect/1:STN:280:DyaK1M%2Fhs1yisA%3D%3D" aria-label="CAS reference 59">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=9800430" aria-label="PubMed reference 59">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 59" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20model%20for%20case%20assessment%20and%20interpretation&amp;journal=Sci.%20Justice&amp;doi=10.1016%2Fs1355-0306%2898%2972099-4&amp;volume=38&amp;pages=151-156&amp;publication_year=1998&amp;author=Cook%2CR&amp;author=Evett%2CIW&amp;author=Jackson%2CG&amp;author=Jones%2CPJ&amp;author=Lambert%2CJA">
                    Google Scholar</a> 
                </p></li><li data-counter="60."><p id="ref-CR60">Gill, P. <i>et al.</i> DNA commission of the International society for forensic genetics: Assessing the value of forensic biological evidence—Guidelines highlighting the importance of propositions. Part II: Evaluation of biological traces considering activity level propositions. <i>Forensic Sci. Int. Genet.</i> <b>44</b>, 102186. <a href="https://doi.org/10.1016/j.fsigen.2019.102186" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1016/j.fsigen.2019.102186">https://doi.org/10.1016/j.fsigen.2019.102186</a> (2020).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.fsigen.2019.102186" data-track-item_id="10.1016/j.fsigen.2019.102186" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.fsigen.2019.102186" aria-label="Article reference 60" data-doi="10.1016/j.fsigen.2019.102186">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://interjectedfuture.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXitVKqs73N" aria-label="CAS reference 60">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31677444" aria-label="PubMed reference 60">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 60" href="http://scholar.google.com/scholar_lookup?&amp;title=DNA%20commission%20of%20the%20International%20society%20for%20forensic%20genetics%3A%20Assessing%20the%20value%20of%20forensic%20biological%20evidence%E2%80%94Guidelines%20highlighting%20the%20importance%20of%20propositions.%20Part%20II%3A%20Evaluation%20of%20biological%20traces%20considering%20activity%20level%20propositions&amp;journal=Forensic%20Sci.%20Int.%20Genet.&amp;doi=10.1016%2Fj.fsigen.2019.102186&amp;volume=44&amp;publication_year=2020&amp;author=Gill%2CP">
                    Google Scholar</a> 
                </p></li><li data-counter="61."><p id="ref-CR61">Biedermann, A. &amp; Taroni, F. Bayesian networks for evaluating forensic DNA profiling evidence: A review and guide to literature. <i>Forensic Sci. Int. Genet.</i> <b>6</b>, 147–157. <a href="https://doi.org/10.1016/j.fsigen.2011.06.009" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1016/j.fsigen.2011.06.009">https://doi.org/10.1016/j.fsigen.2011.06.009</a> (2012).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.fsigen.2011.06.009" data-track-item_id="10.1016/j.fsigen.2011.06.009" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.fsigen.2011.06.009" aria-label="Article reference 61" data-doi="10.1016/j.fsigen.2011.06.009">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="https://interjectedfuture.com/articles/cas-redirect/1:CAS:528:DC%2BC38XitlOrs70%3D" aria-label="CAS reference 61">CAS</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21775236" aria-label="PubMed reference 61">PubMed</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 61" href="http://scholar.google.com/scholar_lookup?&amp;title=Bayesian%20networks%20for%20evaluating%20forensic%20DNA%20profiling%20evidence%3A%20A%20review%20and%20guide%20to%20literature&amp;journal=Forensic%20Sci.%20Int.%20Genet.&amp;doi=10.1016%2Fj.fsigen.2011.06.009&amp;volume=6&amp;pages=147-157&amp;publication_year=2012&amp;author=Biedermann%2CA&amp;author=Taroni%2CF">
                    Google Scholar</a> 
                </p></li><li data-counter="62."><p id="ref-CR62"><i>Making sense of Forensic Genetics</i>, <a href="https://senseaboutscience.org/activities/making-sense-of-forensic-genetics/" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="https://senseaboutscience.org/activities/making-sense-of-forensic-genetics/">https://senseaboutscience.org/activities/making-sense-of-forensic-genetics/</a> (2017).</p></li><li data-counter="63."><p id="ref-CR63">Whitmore, L. <i>et al.</i> Inadvertent human genomic bycatch and intentional capture raise beneficial applications and ethical concerns with environmental DNA. <i>Nat. Ecol. Evol.</i> <a href="https://doi.org/10.1038/s41559-023-02056-2" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1038/s41559-023-02056-2">https://doi.org/10.1038/s41559-023-02056-2</a> (2023).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41559-023-02056-2" data-track-item_id="10.1038/s41559-023-02056-2" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41559-023-02056-2" aria-label="Article reference 63" data-doi="10.1038/s41559-023-02056-2">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=37188965" aria-label="PubMed reference 63">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10250199" aria-label="PubMed Central reference 63">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 63" href="http://scholar.google.com/scholar_lookup?&amp;title=Inadvertent%20human%20genomic%20bycatch%20and%20intentional%20capture%20raise%20beneficial%20applications%20and%20ethical%20concerns%20with%20environmental%20DNA&amp;journal=Nat.%20Ecol.%20Evol.&amp;doi=10.1038%2Fs41559-023-02056-2&amp;publication_year=2023&amp;author=Whitmore%2CL">
                    Google Scholar</a> 
                </p></li><li data-counter="64."><p id="ref-CR64">InnovaPrep. AirPrep™ Cub Sampler User Guide models ACD210 and ACD200. (2021).</p></li><li data-counter="65."><p id="ref-CR65">Clare, E. L. <i>et al.</i> eDNAir: Proof of concept that animal DNA can be collected from air sampling. <i>PeerJ</i> <b>9</b>, e11030. <a href="https://doi.org/10.7717/peerj.11030" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.7717/peerj.11030">https://doi.org/10.7717/peerj.11030</a> (2021).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.7717/peerj.11030" data-track-item_id="10.7717/peerj.11030" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.7717%2Fpeerj.11030" aria-label="Article reference 65" data-doi="10.7717/peerj.11030">Article</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33850648" aria-label="PubMed reference 65">PubMed</a> 
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8019316" aria-label="PubMed Central reference 65">PubMed Central</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 65" href="http://scholar.google.com/scholar_lookup?&amp;title=eDNAir%3A%20Proof%20of%20concept%20that%20animal%20DNA%20can%20be%20collected%20from%20air%20sampling&amp;journal=PeerJ&amp;doi=10.7717%2Fpeerj.11030&amp;volume=9&amp;publication_year=2021&amp;author=Clare%2CEL">
                    Google Scholar</a> 
                </p></li><li data-counter="66."><p id="ref-CR66">Duhaime M. B., Cable R. (2019). <i>DNA Extraction from Filters using QIAgen DNeasy and QIAshredder V.1.</i> <a href="https://doi.org/10.17504/protocols.io.jx5cpq6" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.17504/protocols.io.jx5cpq6">https://doi.org/10.17504/protocols.io.jx5cpq6</a>.</p></li></ol><p><a data-track="click" data-track-action="download citation references" data-track-label="link" rel="nofollow" href="https://citation-needed.springer.com/v2/references/10.1038/s41598-023-46151-7?format=refman&amp;flavour=references">Download references<svg width="16" height="16" focusable="false" role="img" aria-hidden="true"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-download-medium"></use></svg></a></p></div></div></div></section></div><section data-title="Acknowledgements"><div id="Ack1-section"><h2 id="Ack1">Acknowledgements</h2><p>We would like to acknowledge the volunteers that participated in this study.</p></div></section><section aria-labelledby="author-information" data-title="Author information"><div id="author-information-section"><h2 id="author-information">Author information</h2><div id="author-information-content"><h3 id="affiliations">Authors and Affiliations</h3><ol><li id="Aff1"><p>Department of Forensic Sciences, Oslo University Hospital, Oslo, Norway</p><p>Chiara Fantinato, Ane Elida Fonneløp, Øyvind Bleka, Magnus Dehli Vigeland &amp; Peter Gill</p></li><li id="Aff2"><p>Department of Forensic Medicine, Institute of Clinical Medicine, University of Oslo, Oslo, Norway</p><p>Chiara Fantinato &amp; Peter Gill</p></li><li id="Aff3"><p>Centre for Ecological and Evolutionary Synthesis (CEES), Department of Biosciences, University of Oslo, Oslo, Norway</p><p>Ane Elida Fonneløp</p></li></ol><h3 id="contributions">Contributions</h3><p>Conceptualization: C.F., A.E.F., P.G.; Data curation: C.F., A.E.F., P.G., Ø.B.; Formal analysis: C.F., P.G., Ø.B.; Investigation: C.F.; Methodology: C.F., A.E.F., P.G., Ø.B.; Software: P.G., Ø.B., M.D.V.; Supervision: A.E.F., P.G.; Project administration: A.E.F.; Writing original draft: C.F.; Writing-review &amp; editing: C.F., A.E.F., Ø.B., M.D.V., P.G.</p><h3 id="corresponding-author">Corresponding author</h3><p id="corresponding-author-list">Correspondence to
                <a id="corresp-c1" href="mailto:chifan@ous-hf.no">Chiara Fantinato</a>.</p></div></div></section><section data-title="Ethics declarations"><div id="ethics-section"><h2 id="ethics">Ethics declarations</h2><div id="ethics-content">
              
                <h3 id="FPar1">Competing interests</h3>
                <p>The authors declare no competing interests.</p>
              
            </div></div></section><section data-title="Additional information"><div id="additional-information-section"><h2 id="additional-information">Additional information</h2><div id="additional-information-content"><h3>Publisher&#39;s note</h3><p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div></div></section><section data-title="Supplementary Information"><div id="Sec23-section"><h2 id="Sec23">Supplementary Information</h2></div></section><section data-title="Rights and permissions"><div id="rightslink-section"><h2 id="rightslink">Rights and permissions</h2><div id="rightslink-content">
                <p><b>Open Access</b>  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article&#39;s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article&#39;s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>.</p>
              <p><a data-track="click" data-track-action="view rights and permissions" data-track-label="link" href="https://s100.copyright.com/AppDispatchServlet?title=The%20invisible%20witness%3A%20air%20and%20dust%20as%20DNA%20evidence%20of%20human%20occupancy%20in%20indoor%20premises&amp;author=Chiara%20Fantinato%20et%20al&amp;contentID=10.1038%2Fs41598-023-46151-7&amp;copyright=The%20Author%28s%29&amp;publication=2045-2322&amp;publicationDate=2023-11-04&amp;publisherName=SpringerNature&amp;orderBeanReset=true&amp;oa=CC%20BY">Reprints and permissions</a></p></div></div></section><section aria-labelledby="article-info" data-title="About this article"><div id="article-info-section"><h2 id="article-info">About this article</h2><div id="article-info-content"><div><p><a data-crossmark="10.1038/s41598-023-46151-7" target="_blank" rel="noopener" href="https://crossmark.crossref.org/dialog/?doi=10.1038/s41598-023-46151-7" data-track="click" data-track-action="Click Crossmark" data-track-label="link" data-test="crossmark"><img loading="lazy" width="57" height="81" alt="Check for updates. Verify currency and authenticity via CrossMark" src="data:image/svg+xml;base64,<svg height="81" width="57" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="m17.35 35.45 21.3-14.2v-17.03h-21.3" fill="#989898"/><path d="m38.65 35.45-21.3-14.2v-17.03h21.3" fill="#747474"/><path d="m28 .5c-12.98 0-23.5 10.52-23.5 23.5s10.52 23.5 23.5 23.5 23.5-10.52 23.5-23.5c0-6.23-2.48-12.21-6.88-16.62-4.41-4.4-10.39-6.88-16.62-6.88zm0 41.25c-9.8 0-17.75-7.95-17.75-17.75s7.95-17.75 17.75-17.75 17.75 7.95 17.75 17.75c0 4.71-1.87 9.22-5.2 12.55s-7.84 5.2-12.55 5.2z" fill="#535353"/><path d="m41 36c-5.81 6.23-15.23 7.45-22.43 2.9-7.21-4.55-10.16-13.57-7.03-21.5l-4.92-3.11c-4.95 10.7-1.19 23.42 8.78 29.71 9.97 6.3 23.07 4.22 30.6-4.86z" fill="#9c9c9c"/><path d="m.2 58.45c0-.75.11-1.42.33-2.01s.52-1.09.91-1.5c.38-.41.83-.73 1.34-.94.51-.22 1.06-.32 1.65-.32.56 0 1.06.11 1.51.35.44.23.81.5 1.1.81l-.91 1.01c-.24-.24-.49-.42-.75-.56-.27-.13-.58-.2-.93-.2-.39 0-.73.08-1.05.23-.31.16-.58.37-.81.66-.23.28-.41.63-.53 1.04-.13.41-.19.88-.19 1.39 0 1.04.23 1.86.68 2.46.45.59 1.06.88 1.84.88.41 0 .77-.07 1.07-.23s.59-.39.85-.68l.91 1c-.38.43-.8.76-1.28.99-.47.22-1 .34-1.58.34-.59 0-1.13-.1-1.64-.31-.5-.2-.94-.51-1.31-.91-.38-.4-.67-.9-.88-1.48-.22-.59-.33-1.26-.33-2.02zm8.4-5.33h1.61v2.54l-.05 1.33c.29-.27.61-.51.96-.72s.76-.31 1.24-.31c.73 0 1.27.23 1.61.71.33.47.5 1.14.5 2.02v4.31h-1.61v-4.1c0-.57-.08-.97-.25-1.21-.17-.23-.45-.35-.83-.35-.3 0-.56.08-.79.22-.23.15-.49.36-.78.64v4.8h-1.61zm7.37 6.45c0-.56.09-1.06.26-1.51.18-.45.42-.83.71-1.14.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.36c.07.62.29 1.1.65 1.44.36.33.82.5 1.38.5.29 0 .57-.04.83-.13s.51-.21.76-.37l.55 1.01c-.33.21-.69.39-1.09.53-.41.14-.83.21-1.26.21-.48 0-.92-.08-1.34-.25-.41-.16-.76-.4-1.07-.7-.31-.31-.55-.69-.72-1.13-.18-.44-.26-.95-.26-1.52zm4.6-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.07.45-.31.29-.5.73-.58 1.3zm2.5.62c0-.57.09-1.08.28-1.53.18-.44.43-.82.75-1.13s.69-.54 1.1-.71c.42-.16.85-.24 1.31-.24.45 0 .84.08 1.17.23s.61.34.85.57l-.77 1.02c-.19-.16-.38-.28-.56-.37-.19-.09-.39-.14-.61-.14-.56 0-1.01.21-1.35.63-.35.41-.52.97-.52 1.67 0 .69.17 1.24.51 1.66.34.41.78.62 1.32.62.28 0 .54-.06.78-.17.24-.12.45-.26.64-.42l.67 1.03c-.33.29-.69.51-1.08.65-.39.15-.78.23-1.18.23-.46 0-.9-.08-1.31-.24-.4-.16-.75-.39-1.05-.7s-.53-.69-.7-1.13c-.17-.45-.25-.96-.25-1.53zm6.91-6.45h1.58v6.17h.05l2.54-3.16h1.77l-2.35 2.8 2.59 4.07h-1.75l-1.77-2.98-1.08 1.23v1.75h-1.58zm13.69 1.27c-.25-.11-.5-.17-.75-.17-.58 0-.87.39-.87 1.16v.75h1.34v1.27h-1.34v5.6h-1.61v-5.6h-.92v-1.2l.92-.07v-.72c0-.35.04-.68.13-.98.08-.31.21-.57.4-.79s.42-.39.71-.51c.28-.12.63-.18 1.04-.18.24 0 .48.02.69.07.22.05.41.1.57.17zm.48 5.18c0-.57.09-1.08.27-1.53.17-.44.41-.82.72-1.13.3-.31.65-.54 1.04-.71.39-.16.8-.24 1.23-.24s.84.08 1.24.24c.4.17.74.4 1.04.71s.54.69.72 1.13c.19.45.28.96.28 1.53s-.09 1.08-.28 1.53c-.18.44-.42.82-.72 1.13s-.64.54-1.04.7-.81.24-1.24.24-.84-.08-1.23-.24-.74-.39-1.04-.7c-.31-.31-.55-.69-.72-1.13-.18-.45-.27-.96-.27-1.53zm1.65 0c0 .69.14 1.24.43 1.66.28.41.68.62 1.18.62.51 0 .9-.21 1.19-.62.29-.42.44-.97.44-1.66 0-.7-.15-1.26-.44-1.67-.29-.42-.68-.63-1.19-.63-.5 0-.9.21-1.18.63-.29.41-.43.97-.43 1.67zm6.48-3.44h1.33l.12 1.21h.05c.24-.44.54-.79.88-1.02.35-.24.7-.36 1.07-.36.32 0 .59.05.78.14l-.28 1.4-.33-.09c-.11-.01-.23-.02-.38-.02-.27 0-.56.1-.86.31s-.55.58-.77 1.1v4.2h-1.61zm-47.87 15h1.61v4.1c0 .57.08.97.25 1.2.17.24.44.35.81.35.3 0 .57-.07.8-.22.22-.15.47-.39.73-.73v-4.7h1.61v6.87h-1.32l-.12-1.01h-.04c-.3.36-.63.64-.98.86-.35.21-.76.32-1.24.32-.73 0-1.27-.24-1.61-.71-.33-.47-.5-1.14-.5-2.02zm9.46 7.43v2.16h-1.61v-9.59h1.33l.12.72h.05c.29-.24.61-.45.97-.63.35-.17.72-.26 1.1-.26.43 0 .81.08 1.15.24.33.17.61.4.84.71.24.31.41.68.53 1.11.13.42.19.91.19 1.44 0 .59-.09 1.11-.25 1.57-.16.47-.38.85-.65 1.16-.27.32-.58.56-.94.73-.35.16-.72.25-1.1.25-.3 0-.6-.07-.9-.2s-.59-.31-.87-.56zm0-2.3c.26.22.5.37.73.45.24.09.46.13.66.13.46 0 .84-.2 1.15-.6.31-.39.46-.98.46-1.77 0-.69-.12-1.22-.35-1.61-.23-.38-.61-.57-1.13-.57-.49 0-.99.26-1.52.77zm5.87-1.69c0-.56.08-1.06.25-1.51.16-.45.37-.83.65-1.14.27-.3.58-.54.93-.71s.71-.25 1.08-.25c.39 0 .73.07 1 .2.27.14.54.32.81.55l-.06-1.1v-2.49h1.61v9.88h-1.33l-.11-.74h-.06c-.25.25-.54.46-.88.64-.33.18-.69.27-1.06.27-.87 0-1.56-.32-2.07-.95s-.76-1.51-.76-2.65zm1.67-.01c0 .74.13 1.31.4 1.7.26.38.65.58 1.15.58.51 0 .99-.26 1.44-.77v-3.21c-.24-.21-.48-.36-.7-.45-.23-.08-.46-.12-.7-.12-.45 0-.82.19-1.13.59-.31.39-.46.95-.46 1.68zm6.35 1.59c0-.73.32-1.3.97-1.71.64-.4 1.67-.68 3.08-.84 0-.17-.02-.34-.07-.51-.05-.16-.12-.3-.22-.43s-.22-.22-.38-.3c-.15-.06-.34-.1-.58-.1-.34 0-.68.07-1 .2s-.63.29-.93.47l-.59-1.08c.39-.24.81-.45 1.28-.63.47-.17.99-.26 1.54-.26.86 0 1.51.25 1.93.76s.63 1.25.63 2.21v4.07h-1.32l-.12-.76h-.05c-.3.27-.63.48-.98.66s-.73.27-1.14.27c-.61 0-1.1-.19-1.48-.56-.38-.36-.57-.85-.57-1.46zm1.57-.12c0 .3.09.53.27.67.19.14.42.21.71.21.28 0 .54-.07.77-.2s.48-.31.73-.56v-1.54c-.47.06-.86.13-1.18.23-.31.09-.57.19-.76.31s-.33.25-.41.4c-.09.15-.13.31-.13.48zm6.29-3.63h-.98v-1.2l1.06-.07.2-1.88h1.34v1.88h1.75v1.27h-1.75v3.28c0 .8.32 1.2.97 1.2.12 0 .24-.01.37-.04.12-.03.24-.07.34-.11l.28 1.19c-.19.06-.4.12-.64.17-.23.05-.49.08-.76.08-.4 0-.74-.06-1.02-.18-.27-.13-.49-.3-.67-.52-.17-.21-.3-.48-.37-.78-.08-.3-.12-.64-.12-1.01zm4.36 2.17c0-.56.09-1.06.27-1.51s.41-.83.71-1.14c.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.37c.08.62.29 1.1.65 1.44.36.33.82.5 1.38.5.3 0 .58-.04.84-.13.25-.09.51-.21.76-.37l.54 1.01c-.32.21-.69.39-1.09.53s-.82.21-1.26.21c-.47 0-.92-.08-1.33-.25-.41-.16-.77-.4-1.08-.7-.3-.31-.54-.69-.72-1.13-.17-.44-.26-.95-.26-1.52zm4.61-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.08.45-.31.29-.5.73-.57 1.3zm3.01 2.23c.31.24.61.43.92.57.3.13.63.2.98.2.38 0 .65-.08.83-.23s.27-.35.27-.6c0-.14-.05-.26-.13-.37-.08-.1-.2-.2-.34-.28-.14-.09-.29-.16-.47-.23l-.53-.22c-.23-.09-.46-.18-.69-.3-.23-.11-.44-.24-.62-.4s-.33-.35-.45-.55c-.12-.21-.18-.46-.18-.75 0-.61.23-1.1.68-1.49.44-.38 1.06-.57 1.83-.57.48 0 .91.08 1.29.25s.71.36.99.57l-.74.98c-.24-.17-.49-.32-.73-.42-.25-.11-.51-.16-.78-.16-.35 0-.6.07-.76.21-.17.15-.25.33-.25.54 0 .14.04.26.12.36s.18.18.31.26c.14.07.29.14.46.21l.54.19c.23.09.47.18.7.29s.44.24.64.4c.19.16.34.35.46.58.11.23.17.5.17.82 0 .3-.06.58-.17.83-.12.26-.29.48-.51.68-.23.19-.51.34-.84.45-.34.11-.72.17-1.15.17-.48 0-.95-.09-1.41-.27-.46-.19-.86-.41-1.2-.68z" fill="#535353"/></g></svg>"/></a></p><div><h3 id="citeas">Cite this article</h3><p>Fantinato, C., Fonneløp, A.E., Bleka, Ø. <i>et al.</i> The invisible witness: air and dust as DNA evidence of human occupancy in indoor premises.
                    <i>Sci Rep</i> <b>13</b>, 19059 (2023). https://doi.org/10.1038/s41598-023-46151-7</p><p><a data-test="citation-link" data-track="click" data-track-action="download article citation" data-track-label="link" data-track-external="" rel="nofollow" href="https://citation-needed.springer.com/v2/references/10.1038/s41598-023-46151-7?format=refman&amp;flavour=citation">Download citation<svg width="16" height="16" focusable="false" role="img" aria-hidden="true"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-download-medium"></use></svg></a></p><ul data-test="publication-history"><li><p>Received<span>: </span><span><time datetime="2023-07-24">24 July 2023</time></span></p></li><li><p>Accepted<span>: </span><span><time datetime="2023-10-28">28 October 2023</time></span></p></li><li><p>Published<span>: </span><span><time datetime="2023-11-04">04 November 2023</time></span></p></li><li><p><abbr title="Digital Object Identifier">DOI</abbr><span>: </span><span>https://doi.org/10.1038/s41598-023-46151-7</span></p></li></ul></div></div></div></div></section>
            </div></div>
  </body>
</html>
