<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <title>James Routley | Feed</title>
    <link
      rel="stylesheet"
      type="text/css"
      href="../styles.css"
      media="screen"
    />
  </head>
  <body>
    <a href="/index.html">Back</a>
    <a href="https://elifesciences.org/articles/83291">Original</a>
    <h1>George Church lab creates ovarian granulosa-like cells from stem cells</h1>
    
    <div id="readability-page-1" class="page"><div>

                
            
            
                
                
                    <section id="abstract" data-behaviour="ArticleSection">

    <header>
      <h2>Abstract</h2>
    </header>

  <p>An in vitro model of human ovarian follicles would greatly benefit the study of female reproduction. Ovarian development requires the combination of germ cells and several types of somatic cells. Among these, granulosa cells play a key role in follicle formation and support for oogenesis. Whereas efficient protocols exist for generating human primordial germ cell-like cells (hPGCLCs) from human induced pluripotent stem cells (hiPSCs), a method of generating granulosa cells has been elusive. Here, we report that simultaneous overexpression of two transcription factors (TFs) can direct the differentiation of hiPSCs to granulosa-like cells. We elucidate the regulatory effects of several granulosa-related TFs and establish that overexpression of NR5A1 and either RUNX1 or RUNX2 is sufficient to generate granulosa-like cells. Our granulosa-like cells have transcriptomes similar to human fetal ovarian cells and recapitulate key ovarian phenotypes including follicle formation and steroidogenesis. When aggregated with hPGCLCs, our cells form ovary-like organoids (ovaroids) and support hPGCLC development from the premigratory to the gonadal stage as measured by induction of DAZL expression. This model system will provide unique opportunities for studying human ovarian biology and may enable the development of therapies for female reproductive health.</p>

</section>


                
                    <section id="sa0" data-behaviour="ArticleSection">

    <header>
      <h2>Editor&#39;s evaluation</h2>
    </header>

  <div>
      <p>This manuscript addresses a fundamental issue in ovarian biology of deriving granulosa cells from human iPS cells. These findings are important to treat female infertility in the future and may prove valuable in the Ob and Gyn clinical practice. The authors provide compelling evidence by developing and validating their model using in vitro ovaroids. This study provides a novel resource for transcriptomic signatures of ovarian somatic cells derived in vitro.</p>




      <p><span>
          <a href="https://doi.org/10.7554/eLife.83291.sa0">
        https://doi.org/10.7554/eLife.83291.sa0
          </a>
      </span></p><ul>
        <li>
          <a href="https://elifesciences.org/articles/83291#sa1">Decision letter</a>
        </li>
        <li>
          <a href="https://sciety.org/articles/activity/10.1101/2022.07.04.498717">Reviews on Sciety</a>
        </li>
        <li>
          <a href="https://elifesciences.org/about/peer-review">eLife&#39;s review process</a>
        </li>
      </ul>
  </div>

</section>


                
                    <section id="digest" data-behaviour="ArticleSection">

    <header>
      <h2>eLife digest</h2>
    </header>

  <div>
      <p>Ovaries are responsible for forming the eggs humans and other mammals need to reproduce. Once mature, the egg cell is released into the fallopian tube where it can be potentially fertilized by a sperm. Despite their crucial role, how eggs are made in the ovary is poorly understood. This is because ovaries are hard to access, making it difficult to conduct experiments on them.</p>
<p>To overcome this, researchers have built artificial ovaries in the laboratory using stem cells from the embryos of mice which can develop into all cell types in the adult body. By culturing these embryonic stem cells under special conditions, researchers can convert them in to the two main cell types of the developing ovary: germ cells which go on to form eggs, and granulosa cells which help eggs grow and mature. The resulting lab-grown ovary can make eggs that produce live mice when fertilized.</p>
<p>This approach has also been applied to human induced pluripotent stem cells (iPSCs), adult human cells which have been reprogrammed to a stem-like state. While this has produced human germ cells, generating human granulosa cells has been more challenging. Here, Pierson Smela, Kramme et al. show that activating a specific set of transcription factors (proteins that switch genes on or off) in iPSCs can make them transition to granulosa cells.</p>
<p>First, the team tested random combinations of 35 transcription factors which, based on previous literature and genetic data, were likely to play a role in the formation of granulosa cells. This led to the identification of a small number of factors that caused the human iPSCs to develop features and carry out roles seen in mature granulosa cells; this includes producing an important reproductive hormone and supporting the maturation of germ cells. Pierson Smela, Kramme et al. found that growing these granulosa-like cells together with germ cells (also generated via iPSCs) resulted in structures similar to ovarian follicles which help eggs develop.</p>
<p>These findings could help researchers build stable systems for studying how granulosa cells behave in human ovaries. This could lead to new insights about reproductive health.</p>





  </div>

</section>


                
                    <section id="s1" data-behaviour="ArticleSection">

    <header>
      <h2>Introduction</h2>
    </header>

  <div>
      <p>Oogenesis is the central process of female reproduction, yet less is understood about this process in humans relative to other model organisms. This is in part due to difficulties in obtaining samples of human fetal ovaries, and the lack of a suitable model system to study human ovarian development in vitro. The potential power of such a system is illustrated by a recent study in mice, which differentiated mouse embryonic stem cells into primordial germ cell-like cells (PGCLCs) and ovarian-like cells, and combined these cell types. Reconstituted follicles were capable of producing oocytes and live offspring (<a href="#bib78">Yoshino et al., 2021</a>). Using mouse fetal ovarian somatic cells, a similar system was published that allowed the development of human PGCLCs to the oogonia stage (<a href="#bib74">Yamashiro et al., 2018</a>). However, these mixed-species systems are inadequate for fully modeling human ovarian development. Clearly, the construction of fully human ovarian follicles from pluripotent stem cells could enable new advances in the study of female reproduction, epigenetics, and human development.</p>
<p>Ovarian development requires paracrine and juxtacrine interaction between cells from germline and somatic lineages (<a href="#bib19">Høyer et al., 2005</a>; <a href="#bib1">Albertini et al., 2001</a>; <a href="#bib57">Rodrigues et al., 2021</a>; <a href="#bib64">Su et al., 2004</a>; <a href="#bib31">Le Bouffant et al., 2010</a>; <a href="#bib51">Otsuka and Shimasaki, 2002</a>; <a href="#bib20">Hu et al., 2015</a>). During embryonic development, the first cells committed to the germline appear at roughly 2–3 weeks post-fertilization (wpf), when primordial germ cells (PGCs) are specified from the posterior epiblast in response to bone morphogenetic protein (BMP) signaling (<a href="#bib68">Tang et al., 2016</a>; <a href="#bib30">Lawson et al., 1999</a>; <a href="#bib24">Kee et al., 2006</a>; <a href="#bib18">Hayashi et al., 2011</a>). After specification, PGCs migrate to the developing gonad and undergo further development into oogonia (in female) or spermatogonia (in male) (<a href="#bib56">Richardson and Lehmann, 2010</a>; <a href="#bib14">Grimaldi and Raz, 2020</a>). The expression of the RNA-binding proteins DAZL and DDX4 (also known as VASA) begins in gonadal PGCs (<a href="#bib20">Hu et al., 2015</a>; <a href="#bib2">Anderson et al., 2007</a>; <a href="#bib44">Nicholls et al., 2019</a>), with DAZL identified as a key factor induced by the gonadal environment that commits PGCs to the germline (<a href="#bib44">Nicholls et al., 2019</a>), restricting pluripotency and inhibiting differentiation to somatic lineages (<a href="#bib5">Chen et al., 2014</a>). Notably, gonadal PGCs from Carnegie stage 23 primate embryos express mature markers such as DAZL and DDX4, and can complete spermatogenesis upon transplantation into a recipient testis (<a href="#bib7">Clark et al., 2017</a>), whereas earlier-stage PGCLCs cannot (<a href="#bib62">Sosa et al., 2018</a>).</p>
<p>Previous studies have provided efficient protocols for the differentiation of hiPSCs to DAZL-negative premigratory PGCLCs (<a href="#bib22">Irie et al., 2015</a>; <a href="#bib58">Sasaki et al., 2015</a>; <a href="#bib39">Mitsunaga et al., 2017</a>; <a href="#bib61">Sebastiano et al., 2021</a>; <a href="#bib24">Kee et al., 2006</a>). However, the only previously reported method that allows human primordial germ cell-like cells (hPGCLCs) to develop to later DAZL-positive stages in vitro is to aggregate them with mouse fetal ovarian (<a href="#bib74">Yamashiro et al., 2018</a>) or testicular (<a href="#bib21">Hwang et al., 2020</a>; <a href="#bib25">Kobayashi et al., 2022</a>) cells, which mimic the environment of the early gonad. Another study tested aggregation of hPGCLCs with rat neonatal testis cells (<a href="#bib36">Mall et al., 2020</a>), but only showed that the hPGCLCs could survive and did not assess maturation.</p>
<p>The somatic cells of the developing ovary provide crucial support for oogenesis. These cells originate from the WT1<i>-</i>positive intermediate mesoderm (<a href="#bib59">Sasaki et al., 2021</a>; <a href="#bib63">Stévant et al., 2019</a>), and initially exist in a bipotential state, capable of supporting either male or female development. In humans, sex-specific pathways begin to diverge at around 7 wpf (<a href="#bib17">Hartshorne et al., 2009</a>). In male mammals, the Y-chromosomal transcription factor (TF) SRY activates a gene regulatory network promoting testis formation and repressing ovary formation (<a href="#bib26">Koopman et al., 1991</a>). In the absence of SRY, the TF FOXL2 represses the testis factors, allowing ovarian development to proceed (<a href="#bib45">Nicol et al., 2018</a>; <a href="#bib52">Ottolenghi et al., 2005</a>). The ovarian somatic cells further differentiate into several lineages, including the granulosa cells which surround and support developing oocytes. Proper oogenesis depends on a complex interplay of paracrine signaling between somatic and germline cells (<a href="#bib64">Su et al., 2004</a>; <a href="#bib31">Le Bouffant et al., 2010</a>; <a href="#bib51">Otsuka and Shimasaki, 2002</a>; <a href="#bib53">Peng et al., 2013</a>). Granulosa cells additionally participate in endocrine signaling, producing estradiol and progesterone which regulate diverse reproductive functions (<a href="#bib10">Findlay et al., 2019</a>).</p>
<p>Although a few studies have attempted differentiating human pluripotent stem cells to granulosa-like cells through treatment with recombinant signaling proteins, none has resulted in an efficient method, nor has any previous study evaluated their ability to support germ cell development. For example, a study using spontaneous differentiation of hiPSCs produced a small fraction of AMHR2-positive granulosa-like cells which expressed some granulosa marker genes and produced estradiol (<a href="#bib35">Lipskind et al., 2018</a>). However, the efficiency of this spontaneous differentiation was low and not numerically reported. A related study used recombinant signaling proteins in a 12-day multistep protocol to direct the differentiation of human embryonic stem cells (hESCs) toward granulosa-like cells (<a href="#bib29">Lan et al., 2013</a>); again, the yield was low (12–36%), and although the cells upregulated some marker genes, the expression levels were much lower than observed in primary granulosa cells.</p>
<p>Manipulation of TFs allows reprogramming of cell identity and offers a rapid and efficient way to generate granulosa-like cells. Perhaps the best-known example of cellular reprogramming using TFs is the generation of iPSCs from somatic cells (<a href="#bib66">Takahashi and Yamanaka, 2006</a>). TF expression can also guide the differentiation of iPSCs into various somatic lineages, including neurons, fibroblasts, oligodendrocytes, and vascular endothelium (<a href="#bib43">Ng et al., 2021</a>). Furthermore, a recent study demonstrated that overexpression of the TFs NR5A1 and GATA4 is sufficient to reprogram human fibroblasts into Sertoli-like cells (<a href="#bib34">Liang et al., 2019</a>), the male developmental counterparts of granulosa-like cells. Therefore, we set out to determine a set of TFs that could be used for an efficient, scalable method of generating granulosa-like cells from hiPSCs. Our overall experimental workflow is shown in <a href="#fig1">Figure 1</a>.</p>
    <div id="fig1" data-variant="" data-behaviour="AssetNavigation AssetViewer ToggleableCaption" data-selector=".caption-text__body" data-asset-viewer-group="fig1" data-asset-viewer-uri="https://iiif.elifesciences.org/lax:83291%2Felife-83291-fig1-v2.tif/full/1500,/0/default.jpg" data-asset-viewer-width="1500" data-asset-viewer-height="479">
    
      
    
          <figure>
          
              <a href="https://iiif.elifesciences.org/lax:83291%2Felife-83291-fig1-v2.tif/full/1500,/0/default.jpg" target="_blank" rel="noopener noreferrer">
              <picture>
                  <source srcset="https://iiif.elifesciences.org/lax:83291%2Felife-83291-fig1-v2.tif/full/1234,/0/default.webp 2x, https://iiif.elifesciences.org/lax:83291%2Felife-83291-fig1-v2.tif/full/617,/0/default.webp 1x" type="image/webp"/>
                  <source srcset="https://iiif.elifesciences.org/lax:83291%2Felife-83291-fig1-v2.tif/full/1234,/0/default.jpg 2x, https://iiif.elifesciences.org/lax:83291%2Felife-83291-fig1-v2.tif/full/617,/0/default.jpg 1x" type="image/jpeg"/>
                  <img src="https://iiif.elifesciences.org/lax:83291%2Felife-83291-fig1-v2.tif/full/617,/0/default.jpg" alt=""/>
              </picture>
              </a>
          
          
          
          
          
              <figcaption>
          
                  <h6>Experimental workflow of the study.</h6>
                
                
                <p>First, barcoded transcription factor (TF) expression vectors were integrated into FOXL2-T2A-tdTomato reporter human induced pluripotent stem cells (hiPSCs). After induction of TF expression, cells positive for tdTomato and granulosa-related surface markers were sorted, and the barcodes were sequenced. The top TFs based on barcode enrichment were selected for further characterization by combinatorial screening and bulk RNA-seq. Next, monoclonal hiPSC lines were generated that inducibly express the top TFs (see <a href="#fig3sdata2">Figure 3—source data 2</a>) and generate granulosa-like cells with high efficiency (<a href="https://elifesciences.org/articles/83291/figures#fig3s2">Figure 3—figure supplement 2</a>). Granulosa-like cells from these lines were further evaluated for estradiol production in response to follicle-stimulating hormone (FSH). Finally, they were aggregated with human primordial germ cell-like cells (hPGCLCs) to form ovaroids. These ovaroids produced estradiol and progesterone, formed follicle-like structures, and supported hPGCLC maturation as measured by immunofluorescence microscopy and scRNA-seq.</p>
          
          
              </figcaption>
          
          
          
          
          </figure>
    
    
    </div>





  </div>

</section>


                
                    <section id="s2" data-behaviour="ArticleSection" data-initial-state="closed">

    <header>
      <h2>Results</h2>
    </header>

  <div>
      <section id="s2-1">

    <header>
      <h3>In silico prediction of granulosa cell-regulating TFs</h3>
    </header>

  <p>We began by predicting candidate TFs that could direct the differentiation of hiPSCs to granulosa-like cells. We first selected 21 TFs that were differentially expressed in granulosa cells compared to hESCs and early mesoderm, using previously published datasets for these cell types (<a href="#bib22">Irie et al., 2015</a>; <a href="#bib80">Zhang et al., 2018</a>). We also included five TFs based on previous developmental biology studies, mainly showing that these TFs were important for mouse ovarian development (<a href="#bib37">Manuylov et al., 2008</a>; <a href="#bib48">Niu and Spradling, 2020</a>; <a href="#bib71">Voronina et al., 2007</a>; <a href="#bib55">Richards, 2001</a>; <a href="#bib46">Nicol et al., 2019</a>). Finally, we identified nine additional TFs that we predicted to be upstream of the others on the list, based on a gene regulatory network analysis taking into account co-expression data as well as binding motifs (<a href="#bib27">Kramme et al., 2021a</a>). The list of TFs (35 in total) is in <a href="#fig2sdata2">Figure 2—source data 2</a>. Next, we assembled a PiggyBac library by cloning cDNAs into a barcoded destination plasmid (<a href="#bib28">Kramme et al., 2021b</a>). The TFs were expressed under the control of a doxycycline-inducible promoter, which we characterized in previous work (<a href="#bib27">Kramme et al., 2021a</a>).</p>

</section>
<section id="s2-2">

    <header>
      <h3>Overexpression screening identifies TFs that drive granulosa-like cell formation</h3>
    </header>

  <div>
      <p>To enable identification of granulosa-like cells, we used CRISPR/Cas9-mediated homology-directed repair to engineer an hiPSC line with a homozygous knock-in of a T2A-tdTomato reporter at the C-terminus of FOXL2 (<a href="https://elifesciences.org/articles/83291/figures#fig2s1">Figure 2—figure supplement 1</a>). We chose FOXL2 due to its specific expression in granulosa cells (<a href="#bib52">Ottolenghi et al., 2005</a>; <a href="#bib8">Cocquet et al., 2002</a>). Next, the pooled cDNA library was co-electroporated with a PiggyBac transposase expression plasmid, and a population of cells with integrated transposons was selected by treatment with puromycin. In these experiments, we tested several different library compositions and DNA concentrations (see Methods, <a href="#fig2sdata2">Figure 2—source data 2</a>, and <a href="https://elifesciences.org/articles/83291/figures#fig2s2">Figure 2—figure supplement 2</a>).</p>
<p>For screening, we induced TF expression with doxycycline, and sorted reporter-positive cells after 5 days of treatment. In addition to screening TF expression in the pluripotency-supporting mTeSR Plus medium, we also expressed TFs following differentiation of hiPSCs to early-stage mesoderm by treatment with the GSK3 inhibitor CHIR99021. The TF expression resulted in a small fraction of FOXL2+ cells, from which we extracted gDNA and sequenced barcodes. In both conditions, we observed barcodes for <i>NR5A1</i> to be strongly enriched in FOXL2+ cells relative to negative cells, as well as relative to the barcodes in the pre-induced hiPSC population (<a href="#fig2">Figure 2A</a>). Other TFs showed more modest barcode enrichment (<i>RUNX2</i>, <i>GATA4</i>, and <i>TCF21</i>), or enrichment in only one condition (<i>KLF2</i> and <i>NR2F2</i>). Interestingly, barcodes for <i>FOXL2</i> were strongly depleted (<a href="#fig2">Figure 2A</a>), suggesting negative feedback whereby exogenous FOXL2 directly or indirectly suppresses the expression of the endogenous FOXL2 reporter allele.</p>
    <div id="fig2" data-variant="" data-behaviour="AssetNavigation AssetViewer ToggleableCaption" data-selector=".caption-text__body" data-asset-viewer-group="fig2" data-asset-viewer-uri="https://iiif.elifesciences.org/lax:83291%2Felife-83291-fig2-v2.tif/full/1500,/0/default.jpg" data-asset-viewer-width="1500" data-asset-viewer-height="1428">
    
      
    
          <figure>
          
              <a href="https://iiif.elifesciences.org/lax:83291%2Felife-83291-fig2-v2.tif/full/1500,/0/default.jpg" target="_blank" rel="noopener noreferrer">
              <picture>
                  <source srcset="https://iiif.elifesciences.org/lax:83291%2Felife-83291-fig2-v2.tif/full/1234,/0/default.webp 2x, https://iiif.elifesciences.org/lax:83291%2Felife-83291-fig2-v2.tif/full/617,/0/default.webp 1x" type="image/webp"/>
                  <source srcset="https://iiif.elifesciences.org/lax:83291%2Felife-83291-fig2-v2.tif/full/1234,/0/default.jpg 2x, https://iiif.elifesciences.org/lax:83291%2Felife-83291-fig2-v2.tif/full/617,/0/default.jpg 1x" type="image/jpeg"/>
                  <img src="https://iiif.elifesciences.org/lax:83291%2Felife-83291-fig2-v2.tif/full/617,/0/default.jpg" alt=""/>
              </picture>
              </a>
          
          
          
          
          
              <figcaption>
          
                  <h6>Identification of transcription factors (TFs) whose overexpression generates granulosa-like cells.</h6>
                
                
                <p>(<b>A</b>) Pooled screening of barcoded TF cDNA libraries (see Methods for details) identifies TFs enriched in FOXL2-T2A-tdTomato+ cells. Library #1 is the full library of 35 TFs, and library #2 is a library containing a subset of 18 TFs. Empty values correspond to TFs that were absent in library #2. (<b>B</b>) Combinatorial screening identifies minimal TF combinations for inducing granulosa-like cells. TF combinations were integrated into human induced pluripotent stem cells (hiPSCs); a ‘1’ in the left-hand box signifies the presence of the TF in the combination corresponding to that row. For each combination, the polyclonal hiPSC population was differentiated with TF induction (see Methods). For the last 24 hr of differentiation, cells were additionally treated with FSH and androstenedione. Estradiol production and granulosa markers were measured by ELISA and flow cytometry after a total of 5 days. NR5A1 expression induced high levels of estradiol synthesis, but the combination of NR5A1 with RUNX1 or RUNX2 was required to give the best results for granulosa markers. ‘All markers’ signifies FOXL2+CD82+AMHR2+EPCAM−.</p>
          
          
              </figcaption>
          
          
          
          
          </figure>
    
        
    
    </div>
<p>Using the top TFs identified in barcode screening (<i>NR5A1</i>, <i>RUNX1/RUNX2</i>, <i>TCF21</i>, and <i>GATA4</i>) we then further optimized the conditions for generation of FOXL2+ cells. We included <i>RUNX1</i> in this list along with <i>RUNX2</i> because the two TFs are structurally and functionally similar, and <i>RUNX1</i> is known to play an important role for granulosa cell maintenance in the mouse (<a href="#bib46">Nicol et al., 2019</a>). We integrated these TFs into hiPSCs and established monoclonal lines, which we screened by flow cytometry after TF induction. We monitored FOXL2-tdTomato as well as the surface markers CD82, follicle-stimulating hormone receptor (FSHR), and EpCAM. CD82 is absent in hiPSCs, but highly expressed in granulosa cells beginning at the primordial follicle stage (<a href="#bib80">Zhang et al., 2018</a>). FSHR is specific for late-stage (secondary/antral) granulosa cells and Sertoli cells (the male equivalent). By contrast, EpCAM is expressed in hiPSCs and epithelial cells, but not granulosa cells (<a href="#bib80">Zhang et al., 2018</a>).</p>
<p>Out of 23 lines tested, 4 showed &gt;50% FOXL2-tdTomato+CD82+EpCAM− cells after 5 days of differentiation (<a href="https://elifesciences.org/articles/83291/figures#fig3s1">Figure 3—figure supplement 1A</a>). We observed low levels of FSHR expression, suggesting that we were mainly generating cells corresponding to early (primordial/primary follicle) granulosa cells, given that FSHR only becomes strongly expressed at later stages of ovarian follicle development (<a href="#bib49">Oktay et al., 1997</a>). Further optimization by adding the GSK3 inhibitor CHIR99021 for the first 2 days (<a href="https://elifesciences.org/articles/83291/figures#supp2">Supplementary file 2</a>) resulted in a near-homogeneous FOXL2-tdTomato+CD82+EPCAM− population for the top cell lines (<a href="https://elifesciences.org/articles/83291/figures#fig3s2">Figure 3—figure supplement 2</a>), with this effect being doxycycline responsive. Genotyping of two top lines (<a href="https://elifesciences.org/articles/83291/figures#fig3s1">Figure 3—figure supplement 1C</a>) revealed that both had <i>NR5A1</i> expression cassettes integrated, with the first line also having <i>RUNX1</i> and the second line <i>RUNX2</i>.</p>
<p>Subsequently, we set out to determine which TFs were sufficient for induction of granulosa-like cells. We tested combinations of hit TFs from our screening (<i>NR5A1</i>, <i>RUNX2</i>, <i>TCF21</i>, <i>GATA4</i>, <i>KLF2</i>, and <i>NR2F2</i>) as well as TFs reported to be important for ovarian function (<i>RUNX1</i>, <i>WT1</i> [−KTS isoform], and <i>FOXL2</i>). We tested expression of each TF individually, as well as combinations of other TFs and our top hit <i>NR5A1</i>. In addition to flow cytometry, we also measured production of estradiol after treatment of the cells with FSH and androstenedione. We observed that combinations containing <i>NR5A1</i> and <i>RUNX1</i> or <i>RUNX2</i> upregulated FOXL2-tdTomato expression, as well as granulosa surface markers AMHR2 and CD82 (<a href="#fig2">Figure 2B</a>). Estradiol production was strongly induced by <i>NR5A1</i>, and weakly by <i>GATA4</i> and <i>KLF2. RUNX1</i> and <i>RUNX2</i> expression somewhat decreased estradiol production, suggesting a regulatory role of these factors. Overall, these results indicate that <i>NR5A1</i> and either <i>RUNX1</i> or <i>RUNX2</i> are sufficient to induce a granulosa-like phenotype.</p>





  </div>

</section>
<section id="s2-3">

    <header>
      <h3>TF-mediated differentiation drives granulosa-like cell formation based on gene expression signatures</h3>
    </header>

  <div>
      <p>We next examined the gene expression of our granulosa-like cells. We compared the bulk transcriptomes of hiPSCs, COV434 and KGN ovarian tumor cells, and sorted FOXL2+CD82+ granulosa-like cells from day 5 of a polyclonal differentiation with expression of our previously identified top TFs (NR5A1, TCF21, GATA4, and RUNX1; see <a href="#fig2">Figure 2</a>). As an additional control, we included hiPSCs differentiated under the same conditions but without TF induction.</p>
<p>In the absence of TF expression, the hiPSCs differentiated into cells expressing mesoderm markers (full gene expression data are provided in the Source Data for <a href="#fig3">Figure 3</a>). At day 5 of differentiation, we observed strong upregulation relative to hiPSCs of genes associated with both lateral mesoderm (<i>HAND1</i> and <i>BMP5</i>) as well as paraxial mesoderm (<i>PAX3</i>), which potentially indicates the presence of a heterogeneous population. Upregulated genes were enriched for gene ontology terms related to heart, blood vessel, muscle, and skeletal development (<a href="https://elifesciences.org/articles/83291/figures#supp3">Supplementary file 3</a>).</p>
    <div id="fig3" data-variant="" data-behaviour="AssetNavigation AssetViewer ToggleableCaption" data-selector=".caption-text__body" data-asset-viewer-group="fig3" data-asset-viewer-uri="https://iiif.elifesciences.org/lax:83291%2Felife-83291-fig3-v2.tif/full/,1500/0/default.jpg" data-asset-viewer-width="710" data-asset-viewer-height="1500">
    
      
    
          <figure>
          
              <a href="https://iiif.elifesciences.org/lax:83291%2Felife-83291-fig3-v2.tif/full/,1500/0/default.jpg" target="_blank" rel="noopener noreferrer">
              <picture>
                  <source srcset="https://iiif.elifesciences.org/lax:83291%2Felife-83291-fig3-v2.tif/full/1234,/0/default.webp 2x, https://iiif.elifesciences.org/lax:83291%2Felife-83291-fig3-v2.tif/full/617,/0/default.webp 1x" type="image/webp"/>
                  <source srcset="https://iiif.elifesciences.org/lax:83291%2Felife-83291-fig3-v2.tif/full/1234,/0/default.jpg 2x, https://iiif.elifesciences.org/lax:83291%2Felife-83291-fig3-v2.tif/full/617,/0/default.jpg 1x" type="image/jpeg"/>
                  <img src="https://iiif.elifesciences.org/lax:83291%2Felife-83291-fig3-v2.tif/full/617,/0/default.jpg" alt=""/>
              </picture>
              </a>
          
          
          
          
          
              <figcaption>
          
                  <h6>Transcriptomic analysis of transcription factor (TF)-induced granulosa-like cells.</h6>
                
                
                <p>(<b>A</b>) Gene expression of selected markers in granulosa-like cells. Log<sub>2</sub>(TPM) values for gondal/granulosa, adrenal, and pluripotent marker genes were compared between 7 wpf male and female fetal gonad somatic cells, primary and primordial granulosa cells, TF-induced FOXL2+ cells, KGN cells, COV434 cells, and human induced pluripotent stem cells (hiPSCs). (<b>B</b>) Transcriptome overlap measure (TROM) comparison of TF-induced FOXL2+ cells, COV434 cells, and hiPSCs with published in vivo data from different time points in ovarian development. (<b>C</b>) Regulatory effects of granulosa-related TFs. RNA-seq was performed after 2 days of TF overexpression in hiPSCs (TFs shown in magenta). A differential gene expression (DEG) analysis was performed for all samples relative to the hiPSC control (<i>n</i> = 2 biological replicates each). Black arrows represent significant (false discovery rate &lt;0.05) upregulation, with the width proportional to the log<sub>2</sub>-fold change. Interactions are shown between TFs (magenta) and granulosa marker genes (yellow), as well as the stromal/theca marker <i>NR2F2</i> (red) and the pre-granulosa marker <i>LGR5</i> (green). (<b>D–I</b>) Volcano plots showing DEGs in the TF overexpression experiments. Colors are as in panel C; other DEGs not listed in panel C are shown in blue. Not all DEGs could be labeled due to space limits, but they are listed in the Source Data for this figure.</p>
          
          
              </figcaption>
          
          
          
          
          </figure>
    
        
    
    </div>
<p>With TF induction, we observed expression of bipotential gonad and granulosa markers, notably including <i>AMHR2</i>, <i>CD82</i>, <i>FOXL2</i>, <i>FSHR</i>, <i>IGFBP7</i>, <i>KRT19</i>, <i>STAR</i>, and <i>WNT4</i> (<a href="#fig3">Figure 3A</a>). The ovarian stromal/theca cell marker <i>NR2F2</i> was also upregulated. The expression levels (transcripts per million (TPM)) were generally comparable to those observed in previously published data from granulosa cells and human fetal gonad (<a href="#bib80">Zhang et al., 2018</a>; <a href="#bib65">Sybirna et al., 2020</a>). The only major exception was <i>WT1,</i> which had much weaker expression than in vivo (although still greater than hiPSCs and KGN cells). We also examined the expression of pluripotent and adrenal markers to check for incomplete or off-target differentiation. We did not observe adrenal marker expression, although we did note some residual <i>POU5F1</i> (encoding the OCT4 protein) expression that may indicate that our polyclonal population did not fully differentiate. COV434 cells, which were commonly considered as granulosa-like cells but recently reclassified as small cell ovarian carcinoma (<a href="#bib54">Price et al., 2012</a>; <a href="#bib79">Zhang et al., 2000</a>; <a href="#bib23">Karnezis et al., 2021</a>), did not express most granulosa markers, but did express high levels of <i>WT1</i> and <i>NR2F2</i>. By contrast, KGN cells expressed high levels of most granulosa markers, but not <i>WT1</i>.</p>
<p>We also performed a transcriptome-wide comparison of our TF-induced granulosa-like cells with three published datasets spanning weeks 6–21 of human fetal ovarian development (<a href="#bib67">Tang et al., 2015</a>; <a href="#bib32">Lecluze et al., 2020</a>; <a href="#bib77">Yatsenko et al., 2019</a>). Tang et al. sequenced RNA from male and female gonadal somatic cells at gestational week 7, whereas Yatsenko et al. and Leclucze et al. sequenced RNA from whole fetal ovaries over a range of gestational weeks. Using the transcriptome overlap measure (TROM) (<a href="#bib33">Li et al., 2017</a>), we computed similarity scores between our samples and these datasets (<a href="#fig3">Figure 3B</a>). We found a high degree of similarity (−log<sub>10</sub>(p<sub>adj</sub>) = 50.5) between our granulosa-like cells and human fetal ovaries. The closest match was with gestational week 12 whole fetal ovary from the Leclucze et al. dataset (which sequenced RNA from whole ovaries at different gestational weeks) but we also observed a significant similarity (−log<sub>10</sub>(p<sub>adj</sub>) of 12.7–15.8) with the other two datasets we examined. By contrast, COV434 and KGN cells showed a more modest similarity to the fetal ovary, and hiPSCs showed none at all.</p>
<p>To further elucidate the regulatory effects of individual TFs that we identified as granulosa related (FOXL2, WT1, NR5A1, GATA4, TCF21, and RUNX1), we integrated their cDNA plasmids into hiPSCs and induced expression with doxycycline. For these experiments, we harvested RNA after 2 days of TF expression, since we were interested in short-term effects. We performed differential gene expression analysis relative to hiPSCs, and examined regulatory effects among TFs, and between TFs and marker genes (<a href="#fig3">Figure 3C</a>). Overall, we observed that FOXL2, NR5A1, GATA4, and RUNX1 had the greatest effects on granulosa marker genes. Therefore, we generated additional hiPSC lines by integrating these TFs (<a href="https://elifesciences.org/articles/83291/figures#fig3s1">Figure 3—figure supplement 1</a>). To rule out any effects of the T2A-tdTomato reporter allele potentially perturbing FOXL2 function, we generated these in a wild-type background (F66 hiPSC line). We measured surface marker expression by flow cytometry after induction, confirmed expression of the granulosa marker <i>AMHR2</i> by qPCR, and additionally measured estradiol production in the presence of FSH and androstenedione (<a href="https://elifesciences.org/articles/83291/figures#fig3s1">Figure 3—figure supplement 1B</a>). We selected the clones (<a href="#fig3sdata2">Figure 3—source data 2</a>) that showed the most robust overall phenotype based on a combination of surface markers, <i>AMHR2</i>, expression and estradiol production.</p>
<p>In the differential expression analysis, we additionally found that GATA4 upregulated <i>NR2F2</i>, a marker of ovarian stromal and theca cells (<a href="#fig3">Figure 3C, G</a>). <i>LGR5</i>, previously reported as a marker of pre-granulosa cells (<a href="#bib48">Niu and Spradling, 2020</a>), was upregulated by GATA4 and RUNX1 (<a href="#fig3">Figure 3C, F, G</a>). We also confirmed a previous report that FOXL2 expression upregulated the cyclin-dependent kinase inhibitor <i>CDKN1B</i> (<a href="#bib15">Gustin et al., 2016</a>)<i>,</i> as well as downregulating <i>ORC1</i> and <i>MYC</i> (Source Data for <a href="#fig3">Figure 3</a>), suggesting suppression of cellular proliferation.</p>
<p>Finally, we conducted a gene ontology enrichment analysis of upregulated and downregulated genes for each TF (<a href="#bib38">Mi et al., 2021</a>). Significantly enriched terms were mainly related to generic developmental processes (e.g., ‘tissue development’, see <a href="https://elifesciences.org/articles/83291/figures#supp3">Supplementary file 3</a>) although terms related to gonad development were also significantly enriched for NR5A1 upregulated genes.</p>





  </div>

</section>
<section id="s2-4">

    <header>
      <h3>Granulosa-like cells respond to FSH and perform steroidogenesis</h3>
    </header>

  <div>
      <p>We next validated the ability of our granulosa-like cells to carry out one of the key endocrine functions of granulosa cells: the production of estradiol. In the ovary, theca cells convert cholesterol to androstenedione, which is the substrate for estradiol production in granulosa cells. The rate-limiting step is oxidative decarboxylation by CYP19A1 (aromatase), producing estrone, which is subsequently reduced to estradiol by enzymes in the HSD17B family, typically HSD17B1 in granulosa cells (<a href="#bib16">Hakkarainen et al., 2015</a>). In vivo, this pathway of estrogen synthesis is stimulated by FSH (<a href="#bib60">Sasson et al., 2004</a>; <a href="#bib72">Welsh et al., 1984</a>).</p>
<p>We treated our granulosa-like cells with androstenedione, in the presence or absence of FSH or forskolin (which directly increases levels of the FSHR second messenger cAMP). As controls, we used COV434 and KGN ovarian tumor cells, which were previously reported to produce estradiol from androstenedione (<a href="#bib54">Price et al., 2012</a>; <a href="#bib79">Zhang et al., 2000</a>; <a href="#bib47">Nishi et al., 2001</a>), as well as immortalized primary human granulosa cells (HGL5) and primary adult mouse ovarian somatic cells. Our granulosa-like cells produced estradiol from androstenedione, and in seven out of the nine monoclonal lines we tested, this steroidogenic activity significantly increased upon stimulation with FSH or forskolin (<a href="#fig4">Figure 4A</a>, see Source Data for statistical tests). One of our granulosa lines (F3/N.T #5) produced high levels of estradiol in all conditions, and this line, unlike the others, had neither <i>RUNX1</i> nor <i>RUNX2</i> expression vectors integrated (<a href="https://elifesciences.org/articles/83291/figures#fig3s1">Figure 3—figure supplement 1C</a>).</p>
    <div id="fig4" data-variant="" data-behaviour="AssetNavigation AssetViewer ToggleableCaption" data-selector=".caption-text__body" data-asset-viewer-group="fig4" data-asset-viewer-uri="https://iiif.elifesciences.org/lax:83291%2Felife-83291-fig4-v2.tif/full/,1500/0/default.jpg" data-asset-viewer-width="874" data-asset-viewer-height="1500">
    
      
    
          <figure>
          
              <a href="https://iiif.elifesciences.org/lax:83291%2Felife-83291-fig4-v2.tif/full/,1500/0/default.jpg" target="_blank" rel="noopener noreferrer">
              <picture>
                  <source srcset="https://iiif.elifesciences.org/lax:83291%2Felife-83291-fig4-v2.tif/full/1234,/0/default.webp 2x, https://iiif.elifesciences.org/lax:83291%2Felife-83291-fig4-v2.tif/full/617,/0/default.webp 1x" type="image/webp"/>
                  <source srcset="https://iiif.elifesciences.org/lax:83291%2Felife-83291-fig4-v2.tif/full/1234,/0/default.jpg 2x, https://iiif.elifesciences.org/lax:83291%2Felife-83291-fig4-v2.tif/full/617,/0/default.jpg 1x" type="image/jpeg"/>
                  <img src="https://iiif.elifesciences.org/lax:83291%2Felife-83291-fig4-v2.tif/full/617,/0/default.jpg" alt=""/>
              </picture>
              </a>
          
          
          
          
          
              <figcaption>
          
                  <h6>Hormonal signaling by granulosa-like cells.</h6>
                
                
                <p>(<b>A</b>) Granulosa-like cells produce estradiol in the presence of androstenedione and either FSH or forskolin (FK). Results are shown from nine monoclonal populations (see <a href="#fig3sdata2">Figure 3—source data 2</a>) of granulosa-like cells (<i>n</i> = 2 biological replicates for each of 9 clones, error bars are 95% CI), as well as the COV434 and KGN human ovarian cancer cell lines, HGL5 immortalized primary human granulosa cells, and primary adult mouse granulosa cells. Asterisks mark lines where FSH production significantly (two-tailed <i>t</i>-test, p &lt; 0.05) increased upon stimulation. Exact p values are given in the Source Data. (<b>B</b>) Ovaroids produce both estradiol and progesterone. Estradiol production requires androstenedione and is stimulated by FSH. Results are shown for ovaroids formed with six different monoclonal samples of granulosa-like cells (<i>n</i> = 1 sample per ovaroid per condition), at 3 days post-aggregation.</p>
          
          
              </figcaption>
          
          
          
          
          </figure>
    
        
    
    </div>
<p>The levels of estradiol produced by our three FSH-responsive lines were similar to those produced by KGN human granulosa tumor cells, which also showed responses to FSH and forskolin (<a href="#fig4">Figure 4A</a>). In contrast, COV434 cells, which showed no <i>FSHR</i> expression in our RNA-seq data (<a href="#fig3">Figure 3A</a>), were unresponsive to FSH alone, producing estradiol only in the presence of forskolin. HGL5 immortalized human granulosa cells did not produce estradiol under any condition. Primary adult mouse ovarian somatic cells produced similar amounts of estradiol to our hiPSC-derived granulosa-like cells (<a href="#fig4">Figure 4A</a>). However, the mouse cells did not show a response to FSH or forskolin, possibly because they were already exposed to FSH in vivo.</p>
<p>We also investigated whether our granulosa-like cells maintained their steroidogenic activity during ovaroid (ovarian organoid) co-culture with hPGCLCs (see below). We measured hormone levels in ovaroid supernatants in the presence or absence of androstenedione and FSH. In addition to estradiol, we also measured progesterone, which granulosa cells produce in vivo after ovulation and formation of the corpus luteum. We observed production of both hormones in five out of six samples (<a href="#fig4">Figure 4B</a>). Estradiol was produced only in the presence of androstenedione supplementation, and levels increased with FSH treatment. Progesterone was produced in all conditions but was highest in the absence of androstenedione.</p>





  </div>

</section>
<section id="s2-5">

    <header>
      <h3>Granulosa-like cells support germ cell development within ovaroids</h3>
    </header>

  <div>
      <p>Current methods for inducing and culturing hPGCLCs produce cells corresponding to immature, premigratory PGCs that lack expression of gonadal PGC markers such as DAZL (<a href="#bib22">Irie et al., 2015</a>). During fetal development, PGCs mature through interactions with gonadal somatic cells, with DAZL playing a key role in downregulation of pluripotency factors and commitment to gametogenesis (<a href="#bib44">Nicholls et al., 2019</a>; <a href="#bib5">Chen et al., 2014</a>). This process has recently been recreated in vitro using mouse fetal ovarian somatic cells (<a href="#bib74">Yamashiro et al., 2018</a>), which allowed the development of hPGCLCs to the oogonia-like stage. We hypothesized that in vitro-derived human granulosa-like cells could perform a similar role, with the potential for eliminating interspecies developmental mismatches. Therefore, we combined our granulosa-like cells with hPGCLCs to form ovarian organoids, which we termed ovaroids.</p>
<p>To generate ovaroids, we aggregated these two cell types in low-binding U-bottom wells, followed by transfer to air–liquid interface Transwell culture. As a comparison, we followed a previously described protocol (<a href="#bib75">Yamashiro et al., 2020</a>) to isolate fetal mouse ovarian somatic cells and aggregate them with hPGCLCs. By immunofluorescence, we observed expression of the mature marker DAZL beginning in a subset of OCT4 + hPGCLCs at 4 days of co-culture with hiPSC-derived granulosa-like cells (<a href="#fig5">Figure 5A</a>). In contrast, robust DAZL expression in co-culture with mouse cells was not observed until day 32 (<a href="#fig5">Figure 5B</a>), with fainter expression visible at day 26 (<a href="https://elifesciences.org/articles/83291/figures#supp4">Supplementary file 4</a>). Similarly, in a previous study using the same hPGCLC line and anti-DAZL antibody, DAZL expression was observed only after 77 days of co-culture with mouse fetal testis somatic cells (<a href="#bib25">Kobayashi et al., 2022</a>).</p>
    <div id="fig5" data-variant="" data-behaviour="AssetNavigation AssetViewer ToggleableCaption" data-selector=".caption-text__body" data-asset-viewer-group="fig5" data-asset-viewer-uri="https://iiif.elifesciences.org/lax:83291%2Felife-83291-fig5-v2.tif/full/1500,/0/default.jpg" data-asset-viewer-width="1500" data-asset-viewer-height="902">
    
      
    
          <figure>
          
              <a href="https://iiif.elifesciences.org/lax:83291%2Felife-83291-fig5-v2.tif/full/1500,/0/default.jpg" target="_blank" rel="noopener noreferrer">
              <picture>
                  <source srcset="https://iiif.elifesciences.org/lax:83291%2Felife-83291-fig5-v2.tif/full/1234,/0/default.webp 2x, https://iiif.elifesciences.org/lax:83291%2Felife-83291-fig5-v2.tif/full/617,/0/default.webp 1x" type="image/webp"/>
                  <source srcset="https://iiif.elifesciences.org/lax:83291%2Felife-83291-fig5-v2.tif/full/1234,/0/default.jpg 2x, https://iiif.elifesciences.org/lax:83291%2Felife-83291-fig5-v2.tif/full/617,/0/default.jpg 1x" type="image/jpeg"/>
                  <img src="https://iiif.elifesciences.org/lax:83291%2Felife-83291-fig5-v2.tif/full/617,/0/default.jpg" alt=""/>
              </picture>
              </a>
          
          
          
          
          
              <figcaption>
          
                  <h6>Ovaroid development and germ cell maturation.</h6>
                
                
                <p>Scale bars in immunofluorescence images are 40 µm. (<b>A</b>) Human ovaroid (F66/N.R1.G.F #4 granulosa-like cells + hPGCLCs) sections at days 2, 4, 14, and 32 of culture, stained for FOXL2 (granulosa), OCT4 (germ cell/pluripotent), and DAZL (mature germ cell). (<b>B</b>) Mouse ovaroid (fetal mouse ovarian somatic cells + hPGCLCs) sections stained as in panel A. (<b>C</b>) Fraction of OCT4+ and DAZL+ cells relative to the total (DAPI+) over time in human ovaroids and mouse xeno-ovaroids. Counts were performed at 11 time points on images from 2 replicates of human ovaroids (F66/N.R1.G.F #4 and F66/N.R2 #1 granulosa-like cells + hPGCLCs) and 1 replicate of mouse xeno-ovaroids. (<b>D</b>) Human ovaroid (F66/N.R2 #1 granulosa-like cells + hPGCLCs) sections at days 4 and 8 of culture, stained for SOX17 (germ cell), TFAP2C (early germ cell), and AMHR2 (granulosa).</p>
          
          
              </figcaption>
          
          
          
          
          </figure>
    
        
    
    </div>
<p>The fraction of DAZL+ cells reached its maximum at day 14 in human ovaroids and day 38 in mouse ovaroids (<a href="#fig5">Figure 5C</a>). In human ovaroids, the fraction of OCT4+ cells declined after day 8. In mouse ovaroids, the fraction of OCT4+ cells also declined over time. At day 16 in human ovaroids, DAZL+OCT4− cells were also apparent (<a href="https://elifesciences.org/articles/83291/figures#fig5s1">Figure 5—figure supplement 1</a>) in addition to DAZL+OCT4+ cells, and past day 38 there were more DAZL+ cells than OCT4+ cells in total (<a href="#fig5">Figure 5C</a>). The downregulation of OCT4 in DAZL+ oogonia occurs in vivo during the second trimester of human fetal ovarian development (<a href="#bib2">Anderson et al., 2007</a>); however, we did not observe the transition of DAZL to exclusively cytoplasmic localization that was reported to take place at this stage. Expression of TFAP2C, an early-stage PGC marker, declined during ovaroid culture (<a href="#fig5">Figure 5D</a>) and was almost entirely absent by day 8. By contrast, SOX17 expression was still visible on day 8, and OCT4 and DAZL expression continued to day 54 (<a href="#fig5">Figure 5A, C</a>).</p>
<p>Although this system allowed the rapid development of hPGCLCs to the gonadal stage, the number of germ cells in both hiPSC- and mouse-derived ovaroids declined over prolonged culture (<a href="#fig5">Figure 5C</a>), indicating that either they were dying or differentiating to other lineages. Unlike mouse-derived ovaroids, the hiPSC-derived ovaroids cultured on Transwells gradually flattened and widened, and by day 38 were largely collapsed.</p>
<p>Nonetheless, in these longer-term experiments, we observed the formation of empty follicle-like structures composed of cuboidal AMHR2+FOXL2+ granulosa-like cells (<a href="#fig6">Figure 6A</a>), suggesting that the TFs could drive folliculogenesis even in the absence of oocytes. Follicle-like structure formation was first visible at day 16 (<a href="https://elifesciences.org/articles/83291/figures#fig5s1">Figure 5—figure supplement 1</a>), and at day 26 the largest of these structures had grown to 1–2 mm diameter (<a href="#fig6">Figure 6B</a>). At day 70, ovaroids had developed follicles of a variety of sizes, mainly small single-layer follicles (<a href="#fig6">Figure 6C</a>) but also including antral follicles (<a href="#fig6">Figure 6D</a>). Cells outside of the follicles stained positive for NR2F2 (<a href="#fig6">Figure 6C, D</a>), a marker of ovarian stromal and theca cells.</p>
    <div id="fig6" data-variant="" data-behaviour="AssetNavigation AssetViewer ToggleableCaption" data-selector=".caption-text__body" data-asset-viewer-group="fig6" data-asset-viewer-uri="https://iiif.elifesciences.org/lax:83291%2Felife-83291-fig6-v2.tif/full/1500,/0/default.jpg" data-asset-viewer-width="1500" data-asset-viewer-height="776">
    
      
    
          <figure>
          
              <a href="https://iiif.elifesciences.org/lax:83291%2Felife-83291-fig6-v2.tif/full/1500,/0/default.jpg" target="_blank" rel="noopener noreferrer">
              <picture>
                  <source srcset="https://iiif.elifesciences.org/lax:83291%2Felife-83291-fig6-v2.tif/full/1234,/0/default.webp 2x, https://iiif.elifesciences.org/lax:83291%2Felife-83291-fig6-v2.tif/full/617,/0/default.webp 1x" type="image/webp"/>
                  <source srcset="https://iiif.elifesciences.org/lax:83291%2Felife-83291-fig6-v2.tif/full/1234,/0/default.jpg 2x, https://iiif.elifesciences.org/lax:83291%2Felife-83291-fig6-v2.tif/full/617,/0/default.jpg 1x" type="image/jpeg"/>
                  <img src="https://iiif.elifesciences.org/lax:83291%2Felife-83291-fig6-v2.tif/full/617,/0/default.jpg" alt=""/>
              </picture>
              </a>
          
          
          
          
          
              <figcaption>
          
                  <h6>Ovaroid follicle formation.</h6>
                
                
                <p>(<b>A</b>) Day 35 human ovaroid (F66/N.R1.G #7 + hPGCLC) sections stained for FOXL2, OCT4, and AMHR2. Scale bars are 40 µm. Follicle-like structures are marked with yellow triangles. (<b>B</b>) Whole-ovaroid view of follicle-like structures in human ovaroids (F66/N.R1.G #7). Scale bars are 1 mm. (<b>C</b>) Section of human ovaroid (F66/N.R1.G.F #4 + hPGCLC) at day 70 of culture, stained for FOXL2, NR2F2, and AMHR2, showing multiple small follicles (yellow triangles) consisting single layers of FOXL2+AMHR2+ cells. NR2F2+ cells are interspersed between these. Scale bars are 100 µm. (<b>D</b>) Section of human ovaroid (F66/N.R2 #1 + hPGCLC) at day 70 of culture, stained for FOXL2, NR2F2, and AMHR2, showing an antral follicle consisting of FOXL2+AMHR2+ granulosa-like cells arranged in several layers around a central cavity. NR2F2 staining is visible outside of the follicle (marked ‘Stroma’). Scale bars are 100 µm.</p>
          
          
              </figcaption>
          
          
          
          
          </figure>
    
    
    </div>
<p>To further examine the gene expression of hPGCLCs and somatic cells in this system, we performed scRNA-seq on dissociated ovaroids at days 2, 4, 8, and 14 of culture, and clustered cells according to gene expression. As expected, the largest cluster (cluster 0) contained cells expressing granulosa markers such as <i>FOXL2</i>, <i>WNT4</i>, and <i>CD82</i> (<a href="#fig7">Figure 7A, B</a>). Cells expressing markers of secondary/antral granulosa cells such as <i>FSHR</i> and <i>CYP19A1</i> were also found within this cluster, although these were much less numerous. A smaller cluster (cluster 1) expressing the ovarian stromal marker <i>NR2F2</i> was also present. <i>NR2F2</i> is expressed by both stromal and theca cells, but the cells in cluster 1 did not express 17α-hydroxylase (<i>CYP17A1</i>), indicating that they could not produce androgens and were not theca cells. Full expression data for each cluster are given in <a href="#fig7sdata1">Figure 7—source data 1</a>.</p>
    <div id="fig7" data-variant="" data-behaviour="AssetNavigation AssetViewer ToggleableCaption" data-selector=".caption-text__body" data-asset-viewer-group="fig7" data-asset-viewer-uri="https://iiif.elifesciences.org/lax:83291%2Felife-83291-fig7-v2.tif/full/,1500/0/default.jpg" data-asset-viewer-width="1263" data-asset-viewer-height="1500">
    
      
    
          <figure>
          
              <a href="https://iiif.elifesciences.org/lax:83291%2Felife-83291-fig7-v2.tif/full/,1500/0/default.jpg" target="_blank" rel="noopener noreferrer">
              <picture>
                  <source srcset="https://iiif.elifesciences.org/lax:83291%2Felife-83291-fig7-v2.tif/full/1234,/0/default.webp 2x, https://iiif.elifesciences.org/lax:83291%2Felife-83291-fig7-v2.tif/full/617,/0/default.webp 1x" type="image/webp"/>
                  <source srcset="https://iiif.elifesciences.org/lax:83291%2Felife-83291-fig7-v2.tif/full/1234,/0/default.jpg 2x, https://iiif.elifesciences.org/lax:83291%2Felife-83291-fig7-v2.tif/full/617,/0/default.jpg 1x" type="image/jpeg"/>
                  <img src="https://iiif.elifesciences.org/lax:83291%2Felife-83291-fig7-v2.tif/full/617,/0/default.jpg" alt=""/>
              </picture>
              </a>
          
          
          
          
          
              <figcaption>
          
                  <h6>scRNA-seq analysis of ovaroids (F66/N.R1.G.F #4 granulosa-like cells + hPGCLCs). Data from all samples (days 2, 4, 8, and 14) were combined for joint dimensionality reduction and clustering.</h6>
                
                
                <p>(<b>A</b>) Expression (log<sub>2</sub> CPM) of selected granulosa (<i>FOXL2</i>)<i>,</i> stroma/theca (<i>NR2F2</i>), and germ cell (<i>PRDM1</i>) markers. (<b>B</b>) Leiden clustering shows four main clusters; the expression (log<sub>2</sub> CPM) of marker genes is plotted for each. (<b>C</b>) Mapping of cells onto a human fetal ovary reference atlas (<a href="#bib11">Garcia-Alonso et al., 2022</a>) and assignment of cell types. (<b>D</b>) Proportion of somatic cell types, germ cells, <i>DAZL</i>+ cells, and <i>DDX4</i>+ cells in ovaroids from each day.</p>
          
          
              </figcaption>
          
          
          
          
          </figure>
    
        
    
    </div>
<p>We also observed a cluster of hPGCLCs expressing marker genes such as <i>CD38</i>, <i>KIT</i>, <i>PRDM1</i>, <i>TFAP2C</i>, <i>PRDM14</i>, <i>NANOG</i>, and <i>POU5F1</i>. Notably, X-chromosomal lncRNAs <i>XIST</i>, <i>TSIX</i>, and <i>XACT</i> were all more highly expressed (an average of ~80-, ~20-, and ~2900-fold, respectively) in the hPGCLCs relative to other clusters (<a href="#fig7">Figure 7B</a>), suggesting that the hPGCLCs were starting the process of X-reactivation (<a href="#bib70">Vallot et al., 2017</a>), which in hPGCs is associated with high expression of both <i>XIST</i> and <i>XACT</i> (<a href="#bib6">Chitiashvili et al., 2020</a>). The X-chromosomal <i>HPRT1</i> gene, known to be more highly expressed in cells with two active X chromosomes (<a href="#bib9">Epstein, 1972</a>), was also ~3-fold upregulated (<a href="#fig7sdata1">Figure 7—source data 1</a>).</p>
<p>We next compared our in vitro-generated ovaroids to a reference atlas of human fetal ovarian development (<a href="#bib11">Garcia-Alonso et al., 2022</a>). We used scanpy ingest (<a href="#bib73">Wolf et al., 2018</a>) to integrate our samples into the atlas and annotate each cell with the closest cell type from the in vivo data (<a href="#fig7">Figure 7C</a>). The ovaroids consisted mainly of granulosa, gonadal mesenchyme, and pre-granulosa lineages (<a href="#fig7">Figure 7D</a>), with a small fraction of coelomic epithelium. The fraction of granulosa cells increased from day 2 through day 8, potentially representing a maturation of the somatic cell population. As expected, neural, immune, smooth muscle, and erythroid cells, which were present in fetal ovaries, were completely absent from our ovaroids. Epithelial, endothelial, and perivascular cells were detected, but at very low frequency (1% or less), possibly representing a low rate of off-target differentiation.</p>
<p>We additionally examined the overall fraction of germ cells, as well as the fraction of cells expressing the gonadal germ cell markers <i>DAZL</i> and <i>DDX4</i>, over the course of our experiment (<a href="#fig7">Figure 7D</a>). We defined the germ cell population based on the fetal ovary atlas integration. This population increased from days 2 to 4, but declined thereafter. In comparison, the fraction of <i>DAZL+</i> and <i>DDX4+</i> cells also increased from days 2 to 4, but remained roughly constant from days 4 to 14 (<a href="#fig7">Figure 7D</a>). We performed a differential gene expression analysis and gene ontology enrichment on <i>DAZL+</i> cells relative to <i>DAZL−</i> cells. Upregulated genes (log<sub>2</sub>fc &gt;2, <i>n</i> = 221) were most highly enriched for terms related to generic developmental processes but also included terms related to adhesion and migration (e.g., ‘ameboidal-type cell migration’), as well as reproductive system development. Downregulated genes (log<sub>2</sub>fc &lt;−2, <i>n</i> = 6451) were strongly related to metabolic processes and mitotic cell division. Full data on differentially expressed genes and ontology enrichment are provided in <a href="https://elifesciences.org/articles/83291/figures#supp4">Supplementary file 4</a>. These data suggest that DAZL+ cells in our ovaroids are downregulating their metabolism and proliferation, in agreement with the known role of DAZL in suppressing PGC proliferation (<a href="#bib76">Yan et al., 2022</a>; <a href="#bib12">Gill et al., 2011</a>).</p>





  </div>

</section>





  </div>

</section>


                
                    <section id="s3" data-behaviour="ArticleSection" data-initial-state="closed">

    <header>
      <h2>Discussion</h2>
    </header>

  <div>
      <p>This work establishes a rapid (5 days) and efficient (&gt;70% in top monoclonal lines) method to produce granulosa-like cells by TF-mediated differentiation of hiPSCs. These granulosa-like cells express granulosa marker genes and have highly similar transcriptomes to the human fetal ovary, both by bulk RNA-seq (<a href="#fig3">Figure 3</a>) and scRNA-seq (<a href="#fig7">Figure 7</a>). We demonstrate that our system can model several key ovarian phenotypes, including hormonal signaling, germ cell maturation, and follicle formation. Additionally, we characterize the regulatory effects of granulosa-related TFs and determine that co-expression of NR5A1 and RUNX1 or RUNX2 is minimally sufficient to generate granulosa-like cells from hiPSCs (<a href="#fig2">Figure 2B</a>). We show that FOXL2 and GATA4 may help to further stabilize the granulosa-like transcriptome (<a href="#fig3">Figure 3C</a>), and these TFs were also included in some of our top-performing lines (<a href="https://elifesciences.org/articles/83291/figures#fig3s1">Figure 3—figure supplement 1</a>, <a href="#fig3sdata2">Figure 3—source data 2</a>). Finally, our ovaroids formed from granulosa-like cells and hPGCLCs are notable as the first reported fully human ovarian organoids.</p>
<p>Our human ovaroids allow the rapid development of hPGCLCs to the gonadal stage as measured by expression of the markers <i>DAZL</i> and <i>DDX4</i> (<a href="#fig5">Figures 5</a> and <a href="#fig7">7</a>), which was previously only achieved by extended culture with mouse fetal gonadal somatic cells (<a href="#bib74">Yamashiro et al., 2018</a>; <a href="#bib21">Hwang et al., 2020</a>; <a href="#bib42">Murase et al., 2020</a>). As <i>DAZL</i> and <i>DDX4</i> are not expressed in pre-gonadal germ cells and only become expressed after germ cells colonize the gonads (<a href="#bib74">Yamashiro et al., 2018</a>; <a href="#bib44">Nicholls et al., 2019</a>; <a href="#bib25">Kobayashi et al., 2022</a>), this implies that our hiPSC-derived granulosa-like cells are able to simulate the gonadal niche and allow germ cell maturation. In a direct comparison with the previous method (<a href="#fig5">Figure 5</a>), we show that DAZL expression occurs earlier when hPGCLCs are co-cultured with hiPSC-derived granulosa-like cells instead of mouse fetal ovarian somatic cells.</p>
<p>While our results represent a step forward for the in vitro study of granulosa cells and germ cell development, future research will be necessary to construct a full model of the human ovary. Although we can efficiently generate granulosa-like cells and form ovaroids in combination with germ cells, improved methods will be required for maintaining these ovaroids in culture and preventing the loss of DAZL+ germ cells. Recent optimizations in media and culture methods for re-aggregated human fetal ovarian cells (<a href="#bib40">Mizuta et al., 2022</a>), which were specifically developed to avoid the ‘flattening’ issue with Transwell cultures, may be applicable to our system and allow for more extended development. Our results were also similar to a previous study in which rhesus PGCLCs gained expression of the mature marker DDX4 upon transplantation into an adult testis, but did not complete spermatogenesis (<a href="#bib62">Sosa et al., 2018</a>). A better developmental match (using later-stage germ cells or earlier-stage gonadal somatic cells) may provide better results.</p>
<p>Additionally, the ovary consists of more cell types than merely germ cells and granulosa cells (<a href="#fig7">Figure 7C</a>), and these other cells are also functionally important. For example, our ovaroids produced estradiol only in the presence of androstenedione, which is normally produced by ovarian theca cells. Although our granulosa-like cells were capable of a limited amount of differentiation into NR2F2+ ovarian stromal cells (<a href="#fig6">Figures 6</a> and <a href="#fig7">7</a>), these were not functional theca cells. Thus, developing a method to generate theca cells (or earlier-stage gonadal cells that differentiate into both granulosa and theca cells) will provide a more complete model of ovarian biology. We anticipate that our overall approach of identifying TFs to drive the differentiation of hiPSCs will be broadly applicable to generate these cell types of interest, as well as many others.</p>
<p>One further limitation is the random integration of our transposon-based TF expression vectors. This necessitated generating and testing several clonal hiPSC lines before selecting ones that performed well (<a href="https://elifesciences.org/articles/83291/figures#fig3s1">Figure 3—figure supplement 1</a> and <a href="#fig3sdata2">Figure 3—source data 2</a>). Additionally, due to the challenge of growing large numbers of hPGCLCs, we only performed our ovaroid experiments using granulosa-like cells from a limited number of lines (six for hormone assays, two for immunofluorescence, and one for scRNA-seq). We selected the lines for these endpoints based on an assessment of their surface marker expression and estradiol production, as well as preliminary aggregation experiments in which they showed follicle-like morphology. In future studies, we hope to better characterize the relative dosage requirements for our TFs and generate hiPSC lines with controlled TF expression from a known integration site in a safe-harbor locus, or from non-integrating episomal vectors. This should provide a more reproducible way to generate high-performing lines.</p>
<p>Despite these limitations, our system has several advantages in comparison with existing in vitro models of the human ovary. Several studies have used mouse fetal gonad cells to support hPGCLC development (<a href="#bib74">Yamashiro et al., 2018</a>; <a href="#bib25">Kobayashi et al., 2022</a>), but this process is much slower (32 vs. 4 days, <a href="#fig3">Figure 3</a>), the mouse cells may not behave the same as human cells, and mouse fetal gonad cells are difficult to obtain in large quantities. Other studies have used cancer cell lines such as COV434 or KGN to study various aspects of granulosa cells (<a href="#bib54">Price et al., 2012</a>; <a href="#bib79">Zhang et al., 2000</a>; <a href="#bib47">Nishi et al., 2001</a>). However, these cancer cells may be phenotypically aberrant. In particular, we show that unlike KGN cells or our iPSC-derived granulosa-like cells, COV434 cells do not express granulosa marker genes and do not respond to FSH. This is in agreement with a recent study reporting that the COV434 cell line was mis-identified and actually originates from small cell carcinoma of the ovary, not a granulosa tumor (<a href="#bib23">Karnezis et al., 2021</a>). Finally, some previous studies have reported growth-factor-based protocols to differentiate human pluripotent stem cells to ovarian granulosa-like cells (<a href="#bib35">Lipskind et al., 2018</a>; <a href="#bib29">Lan et al., 2013</a>; <a href="#bib13">Gonen et al., 2021</a>). However, the efficiency of such protocols is low (4–12%, or not reported), the protocols require extended culture (multiple weeks vs. 5 days), and those studies did not demonstrate that the cells could support germ cell maturation. Using monoclonal hiPSC lines, our induction efficiency (&gt;70% in top lines, <a href="https://elifesciences.org/articles/83291/figures#fig3s1">Figure 3—figure supplement 1</a>) is sufficient to form ovaroids without the need for sorting FOXL2+ cells.</p>
<p>Overall, our rapid, scalable, and efficient method of producing human granulosa-like cells represents a substantial advance over previous studies. We anticipate that this system will prove useful as a model for the study of ovarian development and function, and that further optimization could allow for human in vitro oogenesis.</p>





  </div>

</section>


                
                    <section id="s4" data-behaviour="ArticleSection" data-initial-state="closed">

    <header>
      <h2>Materials and methods</h2>
    </header>

  <div>
          <div id="keyresource" data-variant="table" data-behaviour="AssetNavigation AssetViewer ToggleableCaption" data-selector=".caption-text__body" data-asset-viewer-group="keyresource">
    
      
    
          <figure>
          
          
          
          
          
          
          
          
          
          
              <div>
              
              
                  <div>
              
                    <table><thead><tr><th>Reagent type (species) or resource</th><th>Designation</th><th>Source or reference</th><th>Identifiers</th><th>Additional information</th></tr></thead><tbody><tr><td>Strain, strain </td><td>CD-1</td><td>Charles River Labs</td><td>RRID:<a href="https://identifiers.org/RRID/RRID:IMSR_CRL:022">IMSR_CRL:022</a></td><td>Used for fetal ovary isolation</td></tr><tr><td>Strain, strain </td><td>BALB/c</td><td>Charles River Labs</td><td>RRID:<a href="https://identifiers.org/RRID/RRID:IMSR_APB:4790">IMSR_APB:4790</a></td><td>Used for adult ovary isolation</td></tr><tr><td>Cell line </td><td>F3 iPSC</td><td>ATCC</td><td>ATCC-BXS0116</td><td></td></tr><tr><td>Cell line </td><td>F66 iPSC</td><td>Other</td><td></td><td>Previously derived in-house </td></tr><tr><td>Cell line </td><td>F2 PGCLC</td><td><a href="#bib25">Kobayashi et al., 2022</a>; PMID:<a href="https://pubmed.ncbi.nlm.nih.gov/35148847/">35148847</a></td><td>PMID:<a href="https://pubmed.ncbi.nlm.nih.gov/35148847/">35148847</a></td><td>Derived from the ATCC-</td></tr><tr><td>Recombinant </td><td>pHDR-FOXL2-T2A-</td><td>This paper</td><td>RRID:<a href="https://identifiers.org/RRID/RRID:Addgene_192892">Addgene_192892</a></td><td>For engineering reporter </td></tr><tr><td>Recombinant </td><td>pX330-FOXL2-cterm</td><td>This paper</td><td>RRID:<a href="https://identifiers.org/RRID/RRID:Addgene_192893">Addgene_192893</a></td><td>For engineering reporter </td></tr><tr><td>Recombinant </td><td>Super PiggyBac </td><td>System Bioscience</td><td>Cat:PB210PA-1</td><td></td></tr><tr><td>Recombinant </td><td>pPB-cT3G-cERP2-TCF21</td><td>This paper</td><td>RRID:<a href="https://identifiers.org/RRID/RRID:Addgene_192894">Addgene_192894</a></td><td>Dox-inducible TF expression</td></tr><tr><td>Recombinant DNA reagent</td><td>pPB-cT3G-cERP2-JUN</td><td>This paper</td><td>RRID:<a href="https://identifiers.org/RRID/RRID:Addgene_192895">Addgene_192895</a></td><td>Dox-inducible TF expression</td></tr><tr><td>Recombinant DNA reagent</td><td>pPB-cT3G-cERP2-GATA4</td><td>This paper</td><td>RRID:<a href="https://identifiers.org/RRID/RRID:Addgene_192896">Addgene_192896</a></td><td>Dox-inducible TF expression</td></tr><tr><td>Recombinant DNA reagent</td><td>pPB-cT3G-cERP2-CEBPD</td><td>This paper</td><td>RRID:<a href="https://identifiers.org/RRID/RRID:Addgene_192897">Addgene_192897</a></td><td>Dox-inducible TF expression</td></tr><tr><td>Recombinant DNA reagent</td><td>pPB-cT3G-cERP2-EMX2</td><td>This paper</td><td>RRID:<a href="https://identifiers.org/RRID/RRID:Addgene_192898">Addgene_192898</a></td><td>Dox-inducible TF expression</td></tr><tr><td>Recombinant DNA reagent</td><td>pPB-cT3G-cERP2-MYC</td><td>This paper</td><td>RRID:<a href="https://identifiers.org/RRID/RRID:Addgene_192899">Addgene_192899</a></td><td>Dox-inducible TF expression</td></tr><tr><td>Recombinant DNA reagent</td><td>pPB-cT3G-cERP2-NR2F2</td><td>This paper</td><td>RRID:<a href="https://identifiers.org/RRID/RRID:Addgene_192900">Addgene_192900</a></td><td>Dox-inducible TF expression</td></tr><tr><td>Recombinant DNA reagent</td><td>pPB-cT3G-cERP2-FOSB</td><td>This paper</td><td>RRID:<a href="https://identifiers.org/RRID/RRID:Addgene_192901">Addgene_192901</a></td><td>Dox-inducible TF expression</td></tr><tr><td>Recombinant DNA reagent</td><td>pPB-cT3G-cERP2-KLF2</td><td>This paper</td><td>RRID:<a href="https://identifiers.org/RRID/RRID:Addgene_192902">Addgene_192902</a></td><td>Dox-inducible TF expression</td></tr><tr><td>Recombinant DNA reagent</td><td>pPB-cT3G-cERP2-ZBTB16</td><td>This paper</td><td>RRID:<a href="https://identifiers.org/RRID/RRID:Addgene_192903">Addgene_192903</a></td><td>Dox-inducible TF expression</td></tr><tr><td>Recombinant DNA reagent</td><td>pPB-cT3G-cERP2-YBX1</td><td>This paper</td><td>RRID:<a href="https://identifiers.org/RRID/RRID:Addgene_192904">Addgene_192904</a></td><td>Dox-inducible TF expression</td></tr><tr><td>Recombinant DNA reagent</td><td>pPB-cT3G-cERP2-LHX1</td><td>This paper</td><td>RRID:<a href="https://identifiers.org/RRID/RRID:Addgene_192905">Addgene_192905</a></td><td>Dox-inducible TF expression</td></tr><tr><td>Recombinant DNA reagent</td><td>pPB-cT3G-cERP2-WT1</td><td>This paper</td><td>RRID:<a href="https://identifiers.org/RRID/RRID:Addgene_192906">Addgene_192906</a></td><td>Dox-inducible TF expression</td></tr><tr><td>Recombinant DNA reagent</td><td>pPB-cT3G-cERP2-JUNB</td><td>This paper</td><td>RRID:<a href="https://identifiers.org/RRID/RRID:Addgene_192907">Addgene_192907</a></td><td>Dox-inducible TF expression</td></tr><tr><td>Recombinant DNA reagent</td><td>pPB-cT3G-cERP2-KLF4</td><td>This paper</td><td>RRID:<a href="https://identifiers.org/RRID/RRID:Addgene_192908">Addgene_192908</a></td><td>Dox-inducible TF expression</td></tr><tr><td>Recombinant DNA reagent</td><td>pPB-cT3G-cERP2-KLF6</td><td>This paper</td><td>RRID:<a href="https://identifiers.org/RRID/RRID:Addgene_192909">Addgene_192909</a></td><td>Dox-inducible TF expression</td></tr><tr><td>Recombinant DNA reagent</td><td>pPB-cT3G-cERP2-HOXC9</td><td>This paper</td><td>RRID:<a href="https://identifiers.org/RRID/RRID:Addgene_192910">Addgene_192910</a></td><td>Dox-inducible TF expression</td></tr><tr><td>Recombinant DNA reagent</td><td>pPB-cT3G-cERP2-ATF4</td><td>This paper</td><td>RRID:<a href="https://identifiers.org/RRID/RRID:Addgene_192911">Addgene_192911</a></td><td>Dox-inducible TF expression</td></tr><tr><td>Recombinant DNA reagent</td><td>pPB-cT3G-cERP2-PPARG</td><td>This paper</td><td>RRID:<a href="https://identifiers.org/RRID/RRID:Addgene_192912">Addgene_192912</a></td><td>Dox-inducible TF expression</td></tr><tr><td>Recombinant DNA reagent</td><td>pPB-cT3G-cERP2-EGR1</td><td>This paper</td><td>RRID:<a href="https://identifiers.org/RRID/RRID:Addgene_192913">Addgene_192913</a></td><td>Dox-inducible TF expression</td></tr><tr><td>Recombinant DNA reagent</td><td>pPB-cT3G-cERP2-FOXL2</td><td>This paper</td><td>RRID:<a href="https://identifiers.org/RRID/RRID:Addgene_192914">Addgene_192914</a></td><td>Dox-inducible TF expression</td></tr><tr><td>Recombinant DNA reagent</td><td>pPB-cT3G-cERP2-NR5A1</td><td>This paper</td><td>RRID:<a href="https://identifiers.org/RRID/RRID:Addgene_192915">Addgene_192915</a></td><td>Dox-inducible TF expression</td></tr><tr><td>Recombinant DNA reagent</td><td>pPB-cT3G-cERP2-TSC22D3</td><td>This paper</td><td>RRID:<a href="https://identifiers.org/RRID/RRID:Addgene_192916">Addgene_192916</a></td><td>Dox-inducible TF expression</td></tr><tr><td>Recombinant DNA reagent</td><td>pPB-cT3G-cERP2-MAFF</td><td>This paper</td><td>RRID:<a href="https://identifiers.org/RRID/RRID:Addgene_192917">Addgene_192917</a></td><td>Dox-inducible TF expression</td></tr><tr><td>Recombinant DNA reagent</td><td>pPB-cT3G-cERP2-ELK1</td><td>This paper</td><td>RRID:<a href="https://identifiers.org/RRID/RRID:Addgene_192918">Addgene_192918</a></td><td>Dox-inducible TF expression</td></tr><tr><td>Recombinant DNA reagent</td><td>pPB-cT3G-cERP2-NR1H2</td><td>This paper</td><td>RRID:<a href="https://identifiers.org/RRID/RRID:Addgene_192919">Addgene_192919</a></td><td>Dox-inducible TF expression</td></tr><tr><td>Recombinant DNA reagent</td><td>pPB-cT3G-cERP2-TOX2</td><td>This paper</td><td>RRID:<a href="https://identifiers.org/RRID/RRID:Addgene_192920">Addgene_192920</a></td><td>Dox-inducible TF expression</td></tr><tr><td>Recombinant DNA reagent</td><td>pPB-cT3G-cERP2-LHX9</td><td>This paper</td><td>RRID:<a href="https://identifiers.org/RRID/RRID:Addgene_192921">Addgene_192921</a></td><td>Dox-inducible TF expression</td></tr><tr><td>Recombinant DNA reagent</td><td>pPB-cT3G-cERP2-ZFPM2</td><td>This paper</td><td>RRID:<a href="https://identifiers.org/RRID/RRID:Addgene_192922">Addgene_192922</a></td><td>Dox-inducible TF expression</td></tr><tr><td>Recombinant DNA reagent</td><td>pPB-cT3G-cERP2-HOPX</td><td>This paper</td><td>RRID:<a href="https://identifiers.org/RRID/RRID:Addgene_192923">Addgene_192923</a></td><td>Dox-inducible TF expression</td></tr><tr><td>Recombinant DNA reagent</td><td>pPB-cT3G-cERP2-FOS</td><td>This paper</td><td>RRID:<a href="https://identifiers.org/RRID/RRID:Addgene_192924">Addgene_192924</a></td><td>Dox-inducible TF expression</td></tr><tr><td>Recombinant DNA reagent</td><td>pPB-cT3G-cERP2-NR4A1</td><td>This paper</td><td>RRID:<a href="https://identifiers.org/RRID/RRID:Addgene_192925">Addgene_192925</a></td><td>Dox-inducible TF expression</td></tr><tr><td>Recombinant DNA reagent</td><td>pPB-cT3G-cERP2-RUNX2</td><td>This paper</td><td>RRID:<a href="https://identifiers.org/RRID/RRID:Addgene_192926">Addgene_192926</a></td><td>Dox-inducible TF expression</td></tr><tr><td>Recombinant DNA reagent</td><td>pPB-cT3G-cERP2-TAF4B</td><td>This paper</td><td>RRID:<a href="https://identifiers.org/RRID/RRID:Addgene_192927">Addgene_192927</a></td><td>Dox-inducible TF expression</td></tr><tr><td>Recombinant DNA reagent</td><td>pPB-cT3G-cERP2-RUNX1</td><td>This paper</td><td>RRID:<a href="https://identifiers.org/RRID/RRID:Addgene_192928">Addgene_192928</a></td><td>Dox-inducible TF expression</td></tr><tr><td>Antibody</td><td>anti-AMHR2 (Rabbit IgG, polyclonal)</td><td>Thermo Fisher</td><td>Cat:PA5-13902; RRID:<a href="https://identifiers.org/RRID/RRID:AB_2305463">AB_2305463</a></td><td>IF (1:100)</td></tr><tr><td>Antibody</td><td>anti-DAZL (Rabbit IgG, monoclonal)</td><td>Abcam</td><td>Cat:ab215718; RRID:<a href="https://identifiers.org/RRID/RRID:AB_2893177">AB_2893177</a></td><td>IF (1:500)</td></tr><tr><td>Antibody</td><td>anti-FOXL2 (Goat IgG, polyclonal)</td><td>Novus</td><td>Cat:NB100-1277; RRID:<a href="https://identifiers.org/RRID/RRID:AB_2106187">AB_2106187</a></td><td>IF (1:250)</td></tr><tr><td>Antibody</td><td>anti-44838 (Mouse IgG, monoclonal)</td><td>BD Bioscience</td><td>Cat:611202; RRID:<a href="https://identifiers.org/RRID/RRID:AB_398736">AB_398736</a></td><td>IF (1:200)</td></tr><tr><td>Antibody</td><td>anti-SOX17 (Goat IgG, polyclonal)</td><td>Novus</td><td>Cat:AF1924; RRID:<a href="https://identifiers.org/RRID/RRID:AB_355060">AB_355060</a></td><td>IF (1:500)</td></tr><tr><td>Antibody</td><td>anti-TFAP2C (Mouse IgG, monoclonal)</td><td>Abcam</td><td>Cat:ab110635; RRID:<a href="https://identifiers.org/RRID/RRID:AB_10858471">AB_10858471</a></td><td>IF (1:250)</td></tr><tr><td>Antibody</td><td>anti-NR2F2 (Mouse IgG, monoclonal)</td><td>Novus</td><td>Cat:H7147; RRID:<a href="https://identifiers.org/RRID/RRID:AB_1964214">AB_1964214</a></td><td>IF (1:250)</td></tr><tr><td>Antibody</td><td>anti-AMHR2-FITC (Rabbit IgG polyclonal)</td><td>Biorbyt</td><td>Cat:orb37457; RRID:<a href="https://identifiers.org/RRID/RRID:AB_10992015">AB_10992015</a></td><td>Flow cytometry (1:60)</td></tr><tr><td>Antibody</td><td>anti-CD82-PerCP-Cy5.5 (Mouse IgG monoclonal)</td><td>BioLegend</td><td>Cat:342111; RRID:<a href="https://identifiers.org/RRID/RRID:AB_2750124">AB_2750124</a></td><td>Flow cytometry (1:60)</td></tr><tr><td>Antibody</td><td>anti-EpCAM-APC-Cy7 </td><td>BioLegend</td><td>Cat:324245; RRID:<a href="https://identifiers.org/RRID/RRID:AB_2783193">AB_2783193</a></td><td>Flow cytometry (1:60)</td></tr><tr><td>Antibody</td><td>anti-FSHR-APC (Mouse </td><td>R&amp;D Systems</td><td>Cat:FAB65591A; RRID:<a href="https://identifiers.org/RRID/RRID:AB_2920602">AB_2920602</a></td><td>Flow cytometry (1:60)</td></tr><tr><td>Antibody</td><td>anti-Mouse IgG-AF647 </td><td>Fisher</td><td>Cat:A31571; RRID:<a href="https://identifiers.org/RRID/RRID:AB_162542">AB_162542</a></td><td>IF secondary (1:250)</td></tr><tr><td>Antibody</td><td>anti-Goat IgG-AF568 </td><td>Fisher</td><td>Cat:A11057; RRID:<a href="https://identifiers.org/RRID/RRID:AB_2534104">AB_2534104</a></td><td>IF secondary (1:500)</td></tr><tr><td>Antibody</td><td>anti-Rabbit IgG-AF488 </td><td>Jackson</td><td>Cat:711-546-152; RRID:<a href="https://identifiers.org/RRID/RRID:AB_2340619">AB_2340619</a></td><td>IF secondary (1:500)</td></tr><tr><td>Commercial </td><td>Estradiol ELISA</td><td>DRG</td><td>Cat:EIA-2693</td><td></td></tr><tr><td>Commercial </td><td>Progesterone ELISA</td><td>DRG</td><td>Cat:EIA-1561</td><td></td></tr><tr><td>Commercial assay or kit</td><td>Evercode WT Mega scRNA-seq kit (v1)</td><td>Parse Biosciences</td><td>Cat:EC-W01050</td><td></td></tr><tr><td>Software, algorithm</td><td>Python scripts for </td><td>This paper (<a href="#bib4">Brixi et al., 2023</a>)</td><td></td><td><a href="https://github.com/programmablebio/granulosa">https://github.com/programmablebio/granulosa</a>, </td></tr><tr><td>Software, algorithm</td><td>Parse Biosciences barcode </td><td>Parse Biosciences</td><td>Version 0.9.6</td><td></td></tr></tbody></table>
              
                  </div>
              
              
              
              </div>
          
          </figure>
    
    
    </div>
<section id="s4-1">

    <header>
      <h3>Cell culture</h3>
        <a href="https://bio-protocol.org/eLIFErap83291?item=s4-1">Request a detailed protocol</a>
    </header>

  <div>
      <p>Two parental hiPSC lines were used in this study: ATCC-BXS0116 female hiPSCs, which we refer to as the F3 line, and the F66 line, an in-house hiPSC line derived from the NIA Aging Cell Repository fibroblast line AG07141 using Epi5 footprint-free episomal reprogramming. The karyotypes of parental lines, as well as engineered reporter lines, were verified by Thermo Fisher Cell ID (single-nucleotide polymorphism-based authentication) + Karyostat, and pluripotency was assessed by Thermo Fisher Pluritest. All lines were identified as normal.</p>
<p>hiPSCs were cultured in mTESR Plus medium (Stemcell Technologies) on standard polystyrene plates coated with hESC-qualified Matrigel (Corning). Medium was changed daily. Passaging was performed using 0.5 mM ethylenediaminetetraacetic acid solution in phosphate-buffered saline (PBS), or TRYPLE for experiments requiring single-cell dissociation. hiPSCs were treated with 10 µM Y-27632 (Ambeed) for 24 hr after each passage. COV434 cells were cultured in Dulbecco’s modified Eagle medium (DMEM) + 10% fetal bovine serum (FBS) + 1× GlutaMax (Gibco). KGN cells (RIKEN, RCB1154) were cultured in DMEM/F12 + 10% FBS + 1× GlutaMax (Gibco). HGL5 cells (ABM cat. T0650) were cultured in Prigrow IV medium (ABM) with 10% FBS. Passaging was performed with TRYPLE (Gibco). hPGCLCs were cultured in S-CM medium as previously described (<a href="#bib25">Kobayashi et al., 2022</a>), and passaged with Accutase (Stemcell Technologies). Mycoplasma testing was performed by polymerase chain reaction (PCR) every 3 months; all cells tested negative.</p>





  </div>

</section>
<section id="s4-2">

    <header>
      <h3>Electroporations</h3>
        <a href="https://bio-protocol.org/eLIFErap83291?item=s4-2">Request a detailed protocol</a>
    </header>

  <p>Electroporations were performed using a Lonza Nucleofector with 96-well shuttle, with 200,000 cells in 20 µl of P3 buffer. Pulse setting CA-137 was used for all electroporations. Selection with the appropriate agent was begun 48 hr after electroporation and continued for 5 days. For the agents used in this study, this time was sufficient to give a high-purity final cell population.</p>

</section>
<section id="s4-3">

    <header>
      <h3>Reporter construction</h3>
        <a href="https://bio-protocol.org/eLIFErap83291?item=s4-3">Request a detailed protocol</a>
    </header>

  <div>
      <p>Homology arms for <i>FOXL2</i> were amplified by PCR from genomic DNA. A targeting plasmid, containing an in-frame C-terminal T2A-tdTomato reporter, as well as a Rox-PGK-PuroTK-Rox selection cassette (<a href="https://elifesciences.org/articles/83291/figures#fig2s1">Figure 2—figure supplement 1A</a>), was constructed by Gibson assembly. The plasmid backbone additionally had an MC1-DTA marker to select against random integration. sgRNA oligos targeting the C-terminal region of <i>FOXL2</i> were cloned into pX330 (Addgene #42230). For generation of the reporter lines, 1 µg donor plasmid and 1 µg sgRNA plasmid were co-electroporated into hiPSCs, which were subsequently plated in one well of a 6-well plate. After selection with puromycin (400 ng/ml), colonies were picked manually with a P20 pipette. The hiPSC lines generated were genotyped by PCR for the presence of wild-type and reporter alleles. Homozygous clones were further verified by PCR amplification of the entire <i>FOXL2</i> locus (<a href="https://elifesciences.org/articles/83291/figures#fig2s1">Figure 2—figure supplement 1B</a>) and Sanger sequencing.</p>
<p>To excise the selection cassette, hiPSCs were electroporated with pCAGGS-Dre (1 µg). Selection was performed with ganciclovir (4 µM) and colonies were picked as described above. The excision of the selection cassette was verified by genotyping. Primers used in this study are listed in <a href="https://elifesciences.org/articles/83291/figures#supp1">Supplementary file 1</a>.</p>





  </div>

</section>
<section id="s4-4">

    <header>
      <h3>TF plasmid construction</h3>
        <a href="https://bio-protocol.org/eLIFErap83291?item=s4-4">Request a detailed protocol</a>
    </header>

  <p>TF cDNAs were obtained from the TFome (<a href="#bib43">Ng et al., 2021</a>) or the ORFeome (<a href="#bib50">ORFeome Collaboration, 2016</a>) as Gateway entry clones. These were cloned into a barcoded Dox-inducible expression vector (Addgene #175503) using MegaGate cloning (<a href="#bib28">Kramme et al., 2021b</a>). The final expression constructs were verified by Sanger sequencing, which also served to determine the barcode sequences for each TF. Two unique barcodes were used per TF during library pooling. Libraries were pooled using an equimolar quantity of each plasmid (measured using QuBit).</p>

</section>
<section id="s4-5">

    <header>
      <h3>TF screening for granulosa differentiation</h3>
        <a href="https://bio-protocol.org/eLIFErap83291?item=s4-5">Request a detailed protocol</a>
    </header>

  <div>
      <p>A pooled library of barcoded TF plasmids was electroporated into FOXL2-tdTomato reporter hiPSCs, typically at 5 fmol library and 500 ng PiggyBac transposase expression plasmid (Systems Bio). These conditions were chosen to give an average copy number of approximately 5/cell (<a href="https://elifesciences.org/articles/83291/figures#fig2s2">Figure 2—figure supplement 2</a>). Some experiments were also performed at 50 fmol to explore the effects of higher copy numbers. For the screening data presented in <a href="#fig2">Figure 2</a>, two libraries were used: library #1, containing 35 TFs, and library #2, containing 18 TFs. Library #1 was used only at 5 fmol, whereas library #2 was used at both 5 and 50 fmol.</p>
<p>After selection with puromycin (400 ng/ml), hiPSCs were treated with doxycycline (1 µg/ml) in mTESR Plus medium. In additional experiments, hiPSCs were first differentiated to mesoderm following a previously published protocol (<a href="#bib41">Morizane and Bonventre, 2017</a>) before doxycycline treatment. In both sets of experiments, doxycycline treatment continued for 5 days, after which the cells were dissociated with TRYPLE and reporter-positive cells were isolated by FACS. Genomic DNA was extracted (QIAamp DNA Micro kit) from reporter-positive and -negative cells, as well as from the initial population before doxycycline treatment.</p>
<p>Barcodes were amplified by PCR (KAPA polymerase), using 10 ng input gDNA per reaction and typically 22 PCR cycles (95°C 15 s denature, 58°C 20 s anneal/extend). PCR products were purified using ProNex beads, and a second round of PCR (NEB Q5 polymerase, six cycles of 98°C 5 s denature, 61°C 20 s anneal, 72°C 5 s extend, final extension 72°C 2 min) was performed to add Illumina indices (primers are given in <a href="https://elifesciences.org/articles/83291/figures#supp1">Supplementary file 1</a>). These amplicons were again purified using ProNex beads. Samples were normalized and pooled, and barcodes were sequenced on an Illumina MiSeq with 10% PhiX spike-in. To call barcodes, reads were aligned to the set of known barcode sequences. Fold changes were calculated by comparing barcode frequencies in the sorted FOXL2+ cells to the frequencies in the starting population.</p>





  </div>

</section>
<section id="s4-6">

    <header>
      <h3>Flow cytometry/cell sorting</h3>
        <a href="https://bio-protocol.org/eLIFErap83291?item=s4-6">Request a detailed protocol</a>
    </header>

  <div>
      <p>Cells were dissociated by treatment with TRYPLE for 5 min, which was quenched with 4 volumes of ice-cold DMEM + 10% FBS. The suspension was passed through a 70-µm cell strainer. Cells were pelleted (200 g, 5 min) and resuspended in staining buffer (PBS + 3% FBS + antibodies, approx. 100 µl per million cells). Staining continued on ice in the dark for 30 min. The suspension was diluted with 9 volumes of PBS + 3% FBS. Cells were pelleted (200 g, 5 min) and resuspended in PBS + 3% FBS + 100 ng/ml 4′,6-diamidino-2-phenylindole (DAPI). The suspension was kept on ice in the dark until analysis. Flow cytometry was performed on a BD LSRFortessa, and sorting was performed on a Sony SH800 with 100-µm chip.</p>
<p>Antibody capture beads (BD Biosciences, RRID <a href="https://identifiers.org/RRID/RRID:AB_10051478">AB_10051478</a>), or hiPSCs expressing tdTomato, were used as compensation controls. Antibodies used are given in the Key Resources table. Data analysis was performed using the Cytoflow Python package (version 1.0.0, <a href="https://github.com/cytoflow/cytoflow">https://github.com/cytoflow/cytoflow</a>, <a href="#bib69">Teague, 2022</a>).</p>





  </div>

</section>
<section id="s4-7">

    <header>
      <h3>Optimized protocol for granulosa differentiation</h3>
        <a href="https://bio-protocol.org/eLIFErap83291?item=s4-7">Request a detailed protocol</a>
    </header>

  <p>iPSCs were dissociated with TRYPLE, and plated in DK10 medium (DMEM–F12, 15 mM 4-(2-hydroxyethyl)-1-piperazineethanesulfonic acid [HEPES], 1× GlutaMax, 10% knockout serum replacement [KSR]) with Y-27632 (10 µM), CHIR99021 (3 µM), and doxycycline (1 µg/ml) at a cell density of 12,500/cm<sup>2</sup> on Matrigel-coated polystyrene plates. For 24-well plates the medium volume per well was 0.5 ml; for 6-well plates it was 2 ml. 48 hr after plating, the medium was changed to DK10 + doxycycline (1 µg/ml), and the medium was subsequently changed every 24 hr. Cells were harvested on day 5 unless otherwise indicated. In the no-TF control differentiation for RNA-seq, the protocol was the same except the cells did not contain TF expression plasmids.</p>

</section>
<section id="s4-8">

    <header>
      <h3>RNA-seq</h3>
        <a href="https://bio-protocol.org/eLIFErap83291?item=s4-8">Request a detailed protocol</a>
    </header>

  <div>
      <p>Total RNA was extracted from sorted FOXL2+ granulosa-like cells using the Arcturus PicoPure kit (Thermo Fisher), or from COV434 cells and hiPSCs using the Monarch Total RNA Miniprep kit (NEB). For experiments involving TF overexpression, TF expression plasmids were integrated into hiPSCs as described above (50 fmol/200,000 cells). After selection with puromycin, TF expression was induced using doxycycline (1000 ng/ml).</p>
<p>Two biological replicates were collected for each sample (iPSC, hiPSC+ individual TFs, sorted FOXL2+, no-TF differentiation, KGN, COV434). Libraries were prepared using the NEBNext Ultra II Directional kit following the manufacturer’s protocol, and sequenced on an Illumina NextSeq 500 (2 × 75 bp paired-end reads). The TPM data shown in <a href="#fig3">Figure 3</a> were generated using kallisto (<a href="#bib3">Bray et al., 2016</a>) to pseudoalign reads to the reference human transcriptome (Ensembl GRCh38 v96). Differential expression analysis was performed using DESeq2. PantherDB (<a href="#bib38">Mi et al., 2021</a>) was used to calculate gene ontology enrichment for significantly upregulated (log<sub>2</sub>fc &gt;3, p<sub>adj</sub> &lt; 0.05) and downregulated (log<sub>2</sub>fc &lt;−3, p<sub>adj</sub> &lt; 0.05) genes for each sample relative to hiPSCs.</p>





  </div>

</section>
<section id="s4-9">

    <header>
      <h3>TROM analysis</h3>
        <a href="https://bio-protocol.org/eLIFErap83291?item=s4-9">Request a detailed protocol</a>
    </header>

  <p>The TROM method was employed to identify associated genes that capture molecular characteristics of biological samples and subsequently comparing the biological samples by testing the overlap of their associated genes (<a href="#bib33">Li et al., 2017</a>). TROM scores were calculated as the −log<sub>10</sub>(Bonferroni corrected p value of association) on a scale of 0–300. The TROM magnitude is positively correlated with similarity between two independent samples, with a standard threshold of 12 as a generally accepted indicator of significant similarity.</p>

</section>
<section id="s4-10">

    <header>
      <h3>Ovaroid formation with hPGCLCs and granulosa-like cells</h3>
        <a href="https://bio-protocol.org/eLIFErap83291?item=s4-10">Request a detailed protocol</a>
    </header>

  <div>
      <p>F2 female hPGCLCs (see Key Resources table) were maintained in long-term culture as previously described (<a href="#bib25">Kobayashi et al., 2022</a>). Briefly, hPGCLCs were cultured on Matrigel in STO-conditioned medium (Glasgow Minimum Essential Medium [GMEM] with 13% KSR and 1× non-essential amino acids, sodium pyruvate, and GlutaMax, all from Gibco), supplemented with stem cell factor (SCF) (100 ng/ml, Peprotech), ascorbic acid (50 µg/ml, Gibco), and 2-mercaptoethanol (25 µM, Gibco). hPGCLCs were harvested with Accutase. To form ovaroids, granulosa-like cells were harvested with TRYPLE, counted, and mixed with F2 hPGCLCs. For hormone assays in <a href="#fig3">Figure 3</a>, we used granulosa-like cells from F3/N.R1 #6, F66/N.R1.G.F #4, F66/N.R1.G #7, F66/N.R2 #1, F66/N.R2 #5, and F66/N.R2.G #3. For immunofluorescence experiments in <a href="#fig4">Figures 4</a> and <a href="#fig5">5</a>, we used F66/N.R1.G.F #4 and F66/N.R2 #1. For scRNA-seq in <a href="#fig6">Figure 6</a>, we used F66/N.R1.G.F #4.</p>
<p>For each ovaroid, 100,000 granulosa-like cells and 10,000 hPGCLCs were added to each well of a 96-well U-bottom low-bind plate (Corning #7007) in 200 µl of GK15 medium (GMEM, 15% KSR, with 1× GlutaMax, sodium pyruvate, and non-essential amino acids) supplemented with 10 mM Y-27632, 0.1 mM 2-mercaptoethanol, 1 µg/ml doxycycline, 100 ng/ml SCF, and 50 µg/ml primocin. The plate was centrifuged (100 × <i>g</i>, 2 min) and incubated (37°C, 5% CO<sub>2</sub>) for 2 days. Subsequently, ovaroids were transferred to Transwells (collagen-coated polytetrafluoroethylene, 3-µm pore size, 24-mm diameter, Corning #3492) for air–liquid interface culture with Alpha Minimum Essential Medium, 10% KSR, 55 µM 2-mercaptoethanol, 500 ng/ml doxycycline, and 50 µg/ml primocin. Typically five to six ovaroids were cultured on each 6-well Transwell. The medium (1.5 ml) was changed every 2 days.</p>





  </div>

</section>
<section id="s4-11">

    <header>
      <h3>Ovaroid formation with hPGCLCs and mouse fetal ovarian somatic cells</h3>
        <a href="https://bio-protocol.org/eLIFErap83291?item=s4-11">Request a detailed protocol</a>
    </header>

  <p>Fetal ovarian somatic cells were isolated from E12.5 female embryos of CD-1 mice (Charles River) as described by <a href="#bib75">Yamashiro et al., 2020</a>. For each ovaroid, 50,000 fetal ovarian somatic cells and 5000 F2 hPGCLCs were combined. Ovaroids were cultured as described above. All mouse experiments were approved by the Harvard Medical School Institutional Animal Care and Use Committee (IACUC).</p>

</section>
<section id="s4-12">

    <header>
      <h3>Immunofluorescence</h3>
        <a href="https://bio-protocol.org/eLIFErap83291?item=s4-12">Request a detailed protocol</a>
    </header>

  <div>
      <p>Ovaroids were washed with PBS and fixed with 1% paraformaldehyde solution in PBS overnight at 4°C. After another PBS wash, ovaroids were detached from the Transwell. In preparation for cryosectioning, ovaroids were transferred to 10% sucrose in PBS. After 24 hr at 4°C, the 10% sucrose solution was removed and replaced with 20% sucrose in PBS. After an additional 24 hr at 4°C, the ovaroids were embedded in OCT compound and stored at −80°C until sectioning.</p>
<p>The ovaroids were sectioned to 10 µm using a Leica CM3050S cryostat. Sections were transferred to Superfrost Plus slides, which were washed with PBS to remove OCT compound. The slides were washed with PBST (0.1% Triton X-100 in PBS) and sections were circled with a Pap pen. Slides were blocked for 30 min at room temp. with blocking buffer (1% bovine serum albumin and 5% normal donkey serum [Jackson ImmunoResearch, #017-000-121, lot #152961] in PBST). The blocking buffer was removed and replaced with a solution of primary antibodies in blocking buffer, and the slides were incubated overnight at 4°C. The antibody solution was removed and the slides were washed with PBST for 3 × 5 min. The slides were incubated with secondary antibody and DAPI solution in blocking buffer for 1 hr. at room temp. in the dark, followed by two 5-min washes with PBST and one wash with PBS. After staining, samples were mounted in Prolong Gold medium and covered with coverslips. Imaging was performed on a Leica SP5 confocal microscope. Antibodies used are given the Key Resources Table. Images were adjusted for brightness (and only for brightness) in ImageJ (version 2.9.0/1.53t), and cell counts for <a href="#fig4">Figure 4C</a> were performed manually by a researcher who was blinded to the species of the ovaroids.</p>





  </div>

</section>
<section id="s4-13">

    <header>
      <h3>Single-cell RNA sequencing</h3>
        <a href="https://bio-protocol.org/eLIFErap83291?item=s4-13">Request a detailed protocol</a>
    </header>

  <p>Ovaroids (6 ovaroids per sample, 2 samples per time point) were dissociated using the Miltenyi Embryoid Body Dissociation Kit (Miltenyi #130-096-348). The cells were passed through a 40-µm strainer, fixed using the Parse Biosciences fixation kit, and stored at −80°C until all time points had been collected. Libraries were prepared using the Parse Biosciences WT Mega v1 kit generating libraries of an average of 450 bp. The ovaroids took up 8 of the 96 samples; the remaining kit capacity was used for other experiments. The libraries were sequenced on an Illumina NovaSeq 2 × 150 bp S4 flow cell using single index, 6 bp, libraries and a 5% PhiX spike-in. Data were demultiplexed into library fastq files and counts matrices were generated using Parse Bioscience’s analysis pipeline (version 0.9.6). Downstream data processing, such as doublet filtering, dimensionality reduction, and clustering, was performed using Scanpy (version 1.8.2) (<a href="#bib73">Wolf et al., 2018</a>). For cell type assignment, the fetal ovarian dataset from the human reproductive cell atlas (<a href="#bib11">Garcia-Alonso et al., 2022</a>) was used as a reference for scanpy ingest.</p>

</section>
<section id="s4-14">

    <header>
      <h3>Collection of primary mouse ovarian somatic cells</h3>
        <a href="https://bio-protocol.org/eLIFErap83291?item=s4-14">Request a detailed protocol</a>
    </header>

  <p>Female BALB/c mice (age 10–12 weeks) were confirmed to be in proestrus by visual examination. Mice were killed by CO<sub>2</sub> exposure followed by cervical dislocation, and ovaries were removed by dissection. Ovaries were placed in HEPES-buffered DMEM/F12 with 0.1% bovine serum albumin (2 ovaries per 1.5-ml tube, with 500-µl medium) and mechanically disrupted by stabbing with forceps. The cell suspensions were strained through a 40-µm strainer to remove oocytes and clumps prior to culture for hormone assays.</p>

</section>
<section id="s4-15">

    <header>
      <h3>Steroid hormone assays</h3>
        <a href="https://bio-protocol.org/eLIFErap83291?item=s4-15">Request a detailed protocol</a>
    </header>

  <div>
      <p>Androstenedione (500 ng/ml) was added to the medium on day 4 of granulosa differentiation. FSH (0.25 IU/ml, BioVision #4781-50 lot 5F07L47810) or forskolin (100 µM, Sigma-Aldrich) were also added as indicated. The total medium volume was 0.5 ml per well of 24-well plate. We performed these assays on each of the lines listed in <a href="#fig3sdata2">Figure 3—source data 2</a>. For controls using human cell lines (COV434, KGN, or HGL5) or mouse primary ovarian somatic cells, 75,000 cells were seeded per well. After 24 hr, the medium was analyzed for estradiol content by ELISA (DRG International, EIA-2693). Concentrations were calculated with a 4-parameter logistic curve fit using the data from the standards provided in the kit. Samples outside the range of the calibration curve were diluted and re-run.</p>
<p>For measuring hormone production in ovaroids, ovaroids were aggregated as described above. Androstenedione (500 ng/ml) and/or FSH (0.25 IU/ml) were added to the aggregation medium (total volume 200 µl per ovaroid). After 3 days of culture, the medium was removed and analyzed by ELISA for estradiol (DRG International, EIA-2693) and progesterone (DRG International, EIA-1561). Hormone concentrations were calculated as described above.</p>





  </div>

</section>
<section id="s4-16">

    <header>
      <h3>Materials availability</h3>
        <a href="https://bio-protocol.org/eLIFErap83291?item=s4-16">Request a detailed protocol</a>
    </header>

  <p>Cell lines generated in this study are available for noncommercial use; contact the authors to negotiate a Material Transfer Agreement. Plasmids generated in this study are available via Addgene; see the Key Resources Table for accession numbers.</p>

</section>





  </div>

</section>


                
                    <section id="data" data-behaviour="ArticleSection">

    <header>
      <h2>Data availability</h2>
    </header>

  <div>
      <p>All data needed to evaluate the conclusions in the paper are present in the paper and supplementary tables and figures. Data analysis code can be found at: <a href="https://github.com/programmablebio/granulosa">https://github.com/programmablebio/granulosa</a>, (copy archived at <a href="https://archive.softwareheritage.org/swh:1:dir:23c3e0d1110d90ccffcd2d8ef5228702c8e02cb0;origin=https://github.com/programmablebio/granulosa;visit=swh:1:snp:d12df48bd51048f2ea79e8177e728bc4922d8d64;anchor=swh:1:rev:3c650290779db376c4d1f3a14960b08b17ae5561">swh:1:rev:3c650290779db376c4d1f3a14960b08b17ae5561</a>). Raw and processed sequencing data have been deposited to GEO under accession GSE213156.</p>
<p>
  The following data sets were generated
</p>
<ol>
    <li>
      
    </li>
</ol>
<p>
  The following previously published data sets were used
</p>
<ol>
    <li>
      
    </li>
    <li>
      
    </li>
    <li>
      
    </li>
    <li>
      
    </li>
</ol>





  </div>

</section>


                
                    <section id="references" data-behaviour="ArticleSection" data-initial-state="closed">

    <header>
      <h2>References</h2>
    </header>

  

</section>


                
                    <section id="sa1" data-behaviour="ArticleSection" data-initial-state="closed">

    <header>
      <h2>Decision letter</h2>
    </header>

  <div>
      
<p><b>Decision letter after peer review:</b></p>
<p>Thank you for submitting your article &#34;Directed Differentiation of Human iPSCs to Functional Ovarian Granulosa-Like Cells via Transcription Factor Overexpression&#34; for consideration by <i>eLife</i>. Your article has been reviewed by 3 peer reviewers, and the evaluation has been overseen by a Reviewing Editor and Marianne Bronner as the Senior Editor. The reviewers have opted to remain anonymous.</p>
<p>The reviewers have discussed their reviews with one another, and the Reviewing Editor has drafted this to help you prepare a revised submission.</p>
<p>Essential revisions:</p>
<p>1) Include a diagram to illustrate the experimental workflow of the study</p>
<p>2) Improve the quality of images and colors in Figure 3 and 4. In Figure 3, the <i>Foxl2</i>+ cells do not seem to be in a position similar to what is typically observed in vivo in ovaries.</p>
<p>3) Provide a clear rationale for using different endpoints for different cell lines.</p>
<p>4) The ovaroid data requires an OCT4+ and DAZL+ expression over the total analysis of the study.</p>
<p>5) The actual transcription factors used in the study need to be clearly described in the Discussion section.</p>
<p><i>Reviewer #1 (Recommendations for the authors):</i></p>
<p>I got lost at the beginning of the manuscript trying to understand exactly all the steps that led to the generation of the granulosa-like cells. It would be helpful to include a diagram to illustrate the experimental workflow of the study similar to a graphical abstract.</p>
<p>In Figure 3, I have several recommendations to facilitate the interpretation of the data</p>
<p>– The authors mention several times that they observe follicle-like formation in their ovaroid model, but in the IF images the <i>FOXL2</i>+ and AMHR2 staining is scattered and looks nothing like the spherical arrangement of squamous (primordial) or cuboidal (growing) cells one would expect to see in the in vivo ovary. In addition, there is no evidence in figures 3 or S5 that DAZL+ hPGCLCs become surrounded by <i>FOXL2</i>+ granulosa-like cells. I suggest replacing the images with better-quality IF stains, or the authors should soften the conclusion that follicle-like structures do form in their ovaroid model.</p>
<p>– The images in panel A Day 2 and Day 4 appear to be on a different scale than the rest, which is confusing.</p>
<p>– The merged images on the right are not legible, consider removing the DAPI channel from these so that the reader can better appreciate the staining overlap or lack thereof.</p>
<p>– In figure 4C, I find the integration onto the human cell atlas very helpful. However, it is confusing that the UMAP no longer corresponds to the cluster distribution shown in figure 4A. I would suggest adding an additional visual that shows the projection of the human cell atlas cell ID onto the original clusters of the ovaroid scRNAseq. Also, how do the authors explain that RUNX1 expression is very low in the <i>FOXL2</i>+ granulosa-like cluster (Figure 4B)?</p>
<p>Some of the data is difficult to evaluate in the figures.</p>
<p>– The bar histograms in figures 2d and e are very compact and difficult to tell apart. Also, the colors are not colorblind-friendly. I recommend changing the colors to more accessible color combinations in these and in figure 4D.</p>
<p>– The images of the ovaroids lack annotations to guide the readers to what is relevant, and the staining quality is suboptimal, making it difficult to interpret the similarity of these ovariods to in vivo ovaries.</p>
<p><i>Reviewer #2 (Recommendations for the authors):</i></p>
<p>This study from Smela et al. describes the production and functional characterisation of granulosa-like cells from induced pluripotent stem cells. The key claim is that two transcription factors – NR5A1, and either RUNX1 or RUNX2 – are collectively necessary and sufficient for the production of these cells (abstract line 9, results page 7 line 21, discussion page 16 line 9). While sufficiency for a selected set of characteristics is in part supported by functional and transcriptional analyses, the claim of necessity is not directly tested. If this claim of necessity is to stand, then the authors must clarify their use of this term, or alternatively, describe experiments they have undertaken that would justify this claim. Given the extensive data presented in figure 1b, the threshold used for this justification should also be described in much greater detail.</p>
<p>Impressively, this paper describes the rapid development of the germline when human-human chimeras are used, and this is a major strength of the study (figure 3). I do however find the use of the term &#34;ovaroid&#34; to be somewhat problematic. An ovary contains many more than two cell types, and a more nuanced term is strongly advised, potentially related to the production of a follicle-like structure.</p>
<p>While many suitable approaches are performed, the use of COV434 cells is not well-founded. As the authors note, there is some uncertainty regarding the identity of COV434 cells, and as a cancer cell type, are unlikely to reflect embryonic granulosa cells. A more appropriate control might be the starting iPS cells (as presented), or the various sublines developed, from which a more realistic assessment of the TROM value can be drawn. As presented, this transcriptional analysis cannot be used to justify the identity of the TF-induced granulosa-like cells. The authors acknowledge the issues with COV434 cells on page 9 lines 11-14, which further justifies an alternative approach.</p>
<p>The regulatory logic described in Figure 2c is not clearly described, and a much more rigorous description of the data underlying these findings should be included.</p>
<p>The section that describing the functional maturation of germline cells (via activation of DAZL and DDX4) is limited in scope, and the manuscript would benefit from orthogonal approaches to demonstrate the engraftment of these cells into follicle-like structures, or transcriptional analyses supporting the claims.</p>
<p><i>Reviewer #3 (Recommendations for the authors):</i></p>
<p>Additional Analyses:</p>
<p>1. Are multiple iPSC lines – biological variables – used to differentiate into granulosa-like cells? It seems this was only tested on lines from ATCC-BXS0116 and not confirmed in another biologically different line. Testing additional lines with the established protocol is necessary to increase robustness.</p>
<p>2. There are no images of ovaroids treated with androstenedione or FSH for the E2 and P4 experiments.</p>
<p>3. The ovaroid data was descriptive and requires an OCT4+ and DAZL+ cell over total analysis overtime to support the descriptive claims that these cell types underwent &#34;rapid development&#34; and &#34;declined over time&#34; and to become compare the results from the hPSCs against the murine fetal cells. While the authors do perform an analysis of Dazl-expressing cells in Figure 4D, this is not what is referred to in the paragraphs of results above and there are no comparisons with murine ovaroids here.</p>
<p>4. Which cells in the ovaroid come from the granulosa-like cell lines versus those that were differentiated into PGCLCs? It will be important to distinguish these populations in &#34;follicle-like&#34; cultures and to confirm that both are still present.</p>
<p>Editorial or data description suggestions:</p>
<p>1. &#34;The description of experimentation within the Results section must be more clear.&#34;</p>
<p>a. The data represented in Figure 2C and SFiles 2 must be presented in a form that demonstrates the relative fold change of expression for each doxy-inducible transcription factor lines in a main figure.</p>
<p>b. The description of iPSC lines that have one or more TF must be clearly stated. And the results must be clearly described for each line tested. For example, on page 11 &#34;The fourth of our granulosa lines produced high levels of estradiol in all conditions, and this line lacked the RUNX1 expression vector that was present in the others.&#34; Which four lines were used and in which order the authors are referring to them to deduce the &#34;fourth line&#34; was not made clear. Which TFs were in the lines that were picked for &#34;phenotype&#34; on page 12? First described on page 13 – which lines were used as the granulosa-like lines?</p>
<p>c. Figure 2 – 2 biological replicates for 4 clones? What does G1, G2, G3, G4 mean in Figure 2E? Are these different monoclonal granulosa-like cells from those shown in 2D? If so, why were they chosen?</p>
<p>2. It is not obvious which iPSC lines were used to generate the hPGCLCs or GLCs in the ovaroid cultures. This should be stated in the results and methods.</p>
<p>3. Page 13 lines 12-14 compare the expression of SOX17, OCT4, DAZL, and TFAP2C. However, the day in culture chosen for these IF analyses were different. This sentence should be reworded to more accurately state that OCT4 and DAZL expression was still visible on day 32 and SOX17 expression on day 8 in human…</p>
<p>4. Are OCT4- and DAZL+ oogonia visible on day 32? The line on page 13, line 14 states that &#34;at later timepoints (day 16), DaZL+…&#34; This is confusing because there is an image for DAZL and OCT4 antibodies at day 32 in Figure 3.</p>
<p>5. The germ cell populations in Figure 4D are not mentioned in the text.</p>
<p>6. The discussion should clearly summarize/restate the transcription factors used and steps to efficiently generate the granulosa-like cells from hiPSCs.</p>
<p>7. There needs to be clarity on which lines were used for each experiment in the Materials and methods.</p>




      <p><span>
          <a href="https://doi.org/10.7554/eLife.83291.sa1">
        https://doi.org/10.7554/eLife.83291.sa1
          </a>
      </span>

  </p></div>

</section>


                
                    <section id="sa2" data-behaviour="ArticleSection" data-initial-state="closed">

    <header>
      <h2>Author response</h2>
    </header>

  <div>
      <blockquote>
  <p>Essential revisions:</p>
<p>1) Include a diagram to illustrate the experimental workflow of the study</p>

  
</blockquote>
<p>We have made this diagram and included it as Figure 1.</p>
<blockquote>
  <p>2) Improve the quality of images and colors in Figure 3 and 4. In Figure 3, the Foxl2+ cells do not seem to be in a position similar to what is typically observed in vivo in ovaries.</p>

  
</blockquote>
<p>We have changed our graphs to have a colorblind-friendly palette. Additionally, we have improved the brightness of our immunofluorescence images (previously we were only showing the raw brightness values), and exported the figures at a higher resolution.</p>
<p>We have also collected additional immunofluorescence images from day 70 of ovaroid culture, which clearly show follicle-like structures (Figure 6).</p>
<blockquote>
  <p>3) Provide a clear rationale for using different endpoints for different cell lines.</p>

  
</blockquote>
<p>We have added this to the Discussion section. Also, we collected additional estradiol production and FSH response data, so that we now have that data for 9 cell lines instead of 4 as before.</p>
<blockquote>
  <p>4) The ovaroid data requires an OCT4+ and DAZL+ expression over the total analysis of the study.</p>

  
</blockquote>
<p>We have included this as Figure 5C.</p>
<blockquote>
  <p>5) The actual transcription factors used in the study need to be clearly described in the Discussion section.</p>

  
</blockquote>
<p>We have added this to the Discussion section, added a table showing which TFs are in which lines (Figure 3 – Source Data 2), and also provided PCR data verifying this (Figure 3—figure supplement 1C). Additionally, we have re-named our cell lines to be more informative about which TFs they express.</p>
<blockquote>
  <p>Reviewer #1 (Recommendations for the authors):</p>
<p>I got lost at the beginning of the manuscript trying to understand exactly all the steps that led to the generation of the granulosa-like cells. It would be helpful to include a diagram to illustrate the experimental workflow of the study similar to a graphical abstract.</p>

  
</blockquote>
<p>We have included this as Figure 1.</p>
<blockquote>
  <p>In Figure 3, I have several recommendations to facilitate the interpretation of the data</p>
<p>– The authors mention several times that they observe follicle-like formation in their ovaroid model, but in the IF images the FOXL2+ and AMHR2 staining is scattered and looks nothing like the spherical arrangement of squamous (primordial) or cuboidal (growing) cells one would expect to see in the in vivo ovary. In addition, there is no evidence in figures 3 or S5 that DAZL+ hPGCLCs become surrounded by FOXL2+ granulosa-like cells. I suggest replacing the images with better-quality IF stains, or the authors should soften the conclusion that follicle-like structures do form in their ovaroid model.</p>

  
</blockquote>
<p>We have collected additional immunofluorescence images from day 70 of ovaroid culture, which clearly show follicle-like structures of <i>FOXL2</i>+ AMHR2+ cells (Figure 5).</p>
<p>We agree that the follicles are empty (not containing hPGCLCs), and we mention this in the Results section.</p>
<blockquote>
  <p>– The images in panel A Day 2 and Day 4 appear to be on a different scale than the rest, which is confusing.</p>

  
</blockquote>
<p>We have swapped these for different images which have the same magnification as the rest.</p>
<blockquote>
  <p>– The merged images on the right are not legible, consider removing the DAPI channel from these so that the reader can better appreciate the staining overlap or lack thereof.</p>

  
</blockquote>
<p>We have done this, and also changed the colors to make them more legible.</p>
<blockquote>
  <p>– In figure 4C, I find the integration onto the human cell atlas very helpful. However, it is confusing that the UMAP no longer corresponds to the cluster distribution shown in figure 4A. I would suggest adding an additional visual that shows the projection of the human cell atlas cell ID onto the original clusters of the ovaroid scRNAseq.</p>

  
</blockquote>
<p>We have done this.</p>
<blockquote>
  <p>Also, how do the authors explain that RUNX1 expression is very low in the FOXL2+ granulosa-like cluster (Figure 4B)?</p>

  
</blockquote>
<p><i>RUNX1</i> expression is not very low in the granulosa cluster. In the original version of the violin plot (which is now figure 7B), the default setting of the width scaling factor was too small, which made it look like there wasn’t much <i>RUNX1</i>. We have since corrected this. The distribution of <i>RUNX1</i> expression is shown in <a href="#sa2fig1">Author response image 1</a>.</p>
    <div id="sa2fig1" data-variant="" data-behaviour="AssetNavigation AssetViewer ToggleableCaption" data-selector=".caption-text__body" data-asset-viewer-group="sa2fig1" data-asset-viewer-uri="https://iiif.elifesciences.org/lax:83291%2Felife-83291-sa2-fig1-v2.tif/full/full/0/default.jpg" data-asset-viewer-width="305" data-asset-viewer-height="256">
    
      
    
          <figure>
          
              <a href="https://iiif.elifesciences.org/lax:83291%2Felife-83291-sa2-fig1-v2.tif/full/full/0/default.jpg" target="_blank" rel="noopener noreferrer">
              <picture>
                  <source srcset="https://iiif.elifesciences.org/lax:83291%2Felife-83291-sa2-fig1-v2.tif/full/full/0/default.webp" type="image/webp"/>
                  <img src="https://iiif.elifesciences.org/lax:83291%2Felife-83291-sa2-fig1-v2.tif/full/full/0/default.jpg" alt=""/>
              </picture>
              </a>
          
          
          
          
          
          
          
          
          
          
          </figure>
    
    
    </div>
<blockquote>
  <p>Some of the data is difficult to evaluate in the figures.</p>
<p>– The bar histograms in figures 2d and e are very compact and difficult to tell apart. Also, the colors are not colorblind-friendly. I recommend changing the colors to more accessible color combinations in these and in figure 4D.</p>

  
</blockquote>
<p>We have changed the colors in these figures to a colorblind-friendly palette. Also, we separated the ELISA and bulk RNA-seq figures so that the bar plots can be bigger.</p>
<blockquote>
  <p>– The images of the ovaroids lack annotations to guide the readers to what is relevant, and the staining quality is suboptimal, making it difficult to interpret the similarity of these ovariods to in vivo ovaries.</p>

  
</blockquote>
<p>We have added annotations to better show the follicle-like structures in figure 6.</p>
<p>The figure files in the original manuscript were at lower resolution because there was a file size limit on the PDF upload. We have provided original-resolution versions of the figures in our resubmission. We note that the original versions of our immunofluorescence figures were simply the raw image files not adjusted for brightness/contrast. We have since adjusted the brightness of these images in the figures (as most other papers do), and this improves their interpretability. We have also chosen a better color palette. There is still some background staining in a few of the images but this should not affect the conclusions of our paper.</p>
<blockquote>
  <p>Reviewer #2 (Recommendations for the authors):</p>
<p>This study from Smela et al. describes the production and functional characterisation of granulosa-like cells from induced pluripotent stem cells. The key claim is that two transcription factors – NR5A1, and either RUNX1 or RUNX2 – are collectively necessary and sufficient for the production of these cells (abstract line 9, results page 7 line 21, discussion page 16 line 9). While sufficiency for a selected set of characteristics is in part supported by functional and transcriptional analyses, the claim of necessity is not directly tested. If this claim of necessity is to stand, then the authors must clarify their use of this term, or alternatively, describe experiments they have undertaken that would justify this claim. Given the extensive data presented in figure 1b, the threshold used for this justification should also be described in much greater detail.</p>

  
</blockquote>
<p>We have removed the claim of necessity, since it’s possible that other TFs outside of the set we tested could also produce granulosa-like cells.</p>
<blockquote>
  <p>Impressively, this paper describes the rapid development of the germline when human-human chimeras are used, and this is a major strength of the study (figure 3). I do however find the use of the term &#34;ovaroid&#34; to be somewhat problematic. An ovary contains many more than two cell types, and a more nuanced term is strongly advised, potentially related to the production of a follicle-like structure.</p>

  
</blockquote>
<p>Although our organoids do not contain all the cell types of the ovary, they do contain several important ones such as <i>FOXL2</i>+ granulosa-like cells, NR2F2+ stroma-like cells, and hPGCLCs. The name “ovaroid” is intended to describe a system resembling an ovary, not an actual ovary.</p>
<blockquote>
  <p>While many suitable approaches are performed, the use of COV434 cells is not well-founded. As the authors note, there is some uncertainty regarding the identity of COV434 cells, and as a cancer cell type, are unlikely to reflect embryonic granulosa cells. A more appropriate control might be the starting iPS cells (as presented), or the various sublines developed, from which a more realistic assessment of the TROM value can be drawn. As presented, this transcriptional analysis cannot be used to justify the identity of the TF-induced granulosa-like cells. The authors acknowledge the issues with COV434 cells on page 9 lines 11-14, which further justifies an alternative approach.</p>

  
</blockquote>
<p>We have collected additional RNA-seq data from KGN cells, which are phenotypically closer to bona fide granulosa cells in terms of steroidogenesis and marker gene expression. In our TROM analysis, the control is the starting iPSCs (which we find to have a TROM score of zero with all of the other samples).</p>
<blockquote>
  <p>The regulatory logic described in Figure 2c is not clearly described, and a much more rigorous description of the data underlying these findings should be included.</p>

  
</blockquote>
<p>We have clarified that this chart displays upregulation of genes of interest in response to TF overexpression. We have also included volcano plots to numerically show the regulatory effects.</p>
<blockquote>
  <p>The section that describing the functional maturation of germline cells (via activation of DAZL and DDX4) is limited in scope, and the manuscript would benefit from orthogonal approaches to demonstrate the engraftment of these cells into follicle-like structures, or transcriptional analyses supporting the claims.</p>

  
</blockquote>
<p>Although we now have strong evidence for the formation of follicle-like structures (Figure 6) we have not observed any germ cells inside of these follicles. We do mention that these follicles are empty in the Results section. Future studies using optimized culture conditions to promote germ cell survival will be necessary to address this. Regarding transcriptional analysis, we have now performed a differential expression and gene ontology enrichment analysis on <i>DAZL+</i> vs <i>DAZL-</i> germ cells in our ovaroids, which we describe in the Results section.</p>
<blockquote>
  <p>Reviewer #3 (Recommendations for the authors):</p>
<p>Additional Analyses:</p>
<p>1. Are multiple iPSC lines – biological variables – used to differentiate into granulosa-like cells? It seems this was only tested on lines from ATCC-BXS0116 and not confirmed in another biologically different line. Testing additional lines with the established protocol is necessary to increase robustness.</p>

  
</blockquote>
<p>We tested this on both F3 (ATCC-BXS0116) and F66 (an hiPSC line derived in-house from NIA Aging Cell Repository fibroblast line AG07141). A full list of lines is given in Figure 3 – Source Data 2.</p>
<blockquote>
  <p>2. There are no images of ovaroids treated with androstenedione or FSH for the E2 and P4 experiments.</p>

  
</blockquote>
<p>We tried to do whole-mount staining on these, but this was unsuccessful and we were not able to get any good images. We do have brightfield images of the ovaroids, but these are not very informative (the samples are too thick to see much detail), and there do not seem to be any major differences in appearance between the conditions.</p>
<blockquote>
  <p>3. The ovaroid data was descriptive and requires an OCT4+ and DAZL+ cell over total analysis overtime to support the descriptive claims that these cell types underwent &#34;rapid development&#34; and &#34;declined over time&#34; and to become compare the results from the hPSCs against the murine fetal cells. While the authors do perform an analysis of Dazl-expressing cells in Figure 4D, this is not what is referred to in the paragraphs of results above and there are no comparisons with murine ovaroids here.</p>

  
</blockquote>
<p>We have performed this analysis and it is now presented in Figure 5C.</p>
<blockquote>
  <p>4. Which cells in the ovaroid come from the granulosa-like cell lines versus those that were differentiated into PGCLCs? It will be important to distinguish these populations in &#34;follicle-like&#34; cultures and to confirm that both are still present.</p>

  
</blockquote>
<p>We do not currently have enough information to definitively assign the origins of the cells in the ovaroids. However, we consider it unlikely that DAZL+ cells could originate from our granulosa-like cells, since DAZL expression is highly specific to gonadal PGCs (see Refs. 15–17). Also we do not see <i>DAZL</i> expression in bulk RNA-seq data of our <i>FOXL2</i>+ cells (see Source Data for figure 3A).</p>
<blockquote>
  <p>Editorial or data description suggestions:</p>
<p>1. &#34;The description of experimentation within the Results section must be more clear.&#34;</p>
<p>a. The data represented in Figure 2C and SFiles 2 must be presented in a form that demonstrates the relative fold change of expression for each doxy-inducible transcription factor lines in a main figure.</p>

  
</blockquote>
<p>We have added volcano plots to Figure 3 which depict the fold-changes for genes of interest, including our transcription factors. Full data are provided in the source data for that figure.</p>
<blockquote>
  <p>b. The description of iPSC lines that have one or more TF must be clearly stated. And the results must be clearly described for each line tested. For example, on page 11 &#34;The fourth of our granulosa lines produced high levels of estradiol in all conditions, and this line lacked the RUNX1 expression vector that was present in the others.&#34; Which four lines were used and in which order the authors are referring to them to deduce the &#34;fourth line&#34; was not made clear. Which TFs were in the lines that were picked for &#34;phenotype&#34; on page 12? First described on page 13 – which lines were used as the granulosa-like lines?</p>

  
</blockquote>
<p>We changed our naming of the clones to clearly describe which TFs are present, and also made a list of all the clones in Figure 3 – Source Data 2.</p>
<blockquote>
  <p>c. Figure 2 – 2 biological replicates for 4 clones? What does G1, G2, G3, G4 mean in Figure 2E? Are these different monoclonal granulosa-like cells from those shown in 2D? If so, why were they chosen?</p>

  
</blockquote>
<p>We have tested additional clones (now all nine of our top clones, instead of just four). We also changed our naming of the clones to be more descriptive.</p>
<blockquote>
  <p>2. It is not obvious which iPSC lines were used to generate the hPGCLCs or GLCs in the ovaroid cultures. This should be stated in the results and methods.</p>

  
</blockquote>
<p>We have added this.</p>
<blockquote>
  <p>3. Page 13 lines 12-14 compare the expression of SOX17, OCT4, DAZL, and TFAP2C. However, the day in culture chosen for these IF analyses were different. This sentence should be reworded to more accurately state that OCT4 and DAZL expression was still visible on day 32 and SOX17 expression on day 8 in human…</p>

  
</blockquote>
<p>We reworded this following your suggestion.</p>
<blockquote>
  <p>4. Are OCT4- and DAZL+ oogonia visible on day 32? The line on page 13, line 14 states that &#34;at later timepoints (day 16), DaZL+…&#34; This is confusing because there is an image for DAZL and OCT4 antibodies at day 32 in Figure 3.</p>

  
</blockquote>
<p>We clarified that DAZL+/OCT4- and DAZL+/OCT4+ cells are both present.</p>
<blockquote>
  <p>5. The germ cell populations in Figure 4D are not mentioned in the text.</p>

  
</blockquote>
<p>The “germ cells” are the cells annotated as germ cells by our fetal ovary atlas integration. We have indicated this in the text.</p>
<blockquote>
  <p>6. The discussion should clearly summarize/restate the transcription factors used and steps to efficiently generate the granulosa-like cells from hiPSCs.</p>

  
</blockquote>
<p>We have added this.</p>
<blockquote>
  <p>7. There needs to be clarity on which lines were used for each experiment in the Materials and methods.</p>

  
</blockquote>
<p>We have added this.</p>




      <p><span>
          <a href="https://doi.org/10.7554/eLife.83291.sa2">
        https://doi.org/10.7554/eLife.83291.sa2
          </a>
      </span>

  </p></div>

</section>


                
                    <section id="info" data-behaviour="ArticleSection" data-initial-state="closed">

    <header>
      <h2>Article and author information</h2>
    </header>

  <div>
      <h3>Author details</h3>
<ol>
    <li><div data-popup-contents="xd43a8e74" id="xd43a8e74">

  <h4>Merrick D Pierson Smela</h4>

    <section>
        <ol>
            <li>Wyss Institute, Harvard University, Boston, United States</li>
            <li>Department of Genetics, Harvard Medical School, Boston, United States</li>
        </ol>
    </section>
    <section>
        <h5>Contribution</h5>
        <span>Conceptualization, Resources, Data curation, Software, Formal analysis, Validation, Investigation, Visualization, Methodology, Writing – original draft, Writing – review and editing</span>
    </section>
    <section>
        <h5>Contributed equally with</h5>
        <span>Christian C Kramme</span>
    </section>
    <section>
        <h5>Competing interests</h5>
        <span>is listed as an inventor for US Provisional Application No. 63/326,640, entitled &#39;Methods and Compositions for Producing Granulosa-Like Cells&#39;</span>
    </section>

  <section>
    <span>
      <a href="https://orcid.org/0000-0001-5816-7098">
        <picture>
          <source srcset="/assets/patterns/img/icons/orcid.b96370b9.svg" type="image/svg+xml"/>
          <img src="https://elifesciences.org/assets/patterns/img/icons/orcid.e0a7f9de.png" alt="ORCID icon"/>
        </picture> <span>&#34;This ORCID iD identifies the author of this article:&#34;</span>
    	 0000-0001-5816-7098</a>
    </span>
  </section>


</div>

</li>
    <li><div data-popup-contents="x579a51db" id="x579a51db">

  <h4>Christian C Kramme</h4>

    <section>
        <ol>
            <li>Wyss Institute, Harvard University, Boston, United States</li>
            <li>Department of Genetics, Harvard Medical School, Boston, United States</li>
        </ol>
    </section>
    <section>
        <h5>Contribution</h5>
        <span>Conceptualization, Resources, Investigation, Methodology</span>
    </section>
    <section>
        <h5>Contributed equally with</h5>
        <span>Merrick D Pierson Smela</span>
    </section>
    <section>
        <h5>Competing interests</h5>
        <span>is listed as an inventor for US Provisional Application No. 63/326,640, entitled &#39;Methods and Compositions for Producing Granulosa-Like Cells&#39;. CK is the VP of Cell Engineering of Gameto, Inc</span>
    </section>

  <section>
    <span>
      <a href="https://orcid.org/0000-0002-7518-8111">
        <picture>
          <source srcset="/assets/patterns/img/icons/orcid.b96370b9.svg" type="image/svg+xml"/>
          <img src="https://elifesciences.org/assets/patterns/img/icons/orcid.e0a7f9de.png" alt="ORCID icon"/>
        </picture> <span>&#34;This ORCID iD identifies the author of this article:&#34;</span>
    	 0000-0002-7518-8111</a>
    </span>
  </section>


</div>

</li>
    <li><div data-popup-contents="x3a159655" id="x3a159655">

  <h4>Patrick RJ Fortuna</h4>

    <section>
        <ol>
            <li>Wyss Institute, Harvard University, Boston, United States</li>
            <li>Department of Genetics, Harvard Medical School, Boston, United States</li>
        </ol>
    </section>
    <section>
        <h5>Contribution</h5>
        <span>Investigation, Visualization, Writing – review and editing</span>
    </section>
    <section>
        <h5>Competing interests</h5>
        <span>No competing interests declared</span>
    </section>



</div>

</li>
    <li><div data-popup-contents="xb046f643" id="xb046f643">

  <h4>Jessica L Adams</h4>

    <section>
        <ol>
            <li>Wyss Institute, Harvard University, Boston, United States</li>
            <li>Department of Genetics, Harvard Medical School, Boston, United States</li>
        </ol>
    </section>
    <section>
        <h5>Contribution</h5>
        <span>Investigation</span>
    </section>
    <section>
        <h5>Competing interests</h5>
        <span>No competing interests declared</span>
    </section>



</div>

</li>
    <li><div data-popup-contents="xf2f2708b" id="xf2f2708b">

  <h4>Rui Su</h4>

    <section>
        <ol>
            <li>Wyss Institute, Harvard University, Boston, United States</li>
            <li>Department of Genetics, Harvard Medical School, Boston, United States</li>
        </ol>
    </section>
    <section>
        <h5>Contribution</h5>
        <span>Investigation</span>
    </section>
    <section>
        <h5>Competing interests</h5>
        <span>No competing interests declared</span>
    </section>



</div>

</li>
    <li><div data-popup-contents="x3a5215cf" id="x3a5215cf">

  <h4>Edward Dong</h4>

    <section>
        <ol>
            <li>Wyss Institute, Harvard University, Boston, United States</li>
            <li>Department of Genetics, Harvard Medical School, Boston, United States</li>
        </ol>
    </section>
    <section>
        <h5>Contribution</h5>
        <span>Investigation</span>
    </section>
    <section>
        <h5>Competing interests</h5>
        <span>No competing interests declared</span>
    </section>



</div>

</li>
    <li><div data-popup-contents="xe026215f" id="xe026215f">

  <h4>Mutsumi Kobayashi</h4>

    <section>
        <span>Massachusetts General Hospital Center for Cancer Research, Harvard Medical School, Charlestown, United States</span>
    </section>
    <section>
        <h5>Contribution</h5>
        <span>Resources</span>
    </section>
    <section>
        <h5>Competing interests</h5>
        <span>is listed as an inventor for US Provisional Application No. 63/326,640, entitled &#39;Methods and Compositions for Producing Granulosa-Like Cells&#39;</span>
    </section>

  <section>
    <span>
      <a href="https://orcid.org/0000-0001-7897-2633">
        <picture>
          <source srcset="/assets/patterns/img/icons/orcid.b96370b9.svg" type="image/svg+xml"/>
          <img src="https://elifesciences.org/assets/patterns/img/icons/orcid.e0a7f9de.png" alt="ORCID icon"/>
        </picture> <span>&#34;This ORCID iD identifies the author of this article:&#34;</span>
    	 0000-0001-7897-2633</a>
    </span>
  </section>


</div>

</li>
    <li><div data-popup-contents="xb76a8c1c" id="xb76a8c1c">

  <h4>Garyk Brixi</h4>

    <section>
        <ol>
            <li>Wyss Institute, Harvard University, Boston, United States</li>
            <li>Department of Genetics, Harvard Medical School, Boston, United States</li>
            <li>Department of Biomedical Engineering, Duke University, Durham, United States</li>
            <li>Department of Computer Science, Duke University, Durham, United States</li>
        </ol>
    </section>
    <section>
        <h5>Contribution</h5>
        <span>Software</span>
    </section>
    <section>
        <h5>Competing interests</h5>
        <span>is listed as an inventor for US Provisional Application No. 63/326,640, entitled &#39;Methods and Compositions for Producing Granulosa-Like Cells&#39;</span>
    </section>



</div>

</li>
    <li><div data-popup-contents="xe0c33951" id="xe0c33951">

  <h4>Venkata Srikar Kavirayuni</h4>

    <section>
        <ol>
            <li>Department of Biomedical Engineering, Duke University, Durham, United States</li>
            <li>Department of Computer Science, Duke University, Durham, United States</li>
        </ol>
    </section>
    <section>
        <h5>Contribution</h5>
        <span>Software</span>
    </section>
    <section>
        <h5>Competing interests</h5>
        <span>is listed as an inventor for US Provisional Application No. 63/326,640, entitled &#39;Methods and Compositions for Producing Granulosa-Like Cells&#39;</span>
    </section>



</div>

</li>
    <li><div data-popup-contents="x7383e021" id="x7383e021">

  <h4>Emma Tysinger</h4>

    <section>
        <ol>
            <li>Department of Biomedical Engineering, Duke University, Durham, United States</li>
            <li>Department of Computer Science, Duke University, Durham, United States</li>
        </ol>
    </section>
    <section>
        <h5>Contribution</h5>
        <span>Software</span>
    </section>
    <section>
        <h5>Competing interests</h5>
        <span>is listed as an inventor for US Provisional Application No. 63/326,640, entitled &#39;Methods and Compositions for Producing Granulosa-Like Cells&#39;</span>
    </section>

  <section>
    <span>
      <a href="https://orcid.org/0000-0002-3958-8097">
        <picture>
          <source srcset="/assets/patterns/img/icons/orcid.b96370b9.svg" type="image/svg+xml"/>
          <img src="https://elifesciences.org/assets/patterns/img/icons/orcid.e0a7f9de.png" alt="ORCID icon"/>
        </picture> <span>&#34;This ORCID iD identifies the author of this article:&#34;</span>
    	 0000-0002-3958-8097</a>
    </span>
  </section>


</div>

</li>
    <li><div data-popup-contents="xa80dc85a" id="xa80dc85a">

  <h4>Richie E Kohman</h4>

    <section>
        <ol>
            <li>Wyss Institute, Harvard University, Boston, United States</li>
            <li>Department of Genetics, Harvard Medical School, Boston, United States</li>
        </ol>
    </section>
    <section>
        <h5>Contribution</h5>
        <span>Project administration</span>
    </section>
    <section>
        <h5>Competing interests</h5>
        <span>is listed as an inventor for US Provisional Application No. 63/326,640, entitled &#39;Methods and Compositions for Producing Granulosa-Like Cells&#39;</span>
    </section>

  <section>
    <span>
      <a href="https://orcid.org/0000-0002-7412-671X">
        <picture>
          <source srcset="/assets/patterns/img/icons/orcid.b96370b9.svg" type="image/svg+xml"/>
          <img src="https://elifesciences.org/assets/patterns/img/icons/orcid.e0a7f9de.png" alt="ORCID icon"/>
        </picture> <span>&#34;This ORCID iD identifies the author of this article:&#34;</span>
    	 0000-0002-7412-671X</a>
    </span>
  </section>


</div>

</li>
    <li><div data-popup-contents="xbfb8f171" id="xbfb8f171">

  <h4>Toshi Shioda</h4>

    <section>
        <span>Massachusetts General Hospital Center for Cancer Research, Harvard Medical School, Charlestown, United States</span>
    </section>
    <section>
        <h5>Contribution</h5>
        <span>Resources, Supervision, Writing – review and editing</span>
    </section>
    <section>
        <h5>Competing interests</h5>
        <span>is listed as an inventor for US Provisional Application No. 63/326,640, entitled &#39;Methods and Compositions for Producing Granulosa-Like Cells&#39;</span>
    </section>

  <section>
    <span>
      <a href="https://orcid.org/0000-0002-9434-7835">
        <picture>
          <source srcset="/assets/patterns/img/icons/orcid.b96370b9.svg" type="image/svg+xml"/>
          <img src="https://elifesciences.org/assets/patterns/img/icons/orcid.e0a7f9de.png" alt="ORCID icon"/>
        </picture> <span>&#34;This ORCID iD identifies the author of this article:&#34;</span>
    	 0000-0002-9434-7835</a>
    </span>
  </section>


</div>

</li>
    <li><div data-popup-contents="xf8ff436e" id="xf8ff436e">

  <h4>Pranam Chatterjee</h4>

    <section>
        <ol>
            <li>Wyss Institute, Harvard University, Boston, United States</li>
            <li>Department of Genetics, Harvard Medical School, Boston, United States</li>
            <li>Department of Biomedical Engineering, Duke University, Durham, United States</li>
            <li>Department of Computer Science, Duke University, Durham, United States</li>
        </ol>
    </section>
    <section>
        <h5>Contribution</h5>
        <span>Software, Project administration, Supervision, Writing – review and editing</span>
    </section>
    <section>
        <h5>Competing interests</h5>
        <span>is listed an an inventor for US Provisional Application No. 63/326,640, entitled &#39;Methods and Compositions for Producing Granulosa-Like Cells&#39;. PC is a co-founder and scientific advisor to Gameto, Inc</span>
    </section>

  <section>
    <span>
      <a href="https://orcid.org/0000-0003-3957-8478">
        <picture>
          <source srcset="/assets/patterns/img/icons/orcid.b96370b9.svg" type="image/svg+xml"/>
          <img src="https://elifesciences.org/assets/patterns/img/icons/orcid.e0a7f9de.png" alt="ORCID icon"/>
        </picture> <span>&#34;This ORCID iD identifies the author of this article:&#34;</span>
    	 0000-0003-3957-8478</a>
    </span>
  </section>


</div>

</li>
    <li><div data-popup-contents="xbf471ab2" id="xbf471ab2">

  <h4>George M Church</h4>

    <section>
        <ol>
            <li>Wyss Institute, Harvard University, Boston, United States</li>
            <li>Department of Genetics, Harvard Medical School, Boston, United States</li>
        </ol>
    </section>
    <section>
        <h5>Contribution</h5>
        <span>Supervision, Funding acquisition, Project administration, Writing – review and editing</span>
    </section>
    <section>
        <h5>For correspondence</h5>
        <span><a href="mailto:gchurch@genetics.med.harvard.edu">gchurch@genetics.med.harvard.edu</a></span>
    </section>
    <section>
        <h5>Competing interests</h5>
        <span>is listed as an inventor for US Provisional Application No. 63/326,640, entitled &#39;Methods and Compositions for Producing Granulosa-Like Cells&#39;. GMC serves on the scientific advisory board of Gameto, Inc, Colossal Biosciences, and GCTx</span>
    </section>

  <section>
    <span>
      <a href="https://orcid.org/0000-0003-3535-2076">
        <picture>
          <source srcset="/assets/patterns/img/icons/orcid.b96370b9.svg" type="image/svg+xml"/>
          <img src="https://elifesciences.org/assets/patterns/img/icons/orcid.e0a7f9de.png" alt="ORCID icon"/>
        </picture> <span>&#34;This ORCID iD identifies the author of this article:&#34;</span>
    	 0000-0003-3535-2076</a>
    </span>
  </section>


</div>

</li>
</ol>
<section>

    <header>
      <h3>Funding</h3>
    </header>

  <div>
      <section>

    <header>
      <h4>Wyss Institute (Synthetic Biology Platform)</h4>
    </header>

  

</section>
<section>

    <header>
      <h4>Gameto, Inc (Sponsored Research Agreement)</h4>
    </header>

  

</section>
<section>

    <header>
      <h4>National Science Foundation (Graduate Research Fellowship.)</h4>
    </header>

  

</section>
<p>The funders had no role in study design, data collection, and interpretation, or the decision to submit the work for publication.</p>





  </div>

</section>
<section>

    <header>
      <h3>Acknowledgements</h3>
    </header>

  <p>This work was funded by the Synthetic Biology Platform at the Wyss Institute for Biologically Inspired Engineering. This work was additionally funded by the Gameto Sponsored Research Agreement at Harvard University and Colossal Sponsored Research Agreement at Harvard University. MPS was supported by the National Science Foundation Graduate Research Fellowship. We thank Dr. Amanda Graveline and Oliver Dodd for assistance with mouse care and euthanasia. The KGN cell line was provided by the RIKEN BRC through the National BioResource Project of the MEXT, Japan.</p>

</section>
<section>

    <header>
      <h3>Ethics</h3>
    </header>

  <p>All protocols involving mice were approved by the Institutional Animal Care and Use Committee (IACUC) of Harvard Medical School, with reference number IS00001087.</p>

</section>
<section>

    <header>
      <h3>Senior Editor</h3>
    </header>

  <div>
      
    <ol>
            <li>Marianne E Bronner, California Institute of Technology, United States</li>
    </ol>







  </div>

</section>
<section>

    <header>
      <h3>Reviewing Editor</h3>
    </header>

  <div>
      
    <ol>
            <li>T Rajendra Kumar, University of Colorado, United States</li>
    </ol>







  </div>

</section>
<section>

    <header>
      <h3>Version history</h3>
    </header>

  <div>
      
    <ol>
            <li>Preprint posted: <a href="https://doi.org/10.1101/2022.07.04.498717">July 4, 2022 (view preprint)</a></li>
            <li>Received: September 6, 2022</li>
            <li>Accepted: January 18, 2023</li>
            <li>Version of Record published: <a href="https://elifesciences.org/articles/83291v1">February 21, 2023 (version 1)</a></li>
            <li>Version of Record updated: <a href="https://elifesciences.org/articles/83291">March 22, 2023 (version 2)</a></li>
    </ol>







  </div>

</section>
<section>

    <header>
      <h3>Copyright</h3>
    </header>

  <div>
      <p>© 2023, Pierson Smela, Kramme et al.</p><p>This article is distributed under the terms of the <a href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution License</a>, which permits unrestricted use and redistribution provided that the original author and source are credited.</p>




  </div>

</section>





  </div>

</section>


                
                    <section id="metrics" data-behaviour="ArticleSection" data-initial-state="closed">

    <header>
      <h2>Metrics</h2>
    </header>

  <div>
      <ul>
    <li>
      <dl>
        <dd>
          5,170
        </dd>
        <dt>
          Page views
        </dt>
      </dl>
    </li>
    <li>
      <dl>
        <dd>
          597
        </dd>
        <dt>
          Downloads
        </dt>
      </dl>
    </li>
    <li>
      <dl>
        <dd>
          7
        </dd>
        <dt>
          Citations
        </dt>
      </dl>
    </li>
</ul>


<p>Article citation count generated by polling the highest count across the following sources: <a href="https://doi.org/10.7554/eLife.83291">Crossref</a>, <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9943069/">PubMed Central</a>, <a href="">Scopus</a>.</p>





  </div>

</section>


                
                    <section>

    <header>
      <h2>Download links</h2>
    </header>

  <div>
      <div data-behaviour="ArticleDownloadLinksList" id="downloads" aria-labelledby="downloads-label">
  <p><span id="downloads-label">A two-part list of links to download the article, or parts of the article, in various formats.</span></p>

    <div>
      <h3 id="downloads">Downloads<span> (link to download the article as PDF)</span></h3>


      
    </div>
    <div>
      <h3 id="downloads">Open citations<span> (links to open the citations from this article in various online reference manager services)</span></h3>


      
    </div>
    <div>
      <h3 id="cite-this-article">Cite this article<span> (links to download the citations from this article in formats compatible with various reference manager tools)</span></h3>

        <div><div>
    <ol>
        <li>
          Merrick D Pierson Smela</li>
        <li>
          Christian C Kramme</li>
        <li>
          Patrick RJ Fortuna</li>
        <li>
          Jessica L Adams</li>
        <li>
          Rui Su</li>
        <li>
          Edward Dong</li>
        <li>
          Mutsumi Kobayashi</li>
        <li>
          Garyk Brixi</li>
        <li>
          Venkata Srikar Kavirayuni</li>
        <li>
          Emma Tysinger</li>
        <li>
          Richie E Kohman</li>
        <li>
          Toshi Shioda</li>
        <li>
          Pranam Chatterjee</li>
        <li>
          George M Church</li>
    </ol>
    <p><span>(2023)</span></p><p>Directed differentiation of human iPSCs to functional ovarian granulosa-like cells via transcription factor overexpression</p>



    <p><i>eLife</i> <b>12</b>:e83291.</p>

    <p><span>
      https://doi.org/10.7554/eLife.83291
    </span>



</p></div>
</div>

      
    </div>

</div>





  </div>

</section>


                
                    
<section>

  

</section>


                
                
            
        

        </div></div>
  </body>
</html>
