<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <title>James Routley | Feed</title>
    <link
      rel="stylesheet"
      type="text/css"
      href="../styles.css"
      media="screen"
    />
  </head>
  <body>
    <a href="/index.html">Back</a>
    <a href="https://www.nature.com/articles/s41593-023-01361-0">Original</a>
    <h1>Gut–brain axis study shows autism-associated molecular, microbial profiles</h1>
    
    <div id="readability-page-1" class="page"><div>
                <section data-title="Main"><div id="Sec1-section"><h2 id="Sec1">Main</h2><div id="Sec1-content"><p>Autism spectrum disorder (ASD) encompasses a broad range of neurodevelopmental conditions defined by heterogeneous cognitive, behavioral and communication impairments that manifest early in childhood<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 1" title="Lord, C. et al. Autism spectrum disorder. Nat. Rev. Dis. Primers 6, 5 (2020)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR1" id="ref-link-section-d23130190e1804">1</a></sup>. To date, over 100 genes have been identified as putatively associated with ASD, with some genotypes now having a standardized clinical diagnosis<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 2" title="Satterstrom, F. K. et al. Large-scale exome sequencing study implicates both developmental and functional changes in the neurobiology of autism. Cell 180, 568–584 (2020)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR2" id="ref-link-section-d23130190e1808">2</a></sup>. However, most of the genetic variants are still associated with heterogeneous phenotypes, making it difficult to identify molecular mechanisms that might be responsible for particular impairments<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 3" title="Iakoucheva, L. M., Muotri, A. R. &amp; Sebat, J. Getting to the cores of autism. Cell 178, 1287–1298 (2019)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR3" id="ref-link-section-d23130190e1812">3</a></sup>. Some studies have also looked at the presence of abnormalities in different brain regions in children with ASD<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 4" title="Schumann, C. M. et al. The amygdala is enlarged in children but not adolescents with autism; the hippocampus is enlarged at all ages. J. Neurosci. 24, 6392–6401 (2004)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR4" id="ref-link-section-d23130190e1816">4</a></sup>. However, whether such neuroanatomical features could mechanistically determine autism, and whether environmental factors could induce analogous ASD-like symptoms, remain unresolved<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 1" title="Lord, C. et al. Autism spectrum disorder. Nat. Rev. Dis. Primers 6, 5 (2020)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR1" id="ref-link-section-d23130190e1820">1</a></sup>.</p><p>In addition to risk factors, one comorbidity that has been linked to ASD with high confidence is the occurrence of gastrointestinal (GI) symptoms, such as constipation, diarrhea or abdominal bloating, but causal insights remain elusive<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 5" title="Lefter, R., Ciobica, A., Timofte, D., Stanciu, C. &amp; Trifan, A. A descriptive review on the prevalence of gastrointestinal disturbances and their multiple associations in autism spectrum disorder. Medicina (Kaunas) 56, 11 (2019)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR5" id="ref-link-section-d23130190e1827">5</a></sup>. Mechanistically, much research has been focused on the interplay between the GI system and processes controlled by the neuroendocrine, neuroimmune and autonomous nervous systems, all of which converge around the GI tract and together modulate the gut–brain axis (GBA)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 6" title="Morais, L. H., Schreiber IV, H. L. &amp; Mazmanian, S. K. The gut microbiota–brain axis in behaviour and brain disorders. Nat. Rev. Microbiol. 19, 241–255 (2021)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR6" id="ref-link-section-d23130190e1831">6</a></sup>.</p><p>The GBA facilitates bidirectional communication between the gut and the brain, contributing to brain homeostasis and helping regulate cognitive and emotional functions<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Mayer, E. A. Gut feelings: the emerging biology of gut–brain communication. Nat. Rev. Neurosci. 12, 453–466 (2011)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR7" id="ref-link-section-d23130190e1838">7</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 8" title="Sarkar, A. et al. The role of the microbiome in the neurobiology of social behaviour. Biol. Rev. Camb. Philos. Soc. 95, 1131–1166 (2020)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR8" id="ref-link-section-d23130190e1841">8</a></sup>. Over the past decade, research on the factors modulating the GBA has revealed the central role played by the gut microbiome—the trillions of microbes that colonize the gut—in regulating neuroimmune networks, modifying neural networks and directly communicating with the brain<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 9" title="Jacobson, A., Yang, D., Vella, M. &amp; Chiu, I. M. The intestinal neuro–immune axis: crosstalk between neurons, immune cells, and microbes. Mucosal Immunol. 14, 555–565 (2021)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR9" id="ref-link-section-d23130190e1845">9</a></sup>. Dysregulation of the gut microbiome and the ensuing disruption of the GBA are thought to contribute to the pathogenesis of neurodevelopmental disorders, including autism, but the underlying mechanisms and the extent to which the microbiome explains these dynamics are still unclear<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 10" title="Xu, M., Xu, X., Li, J. &amp; Li, F. Association between gut microbiota and autism spectrum disorder: a systematic review and meta-analysis. Front. Psychiatry 10, 473 (2019)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR10" id="ref-link-section-d23130190e1849">10</a></sup>.</p><p>Several dozen autism gut metagenomics studies have revealed many, albeit inconsistent, variations in microbial diversity in individuals with ASD compared to neurotypical individuals<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 10" title="Xu, M., Xu, X., Li, J. &amp; Li, F. Association between gut microbiota and autism spectrum disorder: a systematic review and meta-analysis. Front. Psychiatry 10, 473 (2019)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR10" id="ref-link-section-d23130190e1856">10</a></sup>. Similarly, metagenome-based functional reconstructions and metabolic analyses have also shown strong, albeit inconclusive, differences between individuals with ASD and neurotypical individuals<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 11" title="Xu, X.-J. et al. Comparison of the metabolic profiles in the plasma and urine samples between autistic and typically developing boys: a preliminary study. Front. Psychiatry 12, 657105 (2021)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR11" id="ref-link-section-d23130190e1860">11</a></sup>. Comparative analyses at other omic levels have further shown little agreement across studies<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 12" title="Azhari, A., Azizan, F. &amp; Esposito, G. A systematic review of gut-immune-brain mechanisms in autism spectrum disorder. Dev. Psychobiol. 61, 752–771 (2019)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR12" id="ref-link-section-d23130190e1864">12</a></sup>, raising the question of whether the results obtained so far reflect intrinsic biological differences among cohorts, insufficient statistical power or experimental biases that preclude meaningful comparisons<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 13" title="Schloss, P. D. Identifying and overcoming threats to reproducibility, replicability, robustness, and generalizability in microbiome research. mBio 9, e00525-18 (2018)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR13" id="ref-link-section-d23130190e1868">13</a></sup>.</p><p>A wide range of factors could explain the disagreement across studies, including confounding variation due to batch effects, the application of inappropriate statistical methodologies and the vast phenotypic and genotypic heterogeneity of ASD. Batch effects can be caused by many factors, including mis-specified experimental designs, technical variability, geographical location and demographic composition, and several algorithms have been proposed to correct for them, but a lack of standardized statistical methods further complicates interpretation<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 14" title="Wang, Y. &amp; LêCao, K.-A. Managing batch effects in microbiome data. Brief. Bioinform. 21, 1954–1970 (2020)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR14" id="ref-link-section-d23130190e1876">14</a></sup>. Microbiome datasets, like other omic datasets, are compositional, and failure to account for the compositional nature of sequencing counts can lead to high false-positive and false-negative rates when identifying differentially abundant microbes<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 15" title="Morton, J. T. et al. Establishing microbial composition measurement standards with reference frames. Nat. Commun. 10, 2719 (2019)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR15" id="ref-link-section-d23130190e1880">15</a></sup>. Microbiome analysis in ASD is further confounded by the phenotypic and genotypic heterogeneity of the disorder, which is known to be critical for stratifying ASD subtypes and constructing reliable diagnostics, but is typically not measured or controlled for<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 1" title="Lord, C. et al. Autism spectrum disorder. Nat. Rev. Dis. Primers 6, 5 (2020)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR1" id="ref-link-section-d23130190e1884">1</a></sup>.</p><p>Understanding the functional architecture—the network of interactions among different omic levels that determines individual phenotypes—of complex neurodevelopmental disorders, such as autism, requires an accurate and comprehensive characterization of the different omic levels contributing to it<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 16" title="Sullivan, P. F. &amp; Geschwind, D. H. Defining the genetic, genomic, cellular, and diagnostic architectures of psychiatric disorders. Cell 177, 162–183 (2019)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR16" id="ref-link-section-d23130190e1891">16</a></sup>. Traditionally focused on the human genomic, metabolic and cellular components, mounting evidence of the role the GBA plays in phenotype determination raises the need for considering the metagenomic and metabolic contributions of the microbiome as potential key components of the functional architecture of autism<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 17" title="Blekhman, R. et al. Host genetic variation impacts microbiome composition across human body sites. Genome Biol. 16, 191 (2015)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR17" id="ref-link-section-d23130190e1895">17</a></sup>.</p><p>To identify autism-specific omic profiles while reducing cohort-specific confounding factors, we devised a Bayesian differential ranking algorithm to estimate a distribution of microbial differentials, or relative log fold changes<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 15" title="Morton, J. T. et al. Establishing microbial composition measurement standards with reference frames. Nat. Commun. 10, 2719 (2019)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR15" id="ref-link-section-d23130190e1902">15</a></sup>, across multiple potential ASD subtypes implicit in 25 omic datasets (Table <a data-track="click" data-track-label="link" data-track-action="table anchor" href="https://bellmar.medium.com/articles/s41593-023-01361-0#Tab1">1</a>). Ranking microbes by their log fold changes allows us to simultaneously (1) cancel out the compositional bias inherent in microbiome datasets and (2) minimize inflated false positives due to microbe-specific false discovery rate (FDR)-corrected statistical tests<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 15" title="Morton, J. T. et al. Establishing microbial composition measurement standards with reference frames. Nat. Commun. 10, 2719 (2019)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR15" id="ref-link-section-d23130190e1909">15</a></sup>. A key feature of our approach was to match individual study participants by sex and age within each study to adjust for confounders in childhood development. This setup also reduced confounding variation due to cohort-specific processing protocols, because within-study fold change calculations are insensitive to batch effects<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 18" title="McLaren, M. R., Willis, A. D. &amp; Callahan, B. J. Consistent and correctable bias in metagenomic sequencing experiments. eLife 8, e46923 (2019)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR18" id="ref-link-section-d23130190e1913">18</a></sup> (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bellmar.medium.com/articles/s41593-023-01361-0#Fig6">2</a>). The preponderance of autism among males is well documented, and several potentially sex-dependent mechanisms to explain this phenomenon have been proposed. Furthermore, the development of the microbiome during childhood is a hallmark of microbiome dynamics in the human gut. Our analysis reveals strong associations among omic levels along the GBA and in particular of the microbiome in the context of ASD. Ultimately, our analysis highlights the inherent limitations of cross-sectional studies for understanding the dynamics of the functional architecture of autism and provides a framework for future studies aimed at better defining the causal relationship between the microbiome and other omic levels and ASD.</p><div data-test="inline-table" data-container-section="table" id="table-1"><figure><figcaption><b id="Tab1" data-test="table-caption">Table 1 ASD omic datasets included in this study. All sequencing datasets were retrieved from the SRA</b></figcaption></figure></div></div></div></section><section data-title="Results"><div id="Sec2-section"><h2 id="Sec2">Results</h2><div id="Sec2-content"><p>The structure of our analysis consisted of a multi-cohort and multi-omic meta-analysis framework that allowed us to combine independent and dependent omic datasets in one integrated analysis<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 19" title="Shafi, A., Nguyen, T., Peyvandipour, A., Nguyen, H. &amp; Draghici, S. A multi-cohort and multi-omics meta-analysis framework to identify network-based gene signatures. Front. Genet. 10, 159 (2019)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR19" id="ref-link-section-d23130190e2422">19</a></sup>. To minimize issues of compositionality and sequencing depth<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 20" title="Gloor, G. B., Macklaim, J. M., Pawlowsky-Glahn, V. &amp; Egozcue, J. J. Microbiome datasets are compositional: and this is not optional. Front. Microbiol. 8, 2224 (2017)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR20" id="ref-link-section-d23130190e2426">20</a></sup>, we modeled sequencing count data using a negative binomial distribution<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 21" title="Love, M. I., Huber, W. &amp; Anders, S. Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. Genome Biol. 15, 550 (2014)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR21" id="ref-link-section-d23130190e2430">21</a></sup> (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bellmar.medium.com/articles/s41593-023-01361-0#Fig5">1</a>, ‘Study approach’). Our differential ranking approach incorporated a case–control matching component that individually paired children with ASD with age-matched and sex-matched neurotypical control children within each study cohort to adjust for confounding variation and batch effects (<a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://bellmar.medium.com/articles/s41593-023-01361-0#MOESM1">Supplementary Information</a>). Finally, we cross-referenced the microbial differential rankings estimated from 16S rRNA gene (16S) amplicon data obtained from seven age-matched and sex-matched cohorts against 15 other omic datasets to contextualize the potential functional roles that these microbes could play in autism (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bellmar.medium.com/articles/s41593-023-01361-0#Fig1">1</a>).</p><div data-test="figure" data-container-section="figure" id="figure-1" data-title="Diagram delineating the concept of age matching and sex matching."><figure><figcaption><b id="Fig1" data-test="figure-caption-text">Fig. 1: Diagram delineating the concept of age matching and sex matching.</b></figcaption><div><div><a data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="https://bellmar.medium.com/articles/s41593-023-01361-0/figures/1" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-023-01361-0/MediaObjects/41593_2023_1361_Fig1_HTML.png?as=webp"/><img aria-describedby="Fig1" src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-023-01361-0/MediaObjects/41593_2023_1361_Fig1_HTML.png" alt="figure 1" loading="lazy" width="685" height="559"/></picture></a></div><p><b>a</b>, Children with ASD and neurotypical children of the same gender and similar age (±6 months) were matched within studies to reduce batch effects due to experimental and other cohort-specific differences. Matched pairs were then used to compute differentials (log fold ratios) of different omic features (microbes, metabolites, etc.). Downstream analyses across studies compared the within-study differentials determined for the different pairs of matched individuals (numbers inside circles denote age in years). <b>b</b>, The structure of our meta-analysis across multiple omic levels. For Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bellmar.medium.com/articles/s41593-023-01361-0#Fig2">2</a>, 16S differentials computed from age-matched and sex-matched cohorts were cross-referenced against 16S differentials from sibling-matched cohorts as well as against SMS differentials from other age-matched and sex-matched cohorts. For Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bellmar.medium.com/articles/s41593-023-01361-0#Fig3">3</a>, the 16S differentials from the age-matched and sex-matched cohorts were cross-referenced against cytokine differentials and RNA-seq differentials using KEGG pathways as a reference. Figure <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bellmar.medium.com/articles/s41593-023-01361-0#Fig3">3</a> also includes a microbe–diet co-occurrence analysis. For Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bellmar.medium.com/articles/s41593-023-01361-0#Fig4">4</a>, the 16S differentials from the age-matched and sex-matched cohorts were cross-referenced against 16S differentials computed from the Kang et al. FMT trial<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 52" title="Kang, D.-W. et al. Long-term benefit of microbiota transfer therapy on autism symptoms and gut microbiota. Sci. Rep. 9, 5821 (2019)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR52" id="ref-link-section-d23130190e2474">52</a></sup>.</p></div></figure></div><h3 id="Sec3">Age matching and sex matching enhance ASD data analysis</h3><p>To establish the validity and robustness of our age-matched and sex-matched Bayesian differential ranking approach, we performed a series of benchmarking exercises and sensitivity tests.</p><p>We started by investigating the means and the standard deviations for the 16S and shotgun metagenomics sequencing (SMS) differentials from the age-matched and sex-matched cohorts compared to the total sequencing depth for each microbe (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bellmar.medium.com/articles/s41593-023-01361-0#Fig6">2a–d</a>). In both analyses, we observed that the models could use sequencing depth to calibrate the uncertainty estimates, giving larger standard deviations for rare taxa with fewer observed reads (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bellmar.medium.com/articles/s41593-023-01361-0#Fig6">2b,d</a>). Furthermore, rare taxa (with fewer than 100 reads total) were not among the most differentially abundant ASD-associated taxa (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bellmar.medium.com/articles/s41593-023-01361-0#Fig6">2a,c</a>).</p><p>Next, we performed a rarefaction benchmark to test whether the high frequency of rare taxa would influence the results of our log fold change calculations. A comparison of differential abundance estimates between rarefied (9,000 threshold) and non-rarefied data from our 16S cross-sectional datasets showed that rarefaction did not substantially affect our results (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bellmar.medium.com/articles/s41593-023-01361-0#Fig6">2e</a>). We also conducted a data-driven simulation with varying differential sequencing depths between cases and controls and showed that, despite the sequencing depth confounder, our differential abundance method could accurately recover the ground truth log fold changes (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bellmar.medium.com/articles/s41593-023-01361-0#Fig6">2f</a>). We then compared performance of our age-matched and sex-matched differential ranking analysis to the standard group-averaged differential ranking analysis across seven out of the 11 16S studies<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Chen, Y. et al. Gut bacteria shared by children and their mothers associate with developmental level and social deficits in autism spectrum disorder. mSphere 5, e01044–20 (2020)." href="#ref-CR22" id="ref-link-section-d23130190e2514">22</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Fouquier, J. et al. The gut microbiome in autism: study-site effects and longitudinal analysis of behavior change. mSystems 6, e00848-20 (2021)." href="#ref-CR23" id="ref-link-section-d23130190e2514_1">23</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Zurita, M. F. et al. Analysis of gut microbiome, nutrition and immune status in autism spectrum disorder: a case–control study in Ecuador. Gut Microbes 11, 453–464 (2020)." href="#ref-CR24" id="ref-link-section-d23130190e2514_2">24</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Berding, K. &amp; Donovan, S. M. Dietary patterns impact temporal dynamics of fecal microbiota composition in children with autism spectrum disorder. Front. Nutr. 6, 193 (2019)." href="#ref-CR25" id="ref-link-section-d23130190e2514_3">25</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Kang, D.-W. et al. Microbiota transfer therapy alters gut ecosystem and improves gastrointestinal and autism symptoms: an open-label study. Microbiome 5, 10 (2017)." href="#ref-CR26" id="ref-link-section-d23130190e2514_4">26</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Zou, R. et al. Changes in the gut microbiota of children with autism spectrum disorder. Autism Res. 13, 1614–1625 (2020)." href="#ref-CR27" id="ref-link-section-d23130190e2514_5">27</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Dan, Z. et al. Altered gut microbial profile is associated with abnormal metabolism activity of autism spectrum disorder. Gut Microbes 11, 1246–1267 (2020)." href="#ref-CR28" id="ref-link-section-d23130190e2514_6">28</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 29" title="Fecal microbiome of children with autism spectrum disorder, their mothers, and healthy children. 
                  https://www.ncbi.nlm.nih.gov/bioproject/PRJNA687773
                  
                " href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR29" id="ref-link-section-d23130190e2517">29</a></sup> (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bellmar.medium.com/articles/s41593-023-01361-0#Fig6">2g</a>). A side-by-side comparison with a commonly used differential abundance approach, ANCOM-BC (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 30" title="Lin, H. &amp; Peddada, S. D. Analysis of compositions of microbiomes with bias correction. Nat. Commun. 11, 3514 (2020)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR30" id="ref-link-section-d23130190e2524">30</a></sup>), was then conducted to highlight the differences between our methodology and one of the state-of-the-art differential abundance methods (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bellmar.medium.com/articles/s41593-023-01361-0#Fig6">2i–k</a>).</p><p>We benchmarked the overall batch-effect-reducing power of performing within-study differential analyses with our sex-matched and age-matched Bayesian differential ranking approach. We used the MicroBiome Quality Control (MBQC) study (Sinha et al. 2017 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 31" title="Sinha, R. et al. Assessment of variation in microbial community amplicon sequencing by the microbiome quality control (mbqc) project consortium. Nat. Biotechnol. 35, 1077–1086 (2017)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR31" id="ref-link-section-d23130190e2535">31</a></sup>)) to evaluate the extent to which within-study differential analysis removed experimental and other study-related confounders, allowing for meaningful comparisons across independent studies. Focusing on the microbial abundance datasets (16S microbial counts) generated by four independent laboratories (Lab A, Lab B, Lab C and Lab D) processing two identical MBQC microbiome samples (samples ‘4’ and ‘6’), we calculated the differentials between microbial counts for these samples for each laboratory. An initial assessment of overall variability between the two samples (principal coordinate analysis (PCoA) plot with Bray–Curtis dissimilarity) (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bellmar.medium.com/articles/s41593-023-01361-0#Fig6">2l</a>) showed a reasonable separation between both samples just based on microbial counts, but a visualization of study membership revealed that a significant degree of variability was associated with the laboratory that generated the dataset (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bellmar.medium.com/articles/s41593-023-01361-0#Fig6">2n</a>). Given that the metagenomic samples by each laboratory were identical, the high level of variability observed among datasets could be ascribed only to experimental and laboratory-specific batch effects. Consistent with the theoretical findings of McLaren et al.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 18" title="McLaren, M. R., Willis, A. D. &amp; Callahan, B. J. Consistent and correctable bias in metagenomic sequencing experiments. eLife 8, e46923 (2019)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR18" id="ref-link-section-d23130190e2545">18</a></sup>, our differential analysis showed a high degree of correlation among within-study differentials, clearly supporting the use of scale-equivariant log fold change calculations within studies as a way to provide a high-confidence readout of ground truth differentials and to enable cross-comparisons of independent cohort studies (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bellmar.medium.com/articles/s41593-023-01361-0#Fig6">2m</a>).</p><p>To determine if our analyses can generalize between 16S and SMS datasets, we focused on the Dan et al.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 28" title="Dan, Z. et al. Altered gut microbial profile is associated with abnormal metabolism activity of autism spectrum disorder. Gut Microbes 11, 1246–1267 (2020)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR28" id="ref-link-section-d23130190e2556">28</a></sup> cohort that paired 16S and SMS samples. Abundances obtained after mapping reads to the Greengenes2 database<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 32" title="Usyk, M. et al. Comprehensive evaluation of shotgun metagenomics, amplicon sequencing, and harmonization of these platforms for epidemiological studies. Cell Rep. Methods 3, 100391 (2023)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR32" id="ref-link-section-d23130190e2560">32</a></sup> highlight strong agreement between 16S and SMS datasets on the genus level (<i>r</i> = 0.63, <i>P</i> &lt; 1 × 10<sup>−100</sup>). Furthermore, the log fold changes between 16S and SMS obtained from our age-matching and sex-matching approach also show strong agreement on the genus level (<i>r</i> = 0.47, <i>P</i> = 1 × 10<sup>−7</sup>).</p><h3 id="Sec4">Differential ranking analysis reveals strong ASD–microbiome links</h3><p>A global age-matched and sex-matched differential ranking analysis of the seven 16S datasets selected for this study revealed a clear partitioning of microbial differences with respect to ASD and cohort membership (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bellmar.medium.com/articles/s41593-023-01361-0#Fig2">2a</a>). The distribution of the overall case–control differences showed a strong ASD-specific signal driven by 591 microbes more commonly found in children with ASD and 169 microbes more commonly found in their control counterparts (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://bellmar.medium.com/articles/s41593-023-01361-0#MOESM3">4)</a>. The variability observed is most likely due to confounding factors such as cohort demographics and geographic location, with the seven cohorts originating from Asia, Europe, South America and North America. Analogous differential ranking trends could be observed for the virome, SMS and RNA sequencing (RNA-seq) datasets (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bellmar.medium.com/articles/s41593-023-01361-0#Fig7">3</a>). To determine whether these highly significant microbiome signals (<i>P</i> &lt; 0.0025) could be used to distinguish children with ASD from their age-matched and sex-matched control counterparts, we trained random forest classifiers on train/validation/test splits of data derived from 16S-targeted sequencing and SMS–whole-genome sequencing of microbial communities. We fitted gradient boosting classifiers on combined microbiome datasets as well as on individual datasets and measured their performance with area under the receiver operating characteristic (AUROC) curve. Of the nine age-matched and sex-matched cohorts<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Chen, Y. et al. Gut bacteria shared by children and their mothers associate with developmental level and social deficits in autism spectrum disorder. mSphere 5, e01044–20 (2020)." href="#ref-CR22" id="ref-link-section-d23130190e2601">22</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Fouquier, J. et al. The gut microbiome in autism: study-site effects and longitudinal analysis of behavior change. mSystems 6, e00848-20 (2021)." href="#ref-CR23" id="ref-link-section-d23130190e2601_1">23</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Zurita, M. F. et al. Analysis of gut microbiome, nutrition and immune status in autism spectrum disorder: a case–control study in Ecuador. Gut Microbes 11, 453–464 (2020)." href="#ref-CR24" id="ref-link-section-d23130190e2601_2">24</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 25" title="Berding, K. &amp; Donovan, S. M. Dietary patterns impact temporal dynamics of fecal microbiota composition in children with autism spectrum disorder. Front. Nutr. 6, 193 (2019)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR25" id="ref-link-section-d23130190e2604">25</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Zou, R. et al. Changes in the gut microbiota of children with autism spectrum disorder. Autism Res. 13, 1614–1625 (2020)." href="#ref-CR27" id="ref-link-section-d23130190e2607">27</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Dan, Z. et al. Altered gut microbial profile is associated with abnormal metabolism activity of autism spectrum disorder. Gut Microbes 11, 1246–1267 (2020)." href="#ref-CR28" id="ref-link-section-d23130190e2607_1">28</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 29" title="Fecal microbiome of children with autism spectrum disorder, their mothers, and healthy children. 
                  https://www.ncbi.nlm.nih.gov/bioproject/PRJNA687773
                  
                " href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR29" id="ref-link-section-d23130190e2610">29</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 33" title="Wang, M. et al. Alterations in gut glutamate metabolism associated with changes in gut microbiota composition in children with autism spectrum disorder. mSystems 4, e00321-18 (2019)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR33" id="ref-link-section-d23130190e2613">33</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 34" title="Averina, O. V. et al. The bacterial neurometabolic signature of the gut microbiota of young children with autism spectrum disorders. J. Med. Microbiol. 69, 558–571 (2020)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR34" id="ref-link-section-d23130190e2616">34</a></sup>, six of the studies had an AUC &gt; 0.87, highlighting the strong microbial differences between children with ASD and neurotypical children within age-matched and sex-matched cohorts (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bellmar.medium.com/articles/s41593-023-01361-0#Fig2">2b</a>). The classification performance decreased when we trained one classifier across all 1,193 samples across all of the cohorts but is still predictive of ASD (AUC = 0.78). This is consistent with previous observations in other disease meta-analyses<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 35" title="Wirbel, J. et al. Meta-analysis of fecal metagenomes reveals global microbial signatures that are specific for colorectal cancer. Nat. Med. 25, 679–689 (2019)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR35" id="ref-link-section-d23130190e2624">35</a></sup>, where within-study classification performance is greater than across-study classification performance. We suspect that widespread microbial heterogeneity across diverse human populations could play a role in impeding classification performance.</p><div data-test="figure" data-container-section="figure" id="figure-2" data-title="Differential ranking analysis across omics levels."><figure><figcaption><b id="Fig2" data-test="figure-caption-text">Fig. 2: Differential ranking analysis across omics levels.</b></figcaption><div><div><a data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="https://bellmar.medium.com/articles/s41593-023-01361-0/figures/2" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-023-01361-0/MediaObjects/41593_2023_1361_Fig2_HTML.png?as=webp"/><img aria-describedby="Fig2" src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-023-01361-0/MediaObjects/41593_2023_1361_Fig2_HTML.png" alt="figure 2" loading="lazy" width="685" height="556"/></picture></a></div><p><b>a</b>, Global microbial 16S log fold changes between age-matched and sex-matched ASD and control individuals. Error bars represent the 95% credible intervals. Heat map showing all center log ratio (CLR) transformed microbial differentials for each age-matched and sex-matched ASD–control pair across all cohorts. Microbes are binned into ASD-associated, Neutral and Control-associated groups using an age-matched and sex-matched classifier (<a data-track="click" data-track-label="link" data-track-action="section anchor" href="https://bellmar.medium.com/articles/s41593-023-01361-0#Sec11">Methods</a>). <i>K</i> is an unknown bias due to the shift in the microbial load between the ASD and neurotypical control population. <b>b</b>, Sample size, male:female (M:F) ratio and average ages across all 16S and shotgun metagenomics datasets analyzed in this study and held-out gradient boosting ASD prediction performance measured by AUROC. V3–V4, V4 and V4–V5 refer to the variable region of the bacterial ribosomal RNA analyzed. <b>c</b>, Log ratios of microbes that are classified to be ASD associated and control associated were computed for each sample. The <i>x</i> axis represents the case–control differences of these log ratios, where values greater than 0 indicate that there is a separation between children with ASD and neurotypical children. The box plots show the median (line), 25–75% range (box) and 5–95% range (whiskers). <b>d</b>, Effect sizes of different omics levels: viral, 16S, SMS and RNA-seq.</p></div></figure></div><p>In contrast to the age-matched and sex-matched cohorts, the AUC dropped substantially in the sibling-matched cohorts (Son et al.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 36" title="Son, J. S. et al. Comparison of fecal microbiota in children with autism spectrum disorders and neurotypical siblings in the Simons Simplex Collection. PLoS ONE 10, e0137725 (2015)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR36" id="ref-link-section-d23130190e2672">36</a></sup> AUC = 0.69; David et al.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 37" title="David, M. M. et al. Children with autism and their typically developing siblings differ in amplicon sequence variants and predicted functions of stool-associated microbes. mSystems 6, e00193-20 (2021)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR37" id="ref-link-section-d23130190e2676">37</a></sup> AUC = 0.46; Elliot et al.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 38" title="Martin-Brevet, S. et al. Quantifying the effects of 16p11.2 copy number variants on brain structure: a multisite genetic-first study. Biol. Psychiatry 84, 253–264 (2018)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR38" id="ref-link-section-d23130190e2680">38</a></sup> AUC = 0.38). Similarly PERMANOVA detected ASD-specific microbiome differences only in the age-matched and sex-matched cohort (<i>P</i> = 0.002), whereas no such signal was found in the sibling-matched cohort (<i>P</i> = 0.535). In both cohorts, age and sex were significant confounders (<i>P</i> &lt; 0.002), but only in the age-matched and sex-matched cohort could the age and sex differences between case–control pairs be minimized (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bellmar.medium.com/articles/s41593-023-01361-0#Fig10">6a–d</a>), where more than two times more case–control pairs are the within 1-year age difference and the same gender compared to the sibling-matched cohort (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bellmar.medium.com/articles/s41593-023-01361-0#Fig10">6e,f</a>). Although household has been observed to be a confounder in the sibling-matched cohort (<i>P</i> &lt; 0.001), we did see strong classifier generalization in the age-matched and sex-matched cohorts, where none of the children live in the same home. However, it is possible that unmeasured confounders, such as household diet or socioeconomic status, could artificially boost classification performance. To investigate the potential age and sex confounders in the sibling studies, we performed a data-driven simulation with a known ground truth to determine how large age differences (± &gt;2 years) would bias modeling outcomes compared to optimized age matchings (± ≤0.5 years) using a sibling-like age distribution (obtained from David et al.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 37" title="David, M. M. et al. Children with autism and their typically developing siblings differ in amplicon sequence variants and predicted functions of stool-associated microbes. mSystems 6, e00193-20 (2021)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR37" id="ref-link-section-d23130190e2703">37</a></sup>) and our overall sex-matched and age-matched distribution, respectively (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bellmar.medium.com/articles/s41593-023-01361-0#Fig10">6c</a>). The analysis showed that, in case–controls with a sufficiently large age confounder, methods using age and sex matching or sibling matching cannot exactly recover the ground truth log fold changes. However, for sibling-like age distribution, the estimated log fold changes exhibited a large bias (mean squared error = 589.3) that was reduced by an order of magnitude in the sex-matched and age-matched group (mean squared error = 57.8) (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bellmar.medium.com/articles/s41593-023-01361-0#Fig10">6g,h</a>).</p><p>Age-matched and sex-matched differential analysis outperformed standard group averaging with respect to R<sup>2</sup>, and its overall performance strictly improved as more studies were added (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bellmar.medium.com/articles/s41593-023-01361-0#Fig6">2g,h</a>). This performance boost reflected a reduction in model uncertainty with larger cohorts that was indicative of overlapping differentially abundant taxa across studies and of reduced confounding variation. To aid in the interpretation of the classification results, we constructed log ratios of taxa derived from the age-matched and sex-matched differential abundance analysis that strongly separated children with ASD from the neurotypical controls within each study. From these individual analyses, we assembled a single microbial log ratio that highlighted a strong consistent enrichment of taxa in children with ASD relative to their control counterparts with log ratios greater 0 across 88% of pairs (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bellmar.medium.com/articles/s41593-023-01361-0#Fig2">2c</a>). This pattern was consistent across all age-matched and sex-matched cohorts, including two held-out shotgun metagenomics datasets: Wang et al.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 33" title="Wang, M. et al. Alterations in gut glutamate metabolism associated with changes in gut microbiota composition in children with autism spectrum disorder. mSystems 4, e00321-18 (2019)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR33" id="ref-link-section-d23130190e2725">33</a></sup> (log ratios &gt; 0 in 70% of pairs) and Dan et al.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 28" title="Dan, Z. et al. Altered gut microbial profile is associated with abnormal metabolism activity of autism spectrum disorder. Gut Microbes 11, 1246–1267 (2020)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR28" id="ref-link-section-d23130190e2729">28</a></sup> (log-ratios &gt; 0 in 73% of pairs).</p><h3 id="Sec5">ASD-specific patterns are present at several omic levels</h3><p>Differential ranking analysis of three omic levels—microbiome (16S and SMS) and human transcriptome (RNA-seq)—revealed strong and highly significant differences between children with ASD and their age-matched and sex-matched neurotypical counterparts (<i>P</i> &lt; 0.0025) (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bellmar.medium.com/articles/s41593-023-01361-0#Fig2">2d</a> and Supplementary Tables <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://bellmar.medium.com/articles/s41593-023-01361-0#MOESM3">5</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://bellmar.medium.com/articles/s41593-023-01361-0#MOESM3">6)</a>. Two additional omic levels—the metabolome and the virome—did not show significant signals (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bellmar.medium.com/articles/s41593-023-01361-0#Fig8">4</a> and Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://bellmar.medium.com/articles/s41593-023-01361-0#MOESM3">7)</a>.</p><h3 id="Sec6">Host cytokines correlate with microbial abundances</h3><p>Immune dysregulation, ranging from circulating ‘anti-brain’ antibodies and perturbed cytokine profiles to simply having a family history of immune disorders, has been repeatedly associated with ASD<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 39" title="Croen, L. A. et al. Family history of immune conditions and autism spectrum and developmental disorders: findings from the study to explore early development. Autism Res. 12, 123–135 (2019)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR39" id="ref-link-section-d23130190e2769">39</a></sup>. Recently, for example, Zurita et al.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 24" title="Zurita, M. F. et al. Analysis of gut microbiome, nutrition and immune status in autism spectrum disorder: a case–control study in Ecuador. Gut Microbes 11, 453–464 (2020)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR24" id="ref-link-section-d23130190e2773">24</a></sup> showed that concentrations of the inflammatory cytokine transforming growth factor beta (TGF-β) are significantly elevated in children with ASD. We re-analyzed this dataset, after age matching and sex matching, and observed that 16S microbial differentials estimated from Zurita et al.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 24" title="Zurita, M. F. et al. Analysis of gut microbiome, nutrition and immune status in autism spectrum disorder: a case–control study in Ecuador. Gut Microbes 11, 453–464 (2020)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR24" id="ref-link-section-d23130190e2777">24</a></sup> were associated with TGF-β and were positively correlated with the global microbial log fold changes between ASD and control pairs (TGF-β: <i>r</i> = 0.237, <i>P</i> = 2.84 × 10<sup>−5</sup>) (Supplementary Tables <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://bellmar.medium.com/articles/s41593-023-01361-0#MOESM1">1</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://bellmar.medium.com/articles/s41593-023-01361-0#MOESM3">9</a>). In contrast, the global microbial log fold changes had little correlation with interleukin (IL)-6 concentrations (<i>r</i> = 0.07, <i>P</i> = 0.17). However, when we calculated the log ratios of the most differentiating microbial taxa, they were highly correlated with both TGF-β and IL-6 concentrations (TGF-β: <i>r</i> = 0.61, <i>P</i> = 1.84 × 10<sup>−5</sup>; IL-6: <i>r</i> = 0.73, <i>P</i> = 5.74 × 10<sup>−8</sup>) (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bellmar.medium.com/articles/s41593-023-01361-0#Fig3">3a–d</a>). This highlights how IL-6 changes are linked to only a handful of taxa, whereas TGF-β is linked to a much larger number of taxa.</p><div data-test="figure" data-container-section="figure" id="figure-3" data-title="Characterizing the associations among differentially abundant microbes in ASD and cytokines, gene expression in the brain and dietary patterns."><figure><figcaption><b id="Fig3" data-test="figure-caption-text">Fig. 3: Characterizing the associations among differentially abundant microbes in ASD and cytokines, gene expression in the brain and dietary patterns.</b></figcaption><div><div><a data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="https://bellmar.medium.com/articles/s41593-023-01361-0/figures/3" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-023-01361-0/MediaObjects/41593_2023_1361_Fig3_HTML.png?as=webp"/><img aria-describedby="Fig3" src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-023-01361-0/MediaObjects/41593_2023_1361_Fig3_HTML.png" alt="figure 3" loading="lazy" width="685" height="309"/></picture></a></div><p><b>a</b>,<b>b</b>, Comparison of microbial differentials obtained from age matching and sex matching and cytokine analysis. <b>c</b>,<b>d</b>, Microbial log ratios constructed from the 50 top and bottom most differentially abundant microbes corresponding to each cytokine. <i>K</i> and <i>C</i> represent unknown biases due to the shift in the microbial load between the ASD and neurotypical control population. <b>e</b>, Heat map showing the overlap of molecules between ASD-enriched pathways in the microbiome and in the brain. The microbial and human pathways are both sorted alpha-numerically; the dense diagonal is largely indicative of common pathways between microbial and human genomes. <b>f</b>,<b>g</b>, PC3 from microbe–diet co-occurrence analysis is contrasted against microbial log fold changes and dietary differences from Berding et al.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 25" title="Berding, K. &amp; Donovan, S. M. Dietary patterns impact temporal dynamics of fecal microbiota composition in children with autism spectrum disorder. Front. Nutr. 6, 193 (2019)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR25" id="ref-link-section-d23130190e2862">25</a></sup> Dietary compounds that are depleted (<i>P</i> &lt; 0.1) in children with ASD are highlighted as ‘x’ markers. <i>T</i>(ASD-Control) represents the <i>t</i>-statistic that measures the differences between ASD and neurotypical dietary intake. conc., concentration.</p></div></figure></div><p><i>Prevotella</i>, <i>Bacteroides</i> and <i>Bifidobacterium</i> were predominantly associated with the cytokine differentials. Partial mechanistic insights on some of these cytokine–microbe associations were previously published. <i>Bacteroides thetaiotaomicron</i> was the second most highly elevated microbe when TGF-β was depleted and has been suggested to play a role modulating maternal immune activation-dependent metabolites that are linked to behavorial symptoms<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 40" title="Hsiao, E. Y. et al. The microbiota modulates gut physiology and behavioral abnormalities associated with autism. Cell 155, 1451 (2013)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR40" id="ref-link-section-d23130190e2898">40</a></sup>. <i>Bifidobacterium callitrichidarum</i> was the sixth most enriched taxon when IL-6 was in lower concentration. Other <i>Bifidobacteria</i> species, such as <i>Bifidobacterium longum</i>, have been observed to downregulate IL-6 in fetal human enterocytes in vitro<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 41" title="Meng, D., Zhu, W., Ganguli, K., Shi, H. N. &amp; Walker, W. A. Anti-inflammatory effects of bifidobacterium longum subsp infantis secretions on fetal human enterocytes are mediated by TLR-4 receptors. Am. J. Physiol. Gastrointest. Liver Physiol. 311, G744–G753 (2016)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR41" id="ref-link-section-d23130190e2912">41</a></sup>. <i>Prevotella copri</i> was the second most enriched taxon when IL-6 was in lower concentration and the sixth most enriched taxon when TGF-β was in lower concentration. This was consistent with Tett et al.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 42" title="Tett, A. et al. The Prevotella copri complex comprises four distinct clades underrepresented in Westernized populations. Cell Host Microbe 26, 666–679 (2019)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR42" id="ref-link-section-d23130190e2919">42</a></sup>, where <i>P. copri</i> associations with different cytokines were observed in multiple disease contexts. Similarly, <i>P. copri</i> and <i>Bacteroides fragilis</i> both co-occurred with phages enriched in children with ASD or in neurotypical children (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bellmar.medium.com/articles/s41593-023-01361-0#Fig11">7</a> and Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://bellmar.medium.com/articles/s41593-023-01361-0#MOESM3">10)</a>, but, whereas microbes were previously reported to mediate viral infections<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 43" title="Neu, U. &amp; Mainou, B. A. Virus interactions with bacteria: partners in the infectious dance. PLoS Pathog. 16, e1008234 (2020)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR43" id="ref-link-section-d23130190e2939">43</a></sup>, the mechanistic underpinnings of these interactions with the host’s immunity remain poorly understood<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 44" title="Camarillo-Guerrero, L. F., Almeida, A., Rangel-Pineros, G., Finn, R. D. &amp; Lawley, T. D. Massive expansion of human gut bacteriophage diversity. Cell 184, 1098–1109 (2021)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR44" id="ref-link-section-d23130190e2943">44</a></sup>.</p><h3 id="Sec7">Microbiome metabolism mirrors human brain metabolism in ASD</h3><p>To determine potential crosstalk between microbiome physiology and the human brain, we compared the metabolic capacities encoded by the microbial metagenome—combining the individual metabolic capacities of thousands of different microbes—and the differentially expressed human genome in the brain, two omic levels representing entirely different biological contexts. We identified 138 microbial and 1,772 human metabolic encoding genes, inferred from SMS and RNA-seq, respectively, that were linked to ASD phenotype. Ninety-five human metabolic pathways differentially expressed in the brain tissues of individuals with ASD had analogous microbial pathways differentially abundant in the microbiome of children with ASD, suggesting a potential coordination of metabolic pathways across omic levels in ASD (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bellmar.medium.com/articles/s41593-023-01361-0#Fig3">3e</a>). Pathways related to amino acid metabolism, carbohydrate metabolism and lipid metabolism were disproportionately represented among the overlapping pathways (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bellmar.medium.com/articles/s41593-023-01361-0#Fig13">9</a> and Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://bellmar.medium.com/articles/s41593-023-01361-0#MOESM3">14)</a>. Cross-comparison of the ASD-associated microbial enzyme-encoding genes with the gut–brain modules (GBMs), previously defined as part of the GBM framework, also revealed an approximately 48.5% overlap (101/208), further supporting the notion of potential metabolic crosstalk across omic levels<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 45" title="Valles-Colomer, M. et al. The neuroactive potential of the human gut microbiota in quality of life and depression. Nat. Microbiol. 4, 623–632 (2019)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR45" id="ref-link-section-d23130190e2964">45</a></sup> (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://bellmar.medium.com/articles/s41593-023-01361-0#MOESM3">13)</a>.</p><h3 id="Sec8">Microbiome metabolic capacity mirrors diet patterns in ASD</h3><p>Autistic traits in early childhood have been shown to correlate with poor diet quality later in life; however, little is known about how diet quality is directly linked to autistic traits<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 46" title="Harris, H. A., Mou, Y., Dieleman, G. C., Voortman, T. &amp; Jansen, P. W. Child autistic traits, food selectivity and diet quality: a population-based study. J. Nutr. 152, 856–862 (2021)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR46" id="ref-link-section-d23130190e2979">46</a></sup>. Here, we re-analyzed the paired microbiome and dietary survey data from Berding et al.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 25" title="Berding, K. &amp; Donovan, S. M. Dietary patterns impact temporal dynamics of fecal microbiota composition in children with autism spectrum disorder. Front. Nutr. 6, 193 (2019)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR25" id="ref-link-section-d23130190e2983">25</a></sup>. A microbiome–diet co-occurrence analysis revealed startlingly similar amino acid, carbohydrate and lipid metabolism association patterns to those observed in the microbiome–brain metabolic capacity analysis (Supplementary Tables <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://bellmar.medium.com/articles/s41593-023-01361-0#MOESM1">2</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://bellmar.medium.com/articles/s41593-023-01361-0#MOESM3">15)</a> (<i>Q</i><sup>2</sup> = 0.43). From the microbe–diet co-occurrence analysis, only principal component (PC) 3, which explains 3% of the microbe–diet variance, could differentiate between ASD and neurotypical diets (<i>r</i> = 0.26, <i>P</i> = 0.004) and was strongly correlated with the microbial log fold changes between ASD and age-matched and sex-matched controls (<i>r</i> = 0.22, <i>P</i> = 4.3 × 10<sup>−9</sup>). Autistic children were less likely to consume foods high in glutamic acid, serine, choline, phenylalanine, leucine, tyrosine, valine and histidine, all compounds involved in neurotransmitter biosynthesis<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 47" title="Dalangin, R., Kim, A. &amp; Campbell, R. E. The role of amino acids in neurotransmission and fluorescent tools for their detection. Int. J. Mol. Sci. 21, 6197 (2020)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR47" id="ref-link-section-d23130190e3012">47</a></sup>. Interestingly, multiple <i>Bacteroides</i> taxa and <i>P. copri</i> taxa were among the top 20 taxa along MMvec PC3, highlighting how these taxa could be involved in metabolizing amino acid dietary compounds (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bellmar.medium.com/articles/s41593-023-01361-0#Fig3">3f</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bellmar.medium.com/articles/s41593-023-01361-0#Fig12">8</a>). Even though the metabolomic analysis did not yield statistically significant signals after FDR correction, the metabolites that showed the strongest signal included glutamate and phenylalanine, consistent with the microbiome–diet analysis<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Kuwabara, H. et al. Altered metabolites in the plasma of autism spectrum disorder: a capillary electrophoresis time-of-flight mass spectroscopy study. PLoS ONE 8, e73814 (2013)." href="#ref-CR48" id="ref-link-section-d23130190e3029">48</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Needham, B. D. et al. Plasma and fecal metabolite profiles in autism spectrum disorder. Biol. Psychiatry 89, 451–462 (2021)." href="#ref-CR49" id="ref-link-section-d23130190e3029_1">49</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 50" title="Noto, A. et al. The urinary metabolomics profile of an Italian autistic children population and their unaffected siblings. J. Matern. Fetal Neonatal Med. 27 Suppl 2, 46–52 (2014)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR50" id="ref-link-section-d23130190e3032">50</a></sup>. Disruptions in the biosynthesis of these neurotransmitter molecules have been implicated in a wide variety of psychiatric disorders, and a recent blood metabolomics study showed the potential of using branched-chain amino acids to define autism subtypes<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 51" title="Smith, A. M. et al. Amino acid dysregulation metabotypes: potential biomarkers for diagnosis and individualized treatment for subtypes of autism spectrum disorder. Biol. Psychiatry 85, 345–354 (2019)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR51" id="ref-link-section-d23130190e3036">51</a></sup>. Due to the incompatibility among the molecular features across datasets, it was not possible to combine any of the metabolomics datasets to boost the statistical power, which remains a major limitation of metabolomics technologies at present (<a data-track="click" data-track-label="link" data-track-action="section anchor" href="https://bellmar.medium.com/articles/s41593-023-01361-0#Sec11">Methods</a>).</p><h3 id="Sec9">ASD microbiomes mirror behavior improvement after fecal matter transplant</h3><p>Although the preceding cross-sectional analyses showed significant associations among several omic levels (virome, microbiome and immunome) or diet and ASD, insights into causality are still limited. By contrast, longitudinal intervention studies provide an opportunity to obtain stronger insights into causality. To test this, we re-analyzed data from a 2-year, open-label fecal matter transplant (FMT) study with 18 children with ASD<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 52" title="Kang, D.-W. et al. Long-term benefit of microbiota transfer therapy on autism symptoms and gut microbiota. Sci. Rep. 9, 5821 (2019)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR52" id="ref-link-section-d23130190e3052">52</a></sup>. In this study, the children were subjected to a 2-week antibiotic treatment and a bowel cleanse, followed by 2 d of high-dose FMT treatment and 8 weeks of daily maintenance FMT doses. Based on one of the most common evaluation scales for ASD, the Childhood Autism Rating Scale (CARS), significant improvements were achieved after the 10-week course of treatment. Two months later, the initial gains were largely maintained, and a 2-year follow-up showed signs of further improvement in most of the patients. The results are consistent with a potential role of the microbiome in improving autism symptoms, but how the underlying changes in microbiome composition related to those seen in other studies remains unknown.</p><p>In the present study, we re-analyzed the original raw data in the context of the ASD profiles revealed by our cross-sectional differential ranking analysis (Supplementary Tables <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://bellmar.medium.com/articles/s41593-023-01361-0#MOESM1">3</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://bellmar.medium.com/articles/s41593-023-01361-0#MOESM3">16)</a>. All microbes associated with ASD in the 18 children before the FMT treatment had been identified as ASD-associated microbes in our age-matched and sex-matched cross-sectional analysis. After 2 years, 91% of these microbes that had low uncertainty (posterior standard deviation &lt; 3) exhibited a mean decrease in abundance, and this decrease was significant (95% log fold change quantile &lt; 0) in 57% of the microbes (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bellmar.medium.com/articles/s41593-023-01361-0#Fig4">4</a>). Consistent with the original analysis by Kang et al., we detected an increase in <i>Prevotella sp</i>. over the 2-year span of the study. In addition, we also determined an increase in <i>Desulfovibrio piger</i> and no significant changes in <i>Bifidobacteria</i>, counter to the original analysis by Kang et al.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 52" title="Kang, D.-W. et al. Long-term benefit of microbiota transfer therapy on autism symptoms and gut microbiota. Sci. Rep. 9, 5821 (2019)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR52" id="ref-link-section-d23130190e3078">52</a></sup> Interestingly, 305 taxa remained stable throughout the duration of the study. Of these, 13 taxa belonged to the <i>Prevotella</i>, <i>Bifidobacterium</i>, <i>Bacteroides</i> and <i>Desulfovibrio</i> lineages, pointing to a potentially wide functional diversity within these genera not noted in the original study. Some of these taxa, including <i>B. fragilis</i>, <i>B. thetaiotaomicron</i>, <i>B. longum</i> and <i>P. copri</i><sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 42" title="Tett, A. et al. The Prevotella copri complex comprises four distinct clades underrepresented in Westernized populations. Cell Host Microbe 26, 666–679 (2019)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR42" id="ref-link-section-d23130190e3106">42</a></sup>, were previously associated with beneficial immunomodulatory properties. Also worth pointing out are multiple butyrate producers in the <i>Butyricimonas</i> and <i>Anaerobutyricum</i> genera that we detected as being stable throughout the 2 years of the study, indicating a potential role in contributing to GBA homeostasis<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 53" title="Silva, Y. P., Bernardi, A. &amp; Frozza, R. L. The role of short-chain fatty acids from gut microbiota in gut–brain communication. Front. Endocrinol. 11, 25 (2020)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR53" id="ref-link-section-d23130190e3117">53</a></sup>.</p><div data-test="figure" data-container-section="figure" id="figure-4" data-title="FMTs have long-lasting effects on autism gut microbiomes."><figure><figcaption><b id="Fig4" data-test="figure-caption-text">Fig. 4: FMTs have long-lasting effects on autism gut microbiomes.</b></figcaption><div><div><a data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="https://bellmar.medium.com/articles/s41593-023-01361-0/figures/4" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-023-01361-0/MediaObjects/41593_2023_1361_Fig4_HTML.png?as=webp"/><img aria-describedby="Fig4" src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-023-01361-0/MediaObjects/41593_2023_1361_Fig4_HTML.png" alt="figure 4" loading="lazy" width="685" height="764"/></picture></a></div><p><b>a</b>, The improvement of CARS for each child with ASD over time. The children were split into three groups—non-ASD, mild/moderate and severe—based on whether their CARS score fell below 30, was between 30 and 37 or was higher than 37. <b>b</b>, Microbial log fold changes over time: the time series was generated by calculating log fold changes between timepoints for each microbe. ASD-specific microbes highlighted in red were determined in the cross-sectional study. <b>c</b>–<b>f</b>, Microbial log fold changes are re-colored with genera highlighted in cytokine comparisons.</p></div></figure></div></div></div></section><section data-title="Discussion"><div id="Sec10-section"><h2 id="Sec10">Discussion</h2><div id="Sec10-content"><p>The functional architecture of ASD, and in particular the potential role that the microbiome plays in modulating the GBA in the context of autism, remain poorly understood due to disagreements among existing microbiome and other omic studies. However, in contrast to recently reported findings<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 54" title="Yap, C. X. et al. Autism-related dietary preferences mediate autism–gut microbiome associations. Cell 184, 5916–5931 (2021)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR54" id="ref-link-section-d23130190e3161">54</a></sup>, we observed a clear separation between children with ASD and unrelated age-matched and sex-matched neurotypical controls, and this signal was validated using three distinct methodologies—namely PERMANOVA, classification and differential abundance—across multiple cohorts. Unlike the age-matched and sex-matched analysis, no ASD–microbiome signal was detected in the sibling-matched cohorts, including in the Elliott et al.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 38" title="Martin-Brevet, S. et al. Quantifying the effects of 16p11.2 copy number variants on brain structure: a multisite genetic-first study. Biol. Psychiatry 84, 253–264 (2018)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR38" id="ref-link-section-d23130190e3165">38</a></sup> dataset that consists of individuals with chromosome 16p11.2 deletion, a known risk factor for ASD. One possibility is due to age and sex confounding in the sibling cohorts, because age remains a major confounding factor in early childhood microbiome development<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 55" title="Rodríguez, J. M. et al. The composition of the gut microbiota throughout life, with an emphasis on early life. Microb. Ecol. Health Dis. 26, 26050 (2015)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR55" id="ref-link-section-d23130190e3169">55</a></sup>. However, we cannot rule out the possibility that our classifiers are identifying differences between households rather than individuals with ASD in the age-matched and sex-matched cohorts. Previous efforts identified household-specific effects on the human microbiome<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 56" title="Song, S. J. et al. Cohabiting family members share microbiota with one another and with their dogs. eLife 2, e00458 (2013)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR56" id="ref-link-section-d23130190e3173">56</a></sup>, and other studies raised issues with sibling controls in these studies because siblings often exhibit a higher risk of developing ASD compared to the general population<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 57" title="Pisula, E. &amp; Ziegart-Sadowska, K. Broader autism phenotype in siblings of children with ASD—a review. Int. J. Mol. Sci. 16, 13217–13258 (2015)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR57" id="ref-link-section-d23130190e3177">57</a></sup>. However, the fact that we see a clear ASD–microbiome signal that generalizes across households within cohorts highlights the need to control these confounding factors to understand the functional role that these gut microbiota could play. Thus, a follow-up study investigating gut microbiome and genetic variation between households with and without children with ASD is needed.</p><p>Parallel analyses at the immunome, human transcriptome and dietome levels revealed strong associations among omic levels. The virome and the direct metabolome signals, although present, were markedly weaker than the other omic signals. The inferred ASD-specific metabolic profiles from the microbiome and the human transcriptome, on the other hand, showed a high and significant degree of overlap in microbial and human pathways expressed in the gut and in the brain, respectively. The metabolic connection implied by this overlap, which included differentially enriched carbohydrate and amino acid metabolic pathways in ASD, is a remarkable observation given the fundamental difference between the gut and brain physiologies, which would a priori suggest a reduced overlap in metabolic capacities. The microbiome–diet co-occurrence analysis also highlighted a reduced intake of amino acids and carbohydrates linked to specific microbiome profiles in children with ASD. These metabolic and dietary imbalances, particularly regarding glutamate levels, were further apparent, albeit weakly, in the serum, fecal and urine metabolomes that we analyzed. This multi-scale overlap that we observed along the GBA points to the existence of a functional architecture of ASD driven by the metabolic potential at the genomic and metagenomic levels.</p><p>In light of the heterogeneity across studies, our analysis identified several microorganisms consistently detected across omic levels that point to potentially interesting functional connections. The diet co-occurrence analysis also showed a strong association between <i>P. copri</i> and carbohydrate depletion in ASD, in addition to upregulation of IL-6. <i>Bacteroides</i> genera are observed to play a key role in ASD diet differentiation, with <i>B. thetaiotaomicron</i> associated with the depletion of TGF-β. Multiple other microbes, including <i>P. copri</i> and several <i>Bacteroides</i>, stood out in the immune and viral analyses. In the FMT study, we observed a stable core microbiome made up of <i>Bacteroides</i>, <i>Prevotella</i>, <i>Bifidobacteria</i> and <i>Desulfovibrio</i> in addition to multiple butyrate producers. The presence of this core microbiome in combination with the depletion of most ASD-associated taxa further suggests a causal role for these microorganisms in shaping autism symptoms.</p><p>Despite our inability to determine actual metabolomic profiles at this point (<a data-track="click" data-track-label="link" data-track-action="section anchor" href="https://bellmar.medium.com/articles/s41593-023-01361-0#Sec11">Methods</a>), our metabolite analysis based on microbiome-derived and brain-derived metabolite inferences as well as the diet-derived metabolite data reveals a picture of a unifying and distinct ASD functional architecture. With the brain, the immunome and diet as major effectors, the multi-factorial complexity of ASD is reduced to a multi-scale set of interactions centered around human and bacterial metabolism that, in turn, determines phenotypic, genomic and metagenomic attributes via multiple feedback loops. Although we did not observe an effect on genotype to the microbiome, previous studies identified genes that are high risk for ASD<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 2" title="Satterstrom, F. K. et al. Large-scale exome sequencing study implicates both developmental and functional changes in the neurobiology of autism. Cell 180, 568–584 (2020)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR2" id="ref-link-section-d23130190e3221">2</a></sup>. The pivotal role of the immune system in mediating the communication between the gut microbiome and the human brain as well as other peripheral systems is also firmly established. Furthermore, the central role of the microbiome in mediating diet-derived nutrient mobilization has been extensively documented, and several hardwired feedback loops among these effectors, such as the hypothalamus-mediated regulation of appetite and diet, have also been described<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 6" title="Morais, L. H., Schreiber IV, H. L. &amp; Mazmanian, S. K. The gut microbiota–brain axis in behaviour and brain disorders. Nat. Rev. Microbiol. 19, 241–255 (2021)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR6" id="ref-link-section-d23130190e3225">6</a></sup>.</p><p>Our understanding of how the gut microbiome is connected to dietary preferences, host immunity and GI and ASD behavioral symptoms is limited in cross-sectional studies and, thus, restricts our ability to perform causal inference. We envision that obtaining causal insights into the functional architecture of autism will require a multi-arm approach, from culturing key microbes and probing their metabolic capacity, to performing experimental interventions with model organisms and conducting longitudinal observational studies, with multi-omics data collection and extensive phenotypic profiling to observe the effects of natural interventions. Building realistic causal models of autism needs to take into account the multi-factorial complexity underlying different ASD subtypes, which will require a concerted effort to simultaneously analyze several omic levels and at clinically relevant timescales. For instance, understanding the engraftment dynamics of FMT and its functional implications on the recipients’ gut microbiomes requires frequent initial sampling of the microbiome, immunome and metabolome, but tracing any behavioral changes over time requires less frequent sampling over periods of up to several years, in combination with reliable behavioral, medical and dietary surveys<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 58" title="Wilson, B. C. et al. Strain engraftment competition and functional augmentation in a multi-donor fecal microbiota transplantation trial for obesity. Microbiome 9, 107 (2021)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR58" id="ref-link-section-d23130190e3233">58</a></sup>. Collecting and integrating such multi-scale omic datasets presents unique logistical and analytical challenges.</p><p>Managing data acquisition and access will require coordinating multiple sites and potentially centralizing some aspects of sample processing. Recent initiatives, such as The Environmental Determinants of Diabetes in the Young (TEDDY) study, an international long-term, multi-center initiative to link specific environmental triggers to particular type 1 diabetes-associated genotypes, provide a blueprint for similar approaches in ASD. A key component of such an initiative would be the establishment of standardized sampling and processing protocols that would minimize technical confounders, one of the top confounders at most omic levels. Moreover, although extensive efforts are underway to calibrate microbiome datasets, other omic levels, such as the metabolome, present even more fundamental technical issues that make it imperative to develop concerted strategies to be able to include them in an integrated analysis.</p><p>In addition to the considerable variations in statistical properties across datasets, interactions among omic levels are mostly underdetermined, making the construction of informative models a major challenge. Determining the necessary biologically relevant assumptions is a non-trivial process and can inadvertently lead to model mis-specifications, resulting in misleading conclusions. This was the likely consequence from Yap et al.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 54" title="Yap, C. X. et al. Autism-related dietary preferences mediate autism–gut microbiome associations. Cell 184, 5916–5931 (2021)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR54" id="ref-link-section-d23130190e3243">54</a></sup>, where the proposed model that tested for a causal relationship among diet, microbes and the ASD phenotype implicitly assumed that there was no relationship between diet and gut microbiome, prematurely rejecting the potential role between gut microbiota and ASD. Addressing these types of model mis-specication issues will be critical to inferring causal mechanisms from population-scale studies. In addition, and given the vast heterogeneity of ASD, designing cohort studies that minimize confounding factor effects will be key to furthering understanding of autism. For example, although our analysis could not identify ASD subtypes, we determined stronger associations among gut microbes, host immunity, brain expression and dietary patterns than previously reported, highlighting the potential for boosting the statistical power and biological insight with comprehensive omic analyses.</p><p>We conclude that multi-omic longitudinal intervention studies on appropriately stratified cohorts, combined with comprehensive patient metadata, would provide the necessary entry points for advancing mechanistic studies along the GBA in ASD. The experimental framework that we propose for inferring causal mechanisms from population-scale studies will require the development of consensuated multi-disciplinary strategies. For instance, given the central role played by the metabolome in relaying information across omic levels, a unified approach to metabolomics studies will be needed to overcome current differences in data types (targeted versus untargeted and liquid chromatography–mass spectrometry (LC–MS) versus gas chromatography–mass spectrometry (GC–MS)) or origin of the specimens (blood/serum, urine or feces). Phenotyping behavorial and GI symptoms in children with ASD is another issue that is still far from being resolved, making it further challenging to stratify patient cohorts. Issues of timescales—from the molecular to the behavioral—need to be harmonized in statistically relevant ways to allow for proper causality inference. Finally, using appropriate statistical methodologies for identifying potential causal relationships will be critical to ensure the success of the proposed mechanistic studies and of efforts to advance understanding of the role that the microbiome plays in the context of the overall functional architecture of ASD.</p></div></div></section><section data-title="Methods"><div id="Sec11-section"><h2 id="Sec11">Methods</h2><div id="Sec11-content"><h3 id="Sec12">Search strategy and inclusion criteria</h3><p>We performed a systematic search for published and/or publicly deposited or not yet published and/or publicly available human microbiome, metabolome, immunome, transcriptome and autism/ASD datasets in several National Center for Biotechnology Information (NCBI) databases (PubMed, Sequence Read Archive (SRA) and BioProject), UCSD’s MassIVE resource, the PsychENCODE consortium and the American Gut Project and from individual research groups worldwide. About half of the 70+ studies that we identified were already deposited on public data repositories or were made directly available to us by the research groups.</p><p>Most studies consisted of heterogenous—no genotype or phenotype stratification—ASD and neurotypical age-matched and sex-matched cohorts and had one or two datasets (microbiome (16S, SMS), metabolome (urine/serum/fecal), immunome (cytokines), transcriptome (RNA-seq), dietary survey and behavioral survey) associated with them, with only a few studies having three or more omic datasets associated with them (Table <a data-track="click" data-track-label="link" data-track-action="table anchor" href="https://bellmar.medium.com/articles/s41593-023-01361-0#Tab1">1</a>). We adopted a multi-cohort and multi-omics meta-analysis framework that allowed us to combine independent and dependent omic datasets in one overall analysis<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 19" title="Shafi, A., Nguyen, T., Peyvandipour, A., Nguyen, H. &amp; Draghici, S. A multi-cohort and multi-omics meta-analysis framework to identify network-based gene signatures. Front. Genet. 10, 159 (2019)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR19" id="ref-link-section-d23130190e3268">19</a></sup>. In total, we analyzed 528 ASD–control pairs that had either age and sex information or sibling-matching information. To reduce the batch effects and noise associated with primer choice in the 16S datasets, a major confounder in microbiome analyses, we restricted the 16S datasets to include only those targeting the variable region V4 of the bacterial ribosomal RNA, a region exhibiting higher heterogeneity and lower evolution rates than other variable regions<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 64" title="Walters, W. et al. Improved bacterial 16S rRNA gene (v4 and v4–5) and fungal internal transcribed spacer marker gene primers for microbial community surveys. mSystems 1, e00009-15 (2016)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR64" id="ref-link-section-d23130190e3272">64</a></sup>. Previous studies showed how primers targeting adjacent regions in the 16S can yield similar composition estimates up to the genus level<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 65" title="Fadeev, E. et al. Comparison of two 16s rRNA primers (v3–v4 and v4–v5) for studies of Arctic microbial communities. Front. Microbiol. 12, 283 (2021)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR65" id="ref-link-section-d23130190e3276">65</a></sup>. Our analysis included 16S datasets obtained targeting the V4 region exclusively, the V3–V4 region or the V4–V5 region.</p><p>The final metabolomic meta-analysis that we present here consists of the combined analysis of only four independently pre-processed, normalized and analyzed metabolomic datasets. Despite several more ASD-related datasets being available, the disparity in mass spectrometric technologies used to generate them, which results in the detection of different subsets of metabolites, precluded their side-by-side comparison (Table <a data-track="click" data-track-label="link" data-track-action="table anchor" href="https://bellmar.medium.com/articles/s41593-023-01361-0#Tab1">1</a>). For example, targeted mass spectrometry enables the precise determination of concentrations for a finite number of metabolites, whereas untargeted mass spectrometry detects up to two or three orders of magnitude more metabolites but is compositional in nature and, thus, does not yield absolute abundances. Furthermore, batch effects due to sample processing, such as differences in reagents, sample storage and mass spectrometry instruments, can introduce unwanted variation in both the abundances and the detected molecular features<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 66" title="Han, W. &amp; Li, L. Evaluating and minimizing batch effects in metabolomics. Mass Spectrom. Rev. 41, 421–442 (2020)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR66" id="ref-link-section-d23130190e3286">66</a></sup>. One additional obstacle that we encountered was the proprietary nature of many of the metabolomic datasets that made it impossible to access the raw data and run standardized workflows.</p><p>Of the 40 transcriptomic datasets that were available in recount3 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 67" title="Wilks, C. et al. recount3: summaries and queries for large-scale RNA-seq expression and splicing. Genome Biol. 22, 323 (2021)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR67" id="ref-link-section-d23130190e3293">67</a></sup>), the vast majority were obtained from studies with model animals, and only four of them had been obtained from postmortem processing of brain samples from autistic and neurotypical individuals. These four datasets collected different brain tissue types, including from the amygdala, the prefrontal cortex, the anterior cingulate and the dorsolateral prefrontal cortex.</p><h4 id="Sec13">Martin-Brevet et al. cohort</h4><p>Data from Martin-Brevet et al.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 38" title="Martin-Brevet, S. et al. Quantifying the effects of 16p11.2 copy number variants on brain structure: a multisite genetic-first study. Biol. Psychiatry 84, 253–264 (2018)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR38" id="ref-link-section-d23130190e3305">38</a></sup> were acquired from two different cohorts: one from the Simons Variation in Individuals Project, consisting mostly of families from the United States. From this cohort, there are 24 individuals with the 16p11.2 deletion and 24 corresponding siblings from the same family who do not carry the deletion; and a second cohort consisting of individuals from the European 16p11.2 consortium (24 deletion carriers and 24 familial controls). More exact information about this cohort was previously published<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 38" title="Martin-Brevet, S. et al. Quantifying the effects of 16p11.2 copy number variants on brain structure: a multisite genetic-first study. Biol. Psychiatry 84, 253–264 (2018)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR38" id="ref-link-section-d23130190e3309">38</a></sup>. Deletion carriers were ascertained regardless of age or clinical diagnosis. DNA was extracted from stool samples, and 16S sequencing was performed using primers to the V4 region.</p><h3 id="Sec14">Data processing</h3><p>We constructed matched reference databases for 16S and SMS data analyses. The Web of Life 2 (WoL2) reference genome database contains 15,953 bacterial and archaeal genomes sampled from the NCBI to maximize representation of biodiversity. It is a major upgrade from WoL (10,575 genomes). A reference phylogeny was reconstructed based on 387 universal marker genes using uDance, a novel phylogenomic inference workflow employing a divide-and-conquer method. Taxonomic assignments of the genomes were based on Genome Taxonomy Database (GTDB) r207 and curated according to the phylogeny using tax2tree. The Greengenes2 reference 16S rRNA database was constructed based on the WoL2 whole-genome phylogeny and updated with full-length 16S rRNA sequences from the Living Tree Project and 16S from high-quality bacterial operons, using uDance to revise the topology. Into this backbone, we inserted all 16S V4 amplicon sequencing variants from public and private samples Qiita using DEPP. A taxonomy based on GTDB r207, expanded with lineages from the Living Tree Project not present in GTDB, was decorated onto the phylogeny using tax2tree. Full details behind the construction of WoL and Greengenes2 can be found in Usyk et al.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 32" title="Usyk, M. et al. Comprehensive evaluation of shotgun metagenomics, amplicon sequencing, and harmonization of these platforms for epidemiological studies. Cell Rep. Methods 3, 100391 (2023)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR32" id="ref-link-section-d23130190e3322">32</a></sup>.</p><p>The 16S amplicon and shotgun metagenomics samples were downloaded from the SRA. The 16S amplicon samples were processed using Deblur and subsequently mapped to Greenegenes2 using Vsearch with qiime2 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 68" title="Bolyen, E. et al. Reproducible, interactive, scalable and extensible microbiome data science using qiime 2. Nat. Biotechnol. 37, 852–857 (2019)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR68" id="ref-link-section-d23130190e3329">68</a></sup>). Shotgun metagenomics samples were mapped to bacterial whole genomes captured in the WoL2 using Bowtie2 followed by Woltka<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 69" title="Zhu, Q. et al. Phylogeny-aware analysis of metagenome community ecology based on matched reference genomes while bypassing taxonomy. mSystems 7.2, e00167-22 (2022)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR69" id="ref-link-section-d23130190e3333">69</a></sup>. Viral abundances were extracted from shotgun metagenomics samples using GPD and BWA. RNA expression data were obtained directly from recount3 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 67" title="Wilks, C. et al. recount3: summaries and queries for large-scale RNA-seq expression and splicing. Genome Biol. 22, 323 (2021)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR67" id="ref-link-section-d23130190e3337">67</a></sup>); the four metabolomics datasets were provided by the authors.</p><p>To enable age and sex matching, a bipartite matching between individuals with ASD and neurotypical individuals was performed using age and sex covariates. This approach has been shown to be optimal for case–control matching<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 70" title="Rosenbaum, P. R. Optimal matching for observational studies. J. Am. Stat. Assoc. 84, 1024–1032 (1989)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR70" id="ref-link-section-d23130190e3344">70</a></sup>. Individuals who could not be matched were excluded from the meta-analysis. Among the 16S and SMS datasets, there were multiple longitudinal datasets. To integrate these datasets into the cross-sectional analysis, we picked only the first timepoint for each individual.</p><h3 id="Sec15">Differential ranking analysis</h3><p>One of the most common approaches to evaluating microbiome and other omic studies consists of determining differences in the abundances of microbial taxa, human metabolites or other omic features between cases and controls. Such differential abundance analysis is typically performed by computing the log fold changes between the case and control groups<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 21" title="Love, M. I., Huber, W. &amp; Anders, S. Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. Genome Biol. 15, 550 (2014)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR21" id="ref-link-section-d23130190e3356">21</a></sup>. However, confounders, such as sex-related, age-related and geography-related batch effects, compositionality, high dimensionality, overdispersion and sparsity, prevented a reliable estimation of differential abundances and, thus, compromised the side-by-side comparison of these differential abundances across studies in the manner of a traditional meta-analysis. Here, we set out to overcome these inherent limitations of traditional meta-analyses by developing a generalizable approach for controlling for select confounders that would help reveal a comprehensive picture of ASD-specific omic signals.</p><p>To minimize confounder effects, we developed a Bayesian differential ranking algorithm that uses bipartite matching to optimize the age-based and sex-based pairing of ASD and control individuals within each dataset. This approach helped control for potential age and sex confounders while also minimizing batch effects, such as sample collection method, sample processing protocol, different primers and geographical provenance<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 71" title="Tremblay, J. et al. Primer and platform effects on 16S rRNA tag sequencing. Front. Microbiol. 6, 771 (2015)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR71" id="ref-link-section-d23130190e3363">71</a></sup>. Our approach could do this by leveraging recent insights into the multiplicative nature of protocol biases<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 18" title="McLaren, M. R., Willis, A. D. &amp; Callahan, B. J. Consistent and correctable bias in metagenomic sequencing experiments. eLife 8, e46923 (2019)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR18" id="ref-link-section-d23130190e3367">18</a></sup>. Fold change calculations can be designed to be robust to bias induced by protocols, provided that the fold changes are being computed only on samples processed under the same protocol. Similar observations have been made about biases induced by differences in polymerase chain reaction (PCR) primers, with abundance-based beta diversity metrics being robust to primer biases, as long as comparisons are confined to datasets generated with the same protocol<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 71" title="Tremblay, J. et al. Primer and platform effects on 16S rRNA tag sequencing. Front. Microbiol. 6, 771 (2015)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR71" id="ref-link-section-d23130190e3371">71</a></sup>. We extended this strategy to handle age and sex matching, taking advantage of the fact that most of the cohorts that we analyzed selected their participants to be age and sex matched. Most of the case–control pairings in the 16S and SMS datasets were within 1 year apart, providing an opportunity to remove age-related confounders in downstream analyses. Our Bayesian models were fitted via Markov chain Monte Carlo (MCMC) using Stan<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 72" title="Stan Development Team. Stan modeling language userʼs guide and reference manual. 
                  https://mc-stan.org
                  
                " href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR72" id="ref-link-section-d23130190e3375">72</a></sup>. Conceptually, this allowed us to compute log fold change differences of microbes between age-matched and sex-matched individuals, but, because we did not have absolute abundance information, we could estimate this log fold change only up to a constant<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 73" title="Morton, J. T. et al. Learning representations of microbe–metabolite interactions. Nat. Methods 16, 1306–1314 (2019)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR73" id="ref-link-section-d23130190e3379">73</a></sup> (<a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://bellmar.medium.com/articles/s41593-023-01361-0#MOESM1">Supplementary Information</a>).</p><p>To determine how sensitive our proposed differential abundance strategy was to sequencing depth, we conducted a rarefaction benchmark in addition to a simulation benchmark. When comparing unrarefied data (with mean sequencing depth greater than 200,000 reads per sample) and sequencing count data rarefied down to 9,000 reads per sample from the 16S cross-sectional cohort data, we still see strong agreement between the unrarefied log fold changes and the rarefied log fold changes (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bellmar.medium.com/articles/s41593-023-01361-0#Fig6">2e</a>). This supported the theoretical evidence that our differential abundance method was scale equivariant and that changes in sequencing depth would not markedly affect the mean log fold change estimates.</p><p>This was further validated in our simulation benchmarks, where we showed that our model could capture the ground truth log fold changes based on 16S differentials from the age-matched and sex-matched cohort (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bellmar.medium.com/articles/s41593-023-01361-0#Fig6">2f</a>). We compared our proposed age-matched and sex-matched differential ranking method to ANCOM-BC and our differential abundance method without age and sex matching (which we will refer to as group-averaged differential ranking) (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bellmar.medium.com/articles/s41593-023-01361-0#Fig6">2i–k</a>) to showcase the differences between these methods. This benchmark was performed using data-driven simulations derived from the 16S cohort analysis. For the side-by-side comparison, we ran three different configurations of ANCOM-BC: (1) case–control differences only [‘formula=disease status’]; (2) case–control differences adjusted by age and sex confounders [‘formula=disease status + age + sex’]; and (3) case–control differences by age and sex matching [‘formula=disease status + (disease status–age sex matching IDs)’]. The first configuration provided a direct comparator to our ‘standard group-averaged differential ranking’, and the third configuration provided the most direct comparator to our ‘sex- and age-matched differential ranking’. None of the three ANCOM-BC models could recover the ground truth log fold changes in our simulations (<i>r</i> = 0.38, 0.37 and 0.39 for implementations 1, 2 and 3, respectively), whereas both the ‘standard group-averaged differential ranking’ and the ‘sex- and age-matched differential ranking’ models were able to recover the ground truth (<i>r</i> = 0.64 and 0.79, respectively). Ultimately, this illustrates how our method could account for age and sex matching and perform as expected if the assumptions were satisfied.</p><p>Similar to other simulation-based benchmarks, this is not a rigorous benchmark showcasing the improved performance of our method; rather, it is showcasing how all three methods have different assumptions. To determine in which biological scenarios age and sex matching could be more informative than household matching, we generated another simulation incorporating both a household confounder and an age confounder. The subject ages and age differences were sampled from the age distribution observed in David et al.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 37" title="David, M. M. et al. Children with autism and their typically developing siblings differ in amplicon sequence variants and predicted functions of stool-associated microbes. mSystems 6, e00193-20 (2021)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR37" id="ref-link-section-d23130190e3412">37</a></sup>. Similarly to our previous simulations, we simulated the ground truth log fold changes using the model from the 16S cohort analysis. Here, we observed that, with a sufficiently large age confounder, the household matching estimated log fold changes with a noticeably large bias (mean squared error = 589.3) (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bellmar.medium.com/articles/s41593-023-01361-0#Fig10">6e</a>). In contrast, although age and sex matching did not precisely estimate the ground truth log fold changes, we observed a 10×-fold reduction in bias (mean squared error = 57.8) (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bellmar.medium.com/articles/s41593-023-01361-0#Fig10">6f</a>). This simulation also showcased how cohort randomization may play a role in mitigating the bias introduced by age confounding, at the expense of increased variance of the estimator.</p><p>To determine how sensitive our proposed differential abundance strategy is to batch effects, we computed the log fold changes between two samples, ‘sample4’ and ‘sample6’, from the MBQC study<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 31" title="Sinha, R. et al. Assessment of variation in microbial community amplicon sequencing by the microbiome quality control (mbqc) project consortium. Nat. Biotechnol. 35, 1077–1086 (2017)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR31" id="ref-link-section-d23130190e3425">31</a></sup> for each processing laboratory. These samples were replicated and processed by multiple laboratories, providing an experimental setup for validating batch removal methods. Bray–Curtis PCoA shows a weak separation in sample name but a strong separation due to batch effects induced by differences in processing protocols. However, when we compare the log fold changes for each processing laboratory, we see strong agreement (<i>r</i> &gt; 0.5, <i>P</i> &lt; &lt; 0.05) (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bellmar.medium.com/articles/s41593-023-01361-0#Fig10">6m</a>), which supports the claim of McLaren et al.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 18" title="McLaren, M. R., Willis, A. D. &amp; Callahan, B. J. Consistent and correctable bias in metagenomic sequencing experiments. eLife 8, e46923 (2019)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR18" id="ref-link-section-d23130190e3438">18</a></sup> that within-study fold change calculations are insensitive to batch effect, as long as the processing protocol is consistent within the study.</p><p>To determine if there was a significant difference between the age-matched and sex-matched pairs, we constructed an effect size metric using our model’s uncertainty estimation (see Supplementary Methods for more details). A global model for each data type—16S, SMS and RNA-seq—was used to determine if there was a significant difference between age-matched and sex-matched case–control pairs across each datatype. When we evaluated our Bayesian model fit on the 16S, SMS and RNA-seq datasets, our model fits achieved Rhat values below 1.1 and ESS values above 300, indicating that the draws from the posterior distribution are reliable.</p><p>The age-matched and sex-matched classifiers were constructed to build a classifier that generalizes across cohorts, identifying microbes that consistently differentiate between age-matched and sex-matched case–control pairs. To build age-matched and sex-matched classifiers, within each age-matched and sex-matched 16S cohort, we fitted our Bayesian model and assigned taxa into three groups: ASD-associated, Control-associated and Neutral. A taxon is assigned to the ASD-associated group if 70% of their posterior samples are greater than 0; a taxon is assigned to the Control-associated group if 70% of their posterior samples is less than 0. The remaining taxa are assigned to the Neutral group. After assigning taxa to each group, for each sample, a single log ratio, or balance<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 15" title="Morton, J. T. et al. Establishing microbial composition measurement standards with reference frames. Nat. Commun. 10, 2719 (2019)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR15" id="ref-link-section-d23130190e3448">15</a></sup>, is computed by taking the geometric mean of all of the taxon abundances within each group. To create a single log ratio that generalizes across cohorts, we assigned taxa to the ASD-associated group if it appears to be ASD associated in at least two studies. The same procedure is applied to the Control-associated group. The differences of these log ratios across the case–control pairs are shown for the age-matched and sex-matched cohorts in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bellmar.medium.com/articles/s41593-023-01361-0#Fig2">2c</a>. Although we did not apply this approach to the shotgun metagenomics datasets, we showed that the log ratios constructed from the 16S datasets also separated more than 70% of the ASD–control samples, serving as an additional cross-validation.</p><p>To determine if a microbe in increased or decreased between two groups of samples, a reference frame that identifies which group of microbes is stable is required. To do this using our Bayesian models, the quantiles estimated from the posterior distribution of the log fold change is used. A microbe is said to be significantly increased if the log fold change is greater than 0 in 95% of the posterior samples (5% log fold change quantile &gt; 0). Finally, a microbe is said to be stable if the 90% quantile of the posterior distribution overlaps with 0 and the standard deviation of the posterior distribution is less than 3. Similarly, a microbe is said to be significantly decreased if the log fold change is less than 0 in 95% of the posterior samples (95% log fold change quantile &lt; 0). The reference frame in the FMT analysis used microbes that were identified to be neutral or control associated by the age-matched and sex-matched classifier, with the assumption that the average abundance of these taxa is stable throughout the entire 2-year follow-up study. The FMT analysis used the same matching strategy, but, instead of matching on age and sex, the matchings were performed on the subjects to compare different timepoints. When identifying microbes that are the core microbiome, we focused on taxa that overlapped with 0 and had a posterior standard deviation of less than 3. Similarly, when computing the overlap between the cross-sectional cohort and the microbes depleted after the FMT, we focused on taxa with low uncertainty with a posterior standard deviation of less than 3.</p><p>The heat map shown in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bellmar.medium.com/articles/s41593-023-01361-0#Fig2">2b</a> displays the log fold changes for each case–control pair. To do this, a robust center log ratio (CLR) transform was performed, and all zeros were imputed to the mean abundance for visualization purposes. The case–control log fold changes were then computed for each case–control pair.</p><h4 id="Sec16">Bayesian differential ranking</h4><p>Conceptually, the goal of a differential analysis is to make a statement about change in abundance for a given feature <i>i</i> between conditions A and B by evaluating the following null hypothesis:</p><div id="Equa"><p><span>$$\frac{{A}_{i}}{{B}_{i}}=1$$</span></p></div><p>However, most omic datasets do not provide a direct observation of the absolute quantities of <i>A</i><sub><i>i</i></sub> and <i>B</i><sub><i>i</i></sub>, or the total microbial loads <span>\({N}_{{A}_{i}}\)</span> and <span>\({N}_{{B}_{i}}\)</span> but, rather, only an observation of their proportions <span>\({p}_{{A}_{i}}\)</span> and <span>\({p}_{{B}_{i}}\)</span>, respectively, within each dataset, which are determined by a bias term, <span>\(\frac{{N}_{A}}{{N}_{B}}\)</span>. This bias term, given by</p><div id="Equb"><p><span>$$\frac{{A}_{i}}{{B}_{i}}=\frac{{p}_{{A}_{i}}{N}_{A}}{{p}_{{B}_{i}}{N}_{B}}=\frac{{p}_{{A}_{i}}}{{p}_{{B}_{i}}}\frac{{N}_{A}}{{N}_{B}}$$</span></p></div><p>results in high FDRs that cannot be adjusted for in models analyzing compositional omics datasets because the overall contribution of <i>N</i><sub><i>A</i></sub> and <i>N</i><sub><i>B</i></sub> to change cannot be unequivocally quantified<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 74" title="Vandeputte, D. et al. Quantitative microbiome profiling links gut community variation to microbial load. Nature 551, 507–511 (2017)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR74" id="ref-link-section-d23130190e3997">74</a></sup>. To avoid the total biomass bias without having to resort to performing traditional FDR corrections, we adopted a ranking approach that allowed us to sort omic features by their log fold change values independently of how large their change was in absolute terms<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 73" title="Morton, J. T. et al. Learning representations of microbe–metabolite interactions. Nat. Methods 16, 1306–1314 (2019)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR73" id="ref-link-section-d23130190e4001">73</a></sup>. Because the biomass bias impacts every species within a dataset equally, the ranking approach ignores this bias, making the approach scale invariant (Equation 1).</p><div id="Equc"><p><span>$${{{\rm{rank}}}}\left(\frac{A}{B}\right)={{{\rm{rank}}}}\left(\frac{{p}_{A}{N}_{A}}{{p}_{B}{N}_{B}}\right)={{{\rm{rank}}}}\left(\frac{{p}_{A}}{{p}_{B}}\right)$$</span></p></div><p>The overall model that we designed consisted of a customized differential abundance tool that leveraged the experimental design of each study included in the analysis to determine study-specific feature perturbation profiles that could then be combined with the normalized perturbation profiles of other studies to perform a global differential perturbation analysis. The overall model had the following structure:</p><div id="Equd"><p><span>$$\begin{array}{rcl}{y}_{i,\,j}&amp; \sim &amp;{{{\rm{NegativeBinomial}}}}({\lambda }_{i,\,j},{\alpha }_{ij})\\ \log {\lambda }_{i,\,j}&amp;=&amp;\log {N}_{i}+{C}_{k(i),\,j}+{D}_{j}{{{\rm{I}}}}[i=ASD]\end{array}$$</span></p></div><p>where <i>y</i><sub><i>i</i>, <i>j</i></sub> denotes the microbial counts in sample <i>i</i> of species <i>j</i> across <i>d</i> species; <i>λ</i><sub><i>i</i>, <i>j</i></sub>,<i>α</i><sub><i>i</i><i>j</i></sub> represents the expected counts for species <i>j</i>; sample <i>i</i>, <i>j</i> represents a microbe-specific overdispersion term; <i>N</i><sub><i>i</i></sub> represents sequencing depth (self-normalization and preemptive of rarefaction); <i>C</i><sub><i>k</i>(<i>i</i>),<i>j</i></sub> represents the log proportion of species <i>j</i> in the <i>k</i>(<i>i</i>) control subject (age matched and sex matched); and <i>D</i><sub><i>j</i></sub>I[<i>i</i> = <i>A</i><i>S</i><i>D</i>] represents the log fold change difference between control and ASD subjects with a corrective function that equals 1 when <i>i</i> corresponds to the paired ASD subject and 0 when <i>i</i> corresponds to the control subject. Incorporating <i>N</i><sub><i>i</i></sub> into the model renders the model self-normalizing and not dependent on rarefaction, and <i>C</i><sub><i>k</i>(<i>i</i>),<i>j</i></sub> incorporates the age-matching and sex-matching component for a given pair <i>k</i>. The priors for these variables are given below.</p><div id="Eque"><p><span>$$\begin{array}{rll}{\alpha }_{ij}&amp;=&amp;\frac{{a}_{0,k(i),\,j}}{{\lambda }_{ij}}+{a}_{1,k(i),\,j}+{\beta }_{p}\quad {a}_{0} \sim {{{\rm{LogNormal}}}}(0,1)\\ &amp;&amp; {a}_{1} \sim {{{\rm{LogNormal}}}}(\log (10),0.1)\\ {\beta }_{p} &amp;\sim&amp; {{{\rm{Normal}}}}({\beta }_{\mu },{\beta }_{\sigma })\quad{\beta }_{\mu } \sim {{{\rm{Normal}}}}(0,3)\\ &amp;&amp; {\beta }_{\sigma } \sim {{{\rm{LogNormal}}}}(\log (0.1),0.1)\\ {C}_{k(i),\,j} &amp;\sim &amp; {{{\rm{Normal}}}}({C}_{{\mu }_{j}},{C}_{{\sigma }_{j}})\quad{C}_{{\mu }_{j}} \sim {{{\rm{Normal}}}}\left(\frac{1}{d},3\right)\\ &amp;&amp; {C}_{{\sigma }_{j}} \sim {{{\rm{Normal}}}}\left(\frac{1}{d},1\right)\\ {D}_{j} &amp;\sim&amp; {{{\rm{Normal}}}}(0,3)\end{array}$$</span></p></div><p>Here, the overdispersion parameters are estimated for each microbe, for each batch and for the ASD and control groups. This approach is adapted from DESeq2, allowing for the overdispersion to be modeled by both linear and quadratic terms with respect to the abundance. Furthermore, this parameterization does allow for a compositional interpretation owing to the following rationale. The Poisson distribution with an offset term is known to approximate the multi-nomial distribution. Furthermore, the negative binomial can be re-parameterized as a gamma–Poisson distribution, allowing for overdispersion modeling by breaking the mean–variance relationship inherent in the Poisson distribution.</p><p>The age-matched and sex-matched differential abundance has a similar methodology to paired tests, such as the paired <i>t</i>-test and the Wilcoxon test. To this end, we also used this differential abundance methodology to analyze the FMT dataset. Here, instead of matching pairs of subjects, we matched pairs of timepoints and computed the differential abundance across each pair of timepoints. To make these differentials comparable, a common set of taxa that was detected to be associated with controls was selected. Specifically, taxa that had a log fold change less than 0 in the cross-sectional cohort were assigned to this reference set. The estimated log fold changes were adjusted by centering around the mean log fold in the reference dataset as follows:</p><div id="Equf"><p><span>$${{{{\boldsymbol{D}}}}}^{* }={{{\boldsymbol{D}}}}-{\bar{{{{\boldsymbol{D}}}}}}_{R}$$</span></p></div><p>where <span>\({\bar{{{{\boldsymbol{D}}}}}}_{R}\)</span> denotes the mean of the log fold changes of the reference taxa, and <b><i>D</i></b><sup>*</sup> represents the recentered log fold changes. By doing this, all timepoints will have the same reference and will be more directly comparable.</p><p>One of the advantages of the above model is that it will cancel out any multiplicative batch effect, such as PCR amplification bias, with no impact on <i>j</i>. This is because <i>D</i> is computed only within cohorts, and, as a result, cohort-specific batch effects are mitigated. Another advantage of the proposed model is that negative binomial models can be fitted independently for each microbe; as a result, the log fold change estimates for one microbe will not affect the estimates of other microbes. This can be a benefit, because these models will be agnostic to the choice of filtering criteria—filtering certain microbes will not affect the log fold change estimates of the remaining microbes. Furthermore, this differential abundance model can be applied to different types of omics data. Moreover, because we built the differential ranking model in a Bayesian environment, we were able to fit the model using an MCMC approach to estimate uncertainty by sampling the resulting posterior distributions.</p><p>For example, to make a statement about the value of an estimated posterior probability distribution <i>p</i>(<i>D</i><span>∣</span> <i>y</i>), we could compute an average value using the following approximation:</p><div id="Equg"><p><span>$${\mathbb{E}}[D]\approx \frac{1}{m}\mathop{\sum }\limits_{i=0}^{m}{\hat{D}}_{i}\quad {\hat{D}}_{i} \sim p(D| \,y)$$</span></p></div><p>Using this classic application of MCMC sampling in which <i>N</i> samples of <i>i</i> are drawn from the posterior distribution <i>p</i>(<i>D</i><span>∣</span><i>y</i>), we were able to approximate the true mean of the posterior differential abundance distributions and the corresponding effect sizes. With this, we can compute an effect size metric that determines if there is any global difference detected. This metric is analogous to PERMANOVA but one that computes this from log fold changes using the age-matched and sex-matched design. The effect size <i>E</i> is measured as follows:</p><div id="Equh"><p><span>$$E=\frac{| | {\mu }_{D}| {| }_{2}}{{r}_{D}},\quad {\mu }_{D}=\frac{1}{m}\mathop{\sum }\limits_{i=0}^{m}{\hat{D}}_{i}\quad {r}_{D}=\mathop{\max }\limits_{{\hat{D}}_{i} \sim p(D| y)}\parallel | {\hat{D}}_{i}-{\mu }_{D}\parallel {| }_{2}$$</span></p></div><p>where <i>μ</i><sub><i>D</i></sub> is the mean of the posterior distribution, and <i>r</i><sub><i>D</i></sub> represents the radius of a sphere that contains all of the samples from the posterior distribution. If the effect size is greater than 1, that means that 0 is not included in the posterior distribution, and the difference is significant. Bayesian <i>P</i> values are computed from the number of draws of <span>\({\hat{D}}_{i}\)</span> that were simulated from the posterior distribution <i>p</i>(<i>D</i><span>∣</span><i>y</i>). For instance, if 100 draws are sampled from the posterior distribution, and 0 is not within the sphere estimated from those 100 draws, then we say that the posterior distribution is significantly not overlapping with 0 with <i>P</i> &lt; 0.01.</p><h3 id="Sec17">Other methods</h3><p>We fitted gradient boosting classifiers on 10 16S datasets and on three SMS datasets using q2-sample-classifier<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 68" title="Bolyen, E. et al. Reproducible, interactive, scalable and extensible microbiome data science using qiime 2. Nat. Biotechnol. 37, 852–857 (2019)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR68" id="ref-link-section-d23130190e5846">68</a></sup>. We randomly split the samples into 80/20 training and test splits, performed a fivefold cross-validation on the training datasets to obtain optimal model parameters and computed predictions on the held-out test dataset. PERMANOVA with Bray–Curtis distances was used to determine if confounding variation due to household, age and sex was statistically significant in the sibling cohorts.</p><p>We used MMvec<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 73" title="Morton, J. T. et al. Learning representations of microbe–metabolite interactions. Nat. Methods 16, 1306–1314 (2019)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR73" id="ref-link-section-d23130190e5853">73</a></sup> to perform the diet–microbe co-occurrence analysis. Here, microbes were used to predict dietary intake. This analysis enabled the estimation of conditional probabilities, namely the probability of observing a dietary compound given that the microbe was already observed. To estimate these conditional probabilities, MMvec performs a matrix factorization, identifying the factors that explain the most information in these interactions. We compared the MMvec microbial factors against the cross-sectional log fold changes. We then compared the MMvec dietary factors against <i>t</i>-statistics that measure the differences in dietary compounds between children with ASD and neurotypical children.</p><p>To identify candidate viral–microbe interactions, we ran MMvec on each of the SMS datasets. We then pulled out the top co-occurring viral taxa for each microbe that had a conditional log probability greater than 1, amounting to 78,580 microbe–viral interactions. Then, we filtered out the microbe–viral interactions that were not present in the Gut Phage Database (GPD)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 44" title="Camarillo-Guerrero, L. F., Almeida, A., Rangel-Pineros, G., Finn, R. D. &amp; Lawley, T. D. Massive expansion of human gut bacteriophage diversity. Cell 184, 1098–1109 (2021)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR44" id="ref-link-section-d23130190e5863">44</a></sup>, leaving 31,276 microbial–viral interactions. The microbe–viral interactions estimated by Dan et al.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 28" title="Dan, Z. et al. Altered gut microbial profile is associated with abnormal metabolism activity of autism spectrum disorder. Gut Microbes 11, 1246–1267 (2020)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR28" id="ref-link-section-d23130190e5867">28</a></sup> and Wang et al.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 33" title="Wang, M. et al. Alterations in gut glutamate metabolism associated with changes in gut microbiota composition in children with autism spectrum disorder. mSystems 4, e00321-18 (2019)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR33" id="ref-link-section-d23130190e5871">33</a></sup> were weakly generalizable (<i>Q</i><sup>2</sup> =0.0036 &gt; 0 and <i>Q</i><sup>2</sup> =0.0114 &gt; 0). However, the microbe–viral interactions estimated from Averina et al.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 34" title="Averina, O. V. et al. The bacterial neurometabolic signature of the gut microbiota of young children with autism spectrum disorders. J. Med. Microbiol. 69, 558–571 (2020)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR34" id="ref-link-section-d23130190e5884">34</a></sup> were similar to random chance (<i>Q</i><sup>2</sup> = −0.005).</p><p>We used Songbird<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 15" title="Morton, J. T. et al. Establishing microbial composition measurement standards with reference frames. Nat. Commun. 10, 2719 (2019)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR15" id="ref-link-section-d23130190e5895">15</a></sup> to perform the cytokine−microbe analysis via a multinomial regression that used the cytokines to predict microbial abundances. We reported biased microbial log fold changes with respect to cytokine concentration differences. Pearson correlation was used to determine the agreement between the 16S cross-sectional microbial differentials and the microbe−cytokine differentials. To directly link these microbial abundances to the cytokine concentrations, we computed log ratios, or balances, of microbes for each sample. For example, for IL-6, the numerator consisted of the top 50 microbes that are estimated to increase the most in abundance when IL-6 concentration increased, and the denominator consisted of the bottom 50 microbes that are estimated to be the most decreased when IL-6 concentration increases. Once these partitions are defined, the balances for each sample are computed by taking the log ratio of the average abundance of the numerator group and the denominator group<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 15" title="Morton, J. T. et al. Establishing microbial composition measurement standards with reference frames. Nat. Commun. 10, 2719 (2019)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR15" id="ref-link-section-d23130190e5899">15</a></sup>. Pearson correlation between these balances and the cytokine concentrations is then computed to measure the agreement between the microbial abundances and the cytokine concentrations.</p><p>To identify key microbial genes, we performed a comparative genomic analysis in which we binned the microbial genomes into those associated with ASD subjects and those associated with control subjects in the shotgun metagenomics data. We focused on microbes that are strongly associated with ASD, specifically those that are significantly greater than 10% of taxa that are estimated to be enriched in ASD. Using a binomial test, we were able to determine if a particular gene was more commonly observed in ASD-associated microbes than by random chance. Altogether, we identified 2,176 statistically significant microbial genes that differentiated ASD-associated microbial genomes from neurotypical-associated microbial genomes. Similarly, we identified 1,570 human transcripts that were differentially expressed between ASD and neurotypical subjects. Significant microbial genes and RNA transcripts were subsequently mapped to KEGG pathways. To directly compare the two contrasting omics levels and gauge metabolic similarity, we retrieved all the molecules involved in both the microbial and human pathways and calculated their intersection. Because the metabolomics datasets are not discrete values like sequencing count data, we additive log ratio (ALR) transformed the metabolomics datasets using the reference frames highlighted in the original papers. We then performed Wilcoxon tests on age-matched and sex-matched metabolomics samples within each cohort separately. Although our analysis revealed multiple metabolites that were below the 0.05 <i>P</i> value threshold, none of these metabolites passed the FDR-corrected threshold.</p><h3 id="Sec18">Reporting summary</h3><p>Further information on research design is available in the <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://bellmar.medium.com/articles/s41593-023-01361-0#MOESM2">Nature Portfolio Reporting Summary</a> linked to this article.</p></div></div></section>
                </div><div>
            <section data-title="Data availability"><div id="data-availability-section"><h2 id="data-availability">Data availability</h2><p>This study is based on previously published 16S<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Chen, Y. et al. Gut bacteria shared by children and their mothers associate with developmental level and social deficits in autism spectrum disorder. mSphere 5, e01044–20 (2020)." href="#ref-CR22" id="ref-link-section-d23130190e6022">22</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Fouquier, J. et al. The gut microbiome in autism: study-site effects and longitudinal analysis of behavior change. mSystems 6, e00848-20 (2021)." href="#ref-CR23" id="ref-link-section-d23130190e6022_1">23</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Zurita, M. F. et al. Analysis of gut microbiome, nutrition and immune status in autism spectrum disorder: a case–control study in Ecuador. Gut Microbes 11, 453–464 (2020)." href="#ref-CR24" id="ref-link-section-d23130190e6022_2">24</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 25" title="Berding, K. &amp; Donovan, S. M. Dietary patterns impact temporal dynamics of fecal microbiota composition in children with autism spectrum disorder. Front. Nutr. 6, 193 (2019)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR25" id="ref-link-section-d23130190e6025">25</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Zou, R. et al. Changes in the gut microbiota of children with autism spectrum disorder. Autism Res. 13, 1614–1625 (2020)." href="#ref-CR27" id="ref-link-section-d23130190e6028">27</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Dan, Z. et al. Altered gut microbial profile is associated with abnormal metabolism activity of autism spectrum disorder. Gut Microbes 11, 1246–1267 (2020)." href="#ref-CR28" id="ref-link-section-d23130190e6028_1">28</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 29" title="Fecal microbiome of children with autism spectrum disorder, their mothers, and healthy children. 
                  https://www.ncbi.nlm.nih.gov/bioproject/PRJNA687773
                  
                " href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR29" id="ref-link-section-d23130190e6031">29</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 37" title="David, M. M. et al. Children with autism and their typically developing siblings differ in amplicon sequence variants and predicted functions of stool-associated microbes. mSystems 6, e00193-20 (2021)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR37" id="ref-link-section-d23130190e6034">37</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 38" title="Martin-Brevet, S. et al. Quantifying the effects of 16p11.2 copy number variants on brain structure: a multisite genetic-first study. Biol. Psychiatry 84, 253–264 (2018)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR38" id="ref-link-section-d23130190e6037">38</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 52" title="Kang, D.-W. et al. Long-term benefit of microbiota transfer therapy on autism symptoms and gut microbiota. Sci. Rep. 9, 5821 (2019)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR52" id="ref-link-section-d23130190e6041">52</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 36" title="Son, J. S. et al. Comparison of fecal microbiota in children with autism spectrum disorders and neurotypical siblings in the Simons Simplex Collection. PLoS ONE 10, e0137725 (2015)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR36" id="ref-link-section-d23130190e6044">36</a></sup>, metagenomics<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 28" title="Dan, Z. et al. Altered gut microbial profile is associated with abnormal metabolism activity of autism spectrum disorder. Gut Microbes 11, 1246–1267 (2020)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR28" id="ref-link-section-d23130190e6048">28</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 33" title="Wang, M. et al. Alterations in gut glutamate metabolism associated with changes in gut microbiota composition in children with autism spectrum disorder. mSystems 4, e00321-18 (2019)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR33" id="ref-link-section-d23130190e6051">33</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 34" title="Averina, O. V. et al. The bacterial neurometabolic signature of the gut microbiota of young children with autism spectrum disorders. J. Med. Microbiol. 69, 558–571 (2020)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR34" id="ref-link-section-d23130190e6054">34</a></sup>, RNA-seq<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Velmeshev, D. et al. Single-cell genomics identifies cell type-specific molecular changes in autism. Science 364, 685–689 (2019)." href="#ref-CR59" id="ref-link-section-d23130190e6058">59</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Wright, C. et al. Altered expression of histamine signaling genes in autism spectrum disorder. Transl. Psychiatry 7, e1126 (2017)." href="#ref-CR60" id="ref-link-section-d23130190e6058_1">60</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Herrero, M. J. et al. Identification of amygdala-expressed genes associated with autism spectrum disorder. Mol. Autism 11, 39 (2020)." href="#ref-CR61" id="ref-link-section-d23130190e6058_2">61</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 62" title="Homo sapiens transcriptome or gene expression. 
                  https://www.ncbi.nlm.nih.gov/bioproject/PRJNA316853
                  
                " href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR62" id="ref-link-section-d23130190e6061">62</a></sup> and metabolomics<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Kuwabara, H. et al. Altered metabolites in the plasma of autism spectrum disorder: a capillary electrophoresis time-of-flight mass spectroscopy study. PLoS ONE 8, e73814 (2013)." href="#ref-CR48" id="ref-link-section-d23130190e6065">48</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Needham, B. D. et al. Plasma and fecal metabolite profiles in autism spectrum disorder. Biol. Psychiatry 89, 451–462 (2021)." href="#ref-CR49" id="ref-link-section-d23130190e6065_1">49</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 50" title="Noto, A. et al. The urinary metabolomics profile of an Italian autistic children population and their unaffected siblings. J. Matern. Fetal Neonatal Med. 27 Suppl 2, 46–52 (2014)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR50" id="ref-link-section-d23130190e6068">50</a></sup> data. (The 16S sequencing data in Martin-Brevet et al.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 38" title="Martin-Brevet, S. et al. Quantifying the effects of 16p11.2 copy number variants on brain structure: a multisite genetic-first study. Biol. Psychiatry 84, 253–264 (2018)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR38" id="ref-link-section-d23130190e6072">38</a></sup> is available under accession number ERP147524. All processed datasets and harmonized metadata are available on Zenodo at 10.5281/zenodo.7877350 as well as on Github at <a href="https://github.com/mortonjt/asd_multiomics_analyses">https://github.com/mortonjt/asd_multiomics_analyses</a>.</p></div></section><section data-title="Code availability"><div id="code-availability-section"><h2 id="code-availability">Code availability</h2><div id="code-availability-content">
              
              <p>Software implementation of our Bayesian age-matched and sex-matched differential ranking algorithm can be found at <a href="https://github.com/flatironinstitute/q2-matchmaker">https://github.com/flatironinstitute/q2-matchmaker</a>. Our group-averaged differential ranking algorithm can be found at <a href="https://github.com/mortonjt/q2-differential">https://github.com/mortonjt/q2-differential</a>. Finally, our analysis scripts can be found at <a href="https://github.com/mortonjt/asd_multiomics_analyses">https://github.com/mortonjt/asd_multiomics_analyses</a>. We would like to acknowledge Matplotlib, Seaborn, Scipy, Numpy, Xarray, Arviz Scikit-learn, biom-format and Scikit-bio for providing the software foundation that this work was built upon.</p>
            </div></div></section><div id="MagazineFulltextArticleBodySuffix"><section aria-labelledby="Bib1" data-title="References"><div id="Bib1-section"><h2 id="Bib1">References</h2><div id="Bib1-content"><div data-container-section="references"><ol data-track-component="outbound reference"><li data-counter="1."><p id="ref-CR1">Lord, C. et al. Autism spectrum disorder. <i>Nat. Rev. Dis. Primers</i> <b>6</b>, 5 (2020).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/s41572-019-0138-4" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41572-019-0138-4" aria-label="Article reference 1" data-doi="10.1038/s41572-019-0138-4">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31949163" aria-label="PubMed reference 1">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8900942" aria-label="PubMed Central reference 1">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 1" href="http://scholar.google.com/scholar_lookup?&amp;title=Autism%20spectrum%20disorder&amp;journal=Nat.%20Rev.%20Dis.%20Primers&amp;doi=10.1038%2Fs41572-019-0138-4&amp;volume=6&amp;publication_year=2020&amp;author=Lord%2CC">
                    Google Scholar</a> 
                </p></li><li data-counter="2."><p id="ref-CR2">Satterstrom, F. K. et al. Large-scale exome sequencing study implicates both developmental and functional changes in the neurobiology of autism. <i>Cell</i> <b>180</b>, 568–584 (2020).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.cell.2019.12.036" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.cell.2019.12.036" aria-label="Article reference 2" data-doi="10.1016/j.cell.2019.12.036">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 2" href="http://scholar.google.com/scholar_lookup?&amp;title=Large-scale%20exome%20sequencing%20study%20implicates%20both%20developmental%20and%20functional%20changes%20in%20the%20neurobiology%20of%20autism&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;volume=180&amp;publication_year=2020&amp;author=Satterstrom%2CFK">
                    Google Scholar</a> 
                </p></li><li data-counter="3."><p id="ref-CR3">Iakoucheva, L. M., Muotri, A. R. &amp; Sebat, J. Getting to the cores of autism. <i>Cell</i> <b>178</b>, 1287–1298 (2019).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.cell.2019.07.037" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.cell.2019.07.037" aria-label="Article reference 3" data-doi="10.1016/j.cell.2019.07.037">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://bellmar.medium.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhslCgsbzJ" aria-label="CAS reference 3">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31491383" aria-label="PubMed reference 3">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7039308" aria-label="PubMed Central reference 3">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 3" href="http://scholar.google.com/scholar_lookup?&amp;title=Getting%20to%20the%20cores%20of%20autism&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2019.07.037&amp;volume=178&amp;pages=1287-1298&amp;publication_year=2019&amp;author=Iakoucheva%2CLM&amp;author=Muotri%2CAR&amp;author=Sebat%2CJ">
                    Google Scholar</a> 
                </p></li><li data-counter="4."><p id="ref-CR4">Schumann, C. M. et al. The amygdala is enlarged in children but not adolescents with autism; the hippocampus is enlarged at all ages. <i>J. Neurosci.</i> <b>24</b>, 6392–6401 (2004).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1523/JNEUROSCI.1297-04.2004" data-track-action="article reference" href="https://doi.org/10.1523%2FJNEUROSCI.1297-04.2004" aria-label="Article reference 4" data-doi="10.1523/JNEUROSCI.1297-04.2004">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://bellmar.medium.com/articles/cas-redirect/1:CAS:528:DC%2BD2cXmtFersbk%3D" aria-label="CAS reference 4">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15254095" aria-label="PubMed reference 4">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6729537" aria-label="PubMed Central reference 4">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 4" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20amygdala%20is%20enlarged%20in%20children%20but%20not%20adolescents%20with%20autism%3B%20the%20hippocampus%20is%20enlarged%20at%20all%20ages&amp;journal=J.%20Neurosci.&amp;doi=10.1523%2FJNEUROSCI.1297-04.2004&amp;volume=24&amp;pages=6392-6401&amp;publication_year=2004&amp;author=Schumann%2CCM">
                    Google Scholar</a> 
                </p></li><li data-counter="5."><p id="ref-CR5">Lefter, R., Ciobica, A., Timofte, D., Stanciu, C. &amp; Trifan, A. A descriptive review on the prevalence of gastrointestinal disturbances and their multiple associations in autism spectrum disorder. <i>Medicina (Kaunas)</i> <b>56</b>, 11 (2019).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.3390/medicina56010011" data-track-action="article reference" href="https://doi.org/10.3390%2Fmedicina56010011" aria-label="Article reference 5" data-doi="10.3390/medicina56010011">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31892195" aria-label="PubMed reference 5">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 5" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20descriptive%20review%20on%20the%20prevalence%20of%20gastrointestinal%20disturbances%20and%20their%20multiple%20associations%20in%20autism%20spectrum%20disorder&amp;journal=Medicina%20%28Kaunas%29&amp;doi=10.3390%2Fmedicina56010011&amp;volume=56&amp;publication_year=2019&amp;author=Lefter%2CR&amp;author=Ciobica%2CA&amp;author=Timofte%2CD&amp;author=Stanciu%2CC&amp;author=Trifan%2CA">
                    Google Scholar</a> 
                </p></li><li data-counter="6."><p id="ref-CR6">Morais, L. H., Schreiber IV, H. L. &amp; Mazmanian, S. K. The gut microbiota–brain axis in behaviour and brain disorders. <i>Nat. Rev. Microbiol.</i> <b>19</b>, 241–255 (2021).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/s41579-020-00460-0" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41579-020-00460-0" aria-label="Article reference 6" data-doi="10.1038/s41579-020-00460-0">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://bellmar.medium.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXitFGqtb3O" aria-label="CAS reference 6">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33093662" aria-label="PubMed reference 6">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 6" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20gut%20microbiota%E2%80%93brain%20axis%20in%20behaviour%20and%20brain%20disorders&amp;journal=Nat.%20Rev.%20Microbiol.&amp;doi=10.1038%2Fs41579-020-00460-0&amp;volume=19&amp;pages=241-255&amp;publication_year=2021&amp;author=Morais%2CLH&amp;author=Schreiber%20IV%2CHL&amp;author=Mazmanian%2CSK">
                    Google Scholar</a> 
                </p></li><li data-counter="7."><p id="ref-CR7">Mayer, E. A. Gut feelings: the emerging biology of gut–brain communication. <i>Nat. Rev. Neurosci.</i> <b>12</b>, 453–466 (2011).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/nrn3071" data-track-action="article reference" href="https://doi.org/10.1038%2Fnrn3071" aria-label="Article reference 7" data-doi="10.1038/nrn3071">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://bellmar.medium.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXoslynsLY%3D" aria-label="CAS reference 7">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21750565" aria-label="PubMed reference 7">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 7" href="http://scholar.google.com/scholar_lookup?&amp;title=Gut%20feelings%3A%20the%20emerging%20biology%20of%20gut%E2%80%93brain%20communication&amp;journal=Nat.%20Rev.%20Neurosci.&amp;doi=10.1038%2Fnrn3071&amp;volume=12&amp;pages=453-466&amp;publication_year=2011&amp;author=Mayer%2CEA">
                    Google Scholar</a> 
                </p></li><li data-counter="8."><p id="ref-CR8">Sarkar, A. et al. The role of the microbiome in the neurobiology of social behaviour. <i>Biol. Rev. Camb. Philos. Soc.</i> <b>95</b>, 1131–1166 (2020).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1111/brv.12603" data-track-action="article reference" href="https://doi.org/10.1111%2Fbrv.12603" aria-label="Article reference 8" data-doi="10.1111/brv.12603">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32383208" aria-label="PubMed reference 8">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10040264" aria-label="PubMed Central reference 8">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 8" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20role%20of%20the%20microbiome%20in%20the%20neurobiology%20of%20social%20behaviour&amp;journal=Biol.%20Rev.%20Camb.%20Philos.%20Soc.&amp;doi=10.1111%2Fbrv.12603&amp;volume=95&amp;pages=1131-1166&amp;publication_year=2020&amp;author=Sarkar%2CA">
                    Google Scholar</a> 
                </p></li><li data-counter="9."><p id="ref-CR9">Jacobson, A., Yang, D., Vella, M. &amp; Chiu, I. M. The intestinal neuro–immune axis: crosstalk between neurons, immune cells, and microbes. <i>Mucosal Immunol.</i> <b>14</b>, 555–565 (2021).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/s41385-020-00368-1" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41385-020-00368-1" aria-label="Article reference 9" data-doi="10.1038/s41385-020-00368-1">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://bellmar.medium.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXjtFOnt7o%3D" aria-label="CAS reference 9">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33542493" aria-label="PubMed reference 9">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8075967" aria-label="PubMed Central reference 9">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 9" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20intestinal%20neuro%E2%80%93immune%20axis%3A%20crosstalk%20between%20neurons%2C%20immune%20cells%2C%20and%20microbes&amp;journal=Mucosal%20Immunol.&amp;doi=10.1038%2Fs41385-020-00368-1&amp;volume=14&amp;pages=555-565&amp;publication_year=2021&amp;author=Jacobson%2CA&amp;author=Yang%2CD&amp;author=Vella%2CM&amp;author=Chiu%2CIM">
                    Google Scholar</a> 
                </p></li><li data-counter="10."><p id="ref-CR10">Xu, M., Xu, X., Li, J. &amp; Li, F. Association between gut microbiota and autism spectrum disorder: a systematic review and meta-analysis. <i>Front. Psychiatry</i> <b>10</b>, 473 (2019).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.3389/fpsyt.2019.00473" data-track-action="article reference" href="https://doi.org/10.3389%2Ffpsyt.2019.00473" aria-label="Article reference 10" data-doi="10.3389/fpsyt.2019.00473">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31404299" aria-label="PubMed reference 10">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6673757" aria-label="PubMed Central reference 10">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 10" href="http://scholar.google.com/scholar_lookup?&amp;title=Association%20between%20gut%20microbiota%20and%20autism%20spectrum%20disorder%3A%20a%20systematic%20review%20and%20meta-analysis&amp;journal=Front.%20Psychiatry&amp;doi=10.3389%2Ffpsyt.2019.00473&amp;volume=10&amp;publication_year=2019&amp;author=Xu%2CM&amp;author=Xu%2CX&amp;author=Li%2CJ&amp;author=Li%2CF">
                    Google Scholar</a> 
                </p></li><li data-counter="11."><p id="ref-CR11">Xu, X.-J. et al. Comparison of the metabolic profiles in the plasma and urine samples between autistic and typically developing boys: a preliminary study. <i>Front. Psychiatry</i> <b>12</b>, 657105 (2021).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.3389/fpsyt.2021.657105" data-track-action="article reference" href="https://doi.org/10.3389%2Ffpsyt.2021.657105" aria-label="Article reference 11" data-doi="10.3389/fpsyt.2021.657105">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34149478" aria-label="PubMed reference 11">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8211775" aria-label="PubMed Central reference 11">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 11" href="http://scholar.google.com/scholar_lookup?&amp;title=Comparison%20of%20the%20metabolic%20profiles%20in%20the%20plasma%20and%20urine%20samples%20between%20autistic%20and%20typically%20developing%20boys%3A%20a%20preliminary%20study&amp;journal=Front.%20Psychiatry&amp;doi=10.3389%2Ffpsyt.2021.657105&amp;volume=12&amp;publication_year=2021&amp;author=Xu%2CX-J">
                    Google Scholar</a> 
                </p></li><li data-counter="12."><p id="ref-CR12">Azhari, A., Azizan, F. &amp; Esposito, G. A systematic review of gut-immune-brain mechanisms in autism spectrum disorder. <i>Dev. Psychobiol.</i> <b>61</b>, 752–771 (2019).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1002/dev.21803" data-track-action="article reference" href="https://doi.org/10.1002%2Fdev.21803" aria-label="Article reference 12" data-doi="10.1002/dev.21803">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30523646" aria-label="PubMed reference 12">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 12" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20systematic%20review%20of%20gut-immune-brain%20mechanisms%20in%20autism%20spectrum%20disorder&amp;journal=Dev.%20Psychobiol.&amp;doi=10.1002%2Fdev.21803&amp;volume=61&amp;pages=752-771&amp;publication_year=2019&amp;author=Azhari%2CA&amp;author=Azizan%2CF&amp;author=Esposito%2CG">
                    Google Scholar</a> 
                </p></li><li data-counter="13."><p id="ref-CR13">Schloss, P. D. Identifying and overcoming threats to reproducibility, replicability, robustness, and generalizability in microbiome research. <i>mBio</i> <b>9</b>, e00525-18 (2018).</p></li><li data-counter="14."><p id="ref-CR14">Wang, Y. &amp; LêCao, K.-A. Managing batch effects in microbiome data. <i>Brief. Bioinform.</i> <b>21</b>, 1954–1970 (2020).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1093/bib/bbz105" data-track-action="article reference" href="https://doi.org/10.1093%2Fbib%2Fbbz105" aria-label="Article reference 14" data-doi="10.1093/bib/bbz105">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31776547" aria-label="PubMed reference 14">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 14" href="http://scholar.google.com/scholar_lookup?&amp;title=Managing%20batch%20effects%20in%20microbiome%20data&amp;journal=Brief.%20Bioinform.&amp;doi=10.1093%2Fbib%2Fbbz105&amp;volume=21&amp;pages=1954-1970&amp;publication_year=2020&amp;author=Wang%2CY&amp;author=L%C3%AACao%2CK-A">
                    Google Scholar</a> 
                </p></li><li data-counter="15."><p id="ref-CR15">Morton, J. T. et al. Establishing microbial composition measurement standards with reference frames. <i>Nat. Commun.</i> <b>10</b>, 2719 (2019).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/s41467-019-10656-5" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41467-019-10656-5" aria-label="Article reference 15" data-doi="10.1038/s41467-019-10656-5">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31222023" aria-label="PubMed reference 15">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6586903" aria-label="PubMed Central reference 15">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 15" href="http://scholar.google.com/scholar_lookup?&amp;title=Establishing%20microbial%20composition%20measurement%20standards%20with%20reference%20frames&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-019-10656-5&amp;volume=10&amp;publication_year=2019&amp;author=Morton%2CJT">
                    Google Scholar</a> 
                </p></li><li data-counter="16."><p id="ref-CR16">Sullivan, P. F. &amp; Geschwind, D. H. Defining the genetic, genomic, cellular, and diagnostic architectures of psychiatric disorders. <i>Cell</i> <b>177</b>, 162–183 (2019).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.cell.2019.01.015" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.cell.2019.01.015" aria-label="Article reference 16" data-doi="10.1016/j.cell.2019.01.015">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://bellmar.medium.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXlvVWgt7w%3D" aria-label="CAS reference 16">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30901538" aria-label="PubMed reference 16">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6432948" aria-label="PubMed Central reference 16">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 16" href="http://scholar.google.com/scholar_lookup?&amp;title=Defining%20the%20genetic%2C%20genomic%2C%20cellular%2C%20and%20diagnostic%20architectures%20of%20psychiatric%20disorders&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2019.01.015&amp;volume=177&amp;pages=162-183&amp;publication_year=2019&amp;author=Sullivan%2CPF&amp;author=Geschwind%2CDH">
                    Google Scholar</a> 
                </p></li><li data-counter="17."><p id="ref-CR17">Blekhman, R. et al. Host genetic variation impacts microbiome composition across human body sites. <i>Genome Biol.</i> <b>16</b>, 191 (2015).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1186/s13059-015-0759-1" data-track-action="article reference" href="https://doi.org/10.1186%2Fs13059-015-0759-1" aria-label="Article reference 17" data-doi="10.1186/s13059-015-0759-1">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26374288" aria-label="PubMed reference 17">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4570153" aria-label="PubMed Central reference 17">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 17" href="http://scholar.google.com/scholar_lookup?&amp;title=Host%20genetic%20variation%20impacts%20microbiome%20composition%20across%20human%20body%20sites&amp;journal=Genome%20Biol.&amp;doi=10.1186%2Fs13059-015-0759-1&amp;volume=16&amp;publication_year=2015&amp;author=Blekhman%2CR">
                    Google Scholar</a> 
                </p></li><li data-counter="18."><p id="ref-CR18">McLaren, M. R., Willis, A. D. &amp; Callahan, B. J. Consistent and correctable bias in metagenomic sequencing experiments. <i>eLife</i> <b>8</b>, e46923 (2019).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.7554/eLife.46923" data-track-action="article reference" href="https://doi.org/10.7554%2FeLife.46923" aria-label="Article reference 18" data-doi="10.7554/eLife.46923">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31502536" aria-label="PubMed reference 18">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6739870" aria-label="PubMed Central reference 18">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 18" href="http://scholar.google.com/scholar_lookup?&amp;title=Consistent%20and%20correctable%20bias%20in%20metagenomic%20sequencing%20experiments&amp;journal=eLife&amp;doi=10.7554%2FeLife.46923&amp;volume=8&amp;publication_year=2019&amp;author=McLaren%2CMR&amp;author=Willis%2CAD&amp;author=Callahan%2CBJ">
                    Google Scholar</a> 
                </p></li><li data-counter="19."><p id="ref-CR19">Shafi, A., Nguyen, T., Peyvandipour, A., Nguyen, H. &amp; Draghici, S. A multi-cohort and multi-omics meta-analysis framework to identify network-based gene signatures. <i>Front. Genet.</i> <b>10</b>, 159 (2019).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.3389/fgene.2019.00159" data-track-action="article reference" href="https://doi.org/10.3389%2Ffgene.2019.00159" aria-label="Article reference 19" data-doi="10.3389/fgene.2019.00159">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://bellmar.medium.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhtlOnu73E" aria-label="CAS reference 19">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30941158" aria-label="PubMed reference 19">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6434849" aria-label="PubMed Central reference 19">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 19" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20multi-cohort%20and%20multi-omics%20meta-analysis%20framework%20to%20identify%20network-based%20gene%20signatures&amp;journal=Front.%20Genet.&amp;doi=10.3389%2Ffgene.2019.00159&amp;volume=10&amp;publication_year=2019&amp;author=Shafi%2CA&amp;author=Nguyen%2CT&amp;author=Peyvandipour%2CA&amp;author=Nguyen%2CH&amp;author=Draghici%2CS">
                    Google Scholar</a> 
                </p></li><li data-counter="20."><p id="ref-CR20">Gloor, G. B., Macklaim, J. M., Pawlowsky-Glahn, V. &amp; Egozcue, J. J. Microbiome datasets are compositional: and this is not optional. <i>Front. Microbiol.</i> <b>8</b>, 2224 (2017).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.3389/fmicb.2017.02224" data-track-action="article reference" href="https://doi.org/10.3389%2Ffmicb.2017.02224" aria-label="Article reference 20" data-doi="10.3389/fmicb.2017.02224">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29187837" aria-label="PubMed reference 20">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5695134" aria-label="PubMed Central reference 20">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 20" href="http://scholar.google.com/scholar_lookup?&amp;title=Microbiome%20datasets%20are%20compositional%3A%20and%20this%20is%20not%20optional&amp;journal=Front.%20Microbiol.&amp;doi=10.3389%2Ffmicb.2017.02224&amp;volume=8&amp;publication_year=2017&amp;author=Gloor%2CGB&amp;author=Macklaim%2CJM&amp;author=Pawlowsky-Glahn%2CV&amp;author=Egozcue%2CJJ">
                    Google Scholar</a> 
                </p></li><li data-counter="21."><p id="ref-CR21">Love, M. I., Huber, W. &amp; Anders, S. Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. <i>Genome Biol.</i> <b>15</b>, 550 (2014).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1186/s13059-014-0550-8" data-track-action="article reference" href="https://doi.org/10.1186%2Fs13059-014-0550-8" aria-label="Article reference 21" data-doi="10.1186/s13059-014-0550-8">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25516281" aria-label="PubMed reference 21">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4302049" aria-label="PubMed Central reference 21">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 21" href="http://scholar.google.com/scholar_lookup?&amp;title=Moderated%20estimation%20of%20fold%20change%20and%20dispersion%20for%20RNA-seq%20data%20with%20DESeq2&amp;journal=Genome%20Biol.&amp;doi=10.1186%2Fs13059-014-0550-8&amp;volume=15&amp;publication_year=2014&amp;author=Love%2CMI&amp;author=Huber%2CW&amp;author=Anders%2CS">
                    Google Scholar</a> 
                </p></li><li data-counter="22."><p id="ref-CR22">Chen, Y. et al. Gut bacteria shared by children and their mothers associate with developmental level and social deficits in autism spectrum disorder. <i>mSphere</i> <b>5</b>, e01044–20 (2020).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1128/mSphere.01044-20" data-track-action="article reference" href="https://doi.org/10.1128%2FmSphere.01044-20" aria-label="Article reference 22" data-doi="10.1128/mSphere.01044-20">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://bellmar.medium.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXovFyitA%3D%3D" aria-label="CAS reference 22">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33268567" aria-label="PubMed reference 22">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7716279" aria-label="PubMed Central reference 22">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 22" href="http://scholar.google.com/scholar_lookup?&amp;title=Gut%20bacteria%20shared%20by%20children%20and%20their%20mothers%20associate%20with%20developmental%20level%20and%20social%20deficits%20in%20autism%20spectrum%20disorder&amp;journal=mSphere&amp;doi=10.1128%2FmSphere.01044-20&amp;volume=5&amp;pages=e01044-20&amp;publication_year=2020&amp;author=Chen%2CY">
                    Google Scholar</a> 
                </p></li><li data-counter="23."><p id="ref-CR23">Fouquier, J. et al. The gut microbiome in autism: study-site effects and longitudinal analysis of behavior change. <i>mSystems</i> <b>6</b>, e00848-20 (2021).</p></li><li data-counter="24."><p id="ref-CR24">Zurita, M. F. et al. Analysis of gut microbiome, nutrition and immune status in autism spectrum disorder: a case–control study in Ecuador. <i>Gut Microbes</i> <b>11</b>, 453–464 (2020).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1080/19490976.2019.1662260" data-track-action="article reference" href="https://doi.org/10.1080%2F19490976.2019.1662260" aria-label="Article reference 24" data-doi="10.1080/19490976.2019.1662260">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31530087" aria-label="PubMed reference 24">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 24" href="http://scholar.google.com/scholar_lookup?&amp;title=Analysis%20of%20gut%20microbiome%2C%20nutrition%20and%20immune%20status%20in%20autism%20spectrum%20disorder%3A%20a%20case%E2%80%93control%20study%20in%20Ecuador&amp;journal=Gut%20Microbes&amp;doi=10.1080%2F19490976.2019.1662260&amp;volume=11&amp;pages=453-464&amp;publication_year=2020&amp;author=Zurita%2CMF">
                    Google Scholar</a> 
                </p></li><li data-counter="25."><p id="ref-CR25">Berding, K. &amp; Donovan, S. M. Dietary patterns impact temporal dynamics of fecal microbiota composition in children with autism spectrum disorder. <i>Front. Nutr.</i> <b>6</b>, 193 (2019).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.3389/fnut.2019.00193" data-track-action="article reference" href="https://doi.org/10.3389%2Ffnut.2019.00193" aria-label="Article reference 25" data-doi="10.3389/fnut.2019.00193">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31998741" aria-label="PubMed reference 25">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 25" href="http://scholar.google.com/scholar_lookup?&amp;title=Dietary%20patterns%20impact%20temporal%20dynamics%20of%20fecal%20microbiota%20composition%20in%20children%20with%20autism%20spectrum%20disorder&amp;journal=Front.%20Nutr.&amp;doi=10.3389%2Ffnut.2019.00193&amp;volume=6&amp;publication_year=2019&amp;author=Berding%2CK&amp;author=Donovan%2CSM">
                    Google Scholar</a> 
                </p></li><li data-counter="26."><p id="ref-CR26">Kang, D.-W. et al. Microbiota transfer therapy alters gut ecosystem and improves gastrointestinal and autism symptoms: an open-label study. <i>Microbiome</i> <b>5</b>, 10 (2017).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1186/s40168-016-0225-7" data-track-action="article reference" href="https://doi.org/10.1186%2Fs40168-016-0225-7" aria-label="Article reference 26" data-doi="10.1186/s40168-016-0225-7">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28122648" aria-label="PubMed reference 26">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5264285" aria-label="PubMed Central reference 26">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 26" href="http://scholar.google.com/scholar_lookup?&amp;title=Microbiota%20transfer%20therapy%20alters%20gut%20ecosystem%20and%20improves%20gastrointestinal%20and%20autism%20symptoms%3A%20an%20open-label%20study&amp;journal=Microbiome&amp;doi=10.1186%2Fs40168-016-0225-7&amp;volume=5&amp;publication_year=2017&amp;author=Kang%2CD-W">
                    Google Scholar</a> 
                </p></li><li data-counter="27."><p id="ref-CR27">Zou, R. et al. Changes in the gut microbiota of children with autism spectrum disorder. <i>Autism Res.</i> <b>13</b>, 1614–1625 (2020).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1002/aur.2358" data-track-action="article reference" href="https://doi.org/10.1002%2Faur.2358" aria-label="Article reference 27" data-doi="10.1002/aur.2358">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32830918" aria-label="PubMed reference 27">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 27" href="http://scholar.google.com/scholar_lookup?&amp;title=Changes%20in%20the%20gut%20microbiota%20of%20children%20with%20autism%20spectrum%20disorder&amp;journal=Autism%20Res.&amp;doi=10.1002%2Faur.2358&amp;volume=13&amp;pages=1614-1625&amp;publication_year=2020&amp;author=Zou%2CR">
                    Google Scholar</a> 
                </p></li><li data-counter="28."><p id="ref-CR28">Dan, Z. et al. Altered gut microbial profile is associated with abnormal metabolism activity of autism spectrum disorder. <i>Gut Microbes</i> <b>11</b>, 1246–1267 (2020).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1080/19490976.2020.1747329" data-track-action="article reference" href="https://doi.org/10.1080%2F19490976.2020.1747329" aria-label="Article reference 28" data-doi="10.1080/19490976.2020.1747329">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://bellmar.medium.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXns1CisrY%3D" aria-label="CAS reference 28">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32312186" aria-label="PubMed reference 28">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7524265" aria-label="PubMed Central reference 28">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 28" href="http://scholar.google.com/scholar_lookup?&amp;title=Altered%20gut%20microbial%20profile%20is%20associated%20with%20abnormal%20metabolism%20activity%20of%20autism%20spectrum%20disorder&amp;journal=Gut%20Microbes&amp;doi=10.1080%2F19490976.2020.1747329&amp;volume=11&amp;pages=1246-1267&amp;publication_year=2020&amp;author=Dan%2CZ">
                    Google Scholar</a> 
                </p></li><li data-counter="29."><p id="ref-CR29">Fecal microbiome of children with autism spectrum disorder, their mothers, and healthy children. <a href="https://www.ncbi.nlm.nih.gov/bioproject/PRJNA687773">https://www.ncbi.nlm.nih.gov/bioproject/PRJNA687773</a></p></li><li data-counter="30."><p id="ref-CR30">Lin, H. &amp; Peddada, S. D. Analysis of compositions of microbiomes with bias correction. <i>Nat. Commun.</i> <b>11</b>, 3514 (2020).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/s41467-020-17041-7" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41467-020-17041-7" aria-label="Article reference 30" data-doi="10.1038/s41467-020-17041-7">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://bellmar.medium.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhsVSmtr3I" aria-label="CAS reference 30">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32665548" aria-label="PubMed reference 30">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7360769" aria-label="PubMed Central reference 30">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 30" href="http://scholar.google.com/scholar_lookup?&amp;title=Analysis%20of%20compositions%20of%20microbiomes%20with%20bias%20correction&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-020-17041-7&amp;volume=11&amp;publication_year=2020&amp;author=Lin%2CH&amp;author=Peddada%2CSD">
                    Google Scholar</a> 
                </p></li><li data-counter="31."><p id="ref-CR31">Sinha, R. et al. Assessment of variation in microbial community amplicon sequencing by the microbiome quality control (mbqc) project consortium. <i>Nat. Biotechnol.</i> <b>35</b>, 1077–1086 (2017).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/nbt.3981" data-track-action="article reference" href="https://doi.org/10.1038%2Fnbt.3981" aria-label="Article reference 31" data-doi="10.1038/nbt.3981">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://bellmar.medium.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhsFylurvE" aria-label="CAS reference 31">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28967885" aria-label="PubMed reference 31">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5839636" aria-label="PubMed Central reference 31">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 31" href="http://scholar.google.com/scholar_lookup?&amp;title=Assessment%20of%20variation%20in%20microbial%20community%20amplicon%20sequencing%20by%20the%20microbiome%20quality%20control%20%28mbqc%29%20project%20consortium&amp;journal=Nat.%20Biotechnol.&amp;doi=10.1038%2Fnbt.3981&amp;volume=35&amp;pages=1077-1086&amp;publication_year=2017&amp;author=Sinha%2CR">
                    Google Scholar</a> 
                </p></li><li data-counter="32."><p id="ref-CR32">Usyk, M. et al. Comprehensive evaluation of shotgun metagenomics, amplicon sequencing, and harmonization of these platforms for epidemiological studies. <i>Cell Rep. Methods</i> <b>3</b>, 100391 (2023).</p></li><li data-counter="33."><p id="ref-CR33">Wang, M. et al. Alterations in gut glutamate metabolism associated with changes in gut microbiota composition in children with autism spectrum disorder. <i>mSystems</i> <b>4</b>, e00321-18 (2019).</p></li><li data-counter="34."><p id="ref-CR34">Averina, O. V. et al. The bacterial neurometabolic signature of the gut microbiota of young children with autism spectrum disorders. <i>J. Med. Microbiol.</i> <b>69</b>, 558–571 (2020).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1099/jmm.0.001178" data-track-action="article reference" href="https://doi.org/10.1099%2Fjmm.0.001178" aria-label="Article reference 34" data-doi="10.1099/jmm.0.001178">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://bellmar.medium.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhvVenur3E" aria-label="CAS reference 34">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32213246" aria-label="PubMed reference 34">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 34" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20bacterial%20neurometabolic%20signature%20of%20the%20gut%20microbiota%20of%20young%20children%20with%20autism%20spectrum%20disorders&amp;journal=J.%20Med.%20Microbiol.&amp;doi=10.1099%2Fjmm.0.001178&amp;volume=69&amp;pages=558-571&amp;publication_year=2020&amp;author=Averina%2COV">
                    Google Scholar</a> 
                </p></li><li data-counter="35."><p id="ref-CR35">Wirbel, J. et al. Meta-analysis of fecal metagenomes reveals global microbial signatures that are specific for colorectal cancer. <i>Nat. Med.</i> <b>25</b>, 679–689 (2019).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/s41591-019-0406-6" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41591-019-0406-6" aria-label="Article reference 35" data-doi="10.1038/s41591-019-0406-6">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://bellmar.medium.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXosFGqur0%3D" aria-label="CAS reference 35">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30936547" aria-label="PubMed reference 35">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7984229" aria-label="PubMed Central reference 35">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 35" href="http://scholar.google.com/scholar_lookup?&amp;title=Meta-analysis%20of%20fecal%20metagenomes%20reveals%20global%20microbial%20signatures%20that%20are%20specific%20for%20colorectal%20cancer&amp;journal=Nat.%20Med.&amp;doi=10.1038%2Fs41591-019-0406-6&amp;volume=25&amp;pages=679-689&amp;publication_year=2019&amp;author=Wirbel%2CJ">
                    Google Scholar</a> 
                </p></li><li data-counter="36."><p id="ref-CR36">Son, J. S. et al. Comparison of fecal microbiota in children with autism spectrum disorders and neurotypical siblings in the Simons Simplex Collection. <i>PLoS ONE</i> <b>10</b>, e0137725 (2015).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1371/journal.pone.0137725" data-track-action="article reference" href="https://doi.org/10.1371%2Fjournal.pone.0137725" aria-label="Article reference 36" data-doi="10.1371/journal.pone.0137725">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26427004" aria-label="PubMed reference 36">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4591364" aria-label="PubMed Central reference 36">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 36" href="http://scholar.google.com/scholar_lookup?&amp;title=Comparison%20of%20fecal%20microbiota%20in%20children%20with%20autism%20spectrum%20disorders%20and%20neurotypical%20siblings%20in%20the%20Simons%20Simplex%20Collection&amp;journal=PLoS%20ONE&amp;doi=10.1371%2Fjournal.pone.0137725&amp;volume=10&amp;publication_year=2015&amp;author=Son%2CJS">
                    Google Scholar</a> 
                </p></li><li data-counter="37."><p id="ref-CR37">David, M. M. et al. Children with autism and their typically developing siblings differ in amplicon sequence variants and predicted functions of stool-associated microbes. <i>mSystems</i> <b>6</b>, e00193-20 (2021).</p></li><li data-counter="38."><p id="ref-CR38">Martin-Brevet, S. et al. Quantifying the effects of 16p11.2 copy number variants on brain structure: a multisite genetic-first study. <i>Biol. Psychiatry</i> <b>84</b>, 253–264 (2018).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.biopsych.2018.02.1176" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.biopsych.2018.02.1176" aria-label="Article reference 38" data-doi="10.1016/j.biopsych.2018.02.1176">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://bellmar.medium.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXps1Kqsbw%3D" aria-label="CAS reference 38">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29778275" aria-label="PubMed reference 38">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 38" href="http://scholar.google.com/scholar_lookup?&amp;title=Quantifying%20the%20effects%20of%2016p11.2%20copy%20number%20variants%20on%20brain%20structure%3A%20a%20multisite%20genetic-first%20study&amp;journal=Biol.%20Psychiatry&amp;doi=10.1016%2Fj.biopsych.2018.02.1176&amp;volume=84&amp;pages=253-264&amp;publication_year=2018&amp;author=Martin-Brevet%2CS">
                    Google Scholar</a> 
                </p></li><li data-counter="39."><p id="ref-CR39">Croen, L. A. et al. Family history of immune conditions and autism spectrum and developmental disorders: findings from the study to explore early development. <i>Autism Res.</i> <b>12</b>, 123–135 (2019).</p></li><li data-counter="40."><p id="ref-CR40">Hsiao, E. Y. et al. The microbiota modulates gut physiology and behavioral abnormalities associated with autism. <i>Cell</i> <b>155</b>, 1451 (2013).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.cell.2013.11.024" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.cell.2013.11.024" aria-label="Article reference 40" data-doi="10.1016/j.cell.2013.11.024">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://bellmar.medium.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXhvFGiurbF" aria-label="CAS reference 40">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24315484" aria-label="PubMed reference 40">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3897394" aria-label="PubMed Central reference 40">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 40" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20microbiota%20modulates%20gut%20physiology%20and%20behavioral%20abnormalities%20associated%20with%20autism&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2013.11.024&amp;volume=155&amp;publication_year=2013&amp;author=Hsiao%2CEY">
                    Google Scholar</a> 
                </p></li><li data-counter="41."><p id="ref-CR41">Meng, D., Zhu, W., Ganguli, K., Shi, H. N. &amp; Walker, W. A. Anti-inflammatory effects of bifidobacterium longum subsp infantis secretions on fetal human enterocytes are mediated by TLR-4 receptors. <i>Am. J. Physiol. Gastrointest. Liver Physiol.</i> <b>311</b>, G744–G753 (2016).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1152/ajpgi.00090.2016" data-track-action="article reference" href="https://doi.org/10.1152%2Fajpgi.00090.2016" aria-label="Article reference 41" data-doi="10.1152/ajpgi.00090.2016">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27562058" aria-label="PubMed reference 41">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5142200" aria-label="PubMed Central reference 41">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 41" href="http://scholar.google.com/scholar_lookup?&amp;title=Anti-inflammatory%20effects%20of%20bifidobacterium%20longum%20subsp%20infantis%20secretions%20on%20fetal%20human%20enterocytes%20are%20mediated%20by%20TLR-4%20receptors&amp;journal=Am.%20J.%20Physiol.%20Gastrointest.%20Liver%20Physiol.&amp;doi=10.1152%2Fajpgi.00090.2016&amp;volume=311&amp;pages=G744-G753&amp;publication_year=2016&amp;author=Meng%2CD&amp;author=Zhu%2CW&amp;author=Ganguli%2CK&amp;author=Shi%2CHN&amp;author=Walker%2CWA">
                    Google Scholar</a> 
                </p></li><li data-counter="42."><p id="ref-CR42">Tett, A. et al. The <i>Prevotella copri</i> complex comprises four distinct clades underrepresented in Westernized populations. <i>Cell Host Microbe</i> <b>26</b>, 666–679 (2019).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.chom.2019.08.018" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.chom.2019.08.018" aria-label="Article reference 42" data-doi="10.1016/j.chom.2019.08.018">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 42" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20Prevotella%20copri%20complex%20comprises%20four%20distinct%20clades%20underrepresented%20in%20Westernized%20populations&amp;journal=Cell%20Host%20Microbe&amp;doi=10.1016%2Fj.chom.2019.08.018&amp;volume=26&amp;publication_year=2019&amp;author=Tett%2CA">
                    Google Scholar</a> 
                </p></li><li data-counter="43."><p id="ref-CR43">Neu, U. &amp; Mainou, B. A. Virus interactions with bacteria: partners in the infectious dance. <i>PLoS Pathog.</i> <b>16</b>, e1008234 (2020).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1371/journal.ppat.1008234" data-track-action="article reference" href="https://doi.org/10.1371%2Fjournal.ppat.1008234" aria-label="Article reference 43" data-doi="10.1371/journal.ppat.1008234">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://bellmar.medium.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXptVSns7s%3D" aria-label="CAS reference 43">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32045465" aria-label="PubMed reference 43">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7012391" aria-label="PubMed Central reference 43">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 43" href="http://scholar.google.com/scholar_lookup?&amp;title=Virus%20interactions%20with%20bacteria%3A%20partners%20in%20the%20infectious%20dance&amp;journal=PLoS%20Pathog.&amp;doi=10.1371%2Fjournal.ppat.1008234&amp;volume=16&amp;publication_year=2020&amp;author=Neu%2CU&amp;author=Mainou%2CBA">
                    Google Scholar</a> 
                </p></li><li data-counter="44."><p id="ref-CR44">Camarillo-Guerrero, L. F., Almeida, A., Rangel-Pineros, G., Finn, R. D. &amp; Lawley, T. D. Massive expansion of human gut bacteriophage diversity. <i>Cell</i> <b>184</b>, 1098–1109 (2021).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.cell.2021.01.029" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.cell.2021.01.029" aria-label="Article reference 44" data-doi="10.1016/j.cell.2021.01.029">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 44" href="http://scholar.google.com/scholar_lookup?&amp;title=Massive%20expansion%20of%20human%20gut%20bacteriophage%20diversity&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2021.01.029&amp;volume=184&amp;publication_year=2021&amp;author=Camarillo-Guerrero%2CLF&amp;author=Almeida%2CA&amp;author=Rangel-Pineros%2CG&amp;author=Finn%2CRD&amp;author=Lawley%2CTD">
                    Google Scholar</a> 
                </p></li><li data-counter="45."><p id="ref-CR45">Valles-Colomer, M. et al. The neuroactive potential of the human gut microbiota in quality of life and depression. <i>Nat. Microbiol.</i> <b>4</b>, 623–632 (2019).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/s41564-018-0337-x" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41564-018-0337-x" aria-label="Article reference 45" data-doi="10.1038/s41564-018-0337-x">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://bellmar.medium.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXmtFKhsbo%3D" aria-label="CAS reference 45">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30718848" aria-label="PubMed reference 45">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 45" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20neuroactive%20potential%20of%20the%20human%20gut%20microbiota%20in%20quality%20of%20life%20and%20depression&amp;journal=Nat.%20Microbiol.&amp;doi=10.1038%2Fs41564-018-0337-x&amp;volume=4&amp;pages=623-632&amp;publication_year=2019&amp;author=Valles-Colomer%2CM">
                    Google Scholar</a> 
                </p></li><li data-counter="46."><p id="ref-CR46">Harris, H. A., Mou, Y., Dieleman, G. C., Voortman, T. &amp; Jansen, P. W. Child autistic traits, food selectivity and diet quality: a population-based study. <i>J. Nutr</i>. <b>152</b>, 856–862 (2021).</p></li><li data-counter="47."><p id="ref-CR47">Dalangin, R., Kim, A. &amp; Campbell, R. E. The role of amino acids in neurotransmission and fluorescent tools for their detection. <i>Int. J. Mol. Sci.</i> <b>21</b>, 6197 (2020).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.3390/ijms21176197" data-track-action="article reference" href="https://doi.org/10.3390%2Fijms21176197" aria-label="Article reference 47" data-doi="10.3390/ijms21176197">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://bellmar.medium.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXitVKqtbzO" aria-label="CAS reference 47">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32867295" aria-label="PubMed reference 47">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7503967" aria-label="PubMed Central reference 47">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 47" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20role%20of%20amino%20acids%20in%20neurotransmission%20and%20fluorescent%20tools%20for%20their%20detection&amp;journal=Int.%20J.%20Mol.%20Sci.&amp;doi=10.3390%2Fijms21176197&amp;volume=21&amp;publication_year=2020&amp;author=Dalangin%2CR&amp;author=Kim%2CA&amp;author=Campbell%2CRE">
                    Google Scholar</a> 
                </p></li><li data-counter="48."><p id="ref-CR48">Kuwabara, H. et al. Altered metabolites in the plasma of autism spectrum disorder: a capillary electrophoresis time-of-flight mass spectroscopy study. <i>PLoS ONE</i> <b>8</b>, e73814 (2013).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1371/journal.pone.0073814" data-track-action="article reference" href="https://doi.org/10.1371%2Fjournal.pone.0073814" aria-label="Article reference 48" data-doi="10.1371/journal.pone.0073814">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://bellmar.medium.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXhsFeitLbE" aria-label="CAS reference 48">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24058493" aria-label="PubMed reference 48">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3776798" aria-label="PubMed Central reference 48">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 48" href="http://scholar.google.com/scholar_lookup?&amp;title=Altered%20metabolites%20in%20the%20plasma%20of%20autism%20spectrum%20disorder%3A%20a%20capillary%20electrophoresis%20time-of-flight%20mass%20spectroscopy%20study&amp;journal=PLoS%20ONE&amp;doi=10.1371%2Fjournal.pone.0073814&amp;volume=8&amp;publication_year=2013&amp;author=Kuwabara%2CH">
                    Google Scholar</a> 
                </p></li><li data-counter="49."><p id="ref-CR49">Needham, B. D. et al. Plasma and fecal metabolite profiles in autism spectrum disorder. <i>Biol. Psychiatry</i> <b>89</b>, 451–462 (2021).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.biopsych.2020.09.025" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.biopsych.2020.09.025" aria-label="Article reference 49" data-doi="10.1016/j.biopsych.2020.09.025">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://bellmar.medium.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXislSht7nL" aria-label="CAS reference 49">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33342544" aria-label="PubMed reference 49">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 49" href="http://scholar.google.com/scholar_lookup?&amp;title=Plasma%20and%20fecal%20metabolite%20profiles%20in%20autism%20spectrum%20disorder&amp;journal=Biol.%20Psychiatry&amp;doi=10.1016%2Fj.biopsych.2020.09.025&amp;volume=89&amp;pages=451-462&amp;publication_year=2021&amp;author=Needham%2CBD">
                    Google Scholar</a> 
                </p></li><li data-counter="50."><p id="ref-CR50">Noto, A. et al. The urinary metabolomics profile of an Italian autistic children population and their unaffected siblings. <i>J. Matern. Fetal Neonatal Med.</i> <b>27 Suppl 2</b>, 46–52 (2014).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.3109/14767058.2014.954784" data-track-action="article reference" href="https://doi.org/10.3109%2F14767058.2014.954784" aria-label="Article reference 50" data-doi="10.3109/14767058.2014.954784">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25284177" aria-label="PubMed reference 50">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 50" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20urinary%20metabolomics%20profile%20of%20an%20Italian%20autistic%20children%20population%20and%20their%20unaffected%20siblings&amp;journal=J.%20Matern.%20Fetal%20Neonatal%20Med.&amp;doi=10.3109%2F14767058.2014.954784&amp;volume=27%20Suppl%202&amp;pages=46-52&amp;publication_year=2014&amp;author=Noto%2CA">
                    Google Scholar</a> 
                </p></li><li data-counter="51."><p id="ref-CR51">Smith, A. M. et al. Amino acid dysregulation metabotypes: potential biomarkers for diagnosis and individualized treatment for subtypes of autism spectrum disorder. <i>Biol. Psychiatry</i> <b>85</b>, 345–354 (2019).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.biopsych.2018.08.016" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.biopsych.2018.08.016" aria-label="Article reference 51" data-doi="10.1016/j.biopsych.2018.08.016">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://bellmar.medium.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXitFyhtbzN" aria-label="CAS reference 51">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30446206" aria-label="PubMed reference 51">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 51" href="http://scholar.google.com/scholar_lookup?&amp;title=Amino%20acid%20dysregulation%20metabotypes%3A%20potential%20biomarkers%20for%20diagnosis%20and%20individualized%20treatment%20for%20subtypes%20of%20autism%20spectrum%20disorder&amp;journal=Biol.%20Psychiatry&amp;doi=10.1016%2Fj.biopsych.2018.08.016&amp;volume=85&amp;pages=345-354&amp;publication_year=2019&amp;author=Smith%2CAM">
                    Google Scholar</a> 
                </p></li><li data-counter="52."><p id="ref-CR52">Kang, D.-W. et al. Long-term benefit of microbiota transfer therapy on autism symptoms and gut microbiota. <i>Sci. Rep.</i> <b>9</b>, 5821 (2019).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/s41598-019-42183-0" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41598-019-42183-0" aria-label="Article reference 52" data-doi="10.1038/s41598-019-42183-0">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30967657" aria-label="PubMed reference 52">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6456593" aria-label="PubMed Central reference 52">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 52" href="http://scholar.google.com/scholar_lookup?&amp;title=Long-term%20benefit%20of%20microbiota%20transfer%20therapy%20on%20autism%20symptoms%20and%20gut%20microbiota&amp;journal=Sci.%20Rep.&amp;doi=10.1038%2Fs41598-019-42183-0&amp;volume=9&amp;publication_year=2019&amp;author=Kang%2CD-W">
                    Google Scholar</a> 
                </p></li><li data-counter="53."><p id="ref-CR53">Silva, Y. P., Bernardi, A. &amp; Frozza, R. L. The role of short-chain fatty acids from gut microbiota in gut–brain communication. <i>Front. Endocrinol.</i> <b>11</b>, 25 (2020).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.3389/fendo.2020.00025" data-track-action="article reference" href="https://doi.org/10.3389%2Ffendo.2020.00025" aria-label="Article reference 53" data-doi="10.3389/fendo.2020.00025">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 53" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20role%20of%20short-chain%20fatty%20acids%20from%20gut%20microbiota%20in%20gut%E2%80%93brain%20communication&amp;journal=Front.%20Endocrinol.&amp;doi=10.3389%2Ffendo.2020.00025&amp;volume=11&amp;publication_year=2020&amp;author=Silva%2CYP&amp;author=Bernardi%2CA&amp;author=Frozza%2CRL">
                    Google Scholar</a> 
                </p></li><li data-counter="54."><p id="ref-CR54">Yap, C. X. et al. Autism-related dietary preferences mediate autism–gut microbiome associations. <i>Cell</i> <b>184</b>, 5916–5931 (2021).</p></li><li data-counter="55."><p id="ref-CR55">Rodríguez, J. M. et al. The composition of the gut microbiota throughout life, with an emphasis on early life. <i>Microb. Ecol. Health Dis.</i> <b>26</b>, 26050 (2015).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25651996" aria-label="PubMed reference 55">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 55" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20composition%20of%20the%20gut%20microbiota%20throughout%20life%2C%20with%20an%20emphasis%20on%20early%20life&amp;journal=Microb.%20Ecol.%20Health%20Dis.&amp;volume=26&amp;publication_year=2015&amp;author=Rodr%C3%ADguez%2CJM">
                    Google Scholar</a> 
                </p></li><li data-counter="56."><p id="ref-CR56">Song, S. J. et al. Cohabiting family members share microbiota with one another and with their dogs. <i>eLife</i> <b>2</b>, e00458 (2013).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.7554/eLife.00458" data-track-action="article reference" href="https://doi.org/10.7554%2FeLife.00458" aria-label="Article reference 56" data-doi="10.7554/eLife.00458">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23599893" aria-label="PubMed reference 56">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3628085" aria-label="PubMed Central reference 56">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 56" href="http://scholar.google.com/scholar_lookup?&amp;title=Cohabiting%20family%20members%20share%20microbiota%20with%20one%20another%20and%20with%20their%20dogs&amp;journal=eLife&amp;doi=10.7554%2FeLife.00458&amp;volume=2&amp;publication_year=2013&amp;author=Song%2CSJ">
                    Google Scholar</a> 
                </p></li><li data-counter="57."><p id="ref-CR57">Pisula, E. &amp; Ziegart-Sadowska, K. Broader autism phenotype in siblings of children with ASD—a review. <i>Int. J. Mol. Sci.</i> <b>16</b>, 13217–13258 (2015).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.3390/ijms160613217" data-track-action="article reference" href="https://doi.org/10.3390%2Fijms160613217" aria-label="Article reference 57" data-doi="10.3390/ijms160613217">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26068453" aria-label="PubMed reference 57">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4490493" aria-label="PubMed Central reference 57">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 57" href="http://scholar.google.com/scholar_lookup?&amp;title=Broader%20autism%20phenotype%20in%20siblings%20of%20children%20with%20ASD%E2%80%94a%20review&amp;journal=Int.%20J.%20Mol.%20Sci.&amp;doi=10.3390%2Fijms160613217&amp;volume=16&amp;pages=13217-13258&amp;publication_year=2015&amp;author=Pisula%2CE&amp;author=Ziegart-Sadowska%2CK">
                    Google Scholar</a> 
                </p></li><li data-counter="58."><p id="ref-CR58">Wilson, B. C. et al. Strain engraftment competition and functional augmentation in a multi-donor fecal microbiota transplantation trial for obesity. <i>Microbiome</i> <b>9</b>, 107 (2021).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1186/s40168-021-01060-7" data-track-action="article reference" href="https://doi.org/10.1186%2Fs40168-021-01060-7" aria-label="Article reference 58" data-doi="10.1186/s40168-021-01060-7">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://bellmar.medium.com/articles/cas-redirect/1:CAS:528:DC%2BB38XjsVems7c%3D" aria-label="CAS reference 58">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33985595" aria-label="PubMed reference 58">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8120839" aria-label="PubMed Central reference 58">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 58" href="http://scholar.google.com/scholar_lookup?&amp;title=Strain%20engraftment%20competition%20and%20functional%20augmentation%20in%20a%20multi-donor%20fecal%20microbiota%20transplantation%20trial%20for%20obesity&amp;journal=Microbiome&amp;doi=10.1186%2Fs40168-021-01060-7&amp;volume=9&amp;publication_year=2021&amp;author=Wilson%2CBC">
                    Google Scholar</a> 
                </p></li><li data-counter="59."><p id="ref-CR59">Velmeshev, D. et al. Single-cell genomics identifies cell type-specific molecular changes in autism. <i>Science</i> <b>364</b>, 685–689 (2019).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1126/science.aav8130" data-track-action="article reference" href="https://doi.org/10.1126%2Fscience.aav8130" aria-label="Article reference 59" data-doi="10.1126/science.aav8130">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://bellmar.medium.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXpvVChtLY%3D" aria-label="CAS reference 59">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31097668" aria-label="PubMed reference 59">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7678724" aria-label="PubMed Central reference 59">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 59" href="http://scholar.google.com/scholar_lookup?&amp;title=Single-cell%20genomics%20identifies%20cell%20type-specific%20molecular%20changes%20in%20autism&amp;journal=Science&amp;doi=10.1126%2Fscience.aav8130&amp;volume=364&amp;pages=685-689&amp;publication_year=2019&amp;author=Velmeshev%2CD">
                    Google Scholar</a> 
                </p></li><li data-counter="60."><p id="ref-CR60">Wright, C. et al. Altered expression of histamine signaling genes in autism spectrum disorder. <i>Transl. Psychiatry</i> <b>7</b>, e1126 (2017).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/tp.2017.87" data-track-action="article reference" href="https://doi.org/10.1038%2Ftp.2017.87" aria-label="Article reference 60" data-doi="10.1038/tp.2017.87">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://bellmar.medium.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXnvVCku7s%3D" aria-label="CAS reference 60">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28485729" aria-label="PubMed reference 60">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5534955" aria-label="PubMed Central reference 60">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 60" href="http://scholar.google.com/scholar_lookup?&amp;title=Altered%20expression%20of%20histamine%20signaling%20genes%20in%20autism%20spectrum%20disorder&amp;journal=Transl.%20Psychiatry&amp;doi=10.1038%2Ftp.2017.87&amp;volume=7&amp;publication_year=2017&amp;author=Wright%2CC">
                    Google Scholar</a> 
                </p></li><li data-counter="61."><p id="ref-CR61">Herrero, M. J. et al. Identification of amygdala-expressed genes associated with autism spectrum disorder. <i>Mol. Autism</i> <b>11</b>, 39 (2020).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1186/s13229-020-00346-1" data-track-action="article reference" href="https://doi.org/10.1186%2Fs13229-020-00346-1" aria-label="Article reference 61" data-doi="10.1186/s13229-020-00346-1">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://bellmar.medium.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhtVyisbzJ" aria-label="CAS reference 61">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32460837" aria-label="PubMed reference 61">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7251751" aria-label="PubMed Central reference 61">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 61" href="http://scholar.google.com/scholar_lookup?&amp;title=Identification%20of%20amygdala-expressed%20genes%20associated%20with%20autism%20spectrum%20disorder&amp;journal=Mol.%20Autism&amp;doi=10.1186%2Fs13229-020-00346-1&amp;volume=11&amp;publication_year=2020&amp;author=Herrero%2CMJ">
                    Google Scholar</a> 
                </p></li><li data-counter="62."><p id="ref-CR62"><i>Homo sapiens</i> transcriptome or gene expression. <a href="https://www.ncbi.nlm.nih.gov/bioproject/PRJNA316853">https://www.ncbi.nlm.nih.gov/bioproject/PRJNA316853</a></p></li><li data-counter="63."><p id="ref-CR63">Kang, D.-W. et al. Differences in fecal microbial metabolites and microbiota of children with autism spectrum disorders. <i>Anaerobe</i> <b>49</b>, 121–131 (2018).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.anaerobe.2017.12.007" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.anaerobe.2017.12.007" aria-label="Article reference 63" data-doi="10.1016/j.anaerobe.2017.12.007">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://bellmar.medium.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtFKmu7Y%3D" aria-label="CAS reference 63">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29274915" aria-label="PubMed reference 63">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 63" href="http://scholar.google.com/scholar_lookup?&amp;title=Differences%20in%20fecal%20microbial%20metabolites%20and%20microbiota%20of%20children%20with%20autism%20spectrum%20disorders&amp;journal=Anaerobe&amp;doi=10.1016%2Fj.anaerobe.2017.12.007&amp;volume=49&amp;pages=121-131&amp;publication_year=2018&amp;author=Kang%2CD-W">
                    Google Scholar</a> 
                </p></li><li data-counter="64."><p id="ref-CR64">Walters, W. et al. Improved bacterial 16S rRNA gene (v4 and v4–5) and fungal internal transcribed spacer marker gene primers for microbial community surveys. <i>mSystems</i> <b>1</b>, e00009-15 (2016).</p></li><li data-counter="65."><p id="ref-CR65">Fadeev, E. et al. Comparison of two 16s rRNA primers (v3–v4 and v4–v5) for studies of Arctic microbial communities. <i>Front. Microbiol.</i> <b>12</b>, 283 (2021).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.3389/fmicb.2021.637526" data-track-action="article reference" href="https://doi.org/10.3389%2Ffmicb.2021.637526" aria-label="Article reference 65" data-doi="10.3389/fmicb.2021.637526">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 65" href="http://scholar.google.com/scholar_lookup?&amp;title=Comparison%20of%20two%2016s%20rRNA%20primers%20%28v3%E2%80%93v4%20and%20v4%E2%80%93v5%29%20for%20studies%20of%20Arctic%20microbial%20communities&amp;journal=Front.%20Microbiol.&amp;doi=10.3389%2Ffmicb.2021.637526&amp;volume=12&amp;publication_year=2021&amp;author=Fadeev%2CE">
                    Google Scholar</a> 
                </p></li><li data-counter="66."><p id="ref-CR66">Han, W. &amp; Li, L. Evaluating and minimizing batch effects in metabolomics. <i>Mass Spectrom. Rev</i>. <b>41</b>, 421–442 (2020).</p></li><li data-counter="67."><p id="ref-CR67">Wilks, C. et al. recount3: summaries and queries for large-scale RNA-seq expression and splicing. <i>Genome Biol.</i> <b>22</b>, 323 (2021).</p></li><li data-counter="68."><p id="ref-CR68">Bolyen, E. et al. Reproducible, interactive, scalable and extensible microbiome data science using qiime 2. <i>Nat. Biotechnol.</i> <b>37</b>, 852–857 (2019).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/s41587-019-0209-9" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41587-019-0209-9" aria-label="Article reference 68" data-doi="10.1038/s41587-019-0209-9">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://bellmar.medium.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhsVeksr%2FO" aria-label="CAS reference 68">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31341288" aria-label="PubMed reference 68">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7015180" aria-label="PubMed Central reference 68">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 68" href="http://scholar.google.com/scholar_lookup?&amp;title=Reproducible%2C%20interactive%2C%20scalable%20and%20extensible%20microbiome%20data%20science%20using%20qiime%202&amp;journal=Nat.%20Biotechnol.&amp;doi=10.1038%2Fs41587-019-0209-9&amp;volume=37&amp;pages=852-857&amp;publication_year=2019&amp;author=Bolyen%2CE">
                    Google Scholar</a> 
                </p></li><li data-counter="69."><p id="ref-CR69">Zhu, Q. et al. Phylogeny-aware analysis of metagenome community ecology based on matched reference genomes while bypassing taxonomy. <i>mSystems</i> <b>7.2</b>, e00167-22 (2022).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1128/msystems.00167-22" data-track-action="article reference" href="https://doi.org/10.1128%2Fmsystems.00167-22" aria-label="Article reference 69" data-doi="10.1128/msystems.00167-22">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 69" href="http://scholar.google.com/scholar_lookup?&amp;title=Phylogeny-aware%20analysis%20of%20metagenome%20community%20ecology%20based%20on%20matched%20reference%20genomes%20while%20bypassing%20taxonomy&amp;journal=mSystems&amp;doi=10.1128%2Fmsystems.00167-22&amp;volume=7.2&amp;publication_year=2022&amp;author=Zhu%2CQ">
                    Google Scholar</a> 
                </p></li><li data-counter="70."><p id="ref-CR70">Rosenbaum, P. R. Optimal matching for observational studies. <i>J. Am. Stat. Assoc.</i> <b>84</b>, 1024–1032 (1989).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1080/01621459.1989.10478868" data-track-action="article reference" href="https://doi.org/10.1080%2F01621459.1989.10478868" aria-label="Article reference 70" data-doi="10.1080/01621459.1989.10478868">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 70" href="http://scholar.google.com/scholar_lookup?&amp;title=Optimal%20matching%20for%20observational%20studies&amp;journal=J.%20Am.%20Stat.%20Assoc.&amp;doi=10.1080%2F01621459.1989.10478868&amp;volume=84&amp;pages=1024-1032&amp;publication_year=1989&amp;author=Rosenbaum%2CPR">
                    Google Scholar</a> 
                </p></li><li data-counter="71."><p id="ref-CR71">Tremblay, J. et al. Primer and platform effects on 16S rRNA tag sequencing. <i>Front. Microbiol.</i> <b>6</b>, 771 (2015).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.3389/fmicb.2015.00771" data-track-action="article reference" href="https://doi.org/10.3389%2Ffmicb.2015.00771" aria-label="Article reference 71" data-doi="10.3389/fmicb.2015.00771">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26300854" aria-label="PubMed reference 71">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4523815" aria-label="PubMed Central reference 71">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 71" href="http://scholar.google.com/scholar_lookup?&amp;title=Primer%20and%20platform%20effects%20on%2016S%20rRNA%20tag%20sequencing&amp;journal=Front.%20Microbiol.&amp;doi=10.3389%2Ffmicb.2015.00771&amp;volume=6&amp;publication_year=2015&amp;author=Tremblay%2CJ">
                    Google Scholar</a> 
                </p></li><li data-counter="72."><p id="ref-CR72">Stan Development Team. Stan modeling language userʼs guide and reference manual. <a href="https://mc-stan.org">https://mc-stan.org</a></p></li><li data-counter="73."><p id="ref-CR73">Morton, J. T. et al. Learning representations of microbe–metabolite interactions. <i>Nat. Methods</i> <b>16</b>, 1306–1314 (2019).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/s41592-019-0616-3" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41592-019-0616-3" aria-label="Article reference 73" data-doi="10.1038/s41592-019-0616-3">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://bellmar.medium.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXitFSjs7vP" aria-label="CAS reference 73">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31686038" aria-label="PubMed reference 73">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6884698" aria-label="PubMed Central reference 73">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 73" href="http://scholar.google.com/scholar_lookup?&amp;title=Learning%20representations%20of%20microbe%E2%80%93metabolite%20interactions&amp;journal=Nat.%20Methods&amp;doi=10.1038%2Fs41592-019-0616-3&amp;volume=16&amp;pages=1306-1314&amp;publication_year=2019&amp;author=Morton%2CJT">
                    Google Scholar</a> 
                </p></li><li data-counter="74."><p id="ref-CR74">Vandeputte, D. et al. Quantitative microbiome profiling links gut community variation to microbial load. <i>Nature</i> <b>551</b>, 507–511 (2017).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/nature24460" data-track-action="article reference" href="https://doi.org/10.1038%2Fnature24460" aria-label="Article reference 74" data-doi="10.1038/nature24460">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://bellmar.medium.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhvVKqtb%2FM" aria-label="CAS reference 74">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29143816" aria-label="PubMed reference 74">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 74" href="http://scholar.google.com/scholar_lookup?&amp;title=Quantitative%20microbiome%20profiling%20links%20gut%20community%20variation%20to%20microbial%20load&amp;journal=Nature&amp;doi=10.1038%2Fnature24460&amp;volume=551&amp;pages=507-511&amp;publication_year=2017&amp;author=Vandeputte%2CD">
                    Google Scholar</a> 
                </p></li></ol><p><a data-track="click" data-track-action="download citation references" data-track-label="link" rel="nofollow" href="https://citation-needed.springer.com/v2/references/10.1038/s41593-023-01361-0?format=refman&amp;flavour=references">Download references<svg width="16" height="16" focusable="false" role="img" aria-hidden="true"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-download"></use></svg></a></p></div></div></div></section></div><section data-title="Acknowledgements"><div id="Ack1-section"><h2 id="Ack1">Acknowledgements</h2><p>J.T.M. was funded by the intramural research program of the Eunice Kennedy Shriver National Institute of Child Health and Human Development (NICHD). Y.S. and T.D.L. are supported by the Wellcome Trust (WT206194). M.W. is supported by the National Natural Science Foundation of China (program no. 82071733) and Shanghai talent development funding (no. 2020115). E.E. is supported by Israel Science Foundation grant 818/17 and a research grant provided by Teva Pharmaceuticals under their support of the Azrieli Faculty of Medicine. O.K. is supported by the European Research Council (ERC) under the European Union’s Horizon 2020 Research and Innovation Program (grant agreement ERC-2020-COG no. 101001355). We would like to thank A. Packer, P. Wang, N. Volfovsky, K. Martin and J. Spiro for their critical review of the manuscript. We would like to thank S. Mirarab for feedback on the construction of the Greengenes2 and Web of Life databases. We would like to thank A. Amir for insights on processing shotgun metagenomics and 16S sequencing data using the GetData software package. We would also like to thank K. Liu, H. Sherman and X.-J. Kong for insightful discussions.</p></div></section><section aria-labelledby="author-information" data-title="Author information"><div id="author-information-section"><h2 id="author-information">Author information</h2><div id="author-information-content"><p><span id="author-notes">Author notes</span></p><ol><li id="na1"><p>These authors contributed equally: James T. Morton, Gaspar Taroncher-Oldenburg.</p></li></ol><h3 id="affiliations">Authors and Affiliations</h3><ol><li id="Aff1"><p>Center for Computational Biology, Flatiron Institute, Simons Foundation, New York, NY, USA</p><p>James T. Morton, Bob Carpenter &amp; Richard Bonneau</p></li><li id="Aff2"><p>Biostatistics &amp; Bioinformatics Branch, Eunice Kennedy Shriver National Institute of Child Health and Human Development, National Institutes of Health, Bethesda, MD, USA</p><p>James T. Morton</p></li><li id="Aff3"><p>Center for Genomics and Systems Biology, Department of Biology, New York University, New York, NY, USA</p><p>Dong-Min Jin &amp; Richard Bonneau</p></li><li id="Aff4"><p>Precidiag, Inc., Watertown, MA, USA</p><p>Robert H. Mills</p></li><li id="Aff5"><p>Host-Microbiota Interactions Laboratory, Wellcome Sanger Institute, Hinxton, UK</p><p>Yan Shao &amp; Trevor D. Lawley</p></li><li id="Aff6"><p>Bioinformatics and Systems Biology Program, University of California, San Diego, La Jolla, CA, USA</p><p>Gibraan Rahman &amp; Metin Balaban</p></li><li id="Aff7"><p>Department of Pediatrics, School of Medicine, University of California, San Diego, La Jolla, CA, USA</p><p>Gibraan Rahman, Daniel McDonald, Kalen Cantrell, Antonio Gonzalez, Rob Knight &amp; Jack A. Gilbert</p></li><li id="Aff8"><p>School of Life Sciences, Arizona State University, Tempe, AZ, USA</p><p>Qiyun Zhu</p></li><li id="Aff9"><p>Biodesign Center for Fundamental and Applied Microbiomics, Arizona State University, Tempe, AZ, USA</p><p>Qiyun Zhu</p></li><li id="Aff10"><p>Department of Electrical and Computer Engineering, University of California, San Diego, La Jolla, CA, USA</p><p>Yueyu Jiang</p></li><li id="Aff11"><p>Department of Computer Science and Engineering, Jacobs School of Engineering, University of California, San Diego, La Jolla, CA, USA</p><p>Kalen Cantrell &amp; Rob Knight</p></li><li id="Aff12"><p>Azrieli Faculty of Medicine, Bar Ilan University, Safed, Israel</p><p>Julie Carmel, Linoy Mia Frankiensztajn, Omry Koren &amp; Evan Elliott</p></li><li id="Aff13"><p>Laboratory for Research in Neuroimaging, Centre for Research in Neurosciences, Department of Clinical Neurosciences, Centre Hospitalier Universitaire Vaudois, University of Lausanne, Lausanne, Switzerland</p><p>Sandra Martin-Brevet</p></li><li id="Aff14"><p>Division of Nutritional Sciences, University of Illinois, Urbana, IL, USA</p><p>Kirsten Berding &amp; Sharon M. Donovan</p></li><li id="Aff15"><p>Stark Neurosciences Research Institute, Indiana University School of Medicine, Indianapolis, IN, USA</p><p>Brittany D. Needham</p></li><li id="Aff16"><p>Department of Anatomy, Cell Biology and Physiology, Indiana University School of Medicine, Indianapolis, IN, USA</p><p>Brittany D. Needham</p></li><li id="Aff17"><p>Microbiology Institute and Health Science College, Universidad San Francisco de Quito, Quito, Ecuador</p><p>María Fernanda Zurita</p></li><li id="Aff18"><p>Departments of Microbiology &amp; Pharmaceutical Sciences, Oregon State University, Corvallis, OR, USA</p><p>Maude David</p></li><li id="Aff19"><p>Vavilov Institute of General Genetics Russian Academy of Sciences, Moscow, Russia</p><p>Olga V. Averina, Alexey S. Kovtun &amp; Valery N. Danilenko</p></li><li id="Aff20"><p>Skolkovo Institute of Science and Technology, Skolkovo, Russia</p><p>Alexey S. Kovtun</p></li><li id="Aff21"><p>Department of Biomedical Sciences, School of Medicine, University of Cagliari, Cagliari, Italy</p><p>Antonio Noto</p></li><li id="Aff22"><p>Laboratory Medicine, Department of Surgical Sciences, School of Medicine, University of Cagliari, Cagliari, Italy</p><p>Michele Mussap</p></li><li id="Aff23"><p>Shanghai Key Laboratory of Birth Defects, Division of Neonatology, Children’s Hospital of Fudan University, National Center for Children’s Health, Shanghai, China</p><p>Mingbang Wang &amp; Wenhao Zhou</p></li><li id="Aff24"><p>Microbiome Therapy Center, South China Hospital, Health Science Center, Shenzhen University, Shenzhen, China</p><p>Mingbang Wang</p></li><li id="Aff25"><p>Department of Medicine, University of Colorado Anschutz Medical Campus, Aurora, CO, USA</p><p>Daniel N. Frank</p></li><li id="Aff26"><p>Department of Medicine, Division of Gastroenterology and Hepatology, Stony Brook University, Stony Brook, NY, USA</p><p>Ellen Li</p></li><li id="Aff27"><p>Neonatal Intensive Care Unit and Neonatal Pathology, Department of Surgical Sciences, School of Medicine, University of Cagliari, Cagliari, Italy</p><p>Vassilios Fanos</p></li><li id="Aff28"><p>Pediatrics (Systems Medicine), Biomedical Data Science, and Psychiatry and Behavioral Sciences, Stanford University, Stanford, CA, USA</p><p>Dennis P. Wall</p></li><li id="Aff29"><p>Institute of Microbiology, COCIBA, Universidad San Francisco de Quito, Quito, Ecuador</p><p>Paúl Cárdenas</p></li><li id="Aff30"><p>Facultad de Ciencias Médicas, de la Salud y la Vida, Universidad Internacional del Ecuador, Quito, Ecuador</p><p>Manuel E. Baldeón</p></li><li id="Aff31"><p>Sainte Justine Hospital Research Center, Montréal, QC, Canada</p><p>Sébastien Jacquemont</p></li><li id="Aff32"><p>Department of Pediatrics, Université de Montréal, Montréal, QC, Canada</p><p>Sébastien Jacquemont</p></li><li id="Aff33"><p>The Leslie and Susan Gonda Multidisciplinary Brain Research Center, Bar Ilan University, Ramat Gan, Israel</p><p>Evan Elliott</p></li><li id="Aff34"><p>Broad Institute of MIT and Harvard, Cambridge, MA, USA</p><p>Ramnik J. Xavier</p></li><li id="Aff35"><p>Department of Molecular Biology, Massachusetts General Hospital, Boston, MA, USA</p><p>Ramnik J. Xavier</p></li><li id="Aff36"><p>Center for the Study of Inflammatory Bowel Disease, Massachusetts General Hospital, Boston, MA, USA</p><p>Ramnik J. Xavier</p></li><li id="Aff37"><p>Division of Biology &amp; Biological Engineering, California Institute of Technology, Pasadena, CA, USA</p><p>Sarkis K. Mazmanian</p></li><li id="Aff38"><p>Department of Bioengineering, University of California, San Diego, La Jolla, California, USA</p><p>Rob Knight</p></li><li id="Aff39"><p>Center for Microbiome Innovation, University of California, San Diego, La Jolla, California, USA</p><p>Rob Knight &amp; Jack A. Gilbert</p></li><li id="Aff40"><p>Scripps Institution of Oceanography, University of California, San Diego, La Jolla, CA, USA</p><p>Jack A. Gilbert</p></li><li id="Aff41"><p>Prescient Design, a Genentech Accelerator, New York, NY, USA</p><p>Richard Bonneau</p></li><li id="Aff42"><p>Gaspar Taroncher Consulting, Philadelphia, PA, USA</p><p>Gaspar Taroncher-Oldenburg</p></li><li id="Aff43"><p>Simons Foundation Autism Research Initiative, Simons Foundation, New York, NY, USA</p><p>Gaspar Taroncher-Oldenburg</p></li></ol><h3 id="contributions">Contributions</h3><p>J.T.M. and G.T.-O. conceived and designed the study, developed the software package q2-matchmaker, analyzed the data, interpreted the results and wrote the manuscript. R.B. contributed to study design, data analysis and results interpretation. R.H.M. contributed to study design, data analysis and manuscript editing. R.J.X. and S.K.M. contributed to study design. G.R. and B.C. contributed to software development and manuscript editing. D.M., Q.Z., K.C., A.G., M.B. and Y.J. have contributed the Greengenes2 and Web of Life 2 databases. D.-M.J. and Y.S. contributed to data analysis. K.B., B.D.N., M.F.Z., M.D., O.V.A., A.S.K., A.N., M.M., M.W., J.C., S.J., S.M.-B., O.K, E.E., D.N.F., E.L., W.Z., V.F., V.N.D., D.P.W., M.E.B., R.K., J.A.G., S.M.D., T.D.L. J.C. and M.F.Z. provided access to data. All authors contributed to manuscript editing.</p><h3 id="corresponding-author">Corresponding author</h3><p id="corresponding-author-list">Correspondence to
                <a id="corresp-c1" href="mailto:gtaroncher@gmail.com">Gaspar Taroncher-Oldenburg</a>.</p></div></div></section><section data-title="Ethics declarations"><div id="ethics-section"><h2 id="ethics">Ethics declarations</h2><div id="ethics-content">
              
                <h3 id="FPar4">Competing interests</h3>
                <p>R.H.M. is Scientific Director at Precidiag, Inc. T.D.L. is a co-founder and Chief Scientific Officer of Microbiotica. S.K.M. is a co-founder and has equity in Axial Therapeutics. R.J.X. is a co-founder of Celsius Therapeutics and Jnana Therapeutics, a member of the Scientific Advisory Board at Nestle and a member of the Board of Directors at Moonlake Immunotherapeutics. R.B. is currently Executive Director of Prescient Design, a Genentech Accelerator. J.T.M. is the founder of Gutz Analytics and a co-founder of Integrated Omics AI. G.T.-O. is a Consultant-in-Residence at the Simons Foundation. The remaining authors declare no competing interests.</p>
              
            </div></div></section><section data-title="Peer review"><div id="peer-review-section"><h2 id="peer-review">Peer review</h2><div id="peer-review-content">
              
              
                <h3 id="FPar3">Peer review information</h3>
                <p><i>Nature Neuroscience</i> thanks the anonymous reviewers for their contribution to the peer review of this work.</p>
              
            </div></div></section><section data-title="Additional information"><div id="additional-information-section"><h2 id="additional-information">Additional information</h2><p><b>Publisher’s note</b> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div></section><section data-title="Extended data"><div id="Sec20-section"><h2 id="Sec20">Extended data</h2><div id="Sec20-content"><div data-test="supplementary-info"><div data-test="supp-item" id="Fig5"><h3><a data-track="click" data-track-action="view supplementary info" data-track-label="link" data-test="supp-info-link" href="https://bellmar.medium.com/articles/s41593-023-01361-0/figures/5" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41593-023-01361-0/MediaObjects/41593_2023_1361_Fig5_ESM.jpg">Extended Data Fig. 1 Study approach.</a></h3><p>Metagenomic sequence data present unique quantification challenges due to a lack of total microbial load measurements, which precludes the determination of absolute microbe abundances, and to limitations brought about by sampling and sequencing depth limitations, which result in an incomplete representation of the metagenome. We devised a Bayesian differential ranking algorithm to address both these challenges, the compositional challenge and the zero-inflation challenge. <b>The compositional challenge</b>: Most sequencing count datasets lack absolute abundance information in the form of cells, colony forming units, or transcripts per volume. This limitation preempts the reliable estimation of log fold changes (LFCs) and is a defining characteristic of compositional data that can lead to excessive false positives or false negatives depending on the magnitude of the change in absolute abundances<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 15" title="Morton, J. T. et al. Establishing microbial composition measurement standards with reference frames. Nat. Commun. 10, 2719 (2019)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR15" id="ref-link-section-d23130190e6149">15</a></sup>. As illustrated in panels a) through c), microbial counts (a) are typically converted into proportional abundances (b) that are then used to compute log-fold ratios. Fold change calculations adopt the general formula <span>\(\frac{B}{A}=\frac{{N}_{B}{p}_{B}}{{N}_{A}{p}_{A}}=\frac{{p}_{B}}{{p}_{A}}\times \frac{{N}_{B}}{{N}_{A}}\)</span>, where <i>A</i> and <i>B</i> represent the two samples being compared, <i>p</i><sub><i>A</i></sub> and <i>p</i><sub><i>B</i></sub> represent the microbial proportions in <i>A</i> and <i>B</i>, and <i>N</i><sub><i>A</i></sub> and <i>N</i><sub><i>B</i></sub> represent the total number of microbes in <i>A</i> and <i>B</i>, also known as the ground truth. A key limitation of sequencing count data is their lack of proportionality to the corresponding absolute abundances in the original samples due to sequencing depth constraints. Our inability to observe <i>N</i><sub><i>A</i></sub> and <i>N</i><sub><i>B</i></sub> introduces a bias that ultimately prevents us from performing false discovery rate (FDR) correction to identify differentially abundant microbes. This bias depends on the change in microbial population size, with large population shifts leading to increased false positive and false negative rates, and an overall skewed representation of the ground truth (c). <b>The zero-inflation challenge</b>: Sampling errors and shallow sequencing lead to disproportionately high numbers of zero counts, especially for microbes present in low abundances (d). Multinomial, Poisson and Negative Binomial distributions have been used to explicitly handle zero counts<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 21" title="Love, M. I., Huber, W. &amp; Anders, S. Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. Genome Biol. 15, 550 (2014)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR21" id="ref-link-section-d23130190e6382">21</a></sup>. However, estimating log-fold differentials remains problematic when microbes are not observed in any of the samples in one group since log 0 is − <i>∞</i> and thus the true log-fold change of a zero-count microbe can not be determined (e). Bayesian inference avoids this problem by introducing a prior that prevents nonsensical log-fold change estimates (f). Specifically, this introduces a rounded-zero assumption whereby all microbes have a non-zero chance of being observed. Panel h highlights what these log-fold changes would look like using a Dirichlet prior, where every microbe has the same probability of being observed before collecting data.</p></div><div data-test="supp-item" id="Fig6"><h3><a data-track="click" data-track-action="view supplementary info" data-track-label="link" data-test="supp-info-link" href="https://bellmar.medium.com/articles/s41593-023-01361-0/figures/6" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41593-023-01361-0/MediaObjects/41593_2023_1361_Fig6_ESM.jpg">Extended Data Fig. 2 Benchmarks.</a></h3><p>(a-d) Mean and standard deviations of the per-microbe log-fold changes compared to the total sequencing depth (log10 scale) for each microbe. e) Rarefaction benchmark, showcasing how differential abundance analysis is insensitive to rarefaction. (f) Differential abundance estimation derived from a data-driven simulated 16S dataset. (g) Comparison of age- and sex-matching approach compared to standard group averaging with respect to dataset size across 7 of the 11 16S studies (excluding Kang et al<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 52" title="Kang, D.-W. et al. Long-term benefit of microbiota transfer therapy on autism symptoms and gut microbiota. Sci. Rep. 9, 5821 (2019)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR52" id="ref-link-section-d23130190e6413">52</a></sup>, David et al<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 37" title="David, M. M. et al. Children with autism and their typically developing siblings differ in amplicon sequence variants and predicted functions of stool-associated microbes. mSystems 6, e00193-20 (2021)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR37" id="ref-link-section-d23130190e6417">37</a></sup> and Son et al<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 36" title="Son, J. S. et al. Comparison of fecal microbiota in children with autism spectrum disorders and neurotypical siblings in the Simons Simplex Collection. PLoS ONE 10, e0137725 (2015)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR36" id="ref-link-section-d23130190e6421">36</a></sup> The x-axis represents the number of aggregated datasets, the y-axis on the left panel is the average <i>R</i><sup>2</sup> metric to measure the model error. (h) Number of samples analyzed on the y-axis, and the x-axis on the right panel is the number of aggregated dataset. (i-k) Simulated datasets with a sequencing depth differential between matched cases and controls, where matched controls always have a larger sequencing depth than their case counterparts. This benchmark investigates how well ANCOM-BC, group averaged differential ranking and age-sex matched differential ranking can recover the ground truth log-fold changes. The group averaged and age-sex matched differential ranking both use the Negative Binomial (NB) distribution to model sequencing count data.(l-m) Simulated datasets comparing household matching to age-sex matching. (l-n) Bray-Curtis PCoA of 2 samples replicated across 4 processing labs in the MBQC<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 31" title="Sinha, R. et al. Assessment of variation in microbial community amplicon sequencing by the microbiome quality control (mbqc) project consortium. Nat. Biotechnol. 35, 1077–1086 (2017)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR31" id="ref-link-section-d23130190e6429">31</a></sup>. (m) Pairwise comparsions of log-fold change between 2 samples across all 4 labs using group-averaged differential abundance analysis.</p></div><div data-test="supp-item" id="Fig7"><h3><a data-track="click" data-track-action="view supplementary info" data-track-label="link" data-test="supp-info-link" href="https://bellmar.medium.com/articles/s41593-023-01361-0/figures/7" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41593-023-01361-0/MediaObjects/41593_2023_1361_Fig7_ESM.jpg">Extended Data Fig. 3 Differential ranking trends observed for the virome, 16S, SMS, and RNAseq datasets analyzed in this study.</a></h3><p>The top 10% most differentially abundant features are highlighted in red. The x axis for the virome, 16S and SMS datasets is equivalent to showcase the differences in feature counts; the x axes for the RNAseq dataset is larger by a factor of 10, illustrating the stark difference in number of features of this dataset compared to the other three.</p></div><div data-test="supp-item" id="Fig8"><h3><a data-track="click" data-track-action="view supplementary info" data-track-label="link" data-test="supp-info-link" href="https://bellmar.medium.com/articles/s41593-023-01361-0/figures/8" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41593-023-01361-0/MediaObjects/41593_2023_1361_Fig8_ESM.jpg">Extended Data Fig. 4 Metabolomics differential ranking analysis across four studies.</a></h3><p>Paired <i>t</i>-tests were performed to identify differentially abundant metabolites. The metabolites shown in Needham et al consist of both fecal and serum metabolites. None of the metabolites had significant log-fold changes after applying FDR correction.</p></div><div data-test="supp-item" id="Fig9"><h3><a data-track="click" data-track-action="view supplementary info" data-track-label="link" data-test="supp-info-link" href="https://bellmar.medium.com/articles/s41593-023-01361-0/figures/9" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41593-023-01361-0/MediaObjects/41593_2023_1361_Fig9_ESM.jpg">Extended Data Fig. 5 Comparison of log-fold changes computed from 16S and SMS.</a></h3><p>(a) Comparison of taxa proportions across all 16S and SMS samples from Dan et al<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 28" title="Dan, Z. et al. Altered gut microbial profile is associated with abnormal metabolism activity of autism spectrum disorder. Gut Microbes 11, 1246–1267 (2020)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR28" id="ref-link-section-d23130190e6508">28</a></sup> the cross-sectional datasets after mapping to Greengenes2. (b) Comparison of differentials obtained from 16S and SMS on the same samples from Dan et al across taxa observed in both datasets. Only log-fold changes with high confidence (std &lt; 0.5) are shown here.</p></div><div data-test="supp-item" id="Fig10"><h3><a data-track="click" data-track-action="view supplementary info" data-track-label="link" data-test="supp-info-link" href="https://bellmar.medium.com/articles/s41593-023-01361-0/figures/10" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41593-023-01361-0/MediaObjects/41593_2023_1361_Fig10_ESM.jpg">Extended Data Fig. 6 Age differences between case-control matchings.</a></h3><p>(a) 16S age-sex matched dataset, (b) the SMS age-sex matched dataset, (c) the David et al household matched dataset (16S)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 37" title="David, M. M. et al. Children with autism and their typically developing siblings differ in amplicon sequence variants and predicted functions of stool-associated microbes. mSystems 6, e00193-20 (2021)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR37" id="ref-link-section-d23130190e6536">37</a></sup> (d) the Son et al household matched dataset (16S)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 36" title="Son, J. S. et al. Comparison of fecal microbiota in children with autism spectrum disorders and neurotypical siblings in the Simons Simplex Collection. PLoS ONE 10, e0137725 (2015)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR36" id="ref-link-section-d23130190e6540">36</a></sup> all datasets, the age of the control subject is subtracted from the age of the corresponding matched ASD subject. Neither David et al<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 37" title="David, M. M. et al. Children with autism and their typically developing siblings differ in amplicon sequence variants and predicted functions of stool-associated microbes. mSystems 6, e00193-20 (2021)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR37" id="ref-link-section-d23130190e6544">37</a></sup> or Son et al<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 36" title="Son, J. S. et al. Comparison of fecal microbiota in children with autism spectrum disorders and neurotypical siblings in the Simons Simplex Collection. PLoS ONE 10, e0137725 (2015)." href="https://bellmar.medium.com/articles/s41593-023-01361-0#ref-CR36" id="ref-link-section-d23130190e6548">36</a></sup> showed a statistical difference between ages across households. (e) Estimated microbial log-fold changes compared to ground truth microbial log-fold changes in household matched simulation. (f) Estimated microbial log-fold changes compared to ground truth microbial log-fold changes in age-sex matched simulation. (g) Percentage of case-control pairs that are within 1 year in the age-sex matched dataset and the sibling matched dataset. (h) Percentage of case-control pairs that are have the same gender in the age-sex matched dataset and the sibling matched dataset.</p></div><div data-test="supp-item" id="Fig11"><h3><a data-track="click" data-track-action="view supplementary info" data-track-label="link" data-test="supp-info-link" href="https://bellmar.medium.com/articles/s41593-023-01361-0/figures/11" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41593-023-01361-0/MediaObjects/41593_2023_1361_Fig11_ESM.jpg">Extended Data Fig. 7 Microbe-viral co-occurrence network estimated using MMvec.</a></h3><p>Microbes are colored red and viruses are colored blue. Edges are drawn between microbes and viruses if they are highly co-occurring and the interaction was annotated in GPD.</p></div><div data-test="supp-item" id="Fig12"><h3><a data-track="click" data-track-action="view supplementary info" data-track-label="link" data-test="supp-info-link" href="https://bellmar.medium.com/articles/s41593-023-01361-0/figures/12" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41593-023-01361-0/MediaObjects/41593_2023_1361_Fig12_ESM.jpg">Extended Data Fig. 8 Microbe-diet co-occurrences.</a></h3><p>Microbe-diet co-occurrence heatmaps sorted by the (a) first and (b) third principal components estimated from MMvec.</p></div><div data-test="supp-item" id="Fig13"><h3><a data-track="click" data-track-action="view supplementary info" data-track-label="link" data-test="supp-info-link" href="https://bellmar.medium.com/articles/s41593-023-01361-0/figures/13" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41593-023-01361-0/MediaObjects/41593_2023_1361_Fig13_ESM.jpg">Extended Data Fig. 9 Distribution of pathways in ASD and control-associated genes detected in SMS and RNAseq data.</a></h3><p>(a-b) Breakdown of pathways in SMS data that are associated with ASD and neurotypical controls. (c-d) Breakdown of pathways in RNAseq data that are associated with ASD and neurotypical controls. e) Overlap of ASD associated KEGG enzymes derived from the multi-cohort cross-sectional analysis and KEGG enzymes that are found to be present in the microbes that decreased in the Kang et al FMT study. f) Pathway break down of KEGG enyzmes found in both the Kang et al FMT study and ASD children in the multi-cohort cross-sectional analysis. Only microbes that were also found in the SMS data were considered in the Kang et al study.</p></div></div></div></div></section><section data-title="Supplementary information"><div id="Sec21-section"><h2 id="Sec21">Supplementary information</h2><div id="Sec21-content"><div data-test="supplementary-info"><div data-test="supp-item" id="MOESM1"><h3><a data-track="click" data-track-action="view supplementary info" data-track-label="link" data-test="supp-info-link" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41593-023-01361-0/MediaObjects/41593_2023_1361_MOESM1_ESM.pdf" data-supp-info-image="">Supplementary Information</a></h3><p>Supplementary Tables 1–3.</p></div><div data-test="supp-item" id="MOESM3"><h3><a data-track="click" data-track-action="view supplementary info" data-track-label="link" data-test="supp-info-link" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41593-023-01361-0/MediaObjects/41593_2023_1361_MOESM3_ESM.xlsx" data-supp-info-image="">Supplementary Tables 4–16</a></h3><p>Table S4: Table of statistics for 16S differentials, including mean log fold change, standard deviation log fold change, 90% credible intervals and taxonomy for each microbe. Table S5: Table of statistics for SMS differentials, including mean log fold change, standard deviation log fold change, 90% credible intervals and taxonomy for each microbe. Table S6: Table of statistics for RNA-seq differentials, including mean log fold change, standard deviation log fold change and 90% credible intervals for each transcript. Table S7: Table of statistics for viral differentials, including mean log fold change, standard deviation log fold change and 90% credible intervals for each virus. Table S8: PERMANOVA breakdown of sibling-matched cohorts looking at the confounding variation due to age, sex and household. Table S9: Microbial log fold changes due to cytokine differences, including mean log fold change for each cytokine. Table S10: Microbe virus co-occurrences estimated by MMvec, where entries represent the centered log probability of a microbe and a virus both present for a given sample. The first two columns yield the <i>t</i>-statistic and a <i>P</i> value measuring the difference between case and control dietary preferences. Table S11: A list of genes and their associated KEGG pathways that were determined to be statistically abundant in ASD-associated microbial genomes using a one-sided binomial test corrected for multiple comparisons. Table S12: A list of genes and their associated KEGG pathways that were determined to be statistically expressed in humans using a one-sided binomial test corrected for multiple comparisons. Table S13: A list of microbial pathways that were both statistically abundant in ASD-associated microbial genomes and present in GBMs. Table S14: A list of paired microbe and human pathways in addition to the number of overlapping metabolites. Table S15: Microbe diet co-occurrences estimated by MMvec, where entries represent the centered log probability of a microbe and a dietary compound both present for a given subject. Table S16: Microbial log fold changes between paired timepoints across all of the individuals in the FMT study. The reported log fold change was calculated by using the reference frame estimated in the cross-sectional analysis.</p></div></div></div></div></section><section data-title="Rights and permissions"><div id="rightslink-section"><h2 id="rightslink">Rights and permissions</h2><div id="rightslink-content">
                <p><b>Open Access</b>  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this license, visit <a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>.</p>
              <p><a data-track="click" data-track-action="view rights and permissions" data-track-label="link" href="https://s100.copyright.com/AppDispatchServlet?title=Multi-level%20analysis%20of%20the%20gut%E2%80%93brain%20axis%20shows%20autism%20spectrum%20disorder-associated%20molecular%20and%20microbial%20profiles&amp;author=James%20T.%20Morton%20et%20al&amp;contentID=10.1038%2Fs41593-023-01361-0&amp;copyright=The%20Author%28s%29&amp;publication=1097-6256&amp;publicationDate=2023-06-26&amp;publisherName=SpringerNature&amp;orderBeanReset=true&amp;oa=CC%20BY">Reprints and Permissions</a></p></div></div></section><section aria-labelledby="article-info" data-title="About this article"><div id="article-info-section"><h2 id="article-info">About this article</h2><div id="article-info-content"><div><p><a data-crossmark="10.1038/s41593-023-01361-0" target="_blank" rel="noopener" href="https://crossmark.crossref.org/dialog/?doi=10.1038/s41593-023-01361-0" data-track="click" data-track-action="Click Crossmark" data-track-label="link" data-test="crossmark"><img width="57" height="81" alt="Check for updates. Verify currency and authenticity via CrossMark" src="data:image/svg+xml;base64,<svg height="81" width="57" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="m17.35 35.45 21.3-14.2v-17.03h-21.3" fill="#989898"/><path d="m38.65 35.45-21.3-14.2v-17.03h21.3" fill="#747474"/><path d="m28 .5c-12.98 0-23.5 10.52-23.5 23.5s10.52 23.5 23.5 23.5 23.5-10.52 23.5-23.5c0-6.23-2.48-12.21-6.88-16.62-4.41-4.4-10.39-6.88-16.62-6.88zm0 41.25c-9.8 0-17.75-7.95-17.75-17.75s7.95-17.75 17.75-17.75 17.75 7.95 17.75 17.75c0 4.71-1.87 9.22-5.2 12.55s-7.84 5.2-12.55 5.2z" fill="#535353"/><path d="m41 36c-5.81 6.23-15.23 7.45-22.43 2.9-7.21-4.55-10.16-13.57-7.03-21.5l-4.92-3.11c-4.95 10.7-1.19 23.42 8.78 29.71 9.97 6.3 23.07 4.22 30.6-4.86z" fill="#9c9c9c"/><path d="m.2 58.45c0-.75.11-1.42.33-2.01s.52-1.09.91-1.5c.38-.41.83-.73 1.34-.94.51-.22 1.06-.32 1.65-.32.56 0 1.06.11 1.51.35.44.23.81.5 1.1.81l-.91 1.01c-.24-.24-.49-.42-.75-.56-.27-.13-.58-.2-.93-.2-.39 0-.73.08-1.05.23-.31.16-.58.37-.81.66-.23.28-.41.63-.53 1.04-.13.41-.19.88-.19 1.39 0 1.04.23 1.86.68 2.46.45.59 1.06.88 1.84.88.41 0 .77-.07 1.07-.23s.59-.39.85-.68l.91 1c-.38.43-.8.76-1.28.99-.47.22-1 .34-1.58.34-.59 0-1.13-.1-1.64-.31-.5-.2-.94-.51-1.31-.91-.38-.4-.67-.9-.88-1.48-.22-.59-.33-1.26-.33-2.02zm8.4-5.33h1.61v2.54l-.05 1.33c.29-.27.61-.51.96-.72s.76-.31 1.24-.31c.73 0 1.27.23 1.61.71.33.47.5 1.14.5 2.02v4.31h-1.61v-4.1c0-.57-.08-.97-.25-1.21-.17-.23-.45-.35-.83-.35-.3 0-.56.08-.79.22-.23.15-.49.36-.78.64v4.8h-1.61zm7.37 6.45c0-.56.09-1.06.26-1.51.18-.45.42-.83.71-1.14.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.36c.07.62.29 1.1.65 1.44.36.33.82.5 1.38.5.29 0 .57-.04.83-.13s.51-.21.76-.37l.55 1.01c-.33.21-.69.39-1.09.53-.41.14-.83.21-1.26.21-.48 0-.92-.08-1.34-.25-.41-.16-.76-.4-1.07-.7-.31-.31-.55-.69-.72-1.13-.18-.44-.26-.95-.26-1.52zm4.6-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.07.45-.31.29-.5.73-.58 1.3zm2.5.62c0-.57.09-1.08.28-1.53.18-.44.43-.82.75-1.13s.69-.54 1.1-.71c.42-.16.85-.24 1.31-.24.45 0 .84.08 1.17.23s.61.34.85.57l-.77 1.02c-.19-.16-.38-.28-.56-.37-.19-.09-.39-.14-.61-.14-.56 0-1.01.21-1.35.63-.35.41-.52.97-.52 1.67 0 .69.17 1.24.51 1.66.34.41.78.62 1.32.62.28 0 .54-.06.78-.17.24-.12.45-.26.64-.42l.67 1.03c-.33.29-.69.51-1.08.65-.39.15-.78.23-1.18.23-.46 0-.9-.08-1.31-.24-.4-.16-.75-.39-1.05-.7s-.53-.69-.7-1.13c-.17-.45-.25-.96-.25-1.53zm6.91-6.45h1.58v6.17h.05l2.54-3.16h1.77l-2.35 2.8 2.59 4.07h-1.75l-1.77-2.98-1.08 1.23v1.75h-1.58zm13.69 1.27c-.25-.11-.5-.17-.75-.17-.58 0-.87.39-.87 1.16v.75h1.34v1.27h-1.34v5.6h-1.61v-5.6h-.92v-1.2l.92-.07v-.72c0-.35.04-.68.13-.98.08-.31.21-.57.4-.79s.42-.39.71-.51c.28-.12.63-.18 1.04-.18.24 0 .48.02.69.07.22.05.41.1.57.17zm.48 5.18c0-.57.09-1.08.27-1.53.17-.44.41-.82.72-1.13.3-.31.65-.54 1.04-.71.39-.16.8-.24 1.23-.24s.84.08 1.24.24c.4.17.74.4 1.04.71s.54.69.72 1.13c.19.45.28.96.28 1.53s-.09 1.08-.28 1.53c-.18.44-.42.82-.72 1.13s-.64.54-1.04.7-.81.24-1.24.24-.84-.08-1.23-.24-.74-.39-1.04-.7c-.31-.31-.55-.69-.72-1.13-.18-.45-.27-.96-.27-1.53zm1.65 0c0 .69.14 1.24.43 1.66.28.41.68.62 1.18.62.51 0 .9-.21 1.19-.62.29-.42.44-.97.44-1.66 0-.7-.15-1.26-.44-1.67-.29-.42-.68-.63-1.19-.63-.5 0-.9.21-1.18.63-.29.41-.43.97-.43 1.67zm6.48-3.44h1.33l.12 1.21h.05c.24-.44.54-.79.88-1.02.35-.24.7-.36 1.07-.36.32 0 .59.05.78.14l-.28 1.4-.33-.09c-.11-.01-.23-.02-.38-.02-.27 0-.56.1-.86.31s-.55.58-.77 1.1v4.2h-1.61zm-47.87 15h1.61v4.1c0 .57.08.97.25 1.2.17.24.44.35.81.35.3 0 .57-.07.8-.22.22-.15.47-.39.73-.73v-4.7h1.61v6.87h-1.32l-.12-1.01h-.04c-.3.36-.63.64-.98.86-.35.21-.76.32-1.24.32-.73 0-1.27-.24-1.61-.71-.33-.47-.5-1.14-.5-2.02zm9.46 7.43v2.16h-1.61v-9.59h1.33l.12.72h.05c.29-.24.61-.45.97-.63.35-.17.72-.26 1.1-.26.43 0 .81.08 1.15.24.33.17.61.4.84.71.24.31.41.68.53 1.11.13.42.19.91.19 1.44 0 .59-.09 1.11-.25 1.57-.16.47-.38.85-.65 1.16-.27.32-.58.56-.94.73-.35.16-.72.25-1.1.25-.3 0-.6-.07-.9-.2s-.59-.31-.87-.56zm0-2.3c.26.22.5.37.73.45.24.09.46.13.66.13.46 0 .84-.2 1.15-.6.31-.39.46-.98.46-1.77 0-.69-.12-1.22-.35-1.61-.23-.38-.61-.57-1.13-.57-.49 0-.99.26-1.52.77zm5.87-1.69c0-.56.08-1.06.25-1.51.16-.45.37-.83.65-1.14.27-.3.58-.54.93-.71s.71-.25 1.08-.25c.39 0 .73.07 1 .2.27.14.54.32.81.55l-.06-1.1v-2.49h1.61v9.88h-1.33l-.11-.74h-.06c-.25.25-.54.46-.88.64-.33.18-.69.27-1.06.27-.87 0-1.56-.32-2.07-.95s-.76-1.51-.76-2.65zm1.67-.01c0 .74.13 1.31.4 1.7.26.38.65.58 1.15.58.51 0 .99-.26 1.44-.77v-3.21c-.24-.21-.48-.36-.7-.45-.23-.08-.46-.12-.7-.12-.45 0-.82.19-1.13.59-.31.39-.46.95-.46 1.68zm6.35 1.59c0-.73.32-1.3.97-1.71.64-.4 1.67-.68 3.08-.84 0-.17-.02-.34-.07-.51-.05-.16-.12-.3-.22-.43s-.22-.22-.38-.3c-.15-.06-.34-.1-.58-.1-.34 0-.68.07-1 .2s-.63.29-.93.47l-.59-1.08c.39-.24.81-.45 1.28-.63.47-.17.99-.26 1.54-.26.86 0 1.51.25 1.93.76s.63 1.25.63 2.21v4.07h-1.32l-.12-.76h-.05c-.3.27-.63.48-.98.66s-.73.27-1.14.27c-.61 0-1.1-.19-1.48-.56-.38-.36-.57-.85-.57-1.46zm1.57-.12c0 .3.09.53.27.67.19.14.42.21.71.21.28 0 .54-.07.77-.2s.48-.31.73-.56v-1.54c-.47.06-.86.13-1.18.23-.31.09-.57.19-.76.31s-.33.25-.41.4c-.09.15-.13.31-.13.48zm6.29-3.63h-.98v-1.2l1.06-.07.2-1.88h1.34v1.88h1.75v1.27h-1.75v3.28c0 .8.32 1.2.97 1.2.12 0 .24-.01.37-.04.12-.03.24-.07.34-.11l.28 1.19c-.19.06-.4.12-.64.17-.23.05-.49.08-.76.08-.4 0-.74-.06-1.02-.18-.27-.13-.49-.3-.67-.52-.17-.21-.3-.48-.37-.78-.08-.3-.12-.64-.12-1.01zm4.36 2.17c0-.56.09-1.06.27-1.51s.41-.83.71-1.14c.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.37c.08.62.29 1.1.65 1.44.36.33.82.5 1.38.5.3 0 .58-.04.84-.13.25-.09.51-.21.76-.37l.54 1.01c-.32.21-.69.39-1.09.53s-.82.21-1.26.21c-.47 0-.92-.08-1.33-.25-.41-.16-.77-.4-1.08-.7-.3-.31-.54-.69-.72-1.13-.17-.44-.26-.95-.26-1.52zm4.61-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.08.45-.31.29-.5.73-.57 1.3zm3.01 2.23c.31.24.61.43.92.57.3.13.63.2.98.2.38 0 .65-.08.83-.23s.27-.35.27-.6c0-.14-.05-.26-.13-.37-.08-.1-.2-.2-.34-.28-.14-.09-.29-.16-.47-.23l-.53-.22c-.23-.09-.46-.18-.69-.3-.23-.11-.44-.24-.62-.4s-.33-.35-.45-.55c-.12-.21-.18-.46-.18-.75 0-.61.23-1.1.68-1.49.44-.38 1.06-.57 1.83-.57.48 0 .91.08 1.29.25s.71.36.99.57l-.74.98c-.24-.17-.49-.32-.73-.42-.25-.11-.51-.16-.78-.16-.35 0-.6.07-.76.21-.17.15-.25.33-.25.54 0 .14.04.26.12.36s.18.18.31.26c.14.07.29.14.46.21l.54.19c.23.09.47.18.7.29s.44.24.64.4c.19.16.34.35.46.58.11.23.17.5.17.82 0 .3-.06.58-.17.83-.12.26-.29.48-.51.68-.23.19-.51.34-.84.45-.34.11-.72.17-1.15.17-.48 0-.95-.09-1.41-.27-.46-.19-.86-.41-1.2-.68z" fill="#535353"/></g></svg>"/></a></p><div><h3 id="citeas">Cite this article</h3><p>Morton, J.T., Jin, DM., Mills, R.H. <i>et al.</i> Multi-level analysis of the gut–brain axis shows autism spectrum disorder-associated molecular and microbial profiles.
                    <i>Nat Neurosci</i> <b>26</b>, 1208–1217 (2023). https://doi.org/10.1038/s41593-023-01361-0</p><p><a data-test="citation-link" data-track="click" data-track-action="download article citation" data-track-label="link" data-track-external="" rel="nofollow" href="https://citation-needed.springer.com/v2/references/10.1038/s41593-023-01361-0?format=refman&amp;flavour=citation">Download citation<svg width="16" height="16" focusable="false" role="img" aria-hidden="true"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-download"></use></svg></a></p><ul data-test="publication-history"><li><p>Received<span>: </span><span><time datetime="2022-11-11">11 November 2022</time></span></p></li><li><p>Accepted<span>: </span><span><time datetime="2023-05-13">13 May 2023</time></span></p></li><li><p>Published<span>: </span><span><time datetime="2023-06-26">26 June 2023</time></span></p></li><li><p>Issue Date<span>: </span><span><time datetime="2023-07">July 2023</time></span></p></li><li><p><abbr title="Digital Object Identifier">DOI</abbr><span>: </span><span>https://doi.org/10.1038/s41593-023-01361-0</span></p></li></ul></div></div></div></div></section>
            </div></div>
  </body>
</html>
