<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <title>James Routley | Feed</title>
    <link
      rel="stylesheet"
      type="text/css"
      href="../styles.css"
      media="screen"
    />
  </head>
  <body>
    <a href="/index.html">Back</a>
    <a href="https://www.nature.com/articles/d41586-023-01980-4">Original</a>
    <h1>How scientists are hacking the genetic code to give proteins new powers</h1>
    
    <div id="readability-page-1" class="page"><div>
                    <figure>
 <picture>
  <source type="image/webp" srcset="//media.nature.com/lw767/magazine-assets/d41586-023-01980-4/d41586-023-01980-4_25489756.jpg?as=webp 767w, //media.nature.com/lw319/magazine-assets/d41586-023-01980-4/d41586-023-01980-4_25489756.jpg?as=webp 319w" sizes="(max-width: 319px) 319px, (min-width: 1023px) 100vw,  767px"/>
  <img alt="Illustration of a ribosome producing a protein from an mRNA template." loading="lazy" src="https://media.nature.com/lw767/magazine-assets/d41586-023-01980-4/d41586-023-01980-4_25489756.jpg"/>
  <figcaption>
   <p><span>A ribosome (centre) builds a new protein (red) from messenger RNA (multicoloured).</span><span>Credit: Juan Gaertner/Science Photo Library</span></p>
  </figcaption>
 </picture>
</figure><p>For most of the history of life on Earth, genetic information has been carried in a code that specifies just 20 amino acids. Amino acids are the building blocks of proteins, which do most of the heavy lifting in the cell; their side-chains govern protein folding, interactions and chemical activities. By limiting the available side chains, nature effectively restricts the kinds of reaction that proteins can perform.</p><p>As a doctoral student in the 1980s, Peter Schultz found himself wondering why nature had restricted itself in this way — and set about trying to circumvent this limitation. Several years later, as a professor at the University of California, Berkeley, Schultz and his team managed to do so by tinkering with the machinery of protein synthesis. Although confined to a test tube, the work marked a key early success in efforts to hack the genetic code.</p><p>Since then, many researchers have followed in Schultz’s footsteps, tweaking the cellular apparatus for building proteins both to alter existing macromolecules and to create polymers from entirely new building blocks. The resulting molecules can be used in research and for the development of therapeutics and materials. But it’s been a hard slog, because protein synthesis is a crucial cellular function that cannot easily be changed.</p><p>“It’s a super-exciting field,” says Jason Chin, a synthetic and chemical biologist at the MRC Laboratory of Molecular Biology in Cambridge, UK. “Even from the beginning, I think it was clear that we were doing something really special.”</p><h2><b>From </b><b><i>vitro</i></b><b> to </b><b><i>vivo</i></b></h2><p>To understand why, we have to consider how proteins are made (see ‘Protein hacking’).</p><p>During transcription, DNA instructions are copied into RNA, which is then translated into proteins in a molecular machine called a ribosome. Each triplet of bases in the RNA message (adenine, cytosine, guanine and uracil) represents a word, or codon, of the genetic code. There are 64 words in total — 61 that encode for amino acids and 3 that signal the ribosome to stop.</p><figure>
 <picture>
  <source type="image/webp" srcset="//media.nature.com/lw767/magazine-assets/d41586-023-01980-4/d41586-023-01980-4_25493392.png?as=webp 767w, //media.nature.com/lw319/magazine-assets/d41586-023-01980-4/d41586-023-01980-4_25493392.png?as=webp 319w" sizes="(max-width: 319px) 319px, (min-width: 1023px) 100vw,  767px"/>
  <img alt="Protein hacking. Graphic showing how a stop codon is repurposed to insert a new amino acid into a peptide chain." loading="lazy" src="https://media.nature.com/lw767/magazine-assets/d41586-023-01980-4/d41586-023-01980-4_25493392.png"/>
  <figcaption>
   
  </figcaption>
 </picture>
</figure><p>Amino acids are brought to the ribosome by transfer RNAs (tRNAs), each of which corresponds to a complementary codon. Enzymes known as aminoacyl-tRNA synthetases couple the tRNA to its cognate amino acid.</p><p>Making this system accept unnatural, or ‘non-canonical’, amino acids in a test tube required several key tweaks. First, Schultz’s team devised a way to chemically attach unnatural amino acids to a tRNA that could recognize one of the three stop codons. They then introduced a stop codon into the gene for the penicillin-resistance protein, β-lactamase. Using these modifications, the researchers explored how different non-canonical protein variants affected enzyme activity<sup><a href="#ref-CR1" data-track="click" data-action="anchor-link" data-track-label="go to reference" data-track-category="references">1</a></sup>.</p><p>But that was in a test tube. To incorporate non-canonical amino acids into living cells, the team had to supplement the translational apparatus without altering it. They did so by identifying and modifying tRNAs and aminoacyl-tRNA synthetases that were bio-orthogonal to the host cell — that is, unable to recognize (and be recognized by) other parts of the cell’s translational machinery.</p><article data-label="Related">
  <a href="https://www.nature.com/articles/547477a" data-track="click" data-track-label="recommended article"><img alt="" src="https://media.nature.com/w400/magazine-assets/d41586-023-01980-4/d41586-023-01980-4_16251936.jpg"/><p>How to hack the genome</p></a>
 </article><p>“People thought that would be very, very difficult to do,” says Schultz, who is now the president and chief executive of Scripps Research in La Jolla, California. “Luckily, we were naive enough to think that we could do it.” (Schultz has founded several companies, including Ambrx based in La Jolla, which develops biotherapeutics using non-canonical amino acids. He is currently a member of Ambrx’s scientific advisory board.)</p><p>Eventually, Schultz’s team settled on a tRNA-synthetase pair from an archaeon, <i>Methanococcus jannaschii</i>, which did not readily recognize the tRNAs or the synthetases from the <i>Escherichia coli</i> bacteria they planned to modify. The team optimized the molecules to be selective to the new amino acid. In 2001, the researchers engineered this system into <i>E. coli</i>, enabling the cells to incorporate non-standard amino acids into proteins using its own translational machinery<sup><a href="#ref-CR2" data-track="click" data-action="anchor-link" data-track-label="go to reference" data-track-category="references">2</a></sup>.</p><p>“That was probably the beginning of the expansion of the genetic code <i>in</i> <i>vivo</i>,” says Chin, who was formerly a postdoc in Schultz’s laboratory.</p><h2><b>A growing toolkit</b></h2><p>Schultz’s team and others have used this approach to genetically encode more than 200 non-standard amino acids into proteins, providing a powerful tool with which to study protein structure and function. Scientists could, for instance, introduce fluorescent markers or other labels into proteins, or conduct photocaging experiments, in which proteins are rendered inactive (‘caged’) by chemical groups that can be removed with light.</p><p>Scientists have also moved beyond <i>E</i>. <i>coli</i> to hack the genetic codes of nematode worms, fruit flies, plants and even mice. Chin, along with Michael Hastings, a neuroscientist at the University of Cambridge, and their colleagues, for instance, hacked the tRNA and tRNA synthetase for the amino acid pyrrolysine to incorporate an amino acid called alkyne lysine <i>N</i><sup>6</sup>-[(2-propynyloxy)carbonyl]-<span>l</span>-lysine (AlkK) into proteins. They used it as a switch to turn genes on and off in mouse brain cells. The researchers deleted a key protein involved in regulating circadian rhythms — daily cycles controlled by the body’s internal clock — and introduced a new version of the gene into the cells that could be translated only in the presence of AlkK. Because the mice couldn’t make that amino acid on their own, the researchers could control their circadian rhythms by adding or removing the amino acids from the rodents’ drinking water<sup><a href="#ref-CR3" data-track="click" data-action="anchor-link" data-track-label="go to reference" data-track-category="references">3</a></sup>.</p><article data-label="Related">
  <a href="https://www.nature.com/articles/d41586-023-00702-0" data-track="click" data-track-label="recommended article"><img alt="" src="https://media.nature.com/w400/magazine-assets/d41586-023-01980-4/d41586-023-01980-4_24678558.jpg"/><p>Synthetic bacterial genome upgraded for viral defence and biocontainment</p></a>
 </article><p>For his part, Schultz has been using the expanded genetic code to ask what organisms would look like if they had more than 20 amino acids to play with. In one study, his team created a library of variants of <i>E. coli</i> homoserine O-succinyltransferase, an enzyme involved in the biosynthesis of amino acid methionine that is particularly sensitive to temperature, in which each codon outside the enzyme’s catalytic centre was replaced with the non-canonical amino acid, (<i>p</i>-benzoylphenyl)alanine (pBzF). The team discovered a variant that was stable at up to 21 °C above the typical range — a feat that they attributed to the formation of a chemical bond between pBzF and another amino acid<sup><a href="#ref-CR4" data-track="click" data-action="anchor-link" data-track-label="go to reference" data-track-category="references">4</a></sup>.</p><p>Chin, Schultz and other researchers have also begun to apply this technology to the development of therapeutics. Lei Wang, a chemical biologist at the University of California, San Francisco, for instance, is developing protein-based drugs that can form covalent bonds with other biomolecules.</p><p>Proteins typically interact with other molecules through relatively weak, non-covalent interactions, but covalent bonding could enhance their potency, says Wang. In 2020, his team incorporated the non-canonical amino acid fluorosulfate-<span>l</span>-tyrosine (FSY) into PD-1, an immune checkpoint protein that helps to rein in the body’s immune response, to create an anti-tumour drug. Typically, the interaction between PD-1 on T cells, and PD-L1 on tumour cells, dampens the immune response, allowing the tumour to escape immune surveillance. When Wang’s team injected the FSY-containing PD-1 into mice engrafted with human cancer cells, the protein formed an irreversible covalent bond with PD-L1, causing the tumours to shrink<sup><a href="#ref-CR5" data-track="click" data-action="anchor-link" data-track-label="go to reference" data-track-category="references">5</a></sup>. Wang is a scientific adviser at the company Enlaza Therapeutics in La Jolla, which has developed this therapeutic strategy.</p><figure>
 <picture>
  <source type="image/webp" srcset="//media.nature.com/lw767/magazine-assets/d41586-023-01980-4/d41586-023-01980-4_25484704.jpg?as=webp 767w, //media.nature.com/lw319/magazine-assets/d41586-023-01980-4/d41586-023-01980-4_25484704.jpg?as=webp 319w" sizes="(max-width: 319px) 319px, (min-width: 1023px) 100vw,  767px"/>
  <img alt="A rendering of a crystal structure of human PD-1/PD-L1 complex in pink and blue colours" loading="lazy" src="https://media.nature.com/lw767/magazine-assets/d41586-023-01980-4/d41586-023-01980-4_25484704.jpg"/>
  <figcaption>
   <p><span>PD-1 (left), an immune checkpoint protein, bound to its ligand PD-L1.</span><span>Credit: Q. Li <i>et al. Cell</i> <b>182</b>, 85–97 (2020)</span></p>
  </figcaption>
 </picture>
</figure><p>Meanwhile, in Beijing, Tao Liu, a chemical and synthetic biologist at Peking University, and his team have been applying genetic expansion to cell and gene therapies. In a 2021 study, Liu and his team reported engineering cells that, in the presence of the synthetic amino acid <i>O</i>-methyl tyrosine, express insulin<sup><a href="#ref-CR6" data-track="click" data-action="anchor-link" data-track-label="go to reference" data-track-category="references">6</a></sup>. When implanted into diabetic mice, the researchers could control the animal’s blood glucose levels by controlling how much <i>O</i>-methyl tyrosine they dispensed in the animals’ food.</p><h2><b>Expanding applications</b></h2><p>Beyond encoding new amino acids, one application of an extended genetic code is genetic isolation. With 61 codons for 20 amino acids, the genetic code is redundant, meaning that multiple codons encode the same amino acid. It is also nearly universal. By replacing all instances of a given codon for a synonymous one and removing the machinery that uses the original codon, researchers can render the cell effectively immune to foreign DNA — including pathogens.</p><p>In a 2022 study Chin and his group demonstrated this concept; they created an <i>E. coli</i> mutant in which two of its six serine codons were reassigned to encode for other amino acids — then they deleted the tRNAs that recognized the original serine codons. They further tweaked the system to ensure that viruses couldn’t use their own tRNAs, if they had them. The resulting cells were resistant to horizontal gene transfer from other bacteria, as well as to viral infection<sup><a href="#ref-CR7" data-track="click" data-action="anchor-link" data-track-label="go to reference" data-track-category="references">7</a></sup>. Another group, led by synthetic biologist George Church at Harvard University in Cambridge, Massachusetts, applied a similar ‘genetic firewall’ approach to create phage-resistant bacteria in a paper published earlier this year<sup><a href="#ref-CR8" data-track="click" data-action="anchor-link" data-track-label="go to reference" data-track-category="references">8</a></sup>.</p><p>Researchers can also use a modified genetic code to create polymers. In 2021, Chin’s team hacked the genetic code to synthesize short polymers, and even an artificial circular structure called a macrocycle, in <i>E. coli</i><sup><a href="#ref-CR9" data-track="click" data-action="anchor-link" data-track-label="go to reference" data-track-category="references">9</a></sup>. Now, Chin hopes to push this technology further to create cell factories that can churn out entirely new polymers, such as plastics. Like proteins, plastics are composed of long strings of monomers. But, although the genetic code dictates the sequence of amino acids in proteins, no equivalent system exists for artificial polymers.</p><p>“If you could encode the expanded set of building blocks in the same way that we can proteins, then we could turn cells into living factories for the encoded synthesis of polymers for everything from new drugs to materials,” says Chin. (Chin founded a company, Constructive Bio based in Saffron Walden, UK, to advance this goal.)</p><h2><b>Pushing boundaries</b></h2><p>Besides codon reassignment, researchers can also increase the number of available protein building blocks by expanding the nucleic acid alphabet from four bases to six, thus increasing the number of possible triplet codons to 216. In 2014, a team led by biochemist Floyd Romesberg, who was then at Scripps Research, reported creating a bacterial strain with a six-base genetic alphabet that could successfully replicate<sup><a href="#ref-CR10" data-track="click" data-action="anchor-link" data-track-label="go to reference" data-track-category="references">10</a></sup>. The group subsequently demonstrated that these cells could use their expanded DNA to produce proteins containing non-canonical amino acids<sup><a href="#ref-CR11" data-track="click" data-action="anchor-link" data-track-label="go to reference" data-track-category="references">11</a></sup>.</p><article data-label="Related">
  <a href="https://www.nature.com/collections/fxvqrpnlcq" data-track="click" data-track-label="recommended article"><img alt="" src="https://media.nature.com/w400/magazine-assets/d41586-023-01980-4/d41586-023-01980-4_16573690.jpg"/><p>NatureTech hub</p></a>
 </article><p>Another approach is to expand the length of a codon from three bases to four, thus increasing the number of possible codons to 256. This requires modifying multiple pieces of the translation machinery, including the ribosome. Chin and his team have used this strategy to incorporate four non-canonical amino acids into <i>E. coli</i> on a transcript that also contains conventional three-base codons<sup><a href="#ref-CR12" data-track="click" data-action="anchor-link" data-track-label="go to reference" data-track-category="references">12</a></sup>. Other researchers have been exploring whether it might even be possible to create an all-quadruplet genetic code.</p><p>Some researchers are attempting more extreme alterations. These include backbone modifications — creating so-called β- or γ-amino-acids (as opposed to α-amino-acids, which are found in nature), or amino acids that are reverse mirror images of standard amino acids. Polymers built from either type of building block would probably be highly stable, because typical protein-degrading machinery would not be able to recognize them.</p><p>But existing translational machinery isn’t built to accept these exotic amino acids, including the aminoacyl-tRNA synthetases that attach amino acids onto tRNA. Hiroaki Suga, a chemical biologist at the University of Tokyo, developed one workaround in 2006 with the ‘flexizyme’, an RNA-based catalyst that can perform the job of protein synthetases: linking amino acids to tRNAs<sup><a href="#ref-CR13" data-track="click" data-action="anchor-link" data-track-label="go to reference" data-track-category="references">13</a></sup>.</p><p>Suga focuses on synthesizing polymers <i>in</i> <i>vitro</i>, because it provides the greatest flexibility in terms of the modifications he can make. Using the flexizyme, his team combined up to 11 non-canonical amino acids with 12 standard amino acids in a single macrocycle<sup><a href="#ref-CR14" data-track="click" data-action="anchor-link" data-track-label="go to reference" data-track-category="references">14</a></sup>. Suga co-founded the biotechnology company PeptiDream, based in Kawasaki, Japan, to develop drugs using the same technology.</p><p>Again, however, that work is <i>in vitro</i>; applying backbone- or configuration-modified amino acids to proteins in cells remain a challenge. Although there are a few instances in which scientists have been able to incorporate these more exotic amino acids, further steps are necessary to make the process more efficient. As well as finding synthetases that could work with these amino acids <i>in</i> <i>vivo</i>, in many cases scientists would have to design a ribosome that can process these new amino acids — while still carrying out its usual duties.</p><p>In two recent studies Alanna Schepartz, a chemical and synthetic biologist at the University of California, Berkeley, and her team report steps toward solving these problems. In one of the papers, they describe a synthetase from the archaeon <i>Methanomethylophilus alvus</i> that can accept non-α-amino-acids and is bio-orthogonal to <i>E. coli.</i> In the other study, the team reports a computational technique for screening unique backbone monomers that the <i>E. coli</i> ribosome can process efficiently. The researchers say this will help to identify non-α-amino-acids that are most likely to be compatible with an existing ribosome<sup><a href="#ref-CR15" data-track="click" data-action="anchor-link" data-track-label="go to reference" data-track-category="references">15</a></sup><sup>,</sup><sup><a href="#ref-CR16" data-track="click" data-action="anchor-link" data-track-label="go to reference" data-track-category="references">16</a></sup>.</p><p>Other groups are working to develop a ribosome that can produce proteins with exotic amino acids from scratch. Last October, researchers at Westlake University in Hangzhou, China, reported a mirror-image RNA polymerase that could synthesize all the RNA molecules that are needed to produce a mirror-image ribosome<sup><a href="#ref-CR17" data-track="click" data-action="anchor-link" data-track-label="go to reference" data-track-category="references">17</a></sup>. Although many more steps are needed to make a mirror-image ribosome a reality, creating such a molecule would be an important step towards using the translational machinery to make mirror-image proteins.</p><p>Other researchers are working to reprogram the ribosome to create polymers with carbon–carbon bonds, as opposed to the nitrogen–carbon amide bonds they typically forge to link amino acids.</p><p>Should these strategies come to fruition, they would endow scientists with tremendous synthetic power over new polymers. Nobody knows what properties can emerge from a synthetic polymer that’s created with the same length and level of sequence definition as a protein, “because no such molecules have ever been made”, Schepartz says. But she and others hope to get there soon. “It’s a very exciting time for the field.”</p>
                </div><div id="references">
                <section aria-labelledby="Bib1"><div id="Bib1-section"><h2 id="Bib1">References</h2><div id="Bib1-content"><div data-container-section="references"><ol data-track-component="outbound reference"><li data-counter="1."><p id="ref-CR1">Noren, C. J. Anthony-Cahill, S. J., Griffith, M. C. &amp; Schultz, P. G. <i>Science</i> <b>244</b>, 182–188 (1989).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1126/science.2649980" data-track-action="article reference" href="https://doi.org/10.1126%2Fscience.2649980" aria-label="Article reference 1" data-doi="10.1126/science.2649980">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=2649980" aria-label="PubMed reference 1">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 1" href="http://scholar.google.com/scholar_lookup?&amp;title=&amp;journal=Science&amp;doi=10.1126%2Fscience.2649980&amp;volume=244&amp;pages=182-188&amp;publication_year=1989&amp;author=Noren%2CC.%20J.&amp;author=Anthony-Cahill%2CS.%20J.&amp;author=Griffith%2CM.%20C.&amp;author=Schultz%2CP.%20G">
                    Google Scholar</a> 
                </p></li><li data-counter="2."><p id="ref-CR2">Wang, L. Brock, A., Herberich, B. &amp; Schultz, P. G. <i>Science</i> <b>292</b>, 498–500 (2001).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1126/science.1060077" data-track-action="article reference" href="https://doi.org/10.1126%2Fscience.1060077" aria-label="Article reference 2" data-doi="10.1126/science.1060077">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=11313494" aria-label="PubMed reference 2">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 2" href="http://scholar.google.com/scholar_lookup?&amp;title=&amp;journal=Science&amp;doi=10.1126%2Fscience.1060077&amp;volume=292&amp;pages=498-500&amp;publication_year=2001&amp;author=Wang%2CL.&amp;author=Brock%2CA.&amp;author=Herberich%2CB.&amp;author=Schultz%2CP.%20G">
                    Google Scholar</a> 
                </p></li><li data-counter="3."><p id="ref-CR3">Maywood, E. S. <i>et al.</i> <i>Proc. Natl Acad. Sci. USA</i> <b>115</b>, E12388–E12397 (2018).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1073/pnas.1811438115" data-track-action="article reference" href="https://doi.org/10.1073%2Fpnas.1811438115" aria-label="Article reference 3" data-doi="10.1073/pnas.1811438115">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30487216" aria-label="PubMed reference 3">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 3" href="http://scholar.google.com/scholar_lookup?&amp;title=&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;doi=10.1073%2Fpnas.1811438115&amp;volume=115&amp;pages=E12388-E12397&amp;publication_year=2018&amp;author=Maywood%2CE.%20S.">
                    Google Scholar</a> 
                </p></li><li data-counter="4."><p id="ref-CR4">Li, J. C., Liu, T., Wang, Y., Mehta, A. P. &amp; Schultz, P. G. <i>J. Am. Chem. Soc.</i> <b>140</b>, 15997–16000 (2018).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1021/jacs.8b07157" data-track-action="article reference" href="https://doi.org/10.1021%2Fjacs.8b07157" aria-label="Article reference 4" data-doi="10.1021/jacs.8b07157">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30433771" aria-label="PubMed reference 4">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 4" href="http://scholar.google.com/scholar_lookup?&amp;title=&amp;journal=J.%20Am.%20Chem.%20Soc.&amp;doi=10.1021%2Fjacs.8b07157&amp;volume=140&amp;pages=15997-16000&amp;publication_year=2018&amp;author=Li%2CJ.%20C.&amp;author=Liu%2CT.&amp;author=Wang%2CY.&amp;author=Mehta%2CA.%20P.&amp;author=Schultz%2CP.%20G.">
                    Google Scholar</a> 
                </p></li><li data-counter="5."><p id="ref-CR5">Li, Q. <i>et al.</i> <i>Cell</i> <b>182</b>, 85–97 (2020).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.cell.2020.05.028" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.cell.2020.05.028" aria-label="Article reference 5" data-doi="10.1016/j.cell.2020.05.028">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32579975" aria-label="PubMed reference 5">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 5" href="http://scholar.google.com/scholar_lookup?&amp;title=&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2020.05.028&amp;volume=182&amp;pages=85-97&amp;publication_year=2020&amp;author=Li%2CQ.">
                    Google Scholar</a> 
                </p></li><li data-counter="6."><p id="ref-CR6">Chen, C. <i>et al.</i> <i>Nature Chem. Biol.</i> <b>18</b>, 47–55 (2022).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/s41589-021-00899-z" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41589-021-00899-z" aria-label="Article reference 6" data-doi="10.1038/s41589-021-00899-z">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34782743" aria-label="PubMed reference 6">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 6" href="http://scholar.google.com/scholar_lookup?&amp;title=&amp;journal=Nature%20Chem.%20Biol.&amp;doi=10.1038%2Fs41589-021-00899-z&amp;volume=18&amp;pages=47-55&amp;publication_year=2022&amp;author=Chen%2CC.">
                    Google Scholar</a> 
                </p></li><li data-counter="7."><p id="ref-CR7">Zürcher, J. F. <i>et al.</i> <i>Science</i> <b>378</b>, 516–523 (2022).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1126/science.add8943" data-track-action="article reference" href="https://doi.org/10.1126%2Fscience.add8943" aria-label="Article reference 7" data-doi="10.1126/science.add8943">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=36264827" aria-label="PubMed reference 7">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 7" href="http://scholar.google.com/scholar_lookup?&amp;title=&amp;journal=Science&amp;doi=10.1126%2Fscience.add8943&amp;volume=378&amp;pages=516-523&amp;publication_year=2022&amp;author=Z%C3%BCrcher%2CJ.%20F.">
                    Google Scholar</a> 
                </p></li><li data-counter="8."><p id="ref-CR8">Nyerges, A. <i>et al.</i> <i>Nature</i> <b>615</b>, 720–727 (2023).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/s41586-023-05824-z" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41586-023-05824-z" aria-label="Article reference 8" data-doi="10.1038/s41586-023-05824-z">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=36922599" aria-label="PubMed reference 8">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 8" href="http://scholar.google.com/scholar_lookup?&amp;title=&amp;journal=Nature&amp;doi=10.1038%2Fs41586-023-05824-z&amp;volume=615&amp;pages=720-727&amp;publication_year=2023&amp;author=Nyerges%2CA.">
                    Google Scholar</a> 
                </p></li><li data-counter="9."><p id="ref-CR9">Robertson, W. E. <i>et al.</i> <i>Science</i> <b>372</b>, 1057–1062 (2021).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1126/science.abg3029" data-track-action="article reference" href="https://doi.org/10.1126%2Fscience.abg3029" aria-label="Article reference 9" data-doi="10.1126/science.abg3029">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34083482" aria-label="PubMed reference 9">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 9" href="http://scholar.google.com/scholar_lookup?&amp;title=&amp;journal=Science&amp;doi=10.1126%2Fscience.abg3029&amp;volume=372&amp;pages=1057-1062&amp;publication_year=2021&amp;author=Robertson%2CW.%20E.">
                    Google Scholar</a> 
                </p></li><li data-counter="10."><p id="ref-CR10">Malyshev, D. A. <i>et al.</i> <i>Nature</i> <b>509</b>, 385–388 (2014).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/nature13314" data-track-action="article reference" href="https://doi.org/10.1038%2Fnature13314" aria-label="Article reference 10" data-doi="10.1038/nature13314">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24805238" aria-label="PubMed reference 10">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 10" href="http://scholar.google.com/scholar_lookup?&amp;title=&amp;journal=Nature&amp;doi=10.1038%2Fnature13314&amp;volume=509&amp;pages=385-388&amp;publication_year=2014&amp;author=Malyshev%2CD.%20A.">
                    Google Scholar</a> 
                </p></li><li data-counter="11."><p id="ref-CR11">Zhang, Y. <i>et al.</i> <i>Nature</i> <b>551</b>, 644–647 (2017).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/nature24659" data-track-action="article reference" href="https://doi.org/10.1038%2Fnature24659" aria-label="Article reference 11" data-doi="10.1038/nature24659">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29189780" aria-label="PubMed reference 11">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 11" href="http://scholar.google.com/scholar_lookup?&amp;title=&amp;journal=Nature&amp;doi=10.1038%2Fnature24659&amp;volume=551&amp;pages=644-647&amp;publication_year=2017&amp;author=Zhang%2CY.">
                    Google Scholar</a> 
                </p></li><li data-counter="12."><p id="ref-CR12">Dunkelmann, D. L., Oehm, S. B., Beattie, A. T. &amp; Chin, J. W. <i>Nature Chem.</i> <b>13</b>, 1110–1117 (2021).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/s41557-021-00764-5" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41557-021-00764-5" aria-label="Article reference 12" data-doi="10.1038/s41557-021-00764-5">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34426682" aria-label="PubMed reference 12">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 12" href="http://scholar.google.com/scholar_lookup?&amp;title=&amp;journal=Nature%20Chem.&amp;doi=10.1038%2Fs41557-021-00764-5&amp;volume=13&amp;pages=1110-1117&amp;publication_year=2021&amp;author=Dunkelmann%2CD.%20L.&amp;author=Oehm%2CS.%20B.&amp;author=Beattie%2CA.%20T.&amp;author=Chin%2CJ.%20W.">
                    Google Scholar</a> 
                </p></li><li data-counter="13."><p id="ref-CR13">Murakami, H., Ohta, A., Ashigai, H. &amp; Suga, H. <i>Nature Methods</i> <b>3</b>, 357–359 (2006).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/nmeth877" data-track-action="article reference" href="https://doi.org/10.1038%2Fnmeth877" aria-label="Article reference 13" data-doi="10.1038/nmeth877">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16628205" aria-label="PubMed reference 13">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 13" href="http://scholar.google.com/scholar_lookup?&amp;title=&amp;journal=Nature%20Methods&amp;doi=10.1038%2Fnmeth877&amp;volume=3&amp;pages=357-359&amp;publication_year=2006&amp;author=Murakami%2CH.&amp;author=Ohta%2CA.&amp;author=Ashigai%2CH.&amp;author=Suga%2CH.">
                    Google Scholar</a> 
                </p></li><li data-counter="14."><p id="ref-CR14">Peacock, H. &amp; Suga, H. <i>Trends Pharma. Sci.</i> <b>42</b>, 385–397 (2021).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.tips.2021.02.004" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.tips.2021.02.004" aria-label="Article reference 14" data-doi="10.1016/j.tips.2021.02.004">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 14" href="http://scholar.google.com/scholar_lookup?&amp;title=&amp;journal=Trends%20Pharma.%20Sci.&amp;doi=10.1016%2Fj.tips.2021.02.004&amp;volume=42&amp;pages=385-397&amp;publication_year=2021&amp;author=Peacock%2CH.&amp;author=Suga%2CH.">
                    Google Scholar</a> 
                </p></li><li data-counter="15."><p id="ref-CR15">Fricke, R. <i>et al.</i> <i>Nature Chem</i>. https://doi.org/10.1038/s41557-023-01224-y (2023).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/s41557-023-01224-y" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41557-023-01224-y" aria-label="Article reference 15" data-doi="10.1038/s41557-023-01224-y">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 15" href="http://scholar.google.com/scholar_lookup?&amp;title=&amp;journal=Nature%20Chem&amp;doi=10.1038%2Fs41557-023-01224-y&amp;publication_year=2023&amp;author=Fricke%2CR.">
                    Google Scholar</a> 
                </p></li><li data-counter="16."><p id="ref-CR16">Watson, Z. L. <i>et al.</i> <i>Nature Chem</i>. https://doi.org/10.1038/s41557-023-01226-w (2023).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/s41557-023-01226-w" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41557-023-01226-w" aria-label="Article reference 16" data-doi="10.1038/s41557-023-01226-w">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 16" href="http://scholar.google.com/scholar_lookup?&amp;title=&amp;journal=Nature%20Chem&amp;doi=10.1038%2Fs41557-023-01226-w&amp;publication_year=2023&amp;author=Watson%2CZ.%20L.">
                    Google Scholar</a> 
                </p></li><li data-counter="17."><p id="ref-CR17">Xu, Y. &amp; Zhu, T. F. <i>Science</i> <b>378</b>, 405–412 (2022).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1126/science.abm0646" data-track-action="article reference" href="https://doi.org/10.1126%2Fscience.abm0646" aria-label="Article reference 17" data-doi="10.1126/science.abm0646">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=36302022" aria-label="PubMed reference 17">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 17" href="http://scholar.google.com/scholar_lookup?&amp;title=&amp;journal=Science&amp;doi=10.1126%2Fscience.abm0646&amp;volume=378&amp;pages=405-412&amp;publication_year=2022&amp;author=Xu%2CY.&amp;author=Zhu%2CT.%20F.">
                    Google Scholar</a> 
                </p></li></ol><p><a data-track="click" data-track-action="download citation references" data-track-label="link" rel="nofollow" href="https://citation-needed.springer.com/v2/references/10.1038/d41586-023-01980-4?format=refman&amp;flavour=references">Download references<svg width="16" height="16" focusable="false" role="img" aria-hidden="true"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-download"></use></svg></a></p></div></div></div></section>
            </div></div>
  </body>
</html>
