<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <title>James Routley | Feed</title>
    <link
      rel="stylesheet"
      type="text/css"
      href="../styles.css"
      media="screen"
    />
  </head>
  <body>
    <a href="/index.html">Back</a>
    <a href="https://www.nature.com/articles/s41564-023-01426-7">Original</a>
    <h1>Human microbiome myths and misconceptions</h1>
    
    <div id="readability-page-1" class="page"><div>
                <section data-title="Main"><div id="Sec1-section"><h2 id="Sec1">Main</h2><p>Human microbiome research has undergone rapid growth over the past two decades and thousands of research papers on this topic are now published every year. Huge sums of money have been spent investigating the human microbiome as a cause of, or potential therapeutic solution to, a wide range of diseases, including inflammatory bowel disease and cardiometabolic conditions. Although truly exciting, the increasing focus on microbiome research has unfortunately also brought with it hype and entrenched certain misconceptions. As a result, many unsupported, or undersupported, statements have become ‘fact’ by virtue of constant repetition. Some are more widespread than others and some are relatively trivial, but, cumulatively, they highlight that misinformation is pervasive in the human microbiome literature. Given the potential importance of human microbiomes for health, it is crucial that claims are based on evidence. In this Perspective, we shine a light on persistent or emerging microbiome myths and misconceptions, outlining factual inaccuracies. We begin with relatively minor, but illustrative, points and build towards issues with greater potential impacts. We have purposefully tried to avoid unnecessary finger pointing at original sources of erroneous information, and instead hope that our insights and critical assessment are helpful to the field.</p></div></section><section data-title="“Microbiome research is a new field”"><div id="Sec2-section"><h2 id="Sec2">“Microbiome research is a new field”</h2><p>The pace of human microbiome research has greatly accelerated over the past 15 years, but the field is not in its infancy. To state so does a disservice to the excellent research that preceded the advent of high-throughput DNA-sequencing approaches. Indeed, there has been a rich history of research into human-associated microorganisms since at least the late nineteenth century. <i>Escherichia coli</i> was first isolated in 1885<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 1" title="Hacker, J. &amp; Blum-Oehler, G. In appreciation of Theodor Escherich. Nat. Rev. Microbiol. 5, 902 (2007)." href="https://www.nature.com/articles/s41564-023-01426-7#ref-CR1" id="ref-link-section-d187515113e405">1</a></sup>, bifidobacteria were described in 1899<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 2" title="Cruickshank, R. Bacillus bifidus: its characters and isolation from the intestine of infants. J. Hyg. 24, 241–254 (1925)." href="https://www.nature.com/articles/s41564-023-01426-7#ref-CR2" id="ref-link-section-d187515113e409">2</a></sup> and Metchnikoff speculated on the importance of beneficial gut microorganisms in the early 1900s<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 3" title="Metchnikoff, E. The Prolongation of Life: Optimistic Studies (William Heinemann, G. P. Putnam’s Sons, 1907)." href="https://www.nature.com/articles/s41564-023-01426-7#ref-CR3" id="ref-link-section-d187515113e413">3</a></sup>. Similarly, concepts such as the gut–brain axis have been researched for centuries<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 4" title="Miller, I. The gut–brain axis: historical reflections. Microb. Ecol. Health Dis. 29, 1542921 (2018)." href="https://www.nature.com/articles/s41564-023-01426-7#ref-CR4" id="ref-link-section-d187515113e417">4</a></sup> and health impacts of key microbiome-associated metabolites, such as short-chain fatty acids, were first reported more than 40 years ago<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 5" title="Roediger, W. E. W. The colonic epithelium in ulcerative colitis: an energy-deficiency disease? Lancet 316, 712–715 (1980)." href="https://www.nature.com/articles/s41564-023-01426-7#ref-CR5" id="ref-link-section-d187515113e422">5</a></sup>.</p></div></section><section data-title="“Joshua Lederberg coined the term ‘microbiome’”"><div id="Sec3-section"><h2 id="Sec3">“Joshua Lederberg coined the term ‘microbiome’”</h2><p>Although Nobel laureate Joshua Lederberg had many notable achievements in his career, he did not invent the word ‘microbiome’. This oft-repeated claim has been thoroughly refuted elsewhere, with evidence provided that the word was used in its modern context more than a decade before Joshua Lederberg first used it in 2001<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 6" title="Prescott, S. L. History of medicine: origin of the term microbiome and why it matters. Hum. Microb. J. 4, 24–25 (2017)." href="https://www.nature.com/articles/s41564-023-01426-7#ref-CR6" id="ref-link-section-d187515113e434">6</a></sup>. Although relatively trivial, this myth serves as an example of how easy it is for falsehoods to become embedded in the human microbiome literature.</p></div></section><section data-title="“There are 1012 bacterial cells per gram of human faeces”"><div id="Sec4-section"><h2 id="Sec4">“There are 10<sup>12</sup> bacterial cells per gram of human faeces”</h2><p>This figure is commonly mentioned in the microbiome literature, but its source has been difficult to ascertain. It may, however, have originated from dry-weight rather than wet-weight faecal cell counts. Regardless, it is incorrect. The real figure, as ascertained using various methods such as direct cell counts, fluorescence in situ hybridization, flow cytometry and quantitative polymerase chain reaction (qPCR), is typically between 10<sup>10</sup> and 10<sup>11</sup> microbial cells per wet-weight gram of faeces<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Stephen, A. M. &amp; Cummings, J. H. The microbial contribution to human faecal mass. J. Med. Microbiol. 13, 45–56 (1980)." href="#ref-CR7" id="ref-link-section-d187515113e453">7</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Hoyles, L. &amp; McCartney, A. L. What do we mean when we refer to Bacteroidetes populations in the human gastrointestinal microbiota? FEMS Microbiol. Lett. 299, 175–183 (2009)." href="#ref-CR8" id="ref-link-section-d187515113e453_1">8</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 9" title="Vandeputte, D. et al. Quantitative microbiome profiling links gut community variation to microbial load. Nature 551, 507–511 (2017)." href="https://www.nature.com/articles/s41564-023-01426-7#ref-CR9" id="ref-link-section-d187515113e456">9</a></sup>.</p></div></section><section data-title="“The human microbiota weighs 1 to 2 kg”"><div id="Sec5-section"><h2 id="Sec5">“The human microbiota weighs 1 to 2 kg”</h2><p>Although this is mentioned many times in the literature, it is often given without citation and we were unable to find an original source for this claim. Nonetheless, it is unlikely to be true in most cases. The majority of the human microbiota resides in the colon, and these microorganisms typically account for less than half of the weight of faecal solids<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 10" title="Sender, R., Fuchs, S. &amp; Milo, R. Revised estimates for the number of human and bacteria cells in the body. PLoS Biol. 14, e1002533 (2016)." href="https://www.nature.com/articles/s41564-023-01426-7#ref-CR10" id="ref-link-section-d187515113e468">10</a></sup>. The average human stool weighs less than 200 g (wet weight)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 11" title="Cummings, J. H., Bingham, S. A., Heaton, K. W. &amp; Eastwood, M. A. Fecal weight, colon cancer risk, and dietary intake of nonstarch polysaccharides (dietary fiber). Gastroenterology 103, 1783–1789 (1992)." href="https://www.nature.com/articles/s41564-023-01426-7#ref-CR11" id="ref-link-section-d187515113e472">11</a></sup>, with total colonic contents ranging from 83 to 421 g in a small study of sudden-death victims<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 12" title="Cummings, J. H., Pomare, E. W., Branch, W. J., Naylor, C. P. &amp; Macfarlane, G. T. Short chain fatty acids in human large intestine, portal, hepatic and venous blood. Gut 28, 1221–1227 (1987)." href="https://www.nature.com/articles/s41564-023-01426-7#ref-CR12" id="ref-link-section-d187515113e476">12</a></sup>. Therefore, aside perhaps from rare cases of severely constipated individuals with extremely compacted faecal matter in their colons, the total weight of the human microbiota is much more likely to be less than 500 g, and perhaps even considerably lower than this in some cases.</p></div></section><section data-title="“The microbiota outnumbers human cells by 10:1”"><div id="Sec6-section"><h2 id="Sec6">“The microbiota outnumbers human cells by 10:1”</h2><p>This myth is perhaps one of the most pervasive in the human microbiome literature and is one that we have also repeated uncritically in the past (we, sadly, are not immune to mythology). Excellent work<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 10" title="Sender, R., Fuchs, S. &amp; Milo, R. Revised estimates for the number of human and bacteria cells in the body. PLoS Biol. 14, e1002533 (2016)." href="https://www.nature.com/articles/s41564-023-01426-7#ref-CR10" id="ref-link-section-d187515113e489">10</a></sup> has, however, shown that this myth seems to have originated from a ‘back of the envelope’ calculation in the 1970s. More detailed analyses indicate that the true figure, albeit still impressive, is probably closer to a ratio of 1:1. It should be noted that the ratio is likely to vary from person to person and is dependent on factors such as the host’s body size and the amount of faecal material they are carrying in their colon<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 13" title="Sender, R., Fuchs, S. &amp; Milo, R. Are we really vastly outnumbered? Revisiting the ratio of bacterial to host cells in humans. Cell 164, 337–340 (2016)." href="https://www.nature.com/articles/s41564-023-01426-7#ref-CR13" id="ref-link-section-d187515113e493">13</a></sup>. Current estimates are also largely based on observations from adult individuals living in urbanized high-income country settings. More comprehensive estimates will require study of individuals from lower income or rural settings, and also from across the life course.</p></div></section><section data-title="“The microbiota is inherited from the mother at birth”"><div id="Sec7-section"><h2 id="Sec7">“The microbiota is inherited from the mother at birth”</h2><div id="Sec7-content"><p>Although variants of this statement are more often found in popular science articles than the scientific literature, it is an example of how nuance is extremely important when describing the human microbiome. Although some microorganisms are directly transferred from mother to baby during birth<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 14" title="Ferretti, P. et al. Mother-to-infant microbial transmission from different body sites shapes the developing infant gut microbiome. Cell Host Microbe 24, 133–145.e5 (2018)." href="https://www.nature.com/articles/s41564-023-01426-7#ref-CR14" id="ref-link-section-d187515113e505">14</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 15" title="Valles-Colomer, M. et al. The person-to-person transmission landscape of the gut and oral microbiomes. Nature 614, 125–135 (2023)." href="https://www.nature.com/articles/s41564-023-01426-7#ref-CR15" id="ref-link-section-d187515113e508">15</a></sup>, proportionally few microbiota species are truly ‘heritable’ and persist through from birth to adulthood in the offspring<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 15" title="Valles-Colomer, M. et al. The person-to-person transmission landscape of the gut and oral microbiomes. Nature 614, 125–135 (2023)." href="https://www.nature.com/articles/s41564-023-01426-7#ref-CR15" id="ref-link-section-d187515113e512">15</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 16" title="Rothschild, D. et al. Environment dominates over host genetics in shaping human gut microbiota. Nature 555, 210–215 (2018)." href="https://www.nature.com/articles/s41564-023-01426-7#ref-CR16" id="ref-link-section-d187515113e515">16</a></sup>. Indeed, most of the expansion in gut microbiota diversity occurs after birth, during the first few years of life, and increases most dramatically after weaning<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 17" title="Yatsunenko, T. et al. Human gut microbiome viewed across age and geography. Nature 486, 222–227 (2012)." href="https://www.nature.com/articles/s41564-023-01426-7#ref-CR17" id="ref-link-section-d187515113e519">17</a></sup> (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41564-023-01426-7#Fig1">1</a>). Every adult ends up with a unique microbiota configuration, even identical twins that are raised in the same household<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 18" title="Goodrich, J. K. et al. Human genetics shape the gut microbiome. Cell 159, 789–799 (2014)." href="https://www.nature.com/articles/s41564-023-01426-7#ref-CR18" id="ref-link-section-d187515113e526">18</a></sup>. Therefore, although microbiota assembly is not yet fully understood, adult microbial communities seem to be predominantly shaped by prior stochastic environmental exposures, as well as multiple other factors such as diet, antibiotic therapy and host genetics, with direct ‘inheritance’ from the mother at birth playing a similarly lesser role.</p><div data-test="figure" data-container-section="figure" id="figure-1" data-title="Diversity of the human gut microbiota dramatically increases in the years after birth."><figure><figcaption><b id="Fig1" data-test="figure-caption-text">Fig. 1: Diversity of the human gut microbiota dramatically increases in the years after birth.</b></figcaption><div><div><a data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="https://www.nature.com/articles/s41564-023-01426-7/figures/1" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41564-023-01426-7/MediaObjects/41564_2023_1426_Fig1_HTML.png?as=webp"/><img aria-describedby="Fig1" src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41564-023-01426-7/MediaObjects/41564_2023_1426_Fig1_HTML.png" alt="figure 1" loading="lazy" width="685" height="391"/></picture></a></div><p>Diversity (as assessed using number of observed operational taxonomic units) dramatically increases during the first few years of life, particularly after weaning, before beginning to plateau in childhood. This pattern is observed across individuals living in different geographical locations: Malawi (red), Venezuela (green) and the United States (blue). Figure adapted with permission from ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 17" title="Yatsunenko, T. et al. Human gut microbiome viewed across age and geography. Nature 486, 222–227 (2012)." href="https://www.nature.com/articles/s41564-023-01426-7#ref-CR17" id="ref-link-section-d187515113e543">17</a></sup>, Springer Nature Ltd.</p></div></figure></div></div></div></section><section data-title="“Most diseases are characterized by a pathobiome”"><div id="Sec8-section"><h2 id="Sec8">“Most diseases are characterized by a pathobiome”</h2><div id="Sec8-content"><p>It has become increasingly common to read claims in the literature that most diseases are caused by a ‘pathobiome’. This is loosely defined as deleterious interactions between microbial communities and their host that lead to disease. This term is unfortunately overly simplistic and inherently flawed. Microorganisms and their metabolites are neither ‘good’ nor ‘bad’, they merely exist. Their impacts on us as hosts are heavily dependent on context. Microorganisms or metabolites that are deleterious in one context may cause no harm in another. As examples, <i>Clostridioides difficile</i> can be carried asymptomatically throughout life, and only cause problems in older age when the host is immunocompromised and treated with antibiotics<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 19" title="Schäffler, H. &amp; Breitrück, A. Clostridium difficile – from colonization to infection. Front. Microbiol. 9, 646 (2018)." href="https://www.nature.com/articles/s41564-023-01426-7#ref-CR19" id="ref-link-section-d187515113e566">19</a></sup>. Similarly, a strain of <i>E.</i> <i>coli</i> may be relatively harmless in the colon, but cause a urinary tract infection if it invades the urethra<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 20" title="Worby, C. J. et al. Longitudinal multi-omics analyses link gut microbiome dysbiosis with recurrent urinary tract infections in women. Nat. Microbiol. 7, 630–639 (2022)." href="https://www.nature.com/articles/s41564-023-01426-7#ref-CR20" id="ref-link-section-d187515113e576">20</a></sup>. As a result, the term pathobiome remains vague and lacking in the precision required to be truly useful in clinical practice.</p><p>It is true, however, that numerous human conditions have been shown to correlate with alterations in microbiota composition. This is sometimes referred to as ‘dysbiosis’, which is also a vague term with limited clinical applicability<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 21" title="Olesen, S. W. &amp; Alm, E. J. Dysbiosis is not an answer. Nat. Microbiol. 1, 16228 (2016)." href="https://www.nature.com/articles/s41564-023-01426-7#ref-CR21" id="ref-link-section-d187515113e583">21</a></sup>. It is very likely that this may contribute to disease progression in some conditions, including inflammatory bowel diseases<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 22" title="Ni, J., Wu, G. D., Albenberg, L. &amp; Tomov, V. T. Gut microbiota and IBD: causation or correlation? Nat. Rev. Gastroenterol. Hepatol. 14, 573–584 (2017)." href="https://www.nature.com/articles/s41564-023-01426-7#ref-CR22" id="ref-link-section-d187515113e587">22</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 23" title="Pammi, M. et al. Intestinal dysbiosis in preterm infants preceding necrotizing enterocolitis: a systematic review and meta-analysis. Microbiome 5, 31 (2017)." href="https://www.nature.com/articles/s41564-023-01426-7#ref-CR23" id="ref-link-section-d187515113e590">23</a></sup>, however, such alterations are rarely consistent and the microbiota is hugely variable between individuals, both in health and disease. This makes it extremely difficult to identify gut microbiota configurations with the required specificity and reproducibility for clinical practice<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 24" title="Damhorst, G. L., Adelman, M. W., Woodworth, M. H. &amp; Kraft, C. S. Current capabilities of gut microbiome-based diagnostics and the promise of clinical application. J. Infect. Dis. 223, S270–S275 (2021)." href="https://www.nature.com/articles/s41564-023-01426-7#ref-CR24" id="ref-link-section-d187515113e594">24</a></sup>. In addition, correlating gut microbiome changes with systemic markers or data is fraught with challenges. This often fails to account for confounders such as age, body mass index (BMI), sex and medication, factors such as microbial community interactions or for changes that occur as a result of immunological, metabolic or other functional changes in the host rather than being directly causal (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41564-023-01426-7#Fig2">2</a>). Attempts to define ‘tipping points’ at which changes in microbiome composition definitively influence disease progression have so far largely failed to generate a clear consensus due to a lack of consistency between different studies. It is, therefore, a leap that is not yet evidence based to conclude that a characteristic pathobiome has a role in ‘most’ diseases.</p><div data-test="figure" data-container-section="figure" id="figure-2" data-title="Difficulties of establishing causality from correlation-based microbiota studies."><figure><figcaption><b id="Fig2" data-test="figure-caption-text">Fig. 2: Difficulties of establishing causality from correlation-based microbiota studies.</b></figcaption><div><div><a data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="https://www.nature.com/articles/s41564-023-01426-7/figures/2" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41564-023-01426-7/MediaObjects/41564_2023_1426_Fig2_HTML.png?as=webp"/><img aria-describedby="Fig2" src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41564-023-01426-7/MediaObjects/41564_2023_1426_Fig2_HTML.png" alt="figure 2" loading="lazy" width="685" height="460"/></picture></a></div><p>Changes in faecal microbiota have been associated with a range of diseases in humans. Interestingly, despite the diverse nature of these conditions, and the organs they affect, there are some broadly common recurring microbiota features, such as reduced diversity and increases in facultative anaerobes like Enterobacteriaceae. One common theme amongst these conditions is that they often result in increased levels of inflammation, at local and systemic levels. Such inflammation can, in turn, deplete the gut microbiota (and consequently microbial gene diversity), and allow facultative anaerobes such as Enterobacteriaceae to proliferate. This directly impacts the metabolic output of the microbiota, and its interactions with the host. Additionally, there are other host factors that contribute to disease and gut microbiota composition, such as age, BMI and medication, as well as host metabolism and immune response. This makes it very difficult to distinguish cause from effect in correlation-based studies.</p></div></figure></div></div></div></section><section data-title="“The Firmicutes:Bacteroidetes ratio is altered in obesity”"><div id="Sec9-section"><h2 id="Sec9">“The Firmicutes:Bacteroidetes ratio is altered in obesity”</h2><div id="Sec9-content"><p>Related to the previous section, this commonly used but erroneous claim stems primarily from rodent-based research, and from findings in single, or under-powered, human studies. However, as with many other studies that report links between specific microbiota profiles and disease, reproducibility is poor. Indeed, there have now been at least three meta-analyses reporting that this finding is inconsistent between human studies, and that there are, in fact, no reproducible microbial taxonomic signatures of obesity in humans<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Finucane, M. M., Sharpton, T. J., Laurent, T. J. &amp; Pollard, K. S. A taxonomic signature of obesity in the microbiome? Getting to the guts of the matter. PLoS ONE 9, e84689 (2014)." href="#ref-CR25" id="ref-link-section-d187515113e628">25</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Walters, W. A., Xu, Z. &amp; Knight, R. Meta-analyses of human gut microbes associated with obesity and IBD. FEBS Lett. 588, 4223–4233 (2014)." href="#ref-CR26" id="ref-link-section-d187515113e628_1">26</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 27" title="Sze, M. A. &amp; Schloss, P. D. Looking for a signal in the noise: revisiting obesity and the microbiome. mBio 7, e01018-16 (2016)." href="https://www.nature.com/articles/s41564-023-01426-7#ref-CR27" id="ref-link-section-d187515113e631">27</a></sup>.</p><p>This misconception also reflects an unhelpful tendency to examine sequence-based microbiota profiles at very broad taxonomic levels, such as phyla. Although this is appealing from a data simplification point of view, it fails to incorporate the huge and inherent variability within individual phyla. To draw a crude analogy, humans, birds, fish, reptiles and even sea squirts are all members of the phylum Chordata, yet clearly have very different physiologies, lifestyles and impacts on their environments.</p><p>Moreover, this claim was also based on relative-abundance-based DNA sequence data. Compositional data are still useful, and can correlate well with absolute quantifications obtained with techniques such as qPCR<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 28" title="Tettamanti Boshier, F. A. et al. Complementing 16S rRNA gene amplicon sequencing with total bacterial load to infer absolute species concentrations in the vaginal microbiome. mSystems 5, e00777-19 (2020)." href="https://www.nature.com/articles/s41564-023-01426-7#ref-CR28" id="ref-link-section-d187515113e641">28</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 29" title="Reichardt, N. et al. Specific substrate-driven changes in human faecal microbiota composition contrast with functional redundancy in short-chain fatty acid production. ISME J. 12, 610–622 (2018)." href="https://www.nature.com/articles/s41564-023-01426-7#ref-CR29" id="ref-link-section-d187515113e644">29</a></sup>. However, some studies have suggested that relative-abundance-based correlations can lose significance when absolute microbial abundances are also factored into analyses<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 9" title="Vandeputte, D. et al. Quantitative microbiome profiling links gut community variation to microbial load. Nature 551, 507–511 (2017)." href="https://www.nature.com/articles/s41564-023-01426-7#ref-CR9" id="ref-link-section-d187515113e648">9</a></sup>. Moving forwards, increased incorporation of absolute quantification data may help to make conclusions based on compositional analyses more robust.</p></div></div></section><section data-title="“The gut microbiome is functionally redundant”"><div id="Sec10-section"><h2 id="Sec10">“The gut microbiome is functionally redundant”</h2><p>This claim derives from studies showing that, whereas the taxonomic composition of human metagenomes can vary hugely, functional gene prediction profiles remain remarkably consistent. We contend that this is at least partly artefactual, as these functional comparisons are typically carried out after discarding the large proportion of metagenomic data that does not map to reference databases<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 30" title="Méric, G., Wick, R. R., Watts, S. C., Holt, K. E. &amp; Inouye, M. Correcting index databases improves metagenomic studies. Preprint at bioRxiv 
                  https://doi.org/10.1101/712166
                  
                 (2019)." href="https://www.nature.com/articles/s41564-023-01426-7#ref-CR30" id="ref-link-section-d187515113e660">30</a></sup>. Much of what does map to those databases is likely to be derived from common housekeeping and/or well-characterized genes, which are found across many different bacteria and are also relatively well represented in reference databases. These comparative analyses therefore fail to accurately capture specialist, or less well-characterized, functions. As such, the truth is more nuanced. Although there are important functionalities that are conserved across many different human microbiota species, such as short-chain fatty acid production<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 29" title="Reichardt, N. et al. Specific substrate-driven changes in human faecal microbiota composition contrast with functional redundancy in short-chain fatty acid production. ISME J. 12, 610–622 (2018)." href="https://www.nature.com/articles/s41564-023-01426-7#ref-CR29" id="ref-link-section-d187515113e664">29</a></sup>, there are many key functions that are only carried out by a relatively small number of microbiota species. Examples include oxalate<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 31" title="Daniel, S. L. et al. Forty years of Oxalobacter formigenes, a gutsy oxalate-degrading specialist. Appl. Environ. Microbiol. 87, e0054421 (2021)." href="https://www.nature.com/articles/s41564-023-01426-7#ref-CR31" id="ref-link-section-d187515113e668">31</a></sup> and resistant-starch<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 32" title="Ze, X., Duncan, S. H., Louis, P. &amp; Flint, H. J. Ruminococcus bromii is a keystone species for the degradation of resistant starch in the human colon. ISME J. 6, 1535–1543 (2012)." href="https://www.nature.com/articles/s41564-023-01426-7#ref-CR32" id="ref-link-section-d187515113e672">32</a></sup> degradation. In the absence of key species, functionalities such as these may not necessarily be fully replaced by other members of the microbiota.</p></div></section><section data-title="“Sequencing is unbiased”"><div id="Sec11-section"><h2 id="Sec11">“Sequencing is unbiased”</h2><p>Although sequence-based methods have been transformative for microbiome research, they are not perfect. Biases can be introduced at every step of sequence-based studies, from sample collection and storage, through laboratory-based steps such as DNA extraction, to choice of bioinformatic pipelines and reference databases used to analyse the data<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 33" title="Quince, C., Walker, A. W., Simpson, J. T., Loman, N. J. &amp; Segata, N. Shotgun metagenomics, from sampling to analysis. Nat. Biotechnol. 35, 833–844 (2017)." href="https://www.nature.com/articles/s41564-023-01426-7#ref-CR33" id="ref-link-section-d187515113e684">33</a></sup>. Comparisons of sequence-based versus culture-based studies of the microbiota have shown that sequence-based approaches completely failed to detect some species that were only recovered using traditional culturing methods<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 34" title="Rajilić-Stojanović, M., Smidt, H. &amp; de Vos, W. M. Diversity of the human gastrointestinal tract microbiota revisited. Environ. Microbiol. 9, 2125–2136 (2007)." href="https://www.nature.com/articles/s41564-023-01426-7#ref-CR34" id="ref-link-section-d187515113e688">34</a></sup>. Modern sequence-based approaches are hugely powerful but, like all techniques, they are not unbiased. For optimal interpretation of results, it is important to be aware of the inherent limitations of any given method.</p></div></section><section data-title="“We need standardized methodologies”"><div id="Sec12-section"><h2 id="Sec12">“We need standardized methodologies”</h2><p>This opinion is prevalent in the microbiome field and is sensibly grounded in a desire to make it easier and more robust to compare results from different studies. However, as outlined above, there are no methodologies that are perfect, and all are biased in some way. If everyone in the world is using the same method, then everyone is equally blind to the limitations of that particular approach. There is also the problem of deciding which protocol everyone should use. For example, comparisons of results from the Human Microbiome Project with the MetaHIT project showed stark differences in microbiome profiles and indicated that the Human Microbiome Project protocol was less effective at extracting DNA from eukaryotes and Gram-positive bacterial lineages<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 35" title="Wesolowska-Andersen, A. et al. Choice of bacterial DNA extraction method from fecal material influences community structure as evaluated by metagenomic analysis. Microbiome 2, 19 (2014)." href="https://www.nature.com/articles/s41564-023-01426-7#ref-CR35" id="ref-link-section-d187515113e701">35</a></sup>. The truth is that the ‘best’ method fundamentally depends on the underlying structure of the microbial community in a given sample and this can vary hugely between individuals and between body sites. For these reasons we argue, as others have, that optimization and verification of sequence-based results with additional approaches are preferable to asking everyone to adopt the same method<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 36" title="Munafò, M. R. &amp; Smith, G. D. Robust research needs many lines of evidence. Nature 553, 399–401 (2018)." href="https://www.nature.com/articles/s41564-023-01426-7#ref-CR36" id="ref-link-section-d187515113e705">36</a></sup>. An additional advantage of multi-faceted studies using different methods and research platforms is that they can enable more mechanistic understandings of associations between microorganisms and host phenotypes<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Hoyles, L. et al. Molecular phenomics and metagenomics of hepatic steatosis in non-diabetic obese women. Nat. Med. 24, 1070–1080 (2018)." href="#ref-CR37" id="ref-link-section-d187515113e709">37</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Koh, A. et al. Microbially produced imidazole propionate impairs insulin signaling through mTORC1. Cell 175, 947–961.e17 (2018)." href="#ref-CR38" id="ref-link-section-d187515113e709_1">38</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 39" title="Belda, E. et al. Impairment of gut microbial biotin metabolism and host biotin status in severe obesity: effect of biotin and prebiotic supplementation on improved metabolism. Gut 71, 2463–2480 (2022)." href="https://www.nature.com/articles/s41564-023-01426-7#ref-CR39" id="ref-link-section-d187515113e712">39</a></sup>. Increased transparency when reporting methodology choices would be helpful for comparing results from different studies. The recently published STORMS (Strengthening the Organization and Reporting of Microbiome Studies) guidelines<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 40" title="Mirzayi, C. et al. Reporting guidelines for human microbiome research: the STORMS checklist. Nat. Med. 27, 1885–1892 (2021)." href="https://www.nature.com/articles/s41564-023-01426-7#ref-CR40" id="ref-link-section-d187515113e716">40</a></sup>, for example, could greatly aid this process if adopted widely.</p></div></section><section data-title="“Most of the human microbiota is ‘unculturable’”"><div id="Sec13-section"><h2 id="Sec13">“Most of the human microbiota is ‘unculturable’”</h2><p>The adoption of high-throughput sequence-based technologies has also been mirrored by claims from some quarters that these methods must be used because most human-associated microorganisms cannot be cultivated in the laboratory. In fact, a reasonably large proportion of the bacterial and archaeal component of our microbiota has already been cultured<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 41" title="Lagkouvardos, I., Overmann, J. &amp; Clavel, T. Cultured microbes represent a substantial fraction of the human and mouse gut microbiota. Gut Microbes 8, 493–503 (2017)." href="https://www.nature.com/articles/s41564-023-01426-7#ref-CR41" id="ref-link-section-d187515113e728">41</a></sup> (viruses and fungi remain under-represented), with pioneering work from as early as the 1970s establishing the cultivability of a broad diversity of species from the human gut microbiota<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 42" title="Moore, W. E. C. &amp; Holdeman, L. V. Human fecal flora: the normal flora of 20 Japanese-Hawaiians. Appl. Microbiol. 27, 961–979 (1974)." href="https://www.nature.com/articles/s41564-023-01426-7#ref-CR42" id="ref-link-section-d187515113e732">42</a></sup>. Many more species continue to be cultured as laboratory-based efforts have increased throughput<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 43" title="Lagier, J. et al. Culture of previously uncultured members of the human gut microbiota by culturomics. Nat. Microbiol. 1, 16203 (2016)." href="https://www.nature.com/articles/s41564-023-01426-7#ref-CR43" id="ref-link-section-d187515113e736">43</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 44" title="Browne, H. P. et al. Culturing of ‘unculturable’ human microbiota reveals novel taxa and extensive sporulation. Nature 533, 543–546 (2016)." href="https://www.nature.com/articles/s41564-023-01426-7#ref-CR44" id="ref-link-section-d187515113e739">44</a></sup>. This implies that current gaps in culture collections are at least in part attributable to a lack of previous effort rather than an inherent ‘unculturability’. Although cultivation is undeniably labour intensive, has its own biases and often requires expensive specialist equipment and media, there are clear advantages to having microorganisms in culture. These include enabling mechanistic experiments, verifying genomic predictions, and developing them as novel therapeutics<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 45" title="Walker, A. W., Duncan, S. H., Louis, P. &amp; Flint, H. J. Phylogeny, culturing, and metagenomics of the human gut microbiota. Trends Microbiol 22, 267–274 (2014)." href="https://www.nature.com/articles/s41564-023-01426-7#ref-CR45" id="ref-link-section-d187515113e743">45</a></sup>. Given the importance of continued cultivation-based work for the progression of microbiome research, it is gratifying that this myth has become less prevalent in recent years following the publication of the aforementioned high-impact studies that demonstrate it to be false. However, it serves as an excellent example of how previously widely accepted dogma is sometimes simply not true. There are important lessons for many other myths and misconceptions that have yet to become as widely rejected.</p></div></section><section data-title="Conclusions"><div id="Sec14-section"><h2 id="Sec14">Conclusions</h2><div id="Sec14-content"><p>The microbiome field is broad, and there are many other controversial topics that might also have been included here. However, knowledge is still evolving on many of these; consequently, we have largely focused on concepts where we believe there is a strong evidence base for rejecting myths and misconceptions. Although some of the points above may seem trivial, we argue that the accuracy of details such as these matters. If we are consistently repeating falsehoods about minor details, can our accuracy be relied upon when covering more important matters? We hope that, by illustrating just a few examples of microbiome myths and misconceptions, we can draw increased attention to the potential problems of over-simplification and insufficient critical assessment in the microbiome literature.</p><p>Given the many potential health impacts, the huge amount of funding and the keen public interest in microbiomes, rejection of unfounded assertions is crucial if we wish to avoid expending finite resources researching unproductive avenues and undermining public confidence in our conclusions.</p></div></div></section>
                </div><div>
            <div id="MagazineFulltextArticleBodySuffix"><section aria-labelledby="Bib1" data-title="References"><div id="Bib1-section"><h2 id="Bib1">References</h2><div id="Bib1-content"><div data-container-section="references"><ol data-track-component="outbound reference"><li data-counter="1."><p id="ref-CR1">Hacker, J. &amp; Blum-Oehler, G. In appreciation of Theodor Escherich. <i>Nat. Rev. Microbiol.</i> <b>5</b>, 902 (2007).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2sXhtlajt7fM" aria-label="CAS reference 1">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 1" href="http://scholar.google.com/scholar_lookup?&amp;title=In%20appreciation%20of%20Theodor%20Escherich&amp;journal=Nat.%20Rev.%20Microbiol.&amp;volume=5&amp;publication_year=2007&amp;author=Hacker%2CJ&amp;author=Blum-Oehler%2CG">
                    Google Scholar</a> 
                </p></li><li data-counter="2."><p id="ref-CR2">Cruickshank, R. <i>Bacillus bifidus</i>: its characters and isolation from the intestine of infants. <i>J. Hyg.</i> <b>24</b>, 241–254 (1925).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:STN:280:DC%2BC3crktVKrsQ%3D%3D" aria-label="CAS reference 2">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20474862" aria-label="PubMed reference 2">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2167664" aria-label="PubMed Central reference 2">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 2" href="http://scholar.google.com/scholar_lookup?&amp;title=Bacillus%20bifidus%3A%20its%20characters%20and%20isolation%20from%20the%20intestine%20of%20infants&amp;journal=J.%20Hyg.&amp;volume=24&amp;pages=241-254&amp;publication_year=1925&amp;author=Cruickshank%2CR">
                    Google Scholar</a> 
                </p></li><li data-counter="3."><p id="ref-CR3">Metchnikoff, E. <i>The Prolongation of Life: Optimistic Studies</i> (William Heinemann, G. P. Putnam’s Sons, 1907).</p></li><li data-counter="4."><p id="ref-CR4">Miller, I. The gut–brain axis: historical reflections. <i>Microb. Ecol. Health Dis.</i> <b>29</b>, 1542921 (2018).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30425612" aria-label="PubMed reference 4">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6225396" aria-label="PubMed Central reference 4">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 4" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20gut%E2%80%93brain%20axis%3A%20historical%20reflections&amp;journal=Microb.%20Ecol.%20Health%20Dis.&amp;volume=29&amp;publication_year=2018&amp;author=Miller%2CI">
                    Google Scholar</a> 
                </p></li><li data-counter="5."><p id="ref-CR5">Roediger, W. E. W. The colonic epithelium in ulcerative colitis: an energy-deficiency disease? <i>Lancet</i> <b>316</b>, 712–715 (1980).</p><p><a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 5" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20colonic%20epithelium%20in%20ulcerative%20colitis%3A%20an%20energy-deficiency%20disease%3F&amp;journal=Lancet&amp;volume=316&amp;pages=712-715&amp;publication_year=1980&amp;author=Roediger%2CWEW">
                    Google Scholar</a> 
                </p></li><li data-counter="6."><p id="ref-CR6">Prescott, S. L. History of medicine: origin of the term microbiome and why it matters. <i>Hum. Microb. J.</i> <b>4</b>, 24–25 (2017).</p><p><a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 6" href="http://scholar.google.com/scholar_lookup?&amp;title=History%20of%20medicine%3A%20origin%20of%20the%20term%20microbiome%20and%20why%20it%20matters&amp;journal=Hum.%20Microb.%20J.&amp;volume=4&amp;pages=24-25&amp;publication_year=2017&amp;author=Prescott%2CSL">
                    Google Scholar</a> 
                </p></li><li data-counter="7."><p id="ref-CR7">Stephen, A. M. &amp; Cummings, J. H. The microbial contribution to human faecal mass. <i>J. Med. Microbiol.</i> <b>13</b>, 45–56 (1980).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:STN:280:DyaL3c7ktVensQ%3D%3D" aria-label="CAS reference 7">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=7359576" aria-label="PubMed reference 7">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 7" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20microbial%20contribution%20to%20human%20faecal%20mass&amp;journal=J.%20Med.%20Microbiol.&amp;volume=13&amp;pages=45-56&amp;publication_year=1980&amp;author=Stephen%2CAM&amp;author=Cummings%2CJH">
                    Google Scholar</a> 
                </p></li><li data-counter="8."><p id="ref-CR8">Hoyles, L. &amp; McCartney, A. L. What do we mean when we refer to <i>Bacteroidetes</i> populations in the human gastrointestinal microbiota? <i>FEMS Microbiol. Lett.</i> <b>299</b>, 175–183 (2009).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXht1Oksb3O" aria-label="CAS reference 8">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19694813" aria-label="PubMed reference 8">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 8" href="http://scholar.google.com/scholar_lookup?&amp;title=What%20do%20we%20mean%20when%20we%20refer%20to%20Bacteroidetes%20populations%20in%20the%20human%20gastrointestinal%20microbiota%3F&amp;journal=FEMS%20Microbiol.%20Lett.&amp;volume=299&amp;pages=175-183&amp;publication_year=2009&amp;author=Hoyles%2CL&amp;author=McCartney%2CAL">
                    Google Scholar</a> 
                </p></li><li data-counter="9."><p id="ref-CR9">Vandeputte, D. et al. Quantitative microbiome profiling links gut community variation to microbial load. <i>Nature</i> <b>551</b>, 507–511 (2017).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhvVKqtb%2FM" aria-label="CAS reference 9">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29143816" aria-label="PubMed reference 9">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 9" href="http://scholar.google.com/scholar_lookup?&amp;title=Quantitative%20microbiome%20profiling%20links%20gut%20community%20variation%20to%20microbial%20load&amp;journal=Nature&amp;volume=551&amp;pages=507-511&amp;publication_year=2017&amp;author=Vandeputte%2CD">
                    Google Scholar</a> 
                </p></li><li data-counter="10."><p id="ref-CR10">Sender, R., Fuchs, S. &amp; Milo, R. Revised estimates for the number of human and bacteria cells in the body. <i>PLoS Biol.</i> <b>14</b>, e1002533 (2016).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27541692" aria-label="PubMed reference 10">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4991899" aria-label="PubMed Central reference 10">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 10" href="http://scholar.google.com/scholar_lookup?&amp;title=Revised%20estimates%20for%20the%20number%20of%20human%20and%20bacteria%20cells%20in%20the%20body&amp;journal=PLoS%20Biol.&amp;volume=14&amp;publication_year=2016&amp;author=Sender%2CR&amp;author=Fuchs%2CS&amp;author=Milo%2CR">
                    Google Scholar</a> 
                </p></li><li data-counter="11."><p id="ref-CR11">Cummings, J. H., Bingham, S. A., Heaton, K. W. &amp; Eastwood, M. A. Fecal weight, colon cancer risk, and dietary intake of nonstarch polysaccharides (dietary fiber). <i>Gastroenterology</i> <b>103</b>, 1783–1789 (1992).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:STN:280:DyaK3s%2Fotlymug%3D%3D" aria-label="CAS reference 11">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=1333426" aria-label="PubMed reference 11">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 11" href="http://scholar.google.com/scholar_lookup?&amp;title=Fecal%20weight%2C%20colon%20cancer%20risk%2C%20and%20dietary%20intake%20of%20nonstarch%20polysaccharides%20%28dietary%20fiber%29&amp;journal=Gastroenterology&amp;volume=103&amp;pages=1783-1789&amp;publication_year=1992&amp;author=Cummings%2CJH&amp;author=Bingham%2CSA&amp;author=Heaton%2CKW&amp;author=Eastwood%2CMA">
                    Google Scholar</a> 
                </p></li><li data-counter="12."><p id="ref-CR12">Cummings, J. H., Pomare, E. W., Branch, W. J., Naylor, C. P. &amp; Macfarlane, G. T. Short chain fatty acids in human large intestine, portal, hepatic and venous blood. <i>Gut</i> <b>28</b>, 1221–1227 (1987).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:STN:280:DyaL1c%2Fls1CrsA%3D%3D" aria-label="CAS reference 12">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=3678950" aria-label="PubMed reference 12">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1433442" aria-label="PubMed Central reference 12">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 12" href="http://scholar.google.com/scholar_lookup?&amp;title=Short%20chain%20fatty%20acids%20in%20human%20large%20intestine%2C%20portal%2C%20hepatic%20and%20venous%20blood&amp;journal=Gut&amp;volume=28&amp;pages=1221-1227&amp;publication_year=1987&amp;author=Cummings%2CJH&amp;author=Pomare%2CEW&amp;author=Branch%2CWJ&amp;author=Naylor%2CCP&amp;author=Macfarlane%2CGT">
                    Google Scholar</a> 
                </p></li><li data-counter="13."><p id="ref-CR13">Sender, R., Fuchs, S. &amp; Milo, R. Are we really vastly outnumbered? Revisiting the ratio of bacterial to host cells in humans. <i>Cell</i> <b>164</b>, 337–340 (2016).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28Xhs12lsL4%3D" aria-label="CAS reference 13">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26824647" aria-label="PubMed reference 13">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 13" href="http://scholar.google.com/scholar_lookup?&amp;title=Are%20we%20really%20vastly%20outnumbered%3F%20Revisiting%20the%20ratio%20of%20bacterial%20to%20host%20cells%20in%20humans&amp;journal=Cell&amp;volume=164&amp;pages=337-340&amp;publication_year=2016&amp;author=Sender%2CR&amp;author=Fuchs%2CS&amp;author=Milo%2CR">
                    Google Scholar</a> 
                </p></li><li data-counter="14."><p id="ref-CR14">Ferretti, P. et al. Mother-to-infant microbial transmission from different body sites shapes the developing infant gut microbiome. <i>Cell Host Microbe</i> <b>24</b>, 133–145.e5 (2018).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtlSqt7vI" aria-label="CAS reference 14">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30001516" aria-label="PubMed reference 14">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6716579" aria-label="PubMed Central reference 14">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 14" href="http://scholar.google.com/scholar_lookup?&amp;title=Mother-to-infant%20microbial%20transmission%20from%20different%20body%20sites%20shapes%20the%20developing%20infant%20gut%20microbiome&amp;journal=Cell%20Host%20Microbe&amp;volume=24&amp;pages=133-145.e5&amp;publication_year=2018&amp;author=Ferretti%2CP">
                    Google Scholar</a> 
                </p></li><li data-counter="15."><p id="ref-CR15">Valles-Colomer, M. et al. The person-to-person transmission landscape of the gut and oral microbiomes. <i>Nature</i> <b>614</b>, 125–135 (2023).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXhslKisbY%3D" aria-label="CAS reference 15">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=36653448" aria-label="PubMed reference 15">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9892008" aria-label="PubMed Central reference 15">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 15" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20person-to-person%20transmission%20landscape%20of%20the%20gut%20and%20oral%20microbiomes&amp;journal=Nature&amp;volume=614&amp;pages=125-135&amp;publication_year=2023&amp;author=Valles-Colomer%2CM">
                    Google Scholar</a> 
                </p></li><li data-counter="16."><p id="ref-CR16">Rothschild, D. et al. Environment dominates over host genetics in shaping human gut microbiota. <i>Nature</i> <b>555</b>, 210–215 (2018).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXjsFaitLg%3D" aria-label="CAS reference 16">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29489753" aria-label="PubMed reference 16">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 16" href="http://scholar.google.com/scholar_lookup?&amp;title=Environment%20dominates%20over%20host%20genetics%20in%20shaping%20human%20gut%20microbiota&amp;journal=Nature&amp;volume=555&amp;pages=210-215&amp;publication_year=2018&amp;author=Rothschild%2CD">
                    Google Scholar</a> 
                </p></li><li data-counter="17."><p id="ref-CR17">Yatsunenko, T. et al. Human gut microbiome viewed across age and geography. <i>Nature</i> <b>486</b>, 222–227 (2012).</p></li><li data-counter="18."><p id="ref-CR18">Goodrich, J. K. et al. Human genetics shape the gut microbiome. <i>Cell</i> <b>159</b>, 789–799 (2014).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXhvV2msrnJ" aria-label="CAS reference 18">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25417156" aria-label="PubMed reference 18">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4255478" aria-label="PubMed Central reference 18">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 18" href="http://scholar.google.com/scholar_lookup?&amp;title=Human%20genetics%20shape%20the%20gut%20microbiome&amp;journal=Cell&amp;volume=159&amp;pages=789-799&amp;publication_year=2014&amp;author=Goodrich%2CJK">
                    Google Scholar</a> 
                </p></li><li data-counter="19."><p id="ref-CR19">Schäffler, H. &amp; Breitrück, A. <i>Clostridium difficile</i> – from colonization to infection. <i>Front. Microbiol.</i> <b>9</b>, 646 (2018).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29692762" aria-label="PubMed reference 19">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5902504" aria-label="PubMed Central reference 19">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 19" href="http://scholar.google.com/scholar_lookup?&amp;title=Clostridium%20difficile%20%E2%80%93%20from%20colonization%20to%20infection&amp;journal=Front.%20Microbiol.&amp;volume=9&amp;publication_year=2018&amp;author=Sch%C3%A4ffler%2CH&amp;author=Breitr%C3%BCck%2CA">
                    Google Scholar</a> 
                </p></li><li data-counter="20."><p id="ref-CR20">Worby, C. J. et al. Longitudinal multi-omics analyses link gut microbiome dysbiosis with recurrent urinary tract infections in women. <i>Nat. Microbiol.</i> <b>7</b>, 630–639 (2022).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XhtFyqtrzF" aria-label="CAS reference 20">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=35505248" aria-label="PubMed reference 20">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9136705" aria-label="PubMed Central reference 20">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 20" href="http://scholar.google.com/scholar_lookup?&amp;title=Longitudinal%20multi-omics%20analyses%20link%20gut%20microbiome%20dysbiosis%20with%20recurrent%20urinary%20tract%20infections%20in%20women&amp;journal=Nat.%20Microbiol.&amp;volume=7&amp;pages=630-639&amp;publication_year=2022&amp;author=Worby%2CCJ">
                    Google Scholar</a> 
                </p></li><li data-counter="21."><p id="ref-CR21">Olesen, S. W. &amp; Alm, E. J. Dysbiosis is not an answer. <i>Nat. Microbiol.</i> <b>1</b>, 16228 (2016).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXkvFyqsr0%3D" aria-label="CAS reference 21">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27886190" aria-label="PubMed reference 21">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 21" href="http://scholar.google.com/scholar_lookup?&amp;title=Dysbiosis%20is%20not%20an%20answer&amp;journal=Nat.%20Microbiol.&amp;volume=1&amp;publication_year=2016&amp;author=Olesen%2CSW&amp;author=Alm%2CEJ">
                    Google Scholar</a> 
                </p></li><li data-counter="22."><p id="ref-CR22">Ni, J., Wu, G. D., Albenberg, L. &amp; Tomov, V. T. Gut microbiota and IBD: causation or correlation? <i>Nat. Rev. Gastroenterol. Hepatol.</i> <b>14</b>, 573–584 (2017).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28743984" aria-label="PubMed reference 22">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5880536" aria-label="PubMed Central reference 22">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 22" href="http://scholar.google.com/scholar_lookup?&amp;title=Gut%20microbiota%20and%20IBD%3A%20causation%20or%20correlation%3F&amp;journal=Nat.%20Rev.%20Gastroenterol.%20Hepatol.&amp;volume=14&amp;pages=573-584&amp;publication_year=2017&amp;author=Ni%2CJ&amp;author=Wu%2CGD&amp;author=Albenberg%2CL&amp;author=Tomov%2CVT">
                    Google Scholar</a> 
                </p></li><li data-counter="23."><p id="ref-CR23">Pammi, M. et al. Intestinal dysbiosis in preterm infants preceding necrotizing enterocolitis: a systematic review and meta-analysis. <i>Microbiome</i> <b>5</b>, 31 (2017).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28274256" aria-label="PubMed reference 23">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5343300" aria-label="PubMed Central reference 23">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 23" href="http://scholar.google.com/scholar_lookup?&amp;title=Intestinal%20dysbiosis%20in%20preterm%20infants%20preceding%20necrotizing%20enterocolitis%3A%20a%20systematic%20review%20and%20meta-analysis&amp;journal=Microbiome&amp;volume=5&amp;publication_year=2017&amp;author=Pammi%2CM">
                    Google Scholar</a> 
                </p></li><li data-counter="24."><p id="ref-CR24">Damhorst, G. L., Adelman, M. W., Woodworth, M. H. &amp; Kraft, C. S. Current capabilities of gut microbiome-based diagnostics and the promise of clinical application. <i>J. Infect. Dis.</i> <b>223</b>, S270–S275 (2021).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhsVyisrvN" aria-label="CAS reference 24">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33330938" aria-label="PubMed reference 24">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 24" href="http://scholar.google.com/scholar_lookup?&amp;title=Current%20capabilities%20of%20gut%20microbiome-based%20diagnostics%20and%20the%20promise%20of%20clinical%20application&amp;journal=J.%20Infect.%20Dis.&amp;volume=223&amp;pages=S270-S275&amp;publication_year=2021&amp;author=Damhorst%2CGL&amp;author=Adelman%2CMW&amp;author=Woodworth%2CMH&amp;author=Kraft%2CCS">
                    Google Scholar</a> 
                </p></li><li data-counter="25."><p id="ref-CR25">Finucane, M. M., Sharpton, T. J., Laurent, T. J. &amp; Pollard, K. S. A taxonomic signature of obesity in the microbiome? Getting to the guts of the matter. <i>PLoS ONE</i> <b>9</b>, e84689 (2014).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24416266" aria-label="PubMed reference 25">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3885756" aria-label="PubMed Central reference 25">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 25" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20taxonomic%20signature%20of%20obesity%20in%20the%20microbiome%3F%20Getting%20to%20the%20guts%20of%20the%20matter&amp;journal=PLoS%20ONE&amp;volume=9&amp;publication_year=2014&amp;author=Finucane%2CMM&amp;author=Sharpton%2CTJ&amp;author=Laurent%2CTJ&amp;author=Pollard%2CKS">
                    Google Scholar</a> 
                </p></li><li data-counter="26."><p id="ref-CR26">Walters, W. A., Xu, Z. &amp; Knight, R. Meta-analyses of human gut microbes associated with obesity and IBD. <i>FEBS Lett.</i> <b>588</b>, 4223–4233 (2014).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXhslCisbjO" aria-label="CAS reference 26">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25307765" aria-label="PubMed reference 26">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5050012" aria-label="PubMed Central reference 26">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 26" href="http://scholar.google.com/scholar_lookup?&amp;title=Meta-analyses%20of%20human%20gut%20microbes%20associated%20with%20obesity%20and%20IBD&amp;journal=FEBS%20Lett.&amp;volume=588&amp;pages=4223-4233&amp;publication_year=2014&amp;author=Walters%2CWA&amp;author=Xu%2CZ&amp;author=Knight%2CR">
                    Google Scholar</a> 
                </p></li><li data-counter="27."><p id="ref-CR27">Sze, M. A. &amp; Schloss, P. D. Looking for a signal in the noise: revisiting obesity and the microbiome. <i>mBio</i> <b>7</b>, e01018-16 (2016).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27555308" aria-label="PubMed reference 27">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4999546" aria-label="PubMed Central reference 27">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 27" href="http://scholar.google.com/scholar_lookup?&amp;title=Looking%20for%20a%20signal%20in%20the%20noise%3A%20revisiting%20obesity%20and%20the%20microbiome&amp;journal=mBio&amp;volume=7&amp;publication_year=2016&amp;author=Sze%2CMA&amp;author=Schloss%2CPD">
                    Google Scholar</a> 
                </p></li><li data-counter="28."><p id="ref-CR28">Tettamanti Boshier, F. A. et al. Complementing 16S rRNA gene amplicon sequencing with total bacterial load to infer absolute species concentrations in the vaginal microbiome. <i>mSystems</i> <b>5</b>, e00777-19 (2020).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32265316" aria-label="PubMed reference 28">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7141891" aria-label="PubMed Central reference 28">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 28" href="http://scholar.google.com/scholar_lookup?&amp;title=Complementing%2016S%20rRNA%20gene%20amplicon%20sequencing%20with%20total%20bacterial%20load%20to%20infer%20absolute%20species%20concentrations%20in%20the%20vaginal%20microbiome&amp;journal=mSystems&amp;volume=5&amp;publication_year=2020&amp;author=Tettamanti%20Boshier%2CFA">
                    Google Scholar</a> 
                </p></li><li data-counter="29."><p id="ref-CR29">Reichardt, N. et al. Specific substrate-driven changes in human faecal microbiota composition contrast with functional redundancy in short-chain fatty acid production. <i>ISME J.</i> <b>12</b>, 610–622 (2018).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtVOitr8%3D" aria-label="CAS reference 29">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29192904" aria-label="PubMed reference 29">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 29" href="http://scholar.google.com/scholar_lookup?&amp;title=Specific%20substrate-driven%20changes%20in%20human%20faecal%20microbiota%20composition%20contrast%20with%20functional%20redundancy%20in%20short-chain%20fatty%20acid%20production&amp;journal=ISME%20J.&amp;volume=12&amp;pages=610-622&amp;publication_year=2018&amp;author=Reichardt%2CN">
                    Google Scholar</a> 
                </p></li><li data-counter="30."><p id="ref-CR30">Méric, G., Wick, R. R., Watts, S. C., Holt, K. E. &amp; Inouye, M. Correcting index databases improves metagenomic studies. Preprint at <i>bioRxiv</i> <a href="https://doi.org/10.1101/712166" data-track="click" data-track-action="external reference" data-track-label="10.1101/712166">https://doi.org/10.1101/712166</a> (2019).</p></li><li data-counter="31."><p id="ref-CR31">Daniel, S. L. et al. Forty years of <i>Oxalobacter formigenes</i>, a gutsy oxalate-degrading specialist. <i>Appl. Environ. Microbiol.</i> <b>87</b>, e0054421 (2021).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34190610" aria-label="PubMed reference 31">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 31" href="http://scholar.google.com/scholar_lookup?&amp;title=Forty%20years%20of%20Oxalobacter%20formigenes%2C%20a%20gutsy%20oxalate-degrading%20specialist&amp;journal=Appl.%20Environ.%20Microbiol.&amp;volume=87&amp;publication_year=2021&amp;author=Daniel%2CSL">
                    Google Scholar</a> 
                </p></li><li data-counter="32."><p id="ref-CR32">Ze, X., Duncan, S. H., Louis, P. &amp; Flint, H. J. <i>Ruminococcus bromii</i> is a keystone species for the degradation of resistant starch in the human colon. <i>ISME J.</i> <b>6</b>, 1535–1543 (2012).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XhtVOjtL3L" aria-label="CAS reference 32">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22343308" aria-label="PubMed reference 32">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3400402" aria-label="PubMed Central reference 32">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 32" href="http://scholar.google.com/scholar_lookup?&amp;title=Ruminococcus%20bromii%20is%20a%20keystone%20species%20for%20the%20degradation%20of%20resistant%20starch%20in%20the%20human%20colon&amp;journal=ISME%20J.&amp;volume=6&amp;pages=1535-1543&amp;publication_year=2012&amp;author=Ze%2CX&amp;author=Duncan%2CSH&amp;author=Louis%2CP&amp;author=Flint%2CHJ">
                    Google Scholar</a> 
                </p></li><li data-counter="33."><p id="ref-CR33">Quince, C., Walker, A. W., Simpson, J. T., Loman, N. J. &amp; Segata, N. Shotgun metagenomics, from sampling to analysis. <i>Nat. Biotechnol.</i> <b>35</b>, 833–844 (2017).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhsVOht7vJ" aria-label="CAS reference 33">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28898207" aria-label="PubMed reference 33">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 33" href="http://scholar.google.com/scholar_lookup?&amp;title=Shotgun%20metagenomics%2C%20from%20sampling%20to%20analysis&amp;journal=Nat.%20Biotechnol.&amp;volume=35&amp;pages=833-844&amp;publication_year=2017&amp;author=Quince%2CC&amp;author=Walker%2CAW&amp;author=Simpson%2CJT&amp;author=Loman%2CNJ&amp;author=Segata%2CN">
                    Google Scholar</a> 
                </p></li><li data-counter="34."><p id="ref-CR34">Rajilić-Stojanović, M., Smidt, H. &amp; de Vos, W. M. Diversity of the human gastrointestinal tract microbiota revisited. <i>Environ. Microbiol.</i> <b>9</b>, 2125–2136 (2007).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17686012" aria-label="PubMed reference 34">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 34" href="http://scholar.google.com/scholar_lookup?&amp;title=Diversity%20of%20the%20human%20gastrointestinal%20tract%20microbiota%20revisited&amp;journal=Environ.%20Microbiol.&amp;volume=9&amp;pages=2125-2136&amp;publication_year=2007&amp;author=Rajili%C4%87-Stojanovi%C4%87%2CM&amp;author=Smidt%2CH&amp;author=Vos%2CWM">
                    Google Scholar</a> 
                </p></li><li data-counter="35."><p id="ref-CR35">Wesolowska-Andersen, A. et al. Choice of bacterial DNA extraction method from fecal material influences community structure as evaluated by metagenomic analysis. <i>Microbiome</i> <b>2</b>, 19 (2014).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24949196" aria-label="PubMed reference 35">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4063427" aria-label="PubMed Central reference 35">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 35" href="http://scholar.google.com/scholar_lookup?&amp;title=Choice%20of%20bacterial%20DNA%20extraction%20method%20from%20fecal%20material%20influences%20community%20structure%20as%20evaluated%20by%20metagenomic%20analysis&amp;journal=Microbiome&amp;volume=2&amp;publication_year=2014&amp;author=Wesolowska-Andersen%2CA">
                    Google Scholar</a> 
                </p></li><li data-counter="36."><p id="ref-CR36">Munafò, M. R. &amp; Smith, G. D. Robust research needs many lines of evidence. <i>Nature</i> <b>553</b>, 399–401 (2018).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29368721" aria-label="PubMed reference 36">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 36" href="http://scholar.google.com/scholar_lookup?&amp;title=Robust%20research%20needs%20many%20lines%20of%20evidence&amp;journal=Nature&amp;volume=553&amp;pages=399-401&amp;publication_year=2018&amp;author=Munaf%C3%B2%2CMR&amp;author=Smith%2CGD">
                    Google Scholar</a> 
                </p></li><li data-counter="37."><p id="ref-CR37">Hoyles, L. et al. Molecular phenomics and metagenomics of hepatic steatosis in non-diabetic obese women. <i>Nat. Med.</i> <b>24</b>, 1070–1080 (2018).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXht1WqsLjN" aria-label="CAS reference 37">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29942096" aria-label="PubMed reference 37">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6140997" aria-label="PubMed Central reference 37">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 37" href="http://scholar.google.com/scholar_lookup?&amp;title=Molecular%20phenomics%20and%20metagenomics%20of%20hepatic%20steatosis%20in%20non-diabetic%20obese%20women&amp;journal=Nat.%20Med.&amp;volume=24&amp;pages=1070-1080&amp;publication_year=2018&amp;author=Hoyles%2CL">
                    Google Scholar</a> 
                </p></li><li data-counter="38."><p id="ref-CR38">Koh, A. et al. Microbially produced imidazole propionate impairs insulin signaling through mTORC1. <i>Cell</i> <b>175</b>, 947–961.e17 (2018).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXitVWju7jI" aria-label="CAS reference 38">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30401435" aria-label="PubMed reference 38">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 38" href="http://scholar.google.com/scholar_lookup?&amp;title=Microbially%20produced%20imidazole%20propionate%20impairs%20insulin%20signaling%20through%20mTORC1&amp;journal=Cell&amp;volume=175&amp;pages=947-961.e17&amp;publication_year=2018&amp;author=Koh%2CA">
                    Google Scholar</a> 
                </p></li><li data-counter="39."><p id="ref-CR39">Belda, E. et al. Impairment of gut microbial biotin metabolism and host biotin status in severe obesity: effect of biotin and prebiotic supplementation on improved metabolism. <i>Gut</i> <b>71</b>, 2463–2480 (2022).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XivFegtbzE" aria-label="CAS reference 39">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=35017197" aria-label="PubMed reference 39">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 39" href="http://scholar.google.com/scholar_lookup?&amp;title=Impairment%20of%20gut%20microbial%20biotin%20metabolism%20and%20host%20biotin%20status%20in%20severe%20obesity%3A%20effect%20of%20biotin%20and%20prebiotic%20supplementation%20on%20improved%20metabolism&amp;journal=Gut&amp;volume=71&amp;pages=2463-2480&amp;publication_year=2022&amp;author=Belda%2CE">
                    Google Scholar</a> 
                </p></li><li data-counter="40."><p id="ref-CR40">Mirzayi, C. et al. Reporting guidelines for human microbiome research: the STORMS checklist. <i>Nat. Med.</i> <b>27</b>, 1885–1892 (2021).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXisFWks73O" aria-label="CAS reference 40">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34789871" aria-label="PubMed reference 40">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9105086" aria-label="PubMed Central reference 40">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 40" href="http://scholar.google.com/scholar_lookup?&amp;title=Reporting%20guidelines%20for%20human%20microbiome%20research%3A%20the%20STORMS%20checklist&amp;journal=Nat.%20Med.&amp;volume=27&amp;pages=1885-1892&amp;publication_year=2021&amp;author=Mirzayi%2CC">
                    Google Scholar</a> 
                </p></li><li data-counter="41."><p id="ref-CR41">Lagkouvardos, I., Overmann, J. &amp; Clavel, T. Cultured microbes represent a substantial fraction of the human and mouse gut microbiota. <i>Gut Microbes</i> <b>8</b>, 493–503 (2017).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28418756" aria-label="PubMed reference 41">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5628658" aria-label="PubMed Central reference 41">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 41" href="http://scholar.google.com/scholar_lookup?&amp;title=Cultured%20microbes%20represent%20a%20substantial%20fraction%20of%20the%20human%20and%20mouse%20gut%20microbiota&amp;journal=Gut%20Microbes&amp;volume=8&amp;pages=493-503&amp;publication_year=2017&amp;author=Lagkouvardos%2CI&amp;author=Overmann%2CJ&amp;author=Clavel%2CT">
                    Google Scholar</a> 
                </p></li><li data-counter="42."><p id="ref-CR42">Moore, W. E. C. &amp; Holdeman, L. V. Human fecal flora: the normal flora of 20 Japanese-Hawaiians. <i>Appl. Microbiol.</i> <b>27</b>, 961–979 (1974).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:STN:280:DyaE2c7ntVagsQ%3D%3D" aria-label="CAS reference 42">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=4598229" aria-label="PubMed reference 42">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC380185" aria-label="PubMed Central reference 42">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 42" href="http://scholar.google.com/scholar_lookup?&amp;title=Human%20fecal%20flora%3A%20the%20normal%20flora%20of%2020%20Japanese-Hawaiians&amp;journal=Appl.%20Microbiol.&amp;volume=27&amp;pages=961-979&amp;publication_year=1974&amp;author=Moore%2CWEC&amp;author=Holdeman%2CLV">
                    Google Scholar</a> 
                </p></li><li data-counter="43."><p id="ref-CR43">Lagier, J. et al. Culture of previously uncultured members of the human gut microbiota by culturomics. <i>Nat. Microbiol.</i> <b>1</b>, 16203 (2016).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXkvFyruro%3D" aria-label="CAS reference 43">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27819657" aria-label="PubMed reference 43">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 43" href="http://scholar.google.com/scholar_lookup?&amp;title=Culture%20of%20previously%20uncultured%20members%20of%20the%20human%20gut%20microbiota%20by%20culturomics&amp;journal=Nat.%20Microbiol.&amp;volume=1&amp;publication_year=2016&amp;author=Lagier%2CJ">
                    Google Scholar</a> 
                </p></li><li data-counter="44."><p id="ref-CR44">Browne, H. P. et al. Culturing of ‘unculturable’ human microbiota reveals novel taxa and extensive sporulation. <i>Nature</i> <b>533</b>, 543–546 (2016).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28Xnt1Ogsrg%3D" aria-label="CAS reference 44">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27144353" aria-label="PubMed reference 44">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4890681" aria-label="PubMed Central reference 44">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 44" href="http://scholar.google.com/scholar_lookup?&amp;title=Culturing%20of%20%E2%80%98unculturable%E2%80%99%20human%20microbiota%20reveals%20novel%20taxa%20and%20extensive%20sporulation&amp;journal=Nature&amp;volume=533&amp;pages=543-546&amp;publication_year=2016&amp;author=Browne%2CHP">
                    Google Scholar</a> 
                </p></li><li data-counter="45."><p id="ref-CR45">Walker, A. W., Duncan, S. H., Louis, P. &amp; Flint, H. J. Phylogeny, culturing, and metagenomics of the human gut microbiota. <i>Trends Microbiol</i> <b>22</b>, 267–274 (2014).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXlt1CrsLg%3D" aria-label="CAS reference 45">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24698744" aria-label="PubMed reference 45">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 45" href="http://scholar.google.com/scholar_lookup?&amp;title=Phylogeny%2C%20culturing%2C%20and%20metagenomics%20of%20the%20human%20gut%20microbiota&amp;journal=Trends%20Microbiol&amp;volume=22&amp;pages=267-274&amp;publication_year=2014&amp;author=Walker%2CAW&amp;author=Duncan%2CSH&amp;author=Louis%2CP&amp;author=Flint%2CHJ">
                    Google Scholar</a> 
                </p></li></ol><p><a data-track="click" data-track-action="download citation references" data-track-label="link" rel="nofollow" href="https://citation-needed.springer.com/v2/references/10.1038/s41564-023-01426-7?format=refman&amp;flavour=references">Download references<svg width="16" height="16" focusable="false" role="img" aria-hidden="true"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-download"></use></svg></a></p></div></div></div></section></div><section data-title="Acknowledgements"><div id="Ack1-section"><h2 id="Ack1">Acknowledgements</h2><p>A.W.W. would like to thank S. Patrick for the original invitation to give a talk on this topic, which inspired the subsequent writing of this article. A.W.W. and the Rowett Institute receive core funding support from the Scottish Government’s Rural and Environment Science and Analytical Services division. L.H. is funded by Alzheimer’s Research UK, Healthcare Infection Society, Diabetes UK, Cancer Research UK and the European Union’s Horizon 2020 research and innovation programme under grant agreement 874583. This publication reflects only the authors’ views and the European Commission is not responsible for any use that may be made of the information it contains.</p></div></section><section aria-labelledby="author-information" data-title="Author information"><div id="author-information-section"><h2 id="author-information">Author information</h2><div id="author-information-content"><h3 id="affiliations">Authors and Affiliations</h3><ol><li id="Aff1"><p>Microbiome, Food Innovation and Food Security Research Theme, Rowett Institute, University of Aberdeen, Aberdeen, UK</p><p>Alan W. Walker</p></li><li id="Aff2"><p>Department of Biosciences, Nottingham Trent University, Nottingham, UK</p><p>Lesley Hoyles</p></li></ol><h3 id="corresponding-author">Corresponding author</h3><p id="corresponding-author-list">Correspondence to
                <a id="corresp-c1" href="mailto:alan.walker@abdn.ac.uk">Alan W. Walker</a>.</p></div></div></section><section data-title="Ethics declarations"><div id="ethics-section"><h2 id="ethics">Ethics declarations</h2><div id="ethics-content">
              
                <h3 id="FPar2">Competing interests</h3>
                <p>The authors declare no competing interests.</p>
              
            </div></div></section><section data-title="Peer review"><div id="peer-review-section"><h2 id="peer-review">Peer review</h2><div id="peer-review-content">
              
              
                <h3 id="FPar1">Peer review information</h3>
                <p><i>Nature Microbiology</i> thanks Ami Bhatt, Fergus Shanahan and the other, anonymous, reviewer(s) for their contribution to the peer review of this work.</p>
              
            </div></div></section><section data-title="Additional information"><div id="additional-information-section"><h2 id="additional-information">Additional information</h2><p><b>Publisher’s note</b> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div></section><section data-title="Rights and permissions"><div id="rightslink-section"><h2 id="rightslink">Rights and permissions</h2><div id="rightslink-content"><p>Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law.</p><p><a data-track="click" data-track-action="view rights and permissions" data-track-label="link" href="https://s100.copyright.com/AppDispatchServlet?title=Human%20microbiome%20myths%20and%20misconceptions&amp;author=Alan%20W.%20Walker%20et%20al&amp;contentID=10.1038%2Fs41564-023-01426-7&amp;copyright=Springer%20Nature%20Limited&amp;publication=2058-5276&amp;publicationDate=2023-07-31&amp;publisherName=SpringerNature&amp;orderBeanReset=true">Reprints and Permissions</a></p></div></div></section><section aria-labelledby="article-info" data-title="About this article"><div id="article-info-section"><h2 id="article-info">About this article</h2><div id="article-info-content"><div><p><a data-crossmark="10.1038/s41564-023-01426-7" target="_blank" rel="noopener" href="https://crossmark.crossref.org/dialog/?doi=10.1038/s41564-023-01426-7" data-track="click" data-track-action="Click Crossmark" data-track-label="link" data-test="crossmark"><img width="57" height="81" alt="Check for updates. Verify currency and authenticity via CrossMark" src="data:image/svg+xml;base64,<svg height="81" width="57" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="m17.35 35.45 21.3-14.2v-17.03h-21.3" fill="#989898"/><path d="m38.65 35.45-21.3-14.2v-17.03h21.3" fill="#747474"/><path d="m28 .5c-12.98 0-23.5 10.52-23.5 23.5s10.52 23.5 23.5 23.5 23.5-10.52 23.5-23.5c0-6.23-2.48-12.21-6.88-16.62-4.41-4.4-10.39-6.88-16.62-6.88zm0 41.25c-9.8 0-17.75-7.95-17.75-17.75s7.95-17.75 17.75-17.75 17.75 7.95 17.75 17.75c0 4.71-1.87 9.22-5.2 12.55s-7.84 5.2-12.55 5.2z" fill="#535353"/><path d="m41 36c-5.81 6.23-15.23 7.45-22.43 2.9-7.21-4.55-10.16-13.57-7.03-21.5l-4.92-3.11c-4.95 10.7-1.19 23.42 8.78 29.71 9.97 6.3 23.07 4.22 30.6-4.86z" fill="#9c9c9c"/><path d="m.2 58.45c0-.75.11-1.42.33-2.01s.52-1.09.91-1.5c.38-.41.83-.73 1.34-.94.51-.22 1.06-.32 1.65-.32.56 0 1.06.11 1.51.35.44.23.81.5 1.1.81l-.91 1.01c-.24-.24-.49-.42-.75-.56-.27-.13-.58-.2-.93-.2-.39 0-.73.08-1.05.23-.31.16-.58.37-.81.66-.23.28-.41.63-.53 1.04-.13.41-.19.88-.19 1.39 0 1.04.23 1.86.68 2.46.45.59 1.06.88 1.84.88.41 0 .77-.07 1.07-.23s.59-.39.85-.68l.91 1c-.38.43-.8.76-1.28.99-.47.22-1 .34-1.58.34-.59 0-1.13-.1-1.64-.31-.5-.2-.94-.51-1.31-.91-.38-.4-.67-.9-.88-1.48-.22-.59-.33-1.26-.33-2.02zm8.4-5.33h1.61v2.54l-.05 1.33c.29-.27.61-.51.96-.72s.76-.31 1.24-.31c.73 0 1.27.23 1.61.71.33.47.5 1.14.5 2.02v4.31h-1.61v-4.1c0-.57-.08-.97-.25-1.21-.17-.23-.45-.35-.83-.35-.3 0-.56.08-.79.22-.23.15-.49.36-.78.64v4.8h-1.61zm7.37 6.45c0-.56.09-1.06.26-1.51.18-.45.42-.83.71-1.14.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.36c.07.62.29 1.1.65 1.44.36.33.82.5 1.38.5.29 0 .57-.04.83-.13s.51-.21.76-.37l.55 1.01c-.33.21-.69.39-1.09.53-.41.14-.83.21-1.26.21-.48 0-.92-.08-1.34-.25-.41-.16-.76-.4-1.07-.7-.31-.31-.55-.69-.72-1.13-.18-.44-.26-.95-.26-1.52zm4.6-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.07.45-.31.29-.5.73-.58 1.3zm2.5.62c0-.57.09-1.08.28-1.53.18-.44.43-.82.75-1.13s.69-.54 1.1-.71c.42-.16.85-.24 1.31-.24.45 0 .84.08 1.17.23s.61.34.85.57l-.77 1.02c-.19-.16-.38-.28-.56-.37-.19-.09-.39-.14-.61-.14-.56 0-1.01.21-1.35.63-.35.41-.52.97-.52 1.67 0 .69.17 1.24.51 1.66.34.41.78.62 1.32.62.28 0 .54-.06.78-.17.24-.12.45-.26.64-.42l.67 1.03c-.33.29-.69.51-1.08.65-.39.15-.78.23-1.18.23-.46 0-.9-.08-1.31-.24-.4-.16-.75-.39-1.05-.7s-.53-.69-.7-1.13c-.17-.45-.25-.96-.25-1.53zm6.91-6.45h1.58v6.17h.05l2.54-3.16h1.77l-2.35 2.8 2.59 4.07h-1.75l-1.77-2.98-1.08 1.23v1.75h-1.58zm13.69 1.27c-.25-.11-.5-.17-.75-.17-.58 0-.87.39-.87 1.16v.75h1.34v1.27h-1.34v5.6h-1.61v-5.6h-.92v-1.2l.92-.07v-.72c0-.35.04-.68.13-.98.08-.31.21-.57.4-.79s.42-.39.71-.51c.28-.12.63-.18 1.04-.18.24 0 .48.02.69.07.22.05.41.1.57.17zm.48 5.18c0-.57.09-1.08.27-1.53.17-.44.41-.82.72-1.13.3-.31.65-.54 1.04-.71.39-.16.8-.24 1.23-.24s.84.08 1.24.24c.4.17.74.4 1.04.71s.54.69.72 1.13c.19.45.28.96.28 1.53s-.09 1.08-.28 1.53c-.18.44-.42.82-.72 1.13s-.64.54-1.04.7-.81.24-1.24.24-.84-.08-1.23-.24-.74-.39-1.04-.7c-.31-.31-.55-.69-.72-1.13-.18-.45-.27-.96-.27-1.53zm1.65 0c0 .69.14 1.24.43 1.66.28.41.68.62 1.18.62.51 0 .9-.21 1.19-.62.29-.42.44-.97.44-1.66 0-.7-.15-1.26-.44-1.67-.29-.42-.68-.63-1.19-.63-.5 0-.9.21-1.18.63-.29.41-.43.97-.43 1.67zm6.48-3.44h1.33l.12 1.21h.05c.24-.44.54-.79.88-1.02.35-.24.7-.36 1.07-.36.32 0 .59.05.78.14l-.28 1.4-.33-.09c-.11-.01-.23-.02-.38-.02-.27 0-.56.1-.86.31s-.55.58-.77 1.1v4.2h-1.61zm-47.87 15h1.61v4.1c0 .57.08.97.25 1.2.17.24.44.35.81.35.3 0 .57-.07.8-.22.22-.15.47-.39.73-.73v-4.7h1.61v6.87h-1.32l-.12-1.01h-.04c-.3.36-.63.64-.98.86-.35.21-.76.32-1.24.32-.73 0-1.27-.24-1.61-.71-.33-.47-.5-1.14-.5-2.02zm9.46 7.43v2.16h-1.61v-9.59h1.33l.12.72h.05c.29-.24.61-.45.97-.63.35-.17.72-.26 1.1-.26.43 0 .81.08 1.15.24.33.17.61.4.84.71.24.31.41.68.53 1.11.13.42.19.91.19 1.44 0 .59-.09 1.11-.25 1.57-.16.47-.38.85-.65 1.16-.27.32-.58.56-.94.73-.35.16-.72.25-1.1.25-.3 0-.6-.07-.9-.2s-.59-.31-.87-.56zm0-2.3c.26.22.5.37.73.45.24.09.46.13.66.13.46 0 .84-.2 1.15-.6.31-.39.46-.98.46-1.77 0-.69-.12-1.22-.35-1.61-.23-.38-.61-.57-1.13-.57-.49 0-.99.26-1.52.77zm5.87-1.69c0-.56.08-1.06.25-1.51.16-.45.37-.83.65-1.14.27-.3.58-.54.93-.71s.71-.25 1.08-.25c.39 0 .73.07 1 .2.27.14.54.32.81.55l-.06-1.1v-2.49h1.61v9.88h-1.33l-.11-.74h-.06c-.25.25-.54.46-.88.64-.33.18-.69.27-1.06.27-.87 0-1.56-.32-2.07-.95s-.76-1.51-.76-2.65zm1.67-.01c0 .74.13 1.31.4 1.7.26.38.65.58 1.15.58.51 0 .99-.26 1.44-.77v-3.21c-.24-.21-.48-.36-.7-.45-.23-.08-.46-.12-.7-.12-.45 0-.82.19-1.13.59-.31.39-.46.95-.46 1.68zm6.35 1.59c0-.73.32-1.3.97-1.71.64-.4 1.67-.68 3.08-.84 0-.17-.02-.34-.07-.51-.05-.16-.12-.3-.22-.43s-.22-.22-.38-.3c-.15-.06-.34-.1-.58-.1-.34 0-.68.07-1 .2s-.63.29-.93.47l-.59-1.08c.39-.24.81-.45 1.28-.63.47-.17.99-.26 1.54-.26.86 0 1.51.25 1.93.76s.63 1.25.63 2.21v4.07h-1.32l-.12-.76h-.05c-.3.27-.63.48-.98.66s-.73.27-1.14.27c-.61 0-1.1-.19-1.48-.56-.38-.36-.57-.85-.57-1.46zm1.57-.12c0 .3.09.53.27.67.19.14.42.21.71.21.28 0 .54-.07.77-.2s.48-.31.73-.56v-1.54c-.47.06-.86.13-1.18.23-.31.09-.57.19-.76.31s-.33.25-.41.4c-.09.15-.13.31-.13.48zm6.29-3.63h-.98v-1.2l1.06-.07.2-1.88h1.34v1.88h1.75v1.27h-1.75v3.28c0 .8.32 1.2.97 1.2.12 0 .24-.01.37-.04.12-.03.24-.07.34-.11l.28 1.19c-.19.06-.4.12-.64.17-.23.05-.49.08-.76.08-.4 0-.74-.06-1.02-.18-.27-.13-.49-.3-.67-.52-.17-.21-.3-.48-.37-.78-.08-.3-.12-.64-.12-1.01zm4.36 2.17c0-.56.09-1.06.27-1.51s.41-.83.71-1.14c.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.37c.08.62.29 1.1.65 1.44.36.33.82.5 1.38.5.3 0 .58-.04.84-.13.25-.09.51-.21.76-.37l.54 1.01c-.32.21-.69.39-1.09.53s-.82.21-1.26.21c-.47 0-.92-.08-1.33-.25-.41-.16-.77-.4-1.08-.7-.3-.31-.54-.69-.72-1.13-.17-.44-.26-.95-.26-1.52zm4.61-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.08.45-.31.29-.5.73-.57 1.3zm3.01 2.23c.31.24.61.43.92.57.3.13.63.2.98.2.38 0 .65-.08.83-.23s.27-.35.27-.6c0-.14-.05-.26-.13-.37-.08-.1-.2-.2-.34-.28-.14-.09-.29-.16-.47-.23l-.53-.22c-.23-.09-.46-.18-.69-.3-.23-.11-.44-.24-.62-.4s-.33-.35-.45-.55c-.12-.21-.18-.46-.18-.75 0-.61.23-1.1.68-1.49.44-.38 1.06-.57 1.83-.57.48 0 .91.08 1.29.25s.71.36.99.57l-.74.98c-.24-.17-.49-.32-.73-.42-.25-.11-.51-.16-.78-.16-.35 0-.6.07-.76.21-.17.15-.25.33-.25.54 0 .14.04.26.12.36s.18.18.31.26c.14.07.29.14.46.21l.54.19c.23.09.47.18.7.29s.44.24.64.4c.19.16.34.35.46.58.11.23.17.5.17.82 0 .3-.06.58-.17.83-.12.26-.29.48-.51.68-.23.19-.51.34-.84.45-.34.11-.72.17-1.15.17-.48 0-.95-.09-1.41-.27-.46-.19-.86-.41-1.2-.68z" fill="#535353"/></g></svg>"/></a></p><div><h3 id="citeas">Cite this article</h3><p>Walker, A.W., Hoyles, L. Human microbiome myths and misconceptions.
                    <i>Nat Microbiol</i> <b>8</b>, 1392–1396 (2023). https://doi.org/10.1038/s41564-023-01426-7</p><p><a data-test="citation-link" data-track="click" data-track-action="download article citation" data-track-label="link" data-track-external="" rel="nofollow" href="https://citation-needed.springer.com/v2/references/10.1038/s41564-023-01426-7?format=refman&amp;flavour=citation">Download citation<svg width="16" height="16" focusable="false" role="img" aria-hidden="true"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-download"></use></svg></a></p><ul data-test="publication-history"><li><p>Received<span>: </span><span><time datetime="2022-08-31">31 August 2022</time></span></p></li><li><p>Accepted<span>: </span><span><time datetime="2023-06-15">15 June 2023</time></span></p></li><li><p>Published<span>: </span><span><time datetime="2023-07-31">31 July 2023</time></span></p></li><li><p>Issue Date<span>: </span><span><time datetime="2023-08">August 2023</time></span></p></li><li><p><abbr title="Digital Object Identifier">DOI</abbr><span>: </span><span>https://doi.org/10.1038/s41564-023-01426-7</span></p></li></ul></div></div></div></div></section>
            </div></div>
  </body>
</html>
