<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <title>James Routley | Feed</title>
    <link
      rel="stylesheet"
      type="text/css"
      href="../styles.css"
      media="screen"
    />
  </head>
  <body>
    <a href="/index.html">Back</a>
    <a href="https://www.nature.com/articles/s41586-023-06585-5">Original</a>
    <h1>Inhibition of fatty acid oxidation enables heart regeneration in adult mice</h1>
    
    <div id="readability-page-1" class="page"><div>
                <section data-title="Main"><div id="Sec1-section"><h2 id="Sec1">Main</h2><div id="Sec1-content"><p>Shortly after birth, energy metabolism in cardiomyocytes (CMs) shifts from glycolytic to oxidative metabolism, resulting in major rearrangements in mitochondrial homoeostasis, cellular architecture and electrophysiological properties, among others<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 1" title="Maroli, G. &amp; Braun, T. The long and winding road of cardiomyocyte maturation. Cardiovasc. Res. 117, 712–726 (2021)." href="https://riskmusings.substack.com/articles/s41586-023-06585-5#ref-CR1" id="ref-link-section-d180729366e657">1</a></sup>. Most CMs terminally withdraw from the cell cycle during the early postnatal period as part of the maturation program and undergo hypertrophic growth together with changes in cell–cell and cell–extracellular matrix interactions<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 2" title="Yuan, X. &amp; Braun, T. Multimodal regulation of cardiac myocyte proliferation. Circ. Res. 121, 293–309 (2017)." href="https://riskmusings.substack.com/articles/s41586-023-06585-5#ref-CR2" id="ref-link-section-d180729366e661">2</a></sup>. Postnatal maturation of CMs together with oxidative DNA damage, due to high metabolic activity, creates a natural barrier against CM cell division<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Nakada, Y. et al. Hypoxia induces heart regeneration in adult mice. Nature 
                https://doi.org/10.1038/nature20173
                
               (2016)." href="#ref-CR4" id="ref-link-section-d180729366e665">4</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Puente, B. N. et al. The oxygen-rich postnatal environment induces cardiomyocyte cell-cycle arrest through DNA damage response. Cell 157, 565–579 (2014)." href="#ref-CR5" id="ref-link-section-d180729366e665_1">5</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 6" title="Ben-Yair, R. et al. H3K27me3-mediated silencing of structural genes is required for zebrafish heart regeneration. Development 146, dev178632 (2019)." href="https://riskmusings.substack.com/articles/s41586-023-06585-5#ref-CR6" id="ref-link-section-d180729366e668">6</a></sup>.</p><p>Metabolic and structural maturation of CMs are tightly intertwined. Reversal of either of the two processes may re-establish some proliferative capacity in CMs<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Zhu, Y., Do, V. D., Richards, A. M. &amp; Foo, R. What we know about cardiomyocyte dedifferentiation. J. Mol. Cell. Cardiol. 152, 80–91 (2021)." href="https://riskmusings.substack.com/articles/s41586-023-06585-5#ref-CR7" id="ref-link-section-d180729366e675">7</a></sup>. Enforcement of anaerobic metabolism by expression of pyruvate kinase isoenzyme 2 (<i>Pkm2</i>) mRNA, deletion of pyruvate dehydrogenase 4 (<i>Pdk4</i>) and inhibition of succinate dehydrogenase by malonate treatment promote CM cell proliferation<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 4" title="Nakada, Y. et al. Hypoxia induces heart regeneration in adult mice. Nature 
                https://doi.org/10.1038/nature20173
                
               (2016)." href="https://riskmusings.substack.com/articles/s41586-023-06585-5#ref-CR4" id="ref-link-section-d180729366e685">4</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Tao, G. et al. Pitx2 promotes heart repair by activating the antioxidant response after cardiac injury. Nature 534, 119–123 (2016)." href="#ref-CR8" id="ref-link-section-d180729366e688">8</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Magadum, A. et al. Pkm2 regulates cardiomyocyte cell cycle and promotes cardiac regeneration. Circulation 141, 1249–1265 (2020)." href="#ref-CR9" id="ref-link-section-d180729366e688_1">9</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Cardoso, A. C. et al. Mitochondrial substrate utilization regulates cardiomyocyte cell-cycle progression. Nat. Metab. 2, 167–178 (2020)." href="#ref-CR10" id="ref-link-section-d180729366e688_2">10</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 11" title="Bae, J. et al. Malonate promotes adult cardiomyocyte proliferation and heart regeneration. Circulation 143, 1973–1986 (2021)." href="https://riskmusings.substack.com/articles/s41586-023-06585-5#ref-CR11" id="ref-link-section-d180729366e691">11</a></sup>. However, the mechanisms linking metabolic rewiring with transcriptional and structural alterations, limiting proliferation of adult CMs, are not well understood.</p><p>Changes in the concentration of different metabolites, which serve as essential cofactors or substrates for various chromatin modifiers, potentially couple metabolic processes to the chromatin landscape and gene activity. α-ketoglutarate (αKG), a central intermediate of the Krebs cycle, is such a metabolite and is required for the activity of several dioxygenases, including JmjC-domain-containing histone lysine demethylases (KDMs)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 12" title="Atlasi, Y. &amp; Stunnenberg, H. G. The interplay of epigenetic marks during stem cell differentiation and development. Nat. Rev. Genet. 18, 643–658 (2017)." href="https://riskmusings.substack.com/articles/s41586-023-06585-5#ref-CR12" id="ref-link-section-d180729366e698">12</a></sup>. αKG-dependent KDMs play pivotal roles in cardiac development, maturation of CMs and heart function by regulating levels of H3K4me3 and H3K9me3 histone modifications<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 13" title="Kaneda, R. et al. Genome-wide histone methylation profile for heart failure. Genes Cells 14, 69–77 (2009)." href="https://riskmusings.substack.com/articles/s41586-023-06585-5#ref-CR13" id="ref-link-section-d180729366e702">13</a></sup>. H3K4me3 is a classical marker of active promoters, and numerous cell identity genes in different tissues and organs, including the heart, are characterized by broad domains of H3K4me3. Such domains can extend for more than 40 kilobases and often correlate with high levels of gene expression and increased transcription elongation<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Chen, K. et al. Broad H3K4me3 is associated with increased transcription elongation and enhancer activity at tumor-suppressor genes. Nat. Genet. 47, 1149–1157 (2015)." href="#ref-CR14" id="ref-link-section-d180729366e706">14</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Howe, F. S., Fischl, H., Murray, S. C. &amp; Mellor, J. Is H3K4me3 instructive for transcription activation? Bioessays 39, e201600095 (2017)." href="#ref-CR15" id="ref-link-section-d180729366e706_1">15</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Gilsbach, R. et al. Distinct epigenetic programs regulate cardiac myocyte development and disease in the human heart in vivo. Nat. Commun. 9, 391 (2018)." href="#ref-CR16" id="ref-link-section-d180729366e706_2">16</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 17" title="Pokholok, D. K. et al. Genome-wide map of nucleosome acetylation and methylation in yeast. Cell 122, 517–527 (2005)." href="https://riskmusings.substack.com/articles/s41586-023-06585-5#ref-CR17" id="ref-link-section-d180729366e709">17</a></sup>. By contrast, narrow domains of H3K4me3 are usually found in promoters of typical housekeeping genes<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 14" title="Chen, K. et al. Broad H3K4me3 is associated with increased transcription elongation and enhancer activity at tumor-suppressor genes. Nat. Genet. 47, 1149–1157 (2015)." href="https://riskmusings.substack.com/articles/s41586-023-06585-5#ref-CR14" id="ref-link-section-d180729366e713">14</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 18" title="Zhang, Z. et al. H3K4 tri-methylation breadth at transcription start sites impacts the transcriptome of systemic lupus erythematosus. Clin. Epigenetics 8, 14 (2016)." href="https://riskmusings.substack.com/articles/s41586-023-06585-5#ref-CR18" id="ref-link-section-d180729366e716">18</a></sup>. Although αKG is well known as a central metabolic fuel and for its function in numerous signalling pathways<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 19" title="Liu, P. S. et al. α-ketoglutarate orchestrates macrophage activation through metabolic and epigenetic reprogramming. Nat. Immunol. 18, 985–994 (2017)." href="https://riskmusings.substack.com/articles/s41586-023-06585-5#ref-CR19" id="ref-link-section-d180729366e720">19</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 20" title="Chin, R. M. et al. The metabolite α-ketoglutarate extends lifespan by inhibiting ATP synthase and TOR. Nature 510, 397–401 (2014)." href="https://riskmusings.substack.com/articles/s41586-023-06585-5#ref-CR20" id="ref-link-section-d180729366e723">20</a></sup>, information about the role of αKG in chromatin reconfiguration is scarce<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 21" title="Carey, B. W., Finley, L. W., Cross, J. R., Allis, C. D. &amp; Thompson, C. B. Intracellular α-ketoglutarate maintains the pluripotency of embryonic stem cells. Nature 518, 413–416 (2015)." href="https://riskmusings.substack.com/articles/s41586-023-06585-5#ref-CR21" id="ref-link-section-d180729366e728">21</a></sup> and a potential role of αKG in promoting heart regeneration has not been investigated so far.</p></div></div></section><section data-title="Hyperplasia in Cpt1b-deficient hearts"><div id="Sec2-section"><h2 id="Sec2">Hyperplasia in <i>Cpt1b</i>-deficient hearts</h2><div id="Sec2-content"><p>Analysis of RNA-sequencing (RNA-seq) datasets revealed reduced expression levels of several key genes of glycolysis and cell cycle progression in the course of CM maturation during the first week after birth, whereas genes related to fatty acid oxidation (FAO) and the Krebs cycle were upregulated (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#Fig6">1a</a>). FAO-related genes that were upregulated included the muscle-specific isoform of carnitine palmitoyltransferase <i>Cpt1b</i> but not the ubiquitously expressed <i>Cpt1a</i> isoform, which are required for mitochondrial uptake of fatty acids and subsequent FAO (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#Fig6">1b</a>). This observation is in line with previous studies reporting developmental changes in the use of cardiac energy substrates and a correlation between increased FAO and cell cycle withdrawal during CM maturation<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 22" title="Lopaschuk, G. D. &amp; Jaswal, J. S. Energy metabolic phenotype of the cardiomyocyte during development, differentiation, and postnatal maturation. J. Cardiovasc. Pharmacol. 56, 130–140 (2010)." href="https://riskmusings.substack.com/articles/s41586-023-06585-5#ref-CR22" id="ref-link-section-d180729366e756">22</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 23" title="Greco, C. M. et al. DNA hydroxymethylation controls cardiomyocyte gene expression in development and hypertrophy. Nat. Commun. 7, 12418 (2016)." href="https://riskmusings.substack.com/articles/s41586-023-06585-5#ref-CR23" id="ref-link-section-d180729366e759">23</a></sup>. To explore a potential role of FAO in CM maturation and termination of proliferation, we inhibited the activity of CPT1 in CMs from neonatal mice at postnatal days 0–1 (P0–1) by treatment with etomoxir. Etomoxir treatment resulted in enhanced incorporation of 5-ethynyl-2′-deoxyuridine (EdU), increased numbers of Ki67- and pH3(Ser10)-positive cells, elevated protein levels of cyclin E1 and strong re-expression of <i>Nppa</i>, <i>Nppb</i> and <i>Acta1</i>, markers that are highly expressed at embryonic and fetal stages but are downregulated after birth<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 24" title="Man, J., Barnett, P. &amp; Christoffels, V. M. Structure and function of the Nppa–Nppb cluster locus during heart development and disease. Cell. Mol. Life Sci. 75, 1435–1444 (2018)." href="https://riskmusings.substack.com/articles/s41586-023-06585-5#ref-CR24" id="ref-link-section-d180729366e773">24</a></sup> (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#Fig6">1c–f</a>).</p><p>To determine the role of FAO for regulation of CM proliferation and growth in vivo, we generated <i>αMHC-Cre</i><sup><i>pos/+</i></sup>;<i>Cpt1b</i><sup><i>fl/fl</i></sup> mice (hereafter referred to as <i>Cpt1b</i><sup><i>cKO</i></sup>), in which the <i>Cpt1b</i> gene is specifically inactivated in CMs at embryonic stages. Inactivation of the <i>Cpt1b</i> gene was efficient without a compensatory increase in the level of <i>Cpt1a</i> expression (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#Fig1">1a</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#Fig6">1g,h</a>). Body weight (BW), heart weight (HW), HW/BW ratios and morphology of <i>Cpt1b</i><sup><i>cKO</i></sup> hearts were not changed compared to those of <i>αMHC-Cre</i><sup><i>pos/+</i></sup>;<i>Cpt1b</i><sup><i>+/+</i></sup> control mice (hereafter referred to as <i>Ctrl</i><sup><i>Cre</i></sup>) at P7 (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#Fig6">1i,j</a>). However, heart size, HW and HW/BW ratios but not BW increased in 10-week-old <i>Cpt1b</i><sup><i>cKO</i></sup> mice owing to concentric growth of the myocardium (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#Fig1">1b</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#Fig6">1k–o</a>). Notably, we observed a doubling of the absolute numbers of CMs in <i>Cpt1b</i><sup><i>cKO</i></sup> hearts, which was accompanied by markedly higher numbers of PCM1<sup>+</sup>EdU<sup>+</sup> CMs and of Ki67<sup>+</sup>, pH3<sup>+</sup> and aurora B<sup>+</sup> CMs in <i>Cpt1b</i>-mutant hearts at 10 weeks of age (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#Fig1">1c–g</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#Fig7">2a–e</a>). In addition, we noted a relatively modest increase of CM surface area in <i>Cpt1b</i><sup><i>cKO</i></sup> mice, suggesting combined hyperplastic and hypertrophic growth of the myocardium.</p><div data-test="figure" data-container-section="figure" id="figure-1" data-title="Inactivation of Cpt1b induces hyperplastic and hypertrophic growth of CMs."><figure><figcaption><b id="Fig1" data-test="figure-caption-text">Fig. 1: Inactivation of <i>Cpt1b</i> induces hyperplastic and hypertrophic growth of CMs.</b></figcaption><div><div><a data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="https://riskmusings.substack.com/articles/s41586-023-06585-5/figures/1" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41586-023-06585-5/MediaObjects/41586_2023_6585_Fig1_HTML.png?as=webp"/><img aria-describedby="Fig1" src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41586-023-06585-5/MediaObjects/41586_2023_6585_Fig1_HTML.png" alt="figure 1" loading="lazy" width="685" height="615"/></picture></a></div><div data-test="bottom-caption" id="figure-1-desc"><p><b>a</b>, Generation of <i>Cpt1b</i><sup><i>cKO</i></sup> mice. <b>b</b>, HW/BW ratio of 10-week-old <i>Ctrl</i><sup><i>Cre</i></sup> and <i>Cpt1b</i><sup><i>cKO</i></sup> mice (<i>n</i> = 5 per genotype). <b>c</b>, Quantification of CMs in adult <i>Ctrl</i><sup><i>Cre</i></sup> and <i>Cpt1b</i><sup><i>cKO</i></sup> hearts (<i>n</i> = 4 per genotype). <b>d</b>, EdU incorporation in PCM1<sup>+</sup> cardiac nuclei from <i>Ctrl</i><sup><i>Cre</i></sup> hearts (<i>n</i> = 7) and <i>Cpt1b</i><sup><i>cKO</i></sup> hearts (<i>n</i> = 5) by FACS analysis. <b>e</b>–<b>g</b>, Quantification of Ki67<sup>+</sup> (<b>e</b>), pH3<sup>+</sup> (<b>f</b>), aurora B<sup>+</sup> (<b>g</b>) and sarcomeric (sarc)-actinin<sup>+</sup> CMs on heart sections from <i>Ctrl</i><sup><i>Cre</i></sup> and <i>Cpt1b</i><sup><i>cKO</i></sup> mice (<i>n </i>= 3 per genotype). <b>h</b>, Distribution of CM cross-section area (µm<sup>2</sup>) in <i>Ctrl</i><sup><i>Cre</i></sup> and <i>Cpt1b</i><sup><i>cKO</i></sup> mice (<i>n</i> = 3 each). <b>i</b>, Strategy to generate <i>Cpt1b</i><sup><i>iKO</i></sup> mice and experimental outline. <b>j</b>, HW/BW ratio of control and <i>Cpt1b</i><sup><i>iKO</i></sup> mice, 4 and 8 weeks (w) after TAM injection (control, 4 and 8 weeks after TAM, and <i>Cpt1b</i><sup><i>iKO</i></sup>, 8 weeks after TAM, <i>n</i> = 4 per group; <i>Cpt1b</i><sup><i>iKO</i></sup>, 4 weeks after TAM, <i>n</i> = 3). <b>k</b>–<b>m</b>, Quantification of Ki67<sup>+</sup> (<b>k</b>), pH3<sup>+</sup> (<b>l</b>), aurora B<sup>+</sup> (<b>m</b>) and sarc-actinin<sup>+</sup> CMs on heart sections from control and <i>Cpt1b</i><sup><i>iKO</i></sup> mice (<i>n</i> = 4 for <b>k</b>; <i>n</i> = 3 for <b>l</b>,<b>m</b>), 4 weeks after completion of TAM treatment. <b>n</b>, Quantification of CMs in control and <i>Cpt1b</i><sup><i>iKO</i></sup> hearts 4 weeks after TAM injection (<i>n</i> = 3 each). <b>o</b>, Quantification of CMs in <i>Cpt1b</i><sup><i>iKO</i></sup> hearts 4 weeks (<i>n</i> = 3) and 8 weeks (<i>n</i> = 4) after TAM injection. Error bars represent mean ± s.e.m. <i>n</i> numbers refer to individual mice. Two-tailed, unpaired Student <i>t</i>-tests were used for statistical analysis of data in <b>b</b>–<b>h</b>,<b>k</b>–<b>o</b>. One-way analysis of variance (ANOVA) with Tukey tests was used for correction of multiple comparisons in <b>j</b>. *<i>P</i> &lt; 0.05, **<i>P</i> &lt; 0.01, ***<i>P</i> &lt; 0.001, ****<i>P</i> &lt; 0.0001.</p><p><a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#MOESM5">Source data</a></p></div></div></figure></div><p>A more detailed morphometric evaluation revealed a marked increase of the CM population with the smallest cell size (&lt;100 μm<sup>2</sup>), suggesting formation of new CMs in response to <i>Cpt1b</i> inactivation. The percentage of medium-sized CMs (100–300 μm<sup>2</sup>) decreased, whereas the number of larger CMs (&gt;300 μm<sup>2</sup>) increased significantly, reflecting the overall increase of CM surface area (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#Fig1">1h</a>). The relatively modest increase of CM surface area, in contrast to the massive increase in CM numbers in <i>Cpt1b</i><sup><i>cKO</i></sup> hearts, suggests that the increased heart size is mainly caused by hyperplastic growth. Of note, no signs of pathological cardiac hypertrophy such as myocardial fibrosis or cardiac dysfunction were evident in <i>Cpt1b</i><sup><i>cKO</i></sup> mice (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#Fig7">2f–h</a>). The data indicate that loss of <i>Cpt1b</i> stimulates proliferation of CMs, which is normally terminated in neonatal mouse hearts during the first week after birth.</p></div></div></section><section data-title="
                     Cpt1b-deficient CMs are less mature"><div id="Sec3-section"><h2 id="Sec3">
                     <i>Cpt1b</i>-deficient CMs are less mature</h2><div id="Sec3-content"><p>To examine whether inactivation of <i>Cpt1b</i> in fully differentiated adult CMs causes them to exit the post-mitotic state, we generated tamoxifen (TAM)-inducible <i>αMHC-MerCreMer</i><sup><i>pos/+</i></sup>;<i>Cpt1b</i><sup><i>fl/fl</i></sup> mice (hereafter referred to as <i>Cpt1b</i><sup><i>iKO</i></sup>; Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#Fig1">1i</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#Fig7">2i</a>). As for <i>Cpt1b</i><sup><i>cKO</i></sup> mice, the HW and HW/BW ratios but not the BW of <i>Cpt1b</i><sup><i>iKO</i></sup> mice increased markedly compared to those of <i>αMHC-MerCreMer</i><sup><i>pos/+</i></sup>;<i>Cpt1b</i><sup><i>+/+</i></sup> control mice 4 and 8 weeks after TAM injections (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#Fig1">1j</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#Fig7">2j</a>). Likewise, we did not observe fibrosis or impaired cardiac function, although <i>Cpt1b</i><sup><i>iKO</i></sup> mice hearts showed a pronounced cardiomegaly and an increase of CM surface area (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#Fig7">2k–o</a>). Notably, the number of Ki67<sup>+</sup>, pH3<sup>+</sup> and aurora B<sup>+</sup> CMs and the total number of CMs per heart were markedly elevated 4 weeks after TAM injection (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#Fig1">1k–n</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#Fig7">2p–r</a>). We also detected a further increase in the absolute number of CMs per heart and in HW at 8 weeks compared to 4 weeks after TAM administration, indicating that induction of CM proliferation by inactivation of <i>Cpt1b</i> is a lasting and continuous process (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#Fig1">1o</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#Fig7">2j</a>).</p><p>We next carried out RNA-seq analysis of CMs isolated from control, <i>Cpt1b</i><sup><i>cKO</i></sup> and <i>Cpt1b</i><sup><i>iKO</i></sup> mice and identified 1,513 and 1,432 differentially expressed genes (<i>P</i> value &lt; 0.05) in <i>Cpt1b</i><sup><i>cKO</i></sup> and <i>Cpt1b</i><sup><i>iKO</i></sup> hearts, respectively. Gene Ontology (GO) term enrichment analysis recognized several upregulated genes associated with fatty acid and lipid metabolic processes in both mutant mouse lines, most likely reflecting an attempt to compensate for abrogation of FAO<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 4" title="Nakada, Y. et al. Hypoxia induces heart regeneration in adult mice. Nature 
                https://doi.org/10.1038/nature20173
                
               (2016)." href="https://riskmusings.substack.com/articles/s41586-023-06585-5#ref-CR4" id="ref-link-section-d180729366e1366">4</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 5" title="Puente, B. N. et al. The oxygen-rich postnatal environment induces cardiomyocyte cell-cycle arrest through DNA damage response. Cell 157, 565–579 (2014)." href="https://riskmusings.substack.com/articles/s41586-023-06585-5#ref-CR5" id="ref-link-section-d180729366e1369">5</a></sup>. In both mutant mouse lines, we also detected strong upregulation of genes associated with cell cycle progression and reduced expression levels of genes involved in maturation and contraction, concomitant with upregulation of dedifferentiation markers (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#Fig8">3a–c</a>). Downregulated genes were mainly involved in cardiac muscle cell differentiation, heart development and sarcomere organization, suggesting a less mature state of <i>Cpt1b</i>-deficient CMs (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#Fig8">3a–c</a>). This notion was further confirmed by electron microscopy analysis and quantification of sarcomere density through immunofluorescence staining, uncovering reduced density of sarcomeres and irregular positioning of mitochondria, respectively (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#Fig8">3d,e</a>). The reduction of sarcomere density was particularly evident in the centre of CMs, close to the nuclei (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#Fig8">3e</a>). In addition, analysis by quantitative PCR (qPCR) with reverse transcription demonstrated reduced expression levels of mature CM-specific genes involved in Ca<sup>2+</sup> signalling, glucose transport (<i>Slc2a4</i>) and lactate production (<i>Ldha</i>), but increased expression levels of dedifferentiation markers (<i>Nppa</i> and <i>Acta1</i>; Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#Fig8">3f</a>). Integrative analysis of RNA-seq datasets from <i>Cpt1b</i><sup><i>cKO</i></sup> and <i>Cpt1b</i><sup><i>iKO</i></sup> CMs disclosed 172 and 299 genes that were jointly upregulated or downregulated in both mutant strains (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#Fig8">3g</a>). Genes involved in heart development and cell differentiation dominated in the group of downregulated genes (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#Fig8">3h</a>).</p><p>Of note, we observed activation of the anti-apoptotic HIF1A signalling pathway indicated by increased expression levels of <i>Hif1α</i> and its target gene <i>Bcl2</i>. <i>Egln3</i>, a suppressor of HIF1A, was downregulated, as were the pro-apoptotic genes <i>Bnip3</i> and <i>Bcl2l11</i>, which corresponds to the absence of TUNEL<sup>+</sup> CMs in <i>Cpt1b</i><sup><i>cKO</i></sup> hearts (Extended Data Figs. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#Fig8">3c</a> and <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#Fig9">4a,b</a>). Unexpectedly, we did not detect differences in the levels of reactive oxygen species between control and <i>Cpt1b</i>-knockout hearts, indicating that inhibition of FAO does not attenuate generation of reactive oxygen species (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#Fig9">4c</a>). However, the percentage of γH2A.X<sup>+</sup> cardiac nuclei dropped markedly in <i>Cpt1b</i>-deficient hearts compared to control hearts, suggesting that inhibition of FAO reduces DNA damage irrespective of reactive oxygen species or stimulates DNA repair (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#Fig9">4d</a>). Taken together, the data indicate that inhibition of FAO initiates a cascade of events converting adult CMs into a more immature state that enables CM proliferation and to a lesser extent favours hypertrophy.</p></div></div></section><section data-title="FAO blockade enables heart regeneration"><div id="Sec4-section"><h2 id="Sec4">FAO blockade enables heart regeneration</h2><div id="Sec4-content"><p>To investigate whether <i>Cpt1b</i> inactivation and the ensuing proliferation of CMs enables heart regeneration, we subjected <i>Cpt1b</i><sup><i>cKO</i></sup> and <i>Cpt1b</i><sup><i>iKO</i></sup> mice to ischaemia–reperfusion (I–R) injury, a model that closely mimics the situation in human patients receiving a stent for revascularization of an obstructed coronary artery. I–R-induced scars were virtually absent in <i>Cpt1b</i><sup><i>cKO</i></sup> and <i>Cpt1b</i><sup><i>iKO</i></sup> mice after 3 weeks compared to control animals, although the area at risk (AAR) was similar in both mutant hearts 24 h after I–R surgery (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#Fig2">2a–f</a>, Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#Fig10">5a,b</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#MOESM1">Supplementary Information</a>). The nearly complete prevention of scar formation was accompanied by the appearance of numerous small, round-shaped Ki67<sup>+</sup> as well as aurora B<sup>+</sup> CMs, specifically in the border zone of <i>Cpt1b</i>-deficient hearts, 72 h following I–R injury (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#Fig10">5c</a>), suggesting that the proliferative potential gained by abrogation of FAO allows preexisting CMs to re-enter the cell cycle and contribute to heart regeneration.</p><div data-test="figure" data-container-section="figure" id="figure-2" data-title="Cpt1b-mediated abrogation of FAO protects from I–R damage and enables heart regeneration."><figure><figcaption><b id="Fig2" data-test="figure-caption-text">Fig. 2: <i>Cpt1b</i>-mediated abrogation of FAO protects from I–R damage and enables heart regeneration.</b></figcaption><div><div><a data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="https://riskmusings.substack.com/articles/s41586-023-06585-5/figures/2" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41586-023-06585-5/MediaObjects/41586_2023_6585_Fig2_HTML.png?as=webp"/><img aria-describedby="Fig2" src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41586-023-06585-5/MediaObjects/41586_2023_6585_Fig2_HTML.png" alt="figure 2" loading="lazy" width="685" height="615"/></picture></a></div><div data-test="bottom-caption" id="figure-2-desc"><p><b>a</b>–<b>d</b>, Trichrome staining (<b>a</b>,<b>c</b>) and quantification of fibrosis (scar area; <b>b</b>,<b>d</b>) on heart sections from <i>Ctrl</i><sup><i>Cre</i></sup> mice (<i>n</i> = 5) and <i>Cpt1b</i><sup><i>cKO</i></sup> mice (<i>n</i> = 9) (<b>a</b>,<b>b</b>), and control mice (<i>n</i> = 9) and <i>Cpt1b</i><sup><i>iKO</i></sup> mice (<i>n</i> = 7) (<b>c</b>,<b>d</b>), 3 weeks after I–R injury. I–R surgery was carried out using 7-week-old (<b>a</b>) or 14-week-old (<b>c</b>) mice, 4 weeks after completion of TAM treatment. Scale bars, 300 μm. <b>e</b>,<b>f</b>, AAR (left) and infarct area (IF) of AAR (right) in <i>Ctrl</i><sup><i>Cre</i></sup> mice (<i>n</i> = 6) and <i>Cpt1b</i><sup><i>cKO</i></sup> mice (<i>n</i> = 5) (<b>e</b>) and in control mice (<i>n</i> = 4) and <i>Cpt1b</i><sup><i>iKO</i></sup> mice (<i>n</i> = 5) (<b>f</b>) 24 h after I–R injury. <b>g</b>,<b>h</b>, Trichrome staining (<b>g</b>) and quantification (<b>h</b>) of fibrosis on heart sections from control mice (<i>n</i> = 6) and <i>Cpt1b</i><sup><i>iKO</i></sup> mice (<i>n</i> = 5), 31 days after I–R injury. The I–R injury was carried out 1 day before initiation of <i>Cpt1b</i> deletion. Scale bars, 300 μm. <b>i</b>, Magnetic resonance imaging-based assessment of heart function in control and <i>Cpt1b</i><sup><i>iKO</i></sup> mice before and 7, 14 and 28 days after I–R surgery (before and 7 days after I–R, <i>n</i> = 6 each; 14 days after I–R, <i>n</i> = 5 each; 28 days after I–R, control <i>n</i> = 4, <i>Cpt1b</i><sup><i>iKO</i></sup> <i>n</i> = 5). LVEF, left ventricle ejection fraction. Asterisks representing <i>P</i> values in <b>i</b> refer to differences between individual measurements and the measurement before I–R. Asterisks representing <i>P</i> values above lines refer to measurements connected by the lines. Error bars represent mean ± s.e.m. <i>n</i> numbers refer to individual mice. Two-tailed, unpaired Student <i>t</i>-tests were used for statistical analysis in <b>b</b>,<b>d</b>–<b>f</b>,<b>h</b>. Two-way ANOVA with Tukey tests was used for correction of multiple comparisons in <b>i</b>. NS, not significant; *<i>P</i> &lt; 0.05; **<i>P</i> &lt; 0.01; ***<i>P</i> &lt; 0.001; ****<i>P</i> &lt; 0.0001.</p><p><a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#MOESM6">Source data</a></p></div></div></figure></div><p>We detected a reduced infarct area within the AAR in both <i>Cpt1b</i><sup><i>cKO</i></sup> and <i>Cpt1b</i><sup><i>iKO</i></sup> hearts 24 h after I–R injury and better cardiac contraction in <i>Cpt1b</i><sup><i>cKO</i></sup> mice 48 h after I–R injury, suggesting that inhibition of FAO also confers protection against I–R damage (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#Fig2">2e,f</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#Fig10">5d,e</a>). In line with these findings, the rate of cell death was significantly reduced in <i>Cpt1b</i>-deficient CMs compared to control CMs, as measured by uptake of ethidium homodimer 1 after exposure to 1% O<sub>2</sub> (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#Fig10">5f,g</a>). To determine the contribution of CM proliferation to heart regeneration after I–R damage and to exclude enhanced protection as the main cause of the absence of scar formation, we initiated <i>Cpt1b</i> deletion 1 day after applying the I–R injury. Initiation of <i>Cpt1b</i> deletion by injection of TAM did not reduce CPT1B protein levels at 3 and 6 days after I–R injury, corresponding to 2 and 5 days after the first TAM injection. CPT1B protein levels were significantly reduced only 11 days after I–R injury (corresponding to 10 days after the first TAM injection) excluding the possibility that cardioprotection can occur owing to the absence CPT1B (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#Fig10">5h</a>). Histological analysis revealed a strong reduction of the fibrotic scar area in <i>Cpt1b</i><sup><i>iKO</i></sup> animals compared to control animals. Notably, we also observed a major recovery of cardiac functions 4 weeks after I–R surgery, reaching nearly the same level as before the injury (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#Fig2">2g–i</a>, <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#MOESM1">Supplementary Information</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#MOESM4">Supplementary Video</a>), which unequivocally demonstrates that <i>Cpt1b</i> inactivation facilitates cardiac regeneration. We conclude that both enhanced CM proliferation and cardioprotection contribute to reduced scar formation after coronary occlusion in mice deficient for FAO.</p></div></div></section><section data-title="
                     Cpt1b loss increases αKG levels in CMs"><div id="Sec5-section"><h2 id="Sec5">
                     <i>Cpt1b</i> loss increases αKG levels in CMs</h2><div id="Sec5-content"><p>Next we investigated how blockage of FAO reprograms the metabolism of CMs. Metabolic flux and Seahorse assays demonstrated that long-chain fatty acid utilization is efficiently inhibited in <i>Cpt1b</i><sup><i>cKO</i></sup> CMs without evident compensatory usage of CPT1-independent medium- or short-chain fatty acids (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#Fig3">3a</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#Fig11">6a</a>). Furthermore, targeted metabolome analysis revealed that levels of acyl-carnitines derived from medium-chain and long-chain fatty acids are strongly reduced in <i>Cpt1b</i>-deficient CMs, whereas intracellular levels of free carnitine were markedly elevated (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#Fig3">3b</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#Fig11">6b,c</a>). Unexpectedly, intracellular levels of acetyl-CoA and most metabolites of the Krebs cycle, such as citrate, succinate and malate, were not significantly altered, suggesting that CMs use different forms of energy production in the absence of FAO (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#Fig3">3c</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#Fig11">6d</a>). We assume that increased glucose oxidation compensates to some extent for the loss of FAO in <i>Cpt1b</i>-deficient CMs to generate acetyl-CoA, as the expression level of PDH1A but not that of its inhibitor PDK4 increases. Likewise, the expression levels of ACSS1 and ACSS2 that generate acetyl-CoA from acetate remained unchanged (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#Fig11">6e</a>).</p><div data-test="figure" data-container-section="figure" id="figure-3" data-title="Impeded mitochondrial import of fatty acid rewires the metabolism of CMs and boosts αKG levels."><figure><figcaption><b id="Fig3" data-test="figure-caption-text">Fig. 3: Impeded mitochondrial import of fatty acid rewires the metabolism of CMs and boosts αKG levels.</b></figcaption><div><div><a data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="https://riskmusings.substack.com/articles/s41586-023-06585-5/figures/3" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41586-023-06585-5/MediaObjects/41586_2023_6585_Fig3_HTML.png?as=webp"/><img aria-describedby="Fig3" src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41586-023-06585-5/MediaObjects/41586_2023_6585_Fig3_HTML.png" alt="figure 3" loading="lazy" width="685" height="244"/></picture></a></div><div data-test="bottom-caption" id="figure-3-desc"><p><b>a</b>, Metabolic flux assay of <i>Ctrl</i><sup><i>Cre</i></sup> and <i>Cpt1b</i><sup><i>cKO</i></sup> hearts perfused with [<sup>13</sup>C]palmitate (<i>n</i> = 5 each). <b>b</b>, Volcano plot showing different metabolites in <i>Ctrl</i><sup><i>Cre</i></sup> and <i>Cpt1b</i><sup><i>cKO</i></sup> hearts (<i>n</i> = 8 each). Symbol colors indicate increased (red), decreased (blue) or unchanged (black) metabolites in <i>Cpt1b</i><sup><i>cKO</i></sup> CMs. Dashed horizontal line indicates the threshold of changed metabolites with a <i>P</i> value of &lt; 0.05; dashed vertical lines indicate a fold change of &gt;2 or &lt;0.5.  <b>c</b>, Quantification of metabolites associated with the Krebs cycle and glycolysis, as well as amino acids, in <i>Cpt1b</i><sup><i>cKO</i></sup> hearts (pyruvate, <i>n</i> = 6 each; all other metabolites, <i>n</i> = 8 each). Dashed outline indicates the change in αKG. <b>d</b>,<b>e</b>, Quantification of BCAA catabolism-associated metabolites (<i>n</i> = 7 for <b>d</b>; <i>n</i> = 8 for <b>e</b>). KIV, α-ketoisovalerate; KMV, α-keto-β-methylvalerate. <b>f</b>,<b>g</b>, Metabolic flux assay of <i>Ctrl</i><sup><i>Cre</i></sup> and <i>Cpt1b</i><sup><i>cKO</i></sup> hearts perfused with [<sup>13</sup>C]glucose (<b>f</b>) or [<sup>13</sup>C]isoleucine (<b>g</b>) (<i>n</i> = 6 each). <b>h</b>, Western blot analysis of enzymes involved in αKG generation and catabolism (<i>n</i> = 3 each). Pan-actin was used as a loading control. For gel source data, see <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#MOESM1">Supplementary Information</a>. <b>i</b>, Enzymatic activity of OGDH in CMs from adult <i>Ctrl</i><sup><i>Cre</i></sup> and <i>Cpt1b</i><sup><i>cKO</i></sup> mice (<i>n</i> = 3 each). <b>j</b>, Comparison of Krebs cycle metabolites in CMs from control hearts (<i>n</i> = 6) and <i>Ogdh</i><sup><i>iKO</i></sup> hearts (<i>n</i> = 5). Dashed outline indicates the change in αKG <b>k</b>, Trichrome staining of heart sections from control and <i>Ogdh</i><sup><i>iKO</i></sup> mice, 3–4 weeks after termination of TAM treatment for <i>Ogdh</i> deletion. RV, right ventricle; LV, left ventricle. Scale bars, 500 μm. <b>l</b>, Quantification of CMs in adult control and <i>Ogdh</i><sup><i>iKO</i></sup> hearts (<i>n</i> = 3 each). <b>m</b>–<b>o</b>, Quantification of Ki67<sup>+</sup> (<b>m</b>), pH3<sup>+</sup> (<b>n</b>), aurora B<sup>+</sup> (<b>o</b>) and sarc-actinin<sup>+</sup> CMs on heart sections from control and <i>Ogdh</i><sup><i>iKO</i></sup> mice (<b>m</b>, control <i>n</i> = 5; <i>Ogdh</i><sup><i>iKO</i></sup> <i>n</i> = 4; <b>n</b> and <b>o</b>, <i>n</i> = 4 each). Error bar represents mean ± s.e.m. <i>n</i> numbers refer to individual mice. Two-tailed, unpaired Student <i>t</i>-tests were used for statistical analysis in <b>a</b>,<b>c</b>–<b>g</b>,<b>i</b>,<b>j</b>,<b>l</b>–<b>o</b>. *<i>P</i> &lt; 0.05, **<i>P</i> &lt; 0.01, ***<i>P</i> &lt; 0.001, ****<i>P</i> &lt; 0.0001.</p><p><a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#MOESM7">Source data</a></p></div></div></figure></div><p>Another potential source of energy in <i>Cpt1b</i>-depleted CMs is amino acids. We detected a major enrichment of metabolites involved in amino acid turnover, including ammonia recycling and urea cycle (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#Fig11">6f</a>). The levels of several amino acids whose intermediates (for example, acetyl-CoA, succinyl-CoA and fumarate) replenish the Krebs cycle were increased as well (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#Fig3">3c</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#Fig11">6g,h</a>). Likewise, we found a marked elevation of the levels of metabolites from branched chain amino acid (BCAA) catabolism in <i>Cpt1b</i>-deficient hearts, including α-ketoisovalerate and α-keto-β-methylvalerate. The levels of propionyl-CoA and methylmalonyl-CoA, which serve as precursors for succinyl-CoA, were also increased (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#Fig3">3d,e</a>). Thus, we reason that enhanced glucose oxidation and BCAA catabolism in <i>Cpt1b-</i>deficient CMs efficiently compensate for impaired metabolic flux of fatty acid-derived acetyl-CoA into the Krebs cycle in <i>Cpt1b</i>-deficient CMs. This conclusion is also supported by the increased metabolic flux rate of <sup>13</sup>C-labelled glucose and isoleucine to metabolites of the Krebs cycle or BCAA catabolism, respectively. Additional evidence comes from Seahorse analysis of <i>Cpt1b</i>-deficient CMs, indicating more efficient utilization of pyruvate, BCAAs and glucose compared to control CMs (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#Fig3">3f,g</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#Fig11">6i,j</a>).</p><p>Notably, we detected a marked, nearly 20-fold, increase of αKG in CMs after inactivation of <i>Cpt1b</i> (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#Fig3">3c</a>). A direct conversion of glutamine and glutamate to αKG seems unlikely to account for the accumulation of αKG, as neither the concentration of glutamine nor that of glutamate declined (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#Fig3">3c</a>). Instead, we observed significantly reduced isocitrate levels, along with an increase of the levels of IDH1, IDH2 and IDH3A, which catalyse conversion of isocitrate to αKG (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#Fig3">3c,h</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#Fig11">6k</a>). In addition, the enzymatic activity of OGDH, the key component of the αKG dehydrogenase complex converting αKG to succinyl-CoA was reduced in <i>Cpt1b</i>-deficient CMs, despite an increase in protein concentrations of OGDH complex components (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#Fig3">3h,i</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#Fig11">6l</a>). The reduction of the enzymatic activity of OGDH is most likely caused by the well-known inhibitory effects of intermediate metabolites from BCAA catabolism, the levels of which are increased in the <i>Cpt1b</i> mutants<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 25" title="Patel, M. S. Inhibition by the branched-chain 2-oxo acids of the 2-oxoglutarate dehydrogenase complex in developing rat and human brain. Biochem. J. 144, 91–97 (1974)." href="https://riskmusings.substack.com/articles/s41586-023-06585-5#ref-CR25" id="ref-link-section-d180729366e2259">25</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 26" title="Andrade, J. et al. Control of endothelial quiescence by FOXO-regulated metabolites. Nat. Cell Biol. 23, 413–423 (2021)." href="https://riskmusings.substack.com/articles/s41586-023-06585-5#ref-CR26" id="ref-link-section-d180729366e2262">26</a></sup>. Inactivation of <i>Cpt1b</i> and αKG accumulation did not alter protein levels of other Krebs cycle enzymes or mitochondrial DNA content (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#Fig11">6m,n</a>), indicating that FAO-independent functions of mitochondria were not compromised. Taken together, the results indicate that elevated synthesis and reduced metabolization synergize to accumulate αKG in CMs after abrogation of FAO.</p><p>We did not observe significant differences in the concentrations of diacylglycerols, triacylglycerols and very long-chain acyl-CoA in <i>Cpt1b</i>-deficient compared to control CMs (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#Fig12">7a–d</a>). We reason that the strong decline of <i>Lpl</i> expression in <i>Cpt1b</i>-deficient CMs prevents aberrant myocardial lipid accumulation and its sequelae (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#Fig12">7e</a>). To further substantiate the causality between αKG accumulation and CM reprogramming, we inactivated <i>Ogdh</i> in adult CMs using <i>αMHC-MerCreMer</i><sup><i>pos/+</i></sup><i>Ogdh</i><sup><i>fl/fl</i></sup> (<i>Ogdh</i><sup><i>iKO</i></sup>) mice and <i>αMHC-MerCreMer</i><sup><i>pos/+</i></sup><i>Ogdh</i><sup><i>+/+</i></sup> as controls, which—as expected—results in substantial accumulation of αKG (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#Fig3">3j</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#Fig12">7f–h</a>). Notably, CM-specific inactivation of <i>Ogdh</i> essentially recapitulated the phenotype observed in <i>Cpt1b</i>-knockout animals, including the massive increase in CM numbers, increased proliferation of CMs and reactivation of genes characteristic of immature CMs (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#Fig3">3k–o</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#Fig12">7i–m</a>). However, in contrast to <i>Cpt1b</i>-deficient mice, <i>Ogdh</i> mutants die between 5 and 8 weeks after gene inactivation, which makes them unsuitable for cardiac regeneration studies.</p></div></div></section><section data-title="Reduced H3K4me3 in CM identity genes"><div id="Sec6-section"><h2 id="Sec6">Reduced H3K4me3 in CM identity genes</h2><div id="Sec6-content"><p>αKG is an essential cofactor for histone demethylases. Therefore, we wondered whether accumulation of αKG in <i>Cpt1b</i>-deficient CMs affects histone methylation. Analysis of different histone lysine methylation modifications using cardiac nuclei sorted by fluorescence-activated cell sorting (FACS) revealed only a reduction in the level of H3K4me3, whereas the levels of 11 other histone lysine methylation modifications were not reduced. By contrast, the levels of heterochromatin markers such as H3K9me3, H3K27me3 and H4K20me2 were markedly increased after inactivation of <i>Cpt1b</i>, probably owing to secondary effects associated with the shift of CMs to a more immature state (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#Fig4">4a</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#Fig13">8a</a>). Consistently, H3K4me3 levels are decreased in <i>Ogdh</i>-deficient CMs, which also exhibited increased αKG levels (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#Fig13">8b</a>). Expression levels of enzymes determining H3K4me3 levels, including methyltransferases (that is, <i>Kmt2</i> family, <i>Ash1l</i>, <i>Prdm9</i> and <i>Smyd</i> family) and αKG-dependent H3K4me3 demethylases (that is, <i>Kdm5a-d</i>, <i>Kdm2b</i> and <i>Riox1</i>), did not change after inactivation of <i>Cpt1b</i>. Neither did we detect a change in the intracellular levels of <i>S</i>-adenosyl methionine, the methyl-group donor for histone methylation. Therefore, we conclude that αKG boosts the activity of a H3K4me3-specific demethylase, eventually leading to reduction of H3K4me3 levels (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#Fig13">8c,d</a>).</p><div data-test="figure" data-container-section="figure" id="figure-4" data-title="Increased αKG levels induce H3K4me3 demethylation and decrease expression of CM maturity genes."><figure><figcaption><b id="Fig4" data-test="figure-caption-text">Fig. 4: Increased αKG levels induce H3K4me3 demethylation and decrease expression of CM maturity genes.</b></figcaption><div><div><a data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="https://riskmusings.substack.com/articles/s41586-023-06585-5/figures/4" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41586-023-06585-5/MediaObjects/41586_2023_6585_Fig4_HTML.png?as=webp"/><img aria-describedby="Fig4" src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41586-023-06585-5/MediaObjects/41586_2023_6585_Fig4_HTML.png" alt="figure 4" loading="lazy" width="685" height="594"/></picture></a></div><div data-test="bottom-caption" id="figure-4-desc"><p><b>a</b>, Western blot analysis of histone methylation modifications using FACS-isolated CM nuclei from <i>Ctrl</i><sup><i>Cre</i></sup> and <i>Cpt1b</i><sup><i>cKO</i></sup> mice (<i>n</i> = 3 mice each). Histone 3 (H3) was used as a loading control. For gel source data, see <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#MOESM1">Supplementary Information</a>. <b>b</b>, Coverage plots of H3K4me3 ChIP–seq signals within genes in three different groups categorized according to the breadth of H3K4me3 peaks in <i>Ctrl</i><sup><i>Cre</i></sup> CMs. The 25% of genes with the broadest peaks were placed in the ‘broad’ group, the 25% of genes with the narrowest peaks were placed in the ‘narrow’ group, and the ‘medium’ group contains all other genes (25%–75%). RPM, reads per million mapped reads; TSS, transcription start site; TES, transcription end site; kb, kilobases. <b>c</b>, Box plots showing DESeq-normalized expression levels of genes with broad, medium and narrow H3K4me3 peaks in <i>Ctrl</i><sup><i>Cre</i></sup> CMs. The box plot displays data from minimum to maximum, the median (centre line), 25th (bottom line) and 75th (top line) percentiles. One-way ANOVA analysis with multiple testing correction. The false discovery rate was controlled by using the two-stage step-up method of Benjamini, Krieger and Yekutieli. Broad, <i>n</i> = 2,387 genes; medium, <i>n</i> = 4,775 genes; narrow, <i>n</i> = 2,387 genes. <b>d</b>, Venn diagram showing the overlap between genes with reduced expression and genes with reduced H3K4me3 deposition in <i>Cpt1b</i><sup><i>cKO</i></sup> compared to <i>Ctrl</i><sup><i>Cre</i></sup> CMs. Distribution of overlapping peaks in the broad, medium and narrow groups is shown in the pie chart. <b>e</b>, Analysis of top GO terms from overlapping differentially expressed genes using the David tool (<i>n</i> = 3 mice each). <b>f</b>, Genome browser snapshots demonstrating reduced H3K4me3 deposition of representative genes associated with maturation of CMs in <i>Cpt1b</i><sup><i>cKO</i></sup> CMs (normalized to mapped reads). RPKM, reads per kilobase per million mapped reads.</p><p><a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#MOESM8">Source data</a></p></div></div></figure></div><p>Chromatin immunoprecipitation and sequencing (ChIP–seq) analysis of H3K4me3 using FACS-sorted CM nuclei to map gene regions with altered H3K4me3 identified 467 genes with increased and 478 genes with reduced H3K4me3 peaks in <i>Cpt1b</i>-deficient CMs compared to control CMs. Genes with increased H3K4me3 peaks were mainly associated with long-chain fatty acid transport and lipid and fatty acid metabolic processes, reflecting a potential feedback mechanism in response to abrogation of FAO. Notably, genes with reduced H3K4me3 peaks were mainly CM identity genes, involved in pathways regulating cardiac maturation and function (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#Fig13">8e,f</a>). As the breadth of H3K4me3 peaks is tightly linked to transcriptional strength of cell identity genes in various cell types and tissues, including the heart<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 14" title="Chen, K. et al. Broad H3K4me3 is associated with increased transcription elongation and enhancer activity at tumor-suppressor genes. Nat. Genet. 47, 1149–1157 (2015)." href="https://riskmusings.substack.com/articles/s41586-023-06585-5#ref-CR14" id="ref-link-section-d180729366e2521">14</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 27" title="Sze, C. C. et al. Coordinated regulation of cellular identity-associated H3K4me3 breadth by the COMPASS family. Sci. Adv. 6, eaaz4764 (2020)." href="https://riskmusings.substack.com/articles/s41586-023-06585-5#ref-CR27" id="ref-link-section-d180729366e2524">27</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 28" title="Benayoun, B. A. et al. H3K4me3 breadth is linked to cell identity and transcriptional consistency. Cell 158, 673–688 (2014)." href="https://riskmusings.substack.com/articles/s41586-023-06585-5#ref-CR28" id="ref-link-section-d180729366e2527">28</a></sup>, we categorized genes into three groups according to the width of the H3K4me3 peaks. Genes showing the 25% broadest peaks exhibited the highest transcriptional activity and were preferentially associated with CM maturation and function (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#Fig4">4b,c</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#Fig13">8g</a>). Integrated analysis of RNA-seq and H3K4me3 ChIP–seq datasets indicated that 31.6% (151) of genes with reduced H3K4me3 peaks were transcriptionally repressed. A total of 95 genes (62.91%) in this group contained broad H3K4me3 peaks and 44 genes (29.14%) contained medium H3K4me3 peaks. By contrast, only a few genes with narrow H3K4me3 peaks (7.95%) showed reduced expression in <i>Cpt1b</i>-deficient CMs (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#Fig4">4d</a>). Reduction of H3K4me3 deposition in genes with both reduced expression and reduced H3K4me3 peaks was found to be more pronounced inside gene bodies compared to transcription start site regions (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#Fig13">8h</a>). Strikingly, such genes were mostly associated with cardiac muscle contraction, CM differentiation and maturation, including <i>Mylk3</i>, <i>Cacna1g</i> and <i>Myocd</i> (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#Fig4">4e,f</a>). We validated these findings by ChIP–qPCR, demonstrating that H3K4me3 enrichment was broadly diminished in both promoters and gene bodies of genes associated with CM maturation, whereas H3K9me3 and H3K27me3 levels at promoters remained unchanged (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#Fig13">8i–k</a>). Taking these findings together, we conclude that accumulation of αKG activates a H3K4me3-specific demethylase, which subsequently erases H3K4me3 within genes required for cardiac differentiation and maturation, thereby converting <i>Cpt1b</i>-deficient CMs into a more immature, proliferation-competent state.</p></div></div></section><section data-title="Activated KDM5 stimulates proliferation"><div id="Sec7-section"><h2 id="Sec7">Activated KDM5 stimulates proliferation</h2><div id="Sec7-content"><p>To confirm that accumulation of αKG is a critical signal to prevent CM maturation and enable proliferation, we treated P0–1 neonatal CMs for 4 days with cell-permeable αKG. We observed a marked increase of Ki67<sup>+</sup>cTnT<sup>+</sup> and pH3(Ser10)<sup>+</sup>cTnT<sup>+</sup> CMs, consistent with RNA-seq data indicating enrichment of GO terms related to cell divisions in αKG-treated cells (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#Fig5">5a–c</a>). Furthermore, the level of expression of <i>Nppa</i>, <i>Nppb</i>, <i>Acta1</i> and <i>Myh7</i>, genes that are predominantly expressed during fetal and neonatal stages, was strongly elevated, whereas the level of expression of genes associated with CM maturation and differentiation, including <i>Tnni3</i>, <i>Mylk3</i> and <i>Myocd</i>, was reduced (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#Fig5">5c</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#Fig14">9a</a>). Treatment with αKG also diminished H3K4me3 but did not affect H3K9 and H4K20 methylation (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#Fig5">5d</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#Fig14">9b</a>). In line with this finding, overexpression of enzymes in CMs that increase αKG levels, such as those encoded by <i>Idh3b</i> and <i>Idh3g</i>, stimulated CM proliferation, markedly reduced H3K4me3 deposition and lowered mRNA levels of cardiac maturation-related genes (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#Fig14">9c–h</a>). Treatment of neonatal CMs with CPI-455, a specific inhibitor of the αKG-dependent H3K4 demethylase KDM5, or cell-permeable R2HG, a competitive inhibitor of αKG<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 29" title="Intlekofer, A. M. et al. Hypoxia induces production of L-2-hydroxyglutarate. Cell Metab 22, 304–311 (2015)." href="https://riskmusings.substack.com/articles/s41586-023-06585-5#ref-CR29" id="ref-link-section-d180729366e2630">29</a></sup>, prevented enhanced cell cycle activity induced by αKG or overexpression of <i>Idh3b</i> or <i>Idh3g</i> (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#Fig5">5e,f</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#Fig14">9g–k</a>). As expected, treatment with CPI-455 altered the expression of cardiac maturation-related genes accordingly (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#Fig14">9l</a>).</p><div data-test="figure" data-container-section="figure" id="figure-5" data-title="Accumulation of αKG stimulates KDM5 activity, attenuates maturation and enhances proliferation of CMs."><figure><figcaption><b id="Fig5" data-test="figure-caption-text">Fig. 5: Accumulation of αKG stimulates KDM5 activity, attenuates maturation and enhances proliferation of CMs.</b></figcaption><div><div><a data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="https://riskmusings.substack.com/articles/s41586-023-06585-5/figures/5" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41586-023-06585-5/MediaObjects/41586_2023_6585_Fig5_HTML.png?as=webp"/><img aria-describedby="Fig5" src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41586-023-06585-5/MediaObjects/41586_2023_6585_Fig5_HTML.png" alt="figure 5" loading="lazy" width="685" height="302"/></picture></a></div><div data-test="bottom-caption" id="figure-5-desc"><p><b>a</b>,<b>b</b>, Immunofluorescence images and quantification of Ki67 (<i>n</i> = 5 for <b>a</b>) and pH3(Ser10) (<i>n</i> = 3 for <b>b</b>) signals in cTnT<sup>+</sup> neonatal CMs (P0–1) after 4-day culture in the presence of dimethylsulfoxide (DMSO) or (cell permeable) αKG. Top right corner, enlargement of area outlined in main image. Scale bars, 50 μm. <b>c</b>, GO-term enrichment of differentially expressed genes in DMSO- and αKG-treated neonatal CMs. Upregulated genes are in red, and downregulated genes are in blue. <b>d</b>, Western blot analysis of histone methylation modifications in P0–1 neonatal CMs treated with DMSO or αKG. H3 was used as a loading control. <b>e</b>, Top panels: western blot analysis of H3K4me3 in P0–1 CMs after 3-day culture in the presence of DMSO, αKG, CPI and αKG combined with CPI. H3 was used as a loading control. Lower panel: quantification of H3K4me3 levels (DMSO and αKG, <i>n</i> = 4 each; CPI and αKG + CPI, <i>n</i> = 3 each). <b>f</b>, Quantification of Ki67 signals in cTnT<sup>+</sup> P0–1 CMs treated with DMSO, αKG, CPI and αKG combined with CPI (<i>n</i> = 6 each). <b>g</b>, Immunofluorescence micrographs and quantification of Ki67 signals in cTnT<sup>+</sup> P0–1 CMs after lentiviral transduction of <i>Kdm5b</i> (<i>n</i> = 5 each). OE, overexpression. Top right corner, enlargement of area outlined in main image. Scale bars, 50 μm. <b>h</b>, Quantification of Ki67 signals in sarc-actinin<sup>+</sup> neonatal CMs (P0–1), 4 days after lentiviral transduction of <i>Kdm5b</i> with and without αKG (DMSO, αKG, <i>Kdm5b</i> OE, <i>n</i> = 4; αKG + <i>Kdm5b</i> OE, <i>n</i> = 3). <b>i</b>, Quantification of Ki67 signals in sarc-actinin<sup>+</sup> neonatal CMs (P0–1), 4 days after knockdown (KD) of <i>Kdm5b</i>, with and without αKG (DMSO, αKG + <i>Kdm5b</i> KD, <i>Kdm5b</i> KD, <i>n</i> = 4; αKG, <i>n</i> = 3). <b>j</b>, ChIP–qPCR of KDM5B in genes with broad and narrow H3K4me3 peaks in P0–1 CMs (<i>n</i> = 3). bp, base pairs. Error bars show mean ± s.e.m. Two-tailed, unpaired Student <i>t</i>-tests were used for statistical analysis in <b>a</b>,<b>b</b>,<b>g</b>,<b>j</b>. One-way ANOVA with Tukey tests was used for correction of multiple comparisons in <b>e</b>,<b>f</b>,<b>h</b>,<b>i</b>. *<i>P</i> &lt; 0.05, **<i>P</i> &lt; 0.01, ***<i>P</i> &lt; 0.001, ****<i>P</i> &lt; 0.0001. <i>n</i> numbers refer to independent experiments. For gel source data in <b>d</b>,<b>e</b>, see <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#MOESM1">Supplementary Information</a>.</p><p><a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#MOESM9">Source data</a></p></div></div></figure></div><p>Pharmacological inhibition of αKG-dependent effects by the KDM5 inhibitor CPI-455 suggested that effects of αKG on the chromatin and on gene expression are relayed by members of the <i>Kdm5</i> gene family. This hypothesis was confirmed by overexpression of <i>Kdm5b</i>, which strongly reduced H3K4me3 levels and markedly increased CM cell cycle activity when combined with αKG treatment (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#Fig5">5g,h</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#Fig15">10a–d</a>). <i>Kdm5b</i> plays important roles during heart development, and its expression progressively declines during CM maturation<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 30" title="Zaidi, S. et al. De novo mutations in histone-modifying genes in congenital heart disease. Nature 498, 220–223 (2013)." href="https://riskmusings.substack.com/articles/s41586-023-06585-5#ref-CR30" id="ref-link-section-d180729366e2851">30</a></sup>. <i>Kdm5b</i> knockdown mediated by short interfering RNA efficiently antagonized pro-proliferative effects of αKG, fully restoring reduced H3K4me3 levels that were lowered by αKG treatment, and thereby increasing expression of several key genes associated with CM maturation (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#Fig5">5i</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#Fig15">10e–g</a>). To validate potential targets of KDM5B, we carried out ChIP–qPCR experiments. We found that KDM5B preferentially binds to genes with broad H3K4me3 domains involved in maintaining the mature phenotype of CMs, such as <i>Tnni3</i>, <i>Mylk3</i> and <i>Myocd</i>. By contrast, no enrichment of KDM5B was found on housekeeping genes characterized by narrow peaks, such as <i>Dus1l</i>, <i>Snx19</i> and <i>Lsm6</i>. Notably, mRNA levels of these housekeeping genes were not altered in response to αKG, <i>Kdm5b</i> knockdown or combined treatment (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#Fig5">5j</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#Fig15">10h</a>). Taken together, our results indicate that accumulation of αKG caused by <i>Cpt1b</i> inactivation activates KDM5, which lowers broad H3K4me3 peaks in genes required for CM maturation, thereby reverting CMs to a more immature state and enabling CM cell cycle activity (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#Fig15">10i</a>).</p></div></div></section><section data-title="Discussion"><div id="Sec8-section"><h2 id="Sec8">Discussion</h2><div id="Sec8-content"><p>Here we demonstrate that the cellular metabolism directly regulates H3K4me3 histone modifications at CM identity genes, thereby playing a pivotal role in controlling the maturation and proliferation of CMs. Metabolism-induced partial rewinding of the maturation program not only increases the proliferative capacity of CMs but also leads to various cardioprotective changes, including increased activation of adaptive HIF1 signalling<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 31" title="Huang, C. et al. SENP3 is responsible for HIF-1 transactivation under mild oxidative stress via p300 de-SUMOylation. EMBO J. 28, 2748–2762 (2009)." href="https://riskmusings.substack.com/articles/s41586-023-06585-5#ref-CR31" id="ref-link-section-d180729366e2907">31</a></sup> and αKG-dependent DNA damage repair<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 32" title="Sundheim, O. et al. Human ABH3 structure and key residues for oxidative demethylation to reverse DNA/RNA damage. EMBO J. 25, 3389–3397 (2006)." href="https://riskmusings.substack.com/articles/s41586-023-06585-5#ref-CR32" id="ref-link-section-d180729366e2911">32</a></sup>. Increased HIF1 signalling may also contribute to the relatively modest increase of CM surface area in <i>Cpt1b</i><sup><i>cKO</i></sup> mice<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 33" title="Chu, W. et al. Mild hypoxia-induced cardiomyocyte hypertrophy via up-regulation of HIF-1α-mediated TRPC signalling. J. Cell. Mol. Med. 16, 2022–2034 (2012)." href="https://riskmusings.substack.com/articles/s41586-023-06585-5#ref-CR33" id="ref-link-section-d180729366e2921">33</a></sup>, which was limited to larger CMs (&gt; 300 μm<sup>2</sup>). <i>Cpt1b</i>-deficient CMs are characterized by reduced sarcomere density, a hallmark of neonatal CMs, which are able to proliferate but also support cardiac contractility. However, we do not know whether all CMs in <i>Cpt1b</i>-deficient hearts show the same propensity for cytokinesis, or whether only a distinct subset of CMs undergo cell division, which show a more severe disassembly of sarcomeres that cannot be easily identified on tissues sections. Furthermore, we observed improved coupling of glucose oxidation to glycolysis, which will reduce proton production and Ca<sup>2+</sup> overload during ischaemia, preventing mitochondrial damage and cell death especially during reperfusion when the influx of O<sub>2</sub> stimulates FAO <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 34" title="Peoples, J. N., Saraf, A., Ghazal, N., Pham, T. T. &amp; Kwong, J. Q. Mitochondrial dysfunction and oxidative stress in heart disease. Exp. Mol. Med. 51, 1–13 (2019)." href="https://riskmusings.substack.com/articles/s41586-023-06585-5#ref-CR34" id="ref-link-section-d180729366e2938">34</a></sup>. Thus, stimulation of both regeneration and cardioprotection seem to contribute to reduced scar formation in <i>Cpt1b</i>-deficient CMs after I–R injury. However, as inactivation of <i>Cpt1b</i> in CMs after I–R injury facilitates heart repair and restored cardiac functions—processes that cannot be explained by cardioprotection—we postulate that induction of cardiac regeneration dominates, a hypothesis that is also supported by the overall increase in CM numbers and in proliferating CMs in the infarct border zone.</p><p>The metabolic rewiring of CMs after abrogation of FAO causes complex transcriptional changes, which partially rewind the developmental program and endow CMs with a renewed proliferative potential. Metabolic rewiring does not only increase the flux from glucose, but also strongly increases the concentrations of catabolic intermediates generated during conversion of BCAAs to acetyl-CoA and/or succinyl-CoA, which fuel the Krebs cycle during anoxia and ischaemia<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 35" title="Drake, K. J., Sidorov, V. Y., McGuinness, O. P., Wasserman, D. H. &amp; Wikswo, J. P. Amino acids as metabolic substrates during cardiac ischemia. Exp. Biol. Med. 237, 1369–1378 (2012)." href="https://riskmusings.substack.com/articles/s41586-023-06585-5#ref-CR35" id="ref-link-section-d180729366e2952">35</a></sup>. Essentially, <i>Cpt1b</i>-deficient CMs adopt a metabolic profile that facilitates utilization of alternative energy substrates such as amino acids. We did not detect signs of lipotoxicity in <i>Cpt1b</i><sup><i>cKO</i></sup> and <i>Cpt1b</i><sup><i>iKO</i></sup> hearts, which has been reported for other <i>Cpt1b</i>-mutant mouse lines<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Ji, S. et al. Homozygous carnitine palmitoyltransferase 1b (muscle isoform) deficiency is lethal in the mouse. Mol. Genet. Metab. 93, 314–322 (2008)." href="#ref-CR36" id="ref-link-section-d180729366e2975">36</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="He, L. et al. Carnitine palmitoyltransferase-1b deficiency aggravates pressure overload-induced cardiac hypertrophy caused by lipotoxicity. Circulation 126, 1705–1716 (2012)." href="#ref-CR37" id="ref-link-section-d180729366e2975_1">37</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 38" title="Haynie, K. R., Vandanmagsar, B., Wicks, S. E., Zhang, J. &amp; Mynatt, R. L. Inhibition of carnitine palymitoyltransferase1b induces cardiac hypertrophy and mortality in mice. Diabetes Obes. Metab. 16, 757–760 (2014)." href="https://riskmusings.substack.com/articles/s41586-023-06585-5#ref-CR38" id="ref-link-section-d180729366e2978">38</a></sup>. The absence of lipotoxicity might be explained by the catabolization of fatty acids in skeletal muscles, which is not affected in our cardiac-specific models, preventing accumulation of circulating free fatty acids and/or lipoproteins. Moreover, lowered expression levels of <i>Lpl</i> in hearts of <i>Cpt1b</i><sup><i>cKO</i></sup> mice will prevent efficient uptake of fatty acids into CMs owing to reduced hydrolysation of triacylglycerols in lipoproteins<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 39" title="Yagyu, H. et al. Lipoprotein lipase (LpL) on the surface of cardiomyocytes increases lipid uptake and produces a cardiomyopathy. J. Clin. Invest. 111, 419–426 (2003)." href="https://riskmusings.substack.com/articles/s41586-023-06585-5#ref-CR39" id="ref-link-section-d180729366e2991">39</a></sup>.</p><p>The marked, nearly twenty-fold accumulation of αKG is a hallmark of metabolic rewiring in <i>Cpt1b</i>-deficient CMs. In contrast to αKG, concentrations of other Krebs cycle metabolites were only marginally altered. αKG plays a crucial role in the Krebs cycle but also links metabolism to regulation of gene expression by serving as an essential co-substrate for αKG-dependent dioxygenases, including histone and DNA demethylases<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 40" title="Losman, J. A., Koivunen, P. &amp; Kaelin, W. G. Jr 2-Oxoglutarate-dependent dioxygenases in cancer. Nat. Rev. Cancer 20, 710–726 (2020)." href="https://riskmusings.substack.com/articles/s41586-023-06585-5#ref-CR40" id="ref-link-section-d180729366e3001">40</a></sup>. Our data indicate that the increase of αKG activates KDM5, leading to demethylation of H3K4me3 in cardiac identity genes that are required to maintain CMs in a mature state. H3K4me3 is a well-studied histone modification characteristic of actively transcribed genes. H3K4me3 shows a pronounced peak immediately downstream of the transcription start site, which can extend for more than 40 kilobases, probably contributing to increased PolII pausing and release, elongation and transcriptional strength<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 14" title="Chen, K. et al. Broad H3K4me3 is associated with increased transcription elongation and enhancer activity at tumor-suppressor genes. Nat. Genet. 47, 1149–1157 (2015)." href="https://riskmusings.substack.com/articles/s41586-023-06585-5#ref-CR14" id="ref-link-section-d180729366e3005">14</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 17" title="Pokholok, D. K. et al. Genome-wide map of nucleosome acetylation and methylation in yeast. Cell 122, 517–527 (2005)." href="https://riskmusings.substack.com/articles/s41586-023-06585-5#ref-CR17" id="ref-link-section-d180729366e3008">17</a></sup>. Consistent with other studies<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 15" title="Howe, F. S., Fischl, H., Murray, S. C. &amp; Mellor, J. Is H3K4me3 instructive for transcription activation? Bioessays 39, e201600095 (2017)." href="https://riskmusings.substack.com/articles/s41586-023-06585-5#ref-CR15" id="ref-link-section-d180729366e3012">15</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 16" title="Gilsbach, R. et al. Distinct epigenetic programs regulate cardiac myocyte development and disease in the human heart in vivo. Nat. Commun. 9, 391 (2018)." href="https://riskmusings.substack.com/articles/s41586-023-06585-5#ref-CR16" id="ref-link-section-d180729366e3015">16</a></sup>, our findings show that genes with the broadest H3K4me3 peaks (top 25%) are transcriptionally more active than genes with narrow peaks and critical for cellular identity and maturation<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 14" title="Chen, K. et al. Broad H3K4me3 is associated with increased transcription elongation and enhancer activity at tumor-suppressor genes. Nat. Genet. 47, 1149–1157 (2015)." href="https://riskmusings.substack.com/articles/s41586-023-06585-5#ref-CR14" id="ref-link-section-d180729366e3019">14</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 27" title="Sze, C. C. et al. Coordinated regulation of cellular identity-associated H3K4me3 breadth by the COMPASS family. Sci. Adv. 6, eaaz4764 (2020)." href="https://riskmusings.substack.com/articles/s41586-023-06585-5#ref-CR27" id="ref-link-section-d180729366e3022">27</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 28" title="Benayoun, B. A. et al. H3K4me3 breadth is linked to cell identity and transcriptional consistency. Cell 158, 673–688 (2014)." href="https://riskmusings.substack.com/articles/s41586-023-06585-5#ref-CR28" id="ref-link-section-d180729366e3025">28</a></sup>. Recent studies questioned whether H3K4me3 is an instructive signal for gene activation, as reduction of H3K4me3 has a limited impact on global gene expression in yeast and mammalian cells<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 15" title="Howe, F. S., Fischl, H., Murray, S. C. &amp; Mellor, J. Is H3K4me3 instructive for transcription activation? Bioessays 39, e201600095 (2017)." href="https://riskmusings.substack.com/articles/s41586-023-06585-5#ref-CR15" id="ref-link-section-d180729366e3030">15</a></sup>. However, numerous studies described an active role of H3K4me3 in gene expression (for example, by recruiting the NURF complex or by interacting with TAF3)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Cano-Rodriguez, D. et al. Writing of H3K4Me3 overcomes epigenetic silencing in a sustained but context-dependent manner. Nat. Commun. 7, 12284 (2016)." href="#ref-CR41" id="ref-link-section-d180729366e3034">41</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Wysocka, J. et al. A PHD finger of NURF couples histone H3 lysine 4 trimethylation with chromatin remodelling. Nature 442, 86–90 (2006)." href="#ref-CR42" id="ref-link-section-d180729366e3034_1">42</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 43" title="Lauberth, S. M. et al. H3K4me3 interactions with TAF3 regulate preinitiation complex assembly and selective gene activation. Cell 152, 1021–1036 (2013)." href="https://riskmusings.substack.com/articles/s41586-023-06585-5#ref-CR43" id="ref-link-section-d180729366e3037">43</a></sup>. In line with these reports, we identified 151 genes in <i>Cpt1b</i>-mutant CMs with reduced H3K4me3 peaks and diminished gene transcription. Of note, 92% of these genes contain broad or medium peaks and are mainly associated with cardiac development and contraction. We reason that cell-type-specific genes with broad H3K4me3 peaks are preferentially targeted by an αKG-dependent H3K4me3 demethylase. On the basis of inhibitor and overexpression studies, we reason that a member of the KDM5 family, most likely KDM5B, is the responsible H3K4me3 demethylase<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 30" title="Zaidi, S. et al. De novo mutations in histone-modifying genes in congenital heart disease. Nature 498, 220–223 (2013)." href="https://riskmusings.substack.com/articles/s41586-023-06585-5#ref-CR30" id="ref-link-section-d180729366e3044">30</a></sup>. Future studies will unveil how precisely the αKG-dependent stimulation of a H3K4me3 demethylase achieves target selectivity and preferentially deactivates genes associated with cardiac development and contraction. We assume that genes containing broad H3K4me3 peaks provide more docking sites for KDM5 and therefore represent privileged targets. Genes with enhanced recruitment of KDM5 will automatically be more responsive to an increase of αKG and KDM5-mediated silencing. However, it is also possible that maturation-associated genes with broad H3K4me3 peaks possess other distinct features promoting enhanced recruitment of KDM5 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 44" title="Brier, A. B. et al. The KDM5 family is required for activation of pro-proliferative cell cycle genes during adipocyte differentiation. Nucleic Acids Res. 45, 1743–1759 (2017)." href="https://riskmusings.substack.com/articles/s41586-023-06585-5#ref-CR44" id="ref-link-section-d180729366e3048">44</a></sup>).</p><p>Overall, the observation that a single metabolite, αKG, serves a pivotal role in connecting metabolic processes to transcriptional changes mediated by KDM5 offers numerous opportunities to manipulate the fate and function of CMs and stimulate cardiac regeneration.</p></div></div></section><section data-title="Methods"><div id="Sec9-section"><h2 id="Sec9">Methods</h2><div id="Sec9-content"><h3 id="Sec10">Animals</h3><p><i>Cpt1b</i><sup><i>fl/fl</i></sup> mice were generated in-house by using a targeting vector purchased from the European Conditional Mouse Mutagenesis Program, in which exons 10–11 of the <i>Cpt1b</i> gene are flanked by two <i>loxP</i> sites. Generation of <i>Ogdh</i><sup><i>fl/fl</i></sup> animals, in which exons 3 and 4 were flanked by <i>loxP</i> sites, have been described before<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 26" title="Andrade, J. et al. Control of endothelial quiescence by FOXO-regulated metabolites. Nat. Cell Biol. 23, 413–423 (2021)." href="https://riskmusings.substack.com/articles/s41586-023-06585-5#ref-CR26" id="ref-link-section-d180729366e3088">26</a></sup>. <i>αMHC-Cre</i><sup><i>Pos/+</i></sup> and <i>αMHC-MCM</i><sup><i>pos/+</i></sup> mice were obtained from The Jackson Laboratory. C57BL/6 mice were obtained from Charles River. Primers used for genotyping are listed in the <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#MOESM1">Supplementary Information</a>. All mice were maintained in individually ventilated cages, at 22.5 °C ± 1 °C and a relative humidity of 50% ± 5% with controlled illumination (12 h dark/light cycle). Mice were given ad libitum access to food and water. TAM (Sigma) was administered intraperitoneally at 75 mg per kilogram of BW daily for 10 days. In experiments, in which genes were inactivated by Cre recombinase-mediated recombination, corresponding Cre recombinase-expressing strains without the floxed target genes were always used as negative controls, unless indicated otherwise. <i>αMHC-MCM</i> control mice were subjected to the same TAM treatment as in the actual gene inactivation experiment. All mice were maintained on a C57BL/6 background, and littermates were used as controls in all experiments. Animals were assigned to different groups according to genotypes. The genotype of animals, from which individual samples were taken, was not known to the investigator, and experiments were performed in a blinded manner. After data collection, individual genotypes were revealed and the animals were assigned to separate groups for further data analysis. Observed results did not differ between male mice and female mice. No sex-specific experiments were performed. Sample sizes were determined on the basis of established practice and applicable standards. We opted for sample sizes that are commonly used sample sizes in the field. For in vivo studies, a minimum of three biological replicates were analyzed. All animal experiments were carried out in accordance with the Guide for the Care and Use of Laboratory Animals published by the US National Institute of Health (NIH Publication No. 85-23, revised 1996) and according to the regulations issued by the Committee for Animal Rights Protection of the State of Hessen (Regierungspraesidium Darmstadt, Wilhelminenstr. 1–3, 64283 Darmstadt, Germany) with the project numbers B2/1137, B2/1125 and B2/2034.</p><h3 id="Sec11">Neonatal CM isolation and culture in vitro</h3><p>Neonatal hearts were dissected from P0–1 C57bl/6 pups, washed with ice-cold PBS and dissociated using standard procedures. Neonatal CMs were seeded in culture plates precoated with fibronectin (0.8–1 million per 3.5-cm dish or 0.25 to 0.4 million per well of 2-well chamber slides) and cultured in primary neonatal CM culture medium (80% DMEM with 4.5 g l<sup>−1</sup> glucose, 20% Medium 199, 5% FCS and 100 U ml<sup>−1</sup> penicillin and streptomycin). After overnight culture, chemicals were added to the medium and cells were further cultured for 72–96 h before collection. Concentrations of chemical were as follows: etomoxir 100 μM (Cayman, 11969); octyl-αKG 500 μM (Cayman, 11970); CPI-HCl 25 μM (Selleckchem, S8287); R2HG 500 μM (Cayman, 16366). <i>Kdm5b</i> knockdown was achieved through transfection of pooled <i>Kdm5b</i> short interfering RNA (Dharmacon) with DharmaFECT 1 Transfection Reagent. Non-targeting pooled short interfering RNAs were used for the control group.</p><h3 id="Sec12">Immunofluorescence and histological analysis</h3><p>Hearts were immediately fixed in 4% PFA after dissection. For trichrome staining after I–R injury, hearts were embedded in paraffin and continuously sectioned from the apex to the ligation site. Every second section from hearts of each group was used for staining and quantification. In vitro-cultured neonatal CMs were fixed with 4% PFA for 10 min at room temperature and permeabilized (0.3% Triton X-100 and 5% BSA) for 1 h at room temperature. To determine the surface area of CMs, approximately 120 CMs randomly selected from 5–6 paraffin sections of each heart sample were measured using the ImageJ software tool. The sarcomere density of isolated adult CMs was analysed using the ImageJ software tool. For other quantification procedures, using either isolated CMs or tissue sections, hundreds of CMs were analysed for each individual heart. Values from each group of CMs (or sections) were averaged and are presented as one sample (<i>n</i> = 1). For experiments with neonatal CMs, values for each sample represent the results obtained from one isolation of CMs from pooled neonatal hearts. Antibodies for immunofluorescence staining are listed in the <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#MOESM1">Supplementary Information</a>. Microscopic images were acquired with a fluorescence stereomicroscope (Leica M205 FA). Regular immunofluorescence images were acquired with a fluorescence microscope (Zeiss Imager Z1) and processed with ZEN 2 imaging software. Confocal immunofluorescence images were acquired with a Leica SP8 confocal microscope and processed with LAS X software 3.5.7.23225. Acquisition of histological images was carried out with a light microscope (Zeiss Axioplan2).</p><h3 id="Sec13">EdU incorporation assay</h3><p>EdU and other reagents were prepared according to the manufacturer’s instructions (ThermoFisher C10339). In vivo EdU incorporation assays were carried out according to previous publications<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 45" title="Richardson, G. D. Simultaneous assessment of cardiomyocyte DNA synthesis and ploidy: a method to assist quantification of cardiomyocyte regeneration and turnover. J. Vis. Exp. 
                https://doi.org/10.3791/53979
                
               (2016)." href="https://riskmusings.substack.com/articles/s41586-023-06585-5#ref-CR45" id="ref-link-section-d180729366e3150">45</a></sup>. To analyse EdU incorporation in cultured neonatal CMs, cells in 2-well chamber slides were labelled with 10 µM EdU for 12 h. After two washes with pre-warmed PBS, cells were fixed with 4% PFA for 10 min at room temperature and EdU incorporation was visualized using the Click-iT EdU kit (Invitrogen), following the manufacturer’s protocol.</p><h3 id="Sec14">Western blot assays</h3><p>Freshly isolated or cultured cells were washed with ice-cold PBS and lysed in cell lysis buffer (20 mM Tris (pH 7.5), 400 mM NaCl, 1 mM EDTA, 1 mM EGTA, 1% Triton X-100, 1× Complete Protease Inhibitor Cocktail (Roche Diagnostics)) for 10 min on ice, followed by sonication using the Bioruptor (Dianagene) at 4 °C for 5 min. Proteins were separated by SDS–polyacrylamide gel electrophoresis and transferred to nitrocellulose membranes (Millipore). Proteins detected by antibodies were visualized using an enhanced chemiluminescence detection system (GE Healthcare) and quantified using the ChemiDoc gel documentation system (Bio-Rad). Antibodies and dilutions used in this study are listed in the <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#MOESM1">Supplementary Information</a>.</p><h3 id="Sec15">Adult CM isolation and in vitro culture</h3><p>Isolation of adult CMs was carried out as described previously<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 46" title="O’Connell, T. D., Rodrigo, M. C. &amp; Simpson, P. C. Isolation and culture of adult mouse cardiac myocytes. Methods Mol. Biol. 357, 271–296 (2007)." href="https://riskmusings.substack.com/articles/s41586-023-06585-5#ref-CR46" id="ref-link-section-d180729366e3174">46</a></sup>. In brief, dissected hearts were cannulated through the aorta and retrogradely perfused with calcium-free buffer. Cannulated hearts were enzymatically digested by perfusion with enzyme buffer solution and cut off from the cannula. Atria were separated, and ventricles were minced in enzyme buffer. After gentle pipetting, myocytes were centrifuged at 500 r.p.m. for 1 min and cell pellets containing CM fractions were resuspended in stop buffer. The calcium content of the cell suspension was then stepwise adjusted to 1 mM and CM-containing cell pellets were resuspended in M199 cell culture medium, supplemented with creatinine, <span>l</span>-carnitine, HEPES, penicillin–streptavidin, 5% FCS and insulin–transferrin–sodium selenite medium supplement. Cells were seeded in dishes precoated with laminin and maintained in a humidified incubator at 37 °C and 5% CO<sub>2</sub>. To determine CM numbers in adult hearts, the dissected heart was washed with ice-cold PBS and fixed with 1% PFA overnight. After washing with ice-cold PBS, hearts were cut into 1–2-mm<sup>3</sup> pieces and incubated with digestion buffer (PBS containing 0.5 U ml<sup>−1</sup> of collagenase B (Roche no. 11088807001) and 0.2% NaN<sub>3</sub>) with constant shaking at 1,000 r.p.m. at 37 °C. Every 12–24 h, digested CMs were collected, and new digestion buffer was added until the heart was fully digested. CMs were pooled, plated into a Sedgewick rafter chamber and counted therein.</p><h3 id="Sec16">Measurement of oxygen consumption rate with Agilent Seahorse XF</h3><p>Adult CMs were isolated and seeded at a density of 6,000 cells per well in a 96-well plate for Seahorse measurements (Agilent Seahorse XFe96 Analyzer). Cells were washed with PBS and Seahorse base medium after attachment to the plate. The following substrates were added as energy substrates to the medium 1 h before measurements of the oxygen consumption rate: glucose 5 mM (Sigma), pyruvate 0.2 mM (Sigma), glutamine 4 mM (Sigma), palmitate–BSA 0.2 mM (Agilent), BSA control (Agilent), carnitine 0.2 mM (Sigma), valine 1 mM (Sigma), isoleucine 1 mM (Sigma), leucine 1 mM (Sigma), sodium propionate 0.05 mM (Sigma), sodium acetate 0.05 mM (Sigma), sodium octanoate 0.1 mM (Sigma), sodium decanoate 0.1 mM (Sigma). The oxygen consumption rate was measured using the Mito Stress Test kit (Agilent). The following inhibitors were injected: oligomycin (2 µM), FCCP (2 µM), rotenone and antimycin A (1 µM).</p><h3 id="Sec17">Magnetic resonance imaging and data processing</h3><p>Cardiac magnetic resonance imaging (MRI) measurements were carried out using a 7.0T Bruker Pharmascan (Bruker) equipped with a 760 mT m<sup>−1</sup> gradient system, using a cryogenically cooled four-channel phased array element <sup>1</sup>H receiver coil (CryoProbe), a 72-mm room-temperature volume resonator for transmission, and the IntraGate self-gating tool<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 47" title="Larson, A. C. et al. Self-gated cardiac cine MRI. Magn. Reson. Med. 51, 93–102 (2004)." href="https://riskmusings.substack.com/articles/s41586-023-06585-5#ref-CR47" id="ref-link-section-d180729366e3210">47</a></sup>. Electrocardiogram parameters were adapted for one heart slice and transferred afterwards to the navigator signals of the remaining slices. Thus, in-phase reconstruction of all pictures was guaranteed. Measurements are based on the gradient echo method (repetition time = 6.2 ms; echo time = 1.3 ms; field of view = 2.20 × 2.20 cm; slice thickness = 1.0 mm; matrix = 128 × 128; oversampling = 100). The imaging plane was localized using scout images showing the two- and four-chamber view of the heart, followed by acquisition of images in short-axis view, orthogonal on the septum in both scouts. Multiple contiguous short-axis slices consisting of 7 to 10 slices were acquired for complete coverage of the left and right ventricle. Mice were measured under isoflurane (1.5–2.0% in oxygen and air with a flow rate of 1.0 l min<sup>−1</sup>) anaesthesia. Body temperature was maintained at 37 °C by a thermostatically regulated water flow system during the entire imaging protocol. MRI data were analysed using Qmass digital imaging software (Medis Imaging Systems, Leiden, the Netherlands).</p><h3 id="Sec18">FACS-based isolation of cardiac nuclei</h3><p>Ventricles were washed with ice-cold PBS after dissection and snap frozen in liquid N<sub>2</sub>. For isolation of cardiac nuclei, the frozen ventricle was thawed in 3 ml lysis buffer (5 mM CaCl<sub>2</sub>, 3 mM MgAc, 2 mM EDTA, 0.5 mM EGTA and 10 mM Tris-HCl, pH 8) in M-tubes (Miltenyi Biotec) and homogenized using the gentleMACS Dissociator (Miltenyi Biotec), following the manufacturer’s protocol (protein_01). The resultant homogenate was mixed with lysis buffer containing 0.4% Triton X-100, incubated on ice for 10 min, and subsequently filtered through 40-μm cell strainers (BD Bioscience). The flow-through was centrifuged at 1,000<i>g</i> for 5 min at 4 °C to collect nuclei. Nuclei were further purified by centrifugation at 1,000<i>g</i> for 5 min at 4 °C through a 1 M sucrose cushion (3 mM MgAc, 10 mM Tris-HCl, pH 8) and then stained with a PCM1 antibody in nuclei stain buffer (DPBS, 1% BSA, 0.2% Igepal CA-630, 1 mM EDTA). DNA was stained by DAPI before FACS. FACS was carried out using a FACSAria III (BD Biosciences). Quantification of PCM<sup>+</sup> cardiac nuclei and DNA content was carried out with the LSR Fortessa (BD Biosciences) analyser. Data acquisition and analysis were accomplished using the BD FACS Diva v8 software. The gating strategy is shown in the <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#MOESM1">Supplementary Information</a>.</p><h3 id="Sec19">RNA-seq and data analysis</h3><p>RNA was extracted from isolated adult CMs using the Direct-zol Total Kit (Zymo Research) combined with on-column DNase digestion (DNase-Free DNase Set, Qiagen) to avoid contamination by genomic DNA. RNA and library preparation integrity were verified using the LabChip Gx Touch 24 (Perkin Elmer). A 200 ng quantity of total RNA was used as input for the SMARTer Stranded Total RNA Sample Prep Kit - HI Mammalian (Clontech) following the manufacturer’s instructions. Sequencing was carried out on a NextSeq500 instrument (Illumina) using v2 chemistry, resulting in an average of 22 million reads per library with a 1 × 75 bp single-end setup. Raw reads were assessed for quality, adapter content and duplication rates with FastQC 0.11.8 (<a href="http://www.bioinformatics.babraham.ac.uk/projects/fastqc">http://www.bioinformatics.babraham.ac.uk/projects/fastqc</a>). Trimmomatic version ≥ 0.36 was used to trim reads after a quality drop below a mean of Q15 in a window of five nucleotides<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 48" title="Bolger, A. M., Lohse, M. &amp; Usadel, B. Trimmomatic: a flexible trimmer for Illumina sequence data. Bioinformatics 30, 2114–2120 (2014)." href="https://riskmusings.substack.com/articles/s41586-023-06585-5#ref-CR48" id="ref-link-section-d180729366e3255">48</a></sup>. Only reads of at least 15 nucleotides were cleared for subsequent analyses. Trimmed and filtered reads were aligned versus mouse genome version mm10 (GRCm38.p5) using STAR ≥ 2.5.4b with the parameters --outFilterMismatchNoverLmax 0.1 --alignIntronMax 200000 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 49" title="Dobin, A. et al. STAR: ultrafast universal RNA-seq aligner. Bioinformatics 29, 15–21 (2013)." href="https://riskmusings.substack.com/articles/s41586-023-06585-5#ref-CR49" id="ref-link-section-d180729366e3259">49</a></sup>). The number of reads aligning to genes was counted with featureCounts ≥ 1.6.0 from the Subread package<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 50" title="Liao, Y., Smyth, G. K. &amp; Shi, W. featureCounts: an efficient general purpose program for assigning sequence reads to genomic features. Bioinformatics 30, 923–930 (2014)." href="https://riskmusings.substack.com/articles/s41586-023-06585-5#ref-CR50" id="ref-link-section-d180729366e3263">50</a></sup>. Only reads mapping at least partially inside exons were admitted and aggregated per gene. Reads overlapping multiple genes or aligning to multiple regions were excluded. Differentially expressed genes were identified using DESeq2 version ≥ 1.14.0 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 51" title="Love, M. I., Huber, W. &amp; Anders, S. Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. Genome Biol. 15, 550 (2014)." href="https://riskmusings.substack.com/articles/s41586-023-06585-5#ref-CR51" id="ref-link-section-d180729366e3267">51</a></sup>). Genes were classified as significantly differentially expressed with a <i>P</i> value &lt; 0.05. Annotations were enriched using UniProt data (release 24.03.2017) based on Ensembl gene identifiers (Activities at the Universal Protein Resource (UniProt)).</p><h3 id="Sec20">ChIP, ChIP–seq and data analysis</h3><p>Chromatin was prepared using the truChIP Chromatin Shearing Kit (COVARIS) and sheared to an average size of 200–500 bp by sonication (Diagenode Bioruptor). Protein–DNA complexes were immunoprecipitated with IgG or KDM5B antibodies, followed by incubation with Protein A/G magnetic beads (Dynabeads, Invitrogen). For ChIP–qPCR, beads were washed and protein–DNA complexes were eluted and purified using 10% Chelex-100 (w:v, Bio-Rad Laboratories) in Tris–EDTA. Immunoprecipitated chromatin was analysed by qPCR using SYBR Green quantitative real-time analysis with primers that are listed in the <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#MOESM1">Supplementary Information</a>. A detailed description of ChIP–seq analysis is provided in the <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#MOESM1">Supplementary Information</a>.</p><h3 id="Sec21">I–R injury and measurement of AAR and infarct area out of AAR</h3><p>Animals were anaesthetized using 4.5% isoflurane and endotracheally intubated with a 22-gauge intravenous catheter. Mice were placed on a 37 °C heating plate in the supine position and ventilated at a rate of 225 strokes min<sup>−1</sup> and a stroke volume of 250 µl with a mixture of oxygen and 1.5% isoflurane using a MiniVent rodent ventilator. Chest hair was removed, and skin was disinfected and opened with a small incision of several millimetres in length from the left armpit to the sternal border. Pectoralis major and minor muscles were separated, the chest was opened in the third intercostal space, and retractors were inserted. Next, the pericardium was opened to access the heart. The left coronary artery was ligated for 30 min and reopened for reperfusion in a proximal position using a prolene suture (7-0). The retractors were removed, and the chest wall was closed by bringing together the second and third rib using a vicryl suture (5-0). The muscles were placed into their original position, and the skin incision was closed with vicryl (5-0). Mice were ventilated with oxygen until awakening, followed by extubation, and placement into their cages. At 24 h after I–R surgery, the animals were euthanized for AAR and infarct area out of AAR measurement, for which hearts were removed and the aorta quickly cannulated for an injection of 500 µl 1% Evan’s blue solution into the ventricle. Hearts were kept on ice-cold saline for further investigations. Afterwards, the heart was frozen and sliced at 0.5 mm. Heart sections were subsequently stained with TTC solution (1% in PBS) at 37 °C for 30 min and then fixed with formalin solution.</p><h3 id="Sec22">Viability assay of adult CMs under hypoxic conditions</h3><p>Freshly isolated adult CMs seeded in chamber slides were cultured either under normoxia or in a hypoxia chamber with 1% O<sub>2</sub>, 5% CO<sub>2</sub> at 37 °C for 18 h. Cells were washed at room temperature with PBS and incubated with PBS containing ethidium homodimer 1 (4 µM) and calcineurin (2 µM) for 45 min at room temperature. Immunofluorescence images were acquired with a Zeiss Imager Z1 microscope.</p><h3 id="Sec23">Metabolic flux assays and targeted metabolic analysis</h3><p>Metabolic flux assays were carried out with isolated Langendorff-perfused hearts from <i>Ctrl</i><sup><i>Cre</i></sup> and <i>Cpt1b</i><sup><i>cKO</i></sup> mice. Hearts were quickly excised and cannulated through the aorta. The cannulated heart was connected to a perfusion column apparatus maintained at 37 °C using a temperature-controlled water bath. Hearts were perfused retrogradely for 60 min with Krebs–Henseleit buffer with the following substrates: glucose 8 mM; pyruvate 0.12 mM; palmitate–BSA 0.4 mM; isoleucine 0.176 mM. In each perfusion assay, only one metabolite was replaced by a <sup>13</sup>C-labelled metabolite ([<sup>13</sup>C]glucose, [<sup>13</sup>C]isoleucine or [<sup>13</sup>C]palmitate–BSA). Subsequently, hearts were snap frozen, pulverized in liquid nitrogen and subjected to metabolite extraction (acyl-CoAs or metabolites of the Krebs cycle) and quantification using liquid chromatography with triple-quadrupole mass spectrometry. Metabolic flux assays and α-ketoacids measurements were carried out using tissue from isolated hearts after perfusion with different substrates as indicated. Measurements of Krebs cycle metabolites and standardized targeted metabolic analysis were carried out with isolated adult CMs. A detailed description of the quantification of acyl-CoAs, Krebs cycle metabolites, α-ketoacids and targeted metabolome analysis is provided in the <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#MOESM1">Supplementary Information</a>.</p><h3 id="Sec24">Lentiviral transduction of CMs</h3><p>HEK293T cells were grown in DMEM (Sigma) supplemented with 10% FCS (Sigma), 2 mM <span>l</span>-glutamine, 100 U penicillin and 100 µg ml<sup>−1</sup> streptomycin at 37 °C, 5% CO<sub>2</sub>. HEK293T cells (2 × 10<sup>6</sup> per 10-cm dish) were transfected with 5 µg pLJM1-Kdm5b, pLJM1-Idh3b or pLJM1-Idh3g, 4.5 µg psPAX2 (Addgene, no. 12260) and 0.5 µg pMD2.G (Addgene, no. 12259) using the Turbofect transfection reagent and Opti-MEM for 6–8 h. The supernatants containing lentiviral particles were collected at 48 and 72 h after transfection and pooled. Lentiviruses were filtered through a 0.45 µM cell strainer to remove HEK293T cells and concentrated with a Lenti-X concentrator according to the manufacturer’s instructions (TaKaRa, 631231). Primary neonatal CMs were infected in suspension with Polybrene (8 μg ml<sup>−1</sup>) for 6 to 8 h.</p><h3 id="Sec25">Analysis of gene expression using qPCR with reverse transcription and assessment of mitochondrial DNA copy numbers</h3><p>Total RNA was extracted using the TRIzol reagent (Invitrogen) according to the manufacturer’s instructions. RNA was reverse transcribed with Superscript II (Invitrogen) following standard procedures. Real-time PCR was carried out with two technical replicates using the StepOne real-time PCR system and KAPA SYBR FAST qPCR Master Mix (KAPA Biosystems). Relative quantification of gene expression was carried out using the ∆∆CT method. The Ct values of the target genes were normalized to expression of the <i>36b4</i> gene using the equation ΔCt = Ct<sub>reference</sub> − Ct<sub>target</sub> and expressed as ΔCt. Relative mRNA expression values were shown with the average from control samples set as 1. Mitochondrial DNA copy numbers were determined using DNA extracted from isolated adult CMs. The data were normalized to internal controls (<i>H19</i> or <i>Mx1</i>) and cell numbers. Primers and PCR conditions are listed in the <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#MOESM1">Supplementary Information</a>.</p><h3 id="Sec26">Electron microscopy</h3><p>Hearts were isolated and fixed in 1.5% glutaraldehyde (v/v), 1.5% PFA (v/w) in 0.15 M HEPES (v/w), pH 8.0 at 4 °C for at least 24 h, and subsequently incubated with 1% osmium tetroxide for 2 h. Samples were stained en bloc with 50%-saturated watery uranyl acetate, followed by sequential ethanol dehydration (30%, 50%, 75%, 95%), and embedded in Agar 100. Ultrathin sections were cut using an ultramicrotome and image acquisition was carried out with a Philips CM10 electron microscope. All images were captured with a slow-scan 2k CCD (charge-coupled device) camera.</p><h3 id="Sec27">Statistical analysis</h3><p>For all quantitative analyses, a minimum of three biological replicates were analysed. Statistical tests were selected on the basis of the assumption that sample data are from a population following a probability distribution based on a fixed set of parameters. Student’s <i>t</i>-tests were used to determine the statistical significance of differences between two groups. One-way AVOVA was used for multiple comparison tests. The following values were considered to be statistically significant: *<i>P</i> &lt; 0.05, **<i>P</i> &lt; 0.01, ***<i>P</i> &lt; 0.001, ****<i>P</i> &lt; 0.0001. Calculations were carried out using the GraphPad Prism 9 software package. Data are always represented as mean ± the standard error of the mean. No statistical method was used to predetermine sample size.</p><h3 id="Sec28">Reporting summary</h3><p>Further information on research design is available in the <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#MOESM2">Nature Portfolio Reporting Summary</a> linked to this article.</p></div></div></section>
                </div><div>
            <section data-title="Data availability"><div id="data-availability-section"><h2 id="data-availability">Data availability</h2><p>Data have been deposited in public databases. Sequencing data are available under the accession number <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE172415">GSE172415</a> and include results from ChIP–seq and RNA-seq experiments. RNA-seq data for neonatal CMs treated with DMSO and aKG are available under the accession number <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE217188">GSE217188</a>. <a data-track="click" data-track-label="link" data-track-action="section anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#Sec32">Source data</a> are provided with this paper.</p></div></section><section data-title="Code availability"><div id="code-availability-section"><h2 id="code-availability">Code availability</h2><div id="code-availability-content">
            
            <p>Codes used in the study are available at <a href="https://github.com/loosolab/Li_et_al_2023_heart_regeneration">https://github.com/loosolab/Li_et_al_2023_heart_regeneration</a> (<a href="https://doi.org/10.5281/zenodo.7828994">https://doi.org/10.5281/zenodo.7828994</a>). The codes have already been used and published in a different project<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 52" title="Wu, F. et al. Spurious transcription causing innate immune responses is prevented by 5-hydroxymethylcytosine. Nat. Genet. 55, 100–111 (2023)." href="https://riskmusings.substack.com/articles/s41586-023-06585-5#ref-CR52" id="ref-link-section-d180729366e3577">52</a></sup>.</p>
          </div></div></section><div id="MagazineFulltextArticleBodySuffix"><section aria-labelledby="Bib1" data-title="References"><div id="Bib1-section"><h2 id="Bib1">References</h2><div id="Bib1-content"><div data-container-section="references"><ol data-track-component="outbound reference"><li data-counter="1."><p id="ref-CR1">Maroli, G. &amp; Braun, T. The long and winding road of cardiomyocyte maturation. <i>Cardiovasc. Res.</i> <b>117</b>, 712–726 (2021).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1093/cvr/cvaa159" data-track-action="article reference" href="https://doi.org/10.1093%2Fcvr%2Fcvaa159" aria-label="Article reference 1" data-doi="10.1093/cvr/cvaa159">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://riskmusings.substack.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhs1Wrt7vN" aria-label="CAS reference 1">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32514522" aria-label="PubMed reference 1">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 1" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20long%20and%20winding%20road%20of%20cardiomyocyte%20maturation&amp;journal=Cardiovasc.%20Res.&amp;doi=10.1093%2Fcvr%2Fcvaa159&amp;volume=117&amp;pages=712-726&amp;publication_year=2021&amp;author=Maroli%2CG&amp;author=Braun%2CT">
                    Google Scholar</a> 
                </p></li><li data-counter="2."><p id="ref-CR2">Yuan, X. &amp; Braun, T. Multimodal regulation of cardiac myocyte proliferation. <i>Circ. Res.</i> <b>121</b>, 293–309 (2017).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1161/CIRCRESAHA.117.308428" data-track-action="article reference" href="https://doi.org/10.1161%2FCIRCRESAHA.117.308428" aria-label="Article reference 2" data-doi="10.1161/CIRCRESAHA.117.308428">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://riskmusings.substack.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhtF2it7bO" aria-label="CAS reference 2">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28729454" aria-label="PubMed reference 2">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 2" href="http://scholar.google.com/scholar_lookup?&amp;title=Multimodal%20regulation%20of%20cardiac%20myocyte%20proliferation&amp;journal=Circ.%20Res.&amp;doi=10.1161%2FCIRCRESAHA.117.308428&amp;volume=121&amp;pages=293-309&amp;publication_year=2017&amp;author=Yuan%2CX&amp;author=Braun%2CT">
                    Google Scholar</a> 
                </p></li><li data-counter="3."><p id="ref-CR3">Horton, J. R. et al. Characterization of a linked Jumonji domain of the KDM5/JARID1 family of histone H3 lysine 4 demethylases. <i>J. Biol. Chem.</i> <b>291</b>, 2631–2646 (2016).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1074/jbc.M115.698449" data-track-action="article reference" href="https://doi.org/10.1074%2Fjbc.M115.698449" aria-label="Article reference 3" data-doi="10.1074/jbc.M115.698449">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://riskmusings.substack.com/articles/cas-redirect/1:CAS:528:DC%2BC28XitVOqtr0%3D" aria-label="CAS reference 3">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26645689" aria-label="PubMed reference 3">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 3" href="http://scholar.google.com/scholar_lookup?&amp;title=Characterization%20of%20a%20linked%20Jumonji%20domain%20of%20the%20KDM5%2FJARID1%20family%20of%20histone%20H3%20lysine%204%20demethylases&amp;journal=J.%20Biol.%20Chem.&amp;doi=10.1074%2Fjbc.M115.698449&amp;volume=291&amp;pages=2631-2646&amp;publication_year=2016&amp;author=Horton%2CJR">
                    Google Scholar</a> 
                </p></li><li data-counter="4."><p id="ref-CR4">Nakada, Y. et al. Hypoxia induces heart regeneration in adult mice. <i>Nature</i> <a href="https://doi.org/10.1038/nature20173" data-track="click" data-track-action="external reference" data-track-label="10.1038/nature20173">https://doi.org/10.1038/nature20173</a> (2016).</p></li><li data-counter="5."><p id="ref-CR5">Puente, B. N. et al. The oxygen-rich postnatal environment induces cardiomyocyte cell-cycle arrest through DNA damage response. <i>Cell</i> <b>157</b>, 565–579 (2014).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.cell.2014.03.032" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.cell.2014.03.032" aria-label="Article reference 5" data-doi="10.1016/j.cell.2014.03.032">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://riskmusings.substack.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXntFGit7g%3D" aria-label="CAS reference 5">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24766806" aria-label="PubMed reference 5">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4104514" aria-label="PubMed Central reference 5">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 5" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20oxygen-rich%20postnatal%20environment%20induces%20cardiomyocyte%20cell-cycle%20arrest%20through%20DNA%20damage%20response&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2014.03.032&amp;volume=157&amp;pages=565-579&amp;publication_year=2014&amp;author=Puente%2CBN">
                    Google Scholar</a> 
                </p></li><li data-counter="6."><p id="ref-CR6">Ben-Yair, R. et al. H3K27me3-mediated silencing of structural genes is required for zebrafish heart regeneration. <i>Development</i> <b>146</b>, dev178632 (2019).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1242/dev.178632" data-track-action="article reference" href="https://doi.org/10.1242%2Fdev.178632" aria-label="Article reference 6" data-doi="10.1242/dev.178632">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31427288" aria-label="PubMed reference 6">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6803378" aria-label="PubMed Central reference 6">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 6" href="http://scholar.google.com/scholar_lookup?&amp;title=H3K27me3-mediated%20silencing%20of%20structural%20genes%20is%20required%20for%20zebrafish%20heart%20regeneration&amp;journal=Development&amp;doi=10.1242%2Fdev.178632&amp;volume=146&amp;publication_year=2019&amp;author=Ben-Yair%2CR">
                    Google Scholar</a> 
                </p></li><li data-counter="7."><p id="ref-CR7">Zhu, Y., Do, V. D., Richards, A. M. &amp; Foo, R. What we know about cardiomyocyte dedifferentiation. <i>J. Mol. Cell. Cardiol.</i> <b>152</b>, 80–91 (2021).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.yjmcc.2020.11.016" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.yjmcc.2020.11.016" aria-label="Article reference 7" data-doi="10.1016/j.yjmcc.2020.11.016">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://riskmusings.substack.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXis1Wkt7fO" aria-label="CAS reference 7">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33275936" aria-label="PubMed reference 7">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 7" href="http://scholar.google.com/scholar_lookup?&amp;title=What%20we%20know%20about%20cardiomyocyte%20dedifferentiation&amp;journal=J.%20Mol.%20Cell.%20Cardiol.&amp;doi=10.1016%2Fj.yjmcc.2020.11.016&amp;volume=152&amp;pages=80-91&amp;publication_year=2021&amp;author=Zhu%2CY&amp;author=Do%2CVD&amp;author=Richards%2CAM&amp;author=Foo%2CR">
                    Google Scholar</a> 
                </p></li><li data-counter="8."><p id="ref-CR8">Tao, G. et al. Pitx2 promotes heart repair by activating the antioxidant response after cardiac injury. <i>Nature</i> <b>534</b>, 119–123 (2016).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/nature17959" data-track-action="article reference" href="https://doi.org/10.1038%2Fnature17959" aria-label="Article reference 8" data-doi="10.1038/nature17959">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2016Natur.534..119T" aria-label="ADS reference 8">ADS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://riskmusings.substack.com/articles/cas-redirect/1:CAS:528:DC%2BC28XosFWht78%3D" aria-label="CAS reference 8">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27251288" aria-label="PubMed reference 8">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4999251" aria-label="PubMed Central reference 8">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 8" href="http://scholar.google.com/scholar_lookup?&amp;title=Pitx2%20promotes%20heart%20repair%20by%20activating%20the%20antioxidant%20response%20after%20cardiac%20injury&amp;journal=Nature&amp;doi=10.1038%2Fnature17959&amp;volume=534&amp;pages=119-123&amp;publication_year=2016&amp;author=Tao%2CG">
                    Google Scholar</a> 
                </p></li><li data-counter="9."><p id="ref-CR9">Magadum, A. et al. Pkm2 regulates cardiomyocyte cell cycle and promotes cardiac regeneration. <i>Circulation</i> <b>141</b>, 1249–1265 (2020).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1161/CIRCULATIONAHA.119.043067" data-track-action="article reference" href="https://doi.org/10.1161%2FCIRCULATIONAHA.119.043067" aria-label="Article reference 9" data-doi="10.1161/CIRCULATIONAHA.119.043067">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://riskmusings.substack.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXmvFSjt7w%3D" aria-label="CAS reference 9">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32078387" aria-label="PubMed reference 9">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7241614" aria-label="PubMed Central reference 9">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 9" href="http://scholar.google.com/scholar_lookup?&amp;title=Pkm2%20regulates%20cardiomyocyte%20cell%20cycle%20and%20promotes%20cardiac%20regeneration&amp;journal=Circulation&amp;doi=10.1161%2FCIRCULATIONAHA.119.043067&amp;volume=141&amp;pages=1249-1265&amp;publication_year=2020&amp;author=Magadum%2CA">
                    Google Scholar</a> 
                </p></li><li data-counter="10."><p id="ref-CR10">Cardoso, A. C. et al. Mitochondrial substrate utilization regulates cardiomyocyte cell-cycle progression. <i>Nat. Metab.</i> <b>2</b>, 167–178 (2020).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/s42255-020-0169-x" data-track-action="article reference" href="https://doi.org/10.1038%2Fs42255-020-0169-x" aria-label="Article reference 10" data-doi="10.1038/s42255-020-0169-x">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://riskmusings.substack.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXks1Krt74%3D" aria-label="CAS reference 10">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32617517" aria-label="PubMed reference 10">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7331943" aria-label="PubMed Central reference 10">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 10" href="http://scholar.google.com/scholar_lookup?&amp;title=Mitochondrial%20substrate%20utilization%20regulates%20cardiomyocyte%20cell-cycle%20progression&amp;journal=Nat.%20Metab.&amp;doi=10.1038%2Fs42255-020-0169-x&amp;volume=2&amp;pages=167-178&amp;publication_year=2020&amp;author=Cardoso%2CAC">
                    Google Scholar</a> 
                </p></li><li data-counter="11."><p id="ref-CR11">Bae, J. et al. Malonate promotes adult cardiomyocyte proliferation and heart regeneration. <i>Circulation</i> <b>143</b>, 1973–1986 (2021).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1161/CIRCULATIONAHA.120.049952" data-track-action="article reference" href="https://doi.org/10.1161%2FCIRCULATIONAHA.120.049952" aria-label="Article reference 11" data-doi="10.1161/CIRCULATIONAHA.120.049952">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://riskmusings.substack.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhsF2isrrE" aria-label="CAS reference 11">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33666092" aria-label="PubMed reference 11">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 11" href="http://scholar.google.com/scholar_lookup?&amp;title=Malonate%20promotes%20adult%20cardiomyocyte%20proliferation%20and%20heart%20regeneration&amp;journal=Circulation&amp;doi=10.1161%2FCIRCULATIONAHA.120.049952&amp;volume=143&amp;pages=1973-1986&amp;publication_year=2021&amp;author=Bae%2CJ">
                    Google Scholar</a> 
                </p></li><li data-counter="12."><p id="ref-CR12">Atlasi, Y. &amp; Stunnenberg, H. G. The interplay of epigenetic marks during stem cell differentiation and development. <i>Nat. Rev. Genet.</i> <b>18</b>, 643–658 (2017).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/nrg.2017.57" data-track-action="article reference" href="https://doi.org/10.1038%2Fnrg.2017.57" aria-label="Article reference 12" data-doi="10.1038/nrg.2017.57">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://riskmusings.substack.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhtlGktbfP" aria-label="CAS reference 12">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28804139" aria-label="PubMed reference 12">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 12" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20interplay%20of%20epigenetic%20marks%20during%20stem%20cell%20differentiation%20and%20development&amp;journal=Nat.%20Rev.%20Genet.&amp;doi=10.1038%2Fnrg.2017.57&amp;volume=18&amp;pages=643-658&amp;publication_year=2017&amp;author=Atlasi%2CY&amp;author=Stunnenberg%2CHG">
                    Google Scholar</a> 
                </p></li><li data-counter="13."><p id="ref-CR13">Kaneda, R. et al. Genome-wide histone methylation profile for heart failure. <i>Genes Cells</i> <b>14</b>, 69–77 (2009).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1111/j.1365-2443.2008.01252.x" data-track-action="article reference" href="https://doi.org/10.1111%2Fj.1365-2443.2008.01252.x" aria-label="Article reference 13" data-doi="10.1111/j.1365-2443.2008.01252.x">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://riskmusings.substack.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXpsFagtQ%3D%3D" aria-label="CAS reference 13">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19077033" aria-label="PubMed reference 13">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 13" href="http://scholar.google.com/scholar_lookup?&amp;title=Genome-wide%20histone%20methylation%20profile%20for%20heart%20failure&amp;journal=Genes%20Cells&amp;doi=10.1111%2Fj.1365-2443.2008.01252.x&amp;volume=14&amp;pages=69-77&amp;publication_year=2009&amp;author=Kaneda%2CR">
                    Google Scholar</a> 
                </p></li><li data-counter="14."><p id="ref-CR14">Chen, K. et al. Broad H3K4me3 is associated with increased transcription elongation and enhancer activity at tumor-suppressor genes. <i>Nat. Genet.</i> <b>47</b>, 1149–1157 (2015).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/ng.3385" data-track-action="article reference" href="https://doi.org/10.1038%2Fng.3385" aria-label="Article reference 14" data-doi="10.1038/ng.3385">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://riskmusings.substack.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXhtlylsrjL" aria-label="CAS reference 14">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26301496" aria-label="PubMed reference 14">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4780747" aria-label="PubMed Central reference 14">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 14" href="http://scholar.google.com/scholar_lookup?&amp;title=Broad%20H3K4me3%20is%20associated%20with%20increased%20transcription%20elongation%20and%20enhancer%20activity%20at%20tumor-suppressor%20genes&amp;journal=Nat.%20Genet.&amp;doi=10.1038%2Fng.3385&amp;volume=47&amp;pages=1149-1157&amp;publication_year=2015&amp;author=Chen%2CK">
                    Google Scholar</a> 
                </p></li><li data-counter="15."><p id="ref-CR15">Howe, F. S., Fischl, H., Murray, S. C. &amp; Mellor, J. Is H3K4me3 instructive for transcription activation? <i>Bioessays</i> <b>39</b>, e201600095 (2017).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1002/bies.201600095" data-track-action="article reference" href="https://doi.org/10.1002%2Fbies.201600095" aria-label="Article reference 15" data-doi="10.1002/bies.201600095">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 15" href="http://scholar.google.com/scholar_lookup?&amp;title=Is%20H3K4me3%20instructive%20for%20transcription%20activation%3F&amp;journal=Bioessays&amp;doi=10.1002%2Fbies.201600095&amp;volume=39&amp;publication_year=2017&amp;author=Howe%2CFS&amp;author=Fischl%2CH&amp;author=Murray%2CSC&amp;author=Mellor%2CJ">
                    Google Scholar</a> 
                </p></li><li data-counter="16."><p id="ref-CR16">Gilsbach, R. et al. Distinct epigenetic programs regulate cardiac myocyte development and disease in the human heart in vivo. <i>Nat. Commun.</i> <b>9</b>, 391 (2018).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/s41467-017-02762-z" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41467-017-02762-z" aria-label="Article reference 16" data-doi="10.1038/s41467-017-02762-z">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2018NatCo...9..391G" aria-label="ADS reference 16">ADS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29374152" aria-label="PubMed reference 16">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5786002" aria-label="PubMed Central reference 16">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 16" href="http://scholar.google.com/scholar_lookup?&amp;title=Distinct%20epigenetic%20programs%20regulate%20cardiac%20myocyte%20development%20and%20disease%20in%20the%20human%20heart%20in%20vivo&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-017-02762-z&amp;volume=9&amp;publication_year=2018&amp;author=Gilsbach%2CR">
                    Google Scholar</a> 
                </p></li><li data-counter="17."><p id="ref-CR17">Pokholok, D. K. et al. Genome-wide map of nucleosome acetylation and methylation in yeast. <i>Cell</i> <b>122</b>, 517–527 (2005).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.cell.2005.06.026" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.cell.2005.06.026" aria-label="Article reference 17" data-doi="10.1016/j.cell.2005.06.026">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://riskmusings.substack.com/articles/cas-redirect/1:CAS:528:DC%2BD2MXpvVKntLc%3D" aria-label="CAS reference 17">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16122420" aria-label="PubMed reference 17">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 17" href="http://scholar.google.com/scholar_lookup?&amp;title=Genome-wide%20map%20of%20nucleosome%20acetylation%20and%20methylation%20in%20yeast&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2005.06.026&amp;volume=122&amp;pages=517-527&amp;publication_year=2005&amp;author=Pokholok%2CDK">
                    Google Scholar</a> 
                </p></li><li data-counter="18."><p id="ref-CR18">Zhang, Z. et al. H3K4 tri-methylation breadth at transcription start sites impacts the transcriptome of systemic lupus erythematosus. <i>Clin. Epigenetics</i> <b>8</b>, 14 (2016).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1186/s13148-016-0179-4" data-track-action="article reference" href="https://doi.org/10.1186%2Fs13148-016-0179-4" aria-label="Article reference 18" data-doi="10.1186/s13148-016-0179-4">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26839600" aria-label="PubMed reference 18">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4736279" aria-label="PubMed Central reference 18">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 18" href="http://scholar.google.com/scholar_lookup?&amp;title=H3K4%20tri-methylation%20breadth%20at%20transcription%20start%20sites%20impacts%20the%20transcriptome%20of%20systemic%20lupus%20erythematosus&amp;journal=Clin.%20Epigenetics&amp;doi=10.1186%2Fs13148-016-0179-4&amp;volume=8&amp;publication_year=2016&amp;author=Zhang%2CZ">
                    Google Scholar</a> 
                </p></li><li data-counter="19."><p id="ref-CR19">Liu, P. S. et al. α-ketoglutarate orchestrates macrophage activation through metabolic and epigenetic reprogramming. <i>Nat. Immunol.</i> <b>18</b>, 985–994 (2017).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/ni.3796" data-track-action="article reference" href="https://doi.org/10.1038%2Fni.3796" aria-label="Article reference 19" data-doi="10.1038/ni.3796">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://riskmusings.substack.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhtFOltLnM" aria-label="CAS reference 19">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28714978" aria-label="PubMed reference 19">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 19" href="http://scholar.google.com/scholar_lookup?&amp;title=%CE%B1-ketoglutarate%20orchestrates%20macrophage%20activation%20through%20metabolic%20and%20epigenetic%20reprogramming&amp;journal=Nat.%20Immunol.&amp;doi=10.1038%2Fni.3796&amp;volume=18&amp;pages=985-994&amp;publication_year=2017&amp;author=Liu%2CPS">
                    Google Scholar</a> 
                </p></li><li data-counter="20."><p id="ref-CR20">Chin, R. M. et al. The metabolite α-ketoglutarate extends lifespan by inhibiting ATP synthase and TOR. <i>Nature</i> <b>510</b>, 397–401 (2014).</p></li><li data-counter="21."><p id="ref-CR21">Carey, B. W., Finley, L. W., Cross, J. R., Allis, C. D. &amp; Thompson, C. B. Intracellular α-ketoglutarate maintains the pluripotency of embryonic stem cells. <i>Nature</i> <b>518</b>, 413–416 (2015).</p></li><li data-counter="22."><p id="ref-CR22">Lopaschuk, G. D. &amp; Jaswal, J. S. Energy metabolic phenotype of the cardiomyocyte during development, differentiation, and postnatal maturation. <i>J. Cardiovasc. Pharmacol.</i> <b>56</b>, 130–140 (2010).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1097/FJC.0b013e3181e74a14" data-track-action="article reference" href="https://doi.org/10.1097%2FFJC.0b013e3181e74a14" aria-label="Article reference 22" data-doi="10.1097/FJC.0b013e3181e74a14">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://riskmusings.substack.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXhtVektbzM" aria-label="CAS reference 22">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20505524" aria-label="PubMed reference 22">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 22" href="http://scholar.google.com/scholar_lookup?&amp;title=Energy%20metabolic%20phenotype%20of%20the%20cardiomyocyte%20during%20development%2C%20differentiation%2C%20and%20postnatal%20maturation&amp;journal=J.%20Cardiovasc.%20Pharmacol.&amp;doi=10.1097%2FFJC.0b013e3181e74a14&amp;volume=56&amp;pages=130-140&amp;publication_year=2010&amp;author=Lopaschuk%2CGD&amp;author=Jaswal%2CJS">
                    Google Scholar</a> 
                </p></li><li data-counter="23."><p id="ref-CR23">Greco, C. M. et al. DNA hydroxymethylation controls cardiomyocyte gene expression in development and hypertrophy. <i>Nat. Commun.</i> <b>7</b>, 12418 (2016).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/ncomms12418" data-track-action="article reference" href="https://doi.org/10.1038%2Fncomms12418" aria-label="Article reference 23" data-doi="10.1038/ncomms12418">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2016NatCo...712418G" aria-label="ADS reference 23">ADS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://riskmusings.substack.com/articles/cas-redirect/1:CAS:528:DC%2BC28XhtlalsLbK" aria-label="CAS reference 23">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27489048" aria-label="PubMed reference 23">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4976219" aria-label="PubMed Central reference 23">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 23" href="http://scholar.google.com/scholar_lookup?&amp;title=DNA%20hydroxymethylation%20controls%20cardiomyocyte%20gene%20expression%20in%20development%20and%20hypertrophy&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fncomms12418&amp;volume=7&amp;publication_year=2016&amp;author=Greco%2CCM">
                    Google Scholar</a> 
                </p></li><li data-counter="24."><p id="ref-CR24">Man, J., Barnett, P. &amp; Christoffels, V. M. Structure and function of the <i>Nppa</i>–<i>Nppb</i> cluster locus during heart development and disease. <i>Cell. Mol. Life Sci.</i> <b>75</b>, 1435–1444 (2018).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1007/s00018-017-2737-0" data-track-action="article reference" href="https://doi.org/10.1007%2Fs00018-017-2737-0" aria-label="Article reference 24" data-doi="10.1007/s00018-017-2737-0">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://riskmusings.substack.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXjvFejtA%3D%3D" aria-label="CAS reference 24">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29302701" aria-label="PubMed reference 24">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5852170" aria-label="PubMed Central reference 24">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 24" href="http://scholar.google.com/scholar_lookup?&amp;title=Structure%20and%20function%20of%20the%20Nppa%E2%80%93Nppb%20cluster%20locus%20during%20heart%20development%20and%20disease&amp;journal=Cell.%20Mol.%20Life%20Sci.&amp;doi=10.1007%2Fs00018-017-2737-0&amp;volume=75&amp;pages=1435-1444&amp;publication_year=2018&amp;author=Man%2CJ&amp;author=Barnett%2CP&amp;author=Christoffels%2CVM">
                    Google Scholar</a> 
                </p></li><li data-counter="25."><p id="ref-CR25">Patel, M. S. Inhibition by the branched-chain 2-oxo acids of the 2-oxoglutarate dehydrogenase complex in developing rat and human brain. <i>Biochem. J.</i> <b>144</b>, 91–97 (1974).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1042/bj1440091" data-track-action="article reference" href="https://doi.org/10.1042%2Fbj1440091" aria-label="Article reference 25" data-doi="10.1042/bj1440091">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://riskmusings.substack.com/articles/cas-redirect/1:CAS:528:DyaE2MXps1WqsA%3D%3D" aria-label="CAS reference 25">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=4462577" aria-label="PubMed reference 25">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1168468" aria-label="PubMed Central reference 25">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 25" href="http://scholar.google.com/scholar_lookup?&amp;title=Inhibition%20by%20the%20branched-chain%202-oxo%20acids%20of%20the%202-oxoglutarate%20dehydrogenase%20complex%20in%20developing%20rat%20and%20human%20brain&amp;journal=Biochem.%20J.&amp;doi=10.1042%2Fbj1440091&amp;volume=144&amp;pages=91-97&amp;publication_year=1974&amp;author=Patel%2CMS">
                    Google Scholar</a> 
                </p></li><li data-counter="26."><p id="ref-CR26">Andrade, J. et al. Control of endothelial quiescence by FOXO-regulated metabolites. <i>Nat. Cell Biol.</i> <b>23</b>, 413–423 (2021).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/s41556-021-00637-6" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41556-021-00637-6" aria-label="Article reference 26" data-doi="10.1038/s41556-021-00637-6">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://riskmusings.substack.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXotVChtL8%3D" aria-label="CAS reference 26">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33795871" aria-label="PubMed reference 26">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8032556" aria-label="PubMed Central reference 26">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 26" href="http://scholar.google.com/scholar_lookup?&amp;title=Control%20of%20endothelial%20quiescence%20by%20FOXO-regulated%20metabolites&amp;journal=Nat.%20Cell%20Biol.&amp;doi=10.1038%2Fs41556-021-00637-6&amp;volume=23&amp;pages=413-423&amp;publication_year=2021&amp;author=Andrade%2CJ">
                    Google Scholar</a> 
                </p></li><li data-counter="27."><p id="ref-CR27">Sze, C. C. et al. Coordinated regulation of cellular identity-associated H3K4me3 breadth by the COMPASS family. <i>Sci. Adv.</i> <b>6</b>, eaaz4764 (2020).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1126/sciadv.aaz4764" data-track-action="article reference" href="https://doi.org/10.1126%2Fsciadv.aaz4764" aria-label="Article reference 27" data-doi="10.1126/sciadv.aaz4764">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2020SciA....6.4764S" aria-label="ADS reference 27">ADS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://riskmusings.substack.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXitlWitrjK" aria-label="CAS reference 27">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32637595" aria-label="PubMed reference 27">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7314515" aria-label="PubMed Central reference 27">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 27" href="http://scholar.google.com/scholar_lookup?&amp;title=Coordinated%20regulation%20of%20cellular%20identity-associated%20H3K4me3%20breadth%20by%20the%20COMPASS%20family&amp;journal=Sci.%20Adv.&amp;doi=10.1126%2Fsciadv.aaz4764&amp;volume=6&amp;publication_year=2020&amp;author=Sze%2CCC">
                    Google Scholar</a> 
                </p></li><li data-counter="28."><p id="ref-CR28">Benayoun, B. A. et al. H3K4me3 breadth is linked to cell identity and transcriptional consistency. <i>Cell</i> <b>158</b>, 673–688 (2014).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.cell.2014.06.027" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.cell.2014.06.027" aria-label="Article reference 28" data-doi="10.1016/j.cell.2014.06.027">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://riskmusings.substack.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXht1ymtLjP" aria-label="CAS reference 28">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25083876" aria-label="PubMed reference 28">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4137894" aria-label="PubMed Central reference 28">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 28" href="http://scholar.google.com/scholar_lookup?&amp;title=H3K4me3%20breadth%20is%20linked%20to%20cell%20identity%20and%20transcriptional%20consistency&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2014.06.027&amp;volume=158&amp;pages=673-688&amp;publication_year=2014&amp;author=Benayoun%2CBA">
                    Google Scholar</a> 
                </p></li><li data-counter="29."><p id="ref-CR29">Intlekofer, A. M. et al. Hypoxia induces production of L-2-hydroxyglutarate. <i>Cell Metab</i> <b>22</b>, 304–311 (2015).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.cmet.2015.06.023" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.cmet.2015.06.023" aria-label="Article reference 29" data-doi="10.1016/j.cmet.2015.06.023">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://riskmusings.substack.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXht1ChsLrE" aria-label="CAS reference 29">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26212717" aria-label="PubMed reference 29">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4527873" aria-label="PubMed Central reference 29">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 29" href="http://scholar.google.com/scholar_lookup?&amp;title=Hypoxia%20induces%20production%20of%20L-2-hydroxyglutarate&amp;journal=Cell%20Metab&amp;doi=10.1016%2Fj.cmet.2015.06.023&amp;volume=22&amp;pages=304-311&amp;publication_year=2015&amp;author=Intlekofer%2CAM">
                    Google Scholar</a> 
                </p></li><li data-counter="30."><p id="ref-CR30">Zaidi, S. et al. De novo mutations in histone-modifying genes in congenital heart disease. <i>Nature</i> <b>498</b>, 220–223 (2013).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/nature12141" data-track-action="article reference" href="https://doi.org/10.1038%2Fnature12141" aria-label="Article reference 30" data-doi="10.1038/nature12141">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2013Natur.498..220Z" aria-label="ADS reference 30">ADS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://riskmusings.substack.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXotFSns7s%3D" aria-label="CAS reference 30">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23665959" aria-label="PubMed reference 30">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3706629" aria-label="PubMed Central reference 30">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 30" href="http://scholar.google.com/scholar_lookup?&amp;title=De%20novo%20mutations%20in%20histone-modifying%20genes%20in%20congenital%20heart%20disease&amp;journal=Nature&amp;doi=10.1038%2Fnature12141&amp;volume=498&amp;pages=220-223&amp;publication_year=2013&amp;author=Zaidi%2CS">
                    Google Scholar</a> 
                </p></li><li data-counter="31."><p id="ref-CR31">Huang, C. et al. SENP3 is responsible for HIF-1 transactivation under mild oxidative stress via p300 de-SUMOylation. <i>EMBO J.</i> <b>28</b>, 2748–2762 (2009).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/emboj.2009.210" data-track-action="article reference" href="https://doi.org/10.1038%2Femboj.2009.210" aria-label="Article reference 31" data-doi="10.1038/emboj.2009.210">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://riskmusings.substack.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXpsFKgsL4%3D" aria-label="CAS reference 31">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19680224" aria-label="PubMed reference 31">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2750016" aria-label="PubMed Central reference 31">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 31" href="http://scholar.google.com/scholar_lookup?&amp;title=SENP3%20is%20responsible%20for%20HIF-1%20transactivation%20under%20mild%20oxidative%20stress%20via%20p300%20de-SUMOylation&amp;journal=EMBO%20J.&amp;doi=10.1038%2Femboj.2009.210&amp;volume=28&amp;pages=2748-2762&amp;publication_year=2009&amp;author=Huang%2CC">
                    Google Scholar</a> 
                </p></li><li data-counter="32."><p id="ref-CR32">Sundheim, O. et al. Human ABH3 structure and key residues for oxidative demethylation to reverse DNA/RNA damage. <i>EMBO J.</i> <b>25</b>, 3389–3397 (2006).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/sj.emboj.7601219" data-track-action="article reference" href="https://doi.org/10.1038%2Fsj.emboj.7601219" aria-label="Article reference 32" data-doi="10.1038/sj.emboj.7601219">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://riskmusings.substack.com/articles/cas-redirect/1:CAS:528:DC%2BD28Xnt1yqurg%3D" aria-label="CAS reference 32">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16858410" aria-label="PubMed reference 32">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1523172" aria-label="PubMed Central reference 32">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 32" href="http://scholar.google.com/scholar_lookup?&amp;title=Human%20ABH3%20structure%20and%20key%20residues%20for%20oxidative%20demethylation%20to%20reverse%20DNA%2FRNA%20damage&amp;journal=EMBO%20J.&amp;doi=10.1038%2Fsj.emboj.7601219&amp;volume=25&amp;pages=3389-3397&amp;publication_year=2006&amp;author=Sundheim%2CO">
                    Google Scholar</a> 
                </p></li><li data-counter="33."><p id="ref-CR33">Chu, W. et al. Mild hypoxia-induced cardiomyocyte hypertrophy <i>via</i> up-regulation of HIF-1α-mediated TRPC signalling. <i>J. Cell. Mol. Med.</i> <b>16</b>, 2022–2034 (2012).</p></li><li data-counter="34."><p id="ref-CR34">Peoples, J. N., Saraf, A., Ghazal, N., Pham, T. T. &amp; Kwong, J. Q. Mitochondrial dysfunction and oxidative stress in heart disease. <i>Exp. Mol. Med.</i> <b>51</b>, 1–13 (2019).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/s12276-019-0355-7" data-track-action="article reference" href="https://doi.org/10.1038%2Fs12276-019-0355-7" aria-label="Article reference 34" data-doi="10.1038/s12276-019-0355-7">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://riskmusings.substack.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXisVKrsr%2FN" aria-label="CAS reference 34">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31857574" aria-label="PubMed reference 34">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 34" href="http://scholar.google.com/scholar_lookup?&amp;title=Mitochondrial%20dysfunction%20and%20oxidative%20stress%20in%20heart%20disease&amp;journal=Exp.%20Mol.%20Med.&amp;doi=10.1038%2Fs12276-019-0355-7&amp;volume=51&amp;pages=1-13&amp;publication_year=2019&amp;author=Peoples%2CJN&amp;author=Saraf%2CA&amp;author=Ghazal%2CN&amp;author=Pham%2CTT&amp;author=Kwong%2CJQ">
                    Google Scholar</a> 
                </p></li><li data-counter="35."><p id="ref-CR35">Drake, K. J., Sidorov, V. Y., McGuinness, O. P., Wasserman, D. H. &amp; Wikswo, J. P. Amino acids as metabolic substrates during cardiac ischemia. <i>Exp. Biol. Med.</i> <b>237</b>, 1369–1378 (2012).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1258/ebm.2012.012025" data-track-action="article reference" href="https://doi.org/10.1258%2Febm.2012.012025" aria-label="Article reference 35" data-doi="10.1258/ebm.2012.012025">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://riskmusings.substack.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXjsFagtb0%3D" aria-label="CAS reference 35">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 35" href="http://scholar.google.com/scholar_lookup?&amp;title=Amino%20acids%20as%20metabolic%20substrates%20during%20cardiac%20ischemia&amp;journal=Exp.%20Biol.%20Med.&amp;doi=10.1258%2Febm.2012.012025&amp;volume=237&amp;pages=1369-1378&amp;publication_year=2012&amp;author=Drake%2CKJ&amp;author=Sidorov%2CVY&amp;author=McGuinness%2COP&amp;author=Wasserman%2CDH&amp;author=Wikswo%2CJP">
                    Google Scholar</a> 
                </p></li><li data-counter="36."><p id="ref-CR36">Ji, S. et al. Homozygous carnitine palmitoyltransferase 1b (muscle isoform) deficiency is lethal in the mouse. <i>Mol. Genet. Metab.</i> <b>93</b>, 314–322 (2008).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.ymgme.2007.10.006" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.ymgme.2007.10.006" aria-label="Article reference 36" data-doi="10.1016/j.ymgme.2007.10.006">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://riskmusings.substack.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXhslehtbY%3D" aria-label="CAS reference 36">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18023382" aria-label="PubMed reference 36">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 36" href="http://scholar.google.com/scholar_lookup?&amp;title=Homozygous%20carnitine%20palmitoyltransferase%201b%20%28muscle%20isoform%29%20deficiency%20is%20lethal%20in%20the%20mouse&amp;journal=Mol.%20Genet.%20Metab.&amp;doi=10.1016%2Fj.ymgme.2007.10.006&amp;volume=93&amp;pages=314-322&amp;publication_year=2008&amp;author=Ji%2CS">
                    Google Scholar</a> 
                </p></li><li data-counter="37."><p id="ref-CR37">He, L. et al. Carnitine palmitoyltransferase-1b deficiency aggravates pressure overload-induced cardiac hypertrophy caused by lipotoxicity. <i>Circulation</i> <b>126</b>, 1705–1716 (2012).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1161/CIRCULATIONAHA.111.075978" data-track-action="article reference" href="https://doi.org/10.1161%2FCIRCULATIONAHA.111.075978" aria-label="Article reference 37" data-doi="10.1161/CIRCULATIONAHA.111.075978">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://riskmusings.substack.com/articles/cas-redirect/1:CAS:528:DC%2BC38XhsVGiu7jJ" aria-label="CAS reference 37">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22932257" aria-label="PubMed reference 37">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3484985" aria-label="PubMed Central reference 37">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 37" href="http://scholar.google.com/scholar_lookup?&amp;title=Carnitine%20palmitoyltransferase-1b%20deficiency%20aggravates%20pressure%20overload-induced%20cardiac%20hypertrophy%20caused%20by%20lipotoxicity&amp;journal=Circulation&amp;doi=10.1161%2FCIRCULATIONAHA.111.075978&amp;volume=126&amp;pages=1705-1716&amp;publication_year=2012&amp;author=He%2CL">
                    Google Scholar</a> 
                </p></li><li data-counter="38."><p id="ref-CR38">Haynie, K. R., Vandanmagsar, B., Wicks, S. E., Zhang, J. &amp; Mynatt, R. L. Inhibition of carnitine palymitoyltransferase1b induces cardiac hypertrophy and mortality in mice. <i>Diabetes Obes. Metab.</i> <b>16</b>, 757–760 (2014).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1111/dom.12248" data-track-action="article reference" href="https://doi.org/10.1111%2Fdom.12248" aria-label="Article reference 38" data-doi="10.1111/dom.12248">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://riskmusings.substack.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXhtFOlt7bF" aria-label="CAS reference 38">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24330405" aria-label="PubMed reference 38">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4057362" aria-label="PubMed Central reference 38">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 38" href="http://scholar.google.com/scholar_lookup?&amp;title=Inhibition%20of%20carnitine%20palymitoyltransferase1b%20induces%20cardiac%20hypertrophy%20and%20mortality%20in%20mice&amp;journal=Diabetes%20Obes.%20Metab.&amp;doi=10.1111%2Fdom.12248&amp;volume=16&amp;pages=757-760&amp;publication_year=2014&amp;author=Haynie%2CKR&amp;author=Vandanmagsar%2CB&amp;author=Wicks%2CSE&amp;author=Zhang%2CJ&amp;author=Mynatt%2CRL">
                    Google Scholar</a> 
                </p></li><li data-counter="39."><p id="ref-CR39">Yagyu, H. et al. Lipoprotein lipase (LpL) on the surface of cardiomyocytes increases lipid uptake and produces a cardiomyopathy. <i>J. Clin. Invest.</i> <b>111</b>, 419–426 (2003).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1172/JCI16751" data-track-action="article reference" href="https://doi.org/10.1172%2FJCI16751" aria-label="Article reference 39" data-doi="10.1172/JCI16751">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=12569168" aria-label="PubMed reference 39">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC151861" aria-label="PubMed Central reference 39">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 39" href="http://scholar.google.com/scholar_lookup?&amp;title=Lipoprotein%20lipase%20%28LpL%29%20on%20the%20surface%20of%20cardiomyocytes%20increases%20lipid%20uptake%20and%20produces%20a%20cardiomyopathy&amp;journal=J.%20Clin.%20Invest.&amp;doi=10.1172%2FJCI16751&amp;volume=111&amp;pages=419-426&amp;publication_year=2003&amp;author=Yagyu%2CH">
                    Google Scholar</a> 
                </p></li><li data-counter="40."><p id="ref-CR40">Losman, J. A., Koivunen, P. &amp; Kaelin, W. G. Jr 2-Oxoglutarate-dependent dioxygenases in cancer. <i>Nat. Rev. Cancer</i> <b>20</b>, 710–726 (2020).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/s41568-020-00303-3" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41568-020-00303-3" aria-label="Article reference 40" data-doi="10.1038/s41568-020-00303-3">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://riskmusings.substack.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXitFGqtLfK" aria-label="CAS reference 40">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33087883" aria-label="PubMed reference 40">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 40" href="http://scholar.google.com/scholar_lookup?&amp;title=2-Oxoglutarate-dependent%20dioxygenases%20in%20cancer&amp;journal=Nat.%20Rev.%20Cancer&amp;doi=10.1038%2Fs41568-020-00303-3&amp;volume=20&amp;pages=710-726&amp;publication_year=2020&amp;author=Losman%2CJA&amp;author=Koivunen%2CP&amp;author=Kaelin%2CWG">
                    Google Scholar</a> 
                </p></li><li data-counter="41."><p id="ref-CR41">Cano-Rodriguez, D. et al. Writing of H3K4Me3 overcomes epigenetic silencing in a sustained but context-dependent manner. <i>Nat. Commun.</i> <b>7</b>, 12284 (2016).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/ncomms12284" data-track-action="article reference" href="https://doi.org/10.1038%2Fncomms12284" aria-label="Article reference 41" data-doi="10.1038/ncomms12284">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2016NatCo...712284C" aria-label="ADS reference 41">ADS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://riskmusings.substack.com/articles/cas-redirect/1:CAS:528:DC%2BC28XhtlaksLzI" aria-label="CAS reference 41">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27506838" aria-label="PubMed reference 41">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4987519" aria-label="PubMed Central reference 41">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 41" href="http://scholar.google.com/scholar_lookup?&amp;title=Writing%20of%20H3K4Me3%20overcomes%20epigenetic%20silencing%20in%20a%20sustained%20but%20context-dependent%20manner&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fncomms12284&amp;volume=7&amp;publication_year=2016&amp;author=Cano-Rodriguez%2CD">
                    Google Scholar</a> 
                </p></li><li data-counter="42."><p id="ref-CR42">Wysocka, J. et al. A PHD finger of NURF couples histone H3 lysine 4 trimethylation with chromatin remodelling. <i>Nature</i> <b>442</b>, 86–90 (2006).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/nature04815" data-track-action="article reference" href="https://doi.org/10.1038%2Fnature04815" aria-label="Article reference 42" data-doi="10.1038/nature04815">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2006Natur.442...86W" aria-label="ADS reference 42">ADS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://riskmusings.substack.com/articles/cas-redirect/1:CAS:528:DC%2BD28XmsFWnt7g%3D" aria-label="CAS reference 42">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16728976" aria-label="PubMed reference 42">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 42" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20PHD%20finger%20of%20NURF%20couples%20histone%20H3%20lysine%204%20trimethylation%20with%20chromatin%20remodelling&amp;journal=Nature&amp;doi=10.1038%2Fnature04815&amp;volume=442&amp;pages=86-90&amp;publication_year=2006&amp;author=Wysocka%2CJ">
                    Google Scholar</a> 
                </p></li><li data-counter="43."><p id="ref-CR43">Lauberth, S. M. et al. H3K4me3 interactions with TAF3 regulate preinitiation complex assembly and selective gene activation. <i>Cell</i> <b>152</b>, 1021–1036 (2013).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.cell.2013.01.052" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.cell.2013.01.052" aria-label="Article reference 43" data-doi="10.1016/j.cell.2013.01.052">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://riskmusings.substack.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXjsFeqsro%3D" aria-label="CAS reference 43">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23452851" aria-label="PubMed reference 43">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3588593" aria-label="PubMed Central reference 43">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 43" href="http://scholar.google.com/scholar_lookup?&amp;title=H3K4me3%20interactions%20with%20TAF3%20regulate%20preinitiation%20complex%20assembly%20and%20selective%20gene%20activation&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2013.01.052&amp;volume=152&amp;pages=1021-1036&amp;publication_year=2013&amp;author=Lauberth%2CSM">
                    Google Scholar</a> 
                </p></li><li data-counter="44."><p id="ref-CR44">Brier, A. B. et al. The KDM5 family is required for activation of pro-proliferative cell cycle genes during adipocyte differentiation. <i>Nucleic Acids Res.</i> <b>45</b>, 1743–1759 (2017).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1093/nar/gkw1156" data-track-action="article reference" href="https://doi.org/10.1093%2Fnar%2Fgkw1156" aria-label="Article reference 44" data-doi="10.1093/nar/gkw1156">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://riskmusings.substack.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXivFynu7o%3D" aria-label="CAS reference 44">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27899593" aria-label="PubMed reference 44">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 44" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20KDM5%20family%20is%20required%20for%20activation%20of%20pro-proliferative%20cell%20cycle%20genes%20during%20adipocyte%20differentiation&amp;journal=Nucleic%20Acids%20Res.&amp;doi=10.1093%2Fnar%2Fgkw1156&amp;volume=45&amp;pages=1743-1759&amp;publication_year=2017&amp;author=Brier%2CAB">
                    Google Scholar</a> 
                </p></li><li data-counter="45."><p id="ref-CR45">Richardson, G. D. Simultaneous assessment of cardiomyocyte DNA synthesis and ploidy: a method to assist quantification of cardiomyocyte regeneration and turnover. <i>J. Vis. Exp.</i> <a href="https://doi.org/10.3791/53979" data-track="click" data-track-action="external reference" data-track-label="10.3791/53979">https://doi.org/10.3791/53979</a> (2016).</p></li><li data-counter="46."><p id="ref-CR46">O’Connell, T. D., Rodrigo, M. C. &amp; Simpson, P. C. Isolation and culture of adult mouse cardiac myocytes. <i>Methods Mol. Biol.</i> <b>357</b>, 271–296 (2007).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17172694" aria-label="PubMed reference 46">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 46" href="http://scholar.google.com/scholar_lookup?&amp;title=Isolation%20and%20culture%20of%20adult%20mouse%20cardiac%20myocytes&amp;journal=Methods%20Mol.%20Biol.&amp;volume=357&amp;pages=271-296&amp;publication_year=2007&amp;author=O%E2%80%99Connell%2CTD&amp;author=Rodrigo%2CMC&amp;author=Simpson%2CPC">
                    Google Scholar</a> 
                </p></li><li data-counter="47."><p id="ref-CR47">Larson, A. C. et al. Self-gated cardiac cine MRI. <i>Magn. Reson. Med.</i> <b>51</b>, 93–102 (2004).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1002/mrm.10664" data-track-action="article reference" href="https://doi.org/10.1002%2Fmrm.10664" aria-label="Article reference 47" data-doi="10.1002/mrm.10664">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=14705049" aria-label="PubMed reference 47">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2396326" aria-label="PubMed Central reference 47">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 47" href="http://scholar.google.com/scholar_lookup?&amp;title=Self-gated%20cardiac%20cine%20MRI&amp;journal=Magn.%20Reson.%20Med.&amp;doi=10.1002%2Fmrm.10664&amp;volume=51&amp;pages=93-102&amp;publication_year=2004&amp;author=Larson%2CAC">
                    Google Scholar</a> 
                </p></li><li data-counter="48."><p id="ref-CR48">Bolger, A. M., Lohse, M. &amp; Usadel, B. Trimmomatic: a flexible trimmer for Illumina sequence data. <i>Bioinformatics</i> <b>30</b>, 2114–2120 (2014).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1093/bioinformatics/btu170" data-track-action="article reference" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtu170" aria-label="Article reference 48" data-doi="10.1093/bioinformatics/btu170">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://riskmusings.substack.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXht1Sqt7nP" aria-label="CAS reference 48">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24695404" aria-label="PubMed reference 48">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4103590" aria-label="PubMed Central reference 48">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 48" href="http://scholar.google.com/scholar_lookup?&amp;title=Trimmomatic%3A%20a%20flexible%20trimmer%20for%20Illumina%20sequence%20data&amp;journal=Bioinformatics&amp;doi=10.1093%2Fbioinformatics%2Fbtu170&amp;volume=30&amp;pages=2114-2120&amp;publication_year=2014&amp;author=Bolger%2CAM&amp;author=Lohse%2CM&amp;author=Usadel%2CB">
                    Google Scholar</a> 
                </p></li><li data-counter="49."><p id="ref-CR49">Dobin, A. et al. STAR: ultrafast universal RNA-seq aligner. <i>Bioinformatics</i> <b>29</b>, 15–21 (2013).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1093/bioinformatics/bts635" data-track-action="article reference" href="https://doi.org/10.1093%2Fbioinformatics%2Fbts635" aria-label="Article reference 49" data-doi="10.1093/bioinformatics/bts635">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://riskmusings.substack.com/articles/cas-redirect/1:CAS:528:DC%2BC38XhvV2gsbnF" aria-label="CAS reference 49">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23104886" aria-label="PubMed reference 49">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 49" href="http://scholar.google.com/scholar_lookup?&amp;title=STAR%3A%20ultrafast%20universal%20RNA-seq%20aligner&amp;journal=Bioinformatics&amp;doi=10.1093%2Fbioinformatics%2Fbts635&amp;volume=29&amp;pages=15-21&amp;publication_year=2013&amp;author=Dobin%2CA">
                    Google Scholar</a> 
                </p></li><li data-counter="50."><p id="ref-CR50">Liao, Y., Smyth, G. K. &amp; Shi, W. featureCounts: an efficient general purpose program for assigning sequence reads to genomic features. <i>Bioinformatics</i> <b>30</b>, 923–930 (2014).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1093/bioinformatics/btt656" data-track-action="article reference" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtt656" aria-label="Article reference 50" data-doi="10.1093/bioinformatics/btt656">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://riskmusings.substack.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXltFGqu7c%3D" aria-label="CAS reference 50">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24227677" aria-label="PubMed reference 50">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 50" href="http://scholar.google.com/scholar_lookup?&amp;title=featureCounts%3A%20an%20efficient%20general%20purpose%20program%20for%20assigning%20sequence%20reads%20to%20genomic%20features&amp;journal=Bioinformatics&amp;doi=10.1093%2Fbioinformatics%2Fbtt656&amp;volume=30&amp;pages=923-930&amp;publication_year=2014&amp;author=Liao%2CY&amp;author=Smyth%2CGK&amp;author=Shi%2CW">
                    Google Scholar</a> 
                </p></li><li data-counter="51."><p id="ref-CR51">Love, M. I., Huber, W. &amp; Anders, S. Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. <i>Genome Biol.</i> <b>15</b>, 550 (2014).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1186/s13059-014-0550-8" data-track-action="article reference" href="https://doi.org/10.1186%2Fs13059-014-0550-8" aria-label="Article reference 51" data-doi="10.1186/s13059-014-0550-8">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25516281" aria-label="PubMed reference 51">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4302049" aria-label="PubMed Central reference 51">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 51" href="http://scholar.google.com/scholar_lookup?&amp;title=Moderated%20estimation%20of%20fold%20change%20and%20dispersion%20for%20RNA-seq%20data%20with%20DESeq2&amp;journal=Genome%20Biol.&amp;doi=10.1186%2Fs13059-014-0550-8&amp;volume=15&amp;publication_year=2014&amp;author=Love%2CMI&amp;author=Huber%2CW&amp;author=Anders%2CS">
                    Google Scholar</a> 
                </p></li><li data-counter="52."><p id="ref-CR52">Wu, F. et al. Spurious transcription causing innate immune responses is prevented by 5-hydroxymethylcytosine. <i>Nat. Genet.</i> <b>55</b>, 100–111 (2023).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/s41588-022-01252-3" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41588-022-01252-3" aria-label="Article reference 52" data-doi="10.1038/s41588-022-01252-3">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://riskmusings.substack.com/articles/cas-redirect/1:CAS:528:DC%2BB38XjtFCktr7E" aria-label="CAS reference 52">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=36539616" aria-label="PubMed reference 52">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 52" href="http://scholar.google.com/scholar_lookup?&amp;title=Spurious%20transcription%20causing%20innate%20immune%20responses%20is%20prevented%20by%205-hydroxymethylcytosine&amp;journal=Nat.%20Genet.&amp;doi=10.1038%2Fs41588-022-01252-3&amp;volume=55&amp;pages=100-111&amp;publication_year=2023&amp;author=Wu%2CF">
                    Google Scholar</a> 
                </p></li></ol><p><a data-track="click" data-track-action="download citation references" data-track-label="link" rel="nofollow" href="https://citation-needed.springer.com/v2/references/10.1038/s41586-023-06585-5?format=refman&amp;flavour=references">Download references<svg width="16" height="16" focusable="false" role="img" aria-hidden="true"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-download"></use></svg></a></p></div></div></div></section></div><section data-title="Acknowledgements"><div id="Ack1-section"><h2 id="Ack1">Acknowledgements</h2><p>We thank S. Kreutzer, K. Mattonet, S. Krüger, K. Khrievono and K. Richter for technical support; and U. Gärtner for carrying out electron microscopy analysis. This work was supported by the Excellence Cluster Cardio Pulmonary System (CPI), the DFG Transregional Collaborative Research Centres 81 (TP A02) and 267 (TP A05), the DFG Collaborative Research Centres 1213 (TP A02 and B02) and 1531 (TP B08) and the German Centre for Cardiovascular Research. The work of M.P. was supported by the European Research Council Consolidator Grant EMERGE (no. 773047). M.P., X.Y. and T.B. are members of the German Center for Cardiovascular Research (DZHK).</p></div></section><section data-title="Funding"><div id="Fun-section"><h2 id="Fun">Funding</h2><p>Open access funding provided by Max Planck Society.</p></div></section><section aria-labelledby="author-information" data-title="Author information"><div id="author-information-section"><h2 id="author-information">Author information</h2><div id="author-information-content"><p><span id="author-notes">Author notes</span></p><ol><li id="nAff7"><p>Michael Potente</p><p>Present address: Berlin Institute of Health, Charité-Universitätsmedizin Berlin, Berlin, Germany</p></li></ol><h3 id="affiliations">Authors and Affiliations</h3><ol><li id="Aff1"><p>Department of Cardiac Development and Remodeling, Max Planck Institute for Heart and Lung Research, Bad Nauheim, Germany</p><p>Xiang Li, Fan Wu, Stefan Günther, Mario Looso, Carsten Kuenne, Ting Zhang, Marion Wiesnet, Astrid Wietelmann, Ann Atzberger, Xuejun Yuan &amp; Thomas Braun</p></li><li id="Aff2"><p>Institute for Vascular Signaling, Centre for Molecular Medicine, Goethe-University, Frankfurt am Main, Germany</p><p>Stephan Klatt, Sven Zukunft &amp; Ingrid Fleming</p></li><li id="Aff3"><p>Metabolomics Core Technology Platform, Centre for Organismal Studies (COS), Heidelberg University, Heidelberg, Germany</p><p>Gernot Poschet</p></li><li id="Aff4"><p>Angiogenesis and Metabolism Laboratory, Max Planck Institute for Heart and Lung Research, Bad Nauheim, Germany</p><p>Michael Potente</p></li><li id="Aff5"><p>Max Delbrück Center for Molecular Medicine, Helmholtz Association of German Research Centres, Berlin, Germany</p><p>Michael Potente</p></li><li id="Aff6"><p>Instituto de Investigacion en Biomedicina de Buenos Aires (IBioBA) - CONICET - Partner Institute of the Max Planck Society, Buenos Aires, Argentina</p><p>Xuejun Yuan &amp; Thomas Braun</p></li></ol><h3 id="contributions">Contributions</h3><p>X.Y. and T.B. conceived and designed experiments. X.L. carried out most of the experiments, analysed the data and prepared figures. S.K., S.Z., I.F. and G.P. carried out the metabolic analysis. S.G. carried out next-generation deep sequencing. M.L., S.G. and C.K. carried out bio-informatics analysis. T.Z. contributed to analysis of ChIP–seq experiments. M.W. conducted cardiac surgery. A.W. carried out the MRI measurements. M.P. provided transgenic mouse lines and critical comments. F.W. and A.A. contributed to experimental design, data analysis, discussions and advice. T.B., X.Y. and X.L. wrote the manuscript.</p><h3 id="corresponding-author">Corresponding authors</h3><p id="corresponding-author-list">Correspondence to
                <a id="corresp-c1" href="mailto:Xuejun.Yuan@mpi-bn.mpg.de">Xuejun Yuan</a> or <a id="corresp-c2" href="mailto:Thomas.Braun@mpi-bn.mpg.de">Thomas Braun</a>.</p></div></div></section><section data-title="Ethics declarations"><div id="ethics-section"><h2 id="ethics">Ethics declarations</h2><div id="ethics-content">
            
              <h3 id="FPar2">Competing interests</h3>
              <p>The authors declare no competing interests.</p>
            
          </div></div></section><section data-title="Peer review"><div id="peer-review-section"><h2 id="peer-review">Peer review</h2><div id="peer-review-content">
            
            
              <h3 id="FPar1">Peer review information</h3>
              <p><i>Nature</i> thanks Rong Tian and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#MOESM3">Peer reviewer reports</a> are available.</p>
            
          </div></div></section><section data-title="Additional information"><div id="additional-information-section"><h2 id="additional-information">Additional information</h2><p><b>Publisher’s note</b> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div></section><section data-title="Extended data figures and tables"><div id="Sec30-section"><h2 id="Sec30">Extended data figures and tables</h2><div id="Sec30-content"><div data-test="supplementary-info"><div data-test="supp-item" id="Fig6"><h3><a data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 1 inhibition of cpt1 stimulates" href="https://riskmusings.substack.com/articles/s41586-023-06585-5/figures/6" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41586-023-06585-5/MediaObjects/41586_2023_6585_Fig6_ESM.jpg">Extended Data Fig. 1 Inhibition of CPT1 stimulates cell cycle activity and hypertrophic growth of neonatal CMs.</a></h3><div data-component="thumbnail-container"><p><b>a</b>, Heat map representing expression of genes involved in glycolysis, cell cycle, fatty acid oxidation, and the Krebs Cycle. Publicly available RNA-seq data of CMs isolated from mice at E14.5, P1-2, P60, and adult mice one week after transaortic constriction were used (<a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE79883">GSE79883</a>). <b>b</b>, RT-qPCR analysis of <i>Cpt1a</i> (n = 3) and <i>Cpt1b</i> (n = 4) expression in mouse hearts at different developmental stages (P0, P3, P7, P14). <i>36b4</i> expression served as reference. <b>c</b>, EdU incorporation in neonatal CMs, 72 h after DMSO or etomoxir treatment (n = 5, each). Areas of enlarged images are labelled by white frames. Quantification of EdU<sup>+</sup>sarc-actinin<sup>+</sup> CMs is in the right panel. Scale bar: 50 μm. <b>d</b>, Immunofluorescence staining of neonatal CMs for sarc-actinin and Ki67 (n = 4) or pH3 (Ser10) (n = 3) with or without etomoxir treatment. Quantification of Ki67<sup>+</sup>sarc-actinin<sup>+</sup> and pH3<sup>+</sup>sarc-actinin<sup>+</sup> CMs is in the right panels. Scale bar: 50 μm. <b>e</b>, Western blot analysis of Cyclin E1 in P0-1 CMs after 3-days-culture with DMSO or etomoxir (n = 3, each). Pan-actin served as loading control. <b>f</b>, RT-qPCR analysis of hypertrophy-associated genes in P0-1 CMs, 72 h after DMSO or etomoxir treatment (n = 4, each). <i>36b4</i> was used as reference gene. <b>g</b>, RT-qPCR analysis of <i>Cpt1a</i> and <i>Cpt1b</i> expression in CMs from 10-weeks-old <i>Ctrl</i><sup><i>Cre</i></sup> and <i>Cpt1b</i><sup><i>cKO</i></sup> mice (n = 3, each). <i>36b4</i> served as reference gene. <b>h</b>, Western blot analysis of CPT1B in CMs from 10-weeks-old <i>Ctrl</i><sup><i>Cre</i></sup> and <i>Cpt1b</i><sup><i>cKO</i></sup> mice (n = 2, each). Pan-actin served as loading control. <b>i</b>, BW, HW, and HW/BW ratios of P7 <i>Ctrl</i><sup><i>Cre</i></sup> (n = 6) and <i>Cpt1b</i><sup><i>cKO</i></sup> (n = 4) mice. <b>j</b>, H&amp;E-stained heart sections from P7 <i>Ctrl</i><sup><i>Cre</i></sup> and <i>Cpt1b</i><sup><i>cKO</i></sup> mice (n = 3, each). Scale bar: 500 μm. <b>k</b>, Images of hearts from 10-weeks-old <i>Ctrl</i><sup><i>Cre</i></sup> and <i>Cpt1b</i><sup><i>cKO</i></sup> mice (n = 3, each). Scale bar: 2 mm. <b>l</b>, BW and HW of 10-weeks-old <i>Ctrl</i><sup><i>Cre</i></sup> and <i>Cpt1b</i><sup><i>cKO</i></sup> mice (n = 5, each). <b>m</b>, H&amp;E staining of heart sections from 10-weeks-old <i>Ctrl</i><sup><i>Cre</i></sup> and <i>Cpt1b</i><sup><i>cKO</i></sup> mice (n = 3, each). Scale bar: 600 μm. <b>n</b>, Immunofluorescence staining for α-WGA and quantification of cell surface areas on heart sections from 10-weeks-old <i>Ctrl</i><sup><i>Cre</i></sup> and <i>Cpt1b</i><sup><i>cKO</i></sup> mice (n = 3, each). Scale bar: 20 μm. <b>o</b>, Immunofluorescence images of sarc-actinin<sup>+</sup> CMs from 10-weeks-old <i>Ctrl</i><sup><i>Cre</i></sup> and <i>Cpt1b</i><sup><i>cKO</i></sup> mice (n = 3, each). Quantifications of cell length, width, and surface area are in the right panels. Scale bar: 50 μm. Error bar represents mean ± s.e.m. N-numbers refer to individual mice in b, e-o. N-numbers refer to independent experiments in c, d. Two-tailed, unpaired student t-tests for statistical analysis in c-g, i, l, n-o. One-way ANOVA with Tukey tests for correction of multiple comparisons in b. *<i>P</i> &lt; 0.05, **<i>P</i> &lt; 0.01, ****<i>P</i> &lt; 0.0001. For gel source data in e, h, see <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#MOESM1">Supplementary Information</a>.</p><p>
                        <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#MOESM10">Source data</a>
                      </p></div></div><div data-test="supp-item" id="Fig7"><h3><a data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 2 deletion of cpt1b in adult cm" href="https://riskmusings.substack.com/articles/s41586-023-06585-5/figures/7" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41586-023-06585-5/MediaObjects/41586_2023_6585_Fig7_ESM.jpg">Extended Data Fig. 2 Deletion of <i>Cpt1b</i> in adult CMs reverses cell cycle arrest.</a></h3><div data-component="thumbnail-container"><p><b>a</b>–<b>c</b>, Immunofluorescence staining for Ki67 (a), pH3 (b), Aurora B (c), and sarc-actinin on heart sections from 10-weeks-old <i>Ctrl</i><sup><i>Cre</i></sup> and <i>Cpt1b</i><sup><i>cKO</i></sup> mice (n = 3, each). Scale bar: 50 μm. <b>d</b>, Outline of the EdU incorporation assays using isolated cardiac nuclei. <b>e</b>, Strategy for quantification of CMs numbers in adult mouse hearts. <b>f</b>, Trichrome staining of heart sections from 10-weeks-old <i>Ctrl</i><sup><i>Cre</i></sup> and <i>Cpt1b</i><sup><i>cKO</i></sup> mice (n = 3, each). Scale bar: 600 μm. <b>g</b>-<b>h</b>, Cardiac MRI analysis of 10-weeks-old <i>Ctrl</i><sup><i>Cre</i></sup> and <i>Cpt1b</i><sup><i>cKO</i></sup> mice (n = 6, each). <b>i</b>, RT-qPCR analysis of <i>Cpt1a</i> and <i>Cpt1b</i> expression in CMs from adult hearts of Ctrl and <i>Cpt1b</i><sup><i>iKO</i></sup> mice, 4 weeks after termination of tamoxifen treatment for <i>Cpt1b</i> deletion (n = 3, each). <i>36b4</i> served as reference gene. <b>j</b>, Analysis of BW (left panel) and HW (right panel) of Ctrl and <i>Cpt1b</i><sup><i>iKO</i></sup> mice 4 and 8 weeks after TAM injection (Ctrl 4 and 8 weeks after TAM, <i>Cpt1b</i><sup><i>iKO</i></sup> 8 weeks after TAM, n = 4; <i>Cpt1b</i><sup><i>iKO</i></sup> 4 weeks after TAM, n = 3). <b>k</b>, Immunofluorescence staining for α-WGA and quantification of cell surface areas on heart sections of Ctrl and <i>Cpt1b</i><sup><i>iKO</i></sup> mice, 8 weeks after termination of tamoxifen treatment for <i>Cpt1b</i> deletion (n = 4, each). Scale bar: 20 μm. <b>l</b>, Macroscopic images of Ctrl and <i>Cpt1b</i><sup><i>iKO</i></sup> hearts, 4 and 8 weeks after termination of tamoxifen treatment for <i>Cpt1b</i> deletion. Scale bar: 2 mm. <b>m</b>-<b>n</b>, H&amp;E and Trichrome staining of heart sections from Ctrl and <i>Cpt1b</i><sup><i>iKO</i></sup> mice, 4 and 8 weeks after termination of tamoxifen treatment for <i>Cpt1b</i> deletion (n = 3, each). Scale bar: 600 μm. <b>o</b>, MRI analysis of left ventricle ejection fraction (LVEF), ESV (End-Systolic Volume), EDV (End-Diastolic Volume) and cardiac output of Ctrl (n = 7) and <i>Cpt1b</i><sup><i>iKO</i></sup> (n = 8) mice, 8 weeks after termination of tamoxifen treatment for <i>Cpt1b</i> deletion. <b>p</b>-<b>r</b>, Immunofluorescence staining for Ki67 (p), pH3 (q), Aurora B (r), and sarc-actinin on heart sections from Ctrl and <i>Cpt1b</i><sup><i>iKO</i></sup> mice. Scale bar: 50 μm, p, n = 4; q and r, n = 3. Error bar represents mean ± s.e.m. N-numbers refer to individual mice. Two-tailed, unpaired student t-tests for statistical analysis in g,i,k,o. One-way ANOVA with Tukey tests for correction of multiple comparisons in j. *<i>P</i> &lt; 0.05, ***<i>P</i> &lt; 0.001, ****<i>P</i> &lt; 0.0001.</p><p>
                        <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#MOESM11">Source data</a>
                      </p></div></div><div data-test="supp-item" id="Fig8"><h3><a data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 3 abrogation of fao reverts mat" href="https://riskmusings.substack.com/articles/s41586-023-06585-5/figures/8" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41586-023-06585-5/MediaObjects/41586_2023_6585_Fig8_ESM.jpg">Extended Data Fig. 3 Abrogation of FAO reverts maturation of CMs.</a></h3><div data-component="thumbnail-container"><p><b>a</b>-<b>b</b>, GO-term enrichment analysis of differentially expressed genes (DEGs) in <i>Cpt1b</i><sup><i>cKO</i></sup> and <i>Cpt1b</i><sup><i>iKO</i></sup> CMs. Enriched biological processes for up-regulated genes are in red and for down-regulated genes in blue. <b>c</b>, Heat map of selected DEGs involved in cell cycle (Magenta), maturation (red), contraction (blue), and HIF1A signaling (light blue) based on z-score transformed normalized DESeq counts. <b>d</b>, EM images showing the cytoarchitecture of CMs isolated from 10-weeks-old <i>Ctrl</i><sup><i>Cre</i></sup> and <i>Cpt1b</i><sup><i>cKO</i></sup> mice (n = 3, each). Scar bar: 500 nm. <b>e</b>, left: IF staining of isolated single CMs from <i>Ctrl</i><sup><i>Cre</i></sup> and <i>Cpt1b</i><sup><i>cKO</i></sup> adult hearts for sarcomeric actinin (identical exposure time for both CMs); right: quantification of sarcomere density based on IF staining for sarcomeric actinin, comparing <i>Ctrl</i><sup><i>Cre</i></sup> and <i>Cpt1b</i><sup><i>cKO</i></sup> adult CMs (n = 3, each). Scale bar: 10 μm. <b>f</b>, RT-qPCR analysis of selected genes in CMs from 10-weeks-old <i>Ctrl</i><sup><i>Cre</i></sup> and <i>Cpt1b</i><sup><i>cKO</i></sup> mice. <i>36b4</i> served as reference gene (n = 3, each). <b>g</b>, Overlap of up-regulated (left panel) and down-regulated (right panel) genes in <i>Cpt1b</i><sup><i>cKO</i></sup> and <i>Cpt1b</i><sup><i>iKO</i></sup> CMs compared to control CMs. <b>h</b>, GO term enrichment analysis of genes from the overlap shown in (g). Error bar represents mean ± s.e.m. N-numbers refer to individual mice. Two-tailed, unpaired student t-tests for statistical analysis in (e,f). *<i>P</i> &lt; 0.05, **<i>P</i> &lt; 0.01, ***<i>P</i> &lt; 0.001, ****<i>P</i> &lt; 0.0001.</p><p>
                        <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#MOESM12">Source data</a>
                      </p></div></div><div data-test="supp-item" id="Fig9"><h3><a data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 4 inhibition of fao in cms incr" href="https://riskmusings.substack.com/articles/s41586-023-06585-5/figures/9" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41586-023-06585-5/MediaObjects/41586_2023_6585_Fig9_ESM.jpg">Extended Data Fig. 4 Inhibition of FAO in CMs increases HIF1A levels and reduces DNA damage.</a></h3><div data-component="thumbnail-container"><p><b>a</b>, Western blot analysis of HIF1A in CMs from 10-weeks-old <i>Ctrl</i><sup><i>Cre</i></sup> and <i>Cpt1b</i><sup><i>cKO</i></sup> mice (n = 3, each). Pan-actin was used as loading control. For gel source data, see <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#MOESM1">Supplementary Information</a>. <b>b</b>, TUNEL assays on heart sections from 20–25-weeks-old <i>Ctrl</i><sup><i>Cre</i></sup> and <i>Cpt1b</i><sup><i>cKO</i></sup> mice (n = 3, each). DNase-treated samples were used as positive control. Scale bar: 20 μm. <b>c</b>, Immunofluorescence staining of 10-weeks-old <i>Ctrl</i><sup><i>Cre</i></sup> and <i>Cpt1b</i><sup><i>cKO</i></sup> heart sections using DAPI and DHR123. H<sub>2</sub>O<sub>2</sub> treated heart sections were used as positive controls. Quantification of mean fluorescence intensity (MFI) per image is shown in the right panel (n = 3, each). <b>d</b>, Immunofluorescence staining of 10-weeks-old <i>Ctrl</i><sup><i>Cre</i></sup> and <i>Cpt1b</i><sup><i>cKO</i></sup> heart sections using γH2A.X and sarc-actinin antibodies. Quantification of the percentage of γH2A.X<sup>+</sup>sarc-actinin<sup>+</sup> CMs per heart section is shown in the right panel (n = 3, each). Error bar represents mean ± s.e.m. N-numbers refer to individual mice. Two-tailed, unpaired student t-tests for statistical analysis in c and d. **<i>P</i> &lt; 0.01.</p><p>
                        <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#MOESM13">Source data</a>
                      </p></div></div><div data-test="supp-item" id="Fig10"><h3><a data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 5 abrogation of fao by deletion" href="https://riskmusings.substack.com/articles/s41586-023-06585-5/figures/10" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41586-023-06585-5/MediaObjects/41586_2023_6585_Fig10_ESM.jpg">Extended Data Fig. 5 Abrogation of FAO by deletion of <i>Cpt1b</i> protects from I/R damage and enables heart regeneration.</a></h3><div data-component="thumbnail-container"><p><b>a</b>, <b>b</b>, Images of <i>Ctrl</i><sup><i>Cre</i></sup> and <i>Cpt1b</i><sup><i>cKO</i></sup> (a) and Ctrl and <i>Cpt1b</i><sup><i>iKO</i></sup> (b) heart sections after Evans blue injection and TTC staining, 24 h after I/R injury. I/R surgery was done using 7-weeks-old mice (a) or 14-weeks-old mice, 3 weeks after completion of TAM treatment (b). <b>c</b>, Staining for Ki67 and Sarc-actinin (left panel), and Aurora B and cTnT (right panel) on sections from hearts of adult <i>Ctrl</i><sup><i>Cre</i></sup> and <i>Cpt1b</i><sup><i>cKO</i></sup> mice, 72 h after I/R injury (<i>Ctrl</i><sup><i>cre</i></sup> n = 3, <i>Cpt1b</i><sup><i>cKO</i></sup> n = 4). Scale bar: 50 μm (left panel), 20 μm (right panel). <b>d</b>, Immunofluorescence staining for detyrosinated tubulin and cTnI using sections from <i>Ctrl</i><sup><i>Cre</i></sup> and <i>Cpt1b</i><sup><i>cKO</i></sup> hearts, 24 h after IR surgery (n = 3, each). <b>e</b>, MRI-based assessment of heart functions in <i>Ctrl</i><sup><i>Cre</i></sup> and <i>Cpt1b</i><sup><i>cKO</i></sup> mice before and 48 h after I/R surgery (0 h, <i>Ctrl</i><sup><i>Cre</i></sup> n = 8, <i>Cpt1b</i><sup><i>cKO</i></sup> n = 7; 48 h, n = 7, each). <b>f</b>, <b>g</b>, Cell viability assay of CMs isolated from adult <i>Ctrl</i><sup><i>Cre</i></sup> and <i>Cpt1b</i><sup><i>cKO</i></sup> hearts after exposure to 1% O<sub>2</sub> for 18 hrs. The experimental approach is outlined in the upper panel. Quantification of dead cells (EthD-1<sup>+</sup>) is shown in g (n = 3, each). <b>h</b>, Western blot analysis and quantification of CPT1B levels in <i>Cpt1b</i><sup><i>iKO</i></sup> mice, 0, 2, 5, and 10 days after initiation of TAM treatment (n = 3, each). Pan-actin served as loading control. For gel source data, see <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#MOESM1">Supplementary Information</a>. Error bars represent mean ± s.e.m. N-numbers refer to independent experiments in g, others refer to individual mice. Two-tailed, unpaired student t-tests for statistical analysis in c, g. Two-way ANOVA with Tukey tests for correction of multiple comparisons in e and one-way ANOVA with Tukey tests for correction of multiple comparisons in h. ns: not significant, *<i>P</i> &lt; 0.05, **<i>P</i> &lt; 0.01, ****<i>P</i> &lt; 0.0001.</p><p>
                        <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#MOESM14">Source data</a>
                      </p></div></div><div data-test="supp-item" id="Fig11"><h3><a data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 6 characterization of metabolic" href="https://riskmusings.substack.com/articles/s41586-023-06585-5/figures/11" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41586-023-06585-5/MediaObjects/41586_2023_6585_Fig11_ESM.jpg">Extended Data Fig. 6 Characterization of metabolic changes in <i>Ctp1b</i>-deficient CMs.</a></h3><div data-component="thumbnail-container"><p><b>a</b>, Oxygen consumption rate (OCR) measurements of adult CMs isolated from <i>Ctrl</i><sup><i>Cre</i></sup> and <i>Cpt1b</i><sup><i>cKO</i></sup> hearts after addition of Palmitate-BSA (long chain fatty acids (LCFA), left panel), medium chain fatty acids (MCFA, middle panel) and short chain fatty acids (SCFA, right panel) using the Mito Stress kit for Seahorse instruments. <b>b</b>–<b>d</b>, Quantification of acylcarnitine (b), free carnitine (c), and acetyl-CoA (d) in CMs isolated from adult <i>Ctrl</i><sup><i>Cre</i></sup> and <i>Cpt1b</i><sup><i>cKO</i></sup> mice (n = 8, each). <b>e</b>, Western blot analysis and quantification of enzymes involved in acetyl-CoA production (n = 3, each). Pan-actin was utilized as loading control. <b>f</b>, Activated metabolic pathways in adult <i>Cpt1b</i><sup><i>cKO</i></sup> CMs based on targeted metabolome analysis. <b>g</b>, <b>h</b>, Fold-changes of amino acids (g) and biogenic amines (h) in <i>Cpt1b</i><sup><i>cKO</i></sup> compared to <i>Ctrl</i><sup><i>Cre</i></sup> CMs (n = 8, each). <b>i</b>, <b>j</b>, OCR analysis of adult CMs from <i>Ctrl</i><sup><i>Cre</i></sup> and <i>Cpt1b</i><sup>cKO</sup> mice using the Mito Stress kit for Seahorse instruments. Palmitate, glucose, BCAA and pyruvate were added as substrates for energy production. Basal and maximal respiration rates are shown in j (n = 3, each). <b>k</b>, Quantification of western blot analysis for enzymes involved in αKG production (n = 3, each). <b>l</b>, Western blot analysis of enzymes of the αKG dehydrogenase complex in adult <i>Ctrl</i><sup><i>Cre</i></sup> and <i>Cpt1b</i><sup><i>cKO</i></sup> CMs (n = 3, each). <b>m</b>, Western blot analysis of Krebs Cycle enzymes in adult <i>Ctrl</i><sup><i>Cre</i></sup> and <i>Cpt1b</i><sup><i>cKO</i></sup> CMs (n = 5, each). <b>n</b>, Quantification of mitochondrial DNA copy number using DNA from adult CMs of <i>Ctrl</i><sup><i>Cre</i></sup> and <i>Cpt1b</i><sup><i>cKO</i></sup> mice (n = 4, each). <i>H19</i> and <i>Mx1</i> were used as controls. Error bar represents mean ± s.e.m. N-numbers refer to individual mice. Two-tailed, unpaired student t-tests for statistical analysis in b-e, g-h, k-n. One-way ANOVA with Tukey tests for correction of multiple comparisons in j. *<i>P</i> &lt; 0.05, **<i>P</i> &lt; 0.01, ***<i>P</i> &lt; 0.001, ****<i>P</i> &lt; 0.0001. For gel source data in e,l,m, see <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#MOESM1">Supplementary Information</a>.</p><p>
                        <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#MOESM15">Source data</a>
                      </p></div></div><div data-test="supp-item" id="Fig12"><h3><a data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 7 cm-specific inactivation of o" href="https://riskmusings.substack.com/articles/s41586-023-06585-5/figures/12" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41586-023-06585-5/MediaObjects/41586_2023_6585_Fig12_ESM.jpg">Extended Data Fig. 7 CM-specific inactivation of <i>Ogdh</i> in adult mice induces hyperplastic and hypertrophic cardiac growth, similar to <i>Cpt1b</i>-deficient mice.</a></h3><div data-component="thumbnail-container"><p><b>a</b>–<b>d</b>, Fold-changes of diacylglycerols (a), triacylglycerols (b,c) and CoAs (d) in <i>Cpt1b</i><sup><i>cKO</i></sup> compared to <i>Ctrl</i><sup><i>Cre</i></sup> CMs (n = 8, each). <b>e</b>, RT-qPCR expression analysis of <i>Lpl</i> in CMs isolated from adult <i>Ctrl</i><sup><i>Cre</i></sup> and <i>Cpt1b</i><sup><i>cKO</i></sup> mice (<i>Ctrl</i><sup><i>Cre</i></sup> n = 4, <i>Cpt1b</i><sup><i>cKO</i></sup> n = 3). <i>36b4</i> served as reference gene. <b>f</b>, Generation of <i>Ogdh</i> conditional KO mice. <b>g</b>, Western blot analysis of OGDH in CMs from Ctrl and <i>Ogdh</i><sup><i>iKO</i></sup> mice, 3-4 weeks after termination of tamoxifen treatment for <i>Ogdh</i> deletion (n = 3, each). Pan-actin was used as sample processing control. For gel source data, see <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#MOESM1">Supplementary Information</a>. <b>h</b>, Volcano plot showing differentially present metabolites in CMs of Ctrl and <i>Ogdh</i><sup><i>iKO</i></sup> mice. Red dots indicate increased and blue dots reduced concentrations of metabolites in <i>Ogdh</i><sup><i>iKO</i></sup> compared to Ctrl CMs. <b>i</b>, HW, BW and HW/BW ratios of Ctrl and <i>Ogdh</i><sup><i>iKO</i></sup> mice, 3-4 weeks after termination of tamoxifen treatment for <i>Ogdh</i> deletion (Ctrl n = 5, <i>Ogdh</i><sup><i>iKO</i></sup> n = 7). <b>j</b>–<b>l</b>, Immunofluorescence analysis of heart sections from Ctrl and <i>Ogdh</i><sup><i>iKO</i></sup> mice for Ki67 (j), pH3 (k) and Aurora B (l), counterstained for sarc-actinin to identify CMs. Scale bar: 50 μm, (j, Ctrl n = 5, <i>Ogdh</i><sup><i>iKO</i></sup> n = 4; k and l, n = 4, each). <b>m</b>, RT-qPCR analysis of markers related to CMs maturation in Ctrl and <i>Ogdh</i><sup><i>iKO</i></sup> mice, 3-4 weeks after termination of tamoxifen treatment for <i>Ogdh</i> deletion (n = 4, each). <i>36b4</i> served as reference gene. Error bar represents mean ± s.e.m. N-numbers refer to individual mice. Two-tailed, unpaired student t-tests for statistical analysis in a-e, g,i,m. *<i>P</i> &lt; 0.05, **<i>P</i> &lt; 0.01, ***<i>P</i> &lt; 0.001, ****<i>P</i> &lt; 0.0001.</p><p>
                        <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#MOESM16">Source data</a>
                      </p></div></div><div data-test="supp-item" id="Fig13"><h3><a data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 8 analysis of epigenetic and ex" href="https://riskmusings.substack.com/articles/s41586-023-06585-5/figures/13" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41586-023-06585-5/MediaObjects/41586_2023_6585_Fig13_ESM.jpg">Extended Data Fig. 8 Analysis of epigenetic and expression changes in CMs after inactivation of <i>Cpt1b</i> or <i>Ogdh</i> in adult mice.</a></h3><div data-component="thumbnail-container"><p><b>a</b>, Quantification of histone methylation modifications using FACS-isolated CM nuclei from 10-weeks-old <i>Ctrl</i><sup><i>Cre</i></sup> and <i>Cpt1b</i><sup><i>cKO</i></sup> mice (n = 3, each). <b>b</b>, Western blot analysis of H3K4me3 in CMs from Ctrl and <i>Ogdh</i><sup><i>iKO</i></sup> mice, 3-4 weeks after termination of tamoxifen treatment for <i>Ogdh</i> deletion (n = 3, each). H3 was used as loading control. For gel source data, see <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#MOESM1">Supplementary Information</a>. <b>c</b>, Expression levels of selected <i>Jmjc</i> domain-containing histone demethylase (upper panel) and histone methyltransferase (lower panel) genes based on log2-transformation of DESeq-normalized counts (n = 3, each). <b>d</b>, Quantification of SAM concentrations in adult CMs from 10-weeks-old <i>Ctrl</i><sup><i>Cre</i></sup> and <i>Cpt1b</i><sup><i>cKO</i></sup> mice (n = 4, each). <b>e</b>, GO-terms of genes with differentially present H3K4me3 peaks in <i>Cpt1b</i><sup><i>cKO</i></sup> CMs. Enriched biological processes for genes with increased peaks are represented in red and for genes with reduced peaks in blue. <b>f</b>, Heat map of genes with reduced H3K4me3 ChIP-seq signals on promoters normalized for reads mapped in peaks. Corresponding coverage plots are shown in the top panels. <b>g</b>, Biological processes enriched for genes containing broad and narrow H3K4me3 peaks. Enriched GO-terms for genes with broad H3K4me3 peaks are shown in blue and for genes with narrow H3K4me3 peaks in red. <b>h</b>, Coverage plot of H3K4me3 peaks of genes with an overlap as shown in (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#Fig4">4d</a>) between reduced expression and reduced H3K4me3 deposition in <i>Cpt1b</i><sup><i>cKO</i></sup> compared to <i>Ctrl</i><sup><i>Cre</i></sup> CMs. <b>i</b>-<b>k</b>, ChIP-qPCR analysis of H3K4me3, H3K9me3 and H3K27me3 enrichment in genes related to cardiac maturation using FACS-isolated adult CMs nuclei from 10-weeks-old <i>Ctrl</i><sup><i>Cre</i></sup> and <i>Cpt1b</i><sup><i>cKO</i></sup> mice (i, <i>Mylk3 Exon1</i> Ctrl n = 4, others n = 5; j and k, n = 5, each). Error bar represents mean ± s.e.m. N-numbers refer to individual mice. Two-tailed, unpaired student t-tests for statistical analysis in a-b, d, i-k. Wald-test with Benjamini–Hochberg correction in c. *<i>P</i> &lt; 0.05, **<i>P</i> &lt; 0.01, ***<i>P</i> &lt; 0.001.</p><p>
                        <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#MOESM17">Source data</a>
                      </p></div></div><div data-test="supp-item" id="Fig14"><h3><a data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 9 αkg stimulates proliferation " href="https://riskmusings.substack.com/articles/s41586-023-06585-5/figures/14" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41586-023-06585-5/MediaObjects/41586_2023_6585_Fig14_ESM.jpg">Extended Data Fig. 9 αKG stimulates proliferation of CMs.</a></h3><div data-component="thumbnail-container"><p><b>a</b>, RT-qPCR analysis of P0-1 neonatal CMs treated with DMSO or αKG (<i>Acta1</i>, <i>Nppa</i>, <i>Nppb</i>, <i>Myh7, Tnni3</i>, <i>Mylk3</i>, n = 4 in DMSO and n = 3 in αKG; <i>Myocd</i>, n = 5 in DMSO and n = 4 in αKG). <i>36b4</i> served as reference gene. <b>b</b>, Quantification of histone methylation levels in P0-1 neonatal CMs after 3-days in presence of DMSO or αKG (H3K27me3, n = 4; others n = 3, each). <b>c</b>, Quantification of αKG levels in P0-1 CMs transduced with <i>Idh3b</i> and <i>Idh3g</i> lentiviruses (n = 4, each). <b>d</b>, Western blot analysis of His-tagged IDH3B, IDH3G and H3K4me3 in neonatal CMs after transduction with <i>Idh3b</i> and <i>Idh3g</i> lentiviruses. Quantification of H3K4me3 levels is shown in the right panels (n = 3, each). H3 and pan-actin were used as internal controls. For gel source data, see <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#MOESM1">Supplementary Information</a>. <b>e</b>, ChIP-qPCR showing reduced enrichment of H3K4me3 in key cardiac maturation genes with broad but not with narrow H3K4me3 peaks (<i>Snx19</i>) in P0-1 CMs after transduction of <i>Idh3b</i> and <i>Idh3g</i> (Ctrl n = 5, <i>Idh3b</i>, n = 4; <i>Idh3</i>g, <i>Mylk3 TSS</i> and <i>Snx19 Promoter</i> n = 4, others n = 5). <b>f</b>, RT-qPCR analysis of cardiac maturation markers in P0-1 CMs after transduction with <i>Idh3b</i> and <i>Idh3g</i> lentiviruses (n = 5, each). <i>36b4</i> served as reference gene. <b>g</b>, <b>h</b>, Immunofluorescence analysis of Ki67 in sarc-actinin<sup>+</sup> P0-1 CMs, 4 days after transduction of <i>Idh3b</i> (g) or <i>Idh3g</i> (h) lentiviruses with and without CPI treatment (n = 4, each). Scale bar: 50 μm. <b>i</b>, Immunofluorescence analysis of Ki67 in sarc-actinin<sup>+</sup> P0-1 CMs after 3-days-culture in the presence of DMSO, αKG, R2HG, and a combination of αKG and R2HG (n = 3, each). Quantification of Ki67<sup>+</sup>sarc-actinin<sup>+</sup> CMs is shown in the right panel. Scale bar: 50 μm. <b>j</b>, Immunofluorescence analysis of Ki67 in cTnT<sup>+</sup> P0-1 CMs after 3 days in presence of DMSO, αKG, CPI, and a combination of αKG and CPI. Scale bar: 50 μm. <b>k</b>, EdU incorporation in adult CM treated for 96 h with DMSO, αKG, and CPI (n = 3). Enlarged images are labelled by white frames. Quantification of EdU<sup>+</sup>sarc-actinin<sup>+</sup> CMs is shown in the right panel. Scale bar: 50 μm. <b>l</b>, RT-qPCR analysis of genes in P0-1 CMs after 4 days in presence of αKG and CPI (DMSO, n = 4; αKG, CPI, αKG + CPI, n = 3). <i>36b4</i> served as reference gene. Error bars represent mean ± s.e.m. N-numbers refer to independent experiments. Two-tailed, unpaired student t-tests for statistical analysis in a-b. One-way ANOVA with Tukey tests for correction of multiple comparisons in c-i, k-l. *<i>P</i> &lt; 0.05, **<i>P</i> &lt; 0.01, ***<i>P</i> &lt; 0.001, ****<i>P</i> &lt; 0.0001.</p><p>
                        <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#MOESM18">Source data</a>
                      </p></div></div><div data-test="supp-item" id="Fig15"><h3><a data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 10 effects of αkg on gene expre" href="https://riskmusings.substack.com/articles/s41586-023-06585-5/figures/15" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41586-023-06585-5/MediaObjects/41586_2023_6585_Fig15_ESM.jpg">Extended Data Fig. 10 Effects of αKG on gene expression, H3K4me3 deposition, and proliferation of CMs are mediated via KDM5.</a></h3><div data-component="thumbnail-container"><p><b>a</b>, Heat map of gene expression of <i>Kdm5</i> family members at different stages of heart development and after transaortic constriction (TAC) based on z-scores of DESeq-normalized counts (RNAseq data are from <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE79883">GSE79883</a>). <b>b</b>, Western blot analysis of myc-tagged KDM5B and H3K4me3 in neonatal CMs transduced with a <i>Kdm5b</i> lentivirus. Quantification of H3K4me3 levels is shown in the lower panel (Ctrl n = 4, <i>Kdm5b-myc</i> n = 5). H3 was used as loading control and pan-actin as sample processing control. <b>c</b>, Western blot analysis of H3K4me3 in P0-1 CMs transduced with a <i>Kdm5b</i> lentivirus with and without αKG treatment (n = 3, each). H3 was used as sample processing control. Quantification of H3K4me3 levels is shown in the right panel. <b>d</b>, Immunofluorescence analysis of Ki67 in sarc-actinin<sup>+</sup> neonatal CMs (P0-1), 4-days after lentiviral transduction of <i>Kdm5b</i> with and without αKG treatment. Scale bar: 50 μm. <b>e</b>, Western blot analysis of H3K4me3 in P0-1 CMs after 4 days in presence of <i>Kdm5b</i> siRNA, αKG, and a combination of αKG and <i>Kdm5b</i> siRNA (n = 4, each). H3 was used as loading control. Quantification of western blots is in the right panel. <b>f</b>, Immunofluorescence analysis of Ki67 in sarc-actinin<sup>+</sup> P0-1 CMs, 4 days after treatment with <i>Kdm5b</i> siRNA with and without αKG. Scale bar: 50 μm. <b>g</b>-<b>h</b>, RT-qPCR analysis of genes related to cardiac maturation (g) and selected house-keeping genes (h) in P0-1 CMs after treatment with <i>Kdm5b</i> siRNA with and without αKG (DMSO n = 6; αKG, <i>Kdm5b KD</i> and αKG+<i>Kdm5b KD</i>, n = 4). <i>36b4</i> expression served as reference gene. <b>i</b>, Proposed model: inactivation of <i>Cpt1b</i> in CMs increases αKG levels due to increased expression of IDH and reduced enzymatic activity of OGDH, which stimulates KDM5 activity, leading to reduction of H3K4me3 deposition in cardiac maturation genes, enabling proliferation of CMs. The mechanisms underlying the transport of αKG and citrate (dashed lines) into the nucleus are not fully understood. Error bar represents mean ± s.e.m. N-numbers refer to independent experiments. Two-tailed, unpaired student t-tests for statistical analysis in b. One-way ANOVA with Tukey tests for correction of multiple comparisons in c,e, g-h. *<i>P</i> &lt; 0.05, **<i>P</i> &lt; 0.01, ***<i>P</i> &lt; 0.001, ****<i>P</i> &lt; 0.0001. For gel source data in b,c,e, see <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#MOESM1">Supplementary Information</a>.</p><p>
                        <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://riskmusings.substack.com/articles/s41586-023-06585-5#MOESM19">Source data</a>
                      </p></div></div></div></div></div></section><section data-title="Supplementary information"><div id="Sec31-section"><h2 id="Sec31">Supplementary information</h2><div id="Sec31-content"><div data-test="supplementary-info"><div data-test="supp-item" id="MOESM1"><h3><a data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="supplementary information" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41586-023-06585-5/MediaObjects/41586_2023_6585_MOESM1_ESM.pdf" data-supp-info-image="">Supplementary Information</a></h3><p>Supplementary Methods, References, Tables 1 and 2 and Figs. 1–7.</p></div><div data-test="supp-item" id="MOESM4"><h3><a data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="supplementary video" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41586-023-06585-5/MediaObjects/41586_2023_6585_MOESM4_ESM.mp4" data-supp-info-image="">Supplementary Video</a></h3><p>Assessment of heart function in Ctrl and <i>Cpt1b</i><sup><i>iKO</i></sup> mice by cardiac MRI before and 14 and 28 days after I–R surgery. Representative videos of cardiac MRI measurements before and 14 days and 28 days after I–R surgery in Ctrl and <i>Cpt1b</i><sup><i>iKO</i></sup> mice. Inactivation of <i>Cpt1b</i> by injection of TAM was initiated 1 day after I–R surgery, resulting in significant reduction of CPT1B protein levels 11 days, but not 3 and 6 days, after I–R surgery.</p></div></div></div></div></section><section data-title="Source data"></section><section data-title="Rights and permissions"><div id="rightslink-section"><h2 id="rightslink">Rights and permissions</h2><div id="rightslink-content">
              <p><b>Open Access</b>  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>.</p>
            <p><a data-track="click" data-track-action="view rights and permissions" data-track-label="link" href="https://s100.copyright.com/AppDispatchServlet?title=Inhibition%20of%20fatty%20acid%20oxidation%20enables%20heart%20regeneration%20in%20adult%20mice&amp;author=Xiang%20Li%20et%20al&amp;contentID=10.1038%2Fs41586-023-06585-5&amp;copyright=The%20Author%28s%29&amp;publication=0028-0836&amp;publicationDate=2023-09-27&amp;publisherName=SpringerNature&amp;orderBeanReset=true&amp;oa=CC%20BY">Reprints and Permissions</a></p></div></div></section><section aria-labelledby="article-info" data-title="About this article"><div id="article-info-section"><h2 id="article-info">About this article</h2><div id="article-info-content"><div><p><a data-crossmark="10.1038/s41586-023-06585-5" target="_blank" rel="noopener" href="https://crossmark.crossref.org/dialog/?doi=10.1038/s41586-023-06585-5" data-track="click" data-track-action="Click Crossmark" data-track-label="link" data-test="crossmark"><img width="57" height="81" alt="Check for updates. Verify currency and authenticity via CrossMark" src="data:image/svg+xml;base64,<svg height="81" width="57" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="m17.35 35.45 21.3-14.2v-17.03h-21.3" fill="#989898"/><path d="m38.65 35.45-21.3-14.2v-17.03h21.3" fill="#747474"/><path d="m28 .5c-12.98 0-23.5 10.52-23.5 23.5s10.52 23.5 23.5 23.5 23.5-10.52 23.5-23.5c0-6.23-2.48-12.21-6.88-16.62-4.41-4.4-10.39-6.88-16.62-6.88zm0 41.25c-9.8 0-17.75-7.95-17.75-17.75s7.95-17.75 17.75-17.75 17.75 7.95 17.75 17.75c0 4.71-1.87 9.22-5.2 12.55s-7.84 5.2-12.55 5.2z" fill="#535353"/><path d="m41 36c-5.81 6.23-15.23 7.45-22.43 2.9-7.21-4.55-10.16-13.57-7.03-21.5l-4.92-3.11c-4.95 10.7-1.19 23.42 8.78 29.71 9.97 6.3 23.07 4.22 30.6-4.86z" fill="#9c9c9c"/><path d="m.2 58.45c0-.75.11-1.42.33-2.01s.52-1.09.91-1.5c.38-.41.83-.73 1.34-.94.51-.22 1.06-.32 1.65-.32.56 0 1.06.11 1.51.35.44.23.81.5 1.1.81l-.91 1.01c-.24-.24-.49-.42-.75-.56-.27-.13-.58-.2-.93-.2-.39 0-.73.08-1.05.23-.31.16-.58.37-.81.66-.23.28-.41.63-.53 1.04-.13.41-.19.88-.19 1.39 0 1.04.23 1.86.68 2.46.45.59 1.06.88 1.84.88.41 0 .77-.07 1.07-.23s.59-.39.85-.68l.91 1c-.38.43-.8.76-1.28.99-.47.22-1 .34-1.58.34-.59 0-1.13-.1-1.64-.31-.5-.2-.94-.51-1.31-.91-.38-.4-.67-.9-.88-1.48-.22-.59-.33-1.26-.33-2.02zm8.4-5.33h1.61v2.54l-.05 1.33c.29-.27.61-.51.96-.72s.76-.31 1.24-.31c.73 0 1.27.23 1.61.71.33.47.5 1.14.5 2.02v4.31h-1.61v-4.1c0-.57-.08-.97-.25-1.21-.17-.23-.45-.35-.83-.35-.3 0-.56.08-.79.22-.23.15-.49.36-.78.64v4.8h-1.61zm7.37 6.45c0-.56.09-1.06.26-1.51.18-.45.42-.83.71-1.14.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.36c.07.62.29 1.1.65 1.44.36.33.82.5 1.38.5.29 0 .57-.04.83-.13s.51-.21.76-.37l.55 1.01c-.33.21-.69.39-1.09.53-.41.14-.83.21-1.26.21-.48 0-.92-.08-1.34-.25-.41-.16-.76-.4-1.07-.7-.31-.31-.55-.69-.72-1.13-.18-.44-.26-.95-.26-1.52zm4.6-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.07.45-.31.29-.5.73-.58 1.3zm2.5.62c0-.57.09-1.08.28-1.53.18-.44.43-.82.75-1.13s.69-.54 1.1-.71c.42-.16.85-.24 1.31-.24.45 0 .84.08 1.17.23s.61.34.85.57l-.77 1.02c-.19-.16-.38-.28-.56-.37-.19-.09-.39-.14-.61-.14-.56 0-1.01.21-1.35.63-.35.41-.52.97-.52 1.67 0 .69.17 1.24.51 1.66.34.41.78.62 1.32.62.28 0 .54-.06.78-.17.24-.12.45-.26.64-.42l.67 1.03c-.33.29-.69.51-1.08.65-.39.15-.78.23-1.18.23-.46 0-.9-.08-1.31-.24-.4-.16-.75-.39-1.05-.7s-.53-.69-.7-1.13c-.17-.45-.25-.96-.25-1.53zm6.91-6.45h1.58v6.17h.05l2.54-3.16h1.77l-2.35 2.8 2.59 4.07h-1.75l-1.77-2.98-1.08 1.23v1.75h-1.58zm13.69 1.27c-.25-.11-.5-.17-.75-.17-.58 0-.87.39-.87 1.16v.75h1.34v1.27h-1.34v5.6h-1.61v-5.6h-.92v-1.2l.92-.07v-.72c0-.35.04-.68.13-.98.08-.31.21-.57.4-.79s.42-.39.71-.51c.28-.12.63-.18 1.04-.18.24 0 .48.02.69.07.22.05.41.1.57.17zm.48 5.18c0-.57.09-1.08.27-1.53.17-.44.41-.82.72-1.13.3-.31.65-.54 1.04-.71.39-.16.8-.24 1.23-.24s.84.08 1.24.24c.4.17.74.4 1.04.71s.54.69.72 1.13c.19.45.28.96.28 1.53s-.09 1.08-.28 1.53c-.18.44-.42.82-.72 1.13s-.64.54-1.04.7-.81.24-1.24.24-.84-.08-1.23-.24-.74-.39-1.04-.7c-.31-.31-.55-.69-.72-1.13-.18-.45-.27-.96-.27-1.53zm1.65 0c0 .69.14 1.24.43 1.66.28.41.68.62 1.18.62.51 0 .9-.21 1.19-.62.29-.42.44-.97.44-1.66 0-.7-.15-1.26-.44-1.67-.29-.42-.68-.63-1.19-.63-.5 0-.9.21-1.18.63-.29.41-.43.97-.43 1.67zm6.48-3.44h1.33l.12 1.21h.05c.24-.44.54-.79.88-1.02.35-.24.7-.36 1.07-.36.32 0 .59.05.78.14l-.28 1.4-.33-.09c-.11-.01-.23-.02-.38-.02-.27 0-.56.1-.86.31s-.55.58-.77 1.1v4.2h-1.61zm-47.87 15h1.61v4.1c0 .57.08.97.25 1.2.17.24.44.35.81.35.3 0 .57-.07.8-.22.22-.15.47-.39.73-.73v-4.7h1.61v6.87h-1.32l-.12-1.01h-.04c-.3.36-.63.64-.98.86-.35.21-.76.32-1.24.32-.73 0-1.27-.24-1.61-.71-.33-.47-.5-1.14-.5-2.02zm9.46 7.43v2.16h-1.61v-9.59h1.33l.12.72h.05c.29-.24.61-.45.97-.63.35-.17.72-.26 1.1-.26.43 0 .81.08 1.15.24.33.17.61.4.84.71.24.31.41.68.53 1.11.13.42.19.91.19 1.44 0 .59-.09 1.11-.25 1.57-.16.47-.38.85-.65 1.16-.27.32-.58.56-.94.73-.35.16-.72.25-1.1.25-.3 0-.6-.07-.9-.2s-.59-.31-.87-.56zm0-2.3c.26.22.5.37.73.45.24.09.46.13.66.13.46 0 .84-.2 1.15-.6.31-.39.46-.98.46-1.77 0-.69-.12-1.22-.35-1.61-.23-.38-.61-.57-1.13-.57-.49 0-.99.26-1.52.77zm5.87-1.69c0-.56.08-1.06.25-1.51.16-.45.37-.83.65-1.14.27-.3.58-.54.93-.71s.71-.25 1.08-.25c.39 0 .73.07 1 .2.27.14.54.32.81.55l-.06-1.1v-2.49h1.61v9.88h-1.33l-.11-.74h-.06c-.25.25-.54.46-.88.64-.33.18-.69.27-1.06.27-.87 0-1.56-.32-2.07-.95s-.76-1.51-.76-2.65zm1.67-.01c0 .74.13 1.31.4 1.7.26.38.65.58 1.15.58.51 0 .99-.26 1.44-.77v-3.21c-.24-.21-.48-.36-.7-.45-.23-.08-.46-.12-.7-.12-.45 0-.82.19-1.13.59-.31.39-.46.95-.46 1.68zm6.35 1.59c0-.73.32-1.3.97-1.71.64-.4 1.67-.68 3.08-.84 0-.17-.02-.34-.07-.51-.05-.16-.12-.3-.22-.43s-.22-.22-.38-.3c-.15-.06-.34-.1-.58-.1-.34 0-.68.07-1 .2s-.63.29-.93.47l-.59-1.08c.39-.24.81-.45 1.28-.63.47-.17.99-.26 1.54-.26.86 0 1.51.25 1.93.76s.63 1.25.63 2.21v4.07h-1.32l-.12-.76h-.05c-.3.27-.63.48-.98.66s-.73.27-1.14.27c-.61 0-1.1-.19-1.48-.56-.38-.36-.57-.85-.57-1.46zm1.57-.12c0 .3.09.53.27.67.19.14.42.21.71.21.28 0 .54-.07.77-.2s.48-.31.73-.56v-1.54c-.47.06-.86.13-1.18.23-.31.09-.57.19-.76.31s-.33.25-.41.4c-.09.15-.13.31-.13.48zm6.29-3.63h-.98v-1.2l1.06-.07.2-1.88h1.34v1.88h1.75v1.27h-1.75v3.28c0 .8.32 1.2.97 1.2.12 0 .24-.01.37-.04.12-.03.24-.07.34-.11l.28 1.19c-.19.06-.4.12-.64.17-.23.05-.49.08-.76.08-.4 0-.74-.06-1.02-.18-.27-.13-.49-.3-.67-.52-.17-.21-.3-.48-.37-.78-.08-.3-.12-.64-.12-1.01zm4.36 2.17c0-.56.09-1.06.27-1.51s.41-.83.71-1.14c.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.37c.08.62.29 1.1.65 1.44.36.33.82.5 1.38.5.3 0 .58-.04.84-.13.25-.09.51-.21.76-.37l.54 1.01c-.32.21-.69.39-1.09.53s-.82.21-1.26.21c-.47 0-.92-.08-1.33-.25-.41-.16-.77-.4-1.08-.7-.3-.31-.54-.69-.72-1.13-.17-.44-.26-.95-.26-1.52zm4.61-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.08.45-.31.29-.5.73-.57 1.3zm3.01 2.23c.31.24.61.43.92.57.3.13.63.2.98.2.38 0 .65-.08.83-.23s.27-.35.27-.6c0-.14-.05-.26-.13-.37-.08-.1-.2-.2-.34-.28-.14-.09-.29-.16-.47-.23l-.53-.22c-.23-.09-.46-.18-.69-.3-.23-.11-.44-.24-.62-.4s-.33-.35-.45-.55c-.12-.21-.18-.46-.18-.75 0-.61.23-1.1.68-1.49.44-.38 1.06-.57 1.83-.57.48 0 .91.08 1.29.25s.71.36.99.57l-.74.98c-.24-.17-.49-.32-.73-.42-.25-.11-.51-.16-.78-.16-.35 0-.6.07-.76.21-.17.15-.25.33-.25.54 0 .14.04.26.12.36s.18.18.31.26c.14.07.29.14.46.21l.54.19c.23.09.47.18.7.29s.44.24.64.4c.19.16.34.35.46.58.11.23.17.5.17.82 0 .3-.06.58-.17.83-.12.26-.29.48-.51.68-.23.19-.51.34-.84.45-.34.11-.72.17-1.15.17-.48 0-.95-.09-1.41-.27-.46-.19-.86-.41-1.2-.68z" fill="#535353"/></g></svg>"/></a></p><div><h3 id="citeas">Cite this article</h3><p>Li, X., Wu, F., Günther, S. <i>et al.</i> Inhibition of fatty acid oxidation enables heart regeneration in adult mice.
                    <i>Nature</i>  (2023). https://doi.org/10.1038/s41586-023-06585-5</p><p><a data-test="citation-link" data-track="click" data-track-action="download article citation" data-track-label="link" data-track-external="" rel="nofollow" href="https://citation-needed.springer.com/v2/references/10.1038/s41586-023-06585-5?format=refman&amp;flavour=citation">Download citation<svg width="16" height="16" focusable="false" role="img" aria-hidden="true"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-download"></use></svg></a></p><ul data-test="publication-history"><li><p>Received<span>: </span><span><time datetime="2021-08-05">05 August 2021</time></span></p></li><li><p>Accepted<span>: </span><span><time datetime="2023-08-30">30 August 2023</time></span></p></li><li><p>Published<span>: </span><span><time datetime="2023-09-27">27 September 2023</time></span></p></li><li><p><abbr title="Digital Object Identifier">DOI</abbr><span>: </span><span>https://doi.org/10.1038/s41586-023-06585-5</span></p></li></ul></div></div></div></div></section>
            </div></div>
  </body>
</html>
