<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <title>James Routley | Feed</title>
    <link
      rel="stylesheet"
      type="text/css"
      href="../styles.css"
      media="screen"
    />
  </head>
  <body>
    <a href="/index.html">Back</a>
    <a href="https://www.nature.com/articles/s41588-022-01148-2">Original</a>
    <h1>Integrating variants in 42k autism cases identifies mutations in new genes</h1>
    
    <div id="readability-page-1" class="page"><div>
            <section aria-labelledby="Abs1" data-title="Abstract" lang="en"><div id="Abs1-section"><h2 id="Abs1">Abstract</h2><p>To capture the full spectrum of genetic risk for autism, we performed a two-stage analysis of rare de novo and inherited coding variants in 42,607 autism cases, including 35,130 new cases recruited online by SPARK. We identified 60 genes with exome-wide significance (<i>P</i> &lt; 2.5 × 10<sup>−6</sup>), including five new risk genes (<i>NAV3</i>, <i>ITSN1</i>, <i>MARK2</i>, <i>SCAF1</i> and <i>HNRNPUL2</i>). The association of <i>NAV3</i> with autism risk is primarily driven by rare inherited loss-of-function (LoF) variants, with an estimated relative risk of 4, consistent with moderate effect. Autistic individuals with LoF variants in the four moderate-risk genes (<i>NAV3</i>, <i>ITSN1</i>, <i>SCAF1</i> and <i>HNRNPUL2</i>; <i>n</i> = 95) have less cognitive impairment than 129 autistic individuals with LoF variants in highly penetrant genes (<i>CHD8, SCN2A, ADNP, FOXP1</i> and <i>SHANK3</i>) (59% vs 88%, <i>P</i> = 1.9 × 10<sup>−6</sup>). Power calculations suggest that much larger numbers of autism cases are needed to identify additional moderate-risk genes.</p></div></section>
            

                
            
                <section data-title="Main"><div id="Sec1-section"><h2 id="Sec1">Main</h2><div id="Sec1-content"><p>Many previous genetic studies in autism spectrum disorder (ASD), a neurodevelopmental condition characterized by social communication difficulties and repetitive behaviors<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 1" title="Lord, C. et al. Autism spectrum disorder. Nat. Rev. Dis. Primers 6, 5 (2020)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR1" id="ref-link-section-d6323361e6270">1</a></sup>, focused on de novo variants (DNVs) identified from parent–offspring trios<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Iossifov, I. et al. The contribution of de novo coding mutations to autism spectrum disorder. Nature 515, 216–221 (2014)." href="#ref-CR2" id="ref-link-section-d6323361e6274">2</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="De Rubeis, S. et al. Synaptic, transcriptional and chromatin genes disrupted in autism. Nature 515, 209–215 (2014)." href="#ref-CR3" id="ref-link-section-d6323361e6274_1">3</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="O’Roak, B. J. et al. Recurrent de novo mutations implicate novel genes underlying simplex autism risk. Nat. Commun. 5, 5595 (2014)." href="#ref-CR4" id="ref-link-section-d6323361e6274_2">4</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Yuen, R. K. C. et al. Whole-genome sequencing of quartet families with autism spectrum disorder. Nat. Med. 21, 185–191 (2015)." href="#ref-CR5" id="ref-link-section-d6323361e6274_3">5</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Yuen, R. K. C. et al. Whole genome sequencing resource identifies 18 new candidate genes for autism spectrum disorder. Nat. Neurosci. 20, 602–611 (2017)." href="#ref-CR6" id="ref-link-section-d6323361e6274_4">6</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Feliciano, P. et al. Exome sequencing of 457 autism families recruited online provides evidence for autism risk genes. npj Genomic Med 4, 19 (2019)." href="#ref-CR7" id="ref-link-section-d6323361e6274_5">7</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 8" title="Satterstrom, F. K. et al. Large-scale exome sequencing study implicates both developmental and functional changes in the neurobiology of autism. Cell 180, 568–584.e23 (2020)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR8" id="ref-link-section-d6323361e6277">8</a></sup>. Over 100 high-confidence ASD genes enriched with likely deleterious DNVs have been identified<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 8" title="Satterstrom, F. K. et al. Large-scale exome sequencing study implicates both developmental and functional changes in the neurobiology of autism. Cell 180, 568–584.e23 (2020)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR8" id="ref-link-section-d6323361e6281">8</a></sup>, most of which are also enriched for DNVs in other neurodevelopmental disorders (NDDs)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="The Deciphering Developmental Disorders Study. Large-scale discovery of novel genetic causes of developmental disorders. Nature 519, 223–228 (2015)." href="#ref-CR9" id="ref-link-section-d6323361e6285">9</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="The Deciphering Developmental Disorders Study. Prevalence and architecture of de novo mutations in developmental disorders. Nature 542, 433–438 (2017)." href="#ref-CR10" id="ref-link-section-d6323361e6285_1">10</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 11" title="Kaplanis, J. et al. Evidence for 28 genetic disorders discovered by combining healthcare and research data. Nature 586, 757–762 (2020)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR11" id="ref-link-section-d6323361e6288">11</a></sup>. Statistical modeling suggests that there are ~1,000 genes with DNVs in ASD<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 12" title="He, X. et al. Integrated model of de novo and inherited genetic variants yields greater power to identify risk genes. PLoS Genet. 9, e1003671 (2013)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR12" id="ref-link-section-d6323361e6292">12</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 13" title="Nguyen, H. T. et al. Integrated Bayesian analysis of rare exonic variants to identify risk genes for schizophrenia and neurodevelopmental disorders. Genome Med. 9, 114 (2017)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR13" id="ref-link-section-d6323361e6295">13</a></sup>. However, despite the large effect size of individual pathogenic DNVs, all DNVs combined explain only ~2% of variance in liability for ASD<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 8" title="Satterstrom, F. K. et al. Large-scale exome sequencing study implicates both developmental and functional changes in the neurobiology of autism. Cell 180, 568–584.e23 (2020)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR8" id="ref-link-section-d6323361e6300">8</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 14" title="Gaugler, T. et al. Most genetic risk for autism resides with common variation. Nat. Genet. 46, 881–885 (2014)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR14" id="ref-link-section-d6323361e6303">14</a></sup>. ASD is highly heritable<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Gaugler, T. et al. Most genetic risk for autism resides with common variation. Nat. Genet. 46, 881–885 (2014)." href="#ref-CR14" id="ref-link-section-d6323361e6307">14</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Sandin, S. et al. The familial risk of autism. JAMA 311, 1770–1777 (2014)." href="#ref-CR15" id="ref-link-section-d6323361e6307_1">15</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 16" title="Sandin, S. et al. The heritability of autism spectrum disorder. JAMA 318, 1182–1184 (2017)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR16" id="ref-link-section-d6323361e6310">16</a></sup>, and previous studies estimated that common variants explain up to half of the heritability<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 14" title="Gaugler, T. et al. Most genetic risk for autism resides with common variation. Nat. Genet. 46, 881–885 (2014)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR14" id="ref-link-section-d6323361e6314">14</a></sup>, although only five genome-wide significant loci have been identified<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 17" title="Grove, J. et al. Identification of common genetic risk variants for autism spectrum disorder. Nat. Genet. 51, 431–444 (2019)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR17" id="ref-link-section-d6323361e6318">17</a></sup>. Rare LoF variants in genes intolerant of variation<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 9" title="The Deciphering Developmental Disorders Study. Large-scale discovery of novel genetic causes of developmental disorders. Nature 519, 223–228 (2015)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR9" id="ref-link-section-d6323361e6322">9</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 18" title="Lek, M. et al. Analysis of protein-coding genetic variation in 60,706 humans. Nature 536, 285–291 (2016)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR18" id="ref-link-section-d6323361e6325">18</a></sup> are overtransmitted to probands compared with siblings without ASD<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Feliciano, P. et al. Exome sequencing of 457 autism families recruited online provides evidence for autism risk genes. npj Genomic Med 4, 19 (2019)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR7" id="ref-link-section-d6323361e6329">7</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 8" title="Satterstrom, F. K. et al. Large-scale exome sequencing study implicates both developmental and functional changes in the neurobiology of autism. Cell 180, 568–584.e23 (2020)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR8" id="ref-link-section-d6323361e6332">8</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Krumm, N. et al. Excess of rare, inherited truncating mutations in autism. Nat. Genet. 47, 582–588 (2015)." href="#ref-CR19" id="ref-link-section-d6323361e6335">19</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Kosmicki, J. A. et al. Refining the role of de novo protein-truncating variants in neurodevelopmental disorders by using population reference samples. Nat. Genet. 49, 504–510 (2017)." href="#ref-CR20" id="ref-link-section-d6323361e6335_1">20</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Ruzzo, E. K. et al. Inherited and de novo genetic risk for autism impacts shared networks. Cell 178, 850–866.e26 (2019)." href="#ref-CR21" id="ref-link-section-d6323361e6335_2">21</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 22" title="Wilfert, A. B. et al. Recent ultra-rare inherited variants implicate new autism candidate risk genes. Nat. Genet. 53, 1125–1134 (2021)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR22" id="ref-link-section-d6323361e6338">22</a></sup>. However, identification of the individual risk genes enriched by such inherited variants has remained elusive. We have established the largest ASD cohort, Simons Foundation Powering Autism Research for Knowledge (SPARK)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 23" title="The SPARK Consortium. SPARK: a US cohort of 50,000 families to accelerate autism research. Neuron 97, 488–493 (2018)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR23" id="ref-link-section-d6323361e6343">23</a></sup>, which currently includes over 100,000 people with ASD, to advance research on the genetic, behavioral and clinical features associated with ASD.</p><p>Rare LoF variants are enriched in developmental disorders including ASD<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 22" title="Wilfert, A. B. et al. Recent ultra-rare inherited variants implicate new autism candidate risk genes. Nat. Genet. 53, 1125–1134 (2021)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR22" id="ref-link-section-d6323361e6350">22</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 24" title="MacArthur, D. G. &amp; Tyler-Smith, C. Loss-of-function variants in the genomes of healthy humans. Hum. Mol. Genet. 19, R125–R130 (2010)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR24" id="ref-link-section-d6323361e6353">24</a></sup>, but may also result from sequencing and annotation artifacts<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 25" title="MacArthur, D. G. et al. A systematic survey of loss-of-function variants in human protein-coding genes. Science 335, 823–828 (2012)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR25" id="ref-link-section-d6323361e6357">25</a></sup> and present technical challenges in large sequencing studies. Methods to distinguish between high-confidence and low-confidence LoF variants<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 18" title="Lek, M. et al. Analysis of protein-coding genetic variation in 60,706 humans. Nature 536, 285–291 (2016)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR18" id="ref-link-section-d6323361e6361">18</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 26" title="Karczewski, K. J. et al. The mutational constraint spectrum quantified from variation in 141,456 humans. Nature 581, 434–443 (2020)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR26" id="ref-link-section-d6323361e6364">26</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 27" title="Cummings, B. B. et al. Transcript expression-aware annotation improves rare variant interpretation. Nature 581, 452–458 (2020)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR27" id="ref-link-section-d6323361e6367">27</a></sup> have been used to quantify gene-level LoF intolerance<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 18" title="Lek, M. et al. Analysis of protein-coding genetic variation in 60,706 humans. Nature 536, 285–291 (2016)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR18" id="ref-link-section-d6323361e6371">18</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 26" title="Karczewski, K. J. et al. The mutational constraint spectrum quantified from variation in 141,456 humans. Nature 581, 434–443 (2020)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR26" id="ref-link-section-d6323361e6374">26</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 28" title="Cassa, C. A. et al. Estimating the selective effects of heterozygous protein-truncating variants from human exome data. Nat. Genet. 49, 806–810 (2017)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR28" id="ref-link-section-d6323361e6377">28</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 29" title="Fuller, Z. L., Berg, J. J., Mostafavi, H., Sella, G. &amp; Przeworski, M. Measuring intolerance to mutation in human genetics. Nat. Genet. 51, 772–776 (2019)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR29" id="ref-link-section-d6323361e6380">29</a></sup> and to refine the role of LoF DNVs in NDDs<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 20" title="Kosmicki, J. A. et al. Refining the role of de novo protein-truncating variants in neurodevelopmental disorders by using population reference samples. Nat. Genet. 49, 504–510 (2017)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR20" id="ref-link-section-d6323361e6384">20</a></sup>.</p><p>Here, we present an integrated analysis of de novo and inherited coding variants in over 42,607 ASD cases, including cases from previously published ASD cohorts and 35,130 new cases from SPARK. In our two-stage design, we first characterized the contribution of DNVs and rare inherited LoF variants to ASD risk. Results from the first stage informed the second stage meta-analysis of 404 genes. By combining evidence from DNVs, transmission disequilibrium tests (TDTs) and case-control comparisons, we identified 60 ASD risk genes with exome-wide significance, including five new genes not previously implicated in NDDs. Finally, we estimated the effect sizes of known and newly identified genes and conducted power calculations to inform the design of future studies.</p></div></div></section><section data-title="Results"><div id="Sec2-section"><h2 id="Sec2">Results</h2><div id="Sec2-content"><h3 id="Sec3">Overview of data and workflow</h3><p>We aggregated exome or whole genome sequencing (WGS) data of 35,130 new cases from SPARK and 7,665 cases from published ASD studies (ASC<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 3" title="De Rubeis, S. et al. Synaptic, transcriptional and chromatin genes disrupted in autism. Nature 515, 209–215 (2014)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR3" id="ref-link-section-d6323361e6403">3</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 8" title="Satterstrom, F. K. et al. Large-scale exome sequencing study implicates both developmental and functional changes in the neurobiology of autism. Cell 180, 568–584.e23 (2020)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR8" id="ref-link-section-d6323361e6406">8</a></sup>, MSSNG<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 6" title="Yuen, R. K. C. et al. Whole genome sequencing resource identifies 18 new candidate genes for autism spectrum disorder. Nat. Neurosci. 20, 602–611 (2017)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR6" id="ref-link-section-d6323361e6410">6</a></sup> and SSC<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 2" title="Iossifov, I. et al. The contribution of de novo coding mutations to autism spectrum disorder. Nature 515, 216–221 (2014)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR2" id="ref-link-section-d6323361e6414">2</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 30" title="An, J.-Y. et al. Genome-wide de novo risk score implicates promoter variation in autism spectrum disorder. Science 362, aat6576 (2018)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR30" id="ref-link-section-d6323361e6417">30</a></sup>) (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41588-022-01148-2#MOESM4">1</a>) and performed a two-stage analysis (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41588-022-01148-2#Fig1">1</a>). In stage 1, we analyzed DNVs in 16,877 ASD trios and assessed transmission of rare LoF variants from 20,491 parents without ASD diagnoses or intellectual disability to offspring with ASD (including 9,504 trios and 2,966 single-parent-proband duos). For DNVs, we characterized the enrichment pattern in known and candidate risk genes, as well as mutation intolerance (probability of being LoF intolerant as defined by the Exome Aggregation Consortium (ExAC pLI)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 18" title="Lek, M. et al. Analysis of protein-coding genetic variation in 60,706 humans. Nature 536, 285–291 (2016)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR18" id="ref-link-section-d6323361e6428">18</a></sup>, and Genome Aggregation Database (gnomAD) metrics<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 26" title="Karczewski, K. J. et al. The mutational constraint spectrum quantified from variation in 141,456 humans. Nature 581, 434–443 (2020)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR26" id="ref-link-section-d6323361e6432">26</a></sup>), and performed gene-based burden tests of LoF and missense DNVs by DeNovoWEST<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 11" title="Kaplanis, J. et al. Evidence for 28 genetic disorders discovered by combining healthcare and research data. Nature 586, 757–762 (2020)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR11" id="ref-link-section-d6323361e6436">11</a></sup>. For rare inherited LoFs, we estimated the overtransmission from unaffected parents to ASD offspring in all genes and gene sets predefined by functional genomic data or results from DNV analysis. Based on DNV enrichment and overtransmission patterns in gene sets, we selected 404 genes for meta-analysis in stage 2 using 22,764 new cases with exome or WGS data. In stage 2, we applied DeNovoWEST on DNVs, conducted TDTs on inherited LoFs in trios or duos, performed burden tests on rare LoFs in unrelated cases compared with population controls (104,068 subjects from non-neuro gnomAD exomes and 132,345 TOPMed subjects) and combined the <i>P</i> values to estimate a final <i>P</i> value for each of the 404 genes. Finally, we performed a mega-analysis of rare LoFs in all cases and controls to estimate the effect sizes of known or new candidate ASD genes to inform future studies.</p><div data-test="figure" data-container-section="figure" id="figure-1" data-title="Analysis workflow."><figure><figcaption><b id="Fig1" data-test="figure-caption-text">Fig. 1: Analysis workflow.</b></figcaption><div><div><a data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="https://www.nature.com/articles/s41588-022-01148-2/figures/1" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41588-022-01148-2/MediaObjects/41588_2022_1148_Fig1_HTML.png?as=webp"/><img aria-describedby="Fig1" src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41588-022-01148-2/MediaObjects/41588_2022_1148_Fig1_HTML.png" alt="figure 1" loading="lazy" width="685" height="822"/></picture></a></div><p>In the discovery stage, we identified DNVs in 16,877 ASD trios and rare LoF variants in 20,491 parents without ASD diagnoses and intellectual disability. We compared properties of de novo and rare variants to identify rare LoFs that contribute to genetic risk in individuals with ASD. We also evaluated their associations with cognitive impairment and enriched gene sets. We performed an initial exome-wide scan of genes enriched by DNVs or showing transmission disequilibrium of rare LoFs to affected offspring and selected a total of 404 genes for further replication, including 159 de novo enriched genes and 260 prioritized transmission disequilibrium genes from enriched gene sets (15 genes were in both). In the meta-analysis stage, we first evaluated evidence from de novo enrichment and transmission disequilibrium of rare inherited LoFs in an expanded set of family-based samples including over 6,000 additional ASD trios and around 2,000 additional duos. The DNVs in ASD were combined with those from an additional 31,565 NDD trios to refine the filters of high-confidence LoF variants in de novo LoF enriched genes. We also constructed an independent dataset of LoF variants of unknown inheritance from 15,780 cases that were not used in de novo or transmission analysis. We compared LoF rates in cases with two population-based sets of controls (<i>n</i> = ~104,000 and ~132,000, respectively). For 367 LoF-intolerant genes on autosomes, the final gene-level evidence was obtained by meta-analyzing <i>P</i> values of de novo enrichment, transmission disequilibrium of high-confidence rare inherited LoFs, and comparison of high-confidence LoFs from cases and controls not used in the de novo or transmission analysis. We also performed a mega-analysis that analyzed high-confidence LoFs identified in all 31,976 unrelated ASD cases and compared their rates with population-based controls. HC, high-confidence.</p></div></figure></div><h3 id="Sec4">Known ASD or NDD risk genes explain most de novo burden</h3><p>In the first stage, we combined data from four large-scale ASD cohorts, including 16,877 unique ASD trios and 5,764 unaffected trios (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41588-022-01148-2#MOESM4">1</a>). The cohorts show similar exome-wide burden of DNVs in simplex families. The burden of LoF DNVs in cases with an ASD family history is significantly lower than those without (<i>P</i> = 1.1 × 10<sup>−4</sup> by Poisson test), whereas the burden of predicted de novo damaging missense (D-mis, defined by rare exome variant ensemble learner (REVEL) score<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 31" title="Ioannidis, N. M. et al. REVEL: an ensemble method for predicting the pathogenicity of rare missense variants. Am. J. Hum. Genet. 99, 877–885 (2016)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR31" id="ref-link-section-d6323361e6488">31</a></sup> ≥ 0.5) and synonymous variants are similar (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41588-022-01148-2#Fig8">1</a>). Compared with unaffected offspring, the excess of damaging DNVs (de novo LoF and D-mis variants) in individuals with ASD is concentrated in LoF-intolerant genes, defined as genes with an ExAC pLI ≥ 0.5 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 18" title="Lek, M. et al. Analysis of protein-coding genetic variation in 60,706 humans. Nature 536, 285–291 (2016)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR18" id="ref-link-section-d6323361e6496">18</a></sup>). Using LoF observed/expected upper-bound fraction (LOEUF), a recently developed gene constraint metric<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 26" title="Karczewski, K. J. et al. The mutational constraint spectrum quantified from variation in 141,456 humans. Nature 581, 434–443 (2020)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR26" id="ref-link-section-d6323361e6500">26</a></sup>, the burden of damaging DNVs is highest among genes ranked in the top 20% of LOEUF scores (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41588-022-01148-2#Fig2">2a</a>). Overall, the population attributable risk (PAR) from damaging DNVs is about 10%. We assembled 618 previously established dominant (‘known’) ASD or NDD risk genes (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41588-022-01148-2#MOESM4">2</a>). These genes explained about two-thirds of the PAR from damaging DNVs. Excluding these genes, the fold enrichment of damaging DNVs was greatly attenuated (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41588-022-01148-2#Fig2">2a</a>).</p><div data-test="figure" data-container-section="figure" id="figure-2" data-title="Comparison of burden between de novo damaging variants and rare inherited LoFs in ASD."><figure><figcaption><b id="Fig2" data-test="figure-caption-text">Fig. 2: Comparison of burden between de novo damaging variants and rare inherited LoFs in ASD.</b></figcaption><div><div><a data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="https://www.nature.com/articles/s41588-022-01148-2/figures/2" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41588-022-01148-2/MediaObjects/41588_2022_1148_Fig2_HTML.png?as=webp"/><img aria-describedby="Fig2" src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41588-022-01148-2/MediaObjects/41588_2022_1148_Fig2_HTML.png" alt="figure 2" loading="lazy" width="685" height="628"/></picture></a></div><p><b>a</b>, The burden of DNVs was evaluated by the rate ratio and rate difference between 16,877 ASD and 5,764 unaffected trios. The exome-wide burden of de novo LoF and D-mis (REVEL ≥ 0.5) variants are concentrated in constrained genes (ExAC pLI ≥ 0.5) and in genes with the highest levels of LoF intolerance in the population (defined by the top two deciles of gnomAD LOEUF scores). Burden analysis was repeated after removing known ASD or NDD genes. The number of genes before and after removing known genes in each constraint bin is shown below the axis label. Data are presented as mean values and 95% confidence intervals. Among constrained genes (ExAC pLI ≥ 0.5 or the top 20% of gnomAD LOEUF scores), close to two-thirds of case-control rate differences of de novo LoF and D-mis variants can be explained by known genes. Exact <i>P</i> values by Poisson test are listed in Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41588-022-01148-2#MOESM4">19</a>. <b>b</b>, The burden of inherited LoFs was evaluated by looking at the proportion of rare LoFs in 20,491 parents without ASD diagnoses or intellectual disability that are transmitted to affected offspring in 9,504 trios and 2,966 duos and show evidence of overtransmission of LoFs per ASD trio. As a comparison, we also show the transmission disequilibrium pattern to unaffected offspring in 5,110 trios and 129 duos. Data are presented as mean values ± standard errors as error bars. Two-sided binomial test was used to compute the <i>P</i> values for overtransmission or undertransmission. Using ultra-rare LoFs with pExt ≥ 0.1, exome-wide signals of transmission disequilibrium of rare inherited LoF variants also concentrate in constrained genes (ExAC pLI ≥ 0.5) and in genes within the top two deciles of gnomAD LOEUF scores. Analysis was restricted to autosomal genes and repeated after removing known ASD or NDD genes (number of genes in each constrained bin before and after removing known genes is shown below the axis label). Among all constrained genes, only one-fifth of overtransmission of LoFs to ASD trios can be explained by known ASD or NDD genes. Exact <i>P</i> values by binominal test are listed in Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41588-022-01148-2#MOESM4">19</a>.</p></div></figure></div><p>To assess the evidence of DNVs in individual genes, we applied DeNovoWEST<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 11" title="Kaplanis, J. et al. Evidence for 28 genetic disorders discovered by combining healthcare and research data. Nature 586, 757–762 (2020)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR11" id="ref-link-section-d6323361e6557">11</a></sup>, which integrates DNV enrichment with clustering of missense variants in each gene. The initial DeNovoWEST scan of DNVs in 16,877 ASD trios identified 159 genes with <i>P</i> &lt; 0.001 (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41588-022-01148-2#MOESM4">3</a>).</p><h3 id="Sec5">Rare inherited LoFs are mostly in unknown ASD risk genes</h3><p>To analyze the contribution of rare inherited LoF variants to ASD risk, we evaluated transmission disequilibrium in ultra-rare (allele frequency &lt; 1 × 10<sup>−5</sup>) high-confidence (by the loss-of-function transcript effect estimator (LOFTEE)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 26" title="Karczewski, K. J. et al. The mutational constraint spectrum quantified from variation in 141,456 humans. Nature 581, 434–443 (2020)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR26" id="ref-link-section-d6323361e6577">26</a></sup> package and proportion expression across transcripts (pExt)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 27" title="Cummings, B. B. et al. Transcript expression-aware annotation improves rare variant interpretation. Nature 581, 452–458 (2020)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR27" id="ref-link-section-d6323361e6581">27</a></sup>; see <a data-track="click" data-track-label="link" data-track-action="section anchor" href="https://www.nature.com/articles/s41588-022-01148-2#Sec11">Methods</a> and Supplementary Note) LoF variants from parents without ASD diagnoses or intellectual disability to affected offspring with ASD in 9,504 trios and 2,966 duos from the first stage (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41588-022-01148-2#MOESM4">4</a>). For a given set of genes, we quantified transmission disequilibrium using the number of overtransmitted (excess in transmission over nontransmission) LoF variants per trio; parent–offspring duos were considered half-trios. Among autosomal genes, the overall transmission disequilibrium signal of ultra-rare LoF variants is enriched in LoF-intolerant genes (ExAC pLI ≥ 0.5) and in genes within the top 20% of LOEUF scores (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41588-022-01148-2#Fig2">2b</a>), similar to the burden of damaging DNVs. We observed both overtransmission to affected and undertransmission to unaffected offspring, especially in genes within the top 10% of LOEUF scores. However, known ASD or NDD genes explain only ~20% of overtransmission of LoF variants to affected offspring (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41588-022-01148-2#Fig2">2b</a>). On the X chromosome, we only considered transmission from mothers without ASD to 9,883 affected sons and 2,571 affected daughters (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41588-022-01148-2#MOESM4">4</a>). Rare LoF variants in mothers without ASD show significant overtransmission to affected sons but not affected daughters and remain significant after removing known ASD or NDD genes (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41588-022-01148-2#MOESM1">1</a>). Together, these results suggest that most genes conferring inherited ASD risk are yet to be identified. Autosomal rare D-mis variants also show evidence of transmission disequilibrium to affected offspring, although the signal is much weaker (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41588-022-01148-2#MOESM1">2</a>).</p><p>To characterize the properties of genes contributing to ASD risk through rare inherited variants, we defined 25 gene sets from five categories representing both functional and genetic evidence relevant to ASD (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41588-022-01148-2#MOESM4">5</a> and Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41588-022-01148-2#MOESM1">3</a>). We limited the genes to 5,754 autosomal constrained genes (ExAC pLI ≥ 0.5 or top 20% of LOEUF scores) and performed TDT (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41588-022-01148-2#MOESM4">6</a>). For each gene set, we tested if ultra-rare high-confidence rare LoF variants show a higher transmission to ASD offspring than the remaining genes in the overall constrained gene set. As a comparison with DNVs, we also tested if the same set of genes are more frequently disrupted by damaging DNVs than the rest of the genes in ASD trios using DNENRICH<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 32" title="Fromer, M. et al. De novo mutations in schizophrenia implicate synaptic networks. Nature 506, 179–184 (2014)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR32" id="ref-link-section-d6323361e6619">32</a></sup>.</p><p>Using functional gene sets derived from the neuronal transcriptome, proteome or regulome, we confirmed significant enrichment in damaging DNVs (<i>P</i> &lt; 0.005 by simulation) in the gene sets that were previously suggested to be enriched for ASD risk genes including expression module M2/3<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 33" title="Parikshak, N. N. et al. Integrative functional genomic analyses implicate specific molecular pathways and circuits in autism. Cell 155, 1008–1021 (2013)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR33" id="ref-link-section-d6323361e6629">33</a></sup>, RBFOX1/3 targets<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 34" title="Weyn-Vanhentenryck, S. M. et al. HITS-CLIP and integrative modeling define the Rbfox splicing-regulatory network linked to brain development and autism. Cell Rep. 6, 1139–1152 (2014)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR34" id="ref-link-section-d6323361e6633">34</a></sup>, FMRP targets<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 35" title="Darnell, J. C. et al. FMRP stalls ribosomal translocation on mRNAs linked to synaptic function and autism. Cell 146, 247–261 (2011)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR35" id="ref-link-section-d6323361e6637">35</a></sup> and CHD8 targets<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 36" title="Cotney, J. et al. The autism-associated chromatin modifier CHD8 regulates other autism risk genes during human neurodevelopment. Nat. Commun. 6, 6404 (2015)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR36" id="ref-link-section-d6323361e6641">36</a></sup>. However, this enrichment can be largely explained by known ASD or NDD genes (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41588-022-01148-2#Fig9">2</a>). For ultra-rare inherited LoF variants, we found that the proportion of transmission to ASD individuals in most functional gene sets is close to all other genes; only RBFOX targets show a weak enrichment but can be largely explained by known genes (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41588-022-01148-2#Fig3">3</a>). We also applied two machine learning methods to prioritize ASD risk genes: forecASD<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 37" title="Brueggeman, L., Koomar, T. &amp; Michaelson, J. J. Forecasting risk gene discovery in autism with machine learning and genome-scale data. Sci. Rep. 10, 4569 (2020)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR37" id="ref-link-section-d6323361e6652">37</a></sup> and A-risk<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 38" title="Chen, S. et al. Dissecting autism genetic risk using single-cell RNA-seq data. Preprint at 
                https://www.bioRxiv.org/content/10.1101/2020.06.15.153031v1
                
               (2020)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR38" id="ref-link-section-d6323361e6656">38</a></sup>. Although enrichment of DNVs in predicted genes is mainly explained by known genes, genes prioritized by A-risk are significantly enriched with inherited LoFs that are not explained by known genes. Using A-risk ≥ 0.4, 30% of constrained genes (<i>n</i> = 1,464) were prioritized and explain 64% of the overtransmission of LoF variants to ASD offspring (<i>P</i> = 2.6 × 10<sup>−5</sup> by χ<sup>2</sup> test). This enrichment is higher than genes prioritized by the LOEUF score; 33% of genes (<i>n</i> = 1,777) in the top decile of LOEUF account for 55% of the overtransmission (<i>P</i> = 3.5 × 10<sup>−4</sup> by χ<sup>2</sup> test) (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41588-022-01148-2#Fig3">3</a>).</p><div data-test="figure" data-container-section="figure" id="figure-3" data-title="Enrichment of rare LoF variants in ASD cases across gene sets."><figure><figcaption><b id="Fig3" data-test="figure-caption-text">Fig. 3: Enrichment of rare LoF variants in ASD cases across gene sets.</b></figcaption><div><div><a data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="https://www.nature.com/articles/s41588-022-01148-2/figures/3" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41588-022-01148-2/MediaObjects/41588_2022_1148_Fig3_HTML.png?as=webp"/><img aria-describedby="Fig3" src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41588-022-01148-2/MediaObjects/41588_2022_1148_Fig3_HTML.png" alt="figure 3" loading="lazy" width="685" height="1344"/></picture></a></div><p>Gene sets were defined and grouped by transcriptome proteome, neuronal regulome, ASD gene prediction scores, genetic evidence from neuropsychiatric diseases, and gene-level constraint. Analyses were repeated after removing known ASD or NDD genes. (Number of genes in each set before and after removing known genes are shown in parentheses below gene set.) Dots represent fold enrichment of DNVs or odds ratios for overtransmission of LoF variants in each set. Horizontal bars are presented as mean values with 95% confidence interval as error bars. For each gene set, we show the percentage of overtransmission of rare LoFs to cases. Enrichment of rare inherited LoFs was evaluated by the share of overtransmission events (the transmission and nontransmission of ultra-rare LoFs with pExt ≥ 0.1) in the selected gene set vs those in all other constrained genes using a two-by-two table. <i>P</i> values were determined using the χ<sup>2</sup> test. Exact <i>P</i> values are listed in Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41588-022-01148-2#MOESM4">19</a>.</p></div></figure></div><p>We also considered gene sets that have evidence of genetic association with DNVs. Genes nominally enriched by DNVs (<i>P</i> &lt; 0.01 by DeNovoWEST; <i>n</i> = 300) in ASD from the current study have a significantly higher overtransmission rate than other constrained genes (odds ratio = 1.39, <i>P</i> = 3.0 × 10<sup>−5</sup> by χ<sup>2</sup> test) (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41588-022-01148-2#Fig3">3</a>), although these genes account for only 21% of the overtransmission. Genes nominally enriched by DNVs in other NDDs<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 11" title="Kaplanis, J. et al. Evidence for 28 genetic disorders discovered by combining healthcare and research data. Nature 586, 757–762 (2020)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR11" id="ref-link-section-d6323361e6736">11</a></sup> are also significantly enriched by DNVs in ASD and weakly enriched by inherited LoFs in ASD; however, both can be largely explained by known genes (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41588-022-01148-2#Fig3">3</a>). This suggests that a subset of ASD genes increase risk by both DNVs and inherited variants, and new genes can be identified by integrating evidence from DNV enrichment and TDT.</p><h3 id="Sec6">DNVs and some rare inherited LoFs are associated with intellectual disability</h3><p>To evaluate the association of genotypes with phenotype in ASD, we used self-reported cognitive impairment in SPARK, a Vineland score of &lt;70 in the SSC or the presence of intellectual disability in ASC. Damaging DNVs in genes ranked within the top 10% of LOEUF scores show a higher burden (<i>P</i> = 1.1 × 10<sup>−24</sup> by χ<sup>2</sup> test) in ASD cases with evidence of cognitive impairment than in other cases, consistent with previous results<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 2" title="Iossifov, I. et al. The contribution of de novo coding mutations to autism spectrum disorder. Nature 515, 216–221 (2014)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR2" id="ref-link-section-d6323361e6758">2</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 8" title="Satterstrom, F. K. et al. Large-scale exome sequencing study implicates both developmental and functional changes in the neurobiology of autism. Cell 180, 568–584.e23 (2020)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR8" id="ref-link-section-d6323361e6761">8</a></sup> (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41588-022-01148-2#Fig4">4a</a>). Once known ASD or NDD genes were excluded, the residual burden of damaging DNVs in genes within the top 10% of LOEUF scores is greatly reduced and not significantly associated with cognitive phenotype in ASD (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41588-022-01148-2#Fig4">4a</a>). Overtransmission of rare LoFs in genes within the top 10% of LOEUF genes to ASD cases with cognitive impairment is about 2.7 times higher than to cases without cognitive impairment (<i>P</i> = 4.6 × 10<sup>−3</sup> by χ<sup>2</sup> test) and is still 2 times higher (<i>P</i> = 0.04 by χ<sup>2</sup> test) once known ASD or NDD genes were excluded (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41588-022-01148-2#Fig4">4b</a>). However, rare LoFs in genes prioritized by A-risk are not associated with cognitive impairment (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41588-022-01148-2#MOESM1">4</a>). Taken together, these results suggest that rare variants in the top 10% of LOEUF genes—most of which are already known to be ASD or NDD risk genes—are associated with cognitive impairment. However, a subset of rare inherited variants, particularly those prioritized by A-risk, are not associated with cognitive impairment.</p><div data-test="figure" data-container-section="figure" id="figure-4" data-title="Association of rare inherited LoFs with cognitive impairment in ASD cases."><figure><figcaption><b id="Fig4" data-test="figure-caption-text">Fig. 4: Association of rare inherited LoFs with cognitive impairment in ASD cases.</b></figcaption><div><div><a data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="https://www.nature.com/articles/s41588-022-01148-2/figures/4" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41588-022-01148-2/MediaObjects/41588_2022_1148_Fig4_HTML.png?as=webp"/><img aria-describedby="Fig4" src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41588-022-01148-2/MediaObjects/41588_2022_1148_Fig4_HTML.png" alt="figure 4" loading="lazy" width="685" height="432"/></picture></a></div><p>Ultra-rare inherited LoFs with pExt ≥ 0.1 in genes with the top 10% of gnomAD LOEUF scores also show a higher proportion of transmission and a higher overtransmission rate to ASD offspring with cognitive impairment (CogImp) than those without (NoCogImp). Rare LoFs in other constrained genes are not significantly associated with phenotypic severity. The increased burden of inherited LoFs in cases with cognitive impairment remains significant after removing known ASD or NDD genes. Data are presented as mean values ± standard errors as error bars. Poisson test was used to compute the <i>P</i> values to assess the fold enrichment, and binominal test was used for overtransmission. Exact <i>P</i> values are listed in Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41588-022-01148-2#MOESM4">19</a>.</p></div></figure></div><h3 id="Sec7">Meta-analysis identifies five new risk genes</h3><p>Based on results from the first stage, we identified 260 genes with evidence of TDT (TDT statistic<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 39" title="Ewens, W. J. &amp; Spielman, R. S. The transmission/disequilibrium test: history, subdivision, and admixture. Am. J. Hum. Genet. 57, 455–464 (1995)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR39" id="ref-link-section-d6323361e6829">39</a></sup> ≥ 1) and in gene sets enriched with rare inherited LoFs (top 10% LOEUF or within top 20% LOEUF and A-risk ≥ 0.4) (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41588-022-01148-2#MOESM4">6</a>) and 159 genes with <i>P</i> &lt; 0.001 from the DeNovoWEST analysis of DNVs (with 15 genes by both) (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41588-022-01148-2#MOESM4">3</a>). We performed a meta-analysis on the 367 autosomal genes with all data from stage 1 and stage 2, which includes 6,174 new ASD trios, 1,942 new duos, 15,780 unrelated cases (see <a data-track="click" data-track-label="link" data-track-action="section anchor" href="https://www.nature.com/articles/s41588-022-01148-2#Sec11">Methods</a>) and 236,000 population controls.</p><p>We used Fisher’s method<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 40" title="Fisher, R. A. Statistical Methods for Research Workers 11th edn, rev (Oliver and Boyd, 1925)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR40" id="ref-link-section-d6323361e6848">40</a></sup> to combine three <i>P</i> values that estimate independent evidence of DNVs, TDT and case-control comparison: (1) DeNovoWEST with DNVs from both stage 1 and stage 2 (<i>n</i> = 23,039 trios, Supplementary Tables <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41588-022-01148-2#MOESM4">1</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41588-022-01148-2#MOESM4">7</a>) using the parameters estimated in stage 1, (2) TDT with rare LoF variants in parents without ASD diagnoses or intellectual disability with affected offspring in 15,586 trios and 4,907 duos (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41588-022-01148-2#MOESM4">4</a>) and (3) unrelated cases (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41588-022-01148-2#MOESM4">8</a>) compared with population controls using a binomial test. We used two sets of controls: gnomAD exome v2.1.1 non-neuro subset (only samples from individuals who were not ascertained for having a neurological condition in a neurological case-control study, <i>n</i> = 104,068) and TOPMed WGS (freeze 8, <i>n</i> = 132,345). We performed a case-control burden test using the two sets separately and input the larger <i>P</i> value for the meta-analysis. This approach avoids sample overlap and helps ensure that significant genes are not dependent on the choice of population reference. Although population reference data were processed by different pipelines, the cumulative allele frequencies (CAFs) of high-confidence LoF variants (see <a data-track="click" data-track-label="link" data-track-action="section anchor" href="https://www.nature.com/articles/s41588-022-01148-2#Sec11">Methods</a>) are similar between internal pseudocontrols (see <a data-track="click" data-track-label="link" data-track-action="section anchor" href="https://www.nature.com/articles/s41588-022-01148-2#Sec11">Methods</a>) and the two population references after applying the same LoF filters (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41588-022-01148-2#MOESM1">5</a>). Previous population genetic simulations predict that for genes under moderate to strong selection (selection coefficient &gt; 0.001), deleterious variants are expected to arise within 1,000 generations and population demographic histories do not confound the CAFs of deleterious alleles in these genes<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 41" title="Zuk, O. et al. Searching for missing heritability: designing rare variant association studies. Proc. Natl Acad. Sci. USA 111, E455–E464 (2014)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR41" id="ref-link-section-d6323361e6890">41</a></sup>.</p><p>For 367 selected autosomal genes, the point estimates of selection coefficient under the mutation–selection balance model<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 42" title="Wright, S. Evolution in Mendelian populations. Genetics 16, 97–159 (1931)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR42" id="ref-link-section-d6323361e6897">42</a></sup> are all greater than 0.01 (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41588-022-01148-2#MOESM1">6</a>). Most high-confidence LoF variants in these genes are ultra-rare (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41588-022-01148-2#MOESM1">7</a>), and the CAFs of high-confidence LoF variants in European and non-European population samples are highly correlated (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41588-022-01148-2#MOESM1">8</a>). Therefore, we included population samples across all ancestries as controls. The ultra-rare synonymous variant burden is similar between cases and controls across the selected genes (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41588-022-01148-2#Fig10">3</a>). To make use of all genetic data collected, we also included rare variants of unknown inheritance from ASD cases that were analyzed in the first stage. These variants come from cases that are part of unaffected parent–ASD duos; such variants were either inherited from the parent not participating in the study or DNVs. Therefore, these variants are independent of TDT, even though the same cases were included in TDT.</p><p>We identified 60 genes with exome-wide significance (<i>P</i> &lt; 2.5 × 10<sup>−6</sup>), and 72 genes reached study-wide significance accounting for all 5,754 constrained genes (<i>P</i> &lt; 8.7 × 10<sup>−6</sup>, Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41588-022-01148-2#MOESM4">9</a>). Figure <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41588-022-01148-2#Fig5">5</a> summarizes the distribution of LoF variants (with different modes of inheritance) in genes that reached study-wide significance by DNV enrichment (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41588-022-01148-2#Fig5">5a</a>) and other significant genes by meta-analysis (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41588-022-01148-2#Fig5">5b</a> and Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41588-022-01148-2#MOESM1">9</a>). Genes that are significant only in meta-analysis tend to harbor more inherited LoF variants than DNVs, consistent with their lower penetrance for ASD or NDD.</p><div data-test="figure" data-container-section="figure" id="figure-5" data-title="Distribution of de novo and inherited LoF variants in known and novel ASD genes in cases and population controls."><figure><figcaption><b id="Fig5" data-test="figure-caption-text">Fig. 5: Distribution of de novo and inherited LoF variants in known and novel ASD genes in cases and population controls.</b></figcaption><div><div><a data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="https://www.nature.com/articles/s41588-022-01148-2/figures/5" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41588-022-01148-2/MediaObjects/41588_2022_1148_Fig5_HTML.png?as=webp"/><img aria-describedby="Fig5" src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41588-022-01148-2/MediaObjects/41588_2022_1148_Fig5_HTML.png" alt="figure 5" loading="lazy" width="685" height="736"/></picture></a></div><p>From left to right: pyramid plots summarizing the number of de novo LoFs in 15,857 ASD trios, inherited high-confidence LoFs in 18,720 unrelated offspring included in transmission analysis, and high-confidence LoFs in 15,780 unrelated cases; bar plot of transmission vs nontransmission for rare high-confidence LoFs identified in parents without ASD diagnoses or intellectual disability; three plots comparing the high-confidence LoF rate in 31,976 unrelated ASD cases with gnomAD exomes (non-neuro subset, 104,068 individuals). Horizontal bars are presented as mean values ± standard errors as error bars. <b>a</b>, Twenty-eight known ASD or NDD genes that have LOEUF scores in the top 30% of gnomAD, have a <i>P</i> value for enrichment among all DNVs (<i>P</i> &lt; 9 × 10<sup>−6</sup>) in 23,039 ASD trios, and have more than 10 LoFs. <b>b</b>, Nine additional ASD risk genes that achieved a <i>P</i> value of &lt;9 × 10<sup>−6</sup> in stage 2 of this analysis. The majority of genes in <b>b</b> harbor more inherited LoFs than DNVs. All five novel genes (Table <a data-track="click" data-track-label="link" data-track-action="table anchor" href="https://www.nature.com/articles/s41588-022-01148-2#Tab1">1</a>) are shown in <b>b</b>. Note that the <i>x</i> axes of LoF rates are in the squared root scale. Poisson test was used to compute the <i>P</i> values. Exact <i>P</i> values are listed in Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41588-022-01148-2#MOESM4">6</a>.</p></div></figure></div><p>Although most significant genes were previously known, we identified five new genes that have exome-wide significance regardless of the choice of population reference: <i>NAV3</i>, <i>MARK2</i>, <i>ITSN1</i>, <i>SCAF1</i> and <i>HNRNPUL2</i> (Table <a data-track="click" data-track-label="link" data-track-action="table anchor" href="https://www.nature.com/articles/s41588-022-01148-2#Tab1">1</a>). The combined <i>P</i> values based on ancestry-specific case-control analyses are similar to the overall case-control analysis for these five genes (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41588-022-01148-2#MOESM4">10</a>). As expected, most supporting variants are ultra-rare, and results are robust to the allele frequency filter. These five new genes together explain 0.27% of the PAR ratio (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41588-022-01148-2#MOESM4">11</a>). <i>NAV3</i> has a similar PAR to that of <i>CHD8</i> and <i>SCN2A</i> (~0.095%). <i>ITSN1</i> is similar to <i>PTEN</i> (~0.065%).</p><div data-test="inline-table" data-container-section="table" id="table-1"><figure><figcaption><b id="Tab1" data-test="table-caption">Table 1 Statistical evidence for the five novel exome-wide-significant ASD risk genes identified in this study</b></figcaption></figure></div><p>The association of <i>NAV3</i> with ASD risk is primarily driven by rare inherited variants (Table <a data-track="click" data-track-label="link" data-track-action="table anchor" href="https://www.nature.com/articles/s41588-022-01148-2#Tab1">1</a>). <i>NAV3</i> has a high A-risk score, suggesting that the expression pattern of <i>NAV3</i> is highly similar to known ASD genes (Supplementary Data <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41588-022-01148-2#MOESM5">1</a>)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Feliciano, P. et al. Exome sequencing of 457 autism families recruited online provides evidence for autism risk genes. npj Genomic Med 4, 19 (2019)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR7" id="ref-link-section-d6323361e7760">7</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 43" title="Skene, N. G. &amp; Grant, S. G. N. Identification of vulnerable cell types in major brain disorders using single cell transcriptomes and expression weighted cell type enrichment. Front. Neurosci. 10, 16 (2016)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR43" id="ref-link-section-d6323361e7763">43</a></sup>. <i>NAV3</i> has high expression in the inner cortical plate of the developing cortex<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 33" title="Parikshak, N. N. et al. Integrative functional genomic analyses implicate specific molecular pathways and circuits in autism. Cell 155, 1008–1021 (2013)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR33" id="ref-link-section-d6323361e7770">33</a></sup>, and in pyramidal neurons (hippocampus CA1 and somatosensory cortex) and cortical interneurons<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 44" title="Willsey, A. J. et al. Coexpression networks implicate human midfetal deep cortical projection neurons in the pathogenesis of autism. Cell 155, 997–1007 (2013)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR44" id="ref-link-section-d6323361e7774">44</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 45" title="Chang, J., Gilman, S. R., Chiang, A. H., Sanders, S. J. &amp; Vitkup, D. Genotype to phenotype relationships in autism spectrum disorders. Nat. Neurosci. 18, 191–198 (2015)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR45" id="ref-link-section-d6323361e7777">45</a></sup> (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41588-022-01148-2#MOESM1">10</a>). The association of <i>MARK2</i> with ASD risk is primarily driven by DNVs and is also associated with other NDDs<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 11" title="Kaplanis, J. et al. Evidence for 28 genetic disorders discovered by combining healthcare and research data. Nature 586, 757–762 (2020)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR11" id="ref-link-section-d6323361e7788">11</a></sup> (<i>P</i> = 2.7 × 10<sup>−5</sup> by DeNovoWEST) including Tourette syndrome<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 46" title="Wang, S. et al. De novo sequence and copy number variants are strongly associated with Tourette disorder and implicate cell polarity in pathogenesis. Cell Rep. 24, 3441–3454.e12 (2018)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR46" id="ref-link-section-d6323361e7797">46</a></sup> and epilepsy<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 47" title="Heyne, H. O. et al. De novo variants in neurodevelopmental disorders with epilepsy. Nat. Genet. 50, 1048–1053 (2018)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR47" id="ref-link-section-d6323361e7801">47</a></sup>. We find that three out of eight autistic offspring with variants in <i>MARK2</i> report epilepsy, two out of eight report Tourette syndrome and seven out of eight have evidence of cognitive impairment (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41588-022-01148-2#MOESM4">12</a>).</p><p>The remaining three novel genes have support from both DNVs and rare LoFs. <i>ITSN1</i> and <i>SCAF1</i> show nominal significance of DNV enrichment in 31,058 NDD trios<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 11" title="Kaplanis, J. et al. Evidence for 28 genetic disorders discovered by combining healthcare and research data. Nature 586, 757–762 (2020)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR11" id="ref-link-section-d6323361e7821">11</a></sup> (<i>P</i> &lt; 0.05 by DeNovoWEST). <i>SCAF1</i> was among the top 50 genes from a gene-based burden test in a recent schizophrenia case-control study (<i>P</i> = 0.0027 by burden test)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 48" title="Singh, T. et al. Rare coding variants in ten genes confer substantial risk for schizophrenia. Nature 604, 509–516 (2022)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR48" id="ref-link-section-d6323361e7835">48</a></sup>. Both <i>ITSN1</i> and <i>NAV3</i> have moderate effect sizes (point estimate of relative risk 3~6; Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41588-022-01148-2#MOESM4">11</a>). <i>ITSN1</i> has been highlighted in our previous study with evidence of enriched inherited LoFs<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Feliciano, P. et al. Exome sequencing of 457 autism families recruited online provides evidence for autism risk genes. npj Genomic Med 4, 19 (2019)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR7" id="ref-link-section-d6323361e7852">7</a></sup>. We also assessed deletions in these new genes. For both <i>ITSN1</i> and <i>NAV3</i>, we identified four partial or whole gene deletions in 33,083 parents without ASD diagnoses or intellectual disability that also show transmission disequilibrium to affected offspring (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41588-022-01148-2#Fig11">4</a>).</p><p>Although both de novo and rare inherited LoFs in the most constrained genes are strongly associated with intellectual disability in ASD (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41588-022-01148-2#Fig4">4</a>), the association of such variants in individual genes is heterogenous, as suggested by the lack of association of rare inherited variants in genes with high A-risk scores (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41588-022-01148-2#Fig9">2</a>). We calculated the burden of cognitive impairment (see <a data-track="click" data-track-label="link" data-track-action="section anchor" href="https://www.nature.com/articles/s41588-022-01148-2#Sec11">Methods</a>) in 87 ASD individuals with high-confidence LoF variants in the four novel moderate-risk genes and compared it with that in 129 individuals with high-confidence LoF in the well-established ASD risk genes <i>CHD8, SCN2A, SHANK3, ADNP</i> and <i>FOXP1</i>, as well as 8,731 individuals with ASD (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41588-022-01148-2#MOESM1">11</a>). Although most individuals with variants in well-established ASD risk genes have some evidence of cognitive impairment (88%), individuals with LoF variants in the moderate-risk genes had significantly lower burden (56%, <i>P</i> = 4.5 × 10<sup>−7</sup>). Individuals with high-confidence LOFs in the moderate-risk genes did not have a significantly different burden of cognitive impairment than 8,731 individuals with ASD in SPARK (56% vs 50%, <i>P</i> = n.s.). Individuals with LoF variants in the moderate-risk genes also had a similar male:female (4:1) ratio compared with the larger cohort, whereas individuals with variants in the well-established ASD risk genes showed significantly less male bias (1.6:1, <i>P</i> = 0.009) (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41588-022-01148-2#MOESM1">11</a>), as previously reported<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 2" title="Iossifov, I. et al. The contribution of de novo coding mutations to autism spectrum disorder. Nature 515, 216–221 (2014)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR2" id="ref-link-section-d6323361e7902">2</a></sup>. We also predicted full-scale intelligence quotient (IQ) on all participants based on parent-reported data using a machine learning method<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 49" title="Shu, C., Snyder, L. G., Shen, Y. &amp; Chung, W. K. The SPARK Consortium. Imputing cognitive impairment in SPARK, a large autism cohort. Autism Res. 15, 156–170 (2022)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR49" id="ref-link-section-d6323361e7906">49</a></sup>. Heterozygotes for rare LoFs in three (<i>NAV3</i>, <i>SCAF1</i> and <i>HNRNPUL2</i>) of the four new genes with substantial contribution from rare inherited variants have similar IQ distribution as the overall SPARK cohort (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41588-022-01148-2#Fig6">6a</a>), which is substantially higher than heterozygotes with rare LoFs in well-established, highly penetrant genes that contribute to ASD primarily through DNVs (‘DN genes’) such as <i>CHD8</i>, <i>SHANK3</i> and <i>SCN2A</i>. In fact, both novel and established genes with significant contribution from rare inherited LoFs are less associated with intellectual disability than NDD genes (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41588-022-01148-2#Fig6">6b</a>). Across these genes, there is a significant negative correlation (<i>r</i> = 0.78, <i>P</i> = 0.001) of estimated relative risk of rare LoFs with average predicted IQ of the individuals with these variants (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41588-022-01148-2#Fig6">6c</a>).</p><div data-test="figure" data-container-section="figure" id="figure-6" data-title="Predicted full-scale IQ in individuals with pathogenic variants in inherited or de novo genes in SPARK."><figure><figcaption><b id="Fig6" data-test="figure-caption-text">Fig. 6: Predicted full-scale IQ in individuals with pathogenic variants in inherited or de novo genes in SPARK.</b></figcaption><div><div><a data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="https://www.nature.com/articles/s41588-022-01148-2/figures/6" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41588-022-01148-2/MediaObjects/41588_2022_1148_Fig6_HTML.png?as=webp"/><img aria-describedby="Fig6" src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41588-022-01148-2/MediaObjects/41588_2022_1148_Fig6_HTML.png" alt="figure 6" loading="lazy" width="685" height="335"/></picture></a></div><p>We examined the distribution of predicted IQ using a machine learning method<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 49" title="Shu, C., Snyder, L. G., Shen, Y. &amp; Chung, W. K. The SPARK Consortium. Imputing cognitive impairment in SPARK, a large autism cohort. Autism Res. 15, 156–170 (2022)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR49" id="ref-link-section-d6323361e7957">49</a></sup> for 95 individuals with ASD with an LoF mutation in one of the five novel exome-wide-significant genes (<i>MARK2</i>, <i>NAV3</i>, <i>ITSN1</i>, <i>SCAF1</i> and <i>HNRNPUL2</i>) and nine known ASD genes (<i>CHD8</i>, <i>SHANK3</i>, <i>SCN2A</i>, <i>ADNP</i>, <i>ARID1B</i>, <i>FOXP1</i>, <i>KDM5B</i>, <i>GIGYF1</i> and <i>KMT2C</i>), compared with 2,545 SPARK participants with ASD and known IQ scores. The nine known ASD genes include six genes (pink and labeled ‘de novo, known’) that are well-established de novo ASD risk genes that exceed exome-wide significance and were most frequently identified in SPARK, which maximizes the number of samples available for genotype–phenotype analyses. We also included three genes (light blue and labeled ‘inherited, known’) that have some previous evidence for inherited ASD risk (<i>GIGYF1</i><sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Feliciano, P. et al. Exome sequencing of 457 autism families recruited online provides evidence for autism risk genes. npj Genomic Med 4, 19 (2019)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR7" id="ref-link-section-d6323361e8007">7</a></sup>, <i>KDM5B</i><sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 62" title="Martin, H. C. et al. Quantifying the contribution of recessive coding variation to developmental disorders. Science 362, 1161–1164 (2018)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR62" id="ref-link-section-d6323361e8014">62</a></sup> and <i>KMT2C</i><sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 63" title="Guo, H. et al. Inherited and multiple de novo mutations in autism/developmental delay risk genes suggest a multifactorial model. Mol. Autism 9, 64 (2018)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR63" id="ref-link-section-d6323361e8020">63</a></sup>) and were also frequently identified in SPARK. We denote the genes contributing to ASD primarily through de novo LoF variants in our analysis as ‘de novo’ (red), and the genes primarily through inherited LoF variants as ‘inherited’ (blue). <b>a</b>, Distribution of predicted IQ between individuals with ASD with LoF mutations in the five novel genes, nine known genes and all participants with ASD and known IQ scores in SPARK (<i>n</i> = 2,545). We compared the mean predicted IQ between participants with LoF mutations in ASD genes and all participants by two-sample <i>t</i>-test. *, 0.01 ≤ <i>P</i> &lt; 0.05; **, 0.001 ≤ <i>P</i> &lt; 0.01; ***, <i>P</i> &lt; 0.001. Exact <i>P</i> values are listed in Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41588-022-01148-2#MOESM4">19</a>. The box plots represent median as center, and interquartile range (IQR) as bounds of the box; the upper whisker extends from the upper bound of the box to 1.5 × IQR, and the lower whisker extends from the lower bound of the box to 1.5 × IQR. Two-sided <i>t</i>-test was used to compute the <i>P</i> values for comparing mean predicted IQ between ASD individuals with LoF mutation in specific gene and all ASD participants. Individuals with pathogenic variants in de novo risk genes have significantly lower predicted IQ than overall SPARK participants with ASD and known IQ scores, whereas individuals with LoF variants in moderate-risk, inherited genes show similar predicted IQ as the overall SPARK participants, with the exception of <i>ITSN1</i>. <b>b</b>, Distribution of predicted IQ between individuals with ASD gene grouped by both inheritance status (‘de novo’ or ‘inherited’) and whether the ASD genes are novel (‘novel’ or ‘known’). We compared the mean predicted IQ between individuals with pathogenic variants in de novo genes and inherited genes among our five novel genes and nine known genes. Overall, people with LoF mutations in de novo genes have an average of 13–16 points lower predicted IQ than individuals with LoF mutations in inherited genes, regardless of whether the ASD genes are novel or known. The box plots represent median as center, and IQR as the bounds of the box; the upper whisker extends from the upper bound of the box to 1.5 × IQR, and the lower whisker extends from the lower bound of the box to 1.5 × IQR. <b>c</b>, Average relative risk of ASD and average predicted IQ among different groups. Each dot shows the average of individuals with rare LoFs of a gene selected in <b>a</b>. The relative risk is estimated from mega-analysis and capped at 60. Pearson correlation between average IQ and log relative risk is −0.78 (<i>P</i> = 0.001). The horizontal line represents the average IQ (IQ = 79) of all SPARK individuals with predicted IQs. <i>ITSN1</i> is an outlier at the bottom left corner.</p></div></figure></div><p>Most known ASD or NDD genes that are enriched by LoF DNVs harbor more de novo than LoF inherited variants in ~16,000 unrelated ASD trios (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41588-022-01148-2#Fig5">5a</a> and Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41588-022-01148-2#MOESM1">16</a>), consistent with their high penetrance for ASD or NDD phenotypes and strong negative selection. Using population exome or WGS data, we calculated a point estimate of selection coefficient (<span>\(\hat s\)</span>)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 50" title="Arnheim, N. &amp; Calabrese, P. Understanding what determines the frequency and pattern of human germline mutations. Nat. Rev. Genet. 10, 478–488 (2009)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR50" id="ref-link-section-d6323361e8120">50</a></sup> of LoFs in each gene (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41588-022-01148-2#MOESM4">11</a>) and found that the fraction of de novo LoFs in ASD genes is higher in genes with large <span>\(\hat s\)</span>, and smaller in genes with small <span>\(\hat s\)</span> (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41588-022-01148-2#MOESM1">5b</a>), consistent with population genetic theory<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 51" title="Rees, E., Moskvina, V., Owen, M. J., O’Donovan, M. C. &amp; Kirov, G. De novo rates and selection of schizophrenia-associated copy number variants. Biol. Psychiatry 70, 1109–1114 (2011)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR51" id="ref-link-section-d6323361e8187">51</a></sup>. We also estimated average effect size of rare LoFs in ASD genes by comparing CAF in 31,976 unrelated cases and population exome or WGS data. As expected, known and newly significant ASD genes with higher risk for ASD are under stronger selection (larger <span>\(\hat s\)</span>) (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41588-022-01148-2#MOESM1">13</a>).</p><h3 id="Sec8">Functional similarity of new genes and known ASD genes</h3><p>To better appreciate the probable functional implications of the new exome-wide-significant genes that confer inherited risk for ASD, we integrated mechanistic (STRING<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 52" title="Szklarczyk, D. et al. STRING v11: protein–protein association networks with increased coverage, supporting functional discovery in genome-wide experimental datasets. Nucleic Acids Res. 47, D607–D613 (2019)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR52" id="ref-link-section-d6323361e8230">52</a></sup>) and phenotypic (Human Phenotype Ontology (HPO)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 53" title="Köhler, S. et al. The Human Phenotype Ontology in 2021. Nucleic Acids Res. 49, D1207–D1217 (2021)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR53" id="ref-link-section-d6323361e8234">53</a></sup>) data into a single embedding space (six dimensions, one for each archetype coefficient) using a combination of canonical correlation analysis and archetypal analysis (see <a data-track="click" data-track-label="link" data-track-action="section anchor" href="https://www.nature.com/articles/s41588-022-01148-2#Sec11">Methods</a>). This embedding space serves as an interpretive framework for putative ASD risk genes (<i>n</i> = 1,776). Six functional or phenotypic archetypes were identified (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41588-022-01148-2#Fig7">7</a> and Supplementary Tables <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41588-022-01148-2#MOESM4">13</a>–<a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41588-022-01148-2#MOESM4">15</a>) that represent pathways that are well understood to play a role in ASD: neurotransmission (archetype 1 or A1), chromatin modification (archetype 2 or A2), RNA processing (archetype 3 or A3), vesicle-mediated transport (archetype 4 or A4), MAPK signaling and migration (archetype 5 or A5), and cytoskeleton and mitosis (archetype 6 or A6), also enriched for intermediate filaments. These archetypes organize risk genes in a way that jointly maximizes their association with mechanisms (STRING clusters) and phenotypes (HPO terms). For instance, A1 genes (neurotransmission) are enriched for the STRING cluster CL:8435 (ion channel and neuronal system) and are also associated with seizure and epileptic phenotypes. A2 genes (chromatin modifiers) are enriched for nuclear factors and genes linked to growth and morphological phenotypes (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41588-022-01148-2#MOESM4">14</a>). We call genes that strongly map to an archetype (that is, &gt;2× the next highest-ranking archetype) ‘archetypal’; if this criterion is not met, we call the genes ‘mixed’. Archetypal genes are generally less functionally ambiguous than mixed genes. Of the five novel inherited risk genes, two are archetypal (suggesting function within known risk mechanisms): <i>NAV3</i> (A6, cytoskeleton and mitosis) and <i>ITSN1</i> (A4, vesicle-mediated transport). <i>SCAF1</i>, <i>MARK2</i> and <i>HNRNPUL2</i> are mixtures of the identified archetypes, largely A4 and A5. That these new genes did not resolve clearly into archetypes (that were defined by known and suspected autism risk genes) suggests that they may operate in potentially novel mechanisms. To elucidate these possibilities, we constructed an ad hoc archetype, defined by the centroid between <i>SCAF1</i>, <i>MARK2</i> and <i>HNRNPUL2</i> (see Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41588-022-01148-2#Fig7">7c</a>). Cell–cell junction (CL:6549) was the STRING cluster most associated with this centroid (<i>P</i> = 4.12 × 10<sup>−14</sup> by the Kolmogorov–Smirnov test; Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41588-022-01148-2#Fig7">7d</a>), which fits with its location between A4 (vesicle-mediated transport) and A5 (MAPK signaling and migration).</p><div data-test="figure" data-container-section="figure" id="figure-7" data-title="Functional or phenotypic embedding of ASD risk genes."><figure><figcaption><b id="Fig7" data-test="figure-caption-text">Fig. 7: Functional or phenotypic embedding of ASD risk genes.</b></figcaption><div><div><a data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="https://www.nature.com/articles/s41588-022-01148-2/figures/7" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41588-022-01148-2/MediaObjects/41588_2022_1148_Fig7_HTML.png?as=webp"/><img aria-describedby="Fig7" src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41588-022-01148-2/MediaObjects/41588_2022_1148_Fig7_HTML.png" alt="figure 7" loading="lazy" width="685" height="447"/></picture></a></div><p><b>a</b>, Using a combination of archetypal analysis and canonical correlation analysis, putative autism risk genes were organized into <i>k</i> <i>=</i> 6 archetypes that represent distinct mechanistic (STRING) and phenotypic (HPO) categorizations (neurotransmission, chromatin modification, RNA processing, vesicle-mediated transport, MAPK signaling and migration, and cytoskeleton and mitosis). Genes implicated by our meta-analysis are indicated by their label, with novel genes in red. <b>b</b>, For each of the five novel genes, we identified the five nearest neighbors in the embedding space among the 62 meta-analysis genes. <i>SCAF1</i>, <i>MARK2</i> and <i>HNRNPUL2</i> were identified as ‘mixed’ rather than ‘archetypal’ in their probable risk mechanisms. <b>c</b>, To gain further insight into possible risk mechanisms, we calculated the embedding distance to the centroid of these three genes, which was then used as an index variable to perform gene set enrichment analysis. <b>d</b>, A STRING cluster (CL:6549) containing genes related to cell–cell junctions and the gap junction was identified as being highly localized in this region of the embedding space (<i>P</i> = 4.12 × 10<sup>−</sup><sup>14</sup> by the Kolmogorov–Smirnov test). This may suggest that these genes confer autism risk through dysregulation of processes related to cell adhesion and migration.</p></div></figure></div><h3 id="Sec9">Power analysis</h3><p>The power of identifying risk genes with rare inherited variants or DNVs monotonically increases with increasing effect size or expected CAF under the null. New ASD genes to be discovered are likely to have smaller effect size than known ASD genes, as suggested by our results. Additionally, known ASD genes are biased toward longer genes, which have a higher background mutation rate of damaging variants (‘long genes’) (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41588-022-01148-2#Fig12">5</a>). Even though longer genes are more likely to be expressed in the brain and be relevant to ASD or NDD<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 54" title="Raychaudhuri, S. et al. Accurately assessing the risk of schizophrenia conferred by rare copy-number variation affecting genes with brain function. PLoS Genet. 6, e1001097 (2010)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR54" id="ref-link-section-d6323361e8358">54</a></sup>, among most constrained genes, long genes (LoF mutation rate<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 55" title="Carlson, J. et al. Extremely rare variants reveal patterns of germline mutation rate heterogeneity in humans. Nat. Commun. 9, 3753 (2018)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR55" id="ref-link-section-d6323361e8362">55</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 56" title="Samocha, K. E. et al. A framework for the interpretation of de novo mutation in human disease. Nat. Genet. 46, 944–950 (2014)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR56" id="ref-link-section-d6323361e8365">56</a></sup> above 80% quantile) and short genes (below 80%) have similar enrichment of damaging DNVs and rare inherited LoFs (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41588-022-01148-2#MOESM1">14</a>). Notably, for short genes, known genes have virtually no contribution to overtransmitted high-confidence LoFs to affected offspring (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41588-022-01148-2#MOESM1">14b</a>), suggesting that many short ASD risk genes remain to be identified.</p><p>We used a published framework<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 41" title="Zuk, O. et al. Searching for missing heritability: designing rare variant association studies. Proc. Natl Acad. Sci. USA 111, E455–E464 (2014)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR41" id="ref-link-section-d6323361e8378">41</a></sup> to analyze power based on case-control association of rare variants. For rare variants in genes under strong selection, CAF is largely determined by mutation rate and selection coefficient<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 41" title="Zuk, O. et al. Searching for missing heritability: designing rare variant association studies. Proc. Natl Acad. Sci. USA 111, E455–E464 (2014)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR41" id="ref-link-section-d6323361e8382">41</a></sup>. We therefore modeled power of discovering risk genes as a function of relative risk and selection coefficient. With about 5,500 constrained genes, the power of the current study was calculated for 31,976 unrelated cases and an experiment-wise error rate of 9 × 10<sup>−6</sup> (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41588-022-01148-2#Fig13">6</a>).</p><p>We inversed the power calculation to determine the required sample size to achieve 90% power under the same assumptions (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41588-022-01148-2#Fig14">7</a>). For genes at median LoF mutation rate across all genes, we estimated that it requires about 96,000 cases (three times the current sample size) to identify genes with similar effect size as <i>NAV3</i> (relative risk = 4.5) and <i>ITSN1</i> (relative risk = 5), and about 64,000 cases (twice the current sample size) to find genes with similar effect sizes as <i>SCAF1</i> (relative risk = 8) and <i>HNRNPUL2</i> (relative risk = 9). We note that ten and five times the current sample size, respectively, are required to detect genes similar to <i>NAV3</i> or <i>ITSN1</i> and genes similar to <i>SCAF1</i> or <i>HNRNPUL2</i> by DNVs alone.</p></div></div></section><section data-title="Discussion"><div id="Sec10-section"><h2 id="Sec10">Discussion</h2><div id="Sec10-content"><p>In this study, we identified five new ASD risk genes by both DNVs and rare inherited coding variants. We identified rare LoF variants in new ASD risk genes with modest effect size that are not strongly associated with intellectual disability. This finding represents a difference in phenotypic association with intellectual disability compared with other highly penetrant ASD genes. To find new risk genes with relative risks of 2–5 (comparable to the low relative risk genes from this study, <i>NAV3</i> and <i>ITSN1</i>) in the 50th percentile for gene-wide LoF mutation rate (2 × 10<sup>−6</sup>) and the 50th percentile for selection among known risk genes (0.2), our power analysis suggests that 52,000, 73,000, 116,000 or 227,000 total ASD cases are necessary, respectively (cf. equation (1) from power calculation in Supplementary material).</p><p>Our results suggest that the identification of new risk genes with rare inherited variants may substantially improve genetic diagnostic yield. We found that rare inherited LoF variants account for 6% of PAR, similar to de novo LoF variants. Over two-thirds of the PAR from coding DNVs is explained by known ASD or NDD genes. In contrast, less than 20% of PAR from rare inherited LoFs variants is explained by known genes, suggesting that most genes contributing to ASD risk through rare inherited variants are yet to be discovered. These unknown risk genes are still largely constrained to LoFs in the general population and/or have similar expression profiles in developing brains to known ASD risk genes. Combining evidence from both DNVs and rare inherited variants, we identified 60 genes associated with ASD with exome-wide significance, including five novel genes. Rare LoFs in these five new genes account for a PAR of 0.27%, about half of the PAR of the five most common highly penetrant ASD genes (<i>KDM5B, GIGYF1, CHD8, SCN2A</i> and <i>SHANK3</i>).</p><p><i>NAV3</i>’s association to autism is primarily driven by rare inherited variants. Carriers of rare LoFs in <i>NAV3</i> have an average predicted IQ of 81, slightly above the SPARK cohort average (IQ, 79). The prevalence of intellectual disability among <i>NAV3</i> heterozygotes is similar to the SPARK cohort average. This is distinctly different from established ASD risk genes (for example, <i>CHD8</i>, <i>SHANK3</i> and <i>SCN2A</i>), nearly all identified by highly penetrant DNVs, and are associated with intellectual disability in ASD cohorts<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 2" title="Iossifov, I. et al. The contribution of de novo coding mutations to autism spectrum disorder. Nature 515, 216–221 (2014)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR2" id="ref-link-section-d6323361e8469">2</a></sup>. The absence of intellectual disability is also observed in other genes (for example, <i>SCAF1</i>, <i>HNRNPUL2</i>, <i>GIGYF1</i>, <i>KDM5B</i> and <i>KMT2C</i>) with substantial contribution from rare inherited variants and modest effect size. Nevertheless, the data show that many individuals with variants in these new ASD genes are affected with various neuropsychiatric conditions such as epilepsy, schizophrenia, Tourette syndrome and attention deficit hyperactivity disorder (ADHD) (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41588-022-01148-2#MOESM4">12</a>). Detailed phenotyping of larger numbers of individuals carrying these rare inherited variants is needed to understand the phenotype associated with each gene. Such strategies should include a genetic and phenotypic assessment of family members who also carry the rare variant without ASD. Because all individuals consented in SPARK are recontactable, such studies will enable a more complete picture of the broad phenotypic effects of these variants without the bias of clinical ascertainment. Overall, these risk genes with modest effect size may represent a different class of ASD genes that are more directly associated with core symptoms of ASD and/or neuropsychiatric conditions rather than global brain development and intellectual disability.</p><p>The approaches used in this study made full use of rare variation, and this analytical method is generalizable to many conditions. In particular, the multiple methods used to reduce noise in LoF alleles present in control samples were particularly effective in assessing the signal within the novel genes of moderate effect. We also leveraged gene expression profiles informed by machine learning methods to help prioritize genes for the meta-analysis stage of our analysis<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 38" title="Chen, S. et al. Dissecting autism genetic risk using single-cell RNA-seq data. Preprint at 
                https://www.bioRxiv.org/content/10.1101/2020.06.15.153031v1
                
               (2020)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR38" id="ref-link-section-d6323361e8495">38</a></sup>. Future studies that leverage additional multiomic data such as the Genotype–Tissue Expression (GTEx) project may further improve the signal-to-noise ratio.</p><p>Our archetypal analysis (Supplementary Tables <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41588-022-01148-2#MOESM4">13</a>–<a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41588-022-01148-2#MOESM4">15</a>) provides some clues to the potential mechanisms of the five newly identified risk genes. <i>ITSN1</i> was unambiguously mapped to A4 (vesicle-mediated transport) and has a role in coordinating endocytic membrane traffic with the actin cytoskeleton<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 57" title="Yamabhai, M. et al. Intersectin, a novel adaptor protein with two Eps15 homology and five Src homology 3 domains. J. Biol. Chem. 273, 31401–31407 (1998)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR57" id="ref-link-section-d6323361e8512">57</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 58" title="Wang, W. et al. ITSN-1 controls vesicle recycling at the neuromuscular junction and functions in parallel with DAB-1. Traffic 9, 742–754 (2008)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR58" id="ref-link-section-d6323361e8515">58</a></sup><i>. NAV3</i> (A6, cytoskeleton and mitosis) is associated with both axon guidance<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 59" title="Stringham, E. G. &amp; Schmidt, K. L. Navigating the cell: UNC-53 and the navigators, a family of cytoskeletal regulators with multiple roles in cell migration, outgrowth and trafficking. Cell Adhes. Migr. 3, 342–346 (2009)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR59" id="ref-link-section-d6323361e8521">59</a></sup> and malignant growth and invasion<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 60" title="Karenko, L. et al. Primary cutaneous T-cell lymphomas show a deletion or translocation affecting NAV3, the human UNC-53 homologue. Cancer Res. 65, 8101–8110 (2005)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR60" id="ref-link-section-d6323361e8526">60</a></sup> and is thought to regulate cytoskeletal dynamics. Indeed, A6 is enriched for processes related to intermediate filaments (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41588-022-01148-2#MOESM4">14</a>), a known determinant of cell motility and polarity<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 61" title="Sanghvi-Shah, R. &amp; Weber, G. F. Intermediate filaments at the junction of mechanotransduction, migration, and development. Front Cell Dev. Biol. 5, 81 (2017)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR61" id="ref-link-section-d6323361e8533">61</a></sup>. Although <i>MARK2</i>, <i>SCAF1</i> and <i>HNRNPUL2</i> were not identified as archetypal (potentially suggesting divergence from well-known autism risk mechanisms), a search for functional enrichment of this interstitial region between A4 and A5 found that their roles in developmental risk may be most relevant at the cell–cell junction, particularly as it relates to migration (see Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41588-022-01148-2#Fig7">7d</a>).</p><p>Taken together, our results suggest that a continued focus on DNVs for ASD gene discovery may yield diminishing returns. By contrast, studies designed to identify genomic risk from rare and common inherited variants will not only yield new mechanistic insight, but also help explain the high heritability of ASD. SPARK is designed to recruit individuals across the autism spectrum, without relying on ascertainment at medical centers. As a result, SPARK may be better suited to identify genes with transmitted variants that have lower penetrance and to identify the genetic contributions to the full spectrum of autism. The strategy used by SPARK—to recruit and assess large numbers of individuals with autism across the spectrum and their available family members without costly, in-depth clinical phenotyping—is necessary to achieve the required sample size to fully elucidate genetic contributions to ASD. The ability to recontact and follow all SPARK participants will also be critical to deeply assess the phenotypes associated with the newly discovered genes and to develop and test novel treatments.</p></div></div></section><section data-title="Methods"><div id="Sec11-section"><h2 id="Sec11">Methods</h2><div id="Sec11-content"><p>We established the SPARK cohort to facilitate genotype-driven research of ASD at scale<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 23" title="The SPARK Consortium. SPARK: a US cohort of 50,000 families to accelerate autism research. Neuron 97, 488–493 (2018)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR23" id="ref-link-section-d6323361e8561">23</a></sup>. All participants were recruited to SPARK under a centralized institutional review board (IRB) protocol (Western IRB Protocol no. 20151664). All participants provided written informed consent to take part in the study. Written informed consent was obtained from all legal guardians or parents for all participants aged 18 and younger and for all participants aged 18 and older who have a legal guardian. Assent was also obtained from dependent participants aged 10 and older. Participants with autism were compensated US$25–50 depending on other registered family members. The mean age of the SPARK cohort in this analysis was 16.5 years (s.d., 19.2 years). Cases made up 46% of the SPARK cohort (the remaining 54% were controls). The sex breakdown of the full SPARK cohort was 57% male and 43% female. The sex breakdown of the SPARK case cohort was 77% male and 23% female.</p><p>The first stage of analysis included 28,649 SPARK participants, including 10,242 ASD cases from over 9,000 families with exome sequencing data that passed quality control (Supplementary Data <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41588-022-01148-2#MOESM5">1</a>). A subset of 1,379 individuals was part of the published pilot study<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Feliciano, P. et al. Exome sequencing of 457 autism families recruited online provides evidence for autism risk genes. npj Genomic Med 4, 19 (2019)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR7" id="ref-link-section-d6323361e8571">7</a></sup>. To replicate prioritized genes from the discovery stage, we performed a second-stage analysis that included an additional 39,926 individuals with 16,970 ASD cases from over 20,000 families with exome or WGS data available after the analysis in the discovery cohort was completed. New exome sequencing samples in this study were captured using an IDT xGen research panel and sequenced on Illumina NovaSeq. DNA samples were also genotyped for over 600,000 single nucleotide polymorphisms (SNPs) using Infinium Global Screening Array. Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41588-022-01148-2#MOESM4">16</a> outlines the software version and parameter settings for each analysis below. Details on data preprocessing, variant-level quality control procedures, variant annotation, high-confidence LoF variant filtering, and copy number variants, as well as descriptions of other publicly available ASD cohorts (SSC, MSSNG and ASC), are in the Supplementary Note.</p><h3 id="Sec12">DNVs</h3><p>We identified candidate de novo single nucleotide variants (SNVs) or insertion–deletion mutations (indels) from SPARK and SSC cohorts from per-family variant call formats files (VCFs) generated by GATK<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 64" title="Poplin, R. et al. Scaling accurate genetic variant discovery to tens of thousands of samples. Preprint at 
                https://www.bioRxiv.org/content/10.1101/201178v3
                
               (2018)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR64" id="ref-link-section-d6323361e8585">64</a></sup> (v.4.1.2.0) and freebayes<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 65" title="Garrison, E. &amp; Marth, G. Haplotype-based variant detection from short-read sequencing. Preprint at 
                https://arXiv.org/abs/1207.3907
                
               (2012)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR65" id="ref-link-section-d6323361e8589">65</a></sup> (v.1.1.0) and a cohort-wide population VCF generated by weCall<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 66" title="GENOMICS plc - weCall: fast, accurate and simple to use command line tool for variant detection in NGS data. 
                https://github.com/Genomicsplc/wecall
                
              " href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR66" id="ref-link-section-d6323361e8593">66</a></sup> (v.2.0.0) using a set of heuristic filters that aim to maximize the sensitivity while minimizing false negatives in parents<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Feliciano, P. et al. Exome sequencing of 457 autism families recruited online provides evidence for autism risk genes. npj Genomic Med 4, 19 (2019)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR7" id="ref-link-section-d6323361e8597">7</a></sup> (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41588-022-01148-2#MOESM4">16</a>). Candidates were retained if they were called by DeepVariant<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 67" title="Poplin, R. et al. A universal SNP and small-indel variant caller using deep neural networks. Nat. Biotechnol. 36, 983–987 (2018)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR67" id="ref-link-section-d6323361e8605">67</a></sup> (v.0.8.0) in offspring and had no support in parents, identified in multiple offspring that passed the DeepVariant filter in all trios, or were shared by siblings in the same family and the de novo quality estimated by triodenovo was higher than 8 (or 7 for SNVs in CpG context). Before creating the final call set, we selected subsets of variants (see Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41588-022-01148-2#MOESM4">16</a>) for manual evaluation by Integrative Genomics Viewer (IGV) to filter out candidates with failed review. Finally, we merged nearby clustered de novo coding variants (within 2 bp for SNVs or 50 bp for indels) on the same haplotype to form multinucleotide variants (MNVs) or complex indels. We removed variants located in regions known to be difficult for variant calling (HLA, mucin, and olfactory receptors). DNVs in the final call set follow a Poisson distribution with an average of 1.4 coding DNVs per affected offspring and 1.3 per unaffected offspring (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41588-022-01148-2#MOESM1">15</a>). The proportion of different types of DNVs, the mutation spectrum of SNVs, and indel length distributions were similar between SPARK and SSC (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41588-022-01148-2#MOESM1">15</a>). A small fraction of variants in the final call set are likely postzygotic mosaic mutations (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41588-022-01148-2#MOESM1">16</a>).</p><h3 id="Sec13">Rare variants</h3><p>Rare variant genotypes were filtered from cohort-wide population VCFs with quality-control metrics collected from individual and family VCFs (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41588-022-01148-2#MOESM1">17a</a>). In brief, we initially extracted high-quality genotypes for each individual for variants that appear in less than 1% of families in the cohort. Evidence for the variant genotypes was re-evaluated by DeepVariant from aligned reads and collapsed over individuals to create site-level summary statistics including the fraction of individual genotypes that passed DeepVariant filter and mean genotype quality over all individuals. For variant genotypes extracted from GLnexus<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 68" title="Lin, M. F. et al. GLnexus: joint variant calling for large cohort sequencing. Preprint at 
                https://www.bioRxiv.org/content/10.1101/343970v1
                
               (2018)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR68" id="ref-link-section-d6323361e8632">68</a></sup> (v1.1.3) VCFs, we re-examined variant genotype from per-family VCFs by GATK to collect GATK site-level metrics (including QD, MQ and SOR), then took the read-depth weighted average over families to create cohort-wide site metrics. For variant genotypes extracted from GATK joint genotyping VCFs, these site metrics were directly available from INFO fields.</p><h3 id="Sec14">De novo analysis</h3><p>In the discovery-stage analysis, the DNV call sets of SPARK and SSC were merged with published DNVs from ASC<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 3" title="De Rubeis, S. et al. Synaptic, transcriptional and chromatin genes disrupted in autism. Nature 515, 209–215 (2014)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR3" id="ref-link-section-d6323361e8645">3</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 8" title="Satterstrom, F. K. et al. Large-scale exome sequencing study implicates both developmental and functional changes in the neurobiology of autism. Cell 180, 568–584.e23 (2020)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR8" id="ref-link-section-d6323361e8648">8</a></sup> and MSSNG<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 6" title="Yuen, R. K. C. et al. Whole genome sequencing resource identifies 18 new candidate genes for autism spectrum disorder. Nat. Neurosci. 20, 602–611 (2017)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR6" id="ref-link-section-d6323361e8652">6</a></sup>. To infer likely sample overlaps with published trios for which we do not have individual-level data, we tallied the proportion of shared DNVs between all pairs of trios. For a pair of trios, let <i>N</i><sub>1</sub> and <i>N</i><sub>2</sub> be the number of coding DNVs, and let <i>O</i> be the number of shared DNVs between the pair. To account for mutation hot spots, if a DNV is an SNV within CpG context or a known recurrent DNV identified in SPARK and SSC, it contributes 0.5 to the count. Likely overlapping samples were identified if (<i>O</i>/<i>N</i><sub>1</sub>) ≥ 0.5 or (<i>O</i>/<i>N</i><sub>2</sub>) ≥ 0.5 and they have identical sex.</p><p>To determine the expected number of DNVs in the cohort, we used a 7-mer mutation rate model<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 55" title="Carlson, J. et al. Extremely rare variants reveal patterns of germline mutation rate heterogeneity in humans. Nat. Commun. 9, 3753 (2018)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR55" id="ref-link-section-d6323361e8685">55</a></sup> in which the expected haploid mutation rate of each base pair depends on the 3-bp sequence context on both sides. The per-base mutation rates were adjusted by the fraction of callable trios at each base pair, which was the fraction of trios with ≥10× coverage in parents and ≥15× coverage in offspring. For published trios, we used an in-house WGS dataset of 300 trios (average, 36× coverage) to approximate the callable regions. Gene-level haploid mutation rates for different classes of DNVs were calculated by summing up the depth-adjusted per-base mutation rate of all possible SNVs of the same class. The rate for frameshift variants was presumed to be 1.3× the rate of stop-gained SNVs<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 56" title="Samocha, K. E. et al. A framework for the interpretation of de novo mutation in human disease. Nat. Genet. 46, 944–950 (2014)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR56" id="ref-link-section-d6323361e8689">56</a></sup>. Mutation rates in haploid X chromosome regions were adjusted for the observed male:female ratio (4.2), assuming that mutation rates in spermatogenesis are 3.4 times higher than in oogenesis<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 9" title="The Deciphering Developmental Disorders Study. Large-scale discovery of novel genetic causes of developmental disorders. Nature 519, 223–228 (2015)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR9" id="ref-link-section-d6323361e8693">9</a></sup>. The exome-wide rate of synonymous DNVs closely matches the observed number of DNVs (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41588-022-01148-2#Fig8">1</a>). We also observed similar fold enrichment of damaging DNVs (vs expected rate) in ASD cases across four cohorts after accounting for samples with family history (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41588-022-01148-2#Fig8">1</a>).</p><p>To identify risk genes through DNVs, we applied DeNovoWEST<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 11" title="Kaplanis, J. et al. Evidence for 28 genetic disorders discovered by combining healthcare and research data. Nature 586, 757–762 (2020)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR11" id="ref-link-section-d6323361e8706">11</a></sup>. We used the empirical burden of DNVs to derive weights for different variant classes in constrained genes (ExAC pLI ≥ 0.5) and nonconstrained genes separately based on positive predictive values (PPVs) (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41588-022-01148-2#MOESM4">18</a>). For ASD, we defined de novo D-mis variants by REVEL score ≥ 0.5. For other NDDs, we defined two classes of de novo D-mis variants by MPC score ≥ 2 or MPC ≤ 2 and CADD score ≥ 25. We first ran DeNovoWEST to test the enrichment of all nonsynonymous DNVs (pEnrichAll). To account for risk genes that harbor only missense variants, we ran DenovoWEST to test the enrichment of de novo missense variants only and applied a second test for spatial clustering of missense variants using denovonear<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 9" title="The Deciphering Developmental Disorders Study. Large-scale discovery of novel genetic causes of developmental disorders. Nature 519, 223–228 (2015)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR9" id="ref-link-section-d6323361e8713">9</a></sup>, then combined evidence of missense enrichment and clustering (pCombMis). The minimal of pEnrichAll and pCombMis was used as the final <i>P</i> value for DeNovoWEST. The exome-wide significance threshold was set to 1.3 × 10<sup>−6</sup> (=0.05/(18,000 genes × 2)). The analysis on the replication cohort used the same weights as derived from the discovery cohort. Compared with the original publication<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 11" title="Kaplanis, J. et al. Evidence for 28 genetic disorders discovered by combining healthcare and research data. Nature 586, 757–762 (2020)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR11" id="ref-link-section-d6323361e8723">11</a></sup>, our implementation of DeNovoWEST used different ways to stratify genes, determine variant weights and calculate per-base mutation rates. We applied our DeNovoWEST implementation on 31,058 NDD trios. The results are highly concordant with published results on the same data (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41588-022-01148-2#MOESM1">18</a>). We used <i>P</i> values from our reanalysis on other NDD trios in comparative analysis with ASD.</p><p>Gene set enrichment analysis of DNVs was performed using the DNENRICH framework<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 32" title="Fromer, M. et al. De novo mutations in schizophrenia implicate synaptic networks. Nature 506, 179–184 (2014)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR32" id="ref-link-section-d6323361e8736">32</a></sup>. We included all de novo LoF and D-mis variants in 5,754 constrained genes from 16,877 ASD and 5,764 control trios. For each gene set, we calculated the fraction of weighted sums of damaging DNVs using PPV weights of constrained genes (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41588-022-01148-2#MOESM4">18</a>) for cases and controls respectively. The test statistic for each gene set is the ratio of such fractions in cases over controls. To determine the distribution of the test statistic under the null hypothesis, we randomly placed mutations onto the exome of all constrained genes, while holding the number of mutations, their trinucleotide context and functional impact to be the same as observed in cases and controls separately. Note that by conditioning on the observed number of damaging DNVs in cases and controls, we tested enriched gene sets in cases that are not due to an increased overall burden. At each round of simulation, the permuted test statistic in each gene set was calculated. Finally, the <i>P</i> value was calculated as the number of times that the permuted statistic was no less than the observed statistic. Fold enrichment was calculated as the ratio between observed and average test statistics over all permutations. We also approximated the 95% confidence interval for fold enrichment by assuming that log[fold enrichment] follows normal distribution with mean 0 and s.d. determined by the <i>P</i> value.</p><p>In all DNV analyses above, DNVs shared by full or twin siblings represent single mutational events and were counted only once. When an individual carries multiple DNVs within 100 bp in the same gene, only one variant with the most severe effects was included in the analysis.</p><h3 id="Sec15">Transmission disequilibrium analysis</h3><p>The effect of inherited LoF variants was analyzed using TDT in each individual gene or in gene sets. Rare LoF variants were first identified in parents without ASD diagnoses or intellectual disability, then for each parent–offspring pair, the number of times the LoF variant was transmitted from parents to offspring was tallied. For variants in the (non-PAR part of the) X chromosome, we only used rare LoF variants carried by mothers without ASD diagnoses or intellectual disability and analyzed transmission in different types of mother–offspring pairs. For TDT analysis of rare inherited missense variants, different D-mis definitions and allele frequency cutoffs were used (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41588-022-01148-2#MOESM1">2</a>).</p><p>The overtransmission of LoFs to affected offspring was evaluated using a binomial test assuming that under the null hypothesis the chance of transmission is 0.5. In the discovery stage, ultra-rare LoFs with pExt ≥ 0.1 were used in exome-wide transmission disequilibrium and gene set enrichment analysis. For the per-gene test, all rare LoFs with pExt ≥ 0.1 were also used, and the TDT statistic<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 39" title="Ewens, W. J. &amp; Spielman, R. S. The transmission/disequilibrium test: history, subdivision, and admixture. Am. J. Hum. Genet. 57, 455–464 (1995)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR39" id="ref-link-section-d6323361e8767">39</a></sup> for each gene was calculated by <span>\(z = \frac{{T - NT}}{{\sqrt {T + NT} }}\)</span>, where <i>T</i>(<i>NT</i>) is the number of times LoF variants were transmitted (not transmitted) to affected offspring. When offspring included monozygotic twin pairs, only one was kept in the transmission analysis. We prioritized 244 autosomal genes with <i>z</i> &gt; 1 in the top 10% of LOEUF or in the top 20% of LOEUF and A-risk ≥ 0.4. In the second stage gene-based test, if a gene-specific pExt threshold was available, we used high-confidence LoF variants that passed the gene-specific pExt filter (see Supplementary Note).</p><p>In the gene set enrichment analysis of inherited LoFs, the rate of transmission to affected offspring in each gene set was compared with the transmission rate in the rest of the genes in the background using the χ<sup>2</sup> test.</p><h3 id="Sec16">Case-control analysis</h3><p>Pseudocontrols are constructed from parents without ASD diagnoses or intellectual disability in simplex families, using alleles that were not transmitted to affected offspring. Each parent without ASD diagnoses or intellectual disability contributes a sample size of 0.5 to pseudocontrols. Rare LoFs in ASD cases whose parent data are not available and from other cases that were not used in DNV enrichment or TDT analysis were analyzed in this stage. Specifically, for each ASD case, we found out all of their most recent unaffected ancestors without ASD diagnoses or intellectual disability in the pedigree and calculated the contributing sample size as 1 minus the summation of kinship coefficients with these ancestors. If the contributing sample size was greater than 0, then the sample was included in pseudocases after removing alleles that were observed in any unaffected ancestors without ASD diagnoses or intellectual disability used in TDT and alleles included in DNV analysis if any. Examples of such rare LoFs in cases and their contributing sample sizes are given in Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41588-022-01148-2#MOESM1">19</a>.</p><p>Rare LoFs in cases and controls for the X chromosome were categorized separately for males and females. Male controls include all fathers. In contrast, male cases include only those whose mothers do not have ASD diagnoses or intellectual disability (thus not included in TDT analysis). For females, because we include only mothers without ASD diagnoses or intellectual disability and affected sons in TDT, female pseudocases include all affected females. Female pseudocontrols were established from unaffected mothers in simplex families using alleles that do not transmit to affected sons. Each unaffected mother contributes a sample size of 0.5 to pseudocontrols. In both sexes, DNVs were removed from pseudocases.</p><p>For gene-based case-control association in stage 2, we used population data as controls, including gnomAD exomes<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 26" title="Karczewski, K. J. et al. The mutational constraint spectrum quantified from variation in 141,456 humans. Nature 581, 434–443 (2020)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR26" id="ref-link-section-d6323361e8855">26</a></sup> (v2.1.1 non-neuro subset) and TOPMed genomes<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 69" title="Taliun, D. et al. Sequencing of 53,831 diverse genomes from the NHLBI TOPMed Program. Nature 590, 290–299 (2021)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR69" id="ref-link-section-d6323361e8859">69</a></sup> (freeze 8). Variants in the population controls were filtered to keep those that passed the default quality-control filter in released data. For variants in gnomAD data, we further removed variants located in low-complexity regions<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 70" title="Li, H. Toward better understanding of artifacts in variant calling from high-coverage samples. Bioinformatics 30, 2843–2851 (2014)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR70" id="ref-link-section-d6323361e8863">70</a></sup> using the same procedure that was already applied to cases and TOPMed data. In gene-level case-control comparison of LoF burden, we used a baseline pExt ≥ 0.1 filter or gene-specific pExt threshold if available to define high-confidence LoF variants (see Supplementary Note). For LoF variants in selected genes, we also extracted curation results by gnomAD to remove curated non-LoF variants, and manually reviewed IGV snapshots from the gnomAD browser if available to remove likely variant calling artifacts (Supplementary Data <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41588-022-01148-2#MOESM5">1</a>). The number of high-confidence LoF variants was obtained from the allele counts in site-level VCF files. The gene-level burden of high-confidence LoF variants between cases and population controls is tested by comparing the high-confidence LoF variant rates between cases and controls using the Poisson test. The gene-level burden of ultra-rare synonymous variants (allele frequency &lt; 1 × 10<sup>−5</sup>) between cases and population controls is assessed by a quantile–quantile (Q–Q) plot among all ancestry and European ancestry (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41588-022-01148-2#Fig10">3</a>). To account for difference in depth of coverage, sample sizes are multiplied by the fraction of callable coding regions of each gene (≥15× for autosomes or female X chromosome, ≥10× for male X chromosome) in ASD cases and in population controls respectively.</p><p>To account for sample relatedness in case-control analysis, we created a relationship graph in which each node represents an individual and each edge represents a known first-degree or second-degree relationship between two individuals. We also added edges to pairs of individuals without known familial relationship but have an estimated kinship coefficient ≥ 0.1. From the graph, we select one individual from each connected component to create unrelated case-control samples. For the X chromosome, father and sons were treated as unrelated. For population controls, only gnomAD data included sex-specific allele counts and were used in the sex-specific analysis.</p><p>Meta-analysis was performed for prioritized autosomal genes that are constrained (defined as ExAC pLI ≥ 0.5 or ranked in the top 20% of the LOEUF scores; Supplementary Data <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41588-022-01148-2#MOESM6">2</a>). We integrated evidence from the enrichment of all DNVs, transmission disequilibrium and increased burden in cases compared with population controls by combining <i>P</i> values using Fisher’s method<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 40" title="Fisher, R. A. Statistical Methods for Research Workers 11th edn, rev (Oliver and Boyd, 1925)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR40" id="ref-link-section-d6323361e8889">40</a></sup>. Study-wide significance was set at 9 × 10<sup>−6</sup> (Bonferroni correction with 5,754 constrained genes). In mega-analysis, we combined all unrelated ASD cases together and compared CAFs of high-confidence LoF variants with two population control sets.</p><h3 id="Sec17">Power calculation</h3><p>To calculate statistical power of the current study and to estimate sample size for future studies, we adopted the statistical framework from ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 41" title="Zuk, O. et al. Searching for missing heritability: designing rare variant association studies. Proc. Natl Acad. Sci. USA 111, E455–E464 (2014)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR41" id="ref-link-section-d6323361e8903">41</a></sup>, comparing CAF of LoF variants in <i>N</i> unrelated cases <i>f</i><sub>case</sub> with CAF <i>f</i> in the general population. The effects of LoFs in the same gene are assumed to be the same (relative risk = <i>γ</i>). We focus only on constrained genes. We assume <i>f</i> to be at mutation–selection equilibrium <i>f</i> = <i>μ</i><sub>LoF</sub>/<i>s</i>, where <i>μ</i><sub>LoF</sub> is the LoF mutation rate and <i>s</i> is the selection coefficient. The test statistic asymptotically follows a noncentral χ<sup>2</sup> distribution with 1 d.f. and noncentrality parameter (NCP):</p><div id="Equa"><p><span>$$\lambda = 4N\left[ {\gamma f\ln \gamma + (1 - \gamma ){{{\mathrm{ln}}}}\frac{{1 - \gamma f}}{{1 - f}}} \right]$$</span></p></div><p>Given the significance threshold <i>α</i>, power can be calculated analytically by</p><div id="Equb"><p><span>$$1 - \beta = 1 - F\left[ {F^{ - 1}\left( {1 - \alpha ,0} \right),\lambda } \right]$$</span></p></div><p>where <i>F</i>(<i>x</i>, <i>λ</i>) is the cumulative distribution of <span>\(\chi _1^2\)</span> with NCP <i>λ</i>.</p><p>To calculate the sample size to achieve desired power 1 − <i>β</i> at significance level <i>α</i>, we first solve NCP <i>λ</i><sub><i>α</i>,<i>β</i></sub> from the above equation, then estimate sample size by</p><div id="Equc"><p><span>$$n_{\alpha ,\beta } \approx \frac{{\lambda _{\alpha ,\beta }}}{{4f\left[ {\gamma \ln \gamma - \left( {\gamma - 1} \right)} \right]}}$$</span></p></div><p>The current study has a sample size of <i>n</i> = 31,976 and the type I error rate is <i>α</i> = 9 × 10<sup>−6</sup> (experimental wide significant). Given continuing expansion of biobank-scale sequencing, treating <i>f</i> as known without error is a reasonable assumption for future studies. To calculate power for the new genes identified in this study, we used point estimates of <i>γ</i> and <i>f</i> from mega-analysis using gnomAD exomes as population controls, and used <i>μ</i><sub>LoF</sub> computed from the 7-mer context-dependent mutation rate model<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 55" title="Carlson, J. et al. Extremely rare variants reveal patterns of germline mutation rate heterogeneity in humans. Nat. Commun. 9, 3753 (2018)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR55" id="ref-link-section-d6323361e9289">55</a></sup> to convert <i>f</i> to <i>s</i> = <i>μ</i><sub>LoF</sub>/<i>f</i>. The required sample sizes were calculated to achieve 90% power.</p><p>Power and sample size are both calculated as a function of relative risk for ASD (<i>γ</i>) and selection coefficient (<i>s</i>) across different haploid LoF mutation rates (<i>μ</i><sub>LoF</sub>). We only considered <i>s</i> between 0.01 and 0.5, because most prioritized genes have point estimates of <i>s</i> &gt; 0.01 (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41588-022-01148-2#MOESM1">5</a>) and genes with <i>s</i> &gt; 0.5 are expected to harbor more de novo than inherited LoF variants and can be identified from the enrichment of DNVs. We limited relative risk (<i>γ</i>) between 1 and 20 to focus on risk genes with moderate to small effects. The reduction in fitness <i>s</i> is correlated with the increases in ASD risk <i>γ</i> by s = <i>γπ</i> under the assumption of no pleiotropic effect, where <i>π</i> is ASD prevalence and <i>s</i><sub><i>D</i></sub> is decreased reproductive fitness of ASD cases. Based on epidemiological studies, the current estimated prevalence of ASD is <span>\(\hat \pi = 1/54\)</span><sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 71" title="Maenner, M. J. et al. Prevalence of autism spectrum disorder among children aged 8 years — Autism and Developmental Disabilities Monitoring Network, 11 Sites, United States, 2016. MMWR Surveill. Summ. 69, 1–12 (2020)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR71" id="ref-link-section-d6323361e9394">71</a></sup>, and the estimated <i>s</i><sub><i>D</i></sub> is for 0.75 male and for 0.52 female<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 72" title="Power, R. A. et al. Fecundity of patients with schizophrenia, autism, bipolar disorder, depression, anorexia nervosa, or substance abuse vs their unaffected siblings. JAMA Psychiatry 70, 22–30 (2013)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR72" id="ref-link-section-d6323361e9405">72</a></sup>, so sex-averaged <span>\(\hat s_D = 0.71\)</span> (assuming male:female ratio of 4.2). In reality, most known ASD genes also show pleiotropic effects with other NDDs or are associated with prenatal death; therefore, <span>\(s \ge \gamma \pi s_D \approx \gamma \hat \pi \hat s_D = 0.013\gamma\)</span>. So, we considered only combinations of (<i>s</i>, <i>γ</i>) that satisfy the condition <i>s</i> ≥ 0.013<i>γ</i>.</p><h3 id="Sec18">Gene sets</h3><p>To evaluate the contribution of known ASD risk genes to the burdens of DNVs and inherited LoF variants identified in this study, we collected 618 known dominant ASD or NDD genes from the following sources: (1) Known NDD genes from DDG2P<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 73" title="Wright, C. F. et al. Genetic diagnosis of developmental disorders in the DDD study: a scalable analysis of genome-wide research data. Lancet 385, 1305–1314 (2015)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR73" id="ref-link-section-d6323361e9564">73</a></sup> (2020-02) that are dominant or X-linked and have an organ specificity list that includes the brain or cause multisystem syndrome; (2) high-confidence ASD genes collected by the Simons Foundation Autism Research Initiative (SFARI)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 74" title="Abrahams, B. S. et al. SFARI Gene 2.0: a community-driven knowledgebase for the autism spectrum disorders (ASDs). Mol. Autism 4, 36 (2013)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR74" id="ref-link-section-d6323361e9568">74</a></sup> (2019-08) with score of 1 or 2, excluding known recessive genes; and (3) dominant ASD genes reported in recent literature and included in the SPARK genes list<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 75" title="Singer, E. What makes an autism gene? 
                https://SPARKforAutism.org/discover_article/what-makes-an-autism-gene/
                
               (2017)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR75" id="ref-link-section-d6323361e9572">75</a></sup> (2020-07).</p><p>To evaluate the gene sets enriched by damaging DNVs or inherited high-confidence LoFs, we used all constrained genes by ExAC pLI ≥ 0.5 or in the top 20% of the LOEUF as the background. Gene sets of the following five categories were collected for gene set enrichment analysis.</p><h4 id="Sec19">Transcriptome and proteome</h4><p>(1) Genes with brain-specific expression, defined as the genes with average reads per kilobase of transcript per million mapped reads (RPKM) &gt; 1 in the brain and over four times the median RPKM of 27 tissues in processed RNA sequencing (RNA-seq) data from ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 76" title="Fagerberg, L. et al. Analysis of the human tissue-specific expression by genome-wide integration of transcriptomics and antibody-based proteomics. Mol. Cell. Proteomics 13, 397–406 (2014)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR76" id="ref-link-section-d6323361e9586">76</a></sup>. (2) Genes in coexpression modules M2 and M3 derived from weighted gene correlation network analysis (WGCNA) of BrainSpan developmental RNA-seq data, previously reported to be enriched for known ASD genes<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 33" title="Parikshak, N. N. et al. Integrative functional genomic analyses implicate specific molecular pathways and circuits in autism. Cell 155, 1008–1021 (2013)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR33" id="ref-link-section-d6323361e9590">33</a></sup>. (3) Genes expressed in excitatory or inhibitory neurons. We selected genes from ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 77" title="Mo, A. et al. Epigenomic signatures of neuronal diversity in the mammalian brain. Neuron 86, 1369–1384 (2015)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR77" id="ref-link-section-d6323361e9594">77</a></sup> that have average transcripts per million (TPM) &gt; 100 in excitatory neurons and in inhibitory neurons. (4) Synaptic genes collected from SynaptomeDB<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 78" title="Pirooznia, M. et al. SynaptomeDB: an ontology-based knowledgebase for synaptic genes. Bioinformatics 28, 897–899 (2012)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR78" id="ref-link-section-d6323361e9598">78</a></sup>.</p><h4 id="Sec20">Neuronal regulome</h4><p>(1) Putative CELF4 target genes, defined as genes with an iCLIP occupancy &gt; 0.2 in ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 79" title="Wagnon, J. L. et al. CELF4 regulates translation and local abundance of a vast set of mRNAs, including genes associated with regulation of synaptic function. PLoS Genet. 8, e1003067 (2012)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR79" id="ref-link-section-d6323361e9610">79</a></sup>. (2) CHD8 target genes, defined as genes with promoter or enhancer regions that overlap with CHD8 binding peaks in human neural stem cells or midfetal brain in ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 36" title="Cotney, J. et al. The autism-associated chromatin modifier CHD8 regulates other autism risk genes during human neurodevelopment. Nat. Commun. 6, 6404 (2015)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR36" id="ref-link-section-d6323361e9614">36</a></sup>. (3) FMRP target genes<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 35" title="Darnell, J. C. et al. FMRP stalls ribosomal translocation on mRNAs linked to synaptic function and autism. Cell 146, 247–261 (2011)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR35" id="ref-link-section-d6323361e9618">35</a></sup> with a false discovery rate (FDR) &lt; 0.1 that mapped them to orthologous human genes (Mouse Genome Informatics<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 80" title="Eppig, J. T. Mouse Genome Informatics (MGI) Resource: genetic, genomic, and biological knowledgebase for the laboratory mouse. ILAR J. 58, 17–41 (2017)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR80" id="ref-link-section-d6323361e9622">80</a></sup>, 2018-07). (4) RBFOX2 target genes<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 34" title="Weyn-Vanhentenryck, S. M. et al. HITS-CLIP and integrative modeling define the Rbfox splicing-regulatory network linked to brain development and autism. Cell Rep. 6, 1139–1152 (2014)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR34" id="ref-link-section-d6323361e9626">34</a></sup> that have Rbfox2 tag counts ≥ 8. Due to high correlations between RBFOX1 and RBFOX3, targeted genes by the two RNA binding proteins were merged in one gene set and selected to have total Rbfox1 and Rbfox3 tag counts &gt; 24.</p><h4 id="Sec21">Autism gene predictions</h4><p>(1) ForecASD is an ensemble classifier that integrates brain gene expression, heterogeneous network data and previous gene-level predictors of autism association to yield a single prediction score<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 37" title="Brueggeman, L., Koomar, T. &amp; Michaelson, J. J. Forecasting risk gene discovery in autism with machine learning and genome-scale data. Sci. Rep. 10, 4569 (2020)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR37" id="ref-link-section-d6323361e9639">37</a></sup>. We created two sets of genes with forecASD prediction score greater than 0.4 or 0.5. (2) A-risk is a classifier that uses a gradient boosting tree to predict autism candidate genes using cell-type specific expression signatures in the fetal brain<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 38" title="Chen, S. et al. Dissecting autism genetic risk using single-cell RNA-seq data. Preprint at 
                https://www.bioRxiv.org/content/10.1101/2020.06.15.153031v1
                
               (2020)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR38" id="ref-link-section-d6323361e9643">38</a></sup>. We created three sets of genes with prediction score greater than 0.4, 0.5 or 0.6.</p><h4 id="Sec22">Genetic evidence</h4><p>(1) Genes enriched with DNVs in ASD, with nominal statistical evidence (<i>P</i> &lt; 0.01 or <i>P</i> &lt; 0.05 by DeNovoWEST) in the SPARK discovery cohort of 16,877 trios. (2) Genes enriched with DNVs in other NDDs (Supplementary Data <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41588-022-01148-2#MOESM7">3</a>), with nominal statistical evidence in 31,058 NDDs<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 11" title="Kaplanis, J. et al. Evidence for 28 genetic disorders discovered by combining healthcare and research data. Nature 586, 757–762 (2020)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR11" id="ref-link-section-d6323361e9664">11</a></sup>. (3) Genes implicated in schizophrenia, with nominal statistical evidence (<i>P</i> &lt; 0.05)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 48" title="Singh, T. et al. Rare coding variants in ten genes confer substantial risk for schizophrenia. Nature 604, 509–516 (2022)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR48" id="ref-link-section-d6323361e9672">48</a></sup>.</p><h4 id="Sec23">Genetic constraint</h4><p>Four constraint gene sets are defined by genes in top 10% of the gnomAD LOEUF, 10–20% the gnomAD LOEUF and genes with selection coefficient for heterozygous protein-truncating variants estimated from ExAC data (sHet) ≥ 0.25 and ≥ 0.2.</p><h3 id="Sec24">Archetypal analysis</h3><p>Archetypal analysis<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 81" title="Cutler, A. &amp; Breiman, L. Archetypal analysis. Technometrics 36, 338–347 (1994)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR81" id="ref-link-section-d6323361e9693">81</a></sup> is an unsupervised learning approach that has similarities to other dimensionality reduction and clustering approaches. An important distinction of archetypal analysis from comparable approaches is that it seeks a set of <i>k</i> archetypes, which are points along the convex hull of the data, from which all data points may be expressed as a mixture. The output of archetypal analysis is an <i>N</i> by <i>k</i> matrix, α, of [0,1] coefficients that represent the contribution of each archetype to each data point. Whereas cluster centroids are embedded within the interior of a cluster, archetypes are, by design, at the extremes of the data (that is, along the convex hull). In practice, this ensures that the archetypes are appreciably distinct from one another, which is often not the case for cluster centroids. Here, we use archetypal analysis as a means to organize putative ASD risk genes in a space that has meaning from a mechanistic (STRING) and phenotypic (HPO) standpoint. Genes of particular interest to this study can then be considered and interpreted in the context of the archetypes that define this space. STRING (v11)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 52" title="Szklarczyk, D. et al. STRING v11: protein–protein association networks with increased coverage, supporting functional discovery in genome-wide experimental datasets. Nucleic Acids Res. 47, D607–D613 (2019)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR52" id="ref-link-section-d6323361e9706">52</a></sup> clusters and HPO<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 53" title="Köhler, S. et al. The Human Phenotype Ontology in 2021. Nucleic Acids Res. 49, D1207–D1217 (2021)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR53" id="ref-link-section-d6323361e9711">53</a></sup> terms were formatted as gene-by-term binary matrices. The working gene list was taken as the union of forecASD top decile genes and the 62 autism-associated genes from this study (total of 1,776 genes). A total of 583 genes from this set had annotations in both STRING and HPO, and using these genes, a canonical correlation analysis (CCA) was carried out using the RGCCA package for R (<a href="https://cran.r-project.org/web/packages/RGCCA/index.html">https://cran.r-project.org/web/packages/RGCCA/index.html</a>) using five components and sparsity parameter c1 set to 0.8 for both the HPO and STRING matrices. Component scores for all 1,776 genes were calculated using the STRING cluster annotations and the corresponding coefficients from the CCA and was used as input for archetypal analysis<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 81" title="Cutler, A. &amp; Breiman, L. Archetypal analysis. Technometrics 36, 338–347 (1994)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR81" id="ref-link-section-d6323361e9722">81</a></sup>, with the optimal <i>k</i> (number of archetypes) selected using the elbow plot heuristic<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 82" title="Thorndike, R. L. Who belongs in the family? Psychometrika 18, 267–276 (1953)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR82" id="ref-link-section-d6323361e9729">82</a></sup> and the residual sums of squares (RSS) plotted as a function of <i>k</i>. We displayed the archetypal embedding using the simplexplot() function of the archetypes R package. Genes were identified as ‘archetypal’ if their top archetype coefficient was &gt;2× the next highest archetypal coefficient. Those genes that did not fulfill this criterion were classified as ‘mixed’, whereas those that did were assigned to their maximally scoring archetype. We applied the following heuristic to guide the naming of each archetype: each MSigDB<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 83" title="Subramanian, A. et al. Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles. Proc. Natl Acad. Sci. 102, 15545–15550 (2005)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR83" id="ref-link-section-d6323361e9737">83</a></sup> gene set (v7.4) that has two or more autism genes (<i>n</i> = 62 as described above) annotated is tested for association with the archetypal coefficients (A1–A6, simultaneously) in a quasibinomial generalized linear model (GLM) (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41588-022-01148-2#MOESM4">15</a>). The one-sided association <i>P</i> value (that is, positive association) for each archetype is reported in Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41588-022-01148-2#MOESM4">12</a> (top 20 for each archetype). To name each archetype, the top 20 gene sets with the strongest associations with that archetype were considered; this leads to the following naming conventions: A1, neurotransmission; A2, chromatin modification; A3, RNA processing; A4, vesicle-mediated transport; A5, MAPK signaling and migration; and A6, cytoskeleton and mitosis (see Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41588-022-01148-2#MOESM4">15</a> and Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41588-022-01148-2#Fig7">7</a>). Representative genes for each archetype were chosen from among the list of 62 risk genes identified in this study, using the top six genes for each archetype (note that these genes do not necessarily fulfill the ‘archetypal’ criterion described above, but are simply the top six of the 62 for each archetype).</p><h3 id="Sec25">Statistics and reproducibility</h3><p>No statistical method was used to predetermine sample size, and no data were excluded from the analyses.</p><h3 id="Sec26">Reporting summary</h3><p>Further information on research design is available in the <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41588-022-01148-2#MOESM2">Nature Research Reporting Summary</a> linked to this article.</p></div></div></section>
            

            <section data-title="Data availability"><div id="data-availability-section"><h2 id="data-availability">Data availability</h2><div id="data-availability-content">
            
            <p>In order to abide by the informed consents that individuals with autism and their family members signed when agreeing to participate in a SFARI cohort (SSC and SPARK), researchers must be approved by SFARI Base (<a href="https://base.sfari.org">https://base.sfari.org</a>). To access to SPARK or SFARI data, researchers should</p>
            <p>(1) Obtain a SFARI Base account at <a href="https://base.sfari.org">https://base.sfari.org</a>, which will require affiliating with an institution. Currently, there are 271 institutions around the world that have signed SFARI’s Researcher Distribution Agreement (RDA), and any researcher affiliated with those institutions can apply for SFARI Base access.</p>
            <p>(2) Review the institute’s executed RDA. The standard RDA is available at <a href="https://s3.amazonaws.com/sf-web-assets-prod/wp-content/uploads/sites/2/2021/06/15165956/SFARI_RDA.pdf">https://s3.amazonaws.com/sf-web-assets-prod/wp-content/uploads/sites/2/2021/06/15165956/SFARI_RDA.pdf</a></p>
            <p>(3) Create a SFARI Base project, which includes a title, abstract and an IRB approval or exemption document.</p>
            <p>(4) Create a SFARI Base request. All requests are processed in a timely manner.</p>
            <p>The SPARK data is accessible as follows:</p>
            <p>SFARI_SPARK_iWES includes exome and genotyping data on 70,487 participants, including all people analyzed in this paper plus an additional 11,282 participants.</p>
            <p>SFARI_SPARK_WGS_1 includes whole genome data from 2,629 individuals from 645 families with at least one person with autism.</p>
            <p>SFARI_SPARK_WGS_2 includes whole genome data from 2,365 individuals from 587 families with at least one person with autism.</p>
            <p>SFARI_SPARK_WGS_3 includes whole genome data from 2,871 individuals from 803 families with at least one person with autism.</p>
            <p>SSC_WES_3 is whole exome data on the Simons Simplex Collection (SSC) as analyzed and reported in ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 19" title="Krumm, N. et al. Excess of rare, inherited truncating mutations in autism. Nat. Genet. 47, 582–588 (2015)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR19" id="ref-link-section-d6323361e9936">19</a></sup>.</p>
            <p>SFARI_SSC_WGS_pilot contains genomes of 40 families with autism.</p>
            <p>SFARI_SSC_WGS_1 and SFARI_SSC_ WGS_2 contain WGS of the SSC.</p>
            <p>SSC Dataset contains phenotypic information on 2,644 simplex autism families.</p>
            <p>SPARK Phenotype Dataset V7 is the current phenotypic dataset on 290,502 SPARK participants, including 111,720 participants with autism.</p>
          </div></div></section><section data-title="Code availability"><div id="code-availability-section"><h2 id="code-availability">Code availability</h2><div id="code-availability-content">
            
            <p>All software used in this study is publicly available. The code for major figures and analysis can be found at <a href="https://github.com/ShenLab/SPARK_Analysis_V1.git">https://github.com/ShenLab/SPARK_Analysis_V1.git</a> or <a href="https://doi.org/10.5281/zenodo.6646871">https://doi.org/10.5281/zenodo.6646871</a>.</p>
          </div></div></section><div id="MagazineFulltextArticleBodySuffix"><section aria-labelledby="Bib1" data-title="References"><div id="Bib1-section"><h2 id="Bib1">References</h2><div id="Bib1-content"><div data-container-section="references"><ol data-track-category="outbound reference"><li data-counter="1."><p id="ref-CR1">Lord, C. et al. Autism spectrum disorder. <i>Nat. Rev. Dis. Primers</i> <b>6</b>, 5 (2020).</p><p><a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31949163" aria-label="PubMed reference 1">PubMed</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8900942" aria-label="PubMed Central reference 1">PubMed Central</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41572-019-0138-4" aria-label="Article reference 1">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 1" href="http://scholar.google.com/scholar_lookup?&amp;title=Autism%20spectrum%20disorder&amp;journal=Nat.%20Rev.%20Dis.%20Primers&amp;volume=6&amp;publication_year=2020&amp;author=Lord%2CC">
                    Google Scholar</a> 
                </p></li><li data-counter="2."><p id="ref-CR2">Iossifov, I. et al. The contribution of de novo coding mutations to autism spectrum disorder. <i>Nature</i> <b>515</b>, 216–221 (2014).</p><p><a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXhvVemtrnN" aria-label="CAS reference 2">CAS</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25363768" aria-label="PubMed reference 2">PubMed</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4313871" aria-label="PubMed Central reference 2">PubMed Central</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="article reference" href="https://doi.org/10.1038%2Fnature13908" aria-label="Article reference 2">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 2" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20contribution%20of%20de%20novo%20coding%20mutations%20to%20autism%20spectrum%20disorder&amp;journal=Nature&amp;volume=515&amp;pages=216-221&amp;publication_year=2014&amp;author=Iossifov%2CI">
                    Google Scholar</a> 
                </p></li><li data-counter="3."><p id="ref-CR3">De Rubeis, S. et al. Synaptic, transcriptional and chromatin genes disrupted in autism. <i>Nature</i> <b>515</b>, 209–215 (2014).</p><p><a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25363760" aria-label="PubMed reference 3">PubMed</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4402723" aria-label="PubMed Central reference 3">PubMed Central</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="article reference" href="https://doi.org/10.1038%2Fnature13772" aria-label="Article reference 3">Article</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXhvVemtrnJ" aria-label="CAS reference 3">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 3" href="http://scholar.google.com/scholar_lookup?&amp;title=Synaptic%2C%20transcriptional%20and%20chromatin%20genes%20disrupted%20in%20autism&amp;journal=Nature&amp;volume=515&amp;pages=209-215&amp;publication_year=2014&amp;author=Rubeis%2CS">
                    Google Scholar</a> 
                </p></li><li data-counter="4."><p id="ref-CR4">O’Roak, B. J. et al. Recurrent de novo mutations implicate novel genes underlying simplex autism risk. <i>Nat. Commun.</i> <b>5</b>, 5595 (2014).</p><p><a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25418537" aria-label="PubMed reference 4">PubMed</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="article reference" href="https://doi.org/10.1038%2Fncomms6595" aria-label="Article reference 4">Article</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXksVCitrc%3D" aria-label="CAS reference 4">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 4" href="http://scholar.google.com/scholar_lookup?&amp;title=Recurrent%20de%20novo%20mutations%20implicate%20novel%20genes%20underlying%20simplex%20autism%20risk&amp;journal=Nat.%20Commun.&amp;volume=5&amp;publication_year=2014&amp;author=O%E2%80%99Roak%2CBJ">
                    Google Scholar</a> 
                </p></li><li data-counter="5."><p id="ref-CR5">Yuen, R. K. C. et al. Whole-genome sequencing of quartet families with autism spectrum disorder. <i>Nat. Med.</i> <b>21</b>, 185–191 (2015).</p><p><a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXhsVehsbc%3D" aria-label="CAS reference 5">CAS</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25621899" aria-label="PubMed reference 5">PubMed</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="article reference" href="https://doi.org/10.1038%2Fnm.3792" aria-label="Article reference 5">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 5" href="http://scholar.google.com/scholar_lookup?&amp;title=Whole-genome%20sequencing%20of%20quartet%20families%20with%20autism%20spectrum%20disorder&amp;journal=Nat.%20Med.&amp;volume=21&amp;pages=185-191&amp;publication_year=2015&amp;author=Yuen%2CRKC">
                    Google Scholar</a> 
                </p></li><li data-counter="6."><p id="ref-CR6">Yuen, R. K. C. et al. Whole genome sequencing resource identifies 18 new candidate genes for autism spectrum disorder. <i>Nat. Neurosci.</i> <b>20</b>, 602–611 (2017).</p><p><a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXltVWntLo%3D" aria-label="CAS reference 6">CAS</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5501701" aria-label="PubMed Central reference 6">PubMed Central</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="article reference" href="https://doi.org/10.1038%2Fnn.4524" aria-label="Article reference 6">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 6" href="http://scholar.google.com/scholar_lookup?&amp;title=Whole%20genome%20sequencing%20resource%20identifies%2018%20new%20candidate%20genes%20for%20autism%20spectrum%20disorder&amp;journal=Nat.%20Neurosci.&amp;volume=20&amp;pages=602-611&amp;publication_year=2017&amp;author=Yuen%2CRKC">
                    Google Scholar</a> 
                </p></li><li data-counter="7."><p id="ref-CR7">Feliciano, P. et al. Exome sequencing of 457 autism families recruited online provides evidence for autism risk genes. <i>npj Genomic Med</i> <b>4</b>, 19 (2019).</p><p><a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41525-019-0093-8" aria-label="Article reference 7">Article</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhs1ektrrP" aria-label="CAS reference 7">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 7" href="http://scholar.google.com/scholar_lookup?&amp;title=Exome%20sequencing%20of%20457%20autism%20families%20recruited%20online%20provides%20evidence%20for%20autism%20risk%20genes&amp;journal=npj%20Genomic%20Med&amp;volume=4&amp;publication_year=2019&amp;author=Feliciano%2CP">
                    Google Scholar</a> 
                </p></li><li data-counter="8."><p id="ref-CR8">Satterstrom, F. K. et al. Large-scale exome sequencing study implicates both developmental and functional changes in the neurobiology of autism. <i>Cell</i> <b>180</b>, 568–584.e23 (2020).</p><p><a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhvVShsr8%3D" aria-label="CAS reference 8">CAS</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31981491" aria-label="PubMed reference 8">PubMed</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7250485" aria-label="PubMed Central reference 8">PubMed Central</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.cell.2019.12.036" aria-label="Article reference 8">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 8" href="http://scholar.google.com/scholar_lookup?&amp;title=Large-scale%20exome%20sequencing%20study%20implicates%20both%20developmental%20and%20functional%20changes%20in%20the%20neurobiology%20of%20autism&amp;journal=Cell&amp;volume=180&amp;pages=568-584.e23&amp;publication_year=2020&amp;author=Satterstrom%2CFK">
                    Google Scholar</a> 
                </p></li><li data-counter="9."><p id="ref-CR9">The Deciphering Developmental Disorders Study. Large-scale discovery of novel genetic causes of developmental disorders. <i>Nature</i> <b>519</b>, 223–228 (2015).</p><p><a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="article reference" href="https://doi.org/10.1038%2Fnature14135" aria-label="Article reference 9">Article</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXjt1Sntbk%3D" aria-label="CAS reference 9">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 9" href="http://scholar.google.com/scholar_lookup?&amp;title=Large-scale%20discovery%20of%20novel%20genetic%20causes%20of%20developmental%20disorders&amp;journal=Nature&amp;volume=519&amp;pages=223-228&amp;publication_year=2015">
                    Google Scholar</a> 
                </p></li><li data-counter="10."><p id="ref-CR10">The Deciphering Developmental Disorders Study. Prevalence and architecture of de novo mutations in developmental disorders. <i>Nature</i> <b>542</b>, 433–438 (2017).</p><p><a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6016744" aria-label="PubMed Central reference 10">PubMed Central</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="article reference" href="https://doi.org/10.1038%2Fnature21062" aria-label="Article reference 10">Article</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhsFGlt7c%3D" aria-label="CAS reference 10">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 10" href="http://scholar.google.com/scholar_lookup?&amp;title=Prevalence%20and%20architecture%20of%20de%20novo%20mutations%20in%20developmental%20disorders&amp;journal=Nature&amp;volume=542&amp;pages=433-438&amp;publication_year=2017">
                    Google Scholar</a> 
                </p></li><li data-counter="11."><p id="ref-CR11">Kaplanis, J. et al. Evidence for 28 genetic disorders discovered by combining healthcare and research data. <i>Nature</i> <b>586</b>, 757–762 (2020).</p><p><a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXitVyiu7bP" aria-label="CAS reference 11">CAS</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33057194" aria-label="PubMed reference 11">PubMed</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7116826" aria-label="PubMed Central reference 11">PubMed Central</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41586-020-2832-5" aria-label="Article reference 11">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 11" href="http://scholar.google.com/scholar_lookup?&amp;title=Evidence%20for%2028%20genetic%20disorders%20discovered%20by%20combining%20healthcare%20and%20research%20data&amp;journal=Nature&amp;volume=586&amp;pages=757-762&amp;publication_year=2020&amp;author=Kaplanis%2CJ">
                    Google Scholar</a> 
                </p></li><li data-counter="12."><p id="ref-CR12">He, X. et al. Integrated model of de novo and inherited genetic variants yields greater power to identify risk genes. <i>PLoS Genet.</i> <b>9</b>, e1003671 (2013).</p><p><a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXhsVagtLjP" aria-label="CAS reference 12">CAS</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23966865" aria-label="PubMed reference 12">PubMed</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3744441" aria-label="PubMed Central reference 12">PubMed Central</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="article reference" href="https://doi.org/10.1371%2Fjournal.pgen.1003671" aria-label="Article reference 12">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 12" href="http://scholar.google.com/scholar_lookup?&amp;title=Integrated%20model%20of%20de%20novo%20and%20inherited%20genetic%20variants%20yields%20greater%20power%20to%20identify%20risk%20genes&amp;journal=PLoS%20Genet.&amp;volume=9&amp;publication_year=2013&amp;author=He%2CX">
                    Google Scholar</a> 
                </p></li><li data-counter="13."><p id="ref-CR13">Nguyen, H. T. et al. Integrated Bayesian analysis of rare exonic variants to identify risk genes for schizophrenia and neurodevelopmental disorders. <i>Genome Med.</i> <b>9</b>, 114 (2017).</p><p><a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29262854" aria-label="PubMed reference 13">PubMed</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5738153" aria-label="PubMed Central reference 13">PubMed Central</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="article reference" href="https://doi.org/10.1186%2Fs13073-017-0497-y" aria-label="Article reference 13">Article</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXit1Wrs7nP" aria-label="CAS reference 13">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 13" href="http://scholar.google.com/scholar_lookup?&amp;title=Integrated%20Bayesian%20analysis%20of%20rare%20exonic%20variants%20to%20identify%20risk%20genes%20for%20schizophrenia%20and%20neurodevelopmental%20disorders&amp;journal=Genome%20Med.&amp;volume=9&amp;publication_year=2017&amp;author=Nguyen%2CHT">
                    Google Scholar</a> 
                </p></li><li data-counter="14."><p id="ref-CR14">Gaugler, T. et al. Most genetic risk for autism resides with common variation. <i>Nat. Genet.</i> <b>46</b>, 881–885 (2014).</p><p><a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXhtFygs7fK" aria-label="CAS reference 14">CAS</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25038753" aria-label="PubMed reference 14">PubMed</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4137411" aria-label="PubMed Central reference 14">PubMed Central</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="article reference" href="https://doi.org/10.1038%2Fng.3039" aria-label="Article reference 14">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 14" href="http://scholar.google.com/scholar_lookup?&amp;title=Most%20genetic%20risk%20for%20autism%20resides%20with%20common%20variation&amp;journal=Nat.%20Genet.&amp;volume=46&amp;pages=881-885&amp;publication_year=2014&amp;author=Gaugler%2CT">
                    Google Scholar</a> 
                </p></li><li data-counter="15."><p id="ref-CR15">Sandin, S. et al. The familial risk of autism. <i>JAMA</i> <b>311</b>, 1770–1777 (2014).</p><p><a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXhsFems77M" aria-label="CAS reference 15">CAS</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24794370" aria-label="PubMed reference 15">PubMed</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4381277" aria-label="PubMed Central reference 15">PubMed Central</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="article reference" href="https://doi.org/10.1001%2Fjama.2014.4144" aria-label="Article reference 15">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 15" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20familial%20risk%20of%20autism&amp;journal=JAMA&amp;volume=311&amp;pages=1770-1777&amp;publication_year=2014&amp;author=Sandin%2CS">
                    Google Scholar</a> 
                </p></li><li data-counter="16."><p id="ref-CR16">Sandin, S. et al. The heritability of autism spectrum disorder. <i>JAMA</i> <b>318</b>, 1182–1184 (2017).</p><p><a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28973605" aria-label="PubMed reference 16">PubMed</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5818813" aria-label="PubMed Central reference 16">PubMed Central</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="article reference" href="https://doi.org/10.1001%2Fjama.2017.12141" aria-label="Article reference 16">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 16" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20heritability%20of%20autism%20spectrum%20disorder&amp;journal=JAMA&amp;volume=318&amp;pages=1182-1184&amp;publication_year=2017&amp;author=Sandin%2CS">
                    Google Scholar</a> 
                </p></li><li data-counter="17."><p id="ref-CR17">Grove, J. et al. Identification of common genetic risk variants for autism spectrum disorder. <i>Nat. Genet.</i> <b>51</b>, 431–444 (2019).</p><p><a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXmsVyqs70%3D" aria-label="CAS reference 17">CAS</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30804558" aria-label="PubMed reference 17">PubMed</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6454898" aria-label="PubMed Central reference 17">PubMed Central</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41588-019-0344-8" aria-label="Article reference 17">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 17" href="http://scholar.google.com/scholar_lookup?&amp;title=Identification%20of%20common%20genetic%20risk%20variants%20for%20autism%20spectrum%20disorder&amp;journal=Nat.%20Genet.&amp;volume=51&amp;pages=431-444&amp;publication_year=2019&amp;author=Grove%2CJ">
                    Google Scholar</a> 
                </p></li><li data-counter="18."><p id="ref-CR18">Lek, M. et al. Analysis of protein-coding genetic variation in 60,706 humans. <i>Nature</i> <b>536</b>, 285–291 (2016).</p><p><a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XhtlOnsbbP" aria-label="CAS reference 18">CAS</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27535533" aria-label="PubMed reference 18">PubMed</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5018207" aria-label="PubMed Central reference 18">PubMed Central</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="article reference" href="https://doi.org/10.1038%2Fnature19057" aria-label="Article reference 18">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 18" href="http://scholar.google.com/scholar_lookup?&amp;title=Analysis%20of%20protein-coding%20genetic%20variation%20in%2060%2C706%20humans&amp;journal=Nature&amp;volume=536&amp;pages=285-291&amp;publication_year=2016&amp;author=Lek%2CM">
                    Google Scholar</a> 
                </p></li><li data-counter="19."><p id="ref-CR19">Krumm, N. et al. Excess of rare, inherited truncating mutations in autism. <i>Nat. Genet.</i> <b>47</b>, 582–588 (2015).</p><p><a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXht1Wltb7N" aria-label="CAS reference 19">CAS</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25961944" aria-label="PubMed reference 19">PubMed</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4449286" aria-label="PubMed Central reference 19">PubMed Central</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="article reference" href="https://doi.org/10.1038%2Fng.3303" aria-label="Article reference 19">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 19" href="http://scholar.google.com/scholar_lookup?&amp;title=Excess%20of%20rare%2C%20inherited%20truncating%20mutations%20in%20autism&amp;journal=Nat.%20Genet.&amp;volume=47&amp;pages=582-588&amp;publication_year=2015&amp;author=Krumm%2CN">
                    Google Scholar</a> 
                </p></li><li data-counter="20."><p id="ref-CR20">Kosmicki, J. A. et al. Refining the role of de novo protein-truncating variants in neurodevelopmental disorders by using population reference samples. <i>Nat. Genet.</i> <b>49</b>, 504–510 (2017).</p><p><a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXitlGiu7Y%3D" aria-label="CAS reference 20">CAS</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28191890" aria-label="PubMed reference 20">PubMed</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5496244" aria-label="PubMed Central reference 20">PubMed Central</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="article reference" href="https://doi.org/10.1038%2Fng.3789" aria-label="Article reference 20">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 20" href="http://scholar.google.com/scholar_lookup?&amp;title=Refining%20the%20role%20of%20de%20novo%20protein-truncating%20variants%20in%20neurodevelopmental%20disorders%20by%20using%20population%20reference%20samples&amp;journal=Nat.%20Genet.&amp;volume=49&amp;pages=504-510&amp;publication_year=2017&amp;author=Kosmicki%2CJA">
                    Google Scholar</a> 
                </p></li><li data-counter="21."><p id="ref-CR21">Ruzzo, E. K. et al. Inherited and de novo genetic risk for autism impacts shared networks. <i>Cell</i> <b>178</b>, 850–866.e26 (2019).</p><p><a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhsFKktrrL" aria-label="CAS reference 21">CAS</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31398340" aria-label="PubMed reference 21">PubMed</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7102900" aria-label="PubMed Central reference 21">PubMed Central</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.cell.2019.07.015" aria-label="Article reference 21">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 21" href="http://scholar.google.com/scholar_lookup?&amp;title=Inherited%20and%20de%20novo%20genetic%20risk%20for%20autism%20impacts%20shared%20networks&amp;journal=Cell&amp;volume=178&amp;pages=850-866.e26&amp;publication_year=2019&amp;author=Ruzzo%2CEK">
                    Google Scholar</a> 
                </p></li><li data-counter="22."><p id="ref-CR22">Wilfert, A. B. et al. Recent ultra-rare inherited variants implicate new autism candidate risk genes. <i>Nat. Genet.</i> <b>53</b>, 1125–1134 (2021).</p><p><a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhs1Cksb7O" aria-label="CAS reference 22">CAS</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34312540" aria-label="PubMed reference 22">PubMed</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8459613" aria-label="PubMed Central reference 22">PubMed Central</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41588-021-00899-8" aria-label="Article reference 22">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 22" href="http://scholar.google.com/scholar_lookup?&amp;title=Recent%20ultra-rare%20inherited%20variants%20implicate%20new%20autism%20candidate%20risk%20genes&amp;journal=Nat.%20Genet.&amp;volume=53&amp;pages=1125-1134&amp;publication_year=2021&amp;author=Wilfert%2CAB">
                    Google Scholar</a> 
                </p></li><li data-counter="23."><p id="ref-CR23">The SPARK Consortium. SPARK: a US cohort of 50,000 families to accelerate autism research. <i>Neuron</i> <b>97</b>, 488–493 (2018).</p><p><a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7444276" aria-label="PubMed Central reference 23">PubMed Central</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuron.2018.01.015" aria-label="Article reference 23">Article</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXisVCmtLY%3D" aria-label="CAS reference 23">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 23" href="http://scholar.google.com/scholar_lookup?&amp;title=SPARK%3A%20a%20US%20cohort%20of%2050%2C000%20families%20to%20accelerate%20autism%20research&amp;journal=Neuron&amp;volume=97&amp;pages=488-493&amp;publication_year=2018">
                    Google Scholar</a> 
                </p></li><li data-counter="24."><p id="ref-CR24">MacArthur, D. G. &amp; Tyler-Smith, C. Loss-of-function variants in the genomes of healthy humans. <i>Hum. Mol. Genet.</i> <b>19</b>, R125–R130 (2010).</p><p><a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXhsFSnsrbL" aria-label="CAS reference 24">CAS</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20805107" aria-label="PubMed reference 24">PubMed</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2953739" aria-label="PubMed Central reference 24">PubMed Central</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="article reference" href="https://doi.org/10.1093%2Fhmg%2Fddq365" aria-label="Article reference 24">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 24" href="http://scholar.google.com/scholar_lookup?&amp;title=Loss-of-function%20variants%20in%20the%20genomes%20of%20healthy%20humans&amp;journal=Hum.%20Mol.%20Genet.&amp;volume=19&amp;pages=R125-R130&amp;publication_year=2010&amp;author=MacArthur%2CDG&amp;author=Tyler-Smith%2CC">
                    Google Scholar</a> 
                </p></li><li data-counter="25."><p id="ref-CR25">MacArthur, D. G. et al. A systematic survey of loss-of-function variants in human protein-coding genes. <i>Science</i> <b>335</b>, 823–828 (2012).</p><p><a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XitFGqur0%3D" aria-label="CAS reference 25">CAS</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22344438" aria-label="PubMed reference 25">PubMed</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3299548" aria-label="PubMed Central reference 25">PubMed Central</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="article reference" href="https://doi.org/10.1126%2Fscience.1215040" aria-label="Article reference 25">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 25" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20systematic%20survey%20of%20loss-of-function%20variants%20in%20human%20protein-coding%20genes&amp;journal=Science&amp;volume=335&amp;pages=823-828&amp;publication_year=2012&amp;author=MacArthur%2CDG">
                    Google Scholar</a> 
                </p></li><li data-counter="26."><p id="ref-CR26">Karczewski, K. J. et al. The mutational constraint spectrum quantified from variation in 141,456 humans. <i>Nature</i> <b>581</b>, 434–443 (2020).</p><p><a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhtVanu7jF" aria-label="CAS reference 26">CAS</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32461654" aria-label="PubMed reference 26">PubMed</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7334197" aria-label="PubMed Central reference 26">PubMed Central</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41586-020-2308-7" aria-label="Article reference 26">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 26" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20mutational%20constraint%20spectrum%20quantified%20from%20variation%20in%20141%2C456%20humans&amp;journal=Nature&amp;volume=581&amp;pages=434-443&amp;publication_year=2020&amp;author=Karczewski%2CKJ">
                    Google Scholar</a> 
                </p></li><li data-counter="27."><p id="ref-CR27">Cummings, B. B. et al. Transcript expression-aware annotation improves rare variant interpretation. <i>Nature</i> <b>581</b>, 452–458 (2020).</p><p><a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhtVanu7nM" aria-label="CAS reference 27">CAS</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32461655" aria-label="PubMed reference 27">PubMed</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7334198" aria-label="PubMed Central reference 27">PubMed Central</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41586-020-2329-2" aria-label="Article reference 27">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 27" href="http://scholar.google.com/scholar_lookup?&amp;title=Transcript%20expression-aware%20annotation%20improves%20rare%20variant%20interpretation&amp;journal=Nature&amp;volume=581&amp;pages=452-458&amp;publication_year=2020&amp;author=Cummings%2CBB">
                    Google Scholar</a> 
                </p></li><li data-counter="28."><p id="ref-CR28">Cassa, C. A. et al. Estimating the selective effects of heterozygous protein-truncating variants from human exome data. <i>Nat. Genet.</i> <b>49</b>, 806–810 (2017).</p><p><a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXltl2ru78%3D" aria-label="CAS reference 28">CAS</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28369035" aria-label="PubMed reference 28">PubMed</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5618255" aria-label="PubMed Central reference 28">PubMed Central</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="article reference" href="https://doi.org/10.1038%2Fng.3831" aria-label="Article reference 28">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 28" href="http://scholar.google.com/scholar_lookup?&amp;title=Estimating%20the%20selective%20effects%20of%20heterozygous%20protein-truncating%20variants%20from%20human%20exome%20data&amp;journal=Nat.%20Genet.&amp;volume=49&amp;pages=806-810&amp;publication_year=2017&amp;author=Cassa%2CCA">
                    Google Scholar</a> 
                </p></li><li data-counter="29."><p id="ref-CR29">Fuller, Z. L., Berg, J. J., Mostafavi, H., Sella, G. &amp; Przeworski, M. Measuring intolerance to mutation in human genetics. <i>Nat. Genet.</i> <b>51</b>, 772–776 (2019).</p><p><a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXoslans70%3D" aria-label="CAS reference 29">CAS</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30962618" aria-label="PubMed reference 29">PubMed</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6615471" aria-label="PubMed Central reference 29">PubMed Central</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41588-019-0383-1" aria-label="Article reference 29">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 29" href="http://scholar.google.com/scholar_lookup?&amp;title=Measuring%20intolerance%20to%20mutation%20in%20human%20genetics&amp;journal=Nat.%20Genet.&amp;volume=51&amp;pages=772-776&amp;publication_year=2019&amp;author=Fuller%2CZL&amp;author=Berg%2CJJ&amp;author=Mostafavi%2CH&amp;author=Sella%2CG&amp;author=Przeworski%2CM">
                    Google Scholar</a> 
                </p></li><li data-counter="30."><p id="ref-CR30">An, J.-Y. et al. Genome-wide de novo risk score implicates promoter variation in autism spectrum disorder. <i>Science</i> <b>362</b>, aat6576 (2018).</p><p><a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXisVyhsLnN" aria-label="CAS reference 30">CAS</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="article reference" href="https://doi.org/10.1126%2Fscience.aat6576" aria-label="Article reference 30">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 30" href="http://scholar.google.com/scholar_lookup?&amp;title=Genome-wide%20de%20novo%20risk%20score%20implicates%20promoter%20variation%20in%20autism%20spectrum%20disorder&amp;journal=Science&amp;volume=362&amp;publication_year=2018&amp;author=An%2CJ-Y">
                    Google Scholar</a> 
                </p></li><li data-counter="31."><p id="ref-CR31">Ioannidis, N. M. et al. REVEL: an ensemble method for predicting the pathogenicity of rare missense variants. <i>Am. J. Hum. Genet.</i> <b>99</b>, 877–885 (2016).</p><p><a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XhsFGrurnM" aria-label="CAS reference 31">CAS</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27666373" aria-label="PubMed reference 31">PubMed</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5065685" aria-label="PubMed Central reference 31">PubMed Central</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.ajhg.2016.08.016" aria-label="Article reference 31">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 31" href="http://scholar.google.com/scholar_lookup?&amp;title=REVEL%3A%20an%20ensemble%20method%20for%20predicting%20the%20pathogenicity%20of%20rare%20missense%20variants&amp;journal=Am.%20J.%20Hum.%20Genet.&amp;volume=99&amp;pages=877-885&amp;publication_year=2016&amp;author=Ioannidis%2CNM">
                    Google Scholar</a> 
                </p></li><li data-counter="32."><p id="ref-CR32">Fromer, M. et al. De novo mutations in schizophrenia implicate synaptic networks. <i>Nature</i> <b>506</b>, 179–184 (2014).</p><p><a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXisVamurY%3D" aria-label="CAS reference 32">CAS</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24463507" aria-label="PubMed reference 32">PubMed</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4237002" aria-label="PubMed Central reference 32">PubMed Central</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="article reference" href="https://doi.org/10.1038%2Fnature12929" aria-label="Article reference 32">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 32" href="http://scholar.google.com/scholar_lookup?&amp;title=De%20novo%20mutations%20in%20schizophrenia%20implicate%20synaptic%20networks&amp;journal=Nature&amp;volume=506&amp;pages=179-184&amp;publication_year=2014&amp;author=Fromer%2CM">
                    Google Scholar</a> 
                </p></li><li data-counter="33."><p id="ref-CR33">Parikshak, N. N. et al. Integrative functional genomic analyses implicate specific molecular pathways and circuits in autism. <i>Cell</i> <b>155</b>, 1008–1021 (2013).</p><p><a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXhvFWjt73N" aria-label="CAS reference 33">CAS</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24267887" aria-label="PubMed reference 33">PubMed</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3934107" aria-label="PubMed Central reference 33">PubMed Central</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.cell.2013.10.031" aria-label="Article reference 33">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 33" href="http://scholar.google.com/scholar_lookup?&amp;title=Integrative%20functional%20genomic%20analyses%20implicate%20specific%20molecular%20pathways%20and%20circuits%20in%20autism&amp;journal=Cell&amp;volume=155&amp;pages=1008-1021&amp;publication_year=2013&amp;author=Parikshak%2CNN">
                    Google Scholar</a> 
                </p></li><li data-counter="34."><p id="ref-CR34">Weyn-Vanhentenryck, S. M. et al. HITS-CLIP and integrative modeling define the Rbfox splicing-regulatory network linked to brain development and autism. <i>Cell Rep.</i> <b>6</b>, 1139–1152 (2014).</p><p><a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXjvFWitrY%3D" aria-label="CAS reference 34">CAS</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24613350" aria-label="PubMed reference 34">PubMed</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3992522" aria-label="PubMed Central reference 34">PubMed Central</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.celrep.2014.02.005" aria-label="Article reference 34">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 34" href="http://scholar.google.com/scholar_lookup?&amp;title=HITS-CLIP%20and%20integrative%20modeling%20define%20the%20Rbfox%20splicing-regulatory%20network%20linked%20to%20brain%20development%20and%20autism&amp;journal=Cell%20Rep.&amp;volume=6&amp;pages=1139-1152&amp;publication_year=2014&amp;author=Weyn-Vanhentenryck%2CSM">
                    Google Scholar</a> 
                </p></li><li data-counter="35."><p id="ref-CR35">Darnell, J. C. et al. FMRP stalls ribosomal translocation on mRNAs linked to synaptic function and autism. <i>Cell</i> <b>146</b>, 247–261 (2011).</p><p><a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXptleksbo%3D" aria-label="CAS reference 35">CAS</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21784246" aria-label="PubMed reference 35">PubMed</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3232425" aria-label="PubMed Central reference 35">PubMed Central</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.cell.2011.06.013" aria-label="Article reference 35">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 35" href="http://scholar.google.com/scholar_lookup?&amp;title=FMRP%20stalls%20ribosomal%20translocation%20on%20mRNAs%20linked%20to%20synaptic%20function%20and%20autism&amp;journal=Cell&amp;volume=146&amp;pages=247-261&amp;publication_year=2011&amp;author=Darnell%2CJC">
                    Google Scholar</a> 
                </p></li><li data-counter="36."><p id="ref-CR36">Cotney, J. et al. The autism-associated chromatin modifier CHD8 regulates other autism risk genes during human neurodevelopment. <i>Nat. Commun.</i> <b>6</b>, 6404 (2015).</p><p><a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXosVCrsL4%3D" aria-label="CAS reference 36">CAS</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25752243" aria-label="PubMed reference 36">PubMed</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="article reference" href="https://doi.org/10.1038%2Fncomms7404" aria-label="Article reference 36">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 36" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20autism-associated%20chromatin%20modifier%20CHD8%20regulates%20other%20autism%20risk%20genes%20during%20human%20neurodevelopment&amp;journal=Nat.%20Commun.&amp;volume=6&amp;publication_year=2015&amp;author=Cotney%2CJ">
                    Google Scholar</a> 
                </p></li><li data-counter="37."><p id="ref-CR37">Brueggeman, L., Koomar, T. &amp; Michaelson, J. J. Forecasting risk gene discovery in autism with machine learning and genome-scale data. <i>Sci. Rep.</i> <b>10</b>, 4569 (2020).</p><p><a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXlvFCgtr8%3D" aria-label="CAS reference 37">CAS</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32165711" aria-label="PubMed reference 37">PubMed</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7067874" aria-label="PubMed Central reference 37">PubMed Central</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41598-020-61288-5" aria-label="Article reference 37">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 37" href="http://scholar.google.com/scholar_lookup?&amp;title=Forecasting%20risk%20gene%20discovery%20in%20autism%20with%20machine%20learning%20and%20genome-scale%20data&amp;journal=Sci.%20Rep.&amp;volume=10&amp;publication_year=2020&amp;author=Brueggeman%2CL&amp;author=Koomar%2CT&amp;author=Michaelson%2CJJ">
                    Google Scholar</a> 
                </p></li><li data-counter="38."><p id="ref-CR38">Chen, S. et al. Dissecting autism genetic risk using single-cell RNA-seq data. Preprint at <a href="https://www.biorxiv.org/content/10.1101/2020.06.15.153031v1">https://www.bioRxiv.org/content/10.1101/2020.06.15.153031v1</a> (2020).</p></li><li data-counter="39."><p id="ref-CR39">Ewens, W. J. &amp; Spielman, R. S. The transmission/disequilibrium test: history, subdivision, and admixture. <i>Am. J. Hum. Genet.</i> <b>57</b>, 455–464 (1995).</p><p><a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:STN:280:DyaK2Mzpslyhuw%3D%3D" aria-label="CAS reference 39">CAS</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=7668272" aria-label="PubMed reference 39">PubMed</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1801556" aria-label="PubMed Central reference 39">PubMed Central</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="article reference" href="https://doi.org/10.1002%2Fajmg.1320570319" aria-label="Article reference 39">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 39" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20transmission%2Fdisequilibrium%20test%3A%20history%2C%20subdivision%2C%20and%20admixture&amp;journal=Am.%20J.%20Hum.%20Genet.&amp;volume=57&amp;pages=455-464&amp;publication_year=1995&amp;author=Ewens%2CWJ&amp;author=Spielman%2CRS">
                    Google Scholar</a> 
                </p></li><li data-counter="40."><p id="ref-CR40">Fisher, R. A. <i>Statistical Methods for Research Workers</i> 11th edn, rev (Oliver and Boyd, 1925).</p></li><li data-counter="41."><p id="ref-CR41">Zuk, O. et al. Searching for missing heritability: designing rare variant association studies. <i>Proc. Natl Acad. Sci. USA</i> <b>111</b>, E455–E464 (2014).</p><p><a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXhs1Sntbs%3D" aria-label="CAS reference 41">CAS</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24443550" aria-label="PubMed reference 41">PubMed</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3910587" aria-label="PubMed Central reference 41">PubMed Central</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="article reference" href="https://doi.org/10.1073%2Fpnas.1322563111" aria-label="Article reference 41">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 41" href="http://scholar.google.com/scholar_lookup?&amp;title=Searching%20for%20missing%20heritability%3A%20designing%20rare%20variant%20association%20studies&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=111&amp;pages=E455-E464&amp;publication_year=2014&amp;author=Zuk%2CO">
                    Google Scholar</a> 
                </p></li><li data-counter="42."><p id="ref-CR42">Wright, S. Evolution in Mendelian populations. <i>Genetics</i> <b>16</b>, 97–159 (1931).</p><p><a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:STN:280:DC%2BD2s%2FmsVGqsw%3D%3D" aria-label="CAS reference 42">CAS</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17246615" aria-label="PubMed reference 42">PubMed</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1201091" aria-label="PubMed Central reference 42">PubMed Central</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="article reference" href="https://doi.org/10.1093%2Fgenetics%2F16.2.97" aria-label="Article reference 42">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 42" href="http://scholar.google.com/scholar_lookup?&amp;title=Evolution%20in%20Mendelian%20populations&amp;journal=Genetics&amp;volume=16&amp;pages=97-159&amp;publication_year=1931&amp;author=Wright%2CS">
                    Google Scholar</a> 
                </p></li><li data-counter="43."><p id="ref-CR43">Skene, N. G. &amp; Grant, S. G. N. Identification of vulnerable cell types in major brain disorders using single cell transcriptomes and expression weighted cell type enrichment. <i>Front. Neurosci.</i> <b>10</b>, 16 (2016).</p><p><a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26858593" aria-label="PubMed reference 43">PubMed</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4730103" aria-label="PubMed Central reference 43">PubMed Central</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="article reference" href="https://doi.org/10.3389%2Ffnins.2016.00016" aria-label="Article reference 43">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 43" href="http://scholar.google.com/scholar_lookup?&amp;title=Identification%20of%20vulnerable%20cell%20types%20in%20major%20brain%20disorders%20using%20single%20cell%20transcriptomes%20and%20expression%20weighted%20cell%20type%20enrichment&amp;journal=Front.%20Neurosci.&amp;volume=10&amp;publication_year=2016&amp;author=Skene%2CNG&amp;author=Grant%2CSGN">
                    Google Scholar</a> 
                </p></li><li data-counter="44."><p id="ref-CR44">Willsey, A. J. et al. Coexpression networks implicate human midfetal deep cortical projection neurons in the pathogenesis of autism. <i>Cell</i> <b>155</b>, 997–1007 (2013).</p><p><a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXhvFWis7zP" aria-label="CAS reference 44">CAS</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24267886" aria-label="PubMed reference 44">PubMed</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3995413" aria-label="PubMed Central reference 44">PubMed Central</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.cell.2013.10.020" aria-label="Article reference 44">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 44" href="http://scholar.google.com/scholar_lookup?&amp;title=Coexpression%20networks%20implicate%20human%20midfetal%20deep%20cortical%20projection%20neurons%20in%20the%20pathogenesis%20of%20autism&amp;journal=Cell&amp;volume=155&amp;pages=997-1007&amp;publication_year=2013&amp;author=Willsey%2CAJ">
                    Google Scholar</a> 
                </p></li><li data-counter="45."><p id="ref-CR45">Chang, J., Gilman, S. R., Chiang, A. H., Sanders, S. J. &amp; Vitkup, D. Genotype to phenotype relationships in autism spectrum disorders. <i>Nat. Neurosci.</i> <b>18</b>, 191–198 (2015).</p><p><a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXitFKltLjE" aria-label="CAS reference 45">CAS</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25531569" aria-label="PubMed reference 45">PubMed</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="article reference" href="https://doi.org/10.1038%2Fnn.3907" aria-label="Article reference 45">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 45" href="http://scholar.google.com/scholar_lookup?&amp;title=Genotype%20to%20phenotype%20relationships%20in%20autism%20spectrum%20disorders&amp;journal=Nat.%20Neurosci.&amp;volume=18&amp;pages=191-198&amp;publication_year=2015&amp;author=Chang%2CJ&amp;author=Gilman%2CSR&amp;author=Chiang%2CAH&amp;author=Sanders%2CSJ&amp;author=Vitkup%2CD">
                    Google Scholar</a> 
                </p></li><li data-counter="46."><p id="ref-CR46">Wang, S. et al. De novo sequence and copy number variants are strongly associated with Tourette disorder and implicate cell polarity in pathogenesis. <i>Cell Rep.</i> <b>24</b>, 3441–3454.e12 (2018).</p><p><a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhslyqsbfK" aria-label="CAS reference 46">CAS</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30257206" aria-label="PubMed reference 46">PubMed</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6475626" aria-label="PubMed Central reference 46">PubMed Central</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.celrep.2018.08.082" aria-label="Article reference 46">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 46" href="http://scholar.google.com/scholar_lookup?&amp;title=De%20novo%20sequence%20and%20copy%20number%20variants%20are%20strongly%20associated%20with%20Tourette%20disorder%20and%20implicate%20cell%20polarity%20in%20pathogenesis&amp;journal=Cell%20Rep.&amp;volume=24&amp;pages=3441-3454.e12&amp;publication_year=2018&amp;author=Wang%2CS">
                    Google Scholar</a> 
                </p></li><li data-counter="47."><p id="ref-CR47">Heyne, H. O. et al. De novo variants in neurodevelopmental disorders with epilepsy. <i>Nat. Genet.</i> <b>50</b>, 1048–1053 (2018).</p><p><a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXht1WqsLvK" aria-label="CAS reference 47">CAS</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29942082" aria-label="PubMed reference 47">PubMed</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41588-018-0143-7" aria-label="Article reference 47">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 47" href="http://scholar.google.com/scholar_lookup?&amp;title=De%20novo%20variants%20in%20neurodevelopmental%20disorders%20with%20epilepsy&amp;journal=Nat.%20Genet.&amp;volume=50&amp;pages=1048-1053&amp;publication_year=2018&amp;author=Heyne%2CHO">
                    Google Scholar</a> 
                </p></li><li data-counter="48."><p id="ref-CR48">Singh, T. et al. Rare coding variants in ten genes confer substantial risk for schizophrenia. <i>Nature</i> <b>604</b>, 509–516 (2022).</p><p><a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XpvVWjtrs%3D" aria-label="CAS reference 48">CAS</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=35396579" aria-label="PubMed reference 48">PubMed</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41586-022-04556-w" aria-label="Article reference 48">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 48" href="http://scholar.google.com/scholar_lookup?&amp;title=Rare%20coding%20variants%20in%20ten%20genes%20confer%20substantial%20risk%20for%20schizophrenia&amp;journal=Nature&amp;volume=604&amp;pages=509-516&amp;publication_year=2022&amp;author=Singh%2CT">
                    Google Scholar</a> 
                </p></li><li data-counter="49."><p id="ref-CR49">Shu, C., Snyder, L. G., Shen, Y. &amp; Chung, W. K. The SPARK Consortium. Imputing cognitive impairment in SPARK, a large autism cohort. <i>Autism Res.</i> <b>15</b>, 156–170 (2022).</p><p><a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34636158" aria-label="PubMed reference 49">PubMed</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="article reference" href="https://doi.org/10.1002%2Faur.2622" aria-label="Article reference 49">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 49" href="http://scholar.google.com/scholar_lookup?&amp;title=Imputing%20cognitive%20impairment%20in%20SPARK%2C%20a%20large%20autism%20cohort&amp;journal=Autism%20Res.&amp;volume=15&amp;pages=156-170&amp;publication_year=2022&amp;author=Shu%2CC&amp;author=Snyder%2CLG&amp;author=Shen%2CY&amp;author=Chung%2CWK">
                    Google Scholar</a> 
                </p></li><li data-counter="50."><p id="ref-CR50">Arnheim, N. &amp; Calabrese, P. Understanding what determines the frequency and pattern of human germline mutations. <i>Nat. Rev. Genet.</i> <b>10</b>, 478–488 (2009).</p><p><a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXnsVWgtLk%3D" aria-label="CAS reference 50">CAS</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19488047" aria-label="PubMed reference 50">PubMed</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2744436" aria-label="PubMed Central reference 50">PubMed Central</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="article reference" href="https://doi.org/10.1038%2Fnrg2529" aria-label="Article reference 50">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 50" href="http://scholar.google.com/scholar_lookup?&amp;title=Understanding%20what%20determines%20the%20frequency%20and%20pattern%20of%20human%20germline%20mutations&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=10&amp;pages=478-488&amp;publication_year=2009&amp;author=Arnheim%2CN&amp;author=Calabrese%2CP">
                    Google Scholar</a> 
                </p></li><li data-counter="51."><p id="ref-CR51">Rees, E., Moskvina, V., Owen, M. J., O’Donovan, M. C. &amp; Kirov, G. De novo rates and selection of schizophrenia-associated copy number variants. <i>Biol. Psychiatry</i> <b>70</b>, 1109–1114 (2011).</p><p><a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21855053" aria-label="PubMed reference 51">PubMed</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.biopsych.2011.07.011" aria-label="Article reference 51">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 51" href="http://scholar.google.com/scholar_lookup?&amp;title=De%20novo%20rates%20and%20selection%20of%20schizophrenia-associated%20copy%20number%20variants&amp;journal=Biol.%20Psychiatry&amp;volume=70&amp;pages=1109-1114&amp;publication_year=2011&amp;author=Rees%2CE&amp;author=Moskvina%2CV&amp;author=Owen%2CMJ&amp;author=O%E2%80%99Donovan%2CMC&amp;author=Kirov%2CG">
                    Google Scholar</a> 
                </p></li><li data-counter="52."><p id="ref-CR52">Szklarczyk, D. et al. STRING v11: protein–protein association networks with increased coverage, supporting functional discovery in genome-wide experimental datasets. <i>Nucleic Acids Res.</i> <b>47</b>, D607–D613 (2019).</p><p><a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhs1Gqtr%2FM" aria-label="CAS reference 52">CAS</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30476243" aria-label="PubMed reference 52">PubMed</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="article reference" href="https://doi.org/10.1093%2Fnar%2Fgky1131" aria-label="Article reference 52">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 52" href="http://scholar.google.com/scholar_lookup?&amp;title=STRING%20v11%3A%20protein%E2%80%93protein%20association%20networks%20with%20increased%20coverage%2C%20supporting%20functional%20discovery%20in%20genome-wide%20experimental%20datasets&amp;journal=Nucleic%20Acids%20Res.&amp;volume=47&amp;pages=D607-D613&amp;publication_year=2019&amp;author=Szklarczyk%2CD">
                    Google Scholar</a> 
                </p></li><li data-counter="53."><p id="ref-CR53">Köhler, S. et al. The Human Phenotype Ontology in 2021. <i>Nucleic Acids Res.</i> <b>49</b>, D1207–D1217 (2021).</p><p><a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33264411" aria-label="PubMed reference 53">PubMed</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="article reference" href="https://doi.org/10.1093%2Fnar%2Fgkaa1043" aria-label="Article reference 53">Article</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXntlejt7o%3D" aria-label="CAS reference 53">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 53" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20Human%20Phenotype%20Ontology%20in%202021&amp;journal=Nucleic%20Acids%20Res.&amp;volume=49&amp;pages=D1207-D1217&amp;publication_year=2021&amp;author=K%C3%B6hler%2CS">
                    Google Scholar</a> 
                </p></li><li data-counter="54."><p id="ref-CR54">Raychaudhuri, S. et al. Accurately assessing the risk of schizophrenia conferred by rare copy-number variation affecting genes with brain function. <i>PLoS Genet.</i> <b>6</b>, e1001097 (2010).</p><p><a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20838587" aria-label="PubMed reference 54">PubMed</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2936523" aria-label="PubMed Central reference 54">PubMed Central</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="article reference" href="https://doi.org/10.1371%2Fjournal.pgen.1001097" aria-label="Article reference 54">Article</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXhtFyjsLnJ" aria-label="CAS reference 54">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 54" href="http://scholar.google.com/scholar_lookup?&amp;title=Accurately%20assessing%20the%20risk%20of%20schizophrenia%20conferred%20by%20rare%20copy-number%20variation%20affecting%20genes%20with%20brain%20function&amp;journal=PLoS%20Genet.&amp;volume=6&amp;publication_year=2010&amp;author=Raychaudhuri%2CS">
                    Google Scholar</a> 
                </p></li><li data-counter="55."><p id="ref-CR55">Carlson, J. et al. Extremely rare variants reveal patterns of germline mutation rate heterogeneity in humans. <i>Nat. Commun.</i> <b>9</b>, 3753 (2018).</p><p><a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30218074" aria-label="PubMed reference 55">PubMed</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6138700" aria-label="PubMed Central reference 55">PubMed Central</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41467-018-05936-5" aria-label="Article reference 55">Article</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhslamu7fL" aria-label="CAS reference 55">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 55" href="http://scholar.google.com/scholar_lookup?&amp;title=Extremely%20rare%20variants%20reveal%20patterns%20of%20germline%20mutation%20rate%20heterogeneity%20in%20humans&amp;journal=Nat.%20Commun.&amp;volume=9&amp;publication_year=2018&amp;author=Carlson%2CJ">
                    Google Scholar</a> 
                </p></li><li data-counter="56."><p id="ref-CR56">Samocha, K. E. et al. A framework for the interpretation of de novo mutation in human disease. <i>Nat. Genet.</i> <b>46</b>, 944–950 (2014).</p><p><a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXht1Gqtr%2FK" aria-label="CAS reference 56">CAS</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25086666" aria-label="PubMed reference 56">PubMed</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4222185" aria-label="PubMed Central reference 56">PubMed Central</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="article reference" href="https://doi.org/10.1038%2Fng.3050" aria-label="Article reference 56">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 56" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20framework%20for%20the%20interpretation%20of%20de%20novo%20mutation%20in%20human%20disease&amp;journal=Nat.%20Genet.&amp;volume=46&amp;pages=944-950&amp;publication_year=2014&amp;author=Samocha%2CKE">
                    Google Scholar</a> 
                </p></li><li data-counter="57."><p id="ref-CR57">Yamabhai, M. et al. Intersectin, a novel adaptor protein with two Eps15 homology and five Src homology 3 domains. <i>J. Biol. Chem.</i> <b>273</b>, 31401–31407 (1998).</p><p><a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK1cXns1ens7o%3D" aria-label="CAS reference 57">CAS</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=9813051" aria-label="PubMed reference 57">PubMed</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="article reference" href="https://doi.org/10.1074%2Fjbc.273.47.31401" aria-label="Article reference 57">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 57" href="http://scholar.google.com/scholar_lookup?&amp;title=Intersectin%2C%20a%20novel%20adaptor%20protein%20with%20two%20Eps15%20homology%20and%20five%20Src%20homology%203%20domains&amp;journal=J.%20Biol.%20Chem.&amp;volume=273&amp;pages=31401-31407&amp;publication_year=1998&amp;author=Yamabhai%2CM">
                    Google Scholar</a> 
                </p></li><li data-counter="58."><p id="ref-CR58">Wang, W. et al. ITSN-1 controls vesicle recycling at the neuromuscular junction and functions in parallel with DAB-1. <i>Traffic</i> <b>9</b>, 742–754 (2008).</p><p><a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXlsVSrtb0%3D" aria-label="CAS reference 58">CAS</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18298590" aria-label="PubMed reference 58">PubMed</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3791081" aria-label="PubMed Central reference 58">PubMed Central</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="article reference" href="https://doi.org/10.1111%2Fj.1600-0854.2008.00712.x" aria-label="Article reference 58">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 58" href="http://scholar.google.com/scholar_lookup?&amp;title=ITSN-1%20controls%20vesicle%20recycling%20at%20the%20neuromuscular%20junction%20and%20functions%20in%20parallel%20with%20DAB-1&amp;journal=Traffic&amp;volume=9&amp;pages=742-754&amp;publication_year=2008&amp;author=Wang%2CW">
                    Google Scholar</a> 
                </p></li><li data-counter="59."><p id="ref-CR59">Stringham, E. G. &amp; Schmidt, K. L. Navigating the cell: UNC-53 and the navigators, a family of cytoskeletal regulators with multiple roles in cell migration, outgrowth and trafficking. <i>Cell Adhes. Migr.</i> <b>3</b>, 342–346 (2009).</p><p><a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="article reference" href="https://doi.org/10.4161%2Fcam.3.4.9451" aria-label="Article reference 59">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 59" href="http://scholar.google.com/scholar_lookup?&amp;title=Navigating%20the%20cell%3A%20UNC-53%20and%20the%20navigators%2C%20a%20family%20of%20cytoskeletal%20regulators%20with%20multiple%20roles%20in%20cell%20migration%2C%20outgrowth%20and%20trafficking&amp;journal=Cell%20Adhes.%20Migr.&amp;volume=3&amp;pages=342-346&amp;publication_year=2009&amp;author=Stringham%2CEG&amp;author=Schmidt%2CKL">
                    Google Scholar</a> 
                </p></li><li data-counter="60."><p id="ref-CR60">Karenko, L. et al. Primary cutaneous T-cell lymphomas show a deletion or translocation affecting <i>NAV3</i>, the human <i>UNC-53</i> homologue. <i>Cancer Res.</i> <b>65</b>, 8101–8110 (2005).</p><p><a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2MXhtVShsbrJ" aria-label="CAS reference 60">CAS</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16166283" aria-label="PubMed reference 60">PubMed</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="article reference" href="https://doi.org/10.1158%2F0008-5472.CAN-04-0366" aria-label="Article reference 60">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 60" href="http://scholar.google.com/scholar_lookup?&amp;title=Primary%20cutaneous%20T-cell%20lymphomas%20show%20a%20deletion%20or%20translocation%20affecting%20NAV3%2C%20the%20human%20UNC-53%20homologue&amp;journal=Cancer%20Res.&amp;volume=65&amp;pages=8101-8110&amp;publication_year=2005&amp;author=Karenko%2CL">
                    Google Scholar</a> 
                </p></li><li data-counter="61."><p id="ref-CR61">Sanghvi-Shah, R. &amp; Weber, G. F. Intermediate filaments at the junction of mechanotransduction, migration, and development. <i>Front Cell Dev. Biol.</i> <b>5</b>, 81 (2017).</p><p><a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28959689" aria-label="PubMed reference 61">PubMed</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5603733" aria-label="PubMed Central reference 61">PubMed Central</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="article reference" href="https://doi.org/10.3389%2Ffcell.2017.00081" aria-label="Article reference 61">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 61" href="http://scholar.google.com/scholar_lookup?&amp;title=Intermediate%20filaments%20at%20the%20junction%20of%20mechanotransduction%2C%20migration%2C%20and%20development&amp;journal=Front%20Cell%20Dev.%20Biol.&amp;volume=5&amp;publication_year=2017&amp;author=Sanghvi-Shah%2CR&amp;author=Weber%2CGF">
                    Google Scholar</a> 
                </p></li><li data-counter="62."><p id="ref-CR62">Martin, H. C. et al. Quantifying the contribution of recessive coding variation to developmental disorders. <i>Science</i> <b>362</b>, 1161–1164 (2018).</p><p><a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXisVegu7zN" aria-label="CAS reference 62">CAS</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30409806" aria-label="PubMed reference 62">PubMed</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6726470" aria-label="PubMed Central reference 62">PubMed Central</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="article reference" href="https://doi.org/10.1126%2Fscience.aar6731" aria-label="Article reference 62">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 62" href="http://scholar.google.com/scholar_lookup?&amp;title=Quantifying%20the%20contribution%20of%20recessive%20coding%20variation%20to%20developmental%20disorders&amp;journal=Science&amp;volume=362&amp;pages=1161-1164&amp;publication_year=2018&amp;author=Martin%2CHC">
                    Google Scholar</a> 
                </p></li><li data-counter="63."><p id="ref-CR63">Guo, H. et al. Inherited and multiple de novo mutations in autism/developmental delay risk genes suggest a multifactorial model. <i>Mol. Autism</i> <b>9</b>, 64 (2018).</p><p><a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhsF2hu7fK" aria-label="CAS reference 63">CAS</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30564305" aria-label="PubMed reference 63">PubMed</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6293633" aria-label="PubMed Central reference 63">PubMed Central</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="article reference" href="https://doi.org/10.1186%2Fs13229-018-0247-z" aria-label="Article reference 63">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 63" href="http://scholar.google.com/scholar_lookup?&amp;title=Inherited%20and%20multiple%20de%20novo%20mutations%20in%20autism%2Fdevelopmental%20delay%20risk%20genes%20suggest%20a%20multifactorial%20model&amp;journal=Mol.%20Autism&amp;volume=9&amp;publication_year=2018&amp;author=Guo%2CH">
                    Google Scholar</a> 
                </p></li><li data-counter="64."><p id="ref-CR64">Poplin, R. et al. Scaling accurate genetic variant discovery to tens of thousands of samples. Preprint at <a href="https://www.biorxiv.org/content/10.1101/201178v3">https://www.bioRxiv.org/content/10.1101/201178v3</a> (2018).</p></li><li data-counter="65."><p id="ref-CR65">Garrison, E. &amp; Marth, G. Haplotype-based variant detection from short-read sequencing. Preprint at <a href="https://arxiv.org/abs/1207.3907">https://arXiv.org/abs/1207.3907</a> (2012).</p></li><li data-counter="66."><p id="ref-CR66">GENOMICS plc - weCall: fast, accurate and simple to use command line tool for variant detection in NGS data. <a href="https://github.com/Genomicsplc/wecall">https://github.com/Genomicsplc/wecall</a></p></li><li data-counter="67."><p id="ref-CR67">Poplin, R. et al. A universal SNP and small-indel variant caller using deep neural networks. <i>Nat. Biotechnol.</i> <b>36</b>, 983–987 (2018).</p><p><a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhslOhtbjE" aria-label="CAS reference 67">CAS</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30247488" aria-label="PubMed reference 67">PubMed</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="article reference" href="https://doi.org/10.1038%2Fnbt.4235" aria-label="Article reference 67">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 67" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20universal%20SNP%20and%20small-indel%20variant%20caller%20using%20deep%20neural%20networks&amp;journal=Nat.%20Biotechnol.&amp;volume=36&amp;pages=983-987&amp;publication_year=2018&amp;author=Poplin%2CR">
                    Google Scholar</a> 
                </p></li><li data-counter="68."><p id="ref-CR68">Lin, M. F. et al. GLnexus: joint variant calling for large cohort sequencing. Preprint at <a href="https://www.biorxiv.org/content/10.1101/343970v1">https://www.bioRxiv.org/content/10.1101/343970v1</a> (2018).</p></li><li data-counter="69."><p id="ref-CR69">Taliun, D. et al. Sequencing of 53,831 diverse genomes from the NHLBI TOPMed Program. <i>Nature</i> <b>590</b>, 290–299 (2021).</p><p><a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXjslKnsrY%3D" aria-label="CAS reference 69">CAS</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33568819" aria-label="PubMed reference 69">PubMed</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7875770" aria-label="PubMed Central reference 69">PubMed Central</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41586-021-03205-y" aria-label="Article reference 69">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 69" href="http://scholar.google.com/scholar_lookup?&amp;title=Sequencing%20of%2053%2C831%20diverse%20genomes%20from%20the%20NHLBI%20TOPMed%20Program&amp;journal=Nature&amp;volume=590&amp;pages=290-299&amp;publication_year=2021&amp;author=Taliun%2CD">
                    Google Scholar</a> 
                </p></li><li data-counter="70."><p id="ref-CR70">Li, H. Toward better understanding of artifacts in variant calling from high-coverage samples. <i>Bioinformatics</i> <b>30</b>, 2843–2851 (2014).</p><p><a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XhtFOrt7fP" aria-label="CAS reference 70">CAS</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24974202" aria-label="PubMed reference 70">PubMed</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4271055" aria-label="PubMed Central reference 70">PubMed Central</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="article reference" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtu356" aria-label="Article reference 70">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 70" href="http://scholar.google.com/scholar_lookup?&amp;title=Toward%20better%20understanding%20of%20artifacts%20in%20variant%20calling%20from%20high-coverage%20samples&amp;journal=Bioinformatics&amp;volume=30&amp;pages=2843-2851&amp;publication_year=2014&amp;author=Li%2CH">
                    Google Scholar</a> 
                </p></li><li data-counter="71."><p id="ref-CR71">Maenner, M. J. et al. Prevalence of autism spectrum disorder among children aged 8 years — Autism and Developmental Disabilities Monitoring Network, 11 Sites, United States, 2016. <i>MMWR Surveill. Summ.</i> <b>69</b>, 1–12 (2020).</p><p><a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32214087" aria-label="PubMed reference 71">PubMed</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7119644" aria-label="PubMed Central reference 71">PubMed Central</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="article reference" href="https://doi.org/10.15585%2Fmmwr.ss6904a1" aria-label="Article reference 71">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 71" href="http://scholar.google.com/scholar_lookup?&amp;title=Prevalence%20of%20autism%20spectrum%20disorder%20among%20children%20aged%208%20years%20%E2%80%94%20Autism%20and%20Developmental%20Disabilities%20Monitoring%20Network%2C%2011%20Sites%2C%20United%20States%2C%202016&amp;journal=MMWR%20Surveill.%20Summ.&amp;volume=69&amp;pages=1-12&amp;publication_year=2020&amp;author=Maenner%2CMJ">
                    Google Scholar</a> 
                </p></li><li data-counter="72."><p id="ref-CR72">Power, R. A. et al. Fecundity of patients with schizophrenia, autism, bipolar disorder, depression, anorexia nervosa, or substance abuse vs their unaffected siblings. <i>JAMA Psychiatry</i> <b>70</b>, 22–30 (2013).</p><p><a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23147713" aria-label="PubMed reference 72">PubMed</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="article reference" href="https://doi.org/10.1001%2Fjamapsychiatry.2013.268" aria-label="Article reference 72">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 72" href="http://scholar.google.com/scholar_lookup?&amp;title=Fecundity%20of%20patients%20with%20schizophrenia%2C%20autism%2C%20bipolar%20disorder%2C%20depression%2C%20anorexia%20nervosa%2C%20or%20substance%20abuse%20vs%20their%20unaffected%20siblings&amp;journal=JAMA%20Psychiatry&amp;volume=70&amp;pages=22-30&amp;publication_year=2013&amp;author=Power%2CRA">
                    Google Scholar</a> 
                </p></li><li data-counter="73."><p id="ref-CR73">Wright, C. F. et al. Genetic diagnosis of developmental disorders in the DDD study: a scalable analysis of genome-wide research data. <i>Lancet</i> <b>385</b>, 1305–1314 (2015).</p><p><a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25529582" aria-label="PubMed reference 73">PubMed</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4392068" aria-label="PubMed Central reference 73">PubMed Central</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="article reference" href="https://doi.org/10.1016%2FS0140-6736%2814%2961705-0" aria-label="Article reference 73">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 73" href="http://scholar.google.com/scholar_lookup?&amp;title=Genetic%20diagnosis%20of%20developmental%20disorders%20in%20the%20DDD%20study%3A%20a%20scalable%20analysis%20of%20genome-wide%20research%20data&amp;journal=Lancet&amp;volume=385&amp;pages=1305-1314&amp;publication_year=2015&amp;author=Wright%2CCF">
                    Google Scholar</a> 
                </p></li><li data-counter="74."><p id="ref-CR74">Abrahams, B. S. et al. SFARI Gene 2.0: a community-driven knowledgebase for the autism spectrum disorders (ASDs). <i>Mol. Autism</i> <b>4</b>, 36 (2013).</p><p><a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24090431" aria-label="PubMed reference 74">PubMed</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3851189" aria-label="PubMed Central reference 74">PubMed Central</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="article reference" href="https://doi.org/10.1186%2F2040-2392-4-36" aria-label="Article reference 74">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 74" href="http://scholar.google.com/scholar_lookup?&amp;title=SFARI%20Gene%202.0%3A%20a%20community-driven%20knowledgebase%20for%20the%20autism%20spectrum%20disorders%20%28ASDs%29&amp;journal=Mol.%20Autism&amp;volume=4&amp;publication_year=2013&amp;author=Abrahams%2CBS">
                    Google Scholar</a> 
                </p></li><li data-counter="75."><p id="ref-CR75">Singer, E. What makes an autism gene? <a href="https://sparkforautism.org/discover_article/what-makes-an-autism-gene/">https://SPARKforAutism.org/discover_article/what-makes-an-autism-gene/</a> (2017).</p></li><li data-counter="76."><p id="ref-CR76">Fagerberg, L. et al. Analysis of the human tissue-specific expression by genome-wide integration of transcriptomics and antibody-based proteomics. <i>Mol. Cell. Proteomics</i> <b>13</b>, 397–406 (2014).</p><p><a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXhsl2nsrg%3D" aria-label="CAS reference 76">CAS</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24309898" aria-label="PubMed reference 76">PubMed</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="article reference" href="https://doi.org/10.1074%2Fmcp.M113.035600" aria-label="Article reference 76">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 76" href="http://scholar.google.com/scholar_lookup?&amp;title=Analysis%20of%20the%20human%20tissue-specific%20expression%20by%20genome-wide%20integration%20of%20transcriptomics%20and%20antibody-based%20proteomics&amp;journal=Mol.%20Cell.%20Proteomics&amp;volume=13&amp;pages=397-406&amp;publication_year=2014&amp;author=Fagerberg%2CL">
                    Google Scholar</a> 
                </p></li><li data-counter="77."><p id="ref-CR77">Mo, A. et al. Epigenomic signatures of neuronal diversity in the mammalian brain. <i>Neuron</i> <b>86</b>, 1369–1384 (2015).</p><p><a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXht1Sgu7%2FF" aria-label="CAS reference 77">CAS</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26087164" aria-label="PubMed reference 77">PubMed</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4499463" aria-label="PubMed Central reference 77">PubMed Central</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuron.2015.05.018" aria-label="Article reference 77">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 77" href="http://scholar.google.com/scholar_lookup?&amp;title=Epigenomic%20signatures%20of%20neuronal%20diversity%20in%20the%20mammalian%20brain&amp;journal=Neuron&amp;volume=86&amp;pages=1369-1384&amp;publication_year=2015&amp;author=Mo%2CA">
                    Google Scholar</a> 
                </p></li><li data-counter="78."><p id="ref-CR78">Pirooznia, M. et al. SynaptomeDB: an ontology-based knowledgebase for synaptic genes. <i>Bioinformatics</i> <b>28</b>, 897–899 (2012).</p><p><a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38Xks1Sisrs%3D" aria-label="CAS reference 78">CAS</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22285564" aria-label="PubMed reference 78">PubMed</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3307115" aria-label="PubMed Central reference 78">PubMed Central</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="article reference" href="https://doi.org/10.1093%2Fbioinformatics%2Fbts040" aria-label="Article reference 78">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 78" href="http://scholar.google.com/scholar_lookup?&amp;title=SynaptomeDB%3A%20an%20ontology-based%20knowledgebase%20for%20synaptic%20genes&amp;journal=Bioinformatics&amp;volume=28&amp;pages=897-899&amp;publication_year=2012&amp;author=Pirooznia%2CM">
                    Google Scholar</a> 
                </p></li><li data-counter="79."><p id="ref-CR79">Wagnon, J. L. et al. CELF4 regulates translation and local abundance of a vast set of mRNAs, including genes associated with regulation of synaptic function. <i>PLoS Genet.</i> <b>8</b>, e1003067 (2012).</p><p><a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XhvVKjtrfL" aria-label="CAS reference 79">CAS</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23209433" aria-label="PubMed reference 79">PubMed</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3510034" aria-label="PubMed Central reference 79">PubMed Central</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="article reference" href="https://doi.org/10.1371%2Fjournal.pgen.1003067" aria-label="Article reference 79">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 79" href="http://scholar.google.com/scholar_lookup?&amp;title=CELF4%20regulates%20translation%20and%20local%20abundance%20of%20a%20vast%20set%20of%20mRNAs%2C%20including%20genes%20associated%20with%20regulation%20of%20synaptic%20function&amp;journal=PLoS%20Genet.&amp;volume=8&amp;publication_year=2012&amp;author=Wagnon%2CJL">
                    Google Scholar</a> 
                </p></li><li data-counter="80."><p id="ref-CR80">Eppig, J. T. Mouse Genome Informatics (MGI) Resource: genetic, genomic, and biological knowledgebase for the laboratory mouse. <i>ILAR J.</i> <b>58</b>, 17–41 (2017).</p><p><a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXitFOntbbN" aria-label="CAS reference 80">CAS</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28838066" aria-label="PubMed reference 80">PubMed</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5886341" aria-label="PubMed Central reference 80">PubMed Central</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="article reference" href="https://doi.org/10.1093%2Filar%2Filx013" aria-label="Article reference 80">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 80" href="http://scholar.google.com/scholar_lookup?&amp;title=Mouse%20Genome%20Informatics%20%28MGI%29%20Resource%3A%20genetic%2C%20genomic%2C%20and%20biological%20knowledgebase%20for%20the%20laboratory%20mouse&amp;journal=ILAR%20J.&amp;volume=58&amp;pages=17-41&amp;publication_year=2017&amp;author=Eppig%2CJT">
                    Google Scholar</a> 
                </p></li><li data-counter="81."><p id="ref-CR81">Cutler, A. &amp; Breiman, L. Archetypal analysis. <i>Technometrics</i> <b>36</b>, 338–347 (1994).</p><p><a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="article reference" href="https://doi.org/10.1080%2F00401706.1994.10485840" aria-label="Article reference 81">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 81" href="http://scholar.google.com/scholar_lookup?&amp;title=Archetypal%20analysis&amp;journal=Technometrics&amp;volume=36&amp;pages=338-347&amp;publication_year=1994&amp;author=Cutler%2CA&amp;author=Breiman%2CL">
                    Google Scholar</a> 
                </p></li><li data-counter="82."><p id="ref-CR82">Thorndike, R. L. Who belongs in the family? <i>Psychometrika</i> <b>18</b>, 267–276 (1953).</p><p><a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="article reference" href="https://doi.org/10.1007%2FBF02289263" aria-label="Article reference 82">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 82" href="http://scholar.google.com/scholar_lookup?&amp;title=Who%20belongs%20in%20the%20family%3F&amp;journal=Psychometrika&amp;volume=18&amp;pages=267-276&amp;publication_year=1953&amp;author=Thorndike%2CRL">
                    Google Scholar</a> 
                </p></li><li data-counter="83."><p id="ref-CR83">Subramanian, A. et al. Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles. <i>Proc. Natl Acad. Sci.</i> <b>102</b>, 15545–15550 (2005).</p><p><a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2MXht1ShtrnO" aria-label="CAS reference 83">CAS</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16199517" aria-label="PubMed reference 83">PubMed</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1239896" aria-label="PubMed Central reference 83">PubMed Central</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="article reference" href="https://doi.org/10.1073%2Fpnas.0506580102" aria-label="Article reference 83">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 83" href="http://scholar.google.com/scholar_lookup?&amp;title=Gene%20set%20enrichment%20analysis%3A%20a%20knowledge-based%20approach%20for%20interpreting%20genome-wide%20expression%20profiles&amp;journal=Proc.%20Natl%20Acad.%20Sci.&amp;volume=102&amp;pages=15545-15550&amp;publication_year=2005&amp;author=Subramanian%2CA">
                    Google Scholar</a> 
                </p></li><li data-counter="84."><p id="ref-CR84">Packer, J. S. et al. CLAMMS: a scalable algorithm for calling common and rare copy number variants from exome sequencing data. <i>Bioinformatics</i> <b>32</b>, 133–135 (2016).</p><p><a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XhtlaqsLzK" aria-label="CAS reference 84">CAS</a> 
    <a data-track="click" data-track-label="link" rel="nofollow noopener" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26382196" aria-label="PubMed reference 84">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 84" href="http://scholar.google.com/scholar_lookup?&amp;title=CLAMMS%3A%20a%20scalable%20algorithm%20for%20calling%20common%20and%20rare%20copy%20number%20variants%20from%20exome%20sequencing%20data&amp;journal=Bioinformatics&amp;volume=32&amp;pages=133-135&amp;publication_year=2016&amp;author=Packer%2CJS">
                    Google Scholar</a> 
                </p></li></ol><p><a data-track="click" data-track-action="download citation references" data-track-label="link" href="https://citation-needed.springer.com/v2/references/10.1038/s41588-022-01148-2?format=refman&amp;flavour=references">Download references<svg width="16" height="16" focusable="false" role="img" aria-hidden="true"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#global-icon-download"></use></svg></a></p></div></div></div></section></div><section data-title="Acknowledgements"><div id="Ack1-section"><h2 id="Ack1">Acknowledgements</h2><p>We are extremely grateful to the thousands of individuals and families who are participating in this study. We thank the sites, staff and volunteers of the SPARK Clinical Site Network and SFARI for their invaluable contributions. We are grateful to the many ASD advocacy and service organizations that have helped us inform the community about SPARK, including the Autism Society of America and its affiliates, the Global and Regional Autism Spectrum Partnership (GRASP), Autism Speaks, and the Autism Science Foundation. We thank the members of SPARK’s Community Advisory Council for providing feedback and advice. We thank members of our Scientific Advisory Board and SFARI scientists for advice on our protocol and participant outreach and retention strategies. Our de novo analyses involved primary data from SSC and results from published ASD studies, including ASC and MSSNG, and our case-control analysis used published population controls from gnomAD (exome v2.1.1, non-neuro subset) and TOPMed (WGS, freeze 8). Approved researchers can obtain the SPARK population dataset described in this study by applying at <a href="https://base.sfari.org">https://base.sfari.org</a>. The SPARK initiative is funded by the Simons Foundation as part of SFARI. This research was supported, in part, by a grant from the National Institute of Mental Health (NIMH R01MH101221) and grants from the Simons Foundation (SFARI nos. 608045 and 810018EE) to E.E.E., a grant from the National Institutes of Health (NIH R01GM120609) and from the Simons Foundation (SFARI no. 606450) to Y.S., a grant from the Simons Foundation (SFARI no. 644038) to B.J.O. and J.J.M., a grant from the National Institute of Mental Health to T.N.T. (NIMH 1K99MH117165), and grants MH105527 and DC014489 from the National Institute of Health to J.J.M. E.E.E. is an investigator of the Howard Hughes Medical Institute.</p></div></section><section aria-labelledby="author-information" data-title="Author information"><div id="author-information-section"><h2 id="author-information">Author information</h2><div id="author-information-content"><p><span id="author-notes">Author notes</span></p><ol><li id="na1"><p>These authors contributed equally: Xueya Zhou, Pamela Feliciano, Chang Shu, Tianyun Wang, Irina Astrovskaya.</p></li><li id="na2"><p>These authors jointly supervised this work: Yufeng Shen, Wendy K. Chung.</p></li></ol><h3 id="affiliations">Authors and Affiliations</h3><ol><li id="Aff1"><p>Department of Pediatrics, Columbia University Medical Center, New York, NY, USA</p><p>Xueya Zhou, Chang Shu, Joseph U. Obiajulu &amp; Wendy K. Chung</p></li><li id="Aff2"><p>Department of Systems Biology, Columbia University Medical Center, New York, NY, USA</p><p>Xueya Zhou, Chang Shu, Joseph U. Obiajulu, Alex Kitaygorodsky &amp; Yufeng Shen</p></li><li id="Aff3"><p>Simons Foundation, New York, NY, USA</p><p>Pamela Feliciano, Irina Astrovskaya, Jacob B. Hall, Jessica R. Wright, Simon Xuming Xu, Olena Marchenko, Christopher Fleisch, Sarah D. Barns, LeeAnne Green Snyder, Bing Han, Alpha Amatya, Asif Bashar, Alex E. Lash, Alexandra Goler, Alison Holbrook, Andrea J. Ace, Amy M. Daniels, Anup Mankar, Alexandra N. Stephens, Ai Nhu Nguyen, Beverly E. Robertson, Bill Jensen, Brianna M. Vernoia, Brady Schwind, Cheryl Cohen, Chris Diggins, Chris Rigby, Casey White-Lehman, Dennis Vasquez Montes, Dave Cho, Elizabeth Brooks, Erica Jones, Eirene O’Connor, Esmeralda Perez, Emily Singer, Hana Zaydens, John Acampado, Jennylyn Brown, Jessica Hernandez, J. Kiely Law, Julie Manoharan, Jibrielle Polite, Jennifer Tjernagel, Jaimie Toroney, Katharine Diehl, Lindsey A. Cartner, Lori Mann, Luke P. Grosvenor, Malcolm D. Mallardi, Martin E. Butler, Misia Kowanda, Mahfuza Sabiha, Marina Sarris, Noah Lawson, Nathan Lo, Natalie Nagpal, Neelay Shah, Richard Marini, Rishiraj Rana, Richard Remington, Swami Ganesan, Stanley Jean, Swapnil Shah, Sabrina Xiao, Simon Xu, Tunisia Greene, Vincent J. Myers, Wubin Chin, Natalia Volfovsky &amp; Wendy K. Chung</p></li><li id="Aff4"><p>Department of Genome Sciences, University of Washington School of Medicine, Seattle, WA, USA</p><p>Tianyun Wang, Shwetha C. Murali, William T. Harvey &amp; Evan E. Eichler</p></li><li id="Aff5"><p>Department of Medical Genetics, Center for Medical Genetics, School of Basic Medical Sciences, Peking University Health Science Center, Beijing, China</p><p>Tianyun Wang</p></li><li id="Aff6"><p>Neuroscience Research Institute, Department of Neurobiology, School of Basic Medical Sciences, Peking University Health Science Center; Key Laboratory for Neuroscience, Ministry of Education of China &amp; National Health Commission of China, Beijing, China</p><p>Tianyun Wang</p></li><li id="Aff7"><p>Howard Hughes Medical Institute, University of Washington, Seattle, WA, USA</p><p>Shwetha C. Murali &amp; Evan E. Eichler</p></li><li id="Aff8"><p>Department of Psychiatry, University of Iowa Carver College of Medicine, Iowa City, IA, USA</p><p>Leo Brueggeman, Taylor R. Thomas, Cate Buescher, Ethan Bahl, Lucas Casten, Sydney Kramer, Tanner Koomar &amp; Jacob J. Michaelson</p></li><li id="Aff9"><p>Program in Neurogenetics, Department of Neurology, David Geffen School of Medicine, University of California, Los Angeles, Los Angeles, CA, USA</p><p>Timothy S. Chang, Daniel H. Geschwind &amp; Elizabeth Ruzzo</p></li><li id="Aff10"><p>Department of Genetics, Washington University, St. Louis, MO, USA</p><p>Tychele N. Turner</p></li><li id="Aff11"><p>Department of Molecular &amp; Medical Genetics, Oregon Health &amp; Science University, Portland, OR, USA</p><p>Andrew Nishida &amp; Brian J. O’Roak</p></li><li id="Aff12"><p>Department of Biomedical Informatics, Columbia University Medical Center, New York, NY, USA</p><p>Yufeng Shen</p></li><li id="Aff13"><p>Department of Medicine, Columbia University Medical Center, New York, NY, USA</p><p>Wendy K. Chung</p></li><li id="Aff14"><p>Department of Psychiatry and Behavioral Sciences, Rush University Medical Center, Chicago, IL, USA</p><p>Adrienne Adams, Anna Hirshman, Allison L. Wainer, Caroline Leonczyk, Cynthia Pierre, Diamond Phillips, Edith Ocampo, Holly Lechniak, Kathryn A. Schweers, Latha V. Soorya, Lisa Yeh, Madison Printen, Natalie Berger, Norma Calderon, Nicole M. Russo-Ponsaran, Rachel A. Gordon, Soo J. Lee &amp; Sarely Licona</p></li><li id="Aff15"><p>Department of Pediatrics, University of Utah, Salt Lake City, UT, USA</p><p>Alicia Andrus, Dawn Bentley, Gabriele Baraghoshi, Gregory Schoonover, Laurie Lesher &amp; Paul S. Carbone</p></li><li id="Aff16"><p>Vanderbilt Kennedy Center, Vanderbilt University Medical Center, Nashville, TN, USA</p><p>Anna Berman, Alexandra Miceli, Amy Swanson, Ellen Grimes, Nicole Bardett, Nina Harris &amp; S. Carpenter</p></li><li id="Aff17"><p>Center for Autism and Related Disorders, Kennedy Krieger Institute, Baltimore, MD, USA</p><p>Alison Brown, Alexander P. McKenzie, Bonnie VanMetre, Catherine Edmonson, Emily Dillon, Ericka L. Wodka, Hanna Hutter, Jason Neely, Katrina Pama, Melinda Koza, Megan McTaggart, Nicolas Alvarez, Rebecca J. Landa, Roman Shikov, Sarah Michaels &amp; Vini Singh</p></li><li id="Aff18"><p>Department of Psychiatry and Biobehavioral Sciences, University of California, Los Angeles, Los Angeles, CA, USA</p><p>Alexies Camba, Amanda C. Gulsrud, Catherine Lord, Ivette Arriaga, James T. McCracken, Karla Murillo, Katherine Tsai, Monica Haley, Maira Tafolla, Sophia Sandhu &amp; Wha S. Yang</p></li><li id="Aff19"><p>PreventionGenetics, Marshfield, WI, USA</p><p>Anthony D. Krentz, Angela J. Gruber, Gregory J. Fischer &amp; Marc A. Popp</p></li><li id="Aff20"><p>Thompson Center for Autism and Neurodevelopmental Disorders, University of Missouri, Columbia, MO, USA</p><p>Amanda D. Shocklee, Hannah Cottrell, Jennifer Delaporte, Kathy Hirst, Kerri Nowell, Megan Sweeney, Nicole Takahashi, Shelby Birdwell, Samantha Hunter &amp; Stephen M. Kanne</p></li><li id="Aff21"><p>Department of Pediatrics, University of Minnesota, Minneapolis, MN, USA</p><p>Amy Esler</p></li><li id="Aff22"><p>Child Study Center, Yale School of Medicine, New Haven, CT, USA</p><p>Anne Fanta, Abha R. Gupta, Catherine A. W. Sullivan, Hope Koene, Jane Jurayj, Roger Jou &amp; Sarah Boland</p></li><li id="Aff23"><p>Department of Neurogenetics, Kennedy Krieger Institute, Baltimore, MD, USA</p><p>Ali Fatemi</p></li><li id="Aff24"><p>Department of Psychiatry, University of Michigan, Ann Arbor, MI, USA</p><p>Angela Fish, Annes Kim, Ashley Raven, Allyson Zick, Costanza Colombi, Fiona K. Miller, Jessyca Judge, Kate D. Fitzgerald, Mohammad Ghaziuddin, Sharmista Chintalapalli &amp; Sarah Mohiuddin</p></li><li id="Aff25"><p>Department of Psychology, University of Miami’s Center for Autism and Related Disabilities (UM-CARD), Coral Gables, FL, USA</p><p>Antonio Gonzalez, Anibal Gutierrez Jr., Brandon Lobisi, Cindy Zha, Dahriana Correa, David Giancarla, Hoa Lam Schneider, Joaquin Comitre, Jacob Rosewater, Lynette M. Herbert, Michael Alessandri, Michele Cutri, Melissa N. Hale, Madelyn Rayos &amp; Nickelle Decius</p></li><li id="Aff26"><p>Department of Psychiatry and Behavioral Sciences, Stanford University, Stanford, CA, USA</p><p>Antonio Hardan, Daniela Rojas, Elizabeth Orrick, Jared Gong, Margot Frayne, Morgan Steele, Robin Libove &amp; Sandy Littlefield</p></li><li id="Aff27"><p>Division of Pediatric Psychology and Neuropsychology, Nationwide Children’s Hospital (Child Development Center), Columbus, OH, USA</p><p>Amy Hess, Charles Albright, Daniel Lee Coury, Erin Given, Elizabeth Kryszak, Eric M. Butter, Gabrielle Tiede, Jessica Scherr, Kevin Stephenson, Megan Norris, Nancy Long, Nicholas Russell, Sara Eldred &amp; Teresa Ibanez</p></li><li id="Aff28"><p>John P. Hussman Institute for Human Genomics, University of Miami Miller School of Medicine, Miami, FL, USA</p><p>Anthony J. Griswold</p></li><li id="Aff29"><p>Department of Psychiatry, University of Minnesota, Minneapolis, MN, USA</p><p>Andrea Jarratt, Alissa Jorgenson, Angela Tesng, Claudia Campo-Soria, Dain Howes, Dalia Istephanous, Desiree Rambeck, Diksha Srishyla, Gregory Hooks, Jaclyn Gunderson, Katie Beard, Laura Simon, Lucy Wasserburg, Megan DuBois, Natasha Lillie, Olivia Newman, Sunday Francis, Suma Jacob &amp; Wenteng CaI</p></li><li id="Aff30"><p>Department of Psychiatry and Behavioral Neuroscience, Cincinnati Children’s Hospital Medical Center - Research Foundation, Cincinnati, OH, USA</p><p>Anna Jelinek, Craig A. Erickson, Carrie Fassler, Elizabeth Blank, Erica Lampert, Ernest V. Pedapati, Jose Polanco, Kaela O’Brien, Natalie Madi, Nicole Mccoy, Patricia Manning, Rebecca C. Shaffer, Stephanie Booker, Sophia Melnyk &amp; Shelley Randall</p></li><li id="Aff31"><p>Department of Pediatrics, Vanderbilt University Medical Center, Nashville, TN, USA</p><p>A. Pablo Juarez, Amy Nicholson &amp; Zachary E. Warren</p></li><li id="Aff32"><p>Department of Psychiatry, Oregon Health &amp; Science University, Portland, OR, USA</p><p>Addie Luo, Desi Limpoco, Eric J. Fombonne, Faris Fazal, Hadley Morotti, Landon Beeson, Leigh Coppola, Lillian D. Pacheco, Lark Y. Huang-Storms, Shelley Aberle &amp; Sarah Mastel</p></li><li id="Aff33"><p>Department of Pediatrics, JFK Partners/University of Colorado School of Medicine, Aurora, CO, USA</p><p>Angela L. Rachubinski, Amber Tallbull, Cordelia R. Rosenberg, Jeanette Cordova, Lisa Cordiero, Nicole Targalia, Susan Hepburn, Sandra L. Friedman &amp; Tamim Shaikh</p></li><li id="Aff34"><p>Department of Neurosciences, University of California, San Diego and SARRC Phoenix, La Jolla, CA, USA</p><p>Andrew Mason, Anthony Sziklay, Chris Smith, Erin Bower, Eric Courchesne, Hannah E. Kaplan, Hongjian Qi, Haicang Zhang, Haoquan Zhao, Jiayao Wang, Kendra Coleman, Karen L. Pierce, Mojeeb Shir, Sidi Xu, Tia Chen &amp; Tiziano Pramparo</p></li><li id="Aff35"><p>Department of Pediatrics, Boston Children’s Hospital, Boston, MA, USA</p><p>Anna Milliken, Christopher Zaro, Emily T. Matthews, Gabriella Aberbach, Katherine G. Pawlowski, Lisa M. Prock, Michelle Coughlin &amp; Margaret Hojlo</p></li><li id="Aff36"><p>Department of Pediatrics, Montefiore Medical Center and The Albert Einstein College of Medicine, Bronx, NY, USA</p><p>Amy Morales-Lara, Lisa H. Shulman, Molly O’Neil &amp; Maria Valicenti-McDermott</p></li><li id="Aff37"><p>Department of Psychiatry, Weill Cornell Medicine, White Plains, NY, USA</p><p>Anna Marie Paolicelli, Michelle Heyman &amp; Shanping Qiu</p></li><li id="Aff38"><p>Department of Pediatrics, Medical University of South Carolina, Charleston, SC, USA</p><p>Anna Rhea, Audrey Ward, Brandi Bell, Catherine C. Bradley, Caitlin McCarthy, Crissy Ortiz, Casey Roche, Clara Shrier, Candace Van Wade, Erin Doyle, Harper Richardson, Jessie Montezuma, Laura Carpenter, Sarah Conyers, Sophia D’Ambrosi &amp; Virginia Galbraith</p></li><li id="Aff39"><p>Department of Pediatrics, Texas Children’s Hospital (Baylor College of Medicine), Houston, TX, USA</p><p>Andrea Simon, Danica Limon, Gabrielle Duhon, Gabriela Marzano, Jessica Orobio, Kelli Baalman &amp; Robin P. Goin-Kochel</p></li><li id="Aff40"><p>Department of Medicine, Boston Children’s Hospital, Boston, MA, USA</p><p>Aubrie Soucy, Ryan N. Doan &amp; Timothy W. Yu</p></li><li id="Aff41"><p>MIND Institute and Department of Psychiatry and Behavioral Sciences, University of California, Davis, Sacramento, CA, USA</p><p>Brittani A. Hilscher, Claudine Anglo, David G. Amaral, Deana Li, Kaitlin Smith, Leonard Abbeduto, Nicki Rodriguez &amp; Samantha Thompson</p></li><li id="Aff42"><p>Children’s Specialized Hospital, Toms River, NJ, USA</p><p>Barbara Enright, Jill F. Harris, Malia Beckwith, Myriam Casseus, Marilyn Lopez &amp; Natalia Gonzalez</p></li><li id="Aff43"><p>Department of Psychiatry, University of North Carolina (UNC, TEACCH, CIDD), Chapel Hill, NC, USA</p><p>Brenda Hauf, Catherine Gray, Corrie H. Walston, Elena Lamarche, Gabriel S. Dichter, Joseph Piven, Kailey Murray, Kelli Real, Mary Currin, Michelle Jordy &amp; Renee D. Clark</p></li><li id="Aff44"><p>Department of Psychiatry and Behavioral Sciences, Emory University and Marcus Autism Center, Atlanta, GA, USA</p><p>Catherine E. Rice, Joseph F. Cubells, Michael J. Morrier &amp; Opal Y. Ousley</p></li><li id="Aff45"><p>Department of Pediatrics, Emory University and Marcus Autism Center, Atlanta, GA, USA</p><p>Chris Gunter, Cheryl Klaiman, Megan Dunlevy, Sheena Mathai &amp; Stormi White</p></li><li id="Aff46"><p>Geisinger Autism &amp; Developmental Medicine Institute, Lewisburg, PA, USA</p><p>Christa Lese Martin, Cora M. Taylor, Kate Dent, Kaitlyn Singer, Lauren Kasperson Walsh &amp; William Curtis Weaver</p></li><li id="Aff47"><p>Department of Pediatrics, Rush University Medical Center, Chicago, IL, USA</p><p>Cesar Ochoa-Lubinoff, Elizabeth Berry-Kravis, Hai Li, Josh Beeson, Kelsey Gonring, Lorena Ferreira Corzo, Melissa Baer, Peter Heydemann &amp; Thao Tran</p></li><li id="Aff48"><p>Department of Pediatrics, University of Mississippi Medical Center, Jackson, MS, USA</p><p>Dustin E. Sarver, Danielle Stamps, Kristen Callahan, Lacy Malloch, Robert D. Annett &amp; Sabrina White</p></li><li id="Aff49"><p>Department of Psychiatry and Behavioral Sciences, University of Washington/Seattle Children’s Autism Center, Seattle, WA, USA</p><p>Daniel Cho, Emily A. Fox, Haidar Ghina, Jennifer A. Gerdts, Katie Ahlers, Raphael A. Bernier, Rachel K. Earl, Theodore Ho &amp; Tara Rutter</p></li><li id="Aff50"><p>Department of Psychology, University of Michigan, Ann Arbor, MI, USA</p><p>Ivy F. Tso</p></li><li id="Aff51"><p>Center for Autism Research, Children’s Hospital of Philadelphia, Philadelphia, PA, USA</p><p>Juhi Pandey, Lindsey DeMarco, Robert T. Schultz, Susannah Horner, Samiza Palmer, Samantha Plate &amp; Vaikunt Ranganathan</p></li><li id="Aff52"><p>Department of Molecular Physiology and Biophysics, Vanderbilt University, Nashville, TN, USA</p><p>James S. Sutcliffe</p></li><li id="Aff53"><p>Department of Psychiatry, Columbia University Medical Center, New York, NY, USA</p><p>Jeremy Veenstra-Vanderweele</p></li><li id="Aff54"><p>Department of Psychiatry and Behavioral Sciences, Medical University of South Carolina, Charleston, SC, USA</p><p>McLeod F. Gwynette</p></li><li id="Aff55"><p>Department of Neurology, Boston Children’s Hospital, Boston, MA, USA</p><p>Mustafa Sahin</p></li><li id="Aff56"><p>Maine Medical Center Research Institute, Scarborough, ME, USA</p><p>Matthew Siegel, Mary Verdi &amp; Susan Santangelo</p></li><li id="Aff57"><p>Genome Center, MIND Institute, Department of Biochemistry and Molecular Medicine, University of California, Davis, Sacramento, CA, USA</p><p>Megan Y. Dennis</p></li><li id="Aff58"><p>Translational Neuroscience Center, Boston Children’s Hospital, Boston, MA, USA</p><p>Stephanie J. Brewster</p></li><li id="Aff59"><p>Center for Autism and the Developing Brain (CADB), Weill Cornell Medicine, White Plains, NY, USA</p><p>So Hyun &amp; Y. B. Choi</p></li><li id="Aff60"><p>Department of Health Psychology, University of Missouri, Columbia, MO, USA</p><p>Stephen M. Kanne</p></li><li id="Aff61"><p>Greenwood Genetic Center, Greenwood, SC, USA</p><p>Steve Skinner</p></li><li id="Aff62"><p>Department of Pediatrics, University of California, San Diego and SARRC Phoenix, La Jolla, CA, USA</p><p>Vahid Gazestani</p></li></ol><h3 id="groups">Consortia</h3><div id="group-1"><h3>The SPARK Consortium</h3><ul><li>Adrienne Adams</li><li>, Alpha Amatya</li><li>, Alicia Andrus</li><li>, Asif Bashar</li><li>, Anna Berman</li><li>, Alison Brown</li><li>, Alexies Camba</li><li>, Amanda C. Gulsrud</li><li>, Anthony D. Krentz</li><li>, Amanda D. Shocklee</li><li>, Amy Esler</li><li>, Alex E. Lash</li><li>, Anne Fanta</li><li>, Ali Fatemi</li><li>, Angela Fish</li><li>, Alexandra Goler</li><li>, Antonio Gonzalez</li><li>, Anibal Gutierrez Jr.</li><li>, Antonio Hardan</li><li>, Amy Hess</li><li>, Anna Hirshman</li><li>, Alison Holbrook</li><li>, Andrea J. Ace</li><li>, Anthony J. Griswold</li><li>, Angela J. Gruber</li><li>, Andrea Jarratt</li><li>, Anna Jelinek</li><li>, Alissa Jorgenson</li><li>, A. Pablo Juarez</li><li>, Annes Kim</li><li>, Alex Kitaygorodsky</li><li>, Addie Luo</li><li>, Angela L. Rachubinski</li><li>, Allison L. Wainer</li><li>, Amy M. Daniels</li><li>, Anup Mankar</li><li>, Andrew Mason</li><li>, Alexandra Miceli</li><li>, Anna Milliken</li><li>, Amy Morales-Lara</li><li>, Alexandra N. Stephens</li><li>, Ai Nhu Nguyen</li><li>, Amy Nicholson</li><li>, Anna Marie Paolicelli</li><li>, Alexander P. McKenzie</li><li>, Abha R. Gupta</li><li>, Ashley Raven</li><li>, Anna Rhea</li><li>, Andrea Simon</li><li>, Aubrie Soucy</li><li>, Amy Swanson</li><li>, Anthony Sziklay</li><li>, Amber Tallbull</li><li>, Angela Tesng</li><li>, Audrey Ward</li><li>, Allyson Zick</li><li>, Brittani A. Hilscher</li><li>, Brandi Bell</li><li>, Barbara Enright</li><li>, Beverly E. Robertson</li><li>, Brenda Hauf</li><li>, Bill Jensen</li><li>, Brandon Lobisi</li><li>, Brianna M. Vernoia</li><li>, Brady Schwind</li><li>, Bonnie VanMetre</li><li>, Craig A. Erickson</li><li>, Catherine A. W. Sullivan</li><li>, Charles Albright</li><li>, Claudine Anglo</li><li>, Cate Buescher</li><li>, Catherine C. Bradley</li><li>, Claudia Campo-Soria</li><li>, Cheryl Cohen</li><li>, Costanza Colombi</li><li>, Chris Diggins</li><li>, Catherine Edmonson</li><li>, Catherine E. Rice</li><li>, Carrie Fassler</li><li>, Catherine Gray</li><li>, Chris Gunter</li><li>, Corrie H. Walston</li><li>, Cheryl Klaiman</li><li>, Caroline Leonczyk</li><li>, Christa Lese Martin</li><li>, Catherine Lord</li><li>, Cora M. Taylor</li><li>, Caitlin McCarthy</li><li>, Cesar Ochoa-Lubinoff</li><li>, Crissy Ortiz</li><li>, Cynthia Pierre</li><li>, Cordelia R. Rosenberg</li><li>, Chris Rigby</li><li>, Casey Roche</li><li>, Clara Shrier</li><li>, Chris Smith</li><li>, Candace Van Wade</li><li>, Casey White-Lehman</li><li>, Christopher Zaro</li><li>, Cindy Zha</li><li>, Dawn Bentley</li><li>, Dahriana Correa</li><li>, Dustin E. Sarver</li><li>, David Giancarla</li><li>, David G. Amaral</li><li>, Dain Howes</li><li>, Dalia Istephanous</li><li>, Daniel Lee Coury</li><li>, Deana Li</li><li>, Danica Limon</li><li>, Desi Limpoco</li><li>, Diamond Phillips</li><li>, Desiree Rambeck</li><li>, Daniela Rojas</li><li>, Diksha Srishyla</li><li>, Danielle Stamps</li><li>, Dennis Vasquez Montes</li><li>, Daniel Cho</li><li>, Dave Cho</li><li>, Emily A. Fox</li><li>, Ethan Bahl</li><li>, Elizabeth Berry-Kravis</li><li>, Elizabeth Blank</li><li>, Erin Bower</li><li>, Elizabeth Brooks</li><li>, Eric Courchesne</li><li>, Emily Dillon</li><li>, Erin Doyle</li><li>, Erin Given</li><li>, Ellen Grimes</li><li>, Erica Jones</li><li>, Eric J. Fombonne</li><li>, Elizabeth Kryszak</li><li>, Ericka L. Wodka</li><li>, Elena Lamarche</li><li>, Erica Lampert</li><li>, Eric M. Butter</li><li>, Eirene O’Connor</li><li>, Edith Ocampo</li><li>, Elizabeth Orrick</li><li>, Esmeralda Perez</li><li>, Elizabeth Ruzzo</li><li>, Emily Singer</li><li>, Emily T. Matthews</li><li>, Ernest V. Pedapati</li><li>, Faris Fazal</li><li>, Fiona K. Miller</li><li>, Gabriella Aberbach</li><li>, Gabriele Baraghoshi</li><li>, Gabrielle Duhon</li><li>, Gregory Hooks</li><li>, Gregory J. Fischer</li><li>, Gabriela Marzano</li><li>, Gregory Schoonover</li><li>, Gabriel S. Dichter</li><li>, Gabrielle Tiede</li><li>, Hannah Cottrell</li><li>, Hannah E. Kaplan</li><li>, Haidar Ghina</li><li>, Hanna Hutter</li><li>, Hope Koene</li><li>, Hoa Lam Schneider</li><li>, Holly Lechniak</li><li>, Hai Li</li><li>, Hadley Morotti</li><li>, Hongjian Qi</li><li>, Harper Richardson</li><li>, Hana Zaydens</li><li>, Haicang Zhang</li><li>, Haoquan Zhao</li><li>, Ivette Arriaga</li><li>, Ivy F. Tso</li><li>, John Acampado</li><li>, Jennifer A. Gerdts</li><li>, Josh Beeson</li><li>, Jennylyn Brown</li><li>, Joaquin Comitre</li><li>, Jeanette Cordova</li><li>, Jennifer Delaporte</li><li>, Joseph F. Cubells</li><li>, Jill F. Harris</li><li>, Jared Gong</li><li>, Jaclyn Gunderson</li><li>, Jessica Hernandez</li><li>, Jessyca Judge</li><li>, Jane Jurayj</li><li>, J. Kiely Law</li><li>, Julie Manoharan</li><li>, Jessie Montezuma</li><li>, Jason Neely</li><li>, Jessica Orobio</li><li>, Juhi Pandey</li><li>, Joseph Piven</li><li>, Jose Polanco</li><li>, Jibrielle Polite</li><li>, Jacob Rosewater</li><li>, Jessica Scherr</li><li>, James S. Sutcliffe</li><li>, James T. McCracken</li><li>, Jennifer Tjernagel</li><li>, Jaimie Toroney</li><li>, Jeremy Veenstra-Vanderweele</li><li>, Jiayao Wang</li><li>, Katie Ahlers</li><li>, Kathryn A. Schweers</li><li>, Kelli Baalman</li><li>, Katie Beard</li><li>, Kristen Callahan</li><li>, Kendra Coleman</li><li>, Kate D. Fitzgerald</li><li>, Kate Dent</li><li>, Katharine Diehl</li><li>, Kelsey Gonring</li><li>, Katherine G. Pawlowski</li><li>, Kathy Hirst</li><li>, Karen L. Pierce</li><li>, Karla Murillo</li><li>, Kailey Murray</li><li>, Kerri Nowell</li><li>, Kaela O’Brien</li><li>, Katrina Pama</li><li>, Kelli Real</li><li>, Kaitlyn Singer</li><li>, Kaitlin Smith</li><li>, Kevin Stephenson</li><li>, Katherine Tsai</li><li>, Leonard Abbeduto</li><li>, Lindsey A. Cartner</li><li>, Landon Beeson</li><li>, Laura Carpenter</li><li>, Lucas Casten</li><li>, Leigh Coppola</li><li>, Lisa Cordiero</li><li>, Lindsey DeMarco</li><li>, Lillian D. Pacheco</li><li>, Lorena Ferreira Corzo</li><li>, Lisa H. Shulman</li><li>, Lauren Kasperson Walsh</li><li>, Laurie Lesher</li><li>, Lynette M. Herbert</li><li>, Lisa M. Prock</li><li>, Lacy Malloch</li><li>, Lori Mann</li><li>, Luke P. Grosvenor</li><li>, Laura Simon</li><li>, Latha V. Soorya</li><li>, Lucy Wasserburg</li><li>, Lisa Yeh</li><li>, Lark Y. Huang-Storms</li><li>, Michael Alessandri</li><li>, Marc A. Popp</li><li>, Melissa Baer</li><li>, Malia Beckwith</li><li>, Myriam Casseus</li><li>, Michelle Coughlin</li><li>, Mary Currin</li><li>, Michele Cutri</li><li>, Malcolm D. Mallardi</li><li>, Megan DuBois</li><li>, Megan Dunlevy</li><li>, Martin E. Butler</li><li>, Margot Frayne</li><li>, McLeod F. Gwynette</li><li>, Mohammad Ghaziuddin</li><li>, Monica Haley</li><li>, Michelle Heyman</li><li>, Margaret Hojlo</li><li>, Michelle Jordy</li><li>, Michael J. Morrier</li><li>, Misia Kowanda</li><li>, Melinda Koza</li><li>, Marilyn Lopez</li><li>, Megan McTaggart</li><li>, Megan Norris</li><li>, Melissa N. Hale</li><li>, Molly O’Neil</li><li>, Madison Printen</li><li>, Madelyn Rayos</li><li>, Mahfuza Sabiha</li><li>, Mustafa Sahin</li><li>, Marina Sarris</li><li>, Mojeeb Shir</li><li>, Matthew Siegel</li><li>, Morgan Steele</li><li>, Megan Sweeney</li><li>, Maira Tafolla</li><li>, Maria Valicenti-McDermott</li><li>, Mary Verdi</li><li>, Megan Y. Dennis</li><li>, Nicolas Alvarez</li><li>, Nicole Bardett</li><li>, Natalie Berger</li><li>, Norma Calderon</li><li>, Nickelle Decius</li><li>, Natalia Gonzalez</li><li>, Nina Harris</li><li>, Noah Lawson</li><li>, Natasha Lillie</li><li>, Nathan Lo</li><li>, Nancy Long</li><li>, Nicole M. Russo-Ponsaran</li><li>, Natalie Madi</li><li>, Nicole Mccoy</li><li>, Natalie Nagpal</li><li>, Nicki Rodriguez</li><li>, Nicholas Russell</li><li>, Neelay Shah</li><li>, Nicole Takahashi</li><li>, Nicole Targalia</li><li>, Olivia Newman</li><li>, Opal Y. Ousley</li><li>, Peter Heydemann</li><li>, Patricia Manning</li><li>, Paul S. Carbone</li><li>, Raphael A. Bernier</li><li>, Rachel A. Gordon</li><li>, Rebecca C. Shaffer</li><li>, Robert D. Annett</li><li>, Renee D. Clark</li><li>, Roger Jou</li><li>, Rebecca J. Landa</li><li>, Rachel K. Earl</li><li>, Robin Libove</li><li>, Richard Marini</li><li>, Ryan N. Doan</li><li>, Robin P. Goin-Kochel</li><li>, Rishiraj Rana</li><li>, Richard Remington</li><li>, Roman Shikov</li><li>, Robert T. Schultz</li><li>, Shelley Aberle</li><li>, Shelby Birdwell</li><li>, Sarah Boland</li><li>, Stephanie Booker</li><li>, S. Carpenter</li><li>, Sharmista Chintalapalli</li><li>, Sarah Conyers</li><li>, Sophia D’Ambrosi</li><li>, Sara Eldred</li><li>, Sunday Francis</li><li>, Swami Ganesan</li><li>, Susan Hepburn</li><li>, Susannah Horner</li><li>, Samantha Hunter</li><li>, Stephanie J. Brewster</li><li>, Soo J. Lee</li><li>, Suma Jacob</li><li>, Stanley Jean</li><li>, So Hyun</li><li>, Sydney Kramer</li><li>, Sandra L. Friedman</li><li>, Sarely Licona</li><li>, Sandy Littlefield</li><li>, Stephen M. Kanne</li><li>, Sarah Mastel</li><li>, Sheena Mathai</li><li>, Sophia Melnyk</li><li>, Sarah Michaels</li><li>, Sarah Mohiuddin</li><li>, Samiza Palmer</li><li>, Samantha Plate</li><li>, Shanping Qiu</li><li>, Shelley Randall</li><li>, Sophia Sandhu</li><li>, Susan Santangelo</li><li>, Swapnil Shah</li><li>, Steve Skinner</li><li>, Samantha Thompson</li><li>, Sabrina White</li><li>, Stormi White</li><li>, Sabrina Xiao</li><li>, Sidi Xu</li><li>, Simon Xu</li><li>, Tia Chen</li><li>, Tunisia Greene</li><li>, Theodore Ho</li><li>, Teresa Ibanez</li><li>, Tanner Koomar</li><li>, Tiziano Pramparo</li><li>, Tara Rutter</li><li>, Tamim Shaikh</li><li>, Thao Tran</li><li>, Timothy W. Yu</li><li>, Virginia Galbraith</li><li>, Vahid Gazestani</li><li>, Vincent J. Myers</li><li>, Vaikunt Ranganathan</li><li>, Vini Singh</li><li>, William Curtis Weaver</li><li>, Wenteng CaI</li><li>, Wubin Chin</li><li>, Wha S. Yang</li><li>, Y. B. Choi</li><li> &amp; Zachary E. Warren</li></ul></div><h3 id="contributions">Contributions</h3><p>I. Astrovskaya, J.B.H., J.J.M., N.V., P.F., C. Shu, T.W., W.K.C., X.Z. and Y.S. designed and conceived this study. A. Adams, A. Andrus, A. Berman, A. Brown, A.C., A.C.G., A.D.S., A.E., A. Fanta, A. Fatemi, A. Fish, A. Goler, A. Gonzalez, A. Gutierrez Jr., A. Hardan, A. Hess, A. Hirshman, A. Holbrook, A.J.A., A.J. Griswold, A. Jarratt, A. Jelinek, A. Jorgenson, A. Juarez, A. Kim, A. Kitaygorodsky, A.L., A.L.R., A.L.W., A.M.D., A. Mankar, A. Mason, A. Miceli, A. Milliken, A.M.-L., A.N.S., A. Nguyen, A. Nicholson, A. Nishida, A.P., A.P.M., A.R.G., A. Raven, A. Rhea, A. Simon, A. Swanson, A. Sziklay, A. Tallbull, A. Tesng, A.W., A.Z., B.A.H., B.B., B.E., B.E.R., B. Hauf, B.J.O., B.L., B.M.V., B.S., B.V., C.A.E., C.A.W.S., C. Albright, C. Anglo, C.B., C.C.B., C.C.-S., C. Cohen, C. Colombi, C.D., C.E., C.E.R., C. Fassler, C. Gray, C. Gunter, C.H.W., C.K., C. Leonczyk, C.L.M., C. Lord, C.M.T., C.M., C.O.-L., C. Ortiz, C.P., C.R.R., C. Roche, C. Shrier, C. Smith, C.V., C.W.-L., C. Zaro, C. Zha, D.B., Daniel Cho, D. Correa, D.E.S., D.G., D.G.A., D.H., D.I., D.L.C., D. Li, D. Limon, D. Limpoco, D.P., D. Rambeck, D. Rojas, D. Srishyla, D. Stamps, E.A.F., E. Bahl, E.B.-K., E. Blank, E. Bower, E. Brooks, E.C., E. Dillon, E. Doyle, E. Given, E. Grimes, E.J., E.J.F., E.K., E.L.W., E. Lamarche, E. Lampert, E.M.B., E. O’Connor, E. Ocampo, E. Orrick, E.P., E.R., E.S., E.T.M., E.V.P., F.F., F.K.M., G.A., G.B., G.D., G.H., G.M., G.S., G.S.D., G.T., H.C., H.E.K., H.G., H.H., H.K., H.L.S., H. Lechniak, H. Li, H.M., H.R., H. Zaydens, I. Arriaga, I.F.T., J.A., J.A.G., J. Beeson, J. Brown, J. Comitre, J. Cordova, J.D., J.F.C., J.F.H., J. Gong, J. Gunderson, J.H., J.J.M., J. Judge, J. Jurayj, J.K.L., J. Manoharan, J. Montezuma, J.N., J.O., J. Pandey, J. Piven, J. Polanco, J. Polite, J.R., J.R.W., J.S., J.S.S., J.T.M., J. Tjernagel, J. Toroney, J.V.-V., J.W., K.A., K.A.S., K. Baalman, K. Beard, K. Callahan, K. Coleman, K.D.F., K. Dent, K. Diehl, K.G., K.G.P., K.H., K.L.P., K. Murillo, K. Murray, K.N., K.O., K. Pama, K.R., K. Singer, K. Smith, K. Stephenson, K.T., L.A., L.A.C., L. Beeson, L. Carpenter, L. Casten, L. Coppola, L. Cordiero, L.D., L.D.P., L.F.C., L.G.S., L.H.S., L.K.W., L.L., L.M.H., L.M.P., L. Malloch, L. Mann, L.P.G., L.S., L.V.S., L.W., L.Y., L.Y.-H., M.A., M. Baer, M. Beckwith, M. Casseus, M. Coughlin, M. Currin, M. Cutri, M. DuBois, M. Dunlevy, M.F., M.F.G., M.G., M. Haley, M. Heyman, M. Hojlo, M.J., M.J.M., M. Kowanda, M. Koza, M.L., M.M., M.N., M.N.H., M.O., M.P., M.R., M. Sabiha, M. Sahin, M. Sarris, M. Shir, M. Siegel, M. Steele, M. Sweeney, M.T., M.V.-M., M. Verdi, M.Y.D., N.A., N. Bardett, N. Berger, N.C., N.D., N.G., N.H., N. Lillie, N. Long, N.M.R.-P., N. Madi, N. Mccoy, N.N., N. Rodriguez, N. Russell, N.S., N. Takahashi, N. Targalia, N.V., O.N., O.Y.O., P.F., P.H., P.M., P.S.C., R.A.B., R.A.G., R.C.S., R.D.A., R.D.C., R.J., R.J.L., R.K.E., R.L., R.P.G.-K., R. Remington, R.S., R.T.S., S.A., S. Birdwell, S. Boland, S. Booker, S. Carpenter, S. Chintalapalli, S. Conyers, S.D., S.D.B., S.E., S.F., S.G., S. Hepburn, S. Horner, S. Hunter, S.J.B., S.J.L., S. Jacob, S. Jean, S. Kim, S. Kramer, S.L.F., S. Licona, S. Littlefield, S.M.K., S. Mastel, S. Mathai, S. Melnyk, S. Michaels, S. Mohiuddin, S. Palmer, S. Plate, S.Q., S.R., S. Sandhu, S. Santangelo, S. Skinner, S.T., S. Xu, S. Xiao, Sabrina White, Stormi White, T.C., T.G., T.H., T.I., T.K., T.P., T.R., T.S., T.R.T., T.T., V. Galbraith, V. Gazestani, V.J.M., V.R., V.S., W.C.W., W. Cal, W.K.C., W.S.Y., Y.C. and Z.E.W. recruited participants and collected clinical data and biospecimens. A. Amatya, A. Bashar, A.E.L., A. Mankar, A. Nguyen, B.J., C. Rigby, Dave Cho, D.V.M., E. O’Connor, J.A., M.D.M., M.E.B., N. Lawson, N. Lo, N.V., R.M., R. Rana, S.G., S. Jean, S. Shah and W. Chin built and supported the SPARKforAutism.org website, software, databases and systems, and managed SPARK data. A.D.K., A.J. Gruber, A. Nishida, B. Han, B.J.O., C. Fleisch, C. Shu, D.V.M., E. Brooks, G.J.F., I. Astrovskaya, J.B.H., J.J.M., J.R.W., J.U.O., L. Brueggeman, L.G.S., M.A.P., N. Lo, N.V., O.M., P.F., S.D.B., S.C.M., S.X.X., T.N.T., T.S.C., T.W., W.T.H., X.Z. and Y.S. performed analyses, processed biospecimens and sequenced DNA samples. A. Fatemi, A. Kitaygorodsky, A. Soucy, C. Shu, D.H.G., E.B.-K., E.E.E., E.R., H.Q., H. Zhang, H. Zhao, I. Astrovskaya, J.B.H., J.J.M., J.R.W., J.U.O., L. Brueggeman, L.G.S., M.Y.D., N.V., O.M., P.F., R.N.D., S.D.B., S.C.M., S.X.X., T.K., T.N.T., T.P., T.S., T.R.T., T.W., T.W.Y., W.K.C., X.Z. and Y.S. helped with data interpretation. A.E.L., E.E.E., J.J.M., N.V., P.F., W.K.C. and X.Z. supervised the work. B.J.O., I. Astrovskaya, J.B.H., J.J.M., J.U.O., C. Shu, J.R.W., N.V., P.F., T.N.T., T.W., W.K.C., X.Z. and Y.S. wrote this paper.</p><h3 id="corresponding-author">Corresponding author</h3><p id="corresponding-author-list">Correspondence to
                <a id="corresp-c1" href="mailto:wkc15@columbia.edu">Wendy K. Chung</a>.</p></div></div></section><section data-title="Ethics declarations"><div id="ethics-section"><h2 id="ethics">Ethics declarations</h2><div id="ethics-content">
            
              <h3 id="FPar4">Competing interests</h3>
              <p>D.H.G. has received consulting fees or equity participation for scientific advisory board work from Ovid Therapeutics, Axial Biotherapeutics, Acurastem, and Falcon Computing. E.E.E. serves on the Scientific Advisory Board of Variant Bio. W.K.C. serves on Scientific Advisory Board of the Regeneron Genetics Center and is the Director of Clinical Research for SFARI. All other authors declare no competing interests.</p>
            
          </div></div></section><section data-title="Peer review"><div id="peer-review-section"><h2 id="peer-review">Peer review</h2><div id="peer-review-content">
            
            
              <h3 id="FPar3">Peer review information</h3>
              <p><i>Nature Genetics</i> thanks Anders Borglum and Xin He for their contribution to the peer review of this work. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41588-022-01148-2#MOESM3">Peer reviewer reports</a> are available.</p>
            
          </div></div></section><section data-title="Additional information"><div id="additional-information-section"><h2 id="additional-information">Additional information</h2><p><b>Publisher’s note</b> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div></section><section data-title="Extended data"><div id="Sec28-section"><h2 id="Sec28">Extended data</h2><div id="Sec28-content"><div data-test="supplementary-info"><div data-test="supp-item" id="Fig8"><h3><a data-track="click" data-track-action="view supplementary info" data-track-label="link" data-test="supp-info-link" href="https://www.nature.com/articles/s41588-022-01148-2/figures/8" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41588-022-01148-2/MediaObjects/41588_2022_1148_Fig8_ESM.jpg">Extended Data Fig. 1 Overall burden of de novo variants in four ASD cohorts included in the discovery sample.</a></h3><p>(A) Observed rates of de novo LoF, Dmis (REVEL &gt; = 0.5) and silent variants (B) are compared with expected rates. We used a 7mer sequence context dependent mutation rate model<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 55" title="Carlson, J. et al. Extremely rare variants reveal patterns of germline mutation rate heterogeneity in humans. Nat. Commun. 9, 3753 (2018)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR55" id="ref-link-section-d6323361e10003">55</a></sup> to calculate expected rates for different classes of de novo variants after adjusting sequencing coverage, and found a close match with observed de novo rates in control trios. The rates of de novo LoF and Dmis variants in ASD cases are significantly higher than baseline expectation and are reduced in cases with known family history. Data are presented as mean values and 95% confidence interval as error bars.</p></div><div data-test="supp-item" id="Fig9"><h3><a data-track="click" data-track-action="view supplementary info" data-track-label="link" data-test="supp-info-link" href="https://www.nature.com/articles/s41588-022-01148-2/figures/9" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41588-022-01148-2/MediaObjects/41588_2022_1148_Fig9_ESM.jpg">Extended Data Fig. 2 Enrichment of de novo damaging and rare, inherited LoF variants in ASD cases across gene sets.</a></h3><p>Gene sets were defined and grouped by transcriptome proteome, neuronal regulome, ASD gene prediction scores, genetic evidence from neuropsychiatric diseases, and gene level constraint. Analyses were repeated after removing known ASD/NDD genes. Number of genes in each set before and after removing known genes are shown in bracket below gene set. Dots represent fold enrichment of DNVs or odds ratio for over-transmission of LoFs in each set. Horizontal bars indicate 95% confidence interval. For each gene set, also shown are the percentage of excessive DNVs in cases and percentage of over-transmission of rare, inherited LoFs to cases. (A) De novo enrichment analysis was performed by dnEnrich that conditional on the overall increase in burden of de novo damaging variants in cases compared with controls (<a data-track="click" data-track-label="link" data-track-action="section anchor" href="https://www.nature.com/articles/s41588-022-01148-2#Sec11">Methods</a>). P-values were derived from 100,000 random permutations of de novo damaging variants among all 5,754 constrained genes and accounts for the tri-nucleotide sequence context and gene length. (B) Enrichment of rare, inherited LoFs was evaluated by comparing the transmission and non-transmission of ultra-rare LoFs with pExt &gt; =0.1 in the gene set versus those in all other constrained genes using a 2-by-2 table. P-values were given by chi-squared test.</p></div><div data-test="supp-item" id="Fig10"><h3><a data-track="click" data-track-action="view supplementary info" data-track-label="link" data-test="supp-info-link" href="https://www.nature.com/articles/s41588-022-01148-2/figures/10" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41588-022-01148-2/MediaObjects/41588_2022_1148_Fig10_ESM.jpg">Extended Data Fig. 3 QQ plot showing the ultra-rare synonymous burden test among 404 selected genes between SPARK cases and gnomAD controls for allele frequency&lt;1e-5.</a></h3><p><i>HMCN2</i> is excluded (not shown) since it has poor coverage in gnomAD. Panel A shows the cross-ancestry case-control ultra-rare synonymous burden comparison, while Panel B shows the European-only case-control ultra-rare synonymous burden comparison. The observed P values for each gene are sorted from largest to smallest and plotted against expected values from a theoretical chi-square distribution.</p></div><div data-test="supp-item" id="Fig11"><h3><a data-track="click" data-track-action="view supplementary info" data-track-label="link" data-test="supp-info-link" href="https://www.nature.com/articles/s41588-022-01148-2/figures/11" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41588-022-01148-2/MediaObjects/41588_2022_1148_Fig11_ESM.jpg">Extended Data Fig. 4 Transmission disequilibrium of exonic or single gene deletions of ITSN1 (A) and NAV3 (B).</a></h3><p>The read depth signal plots show normalized read depth (NDP) of exome targets used in CNV calling by CLAMMS<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 84" title="Packer, J. S. et al. CLAMMS: a scalable algorithm for calling common and rare copy number variants from exome sequencing data. Bioinformatics 32, 133–135 (2016)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR84" id="ref-link-section-d6323361e10084">84</a></sup> for <i>ITSN1</i> (A) and <i>NAV3</i> (B) in Family 1–8. NDPs of −0.5, 0 and 0.5 correspond to copy number (CN) of 1 (deletion), 2 (normal diploid) and 3 (duplication). NDPs of exon targets within deleted regions are colored red. Dark and gray areas correspond to 1 and 2 estimated standard deviations of NDP for each exon target. CN deletions were initially discovered from all unaffected parents, then subsequently genotyped on Family 1–8. Signal plots of Family 1–8 are shown with parents appear in the top and offspring in the bottom. The deletion regions and affected exons of genes are shown below each plot.</p></div><div data-test="supp-item" id="Fig12"><h3><a data-track="click" data-track-action="view supplementary info" data-track-label="link" data-test="supp-info-link" href="https://www.nature.com/articles/s41588-022-01148-2/figures/12" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41588-022-01148-2/MediaObjects/41588_2022_1148_Fig12_ESM.jpg">Extended Data Fig. 5 Cumulative distribution of haploid mutation rates (per generation) of LoF and D-mis (REVEL &gt; = 0.5) variants of all protein coding genes on autosomes.</a></h3><p>Panel A shows the distribution of all autosomal genes vs. known ASD/NDD genes. Panel B shows all autosomal genes vs. prioritized genes. Baseline mutation rates were calculated using 7mer sequence context dependent mutation rates<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 55" title="Carlson, J. et al. Extremely rare variants reveal patterns of germline mutation rate heterogeneity in humans. Nat. Commun. 9, 3753 (2018)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR55" id="ref-link-section-d6323361e10119">55</a></sup>. Known ASD/NDD genes tend to have higher mutation rates than average genes.</p></div><div data-test="supp-item" id="Fig13"><h3><a data-track="click" data-track-action="view supplementary info" data-track-label="link" data-test="supp-info-link" href="https://www.nature.com/articles/s41588-022-01148-2/figures/13" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41588-022-01148-2/MediaObjects/41588_2022_1148_Fig13_ESM.jpg">Extended Data Fig. 6 Power of case-control association by rare LoFs variants (‘mega-analysis’) with sample size equal to current study.</a></h3><p>The mega-analysis of current study compares the rate of LoF variants in 32,024 unrelated ASD cases with population controls with sample sizes about 76,000~132,000. For power calculation, we assumed that population controls are infinite so that cumulative allele frequency are known and presumed to be at equilibrium under selection-mutation balance for constrained genes (<i>f</i> = <i>μ</i><sub>LoF</sub>/<i>s</i>). Experiment-wide error rate was set at 9e-6 (0.05 divided by the number of autosomal genes at gnomAD LOEUF 30%). Power is calculated as a function of relative risk for ASD (<i>RR</i>) and selection coefficient (<i>s</i>) across different haploid LoF mutation rates (<i>μ</i><sub><i>LoF</i></sub>) using an analytic approximation by Zuk <i>et al</i>.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 41" title="Zuk, O. et al. Searching for missing heritability: designing rare variant association studies. Proc. Natl Acad. Sci. USA 111, E455–E464 (2014)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR41" id="ref-link-section-d6323361e10173">41</a></sup>. We only considered selection coefficient between 0.01 and 0.5 and relative risk to ASD between 1 and 20, because genes with huge effect sizes and larger selection coefficients are expected to be identified from the enrichment of de novo variants. The triangular region where <i>s</i> &lt; 0.013<i>RR</i> are left blank because the parameters in this region are not compatible with the current estimates of prevalence of ASD (1/54)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 71" title="Maenner, M. J. et al. Prevalence of autism spectrum disorder among children aged 8 years — Autism and Developmental Disabilities Monitoring Network, 11 Sites, United States, 2016. MMWR Surveill. Summ. 69, 1–12 (2020)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR71" id="ref-link-section-d6323361e10184">71</a></sup> and sex-averaged reduction of reproductive fitness (0.71)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 72" title="Power, R. A. et al. Fecundity of patients with schizophrenia, autism, bipolar disorder, depression, anorexia nervosa, or substance abuse vs their unaffected siblings. JAMA Psychiatry 70, 22–30 (2013)." href="https://www.nature.com/articles/s41588-022-01148-2#ref-CR72" id="ref-link-section-d6323361e10188">72</a></sup>. Five new ASD genes identified in this study are placed onto the heatmap closest to its LoF mutation rate. Their positions within heatmaps are taken from point estimates of using gnomAD exomes (non-neuro subset) as population controls.</p></div><div data-test="supp-item" id="Fig14"><h3><a data-track="click" data-track-action="view supplementary info" data-track-label="link" data-test="supp-info-link" href="https://www.nature.com/articles/s41588-022-01148-2/figures/14" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41588-022-01148-2/MediaObjects/41588_2022_1148_Fig14_ESM.jpg">Extended Data Fig. 7 Sample sizes required for achieving 90% of power.</a></h3><p>Using the same assumptions and experiment-wide error rate as power calculation, we calculated required sample size for 90% of power. Sample size is shown as a factor relative to the current sample size (32,024) and as a function of relative risk to ASD and selection coefficient across with different LoF mutation rates. Contours 1, 2 and 5 times of current size are shown as dashed lines. Regions of parameter space that require over 10 times current sample sizes are shown in gray.</p></div></div></div></div></section><section data-title="Supplementary information"><div id="Sec29-section"><h2 id="Sec29">Supplementary information</h2></div></section><section data-title="Rights and permissions"><div id="rightslink-section"><h2 id="rightslink">Rights and permissions</h2><div id="rightslink-content">
              <p><b>Open Access</b>  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this license, visit <a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>.</p>
            <p><a data-track="click" data-track-action="view rights and permissions" data-track-label="link" href="https://s100.copyright.com/AppDispatchServlet?title=Integrating%20de%20novo%20and%20inherited%20variants%20in%2042%2C607%20autism%20cases%20identifies%20mutations%20in%20new%20moderate-risk%20genes&amp;author=Xueya%20Zhou%20Adrienne%20Adams%20et%20al&amp;contentID=10.1038%2Fs41588-022-01148-2&amp;copyright=The%20Author%28s%29&amp;publication=1061-4036&amp;publicationDate=2022-08-18&amp;publisherName=SpringerNature&amp;orderBeanReset=true&amp;oa=CC%20BY">Reprints and Permissions</a></p></div></div></section><section aria-labelledby="article-info" data-title="About this article"><div id="article-info-section"><h2 id="article-info">About this article</h2><div id="article-info-content"><div><p><a data-crossmark="10.1038/s41588-022-01148-2" target="_blank" rel="noopener" href="https://crossmark.crossref.org/dialog/?doi=10.1038/s41588-022-01148-2" data-track="click" data-track-action="Click Crossmark" data-track-label="link" data-test="crossmark"><img width="57" height="81" alt="Verify currency and authenticity via CrossMark" src="data:image/svg+xml;base64,<svg height="81" width="57" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="m17.35 35.45 21.3-14.2v-17.03h-21.3" fill="#989898"/><path d="m38.65 35.45-21.3-14.2v-17.03h21.3" fill="#747474"/><path d="m28 .5c-12.98 0-23.5 10.52-23.5 23.5s10.52 23.5 23.5 23.5 23.5-10.52 23.5-23.5c0-6.23-2.48-12.21-6.88-16.62-4.41-4.4-10.39-6.88-16.62-6.88zm0 41.25c-9.8 0-17.75-7.95-17.75-17.75s7.95-17.75 17.75-17.75 17.75 7.95 17.75 17.75c0 4.71-1.87 9.22-5.2 12.55s-7.84 5.2-12.55 5.2z" fill="#535353"/><path d="m41 36c-5.81 6.23-15.23 7.45-22.43 2.9-7.21-4.55-10.16-13.57-7.03-21.5l-4.92-3.11c-4.95 10.7-1.19 23.42 8.78 29.71 9.97 6.3 23.07 4.22 30.6-4.86z" fill="#9c9c9c"/><path d="m.2 58.45c0-.75.11-1.42.33-2.01s.52-1.09.91-1.5c.38-.41.83-.73 1.34-.94.51-.22 1.06-.32 1.65-.32.56 0 1.06.11 1.51.35.44.23.81.5 1.1.81l-.91 1.01c-.24-.24-.49-.42-.75-.56-.27-.13-.58-.2-.93-.2-.39 0-.73.08-1.05.23-.31.16-.58.37-.81.66-.23.28-.41.63-.53 1.04-.13.41-.19.88-.19 1.39 0 1.04.23 1.86.68 2.46.45.59 1.06.88 1.84.88.41 0 .77-.07 1.07-.23s.59-.39.85-.68l.91 1c-.38.43-.8.76-1.28.99-.47.22-1 .34-1.58.34-.59 0-1.13-.1-1.64-.31-.5-.2-.94-.51-1.31-.91-.38-.4-.67-.9-.88-1.48-.22-.59-.33-1.26-.33-2.02zm8.4-5.33h1.61v2.54l-.05 1.33c.29-.27.61-.51.96-.72s.76-.31 1.24-.31c.73 0 1.27.23 1.61.71.33.47.5 1.14.5 2.02v4.31h-1.61v-4.1c0-.57-.08-.97-.25-1.21-.17-.23-.45-.35-.83-.35-.3 0-.56.08-.79.22-.23.15-.49.36-.78.64v4.8h-1.61zm7.37 6.45c0-.56.09-1.06.26-1.51.18-.45.42-.83.71-1.14.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.36c.07.62.29 1.1.65 1.44.36.33.82.5 1.38.5.29 0 .57-.04.83-.13s.51-.21.76-.37l.55 1.01c-.33.21-.69.39-1.09.53-.41.14-.83.21-1.26.21-.48 0-.92-.08-1.34-.25-.41-.16-.76-.4-1.07-.7-.31-.31-.55-.69-.72-1.13-.18-.44-.26-.95-.26-1.52zm4.6-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.07.45-.31.29-.5.73-.58 1.3zm2.5.62c0-.57.09-1.08.28-1.53.18-.44.43-.82.75-1.13s.69-.54 1.1-.71c.42-.16.85-.24 1.31-.24.45 0 .84.08 1.17.23s.61.34.85.57l-.77 1.02c-.19-.16-.38-.28-.56-.37-.19-.09-.39-.14-.61-.14-.56 0-1.01.21-1.35.63-.35.41-.52.97-.52 1.67 0 .69.17 1.24.51 1.66.34.41.78.62 1.32.62.28 0 .54-.06.78-.17.24-.12.45-.26.64-.42l.67 1.03c-.33.29-.69.51-1.08.65-.39.15-.78.23-1.18.23-.46 0-.9-.08-1.31-.24-.4-.16-.75-.39-1.05-.7s-.53-.69-.7-1.13c-.17-.45-.25-.96-.25-1.53zm6.91-6.45h1.58v6.17h.05l2.54-3.16h1.77l-2.35 2.8 2.59 4.07h-1.75l-1.77-2.98-1.08 1.23v1.75h-1.58zm13.69 1.27c-.25-.11-.5-.17-.75-.17-.58 0-.87.39-.87 1.16v.75h1.34v1.27h-1.34v5.6h-1.61v-5.6h-.92v-1.2l.92-.07v-.72c0-.35.04-.68.13-.98.08-.31.21-.57.4-.79s.42-.39.71-.51c.28-.12.63-.18 1.04-.18.24 0 .48.02.69.07.22.05.41.1.57.17zm.48 5.18c0-.57.09-1.08.27-1.53.17-.44.41-.82.72-1.13.3-.31.65-.54 1.04-.71.39-.16.8-.24 1.23-.24s.84.08 1.24.24c.4.17.74.4 1.04.71s.54.69.72 1.13c.19.45.28.96.28 1.53s-.09 1.08-.28 1.53c-.18.44-.42.82-.72 1.13s-.64.54-1.04.7-.81.24-1.24.24-.84-.08-1.23-.24-.74-.39-1.04-.7c-.31-.31-.55-.69-.72-1.13-.18-.45-.27-.96-.27-1.53zm1.65 0c0 .69.14 1.24.43 1.66.28.41.68.62 1.18.62.51 0 .9-.21 1.19-.62.29-.42.44-.97.44-1.66 0-.7-.15-1.26-.44-1.67-.29-.42-.68-.63-1.19-.63-.5 0-.9.21-1.18.63-.29.41-.43.97-.43 1.67zm6.48-3.44h1.33l.12 1.21h.05c.24-.44.54-.79.88-1.02.35-.24.7-.36 1.07-.36.32 0 .59.05.78.14l-.28 1.4-.33-.09c-.11-.01-.23-.02-.38-.02-.27 0-.56.1-.86.31s-.55.58-.77 1.1v4.2h-1.61zm-47.87 15h1.61v4.1c0 .57.08.97.25 1.2.17.24.44.35.81.35.3 0 .57-.07.8-.22.22-.15.47-.39.73-.73v-4.7h1.61v6.87h-1.32l-.12-1.01h-.04c-.3.36-.63.64-.98.86-.35.21-.76.32-1.24.32-.73 0-1.27-.24-1.61-.71-.33-.47-.5-1.14-.5-2.02zm9.46 7.43v2.16h-1.61v-9.59h1.33l.12.72h.05c.29-.24.61-.45.97-.63.35-.17.72-.26 1.1-.26.43 0 .81.08 1.15.24.33.17.61.4.84.71.24.31.41.68.53 1.11.13.42.19.91.19 1.44 0 .59-.09 1.11-.25 1.57-.16.47-.38.85-.65 1.16-.27.32-.58.56-.94.73-.35.16-.72.25-1.1.25-.3 0-.6-.07-.9-.2s-.59-.31-.87-.56zm0-2.3c.26.22.5.37.73.45.24.09.46.13.66.13.46 0 .84-.2 1.15-.6.31-.39.46-.98.46-1.77 0-.69-.12-1.22-.35-1.61-.23-.38-.61-.57-1.13-.57-.49 0-.99.26-1.52.77zm5.87-1.69c0-.56.08-1.06.25-1.51.16-.45.37-.83.65-1.14.27-.3.58-.54.93-.71s.71-.25 1.08-.25c.39 0 .73.07 1 .2.27.14.54.32.81.55l-.06-1.1v-2.49h1.61v9.88h-1.33l-.11-.74h-.06c-.25.25-.54.46-.88.64-.33.18-.69.27-1.06.27-.87 0-1.56-.32-2.07-.95s-.76-1.51-.76-2.65zm1.67-.01c0 .74.13 1.31.4 1.7.26.38.65.58 1.15.58.51 0 .99-.26 1.44-.77v-3.21c-.24-.21-.48-.36-.7-.45-.23-.08-.46-.12-.7-.12-.45 0-.82.19-1.13.59-.31.39-.46.95-.46 1.68zm6.35 1.59c0-.73.32-1.3.97-1.71.64-.4 1.67-.68 3.08-.84 0-.17-.02-.34-.07-.51-.05-.16-.12-.3-.22-.43s-.22-.22-.38-.3c-.15-.06-.34-.1-.58-.1-.34 0-.68.07-1 .2s-.63.29-.93.47l-.59-1.08c.39-.24.81-.45 1.28-.63.47-.17.99-.26 1.54-.26.86 0 1.51.25 1.93.76s.63 1.25.63 2.21v4.07h-1.32l-.12-.76h-.05c-.3.27-.63.48-.98.66s-.73.27-1.14.27c-.61 0-1.1-.19-1.48-.56-.38-.36-.57-.85-.57-1.46zm1.57-.12c0 .3.09.53.27.67.19.14.42.21.71.21.28 0 .54-.07.77-.2s.48-.31.73-.56v-1.54c-.47.06-.86.13-1.18.23-.31.09-.57.19-.76.31s-.33.25-.41.4c-.09.15-.13.31-.13.48zm6.29-3.63h-.98v-1.2l1.06-.07.2-1.88h1.34v1.88h1.75v1.27h-1.75v3.28c0 .8.32 1.2.97 1.2.12 0 .24-.01.37-.04.12-.03.24-.07.34-.11l.28 1.19c-.19.06-.4.12-.64.17-.23.05-.49.08-.76.08-.4 0-.74-.06-1.02-.18-.27-.13-.49-.3-.67-.52-.17-.21-.3-.48-.37-.78-.08-.3-.12-.64-.12-1.01zm4.36 2.17c0-.56.09-1.06.27-1.51s.41-.83.71-1.14c.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.37c.08.62.29 1.1.65 1.44.36.33.82.5 1.38.5.3 0 .58-.04.84-.13.25-.09.51-.21.76-.37l.54 1.01c-.32.21-.69.39-1.09.53s-.82.21-1.26.21c-.47 0-.92-.08-1.33-.25-.41-.16-.77-.4-1.08-.7-.3-.31-.54-.69-.72-1.13-.17-.44-.26-.95-.26-1.52zm4.61-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.08.45-.31.29-.5.73-.57 1.3zm3.01 2.23c.31.24.61.43.92.57.3.13.63.2.98.2.38 0 .65-.08.83-.23s.27-.35.27-.6c0-.14-.05-.26-.13-.37-.08-.1-.2-.2-.34-.28-.14-.09-.29-.16-.47-.23l-.53-.22c-.23-.09-.46-.18-.69-.3-.23-.11-.44-.24-.62-.4s-.33-.35-.45-.55c-.12-.21-.18-.46-.18-.75 0-.61.23-1.1.68-1.49.44-.38 1.06-.57 1.83-.57.48 0 .91.08 1.29.25s.71.36.99.57l-.74.98c-.24-.17-.49-.32-.73-.42-.25-.11-.51-.16-.78-.16-.35 0-.6.07-.76.21-.17.15-.25.33-.25.54 0 .14.04.26.12.36s.18.18.31.26c.14.07.29.14.46.21l.54.19c.23.09.47.18.7.29s.44.24.64.4c.19.16.34.35.46.58.11.23.17.5.17.82 0 .3-.06.58-.17.83-.12.26-.29.48-.51.68-.23.19-.51.34-.84.45-.34.11-.72.17-1.15.17-.48 0-.95-.09-1.41-.27-.46-.19-.86-.41-1.2-.68z" fill="#535353"/></g></svg>"/></a></p><div><h3 id="citeas">Cite this article</h3><p>Zhou, X., Feliciano, P., Shu, C. <i>et al.</i> Integrating de novo and inherited variants in 42,607 autism cases identifies mutations in new moderate-risk genes.
                    <i>Nat Genet</i>  (2022). https://doi.org/10.1038/s41588-022-01148-2</p><p><a data-test="citation-link" data-track="click" data-track-action="download article citation" data-track-label="link" data-track-external="" href="https://citation-needed.springer.com/v2/references/10.1038/s41588-022-01148-2?format=refman&amp;flavour=citation">Download citation<svg width="16" height="16" focusable="false" role="img" aria-hidden="true"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#global-icon-download"></use></svg></a></p><ul data-test="publication-history"><li><p>Received<span>: </span><span><time datetime="2021-09-26">26 September 2021</time></span></p></li><li><p>Accepted<span>: </span><span><time datetime="2022-06-28">28 June 2022</time></span></p></li><li><p>Published<span>: </span><span><time datetime="2022-08-18">18 August 2022</time></span></p></li><li><p><abbr title="Digital Object Identifier">DOI</abbr><span>: </span><span>https://doi.org/10.1038/s41588-022-01148-2</span></p></li></ul></div></div></div></div></section>

            

            

            </div></div>
  </body>
</html>
