<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <title>James Routley | Feed</title>
    <link
      rel="stylesheet"
      type="text/css"
      href="../styles.css"
      media="screen"
    />
  </head>
  <body>
    <a href="/index.html">Back</a>
    <a href="https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1006616">Original</a>
    <h1>Investigating the case of human nose shape and climate adaptation</h1>
    
    <div id="readability-page-1" class="page"><div id="main-content"> <div>

<header>





<ul id="almSignposts">
  <li id="loadingMetrics">
    <p>Loading metrics</p>
  </li>
</ul>







    <div>

<div>
  <p id="licenseShort">Open Access</p>
  <p id="peerReviewed">Peer-reviewed</p>

<p id="artType">Research Article</p>


</div>
    </div>
    <div>






<div id="floatTitleTop" data-js-floater="title_author" role="presentation">
  <div>
    <div>
      

<ul id="floatAuthorList" data-js-floater="floated_authors">

  <li data-float-index="1">Arslan A. Zaidi, 

  </li>
  <li data-float-index="2">Brooke C. Mattern, 

  </li>
  <li data-float-index="3">Peter Claes, 

  </li>
  <li data-float-index="4">Brian McEcoy, 

  </li>
  <li data-float-index="5">Cris Hughes, 

  </li>
  <li data-float-index="6">Mark D. Shriver

  </li>

</ul>



    </div>
    <div id="titleTopCloser">
      <p><img src="https://journals.plos.org/resource/img/logo-plos.png" alt="PLOS"/></p><p>x</p>
    </div>
  </div>
</div>

      <ul>
        <li id="artPubDate">Published: March 16, 2017</li>
        <li id="artDoi">
<a href="https://doi.org/10.1371/journal.pgen.1006616">https://doi.org/10.1371/journal.pgen.1006616</a>
        </li>
        <li></li>
      </ul>

    </div>
  
</header>
  <section>





    <div>












      <div>




  








        <div id="artText">
          



<div xmlns:plos="http://plos.org"><h2>Abstract</h2><div><p>The evolutionary reasons for variation in nose shape across human populations have been subject to continuing debate. An import function of the nose and nasal cavity is to condition inspired air before it reaches the lower respiratory tract. For this reason, it is thought the observed differences in nose shape among populations are not simply the result of genetic drift, but may be adaptations to climate. To address the question of whether local adaptation to climate is responsible for nose shape divergence across populations, we use Qst–Fst comparisons to show that nares width and alar base width are more differentiated across populations than expected under genetic drift alone. To test whether this differentiation is due to climate adaptation, we compared the spatial distribution of these variables with the global distribution of temperature, absolute humidity, and relative humidity. We find that width of the nares is correlated with temperature and absolute humidity, but not with relative humidity. We conclude that some aspects of nose shape may indeed have been driven by local adaptation to climate. However, we think that this is a simplified explanation of a very complex evolutionary history, which possibly also involved other non-neutral forces such as sexual selection.</p>
</div></div><div xmlns:plos="http://plos.org"><h2>Author summary</h2>
<div><p>The study of human adaptation is essential to our understanding of disease etiology. Evolutionary investigations into why certain disease phenotypes such as sickle-cell anemia and lactose intolerance occur at different rates in different populations have led to a better understanding of the genetic and environmental risk factors involved. Similarly, research into the geographical distribution of skin pigmentation continues to yield important clues regarding risk of vitamin D deficiency and skin cancer. Here, we investigate whether variation in the shape of the external nose across populations has been driven by regional differences in climate. We find that variation in both nares width and alar base width appear to have experienced accelerated divergence across human populations. We also find that the geospatial distribution of nares width is correlated with temperature, and absolute humidity, but not with relative humidity. Our results support the claim that local adaptation to climate may have had a role in the evolution of nose shape differences across human populations.</p>
</div></div>


<div xmlns:plos="http://plos.org"><p><strong>Citation: </strong>Zaidi AA, Mattern BC, Claes P, McEcoy B, Hughes C, Shriver MD (2017) Investigating the case of human nose shape and climate adaptation. PLoS Genet 13(3):
           e1006616.
        
        https://doi.org/10.1371/journal.pgen.1006616</p><p><strong>Editor: </strong>Greg Gibson, Georgia Institute of Technology, UNITED STATES</p><p><strong>Received: </strong>December 17, 2015; <strong>Accepted: </strong>February 3, 2017; <strong>Published: </strong> March 16, 2017</p><p><strong>Copyright: </strong> © 2017 Zaidi et al. This is an open access article distributed under the terms of the <a href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution License</a>, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Data Availability: </strong>We have made almost all data, which were used to generate results in the paper, publicly available in a spreadsheet in the Supporting Information section (<a href="#pgen.1006616.s009">S1 File</a>). The spreadsheet includes de-identified data, split by the analyses for which they were used. More specifically, we have provided several phenotypes (nose shape measures, skin pigmentation, height, weight, and BMI) as well as relevant covariates (ancestry components, sex, and age). Where relevant, we have also provided climate values (mean annual temperature, annual aridity index, and ultraviolet B irradiance) ascribed to each individual. These data can be used to replicate almost all the results presented in the manuscript, without compromising the privacy of the participants. Some raw data, such as the 3D photographs, genotypes, and birthplaces, have not been shared because they are highly identifiable in nature and sharing them would be in ethical and legal violation of the informed consent obtained from participants. This restriction is not because of any personal or commercial interests.</p><p><strong>Funding: </strong>This work was supported in part by the United States National Institute of Justice, the United States Department of Defense, the University of Illinois Interdisciplinary Innovation Initiative Research Grant, the Flemish Institute for the Promotion of Innovation by Science and Technology in Flanders (IWT Vlaanderen), the Research Program of the Fund for Scientific Research - Flanders (Belgium), and the Center for Human Evolution and Development at Penn State. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p><p><strong>Competing interests: </strong> The authors have declared that no competing interests exist.</p></div>





<div xmlns:plos="http://plos.org" id="section1"><h2>Introduction</h2><p>The shape of the nose, like many other parts of the face, varies both within as well as across human populations. For example, the distance between the nasal alare (wings of the nose) are significantly larger in individuals of West African, South Asian, and East Asian ancestry as compared to persons with European ancestry [<a href="#pgen.1006616.ref001">1</a>]. The nasal index (width/height of the nasal aperture of the skull) is also known to vary significantly among populations [<a href="#pgen.1006616.ref001">1</a>,<a href="#pgen.1006616.ref002">2</a>]. Whether these population differences in nose shape are due primarily to genetic drift or natural selection is unclear.</p>
<p>A vital function of the nose is to warm inspired air to core body temperature and saturate it with water vapor before it reaches the lower respiratory tract [<a href="#pgen.1006616.ref003">3</a>]. In fact, inhaled air reaches 90% of the required temperature and humidity levels before even reaching the nasopharynx, implicating the nasal cavity as the major conditioning apparatus in the respiratory tract [<a href="#pgen.1006616.ref004">4</a>,<a href="#pgen.1006616.ref005">5</a>]. This conditioning acts to maintain proper functioning of the mucociliary apparatus, which functions in trapping particles and pathogens and removing them from the airways. Low humidity in the respiratory tract leads to impaired mucociliary function and increases the risk of both upper and lower respiratory tract infections [<a href="#pgen.1006616.ref006">6</a>]. Much of the air conditioning occurs as it passes through the turbinates, the walls of which are lined with blood vessels and mucus producing goblet cells [<a href="#pgen.1006616.ref004">4</a>]. Studies have shown that the efficacy of the conditioning process depends on the flow dynamics of the inspired air, which in turn, depend on the geometry of the nasal cavity and inlets [<a href="#pgen.1006616.ref005">5</a>,<a href="#pgen.1006616.ref007">7</a>]. Because of the function of the nose as an air-conditioning apparatus, it is hypothesized that differences in nose shape across populations may have been driven by local adaptation to climate [<a href="#pgen.1006616.ref002">2</a>,<a href="#pgen.1006616.ref003">3</a>,<a href="#pgen.1006616.ref008">8</a>].</p>
<p>There are several challenges in testing this hypothesis. We know there is substantial nose shape variation among human populations, in both the external morphology of the nose as well as the underlying cranial morphology [<a href="#pgen.1006616.ref001">1</a>,<a href="#pgen.1006616.ref002">2</a>,<a href="#pgen.1006616.ref009">9</a>]. While this can be explained by adaptation to local selection pressures, it could also be explained by the fact that phenotypic differences among geographically distant populations can arise simply due to genetic drift. Thus, in order to invoke divergent selection as an explanation, one must demonstrate that the observed variation in nose shape across human populations is greater than that expected under genetic drift alone. This can be carried out using the Qst statistic, which measures the degree of genetic differentiation underlying a quantitative trait [<a href="#pgen.1006616.ref010">10</a>]. In principle, the Qst of a neutrally evolving trait is expected to follow the Fst distribution of neutrally evolving loci [<a href="#pgen.1006616.ref011">11</a>]. Thus, when Qst is much greater than Fst, trait divergence exceeds neutral expectations and can be attributed to divergent selection [<a href="#pgen.1006616.ref011">11</a>]. The problem with Qst is that its calculation requires knowledge of the additive genetic variances within- and among-populations. These can only be estimated reliably using ‘common-garden’ experiments, in which environmental effects on the phenotype can be effectively controlled. Since such experiments are not possible in humans, Qst-based inference regarding divergent selection on human phenotypes necessitates making realistic assumptions about the heritabilities of the phenotypes in question.</p>
<p>Several studies have used this approach and found that while most aspects of the skull seem to be evolving neutrally, the shape of the nasal aperture appears to be more differentiated across human populations than expected under genetic drift [<a href="#pgen.1006616.ref009">9</a>,<a href="#pgen.1006616.ref012">12</a>,<a href="#pgen.1006616.ref013">13</a>]. More recently, it was also reported that the divergence in the shape of the external nose, at least between Europeans and Han Chinese populations, also exceeds neutral expectations [<a href="#pgen.1006616.ref014">14</a>]. While this might be true, most of these studies employed anticonservative heritability assumptions, which overestimate the genetic differentiation underlying a trait, and thus, lead to incorrect conclusions regarding the relative roles of selection and drift in driving phenotypic differences among populations.</p>
<p>Here, we carried out an exploration of external nose shape variation in light of quantitative genetic theory to investigate whether nose shape variation among populations exceeds neutral expectations. In doing so, we discuss the methodological challenges involved in tackling such questions in humans, and show how the limitations of previous studies can be addressed with recent advances in statistical genetics. As proof of principle, we compare the differentiation of nose shape with two highly heritable morphological traits, which are known to have a polygenic basis; namely height and skin pigmentation [<a href="#pgen.1006616.ref015">15</a>–<a href="#pgen.1006616.ref018">18</a>]. Both height and skin pigmentation exhibit substantial variation within and across human populations, and are thought to be under positive selection in various populations [<a href="#pgen.1006616.ref019">19</a>–<a href="#pgen.1006616.ref022">22</a>]. Skin pigmentation is also a good example of a phenotype which is known to have evolved in response to a geospatially varying selection pressure: exposure to ground level ultraviolet B radiation (UVB) [<a href="#pgen.1006616.ref023">23</a>]. Finally, we test whether clinal variation in aspects of nose shape, which appear to be under accelerated divergence across populations, covaries with geographic variation in temperature and humidity, in order to determine whether this divergence is due to climatic selection pressures.</p>
</div>

<div xmlns:plos="http://plos.org" id="section2"><h2>Results</h2>
<div id="section1"><h3>Describing variation in nose shape</h3>
<p>To quantify variation in nose shape, we first captured high resolution 3D images of participants’ faces using the 3dMD Face system (<em>3dMD Atlanta</em>, <em>GA</em>). Five positioning landmarks (two on the inner corner of the eyes, two on the outer corners of the mouth, and one on the tip of the nose) were placed in order to establish facial orientation. A spatially dense mesh of 7,150 quasi-landmarks (QLs) was mapped onto each image and its reflection. Generalized Procrustes Superimposition [<a href="#pgen.1006616.ref024">24</a>] was carried out on both sets of images (original and reflected) to remove differences in position and orientation. The Procrustes coordinates of the original and reflected image for each participant were then averaged to remove effects of bilateral asymmetry following Claes <em>et al</em>. (2014) [<a href="#pgen.1006616.ref025">25</a>]. The nose region, which is comprised of 709 out of the 7,150 QLs, was selected for downstream analyses.</p>
<p>We used linear distances and areas to characterize the shape of the nose (<a href="#pgen-1006616-g001">Fig 1</a>). The linear distances (measured in mm) were calculated using seven standard anthropometric landmarks: i. nasion (n), ii. pronasale (prn), iii. subnasale (sn), iv. Left alar curvature, (al<sub>l</sub>), v. right alar curvature (al<sub>r</sub>), vi. left alar base (ac<sub>l</sub>), and vii. Right alar base (ac<sub>r</sub>) (<a href="#pgen-1006616-g001">Fig 1</a>) [<a href="#pgen.1006616.ref026">26</a>,<a href="#pgen.1006616.ref027">27</a>]. These seven landmarks were placed on the spatially-dense mesh of 709 QLs, which facilitated automated placement on each face. We used the x, y, and z coordinates of these landmarks to calculate five Euclidean distances: i. nares width (al<sub>l</sub>−al<sub>r</sub>), ii. alar base width (ac<sub>l</sub>−ac<sub>r</sub>), iii. nasal height (n–sn), iv. nasal ridge length (n–prn), and v. nasal tip protrusion (sn–prn) (<a href="#pgen-1006616-g001">Fig 1</a>). We also computed two areas (measured in mm<sup>2</sup>): i. total external area of the nose and ii. mean nostril area (<a href="#pgen-1006616-g001">Fig 1</a>) (Methods).</p>
<div data-doi="10.1371/journal.pgen.1006616.g001"><div><p><a title="Click for larger image" href="https://journals.plos.org/plosgenetics/article/figure/image?size=medium&amp;id=10.1371/journal.pgen.1006616.g001" data-doi="10.1371/journal.pgen.1006616" data-uri="10.1371/journal.pgen.1006616.g001"><img src="https://journals.plos.org/plosgenetics/article/figure/image?size=inline&amp;id=10.1371/journal.pgen.1006616.g001" alt="thumbnail" loading="lazy"/></a></p></div><p><span>Fig 1. </span> Measurements used to describe the shape of the nose.</p><p>A) The locations of landmarks, which were used to calculate linear distances, are shown as red points. B) Five Linear distances (red lines) and two surface areas (red mesh) were used to describe nose shape. Linear distances were calculated from the 3D coordinates of landmarks (red points). Surface areas were computed as the sum of the polygons highlighted in red.</p>
<p><a href="https://doi.org/10.1371/journal.pgen.1006616.g001">
              https://doi.org/10.1371/journal.pgen.1006616.g001</a></p></div><p><a href="#pgen-1006616-g002">Fig 2</a> shows the distribution of aspects of nose shape in males and females from four human population groups: West African, East Asian, South Asian, and Northern European. These groups were defined based on genetic and self-reported ancestry (Methods). <a href="#pgen-1006616-g002">Fig 2</a> also shows the distribution of height and melanin index, a measure of skin pigmentation level derived from reflectance spectrophotometry (measured on the upper inner arms; see <a href="#sec009">Methods</a>), for comparison. The sample size, mean, and standard deviation for each phenotype are provided in <a href="#pgen-1006616-t001">Table 1</a>. One clear observation is that all aspects of nose shape, at least those considered here, are highly sexually dimorphic (<a href="#pgen-1006616-g002">Fig 2</a>, <a href="#pgen-1006616-t001">Table 1</a>). Males, on average, tend to have wider nares, longer nasal ridges, more outwardly protruding nasal tips, bigger nostrils, and a larger overall external surface area, compared to females (<a href="#pgen-1006616-g002">Fig 2</a>, <a href="#pgen-1006616-t001">Table 1</a>). All nasal measurements are also significantly different across populations (<a href="#pgen.1006616.s008">S1 Table</a>). Based on <a href="#pgen-1006616-t001">Table 1</a>, we note some general patterns. For example, nares width and alar base width are largest in W. Africans and smallest in N. Europeans. These findings are consistent with previous observations [<a href="#pgen.1006616.ref001">1</a>,<a href="#pgen.1006616.ref002">2</a>]. Nasal tip protrusion is greatest in N. Europeans and smallest in W. Africans and E. Asians. E. Asians also tend to have the smallest noses in terms of external surface area.</p>
<div data-doi="10.1371/journal.pgen.1006616.g002"><div><p><a title="Click for larger image" href="https://journals.plos.org/plosgenetics/article/figure/image?size=medium&amp;id=10.1371/journal.pgen.1006616.g002" data-doi="10.1371/journal.pgen.1006616" data-uri="10.1371/journal.pgen.1006616.g002"><img src="https://journals.plos.org/plosgenetics/article/figure/image?size=inline&amp;id=10.1371/journal.pgen.1006616.g002" alt="thumbnail" loading="lazy"/></a></p></div><p><span>Fig 2. </span> Boxplots of phenotypes by population and sex overlaid with the individual data points.</p><p>Height is measured in centimeters and melanin index is measured in percentage reflectance (Methods). Linear distances are measured in millimeters (mm) and area are measured in mm<sup>2</sup>. Points are individual observations and the color of the boxplots and points represents sex with blue indicating males and red indicating females.</p>
<p><a href="https://doi.org/10.1371/journal.pgen.1006616.g002">
              https://doi.org/10.1371/journal.pgen.1006616.g002</a></p></div></div>

<div id="section2"><h3>Testing for accelerated divergence</h3>
<p>Differences in a phenotype can accumulate across populations simply due to genetic drift. In order to invoke positive directional selection, one must demonstrate that the variation across populations is more than that expected under genetic drift. We used Qst (see Leinonen <em>et al</em>. [<a href="#pgen.1006616.ref028">28</a>] for a recent review), the quantitative genetic analog of Fst [<a href="#pgen.1006616.ref010">10</a>,<a href="#pgen.1006616.ref029">29</a>], to test whether certain aspects of nose shape exhibit greater differentiation across populations than expected under genetic drift alone. Qst measures the degree of genetic differentiation in a quantitative trait across populations and is defined as:
<a name="pgen.1006616.e001" id="pgen.1006616.e001"></a><span><img src="https://journals.plos.org/plosgenetics/article/file?type=thumbnail&amp;id=10.1371/journal.pgen.1006616.e001" loading="lazy"/><span>(1)</span></span>
where <span><img src="https://journals.plos.org/plosgenetics/article/file?type=thumbnail&amp;id=10.1371/journal.pgen.1006616.e002" loading="lazy"/></span> and <span><img src="https://journals.plos.org/plosgenetics/article/file?type=thumbnail&amp;id=10.1371/journal.pgen.1006616.e003" loading="lazy"/></span> are the components of phenotypic variance due to additive genetic effects among and within populations, respectively. It has been shown that, in principle, the distribution of Qst of a quantitative trait that has evolved under genetic drift alone is expected to be equal to Fst of neutral genetic markers [<a href="#pgen.1006616.ref011">11</a>,<a href="#pgen.1006616.ref030">30</a>,<a href="#pgen.1006616.ref031">31</a>]. This expectation allows one to compare Qst to Fst to test whether genetic drift alone is sufficient to explain the divergence of a trait among populations. If the Qst of a trait across a set of populations is much greater than Fst, it means that the phenotypic differentiation exceeds the expectation under neutrality.</p>
<p>The components of additive genetic variance, <span><img src="https://journals.plos.org/plosgenetics/article/file?type=thumbnail&amp;id=10.1371/journal.pgen.1006616.e004" loading="lazy"/></span> and <span><img src="https://journals.plos.org/plosgenetics/article/file?type=thumbnail&amp;id=10.1371/journal.pgen.1006616.e005" loading="lazy"/></span>, are typically estimated from ‘common-garden’ experiments in which the effects due to the environment can be controlled [<a href="#pgen.1006616.ref028">28</a>]. This is often not possible in non-model organisms, especially humans, because of practical and ethical limitations. In such cases, <span><img src="https://journals.plos.org/plosgenetics/article/file?type=thumbnail&amp;id=10.1371/journal.pgen.1006616.e006" loading="lazy"/></span> and <span><img src="https://journals.plos.org/plosgenetics/article/file?type=thumbnail&amp;id=10.1371/journal.pgen.1006616.e007" loading="lazy"/></span> can be estimated from the among- and within-population components of <em>phenotypic</em> variance, <span><img src="https://journals.plos.org/plosgenetics/article/file?type=thumbnail&amp;id=10.1371/journal.pgen.1006616.e008" loading="lazy"/></span> and <span><img src="https://journals.plos.org/plosgenetics/article/file?type=thumbnail&amp;id=10.1371/journal.pgen.1006616.e009" loading="lazy"/></span>, respectively, provided the heritabilities underlying these components are known:
<a name="pgen.1006616.e010" id="pgen.1006616.e010"></a><span><img src="https://journals.plos.org/plosgenetics/article/file?type=thumbnail&amp;id=10.1371/journal.pgen.1006616.e010" loading="lazy"/><span>(2)</span></span></p>
<p>Here <span><img src="https://journals.plos.org/plosgenetics/article/file?type=thumbnail&amp;id=10.1371/journal.pgen.1006616.e011" loading="lazy"/></span> and <span><img src="https://journals.plos.org/plosgenetics/article/file?type=thumbnail&amp;id=10.1371/journal.pgen.1006616.e012" loading="lazy"/></span> are among- and within-population components of the phenotypic variance and <em>c</em> and <em>h</em><sup><em>2</em></sup> are proportions of <span><img src="https://journals.plos.org/plosgenetics/article/file?type=thumbnail&amp;id=10.1371/journal.pgen.1006616.e013" loading="lazy"/></span> and <span><img src="https://journals.plos.org/plosgenetics/article/file?type=thumbnail&amp;id=10.1371/journal.pgen.1006616.e014" loading="lazy"/></span>, respectively, that are due to additive genetic effects. Both <em>c</em> and <em>h</em><sup><em>2</em></sup> are can range from 0 (none of the variance is due to additive genetic effects) to 1 (all of the variance is due to additive genetic effects). Eq (<a href="#pgen.1006616.e010">2</a>) shows that Qst calculated from phenotypic variance components depends on the ratio between <em>c</em> and <em>h</em><sup><em>2</em></sup> [<a href="#pgen.1006616.ref032">32</a>]. Without prior information, it is reasonable to assume <em>c/h</em><sup><em>2</em></sup> = 1, i.e., the proportion of phenotypic variance due to additive genetic effects is the same among- and within-populations. Qst calculated this way is sometimes referred to as Pst [<a href="#pgen.1006616.ref033">33</a>]. However, we will continue to use the term Qst to avoid confusion and will evaluate the validity of the assumption that <em>c/h</em><sup><em>2</em></sup> = 1 in the following section.</p>
<p>We calculated Qst for each aspect of nose shape, described in the previous section, across four human population groups: i) West African (N = 40), ii) North European (N = 236), iii) East Asian (N = 127), and iv) South Asian (N = 73) (see <a href="#sec009">Methods</a> for selection criteria). We used a non-parametric bootstrap approach to generate the empirical distributions of Qst and Fst and to test whether the observed value of Qst is greater than Fst (Methods). The statistic we used is Qst–Fst, which, under the null hypothesis of genetic drift, is expected to be equal to zero. The larger the Qst–Fst of a phenotype, the stronger the evidence that the variation in the phenotype across populations is more than that expected under genetic drift alone. We refer to outliers in the neutral distribution as signals of accelerated divergence for brevity. The strength of evidence for accelerated divergence can be measured using an empirical p-value, which is the proportion of bootstrapped values of Qst–Fst that are less than zero. To compare with other quantitative traits with a polygenic basis, we also tested whether height and skin pigmentation exhibit signals of accelerated divergence. The results are illustrated in <a href="#pgen-1006616-g003">Fig 3</a> and the p-values are listed in <a href="#pgen-1006616-t002">Table 2</a>. We treat phenotypes that pass a stringent Bonferonni correction (p-value &lt;0.0071 = 0.05/7 for seven nose shape traits) as exhibiting signals of accelerated divergence across populations.</p>
<div data-doi="10.1371/journal.pgen.1006616.g003"><div><p><a title="Click for larger image" href="https://journals.plos.org/plosgenetics/article/figure/image?size=medium&amp;id=10.1371/journal.pgen.1006616.g003" data-doi="10.1371/journal.pgen.1006616" data-uri="10.1371/journal.pgen.1006616.g003"><img src="https://journals.plos.org/plosgenetics/article/figure/image?size=inline&amp;id=10.1371/journal.pgen.1006616.g003" alt="thumbnail" loading="lazy"/></a></p></div><p><span>Fig 3. </span> Qst–Fst results across all populations.</p><p>The bootstrapped distribution of Qst–Fst for each phenotype (shown by a violin plot) is compared against the expected value of zero under neutrality (horizontal dashed line). Phenotypes, which exhibit accelerated divergence (using a Bonferronni corrected p-value threshold of 0.0071), are shown in red.</p>
<p><a href="https://doi.org/10.1371/journal.pgen.1006616.g003">
              https://doi.org/10.1371/journal.pgen.1006616.g003</a></p></div><p>As expected, skin pigmentation shows a strong signal of accelerated divergence across populations (Qst = 0.642, p-value = 1.00E-04). This is in accordance with the idea that differences in skin pigmentation, at least across continental populations, have been driven by positive selection. Height, on the other hand, does not show signals of accelerated divergence (Qst = 0.177, p-value = 0.14). While height seems to be under positive <em>selection within</em> European and African populations separately [<a href="#pgen.1006616.ref019">19</a>,<a href="#pgen.1006616.ref034">34</a>], it does not appear to be driven by divergent selection pressures across the populations considered here. Of all the nose shape measurements, only nares width (Qst = 0.467, p-value = 2.80E-03) and alar base width (Qst = 0.440, p-value = 4.90E-03) exhibit signals of accelerated divergence. However, the degree of divergence in these traits is not as high as that observed for skin pigmentation.</p>
</div>

<div id="section3"><h3>Evaluating validity of assumptions regarding <em>c</em> and <em>h</em><sup><em>2</em></sup></h3>
<p>We estimated Qst under the assumption that <em>c/h</em><sup><em>2</em></sup> = 1. In other words, we assume that the proportion of between-population variance in the trait due to additive genetic effects is the same as the proportion of within-population variance due to additive genetic effects. This can be an incorrect assumption if a large proportion of the phenotypic variance between populations is due to direct environmental effects (i.e., phenotypic plasticity). If the true value of <em>c/h</em><sup><em>2</em></sup> is drastically lower than 1, phenotype-derived estimates of the among-population genetic variance, and hence, Qst will be inflated, resulting in false-positive conclusions regarding the role of selection in driving phenotypic differentiation. This fact has largely been ignored in previous studies, in which Qst-based approaches were used to explore craniofacial divergence [<a href="#pgen.1006616.ref012">12</a>–<a href="#pgen.1006616.ref014">14</a>,<a href="#pgen.1006616.ref035">35</a>]. In these studies, <em>h</em><sup><em>2</em></sup> was assumed to be 0.55 and <em>c</em> was implicitly assumed to be 1, resulting in <em>c/h</em><sup><em>2</em></sup> to be greater than 1. While this might be true for some traits (for example, see skin pigmentation in <a href="#pgen-1006616-t003">Table 3</a>), this is an anticonservative approach in our opinion, because it assumes that the heritability across populations is higher than the heritability within populations. If anything, the opposite case (<em>c/h</em><sup><em>2</em></sup> &lt; 1) is more plausible as we expect environmental variation to be large across geographically distant populations. Keeping this in mind, we evaluated the sensitivity of the Qst–Fst results to the case where <em>c</em>/<em>h</em><sup><em>2</em></sup> &lt; 1 [<a href="#pgen.1006616.ref032">32</a>], by determining the ‘critical value’ of <em>c/h</em><sup><em>2</em></sup> at which the 95% lower bound of Qst meets the 95% upper bound of Fst (<a href="#pgen-1006616-g004">Fig 4</a>). Smaller critical values imply that phenotype-based Qst–Fst comparisons are robust to <em>c/h</em><sup><em>2</em></sup> assumptions. Such ‘sensitivity analyses’, while common in the molecular ecology literature, are underrepresented in human genetic and anthropological studies. <a href="#pgen-1006616-g004">Fig 4</a> shows the sensitivity curves for height, skin pigmentation, and nose shape traits. Skin pigmentation, which shows a very high Qst value, has a critical value of 0.25, which is much lower than 1 (<a href="#pgen-1006616-g004">Fig 4</a> and <a href="#pgen-1006616-t002">Table 2</a>). The critical value of nares width is 0.50 and that of alar base width is 0.55, which are also much lower than 1 (<a href="#pgen-1006616-g004">Fig 4</a> and <a href="#pgen-1006616-t002">Table 2</a>). Thus, we expect that these phenotypes would exhibit signals of accelerated divergence, even if the true values of <em>c/h</em><sup><em>2</em></sup> were much less than 1.</p>
<div data-doi="10.1371/journal.pgen.1006616.g004"><div><p><a title="Click for larger image" href="https://journals.plos.org/plosgenetics/article/figure/image?size=medium&amp;id=10.1371/journal.pgen.1006616.g004" data-doi="10.1371/journal.pgen.1006616" data-uri="10.1371/journal.pgen.1006616.g004"><img src="https://journals.plos.org/plosgenetics/article/figure/image?size=inline&amp;id=10.1371/journal.pgen.1006616.g004" alt="thumbnail" loading="lazy"/></a></p></div><p><span>Fig 4. </span> Sensitivity plots showing critical values of <em>c/h</em><sup><em>2</em></sup>.</p><p>Change in median Qst of height, skin pigmentation, and nose shape measures as a function of <em>c/h</em><sup><em>2</em></sup> is shown as a solid red line. Median Fst is shown as a solid blue line. The lower and upper 95% bounds for Qst and Fst are shown as dashed red and blue lines, respectively. The critical value at which the lower bound of Qst meets the upper bound of Fst is shown on the upper left corner of each plot. Lower critical values indicate the Qst–Fst is more robust to <em>c/h</em><sup><em>2</em></sup> assumptions.</p>
<p><a href="https://doi.org/10.1371/journal.pgen.1006616.g004">
              https://doi.org/10.1371/journal.pgen.1006616.g004</a></p></div></div>

<div id="section4"><h3>Investigating heritability of nose shape within- and among-populations</h3>
<p>In addition to the sensitivity analysis, we wanted to demonstrate that differences in nose shape among individuals both within- and between-populations are heritable. Within-population heritability (<em>h</em><sup>2</sup>) for a phenotype is traditionally estimated from data collected on large sets of twins or from pedigrees where the genetic relationships among individuals are known [<a href="#pgen.1006616.ref036">36</a>]. Recently, Yang and colleagues (2010) introduced a linear mixed model approach, which can be used to estimate an alternative statistic in unrelated individuals: the proportion of phenotypic variance explained by genotyped SNPs (<em>h</em><sub><em>g</em></sub><sup><em>2</em></sup>) [<a href="#pgen.1006616.ref018">18</a>]. A major advantage of this approach over traditional twin- and pedigree-based approaches is that it is applicable to data from unrelated individuals, which are more accessible and easier to collect in large numbers. We calculated <em>h</em><sub><em>g</em></sub><sup><em>2</em></sup> as an estimate of <em>h</em><sup><em>2</em></sup> for height, melanin index, and nose shape traits using 118,420 autosomal SNPs in a sample of unrelated persons of European ancestry, implemented in the GCTA software [<a href="#pgen.1006616.ref037">37</a>] (Methods). The values of <em>h</em><sub><em>g</em></sub><sup><em>2</em></sup> are provided in <a href="#pgen-1006616-t003">Table 3</a>. The heritability for height in Europeans (<em>h</em><sub><em>g</em></sub><sup><em>2</em></sup> = 0.394; N = 1,825) is similar to previously reported estimates [<a href="#pgen.1006616.ref018">18</a>]. Interestingly, the heritability of skin pigmentation is very low in Europeans (<em>h</em><sub><em>g</em></sub><sup><em>2</em></sup> = 0.191; N = 1,231), which we think can be explained by reduced genetic variation at skin pigmentation loci due to strong positive selection for lighter skin in Europeans [<a href="#pgen.1006616.ref017">17</a>,<a href="#pgen.1006616.ref021">21</a>,<a href="#pgen.1006616.ref038">38</a>]. Almost all aspects of nose shape seem to be highly heritable in Europeans (<em>h</em><sub><em>g</em></sub><sup><em>2</em></sup> range: 0.401–0.657 N = 1,718) (<a href="#pgen-1006616-t003">Table 3</a>).</p>
<p>Estimation of the among-population heritability (<em>c</em>) is more difficult, since genetic effects between geographically distant populations can be confounded with large environmental effects. This parameter can often only be reliably estimated through ‘common garden’ experiments, in which systematic differences in the environment, which might be confounded with genetic differences, can be taken into account. However, as mentioned earlier, this approach is only possible in model organisms, which are amenable to experimental manipulation. In humans and other non-model organisms, this is a severe limitation, which can be overcome by studying naturally occurring admixed populations.</p>
<p>The process of admixture reunites gene pools from two or more populations, which might have diverged due to genetic drift, mutation, and selection. Admixture, which may occur repeatedly over several generations, followed by recombination, leads to chromosomes that are essentially mosaics of ancestry segments (<a href="#pgen-1006616-g005">Fig 5</a>) [<a href="#pgen.1006616.ref039">39</a>]. In a randomly mating admixed population, ancestry segments segregate randomly with respect to the environment, which decouples the between-population genetic effects on the phenotype from the environmental effects, allowing for the estimation of the genetic variance underlying a phenotype. We propose that a method recently developed to estimate heritability in admixed populations (Zaitlen <em>et al</em>. (2014)) [<a href="#pgen.1006616.ref040">40</a>], might provide a valid estimate of <em>c</em>. Zaitlen <em>et al</em>. (2014) extend the method developed by Yang <em>et al</em>. (2010) [<a href="#pgen.1006616.ref018">18</a>], using local ancestry at SNPs, instead of genotypes, to construct the genetic relationship matrix among admixed individuals. The proportion of phenotypic variance in an admixed population that can be explained by local ancestry (<em>h</em><sub><em>y</em></sub><sup><em>2</em></sup>) is conceptually equivalent to <em>c</em> between the parental populations. Our reasoning is that, on a genotypic level, the genetic variation in an admixed population should be the sum of the genetic variation within the parental populations and the genetic variation between them. Variation at the scale of local ancestry only represents genetic variation between populations (<a href="#pgen-1006616-g005">Fig 5</a>). Thus, the proportion of phenotypic variation that can be explained by local ancestry (<em>h</em><sub><em>y</em></sub><sup><em>2</em></sup>), should be equivalent to <em>c</em>.</p>
<div data-doi="10.1371/journal.pgen.1006616.g005"><div><p><a title="Click for larger image" href="https://journals.plos.org/plosgenetics/article/figure/image?size=medium&amp;id=10.1371/journal.pgen.1006616.g005" data-doi="10.1371/journal.pgen.1006616" data-uri="10.1371/journal.pgen.1006616.g005"><img src="https://journals.plos.org/plosgenetics/article/figure/image?size=inline&amp;id=10.1371/journal.pgen.1006616.g005" alt="thumbnail" loading="lazy"/></a></p></div><p><span>Fig 5. </span> Admixture brings together within- and between-population variation in admixed individuals.</p><p>Genetic variation between the two parental populations is represented by difference in color of the chromosomes, whereas genetic variation within the populations is represented by hue intensity. Admixture brings together genetic variation from both populations. On a genotypic level, genetic variation within admixed populations is composed of both within-population and between-population variation. The variation at the level of ancestry is variation between the two parental populations.</p>
<p><a href="https://doi.org/10.1371/journal.pgen.1006616.g005">
              https://doi.org/10.1371/journal.pgen.1006616.g005</a></p></div><p>Following this reasoning, we estimated <em>h</em><sub><em>y</em></sub><sup><em>2</em></sup> from local ancestry at 623,625 autosomal SNPs in a sample of 409 Cape Verdeans, who derive their ancestry primarily from W. Africans and Europeans (see <a href="#sec009">Methods</a> for details and <a href="#pgen-1006616-t003">Table 3</a> for results). <a href="#pgen-1006616-t003">Table 3</a> shows that the phenotypic variation between W. Africans and Europeans in height (<em>h</em><sub><em>y</em></sub><sup><em>2</em></sup> = 0.224, N = 409) is quite heritable. Results for skin pigmentation data are not presented here as they were unavailable. Differences in many aspects of nose shape are also heritable (Nares width: <em>h</em><sub><em>y</em></sub><sup><em>2</em></sup> = 0.226, Alar base width: <em>h</em><sub><em>y</em></sub><sup><em>2</em></sup> = 0.212, Nasal tip protrusion: 0.177, External surface area: <em>h</em><sub><em>y</em></sub><sup><em>2</em></sup> = 0.121, N = 409). Other aspects of nose shape may not be as heritable between these two populations (Nasal height: <em>h</em><sub><em>y</em></sub><sup><em>2</em></sup> = 0.03, <em>h</em><sub><em>y</em></sub><sup><em>2</em></sup> = 0.059, N = 409). Another interesting observation from <a href="#pgen-1006616-t003">Table 3</a> is that estimates of <em>h</em><sub><em>y</em></sub><sup><em>2</em></sup> are generally lower than estimates of <em>h</em><sub><em>g</em></sub><sup><em>2</em></sup> for all traits. This suggests that for most human traits, the additive genetic variance between populations might be less than the additive genetic variance within populations, which agrees with the fact that most of the genetic variation in humans exists within populations. However, we are cautious of over-stating this conclusion since the heritabilities were estimated in individuals with W. African and European ancestry only and do not reflect the variation within and across other populations. Altogether, our results show that genetic differences underlie the variation in many aspects of nose shape, both within- and between-populations.</p>
</div>

<div id="section5"><h3>Testing for adaptation to climate</h3>
<p>Previously, several studies have shown that the shapes of the nasal aperture and nasal cavity are correlated with climate variables related to temperature and humidity such that individuals from cold-dry climates exhibit narrower nasal cavities compared to individuals from warm-humid climates [<a href="#pgen.1006616.ref041">41</a>,<a href="#pgen.1006616.ref042">42</a>]. We were interested in testing whether aspects of external nose shape showing unusually high differentiation across populations based on Qst–Fst analysis, show correlations with climate. For this purpose, we selected, from the subset used in the Qst analysis, females with genetic data whose parents were born in a region that coincided with their continental ancestry (N = 140) (<a href="#pgen-1006616-g006">Fig 6</a>). This was done to assign to each individual, a climate value that was most similar to their ‘ancestral’ climate. Since we did not have genotype data available for males of Northern European ancestry, we only used females for this analysis. The genotype data were necessary to correct for genetic structure (see <a href="#sec009">Methods</a>). We tested the correlation of nares width and alar base width with three climate variables: i) mean annual temperature (hereafter referred to as temperature), ii) relative humidity, and iii) absolute humidity. The choice of these climate variables follows from the functional importance of the nose in warming and humidifying inspired air. We also tested whether skin pigmentation is correlated with UVB levels. This was used as proof of principle, since multiple lines of evidence suggest that differences in skin pigmentation across populations have evolved primarily in response to ultraviolet radiation [<a href="#pgen.1006616.ref017">17</a>,<a href="#pgen.1006616.ref020">20</a>–<a href="#pgen.1006616.ref022">22</a>]. The association between phenotypes and climate variables was tested using linear mixed models, which correct for age, BMI, and genetic similarity (Methods). Sex was not included as a covariate as only females were used. We used a likelihood ratio test (LRT) to evaluate the statistical significance of the slope between phenotype and climate variable. The LRT statistic and its corresponding p-values were generated by comparing full (climate predictor included) and reduced (climate predictor removed) models. Results are presented in <a href="#pgen-1006616-t004">Table 4</a>.</p>
<div data-doi="10.1371/journal.pgen.1006616.g006"><div><p><a title="Click for larger image" href="https://journals.plos.org/plosgenetics/article/figure/image?size=medium&amp;id=10.1371/journal.pgen.1006616.g006" data-doi="10.1371/journal.pgen.1006616" data-uri="10.1371/journal.pgen.1006616.g006"><img src="https://journals.plos.org/plosgenetics/article/figure/image?size=inline&amp;id=10.1371/journal.pgen.1006616.g006" alt="thumbnail" loading="lazy"/></a></p></div><p><span>Fig 6. </span> Geographic distribution of parents’ birthplaces.</p><p>Individual points represent the birth locations of the parents with a line connecting two parents. A single point indicates that the two parents were born in the same location. Climate values at these locations were used to test for signals of climate adaptation.</p>
<p><a href="https://doi.org/10.1371/journal.pgen.1006616.g006">
              https://doi.org/10.1371/journal.pgen.1006616.g006</a></p></div><p>As expected, we see a strong relationship between skin pigmentation and UVB (N = 126, T = 6.23, LRT = 55.20, p-value = 1.09E-13). The positive values of the slope and t-statistic indicate that people tend to be darker in regions with higher exposure to UVB and vice versa. This pattern provides further support to the notion that adaptation to ultraviolet radiation has had an important role in the evolution of skin pigmentation in humans. Nares width is significantly correlated with temperature (N = 140, T = 3.09, LRT = 9.24, p-value = 2.37E-03) and absolute humidity (N = 140, T = 2.73, LRT = 7.28, p-value = 6.97E-03). The positive slopes indicate that individuals from warm-humid climates, on average, tend to have wider nares whereas individuals from cool-dry climates tend to have narrower nares. This is consistent with previously reported patterns of ecogeographic variation in the width of the nasal aperture and nasal cavity [<a href="#pgen.1006616.ref041">41</a>,<a href="#pgen.1006616.ref042">42</a>]. Nares width was not significantly correlated with relative humidity (N = 140, T = -1.65, LRT = 2.71, p-value = 0.100). This is not surprising since absolute humidity is more important for the physiological functioning of the nose than relative humidity [<a href="#pgen.1006616.ref043">43</a>]. Alar base width is neither significantly correlated with temperature (N = 140, T = 1.97, LRT = 3.82, p-value = 0.051) nor with relative humidity (N = 140, T = -0.79, LRT = 0.62, p-value = 0.431), and is only mildly correlated with absolute humidity (N = 140, T = 2.09, LRT = 4.30, p-value = 0.038). This might suggest that it is not the width of the nose <em>per se</em> but the width of the nares that is under selection pressure. However, even nares width is not as strongly correlated with temperature and humidity as skin pigmentation is to UVB, suggesting that the strength of selection exerted by temperature on nose width is likely to be much weaker than the selection pressure imposed by UVB exposure on skin pigmentation.</p>
<p>Following a suggestion by one of the reviewers, we investigated whether the observed phenotype-climate correlations were driven by any one population. To do so, we re-estimated the phenotype-climate slopes (skin pigmentation vs UVB and nares width vs temperature) after removing each population in turn. <a href="#pgen-1006616-g007">Fig 7</a> shows that the 95% confidence intervals of the slope overlap with zero (red line in <a href="#pgen-1006616-g007">Fig 7</a>) only when the N. Europeans are removed, suggesting that both phenotype-climate correlations are driven primarily by the inclusion of N. Europeans. This effect might be indicative of positive selection for lighter skin pigmentation and narrower nares in higher latitude populations. However, because our sampling of populations at higher latitude is limited (<a href="#pgen-1006616-g006">Fig 6</a>), this result requires further investigation.</p>
<div data-doi="10.1371/journal.pgen.1006616.g007"><div><p><a title="Click for larger image" href="https://journals.plos.org/plosgenetics/article/figure/image?size=medium&amp;id=10.1371/journal.pgen.1006616.g007" data-doi="10.1371/journal.pgen.1006616" data-uri="10.1371/journal.pgen.1006616.g007"><img src="https://journals.plos.org/plosgenetics/article/figure/image?size=inline&amp;id=10.1371/journal.pgen.1006616.g007" alt="thumbnail" loading="lazy"/></a></p></div><p><span>Fig 7. </span> The effect of removing each population on phenotype-climate correlations.</p><p>The point estimate and the 95% confidence intervals of the phenotype-climate slopes after removing each population (on the y-axis) are shown for A) skin pigmentation and UVB, B) nares width and temperature, and C) nares width and absolute humidity. In all cases, the 95% confidence interval of the slope overlaps with zero (red vertical line), only when the Europeans are removed.</p>
<p><a href="https://doi.org/10.1371/journal.pgen.1006616.g007">
              https://doi.org/10.1371/journal.pgen.1006616.g007</a></p></div></div>
</div>

<div xmlns:plos="http://plos.org" id="section3"><h2>Discussion</h2><p>The diversity of facial features across human populations has fascinated scientists for a long time. Even though genetic drift has played a predominant role in human evolution, external physical traits such as facial shape and skin pigmentation, because of their proximity to the environment, have also likely been influenced by natural selection. Substantial evidence has accumulated supporting the hypothesis that differences in skin pigmentation across human populations have evolved largely in response to selection pressures imposed by exposure to ultraviolet radiation [<a href="#pgen.1006616.ref020">20</a>]. How selection may have affected facial shape, a trait that is also quite variable between populations, is unclear, likely because it has received much less attention to date. Given the complexity of the face, we have chosen to study one particularly interesting and variable part of the face; the nose. The broad question driving this study is: Has climate adaptation played an important role in influencing variation in human nose shape? To answer this question, we formulated two hypotheses: i. divergent selection has been involved in the differentiation of certain aspects of nose shape across populations, and ii. climate is the agent of selection in cases where divergent selection can be invoked.</p>
<p>To test the first hypothesis, we used Qst–Fst comparisons to investigate whether the mean difference in nose shape among populations is greater than that expected under genetic drift alone. Estimation of Qst relies on stipulating the within- and among-population components of phenotypic variance that are due to additive genetic effects. These variance components are ideally calculated through experiments in which the effects of environmental variables can be controlled [<a href="#pgen.1006616.ref028">28</a>]. This poses a practical and ethical challenge in non-model organisms, such as humans, who are not amenable to the type of experimentation required. To circumvent these limitations, studies often calculate Qst directly from phenotype data under the assumption that the within-population heritability (<em>h</em><sup><em>2</em></sup>) is equal to the among-population heritability (<em>c</em>). Qst calculated this way is probably best considered to be the ‘minimum’ Qst proposed by Relethford (1994) [<a href="#pgen.1006616.ref035">35</a>]. However, the notion that this takes the minimum value of Qst relies on <em>h</em><sup>2</sup> always being less than <em>c</em>, which is anticonservative as has been noted previously [<a href="#pgen.1006616.ref032">32</a>]. In fact, genetic variation between-populations is likely to be less than the genetic variation within-populations for most phenotypes. Our approach is similar to previous studies in that we also estimated Qst assuming <em>c</em> = <em>h</em><sup><em>2</em></sup>. However, we used sensitivity curves to evaluate the behavior of Qst to cases where <em>c</em> &lt; <em>h</em><sup><em>2</em></sup>. In addition, we demonstrate that both within- and between-population variation in nose shape are heritable.</p>
<p>We carried out Qst–Fst comparisons for seven nose shape traits: nares width, alar base width, nasal height, length of the nasal ridge, nasal tip protrusion, external surface area, and nostril area. We found that the divergence in the width of the nares and alar base deviate from neutral expectations, and that these results are robust to the <em>c</em> = <em>h</em><sup><em>2</em></sup> assumption. This suggests that the width of the nares and alar base may have evolved across populations due to divergent selection.</p>
<p>Next, we hypothesized that the divergence of these two traits is driven by climate adaptation. To investigate this, we tested whether the spatial distribution of these traits is correlated with mean annual temperature and humidity. Our results show that nares width is strongly correlated with temperature and absolutely humidity. The positive direction of the effects indicate that wider noses are more common in warm-humid climates, while narrower noses are more common in cold-dry climates. Nares width is not, however, correlated with relative humidity. This is not surprising since absolute humidity levels are likely more important for respiration than relative humidity [<a href="#pgen.1006616.ref043">43</a>]. Alar base width is only weakly, if at all, correlated with temperature and absolute humidity, suggesting that the signal of climate adaptation might be specific to the width of the nares. Computational fluid dynamics (CFD) studies show that the geometry of the nose is important for its respiratory functions [<a href="#pgen.1006616.ref005">5</a>,<a href="#pgen.1006616.ref044">44</a>]. Inhaled air reaches 90% of the required temperature and humidity levels before even reaching the nasopharynx, implicating the nasal cavity, especially the turbinates, as the major conditioning apparatus in the respiratory tract [<a href="#pgen.1006616.ref004">4</a>,<a href="#pgen.1006616.ref005">5</a>]. We also know that the geometry of the nasal airways influences the velocity of inspired air [<a href="#pgen.1006616.ref004">4</a>,<a href="#pgen.1006616.ref007">7</a>,<a href="#pgen.1006616.ref045">45</a>]. Narrow airways in cold-dry climates might allow better conditioning by increasing the turbulence in inspired air as it reaches the turbinates, thereby facilitating contact with the nasal mucosa [<a href="#pgen.1006616.ref005">5</a>]. However, we note that nostril area does not show unusually high differentiation across populations, which suggests that it is not the size of the nostrils but the shape that might be functionally important.</p>
<p>Another hypothesis regarding the function of the nose that has received less attention compared to the air-conditioning hypothesis is that of selective brain cooling (SBC) [<a href="#pgen.1006616.ref046">46</a>]. Some large mammals are known to regulate brain temperatures in hyperthermic conditions, often caused by sustained exercise [<a href="#pgen.1006616.ref047">47</a>,<a href="#pgen.1006616.ref048">48</a>]. This ability is mostly reported in mammals with a carotid rete, which is a network of capillaries in a cavity under the brain called the cavernous sinus. Arterial blood leaving the rete eventually perfuses the brain through the Circle of Willis. Venous blood traveling back from the nasal mucosa interacts with the carotid rete in the cavernous sinus and is thought to cool the blood entering the brain in arid-zone mammals, such as Oryx [<a href="#pgen.1006616.ref049">49</a>], as well as in winter-acclimatized animals, such as reindeer [<a href="#pgen.1006616.ref050">50</a>]. Although, humans do not possess such a rete and instead have a single artery going through the cavernous sinus, which has led some to question the role of SBC in humans [<a href="#pgen.1006616.ref051">51</a>], proponents argue that the carotid rete is not a pre-requisite as SBC is present in some animals who don’t possess a rete, such as horses [<a href="#pgen.1006616.ref052">52</a>]. Since SBC in humans is highly debated, the role of nose shape differences in contributing to differences to this physiological process, while worth mentioning here, is highly speculative.</p>
<p>Climate may not have been the only factor in contributing to nose shape differences across populations. In fact, we show that temperature is only weakly correlated with nares width, especially when compared with the correlation between skin pigmentation and UVB. What then could be the selective agent driving the divergence of nose shape? We mentioned earlier that all aspects of nose shape studied here are sexually dimorphic, which raises a number of questions. Why does this sexual dimorphism exist? Is it merely a by-product of circulating hormones leading to differences in growth and development in early adulthood, or does it have an adaptive function, such as signaling sex to other males and females? Sexual selection has likely played an important role in human evolution, as evidenced by the presence of sexual dimorphism in many physical traits (e.g., height, waist-hip ratio, facial hair, and breasts to name a few). Could cultural differences in perceptions of dominance and attractiveness have had a role in the divergence of nose shape [<a href="#pgen.1006616.ref053">53</a>]? Could these perceptions have arisen to select mates who signal adaptation to the local environment? Indeed, ecological selection and sexual selection could reinforce each other, accelerating phenotypic divergence across populations in spite of continued gene flow [<a href="#pgen.1006616.ref054">54</a>]. These are interesting avenues of research, which need to be considered in order to sketch out a more complete picture of the evolution of the human nose.</p>
<p>The investigation of nose shape evolution with respect to climate adaptation, while interesting anthropologically, is also relevant medically. As humans are becoming more of a global community, the study of local adaptation is becoming more important to understanding health risks involved in living in ‘foreign’ climates. Obvious examples of such health risks are of increased risk of sunburn, skin cancer, and folate deficiency in light-skinned individuals exposed to high UVB, and of low birth weight and chronic mountain sickness associated with hypoxia at high altitudes [<a href="#pgen.1006616.ref020">20</a>,<a href="#pgen.1006616.ref055">55</a>]. Does the morphology of the external nose, or that of the inner nasal cavity affect risk of respiratory disease in different climates? It’s difficult to say at this point. While our findings provide support for the idea that differences in aspects of nose shape may have evolved across populations as a result of climate-related selection pressures, something that has been demonstrated previously using craniometric data [<a href="#pgen.1006616.ref002">2</a>,<a href="#pgen.1006616.ref012">12</a>,<a href="#pgen.1006616.ref013">13</a>,<a href="#pgen.1006616.ref041">41</a>,<a href="#pgen.1006616.ref042">42</a>], we note that the signal of climate adaptation is not very strong, especially when compared to skin pigmentation. This could be due to weaker selection pressure or selection on standing variation, but also due the sparse sampling of populations shown here, which is a limitation of this study. These results will need to be replicated in a larger set of populations. We expect that studies incorporating diverse populations who have been living long-term in a range of environments, such as the tropics, deserts, and circumpolar regions, will nicely fill in the gaps. Especially useful would be representation of populations from higher altitude regions, such as Andeans, Tibetans, and Ethiopians, who not only have to cope with the stress of a cold and dry climate, but also that of low atmospheric oxygen levels [<a href="#pgen.1006616.ref056">56</a>]. It would also be informative to study non-human primates in this context, who occupy a variety of climes and exhibit extensive variation in nose morphology. Finally, future studies should also focus on genome-wide association studies (GWASs) to identify variants contributing to nose shape. With increasing interest in identifying loci associated with facial shape, some GWASs have recently identified a number of nose shape loci [<a href="#pgen.1006616.ref057">57</a>–<a href="#pgen.1006616.ref060">60</a>]. Genetic variation at these loci will be informative about the nature of selection, as well as for inferring the timing of selection events.</p>
</div>

<div xmlns:plos="http://plos.org" id="section4"><h2>Materials and methods</h2>
<div id="section1"><h3>Ethics statement</h3>
<p>Data collection was carried out with informed consent of participants and with approval from the institutional review boards (IRBs) at The Pennsylvania State University (IRB# 32341, IRB# 45727, IRB# 44929) and at The University of Illinois Urbana-Champaign (IRB# 13103). All participants provided written informed consent.</p>
</div>

<div id="section2"><h3>Participant recruitment and description of the data</h3>
<p>The data used in this paper are part of a larger dataset that were collected in various locations through studies based at the Pennsylvania State University and at the University of Illinois at Urbana-Champaign. The majority of these data were collected in the United States at the Pennsylvania State University, at the World Science Festival in New York, and at the University of Illinois at Urbana-Champaign. Data collection in Europe took place in Dublin (Ireland), in Rome (Italy), in Warsaw (Poland), and in Porto (Portugal). The Cape Verdean data were collected in collaboration with Dr. Sandra Beleza as described in Beleza <em>et al</em>. [<a href="#pgen.1006616.ref061">61</a>]. All data were collected through informed consent from the participants with approval from the institutional review boards at The Pennsylvania State University, at the University of Illinois at Urbana-Champaign, and the National Ethical Committee for Health Research of Cape Verde.</p>
<p>The phenotype data collected include three dimensional (3D) images, skin pigmentation, standing height, and body weight. 3D images of participants were taken using the 3dMD system (<em>3dMD System</em>, <em>Atlanta</em>, <em>GA</em>). Skin pigmentation was measured with melanin index using Derma Spectrometer (<em>Cortex Technology</em>, <em>Hadsund</em>, <em>Denmark</em>). Melanin index is defined as the 100 x log<sub>10</sub> (1/percentage reflectance at 650 nM) [<a href="#pgen.1006616.ref017">17</a>]. We took three readings of melanin index from the inner left and right arm and used the mean of the six measurements for each individual. Statistical testing was performed using the inverse of melanin index since it is more normally distributed (<a href="#pgen.1006616.s007">S7 Fig</a>) but the use of untransformed melanin index produces similar results and does not affect our conclusions. Demographic information was collected through questionnaires and included self-reported ancestry and sex, participant’s birthplace, the birthplaces of the parents and grandparents, and the locations where the participant was raised.</p>
<p>The total number of participants was 4,257. Of these, we selected 2,637 individuals based on genetic ancestry and availability of phenotypic and covariate data, as described in the following sections. A summary of the sample sizes used in different analyses is provided in <a href="#pgen-1006616-t005">Table 5</a>. For a more detailed and accurate representation, refer to the Excel spreadsheet in the supplementary materials (<a href="#pgen.1006616.s009">S1 File</a>), which provides individual-level data for all samples used to generate the results presented here.</p>
</div>

<div id="section3"><h3>Estimation of genetic ancestry</h3>

<div id="section1"><h4>Genotyping and quality control.</h4><p>Out of the 4,257 participants, we had genotype data for 3,746 individuals. Since the data were collected as part of different studies over a long period of time, the samples were genotyped on four different genotyping arrays, namely the Illumina HumanHap300v1 BeadChip, the Illumina Infinium HD Human 1M-Duo Beadarray (<em>Illumina</em>, <em>Inc</em>., <em>San Diego</em>, <em>CA</em>), and the 23andMe v3 and v4 arrays (<em>23andMe Mountainview</em>, <em>CA)</em>. In order to select individuals based on genetic ancestry, we needed to merge genotypes across platforms into a single database. We took several steps to ensure there were no batch effects due to genotyping platform, which could potentially lead to biases in ancestry estimation and Fst calculation. First, we removed SNPs that were not present on all four platforms to ensure there were no systematic differences due to missingness. Second, we performed quality-control separately for each platform by removing individuals and SNPs with a genotyping rate of less than 90%. Finally, we removed palindromic (AT/GC) SNPs and recoded the data in the Illumina A/B format [<a href="#pgen.1006616.ref062">62</a>] to ensure strand differences across platforms were not an issue. This sufficiently removes any potential batch effects due to platform as evidenced by the observation that in a multidimensional scaling plot, individuals cluster by ancestry and not by platform (<a href="#pgen.1006616.s001">S1 Fig</a>). At this stage, the genotypes were merged across platforms into a single dataset. We further removed non-autosomal SNPs from the merged dataset and pruned for linkage disequilibrium (using a pairwise r<sup>2</sup> cutoff of 0.5). This resulted in 118,420 autosomal SNPs. Related individuals were identified using identity-by-state (IBS) analysis (IBS &gt; 0.8). Individuals were removed to minimize relatedness and maximize sample size (e.g. In the case of a mother-father-siblings quartet, the siblings were removed). These QC steps were carried out in Plink 1.9 [<a href="#pgen.1006616.ref063">63</a>,<a href="#pgen.1006616.ref064">64</a>].</p>
</div>

<div id="section2"><h4>Genomic ancestry estimation and principal components analysis.</h4><p>We estimated genetic ancestry for all individuals with available genotype data (N = 3,746) using an unsupervised clustering approach in ADMIXTURE. To assist with the clustering and interpretation of the resulting clusters, we merged this dataset with genotypes from the HapMap dataset for the same SNPs (Populations included: YRI, LWK, CEU, TSI, CHB, CHD, JPT, GIH, MEX; N = 988). We ran ADMIXTURE for three values of k (k = 5, 6, 7), and visually determined the optimum clustering scheme to be k = 6 for our purposes. To further visualize fine-scale structure, we carried out unsupervised principal component analysis (PCA) on the genotype data in Plink 1.9 [<a href="#pgen.1006616.ref064">64</a>]. A combination of filters based on the ADMIXTURE output and the PCA scores allowed us to select individuals from specific populations of interest. These are described below separately for each analysis.</p>
</div>
</div>

<div id="section4"><h3>Qst–Fst analyses</h3>

<div id="section1"><h4>Sub-selection of individuals.</h4><p>For the Qst–Fst analyses, we were interested in selecting individuals with ancestry primarily from one of four populations: i) North European, ii) West African, iii) East Asian, and iv) South Asian. Much of this was based on genetic ancestry calculated from dense genotype data. However, for some individuals (see <em>Northern European</em> below), self-reported ancestry information, corroborated through the birthplace of all four grandparents, was used where genotype data were unavailable.</p>
<p><em>West African (N = 40)</em>: We identified 40 individuals who had close to 90% or greater ancestry from ADMIXTURE cluster 1. ADMIXTURE cluster 1 represents ancestry from West Africa, given that individuals from the HapMap YRI (Yoruba in Ibadan, Nigeria) derive most of their ancestry from ADMIXTURE cluster 1 (<a href="#pgen-1006616-g008">Fig 8</a>). These 40 individuals were included in the ‘West African’ population group.</p>
<div data-doi="10.1371/journal.pgen.1006616.g008"><div><p><a title="Click for larger image" href="https://journals.plos.org/plosgenetics/article/figure/image?size=medium&amp;id=10.1371/journal.pgen.1006616.g008" data-doi="10.1371/journal.pgen.1006616" data-uri="10.1371/journal.pgen.1006616.g008"><img src="https://journals.plos.org/plosgenetics/article/figure/image?size=inline&amp;id=10.1371/journal.pgen.1006616.g008" alt="thumbnail" loading="lazy"/></a></p></div><p><span>Fig 8. </span> ADMIXTURE Ancestry proportions (k = 6) of genotyped individuals used in Qst–Fst analyses.</p><p>Their ADMIXTURE proportions are compared with samples from the HapMap dataset. Each vertical bar in the panels is an individual and the colors represent the proportion of ancestry derived from each of 6 clusters (k = 6). In each panel, the HapMap samples are arranged on the left with a three-letter acronym for the population they are from (e.g. GIH refers to the Gujarati Indians from Houston) listed under them. The samples from our dataset are arranged to the right of the HapMap samples with their population designation (e.g. South Asian) under them.</p>
<p><a href="https://doi.org/10.1371/journal.pgen.1006616.g008">
              https://doi.org/10.1371/journal.pgen.1006616.g008</a></p></div><p><em>East Asian (N = 127)</em>: Comparison with the HapMap CHB (Han Chinese in Beijing, China), HapMap CHD (Chinese in Metropolitan Denver, Colorado), and HapMap JPT (Japanese in Tokyo, Japan) samples shows that cluster 4 represents genetic ancestry from East Asia (<a href="#pgen-1006616-g008">Fig 8</a>). We selected 166 individuals who had 90% or more ancestry from ADMIXTURE cluster 4 to be included in the ‘East Asian’ population group (<a href="#pgen-1006616-g008">Fig 8</a>). We further examined the distribution of the PCA plots and removed six individuals who clustered far from the CHB, CHD, and JPT samples. The clustering of the remaining 160 individuals around the CHB, CHD, and JPT samples is shown in <a href="#pgen.1006616.s002">S2 Fig</a>. Of these, we had high quality 3D images and complete covariate data for 127 individuals, who were then used in our analyses.</p>
<p><em>South Asian (N = 73)</em>: 98 individuals in our dataset had more than 60% ancestry from ADMIXTURE cluster 2, which seems to represent the South Asian cluster (<a href="#pgen-1006616-g008">Fig 8</a>). These 98 individuals were added to the ‘South Asian’ population group. While the cutoff here seems liberal compared to that used for the other populations, it is only meant to be a rough guide and was chosen by comparing with the ancestry proportions of the HapMap GIH (Gujarati Indians in Houston, Texas) samples (<a href="#pgen-1006616-g008">Fig 8</a>). However, we examined the distribution of the 98 individuals around the GIH (Gujarati Indians in Houston, Texas) samples using PCA plots (<a href="#pgen.1006616.s002">S2 Fig</a>), and noticed that six individuals appear to fall far from the main cluster, who were subsequently removed. Of the remaining, we had high quality 3D images and complete covariate data for 73 individuals, who were used for further analyses.</p>
<p><em>Northern European (N = 236)</em>: Our full dataset had a large number of individuals with European ancestry from different regions of Europe. In order to minimize structure within Europe, we only retained individuals with Northern European ancestry. Thus, we restricted the analysis to Europeans collected in Ireland (N = 151) and Poland (N = 85) who reported that the ancestry of all four of their grandparents was from the region of sampling. More details for these samples are described in Candille <em>et al</em>. [<a href="#pgen.1006616.ref065">65</a>]. Of this set, we had genotypes for 73 females (Irish = 37, Polish = 36) but none for males. This is the reason why only females were included in the phenotype-climate analyses, where genotypes were used to construct the genetic relationship matrix. The ADMIXTURE results for these women are shown in <a href="#pgen-1006616-g008">Fig 8</a> alongside the results of the CEU (Utah residents with Northern and Western European ancestry from the CEPH collection) and TSI (Tuscans from Italy) samples.</p>
</div>

<div id="section2"><h4>Qst calculation.</h4><p>Estimation of the variance components, <span><img src="https://journals.plos.org/plosgenetics/article/file?type=thumbnail&amp;id=10.1371/journal.pgen.1006616.e015" loading="lazy"/></span> and <span><img src="https://journals.plos.org/plosgenetics/article/file?type=thumbnail&amp;id=10.1371/journal.pgen.1006616.e016" loading="lazy"/></span>, was carried out using a linear model in which population group was treated as a fixed effect, with sex, age, and BMI as covariates. If <em>MSB</em> is the mean square among populations, <em>MSW</em> is the mean square within populations, <em>n</em><sub><em>i</em></sub> is the number of individuals in the i<sup>th</sup> populations, and <em>a</em> is the number of populations, <span><img src="https://journals.plos.org/plosgenetics/article/file?type=thumbnail&amp;id=10.1371/journal.pgen.1006616.e017" loading="lazy"/></span> and <span><img src="https://journals.plos.org/plosgenetics/article/file?type=thumbnail&amp;id=10.1371/journal.pgen.1006616.e018" loading="lazy"/></span> can be estimated as described as follows [<a href="#pgen.1006616.ref010">10</a>,<a href="#pgen.1006616.ref066">66</a>,<a href="#pgen.1006616.ref067">67</a>]:
<a name="pgen.1006616.e019" id="pgen.1006616.e019"></a><span><img src="https://journals.plos.org/plosgenetics/article/file?type=thumbnail&amp;id=10.1371/journal.pgen.1006616.e019" loading="lazy"/><span>(3)</span></span>
<a name="pgen.1006616.e020" id="pgen.1006616.e020"></a><span><img src="https://journals.plos.org/plosgenetics/article/file?type=thumbnail&amp;id=10.1371/journal.pgen.1006616.e020" loading="lazy"/><span>(4)</span></span>
<a name="pgen.1006616.e021" id="pgen.1006616.e021"></a><span><img src="https://journals.plos.org/plosgenetics/article/file?type=thumbnail&amp;id=10.1371/journal.pgen.1006616.e021" loading="lazy"/><span>(5)</span></span></p>
</div>

<div id="section3"><h4>Fst calculation.</h4><p>Wright’s Fst measures the genetic divergence between two or more populations. We estimated the Fst across the four populations for the LD-pruned set of 118,420 autosomal SNPs, described above, using Weir and Cockerham’s <em>θ</em> [<a href="#pgen.1006616.ref068">68</a>]. The genomic Fst distribution in shown in <a href="#pgen.1006616.s003">S3 Fig</a>.</p>
</div>

<div id="section4"><h4>Qst–Fst comparison.</h4><p>We used a non-parametric bootstrap approach to test whether the Qst of a phenotype is significantly greater than the Fst. This was done by computing Qst from subsamples generated by randomly selecting individuals with replacement, within population and sex such that the number of males and females in each population remains unchanged. Values of Fst were simulated by randomly sampling from the Fst of 118,420 SNPs (<a href="#pgen.1006616.s003">S3 Fig</a>). We generated 10,000 such pseudo-samples and, for each simulated value of Qst and Fst, computed the difference Qst–Fst. High values of Qst–Fst are indicative of divergent selection, and the empirical p-value for this test was determined by calculating the proportion of Qst–Fst values that are less than zero.</p>
</div>
</div>

<div id="section5"><h3>Estimation of <span><img src="https://journals.plos.org/plosgenetics/article/file?type=thumbnail&amp;id=10.1371/journal.pgen.1006616.e022" loading="lazy"/></span> and <em>c</em></h3>

<div id="section1"><h4>Estimation of <span><img src="https://journals.plos.org/plosgenetics/article/file?type=thumbnail&amp;id=10.1371/journal.pgen.1006616.e023" loading="lazy"/></span>.</h4><p>For the calculation of <span><img src="https://journals.plos.org/plosgenetics/article/file?type=thumbnail&amp;id=10.1371/journal.pgen.1006616.e024" loading="lazy"/></span>, we identified 1,825 unrelated individuals with primarily European ancestry, who derived more than 80% of their combined genetic ancestry from Cluster 3 and Cluster 6 based on the ADMIXTURE output (<a href="#pgen-1006616-g009">Fig 9</a>). This cutoff was chosen based on comparison with the genetic ancestries of CEU (Utah residents with Northern and Western European ancestry from the CEPH collection) and TSI (Toscans in Italy) individuals from the HapMap dataset (<a href="#pgen-1006616-g009">Fig 9</a>). We used GCTA to calculate the genome-wide kinship matrix for the 1,825 individuals from 118,420 autosomal SNPs, which overlapped across all four platforms. Of the 1,825 individuals with European ancestry, we had measured height for 1,825, measured melanin index for 1,231, and 3D photos for 1,718 individuals. From these data, we used GCTA to estimate the proportion of phenotypic variance explained by genotyped SNPs (<em>h</em><sub><em>g</em></sub><sup><em>2</em></sup>) for height, skin pigmentation, and nose shape aspects. The top four genetic PCs were included as covariates in the model to correct for population structure, which can inflate heritability estimates [<a href="#pgen.1006616.ref018">18</a>,<a href="#pgen.1006616.ref069">69</a>,<a href="#pgen.1006616.ref070">70</a>]. Also included in the model were sex, age, and BMI.</p>
<div data-doi="10.1371/journal.pgen.1006616.g009"><div><p><a title="Click for larger image" href="https://journals.plos.org/plosgenetics/article/figure/image?size=medium&amp;id=10.1371/journal.pgen.1006616.g009" data-doi="10.1371/journal.pgen.1006616" data-uri="10.1371/journal.pgen.1006616.g009"><img src="https://journals.plos.org/plosgenetics/article/figure/image?size=inline&amp;id=10.1371/journal.pgen.1006616.g009" alt="thumbnail" loading="lazy"/></a></p></div><p><span>Fig 9. </span> ADMIXTURE ancestry proportions (k = 6) of 1,825 individuals with primarily European ancestry, who were used to estimate <em>h<sub>g</sub><sup>2</sup></em>.</p><p>Each bar represents an individual and the different colors represent the ADMIXTURE proportions from six clusters (k = 6). The ADMIXTURE proportions for HapMap European populations (CEU and TSI) are shown on the left for comparison.</p>
<p><a href="https://doi.org/10.1371/journal.pgen.1006616.g009">
              https://doi.org/10.1371/journal.pgen.1006616.g009</a></p></div></div>

<div id="section2"><h4>Estimation of c.</h4><p>We estimated the proportion of phenotypic variance between populations due to additive genetic effects (<em>c</em>) in an admixed population from Cape Verde, who derive their ancestry primarily from W. Africans and Europeans (<a href="#pgen-1006616-g010">Fig 10</a>), using the approach described in Zaitlen <em>et al</em>. [<a href="#pgen.1006616.ref040">40</a>]. 697 Cape Verdeans were genotyped on the Illumina Infinium HD Human 1M-Duo Beadarray (<em>Illumina</em>, <em>Inc</em>., <em>San Diego</em>, <em>CA</em>) for a total of 1,016,423 SNPs. This was reduced to 623,625 autosomal SNPs after removing palindromic SNPs, SNPs that did not overlap with the HapMap dataset, and other SNPs based on filters described in Beleza <em>et al</em>. [<a href="#pgen.1006616.ref017">17</a>]. We had high quality 3D images and complete covariate data for 409 of the 697 Cape Verdeans who were genotyped. For these individuals, we calculated local ancestry at 623,625 autosomal SNPs with LAMP-LD [<a href="#pgen.1006616.ref071">71</a>] using phased genotypes from the HapMap CEU and YRI samples as reference. Averaging local ancestry across all SNPs yields an estimate of global ancestry that is highly correlated with estimates obtained from ADMIXTURE using k = 2 (r<sup>2</sup> ∼ 1) (<a href="#pgen.1006616.s004">S4 Fig</a>). We used GCTA to calculate a kinship matrix based on local ancestry and to estimate the proportion of phenotypic variance explained by local ancestry (<em>h</em><sub><em>y</em></sub><sup><em>2</em></sup>). Global ancestry, calculated from the mean of local ancestry across all SNPs, was included as a covariate to correct for ancestry stratification and any environmental variables that might covary with ancestry [<a href="#pgen.1006616.ref016">16</a>,<a href="#pgen.1006616.ref072">72</a>]. Sex, age, and BMI were also included as covariates.</p>
<div data-doi="10.1371/journal.pgen.1006616.g010"><div><p><a title="Click for larger image" href="https://journals.plos.org/plosgenetics/article/figure/image?size=medium&amp;id=10.1371/journal.pgen.1006616.g010" data-doi="10.1371/journal.pgen.1006616" data-uri="10.1371/journal.pgen.1006616.g010"><img src="https://journals.plos.org/plosgenetics/article/figure/image?size=inline&amp;id=10.1371/journal.pgen.1006616.g010" alt="thumbnail" loading="lazy"/></a></p></div><p><span>Fig 10. </span> ADMIXTURE ancestry proportions (k = 6) of 409 Cape Verdeans, who were used to estimate <em>c</em>.</p><p>Each bar represents an individual and the different colors represent the ADMIXTURE proportions from six clusters (k = 6).</p>
<p><a href="https://doi.org/10.1371/journal.pgen.1006616.g010">
              https://doi.org/10.1371/journal.pgen.1006616.g010</a></p></div></div>
</div>

<div id="section6"><h3>Phenotype-climate associations</h3>

<div id="section1"><h4>Testing for local adaptation.</h4><p>The primary focus here was to answer the question: Does nose shape variation between geographically distant populations show signals of local adaptation? To answer this, we needed to ascribe a climate value to every individual that can serve as a proxy of their ancestral environment. For this reason, we selected individuals used in the Qst–Fst analyses who reported at least one parent’s birthplace. These were further filtered to retain individuals whose parents’ birthplaces coincided with their continental ancestry, for example, East Asian individuals whose parents were born in the United States were removed. Finally, we restricted the analyses to females since genetic data were not available for males sampled in Ireland and Poland. This resulted in 140 females with genetic, phenotypic, and geographic data. We have plotted the geographic distribution of the parents of these individuals in <a href="#pgen-1006616-g006">Fig 6</a> with a line connecting the two parents of each individual. A single dot means that the parents were born in the same location. These locations were geocoded to obtain the latitude and longitude in decimal degrees using the ggmap package in R [<a href="#pgen.1006616.ref073">73</a>,<a href="#pgen.1006616.ref074">74</a>].</p>
<p>Gridded climate data were obtained in latitude/longitude coordinate system. Climate grids with mean monthly values for temperature (measured in degrees Celsius), and vapor pressure (measured in hectopascal or hPa) at 0.5° resolution for the period between 1901 and 2015 were obtained from The Climate Research Unit TS v3.24 Dataset (<a href="https://crudata.uea.ac.uk/cru/data/hrg/cru_ts_3.24/">https://crudata.uea.ac.uk/cru/data/hrg/cru_ts_3.24/</a>) [<a href="#pgen.1006616.ref075">75</a>]. Ground-level Ultraviolet-B (UVB) irradiance data (expressed in J/m<sup>2</sup>), for the period between 1996 and 2005, generated by the Earth Probe Total Ozone Mapping Spectrometer (EP/TOMS) Team at 0.5° resolution, were downloaded from the Goddard Earth Sciences Data and Information Services Center (GES DISC) website (<a href="https://disc.gsfc.nasa.gov/">https://disc.gsfc.nasa.gov/</a>) [<a href="#pgen.1006616.ref076">76</a>]. Relative humidity and absolute humidity grids were derived from temperature and vapor pressure as described in Maddux <em>et al</em>. (2016) [<a href="#pgen.1006616.ref043">43</a>]. For each of the four climate variables (UVB, temperature, absolute humidity, and relative humidity), a single grid was constructed by averaging across all months within the time period to represent long-term climate. R packages RnetCDF [<a href="#pgen.1006616.ref077">77</a>] and Raster [<a href="#pgen.1006616.ref078">78</a>] were used to read gridded data and to extract climate values at each location. We used climate values at the parents’ birthplace to represent the ancestral climate for each individual. Where the birthplace of both parents was known, the mean climate values extracted at the two locations was used. Both parents usually come from nearby regions, at least on a continental scale (<a href="#pgen-1006616-g006">Fig 6</a>), and the climate values for the parents are quite similar (<a href="#pgen.1006616.s005">S5 Fig</a>).</p>
<p>We used a linear mixed model to test the relationship between phenotype and climate variables, while correcting for age and BMI. We took non-independence among observations due to genetic similarity into account by allowing the observations to be correlated according to the SNP-based genome-wide kinship matrix. In matrix notation, the linear model takes the following form:
<a name="pgen.1006616.e025" id="pgen.1006616.e025"></a><span><img src="https://journals.plos.org/plosgenetics/article/file?type=thumbnail&amp;id=10.1371/journal.pgen.1006616.e025" loading="lazy"/><span>(6)</span></span>
Where <em>β</em> are fixed effects and <em>ε</em> is the random error term. Thus, <em>ε</em> ∼ <em>N</em>(0,<em>Gσ</em><sub><em>e</em></sub>), where <em>G</em> is the genome-wide kinship matrix, which was constructed from the genotypes at 118,420 SNPs in GCTA [<a href="#pgen.1006616.ref037">37</a>]. The slopes were estimated using maximum likelihood, implemented in the spaMM package in R [<a href="#pgen.1006616.ref074">74</a>,<a href="#pgen.1006616.ref079">79</a>]. P-values for each slope were generated from the likelihood ratio between full and reduced models (fit by removing each climate predictor).</p>
</div>
</div>

<div id="section7"><h3>Morphometric measurements</h3>

<div id="section1"><h4>Imaging processing.</h4><p>We captured high resolution 3D images of participants’ faces using the 3dMD Face system (<em>3dMD Atlanta</em>, <em>GA</em>). Five positioning landmarks (two on the inner corner of the eyes, two on the outer corners of the mouth, and one on the tip of the nose) were placed on each face in order to establish facial orientation. A spatially dense mesh of 7,150 quasi-landmarks (QLs) was mapped onto each image and its reflection. As a result of this mapping, the vertices and triangles are homologous across individuals [<a href="#pgen.1006616.ref080">80</a>]. Generalized Procrustes Superimposition [<a href="#pgen.1006616.ref024">24</a>] was carried out on both sets of images (original and reflected) to remove differences in position and orientation. The Procrustes coordinates of the original and reflected image for each participant were then averaged to remove effects of bilateral asymmetry. We selected the nose region (see outline in <a href="#pgen-1006616-g001">Fig 1</a>), which is comprised of 709 out of the 7,150 QLs making up the entire face, for further downstream analyses.</p>
</div>

<div id="section2"><h4>Linear distances.</h4><p>Linear distances were calculated using three mid-line landmarks; i. nasion (n), ii. pronasale (prn), iii. subnasale (sn), and two sets of paired landmarks; iv. left and right alar curvature (al<sub>l</sub> and al<sub>r</sub>), and v. left and right alar base (ac<sub>l</sub> and ac<sub>r</sub>). These landmarks have been defined previously [<a href="#pgen.1006616.ref001">1</a>,<a href="#pgen.1006616.ref026">26</a>] and their approximate positions are shown in <a href="#pgen-1006616-g001">Fig 1</a>. Two different observers placed each of these seven landmarks three different times on the spatially-dense QL template mesh that was mapped on all the faces. Because the QLs are homologous across individuals, we can automatically select corresponding landmarks for all individuals by indicating these on the QL template. This is possible because any point on the template surface can be expressed as a function of QLs using barycentric coordinates on triangles. Using this approach, we obtained the x, y, and z coordinates of the above-mentioned landmarks for each individual. The Euclidean distances (in mm) were then calculated separately for each of the six replicate placements, which were averaged across replicates and then averaged between the two observers, to obtain mean distances for each individual.</p>
</div>

<div id="section3"><h4>Nostril area.</h4><p>To measure area of the nostrils, the observers defined regions around the nostrils (approximate location shown in <a href="#pgen-1006616-g001">Fig 1</a>) three different times. The area of a region was calculated as the total area of the polygons within the region. Using this approach, we calculated the mean area of the right and left nostrils (in mm<sup>2</sup>) for each replicate. This was then averaged across replicates for the same observer, then across observers to obtain the mean nostril area for each individual.</p>
</div>

<div id="section4"><h4>External surface area.</h4><p>The external surface area of the nose (also in mm<sup>2</sup>) was calculated by summing the areas of polygons bounded by the 709 QLs, defining the entire nose region (outlined in <a href="#pgen-1006616-g001">Fig 1</a>).</p>
</div>

<div id="section5"><h4>Intra-observer error and inter-observer reliability.</h4><p>We calculated the standard deviation in the measurement of linear distances (in mm) and nostril area (in mm<sup>2</sup>) made by the two observers to estimate the degree of intra-observer error. The quantiles for these standard deviations are shown in <a href="#pgen-1006616-t006">Table 6</a>. The measurements from Observer2 are slightly more internally consistent than Observer1. For Observer1, nasal tip protrusion was the least precise distance measure with 95% of the standard deviations falling between 0.791mm and 1.165mm, with error for the remaining distances well under 1mm. The error distribution for nostril area was similar for both Observer1 (95% CI: 0.914–1.930) and Observer2 (95% CI: 0.888–2.243).</p>
<p>Inter-observer reliability was evaluated using intraclass correlation coefficient (ICC). ICC for each measure was calculated using linear mixed models where the observer was treated as fixed effect and subject was treated as random effect. This was carried out using the nlme package in R [<a href="#pgen.1006616.ref074">74</a>,<a href="#pgen.1006616.ref081">81</a>]. With the exception of nasal tip protrusion (ICC = 0.819), the ICCs for all other measures were quite high (ICC &gt; 0.95). This concordance is evident by comparing the mean measurements between Observer1 and Observer2 (<a href="#pgen.1006616.s006">S6 Fig</a>)</p>
</div>
</div>
</div>

<div xmlns:plos="http://plos.org" id="section5"><h2>Supporting information</h2><div><h3><a href="https://journals.plos.org/plosgenetics/article/file?type=supplementary&amp;id=10.1371/journal.pgen.1006616.s001">S1 Fig. </a></h3><p><strong>MDS plots (k = 2) showing clustering of genotyped individuals by A) genotyping platform and B) by ancestry.</strong> In B) the genotyped individuals from the study set are shown in black, while the HapMap samples are shown in color for reference. The plot shows that individuals cluster by ancestry and not by genotyping platform. Thus, batch effects due to genotyping platform are unlikely to lead to artifacts in ancestry inference.</p>
<p><a href="https://doi.org/10.1371/journal.pgen.1006616.s001">https://doi.org/10.1371/journal.pgen.1006616.s001</a></p><p>(TIF)</p>
</div><div><h3><a href="https://journals.plos.org/plosgenetics/article/file?type=supplementary&amp;id=10.1371/journal.pgen.1006616.s002">S2 Fig. </a></h3><p><strong>PCA plots showing clustering of genotyped individuals by A) South Asian and B) East Asian ancestry around HapMap reference samples.</strong> The colored points are reference samples from the HapMap dataset and the grey points are individuals from our dataset. A) South Asian individuals are shown with the HapMap GIH samples and B) East Asian individuals are shown with the combined East Asian sample (CHB+JPT+CHD) from the HapMap dataset. The red lines indicate the cutoff for removing individuals who appear to cluster far away from the main cluster. The individuals who were removed, based on clustering, are shown as triangles, whereas all other individuals are shown as circles.</p>
<p><a href="https://doi.org/10.1371/journal.pgen.1006616.s002">https://doi.org/10.1371/journal.pgen.1006616.s002</a></p><p>(TIF)</p>
</div><div><h3><a href="https://journals.plos.org/plosgenetics/article/file?type=supplementary&amp;id=10.1371/journal.pgen.1006616.s004">S4 Fig. </a>Global ancestry calculated using LAMP-LD is highly correlated with global ancestry calculated using ADMIXTURE.</h3><p>ADMIXTURE-based ancestry estimates were calculated using a k = 2 assumption. LAMP-LD-based ancestry estimates were calculating the mean local ancestry across 623,625 autosomal SNPs.</p>
<p><a href="https://doi.org/10.1371/journal.pgen.1006616.s004">https://doi.org/10.1371/journal.pgen.1006616.s004</a></p><p>(TIF)</p>
</div><div><h3><a href="https://journals.plos.org/plosgenetics/article/file?type=supplementary&amp;id=10.1371/journal.pgen.1006616.s007">S7 Fig. </a></h3><p><strong>QQplots showing that the residuals for A) the inverse of melanin index are more normally distributed than B) untransformed melanin index.</strong> The points are residuals from linear models where population, sex, age, and BMI were used as predictors.</p>
<p><a href="https://doi.org/10.1371/journal.pgen.1006616.s007">https://doi.org/10.1371/journal.pgen.1006616.s007</a></p><p>(TIF)</p>
</div></div>





<div xmlns:plos="http://plos.org"><h2>Acknowledgments</h2>
<p>The authors would first and foremost like to thank the study participants, without whom none of this would have been possible. We are grateful to the Shriver Lab team, Ripan Malhi, and the members of his lab for their efforts in collection and organization of the data. We are also grateful to Sandra Beleza and Isabel Inês Araújo for their efforts in collecting and sharing the Cape Verdean data. We acknowledge Ine Saye from Peter Claes’ group for helping with morphometric analyses. We also thank Nina Jablonski, George Chaplin, George Perry, Laurel Pearson, Julie White, and Matthew Reimherr for helpful discussions on the analytical aspects of the paper. Finally, we would like to thank two anonymous reviewers for providing helpful critique and comments on the first submission of this manuscript.</p>
</div><div xmlns:plos="http://plos.org"><h2>Author Contributions</h2><ol>

<li><strong>Conceptualization:</strong> AAZ MDS.</li>

<li><strong>Data curation:</strong> AAZ BCM MDS BM.</li>

<li><strong>Formal analysis:</strong> AAZ.</li>

<li><strong>Funding acquisition:</strong> MDS PC CH.</li>

<li><strong>Investigation:</strong> AAZ BCM CH MDS.</li>

<li><strong>Methodology:</strong> AAZ MDS.</li>

<li><strong>Project administration:</strong> MDS CH BCM BM.</li>

<li><strong>Resources:</strong> MDS CH PC BM.</li>

<li><strong>Software:</strong> PC AAZ.</li>

<li><strong>Supervision:</strong> MDS.</li>

<li><strong>Visualization:</strong> AAZ PC.</li>

<li><strong>Writing – original draft:</strong> AAZ.</li>

<li><strong>Writing – review &amp; editing:</strong> AAZ MDS PC CH BCM BM.</li>

</ol>
</div><div xmlns:plos="http://plos.org"><h2>References</h2><ol><li id="ref1"><span>1.
            </span><a name="pgen.1006616.ref001" id="pgen.1006616.ref001"></a>Farkas LG, Kolar JC, Munro IR. Geography of the nose: a morphometric study. Aesthetic Plast Surg. 1986;10: 191–223. pmid:3812136 <ul><li><a href="#" data-author="Farkas" data-cit="FarkasLG%2C%20KolarJC%2C%20MunroIR.%20Geography%20of%20the%20nose%3A%20a%20morphometric%20study.%20Aesthetic%20Plast%20Surg.%201986%3B10%3A%20191%E2%80%93223.%203812136" data-title="Geography%20of%20the%20nose%3A%20a%20morphometric%20study" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/3812136" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Geography+of+the+nose%3A+a+morphometric+study+Farkas+1986" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref2"><span>2.
            </span><a name="pgen.1006616.ref002" id="pgen.1006616.ref002"></a>Franciscus RG, Long JC. Variation in human nasal height and breadth. Am J Phys Anthropol. 1991;85: 419–427.  pmid:1928315 <ul data-doi="10.1002/ajpa.1330850406"><li><a href="https://doi.org/10.1002/ajpa.1330850406" data-author="doi-provided" data-cit="doi-provided" data-title="doi-provided" target="_new" title="Go to article">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/1928315" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Variation+in+human+nasal+height+and+breadth+Franciscus+1991" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref3"><span>3.
            </span><a name="pgen.1006616.ref003" id="pgen.1006616.ref003"></a>Negus VE. Introduction to the comparative anatomy of the nose and paranasal sinuses. Ann R Coll Surg Engl. 1954;15: 141–171. pmid:13198060 <ul><li><a href="#" data-author="Negus" data-cit="NegusVE.%20Introduction%20to%20the%20comparative%20anatomy%20of%20the%20nose%20and%20paranasal%20sinuses.%20Ann%20R%20Coll%20Surg%20Engl.%201954%3B15%3A%20141%E2%80%93171.%2013198060" data-title="Introduction%20to%20the%20comparative%20anatomy%20of%20the%20nose%20and%20paranasal%20sinuses" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/13198060" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Introduction+to+the+comparative+anatomy+of+the+nose+and+paranasal+sinuses+Negus+1954" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref4"><span>4.
            </span><a name="pgen.1006616.ref004" id="pgen.1006616.ref004"></a>Hanna LM, Scherer PW. A Theoretical Model of Localized Heat and Water Vapor Transport in the Human Respiratory Tract. J Biomech Eng. American Society of Mechanical Engineers; 1986;108: 19. pmid:3959548 <ul><li><a href="#" data-author="Hanna" data-cit="HannaLM%2C%20SchererPW.%20A%20Theoretical%20Model%20of%20Localized%20Heat%20and%20Water%20Vapor%20Transport%20in%20the%20Human%20Respiratory%20Tract.%20J%20Biomech%20Eng.%20American%20Society%20of%20Mechanical%20Engineers%3B%201986%3B108%3A%2019.%203959548" data-title="A%20Theoretical%20Model%20of%20Localized%20Heat%20and%20Water%20Vapor%20Transport%20in%20the%20Human%20Respiratory%20Tract" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/3959548" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=A+Theoretical+Model+of+Localized+Heat+and+Water+Vapor+Transport+in+the+Human+Respiratory+Tract+Hanna+1986" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref5"><span>5.
            </span><a name="pgen.1006616.ref005" id="pgen.1006616.ref005"></a>Naftali S, Rosenfeld M, Wolf M, Elad D. The Air-Conditioning Capacity of the Human Nose. Ann Biomed Eng. 2005;33: 545–553. pmid:15909660 <ul><li><a href="#" data-author="Naftali" data-cit="NaftaliS%2C%20RosenfeldM%2C%20WolfM%2C%20EladD.%20The%20Air-Conditioning%20Capacity%20of%20the%20Human%20Nose.%20Ann%20Biomed%20Eng.%202005%3B33%3A%20545%E2%80%93553.%2015909660" data-title="The%20Air-Conditioning%20Capacity%20of%20the%20Human%20Nose" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/15909660" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=The+Air-Conditioning+Capacity+of+the+Human+Nose+Naftali+2005" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref6"><span>6.
            </span><a name="pgen.1006616.ref006" id="pgen.1006616.ref006"></a>Randell SH, Boucher RC. Effective mucus clearance is essential for respiratory health. Am J Respir Cell Mol Biol. 2006;35: 20–8.  pmid:16528010 <ul data-doi="10.1165/rcmb.2006-0082SF"><li><a href="https://doi.org/10.1165/rcmb.2006-0082SF" data-author="doi-provided" data-cit="doi-provided" data-title="doi-provided" target="_new" title="Go to article">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/16528010" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Effective+mucus+clearance+is+essential+for+respiratory+health+Randell+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref7"><span>7.
            </span><a name="pgen.1006616.ref007" id="pgen.1006616.ref007"></a>Churchill SE, Shackelford LL, Georgi JN, Black MT. Morphological variation and airflow dynamics in the human nose. Am J Hum Biol. 2004;16: 625–638.  pmid:15495233 <ul data-doi="10.1002/ajhb.20074"><li><a href="https://doi.org/10.1002/ajhb.20074" data-author="doi-provided" data-cit="doi-provided" data-title="doi-provided" target="_new" title="Go to article">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/15495233" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Morphological+variation+and+airflow+dynamics+in+the+human+nose+Churchill+2004" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref8"><span>8.
            </span><a name="pgen.1006616.ref008" id="pgen.1006616.ref008"></a>Weiner JS. Nose Shape and Climate. Am J Phys Anthropol. 1954;12: 615–618. pmid:14350081 <ul><li><a href="#" data-author="Weiner" data-cit="WeinerJS.%20Nose%20Shape%20and%20Climate.%20Am%20J%20Phys%20Anthropol.%201954%3B12%3A%20615%E2%80%93618.%2014350081" data-title="Nose%20Shape%20and%20Climate" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/14350081" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Nose+Shape+and+Climate+Weiner+1954" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref9"><span>9.
            </span><a name="pgen.1006616.ref009" id="pgen.1006616.ref009"></a>Roseman CC, Weaver TD. Multivariate apportionment of global human craniometric diversity. Am J Phys Anthropol. 2004;125: 257–263.  pmid:15386236 <ul data-doi="10.1002/ajpa.10424"><li><a href="https://doi.org/10.1002/ajpa.10424" data-author="doi-provided" data-cit="doi-provided" data-title="doi-provided" target="_new" title="Go to article">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/15386236" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Multivariate+apportionment+of+global+human+craniometric+diversity+Roseman+2004" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref10"><span>10.
            </span><a name="pgen.1006616.ref010" id="pgen.1006616.ref010"></a>Spitze K. Population structure in Daphnia obtusa: quantitative genetic and allozymic variation. Genetics. 1993;135: 367–374. Available: <a href="http://www.genetics.org/content/135/2/367">http://www.genetics.org/content/135/2/367</a> pmid:8244001 <ul><li><a href="#" data-author="Spitze" data-cit="SpitzeK.%20Population%20structure%20in%20Daphnia%20obtusa%3A%20quantitative%20genetic%20and%20allozymic%20variation.%20Genetics.%201993%3B135%3A%20367%E2%80%93374.%20Available%3A%20http%3A%2F%2Fwww.genetics.org%2Fcontent%2F135%2F2%2F367%208244001" data-title="Population%20structure%20in%20Daphnia%20obtusa%3A%20quantitative%20genetic%20and%20allozymic%20variation" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/8244001" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Population+structure+in+Daphnia+obtusa%3A+quantitative+genetic+and+allozymic+variation+Spitze+1993" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref11"><span>11.
            </span><a name="pgen.1006616.ref011" id="pgen.1006616.ref011"></a>Whitlock MC, Guillaume F. Testing for spatially divergent selection: comparing QST to FST. Genetics. 2009;183: 1055–63.  pmid:19687138 <ul data-doi="10.1534/genetics.108.099812"><li><a href="https://doi.org/10.1534/genetics.108.099812" data-author="doi-provided" data-cit="doi-provided" data-title="doi-provided" target="_new" title="Go to article">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/19687138" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Testing+for+spatially+divergent+selection%3A+comparing+QST+to+FST+Whitlock+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref12"><span>12.
            </span><a name="pgen.1006616.ref012" id="pgen.1006616.ref012"></a>Roseman CC. Detecting interregionally diversifying natural selection on modern human cranial form by using matched molecular and morphometric data. Proc Natl Acad Sci U S A. 2004;101: 12824–9.  pmid:15326305 <ul data-doi="10.1073/pnas.0402637101"><li><a href="https://doi.org/10.1073/pnas.0402637101" data-author="doi-provided" data-cit="doi-provided" data-title="doi-provided" target="_new" title="Go to article">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/15326305" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Detecting+interregionally+diversifying+natural+selection+on+modern+human+cranial+form+by+using+matched+molecular+and+morphometric+data+Roseman+2004" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref13"><span>13.
            </span><a name="pgen.1006616.ref013" id="pgen.1006616.ref013"></a>Hubbe M, Hanihara T, Harvati K. Climate signatures in the morphological differentiation of worldwide modern human populations. Anat Rec. 2009;292: 1720–1733. <ul><li><a href="#" data-author="Hubbe" data-cit="HubbeM%2C%20HaniharaT%2C%20HarvatiK.%20Climate%20signatures%20in%20the%20morphological%20differentiation%20of%20worldwide%20modern%20human%20populations.%20Anat%20Rec.%202009%3B292%3A%201720%E2%80%931733." data-title="Climate%20signatures%20in%20the%20morphological%20differentiation%20of%20worldwide%20modern%20human%20populations" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Climate+signatures+in+the+morphological+differentiation+of+worldwide+modern+human+populations+Hubbe+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref14"><span>14.
            </span><a name="pgen.1006616.ref014" id="pgen.1006616.ref014"></a>Guo J, Tan J, Yang Y, Zhou H, Hu S, Hashan A, et al. Variation and signatures of selection on the human face. J Hum Evol. Elsevier Ltd; 2014;75: 143–152.  pmid:25186351 <ul data-doi="10.1016/j.jhevol.2014.08.001"><li><a href="https://doi.org/10.1016/j.jhevol.2014.08.001" data-author="doi-provided" data-cit="doi-provided" data-title="doi-provided" target="_new" title="Go to article">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/25186351" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Variation+and+signatures+of+selection+on+the+human+face+Guo+2014" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref15"><span>15.
            </span><a name="pgen.1006616.ref015" id="pgen.1006616.ref015"></a>Shriver MD, Parra EJ, Dios S, Bonilla C, Norton H, Jovel C, et al. Skin pigmentation, biogeographical ancestry and admixture mapping. Hum Genet. 2003;112: 387–99.  pmid:12579416 <ul data-doi="10.1007/s00439-002-0896-y"><li><a href="https://doi.org/10.1007/s00439-002-0896-y" data-author="doi-provided" data-cit="doi-provided" data-title="doi-provided" target="_new" title="Go to article">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/12579416" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Skin+pigmentation%2C+biogeographical+ancestry+and+admixture+mapping+Shriver+2003" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref16"><span>16.
            </span><a name="pgen.1006616.ref016" id="pgen.1006616.ref016"></a>Parra EJ, Kittles R a, Shriver MD. Implications of correlations between skin color and genetic ancestry for biomedical research. Nat Genet. 2004;36: S54–S60.  pmid:15508005 <ul data-doi="10.1038/ng1440"><li><a href="https://doi.org/10.1038/ng1440" data-author="doi-provided" data-cit="doi-provided" data-title="doi-provided" target="_new" title="Go to article">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/15508005" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Implications+of+correlations+between+skin+color+and+genetic+ancestry+for+biomedical+research+Parra+2004" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref17"><span>17.
            </span><a name="pgen.1006616.ref017" id="pgen.1006616.ref017"></a>Beleza S, Johnson NA, Candille SI, Absher DM, Coram MA, Lopes J, et al. Genetic architecture of skin and eye color in an African-European admixed population. Spritz RA, editor. PLoS Genet. Public Library of Science; 2013;9: e1003372.  pmid:23555287 <ul data-doi="10.1371/journal.pgen.1003372"><li><a href="https://doi.org/10.1371/journal.pgen.1003372" data-author="doi-provided" data-cit="doi-provided" data-title="doi-provided" target="_new" title="Go to article">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/23555287" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Genetic+architecture+of+skin+and+eye+color+in+an+African-European+admixed+population.+Spritz+RA%2C+editor+Beleza+2013" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref18"><span>18.
            </span><a name="pgen.1006616.ref018" id="pgen.1006616.ref018"></a>Yang J, Benyamin B, McEvoy BP, Gordon S, Henders AK, Nyholt DR, et al. Common SNPs explain a large proportion of the heritability for human height. Nat Genet. Nature Publishing Group, a division of Macmillan Publishers Limited. All Rights Reserved.; 2010;42: 565–9.  pmid:20562875 <ul data-doi="10.1038/ng.608"><li><a href="https://doi.org/10.1038/ng.608" data-author="doi-provided" data-cit="doi-provided" data-title="doi-provided" target="_new" title="Go to article">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/20562875" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Common+SNPs+explain+a+large+proportion+of+the+heritability+for+human+height+Yang+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref19"><span>19.
            </span><a name="pgen.1006616.ref019" id="pgen.1006616.ref019"></a>Turchin MC, Chiang CWK, Palmer CD, Sankararaman S, Reich D, Hirschhorn JN. Evidence of widespread selection on standing variation in Europe at height-associated SNPs. Nat Genet. Nature Publishing Group, a division of Macmillan Publishers Limited. All Rights Reserved.; 2012;44: 1015–9.  pmid:22902787 <ul data-doi="10.1038/ng.2368"><li><a href="https://doi.org/10.1038/ng.2368" data-author="doi-provided" data-cit="doi-provided" data-title="doi-provided" target="_new" title="Go to article">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/22902787" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Evidence+of+widespread+selection+on+standing+variation+in+Europe+at+height-associated+SNPs+Turchin+2012" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref20"><span>20.
            </span><a name="pgen.1006616.ref020" id="pgen.1006616.ref020"></a>Jablonski NG. the Evolution of Human Skin and Skin Color. Annu Rev Anthropol. 2004;33: 585–623. <ul><li><a href="#" data-author="Jablonski" data-cit="JablonskiNG.%20the%20Evolution%20of%20Human%20Skin%20and%20Skin%20Color.%20Annu%20Rev%20Anthropol.%202004%3B33%3A%20585%E2%80%93623." data-title="the%20Evolution%20of%20Human%20Skin%20and%20Skin%20Color" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=the+Evolution+of+Human+Skin+and+Skin+Color+Jablonski+2004" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref21"><span>21.
            </span><a name="pgen.1006616.ref021" id="pgen.1006616.ref021"></a>Lamason RL, Mohideen M-APK, Mest JR, Wong AC, Norton HL, Aros MC, et al. SLC24A5, a putative cation exchanger, affects pigmentation in zebrafish and humans. Science. 2005;310: 1782–6.  pmid:16357253 <ul data-doi="10.1126/science.1116238"><li><a href="https://doi.org/10.1126/science.1116238" data-author="doi-provided" data-cit="doi-provided" data-title="doi-provided" target="_new" title="Go to article">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/16357253" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=SLC24A5%2C+a+putative+cation+exchanger%2C+affects+pigmentation+in+zebrafish+and+humans+Lamason+2005" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref22"><span>22.
            </span><a name="pgen.1006616.ref022" id="pgen.1006616.ref022"></a>Norton HL, Kittles RA, Parra E, McKeigue P, Mao X, Cheng K, et al. Genetic evidence for the convergent evolution of light skin in Europeans and East Asians. Mol Biol Evol. 2007;24: 710–22.  pmid:17182896 <ul data-doi="10.1093/molbev/msl203"><li><a href="https://doi.org/10.1093/molbev/msl203" data-author="doi-provided" data-cit="doi-provided" data-title="doi-provided" target="_new" title="Go to article">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/17182896" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Genetic+evidence+for+the+convergent+evolution+of+light+skin+in+Europeans+and+East+Asians+Norton+2007" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref23"><span>23.
            </span><a name="pgen.1006616.ref023" id="pgen.1006616.ref023"></a>Jablonski NG, Chaplin G. The evolution of human skin coloration. J Hum Evol. 2000;39: 57–106.  pmid:10896812 <ul data-doi="10.1006/jhev.2000.0403"><li><a href="https://doi.org/10.1006/jhev.2000.0403" data-author="doi-provided" data-cit="doi-provided" data-title="doi-provided" target="_new" title="Go to article">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/10896812" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=The+evolution+of+human+skin+coloration+Jablonski+2000" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref24"><span>24.
            </span><a name="pgen.1006616.ref024" id="pgen.1006616.ref024"></a>Gower JC. Generalized procrustes analysis. Psychometrika. 1975;40: 33–51. <ul><li><a href="#" data-author="Gower" data-cit="GowerJC.%20Generalized%20procrustes%20analysis.%20Psychometrika.%201975%3B40%3A%2033%E2%80%9351." data-title="Generalized%20procrustes%20analysis" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Generalized+procrustes+analysis+Gower+1975" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref25"><span>25.
            </span><a name="pgen.1006616.ref025" id="pgen.1006616.ref025"></a>Claes P, Liberton DK, Daniels K, Rosana KM, Quillen EE, Pearson LN, et al. Modeling 3D Facial Shape from DNA. Luquetti D, editor. PLoS Genet. Public Library of Science; 2014;10: e1004224.  pmid:24651127 <ul data-doi="10.1371/journal.pgen.1004224"><li><a href="https://doi.org/10.1371/journal.pgen.1004224" data-author="doi-provided" data-cit="doi-provided" data-title="doi-provided" target="_new" title="Go to article">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/24651127" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Modeling+3D+Facial+Shape+from+DNA.+Luquetti+D%2C+editor+Claes+2014" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref26"><span>26.
            </span><a name="pgen.1006616.ref026" id="pgen.1006616.ref026"></a>Ferrario VF, Mian F, Peretta R, Rosati R, Sforza C. Three-dimensional computerized anthropometry of the nose: Landmark representation compared to surface analysis. Cleft Palate-Craniofacial J. 2007;44: 278–285. <ul><li><a href="#" data-author="Ferrario" data-cit="FerrarioVF%2C%20MianF%2C%20PerettaR%2C%20RosatiR%2C%20SforzaC.%20Three-dimensional%20computerized%20anthropometry%20of%20the%20nose%3A%20Landmark%20representation%20compared%20to%20surface%20analysis.%20Cleft%20Palate-Craniofacial%20J.%202007%3B44%3A%20278%E2%80%93285." data-title="Three-dimensional%20computerized%20anthropometry%20of%20the%20nose%3A%20Landmark%20representation%20compared%20to%20surface%20analysis" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Three-dimensional+computerized+anthropometry+of+the+nose%3A+Landmark+representation+compared+to+surface+analysis+Ferrario+2007" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref27"><span>27.
            </span><a name="pgen.1006616.ref027" id="pgen.1006616.ref027"></a>Nucara A, Pietrafesa M, Rizzo G, Scaccianoce G. Handbook of Anthropometry. Handb Anthr. 2012; 91–114. <ul><li><a href="#" data-author="Nucara" data-cit="NucaraA%2C%20PietrafesaM%2C%20RizzoG%2C%20ScaccianoceG.%20Handbook%20of%20Anthropometry.%20Handb%20Anthr.%202012%3B%2091%E2%80%93114." data-title="Handbook%20of%20Anthropometry" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Handbook+of+Anthropometry+Nucara+2012" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref28"><span>28.
            </span><a name="pgen.1006616.ref028" id="pgen.1006616.ref028"></a>Leinonen T, McCairns RJS, O’Hara RB, Merilä J. Q(ST)-F(ST) comparisons: evolutionary and ecological insights from genomic heterogeneity. Nat Rev Genet. Nature Publishing Group, a division of Macmillan Publishers Limited. All Rights Reserved.; 2013;14: 179–90.  pmid:23381120 <ul data-doi="10.1038/nrg3395"><li><a href="https://doi.org/10.1038/nrg3395" data-author="doi-provided" data-cit="doi-provided" data-title="doi-provided" target="_new" title="Go to article">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/23381120" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Q%28ST%29-F%28ST%29+comparisons%3A+evolutionary+and+ecological+insights+from+genomic+heterogeneity+Leinonen+2013" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref29"><span>29.
            </span><a name="pgen.1006616.ref029" id="pgen.1006616.ref029"></a>Wright S. The Genetical Structure Of Populations. Ann Eugen. 1949;15: 323–354. <ul><li><a href="#" data-author="Wright" data-cit="WrightS.%20The%20Genetical%20Structure%20Of%20Populations.%20Ann%20Eugen.%201949%3B15%3A%20323%E2%80%93354." data-title="The%20Genetical%20Structure%20Of%20Populations" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=The+Genetical+Structure+Of+Populations+Wright+1949" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref30"><span>30.
            </span><a name="pgen.1006616.ref030" id="pgen.1006616.ref030"></a>Lande R. Neutral Theory of Quantitative Genetic Variance in an Island Model with Local Extinction and Colonization. Evolution (N Y). 1992;46: 381–389. <ul><li><a href="#" data-author="Lande" data-cit="LandeR.%20Neutral%20Theory%20of%20Quantitative%20Genetic%20Variance%20in%20an%20Island%20Model%20with%20Local%20Extinction%20and%20Colonization.%20Evolution%20%28N%20Y%29.%201992%3B46%3A%20381%E2%80%93389." data-title="Neutral%20Theory%20of%20Quantitative%20Genetic%20Variance%20in%20an%20Island%20Model%20with%20Local%20Extinction%20and%20Colonization" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Neutral+Theory+of+Quantitative+Genetic+Variance+in+an+Island+Model+with+Local+Extinction+and+Colonization+Lande+1992" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref31"><span>31.
            </span><a name="pgen.1006616.ref031" id="pgen.1006616.ref031"></a>Whitlock MC. Evolutionary inference from QST. Molecular Ecology. 2008. pp. 1885–1896.  pmid:18363667 <ul data-doi="10.1111/j.1365-294X.2008.03712.x"><li><a href="https://doi.org/10.1111/j.1365-294X.2008.03712.x" data-author="doi-provided" data-cit="doi-provided" data-title="doi-provided" target="_new" title="Go to article">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/18363667" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Evolutionary+inference+from+QST+Whitlock+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref32"><span>32.
            </span><a name="pgen.1006616.ref032" id="pgen.1006616.ref032"></a>Brommer JE. Whither Pst? The approximation of Qst by Pst in evolutionary and conservation biology. J Evol Biol. 2011;24: 1160–8.  pmid:21457173 <ul data-doi="10.1111/j.1420-9101.2011.02268.x"><li><a href="https://doi.org/10.1111/j.1420-9101.2011.02268.x" data-author="doi-provided" data-cit="doi-provided" data-title="doi-provided" target="_new" title="Go to article">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/21457173" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Whither+Pst%3F+The+approximation+of+Qst+by+Pst+in+evolutionary+and+conservation+biology+Brommer+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref33"><span>33.
            </span><a name="pgen.1006616.ref033" id="pgen.1006616.ref033"></a>Leinonen T, Cano JM, Mäkinen H, Merilä J. Contrasting patterns of body shape and neutral genetic divergence in marine and lake populations of threespine sticklebacks. J Evol Biol. 2006;19: 1803–12.  pmid:17040377 <ul data-doi="10.1111/j.1420-9101.2006.01182.x"><li><a href="https://doi.org/10.1111/j.1420-9101.2006.01182.x" data-author="doi-provided" data-cit="doi-provided" data-title="doi-provided" target="_new" title="Go to article">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/17040377" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Contrasting+patterns+of+body+shape+and+neutral+genetic+divergence+in+marine+and+lake+populations+of+threespine+sticklebacks+Leinonen+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref34"><span>34.
            </span><a name="pgen.1006616.ref034" id="pgen.1006616.ref034"></a>Perry GH, Foll M, Grenier J-C, Patin E, Nédélec Y, Pacis A, et al. Adaptive, convergent origins of the pygmy phenotype in African rainforest hunter-gatherers. Proc Natl Acad Sci U S A. 2014;111: E3596–603.  pmid:25136101 <ul data-doi="10.1073/pnas.1402875111"><li><a href="https://doi.org/10.1073/pnas.1402875111" data-author="doi-provided" data-cit="doi-provided" data-title="doi-provided" target="_new" title="Go to article">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/25136101" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Adaptive%2C+convergent+origins+of+the+pygmy+phenotype+in+African+rainforest+hunter-gatherers+Perry+2014" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref35"><span>35.
            </span><a name="pgen.1006616.ref035" id="pgen.1006616.ref035"></a>Relethford JH. Craniometric variation among modern human populations. Am J Phys Anthropol. 1994;95: 53–62.  pmid:7527996 <ul data-doi="10.1002/ajpa.1330950105"><li><a href="https://doi.org/10.1002/ajpa.1330950105" data-author="doi-provided" data-cit="doi-provided" data-title="doi-provided" target="_new" title="Go to article">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/7527996" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Craniometric+variation+among+modern+human+populations+Relethford+1994" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref36"><span>36.
            </span><a name="pgen.1006616.ref036" id="pgen.1006616.ref036"></a>Lynch M, Walsh B. Genetics and analysis of quantitative traits. Sunderland, MA: Sinauer Associates, Inc; 1998. <ul></ul></li><li id="ref37"><span>37.
            </span><a name="pgen.1006616.ref037" id="pgen.1006616.ref037"></a>Yang J, Lee SH, Goddard ME, Visscher PM. GCTA: a tool for genome-wide complex trait analysis. Am J Hum Genet. 2011;88: 76–82.  pmid:21167468 <ul data-doi="10.1016/j.ajhg.2010.11.011"><li><a href="https://doi.org/10.1016/j.ajhg.2010.11.011" data-author="doi-provided" data-cit="doi-provided" data-title="doi-provided" target="_new" title="Go to article">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/21167468" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=GCTA%3A+a+tool+for+genome-wide+complex+trait+analysis+Yang+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref38"><span>38.
            </span><a name="pgen.1006616.ref038" id="pgen.1006616.ref038"></a>Sabeti PC, Varilly P, Fry B, Lohmueller J, Hostetter E, Cotsapas C, et al. Genome-wide detection and characterization of positive selection in human populations. Nature. 2007;449: 913–918.  pmid:17943131 <ul data-doi="10.1038/nature06250"><li><a href="https://doi.org/10.1038/nature06250" data-author="doi-provided" data-cit="doi-provided" data-title="doi-provided" target="_new" title="Go to article">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/17943131" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Genome-wide+detection+and+characterization+of+positive+selection+in+human+populations+Sabeti+2007" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref39"><span>39.
            </span><a name="pgen.1006616.ref039" id="pgen.1006616.ref039"></a>Darvasi A, Shifman S. The beauty of admixture. Nat Genet. 2005;37: 118–119.  pmid:15678141 <ul data-doi="10.1038/ng0205-118"><li><a href="https://doi.org/10.1038/ng0205-118" data-author="doi-provided" data-cit="doi-provided" data-title="doi-provided" target="_new" title="Go to article">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/15678141" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=The+beauty+of+admixture+Darvasi+2005" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref40"><span>40.
            </span><a name="pgen.1006616.ref040" id="pgen.1006616.ref040"></a>Zaitlen N, Pasaniuc B, Sankararaman S, Bhatia G, Zhang J, Gusev A, et al. Leveraging population admixture to characterize the heritability of complex traits. Nat Genet. Nature Publishing Group, a division of Macmillan Publishers Limited. All Rights Reserved.; 2014;46: 1356–1362.  pmid:25383972 <ul data-doi="10.1038/ng.3139"><li><a href="https://doi.org/10.1038/ng.3139" data-author="doi-provided" data-cit="doi-provided" data-title="doi-provided" target="_new" title="Go to article">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/25383972" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Leveraging+population+admixture+to+characterize+the+heritability+of+complex+traits+Zaitlen+2014" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref41"><span>41.
            </span><a name="pgen.1006616.ref041" id="pgen.1006616.ref041"></a>Yokley TR. Ecogeographic variation in human nasal passages. Am J Phys Anthropol. 2009;138: 11–22.  pmid:18623075 <ul data-doi="10.1002/ajpa.20893"><li><a href="https://doi.org/10.1002/ajpa.20893" data-author="doi-provided" data-cit="doi-provided" data-title="doi-provided" target="_new" title="Go to article">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/18623075" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Ecogeographic+variation+in+human+nasal+passages+Yokley+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref42"><span>42.
            </span><a name="pgen.1006616.ref042" id="pgen.1006616.ref042"></a>Noback ML, Harvati K, Spoor F. Climate-related variation of the human nasal cavity. Am J Phys Anthropol. 2011;145: 599–614.  pmid:21660932 <ul data-doi="10.1002/ajpa.21523"><li><a href="https://doi.org/10.1002/ajpa.21523" data-author="doi-provided" data-cit="doi-provided" data-title="doi-provided" target="_new" title="Go to article">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/21660932" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Climate-related+variation+of+the+human+nasal+cavity+Noback+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref43"><span>43.
            </span><a name="pgen.1006616.ref043" id="pgen.1006616.ref043"></a>Maddux SD, Yokley TR, Svoma BM, Franciscus RG. Absolute humidity and the human nose: A reanalysis of climate zones and their influence on nasal form and function. Am J Phys Anthropol. 2016;161: 309–320.  pmid:27374937 <ul data-doi="10.1002/ajpa.23032"><li><a href="https://doi.org/10.1002/ajpa.23032" data-author="doi-provided" data-cit="doi-provided" data-title="doi-provided" target="_new" title="Go to article">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/27374937" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Absolute+humidity+and+the+human+nose%3A+A+reanalysis+of+climate+zones+and+their+influence+on+nasal+form+and+function+Maddux+2016" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref44"><span>44.
            </span><a name="pgen.1006616.ref044" id="pgen.1006616.ref044"></a>Keck T, Lindemann J. Numerical simulation and nasal air-conditioning. GMS Curr Top Otorhinolaryngol Head Neck Surg. 2010;9: Doc08.  pmid:22073112 <ul data-doi="10.3205/cto000072"><li><a href="https://doi.org/10.3205/cto000072" data-author="doi-provided" data-cit="doi-provided" data-title="doi-provided" target="_new" title="Go to article">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/22073112" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Numerical+simulation+and+nasal+air-conditioning+Keck+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref45"><span>45.
            </span><a name="pgen.1006616.ref045" id="pgen.1006616.ref045"></a>Inthavong K, Tian ZF, Tu JY. CFD Simulations on the Heating Capability in a Human Nasal Cavity. Rhinology. 2007; 842–847. <ul><li><a href="#" data-author="Inthavong" data-cit="InthavongK%2C%20TianZF%2C%20TuJY.%20CFD%20Simulations%20on%20the%20Heating%20Capability%20in%20a%20Human%20Nasal%20Cavity.%20Rhinology.%202007%3B%20842%E2%80%93847." data-title="CFD%20Simulations%20on%20the%20Heating%20Capability%20in%20a%20Human%20Nasal%20Cavity" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=CFD+Simulations+on+the+Heating+Capability+in+a+Human+Nasal+Cavity+Inthavong+2007" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref46"><span>46.
            </span><a name="pgen.1006616.ref046" id="pgen.1006616.ref046"></a>Irmak MK, Korkmaz A, Erogul O. Selective brain cooling seems to be a mechanism leading to human craniofacial diversity observed in different geographical regions. Med Hypotheses. 2004;63: 974–979.  pmid:15504564 <ul data-doi="10.1016/j.mehy.2004.05.003"><li><a href="https://doi.org/10.1016/j.mehy.2004.05.003" data-author="doi-provided" data-cit="doi-provided" data-title="doi-provided" target="_new" title="Go to article">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/15504564" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Selective+brain+cooling+seems+to+be+a+mechanism+leading+to+human+craniofacial+diversity+observed+in+different+geographical+regions+Irmak+2004" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref47"><span>47.
            </span><a name="pgen.1006616.ref047" id="pgen.1006616.ref047"></a>Maloney SK, Mitchell D, Blache D. The contribution of carotid rete variability to brain temperature variability in sheep in a thermoneutral environment. Am J Physiol Regul Integr Comp Physiol. 2007;292: R1298–305.  pmid:17082355 <ul data-doi="10.1152/ajpregu.00275.2006"><li><a href="https://doi.org/10.1152/ajpregu.00275.2006" data-author="doi-provided" data-cit="doi-provided" data-title="doi-provided" target="_new" title="Go to article">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/17082355" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=The+contribution+of+carotid+rete+variability+to+brain+temperature+variability+in+sheep+in+a+thermoneutral+environment+Maloney+2007" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref48"><span>48.
            </span><a name="pgen.1006616.ref048" id="pgen.1006616.ref048"></a>Mitchell D, Maloney SK, Jessen C, Laburn HP, Kamerman PR, Mitchell G, et al. Adaptive heterothermy and selective brain cooling in arid-zone mammals. Comp Biochem Physiol Part B Biochem Mol Biol. 2002;131: 571–585. <ul><li><a href="#" data-author="Mitchell" data-cit="MitchellD%2C%20MaloneySK%2C%20JessenC%2C%20LaburnHP%2C%20KamermanPR%2C%20MitchellG%2C%20et%20al.%20Adaptive%20heterothermy%20and%20selective%20brain%20cooling%20in%20arid-zone%20mammals.%20Comp%20Biochem%20Physiol%20Part%20B%20Biochem%20Mol%20Biol.%202002%3B131%3A%20571%E2%80%93585." data-title="Adaptive%20heterothermy%20and%20selective%20brain%20cooling%20in%20arid-zone%20mammals" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Adaptive+heterothermy+and+selective+brain+cooling+in+arid-zone+mammals+Mitchell+2002" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref49"><span>49.
            </span><a name="pgen.1006616.ref049" id="pgen.1006616.ref049"></a>Hetem RS, Strauss WM, Fick LG, Maloney SK, Meyer LCR, Fuller A, et al. Selective brain cooling in Arabian oryx (Oryx leucoryx): a physiological mechanism for coping with aridity? J Exp Biol. 2012;215: 3917–24.  pmid:22899527 <ul data-doi="10.1242/jeb.074666"><li><a href="https://doi.org/10.1242/jeb.074666" data-author="doi-provided" data-cit="doi-provided" data-title="doi-provided" target="_new" title="Go to article">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/22899527" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Selective+brain+cooling+in+Arabian+oryx+%28Oryx+leucoryx%29%3A+a+physiological+mechanism+for+coping+with+aridity%3F+Hetem+2012" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref50"><span>50.
            </span><a name="pgen.1006616.ref050" id="pgen.1006616.ref050"></a>Blix AS, Walløe L, Folkow LP. Regulation of brain temperature in winter-acclimatized reindeer under heat stress. J Exp Biol. 2011;214: 3850–6.  pmid:22031750 <ul data-doi="10.1242/jeb.057455"><li><a href="https://doi.org/10.1242/jeb.057455" data-author="doi-provided" data-cit="doi-provided" data-title="doi-provided" target="_new" title="Go to article">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/22031750" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Regulation+of+brain+temperature+in+winter-acclimatized+reindeer+under+heat+stress+Blix+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref51"><span>51.
            </span><a name="pgen.1006616.ref051" id="pgen.1006616.ref051"></a>Crandall CG, Brothers RM, Zhang R, Brengelmann GL, Covaciu L, Jay O, et al. Comments on point:counterpoint: humans do/do not demonstrate selective brain cooling during hyperthermia. J Appl Physiol. American Physiological Society; 2011;110: 575–80.  pmid:21304015 <ul data-doi="10.1152/japplphysiol.01375.2010"><li><a href="https://doi.org/10.1152/japplphysiol.01375.2010" data-author="doi-provided" data-cit="doi-provided" data-title="doi-provided" target="_new" title="Go to article">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/21304015" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Comments+on+point%3Acounterpoint%3A+humans+do%2Fdo+not+demonstrate+selective+brain+cooling+during+hyperthermia+Crandall+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref52"><span>52.
            </span><a name="pgen.1006616.ref052" id="pgen.1006616.ref052"></a>Cabanac M. Selective brain cooling in humans: “fancy” or fact? FASEB J. 1993;7: 1143-1146-1147. <ul><li><a href="#" data-author="Cabanac" data-cit="CabanacM.%20Selective%20brain%20cooling%20in%20humans%3A%20%E2%80%9Cfancy%E2%80%9D%20or%20fact%3F%20FASEB%20J.%201993%3B7%3A%201143-1146-1147." data-title="Selective%20brain%20cooling%20in%20humans%3A%20%E2%80%9Cfancy%E2%80%9D%20or%20fact%3F" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Selective+brain+cooling+in+humans%3A+%E2%80%9Cfancy%E2%80%9D+or+fact%3F+Cabanac+1993" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref53"><span>53.
            </span><a name="pgen.1006616.ref053" id="pgen.1006616.ref053"></a>Puts DA. Beauty and the beast: mechanisms of sexual selection in humans. Evol Hum Behav. Elsevier; 2010;31: 157–175. <ul><li><a href="#" data-author="Puts" data-cit="PutsDA.%20Beauty%20and%20the%20beast%3A%20mechanisms%20of%20sexual%20selection%20in%20humans.%20Evol%20Hum%20Behav.%20Elsevier%3B%202010%3B31%3A%20157%E2%80%93175." data-title="Beauty%20and%20the%20beast%3A%20mechanisms%20of%20sexual%20selection%20in%20humans" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Beauty+and+the+beast%3A+mechanisms+of+sexual+selection+in+humans+Puts+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref54"><span>54.
            </span><a name="pgen.1006616.ref054" id="pgen.1006616.ref054"></a>van Doorn GS, Edelaar P, Weissing FJ. On the origin of species by natural and sexual selection. Science. 2009;326: 1704–1707.  pmid:19965377 <ul data-doi="10.1126/science.1181661"><li><a href="https://doi.org/10.1126/science.1181661" data-author="doi-provided" data-cit="doi-provided" data-title="doi-provided" target="_new" title="Go to article">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/19965377" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=On+the+origin+of+species+by+natural+and+sexual+selection+van+Doorn+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref55"><span>55.
            </span><a name="pgen.1006616.ref055" id="pgen.1006616.ref055"></a>Bigham AW, Mao X, Mei R, Brutsaert T, Wilson MJ, Julian CG, et al. Identifying positive selection candidate loci for high-altitude adaptation in Andean populations. Hum Genomics. 2009;4: 79–90.  pmid:20038496 <ul data-doi="10.1186/1479-7364-4-2-79"><li><a href="https://doi.org/10.1186/1479-7364-4-2-79" data-author="doi-provided" data-cit="doi-provided" data-title="doi-provided" target="_new" title="Go to article">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/20038496" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Identifying+positive+selection+candidate+loci+for+high-altitude+adaptation+in+Andean+populations+Bigham+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref56"><span>56.
            </span><a name="pgen.1006616.ref056" id="pgen.1006616.ref056"></a>Wu T, Li S, Ward MP. Tibetans at extreme altitude. Wilderness Environ Med. 2005;16: 47–54. pmid:15813148 <ul><li><a href="#" data-author="Wu" data-cit="WuT%2C%20LiS%2C%20WardMP.%20Tibetans%20at%20extreme%20altitude.%20Wilderness%20Environ%20Med.%202005%3B16%3A%2047%E2%80%9354.%2015813148" data-title="Tibetans%20at%20extreme%20altitude" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/15813148" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Tibetans+at+extreme+altitude+Wu+2005" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref57"><span>57.
            </span><a name="pgen.1006616.ref057" id="pgen.1006616.ref057"></a>Paternoster L, Zhurov AI, Toma AM, Kemp JP, St Pourcain B, Timpson NJ, et al. Genome-wide association study of three-dimensional facial morphology identifies a variant in PAX3 associated with nasion position. Am J Hum Genet. Elsevier; 2012;90: 478–85.  pmid:22341974 <ul data-doi="10.1016/j.ajhg.2011.12.021"><li><a href="https://doi.org/10.1016/j.ajhg.2011.12.021" data-author="doi-provided" data-cit="doi-provided" data-title="doi-provided" target="_new" title="Go to article">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/22341974" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Genome-wide+association+study+of+three-dimensional+facial+morphology+identifies+a+variant+in+PAX3+associated+with+nasion+position+Paternoster+2012" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref58"><span>58.
            </span><a name="pgen.1006616.ref058" id="pgen.1006616.ref058"></a>Liu F, van der Lijn F, Schurmann C, Zhu G, Chakravarty MM, Hysi PG, et al. A genome-wide association study identifies five loci influencing facial morphology in Europeans. Gibson G, editor. PLoS Genet. Public Library of Science; 2012;8: e1002932.  pmid:23028347 <ul data-doi="10.1371/journal.pgen.1002932"><li><a href="https://doi.org/10.1371/journal.pgen.1002932" data-author="doi-provided" data-cit="doi-provided" data-title="doi-provided" target="_new" title="Go to article">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/23028347" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=A+genome-wide+association+study+identifies+five+loci+influencing+facial+morphology+in+Europeans+Liu+2012" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref59"><span>59.
            </span><a name="pgen.1006616.ref059" id="pgen.1006616.ref059"></a>Shaffer JR, Orlova E, Lee MK, Leslie EJ, Raffensperger ZD, Heike CL, et al. Genome-Wide Association Study Reveals Multiple Loci Influencing Normal Human Facial Morphology. Barsh GS, editor. PLOS Genet. Public Library of Science; 2016;12: e1006149.  pmid:27560520 <ul data-doi="10.1371/journal.pgen.1006149"><li><a href="https://doi.org/10.1371/journal.pgen.1006149" data-author="doi-provided" data-cit="doi-provided" data-title="doi-provided" target="_new" title="Go to article">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/27560520" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Genome-Wide+Association+Study+Reveals+Multiple+Loci+Influencing+Normal+Human+Facial+Morphology+Shaffer+2016" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref60"><span>60.
            </span><a name="pgen.1006616.ref060" id="pgen.1006616.ref060"></a>Adhikari K, Fuentes-Guajardo M, Quinto-Sánchez M, Mendoza-Revilla J, Camilo Chacón-Duque J, Acuña-Alonzo V, et al. A genome-wide association scan implicates DCHS2, RUNX2, GLI3, PAX1 and EDAR in human facial variation. Nat Commun. Nature Research; 2016;7: 11616. <ul><li><a href="#" data-author="Adhikari" data-cit="AdhikariK%2C%20Fuentes-GuajardoM%2C%20Quinto-S%C3%A1nchezM%2C%20Mendoza-RevillaJ%2C%20Camilo%20Chac%C3%B3n-DuqueJ%2C%20Acu%C3%B1a-AlonzoV%2C%20et%20al.%20A%20genome-wide%20association%20scan%20implicates%20DCHS2%2C%20RUNX2%2C%20GLI3%2C%20PAX1%20and%20EDAR%20in%20human%20facial%20variation.%20Nat%20Commun.%20Nature%20Research%3B%202016%3B7%3A%2011616." data-title="A%20genome-wide%20association%20scan%20implicates%20DCHS2%2C%20RUNX2%2C%20GLI3%2C%20PAX1%20and%20EDAR%20in%20human%20facial%20variation" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=A+genome-wide+association+scan+implicates+DCHS2%2C+RUNX2%2C+GLI3%2C+PAX1+and+EDAR+in+human+facial+variation+Adhikari+2016" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref61"><span>61.
            </span><a name="pgen.1006616.ref061" id="pgen.1006616.ref061"></a>Beleza S, Campos J, Lopes J, Araújo II, Hoppfer Almada A, Correia e Silva A, et al. The Admixture Structure and Genetic Variation of the Archipelago of Cape Verde and Its Implications for Admixture Mapping Studies. PLoS One. 2012;7. <ul><li><a href="#" data-author="Beleza" data-cit="BelezaS%2C%20CamposJ%2C%20LopesJ%2C%20Ara%C3%BAjoII%2C%20Hoppfer%20AlmadaA%2C%20Correia%20e%20SilvaA%2C%20et%20al.%20The%20Admixture%20Structure%20and%20Genetic%20Variation%20of%20the%20Archipelago%20of%20Cape%20Verde%20and%20Its%20Implications%20for%20Admixture%20Mapping%20Studies.%20PLoS%20One.%202012%3B7." data-title="The%20Admixture%20Structure%20and%20Genetic%20Variation%20of%20the%20Archipelago%20of%20Cape%20Verde%20and%20Its%20Implications%20for%20Admixture%20Mapping%20Studies" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=The+Admixture+Structure+and+Genetic+Variation+of+the+Archipelago+of+Cape+Verde+and+Its+Implications+for+Admixture+Mapping+Studies+Beleza+2012" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref62"><span>62.
            </span><a name="pgen.1006616.ref062" id="pgen.1006616.ref062"></a>Illumina Inc. “TOP/BOT” strand and “A/B” allele—A guide to Illumina’s method for determining Strand and Allele for the GoldenGate and Infinium Assays. (Technical Note) [Internet]. 2006. Available: <a href="http://www.illumina.com/documents/products/technotes/technote_topbot.pdf">http://www.illumina.com/documents/products/technotes/technote_topbot.pdf</a> <ul></ul></li><li id="ref63"><span>63.
            </span><a name="pgen.1006616.ref063" id="pgen.1006616.ref063"></a>Purcell S, Neale B, Todd-Brown K, Thomas L, Ferreira MAR, Bender D, et al. PLINK: a tool set for whole-genome association and population-based linkage analyses. Am J Hum Genet. 2007;81: 559–75.  pmid:17701901 <ul data-doi="10.1086/519795"><li><a href="https://doi.org/10.1086/519795" data-author="doi-provided" data-cit="doi-provided" data-title="doi-provided" target="_new" title="Go to article">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/17701901" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=PLINK%3A+a+tool+set+for+whole-genome+association+and+population-based+linkage+analyses+Purcell+2007" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref64"><span>64.
            </span><a name="pgen.1006616.ref064" id="pgen.1006616.ref064"></a>Chang CC, Chow CC, Tellier LC, Vattikuti S, Purcell SM, Lee JJ. Second-generation PLINK: rising to the challenge of larger and richer datasets. Gigascience. BioMed Central Ltd; 2015;4: 7.  pmid:25722852 <ul data-doi="10.1186/s13742-015-0047-8"><li><a href="https://doi.org/10.1186/s13742-015-0047-8" data-author="doi-provided" data-cit="doi-provided" data-title="doi-provided" target="_new" title="Go to article">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/25722852" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Second-generation+PLINK%3A+rising+to+the+challenge+of+larger+and+richer+datasets+Chang+2015" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref65"><span>65.
            </span><a name="pgen.1006616.ref065" id="pgen.1006616.ref065"></a>Candille SI, Absher DM, Beleza S, Bauchet M, McEvoy B, Garrison NA, et al. Genome-wide association studies of quantitatively measured skin, hair, and eye pigmentation in four European populations. PLoS One. Public Library of Science; 2012;7: e48294.  pmid:23118974 <ul data-doi="10.1371/journal.pone.0048294"><li><a href="https://doi.org/10.1371/journal.pone.0048294" data-author="doi-provided" data-cit="doi-provided" data-title="doi-provided" target="_new" title="Go to article">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/23118974" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Genome-wide+association+studies+of+quantitatively+measured+skin%2C+hair%2C+and+eye+pigmentation+in+four+European+populations+Candille+2012" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref66"><span>66.
            </span><a name="pgen.1006616.ref066" id="pgen.1006616.ref066"></a>Storz JF. Contrasting patterns of divergence in quantitative traits and neutral DNA markers: analysis of clinal variation. Mol Ecol. 2002;11: 2537–2551. pmid:12453238 <ul><li><a href="#" data-author="Storz" data-cit="StorzJF.%20Contrasting%20patterns%20of%20divergence%20in%20quantitative%20traits%20and%20neutral%20DNA%20markers%3A%20analysis%20of%20clinal%20variation.%20Mol%20Ecol.%202002%3B11%3A%202537%E2%80%932551.%2012453238" data-title="Contrasting%20patterns%20of%20divergence%20in%20quantitative%20traits%20and%20neutral%20DNA%20markers%3A%20analysis%20of%20clinal%20variation" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/12453238" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Contrasting+patterns+of+divergence+in+quantitative+traits+and+neutral+DNA+markers%3A+analysis+of+clinal+variation+Storz+2002" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref67"><span>67.
            </span><a name="pgen.1006616.ref067" id="pgen.1006616.ref067"></a>He Y, Li R, Wang J, Blanchet S, Lek S. Morphological Variation Among Wild Populations of Chinese Rare Minnow (Gobiocypris rarus): Deciphering the Role of Evolutionary Processes. Zoolog Sci. 2013;30: 475–83.  pmid:23721472 <ul data-doi="10.2108/zsj.30.475"><li><a href="https://doi.org/10.2108/zsj.30.475" data-author="doi-provided" data-cit="doi-provided" data-title="doi-provided" target="_new" title="Go to article">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/23721472" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Morphological+Variation+Among+Wild+Populations+of+Chinese+Rare+Minnow+%28Gobiocypris+rarus%29%3A+Deciphering+the+Role+of+Evolutionary+Processes+He+2013" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref68"><span>68.
            </span><a name="pgen.1006616.ref068" id="pgen.1006616.ref068"></a>Weir BS, Cockerham CC. Estimating F-statistics for the analysis of population structure. Evolution (N Y). 1984;38: 1358–1370. <ul><li><a href="#" data-author="Weir" data-cit="WeirBS%2C%20CockerhamCC.%20Estimating%20F-statistics%20for%20the%20analysis%20of%20population%20structure.%20Evolution%20%28N%20Y%29.%201984%3B38%3A%201358%E2%80%931370." data-title="Estimating%20F-statistics%20for%20the%20analysis%20of%20population%20structure" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Estimating+F-statistics+for+the+analysis+of+population+structure+Weir+1984" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref69"><span>69.
            </span><a name="pgen.1006616.ref069" id="pgen.1006616.ref069"></a>Browning SR, Browning BL. Population structure can inflate SNP-based heritability estimates. Am J Hum Genet. 2011;89: 191–193.  pmid:21763486 <ul data-doi="10.1016/j.ajhg.2011.05.025"><li><a href="https://doi.org/10.1016/j.ajhg.2011.05.025" data-author="doi-provided" data-cit="doi-provided" data-title="doi-provided" target="_new" title="Go to article">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/21763486" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Population+structure+can+inflate+SNP-based+heritability+estimates+Browning+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref70"><span>70.
            </span><a name="pgen.1006616.ref070" id="pgen.1006616.ref070"></a>Goddard ME, Lee SH, Yang J, Wray NR, Visscher PM. Response to browning and browning. Am J Hum Genet. 2011;89: 193–195. <ul><li><a href="#" data-author="Goddard" data-cit="GoddardME%2C%20LeeSH%2C%20YangJ%2C%20WrayNR%2C%20VisscherPM.%20Response%20to%20browning%20and%20browning.%20Am%20J%20Hum%20Genet.%202011%3B89%3A%20193%E2%80%93195." data-title="Response%20to%20browning%20and%20browning" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Response+to+browning+and+browning+Goddard+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref71"><span>71.
            </span><a name="pgen.1006616.ref071" id="pgen.1006616.ref071"></a>Baran Y, Pasaniuc B, Sankararaman S, Torgerson DG, Gignoux C, Eng C, et al. Fast and accurate inference of local ancestry in Latino populations. Bioinformatics. 2012;28: 1359–1367.  pmid:22495753 <ul data-doi="10.1093/bioinformatics/bts144"><li><a href="https://doi.org/10.1093/bioinformatics/bts144" data-author="doi-provided" data-cit="doi-provided" data-title="doi-provided" target="_new" title="Go to article">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/22495753" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Fast+and+accurate+inference+of+local+ancestry+in+Latino+populations+Baran+2012" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref72"><span>72.
            </span><a name="pgen.1006616.ref072" id="pgen.1006616.ref072"></a>Halder I, Shriver MD. Measuring and using admixture to study the genetics of complex diseases. Hum Genomics. BioMed Central Ltd; 2003;1: 52.  pmid:15601533 <ul data-doi="10.1186/1479-7364-1-1-52"><li><a href="https://doi.org/10.1186/1479-7364-1-1-52" data-author="doi-provided" data-cit="doi-provided" data-title="doi-provided" target="_new" title="Go to article">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/15601533" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Measuring+and+using+admixture+to+study+the+genetics+of+complex+diseases+Halder+2003" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref73"><span>73.
            </span><a name="pgen.1006616.ref073" id="pgen.1006616.ref073"></a>Kahle D, Wickham H. ggmap: Spatial Visualization with. R J. 2013;5: 144–161. Available: <a href="http://journal.r-project.org/archive/2013-1/kahle-wickham.pdf">http://journal.r-project.org/archive/2013-1/kahle-wickham.pdf</a> <ul><li><a href="#" data-author="Kahle" data-cit="KahleD%2C%20WickhamH.%20ggmap%3A%20Spatial%20Visualization%20with.%20R%20J.%202013%3B5%3A%20144%E2%80%93161.%20Available%3A%20http%3A%2F%2Fjournal.r-project.org%2Farchive%2F2013-1%2Fkahle-wickham.pdf" data-title="ggmap%3A%20Spatial%20Visualization%20with" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=ggmap%3A+Spatial+Visualization+with+Kahle+2013" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref74"><span>74.
            </span><a name="pgen.1006616.ref074" id="pgen.1006616.ref074"></a>R Core Team. R: A language and environment for statistical computing. R Foundation for Statistical Computing, Vienna, Austria. 2013. <ul></ul></li><li id="ref75"><span>75.
            </span><a name="pgen.1006616.ref075" id="pgen.1006616.ref075"></a>Harris I, Jones PD, Osborn TJ, Lister DH. Updated high-resolution grids of monthly climatic observations—the CRU TS3.10 Dataset. Int J Climatol. John Wiley &amp; Sons, Ltd; 2014;34: 623–642. <ul><li><a href="#" data-author="Harris" data-cit="HarrisI%2C%20JonesPD%2C%20OsbornTJ%2C%20ListerDH.%20Updated%20high-resolution%20grids%20of%20monthly%20climatic%20observations%E2%80%94the%20CRU%20TS3.10%20Dataset.%20Int%20J%20Climatol.%20John%20Wiley%20%26%20Sons%2C%20Ltd%3B%202014%3B34%3A%20623%E2%80%93642." data-title="Updated%20high-resolution%20grids%20of%20monthly%20climatic%20observations%E2%80%94the%20CRU%20TS3.10%20Dataset" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Updated+high-resolution+grids+of+monthly+climatic+observations%E2%80%94the+CRU+TS3.10+Dataset+Harris+2014" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref76"><span>76.
            </span><a name="pgen.1006616.ref076" id="pgen.1006616.ref076"></a>TOMS Science Team (1996). TOMS Earth Probe UV Reflectivity Monthly L3 Global 1 deg x 1.25 deg Lat/Lon Grid V008, version 008, Greenbelt, MD, Goddard Earth Sciences Data and Information Services Center (GES DISC) [Internet]. Available: <a href="http://disc.sci.gsfc.nasa.gov/datacollection/TOMSEPL3mref_008.html">http://disc.sci.gsfc.nasa.gov/datacollection/TOMSEPL3mref_008.html</a> <ul></ul></li><li id="ref77"><span>77.
            </span><a name="pgen.1006616.ref077" id="pgen.1006616.ref077"></a>Michna P, Woods M. RNetCDF—A Package for Reading and Writing NetCDF Datasets. R J. 2013;5: 29–35. <ul><li><a href="#" data-author="Michna" data-cit="MichnaP%2C%20WoodsM.%20RNetCDF%E2%80%94A%20Package%20for%20Reading%20and%20Writing%20NetCDF%20Datasets.%20R%20J.%202013%3B5%3A%2029%E2%80%9335." data-title="RNetCDF%E2%80%94A%20Package%20for%20Reading%20and%20Writing%20NetCDF%20Datasets" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=RNetCDF%E2%80%94A+Package+for+Reading+and+Writing+NetCDF+Datasets+Michna+2013" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref78"><span>78.
            </span><a name="pgen.1006616.ref078" id="pgen.1006616.ref078"></a>Hijmans RJ, Etten J Van. raster: Geographic analysis and modeling with raster data. R package version 2.0–12. [Internet]. 2012. Available: <a href="http://cran.r-project.org/package=raster">http://cran.r-project.org/package=raster</a> <ul></ul></li><li id="ref79"><span>79.
            </span><a name="pgen.1006616.ref079" id="pgen.1006616.ref079"></a>Rousset F, Ferdy J-B. Testing environmental and genetic effects in the presence of spatial autocorrelation. Ecography (Cop). 2014;37: 781–790. <ul><li><a href="#" data-author="Rousset" data-cit="RoussetF%2C%20FerdyJ-B.%20Testing%20environmental%20and%20genetic%20effects%20in%20the%20presence%20of%20spatial%20autocorrelation.%20Ecography%20%28Cop%29.%202014%3B37%3A%20781%E2%80%93790." data-title="Testing%20environmental%20and%20genetic%20effects%20in%20the%20presence%20of%20spatial%20autocorrelation" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Testing+environmental+and+genetic+effects+in+the+presence+of+spatial+autocorrelation+Rousset+2014" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref80"><span>80.
            </span><a name="pgen.1006616.ref080" id="pgen.1006616.ref080"></a>Claes P, Reijniers J, Shriver MD, Snyders J, Suetens P, Nielandt J, et al. An investigation of matching symmetry in the human pinnae with possible implications for 3D ear recognition and sound localization. J Anat. 2015;226: 60–72.  pmid:25382291 <ul data-doi="10.1111/joa.12252"><li><a href="https://doi.org/10.1111/joa.12252" data-author="doi-provided" data-cit="doi-provided" data-title="doi-provided" target="_new" title="Go to article">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/25382291" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=An+investigation+of+matching+symmetry+in+the+human+pinnae+with+possible+implications+for+3D+ear+recognition+and+sound+localization+Claes+2015" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref81"><span>81.
            </span><a name="pgen.1006616.ref081" id="pgen.1006616.ref081"></a>Pinheiro J, Bates D, DebRoy S, Sarkar D, Team TRC. nlme: Linear and Nonlinear Mixed Effects Models. October. 2013. pp. 1–3. <ul><li><a href="#" data-author="Pinheiro" data-cit="PinheiroJ%2C%20BatesD%2C%20DebRoyS%2C%20SarkarD%2C%20TeamTRC.%20nlme%3A%20Linear%20and%20Nonlinear%20Mixed%20Effects%20Models.%20October.%202013.%20pp.%201%E2%80%933." data-title="nlme%3A%20Linear%20and%20Nonlinear%20Mixed%20Effects%20Models" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=nlme%3A+Linear+and+Nonlinear+Mixed+Effects+Models+Pinheiro+2013" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li></ol></div>



          

        </div>
      </div>
    </div>

  </section>
  
</div>




</div></div>
  </body>
</html>
