<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <title>James Routley | Feed</title>
    <link
      rel="stylesheet"
      type="text/css"
      href="../styles.css"
      media="screen"
    />
  </head>
  <body>
    <a href="/index.html">Back</a>
    <a href="https://www.nature.com/articles/s41587-022-01618-2">Original</a>
    <h1>Large language models generate functional protein sequences across families</h1>
    
    <div id="readability-page-1" class="page"><div id="MagazineFulltextArticleBodySuffix"><section aria-labelledby="Bib1" data-title="References"><div id="Bib1-section"><h2 id="Bib1">References</h2><div id="Bib1-content"><div data-container-section="references"><ol data-track-component="outbound reference"><li data-counter="1."><p id="ref-CR1">Koga, N. et al. Principles for designing ideal protein structures. <i>Nature</i> <b>491</b>, 222–227 (2012).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/nature11600" data-track-action="article reference" href="https://doi.org/10.1038%2Fnature11600" aria-label="Article reference 1" data-doi="10.1038/nature11600">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://jvns.ca/articles/cas-redirect/1:CAS:528:DC%2BC38Xhs1emu7bF" aria-label="CAS reference 1">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 1" href="http://scholar.google.com/scholar_lookup?&amp;title=Principles%20for%20designing%20ideal%20protein%20structures&amp;journal=Nature&amp;doi=10.1038%2Fnature11600&amp;volume=491&amp;pages=222-227&amp;publication_year=2012&amp;author=Koga%2CN">
                    Google Scholar</a> 
                </p></li><li data-counter="2."><p id="ref-CR2">Lin, Y.-R. et al. Control over overall shape and size in de novo designed proteins. <i>Proc. Natl Acad. Sci. USA</i> <b>112</b>, E5478–E5485 (2015).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1073/pnas.1509508112" data-track-action="article reference" href="https://doi.org/10.1073%2Fpnas.1509508112" aria-label="Article reference 2" data-doi="10.1073/pnas.1509508112">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://jvns.ca/articles/cas-redirect/1:CAS:528:DC%2BC2MXhsFCisbrP" aria-label="CAS reference 2">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 2" href="http://scholar.google.com/scholar_lookup?&amp;title=Control%20over%20overall%20shape%20and%20size%20in%20de%20novo%20designed%20proteins&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;doi=10.1073%2Fpnas.1509508112&amp;volume=112&amp;pages=E5478-E5485&amp;publication_year=2015&amp;author=Lin%2CY-R">
                    Google Scholar</a> 
                </p></li><li data-counter="3."><p id="ref-CR3">Huang, P.-S., Boyken, S. E. &amp; Baker, D. The coming of age of de novo protein design. <i>Nature</i> <b>537</b>, 320–327 (2016).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/nature19946" data-track-action="article reference" href="https://doi.org/10.1038%2Fnature19946" aria-label="Article reference 3" data-doi="10.1038/nature19946">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://jvns.ca/articles/cas-redirect/1:CAS:528:DC%2BC28XhsFajsrzL" aria-label="CAS reference 3">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 3" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20coming%20of%20age%20of%20de%20novo%20protein%20design&amp;journal=Nature&amp;doi=10.1038%2Fnature19946&amp;volume=537&amp;pages=320-327&amp;publication_year=2016&amp;author=Huang%2CP-S&amp;author=Boyken%2CSE&amp;author=Baker%2CD">
                    Google Scholar</a> 
                </p></li><li data-counter="4."><p id="ref-CR4">Huang, P.-S. et al. De novo design of a four-fold symmetric TIM-barrel protein with atomic-level accuracy. <i>Nat. Chem. Biol.</i> <b>12</b>, 29–34 (2016).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/nchembio.1966" data-track-action="article reference" href="https://doi.org/10.1038%2Fnchembio.1966" aria-label="Article reference 4" data-doi="10.1038/nchembio.1966">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://jvns.ca/articles/cas-redirect/1:CAS:528:DC%2BC2MXhvVOgtLnP" aria-label="CAS reference 4">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 4" href="http://scholar.google.com/scholar_lookup?&amp;title=De%20novo%20design%20of%20a%20four-fold%20symmetric%20TIM-barrel%20protein%20with%20atomic-level%20accuracy&amp;journal=Nat.%20Chem.%20Biol.&amp;doi=10.1038%2Fnchembio.1966&amp;volume=12&amp;pages=29-34&amp;publication_year=2016&amp;author=Huang%2CP-S">
                    Google Scholar</a> 
                </p></li><li data-counter="5."><p id="ref-CR5">Boyken, S. E. et al. De novo design of protein homo-oligomers with modular hydrogen-bond network–mediated specificity. <i>Science</i> <b>352</b>, 680–687 (2016).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1126/science.aad8865" data-track-action="article reference" href="https://doi.org/10.1126%2Fscience.aad8865" aria-label="Article reference 5" data-doi="10.1126/science.aad8865">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://jvns.ca/articles/cas-redirect/1:CAS:528:DC%2BC28XntVeqt7g%3D" aria-label="CAS reference 5">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 5" href="http://scholar.google.com/scholar_lookup?&amp;title=De%20novo%20design%20of%20protein%20homo-oligomers%20with%20modular%20hydrogen-bond%20network%E2%80%93mediated%20specificity&amp;journal=Science&amp;doi=10.1126%2Fscience.aad8865&amp;volume=352&amp;pages=680-687&amp;publication_year=2016&amp;author=Boyken%2CSE">
                    Google Scholar</a> 
                </p></li><li data-counter="6."><p id="ref-CR6">Lapedes, A. S., Bertrand, G. G., LonChang, L. &amp; Stormo, G. D. Correlated mutations in models of protein sequences: Phylogenetic and structural effects. <i>Lect. Notes Monogr. Ser.</i> <b>33</b>, 236–256 (1999).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1214/lnms/1215455556" data-track-action="article reference" href="https://doi.org/10.1214%2Flnms%2F1215455556" aria-label="Article reference 6" data-doi="10.1214/lnms/1215455556">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 6" href="http://scholar.google.com/scholar_lookup?&amp;title=Correlated%20mutations%20in%20models%20of%20protein%20sequences%3A%20Phylogenetic%20and%20structural%20effects&amp;journal=Lect.%20Notes%20Monogr.%20Ser.&amp;doi=10.1214%2Flnms%2F1215455556&amp;volume=33&amp;pages=236-256&amp;publication_year=1999&amp;author=Lapedes%2CAS&amp;author=Bertrand%2CGG&amp;author=LonChang%2CL&amp;author=Stormo%2CGD">
                    Google Scholar</a> 
                </p></li><li data-counter="7."><p id="ref-CR7">Russ, W. P. et al. An evolution-based model for designing chorismate mutase enzymes. <i>Science</i> <b>369</b>, 440–445 (2020).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1126/science.aba3304" data-track-action="article reference" href="https://doi.org/10.1126%2Fscience.aba3304" aria-label="Article reference 7" data-doi="10.1126/science.aba3304">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://jvns.ca/articles/cas-redirect/1:CAS:528:DC%2BB3cXhsVCmtL%2FL" aria-label="CAS reference 7">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 7" href="http://scholar.google.com/scholar_lookup?&amp;title=An%20evolution-based%20model%20for%20designing%20chorismate%20mutase%20enzymes&amp;journal=Science&amp;doi=10.1126%2Fscience.aba3304&amp;volume=369&amp;pages=440-445&amp;publication_year=2020&amp;author=Russ%2CWP">
                    Google Scholar</a> 
                </p></li><li data-counter="8."><p id="ref-CR8">Hopf, T. A. et al. The EVcouplings Python framework for coevolutionary sequence analysis. <i>Bioinformatics</i> <b>35</b>, 1582–1584 (2019).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1093/bioinformatics/bty862" data-track-action="article reference" href="https://doi.org/10.1093%2Fbioinformatics%2Fbty862" aria-label="Article reference 8" data-doi="10.1093/bioinformatics/bty862">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://jvns.ca/articles/cas-redirect/1:CAS:528:DC%2BC1MXitlantb7N" aria-label="CAS reference 8">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 8" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20EVcouplings%20Python%20framework%20for%20coevolutionary%20sequence%20analysis&amp;journal=Bioinformatics&amp;doi=10.1093%2Fbioinformatics%2Fbty862&amp;volume=35&amp;pages=1582-1584&amp;publication_year=2019&amp;author=Hopf%2CTA">
                    Google Scholar</a> 
                </p></li><li data-counter="9."><p id="ref-CR9">Morcos, F. et al. Direct-coupling analysis of residue coevolution captures native contacts across many protein families. <i>Proc. Natl Acad. Sci. USA</i> <b>108</b>, E1293–E1301 (2011).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1073/pnas.1111471108" data-track-action="article reference" href="https://doi.org/10.1073%2Fpnas.1111471108" aria-label="Article reference 9" data-doi="10.1073/pnas.1111471108">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://jvns.ca/articles/cas-redirect/1:CAS:528:DC%2BC3MXhs1CmurjN" aria-label="CAS reference 9">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 9" href="http://scholar.google.com/scholar_lookup?&amp;title=Direct-coupling%20analysis%20of%20residue%20coevolution%20captures%20native%20contacts%20across%20many%20protein%20families&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;doi=10.1073%2Fpnas.1111471108&amp;volume=108&amp;pages=E1293-E1301&amp;publication_year=2011&amp;author=Morcos%2CF">
                    Google Scholar</a> 
                </p></li><li data-counter="10."><p id="ref-CR10">Ovchinnikov, S., Kamisetty, H. &amp; Baker, D. Robust and accurate prediction of residue-residue interactions across protein interfaces using evolutionary information. <i>eLife</i> <b>3</b>, e02030 (2014).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.7554/eLife.02030" data-track-action="article reference" href="https://doi.org/10.7554%2FeLife.02030" aria-label="Article reference 10" data-doi="10.7554/eLife.02030">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 10" href="http://scholar.google.com/scholar_lookup?&amp;title=Robust%20and%20accurate%20prediction%20of%20residue-residue%20interactions%20across%20protein%20interfaces%20using%20evolutionary%20information&amp;journal=eLife&amp;doi=10.7554%2FeLife.02030&amp;volume=3&amp;publication_year=2014&amp;author=Ovchinnikov%2CS&amp;author=Kamisetty%2CH&amp;author=Baker%2CD">
                    Google Scholar</a> 
                </p></li><li data-counter="11."><p id="ref-CR11">Alley, E. C., Khimulya, G., Biswas, S., AlQuraishi, M. &amp; Church, G. M. Unified rational protein engineering with sequence-based deep representation learning. <i>Nat. Methods</i> <b>16</b>, 1315–1322 (2019).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/s41592-019-0598-1" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41592-019-0598-1" aria-label="Article reference 11" data-doi="10.1038/s41592-019-0598-1">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://jvns.ca/articles/cas-redirect/1:CAS:528:DC%2BC1MXitVSlsbnJ" aria-label="CAS reference 11">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 11" href="http://scholar.google.com/scholar_lookup?&amp;title=Unified%20rational%20protein%20engineering%20with%20sequence-based%20deep%20representation%20learning&amp;journal=Nat.%20Methods&amp;doi=10.1038%2Fs41592-019-0598-1&amp;volume=16&amp;pages=1315-1322&amp;publication_year=2019&amp;author=Alley%2CEC&amp;author=Khimulya%2CG&amp;author=Biswas%2CS&amp;author=AlQuraishi%2CM&amp;author=Church%2CGM">
                    Google Scholar</a> 
                </p></li><li data-counter="12."><p id="ref-CR12">Wu, Z. et al. Signal peptides generated by attention-based neural networks. <i>ACS Synth. Biol.</i> <b>9</b>, 2154–2161 (2020).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1021/acssynbio.0c00219" data-track-action="article reference" href="https://doi.org/10.1021%2Facssynbio.0c00219" aria-label="Article reference 12" data-doi="10.1021/acssynbio.0c00219">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://jvns.ca/articles/cas-redirect/1:CAS:528:DC%2BB3cXhtlKqsb%2FP" aria-label="CAS reference 12">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 12" href="http://scholar.google.com/scholar_lookup?&amp;title=Signal%20peptides%20generated%20by%20attention-based%20neural%20networks&amp;journal=ACS%20Synth.%20Biol.&amp;doi=10.1021%2Facssynbio.0c00219&amp;volume=9&amp;pages=2154-2161&amp;publication_year=2020&amp;author=Wu%2CZ">
                    Google Scholar</a> 
                </p></li><li data-counter="13."><p id="ref-CR13">Shin, J.-E. et al. Protein design and variant prediction using autoregressive generative models. <i>Nat. Commun.</i> <b>12</b>, 2403 (2021).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/s41467-021-22732-w" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41467-021-22732-w" aria-label="Article reference 13" data-doi="10.1038/s41467-021-22732-w">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://jvns.ca/articles/cas-redirect/1:CAS:528:DC%2BB3MXpsl2ktbY%3D" aria-label="CAS reference 13">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 13" href="http://scholar.google.com/scholar_lookup?&amp;title=Protein%20design%20and%20variant%20prediction%20using%20autoregressive%20generative%20models&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-021-22732-w&amp;volume=12&amp;publication_year=2021&amp;author=Shin%2CJ-E">
                    Google Scholar</a> 
                </p></li><li data-counter="14."><p id="ref-CR14">Jumper, J. et al. Highly accurate protein structure prediction with AlphaFold. <i>Nature</i> <b>596</b>, 583–589 (2021).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/s41586-021-03819-2" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41586-021-03819-2" aria-label="Article reference 14" data-doi="10.1038/s41586-021-03819-2">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://jvns.ca/articles/cas-redirect/1:CAS:528:DC%2BB3MXhvVaktrrL" aria-label="CAS reference 14">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 14" href="http://scholar.google.com/scholar_lookup?&amp;title=Highly%20accurate%20protein%20structure%20prediction%20with%20AlphaFold&amp;journal=Nature&amp;doi=10.1038%2Fs41586-021-03819-2&amp;volume=596&amp;pages=583-589&amp;publication_year=2021&amp;author=Jumper%2CJ">
                    Google Scholar</a> 
                </p></li><li data-counter="15."><p id="ref-CR15">Bryant, D. H. et al. Deep diversification of an AAV capsid protein by machine learning. <i>Nat. Biotechnol.</i> <b>39</b>, 691–696 (2021).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/s41587-020-00793-4" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41587-020-00793-4" aria-label="Article reference 15" data-doi="10.1038/s41587-020-00793-4">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://jvns.ca/articles/cas-redirect/1:CAS:528:DC%2BB3MXjvVOnsb4%3D" aria-label="CAS reference 15">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 15" href="http://scholar.google.com/scholar_lookup?&amp;title=Deep%20diversification%20of%20an%20AAV%20capsid%20protein%20by%20machine%20learning&amp;journal=Nat.%20Biotechnol.&amp;doi=10.1038%2Fs41587-020-00793-4&amp;volume=39&amp;pages=691-696&amp;publication_year=2021&amp;author=Bryant%2CDH">
                    Google Scholar</a> 
                </p></li><li data-counter="16."><p id="ref-CR16">Das, P. et al. Accelerated antimicrobial discovery via deep generative models and molecular dynamics simulations. <i>Nat. Biomed. Eng.</i> <b>5</b>, 613–623 (2021).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/s41551-021-00689-x" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41551-021-00689-x" aria-label="Article reference 16" data-doi="10.1038/s41551-021-00689-x">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://jvns.ca/articles/cas-redirect/1:CAS:528:DC%2BB3MXmvVOhsr8%3D" aria-label="CAS reference 16">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 16" href="http://scholar.google.com/scholar_lookup?&amp;title=Accelerated%20antimicrobial%20discovery%20via%20deep%20generative%20models%20and%20molecular%20dynamics%20simulations&amp;journal=Nat.%20Biomed.%20Eng.&amp;doi=10.1038%2Fs41551-021-00689-x&amp;volume=5&amp;pages=613-623&amp;publication_year=2021&amp;author=Das%2CP">
                    Google Scholar</a> 
                </p></li><li data-counter="17."><p id="ref-CR17">Anishchenko, I. et al. De novo protein design by deep network hallucination. <i>Nature</i> <b>600</b>, 547–552 (2021).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/s41586-021-04184-w" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41586-021-04184-w" aria-label="Article reference 17" data-doi="10.1038/s41586-021-04184-w">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://jvns.ca/articles/cas-redirect/1:CAS:528:DC%2BB3MXis1eqsb7I" aria-label="CAS reference 17">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 17" href="http://scholar.google.com/scholar_lookup?&amp;title=De%20novo%20protein%20design%20by%20deep%20network%20hallucination&amp;journal=Nature&amp;doi=10.1038%2Fs41586-021-04184-w&amp;volume=600&amp;pages=547-552&amp;publication_year=2021&amp;author=Anishchenko%2CI">
                    Google Scholar</a> 
                </p></li><li data-counter="18."><p id="ref-CR18">Moffat, L., Kandathil, S. M. &amp; Jones, D. T. Design in the DARK: Learning deep generative models for De Novo Protein Design. Preprint at bioRxiv <a href="https://doi.org/10.1101/2022.01.27.478087">https://doi.org/10.1101/2022.01.27.478087</a> (2022).</p></li><li data-counter="19."><p id="ref-CR19">Ferruz, N., Schmidt, S. &amp; Höcker, B. ProtGPT2 is a deep unsupervised language model for protein design. <i>Nat. Commun.</i> <b>13</b>, 4348 (2022).</p></li><li data-counter="20."><p id="ref-CR20">Huang, B. et al. A backbone-centred energy function of neural networks for protein design. <i>Nature</i> <b>602</b>, 523–528 (2022).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/s41586-021-04383-5" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41586-021-04383-5" aria-label="Article reference 20" data-doi="10.1038/s41586-021-04383-5">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://jvns.ca/articles/cas-redirect/1:CAS:528:DC%2BB38XjtlSnsr8%3D" aria-label="CAS reference 20">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 20" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20backbone-centred%20energy%20function%20of%20neural%20networks%20for%20protein%20design&amp;journal=Nature&amp;doi=10.1038%2Fs41586-021-04383-5&amp;volume=602&amp;pages=523-528&amp;publication_year=2022&amp;author=Huang%2CB">
                    Google Scholar</a> 
                </p></li><li data-counter="21."><p id="ref-CR21">Leinonen, R. et al. UniProt archive. <i>Bioinformatics</i> <b>20</b>, 3236–3237 (2004).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1093/bioinformatics/bth191" data-track-action="article reference" href="https://doi.org/10.1093%2Fbioinformatics%2Fbth191" aria-label="Article reference 21" data-doi="10.1093/bioinformatics/bth191">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://jvns.ca/articles/cas-redirect/1:CAS:528:DC%2BD2cXhtVSrtbfK" aria-label="CAS reference 21">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 21" href="http://scholar.google.com/scholar_lookup?&amp;title=UniProt%20archive&amp;journal=Bioinformatics&amp;doi=10.1093%2Fbioinformatics%2Fbth191&amp;volume=20&amp;pages=3236-3237&amp;publication_year=2004&amp;author=Leinonen%2CR">
                    Google Scholar</a> 
                </p></li><li data-counter="22."><p id="ref-CR22">Bairoch, A. et al. The Universal Protein Resource (UniProt). <i>Nucleic Acids Res.</i> <b>33</b>, D154–D159 (2005).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1093/nar/gki070" data-track-action="article reference" href="https://doi.org/10.1093%2Fnar%2Fgki070" aria-label="Article reference 22" data-doi="10.1093/nar/gki070">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://jvns.ca/articles/cas-redirect/1:CAS:528:DC%2BD2MXisVCktg%3D%3D" aria-label="CAS reference 22">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 22" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20Universal%20Protein%20Resource%20%28UniProt%29&amp;journal=Nucleic%20Acids%20Res.&amp;doi=10.1093%2Fnar%2Fgki070&amp;volume=33&amp;pages=D154-D159&amp;publication_year=2005&amp;author=Bairoch%2CA">
                    Google Scholar</a> 
                </p></li><li data-counter="23."><p id="ref-CR23">Finn, R. D. et al. Pfam: the protein families database. <i>Nucleic Acids Res.</i> <b>42</b>, D222–D230 (2014).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1093/nar/gkt1223" data-track-action="article reference" href="https://doi.org/10.1093%2Fnar%2Fgkt1223" aria-label="Article reference 23" data-doi="10.1093/nar/gkt1223">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://jvns.ca/articles/cas-redirect/1:CAS:528:DC%2BC2cXos1al" aria-label="CAS reference 23">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 23" href="http://scholar.google.com/scholar_lookup?&amp;title=Pfam%3A%20the%20protein%20families%20database&amp;journal=Nucleic%20Acids%20Res.&amp;doi=10.1093%2Fnar%2Fgkt1223&amp;volume=42&amp;pages=D222-D230&amp;publication_year=2014&amp;author=Finn%2CRD">
                    Google Scholar</a> 
                </p></li><li data-counter="24."><p id="ref-CR24">Vaswani, A. et al. Attention is all you need. In <i>31st Conference on Neural Information Processing Systems</i> (NIPS, 2017).</p></li><li data-counter="25."><p id="ref-CR25">Devlin, J., Chang, M.-W., Lee, K. &amp; Toutanova, K. BERT: Pre-training of deep bidirectional transformers for language understanding. In <i>Proceedings of North American Chapter of the Association for Computational Linguistics: Human Language Technologies</i> (NAACL-HLT, 2019).</p></li><li data-counter="26."><p id="ref-CR26">Brown, T. B. et al. Language models are few-shot learners. In <i>34th Conference on Neural Information Processing Systems</i> (NeurIPS, 2020).</p></li><li data-counter="27."><p id="ref-CR27">Zellers, R. et al. Defending against neural fake news. In <i>33rd Conference on Neural Information Processing Systems</i> (NeurIPS, 2019).</p></li><li data-counter="28."><p id="ref-CR28">Keskar, N. S., McCann, B., Varshney, L. R., Xiong, C. &amp; Socher, R. CTRL: a conditional transformer language model for controllable generation. Preprint at <i>arXiv</i> <a href="https://doi.org/10.48550/arXiv.1909.05858">https://doi.org/10.48550/arXiv.1909.05858</a> (2019).</p></li><li data-counter="29."><p id="ref-CR29">AlQuraishi, M. The future of protein science will not be supervised. <i>Some Thoughts on a Mysterious Universe</i> <a href="https://moalquraishi.wordpress.com/2019/04/01/the-future-of-protein-science-will-not-be-supervised/">https://moalquraishi.wordpress.com/2019/04/01/the-future-of-protein-science-will-not-be-supervised/</a> (2019).</p></li><li data-counter="30."><p id="ref-CR30">Rives, A. et al. Biological structure and function emerge from scaling unsupervised learning to 250 million protein sequences. <i>Proc. Natl Acad. Sci. USA</i> <b>118</b>, e2016239118 (2021).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1073/pnas.2016239118" data-track-action="article reference" href="https://doi.org/10.1073%2Fpnas.2016239118" aria-label="Article reference 30" data-doi="10.1073/pnas.2016239118">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://jvns.ca/articles/cas-redirect/1:CAS:528:DC%2BB3MXovVantro%3D" aria-label="CAS reference 30">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 30" href="http://scholar.google.com/scholar_lookup?&amp;title=Biological%20structure%20and%20function%20emerge%20from%20scaling%20unsupervised%20learning%20to%20250%20million%20protein%20sequences&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;doi=10.1073%2Fpnas.2016239118&amp;volume=118&amp;publication_year=2021&amp;author=Rives%2CA">
                    Google Scholar</a> 
                </p></li><li data-counter="31."><p id="ref-CR31">Elnaggar, A. et al. ProtTrans: towards cracking the language of lifes code through self-supervised deep learning and high performance computing. <i>IEEE Trans. Pattern Anal. Mach. Intell.</i> <b>44</b>, 7112–7127 (2021).</p><p><a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 31" href="http://scholar.google.com/scholar_lookup?&amp;title=ProtTrans%3A%20towards%20cracking%20the%20language%20of%20lifes%20code%20through%20self-supervised%20deep%20learning%20and%20high%20performance%20computing&amp;journal=IEEE%20Trans.%20Pattern%20Anal.%20Mach.%20Intell.&amp;volume=44&amp;pages=7112-7127&amp;publication_year=2021&amp;author=Elnaggar%2CA">
                    Google Scholar</a> 
                </p></li><li data-counter="32."><p id="ref-CR32">Peters, M. E. et al. Deep contextualized word representations. In <i>Proceedings of North American Chapter of the Association for Computational Linguistics: Human Language Technologies</i> (NAACL-HLT, 2018).</p></li><li data-counter="33."><p id="ref-CR33">Howard, J. &amp; Ruder, S. Universal language model fine-tuning for text classification. In <i>Proceedings of the 56th Annual Meeting of the Association for Computational Linguistics</i> (ACL, 2018).</p></li><li data-counter="34."><p id="ref-CR34">Radford, A., Narasimhan, K., Salimans, T. &amp; Sutskever, I. Improving language understanding by generative pre-training. Preprint at <a href="https://s3-us-west-2.amazonaws.com/openai-assets/research-covers/language-unsupervised/language_understanding_paper.pdf">https://s3-us-west-2.amazonaws.com/openai-assets/research-covers/language-unsupervised/language_understanding_paper.pdf</a> (2018).</p></li><li data-counter="35."><p id="ref-CR35">Biswas, S., Khimulya, G., Alley, E. C., Esvelt, K. M. &amp; Church, G. M. Low-N protein engineering with data-efficient deep learning. <i>Nat. Methods</i> <b>18</b>, 389–396 (2021).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/s41592-021-01100-y" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41592-021-01100-y" aria-label="Article reference 35" data-doi="10.1038/s41592-021-01100-y">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://jvns.ca/articles/cas-redirect/1:CAS:528:DC%2BB3MXosVCltrg%3D" aria-label="CAS reference 35">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 35" href="http://scholar.google.com/scholar_lookup?&amp;title=Low-N%20protein%20engineering%20with%20data-efficient%20deep%20learning&amp;journal=Nat.%20Methods&amp;doi=10.1038%2Fs41592-021-01100-y&amp;volume=18&amp;pages=389-396&amp;publication_year=2021&amp;author=Biswas%2CS&amp;author=Khimulya%2CG&amp;author=Alley%2CEC&amp;author=Esvelt%2CKM&amp;author=Church%2CGM">
                    Google Scholar</a> 
                </p></li><li data-counter="36."><p id="ref-CR36">Pfaff, C. W. Constraints on language mixing: Intrasentential code-switching and borrowing in Spanish/English. <i>Language</i> <b>55</b>, 291–318 (1979).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.2307/412586" data-track-action="article reference" href="https://doi.org/10.2307%2F412586" aria-label="Article reference 36" data-doi="10.2307/412586">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 36" href="http://scholar.google.com/scholar_lookup?&amp;title=Constraints%20on%20language%20mixing%3A%20Intrasentential%20code-switching%20and%20borrowing%20in%20Spanish%2FEnglish&amp;journal=Language&amp;doi=10.2307%2F412586&amp;volume=55&amp;pages=291-318&amp;publication_year=1979&amp;author=Pfaff%2CCW">
                    Google Scholar</a> 
                </p></li><li data-counter="37."><p id="ref-CR37">Poplack, S. Sometimes I’ll start a sentence in Spanish Y TERMINO EN ESPAÑOL: toward a typology of code-switching. <i>Linguistics</i> <b>18</b>, 581–618 (1980).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1515/ling.1980.18.7-8.581" data-track-action="article reference" href="https://doi.org/10.1515%2Fling.1980.18.7-8.581" aria-label="Article reference 37" data-doi="10.1515/ling.1980.18.7-8.581">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 37" href="http://scholar.google.com/scholar_lookup?&amp;title=Sometimes%20I%E2%80%99ll%20start%20a%20sentence%20in%20Spanish%20Y%20TERMINO%20EN%20ESPA%C3%91OL%3A%20toward%20a%20typology%20of%20code-switching&amp;journal=Linguistics&amp;doi=10.1515%2Fling.1980.18.7-8.581&amp;volume=18&amp;pages=581-618&amp;publication_year=1980&amp;author=Poplack%2CS">
                    Google Scholar</a> 
                </p></li><li data-counter="38."><p id="ref-CR38">Dathathri, S. et al. Plug and play language models: a simple approach to controlled text generation. In <i>8th International Conference on Learning Representations</i> (ICLR, 2020).</p></li><li data-counter="39."><p id="ref-CR39">Matthews, B. W. Comparison of the predicted and observed secondary structure of T4 phage lysozyme. <i>Biochim. Biophys. Acta</i> <b>405</b>, 442–451 (1975).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/0005-2795(75)90109-9" data-track-action="article reference" href="https://doi.org/10.1016%2F0005-2795%2875%2990109-9" aria-label="Article reference 39" data-doi="10.1016/0005-2795(75)90109-9">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://jvns.ca/articles/cas-redirect/1:CAS:528:DyaE2MXlslCksbk%3D" aria-label="CAS reference 39">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 39" href="http://scholar.google.com/scholar_lookup?&amp;title=Comparison%20of%20the%20predicted%20and%20observed%20secondary%20structure%20of%20T4%20phage%20lysozyme&amp;journal=Biochim.%20Biophys.%20Acta&amp;doi=10.1016%2F0005-2795%2875%2990109-9&amp;volume=405&amp;pages=442-451&amp;publication_year=1975&amp;author=Matthews%2CBW">
                    Google Scholar</a> 
                </p></li><li data-counter="40."><p id="ref-CR40">Broendum, S. S., Buckle, A. M. &amp; McGowan, S. Catalytic diversity and cell wall binding repeats in the phage-encoded endolysins. <i>Mol. Microbiol.</i> <b>110</b>, 879–896 (2018).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1111/mmi.14134" data-track-action="article reference" href="https://doi.org/10.1111%2Fmmi.14134" aria-label="Article reference 40" data-doi="10.1111/mmi.14134">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://jvns.ca/articles/cas-redirect/1:CAS:528:DC%2BC1cXitFOis7jK" aria-label="CAS reference 40">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 40" href="http://scholar.google.com/scholar_lookup?&amp;title=Catalytic%20diversity%20and%20cell%20wall%20binding%20repeats%20in%20the%20phage-encoded%20endolysins&amp;journal=Mol.%20Microbiol.&amp;doi=10.1111%2Fmmi.14134&amp;volume=110&amp;pages=879-896&amp;publication_year=2018&amp;author=Broendum%2CSS&amp;author=Buckle%2CAM&amp;author=McGowan%2CS">
                    Google Scholar</a> 
                </p></li><li data-counter="41."><p id="ref-CR41">Love, M. J., Abeysekera, G. S., Muscroft-Taylor, A. C., Billington, C. &amp; Dobson, R. C. J. On the catalytic mechanism of bacteriophage endolysins: opportunities for engineering. <i>Biochim. Biophys. Acta. Proteins Proteom.</i> <b>1868</b>, 140302 (2020).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.bbapap.2019.140302" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.bbapap.2019.140302" aria-label="Article reference 41" data-doi="10.1016/j.bbapap.2019.140302">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://jvns.ca/articles/cas-redirect/1:CAS:528:DC%2BC1MXitFamtb3P" aria-label="CAS reference 41">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 41" href="http://scholar.google.com/scholar_lookup?&amp;title=On%20the%20catalytic%20mechanism%20of%20bacteriophage%20endolysins%3A%20opportunities%20for%20engineering&amp;journal=Biochim.%20Biophys.%20Acta.%20Proteins%20Proteom.&amp;doi=10.1016%2Fj.bbapap.2019.140302&amp;volume=1868&amp;publication_year=2020&amp;author=Love%2CMJ&amp;author=Abeysekera%2CGS&amp;author=Muscroft-Taylor%2CAC&amp;author=Billington%2CC&amp;author=Dobson%2CRCJ">
                    Google Scholar</a> 
                </p></li><li data-counter="42."><p id="ref-CR42">Martin, P. P. <i>Potts Models And Related Problems In Statistical Mechanics</i> (World Scientific, 1991).</p></li><li data-counter="43."><p id="ref-CR43">Thomas, J., Ramakrishnan, N. &amp; Bailey-Kellogg, C. Graphical models of residue coupling in protein families. <i>IEEE/ACM Trans. Comput. Biol. Bioinform.</i> <b>5</b>, 183–197 (2008).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1109/TCBB.2007.70225" data-track-action="article reference" href="https://doi.org/10.1109%2FTCBB.2007.70225" aria-label="Article reference 43" data-doi="10.1109/TCBB.2007.70225">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://jvns.ca/articles/cas-redirect/1:CAS:528:DC%2BD1cXns1artr8%3D" aria-label="CAS reference 43">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 43" href="http://scholar.google.com/scholar_lookup?&amp;title=Graphical%20models%20of%20residue%20coupling%20in%20protein%20families&amp;journal=IEEE%2FACM%20Trans.%20Comput.%20Biol.%20Bioinform.&amp;doi=10.1109%2FTCBB.2007.70225&amp;volume=5&amp;pages=183-197&amp;publication_year=2008&amp;author=Thomas%2CJ&amp;author=Ramakrishnan%2CN&amp;author=Bailey-Kellogg%2CC">
                    Google Scholar</a> 
                </p></li><li data-counter="44."><p id="ref-CR44">Weigt, M., White, R. A., Szurmant, H., Hoch, J. A. &amp; Hwa, T. Identification of direct residue contacts in protein–protein interaction by message passing. <i>Proc. Natl Acad. Sci. USA</i> <b>106</b>, 67–72 (2009).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1073/pnas.0805923106" data-track-action="article reference" href="https://doi.org/10.1073%2Fpnas.0805923106" aria-label="Article reference 44" data-doi="10.1073/pnas.0805923106">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://jvns.ca/articles/cas-redirect/1:CAS:528:DC%2BD1MXltF2jug%3D%3D" aria-label="CAS reference 44">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 44" href="http://scholar.google.com/scholar_lookup?&amp;title=Identification%20of%20direct%20residue%20contacts%20in%20protein%E2%80%93protein%20interaction%20by%20message%20passing&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;doi=10.1073%2Fpnas.0805923106&amp;volume=106&amp;pages=67-72&amp;publication_year=2009&amp;author=Weigt%2CM&amp;author=White%2CRA&amp;author=Szurmant%2CH&amp;author=Hoch%2CJA&amp;author=Hwa%2CT">
                    Google Scholar</a> 
                </p></li><li data-counter="45."><p id="ref-CR45">Balakrishnan, S., Kamisetty, H., Carbonell, J. G., Lee, S.-I. &amp; Langmead, C. J. Learning generative models for protein fold families. <i>Proteins</i> <b>79</b>, 1061–1078 (2011).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1002/prot.22934" data-track-action="article reference" href="https://doi.org/10.1002%2Fprot.22934" aria-label="Article reference 45" data-doi="10.1002/prot.22934">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://jvns.ca/articles/cas-redirect/1:CAS:528:DC%2BC3MXkvVWrsLs%3D" aria-label="CAS reference 45">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 45" href="http://scholar.google.com/scholar_lookup?&amp;title=Learning%20generative%20models%20for%20protein%20fold%20families&amp;journal=Proteins&amp;doi=10.1002%2Fprot.22934&amp;volume=79&amp;pages=1061-1078&amp;publication_year=2011&amp;author=Balakrishnan%2CS&amp;author=Kamisetty%2CH&amp;author=Carbonell%2CJG&amp;author=Lee%2CS-I&amp;author=Langmead%2CCJ">
                    Google Scholar</a> 
                </p></li><li data-counter="46."><p id="ref-CR46">Stein, R. R., Marks, D. S. &amp; Sander, C. Inferring pairwise interactions from biological data using maximum-entropy probability models. <i>PLoS Comput. Biol.</i> <b>11</b>, e1004182 (2015).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1371/journal.pcbi.1004182" data-track-action="article reference" href="https://doi.org/10.1371%2Fjournal.pcbi.1004182" aria-label="Article reference 46" data-doi="10.1371/journal.pcbi.1004182">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 46" href="http://scholar.google.com/scholar_lookup?&amp;title=Inferring%20pairwise%20interactions%20from%20biological%20data%20using%20maximum-entropy%20probability%20models&amp;journal=PLoS%20Comput.%20Biol.&amp;doi=10.1371%2Fjournal.pcbi.1004182&amp;volume=11&amp;publication_year=2015&amp;author=Stein%2CRR&amp;author=Marks%2CDS&amp;author=Sander%2CC">
                    Google Scholar</a> 
                </p></li><li data-counter="47."><p id="ref-CR47">Mirdita, M., Steinegger, M., Breitwieser, F., Söding, J. &amp; Levy Karin, E. Fast and sensitive taxonomic assignment to metagenomic contigs. <i>Binformatics</i> <b>37</b>, 3029–3031 (2021).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1093/bioinformatics/btab184" data-track-action="article reference" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtab184" aria-label="Article reference 47" data-doi="10.1093/bioinformatics/btab184">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://jvns.ca/articles/cas-redirect/1:CAS:528:DC%2BB38Xitl2lsro%3D" aria-label="CAS reference 47">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 47" href="http://scholar.google.com/scholar_lookup?&amp;title=Fast%20and%20sensitive%20taxonomic%20assignment%20to%20metagenomic%20contigs&amp;journal=Binformatics&amp;doi=10.1093%2Fbioinformatics%2Fbtab184&amp;volume=37&amp;pages=3029-3031&amp;publication_year=2021&amp;author=Mirdita%2CM&amp;author=Steinegger%2CM&amp;author=Breitwieser%2CF&amp;author=S%C3%B6ding%2CJ&amp;author=Levy%20Karin%2CE">
                    Google Scholar</a> 
                </p></li><li data-counter="48."><p id="ref-CR48">Mooers, B. H. M., Tronrud, D. E. &amp; Matthews, B. W. Evaluation at atomic resolution of the role of strain in destabilizing the temperature-sensitive T4 lysozyme mutant Arg 96 → His. <i>Protein Sci.</i> <b>18</b>, 863–870 (2009).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1002/pro.93" data-track-action="article reference" href="https://doi.org/10.1002%2Fpro.93" aria-label="Article reference 48" data-doi="10.1002/pro.93">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://jvns.ca/articles/cas-redirect/1:CAS:528:DC%2BD1MXms1agtrs%3D" aria-label="CAS reference 48">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 48" href="http://scholar.google.com/scholar_lookup?&amp;title=Evaluation%20at%20atomic%20resolution%20of%20the%20role%20of%20strain%20in%20destabilizing%20the%20temperature-sensitive%20T4%20lysozyme%20mutant%20Arg%2096%20%E2%86%92%20His&amp;journal=Protein%20Sci.&amp;doi=10.1002%2Fpro.93&amp;volume=18&amp;pages=863-870&amp;publication_year=2009&amp;author=Mooers%2CBHM&amp;author=Tronrud%2CDE&amp;author=Matthews%2CBW">
                    Google Scholar</a> 
                </p></li><li data-counter="49."><p id="ref-CR49">Baase, W. A., Liu, L., Tronrud, D. E. &amp; Matthews, B. W. Lessons from the lysozyme of phage T4. <i>Protein Sci.</i> <b>19</b>, 631–641 (2010).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1002/pro.344" data-track-action="article reference" href="https://doi.org/10.1002%2Fpro.344" aria-label="Article reference 49" data-doi="10.1002/pro.344">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://jvns.ca/articles/cas-redirect/1:CAS:528:DC%2BC3cXktFyrtL8%3D" aria-label="CAS reference 49">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 49" href="http://scholar.google.com/scholar_lookup?&amp;title=Lessons%20from%20the%20lysozyme%20of%20phage%20T4&amp;journal=Protein%20Sci.&amp;doi=10.1002%2Fpro.344&amp;volume=19&amp;pages=631-641&amp;publication_year=2010&amp;author=Baase%2CWA&amp;author=Liu%2CL&amp;author=Tronrud%2CDE&amp;author=Matthews%2CBW">
                    Google Scholar</a> 
                </p></li><li data-counter="50."><p id="ref-CR50">Kuroki, R., Weaver, L. H. &amp; Matthews, B. W. A covalent enzyme–substrate intermediate with saccharide distortion in a mutant T4 lysozyme. <i>Science</i> <b>262</b>, 2030–2033 (1993).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1126/science.8266098" data-track-action="article reference" href="https://doi.org/10.1126%2Fscience.8266098" aria-label="Article reference 50" data-doi="10.1126/science.8266098">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://jvns.ca/articles/cas-redirect/1:CAS:528:DyaK2cXhsFOgt74%3D" aria-label="CAS reference 50">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 50" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20covalent%20enzyme%E2%80%93substrate%20intermediate%20with%20saccharide%20distortion%20in%20a%20mutant%20T4%20lysozyme&amp;journal=Science&amp;doi=10.1126%2Fscience.8266098&amp;volume=262&amp;pages=2030-2033&amp;publication_year=1993&amp;author=Kuroki%2CR&amp;author=Weaver%2CLH&amp;author=Matthews%2CBW">
                    Google Scholar</a> 
                </p></li><li data-counter="51."><p id="ref-CR51">Mchaourab, H. S., Oh, K. J., Fang, C. J. &amp; Hubbell, W. L. Conformation of T4 lysozyme in solution. Hinge-bending motion and the substrate-induced conformational transition studied by site-directed spin labeling. <i>Biochemistry</i> <b>36</b>, 307–316 (1997).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1021/bi962114m" data-track-action="article reference" href="https://doi.org/10.1021%2Fbi962114m" aria-label="Article reference 51" data-doi="10.1021/bi962114m">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://jvns.ca/articles/cas-redirect/1:CAS:528:DyaK2sXmtVOqsA%3D%3D" aria-label="CAS reference 51">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 51" href="http://scholar.google.com/scholar_lookup?&amp;title=Conformation%20of%20T4%20lysozyme%20in%20solution.%20Hinge-bending%20motion%20and%20the%20substrate-induced%20conformational%20transition%20studied%20by%20site-directed%20spin%20labeling&amp;journal=Biochemistry&amp;doi=10.1021%2Fbi962114m&amp;volume=36&amp;pages=307-316&amp;publication_year=1997&amp;author=Mchaourab%2CHS&amp;author=Oh%2CKJ&amp;author=Fang%2CCJ&amp;author=Hubbell%2CWL">
                    Google Scholar</a> 
                </p></li><li data-counter="52."><p id="ref-CR52">Kim, J.-K. et al. BetaCavityWeb: a webserver for molecular voids and channels. <i>Nucleic Acids Res.</i> <b>43</b>, W413–W418 (2015).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1093/nar/gkv360" data-track-action="article reference" href="https://doi.org/10.1093%2Fnar%2Fgkv360" aria-label="Article reference 52" data-doi="10.1093/nar/gkv360">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://jvns.ca/articles/cas-redirect/1:CAS:528:DC%2BC2sXhtVymtL3O" aria-label="CAS reference 52">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 52" href="http://scholar.google.com/scholar_lookup?&amp;title=BetaCavityWeb%3A%20a%20webserver%20for%20molecular%20voids%20and%20channels&amp;journal=Nucleic%20Acids%20Res.&amp;doi=10.1093%2Fnar%2Fgkv360&amp;volume=43&amp;pages=W413-W418&amp;publication_year=2015&amp;author=Kim%2CJ-K">
                    Google Scholar</a> 
                </p></li><li data-counter="53."><p id="ref-CR53">Rost, B. Twilight zone of protein sequence alignments. <i>Protein Eng.</i> <b>12</b>, 85–94 (1999).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1093/protein/12.2.85" data-track-action="article reference" href="https://doi.org/10.1093%2Fprotein%2F12.2.85" aria-label="Article reference 53" data-doi="10.1093/protein/12.2.85">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://jvns.ca/articles/cas-redirect/1:CAS:528:DyaK1MXhvFehs70%3D" aria-label="CAS reference 53">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 53" href="http://scholar.google.com/scholar_lookup?&amp;title=Twilight%20zone%20of%20protein%20sequence%20alignments&amp;journal=Protein%20Eng.&amp;doi=10.1093%2Fprotein%2F12.2.85&amp;volume=12&amp;pages=85-94&amp;publication_year=1999&amp;author=Rost%2CB">
                    Google Scholar</a> 
                </p></li><li data-counter="54."><p id="ref-CR54">Pearson, W. R. An introduction to sequence similarity (‘homology’) searching. <i>Curr. Protoc. Bioinforma.</i> <b>3</b>, 3.1 (2013). ChapterUnit.</p><p><a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 54" href="http://scholar.google.com/scholar_lookup?&amp;title=An%20introduction%20to%20sequence%20similarity%20%28%E2%80%98homology%E2%80%99%29%20searching&amp;journal=Curr.%20Protoc.%20Bioinforma.&amp;volume=3&amp;publication_year=2013&amp;author=Pearson%2CWR">
                    Google Scholar</a> 
                </p></li><li data-counter="55."><p id="ref-CR55">Repecka, D. et al. Expanding functional protein sequence spaces using generative adversarial networks. <i>Nat. Mach. Intell.</i> <b>3</b>, 324–333 (2021).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/s42256-021-00310-5" data-track-action="article reference" href="https://doi.org/10.1038%2Fs42256-021-00310-5" aria-label="Article reference 55" data-doi="10.1038/s42256-021-00310-5">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 55" href="http://scholar.google.com/scholar_lookup?&amp;title=Expanding%20functional%20protein%20sequence%20spaces%20using%20generative%20adversarial%20networks&amp;journal=Nat.%20Mach.%20Intell.&amp;doi=10.1038%2Fs42256-021-00310-5&amp;volume=3&amp;pages=324-333&amp;publication_year=2021&amp;author=Repecka%2CD">
                    Google Scholar</a> 
                </p></li><li data-counter="56."><p id="ref-CR56">Ruder, S., Peters, M. E., Swayamdipta, S. &amp; Wolf, T. Transfer learning in natural language processing. In <i>Proc. 2019 Conference of the North American Chapter of the Association for Computational Linguistics</i> (eds Jill Burstein, J., Doran, C. &amp; Solorio T.) (Association for Computational Linguistics, 2019).</p></li><li data-counter="57."><p id="ref-CR57">Huh, M., Agrawal, P. &amp; Efros, A. A. What makes ImageNet good for transfer learning? Preprint at <i>arXiv</i> <a href="https://doi.org/10.48550/arXiv.1608.08614">https://doi.org/10.48550/arXiv.1608.08614</a> (2016).</p></li><li data-counter="58."><p id="ref-CR58">LeCun, Y., Bengio, Y. &amp; Hinton, G. Deep learning. <i>Nature</i> <b>521</b>, 436–444 (2015).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/nature14539" data-track-action="article reference" href="https://doi.org/10.1038%2Fnature14539" aria-label="Article reference 58" data-doi="10.1038/nature14539">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://jvns.ca/articles/cas-redirect/1:CAS:528:DC%2BC2MXht1WlurzP" aria-label="CAS reference 58">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 58" href="http://scholar.google.com/scholar_lookup?&amp;title=Deep%20learning&amp;journal=Nature&amp;doi=10.1038%2Fnature14539&amp;volume=521&amp;pages=436-444&amp;publication_year=2015&amp;author=LeCun%2CY&amp;author=Bengio%2CY&amp;author=Hinton%2CG">
                    Google Scholar</a> 
                </p></li><li data-counter="59."><p id="ref-CR59">Norn, C. et al. Protein sequence design by conformational landscape optimization. <i>Proc. Natl Acad. Sci. USA</i> <b>118</b>, e2017228118 (2021).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1073/pnas.2017228118" data-track-action="article reference" href="https://doi.org/10.1073%2Fpnas.2017228118" aria-label="Article reference 59" data-doi="10.1073/pnas.2017228118">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://jvns.ca/articles/cas-redirect/1:CAS:528:DC%2BB3MXmsl2ns7o%3D" aria-label="CAS reference 59">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 59" href="http://scholar.google.com/scholar_lookup?&amp;title=Protein%20sequence%20design%20by%20conformational%20landscape%20optimization&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;doi=10.1073%2Fpnas.2017228118&amp;volume=118&amp;publication_year=2021&amp;author=Norn%2CC">
                    Google Scholar</a> 
                </p></li><li data-counter="60."><p id="ref-CR60">Anand, N. et al. Protein sequence design with a learned potential. <i>Nat. Commun.</i> <b>13</b>, 746 (2022).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/s41467-022-28313-9" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41467-022-28313-9" aria-label="Article reference 60" data-doi="10.1038/s41467-022-28313-9">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://jvns.ca/articles/cas-redirect/1:CAS:528:DC%2BB38XjtFSlu70%3D" aria-label="CAS reference 60">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 60" href="http://scholar.google.com/scholar_lookup?&amp;title=Protein%20sequence%20design%20with%20a%20learned%20potential&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-022-28313-9&amp;volume=13&amp;publication_year=2022&amp;author=Anand%2CN">
                    Google Scholar</a> 
                </p></li><li data-counter="61."><p id="ref-CR61">Federhen, S. The NCBI Taxonomy database. <i>Nucleic Acids Res.</i> <b>40</b>, D136–D143 (2012).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1093/nar/gkr1178" data-track-action="article reference" href="https://doi.org/10.1093%2Fnar%2Fgkr1178" aria-label="Article reference 61" data-doi="10.1093/nar/gkr1178">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://jvns.ca/articles/cas-redirect/1:CAS:528:DC%2BC3MXhs12htb7M" aria-label="CAS reference 61">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 61" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20NCBI%20Taxonomy%20database&amp;journal=Nucleic%20Acids%20Res.&amp;doi=10.1093%2Fnar%2Fgkr1178&amp;volume=40&amp;pages=D136-D143&amp;publication_year=2012&amp;author=Federhen%2CS">
                    Google Scholar</a> 
                </p></li><li data-counter="62."><p id="ref-CR62">Pettit, L. D. The IUPAC stability constants database. <i>Chem. Int.</i> <b>28</b>, 14–15 (2006).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://jvns.ca/articles/cas-redirect/1:CAS:528:DC%2BD28XhtFWlurvL" aria-label="CAS reference 62">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 62" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20IUPAC%20stability%20constants%20database&amp;journal=Chem.%20Int.&amp;volume=28&amp;pages=14-15&amp;publication_year=2006&amp;author=Pettit%2CLD">
                    Google Scholar</a> 
                </p></li><li data-counter="63."><p id="ref-CR63">Ashburner, M. et al. Gene ontology: tool for the unification of biology. The Gene Ontology Consortium. <i>Nat. Genet.</i> <b>25</b>, 25–29 (2000).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/75556" data-track-action="article reference" href="https://doi.org/10.1038%2F75556" aria-label="Article reference 63" data-doi="10.1038/75556">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://jvns.ca/articles/cas-redirect/1:CAS:528:DC%2BD3cXjtFSlsbc%3D" aria-label="CAS reference 63">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 63" href="http://scholar.google.com/scholar_lookup?&amp;title=Gene%20ontology%3A%20tool%20for%20the%20unification%20of%20biology.%20The%20Gene%20Ontology%20Consortium&amp;journal=Nat.%20Genet.&amp;doi=10.1038%2F75556&amp;volume=25&amp;pages=25-29&amp;publication_year=2000&amp;author=Ashburner%2CM">
                    Google Scholar</a> 
                </p></li><li data-counter="64."><p id="ref-CR64">Bengio, Y., Ducharme, R., Vincent, P. &amp; Janvin, C. A neural probabilistic language model. <i>J. Mach. Learn. Res.</i> <b>3</b>, 1137–1155 (2003).</p><p><a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 64" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20neural%20probabilistic%20language%20model&amp;journal=J.%20Mach.%20Learn.%20Res.&amp;volume=3&amp;pages=1137-1155&amp;publication_year=2003&amp;author=Bengio%2CY&amp;author=Ducharme%2CR&amp;author=Vincent%2CP&amp;author=Janvin%2CC">
                    Google Scholar</a> 
                </p></li><li data-counter="65."><p id="ref-CR65">Madani, A. et al. ProGen: language modeling for protein generation. Preprint at arXiv <a href="https://doi.org/10.1101/2020.03.07.982272">https://doi.org/10.1101/2020.03.07.982272</a> (2020).</p></li><li data-counter="66."><p id="ref-CR66">Vig, J. et al. BERTology meets biology: Interpreting attention in protein language models. In International Conference on Learning Representations (ICLR, 2020).</p></li><li data-counter="67."><p id="ref-CR67">Goyal, K., Dyer, C. &amp; Berg-Kirkpatrick, T. Exposing the implicit energy networks behind masked language models via metropolis–hastings. In <i>10th International Conference on Learning Representations</i> (ICLR, 2022).</p></li><li data-counter="68."><p id="ref-CR68">Bhattacharya, N. et al. Single layers of attention suffice to predict protein contacts. Preprint at <i>bioRxiv</i> <a href="https://doi.org/10.1101/2020.12.21.423882">https://doi.org/10.1101/2020.12.21.423882</a> (2020).</p></li><li data-counter="69."><p id="ref-CR69">Ramsauer, H. et al. Hopfield Networks is All You Need. Preprint at <i>arXiv</i> <a href="https://doi.org/10.48550/arXiv.2008.02217">https://doi.org/10.48550/arXiv.2008.02217</a> (2020).</p></li><li data-counter="70."><p id="ref-CR70">Alley, E., Khimulya, G., Biswas, S., AlQuraishi, M. &amp; Church, G. M. Unified rational protein engineering with sequence-based deep representation learning. <i>Nat. Methods</i> <b>16</b>, 1315–1322 (2019).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/s41592-019-0598-1" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41592-019-0598-1" aria-label="Article reference 70" data-doi="10.1038/s41592-019-0598-1">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://jvns.ca/articles/cas-redirect/1:CAS:528:DC%2BC1MXitVSlsbnJ" aria-label="CAS reference 70">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 70" href="http://scholar.google.com/scholar_lookup?&amp;title=Unified%20rational%20protein%20engineering%20with%20sequence-based%20deep%20representation%20learning&amp;journal=Nat.%20Methods&amp;doi=10.1038%2Fs41592-019-0598-1&amp;volume=16&amp;pages=1315-1322&amp;publication_year=2019&amp;author=Alley%2CE&amp;author=Khimulya%2CG&amp;author=Biswas%2CS&amp;author=AlQuraishi%2CM&amp;author=Church%2CGM">
                    Google Scholar</a> 
                </p></li><li data-counter="71."><p id="ref-CR71">Rao, R. et al. Evaluating protein transfer learning with TAPE. <i>Adv. Neural Inf. Process. Syst.</i> <b>32</b>, 9689–9701 (2019).</p><p><a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 71" href="http://scholar.google.com/scholar_lookup?&amp;title=Evaluating%20protein%20transfer%20learning%20with%20TAPE&amp;journal=Adv.%20Neural%20Inf.%20Process.%20Syst.&amp;volume=32&amp;pages=9689-9701&amp;publication_year=2019&amp;author=Rao%2CR">
                    Google Scholar</a> 
                </p></li><li data-counter="72."><p id="ref-CR72">Kingma, D. P. &amp; Ba, J. Adam: a method for stochastic optimization. Preprint <i>arXiv</i> <a href="https://doi.org/10.48550/arXiv.1412.6980">https://doi.org/10.48550/arXiv.1412.6980</a> (2014).</p></li><li data-counter="73."><p id="ref-CR73">Pascanu, R., Mikolov, T. &amp; Bengio, Y. On the difficulty of training recurrent neural networks. In <i>Proc. 30th International Conference on Machine Learning</i> (eds. Dasgupta, S. &amp; McAllester, D.) 1310–1318 (PMLR, 2013).</p></li><li data-counter="74."><p id="ref-CR74">Srivastava, N., Hinton, G., Krizhevsky, A., Sutskever, I. &amp; Salakhutdinov, R. Dropout: a simple way to prevent neural networks from overfitting. <i>J. Mach. Learn. Res.</i> <b>15</b>, 1929–1958 (2014).</p><p><a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 74" href="http://scholar.google.com/scholar_lookup?&amp;title=Dropout%3A%20a%20simple%20way%20to%20prevent%20neural%20networks%20from%20overfitting&amp;journal=J.%20Mach.%20Learn.%20Res.&amp;volume=15&amp;pages=1929-1958&amp;publication_year=2014&amp;author=Srivastava%2CN&amp;author=Hinton%2CG&amp;author=Krizhevsky%2CA&amp;author=Sutskever%2CI&amp;author=Salakhutdinov%2CR">
                    Google Scholar</a> 
                </p></li><li data-counter="75."><p id="ref-CR75">Holtzman, A., Buys, J., Du, L., Forbes, M. &amp; Choi, Y. The curious case of neural text degeneration. In <i>8th International Conference on Learning Representations</i> (ICLR, 2020).</p></li><li data-counter="76."><p id="ref-CR76">Goodfellow, I. J. et al. Generative adversarial networks. In <i>28th Conference on Neural Information Processing Systems</i> (NIPS, 2014).</p></li><li data-counter="77."><p id="ref-CR77">Koehn, P. Pharaoh: a beam search decoder for phrase-based statistical machine translation models. in <i>Machine Translation: From Real Users to Research</i> 115–124 (Springer, 2004).</p></li><li data-counter="78."><p id="ref-CR78">Sun, Z. Z. et al. Protocols for implementing an <i>Escherichia coli</i> based TX-TL cell-free expression system for synthetic biology. <i>J. Vis. Exp.</i> <b>16</b>, e50762 (2013).</p><p><a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 78" href="http://scholar.google.com/scholar_lookup?&amp;title=Protocols%20for%20implementing%20an%20Escherichia%20coli%20based%20TX-TL%20cell-free%20expression%20system%20for%20synthetic%20biology&amp;journal=J.%20Vis.%20Exp.&amp;volume=16&amp;publication_year=2013&amp;author=Sun%2CZZ">
                    Google Scholar</a> 
                </p></li><li data-counter="79."><p id="ref-CR79">Kabsch, W. XDS. <i>Acta Crystallogr. D Biol. Crystallogr.</i> <b>66</b>, 125–132 (2010).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1107/S0907444909047337" data-track-action="article reference" href="https://doi.org/10.1107%2FS0907444909047337" aria-label="Article reference 79" data-doi="10.1107/S0907444909047337">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://jvns.ca/articles/cas-redirect/1:CAS:528:DC%2BC3cXhs1SisLc%3D" aria-label="CAS reference 79">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 79" href="http://scholar.google.com/scholar_lookup?&amp;title=XDS&amp;journal=Acta%20Crystallogr.%20D%20Biol.%20Crystallogr.&amp;doi=10.1107%2FS0907444909047337&amp;volume=66&amp;pages=125-132&amp;publication_year=2010&amp;author=Kabsch%2CW">
                    Google Scholar</a> 
                </p></li><li data-counter="80."><p id="ref-CR80">McCoy, A. J. et al. Phaser crystallographic software. <i>J. Appl. Crystallogr.</i> <b>40</b>, 658–674 (2007).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1107/S0021889807021206" data-track-action="article reference" href="https://doi.org/10.1107%2FS0021889807021206" aria-label="Article reference 80" data-doi="10.1107/S0021889807021206">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://jvns.ca/articles/cas-redirect/1:CAS:528:DC%2BD2sXnslWqsLk%3D" aria-label="CAS reference 80">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 80" href="http://scholar.google.com/scholar_lookup?&amp;title=Phaser%20crystallographic%20software&amp;journal=J.%20Appl.%20Crystallogr.&amp;doi=10.1107%2FS0021889807021206&amp;volume=40&amp;pages=658-674&amp;publication_year=2007&amp;author=McCoy%2CAJ">
                    Google Scholar</a> 
                </p></li><li data-counter="81."><p id="ref-CR81">Kovalevskiy, O., Nicholls, R. A., Long, F., Carlon, A. &amp; Murshudov, G. N. Overview of refinement procedures within REFMAC5: utilizing data from different sources. <i>Acta Crystallogr D Struct. Biol.</i> <b>74</b>, 215–227 (2018).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1107/S2059798318000979" data-track-action="article reference" href="https://doi.org/10.1107%2FS2059798318000979" aria-label="Article reference 81" data-doi="10.1107/S2059798318000979">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://jvns.ca/articles/cas-redirect/1:CAS:528:DC%2BC1cXksFartbw%3D" aria-label="CAS reference 81">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 81" href="http://scholar.google.com/scholar_lookup?&amp;title=Overview%20of%20refinement%20procedures%20within%20REFMAC5%3A%20utilizing%20data%20from%20different%20sources&amp;journal=Acta%20Crystallogr%20D%20Struct.%20Biol.&amp;doi=10.1107%2FS2059798318000979&amp;volume=74&amp;pages=215-227&amp;publication_year=2018&amp;author=Kovalevskiy%2CO&amp;author=Nicholls%2CRA&amp;author=Long%2CF&amp;author=Carlon%2CA&amp;author=Murshudov%2CGN">
                    Google Scholar</a> 
                </p></li><li data-counter="82."><p id="ref-CR82">Terwilliger, T. C. et al. Iterative model building, structure refinement and density modification with the PHENIX AutoBuild wizard. <i>Acta Crystallogr. D Biol. Crystallogr.</i> <b>64</b>, 61–69 (2008).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1107/S090744490705024X" data-track-action="article reference" href="https://doi.org/10.1107%2FS090744490705024X" aria-label="Article reference 82" data-doi="10.1107/S090744490705024X">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://jvns.ca/articles/cas-redirect/1:CAS:528:DC%2BD2sXhsVGjt7nF" aria-label="CAS reference 82">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 82" href="http://scholar.google.com/scholar_lookup?&amp;title=Iterative%20model%20building%2C%20structure%20refinement%20and%20density%20modification%20with%20the%20PHENIX%20AutoBuild%20wizard&amp;journal=Acta%20Crystallogr.%20D%20Biol.%20Crystallogr.&amp;doi=10.1107%2FS090744490705024X&amp;volume=64&amp;pages=61-69&amp;publication_year=2008&amp;author=Terwilliger%2CTC">
                    Google Scholar</a> 
                </p></li><li data-counter="83."><p id="ref-CR83">Hoh, S. W., Burnley, T. &amp; Cowtan, K. Current approaches for automated model building into cryo-EM maps using Buccaneer with CCP-EM. <i>Acta Crystallogr D Struct. Biol.</i> <b>76</b>, 531–541 (2020).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1107/S2059798320005513" data-track-action="article reference" href="https://doi.org/10.1107%2FS2059798320005513" aria-label="Article reference 83" data-doi="10.1107/S2059798320005513">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://jvns.ca/articles/cas-redirect/1:CAS:528:DC%2BB3cXhtVOlurfM" aria-label="CAS reference 83">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 83" href="http://scholar.google.com/scholar_lookup?&amp;title=Current%20approaches%20for%20automated%20model%20building%20into%20cryo-EM%20maps%20using%20Buccaneer%20with%20CCP-EM&amp;journal=Acta%20Crystallogr%20D%20Struct.%20Biol.&amp;doi=10.1107%2FS2059798320005513&amp;volume=76&amp;pages=531-541&amp;publication_year=2020&amp;author=Hoh%2CSW&amp;author=Burnley%2CT&amp;author=Cowtan%2CK">
                    Google Scholar</a> 
                </p></li><li data-counter="84."><p id="ref-CR84">Emsley, P., Lohkamp, B., Scott, W. G. &amp; Cowtan, K. Features and development of Coot. <i>Acta Crystallogr. D Biol. Crystallogr.</i> <b>66</b>, 486–501 (2010).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1107/S0907444910007493" data-track-action="article reference" href="https://doi.org/10.1107%2FS0907444910007493" aria-label="Article reference 84" data-doi="10.1107/S0907444910007493">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://jvns.ca/articles/cas-redirect/1:CAS:528:DC%2BC3cXksFKisb8%3D" aria-label="CAS reference 84">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 84" href="http://scholar.google.com/scholar_lookup?&amp;title=Features%20and%20development%20of%20Coot&amp;journal=Acta%20Crystallogr.%20D%20Biol.%20Crystallogr.&amp;doi=10.1107%2FS0907444910007493&amp;volume=66&amp;pages=486-501&amp;publication_year=2010&amp;author=Emsley%2CP&amp;author=Lohkamp%2CB&amp;author=Scott%2CWG&amp;author=Cowtan%2CK">
                    Google Scholar</a> 
                </p></li><li data-counter="85."><p id="ref-CR85">Afonine, P. V. et al. Towards automated crystallographic structure refinement with phenix.refine. <i>Acta Crystallogr. D Biol. Crystallogr.</i> <b>68</b>, 352–367 (2012).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1107/S0907444912001308" data-track-action="article reference" href="https://doi.org/10.1107%2FS0907444912001308" aria-label="Article reference 85" data-doi="10.1107/S0907444912001308">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://jvns.ca/articles/cas-redirect/1:CAS:528:DC%2BC38Xlt1Ggu7c%3D" aria-label="CAS reference 85">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 85" href="http://scholar.google.com/scholar_lookup?&amp;title=Towards%20automated%20crystallographic%20structure%20refinement%20with%20phenix.refine&amp;journal=Acta%20Crystallogr.%20D%20Biol.%20Crystallogr.&amp;doi=10.1107%2FS0907444912001308&amp;volume=68&amp;pages=352-367&amp;publication_year=2012&amp;author=Afonine%2CPV">
                    Google Scholar</a> 
                </p></li><li data-counter="86."><p id="ref-CR86">Raffel, C. et al. Exploring the limits of transfer learning with a unified text-to-text transformer. Preprint at <i>arXiv</i> <a href="https://doi.org/10.48550/arXiv.1910.10683">https://doi.org/10.48550/arXiv.1910.10683</a> (2019).</p></li><li data-counter="87."><p id="ref-CR87">Studier, F. W. Protein production by auto-induction in high density shaking cultures. <i>Protein Expr. Purif.</i> <b>41</b>, 207–234 (2005).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.pep.2005.01.016" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.pep.2005.01.016" aria-label="Article reference 87" data-doi="10.1016/j.pep.2005.01.016">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://jvns.ca/articles/cas-redirect/1:CAS:528:DC%2BD2MXjsFCktLw%3D" aria-label="CAS reference 87">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 87" href="http://scholar.google.com/scholar_lookup?&amp;title=Protein%20production%20by%20auto-induction%20in%20high%20density%20shaking%20cultures&amp;journal=Protein%20Expr.%20Purif.&amp;doi=10.1016%2Fj.pep.2005.01.016&amp;volume=41&amp;pages=207-234&amp;publication_year=2005&amp;author=Studier%2CFW">
                    Google Scholar</a> 
                </p></li><li data-counter="88."><p id="ref-CR88">Mirdita, M. et al. ColabFold: making protein folding accessible to all. <i>Nat. Methods</i> <b>19</b>, 679–682 (2022).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/s41592-022-01488-1" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41592-022-01488-1" aria-label="Article reference 88" data-doi="10.1038/s41592-022-01488-1">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://jvns.ca/articles/cas-redirect/1:CAS:528:DC%2BB38XhsVWjsLfL" aria-label="CAS reference 88">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 88" href="http://scholar.google.com/scholar_lookup?&amp;title=ColabFold%3A%20making%20protein%20folding%20accessible%20to%20all&amp;journal=Nat.%20Methods&amp;doi=10.1038%2Fs41592-022-01488-1&amp;volume=19&amp;pages=679-682&amp;publication_year=2022&amp;author=Mirdita%2CM">
                    Google Scholar</a> 
                </p></li></ol><p><a data-track="click" data-track-action="download citation references" data-track-label="link" rel="nofollow" href="https://citation-needed.springer.com/v2/references/10.1038/s41587-022-01618-2?format=refman&amp;flavour=references">Download references<svg width="16" height="16" focusable="false" role="img" aria-hidden="true"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-download"></use></svg></a></p></div></div></div></section></div></div>
  </body>
</html>
