<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <title>James Routley | Feed</title>
    <link
      rel="stylesheet"
      type="text/css"
      href="../styles.css"
      media="screen"
    />
  </head>
  <body>
    <a href="/index.html">Back</a>
    <a href="https://news.ycombinator.com/item?id=44986809">Original</a>
    <h1>Launch HN: BlankBio (YC S25) – Making RNA Programmable</h1>
    
    <div id="readability-page-1" class="page"><div><td colspan="2"></td><td><div><p>Hey HN, we&#39;re Phil, Ian and Jonny, and we&#39;re building BlankBio (<a href="https://blank.bio">https://blank.bio</a>). We&#39;re training RNA foundation models to power a computational toolkit for therapeutics. The first application is in mRNA design where our vision is for any biologist to design an effective therapeutic sequence (<a href="https://www.youtube.com/watch?v=ZgI7WJ1SygI" rel="nofollow">https://www.youtube.com/watch?v=ZgI7WJ1SygI</a>).</p><p>BlankBio started from our PhD work in this area, which is open-sourced. There’s a model [2] and a benchmark with APIs access [0].</p><p>mRNA has the potential to encode vaccines, gene therapies, and cancer treatments. Yet designing effective mRNA remains a bottleneck. Today, scientists design mRNA by manually editing sequences AUGCGUAC... and testing the results through trial and error. It&#39;s like writing assembly code and managing individual memory addresses. The field is flooded with capital aimed at therapeutics companies: Strand ($153M), Orna ($221M), Sail Biomedicines ($440M) but the tooling to approach these problems remains low-level. That’s what we’re aiming to solve.</p><p>The big problem is that mRNA sequences are incomprehensible. They encode properties like half-life (how long RNA survives in cells) and translation efficiency (protein output), but we don&#39;t know how to optimize them. To get effective treatments, we need more precision. Scientists need sequences that target specific cell types to reduce dosage and side effects.</p><p>We envision a future where RNA designers operate at a higher level of abstraction. Imagine code like this:</p><pre><code>  seq = &#34;AUGCAUGCAUGC...&#34;
  seq = BB.half_life(seq, target=&#34;6 hours&#34;)
  seq = BB.cell_type(seq, target=&#34;hepatocytes&#34;)
  seq = BB.expression(seq, level=&#34;high&#34;)
</code></pre><p>
To get there we need generalizable RNA embeddings from pre-trained models. During our PhDs, Ian and I worked on self-supervised learning (SSL) objectives for RNA. This approach allows us to train on unlabeled data and has advantages: (1) we don&#39;t require noisy experimental data, and (2) the amount of unlabeled data is significantly greater than labeled. However the challenge is that standard NLP approaches don&#39;t work well on genomic sequences.</p><p>Using joint embedding architecture approaches (contrastive learning), we trained model to recognize functionally similar sequences rather than predict every nucleotide. This worked remarkably well. Our 10M parameter model, Orthrus, trained on 4 GPUs for 14 hours, beats Evo2, a 40B parameter model trained on 1000 GPUs for a month [0]. On mRNA half-life prediction, just by fitting a linear regression on our embeddings, we outperform supervised models. This work done during our academic days is the foundation for what we&#39;re building. We&#39;re improving training algorithms, growing the pre-training dataset, and making use of parameter scaling with the goal of designing effective mRNA therapeutics.</p><p>We have a lot to say about why other SSL approaches work better than next-token prediction and masked language modeling: some of which you can check out in Ian&#39;s blog post [1] and our paper [2]. The big takeaway is that the current approaches of applying NLP to scaling models for biological sequences won&#39;t get us all the way there. 90% of the genome can mutate without affecting fitness so training models to predict this noisy sequence results in suboptimal embeddings [3].</p><p>We think there are strong parallels between the digital and RNA revolutions. In the early days of computing, programmers wrote assembly code, managing registers and memory addresses directly. Today&#39;s RNA designers are manually tweaking sequences, improving stability or reduce immunogenicity through trial and error. As compilers freed programmers from low-level details, we&#39;re building the abstraction layer for RNA.</p><p>We currently have pilots with a few early stage biotechs proving out utility of our embeddings and our open source model is used by folks at Sanofi &amp; GSK. We&#39;re looking for: (1) partners working on RNA adjacent modalities (2) feedback from anyone who&#39;s tried to design RNA sequences what were your pain points?, and (3) Ideas for other applications! We chatted with some biomarker providing companies, and some preliminary analyses demonstrate improved stratification.</p><p>Thanks for reading. Happy to answer questions about the technical approach, why genomics is different from language, or anything else.</p><p>- Phil, Ian, and Jonny</p><p>founders@blankbio.com</p><p>[0] mRNABench: <a href="https://www.biorxiv.org/content/10.1101/2025.07.05.662870v1" rel="nofollow">https://www.biorxiv.org/content/10.1101/2025.07.05.662870v1</a></p><p>[1] Ian’s Blog on Scaling: <a href="https://quietflamingo.substack.com/p/scaling-is-dead-long-live-scaling" rel="nofollow">https://quietflamingo.substack.com/p/scaling-is-dead-long-li...</a></p><p>[2] Orthrus: <a href="https://www.biorxiv.org/content/10.1101/2024.10.10.617658v3" rel="nofollow">https://www.biorxiv.org/content/10.1101/2024.10.10.617658v3</a></p><p>[3] Zoonomia: <a href="https://www.science.org/doi/10.1126/science.abn3943" rel="nofollow">https://www.science.org/doi/10.1126/science.abn3943</a></p></div></td></div></div>
  </body>
</html>
