<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <title>James Routley | Feed</title>
    <link
      rel="stylesheet"
      type="text/css"
      href="../styles.css"
      media="screen"
    />
  </head>
  <body>
    <a href="/index.html">Back</a>
    <a href="https://www.nature.com/articles/s41557-022-00910-7">Original</a>
    <h1>Making the collective knowledge of chemistry open and machine actionable</h1>
    
    <div id="readability-page-1" class="page"><div>
            <section aria-labelledby="Abs1" data-title="Abstract" lang="en"><div id="Abs1-section"><h2 id="Abs1">Abstract</h2><div id="Abs1-content"><p>Large amounts of data are generated in chemistry labs—nearly all instruments record data in a digital form, yet a considerable proportion is also captured non-digitally and reported in ways non-accessible to both humans and their computational agents. Chemical research is still largely centred around paper-based lab notebooks, and the publication of data is often more an afterthought than an integral part of the process. Here we argue that a modular open-science platform for chemistry would be beneficial not only for data-mining studies but also, well beyond that, for the entire chemistry community. Much progress has been made over the past few years in developing technologies such as electronic lab notebooks that aim to address data-management concerns. This will help make chemical data reusable, however it is only one step. We highlight the importance of centring open-science initiatives around open, machine-actionable data and emphasize that most of the required technologies already exist—we only need to connect, polish and embrace them.</p>
                <figure><div data-test="figure" data-container-section="figure"><div><div><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41557-022-00910-7/MediaObjects/41557_2022_910_Figa_HTML.png?as=webp"/><img src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41557-022-00910-7/MediaObjects/41557_2022_910_Figa_HTML.png" alt="" loading="lazy" width="685" height="385"/></picture></div></div></div></figure>
              </div></div></section>
            

                
            
                <section data-title="Main"><div id="Sec1-section"><h2 id="Sec1">Main</h2><div id="Sec1-content"><p>In the era when scientific results were published only on real paper, the compression of information was of paramount importance. As a consequence of limited page counts, most scientific data were not published. Now, we live in a digital era and a large fraction of our data is captured in digital form. Yet, most scientific data that are collected are still not published<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 1" title="Heidorn, P. B. Shedding light on the dark data in the long tail of science. Libr. Trends 57, 280–299 (2008)." href="https://www.nature.com/articles/s41557-022-00910-7#ref-CR1" id="ref-link-section-d49829465e439">1</a></sup>, and the part that is often in a form that makes it difficult for other researchers to build on.</p><p>Scientists have also long been concerned about the reproducibility of results<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 2" title="Baker, M. 1,500 scientists lift the lid on reproducibility. Nature 533, 452–454 (2016)." href="https://www.nature.com/articles/s41557-022-00910-7#ref-CR2" id="ref-link-section-d49829465e446">2</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 3" title="Prinz, F., Schlange, T. &amp; Asadullah, K. Believe it or not: how much can we rely on published data on potential drug targets? Nat. Rev. Drug Discov. 10, 712–712 (2011)." href="https://www.nature.com/articles/s41557-022-00910-7#ref-CR3" id="ref-link-section-d49829465e449">3</a></sup>. This has lead most funding agencies to insist on a commitment by researchers as to how scientific data are managed (for instance in the form of a data management plan, that is, a clear outline of the types of data generated and used during a study, where and by whom they can be accessed, how and by whom they are protected and how and by whom they can be shared or published) and often to require all data to be made publicly available. Having a data management plan is important but, as we argue here, it does not guarantee that data will be shared in an easily findable, accessible, interoperable and reusable (FAIR) and ultimately machine-actionable, form<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 4" title="Wilkinson, M. D. et al. The FAIR guiding principles for scientific data management and stewardship. Sci. Data 3, 160018 (2016)." href="https://www.nature.com/articles/s41557-022-00910-7#ref-CR4" id="ref-link-section-d49829465e453">4</a></sup>.</p><p>Additionally, recent advances in machine learning illustrate very clearly why chemistry would benefit from embracing open and reusable data. In chemistry, we have many problems of irreducible complexity<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 5" title="Pietsch, W. &amp; Wernecke, J. in Berechenbarkeit der Welt? (eds Pietsch, W., Wernecke, J. Ott, M.) 37–57 (Springer, 2017)." href="https://www.nature.com/articles/s41557-022-00910-7#ref-CR5" id="ref-link-section-d49829465e460">5</a></sup>, such as the prediction of synthesizability, where complexity arises from the interaction of many diverse components (such as kinetics of side reactions or impurities) that are often not fully understood. Owing to these unknowns and complex interactions, some problems seem impossible to address with the current theory. Here, data-intensive research might be key. For example, many chemists would welcome a tool that recommends reaction conditions. One can envision building such a recommender system that harvests knowledge from all reactions that have been performed (including the ‘failed’ ones) to recommend conditions for the desired reaction. Building this tool, however, will only be possible if all the data are automatically collected in an interoperable and reusable form, such that machines can read datasets then rather autonomously discover the ones that are most relevant and in turn make decisions. This requires machines to not only parse the data but also understand the data and its context — that is, data must be machine actionable.</p><p>Our key thesis is that, if we want to advance chemistry with data-intensive research and also address reproducibility issues we need to change how experimental data are collected and reported. Structured data alone is not enough; open data alone is also not enough. We need to have both (thesis 1 in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41557-022-00910-7#Fig1">1</a>), together with additional tools such as semantic web technologies, that allow chemists and their computational agents to understand the meaning and intent of the data objects.</p><div data-test="figure" data-container-section="figure" id="figure-1" data-title="The five core theses of this perspective."><figure><figcaption><b id="Fig1" data-test="figure-caption-text">Fig. 1: The five core theses of this perspective.</b></figcaption><div><div><a data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="https://www.nature.com/articles/s41557-022-00910-7/figures/1" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41557-022-00910-7/MediaObjects/41557_2022_910_Fig1_HTML.png?as=webp"/><img aria-describedby="Fig1" src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41557-022-00910-7/MediaObjects/41557_2022_910_Fig1_HTML.png" alt="figure 1" loading="lazy" width="685" height="289"/></picture></a></div><p>Machine learning has fundamentally changed the way that data can be used in chemistry, which in turn requires a change in how it is reported. In addition, raw data are also needed to verify any conclusions presented in scientific publications—as stated with the ‘Nullius in verba’ principle (take nobody’s word for it)—as the results presented in a paper are always a compression of the original research record<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 6" title="Hunter, M. Establishing the New Science: the Experience of the Early Royal Society (Boydell Press, 1989)." href="https://www.nature.com/articles/s41557-022-00910-7#ref-CR6" id="ref-link-section-d49829465e482">6</a></sup>. Only a few groups creating and sharing FAIR data is not sufficient, it needs to be embraced by all chemists. Importantly, this can only happen if there are little or no overheads in publishing all the data in a FAIR, machine-actionable form. For this reason, the most crucial functionality an ELN can provide is to assist chemists in doing so; it is essential to avoid that chemical data become an afterthought in the publication process. Following this logic, developers of ELNs need to work together towards this goal of machine-actionable open science. We can only expect this to be widely adopted if ELNs implement a common standard for data representation and exchange, also with computational tools<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 69" title="Jablonka, K. M. et al. Connecting lab experiments with computer experiments: making ‘routine’ simulations routine. Preprint at ChemRxiv 
                  https://doi.org/10.26434/chemrxiv-2021-h3381-v2
                  
                 (2021)." href="https://www.nature.com/articles/s41557-022-00910-7#ref-CR69" id="ref-link-section-d49829465e486">69</a></sup>, and allow the integration of reusable plugins that can be used to create a custom data management infrastructure that is interoperable with other solutions. Clearly, there will not be one perfect solution that works for all subfields of chemistry. However, we can start by reusing the many existing parts, making them interoperable and ensuring the code is open source, and in this way create a practical solution that works today. This seems more effective than to aim for large-scale, all-encompassing and overcomplicated solutions. Importantly, the development of new data formats (alone) will also not lead us towards the goal of FAIR chemical data.</p></div></figure></div><p>To make this feasible, we envision a platform that seamlessly integrates the process of data collection, data processing and data publication with minimal overheads for the researcher:</p><ol>
                <li>
                  <span>(1)</span>
                  
                    <p>Data collection. A key component of chemistry research is the collection of chemical data (for example, reaction conditions and characterization data). Ideally, the raw<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Hunter, M. Establishing the New Science: the Experience of the Early Royal Society (Boydell Press, 1989)." href="#ref-CR6" id="ref-link-section-d49829465e512">6</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="McAlpine, J. B. et al. The value of universally available raw NMR data for transparency, reproducibility, and integrity in natural product research. Nat. Prod. Rep. 36, 35–107 (2019)." href="#ref-CR7" id="ref-link-section-d49829465e512_1">7</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 8" title="Helliwell, J. R., McMahon, B., Guss, J. M. &amp; Kroon-Batenburg, L. M. J. The science is in the data. IUCrJ 4, 714–722 (2017)." href="https://www.nature.com/articles/s41557-022-00910-7#ref-CR8" id="ref-link-section-d49829465e515">8</a></sup> (characterization) data are directly captured from the instrument, directly converted into a standard structured form<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 4" title="Wilkinson, M. D. et al. The FAIR guiding principles for scientific data management and stewardship. Sci. Data 3, 160018 (2016)." href="https://www.nature.com/articles/s41557-022-00910-7#ref-CR4" id="ref-link-section-d49829465e519">4</a></sup>, in which all the important metadata are systematically added and all the field names, such as ‘adsorption’ or ‘pressure’, are linked to an open vocabulary or ontology (which defines the meaning of the terms and their relation). One should not rely on individual chemists to manually perform such file transfer, annotation or conversion operations. This is not only time consuming and error prone, but also, more importantly, ensuring that all the data are in a form ready for FAIR sharing should not be an afterthought, it should the very first step.</p>
                  
                </li>
                <li>
                  <span>(2)</span>
                  
                    <p>Data processing and collaboration. Once we have converted our data into a standard form, we can apply the same analysis tools to all data types—which makes developments dramatically more efficient. Research groups that use different instruments could compare the data directly, and use the same analysis tools. Also, as soon as all the data are stored in a structured form, an electronic lab notebook (ELN) can make it searchable. For example, if an instrument was incorrectly calibrated, the ELN could allow the users to search for all the spectra that were measured with a specific instrument configuration at a specific range of time (or even automatically apply the correct calibration).</p>
                  
                </li>
                <li>
                  <span>(3)</span>
                  
                    <p>Data publishing. Data that remain locked in an ELN are not useful for the community. As soon as the researcher(s) are ready to publish a project, they could choose the relevant samples from the ELN and export them to a repository from where it can be used by machines, but also reimported by other ELNs.</p>
                  
                </li>
              </ol><p>From this viewpoint, the ELN is the central hub for all chemical research, from which analyses can be requested, analysed, shared, published and integrated with other platforms—and, also, a place to take notes. However, we emphasize that the most important functionality an ELN can provide is to automatically convert the data into an open, standardized and interoperable form (thesis 2 in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41557-022-00910-7#Fig1">1</a>). Only in this way can we leverage web technologies that allow computational tools to autonomously understand data and hence provide more meaningful (search) results (Box <a data-track="click" data-track-label="link" data-track-action="section anchor" href="https://www.nature.com/articles/s41557-022-00910-7#Sec2">1</a>). Note that this is quite different from the functionality most current ELNs offer. The majority of current ELNs only store data digitally as an attachment—they do not convert it into such a reusable form (thesis 2 in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41557-022-00910-7#Fig1">1</a>).</p><p>Over time, an ‘insane’<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 9" title="Kwok, R. How to pick an electronic laboratory notebook. Nature 560, 269–270 (2018)." href="https://www.nature.com/articles/s41557-022-00910-7#ref-CR9" id="ref-link-section-d49829465e563">9</a></sup> number of different ELNs and laboratory infrastructure management systems (LIMS) have been developed. Many of these different ELNs have been compared in previous works (for example, by the <a href="http://www.datamanagement.hms.harvard.edu/analyze/electronic-lab-notebook">Harvard Medical School</a>, the <a href="http://www.data.cam.ac.uk/data-management-guide/electronic-research-notebooks/electronic-research-notebook-products">Library of the University of Cambridge</a>, <a href="http://www.limswiki.org/">LIMSWiki</a> or peer-reviewed articles<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Kanza, S. et al. Electronic lab notebooks: can they replace paper? J. Cheminformatics 9, 31 (2017)." href="#ref-CR10" id="ref-link-section-d49829465e588">10</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Rubacha, M., Rattan, A. K. &amp; Hosselet, S. C. A review of electronic laboratory notebooks available in the market today. J. Lab. Autom. 16, 90–98 (2011)." href="#ref-CR11" id="ref-link-section-d49829465e588_1">11</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Guerrero, S. et al. Analysis and implementation of an electronic laboratory notebook in a biomedical research institute. PLoS ONE 11, e0160428 (2016)." href="#ref-CR12" id="ref-link-section-d49829465e588_2">12</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 13" title="Dirnagl, U. &amp; Przesdzing, I. A pocket guide to electronic laboratory notebooks in the academic life sciences. F1000Research 5, 2 (2016)." href="https://www.nature.com/articles/s41557-022-00910-7#ref-CR13" id="ref-link-section-d49829465e591">13</a></sup>). In this Perspective, we aim to focus on the ideas and design principles that we think are essential to create a successful open-science infrastructure—for the full lifetime of data from inception, creation and processing to publication. As the infrastructure we propose to embrace is already implemented in parts, we review examples (from Table <a data-track="click" data-track-label="link" data-track-action="table anchor" href="https://www.nature.com/articles/s41557-022-00910-7#Tab1">1</a>) that we think offer some key aspects of such an infrastructure to support open science. In a similar vein, we highlight examples in which chemical data have already been shared in a reusable form. Taking into account the many attempts to generate new data schema—describing the abstract structure of the data—and file formats for chemical data, we propose that a more efficient route to open science would be for the chemistry community to embrace and connect existing systems instead (thesis 4 in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41557-022-00910-7#Fig1">1</a>).</p><div data-test="inline-table" data-container-section="table" id="table-1"><figure><figcaption><b id="Tab1" data-test="table-caption">Table 1 Examples for some infrastructure management system ELN systems</b></figcaption></figure></div><div data-expandable-box-container="true"><div data-expandable-box="true" aria-hidden="true" id="box-Sec2"><h3 id="Sec2">Box 1 Machine-actionable data in chemistry</h3><div><p>Data structured in standardized ways can make information findable and interpretable by chemists and their machines and thus can enable humans, as well as their computational agents, to perform actions based on interpretation of the data.</p><p>If we perform web searches, major search engines display meaningful information (sometimes even formatted in infoboxes with tables that allow for easy comparison) and can show related content instead of just a list of hyperlinks. For instance, search engines will show, when queried for ‘old fashioned pancakes’, a compilation of recipes from different sites—similar to that shown in our example (see left panel in the figure). This is possible because the websites embed the information into the website in a standardized form using in-page mark-up, typically <a href="http://schema.org">Schema.org</a> (as in the code snippet on the right-hand side of our example). In summary, the recipe data are reported in a standard, open format, using linked vocabularies, described with metadata and accessible under URIs.</p><p>Similar mark-ups are used to encode COVID-19 announcements on some websites (including those from the US federal government<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 72" title="Kratsios, M., Kent, S. &amp; Rinat. O. Connecting Americans to coronavirus information online. Trump White House Archives 
                  https://trumpwhitehouse.archives.gov/articles/connecting-americans-coronavirus-information-online/
                  
                 (2020)." href="https://www.nature.com/articles/s41557-022-00910-7#ref-CR72" id="ref-link-section-d49829465e845">72</a></sup>), such as special opening hours or prevention measures; those can then be highlighted by search engines<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 73" title="COVID-19 Announcements Structured Data (Google Search Central, 2021); 
                  https://developers.google.com/search/docs/advanced/structured-data/special-announcements
                  
                " href="https://www.nature.com/articles/s41557-022-00910-7#ref-CR73" id="ref-link-section-d49829465e849">73</a></sup>. Readers can find such mark-ups by using the ‘inspect’ or ‘view page source’ tools of their browser (which can typically be accessed with a right-click on the page) and then searching for ‘schema.org’.</p><p>If similar metadata were embedded in, for example, all published spectra (such as from NMR, infrared, Raman and X-ray photoelectron spectroscopy), we could simply use a web search to find all the spectra published for a particular compound in a particular time period. With proper semantic annotation, we could, for instance, also specifically query for ‘vibrational spectroscopy’ to receive infrared, Raman and sum frequency generation spectra. Clearly, we can also envision the use of such standardized structured data for synthesis ‘recipes’. This might facilitate a comparison of different synthetic conditions and also incorporate the feedback of other chemists. The <a href="https://bioschemas.org/">Bioschemas</a><sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 70" title="Gray, A. J., Goble, C. A., Jimenez, R. et al. Bioschemas: from potato salad to protein annotation. In 16th International Semantic Web Conference (2017)." href="https://www.nature.com/articles/s41557-022-00910-7#ref-CR70" id="ref-link-section-d49829465e862">70</a></sup> and <a href="https://pages.nist.gov/material-schema/">Material Schemas</a> efforts attempt to move the life and materials sciences closer to this ideal.</p><p>Same concrete steps, and questions chemists can ask themselves to check their data objects for reusability and reliability<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 74" title="Fletcher, G., Groth, P. &amp; Sequeda, J. Knowledge scientists: unlocking the data-driven organization. Preprint at 
                  https://arxiv.org/abs/2004.07917
                  
                 (2020)." href="https://www.nature.com/articles/s41557-022-00910-7#ref-CR74" id="ref-link-section-d49829465e877">74</a></sup>, are the following.</p><ul>
                  <li>
                    <p>Data should be structured using standard, open conventions: can others (humans and machines) easily use my data objects with their tools? In practice, this means that an open format is always preferred over a proprietary one. Standard formats (JSON, XML, JCAMP-DX) ensure that others can use standard tools to read the data objects.</p>
                  </li>
                  <li>
                    <p>Entries in a data object should use a controlled vocabulary and ideally reference an ontology: can others (humans and machines) easily understand the meaning and format of all the fields in the data object? Ontologies explain the meaning and relation of the fields. For example, when reporting a bandgap, one needs to ensure that the field ‘bandgap’ can be correctly interpreted (as it might refer to the optical gap, fundamental gap or transport gap). A key challenge is that the documentation for the dataset is often transported ‘out of band’ if the data are, for example, described in the supplementary information of a paper, instead of directly ‘in band’ with the data object. <a href="https://json-ld.org/">JSON-LD</a><sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 75" title="Kellogg, G., Champin, P.-A. &amp; Longley, D. JSON-LD 1.1—A JSON-based Serialization for Linked Data. (W3C, 2020)." href="https://www.nature.com/articles/s41557-022-00910-7#ref-CR75" id="ref-link-section-d49829465e899">75</a></sup> (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41557-022-00910-7#Fig5">2</a>) and <a href="https://www.w3.org/2013/csvw/wiki/CSV-LD">CSV-LD</a><sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 76" title="Tennison, J. CSV on the Web: A Primer (W3C, 2016)." href="https://www.nature.com/articles/s41557-022-00910-7#ref-CR76" id="ref-link-section-d49829465e912">76</a></sup> are great ways of providing the context ‘in band’ with the data.</p>
                  </li>
                  <li>
                    <p>Data should be annotated with metadata, and ideally indicate the provenance of the data: do others (humans and machines) understand where the data came from and the context within which they were produced? This information can, for example, be important when issues with the data arise. For example, metadata might help us to find that the reason for all the reactions being unsuccessful is that a batch of the (commercial) starting material was impure or that the humidity or temperature in the room was too high. In chemistry, there is no widely used standard for recording basic metadata of ELN entries, even though proposals such as the elnItemManifest, which builds on the Dublin Core scheme, have been made<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 77" title="Coles, S. J., Frey, J. G., Bird, C. L., Whitby, R. J. &amp; Day, A. E. First steps towards semantic descriptions of electronic laboratory notebook records. J. Cheminformatics 5, 52 (2013)." href="https://www.nature.com/articles/s41557-022-00910-7#ref-CR77" id="ref-link-section-d49829465e922">77</a></sup>.</p>
                  </li>
                  <li>
                    <p>Data should also be uniquely identifiable, and citable, using a stable, and indexed, URI: can others (machines and humans) rely on finding the data in a stable form, see any change history, and do they know the usage conditions? If the aim is for data to be reused, it should be accompanied by a license that allows this (for example, a creative commons license such as a CC0, donation to the public domain or a CC-BY, which also requires attribution of the originator). Using a URL that points to a GitHub repository or personal web page is hereby not enough—the problem is that the content of such URLs can easily change, for example, by deleting a repository on GitHub (a phenomenon called link rot). For this reason, data should be shared via data repositories where it is assigned a stable identifier (such as a DOI) that is guaranteed to point to the content. Also, repositories will make sure that the metadata and identifier are indexed and hence can be found. For organic chemistry, a domain specific repository is the chemotion repository<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 37" title="Tremouilhac, P. et al. Chemotion repository, a curated repository for reaction information and analytical data. Chem. Methods 1, 8–11 (2020)." href="https://www.nature.com/articles/s41557-022-00910-7#ref-CR37" id="ref-link-section-d49829465e932">37</a></sup>. Also for identifiers (for example for samples, instruments) it is best to use hypertext URIs such that they can be easily looked up by others, humans and machines. Additionally, others should be able to find out the history of changes of the data and if they are still maintained. Most repositories can provide this functionality in the form of ‘versions’ of the dataset.</p>
                  </li>
                  <li>
                    <p>Data should be linked to other data: can others (humans and machines) easily find related data (for example, computational work that supports experimental measurements)? Linking data provides context and lets users discover related datasets. From our recipe example we can imagine that related content can give us useful information, for example, direct us to the recipe the original author was inspired by. In the chemistry context, we should link together, for instance, computational and experimental aspects of a study, or crystal structures deposited in different databases.</p>
                  </li>
                </ul><div data-test="illustration" id="i1"><div><picture><source type="image/webp" srcset="//media.springernature.com/relative-r300-703_m1050/springer-static/image/art%3A10.1038%2Fs41557-022-00910-7/MediaObjects/41557_2022_910_Figb_HTML.png?as=webp"/><img alt="" width="703" src="https://media.springernature.com/relative-r300-703_m1050/springer-static/image/art%3A10.1038%2Fs41557-022-00910-7/MediaObjects/41557_2022_910_Figb_HTML.png"/></picture></div></div></div></div></div></div></div></section><section data-title="Data capture, data processing and data publication"><div id="Sec3-section"><h2 id="Sec3">Data capture, data processing and data publication</h2><div id="Sec3-content"><p>To be practical, the data capture step needs to be both as close as possible to the way chemists work and it should ensure that the chemical data generated can be practically reused by other researchers. We give examples for what ‘machine-actionable data’ means in Box <a data-track="click" data-track-label="link" data-track-action="section anchor" href="https://www.nature.com/articles/s41557-022-00910-7#Sec2">1</a>.</p><p>In chemistry, most samples in the lab are produced with a chemical reaction. Trying to predict the conditions at which a reaction can take place optimally is still one of the major challenges in chemistry. Machine-learning methods are expected to help us in this area<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 14" title="Coley, C. W. in Artificial Intelligence in Drug Discovery (ed. Brown, N) 327–348 (Royal Society of Chemistry, 2020)." href="https://www.nature.com/articles/s41557-022-00910-7#ref-CR14" id="ref-link-section-d49829465e966">14</a></sup>. However, for this to work we need to report data in a format that can be used in machine learning, and also report ‘failed’ experiments<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 15" title="Raccuglia, P. et al. Machine-learning-assisted materials discovery using failed experiments. Nature 533, 73–76 (2016)." href="https://www.nature.com/articles/s41557-022-00910-7#ref-CR15" id="ref-link-section-d49829465e970">15</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 16" title="Moosavi, S. M. et al. Capturing chemical intuition in synthesis of metal–organic frameworks. Nat. Commun. 10, 539 (2019)." href="https://www.nature.com/articles/s41557-022-00910-7#ref-CR16" id="ref-link-section-d49829465e973">16</a></sup>. One can easily see the dilemma here; if an experiment—after 99 ‘failed’ attempts—finally works, there is little motivation, if any, for a researcher to spend 1% of their time in reporting the one successful experiment and the remaining 99% of the time on the ‘failed’ ones.</p><h3 id="Sec4">Capturing synthetic data</h3><p>In chemistry, the number of possible steps and combinations of steps is nearly infinite. For example, the order in which the reagents are added can clearly decide whether a reaction will be successful or not<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 17" title="Ojea-Jiménez, I., Bastús, N. G. &amp; Puntes, V. Influence of the sequence of the reagents addition in the citrate-mediated synthesis of gold nanoparticles. J. Phys. Chem. C 115, 15752–15757 (2011)." href="https://www.nature.com/articles/s41557-022-00910-7#ref-CR17" id="ref-link-section-d49829465e984">17</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 18" title="Huang, Y. et al. Importance of reagent addition order in contaminant degradation in an Fe(II)/PMS system. RSC Adv. 6, 70271–70276 (2016)." href="https://www.nature.com/articles/s41557-022-00910-7#ref-CR18" id="ref-link-section-d49829465e987">18</a></sup>—and any machine-learning efforts will fail if such information is not reported correctly. This is exactly what is missing in many of the existing databases. For example, by mining the patent literature<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 19" title="Lowe, D. M. Extraction of Chemical Structures and Reactions from the Literature. PhD thesis, Univ. Cambridge (2012)." href="https://www.nature.com/articles/s41557-022-00910-7#ref-CR19" id="ref-link-section-d49829465e991">19</a></sup> one can obtain a wealth of information on which chemicals can be synthesized<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 20" title="Jin, W., Coley, C. W., Barzilay, R. &amp; Jaakkola, T. Predicting organic reaction outcomes with Weisfeiler-Lehman network. In Proc. 31st International Conference on Neural Information Processing Systems 2604–2613 (NIPS, 2017)." href="https://www.nature.com/articles/s41557-022-00910-7#ref-CR20" id="ref-link-section-d49829465e995">20</a></sup>. However, the actual procedure of the syntheses cannot be mined systematically: the order of addition, the heating, the stirring and, of course, the workup and purification. And the situation is even more dire for inorganic chemistry<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 21" title="Kim, E., Huang, K., Kononova, O., Ceder, G. &amp; Olivetti, E. Distilling a materials synthesis ontology. Matter 1, 8–12 (2019)." href="https://www.nature.com/articles/s41557-022-00910-7#ref-CR21" id="ref-link-section-d49829465e999">21</a></sup>. Similarly, all the databases contain no information about the attempts that did not work and are biased towards certain reaction types<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Roughley, S. D. &amp; Jordan, A. M. The medicinal chemist’s toolbox: an analysis of reactions used in the pursuit of drug candidates. J. Med. Chem. 54, 3451–3479 (2011)." href="#ref-CR22" id="ref-link-section-d49829465e1003">22</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Schneider, N., Lowe, D. M., Sayle, R. A., Tarselli, M. A. &amp; Landrum, G. A. Big data from pharmaceutical patents: a computational analysis of medicinal chemists’ bread and butter. J. Med. Chem. 59, 4385–4402 (2016)." href="#ref-CR23" id="ref-link-section-d49829465e1003_1">23</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 24" title="Brown, D. G., Gagnon, M. M. &amp; Boström, J. Understanding our love affair with p-chlorophenyl: present day implications from historical biases of reagent selection. J. Med. Chem. 58, 2390–2405 (2015)." href="https://www.nature.com/articles/s41557-022-00910-7#ref-CR24" id="ref-link-section-d49829465e1006">24</a></sup>. This lack of reports on ‘failed’ reactions adds to other factors that lead to certain types of reactions being more prominent than others—for example, looking into the most used reactions in medicinal chemistry, Brown and Boström found that amide formation was mentioned at least once in about half of the selected set of manuscripts published in the <i>Journal of Medicinal Chemistry</i> in 2014 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 25" title="Brown, D. G. &amp; Boström, J. Analysis of past and present synthetic methodologies on medicinal chemistry: where have all the new reactions gone? J. Med. Chem. 59, 4443–4458 (2015)." href="https://www.nature.com/articles/s41557-022-00910-7#ref-CR25" id="ref-link-section-d49829465e1014">25</a></sup>).</p><p>Ideally, to capture synthesis information we need to find a balance between the flexibility of a sheet of paper, on which chemists can record anything they want in any format they like<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 26" title="L. Bird, C., Willoughby, C. &amp; G. Frey, J. Laboratory notebooks in the digital era: the role of ELNs in record keeping for chemistry and other sciences. Chem. Soc. Rev. 42, 8157–8175 (2013)." href="https://www.nature.com/articles/s41557-022-00910-7#ref-CR26" id="ref-link-section-d49829465e1021">26</a></sup>, and imposing a structure such that the captured data can be easily reused for machine-learning applications. The flexibility is key to ensure chemists will widely adopt the tool<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 10" title="Kanza, S. et al. Electronic lab notebooks: can they replace paper? J. Cheminformatics 9, 31 (2017)." href="https://www.nature.com/articles/s41557-022-00910-7#ref-CR10" id="ref-link-section-d49829465e1025">10</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 27" title="Oleksik, G., Milic-Frayling, N. &amp; Jones, R. Study of electronic lab notebook design and practices that emerged in a collaborative scientific environment. In CSCW’14 Proceedings of the 17th ACM Conference on Computer Supported Cooperative Work &amp; Social Computing (ACM Press, 2014)." href="https://www.nature.com/articles/s41557-022-00910-7#ref-CR27" id="ref-link-section-d49829465e1028">27</a></sup>, whereas from a data-management perspective a highly structured database (for example, filled via a long form) would be much easier to use. In high-throughput experimentation settings the latter might clearly be a natural approach, but for many manually created, small datasets<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 1" title="Heidorn, P. B. Shedding light on the dark data in the long tail of science. Libr. Trends 57, 280–299 (2008)." href="https://www.nature.com/articles/s41557-022-00910-7#ref-CR1" id="ref-link-section-d49829465e1032">1</a></sup>, this might not be a feasible approach, as to capture all the possible scenarios would result in such a gigantic form that chemists would need special training to navigate it.</p><p>Among the different ELNs no consensus has been reached about this design point. Some allow complete flexibility and have the look and feel of a typical note-taking app, whereby one needs natural-language processing to make the information machine-readable which, unavoidably, leads to information loss. At the other end of the spectrum are those that have a lot of structure with the design of a new form for every eventuality, which might be ideal for machine learning but poses a burden to use for non-routine chemistry.</p><p>A possible solution to these challenges, which is implemented in the chemotion and the cheminfo ELNs (Table <a data-track="click" data-track-label="link" data-track-action="table anchor" href="https://www.nature.com/articles/s41557-022-00910-7#Tab1">1</a>), is to stick to the text-based form chemists are used to, but to combine it with templates to structure the text. This hybrid approach is described in Box <a data-track="click" data-track-label="link" data-track-action="section anchor" href="https://www.nature.com/articles/s41557-022-00910-7#Sec5">2</a>. In practice, we found that some free text fields are always required to give chemists the necessary flexibility to express their motivation, thought process and interpretation). Parts of this can be captured via specific fields, for instance, the related literature, or spectral annotations. For many other parts, the free, potentially unstructured, thought process is exactly what one would like to capture (for example, to annotate when an experiment failed for an unexpected reason, such as a beam drop at the synchrotron).</p><div data-expandable-box-container="true"><div data-expandable-box="true" aria-hidden="true" id="box-Sec5"><h3 id="Sec5">Box 2 Capturing the reaction process</h3><div><p>A paper notebook (panel <b>a</b>) would typically read:</p><p>… we added 10 mg of chemical A (batch 4, see page 25), 5 mg of chemical B (batch 5, see page 61 of notebook 6 of colleague Y), 5 mg of chemical C (Chem-R-Us) in a 50% DMF/50% water mixture and put the solution in oven Y for 11 h at 70 °C.</p><p>It can envisioned that this is a simple step in a complex synthesis in which we are trying to find the optimal conditions for a particular reaction. The question is now how to convert such chemical data into a format that can be practically mined and possibly used for machine-learning studies and yet maintain a level of flexibility that is essential for chemists.</p><p>The idea of such a workflow is to find a compromise between being able to easily extract process variables (such as the heating time and temperature) and still provide the chemist with a natural interface of a text and structure editor such that the structure of the ELN remains similar to that of paper-based notebooks (panel <b>b</b>). In this scenario, research groups—or, ideally, consortia of research groups—can define predefined sentences (with fillable fields) for common operations, such as heating to reflux and filtering, that can be inserted with a shortcut such that the outcome is:</p><p>… we added <b>R1</b> (<i>x</i>R1 g), <b>R2</b> (<i>x</i>R2 g), <b>R3</b> (<i>x</i>R3 g), in a <i>y</i>%<b>R4</b>/(100 – <i>y</i>)%<b>R5</b> mixture and put the solution in <b>oven y</b> for <i>t</i> h at <i>T</i> °C ...</p><p>in which all the bold elements resolve to some URI. If, behind the scenes, the predefined sentences map to a well-defined set of concepts (in standard vocabularies), the description also becomes independent of the language it is written in.</p><p>The real advantages of this approach become clear if we look at the different shortcuts. Each reagent (which can be a previously produced sample or one from a manufacturer catalogue) can be referred to via the hyperlink. Following these links, the researcher has direct access to all the information about the provenance of the reactants, and from the order of the links one can extract the order of the synthesis procedure which is typically described sequentially. At the same time, this approach reduces the time needed to record experiments as most of the usual operations can be inserted with tab completion, and observations such as ‘the solution turned blue’ can be seamlessly integrated.</p><p>In this context it is important to realize that the ways in which observations are typically reported in chemistry are inadequate<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 71" title="Jablonka, K. M. et al. A data-driven perspective on the colours of metal–organic frameworks. Chem. Sci. 12, 3587–3598 (2021)." href="https://www.nature.com/articles/s41557-022-00910-7#ref-CR71" id="ref-link-section-d49829465e1125">71</a></sup>. For example, colours are usually reported as colour names (such as ‘dull blue’) in papers and databases, which are subject to perceptive spread and which therefore can limit the utility of such observations for replication studies or machine-learning approaches. In the case of colours, for example, we recommend images be recorded with colour calibration cards, from which a numerical colour value can be easily extracted. At the same time, the image also gives information about the morphology of the material.</p><p>Another promising approach is lab automation, as proposed by the company <a href="https://www.labforward.io/">labforward</a>, which, for example, allows us to connect balances, rotary evaporators or vacuum pumps to an ELN and, in this way, capture (automatically) more data in a structured and objective way<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 78" title="Lütjohann, D. S., Jung, N. &amp; Bräse, S. Open source life science automation: design of experiments and data acquisition via ‘dial-a-device’. Chemometr. Intell. Lab. Syst. 144, 100–107 (2015)." href="https://www.nature.com/articles/s41557-022-00910-7#ref-CR78" id="ref-link-section-d49829465e1139">78</a></sup>.</p><div data-test="illustration" id="i2"><div><picture><source type="image/webp" srcset="//media.springernature.com/relative-r300-703_m1050/springer-static/image/art%3A10.1038%2Fs41557-022-00910-7/MediaObjects/41557_2022_910_Figc_HTML.png?as=webp"/><img alt="" width="703" src="https://media.springernature.com/relative-r300-703_m1050/springer-static/image/art%3A10.1038%2Fs41557-022-00910-7/MediaObjects/41557_2022_910_Figc_HTML.png"/></picture></div></div></div></div></div><h3 id="Sec6">Characterization data formats and metadata</h3><p>After a sample has been synthesized, it needs to be characterized. Thereby, we want to ensure that researchers all over the world, as well as their computational agents, can use the data. Clearly, data models, which describe how data are stored in a data format, and metadata, which describe datasets, are not the typical focus of a chemist. However, a lot of chemical data is currently stored in a wide variety of proprietary files (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41557-022-00910-7#MOESM1">2)</a>. In the short term, this might not look like a real problem, but in the long term, this is not sustainable. For example, one can lose access to all the files once the software license associated with particular equipment expires; or collaborators in another institute that want to use the data do not have access to the same software. Also, a hodgepodge of inconsistent formats clearly hampers data mining efforts.</p><p>Requiring all individual researchers to manually convert all their spectra into a standard format will be a large, potentially insurmountable and non-scalable burden on the researchers. Therefore, an essential step in progressing towards such an open platform is to convert the data into a standardized structured form before it even enters the ELN (thesis 2 in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41557-022-00910-7#Fig1">1</a>). This is an essential service an ELN must provide to a user. That is, the ELN will take the data as they are provided by the spectrometer, and convert it into a standardized form. The cheminfo implementation, for example, uses JCAMP-DX files (Joint Committee on Atomic and Molecular Physical Data Exchange format; see Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41557-022-00910-7#Fig4">1</a> for an example) as a standard representation for most spectra. This format has been recommended by IUPAC (International Union of Pure and Applied Chemistry) for many spectra together with recommended vocabulary<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 28" title="McDonald, R. S. &amp; Wilks, P. A. Jcamp-dx: a standard form for exchange of infrared spectra in computer readable form. Appl. Spectrosc. 42, 151–162 (1988)." href="https://www.nature.com/articles/s41557-022-00910-7#ref-CR28" id="ref-link-section-d49829465e1171">28</a></sup>, and is also recommended by the chemotion ELN, and used in the Open Spectral Database<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 29" title="Chalk, S. J. The open spectral database: an open platform for sharing and searching spectral data. J. Cheminformatics 8, 55 (2016)." href="https://www.nature.com/articles/s41557-022-00910-7#ref-CR29" id="ref-link-section-d49829465e1175">29</a></sup>. However, in principle, any other format (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41557-022-00910-7#MOESM1">4)</a> can be used as long as it is standardized and openly documented. Indeed, some newer formats have a native support for advanced features, such as linking to standardized vocabularies, and might be preferable (see Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41557-022-00910-7#Fig5">2</a> for an example). For example, there were efforts (spearheaded by the pharmaceutical industry) to develop a ‘unified data model’ for compound synthesis and testing, or the ‘Allotrope data format’, which tries to collect the full data life cycle in one file. Some, like the autoprotocol or XDL<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 30" title="Mehr, S. H. M., Craven, M., Leonov, A. I., Keenan, G. &amp; Cronin, L. A universal system for digitization and automatic execution of the chemical synthesis literature. Science 370, 101–108 (2020)." href="https://www.nature.com/articles/s41557-022-00910-7#ref-CR30" id="ref-link-section-d49829465e1186">30</a></sup>, even try to capture the link between hardware (such as reaction vessels) and synthesis steps in a way that can be understood (and executed) by both robots and humans.</p><p>One can argue that some existing formats and data schema are old-fashioned and that we should develop new ones. However, anyone proposing a new format should realize that if a characterization method has <i>N</i> formats provided by the instrument manufacturers and <i>M</i> ‘standard’ formats are invented, we need to write and maintain <i>N</i> × <i>M</i> conversion programs and <i>M</i><sup>2</sup> programs to be able to compare the different ‘standard’ formats. This indicates that it can be more productive to update existing solutions and make them interoperable compared with creating new ones (thesis 5 in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41557-022-00910-7#Fig1">1</a>).</p><p>It is important to note that data become much more useful, and interoperable, if they are linked and described using a controlled, hierarchical vocabulary, that is, an ontology. Using a formal ontology allows us to infer information from the context encoded in the vocabulary. For example, we might have Raman and infrared spectra, as well as the cities of the measurement stored in our database. The ontology will not only remove ambiguities in spelling of the cities, but it will also tell us which cities to include if we search for, say, all organic samples with vibrational spectra measured in a particular country. At the technical level, this is enabled by the fact that the ontology will encode that both infrared and Raman spectroscopy are forms of vibrational spectroscopy and that cities are located in countries. That is, it allows us to go from machine-readable to machine interpretable on a global scale (global because the terms are standardized and shared via uniform resource identifiers (URIs)). In practice, however, ontologies (and related semantic web technologies) remain underused. The main reasons are probably that the diversity of the ontologies is too large and that existing ones are not well integrated<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 31" title="Directorate General for Research and Innovation (European Commission) Turning FAIR into Reality: Final Report and Action Plan from the European Commission Expert Group on FAIR Data (Publications Office, 2018)." href="https://www.nature.com/articles/s41557-022-00910-7#ref-CR31" id="ref-link-section-d49829465e1216">31</a></sup>. Clearly, we cannot expect chemists to manually annotate their data using an ontology. This is something an ELN needs to do automatically in the background. However, for this to be practical, ELN developers need to connect with other initiatives to register, standardize, link<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 32" title="Harrow, I. et al. Ontology mapping for semantically enabled applications. Drug Discov. Today 24, 2068–2075 (2019)." href="https://www.nature.com/articles/s41557-022-00910-7#ref-CR32" id="ref-link-section-d49829465e1220">32</a></sup> and adopt ontologies.</p><p>Let us now assume the ideal situation that most chemists have settled on a standard data reporting form (for the most important characterization techniques in a subdomain, such as gas adsorption isotherms, X-ray adsorption spectroscopy and cyclic voltammetry), and also accept that open science should not be an afterthought. This implies that the ELN must take the file in whatever form it comes from the instrument, convert into a standard form and permanently connect it to the chemical that was characterized (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41557-022-00910-7#Fig2">2</a>). Such conversion tools (see Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41557-022-00910-7#MOESM1">2</a> for examples) can be developed independently of each other and reused in all ELNs. For instance, the chemotion ELN reuses some of the libraries that we have been developing for the cheminfo ELN (<a href="http://cheminfo.github.io">cheminfo.github.io</a>). Having such common conversion tools would also create the incentive to adopt a common schema.</p><div data-test="figure" data-container-section="figure" id="figure-2" data-title="Overview of a possible importation procedure of the ELN."><figure><figcaption><b id="Fig2" data-test="figure-caption-text">Fig. 2: Overview of a possible importation procedure of the ELN.</b></figcaption><div><div><a data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="https://www.nature.com/articles/s41557-022-00910-7/figures/2" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41557-022-00910-7/MediaObjects/41557_2022_910_Fig2_HTML.png?as=webp"/><img aria-describedby="Fig2" src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41557-022-00910-7/MediaObjects/41557_2022_910_Fig2_HTML.png" alt="figure 2" loading="lazy" width="685" height="570"/></picture></a></div><p>If an instrument is coupled to the network one can, through scanning the barcode on the sample, upload the analysis result directly into a database. Alternatively, one can upload files via drag and drop through a web interface (front end). In both cases, the ELN ensures that the data are converted into a standard form such that anyone with a web browser can visualize and further analyse it. Other parties can access the data, for example, using an access token mechanism<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 70" title="Gray, A. J., Goble, C. A., Jimenez, R. et al. Bioschemas: from potato salad to protein annotation. In 16th International Semantic Web Conference (2017)." href="https://www.nature.com/articles/s41557-022-00910-7#ref-CR70" id="ref-link-section-d49829465e1253">70</a></sup>, via a representational state transfer (REST) application programming interface (API) or published on a repository. Importantly, all the steps can take place from a different location, and hence enable collaboration. This data infrastructure is implemented in the open-source cheminfo ELN. Folder icon reproduced from image designed using resources from <a href="https://www.flaticon.com/">Flaticon.com</a>; laptop photo by <a href="https://unsplash.com/@homajob">Scott Graham</a> on <a href="https://unsplash.com">Unsplash</a>.</p></div></figure></div><h3 id="Sec7">Provenance of data</h3><p>One crucial step in this process is to match the spectrum with the correct sample. A URI system (can be printed as barcodes) can help to avoid mistakes in this step. For instance, in the cheminfo ELN, scanning the barcode will create the upload information for automatic importation from the computers to which the spectrometers are connected. From there, the system can take the file from the computer, convert it into the standard form and store it as an attachment to a sample that has been created in the ELN (for example, as the product of some reaction). This automatic importation not only makes it much easier, and less error-prone, for the chemist to store the data in the ELN, but also it allows us to automatically record a lot of metadata—for example, the importation workflow can fill in information about the instrument (such as the manufacturer, serial number, humidity and temperature of the room) that is not always recorded in the output files of the measurements (see Extended Figs. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41557-022-00910-7#Fig4">1</a> and <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41557-022-00910-7#Fig5">2</a> for examples).</p><h3 id="Sec8">Data processing</h3><p>After data have been produced and imported into the ELN, they usually need to be further analysed. At present, chemists have to switch between different, often proprietary, software to carry out this analysis. They might rely on the software provided by the instrument manufacturer to perform peak picking or baseline correction, and then use another plotting tool to overlay the data. In an open-science vision, one would like to ensure that one can not only access data, but, equally important, can also reproduce the subsequent analyses. Likewise, if the chemistry community embraced the view that the ELN converted data into a commonly agreed standard form, the analysis tools become independent of a particular instrument or even characterization technique (Box <a data-track="click" data-track-label="link" data-track-action="section anchor" href="https://www.nature.com/articles/s41557-022-00910-7#Sec9">3</a>).</p><p>If we design the platform with a common interface, ensure a modular architecture and ensure a reusability of the key components, we have the first step towards an ecosystem in which libraries are developed for specific tools that accelerate the workflow of chemists (thesis 4 in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41557-022-00910-7#Fig1">1</a> and Box <a data-track="click" data-track-label="link" data-track-action="section anchor" href="https://www.nature.com/articles/s41557-022-00910-7#Sec9">3</a>). The modular nature would allow that experts in one technique, for example, NMR spectroscopy develop tools that can then be reused by other ELNs. An example for this is the <a href="https://www.nmrium.org/">NMRium</a> project<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 33" title="Davies, A. &amp; Patiny, L. NMRium browser-based nuclear magnetic resonance data processing. Spectrosc. Eur. 
                  https://doi.org/10.1255/sew.2021.a18
                  
                 (2021)." href="https://www.nature.com/articles/s41557-022-00910-7#ref-CR33" id="ref-link-section-d49829465e1327">33</a></sup>, which is a reusable web component that can, with three lines of code, be plugged into another ELN system. To make this work, it is important that the components can talk with each other via standardized protocols.</p><p>In an open-science vision, the code for these components should be open. One of the concerns regarding open-source software is the danger that a project might ‘die out’ if one maintainer leaves the project, whereas a successful commercial software might seem to have the promise of continuity. However, there are many successful examples (such as Linux and Python) of open-source projects are maintained by the community, yet leave many options for commercial initiatives (for instance, support contracts and maintenance of a custom installation). Similarly, at universities a common analytical infrastructure (such as the routine NMR service) is often supported using institutional funding—a similar model might also be appropriate for a digital infrastructure. Importantly, open-source code has the advantage that the underlying assumptions and equations for any analysis are documented and everyone can verify, replicate or even improve the analysis. Also, in contrast to closed-source (commercial) tools that are discontinued because of a change in business interests, the development can be reanimated at any time, as the code is openly accessible and reusable.</p><div data-expandable-box-container="true"><div data-expandable-box="true" aria-hidden="true" id="box-Sec9"><h3 id="Sec9">Box 3 Example of the online chemical processing of data</h3><div><p>A common operation in materials science and inorganic chemistry is to characterize a material with powder X-ray diffraction. One then typically compares the measured spectrum with some reference, which might be a predicted pattern, a single-crystal structure, an entry from a reference database or a pattern from the past, for example, with a pattern that has been measured by a student that left the group. In the worst case, the latter is completely lost or only findable as an image in some publication.</p><p>In the cheminfo ELN, the same interface can be used to compute an X-ray diffraction pattern based on any crystal structure in the database, overlay it with experimental patterns measured in the past in the research group or deposited in CoRE MOF (computation-ready, experimental metal−organic framework)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 79" title="Chung, Y. G. et al. Advances, updates, and analytics for the computation-ready, experimental metal–organic framework database: CoRE MOF 2019. J. Chem. Eng. Data 64, 5985–5998 (2019)." href="https://www.nature.com/articles/s41557-022-00910-7#ref-CR79" id="ref-link-section-d49829465e1345">79</a></sup> or the crystallographic open database<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 80" title="Gražulis, S. et al. Crystallography Open Database—an open-access collection of crystal structures. J. Appl. Crystallogr. 42, 726–729 (2009)." href="https://www.nature.com/articles/s41557-022-00910-7#ref-CR80" id="ref-link-section-d49829465e1349">80</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 81" title="Gražulis, S. et al. Crystallography Open Database (COD): an open-access collection of crystal structures and platform for world-wide collaboration. Nucleic Acids Res. 40, D420–D427 (2012)." href="https://www.nature.com/articles/s41557-022-00910-7#ref-CR81" id="ref-link-section-d49829465e1352">81</a></sup> (screenshot). A typical question in this context is whether a structure is a distorted analogue of a known structure. When our experimental partners approached us with this question, we extended the toolbox in the ELN to allow the calculation of X-ray diffraction patterns for distorted cells of reference crystal structures—we see this collaboration with experimentalists as a key for the success of an ELN platform. In a similar vein, one can link computational infrastructure to give experimentalists easy access to ‘routine’ simulations<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 70" title="Gray, A. J., Goble, C. A., Jimenez, R. et al. Bioschemas: from potato salad to protein annotation. In 16th International Semantic Web Conference (2017)." href="https://www.nature.com/articles/s41557-022-00910-7#ref-CR70" id="ref-link-section-d49829465e1356">70</a></sup>. Again, the tools are reusable by other researchers—in the form of the source code and a web service that exposes a REST API that can be queried from other systems, such as other ELNs. We envision that web services such as this can be an important part of a platform in which the chemical processing of data happens online. Indeed, different web services can be developed and maintained by research groups in their field of expertise (and in an appropriate programming language) and reused by the chemistry community on any platform with any programming language.</p><div data-test="illustration" id="i3"><div><picture><source type="image/webp" srcset="//media.springernature.com/relative-r300-703_m1050/springer-static/image/art%3A10.1038%2Fs41557-022-00910-7/MediaObjects/41557_2022_910_Figd_HTML.png?as=webp"/><img alt="" width="703" src="https://media.springernature.com/relative-r300-703_m1050/springer-static/image/art%3A10.1038%2Fs41557-022-00910-7/MediaObjects/41557_2022_910_Figd_HTML.png"/></picture></div></div></div></div></div><h3 id="Sec10">Publication of reusable and machine-actionable data</h3><p>The work of a scientist is not completed when all the materials are synthesized and characterized. An essential part of the scientific process is the dissemination of the results to make sure that others can build on top of one’s work. Typically, we are used to thinking of ‘others’ as other scientists in the same field. However, science is increasingly multidisciplinary, and hence non-specialists might also need to understand the data. Additionally, the move towards open science is a logical consequence of the notion that if the taxpayer paid for the research, the ownership of the research data should be the public at large, which can empower citizen (data) science<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 34" title="Bonney, R. et al. Next steps for citizen science. Science 343, 1436–1437 (2014)." href="https://www.nature.com/articles/s41557-022-00910-7#ref-CR34" id="ref-link-section-d49829465e1375">34</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 35" title="Nielsen, M. Reinventing Discovery: the New Era of Networked Science (Princeton Univ. Press, 2012)." href="https://www.nature.com/articles/s41557-022-00910-7#ref-CR35" id="ref-link-section-d49829465e1378">35</a></sup>. We have a glimpse of the power of the reuse of data with the discoveries of Don Swanson, an information scientist without formal training in medicine who analysed literature from the Medline database and found previously undiscovered knowledge, such as links between magnesium deficiency and migraine<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 35" title="Nielsen, M. Reinventing Discovery: the New Era of Networked Science (Princeton Univ. Press, 2012)." href="https://www.nature.com/articles/s41557-022-00910-7#ref-CR35" id="ref-link-section-d49829465e1382">35</a></sup>. Clearly, there is nothing fundamental about chemistry that prohibits us from leveraging such approaches to science.</p><p>Usually, however, in contrast to the publication of an article, the publication of all the scientific data on which the article is based is reduced to an afterthought. Most of us have still been educated with the idea that we need to be selective about which data to publish, instead of embracing the idea that all the scientific data we generate is an integral part of the science we do: data are typically only published to fulfil the requirements of journal policy or data management plan—without reuse in mind. This probably explains why many ELNs do not feature an option to export data to a repository.</p><p>In the open-science platform we propose, the publication of the scientific data is simply seen as one of the applications of the ELN. The users can select the samples which they want to publish and create an entry on a repository that contains all the relevant raw data (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41557-022-00910-7#Fig3">3</a>). The application ensures that data are reported in a form that can be easily reused by other researchers as well as by machines. For the chemists writing a publication, this means that they can provide a DOI (digital object identifier) to supplementary material and augment every figure with a link at which readers can interact with the raw data or download it for follow-up studies. Both the chemotion and cheminfo ELNs implement parts of this functionality. The cheminfo ELN exports data to the general-purpose Zenodo<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 36" title="European Organization For Nuclear Research &amp; OpenAIRE Zenodo 
                  https://www.zenodo.org/
                  
                 (2013)." href="https://www.nature.com/articles/s41557-022-00910-7#ref-CR36" id="ref-link-section-d49829465e1395">36</a></sup> repository whereas the chemotion ELN can export data to the chemotion repository<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 37" title="Tremouilhac, P. et al. Chemotion repository, a curated repository for reaction information and analytical data. Chem. Methods 1, 8–11 (2020)." href="https://www.nature.com/articles/s41557-022-00910-7#ref-CR37" id="ref-link-section-d49829465e1399">37</a></sup>, which focuses on chemical synthesis and characterization data).</p><div data-test="figure" data-container-section="figure" id="figure-3" data-title="Example of the flow of data from an ELN to an interactive visualization for the reader of a paper."><figure><figcaption><b id="Fig3" data-test="figure-caption-text">Fig. 3: Example of the flow of data from an ELN to an interactive visualization for the reader of a paper.</b></figcaption><div><div><a data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="https://www.nature.com/articles/s41557-022-00910-7/figures/3" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41557-022-00910-7/MediaObjects/41557_2022_910_Fig3_HTML.png?as=webp"/><img aria-describedby="Fig3" src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41557-022-00910-7/MediaObjects/41557_2022_910_Fig3_HTML.png" alt="figure 3" loading="lazy" width="685" height="174"/></picture></a></div><p>Once all the chemicals for which the synthesis and characterization data needs to be published are selected, the ELN compiles the data and uploads it to a repository (in this case Zenodo<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 36" title="European Organization For Nuclear Research &amp; OpenAIRE Zenodo 
                  https://www.zenodo.org/
                  
                 (2013)." href="https://www.nature.com/articles/s41557-022-00910-7#ref-CR36" id="ref-link-section-d49829465e1415">36</a></sup>). These data are not only machine-readable, but also can be accessed through a browser, and a human reader can also use the same visualization tools as the authors of the article<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 71" title="Jablonka, K. M. et al. A data-driven perspective on the colours of metal–organic frameworks. Chem. Sci. 12, 3587–3598 (2021)." href="https://www.nature.com/articles/s41557-022-00910-7#ref-CR71" id="ref-link-section-d49829465e1419">71</a></sup>. The implementation sketched in this figure is implemented in the open-source cheminfo ELN. Panel <b>b</b> screenshot reproduced from Zenodo under a Creative Commons license <a href="https://creativecommons.org/licenses/by/4.0/">CC BY 4.0</a>.</p></div></figure></div><p>In a similar vein, an ELN might also allow importing entries from a repository. This means that researchers might import the entire lab notebook used to produce the published results. Importantly, as the characterization data are also provided in the repository, researchers also have access to the original characterization data and might overlay them with their new results. To our knowledge, at the moment no ELN fully implements this automatic reimportation procedure.</p></div></div></section><section data-title="Discussion and outlook"><div id="Sec11-section"><h2 id="Sec11">Discussion and outlook</h2><div id="Sec11-content"><p>The open-science platform we propose in this Perspective provides a central hub for all the synthetic or analytical work of a chemist or materials scientist. Underpinning this platform are two common principles we feel are essential to make it truly open science, such that it can benefit data-intensive research and address reproducibility problems (thesis 1 in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41557-022-00910-7#Fig1">1</a>). First, FAIR data should be at the core; all data that enter the platform need to be converted into an open, structured and standardized form with the appropriate linked metadata—this is the main functionality that an ELN should provide (thesis 2). Second, open science also implies ensuring that other researchers can reproduce and build on the results. Therefore, the platform should be able to export the data in a form that is machine-readable and interpretable and that can easily be reused by other groups (thesis 3). In addition, in an open-science vision the tools used to analyse the data should be made available to anyone in the world who might be interested in reproducing the results or reinterpreting the data. This leads to the notion that such a platform is ideally developed as a modular open-source infrastructure in which the analysis code can be scrutinized, reused and improved by the community (thesis 4).</p><p>If such a platform becomes widely used and supported by the community, the possibilities are unlimited. The way we assess scientific work and credit scientific outputs has the potential to change. Trusted time stamps can provide unique proofs of discovery, going beyond the compressed and delayed priority claim that preprints can provide<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 38" title="Coudert, F.-X. The rise of preprints in chemistry. Nat. Chem. 12, 499–502 (2020)." href="https://www.nature.com/articles/s41557-022-00910-7#ref-CR38" id="ref-link-section-d49829465e1459">38</a></sup>, and peers can continuously provide feedback about the raw data, the analyses and the conclusions. An interesting form of making the full research record public, and hence open for feedback, has already been proposed in the context of open notebook science<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 39" title="Bradley, J.-C. Open notebook science using blogs and wikis. Nat. Prec. 
                  https://doi.org/10.1038/npre.2007.39.1
                  
                 (2007)." href="https://www.nature.com/articles/s41557-022-00910-7#ref-CR39" id="ref-link-section-d49829465e1463">39</a></sup>. If this information is shared with the community, one can build a community-driven version of the <i>Organic Syntheses</i> journal in which the verification of the results is done continuously by the community and not (only) in a lab of one of the members of the editorial board. Importantly, this version would also contain information about the attempts that did not work and in this way document the process, and the learnings, that led to the final result. If data are available in digital form, the peer-review process can be supported with automated checks, for example, to verify the consistency of NMR assignments, and so highlight potential issues for peer reviewers.</p><p>The most important reason for embracing the approach described in this Perspective is that it can change the way we do chemistry. Many of us were educated before the digital era, with the idea that if we publish all the data that we generate, any human being will become lost in the sheer volume of data. Data-intensive science, however, fundamentally changed this point of view. With machine learning, we have the tools to analyse orders of magnitude more data than human being can process, discover correlations in millions of data points and build predictive models<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 40" title="Jablonka, K. M., Ongari, D., Moosavi, S. M. &amp; Smit, B. Big-data science in porous materials: materials genomics and machine learning. Chem. Rev. 120, 8066–8129 (2020)." href="https://www.nature.com/articles/s41557-022-00910-7#ref-CR40" id="ref-link-section-d49829465e1473">40</a></sup>. For example, if we aim to synthesize a compound, a simple query in the collective ELN database might show that for one synthesis route there are 100 ‘failed’ reactions and two successful ones, whereas another route shows 90 successful and ten ‘failed’ attempts—which clearly indicates which synthesis route should be tried first. Undoubtedly, a very experienced chemist might have very good intuitions about what works and what does not. However, for a new student in the field, this collective knowledge now becomes accessible. Clearly, we can go beyond this simple search and try to harvest the collective knowledge generated by all chemists, using machine-learning techniques to capture subtle correlations across the chemical space of the millions of reactions that have been carried out in the world. In this respect, machine learning is not different from the experienced chemist; most probably, it can learn even more from ‘failed’ and partially successful experiments as from the successful ones. However, in contrast with the chemist, it typically needs large amounts of structured data—which we could easily generate in chemistry.</p><p>Another issue that the chemistry community faces with open data is that everyone agrees that there are benefits in making data reusable and in reporting ‘failed’ experiments, but often there is hesitation from individual researchers to adopt this behaviour until all members of the community do so. The social sciences give us a range of possible solutions to this problem setting<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 35" title="Nielsen, M. Reinventing Discovery: the New Era of Networked Science (Princeton Univ. Press, 2012)." href="https://www.nature.com/articles/s41557-022-00910-7#ref-CR35" id="ref-link-section-d49829465e1480">35</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 41" title="Olson, M. The Logic of Collective Action; Public Goods and the Theory of Groups (Schocken Books, 1971)." href="https://www.nature.com/articles/s41557-022-00910-7#ref-CR41" id="ref-link-section-d49829465e1483">41</a></sup>. One approach is some kind of compulsion. For example, the fact that the submission of DNA sequences is a condition for publication in the leading scientific journals of the field is seen as one of the reasons for the success of the GenBank database<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 42" title="Strasser, B. GENETICS: genbank—natural history in the 21st century? Science 322, 537–538 (2008)." href="https://www.nature.com/articles/s41557-022-00910-7#ref-CR42" id="ref-link-section-d49829465e1487">42</a></sup>. This, in turn, opened many doors for bioinformatics research. We also witnessed that for small groups, which include leaders of the field, agreements such as the ‘Bermuda Principles’, which require that DNA sequence data are automatically released in publicly accessible databases directly after the measurement, can be achieved. In chemistry, we have observed similar dynamics in crystallography, in which crystallographic information files must be deposited with the Cambridge Structural Database, where they are made freely accessible (and searchable) on publication. This led the European Commission to conclude that “the requirement from academic journals that authors provide data in support to their papers has proven to be potentially culture-changing, as has been the case in crystallography”<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 31" title="Directorate General for Research and Innovation (European Commission) Turning FAIR into Reality: Final Report and Action Plan from the European Commission Expert Group on FAIR Data (Publications Office, 2018)." href="https://www.nature.com/articles/s41557-022-00910-7#ref-CR31" id="ref-link-section-d49829465e1491">31</a></sup>. What we can also learn from crystallography is that once some standards are adopted, automatic checks (such as <a href="https://checkcif.iucr.org/">checkCIF</a>) can be implemented.</p><p>From the Structural Genomics Consortium and related initiatives (for example, Open Source Malaria<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 43" title="Williamson, A. E. et al. Open source drug discovery: highly potent antimalarial compounds derived from the Tres Cantos arylpyrroles. ACS Centr. Sci. 2, 687–701 (2016)." href="https://www.nature.com/articles/s41557-022-00910-7#ref-CR43" id="ref-link-section-d49829465e1506">43</a></sup> and COVID Moonshot<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 44" title="Chodera, J., Lee, A. A., London, N. &amp; von Delft, F. Crowdsourcing drug discovery for pandemics. Nat. Chem. 12, 581–581 (2020)." href="https://www.nature.com/articles/s41557-022-00910-7#ref-CR44" id="ref-link-section-d49829465e1510">44</a></sup>) we can learn that openness can also be enforced at the level of a consortium, for example, by requesting that members openly publish the protein structures and not file patents for the research outputs. This public–private partnership model seems to be successful because the private sector, which provides the funding and ‘chemical probes’ (potent inhibitors of protein function), can guide the research—that is, prioritize structures that should be solved—without disclosing the companies research and development priorities as the consortium anonymizes the ‘wish lists’<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 45" title="Perkmann, M. &amp; Schildt, H. Open data partnerships between firms and universities: the role of boundary organizations. Res. Policy 44, 1133–1143 (2015)." href="https://www.nature.com/articles/s41557-022-00910-7#ref-CR45" id="ref-link-section-d49829465e1514">45</a></sup>. The utility of such a consortium can best be seen at the precompetitive stage (that is, the early stages of drug discovery) during which it can share risks, enhance collective learning and avoid duplication in new areas of (basic) science<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 46" title="Jones, M. M. &amp; Chataway, J. The structural genomics consortium: successful organisational technology experiment or new institutional infrastructure for health research? Technol. Anal. Strategic Manage. 33, 296–306 (2021)." href="https://www.nature.com/articles/s41557-022-00910-7#ref-CR46" id="ref-link-section-d49829465e1518">46</a></sup>. This is particularly interesting in the case of ‘chemical probes’, which are best produced by experienced industrial medicinal chemists. However, industry would profit enormously if academia could use such probes to validate drug targets<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 47" title="Edwards, A. M., Bountra, C., Kerr, D. J. &amp; Willson, T. M. Open access chemical and clinical probes to support drug discovery. Nat. Chem. Biol. 5, 436–440 (2009)." href="https://www.nature.com/articles/s41557-022-00910-7#ref-CR47" id="ref-link-section-d49829465e1522">47</a></sup>. For this reason, the Structural Genomics Consortium makes them available as ‘open access’ reagents—under the conditions that the research outputs are made available in the public domain. A similar ‘physical open access’ approach is pursued by the <a href="https://compound-platform.eu/home">Molecule Archive of the Compound Platform</a> at the Karlsruhe Institute of Technology, which acts as a mediator for compound exchange: synthetic chemists can ‘archive’ their compounds (which increases their visibility), which can then be requested for biological screenings<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 48" title="Jung, N., Deckers, A. &amp; Bräse, S. Ein molekülarchiv als akademisch integrierte service-einrichtung. Biospektrum 23, 212–214 (2017)." href="https://www.nature.com/articles/s41557-022-00910-7#ref-CR48" id="ref-link-section-d49829465e1534">48</a></sup>.</p><p>Beyond these measures, we need to change incentive structures by creating better ways to give researchers credit for curating data. ELNs could help in this regard by storing the ‘credit’ chain when data are imported and automatically append the citation when datasets are prepared for publication.</p><p>Beyond that, the adaption of this data-centric approach to chemistry requires changes in the curriculum at universities to raise the awareness of such new developments, as well as the need for, and the promises of, data curation. Ideally, open-science solutions, such as the infrastructure we describe here, should already be introduced in the undergraduate curriculum. Students can record the results of their labs in ELNs, harvest the data in machine-learning classes, predict the infrared spectrum they just measured in computational chemistry classes<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 49" title="Jablonka, K. M., Patiny, L. &amp; Smit, B. Making molecules vibrate: Interactive web environment for the teaching of infrared spectroscopy. J. Chem. Educ. 
                  https://doi.org/10.1021/acs.jchemed.1c01101
                  
                 (2022)." href="https://www.nature.com/articles/s41557-022-00910-7#ref-CR49" id="ref-link-section-d49829465e1544">49</a></sup> and use open notebooks to comment on and improve each other’s work. Towards this goal, we define commonly used technical terms in a glossary in the <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41557-022-00910-7#MOESM1">Supplementary Information</a>.</p><p>The question that might still be open at this point is how realistic the widespread adoption of such an open-data platform across the chemistry community is. We argue that we have all the basic tools and technology in place. For many of the key design aspects, here we use examples from our own work, which is openly available, can be tried out by the community and can be reused in other implementations. There are also several initiatives (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41557-022-00910-7#MOESM1">3)</a> that work on some of the aspects we emphasize in this Perspective. One example is the German NFDI4Chem consortium<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 50" title="Herres-Pawlis, S., Koepler, O. &amp; Steinbeck, C. NFDI4chem: shaping a digital and cultural change in chemistry. Angew. Chem. Int. Ed. 58, 10766–10768 (2019)." href="https://www.nature.com/articles/s41557-022-00910-7#ref-CR50" id="ref-link-section-d49829465e1557">50</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 51" title="Steinbeck, C. et al. NFDI4chem—towards a national research data infrastructure for chemistry in Germany. Res. Ideas Outcomes 6, e55852 (2020)." href="https://www.nature.com/articles/s41557-022-00910-7#ref-CR51" id="ref-link-section-d49829465e1560">51</a></sup>, which is embedded in the larger German initiative for the creation of National Research Data Management Infrastructures (which also includes NFDI4Cat<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 52" title="Wulf, C. et al. A unified research data infrastructure for catalysis research—challenges and concepts. ChemCatChem 13, 3223–3236 (2021)." href="https://www.nature.com/articles/s41557-022-00910-7#ref-CR52" id="ref-link-section-d49829465e1564">52</a></sup> for catalysis research and NFDI4Ing for the engineering sciences), and aims to ‘FAIRify’ the full data life cycle in chemistry. However, we, as a community, also have to realize that we are in a phase in which there are an insane number of initiatives, proposed data schema and ELNs. The task we as a community face is to embrace and connect the efforts. Only if we succeed in making these tools interoperable we will be able to leverage the full potential of data and the digital age. One promising way forward is the formation of data communities<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 53" title="Cooper, D. &amp; Springer, R. Data Communities: A New Model for Supporting STEM Data Sharing Technical Report (Univ. Nebraska-Lincoln, 2019)." href="https://www.nature.com/articles/s41557-022-00910-7#ref-CR53" id="ref-link-section-d49829465e1568">53</a></sup>, in which experimentalists and ELN developers work together to develop a domain-specific (for example, porous materials or batteries) open-science infrastructure by combining, extending and polishing the existing building blocks.</p><p>From our perspective, there are a concrete few steps that need to be implemented to reach this goal:</p><ul>
                <li>
                  <p>The chemistry community should embrace their own existing standards and solutions. We will only be able to make progress as a community if we start to connect and use existing solutions. The feedback can then be used to improve the tools. If we as community do not move beyond the stage of just proposing new formats or implementations—instead of using them in practice—we will not make any progress. Clearly, this also requires that the existing tools are made reusable (that is, packages are extracted from monolithic code bases and augmented with documentation) and shared on platforms such as GitHub.</p>
                </li>
                <li>
                  <p>Where community standards exist, journals need to make the deposition of reusable raw data mandatory. This is motivated by the success of the Bermuda agreement and the deposition of crystallographic information files, and is needed to address the collective-action problem. Just using ELNs does not solve the problem. We also need to open our ELNs. Notably, this does not mean that data should be provided as PDFs, but in a standard machine-actionable form. Where community standards exist or are emerging, for example, as is happening in the field of gas adsorption<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 54" title="Evans, J. D., Bon, V., Senkovska, I. &amp; Kaskel, S. A universal standard archive file for adsorption data. Langmuir 37, 4222–4226 (2021)." href="https://www.nature.com/articles/s41557-022-00910-7#ref-CR54" id="ref-link-section-d49829465e1587">54</a></sup>, journals should start to embrace such formats by requesting the deposition in a community repository<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 53" title="Cooper, D. &amp; Springer, R. Data Communities: A New Model for Supporting STEM Data Sharing Technical Report (Univ. Nebraska-Lincoln, 2019)." href="https://www.nature.com/articles/s41557-022-00910-7#ref-CR53" id="ref-link-section-d49829465e1591">53</a></sup>. The same holds for the basic characterization of organic compounds (NMR, infrared and mass spectroscopy), for which the chemotion repository already offers tools and curation that are reminiscent of the Cambridge Structural Database. Importantly, often disconnected pieces of data in different repositories can only practically be used if they are linked. Therefore, for instance, the gas adsorption data in one community repository (such as the NIST/ARPA-E database of Novel and Emerging Adsorbent Materials<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 55" title="Siderius, D. NIST/ARPA-E Database of Novel and Emerging Adsorbent Materials (NIST, accessed 29 June 2020); 
                  https://doi.org/10.18434/T43882
                  
                " href="https://www.nature.com/articles/s41557-022-00910-7#ref-CR55" id="ref-link-section-d49829465e1595">55</a></sup>) needs to be linked, ideally using hyperlinks, to the crystal structure in the Cambridge Structural Database<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 56" title="Ongari, D., Talirz, L., Jablonka, K. M., Siderius, D. W. &amp; Smit, B. Data-driven matching of experimental crystal structures and gas adsorption isotherms of Metal–Organic frameworks. J. Chem. Eng. Data 
                  https://doi.org/10.1021/acs.jced.1c00958
                  
                 (2022)." href="https://www.nature.com/articles/s41557-022-00910-7#ref-CR56" id="ref-link-section-d49829465e1599">56</a></sup>.</p>
                </li>
                <li>
                  <p>We need to embrace the publication of ‘failed’ experiments. With a digital infrastructure this can be easily done to tell the story of how the final result was reached. It also requires that we as a community realize that the outcome of an experiment is not a binary ‘is this a breakthrough or not’, but simply an observation that is valuable and can be reported. For this to be successful we must take care to properly acknowledge such datasets, for example, when we use them for data-mining exercises or they helped us to avoid some costly experiments.</p>
                </li>
                <li>
                  <p>ELNs that do not allow the export of all data into an open machine-actionable form should be avoided. This reflects the core of thesis 2: the most important service an ELN can provide is to remove the hassle from making data FAIR. This is not only to avoid losing access to the data if a licence expires or being unable to build on previous work as it was in the ‘old ELN’ format, but also it is about being able to collaborate and share data with groups independent of the ELN. ELNs that just store data as provided, and might not even allow the export of this data, do not bring us closer to the goal of reusable data in chemistry.</p>
                </li>
                <li>
                  <p>Data-intensive research must enter our curricula. ‘Open science’ is gaining momentum in the chemistry community and increasing numbers of researchers are engaging with it (to various extents). We need to raise the awareness of these new developments at the undergraduate level, use ELNs for our lab courses and teach that open science is just science done properly<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 57" title="Watson, M. When will ‘open science’ become simply ‘science’? Genome Biol. 16, 101 (2015)." href="https://www.nature.com/articles/s41557-022-00910-7#ref-CR57" id="ref-link-section-d49829465e1621">57</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 58" title="Tennant, J. Open science: Just science done right? 
                  https://figshare.com/articles/Open_Science_Just_science_done_right_/9759353/1
                  
                 (2019)." href="https://www.nature.com/articles/s41557-022-00910-7#ref-CR58" id="ref-link-section-d49829465e1624">58</a></sup>. For example, at the École Polytechnique Fédérale de Lausanne, we teach machine learning and the use of ELNs in the same course, and plan to couple the lab courses with data analysis exercises in the ELN. This also implies that our institutions need to provide faculties with appropriate support, for instance, via the campus library<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 59" title="Long, M. &amp; Schonfeld, R. Supporting the Changing Research Practices of Chemists Technical Report (Ithaca, 2013)." href="https://www.nature.com/articles/s41557-022-00910-7#ref-CR59" id="ref-link-section-d49829465e1628">59</a></sup>.</p>
                </li>
              </ul><p>To conclude, we emphasize that the technology is here not only to facilitate the process of publishing data in a FAIR format to satisfy the sponsors, but also to ensure that the combination of chemical data, FAIR principles and openness gives scientists the possibility to harvest all data so that all chemists can have access to the collective knowledge of everybody’s successful, partly successful and even ‘failed’ experiments.</p></div></div></section>
            

            <div id="MagazineFulltextArticleBodySuffix"><section aria-labelledby="Bib1" data-title="References"><div id="Bib1-section"><h2 id="Bib1">References</h2><div id="Bib1-content"><div data-container-section="references"><ol><li data-counter="1."><p id="ref-CR1">Heidorn, P. B. Shedding light on the dark data in the long tail of science. <i>Libr. Trends</i> <b>57</b>, 280–299 (2008).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 1" href="http://scholar.google.com/scholar_lookup?&amp;title=Shedding%20light%20on%20the%20dark%20data%20in%20the%20long%20tail%20of%20science&amp;journal=Libr.%20Trends&amp;volume=57&amp;pages=280-299&amp;publication_year=2008&amp;author=Heidorn%2CPB">
                    Google Scholar</a> 
                </p></li><li data-counter="2."><p id="ref-CR2">Baker, M. 1,500 scientists lift the lid on reproducibility. <i>Nature</i> <b>533</b>, 452–454 (2016).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28Xoslyrt70%3D" aria-label="CAS reference 2">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27225100" aria-label="PubMed reference 2">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 2" href="http://scholar.google.com/scholar_lookup?&amp;title=1%2C500%20scientists%20lift%20the%20lid%20on%20reproducibility&amp;journal=Nature&amp;volume=533&amp;pages=452-454&amp;publication_year=2016&amp;author=Baker%2CM">
                    Google Scholar</a> 
                </p></li><li data-counter="3."><p id="ref-CR3">Prinz, F., Schlange, T. &amp; Asadullah, K. Believe it or not: how much can we rely on published data on potential drug targets? <i>Nat. Rev. Drug Discov.</i> <b>10</b>, 712–712 (2011).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXhtFegtbjN" aria-label="CAS reference 3">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21892149" aria-label="PubMed reference 3">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 3" href="http://scholar.google.com/scholar_lookup?&amp;title=Believe%20it%20or%20not%3A%20how%20much%20can%20we%20rely%20on%20published%20data%20on%20potential%20drug%20targets%3F&amp;journal=Nat.%20Rev.%20Drug%20Discov.&amp;volume=10&amp;pages=712-712&amp;publication_year=2011&amp;author=Prinz%2CF&amp;author=Schlange%2CT&amp;author=Asadullah%2CK">
                    Google Scholar</a> 
                </p></li><li data-counter="4."><p id="ref-CR4">Wilkinson, M. D. et al. The FAIR guiding principles for scientific data management and stewardship. <i>Sci. Data</i> <b>3</b>, 160018 (2016).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26978244" aria-label="PubMed reference 4">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4792175" aria-label="PubMed Central reference 4">PubMed Central</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 4" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20FAIR%20guiding%20principles%20for%20scientific%20data%20management%20and%20stewardship&amp;journal=Sci.%20Data&amp;volume=3&amp;publication_year=2016&amp;author=Wilkinson%2CMD">
                    Google Scholar</a> 
                </p></li><li data-counter="5."><p id="ref-CR5">Pietsch, W. &amp; Wernecke, J. in <i>Berechenbarkeit der Welt?</i> (eds Pietsch, W., Wernecke, J. Ott, M.) 37–57 (Springer, 2017).</p></li><li data-counter="6."><p id="ref-CR6">Hunter, M. <i>Establishing the New Science: the Experience of the Early Royal Society</i> (Boydell Press, 1989).</p></li><li data-counter="7."><p id="ref-CR7">McAlpine, J. B. et al. The value of universally available raw NMR data for transparency, reproducibility, and integrity in natural product research. <i>Nat. Prod. Rep.</i> <b>36</b>, 35–107 (2019).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtlamsrzP" aria-label="CAS reference 7">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30003207" aria-label="PubMed reference 7">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 7" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20value%20of%20universally%20available%20raw%20NMR%20data%20for%20transparency%2C%20reproducibility%2C%20and%20integrity%20in%20natural%20product%20research&amp;journal=Nat.%20Prod.%20Rep.&amp;volume=36&amp;pages=35-107&amp;publication_year=2019&amp;author=McAlpine%2CJB">
                    Google Scholar</a> 
                </p></li><li data-counter="8."><p id="ref-CR8">Helliwell, J. R., McMahon, B., Guss, J. M. &amp; Kroon-Batenburg, L. M. J. The science is in the data. <i>IUCrJ</i> <b>4</b>, 714–722 (2017).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhslKrurvO" aria-label="CAS reference 8">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29123672" aria-label="PubMed reference 8">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5668855" aria-label="PubMed Central reference 8">PubMed Central</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 8" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20science%20is%20in%20the%20data&amp;journal=IUCrJ&amp;volume=4&amp;pages=714-722&amp;publication_year=2017&amp;author=Helliwell%2CJR&amp;author=McMahon%2CB&amp;author=Guss%2CJM&amp;author=Kroon-Batenburg%2CLMJ">
                    Google Scholar</a> 
                </p></li><li data-counter="9."><p id="ref-CR9">Kwok, R. How to pick an electronic laboratory notebook. <i>Nature</i> <b>560</b>, 269–270 (2018).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhsVGis7rL" aria-label="CAS reference 9">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30082695" aria-label="PubMed reference 9">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 9" href="http://scholar.google.com/scholar_lookup?&amp;title=How%20to%20pick%20an%20electronic%20laboratory%20notebook&amp;journal=Nature&amp;volume=560&amp;pages=269-270&amp;publication_year=2018&amp;author=Kwok%2CR">
                    Google Scholar</a> 
                </p></li><li data-counter="10."><p id="ref-CR10">Kanza, S. et al. Electronic lab notebooks: can they replace paper? <i>J. Cheminformatics</i> <b>9</b>, 31 (2017).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 10" href="http://scholar.google.com/scholar_lookup?&amp;title=Electronic%20lab%20notebooks%3A%20can%20they%20replace%20paper&amp;journal=J.%20Cheminformatics&amp;volume=9&amp;publication_year=2017&amp;author=Kanza%2CS">
                    Google Scholar</a> 
                </p></li><li data-counter="11."><p id="ref-CR11">Rubacha, M., Rattan, A. K. &amp; Hosselet, S. C. A review of electronic laboratory notebooks available in the market today. <i>J. Lab. Autom.</i> <b>16</b>, 90–98 (2011).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXovFKrt7k%3D" aria-label="CAS reference 11">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21609689" aria-label="PubMed reference 11">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 11" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20review%20of%20electronic%20laboratory%20notebooks%20available%20in%20the%20market%20today&amp;journal=J.%20Lab.%20Autom.&amp;volume=16&amp;pages=90-98&amp;publication_year=2011&amp;author=Rubacha%2CM&amp;author=Rattan%2CAK&amp;author=Hosselet%2CSC">
                    Google Scholar</a> 
                </p></li><li data-counter="12."><p id="ref-CR12">Guerrero, S. et al. Analysis and implementation of an electronic laboratory notebook in a biomedical research institute. <i>PLoS ONE</i> <b>11</b>, e0160428 (2016).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27479083" aria-label="PubMed reference 12">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4968837" aria-label="PubMed Central reference 12">PubMed Central</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 12" href="http://scholar.google.com/scholar_lookup?&amp;title=Analysis%20and%20implementation%20of%20an%20electronic%20laboratory%20notebook%20in%20a%20biomedical%20research%20institute&amp;journal=PLoS%20ONE&amp;volume=11&amp;publication_year=2016&amp;author=Guerrero%2CS">
                    Google Scholar</a> 
                </p></li><li data-counter="13."><p id="ref-CR13">Dirnagl, U. &amp; Przesdzing, I. A pocket guide to electronic laboratory notebooks in the academic life sciences. <i>F1000Research</i> <b>5</b>, 2 (2016).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26835004" aria-label="PubMed reference 13">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4722687" aria-label="PubMed Central reference 13">PubMed Central</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 13" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20pocket%20guide%20to%20electronic%20laboratory%20notebooks%20in%20the%20academic%20life%20sciences&amp;journal=F1000Research&amp;volume=5&amp;publication_year=2016&amp;author=Dirnagl%2CU&amp;author=Przesdzing%2CI">
                    Google Scholar</a> 
                </p></li><li data-counter="14."><p id="ref-CR14">Coley, C. W. in <i>Artificial Intelligence in Drug Discovery</i> (ed. Brown, N) 327–348 (Royal Society of Chemistry, 2020).</p></li><li data-counter="15."><p id="ref-CR15">Raccuglia, P. et al. Machine-learning-assisted materials discovery using failed experiments. <i>Nature</i> <b>533</b>, 73–76 (2016).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XntlKhtL0%3D" aria-label="CAS reference 15">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27147027" aria-label="PubMed reference 15">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 15" href="http://scholar.google.com/scholar_lookup?&amp;title=Machine-learning-assisted%20materials%20discovery%20using%20failed%20experiments&amp;journal=Nature&amp;volume=533&amp;pages=73-76&amp;publication_year=2016&amp;author=Raccuglia%2CP">
                    Google Scholar</a> 
                </p></li><li data-counter="16."><p id="ref-CR16">Moosavi, S. M. et al. Capturing chemical intuition in synthesis of metal–organic frameworks. <i>Nat. Commun.</i> <b>10</b>, 539 (2019).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXntlKltbc%3D" aria-label="CAS reference 16">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30710082" aria-label="PubMed reference 16">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6358622" aria-label="PubMed Central reference 16">PubMed Central</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 16" href="http://scholar.google.com/scholar_lookup?&amp;title=Capturing%20chemical%20intuition%20in%20synthesis%20of%20metal%E2%80%93organic%20frameworks&amp;journal=Nat.%20Commun.&amp;volume=10&amp;publication_year=2019&amp;author=Moosavi%2CSM">
                    Google Scholar</a> 
                </p></li><li data-counter="17."><p id="ref-CR17">Ojea-Jiménez, I., Bastús, N. G. &amp; Puntes, V. Influence of the sequence of the reagents addition in the citrate-mediated synthesis of gold nanoparticles. <i>J. Phys. Chem. C</i> <b>115</b>, 15752–15757 (2011).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 17" href="http://scholar.google.com/scholar_lookup?&amp;title=Influence%20of%20the%20sequence%20of%20the%20reagents%20addition%20in%20the%20citrate-mediated%20synthesis%20of%20gold%20nanoparticles&amp;journal=J.%20Phys.%20Chem.%20C&amp;volume=115&amp;pages=15752-15757&amp;publication_year=2011&amp;author=Ojea-Jim%C3%A9nez%2CI&amp;author=Bast%C3%BAs%2CNG&amp;author=Puntes%2CV">
                    Google Scholar</a> 
                </p></li><li data-counter="18."><p id="ref-CR18">Huang, Y. et al. Importance of reagent addition order in contaminant degradation in an Fe(II)/PMS system. <i>RSC Adv.</i> <b>6</b>, 70271–70276 (2016).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XhtFOlur3P" aria-label="CAS reference 18">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 18" href="http://scholar.google.com/scholar_lookup?&amp;title=Importance%20of%20reagent%20addition%20order%20in%20contaminant%20degradation%20in%20an%20Fe%28II%29%2FPMS%20system&amp;journal=RSC%20Adv.&amp;volume=6&amp;pages=70271-70276&amp;publication_year=2016&amp;author=Huang%2CY">
                    Google Scholar</a> 
                </p></li><li data-counter="19."><p id="ref-CR19">Lowe, D. M. <i>Extraction of Chemical Structures and Reactions from the Literature</i>. PhD thesis, Univ. Cambridge (2012).</p></li><li data-counter="20."><p id="ref-CR20">Jin, W., Coley, C. W., Barzilay, R. &amp; Jaakkola, T. Predicting organic reaction outcomes with Weisfeiler-Lehman network. In <i>Proc. 31st International Conference on Neural Information Processing Systems</i> 2604–2613 (NIPS, 2017).</p></li><li data-counter="21."><p id="ref-CR21">Kim, E., Huang, K., Kononova, O., Ceder, G. &amp; Olivetti, E. Distilling a materials synthesis ontology. <i>Matter</i> <b>1</b>, 8–12 (2019).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 21" href="http://scholar.google.com/scholar_lookup?&amp;title=Distilling%20a%20materials%20synthesis%20ontology&amp;journal=Matter&amp;volume=1&amp;pages=8-12&amp;publication_year=2019&amp;author=Kim%2CE&amp;author=Huang%2CK&amp;author=Kononova%2CO&amp;author=Ceder%2CG&amp;author=Olivetti%2CE">
                    Google Scholar</a> 
                </p></li><li data-counter="22."><p id="ref-CR22">Roughley, S. D. &amp; Jordan, A. M. The medicinal chemist’s toolbox: an analysis of reactions used in the pursuit of drug candidates. <i>J. Med. Chem.</i> <b>54</b>, 3451–3479 (2011).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXlsVyhsrg%3D" aria-label="CAS reference 22">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21504168" aria-label="PubMed reference 22">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 22" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20medicinal%20chemist%E2%80%99s%20toolbox%3A%20an%20analysis%20of%20reactions%20used%20in%20the%20pursuit%20of%20drug%20candidates&amp;journal=J.%20Med.%20Chem.&amp;volume=54&amp;pages=3451-3479&amp;publication_year=2011&amp;author=Roughley%2CSD&amp;author=Jordan%2CAM">
                    Google Scholar</a> 
                </p></li><li data-counter="23."><p id="ref-CR23">Schneider, N., Lowe, D. M., Sayle, R. A., Tarselli, M. A. &amp; Landrum, G. A. Big data from pharmaceutical patents: a computational analysis of medicinal chemists’ bread and butter. <i>J. Med. Chem.</i> <b>59</b>, 4385–4402 (2016).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XltFGgurs%3D" aria-label="CAS reference 23">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27028220" aria-label="PubMed reference 23">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 23" href="http://scholar.google.com/scholar_lookup?&amp;title=Big%20data%20from%20pharmaceutical%20patents%3A%20a%20computational%20analysis%20of%20medicinal%20chemists%E2%80%99%20bread%20and%20butter&amp;journal=J.%20Med.%20Chem.&amp;volume=59&amp;pages=4385-4402&amp;publication_year=2016&amp;author=Schneider%2CN&amp;author=Lowe%2CDM&amp;author=Sayle%2CRA&amp;author=Tarselli%2CMA&amp;author=Landrum%2CGA">
                    Google Scholar</a> 
                </p></li><li data-counter="24."><p id="ref-CR24">Brown, D. G., Gagnon, M. M. &amp; Boström, J. Understanding our love affair with <i>p</i>-chlorophenyl: present day implications from historical biases of reagent selection. <i>J. Med. Chem.</i> <b>58</b>, 2390–2405 (2015).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXit1Snsr0%3D" aria-label="CAS reference 24">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25658759" aria-label="PubMed reference 24">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 24" href="http://scholar.google.com/scholar_lookup?&amp;title=Understanding%20our%20love%20affair%20with%20p-chlorophenyl%3A%20present%20day%20implications%20from%20historical%20biases%20of%20reagent%20selection&amp;journal=J.%20Med.%20Chem.&amp;volume=58&amp;pages=2390-2405&amp;publication_year=2015&amp;author=Brown%2CDG&amp;author=Gagnon%2CMM&amp;author=Bostr%C3%B6m%2CJ">
                    Google Scholar</a> 
                </p></li><li data-counter="25."><p id="ref-CR25">Brown, D. G. &amp; Boström, J. Analysis of past and present synthetic methodologies on medicinal chemistry: where have all the new reactions gone? <i>J. Med. Chem.</i> <b>59</b>, 4443–4458 (2015).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26571338" aria-label="PubMed reference 25">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 25" href="http://scholar.google.com/scholar_lookup?&amp;title=Analysis%20of%20past%20and%20present%20synthetic%20methodologies%20on%20medicinal%20chemistry%3A%20where%20have%20all%20the%20new%20reactions%20gone%3F&amp;journal=J.%20Med.%20Chem.&amp;volume=59&amp;pages=4443-4458&amp;publication_year=2015&amp;author=Brown%2CDG&amp;author=Bostr%C3%B6m%2CJ">
                    Google Scholar</a> 
                </p></li><li data-counter="26."><p id="ref-CR26">L. Bird, C., Willoughby, C. &amp; G. Frey, J. Laboratory notebooks in the digital era: the role of ELNs in record keeping for chemistry and other sciences. <i>Chem. Soc. Rev.</i> <b>42</b>, 8157–8175 (2013).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 26" href="http://scholar.google.com/scholar_lookup?&amp;title=Laboratory%20notebooks%20in%20the%20digital%20era%3A%20the%20role%20of%20ELNs%20in%20record%20keeping%20for%20chemistry%20and%20other%20sciences&amp;journal=Chem.%20Soc.%20Rev.&amp;volume=42&amp;pages=8157-8175&amp;publication_year=2013&amp;author=L.%20Bird%2CC&amp;author=Willoughby%2CC&amp;author=G.%20Frey%2CJ">
                    Google Scholar</a> 
                </p></li><li data-counter="27."><p id="ref-CR27">Oleksik, G., Milic-Frayling, N. &amp; Jones, R. Study of electronic lab notebook design and practices that emerged in a collaborative scientific environment. In <i>CSCW’14</i> <i>Proceedings of the 17th ACM Conference on Computer Supported Cooperative Work &amp; Social Computing</i> (ACM Press, 2014).</p></li><li data-counter="28."><p id="ref-CR28">McDonald, R. S. &amp; Wilks, P. A. Jcamp-dx: a standard form for exchange of infrared spectra in computer readable form. <i>Appl. Spectrosc.</i> <b>42</b>, 151–162 (1988).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaL1cXhtVeht7o%3D" aria-label="CAS reference 28">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 28" href="http://scholar.google.com/scholar_lookup?&amp;title=Jcamp-dx%3A%20a%20standard%20form%20for%20exchange%20of%20infrared%20spectra%20in%20computer%20readable%20form&amp;journal=Appl.%20Spectrosc.&amp;volume=42&amp;pages=151-162&amp;publication_year=1988&amp;author=McDonald%2CRS&amp;author=Wilks%2CPA">
                    Google Scholar</a> 
                </p></li><li data-counter="29."><p id="ref-CR29">Chalk, S. J. The open spectral database: an open platform for sharing and searching spectral data. <i>J. Cheminformatics</i> <b>8</b>, 55 (2016).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 29" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20open%20spectral%20database%3A%20an%20open%20platform%20for%20sharing%20and%20searching%20spectral%20data&amp;journal=J.%20Cheminformatics&amp;volume=8&amp;publication_year=2016&amp;author=Chalk%2CSJ">
                    Google Scholar</a> 
                </p></li><li data-counter="30."><p id="ref-CR30">Mehr, S. H. M., Craven, M., Leonov, A. I., Keenan, G. &amp; Cronin, L. A universal system for digitization and automatic execution of the chemical synthesis literature. <i>Science</i> <b>370</b>, 101–108 (2020).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhvF2itLnJ" aria-label="CAS reference 30">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33004517" aria-label="PubMed reference 30">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 30" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20universal%20system%20for%20digitization%20and%20automatic%20execution%20of%20the%20chemical%20synthesis%20literature&amp;journal=Science&amp;volume=370&amp;pages=101-108&amp;publication_year=2020&amp;author=Mehr%2CSHM&amp;author=Craven%2CM&amp;author=Leonov%2CAI&amp;author=Keenan%2CG&amp;author=Cronin%2CL">
                    Google Scholar</a> 
                </p></li><li data-counter="31."><p id="ref-CR31">Directorate General for Research and Innovation (European Commission) <i>Turning FAIR into Reality: Final Report and Action Plan from the European Commission Expert Group on FAIR Data</i> (Publications Office, 2018).</p></li><li data-counter="32."><p id="ref-CR32">Harrow, I. et al. Ontology mapping for semantically enabled applications. <i>Drug Discov. Today</i> <b>24</b>, 2068–2075 (2019).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31158512" aria-label="PubMed reference 32">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 32" href="http://scholar.google.com/scholar_lookup?&amp;title=Ontology%20mapping%20for%20semantically%20enabled%20applications&amp;journal=Drug%20Discov.%20Today&amp;volume=24&amp;pages=2068-2075&amp;publication_year=2019&amp;author=Harrow%2CI">
                    Google Scholar</a> 
                </p></li><li data-counter="33."><p id="ref-CR33">Davies, A. &amp; Patiny, L. NMRium browser-based nuclear magnetic resonance data processing. <i>Spectrosc. Eur</i>. <a href="https://doi.org/10.1255/sew.2021.a18">https://doi.org/10.1255/sew.2021.a18</a> (2021).</p></li><li data-counter="34."><p id="ref-CR34">Bonney, R. et al. Next steps for citizen science. <i>Science</i> <b>343</b>, 1436–1437 (2014).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24675940" aria-label="PubMed reference 34">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 34" href="http://scholar.google.com/scholar_lookup?&amp;title=Next%20steps%20for%20citizen%20science&amp;journal=Science&amp;volume=343&amp;pages=1436-1437&amp;publication_year=2014&amp;author=Bonney%2CR">
                    Google Scholar</a> 
                </p></li><li data-counter="35."><p id="ref-CR35">Nielsen, M. <i>Reinventing Discovery: the New Era of Networked Science</i> (Princeton Univ. Press, 2012).</p></li><li data-counter="36."><p id="ref-CR36">European Organization For Nuclear Research &amp; OpenAIRE <i>Zenodo</i> <a href="https://www.zenodo.org/">https://www.zenodo.org/</a> (2013).</p></li><li data-counter="37."><p id="ref-CR37">Tremouilhac, P. et al. Chemotion repository, a curated repository for reaction information and analytical data. <i>Chem. Methods</i> <b>1</b>, 8–11 (2020).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 37" href="http://scholar.google.com/scholar_lookup?&amp;title=Chemotion%20repository%2C%20a%20curated%20repository%20for%20reaction%20information%20and%20analytical%20data&amp;journal=Chem.%20Methods&amp;volume=1&amp;pages=8-11&amp;publication_year=2020&amp;author=Tremouilhac%2CP">
                    Google Scholar</a> 
                </p></li><li data-counter="38."><p id="ref-CR38">Coudert, F.-X. The rise of preprints in chemistry. <i>Nat. Chem.</i> <b>12</b>, 499–502 (2020).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXpvVaitrc%3D" aria-label="CAS reference 38">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32424256" aria-label="PubMed reference 38">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 38" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20rise%20of%20preprints%20in%20chemistry&amp;journal=Nat.%20Chem.&amp;volume=12&amp;pages=499-502&amp;publication_year=2020&amp;author=Coudert%2CF-X">
                    Google Scholar</a> 
                </p></li><li data-counter="39."><p id="ref-CR39">Bradley, J.-C. Open notebook science using blogs and wikis. <i>Nat. Prec</i>. <a href="https://doi.org/10.1038/npre.2007.39.1">https://doi.org/10.1038/npre.2007.39.1</a> (2007).</p></li><li data-counter="40."><p id="ref-CR40">Jablonka, K. M., Ongari, D., Moosavi, S. M. &amp; Smit, B. Big-data science in porous materials: materials genomics and machine learning. <i>Chem. Rev.</i> <b>120</b>, 8066–8129 (2020).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhtFWjsLnL" aria-label="CAS reference 40">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32520531" aria-label="PubMed reference 40">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7453404" aria-label="PubMed Central reference 40">PubMed Central</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 40" href="http://scholar.google.com/scholar_lookup?&amp;title=Big-data%20science%20in%20porous%20materials%3A%20materials%20genomics%20and%20machine%20learning&amp;journal=Chem.%20Rev.&amp;volume=120&amp;pages=8066-8129&amp;publication_year=2020&amp;author=Jablonka%2CKM&amp;author=Ongari%2CD&amp;author=Moosavi%2CSM&amp;author=Smit%2CB">
                    Google Scholar</a> 
                </p></li><li data-counter="41."><p id="ref-CR41">Olson, M. <i>The Logic of Collective Action; Public Goods and the Theory of Groups</i> (Schocken Books, 1971).</p></li><li data-counter="42."><p id="ref-CR42">Strasser, B. GENETICS: genbank—natural history in the 21st century? <i>Science</i> <b>322</b>, 537–538 (2008).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXhtlWjs7nP" aria-label="CAS reference 42">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18948528" aria-label="PubMed reference 42">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 42" href="http://scholar.google.com/scholar_lookup?&amp;title=GENETICS%3A%20genbank%E2%80%94natural%20history%20in%20the%2021st%20century%3F&amp;journal=Science&amp;volume=322&amp;pages=537-538&amp;publication_year=2008&amp;author=Strasser%2CB">
                    Google Scholar</a> 
                </p></li><li data-counter="43."><p id="ref-CR43">Williamson, A. E. et al. Open source drug discovery: highly potent antimalarial compounds derived from the Tres Cantos arylpyrroles. <i>ACS Centr. Sci.</i> <b>2</b>, 687–701 (2016).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XhsV2gsLnO" aria-label="CAS reference 43">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 43" href="http://scholar.google.com/scholar_lookup?&amp;title=Open%20source%20drug%20discovery%3A%20highly%20potent%20antimalarial%20compounds%20derived%20from%20the%20Tres%20Cantos%20arylpyrroles&amp;journal=ACS%20Centr.%20Sci.&amp;volume=2&amp;pages=687-701&amp;publication_year=2016&amp;author=Williamson%2CAE">
                    Google Scholar</a> 
                </p></li><li data-counter="44."><p id="ref-CR44">Chodera, J., Lee, A. A., London, N. &amp; von Delft, F. Crowdsourcing drug discovery for pandemics. <i>Nat. Chem.</i> <b>12</b>, 581–581 (2020).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXht1WjsbnE" aria-label="CAS reference 44">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32555379" aria-label="PubMed reference 44">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 44" href="http://scholar.google.com/scholar_lookup?&amp;title=Crowdsourcing%20drug%20discovery%20for%20pandemics&amp;journal=Nat.%20Chem.&amp;volume=12&amp;pages=581-581&amp;publication_year=2020&amp;author=Chodera%2CJ&amp;author=Lee%2CAA&amp;author=London%2CN&amp;author=Delft%2CF">
                    Google Scholar</a> 
                </p></li><li data-counter="45."><p id="ref-CR45">Perkmann, M. &amp; Schildt, H. Open data partnerships between firms and universities: the role of boundary organizations. <i>Res. Policy</i> <b>44</b>, 1133–1143 (2015).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 45" href="http://scholar.google.com/scholar_lookup?&amp;title=Open%20data%20partnerships%20between%20firms%20and%20universities%3A%20the%20role%20of%20boundary%20organizations&amp;journal=Res.%20Policy&amp;volume=44&amp;pages=1133-1143&amp;publication_year=2015&amp;author=Perkmann%2CM&amp;author=Schildt%2CH">
                    Google Scholar</a> 
                </p></li><li data-counter="46."><p id="ref-CR46">Jones, M. M. &amp; Chataway, J. The structural genomics consortium: successful organisational technology experiment or new institutional infrastructure for health research? <i>Technol. Anal. Strategic Manage.</i> <b>33</b>, 296–306 (2021).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 46" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20structural%20genomics%20consortium%3A%20successful%20organisational%20technology%20experiment%20or%20new%20institutional%20infrastructure%20for%20health%20research%3F&amp;journal=Technol.%20Anal.%20Strategic%20Manage.&amp;volume=33&amp;pages=296-306&amp;publication_year=2021&amp;author=Jones%2CMM&amp;author=Chataway%2CJ">
                    Google Scholar</a> 
                </p></li><li data-counter="47."><p id="ref-CR47">Edwards, A. M., Bountra, C., Kerr, D. J. &amp; Willson, T. M. Open access chemical and clinical probes to support drug discovery. <i>Nat. Chem. Biol.</i> <b>5</b>, 436–440 (2009).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXnsVags7g%3D" aria-label="CAS reference 47">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19536100" aria-label="PubMed reference 47">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 47" href="http://scholar.google.com/scholar_lookup?&amp;title=Open%20access%20chemical%20and%20clinical%20probes%20to%20support%20drug%20discovery&amp;journal=Nat.%20Chem.%20Biol.&amp;volume=5&amp;pages=436-440&amp;publication_year=2009&amp;author=Edwards%2CAM&amp;author=Bountra%2CC&amp;author=Kerr%2CDJ&amp;author=Willson%2CTM">
                    Google Scholar</a> 
                </p></li><li data-counter="48."><p id="ref-CR48">Jung, N., Deckers, A. &amp; Bräse, S. Ein molekülarchiv als akademisch integrierte service-einrichtung. <i>Biospektrum</i> <b>23</b>, 212–214 (2017).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 48" href="http://scholar.google.com/scholar_lookup?&amp;title=Ein%20molek%C3%BClarchiv%20als%20akademisch%20integrierte%20service-einrichtung&amp;journal=Biospektrum&amp;volume=23&amp;pages=212-214&amp;publication_year=2017&amp;author=Jung%2CN&amp;author=Deckers%2CA&amp;author=Br%C3%A4se%2CS">
                    Google Scholar</a> 
                </p></li><li data-counter="49."><p id="ref-CR49">Jablonka, K. M., Patiny, L. &amp; Smit, B. Making molecules vibrate: Interactive web environment for the teaching of infrared spectroscopy. <i>J. Chem. Educ</i>. <a href="https://doi.org/10.1021/acs.jchemed.1c01101">https://doi.org/10.1021/acs.jchemed.1c01101</a> (2022).</p></li><li data-counter="50."><p id="ref-CR50">Herres-Pawlis, S., Koepler, O. &amp; Steinbeck, C. NFDI4chem: shaping a digital and cultural change in chemistry. <i>Angew. Chem. Int. Ed.</i> <b>58</b>, 10766–10768 (2019).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhtl2rtbbF" aria-label="CAS reference 50">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 50" href="http://scholar.google.com/scholar_lookup?&amp;title=NFDI4chem%3A%20shaping%20a%20digital%20and%20cultural%20change%20in%20chemistry&amp;journal=Angew.%20Chem.%20Int.%20Ed.&amp;volume=58&amp;pages=10766-10768&amp;publication_year=2019&amp;author=Herres-Pawlis%2CS&amp;author=Koepler%2CO&amp;author=Steinbeck%2CC">
                    Google Scholar</a> 
                </p></li><li data-counter="51."><p id="ref-CR51">Steinbeck, C. et al. NFDI4chem—towards a national research data infrastructure for chemistry in Germany. <i>Res. Ideas Outcomes</i> <b>6</b>, e55852 (2020).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 51" href="http://scholar.google.com/scholar_lookup?&amp;title=NFDI4chem%E2%80%94towards%20a%20national%20research%20data%20infrastructure%20for%20chemistry%20in%20Germany&amp;journal=Res.%20Ideas%20Outcomes&amp;volume=6&amp;publication_year=2020&amp;author=Steinbeck%2CC">
                    Google Scholar</a> 
                </p></li><li data-counter="52."><p id="ref-CR52">Wulf, C. et al. A unified research data infrastructure for catalysis research—challenges and concepts. <i>ChemCatChem</i> <b>13</b>, 3223–3236 (2021).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXnt1Oms7k%3D" aria-label="CAS reference 52">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 52" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20unified%20research%20data%20infrastructure%20for%20catalysis%20research%E2%80%94challenges%20and%20concepts&amp;journal=ChemCatChem&amp;volume=13&amp;pages=3223-3236&amp;publication_year=2021&amp;author=Wulf%2CC">
                    Google Scholar</a> 
                </p></li><li data-counter="53."><p id="ref-CR53">Cooper, D. &amp; Springer, R. <i>Data Communities: A New Model for Supporting STEM Data Sharing</i> Technical Report (Univ. Nebraska-Lincoln, 2019).</p></li><li data-counter="54."><p id="ref-CR54">Evans, J. D., Bon, V., Senkovska, I. &amp; Kaskel, S. A universal standard archive file for adsorption data. <i>Langmuir</i> <b>37</b>, 4222–4226 (2021).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXnsl2ksLg%3D" aria-label="CAS reference 54">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33797923" aria-label="PubMed reference 54">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 54" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20universal%20standard%20archive%20file%20for%20adsorption%20data&amp;journal=Langmuir&amp;volume=37&amp;pages=4222-4226&amp;publication_year=2021&amp;author=Evans%2CJD&amp;author=Bon%2CV&amp;author=Senkovska%2CI&amp;author=Kaskel%2CS">
                    Google Scholar</a> 
                </p></li><li data-counter="55."><p id="ref-CR55">Siderius, D. <i>NIST/ARPA-E Database of Novel and Emerging Adsorbent Materials</i> (NIST, accessed 29 June 2020); <a href="https://doi.org/10.18434/T43882">https://doi.org/10.18434/T43882</a></p></li><li data-counter="56."><p id="ref-CR56">Ongari, D., Talirz, L., Jablonka, K. M., Siderius, D. W. &amp; Smit, B. Data-driven matching of experimental crystal structures and gas adsorption isotherms of Metal–Organic frameworks. <i>J. Chem. Eng. Data</i> <a href="https://doi.org/10.1021/acs.jced.1c00958">https://doi.org/10.1021/acs.jced.1c00958</a> (2022).</p></li><li data-counter="57."><p id="ref-CR57">Watson, M. When will ‘open science’ become simply ‘science’? <i>Genome Biol.</i> <b>16</b>, 101 (2015).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25986601" aria-label="PubMed reference 57">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4436110" aria-label="PubMed Central reference 57">PubMed Central</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 57" href="http://scholar.google.com/scholar_lookup?&amp;title=When%20will%20%E2%80%98open%20science%E2%80%99%20become%20simply%20%E2%80%98science%E2%80%99&amp;journal=Genome%20Biol.&amp;volume=16&amp;publication_year=2015&amp;author=Watson%2CM">
                    Google Scholar</a> 
                </p></li><li data-counter="58."><p id="ref-CR58">Tennant, J. Open science: Just science done right? <a href="https://figshare.com/articles/Open_Science_Just_science_done_right_/9759353/1">https://figshare.com/articles/Open_Science_Just_science_done_right_/9759353/1</a> (2019).</p></li><li data-counter="59."><p id="ref-CR59">Long, M. &amp; Schonfeld, R. <i>Supporting the Changing Research Practices of Chemists</i> Technical Report (Ithaca, 2013).</p></li><li data-counter="60."><p id="ref-CR60">Tremouilhac, P. et al. Chemotion ELN: an open source electronic lab notebook for chemists in academia. <i>J. Cheminformatics</i> <b>9</b>, 54 (2017).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 60" href="http://scholar.google.com/scholar_lookup?&amp;title=Chemotion%20ELN%3A%20an%20open%20source%20electronic%20lab%20notebook%20for%20chemists%20in%20academia&amp;journal=J.%20Cheminformatics&amp;volume=9&amp;publication_year=2017&amp;author=Tremouilhac%2CP">
                    Google Scholar</a> 
                </p></li><li data-counter="61."><p id="ref-CR61">Huang, Y.-C., Tremouilhac, P., Nguyen, A., Jung, N. &amp; Bräse, S. ChemSpectra: a web-based spectra editor for analytical data. <i>J. Cheminformatics</i> <b>13</b>, 8 (2021).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXht1WrtL3M" aria-label="CAS reference 61">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 61" href="http://scholar.google.com/scholar_lookup?&amp;title=ChemSpectra%3A%20a%20web-based%20spectra%20editor%20for%20analytical%20data&amp;journal=J.%20Cheminformatics&amp;volume=13&amp;publication_year=2021&amp;author=Huang%2CY-C&amp;author=Tremouilhac%2CP&amp;author=Nguyen%2CA&amp;author=Jung%2CN&amp;author=Br%C3%A4se%2CS">
                    Google Scholar</a> 
                </p></li><li data-counter="62."><p id="ref-CR62">Barillari, C. et al. openBIS ELN-LIMS: an open-source database for academic laboratories. <i>Bioinformatics</i> <b>32</b>, 638–640 (2016).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XhtlaqsLbL" aria-label="CAS reference 62">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26508761" aria-label="PubMed reference 62">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 62" href="http://scholar.google.com/scholar_lookup?&amp;title=openBIS%20ELN-LIMS%3A%20an%20open-source%20database%20for%20academic%20laboratories&amp;journal=Bioinformatics&amp;volume=32&amp;pages=638-640&amp;publication_year=2016&amp;author=Barillari%2CC">
                    Google Scholar</a> 
                </p></li><li data-counter="63."><p id="ref-CR63">Patiny, L. et al. The c6h6 NMR repository: an integral solution to control the flow of your data from the magnet to the public. <i>Magn. Reson. Chem.</i> <b>56</b>, 520–528 (2017).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28981966" aria-label="PubMed reference 63">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 63" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20c6h6%20NMR%20repository%3A%20an%20integral%20solution%20to%20control%20the%20flow%20of%20your%20data%20from%20the%20magnet%20to%20the%20public&amp;journal=Magn.%20Reson.%20Chem.&amp;volume=56&amp;pages=520-528&amp;publication_year=2017&amp;author=Patiny%2CL">
                    Google Scholar</a> 
                </p></li><li data-counter="64."><p id="ref-CR64">A. Badiola, K. et al. Experiences with a researcher-centric ELN. <i>Chem. Sci.</i> <b>6</b>, 1614–1629 (2015).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29308130" aria-label="PubMed reference 64">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 64" href="http://scholar.google.com/scholar_lookup?&amp;title=Experiences%20with%20a%20researcher-centric%20ELN&amp;journal=Chem.%20Sci.&amp;volume=6&amp;pages=1614-1629&amp;publication_year=2015&amp;author=A.%20Badiola%2CK">
                    Google Scholar</a> 
                </p></li><li data-counter="65."><p id="ref-CR65">Woelfle, M., Olliaro, P. &amp; Todd, M. H. Open science is a research accelerator. <i>Nat. Chem.</i> <b>3</b>, 745–748 (2011).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXht1ajtbbL" aria-label="CAS reference 65">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21941234" aria-label="PubMed reference 65">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 65" href="http://scholar.google.com/scholar_lookup?&amp;title=Open%20science%20is%20a%20research%20accelerator&amp;journal=Nat.%20Chem.&amp;volume=3&amp;pages=745-748&amp;publication_year=2011&amp;author=Woelfle%2CM&amp;author=Olliaro%2CP&amp;author=Todd%2CMH">
                    Google Scholar</a> 
                </p></li><li data-counter="66."><p id="ref-CR66">Carpi, N., Minges, A. &amp; Piel, M. eLabFTW: an open source laboratory notebook for research labs. <i>J. Open Source Softw.</i> <b>2</b>, 146 (2017).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 66" href="http://scholar.google.com/scholar_lookup?&amp;title=eLabFTW%3A%20an%20open%20source%20laboratory%20notebook%20for%20research%20labs&amp;journal=J.%20Open%20Source%20Softw.&amp;volume=2&amp;publication_year=2017&amp;author=Carpi%2CN&amp;author=Minges%2CA&amp;author=Piel%2CM">
                    Google Scholar</a> 
                </p></li><li data-counter="67."><p id="ref-CR67">Rudolphi, F. Ein elektronisches laborjournal als open-source-software. <i>Nachr. Chem.</i> <b>58</b>, 548–550 (2010).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXosl2qsLg%3D" aria-label="CAS reference 67">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 67" href="http://scholar.google.com/scholar_lookup?&amp;title=Ein%20elektronisches%20laborjournal%20als%20open-source-software&amp;journal=Nachr.%20Chem.&amp;volume=58&amp;pages=548-550&amp;publication_year=2010&amp;author=Rudolphi%2CF">
                    Google Scholar</a> 
                </p></li><li data-counter="68."><p id="ref-CR68">Brandt, N. et al. Kadi4mat: a research data infrastructure for materials science. <i>Data Sci. J.</i> <b>20</b>, 8 (2021).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 68" href="http://scholar.google.com/scholar_lookup?&amp;title=Kadi4mat%3A%20a%20research%20data%20infrastructure%20for%20materials%20science&amp;journal=Data%20Sci.%20J.&amp;volume=20&amp;publication_year=2021&amp;author=Brandt%2CN">
                    Google Scholar</a> 
                </p></li><li data-counter="69."><p id="ref-CR69">Jablonka, K. M. et al. Connecting lab experiments with computer experiments: making ‘routine’ simulations routine. Preprint at <i>ChemRxiv</i> <a href="https://doi.org/10.26434/chemrxiv-2021-h3381-v2">https://doi.org/10.26434/chemrxiv-2021-h3381-v2</a> (2021).</p></li><li data-counter="70."><p id="ref-CR70">Gray, A. J., Goble, C. A., Jimenez, R. et al. Bioschemas: from potato salad to protein annotation. In <i>16th International Semantic Web Conference</i> (2017).</p></li><li data-counter="71."><p id="ref-CR71">Jablonka, K. M. et al. A data-driven perspective on the colours of metal–organic frameworks. <i>Chem. Sci.</i> <b>12</b>, 3587–3598 (2021).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXis12ltrjE" aria-label="CAS reference 71">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 71" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20data-driven%20perspective%20on%20the%20colours%20of%20metal%E2%80%93organic%20frameworks&amp;journal=Chem.%20Sci.&amp;volume=12&amp;pages=3587-3598&amp;publication_year=2021&amp;author=Jablonka%2CKM">
                    Google Scholar</a> 
                </p></li><li data-counter="72."><p id="ref-CR72">Kratsios, M., Kent, S. &amp; Rinat. O. Connecting Americans to coronavirus information online. <i>Trump White House Archives</i> <a href="https://trumpwhitehouse.archives.gov/articles/connecting-americans-coronavirus-information-online/">https://trumpwhitehouse.archives.gov/articles/connecting-americans-coronavirus-information-online/</a> (2020).</p></li><li data-counter="73."><p id="ref-CR73"><i>COVID-19 Announcements Structured Data</i> (Google Search Central, 2021); <a href="https://developers.google.com/search/docs/advanced/structured-data/special-announcements">https://developers.google.com/search/docs/advanced/structured-data/special-announcements</a></p></li><li data-counter="74."><p id="ref-CR74">Fletcher, G., Groth, P. &amp; Sequeda, J. Knowledge scientists: unlocking the data-driven organization. Preprint at <a href="https://arxiv.org/abs/2004.07917">https://arxiv.org/abs/2004.07917</a> (2020).</p></li><li data-counter="75."><p id="ref-CR75">Kellogg, G., Champin, P.-A. &amp; Longley, D. <i>JSON-LD 1.1—A JSON-based Serialization for Linked Data</i>. (W3C, 2020).</p></li><li data-counter="76."><p id="ref-CR76">Tennison, J. <i>CSV on the Web: A Primer</i> (W3C, 2016).</p></li><li data-counter="77."><p id="ref-CR77">Coles, S. J., Frey, J. G., Bird, C. L., Whitby, R. J. &amp; Day, A. E. First steps towards semantic descriptions of electronic laboratory notebook records. <i>J. Cheminformatics</i> <b>5</b>, 52 (2013).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 77" href="http://scholar.google.com/scholar_lookup?&amp;title=First%20steps%20towards%20semantic%20descriptions%20of%20electronic%20laboratory%20notebook%20records&amp;journal=J.%20Cheminformatics&amp;volume=5&amp;publication_year=2013&amp;author=Coles%2CSJ&amp;author=Frey%2CJG&amp;author=Bird%2CCL&amp;author=Whitby%2CRJ&amp;author=Day%2CAE">
                    Google Scholar</a> 
                </p></li><li data-counter="78."><p id="ref-CR78">Lütjohann, D. S., Jung, N. &amp; Bräse, S. Open source life science automation: design of experiments and data acquisition via ‘dial-a-device’. <i>Chemometr. Intell. Lab. Syst.</i> <b>144</b>, 100–107 (2015).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 78" href="http://scholar.google.com/scholar_lookup?&amp;title=Open%20source%20life%20science%20automation%3A%20design%20of%20experiments%20and%20data%20acquisition%20via%20%E2%80%98dial-a-device&amp;journal=Chemometr.%20Intell.%20Lab.%20Syst.&amp;volume=144&amp;pages=100-107&amp;publication_year=2015&amp;author=L%C3%BCtjohann%2CDS&amp;author=Jung%2CN&amp;author=Br%C3%A4se%2CS">
                    Google Scholar</a> 
                </p></li><li data-counter="79."><p id="ref-CR79">Chung, Y. G. et al. Advances, updates, and analytics for the computation-ready, experimental metal–organic framework database: CoRE MOF 2019. <i>J. Chem. Eng. Data</i> <b>64</b>, 5985–5998 (2019).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXitVOmtrrL" aria-label="CAS reference 79">CAS</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 79" href="http://scholar.google.com/scholar_lookup?&amp;title=Advances%2C%20updates%2C%20and%20analytics%20for%20the%20computation-ready%2C%20experimental%20metal%E2%80%93organic%20framework%20database%3A%20CoRE%20MOF%202019&amp;journal=J.%20Chem.%20Eng.%20Data&amp;volume=64&amp;pages=5985-5998&amp;publication_year=2019&amp;author=Chung%2CYG">
                    Google Scholar</a> 
                </p></li><li data-counter="80."><p id="ref-CR80">Gražulis, S. et al. Crystallography Open Database—an open-access collection of crystal structures. <i>J. Appl. Crystallogr.</i> <b>42</b>, 726–729 (2009).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22477773" aria-label="PubMed reference 80">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3253730" aria-label="PubMed Central reference 80">PubMed Central</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 80" href="http://scholar.google.com/scholar_lookup?&amp;title=Crystallography%20Open%20Database%E2%80%94an%20open-access%20collection%20of%20crystal%20structures&amp;journal=J.%20Appl.%20Crystallogr.&amp;volume=42&amp;pages=726-729&amp;publication_year=2009&amp;author=Gra%C5%BEulis%2CS">
                    Google Scholar</a> 
                </p></li><li data-counter="81."><p id="ref-CR81">Gražulis, S. et al. Crystallography Open Database (COD): an open-access collection of crystal structures and platform for world-wide collaboration. <i>Nucleic Acids Res.</i> <b>40</b>, D420–D427 (2012).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22070882" aria-label="PubMed reference 81">PubMed</a> 
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 81" href="http://scholar.google.com/scholar_lookup?&amp;title=Crystallography%20Open%20Database%20%28COD%29%3A%20an%20open-access%20collection%20of%20crystal%20structures%20and%20platform%20for%20world-wide%20collaboration&amp;journal=Nucleic%20Acids%20Res.&amp;volume=40&amp;pages=D420-D427&amp;publication_year=2012&amp;author=Gra%C5%BEulis%2CS">
                    Google Scholar</a> 
                </p></li><li data-counter="82."><p id="ref-CR82">Chalk, S. J. SciData: a data model and ontology for semantic representation of scientific data. <i>J. Cheminformatics</i> <b>8</b>, 54 (2016).</p><p><a data-track="click" data-track-action="outbound reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 82" href="http://scholar.google.com/scholar_lookup?&amp;title=SciData%3A%20a%20data%20model%20and%20ontology%20for%20semantic%20representation%20of%20scientific%20data&amp;journal=J.%20Cheminformatics&amp;volume=8&amp;publication_year=2016&amp;author=Chalk%2CSJ">
                    Google Scholar</a> 
                </p></li></ol><p><a data-track="click" data-track-action="download citation references" data-track-label="link" href="https://citation-needed.springer.com/v2/references/10.1038/s41557-022-00910-7?format=refman&amp;flavour=references">Download references<svg width="16" height="16" focusable="false" role="img" aria-hidden="true"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#global-icon-download"></use></svg></a></p></div></div></div></section></div><section data-title="Acknowledgements"><div id="Ack1-section"><h2 id="Ack1">Acknowledgements</h2><p>This work was partially supported by the European Research Council (ERC) under the European Union’s Horizon 2020 research and innovation programme (grant agreement no. 666983, MaGic) and the Swiss National Science Foundation (SNSF) through the National Center of Competence in Research (NCCR) and Materials’ Revolution: Computational Design and Discovery of Novel Materials (MARVEL). We thank M. Evans, L. Talirz, M. Moosavi, M. Asgari, N. Marzari, G. Pizzi and fellow EPFL Data Champions for discussion and inputs and thank the cheminfo and Zakodium developers (among others, M. Zasso, D. Kostro, J. Wiest, A. M. Castillo, A. Bolaños, J. Osorio and N. Pellet; also see <a href="https://cheminfo.github.io/team">https://cheminfo.github.io/team</a> for a list of contributors) for their invaluable contributions (conceiving and implementing many of the examples discussed in this perspective). Of course, we also thank the chemists whose feedback about our ELN implementation shaped our Perspective.</p></div></section><section aria-labelledby="author-information" data-title="Author information"><div id="author-information-section"><h2 id="author-information">Author information</h2><div id="author-information-content"><h3 id="affiliations">Authors and Affiliations</h3><ol><li id="Aff1"><p>Laboratory of Molecular Simulation (LSMO), Institut des Sciences et Ingenierie Chimiques (ISIC), École Polytechnique Fédérale de Lausanne (EPFL), Sion, Switzerland</p><p>Kevin Maik Jablonka &amp; Berend Smit</p></li><li id="Aff2"><p>Institut des Sciences et Ingénierie Chimiques (ISIC), École Polytechnique Fédérale de Lausanne (EPFL), Lausanne, Switzerland</p><p>Luc Patiny</p></li></ol><h3 id="contributions">Contributions</h3><p>K.M.J. and B.S. wrote the manuscript with inputs from L.P. All the authors contributed to discussions.</p><h3 id="corresponding-author">Corresponding authors</h3><p id="corresponding-author-list">Correspondence to
                <a id="corresp-c1" href="mailto:luc.patiny@epfl.ch">Luc Patiny</a> or <a id="corresp-c2" href="mailto:berend.smit@epfl.ch">Berend Smit</a>.</p></div></div></section><section data-title="Ethics declarations"><div id="ethics-section"><h2 id="ethics">Ethics declarations</h2><div id="ethics-content">
              
                <h3 id="FPar4">Competing interests</h3>
                <p>L.P. is chief scientific officer of Zakodium Sàrl, a company dedicated to the development of tools for storing, processing and analysis of scientific information. All the authors are contributors to the cheminfo ecoystem.</p>
              
            </div></div></section><section data-title="Peer review"><div id="peer-review-section"><h2 id="peer-review">Peer review</h2><div id="peer-review-content">
              
              
                <h3 id="FPar3">Peer review information</h3>
                <p><i>Nature Chemistry</i> thanks Samantha Kanza, Matthew Todd and the other, anonymous, reviewer(s) for their contribution to the peer review of this work.</p>
              
            </div></div></section><section data-title="Additional information"><div id="additional-information-section"><h2 id="additional-information">Additional information</h2><p><b>Publisher’s note</b> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div></section><section data-title="Extended data"><div id="Sec12-section"><h2 id="Sec12">Extended data</h2><div id="Sec12-content"><div data-test="supplementary-info"><div data-test="supp-item" id="Fig4"><h3><a data-track="click" data-track-action="view supplementary info" data-track-label="link" data-test="supp-info-link" href="https://www.nature.com/articles/s41557-022-00910-7/figures/4" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41557-022-00910-7/MediaObjects/41557_2022_910_Fig4_ESM.jpg">Extended Data Fig. 1 Fragment of a NMR spectrum serialized to a classic standard format.</a></h3><p>This is an example of a JCAMP-DX file. This format is a widely used IUPAC-recommended format for spectra that is, for example, supported by the cheminfo and chemotion ELNs. Also, spectra in many databases such as the <a href="http://www.webbook.nist.gov">NIST webbook</a> or the <a href="http://www.irug.org/search-spectral-database">Infrared &amp; Raman Users Group (IRUG) Spectral Database</a> can be downloaded in JCAMP-DX format. A JCAMP-DX file can contain multiple blocks of labelled data records (LDR). That is, one can store multiple related spectra (such as repeated measurements) in the same file. All data blocks must contain a <span>CORE</span> header with basic metadata such as <span>OWNER</span>, <span>DATATYPE</span>. The IUPAC working group also provides a vocabulary of further global labels such as for the temperature/pressure/CAS-number. Data can also be compressed using various compression schemes. Note that the JCAMP-DX format is only one, old standard, and many others have been proposed. The JCAMP-DX format, however, does allow for the addition of an unlimited number of private labels by using the <span>##$</span> prefix, which allows every system to tailor the format to its own needs. Drawbacks of this format are, however, that it does not come with native, standardised, support for semantic web features (such as linking to a vocabulary) and, in contrast to formats like xml, csv, or json, that it is not natively supported by many general purpose tools.</p></div><div data-test="supp-item" id="Fig5"><h3><a data-track="click" data-track-action="view supplementary info" data-track-label="link" data-test="supp-info-link" href="https://www.nature.com/articles/s41557-022-00910-7/figures/5" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41557-022-00910-7/MediaObjects/41557_2022_910_Fig5_ESM.jpg">Extended Data Fig. 2 Fragment of a NMR spectrum serialized to a modern standard format.</a></h3><p>We show another NMR dataset (taken from <a href="https://stuchalk.github.io/scidata/examples/nmr.jsonld">the SciData website from the Chalk Group at the University of North Florida</a>) serialized to JSON-LD using the SciData data model<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 82" title="Chalk, S. J. SciData: a data model and ontology for semantic representation of scientific data. J. Cheminformatics 8, 54 (2016)." href="https://www.nature.com/articles/s41557-022-00910-7#ref-CR82" id="ref-link-section-d49829465e1812">82</a></sup>. One important part on the JSON-LD file is the <span>@context</span> field. The values in this field links to the vocabularies that are used for naming things in this datafile. For instance, for units, the vocabularies provided by <a href="http://qudt.org/">qudt</a> are used, whereas the method is described using the <a href="https://www.ebi.ac.uk/ols/ontologies/pride/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FCHMO_0000591">chemical methods ontology</a> (from which it is clear that, for instance, NMR spectroscopy is—similar to electron spin resonance spectroscopy–a magnetic resonance method). Importantly, almost all modern programming languages provide support for reading such json files. The <span>@type</span> field can describe the format of the data, for instance, to let a computer now that it can expect a list of doubles. Different parts of the file (such as methodology, the dataset) can be access by their own address.</p></div></div></div></div></section><section data-title="Supplementary information"><div id="Sec13-section"><h2 id="Sec13">Supplementary information</h2></div></section><section data-title="Rights and permissions"></section><section aria-labelledby="article-info" data-title="About this article"><div id="article-info-section"><h2 id="article-info">About this article</h2><div id="article-info-content"><div><p><a data-crossmark="10.1038/s41557-022-00910-7" target="_blank" rel="noopener" href="https://crossmark.crossref.org/dialog/?doi=10.1038/s41557-022-00910-7" data-track="click" data-track-action="Click Crossmark" data-track-label="link" data-test="crossmark"><img width="57" height="81" alt="Verify currency and authenticity via CrossMark" src="data:image/svg+xml;base64,<svg height="81" width="57" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="m17.35 35.45 21.3-14.2v-17.03h-21.3" fill="#989898"/><path d="m38.65 35.45-21.3-14.2v-17.03h21.3" fill="#747474"/><path d="m28 .5c-12.98 0-23.5 10.52-23.5 23.5s10.52 23.5 23.5 23.5 23.5-10.52 23.5-23.5c0-6.23-2.48-12.21-6.88-16.62-4.41-4.4-10.39-6.88-16.62-6.88zm0 41.25c-9.8 0-17.75-7.95-17.75-17.75s7.95-17.75 17.75-17.75 17.75 7.95 17.75 17.75c0 4.71-1.87 9.22-5.2 12.55s-7.84 5.2-12.55 5.2z" fill="#535353"/><path d="m41 36c-5.81 6.23-15.23 7.45-22.43 2.9-7.21-4.55-10.16-13.57-7.03-21.5l-4.92-3.11c-4.95 10.7-1.19 23.42 8.78 29.71 9.97 6.3 23.07 4.22 30.6-4.86z" fill="#9c9c9c"/><path d="m.2 58.45c0-.75.11-1.42.33-2.01s.52-1.09.91-1.5c.38-.41.83-.73 1.34-.94.51-.22 1.06-.32 1.65-.32.56 0 1.06.11 1.51.35.44.23.81.5 1.1.81l-.91 1.01c-.24-.24-.49-.42-.75-.56-.27-.13-.58-.2-.93-.2-.39 0-.73.08-1.05.23-.31.16-.58.37-.81.66-.23.28-.41.63-.53 1.04-.13.41-.19.88-.19 1.39 0 1.04.23 1.86.68 2.46.45.59 1.06.88 1.84.88.41 0 .77-.07 1.07-.23s.59-.39.85-.68l.91 1c-.38.43-.8.76-1.28.99-.47.22-1 .34-1.58.34-.59 0-1.13-.1-1.64-.31-.5-.2-.94-.51-1.31-.91-.38-.4-.67-.9-.88-1.48-.22-.59-.33-1.26-.33-2.02zm8.4-5.33h1.61v2.54l-.05 1.33c.29-.27.61-.51.96-.72s.76-.31 1.24-.31c.73 0 1.27.23 1.61.71.33.47.5 1.14.5 2.02v4.31h-1.61v-4.1c0-.57-.08-.97-.25-1.21-.17-.23-.45-.35-.83-.35-.3 0-.56.08-.79.22-.23.15-.49.36-.78.64v4.8h-1.61zm7.37 6.45c0-.56.09-1.06.26-1.51.18-.45.42-.83.71-1.14.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.36c.07.62.29 1.1.65 1.44.36.33.82.5 1.38.5.29 0 .57-.04.83-.13s.51-.21.76-.37l.55 1.01c-.33.21-.69.39-1.09.53-.41.14-.83.21-1.26.21-.48 0-.92-.08-1.34-.25-.41-.16-.76-.4-1.07-.7-.31-.31-.55-.69-.72-1.13-.18-.44-.26-.95-.26-1.52zm4.6-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.07.45-.31.29-.5.73-.58 1.3zm2.5.62c0-.57.09-1.08.28-1.53.18-.44.43-.82.75-1.13s.69-.54 1.1-.71c.42-.16.85-.24 1.31-.24.45 0 .84.08 1.17.23s.61.34.85.57l-.77 1.02c-.19-.16-.38-.28-.56-.37-.19-.09-.39-.14-.61-.14-.56 0-1.01.21-1.35.63-.35.41-.52.97-.52 1.67 0 .69.17 1.24.51 1.66.34.41.78.62 1.32.62.28 0 .54-.06.78-.17.24-.12.45-.26.64-.42l.67 1.03c-.33.29-.69.51-1.08.65-.39.15-.78.23-1.18.23-.46 0-.9-.08-1.31-.24-.4-.16-.75-.39-1.05-.7s-.53-.69-.7-1.13c-.17-.45-.25-.96-.25-1.53zm6.91-6.45h1.58v6.17h.05l2.54-3.16h1.77l-2.35 2.8 2.59 4.07h-1.75l-1.77-2.98-1.08 1.23v1.75h-1.58zm13.69 1.27c-.25-.11-.5-.17-.75-.17-.58 0-.87.39-.87 1.16v.75h1.34v1.27h-1.34v5.6h-1.61v-5.6h-.92v-1.2l.92-.07v-.72c0-.35.04-.68.13-.98.08-.31.21-.57.4-.79s.42-.39.71-.51c.28-.12.63-.18 1.04-.18.24 0 .48.02.69.07.22.05.41.1.57.17zm.48 5.18c0-.57.09-1.08.27-1.53.17-.44.41-.82.72-1.13.3-.31.65-.54 1.04-.71.39-.16.8-.24 1.23-.24s.84.08 1.24.24c.4.17.74.4 1.04.71s.54.69.72 1.13c.19.45.28.96.28 1.53s-.09 1.08-.28 1.53c-.18.44-.42.82-.72 1.13s-.64.54-1.04.7-.81.24-1.24.24-.84-.08-1.23-.24-.74-.39-1.04-.7c-.31-.31-.55-.69-.72-1.13-.18-.45-.27-.96-.27-1.53zm1.65 0c0 .69.14 1.24.43 1.66.28.41.68.62 1.18.62.51 0 .9-.21 1.19-.62.29-.42.44-.97.44-1.66 0-.7-.15-1.26-.44-1.67-.29-.42-.68-.63-1.19-.63-.5 0-.9.21-1.18.63-.29.41-.43.97-.43 1.67zm6.48-3.44h1.33l.12 1.21h.05c.24-.44.54-.79.88-1.02.35-.24.7-.36 1.07-.36.32 0 .59.05.78.14l-.28 1.4-.33-.09c-.11-.01-.23-.02-.38-.02-.27 0-.56.1-.86.31s-.55.58-.77 1.1v4.2h-1.61zm-47.87 15h1.61v4.1c0 .57.08.97.25 1.2.17.24.44.35.81.35.3 0 .57-.07.8-.22.22-.15.47-.39.73-.73v-4.7h1.61v6.87h-1.32l-.12-1.01h-.04c-.3.36-.63.64-.98.86-.35.21-.76.32-1.24.32-.73 0-1.27-.24-1.61-.71-.33-.47-.5-1.14-.5-2.02zm9.46 7.43v2.16h-1.61v-9.59h1.33l.12.72h.05c.29-.24.61-.45.97-.63.35-.17.72-.26 1.1-.26.43 0 .81.08 1.15.24.33.17.61.4.84.71.24.31.41.68.53 1.11.13.42.19.91.19 1.44 0 .59-.09 1.11-.25 1.57-.16.47-.38.85-.65 1.16-.27.32-.58.56-.94.73-.35.16-.72.25-1.1.25-.3 0-.6-.07-.9-.2s-.59-.31-.87-.56zm0-2.3c.26.22.5.37.73.45.24.09.46.13.66.13.46 0 .84-.2 1.15-.6.31-.39.46-.98.46-1.77 0-.69-.12-1.22-.35-1.61-.23-.38-.61-.57-1.13-.57-.49 0-.99.26-1.52.77zm5.87-1.69c0-.56.08-1.06.25-1.51.16-.45.37-.83.65-1.14.27-.3.58-.54.93-.71s.71-.25 1.08-.25c.39 0 .73.07 1 .2.27.14.54.32.81.55l-.06-1.1v-2.49h1.61v9.88h-1.33l-.11-.74h-.06c-.25.25-.54.46-.88.64-.33.18-.69.27-1.06.27-.87 0-1.56-.32-2.07-.95s-.76-1.51-.76-2.65zm1.67-.01c0 .74.13 1.31.4 1.7.26.38.65.58 1.15.58.51 0 .99-.26 1.44-.77v-3.21c-.24-.21-.48-.36-.7-.45-.23-.08-.46-.12-.7-.12-.45 0-.82.19-1.13.59-.31.39-.46.95-.46 1.68zm6.35 1.59c0-.73.32-1.3.97-1.71.64-.4 1.67-.68 3.08-.84 0-.17-.02-.34-.07-.51-.05-.16-.12-.3-.22-.43s-.22-.22-.38-.3c-.15-.06-.34-.1-.58-.1-.34 0-.68.07-1 .2s-.63.29-.93.47l-.59-1.08c.39-.24.81-.45 1.28-.63.47-.17.99-.26 1.54-.26.86 0 1.51.25 1.93.76s.63 1.25.63 2.21v4.07h-1.32l-.12-.76h-.05c-.3.27-.63.48-.98.66s-.73.27-1.14.27c-.61 0-1.1-.19-1.48-.56-.38-.36-.57-.85-.57-1.46zm1.57-.12c0 .3.09.53.27.67.19.14.42.21.71.21.28 0 .54-.07.77-.2s.48-.31.73-.56v-1.54c-.47.06-.86.13-1.18.23-.31.09-.57.19-.76.31s-.33.25-.41.4c-.09.15-.13.31-.13.48zm6.29-3.63h-.98v-1.2l1.06-.07.2-1.88h1.34v1.88h1.75v1.27h-1.75v3.28c0 .8.32 1.2.97 1.2.12 0 .24-.01.37-.04.12-.03.24-.07.34-.11l.28 1.19c-.19.06-.4.12-.64.17-.23.05-.49.08-.76.08-.4 0-.74-.06-1.02-.18-.27-.13-.49-.3-.67-.52-.17-.21-.3-.48-.37-.78-.08-.3-.12-.64-.12-1.01zm4.36 2.17c0-.56.09-1.06.27-1.51s.41-.83.71-1.14c.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.37c.08.62.29 1.1.65 1.44.36.33.82.5 1.38.5.3 0 .58-.04.84-.13.25-.09.51-.21.76-.37l.54 1.01c-.32.21-.69.39-1.09.53s-.82.21-1.26.21c-.47 0-.92-.08-1.33-.25-.41-.16-.77-.4-1.08-.7-.3-.31-.54-.69-.72-1.13-.17-.44-.26-.95-.26-1.52zm4.61-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.08.45-.31.29-.5.73-.57 1.3zm3.01 2.23c.31.24.61.43.92.57.3.13.63.2.98.2.38 0 .65-.08.83-.23s.27-.35.27-.6c0-.14-.05-.26-.13-.37-.08-.1-.2-.2-.34-.28-.14-.09-.29-.16-.47-.23l-.53-.22c-.23-.09-.46-.18-.69-.3-.23-.11-.44-.24-.62-.4s-.33-.35-.45-.55c-.12-.21-.18-.46-.18-.75 0-.61.23-1.1.68-1.49.44-.38 1.06-.57 1.83-.57.48 0 .91.08 1.29.25s.71.36.99.57l-.74.98c-.24-.17-.49-.32-.73-.42-.25-.11-.51-.16-.78-.16-.35 0-.6.07-.76.21-.17.15-.25.33-.25.54 0 .14.04.26.12.36s.18.18.31.26c.14.07.29.14.46.21l.54.19c.23.09.47.18.7.29s.44.24.64.4c.19.16.34.35.46.58.11.23.17.5.17.82 0 .3-.06.58-.17.83-.12.26-.29.48-.51.68-.23.19-.51.34-.84.45-.34.11-.72.17-1.15.17-.48 0-.95-.09-1.41-.27-.46-.19-.86-.41-1.2-.68z" fill="#535353"/></g></svg>"/></a></p><div><h3 id="citeas">Cite this article</h3><p>Jablonka, K.M., Patiny, L. &amp; Smit, B. Making the collective knowledge of chemistry open and machine actionable.
                    <i>Nat. Chem.</i> <b>14, </b>365–376 (2022). https://doi.org/10.1038/s41557-022-00910-7</p><p><a data-test="citation-link" data-track="click" data-track-action="download article citation" data-track-label="link" data-track-external="" href="https://citation-needed.springer.com/v2/references/10.1038/s41557-022-00910-7?format=refman&amp;flavour=citation">Download citation<svg width="16" height="16" focusable="false" role="img" aria-hidden="true"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#global-icon-download"></use></svg></a></p><ul data-test="publication-history"><li><p>Received<span>: </span><span><time datetime="2021-03-09">09 March 2021</time></span></p></li><li><p>Accepted<span>: </span><span><time datetime="2022-02-10">10 February 2022</time></span></p></li><li><p>Published<span>: </span><span><time datetime="2022-04-04">04 April 2022</time></span></p></li><li><p>Issue Date<span>: </span><span><time datetime="2022-04">April 2022</time></span></p></li><li><p><abbr title="Digital Object Identifier">DOI</abbr><span>: </span><span>https://doi.org/10.1038/s41557-022-00910-7</span></p></li></ul></div></div></div></div></section>

            

            

            </div></div>
  </body>
</html>
