<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <title>James Routley | Feed</title>
    <link
      rel="stylesheet"
      type="text/css"
      href="../styles.css"
      media="screen"
    />
  </head>
  <body>
    <a href="/index.html">Back</a>
    <a href="https://www.owlposting.com/p/mapping-the-off-target-effects-of">Original</a>
    <h1>Mapping the off-target effects of every FDA-approved drug in existence</h1>
    
    <div id="readability-page-1" class="page"><div><div><div><article><div><div><div dir="auto"><div><figure><a target="_blank" href="https://substackcdn.com/image/fetch/$s_!EovP!,f_auto,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F5ac3c4de-42fa-4d96-a5c4-f8edff920e6e_2912x1632.png" data-component-name="Image2ToDOM" rel=""><div><picture><source type="image/webp" srcset="https://substackcdn.com/image/fetch/$s_!EovP!,w_424,c_limit,f_webp,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F5ac3c4de-42fa-4d96-a5c4-f8edff920e6e_2912x1632.png 424w, https://substackcdn.com/image/fetch/$s_!EovP!,w_848,c_limit,f_webp,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F5ac3c4de-42fa-4d96-a5c4-f8edff920e6e_2912x1632.png 848w, https://substackcdn.com/image/fetch/$s_!EovP!,w_1272,c_limit,f_webp,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F5ac3c4de-42fa-4d96-a5c4-f8edff920e6e_2912x1632.png 1272w, https://substackcdn.com/image/fetch/$s_!EovP!,w_1456,c_limit,f_webp,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F5ac3c4de-42fa-4d96-a5c4-f8edff920e6e_2912x1632.png 1456w" sizes="100vw"/><img src="https://substackcdn.com/image/fetch/$s_!EovP!,w_2400,c_limit,f_auto,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F5ac3c4de-42fa-4d96-a5c4-f8edff920e6e_2912x1632.png" width="1200" height="672.5274725274726" data-attrs="{&#34;src&#34;:&#34;https://substack-post-media.s3.amazonaws.com/public/images/5ac3c4de-42fa-4d96-a5c4-f8edff920e6e_2912x1632.png&#34;,&#34;srcNoWatermark&#34;:null,&#34;fullscreen&#34;:false,&#34;imageSize&#34;:&#34;large&#34;,&#34;height&#34;:816,&#34;width&#34;:1456,&#34;resizeWidth&#34;:1200,&#34;bytes&#34;:7436874,&#34;alt&#34;:null,&#34;title&#34;:null,&#34;type&#34;:&#34;image/png&#34;,&#34;href&#34;:null,&#34;belowTheFold&#34;:false,&#34;topImage&#34;:true,&#34;internalRedirect&#34;:&#34;https://www.owlposting.com/i/160871483?img=https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F5ac3c4de-42fa-4d96-a5c4-f8edff920e6e_2912x1632.png&#34;,&#34;isProcessing&#34;:false,&#34;align&#34;:&#34;center&#34;,&#34;offset&#34;:false}" alt="" srcset="https://substackcdn.com/image/fetch/$s_!EovP!,w_424,c_limit,f_auto,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F5ac3c4de-42fa-4d96-a5c4-f8edff920e6e_2912x1632.png 424w, https://substackcdn.com/image/fetch/$s_!EovP!,w_848,c_limit,f_auto,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F5ac3c4de-42fa-4d96-a5c4-f8edff920e6e_2912x1632.png 848w, https://substackcdn.com/image/fetch/$s_!EovP!,w_1272,c_limit,f_auto,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F5ac3c4de-42fa-4d96-a5c4-f8edff920e6e_2912x1632.png 1272w, https://substackcdn.com/image/fetch/$s_!EovP!,w_1456,c_limit,f_auto,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F5ac3c4de-42fa-4d96-a5c4-f8edff920e6e_2912x1632.png 1456w" sizes="100vw" fetchpriority="high"/></picture><div><div></div></div></div></a></figure></div><p><em><span>Note: Thank you to</span><a href="https://www.linkedin.com/in/williambusa/overlay/about-this-profile/" rel=""> Bill Busa</a><span>, CEO and co-founder of EvE Bio, for an extremely helpful discussion while working on this essay.</span></em></p><p><em><span>This essay is long, and I recognize that many people don’t necessarily care about the details. The real headline point you need to be aware of is </span><a href="https://data.evebio.org/" rel="">this dataset</a><span>, which was produced by EvE Bio underneath a </span><a href="https://creativecommons.org/licenses/by-nc-sa/4.0/" rel="">CC-NA license</a><span>, and is a </span><strong>comprehensive mapping of the interactions between a significant fraction of clinically important human cellular receptors and 1,600~ FDA-approved drugs</strong><span>. I strongly believe that this data is really, really useful, and more people should be aware it exists. </span></em></p><p><em>If you’d like to understand why I think it is useful, and what the dataset exactly contains, read on! </em></p><ol><li><p><a href="https://www.owlposting.com/i/160871483/introduction" rel="">Introduction</a></p></li><li><p><a href="https://www.owlposting.com/i/160871483/why-is-understanding-off-target-effects-important" rel="">Why is understanding off-target effects important?</a></p><ol><li><p><a href="https://www.owlposting.com/i/160871483/drug-repurposing" rel="">Drug repurposing</a></p></li><li><p><a href="https://www.owlposting.com/i/160871483/validation-data-for-models" rel="">Validation data for models</a></p></li><li><p><a href="https://www.owlposting.com/i/160871483/maybe-polypharmacology" rel="">(Maybe) Polypharmacology</a></p></li></ol></li><li><p><a href="https://www.owlposting.com/i/160871483/how-do-you-understand-off-target-effects-in-a-tractable-way" rel="">How do you understand off-target effects in a tractable way?</a></p></li><li><p><a href="https://www.owlposting.com/i/160871483/why-hasnt-anyone-done-this-before" rel="">Why hasn’t anyone done this before?</a></p></li><li><p><a href="https://www.owlposting.com/i/160871483/what-does-the-future-look-like" rel="">What does the future look like?</a></p></li></ol><p><span>If you were to be a fly on the wall during the 1-6 years of preclinical drug discovery research within a pharmaceutical company, one observation you may walk away with is that, while the work is certainly complicated, it is also frighteningly limited in scope. What you’ll learn is that drugs are made by corporations that are optimizing for one primary thing, and one thing only: </span><strong>work</strong><span>. ‘Working’ is obviously contextual, but it is a simple concept no matter the situation: reduce a worrying biomarker, improve mood, lengthen lifespan and so on and so on. What does this discovery process ignore? Simply put: everything else a drug could do beyond that.</span></p><p><span>Yes, that’s a roundabout way of describing ‘off-target effects’ —  defined as the action of a drug at a gene product other than the gene product it was intended to affect — but I think it’s a helpful intuition pump. Viewing the drug discovery process as ‘</span><em><span>not paying attention to anything that is unrelated to the drug </span><strong>working</strong></em><span>’ is useful in that it contextualizes the situation we’re in. Drugs are meant to make money, and money is derived from </span><strong>drugs working</strong><span>. To spend time on understanding what else a particular drug does beyond It Working for its intended task is time lost and money lost.</span></p><p><span>One unfamiliar with the drug discovery process may find this bizarre; why wouldn’t the well meaning scientists in charge of developing drugs try to deeply understand how it interacts with the body? On the other hand, those deeply in the medical field would find this thesis so obvious that stating it is unnecessary; of </span><strong>course</strong><span> a pharmaceutical company would limit their scope of understanding a drug to things that lie between it working and not working. There’s only so much time and resources to go around. Priorities! </span></p><p><span>Of course, if an off-target effect comes between the drug and </span><strong>It Working</strong><span>, then certainly resources will be allocated to deal with it. But beyond that, mapping everything else a clinical-stage drug does — every receptor it unintentionally binds, every pathway it nudges sideways, every gene it perturbs slightly — is deemed so high effort and so low ROI, that it is relegated to hoping an academic will study it. Only if post-marketing surveillance turns up something worrying shall further exploration occur. Because, again, a deep understanding of </span><strong>what exactly an exogenous chemical is doing inside a body is not the point of the drug discovery process. </strong><span>Working is the point!</span></p><p>With that background context, I am ready to present three claims I’m going to make in this essay and spend the remaining sections trying to prove:</p><ol><li><p>Understanding off-target effects is really useful.</p></li><li><p>Learning about off-target effects at scale is possible.</p></li><li><p>No for-profit institution has a strong incentive to do this work.</p></li></ol><p>For the moment, let’s accept that these three are indeed true, and we can put our skeptic hat back on at the end of this section.</p><p><span>The subject of today’s essay is </span><strong><a href="https://evebio.org/" rel="">EvE Bio</a></strong><span>, and why I think they are doing something incredible.</span></p><p><span>EvE is a bit unlike the typical </span><a href="https://www.owlposting.com/s/startups" rel="">startups</a><span> I write about, because they aren’t really a startup. They are a FRO, or Focused Research Organization. Many reading this blog are likely already familiar with this recent renaissance of strange scientific organizations (</span><a href="https://www.owlposting.com/p/things-i-learned-talking-to-the-new" rel="">something I’ve written about in the past</a><span>) and already understand this acronym, but to those who don’t, this Venn diagram is quite instructive:</span></p><div><figure><a target="_blank" href="https://substackcdn.com/image/fetch/$s_!D_9w!,f_auto,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F010d6a6f-e7ee-4453-81e3-12dbc40545ad_1456x1211.jpeg" data-component-name="Image2ToDOM" rel=""><div><picture><source type="image/webp" srcset="https://substackcdn.com/image/fetch/$s_!D_9w!,w_424,c_limit,f_webp,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F010d6a6f-e7ee-4453-81e3-12dbc40545ad_1456x1211.jpeg 424w, https://substackcdn.com/image/fetch/$s_!D_9w!,w_848,c_limit,f_webp,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F010d6a6f-e7ee-4453-81e3-12dbc40545ad_1456x1211.jpeg 848w, https://substackcdn.com/image/fetch/$s_!D_9w!,w_1272,c_limit,f_webp,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F010d6a6f-e7ee-4453-81e3-12dbc40545ad_1456x1211.jpeg 1272w, https://substackcdn.com/image/fetch/$s_!D_9w!,w_1456,c_limit,f_webp,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F010d6a6f-e7ee-4453-81e3-12dbc40545ad_1456x1211.jpeg 1456w" sizes="100vw"/><img src="https://substackcdn.com/image/fetch/$s_!D_9w!,w_1456,c_limit,f_auto,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F010d6a6f-e7ee-4453-81e3-12dbc40545ad_1456x1211.jpeg" width="617" height="513.1778846153846" data-attrs="{&#34;src&#34;:&#34;https://substack-post-media.s3.amazonaws.com/public/images/010d6a6f-e7ee-4453-81e3-12dbc40545ad_1456x1211.jpeg&#34;,&#34;srcNoWatermark&#34;:null,&#34;fullscreen&#34;:null,&#34;imageSize&#34;:null,&#34;height&#34;:1211,&#34;width&#34;:1456,&#34;resizeWidth&#34;:617,&#34;bytes&#34;:null,&#34;alt&#34;:null,&#34;title&#34;:null,&#34;type&#34;:null,&#34;href&#34;:null,&#34;belowTheFold&#34;:true,&#34;topImage&#34;:false,&#34;internalRedirect&#34;:null,&#34;isProcessing&#34;:false,&#34;align&#34;:null,&#34;offset&#34;:false}" alt="" srcset="https://substackcdn.com/image/fetch/$s_!D_9w!,w_424,c_limit,f_auto,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F010d6a6f-e7ee-4453-81e3-12dbc40545ad_1456x1211.jpeg 424w, https://substackcdn.com/image/fetch/$s_!D_9w!,w_848,c_limit,f_auto,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F010d6a6f-e7ee-4453-81e3-12dbc40545ad_1456x1211.jpeg 848w, https://substackcdn.com/image/fetch/$s_!D_9w!,w_1272,c_limit,f_auto,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F010d6a6f-e7ee-4453-81e3-12dbc40545ad_1456x1211.jpeg 1272w, https://substackcdn.com/image/fetch/$s_!D_9w!,w_1456,c_limit,f_auto,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F010d6a6f-e7ee-4453-81e3-12dbc40545ad_1456x1211.jpeg 1456w" sizes="100vw" loading="lazy"/></picture><div><div></div></div></div></a></figure></div><p><span>Entire essays can (</span><a href="https://calvinball.substack.com/p/three-learnings-about-focused-research" rel="">and have been!</a><span>) written about the intricacies of FRO’s, but this essay will ignore much of their organizational structure, since it isn’t super relevant to what EvE is doing.</span></p><p><span>So what </span><strong>is</strong><span> EvE doing? EvE Bio is a scientific non-profit that has a clear, singular mission: </span><strong>map the off-target effects of every FDA-approved drug in existence and share the data.</strong><span> The data will be released underneath a non-commercial, creative commons license — free to use by academics, and available for licensing for commercial entities. Once they accomplish this task, they close up shop or spin off into their own thing. And if they don’t do it within 5-6 years, the same end result still happens. They do have some future plans that may come into the picture with time, which I’ll cover at the end, but the bolded bit is their primary thesis!</span></p><p>So why are they doing this? How will they do it? And why hasn’t anyone else done it yet? </p><p><span>There is a lazy answer that could be given here: &#34;</span><em>because we want to know if potential side effects of a drug exist</em><span>&#34;. This is partially correct, but I think it pays to be more specific. On EvE’s website, they list six reasons why off-target effects are worth studying:</span></p><div><figure><a target="_blank" href="https://substackcdn.com/image/fetch/$s_!x35K!,f_auto,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2Fe7eb11aa-9d96-4b36-8720-11d8cdffdc6f_1456x1047.jpeg" data-component-name="Image2ToDOM" rel=""><div><picture><source type="image/webp" srcset="https://substackcdn.com/image/fetch/$s_!x35K!,w_424,c_limit,f_webp,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2Fe7eb11aa-9d96-4b36-8720-11d8cdffdc6f_1456x1047.jpeg 424w, https://substackcdn.com/image/fetch/$s_!x35K!,w_848,c_limit,f_webp,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2Fe7eb11aa-9d96-4b36-8720-11d8cdffdc6f_1456x1047.jpeg 848w, https://substackcdn.com/image/fetch/$s_!x35K!,w_1272,c_limit,f_webp,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2Fe7eb11aa-9d96-4b36-8720-11d8cdffdc6f_1456x1047.jpeg 1272w, https://substackcdn.com/image/fetch/$s_!x35K!,w_1456,c_limit,f_webp,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2Fe7eb11aa-9d96-4b36-8720-11d8cdffdc6f_1456x1047.jpeg 1456w" sizes="100vw"/><img src="https://substackcdn.com/image/fetch/$s_!x35K!,w_1456,c_limit,f_auto,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2Fe7eb11aa-9d96-4b36-8720-11d8cdffdc6f_1456x1047.jpeg" width="1456" height="1047" data-attrs="{&#34;src&#34;:&#34;https://substack-post-media.s3.amazonaws.com/public/images/e7eb11aa-9d96-4b36-8720-11d8cdffdc6f_1456x1047.jpeg&#34;,&#34;srcNoWatermark&#34;:null,&#34;fullscreen&#34;:null,&#34;imageSize&#34;:null,&#34;height&#34;:1047,&#34;width&#34;:1456,&#34;resizeWidth&#34;:null,&#34;bytes&#34;:null,&#34;alt&#34;:null,&#34;title&#34;:null,&#34;type&#34;:null,&#34;href&#34;:null,&#34;belowTheFold&#34;:true,&#34;topImage&#34;:false,&#34;internalRedirect&#34;:null,&#34;isProcessing&#34;:false,&#34;align&#34;:null,&#34;offset&#34;:false}" alt="" srcset="https://substackcdn.com/image/fetch/$s_!x35K!,w_424,c_limit,f_auto,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2Fe7eb11aa-9d96-4b36-8720-11d8cdffdc6f_1456x1047.jpeg 424w, https://substackcdn.com/image/fetch/$s_!x35K!,w_848,c_limit,f_auto,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2Fe7eb11aa-9d96-4b36-8720-11d8cdffdc6f_1456x1047.jpeg 848w, https://substackcdn.com/image/fetch/$s_!x35K!,w_1272,c_limit,f_auto,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2Fe7eb11aa-9d96-4b36-8720-11d8cdffdc6f_1456x1047.jpeg 1272w, https://substackcdn.com/image/fetch/$s_!x35K!,w_1456,c_limit,f_auto,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2Fe7eb11aa-9d96-4b36-8720-11d8cdffdc6f_1456x1047.jpeg 1456w" sizes="100vw" loading="lazy"/></picture><div><div></div></div></div></a></figure></div><p><span>Now, fairly, some of these are at least a </span><strong>little</strong><span> fluffy. Is the institution doing off-target mapping really going to be the ones developing the autonomous lab assays of the future? Maybe! But it feels like a third-order, fourth-order, or even further consequence of their main mission. Bill did mention to me that there are already promising results in that direction, such as better reporting cell lines, but still. I think it’s generally good to limit ones assessment of an institution based on what their first-order impact will be, and, there, I think there will be three distinct areas that EvE will service</span><strong>: drug repurposing, validation for machine-learning models, </strong><span>and</span><strong> </strong><span>to a weaker degree</span><strong>, polypharmacology.</strong></p><p><span>What about industrial chemical profiling and pharmacology profiling? I think EvE will certainly be important there, but it’s a bit fuzzier. Industrial chemical profiling may occur in the future but isn’t part of the current cohort of FDA-approved drugs that EvE is focusing on, and there’s a similar problem for pharmacology profiling as there is for ML-for-toxicity (</span><a href="https://www.owlposting.com/p/a-primer-on-why-computational-predictive" rel="">which I have written about before as being a challenging proposition</a><span>).</span></p><p><span>But even if we take my somewhat pessimistic stance that only three of these six things are genuinely tractable in the short term, </span><strong>those areas alone are extremely valuable.</strong><span> Let’s go over them.</span></p><p><span>I think it is under-appreciated just how </span><strong>rich</strong><span> the cohort of FDA-approved drugs that are out there. Consider the fact that basically all drugs start off with singular indications, meant to cure, alleviate, or address </span><strong>one</strong><span> thing.</span><a href="https://link.springer.com/article/10.1186/s40545-020-00282-8" rel=""> Yet, 30%~ of FDA-approved drugs gain a new post-approval indication,</a><span> based on a study of the 197 drugs approved by the FDA from 1997-2020. Funnily enough, the same paper that came up with that 30% number almost treats it as a matter of disappointment, </span><strong>given that 38% of all prescriptions written in the US are off-label!</strong><span> This implies that there are, potentially, hundreds of drugs that are </span><em>already</em><span> being used beyond their original scope, just without the formal validation or regulatory blessing. Which, in turn, implies that we’re sitting on a vast, under-explored landscape of therapeutic potential, one that clinicians are already intuitively poking into, but which the formal system has barely begun to chart.</span></p><p><span>Now, I think some caution is warranted. This 38% number does vary from paper to paper,</span><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC11892052/" rel=""> one other study claims off-label prescriptions are as low as 25%.</a><span> If we’re being even </span><strong>more</strong><span> fair, it’s questionable exactly how proven-out these off-label indications are.</span><a href="https://jamanetwork.com/journals/jamainternalmedicine/fullarticle/410250" rel=""> One 2006 study claims that of the 21% of off-label prescriptions</a><span> they found, 73% of them had little-to-no scientific support. Hard to tell whether this is because there simply are no studies, or because the off-label usage was actively disproved!</span></p><p><span>Consider gabapentin, one of the most egregious cases of off-label drug prescriptions. Typically, most people view gabapentin as the nerve injury drug, right? But it, in fact, was not originally approved for that, only for seizures!</span><a href="https://lowninstitute.org/does-the-benefit-of-off-label-gabapentinoid-use-outweigh-the-harms/" rel=""> Yet, 95% of its prescriptions usage are for pain;</a><span> nerve pain, low-back pain, post-operative pain, and so on. But while the gabapentin is indeed effective for some specific types of nerve pain (diabetic neuropathy), it is ineffective for many other types (e.g sciatica), as confirmed by</span><a href="https://en.wikipedia.org/wiki/Gabapentin" rel=""> follow-up studies by Pfizer</a><span>.</span></p><p>Yet, prescriptions for these ineffective off-label usages continue.</p><p><span>But even if the true rate of valid, effective off-label use is lower than we’d like to imagine, the value of actually stumbling across a chance to repurpose a drug is</span><strong> </strong><span>high enough as to almost certainly still be worth it! Why? New chemical entities must follow the typical clinical phase progression timeline, whereas any repurposed drugs can skip preclinical, phase 1, and (sometimes)</span><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10627937/" rel=""> phase 2 trials as a result of their already-collected toxicity data</a><span>. Billions of dollars and years of time could be saved! </span></p><p><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC5694537/" rel="">From a 2017 review paper.</a></p><blockquote><p><em><span>…repurposed drugs are generally approved sooner (3–12 years) and at reduced (50–60%) cost (</span><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC5694537/#B5" rel="">5</a><span>,</span><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC5694537/#B6" rel=""> 6</a><span>). In addition, while ~10% of new drug applications gain market approval, approximately 30% of repurposed drugs are approved, giving companies a market-driven incentive to repurpose existing assets (</span><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC5694537/#B5" rel="">5</a><span>)….</span></em></p><p><em><span>For example, repurposing of the emergency contraceptive, mifepristone, for Cushing’s syndrome required a cohort of less than 30 patients to test its efficacy, whereas a clinical trial</span><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC5694537/#fn1" rel=""><sup>1</sup></a><span> for the same indication evaluating the safety and efficacy of a new chemical entity, levoketoconazole, required ~90 individuals (</span><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC5694537/#B2" rel="">2</a><span>,</span><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC5694537/#B3" rel=""> 3</a><span>)…..</span></em></p></blockquote><p>But, as it stands today, most drug repurposing efforts are done somewhat blindly; haphazardly glancing through the literature, relying on anecdotal case reports, or waiting for some academic lab to publish a five-mouse study from 2013 that hints at a secondary use. In many ways, it isn’t too dissimilar to the usual drug-discovery process! Given how promising (and relatively limited) the list of FDA-approved drugs are, the simple act of a pre-triaged list of drug-target maps (EvE’s mission!) may be extraordinarily impactful.</p><p><span>In such a world where this data is easily accessible, perhaps an order of magnitude more energy would be devoted to repurposing efforts,</span><a href="https://x.com/cremieuxrecueil/status/1921986782523978044" rel=""> maybe vastly improving the currently horrific finances of modern day drug discovery.</a></p><p><span>But as with all seeming free-lunches, there’s a reason drug repurposing hasn’t been aggressively exploited beyond a few cases: economics. Unlike novel drugs, which come with fresh patents and a full runway of exclusivity, repurposed drugs necessarily rely on compounds whose original patents have expired or are near expiration</span><span>. This limits the sponsor’s ability to recoup development costs, because generic competition can quickly erode any profits once the drug hits the market, even if it’s approved for a new use. There </span><em>are</em><span> mechanisms to extend exclusivity for repurposed indications — such as the</span><a href="https://www.fda.gov/industry/medical-products-rare-diseases-and-conditions/designating-orphan-product-drugs-and-biological-products" rel=""> 7-year exclusivity period given by the FDA’s Orphan Drug Act for treatments of rare diseases</a><span> or the</span><a href="https://www.mayerbrown.com/-/media/files/perspectives-events/events/2023/05/fda-lifecycle-management-webinar-3year-new-clinical-investigation-exclusivity--may-11-2023-final.pdf%3Frev=478b889bb39944e086c77a3c8de4faeb" rel=""> 3-year-exclusivity granted in cases where new clinical data was needed to repurpose a drug</a><span> — but it is a risky enough bet that most companies will shy away from it.</span></p><p><span>But as EvE is a non-profit, the economics don’t </span><strong>need</strong><span> to make sense. They plan to periodically announce opportunities for repurposing to the world, in hopes that other well-meaning non-profits take it on or, if the evidence is sufficiently convincing, that doctors simply take it as a useful datapoint for deciding whether an off-label prescription may be useful. And if they do most of the legwork in identifying good candidates for repurposing, it may even make the economics worth it for for-profit entities to pursue further. </span></p><p><span>One of the easiest ways to assure yourself that what you’re doing is valuable is if people come up to you and ask if they could use whatever you’re producing. This is true in typical SaaS products, and it is true for the fruits of R&amp;D work. But beyond assessing value outright, it also helps you learn </span><strong>what</strong><span> your work is most valuable for.</span></p><p><span>And, curiously, the primary area in which EvE has found ‘product market fit’ is in companies asking to use their data for internal model validation efforts. As I mentioned before, while EvE’s dataset is free-to-use by academics, it requires a commercial license to be used by any for-profit entity. And they are currently in discussions with 4 such commercial entities, all of whom desire to use EvE’s dataset to </span><strong>validate their machine-learning models predictions.</strong></p><p><span>Historically, model builders in drug discovery have had to make do with whatever internal datasets they could get their hands on, which were typically limited in scope, biased toward certain classes of molecules, or simply not reproducible. Public data from sources like</span><a href="https://www.ebi.ac.uk/chembl/" rel=""> ChEMBL</a><span>,</span><a href="https://www.bindingdb.org/rwd/bind/index.jsp" rel=""> BindingDB</a><span>, or</span><a href="https://pubchem.ncbi.nlm.nih.gov/docs/bioassays" rel=""> PubChem BioAssay</a><span> are much larger in size, but they tend to be noisy, heterogeneous in experimental methodology, and </span><strong>always</strong><span> lack negative results. Worse, they’re often cherry-picked around success stories or clustered around well-studied targets, introducing systemic biases that hamper generalization. We need not look further than Pat Walter’s famous essay on the topic:</span><a href="https://practicalcheminformatics.blogspot.com/2023/08/we-need-better-benchmarks-for-machine.html" rel=""> We Need Better Benchmarks for Machine Learning in Drug Discovery</a><span>, which expands on these issues even more.</span></p><p>This is an area of EvE’s work that I cannot personally shed much light on, and obviously, Bill cannot tell me the exact details on what the commercial entities are working on. But it was a surprising learning from our conversation that this particular topic is where public interest is most rapidly coalescing! Very excited to hear about more public statements they make in this area soon.</p><p>I do think this is the weakest, day-one value-add for EvE’s dataset. So take this section with a grain of salt! It just felt too interesting to not cover.</p><p><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10243259/" rel="">Polypharmacology is a drug discovery approach where a drug is designed to target multiple molecular targets</a><span>, instead of a more traditional single-target approach. It’s not a particularly new idea, most clinically useful drugs exhibit multi-target activity whether they were designed that way or not. But what’s changed in the past decade is the </span><strong>intentionality</strong><span>.</span></p><p><span>I think there are a lot of different arguments for the value of polypharmacology, the easiest one hinging on efficacy. There’s a very interesting story that could be told here about drugs that worked better because they modulated the activity of multiple receptors in parallel. A great, recent example is that of drugs that followed Ozempic. Ozempic simply targeted GLP-1, which reduces appetite and slows digestion. But the second-generation (e.g. Zepbound) also targeted GIP, which amplifies insulin response and regulates lipid metabolism differently in adipose tissue. The effects were incredible: </span><a href="https://www.nejm.org/doi/10.1056/NEJMoa2416394?" rel="">13.7% weight loss with Ozempic, 20.2% weight loss with Zepbound</a><span> over 48 weeks. Synergistic effects! The third generation (e.g. retatrutide) tacks on interactions with glucagon receptors — potentially increasing metabolic rate — </span><a href="https://www.nejm.org/doi/full/10.1056/NEJMoa2301972" rel="">with early phase 2 results looking once again promising. </a></p><p><span>But a more interesting place to start is the very similarly named concept of </span><strong>polypharmacy.</strong></p><p><span>Polypharmacy refers to the clinical practice of prescribing multiple drugs simultaneously (usually 5+), typically to manage complex or co-occurring conditions. It’s common in geriatrics, psychiatry, oncology, and increasingly just about everywhere else in medicine: </span><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10337167/" rel="">~17% of all adults in the US meet the definition for polypharmacy</a><span>. The logic is straightforward: most diseases aren’t governed by a single pathway, and so tackling them with a single drug is often insufficient. Instead, clinicians stack therapies: an ACE inhibitor for the blood pressure, a statin for the cholesterol, metformin for the glucose, a GLP-1 for the weight, and so on.</span></p><p><span>As you may expect, polypharmacy is </span><strong>awful</strong><span> on the patient&#39;s physiology.</span><a href="https://pubmed.ncbi.nlm.nih.gov/28251277/" rel=""> One study estimates that</a><span> nearly </span><strong>10% of hospital admissions among older adults</strong><span> are directly attributable to adverse drug events from polypharmacy-related side effects. The more drugs we stack onto people, the more unpredictable the net interaction becomes, because even if each one has been individually safety-tested, nobody tests all the pairwise combinations in a clinically realistic setting.</span></p><p>The solution may very well be to bundle things up.</p><p><span>Rather than throwing five separately optimized molecules at a patient and hoping for cooperative behavior, we could, in principle, design a </span><strong>single </strong><span>molecule that alone engages the same therapeutic targets. This, in turn, allows clinical trials to suss out the net effect of such a drug in a controlled, interpretable way. Which naturally leads us to the utility of polypharmacology; not necessarily because it will give us magic drugs with efficacy far better than current ones (though it may!), but rather that it will simply avoid us having to deal with the current issues that polypharmacy presents.</span></p><p>But the obvious question: does EvE’s dataset help with polypharmacology efforts? There isn’t any current, empirical proof of this, but I think it will. If you squint, you could see it functioning as missing infrastructure, a dataset that is necessary for rational polypharmacology to occur at scale. But this is necessarily tied up with machine-learning for chemical design accelerating, so, again, this is not necessarily something I’d expect EvE’s work to contribute to by the end of the year. But perhaps soon!</p><p>This all said, even if you agreed that the value proposition that EvE is claiming is real, you may struggle to verbalize exactly how you would understand the off-target effects of the 13,000~ FDA approved drugs out there. What assays would you use? How do you dose any given drug? How do you understand the translation of your assay to real-world settings?</p><p>Let’s walk through the EvE workflow.</p><p>First, you need to decide what drugs you&#39;re actually going to test. While there are technically around 13,000 FDA-approved drugs out there, many of them aren&#39;t particularly relevant for this kind of screening. You can immediately exclude things like topical medications, inhalants, radioisotopes, and simple nutrients, stuff that is known to be largely innocuous or not have much systemic impact. After this initial filtering, you end up with about 1,600 small molecule drugs that are worth investigating. But this number gets further whittled down further based on practical constraints; availability, cost, licensing requirements, etc.</p><p>From this, EvE ended up with a library of 1,397 compounds to screen.</p><p><span>Then comes the harder question: what exactly are you screening against? The human body has somewhere around 20,000 protein-coding genes, and there is an argument that any drug could interact with any of them. But perhaps we’d be too zealous to immediately do an </span><em>(everything x everything)</em><span> screen. Shouldn’t we try to do something that’s closer to the Pareto optimal frontier? What if we suspect that the vast majority of clinically meaningful drug interactions occur with a tiny subset of those 20,000 genes?</span></p><p>And, indeed, that turns out to be the case.</p><p><span>The vast majority of genes have some nominal physiological function, yes, but when it comes to drug interactions, only a minority are commonly targeted. At least a minority of classes: </span><strong>nuclear receptors</strong><span> </span><strong>(NRs)</strong><span> and </span><strong>7-transmembrane receptors (also known as GPCRs). </strong><span>In total, there are about 800~ GPCRs and 48 NRs, but only</span><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC11089380/" rel=""> 110 GPCR’s</a><span> and</span><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC11279859/" rel=""> 12-13 NR’s</a><span> are actually targeted by drugs. Per last count, EvE has currently created data for</span><a href="https://data.evebio.org/data_explorer__3.html#targets" rel=""> 56 GPCR’s and 29 NR’s</a><span>. Over the course of their existence, they plan to cover, in total, a select set of the 200 GPCR’s and all 48 NR’s. Why not all 800 GPCR’s? I attached that information in the footnotes.</span><span> </span></p><p>They hope to do much more than this too, but we’ll cover that in the last section.</p><p><span>Both NRs and GPCRs have some nice properties, but most pertinent to EvE, they are known to be very ‘druggable’ classes of drugs, given that the cell often uses them to convey information from the outside world, and evolution has therefore made their binding pockets unusually receptive to small molecules.</span><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC8233523/" rel=""> GPCRs, sitting on the cell surface, are natural sensors for hormones, neurotransmitters, and other circulating ligands</a><span>, many of which resemble or inspire drug scaffolds</span><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC3677810/" rel="">. NRs, meanwhile, act as intracellular switches that come with protected internal pockets</a><span> meant to bind to estrogen, cortisol, and so on, making them ideal for selective small-molecule engagement. As a result, both are involved in a </span><strong>lot</strong><span> of important physiological processes.</span></p><p><span>This ‘physiological importance’ is useful in two ways! One, a plurality of drugs target the two —</span><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10252477/" rel=""> 13% of FDA-approved drugs target NR’s</a><span>, with that number jumping to</span><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC5820538/" rel=""> 35% for GPCR’s</a><span> — so mapping the interactions here may give a clinically meaningful view of off-target effects. And two, given the extreme importance of GPCR’s and NR’s in modern-day drug development, there has been a fair bit of work in improving how we study their interactions with ligands of interest. As in, new assays outright shouldn’t need to be developed to study them.</span></p><p><span>Speaking of that, let’s start talking about </span><strong>how</strong><span> they are building this drug x receptor interaction map. They rely on two well-established assays which I’ll discuss here, but feel free to skip, understanding the two isn’t particularly important.</span></p><ol><li><p><strong>TR-FRET-based co-factor recruitment assays for NRs</strong></p><ol><li><p><span>When a drug successfully activates an NR, it usually causes a conformational shift that allows the receptor to recruit a specific co-factor protein, exposing what is often called the ‘AF2 domain’</span><strong>. These co-factors tend to have little peptide motifs (like an LXXLL motif) that latch onto that domain.</strong></p></li><li><p><span>TF-FRET exploits this. A chemical is tagged onto the NR domain and a chemical is tagged onto the co-factor protein, both of which are fluorophores. </span><strong>If the FDA-approved drug is an agonist, you’ll see a spike of light appear as the two fluorophores interact.</strong></p></li></ol></li><li><p><strong>Tango β-arrestin recruitment assays for 7TMs/GPCRs</strong></p><ol><li><p><span>Instead of recruiting co-factors inside the nucleus like NR’s, GPCR’s sit on the surface of a cell and transmit signals inward. As the name of the protein class implies, this involves utilizing</span><a href="https://en.wikipedia.org/wiki/G_protein" rel=""> G-proteins</a><span>. Unfortunately, G-proteins are quite specific to </span><strong>their</strong><span> GPCR, so using them in our assay as a way to understand activation would be difficult to scale. Luckily, there is a nearly universal binding protein:</span><a href="https://en.wikipedia.org/wiki/Arrestin_beta_1" rel=""> β-arrestin</a><span>. When a GPCR is activated by [something], their signaling process almost always involves binding to that protein.</span></p></li><li><p><span>In the assay, the GPCR (attached to a cell surface) is engineered to have a built-in “trap”, a little molecular tag connected to a transcription factor. When β-arrestin is recruited, it brings along a protease that snips the tag, releasing the transcription factor. That transcription factor then moves into the nucleus and turns on a reporter gene, which encodes for the enzyme β-lactamase. Meanwhile, the cell is loaded with </span><strong>CCF4-AM</strong><span>, a fluorescent substrate that shifts its emission profile when cleaved by β-lactamase. </span><strong>The stronger the GPCR activation by a drug, the more β-lactamase is produced, the more substrate is cleaved, and the bigger the fluorescence shift.</strong><span> </span><strong>That shift, measured as a ratio between ‘starting’ and ‘ending’ wavelengths, serves as a readout of how strongly the receptor was activated.</strong></p></li></ol></li></ol><p>Reasonably simple! One note: the explanations I gave above is for assessing the difference between an inactive drug and an agonist. For assessing inactive versus antagonist, a separate experiment is run with a known ligand included.</p><p><span>Well, wait a minute. Aren’t we missing something? Off-target effects of a small molecule can be summarized purely by these GPCR/NR measurements, but we’d be failing to capture something else that is of vital importance: whether the drug outright kills the cell. One could imagine this also affecting our receptor experiments! Perhaps a drug is an antagonist and there is no color shift, or perhaps the cell is just </span><strong>dead</strong><span>, and nothing is being expressed at all. Conversely, a drug might look like an agonist due to signal drift as the cell’s internal environment falls apart.</span></p><p><strong>EvE solves this in a pragmatic way: run a third assay which measures how healthy the cell is</strong><span>. How do you measure that? Well, one good proxy for how functional a cell is </span><strong>ATP production</strong><span>. Metabolically active cells generate ATP to power all their intracellular processes. Dead ones don’t. The assay EvE uses is called </span><strong>CellTiter-Glo</strong><span>. It works by adding a reagent that causes a fluorescent reaction in the presence of ATP. More ATP, more light. Less ATP, less light. No ATP? No light (and likely dead). Again, simple!</span></p><p><span>Is that all? One last thing: accounting for pan-assay interference compounds, or PAINS. These are molecules that often give false positives in high-throughput screening regimes. This can occur for </span><strong>many</strong><span> different reasons, but one relevant example is if a molecule itself is a fluorophore, leading to us falsely believing that it is an agonist during a run. EvE simply tracks how often a drug is leading to positive results, and flags it in their results if they believe it is a PAINS.</span></p><p>So they run these three assays across their pairwise (drug x receptor) combinations, producing readouts at multiple different concentrations with replicates for each one.</p><p><span>I’m going to skip over a lot at this point. EvE clearly put an immense amount of work into QA’ing this process and filtering through the data, and I think I would do both a disservice and detract from the point of this essay if I were to attempt to repeat it here. Summarizing it all down, using a complex logic table</span><a href="https://data.evebio.org/data_explorer__4b.html#methods" rel=""> detailed here in Fig 7</a><span>, EvE assigns 1 of 4 categories to each (drug x receptor) combination:</span></p><ol><li><p><strong>Inactive.</strong><span> Drug likely has no effect on the receptor, across all tested concentrations. Maybe it doesn’t bind or maybe it binds but does nothing.</span></p></li><li><p><strong>Likely Inactive.</strong><span> A little more ambiguous, perhaps there’s a single noisy point above baseline, but nothing more.</span></p></li><li><p><strong>Active – Unquantified.</strong><span> Something is happening, since there’s reproducible activity, but not enough clean data to fit a proper dose-response curve.</span></p></li><li><p><strong>Active – Quantified</strong><span>. The drug produced a clear, dose-dependent response (as either an agonist or antagonist) with a well-behaved curve. From this, EvE fits a 4-parameter logistic model and extracts a pXC₅₀; the negative log concentration at which the drug produces half its maximal effect. </span></p></li></ol><p>And…that’s it. A clean, rigorous, and tractable approach to understanding off-target effects, across hundreds of receptors, at multiple concentrations, using multiple modes of detection, with full transparency around the data.</p><p><span>How far along is EvE on their mission? Circa their last data release on 5/7/2025, </span><strong>237,490 (drug x concentration x receptor) combinations have been screened, revealing 8 median agonists and 31 median antagonists per target. </strong><span>They run these experiments in 384 well plates, so that means they’ve run the process a little bit over 600~ times to generate their current dataset — though much of the current process is automated, very little human-done pipetting is going on.</span><a href="https://data.evebio.org/" rel=""> Data dumps of the data started in November 2024, with new ones dropping every few months.</a></p><p>I haven’t worked in a wet lab before, but I’ve been assured by at least one person I trust that the effort that went into assembling this all together is nothing short of extraordinary. But it is worth asking the question…</p><p><span>When assessing the value of a seeming scientific achievement, it’s usually good to step back and ask one question:</span><strong> why wasn’t this done a decade back?</strong></p><p>In some cases, the answer is boring: the technology wasn’t there yet to achieve it.</p><p><span>But here, the technology was almost certainly available! Eve’s assay for measuring NR activity has been around</span><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC3025134/" rel=""> at least since 2008</a><span>, and the one for GPCR</span><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC3014586/" rel=""> since 2010</a><span>, maybe even earlier for both. If it’s really </span><strong>that</strong><span> useful, why did it take so long for someone to start assembling this drug x receptor mapping together?</span></p><p><span>Haven’t I already given away this answer? In the introduction, I implied that pharma groups have no direct financial incentive to create such a dataset. And that is true to some degree, especially for smaller therapeutic companies that have bigger issues to focus on, </span><strong>but is that true for big pharma?</strong><span> A small slice of the billions in pharma spending couldn’t be sliced off to hand over to an internal research team? It’s not as if the data wouldn’t be useful for their </span><strong>own</strong><span> drug development pipelines. After all, off-target effects are among the most common reasons for late-stage trial failures and post-approval black box warnings, and even if the creation of an EvE-like dataset doesn’t fix the problem, I can’t imagine it’d hurt.</span></p><p><span>I should be fair: pharma companies do indeed do </span><strong>some</strong><span> of this. EvE’s own blog discusses this a little,</span><a href="https://www.nature.com/articles/s41573-024-00942-3" rel=""> referencing this paper:</a></p><blockquote><p><em>The report’s authors, luminaries in the discipline of safety pharmacology, surveyed 18 major pharmaceutical companies regarding the numbers and identities of potential off-targets against which they test each and every one of their new drug candidates in the interest of safety. The numbers ranged from a low of 11 to a high of 104 potential off-targets routinely profiled per company, with a median of about 45. Interestingly, the industry’s opinions regarding which potential off-targets to screen vary widely. The total number of potential off-targets screened, across the universe of all 18 pharmas, was 763, yet only 12% of them were screened by more than a third of those companies.</em></p></blockquote><p><span>So, yes, pharma companies do their own off-target screening. But, as we’ve discussed, this is a far cry from the universe of druggable receptors, and is only concentrated on </span><strong>their</strong><span> particular assets, not other ones. No attempt at creating a universal map!</span></p><p><span>But the same blogpost did reference another big pharma,</span><a href="https://www.nature.com/articles/s41467-023-40064-9" rel=""> Novartis, who also open-source</a><span> a </span><strong>much</strong><span> larger map:</span></p><blockquote><p><em>Novartis, who presented data collected “over a multi-year period” profiling drug/target interactions across a median of about 800 drugs per target and 105 gene product targets…</em></p></blockquote><p><span>This is impressive! One may imagine that if a big pharma was willing to release this, why does an entity like EvE need to exist? For interest&#39;s sake, let’s ignore the obvious answer of ‘</span><em>it is better for everyone if such a dataset is collected using a single, standardized protocol instead of compiled from unrelated experiments over years.</em><span>’</span></p><p>I asked Bill exactly this question, and the answer was a two-parter.</p><p><span>For one, the dataset that was collected by Novartis, and indeed every large-scale dataset that will ever be collected by big pharma, will always be limited by the constraint we mentioned at the start: everybody only cares about the drug working. </span><strong>A logical conclusion of this is that nearly every receptor covered in these sorts of screens is a safety-oriented receptor</strong><span>. Cytochrome P450, hERG, serotonin subtypes, dopamine D₂, and the like. These are important receptors, not because of how mechanistically interesting they are, but because they are </span><strong>dangerous</strong><span>.</span><a href="https://www.nature.com/articles/s41573-024-00942-3" rel=""> Indeed, the vast majority of screened receptors lie within the so-called Bowes-44 set</a><span>, which comes from a</span><a href="https://www.nature.com/articles/nrd3845" rel=""> 2012 paper that identified 44 receptors known to be often implicated in safety-related drug failures.</a><span> Though these do include NR’s and GPCR’s, it is a minimal set of them, as, again, the screening is not meant to assess how mechanistically interesting the receptors are.</span></p><p><span>And if a big pharma </span><strong>does</strong><span> decide to explore beyond the realm of safety-oriented receptors, they will almost certainly keep that dataset to themselves. Why release potential </span><a href="https://en.wikipedia.org/wiki/Alpha_(finance)" rel="">alpha</a><span> to competitors? Hence, why nothing quite like EvE has come out in the past and it is unlikely it ever will in the future, at least from a for-profit entity.</span></p><p><span>And two, EvE eventually hopes to cover a </span><strong>lot</strong><span> more ground than any of the publicly available datasets. Currently, yes, the Novartis dataset is larger than EvE’s, but it won’t be for long. In fact, their plans for the upcoming few years ended up being so interesting that I decided to split it off into another section:</span></p><p>EvE is still quite young, just over 2 years old, and I think the future of it is going to look really, really crazy. At the end of my startup coverage articles, I typically focus on commercial/scientific risks. But given that EvE is assured funding on a multi-year horizon without needing to care about market demands, it may be much more instructive (and interesting!) to instead discuss their upcoming plans.</p><p><span>Earlier I noted that EvE has currently released data for 29 NRs and 56 GPCRs, out of a planned 40 NR’s and 200 GPCR’s. In my conversation with Bill, I asked him how much time is left till the remaining ones are released. I expected the answer to be, optimistically, ‘</span><em>over the next few years</em><span>’, given how EvE only started to release data back in November 2024 </span><strong>and</strong><span> that the Novartis dataset collection process also took several years. </span><strong>I was astonished to learn that he expected to have released the remainder of all GPCR + NR screens dataset by the end of this year.</strong><span> Setting up the assays, validation, and automation was the hard part, which is why their data releases have only started recently. But now that that’s all set up, they simply must turn the crank to get the rest out of the door.</span></p><p><span>What’s next? Bill told me that the next target of receptors are kinases, 500~ or so receptors that have been</span><a href="https://www.nature.com/articles/s41573-024-00942-3" rel=""> increasingly valuable drug targets over the last 20 years.</a></p><p>Then what? Bill said he’s open to exploring even more drug targets, but he also said, surprisingly, that EvE may add more chemicals on top of the 1,600~ planned FDA-approved drugs. The FDA-approved drugs, he said, are success stories. Potentially it’d be even more interesting to consider the failures as well. Especially the ones that everybody expected to work, arrived at phase 3, and set billions of dollars on fire after the trial results came out.</p><p><span>Even more exotic options are also on the table. For example, Bill discussed exploring how metabolites of approved drugs interact with targets. Some context:</span><a href="https://www.nature.com/articles/s41573-024-00942-3" rel=""> most secondary pharmacology work stops at the parent compound</a><span>, but metabolic byproducts of a drug can have entirely different binding profiles, and, in some cases, they’re the ones responsible for efficacy (e.g codeine, which metabolizes into the much more effective morphine) </span><em>or</em><span> for toxicity (e.g. acetaminophen, which metabolizes into the very toxic</span><a href="https://en.wikipedia.org/wiki/NAPQI" rel=""> NAPQI</a><span>). He also mentioned potentially using EvE’s assaying work to develop our understanding of tool compounds, which are chemicals that don’t necessarily have therapeutic value themselves, but are used in research to probe specific biological pathways or validate target function.</span><a href="https://axial.acs.org/medicinal-chemistry/call-for-reviews-recommended-tool-compounds-for-biological-and-pharmacological-studies" rel=""> An ACS page has this to say about it:</a></p><blockquote><p><em>While tool compounds have tremendous potential for advancing life science research, they are broadly defined, and it is often difficult for a researcher to determine the best tool compounds to employ during the research process. There remains a great need for more tool compound databases and authoritative sources of information from experts in the field.</em></p></blockquote><p><a href="https://www.science.org/content/blog-post/tool-compound-s-new-personality" rel="">And, as always, there is a (very short) Derek Lowe piece</a><span> on how a commonly-relied upon tool compound moonlights as a ligand for a structurally unrelated receptor, likely muddying the literature the tiniest bit. More work here would almost certainly be deeply appreciated by those in the field.</span></p><p><span>Overall, EvE really exemplifies the thesis I put forward in a past essay about how</span><a href="https://www.owlposting.com/p/there-arent-enough-smart-people-in" rel=""> smart people in biology should do more boring things.</a><span> Very little that is directly sexy about doing an N x M screen, but the impact of doing something like it </span><strong>well</strong><span> can be immense. And I have little doubt that EvE Bio has been doing it well, and will continue to do so in their future projects. If you’re interested in checking out their dataset, check it out </span><a href="https://data.evebio.org/data_explorer__4b.html" rel="">here</a><span>.</span></p></div></div></div></article></div></div></div><div><div id="discussion"><div><h4>Discussion about this post</h4></div></div></div></div>
  </body>
</html>
