<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <title>James Routley | Feed</title>
    <link
      rel="stylesheet"
      type="text/css"
      href="../styles.css"
      media="screen"
    />
  </head>
  <body>
    <a href="/index.html">Back</a>
    <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4249026/">Original</a>
    <h1>Microbial Odor Profile of Polyester and Cotton Clothes After a Fitness Session</h1>
    
    <div id="readability-page-1" class="page"><div id="main-content">
    <article>
        <section>
         
    <ul>
            
                <li>
                    <a href="https://www.ncbi.nlm.nih.gov/pmc/journals/">Journal List</a>
                </li>
            
                <li>
                    <a href="https://www.ncbi.nlm.nih.gov/pmc/journals/83/">Appl Environ Microbiol</a>
                </li>
            
                <li>
                    <a href="https://www.ncbi.nlm.nih.gov/pmc/issues/245899/">v.80(21); 2014 Nov</a>
                </li>
            
                <li aria-current="page">
                    PMC4249026
                </li>
            
    </ul>
 

        </section>
        
  <p>
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.
    <a data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="https://www.ncbi.nlm.nih.gov/pmc/about/disclaimer/">
        Learn more about our disclaimer.
    </a>
</p>

        
        <section role="document">
            
                <div id="mc"><!--main-content--><div data-jigconfig="smoothScroll: false, allHeadingLevels: [&#39;h2&#39;], headingExclude: &#39;:hidden,.nomenu&#39;"><div><div><div><div><p><span id="pmcmata">Appl Environ Microbiol.</span> 2014 Nov; 80(21): 6611–6619. </p></div></div></div><p>H. Goodrich-Blair, <span>Editor</span></p></div><div id="abstract-1" lang="en"><h2 id="abstract-1title">Abstract</h2><!--article-meta--><p>Clothing textiles protect our human body against external factors. These textiles are not sterile and can harbor high bacterial counts as sweat and bacteria are transmitted from the skin. We investigated the microbial growth and odor development in cotton and synthetic clothing fabrics. T-shirts were collected from 26 healthy individuals after an intensive bicycle spinning session and incubated for 28 h before analysis. A trained odor panel determined significant differences between polyester versus cotton fabrics for the hedonic value, the intensity, and five qualitative odor characteristics. The polyester T-shirts smelled significantly less pleasant and more intense, compared to the cotton T-shirts. A dissimilar bacterial growth was found in cotton versus synthetic clothing textiles. Micrococci were isolated in almost all synthetic shirts and were detected almost solely on synthetic shirts by means of denaturing gradient gel electrophoresis fingerprinting. A selective enrichment of micrococci in an <em>in vitro</em> growth experiment confirmed the presence of these species on polyester. Staphylococci were abundant on both cotton and synthetic fabrics. Corynebacteria were not enriched on any textile type. This research found that the composition of clothing fibers promotes differential growth of textile microbes and, as such, determines possible malodor generation.</p></div><div id="sec1"><h2 id="sec1title">INTRODUCTION</h2><p>Clothing textiles are in close contact with the microorganisms of the skin and those of the environment. The clothes create a warm and often moist environment on the skin, which leads to the growth of bacteria. In some cases, these microorganisms lead to unpleasant odors, staining, fabric deterioration, and even physical irritation, such as skin allergies and skin infections (<a href="#B1" rid="B1">1</a>). The skin consists of various niches, each with its specific bacterial community present (<a href="#B2" rid="B2">2</a>, <a href="#B3" rid="B3">3</a>). Very dry areas, such as the forearm, trunk, and legs, harbor only 10<sup>2</sup> bacteria per cm<sup>2</sup>, while the axillae, umbilicus, and toe web spaces contain up to 10<sup>7</sup> bacteria per cm<sup>2</sup> (<a href="#B4" rid="B4">4</a>). The human skin contains up to 19 different phyla (<a href="#B5" rid="B5">5</a>) and even in one niche, the axillae, up to 9 different phyla are present (<a href="#B6" rid="B6">6</a>). Skin microorganisms transfer to the clothing fibers and interact with these in several phases: adherence, growth, and damage to the fibers. Growth of bacteria is due to sweat secretions, skin desquamation, natural particles present in the clothing fibers or on the fibers itself, or nutrition from elsewhere in the environment. An important factor determining bacterium-fiber interaction is the origin and the composition of the clothing textile. A large discrepancy exists in the way bacteria adhere to natural versus synthetic fibers. It is posed that natural fibers are more easily affected by the microbiota due to the natural nutrients present in the clothing and the ability to adsorb sweat components (<a href="#B1" rid="B1">1</a>). Cellulose fibers are degraded by a range of bacteria and fungi, possessing cellulolytic enzymes (<a href="#B7" rid="B7">7</a>). Synthetic fibers gather moisture in the free space between the fibers but do not adsorb it on the fibers themselves. Synthetic fibers are therefore less susceptible toward bacterial breakdown, also due to the polyethylene terephthalate (PET) basis of the fiber (<a href="#B1" rid="B1">1</a>).</p><p>Axillary malodor does not only emanate from the axillary skin but also from the textiles near the axillary region (<a href="#B8" rid="B8">8</a>, <a href="#B9" rid="B9">9</a>). Dravniek et al. (<a href="#B9" rid="B9">9</a>) refers to this as the primary odor, originating from the axilla itself, and the secondary odor, originating from clothing in contact with the axilla. The odor would then differ between the two sites (<a href="#B10" rid="B10">10</a>). It is found that a stronger body odor is generated by wearing synthetic clothing textiles compared to natural textiles (<a href="#B10" rid="B10">10</a>). This is held as a common belief; nevertheless, very few published data support this finding. Much research has nonetheless been conducted on controlling body odor by adding antimicrobials to textile fabrics (<a href="#B11" rid="B11">11</a>,<a href="#B12" rid="B12">–</a><a href="#B14" rid="B14">14</a>).</p><p><em>Corynebacterium</em> spp. are determined as the odor causing microorganisms in the human axilla (<a href="#B15" rid="B15">15</a>). It is yet unclear which microorganisms are associated with the odor formation in clothing textiles. Few studies have been performed on determining the microbiota living in clothes. Therefore, this research focuses on (i) the determination of the microbial communities living in clothes, (ii) determining whether different textiles host different communities, and (iii) determining the odor profile of different used fabrics after a sport session. This study focuses primarily on cotton (natural, consisting mainly of cellulose) versus polyester (synthetic) clothing textiles. An <em>in vivo</em> case study is performed on 26 healthy people, wearing 100% cotton, 100% polyester, and intermediate cotton/synthetic clothing, doing a bicycle spinning session for 1 h. A period of 28 h was left between fitness and odor assessment, in order to let the bacteria grow on the textiles. A selected and trained odor panel assessed the odor of the individual T-shirts. The bacterial community is analyzed by means of denaturing gradient gel electrophoresis (DGGE). An <em>in vitro</em> growth experiment is performed to analyze the selective enrichment of isolates on different clothing fabrics.</p></div><div id="sec2"><h2 id="sec2title">MATERIALS AND METHODS</h2><div id="sec2-1"><h3 id="sec2-1title">Study design. </h3><p>First, an <em>in vivo</em> experiment was conducted with 26 healthy subjects, wearing cotton, synthetic, and mixed cotton-synthetic T-shirts, participating in an intensive bicycle spinning session of 1 h. The T-shirts were collected, sealed in plastic bags, and stored at room temperature in the dark, so bacterial growth occurred. Axillary swabs were taken to analyze the bacterial community on the skin. Odor assessment by a trained odor panel and subsequent bacterial extraction was performed on the whole T-shirt. The individual samples were plated to obtain pure colonies for sequencing. The DNA was extracted from axillary and T-shirt samples and the microbial community was investigated by means of DGGE. Descriptive diversity and dynamics analysis was performed on the results. Second, an <em>in vitro</em> growth experiment was conducted in which typical skin/textile microbial isolates were incubated on a range of sterile textile fibers in order to identify the selective growth or inhibition on the textiles. Third, contact angle measurements were performed to detect the affinity of micrococci toward polyester and cotton textiles.</p></div><div id="sec2-2"><h3 id="sec2-2title">Sampling. </h3><p>Samples were taken from the T-shirt and the armpit skin of 26 healthy subjects (13 males and 13 females), participating in an intensive bicycle spinning session of 1 h. The median age was 39 years old (range, 20 to 60 years old) (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4249026/table/T1/" target="table" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>). Every subject wore a freshly washed T-shirt. All were in good health and had not received any antibiotics for at least 2 months. The participants had no history of dermatological disorders or other chronic medical disorders and had no current skin infections. No attempts were made to control the subjects&#39; diet or hygiene habits. All participants were residents living in the area of Willebroek (Belgium), with a temperate maritime climate by the North Sea and Atlantic Ocean. After 1 h of intensive bicycle spinning, the T-shirts were aseptically collected and separately sealed in plastic bags. The bags were kept at room temperature (20°C) in the dark for 28 h. This was done to simulate the home conditions and to let the microbial community grow on the specific clothing textiles. An axillary swab was taken from each participant, using a sterile cotton swab (Biolab, Belgium) that was formerly moistened with sterile physiological water. The swab was thoroughly swabbed for 15 s in the axillary region to detach and absorb the microorganisms, after which the tip was broken in a sterilized reaction tube filled with 1.0 ml of sterile physiological water (<a href="#B16" rid="B16">16</a>). The bacterial samples were pelletized and frozen at −20°C until DNA extraction.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div id="T1"><h3>TABLE 1</h3><!--caption a7--><p>Metadata of the participating subjects</p><div><table><thead><tr><th rowspan="1" colspan="1">Subject</th><th rowspan="1" colspan="1">Gender</th><th rowspan="1" colspan="1">Age (yr)</th><th rowspan="1" colspan="1">No. of washes/wk</th><th rowspan="1" colspan="1">No. of deo/wk<sup><em><sup>a</sup></em></sup></th><th rowspan="1" colspan="1">Textile type</th></tr></thead><tbody><tr><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">M</td><td rowspan="1" colspan="1">36</td><td rowspan="1" colspan="1">10</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">100% polyester</td></tr><tr><td rowspan="1" colspan="1">2</td><td rowspan="1" colspan="1">F</td><td rowspan="1" colspan="1">28</td><td rowspan="1" colspan="1">10</td><td rowspan="1" colspan="1">7</td><td rowspan="1" colspan="1">82% polyester + 18% elastane</td></tr><tr><td rowspan="1" colspan="1">3</td><td rowspan="1" colspan="1">M</td><td rowspan="1" colspan="1">29</td><td rowspan="1" colspan="1">12</td><td rowspan="1" colspan="1">7</td><td rowspan="1" colspan="1">100% cotton</td></tr><tr><td rowspan="1" colspan="1">4</td><td rowspan="1" colspan="1">M</td><td rowspan="1" colspan="1">52</td><td rowspan="1" colspan="1">7</td><td rowspan="1" colspan="1">7</td><td rowspan="1" colspan="1">100% cotton</td></tr><tr><td rowspan="1" colspan="1">6</td><td rowspan="1" colspan="1">M</td><td rowspan="1" colspan="1">40</td><td rowspan="1" colspan="1">7</td><td rowspan="1" colspan="1">7</td><td rowspan="1" colspan="1">100% polyester</td></tr><tr><td rowspan="1" colspan="1">8</td><td rowspan="1" colspan="1">M</td><td rowspan="1" colspan="1">44</td><td rowspan="1" colspan="1">9</td><td rowspan="1" colspan="1">7</td><td rowspan="1" colspan="1">100% polyester</td></tr><tr><td rowspan="1" colspan="1">9</td><td rowspan="1" colspan="1">M</td><td rowspan="1" colspan="1">36</td><td rowspan="1" colspan="1">7</td><td rowspan="1" colspan="1">10</td><td rowspan="1" colspan="1">100% polyester</td></tr><tr><td rowspan="1" colspan="1">10</td><td rowspan="1" colspan="1">F</td><td rowspan="1" colspan="1">43</td><td rowspan="1" colspan="1">7</td><td rowspan="1" colspan="1">7</td><td rowspan="1" colspan="1">100% cotton</td></tr><tr><td rowspan="1" colspan="1">11</td><td rowspan="1" colspan="1">M</td><td rowspan="1" colspan="1">42</td><td rowspan="1" colspan="1">7</td><td rowspan="1" colspan="1">9</td><td rowspan="1" colspan="1">100% polyester</td></tr><tr><td rowspan="1" colspan="1">12</td><td rowspan="1" colspan="1">M</td><td rowspan="1" colspan="1">32</td><td rowspan="1" colspan="1">7</td><td rowspan="1" colspan="1">0</td><td rowspan="1" colspan="1">100% polyester</td></tr><tr><td rowspan="1" colspan="1">13</td><td rowspan="1" colspan="1">F</td><td rowspan="1" colspan="1">35</td><td rowspan="1" colspan="1">7</td><td rowspan="1" colspan="1">0</td><td rowspan="1" colspan="1">100% polyester</td></tr><tr><td rowspan="1" colspan="1">14</td><td rowspan="1" colspan="1">F</td><td rowspan="1" colspan="1">42</td><td rowspan="1" colspan="1">7</td><td rowspan="1" colspan="1">0</td><td rowspan="1" colspan="1">100% cotton</td></tr><tr><td rowspan="1" colspan="1">15</td><td rowspan="1" colspan="1">F</td><td rowspan="1" colspan="1">41</td><td rowspan="1" colspan="1">7</td><td rowspan="1" colspan="1">10</td><td rowspan="1" colspan="1">34% cotton + 28% lyocell + 35% polyester + 3% elastane</td></tr><tr><td rowspan="1" colspan="1">16</td><td rowspan="1" colspan="1">F</td><td rowspan="1" colspan="1">60</td><td rowspan="1" colspan="1">7</td><td rowspan="1" colspan="1">14</td><td rowspan="1" colspan="1">95% cotton + 5% elastane</td></tr><tr><td rowspan="1" colspan="1">17</td><td rowspan="1" colspan="1">M</td><td rowspan="1" colspan="1">42</td><td rowspan="1" colspan="1">12</td><td rowspan="1" colspan="1">0</td><td rowspan="1" colspan="1">100% cotton</td></tr><tr><td rowspan="1" colspan="1">18</td><td rowspan="1" colspan="1">F</td><td rowspan="1" colspan="1">54</td><td rowspan="1" colspan="1">7</td><td rowspan="1" colspan="1">7</td><td rowspan="1" colspan="1">95% cotton + 5% elastane</td></tr><tr><td rowspan="1" colspan="1">19</td><td rowspan="1" colspan="1">M</td><td rowspan="1" colspan="1">21</td><td rowspan="1" colspan="1">7</td><td rowspan="1" colspan="1">10</td><td rowspan="1" colspan="1">100% cotton</td></tr><tr><td rowspan="1" colspan="1">20</td><td rowspan="1" colspan="1">M</td><td rowspan="1" colspan="1">56</td><td rowspan="1" colspan="1">7</td><td rowspan="1" colspan="1">7</td><td rowspan="1" colspan="1">100% cotton</td></tr><tr><td rowspan="1" colspan="1">21</td><td rowspan="1" colspan="1">F</td><td rowspan="1" colspan="1">30</td><td rowspan="1" colspan="1">7</td><td rowspan="1" colspan="1">9</td><td rowspan="1" colspan="1">95% cotton + 5% elastane</td></tr><tr><td rowspan="1" colspan="1">22</td><td rowspan="1" colspan="1">F</td><td rowspan="1" colspan="1">49</td><td rowspan="1" colspan="1">14</td><td rowspan="1" colspan="1">7</td><td rowspan="1" colspan="1">100% cotton</td></tr><tr><td rowspan="1" colspan="1">23</td><td rowspan="1" colspan="1">F</td><td rowspan="1" colspan="1">20</td><td rowspan="1" colspan="1">6</td><td rowspan="1" colspan="1">7</td><td rowspan="1" colspan="1">100% polyester</td></tr><tr><td rowspan="1" colspan="1">24</td><td rowspan="1" colspan="1">M</td><td rowspan="1" colspan="1">31</td><td rowspan="1" colspan="1">10</td><td rowspan="1" colspan="1">5</td><td rowspan="1" colspan="1">100% cotton</td></tr><tr><td rowspan="1" colspan="1">25</td><td rowspan="1" colspan="1">F</td><td rowspan="1" colspan="1">43</td><td rowspan="1" colspan="1">7</td><td rowspan="1" colspan="1">10</td><td rowspan="1" colspan="1">100% cotton</td></tr><tr><td rowspan="1" colspan="1">26</td><td rowspan="1" colspan="1">M</td><td rowspan="1" colspan="1">38</td><td rowspan="1" colspan="1">4</td><td rowspan="1" colspan="1">9</td><td rowspan="1" colspan="1">100% polyester</td></tr><tr><td rowspan="1" colspan="1">27</td><td rowspan="1" colspan="1">F</td><td rowspan="1" colspan="1">37</td><td rowspan="1" colspan="1">7</td><td rowspan="1" colspan="1">7</td><td rowspan="1" colspan="1">100% polyester</td></tr><tr><td rowspan="1" colspan="1">30</td><td rowspan="1" colspan="1">F</td><td rowspan="1" colspan="1">36</td><td rowspan="1" colspan="1">4</td><td rowspan="1" colspan="1">9</td><td rowspan="1" colspan="1">95% cotton + 5% elastane</td></tr></tbody></table></div></div></div><div id="sec2-3"><h3 id="sec2-3title">Odor assessment. </h3><p>Individual T-shirts in the plastic bags were presented to a panel of seven selected and screened human assessors. Assessors were selected by means of sensitivity to dilutions of <em>n</em>-butanol and wastewater and by means of the triangle test (<a href="#B17" rid="B17">17</a>). Each member of the panel was presented three flasks, two of which were the same while the third contained a different odor. The flask was shaken, the stopper was removed, after which the vapors were sniffed. The panelists had to correctly identify the different flask. The triangle test was repeated three times, with a minimum of 2 days in between each measurement. The room in which the tests were conducted was free from extraneous odor stimuli, e.g., such as odors caused by smoking, eating, soaps, perfume, etc. A representative team of odor assessors was chosen from the pool of assessors. The odor assessors were familiar with the olfactometric procedures and met the following conditions: (i) older than 16 years and willing to follow the instructions; (ii) no smoking, eating, drinking (except water), or using chewing gum or sweets for 30 min before olfactometric measurement; (iii) free from colds, allergies, or other infections; (iv) no interference by perfumes, deodorants, body lotions, cosmetics, or personal body odor; and (v) no communication during odor assessment. The samples were assessed by seven odor characteristics: hedonic value (between −4 and +4), intensity (scale 0 to 6), musty (scale 0 to 10), ammonia (scale 0 to 10), “strongness” (scale 0 to 10), sweatiness (scale 0 to 10), and sourness (scale 0 to 10). A control odor measurement, a clean cotton T-shirt with random number, was served to the odor panel together with the other samples.</p></div><div id="sec2-4"><h3 id="sec2-4title">Statistical analysis odor characteristics. </h3><p>The generated data set from the odor assessment was statistically analyzed and visualized in R (<a href="#B18" rid="B18">18</a>). A heat map and scatterplot were generated to visually interpret the correlations between sensory variables. Significance cutoff values were set at 95% (α = 0.05), unless otherwise mentioned in the manuscript. Both a multivariate comparison of means as well as univariate analysis were run after assessment of the hypothesis. Univariate normality was assessed using a Shapiro-Wilk normality test. If normality could not be assumed, the Mann-Whitney (or Wilcoxon rank sum) test was executed to assess null hypothesis of a location shift μ = 0. The alternative hypotheses were selected based upon exploratory data analysis. Nonavailable observations were handled by case-wise deletions. Multivariate data sets were analyzed on their normal distribution using Mahalanobis distances in quantile-quantile (QQ) plots. Also, an E-statistic test of multivariate normality was executed (<a href="#B19" rid="B19">19</a>). Multivariate homogeneity of group dispersions (variances) was assessed using the betadisper function from the package Vegan (<a href="#B20" rid="B20">20</a>), an implementation of the PERMDISP2 procedure (<a href="#B21" rid="B21">21</a>). Euclidean distance measures were used, as well as the spatial median for the group centroid. A Hotelling&#39;s T<sup>2</sup> test was used to compare the multivariate data sets, comparing the multivariate means of each population (<a href="#B22" rid="B22">22</a>). When necessary a chi-squared approximation was used for the test to allow for relaxation of the normality assumption.</p></div><div id="sec2-5"><h3 id="sec2-5title">Bacterial extraction from T-shirts. </h3><p>The bacterial extraction occurred on the complete T-shirt, using TNE buffer (10 mM Tris-HCl [pH 8.0], 10 mM NaCl, 10 mM EDTA) (<a href="#B23" rid="B23">23</a>). A 300-ml portion of TNE buffer was added to the plastic bag with the T-shirt, firmly sealed with tape, and vortexed for 10 min. The buffer was subsequently manually pressed out of the T-shirt and transferred into sterile 50-ml reaction tubes. The extracts were respectively used for isolation of bacteria and for DNA extraction. The bacterial extraction procedure was chosen after an optimization procedure (see Fig. S1 in the supplemental material). The method focused on the extraction of the bacteria of the whole T-shirt. It was not possible to extract the bacteria from one region (e.g., axillary region) of the T-shirt. A clean T-shirt was extracted, together with the other samples, as a control measurement.</p></div><div id="sec2-6"><h3 id="sec2-6title">Sanger sequencing of bacterial isolates. </h3><p>The microorganisms were isolated from the T-shirts by the standard method of dilution plating on nutrient agar. Incubation of all plates was performed at 37°C in aerobic conditions and facultative anaerobic conditions using a gas-pack cultivation jar. The colonies were plated three times on new agar plates using the streak plate method to obtain bacterial isolates. A total of 91 isolates was obtained. The isolates were transferred into a 1.5-ml Eppendorf with 50 μl of sterile PCR water, vortexed, and stored at −20°C to extract DNA. Dereplication was done using DGGE after amplification by PCR using the 338F and 518R primers (<a href="#B24" rid="B24">24</a>, <a href="#B25" rid="B25">25</a>). The analysis involved 31 nucleotide sequences. The 16S rRNA genes were subsequently amplified by PCR using 63F and 1378R (<a href="#B26" rid="B26">26</a>). The PCR program were performed and checked as described below. Sanger sequencing was performed on the 16S rRNA amplicons, aligned, and compared to sequences from the National Center for Biotechnology Information (NCBI) database. The closest match of each isolate was identified. The bacterial isolates were constructed in an evolutionary taxonomic circular tree (see <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4249026/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2</span></a>) using the neighbor-joining method (<a href="#B27" rid="B27">27</a>), conducted in MEGA5 (<a href="#B28" rid="B28">28</a>). The tree has branch lengths in the same units as those of the evolutionary distances used to infer the phylogenetic tree. The evolutionary distances were computed using the Jukes-Cantor method (<a href="#B29" rid="B29">29</a>) and are in the units of the number of base substitutions per site. The codon positions included were first + second + third + noncoding. All ambiguous positions were removed for each sequence pair. There were a total of 1,172 positions in the final data set.</p><!--fig ft0--><!--fig mode=article f1--><div id="F2" co-legend-rid="lgnd_F2"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4249026/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140103170140256"><a href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4249026_zam9991057430002.jpg" target="tileshopwindow" rel="noopener"><img loading="lazy" alt="An external file that holds a picture, illustration, etc.
Object name is zam9991057430002.jpg" title="Click on image to zoom" src="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4249026/bin/zam9991057430002.jpg"/></a></div><div id="lgnd_F2"><!--caption a7--><p>Bacterial isolates obtained from the T-shirts after the spinning session represented in an evolutionary taxonomic circular tree, using the neighbor-joining method.</p></div></div></div><div id="sec2-7"><h3 id="sec2-7title">DNA extraction, PCR, and DGGE. </h3><p>The bacterial solution in the TNE buffer was centrifuged for 5 min at 6,000 × <em>g</em>. The supernatant was discarded, and the obtained pellet was used for further DNA extraction. Total DNA extraction was performed using an UltraClean water DNA isolation kit (Mo Bio, USA). The DNA was stored at −20°C until further analysis. The DNA extraction was chosen after a comparative study of different DNA extraction methods (see Fig. S2 in the supplemental material). The 16S rRNA gene regions were amplified by PCR using 338F and 518R (<a href="#B24" rid="B24">24</a>, <a href="#B25" rid="B25">25</a>). A GC clamp of 40 bp (<a href="#B24" rid="B24">24</a>, <a href="#B25" rid="B25">25</a>) was added to the forward primer. The PCR program consisted of 10 min 95°C, followed by 35 cycles of 1 min at 94°C, 1 min at 53°C, and 2 min at 72°C, with a final elongation for 10 min at 72°C. Amplification products were analyzed by electrophoresis in 1.5% (wt/vol) agarose gels stained with ethidium bromide. DGGE was performed as previously reported (<a href="#B6" rid="B6">6</a>). A control measurement was taken into account. To process and compare the different gels, a homemade marker of different PCR fragments was loaded onto each gel (<a href="#B6" rid="B6">6</a>). Normalization and analysis of DGGE gel patterns was done with the BioNumerics software 5.10 (Applied Maths, Sint-Martens-Latem, Belgium). The different lanes were defined, the background was subtracted, differences in the intensity of the lanes were compensated for during normalization, and bands and band classes were detected.</p></div><div id="sec2-8"><h3 id="sec2-8title">Selective growth of bacteria on textiles. </h3><p>To analyze the selective growth of pure bacterial strains on different clothing textiles, bacteria were inoculated and incubated on a sterile piece of textile in an <em>in vitro</em> growth experiment. A wide range of clothing textiles was screened: polyester, acryl, nylon, fleece, viscose, cotton, and wool. Five common skin bacteria were grown on the textiles: <em>Staphylococcus epidermidis</em> CC6 (GenBank accession no. <a id="__tag_1137564884" href="https://www.ncbi.nlm.nih.gov/nuccore/KJ016246" ref="/nuccore/KJ016246">KJ016246</a>), <em>Micrococcus luteus</em> CC27 (GenBank accession no. <a id="__tag_1137564886" href="https://www.ncbi.nlm.nih.gov/nuccore/KJ016267" ref="/nuccore/KJ016267">KJ016267</a>), <em>Enhydrobacter aerosaccus</em> (LMG 21877), <em>Corynebacterium jeikeium</em> (LMG 19049), and <em>Propionibacterium acnes</em> (LMG 16711). The bacteria were cultivated for 48 h in nutrient broth, washed in M9 medium and finally dissolved in fresh M9 medium. A sterile piece of textile of 25 cm<sup>2</sup> was inoculated with 100 μl of the bacterial culture in a petri dish. The inoculated bacteria were incubated for 3 days at 37°C. The bacteria were subsequently extracted using 10 ml of TNE buffer (<a href="#B23" rid="B23">23</a>). The bacterial suspensions were measured using flow cytometry. To verify the extraction efficiency of the different clothing textiles, the bacterial strains were immediately extracted after inoculation using 10 ml of TNE buffer. All experiments were carried out in triplicate. A control measurement, where bacteria were grown without textiles, was each time taken into account and deducted from the measurements.</p></div><div id="sec2-9"><h3 id="sec2-9title">Flow cytometry. </h3><p>Flow cytometry was used as a fast microbial measurement technique. The laser detection point of the device beams one cell at the time (λ<sub>max</sub> = 488 nm), while the forward and side light scatter are detected. The samples were diluted 100 times in filtered Evian water (Danone Group, Paris, France) and stained with 1/100 SYBR green I dye (Invitrogen), as described in previous studies (<a href="#B30" rid="B30">30</a>). The DNA-dye complex absorbs blue light (λ<sub>max</sub> = 497 nm) and emits green light (λ<sub>max</sub> = 520 nm). Prior to flow cytometric analysis, the stained samples were incubated for 15 min in the dark at room temperature. Every sample was measured in triplicate, using a BD Accuri C6 flow cytometer (BD Biosciences, Belgium). The measurements were processed using the BD Accuri C6 software.</p></div><div id="sec2-10"><h3 id="sec2-10title">Contact angle measurements. </h3><p>The affinity of micrococci (<em>Micrococcus luteus</em>) toward specific clothing textiles (cotton and polyester) was measured by means of contact angle measurements on the fabrics and the micrococci, as described earlier (<a href="#B31" rid="B31">31</a>). Drops of three different solutes were applied on the tissues to determine Lifshitz-Van der Waals and electron-donor and -acceptor components of the surface tension, using the Young-Dupré equation and the extended DLVO approach (<a href="#B31" rid="B31">31</a>). The solutes (Milli-Q water, diiodomethane, and glycerol) had different physicochemical properties with known physicochemical parameters. Since the textile fabrics absorbed much moisture due to the large voids between the fibers, contact angles were carried out on substitute materials: PET plastic to simulate polyester fibers, since PET is the basic substance for polyester, and cardboard (cellulose) for cotton. <em>Micrococcus luteus</em> was cultivated in nutrient broth for 3 days at 37°C. The bacteria were filtered on a 0.45-μm-pore-size filter until a firm layer of micrococci was obtained, on which the contact angles were measured. Drop measurements were repeated at least 10 times for each liquid, whereby the average was taken. Anomalous measurements were rejected. All contact angles were measured using contact angle equipment (Krüss DSA10 goniometer; Krüss GmbH, Hamburg, Germany) equipped with contact angle calculation software (Drop Shape Analysis; Krüss GmbH).</p></div><div id="sec2-11"><h3 id="sec2-11title">Ethics statement. </h3><p>The study was approved by the Ghent University Ethical Committee with approval number B670201112035. All participants gave their written consent to participate in this study, as well as consent to publish these case details.</p></div><div id="sec2-12"><h3 id="sec2-12title">Nucleotide sequence accession numbers. </h3><p>Sequences for all of the strains were submitted to GenBank under accession numbers <a id="__tag_1137564885" href="https://www.ncbi.nlm.nih.gov/nuccore/KJ016241" ref="/nuccore/KJ016241">KJ016241</a> to <a id="__tag_1137564883" href="https://www.ncbi.nlm.nih.gov/nuccore/KJ016271" ref="/nuccore/KJ016271">KJ016271</a>.</p></div></div><div id="sec3"><h2 id="sec3title">RESULTS</h2><div id="sec3-1"><h3 id="sec3-1title">Odor differences between cotton and polyester clothing textiles. </h3><p>The hedonic value (i.e., the pleasantness of the odor) was qualified by the odor panel on a scale from −4 (very unpleasant) to +4 (very pleasant). The average hedonic value of 100% cotton T-shirts was −0.61 ± 1.08, while for 100% polyester T-shirts, a significantly lower value of −2.04 ± 0.90 was determined (see Table S1 in the supplemental material). Polyester clothing after the spinning session smelled significantly less pleasant, and additionally, more intense, more musty, more ammonia, more strong, more sweaty and more sour (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4249026/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1</span></a>). The qualitative differences were the largest for the sourness, strongness, and mustiness. The data set of the odor analysis was examined on its multivariate normal distribution by means of Mahalanobis QQ-plots (data not shown). Deviation from the bisector and, as such, from multivariate normality was observed, as confirmed formally by the E-statistic test (<em>P</em> &lt; 0.05). The multivariate means of cotton and polyester were compared to each other with the Hotelling two-sample T<sup>2</sup> test. This gave a <em>P</em> value of 5.72 × 10<sup>−6</sup>, meaning that a significant difference was found between the multivariate means of the cotton and polyester samples. The correlations between the different variables are visually represented in the heat map in Fig. S3 in the supplemental material. The <em>t</em> test indicated no differences in deodorant/antiperspirant use among the 100% cotton and 100% polyester group (<em>P</em> = 0.86) (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4249026/table/T1/" target="table" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div id="F1" co-legend-rid="lgnd_F1"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4249026/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140103176956192"><a href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4249026_zam9991057430001.jpg" target="tileshopwindow" rel="noopener"><img loading="lazy" alt="An external file that holds a picture, illustration, etc.
Object name is zam9991057430001.jpg" title="Click on image to zoom" src="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4249026/bin/zam9991057430001.jpg"/></a></div><div id="lgnd_F1"><!--caption a7--><p>Odor characterization of cotton (green) and polyester (red) clothing after a fitness experiment, assessed by the odor panel. The hedonic value was assessed between a value −4 (very unpleasant), 0 (neutral), and +4 (very pleasant) and rescaled between 0 and 8. The intensity represents the quantity of the odor, in a value between 0 (no odor) and 10 (very strong/intolerable). The qualitative odor characteristics musty, ammonia, strongness, sweatiness, and sourness were assessed between 0 and 10. The odor assessment is represented in box plots, with the middle black line as the median odor value and the small circles as the outliers. Polyester clothing smelled significantly more after a fitness session than cotton.</p></div></div></div><div id="sec3-2"><h3 id="sec3-2title">Bacterial isolation and identification. </h3><p>Isolates of pure bacterial colonies were identified and are represented in <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4249026/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2</span></a>. A total of 91 isolates was obtained from aerobic and anaerobic plating. The isolates were screened by DGGE and sequenced to allow identification. <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4249026/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2</span></a> represents 31 unique species found on the T-shirts. Not only Gram-positive but also many Gram-negative bacteria were found. Many skin-resident staphylococci were isolated from the textiles. Isolates also belonged to the Gram-positive <em>Bacillus</em> spp., Gram-positive <em>Micrococcus</em> spp., and Gram-negative <em>Acinetobacter</em> spp. and to the Gram-negative <em>Enterobacteriaceae</em> family, among others, which are generally not found on the axillary skin. The isolates were classified into three bacterial phyla: <em>Firmicutes</em>, <em>Actinobacteria</em>, and <em>Proteobacteria</em>.</p></div><div id="sec3-3"><h3 id="sec3-3title">Bacterial fingerprinting of the textile microbiome. </h3><p>DGGE fingerprinting analyses showed large diversities among the individual shirts. Although similar bacterial species were noticed, every textile microbiome was rather unique. <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4249026/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Figure 3</span></a> shows the fingerprinting results of the 26 individual T-shirts. Apparent differences were found between cotton and synthetic clothing textiles after the fitness session. Particular bands were identified that correlated more with specific clothing fibers. <em>Micrococcus</em> spp. were predominantly found in synthetic clothing fabrics. Many micrococci were found on 100% polyester clothes, but they were also on mixed synthetic textiles, such as 82% polyester plus 18% elastane. Micrococci were also found on mixed synthetic/natural textiles, such as 95% cotton + 5% elastane and 35% polyester + 34% cotton + 28% lyocell + 3% elastane (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4249026/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3</span></a>). <em>Staphylococcus hominis</em> bands were solely present on the 100% cotton clothing. <em>Staphylococcus</em> spp. were detected in relatively large amounts in practically all T-shirts. Individual DGGE fingerprinting was performed on both textiles and axillary skin (see Fig. S4 in the supplemental material). The axillary region was chosen as a representative skin area and compared to the textile microbiome, since both are known to generate malodor. Large differences were seen in the bacterial fingerprint patterns between the axillary and textile microbiome. An enrichment of skin bacteria on the textile was frequently observed, such as the apparent enrichment of <em>Staphylococcus epidermidis</em> (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4249026/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3</span></a>). The fingerprint results show that selective bacterial growth occurs in synthetic and cotton clothing.</p><!--fig ft0--><!--fig mode=article f1--><div id="F3" co-legend-rid="lgnd_F3"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4249026/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140103179550592"><a href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4249026_zam9991057430003.jpg" target="tileshopwindow" rel="noopener"><img loading="lazy" alt="An external file that holds a picture, illustration, etc.
Object name is zam9991057430003.jpg" title="Click on image to zoom" src="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4249026/bin/zam9991057430003.jpg"/></a></div><div id="lgnd_F3"><!--caption a7--><p>DGGE bacterial profile of 26 individual T-shirts after the bicycle spinning session. The legend on the right represents the subject number, and the textile fibers are indicated as follows: P, polyester; C, cotton; E, elastane; and L, lyocell. The samples were separated between cotton and synthetic clothing fibers.</p></div></div></div><div id="sec3-4"><h3 id="sec3-4title">Selective bacterial growth on clothing textiles. </h3><p>The selective growth of pure bacterial cultures was examined by means of an <em>in vitro</em> growth experiment on a range of different fabrics. The results, presented in <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4249026/table/T2/" target="table" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>, clearly indicated selective growth and inhibition for several species on the different fabrics. <em>Enhydrobacter aerosaccus</em> and <em>Propionibacterium acnes</em> were able to grow on almost every textile. Under the same conditions, <em>Corynebacterium jeikeium</em> was not able to grow on the textiles, as the log counts decreased. <em>Staphylococcus epidermidis</em> was able to grow on almost every textile, except viscose and fleece. <em>Propionibacterium acnes</em> showed a remarkable growth on nylon textile, with bacterial counts up to 2.25 × 10<sup>8</sup> CFU per cm<sup>2</sup>. The log count difference among textiles was the most dissimilar for <em>Micrococcus luteus</em>. The largest growth was noted on polyester textiles (1-log growth increase; up to 1.72 × 10<sup>7</sup> CFU per cm<sup>2</sup>), whereas the largest inhibition was noted on fleece textiles. This experiment confirmed the finding of selective growth of <em>Micrococcus</em> spp. on polyester clothing textiles, as well as no selective growth of <em>Micrococcus</em> spp. on cotton textiles. According to these results, viscose did not permit any growth of bacterial species. Wool, on the other hand, supported the growth of almost all bacteria. Nylon showed very selective bacterial growth. The growth of <em>Staphylococcus</em>, <em>Propionibacterium</em>, and <em>Enhydrobacter</em> spp. was enhanced, while the growth of <em>Micrococcus</em> and <em>Corynebacterium</em> spp. was inhibited. Growth on fleece likewise showed a selective profile. <em>Enhydrobacter</em> spp. were enhanced, <em>Propionibacterium</em> and <em>Corynebacterium</em> spp. remained at the same level, and <em>Staphylococcus</em> and <em>Micrococcus</em> spp. were inhibited. No growth (or inhibition) was observed on acryl textile for practically all species. Cotton textile indicated a growth for <em>Propionibacterium</em>, <em>Staphylococcus</em>, and <em>Enhydrobacter</em> spp., while practically no growth (or inhibition) was noted for <em>Micrococcus</em> and <em>Corynebacterium</em> spp. Polyester textile was associated the greatest growth for <em>Propionibacterium</em>, <em>Enhydrobacter</em>, and <em>Micrococcus</em> spp. Inhibition was recorded for <em>Corynebacterium</em> spp. on polyester. No growth (or inhibition) was noted for <em>Staphylococcus</em> spp.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div id="T2"><h3>TABLE 2</h3><!--caption a7--><p>Growth or inhibition (in log numbers) of bacterial species after a 3-day inoculation on different clothing textiles<sup><em><sup>a</sup></em></sup></p><p><img loading="lazy" alt="An external file that holds a picture, illustration, etc.
Object name is zam9991057430004.jpg" title="An external file that holds a picture, illustration, etc.
Object name is zam9991057430004.jpg" src="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4249026/bin/zam9991057430004.jpg"/></p></div></div><div id="sec3-5"><h3 id="sec3-5title">Contact angle measurements. </h3><p>A potential explanation for the selective growth is a dissimilar nonelectrostatic attraction between the bacterium and the different textile surfaces. Contact angle measurements were carried out (see Table S2 in the supplemental material) to determine the attraction or repulsion for <em>Micrococcus luteus</em> toward cotton (cellulose) and polyester (PET). Using the Young-Dupré equation, the contact angles were transformed into surface tension components, represented in Table S3 in the supplemental material. The interaction energy between micrococci and cotton (Δ<em>G</em> = −1.22 ± 1.00 J) was in the same range as the interaction energy between micrococci and polyester (Δ<em>G</em> = 0.24 ± 1.00 J). Both values were determined to be around 0. No differences were found in the interaction energies for micrococci and cotton and for micrococci and polyester.</p></div></div><div id="sec4"><h2 id="sec4title">DISCUSSION</h2><p>It is generally accepted that the choice of clothing has an impact on malodor formation (<a href="#B10" rid="B10">10</a>). This research showed that polyester clothes create a significantly higher malodor compared to cotton clothing after a fitness session and an incubation period. Significant differences were found for the hedonic value and the intensity of the odor, as well as all qualitative odor characteristics (musty, ammonia, strongness, sweatiness, and sourness). This corroborates earlier findings, where higher odor intensities were detected in polyester fabrics (<a href="#B10" rid="B10">10</a>). The first reason for the different odor profile is explained by the difference in odor adsorbance. Polyester is a petroleum-based synthetic fiber and has no natural properties. Synthetic fibers hence have a very poor adsorbing capacity, due to their molecular structure. Cotton is a natural fiber, originating from the <em>Gossypium</em> cotton plants. These cotton fibers almost purely consist of cellulose, which has a high adsorbing capacity (<a href="#B32" rid="B32">32</a>). Next to moisture, odors are adsorbed, and less malodor is emitted. A second reason can be explained by the dissimilar bacterial growth on the different textiles, where the malodor causing <em>Micrococcus</em> spp. tends to grow better on synthetic textiles. The poor adsorbing properties and the selective bacterial growth of micrococci may account for the malodor emission by certain synthetic sport clothes.</p><p>The microbial community of the textiles differs with the community living on the axillary skin (see Fig. S4 in the supplemental material). While the axillary microbiome is generally dominated by <em>Staphylococcus</em> and <em>Corynebacterium</em> species (<a href="#B6" rid="B6">6</a>), the textile microbiome was rather dominated by <em>Staphylococcus</em> and <em>Micrococcus</em> spp. (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4249026/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3</span></a>). The three main bacterial phyla found in the textiles (<em>Firmicutes</em>, <em>Actinobacteria</em>, and <em>Proteobacteria</em>) are also three important phyla of the skin microbiome (<a href="#B5" rid="B5">5</a>). Certain species were able to grow in more abundant quantities on the textile fibers. It is suggested that malodor generation is associated with the selective growth of those species. The bacterial enrichment was studied and differed depending on the bacterial species and the type of clothing textile, as shown by an <em>in vitro</em> growth experiment (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4249026/table/T2/" target="table" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). Micrococci were selectively enriched on polyester and wool but were inhibited on fleece and viscose. Polyester textiles showed an enrichment for <em>Micrococcus</em>, for <em>Enhydrobacter</em>, and <em>Propionibacterium</em> spp. These enrichments can have an important impact on the malodor creation from excreted sweat compounds. <em>Staphylococcus epidermidis</em> was enriched on both cotton and polyester textiles, as seen in the fitness clothes (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4249026/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3</span></a>). These results are in close correlation with previous findings, where a high affinity of <em>Staphylococcus</em> spp. for cotton and polyester was reported (<a href="#B33" rid="B33">33</a>, <a href="#B34" rid="B34">34</a>). The enrichment was confirmed by the <em>in vitro</em> growth experiment, with a growth reaching up to 10<sup>7</sup> CFU per cm<sup>2</sup> textile for cotton, wool, and nylon. On polyester, the presence was maintained on a level of 10<sup>6</sup> CFU per cm<sup>2</sup>. In addition, <em>Staphylococcus hominis</em> was often able to gain dominance on cotton textiles, as seen in the fitness experiment. This was not seen for synthetic clothing textiles. No bacterial enrichment was seen on viscose, a textile made from regenerated wood cellulose. Viscose showed very low bacterial extraction efficiencies. Further research is needed to confirm the absence of bacterial growth on viscose. If bacterial growth is indeed impeded on these fiber types, viscose could be used as bacterium- and odor-preventing textile in functional clothes. Wool, on the other hand, promoted the growth of almost all bacteria. This is in correlation with earlier findings, where the highest bacterial growth was noted for wool compared to the other tested clothing textiles. Although wool was associated with high bacterial counts, the odor intensity ratings were the lowest for wool (<a href="#B10" rid="B10">10</a>). nylon showed a very selective bacterial growth, with the biggest enrichment noted for <em>Propionibacterium</em> spp. (up to 10<sup>8</sup> CFU per cm<sup>2</sup>). <em>Staphylococcus</em> and <em>Enhydrobacter</em> spp. were enhanced as well, whereas the growth of <em>Micrococcus</em> and <em>Corynebacterium</em> spp. were inhibited. The <em>Propionibacterium</em> spp. are known to cause an acidic, intense foot odor (<a href="#B35" rid="B35">35</a>). The enrichment of these species on nylon socks has an important consequence on the foot malodor generation.</p><p>The <em>Corynebacterium</em> genus was not able to grow under the circumstances of the <em>in vitro</em> growth experiment. The genus was likewise not detectable by DGGE, nor could it be isolated from any clothing textile after the fitness experiment, although it was initially present in the axillae of many subjects (see Fig. S4 in the supplemental material). These findings are consistent with previous findings, where no growth of corynebacteria on clothing textiles was found (<a href="#B10" rid="B10">10</a>, <a href="#B34" rid="B34">34</a>). Corynebacteria are generally known as the most important species causing axillary malodor (<a href="#B36" rid="B36">36</a>). These bacterial species are thought to be involved in the conversion of sweat compounds into volatile short branched-chain fatty acids, steroid derivatives, and sulfanylalkanols—the three main axillary malodor classes (<a href="#B15" rid="B15">15</a>). The results of the present study, together with former research, indicated that corynebacteria are not the abundant bacterial species on clothing textiles. The absence or inability of corynebacteria to grow on clothing textiles implies that there are other bacterial types involved in the malodor creation in fabrics.</p><p>This research showed an overall enrichment of micrococci on the synthetic fabrics after the fitness session and incubation period. The bands were clearly visible on DGGE, meaning that the bacteria were present for at least more than 1% of the bacterial community (<a href="#B37" rid="B37">37</a>). Isolates of <em>Micrococcus</em> spp. were identified not only in 100% polyester textiles but also in almost every shirt where synthetic fibers were present (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4249026/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3</span></a>). The results were confirmed by the <em>in vitro</em> growth experiment (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4249026/table/T2/" target="table" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). Of the seven tested textile types, micrococci were able to gain the highest abundance on polyester fabrics (up to 10<sup>7</sup> CFU per cm<sup>2</sup>). No selective growth was found for micrococci on cotton textiles after 3 days. Previous research found a single enrichment of micrococci on polyester (<a href="#B34" rid="B34">34</a>). These findings confirm that micrococci are selectively enriched on polyester fabrics. It is hypothesized that the circumstances on synthetic clothing textiles are favorable for the growth and activity of <em>Micrococcus</em> spp. Their enrichment was not caused by a higher nonelectrostatic adsorption affinity for polyester. Other factors play a role in the enrichment of the micrococci. The aerobic growth conditions on polyester favor the growth of aerobic micrococci. Bacteria in clothing textiles are no longer suppressed by the innate immune system present on the skin. The nutritious environment, as well as quorum sensing (<a href="#B38" rid="B38">38</a>, <a href="#B39" rid="B39">39</a>), can additionally play a role in the growth of micrococci. A multiplicity of these favorable situations causes the selective enrichment of micrococci on polyester fabrics. <em>Micrococcus</em> spp. are known for their ability to create malodor from sweat secretions. They are able to fully catabolize saturated, monounsaturated, and methyl-branched fatty acids into malodor compounds (<a href="#B4" rid="B4">4</a>, <a href="#B40" rid="B40">40</a>). Next to corynebacteria, micrococci have been held responsible for the formation of body odor. These species have a high GC% content and are related to corynebacteria (both are members of the <em>Actinobacteria</em> phylum). Micrococci were frequently found in the axillary region, yet always by means of culturing techniques (<a href="#B4" rid="B4">4</a>, <a href="#B41" rid="B41">41</a>). In molecular studies, micrococci have not been found in large quantities on the human axillary skin (<a href="#B6" rid="B6">6</a>, <a href="#B42" rid="B42">42</a>). We suggest that micrococci were detected as they preferentially grow on the textiles worn close to the axillae and due to the practice of culturing techniques, which favor the growth of micrococci. It is suggested that micrococci prefer the aerobic environment of the textile fibers, whereas corynebacteria prefer the lipid-rich and more anaerobic environment on/in the (axillary) skin (<a href="#B43" rid="B43">43</a>). This may also explain the odor differences frequently perceived between axillary skin and the textile worn at the axillary skin. The use of underarm cosmetics may additionally impact the skin microbiome and the subjects body odor. Stopping or resuming deodorant/antiperspirant usage leads toward an altered underarm microbiome. Especially the use of antiperspirants causes significant changes (<a href="#B44" rid="B44">44</a>). Other factors include the general hygiene habits (frequency of washing, soap/shower gel type, etc.), the occupational lifestyle (physical activities, food habits, etc.), and the environment (place of residence and work, climate, humidity, etc.) which can impact the skin microbiome.</p><p>This research indicated that enrichment of micrococci occurred on polyester and, in general, on synthetic clothing textiles. Micrococci were frequently isolated, identified by means of DGGE fingerprinting, and enriched by an <em>in vitro</em> growth experiment on these textiles. The odor of the synthetic textiles was perceived as remarkably less pleasant after an intensive sport session. Microbial exchange occurs from skin to clothing textiles. A selective bacterial enrichment takes place, resulting in another microbiome compared to the autochthonous skin microbiome. The enrichment depended on the type of clothing textile and the type of bacterial species. With the current knowledge, the textile industry can design adjusted clothing fabrics that promote a non-odor-causing microbiome. This research opens perspectives toward better and functionalized sports clothing, which emit less malodor after use. Antimicrobial agents may be added to washing machine powders specifically against the odor causing microbiota, rather than using broad-spectrum antimicrobials. The enhancement of the non-odor-causing bacteria and the inhibition of the odor-causing bacteria, which are enriched on certain textiles, could greatly improve the quality of the fabrics.</p></div><div id="ack-1"><h2 id="ack-1title">ACKNOWLEDGMENTS</h2><div><p>This research was funded by the Flemish Government and Ghent University through the assistantship of C.C. F.-M.K. was supported by a research grant from the Geconcerteerde Onderzoeksactie (GOA) of Ghent University (<span>BOF09/GOA/005</span>).</p><p>C.C., E.D.M., T.V.D.W., and N.B. designed the experiments. E.D.M. and C.C. performed the experiments and analyzed the data. The statistical analysis was done by F.-M.K. The contact angle measurements and analysis was made possible by A.V. C.C. wrote the paper. A.V., T.V.D.W., and N.B. commented on the manuscript.</p><p>We acknowledge the odor panel and the persons attending the spinning session for their willingness to participate in this research. We thank Tim Lacoere for his assistance during the molecular work. We thank Francis de los Reyes III and Eleni Vaiopoulou for their critical review of the manuscript and the inspiring discussions.</p><p>The authors declare that they have no conflict of interest.</p></div></div><div id="ref-list-1"><h2 id="ref-list-1title">REFERENCES</h2><div id="reference-list"><p>1. <span>Szostak-Kotowa J.
2004. 
<span>Biodeterioration of textiles. Int</span>. <span>Biodeterior. Biodegrad.</span>
<span>53</span>:165–170. 10.1016/S0964-8305(03)00090-8. [<a href="https://doi.org/10.1016%2FS0964-8305(03)00090-8" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span>[<a href="https://scholar.google.com/scholar_lookup?journal=Biodeterior.+Biodegrad.&amp;title=Biodeterioration+of+textiles.+Int&amp;author=J+Szostak-Kotowa&amp;volume=53&amp;publication_year=2004&amp;pages=165-170&amp;doi=10.1016/S0964-8305(03)00090-8&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></p><p>2. <span>Marples MJ.
1969. 
<span>Life on the human skin</span>. <span>Sci. Am.</span>
<span>220</span>:108–115. 10.1038/scientificamerican0169-108. [<a href="https://pubmed.ncbi.nlm.nih.gov/5761729" ref="reftype=pubmed&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="https://doi.org/10.1038%2Fscientificamerican0169-108" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span>[<a href="https://scholar.google.com/scholar_lookup?journal=Sci.+Am.&amp;title=Life+on+the+human+skin&amp;author=MJ+Marples&amp;volume=220&amp;publication_year=1969&amp;pages=108-115&amp;pmid=5761729&amp;doi=10.1038/scientificamerican0169-108&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></p><p>3. <span>Fredricks DN.
2001. 
<span>Microbial ecology of human skin in health and disease</span>. <span>J. Invest. Dermatol. Symp. Proc.</span>
<span>6</span>:167–169. 10.1046/j.0022-202x.2001.00039.x. [<a href="https://pubmed.ncbi.nlm.nih.gov/11924822" ref="reftype=pubmed&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="https://doi.org/10.1046%2Fj.0022-202x.2001.00039.x" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span>[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Invest.+Dermatol.+Symp.+Proc.&amp;title=Microbial+ecology+of+human+skin+in+health+and+disease&amp;author=DN+Fredricks&amp;volume=6&amp;publication_year=2001&amp;pages=167-169&amp;doi=10.1046/j.0022-202x.2001.00039.x&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></p><p>4. <span>Leyden JJ, McGinley KJ, Holzle E, Labows JN, Kligman AM.
1981. 
<span>The microbiology of the human axilla and its relationship to axillary odor</span>. <span>J. Invest. Dermatol.</span>
<span>77</span>:413–416. 10.1111/1523-1747.ep12494624. [<a href="https://pubmed.ncbi.nlm.nih.gov/7288207" ref="reftype=pubmed&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="https://doi.org/10.1111%2F1523-1747.ep12494624" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span>[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Invest.+Dermatol.&amp;title=The+microbiology+of+the+human+axilla+and+its+relationship+to+axillary+odor&amp;author=JJ+Leyden&amp;author=KJ+McGinley&amp;author=E+Holzle&amp;author=JN+Labows&amp;author=AM+Kligman&amp;volume=77&amp;publication_year=1981&amp;pages=413-416&amp;pmid=7288207&amp;doi=10.1111/1523-1747.ep12494624&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></p><p>5. <span>Grice EA, Kong HH, Conlan S, Deming CB, Davis J, Young AC, Bouffard GG, Blakesley RW, Murray PR, Green ED, Turner ML, Segre JA, Progra NCS.
2009. 
<span>Topographical and temporal diversity of the human skin microbiome</span>. <span>Science</span>
<span>324</span>:1190–1192. 10.1126/science.1171700. <span>[<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2805064/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19478181" ref="reftype=pubmed&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="https://doi.org/10.1126%2Fscience.1171700" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span>[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Topographical+and+temporal+diversity+of+the+human+skin+microbiome&amp;author=EA+Grice&amp;author=HH+Kong&amp;author=S+Conlan&amp;author=CB+Deming&amp;author=J+Davis&amp;volume=324&amp;publication_year=2009&amp;pages=1190-1192&amp;pmid=19478181&amp;doi=10.1126/science.1171700&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></p><p>6. <span>Callewaert C, Kerckhof FM, Granitsiotis MS, van Gele M, van de Wiele T, Boon N.
2013. 
<span>Characterization of Staphylococcus and Corynebacterium clusters in the human axillary region</span>. <span>PLoS One</span>
<span>8</span>:e50538. 10.1371/journal.pone.0070538. <span>[<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3741381/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/23950955" ref="reftype=pubmed&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="https://doi.org/10.1371%2Fjournal.pone.0070538" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span>[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=Characterization+of+Staphylococcus+and+Corynebacterium+clusters+in+the+human+axillary+region&amp;author=C+Callewaert&amp;author=FM+Kerckhof&amp;author=MS+Granitsiotis&amp;author=M+van+Gele&amp;author=T+van+de+Wiele&amp;volume=8&amp;publication_year=2013&amp;pages=e50538&amp;doi=10.1371/journal.pone.0070538&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></p><p>7. <span>Buschlediller G, Zeronian SH, Pan N, Yoon MY.
1994. 
<span>Enzymatic hydrolysis of cotton, linen, ramie, and viscose rayon fabrics</span>. <span>Text. Res. J.</span>
<span>64</span>:270–279. 10.1177/004051759406400504. [<a href="https://doi.org/10.1177%2F004051759406400504" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span>[<a href="https://scholar.google.com/scholar_lookup?journal=Text.+Res.+J.&amp;title=Enzymatic+hydrolysis+of+cotton,+linen,+ramie,+and+viscose+rayon+fabrics&amp;author=G+Buschlediller&amp;author=SH+Zeronian&amp;author=N+Pan&amp;author=MY+Yoon&amp;volume=64&amp;publication_year=1994&amp;pages=270-279&amp;doi=10.1177/004051759406400504&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></p><p>8. <span>Shelley WB, Hurley HJ, Nicholas AC.
1953. 
<span>Axillary odor: experimental study of the role of bacteria, apocrine sweat, and deodorants</span>. <span>Arch. Dermatol. Syphilol.</span>
<span>68</span>:430–446. [<a href="https://pubmed.ncbi.nlm.nih.gov/13091383" ref="reftype=pubmed&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span>[<a href="https://scholar.google.com/scholar_lookup?journal=Arch.+Dermatol.+Syphilol.&amp;title=Axillary+odor:+experimental+study+of+the+role+of+bacteria,+apocrine+sweat,+and+deodorants&amp;author=WB+Shelley&amp;author=HJ+Hurley&amp;author=AC+Nicholas&amp;volume=68&amp;publication_year=1953&amp;pages=430-446&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></p><p>9. <span>Dravniek A, Krotoszy B, Lieb WE, Jungerma E.
1968. 
<span>Influence of an antibacterial soap on various effluents from axillae</span>. <span>J. Soc. Cosmet. Chem.</span>
<span>19</span>:611–626. <span>[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Soc.+Cosmet.+Chem.&amp;title=Influence+of+an+antibacterial+soap+on+various+effluents+from+axillae&amp;author=A+Dravniek&amp;author=B+Krotoszy&amp;author=WE+Lieb&amp;author=E+Jungerma&amp;volume=19&amp;publication_year=1968&amp;pages=611-626&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></p><p>10. <span>McQueen RH, Laing RM, Brooks HJL, Niven BE.
2007. 
<span>Odor intensity in apparel fabrics and the link with bacterial populations</span>. <span>Text. Res. J.</span>
<span>77</span>:449–456. 10.1177/0040517507074816. [<a href="https://doi.org/10.1177%2F0040517507074816" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span>[<a href="https://scholar.google.com/scholar_lookup?journal=Text.+Res.+J.&amp;title=Odor+intensity+in+apparel+fabrics+and+the+link+with+bacterial+populations&amp;author=RH+McQueen&amp;author=RM+Laing&amp;author=HJL+Brooks&amp;author=BE+Niven&amp;volume=77&amp;publication_year=2007&amp;pages=449-456&amp;doi=10.1177/0040517507074816&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></p><p>11. <span>Alonso D, Gimeno M, Olayo R, Vazquez-Torres H, Sepulveda-Sanchez JD, Shirai K.
2009. 
<span>Cross-linking chitosan into UV-irradiated cellulose fibers for the preparation of antimicrobial-finished textiles</span>. <span>Carbohydr. Polym.</span>
<span>77</span>:536–543. 10.1016/j.carbpol.2009.01.027. [<a href="https://doi.org/10.1016%2Fj.carbpol.2009.01.027" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span>[<a href="https://scholar.google.com/scholar_lookup?journal=Carbohydr.+Polym.&amp;title=Cross-linking+chitosan+into+UV-irradiated+cellulose+fibers+for+the+preparation+of+antimicrobial-finished+textiles&amp;author=D+Alonso&amp;author=M+Gimeno&amp;author=R+Olayo&amp;author=H+Vazquez-Torres&amp;author=JD+Sepulveda-Sanchez&amp;volume=77&amp;publication_year=2009&amp;pages=536-543&amp;doi=10.1016/j.carbpol.2009.01.027&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></p><p>12. <span>Lee J, Broughton RM, Akdag A, Worley SD, Huang TS.
2007. 
<span>Antimicrobial fibers created via polycarboxylic acid durable press finishing</span>. <span>Text. Res. J.</span>
<span>77</span>:604–611. 10.1177/0040517507081832. [<a href="https://doi.org/10.1177%2F0040517507081832" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span>[<a href="https://scholar.google.com/scholar_lookup?journal=Text.+Res.+J.&amp;title=Antimicrobial+fibers+created+via+polycarboxylic+acid+durable+press+finishing&amp;author=J+Lee&amp;author=RM+Broughton&amp;author=A+Akdag&amp;author=SD+Worley&amp;author=TS+Huang&amp;volume=77&amp;publication_year=2007&amp;pages=604-611&amp;doi=10.1177/0040517507081832&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></p><p>13. <span>El-Tahlawy KF, El-Bendary MA, Elhendawy AG, Hudson SM.
2005. 
<span>The antimicrobial activity of cotton fabrics treated with different cross-linking agents and chitosan</span>. <span>Carbohydr. Polym.</span>
<span>60</span>:421–430. 10.1016/j.carbpol.2005.02.019. [<a href="https://doi.org/10.1016%2Fj.carbpol.2005.02.019" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span>[<a href="https://scholar.google.com/scholar_lookup?journal=Carbohydr.+Polym.&amp;title=The+antimicrobial+activity+of+cotton+fabrics+treated+with+different+cross-linking+agents+and+chitosan&amp;author=KF+El-Tahlawy&amp;author=MA+El-Bendary&amp;author=AG+Elhendawy&amp;author=SM+Hudson&amp;volume=60&amp;publication_year=2005&amp;pages=421-430&amp;doi=10.1016/j.carbpol.2005.02.019&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></p><p>14. <span>Kathirvelu S, D&#39;Souza L, Dhurai B.
2009. 
<span>A study on functional finishing of cotton fabrics using nano-particles of zinc oxide</span>. <span>Mater. Sci. (Medziagotyra)</span>
<span>15</span>:75–79. <span>[<a href="https://scholar.google.com/scholar_lookup?journal=Mater.+Sci.+(Medziagotyra)&amp;title=A+study+on+functional+finishing+of+cotton+fabrics+using+nano-particles+of+zinc+oxide&amp;author=S+Kathirvelu&amp;author=L+D&#39;Souza&amp;author=B+Dhurai&amp;volume=15&amp;publication_year=2009&amp;pages=75-79&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></p><p>15. <span>Barzantny H, Brune I, Tauch A.
2012. 
<span>Molecular basis of human body odour formation: insights deduced from corynebacterial genome sequences</span>. <span>Int. J. Cosmet. Sci.</span>
<span>34</span>:2–11. 10.1111/j.1468-2494.2011.00669.x. [<a href="https://pubmed.ncbi.nlm.nih.gov/21790661" ref="reftype=pubmed&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="https://doi.org/10.1111%2Fj.1468-2494.2011.00669.x" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span>[<a href="https://scholar.google.com/scholar_lookup?journal=Int.+J.+Cosmet.+Sci.&amp;title=Molecular+basis+of+human+body+odour+formation:+insights+deduced+from+corynebacterial+genome+sequences&amp;author=H+Barzantny&amp;author=I+Brune&amp;author=A+Tauch&amp;volume=34&amp;publication_year=2012&amp;pages=2-11&amp;pmid=21790661&amp;doi=10.1111/j.1468-2494.2011.00669.x&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></p><p>16. <span>Evans CA, Stevens RJ.
1976. 
<span>Differential quantitation of surface and subsurface bacteria of normal skin by combined use of cotton swab and scrub methods</span>. <span>J. Clin. Microbiol.</span>
<span>3</span>:576–581. <span>[<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC274358/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/950377" ref="reftype=pubmed&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span>[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Clin.+Microbiol.&amp;title=Differential+quantitation+of+surface+and+subsurface+bacteria+of+normal+skin+by+combined+use+of+cotton+swab+and+scrub+methods&amp;author=CA+Evans&amp;author=RJ+Stevens&amp;volume=3&amp;publication_year=1976&amp;pages=576-581&amp;pmid=950377&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></p><p>17. <span>Amoore JE, Venstrom D, Nutting MD.
1972. 
<span>Sweaty odor in fatty acids: measurements of similarity, confusion, and fatigue</span>. <span>J. Food Sci.</span>
<span>37</span>:33–35. 10.1111/j.1365-2621.1972.tb03378.x. [<a href="https://doi.org/10.1111%2Fj.1365-2621.1972.tb03378.x" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span>[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Food+Sci.&amp;title=Sweaty+odor+in+fatty+acids:+measurements+of+similarity,+confusion,+and+fatigue&amp;author=JE+Amoore&amp;author=D+Venstrom&amp;author=MD+Nutting&amp;volume=37&amp;publication_year=1972&amp;pages=33-35&amp;doi=10.1111/j.1365-2621.1972.tb03378.x&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></p><p>18. <span>R Development Core Team. 2013. 
<span>R: a language and environment for statistical computing</span>, 3rd ed.
R Foundation for Statistical Computing, Vienna, Austria. <span>[<a href="https://scholar.google.com/scholar_lookup?title=R:+a+language+and+environment+for+statistical+computing&amp;publication_year=2013&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></p><p>19. <span>Szekely GJ, Rizzo ML.
2005. 
<span>A new test for multivariate normality</span>. <span>J. Multivariate Anal.</span>
<span>93</span>:58–80. 10.1016/j.jmva.2003.12.002. [<a href="https://doi.org/10.1016%2Fj.jmva.2003.12.002" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span>[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Multivariate+Anal.&amp;title=A+new+test+for+multivariate+normality&amp;author=GJ+Szekely&amp;author=ML+Rizzo&amp;volume=93&amp;publication_year=2005&amp;pages=58-80&amp;doi=10.1016/j.jmva.2003.12.002&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></p><p>20. <span>Oksanen J, Blanchet FG, Kindt R, Legendre P, Minchin PR, O&#39;Hara RB, Simpson GL, Solymos P, Stevens MHH, Wagner H.
2013. 
<span>Package “vegan”: community ecology package. R package version 2.0-7</span>. <a href="http://cran.r-project.org/web/packages/vegan/vegan.pdf" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">http://cran.r-project.org/web/packages/vegan/vegan.pdf</a>.</span></p><p>21. <span>Anderson MJ.
2006. 
<span>Distance-based tests for homogeneity of multivariate dispersions</span>. <span>Biometrics</span>
<span>62</span>:245–253. 10.1111/j.1541-0420.2005.00440.x. [<a href="https://pubmed.ncbi.nlm.nih.gov/16542252" ref="reftype=pubmed&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="https://doi.org/10.1111%2Fj.1541-0420.2005.00440.x" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span>[<a href="https://scholar.google.com/scholar_lookup?journal=Biometrics&amp;title=Distance-based+tests+for+homogeneity+of+multivariate+dispersions&amp;author=MJ+Anderson&amp;volume=62&amp;publication_year=2006&amp;pages=245-253&amp;pmid=16542252&amp;doi=10.1111/j.1541-0420.2005.00440.x&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></p><p>23. <span>Teufel L, Schuster KC, Merschak P, Bechtold T, Redl B.
2008. 
<span>Development of a fast and reliable method for the assessment of microbial colonization and growth on textiles by DNA quantification</span>. <span>J. Mol. Microbiol. Biotechnol.</span>
<span>14</span>:193–200. 10.1159/000108657. [<a href="https://pubmed.ncbi.nlm.nih.gov/17878710" ref="reftype=pubmed&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="https://doi.org/10.1159%2F000108657" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span>[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Mol.+Microbiol.+Biotechnol.&amp;title=Development+of+a+fast+and+reliable+method+for+the+assessment+of+microbial+colonization+and+growth+on+textiles+by+DNA+quantification&amp;author=L+Teufel&amp;author=KC+Schuster&amp;author=P+Merschak&amp;author=T+Bechtold&amp;author=B+Redl&amp;volume=14&amp;publication_year=2008&amp;pages=193-200&amp;pmid=17878710&amp;doi=10.1159/000108657&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></p><p>24. <span>Muyzer G, de Waal EC, Uitterlinden AG.
1993. 
<span>Profiling of complex microbial populations by denaturing gradient gel electrophoresis analysis of polymerase chain reaction-amplified genes coding for 16S rRNA</span>. <span>Appl. Environ. Microbiol.</span>
<span>59</span>:695–700. <span>[<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC202176/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/7683183" ref="reftype=pubmed&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span>[<a href="https://scholar.google.com/scholar_lookup?journal=Appl.+Environ.+Microbiol.&amp;title=Profiling+of+complex+microbial+populations+by+denaturing+gradient+gel+electrophoresis+analysis+of+polymerase+chain+reaction-amplified+genes+coding+for+16S+rRNA&amp;author=G+Muyzer&amp;author=EC+de+Waal&amp;author=AG+Uitterlinden&amp;volume=59&amp;publication_year=1993&amp;pages=695-700&amp;pmid=7683183&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></p><p>25. <span>Ovreas L, Forney L, Daae FL, Torsvik V.
1997. 
<span>Distribution of bacterioplankton in meromictic Lake Saelenvannet, as determined by denaturing gradient gel electrophoresis of PCR-amplified gene fragments coding for 16S rRNA</span>. <span>Appl. Environ. Microbiol.</span>
<span>63</span>:3367–3373. <span>[<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC168642/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/9292986" ref="reftype=pubmed&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span>[<a href="https://scholar.google.com/scholar_lookup?journal=Appl.+Environ.+Microbiol.&amp;title=Distribution+of+bacterioplankton+in+meromictic+Lake+Saelenvannet,+as+determined+by+denaturing+gradient+gel+electrophoresis+of+PCR-amplified+gene+fragments+coding+for+16S+rRNA&amp;author=L+Ovreas&amp;author=L+Forney&amp;author=FL+Daae&amp;author=V+Torsvik&amp;volume=63&amp;publication_year=1997&amp;pages=3367-3373&amp;pmid=9292986&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></p><p>26. <span>Lane DJ.
1991. 
<span>16S/23S rRNA sequencing</span>, p 115–175
<em>In</em>
Stackebrandt E, Goodfellow M. (ed), <span>Nucleic acid techniques in bacterial systematics</span>. John Wiley &amp; Sons, Chichester, United Kingdom. <span>[<a href="https://scholar.google.com/scholar_lookup?title=Nucleic+acid+techniques+in+bacterial+systematics&amp;author=DJ+Lane&amp;publication_year=1991&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></p><p>27. <span>Saitou N, Nei M.
1987. 
<span>The neighbor-joining method: a new method for reconstructing phylogenetic trees</span>. <span>Mol. Biol. Evol.</span>
<span>4</span>:406–425. [<a href="https://pubmed.ncbi.nlm.nih.gov/3447015" ref="reftype=pubmed&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span>[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Biol.+Evol.&amp;title=The+neighbor-joining+method:+a+new+method+for+reconstructing+phylogenetic+trees&amp;author=N+Saitou&amp;author=M+Nei&amp;volume=4&amp;publication_year=1987&amp;pages=406-425&amp;pmid=3447015&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></p><p>28. <span>Tamura K, Peterson D, Peterson N, Stecher G, Nei M, Kumar S.
2011. 
<span>MEGA5: molecular evolutionary genetics analysis using maximum likelihood, evolutionary distance, and maximum-parsimony methods</span>. <span>Mol. Biol. Evol.</span>
<span>28</span>:2731–2739. 10.1093/molbev/msr121. <span>[<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3203626/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/21546353" ref="reftype=pubmed&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="https://doi.org/10.1093%2Fmolbev%2Fmsr121" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span>[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Biol.+Evol.&amp;title=MEGA5:+molecular+evolutionary+genetics+analysis+using+maximum+likelihood,+evolutionary+distance,+and+maximum-parsimony+methods&amp;author=K+Tamura&amp;author=D+Peterson&amp;author=N+Peterson&amp;author=G+Stecher&amp;author=M+Nei&amp;volume=28&amp;publication_year=2011&amp;pages=2731-2739&amp;pmid=21546353&amp;doi=10.1093/molbev/msr121&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></p><p>29. <span>Jukes TH, Cantor CR.
1969. 
<span>Evolution of protein molecules</span>, p 21–132
<em>In</em>
Munro HN. (ed), <span>Mammalian protein metabolism</span>. Academic Press, Inc, New York, NY. <span>[<a href="https://scholar.google.com/scholar_lookup?title=Mammalian+protein+metabolism&amp;author=TH+Jukes&amp;author=CR+Cantor&amp;publication_year=1969&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></p><p>30. <span>De Roy K, Clement L, Thas O, Wang YY, Boon N.
2012. 
<span>Flow cytometry for fast microbial community fingerprinting</span>. <span>Water Res.</span>
<span>46</span>:907–919. 10.1016/j.watres.2011.11.076. [<a href="https://pubmed.ncbi.nlm.nih.gov/22192760" ref="reftype=pubmed&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="https://doi.org/10.1016%2Fj.watres.2011.11.076" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span>[<a href="https://scholar.google.com/scholar_lookup?journal=Water+Res.&amp;title=Flow+cytometry+for+fast+microbial+community+fingerprinting&amp;author=K+De+Roy&amp;author=L+Clement&amp;author=O+Thas&amp;author=YY+Wang&amp;author=N+Boon&amp;volume=46&amp;publication_year=2012&amp;pages=907-919&amp;pmid=22192760&amp;doi=10.1016/j.watres.2011.11.076&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></p><p>31. <span>Verliefde ARD, Cornelissen ER, Heijman SGJ, Hoek EMV, Amy GL, Van der Bruggen B, Van Dijk JC.
2009. 
<span>Influence of solute-membrane affinity on rejection of uncharged organic solutes by nanofiltration membranes</span>. <span>Environ. Sci. Technol.</span>
<span>43</span>:2400–2406. 10.1021/es803146r. [<a href="https://pubmed.ncbi.nlm.nih.gov/19452893" ref="reftype=pubmed&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="https://doi.org/10.1021%2Fes803146r" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span>[<a href="https://scholar.google.com/scholar_lookup?journal=Environ.+Sci.+Technol.&amp;title=Influence+of+solute-membrane+affinity+on+rejection+of+uncharged+organic+solutes+by+nanofiltration+membranes&amp;author=ARD+Verliefde&amp;author=ER+Cornelissen&amp;author=SGJ+Heijman&amp;author=EMV+Hoek&amp;author=GL+Amy&amp;volume=43&amp;publication_year=2009&amp;pages=2400-2406&amp;pmid=19452893&amp;doi=10.1021/es803146r&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></p><p>32. <span>Shorter SA.
1924. 
<span>The thermodynamics of water absorption by textile materials</span>. <span>J. Text. Inst. Trans.</span>
<span>15</span>:T328–T336. 10.1080/19447022408661305. [<a href="https://doi.org/10.1080%2F19447022408661305" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span>[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Text.+Inst.+Trans.&amp;title=The+thermodynamics+of+water+absorption+by+textile+materials&amp;author=SA+Shorter&amp;volume=15&amp;publication_year=1924&amp;pages=T328-T336&amp;doi=10.1080/19447022408661305&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></p><p>33. <span>Hsieh YL, Merry J.
1986. 
<span>The adherence of Staphylococcus aureus, Staphylococcus epidermidis, and Escherichia coli on cotton, polyester, and their blends</span>. <span>J. Appl. Bacteriol.</span>
<span>60</span>:535–544. 10.1111/j.1365-2672.1986.tb01093.x. [<a href="https://pubmed.ncbi.nlm.nih.gov/3528111" ref="reftype=pubmed&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="https://doi.org/10.1111%2Fj.1365-2672.1986.tb01093.x" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span>[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Appl.+Bacteriol.&amp;title=The+adherence+of+Staphylococcus+aureus,+Staphylococcus+epidermidis,+and+Escherichia+coli+on+cotton,+polyester,+and+their+blends&amp;author=YL+Hsieh&amp;author=J+Merry&amp;volume=60&amp;publication_year=1986&amp;pages=535-544&amp;pmid=3528111&amp;doi=10.1111/j.1365-2672.1986.tb01093.x&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></p><p>34. <span>Teufel L, Pipal A, Schuster KC, Staudinger T, Redl B.
2010. 
<span>Material-dependent growth of human skin bacteria on textiles investigated using challenge tests and DNA genotyping</span>. <span>J. Appl. Microbiol.</span>
<span>108</span>:450–461. 10.1111/j.1365-2672.2009.04434.x. [<a href="https://pubmed.ncbi.nlm.nih.gov/19645767" ref="reftype=pubmed&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="https://doi.org/10.1111%2Fj.1365-2672.2009.04434.x" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span>[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Appl.+Microbiol.&amp;title=Material-dependent+growth+of+human+skin+bacteria+on+textiles+investigated+using+challenge+tests+and+DNA+genotyping&amp;author=L+Teufel&amp;author=A+Pipal&amp;author=KC+Schuster&amp;author=T+Staudinger&amp;author=B+Redl&amp;volume=108&amp;publication_year=2010&amp;pages=450-461&amp;pmid=19645767&amp;doi=10.1111/j.1365-2672.2009.04434.x&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></p><p>35. <span>Ara K, Hama M, Akiba S, Koike K, Okisaka K, Hagura T, Kamiya T, Tomita F.
2006. 
<span>Foot odor due to microbial metabolism and its control</span>. <span>Can. J. Microbiol.</span>
<span>52</span>:357–364. 10.1139/w05-130. [<a href="https://pubmed.ncbi.nlm.nih.gov/16699586" ref="reftype=pubmed&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="https://doi.org/10.1139%2Fw05-130" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span>[<a href="https://scholar.google.com/scholar_lookup?journal=Can.+J.+Microbiol.&amp;title=Foot+odor+due+to+microbial+metabolism+and+its+control&amp;author=K+Ara&amp;author=M+Hama&amp;author=S+Akiba&amp;author=K+Koike&amp;author=K+Okisaka&amp;volume=52&amp;publication_year=2006&amp;pages=357-364&amp;pmid=16699586&amp;doi=10.1139/w05-130&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></p><p>36. <span>James AG, Austin CJ, Cox DS, Taylor D, Calvert R.
2013. 
<span>Microbiological and biochemical origins of human axillary odour</span>. <span>FEMS Microbiol. Ecol.</span>
<span>83</span>:527–540. 10.1111/1574-6941.12054. [<a href="https://pubmed.ncbi.nlm.nih.gov/23278215" ref="reftype=pubmed&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="https://doi.org/10.1111%2F1574-6941.12054" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span>[<a href="https://scholar.google.com/scholar_lookup?journal=FEMS+Microbiol.+Ecol.&amp;title=Microbiological+and+biochemical+origins+of+human+axillary+odour&amp;author=AG+James&amp;author=CJ+Austin&amp;author=DS+Cox&amp;author=D+Taylor&amp;author=R+Calvert&amp;volume=83&amp;publication_year=2013&amp;pages=527-540&amp;pmid=23278215&amp;doi=10.1111/1574-6941.12054&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></p><p>37. <span>Muyzer G, Smalla K.
1998. 
<span>Application of denaturing gradient gel electrophoresis (DGGE) and temperature gradient gel electrophoresis (TGGE) in microbial ecology</span>. <span>Antonie Van Leeuwenhoek</span>
<span>73</span>:127–141. [<a href="https://pubmed.ncbi.nlm.nih.gov/9602286" ref="reftype=pubmed&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span>[<a href="https://scholar.google.com/scholar_lookup?journal=Antonie+Van+Leeuwenhoek&amp;title=Application+of+denaturing+gradient+gel+electrophoresis+(DGGE)+and+temperature+gradient+gel+electrophoresis+(TGGE)+in+microbial+ecology&amp;author=G+Muyzer&amp;author=K+Smalla&amp;volume=73&amp;publication_year=1998&amp;pages=127-141&amp;pmid=9602286&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></p><p>38. <span>Miller MB, Bassler BL.
2001. 
<span>Quorum sensing in bacteria</span>. <span>Annu. Rev. Microbiol.</span>
<span>55</span>:165–199. 10.1146/annurev.micro.55.1.165. [<a href="https://pubmed.ncbi.nlm.nih.gov/11544353" ref="reftype=pubmed&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="https://doi.org/10.1146%2Fannurev.micro.55.1.165" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span>[<a href="https://scholar.google.com/scholar_lookup?journal=Annu.+Rev.+Microbiol.&amp;title=Quorum+sensing+in+bacteria&amp;author=MB+Miller&amp;author=BL+Bassler&amp;volume=55&amp;publication_year=2001&amp;pages=165-199&amp;pmid=11544353&amp;doi=10.1146/annurev.micro.55.1.165&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></p><p>39. <span>Mukamolova GV, Kormer SS, Kell DB, Kaprelyants AS.
1999. 
<span>Stimulation of the multiplication of Micrococcus luteus by an autocrine growth factor</span>. <span>Arch. Microbiol.</span>
<span>172</span>:9–14. 10.1007/s002030050733. [<a href="https://pubmed.ncbi.nlm.nih.gov/10398746" ref="reftype=pubmed&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="https://doi.org/10.1007%2Fs002030050733" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span>[<a href="https://scholar.google.com/scholar_lookup?journal=Arch.+Microbiol.&amp;title=Stimulation+of+the+multiplication+of+Micrococcus+luteus+by+an+autocrine+growth+factor&amp;author=GV+Mukamolova&amp;author=SS+Kormer&amp;author=DB+Kell&amp;author=AS+Kaprelyants&amp;volume=172&amp;publication_year=1999&amp;pages=9-14&amp;pmid=10398746&amp;doi=10.1007/s002030050733&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></p><p>40. <span>James AG, Casey J, Hyliands D, Mycock G.
2004. 
<span>Fatty acid metabolism by cutaneous bacteria and its role in axillary malodour</span>. <span>World J. Microbiol. Biotechnol.</span>
<span>20</span>:787–793. 10.1007/s11274-004-5843-8. [<a href="https://doi.org/10.1007%2Fs11274-004-5843-8" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span>[<a href="https://scholar.google.com/scholar_lookup?journal=World+J.+Microbiol.+Biotechnol.&amp;title=Fatty+acid+metabolism+by+cutaneous+bacteria+and+its+role+in+axillary+malodour&amp;author=AG+James&amp;author=J+Casey&amp;author=D+Hyliands&amp;author=G+Mycock&amp;volume=20&amp;publication_year=2004&amp;pages=787-793&amp;doi=10.1007/s11274-004-5843-8&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></p><p>41. <span>Taylor D, Daulby A, Grimshaw S, James G, Mercer J, Vaziri S.
2003. 
<span>Characterization of the microflora of the human axilla</span>. <span>Int. J. Cosmet. Sci.</span>
<span>25</span>:137–145. 10.1046/j.1467-2494.2003.00181.x. [<a href="https://pubmed.ncbi.nlm.nih.gov/18494895" ref="reftype=pubmed&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="https://doi.org/10.1046%2Fj.1467-2494.2003.00181.x" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span>[<a href="https://scholar.google.com/scholar_lookup?journal=Int.+J.+Cosmet.+Sci.&amp;title=Characterization+of+the+microflora+of+the+human+axilla&amp;author=D+Taylor&amp;author=A+Daulby&amp;author=S+Grimshaw&amp;author=G+James&amp;author=J+Mercer&amp;volume=25&amp;publication_year=2003&amp;pages=137-145&amp;pmid=18494895&amp;doi=10.1046/j.1467-2494.2003.00181.x&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></p><p>42. <span>Costello EK, Lauber CL, Hamady M, Fierer N, Gordon JI, Knight R.
2009. 
<span>Bacterial community variation in human body habitats across space and time</span>. <span>Science</span>
<span>326</span>:1694–1697. 10.1126/science.1177486. <span>[<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3602444/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19892944" ref="reftype=pubmed&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="https://doi.org/10.1126%2Fscience.1177486" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span>[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Bacterial+community+variation+in+human+body+habitats+across+space+and+time&amp;author=EK+Costello&amp;author=CL+Lauber&amp;author=M+Hamady&amp;author=N+Fierer&amp;author=JI+Gordon&amp;volume=326&amp;publication_year=2009&amp;pages=1694-1697&amp;pmid=19892944&amp;doi=10.1126/science.1177486&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></p><p>43. <span>Marples RR, McGinley KJ.
1974. 
<span>Corynebacterium
<em>acnes</em> and other anaerobic diphteroids from human skin</span>. <span>J. Med. Microbiol.</span>
<span>7</span>:349–352. 10.1099/00222615-7-3-349. [<a href="https://pubmed.ncbi.nlm.nih.gov/4278837" ref="reftype=pubmed&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="https://doi.org/10.1099%2F00222615-7-3-349" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span>[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Med.+Microbiol.&amp;title=Corynebacterium
acnes+and+other+anaerobic+diphteroids+from+human+skin&amp;author=RR+Marples&amp;author=KJ+McGinley&amp;volume=7&amp;publication_year=1974&amp;pages=349-352&amp;pmid=4278837&amp;doi=10.1099/00222615-7-3-349&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></p><p>44. <span>Callewaert C, Hutapea P, Van de Wiele T, Boon N.
2014. 
<span>Deodorants and antiperspirants affect the axillary bacterial community</span>. <span>Arch. Dermatol. Res.</span>
<span>2014</span>:1–10. 10.1007/s00403-014-1487-1. [<a href="https://pubmed.ncbi.nlm.nih.gov/25077920" ref="reftype=pubmed&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="https://doi.org/10.1007%2Fs00403-014-1487-1" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span>[<a href="https://scholar.google.com/scholar_lookup?journal=Arch.+Dermatol.+Res.&amp;title=Deodorants+and+antiperspirants+affect+the+axillary+bacterial+community&amp;author=C+Callewaert&amp;author=P+Hutapea&amp;author=T+Van+de+Wiele&amp;author=N+Boon&amp;volume=2014&amp;publication_year=2014&amp;pages=1-10&amp;doi=10.1007/s00403-014-1487-1&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4249026&amp;issue-id=245899&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></p></div></div></div><!--post-content--><div><hr/><p>Articles from <span>Applied and Environmental Microbiology</span> are provided here courtesy of <strong>American Society for Microbiology (ASM)</strong></p><hr/></div></div>
            
        </section>
    </article>
    
</div></div>
  </body>
</html>
