<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <title>James Routley | Feed</title>
    <link
      rel="stylesheet"
      type="text/css"
      href="../styles.css"
      media="screen"
    />
  </head>
  <body>
    <a href="/index.html">Back</a>
    <a href="https://www.rockefeller.edu/news/38239-hundreds-of-new-bacteria-and-two-potential-antibiotics-found-in-soil/">Original</a>
    <h1>New bacteria, and two potential antibiotics, discovered in soil</h1>
    
    <div id="readability-page-1" class="page"><div data-image-xs="" data-image-sm="" data-image-md="" data-image-lg="" data-image-xl="">
			
<!-- .ru-search -->


			<div id="main_content" data-image-xs="" data-image-sm="" data-image-md="" data-image-lg="" data-image-xl="">
														<section>
    <div>
        <div>
            <div>
                <section>
                    	<nav aria-label="Breadcrumb"><span aria-hidden="true"> &gt; </span></nav>
                    








                    
  <section>
    <div id="attachment_38240"><p><img decoding="async" loading="lazy" aria-describedby="caption-attachment-38240" src="https://www.rockefeller.edu/news/uploads/www.rockefeller.edu/sites/13/2025/09/Brady_81325_HERO.jpg" alt="bacterial DNA extracted from soils" width="1000" height="666" srcset="https://www.rockefeller.edu/news/../../uploads/www.rockefeller.edu/sites/13/2025/09/Brady_81325_HERO.jpg 1000w, https://www.rockefeller.edu/news/../../uploads/www.rockefeller.edu/sites/13/2025/09/Brady_81325_HERO-700x467.jpg 700w, https://www.rockefeller.edu/news/../../uploads/www.rockefeller.edu/sites/13/2025/09/Brady_81325_HERO-400x267.jpg 400w" sizes="(max-width: 1000px) 100vw, 1000px"/></p>
<p id="caption-attachment-38240">To find bioactive molecules with the potential to become new drugs less prone to antibiotic resistance, the researchers sequenced bacterial DNA extracted from soils from Rockefeller’s field center in upstate New York. (Brady lab)</p>
</div>
<p>Most bacteria cannot be cultured in the lab—and that’s been bad news for medicine. Many of our frontline antibiotics originated from microbes, yet as antibiotic resistance spreads and drug pipelines run dry, the soil beneath our feet has a vast hidden reservoir of untapped lifesaving compounds.</p>
<p>Now, researchers have developed a way to access this microbial goldmine. Their approach, published in <em><a href="https://www.nature.com/articles/s41587-025-02810-w">Nature Biotechnology</a></em>, circumvents the need to grow bacteria in the lab by extracting very large DNA fragments directly from soil to piece together the genomes of previously hidden microbes, and then mines resulting genomes for bioactive molecules.</p>
<p>From a single forest sample, the team generated hundreds of complete bacterial genomes never seen before, as well as two new antibiotic leads. The findings offer a scalable way to scour unculturable bacteria for new drug leads—and expose the vast, uncharted microbial frontier that shapes our environment.</p>
<p>“We finally have the technology to see the microbial world that have been previously inaccessible to humans,” says <a href="https://www.rockefeller.edu/our-scientists/heads-of-laboratories/965-sean-f-brady/">Sean F. Brady</a>, head of the <a href="https://bradylab.rockefeller.edu/">Laboratory of Genetically Encoded Small Molecules</a> at Rockefeller. “And we’re not just seeing this information; we’re already turning it into potentially useful antibiotics. This is just the tip of the spear.”</p>
<p><strong>Microbial dark matter</strong></p>
<p>When hunting for bacteria, soil is an obvious choice. It’s the largest, most biodiverse reservoir of bacteria on the planet—a single teaspoon of it may contain thousands of different species. Many important therapeutics, including most of our antibiotic arsenal, were discovered in the tiny fraction of soil bacteria that can be grown in the laboratory. And soil is dirt cheap.</p>
<p>Yet we know very little about the millions of microbes packed into the earth. Scientists suspect that these hidden bacteria hold not only an untapped reservoir of new therapeutics, but clues as to how microbes shape climate, agriculture, and the larger environment that we live in. “All over the world there’s this hidden ecosystem of microbes that could have dramatic effects on our lives,” Brady adds. “We wanted to finally see them.”</p>
<p>Getting that glimpse involved weaving together several approaches. First, the team optimized a method for isolating large, high-quality DNA fragments directly from soil. Pairing this advance with emerging long-read nanopore sequencing allowed Jan Burian, a postdoctoral associate in the Brady lab, to produce continuous stretches of DNA that were tens of thousands of base pairs long—200 times longer than any previously existing technology could manage. Soil DNA contains a huge number of different bacteria; without such large DNA sequences to work with, resolving that complex genetic puzzle into complete and contiguous genomes for disparate bacteria proved exceedingly difficult.</p>
<p>“It’s easier to assemble a whole genome out of bigger pieces of DNA, rather than the millions of tiny snippets that were available before,” Brady says. “And that makes a dramatic difference in your confidence in your results.”</p>
<p>Unique small molecules, like antibiotics, that bacteria produce are called “natural products”. To convert the newly uncovered sequences into bioactive molecules, the team applied a synthetic bioinformatic natural products (synBNP) approach. They bioinformatically predicted the chemical structures of natural products directly from the genome data and then chemically synthesized them in the lab. With the synBNP approach, Brady and colleagues managed to turn the genetic blueprints from uncultured bacteria into actual molecules—including two potent antibiotics.</p>
<p>Brady describes the method, which is scalable and can be adapted to virtually any metagenomic space beyond soil, as a three-step strategy that could kick off a new era of microbiology: “Isolate big DNA, sequence it, and computationally convert it into something useful.”</p>
<p><strong>Two new drug candidates, and counting</strong></p>
<p>Applied to their single forest soil sample, the team’s approach produced 2.5 terabase-pairs of sequence data—the deepest long-read exploration of a single soil sample to date. Their analysis uncovered hundreds of complete contiguous bacterial genomes, more than 99 percent of which were entirely new to science and identified members from 16 major branches of the bacterial family tree.</p>
<p>The two lead compounds discovered could translate into potent antibiotics. One, called erutacidin, disrupts bacterial membranes through an uncommon interaction with the lipid cardiolipin and is effective against even the most challenging drug-resistant bacteria. The other, trigintamicin, acts on a protein-unfolding motor known as ClpX, a rare antibacterial target.</p>
<p>Brady emphasizes that these discoveries are only the beginning. The study demonstrates that previously inaccessible microbial genomes can now be decoded and mined for bioactive molecules at scale without culturing the organisms. Unlocking the genetic potential of microbial dark matter may also provide new insights into the hidden microbial networks that sustain ecosystems.</p>
<p>“We’re mainly interested in small molecules as therapeutics, but there are applications beyond medicine,” Burian says. “Studying culturable bacteria led to advances that helped shape the modern world and finally seeing and accessing the uncultured majority will drive a new generation of discovery.”</p>

  </section>











  <hr/>



                </section>
                
            </div>
        </div>

                    
        
    </div>
</section>

												</div>
			<!-- .ru-site-footer -->
		</div></div>
  </body>
</html>
