<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <title>James Routley | Feed</title>
    <link
      rel="stylesheet"
      type="text/css"
      href="../styles.css"
      media="screen"
    />
  </head>
  <body>
    <a href="/index.html">Back</a>
    <a href="https://phys.org/news/2023-04-genomes-species-redefine-mammalian-tree.html">Original</a>
    <h1>Researchers use genomes of 241 species to redefine mammalian tree of life</h1>
    
    <div id="readability-page-1" class="page"><div>
										
<div>
    <div data-thumb="https://scx1.b-cdn.net/csz/news/tmb/2023/texas-aampm-research-r.jpg" data-src="https://scx2.b-cdn.net/gfx/news/hires/2023/texas-aampm-research-r.jpg" data-sub-html="Foley&#39;s efforts in the research produced the world&#39;s largest mammalian phylogenetic tree to date. The “mammalian tree of life” maps out the evolution of mammals over more than 100 million years and is crucial to the goals of the Zoonomia Project. Credit: Texas A&amp;M University">
        <figure>
            <img src="https://scx1.b-cdn.net/csz/news/800a/2023/texas-aampm-research-r.jpg" alt="Texas A&amp;amp;M research redefines mammalian tree of life" title="Foley&#39;s efforts in the research produced the world&#39;s largest mammalian phylogenetic tree to date. The “mammalian tree of life” maps out the evolution of mammals over more than 100 million years and is crucial to the goals of the Zoonomia Project. Credit: Texas A&amp;M University" width="800" height="530"/>
             <figcaption>
                Foley&#39;s efforts in the research produced the world&#39;s largest mammalian phylogenetic tree to date. The “mammalian tree of life” maps out the evolution of mammals over more than 100 million years and is crucial to the goals of the Zoonomia Project. Credit: Texas A&amp;M University
            </figcaption>        </figure>
    </div>
</div>
<p>Research led by a team of scientists from the Texas A&amp;M School of Veterinary Medicine and Biomedical Sciences puts to bed the heated scientific debate regarding the history of mammal diversification as it relates to the extinction of the non-avian dinosaurs. Their work provides a definitive answer to the evolutionary timeline of mammals throughout the last 100 million years.

										  
											        </p><section>
         <!-- /4988204/Phys_Story_InText_Box -->
        
      </section>
										 
										 											  
<p>The study, published in <a href="https://www.science.org/doi/10.1126/science.abl8189"><i>Science</i></a>, is part of a series of articles released by the <a href="https://zoonomiaproject.org/">Zoonomia Project</a>, a consortium of scientists from around the globe that is using the largest mammalian genomic dataset in history to determine the evolutionary history of the human <a href="https://phys.org/tags/genome/" rel="tag">genome</a> in the context of mammalian evolutionary history. Their ultimate goal is to better identify the <a href="https://phys.org/tags/genetic+basis/" rel="tag">genetic basis</a> for traits and diseases in people and other species.
</p><p>The Texas A&amp;M University research—led by Dr. William J. Murphy, a professor in the Department of Veterinary Integrative Biosciences, and Dr. Nicole Foley, an associate research scientist in Murphy&#39;s lab—is rooted in phylogeny, a branch of biology that deals with the evolutionary relationships and diversification of living and extinct organisms.
</p><p>&#34;The central argument is about whether placental mammals (mammals that develop within placentas) diverged before or after the Cretaceous-Paleogene (or K-Pg) extinction event that wiped out the non-avian dinosaurs,&#34; Foley shared. &#34;By performing new types of analyses only possible because of Zoonomia&#39;s massive scope, we answer the question of where and when mammals diversified and evolved in relation to the K-Pg mass extinction.&#34;
</p><p>The research—which was conducted with collaborators at the University of California, Davis; University of California, Riverside; and the American Museum of Natural History—concludes that mammals began diversifying before the K-Pg extinction as the result of continental drifting, which caused the Earth&#39;s land masses to drift apart and come back together over millions of years. Another pulse of diversification occurred immediately following the K-Pg extinction of the dinosaurs, when mammals had more room, resources and stability.
</p><p>This accelerated rate of diversification led to the rich diversity of mammal lineages—such as carnivores, primates and hoofed animals—that share the Earth today.


											  													    </p>
											  
											  <p>Murphy and Foley&#39;s research was funded by the National Science Foundation and is one part of the Zoonomia Project led by Elinor Karlsson and Kerstin Lindblad-Toh, of the Broad Institute, which also compares mammal genomes to understand the basis of remarkable phenotypes—the expression of certain genes such as brown vs. blue eyes—and the origins of disease.
</p><p>Foley pointed out that the diversity among <a href="https://phys.org/tags/placental+mammals/" rel="tag">placental mammals</a> is exhibited both in their physical traits and in their extraordinary abilities.
</p><p>&#34;Mammals today represent enormous evolutionary diversity—from the whizzing flight of the tiny bumblebee bat to the languid glide of the enormous Blue Whale as it swims through Earth&#39;s vast oceans. Multiple species have evolved to echolocate, some produce venom, while others have evolved cancer resistance and viral tolerance,&#34; she said.
</p><p>&#34;Being able to look at shared differences and similarities across the <a href="https://phys.org/tags/mammalian+species/" rel="tag">mammalian species</a> at a genetic level can help us figure out the parts of the genome that are critical to regulate the expression of genes,&#34; she continued. &#34;Tweaking this genomic machinery in different species has led to the diversity of traits that we see across today&#39;s living mammals.&#34;
</p><p>Murphy shared that Foley&#39;s resolved phylogeny of mammals is crucial to the goals of the Zoonomia Project, which aims to harness the power of comparative genomics as a tool for human medicine and biodiversity conservation.
</p><p>&#34;The Zoonomia Project is really impactful because it&#39;s the first analysis to align 241 diverse mammalian genomes at one time and use that information to better understand the human genome,&#34; he explained. &#34;The major impetus for putting together this big data set was to be able to compare all of these genomes to the human genome and then determine which parts of the human genome have changed over the course of mammalian evolutionary history.&#34;
</p><p>Determining which parts of genes can be manipulated and which parts cannot be changed without causing harm to the gene&#39;s function is important for human medicine. A recent study in <a href="https://www.science.org/doi/10.1126/scitranslmed.abf4077"><i>Science Translational Medicine</i></a> led by one of Murphy and Foley&#39;s colleagues, Texas A&amp;M geneticist Dr. Scott Dindot, used the comparative genomics approach to develop a <a href="https://medicalxpress.com/news/2023-03-groundbreaking-angelman-syndrome.html">molecular therapy for Angelman syndrome</a>, a devastating, rare neurogenetic disorder that is triggered by the loss of function of the maternal <i>UBE3A</i> gene in the brain.
</p><p>Dindot&#39;s team took advantage of the same measures of evolutionary constraint identified by the Zoonomia Project and applied them to identify a crucial but previously unknown genetic target that can be used to rescue the expression of <i>UBE3A</i> in human neurons.
</p><p>Murphy said expanding the ability to compare mammalian genomes by using the largest dataset in history will help develop more cures and treatments for other species&#39; ailments rooted in genetics, including cats and dogs.
</p><p>&#34;For example, cats have physiological adaptations rooted in unique mutations that allow them to consume an exclusively high-fat, high-protein diet that is extremely unhealthy for humans,&#34; Murphy explained. &#34;One of the beautiful aspects of Zoonomia&#39;s 241-species alignment is that we can pick any species (not just human) as the reference and determine which parts of that species&#39; genome are free to change and which ones cannot tolerate change. In the case of cats, for example, we may be able to help identify genetic adaptations in those species that could lead to therapeutic targets for cardiovascular disease in people.&#34;
</p><p>Murphy and Foley&#39;s phylogeny also played an instrumental role in many of the subsequent papers that are part of the project.
</p><p>&#34;It&#39;s trickle-down genomics,&#34; Foley explained. &#34;One of the most gratifying things for me in working as part of the wider project was seeing how many different research projects were enhanced by including our phylogeny in their analyses. This includes studies on conservation genomics of endangered species to those that looked at the evolution of different complex human traits.&#34;
</p><p>Foley said it was both meaningful and rewarding to definitively answer the heavily debated question about the timing of <a href="https://phys.org/tags/mammal/" rel="tag">mammal</a> origins and to produce an expanded phylogeny that lays the foundation for the next several generations of researchers.
</p><p>&#34;Going forward, this massive genome alignment and its historical record of mammalian genome evolution will be the basis of everything that everyone&#39;s going to do when they&#39;re asking comparative questions in mammals,&#34; she said. &#34;That is pretty cool.&#34;
										 																				
																				</p><div>
																						<p><strong>More information:</strong>
												Nicole M. Foley et al, A genomic timescale for placental mammal evolution, <i>Science</i> (2023).  <a data-doi="1" href="https://dx.doi.org/10.1126/science.abl8189" target="_blank">DOI: 10.1126/science.abl8189</a>
																						
																						</p>
																					</div>
                               											
																					
                              										                                        
										<!-- print only -->
										<div>
											 <p><strong>Citation</strong>:
												Researchers use genomes of 241 species to redefine mammalian tree of life (2023, April 28)
												retrieved 14 May 2023
												from https://phys.org/news/2023-04-genomes-species-redefine-mammalian-tree.html
											 </p>
											 <p>
											 This document is subject to copyright. Apart from any fair dealing for the purpose of private study or research, no
											 part may be reproduced without the written permission. The content is provided for information purposes only.
											 </p>
										</div>
                                        
									</div></div>
  </body>
</html>
