<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <title>James Routley | Feed</title>
    <link
      rel="stylesheet"
      type="text/css"
      href="../styles.css"
      media="screen"
    />
  </head>
  <body>
    <a href="/index.html">Back</a>
    <a href="https://www.owlposting.com/p/rna-structure-prediction-is-hard">Original</a>
    <h1>RNA structure prediction is hard. How much does that matter?</h1>
    
    <div id="readability-page-1" class="page"><div><div dir="auto"><div><figure><a target="_blank" href="https://substackcdn.com/image/fetch/$s_!5rzr!,f_auto,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F4f882018-1b26-4247-9b63-352fcec1d49a_2912x1632.png" data-component-name="Image2ToDOM" rel=""><div><picture><source type="image/webp" srcset="https://substackcdn.com/image/fetch/$s_!5rzr!,w_424,c_limit,f_webp,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F4f882018-1b26-4247-9b63-352fcec1d49a_2912x1632.png 424w, https://substackcdn.com/image/fetch/$s_!5rzr!,w_848,c_limit,f_webp,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F4f882018-1b26-4247-9b63-352fcec1d49a_2912x1632.png 848w, https://substackcdn.com/image/fetch/$s_!5rzr!,w_1272,c_limit,f_webp,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F4f882018-1b26-4247-9b63-352fcec1d49a_2912x1632.png 1272w, https://substackcdn.com/image/fetch/$s_!5rzr!,w_1456,c_limit,f_webp,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F4f882018-1b26-4247-9b63-352fcec1d49a_2912x1632.png 1456w" sizes="100vw"/><img src="https://substackcdn.com/image/fetch/$s_!5rzr!,w_2400,c_limit,f_auto,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F4f882018-1b26-4247-9b63-352fcec1d49a_2912x1632.png" width="1200" height="672.5274725274726" data-attrs="{&#34;src&#34;:&#34;https://substack-post-media.s3.amazonaws.com/public/images/4f882018-1b26-4247-9b63-352fcec1d49a_2912x1632.png&#34;,&#34;srcNoWatermark&#34;:null,&#34;fullscreen&#34;:false,&#34;imageSize&#34;:&#34;large&#34;,&#34;height&#34;:816,&#34;width&#34;:1456,&#34;resizeWidth&#34;:1200,&#34;bytes&#34;:7963058,&#34;alt&#34;:null,&#34;title&#34;:null,&#34;type&#34;:&#34;image/png&#34;,&#34;href&#34;:null,&#34;belowTheFold&#34;:false,&#34;topImage&#34;:true,&#34;internalRedirect&#34;:&#34;https://www.owlposting.com/i/173694583?img=https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F4f882018-1b26-4247-9b63-352fcec1d49a_2912x1632.png&#34;,&#34;isProcessing&#34;:false,&#34;align&#34;:&#34;center&#34;,&#34;offset&#34;:false}" alt="" srcset="https://substackcdn.com/image/fetch/$s_!5rzr!,w_424,c_limit,f_auto,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F4f882018-1b26-4247-9b63-352fcec1d49a_2912x1632.png 424w, https://substackcdn.com/image/fetch/$s_!5rzr!,w_848,c_limit,f_auto,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F4f882018-1b26-4247-9b63-352fcec1d49a_2912x1632.png 848w, https://substackcdn.com/image/fetch/$s_!5rzr!,w_1272,c_limit,f_auto,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F4f882018-1b26-4247-9b63-352fcec1d49a_2912x1632.png 1272w, https://substackcdn.com/image/fetch/$s_!5rzr!,w_1456,c_limit,f_auto,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F4f882018-1b26-4247-9b63-352fcec1d49a_2912x1632.png 1456w" sizes="100vw" fetchpriority="high"/></picture><div><div></div></div></div></a></figure></div><p><em><span>Note: I am not an expert in RNA structure, and am extremely grateful to </span><a href="https://x.com/ceejhayess" rel="">Connor Stephens</a><span>, </span><a href="https://x.com/rishabh16_" rel="">Rishabh Anand</a><strong>, </strong><a href="https://x.com/ramyarangan" rel="">Ramya Rangan</a><span>, and </span><a href="https://x.com/chaitjo" rel="">Chaitanya K. Joshi</a><span>—all of whom are actual, bonafide experts—for their incredibly detailed comments on earlier drafts of this essay.</span></em><span> </span><em>All mistakes are, of course mine, and this essay should not be trusted to function as anything more than entertainment. Do your own research!</em></p><ol><li><p><a href="https://www.owlposting.com/i/173694583/introduction" rel="">Introduction</a></p></li><li><p><a href="https://www.owlposting.com/i/173694583/why-is-rna-structure-so-hard-to-model" rel="">Why is RNA structure so hard to model?</a></p></li><li><p><a href="https://www.owlposting.com/i/173694583/why-even-predict-rna-structure-in-the-first-place" rel="">Why even predict RNA structure in the first place?</a></p></li><li><p><a href="https://www.owlposting.com/i/173694583/how-much-do-we-stand-to-gain-if-rna-structure-prediction-improves" rel="">How much do we stand to gain if RNA structure prediction improves?</a></p></li><li><p><a href="https://www.owlposting.com/i/173694583/some-nuance" rel="">Some nuance</a></p></li></ol><p><span>One thing I’ve always wanted to write was ‘</span><em>a primer to RNA structure modeling</em><span>’. I know literally nothing about the field, other than that there are a few startups playing in the space, and have always been curious what exactly they were up to. But the release of Alphafold3—which can model RNA alongside proteins, DNA, and small molecules—dampened this desire. If a singular model solved the problem of RNA structure, who cares about the specifics of the field at large?</span></p><p><span>But while I was in San Francisco a few months back, I happened to chat with</span><a href="https://www.linkedin.com/in/connor-james-stephens/" rel=""> Connor Stephens</a><span>, a machine learning scientist at</span><a href="https://atomic.ai/" rel=""> Atomic AI</a><span>. You may recognize that startup, since their founder has the distinct honor of their PhD work in RNA structure modeling being on the cover of Science in 2021 for making</span><a href="https://www.science.org/doi/10.1126/science.abe5650" rel=""> a substantial advance in RNA structure prediction.</a></p><div><figure><a target="_blank" href="https://substackcdn.com/image/fetch/$s_!DyIx!,f_auto,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F867a6a37-bbb4-4656-bfff-76674cab7869_731x1024.png" data-component-name="Image2ToDOM" rel=""><div><picture><source type="image/webp" srcset="https://substackcdn.com/image/fetch/$s_!DyIx!,w_424,c_limit,f_webp,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F867a6a37-bbb4-4656-bfff-76674cab7869_731x1024.png 424w, https://substackcdn.com/image/fetch/$s_!DyIx!,w_848,c_limit,f_webp,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F867a6a37-bbb4-4656-bfff-76674cab7869_731x1024.png 848w, https://substackcdn.com/image/fetch/$s_!DyIx!,w_1272,c_limit,f_webp,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F867a6a37-bbb4-4656-bfff-76674cab7869_731x1024.png 1272w, https://substackcdn.com/image/fetch/$s_!DyIx!,w_1456,c_limit,f_webp,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F867a6a37-bbb4-4656-bfff-76674cab7869_731x1024.png 1456w" sizes="100vw"/><img src="https://substackcdn.com/image/fetch/$s_!DyIx!,w_1456,c_limit,f_auto,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F867a6a37-bbb4-4656-bfff-76674cab7869_731x1024.png" width="276" height="386.6265389876881" data-attrs="{&#34;src&#34;:&#34;https://substack-post-media.s3.amazonaws.com/public/images/867a6a37-bbb4-4656-bfff-76674cab7869_731x1024.png&#34;,&#34;srcNoWatermark&#34;:null,&#34;fullscreen&#34;:null,&#34;imageSize&#34;:null,&#34;height&#34;:1024,&#34;width&#34;:731,&#34;resizeWidth&#34;:276,&#34;bytes&#34;:null,&#34;alt&#34;:null,&#34;title&#34;:null,&#34;type&#34;:null,&#34;href&#34;:null,&#34;belowTheFold&#34;:false,&#34;topImage&#34;:false,&#34;internalRedirect&#34;:null,&#34;isProcessing&#34;:false,&#34;align&#34;:null,&#34;offset&#34;:false}" alt="" srcset="https://substackcdn.com/image/fetch/$s_!DyIx!,w_424,c_limit,f_auto,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F867a6a37-bbb4-4656-bfff-76674cab7869_731x1024.png 424w, https://substackcdn.com/image/fetch/$s_!DyIx!,w_848,c_limit,f_auto,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F867a6a37-bbb4-4656-bfff-76674cab7869_731x1024.png 848w, https://substackcdn.com/image/fetch/$s_!DyIx!,w_1272,c_limit,f_auto,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F867a6a37-bbb4-4656-bfff-76674cab7869_731x1024.png 1272w, https://substackcdn.com/image/fetch/$s_!DyIx!,w_1456,c_limit,f_auto,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F867a6a37-bbb4-4656-bfff-76674cab7869_731x1024.png 1456w" sizes="100vw"/></picture><div><div></div></div></div></a></figure></div><p><span>But it was long unclear to me what exactly Atomic AI exactly did in terms of R&amp;D. This isn’t a startup post, I’m not planning to explain what their therapeutic goals are. What I </span><strong>was</strong><span> curious about was why they continue to have an ML team despite the RNA problem being seemingly solved by Alphafold3. So, I posed that question to Connor.</span></p><p>Connor told me something very fascinating: not only did Alphafold3 not solve the problem of RNA structure prediction, RNA may be one of the last structure prediction problems to be solved. The rest of the conversation was so incredibly fun that, midway through it, I decided it’d make for a great article to write about.</p><p><span>On face value, the answer is pretty simple: experimentally determined RNA structures deposited in public repositories are both ridiculously small in number and of much lower quality than you’d naively expect.</span><a href="https://www.biorxiv.org/content/10.1101/2024.06.13.598780v1.full.pdf" rel=""> A quote from a paper best explains this</a><span>:</span></p><blockquote><p><em><span>There is a huge disparity in protein and RNA data. Even if there is a higher proportion of RNAs than proteins in the living, this is not reflected in the available data: only a small amount of 3D RNA structures are known. Up to June 2024, 7,759 RNA structures were deposited in the Protein Data Bank (34), compared to 216,212 protein structures. The quality and diversity of data are also different: a huge proportion of RNAs come from the same families. It implies several redundant structures that could prevent a model from being generalized to other families. In addition, a huge amount of RNA families have not yet solved structures in the PDB. </span><strong>This means there is no balanced and representative proportion of RNA families through the known structures.</strong></em></p></blockquote><p>The obvious follow-up question is: why? Apparently, RNA is a good fit for basically none of the existing structure determination methods. But again, why?</p><p><span>Connor told me that RNA is famous for being perhaps one of the most flexible biomolecules to exist as a category, with an almost absurd number of conformational degrees of freedom. Each nucleotide has more torsion angles than an amino acid, and the lack of a bulky side chain—like those in amino acids—means there’s very little steric hindrance to keep the backbone from flopping around. Now, keep in mind, this is not to say that RNA is </span><strong>unstructured</strong><span>. Unstructured has a particular meaning, that the energy landscape is flat, with no favored conformational structure.</span><a href="https://www.pnas.org/doi/10.1073/pnas.2112677119" rel=""> But this isn’t the case for RNA</a><span>, which do have preferred conformational structures, there are just many of them that they constantly flip in between.</span></p><p><span>This all implies that RNA is a very bad fit for X-ray crystallography, which requires orderly, repeating conformations to arrange into a crystal. It is also a bad fit for cryo-EM (</span><a href="https://www.owlposting.com/p/a-primer-on-ml-in-cryo-electron-microscopy" rel="">a subject I’ve written about in detail before</a><span>), given both the </span><strong>extreme</strong><span> conformational heterogeneity of it and how typically small the biomolecule is,</span><a href="https://www.nature.com/articles/s41592-020-0878-9" rel=""> though this is increasingly being addressed</a><span>. Finally, NMR, which, while more forgiving when it comes to flexibility and heterogeneity, is generally limited to </span><strong>very</strong><span> small RNA structures. Once the RNA goes beyond ~50 nucleotides, the spectra start overlapping and the resolution being insufficient to observe anything useful. And lots of important RNA lies beyond that size! </span></p><p><span>I’ve attached some nuance about NMR and cryo-EM in the footnotes.</span><span><a data-component-name="FootnoteAnchorToDOM" id="footnote-anchor-1-173694583" href="https://www.owlposting.com/p/rna-structure-prediction-is-hard#footnote-1-173694583" target="_self" rel="">1</a></span></p><p>This means that there are really only two RNA structures that can be physically characterized: ones that have been artificially stabilized, or ones that are evolutionarily constrained to hold a single dominant conformation.</p><p>The first category includes structures coaxed into rigidity by heavy metal ions, engineered base modifications, or even crystallization chaperones. But of course, this raises a worrying question: are you really measuring the native structure, or just the structure you forced it into? The second category is rarer: RNAs that, through evolutionary pressure, have converged on a stable structure for a functional reason. There are no caveats there, only that trying to train a model on these nucleotide sequences will inevitably bias it towards unusually stable RNA structures.</p><p><span>Well, we shouldn’t let all of this get us down. Many impossible problems are being solved day-after-day in this field. Even if RNA modeling has all the characteristics of being hard to do—huge distributional space of possible outputs for a given input </span><strong>and</strong><span> low number of input data points—surely, some headway has been made in the problem. Consider Alphafold3: how well does it </span><strong>actually</strong><span> do on the RNA structure prediction problem?</span></p><p><span>A well-named paper titled</span><a href="https://journals.iucr.org/d/issues/2025/02/00/lie5001/index.html" rel=""> Has AlphaFold3 achieved success for RNA?</a><span> tries to answer this question. From the article:</span></p><blockquote><p><em>The best models from the CASP-RNA competition, which are human-guided, outperform AlphaFold3….</em></p><p><em><span>….On the other hand, AlphaFold3 shows a cumulative sum of metrics greater than the other methods for the other test sets (p-value &lt; 10</span><sup>−5</sup><span> for RNA-Puzzles, p-value &lt; 10</span><sup>−4</sup><span> for RNASolo). </span></em></p><p><em>For RNA-Puzzles, the challenge-best solutions are from older solutions with less advanced architectures compared with the more recent CASP-RNA solutions.</em></p><p><em>For the RNA3DB_0 data set, the performance of AlphaFold3 is slightly better compared with RhoFold, which gives a better RMSD but a worse MCQ and LCS-TA. </em></p><p><em>AlphaFold3 always has a high MCQ value, indicating that it returns structures which are more physically plausible than ab initio methods (which use physics properties in their predictions).</em></p><p><em> Nonetheless, it does not always have the best RMSD (outperformed in CASP-RNA and RNA3DB_0), suggesting that AlphaFold3 does not always have the best alignment (in terms of all atoms) compared with the reference structure.</em></p></blockquote><p><span>In short, while Alphafold3 is certainly an improvement in some categories of RNA—namely being the only RNA structure prediction method that can model very large RNA’s well—</span><strong>it does not solve the problem outright, and can be outperformed through tailored methods.</strong></p><p><span>Another </span><a href="https://www.biorxiv.org/content/10.1101/2025.04.30.651414v1.full.pdf" rel="">slightly more recent paper </a><span>says something similar, and gives some insight into the practical meaning of these benchmarks, saying ‘</span><em>Boltz-1 and AlphaFold3, make acceptable predictions for about half of the individual RNA chains and complexes.</em><span>’. The authors further note that the results get far worse if you deviate into more structurally unique RNA space (bolding added by me):</span></p><blockquote><p><em><span>We observed that prediction accuracy, as measured by TM-score, generally increased with higher structural similarity to the training set for all methods. </span><strong>The mean TM-score is below 0.1 for the category with the least similarity and increases gradually to over 0.6 for the category with the highest similarity to the training set</strong><span>. This suggests that AlphaFold3 and other methods tend to perform better when the target structure is more similar to motifs it encountered during training, highlighting the limitation of current methods in predicting unseen and structurally divergent RNAs.</span></em></p></blockquote><p>Neat! </p><p><span>I could end the essay here, because this really did cover most of Connor and I’s conversation. There is a lot more that could be said about how difficult benchmarking can be in the RNA ML world, the weak co-evolutionary signal in RNA MSA’s, how even the existing set of RNA structures are made worse by the fact that they are almost always in complex with a protein, and (hearsay) that you likely need experimentally-determined templates/molecular-dynamics to get good structure predictions. </span><a href="https://academic.oup.com/nar/article/51/18/9522/7272628?login=false" rel="">This paper discusses all that in more</a><span> detail if you&#39;re curious, but my main question got answered!</span></p><p>But the more I talked to people in the RNA space while writing this essay, the more I began to ask a new question: how important is this problem anyway?</p><p><span>For the protein-heads reading this, we know that protein structure actually means something quite fundamental. A protein’s three-dimensional fold is usually synonymous with its biological role: an enzyme pocket is what catalyzes a reaction, an antibody groove is what binds an antigen, a receptor domain is what recognizes a ligand. We can hem and haw about dynamics or post-translational tweaks, but the basic architecture is what makes the protein what it is. </span><a href="https://www.cell.com/cell/fulltext/S0092-8674(24)00005-9?__cf_chl_rt_tk=zSgWgFxiHRjxK1KWs_juQiWifKg.CYPamR_HBg0GuBY-1755989805-1.0.1.1-RWSLrfc.fJDowEjjNVanp4kgVjTFHoRUF9k_bvmN_sE" rel="">Protein structure isn’t exactly truth</a><span>, but structure </span><strong>can</strong><span> be a proxy for truth a sufficiently high fraction of the time.</span></p><p><span>RNA is not like this at all. It’s actually really, really, really situational when the structure of RNA matters in a therapeutic context. Well, to be more nuanced, structure </span><strong>always</strong><span> matters, but there is a very significant split what ‘structure’ even means for this biomolecule: secondary structure and tertiary structure (</span><a href="https://www.researchgate.net/figure/RNA-primary-left-secondary-middle-and-tertiary-structures-right-The-RNA-folding_fig1_354169706" rel="">image from here</a><span>):</span></p><div><figure><a target="_blank" href="https://substackcdn.com/image/fetch/$s_!Bwzd!,f_auto,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F4fa53b91-7de4-4403-a17c-b83d388f7e21_1456x807.png" data-component-name="Image2ToDOM" rel=""><div><picture><source type="image/webp" srcset="https://substackcdn.com/image/fetch/$s_!Bwzd!,w_424,c_limit,f_webp,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F4fa53b91-7de4-4403-a17c-b83d388f7e21_1456x807.png 424w, https://substackcdn.com/image/fetch/$s_!Bwzd!,w_848,c_limit,f_webp,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F4fa53b91-7de4-4403-a17c-b83d388f7e21_1456x807.png 848w, https://substackcdn.com/image/fetch/$s_!Bwzd!,w_1272,c_limit,f_webp,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F4fa53b91-7de4-4403-a17c-b83d388f7e21_1456x807.png 1272w, https://substackcdn.com/image/fetch/$s_!Bwzd!,w_1456,c_limit,f_webp,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F4fa53b91-7de4-4403-a17c-b83d388f7e21_1456x807.png 1456w" sizes="100vw"/><img src="https://substackcdn.com/image/fetch/$s_!Bwzd!,w_1456,c_limit,f_auto,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F4fa53b91-7de4-4403-a17c-b83d388f7e21_1456x807.png" width="531" height="294.3111263736264" data-attrs="{&#34;src&#34;:&#34;https://substack-post-media.s3.amazonaws.com/public/images/4fa53b91-7de4-4403-a17c-b83d388f7e21_1456x807.png&#34;,&#34;srcNoWatermark&#34;:null,&#34;fullscreen&#34;:null,&#34;imageSize&#34;:null,&#34;height&#34;:807,&#34;width&#34;:1456,&#34;resizeWidth&#34;:531,&#34;bytes&#34;:null,&#34;alt&#34;:null,&#34;title&#34;:null,&#34;type&#34;:null,&#34;href&#34;:null,&#34;belowTheFold&#34;:true,&#34;topImage&#34;:false,&#34;internalRedirect&#34;:null,&#34;isProcessing&#34;:false,&#34;align&#34;:null,&#34;offset&#34;:false}" alt="" srcset="https://substackcdn.com/image/fetch/$s_!Bwzd!,w_424,c_limit,f_auto,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F4fa53b91-7de4-4403-a17c-b83d388f7e21_1456x807.png 424w, https://substackcdn.com/image/fetch/$s_!Bwzd!,w_848,c_limit,f_auto,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F4fa53b91-7de4-4403-a17c-b83d388f7e21_1456x807.png 848w, https://substackcdn.com/image/fetch/$s_!Bwzd!,w_1272,c_limit,f_auto,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F4fa53b91-7de4-4403-a17c-b83d388f7e21_1456x807.png 1272w, https://substackcdn.com/image/fetch/$s_!Bwzd!,w_1456,c_limit,f_auto,q_auto:good,fl_progressive:steep/https%3A%2F%2Fsubstack-post-media.s3.amazonaws.com%2Fpublic%2Fimages%2F4fa53b91-7de4-4403-a17c-b83d388f7e21_1456x807.png 1456w" sizes="100vw" loading="lazy"/></picture><div><div></div></div></div></a></figure></div><p><strong>Thus far, everything we’ve talked about regarding the ‘difficulty of structure prediction’ has been for tertiary structure.</strong></p><p>Now, this separation exists for proteins as well! But it (somewhat) matters less for proteins. Usually we treat “protein structure” as a single concept because the hierarchy is tightly coupled: secondary structure (α-helices, β-sheets) stacks neatly into tertiary folds, which in turn map directly to function. You can often ignore the distinction because the two levels reinforce each other, and so everyone hyper-focuses on tertiary structures being the most important thing.</p><p><span>But for RNA, the distinction matters a lot, because secondary structure seems to be where </span><strong>most of the clinically relevant value of structure is</strong><span>. Tertiary RNA structure is important! But, as far as I can tell, the value of it is actually relatively limited in scope for therapeutic-relevant problems, partially due to the fact that RNA is just so flexible that a tertiary structure phenomenon like ‘</span><em>the binding site is buried in the core</em><span>’ can immediately be undercut by that same core suddenly flopping out in a new conformation.</span></p><p><span>And, just as is the case for proteins, RNA secondary structure is far easier to predict than RNA tertiary structure. It’s still comparatively hard, in the sense that secondary protein structure is basically something people don’t ever worry about, and secondary RNA structure has only just recently reached those same accuracy levels. </span><a href="https://www.biorxiv.org/content/10.1101/2025.05.06.652459v1" rel="">A paper that analyzed the performance of RNA models at CASP16 had this to say:</a></p><blockquote><p><em>Complex and novel targets appear well beyond current capabilities for NA 3D structure prediction. However, RNA folding can be simplified into a hierarchical process: secondary structure – the pattern of canonical base pairs – forms creating a set of RNA stems which are then stitched into the overall 3D fold…</em></p><p><em><span>CASP16 offered the prospect of carrying out tests of secondary structure accuracy prospectively. The secondary structure of all targets, here defined as the list of all Watson-Crick-Franklin and Wobble pairs, turned out to be predicted to a high level of accuracy (Supplemental Figure 3A)...</span><strong>The trend in RNA secondary structure performance is more reminiscent of the performance observed in current protein 3D structure prediction, suggesting these prediction algorithms are reaching sufficient accuracy in their prediction of secondary structure to be important and useful in structural research.</strong></em></p></blockquote><p>Not too bad!</p><p><span>Returning back to our claim that ‘</span><em>secondary structure is most of what you need</em><span>’, let’s convince ourselves of this by walking through the major classes of RNA-based therapeutics and the importance of secondary versus tertiary structure.</span></p><p><span>The most famous form of therapy here is exogenous </span><strong>mRNA</strong><span>, and tertiary structure doesn’t seem to matter much there. I have two proof points for this. One, this</span><a href="https://blog.genewiz.com/fine-tuning-mrna-structure-for-better-mrna-therapies" rel=""> mRNA optimization article from GeneWiz</a><span> mentions secondary-structure optimization (e.g. preventing</span><a href="https://microbenotes.com/stem-loop-hairpin-loop/" rel=""> hairpins</a><span>), but not tertiary structure. Two, just logically thinking about it, the job of mRNA is to be fed into the ribosome and translated into protein, so as long as the coding region is readable and initiation isn’t blocked (hence probably why hairpins are undesirable), why would it matter for the RNA to maintain any particular higher-order fold?</span></p><p><span>Then there’s </span><strong>antisense oligonucleotides</strong><span>, or ASO. All this is is a short synthetic strand of RNA (usually 15–25 bases long) that binds to a complementary sequence of an RNA. Once bound, it can block translation directly by preventing ribosome access, alter splicing by blocking splice sites or enhancers/silencers, or a few other things. But in all of these cases, </span><strong>all that matters is that the ASO can actually base-pair with its intended target</strong><span>.</span><a href="https://journals.plos.org/plosone/article?id=10.1371%2Fjournal.pone.0101752&amp;u" rel=""> And that comes down to secondary structure accessibility:</a><span> </span><strong>is the binding site exposed or not?</strong><span> Once again, this seems to be something that is largely answerable from secondary structure information, especially given how small ASO’s are.</span></p><p><span>For </span><strong>siRNA’s</strong><span>, or small interfering RNA, it’s nearly the same story as ASO’s,</span></p><p><span>Virtually the only time tertiary structure seems to matter for an RNA therapeutic is for </span><strong>aptamers </strong><span>and </span><strong>ribozymes</strong><span>. The former refers to short RNAs that fold into precise three-dimensional shapes capable of binding proteins or small molecules (e.g.</span><a href="https://pubmed.ncbi.nlm.nih.gov/32142605/" rel=""> theophylline aptamer</a><span>), and the latter refers to enzymatic RNAs with a precise catalytic site that are able to carry out chemical reactions. But, unlike all other classes of RNA therapeutics, approved drugs here are quite rare; aptamers have two and ribozymes</span><strong> </strong><span>have zero. There’s also </span><a href="https://pubmed.ncbi.nlm.nih.gov/20685165/" rel="">riboswitches</a><span>, which are a hazy combination of the two, and also have no released therapies.</span></p><p><span>This all said, we should also consider the other side too: RNA as </span><strong>targets</strong><span>. How important is secondary versus tertiary structure there?</span></p><p>Well, things do get muddier, because there isn’t really a standardized list of established RNA targets the same way there are for proteins. There’s mRNA, the tertiary structure of which is not exploited in any FDA-approved drugs (though we’ll discuss this again later on), but what else?</p><p><span>Well, for one, non-coding regions! Specifically,</span><a href="https://en.wikipedia.org/wiki/MicroRNA" rel=""> microRNAs</a><span> and</span><a href="https://tataa.com/tataa-blog/incrna-from-junk-to-a-therapeutic-goldmine/" rel=""> IncRNA</a><span>.</span></p><p>Given how small microRNA’s are (20~ nucleotides), I’d guess that tertiary structures don’t matter much there.</p><p><span>Curiously, LLM’s will, at first, insist that </span><strong>IncRNA’s</strong><span>, or “long noncoding RNAs”’ really benefit from accurate tertiary structure prediction. There’s some reason to believe that they are right. After all, they are usually above 200 (or 500,</span><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10213152/" rel=""> depending on who you ask</a><span>) nucleotides in length, so, unlike ASOs/siRNAs/microRNAs, IncRNA’s are sufficiently large where tertiary structures may have significant impacts. Unfortunately, the LLM seems to be a bit wrong here, partially because whether IncRNA’s even form global tertiary structures at all</span><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC3527312/" rel=""> has been a matter of intense debate</a><span> for a while, though</span><a href="https://www.tandfonline.com/doi/full/10.1080/10409238.2020.1828259#abstract" rel=""> circa 2020</a><span> it is seeming like at least some IncRNA’s do. But really, whether IncRNA’s have a global structure or not wouldn’t have even mattered anyway,</span><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC7850294/?" rel=""> </a><strong><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC7850294/?" rel="">because their modulation does not seem to actually depend on that global structure</a></strong><span>. Rather, it depends on a set of short nucleotide motifs scattered along an otherwise floppy backbone. Even if we could perfectly predict the full structure of an IncRNA tomorrow, it feels like it wouldn’t change any therapeutic decisions. Perhaps predictions of those local 3D motifs are valuable, but that’s an open question!</span></p><p><span>As far as I can tell, the only type of RNA target where tertiary structure is known to be important is </span><strong>rRNA, </strong><span>or ribosomal RNA. Unlike most RNAs, ribosomal RNAs actually must maintain specific tertiary folds, because, like ribozymes, they are enzymes in every meaningful sense. The</span><a href="https://en.wikipedia.org/wiki/Peptidyl_transferase_center" rel=""> peptidyl transferase center</a><span> of rRNA requires a highly specific</span><a href="https://pubmed.ncbi.nlm.nih.gov/10937990/" rel=""> three-dimensional geometry to orient its usual substrate</a><span>: tRNA. And some classes of approved antibiotics,</span><a href="https://microbeonline.com/macrolides-action-resistance/#:~:text=The%20macrolides%20bind%20to%20the%2050S%20ribosomal%20subunit,synthesis%2C%20however%2C%20different%20macrolide%20act%20at%20different%20steps." rel=""> macrolides for example</a><span>, are able to</span><a href="https://www.cell.com/structure/fulltext/S0969-2126%2803%2900022-4?" rel=""> block this catalysis site</a><span>, preventing (some) forms of bacteria from making proteins at all, eventually killing them. </span></p><p><span>It does seem like, from the outside, that accurate RNA tertiary structure predictions here would be helpful, given this line from a </span><a href="https://pubmed.ncbi.nlm.nih.gov/10347223/" rel="">paper discussing where antibiotics bind to RNA</a><span>:</span></p><blockquote><p><em>For spectinomycin, the apparent binding site and the affected cross linking site are distant in the secondary structure but are close in tertiary structure in several recent models, indicating a localized effect. For tetracycline, the apparent binding sites are significantly separated in both the secondary and the three-dimensional structures, suggesting a more regional effect.</em></p></blockquote><p>In other words, there is a large deviation in what secondary structure tells you, and what tertiary structure tells you! </p><p><span>This said, a few commenters on this essay noted that while this is an area where 3D structure is useful,</span><strong> it almost certainly isn’t a bottleneck</strong><span> </span><strong>due to the relative abundance of existing rRNA structures and ease of gathering new ones.</strong></p><p><span>So, aptamers and rRNA are virtually the only two areas that (today) truly benefit from detailed tertiary structure modeling </span><strong>and</strong><span> have some things in the clinic. For mRNAs, ASOs, siRNAs, and most lncRNAs, the biology seems to collapse down to local accessibility and motif recognition. Both of these are sufficiently described by secondary structure, and that is decently well predicted by existing models! Tertiary folds, though definitively far from being well-predicted, don’t actually seem to influence much…at least as far as I can tell.</span></p><p>So why do people still work on the tertiary structure prediction problem? Is it all just for better ribosome-centric antibiotics and aptamers?</p><p>Well, in the immediate short term, it does seem like antibiotics and aptamers are really the field&#39;s best bets.</p><p><span>This is nothing to sneeze at! On the antibiotic side, we </span><strong>do</strong><span> need better antibiotics to account for the current ‘</span><em>antibiotic resistance</em><span>’ thing that’s been going on for the past decade, so why not elect ribosome-targeting antibiotics? This said, we should immediately drown our hopes that better ribosomal drugs will actually change the resistance trend-line. Naively, one would hope that things that interfere with rRNA functioning should be quite hard to adapt to—after all, elements of the ribosome are canonically known for being extremely conserved. And that is true, but resistance manages to evolve anyway, including via, interestingly enough,</span><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC90232/?" rel=""> post-transcriptional-modifications that prevent the antibiotic from binding to rRNA.</a></p><p><span>Of course, the real issue with antibiotics has little to do with scientific ideas, and more to do with economics. A funny paragraph I found from an interview with the lead author of a recent</span><a href="https://en.wikipedia.org/wiki/Cresomycin" rel=""> ‘new rRNA antibiotic</a><span>’ paper</span><a href="https://cen.acs.org/pharmaceuticals/drug-discovery/antibiotic-built-better-binding/102/web/2024/02" rel=""> had this to say:</a></p><blockquote><p><em>…there is an argument that the difficulty making successful antibiotic drugs has more to do with business models than with molecules. When asked about this, Myers says, “Do I worry about the broken business model for antibiotics development? Are you kidding? Every day. That may be the most challenging problem of the lot, and it is not one that I can solve. Synthesizing new antibiotics—in that, I feel confident.”</em></p></blockquote><p><span>One related note is that RNA structure may not only be useful for targeting bacterial rRNA, but also viral RNA. A particularly famous case here is a</span><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC6061845/#abstract1" rel=""> paper that developed a protein that can bind to a structured RNA element in the HIV virus</a><span>, impairing transcription of it (albeit in an in-vitro setting). Though this has yet to lead to any approved drugs, the subject is, </span><a href="https://pubmed.ncbi.nlm.nih.gov/37686306/" rel="">according to one review paper</a><span>, promising.</span></p><p><span>Moving onto the aptamer side, though it is still early days, the future is interesting. Circa 2024, de novo RNA aptamer design is currently at the ‘</span><em><a href="https://www.nature.com/articles/s43588-024-00720-6" rel="">we can redesign existing things</a></em><span>’, which is a necessary step on the way to ‘</span><em>we can redesign existing things to make them bette</em><span>r’, but we’re not there yet. What’s the therapeutic utility of an aptamer anyway? Basically the same uses one would have for an antibody for, </span><a href="https://www.lubio.ch/blog/aptamers" rel="">with a ton of side benefits:</a></p><blockquote><p><em>Aptamers have several advantages over antibodies, not least the fact that they can be produced quickly and easily without the need for animal use. Aptamers also benefit from low production costs, high batch-to-batch consistency, and functional stability when stored at room temperature, which gives them a long shelf-life and simplifies both transportation and storage. In addition, the low immunogenicity of aptamers makes them valuable tools for in vivo applications, while their small size compared to antibodies allows them to better penetrate cells and tissues. This can be especially useful when studying difficult-to-access targets such as those found within the tumor microenvironment. </em></p><p><em>On the flipside, aptamers are poorly suited to applications in which it is desirable to stimulate an immune response and may undergo rapid clearance in vivo unless they have been modified to prevent this.</em></p></blockquote><p>This is quite nice, but there’s a lot of modalities vying for the antibody throne, and many of those share similar benefits as aptamers. Beyond the scope of this essay for me to judge how large the value is here, but I’m sure it’s non-zero!</p><p>Every essay I write, I try to form a strong opinion to build my story on, and I’ve sketched out one such opinion here: most of the value of RNA structure is in secondary structure, predicted secondary structure is quite good, and tertiary structure has a limited set of use cases. I think the argument for this position is decently strong.</p><p><span>But I should note that the take I have here is not a universally held opinion for those in the field, and is very much a ‘</span><em>I did my research, and this is the conclusion I came to</em><span>’. There are, I think, reasonable disagreements that people have had to this.</span></p><p><span>First, one paper titled </span><em><a href="https://www.pnas.org/doi/10.1073/pnas.2112677119" rel="">Thoughts on how to think (and talk) about RNA structure</a></em><span> argues that the seemingly high utility of secondary structure has a lot more to do with </span><strong>its historical ease of accessibility rather than the low utility of tertiary structure</strong><span>. Some context: most RNA secondary structure consists of what is called ‘Watson-Crick Pairs’, or just the tendency for RNA adenine (A) to match with Uracil (U) and Guanine (G) to pair with Cytosine (C). Non-Watson–Crick are just any hydrogen bond that forms outside of this, which typically can only be noticed in 3D space. The aforementioned paper says this about the two:</span></p><blockquote><p><em><span>Overall, the tendency to focus on Watson–Crick pairs may stem from the fact that they are the basis of nucleic acid hybridization and that they are easier to identify, draw, and rationally mutate. However, non-Watson–Crick pairing and stacking patterns in helical junctions and internal loops preform a 3D architecture that dictates the angles of emerging helices. </span><strong>As a result, specific parts of the RNA are spatially positioned to readily establish interactions often involving nucleotides that are far apart in sequence, but not in three dimensions….</strong><span>Non-Watson–Crick pairings combined with helical stacking give rise to structural motifs that provide the building blocks of many higher-order structures, including ultrastable tetraloops and their receptors, kink-turns, E-loops, etc.</span></em></p></blockquote><p><span>For instance, I mentioned earlier that the tertiary structure of mRNA targets is not exploited in any FDA-approved drugs. This is true, but they </span><strong>are</strong><span> being exploited in preclinical settings! For instance, </span><a href="https://arrakistx.com/" rel="">Arrakis Therapeutics</a><span>, a RNA-targeting-with-small molecules biotech startup with a very fun name, </span><a href="https://arrakistx.com/wp-content/uploads/2024/12/Cell-Symposia_A-Structure-based-Approach-to-Drugging-RNA-with-Small-Molecules.pdf" rel="">has this really interesting presentation</a><span> showing that multiple of their ligands are able to bind to conserved, accessible 3D pockets of mRNA of the MYC protein. This is </span><a href="https://ascopubs.org/doi/10.14694/EdBook_AM.2014.34.e497" rel="">a notoriously difficult protein to directly bind to</a><span>, but seemingly accessible through its mRNA. </span></p><p><span>Second and relatedly, I dismissed the value of mRNA tertiary structure, but there is an RNA modality that does something very similar to exogenous mRNA </span><strong>and</strong><span> has a very important tertiary structure: </span><a href="https://en.wikipedia.org/wiki/Circular_RNA" rel="">circRNA’s</a><span>, or circular RNA, which form a covalently closed continuous loop. One of the giants of the field (Mihir Metkar, </span><a href="https://www.biotecnika.org/2021/11/first-named-inventor-of-modernas-vaccine-is-an-indian-origin-scientist/" rel="">who was one of the primary contributors of the Moderna COVID-19 mRNA vaccine</a><span>) has written a great </span><a href="https://www.nature.com/articles/s41573-023-00827-x" rel="">Nature review article</a><span> over mRNA broadly, and did mention that circRNA’s must rely on a fundamentally different mechanism to initiate protein translation:</span></p><blockquote><p><em>Because canonical mammalian translation depends on 5′-cap recognition, mRNAs that lack a cap [e.g. circRNA’s] require an alternative means of translation initiation. One answer is an IRES (Fig. 6). </em></p><p><em>First discovered in picornaviruses, IRESs vary with respect to both their structural complexity and their reliance on endogenous initiation factors. In general, these two features are inversely correlated, with the simplest IRESs bypassing only the cap recognition step, whereas the most structurally complex bypass even AUG recognition, relying instead on intimate direct interactions with both the large and small ribosomal subunits.</em></p></blockquote><p><span>In other words, the IRES’s, or internal ribosome entry site, on a circRNA is the primary way it is recruited to the ribosome. </span><strong>This means that translation efficiency, tissue specificity, and even coding potential can hinge on whether the IRES is stable, accessible, and folded in the right way, meaning that it is a strong axis of control of a circRNA therapeutic!</strong><span> For example, </span><a href="https://www.nature.com/articles/s41587-022-01393-0#Sec16" rel="">engineering an IRES to improve translation efficacy is something that is fully possible to do.</a><span> But to do this at extreme scales, we’d likely need to be able do tertiary RNA structure prediction very well, since the three-dimensional structure of </span><a href="https://www.biorxiv.org/content/10.1101/2021.05.13.444024v1.full" rel="">IRES seems to matter a fair bit</a><span> (though, admittedly, most of the experimental structure studies of IRES are for non-therapeutically relevant ones). But why even use circRNA’s over mRNA’s? </span><a href="https://www.nature.com/articles/s41392-023-01561-x" rel="">One paper explains that quite well:</a></p><blockquote><p><em>Compared with the canonical linear mRNA used in vaccines, circRNAs have multiple advantages. </em></p><p><em>(1) CircRNAs are more stable and easy to store, whereas mRNA vaccines exhibit extreme instability because it is susceptible to degradation by RNases during transportation, storage, delivery, etc. Although nucleotide modifications of the mRNA backbone and UTR regions make mRNA more stable, this increases cost and complicates the manufacturing process, and the storage of the resulting vaccine still requires a low-temperature cold chain due to its suboptimal thermostability. CircRNAs without any modifications exhibit high stability and RNase resistance and can be stored at room temperature or under repeated freeze‒thaw conditions. </em></p><p><em>(2) CircRNAs without any modification exhibit fewer side effects. The cytotoxicity and side effects caused by mRNA vaccines are partly due to their high immunogenicity. Compared with modified mRNA, which has somewhat modulated high immunogenicity, circRNA exhibits lower immunogenicity, and lower cytotoxicity in the absence of modification. </em></p><p><em>(3) CircRNAs possess prolonged antigen-yielding capabilities and durable immune responses. The resulting longevity and thus prolonged antigen production contribute to antigen retention in antigen-presenting cells (APCs) and prolong antigen presentation.</em></p></blockquote><p>Convincing to me! Very excited to see how the circRNA space plays out.</p><p><span>Thirdly and finally, claiming that secondary structure for RNA is nearly solved is false, at least for mRNA used in the clinic. After all, the mRNA used in vaccines is quite biochemically distinct from the mRNA we naturally produce in one important element: the uridine nucleotide is replaced with a different chemical (the most common one being 1-methyl-pseudouridine, or m1Ψ), which is more immunologically ‘quiet’. </span><strong>This, as you may expect, messes up secondary structure prediction a fair bit, since there are basically zero experimentally determined mRNA structures with modified nucleotides</strong><span>. The same Mihir Metkar paper mentioned earlier says this:</span></p><blockquote><p><em>Although m1Ψ substitutions have little consequence on in vitro transcription or translational fidelity, as with other naturally occurring modified nucleotide, m1Ψ can substantially alter RNA secondary structure…these subtle differences in individual base-pair stabilities can lead to structural changes that alter mRNA functionality (for example, creating or disrupting a RNA binding protein (RBP) binding site)...</em></p><p><em>At present, the functional competence of RNA structures that contain modified nucleotides can only be assured by empirical testing.</em></p></blockquote><p><a href="https://arxiv.org/html/2501.04056v1#S4" rel="">There is ongoing work to solve this</a><span> problem but the datasets are still all quite small, as is typical in the RNA world.</span></p><p>And that’s it! Thank you for reading!</p><div data-component-name="FootnoteToDOM"><p><a id="footnote-1-173694583" href="https://www.owlposting.com/p/rna-structure-prediction-is-hard#footnote-anchor-1-173694583" contenteditable="false" target="_self" rel="">1</a></p><p><span>So, one, there are some cases of cryo-EM being useful for at least some RNA structures, like </span><a href="https://www.nature.com/articles/s41592-020-0878-9" rel="">here</a><span>, and that may accelerate as the field of cryo-EM reconstruction gets better and better. Second, NMR </span><strong>can</strong><span> be useful for RNA structure prediction problems in cases you have a crudely-predicted structure, but think you improve it by confirming the pairwise proximity of a </span><strong>handful</strong><span> of nucleotides. This is significantly more tractable, even for larger RNA, to do via NMR!</span></p></div></div></div></div>
  </body>
</html>
