<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <title>James Routley | Feed</title>
    <link
      rel="stylesheet"
      type="text/css"
      href="../styles.css"
      media="screen"
    />
  </head>
  <body>
    <a href="/index.html">Back</a>
    <a href="https://www.frontiersin.org/articles/10.3389/fviro.2022.834808/full">Original</a>
    <h1>SARS-CoV-2 contains part of a patented genetic sequence</h1>
    
    <div id="readability-page-1" class="page"><div>
<h2>Introduction</h2>
<p>Based on a recent publication describing insertion variants of SARS-CoV-2 (<a href="#B1">1</a>) we would like to bring the attention to our recent findings related to the sequence of the furin cleavage site (FCS) in SARS-CoV-2 Spike (S) protein. The SARS-CoV-2 causing the COVID-19 pandemic (<a href="#B2">2</a>) has 82.3% amino acid identity to bat coronavirus SL-CoVZC45, 77.2% amino acid identity to SARS-CoV, and 96.2% genome sequence identity to bat coronavirus RaTG13. While numerous point mutation differences exist between SARS-CoV-2 and RaTG13, only one insertion and dissimilarity exceeding 3 nucleotides (nt): a 12-nucleotide insertion coding for four amino acids (aa 681-684, PRRA) in the SARS-CoV-2 S protein has been discovered. This polybasic FCS differentiates SARS-CoV-2 from other b-lineage betacoronaviruses or any other sarbecovirus (<a href="#B3">3</a>). An FCS addition enhanced the infectivity of SARS Co-V-2 in 2019 (<a href="#B4">4</a>). The absence of this FCS results in attenuated SARS-CoV-2 variants useful for animal vaccination, accentuating its relevance to human infection (<a href="#B5">5</a>). This FCS is vital for human and ferret transmission (<a href="#B6">6</a>), expands viral tropism to human cells (<a href="#B7">7</a>), and is requisite for severe disease in two animal models of SARS-CoV-2 (<a href="#B8">8</a>).</p>
<h2>SARS-CoV-2 Spike Protein and MSH3</h2>
<p>A peculiar feature of the nucleotide sequence encoding the PRRA furin cleavage site in the SARS-CoV-2 S protein is its two consecutive CGG codons. This arginine codon is rare in coronaviruses: relative synonymous codon usage (RSCU) of CGG in pangolin CoV is 0, in bat CoV 0.08, in SARS-CoV 0.19, in MERS-CoV 0.25, and in SARS-CoV-2 0.299 (<a href="#B9">9</a>).</p>
<p>A BLAST search for the 12-nucleotide insertion led us to a 100% reverse match in a proprietary sequence (SEQ ID11652, nt 2751-2733) found in the US patent 9,587,003 filed on Feb. 4, 2016 (<a href="#B10">10</a>) (<a href="#F1">Figure 1</a>). Examination of SEQ ID11652 revealed that the match extends beyond the 12-nucleotide insertion to a 19-nucleotide sequence: 5′-CTACGTGCCCGCCGAGGAG-3′ (nt 2733-2751 of SEQ ID11652), such that the resulting mRNA would have 3′- GAUGCACGGGCGGCUCCUC-5′, or equivalently 5′- CU <strong>CCU CGG CGG GCA</strong> CGU AG-3′ (nucleotides 23547-23565 in the SARS-CoV-2 genome, in which the four bold codons yield PRRA, amino acids 681–684 of its spike protein). This is very rare in the NCBI BLAST database.</p>


<div>
<p><a href="https://www.frontiersin.org/files/Articles/834808/fviro-02-834808-HTML/image_m/fviro-02-834808-g001.jpg" name="figure1" target="_blank">
<img src="https://www.frontiersin.org/files/Articles/834808/fviro-02-834808-HTML/image_t/fviro-02-834808-g001.gif" id="F1" alt="www.frontiersin.org"/></a></p><p><strong>Figure 1</strong>. The origin of the furin sequence in SARS-CoV-2. Comparison of the protein sequences at the S1/S2 junction in SARS-CoV, RaTG13, and SARS-CoV-2 demonstrating the presence of the furin cleavage site (FCS) PRRA only in SARS-CoV-2. Based on a BLAST search of the 12-nucleotide stretch coding for the FCS PRRA, a 19-nucleotide long identical sequence was identified in the patented (US 958 7003) sequence Seq ID11652. SEQ ID11652 is transcribed to a MSH3 mRNA that appears to be codon optimized for humans. This 19-nucleotide sequence including 12 nucleotides coding for the FCS PRRA, present in the human MSH3 gene might have been introduced into the SARS-CoV-2 genome by the illustrated copy choice recombination mechanism in SARS-CoV-2 infected human cells overexpressing the MSH3 gene.</p>
</div>


<p>The correlation between this SARS-CoV-2 sequence and the reverse complement of a proprietary mRNA sequence is of uncertain origin. Conventional biostatistical analysis indicates that the probability of this sequence randomly being present in a 30,000-nucleotide viral genome is 3.21 ×10<sup>−11</sup> (<a href="#F2">Figure 2</a>).</p>


<div>
<p><a href="https://www.frontiersin.org/files/Articles/834808/fviro-02-834808-HTML/image_m/fviro-02-834808-g002.jpg" name="figure2" target="_blank">
<img src="https://www.frontiersin.org/files/Articles/834808/fviro-02-834808-HTML/image_t/fviro-02-834808-g002.gif" id="F2" alt="www.frontiersin.org"/></a></p><p><strong>Figure 2</strong>. Calculations of the probability of natural occurrence of the 19nt sequence under study. The SARS-CoV-2 genome is ~30,000 nucleotides long (P1). The patented sequence is ~3,300 nucleotides long (P2). The patented library encompasses 24&#39;712 sequences of varying lengths with median length being in the range of 3,300 nucleotides. Conventional probability calculations are given of the probability of the presence of a 19-nucleotide sequence in the human genome and in one of the patented library sequences.</p>
</div>


<p>The proprietary sequence SEQ ID11652, read in the forward direction, encodes a 100% amino acid match to the human mut S homolog 3 (MSH3) (<a href="#B9">9</a>). MSH3 is a DNA mismatch repair protein (part of the MutS beta complex) (<a href="#B11">11</a>). SEQ ID11652 is transcribed to a MSH3 mRNA that appears to be codon optimized for humans (<a href="#B12">12</a>). We did not find the 19-nucleotide sequence CTCCTCGGCGGGCACGTAG in any eukaryotic or viral genomes except SARS-CoV-2 with 100% coverage and identity in the BLAST database (<a href="#SM1">Supplementary Tables 1</a>–<a href="#SM1">3</a>).</p>
<h2>Discussion</h2>
<p>MSH3 replacement with a codon-optimized mRNA sequence for human expression likely has applications in cancers with mismatch repair deficiencies. While a portion of a reverse complement sequence being present in SARS-CoV-2 could be a random coincidence, other possibilities merit consideration.</p>
<p>Overexpression of MSH3 is known to interfere with mismatch repair (MSH2 sequestration from the MutS alpha complex comprising MSH2 and MSH6 results in MSH6 degradation and MutS alpha depletion) (<a href="#B13">13</a>), which holds virologic importance. Induction of DNA mismatch repair deficiency results in permissiveness of influenza A virus (IAV) infection of human respiratory cells and increased pathogenicity (<a href="#B14">14</a>). Mismatch repair deficiency may extend shedding of SARS-CoV-2 (<a href="#B15">15</a>, <a href="#B16">16</a>).</p>
<p>The absence of CTCCTCGGCGGGCACGTAG from any eukaryotic or viral genome in the BLAST database makes recombination in an intermediate host an unlikely explanation for its presence in SARS-CoV-2. A human-codon-optimized mRNA encoding a protein 100% homologous to human MSH3 could, during the course of viral research, inadvertently or intentionally induce mismatch repair deficiency in a human cell line, which would increase susceptibility to SARS-like viral infection. Infection of SEQ ID11652-MSH3-transduced human cells by a SARS-like virus could enable copy choice recombination (<a href="#B15">15</a>). Replication of SARS-CoV-2 and other single stranded RNA viruses with an RNA genome of positive polarity is initiated by the synthesis of negative strand RNA in the cytoplasm of infected cells (<a href="#B17">17</a>) (<a href="#F1">Figure 1</a>). The negative strand RNA is a template for synthesis of positive stranded RNA utilized for translation of non-structural proteins, the replication and transcription complex, or new virion capsids. Coronaviruses generate double stranded RNA at an early stage of infection through genomic replication and mRNA transcription (<a href="#B18">18</a>).</p>
<p>Acquisition of the reverse complement FCS sequence from an overexpressed positive sense MSH3 mRNA could occur through copy choice recombination with a negative sense SARS-CoV-2 RNA intermediate (<a href="#B15">15</a>), involving jumping from one template to another (<a href="#B19">19</a>) (<a href="#F1">Figure 1</a>). The homology between SARS-CoV-2 and other known coronaviruses is discontinued and most SARS-CoV-2 sequences derive from a relatively recent common ancestor with bat RaTG13. Moreover, similarity plots (SimPlots) have identified sudden changes in sequence identity between SARS-CoV-2 and RaTG13, signaling potential recombination events, which could explain the capability of SARS-CoV-2 binding to ACE2 through its RBD, which is not the case for the RaTG13 RBD (<a href="#B15">15</a>).</p>
<p>A criticism of this hypothesis is that the identified sequence is on the opposite strand of the open reading frame in SEQ ID11652. However, cells transfected with MSH3, which induce mismatch repair deficiency could have targeted double-stranded cDNA encoding SEQ ID11652. Such cells co-transfected with a SARS-like virus expressing RdRp could attach to this 19-nucleotide sequence (<a href="#B15">15</a>) and permit integration of a fragment from the negative strand into the viral genome, including the FCS, despite being on the opposite strand of the open reading frame. Mismatch repair mechanisms have enabled integration of short fragments from antisense strands in experimental models (<a href="#B20">20</a>, <a href="#B21">21</a>). Microhomology can direct recombination between the MSH3 and a SARS-like virus, which could take place at the 19-nucleotide sequence of interest.</p>
<p>The presence in SARS-CoV-2 of a 19-nucleotide RNA sequence encoding an FCS at amino acid 681 of its spike protein with 100% identity to the reverse complement of a proprietary MSH3 mRNA sequence is highly unusual. Potential explanations for this correlation should be further investigated.</p>
<h2>Data Availability Statement</h2>
<p>Publicly available datasets were analyzed in this study. This data can be found here: SEQ ID11652.</p>
<h2>Author Contributions</h2>
<p>The investigation and original draft were initiated by BA, AV, and AB. The visualization was performed by BA, AV, AB, and GP. The writing and editing were done by BA, AV, KL, GP, BU, VU, and AB. All authors contributed to the article and approved the submitted version.</p>
<h2>Conflict of Interest</h2>
<p>KL was employed by PanTherapeutics.</p> 
<p>The remaining authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.</p>
<h2>Publisher&#39;s Note</h2>
<p>All claims expressed in this article are solely those of the authors and do not necessarily represent those of their affiliated organizations, or those of the publisher, the editors and the reviewers. Any product that may be evaluated in this article, or claim that may be made by its manufacturer, is not guaranteed or endorsed by the publisher.</p>
<h2>Acknowledgments</h2>
<p>We are thankful for the contribution of Dr. Jian Ying, University of Utah, Eureka, USA to the probability analysis.</p>
<h2>Supplementary Material</h2>
<p id="SM1">The Supplementary Material for this article can be found online at: <a href="https://www.frontiersin.org/articles/10.3389/fviro.2022.834808/full#supplementary-material">https://www.frontiersin.org/articles/10.3389/fviro.2022.834808/full#supplementary-material</a></p>
<h2>References</h2>
<div>
<p><a name="B1" id="B1"></a> 1. Garushyants SK, Rogozin IB, Koonin EV. Template switching and duplications in SARS-CoV-2 genomes give rise to insertion variants that merit monitoring. <i>Commun Biol.</i> (2021) 4:1343. doi: 10.1038/s42003-021-02858-9</p>
<p><a href="https://pubmed.ncbi.nlm.nih.gov/34848826" target="_blank">PubMed Abstract</a> | <a href="https://doi.org/10.1038/s42003-021-02858-9" target="_blank">CrossRef Full Text</a> | <a href="http://scholar.google.com/scholar_lookup?author=SK+Garushyants&amp;author=IB+Rogozin&amp;author=EV+Koonin+&amp;publication_year=2021&amp;title=Template+switching+and+duplications+in+SARS-CoV-2+genomes+give+rise+to+insertion+variants+that+merit+monitoring&amp;journal=Commun+Biol.&amp;volume=4&amp;pages=1343" target="_blank">Google Scholar</a></p>
</div>
<div>
<p><a name="B2" id="B2"></a> 2. Wu F, Zhao S, Yu B, Chen YM, Wang W, Song ZG, et al. A new coronavirus associated with human respiratory disease in China. <i>Nature</i>. (2020) 579:265–9. doi: 10.1038/s41586-020-2008-3</p>
<p><a href="https://pubmed.ncbi.nlm.nih.gov/32296181" target="_blank">PubMed Abstract</a> | <a href="https://doi.org/10.1038/s41586-020-2008-3" target="_blank">CrossRef Full Text</a> | <a href="http://scholar.google.com/scholar_lookup?author=F+Wu&amp;author=S+Zhao&amp;author=B+Yu&amp;author=YM+Chen&amp;author=W+Wang&amp;author=ZG+Song+&amp;publication_year=2020&amp;title=A+new+coronavirus+associated+with+human+respiratory+disease+in+China&amp;journal=Nature&amp;volume=579&amp;pages=265-9" target="_blank">Google Scholar</a></p>
</div>
<div>
<p><a name="B3" id="B3"></a> 3. Coutard B, Valle C, de Lamballerie X, Canard B, Seidah NG, Decroly E. The spike glycoprotein of the new coronavirus 2019-nCoV contains a furin-like cleavage site absent in CoV of the same clade. <i>Antiviral Res.</i> (2020) 176:104742. doi: 10.1016/j.antiviral.2020.104742</p>
<p><a href="https://pubmed.ncbi.nlm.nih.gov/32057769" target="_blank">PubMed Abstract</a> | <a href="https://doi.org/10.1016/j.antiviral.2020.104742" target="_blank">CrossRef Full Text</a> | <a href="http://scholar.google.com/scholar_lookup?author=B+Coutard&amp;author=C+Valle&amp;author=X+de+Lamballerie&amp;author=B+Canard&amp;author=NG+Seidah&amp;author=E+Decroly+&amp;publication_year=2020&amp;title=The+spike+glycoprotein+of+the+new+coronavirus+2019-nCoV+contains+a+furin-like+cleavage+site+absent+in+CoV+of+the+same+clade&amp;journal=Antiviral+Res.&amp;volume=176&amp;pages=104742" target="_blank">Google Scholar</a></p>
</div>
<div>
<p><a name="B4" id="B4"></a> 4. Walls AC, Park YJ, Tortorici MA, Wall A, McGuire AT, Veesler D. Structure, Function, and Antigenicity of the SARS-CoV-2 Spike Glycoprotein. <i>Cell</i>. (2020) 181:281–92.e286. doi: 10.1016/j.cell.2020.02.058</p>
<p><a href="https://pubmed.ncbi.nlm.nih.gov/33306958" target="_blank">PubMed Abstract</a> | <a href="https://doi.org/10.1016/j.cell.2020.02.058" target="_blank">CrossRef Full Text</a> | <a href="http://scholar.google.com/scholar_lookup?author=AC+Walls&amp;author=YJ+Park&amp;author=MA+Tortorici&amp;author=A+Wall&amp;author=AT+McGuire&amp;author=D+Veesler+&amp;publication_year=2020&amp;title=Structure,+Function,+and+Antigenicity+of+the+SARS-CoV-2+Spike+Glycoprotein&amp;journal=Cell&amp;volume=181&amp;pages=281-92" target="_blank">Google Scholar</a></p>
</div>
<div>
<p><a name="B5" id="B5"></a> 5. Lau SY, Wang P, Mok BWY, Zhang AJ, Chu H, Lee ACY, et al. Attenuated SARS-CoV-2 variants with deletions at the S1/S2 junction. <i>Emerg Microbes Infect</i>. (2020) 9:837–42. doi: 10.1080/22221751.2020.1756700</p>
<p><a href="https://pubmed.ncbi.nlm.nih.gov/32301390" target="_blank">PubMed Abstract</a> | <a href="https://doi.org/10.1080/22221751.2020.1756700" target="_blank">CrossRef Full Text</a> | <a href="http://scholar.google.com/scholar_lookup?author=SY+Lau&amp;author=P+Wang&amp;author=BWY+Mok&amp;author=AJ+Zhang&amp;author=H+Chu&amp;author=ACY+Lee+&amp;publication_year=2020&amp;title=Attenuated+SARS-CoV-2+variants+with+deletions+at+the+S1%2FS2+junction&amp;journal=Emerg+Microbes+Infect&amp;volume=9&amp;pages=837-42" target="_blank">Google Scholar</a></p>
</div>
<div>
<p><a name="B6" id="B6"></a> 6. Peacock TP, Goldhill DH, Zhou J, Baillon L, Frise R, Swann OC, et al. The furin cleavage site in the SARS-CoV-2 spike protein is required for transmission in ferrets. <i>Nat Microbiol</i>. (2020) 6:899–909. doi: 10.1038/s41564-021-00908-w</p>
<p><a href="https://pubmed.ncbi.nlm.nih.gov/33907312" target="_blank">PubMed Abstract</a> | <a href="https://doi.org/10.1038/s41564-021-00908-w" target="_blank">CrossRef Full Text</a> | <a href="http://scholar.google.com/scholar_lookup?author=TP+Peacock&amp;author=DH+Goldhill&amp;author=J+Zhou&amp;author=L+Baillon&amp;author=R+Frise&amp;author=OC+Swann+&amp;publication_year=2020&amp;title=The+furin+cleavage+site+in+the+SARS-CoV-2+spike+protein+is+required+for+transmission+in+ferrets&amp;journal=Nat+Microbiol&amp;volume=6&amp;pages=899-909" target="_blank">Google Scholar</a></p>
</div>
<div>
<p><a name="B7" id="B7"></a> 7. Xia S, Lan Q, Su S, Wang X, Xu W, Liu Z, et al. The role of furin cleavage site in SARS-CoV-2 spike protein-mediated membrane fusion in the presence or absence of trypsin. <i>Signal Transd. Targeted Ther</i>. (2020) 5:92. doi: 10.1038/s41392-020-0184-0</p>
<p><a href="https://pubmed.ncbi.nlm.nih.gov/32532959" target="_blank">PubMed Abstract</a> | <a href="https://doi.org/10.1038/s41392-020-0184-0" target="_blank">CrossRef Full Text</a> | <a href="http://scholar.google.com/scholar_lookup?author=S+Xia&amp;author=Q+Lan&amp;author=S+Su&amp;author=X+Wang&amp;author=W+Xu&amp;author=Z+Liu+&amp;publication_year=2020&amp;title=The+role+of+furin+cleavage+site+in+SARS-CoV-2+spike+protein-mediated+membrane+fusion+in+the+presence+or+absence+of+trypsin&amp;journal=Signal+Transd.+Targeted+Ther&amp;volume=5&amp;pages=92" target="_blank">Google Scholar</a></p>
</div>

<div>
<p><a name="B9" id="B9"></a> 9. Kandeel M, Ibrahim A, Fayez M, Al-Nazawi M. From SARS and MERS CoVs to SARS-CoV-2: Moving toward more biased codon usage in viral structural and nonstructural genes. <i>J Med Virol</i>. (2020) 92:660–6. doi: 10.1002/jmv.25754</p>
<p><a href="https://pubmed.ncbi.nlm.nih.gov/32159237" target="_blank">PubMed Abstract</a> | <a href="https://doi.org/10.1002/jmv.25754" target="_blank">CrossRef Full Text</a> | <a href="http://scholar.google.com/scholar_lookup?author=M+Kandeel&amp;author=A+Ibrahim&amp;author=M+Fayez&amp;author=M+Al-Nazawi+&amp;publication_year=2020&amp;title=From+SARS+and+MERS+CoVs+to+SARS-CoV-2%3A+Moving+toward+more+biased+codon+usage+in+viral+structural+and+nonstructural+genes&amp;journal=J+Med+Virol&amp;volume=92&amp;pages=660-6" target="_blank">Google Scholar</a></p>
</div>
<div>
<p><a name="B10" id="B10"></a> 10. Bancel S, Chakraborty T, De Fougerolles A, Elbashir SM, John M, Roy A, et al. <i>Modified Polynucleotides for the Production of Oncology-Related Proteins and Peptides</i>. Cambridge, MA: United States Patent. (2016). </p>
<p><a href="http://scholar.google.com/scholar_lookup?author=S+Bancel&amp;author=T+Chakraborty&amp;author=A+De+Fougerolles&amp;author=SM+Elbashir&amp;author=M+John&amp;author=A+Roy+&amp;publication_year=2016&amp;title=Modified+Polynucleotides+for+the+Production+of+Oncology-Related+Proteins+and+Peptides" target="_blank">Google Scholar</a></p>
</div>
<div>
<p><a name="B11" id="B11"></a> 11. MacRae SL, McKnight Croken M, Calder RB, Aliper A, Milholland B, White RR, et al. DNA repair in species with extreme lifespan differences. <i>Aging</i>. (2015) 7:1171–84. doi: 10.18632/aging.100866</p>
<p><a href="https://pubmed.ncbi.nlm.nih.gov/26729707" target="_blank">PubMed Abstract</a> | <a href="https://doi.org/10.18632/aging.100866" target="_blank">CrossRef Full Text</a> | <a href="http://scholar.google.com/scholar_lookup?author=SL+MacRae&amp;author=M+McKnight+Croken&amp;author=RB+Calder&amp;author=A+Aliper&amp;author=B+Milholland&amp;author=RR+White+&amp;publication_year=2015&amp;title=DNA+repair+in+species+with+extreme+lifespan+differences&amp;journal=Aging&amp;volume=7&amp;pages=1171-84" target="_blank">Google Scholar</a></p>
</div>

<div>
<p><a name="B13" id="B13"></a> 13. Marra G, Iaccarino I, Lettieri T, Roscilli G, Delmastro P, Jiricny J. Mismatch repair deficiency associated with overexpression of the MSH3 gene. <i>Proc Natl Acad Sci USA</i>. (1998) 95:8568. doi: 10.1073/pnas.95.15.8568</p>
<p><a href="https://pubmed.ncbi.nlm.nih.gov/9671718" target="_blank">PubMed Abstract</a> | <a href="https://doi.org/10.1073/pnas.95.15.8568" target="_blank">CrossRef Full Text</a> | <a href="http://scholar.google.com/scholar_lookup?author=G+Marra&amp;author=I+Iaccarino&amp;author=T+Lettieri&amp;author=G+Roscilli&amp;author=P+Delmastro&amp;author=J+Jiricny+&amp;publication_year=1998&amp;title=Mismatch+repair+deficiency+associated+with+overexpression+of+the+MSH3+gene&amp;journal=Proc+Natl+Acad+Sci+USA&amp;volume=95&amp;pages=8568" target="_blank">Google Scholar</a></p>
</div>
<div>
<p><a name="B14" id="B14"></a> 14. Chambers BS, Heaton BE, Rausch K, Dumm RE, Hamilton JR, Cherry S, et al. DNA mismatch repair is required for the host innate response and controls cellular fate after influenza virus infection. <i>Nat Microbiol</i>. (2019) 4:1964–77. doi: 10.1038/s41564-019-0509-3</p>
<p><a href="https://pubmed.ncbi.nlm.nih.gov/31358986" target="_blank">PubMed Abstract</a> | <a href="https://doi.org/10.1038/s41564-019-0509-3" target="_blank">CrossRef Full Text</a> | <a href="http://scholar.google.com/scholar_lookup?author=BS+Chambers&amp;author=BE+Heaton&amp;author=K+Rausch&amp;author=RE+Dumm&amp;author=JR+Hamilton&amp;author=S+Cherry+&amp;publication_year=2019&amp;title=DNA+mismatch+repair+is+required+for+the+host+innate+response+and+controls+cellular+fate+after+influenza+virus+infection&amp;journal=Nat+Microbiol&amp;volume=4&amp;pages=1964-77" target="_blank">Google Scholar</a></p>
</div>
<div>
<p><a name="B15" id="B15"></a> 15. Gallaher WR. A palindromic RNA sequence as a common breakpoint contributor to copy-choice recombination in SARS-COV-2. <i>Arch Virol.</i> (2020) 165:2341–8. doi: 10.1007/s00705-020-04750-z</p>
<p><a href="https://pubmed.ncbi.nlm.nih.gov/32737584" target="_blank">PubMed Abstract</a> | <a href="https://doi.org/10.1007/s00705-020-04750-z" target="_blank">CrossRef Full Text</a> | <a href="http://scholar.google.com/scholar_lookup?author=WR+Gallaher+&amp;publication_year=2020&amp;title=A+palindromic+RNA+sequence+as+a+common+breakpoint+contributor+to+copy-choice+recombination+in+SARS-COV-2&amp;journal=Arch+Virol.&amp;volume=165&amp;pages=2341-8" target="_blank">Google Scholar</a></p>
</div>
<div>
<p><a name="B16" id="B16"></a> 16. Haque F, Lillie P, Haque F, Maraveyas A. Deficient DNA mismatch repair and persistence of SARS-CoV-2 RNA shedding: a case report of Hereditary Nonpolyposis Colorectal Cancer with COVID-19 infection. <i>BMC Infect Dis.</i> (2020) 21:854. doi: 10.1186/s12879-021-06500-1</p>
<p><a href="https://pubmed.ncbi.nlm.nih.gov/34418963" target="_blank">PubMed Abstract</a> | <a href="https://doi.org/10.1186/s12879-021-06500-1" target="_blank">CrossRef Full Text</a> | <a href="http://scholar.google.com/scholar_lookup?author=F+Haque&amp;author=P+Lillie&amp;author=F+Haque&amp;author=A+Maraveyas+&amp;publication_year=2020&amp;title=Deficient+DNA+mismatch+repair+and+persistence+of+SARS-CoV-2+RNA+shedding%3A+a+case+report+of+Hereditary+Nonpolyposis+Colorectal+Cancer+with+COVID-19+infection&amp;journal=BMC+Infect+Dis.&amp;volume=21&amp;pages=854" target="_blank">Google Scholar</a></p>
</div>
<div>
<p><a name="B17" id="B17"></a> 17. V&#39;kovski P, Kratzel A, Steiner S, Stalder H, Thiel V. Coronavirus biology and replication: implications for SARS-CoV-2. <i>Nat Rev</i>. (2021) 19:155–70. doi: 10.1038/s41579-020-00468-6</p>
<p><a href="https://pubmed.ncbi.nlm.nih.gov/33116300" target="_blank">PubMed Abstract</a> | <a href="https://doi.org/10.1038/s41579-020-00468-6" target="_blank">CrossRef Full Text</a> | <a href="http://scholar.google.com/scholar_lookup?author=P+V&#39;kovski&amp;author=A+Kratzel&amp;author=S+Steiner&amp;author=H+Stalder&amp;author=V+Thiel+&amp;publication_year=2021&amp;title=Coronavirus+biology+and+replication%3A+implications+for+SARS-CoV-2&amp;journal=Nat+Rev&amp;volume=19&amp;pages=155-70" target="_blank">Google Scholar</a></p>
</div>

<div>
<p><a name="B19" id="B19"></a> 19. Sola I, Almazan F, Zuniga S, Enjuanes L. Continuous and discontinuous RNA synthesis in coronaviruses. <i>Annu Rev Virol.</i> (2015) 2:265–88. doi: 10.1146/annurev-virology-100114-055218</p>
<p><a href="https://pubmed.ncbi.nlm.nih.gov/26958916" target="_blank">PubMed Abstract</a> | <a href="https://doi.org/10.1146/annurev-virology-100114-055218" target="_blank">CrossRef Full Text</a> | <a href="http://scholar.google.com/scholar_lookup?author=I+Sola&amp;author=F+Almazan&amp;author=S+Zuniga&amp;author=L+Enjuanes+&amp;publication_year=2015&amp;title=Continuous+and+discontinuous+RNA+synthesis+in+coronaviruses&amp;journal=Annu+Rev+Virol.&amp;volume=2&amp;pages=265-88" target="_blank">Google Scholar</a></p>
</div>
<div>
<p><a name="B20" id="B20"></a> 20. Rakosy-Tican E, Lörincz-Besenyei E, Molnár I, Thieme R, Hartung F, Sprink T, et al. New Phenotypes of potato co-induced by mismatch repair deficiency and somatic hybridization. <i>Front Plant Sci.</i> (2019) 10:3. doi: 10.3389/fpls.2019.00003</p>
<p><a href="https://pubmed.ncbi.nlm.nih.gov/30723483" target="_blank">PubMed Abstract</a> | <a href="https://doi.org/10.3389/fpls.2019.00003" target="_blank">CrossRef Full Text</a> | <a href="http://scholar.google.com/scholar_lookup?author=E+Rakosy-Tican&amp;author=E+Lörincz-Besenyei&amp;author=I+Molnár&amp;author=R+Thieme&amp;author=F+Hartung&amp;author=T+Sprink+&amp;publication_year=2019&amp;title=New+Phenotypes+of+potato+co-induced+by+mismatch+repair+deficiency+and+somatic+hybridization&amp;journal=Front+Plant+Sci.&amp;volume=10&amp;pages=3" target="_blank">Google Scholar</a></p>
</div>
<div>
<p><a name="B21" id="B21"></a> 21. Harmsen T, Klaasen S, van de Vrugt H, te Riele H. DNA mismatch repair and oligonucleotide end-protection promote base-pair substitution distal from a CRISPR/Cas9- induced DNA break. <i>Nucl Acids Res</i>. (2018) 46:2945–55. doi: 10.1093/nar/gky076</p>
<p><a href="https://pubmed.ncbi.nlm.nih.gov/29447381" target="_blank">PubMed Abstract</a> | <a href="https://doi.org/10.1093/nar/gky076" target="_blank">CrossRef Full Text</a> | <a href="http://scholar.google.com/scholar_lookup?author=T+Harmsen&amp;author=S+Klaasen&amp;author=H+van+de+Vrugt&amp;author=H+te+Riele+&amp;publication_year=2018&amp;title=DNA+mismatch+repair+and+oligonucleotide+end-protection+promote+base-pair+substitution+distal+from+a+CRISPR%2FCas9-+induced+DNA+break&amp;journal=Nucl+Acids+Res&amp;volume=46&amp;pages=2945-55" target="_blank">Google Scholar</a></p>
</div>
</div></div>
  </body>
</html>
