<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <title>James Routley | Feed</title>
    <link
      rel="stylesheet"
      type="text/css"
      href="../styles.css"
      media="screen"
    />
  </head>
  <body>
    <a href="/index.html">Back</a>
    <a href="https://www.nature.com/articles/s43588-024-00738-w">Original</a>
    <h1>Simulating C. elegans brain, body and environment interactions</h1>
    
    <div id="readability-page-1" class="page"><div>
                    
                        <section data-title="Main"><div id="Sec1-section"><h2 id="Sec1">Main</h2><div id="Sec1-content"><p>The behaviors of an organism are not simply the product of brain activity; rather, they emerge from a dynamic interplay among the brain, body and environment. To unravel the underlying neural control mechanisms, it is crucial to develop an integrative data-driven simulation that is constrained and parameterized by experimental data<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 1" title="Ramaswamy, S. Data-driven multiscale computational models of cortical and subcortical regions. Curr. Opin. Neurobiol. 85, 102842 (2024)." href="https://www.nature.com/articles/s43588-024-00738-w#ref-CR1" id="ref-link-section-d492457827e520">1</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 2" title="Ramaswamy, S., Colangelo, C. &amp; Markram, H. Data-driven modeling of cholinergic modulation of neural microcircuits: bridging neurons, synapses and network activity. Front. Neural Circuits 12, 77 (2018)." href="https://www.nature.com/articles/s43588-024-00738-w#ref-CR2" id="ref-link-section-d492457827e523">2</a></sup> and integrates detailed models of the brain, body and environment. This model would accurately capture the characteristics and dynamics of the biological system being studied, thus both validating theories and making predictions for biological experiments.</p><p>Currently, there are specific data-driven models that simulate either the brain, such as the mouse primary visual cortex<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 3" title="Billeh, Y. N. et al. Systematic integration of structural and functional data into multi-scale models of mouse primary visual cortex. Neuron 106, 388–403. e318 (2020)." href="https://www.nature.com/articles/s43588-024-00738-w#ref-CR3" id="ref-link-section-d492457827e530">3</a></sup>, the mouse striatum<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 4" title="Hjorth, J. J. et al. The microcircuits of striatum in silico. Proc. Natl Acad. Sci. USA 117, 9554–9565 (2020)." href="https://www.nature.com/articles/s43588-024-00738-w#ref-CR4" id="ref-link-section-d492457827e534">4</a></sup> and the rat somatosensory cortex<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 5" title="Markram, H. et al. Reconstruction and simulation of neocortical microcircuitry. Cell 163, 456–492 (2015)." href="https://www.nature.com/articles/s43588-024-00738-w#ref-CR5" id="ref-link-section-d492457827e538">5</a></sup>, or exclusively the body and environment to reproduce animal behaviors, such as neuromechanical models for <i>Drosophila</i><sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 6" title="Lobato-Rios, V. et al. NeuroMechFly, a neuromechanical model of adult Drosophila melanogaster. Nat. Methods 19, 620–627 (2022)." href="https://www.nature.com/articles/s43588-024-00738-w#ref-CR6" id="ref-link-section-d492457827e544">6</a></sup> and rodents<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Merel, J. et al. Deep neuroethology of a virtual rodent. In Proc. 8th International Conference on Learning Representations 11686–11705 (ICLR, 2020)." href="https://www.nature.com/articles/s43588-024-00738-w#ref-CR7" id="ref-link-section-d492457827e548">7</a></sup>. OpenWorm<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 8" title="Sarma, G. P. et al. OpenWorm: overview and recent advances in integrative biological simulation of Caenorhabditis elegans. Philos. Trans. R. Soc. B 373, 20170382 (2018)." href="https://www.nature.com/articles/s43588-024-00738-w#ref-CR8" id="ref-link-section-d492457827e553">8</a></sup> developed data-driven models of both the nervous system (c302, ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 9" title="Gleeson, P., Lung, D., Grosu, R., Hasani, R. &amp; Larson, S. D. c302: a multiscale framework for modelling the nervous system of Caenorhabditis elegans. Philos. Trans. R. Soc. B 373, 20170379 (2018)." href="https://www.nature.com/articles/s43588-024-00738-w#ref-CR9" id="ref-link-section-d492457827e557">9</a></sup>) and the body (Sibernetic<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 10" title="Palyanov, A. Y. &amp; Khayrulin, S. S. Sibernetic: a software complex based on the PCI SPH algorithm aimed at simulation problems in biomechanics. Russ. J. Genet. Appl. Res. 5, 635–641 (2015)." href="https://www.nature.com/articles/s43588-024-00738-w#ref-CR10" id="ref-link-section-d492457827e561">10</a></sup>) of <i>Caenorhabditis</i> <i>elegans</i>, which is a pioneering initiative in computational biology. However, the integration of these models is open loop, meaning they lack feedback from the environment. Therefore, no data-driven model currently meets all requirements of an integrative data-driven model, although such integration is urgently needed to progress our understanding of neural control mechanisms<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 11" title="Urai, A. E., Doiron, B., Leifer, A. M. &amp; Churchland, A. K. Large-scale neural recordings call for new insights to link brain and behavior. Nat. Neurosci. 25, 11–19 (2022)." href="https://www.nature.com/articles/s43588-024-00738-w#ref-CR11" id="ref-link-section-d492457827e571">11</a></sup>.</p><p>An integrative data-driven model must meet certain requirements. First, it should incorporate a biophysically detailed brain model that has neural structures and neural activity patterns similar to those of the real organism. Second, it should consist of a realistic and high-performance body and environment model that facilitates easy behavior quantification. Third, and most crucially, the brain model should not only control the body model interacting with the virtual environment but also receive the feedback from the body and environment model, establishing a closed-loop interaction<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 12" title="Zador, A. et al. Catalyzing next-generation artificial intelligence through NeuroAI. Nat. Commun. 14, 1597 (2023)." href="https://www.nature.com/articles/s43588-024-00738-w#ref-CR12" id="ref-link-section-d492457827e578">12</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 13" title="Roth, E., Sponberg, S. &amp; Cowan, N. J. A comparative approach to closed-loop computation. Curr. Opin. Neurobiol. 25, 54–62 (2014)." href="https://www.nature.com/articles/s43588-024-00738-w#ref-CR13" id="ref-link-section-d492457827e581">13</a></sup>.</p><p>We identified <i>C. elegans</i> as an exemplary model for developing such an integrative data-driven model that bridges brain, body and environment. It has completely mapped morphologies of all 302 neurons<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 14" title="White, J. G., Southgate, E., Thomson, J. N. &amp; Brenner, S. The structure of the nervous system of the nematode Caenorhabditis elegans. Philos. Trans. R. Soc. Lond. B 314, 1–340 (1986)." href="https://www.nature.com/articles/s43588-024-00738-w#ref-CR14" id="ref-link-section-d492457827e591">14</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 15" title="Cook, S. J. et al. Whole-animal connectomes of both Caenorhabditis elegans sexes. Nature 571, 63–71 (2019)." href="https://www.nature.com/articles/s43588-024-00738-w#ref-CR15" id="ref-link-section-d492457827e594">15</a></sup> (in adult hermaphrodite), along with the connectome and synapse-level structure<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Witvliet, D. et al. Connectomes across development reveal principles of brain maturation. Nature 596, 257–261 (2021)." href="#ref-CR16" id="ref-link-section-d492457827e598">16</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Ruach, R., Ratner, N., Emmons, S. W. &amp; Zaslaver, A. The synaptic organization in the Caenorhabditis elegans neural network suggests significant local compartmentalized computations. Proc. Natl Acad. Sci. USA 120, e2201699120 (2023)." href="#ref-CR17" id="ref-link-section-d492457827e598_1">17</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 18" title="Brittin, C. A., Cook, S. J., Hall, D. H., Emmons, S. W. &amp; Cohen, N. A multi-scale brain map derived from whole-brain volumetric reconstructions. Nature 591, 105–110 (2021)." href="https://www.nature.com/articles/s43588-024-00738-w#ref-CR18" id="ref-link-section-d492457827e601">18</a></sup>. Recordings of single neural activity<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Venkatachalam, V. et al. Pan-neuronal imaging in roaming Caenorhabditis elegans. Proc. Natl Acad. Sci. USA 113, E1082–E1088 (2016)." href="#ref-CR19" id="ref-link-section-d492457827e605">19</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Li, H. et al. Fast whole‐body motor neuron calcium imaging of freely moving Caenorhabditis elegans without coverslip pressed. Cytometry Part A 99, 1143–1157 (2021)." href="#ref-CR20" id="ref-link-section-d492457827e605_1">20</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Kato, S. et al. Global brain dynamics embed the motor command sequence of Caenorhabditis elegans. Cell 163, 656–669 (2015)." href="#ref-CR21" id="ref-link-section-d492457827e605_2">21</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Randi, F., Sharma, A. K., Dvali, S. &amp; Leifer, A. M. Neural signal propagation atlas of Caenorhabditis elegans. Nature 623, 406–414 (2023)." href="#ref-CR22" id="ref-link-section-d492457827e605_3">22</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 23" title="Bergs, A. C. et al. All-optical closed-loop voltage clamp for precise control of muscles and neurons in live animals. Nat. Commun. 14, 1939 (2023)." href="https://www.nature.com/articles/s43588-024-00738-w#ref-CR23" id="ref-link-section-d492457827e608">23</a></sup> as well as brain-wide neural dynamics<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 21" title="Kato, S. et al. Global brain dynamics embed the motor command sequence of Caenorhabditis elegans. Cell 163, 656–669 (2015)." href="https://www.nature.com/articles/s43588-024-00738-w#ref-CR21" id="ref-link-section-d492457827e612">21</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 22" title="Randi, F., Sharma, A. K., Dvali, S. &amp; Leifer, A. M. Neural signal propagation atlas of Caenorhabditis elegans. Nature 623, 406–414 (2023)." href="https://www.nature.com/articles/s43588-024-00738-w#ref-CR22" id="ref-link-section-d492457827e615">22</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 24" title="Uzel, K., Kato, S. &amp; Zimmer, M. A set of hub neurons and non-local connectivity features support global brain dynamics in C. elegans. Curr. Biol. 32, 3443–3459 (2022)." href="https://www.nature.com/articles/s43588-024-00738-w#ref-CR24" id="ref-link-section-d492457827e618">24</a></sup> are available. Its body is well reconstructed, consisting of only 95 muscle cells. It also exhibits easily quantifiable behaviors, such as crawling, swimming and foraging<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 25" title="Bargmann, C. I. Genetic and cellular analysis of behavior in C. elegans. Annu. Rev. Neurosci. 16, 47–71 (1993)." href="https://www.nature.com/articles/s43588-024-00738-w#ref-CR25" id="ref-link-section-d492457827e623">25</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 26" title="Rankin, C. H. From gene to identified neuron to behaviour in Caenorhabditis elegans. Nat. Rev. Genet. 3, 622–630 (2002)." href="https://www.nature.com/articles/s43588-024-00738-w#ref-CR26" id="ref-link-section-d492457827e626">26</a></sup>. Therefore, these simple structures and abundant data provide solid foundations for creating an integrative data-driven model.</p><p>In this Resource, we present an integrative data-driven model of <i>C. elegans</i>, BAAIWorm (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s43588-024-00738-w#Fig1">1</a>). First, we developed a neural network model (brain model) of a <i>C. elegans</i> foraging neural circuit. It was built using biophysically detailed compartmental neuronal models, with compartments under 2 μm in length, encoding neural activity similar to live <i>C. elegans</i>. Second, we developed a body–environment model of <i>C. elegans</i>. The body model contained 96 muscles and enabled efficient real-time simulation at 30 frames s<sup>−1</sup> and easy quantification of behaviors. Third, we established a closed-loop interaction between the brain model and the body–environment model to simulate <i>C. elegans</i> moving toward an attractor in a zigzag shape. Sensory neurons in the neural network model were activated by the attractor’s concentration in the environment. Muscles in the body model were controlled by motor neurons in the neural network model. Finally, we performed synthetic perturbations on the neural network model and suggested that the absence of a neurite or synaptic/gap junction disrupts global neural dynamics and hampers accurate forward motion.</p><div data-test="figure" data-container-section="figure" id="figure-1" data-title="Overview of BAAIWorm."><figure><figcaption><b id="Fig1" data-test="figure-caption-text">Fig. 1: Overview of BAAIWorm.</b></figcaption><div><div><a data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="https://www.nature.com/articles/s43588-024-00738-w/figures/1" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs43588-024-00738-w/MediaObjects/43588_2024_738_Fig1_HTML.png?as=webp"/><img aria-describedby="Fig1" src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs43588-024-00738-w/MediaObjects/43588_2024_738_Fig1_HTML.png" alt="figure 1" loading="lazy" width="685" height="492"/></picture></a></div><p>BAAIWorm comprises two submodels: the neural network model and the body–environment model. The neural network model is biophysically detailed. Its neuron models are multicompartmental, exhibiting electrophysiological features similar to real <i>C. elegans</i> neurons. These neuron models are connected by synapses and gap junctions on neurites. An optimization tool is used to tune the weights and polarities of these connections. The body–environment model consists of a biomechanical body, a 3D fluid environment and a real-time simulation engine. The soft body is constructed using 3,341 tetrahedrons, with 96 muscles arranged from head to tail as actuators. The large-scale 3D environment allows the integration of various agents and elements including <i>C. elegans</i>, <i>Escherichia coli</i> (food) and fluid, and it provides a range of auxiliary tools for various lighting effects, visualization, tracking and behavior analysis. Real-time simulation is achieved through a FEM solver and simplified hydrodynamics. To achieve foraging behavior in <i>C. elegans</i>, the neural network senses the attractor’s concentration in the environment, performs neural computations to control muscles’ contraction and relaxation, and ultimately deforms the body to propel it toward the attractor.</p></div></figure></div><p>BAAIWorm is open source and modular as the brain, body and environment models and their interactions can be independently modified, improved or extended by the scientific community. BAAIWorm will contribute to a deeper understanding of the intricate relationships between neural structures, neural activities and behaviors. This work also provides a framework for developing brain–body–environment models of other organisms. This approach paves the way for new avenues of systems biology and has the potential to progress traditional neural mechanism research.</p></div></div></section><section data-title="Results"><div id="Sec2-section"><h2 id="Sec2">Results</h2><div id="Sec2-content"><h3 id="Sec3">A data-driven biophysically detailed neural network model</h3><p>Our neural network model of <i>C. elegans</i> contained 136 neurons that participated in sensory and locomotion functions, as indicated by published studies<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 24" title="Uzel, K., Kato, S. &amp; Zimmer, M. A set of hub neurons and non-local connectivity features support global brain dynamics in C. elegans. Curr. Biol. 32, 3443–3459 (2022)." href="https://www.nature.com/articles/s43588-024-00738-w#ref-CR24" id="ref-link-section-d492457827e705">24</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Gray, J. M., Hill, J. J. &amp; Bargmann, C. I. A circuit for navigation in Caenorhabditis elegans. Proc. Natl Acad. Sci. USA 102, 3184–3191 (2005)." href="#ref-CR27" id="ref-link-section-d492457827e708">27</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Chalasani, S. H. et al. Dissecting a circuit for olfactory behaviour in Caenorhabditis elegans. Nature 450, 63–70 (2007)." href="#ref-CR28" id="ref-link-section-d492457827e708_1">28</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Larsch, J. et al. A circuit for gradient climbing in C. elegans chemotaxis. Cell Rep. 12, 1748–1760 (2015)." href="#ref-CR29" id="ref-link-section-d492457827e708_2">29</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Bargmann, C. I., Hartwieg, E. &amp; Horvitz, H. R. Odorant-selective genes and neurons mediate olfaction in C. elegans. Cell 74, 515–527 (1993)." href="#ref-CR30" id="ref-link-section-d492457827e708_3">30</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 31" title="Piggott, B. J., Liu, J., Feng, Z., Wescott, S. A. &amp; Xu, X. S. The neural circuits and synaptic mechanisms underlying motor initiation in C. elegans. Cell 147, 922–933 (2011)." href="https://www.nature.com/articles/s43588-024-00738-w#ref-CR31" id="ref-link-section-d492457827e711">31</a></sup>. To construct this model, we first collected the necessary data including neural morphology, ion channel models, electrophysiology of single neurons, connectome, connection models and network activities (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s43588-024-00738-w#Fig2">2a</a>). Next, we constructed the individual neuron models and their connections (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s43588-024-00738-w#Fig2">2b</a>). At this stage, the biophysically detailed model was only structurally accurate (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s43588-024-00738-w#Fig2">2c</a>), without network-level realistic dynamics. Finally, we optimized the weights and polarities of the connections to obtain a model that reflected network-level realistic dynamics (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s43588-024-00738-w#Fig2">2d</a>). An overview of the model construction is shown in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s43588-024-00738-w#Fig2">2</a>.</p><div data-test="figure" data-container-section="figure" id="figure-2" data-title="Construction of biophysically detailed neural network model of C. elegans."><figure><figcaption><b id="Fig2" data-test="figure-caption-text">Fig. 2: Construction of biophysically detailed neural network model of <i>C. elegans.</i></b></figcaption><div><div><a data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="https://www.nature.com/articles/s43588-024-00738-w/figures/2" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs43588-024-00738-w/MediaObjects/43588_2024_738_Fig2_HTML.png?as=webp"/><img aria-describedby="Fig2" src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs43588-024-00738-w/MediaObjects/43588_2024_738_Fig2_HTML.png" alt="figure 2" loading="lazy" width="685" height="493"/></picture></a></div><p><b>a</b>, Experimental data collection to constrain models. These data include neural morphologies, ion channel models, electrophysiological recordings of single neurons, connectome, connection models and neural network activities. <b>b</b>, The construction of multicompartmental neuron models and connection models (synapses and gap junctions). <b>c</b>, The biophysically detailed <i>C. elegans</i> neural network model without functional neural activities. <b>d</b>, Optimization of the biophysically detailed <i>C. elegans</i> neural network model to achieve realistic network dynamics. Neurons are color-coded to represent membrane potential.</p></div></figure></div><p>To achieve a high level of biophysical and morphological realism in our model, we used multicompartment models to represent individual neurons. The morphologies of neuron models were constructed on the basis of published morphological data<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 9" title="Gleeson, P., Lung, D., Grosu, R., Hasani, R. &amp; Larson, S. D. c302: a multiscale framework for modelling the nervous system of Caenorhabditis elegans. Philos. Trans. R. Soc. B 373, 20170379 (2018)." href="https://www.nature.com/articles/s43588-024-00738-w#ref-CR9" id="ref-link-section-d492457827e774">9</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 32" title="Grove, C. &amp; Sternberg, P. in 18th International C. elegans Meeting (GSA, 2011)." href="https://www.nature.com/articles/s43588-024-00738-w#ref-CR32" id="ref-link-section-d492457827e777">32</a></sup>. Soma and neurite sections were further divided into several segments, where each segment was less than 2 μm in length. We integrated 14 established classes of ion channels (Supplementary Tables <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s43588-024-00738-w#MOESM1">1</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s43588-024-00738-w#MOESM1">2</a>)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 33" title="Nicoletti, M. et al. Biophysical modeling of C. elegans neurons: single ion currents and whole-cell dynamics of AWCon and RMD. PLoS ONE 14, e0218738 (2019)." href="https://www.nature.com/articles/s43588-024-00738-w#ref-CR33" id="ref-link-section-d492457827e787">33</a></sup> in neuron models and tuned the passive parameters and ion channel conductance densities for each neuron model using an optimization algorithm<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 34" title="Sheng, K. et al. Domain adaptive neural inference for neurons, microcircuits and networks. Preprint at bioRxiv 
                  https://www.biorxiv.org/content/10.1101/2022.10.03.510694v1
                  
                 (2022)." href="https://www.nature.com/articles/s43588-024-00738-w#ref-CR34" id="ref-link-section-d492457827e791">34</a></sup>. This tuning was done to accurately reproduce the electrophysiological recordings obtained from patch-clamp experiments<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Ramot, D., MacInnis, B. L. &amp; Goodman, M. B. Bidirectional temperature-sensing by a single thermosensory neuron in C. elegans. Nat. Neurosci. 11, 908–915 (2008)." href="#ref-CR35" id="ref-link-section-d492457827e796">35</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Liu, Q., Kidd, P. B., Dobosiewicz, M. &amp; Bargmann, C. I. C. elegans AWA olfactory neurons fire calcium-mediated all-or-none action potentials. Cell 175, 57–70.e17 (2018)." href="#ref-CR36" id="ref-link-section-d492457827e796_1">36</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Mellem, J. E., Brockie, P. J., Madsen, D. M. &amp; Maricq, A. V. Action potentials contribute to neuronal signaling in C. elegans. Nat. Neurosci. 11, 865–867 (2008)." href="#ref-CR37" id="ref-link-section-d492457827e796_2">37</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 38" title="Liu, P., Chen, B. &amp; Wang, Z.-W. SLO-2 potassium channel is an important regulator of neurotransmitter release in Caenorhabditis elegans. Nat. Commun. 5, 5155 (2014)." href="https://www.nature.com/articles/s43588-024-00738-w#ref-CR38" id="ref-link-section-d492457827e799">38</a></sup> at the single-neuron level. Based on the few available electrophysiological data, we digitally reconstructed models of five representative neurons: AWC, AIY, AVA, RIM and VD5. The current-voltage characteristic curves (<i>I</i>–<i>V</i> curves) of our neuron models match those in the experiments (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s43588-024-00738-w#MOESM1">1</a>). The optimized parameters of neuron models are listed in Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s43588-024-00738-w#MOESM1">3</a>.</p><p>These five neurons are representative for five different functional group: sensory neuron, interneuron, command neuron, head motor neuron and body motor neuron. They serve as the initial pool of digital neuron models to reconstruct other neurons in the neural network model. For neurons that are not representative neurons, their biophysical parameters were set the same as the representative neuron models that belonged to the same functional group. The referenced neurons of all neurons are listed in Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s43588-024-00738-w#MOESM1">4</a>. This approach allows us to achieve a high level of biophysical and morphological realism in our <i>C. elegans</i> neural network model with limited accessible experimental data.</p><p>For connections, we first modeled gap junctions as simple ohmic resistances, and synapses as continuously transmitting synapse, according to published models<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 9" title="Gleeson, P., Lung, D., Grosu, R., Hasani, R. &amp; Larson, S. D. c302: a multiscale framework for modelling the nervous system of Caenorhabditis elegans. Philos. Trans. R. Soc. B 373, 20170379 (2018)." href="https://www.nature.com/articles/s43588-024-00738-w#ref-CR9" id="ref-link-section-d492457827e827">9</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Mainen, Z. F. &amp; Sejnowski, T. J. Influence of dendritic structure on firing pattern in model neocortical neurons. Nature 382, 363 (1996)." href="#ref-CR39" id="ref-link-section-d492457827e830">39</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Wicks, S. R., Roehrig, C. J. &amp; Rankin, C. H. A dynamic network simulation of the nematode tap withdrawal circuit: predictions concerning synaptic function using behavioral criteria. J. Neurosci. 16, 4017–4031 (1996)." href="#ref-CR40" id="ref-link-section-d492457827e830_1">40</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Prinz, A. A., Bucher, D. &amp; Marder, E. Similar network activity from disparate circuit parameters. Nat. Neurosci. 7, 1345–1352 (2004)." href="#ref-CR41" id="ref-link-section-d492457827e830_2">41</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 42" title="Sakata, K. &amp; Shingai, R. Neural network model to generate head swing in locomotion of Caenorhabditis elegans. Netw., Comput. Neural Syst. 15, 199 (2004)." href="https://www.nature.com/articles/s43588-024-00738-w#ref-CR42" id="ref-link-section-d492457827e833">42</a></sup>. Then, we developed an algorithm to determine the number and locations of connections. The number of connections between two cells was from the cell adjacency matrix<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 15" title="Cook, S. J. et al. Whole-animal connectomes of both Caenorhabditis elegans sexes. Nature 571, 63–71 (2019)." href="https://www.nature.com/articles/s43588-024-00738-w#ref-CR15" id="ref-link-section-d492457827e837">15</a></sup> (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s43588-024-00738-w#MOESM1">2a</a>). For each connection, we randomly assigned a distance following the neurite centroid distance distributions of synapse or gap junctions in an experiment<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 17" title="Ruach, R., Ratner, N., Emmons, S. W. &amp; Zaslaver, A. The synaptic organization in the Caenorhabditis elegans neural network suggests significant local compartmentalized computations. Proc. Natl Acad. Sci. USA 120, e2201699120 (2023)." href="https://www.nature.com/articles/s43588-024-00738-w#ref-CR17" id="ref-link-section-d492457827e844">17</a></sup> (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s43588-024-00738-w#MOESM1">2b</a>). Each connection linked two segments whose distance was closest to the randomly assigned distance. This approach allowed us to generate connections that were constrained by experimental connection numbers and distance distributions (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s43588-024-00738-w#MOESM1">2c</a>). Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s43588-024-00738-w#MOESM1">2d–f</a> shows some examples of the connections between neurons in our neural network model. Most synapses and gap junctions were located on neurites that were relatively close to each other. Also, as shown in Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s43588-024-00738-w#MOESM1">2f</a>, the constructed connections tended to cluster together, which followed the clustered organization principle in published research<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 17" title="Ruach, R., Ratner, N., Emmons, S. W. &amp; Zaslaver, A. The synaptic organization in the Caenorhabditis elegans neural network suggests significant local compartmentalized computations. Proc. Natl Acad. Sci. USA 120, e2201699120 (2023)." href="https://www.nature.com/articles/s43588-024-00738-w#ref-CR17" id="ref-link-section-d492457827e861">17</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 18" title="Brittin, C. A., Cook, S. J., Hall, D. H., Emmons, S. W. &amp; Cohen, N. A multi-scale brain map derived from whole-brain volumetric reconstructions. Nature 591, 105–110 (2021)." href="https://www.nature.com/articles/s43588-024-00738-w#ref-CR18" id="ref-link-section-d492457827e864">18</a></sup>.</p><p>To determine the polarities of synapses (excitatory or inhibitory) and the connection weights, we used a gradient-descent-based algorithm to minimize the mean squared error between simulation results and the optimization target. In this study, our optimization target was a Pearson correlation matrix of 65 identified neurons in a prior study<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 24" title="Uzel, K., Kato, S. &amp; Zimmer, M. A set of hub neurons and non-local connectivity features support global brain dynamics in C. elegans. Curr. Biol. 32, 3443–3459 (2022)." href="https://www.nature.com/articles/s43588-024-00738-w#ref-CR24" id="ref-link-section-d492457827e872">24</a></sup>, where activity series were recorded through whole-brain single-cell-resolution Ca<sup>2+</sup> imaging. Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s43588-024-00738-w#MOESM1">3a</a> shows the correlation matrices generated from our simulation and the experimental data. These two maps exhibited similar correlation strengths, whose mean squared error was only 0.076.</p><p>To further analyze the optimized neural network model, we conducted principal components analysis (PCA) on the membrane potential of the 65 identified neurons, involving the calculation of principal components (PCs) based on the covariance structure in the normalized data (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s43588-024-00738-w#MOESM1">3b</a>). Upon sorting based on PC1, most neurons controlling forward locomotion (for example, AVB, VB and DB neurons) and backward locomotion (for example, AVA, VA and DA neurons) were distinctly classified into two groups (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s43588-024-00738-w#MOESM1">3b</a>). The result of this analysis indicates that our trained neural network model accurately captures realistic neural dynamical relationships, and the dynamics of neurons encode functions similar to those observed in real <i>C. elegans</i>, to some extent. However, it is essential to clarify that this validation primarily confirms the effectiveness of our optimization algorithm rather than implying importance of the optimized parameters. These parameters represent just one of the possible solutions that can accurately reproduce the desired dynamics.</p><h3 id="Sec4">A high-performance body–environment model</h3><p>Next, we developed the body–environment model of <i>C. elegans</i>. Our primary contributions are (1) modeling the soft body and muscles and solving soft-body deformation, (2) creating the three-dimensional (3D) simulation environment and solving soft-body–fluid interaction and (3) proposing a reference coordinate system for behavior analysis. The framework of this model is shown in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s43588-024-00738-w#Fig3">3a,b</a>.</p><div data-test="figure" data-container-section="figure" id="figure-3" data-title="A biomechanical body model of C. elegans."><figure><figcaption><b id="Fig3" data-test="figure-caption-text">Fig. 3: A biomechanical body model of <i>C. elegans.</i></b></figcaption><div><div><a data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="https://www.nature.com/articles/s43588-024-00738-w/figures/3" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs43588-024-00738-w/MediaObjects/43588_2024_738_Fig3_HTML.png?as=webp"/><img aria-describedby="Fig3" src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs43588-024-00738-w/MediaObjects/43588_2024_738_Fig3_HTML.png" alt="figure 3" loading="lazy" width="685" height="624"/></picture></a></div><p><b>a</b>, A schematic diagram of the <i>C. elegans</i> body model. The body was composed of tetrahedrons. Ninety-six muscle cells along four muscle strings served as actuators for soft-body deformation. Different colors on muscles indicate distinct muscle cells. The body was situated in a large 3D fluid environment and could move in the environment. Projective dynamics functions were used as the soft-body solver, and simplified hydrodynamics calculated thrust force and drag force on the body surface. <b>b</b>, An internal view of the body model, consisting of four muscle strings: VR, VL, DR and DL. <b>c</b>, The triangulated tetrahedron mesh of the body. <b>d</b>, Four muscle strings: VR (green), VL (yellow), DR (red) and DL (blue). The color intensity reflects the strength of muscle contraction, with high intensity indicating contraction and transparent intensity indicating relaxation. <b>e</b>, Tetrahedrons driven by four muscle strings: VR (green), VL (yellow), DR (red) and DL (blue). <b>f</b>, Deformation of the soft body. Blue tetrahedrons are increasing their volume whereas red ones are decreasing. <b>g</b>,<b>h</b>, The thrust force (<b>g</b>) and drag force (<b>h</b>) on the body surface. Red indicates strong force, and blue indicates weak force. <b>i</b>, Positions and velocities of the body at four moments. The body was moving from left to right. Color indicates velocities, with red representing fast and blue representing slow.</p></div></figure></div><p>The body model was built with tetrahedral mesh, including 984 vertices, 3,341 tetrahedrons and 1,466 triangles on the body surface (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s43588-024-00738-w#Fig3">3c</a>). It had four muscle strings: ventral right (VR), ventral left (VL), dorsal right (DR) and dorsal left (DL). Each string had 24 muscle cells arranged from head to tail (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s43588-024-00738-w#Fig3">3d</a>). Tetrahedrons were controlled by their proximal muscle strings. The corresponding strings are visualized with different colors in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s43588-024-00738-w#Fig3">3e</a>. We used a finite element method (FEM) solver to compute positions and velocities of all vertices on the deformable body at any given time. In the computation, the solver used 4,558 muscle constraints and 3,341 corotated elasticity constraints. Thus, our soft-body simulation engaged a sum of 7,899 mechanical constraints. After the geometry construction, we need to solve the body deformation efficiently. In our model, we selected a real-time, highly robust and accurate approach named projective dynamics as the FEM solver for calculation. The solved body state is shown in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s43588-024-00738-w#Fig3">3f</a>.</p><p>We constructed a large 3D fluid simulation environment, where the body and food were embedded. This environment was essentially a large cube, with each edge equating to approximately 1,200 times the length of a <i>C. elegans</i>. The mechanic of the body movement in fluid is essentially a soft-body–fluid coupling problem. We used a mechanical model inspired by worm swimming, focused primarily on the thrust force (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s43588-024-00738-w#Fig3">3g</a>) and drag force (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s43588-024-00738-w#Fig3">3h</a>) on the body surface<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 43" title="Min, S., Won, J., Lee, S., Park, J. &amp; Lee, J. Softcon: simulation and control of soft-bodied animals with biomimetic actuators. ACM Trans. Graph. 38, 1–12 (2019)." href="https://www.nature.com/articles/s43588-024-00738-w#ref-CR43" id="ref-link-section-d492457827e993">43</a></sup>. Gravity and buoyancy were assumed to be equal and cancel each other out, allowing the simulation of a much larger space and improved computational efficiency. To achieve forward locomotion behavior, we used projective dynamics to calculate real-time bending waves that propagated from head to tail. The interaction between these bending waves and the fluid generated impetus for forward locomotion (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s43588-024-00738-w#Fig3">3i</a>).</p><p><i>C. elegans</i> is an animal with soft body that exhibits special patterns during locomotion, wherein every point on its body oscillates continuously. Therefore, it is difficult to quantify the trajectory and body state during locomotion. Here, we proposed an effective and simple method to measure <i>C. elegans</i> locomotion. The main idea of this method is to establish a numerically stable coordinate system relative to the body itself, named Target Body Reference Coordinate System (TBRCS) (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s43588-024-00738-w#Fig4">4a</a>). First, we defined the target body, standard body and Standard Body Coordinate System (SBCS). Then, we computed transformation <i>M</i> between the target body and the standard body. Finally, we used <i>M</i> to perform 3D transformation on the SBCS to obtain the TBRCS. Details of this method are described in <a data-track="click" data-track-label="link" data-track-action="section anchor" href="https://www.nature.com/articles/s43588-024-00738-w#Sec8">Methods</a>. Using this method, we can stably quantify not only the body’s locomotion trajectory (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s43588-024-00738-w#Fig4">4a</a>) but also the body states including the relative position and velocities of any points on the <i>C. elegans</i> body (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s43588-024-00738-w#Fig4">4b,c</a>) and steering angle (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s43588-024-00738-w#Fig4">4d</a>). For example, Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s43588-024-00738-w#Fig4">4e</a> shows the relative positions of 17 sampled 3D tracking points, while Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s43588-024-00738-w#Fig4">4f</a> shows their corresponding velocities. In Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s43588-024-00738-w#Fig4">4e</a>, the swing direction of the body was the <i>Z</i> direction of the TBRCS (dorsal–ventral direction). Also, there was a spring movement in the <i>X</i> direction of the TBRCS (head–tail direction) during forward locomotion. Locomotion in the <i>Y</i> direction of the TBRCS (left–right direction) was nearly flat because muscle activations were approximately symmetric in this direction. Figure <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s43588-024-00738-w#Fig4">4f</a> shows that, in this case, the primary propulsion of forward locomotion originated from the tail (pink line), as faster swing generates more thrust force. Moreover, our method enables the quantification of the body’s steering. Figure <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s43588-024-00738-w#Fig4">4d</a> shows an example of a steering angle <i>θ</i>, which was defined as the angle between the TBRCS <i>x</i> axis and the direction of velocity. The steering angle <i>θ</i> was also numerically stable.</p><div data-test="figure" data-container-section="figure" id="figure-4" data-title="Quantification of the body’s behavior."><figure><figcaption><b id="Fig4" data-test="figure-caption-text">Fig. 4: Quantification of the body’s behavior.</b></figcaption><div><div><a data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="https://www.nature.com/articles/s43588-024-00738-w/figures/4" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs43588-024-00738-w/MediaObjects/43588_2024_738_Fig4_HTML.png?as=webp"/><img aria-describedby="Fig4" src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs43588-024-00738-w/MediaObjects/43588_2024_738_Fig4_HTML.png" alt="figure 4" loading="lazy" width="685" height="275"/></picture></a></div><div data-test="bottom-caption" id="figure-4-desc"><p><b>a</b>, A schematic diagram of TBRCS to quantify behavior stably. The standard body and SBCS are depicted on the left. The target body and TBRCS are illustrated on the right. Matching points (magenta dots) are positioned on both the standard and target body. The magenta lines link the corresponding points on two body. The transformation matrix <i>M</i> is obtained through singular value decomposition, enabling the conversion from SBCS to TBRCS. The trajectory of the target body center (orange line) exhibits a zigzag pattern, whereas the trajectory of TBRCS (green line) is smooth. <b>b</b>, Tracking body movement with TBRCS. Seventeen tracking points were defined on the surface of both the standard body and the target body. The magenta lines represent relative positions, while the blue arrows indicate relative velocities. Coordinate systems of the head on standard body and target body are drawn out to display translation and rotation of the head on target body. <b>c</b>, Trajectories of 17 tracking points on the body. <b>d</b>, Steering angle <i>θ</i> of the body. The velocity vector is depicted as the yellow arrow, and the <i>X</i>, <i>Y</i> and <i>Z</i> axes of TBRCS are represented by red, green and blue arrows, respectively. <i>θ</i> is defined as the angle between the red arrow and the yellow arrow and remains stable over time. <b>e</b>,<b>f</b>, Relative positions (<b>e</b>) and velocities (<b>f</b>) of 17 tracking points on the body during the movement.</p><p><a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s43588-024-00738-w#MOESM7">Source data</a></p></div></div></figure></div><h3 id="Sec5">Replicating locomotion behavior of <i>C. elegans</i>
                        </h3><p>In our model, we successfully replicated the locomotion behavior of <i>C. elegans</i> by a closed-loop interaction between the neural network model and the body–environment model. Specifically, the neural network model sensed the dynamic food concentration in the environment by sensory neurons. This sensory information was used to calculate the membrane potentials of motor neurons. The membrane potentials of motor neurons were subsequently translated into muscle activation signals, which controlled the movement of the <i>C. elegans</i> body to navigate toward the food in the environment. An overview of this closed-loop simulation is shown in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s43588-024-00738-w#Fig5">5a</a>. Notably, we observed that the trajectory of the <i>C. elegans</i> movement in our model was similar to the zigzag trajectory recorded in the experiments<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 44" title="Sznitman, J., Purohit, P. K., Krajacic, P., Lamitina, T. &amp; Arratia, P. E. Material properties of Caenorhabditis elegans swimming at low reynolds number. Biophys. J. 98, 617–626 (2010)." href="https://www.nature.com/articles/s43588-024-00738-w#ref-CR44" id="ref-link-section-d492457827e1159">44</a></sup> (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s43588-024-00738-w#Fig5">5b</a>). In addition, <i>C. elegans</i> in both the simulations (Supplementary Videos <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s43588-024-00738-w#MOESM4">1</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s43588-024-00738-w#MOESM5">2</a>) and experiments exhibited a realistic dorsoventral fluctuation during 3D movement<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 45" title="Bilbao, A., Patel, A. K., Rahman, M., Vanapalli, S. A. &amp; Blawzdziewicz, J. Roll maneuvers are essential for active reorientation of Caenorhabditis elegans in 3D media. Proc. Natl Acad. Sci. USA 115, E3616–E3625 (2018)." href="https://www.nature.com/articles/s43588-024-00738-w#ref-CR45" id="ref-link-section-d492457827e1176">45</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 46" title="Salfelder, F. et al. Markerless 3D spatio-temporal reconstruction of microscopic swimmers from video. In 25th International Conference on Pattern Recognition 1965–1972 (IEEE, 2020)." href="https://www.nature.com/articles/s43588-024-00738-w#ref-CR46" id="ref-link-section-d492457827e1179">46</a></sup>.</p><div data-test="figure" data-container-section="figure" id="figure-5" data-title="The closed-loop 3D simulation of C. elegans locomotion behavior."><figure><figcaption><b id="Fig5" data-test="figure-caption-text">Fig. 5: The closed-loop 3D simulation of <i>C. elegans</i> locomotion behavior.</b></figcaption><div><div><a data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="https://www.nature.com/articles/s43588-024-00738-w/figures/5" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs43588-024-00738-w/MediaObjects/43588_2024_738_Fig5_HTML.png?as=webp"/><img aria-describedby="Fig5" src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs43588-024-00738-w/MediaObjects/43588_2024_738_Fig5_HTML.png" alt="figure 5" loading="lazy" width="685" height="455"/></picture></a></div><div data-test="bottom-caption" id="figure-5-desc"><p><b>a</b>, The information flow diagram of the closed-loop interaction between the neural network model and the body–environment model. <b>b</b>, A comparison of the <i>C. elegans</i> trajectory during forward locomotion in the simulation (left) and the experiment (right). <b>c</b>, The input current injected to sensory neurons during locomotion. <b>d</b>, Left: the neural network model of <i>C. elegans</i>. Right: the membrane potentials of all the motor neurons in the model during locomotion. <b>e</b>. Left: the body model with 96 muscle cells. Right: the activation signals of all muscle cells during locomotion. Panel <b>b</b> (right) reproduced with permission from ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 44" title="Sznitman, J., Purohit, P. K., Krajacic, P., Lamitina, T. &amp; Arratia, P. E. Material properties of Caenorhabditis elegans swimming at low reynolds number. Biophys. J. 98, 617–626 (2010)." href="https://www.nature.com/articles/s43588-024-00738-w#ref-CR44" id="ref-link-section-d492457827e1223">44</a></sup>, Elsevier.</p><p><a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s43588-024-00738-w#MOESM8">Source data</a></p></div></div></figure></div><p>In our current configuration for sensation, the food concentration in the virtual environment followed a linear distribution. To incorporate this sensory information into our model, we used the derivative of the food concentration as the amplitude of an external current injected into the soma of all sensory neurons. For neuromuscular coupling, we used a linear transformation, treating the neural network model as a reservoir to generate muscle activities<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 47" title="Bhovad, P. &amp; Li, S. Physical reservoir computing with origami and its application to robotic crawling. Sci. Rep. 11, 13002 (2021)." href="https://www.nature.com/articles/s43588-024-00738-w#ref-CR47" id="ref-link-section-d492457827e1242">47</a></sup> (<a data-track="click" data-track-label="link" data-track-action="section anchor" href="https://www.nature.com/articles/s43588-024-00738-w#Sec8">Methods</a>).</p><p>During locomotion, the input of sensory neurons exhibited fluctuations due to the zigzag movement of the body’s head (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s43588-024-00738-w#Fig5">5c</a>). The membrane potentials of each individual motor neuron also oscillated in response to the sensory input, especially the head motor neurons (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s43588-024-00738-w#Fig5">5d</a>). The activation of muscle cells revealed traveling waves from the head to the tail (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s43588-024-00738-w#MOESM1">4</a>), accompanied by alternating contractions and relaxations of dorsal and ventral muscle cells (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s43588-024-00738-w#Fig5">5e</a>). These findings resemble observations in biological experiments<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 48" title="Wen, Q. et al. Proprioceptive coupling within motor neurons drives C. elegans forward locomotion. Neuron 76, 750–761 (2012)." href="https://www.nature.com/articles/s43588-024-00738-w#ref-CR48" id="ref-link-section-d492457827e1264">48</a></sup>.</p><p>To facilitate visualization, we have incorporated the locomotion of <i>C. elegans</i>, the activities of its neural network model, and muscle activities into our graphical user interface (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s43588-024-00738-w#MOESM1">5</a>).</p><h3 id="Sec6">Impact of neural network structure on dynamics and behaviors</h3><p>We are interested in exploring the extent to which network structure influences the neural dynamics and behaviors. Thus, we conducted synthetic perturbations on the structure of our neural network model. Synthetic perturbations of the neural system often exceed individual tolerance level in animals. However, our model enables researchers to carry out experiments without concerns about experimental technique limitations or animal tolerance. Based on BAAIWorm, we performed experiments involving synthetic perturbations to quantitatively examine the impact of network structure on network dynamics and behaviors. The experiments include removing neurites, shuffling synapse and gap junction locations, shuffling synapse weights, shuffling gap junction weights, removing synapses and removing gap junctions. For shuffling operations, different random seeds produce different results, whereas for other perturbations, simulation results are consistent. The results in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s43588-024-00738-w#Fig6">6</a> represent one of the trials (random seed 64). The results of multiple simulation with different seeds are shown in Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s43588-024-00738-w#MOESM1">6a</a>.</p><div data-test="figure" data-container-section="figure" id="figure-6" data-title="Synthetic perturbation of on the neural network model in BAAIWorm."><figure><figcaption><b id="Fig6" data-test="figure-caption-text">Fig. 6: Synthetic perturbation of on the neural network model in BAAIWorm.</b></figcaption><div><div><a data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="https://www.nature.com/articles/s43588-024-00738-w/figures/6" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs43588-024-00738-w/MediaObjects/43588_2024_738_Fig6_HTML.png?as=webp"/><img aria-describedby="Fig6" src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs43588-024-00738-w/MediaObjects/43588_2024_738_Fig6_HTML.png" alt="figure 6" loading="lazy" width="685" height="965"/></picture></a></div><div data-test="bottom-caption" id="figure-6-desc"><p><b>a</b>, Column 1: a schematic diagram illustrating connections between two neurons in the neural network model. The ball and stick denote soma and neurite, respectively. The rectangles with solid and dashed lines indicate synapses and gap junctions. The rectangles with different colors represent different connections. The width of rectangles signifies connection weights. The depicted connections between this neuron pair represent all connections in the neural network model. Column 2: a Pearson correlation matrix of neurons during simulation of the neural network model. Column 3: locomotion behavior of the <i>C. elegans</i> body controlled by the neural network model in column 1. Columns 4 and 5: the relative position and velocity of 3D tracking points on the head (red), center (blue) and tail (green) of the <i>C. elegans</i> body during movement (Euclidean distance). <i>n</i> = 200, time step 0.1 s. Each point represents one relative velocity exported at each time step. These data are replicable using our code once the random seed is fixed, regardless of the number of replications. The bottom, middle and top lines of the box plot respectively indicate the 25th, 50th (median) and 75th percentiles. The whiskers extend to the minima and maxima within 1.5 times the range between the 25th and 75th percentiles from the bottom and top bounds of the box plot. Flier points are those past the end of the whiskers. <b>b</b>–<b>g</b>, Diagrams represented as in (<b>a</b>), but with different perturbation types: removal of neurites from all neurons in the neural network model, relocating connections onto the soma (<b>b</b>); shuffling the location of connections on neurites while fixing source and target neurons (<b>c</b>); shuffling the weights of synapses while fixing their locations (<b>d</b>); shuffling the weights of gap junctions while fixing their locations (<b>e</b>); removal of all synapses in the neural network model (<b>f</b>); and removal of all gap junctions in the neural network model (<b>g</b>). These perturbations are sequenced on the basis of their distance to the control group in terms of the Pearson correlation matrix (<b>b</b>: closest, <b>g</b>: farthest).</p><p><a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s43588-024-00738-w#MOESM9">Source data</a></p></div></div></figure></div><p>In all the perturbation scenarios, the body exhibits a higher degree of twisting during locomotion, as indicated by the larger relative locations of tracking points on the head and tail compared with the control group (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s43588-024-00738-w#Fig6">6</a> and Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s43588-024-00738-w#MOESM1">6a–c</a>). This suggests that neurites play crucial roles in the computations associated with network information processing. Moreover, shuffling connection locations within the model usually leads to faster oscillations of the body but slower forward locomotion, as indicated by larger relative velocities compared with the control group (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s43588-024-00738-w#Fig6">6c</a> and Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s43588-024-00738-w#MOESM1">6d–f</a>). This suggests that the specific locations of connections along the neurites are important, potentially facilitating localized computations within neurons. In addition, despite that the neural network model contains fewer gap junctions than synapses, the role of gap junctions in the <i>C. elegans</i> neural network appears to be more important than that of synapses (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s43588-024-00738-w#Fig6">6d–g</a>) because perturbations on gap junctions result in more obvious changes in the Pearson correlation matrix compared with perturbations on synapses. Removing gap junctions accelerates the twisting of the body compared with the control group, whereas removing synapses results in slower twisting (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s43588-024-00738-w#Fig6">6f,g</a> and Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s43588-024-00738-w#MOESM1">6d–f</a>). These results emphasize the importance of network structure in shaping network dynamics and behavior patterns. By using our model, researchers can gain insights into the contributions of network structure to neural network.</p></div></div></section><section data-title="Discussion"><div id="Sec7-section"><h2 id="Sec7">Discussion</h2><div id="Sec7-content"><p>There exist many typical models simulating body dynamics and their interactions with the environment. For example, OpenSim<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 49" title="Delp, S. L. et al. OpenSim: open-source software to create and analyze dynamic simulations of movement. IEEE Trans. Biomed. Eng. 54, 1940–1950 (2007)." href="https://www.nature.com/articles/s43588-024-00738-w#ref-CR49" id="ref-link-section-d492457827e1398">49</a></sup> aims to develop models of musculoskeletal structures and create dynamic simulations of movement in closed-loop style in three dimensions, but mainly for humans. There are many two-dimensional worm body models<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 50" title="Boyle, J. H., Berri, S. &amp; Cohen, N. Gait modulation in C. elegans: an integrated neuromechanical model. Front. Comput. Neurosci. 6, 10 (2012)." href="https://www.nature.com/articles/s43588-024-00738-w#ref-CR50" id="ref-link-section-d492457827e1402">50</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 51" title="Izquierdo, E. J. &amp; Beer, R. D. From head to tail: a neuromechanical model of forward locomotion in Caenorhabditis elegans. Philos. Trans. R. Soc. B 373, 20170374 (2018)." href="https://www.nature.com/articles/s43588-024-00738-w#ref-CR51" id="ref-link-section-d492457827e1405">51</a></sup> that elegantly simulate behaviors and provide insights into proprioception and the propagation of neuronal oscillations during worm locomotion. However, although more challenging, we believe that 3D body and physical environment simulations of <i>C. elegans</i> can more precisely replicate natural motion patterns such as twisting and postural changes<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 45" title="Bilbao, A., Patel, A. K., Rahman, M., Vanapalli, S. A. &amp; Blawzdziewicz, J. Roll maneuvers are essential for active reorientation of Caenorhabditis elegans in 3D media. Proc. Natl Acad. Sci. USA 115, E3616–E3625 (2018)." href="https://www.nature.com/articles/s43588-024-00738-w#ref-CR45" id="ref-link-section-d492457827e1412">45</a></sup>.</p><p>OpenWorm provides a platform for understanding the biology of <i>C. elegans</i>. BAAIWorm builds upon OpenWorm’s valuable tools and data, such as cell model morphology, synaptic dynamic and 3D worm body information, but presents several essential advancements over OpenWorm. First, our neural network model is highly detailed and trained to fit data at both neuron and network levels. Second, we used worm body and environment simulation methods supporting larger-scale and highly efficient simulations. In addition, we accomplished a closed-loop interaction with sensory feedback from the 3D environment. Details of comparisons are presented in Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s43588-024-00738-w#MOESM1">5</a>.</p><p>As we focused initially on where we have the most robust data and urgent needs, BAAIWorm includes several simplifications. For example, considering that several neurons in <i>C. elegans</i> cotransmit multiple neurotransmitters<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 52" title="Wang, C. et al. A neurotransmitter atlas of the nervous system of C. elegans males and hermaphrodites. eLife 13, RP95402 (2024)." href="https://www.nature.com/articles/s43588-024-00738-w#ref-CR52" id="ref-link-section-d492457827e1431">52</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 53" title="Serrano-Saiz, E. et al. A neurotransmitter atlas of the Caenorhabditis elegans male nervous system reveals sexually dimorphic neurotransmitter usage. Genetics 206, 1251–1269 (2017)." href="https://www.nature.com/articles/s43588-024-00738-w#ref-CR53" id="ref-link-section-d492457827e1434">53</a></sup>, we did not strictly constrain the model with Dale’s law during the training process. BAAIWorm can be improved to more accurately reflect synaptic polarities as more experimental data become available. For sensation, we simplified that the food concentration gradient at the head was the sole input, which was transferred into current injected into all sensory neurons. In future work, the sensor can be designed to detect the absolute concentration rather than the gradient, and we will incorporate more complex sensory inputs and integrate multimodal sensory feedback, such as mechanical stimuli, temperature stimuli and body proprioception<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 48" title="Wen, Q. et al. Proprioceptive coupling within motor neurons drives C. elegans forward locomotion. Neuron 76, 750–761 (2012)." href="https://www.nature.com/articles/s43588-024-00738-w#ref-CR48" id="ref-link-section-d492457827e1438">48</a></sup>, into the model. Moreover, we adopted a simple but more functional approach from prior studies<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 47" title="Bhovad, P. &amp; Li, S. Physical reservoir computing with origami and its application to robotic crawling. Sci. Rep. 11, 13002 (2021)." href="https://www.nature.com/articles/s43588-024-00738-w#ref-CR47" id="ref-link-section-d492457827e1442">47</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 54" title="Horii, Y. et al. Physical reservoir computing in a soft swimming robot. In ALIFE 2021: The 2021 Conference on Artificial Life (MIT Press, 2021)." href="https://www.nature.com/articles/s43588-024-00738-w#ref-CR54" id="ref-link-section-d492457827e1445">54</a></sup> (reservoir computation, for motor control) and we used matrix analysis (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s43588-024-00738-w#MOESM1">7</a>) to ensure that many neurons contribute to the signaling of the muscles. In future iterations, we will use a more realistic model of the neuromuscular junction with specific ion channels such as calcium channels and incorporate muscle morphology, connection location and stretch forces to establish a more biophysically accurate model.</p><p>When the input of the neural network model was a periodic standard signal, that is, without sensory feedback from the environment, the membrane potentials of motor neurons and the activations of muscle cells were periodic (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s43588-024-00738-w#MOESM1">8</a>). This resulted in the movement trajectory of the <i>C. elegans</i> body following a standard Z shape, which was extremely similar to the trajectory in the experiment<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 44" title="Sznitman, J., Purohit, P. K., Krajacic, P., Lamitina, T. &amp; Arratia, P. E. Material properties of Caenorhabditis elegans swimming at low reynolds number. Biophys. J. 98, 617–626 (2010)." href="https://www.nature.com/articles/s43588-024-00738-w#ref-CR44" id="ref-link-section-d492457827e1461">44</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 55" title="Kwon, N., Pyo, J., Lee, S.-J. &amp; Je, J. H. 3-D worm tracker for freely moving C. elegans. PLoS ONE 8, e57484 (2013)." href="https://www.nature.com/articles/s43588-024-00738-w#ref-CR55" id="ref-link-section-d492457827e1464">55</a></sup> (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s43588-024-00738-w#MOESM1">8b</a> and Supplementary Video <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s43588-024-00738-w#MOESM6">3</a>). Conversely, When the input was a sensory signal dependent on the body’s location in the environment (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s43588-024-00738-w#Fig5">5</a>), the input was not periodic, leading to nonperiodic dynamics in both neurons and muscle cells. Therefore, the trajectory of the body was not as regular as that observed in the experiment (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s43588-024-00738-w#Fig5">5</a>). Future studies will focus on designing more reasonable central pattern generators or motor control systems to achieve more realistic movement.</p><p>In our current model, we modeled 136 neurons rather than all 302 neurons, and the only behavior considered was zigzag locomotion. Training biophysically detailed neural network model is challenging and consumes much time and graphics processing unit memory resources. In the future, we will improve the optimization algorithm to effectively train larger detailed neural networks that contain all 302 neurons, with more constraints such as single-neuron voltages, and reproduce more behaviors beyond zigzag locomotion.</p><p>The neural network model of BAAIWorm integrated experimental data including ion channel dynamics, neural morphologies, electrophysiology, connectome, synaptic organization rules on neurites, and neural activities. Although we collected and integrated as much data as possible up until 2023, there remained a substantial amount of detail that could not be fully implemented due to a lack of data. Also, the physical parameters in the body–environment model did not correspond precisely to those of actual <i>C. elegans</i> because they were difficult to measure accurately. However, with the influx of new experimental data, these models can be enhanced and refined.</p><p>With ongoing experimental data and updates on BAAIWorm, it has the potential to enhance our computational technical capabilities to study <i>C. elegans</i>. For example, we can modulate ion channels or alter specific connections in our neural network model to investigate how experimental manipulations (for example, through genetic mutations or pharmacological interventions) may impact neural dynamics and behavior. Moreover, BAAIWorm enables us to test hypotheses that are currently challenging to verify experimentally. For example, some studies have found that different locations along the neurite of a <i>C. elegans</i> neuron may have distinct local neurite computations<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 17" title="Ruach, R., Ratner, N., Emmons, S. W. &amp; Zaslaver, A. The synaptic organization in the Caenorhabditis elegans neural network suggests significant local compartmentalized computations. Proc. Natl Acad. Sci. USA 120, e2201699120 (2023)." href="https://www.nature.com/articles/s43588-024-00738-w#ref-CR17" id="ref-link-section-d492457827e1500">17</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 56" title="Donato, A., Kagias, K., Zhang, Y. &amp; Hilliard, M. A. Neuronal sub-compartmentalization: a strategy to optimize neuronal function. Biol. Rev. 94, 1023–1037 (2019)." href="https://www.nature.com/articles/s43588-024-00738-w#ref-CR56" id="ref-link-section-d492457827e1503">56</a></sup>. With BAAIWorm, we can precisely move the connection between two neurons or change the connection strengths. Feedback from the environment can help to investigate questions such as how the neural network adapts to environmental stimuli. These in silico experiments can guide the design and interpretation of electrophysiology, imaging and behavioral experiments.</p><p>Beyond its technical capabilities, BAAIWorm represents an advancement in multiscale computational modeling of biology. ‘Worm-As-a-Whole’ enables a variety of scientific discoveries that were previously unattainable. For instance, the platform has the potential to establish cause-and-effect relationships between genetic mutations and neuronal behaviors by enabling precise manipulation and observation of neural activity and its downstream effects. Furthermore, BAAIWorm facilitates the study of multiscale interactions, allowing researchers to explore the impact of molecular and cellular processes on larger network dynamics and behavior. This is critical for understanding system-wide functions<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 57" title="Bargmann, C. I. &amp; Marder, E. From the connectome to brain function. Nat. Methods 10, 483–490 (2013)." href="https://www.nature.com/articles/s43588-024-00738-w#ref-CR57" id="ref-link-section-d492457827e1510">57</a></sup>. In addition, while the current version of BAAIWorm does not model synaptic plasticity explicitly, its modular structure provides a foundation for future integration of plasticity mechanisms, potentially offering insights into adaptive neural reorganizations. These features make BAAIWorm a promising tool for advancing our understanding of neural dynamics across different biological scales.</p><p>BAAIWorm can also be used in computer science and other interdisciplinary fields. For example, it can be used to test and refine simulation methods, enhancing the accuracy of virtual representations. Moreover, in reinforcement learning, BAAIWorm’s closed-loop interactions present an ideal scenario for developing and testing algorithms in a biologically relevant context.</p></div></div></section><section data-title="Methods"><div id="Sec8-section"><h2 id="Sec8">Methods</h2><div id="Sec8-content"><h3 id="Sec9">Details of the neural network model</h3><h4 id="Sec10">Neuron models</h4><p>Neurons were modeled by morphologically derived multicompartmental models with somatic Hodgkin–Huxley dynamics<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 58" title="Hodgkin, A. L. &amp; Huxley, A. F. A quantitative description of membrane current and its application to conduction and excitation in nerve. J. Physiol. 117, 500–544 (1952)." href="https://www.nature.com/articles/s43588-024-00738-w#ref-CR58" id="ref-link-section-d492457827e1533">58</a></sup> and passive neurites. These neuron models captured both the biophysical properties and the spatial complexity of neurons. The models integrated three kinds of neuron data: ion channel, morphology and electrophysiology.</p><p>Ion channel models were determined according to published studies<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 33" title="Nicoletti, M. et al. Biophysical modeling of C. elegans neurons: single ion currents and whole-cell dynamics of AWCon and RMD. PLoS ONE 14, e0218738 (2019)." href="https://www.nature.com/articles/s43588-024-00738-w#ref-CR33" id="ref-link-section-d492457827e1540">33</a></sup>. There were 14 ion channels divided into 4 categories: voltage-gated potassium channels (SHK-1, EGL-36, SHL-1, KVS-1, KQT-3, EGL-2 and IRK1/3), calcium-regulated potassium channels (SLO1, SLO2 and KCNL), voltage-gated calcium channels (EGL19, UNC2 and CCA1) and passive sodium channel (NCA). Model equations and parameters for ion channels are presented in Supplementary Tables <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s43588-024-00738-w#MOESM1">1</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s43588-024-00738-w#MOESM1">2</a>.</p><p>For morphologies of neuron models, the spatial structures were adopted from the <i>C. elegans</i> neuronal anatomy models created by the Virtual Worm Project<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 32" title="Grove, C. &amp; Sternberg, P. in 18th International C. elegans Meeting (GSA, 2011)." href="https://www.nature.com/articles/s43588-024-00738-w#ref-CR32" id="ref-link-section-d492457827e1556">32</a></sup> and the OpenWorm Project<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 9" title="Gleeson, P., Lung, D., Grosu, R., Hasani, R. &amp; Larson, S. D. c302: a multiscale framework for modelling the nervous system of Caenorhabditis elegans. Philos. Trans. R. Soc. B 373, 20170379 (2018)." href="https://www.nature.com/articles/s43588-024-00738-w#ref-CR9" id="ref-link-section-d492457827e1560">9</a></sup>. Each neuron model had a soma section and several neurite sections. Neurite sections were further divided into several compartments, each of which was shorter than 2 μm.</p><p>Only soma compartments had active ion currents. Soma compartments and neurite compartments shared the same passive properties. To identify the passive and active parameters that reproduce experimental data of voltage–current clamps, we initially used a deep learning tool<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 59" title="Sheng, K. et al. Distilling multi-scale neural mechanisms from diverse unlabeled experimental data using deep domain-adaptive inference framework. Preprint at bioRxiv 
                  https://www.biorxiv.org/content/10.1101/2022.10.03.510694v2
                  
                 (2023)." href="https://www.nature.com/articles/s43588-024-00738-w#ref-CR59" id="ref-link-section-d492457827e1567">59</a></sup> that uses model-generated synthetic data to accurately and efficiently infer neural parameters. Then we manually fine-tuned these parameters. Parameters with extremely low values (&lt;0.00001 nS μm<sup>−2</sup>) have negligible impact on the target dynamics. Therefore, they were manually set to 0 in simulation to simplify the model without influencing the model’s accuracy. The value of the parameters are presented in Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s43588-024-00738-w#MOESM1">3</a>.</p><p>We compared two features of the patch-clamp traces between experimental data and our single-neuron models, the voltage or current at steady state and the initial peak point (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s43588-024-00738-w#MOESM1">1</a>). The steady state is where the membrane potential becomes steady. The initial peak point is identified as the first time its gradient decreased to a certain threshold (150 mV s<sup>−1</sup>) after the peak of the voltage or current gradient. Before identifying the initial peak point, the voltage or current gradient is smoothed using a Hamming filter to reduce the impact of experimental noise.</p><p>Less than 20% of the <i>C. elegans</i> neurons have been recorded for the desired electrophysiology data. To estimate model parameters for those neurons that were not recorded, a method was implemented to generalize parameter values. First, considering the available data and hierarchy of information flow in <i>C. elegans</i> nervous system<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 15" title="Cook, S. J. et al. Whole-animal connectomes of both Caenorhabditis elegans sexes. Nature 571, 63–71 (2019)." href="https://www.nature.com/articles/s43588-024-00738-w#ref-CR15" id="ref-link-section-d492457827e1594">15</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 27" title="Gray, J. M., Hill, J. J. &amp; Bargmann, C. I. A circuit for navigation in Caenorhabditis elegans. Proc. Natl Acad. Sci. USA 102, 3184–3191 (2005)." href="https://www.nature.com/articles/s43588-024-00738-w#ref-CR27" id="ref-link-section-d492457827e1597">27</a></sup>, the neurons were divided into five functional groups (sensory neuron, interneuron, command interneuron, head motor neuron and body motor neuron; Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s43588-024-00738-w#MOESM1">4</a>). Second, five neurons of each functional group (AWC<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 35" title="Ramot, D., MacInnis, B. L. &amp; Goodman, M. B. Bidirectional temperature-sensing by a single thermosensory neuron in C. elegans. Nat. Neurosci. 11, 908–915 (2008)." href="https://www.nature.com/articles/s43588-024-00738-w#ref-CR35" id="ref-link-section-d492457827e1604">35</a></sup>, AIY<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 36" title="Liu, Q., Kidd, P. B., Dobosiewicz, M. &amp; Bargmann, C. I. C. elegans AWA olfactory neurons fire calcium-mediated all-or-none action potentials. Cell 175, 57–70.e17 (2018)." href="https://www.nature.com/articles/s43588-024-00738-w#ref-CR36" id="ref-link-section-d492457827e1609">36</a></sup>, AVA<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 37" title="Mellem, J. E., Brockie, P. J., Madsen, D. M. &amp; Maricq, A. V. Action potentials contribute to neuronal signaling in C. elegans. Nat. Neurosci. 11, 865–867 (2008)." href="https://www.nature.com/articles/s43588-024-00738-w#ref-CR37" id="ref-link-section-d492457827e1613">37</a></sup>, RIM<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 36" title="Liu, Q., Kidd, P. B., Dobosiewicz, M. &amp; Bargmann, C. I. C. elegans AWA olfactory neurons fire calcium-mediated all-or-none action potentials. Cell 175, 57–70.e17 (2018)." href="https://www.nature.com/articles/s43588-024-00738-w#ref-CR36" id="ref-link-section-d492457827e1617">36</a></sup> and VD5 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 38" title="Liu, P., Chen, B. &amp; Wang, Z.-W. SLO-2 potassium channel is an important regulator of neurotransmitter release in Caenorhabditis elegans. Nat. Commun. 5, 5155 (2014)." href="https://www.nature.com/articles/s43588-024-00738-w#ref-CR38" id="ref-link-section-d492457827e1621">38</a></sup>)) were selected and their model parameters were optimized as described above. These parameter values of optimized neuron models were then generalized to other neuron models according to their functional categories. In other words, neuron models in the same functional group share identical electrical parameters. Each neuron model has its own morphologies, yet it has electrical properties similar to those of neurons in the same functional group.</p><p>The electrophysiology data in Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s43588-024-00738-w#MOESM1">1</a> (gray traces) were collected from published papers. For the AIY (Liu et al.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 36" title="Liu, Q., Kidd, P. B., Dobosiewicz, M. &amp; Bargmann, C. I. C. elegans AWA olfactory neurons fire calcium-mediated all-or-none action potentials. Cell 175, 57–70.e17 (2018)." href="https://www.nature.com/articles/s43588-024-00738-w#ref-CR36" id="ref-link-section-d492457827e1631">36</a></sup>, fig. 1a) and RIM (Liu et al.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 36" title="Liu, Q., Kidd, P. B., Dobosiewicz, M. &amp; Bargmann, C. I. C. elegans AWA olfactory neurons fire calcium-mediated all-or-none action potentials. Cell 175, 57–70.e17 (2018)." href="https://www.nature.com/articles/s43588-024-00738-w#ref-CR36" id="ref-link-section-d492457827e1635">36</a></sup>, fig. 1a), the raw data were included as supplementary files in the original publications. However, for the AWC (Ramot et al.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 35" title="Ramot, D., MacInnis, B. L. &amp; Goodman, M. B. Bidirectional temperature-sensing by a single thermosensory neuron in C. elegans. Nat. Neurosci. 11, 908–915 (2008)." href="https://www.nature.com/articles/s43588-024-00738-w#ref-CR35" id="ref-link-section-d492457827e1639">35</a></sup>, fig. 8b), AVA (Mellem et al.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 37" title="Mellem, J. E., Brockie, P. J., Madsen, D. M. &amp; Maricq, A. V. Action potentials contribute to neuronal signaling in C. elegans. Nat. Neurosci. 11, 865–867 (2008)." href="https://www.nature.com/articles/s43588-024-00738-w#ref-CR37" id="ref-link-section-d492457827e1643">37</a></sup>, fig. 1d) and VD5 (Liu et al.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 38" title="Liu, P., Chen, B. &amp; Wang, Z.-W. SLO-2 potassium channel is an important regulator of neurotransmitter release in Caenorhabditis elegans. Nat. Commun. 5, 5155 (2014)." href="https://www.nature.com/articles/s43588-024-00738-w#ref-CR38" id="ref-link-section-d492457827e1648">38</a></sup>, fig. 8 (wild type)), the numerical data were not provided. Therefore, we extracted the numerical data from the figures using a digital image processing software (GetData Graph Digitizer). The extracted numerical traces are saved as data files in our GitHub respository (<a href="https://github.com/Jessie940611/BAAIWorm/tree/main/eworm/model_figure/single_neuron/electrophysiology_data">https://github.com/Jessie940611/BAAIWorm/tree/main/eworm/model_figure/single_neuron/electrophysiology_data</a>).</p><h4 id="Sec11">Graded synapse and gap junction models</h4><p>The neuron models were connected by graded synapses<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 36" title="Liu, Q., Kidd, P. B., Dobosiewicz, M. &amp; Bargmann, C. I. C. elegans AWA olfactory neurons fire calcium-mediated all-or-none action potentials. Cell 175, 57–70.e17 (2018)." href="https://www.nature.com/articles/s43588-024-00738-w#ref-CR36" id="ref-link-section-d492457827e1667">36</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 60" title="Liu, Q., Hollopeter, G. &amp; Jorgensen, E. M. Graded synaptic transmission at the Caenorhabditis elegans neuromuscular junction. Proc. Natl Acad. Sci. USA 106, 10823–10828 (2009)." href="https://www.nature.com/articles/s43588-024-00738-w#ref-CR60" id="ref-link-section-d492457827e1670">60</a></sup> and gap junctions<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 61" title="Shui, Y., Liu, P., Zhan, H., Chen, B. &amp; Wang, Z.-W. Molecular basis of junctional current rectification at an electrical synapse. Sci. Adv. 6, eabb3076 (2020)." href="https://www.nature.com/articles/s43588-024-00738-w#ref-CR61" id="ref-link-section-d492457827e1674">61</a></sup> in the <i>C. elegans</i> neural network. Graded synapses, including excitatory and inhibitory synapses, were modeled according to published <i>C. elegans</i> models<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 9" title="Gleeson, P., Lung, D., Grosu, R., Hasani, R. &amp; Larson, S. D. c302: a multiscale framework for modelling the nervous system of Caenorhabditis elegans. Philos. Trans. R. Soc. B 373, 20170379 (2018)." href="https://www.nature.com/articles/s43588-024-00738-w#ref-CR9" id="ref-link-section-d492457827e1684">9</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 40" title="Wicks, S. R., Roehrig, C. J. &amp; Rankin, C. H. A dynamic network simulation of the nematode tap withdrawal circuit: predictions concerning synaptic function using behavioral criteria. J. Neurosci. 16, 4017–4031 (1996)." href="https://www.nature.com/articles/s43588-024-00738-w#ref-CR40" id="ref-link-section-d492457827e1687">40</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 41" title="Prinz, A. A., Bucher, D. &amp; Marder, E. Similar network activity from disparate circuit parameters. Nat. Neurosci. 7, 1345–1352 (2004)." href="https://www.nature.com/articles/s43588-024-00738-w#ref-CR41" id="ref-link-section-d492457827e1690">41</a></sup>. The synaptic conductance is continuously changed with the presynaptic membrane potential. The postsynaptic current is given by</p><div id="Equa"><p><span>$$\left\{\begin{array}{c}I=-ws(E-V)\\ \displaystyle\frac{{\mathrm{d}}s}{{\mathrm{d}}t}=\displaystyle\frac{{s}_{\infty }-s}{\tau }\\ {s}_{\infty }=\displaystyle\frac{1}{1+{{\mathrm{e}}}^{\displaystyle\frac{{V}_{{\mathrm{th}}}-{V}_{{\mathrm{pre}}}}{\delta }}},\tau =\displaystyle\frac{1-{s}_{\infty }}{k}\end{array}\right.,$$</span></p></div><p>where <i>w</i> is the maximal postsynaptic membrane conductance for the synapse, <i>s</i> is a voltage-sensitive parameter controlling the conductance, <i>E</i> is the reversal potential, <i>τ</i> is the time constant of <i>s</i>, <i>V</i><sub>th</sub> is the presynaptic equilibrium potential at the middle of the voltage-sensitive range and <i>V</i><sub>pre</sub> is the presynaptic membrane potential.</p><p>Gap junctions were modeled as simple resistances, where current flowing from presynaptic cell to postsynaptic cell is given by</p><div id="Equb"><p><span>$$I=-w\left({V}_{{{\mathrm{pre}}}}-V\right),$$</span></p></div><p>where <i>w</i> is the conductance of gap junction.</p><p>The parameter values of these synapses were set according to published models<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 61" title="Shui, Y., Liu, P., Zhan, H., Chen, B. &amp; Wang, Z.-W. Molecular basis of junctional current rectification at an electrical synapse. Sci. Adv. 6, eabb3076 (2020)." href="https://www.nature.com/articles/s43588-024-00738-w#ref-CR61" id="ref-link-section-d492457827e2022">61</a></sup>: the conductance of the synapses and gap junctions had the same order of magnitude (1 × 10<sup>−4</sup> μS). The specific weight of each synaptic connection in a specific neural network can be optimized according to a target, as described in ‘Connection parameter optimization’ section.</p><h4 id="Sec12">Connection locations</h4><p>Detailed neuron connections in the neural network model were located on the neuronal neurite (axons or dendrites) of the neurons, which were based on the neuron connectivity matrices and a neurite connectivity algorithm.</p><p>The neuron connectivity matrices were acquired from published connectome data<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 15" title="Cook, S. J. et al. Whole-animal connectomes of both Caenorhabditis elegans sexes. Nature 571, 63–71 (2019)." href="https://www.nature.com/articles/s43588-024-00738-w#ref-CR15" id="ref-link-section-d492457827e2039">15</a></sup>. The original data in the matrices were the total number of electron microscopy (EM) series of all synapses (or gap junctions) between two neurons. In our work, the number of EM series was transformed into the number of synapses (or gap junctions) by</p><div id="Equc"><p><span>$$N=a\times\tanh \left(b\times C\right),$$</span></p></div><p>where <i>C</i> is the number of EM series and <i>a</i> and <i>b</i> are constants. For synapses, <span>\(a=23.91\)</span> and <span>\(b=0.02285\)</span>, while for gap junctions, <span>\(a=20.49\)</span> and <span>\(b=0.02184\)</span>. These parameters were inferred from partial-animal connectome data<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 15" title="Cook, S. J. et al. Whole-animal connectomes of both Caenorhabditis elegans sexes. Nature 571, 63–71 (2019)." href="https://www.nature.com/articles/s43588-024-00738-w#ref-CR15" id="ref-link-section-d492457827e2192">15</a></sup>.</p><p>After the number of synapses (or gap junctions) was determined, the locations of synapses (or gap junction) on neural neurites were identified by a connection algorithm. The algorithm was based on the rule that distances between the centroid coordinates of possible presynaptic and postsynaptic neurites follow a certain statistical distribution<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 17" title="Ruach, R., Ratner, N., Emmons, S. W. &amp; Zaslaver, A. The synaptic organization in the Caenorhabditis elegans neural network suggests significant local compartmentalized computations. Proc. Natl Acad. Sci. USA 120, e2201699120 (2023)." href="https://www.nature.com/articles/s43588-024-00738-w#ref-CR17" id="ref-link-section-d492457827e2199">17</a></sup>. We fit the synapse and gap junction distribution by two inverse Gaussian distributions</p><div id="Equd"><p><span>$$f\left(x,\mu \right)=\frac{1}{k\times \sqrt{2\uppi {x}^{3}}}\exp \left(-\frac{{(x-\mu )}^{2}}{2x{\mu }^{2}}\right),$$</span></p></div><p>where <i>μ</i> = 0.44 and <i>k</i> = 0.63 for synapse distribution, and <i>μ</i> = 0.70 and <i>k</i> = 0.40 for gap junction distribution. The algorithm of generating connection locations is in <a data-track="click" data-track-label="link" data-track-action="section anchor" href="https://www.nature.com/articles/s43588-024-00738-w#Sec30">Supplementary Algorithm</a>.</p><h4 id="Sec13">
                              <i>C. elegans</i> neural network model</h4><p>Our neural network model consists of 136 neurons, including 15 sensory neurons for inputs and 80 motor neurons for outputs (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s43588-024-00738-w#MOESM1">6</a>). We chose these neurons for the following reasons. First, 65 head ganglia neurons, whose neural dynamics were recorded in the published whole-brain Ca<sup>2+</sup> imaging experiment<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 24" title="Uzel, K., Kato, S. &amp; Zimmer, M. A set of hub neurons and non-local connectivity features support global brain dynamics in C. elegans. Curr. Biol. 32, 3443–3459 (2022)." href="https://www.nature.com/articles/s43588-024-00738-w#ref-CR24" id="ref-link-section-d492457827e2375">24</a></sup>, should be selected to enable the comparison between experiment and simulation. Second, other 71 neurons, most of which were motor neurons in the locomotion circuit of <i>C. elegans</i><sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 27" title="Gray, J. M., Hill, J. J. &amp; Bargmann, C. I. A circuit for navigation in Caenorhabditis elegans. Proc. Natl Acad. Sci. USA 102, 3184–3191 (2005)." href="https://www.nature.com/articles/s43588-024-00738-w#ref-CR27" id="ref-link-section-d492457827e2381">27</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 28" title="Chalasani, S. H. et al. Dissecting a circuit for olfactory behaviour in Caenorhabditis elegans. Nature 450, 63–70 (2007)." href="https://www.nature.com/articles/s43588-024-00738-w#ref-CR28" id="ref-link-section-d492457827e2384">28</a></sup>, were added to the network so that the model could be complete in a sensory–motor functional structure. In summary, this model contained neurons for chemosensory, decision and locomotion.</p><p>In our multicompartment neural network, our single-neuron models are not isopotential. For example, in the AVAL neuron, depolarization from the soma attenuates as it travels along the axon (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s43588-024-00738-w#MOESM1">9</a>). However, distal synapses receive inputs from other neurons, which helps to achieve sufficient depolarization. This design ensures that the model works well in the network.</p><h4 id="Sec14">Connection parameter optimization</h4><p>The weights of synapses and gap junctions in the neural network were initialized randomly. The polarities of synapses were initialized randomly according to a given excitatory/inhibitory ratio (2.33). These parameters were optimized using an unpublished optimization algorithm for multicompartment neural network. The algorithm was adapted from our previous work<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 62" title="Zhang, Y. et al. A GPU-based computational framework that bridges neuron simulation and artificial intelligence. Nat. Commun. 14, 5798 (2023)." href="https://www.nature.com/articles/s43588-024-00738-w#ref-CR62" id="ref-link-section-d492457827e2403">62</a></sup>, which combined cable theory and gradient-descent methods. It used a mathematical approximation approach to calculate gradients for backpropagation through time to update parameters. Except for the weights (of synapses and gap junctions) and the polarities (of synapses), the inputs (of some sensory neurons) can be optimized. The self-contained and executable code of this optimization is available in our GitHub repository (<a href="https://github.com/Jessie940611/BAAIWorm/tree/main/eworm_learn">https://github.com/Jessie940611/BAAIWorm/tree/main/eworm_learn</a>).</p><h4 id="Sec15">PCA of neural activities</h4><p>PCA was applied to the time series data of soma membrane potential of the 65 head neurons. Before conducting the analysis, the time series data were preprocessed by subtracting the mean value of the trace.</p><h4 id="Sec16">Correlation analysis</h4><p>The correlation between two neurons was quantified by the Pearson correlation coefficient of their neural activities, which ranged from −1 to 1. A coefficient of −1 or 1 indicates a strong negative or positive correlation of neural activity, respectively. A coefficient of 0 indicates that the neural activities of the two neurons are unrelated. In this analysis, the neural activity was represented by the membrane potential of soma.</p><h4 id="Sec17">Simulation environment</h4><p>The <i>C. elegans</i> neural network simulation and optimization were conducted with the code written in Python 3.8, and NEURON 8.0.0 was used as the simulation engine<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 63" title="Hines, M. L. &amp; Carnevale, N. T. The NEURON simulation environment. Neural Comput. 9, 1179–1209 (1997)." href="https://www.nature.com/articles/s43588-024-00738-w#ref-CR63" id="ref-link-section-d492457827e2441">63</a></sup>. Experiments were performed primarily on a system configured with Ubuntu 18.04, an NVIDIA GeForce RTX 3060 graphics processing unit, 16 GB random-access memory and an Intel Core i7-10700K central processing unit.</p><h3 id="Sec18">Details of the body–environment model</h3><h4 id="Sec19">The body model</h4><p>Tetrahedron is the simplest ordinary convex polyhedra in 3D space. It has four vertices and six edges and is bounded by four triangular faces. We used the tetrahedron as the basic element to construct the body model. The surface shape of the body was extracted as a triangle list from Sibernetic membrane data<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 64" title="Palyanov, A., Khayrulin, S. &amp; Larson, S. D. Three-dimensional simulation of the Caenorhabditis elegans body and muscle cells in liquid and gel environments for behavioural analysis. Philos. Trans. R. Soc. B 373, 20170376 (2018)." href="https://www.nature.com/articles/s43588-024-00738-w#ref-CR64" id="ref-link-section-d492457827e2458">64</a></sup>. Then, we triangulated the body with a robust and high-quality algorithm<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 65" title="Hu, Y. et al. Tetrahedral meshing in the wild. ACM Trans. Graph. 37, 1–14 (2018)." href="https://www.nature.com/articles/s43588-024-00738-w#ref-CR65" id="ref-link-section-d492457827e2462">65</a></sup>. The triangulated tetrahedron mesh was refined for the mechanics solver. The final body mesh was in a straight, natural resting pose that we defined as the standard body (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s43588-024-00738-w#Fig3">3a</a>). The triangulated tetrahedron body was not strictly symmetric in the dorsal–ventral direction and the left–right direction. The triangulation algorithm had additional constraints on the quality of tetrahedrons. This nonstrict symmetry is probably more consistent with real <i>C. elegans</i>. The above-mentioned Sibernetic worm body data can be obtained from OpenWorm<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 8" title="Sarma, G. P. et al. OpenWorm: overview and recent advances in integrative biological simulation of Caenorhabditis elegans. Philos. Trans. R. Soc. B 373, 20170382 (2018)." href="https://www.nature.com/articles/s43588-024-00738-w#ref-CR8" id="ref-link-section-d492457827e2472">8</a></sup> version ow-0.9.5. The triangulation algorithm of tetrahedron mesh can be obtained from TetWild<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 65" title="Hu, Y. et al. Tetrahedral meshing in the wild. ACM Trans. Graph. 37, 1–14 (2018)." href="https://www.nature.com/articles/s43588-024-00738-w#ref-CR65" id="ref-link-section-d492457827e2477">65</a></sup>.</p><h4 id="Sec20">Muscles</h4><p>Ninety-sixmuscles were arranged from head to tail according to the <i>C. elegans</i> anatomy. Each of the 24 muscles was grouped into a longitudinal string, marked as VR, VL, DR and DL in four quadrants (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s43588-024-00738-w#Fig3">3b</a>)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 64" title="Palyanov, A., Khayrulin, S. &amp; Larson, S. D. Three-dimensional simulation of the Caenorhabditis elegans body and muscle cells in liquid and gel environments for behavioural analysis. Philos. Trans. R. Soc. B 373, 20170376 (2018)." href="https://www.nature.com/articles/s43588-024-00738-w#ref-CR64" id="ref-link-section-d492457827e2495">64</a></sup>. We imported the body membrane mesh into Blender and modeled VR, VL, DR and DL by four Bezier curves. Then, the muscle geometry was exported as a wavefront obj file. We used FEM to simulate the soft body and used finite element constraints to simulate elastic muscle cells of the body. Each muscle cell (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s43588-024-00738-w#Fig3">3a</a>) corresponds to one or more FEM constraints according to the discretized tetrahedron body (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s43588-024-00738-w#Fig3">3c</a>). Only nearby tetrahedrons can be driven by corresponding muscle cells.</p><h4 id="Sec21">Soft-body solver</h4><p>We used projective dynamics<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 66" title="Bouaziz, S., Martin, S., Liu, T., Kavan, L. &amp; Pauly, M. Projective dynamics: fusing constraint projections for fast simulation. ACM Trans. Graph. 33, 1–11 (2014)." href="https://www.nature.com/articles/s43588-024-00738-w#ref-CR66" id="ref-link-section-d492457827e2513">66</a></sup> as the solver of the soft body. The purpose of the mechanics solver is to calculate the positions and velocities of all 3D vertices of the deformed body over time. The key idea of projective dynamics is to compute a local step and a global step iteratively. At the local step, vertex positions of the body are fixed and projection variables are updated. At the global step, projections are fixed and vertex positions of the body are updated. The local step is well suited for parallelism and, thus, can be optimized for performance. The global step is a linear system and can be precomputed if the body geometry and constraints are not changed. After a few local–global iterations, the vertex positions of deformed body are converged. Then velocity of deformed body can be computed by Euler integration.</p><h4 id="Sec22">Interaction with environment</h4><p>A simplified model of hydrodynamics was used to simulate the interaction between the body and the fluid, similarly to ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 43" title="Min, S., Won, J., Lee, S., Park, J. &amp; Lee, J. Softcon: simulation and control of soft-bodied animals with biomimetic actuators. ACM Trans. Graph. 38, 1–12 (2019)." href="https://www.nature.com/articles/s43588-024-00738-w#ref-CR43" id="ref-link-section-d492457827e2525">43</a></sup>. For simplicity, we supposed that external force is formed only at the surface of the body. Two kinds of external force were considered here: thrust force by body movement and drag force by fluid. With this simplification, computation in the whole 3D fluid environment is avoided. Thus, more agents, larger environments and more complex tasks are achievable through this simplification. SoftCon<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 43" title="Min, S., Won, J., Lee, S., Park, J. &amp; Lee, J. Softcon: simulation and control of soft-bodied animals with biomimetic actuators. ACM Trans. Graph. 38, 1–12 (2019)." href="https://www.nature.com/articles/s43588-024-00738-w#ref-CR43" id="ref-link-section-d492457827e2529">43</a></sup> was used as the basis of the body–environment model.</p><h4 id="Sec23">Numerical stable coordinate system for body movement</h4><p><i>C. elegans</i> is a soft-body animal that has special patterns during movement. Every position of the body is oscillating during movement. Moreover, as there are no apparent important marks on the body during locomotion, it is hard to find a stable coordinate system for <i>C. elegans</i> movement. Simply using the head or body center as a coordinate system is not sufficient to analyze the body’s movement quantitatively and stably. Here, we proposed an effective and simple method to measure <i>C. elegans</i> locomotion. The core of our method is to find a numerically stable coordinate system relative to the body itself. The main steps of our method are as follows:</p><ol>
                    <li>
                      <span>(1)</span>
                      
                        <p>Define the target body, standard body and SBCS.</p>
                      
                    </li>
                    <li>
                      <span>(2)</span>
                      
                        <p>Obtain transformation between the target body and the standard body.</p>
                      
                    </li>
                    <li>
                      <span>(3)</span>
                      
                        <p>Use the transformation in step 2 to perform 3D transformation on the SBCS to obtain the TBRCS.</p>
                      
                    </li>
                  </ol><p>Details of the three steps are described below:</p><ul>
                    <li>
                      <p>The target body is the research subject. It can change its shape over time and exhibits various behaviors. The standard body is a special case of the target body. Here, we define the standard body in a natural state where the head and tail are in a straight line, without stretching and bending. There is no internal force in the standard body soft body, and no external force acting on the standard body surface. The SBCS is a 3D orthogonal coordinate system used to reflect the overall structure and function of the standard body. In this case, we define the SBCS with origin at the standard body center. The <i>X</i> axis of the SBCS is toward the head of the standard body. The <i>Z</i> axis of the SBCS is toward the ventral–dorsal direction of the standard body. Three axes of the SBCS form a right-handed coordinate system. A schematic diagram of the target body, the standard body and the SBCS is shown in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s43588-024-00738-w#Fig4">4a</a>.</p>
                    </li>
                    <li>
                      <p>In this step, we define the corresponding matching points on the target body and the standard body by using all the vertices of the body. The transformation between the matching points on the target body and the standard body is calculated as the transformation between the target body and the standard body. The calculation formulas of the transformation are as follows:</p><div id="Eque"><p><span>$$\overline{{p}^{0}}=\frac{1}{m}\mathop{\sum }\limits_{i=1}^{m}{p}_{i}^{0}$$</span></p></div><div id="Equf"><p><span>$$\bar{p}=\frac{1}{m}\mathop{\sum }\limits_{i=1}^{m}{p}_{i}$$</span></p></div><div id="Equg"><p><span>$${q}_{i}^{0}={p}_{i}^{0}-\overline{{p}^{0}}$$</span></p></div><div id="Equh"><p><span>$${q}_{i}={p}_{i}-\bar{p}$$</span></p></div><div id="Equi"><p><span>$$H={q}_{i}^{0}{{q}_{i}}^{{\mathrm{T}}}$$</span></p></div><div id="Equj"><p><span>$$R=V{U}^{{\mathrm{T}}}$$</span></p></div><div id="Equk"><p><span>$$T=\bar{p}-R\overline{{p}^{0}}$$</span></p></div><div id="Equl"><p><span>$$M=\left[\begin{array}{cc}R &amp; T\\ 0 &amp; 1\end{array}\right].$$</span></p></div>
                      <p>The matching points are represented by column vectors. <span>\({p}_{i}^{0}\)</span> is matching points on the standard body, and <i>p</i><sub><i>i</i></sub> is matching points on the target body. <i>m</i> is the number of matching points. <span>\(\overline{{p}^{0}}\)</span> is the center point of matching points on the standard body, and <span>\(\overline{p}\)</span> is the center point of matching points on the standard body. <span>\({q}_{i}^{0}\)</span> refers to the standard body matching points with the center offset subtracted, and <i>q</i><sub><i>i</i></sub> denotes the standard body matching points with the center offset subtracted. <i>H</i> is a matrix computed by <span>\({q}_{i}^{0}{{q}_{i}}^{{\mathrm{T}}}\)</span>. <span>\({{q}_{i}}^{{\mathrm{T}}}\)</span> is the transpose of <i>q</i><sub><i>i</i></sub>, <span>\({q}_{i}=\left[{q}_{1},{q}_{2}\ldots {q}_{m}\right]\)</span>, <span>\({q}_{i}^{0}=\left[{q}_{1}^{0},{q}_{2}^{0}\ldots {q}_{m}^{0}\right]\)</span>. <i>U</i> and <i>V</i> are obtained by the singular value decomposition <span>\(H=U\Lambda V\)</span> of the matrix <i>H</i>. <i>U</i><sup>T</sup> is the transpose of <i>U</i>. <i>M</i> is the transformation matrix between the target body and the standard body, which will be used in the next step. <i>R</i> is the rotation part of <i>M</i>, and <i>T</i> is the translation part of <i>M</i>. A schematic diagram of matching points and transformation <i>M</i> is shown in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s43588-024-00738-w#Fig4">4a</a>.</p>
                    </li>
                    <li>
                      <p>In this step, we use matrix <i>M</i> to perform 3D transformation on the SBCS to obtain the TBRCS. The matrix of the TBRCS can be formulated as <span>\(\widehat{M}=M{M}^{0}=\left[\begin{array}{cc}R{R}^{0} &amp; R{T}^{\,0}+T\\ 0 &amp; 1\end{array}\right]\)</span>. <span>\(M=\left[\begin{array}{cc}R &amp; T\\ 0 &amp; 1\end{array}\right]\)</span>. <span>\({M}^{0}=\left[\begin{array}{cc}{R}^{0} &amp; {T}^{\,0}\\ 0 &amp; 1\end{array}\right]\)</span> is a matrix used to represent the SBCS. <i>T</i><sup>0</sup> is the origin of the SBCS, and <i>R</i><sup>0</sup> indicates orientation of the SBCS. The column vector of <i>R</i><sup>0</sup> represents the orthogonal coordinate axis. Finally, weobtain the TBRCS through <span>\(\widehat{M}\)</span>: <span>\(R{T}^{\,0}+T\)</span> as the origin and <span>\({{RR}}^{0}\)</span> as the orthogonal coordinate axis.</p>
                    </li>
                  </ul><h4 id="Sec24">Numerically stable measurements of body movement</h4>
                    <h3 id="FPar1">Measurement of body trajectory</h3>
                    <p>During behavior analysis, <i>C. elegans</i> can be seen as a deformable biology composed of multiple points. When studying the locomotion of a deformable body, generally one or more points of the biology body is defined as a tracking object, and locomotion of the deformable body is studied through tracking these defined points. For deformable biology bodies such as <i>C. elegans</i>, the position and the velocity of local tracking points do not reflect well the overall state of body locomotion (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s43588-024-00738-w#Fig4">4c</a>). As a result, values such as body trajectory or steering angle obtained through these tracking points are usually unstable over time. Using the TBRCS, we can better describe the overall trend of body movement. A comparison of trajectories is shown in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s43588-024-00738-w#Fig4">4a</a>. If the center point of the body is used as the trajectory, the trajectory has a zigzag shape. However, if the TBRCS is used as the trajectory of body movement, the trajectory is a smooth curve, which means that it has stable values. These results indicate that the TBRCS is suitable as a measurement of overall path and direction of body locomotion.</p>
                  
                    <h3 id="FPar2">Tracking body movement</h3>
                    <p>Here, we demonstrate how to track body movement with the TBRCS. If the standard body is drawn in the TBRCS, then it can be used as a reference for the target body. The standard body and the target body are centrally aligned, translated and rotated together. We define <span>\({s}_{k}^{0}\)</span> as 3D tracking points of the standard body and <i>s</i><sub><i>k</i></sub> as 3D tracking points of the target body (<i>k</i> = 1, 2, &amp;hellips;, <i>n</i>). <i>n</i> is the number of tracking points. The relative movement of the body can be defined as <span>\({s}_{k}-{s}_{k}^{0}\)</span>. It is a measurement of body locomotion relative to itself (the standard body), which is more numerically stable than relative to the environment. <span>\({s}_{k}-{s}_{k}^{0}\)</span> is also a better calculation than that relative to the body center, since both the translation part and the rotation part of the whole deformation body are considered in the TBRCS. Moreover, the sampling bias of tracking points is eliminated through subtraction, since bias minus bias equals zero.</p>
                    <p>In Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s43588-024-00738-w#Fig4">4b</a>, we sampled 17 tracking points on the body uniformly. The offset of the tracking points is indicated by the pink lines between the tracking points. The relative velocity of 3D tracking points is indicated with blue arrows. As a common method, trajectories of sampled 17 tracking points that relate to world are also shown in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s43588-024-00738-w#Fig4">4c</a>. We can see that, compared with all 17 tracking points trajectories, the trajectory of the TBRCS is the smoothest. The relative position and the relative velocity of 3D tracking points over time are plotted in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s43588-024-00738-w#Fig4">4e,f</a>. In Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s43588-024-00738-w#Fig4">4e</a>, the positions of the 3D tracking points are tiled from bottom to top for clear seeing. In this case, the swing direction (<i>Z</i> direction of the TBRCS) of the swimming body can be clearly shown with the TBRCS. We also find that there is a spring movement in the <i>X</i> direction of the TBRCS (tail to head) during forward swimming. The locomotion in the <i>Y</i> direction of the TBRCS is nearly flat because muscle activations are approximately symmetrical in this direction. From Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s43588-024-00738-w#Fig4">4f</a> we can further deduce that the biggest propulsion of forward swimming comes from the tail (pink line), as faster swing generates more thrust force.</p>
                  
                    <h3 id="FPar3">Steering angle of body movement</h3>
                    <p>If <span>\({O}_{{t}_{i}}\)</span> is the trajectory of TBRCS origin at time <i>t</i><sub><i>i</i></sub>, <span>\({t}_{i}={t}_{1},{t}_{2}\ldots {t}_{m}\)</span>, then we can calculate the velocity of TBRCS by <span>\({v}_{i}=\frac{{O}_{{t}_{i}}-{O}_{{t}_{i-1}}}{{t}_{i}-{t}_{i-1}}\)</span>. In this way, velocity magnitude and velocity direction over time are numerically stable. We used angle <i>θ</i>, which is between the TBRCS <i>X</i> axis and <i>v</i><sub><i>i</i></sub> as a measurement of steering, as shown in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s43588-024-00738-w#Fig4">4d</a>. Both <i>v</i><sub><i>i</i></sub> and the TBRCS are numerically stable over time; thus, <i>θ</i> is also numerically stable.</p>
                  <h3 id="Sec25">Locomotion behavior simulation of <i>C. elegans</i>
                        </h3><h4 id="Sec26">Simulation configuration</h4><p>The neural network simulation was coupled with the physical body–environment simulation to achieve closed-loop interaction. Food was at the center of the simulated physical environment, acting as an attractor to <i>C. elegans</i>. The food concentration was linearly distributed in the environment. The neural system received currents regarded as sensory signals. The amplitude of the currents was proportional to the gradient of food concentration. We assumed that all sensors were located at the head of <i>C. elegans</i> and that the injected currents of all sensory neurons were the same (the gradient at the tracking point at the head of the <i>C. elegans</i> body). The outputs of the neural network controlled the movements of the body in the environment through a reservoir readout weight matrix. As the body moved in the physical environment, the distance between the body and the food changed, causing the food concentration gradient changed, which was transformed into currents injected into the soma of sensory neurons.</p><p>During the interaction of the two submodels, the simulation time step of the neural network was 5/3 ms and that of the physical environment simulation was 100 ms. The two systems synchronized every 100 ms to update the sensory signals and muscle forces.</p><h4 id="Sec27">Reservoir configuration</h4><p>To establish the neuron–muscle connection between the neural system and physical body muscles, a reservoir computing framework was used. The neural network model acted as the nonlinear reservoir. The membrane potentials of 80 motor neurons were linearly combined and mapped to 96 muscle forces via an 80-by-96 matrix.</p><p>The reservoir was trained using data extracted from a 10 s movement generated in the simulated physical environment. The pairs of food concentration and muscle force during the movements were used as inputs to the neural network and target outputs of the readout layer, respectively. Only the reservoir readout matrix was optimized, whereas the parameters in the neural network model were fixed.</p><h3 id="Sec28">User-friendly interfaces</h3><p>For simulation configuration, the neural network model consists of ion channel, neuron and network connection modules. The parameter and algorithm settings of each module can be modified independently. Parameters were saved in JavaScript Object Notation (JSON) files that are easy to read and modify.</p><p>For quantification, membrane potential of neurons, worm body data outputs, such as positions, velocities and trajectories of tracking points (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s43588-024-00738-w#Fig4">4</a>), and muscle signals (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s43588-024-00738-w#MOESM1">5</a>) can be exported seamlessly with our latest code (<a href="https://github.com/Jessie940611/BAAIWorm/blob/main/eworm/ghost_in_mesh_sim/worm_in_env.py">https://github.com/Jessie940611/BAAIWorm/blob/main/eworm/ghost_in_mesh_sim/worm_in_env.py</a>). Tracking points, including the head, center and tail of the worm body, are configurable within the model. Future updates will include 3D tetrahedron analysis algorithms and an extended Worm tracker Commons Object Notation (WCON)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 67" title="Javer, A. et al. An open-source platform for analyzing and sharing worm-behavior data. Nat. Methods 15, 645–646 (2018)." href="https://www.nature.com/articles/s43588-024-00738-w#ref-CR67" id="ref-link-section-d492457827e4577">67</a></sup> file format for the worm’s 3D structure.</p><h3 id="Sec29">Reporting summary</h3><p>Further information on research design is available in the <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s43588-024-00738-w#MOESM2">Nature Portfolio Reporting Summary</a> linked to this article.</p></div></div></section>
                    
                </div><div>
                <section data-title="Data availability"><div id="data-availability-section"><h2 id="data-availability">Data availability</h2><div id="data-availability-content">
              
              <p><a data-track="click" data-track-label="link" data-track-action="section anchor" href="https://www.nature.com/articles/s43588-024-00738-w#Sec31">Source data</a> are provided with this paper. All other data supporting the findings of this study are available in this Resource and <a data-track="click" data-track-label="link" data-track-action="section anchor" href="https://www.nature.com/articles/s43588-024-00738-w#Sec30">Supplementary Information</a>. These data are also available via Zenodo at <a href="https://doi.org/10.5281/zenodo.13951773">https://doi.org/10.5281/zenodo.13951773</a> (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 68" title="Mengdi, Z. et al. Data for an integrative data-driven model simulating C. elegans brain, body and environment interactions. Zenodo 
                  https://doi.org/10.5281/zenodo.13951773
                  
                 (2024)." href="https://www.nature.com/articles/s43588-024-00738-w#ref-CR68" id="ref-link-section-d492457827e4678">68</a></sup>) and GitHub at <a href="https://github.com/Jessie940611/BAAIWorm">https://github.com/Jessie940611/BAAIWorm</a>.</p>
            </div></div></section><section data-title="Code availability"></section><div id="MagazineFulltextArticleBodySuffix"><section aria-labelledby="Bib1" data-title="References"><div id="Bib1-section"><h2 id="Bib1">References</h2><div id="Bib1-content"><div data-container-section="references"><ol data-track-component="outbound reference" data-track-context="references section"><li data-counter="1."><p id="ref-CR1">Ramaswamy, S. Data-driven multiscale computational models of cortical and subcortical regions. <i>Curr. Opin. Neurobiol.</i> <b>85</b>, 102842 (2024).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.conb.2024.102842" data-track-item_id="10.1016/j.conb.2024.102842" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.conb.2024.102842" aria-label="Article reference 1" data-doi="10.1016/j.conb.2024.102842">Article</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 1" href="http://scholar.google.com/scholar_lookup?&amp;title=Data-driven%20multiscale%20computational%20models%20of%20cortical%20and%20subcortical%20regions&amp;journal=Curr.%20Opin.%20Neurobiol.&amp;doi=10.1016%2Fj.conb.2024.102842&amp;volume=85&amp;publication_year=2024&amp;author=Ramaswamy%2CS">
                    Google Scholar</a> 
                </p></li><li data-counter="2."><p id="ref-CR2">Ramaswamy, S., Colangelo, C. &amp; Markram, H. Data-driven modeling of cholinergic modulation of neural microcircuits: bridging neurons, synapses and network activity. <i>Front. Neural Circuits</i> <b>12</b>, 77 (2018).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.3389/fncir.2018.00077" data-track-item_id="10.3389/fncir.2018.00077" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.3389%2Ffncir.2018.00077" aria-label="Article reference 2" data-doi="10.3389/fncir.2018.00077">Article</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 2" href="http://scholar.google.com/scholar_lookup?&amp;title=Data-driven%20modeling%20of%20cholinergic%20modulation%20of%20neural%20microcircuits%3A%20bridging%20neurons%2C%20synapses%20and%20network%20activity&amp;journal=Front.%20Neural%20Circuits&amp;doi=10.3389%2Ffncir.2018.00077&amp;volume=12&amp;publication_year=2018&amp;author=Ramaswamy%2CS&amp;author=Colangelo%2CC&amp;author=Markram%2CH">
                    Google Scholar</a> 
                </p></li><li data-counter="3."><p id="ref-CR3">Billeh, Y. N. et al. Systematic integration of structural and functional data into multi-scale models of mouse primary visual cortex. <i>Neuron</i> <b>106</b>, 388–403. e318 (2020).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.neuron.2020.01.040" data-track-item_id="10.1016/j.neuron.2020.01.040" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuron.2020.01.040" aria-label="Article reference 3" data-doi="10.1016/j.neuron.2020.01.040">Article</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 3" href="http://scholar.google.com/scholar_lookup?&amp;title=Systematic%20integration%20of%20structural%20and%20functional%20data%20into%20multi-scale%20models%20of%20mouse%20primary%20visual%20cortex&amp;journal=Neuron&amp;doi=10.1016%2Fj.neuron.2020.01.040&amp;volume=106&amp;pages=388-403.%20e318&amp;publication_year=2020&amp;author=Billeh%2CYN">
                    Google Scholar</a> 
                </p></li><li data-counter="4."><p id="ref-CR4">Hjorth, J. J. et al. The microcircuits of striatum in silico. <i>Proc. Natl Acad. Sci. USA</i> <b>117</b>, 9554–9565 (2020).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1073/pnas.2000671117" data-track-item_id="10.1073/pnas.2000671117" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1073%2Fpnas.2000671117" aria-label="Article reference 4" data-doi="10.1073/pnas.2000671117">Article</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 4" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20microcircuits%20of%20striatum%20in%20silico&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;doi=10.1073%2Fpnas.2000671117&amp;volume=117&amp;pages=9554-9565&amp;publication_year=2020&amp;author=Hjorth%2CJJ">
                    Google Scholar</a> 
                </p></li><li data-counter="5."><p id="ref-CR5">Markram, H. et al. Reconstruction and simulation of neocortical microcircuitry. <i>Cell</i> <b>163</b>, 456–492 (2015).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.cell.2015.09.029" data-track-item_id="10.1016/j.cell.2015.09.029" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.cell.2015.09.029" aria-label="Article reference 5" data-doi="10.1016/j.cell.2015.09.029">Article</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 5" href="http://scholar.google.com/scholar_lookup?&amp;title=Reconstruction%20and%20simulation%20of%20neocortical%20microcircuitry&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2015.09.029&amp;volume=163&amp;pages=456-492&amp;publication_year=2015&amp;author=Markram%2CH">
                    Google Scholar</a> 
                </p></li><li data-counter="6."><p id="ref-CR6">Lobato-Rios, V. et al. NeuroMechFly, a neuromechanical model of adult <i>Drosophila melanogaster</i>. <i>Nat. Methods</i> <b>19</b>, 620–627 (2022).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41592-022-01466-7" data-track-item_id="10.1038/s41592-022-01466-7" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41592-022-01466-7" aria-label="Article reference 6" data-doi="10.1038/s41592-022-01466-7">Article</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 6" href="http://scholar.google.com/scholar_lookup?&amp;title=NeuroMechFly%2C%20a%20neuromechanical%20model%20of%20adult%20Drosophila%20melanogaster&amp;journal=Nat.%20Methods&amp;doi=10.1038%2Fs41592-022-01466-7&amp;volume=19&amp;pages=620-627&amp;publication_year=2022&amp;author=Lobato-Rios%2CV">
                    Google Scholar</a> 
                </p></li><li data-counter="7."><p id="ref-CR7">Merel, J. et al. Deep neuroethology of a virtual rodent. In <i>Proc. 8th International Conference on Learning Representations</i> 11686–11705 (ICLR, 2020).</p></li><li data-counter="8."><p id="ref-CR8">Sarma, G. P. et al. OpenWorm: overview and recent advances in integrative biological simulation of <i>Caenorhabditis elegans</i>. <i>Philos. Trans. R. Soc. B</i> <b>373</b>, 20170382 (2018).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1098/rstb.2017.0382" data-track-item_id="10.1098/rstb.2017.0382" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1098%2Frstb.2017.0382" aria-label="Article reference 8" data-doi="10.1098/rstb.2017.0382">Article</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 8" href="http://scholar.google.com/scholar_lookup?&amp;title=OpenWorm%3A%20overview%20and%20recent%20advances%20in%20integrative%20biological%20simulation%20of%20Caenorhabditis%20elegans&amp;journal=Philos.%20Trans.%20R.%20Soc.%20B&amp;doi=10.1098%2Frstb.2017.0382&amp;volume=373&amp;publication_year=2018&amp;author=Sarma%2CGP">
                    Google Scholar</a> 
                </p></li><li data-counter="9."><p id="ref-CR9">Gleeson, P., Lung, D., Grosu, R., Hasani, R. &amp; Larson, S. D. c302: a multiscale framework for modelling the nervous system of <i>Caenorhabditis elegans</i>. <i>Philos. Trans. R. Soc. B</i> <b>373</b>, 20170379 (2018).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1098/rstb.2017.0379" data-track-item_id="10.1098/rstb.2017.0379" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1098%2Frstb.2017.0379" aria-label="Article reference 9" data-doi="10.1098/rstb.2017.0379">Article</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 9" href="http://scholar.google.com/scholar_lookup?&amp;title=c302%3A%20a%20multiscale%20framework%20for%20modelling%20the%20nervous%20system%20of%20Caenorhabditis%20elegans&amp;journal=Philos.%20Trans.%20R.%20Soc.%20B&amp;doi=10.1098%2Frstb.2017.0379&amp;volume=373&amp;publication_year=2018&amp;author=Gleeson%2CP&amp;author=Lung%2CD&amp;author=Grosu%2CR&amp;author=Hasani%2CR&amp;author=Larson%2CSD">
                    Google Scholar</a> 
                </p></li><li data-counter="10."><p id="ref-CR10">Palyanov, A. Y. &amp; Khayrulin, S. S. Sibernetic: a software complex based on the PCI SPH algorithm aimed at simulation problems in biomechanics. <i>Russ. J. Genet. Appl. Res.</i> <b>5</b>, 635–641 (2015).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1134/S2079059715060052" data-track-item_id="10.1134/S2079059715060052" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1134%2FS2079059715060052" aria-label="Article reference 10" data-doi="10.1134/S2079059715060052">Article</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 10" href="http://scholar.google.com/scholar_lookup?&amp;title=Sibernetic%3A%20a%20software%20complex%20based%20on%20the%20PCI%20SPH%20algorithm%20aimed%20at%20simulation%20problems%20in%20biomechanics&amp;journal=Russ.%20J.%20Genet.%20Appl.%20Res.&amp;doi=10.1134%2FS2079059715060052&amp;volume=5&amp;pages=635-641&amp;publication_year=2015&amp;author=Palyanov%2CAY&amp;author=Khayrulin%2CSS">
                    Google Scholar</a> 
                </p></li><li data-counter="11."><p id="ref-CR11">Urai, A. E., Doiron, B., Leifer, A. M. &amp; Churchland, A. K. Large-scale neural recordings call for new insights to link brain and behavior. <i>Nat. Neurosci.</i> <b>25</b>, 11–19 (2022).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41593-021-00980-9" data-track-item_id="10.1038/s41593-021-00980-9" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41593-021-00980-9" aria-label="Article reference 11" data-doi="10.1038/s41593-021-00980-9">Article</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 11" href="http://scholar.google.com/scholar_lookup?&amp;title=Large-scale%20neural%20recordings%20call%20for%20new%20insights%20to%20link%20brain%20and%20behavior&amp;journal=Nat.%20Neurosci.&amp;doi=10.1038%2Fs41593-021-00980-9&amp;volume=25&amp;pages=11-19&amp;publication_year=2022&amp;author=Urai%2CAE&amp;author=Doiron%2CB&amp;author=Leifer%2CAM&amp;author=Churchland%2CAK">
                    Google Scholar</a> 
                </p></li><li data-counter="12."><p id="ref-CR12">Zador, A. et al. Catalyzing next-generation artificial intelligence through NeuroAI. <i>Nat. Commun.</i> <b>14</b>, 1597 (2023).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41467-023-37180-x" data-track-item_id="10.1038/s41467-023-37180-x" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41467-023-37180-x" aria-label="Article reference 12" data-doi="10.1038/s41467-023-37180-x">Article</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 12" href="http://scholar.google.com/scholar_lookup?&amp;title=Catalyzing%20next-generation%20artificial%20intelligence%20through%20NeuroAI&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-023-37180-x&amp;volume=14&amp;publication_year=2023&amp;author=Zador%2CA">
                    Google Scholar</a> 
                </p></li><li data-counter="13."><p id="ref-CR13">Roth, E., Sponberg, S. &amp; Cowan, N. J. A comparative approach to closed-loop computation. <i>Curr. Opin. Neurobiol.</i> <b>25</b>, 54–62 (2014).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.conb.2013.11.005" data-track-item_id="10.1016/j.conb.2013.11.005" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.conb.2013.11.005" aria-label="Article reference 13" data-doi="10.1016/j.conb.2013.11.005">Article</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 13" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20comparative%20approach%20to%20closed-loop%20computation&amp;journal=Curr.%20Opin.%20Neurobiol.&amp;doi=10.1016%2Fj.conb.2013.11.005&amp;volume=25&amp;pages=54-62&amp;publication_year=2014&amp;author=Roth%2CE&amp;author=Sponberg%2CS&amp;author=Cowan%2CNJ">
                    Google Scholar</a> 
                </p></li><li data-counter="14."><p id="ref-CR14">White, J. G., Southgate, E., Thomson, J. N. &amp; Brenner, S. The structure of the nervous system of the nematode <i>Caenorhabditis elegans</i>. <i>Philos. Trans. R. Soc. Lond. B</i> <b>314</b>, 1–340 (1986).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1098/rstb.1986.0056" data-track-item_id="10.1098/rstb.1986.0056" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1098%2Frstb.1986.0056" aria-label="Article reference 14" data-doi="10.1098/rstb.1986.0056">Article</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 14" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20structure%20of%20the%20nervous%20system%20of%20the%20nematode%20Caenorhabditis%20elegans&amp;journal=Philos.%20Trans.%20R.%20Soc.%20Lond.%20B&amp;doi=10.1098%2Frstb.1986.0056&amp;volume=314&amp;pages=1-340&amp;publication_year=1986&amp;author=White%2CJG&amp;author=Southgate%2CE&amp;author=Thomson%2CJN&amp;author=Brenner%2CS">
                    Google Scholar</a> 
                </p></li><li data-counter="15."><p id="ref-CR15">Cook, S. J. et al. Whole-animal connectomes of both <i>Caenorhabditis elegans</i> sexes. <i>Nature</i> <b>571</b>, 63–71 (2019).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41586-019-1352-7" data-track-item_id="10.1038/s41586-019-1352-7" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41586-019-1352-7" aria-label="Article reference 15" data-doi="10.1038/s41586-019-1352-7">Article</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 15" href="http://scholar.google.com/scholar_lookup?&amp;title=Whole-animal%20connectomes%20of%20both%20Caenorhabditis%20elegans%20sexes&amp;journal=Nature&amp;doi=10.1038%2Fs41586-019-1352-7&amp;volume=571&amp;pages=63-71&amp;publication_year=2019&amp;author=Cook%2CSJ">
                    Google Scholar</a> 
                </p></li><li data-counter="16."><p id="ref-CR16">Witvliet, D. et al. Connectomes across development reveal principles of brain maturation. <i>Nature</i> <b>596</b>, 257–261 (2021).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41586-021-03778-8" data-track-item_id="10.1038/s41586-021-03778-8" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41586-021-03778-8" aria-label="Article reference 16" data-doi="10.1038/s41586-021-03778-8">Article</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 16" href="http://scholar.google.com/scholar_lookup?&amp;title=Connectomes%20across%20development%20reveal%20principles%20of%20brain%20maturation&amp;journal=Nature&amp;doi=10.1038%2Fs41586-021-03778-8&amp;volume=596&amp;pages=257-261&amp;publication_year=2021&amp;author=Witvliet%2CD">
                    Google Scholar</a> 
                </p></li><li data-counter="17."><p id="ref-CR17">Ruach, R., Ratner, N., Emmons, S. W. &amp; Zaslaver, A. The synaptic organization in the <i>Caenorhabditis elegans</i> neural network suggests significant local compartmentalized computations. <i>Proc. Natl Acad. Sci. USA</i> <b>120</b>, e2201699120 (2023).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1073/pnas.2201699120" data-track-item_id="10.1073/pnas.2201699120" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1073%2Fpnas.2201699120" aria-label="Article reference 17" data-doi="10.1073/pnas.2201699120">Article</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 17" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20synaptic%20organization%20in%20the%20Caenorhabditis%20elegans%20neural%20network%20suggests%20significant%20local%20compartmentalized%20computations&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;doi=10.1073%2Fpnas.2201699120&amp;volume=120&amp;publication_year=2023&amp;author=Ruach%2CR&amp;author=Ratner%2CN&amp;author=Emmons%2CSW&amp;author=Zaslaver%2CA">
                    Google Scholar</a> 
                </p></li><li data-counter="18."><p id="ref-CR18">Brittin, C. A., Cook, S. J., Hall, D. H., Emmons, S. W. &amp; Cohen, N. A multi-scale brain map derived from whole-brain volumetric reconstructions. <i>Nature</i> <b>591</b>, 105–110 (2021).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41586-021-03284-x" data-track-item_id="10.1038/s41586-021-03284-x" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41586-021-03284-x" aria-label="Article reference 18" data-doi="10.1038/s41586-021-03284-x">Article</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 18" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20multi-scale%20brain%20map%20derived%20from%20whole-brain%20volumetric%20reconstructions&amp;journal=Nature&amp;doi=10.1038%2Fs41586-021-03284-x&amp;volume=591&amp;pages=105-110&amp;publication_year=2021&amp;author=Brittin%2CCA&amp;author=Cook%2CSJ&amp;author=Hall%2CDH&amp;author=Emmons%2CSW&amp;author=Cohen%2CN">
                    Google Scholar</a> 
                </p></li><li data-counter="19."><p id="ref-CR19">Venkatachalam, V. et al. Pan-neuronal imaging in roaming <i>Caenorhabditis elegans</i>. <i>Proc. Natl Acad. Sci. USA</i> <b>113</b>, E1082–E1088 (2016).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1073/pnas.1507109113" data-track-item_id="10.1073/pnas.1507109113" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1073%2Fpnas.1507109113" aria-label="Article reference 19" data-doi="10.1073/pnas.1507109113">Article</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 19" href="http://scholar.google.com/scholar_lookup?&amp;title=Pan-neuronal%20imaging%20in%20roaming%20Caenorhabditis%20elegans&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;doi=10.1073%2Fpnas.1507109113&amp;volume=113&amp;pages=E1082-E1088&amp;publication_year=2016&amp;author=Venkatachalam%2CV">
                    Google Scholar</a> 
                </p></li><li data-counter="20."><p id="ref-CR20">Li, H. et al. Fast whole‐body motor neuron calcium imaging of freely moving <i>Caenorhabditis elegans</i> without coverslip pressed. <i>Cytometry Part A</i> <b>99</b>, 1143–1157 (2021).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1002/cyto.a.24483" data-track-item_id="10.1002/cyto.a.24483" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1002%2Fcyto.a.24483" aria-label="Article reference 20" data-doi="10.1002/cyto.a.24483">Article</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 20" href="http://scholar.google.com/scholar_lookup?&amp;title=Fast%20whole%E2%80%90body%20motor%20neuron%20calcium%20imaging%20of%20freely%20moving%20Caenorhabditis%20elegans%20without%20coverslip%20pressed&amp;journal=Cytometry%20Part%20A&amp;doi=10.1002%2Fcyto.a.24483&amp;volume=99&amp;pages=1143-1157&amp;publication_year=2021&amp;author=Li%2CH">
                    Google Scholar</a> 
                </p></li><li data-counter="21."><p id="ref-CR21">Kato, S. et al. Global brain dynamics embed the motor command sequence of <i>Caenorhabditis elegans</i>. <i>Cell</i> <b>163</b>, 656–669 (2015).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.cell.2015.09.034" data-track-item_id="10.1016/j.cell.2015.09.034" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.cell.2015.09.034" aria-label="Article reference 21" data-doi="10.1016/j.cell.2015.09.034">Article</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 21" href="http://scholar.google.com/scholar_lookup?&amp;title=Global%20brain%20dynamics%20embed%20the%20motor%20command%20sequence%20of%20Caenorhabditis%20elegans&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2015.09.034&amp;volume=163&amp;pages=656-669&amp;publication_year=2015&amp;author=Kato%2CS">
                    Google Scholar</a> 
                </p></li><li data-counter="22."><p id="ref-CR22">Randi, F., Sharma, A. K., Dvali, S. &amp; Leifer, A. M. Neural signal propagation atlas of Caenorhabditis elegans. <i>Nature</i> <b>623</b>, 406–414 (2023).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41586-023-06683-4" data-track-item_id="10.1038/s41586-023-06683-4" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41586-023-06683-4" aria-label="Article reference 22" data-doi="10.1038/s41586-023-06683-4">Article</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 22" href="http://scholar.google.com/scholar_lookup?&amp;title=Neural%20signal%20propagation%20atlas%20of%20Caenorhabditis%20elegans&amp;journal=Nature&amp;doi=10.1038%2Fs41586-023-06683-4&amp;volume=623&amp;pages=406-414&amp;publication_year=2023&amp;author=Randi%2CF&amp;author=Sharma%2CAK&amp;author=Dvali%2CS&amp;author=Leifer%2CAM">
                    Google Scholar</a> 
                </p></li><li data-counter="23."><p id="ref-CR23">Bergs, A. C. et al. All-optical closed-loop voltage clamp for precise control of muscles and neurons in live animals. <i>Nat. Commun.</i> <b>14</b>, 1939 (2023).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41467-023-37622-6" data-track-item_id="10.1038/s41467-023-37622-6" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41467-023-37622-6" aria-label="Article reference 23" data-doi="10.1038/s41467-023-37622-6">Article</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 23" href="http://scholar.google.com/scholar_lookup?&amp;title=All-optical%20closed-loop%20voltage%20clamp%20for%20precise%20control%20of%20muscles%20and%20neurons%20in%20live%20animals&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-023-37622-6&amp;volume=14&amp;publication_year=2023&amp;author=Bergs%2CAC">
                    Google Scholar</a> 
                </p></li><li data-counter="24."><p id="ref-CR24">Uzel, K., Kato, S. &amp; Zimmer, M. A set of hub neurons and non-local connectivity features support global brain dynamics in <i>C. elegans</i>. <i>Curr. Biol.</i> <b>32</b>, 3443–3459 (2022).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.cub.2022.06.039" data-track-item_id="10.1016/j.cub.2022.06.039" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.cub.2022.06.039" aria-label="Article reference 24" data-doi="10.1016/j.cub.2022.06.039">Article</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 24" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20set%20of%20hub%20neurons%20and%20non-local%20connectivity%20features%20support%20global%20brain%20dynamics%20in%20C.%20elegans&amp;journal=Curr.%20Biol.&amp;doi=10.1016%2Fj.cub.2022.06.039&amp;volume=32&amp;pages=3443-3459&amp;publication_year=2022&amp;author=Uzel%2CK&amp;author=Kato%2CS&amp;author=Zimmer%2CM">
                    Google Scholar</a> 
                </p></li><li data-counter="25."><p id="ref-CR25">Bargmann, C. I. Genetic and cellular analysis of behavior in <i>C. elegans</i>. <i>Annu. Rev. Neurosci.</i> <b>16</b>, 47–71 (1993).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1146/annurev.ne.16.030193.000403" data-track-item_id="10.1146/annurev.ne.16.030193.000403" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1146%2Fannurev.ne.16.030193.000403" aria-label="Article reference 25" data-doi="10.1146/annurev.ne.16.030193.000403">Article</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 25" href="http://scholar.google.com/scholar_lookup?&amp;title=Genetic%20and%20cellular%20analysis%20of%20behavior%20in%20C.%20elegans&amp;journal=Annu.%20Rev.%20Neurosci.&amp;doi=10.1146%2Fannurev.ne.16.030193.000403&amp;volume=16&amp;pages=47-71&amp;publication_year=1993&amp;author=Bargmann%2CCI">
                    Google Scholar</a> 
                </p></li><li data-counter="26."><p id="ref-CR26">Rankin, C. H. From gene to identified neuron to behaviour in <i>Caenorhabditis elegans</i>. <i>Nat. Rev. Genet.</i> <b>3</b>, 622–630 (2002).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/nrg864" data-track-item_id="10.1038/nrg864" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fnrg864" aria-label="Article reference 26" data-doi="10.1038/nrg864">Article</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 26" href="http://scholar.google.com/scholar_lookup?&amp;title=From%20gene%20to%20identified%20neuron%20to%20behaviour%20in%20Caenorhabditis%20elegans&amp;journal=Nat.%20Rev.%20Genet.&amp;doi=10.1038%2Fnrg864&amp;volume=3&amp;pages=622-630&amp;publication_year=2002&amp;author=Rankin%2CCH">
                    Google Scholar</a> 
                </p></li><li data-counter="27."><p id="ref-CR27">Gray, J. M., Hill, J. J. &amp; Bargmann, C. I. A circuit for navigation in <i>Caenorhabditis elegans</i>. <i>Proc. Natl Acad. Sci. USA</i> <b>102</b>, 3184–3191 (2005).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1073/pnas.0409009101" data-track-item_id="10.1073/pnas.0409009101" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1073%2Fpnas.0409009101" aria-label="Article reference 27" data-doi="10.1073/pnas.0409009101">Article</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 27" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20circuit%20for%20navigation%20in%20Caenorhabditis%20elegans&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;doi=10.1073%2Fpnas.0409009101&amp;volume=102&amp;pages=3184-3191&amp;publication_year=2005&amp;author=Gray%2CJM&amp;author=Hill%2CJJ&amp;author=Bargmann%2CCI">
                    Google Scholar</a> 
                </p></li><li data-counter="28."><p id="ref-CR28">Chalasani, S. H. et al. Dissecting a circuit for olfactory behaviour in <i>Caenorhabditis elegans</i>. <i>Nature</i> <b>450</b>, 63–70 (2007).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/nature06292" data-track-item_id="10.1038/nature06292" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fnature06292" aria-label="Article reference 28" data-doi="10.1038/nature06292">Article</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 28" href="http://scholar.google.com/scholar_lookup?&amp;title=Dissecting%20a%20circuit%20for%20olfactory%20behaviour%20in%20Caenorhabditis%20elegans&amp;journal=Nature&amp;doi=10.1038%2Fnature06292&amp;volume=450&amp;pages=63-70&amp;publication_year=2007&amp;author=Chalasani%2CSH">
                    Google Scholar</a> 
                </p></li><li data-counter="29."><p id="ref-CR29">Larsch, J. et al. A circuit for gradient climbing in <i>C. elegans</i> chemotaxis. <i>Cell Rep.</i> <b>12</b>, 1748–1760 (2015).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.celrep.2015.08.032" data-track-item_id="10.1016/j.celrep.2015.08.032" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.celrep.2015.08.032" aria-label="Article reference 29" data-doi="10.1016/j.celrep.2015.08.032">Article</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 29" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20circuit%20for%20gradient%20climbing%20in%20C.%20elegans%20chemotaxis&amp;journal=Cell%20Rep.&amp;doi=10.1016%2Fj.celrep.2015.08.032&amp;volume=12&amp;pages=1748-1760&amp;publication_year=2015&amp;author=Larsch%2CJ">
                    Google Scholar</a> 
                </p></li><li data-counter="30."><p id="ref-CR30">Bargmann, C. I., Hartwieg, E. &amp; Horvitz, H. R. Odorant-selective genes and neurons mediate olfaction in <i>C. elegans</i>. <i>Cell</i> <b>74</b>, 515–527 (1993).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/0092-8674(93)80053-H" data-track-item_id="10.1016/0092-8674(93)80053-H" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2F0092-8674%2893%2980053-H" aria-label="Article reference 30" data-doi="10.1016/0092-8674(93)80053-H">Article</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 30" href="http://scholar.google.com/scholar_lookup?&amp;title=Odorant-selective%20genes%20and%20neurons%20mediate%20olfaction%20in%20C.%20elegans&amp;journal=Cell&amp;doi=10.1016%2F0092-8674%2893%2980053-H&amp;volume=74&amp;pages=515-527&amp;publication_year=1993&amp;author=Bargmann%2CCI&amp;author=Hartwieg%2CE&amp;author=Horvitz%2CHR">
                    Google Scholar</a> 
                </p></li><li data-counter="31."><p id="ref-CR31">Piggott, B. J., Liu, J., Feng, Z., Wescott, S. A. &amp; Xu, X. S. The neural circuits and synaptic mechanisms underlying motor initiation in <i>C. elegans</i>. <i>Cell</i> <b>147</b>, 922–933 (2011).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.cell.2011.08.053" data-track-item_id="10.1016/j.cell.2011.08.053" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.cell.2011.08.053" aria-label="Article reference 31" data-doi="10.1016/j.cell.2011.08.053">Article</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 31" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20neural%20circuits%20and%20synaptic%20mechanisms%20underlying%20motor%20initiation%20in%20C.%20elegans&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2011.08.053&amp;volume=147&amp;pages=922-933&amp;publication_year=2011&amp;author=Piggott%2CBJ&amp;author=Liu%2CJ&amp;author=Feng%2CZ&amp;author=Wescott%2CSA&amp;author=Xu%2CXS">
                    Google Scholar</a> 
                </p></li><li data-counter="32."><p id="ref-CR32">Grove, C. &amp; Sternberg, P. in <i>18th International C. elegans Meeting</i> (GSA, 2011).</p></li><li data-counter="33."><p id="ref-CR33">Nicoletti, M. et al. Biophysical modeling of <i>C. elegans</i> neurons: single ion currents and whole-cell dynamics of AWCon and RMD. <i>PLoS ONE</i> <b>14</b>, e0218738 (2019).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1371/journal.pone.0218738" data-track-item_id="10.1371/journal.pone.0218738" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1371%2Fjournal.pone.0218738" aria-label="Article reference 33" data-doi="10.1371/journal.pone.0218738">Article</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 33" href="http://scholar.google.com/scholar_lookup?&amp;title=Biophysical%20modeling%20of%20C.%20elegans%20neurons%3A%20single%20ion%20currents%20and%20whole-cell%20dynamics%20of%20AWCon%20and%20RMD&amp;journal=PLoS%20ONE&amp;doi=10.1371%2Fjournal.pone.0218738&amp;volume=14&amp;publication_year=2019&amp;author=Nicoletti%2CM">
                    Google Scholar</a> 
                </p></li><li data-counter="34."><p id="ref-CR34">Sheng, K. et al. Domain adaptive neural inference for neurons, microcircuits and networks. Preprint at <i>bioRxiv</i> <a href="https://www.biorxiv.org/content/10.1101/2022.10.03.510694v1" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="https://www.biorxiv.org/content/10.1101/2022.10.03.510694v1">https://www.biorxiv.org/content/10.1101/2022.10.03.510694v1</a> (2022).</p></li><li data-counter="35."><p id="ref-CR35">Ramot, D., MacInnis, B. L. &amp; Goodman, M. B. Bidirectional temperature-sensing by a single thermosensory neuron in <i>C. elegans</i>. <i>Nat. Neurosci.</i> <b>11</b>, 908–915 (2008).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/nn.2157" data-track-item_id="10.1038/nn.2157" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fnn.2157" aria-label="Article reference 35" data-doi="10.1038/nn.2157">Article</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 35" href="http://scholar.google.com/scholar_lookup?&amp;title=Bidirectional%20temperature-sensing%20by%20a%20single%20thermosensory%20neuron%20in%20C.%20elegans&amp;journal=Nat.%20Neurosci.&amp;doi=10.1038%2Fnn.2157&amp;volume=11&amp;pages=908-915&amp;publication_year=2008&amp;author=Ramot%2CD&amp;author=MacInnis%2CBL&amp;author=Goodman%2CMB">
                    Google Scholar</a> 
                </p></li><li data-counter="36."><p id="ref-CR36">Liu, Q., Kidd, P. B., Dobosiewicz, M. &amp; Bargmann, C. I. C. elegans AWA olfactory neurons fire calcium-mediated all-or-none action potentials. <i>Cell</i> <b>175</b>, 57–70.e17 (2018).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.cell.2018.08.018" data-track-item_id="10.1016/j.cell.2018.08.018" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.cell.2018.08.018" aria-label="Article reference 36" data-doi="10.1016/j.cell.2018.08.018">Article</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 36" href="http://scholar.google.com/scholar_lookup?&amp;title=elegans%20AWA%20olfactory%20neurons%20fire%20calcium-mediated%20all-or-none%20action%20potentials&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2018.08.018&amp;volume=175&amp;pages=57-70.e17&amp;publication_year=2018&amp;author=Liu%2CQ&amp;author=Kidd%2CPB&amp;author=Dobosiewicz%2CM&amp;author=Bargmann%2CCIC">
                    Google Scholar</a> 
                </p></li><li data-counter="37."><p id="ref-CR37">Mellem, J. E., Brockie, P. J., Madsen, D. M. &amp; Maricq, A. V. Action potentials contribute to neuronal signaling in <i>C. elegans</i>. <i>Nat. Neurosci.</i> <b>11</b>, 865–867 (2008).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/nn.2131" data-track-item_id="10.1038/nn.2131" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fnn.2131" aria-label="Article reference 37" data-doi="10.1038/nn.2131">Article</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 37" href="http://scholar.google.com/scholar_lookup?&amp;title=Action%20potentials%20contribute%20to%20neuronal%20signaling%20in%20C.%20elegans&amp;journal=Nat.%20Neurosci.&amp;doi=10.1038%2Fnn.2131&amp;volume=11&amp;pages=865-867&amp;publication_year=2008&amp;author=Mellem%2CJE&amp;author=Brockie%2CPJ&amp;author=Madsen%2CDM&amp;author=Maricq%2CAV">
                    Google Scholar</a> 
                </p></li><li data-counter="38."><p id="ref-CR38">Liu, P., Chen, B. &amp; Wang, Z.-W. SLO-2 potassium channel is an important regulator of neurotransmitter release in <i>Caenorhabditis elegans</i>. <i>Nat. Commun.</i> <b>5</b>, 5155 (2014).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/ncomms6155" data-track-item_id="10.1038/ncomms6155" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fncomms6155" aria-label="Article reference 38" data-doi="10.1038/ncomms6155">Article</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 38" href="http://scholar.google.com/scholar_lookup?&amp;title=SLO-2%20potassium%20channel%20is%20an%20important%20regulator%20of%20neurotransmitter%20release%20in%20Caenorhabditis%20elegans&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fncomms6155&amp;volume=5&amp;publication_year=2014&amp;author=Liu%2CP&amp;author=Chen%2CB&amp;author=Wang%2CZ-W">
                    Google Scholar</a> 
                </p></li><li data-counter="39."><p id="ref-CR39">Mainen, Z. F. &amp; Sejnowski, T. J. Influence of dendritic structure on firing pattern in model neocortical neurons. <i>Nature</i> <b>382</b>, 363 (1996).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/382363a0" data-track-item_id="10.1038/382363a0" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2F382363a0" aria-label="Article reference 39" data-doi="10.1038/382363a0">Article</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 39" href="http://scholar.google.com/scholar_lookup?&amp;title=Influence%20of%20dendritic%20structure%20on%20firing%20pattern%20in%20model%20neocortical%20neurons&amp;journal=Nature&amp;doi=10.1038%2F382363a0&amp;volume=382&amp;publication_year=1996&amp;author=Mainen%2CZF&amp;author=Sejnowski%2CTJ">
                    Google Scholar</a> 
                </p></li><li data-counter="40."><p id="ref-CR40">Wicks, S. R., Roehrig, C. J. &amp; Rankin, C. H. A dynamic network simulation of the nematode tap withdrawal circuit: predictions concerning synaptic function using behavioral criteria. <i>J. Neurosci.</i> <b>16</b>, 4017–4031 (1996).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1523/JNEUROSCI.16-12-04017.1996" data-track-item_id="10.1523/JNEUROSCI.16-12-04017.1996" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1523%2FJNEUROSCI.16-12-04017.1996" aria-label="Article reference 40" data-doi="10.1523/JNEUROSCI.16-12-04017.1996">Article</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 40" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20dynamic%20network%20simulation%20of%20the%20nematode%20tap%20withdrawal%20circuit%3A%20predictions%20concerning%20synaptic%20function%20using%20behavioral%20criteria&amp;journal=J.%20Neurosci.&amp;doi=10.1523%2FJNEUROSCI.16-12-04017.1996&amp;volume=16&amp;pages=4017-4031&amp;publication_year=1996&amp;author=Wicks%2CSR&amp;author=Roehrig%2CCJ&amp;author=Rankin%2CCH">
                    Google Scholar</a> 
                </p></li><li data-counter="41."><p id="ref-CR41">Prinz, A. A., Bucher, D. &amp; Marder, E. Similar network activity from disparate circuit parameters. <i>Nat. Neurosci.</i> <b>7</b>, 1345–1352 (2004).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/nn1352" data-track-item_id="10.1038/nn1352" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fnn1352" aria-label="Article reference 41" data-doi="10.1038/nn1352">Article</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 41" href="http://scholar.google.com/scholar_lookup?&amp;title=Similar%20network%20activity%20from%20disparate%20circuit%20parameters&amp;journal=Nat.%20Neurosci.&amp;doi=10.1038%2Fnn1352&amp;volume=7&amp;pages=1345-1352&amp;publication_year=2004&amp;author=Prinz%2CAA&amp;author=Bucher%2CD&amp;author=Marder%2CE">
                    Google Scholar</a> 
                </p></li><li data-counter="42."><p id="ref-CR42">Sakata, K. &amp; Shingai, R. Neural network model to generate head swing in locomotion of <i>Caenorhabditis elegans</i>. <i>Netw., Comput. Neural Syst.</i> <b>15</b>, 199 (2004).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1088/0954-898X_15_3_003" data-track-item_id="10.1088/0954-898X_15_3_003" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1088%2F0954-898X_15_3_003" aria-label="Article reference 42" data-doi="10.1088/0954-898X_15_3_003">Article</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 42" href="http://scholar.google.com/scholar_lookup?&amp;title=Neural%20network%20model%20to%20generate%20head%20swing%20in%20locomotion%20of%20Caenorhabditis%20elegans&amp;journal=Netw.%2C%20Comput.%20Neural%20Syst.&amp;doi=10.1088%2F0954-898X_15_3_003&amp;volume=15&amp;publication_year=2004&amp;author=Sakata%2CK&amp;author=Shingai%2CR">
                    Google Scholar</a> 
                </p></li><li data-counter="43."><p id="ref-CR43">Min, S., Won, J., Lee, S., Park, J. &amp; Lee, J. Softcon: simulation and control of soft-bodied animals with biomimetic actuators. <i>ACM Trans. Graph.</i> <b>38</b>, 1–12 (2019).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1145/3355089.3356497" data-track-item_id="10.1145/3355089.3356497" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1145%2F3355089.3356497" aria-label="Article reference 43" data-doi="10.1145/3355089.3356497">Article</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 43" href="http://scholar.google.com/scholar_lookup?&amp;title=Softcon%3A%20simulation%20and%20control%20of%20soft-bodied%20animals%20with%20biomimetic%20actuators&amp;journal=ACM%20Trans.%20Graph.&amp;doi=10.1145%2F3355089.3356497&amp;volume=38&amp;pages=1-12&amp;publication_year=2019&amp;author=Min%2CS&amp;author=Won%2CJ&amp;author=Lee%2CS&amp;author=Park%2CJ&amp;author=Lee%2CJ">
                    Google Scholar</a> 
                </p></li><li data-counter="44."><p id="ref-CR44">Sznitman, J., Purohit, P. K., Krajacic, P., Lamitina, T. &amp; Arratia, P. E. Material properties of <i>Caenorhabditis elegans</i> swimming at low reynolds number. <i>Biophys. J.</i> <b>98</b>, 617–626 (2010).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.bpj.2009.11.010" data-track-item_id="10.1016/j.bpj.2009.11.010" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.bpj.2009.11.010" aria-label="Article reference 44" data-doi="10.1016/j.bpj.2009.11.010">Article</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 44" href="http://scholar.google.com/scholar_lookup?&amp;title=Material%20properties%20of%20Caenorhabditis%20elegans%20swimming%20at%20low%20reynolds%20number&amp;journal=Biophys.%20J.&amp;doi=10.1016%2Fj.bpj.2009.11.010&amp;volume=98&amp;pages=617-626&amp;publication_year=2010&amp;author=Sznitman%2CJ&amp;author=Purohit%2CPK&amp;author=Krajacic%2CP&amp;author=Lamitina%2CT&amp;author=Arratia%2CPE">
                    Google Scholar</a> 
                </p></li><li data-counter="45."><p id="ref-CR45">Bilbao, A., Patel, A. K., Rahman, M., Vanapalli, S. A. &amp; Blawzdziewicz, J. Roll maneuvers are essential for active reorientation of <i>Caenorhabditis elegans</i> in 3D media. <i>Proc. Natl Acad. Sci. USA</i> <b>115</b>, E3616–E3625 (2018).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1073/pnas.1706754115" data-track-item_id="10.1073/pnas.1706754115" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1073%2Fpnas.1706754115" aria-label="Article reference 45" data-doi="10.1073/pnas.1706754115">Article</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 45" href="http://scholar.google.com/scholar_lookup?&amp;title=Roll%20maneuvers%20are%20essential%20for%20active%20reorientation%20of%20Caenorhabditis%20elegans%20in%203D%20media&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;doi=10.1073%2Fpnas.1706754115&amp;volume=115&amp;pages=E3616-E3625&amp;publication_year=2018&amp;author=Bilbao%2CA&amp;author=Patel%2CAK&amp;author=Rahman%2CM&amp;author=Vanapalli%2CSA&amp;author=Blawzdziewicz%2CJ">
                    Google Scholar</a> 
                </p></li><li data-counter="46."><p id="ref-CR46">Salfelder, F. et al. Markerless 3D spatio-temporal reconstruction of microscopic swimmers from video. In <i>25th International Conference on Pattern Recognition</i> 1965–1972 (IEEE, 2020).</p></li><li data-counter="47."><p id="ref-CR47">Bhovad, P. &amp; Li, S. Physical reservoir computing with origami and its application to robotic crawling. <i>Sci. Rep.</i> <b>11</b>, 13002 (2021).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41598-021-92257-1" data-track-item_id="10.1038/s41598-021-92257-1" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41598-021-92257-1" aria-label="Article reference 47" data-doi="10.1038/s41598-021-92257-1">Article</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 47" href="http://scholar.google.com/scholar_lookup?&amp;title=Physical%20reservoir%20computing%20with%20origami%20and%20its%20application%20to%20robotic%20crawling&amp;journal=Sci.%20Rep.&amp;doi=10.1038%2Fs41598-021-92257-1&amp;volume=11&amp;publication_year=2021&amp;author=Bhovad%2CP&amp;author=Li%2CS">
                    Google Scholar</a> 
                </p></li><li data-counter="48."><p id="ref-CR48">Wen, Q. et al. Proprioceptive coupling within motor neurons drives <i>C. elegans</i> forward locomotion. <i>Neuron</i> <b>76</b>, 750–761 (2012).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.neuron.2012.08.039" data-track-item_id="10.1016/j.neuron.2012.08.039" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuron.2012.08.039" aria-label="Article reference 48" data-doi="10.1016/j.neuron.2012.08.039">Article</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 48" href="http://scholar.google.com/scholar_lookup?&amp;title=Proprioceptive%20coupling%20within%20motor%20neurons%20drives%20C.%20elegans%20forward%20locomotion&amp;journal=Neuron&amp;doi=10.1016%2Fj.neuron.2012.08.039&amp;volume=76&amp;pages=750-761&amp;publication_year=2012&amp;author=Wen%2CQ">
                    Google Scholar</a> 
                </p></li><li data-counter="49."><p id="ref-CR49">Delp, S. L. et al. OpenSim: open-source software to create and analyze dynamic simulations of movement. <i>IEEE Trans. Biomed. Eng.</i> <b>54</b>, 1940–1950 (2007).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1109/TBME.2007.901024" data-track-item_id="10.1109/TBME.2007.901024" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1109%2FTBME.2007.901024" aria-label="Article reference 49" data-doi="10.1109/TBME.2007.901024">Article</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 49" href="http://scholar.google.com/scholar_lookup?&amp;title=OpenSim%3A%20open-source%20software%20to%20create%20and%20analyze%20dynamic%20simulations%20of%20movement&amp;journal=IEEE%20Trans.%20Biomed.%20Eng.&amp;doi=10.1109%2FTBME.2007.901024&amp;volume=54&amp;pages=1940-1950&amp;publication_year=2007&amp;author=Delp%2CSL">
                    Google Scholar</a> 
                </p></li><li data-counter="50."><p id="ref-CR50">Boyle, J. H., Berri, S. &amp; Cohen, N. Gait modulation in <i>C. elegans</i>: an integrated neuromechanical model. <i>Front. Comput. Neurosci.</i> <b>6</b>, 10 (2012).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.3389/fncom.2012.00010" data-track-item_id="10.3389/fncom.2012.00010" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.3389%2Ffncom.2012.00010" aria-label="Article reference 50" data-doi="10.3389/fncom.2012.00010">Article</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 50" href="http://scholar.google.com/scholar_lookup?&amp;title=Gait%20modulation%20in%20C.%20elegans%3A%20an%20integrated%20neuromechanical%20model&amp;journal=Front.%20Comput.%20Neurosci.&amp;doi=10.3389%2Ffncom.2012.00010&amp;volume=6&amp;publication_year=2012&amp;author=Boyle%2CJH&amp;author=Berri%2CS&amp;author=Cohen%2CN">
                    Google Scholar</a> 
                </p></li><li data-counter="51."><p id="ref-CR51">Izquierdo, E. J. &amp; Beer, R. D. From head to tail: a neuromechanical model of forward locomotion in <i>Caenorhabditis elegans</i>. <i>Philos. Trans. R. Soc. B</i> <b>373</b>, 20170374 (2018).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1098/rstb.2017.0374" data-track-item_id="10.1098/rstb.2017.0374" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1098%2Frstb.2017.0374" aria-label="Article reference 51" data-doi="10.1098/rstb.2017.0374">Article</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 51" href="http://scholar.google.com/scholar_lookup?&amp;title=From%20head%20to%20tail%3A%20a%20neuromechanical%20model%20of%20forward%20locomotion%20in%20Caenorhabditis%20elegans&amp;journal=Philos.%20Trans.%20R.%20Soc.%20B&amp;doi=10.1098%2Frstb.2017.0374&amp;volume=373&amp;publication_year=2018&amp;author=Izquierdo%2CEJ&amp;author=Beer%2CRD">
                    Google Scholar</a> 
                </p></li><li data-counter="52."><p id="ref-CR52">Wang, C. et al. A neurotransmitter atlas of the nervous system of <i>C. elegans</i> males and hermaphrodites. <i>eLife</i> <b>13</b>, RP95402 (2024).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.7554/eLife.95402.3" data-track-item_id="10.7554/eLife.95402.3" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.7554%2FeLife.95402.3" aria-label="Article reference 52" data-doi="10.7554/eLife.95402.3">Article</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 52" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20neurotransmitter%20atlas%20of%20the%20nervous%20system%20of%20C.%20elegans%20males%20and%20hermaphrodites&amp;journal=eLife&amp;doi=10.7554%2FeLife.95402.3&amp;volume=13&amp;publication_year=2024&amp;author=Wang%2CC">
                    Google Scholar</a> 
                </p></li><li data-counter="53."><p id="ref-CR53">Serrano-Saiz, E. et al. A neurotransmitter atlas of the <i>Caenorhabditis elegans</i> male nervous system reveals sexually dimorphic neurotransmitter usage. <i>Genetics</i> <b>206</b>, 1251–1269 (2017).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1534/genetics.117.202127" data-track-item_id="10.1534/genetics.117.202127" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1534%2Fgenetics.117.202127" aria-label="Article reference 53" data-doi="10.1534/genetics.117.202127">Article</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 53" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20neurotransmitter%20atlas%20of%20the%20Caenorhabditis%20elegans%20male%20nervous%20system%20reveals%20sexually%20dimorphic%20neurotransmitter%20usage&amp;journal=Genetics&amp;doi=10.1534%2Fgenetics.117.202127&amp;volume=206&amp;pages=1251-1269&amp;publication_year=2017&amp;author=Serrano-Saiz%2CE">
                    Google Scholar</a> 
                </p></li><li data-counter="54."><p id="ref-CR54">Horii, Y. et al. Physical reservoir computing in a soft swimming robot. In <i>ALIFE 2021: The 2021 Conference on Artificial Life</i> (MIT Press, 2021).</p></li><li data-counter="55."><p id="ref-CR55">Kwon, N., Pyo, J., Lee, S.-J. &amp; Je, J. H. 3-D worm tracker for freely moving <i>C. elegans</i>. <i>PLoS ONE</i> <b>8</b>, e57484 (2013).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1371/journal.pone.0057484" data-track-item_id="10.1371/journal.pone.0057484" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1371%2Fjournal.pone.0057484" aria-label="Article reference 55" data-doi="10.1371/journal.pone.0057484">Article</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 55" href="http://scholar.google.com/scholar_lookup?&amp;title=3-D%20worm%20tracker%20for%20freely%20moving%20C.%20elegans&amp;journal=PLoS%20ONE&amp;doi=10.1371%2Fjournal.pone.0057484&amp;volume=8&amp;publication_year=2013&amp;author=Kwon%2CN&amp;author=Pyo%2CJ&amp;author=Lee%2CS-J&amp;author=Je%2CJH">
                    Google Scholar</a> 
                </p></li><li data-counter="56."><p id="ref-CR56">Donato, A., Kagias, K., Zhang, Y. &amp; Hilliard, M. A. Neuronal sub-compartmentalization: a strategy to optimize neuronal function. <i>Biol. Rev.</i> <b>94</b>, 1023–1037 (2019).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1111/brv.12487" data-track-item_id="10.1111/brv.12487" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1111%2Fbrv.12487" aria-label="Article reference 56" data-doi="10.1111/brv.12487">Article</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 56" href="http://scholar.google.com/scholar_lookup?&amp;title=Neuronal%20sub-compartmentalization%3A%20a%20strategy%20to%20optimize%20neuronal%20function&amp;journal=Biol.%20Rev.&amp;doi=10.1111%2Fbrv.12487&amp;volume=94&amp;pages=1023-1037&amp;publication_year=2019&amp;author=Donato%2CA&amp;author=Kagias%2CK&amp;author=Zhang%2CY&amp;author=Hilliard%2CMA">
                    Google Scholar</a> 
                </p></li><li data-counter="57."><p id="ref-CR57">Bargmann, C. I. &amp; Marder, E. From the connectome to brain function. <i>Nat. Methods</i> <b>10</b>, 483–490 (2013).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/nmeth.2451" data-track-item_id="10.1038/nmeth.2451" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fnmeth.2451" aria-label="Article reference 57" data-doi="10.1038/nmeth.2451">Article</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 57" href="http://scholar.google.com/scholar_lookup?&amp;title=From%20the%20connectome%20to%20brain%20function&amp;journal=Nat.%20Methods&amp;doi=10.1038%2Fnmeth.2451&amp;volume=10&amp;pages=483-490&amp;publication_year=2013&amp;author=Bargmann%2CCI&amp;author=Marder%2CE">
                    Google Scholar</a> 
                </p></li><li data-counter="58."><p id="ref-CR58">Hodgkin, A. L. &amp; Huxley, A. F. A quantitative description of membrane current and its application to conduction and excitation in nerve. <i>J. Physiol.</i> <b>117</b>, 500–544 (1952).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1113/jphysiol.1952.sp004764" data-track-item_id="10.1113/jphysiol.1952.sp004764" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1113%2Fjphysiol.1952.sp004764" aria-label="Article reference 58" data-doi="10.1113/jphysiol.1952.sp004764">Article</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 58" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20quantitative%20description%20of%20membrane%20current%20and%20its%20application%20to%20conduction%20and%20excitation%20in%20nerve&amp;journal=J.%20Physiol.&amp;doi=10.1113%2Fjphysiol.1952.sp004764&amp;volume=117&amp;pages=500-544&amp;publication_year=1952&amp;author=Hodgkin%2CAL&amp;author=Huxley%2CAF">
                    Google Scholar</a> 
                </p></li><li data-counter="59."><p id="ref-CR59">Sheng, K. et al. Distilling multi-scale neural mechanisms from diverse unlabeled experimental data using deep domain-adaptive inference framework. Preprint at <i>bioRxiv</i> <a href="https://www.biorxiv.org/content/10.1101/2022.10.03.510694v2" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="https://www.biorxiv.org/content/10.1101/2022.10.03.510694v2">https://www.biorxiv.org/content/10.1101/2022.10.03.510694v2</a> (2023).</p></li><li data-counter="60."><p id="ref-CR60">Liu, Q., Hollopeter, G. &amp; Jorgensen, E. M. Graded synaptic transmission at the <i>Caenorhabditis elegans</i> neuromuscular junction. <i>Proc. Natl Acad. Sci. USA</i> <b>106</b>, 10823–10828 (2009).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1073/pnas.0903570106" data-track-item_id="10.1073/pnas.0903570106" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1073%2Fpnas.0903570106" aria-label="Article reference 60" data-doi="10.1073/pnas.0903570106">Article</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 60" href="http://scholar.google.com/scholar_lookup?&amp;title=Graded%20synaptic%20transmission%20at%20the%20Caenorhabditis%20elegans%20neuromuscular%20junction&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;doi=10.1073%2Fpnas.0903570106&amp;volume=106&amp;pages=10823-10828&amp;publication_year=2009&amp;author=Liu%2CQ&amp;author=Hollopeter%2CG&amp;author=Jorgensen%2CEM">
                    Google Scholar</a> 
                </p></li><li data-counter="61."><p id="ref-CR61">Shui, Y., Liu, P., Zhan, H., Chen, B. &amp; Wang, Z.-W. Molecular basis of junctional current rectification at an electrical synapse. <i>Sci. Adv.</i> <b>6</b>, eabb3076 (2020).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1126/sciadv.abb3076" data-track-item_id="10.1126/sciadv.abb3076" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1126%2Fsciadv.abb3076" aria-label="Article reference 61" data-doi="10.1126/sciadv.abb3076">Article</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 61" href="http://scholar.google.com/scholar_lookup?&amp;title=Molecular%20basis%20of%20junctional%20current%20rectification%20at%20an%20electrical%20synapse&amp;journal=Sci.%20Adv.&amp;doi=10.1126%2Fsciadv.abb3076&amp;volume=6&amp;publication_year=2020&amp;author=Shui%2CY&amp;author=Liu%2CP&amp;author=Zhan%2CH&amp;author=Chen%2CB&amp;author=Wang%2CZ-W">
                    Google Scholar</a> 
                </p></li><li data-counter="62."><p id="ref-CR62">Zhang, Y. et al. A GPU-based computational framework that bridges neuron simulation and artificial intelligence. <i>Nat. Commun.</i> <b>14</b>, 5798 (2023).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41467-023-41553-7" data-track-item_id="10.1038/s41467-023-41553-7" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41467-023-41553-7" aria-label="Article reference 62" data-doi="10.1038/s41467-023-41553-7">Article</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 62" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20GPU-based%20computational%20framework%20that%20bridges%20neuron%20simulation%20and%20artificial%20intelligence&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-023-41553-7&amp;volume=14&amp;publication_year=2023&amp;author=Zhang%2CY">
                    Google Scholar</a> 
                </p></li><li data-counter="63."><p id="ref-CR63">Hines, M. L. &amp; Carnevale, N. T. The NEURON simulation environment. <i>Neural Comput.</i> <b>9</b>, 1179–1209 (1997).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1162/neco.1997.9.6.1179" data-track-item_id="10.1162/neco.1997.9.6.1179" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1162%2Fneco.1997.9.6.1179" aria-label="Article reference 63" data-doi="10.1162/neco.1997.9.6.1179">Article</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 63" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20NEURON%20simulation%20environment&amp;journal=Neural%20Comput.&amp;doi=10.1162%2Fneco.1997.9.6.1179&amp;volume=9&amp;pages=1179-1209&amp;publication_year=1997&amp;author=Hines%2CML&amp;author=Carnevale%2CNT">
                    Google Scholar</a> 
                </p></li><li data-counter="64."><p id="ref-CR64">Palyanov, A., Khayrulin, S. &amp; Larson, S. D. Three-dimensional simulation of the <i>Caenorhabditis elegans</i> body and muscle cells in liquid and gel environments for behavioural analysis. <i>Philos. Trans. R. Soc. B</i> <b>373</b>, 20170376 (2018).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1098/rstb.2017.0376" data-track-item_id="10.1098/rstb.2017.0376" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1098%2Frstb.2017.0376" aria-label="Article reference 64" data-doi="10.1098/rstb.2017.0376">Article</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 64" href="http://scholar.google.com/scholar_lookup?&amp;title=Three-dimensional%20simulation%20of%20the%20Caenorhabditis%20elegans%20body%20and%20muscle%20cells%20in%20liquid%20and%20gel%20environments%20for%20behavioural%20analysis&amp;journal=Philos.%20Trans.%20R.%20Soc.%20B&amp;doi=10.1098%2Frstb.2017.0376&amp;volume=373&amp;publication_year=2018&amp;author=Palyanov%2CA&amp;author=Khayrulin%2CS&amp;author=Larson%2CSD">
                    Google Scholar</a> 
                </p></li><li data-counter="65."><p id="ref-CR65">Hu, Y. et al. Tetrahedral meshing in the wild. <i>ACM Trans. Graph</i>. <b>37</b>, 1–14 (2018).</p></li><li data-counter="66."><p id="ref-CR66">Bouaziz, S., Martin, S., Liu, T., Kavan, L. &amp; Pauly, M. Projective dynamics: fusing constraint projections for fast simulation. <i>ACM Trans. Graph.</i> <b>33</b>, 1–11 (2014).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1145/2601097.2601116" data-track-item_id="10.1145/2601097.2601116" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1145%2F2601097.2601116" aria-label="Article reference 66" data-doi="10.1145/2601097.2601116">Article</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 66" href="http://scholar.google.com/scholar_lookup?&amp;title=Projective%20dynamics%3A%20fusing%20constraint%20projections%20for%20fast%20simulation&amp;journal=ACM%20Trans.%20Graph.&amp;doi=10.1145%2F2601097.2601116&amp;volume=33&amp;pages=1-11&amp;publication_year=2014&amp;author=Bouaziz%2CS&amp;author=Martin%2CS&amp;author=Liu%2CT&amp;author=Kavan%2CL&amp;author=Pauly%2CM">
                    Google Scholar</a> 
                </p></li><li data-counter="67."><p id="ref-CR67">Javer, A. et al. An open-source platform for analyzing and sharing worm-behavior data. <i>Nat. Methods</i> <b>15</b>, 645–646 (2018).</p><p><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41592-018-0112-1" data-track-item_id="10.1038/s41592-018-0112-1" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41592-018-0112-1" aria-label="Article reference 67" data-doi="10.1038/s41592-018-0112-1">Article</a> 
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 67" href="http://scholar.google.com/scholar_lookup?&amp;title=An%20open-source%20platform%20for%20analyzing%20and%20sharing%20worm-behavior%20data&amp;journal=Nat.%20Methods&amp;doi=10.1038%2Fs41592-018-0112-1&amp;volume=15&amp;pages=645-646&amp;publication_year=2018&amp;author=Javer%2CA">
                    Google Scholar</a> 
                </p></li><li data-counter="68."><p id="ref-CR68">Mengdi, Z. et al. Data for an integrative data-driven model simulating <i>C. elegans</i> brain, body and environment interactions. <i>Zenodo</i> <a href="https://doi.org/10.5281/zenodo.13951773" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.5281/zenodo.13951773">https://doi.org/10.5281/zenodo.13951773</a> (2024).</p></li><li data-counter="69."><p id="ref-CR69">Mengdi, Z. et al. An integrative data-driven model simulating <i>C. elegans</i> brain, body and environment interactions. <i>Zenodo</i> <a href="https://doi.org/10.5281/zenodo.13943857" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.5281/zenodo.13943857">https://doi.org/10.5281/zenodo.13943857</a> (2024).</p></li></ol><p><a data-track="click" data-track-action="download citation references" data-track-label="link" rel="nofollow" href="https://citation-needed.springer.com/v2/references/10.1038/s43588-024-00738-w?format=refman&amp;flavour=references">Download references<svg width="16" height="16" focusable="false" role="img" aria-hidden="true"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-download-medium"></use></svg></a></p></div></div></div></section></div><section data-title="Acknowledgements"><div id="Ack1-section"><h2 id="Ack1">Acknowledgements</h2><p>This research was supported by the by Beijing Academy of Artificial Intelligence (BAAI). We thank X. Wu and B. Lei from BAAI for discussions. We thank Y. Wang from BAAI for writing the Application Programming Interface (API) of closed-loop interaction.</p></div></section><section aria-labelledby="author-information" data-title="Author information"><div id="author-information-section"><h2 id="author-information">Author information</h2><div id="author-information-content"><p><span id="author-notes">Author notes</span></p><ol><li id="na1"><p>These authors contributed equally: Mengdi Zhao, Ning Wang.</p></li></ol><h3 id="affiliations">Authors and Affiliations</h3><ol><li id="Aff1"><p>Beijing Academy of Artificial Intelligence, Beijing, China</p><p>Mengdi Zhao, Ning Wang, Xinrui Jiang, Xiaoyang Ma, Haixin Ma, Gan He, Kai Du, Lei Ma &amp; Tiejun Huang</p></li><li id="Aff2"><p>National Key Laboratory for Multimedia Information Processing, School of Computer Science, Peking University, Beijing, China</p><p>Gan He, Lei Ma &amp; Tiejun Huang</p></li><li id="Aff3"><p>Institute for Artificial Intelligence, Peking University, Beijing, China</p><p>Kai Du, Lei Ma &amp; Tiejun Huang</p></li><li id="Aff4"><p>National Biomedical Imaging Center, College of Future Technology, Peking University, Beijing, China</p><p>Lei Ma</p></li></ol><h3 id="contributions">Contributions</h3><p>M.Z. developed the neural network model module, designed and conducted experiments, and interpreted results. N.W. developed the body–environment module. H.M. and L.M. developed the 3D neural visualization software. X.J. and M.Z. collected experimental data for constructing the neural network model. X.M. and M.Z. optimized the connection algorithm and rewrote the code of the neural network model module for open source. G.H. developed the optimization algorithm for the multicompartment neural network model. M.Z., N.W., X.J. and L.M. wrote the Resource. T.H. and K.D. reviewed the research and provided advice. L.M. and T.H. supervised the research, coordinated and conceptualized the study, and contributed to writing the Resource. All authors participated in the paper revision process.</p><h3 id="corresponding-author">Corresponding author</h3><p id="corresponding-author-list">Correspondence to
                <a id="corresp-c1" href="mailto:lei.ma@pku.edu.cn">Lei Ma</a>.</p></div></div></section><section data-title="Ethics declarations"><div id="ethics-section"><h2 id="ethics">Ethics declarations</h2><div id="ethics-content">
              
                <h3 id="FPar5">Competing interests</h3>
                <p>The authors declare no competing interests.</p>
              
            </div></div></section><section data-title="Peer review"><div id="peer-review-section"><h2 id="peer-review">Peer review</h2><div id="peer-review-content">
              
              
                <h3 id="FPar4">Peer review information</h3>
                <p><i>Nature Computational Science</i> thanks the anonymous reviewer(s) for their contribution to the peer review of this work. Primary Handling Editor: Ananya Rastogi, in collaboration with the <i>Nature Computational Science</i> team.<a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s43588-024-00738-w#MOESM3"> Peer reviewer reports</a> are available.</p>
              
            </div></div></section><section data-title="Additional information"><div id="additional-information-section"><h2 id="additional-information">Additional information</h2><p><b>Publisher’s note</b> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div></section><section data-title="Supplementary information"><div id="Sec30-section"><h2 id="Sec30">Supplementary information</h2></div></section><section data-title="Source data"></section><section data-title="Rights and permissions"><div id="rightslink-section"><h2 id="rightslink">Rights and permissions</h2><div id="rightslink-content">
                <p><b>Open Access</b>  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="http://creativecommons.org/licenses/by-nc-nd/4.0/" rel="license">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
              <p><a data-track="click" data-track-action="view rights and permissions" data-track-label="link" href="https://s100.copyright.com/AppDispatchServlet?title=An%20integrative%20data-driven%20model%20simulating%20C.%20elegans%20brain%2C%20body%20and%20environment%20interactions&amp;author=Mengdi%20Zhao%20et%20al&amp;contentID=10.1038%2Fs43588-024-00738-w&amp;copyright=The%20Author%28s%29&amp;publication=2662-8457&amp;publicationDate=2024-12-16&amp;publisherName=SpringerNature&amp;orderBeanReset=true&amp;oa=CC%20BY-NC-ND">Reprints and permissions</a></p></div></div></section><section aria-labelledby="article-info" data-title="About this article"><div id="article-info-section"><h2 id="article-info">About this article</h2><div id="article-info-content"><div><p><a data-crossmark="10.1038/s43588-024-00738-w" target="_blank" rel="noopener" href="https://crossmark.crossref.org/dialog/?doi=10.1038/s43588-024-00738-w" data-track="click" data-track-action="Click Crossmark" data-track-label="link" data-test="crossmark"><img loading="lazy" width="57" height="81" alt="Check for updates. Verify currency and authenticity via CrossMark" src="data:image/svg+xml;base64,<svg height="81" width="57" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="m17.35 35.45 21.3-14.2v-17.03h-21.3" fill="#989898"/><path d="m38.65 35.45-21.3-14.2v-17.03h21.3" fill="#747474"/><path d="m28 .5c-12.98 0-23.5 10.52-23.5 23.5s10.52 23.5 23.5 23.5 23.5-10.52 23.5-23.5c0-6.23-2.48-12.21-6.88-16.62-4.41-4.4-10.39-6.88-16.62-6.88zm0 41.25c-9.8 0-17.75-7.95-17.75-17.75s7.95-17.75 17.75-17.75 17.75 7.95 17.75 17.75c0 4.71-1.87 9.22-5.2 12.55s-7.84 5.2-12.55 5.2z" fill="#535353"/><path d="m41 36c-5.81 6.23-15.23 7.45-22.43 2.9-7.21-4.55-10.16-13.57-7.03-21.5l-4.92-3.11c-4.95 10.7-1.19 23.42 8.78 29.71 9.97 6.3 23.07 4.22 30.6-4.86z" fill="#9c9c9c"/><path d="m.2 58.45c0-.75.11-1.42.33-2.01s.52-1.09.91-1.5c.38-.41.83-.73 1.34-.94.51-.22 1.06-.32 1.65-.32.56 0 1.06.11 1.51.35.44.23.81.5 1.1.81l-.91 1.01c-.24-.24-.49-.42-.75-.56-.27-.13-.58-.2-.93-.2-.39 0-.73.08-1.05.23-.31.16-.58.37-.81.66-.23.28-.41.63-.53 1.04-.13.41-.19.88-.19 1.39 0 1.04.23 1.86.68 2.46.45.59 1.06.88 1.84.88.41 0 .77-.07 1.07-.23s.59-.39.85-.68l.91 1c-.38.43-.8.76-1.28.99-.47.22-1 .34-1.58.34-.59 0-1.13-.1-1.64-.31-.5-.2-.94-.51-1.31-.91-.38-.4-.67-.9-.88-1.48-.22-.59-.33-1.26-.33-2.02zm8.4-5.33h1.61v2.54l-.05 1.33c.29-.27.61-.51.96-.72s.76-.31 1.24-.31c.73 0 1.27.23 1.61.71.33.47.5 1.14.5 2.02v4.31h-1.61v-4.1c0-.57-.08-.97-.25-1.21-.17-.23-.45-.35-.83-.35-.3 0-.56.08-.79.22-.23.15-.49.36-.78.64v4.8h-1.61zm7.37 6.45c0-.56.09-1.06.26-1.51.18-.45.42-.83.71-1.14.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.36c.07.62.29 1.1.65 1.44.36.33.82.5 1.38.5.29 0 .57-.04.83-.13s.51-.21.76-.37l.55 1.01c-.33.21-.69.39-1.09.53-.41.14-.83.21-1.26.21-.48 0-.92-.08-1.34-.25-.41-.16-.76-.4-1.07-.7-.31-.31-.55-.69-.72-1.13-.18-.44-.26-.95-.26-1.52zm4.6-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.07.45-.31.29-.5.73-.58 1.3zm2.5.62c0-.57.09-1.08.28-1.53.18-.44.43-.82.75-1.13s.69-.54 1.1-.71c.42-.16.85-.24 1.31-.24.45 0 .84.08 1.17.23s.61.34.85.57l-.77 1.02c-.19-.16-.38-.28-.56-.37-.19-.09-.39-.14-.61-.14-.56 0-1.01.21-1.35.63-.35.41-.52.97-.52 1.67 0 .69.17 1.24.51 1.66.34.41.78.62 1.32.62.28 0 .54-.06.78-.17.24-.12.45-.26.64-.42l.67 1.03c-.33.29-.69.51-1.08.65-.39.15-.78.23-1.18.23-.46 0-.9-.08-1.31-.24-.4-.16-.75-.39-1.05-.7s-.53-.69-.7-1.13c-.17-.45-.25-.96-.25-1.53zm6.91-6.45h1.58v6.17h.05l2.54-3.16h1.77l-2.35 2.8 2.59 4.07h-1.75l-1.77-2.98-1.08 1.23v1.75h-1.58zm13.69 1.27c-.25-.11-.5-.17-.75-.17-.58 0-.87.39-.87 1.16v.75h1.34v1.27h-1.34v5.6h-1.61v-5.6h-.92v-1.2l.92-.07v-.72c0-.35.04-.68.13-.98.08-.31.21-.57.4-.79s.42-.39.71-.51c.28-.12.63-.18 1.04-.18.24 0 .48.02.69.07.22.05.41.1.57.17zm.48 5.18c0-.57.09-1.08.27-1.53.17-.44.41-.82.72-1.13.3-.31.65-.54 1.04-.71.39-.16.8-.24 1.23-.24s.84.08 1.24.24c.4.17.74.4 1.04.71s.54.69.72 1.13c.19.45.28.96.28 1.53s-.09 1.08-.28 1.53c-.18.44-.42.82-.72 1.13s-.64.54-1.04.7-.81.24-1.24.24-.84-.08-1.23-.24-.74-.39-1.04-.7c-.31-.31-.55-.69-.72-1.13-.18-.45-.27-.96-.27-1.53zm1.65 0c0 .69.14 1.24.43 1.66.28.41.68.62 1.18.62.51 0 .9-.21 1.19-.62.29-.42.44-.97.44-1.66 0-.7-.15-1.26-.44-1.67-.29-.42-.68-.63-1.19-.63-.5 0-.9.21-1.18.63-.29.41-.43.97-.43 1.67zm6.48-3.44h1.33l.12 1.21h.05c.24-.44.54-.79.88-1.02.35-.24.7-.36 1.07-.36.32 0 .59.05.78.14l-.28 1.4-.33-.09c-.11-.01-.23-.02-.38-.02-.27 0-.56.1-.86.31s-.55.58-.77 1.1v4.2h-1.61zm-47.87 15h1.61v4.1c0 .57.08.97.25 1.2.17.24.44.35.81.35.3 0 .57-.07.8-.22.22-.15.47-.39.73-.73v-4.7h1.61v6.87h-1.32l-.12-1.01h-.04c-.3.36-.63.64-.98.86-.35.21-.76.32-1.24.32-.73 0-1.27-.24-1.61-.71-.33-.47-.5-1.14-.5-2.02zm9.46 7.43v2.16h-1.61v-9.59h1.33l.12.72h.05c.29-.24.61-.45.97-.63.35-.17.72-.26 1.1-.26.43 0 .81.08 1.15.24.33.17.61.4.84.71.24.31.41.68.53 1.11.13.42.19.91.19 1.44 0 .59-.09 1.11-.25 1.57-.16.47-.38.85-.65 1.16-.27.32-.58.56-.94.73-.35.16-.72.25-1.1.25-.3 0-.6-.07-.9-.2s-.59-.31-.87-.56zm0-2.3c.26.22.5.37.73.45.24.09.46.13.66.13.46 0 .84-.2 1.15-.6.31-.39.46-.98.46-1.77 0-.69-.12-1.22-.35-1.61-.23-.38-.61-.57-1.13-.57-.49 0-.99.26-1.52.77zm5.87-1.69c0-.56.08-1.06.25-1.51.16-.45.37-.83.65-1.14.27-.3.58-.54.93-.71s.71-.25 1.08-.25c.39 0 .73.07 1 .2.27.14.54.32.81.55l-.06-1.1v-2.49h1.61v9.88h-1.33l-.11-.74h-.06c-.25.25-.54.46-.88.64-.33.18-.69.27-1.06.27-.87 0-1.56-.32-2.07-.95s-.76-1.51-.76-2.65zm1.67-.01c0 .74.13 1.31.4 1.7.26.38.65.58 1.15.58.51 0 .99-.26 1.44-.77v-3.21c-.24-.21-.48-.36-.7-.45-.23-.08-.46-.12-.7-.12-.45 0-.82.19-1.13.59-.31.39-.46.95-.46 1.68zm6.35 1.59c0-.73.32-1.3.97-1.71.64-.4 1.67-.68 3.08-.84 0-.17-.02-.34-.07-.51-.05-.16-.12-.3-.22-.43s-.22-.22-.38-.3c-.15-.06-.34-.1-.58-.1-.34 0-.68.07-1 .2s-.63.29-.93.47l-.59-1.08c.39-.24.81-.45 1.28-.63.47-.17.99-.26 1.54-.26.86 0 1.51.25 1.93.76s.63 1.25.63 2.21v4.07h-1.32l-.12-.76h-.05c-.3.27-.63.48-.98.66s-.73.27-1.14.27c-.61 0-1.1-.19-1.48-.56-.38-.36-.57-.85-.57-1.46zm1.57-.12c0 .3.09.53.27.67.19.14.42.21.71.21.28 0 .54-.07.77-.2s.48-.31.73-.56v-1.54c-.47.06-.86.13-1.18.23-.31.09-.57.19-.76.31s-.33.25-.41.4c-.09.15-.13.31-.13.48zm6.29-3.63h-.98v-1.2l1.06-.07.2-1.88h1.34v1.88h1.75v1.27h-1.75v3.28c0 .8.32 1.2.97 1.2.12 0 .24-.01.37-.04.12-.03.24-.07.34-.11l.28 1.19c-.19.06-.4.12-.64.17-.23.05-.49.08-.76.08-.4 0-.74-.06-1.02-.18-.27-.13-.49-.3-.67-.52-.17-.21-.3-.48-.37-.78-.08-.3-.12-.64-.12-1.01zm4.36 2.17c0-.56.09-1.06.27-1.51s.41-.83.71-1.14c.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.37c.08.62.29 1.1.65 1.44.36.33.82.5 1.38.5.3 0 .58-.04.84-.13.25-.09.51-.21.76-.37l.54 1.01c-.32.21-.69.39-1.09.53s-.82.21-1.26.21c-.47 0-.92-.08-1.33-.25-.41-.16-.77-.4-1.08-.7-.3-.31-.54-.69-.72-1.13-.17-.44-.26-.95-.26-1.52zm4.61-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.08.45-.31.29-.5.73-.57 1.3zm3.01 2.23c.31.24.61.43.92.57.3.13.63.2.98.2.38 0 .65-.08.83-.23s.27-.35.27-.6c0-.14-.05-.26-.13-.37-.08-.1-.2-.2-.34-.28-.14-.09-.29-.16-.47-.23l-.53-.22c-.23-.09-.46-.18-.69-.3-.23-.11-.44-.24-.62-.4s-.33-.35-.45-.55c-.12-.21-.18-.46-.18-.75 0-.61.23-1.1.68-1.49.44-.38 1.06-.57 1.83-.57.48 0 .91.08 1.29.25s.71.36.99.57l-.74.98c-.24-.17-.49-.32-.73-.42-.25-.11-.51-.16-.78-.16-.35 0-.6.07-.76.21-.17.15-.25.33-.25.54 0 .14.04.26.12.36s.18.18.31.26c.14.07.29.14.46.21l.54.19c.23.09.47.18.7.29s.44.24.64.4c.19.16.34.35.46.58.11.23.17.5.17.82 0 .3-.06.58-.17.83-.12.26-.29.48-.51.68-.23.19-.51.34-.84.45-.34.11-.72.17-1.15.17-.48 0-.95-.09-1.41-.27-.46-.19-.86-.41-1.2-.68z" fill="#535353"/></g></svg>"/></a></p><div><h3 id="citeas">Cite this article</h3><p>Zhao, M., Wang, N., Jiang, X. <i>et al.</i> An integrative data-driven model simulating <i>C. elegans</i> brain, body and environment interactions.
                    <i>Nat Comput Sci</i> <b>4</b>, 978–990 (2024). https://doi.org/10.1038/s43588-024-00738-w</p><p><a data-test="citation-link" data-track="click" data-track-action="download article citation" data-track-label="link" data-track-external="" rel="nofollow" href="https://citation-needed.springer.com/v2/references/10.1038/s43588-024-00738-w?format=refman&amp;flavour=citation">Download citation<svg width="16" height="16" focusable="false" role="img" aria-hidden="true"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-download-medium"></use></svg></a></p><ul data-test="publication-history"><li><p>Received<span>: </span><span><time datetime="2024-03-10">10 March 2024</time></span></p></li><li><p>Accepted<span>: </span><span><time datetime="2024-11-05">05 November 2024</time></span></p></li><li><p>Published<span>: </span><span><time datetime="2024-12-16">16 December 2024</time></span></p></li><li><p>Issue Date<span>: </span><span><time datetime="2024-12">December 2024</time></span></p></li><li><p><abbr title="Digital Object Identifier">DOI</abbr><span>: </span><span>https://doi.org/10.1038/s43588-024-00738-w</span></p></li></ul></div></div></div></div></section>
            </div></div>
  </body>
</html>
