<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <title>James Routley | Feed</title>
    <link
      rel="stylesheet"
      type="text/css"
      href="../styles.css"
      media="screen"
    />
  </head>
  <body>
    <a href="/index.html">Back</a>
    <a href="https://blogs.nvidia.com/blog/2022/01/20/living-cell-simulation/">Original</a>
    <h1>Simulation of a 2B-atom cell that metabolizes and grows like a living cell</h1>
    
    <div id="readability-page-1" class="page"><div id="bsf_rt_marker"><div data-url="https://blogs.nvidia.com/blog/2022/01/20/living-cell-simulation/" data-title="NVIDIA GPUs Enable Simulation of a Living Cell" data-hashtags=""><p>Every living cell contains its own bustling microcosm, with thousands of components responsible for energy production, protein building, gene transcription and more.</p>
<p>Scientists at the University of Illinois at Urbana-Champaign have built a 3D simulation that replicates these physical and chemical characteristics at a particle scale — creating a fully dynamic model that mimics the behavior of a living cell.</p>
<p><a href="https://doi.org/10.1016/j.cell.2021.12.025" target="_blank" rel="noopener">Published in the journal <i>Cell</i></a><i>, </i>the project simulates a living minimal cell, which contains a pared-down set of genes essential for the cell’s survival, function and replication. The model uses NVIDIA GPUs to simulate 7,000 genetic information processes over a 20-minute span of the cell cycle – making it what the scientists believe is the longest, most complex cell simulation to date.</p>
<p>Minimal cells are simpler than naturally occurring ones, making them easier to recreate digitally.</p>
<p>“Even a minimal cell requires 2 billion atoms,” said Zaida Luthey-Schulten, chemistry professor and co-director of the university’s Center for the Physics of Living Cells. “You cannot do a 3D model like this in a realistic human time scale without GPUs.”</p>
<p>Once further tested and refined, whole-cell models can help scientists predict how changes to the conditions or genomes of real-world cells will affect their function. But even at this stage, minimal cell simulation can give scientists insight into the physical and chemical processes that form the foundation of living cells.</p>
<p>“What we found is that fundamental behaviors emerge from the simulated cell — not because we programmed them in, but because we had the kinetic parameters and lipid mechanisms correct in our model,” she said.</p>
<p><a href="https://ngc.nvidia.com/catalog/containers/hpc:lattice-microbes">Lattice Microbes</a>, the GPU-accelerated software co-developed by Luthey-Schulten and used to simulate the 3D minimal cell, is available on the <a href="https://ngc.nvidia.com/">NVIDIA NGC</a> software hub.</p>
<h2><b>Minimal Cell With Maximum Realism</b></h2>
<p>To build the living cell model, the Illinois researchers simulated one of the simplest living cells, a parasitic bacteria called mycoplasma. They based the model on a trimmed-down version of a mycoplasma cell synthesized by scientists at J. Craig Venter Institute in La Jolla, Calif., which had just under 500 genes to keep it viable.</p>
<p>For comparison, a single E. coli cell has around 5,000 genes. A human cell has more than 20,000.</p>
<p>Luthy-Schulten’s team then used known properties of the mycoplasma’s inner workings, including amino acids, nucleotides, lipids and small molecule metabolites to build out the model with DNA, RNA, proteins and membranes.</p>
<p>“We had enough of the reactions that we could reproduce everything known,” she said.</p>
<p>Using Lattice Microbes software on <a href="https://www.nvidia.com/en-us/data-center/tensor-cores/">NVIDIA Tensor Core GPUs</a>, the researchers ran a 20-minute 3D simulation of the cell’s life cycle, before it starts to substantially expand or replicate its DNA. The model showed that the cell dedicated most of its energy to transporting molecules across the cell membrane, which fits its profile as a parasitic cell.</p>
<p>“If you did these calculations serially, or at an all-atom level, it’d take years,” said graduate student and paper lead author Zane Thornburg. “But because they’re all independent processes, we could bring parallelization into the code and make use of GPUs.”</p>
<p>Thornburg is working on another GPU-accelerated project to simulate growth and cell division in 3D. The team has recently adopted <a href="https://www.nvidia.com/en-us/data-center/dgx-systems/">NVIDIA DGX systems</a> and <a href="https://www.nvidia.com/en-us/design-visualization/rtx-a5000/">RTX A5000 GPUs</a> to further accelerate its work, and found that using A5000 GPUs sped up the benchmark simulation time by 40 percent compared to a development workstation with a previous-generation NVIDIA GPU.</p>
<p>Learn more about researchers using NVIDIA GPUs to accelerate science breakthroughs by <a href="https://reg.rainfocus.com/flow/nvidia/gtcspring2022/registration">registering free for NVIDIA GTC</a>, running online March 21-24.</p>
<p><i>Main image is a snapshot from the 20-minute 3D spatial simulation, showing yellow and purple ribosomes, red and blue degradasomes, and smaller spheres representing DNA polymers and proteins. </i></p>
</div></div></div>
  </body>
</html>
