<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <title>James Routley | Feed</title>
    <link
      rel="stylesheet"
      type="text/css"
      href="../styles.css"
      media="screen"
    />
  </head>
  <body>
    <a href="/index.html">Back</a>
    <a href="https://markusstrasser.org/extracting-knowledge-from-literature/">Original</a>
    <h1>The business of extracting knowledge from academic publications</h1>
    
    <div id="readability-page-1" class="page"><div>
        <header>
            
        </header>
            <p>

         <time datetime="2021-07-12T10:37:00-05:00">
         December 7, 2021.
         </time>Send your thoughts via

         <a target="_blank" href="https://twitter.com/mkstra">
         twitter</a><iconify-icon data-icon="ei-sc-twitter" data-height="50" data-inline="false"></iconify-icon>

          or <a target="_blank" href="mailto:strasser.ms@gmail.com">
           mail</a><iconify-icon data-icon="clarity:email-solid" data-height="50" data-inline="false"></iconify-icon>
        </p>
        <section itemprop="text">

            <!--kg-card-begin: html-->
<p>
<em>I ment to post this much earlier but have been fighting with neurological longhaul covid symptoms for most of the year. </em>
</p><!--kg-card-end: html--><!--kg-card-begin: html--><!--kg-card-end: html--><!--kg-card-begin: html-->

<div>
    <p> <b> TL;DR: I worked on biomedical literature search, discovery and recommender web applications for many months and concluded that extracting, structuring or synthesizing &#34;insights&#34; from academic publications (papers) or building knowledge bases from a domain corpus of literature has negligible value in industry.</b> </p>
    
<p><b> Close to nothing of what makes science actually work is published as text on the web </b></p></div><!--kg-card-end: html--><figure><img src="https://markusstrasser.org/content/images/2021/11/Screen-Shot-2021-11-17-at-1.14.07-PM.png" alt="Multiple images of UIs that serve up semantic knowledge products" loading="lazy" width="1338" height="938"/><figcaption>UIs serving up “insights” from machine-reading millions of papers through specialized search engines, paper recommenders, citation analysis, correlation plots of mentions, hierarchical clusterings of topics, interaction networks (relation extraction), causation graphs [...]</figcaption></figure><p>Here’s the outline:</p><!--kg-card-begin: html-->
<!--kg-card-end: html--><h2 id="psychoanalysis-of-a-troubled-industry">Psychoanalysis of a Troubled Industry </h2><p>Atop the published biomedical literature is an evolved industry  around the extracting, semantic structuring and synthesizing of research papers into search, discovery and knowledge graph software applications (<em><a href="https://markusstrasser.org/companies-knowledge-discovery/">table of example companies</a></em>). The usual sales pitch goes something like this:</p><blockquote><em>Generate and Validate Drug Discovery Hypotheses Faster Using our Knowledge Graph</em></blockquote><blockquote><em>Find the relevant knowledge for your next breakthrough in the X million documents and Y million extracted causal interactions between genes, chemicals, drugs, cells… </em></blockquote><blockquote><em>Our insight prediction pipeline and dynamic knowledge map is like a digital scientist </em>[<a href="https://www.occamzrazor.com/work#our-approach">OccamzRazor</a>]</blockquote><blockquote><em>Try our sentence-level, context-aware, and linguistically informed extractive search system</em> [AllenAI’s <a href="https://spike.apps.allenai.org/datasets">Spike</a>]</blockquote><p>Or from a grant application of yours truly:</p><blockquote><em>a high-level semantic search engine and user interface for answering complex, quantitative research questions in biomedicine </em></blockquote><p>You get the idea. All the projects above spring from similar premises:</p><ol><li>The right piece of information is “out there”</li><li>If only research outputs were more machine interpretable, searchable and discoverable then “research” could be incrementally automated and progress would go through the roof</li><li>No mainstream public platform <em>(eg. arxiv, pubmed, google scholar)</em> provides features that leverage the last decade’s advances in natural language processing (<em>NLP</em>) and they do not improve on tasks mentioned in <strong>2.</strong></li></ol><p>Which leads them to conclude: Let’s <a href="https://www.ncbi.nlm.nih.gov/research/pubtator/?view=docsum&amp;query=ESR1%20breast%20cancer">semantically annotate every piece of literature</a> and incorporate it into a structured knowledge base<em> </em>that enables complex research queries, revealing <a href="https://www.aaai.org/Papers/KDD/1996/KDD96-051.pdf">undiscovered public knowledge</a> by connecting causal chains (<em>swanson linking</em>) or generating<a href="https://en.wikipedia.org/wiki/Literature-based_discovery"> new hypotheses</a> altogether.</p><figure><img src="https://lh3.googleusercontent.com/m-hN9_t7x-GAVMQ6k7qideWVKf1LHBWNjUatBI6C6JacZFwarqlBxLadHMqF6qf9q_5Qnl88tMG5_OKBrsHqasDnchrBO8TnkUsHbN2eny6EKHBVNUWt4IhnBUMhu35N8YNU9R8L" alt="" loading="lazy"/><figcaption>Two sentences semantically with multiple layers of semantic annotation</figcaption></figure><figure><img src="https://lh6.googleusercontent.com/qFFjxQaLZodS5Lau0Zof_YEUvgJp4Q3XKBUyHPmceJMvn0pq1JrCo-wlF6CvIQiWZ0LtG7_uh43YTFxqD1ihK2qoynyre-fkx9BhWVeSzoUiMhK0qTrELxOyOSmqdUN1D5Gv-3ab" alt="" loading="lazy"/><figcaption>An example of a generic biomedical entity-relation schema that might power a knowledge base on top</figcaption></figure><p>It really does sound exciting and VCs and governments like to fund the promise of it. Companies get started, often get a few pivot customers, end up as software consultancies, or silently fail. In <a href="https://www.scibite.com/news/elsevier-acquires-scibite-to-accelerate-solutions-for-life-sciences-and-corporate-rd-industries/">rare cases they get acquired</a> for cheap or survive as a modest “<strong>request a demo” </strong>B2B SaaS with 70% of the company in sales with unit economics that lock them in to only sell to enterprises and never to individuals <sup>0</sup>.</p><p>Meanwhile big players with all the means to provide a category leading platform have either stopped developing such an effort altogether (<em>Google Scholar</em>), shut it down and nobody cares (ie. <a href="https://www.microsoft.com/en-us/research/project/academic/articles/microsoft-academic-to-expand-horizons-with-community-driven-approach/">Microsoft Academic Search</a>) or are unable to commercialize any of it (<em>AllenAI with SemanticScholar, Spike etc.</em>).</p><p>Let’s take <a href="https://allenai.org/">AllenAI</a> as an example. AllenAI is a world-renowned AI research organization with a focus on NLP and automated reasoning. They sponsor a startup incubator to spin-off and commercialize their technologies. The <a href="https://allennlp.org/">AllenNLP platform</a> and <a href="https://www.semanticscholar.org/">SemanticScholar</a> is used by millions of engineers and researchers. And unlike most big AI research groups they also build domain-focused apps for biomedicine (<em>like <a href="https://supp.ai/">supp.ai</a> and <a href="https://spike.apps.allenai.org/datasets/pubmed/search">Spike</a></em>).</p><p>And yet, despite all of that momentum, they can’t commercialize any of it:</p><blockquote><em>We have been looking at opportunities to commercialize the technologies we developed in<a href="https://www.semanticscholar.org/"> Semantic Scholar</a>,<a href="https://spike.pubmed.apps.allenai.org/"> Spike</a>, <a href="https://en.wikipedia.org/wiki/Biomedical_text_mining">bioNLP</a>, etc. The challenge for us so far is in finding significant market demand. I&#39;m curious if you have found an angle on this ? [...]</em></blockquote><p>This was an email reply I received from a technical director at the AllenAI institute. At that point, I had already spent months building and trying to sell biomedical NLP applications (<a href="https://en.wikipedia.org/wiki/Biomedical_text_mining">bioNLP</a>) to biotechs which had made me cynical about the entire space. Sure, I might just be bad at sales but if the AllenAI with 10/10 engineering, distribution and industry credibility can’t sell their offerings, then <strong>literature review, search and knowledge discovery are just not that useful…</strong></p><p>When I then looked at <strong>the SUM of all combined funding, valuations, exits of _NLP_ companies in the semantic search / biomedical literature search / biomedical relation extraction space I could find it was <u>less than</u> the valuation of any single bigger _bioinformatic_ drug discovery platform</strong> (<em> ie. </em><a href="https://atomwise.com/"><em>Atomwise</em></a><em>, </em><a href="https://insitro.com/"><em>Insitro</em></a>, <a href="https://nference.com"><em>nFerence</em></a>). Assuming the market is efficient and not lagging, it shows how little of a problem biomedical literature search and discovery actually is. </p><p>Just to clarify: <strong>This post is about the issues with semantic intelligence platforms that predominantly leverage the published academic literature.</strong> Bioinformatic knowledge apps that integrate biological <a href="https://en.wikipedia.org/wiki/Omics">omics</a> data or clinical datasets are actually very valuable (ie. <a href="https://data4cure.com/solutions.html">Data4cure</a>), as is information extraction from Electronic Health Records [<a href="https://en.wikipedia.org/wiki/Electronic_health_record">EHRs</a>] (<em>like <a href="https://nference.com/">nFerence</a>, <a href="https://www.science.io/">scienceIO</a></em>) and patent databases.</p><p>Back in March 2020 when the first covid lockdowns started I evacuated San Francisco and moved back to Austria. I decided to use the time to get a deeper understanding of molecular biology and began reading textbooks.</p><figure><img src="https://markusstrasser.org/content/images/2021/11/image.png" alt="" loading="lazy" width="1600" height="901"/></figure><p>Before biology, I had worked on text mining, natural language processing and knowledge graphs and it got me thinking ... <strong>could you build something that can reason at the level of a mediocre biology graduate?</strong></p><p>After a few weeks of reading papers in ontology learning, information extraction and reasoning systems and experimenting with toy programs, I had some idea of what&#39;s technologically possible. I figured that a domain-focused search engine would be much simpler to build than a reasoner/chatbot and that the basic building blocks are similar in any case <sup>0.5</sup>.</p><p>I wrote up what I had in mind, applied to <a href="https://www.mercatus.org/emergent-ventures">Emergent Ventures</a> and was given a grant to continue working on the idea.</p><p>At that time I also moved to London as an <a href="https://joinef.com/">Entrepreneur First </a>(EF) fellow. I made good friends in the program and teamed up with<a href="https://rmeinl.com/"> Rico</a>. We worked together for the entirety of the three month program. During that time we prototyped:</p><ul><li><strong>a search engine with entity, relation and quantitative options</strong>: A user could go detailed, expressive queries like:</li><li><strong>a query builder (by demonstration):</strong> A user would upvote study abstracts or sentences that fit their requirements and it would iteratively build configuration. In a sense it was a no-code way for biologists to write interpretable labeling functions while also training a classifier (<em>a constrained case of </em><a href="https://www.microsoft.com/en-us/research/publication/program-synthesis/"><em>program synthesis</em></a>)</li><li><strong>A paper recommender system</strong> (<em>and later claim/sentence/statement level</em>) that, in addition to academic literature, included tweets, company websites and other non-scholarly sources (<em>lots of scientific discourse is happening on Twitter these days</em>)</li><li><strong>A claim explorer</strong> that takes a sentence/paragraph and lets you browse through similar claims</li><li><strong>A claim verifier</strong> that showed sentences that confirm or contradict an entered claim sentence</li><li>And a few more<a href="https://en.wikipedia.org/wiki/Wizard_of_Oz_experiment"> Wizard of Oz experiments</a> to test different variants and combinations of the features above</li></ul><p>At that point, lots of postdocs had told us that some of the apps would have saved them months during their PhD, but actual viable customers were only moderately excited. It became clear that this would have to turn into another B2B SaaS specialized tool with a lot of software consulting and ongoing efforts from a sales team ...</p><p>We definitely did not want to go down that route. We wanted to make a consumer product for consumers, startups, academics or independent researchers and had tested good-enough proxies for most ideas we thought of as useful w.r.t. the biomedical literature. We also knew that pivoting to Electronic Health Records (<em>EHRs</em>) or patents instead of research text was potentially a great business, but neither of us was excited to spend years working on that, even if successful.</p><p>So we were stuck. Rico, who didn&#39;t have exposure to biotech before we teamed up, understandably wanted to branch out and so we decided to build a discovery tool that we could use and evaluate the merits of ourselves.</p><p>And so we loosened up and spent three weeks playing around with <em>knowledge search and discovery</em> outside of biomed. We built:</p><ul><li>a <strong>browser extension</strong> that finds similar paragraphs (<em>in articles/essays from your browser history</em>) to the text you&#39;re currently highlighting (<em>when reading on the web</em>)</li><li><strong>similar</strong>: <strong>an app that pops up serendipitous connections</strong> between a corpus (<em>previous writings, saved articles, bookmarks ...</em>) and the active writing session or paragraph. The corpus, preferably your own, could be from folders, text files, blog archive, a Roam Research graph or a Notion/Evernote database. </li><li><strong>a</strong> semantic <strong>podcast search</strong> (<em>one</em><a href="https://share.streamlit.io/rmeinl/podcast_search/app.py"><em> version for the Artificial Intelligence Podcast is still live</em></a>)</li><li>an extension that after a few days of development was turning into<a href="https://ampie.app/"> Ampie</a> and was shelved</li><li>an extension that after a few days of development was turning into<a href="https://twemex.app/"> Twemex</a> and was shelved</li></ul><p>Some of the above had promise for journalists, VCs, essayists or people that read, cite and tweet all day, but that group is too heterogeneous and fragmented and we couldn&#39;t trust our intuition building for them.</p><p>To us, <strong>these experiments</strong> <strong>felt mostly like gimmicks</strong> that, with more love and industry focus <sup>1.5</sup>, could become useful but probably not essential.</p><p>Now it was almost Christmas and the EF program was over. Rico and I decided to split up as a team because we didn&#39;t have any clear next step in mind <sup>1</sup>. I flew to the south of Portugal for a few weeks to escape London&#39;s food, bad weather and the upcoming covid surge.</p><p>There, I returned to biomed and tinkered with interactive, extractive search interfaces and no-code<a href="https://arxiv.org/abs/1605.07723"> data programming</a> UIs (<em>users can design labeling functions to bootstrap datasets without coding expertise</em>)</p><p>Ironically, I got covid in Portugal and developed scary neurological long-hauler symptoms after the acute infection. Except for a handful of ‘good days&#39;, spring and summer came and went without me being able to do meaningful cognitive work. Fortunately, this month symptoms have improved enough to finish this essay.</p><h2 id="fundamental-issues-with-structuring-academic-literature-as-a-business"><strong>Fundamental Issues with Structuring Academic Literature as a Business</strong></h2><p>As I said in the beginning:</p><blockquote><em>extracting, structuring or synthesizing &#34;insights&#34; from academic publications (papers) or building knowledge bases from a domain corpus of literature has negligible value in industry</em></blockquote><p>To me the reasons feel elusive and trite simultaneously. All are blatantly obvious in hindsight.</p><h3 id="just-a-paper-an-idea-an-insight-does-not-get-you-innovation"><strong>Just a Paper, an Idea, an Insight Does Not Get You Innovation</strong></h3><p><strong>Systems, teams and researchers matter much more than ideas. <sup>2</sup> </strong></p><p>It wasn’t always like that. In the 19th century ideas (<em>inventions</em>) were actually the main mechanism for innovation. From<a href="https://benjaminreinhardt.com/notes-on-the-changing-structure-of-american-innovation"> Notes on The changing structure of American innovation</a>:</p><blockquote><em>​​The period from 1850-1900 could be described as the age of the inventor. During this period inventors were the main mechanism for creating innovations. These folks would usually sell these patents directly to large companies like the telegraph operators, railroads, or large chemical companies. The companies themselves did little R&amp;D and primarily operated labs to test the patents that inventors brought them</em></blockquote><p>But the complexity threshold kept rising and now we need to grow companies around inventions to actually make them happen:</p><blockquote><em>DuPont bought the patent for creating viscose rayon (processed cellulose used for creating artificial silk and other fibers.) However, Dupont was unable to replicate the process successfully and eventually had to partner with the original inventors to get it to work</em></blockquote><p>That’s why incumbents increasingly <a href="https://en.wikipedia.org/wiki/Acqui-hiring"><em>acqui-hire</em></a> instead of just buying the IP and most successful companies that spin out of labs have someone who did the research as a cofounder.<strong> </strong>Technological utopians and ideologists <em>like my former self</em> underrate how important context and tacit knowledge is. </p><p>And even if you had that context and were perfectly set up to make use of new literature ... a significant share of actionable, relevant research findings aren’t published when they’re hot but after the authors milk them and the datasets for a sequence of derivative publications or patent them before publishing the accompanying paper (<em>half the time behind paywalls</em>) months later<strong>.</strong></p><p><strong>In the market of ideas information asymmetries turn into knowledge monopolies.</strong> </p><p>As mentioned in <a href="https://milan.cvitkovic.net/writing/market_failures_in_science/">Market Failures in Science</a>:</p><blockquote>Scientists are incentivized to, and often do, <strong>withhold as much information as possible</strong> about their innovations in their publications to maintain a monopoly over future innovations. This slows the overall progress of science.</blockquote><p>All that is to say: <strong>discovering relevant literature, compiling evidence, finding mechanisms turns out to be a tiny percentage of actual, real life R&amp;D</strong></p><h3 id="contextual-tacit-knowledge-is-not-digital-not-encoded-or-just-not-machine-interpretable-yet">Contextual, Tacit Knowledge is not Digital, not Encoded or just not Machine-Interpretable yet</h3><p>Most knowledge necessary to make scientific progress is not online and not encoded. All tools on top of that bias the exploration towards encoded knowledge only (<a href="https://en.wikipedia.org/wiki/Streetlight_effect">drunkard&#39;s search</a>), which is vanishingly small compared to what scientists need to embody to make progress. </p><p>Expert and crowd-curated world knowledge graphs (<em>eg. <a href="https://conceptnet.io">ConceptNet</a></em>) can partially ground a machine learning model with some context and common sense but that is light years away from an expert’s ability to understand implicature and weigh studies and claims appropriately. </p><p>ML systems are fine for pattern matching, recommendations and generating variants but in the end great meta-research, including literature reviews, comes down to formulating an incisive research question, selecting the right studies and defining adequate evaluation criteria (<em>metrics</em>) <sup>2.5</sup>.</p><p>Besides, accurately and programmatically transforming an entire piece of literature into a computer-interpretable, complete and actionable knowledge artifact remains a pipe dream.</p><figure><img src="https://lh3.googleusercontent.com/QiBbp7SWQ8gIcItL81Gg8f3TxF8vv26hbyHMBfcmpDl4zW4ts2HZA7-2lSChIxdv7c6pcqjlDIYCDCt3jKlo4OHpTK1brJZDb94szTLpb1NwZmeqHWclmjmQvZ1B9LILYvN8ICC7" alt="" loading="lazy"/><figcaption>Automatically generating complex schemas like <a href="https://www.micropublication.org/">micropubs</a> (above) from text is many years away. Micropubs (2014), <a href="https://nanopub.org/">nanopubs</a>, <a href="https://biological-expression-language.github.io/">BEL</a> (<em>Biological Expression Language</em>) and many more are all dead in my view (<em>without admitting it</em>)</figcaption></figure><h3 id="experts-have-well-defined-internalized-maps-of-their-field"><strong>Experts have well defined, internalized maps of their field</strong></h3><p>In industry, open literature reviews are a sporadic, non-standardized task. Professionals spend years building internal schemas of a domain and have social networks for sourcing (<em>ie. discovery</em>) and vetting relevant information if needed. That&#39;s why the most excited of our initial user cohorts were graduate and PhD students and life science VCs, not professionals who specialized and actively work in biotech or pharma companies.</p><p>The VC use case of doing due diligence was predominantly around legal and IP concerns rather than anything that a literature review might produce. The conceptual clearance or feedback on the idea itself was done by sending it to a referred expert in their network, not by searching the literature for contradictions or feasibility.</p><h3 id="scientific-publishing-comes-with-signaling-status-games-fraud-and-often-very-little-information"><strong>Scientific Publishing comes with Signaling, Status Games, Fraud and often Very Little Information</strong></h3><p>Many published papers have methodical or statistical errors, are derivative and don&#39;t add anything to the discourse, are misleading or obfuscated, sometimes even fraudulent or were just bad research to begin with. <strong>Papers are first and foremost career instruments. </strong></p><figure><img src="https://markusstrasser.org/content/images/2021/11/image-2.png" alt="" loading="lazy" width="500" height="500"/></figure><p>A naive system will weigh the insights extracted from a useless publication equally to a seminal work. You can correct for that by normalizing on citations and other tricks but that will just mimic and propagate existing biases and issues with current scientific publishing.</p><p>For example, if I design a system that mimics the practices of an expert reader, it would result in biases towards institutions, countries, the spelling of names, clout of authors and so on. That’s either unfair, biased and unequal or it is efficient, resourceful and a reasonable response based on the reader&#39;s priors. In the end, there is no technological solution. </p><p>To push this point: if you believe in the <a href="https://en.wikipedia.org/wiki/Great_man_theory">great man theory</a> of scientific progress, which has more merit than most want to admit, then <strong>why waste time making the other 99%+ of &#34;unimportant&#34; publications more searchable and discoverable</strong>? The &#34;greats&#34; (<em>of your niche</em>) will show up in your feed anyway, right? Shouldn&#39;t you just follow the best institutions and ~50 top individuals and be done with your research feed?</p><p>Well in fact that&#39;s what most people do.</p><figure><img src="https://markusstrasser.org/content/images/2021/11/image-1.png" alt="" loading="lazy" width="778" height="253"/><figcaption>From <a href="http://www.denisdutton.com/bad_writing.htm">Bad Writing</a></figcaption></figure><h3 id="non-technical-life-science-labor-is-cheap"><strong>Non-technical Life Science Labor Is Cheap</strong></h3><p>Why purchase access to a 3rd party AI reading engine or a knowledge graph when you can just hire hundreds of postdocs in Hyderabad to parse papers into JSON? (<em>at a $6,000 yearly salary</em>)</p><p>Would you invest in automation if you have billions of disposable income and access to cheap labor? After talking with employees of huge companies like GSK, AZ and <a href="https://www.medscape.com/">Medscape</a> the answer is a clear no.</p><p>Life science grads work for cheap, even at the post-grad level. In the UK a good bioinformatician can make 2-4 times of what non-technical lab technicians or early career biologists make. In the US the gap is even larger. The Oxford Chemistry and Biology postdocs I met during bus rides to the science park (<em>from my time at Oxford Nanopore</em>) earned £35k at AstraZeneca <sup>3</sup>. That&#39;s half of what someone slightly competent earns after four months of youtubing Javascript tutorials 🤷‍♂️.</p><p>When we gave our pilot customers (<em>biologists</em>) spreadsheets with relations, mentions and paper recommendations that fit their exact requirements they were quite happy and said it saved them many hours per week. <strong>But it wasn’t a burning pain for them since their hourly wage is low either way.</strong></p><p>I suspect the inconsequential costs of lab labor is a reason why computer aided biology (CAB) and lab automation, including cloud labs, are slow on the uptake …it can’t just be the clogging of liquid handling robots, right?</p><h3 id="the-literature-is-implicitly-reified-in-public-structured-knowledge-bases"><strong>The Literature is Implicitly Reified in Public Structured Knowledge Bases</strong></h3><p>There are a ton of <a href="https://markusstrasser.org/biomedical-datasets/">biomedical knowledge bases, databases</a>, ontologies that are updated regularly <sup>4</sup>. They are high signal because groups of experts curate and denoise them. </p><p>They&#39;re tremendously useful and will eventually make biomedical intelligent systems and reasoners easier to build. Bayer, AZ, GSK all integrate them into their production knowledge graphs and their existence makes any additional commercial attempts to extract relations from the literature less needed.</p><h3 id="advanced-interactives-and-visualizations-are-surprisingly-unsatisfying-to-consume"><strong>Advanced Interactives and Visualizations are Surprisingly Unsatisfying to Consume</strong></h3><p>As an artist this was the most painful lesson: Interactive graphs, trees, cluster visualizations, dendrograms, causal diagrams and what have you are much less satisfying than just lists, and most often lists will do.</p><p>On the other hand <strong>figures, plots and tables are at least as valuable as the actual text content </strong>of a paper but programs can&#39;t intelligibly process and extract their contents (<em>too much implied context</em>).</p><p><strong>That alone cuts the value of tools based on extracting insights from text in half </strong><em>(even with perfect extraction)</em><strong>.</strong></p><p>I realized that when I sat next to a pro &#34;reading&#34; a paper. It goes like this:</p><blockquote><em>Scan abstract. Read last sentence of introduction. Scan figures. Check results that discuss the figures</em>....</blockquote><p>which was entirely different from my non-expert approach to reading papers. It makes sense: If you don&#39;t have a strong domain model, you have no map that guides your scan, so you have to read it all top to bottom … like a computer.</p><figure><div><div><p><img src="https://markusstrasser.org/content/images/2021/11/Screen-Shot-2021-11-10-at-2.09.00-PM.png" width="1436" height="1188" loading="lazy" alt=""/></p><p><img src="https://markusstrasser.org/content/images/2021/11/Screen-Shot-2021-11-10-at-2.04.54-PM.png" width="1276" height="1318" loading="lazy" alt=""/></p><p><img src="https://markusstrasser.org/content/images/2021/11/Screen-Shot-2021-11-10-at-2.04.33-PM.png" width="1080" height="844" loading="lazy" alt=""/></p></div><div><p><img src="https://markusstrasser.org/content/images/2021/11/Screen-Shot-2021-11-10-at-1.54.39-PM-2.png" width="1248" height="1092" loading="lazy" alt=""/></p><p><img src="https://markusstrasser.org/content/images/2021/11/Screen-Shot-2021-11-16-at-2.59.29-PM.png" width="1594" height="800" loading="lazy" alt=""/></p></div></div><figcaption>Is this actually useful to experts?</figcaption></figure><p>This skip-and-scan selective processing also explains why agglomerative, auto-generated and compiled visualizations that incorporate a large corpus of papers are not that valuable: most of the sources would’ve been discarded up front. </p><p>Research contexts are so multiplicitous that every compiled report or visualization has huge amounts of noise and no user interface ever can be expressive enough to perfectly parameterize that context.</p><h3 id="unlike-other-business-software-domain-knowledge-bases-of-research-companies-are-maximally-idiosyncratic"><strong>Unlike other Business Software, Domain Knowledge Bases of Research Companies are Maximally Idiosyncratic</strong></h3><p>Unlike other SaaS products that have standardized components (<em>auth, storage, messaging, ...</em>), research knowledge graphs are always designed around the domain and approach of a company and are tightly integrated with their infrastructure, proprietary datasets and IP. </p><p><strong>Two enterprises can encode the same corpus and will produce starkly different knowledge base artifacts. </strong></p><p>Pharmas all have their own custom knowledge graphs and have entire engineering teams working full time on keeping schemas consistent (<a href="https://www.youtube.com/watch?v=-N2NNVVPULM">AstraZeneca and their setup</a>). </p><p><strong>Incompatibilities are ontological, not technological.</strong> Knowledge representations are design decisions. That&#39;s why 3rd party knowledge graph vendors almost always have to do multi-month ongoing integration work and software consulting to make the sale. “<em>Request a demo</em>” often translates to “<em>hire us for a bespoke data science project</em>”.</p><p>To put knowledge base interoperability issues into perspective: even small database merges within the same company are still a huge problem in the software industry with no real, verifiable loss-free solution yet. Database incompatibilities have left many database administrators traumatized and knowledge bases have orders of magnitude higher <em>schematic complexity than databases</em>.</p><h3 id="divergent-tasks-are-hard-to-evaluate-and-reason-about"><strong>Divergent Tasks are Hard to Evaluate and Reason About</strong></h3><p>By “divergent” I mean loosely defined tasks where it&#39;s unclear when they&#39;re done. That includes &#34;mapping a domain&#34;, &#34;gathering evidence&#34;, &#34;due diligence&#34; and generally anything without a clear outcome like &#34;book a hotel&#34;, &#34;find a barber&#34;, &#34;run this assay&#34;, &#34;order new vials&#34;...</p><p><strong>It’s easy to reason about convergent tasks</strong>, like getting an answer to a factual question or checking a stock price. It’s hard to reason and quantify a divergent discovery process, like getting a map of a field or exploring a space of questions. You can&#39;t put a price tag on exploration and so it is set very low by default. Companies employ “researchers” not for reading literature, but for lab or coding work and the prestige of the “researcher” title pushes salaries down even further.</p><p>For example, take a product that offers corpus analytics:</p><p>What’s the value of getting a topic hierarchy, causal diagram or paper recommendations? It has some value if the user does not already have an operational model in his head but how often is that the case? Most biotech research jobs require a PhD level schema from the get go, so what’s the added value to a noisy AI-generated one?</p><p><strong>When the components  (</strong><em>claims, descriptions, papers</em><strong>) and tasks (</strong><em>evidence review, due diligence, mapping</em><strong>) are ambiguous it is tough to reason about returns on investment with clients. This continues inside the company: employees can defend hours in the lab much better than hours “researching”.</strong></p><p><strong>We’re also naturally good at diverging and auto-association which makes an in silico version of that feature less valuable</strong>. It often ends up being more effort to parse what those interfaces return than to actually, simply think.</p><p>Besides, most biotechs (<em>before Series C</em>) don’t have the infrastructure to try out new ideas faster than they can read them. <em>A week in the lab can save you a day in the library</em> <sup>6</sup></p><h2 id="public-penance-my-mistakes-biases-and-self-deceptions"><strong>Public Penance: My Mistakes, Biases and Self-Deceptions </strong></h2><p>Yes, I was raised catholic. How did you know?</p><ol><li>My biggest mistake was that <strong>I didn&#39;t have experience as a biotech researcher or postdoc working in a lab</strong>. Sometimes being an outsider is an advantage, but in a field full of smart, creative people the majority of remaining inefficiencies are likely to come from incentives, politics and culture and not bad tooling.</li><li><strong>I used to be friends with someone exceptional who started a biomedical cause-effect search engine</strong> and it biased me towards thinking about this category of products. Also, I had met employees at<a href="https://www.scibite.com/"> SciBite</a>, a semantics company that was acquired by Elsevier for £65M, and was so unimpressed by the talent there that I was convinced that there’s lots left to do.</li><li>I wanted to make this my thing, my speciality. <strong>I had all the skills for making augmented interfaces for scientific text</strong>: machine learning, UI design and fine arts, frontend development, text mining and knowledge representation...</li><li>I had a good idea of the business model issues after a few weeks of talking to users, but <strong>I didn&#39;t want to stop without a first-principles explanation</strong> of why that is. Sunk-cost fallacy and a feeling of fiduciary responsibility to Emergent Ventures played a part too, but mainly because I wanted to know decisively why semantic structuring, academic search, discovery and publishing are such <strong>low-innovation zones</strong>. Looking back, I should’ve been OK with 80% certainty that it’s a lost cause and moved on.</li><li>I wanted to work on these tools because I could work with scientists and contribute something <strong>meaningful without needing to start from scratch teaching myself biology, bioinformatics and actually working in a lab</strong>. I started from nothing too many times in my life and had gotten impatient. I wanted to leverage what I already knew. <strong>But there aren’t real shortcuts </strong>and impatience made me climb a local optima.<strong> </strong></li></ol><h2 id="onwards"><strong>Onwards</strong></h2><p>The initial not so modest proposal I sent to Emergent Ventures was along the lines of:</p><blockquote><em>a next-generation scientific search and discovery web interface that can answer complex quantitative questions, built on extracted entities and relations from scientific text, such as causations, effects, biomarkers, quantities, methods and so on</em></blockquote><p>I mentioned that</p><ul><li>there <a href="https://markusstrasser.org/hacking-scientific-text/">were major advances in relevant fields of machine learning</a></li><li>that current interfaces are impoverished</li><li>that an hour of searching could be collapsed into a minute in many cases</li></ul><p><strong>All of that is still true but for the reasons I tried to share in this essay nothing of it matters.</strong></p><p>I had to wrap my head around the fact that close to <strong>nothing of what makes science actually work is published as text on the web.</strong></p><p>Research questions that can be answered logically through just reading papers and connecting the dots don&#39;t require a biotech corp to be formed around them. There&#39;s much less logic and deduction happening than you&#39;d expect in a scientific discipline <sup>5</sup>.</p><p>It&#39;s obvious in retrospect and I likely persisted for too long. I had idealistic notions of how scientific search and knowledge synthesis &#34;should work&#34;.</p><p>I’ve been flirting with this entire cluster of ideas including open source <a href="https://markusstrasser.org/web-annotation-a-short-dive/">web annotation</a>, semantic search and semantic web, <a href="https://www.knowledgefutures.org/work">public knowledge graphs</a>, nano-publications, knowledge maps, <a href="https://www.underlay.org/">interoperable protocols</a> and <a href="https://schema.org/">structured data</a>, serendipitous discovery apps, knowledge organization, communal sense making and academic literature/publishing toolchains for a few years on and off ... <strong>nothing of it will go anywhere</strong>. </p><p><strong>Don’t take that as a challenge. Take it as a red flag and run. Run towards better problems.</strong></p><!--kg-card-begin: html--><hr size="5" noshade=""/><!--kg-card-end: html--><!--kg-card-begin: html--><!--kg-card-end: html-->

        </section>

                <!-- Begin Mailchimp Signup Form -->


    </div></div>
  </body>
</html>
