<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <title>James Routley | Feed</title>
    <link
      rel="stylesheet"
      type="text/css"
      href="../styles.css"
      media="screen"
    />
  </head>
  <body>
    <a href="/index.html">Back</a>
    <a href="https://jsomers.net/e-coli-chemotaxis/">Original</a>
    <h1>The baffling intelligence of a single cell: The story of E. coli chemotaxis</h1>
    
    <div id="readability-page-1" class="page"><article>
        <p>
          I want to tell the story of a beautiful phenomenon in biology. In some
          sense it’s the prototype of much of the activity of life. The
          phenomenon is the way in which an individual cell of
          <em>E. coli</em> forages for nutrients. This process, known as
          “chemotaxis”—the “chemo-” for chemical and the “taxis” from the Greek
          τάξις, for tactics—is intelligence in one of
          its most elemental forms. An individual <em>E. coli</em> has no brain,
          obviously, and is even many orders of magnitude simpler than a human
          cell, and yet already it possesses something like a sense of smell,
          drive, even a memory. Chemotaxis recasts <em>E. coli</em> not as some
          aimless gut-pest but rather as an exquisitely sophisticated
          <a href="https://yalebooks.yale.edu/book/9780300167849/wetware/#:~:text=This%20book%20offers%20a%20startling,is%20a%20form%20of%20computation.">physical computer</a>.
        </p>
        <p>
          I’m also telling this story because
          <a href="https://jsomers.net/i-should-have-loved-biology">I never liked the way biology was taught in high school</a>. It was too much about the names of things. A subject so vast is
          spoiled by a textbook, which can only point at the endless parade of
          stuff-there-is-to-know. It’s better approached with questions—like
          “what’s happening when you smell?” or ”what is a fever,
          actually?”—that contemplate narrow, deep slices.
        </p>

        <p>
          Chemotaxis is a great slice: it’s a triumph of systems biology—we
          understand it holistically but also in fine detail at almost every
          level. It acquaints you with many of the most important motifs in
          biology, including the way in which protein structure determines
          function; how membranes control the information flow into cells; and
          how chemical modifications store and communicate state. It involves
          one of the most sophisticated and beautiful pieces of molecular
          nanotechnology, the flagellar motor. And it helps give an intuition
          for how a bag of unthinking chemicals could possibly give rise to a
          being.
        </p>
        <h3>The 30,000-foot view</h3>
        <figure>
          
          <figcaption>
            Even with simple rules, the E. coli finds food more often than not
          </figcaption>
        </figure>
        <p>
          The basic idea is this: <em>E. coli</em> “smells” chemicals it’s
          attracted to with a set of nose-like receptors and decides how to
          swim. Depending on what it senses, it can either use its flagellar
          tails to swim forward—this is known as a “run”—or it can spin in a
          random direction (a “tumble”). By running when the getting is good and
          tumbling when it isn’t, the <em>E. coli</em> takes a meandering path
          toward the attractant.
        </p>
        <p>
          A little more detail now: there are half a dozen or so rotors on the
          <em>E. coli’s</em> body, each controlling a long whip-like tail that
          flows behind it. When all the rotors are spinning in the same
          direction, the tails join together into a coil that torques the cell
          forward into a run. When even one rotor is spinning against the
          others, the coil unbundles and <em>E. coli</em> spins into a tumble.
        </p>
        <p>
          In a uniform chemical environment, the <em>E. coli</em> swims in a
          random walk by balancing runs with periodic tumbles. By default, a run
          lasts about a second, or ten times longer than a tumble. The rate of
          runs versus tumbles, and their relative duration, is carefully tuned
          to balance “exploitation” and “exploration”: if runs happened too
          often or lasted too long, <em>E. coli</em> would range too widely and
          zip past its food; too seldom or too short, and it’d likely never find
          food in the first place.
        </p>
        <p>
          But how is this balance achieved? The crux of it is a signaling
          molecule called CheY (pronounced “KEY-why”). CheY is constantly
          bouncing around in the cytoplasm of the <em>E. coli</em>, interacting
          with both the receptor complex (the “nose”) and the rotors, carrying
          information between them. In the steady state, when CheY encounters
          the receptor complex, it gets chemically modified, or
          “phosphorylated,” at a certain rate to become CheY-p. Unlike the
          unmodified version, CheY-p has a strong affinity for the rotors, and
          when enough copies bind to one, it reverses its spin, causing a
          tumble.
        </p>

        <p>
          The trick is that when the nose detects an <em>increase</em> in the
          concentration of attractant, that steady turning of CheYs into CheY-ps
          is interrupted. As a result, fewer CheY-ps bind to the rotors; fewer
          reversals take place; and so the <em>E. coli</em> runs more and
          tumbles less. In other words, the all-important relative rate of runs
          versus tumbles is determined entirely by how often the
          CheY→CheY-p process churns—and this, in turn, is determined by
          how much attractant is detected by the nose.
        </p>
        <p>
          You can see this process in action in the interactive illustration.
          Try altering the ratio of CheY-p (white) to CheY (blue) by adding some
          attractant (pink). You‘ll end up inducing a long run.
        </p>
        
        <div>
          <p>Less attractant</p>
          <p>More attractant</p>
        </div>
        <p>
          Why do you need all this complexity? You could imagine a system in
          which the motors themselves responded directly to attractant. We’ll
          see later on that the stream of CheY-ps acts as a kind of adaptable,
          tunable chemical amplifier. “<a href="https://www.ks.uiuc.edu/Research/chemotaxis/#:~:text=In%20particular%2C%20bacterial,per%20cell%20volume!">Bacterial cells can amplify</a>
          signals more than 50-fold; that is to say, a 2% change in receptor
          occupancy can bring about a 100% change in the output of the system at
          the flagellar motors. This feature allows cells to sense minute
          changes in concentration—less than three molecules per cell volume!”
        </p>
        <h3>
          The story gets more complicated: adaptation
        </h3>
        <p>
          If the system were as described above, then
          <em>E. coli</em> wouldn’t have much dynamic range. Imagine: if the
          cell has a huge reaction to just three molecules of attractant,
          wouldn’t a thousand times as many just completely overwhelm it?
        </p>
        <p>
          In reality, the <em>E. coli</em> is able to respond sensitively across
          <a href="https://www.youtube.com/watch?t=58&amp;v=cT855rpX8bc&amp;feature=youtu.be">five orders of magnitude</a>
          of attractant concentration. The cell learns to treat whatever
          concentration it stumbles into as the new normal, so that the
          slightest increase triggers the same hypersensitive response as
          always.
        </p>
        <p>
          The mechanism powering this adaptation is extremely clever. You can
          think of each receptor as being equipped with “struts” that have
          pockets in them. When the receptor is bound to attractant, its struts
          change shape so that these pockets open up, and become the targets for
          little molecules known as methyl groups. Methyl groups are ubiquitous
          in biochemistry: for instance, they help determine which parts of your
          DNA get expressed. Methyl groups bind to the structural proteins your
          DNA strands coil around, called histones; the “methylated” histone can
          kink the DNA strand into or out of view of your transcription
          machinery, turning it on or off.
        </p>
        <p>
          In this case, methylation serves to fill up the strut’s pockets,
          causing it to become more rigid. (I’m simplifying the actual physical
          details somewhat, as we’ll see later.) With more rigid struts the
          receptor’s signaling power is dampened: it takes more attractant to
          elicit the same response. Because there are many methylation sites per
          strut and many struts per receptor, there’s a wide range of possible
          dampening values—as if those pockets were really the holes of an
          elaborate wind instrument.
          <span>[<a id="bray-maybe-referent" href="#bray-maybe">Bray</a>]</span>
          This wide dynamic range is what allows the bacteria not just to find a
          favorable environment but to keenly and speedily nose its way up a
          chemical gradient. No wonder a similar mechanism is used by cells in
          your immune system to track and hunt down invaders.
        </p>
        <p>
          Methylation of the receptors gives <em>E. coli</em> a “<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2899694/#:~:text=The%20receptor%20methylation%20level%20also%20provides%20a%20simple%20chemical%20memory%20used%20to%20ascertain%20whether%20the%20current%20direction%20of%20swimming%20is%20favorable%20or%20unfavorable.">simple chemical memory</a>.” This is a powerful and somewhat profound idea: individual bacteria
          can model their environment and remember important features of it by
          encoding that information in internal chemical modifications.
          <em>E. coli</em> “knows” whether attractant has become more or less
          concentrated in its surroundings going back
          <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2899694">several seconds</a>; that helps it determine whether it’s swimming in a good or bad
          direction. Which is not that different in principle from what brains
          do. In fact one reason that it requires an artificial neural network
          of about a
          <a href="https://www.sciencedirect.com/science/article/pii/S0896627321005018">thousand</a>
          elements just to model the computational capabilities of a single real
          neuron is that the real neuron stores so much “state” in its internal
          chemistry.
        </p>
        <p>
          (Here‘s an aside: should we be surprised at how resilient people can
          be, given the mechanisms available to a single cell for accepting
          previously extreme conditions as “a new normal”? No doubt our macro
          resilience is in some cases actually underwritten by similar cellular
          mechanisms.)
        </p>
        <h3>
          The full picture: a complex signaling network
        </h3>
        <figure>
          <iframe src="https://www.youtube.com/embed/LgPDOSou1tw?controls=0&amp;autoplay=0" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen=""></iframe>
        </figure>
        <p>
          The video above is a very legible overview of
          <em>E. coli</em> chemotaxis, from a popular textbook. It layers in
          even more detail, including not just the proteins that phosphorylate
          CheY but those that dephosphorylate it; and not just the proteins that
          methylate the receptors but those that demethylate it. What you come
          to see is that these doers and undoers define a sort of equilibrated
          circuit whose activity can be conveniently dialed up or down.
        </p>
        <p>
          Dennis Bray describes these sorts of circuits nicely in his book,
          <em>Wetware: A Computer in Every Living Cell</em>:
        </p>
        <blockquote>
          <p>
            In a typical signaling pathway, proteins are continually being
            modified and demodified. Kinases and phosphatases work ceaselessly
            like ants in a nest, adding phosphate groups to proteins and
            removing them again. It seems a pointless exercise, especially when
            you consider that each cycle of addition and removal costs the cell
            one molecule of ATP—one unit of precious energy. Indeed, cyclic
            reactions of this kind were originally labeled “futile.” But the
            adjective is misleading. The addition of phosphate groups to
            proteins is the single most common reaction in cells and underpins a
            large proportion of the computations they perform. Far from being
            futile, this cyclic reaction provides the cell with an essential
            resource: a flexible and rapidly tunable device.
          </p>
          <p>
            If the cell really needs to change the concentration of the modified
            protein very quickly, it can. All it has to do is to switch on or
            shut off the phosphate-adding reaction and the concentration will
            fall precipitously—at the speed of the spinning cycle. There is no
            buildup of products or depletion of substrates to slow down the
            process, as there would be in a linear chain of enzyme reactions.
          </p>
        </blockquote>
        <p>
          This is a clever way to regulate the level of some protein or
          metabolite. Rather than producing the thing you want via a lengthy
          chain reaction, you just have this running cycle that activates and
          then de-activates it, for example via phosphorylation and
          de-phosphorylation. When you want more of the active version, you just
          tamp down the de-activating reaction in the cycle, as if sliding down
          the volume on a stereo.
        </p>
        <p>
          Regulation in this manner via phosphorylation and dephosphorylation
          (by “kinases” and “phosphatases” respectively) is an extremely general
          feature of life. “About 30–50% of human proteins contain covalently
          attached phosphate. [. . .] A typical mammalian cell makes use of
          hundreds of distinct types of protein kinases at any moment.”
          <span>[<a id="alberts-referent" href="#alberts">Alberts</a>]</span>
        </p>
        <p>
          In the interactive figure above, phosphorylation is represented by the
          blue dots becoming white, and de-phosphorylation happens when they
          turn blue again. This cycle is constantly running. The speed of the
          cycle determines how quickly the cell can react to levels of
          attractant. Notice that when you add some, the blue→white
          reaction stops happening as much. But the blue←white reaction
          keeps going at the same rate. So blue CheY proliferates, and the cell
          runs more. (If the cycle spun more slowly, the blues wouldn‘t take
          over so quickly.)
        </p>
        <h3>Down the rabbit-hole…</h3>
        <p>
          One thing I don’t love in presentations of chemotaxis—and of
          biological concepts generally—is that they often prominently feature
          flowcharts and network diagrams. In the case of chemotaxis, as you can
          gather from the video above, there are many players with nearly
          indistinguishable names: CheA phosphorylates CheY to become CheY-p,
          and CheZ dephosphorylates it back to CheY; CheW couples CheA to the
          receptors, and CheR methylates those receptors’ struts; CheB,
          meanwhile, “clips off” the methyl groups added to the struts by CheR.
          A network diagram is no doubt useful for organizing this sea of names
          but in a sense it foregrounds the most abstract view of the process.
          I’d rather try to get a sense of the parts as a living whole or in
          their individual physical detail.
        </p>
        <p>When you do that, it’s amazing what you find.</p>
        <h3>
          What does it mean for a receptor to detect attractant?
        </h3>
        <p>
          Almost every action in a cell depends on proteins changing shape and
          binding to each other. It’s no different in the
          <em>E. coli</em> receptor complex.
        </p>
        <p>
          The way it works is that there are stimulus-specific proteins embedded
          in the <em>E. coli’s</em> cell membrane, protruding into what’s known
          as the periplasm. These proteins are “stimulus-specific” in the
          literal sense that they are shaped so as to bind favorably with
          individual molecules of attractant. <em>E. coli</em> has five or six
          of these, for instance one that detects a crucial amino acid called
          aspartate. This sensor protein has little clefts in it that are shaped
          just so for molecules of aspartate to fit snugly into them.
          <span>[<a id="falke-referent" href="#falke">Falke</a>]</span>
          In schematic form the aspartate receptor looks like this:
        </p>
        <figure>
          <img loading="lazy" width="2032" height="3280" alt="" src="https://jsomers.net/e-coli-chemotaxis/articleImages/transmembrane.jpg"/>
          <figcaption>
            Source: Falke,
            <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2899694/">“The Two-Component Signaling Pathway of Bacterial Chemotaxis: A
              Molecular View of Signal Transduction by Receptors, Kinases, and
              Adaptation Enzymes”</a>
          </figcaption>
        </figure>
        <p>
          You can see that the sensory part—up top, where the aspartate binds—is
          connected to the signaling proteins CheW and CheA by a columnar
          structure that straddles the cell’s membrane. What does this protein
          complex “actually” look like?
        </p>
        <p>
          An individual protein is small enough—<a href="http://book.bionumbers.org/how-big-is-the-average-protein/">like a few nanometers wide</a>—that it can’t really be seen through a regular light microscope.
          This receptor from top to bottom measures about 350 angstroms, or 35
          nanometers. But modern biology is all about seeing the unseeable.
          Nowadays, we try to find out what nanostructures look like by
          <a href="https://en.wikipedia.org/wiki/X-ray_crystallography">X-ray diffraction</a>
          or, more and more often, by
          <a href="https://en.wikipedia.org/wiki/Cryogenic_electron_microscopy">cryo-freezing them</a>
          in an electron microscope. Once we determine a protein’s structure
          it’s usually rendered using ribbon diagrams, a style
          <a href="https://stories.duke.edu/sciences-mother-of-ribbon-diagrams-celebrates-50-years-at-duke">invented by the biochemist Jane Richardson</a>
          in the late 1970s. Here’s a ribbon diagram for the
          <em>E. coli</em> serine receptor (really it’s a “trimer of dimers,” or
          a complex of six receptors):
        </p>
        <figure>
          <img loading="lazy" width="1079" height="1373" alt="" src="https://jsomers.net/e-coli-chemotaxis/articleImages/image10.png"/>
          <figcaption>
            Source:
            <a href="https://ckcassidycom.files.wordpress.com/2015/12/full_trimer.jpg">Keith Cassidy</a>
          </figcaption>
        </figure>
        <p>
          This whole thing is the receptor. (Those parts just inside the
          membrane, with little yellow methyl groups lingering stuck to them,
          are the struts.) How exactly it works is quite complex, and the
          subject of current research. But in simplified terms it acts like one
          big piston: when the asparate binds to the part in the periplasm, the
          columnar structure it’s attached to changes shape—a real biologist
          would call these subtle allosteric effects; to me it looks like
          dipping and tilting—in such a way to lock the thing that’s supposed to
          be phosphorylating CheY, the CheA kinase, into an inactive state.
        </p>
        <figure>
          <video loading="lazy" width="480" height="480" src="articleImages/CheA-conformational-change.mp4" autoplay="" muted="" loop="" playsinline=""></video>
          <figcaption>
            Adapted from:
            <a href="https://www.youtube.com/watch?v=MCobqYrE67w">CheA conformation change, posted by Keith Cassidy</a>
          </figcaption>
        </figure>
        <p>
          When I think of a cell I imagine a Rube Goldberg–type contraption
          where an arm swings here, which drops a ball into a slide there, which
          rolls down and opens a trap door, which… eventually turns on or off
          some important cellular function. Indeed, <em>E. coli</em>’s “sense of
          smell” rests ultimately in a series of physical lock-and-key
          mechanisms, starting with literal molecules of e.g. aspartate nuzzling
          into a protein and transmitting that physical shape-change across the
          membrane.
        </p>
        <p>
          This piston-shaped receptor complex is just one of a huge array
          arranged near the front of <em>E. coli</em>’s body. In cross section
          they appear almost to have been laid down through a lithography
          process, in a neat hexagonal pattern:
        </p>
        <figure>
          <img loading="lazy" width="1862" height="598" alt="" src="https://jsomers.net/e-coli-chemotaxis/articleImages/image2.png"/>
          <figcaption>
            Source:
            <a href="https://www.ks.uiuc.edu/Research/chemotaxis/">Theoretical and Computational Biophysics Group, UIUC</a>
          </figcaption>
        </figure>
        <figure>
          <img loading="lazy" width="1116" height="762" alt="" src="https://jsomers.net/e-coli-chemotaxis/articleImages/image13.png"/>
          <figcaption>
            Source:
            <a href="https://www.pnas.org/doi/10.1073/pnas.0610106104">Direct visualization of Escherichia coli chemotaxis receptor
              arrays using cryo-electron microscopy</a>
          </figcaption>
        </figure>
        <p>
          Calling <em>E. coli’s</em> receptor complex its “nose” is no mere
          metaphor. Our own noses operate on a similar principle: when you smell
          a flower, it means that actual flower-molecules—possibly only a tiny
          number of them—have reached the inside of your nose and bound to some
          protein with a specific affinity for that very molecule. This signal
          is then transmitted via nerves to your brain. The human nose has
          several hundred receptor proteins for smell; a dog has
          <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5884888/#:~:text=The%20canine%20OR%20repertoire%20is%20composed%20of%201%2C094%20genes%2C%20approximately%20three%20times%20more%20than%20a%20human.">more than a thousand</a>.
        </p>

        <p>
          Every one of our senses works like this. Touch is underwritten by
          proteins that get “squished” by tactile forces into cell membranes,
          triggering a set of downstream responses. Sight is my favorite
          example. There’s a protein called opsin that lives in the cells of our
          retina. What’s so cool about it is that the thing that changes its
          shape is a literal photon. That is,
          <a href="https://pdb101.rcsb.org/motm/147#:~:text=Rhodopsin%2C%20shown%20here,to%20the%20brain.">opsin converts the electromagnetic force of an incoming photon into
            a biomechanical / biochemical signal</a>. This is why I tend to think of molecular biology as the science of
          shapes bumping into each other.
        </p>
        <p>
          I think of <em>E. coli’s</em> receptor complex as a protoversion of
          our own sensory apparatus. Its nose has only
          <a href="https://www.youtube.com/watch?t=2596&amp;v=cT855rpX8bc&amp;feature=youtu.be">five or six</a>
          attractant-specific sensory proteins, but their signals are
          integrated, as if different sets of receptor-protein activations were
          playing different “chords” on the
          <em>E. coli’s</em> sensorium. “In short, the chemosensory array
          functions as an ultrasensitive, ultrastable biological integrated
          circuit or sensory chip.”
          <span>[<a id="falke-referent-2" href="#falke-2">Falke</a>]</span>
        </p>
        <h3>How the signal is carried</h3>
        <p>
          So a bit of attractant binds one of the receptors, and lo, the
          equilibrium inside the cell begins to shift. Because the CheA kinase
          is now inactive, CheYs are no longer getting phosphorylated as
          quickly; the process that <em>de</em>-phosphorylates existing CheY-ps
          starts winning out. Recall that this is a response that is dynamic, a
          flow that is tuned. The net number of CheY-ps in the cell is carefully
          faded down. And then what?
        </p>
        <p>
          The CheY-ps had been binding to the flagellar rotors, flipping them,
          causing tumbles. That now no longer happens as much, because the
          unphosphorylated CheY doesn’t have the same affinity for the rotor as
          CheY-p. As a result, the cell tumbles less, runs more, and biases its
          random walk toward the attractant.
        </p>
        <p>
          There’s something really important worth dwelling on here. When we say
          that CheY-p has an “affinity” for the rotor protein, it’s not like it
          gets directed there; nor does it have some long-acting magnetic
          attraction for it. What this really means is that it has a strong
          inclination to bind to the rotor protein when it gets really really
          close to it. (And CheY, without the -p, doesn&#39;t have such an
          inclination.) Given how small a single CheY-p is in the scheme of the
          whole cell’s cytoplasm, it might seem improbable that it’ll somehow
          sidle up right next to one of these rotor proteins somewhere on the
          other end of the <em>E. coli’s</em> body. But that gets at the heart
          of
          <a href="https://www.newyorker.com/magazine/2022/03/07/a-journey-to-the-center-of-our-cells">the crazy kinetic chaos inside our cells</a>.
        </p>
        <figure>
          <img loading="lazy" width="1272" height="832" alt="" src="https://jsomers.net/e-coli-chemotaxis/articleImages/image19.png"/>
          <img loading="lazy" width="1376" height="1829" alt="" src="https://jsomers.net/e-coli-chemotaxis/articleImages/image7.jpg"/>
          <figcaption>
            Source: David Goodsell, <em>The Machinery of Life</em>
          </figcaption>
        </figure>
        <p>
          Cells are dense with stuff, but everything in it is also
          <em>extremely</em> fast-moving:
        </p>
        <blockquote>
          <p>
            To get an idea of how fast this motion is, imagine a typical
            bacterial cell, and place an enzyme at one end and a sugar molecule
            at the other. They will bump around and wander through the whole
            cell, encountering many molecules along the way. On average, though,
            it will only take about a second for those two molecules to bump
            into each other at least once. This is truly remarkable: this means
            that any molecule in a typical bacterial cell, during its chaotic
            journey through the cell, will encounter almost every other molecule
            in a matter of seconds.
            <span>[<a id="goodsell-referent" href="#goodsell">Goodsell</a>]</span>
          </p>
        </blockquote>
        <p>
          Just to put this in perspective: imagine you took an
          <em>E. coli</em> cell and scaled it up so that it was the length of a
          football field. And imagine you kept all the physics the same. A water
          molecule would be about an inch wide; a protein would be about the
          size of a basketball.
          <span>[<a href="#bionumbers">BioNumbers</a>]</span>
          The proteins would be juddering violently due to the thermal motion of
          the water particles bombarding them—so violently in fact that if left
          unchecked they’d be moving at 500 meters per second. But they aren’t
          left unchecked: if you were in such an environment it would be so
          crowded as to be nearly impossible to see. What you really get, then,
          is an incredible ceaseless shaking and bouncing-into-each-other of all
          the component parts.
        </p>
        <p>
          This is why shape changes that lead to different bonding affinities
          are so important in biology. It’s as if inside a cell everyone is
          constantly going up to everyone else, seeing if they fit together.
          Proteins sample the space of interactions with other proteins so
          quickly that for a long time, most biologists didn’t really
          contemplate where in the cytoplasm two reactants lived; they knew that
          you never had to wait too long for them to meet each other. In fact it
          was a
          <a href="https://www.quantamagazine.org/a-newfound-source-of-cellular-order-in-the-chemistry-of-life-20210107/">relatively recent discovery</a>
          that inside the cytoplasm certain proteins that share functional
          relationships do seem to keep especially close together, inside little
          oil drops known as “phase-separated liquids.” Weak interactive forces
          between the floppy tails of different proteins cause them to
          spontaneously “phase separate” into these more viscous pools, and this
          biases certain proteins to interact more frequently.
        </p>
        <p>
          The rate-limiting step in <em>E. coli’s</em> reaction to attractant is
          the time it takes for CheY-p to diffuse from the nose to the motor. It
          takes about a tenth of a second. The journey has actually been tracked
          on camera, using a fluorescent version of the protein:
        </p>
        <figure>
          <img loading="lazy" width="1626" height="660" alt="" src="https://jsomers.net/e-coli-chemotaxis/articleImages/electroporated-dye-labelled-CheY.png"/>
          <figcaption>
            Source:
            <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5052738/">Single-molecule imaging of electroporated dye-labelled CheY in
              live Escherichia coli</a>
          </figcaption>
        </figure>
        <h3>
          What happens when the signal reaches the motor?
        </h3>
        <p>
          Let’s talk about these motors. These things are so intricate and
          beautiful and seem so reminiscent of machines we’d engineer ourselves
          that they’re sometimes cited as evidence for intelligent design.
        </p>
        <p>
          The flagellar motor operates with close to 100% energy efficiency. It
          spins at about 1,500 rotations per second. And the craziest part is
          that like all molecular nanomachines it is entirely self-assembled.
          There’s an amazing 30-minute documentary
          <a href="https://www.youtube.com/watch?v=uw0-MHI_248">available on YouTube</a>
          that details the mechanics of the self-assembly process—and,
          refreshingly, profiles some of the scientists who figured it out,
          describing the methods they used to make their discoveries.
        </p>
        <figure>
          <video loading="lazy" width="320" height="240" src="articleImages/self-assembly.mp4" autoplay="" muted="" loop="" playsinline=""></video>
          <figcaption>
            From:
            <a href="https://www.youtube.com/watch?v=uw0-MHI_248">Self-Assembling NanoMachine: a film about flagellar biosynthesis
            </a>
          </figcaption>
        </figure>
        <p>
          My favorite part of the self-assembly process is that after building a
          base for the rotor, a sort of tunnel is built and the proteins that
          comprise the whip-like “hook” of the flagellum are extruded through
          it—as if the flagellum were built by vomiting forth parts of itself.
        </p>
        <p>
          Anyway, at the base of each rotor there are a series of proteins
          called FliG, FliM, and FliN—pronounced like “Fly G,” “Fly M,” “Fly
          N”—to which CheY-p, our Frodo-esque bearer of the message from the
          nose, attaches once it finally arrives. CheY-p has a strong affinity
          for FliG and will readily glom onto it. We’ll see how that actually
          affects the flagellum in a second. But for now it’s worth noting that
          there’s a thresholding mechanism here: just one CheY-p attaching to
          FliG won’t be enough to flip the motor from counter-clockwise to
          clockwise (thereby causing a tumble)—it actually takes a handful of
          CheY-ps conspiring to make that happen. In fact the motor has
          something like seven states, from rotating quickly counterclockwise at
          three discrete levels of decreasing speed—as if stepping through three
          gears on a bike—to stalling entirely, to starting back up again in the
          clockwise direction, also with three speeds.
        </p>
        <p>
          Even as the motor is in the process of changing direction, any CheY-ps
          that do attach to FliG are under constant threat of being removed by
          yet another player, CheZ. That is, the proteins that would reverse the
          motors are subject to removal by other proteins that un-reverse it.
          Again we have a responsive regulatory circuit reminiscent of the one
          that phosphorylated and de-phosphorylated CheY in the first place
          upstream at the receptor. The idea is that every effect is reversible,
          and in fact is reversed at a regular rate. This means that in the
          absence of further signal the cell will quickly return to baseline.
        </p>
        <h3>
          How does the motor actually change directions?
        </h3>
        <p>
          As a matter of pure mechanics this might be the most ingenious part of
          the story. It took quite a long time to figure out and even still it
          seems that we’re not entirely confident with our explanation. But one
          mechanism that’s been proposed is that CheY-p binds to a protein
          called FliM (“Fly EM”) embedded in that ring that defines the base of
          the rotor. This tilts it and causes a 90-degree rotation in an
          attached protein called FliGc. That protein sits at the interface
          between the rotating part of the motor and the so-called “stator,”
          which drives it from the part that’s anchored solidly in the cell
          membrane.
        </p>
        <figure>
          <p><img loading="lazy" width="438" height="539" alt="" src="https://jsomers.net/e-coli-chemotaxis/articleImages/flagellar-motor-a.png"/>
            <img loading="lazy" width="449" height="699" alt="" src="https://jsomers.net/e-coli-chemotaxis/articleImages/flagellar-motor-b-c.png"/>
          </p>
          <figcaption>
            Source:
            <a href="https://www.pnas.org/doi/10.1073/pnas.1110111108">A molecular mechanism of direction switching in the flagellar
              motor of Escherichia coli</a>
          </figcaption>
        </figure>
        <p>
          When FliGc changes orientation, the stepper-motor-like cycle that
          normally drives the motor counterclockwise starts driving it clockwise
          instead. In the illustration below, Figure A shows the stator, i.e.,
          the driving mechanism of the motor. It works by stepping back and
          forth between the “open” and “closed” states, schematized by the
          <img loading="lazy" alt="" src="https://jsomers.net/e-coli-chemotaxis/articleImages/image14.png"/>
          and
          <img loading="lazy" alt="" src="https://jsomers.net/e-coli-chemotaxis/articleImages/image3.png"/>
          symbols respectively. In Figure B you can see how, in the normal CCW
          direction, the repeated cycling between these two states drives the
          “teeth” of the motor—the crucial FliGc proteins, here tilted
          left-to-right.
        </p>
        <figure>
          <img loading="lazy" width="1328" height="1068" alt="" src="https://jsomers.net/e-coli-chemotaxis/articleImages/image9.png"/>
          <figcaption>
            Source: Supplementary figure S9,
            <a href="https://www.pnas.org/doi/10.1073/pnas.1110111108">A molecular mechanism of direction switching in the flagellar
              motor of Escherichia coli</a>
          </figcaption>
        </figure>
        <p>
          When the CheY-p arrives at the rotor it has the effect of flipping the
          FliGc proteins so that now they tilt right-to-left. In that
          orientation the step-drive action works the opposite way, and the
          motor rotates clockwise:
        </p>
        <figure>
          <img loading="lazy" width="1512" height="816" alt="" src="https://jsomers.net/e-coli-chemotaxis/articleImages/image6.png"/>
        </figure>
        <p>
          You can see this more clearly below in the interactive version of
          those figures. Click “Step” to drive the motor in one direction, and
          “Reverse” to flip the orientation of the FliGc proteins; Step again
          and you’ll see it run in the opposite direction.
        </p>
        <div>
          <p><img id="stepReverse-image" src="https://jsomers.net/e-coli-chemotaxis/articleImages/rotor/rotor-0.png"/>
          </p>
          
        </div>
        <p>
          It would be nice if the original paper presenting this theory included
          an illustration like this. But even this crude version took me many
          hours to make. As Bret Victor argues in
          <a href="https://www.youtube.com/watch?v=ZfytHvgHybA">Stop Drawing Dead Fish</a>, making moving pictures shouldn’t be so hard. If it were easier,
          such animations would spread everywhere in scientific communication,
          because so often what a paper describes is some kind of dynamic
          process.
        </p>

        <p>
          Dynamic illustrations would help readers grasp proposed mechanisms
          more quickly. As it is, someone who understands a complex mechanism
          usually has to explain it in patient detail to someone
          <em>else</em> who’s good at animating; this costs time and money; and
          most people simply opt not to go through with it. Perhaps someday the
          process will be democratized by better tools, or by a multimodal AI
          system.
        </p>
        <h3>
          How the motor changing directions causes the <em>E. coli</em> to
          tumble
        </h3>
        <p>
          The final part of the story—for me, anyway; there’s a lot more to
          explore!—is why exactly the clockwise rotation of just one of the
          flagellar motors would send the whole cell a-tumbling. It helps to
          understand how the thing works in “run” mode, when all the flagella
          are oriented the same way.
        </p>
        <figure>
          <img loading="lazy" width="1398" height="797" alt="" src="https://jsomers.net/e-coli-chemotaxis/articleImages/image4.png"/>
          <figcaption>
            Traced from:
            <a href="https://www.nsf.gov/news/mmg/mmg_disp.jsp?med_id=66718&amp;from=">How Escherichia coli Move | National Science Foundation</a>
          </figcaption>
        </figure>
        <p>
          Even though this bundle of flagella sort of looks like a propeller,
          when you actually think about it, that’s not really what it is. It’s
          more like a pig’s curly tail that spins with a whip-y sort of motion.
          How exactly does that propel the entire cell? A wonderful book called
          <em>Random Walks in Biology</em> gets into the physics in some detail:
        </p>

        <blockquote>
          <figure>
            <img loading="lazy" width="734" height="366" alt="" src="https://jsomers.net/e-coli-chemotaxis/articleImages/berg-random-walks-in-biology.png"/>
            <figcaption>
              <p>Source: Howard C. Berg, Random Walks in Biology.</p>
            </figcaption>
          </figure>
          <p>
            “Fig. 6.3. Analysis of viscous drag on two segments of a flagellar
            filament moving slowly to the right and turning rapidly
            counterclockwise. The velocity of each segment, <i>v</i>, is
            decomposed into velocities normal and parallel to the segment,
            <i>v<sub>n</sub></i> and <i>v<sub>p</sub></i>, respectively. The segment shown on the left is moving upward in
            front of the plane of the paper; the one shown on the right (denoted
            by primes) is moving downward behind the plane of the paper. The
            frictional drags normal and parallel to each segment,
            <i>F<sub>n</sub></i> and <i>F<sub>p</sub></i>, act in directions opposite to <i>v<sub>n</sub></i> and
            <i>v<sub>p</sub></i>, respectively. Note that their magnitudes are in the ratio
            <i>F<sub>n</sub></i>/<i>F<sub>p</sub></i> = 2<i>v<sub>n</sub></i>/<i>v<sub>p</sub></i>. <i>F<sub>n</sub></i> and <i>F<sub>p</sub></i> are decomposed into
            components normal and parallel to the helical axis,
            <i>F<sub>Ω</sub></i> and <i>F<sub>v</sub></i>, respectively. <i>F<sub>Ω</sub></i> and <i>F&#39;<sub>Ω</sub></i> act
            in opposite directions and form a couple that contributes to the
            torque. <i>F<sub>v</sub></i> and <i>F&#39;<sub>v</sub></i> act in the
            same direction and contribute to the thrust.”
          </p>
        </blockquote>
        <p>
          To get a grip on things like this, it helps to have a model of some
          kind that you can hold in your hand. And actually the question of how
          separate filaments running in phase near each other would come to
          bundle was explored nicely in
          <a href="https://www.pnas.org/doi/10.1073/pnas.2633596100">this paper</a>. The authors used physical models of the flagella by wrapping hollow
          Tygon tubes around a mandrel and filling them with epoxy. They then
          used a couple stepper motors to drive the counter-clockwise rotation.
          “The flow field generated by each helix tilts the other helix, causing
          the helices to roll around each other and form a right-handed
          wrapping”:
        </p>

        <figure>
          <video loading="lazy" width="720" height="480" src="articleImages/bundling.mp4" autoplay="" muted="" loop="" playsinline=""></video>
          <figcaption>
            From:
            <a href="https://www.youtube.com/watch?v=25FtMdIFtXM">Macro Scale Model E.Coli Flagella Bundling
            </a>
          </figcaption>
        </figure>

        <h3>
          Individuality in the bacterial population
        </h3>
        <p>
          We tend to think of a colony of something like <em>E. coli</em> as an
          undifferentiated evil goo, each bacterium identical to its neighbors.
          But people who’ve studied these organisms under the microscope observe
          a surprising amount of individual personality.
          <a href="https://www.nature.com/articles/262467a0">A 1976 <em>Nature</em> paper</a>, “Non-genetic individuality: chance in the single cell,” explores
          variation in the context of chemotaxis using strains of Salmonella and
          Enterobacter bacteria.
        </p>
        <p>
          The paper came out before the exact mechanism behind chemotactic
          regulation was well-understood; all the authors knew was that “control
          of tumbling can be rationalised as caused by changes in the levels of
          a tumble regulator.” They hypothesized that although bacteria of the
          same strain would all share the exact same DNA, there might be a
          relatively small number of copies of that “tumble regulator,” and
          natural variation in the transcription, translation, and destruction
          of these regulator proteins could account for differences in behavior.
          Their experiments were mostly at the behavioral level. They observed
          how different individuals—including those in a particularly “tumbly”
          mutant strain (I love that word)—reacted to environments with and
          without attractant, and found plenty of variance.
        </p>
        <p>
          Their theory was spot-on. We talked above about how
          <em>E. coli</em> adapt to higher and higher concentrations of
          attractant via a clever methylation mechanism. Well, it turns out that
          the methylation of the receptor struts is governed by only about
          <a href="https://www.youtube.com/watch?t=3032&amp;v=cT855rpX8bc&amp;feature=youtu.be">100 CheR proteins</a>
          in the cell. The number of those proteins—along with CheB, which
          un-methylates the struts—determines the speed of the “futile cycle”
          that reacts to changes in attractant concentration. That is, it
          affects how quickly the bacterium adapts when the concentration goes
          up and refracts when it goes down.
          <span>[<a href="#gore">Gore</a>
            <a href="https://www.youtube.com/watch?v=cT855rpX8bc&amp;t=3989s">1:06:30</a>]</span>
          Because 100 copies of CheR is so extraordinarily tiny in the context
          of the full cell buzzing with something like ten million proteins,
          variation by just a handful can have a relatively large effect on the
          cell’s behavior.
          <span>[<a href="#gore">Gore</a>
            <a href="https://www.youtube.com/watch?v=cT855rpX8bc&amp;t=4390s">1:13:20</a>]</span>
          That helps account for why different <em>E. coli</em> with the exact
          same genetic sequence will tumble and adapt at different frequencies.
        </p>
        <p>
          Recent
          <a href="https://elifesciences.org/articles/27455">experiments</a>
          have used fluorescent microscopy to quantify the individuality of
          different E. coli cells, individuality that arises not from
          differences in gene expression but from the dynamics of signaling
          networks.
        </p>
        <h3>How did we figure all this stuff out?</h3>
        <p>
          We don’t yet have the technology to just observe all of the activity
          inside a living cell. That Goodsell painting above that shows the
          crowded cytoplasm packed with proteins is an artistic composite—backed
          by rigorous research to be sure—because there’s no way to capture all
          the different players in situ at once. And obviously it’s a “still
          life,” not a video. So how could we possibly know all this detail
          about what exactly a given protein looks like, and how and when it
          interacts with others to kick off some particular part of the
          chemotaxis process?
        </p>
        <p>
          There seem to be three or four major kinds of experiment. Probably the
          most common is genetic: you can selectively disrupt one gene at a time
          and, by observing how the mutant <em>E. coli</em> behaves, begin to
          get a grip on each gene’s function. All of the proteins “CheY,”
          “CheZ,” “CheW,” and so on are named simply because they are the
          products of genes that, when excised, “cause a general defect in
          chemotaxis.”
          <span>[<a href="#blair">Blair</a>]</span> As you
          can imagine, identifying all of these is painstaking work, and
          involves a considerable amount of clever inference. For instance you
          might observe that without gene X the bacteria never seems to tumble;
          is that because that gene is involved in recognizing attractant or in
          forcing the rotor to run clockwise?
        </p>
        <p>
          Once you have a hypothesis, a second kind of experiment involves
          purifying some subset of these proteins-of-interest in vitro to see
          how they work together to form a particular signaling pathway. For
          example you could put CheA and CheY along with some phosphate groups
          and other necessary reactants and observe whether and how much
          phosphorylation takes place. That’s what the authors did in
          <a href="https://doi.org/10.1016/0092-8674(90)90429-I">this paper</a>
          in <em>Cell</em>, in 1990. They used a radioactive version of
          phosphate as a tracer. “Incorporation of [32P]phosphate into CheA or
          CheY was determined by excising the radioactive band out of the dried
          gel and quantitating in scintillation fluid or by analysis of the
          intact gel using a Phosphorimager (Molecular Dynamics, Sunnyvale, CA)
          and compare with known radioactive standards.” Another common method
          for observing in vitro dynamics is to genetically modify proteins to
          fluoresce; or to “find” a protein in solution using an antibody that
          recognizes some part of it—you attach that antibody to another
          protein, and that one you fluoresce, so you can find the hidden one.
        </p>
        <p>
          To understand the literal lock-and-key mechanics at a particular
          binding site—for instance how exactly a molecule of aspartate causes a
          receptor to deform, kicking off a signaling cascade—involves
          “structural” biology work, i.e., taking pictures of individual
          proteins or, increasingly, ensembles of them in situ. For this you can
          use X-ray crystallography, nuclear magnetic resonance imaging,
          cryo-electron microscopy, super-resolution light microscopy, or some
          combination.
        </p>
        <p>
          A <a href="https://www.ks.uiuc.edu/Research/chemotaxis/">group</a> at
          University of Illinois at Urbana-Champagne uses atomic-scale molecular
          dynamics simulations, in software, to understand structural
          details—like the exact
          <a href="https://www.youtube.com/watch?v=MCobqYrE67w">way</a> that
          CheA changes shape to kick off a downstream signaling process—that
          wouldn’t be apparent from high-resolution imaging alone. (Keith
          Cassidy, whose figures appeared above, now has a
          <a href="https://ckcassidy.mufaculty.umsystem.edu/">lab at the University of Missouri-Columbia</a>
          that’s studying the molecular dynamics of the receptor signaling
          complex.)
        </p>
        <p>
          Sometimes you can’t get a direct picture. It may require deduction to
          understand, say, how exactly a protein fits in.
          <a href="https://pubmed.ncbi.nlm.nih.gov/1326408/">One experiment</a>
          found that CheA didn’t bind to a receptor except in the presence of
          CheW; that plus the fact that adding too much CheW into the mixture
          actually led to a decrease in the ability of CheA–CheW complexes to
          bind receptor suggested that CheW competed with that complex for the
          binding site on the receptor and that therefore it must sit between
          the receptor and CheW in the receptor–CheW–CheA trimer
          <span>[via <a href="#blair">Blair</a>]</span>.
          Biology is lousy with heroic inferences like that. It’s a world that’s
          hard to see; sometimes you just have to imagine what’s going on down
          there, and back up those imaginings with the right experiments.
        </p>
        <p>
          The very idea that bacteria run and tumble came from experiments
          <a href="https://www.nature.com/articles/239500a0">published in 1972</a>
          by Howard Berg and Douglas Brown, who used a special three-dimensional
          tracking microscope of their own design to watch the little suckers in
          action. (A fun fact is that they called the non-runs “twiddles”
          instead of “tumbles.”) Some of the physics of flagellar
          propulsion—like how much force the little tails generate—<a href="https://www.google.com/books/edition/An_Introduction_to_Systems_Biology/tcxCkIxzCO4C?hl=en&amp;gbpv=1&amp;pg=PAfrontcover&amp;printsec=frontcover">was discovered</a>
          later by tethering the flagella to a microscope slide: because it’s
          anchored, the “tail wags the dog“ and you can measure how fast the
          <em>E. coli’s</em> body spins. We know that bacterial flagellar motors
          are powered by the
          <a href="https://link.springer.com/referenceworkentry/10.1007/978-3-642-11274-4_141">proton motive force</a>
          from a
          <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC431412/">1977 paper</a>
          that measured how cells ran or “twiddled” in the presence or absence
          of an electrical potential. But the research has become even more
          refined than that. Just by observing the strength of the rotation
          under various conditions—different viscosities, temperatures, and so
          on—we know that “rotation is tightly coupled to proton flow, with a
          fixed number of protons (~500) used to drive each revolution.”
          <span>[<a href="#blair">Blair</a>]</span>
          Think of how detailed an understanding we’ve gotten!
        </p>
        <p>
          One reason I’m particularly attracted to studies of
          <em>E. coli</em> chemotaxis is that it’s an early star of what’s been
          called “in silico” biology. It’s been the subject of many computer
          models. Dennis Bray, the author of that book that put me onto this
          stuff in the first place, made one of the more
          <a href="https://pubmed.ncbi.nlm.nih.gov/17208180/">popular models</a>. Here’s a nice screenshot of the model in action:
        </p>
        <figure>
          <img loading="lazy" width="562" height="425" alt="" src="https://jsomers.net/e-coli-chemotaxis/articleImages/image16.jpg"/>
        </figure>
        <p>
          Maybe the chief role of a computer model is that to get it working in
          the first place you have to explicitly articulate every one of your
          assumptions. In much the same way that writing tends to clarify your
          thinking (or at least reveal how unclear it really is), a computer
          model forces you to synthesize what you know. If anything it’s even
          more exacting than a blank page.
        </p>
        <p>
          Once you have a model, you can use it to explore variations on those
          assumptions. “The program gives the correct phenotype of over 60
          mutants in which chemotaxis-pathway components are deleted or
          overexpressed,” Bray writes. At best, a good enough model lets you
          discover things you didn’t already know, or suggests your next
          experiment. “In order to match the impulse response to a brief
          stimulus [. . .] we also had to increase the activities of the
          adaptational enzymes CheR and CheB at least an order of magnitude
          greater than published values.”
        </p>
        <h3>So what?</h3>
        <p>
          Why should you care about <em>E. coli</em> chemotaxis? A typical
          answer to that sort of question—and I’m sure the answer given in many
          of the grant applications supporting the work cited here—is that there
          are medical and practical uses. For instance: if you understand the
          signaling pathways of bacterial chemotaxis you can disrupt them; that
          work might lead to a new kind of antibiotic, which, in an era of
          increasing resistance, is direly needed. Or you might hijack
          chemotaxis pathways to create “intelligent sniffers” (Keith Cassidy’s
          phrase) that could home in on cancer cells or environmental waste.
        </p>
        <p>
          More generally you might say—and in fact I led with this up top—that
          understanding this specific phenomenon equips you to understand all
          kinds of others. “Bacterial two-component pathways
          <span>[<a href="https://en.wikipedia.org/wiki/Two-component_regulatory_system">def’n</a>]</span>
          control a dazzling array of functions including cell division,
          virulence, antibiotic resistance, metabolite fixation and utilization,
          response to environmental stress, sporulation, and taxis.”
        </p>
        <p>
          But I don’t know, to me the real reason is that it’s neat. It’s just
          fun to find out about. “To learn, and at due times to repeat what one
          has learnt, is that not after all a pleasure?”
        </p>
        <p><em>Please send feedback or corrections to <a href="https://jsomers.net/">James</a>.</em></p>
      </article></div>
  </body>
</html>
