<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <title>James Routley | Feed</title>
    <link
      rel="stylesheet"
      type="text/css"
      href="../styles.css"
      media="screen"
    />
  </head>
  <body>
    <a href="/index.html">Back</a>
    <a href="https://www.nature.com/articles/s42003-023-05587-3">Original</a>
    <h1>The origins of the Guinness stout yeast</h1>
    
    <div id="readability-page-1" class="page"><div>
            <section aria-labelledby="Abs1" data-title="Abstract" lang="en"><div id="Abs1-section"><h2 id="Abs1">Abstract</h2><p>Beer is made via the fermentation of an aqueous extract predominantly composed of malted barley flavoured with hops. The transforming microorganism is typically a single strain of <i>Saccharomyces cerevisiae</i>, and for the majority of major beer brands the yeast strain is a unique component. The present yeast used to make Guinness stout brewed in Dublin, Ireland, can be traced back to 1903, but its origins are unknown. To that end, we used Illumina and Nanopore sequencing to generate whole-genome sequencing data for a total of 22 <i>S. cerevisiae</i> yeast strains: 16 from the Guinness collection and 6 other historical Irish brewing. The origins of the Guinness yeast were determined with a SNP-based analysis, demonstrating that the Guinness strains occupy a distinct group separate from other historical Irish brewing yeasts. Assessment of chromosome number, copy number variation and phenotypic evaluation of key brewing attributes established Guinness yeast-specific SNPs but no specific chromosomal amplifications. Our analysis also demonstrated the effects of yeast storage on phylogeny. Altogether, our results suggest that the Guinness yeast used today is related to the first deposited Guinness yeast; the 1903 Watling Laboratory Guinness yeast.</p></div></section>

            

            

            
                
                    
                <div>
                    <section data-title="Introduction"><div id="Sec1-section"><h2 id="Sec1">Introduction</h2><div id="Sec1-content"><p>To mitigate against unpredictable climate and food shortages humans moved from hunter gathering to an agrarian existence<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 1" title="Diamond, J. Evolution, consequences and future of plant and animal domestication. Nature 418, 700–707 (2002)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR1" id="ref-link-section-d893017906e419">1</a></sup>. In consequence, animals, plants and unwittingly, some microorganisms were domesticated<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 2" title="Steensels, J., Gallone, B., Voordeckers, K. &amp; Verstrepen, K. J. Domestication of industrial microbes. Curr. Biol. 29, R381–R393 (2019)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR2" id="ref-link-section-d893017906e423">2</a></sup> including the ethanol-forming and CO<sub>2</sub>-generating yeast <i>S. cerevisiae</i>. The latter event occurred in multiple geographical locations resulting in the almost universal exploitation of the abilities of <i>Saccharomyces</i> yeasts to leaven bread and produce a multitude of alcoholic beverages<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 3" title="Peter, J. et al. Genome evolution across 1011 Saccharomyces cerevisiae isolates. Nature 556, 339–344 (2018)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR3" id="ref-link-section-d893017906e436">3</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 4" title="Gallone, B. et al. Domestication and divergence of Saccharomyces cerevisiae beer yeasts. Cell 166, 1397–1410 (2016)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR4" id="ref-link-section-d893017906e439">4</a></sup>.</p><p>Largely based on its ease of cultivation and <i>GRAS</i> designation <i>S. cerevisiae</i> has become the model for eukaryotic cell biology<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 5" title="Botstein, D. &amp; Fink, G. R. Yeast: an experimental organism for 21st Century biology. Genetics 189, 695–704 (2011)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR5" id="ref-link-section-d893017906e452">5</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 6" title="Johnston, M. Genome sequencing: the complete code for a eukaryotic cell. Curr. Biol. 6, 500–503 (1996)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR6" id="ref-link-section-d893017906e455">6</a></sup>. Principal component analysis of single nucleotide polymorphisms (SNPs) have established a Chinese origin<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 3" title="Peter, J. et al. Genome evolution across 1011 Saccharomyces cerevisiae isolates. Nature 556, 339–344 (2018)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR3" id="ref-link-section-d893017906e459">3</a></sup> of <i>S. cerevisiae</i> with the <i>Saccharomyces</i> genus believed to be of Asian origin<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Peris, D. et al. Macroevolutionary diversity of traits and genomes in the model yeast genus Saccharomyces. Nat. Commun. 14, 690 (2023)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR7" id="ref-link-section-d893017906e470">7</a></sup>. Publications using industrial strains have provided further understanding of the phylogenetic origins of <i>S. cerevisiae</i> used in wine and beer production<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 4" title="Gallone, B. et al. Domestication and divergence of Saccharomyces cerevisiae beer yeasts. Cell 166, 1397–1410 (2016)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR4" id="ref-link-section-d893017906e477">4</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 8" title="Gonçalves, M. et al. Distinct domestication trajectories in top-fermenting beer yeasts and wine yeasts. Curr. Biol. 26, 2750–2761 (2016)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR8" id="ref-link-section-d893017906e480">8</a></sup>. Recent publications have established the effects of European and Asian wine admixture on today’s industrial brewing yeast<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 9" title="Fay, J. C. et al. A polyploid admixed origin of beer yeasts derived from European and Asian wine populations. PLoS Biol. 17, e3000147 (2019)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR9" id="ref-link-section-d893017906e484">9</a></sup> with those yeast more readily associated with commercial brewing activities dominated by Asian admixture<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 10" title="Abou Saada, O. et al. Phased polyploid genomes provide deeper insight into the multiple origins of domesticated Saccharomyces cerevisiae beer yeasts. Curr. Biol. 32, 1350–1361 (2022)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR10" id="ref-link-section-d893017906e488">10</a></sup>. Admixture within domesticated <i>S. cerevisiae</i> is incongruous with other <i>Saccharomyces</i> yeast, where admixture and heterozygosity levels are low<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Peris, D. et al. Macroevolutionary diversity of traits and genomes in the model yeast genus Saccharomyces. Nat. Commun. 14, 690 (2023)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR7" id="ref-link-section-d893017906e499">7</a></sup>. Many of these industrial <i>S. cerevisiae</i> strains have genomes which feature polyploidy, aneuploidy and loss of fitness, all characteristic of domestication<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 3" title="Peter, J. et al. Genome evolution across 1011 Saccharomyces cerevisiae isolates. Nature 556, 339–344 (2018)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR3" id="ref-link-section-d893017906e506">3</a></sup>. These domesticated yeasts carry mutations conferring properties that make them suitable for beer fermentations. These include the formation of a spectrum of beer flavour yeast metabolites, the ability to utilise sugars such as maltotriose and maltose; ethanol tolerance and an ability to separate from beer at the end of fermentation via the process of flocculation<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 8" title="Gonçalves, M. et al. Distinct domestication trajectories in top-fermenting beer yeasts and wine yeasts. Curr. Biol. 26, 2750–2761 (2016)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR8" id="ref-link-section-d893017906e510">8</a></sup>.</p><p>Historically, distinct beer styles were associated with specific geographical regions based on the availability of raw materials and the mineral composition of the water supply<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 11" title="Boulton, C. &amp; Quain, D. Brewing Yeast and Fermentation (John Wiley &amp; Sons, 2008)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR11" id="ref-link-section-d893017906e517">11</a></sup>. In addition, specific yeast strains were selected to enhance and meet the desired beer style qualities. Initially this was unwitting since the role of yeast was only proven in the early 19th century independently in France by Cagniard-Latour and Germany by Schwann and Kutzing<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 11" title="Boulton, C. &amp; Quain, D. Brewing Yeast and Fermentation (John Wiley &amp; Sons, 2008)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR11" id="ref-link-section-d893017906e521">11</a></sup>. Nevertheless, brewers had actively been selecting yeast for their phenotypes in Europe from at least the 1600s<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 4" title="Gallone, B. et al. Domestication and divergence of Saccharomyces cerevisiae beer yeasts. Cell 166, 1397–1410 (2016)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR4" id="ref-link-section-d893017906e525">4</a></sup>. This led to selection of specific industrial brewing yeast strains from geographical locations containing specific SNPs. Confirmation of geographical specific SNPs have been observed in yeasts isolated from beers brewed in Germany, Belgium, Britain, the USA and Norway<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 4" title="Gallone, B. et al. Domestication and divergence of Saccharomyces cerevisiae beer yeasts. Cell 166, 1397–1410 (2016)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR4" id="ref-link-section-d893017906e529">4</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 12" title="Preiss, R., Tyrawa, C., Krogerus, K., Garshol, L. M. &amp; Van Der Merwe, G. Traditional Norwegian Kveik are a genetically distinct group of domesticated Saccharomyces cerevisiae brewing yeasts. Front. Microbiol. 9, 2137 (2018)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR12" id="ref-link-section-d893017906e532">12</a></sup>.</p><p>Founded in 1759 by Arthur Guinness, the St James’s Gate Dublin brewery was by 1886 the world’s largest by volume<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 13" title="GDB/FN05/0002 Articles of Association. Two copies of the Memorandum and Articles of Association of Arthur Guinness Son and Company, Guinness Archives. (1886)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR13" id="ref-link-section-d893017906e539">13</a></sup> (1.8 MHl per annum) becoming synonymous with stout beer brewed in the Irish dry stout style. The brewery archives provide comprehensive details of the yeast studies performed by scientists employed by Guinness; however, no information is held regarding the origins of the yeast first used to brew beer at St James’s Gate. The first mention of a starting culture (yeast) is detailed in the 1809 Guinness Brewing Book on the 10th and 11th of May. At the time of writing, the two principal Guinness stouts; Guinness Irish Draught Stout (IDS) and Guinness Foreign Extra Stout (FES) are brewed using yeast isolated from the 1959 Guinness pitching yeast. The Guinness archives confirm that the 1959 isolates were selected from the 1903 Watling Laboratory Guinness yeast.</p><p>In this study, 13 Guinness yeast strains, the two current production yeasts and six other historical Guinness strains were assessed to establish the origins and nature of the Guinness yeast. Our analyses have established that the Guinness yeast form a subgrouping within the previously described Beer 1 clade<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 4" title="Gallone, B. et al. Domestication and divergence of Saccharomyces cerevisiae beer yeasts. Cell 166, 1397–1410 (2016)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR4" id="ref-link-section-d893017906e547">4</a></sup>. Furthermore, the Guinness strains are mosaic sharing ancestral lineage that is different to other historical Irish brewing yeast. Genomic assessment of the Guinness yeasts confirmed that the Guinness yeast are genetically similar with different phenotypes, demonstrating the importance of phenotypic validation of yeast for brewing. The assessment of the Guinness yeast chromosome and copy number variation (CNVs) supports previous conclusions regarding the role of gene copy number and phenotype. Consequently, the principal findings of this paper are that the Guinness yeast strains are intimately related and are derived from a common ancestor.</p></div></div></section><section data-title="Results"><div id="Sec2-section"><h2 id="Sec2">Results</h2><div id="Sec2-content"><h3 id="Sec3">Determining the origins of the Guinness yeast</h3><p>The two current Guinness production and eleven historical Guinness yeast were chosen to understand the origins and phylogeny of Guinness yeast. Selection was based on a review of historical records held in the Guinness archives (Table <a data-track="click" data-track-label="link" data-track-action="table anchor" href="https://www.nature.com/articles/s42003-023-05587-3#Tab1">1</a>). The purity of the 13 Guinness yeast strains was determined using the interdelta PCR method of the TY retrotransposon elements<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 14" title="Legras, J. L. &amp; Karst, F. Optimisation of interdelta analysis for Saccharomyces cerevisiae strain characterisation. FEMS Microbiol. Lett. 221, 249–255 (2003)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR14" id="ref-link-section-d893017906e566">14</a></sup>. The method generated a ‘fingerprint’ for each of the yeast strains (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s42003-023-05587-3#MOESM2">1</a>). Dendrograms of the PCR fragment lengths were assessed using hierarchical clustering (Euclidean distance) with yeast confirmed as being identical given a 100% similarity score. For the two Guinness production yeast, FES and IDS, the interdelta PCR method established that all three FES strains were &lt;100% similar, two of IDS strains were 100% similar and the third 98%. Interdelta PCR assessment of the other eleven historical Guinness yeast established the principal fingerprint of the strains; subsequently representative samples of each of the eleven historical Guinness yeast were determined. The fingerprinting evaluation of the 13 Guinness yeast, two production yeast encompassing the five isolated individual yeast strains, and the eleven historical yeasts, resulted in a selection of 16 Guinness yeast for whole-genome sequencing.</p><div data-test="inline-table" data-container-section="table" id="table-1"><figure><figcaption><b id="Tab1" data-test="table-caption">Table 1 Name and description of yeast strains used in this study and selected following a literature review of the Guinness archives.</b></figcaption></figure></div><p>In addition to the 16 selected Guinness yeast a further 6 historical Irish brewing yeast (Table <a data-track="click" data-track-label="link" data-track-action="table anchor" href="https://www.nature.com/articles/s42003-023-05587-3#Tab1">1</a>) were sequenced using an Illumina MiSeq machine reading with a minimum depth of 30× coverage using 2 × 250 bp paired reads. The Sequences were assembled de novo using the <i>S. cerevisiae</i> S288C reference genome (R64 1-1). Origins of the Guinness strains were probed by comparing their genomes with those of the 6 other Irish Brewing yeasts and 154 previously published <i>S. cerevisiae</i> strains<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 4" title="Gallone, B. et al. Domestication and divergence of Saccharomyces cerevisiae beer yeasts. Cell 166, 1397–1410 (2016)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR4" id="ref-link-section-d893017906e1152">4</a></sup>. A total of 466,327 filtered variant sites were identified: 434,890 SNPs and 31,427 indels. For the Guinness yeast, 96,821 filtered variant sites were identified with: 88,271 SNPs and 8550 indels (Table <a data-track="click" data-track-label="link" data-track-action="table anchor" href="https://www.nature.com/articles/s42003-023-05587-3#Tab2">2</a>). Some 5407 filtered variant sites were exclusive to the Guinness yeasts, a total of 4907 SNPs and 500 indels.</p><div data-test="inline-table" data-container-section="table" id="table-2"><figure><figcaption><b id="Tab2" data-test="table-caption">Table 2 Sporulation percentage; mean sequencing coverage along <i>S. cerevisiae</i> S288C genome, transition, transversion and singleton SNPs, and total indels of the 16 sequenced Guinness yeast.</b></figcaption></figure></div><p>A maximum-likelihood (ML) phylogenetic tree of the 176 yeast was constructed using RAxML v8.2.12<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 15" title="Stamatakis, A. RAxML version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies. Bioinformatics 30, 1312–1313 (2014)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR15" id="ref-link-section-d893017906e1881">15</a></sup> based on the concatenated alignment of orthologues protein coding genes and visualised using ggtree (v 3.6.2)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 16" title="Yu, G., Lam, T. T. Y., Zhu, H. &amp; Guan, Y. Two methods for mapping and visualizing associated data on phylogeny using ggtree. Mol. Biol. Evol. 35, 3041–3043 (2018)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR16" id="ref-link-section-d893017906e1885">16</a></sup> (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s42003-023-05587-3#Fig1">1</a>). Previous work has placed the 154 <i>S. cerevisiae</i> strains into 8 separate lineages<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 4" title="Gallone, B. et al. Domestication and divergence of Saccharomyces cerevisiae beer yeasts. Cell 166, 1397–1410 (2016)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR4" id="ref-link-section-d893017906e1895">4</a></sup>. Brewing strains were located in either the Beer 1 or Beer 2 lineage. Our analysis confirmed these observations with the Guinness and other historical Irish brewing yeast placed within the Beer 1 clade (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s42003-023-05587-3#Fig1">1</a>).</p><div data-test="figure" data-container-section="figure" id="figure-1" data-title="Phylogeny and population structure of the Guinness yeast and other industrial S. cerevisiae strains."><figure><figcaption><b id="Fig1" data-test="figure-caption-text">Fig. 1: Phylogeny and population structure of the Guinness yeast and other industrial <i>S. cerevisiae</i> strains.</b></figcaption><div><div><a data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="https://www.nature.com/articles/s42003-023-05587-3/figures/1" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs42003-023-05587-3/MediaObjects/42003_2023_5587_Fig1_HTML.png?as=webp"/><img aria-describedby="Fig1" src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs42003-023-05587-3/MediaObjects/42003_2023_5587_Fig1_HTML.png" alt="figure 1" loading="lazy" width="685" height="660"/></picture></a></div><p><b>a</b> Guinness and other Irish brewing yeast within the maximum-likelihood phylogenetic tree of <i>S. cerevisiae</i>. Guinness and other Irish brewing yeast were sequenced using an Illumina MiSeq platform and combined with 154 previously sequenced <i>S. cerevisiae</i><sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 4" title="Gallone, B. et al. Domestication and divergence of Saccharomyces cerevisiae beer yeasts. Cell 166, 1397–1410 (2016)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR4" id="ref-link-section-d893017906e1926">4</a></sup>. Branch length reflects the number of substitutions per site, with colour denoting the yeast lineage. A maximum-likelihood (ML) phylogenetic tree was reconstructed in RAxML v8.2.4<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 15" title="Stamatakis, A. RAxML version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies. Bioinformatics 30, 1312–1313 (2014)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR15" id="ref-link-section-d893017906e1930">15</a></sup>, performing 100 iterations to search for the best tree, using a discrete GTRGAMMA model of rate heterogeneity. Bootstrap branch support was assessed by performing 1000 pseudoreplicates. Trees were visualised using ggtree (v 3.6.2)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 16" title="Yu, G., Lam, T. T. Y., Zhu, H. &amp; Guan, Y. Two methods for mapping and visualizing associated data on phylogeny using ggtree. Mol. Biol. Evol. 35, 3041–3043 (2018)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR16" id="ref-link-section-d893017906e1934">16</a></sup>. Yeasts that are marked in highlighted orange are described as being Mosaic<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 4" title="Gallone, B. et al. Domestication and divergence of Saccharomyces cerevisiae beer yeasts. Cell 166, 1397–1410 (2016)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR4" id="ref-link-section-d893017906e1939">4</a></sup>. Yeast marked with three asterisk are used to brew beers in the Hefeweizen style. <b>b</b> Principal component analysis of 434,890 SNPs sites from the assessed 176 <i>S. cerevisiae</i> strains. Population differences indicated by colour; NS not specified. <b>c</b> Population structure of the 434,890 SNPs sites of the <i>S. cerevisiae</i> strains used in this study. IDS1 was used as a representative Guinness yeast sample for all Guinness yeast due to the high degree of sequence similarity analysis consequently 161 genomes admixture were assessed. Resolved population fractions are represented by the vertical axis; colours denote estimated ancestral membership. Varying the number of ancestral populations (K) between 1 and 10 using the simple prior implemented in fastSTRUCTURE<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 17" title="Raj, A., Stephens, M. &amp; Pritchard, J. K. fastSTRUCTURE: variational inference of population structure in large SNP data sets. Genetics 197, 573–589 (2014)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR17" id="ref-link-section-d893017906e1955">17</a></sup>, <i>K</i> = 8 found to be optimal.</p></div></figure></div><p>The Beer 1 clade contains three separate subpopulations; a consequence of allopatric activity. These three distinct geographical groupings are Belgium/Germany, Britain and the USA. The non-Guinness Irish brewing strains were located in the ‘Britain’ subpopulations, whereas the Guinness yeast occupy their own subgroup outside the USA and ‘Britain’ subpopulations. This was an unexpected observation as the Guinness archives contain multiple records of the company requesting and supplying yeast to other Irish brewers, examples of a practice common until the mid-20th Century.</p><p>To understand the origins of the Guinness yeast populations, structure and degree of admixture were determined for the 176 genomes used in this study using FastSTRUCTURE (Version 1.0)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 17" title="Raj, A., Stephens, M. &amp; Pritchard, J. K. fastSTRUCTURE: variational inference of population structure in large SNP data sets. Genetics 197, 573–589 (2014)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR17" id="ref-link-section-d893017906e1978">17</a></sup>. Owing to the high degree of sequence similarity between the Guinness samples a single representative, IDS1, was selected for analysis. Varying the number of ancestral populations (K) between 1 and 10, <i>K</i> = 8 was found to be optimal (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s42003-023-05587-3#Fig1">1</a>); an observation in accordance with previous ancestral population investigations<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 4" title="Gallone, B. et al. Domestication and divergence of Saccharomyces cerevisiae beer yeasts. Cell 166, 1397–1410 (2016)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR4" id="ref-link-section-d893017906e1988">4</a></sup>. The non-Guinness Irish brewing strains were all placed within the ‘Britain’ subpopulation, based on &gt;80% common ancestry. The brewing strains of companies Perry, Cherry and Smithwicks aligned completely to the Britain subpopulation, whereas the Great Northern, Macardles 1966, and Macardles 1993 yeast aligned with the ‘Britain’ group (&gt;88%) but also the US and Belgium/Germany subpopulations.</p><p>Mosaicism within the phylogenetic tree is defined as a yeast possessing an ancestry of &lt;80% from a single population<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 4" title="Gallone, B. et al. Domestication and divergence of Saccharomyces cerevisiae beer yeasts. Cell 166, 1397–1410 (2016)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR4" id="ref-link-section-d893017906e1995">4</a></sup>. Within the Beer 1 subpopulation, 10 yeasts were designated as being mosaic; these yeasts that are marked in highlighted orange in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s42003-023-05587-3#Fig1">1</a>. Analysis of the representative Guinness yeast, IDS1, established the ancestry of IDS1 being &lt;80% from one grouping. The Guinness yeast are therefore Mosaic. Further analysis of the admixture (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s42003-023-05587-3#MOESM2">2</a>) using the Alpaca<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 18" title="Salazar, A. N. &amp; Abeel, T., Alpaca: a kmer-based approach for investigating mosaic structures in microbial genomes. Preprint at bioRxiv 
                  https://doi.org/10.1101/551234
                  
                 (2019)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR18" id="ref-link-section-d893017906e2005">18</a></sup> software confirmed the observations of the FastSTRUCTURE<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 17" title="Raj, A., Stephens, M. &amp; Pritchard, J. K. fastSTRUCTURE: variational inference of population structure in large SNP data sets. Genetics 197, 573–589 (2014)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR17" id="ref-link-section-d893017906e2009">17</a></sup> analysis, with the genome hereditary of the Guinness yeast belonging to multiple ancestry origins. The admixture and Alpaca<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 18" title="Salazar, A. N. &amp; Abeel, T., Alpaca: a kmer-based approach for investigating mosaic structures in microbial genomes. Preprint at bioRxiv 
                  https://doi.org/10.1101/551234
                  
                 (2019)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR18" id="ref-link-section-d893017906e2014">18</a></sup> analysis establish a non-linear monophyletic origin of the Guinness strains. The non-overlapping segments across the 16 chromosomes display a high degree of similarity with segments from yeasts derived from very distinct phylogenetic subclades and geographical locations. Consequently, analysis present in Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s42003-023-05587-3#MOESM2">2</a> confirmed the highly non-linear genetic content of the Guinness yeast relative to other members of the Beer 1 clade. Furthermore, the phylogenetic tree presented in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s42003-023-05587-3#Fig1">1</a> placed the yeast Beer042 as the closest relative to the Guinness yeast. The Alpaca<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 18" title="Salazar, A. N. &amp; Abeel, T., Alpaca: a kmer-based approach for investigating mosaic structures in microbial genomes. Preprint at bioRxiv 
                  https://doi.org/10.1101/551234
                  
                 (2019)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR18" id="ref-link-section-d893017906e2024">18</a></sup> analysis confirms this observation with all of the chromosomes sharing SNPs observed in Beer042. Omission of the Beer042 from the analysis confirmed that SNPs from the ‘not specified’ ancestry were the most significant (38.8%) (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s42003-023-05587-3#MOESM2">2</a>).</p><h3 id="Sec4">Phylogeny assessment of the Guinness yeast using long read sequencing technology</h3><p>Short read sequencing de novo assembly against a reference genome introduces potential scaffolding bias<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 19" title="Salazar, A. N. et al. Nanopore sequencing enables near-complete de novo assembly of Saccharomyces cerevisiae reference strain CEN. PK113-7D. FEMS Yeast Res. 17, p.fox074 (2017)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR19" id="ref-link-section-d893017906e2039">19</a></sup>. It has been observed that using <i>S. cerevisiae</i> S288C as the reference genome results in large sequencing gaps in sub-telomeric regions and in consequence poor yeast genome assembly<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 20" title="Nijkamp, J. F. et al. De novo sequencing, assembly and analysis of the genome of the laboratory strain Saccharomyces cerevisiae CEN. PK113-7D, a model for modern industrial biotechnology. Microb. Cell Factories 11, 1–17 (2012)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR20" id="ref-link-section-d893017906e2046">20</a></sup>. Long read sequencing provides a more comprehensive genome assessment from which a reference genome can be assembled and annotated<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 20" title="Nijkamp, J. F. et al. De novo sequencing, assembly and analysis of the genome of the laboratory strain Saccharomyces cerevisiae CEN. PK113-7D, a model for modern industrial biotechnology. Microb. Cell Factories 11, 1–17 (2012)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR20" id="ref-link-section-d893017906e2050">20</a></sup>. Using a Guinness yeast as the reference genome is suitable since <i>S. cerevisiae</i> S288C is a distant relation and this could result in potential phylogeny misrepresentation. The Guinness yeast IDS1 was selected for long read sequencing using the Minion system (Oxford Nanopore Technology, Oxford, UK). 421,040 reads were generated and the genome assembled using Flye<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 21" title="Kolmogorov, M. et al. metaFlye: scalable long-read metagenome assembly using repeat graphs. Nat. Methods 17, 1103–1110 (2020)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR21" id="ref-link-section-d893017906e2058">21</a></sup> (v 2.9) programme. The other 15 Guinness yeast were then assembled against IDS1. Following removal of intergenic variants, a total of 20,039 SNPs present in protein coding genes were identified in the 16 Guinness strains. Hierarchical clustering of the 16 Guinness strains was established using standard dissimilarity matrix analysis. The analysis of all SNPs identified in protein coding genes resulted in assessment of 20,000 protein coding biallelic SNPs<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 3" title="Peter, J. et al. Genome evolution across 1011 Saccharomyces cerevisiae isolates. Nature 556, 339–344 (2018)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR3" id="ref-link-section-d893017906e2062">3</a></sup> (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s42003-023-05587-3#Fig2">2</a>). With the exception of the 1955 Brewing Pitching yeast the Guinness yeast divide into two groupings, pre and post-1959. The observed hierarchical clustering generated using IDS1 as the reference genome was different from the observed clustering derived using <i>S. cerevisiae</i> S288C. Analysis using the former indicated that there was a deliberate selection of the Guinness yeast in 1959 confirming the archival account of Robert Gilliland’s 1959 Guinness yeast reselection programme.</p><div data-test="figure" data-container-section="figure" id="figure-2" data-title="Fig. 2"><figure><figcaption><b id="Fig2" data-test="figure-caption-text">Fig. 2</b></figcaption><div><div><a data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="https://www.nature.com/articles/s42003-023-05587-3/figures/2" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs42003-023-05587-3/MediaObjects/42003_2023_5587_Fig2_HTML.png?as=webp"/><img aria-describedby="Fig2" src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs42003-023-05587-3/MediaObjects/42003_2023_5587_Fig2_HTML.png" alt="figure 2" loading="lazy" width="685" height="279"/></picture></a></div><p>Hierarchical clustering of the 16 Guinness yeast using standard dissimilarity matrix of 20,000 protein coding biallelic SNPs as determined by de novo assembly of the Guinness yeast to the MinION sequenced reference genome IDS1.</p></div></figure></div><h3 id="Sec5">Copy number variation and chromosomal arrangement of the Guinness yeast</h3><p>To determine the CNV for the different Guinness strains a full normalised read depth analysis was performed in 250 bp windows across each chromosome. The result was normalised against an estimated copy number of 4 as this was the average chromosome copy number estimated for the Guinness strains. The likely accuracy of Guinness ploidy estimates was confirmed by reassessing the ploidy of previously published yeast samples<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 4" title="Gallone, B. et al. Domestication and divergence of Saccharomyces cerevisiae beer yeasts. Cell 166, 1397–1410 (2016)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR4" id="ref-link-section-d893017906e2097">4</a></sup>. The 250 bp window assessment of the Guinness strains (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s42003-023-05587-3#Fig3">3</a>) showed the presence of multiple copies of chromosomes and CNVs within individual chromosomes. Whilst there are chromosomal CNVs across all 16 chromosomes only chromosomes II, XIII and XV have 5 chromosomal copies in 6 or more Guinness strains; even yeast that are closely related, IDS1 and IDS2, exhibit difference in CNV in chromosome II, VI, X and XVI. Consequently all 16 Guinness yeast are aneuploid.</p><div data-test="figure" data-container-section="figure" id="figure-3" data-title="Estimated CNV in 250 base pair non-overlapping windows across the entire genome of the 16 Guinness yeast."><figure><figcaption><b id="Fig3" data-test="figure-caption-text">Fig. 3: Estimated CNV in 250 base pair non-overlapping windows across the entire genome of the 16 Guinness yeast.</b></figcaption><div><div><a data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="https://www.nature.com/articles/s42003-023-05587-3/figures/3" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs42003-023-05587-3/MediaObjects/42003_2023_5587_Fig3_HTML.png?as=webp"/><img aria-describedby="Fig3" src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs42003-023-05587-3/MediaObjects/42003_2023_5587_Fig3_HTML.png" alt="figure 3" loading="lazy" width="685" height="1136"/></picture></a></div><p>A black dot on a plot represents a window where the estimate copy number is 4. A blue dot represents a region with an estimated loss of copy number (&lt;4) and an orange dot represents a region of estimated increased copy number (&gt;4).</p></div></figure></div><p>Aneuploidy is common for ale brewing strains and has been previously reported in multiple studies<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 4" title="Gallone, B. et al. Domestication and divergence of Saccharomyces cerevisiae beer yeasts. Cell 166, 1397–1410 (2016)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR4" id="ref-link-section-d893017906e2126">4</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 11" title="Boulton, C. &amp; Quain, D. Brewing Yeast and Fermentation (John Wiley &amp; Sons, 2008)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR11" id="ref-link-section-d893017906e2129">11</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 22" title="Morard, M. et al. Aneuploidy and ethanol tolerance in Saccharomyces cerevisiae. Front. Genet. 10, 82 (2019)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR22" id="ref-link-section-d893017906e2132">22</a></sup>. Unlike natural isolates, domesticated yeast have been influenced by human activity. This selection pressures result in polyploidy for genes conferring sought-after phenotypic qualities. In concert with this, decreased global cellular fitness occurs<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 3" title="Peter, J. et al. Genome evolution across 1011 Saccharomyces cerevisiae isolates. Nature 556, 339–344 (2018)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR3" id="ref-link-section-d893017906e2136">3</a></sup>. Consequently, wild type <i>S. cerevisiae</i> are more likely to be diploid with a functioning sexual phenotype<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 22" title="Morard, M. et al. Aneuploidy and ethanol tolerance in Saccharomyces cerevisiae. Front. Genet. 10, 82 (2019)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR22" id="ref-link-section-d893017906e2143">22</a></sup>. In contrast and presumably a consequence of the effects of aneuploidy, all 16 of the Guinness strains exhibited poor sporulation (Table <a data-track="click" data-track-label="link" data-track-action="table anchor" href="https://www.nature.com/articles/s42003-023-05587-3#Tab2">2</a>); 7 strains did not sporulate at all and of the other 9 the 1959 pitching yeast had the highest sporulation percentage of just 2.8%. This observation is concurrent with Bilinski and Casey<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 23" title="Bilinski, C. A. &amp; Casey, G. P. Developments in sporulation and breeding of brewer’s yeast. Yeast 5, 429–438 (1989)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR23" id="ref-link-section-d893017906e2151">23</a></sup> who also reported poor sporulation in aneuploid yeast.</p><h3 id="Sec6">Phenotype analysis of the Guinness yeast</h3><p>In order to determine whether the genetic similarity defined a ‘Guinness yeast phenotype’, the 16 strains were assessed for phenotypic traits using mini-fermentations as described in the methods section. Some differences in both patterns and extent of attenuation, as well as ethanol production were observed (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s42003-023-05587-3#Fig4">4</a>). A one way ANOVA with Tukey’s post-hoc test of the 16 Guinness yeast determined that ethanol production was significantly different (<i>P</i> = 3.09 × 10<sup>−9</sup>). Assessment of all of the Guinness yeast using a single representative IDS, FES and Park Royal yeast confirms that ethanol production is significantly different (one way ANOVA <i>P</i> = 0.00036), but separate one way ANOVA analysis of the ethanol production of the two yeast identified within the IDS yeast were not statistically significant (<i>P</i> = 0.64) likewise analysis of the Park Royal yeast showed that there was no statistical difference between the Park Royal yeast (<i>P</i> = 0.13). Only the FES yeast demonstrated significant difference in ethanol production (One way ANOVA <i>P</i> = 0.00011) but when FES 2 and FES 3 were assessed without the inclusion of FES 1 ethanol production was determined not to be statistically significant (<i>P</i> = 0.08). Likewise, time to attenuation varied between strains with the Park Royal 1979 yeast achieving attenuation in 22 h compared with 72 h for the 1959 Guinness pitching yeast (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s42003-023-05587-3#Fig4">4</a>).</p><div data-test="figure" data-container-section="figure" id="figure-4" data-title="Phenotypic assessment of the Guinness yeast."><figure><figcaption><b id="Fig4" data-test="figure-caption-text">Fig. 4: Phenotypic assessment of the Guinness yeast.</b></figcaption><div><div><a data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="https://www.nature.com/articles/s42003-023-05587-3/figures/4" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs42003-023-05587-3/MediaObjects/42003_2023_5587_Fig4_HTML.png?as=webp"/><img aria-describedby="Fig4" src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs42003-023-05587-3/MediaObjects/42003_2023_5587_Fig4_HTML.png" alt="figure 4" loading="lazy" width="685" height="1070"/></picture></a></div><p><b>a</b> Percentage weight loss, <b>b</b> ethanol production, <b>c</b> sugar concentration <b>d</b> assessment of esters, <b>e</b> higher alcohols and <b>f</b> 2, 3 butanedione (diacetyl) production at the cessation of fermentation using the different Guinness yeast. Fermentations were performed using 100 ml of 12<sup>o</sup>P all-malt wort, with an inoculation rate of 1 × 10<sup>7</sup> ml<sup>−1</sup> cells. Samples were incubated at 25 °C and stirred at 250 rpm. Observations presented are <i>n</i> = 3 biologically independent experiments. In <b>a</b> each time point represents the mean of 3 independent replicates and standard errors are shown as SD ± error bars.</p></div></figure></div><p>The progress of brewing fermentations is typically assessed by recording the decrease in wort density. Measuring loss of weight, in the mini-fermentations described here is an indirect method which also relies on a fall in density. This decrease is a consequence of the utilisation of wort sugars by yeast and the formation of ethanol. Brewing yeast strains can assimilate simple wort sugars which include glucose, fructose, sucrose, maltose and maltotriose; dextrins are not fermented<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 24" title="Briggs, D. E., Brookes, P. A., Boulton, C. A. &amp; Stevens, R. Brewing: Science and Practice Vol. 108 (Woodhead Publ., 2004)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR24" id="ref-link-section-d893017906e2243">24</a></sup>.</p><p>Maltotriose utilisation correlates with the maltose multigene loci, five of which (<i>MAL1, 2, 3, 4</i> and 6) have been identified in <i>S. cerevisiae</i><sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 25" title="Charron, M. J., Read, E., Haut, S. R. &amp; Michels, C. A. Molecular evolution of the telomere-associated MAL loci of Saccharomyces. Genetics 122, 307–316 (1989)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR25" id="ref-link-section-d893017906e2255">25</a></sup>. Only the <i>MAL1</i> and <i>MAL3</i> multigene are present within the reference yeast <i>S. cerevisiae</i> S288C. The <i>MAL</i> locus comprises three genes, a maltose permease (gene 1), maltase (gene 2) and a <i>Trans</i> acting <i>MAL</i>-activator (gene 3). Illumina sequencing of the 16 Guinness strains confirmed the presence of <i>MAL1</i> and <i>MAL3</i>. A homozygous premature stop codon was identified in the <i>MAL1</i> maltose permease gene (<i>MAL11</i>) for all 16 Guinness yeast (Supplementary data <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s42003-023-05587-3#MOESM4">1</a>). This stop codon mutation potentially prevents the loss of gene function, although the occurrence of this stop codon is present within 145 of the 176 yeast assessed in this study and maltotriose is utilised by all of the Guinness yeast (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s42003-023-05587-3#Fig4">4</a>). Within the phylogenetic tree disruption to <i>MAL11</i>, whilst present in a small number of Beer 1 yeast, is more readily associated with brewing yeast present in the Beer 2 brewing yeast grouping<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 4" title="Gallone, B. et al. Domestication and divergence of Saccharomyces cerevisiae beer yeasts. Cell 166, 1397–1410 (2016)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR4" id="ref-link-section-d893017906e2300">4</a></sup>.</p><p>Illumina analysis of <i>MAL3</i> of S288C, established the presence of 16 heterozygous frameshift mutations in the maltose permease, <i>MAL31</i>. Similar mutations were present in all the different Guinness yeast strains (Supplementary Data <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s42003-023-05587-3#MOESM4">1</a>). The frameshift mutations in <i>MAL31</i> did not result in loss of maltose and maltotriose utilisation as assessment of maltose and maltotriose at the end of the fermentation confirmed the consumption of these wort sugars (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s42003-023-05587-3#Fig4">4</a>).</p><p>Further analysis of the Guinness strains using the longer nanopore sequencing reads determined that the <i>MAL6</i> locus was present. When detected in other <i>S. cerevisiae</i> strains the <i>MAL6</i> multigene locus is located on Chromosome VIII and is arranged from the centromere as <i>MAL63</i>, <i>MAL61</i> and <i>MAL62</i><sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 26" title="Naumov, G. I., Naumova, E. S. &amp; Michels, C. A. Genetic variation of the repeated MAL loci in natural populations of Saccharomyces cerevisiae and Saccharomyces paradoxus. Genetics 136, 803–812 (1994)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR26" id="ref-link-section-d893017906e2344">26</a></sup>. In contrast, assessment of the Guinness strains showed that <i>MAL61</i> and <i>MAL62</i> were arranged on Chromosome VIII, as expected; however, <i>MAL63</i> mapped to the sub-telomeric region of chromosome XVI. In the reference strain <i>S. cerevisiae</i> S288C this Open Reading Frame (ORF) on chromosome XVI is designated as gene YPR196W and is described as a “Putative maltose-responsive transcription factor”. The unusual arrangement of the <i>MAL6</i> locus appears to be specific to the Guinness yeast and should be considered provisional until confirmed through additional experimental data.</p><h3 id="Sec7">Production of flavour metabolites by Guinness yeast</h3><p>Beer recovered from the completed mini-fermentations were analysed for the flavour active esters: isoamyl acetate, ethyl butyrate, ethyl hexanoate and ethyl acetate (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s42003-023-05587-3#Fig4">4</a>). All were detected in the Guinness yeast fermentations apart from ethyl butyrate. Ethyl hexanoate was produced by 7 of the 16 Guinness strains, but at concentrations lower than the flavour threshold of 0.2 ppm<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 27" title="Meilgaard, M. C., Reid, D. S. &amp; Wyborski, K. A. Reference standards for beer flavor terminology system. J. Am. Soc. Brew. Chem. 40, 119–128 (1982)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR27" id="ref-link-section-d893017906e2376">27</a></sup>. The concentrations of ethyl esters varied with strain. In the case of ethyl hexanoate, isoamyl acetate and ethyl acetate the differences were statistically significant; more so in the case of the latter two esters, one way ANOVA <i>p</i> = 0.011, <i>p</i> = 9.32 × 10<sup>−20</sup> and <i>p</i> = 6.91 × 10<sup>−20</sup>, respectively. The 1947, 1950 and Park Royal 1960 yeasts produce isoamyl acetate at concentrations above the flavour threshold (1.1 ppm<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 27" title="Meilgaard, M. C., Reid, D. S. &amp; Wyborski, K. A. Reference standards for beer flavor terminology system. J. Am. Soc. Brew. Chem. 40, 119–128 (1982)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR27" id="ref-link-section-d893017906e2394">27</a></sup>) whereas ethyl acetate is the most widely produced ethyl ester. Eight of the 16 Guinness yeast strains produce ethyl acetate above the flavour threshold of 10 ppm<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 27" title="Meilgaard, M. C., Reid, D. S. &amp; Wyborski, K. A. Reference standards for beer flavor terminology system. J. Am. Soc. Brew. Chem. 40, 119–128 (1982)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR27" id="ref-link-section-d893017906e2398">27</a></sup>. A one way ANOVA of these 8 yeast confirms that ethyl acetate production in the Guinness yeast is strain-specific (<i>p</i> = 8.06 × 10<sup>−5</sup>); however, further analysis of ethyl acetate production of the four Guinness yeast collected in 1959 and 1960 show no statistical significance (One way ANOVA <i>p</i> = 0.86), likewise there is no statistical significance between the 1947 and 1950 pitching yeast (One way ANOVA <i>p</i> = 0.51).</p><p>Higher alcohols (fusel alcohols) are the most abundant yeast-derived organoleptic compounds present in beer apart from ethanol<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 28" title="Pires, E. J., Teixeira, J. A., Brányik, T. &amp; Vicente, A. A. Yeast: the soul of beer’s aroma—a review of flavour-active esters and higher alcohols produced by the brewing yeast. Appl. Microbiol. Biotechnol. 98, 1937–1949 (2014)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR28" id="ref-link-section-d893017906e2417">28</a></sup>. Isobutanol and propanol impart solvent and alcohol/sweet aromas to the beer. As in the case of ethanol production, the concentrations of higher alcohols arising in the beers varied with strain (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s42003-023-05587-3#Fig4">4</a>); one way ANOVA isobutanol <i>p</i> = 4.79 × 10<sup>−21</sup> and propanol <i>p</i> = 4.42 × 10<sup>−32</sup>.</p><p>Higher alcohols are formed by transamination, decarboxylation and reduction via the Ehrlich pathway<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 24" title="Briggs, D. E., Brookes, P. A., Boulton, C. A. &amp; Stevens, R. Brewing: Science and Practice Vol. 108 (Woodhead Publ., 2004)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR24" id="ref-link-section-d893017906e2438">24</a></sup>. Transamination is rate-determining and over-expression of <i>BAT2</i> results in increased higher alcohol production<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 29" title="Lilly, M., Bauer, F. F., Styger, G., Lambrechts, M. G. &amp; Pretorius, I. S. The effect of increased branched-chain amino acid transaminase activity in yeast on the production of higher alcohols and on the flavour profiles of wine and distillates. FEMS Yeast Res. 6, 726–743 (2006)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR29" id="ref-link-section-d893017906e2445">29</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 30" title="Styger, G., Jacobson, D., Prior, B. A. &amp; Bauer, F. F. Genetic analysis of the metabolic pathways responsible for aroma metabolite production by Saccharomyces cerevisiae. Appl. Microbiol. Biotechnol. 97, 4429–4442 (2013)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR30" id="ref-link-section-d893017906e2448">30</a></sup>. CNV of the genes responsible for the Ehrlich pathway (Supplementary data <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s42003-023-05587-3#MOESM5">2</a>) established that CNV of <i>BAT2</i> differed between the Guinness strains with a median value of 4. The Park Royal 1960 produced the highest concentration of isobutanol and was shown to have 6 copies of the <i>BAT2</i> gene. The Guinness production yeast IDS2 produce similar amounts of isobutanol compared to the Park Royal 1960 strain (<i>t</i>-test <i>P</i> = 0.61) and has 4 copies of <i>BAT2</i>. Both IDS1 and IDS2, have the same CNV of <i>BAT2</i>, but IDS1 produced significantly less isobutanol (<i>t</i>-test <i>P</i> = 0.00048) indicating the involvement of other factors. In fact, a frameshift mutation was identified in the <i>THI3</i> decarboxylase gene and the dehydrogenase <i>AAD6</i> gene was not detected in the Guinness yeasts; deletions of <i>THI3</i> and <i>AAD6</i> have been observed to negatively affect isobutanol production<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 30" title="Styger, G., Jacobson, D., Prior, B. A. &amp; Bauer, F. F. Genetic analysis of the metabolic pathways responsible for aroma metabolite production by Saccharomyces cerevisiae. Appl. Microbiol. Biotechnol. 97, 4429–4442 (2013)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR30" id="ref-link-section-d893017906e2493">30</a></sup>.</p><p>The vicinal diketone, diacetyl (2,3 butanedione) arises in beer during fermentation where it imparts a buttery or butterscotch like flavour<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 31" title="Gibson, B., Vidgren, V., Peddinti, G. &amp; Krogerus, K. Diacetyl control during brewery fermentation via adaptive laboratory engineering of the lager yeast Saccharomyces pastorianus. J. Ind. Microbiol. Biotechnol. 45, 1103–1112 (2018)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR31" id="ref-link-section-d893017906e2500">31</a></sup>. Diacetyl is formed indirectly by brewing yeast during fermentation from α-acetolactate. The latter is an intermediate in the isoleucine valine (ILV) synthetic pathway and part of the pool is exported from yeast cells where it undergoes spontaneous oxidative decarboxylation in fermenting wort to form diacetyl. Diacetyl was present at the end of fermentation for all of the Guinness yeast (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s42003-023-05587-3#Fig4">4</a>); however, only 7 of the 16 strains at a concentration greater than the flavour threshold of 100–400 ppb<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 32" title="Krogerus, K. &amp; Gibson, B. R. 125th anniversary review: diacetyl and its control during brewery fermentation. J. Inst. Brew. 119, 86–97 (2013)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR32" id="ref-link-section-d893017906e2507">32</a></sup>. Differences in diacetyl concentrations for individual strains were highly statistically significant (one way ANOVA <i>p</i> = 7.09 × 10<sup>−13</sup>).</p><p>The diacetyl precursor α-acetolactate is produced by the enzyme acetolactate synthase<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 32" title="Krogerus, K. &amp; Gibson, B. R. 125th anniversary review: diacetyl and its control during brewery fermentation. J. Inst. Brew. 119, 86–97 (2013)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR32" id="ref-link-section-d893017906e2520">32</a></sup>. The responsible genes <i>ILV2</i> and <i>ILV6</i><sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 33" title="Cullin, C., Baudin‐Baillieu, A., Guillemet, E. &amp; Ozier‐Kalogeropoulos, O. Functional analysis of YCL09C: evidence for a role as the regulatory subunit of acetolactate synthase. Yeast 12, 1511–1518 (1996)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR33" id="ref-link-section-d893017906e2529">33</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 34" title="Falco, S. C., Dumas, K. S. &amp; Livak, K. J. Nucleotide sequence of the yeast ILV2 gene which encodes acetolactate synthase. Nucleic acids Res. 13, 4011–4027 (1985)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR34" id="ref-link-section-d893017906e2532">34</a></sup> were found in all Guinness yeasts, with a total of 11 SNPs present in <i>ILV2</i> compared to the reference yeast <i>S. cerevisiae</i> S288C. A total of 5 SNPs, and two non-synonymous mutations, for <i>ILV6</i> were found in all of the Guinness strains (Supplementary data <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s42003-023-05587-3#MOESM6">3</a>). 4 of the 5 Guinness yeasts which produced the highest residual diacetyl concentration had 5 copies of the <i>ILV2</i> gene compared to a median value of 4 CNV. The apparent correlation between copy number of <i>ILV2</i> and residual diacetyl concentration could be causative; for many traditional beers’ diacetyl removal occurs via lengthy periods of storage, post-fermentation, at cool temperatures in the presence of yeast. For the Guinness yeast it was observed that the strains with the most rapid fermentations and corresponding longer post-fermentation time had the lowest diacetyl concentration (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s42003-023-05587-3#Fig4">4</a>). This is not surprising since diacetyl is reduced principally in late fermentation through passive uptake by yeast and subsequently enzymatic conversion of diacetyl first to acetoin and thence to 2,3-butanediol<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 35" title="Heidlas, J. &amp; Tressl, R. Purification and properties of two oxidoreductases catalyzing the enantioselective reduction of diacetyl and other diketones from baker’s yeast. Eur. J. Biochem. 188, 165–174 (1990)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR35" id="ref-link-section-d893017906e2558">35</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 36" title="Katz, M., Hahn-Hägerdal, B. &amp; Gorwa-Grauslund, M. F. Screening of two complementary collections of Saccharomyces cerevisiae to identify enzymes involved in stereo-selective reductions of specific carbonyl compounds: an alternative to protein purification. Enzym. Microb. Technol. 33, 163–172 (2003)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR36" id="ref-link-section-d893017906e2561">36</a></sup>. Consequently for FES production, where the presence of diacetyl is part of the beer’s flavour profile, reducing yeast contact time post attenuation ensures that the diacetyl flavour remains within the beer.</p><p>Fermentations were repeated at a scale of 1hl as fermentations at this increased scale are more representative of commercial-scale brewing. Additionally, the increased volume allowed for the resultant beers to be subjected to standard sensory profiling. Fermentations were carried out in a pilot brewery using 12-degree Plato (<sup>o</sup>P) wort. Degree Plato is a measurement, related to density, and used by brewers to determine the concentration of dissolved solids including fermentable and non-fermentable sugars in brewers’ wort. Guinness stout wort was sourced from St James’s Gate Brewery, Dublin. Two Guinness strains, Park Royal 1979 and IDS2 were chosen for the trial as the Park Royal 1979 time to attenuation was the shortest of the 16 Guinness strains and the IDS2 yeast was chosen as it was the atypical IDS production yeast. The chosen yeasts were compared with a control, a third-generation production Irish Draught yeast culture taken from the St James’s Gate brewery. Third generation refers to a culture that had already been used for 3 previous cycles of serial fermentations with intermediate cropping and storage. Yeast cultures of this “age” is considered to produce standard fermentation performance and generate typical beer. When the fermentations were completed, the resultant beers were processed using the standard Irish Guinness stout procedure. Beers were assessed chemically via analysis and organoleptically via the Guinness external taste panel using quantitative descriptive methodology.</p><p>The results were largely in accord with those obtained from the mini-fermentation study (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s42003-023-05587-3#Fig5">5</a>). The flavour panel detected isoamyl acetate and phenolic off-flavour in the beer made with Park Royal 1979, and diacetyl in Guinness made using IDS2; these observations were confirmed by GC-MS analysis (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s42003-023-05587-3#MOESM2">3</a>). Times to attenuation and final ethanol concentrations for both small and larger scale fermentations were also similar.</p><div data-test="figure" data-container-section="figure" id="figure-5" data-title="1HL fermentation and flavour assessment of the Guinness Park Royal 1979 and IDS2 yeast strains compared to present Guinness production yeast."><figure><figcaption><b id="Fig5" data-test="figure-caption-text">Fig. 5: 1HL fermentation and flavour assessment of the Guinness Park Royal 1979 and IDS2 yeast strains compared to present Guinness production yeast.</b></figcaption><div><div><a data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="https://www.nature.com/articles/s42003-023-05587-3/figures/5" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs42003-023-05587-3/MediaObjects/42003_2023_5587_Fig5_HTML.png?as=webp"/><img aria-describedby="Fig5" src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs42003-023-05587-3/MediaObjects/42003_2023_5587_Fig5_HTML.png" alt="figure 5" loading="lazy" width="685" height="267"/></picture></a></div><p><b>a</b> Rate of fermentation and <b>b</b> flavour of Guinness Irish Draught Stout brewed using a control Guinness yeast from Dublin St James’s Gate and the Guinness yeasts: Park Royal 1979 and IDS2. All fermentations were conducted in 100 L fermentation vessels with Guinness wort collected from St James’s Gate Brewery. The tasting samples were assessed, in duplicate using the Guinness Draught attribute list by an expert panel using Quantitative Descriptive methodology. A minimum of <i>n</i> = 18 assessors were used to determine flavour attributes.</p></div></figure></div><h3 id="Sec8">Phenolic off-flavour (POF) phenotype of the Guinness yeast</h3><p>The formation of 4-vinyl guaiacol, also known as <i>Phenolic off-flavour</i> (POF), imbues beers with a medicinal, clove-like aroma and flavour. It is produced from a precursor, ferulic acid, derived from cereal grains, via expression of yeast genes<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 37" title="Brumsted, D., Lauterbach, A. F. &amp; West, D. B. Phenolic characteristics in brewing 111. The role of yeast. In Proc. American Society of Brewing Chemists Annual Meeting, St Louis 142–145 (Am. Soc. Brewing Chemists, Madison, WI, 1965)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR37" id="ref-link-section-d893017906e2618">37</a></sup>. All the Guinness yeast strains used in this study were POF<sup>+</sup> (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s42003-023-05587-3#Fig6">6</a>). The degree of POF character which developed varied with yeast strain (one way ANOVA <i>P</i> value = 4.7 × 10<sup>−21</sup>). The flavour threshold of 4-vinyl guaiacol in beer is reportedly 200–400 ppb<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 38" title="Madigan, D., McMurrough, I. &amp; Smyth, M. R. Rapid determination of 4-vinyl guaiacol and ferulic acid in beers and worts by high-performance liquid chromatography. J. Am. Soc. Brew. Chem. 52, 152–155 (1994)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR38" id="ref-link-section-d893017906e2633">38</a></sup>. Here, the Guinness yeasts: 1950, 1959BY, Ikeja and 1981 yeast all produced 4-vinyl guaiacol at concentrations below the flavour threshold limit, whereas, the 1947, 1960 and Park Royal 1960 yeast all produced 4-vinyl guaiacol at a mean concentrations &gt;1000 ppb.</p><div data-test="figure" data-container-section="figure" id="figure-6" data-title="Phenolic off-flavour and flocculation phenotype of the Guinness yeast."><figure><figcaption><b id="Fig6" data-test="figure-caption-text">Fig. 6: Phenolic off-flavour and flocculation phenotype of the Guinness yeast.</b></figcaption><div><div><a data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="https://www.nature.com/articles/s42003-023-05587-3/figures/6" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs42003-023-05587-3/MediaObjects/42003_2023_5587_Fig6_HTML.png?as=webp"/><img aria-describedby="Fig6" src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs42003-023-05587-3/MediaObjects/42003_2023_5587_Fig6_HTML.png" alt="figure 6" loading="lazy" width="685" height="523"/></picture></a></div><p><b>a</b> Phenolic Off Flavours (POF) production as determined by the Analytica-EBC Method 2.3.9.5<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 97" title="Brewers of Europe. Phenolic off Flavour Method. Analytica-EBC 2.3.9.5. (2018)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR97" id="ref-link-section-d893017906e2651">97</a></sup> and Gas chromatography mass spectrometry, and <b>b</b> Illumina sequencing data of the single nucleotide polymorphism mutations of the POF genes <i>FDC1</i> and <i>PAD1</i> of the different Guinness yeast. The effects of the SNP mutations result in amino acid substitutions that are non-synonymous (NS) or synonymous (S). Each POF observations presented are <i>n</i> = 3 biologically independent experiments. <b>c</b> Flocculence characteristics of the different Guinness yeast as determined by the Analytica-EBC Gilliland Method EBC 3.5.3.1<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 48" title="Gilliland, R. B. Proc. European Brewery Convention, Brighton (1951)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR48" id="ref-link-section-d893017906e2671">48</a></sup>. The method class yeast flocculence in terms of non-flocculant (Class 1), slightly flocculant (Class 2), moderately flocculant (Class 3) and highly flocculant (Class 4). Observations presented are <i>n</i> = 10 biologically independent experiments.</p></div></figure></div><p>The POF phenotype performs an important environmental fitness function for wild <i>S. cerevisiae</i> as it enables the yeast cell to detoxify the phenylacrylic acids present in plant cell walls<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 39" title="Maillard, M. N. &amp; Berset, C. Evolution of antioxidant activity during kilning: role of insoluble bound phenolic acids of barley and malt. J. Agric. Food Chem. 43, 1789–1793 (1995)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR39" id="ref-link-section-d893017906e2692">39</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 40" title="Mukai, N., Masaki, K., Fujii, T. &amp; Iefuji, H. Single nucleotide polymorphisms of PAD1 and FDC1 show a positive relationship with ferulic acid decarboxylation ability among industrial yeasts used in alcoholic beverage production. J. Biosci. Bioeng. 118, 50–55 (2014)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR40" id="ref-link-section-d893017906e2695">40</a></sup>. The genes <i>PAD1</i> and <i>FDC1</i>, encoding a phenylacrylic acid decarboxylase and a ferulic acid decarboxylase respectively decarboxylate the phenylacrylic acid, ferulic acid, to 4-vinyl guaiacol<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 40" title="Mukai, N., Masaki, K., Fujii, T. &amp; Iefuji, H. Single nucleotide polymorphisms of PAD1 and FDC1 show a positive relationship with ferulic acid decarboxylation ability among industrial yeasts used in alcoholic beverage production. J. Biosci. Bioeng. 118, 50–55 (2014)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR40" id="ref-link-section-d893017906e2705">40</a></sup>. Illumina sequence analysis of the Guinness yeasts identified two SNPs in <i>FDC1</i> gene, and 9 SNPs in <i>PAD1</i> (8 homozygous and 1 heterozygous). Of the 11 identified SNPs, 9 SNPs have been previously identified in other strains of <i>S. cerevisiae</i><sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 40" title="Mukai, N., Masaki, K., Fujii, T. &amp; Iefuji, H. Single nucleotide polymorphisms of PAD1 and FDC1 show a positive relationship with ferulic acid decarboxylation ability among industrial yeasts used in alcoholic beverage production. J. Biosci. Bioeng. 118, 50–55 (2014)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR40" id="ref-link-section-d893017906e2718">40</a></sup>. The two SNPs identified only in the Guinness yeast are the heterozygous SNP at position 425 in <i>PAD1</i> gene and the homozygous SNP at position 790 in <i>FDC1</i>. The identified non-synonymous changes do not result in a loss of POF production function.</p><p>The median average of the CNV of <i>PAD1</i> and <i>FDC1</i> within the Guinness yeast was 4. Examination of the data showed that CNV and 4-vinyl guaiacol occurrence in beers were not related for the production for the Guinness yeast studied here (<i>f</i>-test <i>P</i> = 0.86; <i>t</i>-test <i>P</i> = 0.17) POF; yeasts with identical SNPs and CNV within <i>PAD1</i> and <i>FDC1</i> genes produced different concentrations of 4-vinyl guaiacol under identical experimental conditions.</p><h3 id="Sec9">Flocculation phenotype of the Guinness yeast</h3><p>Yeast flocculation is a reversible, non-sexual aggregation of cells which is of benefit to brewers since it improves the efficiency of sedimentation or separation from beer at the end of fermentation<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 41" title="Vidgren, V. &amp; Londesborough, J. 125th anniversary review: yeast flocculation and sedimentation in brewing. J. Inst. Brew. 117, 475–487 (2011)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR41" id="ref-link-section-d893017906e2764">41</a></sup>. The timing of flocculation in <i>S. cerevisiae</i> is dependent upon expression of the flocculation genes <i>FLO1, FLO5, FLO8, FLO9, FLO10</i> and <i>FLO11</i> and environmental factors such as calcium, pH, temperature, fermentable sugars and other nutrients<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Fichtner, L., Schulze, F. &amp; Braus, G. H. Differential Flo8p‐dependent regulation of FLO1 and FLO11 for cell–cell and cell–substrate adherence of S. cerevisiae S288C. Mol. Microbiol. 66, 1276–1289 (2007)." href="#ref-CR42" id="ref-link-section-d893017906e2777">42</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Stratford, M. Yeast flocculation: calcium specificity. Yeast 5, 487–496 (1989)." href="#ref-CR43" id="ref-link-section-d893017906e2777_1">43</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Stratford, M. Yeast flocculation: a new perspective. Adv. Microb. Physiol. 33, 1–71 (1992)." href="#ref-CR44" id="ref-link-section-d893017906e2777_2">44</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Straver, M. H., Aar, P. C., Smit, G. &amp; Kijne, J. W. Determinants of flocculence of brewer’s yeast during fermentation in wort. Yeast 9, 527–532 (1993)." href="#ref-CR45" id="ref-link-section-d893017906e2777_3">45</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Stratford, M. Induction of flocculation in brewing yeasts by change in pH value. FEMS Microbiol. Lett. 136, 13–18 (1996)." href="#ref-CR46" id="ref-link-section-d893017906e2777_4">46</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 47" title="Goossens, K. &amp; Willaert, R. Flocculation protein structure and cell–cell adhesion mechanism in Saccharomyces cerevisiae. Biotechnol. Lett. 32, 1571–1585 (2010)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR47" id="ref-link-section-d893017906e2780">47</a></sup>. For the Guinness yeasts, the degree of flocculation observed was strain-specific (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s42003-023-05587-3#Fig6">6</a>).</p><p>Gilliland<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 48" title="Gilliland, R. B. Proc. European Brewery Convention, Brighton (1951)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR48" id="ref-link-section-d893017906e2791">48</a></sup> developed a method for describing the flocculence characteristics of brewing yeast. Four classification types were defined: Class I non-flocculant, Class II slight flocculant, Class III moderately flocculant and Class IV highly flocculant. The production yeast IDS and FES were shown to be Class II and Class I respectively, confirming that the flocculation phenotype of the current Guinness production yeast is the same as those selected in 1959 and 1960 (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s42003-023-05587-3#Fig6">6c</a>).</p><p>Nanopore assessment of the IDS1 Guinness production yeast established that three complete <i>FLO</i> genes, <i>FLO9, FLO11</i> and <i>FLO8</i> were present. Scaffolding of the other Guinness yeast using IDS1 as the reference genome confirmed the presence of the three <i>FLO</i> genes indicating that <i>FLO9, FLO11</i> and <i>FLO8</i> are common to the Guinness yeasts. Of the incomplete <i>FLO</i> genes, <i>FLO1, FLO5</i> and <i>FLO10</i>, a truncated version of <i>FLO1</i> was identified. A partial read of <i>FLO5</i> was identified in the sub-telomeric region of chromosome VIII. The presence of <i>FLO10</i> could not be confirmed as the sub-telomeric region of chromosome XI was absent in the nanopore sequencing data resulting in the loss of <i>FLO10</i> and the adjacent genes <i>VBA5, NFT1</i> and <i>GEX2</i>.</p><p>The <i>FLO</i> genes identified in the Guinness yeast encode flocculins, <i>FLO9</i> and <i>FLO11</i>, and the flocculation transcription factor <i>FLO8</i>. Flocculation phenotype is affected by the length of the flocculin molecules which projects from the cell wall surface. The longer the flocculin the stronger flocculation competence<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 49" title="Verstrepen, K. J., Jansen, A., Lewitter, F. &amp; Fink, G. R. Intragenic tandem repeats generate functional variability. Nat. Genet. 37, 986–990 (2005)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR49" id="ref-link-section-d893017906e2863">49</a></sup>. The size of the <i>FLO</i> encoded flocculins is a consequence of the number of tandem repeats within the ORF<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 49" title="Verstrepen, K. J., Jansen, A., Lewitter, F. &amp; Fink, G. R. Intragenic tandem repeats generate functional variability. Nat. Genet. 37, 986–990 (2005)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR49" id="ref-link-section-d893017906e2871">49</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 50" title="Watari, J., Nomura, M., Sahara, H., Koshino, S. &amp; Keränen, S. Construction of flocculent brewer’s yeast by chromosomal integration of the yeast flocculation gene FLO1. J. Inst. Brew. 100, 73–77 (1994)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR50" id="ref-link-section-d893017906e2874">50</a></sup> (Supplementary data <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s42003-023-05587-3#MOESM7">4</a>). The <i>FLO11</i> ORF in the Guinness yeasts transcribes a 744 amino acid protein, in comparison that of the reference <i>S. cerevisiae</i> S288C genome comprises 1367 amino acid residues<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 51" title="Lo, W. S. &amp; Dranginis, A. FLO11, a yeast gene related to the STA genes, encodes a novel cell surface flocculin. J. Bacteriol. 178, 7144–7151 (1996)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR51" id="ref-link-section-d893017906e2887">51</a></sup>. Furthermore, <i>FLO9</i> contains two ORF encoding two fragments of the flocculin gene. For the Guinness yeasts FES 3, Ikeja, 1960 pitching yeast, and the 1979 and 1986 Park Royal yeasts a heterozygous SNP mutation at position 12,114 induces a premature stop codon, all the other Guinness yeast retain the consensus SNP resulting in a functioning ORF. The five Guinness yeast that carry the heterozygous SNP in <i>FLO9</i> have a flocculation phenotype of &lt;1.5 with three of the yeast observed to be Class I and therefore described as being non-flocculant. The <i>FLO8</i> transcription factor is functional in all of the Guinness yeasts examined here.</p></div></div></section><section data-title="Discussion"><div id="Sec10-section"><h2 id="Sec10">Discussion</h2><div id="Sec10-content"><p>Our analysis established that the Guinness yeast form their own subgroup within the previously described Beer 1 brewing clade and that the grouping of the Guinness yeast is separate from other historical Irish Brewing yeast strains. The Beer 1 clade SNPs are of European and Asian origin<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 9" title="Fay, J. C. et al. A polyploid admixed origin of beer yeasts derived from European and Asian wine populations. PLoS Biol. 17, e3000147 (2019)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR9" id="ref-link-section-d893017906e2910">9</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 10" title="Abou Saada, O. et al. Phased polyploid genomes provide deeper insight into the multiple origins of domesticated Saccharomyces cerevisiae beer yeasts. Curr. Biol. 32, 1350–1361 (2022)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR10" id="ref-link-section-d893017906e2913">10</a></sup> and unlike the other historical Irish Brewing yeast that group within the ‘Britain’ subpopulations, the data presented in this study indicates the contribution of several lineages to the genetic make-up of the Guinness strains. These different lineages presented in this study establishes that the Guinness yeast are mosaic with an ancestry &lt;80% from a single geographical population.</p><p>The analysis presented in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s42003-023-05587-3#Fig1">1</a> and Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s42003-023-05587-3#MOESM2">2</a> establishes that the yeast Beer042 shares a recent common ancestor with the Guinness yeast. The Beer042 was deposited in 1979 and at that time was used to brew a lager-style beer in Belgium (personal communication with the owners of Beer042). Beer042 was deposited using the then accepted nomenclature for lager strains <i>Saccharomyces carlsbergensis</i> Hansen. Subsequent whole-genome sequencing has confirmed that it was mislabelled and that it is a <i>S. cerevisiae</i> yeast. Within the Guinness archive, the last mention of yeast being brought into the brewery is on the 2nd of January 1854. The performance of this yeast was described as poor, consequently it was disposed of, and brewing continued using the then house yeast. There are no subsequent entries of additional yeast in the Guinness archives, other than the Guinness yeast, being used to brew Guinness. As Beer042 was labelled as <i>Saccharomyces carlsbergensis</i> Hansen it is likely that it originated from Emil Hansen’s Carlsberg group yeast collection. At the end of the 19th century Emil Hansen pioneered the selection and propagation of pure yeast strains for use in brewing<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 11" title="Boulton, C. &amp; Quain, D. Brewing Yeast and Fermentation (John Wiley &amp; Sons, 2008)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR11" id="ref-link-section-d893017906e2936">11</a></sup>. At that time this was a novel concept, and it prompted much interaction between various European brewers<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 52" title="Anderson, R. One yeast or two? Pure yeast and top fermentation. Brew. Hist. 149, 30–38 (2012)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR52" id="ref-link-section-d893017906e2940">52</a></sup>. In the case of Guinness, company scientists visited numerous breweries in the UK on multiple occasions<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 53" title="GDB/RD02/0365. File No 521A - Visits in Great Britain and Ireland 1902-11-21-1978-12-07. In Guinness Archives." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR53" id="ref-link-section-d893017906e2944">53</a></sup>. Consequently, a plausible reason for the relationship demonstrated here with Beer042 and the Guinness yeast is that a common ancestor was shared from Dublin to other European brewers. Originally, this common ancestor would have been deposited in Emil Hansen’s collection. The resulting observed differences in SNPs between Beer042 and the Guinness yeast are likely due to the consequences of yeast evolution driven by differences in handling practices.</p><p>The phylogeny assessment of the Guinness yeast confirmed the expected genealogical relationship between the historical and current production strains, with a division being observed at a pre- and post- ‘1959’ timeline. The 1960 pitching yeast and Park Royal 1960, were collected in 1960 but are yeasts derived from pre-1959 stock. The selection that was undertaken in 1959 used single-cell isolates obtained from the then Guinness Pitching yeast (Guinness Archives). These isolates were tested in the Guinness Research Laboratory with the flocculation phenotype used as the principal differentiating selection criterion. Subsequently the Class II IDS yeast was chosen to brew Guinness Irish Stout, with a second selection from the single-cell isolates undertaken in 1960. This produced a Class I flocculant yeast which was chosen to produce FES on the basis that it was advantageous for yeast to remain suspended in bottle conditioned stout intended for the export market.</p><p>The flocculation assessment of the Guinness yeast reported in this study confirmed that the phenotype of the production yeasts FES and IDS have been preserved when compared with their 1959 and 1960 phenotypes. Whilst the flocculation phenotype of the production yeast has been maintained the present production yeast have diverged from the 1959 Guinness pitching yeast with regard to other characteristics. The reason for this may be related to changes in the methods used for preserving cultures. Prior to 1986 all yeast cultures were maintained on wort agar slopes stored at 4 °C and sub-cultured every 6 months. This method was standard industry practice up until the 1970s and 1980s. Following the work by Labatt’s Brewing Company<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 54" title="Russell, I. &amp; Stewart, G. G. Liquid nitrogen storage of yeast cultures compared to more traditional storage methods. J. Am. Soc. Brew. Chem. 39, 19–24 (1981)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR54" id="ref-link-section-d893017906e2954">54</a></sup> yeast culture storage in liquid nitrogen was introduced and widely adopted.</p><p>The production yeast IDS and FES were subjected to an additional reselection procedure which started with individual non-petite yeast colonies from which phenotypes were chosen that were ‘prone to spontaneous changes’<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 55" title="Donnelly, D. &amp; Hurley, J. Yeast monitoring: the Guinness experience. Ferment 9, 283–286 (1996)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR55" id="ref-link-section-d893017906e2962">55</a></sup>. The phenotypic characters of interest were flocculation, maltotriose utilisation and head formation (cropping behaviour at the end of fermentation). Some 50 colonies were selected, pooled, and cultured on fresh agar slopes. The rationale was that this should minimise the risk of selection of a potential defect or mutation from a single source which would adversely affect the Guinness yeast. This strategy was adopted since it was concluded that it would mitigate the potential adverse effects of long-term maintenance of yeast cultures on slopes<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 56" title="Berry, D. R. &amp; Slaughter, J. C., Alcoholic beverage fermentations. In Fermented beverage production. 25-39 (Springer, Boston, MA, US, 2003)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR56" id="ref-link-section-d893017906e2966">56</a></sup>. This process of selecting positive phenotypic traits could over time increase dissimilarities resulting in potential divergence from the original 1959 Guinness yeast. This process is similar to adaptive evolution, the process of positive selection of an advantageous phenotype. Adaptive evolution has been used successfully by others to enhance yeast phenotypes<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 31" title="Gibson, B., Vidgren, V., Peddinti, G. &amp; Krogerus, K. Diacetyl control during brewery fermentation via adaptive laboratory engineering of the lager yeast Saccharomyces pastorianus. J. Ind. Microbiol. Biotechnol. 45, 1103–1112 (2018)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR31" id="ref-link-section-d893017906e2970">31</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 57" title="Ekberg, J. et al. Adaptive evolution of the lager brewing yeast Saccharomyces pastorianus for improved growth under hyperosmotic conditions and its influence on fermentation performance. FEMS Yeast Res. 13, 335–349 (2013)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR57" id="ref-link-section-d893017906e2973">57</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 58" title="Gibson, B. et al. Adaptive laboratory evolution of ale and lager yeasts for improved brewing efficiency and beer quality. Annu. Rev. Food Sci. Technol. 11, 23–44 (2020)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR58" id="ref-link-section-d893017906e2976">58</a></sup>. The ‘adaptive evolution’ hypothesis is further supported by the observable differences in the 1981 IDS Guinness yeast compared to the IDS1 and IDS2 production yeast. The last reselection of the Guinness production yeast took place in 1989, 8 years after the 1981 Guinness pitching yeast was deposited in the yeast library. The 1981 IDS yeast is chronologically the closest yeast to the current IDS production yeast, unlike the IDS yeast, the 1981 IDS yeast was not reselected therefore the 1981 yeast is a record of IDS yeast at that time. The observable hierarchal clustering of the Nanopore phylogeny assessment places the 1981 IDS yeast closest to the IDS yeasts, accordingly the observable differences between the IDS yeasts and the 1981 IDS yeast are likely to be a consequence of the Guinness yeast ‘adaptive evolution’ reselection process.</p><p>The loss of meiotic cell division and observed aneuploidy of the Guinness yeast are congruent with yeast domestication<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 59" title="Albertin, W. et al. Evidence for autotetraploidy associated with reproductive isolation in Saccharomyces cerevisiae: towards a new domesticated species. J. Evolut. Biol. 22, 2157–2170 (2009)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR59" id="ref-link-section-d893017906e2983">59</a></sup>. Aneuploidy can confer phenotypic advantages such as enhanced tolerance to ethanol, temperature and oxidative stresses<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Voordeckers, K. &amp; Verstrepen, K. J. Experimental evolution of the model eukaryote Saccharomyces cerevisiae yields insight into the molecular mechanisms underlying adaptation. Curr. Opin. Microbiol. 28, 1–9 (2015)." href="#ref-CR60" id="ref-link-section-d893017906e2987">60</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Linder, R. A., Greco, J. P., Seidl, F., Matsui, T. &amp; Ehrenreich, I. M. The stress-inducible peroxidase TSA2 underlies a conditionally beneficial chromosomal duplication in Saccharomyces cerevisiae. G3 Genes Genomes Genet. 7, 3177–3184 (2017)." href="#ref-CR62" id="ref-link-section-d893017906e2987_1">62</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Scopel, E. F., Hose, J., Bensasson, D. &amp; Gasch, A. P. Genetic variation in aneuploidy prevalence and tolerance across Saccharomyces cerevisiae lineages. Genetics 217, iyab015 (2021)." href="#ref-CR63" id="ref-link-section-d893017906e2987_2">63</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 63" title="Scopel, E. F., Hose, J., Bensasson, D. &amp; Gasch, A. P. Genetic variation in aneuploidy prevalence and tolerance across Saccharomyces cerevisiae lineages. Genetics 217, iyab015 (2021)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR63" id="ref-link-section-d893017906e2990">63</a></sup>. Although assessment of stress resistance was not in the scope of this study the observed variable ploidy together with the phenotypic assessments made via studies of fermentation performance suggest that the Guinness strains have evolved to manage environmental stresses associated with commercial brewing. For example, acquisition of additional copies of chromosome III which correlated with improved ethanol tolerance has been reported<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 22" title="Morard, M. et al. Aneuploidy and ethanol tolerance in Saccharomyces cerevisiae. Front. Genet. 10, 82 (2019)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR22" id="ref-link-section-d893017906e2994">22</a></sup>. Others, studying industrial processes employing <i>S. cerevisiae</i>, including baking and sake brewing, have suggested there is no evidence of amplification of specific chromosomes carrying traits which can be directly attributable to industrial practices<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 63" title="Scopel, E. F., Hose, J., Bensasson, D. &amp; Gasch, A. P. Genetic variation in aneuploidy prevalence and tolerance across Saccharomyces cerevisiae lineages. Genetics 217, iyab015 (2021)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR63" id="ref-link-section-d893017906e3001">63</a></sup>. The studies reported here support the latter contention. For the strains examined, no specific chromosome was identified as being responsible for the Guinness yeast phenotype.</p><p>In addition to ploidy, gene copy number correlates with gene expression. Aneuploid yeast with multiple gene copy numbers will have increased expression levels compared to a haploid yeast with a single copy gene<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 22" title="Morard, M. et al. Aneuploidy and ethanol tolerance in Saccharomyces cerevisiae. Front. Genet. 10, 82 (2019)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR22" id="ref-link-section-d893017906e3008">22</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 61" title="Yona, A. H. et al. Chromosomal duplication is a transient evolutionary solution to stress. Proc. Natl Acad. Sci. 109, 21010–21015 (2012)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR61" id="ref-link-section-d893017906e3011">61</a></sup>. Increased gene copy numbers does not always result in an increase in the concentrations of the resultant proteins, even though the genes are translated since turnover via proteolysis also occurs<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 64" title="Torres, E. M. et al. Effects of aneuploidy on cellular physiology and cell division in haploid yeast. Science 317, 916–924 (2007)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR64" id="ref-link-section-d893017906e3015">64</a></sup>. Data presented in this study established that there are gene CNVs between the different Guinness yeast strains but with reference to the concentrations of important beer flavour yeast-derived metabolites: higher alcohols, diacetyl and phenolic off-flavour (POF), the CNV did not significantly influence the phenotypic outcome.</p><p>The observed difference in phenotype between the Guinness yeasts are of interest to brewers as the data presented in this study establishes that yeast that are genotypically similar can be phenotypically diverse. The phenotypic data presented in this study does not correlate with the group of yeast. For example, there are differences between the Park Royal Guinness yeast, present production yeast, pre-1959 and post-1959 Guinness yeast. The difference in POF, esters, higher alcohols and ethanol production is yeast strain-specific. Predicting phenotype based upon genotype is challenging<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Heavner, B. D. &amp; Price, N. D. Comparative analysis of yeast metabolic network models highlights progress, opportunities for metabolic reconstruction. PLoS Comput. Biol. 11, e1004530 (2015)." href="#ref-CR65" id="ref-link-section-d893017906e3022">65</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Pereira, R., Nielsen, J. &amp; Rocha, I. Improving the flux distributions simulated with genome-scale metabolic models of Saccharomyces cerevisiae. Metab. Eng. Commun. 3, 153–163 (2016)." href="#ref-CR66" id="ref-link-section-d893017906e3022_1">66</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Szappanos, B. et al. An integrated approach to characterize genetic interaction networks in yeast metabolism. Nat. Genet. 43, 656–662 (2011)." href="#ref-CR67" id="ref-link-section-d893017906e3022_2">67</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 68" title="Brochado, A. R., Andrejev, S., Maranas, C. D. &amp; Patil, K. R. Impact of stoichiometry representation on simulation of genotype-phenotype relationships in metabolic networks. PLoS Comput. Biol. 8, e1002758 (2012)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR68" id="ref-link-section-d893017906e3025">68</a></sup>, even with the use of machine learning predicting phenotype based upon genotype<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 69" title="Grinberg, N. F., Orhobor, O. I. &amp; King, R. D. An evaluation of machine-learning for predicting phenotype: studies in yeast, rice, and wheat. Mach. Learn. 109, 251–277 (2020)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR69" id="ref-link-section-d893017906e3029">69</a></sup> has a poor correlation for <i>S. cerevisiae</i> (&lt;22%). The data presented in this study provides further valuable awareness of the relationship of genotype and phenotype and confirms previous studies’ conclusions on the difficulty of predicting phenotype from genotype.</p><p>Good brewing practice is to replace pitching yeast; the yeast added to wort, after 8–15 re-pitching procedures<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 70" title="Boulton, C. A. 4 Developments in Brewery Fermentation. Biotechnol. Genet. Eng. Rev. 9, 127–181 (1991)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR70" id="ref-link-section-d893017906e3039">70</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 71" title="Smart, K. A. &amp; Whisker, S. Effect of serial repitching on the fermentation properties and condition of brewing yeast. J. Am. Soc. Brew. Chem. 54, 41–44 (1996)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR71" id="ref-link-section-d893017906e3042">71</a></sup>. A major reason for this practice is to avoid genetic drift so that the fermentation outputs remain consistent<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 72" title="Powell, C. D. &amp; Diacetis, A. N. Long term serial repitching and the genetic and phenotypic stability of brewer’s yeast. J. Inst. Brew. 113, 67–74 (2007)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR72" id="ref-link-section-d893017906e3046">72</a></sup>. The data presented in this study offers another potential explanation; especially for pitching yeast that are not pooled from a pure culture. The differences in fermentation behaviour maybe a consequence of the difference in phenotype. The Guinness production yeast, IDS and FES contain yeast that are phenotypically diverse although they are genotypically similar, consequently, as the yeast is re-pitched there is potential for the concentration of the different yeast to change resulting in a different fermentation/phenotype response. The observations of phenotype and genotype in this study raises important questions for brewers and other groups that use <i>S. cerevisiae</i> yeast for industrial processes and highlights the importance of brewers maintaining their production yeast. Further consideration should be given to understanding the role of genotype on phenotype as this will improve <i>S. cerevisiae</i> industrial comprehension.</p><p>All Guinness yeast strains examined had a POF<sup>+</sup> phenotype. This phenotype is widely found within the wild <i>S. cerevisiae</i> population but much less common in industrial strains<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 4" title="Gallone, B. et al. Domestication and divergence of Saccharomyces cerevisiae beer yeasts. Cell 166, 1397–1410 (2016)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR4" id="ref-link-section-d893017906e3064">4</a></sup>. The same authors and others have reported that none of the examples of the Britain, Ireland and USA brewing yeasts assessed were POF<sup>+</sup> <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 4" title="Gallone, B. et al. Domestication and divergence of Saccharomyces cerevisiae beer yeasts. Cell 166, 1397–1410 (2016)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR4" id="ref-link-section-d893017906e3070">4</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 8" title="Gonçalves, M. et al. Distinct domestication trajectories in top-fermenting beer yeasts and wine yeasts. Curr. Biol. 26, 2750–2761 (2016)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR8" id="ref-link-section-d893017906e3073">8</a></sup>. This suggested that the loss of POF production is a consequence of deliberate selection by the brewer. In contrast, where the POF flavour is an important characteristic; as with German Hefeweizen beers (wheat beers), the POF phenotype has been retained<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 4" title="Gallone, B. et al. Domestication and divergence of Saccharomyces cerevisiae beer yeasts. Cell 166, 1397–1410 (2016)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR4" id="ref-link-section-d893017906e3078">4</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 8" title="Gonçalves, M. et al. Distinct domestication trajectories in top-fermenting beer yeasts and wine yeasts. Curr. Biol. 26, 2750–2761 (2016)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR8" id="ref-link-section-d893017906e3081">8</a></sup>. In the case of the Guinness yeast retention of the POF genotype is unusual for domesticated brewing yeast<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 4" title="Gallone, B. et al. Domestication and divergence of Saccharomyces cerevisiae beer yeasts. Cell 166, 1397–1410 (2016)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR4" id="ref-link-section-d893017906e3085">4</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 8" title="Gonçalves, M. et al. Distinct domestication trajectories in top-fermenting beer yeasts and wine yeasts. Curr. Biol. 26, 2750–2761 (2016)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR8" id="ref-link-section-d893017906e3088">8</a></sup>.</p><p>Retention of the POF phenotype in the Guinness yeasts was unlikely to have been a deliberate act; rather at the concentrations found in the beers the effect was benign as it did not create flavour issues. The precursor of 4-vinyl guaiacol, the causative agent of POF, is ferulic acid a component of cell wall polysaccharides of barley, wheat, rice and maize<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 38" title="Madigan, D., McMurrough, I. &amp; Smyth, M. R. Rapid determination of 4-vinyl guaiacol and ferulic acid in beers and worts by high-performance liquid chromatography. J. Am. Soc. Brew. Chem. 52, 152–155 (1994)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR38" id="ref-link-section-d893017906e3096">38</a></sup>. Free ferulic acid is released during the mashing stage, a process used by brewers to convert grain starches into fermentable sugars during wort production. The free ferulic acid is then converted to 4-vinyl guaiacol by POF<sup>+</sup> yeast during fermentation. The extent of ferulic acid release is influenced by the conditions employed during mashing<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 73" title="McMurrough, I. et al. Control of ferulic acid and 4‐vinyl guaiacol in brewing. J. Inst. Brew. 102, 327–332 (1996)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR73" id="ref-link-section-d893017906e3102">73</a></sup>. A mash temperature stand of 45–50 °C is optimal for releasing ferulic acid<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 24" title="Briggs, D. E., Brookes, P. A., Boulton, C. A. &amp; Stevens, R. Brewing: Science and Practice Vol. 108 (Woodhead Publ., 2004)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR24" id="ref-link-section-d893017906e3106">24</a></sup>. This is a typical starting temperature for Continental European brewers and consequently this suits those beers that have a pronounced POF character, otherwise POF<sup>−</sup> yeast strains must be used<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 74" title="Stewart, G. G., Russell, I. &amp; Anstruther, A. eds. Handbook of Brewing (CRC Press, 2017)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR74" id="ref-link-section-d893017906e3113">74</a></sup>. Irish and British brewers prefer isothermal or infusion mashing using a temperature of 67 °C<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 74" title="Stewart, G. G., Russell, I. &amp; Anstruther, A. eds. Handbook of Brewing (CRC Press, 2017)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR74" id="ref-link-section-d893017906e3117">74</a></sup>. This regime does not favour extensive release of ferulic acid<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 74" title="Stewart, G. G., Russell, I. &amp; Anstruther, A. eds. Handbook of Brewing (CRC Press, 2017)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR74" id="ref-link-section-d893017906e3121">74</a></sup> and therefore the potential for development of POF in Guinness stouts would not be great. The importance of the presence of POF in Guinness beers is further reduced since an internal expert sensory panel has determined that the flavour threshold concentration for 4-vinyl guaiacol in Guinness stouts is higher than in other beer styles. These factors in concert would reduce pressures to eliminate POF genes in Guinness production yeast.</p><p>All Guinness yeasts are POF<sup>+</sup> but the concentration of 4-vinyl guaiacol formed varies with individual yeast strains. The Guinness yeast all share the same SNP mutations and the CNV number does not affect POF production. The presence of stop codon in <i>FDC1</i> and <i>PAD1</i> result in the loss of POF in negative yeast strains<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 4" title="Gallone, B. et al. Domestication and divergence of Saccharomyces cerevisiae beer yeasts. Cell 166, 1397–1410 (2016)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR4" id="ref-link-section-d893017906e3136">4</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 8" title="Gonçalves, M. et al. Distinct domestication trajectories in top-fermenting beer yeasts and wine yeasts. Curr. Biol. 26, 2750–2761 (2016)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR8" id="ref-link-section-d893017906e3139">8</a></sup>. However, Gonçalves et al.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 8" title="Gonçalves, M. et al. Distinct domestication trajectories in top-fermenting beer yeasts and wine yeasts. Curr. Biol. 26, 2750–2761 (2016)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR8" id="ref-link-section-d893017906e3143">8</a></sup> observed that in the yeast TUM 507 stop codons present in <i>PAD1</i> and <i>FDC1</i> did not result in loss of POF phenotype, moreover, in the TUM 380 yeast where there were functioning <i>PAD1</i> and <i>FDC1</i> genes present, there was a loss of POF production phenotype<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 8" title="Gonçalves, M. et al. Distinct domestication trajectories in top-fermenting beer yeasts and wine yeasts. Curr. Biol. 26, 2750–2761 (2016)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR8" id="ref-link-section-d893017906e3160">8</a></sup>. Gonçalves<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 8" title="Gonçalves, M. et al. Distinct domestication trajectories in top-fermenting beer yeasts and wine yeasts. Curr. Biol. 26, 2750–2761 (2016)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR8" id="ref-link-section-d893017906e3164">8</a></sup> concluded that as yet unidentified compounds or enzymes were affecting POF production in TUM 380 and 507 yeasts. Observations made in this study may corroborate Gonçalves’s<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 8" title="Gonçalves, M. et al. Distinct domestication trajectories in top-fermenting beer yeasts and wine yeasts. Curr. Biol. 26, 2750–2761 (2016)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR8" id="ref-link-section-d893017906e3169">8</a></sup> conclusions since individual Guinness strains produce significantly different concentrations of 4-vinyl guaiacol even though all yeast share the same gene content. These observations warrant further investigation.</p><p>This study provides evidence of beer style influencing yeast selection. With the exception of Hefeweizen specific yeast, previous studies have demonstrated that yeast subdivide along geographical locations and not beer style<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 4" title="Gallone, B. et al. Domestication and divergence of Saccharomyces cerevisiae beer yeasts. Cell 166, 1397–1410 (2016)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR4" id="ref-link-section-d893017906e3176">4</a></sup>; our findings run contradictory to this observation. The data presented in this study establishes that yeast from a similar geographical location, Ireland, are genealogically dissimilar despite documented evidence of the wide sharing of yeast between brewers. All of the non-Guinness Irish brewing yeast were used to brew ales, whilst the Guinness yeast were used to make stout. The phylogeny assessment of the Irish yeast divide on the stout/ale brewing axis. Perhaps the reason for the difference is that yeast used to brew ales are generalist, brewing different types of beers, consequently yeast with universally preferable characteristics such as good flocculation and POF<sup>−</sup> would be selected for by the brewer. For yeast that brew a specific beer style these universal characteristics are not essential subsequently brewers can select a yeast that enhances the features of a particular beer style even if these characteristics are unsuitable for generalist yeast. This hypothesis is supported by the findings of both Gallone et al.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 4" title="Gallone, B. et al. Domestication and divergence of Saccharomyces cerevisiae beer yeasts. Cell 166, 1397–1410 (2016)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR4" id="ref-link-section-d893017906e3182">4</a></sup> and Gonçalves et al.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 8" title="Gonçalves, M. et al. Distinct domestication trajectories in top-fermenting beer yeasts and wine yeasts. Curr. Biol. 26, 2750–2761 (2016)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR8" id="ref-link-section-d893017906e3186">8</a></sup> who observed that yeast used to brew beers in the Hefeweizen style are distinct, forming their own subgrouping; interestingly like the Guinness yeast Hefeweizen yeast are also mosaic<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 4" title="Gallone, B. et al. Domestication and divergence of Saccharomyces cerevisiae beer yeasts. Cell 166, 1397–1410 (2016)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR4" id="ref-link-section-d893017906e3190">4</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 8" title="Gonçalves, M. et al. Distinct domestication trajectories in top-fermenting beer yeasts and wine yeasts. Curr. Biol. 26, 2750–2761 (2016)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR8" id="ref-link-section-d893017906e3193">8</a></sup>. The effects of raw material, especially the mineral content of water are well known<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 24" title="Briggs, D. E., Brookes, P. A., Boulton, C. A. &amp; Stevens, R. Brewing: Science and Practice Vol. 108 (Woodhead Publ., 2004)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR24" id="ref-link-section-d893017906e3198">24</a></sup>. This has led to the association of geographical locations with certain beer types such as Burton on Trent in the United Kingdom with ales, and delicate lagers associated with Pilsen in the Czech Republic<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 24" title="Briggs, D. E., Brookes, P. A., Boulton, C. A. &amp; Stevens, R. Brewing: Science and Practice Vol. 108 (Woodhead Publ., 2004)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR24" id="ref-link-section-d893017906e3202">24</a></sup>. Unlike brewers vintners use the microflora of the raw material grapes to ferment. Studies have demonstrated that the microflora of geographical regions are associated with certain types of wine<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 75" title="Knight, S., Klaere, S., Fedrizzi, B. &amp; Goddard, M. R. Regional microbial signatures positively correlate with differential wine phenotypes: evidence for a microbial aspect to terroir. Sci. Rep. 5, 1–10 (2015)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR75" id="ref-link-section-d893017906e3206">75</a></sup> this has resulted in the concept of terroir. For wine terroir is well-established but the concept of terroir in beer is still in its infancy despite the recent studies on the terroir of hops<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 76" title="Morcol, T. B., Negrin, A., Matthews, P. D. &amp; Kennelly, E. J. Hop (Humulus lupulus L.) terroir has large effect on a glycosylated green leaf volatile but not on other aroma glycosides. Food Chem. 321, 126644 (2020)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR76" id="ref-link-section-d893017906e3210">76</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 77" title="Van Holle, A. et al. Relevance of hop terroir for beer flavour. J. Inst. Brew. 127, 238–247 (2021)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR77" id="ref-link-section-d893017906e3213">77</a></sup>. Our findings provide another possible avenue for brewers to further explore the concept of beer associated terroir.</p><p>In conclusion the analysis presented in this study establishes that the Guinness yeast are not only significantly different from other historical Irish Brewing yeast but they form a sub- group within the brewing yeast clade. The genealogy of the different Guinness yeast is confirmed by our analysis and supports the Guinness archive historical records that the Guinness yeast used today is related to the first deposited Guinness yeast; the 1903 Watling Laboratory Guinness yeast.</p></div></div></section><section data-title="Methods"><div id="Sec11-section"><h2 id="Sec11">Methods</h2><div id="Sec11-content"><h3 id="Sec12">Yeast strain selection and maintenance</h3><p>A total of 19 Irish brewing yeast strains including 13 from the Guinness collection were selected for assessment (Table <a data-track="click" data-track-label="link" data-track-action="table anchor" href="https://www.nature.com/articles/s42003-023-05587-3#Tab1">1</a>). The Guinness strains included two current production strains and 11 other historical strains selected as they were the principal brewing strains used to produce Guinness at that time. Cultures were stored in cryo vials (Fisher) in liquid nitrogen at −196 °C using 50% glycerol (Sigma–Aldrich) as a cryo-preservative. Cultures were recovered and inoculated into 25 ml tubes containing 10 ml of YPD (10 g l<sup>−1</sup> yeast extract, 20 g l<sup>−1</sup> peptone, 20 g l<sup>−1</sup> glucose) (Oxoid) and incubated at 25 °C in an orbital shaker (Stuart Scientific) at 120 rpm for 24 h. Serial dilutions of 100 µl of cultures were spread plated onto Wallerstein Nutrient Agar (Oxoid) and incubated at 25 °C for 12 days in accordance with the EBC Yeast Giant Colony method 3.2.1.1<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 78" title="Richards, M. The use of giant‐colony morphology for the differentiation of brewing yeasts. J. Inst. Brew. 73, 162–166 (1967)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR78" id="ref-link-section-d893017906e3241">78</a></sup>. At the end of the incubation, single yeast colonies were selected.</p><h3 id="Sec13">DNA extraction and interdelta yeast typing</h3><p>Three giant colonies of each culture were selected and transferred to microfuge tubes containing 700 µl of molecular grade water (Fisher). Yeast cells were recovered by centrifugation and DNA was extracted in accordance with the manufacture’s guidelines using a PureLink Microbiome DNA Purification Kit (Invitrogen). Individual strains were identified using the interdelta (ITS) Polymerase chain reaction PCR method<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 14" title="Legras, J. L. &amp; Karst, F. Optimisation of interdelta analysis for Saccharomyces cerevisiae strain characterisation. FEMS Microbiol. Lett. 221, 249–255 (2003)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR14" id="ref-link-section-d893017906e3253">14</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 79" title="ASBC method yeast 13. American Society of Brewing Chemists. Report of subcommittee on differentiation of brewing yeast strains by PCR fingerprinting. J. Am. Soc. Brew. Chem. 66, 256–260 (2008)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR79" id="ref-link-section-d893017906e3256">79</a></sup>; primers δ2 (5′-GTGGATTTTTATTCCAAC-3′) and δ12 (5′-TCAACAATGGAATCCCAAC-3′), using a BioRad T100 Thermocycler and Invitrogen’s Platinum Hot start PCR Master Mix. PCR products were analysed on an Agilent 2100 Bioanalyzer using the Agilent DNA 7500 chip. The resulting bands were analysed using Minitab 19 Statistical software (2019) hierarchical clustering function with dendrograms produced using Euclidean distance function.</p><h3 id="Sec14">Illumina whole-genome sequencing and de novo assembly</h3><p>The ITS results were used to select strains for whole-genome sequencing with typical ITS banding used as the selection criterion for the historical Guinness and Irish brewing yeast. In the case of the FES and IDS production yeast all yeasts that were determined to be unique were selected. In total, 16 Guinness strains and 6 other historical Irish brewing yeast were subjected to whole-genome sequencing performed by Elda Biotech (Kildare, Ireland). Yeast samples were sub-cultured onto Wallerstein Nutrient Agar (Oxoid) and single colonies were picked for DNA extraction using the Thermo Scientific Yeast DNA extraction kit (Thermo Scientific). Extracted DNA was analysed using a Qubit (Thermo Fisher Scientific) to determine dsDNA content. Aliquots of 1 ng of DNA was used as input for library preparation using the Illumina Nextera XT DNA library prep protocol with no deviations. Stock libraries of 1–4 nM were generated and samples were pooled for sequencing and denatured according to the manufacturer’s instructions for loading on the Illumina MiSeq(12 pM) sequencer. Samples were sequenced using the Illumina MiSeq machine reading with a minimum depth of 30× coverage using 2 × 250 bp paired reads. All samples were quality checked for low-quality sequence bases and the presence of adaptor contamination using Trimgalore (Version 0.6.1). All identified adaptors were cleaved from both the forward and reverse sequencing reads and those with runs of low-quality bases were trimmed using a Phred scale cutoff of 10. All samples were aligned to the reference genome <i>S. cerevisiae</i> S288C (<a href="http://downloads.yeastgenome.org/sequence/S288C_reference/genome_releases/S288C_reference_genome_R64-1-1_20110203.tgz">http://downloads.yeastgenome.org/sequence/S288C_reference/genome_releases/S288C_reference_genome_R64-1-1_20110203.tgz</a>) using BWA (Version 0.7.17)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 80" title="Li, H. &amp; Durbin, R. Fast and accurate short read alignment with Burrows–Wheeler transform. Bioinformatics 25, 1754–1760 (2009)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR80" id="ref-link-section-d893017906e3278">80</a></sup>. Alignments were sorted and duplicate reads were identified and marked for exclusion from downstream analysis using Samtools (Version 1.10)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 81" title="Danecek, P. et al. Twelve years of SAMtools and BCFtools. Gigascience 10, giab008 (2021)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR81" id="ref-link-section-d893017906e3282">81</a></sup>. Alignment metrics for each sample were collated using Qualimap (Version 2.2.1)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 82" title="Okonechnikov, K., Conesa, A. &amp; García-Alcalde, F. Qualimap 2: advanced multi-sample quality control for high-throughput sequencing data. Bioinformatics 32, 292–294 (2016)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR82" id="ref-link-section-d893017906e3286">82</a></sup>. All samples were de novo assembled using Spades (Version 3.14). For each sample all contigs shorter than 500 base pairs in length were discarded. A reference guided scaffold of each assembled sample genome against the <i>Saccharomyces cerevisiae</i> S288C genome sequence was generated using Ragtag (v. 1.0.2)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 83" title="Alonge, M. et al. RaGOO: fast and accurate reference-guided scaffolding of draft genomes. Genome Biol. 20, 1–17 (2019)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR83" id="ref-link-section-d893017906e3294">83</a></sup> Artificial padding of “N” characters was placed between the reference scaffolded contigs. All bioinformatic software used in this study is specified in Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s42003-023-05587-3#MOESM2">1</a>.</p><h3 id="Sec15">Nanopore MINION sequencing</h3><p>Two nanopore Minion runs of Irish Draught Stout yeast number 1 (IDS1) were carried out by ELDA Biotech (Kildare, Ireland). DNA from an IDS1 colony grown on Wallerstein Nutrient Agar (Oxoid) was extract using the Thermo Scientific Yeast DNA extraction kit (Thermo Scientific). Extracted DNA was processed using the 1D Genomic DNA by ligation (SQK-LSK108) protocol from Oxford Nanopore Technologies. DNA was fragmented using a Covaris g-TUBE (Covaris) with DNA repair performed using End Prep (New England Biolabs). Library clean up and adaptor ligation were performed with AMPure XP beads (Beckman Coulter), and extracted DNA measured using a Qubit (Thermo Fisher Scientific). 3.6fmol of DNA library was loaded onto the flongle for sequencing (Oxford Nanopore Technologies). Sequencing was conducted according to the Nanopore Minion manufacturer’s instructions (Oxford Nanopore Technologies). All Minion run Fast5 files were converted to FASTQ format using Guppy (3.6). NanoFilt was used to remove low-quality reads (Q10) and reads shorter than 1000 bases, with Porechop v6<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 84" title="Murigneux, V. et al. Comparison of long-read methods for sequencing and assembly of a plant genome. GigaScience 9, giaa146 (2020)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR84" id="ref-link-section-d893017906e3309">84</a></sup> used to find and remove adaptors located at the start, end or internal reads. Contaminant (non-fungal) reads were identified using both Kraken2 and BLAST searches against the NCBI non-redundant database. Identified contaminant reads were removed using Seqtk (v1.3). The first Nanopore Minion run resulted in 421,040 usable reads and 64,105 in the second. Reads from both runs were combined for a unified assembly using Flye (v. 2.8)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 21" title="Kolmogorov, M. et al. metaFlye: scalable long-read metagenome assembly using repeat graphs. Nat. Methods 17, 1103–1110 (2020)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR21" id="ref-link-section-d893017906e3313">21</a></sup>. A corrected consensus sequence for the Flye assembly was generated using Medaka (v 1.0.3). Racon was used as a polishing tool for the Medaka consensus sequence using the previously generated IDS1 Illumina data and an assembly evaluation using Quast 5.10.0 was carried out using this nanopore assembled genome and the previously assembled Illumina only IDS1 genome. Gene content completeness of the assembled genome was estimated using Busco (v3)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 85" title="Waterhouse, R. M. et al. BUSCO applications from quality assessments to gene prediction and phylogenomics. Mol. Biol. Evol. 35, 543–548 (2018)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR85" id="ref-link-section-d893017906e3317">85</a></sup> with the assembled nanopore genome scaffolded against the <i>S. cerevisiae</i> S288C reference genome using Ragtag (v. 1.0.2)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 83" title="Alonge, M. et al. RaGOO: fast and accurate reference-guided scaffolding of draft genomes. Genome Biol. 20, 1–17 (2019)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR83" id="ref-link-section-d893017906e3324">83</a></sup>. Finally, the scaffolded assembly was annotated using Funannotate (v 1.74)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 86" title="Palmer, J. &amp; Stajich, J. nextgenusfs/funannotate: funannotate v1.5.3 (Version 1.5.3). Zenodo 
                  https://doi.org/10.5281/zenodo.2604804
                  
                 (2019)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR86" id="ref-link-section-d893017906e3329">86</a></sup>.</p><h3 id="Sec16">Determination of the Guinness yeast phylogeny</h3><p>Sequencing data for 154 <i>S. cerevisiae</i> samples<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 4" title="Gallone, B. et al. Domestication and divergence of Saccharomyces cerevisiae beer yeasts. Cell 166, 1397–1410 (2016)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR4" id="ref-link-section-d893017906e3345">4</a></sup> were retrieved from NCBI (BioProject PRJNA323691) and combined with the 22 <i>S. cerevisiae</i> sequenced for this investigation. All retrieved samples were quality checked for low-quality sequence bases and the presence of adaptor contamination using Trimgalore (Version 0.6.1). All identified adaptors were cleaved from both the forward and reverse sequencing reads and reads with runs of low-quality bases trimmed using a Phred scale cutoff of 10. All samples were aligned to <i>S. cerevisiae</i> S288C reference genome using BWA mem (version 0.7.17)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 80" title="Li, H. &amp; Durbin, R. Fast and accurate short read alignment with Burrows–Wheeler transform. Bioinformatics 25, 1754–1760 (2009)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR80" id="ref-link-section-d893017906e3355">80</a></sup>. Alignments were sorted and duplicate reads were identified and marked for exclusion from downstream analysis using Picard (Version 2.18.23). Alignment metrics for each sample were collated using Qualimap (Version 2.2.1)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 82" title="Okonechnikov, K., Conesa, A. &amp; García-Alcalde, F. Qualimap 2: advanced multi-sample quality control for high-throughput sequencing data. Bioinformatics 32, 292–294 (2016)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR82" id="ref-link-section-d893017906e3360">82</a></sup>. Misalignment of reads in original BWA alignments were corrected using GATK (Version 4.1.4-1)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 87" title="Van der Auwera, G. A. &amp; O’Connor, B. D. Genomics in the Cloud: Using Docker, GATK, and WDL in Terra (O’Reilly Media, 2020)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR87" id="ref-link-section-d893017906e3364">87</a></sup> with the base score recalibration carried out on the corrected alignments. SNP and Indel discovery and genotyping was performed across all 176 samples simultaneously with GATK used to filter sites based on the following metrics: quality score &gt;30, mapping scores &gt;40, read position rank sum &lt;8. All individual genotypes with less than 10× coverage were set to uncalled. Annotation and effect prediction for each variant was estimated using SnpEff (Version 4.3)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 88" title="Cingolani, P. et al. A program for annotating and predicting the effects of single nucleotide polymorphisms, SnpEff: SNPs in the genome of Drosophila melanogaster strain w1118; iso-2; iso-3. Fly 6, 80–92 (2012)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR88" id="ref-link-section-d893017906e3368">88</a></sup>.</p><p>Orthologous genes across all assembled genomes were inferred using Orthofinder (Version 2.3.3)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 89" title="Emms, D. M. &amp; Kelly, S. OrthoFinder: phylogenetic orthology inference for comparative genomics. Genome Biol. 20, 1–14 (2019)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR89" id="ref-link-section-d893017906e3375">89</a></sup>. Sequences from orthologous genes were concatenated and aligned using MUSCLE (Version 3.8.31). A phylogenetic analysis of the concatenated alignment of data from all orthologous genes was carried out using the maximum-likelihood approach implemented in RAxML (Version 8.2.4)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 15" title="Stamatakis, A. RAxML version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies. Bioinformatics 30, 1312–1313 (2014)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR15" id="ref-link-section-d893017906e3379">15</a></sup> based on the GTRGAMMA model of sequence evolution and a rapid bootstrap analysis for 1000 bootstrap replicates. The tree which was rooted using the outgroup species <i>S. paradoxus</i> was visualised and annotated using the ggtree<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 16" title="Yu, G., Lam, T. T. Y., Zhu, H. &amp; Guan, Y. Two methods for mapping and visualizing associated data on phylogeny using ggtree. Mol. Biol. Evol. 35, 3041–3043 (2018)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR16" id="ref-link-section-d893017906e3386">16</a></sup> package in R.</p><p>FastSTRUCTURE (Version 1.0)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 17" title="Raj, A., Stephens, M. &amp; Pritchard, J. K. fastSTRUCTURE: variational inference of population structure in large SNP data sets. Genetics 197, 573–589 (2014)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR17" id="ref-link-section-d893017906e3393">17</a></sup> was used to quantify the number of populations and the degree of admixture in the genomes examined in this study. Owing to the high degree of sequence similarity between the Guinness samples a single representative sample (IDS1) was used in this analysis consequently 161 genomes admixture were assessed. The full set of biallelic segregating sites identified across all samples was filtered based on a minor allele frequency (MAF) &lt; 0.05 and SNPs in linkage-disequilibrium, using PLINK (v1.09)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 90" title="Purcell, S. et al. PLINK: a tool set for whole-genome association and population-based linkage analyses. Am. J. Hum. Genet. 81, 559–575 (2007)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR90" id="ref-link-section-d893017906e3397">90</a></sup>. FastSTRUCTURE<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 17" title="Raj, A., Stephens, M. &amp; Pritchard, J. K. fastSTRUCTURE: variational inference of population structure in large SNP data sets. Genetics 197, 573–589 (2014)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR17" id="ref-link-section-d893017906e3401">17</a></sup> was run on a filtered set of SNPs, varying the number of ancestral populations (K) between 1 and 10 using the simple prior implemented in fastSTRUCTURE<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 17" title="Raj, A., Stephens, M. &amp; Pritchard, J. K. fastSTRUCTURE: variational inference of population structure in large SNP data sets. Genetics 197, 573–589 (2014)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR17" id="ref-link-section-d893017906e3405">17</a></sup> with <i>K</i> = 8 found to be optimal. The admixture of IDS1 was determined from the sequence data of the 154 <i>S. cerevisiae</i> samples<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 4" title="Gallone, B. et al. Domestication and divergence of Saccharomyces cerevisiae beer yeasts. Cell 166, 1397–1410 (2016)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR4" id="ref-link-section-d893017906e3416">4</a></sup> using Alpaca (v1)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 18" title="Salazar, A. N. &amp; Abeel, T., Alpaca: a kmer-based approach for investigating mosaic structures in microbial genomes. Preprint at bioRxiv 
                  https://doi.org/10.1101/551234
                  
                 (2019)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR18" id="ref-link-section-d893017906e3420">18</a></sup> and a kmer length of 21 over 5000 base pair sliding windows.</p><h3 id="Sec17">Copy number variation</h3><p>Analysis of the heterozygous biallelic SNPs for each Guinness yeast established variable copy number across the chromosomes and subsequently CNVs was normalised against an appropriate background copy number for each strain. In addition, CNVs was estimated in 250 base pair non-overlapping windows across the entire ~12 million bases of the <i>S. cerevisiae</i> genome using Control-FREEC (Version 5.7)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 91" title="Boeva, V. et al. Control-FREEC: a tool for assessing copy number and allelic content using next-generation sequencing data. Bioinformatics 28, 423–425 (2012)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR91" id="ref-link-section-d893017906e3435">91</a></sup>. Plots depicting CNVs for the Guinness yeasts were generated in R using publicly available code<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 92" title="Fijarczyk, A. et al. The genome sequence of the Jean-Talon strain, an archeological beer yeast from Québec, reveals traces of adaptation to specific brewing conditions. G3 Genes Genomes Genet. 10, 3087–3097 (2020)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR92" id="ref-link-section-d893017906e3439">92</a></sup>.</p><h3 id="Sec18">Sporulation</h3><p>The sporulation potential of the different Guinness yeast strains was assessed using the ASBC Yeast 7 sporulation method<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 93" title="ASBC method yeast 7. American Society of Brewing Chemists. Report of Subcommittee on Microbiology. J. Am. Soc. Brew. Chem. 43, 16 (1985)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR93" id="ref-link-section-d893017906e3451">93</a></sup>. A total of 1000 cells per sample were examined using a Nikon Eclipse C<i>i</i> microscope 100× magnification. Ascospores stained green to blue green while vegetative yeast cells-stained pink to red. independent triplicate analyses were performed for each strain. The incidence of sporulation was expressed as a percentage.</p><h3 id="Sec19">Assessment of fermentation properties</h3><p>Fermentation ability was assessed using 180 ml mini fermenters (Fisher) containing 120 ml of 12<sup>o</sup>P wort. Cultures were recovered from liquid nitrogen and sufficient yeast for the experiments generated by successive serial aerobic incubations in 10 ml YPD, 90 ml 12<sup>o</sup>P wort and 900 ml 12<sup>o</sup>P wort. A single batch of all-malt hopped wort was used for all experiments to eliminate batch to batch variation. Wort was produced in the Guinness Pilot plant and stored at −20 °C in 5 l aliquots. Prior to use it was thawed and sterilised by autoclaving.</p><p>Yeast cells were recovered by centrifugation and washed three times by successive suspension in distilled water and re-centrifugation. Viability and yeast cell concentration of each culture was determined using the EBC methods, EBC 3.1.1.1 Haemocytometer<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 94" title="Jorgensen A. &amp; Hansen A. Mikroorganismen der Gärungsindustrie 7 Aufl (Verlag Hans Carl, Nürnberg, 1956)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR94" id="ref-link-section-d893017906e3475">94</a></sup> and EBC 3.2.1.1 Methylene Blue<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 95" title="Smart, K. A., Chambers, K. M., Lambert, I. &amp; Jenkins, C. J. Am. Soc. Brew. Chem. 57, 18 (1999)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR95" id="ref-link-section-d893017906e3479">95</a></sup>. Triplicate fermentations were inoculated with 1 × 10<sup>7</sup> viable yeast cells per ml into 180 ml mini fermenters containing 120 ml of air-saturated 12<sup>o</sup>P wort. Fermentations were incubated at 25 °C and stirred continuously using a stirrer plate (mix 15 eco plate Camlab) set at 250 rpm. Mini fermenters were sealed with a butyl rubber plug secured with an aluminium cap (Fisher) and fitted with a Bunsen valve to allow CO<sub>2</sub> to be released. Fermentation progression was measured by periodically monitoring weight loss. The endpoint was established when three successive identical readings were recorded.</p><h3 id="Sec20">Analysis of fermentation metabolites</h3><p>Concentrations of selected yeast-derived flavour compounds were measured using a gas chromatographic procedure using a modified version of the EBC Vicinal Diketone method Analytica-EBC Method 9.24.2<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 96" title="Buckee, G. K. &amp; Mundy, A. P. Determination of vicinal diketones in beer by gas chromatography (headspace technique) – collaborative trial. J. Inst. Brew. 100, 247–253 (1994)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR96" id="ref-link-section-d893017906e3497">96</a></sup>. End-fermentation samples (30 ml) previously clarified by centrifugation were transferred to McCartney bottles which after sealing were heated at 65 °C for 30 min to convert precursor α-acetolactate into free diacetyl. Diacetyl (2,3 butanedione) concentration was determined using an ECD detector; with esters and higher alcohol concentrations determined using an FID detector. Peak areas for the metabolites were normalised using appropriate internal standards.</p><h3 id="Sec21">Analysis for phenolic off-flavour (4-vinyl guaiacol, 4-VG)</h3><p>The ability of yeast to produce 4-vinyl guaiacol was determined according to Analytica-EBC Method 2.3.9.5<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 97" title="Brewers of Europe. Phenolic off Flavour Method. Analytica-EBC 2.3.9.5. (2018)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR97" id="ref-link-section-d893017906e3509">97</a></sup> phenolic off-flavour method using gas chromatography mass spectrometry. Washed yeast samples were inoculated at a concentration of 1 × 10<sup>6</sup> viable cells ml<sup>−1</sup> into 25 ml tubes containing 10 mls of YPD medium supplemented with 0.1 ml of ferulic acid (hydroxycinnamic acid) solution. Triplicate incubations were performed for each yeast strain. After incubation at 25 °C for 48 h. 5 ml was transferred to an autosampler vial (Fisher) containing 2 µl of the internal standard containing: 4-vinyl guaiacol (Sigma–Aldrich). Analyses were carried out using an Agilent 6890/7890 GC systems fitted with a Zebron ZB-Wax 60.0 m × 250.00 μm × 0.25μm column. The initial oven temperature was 60 °C. After 10 min this was increased to 220 °C at a rate of 10 °C min<sup>−1</sup> then held for 2 min. 4-vinyl guaiacol concentration was determined using an ECB detector; temperature 150 °C, make-up flow rate of 60 ml min<sup>−1</sup> (Helium gas). Peak area for 4-vinyl guaiacol were normalised against the internal 4-vinyl guaiacol standard.</p><h3 id="Sec22">Alcohol concentration</h3><p>Ethanol concentration was determined using near infrared spectroscopy using an Anton Paar Alcolyser in accordance with the manufacture’s guidelines.</p><h3 id="Sec23">Sugar concentration</h3><p>Samples were analysed tested using an Agilent 1260 Infinity II system with a refractive index detector (Infinity II 1260 WR RID) and a Zorbax Carbohydrate column (4.6 × 250 mm, 5 µm, P/N: 840300-908). The other acquisition conditions were as follows: mobile phase was a 70/30 mix of acetonitrile and water; sample injection volume was 50 µL; flow rate was isocratic and set at 1.5 ml min<sup>−1</sup>. The column oven was kept at a constant 35 °C. No internal standard was used. However, samples were bracketed either side with freshly made known standard.</p><h3 id="Sec24">Flocculation</h3><p>Flocculation was assessed using EBC Gilliland method EBC 3.5.31<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 48" title="Gilliland, R. B. Proc. European Brewery Convention, Brighton (1951)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR48" id="ref-link-section-d893017906e3548">48</a></sup>. The EBC method uses visual inspection of flocculation behaviour categorising the yeast using prescribed classifications: Class 1 non-flocculant, Class 2 slightly flocculant, Class 3 moderately flocculant and Class 4 highly flocculant. An addendum to the EBC method was the addition of four control yeasts representing the different classifications.</p><h3 id="Sec25">1HL fermentations</h3><p>In order to prepare sufficient beer for taste testing 1 hL fermentations using selected yeast strains were carried out using the Guinness pilot scale plant. Standard Guinness wort was taken from the St James’ Gate Brewery and diluted to 12<sup>o</sup>P with deaerated water and autoclaved prior to use. Yeast strains were retrieved from long-term liquid nitrogen and propagated by successive serial aerobic incubations in 10 ml YPD, 90 ml 12<sup>o</sup>P wort and 900 ml 12<sup>o</sup>P wort. To ensure that sufficient yeast was available the terminal cultures were prepared in a Carlsberg flask (GEA) containing 15 litres of sterile 12<sup>o</sup>P Guinness wort and incubated at 25 °C for 48 h with continuous oxygenation. This generated sufficient yeast to inoculate 80 L of wort at an initial count of 1 × 10<sup>7</sup> viable yeast cells ml<sup>−1</sup>. Fermentations were attemperated at 22 °C. After the desired final gravity was achieved the beer was held at 25 °C for 24 h to allow removal of diacetyl. A 20 L sample of beer was then removed and transferred to a sterile keg. After storage at 4 °C for 48 h the beer was clarified by passage through a sheet filter then bottled and pasteurised (25 PU). Triplicate samples of the beers were analysed for alcohol, fermentation metabolites, and POF production using an Anton Paar Alcolyser, gas chromatographic procedure using a modified version of the EBC Vicinal Diketone method Analytica-EBC Method 9.24.2<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 96" title="Buckee, G. K. &amp; Mundy, A. P. Determination of vicinal diketones in beer by gas chromatography (headspace technique) – collaborative trial. J. Inst. Brew. 100, 247–253 (1994)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR96" id="ref-link-section-d893017906e3573">96</a></sup> and Analytica-EBC Method 2.3.9.5<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 97" title="Brewers of Europe. Phenolic off Flavour Method. Analytica-EBC 2.3.9.5. (2018)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR97" id="ref-link-section-d893017906e3577">97</a></sup>, phenolic off-flavour method using gas chromatography mass spectrometry. Organoleptic properties were assessed by a trained beer sensory panel consisting of 18 members. Using Quantitative Descriptive methodology<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 98" title="dos Santos Navarro, R. D. C. et al. Optimized Descriptive Profile: a rapid methodology for sensory description. Food Qual. Preference 24, 190–200 (2012)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR98" id="ref-link-section-d893017906e3581">98</a></sup> and a list of predefined Guinness Stout sensory attributes all three samples were tested in duplicate by the panel in a single tasting session. Individual sensory attributes were rated on a linear scale (0–10)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 99" title="Lawless, H. T. &amp; Heymann, H. Sensory Evaluation of Food: Principles and Practices Vol. 2 (Springer, New York, 2010)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR99" id="ref-link-section-d893017906e3585">99</a></sup>, with a subset of these attributes identified to explain differences and similarities across samples. The samples were randomised and presented to the taste panel labelled with a three-digit code. Sensory scores were analysed using a 2-way ANOVA including sensory assessor’s as a random factor<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 99" title="Lawless, H. T. &amp; Heymann, H. Sensory Evaluation of Food: Principles and Practices Vol. 2 (Springer, New York, 2010)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR99" id="ref-link-section-d893017906e3589">99</a></sup>.</p><h3 id="Sec26">Statistics and reproducibility</h3><p>Statistical analyses were preformed using Minitab 19 Statistical software (2019) and Xlstat 20 Excel statistical package (2020). Mini-fermentations were performed using three independent biological replicates and statistical significance of ethanol production, fermentation metabolites and POF determined using one way ANOVAs. The organoleptic properties of the 1Hl fermentation Guinness brews were determined by Quantitative Descriptive methodology<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 98" title="dos Santos Navarro, R. D. C. et al. Optimized Descriptive Profile: a rapid methodology for sensory description. Food Qual. Preference 24, 190–200 (2012)." href="https://www.nature.com/articles/s42003-023-05587-3#ref-CR98" id="ref-link-section-d893017906e3601">98</a></sup> using a trained taste panel of 18 independent members, and the resulting data analysed using 2-way ANOVA. <i>T-</i>test were performed on isobutanol production of IDS1 and IDS2. The effects of CNV on POF production was established using <i>f</i> and <i>t-tests</i>.</p><h3 id="Sec27">Reporting summary</h3><p>Further information on research design is available in the <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s42003-023-05587-3#MOESM9">Nature Portfolio Reporting Summary</a> linked to this article.</p></div></div></section>
                </div>
            

            <div>
            <section data-title="Data availability"><div id="data-availability-section"><h2 id="data-availability">Data availability</h2><p>Illumina and Nanopore (basecalled, demultiplexed) reads for all sequenced samples in this manuscript are deposited in the European Nucleotide Archive (ENA) under the project accession PRJEB62101. All experimental data underlying figures are presented in Supplementary Data <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s42003-023-05587-3#MOESM8">5</a>.</p></div></section><div id="MagazineFulltextArticleBodySuffix"><section aria-labelledby="Bib1" data-title="References"><div id="Bib1-section"><h2 id="Bib1">References</h2><div id="Bib1-content"><div data-container-section="references"><ol data-track-component="outbound reference"><li data-counter="1."><p id="ref-CR1">Diamond, J. Evolution, consequences and future of plant and animal domestication. <i>Nature</i> <b>418</b>, 700–707 (2002).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/nature01019" data-track-action="article reference" href="https://doi.org/10.1038%2Fnature01019" aria-label="Article reference 1" data-doi="10.1038/nature01019">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD38XlvVyltbY%3D" aria-label="CAS reference 1">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=12167878" aria-label="PubMed reference 1">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 1" href="http://scholar.google.com/scholar_lookup?&amp;title=Evolution%2C%20consequences%20and%20future%20of%20plant%20and%20animal%20domestication&amp;journal=Nature&amp;doi=10.1038%2Fnature01019&amp;volume=418&amp;pages=700-707&amp;publication_year=2002&amp;author=Diamond%2CJ">
                    Google Scholar</a> 
                </p></li><li data-counter="2."><p id="ref-CR2">Steensels, J., Gallone, B., Voordeckers, K. &amp; Verstrepen, K. J. Domestication of industrial microbes. <i>Curr. Biol.</i> <b>29</b>, R381–R393 (2019).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.cub.2019.04.025" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.cub.2019.04.025" aria-label="Article reference 2" data-doi="10.1016/j.cub.2019.04.025">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhtVSnu7%2FN" aria-label="CAS reference 2">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31112692" aria-label="PubMed reference 2">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 2" href="http://scholar.google.com/scholar_lookup?&amp;title=Domestication%20of%20industrial%20microbes&amp;journal=Curr.%20Biol.&amp;doi=10.1016%2Fj.cub.2019.04.025&amp;volume=29&amp;pages=R381-R393&amp;publication_year=2019&amp;author=Steensels%2CJ&amp;author=Gallone%2CB&amp;author=Voordeckers%2CK&amp;author=Verstrepen%2CKJ">
                    Google Scholar</a> 
                </p></li><li data-counter="3."><p id="ref-CR3">Peter, J. et al. Genome evolution across 1011 Saccharomyces cerevisiae isolates. <i>Nature</i> <b>556</b>, 339–344 (2018).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/s41586-018-0030-5" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41586-018-0030-5" aria-label="Article reference 3" data-doi="10.1038/s41586-018-0030-5">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXosVCgt70%3D" aria-label="CAS reference 3">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29643504" aria-label="PubMed reference 3">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6784862" aria-label="PubMed Central reference 3">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 3" href="http://scholar.google.com/scholar_lookup?&amp;title=Genome%20evolution%20across%201011%20Saccharomyces%20cerevisiae%20isolates&amp;journal=Nature&amp;doi=10.1038%2Fs41586-018-0030-5&amp;volume=556&amp;pages=339-344&amp;publication_year=2018&amp;author=Peter%2CJ">
                    Google Scholar</a> 
                </p></li><li data-counter="4."><p id="ref-CR4">Gallone, B. et al. Domestication and divergence of Saccharomyces cerevisiae beer yeasts. <i>Cell</i> <b>166</b>, 1397–1410 (2016).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.cell.2016.08.020" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.cell.2016.08.020" aria-label="Article reference 4" data-doi="10.1016/j.cell.2016.08.020">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XhsFSltL%2FJ" aria-label="CAS reference 4">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27610566" aria-label="PubMed reference 4">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5018251" aria-label="PubMed Central reference 4">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 4" href="http://scholar.google.com/scholar_lookup?&amp;title=Domestication%20and%20divergence%20of%20Saccharomyces%20cerevisiae%20beer%20yeasts&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2016.08.020&amp;volume=166&amp;pages=1397-1410&amp;publication_year=2016&amp;author=Gallone%2CB">
                    Google Scholar</a> 
                </p></li><li data-counter="5."><p id="ref-CR5">Botstein, D. &amp; Fink, G. R. Yeast: an experimental organism for 21st Century biology. <i>Genetics</i> <b>189</b>, 695–704 (2011).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1534/genetics.111.130765" data-track-action="article reference" href="https://doi.org/10.1534%2Fgenetics.111.130765" aria-label="Article reference 5" data-doi="10.1534/genetics.111.130765">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XitFKgu7c%3D" aria-label="CAS reference 5">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22084421" aria-label="PubMed reference 5">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3213361" aria-label="PubMed Central reference 5">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 5" href="http://scholar.google.com/scholar_lookup?&amp;title=Yeast%3A%20an%20experimental%20organism%20for%2021st%20Century%20biology&amp;journal=Genetics&amp;doi=10.1534%2Fgenetics.111.130765&amp;volume=189&amp;pages=695-704&amp;publication_year=2011&amp;author=Botstein%2CD&amp;author=Fink%2CGR">
                    Google Scholar</a> 
                </p></li><li data-counter="6."><p id="ref-CR6">Johnston, M. Genome sequencing: the complete code for a eukaryotic cell. <i>Curr. Biol.</i> <b>6</b>, 500–503 (1996).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/S0960-9822(02)00526-2" data-track-action="article reference" href="https://doi.org/10.1016%2FS0960-9822%2802%2900526-2" aria-label="Article reference 6" data-doi="10.1016/S0960-9822(02)00526-2">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK28XjtFCrsbs%3D" aria-label="CAS reference 6">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=8805271" aria-label="PubMed reference 6">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 6" href="http://scholar.google.com/scholar_lookup?&amp;title=Genome%20sequencing%3A%20the%20complete%20code%20for%20a%20eukaryotic%20cell&amp;journal=Curr.%20Biol.&amp;doi=10.1016%2FS0960-9822%2802%2900526-2&amp;volume=6&amp;pages=500-503&amp;publication_year=1996&amp;author=Johnston%2CM">
                    Google Scholar</a> 
                </p></li><li data-counter="7."><p id="ref-CR7">Peris, D. et al. Macroevolutionary diversity of traits and genomes in the model yeast genus Saccharomyces. <i>Nat. Commun.</i> <b>14</b>, 690 (2023).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/s41467-023-36139-2" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41467-023-36139-2" aria-label="Article reference 7" data-doi="10.1038/s41467-023-36139-2">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3sXjtVegt78%3D" aria-label="CAS reference 7">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=36755033" aria-label="PubMed reference 7">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9908912" aria-label="PubMed Central reference 7">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 7" href="http://scholar.google.com/scholar_lookup?&amp;title=Macroevolutionary%20diversity%20of%20traits%20and%20genomes%20in%20the%20model%20yeast%20genus%20Saccharomyces&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-023-36139-2&amp;volume=14&amp;publication_year=2023&amp;author=Peris%2CD">
                    Google Scholar</a> 
                </p></li><li data-counter="8."><p id="ref-CR8">Gonçalves, M. et al. Distinct domestication trajectories in top-fermenting beer yeasts and wine yeasts. <i>Curr. Biol.</i> <b>26</b>, 2750–2761 (2016).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.cub.2016.08.040" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.cub.2016.08.040" aria-label="Article reference 8" data-doi="10.1016/j.cub.2016.08.040">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27720622" aria-label="PubMed reference 8">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 8" href="http://scholar.google.com/scholar_lookup?&amp;title=Distinct%20domestication%20trajectories%20in%20top-fermenting%20beer%20yeasts%20and%20wine%20yeasts&amp;journal=Curr.%20Biol.&amp;doi=10.1016%2Fj.cub.2016.08.040&amp;volume=26&amp;pages=2750-2761&amp;publication_year=2016&amp;author=Gon%C3%A7alves%2CM">
                    Google Scholar</a> 
                </p></li><li data-counter="9."><p id="ref-CR9">Fay, J. C. et al. A polyploid admixed origin of beer yeasts derived from European and Asian wine populations. <i>PLoS Biol.</i> <b>17</b>, e3000147 (2019).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1371/journal.pbio.3000147" data-track-action="article reference" href="https://doi.org/10.1371%2Fjournal.pbio.3000147" aria-label="Article reference 9" data-doi="10.1371/journal.pbio.3000147">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhtFemu7nF" aria-label="CAS reference 9">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30835725" aria-label="PubMed reference 9">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6400334" aria-label="PubMed Central reference 9">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 9" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20polyploid%20admixed%20origin%20of%20beer%20yeasts%20derived%20from%20European%20and%20Asian%20wine%20populations&amp;journal=PLoS%20Biol.&amp;doi=10.1371%2Fjournal.pbio.3000147&amp;volume=17&amp;publication_year=2019&amp;author=Fay%2CJC">
                    Google Scholar</a> 
                </p></li><li data-counter="10."><p id="ref-CR10">Abou Saada, O. et al. Phased polyploid genomes provide deeper insight into the multiple origins of domesticated Saccharomyces cerevisiae beer yeasts. <i>Curr. Biol.</i> <b>32</b>, 1350–1361 (2022).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.cub.2022.01.068" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.cub.2022.01.068" aria-label="Article reference 10" data-doi="10.1016/j.cub.2022.01.068">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 10" href="http://scholar.google.com/scholar_lookup?&amp;title=Phased%20polyploid%20genomes%20provide%20deeper%20insight%20into%20the%20multiple%20origins%20of%20domesticated%20Saccharomyces%20cerevisiae%20beer%20yeasts&amp;journal=Curr.%20Biol.&amp;doi=10.1016%2Fj.cub.2022.01.068&amp;volume=32&amp;pages=1350-1361&amp;publication_year=2022&amp;author=Abou%20Saada%2CO">
                    Google Scholar</a> 
                </p></li><li data-counter="11."><p id="ref-CR11">Boulton, C. &amp; Quain, D. <i>Brewing Yeast and Fermentation</i> (John Wiley &amp; Sons, 2008).</p></li><li data-counter="12."><p id="ref-CR12">Preiss, R., Tyrawa, C., Krogerus, K., Garshol, L. M. &amp; Van Der Merwe, G. Traditional Norwegian Kveik are a genetically distinct group of domesticated Saccharomyces cerevisiae brewing yeasts. <i>Front. Microbiol.</i> <b>9</b>, 2137 (2018).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.3389/fmicb.2018.02137" data-track-action="article reference" href="https://doi.org/10.3389%2Ffmicb.2018.02137" aria-label="Article reference 12" data-doi="10.3389/fmicb.2018.02137">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30258422" aria-label="PubMed reference 12">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6145013" aria-label="PubMed Central reference 12">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 12" href="http://scholar.google.com/scholar_lookup?&amp;title=Traditional%20Norwegian%20Kveik%20are%20a%20genetically%20distinct%20group%20of%20domesticated%20Saccharomyces%20cerevisiae%20brewing%20yeasts&amp;journal=Front.%20Microbiol.&amp;doi=10.3389%2Ffmicb.2018.02137&amp;volume=9&amp;publication_year=2018&amp;author=Preiss%2CR&amp;author=Tyrawa%2CC&amp;author=Krogerus%2CK&amp;author=Garshol%2CLM&amp;author=Merwe%2CG">
                    Google Scholar</a> 
                </p></li><li data-counter="13."><p id="ref-CR13">GDB/FN05/0002 Articles of Association. Two copies of the Memorandum and Articles of Association of Arthur Guinness Son and Company, <i>Guinness Archives.</i> (1886).</p></li><li data-counter="14."><p id="ref-CR14">Legras, J. L. &amp; Karst, F. Optimisation of interdelta analysis for Saccharomyces cerevisiae strain characterisation. <i>FEMS Microbiol. Lett.</i> <b>221</b>, 249–255 (2003).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/S0378-1097(03)00205-2" data-track-action="article reference" href="https://doi.org/10.1016%2FS0378-1097%2803%2900205-2" aria-label="Article reference 14" data-doi="10.1016/S0378-1097(03)00205-2">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3sXjtlSlu7w%3D" aria-label="CAS reference 14">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=12725935" aria-label="PubMed reference 14">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 14" href="http://scholar.google.com/scholar_lookup?&amp;title=Optimisation%20of%20interdelta%20analysis%20for%20Saccharomyces%20cerevisiae%20strain%20characterisation&amp;journal=FEMS%20Microbiol.%20Lett.&amp;doi=10.1016%2FS0378-1097%2803%2900205-2&amp;volume=221&amp;pages=249-255&amp;publication_year=2003&amp;author=Legras%2CJL&amp;author=Karst%2CF">
                    Google Scholar</a> 
                </p></li><li data-counter="15."><p id="ref-CR15">Stamatakis, A. RAxML version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies. <i>Bioinformatics</i> <b>30</b>, 1312–1313 (2014).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1093/bioinformatics/btu033" data-track-action="article reference" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtu033" aria-label="Article reference 15" data-doi="10.1093/bioinformatics/btu033">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXmvFCjsbc%3D" aria-label="CAS reference 15">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24451623" aria-label="PubMed reference 15">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3998144" aria-label="PubMed Central reference 15">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 15" href="http://scholar.google.com/scholar_lookup?&amp;title=RAxML%20version%208%3A%20a%20tool%20for%20phylogenetic%20analysis%20and%20post-analysis%20of%20large%20phylogenies&amp;journal=Bioinformatics&amp;doi=10.1093%2Fbioinformatics%2Fbtu033&amp;volume=30&amp;pages=1312-1313&amp;publication_year=2014&amp;author=Stamatakis%2CA">
                    Google Scholar</a> 
                </p></li><li data-counter="16."><p id="ref-CR16">Yu, G., Lam, T. T. Y., Zhu, H. &amp; Guan, Y. Two methods for mapping and visualizing associated data on phylogeny using ggtree. <i>Mol. Biol. Evol.</i> <b>35</b>, 3041–3043 (2018).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1093/molbev/msy194" data-track-action="article reference" href="https://doi.org/10.1093%2Fmolbev%2Fmsy194" aria-label="Article reference 16" data-doi="10.1093/molbev/msy194">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhtFKntrbF" aria-label="CAS reference 16">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30351396" aria-label="PubMed reference 16">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6278858" aria-label="PubMed Central reference 16">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 16" href="http://scholar.google.com/scholar_lookup?&amp;title=Two%20methods%20for%20mapping%20and%20visualizing%20associated%20data%20on%20phylogeny%20using%20ggtree&amp;journal=Mol.%20Biol.%20Evol.&amp;doi=10.1093%2Fmolbev%2Fmsy194&amp;volume=35&amp;pages=3041-3043&amp;publication_year=2018&amp;author=Yu%2CG&amp;author=Lam%2CTTY&amp;author=Zhu%2CH&amp;author=Guan%2CY">
                    Google Scholar</a> 
                </p></li><li data-counter="17."><p id="ref-CR17">Raj, A., Stephens, M. &amp; Pritchard, J. K. fastSTRUCTURE: variational inference of population structure in large SNP data sets. <i>Genetics</i> <b>197</b>, 573–589 (2014).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1534/genetics.114.164350" data-track-action="article reference" href="https://doi.org/10.1534%2Fgenetics.114.164350" aria-label="Article reference 17" data-doi="10.1534/genetics.114.164350">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24700103" aria-label="PubMed reference 17">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4063916" aria-label="PubMed Central reference 17">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 17" href="http://scholar.google.com/scholar_lookup?&amp;title=fastSTRUCTURE%3A%20variational%20inference%20of%20population%20structure%20in%20large%20SNP%20data%20sets&amp;journal=Genetics&amp;doi=10.1534%2Fgenetics.114.164350&amp;volume=197&amp;pages=573-589&amp;publication_year=2014&amp;author=Raj%2CA&amp;author=Stephens%2CM&amp;author=Pritchard%2CJK">
                    Google Scholar</a> 
                </p></li><li data-counter="18."><p id="ref-CR18">Salazar, A. N. &amp; Abeel, T., Alpaca: a kmer-based approach for investigating mosaic structures in microbial genomes. Preprint at <i>bioRxiv</i> <a href="https://doi.org/10.1101/551234" data-track="click" data-track-action="external reference" data-track-label="10.1101/551234">https://doi.org/10.1101/551234</a> (2019).</p></li><li data-counter="19."><p id="ref-CR19">Salazar, A. N. et al. Nanopore sequencing enables near-complete de novo assembly of Saccharomyces cerevisiae reference strain CEN. PK113-7D. <i>FEMS Yeast Res.</i> <b>17</b>, p.fox074 (2017).</p></li><li data-counter="20."><p id="ref-CR20">Nijkamp, J. F. et al. De novo sequencing, assembly and analysis of the genome of the laboratory strain Saccharomyces cerevisiae CEN. PK113-7D, a model for modern industrial biotechnology. <i>Microb. Cell Factories</i> <b>11</b>, 1–17 (2012).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1186/1475-2859-11-36" data-track-action="article reference" href="https://doi.org/10.1186%2F1475-2859-11-36" aria-label="Article reference 20" data-doi="10.1186/1475-2859-11-36">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 20" href="http://scholar.google.com/scholar_lookup?&amp;title=De%20novo%20sequencing%2C%20assembly%20and%20analysis%20of%20the%20genome%20of%20the%20laboratory%20strain%20Saccharomyces%20cerevisiae%20CEN.%20PK113-7D%2C%20a%20model%20for%20modern%20industrial%20biotechnology&amp;journal=Microb.%20Cell%20Factories&amp;doi=10.1186%2F1475-2859-11-36&amp;volume=11&amp;pages=1-17&amp;publication_year=2012&amp;author=Nijkamp%2CJF">
                    Google Scholar</a> 
                </p></li><li data-counter="21."><p id="ref-CR21">Kolmogorov, M. et al. metaFlye: scalable long-read metagenome assembly using repeat graphs. <i>Nat. Methods</i> <b>17</b>, 1103–1110 (2020).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/s41592-020-00971-x" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41592-020-00971-x" aria-label="Article reference 21" data-doi="10.1038/s41592-020-00971-x">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhvF2jtb7I" aria-label="CAS reference 21">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33020656" aria-label="PubMed reference 21">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10699202" aria-label="PubMed Central reference 21">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 21" href="http://scholar.google.com/scholar_lookup?&amp;title=metaFlye%3A%20scalable%20long-read%20metagenome%20assembly%20using%20repeat%20graphs&amp;journal=Nat.%20Methods&amp;doi=10.1038%2Fs41592-020-00971-x&amp;volume=17&amp;pages=1103-1110&amp;publication_year=2020&amp;author=Kolmogorov%2CM">
                    Google Scholar</a> 
                </p></li><li data-counter="22."><p id="ref-CR22">Morard, M. et al. Aneuploidy and ethanol tolerance in Saccharomyces cerevisiae. <i>Front. Genet.</i> <b>10</b>, 82 (2019).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.3389/fgene.2019.00082" data-track-action="article reference" href="https://doi.org/10.3389%2Ffgene.2019.00082" aria-label="Article reference 22" data-doi="10.3389/fgene.2019.00082">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhtlWms7%2FF" aria-label="CAS reference 22">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30809248" aria-label="PubMed reference 22">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6379819" aria-label="PubMed Central reference 22">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 22" href="http://scholar.google.com/scholar_lookup?&amp;title=Aneuploidy%20and%20ethanol%20tolerance%20in%20Saccharomyces%20cerevisiae&amp;journal=Front.%20Genet.&amp;doi=10.3389%2Ffgene.2019.00082&amp;volume=10&amp;publication_year=2019&amp;author=Morard%2CM">
                    Google Scholar</a> 
                </p></li><li data-counter="23."><p id="ref-CR23">Bilinski, C. A. &amp; Casey, G. P. Developments in sporulation and breeding of brewer’s yeast. <i>Yeast</i> <b>5</b>, 429–438 (1989).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1002/yea.320050603" data-track-action="article reference" href="https://doi.org/10.1002%2Fyea.320050603" aria-label="Article reference 23" data-doi="10.1002/yea.320050603">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK3cXhvFykug%3D%3D" aria-label="CAS reference 23">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 23" href="http://scholar.google.com/scholar_lookup?&amp;title=Developments%20in%20sporulation%20and%20breeding%20of%20brewer%E2%80%99s%20yeast&amp;journal=Yeast&amp;doi=10.1002%2Fyea.320050603&amp;volume=5&amp;pages=429-438&amp;publication_year=1989&amp;author=Bilinski%2CCA&amp;author=Casey%2CGP">
                    Google Scholar</a> 
                </p></li><li data-counter="24."><p id="ref-CR24">Briggs, D. E., Brookes, P. A., Boulton, C. A. &amp; Stevens, R. <i>Brewing: Science and Practice</i> Vol. 108 (Woodhead Publ., 2004).</p></li><li data-counter="25."><p id="ref-CR25">Charron, M. J., Read, E., Haut, S. R. &amp; Michels, C. A. Molecular evolution of the telomere-associated MAL loci of Saccharomyces. <i>Genetics</i> <b>122</b>, 307–316 (1989).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1093/genetics/122.2.307" data-track-action="article reference" href="https://doi.org/10.1093%2Fgenetics%2F122.2.307" aria-label="Article reference 25" data-doi="10.1093/genetics/122.2.307">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaL1MXltFSjt78%3D" aria-label="CAS reference 25">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=2548922" aria-label="PubMed reference 25">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1203703" aria-label="PubMed Central reference 25">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 25" href="http://scholar.google.com/scholar_lookup?&amp;title=Molecular%20evolution%20of%20the%20telomere-associated%20MAL%20loci%20of%20Saccharomyces&amp;journal=Genetics&amp;doi=10.1093%2Fgenetics%2F122.2.307&amp;volume=122&amp;pages=307-316&amp;publication_year=1989&amp;author=Charron%2CMJ&amp;author=Read%2CE&amp;author=Haut%2CSR&amp;author=Michels%2CCA">
                    Google Scholar</a> 
                </p></li><li data-counter="26."><p id="ref-CR26">Naumov, G. I., Naumova, E. S. &amp; Michels, C. A. Genetic variation of the repeated MAL loci in natural populations of Saccharomyces cerevisiae and Saccharomyces paradoxus. <i>Genetics</i> <b>136</b>, 803–812 (1994).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1093/genetics/136.3.803" data-track-action="article reference" href="https://doi.org/10.1093%2Fgenetics%2F136.3.803" aria-label="Article reference 26" data-doi="10.1093/genetics/136.3.803">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK2MXpsFWq" aria-label="CAS reference 26">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=8005435" aria-label="PubMed reference 26">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1205886" aria-label="PubMed Central reference 26">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 26" href="http://scholar.google.com/scholar_lookup?&amp;title=Genetic%20variation%20of%20the%20repeated%20MAL%20loci%20in%20natural%20populations%20of%20Saccharomyces%20cerevisiae%20and%20Saccharomyces%20paradoxus&amp;journal=Genetics&amp;doi=10.1093%2Fgenetics%2F136.3.803&amp;volume=136&amp;pages=803-812&amp;publication_year=1994&amp;author=Naumov%2CGI&amp;author=Naumova%2CES&amp;author=Michels%2CCA">
                    Google Scholar</a> 
                </p></li><li data-counter="27."><p id="ref-CR27">Meilgaard, M. C., Reid, D. S. &amp; Wyborski, K. A. Reference standards for beer flavor terminology system. <i>J. Am. Soc. Brew. Chem.</i> <b>40</b>, 119–128 (1982).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaL3sXltFyltg%3D%3D" aria-label="CAS reference 27">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 27" href="http://scholar.google.com/scholar_lookup?&amp;title=Reference%20standards%20for%20beer%20flavor%20terminology%20system&amp;journal=J.%20Am.%20Soc.%20Brew.%20Chem.&amp;volume=40&amp;pages=119-128&amp;publication_year=1982&amp;author=Meilgaard%2CMC&amp;author=Reid%2CDS&amp;author=Wyborski%2CKA">
                    Google Scholar</a> 
                </p></li><li data-counter="28."><p id="ref-CR28">Pires, E. J., Teixeira, J. A., Brányik, T. &amp; Vicente, A. A. Yeast: the soul of beer’s aroma—a review of flavour-active esters and higher alcohols produced by the brewing yeast. <i>Appl. Microbiol. Biotechnol.</i> <b>98</b>, 1937–1949 (2014).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1007/s00253-013-5470-0" data-track-action="article reference" href="https://doi.org/10.1007%2Fs00253-013-5470-0" aria-label="Article reference 28" data-doi="10.1007/s00253-013-5470-0">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXhvVaquw%3D%3D" aria-label="CAS reference 28">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24384752" aria-label="PubMed reference 28">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 28" href="http://scholar.google.com/scholar_lookup?&amp;title=Yeast%3A%20the%20soul%20of%20beer%E2%80%99s%20aroma%E2%80%94a%20review%20of%20flavour-active%20esters%20and%20higher%20alcohols%20produced%20by%20the%20brewing%20yeast&amp;journal=Appl.%20Microbiol.%20Biotechnol.&amp;doi=10.1007%2Fs00253-013-5470-0&amp;volume=98&amp;pages=1937-1949&amp;publication_year=2014&amp;author=Pires%2CEJ&amp;author=Teixeira%2CJA&amp;author=Br%C3%A1nyik%2CT&amp;author=Vicente%2CAA">
                    Google Scholar</a> 
                </p></li><li data-counter="29."><p id="ref-CR29">Lilly, M., Bauer, F. F., Styger, G., Lambrechts, M. G. &amp; Pretorius, I. S. The effect of increased branched-chain amino acid transaminase activity in yeast on the production of higher alcohols and on the flavour profiles of wine and distillates. <i>FEMS Yeast Res.</i> <b>6</b>, 726–743 (2006).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1111/j.1567-1364.2006.00057.x" data-track-action="article reference" href="https://doi.org/10.1111%2Fj.1567-1364.2006.00057.x" aria-label="Article reference 29" data-doi="10.1111/j.1567-1364.2006.00057.x">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD28Xotlymsbc%3D" aria-label="CAS reference 29">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16879424" aria-label="PubMed reference 29">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 29" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20effect%20of%20increased%20branched-chain%20amino%20acid%20transaminase%20activity%20in%20yeast%20on%20the%20production%20of%20higher%20alcohols%20and%20on%20the%20flavour%20profiles%20of%20wine%20and%20distillates&amp;journal=FEMS%20Yeast%20Res.&amp;doi=10.1111%2Fj.1567-1364.2006.00057.x&amp;volume=6&amp;pages=726-743&amp;publication_year=2006&amp;author=Lilly%2CM&amp;author=Bauer%2CFF&amp;author=Styger%2CG&amp;author=Lambrechts%2CMG&amp;author=Pretorius%2CIS">
                    Google Scholar</a> 
                </p></li><li data-counter="30."><p id="ref-CR30">Styger, G., Jacobson, D., Prior, B. A. &amp; Bauer, F. F. Genetic analysis of the metabolic pathways responsible for aroma metabolite production by Saccharomyces cerevisiae. <i>Appl. Microbiol. Biotechnol.</i> <b>97</b>, 4429–4442 (2013).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1007/s00253-012-4522-1" data-track-action="article reference" href="https://doi.org/10.1007%2Fs00253-012-4522-1" aria-label="Article reference 30" data-doi="10.1007/s00253-012-4522-1">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXntFyrtb4%3D" aria-label="CAS reference 30">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23111598" aria-label="PubMed reference 30">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 30" href="http://scholar.google.com/scholar_lookup?&amp;title=Genetic%20analysis%20of%20the%20metabolic%20pathways%20responsible%20for%20aroma%20metabolite%20production%20by%20Saccharomyces%20cerevisiae&amp;journal=Appl.%20Microbiol.%20Biotechnol.&amp;doi=10.1007%2Fs00253-012-4522-1&amp;volume=97&amp;pages=4429-4442&amp;publication_year=2013&amp;author=Styger%2CG&amp;author=Jacobson%2CD&amp;author=Prior%2CBA&amp;author=Bauer%2CFF">
                    Google Scholar</a> 
                </p></li><li data-counter="31."><p id="ref-CR31">Gibson, B., Vidgren, V., Peddinti, G. &amp; Krogerus, K. Diacetyl control during brewery fermentation via adaptive laboratory engineering of the lager yeast Saccharomyces pastorianus. <i>J. Ind. Microbiol. Biotechnol.</i> <b>45</b>, 1103–1112 (2018).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1007/s10295-018-2087-4" data-track-action="article reference" href="https://doi.org/10.1007%2Fs10295-018-2087-4" aria-label="Article reference 31" data-doi="10.1007/s10295-018-2087-4">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhvV2ns7zI" aria-label="CAS reference 31">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30306366" aria-label="PubMed reference 31">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 31" href="http://scholar.google.com/scholar_lookup?&amp;title=Diacetyl%20control%20during%20brewery%20fermentation%20via%20adaptive%20laboratory%20engineering%20of%20the%20lager%20yeast%20Saccharomyces%20pastorianus&amp;journal=J.%20Ind.%20Microbiol.%20Biotechnol.&amp;doi=10.1007%2Fs10295-018-2087-4&amp;volume=45&amp;pages=1103-1112&amp;publication_year=2018&amp;author=Gibson%2CB&amp;author=Vidgren%2CV&amp;author=Peddinti%2CG&amp;author=Krogerus%2CK">
                    Google Scholar</a> 
                </p></li><li data-counter="32."><p id="ref-CR32">Krogerus, K. &amp; Gibson, B. R. 125th anniversary review: diacetyl and its control during brewery fermentation. <i>J. Inst. Brew.</i> <b>119</b>, 86–97 (2013).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXhsFGgurnM" aria-label="CAS reference 32">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 32" href="http://scholar.google.com/scholar_lookup?&amp;title=125th%20anniversary%20review%3A%20diacetyl%20and%20its%20control%20during%20brewery%20fermentation&amp;journal=J.%20Inst.%20Brew.&amp;volume=119&amp;pages=86-97&amp;publication_year=2013&amp;author=Krogerus%2CK&amp;author=Gibson%2CBR">
                    Google Scholar</a> 
                </p></li><li data-counter="33."><p id="ref-CR33">Cullin, C., Baudin‐Baillieu, A., Guillemet, E. &amp; Ozier‐Kalogeropoulos, O. Functional analysis of YCL09C: evidence for a role as the regulatory subunit of acetolactate synthase. <i>Yeast</i> <b>12</b>, 1511–1518 (1996).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1002/(SICI)1097-0061(199612)12:15&lt;1511::AID-YEA41&gt;3.0.CO;2-B" data-track-action="article reference" href="https://doi.org/10.1002%2F%28SICI%291097-0061%28199612%2912%3A15%3C1511%3A%3AAID-YEA41%3E3.0.CO%3B2-B" aria-label="Article reference 33" data-doi="10.1002/(SICI)1097-0061(199612)12:15&lt;1511::AID-YEA41&gt;3.0.CO;2-B">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK2sXitVCjsA%3D%3D" aria-label="CAS reference 33">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=8972574" aria-label="PubMed reference 33">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 33" href="http://scholar.google.com/scholar_lookup?&amp;title=Functional%20analysis%20of%20YCL09C%3A%20evidence%20for%20a%20role%20as%20the%20regulatory%20subunit%20of%20acetolactate%20synthase&amp;journal=Yeast&amp;doi=10.1002%2F%28SICI%291097-0061%28199612%2912%3A15%3C1511%3A%3AAID-YEA41%3E3.0.CO%3B2-B&amp;volume=12&amp;pages=1511-1518&amp;publication_year=1996&amp;author=Cullin%2CC&amp;author=Baudin%E2%80%90Baillieu%2CA&amp;author=Guillemet%2CE&amp;author=Ozier%E2%80%90Kalogeropoulos%2CO">
                    Google Scholar</a> 
                </p></li><li data-counter="34."><p id="ref-CR34">Falco, S. C., Dumas, K. S. &amp; Livak, K. J. Nucleotide sequence of the yeast ILV2 gene which encodes acetolactate synthase. <i>Nucleic acids Res.</i> <b>13</b>, 4011–4027 (1985).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1093/nar/13.11.4011" data-track-action="article reference" href="https://doi.org/10.1093%2Fnar%2F13.11.4011" aria-label="Article reference 34" data-doi="10.1093/nar/13.11.4011">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaL2MXkvValsr8%3D" aria-label="CAS reference 34">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=2989783" aria-label="PubMed reference 34">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC341293" aria-label="PubMed Central reference 34">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 34" href="http://scholar.google.com/scholar_lookup?&amp;title=Nucleotide%20sequence%20of%20the%20yeast%20ILV2%20gene%20which%20encodes%20acetolactate%20synthase&amp;journal=Nucleic%20acids%20Res.&amp;doi=10.1093%2Fnar%2F13.11.4011&amp;volume=13&amp;pages=4011-4027&amp;publication_year=1985&amp;author=Falco%2CSC&amp;author=Dumas%2CKS&amp;author=Livak%2CKJ">
                    Google Scholar</a> 
                </p></li><li data-counter="35."><p id="ref-CR35">Heidlas, J. &amp; Tressl, R. Purification and properties of two oxidoreductases catalyzing the enantioselective reduction of diacetyl and other diketones from baker’s yeast. <i>Eur. J. Biochem.</i> <b>188</b>, 165–174 (1990).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1111/j.1432-1033.1990.tb15384.x" data-track-action="article reference" href="https://doi.org/10.1111%2Fj.1432-1033.1990.tb15384.x" aria-label="Article reference 35" data-doi="10.1111/j.1432-1033.1990.tb15384.x">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK3cXhsFCitLY%3D" aria-label="CAS reference 35">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=2180695" aria-label="PubMed reference 35">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 35" href="http://scholar.google.com/scholar_lookup?&amp;title=Purification%20and%20properties%20of%20two%20oxidoreductases%20catalyzing%20the%20enantioselective%20reduction%20of%20diacetyl%20and%20other%20diketones%20from%20baker%E2%80%99s%20yeast&amp;journal=Eur.%20J.%20Biochem.&amp;doi=10.1111%2Fj.1432-1033.1990.tb15384.x&amp;volume=188&amp;pages=165-174&amp;publication_year=1990&amp;author=Heidlas%2CJ&amp;author=Tressl%2CR">
                    Google Scholar</a> 
                </p></li><li data-counter="36."><p id="ref-CR36">Katz, M., Hahn-Hägerdal, B. &amp; Gorwa-Grauslund, M. F. Screening of two complementary collections of Saccharomyces cerevisiae to identify enzymes involved in stereo-selective reductions of specific carbonyl compounds: an alternative to protein purification. <i>Enzym. Microb. Technol.</i> <b>33</b>, 163–172 (2003).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/S0141-0229(03)00086-3" data-track-action="article reference" href="https://doi.org/10.1016%2FS0141-0229%2803%2900086-3" aria-label="Article reference 36" data-doi="10.1016/S0141-0229(03)00086-3">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3sXlvVWju74%3D" aria-label="CAS reference 36">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 36" href="http://scholar.google.com/scholar_lookup?&amp;title=Screening%20of%20two%20complementary%20collections%20of%20Saccharomyces%20cerevisiae%20to%20identify%20enzymes%20involved%20in%20stereo-selective%20reductions%20of%20specific%20carbonyl%20compounds%3A%20an%20alternative%20to%20protein%20purification&amp;journal=Enzym.%20Microb.%20Technol.&amp;doi=10.1016%2FS0141-0229%2803%2900086-3&amp;volume=33&amp;pages=163-172&amp;publication_year=2003&amp;author=Katz%2CM&amp;author=Hahn-H%C3%A4gerdal%2CB&amp;author=Gorwa-Grauslund%2CMF">
                    Google Scholar</a> 
                </p></li><li data-counter="37."><p id="ref-CR37">Brumsted, D., Lauterbach, A. F. &amp; West, D. B. Phenolic characteristics in brewing 111. The role of yeast. In <i>Proc. American Society of Brewing Chemists Annual Meeting, St Louis</i> 142–145 (Am. Soc. Brewing Chemists, Madison, WI, 1965).</p></li><li data-counter="38."><p id="ref-CR38">Madigan, D., McMurrough, I. &amp; Smyth, M. R. Rapid determination of 4-vinyl guaiacol and ferulic acid in beers and worts by high-performance liquid chromatography. <i>J. Am. Soc. Brew. Chem.</i> <b>52</b>, 152–155 (1994).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK2MXivVaktrs%3D" aria-label="CAS reference 38">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 38" href="http://scholar.google.com/scholar_lookup?&amp;title=Rapid%20determination%20of%204-vinyl%20guaiacol%20and%20ferulic%20acid%20in%20beers%20and%20worts%20by%20high-performance%20liquid%20chromatography&amp;journal=J.%20Am.%20Soc.%20Brew.%20Chem.&amp;volume=52&amp;pages=152-155&amp;publication_year=1994&amp;author=Madigan%2CD&amp;author=McMurrough%2CI&amp;author=Smyth%2CMR">
                    Google Scholar</a> 
                </p></li><li data-counter="39."><p id="ref-CR39">Maillard, M. N. &amp; Berset, C. Evolution of antioxidant activity during kilning: role of insoluble bound phenolic acids of barley and malt. <i>J. Agric. Food Chem.</i> <b>43</b>, 1789–1793 (1995).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1021/jf00055a008" data-track-action="article reference" href="https://doi.org/10.1021%2Fjf00055a008" aria-label="Article reference 39" data-doi="10.1021/jf00055a008">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK2MXmsFCrsb4%3D" aria-label="CAS reference 39">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 39" href="http://scholar.google.com/scholar_lookup?&amp;title=Evolution%20of%20antioxidant%20activity%20during%20kilning%3A%20role%20of%20insoluble%20bound%20phenolic%20acids%20of%20barley%20and%20malt&amp;journal=J.%20Agric.%20Food%20Chem.&amp;doi=10.1021%2Fjf00055a008&amp;volume=43&amp;pages=1789-1793&amp;publication_year=1995&amp;author=Maillard%2CMN&amp;author=Berset%2CC">
                    Google Scholar</a> 
                </p></li><li data-counter="40."><p id="ref-CR40">Mukai, N., Masaki, K., Fujii, T. &amp; Iefuji, H. Single nucleotide polymorphisms of PAD1 and FDC1 show a positive relationship with ferulic acid decarboxylation ability among industrial yeasts used in alcoholic beverage production. <i>J. Biosci. Bioeng.</i> <b>118</b>, 50–55 (2014).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.jbiosc.2013.12.017" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.jbiosc.2013.12.017" aria-label="Article reference 40" data-doi="10.1016/j.jbiosc.2013.12.017">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXhvVahsLo%3D" aria-label="CAS reference 40">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24507903" aria-label="PubMed reference 40">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 40" href="http://scholar.google.com/scholar_lookup?&amp;title=Single%20nucleotide%20polymorphisms%20of%20PAD1%20and%20FDC1%20show%20a%20positive%20relationship%20with%20ferulic%20acid%20decarboxylation%20ability%20among%20industrial%20yeasts%20used%20in%20alcoholic%20beverage%20production&amp;journal=J.%20Biosci.%20Bioeng.&amp;doi=10.1016%2Fj.jbiosc.2013.12.017&amp;volume=118&amp;pages=50-55&amp;publication_year=2014&amp;author=Mukai%2CN&amp;author=Masaki%2CK&amp;author=Fujii%2CT&amp;author=Iefuji%2CH">
                    Google Scholar</a> 
                </p></li><li data-counter="41."><p id="ref-CR41">Vidgren, V. &amp; Londesborough, J. 125th anniversary review: yeast flocculation and sedimentation in brewing. <i>J. Inst. Brew.</i> <b>117</b>, 475–487 (2011).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1002/j.2050-0416.2011.tb00495.x" data-track-action="article reference" href="https://doi.org/10.1002%2Fj.2050-0416.2011.tb00495.x" aria-label="Article reference 41" data-doi="10.1002/j.2050-0416.2011.tb00495.x">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XnvVOiur4%3D" aria-label="CAS reference 41">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 41" href="http://scholar.google.com/scholar_lookup?&amp;title=125th%20anniversary%20review%3A%20yeast%20flocculation%20and%20sedimentation%20in%20brewing&amp;journal=J.%20Inst.%20Brew.&amp;doi=10.1002%2Fj.2050-0416.2011.tb00495.x&amp;volume=117&amp;pages=475-487&amp;publication_year=2011&amp;author=Vidgren%2CV&amp;author=Londesborough%2CJ">
                    Google Scholar</a> 
                </p></li><li data-counter="42."><p id="ref-CR42">Fichtner, L., Schulze, F. &amp; Braus, G. H. Differential Flo8p‐dependent regulation of FLO1 and FLO11 for cell–cell and cell–substrate adherence of S. cerevisiae S288C. <i>Mol. Microbiol.</i> <b>66</b>, 1276–1289 (2007).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1111/j.1365-2958.2007.06014.x" data-track-action="article reference" href="https://doi.org/10.1111%2Fj.1365-2958.2007.06014.x" aria-label="Article reference 42" data-doi="10.1111/j.1365-2958.2007.06014.x">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2sXhsVeku7rM" aria-label="CAS reference 42">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18001350" aria-label="PubMed reference 42">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2780560" aria-label="PubMed Central reference 42">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 42" href="http://scholar.google.com/scholar_lookup?&amp;title=Differential%20Flo8p%E2%80%90dependent%20regulation%20of%20FLO1%20and%20FLO11%20for%20cell%E2%80%93cell%20and%20cell%E2%80%93substrate%20adherence%20of%20S.%20cerevisiae%20S288C&amp;journal=Mol.%20Microbiol.&amp;doi=10.1111%2Fj.1365-2958.2007.06014.x&amp;volume=66&amp;pages=1276-1289&amp;publication_year=2007&amp;author=Fichtner%2CL&amp;author=Schulze%2CF&amp;author=Braus%2CGH">
                    Google Scholar</a> 
                </p></li><li data-counter="43."><p id="ref-CR43">Stratford, M. Yeast flocculation: calcium specificity. <i>Yeast</i> <b>5</b>, 487–496 (1989).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1002/yea.320050608" data-track-action="article reference" href="https://doi.org/10.1002%2Fyea.320050608" aria-label="Article reference 43" data-doi="10.1002/yea.320050608">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK3cXhvVyiug%3D%3D" aria-label="CAS reference 43">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 43" href="http://scholar.google.com/scholar_lookup?&amp;title=Yeast%20flocculation%3A%20calcium%20specificity&amp;journal=Yeast&amp;doi=10.1002%2Fyea.320050608&amp;volume=5&amp;pages=487-496&amp;publication_year=1989&amp;author=Stratford%2CM">
                    Google Scholar</a> 
                </p></li><li data-counter="44."><p id="ref-CR44">Stratford, M. Yeast flocculation: a new perspective. <i>Adv. Microb. Physiol.</i> <b>33</b>, 1–71 (1992).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/S0065-2911(08)60215-5" data-track-action="article reference" href="https://doi.org/10.1016%2FS0065-2911%2808%2960215-5" aria-label="Article reference 44" data-doi="10.1016/S0065-2911(08)60215-5">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK3sXhtVSgu7k%3D" aria-label="CAS reference 44">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 44" href="http://scholar.google.com/scholar_lookup?&amp;title=Yeast%20flocculation%3A%20a%20new%20perspective&amp;journal=Adv.%20Microb.%20Physiol.&amp;doi=10.1016%2FS0065-2911%2808%2960215-5&amp;volume=33&amp;pages=1-71&amp;publication_year=1992&amp;author=Stratford%2CM">
                    Google Scholar</a> 
                </p></li><li data-counter="45."><p id="ref-CR45">Straver, M. H., Aar, P. C., Smit, G. &amp; Kijne, J. W. Determinants of flocculence of brewer’s yeast during fermentation in wort. <i>Yeast</i> <b>9</b>, 527–532 (1993).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1002/yea.320090509" data-track-action="article reference" href="https://doi.org/10.1002%2Fyea.320090509" aria-label="Article reference 45" data-doi="10.1002/yea.320090509">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK3sXkslWhsLc%3D" aria-label="CAS reference 45">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=8322515" aria-label="PubMed reference 45">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 45" href="http://scholar.google.com/scholar_lookup?&amp;title=Determinants%20of%20flocculence%20of%20brewer%E2%80%99s%20yeast%20during%20fermentation%20in%20wort&amp;journal=Yeast&amp;doi=10.1002%2Fyea.320090509&amp;volume=9&amp;pages=527-532&amp;publication_year=1993&amp;author=Straver%2CMH&amp;author=Aar%2CPC&amp;author=Smit%2CG&amp;author=Kijne%2CJW">
                    Google Scholar</a> 
                </p></li><li data-counter="46."><p id="ref-CR46">Stratford, M. Induction of flocculation in brewing yeasts by change in pH value. <i>FEMS Microbiol. Lett.</i> <b>136</b>, 13–18 (1996).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1111/j.1574-6968.1996.tb08018.x" data-track-action="article reference" href="https://doi.org/10.1111%2Fj.1574-6968.1996.tb08018.x" aria-label="Article reference 46" data-doi="10.1111/j.1574-6968.1996.tb08018.x">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK28XhsFOhur4%3D" aria-label="CAS reference 46">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=8919449" aria-label="PubMed reference 46">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 46" href="http://scholar.google.com/scholar_lookup?&amp;title=Induction%20of%20flocculation%20in%20brewing%20yeasts%20by%20change%20in%20pH%20value&amp;journal=FEMS%20Microbiol.%20Lett.&amp;doi=10.1111%2Fj.1574-6968.1996.tb08018.x&amp;volume=136&amp;pages=13-18&amp;publication_year=1996&amp;author=Stratford%2CM">
                    Google Scholar</a> 
                </p></li><li data-counter="47."><p id="ref-CR47">Goossens, K. &amp; Willaert, R. Flocculation protein structure and cell–cell adhesion mechanism in Saccharomyces cerevisiae. <i>Biotechnol. Lett.</i> <b>32</b>, 1571–1585 (2010).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1007/s10529-010-0352-3" data-track-action="article reference" href="https://doi.org/10.1007%2Fs10529-010-0352-3" aria-label="Article reference 47" data-doi="10.1007/s10529-010-0352-3">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXht1KgsrzE" aria-label="CAS reference 47">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20640875" aria-label="PubMed reference 47">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 47" href="http://scholar.google.com/scholar_lookup?&amp;title=Flocculation%20protein%20structure%20and%20cell%E2%80%93cell%20adhesion%20mechanism%20in%20Saccharomyces%20cerevisiae&amp;journal=Biotechnol.%20Lett.&amp;doi=10.1007%2Fs10529-010-0352-3&amp;volume=32&amp;pages=1571-1585&amp;publication_year=2010&amp;author=Goossens%2CK&amp;author=Willaert%2CR">
                    Google Scholar</a> 
                </p></li><li data-counter="48."><p id="ref-CR48">Gilliland, R. B. <i>Proc. European Brewery Convention, Brighton</i> (1951).</p></li><li data-counter="49."><p id="ref-CR49">Verstrepen, K. J., Jansen, A., Lewitter, F. &amp; Fink, G. R. Intragenic tandem repeats generate functional variability. <i>Nat. Genet.</i> <b>37</b>, 986–990 (2005).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/ng1618" data-track-action="article reference" href="https://doi.org/10.1038%2Fng1618" aria-label="Article reference 49" data-doi="10.1038/ng1618">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2MXpsFWisLk%3D" aria-label="CAS reference 49">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16086015" aria-label="PubMed reference 49">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1462868" aria-label="PubMed Central reference 49">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 49" href="http://scholar.google.com/scholar_lookup?&amp;title=Intragenic%20tandem%20repeats%20generate%20functional%20variability&amp;journal=Nat.%20Genet.&amp;doi=10.1038%2Fng1618&amp;volume=37&amp;pages=986-990&amp;publication_year=2005&amp;author=Verstrepen%2CKJ&amp;author=Jansen%2CA&amp;author=Lewitter%2CF&amp;author=Fink%2CGR">
                    Google Scholar</a> 
                </p></li><li data-counter="50."><p id="ref-CR50">Watari, J., Nomura, M., Sahara, H., Koshino, S. &amp; Keränen, S. Construction of flocculent brewer’s yeast by chromosomal integration of the yeast flocculation gene FLO1. <i>J. Inst. Brew.</i> <b>100</b>, 73–77 (1994).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1002/j.2050-0416.1994.tb00809.x" data-track-action="article reference" href="https://doi.org/10.1002%2Fj.2050-0416.1994.tb00809.x" aria-label="Article reference 50" data-doi="10.1002/j.2050-0416.1994.tb00809.x">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK2MXmtFartA%3D%3D" aria-label="CAS reference 50">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 50" href="http://scholar.google.com/scholar_lookup?&amp;title=Construction%20of%20flocculent%20brewer%E2%80%99s%20yeast%20by%20chromosomal%20integration%20of%20the%20yeast%20flocculation%20gene%20FLO1&amp;journal=J.%20Inst.%20Brew.&amp;doi=10.1002%2Fj.2050-0416.1994.tb00809.x&amp;volume=100&amp;pages=73-77&amp;publication_year=1994&amp;author=Watari%2CJ&amp;author=Nomura%2CM&amp;author=Sahara%2CH&amp;author=Koshino%2CS&amp;author=Ker%C3%A4nen%2CS">
                    Google Scholar</a> 
                </p></li><li data-counter="51."><p id="ref-CR51">Lo, W. S. &amp; Dranginis, A. FLO11, a yeast gene related to the STA genes, encodes a novel cell surface flocculin. <i>J. Bacteriol.</i> <b>178</b>, 7144–7151 (1996).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1128/jb.178.24.7144-7151.1996" data-track-action="article reference" href="https://doi.org/10.1128%2Fjb.178.24.7144-7151.1996" aria-label="Article reference 51" data-doi="10.1128/jb.178.24.7144-7151.1996">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK2sXhtlWhtg%3D%3D" aria-label="CAS reference 51">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=8955395" aria-label="PubMed reference 51">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC178626" aria-label="PubMed Central reference 51">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 51" href="http://scholar.google.com/scholar_lookup?&amp;title=FLO11%2C%20a%20yeast%20gene%20related%20to%20the%20STA%20genes%2C%20encodes%20a%20novel%20cell%20surface%20flocculin&amp;journal=J.%20Bacteriol.&amp;doi=10.1128%2Fjb.178.24.7144-7151.1996&amp;volume=178&amp;pages=7144-7151&amp;publication_year=1996&amp;author=Lo%2CWS&amp;author=Dranginis%2CA">
                    Google Scholar</a> 
                </p></li><li data-counter="52."><p id="ref-CR52">Anderson, R. One yeast or two? Pure yeast and top fermentation. <i>Brew. Hist.</i> <b>149</b>, 30–38 (2012).</p><p><a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 52" href="http://scholar.google.com/scholar_lookup?&amp;title=One%20yeast%20or%20two%3F%20Pure%20yeast%20and%20top%20fermentation&amp;journal=Brew.%20Hist.&amp;volume=149&amp;pages=30-38&amp;publication_year=2012&amp;author=Anderson%2CR">
                    Google Scholar</a> 
                </p></li><li data-counter="53."><p id="ref-CR53">GDB/RD02/0365. File No 521A - Visits in Great Britain and Ireland 1902-11-21-1978-12-07. In <i>Guinness Archives.</i></p></li><li data-counter="54."><p id="ref-CR54">Russell, I. &amp; Stewart, G. G. Liquid nitrogen storage of yeast cultures compared to more traditional storage methods. <i>J. Am. Soc. Brew. Chem.</i> <b>39</b>, 19–24 (1981).</p><p><a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 54" href="http://scholar.google.com/scholar_lookup?&amp;title=Liquid%20nitrogen%20storage%20of%20yeast%20cultures%20compared%20to%20more%20traditional%20storage%20methods&amp;journal=J.%20Am.%20Soc.%20Brew.%20Chem.&amp;volume=39&amp;pages=19-24&amp;publication_year=1981&amp;author=Russell%2CI&amp;author=Stewart%2CGG">
                    Google Scholar</a> 
                </p></li><li data-counter="55."><p id="ref-CR55">Donnelly, D. &amp; Hurley, J. Yeast monitoring: the Guinness experience. <i>Ferment</i> <b>9</b>, 283–286 (1996).</p><p><a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 55" href="http://scholar.google.com/scholar_lookup?&amp;title=Yeast%20monitoring%3A%20the%20Guinness%20experience&amp;journal=Ferment&amp;volume=9&amp;pages=283-286&amp;publication_year=1996&amp;author=Donnelly%2CD&amp;author=Hurley%2CJ">
                    Google Scholar</a> 
                </p></li><li data-counter="56."><p id="ref-CR56">Berry, D. R. &amp; Slaughter, J. C., Alcoholic beverage fermentations. In <i>Fermented beverage production</i>. 25-39 (Springer, Boston, MA, US, 2003).</p></li><li data-counter="57."><p id="ref-CR57">Ekberg, J. et al. Adaptive evolution of the lager brewing yeast Saccharomyces pastorianus for improved growth under hyperosmotic conditions and its influence on fermentation performance. <i>FEMS Yeast Res.</i> <b>13</b>, 335–349 (2013).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1111/1567-1364.12038" data-track-action="article reference" href="https://doi.org/10.1111%2F1567-1364.12038" aria-label="Article reference 57" data-doi="10.1111/1567-1364.12038">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXntlers7g%3D" aria-label="CAS reference 57">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23414064" aria-label="PubMed reference 57">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 57" href="http://scholar.google.com/scholar_lookup?&amp;title=Adaptive%20evolution%20of%20the%20lager%20brewing%20yeast%20Saccharomyces%20pastorianus%20for%20improved%20growth%20under%20hyperosmotic%20conditions%20and%20its%20influence%20on%20fermentation%20performance&amp;journal=FEMS%20Yeast%20Res.&amp;doi=10.1111%2F1567-1364.12038&amp;volume=13&amp;pages=335-349&amp;publication_year=2013&amp;author=Ekberg%2CJ">
                    Google Scholar</a> 
                </p></li><li data-counter="58."><p id="ref-CR58">Gibson, B. et al. Adaptive laboratory evolution of ale and lager yeasts for improved brewing efficiency and beer quality. <i>Annu. Rev. Food Sci. Technol.</i> <b>11</b>, 23–44 (2020).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1146/annurev-food-032519-051715" data-track-action="article reference" href="https://doi.org/10.1146%2Fannurev-food-032519-051715" aria-label="Article reference 58" data-doi="10.1146/annurev-food-032519-051715">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:STN:280:DC%2BB38%2FhsV2nsw%3D%3D" aria-label="CAS reference 58">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31951488" aria-label="PubMed reference 58">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 58" href="http://scholar.google.com/scholar_lookup?&amp;title=Adaptive%20laboratory%20evolution%20of%20ale%20and%20lager%20yeasts%20for%20improved%20brewing%20efficiency%20and%20beer%20quality&amp;journal=Annu.%20Rev.%20Food%20Sci.%20Technol.&amp;doi=10.1146%2Fannurev-food-032519-051715&amp;volume=11&amp;pages=23-44&amp;publication_year=2020&amp;author=Gibson%2CB">
                    Google Scholar</a> 
                </p></li><li data-counter="59."><p id="ref-CR59">Albertin, W. et al. Evidence for autotetraploidy associated with reproductive isolation in Saccharomyces cerevisiae: towards a new domesticated species. <i>J. Evolut. Biol.</i> <b>22</b>, 2157–2170 (2009).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1111/j.1420-9101.2009.01828.x" data-track-action="article reference" href="https://doi.org/10.1111%2Fj.1420-9101.2009.01828.x" aria-label="Article reference 59" data-doi="10.1111/j.1420-9101.2009.01828.x">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:STN:280:DC%2BC3c%2FhtlymsA%3D%3D" aria-label="CAS reference 59">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 59" href="http://scholar.google.com/scholar_lookup?&amp;title=Evidence%20for%20autotetraploidy%20associated%20with%20reproductive%20isolation%20in%20Saccharomyces%20cerevisiae%3A%20towards%20a%20new%20domesticated%20species&amp;journal=J.%20Evolut.%20Biol.&amp;doi=10.1111%2Fj.1420-9101.2009.01828.x&amp;volume=22&amp;pages=2157-2170&amp;publication_year=2009&amp;author=Albertin%2CW">
                    Google Scholar</a> 
                </p></li><li data-counter="60."><p id="ref-CR60">Voordeckers, K. &amp; Verstrepen, K. J. Experimental evolution of the model eukaryote Saccharomyces cerevisiae yields insight into the molecular mechanisms underlying adaptation. <i>Curr. Opin. Microbiol.</i> <b>28</b>, 1–9 (2015).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.mib.2015.06.018" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.mib.2015.06.018" aria-label="Article reference 60" data-doi="10.1016/j.mib.2015.06.018">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXhtF2mt7%2FP" aria-label="CAS reference 60">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26202939" aria-label="PubMed reference 60">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 60" href="http://scholar.google.com/scholar_lookup?&amp;title=Experimental%20evolution%20of%20the%20model%20eukaryote%20Saccharomyces%20cerevisiae%20yields%20insight%20into%20the%20molecular%20mechanisms%20underlying%20adaptation&amp;journal=Curr.%20Opin.%20Microbiol.&amp;doi=10.1016%2Fj.mib.2015.06.018&amp;volume=28&amp;pages=1-9&amp;publication_year=2015&amp;author=Voordeckers%2CK&amp;author=Verstrepen%2CKJ">
                    Google Scholar</a> 
                </p></li><li data-counter="61."><p id="ref-CR61">Yona, A. H. et al. Chromosomal duplication is a transient evolutionary solution to stress. <i>Proc. Natl Acad. Sci.</i> <b>109</b>, 21010–21015 (2012).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1073/pnas.1211150109" data-track-action="article reference" href="https://doi.org/10.1073%2Fpnas.1211150109" aria-label="Article reference 61" data-doi="10.1073/pnas.1211150109">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXnt1emsg%3D%3D" aria-label="CAS reference 61">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23197825" aria-label="PubMed reference 61">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3529009" aria-label="PubMed Central reference 61">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 61" href="http://scholar.google.com/scholar_lookup?&amp;title=Chromosomal%20duplication%20is%20a%20transient%20evolutionary%20solution%20to%20stress&amp;journal=Proc.%20Natl%20Acad.%20Sci.&amp;doi=10.1073%2Fpnas.1211150109&amp;volume=109&amp;pages=21010-21015&amp;publication_year=2012&amp;author=Yona%2CAH">
                    Google Scholar</a> 
                </p></li><li data-counter="62."><p id="ref-CR62">Linder, R. A., Greco, J. P., Seidl, F., Matsui, T. &amp; Ehrenreich, I. M. The stress-inducible peroxidase TSA2 underlies a conditionally beneficial chromosomal duplication in Saccharomyces cerevisiae. <i>G3 Genes Genomes Genet.</i> <b>7</b>, 3177–3184 (2017).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1534/g3.117.300069" data-track-action="article reference" href="https://doi.org/10.1534%2Fg3.117.300069" aria-label="Article reference 62" data-doi="10.1534/g3.117.300069">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXisVOrtrzO" aria-label="CAS reference 62">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 62" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20stress-inducible%20peroxidase%20TSA2%20underlies%20a%20conditionally%20beneficial%20chromosomal%20duplication%20in%20Saccharomyces%20cerevisiae&amp;journal=G3%20Genes%20Genomes%20Genet.&amp;doi=10.1534%2Fg3.117.300069&amp;volume=7&amp;pages=3177-3184&amp;publication_year=2017&amp;author=Linder%2CRA&amp;author=Greco%2CJP&amp;author=Seidl%2CF&amp;author=Matsui%2CT&amp;author=Ehrenreich%2CIM">
                    Google Scholar</a> 
                </p></li><li data-counter="63."><p id="ref-CR63">Scopel, E. F., Hose, J., Bensasson, D. &amp; Gasch, A. P. Genetic variation in aneuploidy prevalence and tolerance across Saccharomyces cerevisiae lineages. <i>Genetics</i> <b>217</b>, iyab015 (2021).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1093/genetics/iyab015" data-track-action="article reference" href="https://doi.org/10.1093%2Fgenetics%2Fiyab015" aria-label="Article reference 63" data-doi="10.1093/genetics/iyab015">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33734361" aria-label="PubMed reference 63">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8049548" aria-label="PubMed Central reference 63">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 63" href="http://scholar.google.com/scholar_lookup?&amp;title=Genetic%20variation%20in%20aneuploidy%20prevalence%20and%20tolerance%20across%20Saccharomyces%20cerevisiae%20lineages&amp;journal=Genetics&amp;doi=10.1093%2Fgenetics%2Fiyab015&amp;volume=217&amp;publication_year=2021&amp;author=Scopel%2CEF&amp;author=Hose%2CJ&amp;author=Bensasson%2CD&amp;author=Gasch%2CAP">
                    Google Scholar</a> 
                </p></li><li data-counter="64."><p id="ref-CR64">Torres, E. M. et al. Effects of aneuploidy on cellular physiology and cell division in haploid yeast. <i>Science</i> <b>317</b>, 916–924 (2007).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1126/science.1142210" data-track-action="article reference" href="https://doi.org/10.1126%2Fscience.1142210" aria-label="Article reference 64" data-doi="10.1126/science.1142210">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2sXptVaisro%3D" aria-label="CAS reference 64">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17702937" aria-label="PubMed reference 64">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 64" href="http://scholar.google.com/scholar_lookup?&amp;title=Effects%20of%20aneuploidy%20on%20cellular%20physiology%20and%20cell%20division%20in%20haploid%20yeast&amp;journal=Science&amp;doi=10.1126%2Fscience.1142210&amp;volume=317&amp;pages=916-924&amp;publication_year=2007&amp;author=Torres%2CEM">
                    Google Scholar</a> 
                </p></li><li data-counter="65."><p id="ref-CR65">Heavner, B. D. &amp; Price, N. D. Comparative analysis of yeast metabolic network models highlights progress, opportunities for metabolic reconstruction. <i>PLoS Comput. Biol.</i> <b>11</b>, e1004530 (2015).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1371/journal.pcbi.1004530" data-track-action="article reference" href="https://doi.org/10.1371%2Fjournal.pcbi.1004530" aria-label="Article reference 65" data-doi="10.1371/journal.pcbi.1004530">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26566239" aria-label="PubMed reference 65">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4643975" aria-label="PubMed Central reference 65">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 65" href="http://scholar.google.com/scholar_lookup?&amp;title=Comparative%20analysis%20of%20yeast%20metabolic%20network%20models%20highlights%20progress%2C%20opportunities%20for%20metabolic%20reconstruction&amp;journal=PLoS%20Comput.%20Biol.&amp;doi=10.1371%2Fjournal.pcbi.1004530&amp;volume=11&amp;publication_year=2015&amp;author=Heavner%2CBD&amp;author=Price%2CND">
                    Google Scholar</a> 
                </p></li><li data-counter="66."><p id="ref-CR66">Pereira, R., Nielsen, J. &amp; Rocha, I. Improving the flux distributions simulated with genome-scale metabolic models of Saccharomyces cerevisiae. <i>Metab. Eng. Commun.</i> <b>3</b>, 153–163 (2016).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.meteno.2016.05.002" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.meteno.2016.05.002" aria-label="Article reference 66" data-doi="10.1016/j.meteno.2016.05.002">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29468121" aria-label="PubMed reference 66">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5779720" aria-label="PubMed Central reference 66">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 66" href="http://scholar.google.com/scholar_lookup?&amp;title=Improving%20the%20flux%20distributions%20simulated%20with%20genome-scale%20metabolic%20models%20of%20Saccharomyces%20cerevisiae&amp;journal=Metab.%20Eng.%20Commun.&amp;doi=10.1016%2Fj.meteno.2016.05.002&amp;volume=3&amp;pages=153-163&amp;publication_year=2016&amp;author=Pereira%2CR&amp;author=Nielsen%2CJ&amp;author=Rocha%2CI">
                    Google Scholar</a> 
                </p></li><li data-counter="67."><p id="ref-CR67">Szappanos, B. et al. An integrated approach to characterize genetic interaction networks in yeast metabolism. <i>Nat. Genet.</i> <b>43</b>, 656–662 (2011).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/ng.846" data-track-action="article reference" href="https://doi.org/10.1038%2Fng.846" aria-label="Article reference 67" data-doi="10.1038/ng.846">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXmslSrtb8%3D" aria-label="CAS reference 67">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21623372" aria-label="PubMed reference 67">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3125439" aria-label="PubMed Central reference 67">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 67" href="http://scholar.google.com/scholar_lookup?&amp;title=An%20integrated%20approach%20to%20characterize%20genetic%20interaction%20networks%20in%20yeast%20metabolism&amp;journal=Nat.%20Genet.&amp;doi=10.1038%2Fng.846&amp;volume=43&amp;pages=656-662&amp;publication_year=2011&amp;author=Szappanos%2CB">
                    Google Scholar</a> 
                </p></li><li data-counter="68."><p id="ref-CR68">Brochado, A. R., Andrejev, S., Maranas, C. D. &amp; Patil, K. R. Impact of stoichiometry representation on simulation of genotype-phenotype relationships in metabolic networks. <i>PLoS Comput. Biol.</i> <b>8</b>, e1002758 (2012).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1371/journal.pcbi.1002758" data-track-action="article reference" href="https://doi.org/10.1371%2Fjournal.pcbi.1002758" aria-label="Article reference 68" data-doi="10.1371/journal.pcbi.1002758">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XhslWjsr%2FN" aria-label="CAS reference 68">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23133362" aria-label="PubMed reference 68">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3486866" aria-label="PubMed Central reference 68">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 68" href="http://scholar.google.com/scholar_lookup?&amp;title=Impact%20of%20stoichiometry%20representation%20on%20simulation%20of%20genotype-phenotype%20relationships%20in%20metabolic%20networks&amp;journal=PLoS%20Comput.%20Biol.&amp;doi=10.1371%2Fjournal.pcbi.1002758&amp;volume=8&amp;publication_year=2012&amp;author=Brochado%2CAR&amp;author=Andrejev%2CS&amp;author=Maranas%2CCD&amp;author=Patil%2CKR">
                    Google Scholar</a> 
                </p></li><li data-counter="69."><p id="ref-CR69">Grinberg, N. F., Orhobor, O. I. &amp; King, R. D. An evaluation of machine-learning for predicting phenotype: studies in yeast, rice, and wheat. <i>Mach. Learn.</i> <b>109</b>, 251–277 (2020).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1007/s10994-019-05848-5" data-track-action="article reference" href="https://doi.org/10.1007%2Fs10994-019-05848-5" aria-label="Article reference 69" data-doi="10.1007/s10994-019-05848-5">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32174648" aria-label="PubMed reference 69">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 69" href="http://scholar.google.com/scholar_lookup?&amp;title=An%20evaluation%20of%20machine-learning%20for%20predicting%20phenotype%3A%20studies%20in%20yeast%2C%20rice%2C%20and%20wheat&amp;journal=Mach.%20Learn.&amp;doi=10.1007%2Fs10994-019-05848-5&amp;volume=109&amp;pages=251-277&amp;publication_year=2020&amp;author=Grinberg%2CNF&amp;author=Orhobor%2COI&amp;author=King%2CRD">
                    Google Scholar</a> 
                </p></li><li data-counter="70."><p id="ref-CR70">Boulton, C. A. 4 Developments in Brewery Fermentation. <i>Biotechnol. Genet. Eng. Rev.</i> <b>9</b>, 127–181 (1991).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK3sXhtVSmtLs%3D" aria-label="CAS reference 70">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=1797013" aria-label="PubMed reference 70">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 70" href="http://scholar.google.com/scholar_lookup?&amp;title=4%20Developments%20in%20Brewery%20Fermentation&amp;journal=Biotechnol.%20Genet.%20Eng.%20Rev.&amp;volume=9&amp;pages=127-181&amp;publication_year=1991&amp;author=Boulton%2CCA">
                    Google Scholar</a> 
                </p></li><li data-counter="71."><p id="ref-CR71">Smart, K. A. &amp; Whisker, S. Effect of serial repitching on the fermentation properties and condition of brewing yeast. <i>J. Am. Soc. Brew. Chem.</i> <b>54</b>, 41–44 (1996).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK28XovFWjtQ%3D%3D" aria-label="CAS reference 71">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 71" href="http://scholar.google.com/scholar_lookup?&amp;title=Effect%20of%20serial%20repitching%20on%20the%20fermentation%20properties%20and%20condition%20of%20brewing%20yeast&amp;journal=J.%20Am.%20Soc.%20Brew.%20Chem.&amp;volume=54&amp;pages=41-44&amp;publication_year=1996&amp;author=Smart%2CKA&amp;author=Whisker%2CS">
                    Google Scholar</a> 
                </p></li><li data-counter="72."><p id="ref-CR72">Powell, C. D. &amp; Diacetis, A. N. Long term serial repitching and the genetic and phenotypic stability of brewer’s yeast. <i>J. Inst. Brew.</i> <b>113</b>, 67–74 (2007).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1002/j.2050-0416.2007.tb00258.x" data-track-action="article reference" href="https://doi.org/10.1002%2Fj.2050-0416.2007.tb00258.x" aria-label="Article reference 72" data-doi="10.1002/j.2050-0416.2007.tb00258.x">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2sXntlOntLY%3D" aria-label="CAS reference 72">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 72" href="http://scholar.google.com/scholar_lookup?&amp;title=Long%20term%20serial%20repitching%20and%20the%20genetic%20and%20phenotypic%20stability%20of%20brewer%E2%80%99s%20yeast&amp;journal=J.%20Inst.%20Brew.&amp;doi=10.1002%2Fj.2050-0416.2007.tb00258.x&amp;volume=113&amp;pages=67-74&amp;publication_year=2007&amp;author=Powell%2CCD&amp;author=Diacetis%2CAN">
                    Google Scholar</a> 
                </p></li><li data-counter="73."><p id="ref-CR73">McMurrough, I. et al. Control of ferulic acid and 4‐vinyl guaiacol in brewing. <i>J. Inst. Brew.</i> <b>102</b>, 327–332 (1996).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1002/j.2050-0416.1996.tb00918.x" data-track-action="article reference" href="https://doi.org/10.1002%2Fj.2050-0416.1996.tb00918.x" aria-label="Article reference 73" data-doi="10.1002/j.2050-0416.1996.tb00918.x">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK28XmsFCjtLk%3D" aria-label="CAS reference 73">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 73" href="http://scholar.google.com/scholar_lookup?&amp;title=Control%20of%20ferulic%20acid%20and%204%E2%80%90vinyl%20guaiacol%20in%20brewing&amp;journal=J.%20Inst.%20Brew.&amp;doi=10.1002%2Fj.2050-0416.1996.tb00918.x&amp;volume=102&amp;pages=327-332&amp;publication_year=1996&amp;author=McMurrough%2CI">
                    Google Scholar</a> 
                </p></li><li data-counter="74."><p id="ref-CR74">Stewart, G. G., Russell, I. &amp; Anstruther, A. eds. <i>Handbook of Brewing</i> (CRC Press, 2017).</p></li><li data-counter="75."><p id="ref-CR75">Knight, S., Klaere, S., Fedrizzi, B. &amp; Goddard, M. R. Regional microbial signatures positively correlate with differential wine phenotypes: evidence for a microbial aspect to terroir. <i>Sci. Rep.</i> <b>5</b>, 1–10 (2015).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/srep14233" data-track-action="article reference" href="https://doi.org/10.1038%2Fsrep14233" aria-label="Article reference 75" data-doi="10.1038/srep14233">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 75" href="http://scholar.google.com/scholar_lookup?&amp;title=Regional%20microbial%20signatures%20positively%20correlate%20with%20differential%20wine%20phenotypes%3A%20evidence%20for%20a%20microbial%20aspect%20to%20terroir&amp;journal=Sci.%20Rep.&amp;doi=10.1038%2Fsrep14233&amp;volume=5&amp;pages=1-10&amp;publication_year=2015&amp;author=Knight%2CS&amp;author=Klaere%2CS&amp;author=Fedrizzi%2CB&amp;author=Goddard%2CMR">
                    Google Scholar</a> 
                </p></li><li data-counter="76."><p id="ref-CR76">Morcol, T. B., Negrin, A., Matthews, P. D. &amp; Kennelly, E. J. Hop (Humulus lupulus L.) terroir has large effect on a glycosylated green leaf volatile but not on other aroma glycosides. <i>Food Chem.</i> <b>321</b>, 126644 (2020).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.foodchem.2020.126644" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.foodchem.2020.126644" aria-label="Article reference 76" data-doi="10.1016/j.foodchem.2020.126644">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXmt1Kgs78%3D" aria-label="CAS reference 76">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32247886" aria-label="PubMed reference 76">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 76" href="http://scholar.google.com/scholar_lookup?&amp;title=Hop%20%28Humulus%20lupulus%20L.%29%20terroir%20has%20large%20effect%20on%20a%20glycosylated%20green%20leaf%20volatile%20but%20not%20on%20other%20aroma%20glycosides&amp;journal=Food%20Chem.&amp;doi=10.1016%2Fj.foodchem.2020.126644&amp;volume=321&amp;publication_year=2020&amp;author=Morcol%2CTB&amp;author=Negrin%2CA&amp;author=Matthews%2CPD&amp;author=Kennelly%2CEJ">
                    Google Scholar</a> 
                </p></li><li data-counter="77."><p id="ref-CR77">Van Holle, A. et al. Relevance of hop terroir for beer flavour. <i>J. Inst. Brew.</i> <b>127</b>, 238–247 (2021).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1002/jib.648" data-track-action="article reference" href="https://doi.org/10.1002%2Fjib.648" aria-label="Article reference 77" data-doi="10.1002/jib.648">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 77" href="http://scholar.google.com/scholar_lookup?&amp;title=Relevance%20of%20hop%20terroir%20for%20beer%20flavour&amp;journal=J.%20Inst.%20Brew.&amp;doi=10.1002%2Fjib.648&amp;volume=127&amp;pages=238-247&amp;publication_year=2021&amp;author=Holle%2CA">
                    Google Scholar</a> 
                </p></li><li data-counter="78."><p id="ref-CR78">Richards, M. The use of giant‐colony morphology for the differentiation of brewing yeasts. <i>J. Inst. Brew.</i> <b>73</b>, 162–166 (1967).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1002/j.2050-0416.1967.tb03028.x" data-track-action="article reference" href="https://doi.org/10.1002%2Fj.2050-0416.1967.tb03028.x" aria-label="Article reference 78" data-doi="10.1002/j.2050-0416.1967.tb03028.x">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 78" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20use%20of%20giant%E2%80%90colony%20morphology%20for%20the%20differentiation%20of%20brewing%20yeasts&amp;journal=J.%20Inst.%20Brew.&amp;doi=10.1002%2Fj.2050-0416.1967.tb03028.x&amp;volume=73&amp;pages=162-166&amp;publication_year=1967&amp;author=Richards%2CM">
                    Google Scholar</a> 
                </p></li><li data-counter="79."><p id="ref-CR79">ASBC method yeast 13. American Society of Brewing Chemists. Report of subcommittee on differentiation of brewing yeast strains by PCR fingerprinting. <i>J. Am. Soc. Brew. Chem.</i> <b>66</b>, 256–260 (2008).</p><p><a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 79" href="http://scholar.google.com/scholar_lookup?&amp;title=Report%20of%20subcommittee%20on%20differentiation%20of%20brewing%20yeast%20strains%20by%20PCR%20fingerprinting&amp;journal=J.%20Am.%20Soc.%20Brew.%20Chem.&amp;volume=66&amp;pages=256-260&amp;publication_year=2008">
                    Google Scholar</a> 
                </p></li><li data-counter="80."><p id="ref-CR80">Li, H. &amp; Durbin, R. Fast and accurate short read alignment with Burrows–Wheeler transform. <i>Bioinformatics</i> <b>25</b>, 1754–1760 (2009).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1093/bioinformatics/btp324" data-track-action="article reference" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtp324" aria-label="Article reference 80" data-doi="10.1093/bioinformatics/btp324">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXot1Cjtbo%3D" aria-label="CAS reference 80">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19451168" aria-label="PubMed reference 80">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2705234" aria-label="PubMed Central reference 80">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 80" href="http://scholar.google.com/scholar_lookup?&amp;title=Fast%20and%20accurate%20short%20read%20alignment%20with%20Burrows%E2%80%93Wheeler%20transform&amp;journal=Bioinformatics&amp;doi=10.1093%2Fbioinformatics%2Fbtp324&amp;volume=25&amp;pages=1754-1760&amp;publication_year=2009&amp;author=Li%2CH&amp;author=Durbin%2CR">
                    Google Scholar</a> 
                </p></li><li data-counter="81."><p id="ref-CR81">Danecek, P. et al. Twelve years of SAMtools and BCFtools. <i>Gigascience</i> <b>10</b>, giab008 (2021).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1093/gigascience/giab008" data-track-action="article reference" href="https://doi.org/10.1093%2Fgigascience%2Fgiab008" aria-label="Article reference 81" data-doi="10.1093/gigascience/giab008">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33590861" aria-label="PubMed reference 81">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7931819" aria-label="PubMed Central reference 81">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 81" href="http://scholar.google.com/scholar_lookup?&amp;title=Twelve%20years%20of%20SAMtools%20and%20BCFtools&amp;journal=Gigascience&amp;doi=10.1093%2Fgigascience%2Fgiab008&amp;volume=10&amp;publication_year=2021&amp;author=Danecek%2CP">
                    Google Scholar</a> 
                </p></li><li data-counter="82."><p id="ref-CR82">Okonechnikov, K., Conesa, A. &amp; García-Alcalde, F. Qualimap 2: advanced multi-sample quality control for high-throughput sequencing data. <i>Bioinformatics</i> <b>32</b>, 292–294 (2016).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1093/bioinformatics/btv566" data-track-action="article reference" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtv566" aria-label="Article reference 82" data-doi="10.1093/bioinformatics/btv566">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XhtlaqsLzL" aria-label="CAS reference 82">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26428292" aria-label="PubMed reference 82">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 82" href="http://scholar.google.com/scholar_lookup?&amp;title=Qualimap%202%3A%20advanced%20multi-sample%20quality%20control%20for%20high-throughput%20sequencing%20data&amp;journal=Bioinformatics&amp;doi=10.1093%2Fbioinformatics%2Fbtv566&amp;volume=32&amp;pages=292-294&amp;publication_year=2016&amp;author=Okonechnikov%2CK&amp;author=Conesa%2CA&amp;author=Garc%C3%ADa-Alcalde%2CF">
                    Google Scholar</a> 
                </p></li><li data-counter="83."><p id="ref-CR83">Alonge, M. et al. RaGOO: fast and accurate reference-guided scaffolding of draft genomes. <i>Genome Biol.</i> <b>20</b>, 1–17 (2019).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1186/s13059-019-1829-6" data-track-action="article reference" href="https://doi.org/10.1186%2Fs13059-019-1829-6" aria-label="Article reference 83" data-doi="10.1186/s13059-019-1829-6">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 83" href="http://scholar.google.com/scholar_lookup?&amp;title=RaGOO%3A%20fast%20and%20accurate%20reference-guided%20scaffolding%20of%20draft%20genomes&amp;journal=Genome%20Biol.&amp;doi=10.1186%2Fs13059-019-1829-6&amp;volume=20&amp;pages=1-17&amp;publication_year=2019&amp;author=Alonge%2CM">
                    Google Scholar</a> 
                </p></li><li data-counter="84."><p id="ref-CR84">Murigneux, V. et al. Comparison of long-read methods for sequencing and assembly of a plant genome. <i>GigaScience</i> <b>9</b>, giaa146 (2020).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1093/gigascience/giaa146" data-track-action="article reference" href="https://doi.org/10.1093%2Fgigascience%2Fgiaa146" aria-label="Article reference 84" data-doi="10.1093/gigascience/giaa146">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33347571" aria-label="PubMed reference 84">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7751402" aria-label="PubMed Central reference 84">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 84" href="http://scholar.google.com/scholar_lookup?&amp;title=Comparison%20of%20long-read%20methods%20for%20sequencing%20and%20assembly%20of%20a%20plant%20genome&amp;journal=GigaScience&amp;doi=10.1093%2Fgigascience%2Fgiaa146&amp;volume=9&amp;publication_year=2020&amp;author=Murigneux%2CV">
                    Google Scholar</a> 
                </p></li><li data-counter="85."><p id="ref-CR85">Waterhouse, R. M. et al. BUSCO applications from quality assessments to gene prediction and phylogenomics. <i>Mol. Biol. Evol.</i> <b>35</b>, 543–548 (2018).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1093/molbev/msx319" data-track-action="article reference" href="https://doi.org/10.1093%2Fmolbev%2Fmsx319" aria-label="Article reference 85" data-doi="10.1093/molbev/msx319">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXitlyjt7jP" aria-label="CAS reference 85">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29220515" aria-label="PubMed reference 85">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 85" href="http://scholar.google.com/scholar_lookup?&amp;title=BUSCO%20applications%20from%20quality%20assessments%20to%20gene%20prediction%20and%20phylogenomics&amp;journal=Mol.%20Biol.%20Evol.&amp;doi=10.1093%2Fmolbev%2Fmsx319&amp;volume=35&amp;pages=543-548&amp;publication_year=2018&amp;author=Waterhouse%2CRM">
                    Google Scholar</a> 
                </p></li><li data-counter="86."><p id="ref-CR86">Palmer, J. &amp; Stajich, J. nextgenusfs/funannotate: funannotate v1.5.3 (Version 1.5.3). <i>Zenodo</i> <a href="https://doi.org/10.5281/zenodo.2604804" data-track="click" data-track-action="external reference" data-track-label="10.5281/zenodo.2604804">https://doi.org/10.5281/zenodo.2604804</a> (2019).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.5281/zenodo.2604804" data-track-action="article reference" href="https://doi.org/10.5281%2Fzenodo.2604804" aria-label="Article reference 86" data-doi="10.5281/zenodo.2604804">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 86" href="http://scholar.google.com/scholar_lookup?&amp;title=nextgenusfs%2Ffunannotate%3A%20funannotate%20v1.5.3%20%28Version%201.5.3%29&amp;journal=Zenodo&amp;doi=10.5281%2Fzenodo.2604804&amp;publication_year=2019&amp;author=Palmer%2CJ&amp;author=Stajich%2CJ">
                    Google Scholar</a> 
                </p></li><li data-counter="87."><p id="ref-CR87">Van der Auwera, G. A. &amp; O’Connor, B. D. <i>Genomics in the Cloud: Using Docker, GATK, and WDL in Terra</i> (O’Reilly Media, 2020).</p></li><li data-counter="88."><p id="ref-CR88">Cingolani, P. et al. A program for annotating and predicting the effects of single nucleotide polymorphisms, SnpEff: SNPs in the genome of Drosophila melanogaster strain w1118; iso-2; iso-3. <i>Fly</i> <b>6</b>, 80–92 (2012).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.4161/fly.19695" data-track-action="article reference" href="https://doi.org/10.4161%2Ffly.19695" aria-label="Article reference 88" data-doi="10.4161/fly.19695">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38Xht1GmtL3E" aria-label="CAS reference 88">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22728672" aria-label="PubMed reference 88">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3679285" aria-label="PubMed Central reference 88">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 88" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20program%20for%20annotating%20and%20predicting%20the%20effects%20of%20single%20nucleotide%20polymorphisms%2C%20SnpEff%3A%20SNPs%20in%20the%20genome%20of%20Drosophila%20melanogaster%20strain%20w1118%3B%20iso-2%3B%20iso-3&amp;journal=Fly&amp;doi=10.4161%2Ffly.19695&amp;volume=6&amp;pages=80-92&amp;publication_year=2012&amp;author=Cingolani%2CP">
                    Google Scholar</a> 
                </p></li><li data-counter="89."><p id="ref-CR89">Emms, D. M. &amp; Kelly, S. OrthoFinder: phylogenetic orthology inference for comparative genomics. <i>Genome Biol.</i> <b>20</b>, 1–14 (2019).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1186/s13059-019-1832-y" data-track-action="article reference" href="https://doi.org/10.1186%2Fs13059-019-1832-y" aria-label="Article reference 89" data-doi="10.1186/s13059-019-1832-y">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 89" href="http://scholar.google.com/scholar_lookup?&amp;title=OrthoFinder%3A%20phylogenetic%20orthology%20inference%20for%20comparative%20genomics&amp;journal=Genome%20Biol.&amp;doi=10.1186%2Fs13059-019-1832-y&amp;volume=20&amp;pages=1-14&amp;publication_year=2019&amp;author=Emms%2CDM&amp;author=Kelly%2CS">
                    Google Scholar</a> 
                </p></li><li data-counter="90."><p id="ref-CR90">Purcell, S. et al. PLINK: a tool set for whole-genome association and population-based linkage analyses. <i>Am. J. Hum. Genet.</i> <b>81</b>, 559–575 (2007).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1086/519795" data-track-action="article reference" href="https://doi.org/10.1086%2F519795" aria-label="Article reference 90" data-doi="10.1086/519795">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2sXhtVSqurrL" aria-label="CAS reference 90">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17701901" aria-label="PubMed reference 90">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1950838" aria-label="PubMed Central reference 90">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 90" href="http://scholar.google.com/scholar_lookup?&amp;title=PLINK%3A%20a%20tool%20set%20for%20whole-genome%20association%20and%20population-based%20linkage%20analyses&amp;journal=Am.%20J.%20Hum.%20Genet.&amp;doi=10.1086%2F519795&amp;volume=81&amp;pages=559-575&amp;publication_year=2007&amp;author=Purcell%2CS">
                    Google Scholar</a> 
                </p></li><li data-counter="91."><p id="ref-CR91">Boeva, V. et al. Control-FREEC: a tool for assessing copy number and allelic content using next-generation sequencing data. <i>Bioinformatics</i> <b>28</b>, 423–425 (2012).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1093/bioinformatics/btr670" data-track-action="article reference" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtr670" aria-label="Article reference 91" data-doi="10.1093/bioinformatics/btr670">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XhvVertbo%3D" aria-label="CAS reference 91">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22155870" aria-label="PubMed reference 91">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 91" href="http://scholar.google.com/scholar_lookup?&amp;title=Control-FREEC%3A%20a%20tool%20for%20assessing%20copy%20number%20and%20allelic%20content%20using%20next-generation%20sequencing%20data&amp;journal=Bioinformatics&amp;doi=10.1093%2Fbioinformatics%2Fbtr670&amp;volume=28&amp;pages=423-425&amp;publication_year=2012&amp;author=Boeva%2CV">
                    Google Scholar</a> 
                </p></li><li data-counter="92."><p id="ref-CR92">Fijarczyk, A. et al. The genome sequence of the Jean-Talon strain, an archeological beer yeast from Québec, reveals traces of adaptation to specific brewing conditions. <i>G3 Genes Genomes Genet.</i> <b>10</b>, 3087–3097 (2020).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1534/g3.120.401149" data-track-action="article reference" href="https://doi.org/10.1534%2Fg3.120.401149" aria-label="Article reference 92" data-doi="10.1534/g3.120.401149">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXmvVWmur4%3D" aria-label="CAS reference 92">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 92" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20genome%20sequence%20of%20the%20Jean-Talon%20strain%2C%20an%20archeological%20beer%20yeast%20from%20Qu%C3%A9bec%2C%20reveals%20traces%20of%20adaptation%20to%20specific%20brewing%20conditions&amp;journal=G3%20Genes%20Genomes%20Genet.&amp;doi=10.1534%2Fg3.120.401149&amp;volume=10&amp;pages=3087-3097&amp;publication_year=2020&amp;author=Fijarczyk%2CA">
                    Google Scholar</a> 
                </p></li><li data-counter="93."><p id="ref-CR93">ASBC method yeast 7. American Society of Brewing Chemists. Report of Subcommittee on Microbiology. <i>J. Am. Soc. Brew. Chem.</i> <b>43</b>, 16 (1985).</p><p><a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 93" href="http://scholar.google.com/scholar_lookup?&amp;title=Report%20of%20Subcommittee%20on%20Microbiology&amp;journal=J.%20Am.%20Soc.%20Brew.%20Chem.&amp;volume=43&amp;publication_year=1985">
                    Google Scholar</a> 
                </p></li><li data-counter="94."><p id="ref-CR94">Jorgensen A. &amp; Hansen A. <i>Mikroorganismen der Gärungsindustrie</i> 7 Aufl (Verlag Hans Carl, Nürnberg, 1956).</p></li><li data-counter="95."><p id="ref-CR95">Smart, K. A., Chambers, K. M., Lambert, I. &amp; Jenkins, C. <i>J. Am. Soc. Brew. Chem.</i> <b>57</b>, 18 (1999).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK1MXhsVyms7c%3D" aria-label="CAS reference 95">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 95" href="http://scholar.google.com/scholar_lookup?&amp;title=&amp;journal=J.%20Am.%20Soc.%20Brew.%20Chem.&amp;volume=57&amp;publication_year=1999&amp;author=Smart%2CKA&amp;author=Chambers%2CKM&amp;author=Lambert%2CI&amp;author=Jenkins%2CC">
                    Google Scholar</a> 
                </p></li><li data-counter="96."><p id="ref-CR96">Buckee, G. K. &amp; Mundy, A. P. Determination of vicinal diketones in beer by gas chromatography (headspace technique) – collaborative trial. <i>J. Inst. Brew.</i> <b>100</b>, 247–253 (1994).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1002/j.2050-0416.1994.tb00820.x" data-track-action="article reference" href="https://doi.org/10.1002%2Fj.2050-0416.1994.tb00820.x" aria-label="Article reference 96" data-doi="10.1002/j.2050-0416.1994.tb00820.x">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK2MXhtVCisrc%3D" aria-label="CAS reference 96">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 96" href="http://scholar.google.com/scholar_lookup?&amp;title=Determination%20of%20vicinal%20diketones%20in%20beer%20by%20gas%20chromatography%20%28headspace%20technique%29%20%E2%80%93%20collaborative%20trial&amp;journal=J.%20Inst.%20Brew.&amp;doi=10.1002%2Fj.2050-0416.1994.tb00820.x&amp;volume=100&amp;pages=247-253&amp;publication_year=1994&amp;author=Buckee%2CGK&amp;author=Mundy%2CAP">
                    Google Scholar</a> 
                </p></li><li data-counter="97."><p id="ref-CR97">Brewers of Europe. Phenolic off Flavour Method. <i>Analytica-EBC</i> 2.3.9.5. (2018).</p></li><li data-counter="98."><p id="ref-CR98">dos Santos Navarro, R. D. C. et al. Optimized Descriptive Profile: a rapid methodology for sensory description. <i>Food Qual. Preference</i> <b>24</b>, 190–200 (2012).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.foodqual.2011.10.014" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.foodqual.2011.10.014" aria-label="Article reference 98" data-doi="10.1016/j.foodqual.2011.10.014">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 98" href="http://scholar.google.com/scholar_lookup?&amp;title=Optimized%20Descriptive%20Profile%3A%20a%20rapid%20methodology%20for%20sensory%20description&amp;journal=Food%20Qual.%20Preference&amp;doi=10.1016%2Fj.foodqual.2011.10.014&amp;volume=24&amp;pages=190-200&amp;publication_year=2012&amp;author=Santos%20Navarro%2CRDC">
                    Google Scholar</a> 
                </p></li><li data-counter="99."><p id="ref-CR99">Lawless, H. T. &amp; Heymann, H. <i>Sensory Evaluation of Food: Principles and Practices</i> Vol. 2 (Springer, New York, 2010).</p></li></ol><p><a data-track="click" data-track-action="download citation references" data-track-label="link" rel="nofollow" href="https://citation-needed.springer.com/v2/references/10.1038/s42003-023-05587-3?format=refman&amp;flavour=references">Download references<svg width="16" height="16" focusable="false" role="img" aria-hidden="true"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-download-medium"></use></svg></a></p></div></div></div></section></div><section data-title="Acknowledgements"><div id="Ack1-section"><h2 id="Ack1">Acknowledgements</h2><p>The authors would like to thank the Guinness Brand Team for sponsoring the work. The historic records presented in this study were curated with the help of former Guinness Group Microbiologists Dr Edward Bourke. The authors would like to thank Dr Kieran Joyce and his team for their analytical support and Marcus Bengelstorff and Patrick Kerr for their brewing expertise. D.K. and P.C. would like to thank Dr Brigida Gallone, Dr Jan Steensels, Prof Kevin Verstrepen et al. for their 2016 investigation. The St James’s Gate Yeast Library has been maintained since 1903 by numerous Guinness’ Microbiologists however the authors would like to give special thanks to June Hurley, Dr Barbara Cantwell, Dr Daniel Donnelly, Angela Larkin, Dr Vidya Dixit and Noel Early.</p></div></section><section aria-labelledby="author-information" data-title="Author information"><div id="author-information-section"><h2 id="author-information">Author information</h2><div id="author-information-content"><h3 id="affiliations">Authors and Affiliations</h3><ol><li id="Aff1"><p>Diageo Ireland, St James’s Gate, The Liberties, Dublin, Ireland</p><p>Daniel W. M. Kerruish, Jessica Kearns, Eibhlin Colgan &amp; Sandra N. E. Stelma</p></li><li id="Aff2"><p>ELDA biotech, Kildare, Ireland</p><p>Paul Cormican &amp; Elaine M. Kenny</p></li><li id="Aff3"><p>Brewing Consultant, Burton on Trent, UK</p><p>Chris A. Boulton</p></li></ol><h3 id="contributions">Contributions</h3><p>Illumina and Nanopore sequencing were performed by E.K. with bioinformatic analysis done by P.C. Phenotypic analysis was conducted by D.K. and J.K. Historical archive analysis was provided by E.C. D.K designed the study with bioinformatic experimental design made with P.C. and E.K. D.K. wrote the manuscript with editorial input from E.K., P.C., C.B. and S.S.</p><h3 id="corresponding-author">Corresponding author</h3><p id="corresponding-author-list">Correspondence to
                <a id="corresp-c1" href="mailto:daniel.kerruish@diageo.com">Daniel W. M. Kerruish</a>.</p></div></div></section><section data-title="Ethics declarations"><div id="ethics-section"><h2 id="ethics">Ethics declarations</h2><div id="ethics-content">
              
                <h3 id="FPar2">Competing interests</h3>
                <p>Daniel Kerrruish, Jessica Kearns, Eibhlin Colgan, and Sandra Stelma are employees of Diageo Ireland, the owners of Guinness. Chris Boulton is employed as a consultant by Diageo Ireland. All other authors declare no competing interests.</p>
              
            </div></div></section><section data-title="Peer review"><div id="peer-review-section"><h2 id="peer-review">Peer review</h2><div id="peer-review-content">
              
              
                <h3 id="FPar1">Peer review information</h3>
                <p><i>Communications Biology</i> thanks Isheng Tsai and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Primary Handling Editor: George Inglis. A peer review file is available.</p>
              
            </div></div></section><section data-title="Additional information"><div id="additional-information-section"><h2 id="additional-information">Additional information</h2><p><b>Publisher’s note</b> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div></section><section data-title="Supplementary information"><div id="Sec28-section"><h2 id="Sec28">Supplementary information</h2></div></section><section data-title="Rights and permissions"><div id="rightslink-section"><h2 id="rightslink">Rights and permissions</h2><div id="rightslink-content">
                <p><b>Open Access</b>  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this license, visit <a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>.</p>
              <p><a data-track="click" data-track-action="view rights and permissions" data-track-label="link" href="https://s100.copyright.com/AppDispatchServlet?title=The%20origins%20of%20the%20Guinness%20stout%20yeast&amp;author=Daniel%20W.%20M.%20Kerruish%20et%20al&amp;contentID=10.1038%2Fs42003-023-05587-3&amp;copyright=The%20Author%28s%29&amp;publication=2399-3642&amp;publicationDate=2024-01-12&amp;publisherName=SpringerNature&amp;orderBeanReset=true&amp;oa=CC%20BY">Reprints and permissions</a></p></div></div></section><section aria-labelledby="article-info" data-title="About this article"><div id="article-info-section"><h2 id="article-info">About this article</h2><div id="article-info-content"><div><p><a data-crossmark="10.1038/s42003-023-05587-3" target="_blank" rel="noopener" href="https://crossmark.crossref.org/dialog/?doi=10.1038/s42003-023-05587-3" data-track="click" data-track-action="Click Crossmark" data-track-label="link" data-test="crossmark"><img loading="lazy" width="57" height="81" alt="Check for updates. Verify currency and authenticity via CrossMark" src="data:image/svg+xml;base64,<svg height="81" width="57" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="m17.35 35.45 21.3-14.2v-17.03h-21.3" fill="#989898"/><path d="m38.65 35.45-21.3-14.2v-17.03h21.3" fill="#747474"/><path d="m28 .5c-12.98 0-23.5 10.52-23.5 23.5s10.52 23.5 23.5 23.5 23.5-10.52 23.5-23.5c0-6.23-2.48-12.21-6.88-16.62-4.41-4.4-10.39-6.88-16.62-6.88zm0 41.25c-9.8 0-17.75-7.95-17.75-17.75s7.95-17.75 17.75-17.75 17.75 7.95 17.75 17.75c0 4.71-1.87 9.22-5.2 12.55s-7.84 5.2-12.55 5.2z" fill="#535353"/><path d="m41 36c-5.81 6.23-15.23 7.45-22.43 2.9-7.21-4.55-10.16-13.57-7.03-21.5l-4.92-3.11c-4.95 10.7-1.19 23.42 8.78 29.71 9.97 6.3 23.07 4.22 30.6-4.86z" fill="#9c9c9c"/><path d="m.2 58.45c0-.75.11-1.42.33-2.01s.52-1.09.91-1.5c.38-.41.83-.73 1.34-.94.51-.22 1.06-.32 1.65-.32.56 0 1.06.11 1.51.35.44.23.81.5 1.1.81l-.91 1.01c-.24-.24-.49-.42-.75-.56-.27-.13-.58-.2-.93-.2-.39 0-.73.08-1.05.23-.31.16-.58.37-.81.66-.23.28-.41.63-.53 1.04-.13.41-.19.88-.19 1.39 0 1.04.23 1.86.68 2.46.45.59 1.06.88 1.84.88.41 0 .77-.07 1.07-.23s.59-.39.85-.68l.91 1c-.38.43-.8.76-1.28.99-.47.22-1 .34-1.58.34-.59 0-1.13-.1-1.64-.31-.5-.2-.94-.51-1.31-.91-.38-.4-.67-.9-.88-1.48-.22-.59-.33-1.26-.33-2.02zm8.4-5.33h1.61v2.54l-.05 1.33c.29-.27.61-.51.96-.72s.76-.31 1.24-.31c.73 0 1.27.23 1.61.71.33.47.5 1.14.5 2.02v4.31h-1.61v-4.1c0-.57-.08-.97-.25-1.21-.17-.23-.45-.35-.83-.35-.3 0-.56.08-.79.22-.23.15-.49.36-.78.64v4.8h-1.61zm7.37 6.45c0-.56.09-1.06.26-1.51.18-.45.42-.83.71-1.14.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.36c.07.62.29 1.1.65 1.44.36.33.82.5 1.38.5.29 0 .57-.04.83-.13s.51-.21.76-.37l.55 1.01c-.33.21-.69.39-1.09.53-.41.14-.83.21-1.26.21-.48 0-.92-.08-1.34-.25-.41-.16-.76-.4-1.07-.7-.31-.31-.55-.69-.72-1.13-.18-.44-.26-.95-.26-1.52zm4.6-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.07.45-.31.29-.5.73-.58 1.3zm2.5.62c0-.57.09-1.08.28-1.53.18-.44.43-.82.75-1.13s.69-.54 1.1-.71c.42-.16.85-.24 1.31-.24.45 0 .84.08 1.17.23s.61.34.85.57l-.77 1.02c-.19-.16-.38-.28-.56-.37-.19-.09-.39-.14-.61-.14-.56 0-1.01.21-1.35.63-.35.41-.52.97-.52 1.67 0 .69.17 1.24.51 1.66.34.41.78.62 1.32.62.28 0 .54-.06.78-.17.24-.12.45-.26.64-.42l.67 1.03c-.33.29-.69.51-1.08.65-.39.15-.78.23-1.18.23-.46 0-.9-.08-1.31-.24-.4-.16-.75-.39-1.05-.7s-.53-.69-.7-1.13c-.17-.45-.25-.96-.25-1.53zm6.91-6.45h1.58v6.17h.05l2.54-3.16h1.77l-2.35 2.8 2.59 4.07h-1.75l-1.77-2.98-1.08 1.23v1.75h-1.58zm13.69 1.27c-.25-.11-.5-.17-.75-.17-.58 0-.87.39-.87 1.16v.75h1.34v1.27h-1.34v5.6h-1.61v-5.6h-.92v-1.2l.92-.07v-.72c0-.35.04-.68.13-.98.08-.31.21-.57.4-.79s.42-.39.71-.51c.28-.12.63-.18 1.04-.18.24 0 .48.02.69.07.22.05.41.1.57.17zm.48 5.18c0-.57.09-1.08.27-1.53.17-.44.41-.82.72-1.13.3-.31.65-.54 1.04-.71.39-.16.8-.24 1.23-.24s.84.08 1.24.24c.4.17.74.4 1.04.71s.54.69.72 1.13c.19.45.28.96.28 1.53s-.09 1.08-.28 1.53c-.18.44-.42.82-.72 1.13s-.64.54-1.04.7-.81.24-1.24.24-.84-.08-1.23-.24-.74-.39-1.04-.7c-.31-.31-.55-.69-.72-1.13-.18-.45-.27-.96-.27-1.53zm1.65 0c0 .69.14 1.24.43 1.66.28.41.68.62 1.18.62.51 0 .9-.21 1.19-.62.29-.42.44-.97.44-1.66 0-.7-.15-1.26-.44-1.67-.29-.42-.68-.63-1.19-.63-.5 0-.9.21-1.18.63-.29.41-.43.97-.43 1.67zm6.48-3.44h1.33l.12 1.21h.05c.24-.44.54-.79.88-1.02.35-.24.7-.36 1.07-.36.32 0 .59.05.78.14l-.28 1.4-.33-.09c-.11-.01-.23-.02-.38-.02-.27 0-.56.1-.86.31s-.55.58-.77 1.1v4.2h-1.61zm-47.87 15h1.61v4.1c0 .57.08.97.25 1.2.17.24.44.35.81.35.3 0 .57-.07.8-.22.22-.15.47-.39.73-.73v-4.7h1.61v6.87h-1.32l-.12-1.01h-.04c-.3.36-.63.64-.98.86-.35.21-.76.32-1.24.32-.73 0-1.27-.24-1.61-.71-.33-.47-.5-1.14-.5-2.02zm9.46 7.43v2.16h-1.61v-9.59h1.33l.12.72h.05c.29-.24.61-.45.97-.63.35-.17.72-.26 1.1-.26.43 0 .81.08 1.15.24.33.17.61.4.84.71.24.31.41.68.53 1.11.13.42.19.91.19 1.44 0 .59-.09 1.11-.25 1.57-.16.47-.38.85-.65 1.16-.27.32-.58.56-.94.73-.35.16-.72.25-1.1.25-.3 0-.6-.07-.9-.2s-.59-.31-.87-.56zm0-2.3c.26.22.5.37.73.45.24.09.46.13.66.13.46 0 .84-.2 1.15-.6.31-.39.46-.98.46-1.77 0-.69-.12-1.22-.35-1.61-.23-.38-.61-.57-1.13-.57-.49 0-.99.26-1.52.77zm5.87-1.69c0-.56.08-1.06.25-1.51.16-.45.37-.83.65-1.14.27-.3.58-.54.93-.71s.71-.25 1.08-.25c.39 0 .73.07 1 .2.27.14.54.32.81.55l-.06-1.1v-2.49h1.61v9.88h-1.33l-.11-.74h-.06c-.25.25-.54.46-.88.64-.33.18-.69.27-1.06.27-.87 0-1.56-.32-2.07-.95s-.76-1.51-.76-2.65zm1.67-.01c0 .74.13 1.31.4 1.7.26.38.65.58 1.15.58.51 0 .99-.26 1.44-.77v-3.21c-.24-.21-.48-.36-.7-.45-.23-.08-.46-.12-.7-.12-.45 0-.82.19-1.13.59-.31.39-.46.95-.46 1.68zm6.35 1.59c0-.73.32-1.3.97-1.71.64-.4 1.67-.68 3.08-.84 0-.17-.02-.34-.07-.51-.05-.16-.12-.3-.22-.43s-.22-.22-.38-.3c-.15-.06-.34-.1-.58-.1-.34 0-.68.07-1 .2s-.63.29-.93.47l-.59-1.08c.39-.24.81-.45 1.28-.63.47-.17.99-.26 1.54-.26.86 0 1.51.25 1.93.76s.63 1.25.63 2.21v4.07h-1.32l-.12-.76h-.05c-.3.27-.63.48-.98.66s-.73.27-1.14.27c-.61 0-1.1-.19-1.48-.56-.38-.36-.57-.85-.57-1.46zm1.57-.12c0 .3.09.53.27.67.19.14.42.21.71.21.28 0 .54-.07.77-.2s.48-.31.73-.56v-1.54c-.47.06-.86.13-1.18.23-.31.09-.57.19-.76.31s-.33.25-.41.4c-.09.15-.13.31-.13.48zm6.29-3.63h-.98v-1.2l1.06-.07.2-1.88h1.34v1.88h1.75v1.27h-1.75v3.28c0 .8.32 1.2.97 1.2.12 0 .24-.01.37-.04.12-.03.24-.07.34-.11l.28 1.19c-.19.06-.4.12-.64.17-.23.05-.49.08-.76.08-.4 0-.74-.06-1.02-.18-.27-.13-.49-.3-.67-.52-.17-.21-.3-.48-.37-.78-.08-.3-.12-.64-.12-1.01zm4.36 2.17c0-.56.09-1.06.27-1.51s.41-.83.71-1.14c.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.37c.08.62.29 1.1.65 1.44.36.33.82.5 1.38.5.3 0 .58-.04.84-.13.25-.09.51-.21.76-.37l.54 1.01c-.32.21-.69.39-1.09.53s-.82.21-1.26.21c-.47 0-.92-.08-1.33-.25-.41-.16-.77-.4-1.08-.7-.3-.31-.54-.69-.72-1.13-.17-.44-.26-.95-.26-1.52zm4.61-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.08.45-.31.29-.5.73-.57 1.3zm3.01 2.23c.31.24.61.43.92.57.3.13.63.2.98.2.38 0 .65-.08.83-.23s.27-.35.27-.6c0-.14-.05-.26-.13-.37-.08-.1-.2-.2-.34-.28-.14-.09-.29-.16-.47-.23l-.53-.22c-.23-.09-.46-.18-.69-.3-.23-.11-.44-.24-.62-.4s-.33-.35-.45-.55c-.12-.21-.18-.46-.18-.75 0-.61.23-1.1.68-1.49.44-.38 1.06-.57 1.83-.57.48 0 .91.08 1.29.25s.71.36.99.57l-.74.98c-.24-.17-.49-.32-.73-.42-.25-.11-.51-.16-.78-.16-.35 0-.6.07-.76.21-.17.15-.25.33-.25.54 0 .14.04.26.12.36s.18.18.31.26c.14.07.29.14.46.21l.54.19c.23.09.47.18.7.29s.44.24.64.4c.19.16.34.35.46.58.11.23.17.5.17.82 0 .3-.06.58-.17.83-.12.26-.29.48-.51.68-.23.19-.51.34-.84.45-.34.11-.72.17-1.15.17-.48 0-.95-.09-1.41-.27-.46-.19-.86-.41-1.2-.68z" fill="#535353"/></g></svg>"/></a></p><div><h3 id="citeas">Cite this article</h3><p>Kerruish, D.W.M., Cormican, P., Kenny, E.M. <i>et al.</i> The origins of the Guinness stout yeast.
                    <i>Commun Biol</i> <b>7</b>, 68 (2024). https://doi.org/10.1038/s42003-023-05587-3</p><p><a data-test="citation-link" data-track="click" data-track-action="download article citation" data-track-label="link" data-track-external="" rel="nofollow" href="https://citation-needed.springer.com/v2/references/10.1038/s42003-023-05587-3?format=refman&amp;flavour=citation">Download citation<svg width="16" height="16" focusable="false" role="img" aria-hidden="true"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-download-medium"></use></svg></a></p><ul data-test="publication-history"><li><p>Received<span>: </span><span><time datetime="2022-07-12">12 July 2022</time></span></p></li><li><p>Accepted<span>: </span><span><time datetime="2023-11-14">14 November 2023</time></span></p></li><li><p>Published<span>: </span><span><time datetime="2024-01-12">12 January 2024</time></span></p></li><li><p><abbr title="Digital Object Identifier">DOI</abbr><span>: </span><span>https://doi.org/10.1038/s42003-023-05587-3</span></p></li></ul></div></div></div></div></section>
            </div>

            

            
                <section data-title="Comments"><div id="article-comments-section"><h2 id="article-comments">Comments</h2><p>By submitting a comment you agree to abide by our <a href="https://www.nature.com/info/tandc.html">Terms</a> and <a href="https://www.nature.com/info/community-guidelines.html">Community Guidelines</a>. If you find something abusive or that does not comply with our terms or guidelines please flag it as inappropriate.</p></div></section>
                
            
</div></div>
  </body>
</html>
