<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <title>James Routley | Feed</title>
    <link
      rel="stylesheet"
      type="text/css"
      href="../styles.css"
      media="screen"
    />
  </head>
  <body>
    <a href="/index.html">Back</a>
    <a href="https://www.nature.com/articles/d41586-022-00077-8">Original</a>
    <h1>Turning back time with epigenetic clocks</h1>
    
    <div id="readability-page-1" class="page"><div>
                    <p>In 2019, a small study raised the tantalizing prospect that ageing could be reversed. Scientists in California gave 9 men aged 51 to 65 a growth hormone and two diabetes medications for a year<sup><a href="#ref-CR1" data-track="click" data-action="anchor-link" data-track-label="go to reference" data-track-category="references">1</a></sup>. The drugs seemed to rejuvenate the men’s thymus glands and immune function. They also shaved 2.5 years off the men’s biological age, as measured by one of the most talked-about technologies in ageing research: epigenetic clocks.</p><p>Biological age is an important concept, albeit a slippery one. Everyone’s physical and mental functioning gradually declines from early adulthood onwards, but this occurs at different rates in different people. A technique for measuring biological age detects a signal that is a better guide to a person’s functional capacity than their actual, chronological age.</p><p>As more and more scientists seek to slow, halt or rewind ageing, such methods will be needed to assess whether the new manipulations achieve these goals.</p><p>Epigenetic clocks use algorithms to calculate biological age on the basis of a read-out of the extent to which dozens or even hundreds of sites across an individual’s genome are bound by methyl groups — a form of epigenetic modification.</p><div data-label="Related">
  <a href="https://www.nature.com/collections/hgiiegdbjb" data-track="click" data-track-label="recommended article"><img alt="" src="https://media.nature.com/w400/magazine-assets/d41586-022-00077-8/d41586-022-00077-8_20027734.jpg"/><p>Part of Nature Outlook: Ageing</p></a>
 </div><p>In 2017, the scientists behind the growth-hormone trial — based at Intervene Immune, an anti-ageing biotech company in Torrance, California — were excited by their observation of thymus and immune renewal. They contacted Steve Horvath, an anti-ageing researcher at the University of California, Los Angeles, to ask if he would use an epigenetic clock to analyse blood samples they had taken during the trial. Horvath agreed. “Anybody who has an exciting study,” he says, “I love to get involved.”</p><p>But critics have questioned the purported decrease in biological age, stressing that it was a small, unblinded study with no placebo control arm. “If you have nine people,” says Horvath, “and you get a statistically significant result, it means there’s a strong effect.” He and the company are now running a randomized and placebo-controlled phase II replication on a larger group of 85 people.</p><p>The study is one of many, in humans and in animals, that seek ways to reduce epigenetic-clock scores — and thereby develop new anti-ageing interventions.</p><p>But some experts are concerned by the unknowns that still surround this technology. Matt Kaeberlein, who studies ageing at the University of Washington in Seattle, says: “It’s become a sort of dogma in the field — and in the popular perception — that these things are really measuring biological ageing. We really need to understand how these things are working.”</p><p>Horvath acknowledges this. “That’s the weakness of these biomarkers,” he says. “They come out of a machine-learning algorithm. They work beautifully in a mathematical sense, but biologists want more.”</p><p>The US Food and Drug Administration does not currently recognize epigenetic-clock scores as surrogate end points for clinical trials. It wants their mechanistic basis to be better defined. And it wants an answer to the crucial question of whether a short-term decrease in someone’s epigenetic-clock score definitively lowers their chances of developing age-related ill health.</p><h2><b>Molecular horology</b></h2><p>The DNA methylation that underpins epigenetic clocks is a reversible process that is catalysed by enzymes. It involves the addition of a tag known as a methyl group to parts of the genome in which cytosine bases are bound to guanine bases through a phosphate group (CpG). When methylated, CpGs can act as binding sites for proteins that alter DNA’s 3D structure. At numerous CpGs, methylation has been shown to profoundly decrease gene expression, offering a clear mechanism by which it can affect biological function.</p><p>In the late 1990s, researchers at Johns Hopkins University in Baltimore, Maryland, discovered ageing- and cancer-related changes to DNA methylation in cells of the human colon<sup><a href="#ref-CR2" data-track="click" data-action="anchor-link" data-track-label="go to reference" data-track-category="references">2</a></sup>. Theories of ageing have long considered ways in which genomic integrity might be lost progressively, such as by mutations accumulating or the telomere caps of chromosomes shortening. This observation of shifting DNA methylation bolstered the nascent idea that epigenetic disruption might drive ageing by increasingly dysregulating gene expression.</p><p>But whereas early studies of methylation focused on genes selected for their known relevance to ageing, epigenetic clocks are the fruit of ‘big data’ science.</p><p>In the late 2000s, arrays emerged that take DNA from many cells and analyse thousands of CpGs to determine what fraction of them bear a methyl group. Combined with machine learning, these arrays were quickly used to seekepigenetic signatures for numerous traits.</p><p>But even if a signal is uncovered, much work typically remains. The effects of methylation have been characterized at only a tiny portion of the human genome’s roughly 28 million CpGs; many CpGs are not obviously associated with genes; and even statistically robust signals often represent only small percentage changes in methylation.</p><p>In 2011, Horvath was asked to probe methylome data from blood samples provided in a study of sexual orientation. There was no signal for homosexuality, but Horvath then programmed algorithms to identify age-related signals instead. The result was the first epigenetic age predictor<sup><a href="#ref-CR3" data-track="click" data-action="anchor-link" data-track-label="go to reference" data-track-category="references">3</a></sup>, based on the combined methylation status of a few CpGs.</p><p>Two years later, researchers at the University of California, San Diego, published a blood-based DNA methylation clock<sup><a href="#ref-CR4" data-track="click" data-action="anchor-link" data-track-label="go to reference" data-track-category="references">4</a></sup> that used 71 CpGs to predict age more accurately, and Horvath produced a multi-organ clock. Having gathered published methylome data from as many tissues as possible, he sought age-related signals that applied to them all<sup><a href="#ref-CR5" data-track="click" data-action="anchor-link" data-track-label="go to reference" data-track-category="references">5</a></sup>. Clocks based on protein, RNA or metabolite levels, for example, relate only to the tissues from which the samples are taken. Uncovering a ubiquitous methylation signal would potentially indicate a universal hallmark of ageing. And that’s what Horvath found.</p><p>Data from 353 CpGs predicted human age from the embryo to old age. Embryonic stem cells had an epigenetic age of nearly zero, whereas cancer cells showed accelerated ageing. The clock even works in chimpanzees. “I still think it’s a bit of a miracle that it’s possible,” Horvath says. Since then clock scores have even also been show to tick up upwards as cultured cells age in a dish.</p><h2><b>Clock watching</b></h2><p>Numerous epigenetic clocks have since been developed, and important shifts in methodology have occurred — most notably in clinically oriented clocks. To be clinically useful, a biomarker, in this case the pattern of methylation, must predict functional outcomes: here, the occurrence of age-related ill health and lifespan.</p><p>Early clocks did this quite well when applied retrospectively to pre-existing data sets. But an intrinsic limitation of first-generation clocks was neatly illustrated in a 2019 study<sup><a href="#ref-CR6" data-track="click" data-action="anchor-link" data-track-label="go to reference" data-track-category="references">6</a></sup> by researchers at the University of Queensland in Brisbane, Australia: given enough training data, an algorithm programmed to detect age-dependent patterns could almost perfectly compute chronological age. But the better it did this, the worse it predicted mortality.</p><p>“We actually want deviation from chronological age,” says Morgan Levine, who studies ageing and epigenetics at Yale University in New Haven, Connecticut. “But we don’t want the deviation to be error. We want it to be biologically meaningful.” Clock scores should be higher in people who are ageing faster, and lower in those declining slowly.</p><p>In 2018, Levine, who was then a postdoc in Horvath’s lab, helped develop the first second-generation epigenetic clock<sup><a href="#ref-CR7" data-track="click" data-action="anchor-link" data-track-label="go to reference" data-track-category="references">7</a></sup>. Called DNAm PhenoAge, it used algorithms trained to find methylation signals associated not only with chronological age, but also with a panel of ageing-related phenotypic indicators, such as blood glucose and markers of liver and kidney function. It predicts age-related health outcomes better than earlier clocks can.</p><figure>
 <div>
  <p><img alt="Morgan Levine sits at a lab bench using a pipette" data-src="//media.nature.com/lw800/magazine-assets/d41586-022-00077-8/d41586-022-00077-8_20028576.jpg"/></p><figcaption>
   <p><span>Morgan Levine explores how age-related processes affect epigenetic clocks.</span><span>Credit: Zachary Levine</span></p>
  </figcaption>
 </div>
</figure><p>Horvath then developed GrimAge. Also trained on phenotypic markers, it is even better at predicting age-related disease and mortality<sup><a href="#ref-CR8" data-track="click" data-action="anchor-link" data-track-label="go to reference" data-track-category="references">8</a></sup>. It is being used as the end point in Intervene Immune’s phase II trial, emphasizing an essential question: if a person’s epigenetic clock score decreases, does it definitively indicate that they have become meaningfully, biologically younger?</p><p>Answering this question conclusively will require studies that follow the same people over time. Those whose epigenetic age is lowered should live longer and be less prone to age-related disease than those whose epigenetic age is unaffected, Horvath explains. Testing this directly, however, will take decades, but scientists want quicker confirmation.</p><p>There are no recognized anti-ageing interventions for humans, so it is impossible to test whether a proven treatment actually reduces clock scores. An alternative is to look at interventions that extend the lifespan of other animals. Some, such as calorie restriction, do seem to reduce epigenetic age in mice. Nevertheless, Kaeberlein wants to see all such manipulations systematically investigated — and the proposition that clock scores can predict lifespans at the individual level tested.</p><p>He stresses the importance of reporting negative results, and determining whether all treatments that extend lifespan or improve late-life functionality decrease epigenetic age — as well as whether all interventions that decrease clock scores increase longevity.</p><h2><b>What makes it tick? </b></h2><p>To some extent, determining whether epigenetic clocks might serve as biomarkers is separate from understanding their mechanistic underpinnings. “They’re answering different questions,” says Chris Bell, an epigenetics researcher at Queen Mary University of London. If a biomarker can reliably inform clinical decisions — perhaps by identifying people at high risk of specific outcomes — knowing exactly how it works is less important.</p><p>Nevertheless, understanding what drives epigenetic change should help the development of new clocks and improve our understanding of what the scores they produce really mean.</p><p>Horvath thinks that epigenetic clock scores reflect the progression of a regulated process that is evolutionarily conserved across mammals. He sees support for this in the rate at which DNA methylation changes throughout life: it moves quickly during development, then slows considerably after sexual maturity. Also, some clock CpGs become more methylated and some less so, arguing against a process that drifts towards either increased or decreased methylation. Horvath speculates that changes affecting stem cells, which are present at low levels in adult tissues, might make a big contribution to the signals epigenetic clocks detect. And the field eagerly awaits the maturation of techniques that can measure the methylomes of individual cells.</p><p>By contrast, David Sinclair, an anti-ageing researcher at Harvard University in Boston, Massachusetts, thinks clock algorithms detect the results of stochastic damage to DNA, which induces CpG methylation as breakages are repaired, leading to widespread, random changes to the methylome.</p><p>Sinclair genetically engineered mice to undergo a barrage of DNA damage that is repaired in such a way as to change the animal’s epigenome without inducing sequence mutations. “The more cycles you do of this, the more you accumulate epigenetic noise,” Sinclair says, “ultimately leading to a loss of cell identity, which we call ageing.”</p><p>These mice, from the molecular level to overt functionality, age prematurely. At ten months old — normally typical adulthood for a mouse — they look geriatric. And their epigenetic clocks run fast.</p><p>But Horvath — working with radiation biologist Ken Raj of the UK Health Security Agency, a government body responsible for protecting public health — has studied the effects of ionizing radiation, which also causes random DNA damage and methylation as DNA is repaired. Across multiple epigenetic clocks, they found no evidence of increased epigenetic ageing<sup><a href="#ref-CR9" data-track="click" data-action="anchor-link" data-track-label="go to reference" data-track-category="references">9</a></sup>.</p><p>These hypotheses focus on epigenetic clocks measuring a unitary process — indeed, these clocks calculate a single age score. “A single measure keeps things simple,” says Levine, but she explains that two people can get the same elevated score for different reasons. One person might have accelerated ageing associated with inflammation, whereas another might have advanced cardiovascular or brain ageing. This suggests that multiple distinct age-related processes can run at different rates in different people — or even in the same person.</p><p>Levine’s current work addresses this issue by deconstructing the signals that epigenetic clocks detect. Perhaps surprisingly, successive clocks have not converged on a set of CpGs whose methylation most powerfully correlates with ageing. Levine’s group, therefore, took all the CPGs from multiple published clocks and looked for CpG clusters whose methylation changed in temporally and spatially correlated ways<sup><a href="#ref-CR10" data-track="click" data-action="anchor-link" data-track-label="go to reference" data-track-category="references">10</a></sup>. This approach revealed that CpGs can be grouped in 18 co-varying modules.</p><p>Although the individual CpGs vary, different clocks tend to contain CpGs from each module, meaning that every module is represented. Levine hypothesizes that each module tracks a different biological process, so they change at different rates in different tissues. By testing how factors robustly associated with ageing — such as cell division, cancer or tissue hypoxia — affect different modules, her lab found that particular factors affected modules in different ways. Cancer, for example, accelerated ageing in some modules but slowed it in others.</p><p>If each module is driven by a distinct biological process, with varying implications for different organ systems and disease risk, and they can change independently, then looking at this level of granularity might be more useful for assessing someone’s health than deriving a single age score for the person as a whole.</p><p>Bell says this issue highlights a basic dichotomy between researchers who want to treat ageing itself and those who study ageing to see how it increases the risk of specific diseases. The former group might be interested in an overall age score, whereas a neurologist or cardiologist might wish to access more domain-specific epigenetic biomarkers.</p><p>Levine is interested in both. “I think as we become more sophisticated,” she says, “we’ll want to apply a lot of different clocks.”</p><h2><b>Young again</b></h2><p>As well as knowing what causes DNA methylation to change, researchers also need to address whether methylation changes are a primary driver of ageing. And whether resetting the methylome to a younger state is sufficient to rejuvenate the body.</p><p>Levine is frank: “I don’t think anyone’s showed any convincing evidence that these are causal.” However, as support for the centrality of epigenetic dysregulation, Sinclair cites his own study on prematurely ageing mice and work his lab has done to use cellular reprogramming to rejuvenate the eyesight of mice.</p><p>In 2020, his lab adapted a method previously developed to make differentiated cells revert back to a pluripotent stem-cell state. By introducing genes for certain transcription factors, his team hoped to partly reprogramme the cells. Working on mouse retinas, the researchers managed to revitalize ageing and injured neurons<sup><a href="#ref-CR11" data-track="click" data-action="anchor-link" data-track-label="go to reference" data-track-category="references">11</a></sup>, arguing that the cells became phenotypically younger without being de-differentiated. And he related this to epigenetic clock signals.</p><p>Mature retinal neurons, unlike young ones, cannot regrow axons if their nerve fibres are severed or damaged by disease. In a mouse model of glaucoma or after injury, neurons’ epigenetic clock scores were prematurely aged. Reprogramming them restored the ability to sprout axons, reversed age-related changes in gene expression and returned the epigenetic clock scores to normal.</p><div data-label="Related">
  <a href="https://www.nature.com/nature/collections?type=outlook" data-track="click" data-track-label="recommended article"><img alt="" src="https://media.nature.com/w400/magazine-assets/d41586-022-00077-8/d41586-022-00077-8_17201086.jpg"/><p>More from Nature Outlooks</p></a>
 </div><p>The finding that injury recapitulates ageing in epigenetic clock assays, Sinclair says, “points to a universal stress response that is beneficial because it tries to help the cells survive, but ultimately leads to ageing”.</p><p>Most intriguingly, in terms of establishing causality, if two enzymes that add methyl groups to DNA were inhibited during the reprogramming, the epigenetic clocks did not reset and axon regeneration was not restored.</p><p>“It doesn’t mean the clock alone is the cause of ageing,” says Sinclair. “But the clock is representative of these changes in the DNA methylome that are necessary, but perhaps not sufficient, to reset the age of the tissue and get it to work again.”</p><p>This single experiment will not settle the matter, but it keeps epigenetic clocks at the heart of the growing quest for ways to turn back time. Sinclair is now using the epigenetic age of cultured cells to screen potential anti-ageing treatments. “It means we can screen thousands of samples in just a day,” he says. His efforts are indicative of the escalating drive to discover an intervention that could profoundly affect human health.</p><p>In the meantime, Intervene Immune’s phase II replication of their mixture of growth hormone and diabetes medication carries on. Such is Horvath’s interest in the trial he even enrolled as a participant. “I will withhold any judgement until 12 months from now,” he says. “I will analyse the data, and if it didn’t replicate, I will tell this to the world. But I hope it will replicate because wouldn’t it be nice?”</p>
                </div></div>
  </body>
</html>
