<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <title>James Routley | Feed</title>
    <link
      rel="stylesheet"
      type="text/css"
      href="../styles.css"
      media="screen"
    />
  </head>
  <body>
    <a href="/index.html">Back</a>
    <a href="http://omicsomics.blogspot.com/2024/02/want-to-build-sequencer-454bio-opens-up.html">Original</a>
    <h1>Want to build a sequencer? 454.bio opens up their plans</h1>
    
    <div id="readability-page-1" class="page"><div id="post-body-965125533024651667" itemprop="description articleBody">
<p>Just as the AGBT hype cycle was firing up (with me contributing multiple sparks), serial entrepreneur Jonathan Rothberg&#39;s latest sequencing startup 454.bio fully de-stealthed their technology this weekend, going so far as to release<a href="https://454.bio/docs/"> open source plans</a> to build an instrument prototype.  454.bio  is aiming to build a Keurig-sized device to retail for $100, with sequencing runs in the $20 range.  To accomplish this, they&#39;re attempting a novel twist on sequencing-by-synthesis.  It&#39;s an unconventional strategy by someone who has succeeded twice before in DNA sequencing (454 and Ion Torrent) and has multiple other companies going (if I&#39;ve counted correctly)  - QuantumSI in protein sequencing (a future topic for this space, I promise!), ButterflyNetworks with inexpensive, compact diagnostics ultrasound and Hyperfine with inexpensive, compact MRI diagnostic devices.  Then I went to the <a href="https://www.4catalyzer.com/">4Catalyzer site</a> - Rothberg&#39;s incubator - and discovered a bunch of companies I hadn&#39;t heard of or had forgotten about -- Protein Evolution in synthetic biology for plastics production, Detect for home-based diagnostics instruments, AI Therapeutics in the rare disease space and Liminal with what looks like consumer brain scanning.  That&#39;s quite a series of companies!   But the one closest to my heart (sorry QuantumSI :-) is  454.bio, and their announcements have many interesting facets which I&#39;ll dive into.</p><p>[2024-02-06 01:41 - &#39;used&#34;--&gt; iSeq fix -- stupid autocorrect!]</p><h2><span><a name="more"></a></span>New Wine in an Old Bottle</h2><p>Rothberg loves the 454 story and the 454 moniker, and so went and bought all the trademarks back from Roche.  454, after all, launched the first commercial post-Sanger sequencer and also generated the first genome of a previously-named individual, controversial DNA pioneer (and knucklehead) James D. Watson (yes, everybody knew Celera sequenced J.Craig Venter, but there was a claim of anonymity at first).   Whether reviving the name was a great idea is good over-drinks conversation - already I&#39;ve seen confusion as to why anyone would reboot 454 technology -- and 454.bio&#39;s approach has essentially nothing to do with 454&#39;s approach.  </p><p>In the molecular biology analytics space, Rothberg has been responsible for founding 4  companies before 454.bio, starting with what I&#39;ll refer to after this as 454 classic.</p><p>Raindance was the least successful, due to never finding a good market for their really cool picoliter droplet technology.  They first tried to go after target enrichment for NGS, but soon were plowed over by in-solution technologies from Agilent and Nimblegen.  Later they pivoted to digital PCR, but that market was still developing and the Raindance instrument was too big and expensive.  Perhaps there was another product pivot or two in between.  Eventually it was sold for IP to BioRad.</p><p>Ion Torrent did not achieve the lofty goals it set out to hit - we never saw a single chip generate an entire human genome - but did create a working sequencer and was enough of a threat that Illumina brought out MiSeq to compete with the PGM.  Ion Torrent was also the first non-optical detection NGS instrument, and found at least some showcase uses (if not much market) in places where optical sequencers wouldn&#39;t perform well, such as on a boat.  And Ion carved out a lucrative niche in genetic panel testing, driven by being paired with AmpliSeq multiplex PCR technology - another great example (like Nextera which I cited in my previous piece) of lock-down a complementary technology driving market share.  But even that wasn&#39;t enough - eventually ThermoFisher capitulated and licensed AmpliSeq for Illumina platforms.  Ion is still on the market, but hasn&#39;t innovated in years in terms of performance.  Indeed, it was around the time Rothberg left that performance improvement tanked; he picked a good time to leave.</p><p>QuantumSI, which I&#39;ll cover sometime in the future, is the first (and so far only) entrant in next-gen protein sequencing.  There&#39;s an important parallel between the technical approaches of QuantumSI and 454.bio: one pot operation.</p><h2>One Pot Sequencing</h2><p>Every short read sequencer to date has used a cyclic chemistry scheme.  Each base (or series of the same base for flow chemistries such as 454classic, Ion Torrent, Genapsys and Ultima) requires adding reagents, and then there are stripping steps (for reversible terminator schemes) and washes.  Then another cycle happens.  Each cycle is distinct and requires fresh reagents.  </p><p>Delivering those reagents requires microfluidic pumps and lines.  Delivery to the flowcell can be dogged by challenges of even dispersal; Ion Torrent raw signals had patterns of light and dark looking like flow lines.  The smaller Ion chips had almond-shaped gaskets which lost much surface area but had more uniform fluid flow.  Later, higher capacity chips tried to go for something more like a square to maximize area, but the corners could be troublesome -- and if reagent from one cycle hung up there it would contaminate the next cycle.  </p><p>Also troublesome is that the reagents are very highly concentrated, to drive the reaction to near completion.  But only tiny amounts of the volume actually go anywhere useful and very little of those expensive reagents actually are consumed by the sequencing reactions; most are just flushed away each cycle.  That adds to the expense.</p><p>454.bio is introducing &#34;One Pot Sequencing&#34; - there are no cycles of reagents.  QuantumSI&#39;s chemistry has a similar notion, though that is a degradative not synthesis chemistry, but again no cycles -- the &#34;one pot&#34; of reagents runs the reactions continuously.   454.bio is using a reversible terminator chemistry, except instead of using chemical cleavage to remove the terminator and fluorescent moieties, ultraviolet light is employed.  These &#34;Lightning Terminators&#34; were once the basis of LaserGen - a company that was acquired by Agilent and then somewhat quietly shuttered.  LaserGen was developing a more conventional cycling instrument, but with very fast cycle times.</p><p>Like all other optical readout short read sequencers (except 454!), 454.bio employs Total Internal Reflection Fluorescence (TIRF) microscopy.  The basic idea of TIRF is that if you aim light at the right angle, by the precepts of conventional physics it will all reflect off the interface between the glass surface and the reagents above -- but in quantum land the light&#39;s field extends a bit beyond that surface - evanescent illumination.  So only a tiny volume of reagent will give any signal - the tiny volume that contains polonies.  PacBio uses a similar logic with Zero Mode Waveguides - when they are imaging fluorescence can only occur close to the surface and so the huge background of unincorporated nucleotides and released labels is essentially invisible.  </p><p>For 454.bio, this is also how the removal of the fluorophore/terminating moiety occurs - a cycle of illumination with the appropriate wavelength of light to cleave the terminator moiety.  Lighting Terminators are 3&#39; unblocked; the 3&#39; hydroxyl is always there but steric effects prevent the polymerase from accessing it.  So the evanescent illumination is critical again - you don&#39;t want to be deblocking all the terminators that haven&#39;t been incorporated.  Of course, there will be free terminators that do get into the critical zone -- such as blank areas with no polony or diffusion into the polony and so forth.  So 454.bio will build up problematic unterminated nucleotide triphosphates after every deblock cycle.  </p><p>And that is their current problem - read lengths are in the single digits because the polymerase they are using prefers the unterminated nucleotides to terminated ones, so after a few cycles of deblocking there&#39;s enough unterminated nucleotides to cause serious dephasing -- the molecules in the polony are no longer moving in lockstep.  In one of their unconventional steps (more on this below), 454.bio has announced a contest for improving the discrimination of the polymerase.  If you win by delivering a much improved polymerase a year from now, you get $200K  and can patent and/or public - but grant 454.bio a royalty-free license to use your invention and they have the rights to further engineer based on your design.  </p><p>Here&#39;s a plot of the <a href="https://454.bio/blog/2023/12/18/s0592-sequencing-results/">most recent sequencing run described on their blog</a>.  I&#39;d still prefer to see the error rate plotted log scale, but in this range it&#39;s not that important a point.  Claim is those first five bases are in the phred 20-25 range.  Plotting it log scale would help resolve to what degree is that jump from 4 to 5 truly a discontinuity or is it where the trend was going.  </p><div><p><a href="https://blogger.googleusercontent.com/img/a/AVvXsEjJu7bl8nCoH0HzxA06xxC8EkOlXqZQZVpMXYnN05jtX8yNxYVSUYRX06ay6FWSn_GA7EAaPksTrMDHG3liFjedlT-_4dXTvp1ZOe051G8_Z563K8Mvisc5iwebnCIVnS2sx3gaPNLfCUAogItEpPy7be7hNQgH2EUKNU1GT_l0I_moyApCs_P_lw"><img alt="" data-original-height="339" data-original-width="480" height="226" src="https://blogger.googleusercontent.com/img/a/AVvXsEjJu7bl8nCoH0HzxA06xxC8EkOlXqZQZVpMXYnN05jtX8yNxYVSUYRX06ay6FWSn_GA7EAaPksTrMDHG3liFjedlT-_4dXTvp1ZOe051G8_Z563K8Mvisc5iwebnCIVnS2sx3gaPNLfCUAogItEpPy7be7hNQgH2EUKNU1GT_l0I_moyApCs_P_lw" width="320"/></a></p></div><div><p><a href="https://blogger.googleusercontent.com/img/a/AVvXsEjBrwdcerJnSkC9bY10hDUC8THFpSOM9BAL6WccajQ3qOExXl6fOkEbSaZgCmiWmZ3PKBpPNAeICeMSMLFNeJHNXilKIWeS-EVi5YkVUK2yMreUlLS5uqLkkxXgn-mtHPFM1H5Wv_zcDnX3Y8IOszeWWAqH6US8zQ10HpMRt1k-4UCtVQQZ91BwHw"><img alt="" data-original-height="180" data-original-width="475" height="121" src="https://blogger.googleusercontent.com/img/a/AVvXsEjBrwdcerJnSkC9bY10hDUC8THFpSOM9BAL6WccajQ3qOExXl6fOkEbSaZgCmiWmZ3PKBpPNAeICeMSMLFNeJHNXilKIWeS-EVi5YkVUK2yMreUlLS5uqLkkxXgn-mtHPFM1H5Wv_zcDnX3Y8IOszeWWAqH6US8zQ10HpMRt1k-4UCtVQQZ91BwHw" width="320"/></a></p></div><p>By the way, how are polonies (or clusters, if you prefer) formed?  454.bio is using circular library molecules which are isothermally amplified by something I see as rolling circle amplification (RCA) crossed with bridge amplification.  The surface has forward and reverse primers covalently bound, and initial amplification is by one of those primers driving an RCA reaction - but those products can now find other bound primers to drive more reactions and so on.  The result is a hyperbranched structure.  The reverse primers all contain deoxyuracil, so a USER reaction destroys these to loosen up the snarl.  Note that because each cluster results from multiple priming events, this style of cluster generation will make copies of copies, unlike pure RCA cluster generation on platforms such as AVITI and Complete Genomics.</p><h2>Unconventional Company</h2><p>Going open source is certainly not the usual strategy for a company - most companies stay in stealth mode, then get some alpha sites without revealing much to the world - Ultima&#39;s technology first ran at the Broad in a locked room that only a select cadre of Broad employees had access to or even knew existed.  Oxford Nanopore went a bit differently by <a href="https://omicsomics.blogspot.com/2012/02/oxford-nanopore-doesnt-disappoint.html">suddenly destealthing at AGBT 2012</a>, then going relatively quiet for two years, but then executing the MinION Access Program (MAP) for many researchers around the world.  </p><p>Appealing to the idea of garage hobbyists has an almost romantic appeal.  When I was about 8, my brother and father got a kit-based computer for a few hundred dollars when there were many such kits and no fully functional home computers yet.  Ours was a <a href="http://retro.hansotten.nl/6502-sbc/datac-1000-a-tim-6502-sbc-from-1976/">DATAC-1000</a>, the input method was a series of metal pads and the only output device was a row of LEDs above the metal pads.  Well, it had a cassette tape interface for storing programs.  There were others like the KIM-1 and the Altair and some tree fruit named one we started hearing about.  For at most a few thousand dollars you could flesh these out with calculator-style keypad+display or even simple video interfaces.  From that era came many early programmers and hardware tinkerers -- big brother put together our video display (he may have even designed it).   In terms of financial outlays, a serious computer building habit cost not much different than diving into 35mm photography.</p><p>Sequencing technology seems so digital and potentially as impactful, plus there is so much burnishing of the legend of the early computers or companies like Hewlett-Packard that started in garages.  Ion Torrent tried to tap into this, but the catch was that their claimed price of $50K isn&#39;t hobbyist money - and the real upfront outlay was $100K.  It&#39;s also a lot harder to get things running the first time - with some coaching an 8-year old me could write a simple 6502 machine code program to add two numbers but making a sequencing library is a much bigger lift.  Plus all the accessories you need.  Those old computing days you mostly got by with pliers and a soldering iron (we did have not one but two oscilloscopes - didn&#39;t everyone in the 1970s have one in their house?).  But for even the most basic library prep, you must have temperature control of liquids, a complete set of pipettors, a minfuge and probably a few more bits.  </p><p>There was a <a href="https://omicsomics.blogspot.com/2009/10/pondering-polonators.html">build-it-from-plans sequencer called the Polonator</a>, but at the time it cost around $200K to put one together.   Some academics did so, but it never had a large user base - and then the supplier of reagents was bought by QIAGEN and that was the end of Polonator. </p><p>Oxford Nanopore tried to tap into the vibe with MinION, but discovered its not easy.  They certainly reaped large dividends from the MAP getting MinIONs into the hands of many early career folks who didn&#39;t have a vested interest in sustaining the Illumina ecosystem and could make a name for themselves pioneering nanopore sequencing.  So folks like Josh Quick, Nick Loman, Miten Jain, Matt Loose (and far too many I&#39;ll omit here - my apologies!) held on through ONT&#39;s initial unreliability and zigzagging platform changes and showed novel applications and strengths of nanopore sequencing and shared their protocols and software with the world.  But for better or worse, you didn&#39;t actually build any of the hardware.</p><p>Mid last decade I did have one sequencer startup offer to send me plans to build a very low cost instrument -- they claimed around $1000 in parts.  It never came to fruition - and I&#39;m not sure sending me lab plans made much sense though I was game to try it out, but the idea has been out there before.  The NDA I had with them is probably enforceable, so I won&#39;t spill who it was (sorry!).</p><p>Now 454.bio is really, truly releasing complete plans.  I&#39;m not sure I&#39;m capable of quite following them and I would expect that early users will be sending in pointers about where they can be improved.  But the directions are there as are the design files for the many 3D-printed parts in the instrument.  </p><p>I tried my best to price out the most expensive components - it would have been nice if this happened about two years ago as I could have asked a favor of my nephew, who was in customer support at one of the optical houses listed in the parts inventory.  So please check yourself.  The biggest ticket item is the camera, which I have at $700 -- but the description in the parts brings up many variations with variation in price range -- perhaps one thing 454.bio could nail down a bit better.  There&#39;s a positioning table for $300, four different bandpass filters for $1000 total ($250 each), and 16 UV LEDs that add up to $130.  No other component seems to be more than $100, and my total is up to about $2500.  That doesn&#39;t include things that look inexpensive like nuts and bolts and such or the 3D printing, but suppose that somehow added $500.  $3K is less than it can be easy to spend on a mirrorless digital camera and a few lenses for it. So not unreasonable.</p><p>The announcement shows some signs of being rushed - one image talked about &#34;complimentary strands&#34; of DNA in two different locations. But worst, the original price in the store for the critical reversible terminator mix was $33,999!  That&#39;s a hell of a lot more than I ever spent on film, photo prints or photo paper! After pushback, Rothberg declared &#34;typo&#34;  and the price changed to $1299  - the Levenshtein distance between that and 33,999 engenders skepticism of the typo explanation.  That $1299 is said to enable 60 runs.  That was the advantage of the old kit computers - once you bought them the only consumable was blank cassette tapes, which were cheap, reusable and you didn&#39;t need many. </p><p>You&#39;ll also need the sequencing reservoir components, $49.99 for a pack of 5.  As my colleague (and synthetic biology legend) Tom Knight pointed out on X/Twitter, the finishing directions for the reservoirs aren&#39;t for amateurs, involving several solvents and something he called &#34;piranha&#34;.  Yikes!  Rothberg responded that they&#39;ll work towards something more consumer friendly.   So that brings running costs to about $33 per run - $23 dollars of terminators and $10 for the reservoir (note: unlike their store, I like to round!)</p><h2>What&#39;s The Market?</h2><p>In the short term, you&#39;ll need to be very interested in playing with a novel technology that doesn&#39;t really do very much.  The early kit computers, and even the first all-in-one home computers such at PET and TRS-80, weren&#39;t much better - you could learn to program or you could play games, but not very much else no matter how many journalists tried to claim it could organize recipes or help prepare taxes -- that was all long in the future (and who organizes recipes when you can just google them?).  With the current tiny readlength, you&#39;ll be very hard pressed to get much interesting biology out - though it will certainly be hyped as &#34;go explore the biosphere&#34;.  With custom sequencing primers, it should be possible to make those 5 or so bases count, but the current 454.bio don&#39;t make the process for designing such obvious.</p><p>It may well be that many people will be excited to through their machine learning skills at trying to improve the basecalling and deal with the rampant phasing.  It sounds like some diversity of the initial bases is required for good cluster finding - a common aspect of systems using unpatterned flowcells - but it would seem that a set of templates could be cloned (don&#39;t want oligo synthesis errors confounding) and sequenced conventionally, then used as 454.bio targets.  Each one should have the first 5 bases as a barcode that defines the rest of the sequence -- I&#39;ll leave figuring out the number of distinct barcodes with different Hamming or Levenshtein distances as an exercise for the reader..  Perhaps some big open repositories will be built for 454.bio data, to increase the training set sizes.  And perhaps an advanced machine learning algorithm could make the downstream snarl useful for some basic tasks like &#34;which RNA virus&#34; is this -- at least if you only want &#34;flu A vs COVID vs RSV&#34; level differentiation.</p><p>Longer term, if the phasing issue can be solved and reads could get in the 50 to 100 range, I could imagine diagnostics applications.  Maybe.  <span face="Arial, sans-serif">It will depend in part on how many reads per floral - I didn’t see that described but it’s likely to change over the evolution of the chemistry and software. Clearly some evolution of the hardware will be required to get to the goal of $100 a box. </span></p>
</div></div>
  </body>
</html>
