<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <title>James Routley | Feed</title>
    <link
      rel="stylesheet"
      type="text/css"
      href="../styles.css"
      media="screen"
    />
  </head>
  <body>
    <a href="/index.html">Back</a>
    <a href="https://bioinformatics.stackexchange.com/questions/11227/why-does-the-sars-cov2-coronavirus-genome-end-in-aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa">Original</a>
    <h1>Why does the SARS-Cov2 genome end in aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa? (2020)</h1>
    
    <div id="readability-page-1" class="page"><div itemprop="text">
<p>This question is quite general, so I&#39;m going to attempt to tie it back to bioinformatics.</p>

<p><strong>Background</strong>
The tree for the current coronavirus is <a href="https://www.ecohealthalliance.org/2020/01/phylogenetic-analysis-shows-novel-wuhan-coronavirus-clusters-with-sars" rel="noreferrer">here</a>, showing it is closely related to bat-coronavirus and in particular SARS. </p>

<p><strong>Question</strong>
The bioinformatics question for the current coronavirus is why this virus appears to be able to infect humans and transmit to human.</p>

<p><strong>Genome size</strong>
Firstly, you said that 30kb was large ... this is a standard size for a coronavirus genome, albeit it is unusual in that the family Coronaviridae are the largest genomes for a single stranded RNA virus, for example flaviviruses are 10kb. Thus, all coronaviruses are all approximately 30Kb. Some coronaviruses don&#39;t infect humans (zero symptoms), some cause very mild symptoms, others are MERS and SARS with 40-60% and 10% mortality rates, respectively. So, genome size is of little bioinformatics interest in my opinion.</p>

<p><strong>Polyadenylation</strong>
Polyadenylation and capping (5&#39; methylation) enable the RNA to be trafficked and transcribed by ribosomes and the mechanism is widely used by viruses. Methylation would also prevent the innate immune response from the shredding the vRNA. <a href="https://www.pnas.org/content/107/8/3283#ref-1" rel="noreferrer">Koonin and Moss (2010)</a>, interpreted a given capping mechanism as being common to the Mononegavirales - a viral Order including measles, mumps, Ebolavirus. Its a big statement, but regardless poly-A and capping are simply mimicking the host mRNA which a lot of viruses use. Poly-A and capping per se are not really interesting.</p>

<p><strong><a href="https://bioinformatics.stackexchange.com/questions/11250/ancestry-of-the-coronavirus-2019-ncov-wuhan-city-china">Evolution and SARS</a></strong> A more detailed examination the evolution of 2019-nCov and its epidemiology in relation to SARS can be found <a href="https://bioinformatics.stackexchange.com/questions/11250/ancestry-of-the-coronavirus-2019-ncov-wuhan-city-china">here</a></p>

<p><strong>Conclusion</strong>
The bioinformatics question is the genome size wierd - no, its standard for a coronavirus, is the poly-A weird - no its generic amongst lots of viruses as is capping. Is the length of the poly-A excessive (33 As), it looks odd  but a human genecist/bioinformaticist needs to answer that ... so is it (potentially) linked with its epidemiology/clinical symptoms?</p>

<p>I don&#39;t think 33 poly-As are linked with anything bioinformatically interesting. This is because it will likely vary dramatically between genomes (not simply epidemic vs. non-epidemic strains). I don&#39;t know the mechanism for poly-adenylation, but I think slippage is a likely mutation resulting in large variations between individual genomes, particularly for poly-A - which notorious for slippage. </p>

<p>So ultimately could poly-As be linked with the ability of the new coronavirus to infect/transmit and could we therefore explore that bioinformatically? I personally think slippage mutations would prevent a clonal lineage emerging, i.e. that the size of the poly-As is not stable between genomes, but that would assume a given given mechanism of polyadenylation. Thus as a bioinformatics question I wouldn&#39;t pursue it, because I don&#39;t think there is sufficient biological rationale. I agree weird stuff should be questioned and that bit of the genome jumps out ... but I doubt it would go anywhere.</p>

<p><strong>Slippage</strong>
The definition of a slippage mutation is <a href="https://en.m.wikipedia.org/wiki/Slipped_strand_mispairing" rel="noreferrer">here</a>, but basically it means this genome has 33 poly-As, however another isolate from the same epidemic could say have 30 poly-As (just an example), another might have 25 poly-As and so on.</p>

<p>Just my 2 cents</p>
    </div></div>
  </body>
</html>
