<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <title>James Routley | Feed</title>
    <link
      rel="stylesheet"
      type="text/css"
      href="../styles.css"
      media="screen"
    />
  </head>
  <body>
    <a href="/index.html">Back</a>
    <a href="https://www.nature.com/articles/s41565-023-01496-y">Original</a>
    <h1>Wireless electrical–molecular quantum signalling for cancer cell apoptosis</h1>
    
    <div id="readability-page-1" class="page"><div>
                <section data-title="Main"><div id="Sec1-section"><h2 id="Sec1">Main</h2><div id="Sec1-content"><p>We are entering an era where it has been realized that bioelectricity, defined as the electrical language of cells, programs cell function<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 1" title="McLaughlin, K. A. &amp; Levin, M. Bioelectric signaling in regeneration: mechanisms of ionic controls of growth and form. Dev. Biol. 433, 177–189 (2018)." href="https://bytes.zone/articles/s41565-023-01496-y#ref-CR1" id="ref-link-section-d444612993e752">1</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 2" title="Schofield, Z. et al. Bioelectrical understanding and engineering of cell biology. J. R. Soc. Interface 17, 20200013 (2020)." href="https://bytes.zone/articles/s41565-023-01496-y#ref-CR2" id="ref-link-section-d444612993e755">2</a></sup>. The cell is increasingly viewed as a mass of bioelectrical interconnected circuits that use an endogenous current generated by electron transfer processes to communicate with each other to maintain homoeostasis<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 3" title="Levin, M. Bioelectric signaling: reprogrammable circuits underlying embryogenesis, regeneration, and cancer. Cell 184, 1971–1989 (2021)." href="https://bytes.zone/articles/s41565-023-01496-y#ref-CR3" id="ref-link-section-d444612993e759">3</a></sup>. These electron transfer processes in biology, when governed by quantum mechanical effects such as electron tunnelling, are classified as quantum biological tunnelling for electron transfer (QBET). For instance, one of the most well-known electron transfer pathways is photosynthesis, the mechanism of which was one of the first to be linked to quantum mechanical effects<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 4" title="Lambert, N. et al. Quantum biology. Nat. Phys. 9, 10–18 (2013)." href="https://bytes.zone/articles/s41565-023-01496-y#ref-CR4" id="ref-link-section-d444612993e763">4</a></sup>. Cytochrome <i>c</i> (Cyt <i>c</i>) is known for its vital role in the mitochondrial electron transfer chains that are governed by the redox activity (Fe<sup>2+</sup> to Fe<sup>3+</sup>, and vice versa) of the haem ring present in its structure<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 5" title="Garrido, C. et al. Mechanisms of cytochrome c release from mitochondria. Cell Death Differ. 13, 1423–1433 (2006)." href="https://bytes.zone/articles/s41565-023-01496-y#ref-CR5" id="ref-link-section-d444612993e778">5</a></sup>. These redox processes in Cyt <i>c</i> are also known to occur via electron tunnelling<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Devault, D., Parkes, J. H. &amp; Chance, B. Electron tunnelling in cytochromes. Nature 215, 642–644 (1967)." href="#ref-CR6" id="ref-link-section-d444612993e785">6</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Wuttke, D. S., Bjerrum, M. J., Winkler, J. R. &amp; Gray, H. B. Electron-tunneling pathways in cytochrome c. Science 256, 1007–1009 (1992)." href="#ref-CR7" id="ref-link-section-d444612993e785_1">7</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 8" title="Xin, H. et al. Quantum biological tunnel junction for electron transfer imaging in live cells. Nat. Commun. 10, 3245 (2019)." href="https://bytes.zone/articles/s41565-023-01496-y#ref-CR8" id="ref-link-section-d444612993e788">8</a></sup> and are regulated by the Warburg effect, which is underpinned by bioelectrical faradaic currents<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 9" title="Vaughn, A. E. &amp; Deshmukh, M. Glucose metabolism inhibits apoptosis in neurons and cancer cells by redox inactivation of cytochrome c. Nat. Cell Biol. 10, 1477–1483 (2008)." href="https://bytes.zone/articles/s41565-023-01496-y#ref-CR9" id="ref-link-section-d444612993e792">9</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 10" title="Robinson, A. J. et al. Toward hijacking bioelectricity in cancer to develop new bioelectronic medicine. Adv. Therapeutics 4, 2000248 (2021)." href="https://bytes.zone/articles/s41565-023-01496-y#ref-CR10" id="ref-link-section-d444612993e795">10</a></sup>. Furthermore, these processes are crucial for the translocation of Cyt <i>c</i> to the cytosol<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 11" title="Kuwana, T. et al. Bid, Bax, and lipids cooperate to form supramolecular openings in the outer mitochondrial membrane. Cell 111, 331–342 (2002)." href="https://bytes.zone/articles/s41565-023-01496-y#ref-CR11" id="ref-link-section-d444612993e803">11</a></sup> and for modulating its binding conformation<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 12" title="Brown, G. C. &amp; Borutaite, V. Regulation of apoptosis by the redox state of cytochrome c. Biochim. Biophys. Acta Bioenerg. 1777, 877–881 (2008)." href="https://bytes.zone/articles/s41565-023-01496-y#ref-CR12" id="ref-link-section-d444612993e807">12</a></sup> for interaction with apoptotic protease activating factor 1 (APAF-1)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 13" title="Zou, H., Henzel, W. J., Liu, X., Lutschg, A. &amp; Wang, X. Apaf-1, a human protein homologous to C. elegans CED-4, participates in cytochrome c–dependent activation of caspase-3. Cell 90, 405–413 (1997)." href="https://bytes.zone/articles/s41565-023-01496-y#ref-CR13" id="ref-link-section-d444612993e811">13</a></sup>. Although quantum biology is still in its infancy, we envisage future interdisciplinary developments, which will be reinforced by the understanding and modulation of these quantum mechanical processes<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 14" title="Cordier, B. A., Sawaya, N. P., Guerreschi, G. G. &amp; McWeeney, S. K. Biology and medicine in the landscape of quantum advantages. J. R. Soc. Interface 19, 20220541 (2022)." href="https://bytes.zone/articles/s41565-023-01496-y#ref-CR14" id="ref-link-section-d444612993e815">14</a></sup>. This highlights the need for technology to electrically interact with redox molecules such as Cyt <i>c</i> whose functions are supported by quantum mechanical processes.</p><p>Remotely induced electrical–molecular communication (electrical input causing a redox change in a targeted molecule, inducing an alteration in cell behaviour) inside cells opens the possibility of creating disruptive technologies, including the development of quantum medicines for cancer treatments<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 15" title="Bordonaro, M. Quantum biology and human carcinogenesis. Biosystems 178, 16–24 (2019)." href="https://bytes.zone/articles/s41565-023-01496-y#ref-CR15" id="ref-link-section-d444612993e825">15</a></sup>. However, on-demand targeted electrical–molecular communication within cells has yet to be realized. This is a result of the lack of technological innovation that is suitable for interfacing with cells in a medium at a spatial/temporal level in which native biological interaction occurs. The aim of this study is to pioneer solutions to these challenges.</p><p>Gold nanoparticles (GNPs) and carbon nanotubes can behave as bipolar nanoelectrodes<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 16" title="Sanjuan-Alberte, P. et al. Wireless nanobioelectronics for electrical intracellular sensing. ACS Appl. Nano Mater. 2, 6397–6408 (2019)." href="https://bytes.zone/articles/s41565-023-01496-y#ref-CR16" id="ref-link-section-d444612993e832">16</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 17" title="Warakulwit, C. et al. Dissymmetric carbon nanotubes by bipolar electrochemistry. Nano Lett. 8, 500–504 (2008)." href="https://bytes.zone/articles/s41565-023-01496-y#ref-CR17" id="ref-link-section-d444612993e835">17</a></sup> within cells when external electric fields (EFs) are applied. The bipolar nanoelectrodes become polarized when an EF is applied, leading to a voltage gradient across the particle<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 18" title="Lei, X., Lei, C., Chen, W., Guo, Q. &amp; Huang, B. Photoelectrochemically driven nanoparticulate semiconductors as nanobipolar electrodes for advanced water remediation. Cell Rep. Phys. Sci. 3, 101132 (2022)." href="https://bytes.zone/articles/s41565-023-01496-y#ref-CR18" id="ref-link-section-d444612993e839">18</a></sup>. With a sufficiently large potential difference between the poles of the electrode, the thermodynamic driving force can cause electrochemically induced redox reactions to occur by virtue of bipolar electrochemistry (also known as wireless electrochemistry)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 19" title="Loget, G. &amp; Kuhn, A. Electric field-induced chemical locomotion of conducting objects. Nat. Commun. 2, 535 (2011)." href="https://bytes.zone/articles/s41565-023-01496-y#ref-CR19" id="ref-link-section-d444612993e843">19</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 20" title="Forster, R. J. Wirefree electroceuticals: 3D electrical and electrochemical stimulation of biological systems. Curr. Opin. Electrochem. 39, 101297 (2023)." href="https://bytes.zone/articles/s41565-023-01496-y#ref-CR20" id="ref-link-section-d444612993e846">20</a></sup>. It was thought that nanoscale wireless electrochemistry was not possible in the presence of cells even at very high applied voltages<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 21" title="Loget, G., Roche, J. &amp; Kuhn, A. True bulk synthesis of Janus objects by bipolar electrochemistry. Adv. Mater. 24, 5111–5116 (2012)." href="https://bytes.zone/articles/s41565-023-01496-y#ref-CR21" id="ref-link-section-d444612993e850">21</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 22" title="Loget, G. &amp; Kuhn, A. Shaping and exploring the micro- and nanoworld using bipolar electrochemistry. Anal. Bioanal. Chem. 400, 1691–1704 (2011)." href="https://bytes.zone/articles/s41565-023-01496-y#ref-CR22" id="ref-link-section-d444612993e853">22</a></sup>. Importantly, most recently, bipolar electrodes in series lead to a dramatic drop in cell impedance<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 23" title="Robinson, A. J. et al. Impedimetric characterization of bipolar nanoelectrodes with cancer cells. ACS Omega 6, 29495–29505 (2021)." href="https://bytes.zone/articles/s41565-023-01496-y#ref-CR23" id="ref-link-section-d444612993e857">23</a></sup>, which may allow electrical inputs to be used without inducing cell damage. Redox reactions at carbon nanotube porins, acting as bipolar nanoelectrodes, can occur at unprecedentedly low applied voltages in cells without inducing direct cell death<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 24" title="Hicks, J. M. et al. Electric field induced biomimetic transmembrane electron transport using carbon nanotube porins. Small 17, 2102517 (2021)." href="https://bytes.zone/articles/s41565-023-01496-y#ref-CR24" id="ref-link-section-d444612993e862">24</a></sup>. We, therefore, designed an approach that uses bifunctionalized bipolar nanoelectrodes with attached redox-active molecules, which we term bio-nanoantennae (they are capable of receiving a remotely applied external EF input and converting this to bio-signalling events). Furthermore, alternating current (a.c.) EFs are reported to notably lower the impedance of the cell membrane at higher frequencies to penetrate the cytoplasm<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 25" title="Hart, F. X. &amp; Palisano, J. R. The application of electric fields in biology and medicine. Electr. Field 23, 161–186 (2018)." href="https://bytes.zone/articles/s41565-023-01496-y#ref-CR25" id="ref-link-section-d444612993e866">25</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 26" title="Taghian, T., Narmoneva, D. &amp; Kogan, A. Modulation of cell function by electric field: a high-resolution analysis. J. R. Soc. Interface 12, 20150153 (2015)." href="https://bytes.zone/articles/s41565-023-01496-y#ref-CR26" id="ref-link-section-d444612993e869">26</a></sup>, which together with the enhanced voltage gradient observed at the nanoscale would facilitate bipolar electrochemistry<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 27" title="Dickinson, E. J. &amp; Compton, R. G. Diffuse double layer at nanoelectrodes. J. Phys. Chem. C 113, 17585–17589 (2009)." href="https://bytes.zone/articles/s41565-023-01496-y#ref-CR27" id="ref-link-section-d444612993e873">27</a></sup>. Therefore, we hypothesized that these bio-nanoantennae in combination with applied a.c. EFs could be used to modulate electron transfer, which then could be converted into molecular actuation via targeting a specific metabolic pathway (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bytes.zone/articles/s41565-023-01496-y#Fig1">1</a>).</p><div data-test="figure" data-container-section="figure" id="figure-1" data-title="Illustration of wireless electrical–molecular quantum signalling mediated by a.c.-EF-responsive bio-nanoantennae to induce cell death."><figure><figcaption><b id="Fig1" data-test="figure-caption-text">Fig. 1: Illustration of wireless electrical–molecular quantum signalling mediated by a.c.-EF-responsive bio-nanoantennae to induce cell death.</b></figcaption><div><div><a data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="https://bytes.zone/articles/s41565-023-01496-y/figures/1" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41565-023-01496-y/MediaObjects/41565_2023_1496_Fig1_HTML.png?as=webp"/><img aria-describedby="Fig1" src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41565-023-01496-y/MediaObjects/41565_2023_1496_Fig1_HTML.png" alt="figure 1" loading="lazy" width="685" height="782"/></picture></a></div><p><b>a</b>, Bio-nanoantennae (GNP100@r.Cyt <i>c</i>@Z) were synthesized by covalently conjugating r.Cyt <i>c</i> and Z to carboxylic GNP100s using <i>N</i>-(3-dimethylaminopropyl)-N′-ethylcarbodiimide hydrochloride (EDC)/<i>N</i>-hydroxysuccinimide (NHS) chemistry. <b>b</b>, Primary patient-derived GBM cells, namely, GIN (derived from the GBM infiltrative margin) and GCE (derived from the GBM proliferative core) cells, were incubated with bio-nanoantennae to enable their uptake. These GBM cells were electrically stimulated with a.c. EFs of 3 MHz at 0.65 V cm<sup>–1</sup>. V, voltage; ES, electrical stimulation; Ox, oxidized; Red, reduced. <b>c</b>, These applied a.c. EFs caused the intracellular wireless electrochemistry to occur at the bio-nanoantennae surface to induce caspase-3/7-mediated apoptosis of GBM cells. We examined the electrical–molecular signalling by connecting gene regulation with the a.c.-EF-mediated cell death in GBM cells. <b>d</b>, The a.c. EFs were applied to induce wireless electrochemistry at the surface of bio-nanoantennae to wirelessly switch the redox state of Cyt <i>c</i> (reduced to oxidized), ex situ, suggesting QBET in the proposed system. e<sup>–</sup>, electron. <b>e</b>, Diagram representing the a.c.-EF-responsive bio-nanoantennae for electrical–molecular quantum signalling with GBM cells. Protein Data Bank (PDB) structural entries were used to represent Cyt <i>c</i> (PDB <a href="https://doi.org/10.2210/pdb1HRC/pdb">1HRC</a>; refs. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 39" title="Bushnell, G. W., Louie, G. V. &amp; Brayer, G. D. High-resolution three-dimensional structure of horse heart cytochrome c. Protein Data Bank, 
                https://doi.org/10.2210/pdb1HRC/pdb
                
               (2021)." href="https://bytes.zone/articles/s41565-023-01496-y#ref-CR39" id="ref-link-section-d444612993e937">39</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 40" title="Bushnell, G. W., Louie, G. V. &amp; Brayer, G. D. High-resolution three-dimensional structure of horse heart cytochrome c. J. Mol. Biol., 
                https://doi.org/10.1016/0022-2836(90)90200-6
                
               (1990)." href="https://bytes.zone/articles/s41565-023-01496-y#ref-CR40" id="ref-link-section-d444612993e940">40</a></sup>), apoptosome (PDB <a href="https://doi.org/10.2210/pdb3J2T/pdb">3J2T</a>; refs. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 41" title="Yuan, S., Topf, M. &amp; Akey, C. W. An improved model of the human apoptosome. Protein Data Bank, 
                https://doi.org/10.2210/pdb3J2T/pdb
                
               (2018)." href="https://bytes.zone/articles/s41565-023-01496-y#ref-CR41" id="ref-link-section-d444612993e951">41</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 42" title="Yuan, S., Topf, M., Reubold, T. F., Eschenburg, S. &amp; Akey, C. W. Changes in Apaf-1 conformation that drive apoptosome assembly. Biochemistry, 
                https://doi.org/10.1021/bi301721g
                
               (2013)." href="https://bytes.zone/articles/s41565-023-01496-y#ref-CR42" id="ref-link-section-d444612993e954">42</a></sup>) and caspase 3/7 (PDB <a href="https://doi.org/10.2210/pdb3GJQ/pdb">3GJQ</a>; refs. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 43" title="Fang, B., Fu, G., Agniswamy, J., Harrison, R. W. &amp; Weber, I. T. Caspase-3 binds diverse P4 residues in peptides. Protein Data Bank, 
                https://doi.org/10.2210/pdb3GJQ/pdb
                
               (2011)." href="https://bytes.zone/articles/s41565-023-01496-y#ref-CR43" id="ref-link-section-d444612993e966">43</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 44" title="Fang, B., Fu, G., Agniswamy, J., Harrison, R. W. &amp; Weber, I. T. Caspase-3 binds diverse P4 residues in peptides as revealed by crystallography and structural modeling. Apoptosis, 
                https://doi.org/10.1007/s10495-009-0333-y
                
               (2009)." href="https://bytes.zone/articles/s41565-023-01496-y#ref-CR44" id="ref-link-section-d444612993e969">44</a></sup>). Chemspider structure no. <a href="http://www.chemspider.com/Chemical-Structure.19989078.html">19989078</a> (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 45" title="Zinc porphyrin. ChemSpider 
                http://www.chemspider.com/Chemical-Structure.19989078.html
                
              " href="https://bytes.zone/articles/s41565-023-01496-y#ref-CR45" id="ref-link-section-d444612993e980">45</a></sup>) was used to represent Z. Structure illustrations by Leonora Martínez Nuñez. <b>a</b>,<b>d</b>, Z from ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 45" title="Zinc porphyrin. ChemSpider 
                http://www.chemspider.com/Chemical-Structure.19989078.html
                
              " href="https://bytes.zone/articles/s41565-023-01496-y#ref-CR45" id="ref-link-section-d444612993e990">45</a></sup>; <b>a</b>,<b>c</b>,<b>d</b>, Cyt <i>c</i> from refs. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 39" title="Bushnell, G. W., Louie, G. V. &amp; Brayer, G. D. High-resolution three-dimensional structure of horse heart cytochrome c. Protein Data Bank, 
                https://doi.org/10.2210/pdb1HRC/pdb
                
               (2021)." href="https://bytes.zone/articles/s41565-023-01496-y#ref-CR39" id="ref-link-section-d444612993e1007">39</a></sup><sup>,</sup><sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 40" title="Bushnell, G. W., Louie, G. V. &amp; Brayer, G. D. High-resolution three-dimensional structure of horse heart cytochrome c. J. Mol. Biol., 
                https://doi.org/10.1016/0022-2836(90)90200-6
                
               (1990)." href="https://bytes.zone/articles/s41565-023-01496-y#ref-CR40" id="ref-link-section-d444612993e1011">40</a></sup>; <b>c</b>, Apoptosome from refs. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 41" title="Yuan, S., Topf, M. &amp; Akey, C. W. An improved model of the human apoptosome. Protein Data Bank, 
                https://doi.org/10.2210/pdb3J2T/pdb
                
               (2018)." href="https://bytes.zone/articles/s41565-023-01496-y#ref-CR41" id="ref-link-section-d444612993e1019">41</a></sup><sup>,</sup><sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 42" title="Yuan, S., Topf, M., Reubold, T. F., Eschenburg, S. &amp; Akey, C. W. Changes in Apaf-1 conformation that drive apoptosome assembly. Biochemistry, 
                https://doi.org/10.1021/bi301721g
                
               (2013)." href="https://bytes.zone/articles/s41565-023-01496-y#ref-CR42" id="ref-link-section-d444612993e1023">42</a></sup>; Caspase 3/7 from refs. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 43" title="Fang, B., Fu, G., Agniswamy, J., Harrison, R. W. &amp; Weber, I. T. Caspase-3 binds diverse P4 residues in peptides. Protein Data Bank, 
                https://doi.org/10.2210/pdb3GJQ/pdb
                
               (2011)." href="https://bytes.zone/articles/s41565-023-01496-y#ref-CR43" id="ref-link-section-d444612993e1027">43</a></sup><sup>,</sup><sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 44" title="Fang, B., Fu, G., Agniswamy, J., Harrison, R. W. &amp; Weber, I. T. Caspase-3 binds diverse P4 residues in peptides as revealed by crystallography and structural modeling. Apoptosis, 
                https://doi.org/10.1007/s10495-009-0333-y
                
               (2009)." href="https://bytes.zone/articles/s41565-023-01496-y#ref-CR44" id="ref-link-section-d444612993e1031">44</a></sup>.</p></div></figure></div><p>The strategy to fulfil the development of a quantum electrical–molecular communication tool was multipronged and inspired by the observation that electron transfer in Cyt <i>c</i> is mediated through QBET<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 8" title="Xin, H. et al. Quantum biological tunnel junction for electron transfer imaging in live cells. Nat. Commun. 10, 3245 (2019)." href="https://bytes.zone/articles/s41565-023-01496-y#ref-CR8" id="ref-link-section-d444612993e1049">8</a></sup>. Therefore, we functionalized GNPs with an electron-donor-reduced Cyt <i>c</i> (r.Cyt <i>c</i>) and a redox mediator, zinc porphyrin (zinc 5-(4-aminophenyl)-10,15,20-(tri-4-sulfonatophenyl)porphyrin triammonium; Z). When these particles are exposed to an electrical input, a voltage gradient is induced at the surface; these are therefore described as electronic bio-nanoantennae (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bytes.zone/articles/s41565-023-01496-y#Fig1">1a</a>). We show that the electrical input confers specific signalling in cells via the bio-nanoantennae to induce apoptosis-mediated glioblastoma (GBM) cell death. The transcriptomic analysis enabled elucidation of the biochemical signalling when using the electrical–molecular communication tool (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bytes.zone/articles/s41565-023-01496-y#Fig1">1b,c</a>). We show that on the application of a resonant electrical input (a.c. EF, 3 MHz, 0.65 V cm<sup>–1</sup>), wireless electrochemistry is induced at the bio-nanoantennae surface and switches the redox state of the Cyt <i>c</i>. We propose that the QBET phenomenon and resonant electron transfer between the Cyt <i>c</i> and Z facilitate cellular apoptosis (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bytes.zone/articles/s41565-023-01496-y#Fig1">1d</a>). The data suggest a.c.-EF-induced molecular actuation via bio-nanoantennae in the treatment of a disease, which is a quantum functional medicine tool (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bytes.zone/articles/s41565-023-01496-y#Fig1">1e</a>). This could lead to exciting opportunities for the development of quantum nanomedicines and cancer treatments and provide a state-of-the-art means of modulating cell metabolism.</p></div></div></section><section data-title="Design of bio-nanoantennae for actuating cell death"><div id="Sec2-section"><h2 id="Sec2">Design of bio-nanoantennae for actuating cell death</h2><div id="Sec2-content"><p>To electrically communicate with cells at a level equivalent to that underpinning molecular biochemistry, we initially used carboxylic PEG modified 100 nm spherical GNPs (GNP100s; PEG, polyethylene glycol) as bipolar nanoelectrodes. These GNP100s can sense an EF applied extracellularly<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 16" title="Sanjuan-Alberte, P. et al. Wireless nanobioelectronics for electrical intracellular sensing. ACS Appl. Nano Mater. 2, 6397–6408 (2019)." href="https://bytes.zone/articles/s41565-023-01496-y#ref-CR16" id="ref-link-section-d444612993e1088">16</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 23" title="Robinson, A. J. et al. Impedimetric characterization of bipolar nanoelectrodes with cancer cells. ACS Omega 6, 29495–29505 (2021)." href="https://bytes.zone/articles/s41565-023-01496-y#ref-CR23" id="ref-link-section-d444612993e1091">23</a></sup>. This occurs when they are functionalized with redox-active biomolecules to yield a bio-nanoantenna that can enable surface redox reactions (via wireless electrochemistry) to actuate a cell-specific signalling pathway. Apoptosis is mediated (in part) by Cyt <i>c</i> bioelectrochemistry in which the oxidized state (Fe<sup>3+</sup>) facilitates apoptosis-mediated cell death<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 12" title="Brown, G. C. &amp; Borutaite, V. Regulation of apoptosis by the redox state of cytochrome c. Biochim. Biophys. Acta Bioenerg. 1777, 877–881 (2008)." href="https://bytes.zone/articles/s41565-023-01496-y#ref-CR12" id="ref-link-section-d444612993e1100">12</a></sup>. Therefore, we conjugated GNP100 with an electrical donor, that is, r.Cyt <i>c</i> (inactive, Fe<sup>2+</sup>), and a redox mediator Z using carbodiimide coupling chemistry to form bio-nanoantennae (GNP100@r.Cyt <i>c</i>@Z). On application of a resonant a.c. EF, the bio-nanoantennae would polarize, providing the thermodynamic driving force for nanoscale wireless electrochemistry to occur and switch the redox state of r.Cyt <i>c</i> from Fe<sup>2+</sup> to Fe<sup>3+</sup> (oxidized Cyt <i>c</i> or o.Cyt <i>c</i>)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 28" title="Fosdick, S. E., Knust, K. N., Scida, K. &amp; Crooks, R. M. Bipolar electrochemistry. Angew. Chem. Int. Ed. 52, 10438–10456 (2013)." href="https://bytes.zone/articles/s41565-023-01496-y#ref-CR28" id="ref-link-section-d444612993e1127">28</a></sup>.</p><p>The successful bifunctionalization of carboxylic PEG modified GNP100s (2 kDa PEG) with r.Cyt <i>c</i> and Z is evident from transmission electron microscopy (TEM) images (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bytes.zone/articles/s41565-023-01496-y#Fig2">2a</a>), dynamic light scattering (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://bytes.zone/articles/s41565-023-01496-y#MOESM1">1</a>) and zeta potential (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://bytes.zone/articles/s41565-023-01496-y#MOESM1">2</a>) and UV–visible (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bytes.zone/articles/s41565-023-01496-y#Fig2">2b,c</a> and Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://bytes.zone/articles/s41565-023-01496-y#MOESM1">3a–e</a>) characterizations of GNP100@r.Cyt <i>c</i>@Z. A quantification of redox molecules bound to each nanoparticle is presented in Supplementary Tables <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://bytes.zone/articles/s41565-023-01496-y#MOESM1">1</a>–<a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://bytes.zone/articles/s41565-023-01496-y#MOESM1">4</a>. Cyclic voltammetry was carried out to study the redox behaviour of r.Cyt <i>c</i> and Z on a bifunctionalized system (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bytes.zone/articles/s41565-023-01496-y#Fig2">2d</a> and Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://bytes.zone/articles/s41565-023-01496-y#MOESM1">4a–c</a>). The heterogeneous electron transfer rate coefficient (<i>k</i><sup>0</sup>) of Cyt <i>c</i> for GNP100@r.Cyt <i>c</i> was calculated to be 9.6 × 10<sup>−3</sup> cm s<sup>–1</sup> (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://bytes.zone/articles/s41565-023-01496-y#MOESM1">5</a>), while that for GNP100@r.Cyt <i>c</i>@Z was 3.75 × 10<sup>−3</sup> cm s<sup>–1</sup> (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://bytes.zone/articles/s41565-023-01496-y#MOESM1">5a–f</a>), suggesting a slight decrease in the electron transfer rate of Cyt <i>c</i> in a bifunctionalized system. A detailed discussion is in Supplementary Note <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://bytes.zone/articles/s41565-023-01496-y#MOESM1">1</a>.</p><div data-test="figure" data-container-section="figure" id="figure-2" data-title="Physico-chemical and electro-analytical characterization of bio-nanoantennae and their interaction with patient-derived GBM cells."><figure><figcaption><b id="Fig2" data-test="figure-caption-text">Fig. 2: Physico-chemical and electro-analytical characterization of bio-nanoantennae and their interaction with patient-derived GBM cells.</b></figcaption><div><div><a data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="https://bytes.zone/articles/s41565-023-01496-y/figures/2" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41565-023-01496-y/MediaObjects/41565_2023_1496_Fig2_HTML.png?as=webp"/><img aria-describedby="Fig2" src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41565-023-01496-y/MediaObjects/41565_2023_1496_Fig2_HTML.png" alt="figure 2" loading="lazy" width="685" height="498"/></picture></a></div><div data-test="bottom-caption" id="figure-2-desc"><p><b>a</b>, TEM image of bio-nanoantennae (GNP100@r.Cyt <i>c</i>@Z) prepared by coupling r.Cyt <i>c</i> and Z to GNP100s. The inset is a high-resolution TEM image of 100 nm bio-nanoantennae. The TEM analysis was done on three different samples of bio-nanoantennae that were synthesized over the course of three individual experiments. <b>b</b>, UV–visible absorption spectra of bio-nanoantennae dispersed in phosphate buffer saline (PBS) before ES. <b>c</b>, Zoomed-in UV–visible absorption spectra from 360 to 460 nm showing surface functionalization with r.Cyt <i>c</i> (GNP100@r.Cyt <i>c</i>), Z (GNP100@Z) and both r.Cyt <i>c</i> and Z (GNP100@r.Cyt <i>c</i>@Z). <b>d</b>, Cyclic voltammetry scan rate studies to analyze the redox properties of GNP100@r.Cyt <i>c</i>@Z. Redox potentials were measured using an ITO working electrode, platinum wire counter electrode and Ag/AgCl reference electrode with samples (25 µg mL<sup>–1</sup>) dispersed in 10 mM PBS, scanned from +1.2 V to –0.25 V. ITO, indium tin oxide. <b>e</b>, ICP-MS analysis to determine the association of bio-nanoantennae with different patient-derived GBM cells and cortical astrocytes; the data are expressed as the number of GNPs per cell. Results are expressed as ±s.d. of the mean obtained from a triplicate experiment and repeated three times. The data were considered significant if *<i>P</i> ≤ 0.05, **<i>P</i> ≤ 0.01, ***<i>P</i> ≤ 0.001 and ***<i>P</i> ≤ 0.0001 versus GNP100, obtained using two-way analysis of variance (ANOVA) with a Tukey post-test. NS, not significant.</p><p><a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://bytes.zone/articles/s41565-023-01496-y#MOESM3">Source data</a></p></div></div></figure></div><p>To explore the potential of bio-nanoantennae on the a.c.-EF-mediated redox switching of Cyt <i>c</i>, first we investigated the association and uptake of bio-nanoantennae on four types of preclinical GBM cells, isolated from two GBM patients, glioma invasive margin (GIN 28 and GIN 31) and glioma contrast-enhanced core (GCE 28 and GCE 31), which echo similar characteristics to different regions of GBM tumours<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 29" title="Smith, S. J. et al. Metabolism-based isolation of invasive glioblastoma cells with specific gene signatures and tumorigenic potential. Neuro-Oncol. Adv. 2, vdaa087 (2020)." href="https://bytes.zone/articles/s41565-023-01496-y#ref-CR29" id="ref-link-section-d444612993e1288">29</a></sup>, and a commercial GBM cancer cell line, U251. Human-derived cortical astrocytes were also included as a control for non-tumorigenic cells. A three-dimensional analysis of <i>z</i>-stack confocal microscopy images (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://bytes.zone/articles/s41565-023-01496-y#MOESM1">6</a>) confirmed that the bio-nanoantennae were internalized by all types of cells after 8 hours of incubation, and the number per cell was quantified using inductively coupled plasma mass spectrometry (ICP-MS; Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bytes.zone/articles/s41565-023-01496-y#Fig2">2e</a>). PrestoBlue assay data revealed that the bio-nanoantennae are biocompatible up to a tested range of 100 µg mL<sup>–1</sup> (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://bytes.zone/articles/s41565-023-01496-y#MOESM1">7</a>).</p></div></div></section><section data-title="EF responsive bio-nanoantennae induce apoptosis in GBM"><div id="Sec3-section"><h2 id="Sec3">EF responsive bio-nanoantennae induce apoptosis in GBM</h2><div id="Sec3-content"><p>To examine the electrical input that can be sensed by intracellular bio-nanoantennae for inducing wireless electrochemistry, we optimized the voltage and frequency (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://bytes.zone/articles/s41565-023-01496-y#MOESM1">8</a>) by assessing the change in metabolic activity of GIN 31 cells as the preliminary read-out of Cyt <i>c</i> redox switching. The maximum effect on the metabolic activity was observed at 3 MHz, at an applied potential of 1 V cm<sup>–1</sup> or 0.65 V cm<sup>–1</sup> (no significant difference between 0.65 V cm<sup>–1</sup> and 1 V cm<sup>–1</sup>; <i>P</i> value = 0.23), with significant differences between the tested controls (GNP100, GNP100@r.Cyt <i>c</i> and GNP100@Z) and the bifunctionalized bio-nanoantennae (GNP100@r.Cyt <i>c</i>@Z). Importantly, the observed decrease in metabolic activity of GNP100@r.Cyt <i>c</i>@Z-treated cells was significantly higher than that following 24 h application of in vitro tumour-treating fields that have been approved by the US Food and Drug Administration<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 30" title="Jain, A. et al. Electric field responsive nanotransducers for glioblastoma. Bioelectron. Med. 8, 17 (2022)." href="https://bytes.zone/articles/s41565-023-01496-y#ref-CR30" id="ref-link-section-d444612993e1342">30</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 31" title="Kim, E. H., Song, H. S., Yoo, S. H. &amp; Yoon, M. Tumor treating fields inhibit glioblastoma cell migration, invasion and angiogenesis. Oncotarget 7, 65125 (2016)." href="https://bytes.zone/articles/s41565-023-01496-y#ref-CR31" id="ref-link-section-d444612993e1345">31</a></sup>. We ascribe this decrease in metabolic activity to the electrical–molecular communication via redox switching of r.Cyt <i>c</i> to o.Cyt <i>c</i>, thus inducing cell stress. To eliminate any potential effect of the applied 1 V on water electrolysis (1.23 V versus normal hydrogen electrode), we chose 0.65 V cm<sup>–1</sup> for further studies. The response of the cells to the treatment with bio-nanoantennae (GNP100@r.Cyt <i>c</i>@Z) was comparable in different patient-derived GIN and GCE cells, as well as in the U251 cell line (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bytes.zone/articles/s41565-023-01496-y#Fig3">3a,b</a> and Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://bytes.zone/articles/s41565-023-01496-y#MOESM1">9</a>), with a reported, ~50% decrease in metabolic activity achieved after 12 h compared to ~20% decrease after 2 h of a.c. EF stimulation (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://bytes.zone/articles/s41565-023-01496-y#MOESM1">10</a>). This decrease in metabolic activity was significantly higher compared to all other experimental controls (<i>P</i> values obtained from the statistical analysis are listed in Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://bytes.zone/articles/s41565-023-01496-y#MOESM1">6</a>). By contrast, a weaker effect (~20% decrease in metabolic activity) was observed in cortical astrocytes (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bytes.zone/articles/s41565-023-01496-y#Fig3">3c</a>; <i>P</i> value = 0.011), which was found to be significantly different from the control (no treatment with either bio-nanoantennae or a.c. EFs). A brief discussion on the response of other normal cells (liver and cerebellar astrocyte; Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://bytes.zone/articles/s41565-023-01496-y#MOESM1">11a,b</a>) to the treatment can be found in Supplementary Note <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://bytes.zone/articles/s41565-023-01496-y#MOESM1">2</a>. Thus, based on the obtained data, we conclude that the change in metabolic activity depends on the duration of treatment and cell type.</p><div data-test="figure" data-container-section="figure" id="figure-3" data-title="Wireless electrical–molecular communication mediated by a.c.-EF-responsive bio-nanoantennae induces caspase-3/7-mediated apoptosis in preclinical GBM cells."><figure><figcaption><b id="Fig3" data-test="figure-caption-text">Fig. 3: Wireless electrical–molecular communication mediated by a.c.-EF-responsive bio-nanoantennae induces caspase-3/7-mediated apoptosis in preclinical GBM cells.</b></figcaption><div><div><a data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="https://bytes.zone/articles/s41565-023-01496-y/figures/3" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41565-023-01496-y/MediaObjects/41565_2023_1496_Fig3_HTML.png?as=webp"/><img aria-describedby="Fig3" src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41565-023-01496-y/MediaObjects/41565_2023_1496_Fig3_HTML.png" alt="figure 3" loading="lazy" width="685" height="636"/></picture></a></div><div data-test="bottom-caption" id="figure-3-desc"><p><b>a</b>–<b>c</b>, Metabolic activity of GIN/GCE 31, GIN/GCE 28 and cortical astrocytes was analyzed using a PrestoBlue HS assay. GIN cells, GCE cells and human cortical astrocytes were treated with GNP100@r.Cyt <i>c</i>@Z for 8 h followed by a.c. EF stimulation (3 MHz, 0.65 V cm<sup>–1</sup>) for 12 h. The control is no treatment with either bio-nanoantennae or a.c. EFs. Error bars represent mean ± standard error of mean (s.e.m.) obtained from triplicate experiments repeated three times. Statistical analysis was performed by applying a two-way ANOVA with a Tukey’s post-test. <b>d</b>–<b>g</b>, Representative samples from flow cytometric analysis of cells stained with CellEvent Caspase-3/7 Green to detect caspase 3/7 apoptotic activity and Zombie NIR fixable dye to detect the dead cell population. The quadrants in the figure represents the following: A-D-, no cell death either due to caspase 3/7 apoptosis or necrosis; A+D-, caspase 3/7 positive apoptotic cells; A+D+, non-viable/dead cells caused by apoptosis; and A−D+, necrotic cells. The gating strategy is shown in Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://bytes.zone/articles/s41565-023-01496-y#MOESM1">14</a>. FITC, fluorescein isothiocyanate and NIR, near infra-red are filters. <b>h</b>, High-magnification confocal microscopy images to demonstrate caspase 3/7 activation immediately after the treatment with a.c. EFs (3 MHz, 0.65 V cm<sup>–1</sup>) for 12 h in the presence of bio-nanoantennae. Cells were fixed with paraformaldehyde followed by counterstaining with a caspase 3/7 detection kit (green), Cytopainter actin phalloidin (Texas Red 591, in red) and Hoechst nuclear stain (blue). Scale bars, 20 µm. To confirm caspase 3/7 activation, at least five confocal images (each with nearly 30 cells) were taken at ×10. High-resolution (×63) images show both morphology and caspase 3/7 activation (in green). The –EF and +EF indicate before and after the EF treatment. <b>i</b>, Confocal microscopy images to demonstrate the cytoplasmic localization of GNP100@r.Cyt <i>c</i>@Z immediately after the treatment with a.c. EFs (3 MHz, 0.65 V cm<sup>–1</sup>) for 12 h in the presence of bio-nanoantennae. Cells were stained with late endosome dye (green) and imaged using a Leica confocal microscope with green fluorescent protein, GFP (late endosomes) and Alexa 633 (GNP100@r.Cyt <i>c</i>@Z) filter settings. Scale bars, 100 µm. White arrows represent the endosomal escape and localization of bio-nanoantennae, at least 60 cells were analysed.</p><p><a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://bytes.zone/articles/s41565-023-01496-y#MOESM4">Source data</a></p></div></div></figure></div><p>The alteration in metabolic activity in cells treated with GNP100@r.Cyt <i>c</i>@Z plus a.c. EFs (3 MHz at 0.65 V cm<sup>–1</sup> for 12 hours) is correlated with increased cell death, as clearly evidenced by the results of a live–dead assay (Supplementary Figs. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://bytes.zone/articles/s41565-023-01496-y#MOESM1">12</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://bytes.zone/articles/s41565-023-01496-y#MOESM1">13</a>). The mechanism of cell death was probed by flow cytometry (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bytes.zone/articles/s41565-023-01496-y#Fig3">3d–g</a> and Supplementary Figs. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://bytes.zone/articles/s41565-023-01496-y#MOESM1">14</a>–<a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://bytes.zone/articles/s41565-023-01496-y#MOESM1">16</a>) and confocal microscopy (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bytes.zone/articles/s41565-023-01496-y#Fig3">3h</a> and Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://bytes.zone/articles/s41565-023-01496-y#MOESM1">17</a>), where the induced caspase 3/7 activity was observed in GIN/GCE cells treated with GNP100@r.Cyt <i>c</i>@Z followed by electrical stimulation (ES), indicative of apoptosis. This could be attributed to the increased affinity of activated Cyt <i>c</i> (Fe<sup>3+</sup>) on bio-nanoantennae with APAF-1 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 12" title="Brown, G. C. &amp; Borutaite, V. Regulation of apoptosis by the redox state of cytochrome c. Biochim. Biophys. Acta Bioenerg. 1777, 877–881 (2008)." href="https://bytes.zone/articles/s41565-023-01496-y#ref-CR12" id="ref-link-section-d444612993e1490">12</a></sup>). Confocal microscopy revealed that bio-nanoantennae, upon ES, escape the endo/lysosomal degradation to localize in the cytosol (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bytes.zone/articles/s41565-023-01496-y#Fig3">3i</a> and Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://bytes.zone/articles/s41565-023-01496-y#MOESM1">18</a>). Further studies are required to investigate the precise mechanism. We note that no reactive oxygen species and temperature changes were observed (Supplementary Figs. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://bytes.zone/articles/s41565-023-01496-y#MOESM1">19</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://bytes.zone/articles/s41565-023-01496-y#MOESM1">20</a>). Therefore, we successfully conducted the ES of GBM cells and demonstrated the wireless electrochemistry-mediated redox switching and activation of Cyt <i>c</i> on the surface of GNP100@r.Cyt <i>c</i>@Z, leading to the apoptosis of GBM cells.</p><p>To further understand this electrical–molecular signalling, we performed transcriptomic analysis on a sample of GBM cells and healthy cortical astrocytes (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bytes.zone/articles/s41565-023-01496-y#Fig4">4</a> and Supplementary Figs. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://bytes.zone/articles/s41565-023-01496-y#MOESM1">21</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://bytes.zone/articles/s41565-023-01496-y#MOESM1">22</a><b>)</b> to explore the effect of the bioelectronic communication tool on gene expression and regulation. Hierarchical clustering analysis showed a differential expression of genes related to apoptosis, cancer proliferation and angiogenesis, and tumour suppression (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bytes.zone/articles/s41565-023-01496-y#Fig4">4</a><b>)</b>. This change in differentially expressed genes was highest for GIN/GCE 31 cells treated with GNP100@r.Cyt <i>c</i>@Z and followed by ES compared to untreated cells and other experimental controls. Interestingly, we observed minimal changes in the gene regulation for astrocytes, which suggests that the astrocytic transcriptomic landscape remains largely unperturbed upon the treatment. By contrast, an important deviation in the transcriptome has been monitored for GBM cell lines, implying that the treatment modulates signalling pathways that are specific only to GBM. To explain this effect, we conducted gene-set enrichment analysis of gene ontology biological processes (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bytes.zone/articles/s41565-023-01496-y#Fig7">1</a> and Supplementary Figs. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://bytes.zone/articles/s41565-023-01496-y#MOESM1">23</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://bytes.zone/articles/s41565-023-01496-y#MOESM1">24</a>). A detailed discussion on transcriptomics data and gene ontology biological process analysis is in Supplementary Note <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://bytes.zone/articles/s41565-023-01496-y#MOESM1">3</a>. Collectively, the data obtained from Figs. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bytes.zone/articles/s41565-023-01496-y#Fig3">3</a>–<a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bytes.zone/articles/s41565-023-01496-y#Fig4">4</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bytes.zone/articles/s41565-023-01496-y#Fig7">1</a> imply that GNP100@r.Cyt <i>c</i>@Z treatment followed by ES leads to reduced proliferation and apoptosis induction in patient-derived GBM cells. Overall, the in vitro results indicate successful communication with biology at a molecular scale using electricity. Importantly, this approach enables selective actuation of cancer cell behaviour compared to normal healthy cells (astrocytes and liver bile duct cells).</p><div data-test="figure" data-container-section="figure" id="figure-4" data-title="Transcriptomic analysis to connect bio-nanoantennae-mediated wireless electrical -molecular communication with gene expression and regulation."><figure><figcaption><b id="Fig4" data-test="figure-caption-text">Fig. 4: Transcriptomic analysis to connect bio-nanoantennae-mediated wireless electrical -molecular communication with gene expression and regulation.</b></figcaption><div><div><a data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="https://bytes.zone/articles/s41565-023-01496-y/figures/4" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41565-023-01496-y/MediaObjects/41565_2023_1496_Fig4_HTML.png?as=webp"/><img aria-describedby="Fig4" src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41565-023-01496-y/MediaObjects/41565_2023_1496_Fig4_HTML.png" alt="figure 4" loading="lazy" width="685" height="1144"/></picture></a></div><div data-test="bottom-caption" id="figure-4-desc"><p>Heat map demonstrating hierarchical clustering of top 35 genes that were regulated after the treatment with GNP100@r.Cyt <i>c</i>@Z for 8 h followed by a.c. EF stimulation (3 MHz, 0.65 V cm<sup>–1</sup>) for 2 h. A variance-stabilized transformation was performed on the raw count matrix, and 35 genes with the highest variance across samples were selected for hierarchical clustering. Each row represents one gene, and each column represents one sample. The colour represents the difference of the count value to the row mean. GIN 31 and GCE 31, which showed the maximum response to the treatment with GNP100@r.Cyt <i>c</i>@Z and a.c. EFs, were chosen. As a control to cancer cells, healthy cortical astrocytes were used. Immediately after the treatment, cells were washed and centrifuged to obtain a pellet, which was snap-frozen in liquid nitrogen and shipped (in dry ice) to Qiagen in Germany for RNA sequencing. The treatment codes are as follow: EF–, control (no treatment with either bio-nanoantennae or a.c. EFs); EF+, cells treated with a.c. EFs; NP1 EF+, cells treated with GNP100@r.Cyt <i>c</i>; and NP2 EF+, cells treated with GNP100@r.Cyt <i>c</i>@Z; treatment was for 8 h followed by 2 h with the a.c. EF.</p><p><a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://bytes.zone/articles/s41565-023-01496-y#MOESM5">Source data</a></p></div></div></figure></div></div></div></section><section data-title="Nanoscale wireless electrochemistry and in vitro QBET"><div id="Sec4-section"><h2 id="Sec4">Nanoscale wireless electrochemistry and in vitro QBET</h2><div id="Sec4-content"><p>The electrical–molecular communication via wireless electrochemistry induced at bio-nanoantennae was probed by circular dichroism analysis (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bytes.zone/articles/s41565-023-01496-y#Fig5">5a</a> and Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://bytes.zone/articles/s41565-023-01496-y#MOESM1">25</a>) and UV–visible absorption spectroscopy (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bytes.zone/articles/s41565-023-01496-y#Fig5">5b,c</a> and Supplementary Figs. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://bytes.zone/articles/s41565-023-01496-y#MOESM1">26</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://bytes.zone/articles/s41565-023-01496-y#MOESM1">27</a>). There was no significant change in the hydrodynamic diameter <i>h</i><sub>d</sub> and zeta potential <i>ζ</i> after ES with an a.c. EF (Supplementary Figs. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://bytes.zone/articles/s41565-023-01496-y#MOESM1">28</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://bytes.zone/articles/s41565-023-01496-y#MOESM1">29</a>). A detailed discussion on the data demonstrating that the bio-nanoantennae act as bipolar electrodes and can modulate the redox state of the Cyt <i>c</i> under application of the EF is in Supplementary Note <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://bytes.zone/articles/s41565-023-01496-y#MOESM1">4</a>.</p><div data-test="figure" data-container-section="figure" id="figure-5" data-title="Nanoscale wireless electrochemistry and in vitro electron tunnelling via bio-nanoantennae for inducing cell death in GBM cells."><figure><figcaption><b id="Fig5" data-test="figure-caption-text">Fig. 5: Nanoscale wireless electrochemistry and in vitro electron tunnelling via bio-nanoantennae for inducing cell death in GBM cells.</b></figcaption><div><div><a data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="https://bytes.zone/articles/s41565-023-01496-y/figures/5" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41565-023-01496-y/MediaObjects/41565_2023_1496_Fig5_HTML.png?as=webp"/><img aria-describedby="Fig5" src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41565-023-01496-y/MediaObjects/41565_2023_1496_Fig5_HTML.png" alt="figure 5" loading="lazy" width="685" height="497"/></picture></a></div><div data-test="bottom-caption" id="figure-5-desc"><p><b>a</b>, The effect of a.c. EFs on the conformation of redox moieties<b>:</b> circular dichroism spectra in the Soret region to emphasize the redox-mediated change in haem moieties of free native o.Cyt <i>c</i>, r.Cyt <i>c</i> and bifunctionalized GNP bio-nanoantennae in 10 mM PBS (pH 7.4).θ, molar ellipticity. All samples with identical concentrations (25 µg mL<sup>–1</sup>) were used for spectrum acquisition. Three spectra of each sample were collected and averaged. Insulated, coated steel electrodes were used as a control to demonstrate the occurrence of wireless electrochemistry at the nanoscale. <b>b</b>,<b>c</b>, UV–visible spectra of bio-nanoantennae after stimulation with a.c. EFs of 3 MHz, 0.65 V cm<sup>–1</sup> indicating a blueshift in the absorption maxima in GNP100@r.Cyt <i>c</i>@Z, indicating the redox switching of Cyt <i>c</i> (from r.Cyt <i>c</i> to o.Cyt <i>c</i>). <b>d</b>–<b>f</b>, Metabolic activity of GIN 31 cells as a function of different-sized bio-nanoantennae (20 nm, 50 nm and 100 nm; GNP20@r.Cyt <i>c</i>@Z, GNP50@r.Cyt <i>c</i>@Z and GNP100@r.Cyt <i>c</i>@Z, respectively) synthesized using various linker lengths (1, 2, 3.5 and 5 kDa). GIN 31 cells were treated with bifunctionalized bio-nanoantennae for 8 h followed by a.c. EF stimulation (3 MHz, 0.65 V cm<sup>–1</sup>) for 12 h. Error bars represent ±s.e.m. of the mean obtained from triplicate experiments repeated three times. Statistical analysis was performed by applying a two-way ANOVA with a Tukey’s post-test. Exact <i>P</i> values are in Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://bytes.zone/articles/s41565-023-01496-y#MOESM1">7</a>.</p><p><a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://bytes.zone/articles/s41565-023-01496-y#MOESM6">Source data</a></p></div></div></figure></div><p>To facilitate the translation of the technology more broadly, understanding of the electrically induced mode of electron transfer is needed. The frequency we used for electrical communication was 3 MHz; considering the electron transfer rate constant of 3.75 × 10<sup>−3</sup> cm s<sup>–1</sup> calculated from Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bytes.zone/articles/s41565-023-01496-y#Fig2">2d</a>, the maximum distance the electron could travel at this frequency is 0.01 nm. Thus, theoretically at this given frequency, the redox event should not occur. We propose that the observed activity is enabled by QBET induced from the Cyt <i>c</i>, as was indicated previously<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 8" title="Xin, H. et al. Quantum biological tunnel junction for electron transfer imaging in live cells. Nat. Commun. 10, 3245 (2019)." href="https://bytes.zone/articles/s41565-023-01496-y#ref-CR8" id="ref-link-section-d444612993e1739">8</a></sup>. However, this process has not been controlled using ES.</p><p>To gather experimental evidence for QBET in our system, we consider molecular tunnel junctions, where quantum electron tunnelling can occur<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 32" title="Zhao, J. et al. ‘Size-independent’ single-electron tunneling. J. Phys. Chem. Lett. 6, 4986–4990 (2015)." href="https://bytes.zone/articles/s41565-023-01496-y#ref-CR32" id="ref-link-section-d444612993e1746">32</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 33" title="Gupta, C. et al. Quantum tunneling currents in a nanoengineered electrochemical system. J. Phys. Chem. C 121, 15085–15105 (2017)." href="https://bytes.zone/articles/s41565-023-01496-y#ref-CR33" id="ref-link-section-d444612993e1749">33</a></sup>. We altered the tunnel junction energy by using GNPs with different sizes (20, 50 and 100 nm) and a PEG linker with different lengths (1, 2, 3.5 and 5 kDa; Supplementary Figs. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://bytes.zone/articles/s41565-023-01496-y#MOESM1">30</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://bytes.zone/articles/s41565-023-01496-y#MOESM1">31</a>). Toxicity studies of different-sized bio-nanoantennae synthesized using PEG linkers of various lengths without ES at 25 µg mL<sup>–1</sup> showed no significant changes in the metabolic activity compared to an untreated control (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://bytes.zone/articles/s41565-023-01496-y#MOESM1">32</a><b>)</b>. The metabolic assays (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bytes.zone/articles/s41565-023-01496-y#Fig5">5d–f</a>, Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://bytes.zone/articles/s41565-023-01496-y#MOESM1">7</a> and Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://bytes.zone/articles/s41565-023-01496-y#MOESM1">33</a>) and viability studies (Supplementary Figs. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://bytes.zone/articles/s41565-023-01496-y#MOESM1">34</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://bytes.zone/articles/s41565-023-01496-y#MOESM1">35</a>) on GIN 31 and GCE 31 cells with ES (3 MHz and 0.65 V cm<sup>–1</sup>) indicated a resonant biological effect with bio-nanoantennae using the 1 and 2 kDa PEG linkers with 50 nm and 100 nm GNPs. Importantly, this effect is seen only for cells treated with bio-nanoantennae (GNP@r.Cyt <i>c</i>@Z) following 12 h of ES. ICP-MS analysis revealed no significant difference in the number of Z or r.Cyt <i>c</i> per cell when treated with GNP20@r.Cyt <i>c</i>@Z, GNP50@r.Cyt <i>c</i>@Z or GNP100@r.Cyt <i>c</i>@Z (Supplementary Figs. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://bytes.zone/articles/s41565-023-01496-y#MOESM1">36</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://bytes.zone/articles/s41565-023-01496-y#MOESM1">37</a><b>)</b>. We propose that a QBET is occurring in our system, as evidenced by the resonance at specific conditions, and is indicative of wave-like behaviour<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 8" title="Xin, H. et al. Quantum biological tunnel junction for electron transfer imaging in live cells. Nat. Commun. 10, 3245 (2019)." href="https://bytes.zone/articles/s41565-023-01496-y#ref-CR8" id="ref-link-section-d444612993e1809">8</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 34" title="Kong, F.-F. et al. Wavelike electronic energy transfer in donor–acceptor molecular systems through quantum coherence. Nat. Nanotechnol. 17, 729–736 (2022)." href="https://bytes.zone/articles/s41565-023-01496-y#ref-CR34" id="ref-link-section-d444612993e1812">34</a></sup>. Thus, we suggest that the electron transfer in this system is an example of electrically stimulated QBET in biology and consequently an example of facilitating electrical–molecular communication.</p><p>To provide further evidence of the mechanism of electron transfer, we established a mathematical model. The cell metabolic rate (shown in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bytes.zone/articles/s41565-023-01496-y#Fig5">5d–f</a>) was correlated to the rate of charge transfer from Cyt <i>c</i>, derived as Equation (<a data-track="click" data-track-label="link" data-track-action="equation anchor" href="https://bytes.zone/articles/s41565-023-01496-y#Equ1">1</a>) (full details of the derivation are presented in the <a data-track="click" data-track-label="link" data-track-action="section anchor" href="https://bytes.zone/articles/s41565-023-01496-y#Sec6">Methods</a>). These values were then plotted as a function of the barrier junction, altered using the PEG ligands with different length and different-sized nanoparticles (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bytes.zone/articles/s41565-023-01496-y#Fig6">6a,b</a>).</p><div id="Equ1"><p><span>$${r}_{{\mathrm{d}}}=-\frac{{\rm{ln}}\left(M\left(t\right)\right)}{t}$$</span></p><p>
                    (1)
                </p></div><p>where <i>r</i><sub>d</sub> is the rate of donor charging and <i>M</i>(<i>t</i>) is the metabolic activity at time <i>t</i>.</p><div data-test="figure" data-container-section="figure" id="figure-6" data-title="Mathematical modelling and plasmon resonance energy transfer analysis for probing quantum tunnelling in bio-nanoantennae system."><figure><figcaption><b id="Fig6" data-test="figure-caption-text">Fig. 6: Mathematical modelling and plasmon resonance energy transfer analysis for probing quantum tunnelling in bio-nanoantennae system.</b></figcaption><div><div><a data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="https://bytes.zone/articles/s41565-023-01496-y/figures/6" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41565-023-01496-y/MediaObjects/41565_2023_1496_Fig6_HTML.png?as=webp"/><img aria-describedby="Fig6" src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41565-023-01496-y/MediaObjects/41565_2023_1496_Fig6_HTML.png" alt="figure 6" loading="lazy" width="685" height="670"/></picture></a></div><div data-test="bottom-caption" id="figure-6-desc"><p><b>a</b>,<b>b</b>, Mathematic modelling to determine the rate of donor charging, <i>r</i><sub>d</sub>, calculated using the metabolic activity, using equation (<a data-track="click" data-track-label="link" data-track-action="equation anchor" href="https://bytes.zone/articles/s41565-023-01496-y#Equ1">1</a>), compared to the PEG linker length, <i>L</i>, for GNP20@r.Cyt <i>c</i>@Z (<b>a</b>) and GNP100@r.Cyt <i>c</i>@Z (<b>b</b>) in GIN 31 cell samples. The black lines show the exponential behaviour expected for quantum tunnelling with an inverse localization radius <i>α</i> and constant of proportionality <i>β</i>, shown in the figure legends. Error bars represent ±s.d. of the mean. The <i>x</i>-axis error bar represents the s.d. of the mean PEG linker length, and the <i>y</i>-axis error bar represents the s.d. of the mean <i>r</i><sub>d</sub> (rate of Cyt <i>c</i> charging), which was obtained from equations (<a data-track="click" data-track-label="link" data-track-action="equation anchor" href="https://bytes.zone/articles/s41565-023-01496-y#Equ5">5</a>)–(<a data-track="click" data-track-label="link" data-track-action="equation anchor" href="https://bytes.zone/articles/s41565-023-01496-y#Equ7">7</a>) in the <a data-track="click" data-track-label="link" data-track-action="section anchor" href="https://bytes.zone/articles/s41565-023-01496-y#Sec6">Methods</a>. R<sup>2</sup>, coefficient of determination. <b>c</b>, Scattering spectra (<i>I</i>, intensity) and spectra difference (<i>Δ</i>) for QBET obtained for GNP100@r.Cyt <i>c</i>@Z bio-nanoantennae. The quantized peaks were obtained from the difference of scattering spectra between the samples functionalized with r.Cyt <i>c</i> and Z using a 2 kDa linker and GNP100. Solid curves are captured scattering spectra (linked to left axis) of GNP100@r.Cyt <i>c</i>@Z, and dashed curves are quantized peaks, that is, the corresponding spectra difference (linked to right axis). Blue arrows indicate a peak shift. LSPR, localized surface plasmon resonance. <b>d</b>, Quantized peaks of GNP100@r.Cyt <i>c</i>@Z within a 530–550 nm region (zoomed-in from <b>c</b>) confirming the presence of o.Cyt <i>c</i> in samples exposed to a.c. EFs.</p><p><a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://bytes.zone/articles/s41565-023-01496-y#MOESM7">Source data</a></p></div></div></figure></div><p>The results fit well with the exponential dependences expected for the tunnelling of an electron through a barrier<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 35" title="Greensite, J. An Introduction to Quantum Theory (IOP Publishing, 2017)." href="https://bytes.zone/articles/s41565-023-01496-y#ref-CR35" id="ref-link-section-d444612993e2037">35</a></sup>, supporting the hypothesis that the resonant biological effects observed with the electrical input result in QBET. Further primary experimental evidence for the quantum tunnelling is provided by plasmon resonance scattering probing the plasmon resonance energy transfer<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 36" title="Choi, Y., Kang, T. &amp; Lee, L. P. Plasmon resonance energy transfer (PRET)-based molecular imaging of cytochrome c in living cells. Nano Lett. 9, 85–90 (2009)." href="https://bytes.zone/articles/s41565-023-01496-y#ref-CR36" id="ref-link-section-d444612993e2041">36</a></sup>. Without ES, quantized dips from scattering spectra were obtained by subtracting the spectra for the unmodified particle and vice versa. For GNP100@r.Cyt <i>c</i>@Z samples (no ES), the scattering peaks were observed at ~465 nm (ITO substrate); a broad peak due to r.Cyt <i>c</i> and Z, at around 568 nm; and a localized surface plasmon resonance peak (LSPR), shifted to 711 nm (LSPR of GNP100 = 612 nm; Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bytes.zone/articles/s41565-023-01496-y#Fig6">6c</a> and Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://bytes.zone/articles/s41565-023-01496-y#MOESM1">38a</a>). This equates to the frequencies with the absorption peak of r.Cyt <i>c</i> or Z and subsequently induces the electronic excitation of the r.Cyt <i>c</i> at the specific wavelength<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 8" title="Xin, H. et al. Quantum biological tunnel junction for electron transfer imaging in live cells. Nat. Commun. 10, 3245 (2019)." href="https://bytes.zone/articles/s41565-023-01496-y#ref-CR8" id="ref-link-section-d444612993e2064">8</a></sup>. Importantly, the QBETs captured are cumulative. Under ES, we see a quantized dip at 536 nm due to the switching of r.Cyt <i>c</i> to o.Cyt <i>c</i> and thus representing QBET (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bytes.zone/articles/s41565-023-01496-y#Fig6">6d</a>). We also see a 20 mV shift in the surface plasmon resonance peak (heat maps in Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bytes.zone/articles/s41565-023-01496-y#Fig8">2a</a> (no EF) and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bytes.zone/articles/s41565-023-01496-y#Fig8">2b</a> (EF applied)) indicative of charge transfer, which we associate with the redox behavor of an electron relay donor complex<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 37" title="Jing, C. et al. New insights into electrocatalysis based on plasmon resonance for the real-time monitoring of catalytic events on single gold nanorods. Anal. Chem. 86, 5513–5518 (2014)." href="https://bytes.zone/articles/s41565-023-01496-y#ref-CR37" id="ref-link-section-d444612993e2084">37</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 38" title="Zhou, H. et al. Optical monitoring of faradaic reaction using single plasmon-resonant nanorods functionalized with graphene. Chem. Commun. 51, 3223–3226 (2015)." href="https://bytes.zone/articles/s41565-023-01496-y#ref-CR38" id="ref-link-section-d444612993e2087">38</a></sup>. The appearance of o.Cyt <i>c</i> was not observed in all other control samples with ES (3 MHz, 0.65 V; Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://bytes.zone/articles/s41565-023-01496-y#MOESM1">38b–i</a>). The combined data indicate that resonant frequency effects resulting in cell death, model exponential decay based on barrier junction energetics and quantized dips from plasmon resonance energy transfer all represent electron tunnelling. Linker length, applied frequency and voltage were found to be critical for QBET-mediated electrical–molecular communication and inducing cancer cell apoptosis. A detailed discussion on the role of these parameters is in Supplementary Note <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://bytes.zone/articles/s41565-023-01496-y#MOESM1">5</a>. Collectively and to the best of our knowledge, this is the first successful demonstration of electrical–molecular quantum signalling technology in biology, which could be used for the induced apoptosis of cancer cells as a biomedical exemplar.</p></div></div></section><section data-title="Conclusions"><div id="Sec5-section"><h2 id="Sec5">Conclusions</h2><p>Inspired by wireless electrochemistry, we tuned the redox state of molecules functionalized on the surface of nanoantennae by applying a.c. EFs. This approach is shown to selectively facilitate electrical–molecular communication to induce apoptosis in patient-derived GBM cells by switching the redox state of Cyt <i>c</i> at a resonant electric field. Based on the obtained data, we infer that the electron transfer in the bio-nanoantennae occurs through EF-induced quantum tunnelling and is thus QBET, with the applied frequency, potential and linker length playing a critical role. Moreover, transcriptomics shows that electrical–molecular communication is specifically targeted in cancer cells. This represents a wireless electrical–molecular communication tool that facilitates the killing of cancer cells.</p></div></section><section data-title="Methods"><div id="Sec6-section"><h2 id="Sec6">Methods</h2><div id="Sec6-content"><h3 id="Sec7">Reagents</h3><p>The following reagents were used: horse heart Cyt <i>c</i> (C7752, Sigma-Aldrich), EDC (E6383, Sigma-Aldrich), NHS (130672, Sigma-Aldrich), 2-(<i>N</i>-morpholino)ethane sulfonic acid hydrate, 4-morpholineethanesulfonic acid (M8250, Sigma-Aldrich), <span>l</span>-ascorbic acid (A92902, Sigma-Aldrich), μ-plate 24-well black ibiTreat surface (IB-82426, Thistle Scientific), Hoechst 33342 (NucBlue Live ReadyProbes Reagent, R37605, Thermo Fisher Scientific), actin using Phalloidin–iFluor 488 conjugate (23115, AAT Bioquest, Stratech), PrestoBlue HS cell viability reagent (P50200, Invitrogen), calcein AM (C1430, Thermo Fisher Scientific), propidium iodide (P4170, Sigma-Aldrich), H2DCFDA (D399, Invitrogen), CellEvent Caspase-3/7 Green Flow Cytometry Assay Kit (C10427, Invitrogen), Zombie NIR Fixable Viability Kit (423105, BioLegend), CellEvent Caspase-3/7 Green ReadyProbes Reagent (R37111, Invitrogen), CellLight Late Endosomes-GFP, BacMam 2.0 (C10588, Thermo Fisher Scientific) and LysoTracker Green DND-26 (L7526, Thermo Fisher Scientific). A catalogue number is not available for the carboxylic-PEG-coated GNPs or Z as they were customized and purchased from Nanopartz and Porphychem, respectively.</p><h3 id="Sec8">Synthesis of bifunctionalized gold bipolar bio-nanoantennae</h3><p>Spherical GNPs with a diameter (<i>d</i>) of 100 nm and capped with thiol-carboxylic-PEG (SH-PEG-COOH; molecular weight <i>M</i> = 1 kDa, 2 kDa, 3.5 kDa and 5 kDa) with a PEG density of 1–1.5 nm<sup>2</sup> were purchased from Nanopartz. The r.Cyt <i>c</i> was obtained by adding 10 mg of the oxidized form of the horse heart Cyt <i>c</i> (o.Cyt <i>c</i>) into 5 mL <span>l</span>-ascorbic acid solution (1 mg mL<sup>–1</sup> in PBS) and purifying by dialysis at 4 °C in dark conditions for 36 h to remove excess ascorbic acid. The r.Cyt <i>c</i> and Z were covalently conjugated to a carboxylic group on the capping ligands of GNPs using EDC/NHS carbodiimide coupling chemistry. Briefly, 20 μL GNP solution (3.6 mg mL<sup>–1</sup> in ultrapure water) was mixed with 20 μL EDC/NHS solution in 2-(<i>N</i>-morpholino)ethane sulfonic acid buffer (10 mM, pH 5.5) at a concentration of 30 and 36 mg mL<sup>–1</sup>. The solution was mixed for 1 h at room temperature; then 1 mL of washing buffer (×1 PBS with 0.01% (w/v) Tween 20) was added, and the solution was centrifuged at 450<i>g</i> for 20 min. The supernatant was discarded and 20 μL r.Cyt <i>c</i> (1 mg mL<sup>–1</sup>) and Z (0.5 mg mL<sup>–1</sup>) were added to the pellet and sonicated using a Fisherbrand ultrasonic bath (FB11201, 80 KHz, 50% power, 1 min). Next, the solution was incubated for 4 h at room temperature under mixing; then 1 mL washing buffer was added, and the solution was centrifuged at 450<i>g</i> for 20 min. To ensure the complete removal of unbound r.Cyt <i>c</i> and Z, the washing step was repeated twice. The obtained pellet consist of GNPs conjugated with r.Cyt <i>c</i> and Z (GNP100@r.Cyt <i>c</i>@Z) was dispersed in PBS and stored at 4 °C until further use.</p><p>To prepare the control samples of GNPs covalently conjugated with only one molecule, either r.Cyt <i>c</i> (GNP100@r.Cyt <i>c</i>) or Z (GNP100@Z), only one of these compounds was introduced during the EDC/NHS step with concentrations of 0.25 mg mL<sup>–1</sup> or 0.1 mg mL<sup>–1</sup>, respectively. The same protocol was used to synthesize bio-nanoantennae with different GNP diameters (20 nm, 50 nm and 100 nm) and different PEG lengths (1 kDa, 2 kDa, 3.5 kDa, 5 kDa). The concentration of Cyt <i>c</i> and Z during the EDC/NHS step was optimized (0.25, 0.5 and 1 mg mL<sup>–1</sup>) to obtain a similar binding ratio of Cyt <i>c</i> and Z per GNP.</p><h3 id="Sec9">Characterization techniques</h3><h4 id="Sec10">Dynamic light scattering and zeta potential measurements</h4><p>The hydrodynamic diameter (<i>h</i><sub>d</sub>) and zeta potential (<i>ζ</i>) of bio-nanoantennae (in ultrapure water) was measured using a Malvern Zetasizer Nano-ZS ((Malvern Instruments).</p><h4 id="Sec11">Transmission electron microscopy</h4><p>A TEM instrument (JEOL 2000 FX TEM) operating at 200 kV accelerating voltage was used to record TEM images. The samples were prepared by drop-casting 10 μL of sample onto a carbon-coated copper grid (400 Mesh, Agar Scientific) twice in an interval of 1 h. The sample was dried for at least 30 min before TEM imaging.</p><h4 id="Sec12">UV–visible absorption spectroscopy</h4><p>UV–visible absorption spectra of bio-nanoantennae dispersed in PBS were recorded on a Cary 3500 UV–visible instrument (Agilent Technologies).</p><h4 id="Sec13">Circular dichroism</h4><p>Far- and near-UV circular dichroism spectra were recorded at 20 °C on a Chirascan circular dichroism spectrophotometer (Applied Photophysics) equipped with a temperature control unit TC125 (Quantum Northwest). Samples were dispersed in 10 mM PBS at pH 7.4. At least three spectra were recorded for each sample and averaged. A quartz cuvette with an optical path length of 1 cm was used for the circular dichroism measurements.</p><h4 id="Sec14">Cyclic voltammetry</h4><p>The electrochemical analyses were conducted using a Metrohm Autolab M204 potentiostat equipped with a three-electrode system consisting of a platinum wire counter electrode, an Ag/AgCl reference electrode and an ITO working electrode (Delta Technologies). ITO-coated glass (10 mm × 20 mm) was washed with acetone and water, dried with argon and assembled into an electrochemical cell with an exposed working area of 38.5 mm<sup>2</sup>. Bifunctionalized gold bipolar nanoelectrodes were dispersed in PBS to a final concentration of 25 μg mL<sup>–1</sup> (determined using UV–visible spectroscopy). Cyclic voltammetry was conducted between 1.2 V and −0.2 V with varying scan rates between 50 mV s<sup>−1</sup> and 2 V s<sup>−1</sup>. Repetitive consecutive cyclic voltammetry measurements were conducted at a fixed scan rate of 100 mV s<sup>−1</sup>. Control cyclic voltammetry measurements were conducted with carboxylic-PEG-modified GNPs using PBS as the supporting electrolyte.</p><p>The heterogeneous rate constant (<i>k</i><sup>0</sup>) was calculated using the Nicholson and Shain method<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 46" title="Nicholson, R. S. &amp; Shain, I. Theory of stationary electrode polarography. Single scan and cyclic methods applied to reversible, irreversible, and kinetic systems. Anal. Chem. 36, 706–723 (1964)." href="https://bytes.zone/articles/s41565-023-01496-y#ref-CR46" id="ref-link-section-d444612993e2290">46</a></sup>. The rate transfer coefficient, <i>α</i>, was calculated from the scan rate study (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://bytes.zone/articles/s41565-023-01496-y#MOESM1">5</a>). The slope of the logarithm of the scan rate versus the difference between the peak potential and formal potential of the cell is given by equation (<a data-track="click" data-track-label="link" data-track-action="equation anchor" href="https://bytes.zone/articles/s41565-023-01496-y#Equ2">2</a>):</p><div id="Equ2"><p><span>$${\rm{Slope}}=-\frac{2.3\,{RT}}{\alpha {nF}}$$</span></p><p>
                    (2)
                </p></div><p>where <i>R</i> is the gas constant, <i>T</i> is temperature, <i>n</i> is the number of electrons transferred in the redox reaction and <i>F</i> is the Faraday constant.</p><p>However, the Nicholson and Shain method for determining <i>ψ</i> (<i>ψ</i> is a function of the peak separation) assumes that <i>α</i> = 0.5. But in this work, the value of <i>α</i> is different (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://bytes.zone/articles/s41565-023-01496-y#MOESM1">5</a>); therefore, the Lavagnini method<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 47" title="Lavagnini, I., Antiochia, R. &amp; Magno, F. An extended method for the practical evaluation of the standard rate constant from cyclic voltammetric data. Electroanalysis 16, 505–506 (2004)." href="https://bytes.zone/articles/s41565-023-01496-y#ref-CR47" id="ref-link-section-d444612993e2385">47</a></sup> is used to calculate <i>ψ</i> as the function of the peak separation Δ<i>E</i><sub>p</sub> using equation (<a data-track="click" data-track-label="link" data-track-action="equation anchor" href="https://bytes.zone/articles/s41565-023-01496-y#Equ3">3</a>):</p><div id="Equ3"><p><span>$$\psi =2.18{\left(\frac{\alpha }{\uppi }\right)}^{0.5}{{\exp }}\left[\right.-\left(\frac{{\alpha }^{2}F}{{RT}}\right)n\Delta {E}_{\mathrm{{p}}}.$$</span></p><p>
                    (3)
                </p></div><p>Then <i>k</i><sup>0</sup> for the nanoantennae is calculated using equation (<a data-track="click" data-track-label="link" data-track-action="equation anchor" href="https://bytes.zone/articles/s41565-023-01496-y#Equ4">4</a>):</p><div id="Equ4"><p><span>$$\psi ={k}^{0}{\left[\frac{{{\uppi }}{{D}}{nv}{{F}}}{{{RT}}}\right]}^{-0.5}$$</span></p><p>
                    (4)
                </p></div><p>where <i>D</i> is the diffusion coefficient.</p><h3 id="Sec15">Cell lines</h3><p>GIN cells were isolated from the 5-aminolevulinic acid (5-ALA) fluorescing infiltrative tumour margin, and GCE cells were isolated from the core central region of the tumour, from GBM patients who underwent surgery at the Queen’s Medical Centre, University of Nottingham (Nottingham, UK), using a previously described method<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 29" title="Smith, S. J. et al. Metabolism-based isolation of invasive glioblastoma cells with specific gene signatures and tumorigenic potential. Neuro-Oncol. Adv. 2, vdaa087 (2020)." href="https://bytes.zone/articles/s41565-023-01496-y#ref-CR29" id="ref-link-section-d444612993e2623">29</a></sup>. Low-passage U251 cell lines (purchased from ATCC) and patient-derived GIN 28, GIN 31, GCE 28 and GCE 31 cells were cultured in DMEM medium (Gibco) supplemented with 10% foetal bovine serum (FBS), 1% penicillin/streptomycin and 1% <span>l</span>-glutamine. Human-derived cortical astrocytes (HCOA; catalogue no. 1800, batch no. 24490, ScienCell) and cerebellar astrocytes (HCEA; catalogue no. 1810, ScienCell) were cultured in astrocyte medium containing 2% FBS, 1% astrocyte growth supplement and 1% penicillin/streptomycin from ScienCell. The human intrahepatic biliary epithelial cells (HIBEpiC) isolated from healthy human liver tissue were acquired from Innoprot (P10654) and cultured in epithelial basal medium containing 2% FBS, 1% epithelial cell growth supplement and 1% penicillin/streptomycin. All cells were maintained at 37 °C in an incubator with a humidified atmosphere, containing 5% CO<sub>2</sub>. Cells were routinely tested for mycoplasma (once a month), and they were grown in an antibiotic-free medium for one week before mycoplasma testing. All cells used were mycoplasma-free.</p><h3 id="Sec16">PrestoBlue HS assay for biocompatibility and metabolic activity studies</h3><p>The cells (U251, HCOA, GIN and GCE) were seeded in a 96-well plate at a density of 5 × 10<sup>3</sup> cells per well and allowed to adhere for 24 h. The medium was replaced with fresh medium containing GNP conjugates at different concentrations (25, 50 and 100 μg mL<sup>–1</sup>), and the cells were incubated for 8 h. The medium was removed, and the cells were washed with PBS and incubated for another 48 h in fresh medium. The medium was replaced with a complete medium containing 10% PrestoBlue HS cell viability reagent and incubated for an hour before reading the fluorescence at 590 nm and 610 nm (excitation and emission) in a Tecan microplate reader (Infinite M Plex and Spark 10M). Cells grown in culture media provided only the negative control. Values are presented relative to the negative control. The data are represented as an average of a triplicate experiment with three independent repeats.</p><h3 id="Sec17">Cellular association using ICP-MS and uptake using confocal microscopy</h3><p>GIN and GCE cells were seeded into a 24-well plate at a density of 1 × 10<sup>5</sup> cells per well and incubated at 37 °C for 24 h. After 24 h, the culture medium was replaced with fresh medium containing 25 μg mL<sup>–1</sup> of GNP100@r.Cyt <i>c</i>@Z and incubated for 8 h. Then the medium was removed, and the cells were washed with PBS (300 μL, repeated two times). The cells were trypsinized, and 50 μL of cell suspension was used for trypan blue cell viability and counting. The remaining cell suspension was centrifuged at 300<i>g</i> for 5 min. The obtained cell pellet was digested overnight with 70% nitric acid, diluted with Milli-Q water to bring the acid concentration to 2% and used for ICP-MS analysis (iCAPQ, Thermo Fischer).</p><p>To confirm the cellular uptake of the bipolar nanoelectrode, after 8 h exposure, the cells were washed with PBS (300 μL, two times), fixed with 4% paraformaldehyde for 15 min and washed twice with PBS. The cell nuclei were stained with Hoechst 33342 and actin using Phalloidin–iFluor 488 conjugate and incubated for 1 h at 37 °C in the dark. After washing the cells twice and immersing them in PBS, the fluorescence imaging was performed using a Leica TCS SPE confocal microscope. The orthogonal sections of <i>z</i> stacks were obtained, and the images were analysed using ImageJ.</p><h3 id="Sec18">Electrical stimulation studies</h3><p>U251, HIBEpiC, GIN 28, GIN 31, GCE 28 and GCE 31 cells were seeded in a 24-well plate (μ-plate 24-well black ibiTreat, Thistle Scientific) at a density of 7.5 × 10<sup>4</sup> cells per well, while HCOA and HCEA were seeded at a density of 5 × 10<sup>4</sup> cells per well in a poly-<span>l</span>-lysine-coated 24-well plate. The cells were incubated for 24 h at 37 °C and 5% CO<sub>2</sub>, and then the cell culture medium was replaced with fresh medium containing bio-nanoantennae (25 μg mL<sup>–1</sup>) and incubated for 8 h. The cells were washed twice with PBS. Two steel electrodes (0.5 mm × 25 mm) were placed at a fixed distance (at opposite sides of the well and 10 mm from each other) in each well of a 24-well plate and dipped in cell culture medium. The electrodes were connected to an arbitrary function generator (AFG-21225, RS PRO) to deliver the required a.c. sine-wave signals, frequency and amplitude. The cells were stimulated with a.c. EFs with a frequency of 3 MHz and a peak voltage amplitude of 0.65 V cm<sup>–1</sup> for a period of 2 h or 12 h. The EF between the electrodes was measured using a digital oscilloscope (TDS 210, Tektronix), and the temperature was monitored every 2 h using an infrared laser gun (IR-801, ATP). The intensity of the EF is expressed in peak voltage amplitude per centimetre (V cm<sup>–1</sup>). The metabolic activity of cells after ES was analyzed using a PrestoBlue HS assay.</p><h3 id="Sec19">Calcein AM and propidium iodide live–dead assay</h3><p>Immediately after the ES, the medium was removed and replaced with fresh medium containing mixed dyes 1 μM calcein AM and 1 μg mL<sup>−1</sup> propidium iodide, and incubated for 30 min at 37 °C and 5% CO<sub>2</sub>. The cells were washed twice with PBS, and fresh phenol red free medium was added. The cells were imaged using a Nikon Eclipse Ti fluorescent microscope with GFP and mCherry filter settings. The populations of live and dead cells were quantified using ImageJ software.</p><h3 id="Sec20">H<sub>2</sub>DCFDA/DCF reactive oxygen species generation assay</h3><p>The cells were incubated with non-fluorescent cell-permeant 2′,7′-dichlorodihydrofluorescein diacetate (H<sub>2</sub>DCFDA, 5 μM) probe for 30 min prior to ES. Immediately after the ES, the cells were washed with PBS. The generated reactive oxygen species converted H<sub>2</sub>DCFDA into 2′,7′-dichlorofluorescein. Green fluorescence of 2′,7′-dichlorofluorescein was detected using a Nikon Eclipse Ti with FITC filter settings.</p><h3 id="Sec21">Caspase 3/7 flow cytometry analysis of cell death</h3><p>Immediately after the ES, cells were trypsinized and centrifuged (300<i>g</i> for 5 min) to obtain a cell pellet. After washing with PBS, cells were incubated with a dye master mix containing CellEvent Caspase-3/7 Green Detection Reagent (1:1,000) and Zombie NIR fixable viability stain (1:2,500) for 30 min. Then the cells were centrifuged at 300<i>g</i> for 5 min, washed with PBS and fixed with 4% paraformaldehyde. The fluorescence signals of the caspase 3/7 dye (excitation and emission, 511 nm and 523 nm) and Zombie NIR dye (excitation and emission, 719 nm and 746 nm), characteristic for apoptotic and necrotic cell populations, respectively, were detected using a Sony ID7000 spectral flow cytometer. Kaluza software (v.2.1) was used to analyze the data.</p><h3 id="Sec22">Caspase 3/7 detection using confocal microscope</h3><p>Immediately after the ES, the medium was removed and the cells were incubated with 8 µM CellEvent Caspase-3/7 Green ReadyProbes Reagent in PBS containing 5% FBS for 30 min at 37 °C. Afterwards, the cells were fixed with 4% paraformaldehyde for 20 min and subsequently washed twice with PBS. Later the cells were treated with actin stain Phalloidin–iFluor 594 conjugate for 90 min at 37 °C in the dark and washed again with PBS, followed by staining with Hoechst 33342 for 10 min. Finally, the cells were washed with PBS (twice) and imaged using a Leica TCS SPE confocal microscope with a ×63 objective using the filter settings of Alexa Fluor 488 and Texas Red 594 dyes.</p><h3 id="Sec23">Colocalization studies</h3><p>GIN and GCE cells were seeded at a density of 4 × 10<sup>4</sup> cells per well in a 24-well plate and incubated at 37 °C for 24 h. After 24 h, the culture medium was replaced with fresh medium containing CellLight Late Endosomes-GFP, BacMam 2.0 and incubated overnight at 37 °C and 5% CO<sub>2</sub>. Later, the medium was replaced with fresh medium containing 25 μg mL<sup>–1</sup> of GNP100@r.Cyt <i>c</i>@Z and incubated for 8 h. Immediately after the ES, the cells were washed with PBS and imaged using a Leica confocal microscope. For lysosomal staining, immediately after ES, the cells were incubated with medium containing 100 nM LysoTracker Green DND-26 for 30 min followed by washing and imaging.</p><h3 id="Sec24">Gene regulation analysis</h3><h4 id="Sec25">Differential gene regulation analysis</h4><p>Immediately after the ES (3 MHz, 0.65 V cm<sup>–1</sup>, 2 hours) cells were washed with PBS, trypsinized and centrifuged to obtain a pellet. The cell pellets were snap-frozen in liquid nitrogen for 5 min and stored at −80 °C until shipment (in dry ice) to the Qiagen genomics facility at Hilden, Germany.</p><h4 id="Sec26">Sample preparation</h4><p>RNA was isolated from 200,000 cells using the RNeasy Micro (Qiagen) according to the manufacturer’s instructions with an elution volume of 14 µL.</p><h4 id="Sec27">Library preparation and sequencing</h4><p>The library preparation was done using the QIAseq UPX 3′ Transcriptome Kit (Qiagen). A total of 10 ng purified RNA was converted into cDNA Next Generation Sequencing (NGS libraries). During reverse transcription, each cell is tagged with a unique ID and each RNA molecule is tagged with a unique molecular index (UMI). Then the RNA is converted to cDNA (complementary Deoxyribonucleic acid). The cDNA was amplified, the PCR (polymerase chain reaction) indices were added and the libraries were purified. Library preparation was quality controlled using capillary electrophoresis (Agilent DNA 7500 Chip). Based on the quality of the inserts and the concentration measurements, the libraries were pooled in equimolar ratios. The library pool(s) were quantified using qPCR (quantitative polymerase chain reaction). Each library pool was then sequenced on a NextSeq (Illumina) sequencing instrument according to the manufacturer instructions with 100 bp read length for read 1 and 27 bp for read2. Raw data were de-multiplexed, and FASTQ files for each sample were generated using the bcl2fastq2 software (Illumina).</p><h4 id="Sec28">Read demultiplexing, mapping and quantification of gene expression</h4><p>The ‘Demultiplex QIAseq UPX 3′ reads’ tool of the CLC Genomics Workbench v.20.0.4 was used to de-multiplex the raw sequencing reads according to the sample indices. The ‘Quantify QIAseq UPX 3′ workflow’ was used to process the de-multiplexed sequencing reads with default settings. In short, the reads are annotated with their UMI and are then trimmed for poly(A) and adapter sequences, minimum reads length (15 nucleotides), read quality and ambiguous nucleotides (maximum of 2). They are then deduplicated using their UMI. Reads are grouped into UMI groups when they (1) start at the same position based on the end of the read to which the UMI is ligated (that is, Read2 for paired data), (2) are from the same strand and (3) have identical UMIs. Groups that contain only one read (singletons) are merged into non-singleton groups if the singleton’s UMI can be converted to a UMI of a non-singleton group by introducing an single nucleotide polymorphism (the biggest group is chosen). The reads were then mapped to the human genome hg38 and annotated using the refseq GRCh38.p13 mRNA (messenger Ribonucelic acid) annotation.</p><p>The ‘Empirical analysis of DGE’ algorithm of the CLC Genomics Workbench v.21.0.4 was used for differential expression analysis with default settings. It is an implementation of the ‘Exact Test’ for two-group comparisons developed by Robinson and Smyth<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 48" title="Robinson, M. &amp; Smyth, G. Small-sample estimation of negative binomial dispersion, with applications to SAGE data. Biostatistics 9, 321–332 (2008)." href="https://bytes.zone/articles/s41565-023-01496-y#ref-CR48" id="ref-link-section-d444612993e2801">48</a></sup> and incorporated in the EdgeR Bioconductor Package (Robinson et al., 2010)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 49" title="Robinson, M., McCarthy, D. &amp; Smyth, G. edgeR: a Bioconductor package for differential expression analysis of digital geneexpression data. Bioinformatics 26, 139–140 (2010)." href="https://bytes.zone/articles/s41565-023-01496-y#ref-CR49" id="ref-link-section-d444612993e2805">49</a></sup>.</p><p>For all unsupervised analysis, only genes were considered with at least ten counts summed over all samples. A variance stabilizing transformation was performed on the raw count matrix using the function vst of the R package DESeq2 v.1.28.1. Some 500 genes with the highest variance were used for the principal component analysis. The variance was calculated agnostically to the predefined groups (blind = TRUE). Some 35 genes with the highest variance across samples were selected for hierarchical clustering.</p><h4 id="Sec29">Differential gene expression and gene-set enrichment analysis</h4><p>To identify differentially expressed genes, we used the linear modelling-based limma algorithm on the transcriptome dataset<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 50" title="Reinhardt, J. et al. MAPK signaling and inflammation link melanoma phenotype switching to induction of CD73 during immunotherapy. Cancer Res. 77, 4697–4709 (2017)." href="https://bytes.zone/articles/s41565-023-01496-y#ref-CR50" id="ref-link-section-d444612993e2821">50</a></sup>. Briefly, we compared differential mRNA expression among the different conditions (treated versus untreated) across the cell lines. The significantly regulated genes were selected with an adjusted <i>P</i> value below 0.05 using the Benjamini–Hochberg correction method for multiple testing. Enrichment analysis of the biological processes was carried out by a gene-set enrichment analysis algorithm<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 51" title="Subramanian, A. et al. Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles. Proc. Natl Acad. Sci. USA 102, 15545–15550 (2005)." href="https://bytes.zone/articles/s41565-023-01496-y#ref-CR51" id="ref-link-section-d444612993e2828">51</a></sup>. Briefly, the gene sets were obtained from MSigDB (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 52" title="Liberzon, A. et al. The Molecular Signatures Database hallmark gene set collection. Cell Syst. 1, 417–425 (2015)." href="https://bytes.zone/articles/s41565-023-01496-y#ref-CR52" id="ref-link-section-d444612993e2832">52</a></sup>), and enrichment analyses were conducted among the different conditions across cell lines. Normalized enrichment scores, <i>P</i> values and adjusted <i>P</i> values (calculated with a standard Benjamini–Hochberg procedure) were retrieved for each of the gene sets. The gene sets with higher normalized enrichment score values and an adjusted <i>P</i> value &lt; 0.05 were considered as enriched for a specific GBM region.</p><p>To identify the cell-line-specific responses to treatment, the top differentially regulated pathways (<i>P</i> value &lt; 0.05) with high normalized enrichment scores across the astrocytic and GBM-derived cell lines were selected.</p><h3 id="Sec30">Mathematical model of metabolic activity, charging rate and quantum tunnelling</h3><p>We looked to develop a mathematical model to support the characteristic exponential decay connected with quantum mechanics:</p><div id="Equ5"><p><span>$$P\propto {\mathrm{{e}}}^{-\alpha r}$$</span></p><p>
                    (5)
                </p></div><p>where <i>P</i> is the probability of electron tunnelling; <i>α</i> is the inverse localization length, which scales with the energy barrier that an electron must tunnel through; and <i>r</i> is the length of the energy barrier. We define an intrinsic rate of cell death (directly proportional to the probability of a given cell dying within a given time frame), <i>r</i><sub>d</sub>, which we assume is proportional to the rate of cytochrome charging. Then, the number of dead cells (<i>D</i>) at time <i>t</i> is</p><div id="Equ6"><p><span>$$D\left(t\right)={\int }_{\!\!0}^{t}{\rm{d}}{t}^{{\prime} }A\left({t}^{{\prime} }\right){r}_{\mathrm{{d}}}$$</span></p><p>
                    (6)
                </p></div><p>where <i>A</i> is the number of alive cells. The metabolic activity is given by</p><div id="Equ7"><p><span>$$M\left(t\right)=\frac{A(t)}{A(t)+D(t)}=\frac{T-{r}_{\mathrm{{d}}}{\int }_{\!\!0}^{t}{\rm{d}}{t}^{{\prime} }A\left({t}^{{\prime} }\right)}{T}$$</span></p><p>
                    (7)
                </p></div><p>where <i>T</i> = <i>D</i> + <i>A</i> is the total number of cells. Solving the above equation, we find that <span>\(M\left(t\right)={{\mathrm{e}}}^{-{r}_{{\mathrm{d}}}t}.\)</span> therefore, the rate of charging at any given time (<i>t</i>) is given by <i>r</i><sub>d</sub> <span>∝</span> –ln[<i>M</i>(<i>t</i>)].</p><h3 id="Sec31">Dark-field microscopy and plasmon resonance scattering spectroscopy</h3><p>The plasmon resonance scattering measurements were carried out on an inverted dark-field microscope (eclipse Ti-U, Nikon) using a ×40 objective lens (numerical aperture, 0.6) and a dark-field condenser (0.8 &lt; numerical aperture &lt; 0.95). A halogen lamp (100 W) was used as a source of white light to generate plasmon resonance scattering light. The dark-field images were captured by a true-colour digital camera (Nikon DS-fi). The light scattered from the bifunctionalized nanoantennae was split by a monochromator (grating density, 300 lines mm<sup>–1</sup>; blazed wavelength, 500 nm; Acton SP2300i, Princeton Instruments). An IsoPlane-320 spectrometer was used, and the split light was collected by a charge-coupled device (Pixis 100BX, Princeton Instruments). An a.c. EF of 3 MHz at 0.65 V was applied for 10 min, and scattering spectra were monitored (1,000 frames recorded). The exposure time was 500 ms. The samples for plasmon resonance scattering were prepared by immobilizing nanoantennae on ITO. First, ITO slides were treatment with ethanol, acetone and water under sonication. Next, 50 µl nanoantennae solution was drop-casted on the ITO slides for 10 min, followed by a single-step washing and rinsing with water. Finally, the slides were dried with N<sub>2</sub> gas.</p><h3 id="Sec32">Statistics and reproducibility</h3><p>All the statistical analyses were performed using GraphPad Prism v.9.4.1 software (GraphPad Software). All the data are expressed as mean ± s.e.m., unless specified. For responses that were affected by two variables, a two-way ANOVA with a Tukey post-test was used. The value <i>P</i> ≤ 0.05 was considered significant. The number of technical replicates and independent repeats is included in the figure legends.</p><h3 id="Sec33">Reporting summary</h3><p>Further information on research design is available in the <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://bytes.zone/articles/s41565-023-01496-y#MOESM2">Nature Portfolio Reporting Summary</a> linked to this article.</p></div></div></section>
                </div><div>
            <section data-title="Data availability"><div id="data-availability-section"><h2 id="data-availability">Data availability</h2><p><a data-track="click" data-track-label="link" data-track-action="section anchor" href="https://bytes.zone/articles/s41565-023-01496-y#Sec37">Source data</a> are provided with this paper. All other data associated with this manuscript (including the Supplementary Information) can be found at <a href="https://doi.org/10.17639/nott.7303">https://doi.org/10.17639/nott.7303</a>. The complete dataset of transcriptomics analysis can be found at the Gene Expression Omnibus (GEO) under accession number <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE233380">GSE233380</a>.</p></div></section><div id="MagazineFulltextArticleBodySuffix"><section aria-labelledby="Bib1" data-title="References"><div id="Bib1-section"><h2 id="Bib1">References</h2><div id="Bib1-content"><div data-container-section="references"><ol data-track-component="outbound reference"><li data-counter="1."><p id="ref-CR1">McLaughlin, K. A. &amp; Levin, M. Bioelectric signaling in regeneration: mechanisms of ionic controls of growth and form. <i>Dev. Biol.</i> <b>433</b>, 177–189 (2018).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.ydbio.2017.08.032" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.ydbio.2017.08.032" aria-label="Article reference 1" data-doi="10.1016/j.ydbio.2017.08.032">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://bytes.zone/articles/cas-redirect/1:CAS:528:DC%2BC1cXhslWgtQ%3D%3D" aria-label="CAS reference 1">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 1" href="http://scholar.google.com/scholar_lookup?&amp;title=Bioelectric%20signaling%20in%20regeneration%3A%20mechanisms%20of%20ionic%20controls%20of%20growth%20and%20form&amp;journal=Dev.%20Biol.&amp;doi=10.1016%2Fj.ydbio.2017.08.032&amp;volume=433&amp;pages=177-189&amp;publication_year=2018&amp;author=McLaughlin%2CKA&amp;author=Levin%2CM">
                    Google Scholar</a> 
                </p></li><li data-counter="2."><p id="ref-CR2">Schofield, Z. et al. Bioelectrical understanding and engineering of cell biology. <i>J. R. Soc. Interface</i> <b>17</b>, 20200013 (2020).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1098/rsif.2020.0013" data-track-action="article reference" href="https://doi.org/10.1098%2Frsif.2020.0013" aria-label="Article reference 2" data-doi="10.1098/rsif.2020.0013">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://bytes.zone/articles/cas-redirect/1:CAS:528:DC%2BB3cXitVOmt77O" aria-label="CAS reference 2">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 2" href="http://scholar.google.com/scholar_lookup?&amp;title=Bioelectrical%20understanding%20and%20engineering%20of%20cell%20biology&amp;journal=J.%20R.%20Soc.%20Interface&amp;doi=10.1098%2Frsif.2020.0013&amp;volume=17&amp;publication_year=2020&amp;author=Schofield%2CZ">
                    Google Scholar</a> 
                </p></li><li data-counter="3."><p id="ref-CR3">Levin, M. Bioelectric signaling: reprogrammable circuits underlying embryogenesis, regeneration, and cancer. <i>Cell</i> <b>184</b>, 1971–1989 (2021).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.cell.2021.02.034" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.cell.2021.02.034" aria-label="Article reference 3" data-doi="10.1016/j.cell.2021.02.034">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://bytes.zone/articles/cas-redirect/1:CAS:528:DC%2BB3MXot1KjsbY%3D" aria-label="CAS reference 3">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 3" href="http://scholar.google.com/scholar_lookup?&amp;title=Bioelectric%20signaling%3A%20reprogrammable%20circuits%20underlying%20embryogenesis%2C%20regeneration%2C%20and%20cancer&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2021.02.034&amp;volume=184&amp;pages=1971-1989&amp;publication_year=2021&amp;author=Levin%2CM">
                    Google Scholar</a> 
                </p></li><li data-counter="4."><p id="ref-CR4">Lambert, N. et al. Quantum biology. <i>Nat. Phys.</i> <b>9</b>, 10–18 (2013).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/nphys2474" data-track-action="article reference" href="https://doi.org/10.1038%2Fnphys2474" aria-label="Article reference 4" data-doi="10.1038/nphys2474">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://bytes.zone/articles/cas-redirect/1:CAS:528:DC%2BC38Xhsl2lu7nK" aria-label="CAS reference 4">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 4" href="http://scholar.google.com/scholar_lookup?&amp;title=Quantum%20biology&amp;journal=Nat.%20Phys.&amp;doi=10.1038%2Fnphys2474&amp;volume=9&amp;pages=10-18&amp;publication_year=2013&amp;author=Lambert%2CN">
                    Google Scholar</a> 
                </p></li><li data-counter="5."><p id="ref-CR5">Garrido, C. et al. Mechanisms of cytochrome <i>c</i> release from mitochondria. <i>Cell Death Differ.</i> <b>13</b>, 1423–1433 (2006).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/sj.cdd.4401950" data-track-action="article reference" href="https://doi.org/10.1038%2Fsj.cdd.4401950" aria-label="Article reference 5" data-doi="10.1038/sj.cdd.4401950">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://bytes.zone/articles/cas-redirect/1:CAS:528:DC%2BD28XotVylurs%3D" aria-label="CAS reference 5">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 5" href="http://scholar.google.com/scholar_lookup?&amp;title=Mechanisms%20of%20cytochrome%20c%20release%20from%20mitochondria&amp;journal=Cell%20Death%20Differ.&amp;doi=10.1038%2Fsj.cdd.4401950&amp;volume=13&amp;pages=1423-1433&amp;publication_year=2006&amp;author=Garrido%2CC">
                    Google Scholar</a> 
                </p></li><li data-counter="6."><p id="ref-CR6">Devault, D., Parkes, J. H. &amp; Chance, B. Electron tunnelling in cytochromes. <i>Nature</i> <b>215</b>, 642–644 (1967).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/215642a0" data-track-action="article reference" href="https://doi.org/10.1038%2F215642a0" aria-label="Article reference 6" data-doi="10.1038/215642a0">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://bytes.zone/articles/cas-redirect/1:CAS:528:DyaF2sXkvFKrtr4%3D" aria-label="CAS reference 6">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 6" href="http://scholar.google.com/scholar_lookup?&amp;title=Electron%20tunnelling%20in%20cytochromes&amp;journal=Nature&amp;doi=10.1038%2F215642a0&amp;volume=215&amp;pages=642-644&amp;publication_year=1967&amp;author=Devault%2CD&amp;author=Parkes%2CJH&amp;author=Chance%2CB">
                    Google Scholar</a> 
                </p></li><li data-counter="7."><p id="ref-CR7">Wuttke, D. S., Bjerrum, M. J., Winkler, J. R. &amp; Gray, H. B. Electron-tunneling pathways in cytochrome c. <i>Science</i> <b>256</b>, 1007–1009 (1992).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1126/science.256.5059.1007" data-track-action="article reference" href="https://doi.org/10.1126%2Fscience.256.5059.1007" aria-label="Article reference 7" data-doi="10.1126/science.256.5059.1007">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://bytes.zone/articles/cas-redirect/1:CAS:528:DyaK38XktVejsrw%3D" aria-label="CAS reference 7">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 7" href="http://scholar.google.com/scholar_lookup?&amp;title=Electron-tunneling%20pathways%20in%20cytochrome%20c&amp;journal=Science&amp;doi=10.1126%2Fscience.256.5059.1007&amp;volume=256&amp;pages=1007-1009&amp;publication_year=1992&amp;author=Wuttke%2CDS&amp;author=Bjerrum%2CMJ&amp;author=Winkler%2CJR&amp;author=Gray%2CHB">
                    Google Scholar</a> 
                </p></li><li data-counter="8."><p id="ref-CR8">Xin, H. et al. Quantum biological tunnel junction for electron transfer imaging in live cells. <i>Nat. Commun.</i> <b>10</b>, 3245 (2019).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/s41467-019-11212-x" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41467-019-11212-x" aria-label="Article reference 8" data-doi="10.1038/s41467-019-11212-x">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 8" href="http://scholar.google.com/scholar_lookup?&amp;title=Quantum%20biological%20tunnel%20junction%20for%20electron%20transfer%20imaging%20in%20live%20cells&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-019-11212-x&amp;volume=10&amp;publication_year=2019&amp;author=Xin%2CH">
                    Google Scholar</a> 
                </p></li><li data-counter="9."><p id="ref-CR9">Vaughn, A. E. &amp; Deshmukh, M. Glucose metabolism inhibits apoptosis in neurons and cancer cells by redox inactivation of cytochrome <i>c</i>. <i>Nat. Cell Biol.</i> <b>10</b>, 1477–1483 (2008).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/ncb1807" data-track-action="article reference" href="https://doi.org/10.1038%2Fncb1807" aria-label="Article reference 9" data-doi="10.1038/ncb1807">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://bytes.zone/articles/cas-redirect/1:CAS:528:DC%2BD1cXhsVentbbE" aria-label="CAS reference 9">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 9" href="http://scholar.google.com/scholar_lookup?&amp;title=Glucose%20metabolism%20inhibits%20apoptosis%20in%20neurons%20and%20cancer%20cells%20by%20redox%20inactivation%20of%20cytochrome%20c&amp;journal=Nat.%20Cell%20Biol.&amp;doi=10.1038%2Fncb1807&amp;volume=10&amp;pages=1477-1483&amp;publication_year=2008&amp;author=Vaughn%2CAE&amp;author=Deshmukh%2CM">
                    Google Scholar</a> 
                </p></li><li data-counter="10."><p id="ref-CR10">Robinson, A. J. et al. Toward hijacking bioelectricity in cancer to develop new bioelectronic medicine. <i>Adv. Therapeutics</i> <b>4</b>, 2000248 (2021).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1002/adtp.202000248" data-track-action="article reference" href="https://doi.org/10.1002%2Fadtp.202000248" aria-label="Article reference 10" data-doi="10.1002/adtp.202000248">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://bytes.zone/articles/cas-redirect/1:CAS:528:DC%2BB3MXnsVyrsLo%3D" aria-label="CAS reference 10">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 10" href="http://scholar.google.com/scholar_lookup?&amp;title=Toward%20hijacking%20bioelectricity%20in%20cancer%20to%20develop%20new%20bioelectronic%20medicine&amp;journal=Adv.%20Therapeutics&amp;doi=10.1002%2Fadtp.202000248&amp;volume=4&amp;publication_year=2021&amp;author=Robinson%2CAJ">
                    Google Scholar</a> 
                </p></li><li data-counter="11."><p id="ref-CR11">Kuwana, T. et al. Bid, Bax, and lipids cooperate to form supramolecular openings in the outer mitochondrial membrane. <i>Cell</i> <b>111</b>, 331–342 (2002).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/S0092-8674(02)01036-X" data-track-action="article reference" href="https://doi.org/10.1016%2FS0092-8674%2802%2901036-X" aria-label="Article reference 11" data-doi="10.1016/S0092-8674(02)01036-X">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://bytes.zone/articles/cas-redirect/1:CAS:528:DC%2BD38XovVakurc%3D" aria-label="CAS reference 11">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 11" href="http://scholar.google.com/scholar_lookup?&amp;title=Bid%2C%20Bax%2C%20and%20lipids%20cooperate%20to%20form%20supramolecular%20openings%20in%20the%20outer%20mitochondrial%20membrane&amp;journal=Cell&amp;doi=10.1016%2FS0092-8674%2802%2901036-X&amp;volume=111&amp;pages=331-342&amp;publication_year=2002&amp;author=Kuwana%2CT">
                    Google Scholar</a> 
                </p></li><li data-counter="12."><p id="ref-CR12">Brown, G. C. &amp; Borutaite, V. Regulation of apoptosis by the redox state of cytochrome <i>c</i>. <i>Biochim. Biophys. Acta Bioenerg.</i> <b>1777</b>, 877–881 (2008).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.bbabio.2008.03.024" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.bbabio.2008.03.024" aria-label="Article reference 12" data-doi="10.1016/j.bbabio.2008.03.024">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://bytes.zone/articles/cas-redirect/1:CAS:528:DC%2BD1cXnsFCmtbY%3D" aria-label="CAS reference 12">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 12" href="http://scholar.google.com/scholar_lookup?&amp;title=Regulation%20of%20apoptosis%20by%20the%20redox%20state%20of%20cytochrome%20c&amp;journal=Biochim.%20Biophys.%20Acta%20Bioenerg.&amp;doi=10.1016%2Fj.bbabio.2008.03.024&amp;volume=1777&amp;pages=877-881&amp;publication_year=2008&amp;author=Brown%2CGC&amp;author=Borutaite%2CV">
                    Google Scholar</a> 
                </p></li><li data-counter="13."><p id="ref-CR13">Zou, H., Henzel, W. J., Liu, X., Lutschg, A. &amp; Wang, X. Apaf-1, a human protein homologous to C. elegans CED-4, participates in cytochrome c–dependent activation of caspase-3. <i>Cell</i> <b>90</b>, 405–413 (1997).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/S0092-8674(00)80501-2" data-track-action="article reference" href="https://doi.org/10.1016%2FS0092-8674%2800%2980501-2" aria-label="Article reference 13" data-doi="10.1016/S0092-8674(00)80501-2">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://bytes.zone/articles/cas-redirect/1:CAS:528:DyaK2sXlsVKrsrk%3D" aria-label="CAS reference 13">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 13" href="http://scholar.google.com/scholar_lookup?&amp;title=Apaf-1%2C%20a%20human%20protein%20homologous%20to%20C.%20elegans%20CED-4%2C%20participates%20in%20cytochrome%20c%E2%80%93dependent%20activation%20of%20caspase-3&amp;journal=Cell&amp;doi=10.1016%2FS0092-8674%2800%2980501-2&amp;volume=90&amp;pages=405-413&amp;publication_year=1997&amp;author=Zou%2CH&amp;author=Henzel%2CWJ&amp;author=Liu%2CX&amp;author=Lutschg%2CA&amp;author=Wang%2CX">
                    Google Scholar</a> 
                </p></li><li data-counter="14."><p id="ref-CR14">Cordier, B. A., Sawaya, N. P., Guerreschi, G. G. &amp; McWeeney, S. K. Biology and medicine in the landscape of quantum advantages. <i>J. R. Soc. Interface</i> <b>19</b>, 20220541 (2022).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1098/rsif.2022.0541" data-track-action="article reference" href="https://doi.org/10.1098%2Frsif.2022.0541" aria-label="Article reference 14" data-doi="10.1098/rsif.2022.0541">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://bytes.zone/articles/cas-redirect/1:CAS:528:DC%2BB3sXnt1KksQ%3D%3D" aria-label="CAS reference 14">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 14" href="http://scholar.google.com/scholar_lookup?&amp;title=Biology%20and%20medicine%20in%20the%20landscape%20of%20quantum%20advantages&amp;journal=J.%20R.%20Soc.%20Interface&amp;doi=10.1098%2Frsif.2022.0541&amp;volume=19&amp;publication_year=2022&amp;author=Cordier%2CBA&amp;author=Sawaya%2CNP&amp;author=Guerreschi%2CGG&amp;author=McWeeney%2CSK">
                    Google Scholar</a> 
                </p></li><li data-counter="15."><p id="ref-CR15">Bordonaro, M. Quantum biology and human carcinogenesis. <i>Biosystems</i> <b>178</b>, 16–24 (2019).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.biosystems.2019.01.010" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.biosystems.2019.01.010" aria-label="Article reference 15" data-doi="10.1016/j.biosystems.2019.01.010">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://bytes.zone/articles/cas-redirect/1:CAS:528:DC%2BC1MXitlCrurs%3D" aria-label="CAS reference 15">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 15" href="http://scholar.google.com/scholar_lookup?&amp;title=Quantum%20biology%20and%20human%20carcinogenesis&amp;journal=Biosystems&amp;doi=10.1016%2Fj.biosystems.2019.01.010&amp;volume=178&amp;pages=16-24&amp;publication_year=2019&amp;author=Bordonaro%2CM">
                    Google Scholar</a> 
                </p></li><li data-counter="16."><p id="ref-CR16">Sanjuan-Alberte, P. et al. Wireless nanobioelectronics for electrical intracellular sensing. <i>ACS Appl. Nano Mater.</i> <b>2</b>, 6397–6408 (2019).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1021/acsanm.9b01374" data-track-action="article reference" href="https://doi.org/10.1021%2Facsanm.9b01374" aria-label="Article reference 16" data-doi="10.1021/acsanm.9b01374">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://bytes.zone/articles/cas-redirect/1:CAS:528:DC%2BC1MXhslWnu7jN" aria-label="CAS reference 16">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 16" href="http://scholar.google.com/scholar_lookup?&amp;title=Wireless%20nanobioelectronics%20for%20electrical%20intracellular%20sensing&amp;journal=ACS%20Appl.%20Nano%20Mater.&amp;doi=10.1021%2Facsanm.9b01374&amp;volume=2&amp;pages=6397-6408&amp;publication_year=2019&amp;author=Sanjuan-Alberte%2CP">
                    Google Scholar</a> 
                </p></li><li data-counter="17."><p id="ref-CR17">Warakulwit, C. et al. Dissymmetric carbon nanotubes by bipolar electrochemistry. <i>Nano Lett.</i> <b>8</b>, 500–504 (2008).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1021/nl072652s" data-track-action="article reference" href="https://doi.org/10.1021%2Fnl072652s" aria-label="Article reference 17" data-doi="10.1021/nl072652s">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://bytes.zone/articles/cas-redirect/1:CAS:528:DC%2BD1cXks12jtw%3D%3D" aria-label="CAS reference 17">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 17" href="http://scholar.google.com/scholar_lookup?&amp;title=Dissymmetric%20carbon%20nanotubes%20by%20bipolar%20electrochemistry&amp;journal=Nano%20Lett.&amp;doi=10.1021%2Fnl072652s&amp;volume=8&amp;pages=500-504&amp;publication_year=2008&amp;author=Warakulwit%2CC">
                    Google Scholar</a> 
                </p></li><li data-counter="18."><p id="ref-CR18">Lei, X., Lei, C., Chen, W., Guo, Q. &amp; Huang, B. Photoelectrochemically driven nanoparticulate semiconductors as nanobipolar electrodes for advanced water remediation. <i>Cell Rep. Phys. Sci.</i> <b>3</b>, 101132 (2022).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.xcrp.2022.101132" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.xcrp.2022.101132" aria-label="Article reference 18" data-doi="10.1016/j.xcrp.2022.101132">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://bytes.zone/articles/cas-redirect/1:CAS:528:DC%2BB38XivVCnsLrE" aria-label="CAS reference 18">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 18" href="http://scholar.google.com/scholar_lookup?&amp;title=Photoelectrochemically%20driven%20nanoparticulate%20semiconductors%20as%20nanobipolar%20electrodes%20for%20advanced%20water%20remediation&amp;journal=Cell%20Rep.%20Phys.%20Sci.&amp;doi=10.1016%2Fj.xcrp.2022.101132&amp;volume=3&amp;publication_year=2022&amp;author=Lei%2CX&amp;author=Lei%2CC&amp;author=Chen%2CW&amp;author=Guo%2CQ&amp;author=Huang%2CB">
                    Google Scholar</a> 
                </p></li><li data-counter="19."><p id="ref-CR19">Loget, G. &amp; Kuhn, A. Electric field-induced chemical locomotion of conducting objects. <i>Nat. Commun.</i> <b>2</b>, 535 (2011).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/ncomms1550" data-track-action="article reference" href="https://doi.org/10.1038%2Fncomms1550" aria-label="Article reference 19" data-doi="10.1038/ncomms1550">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 19" href="http://scholar.google.com/scholar_lookup?&amp;title=Electric%20field-induced%20chemical%20locomotion%20of%20conducting%20objects&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fncomms1550&amp;volume=2&amp;publication_year=2011&amp;author=Loget%2CG&amp;author=Kuhn%2CA">
                    Google Scholar</a> 
                </p></li><li data-counter="20."><p id="ref-CR20">Forster, R. J. Wirefree electroceuticals: 3D electrical and electrochemical stimulation of biological systems. <i>Curr. Opin. Electrochem.</i> <b>39</b>, 101297 (2023).</p></li><li data-counter="21."><p id="ref-CR21">Loget, G., Roche, J. &amp; Kuhn, A. True bulk synthesis of Janus objects by bipolar electrochemistry. <i>Adv. Mater.</i> <b>24</b>, 5111–5116 (2012).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1002/adma.201201623" data-track-action="article reference" href="https://doi.org/10.1002%2Fadma.201201623" aria-label="Article reference 21" data-doi="10.1002/adma.201201623">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://bytes.zone/articles/cas-redirect/1:CAS:528:DC%2BC38XhtVehtL7J" aria-label="CAS reference 21">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 21" href="http://scholar.google.com/scholar_lookup?&amp;title=True%20bulk%20synthesis%20of%20Janus%20objects%20by%20bipolar%20electrochemistry&amp;journal=Adv.%20Mater.&amp;doi=10.1002%2Fadma.201201623&amp;volume=24&amp;pages=5111-5116&amp;publication_year=2012&amp;author=Loget%2CG&amp;author=Roche%2CJ&amp;author=Kuhn%2CA">
                    Google Scholar</a> 
                </p></li><li data-counter="22."><p id="ref-CR22">Loget, G. &amp; Kuhn, A. Shaping and exploring the micro- and nanoworld using bipolar electrochemistry. <i>Anal. Bioanal. Chem.</i> <b>400</b>, 1691–1704 (2011).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1007/s00216-011-4862-1" data-track-action="article reference" href="https://doi.org/10.1007%2Fs00216-011-4862-1" aria-label="Article reference 22" data-doi="10.1007/s00216-011-4862-1">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://bytes.zone/articles/cas-redirect/1:CAS:528:DC%2BC3MXktVOmtrs%3D" aria-label="CAS reference 22">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 22" href="http://scholar.google.com/scholar_lookup?&amp;title=Shaping%20and%20exploring%20the%20micro-%20and%20nanoworld%20using%20bipolar%20electrochemistry&amp;journal=Anal.%20Bioanal.%20Chem.&amp;doi=10.1007%2Fs00216-011-4862-1&amp;volume=400&amp;pages=1691-1704&amp;publication_year=2011&amp;author=Loget%2CG&amp;author=Kuhn%2CA">
                    Google Scholar</a> 
                </p></li><li data-counter="23."><p id="ref-CR23">Robinson, A. J. et al. Impedimetric characterization of bipolar nanoelectrodes with cancer cells. <i>ACS Omega</i> <b>6</b>, 29495–29505 (2021).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1021/acsomega.1c03547" data-track-action="article reference" href="https://doi.org/10.1021%2Facsomega.1c03547" aria-label="Article reference 23" data-doi="10.1021/acsomega.1c03547">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://bytes.zone/articles/cas-redirect/1:CAS:528:DC%2BB3MXitlCrtb%2FN" aria-label="CAS reference 23">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 23" href="http://scholar.google.com/scholar_lookup?&amp;title=Impedimetric%20characterization%20of%20bipolar%20nanoelectrodes%20with%20cancer%20cells&amp;journal=ACS%20Omega&amp;doi=10.1021%2Facsomega.1c03547&amp;volume=6&amp;pages=29495-29505&amp;publication_year=2021&amp;author=Robinson%2CAJ">
                    Google Scholar</a> 
                </p></li><li data-counter="24."><p id="ref-CR24">Hicks, J. M. et al. Electric field induced biomimetic transmembrane electron transport using carbon nanotube porins. <i>Small</i> <b>17</b>, 2102517 (2021).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1002/smll.202102517" data-track-action="article reference" href="https://doi.org/10.1002%2Fsmll.202102517" aria-label="Article reference 24" data-doi="10.1002/smll.202102517">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://bytes.zone/articles/cas-redirect/1:CAS:528:DC%2BB3MXhs1emtLjE" aria-label="CAS reference 24">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 24" href="http://scholar.google.com/scholar_lookup?&amp;title=Electric%20field%20induced%20biomimetic%20transmembrane%20electron%20transport%20using%20carbon%20nanotube%20porins&amp;journal=Small&amp;doi=10.1002%2Fsmll.202102517&amp;volume=17&amp;publication_year=2021&amp;author=Hicks%2CJM">
                    Google Scholar</a> 
                </p></li><li data-counter="25."><p id="ref-CR25">Hart, F. X. &amp; Palisano, J. R. The application of electric fields in biology and medicine. <i>Electr. Field</i> <b>23</b>, 161–186 (2018).</p><p><a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 25" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20application%20of%20electric%20fields%20in%20biology%20and%20medicine&amp;journal=Electr.%20Field&amp;volume=23&amp;pages=161-186&amp;publication_year=2018&amp;author=Hart%2CFX&amp;author=Palisano%2CJR">
                    Google Scholar</a> 
                </p></li><li data-counter="26."><p id="ref-CR26">Taghian, T., Narmoneva, D. &amp; Kogan, A. Modulation of cell function by electric field: a high-resolution analysis. <i>J. R. Soc. Interface</i> <b>12</b>, 20150153 (2015).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1098/rsif.2015.0153" data-track-action="article reference" href="https://doi.org/10.1098%2Frsif.2015.0153" aria-label="Article reference 26" data-doi="10.1098/rsif.2015.0153">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 26" href="http://scholar.google.com/scholar_lookup?&amp;title=Modulation%20of%20cell%20function%20by%20electric%20field%3A%20a%20high-resolution%20analysis&amp;journal=J.%20R.%20Soc.%20Interface&amp;doi=10.1098%2Frsif.2015.0153&amp;volume=12&amp;publication_year=2015&amp;author=Taghian%2CT&amp;author=Narmoneva%2CD&amp;author=Kogan%2CA">
                    Google Scholar</a> 
                </p></li><li data-counter="27."><p id="ref-CR27">Dickinson, E. J. &amp; Compton, R. G. Diffuse double layer at nanoelectrodes. <i>J. Phys. Chem. C</i> <b>113</b>, 17585–17589 (2009).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1021/jp906404h" data-track-action="article reference" href="https://doi.org/10.1021%2Fjp906404h" aria-label="Article reference 27" data-doi="10.1021/jp906404h">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://bytes.zone/articles/cas-redirect/1:CAS:528:DC%2BD1MXhtFGqs7jN" aria-label="CAS reference 27">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 27" href="http://scholar.google.com/scholar_lookup?&amp;title=Diffuse%20double%20layer%20at%20nanoelectrodes&amp;journal=J.%20Phys.%20Chem.%20C&amp;doi=10.1021%2Fjp906404h&amp;volume=113&amp;pages=17585-17589&amp;publication_year=2009&amp;author=Dickinson%2CEJ&amp;author=Compton%2CRG">
                    Google Scholar</a> 
                </p></li><li data-counter="28."><p id="ref-CR28">Fosdick, S. E., Knust, K. N., Scida, K. &amp; Crooks, R. M. Bipolar electrochemistry. <i>Angew. Chem. Int. Ed.</i> <b>52</b>, 10438–10456 (2013).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1002/anie.201300947" data-track-action="article reference" href="https://doi.org/10.1002%2Fanie.201300947" aria-label="Article reference 28" data-doi="10.1002/anie.201300947">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://bytes.zone/articles/cas-redirect/1:CAS:528:DC%2BC3sXhtFSgs7vK" aria-label="CAS reference 28">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 28" href="http://scholar.google.com/scholar_lookup?&amp;title=Bipolar%20electrochemistry&amp;journal=Angew.%20Chem.%20Int.%20Ed.&amp;doi=10.1002%2Fanie.201300947&amp;volume=52&amp;pages=10438-10456&amp;publication_year=2013&amp;author=Fosdick%2CSE&amp;author=Knust%2CKN&amp;author=Scida%2CK&amp;author=Crooks%2CRM">
                    Google Scholar</a> 
                </p></li><li data-counter="29."><p id="ref-CR29">Smith, S. J. et al. Metabolism-based isolation of invasive glioblastoma cells with specific gene signatures and tumorigenic potential. <i>Neuro-Oncol. Adv.</i> <b>2</b>, vdaa087 (2020).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1093/noajnl/vdaa087" data-track-action="article reference" href="https://doi.org/10.1093%2Fnoajnl%2Fvdaa087" aria-label="Article reference 29" data-doi="10.1093/noajnl/vdaa087">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 29" href="http://scholar.google.com/scholar_lookup?&amp;title=Metabolism-based%20isolation%20of%20invasive%20glioblastoma%20cells%20with%20specific%20gene%20signatures%20and%20tumorigenic%20potential&amp;journal=Neuro-Oncol.%20Adv.&amp;doi=10.1093%2Fnoajnl%2Fvdaa087&amp;volume=2&amp;publication_year=2020&amp;author=Smith%2CSJ">
                    Google Scholar</a> 
                </p></li><li data-counter="30."><p id="ref-CR30">Jain, A. et al. Electric field responsive nanotransducers for glioblastoma. <i>Bioelectron. Med.</i> <b>8</b>, 17 (2022).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1186/s42234-022-00099-7" data-track-action="article reference" href="https://doi.org/10.1186%2Fs42234-022-00099-7" aria-label="Article reference 30" data-doi="10.1186/s42234-022-00099-7">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 30" href="http://scholar.google.com/scholar_lookup?&amp;title=Electric%20field%20responsive%20nanotransducers%20for%20glioblastoma&amp;journal=Bioelectron.%20Med.&amp;doi=10.1186%2Fs42234-022-00099-7&amp;volume=8&amp;publication_year=2022&amp;author=Jain%2CA">
                    Google Scholar</a> 
                </p></li><li data-counter="31."><p id="ref-CR31">Kim, E. H., Song, H. S., Yoo, S. H. &amp; Yoon, M. Tumor treating fields inhibit glioblastoma cell migration, invasion and angiogenesis. <i>Oncotarget</i> <b>7</b>, 65125 (2016).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.18632/oncotarget.11372" data-track-action="article reference" href="https://doi.org/10.18632%2Foncotarget.11372" aria-label="Article reference 31" data-doi="10.18632/oncotarget.11372">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 31" href="http://scholar.google.com/scholar_lookup?&amp;title=Tumor%20treating%20fields%20inhibit%20glioblastoma%20cell%20migration%2C%20invasion%20and%20angiogenesis&amp;journal=Oncotarget&amp;doi=10.18632%2Foncotarget.11372&amp;volume=7&amp;publication_year=2016&amp;author=Kim%2CEH&amp;author=Song%2CHS&amp;author=Yoo%2CSH&amp;author=Yoon%2CM">
                    Google Scholar</a> 
                </p></li><li data-counter="32."><p id="ref-CR32">Zhao, J. et al. ‘Size-independent’ single-electron tunneling. <i>J. Phys. Chem. Lett.</i> <b>6</b>, 4986–4990 (2015).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1021/acs.jpclett.5b02323" data-track-action="article reference" href="https://doi.org/10.1021%2Facs.jpclett.5b02323" aria-label="Article reference 32" data-doi="10.1021/acs.jpclett.5b02323">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://bytes.zone/articles/cas-redirect/1:CAS:528:DC%2BC2MXhvFWrtr3L" aria-label="CAS reference 32">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 32" href="http://scholar.google.com/scholar_lookup?&amp;title=%E2%80%98Size-independent%E2%80%99%20single-electron%20tunneling&amp;journal=J.%20Phys.%20Chem.%20Lett.&amp;doi=10.1021%2Facs.jpclett.5b02323&amp;volume=6&amp;pages=4986-4990&amp;publication_year=2015&amp;author=Zhao%2CJ">
                    Google Scholar</a> 
                </p></li><li data-counter="33."><p id="ref-CR33">Gupta, C. et al. Quantum tunneling currents in a nanoengineered electrochemical system. <i>J. Phys. Chem. C</i> <b>121</b>, 15085–15105 (2017).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1021/acs.jpcc.7b04350" data-track-action="article reference" href="https://doi.org/10.1021%2Facs.jpcc.7b04350" aria-label="Article reference 33" data-doi="10.1021/acs.jpcc.7b04350">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://bytes.zone/articles/cas-redirect/1:CAS:528:DC%2BC2sXhtFentbfN" aria-label="CAS reference 33">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 33" href="http://scholar.google.com/scholar_lookup?&amp;title=Quantum%20tunneling%20currents%20in%20a%20nanoengineered%20electrochemical%20system&amp;journal=J.%20Phys.%20Chem.%20C&amp;doi=10.1021%2Facs.jpcc.7b04350&amp;volume=121&amp;pages=15085-15105&amp;publication_year=2017&amp;author=Gupta%2CC">
                    Google Scholar</a> 
                </p></li><li data-counter="34."><p id="ref-CR34">Kong, F.-F. et al. Wavelike electronic energy transfer in donor–acceptor molecular systems through quantum coherence. <i>Nat. Nanotechnol.</i> <b>17</b>, 729–736 (2022).</p></li><li data-counter="35."><p id="ref-CR35">Greensite, J. <i>An Introduction to Quantum Theory</i> (IOP Publishing, 2017).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1088/978-0-7503-1167-0" data-track-action="book reference" href="https://doi.org/10.1088%2F978-0-7503-1167-0" aria-label="Book reference 35" data-doi="10.1088/978-0-7503-1167-0">Book</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 35" href="http://scholar.google.com/scholar_lookup?&amp;title=An%20Introduction%20to%20Quantum%20Theory&amp;doi=10.1088%2F978-0-7503-1167-0&amp;publication_year=2017&amp;author=Greensite%2CJ">
                    Google Scholar</a> 
                </p></li><li data-counter="36."><p id="ref-CR36">Choi, Y., Kang, T. &amp; Lee, L. P. Plasmon resonance energy transfer (PRET)-based molecular imaging of cytochrome <i>c</i> in living cells. <i>Nano Lett.</i> <b>9</b>, 85–90 (2009).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1021/nl802511z" data-track-action="article reference" href="https://doi.org/10.1021%2Fnl802511z" aria-label="Article reference 36" data-doi="10.1021/nl802511z">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://bytes.zone/articles/cas-redirect/1:CAS:528:DC%2BD1cXhsFalsL7E" aria-label="CAS reference 36">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 36" href="http://scholar.google.com/scholar_lookup?&amp;title=Plasmon%20resonance%20energy%20transfer%20%28PRET%29-based%20molecular%20imaging%20of%20cytochrome%20c%20in%20living%20cells&amp;journal=Nano%20Lett.&amp;doi=10.1021%2Fnl802511z&amp;volume=9&amp;pages=85-90&amp;publication_year=2009&amp;author=Choi%2CY&amp;author=Kang%2CT&amp;author=Lee%2CLP">
                    Google Scholar</a> 
                </p></li><li data-counter="37."><p id="ref-CR37">Jing, C. et al. New insights into electrocatalysis based on plasmon resonance for the real-time monitoring of catalytic events on single gold nanorods. <i>Anal. Chem.</i> <b>86</b>, 5513–5518 (2014).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1021/ac500785u" data-track-action="article reference" href="https://doi.org/10.1021%2Fac500785u" aria-label="Article reference 37" data-doi="10.1021/ac500785u">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://bytes.zone/articles/cas-redirect/1:CAS:528:DC%2BC2cXmvFCjs7Y%3D" aria-label="CAS reference 37">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 37" href="http://scholar.google.com/scholar_lookup?&amp;title=New%20insights%20into%20electrocatalysis%20based%20on%20plasmon%20resonance%20for%20the%20real-time%20monitoring%20of%20catalytic%20events%20on%20single%20gold%20nanorods&amp;journal=Anal.%20Chem.&amp;doi=10.1021%2Fac500785u&amp;volume=86&amp;pages=5513-5518&amp;publication_year=2014&amp;author=Jing%2CC">
                    Google Scholar</a> 
                </p></li><li data-counter="38."><p id="ref-CR38">Zhou, H. et al. Optical monitoring of faradaic reaction using single plasmon-resonant nanorods functionalized with graphene. <i>Chem. Commun.</i> <b>51</b>, 3223–3226 (2015).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1039/C4CC07939F" data-track-action="article reference" href="https://doi.org/10.1039%2FC4CC07939F" aria-label="Article reference 38" data-doi="10.1039/C4CC07939F">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://bytes.zone/articles/cas-redirect/1:CAS:528:DC%2BC2MXhs1Chug%3D%3D" aria-label="CAS reference 38">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 38" href="http://scholar.google.com/scholar_lookup?&amp;title=Optical%20monitoring%20of%20faradaic%20reaction%20using%20single%20plasmon-resonant%20nanorods%20functionalized%20with%20graphene&amp;journal=Chem.%20Commun.&amp;doi=10.1039%2FC4CC07939F&amp;volume=51&amp;pages=3223-3226&amp;publication_year=2015&amp;author=Zhou%2CH">
                    Google Scholar</a> 
                </p></li><li data-counter="39."><p id="ref-CR39">Bushnell, G. W., Louie, G. V. &amp; Brayer, G. D. High-resolution three-dimensional structure of horse heart cytochrome <i>c</i>. <i>Protein Data Bank</i>, <a href="https://doi.org/10.2210/pdb1HRC/pdb" data-track="click" data-track-action="external reference" data-track-label="10.2210/pdb1HRC/pdb">https://doi.org/10.2210/pdb1HRC/pdb</a> (2021).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.2210/pdb1HRC/pdb" data-track-action="article reference" href="https://doi.org/10.2210%2Fpdb1HRC%2Fpdb" aria-label="Article reference 39" data-doi="10.2210/pdb1HRC/pdb">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 39" href="http://scholar.google.com/scholar_lookup?&amp;title=High-resolution%20three-dimensional%20structure%20of%20horse%20heart%20cytochrome%20c&amp;journal=Protein%20Data%20Bank&amp;doi=10.2210%2Fpdb1HRC%2Fpdb&amp;publication_year=2021&amp;author=Bushnell%2CGW&amp;author=Louie%2CGV&amp;author=Brayer%2CGD">
                    Google Scholar</a> 
                </p></li><li data-counter="40."><p id="ref-CR40">Bushnell, G. W., Louie, G. V. &amp; Brayer, G. D. High-resolution three-dimensional structure of horse heart cytochrome <i>c</i>. <i>J. Mol. Biol.</i>, <a href="https://doi.org/10.1016/0022-2836(90)90200-6" data-track="click" data-track-action="external reference" data-track-label="10.1016/0022-2836(90)90200-6">https://doi.org/10.1016/0022-2836(90)90200-6</a> (1990).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/0022-2836(90)90200-6" data-track-action="article reference" href="https://doi.org/10.1016%2F0022-2836%2890%2990200-6" aria-label="Article reference 40" data-doi="10.1016/0022-2836(90)90200-6">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 40" href="http://scholar.google.com/scholar_lookup?&amp;title=High-resolution%20three-dimensional%20structure%20of%20horse%20heart%20cytochrome%20c&amp;journal=J.%20Mol.%20Biol.&amp;doi=10.1016%2F0022-2836%2890%2990200-6&amp;publication_year=1990&amp;author=Bushnell%2CGW&amp;author=Louie%2CGV&amp;author=Brayer%2CGD">
                    Google Scholar</a> 
                </p></li><li data-counter="41."><p id="ref-CR41">Yuan, S., Topf, M. &amp; Akey, C. W. An improved model of the human apoptosome. <i>Protein Data Bank</i>, <a href="https://doi.org/10.2210/pdb3J2T/pdb" data-track="click" data-track-action="external reference" data-track-label="10.2210/pdb3J2T/pdb">https://doi.org/10.2210/pdb3J2T/pdb</a> (2018).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.2210/pdb3J2T/pdb" data-track-action="article reference" href="https://doi.org/10.2210%2Fpdb3J2T%2Fpdb" aria-label="Article reference 41" data-doi="10.2210/pdb3J2T/pdb">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 41" href="http://scholar.google.com/scholar_lookup?&amp;title=An%20improved%20model%20of%20the%20human%20apoptosome&amp;journal=Protein%20Data%20Bank&amp;doi=10.2210%2Fpdb3J2T%2Fpdb&amp;publication_year=2018&amp;author=Yuan%2CS&amp;author=Topf%2CM&amp;author=Akey%2CCW">
                    Google Scholar</a> 
                </p></li><li data-counter="42."><p id="ref-CR42">Yuan, S., Topf, M., Reubold, T. F., Eschenburg, S. &amp; Akey, C. W. Changes in Apaf-1 conformation that drive apoptosome assembly. <i>Biochemistry</i>, <a href="https://doi.org/10.1021/bi301721g" data-track="click" data-track-action="external reference" data-track-label="10.1021/bi301721g">https://doi.org/10.1021/bi301721g</a> (2013).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1021/bi301721g" data-track-action="article reference" href="https://doi.org/10.1021%2Fbi301721g" aria-label="Article reference 42" data-doi="10.1021/bi301721g">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 42" href="http://scholar.google.com/scholar_lookup?&amp;title=Changes%20in%20Apaf-1%20conformation%20that%20drive%20apoptosome%20assembly&amp;journal=Biochemistry&amp;doi=10.1021%2Fbi301721g&amp;publication_year=2013&amp;author=Yuan%2CS&amp;author=Topf%2CM&amp;author=Reubold%2CTF&amp;author=Eschenburg%2CS&amp;author=Akey%2CCW">
                    Google Scholar</a> 
                </p></li><li data-counter="43."><p id="ref-CR43">Fang, B., Fu, G., Agniswamy, J., Harrison, R. W. &amp; Weber, I. T. Caspase-3 binds diverse P4 residues in peptides. <i>Protein Data Bank</i>, <a href="https://doi.org/10.2210/pdb3GJQ/pdb" data-track="click" data-track-action="external reference" data-track-label="10.2210/pdb3GJQ/pdb">https://doi.org/10.2210/pdb3GJQ/pdb</a> (2011).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.2210/pdb3GJQ/pdb" data-track-action="article reference" href="https://doi.org/10.2210%2Fpdb3GJQ%2Fpdb" aria-label="Article reference 43" data-doi="10.2210/pdb3GJQ/pdb">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 43" href="http://scholar.google.com/scholar_lookup?&amp;title=Caspase-3%20binds%20diverse%20P4%20residues%20in%20peptides&amp;journal=Protein%20Data%20Bank&amp;doi=10.2210%2Fpdb3GJQ%2Fpdb&amp;publication_year=2011&amp;author=Fang%2CB&amp;author=Fu%2CG&amp;author=Agniswamy%2CJ&amp;author=Harrison%2CRW&amp;author=Weber%2CIT">
                    Google Scholar</a> 
                </p></li><li data-counter="44."><p id="ref-CR44">Fang, B., Fu, G., Agniswamy, J., Harrison, R. W. &amp; Weber, I. T. Caspase-3 binds diverse P4 residues in peptides as revealed by crystallography and structural modeling. <i>Apoptosis</i>, <a href="https://doi.org/10.1007/s10495-009-0333-y" data-track="click" data-track-action="external reference" data-track-label="10.1007/s10495-009-0333-y">https://doi.org/10.1007/s10495-009-0333-y</a> (2009).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1007/s10495-009-0333-y" data-track-action="article reference" href="https://doi.org/10.1007%2Fs10495-009-0333-y" aria-label="Article reference 44" data-doi="10.1007/s10495-009-0333-y">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 44" href="http://scholar.google.com/scholar_lookup?&amp;title=Caspase-3%20binds%20diverse%20P4%20residues%20in%20peptides%20as%20revealed%20by%20crystallography%20and%20structural%20modeling&amp;journal=Apoptosis&amp;doi=10.1007%2Fs10495-009-0333-y&amp;publication_year=2009&amp;author=Fang%2CB&amp;author=Fu%2CG&amp;author=Agniswamy%2CJ&amp;author=Harrison%2CRW&amp;author=Weber%2CIT">
                    Google Scholar</a> 
                </p></li><li data-counter="45."><p id="ref-CR45">Zinc porphyrin. <i>ChemSpider</i> <a href="http://www.chemspider.com/Chemical-Structure.19989078.html" data-track="click" data-track-action="external reference" data-track-label="http://www.chemspider.com/Chemical-Structure.19989078.html">http://www.chemspider.com/Chemical-Structure.19989078.html</a></p></li><li data-counter="46."><p id="ref-CR46">Nicholson, R. S. &amp; Shain, I. Theory of stationary electrode polarography. Single scan and cyclic methods applied to reversible, irreversible, and kinetic systems. <i>Anal. Chem.</i> <b>36</b>, 706–723 (1964).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1021/ac60210a007" data-track-action="article reference" href="https://doi.org/10.1021%2Fac60210a007" aria-label="Article reference 46" data-doi="10.1021/ac60210a007">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://bytes.zone/articles/cas-redirect/1:CAS:528:DyaF2cXktV2ms7s%3D" aria-label="CAS reference 46">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 46" href="http://scholar.google.com/scholar_lookup?&amp;title=Theory%20of%20stationary%20electrode%20polarography.%20Single%20scan%20and%20cyclic%20methods%20applied%20to%20reversible%2C%20irreversible%2C%20and%20kinetic%20systems&amp;journal=Anal.%20Chem.&amp;doi=10.1021%2Fac60210a007&amp;volume=36&amp;pages=706-723&amp;publication_year=1964&amp;author=Nicholson%2CRS&amp;author=Shain%2CI">
                    Google Scholar</a> 
                </p></li><li data-counter="47."><p id="ref-CR47">Lavagnini, I., Antiochia, R. &amp; Magno, F. An extended method for the practical evaluation of the standard rate constant from cyclic voltammetric data. <i>Electroanalysis</i> <b>16</b>, 505–506 (2004).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1002/elan.200302851" data-track-action="article reference" href="https://doi.org/10.1002%2Felan.200302851" aria-label="Article reference 47" data-doi="10.1002/elan.200302851">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://bytes.zone/articles/cas-redirect/1:CAS:528:DC%2BD2cXjtlOlsLk%3D" aria-label="CAS reference 47">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 47" href="http://scholar.google.com/scholar_lookup?&amp;title=An%20extended%20method%20for%20the%20practical%20evaluation%20of%20the%20standard%20rate%20constant%20from%20cyclic%20voltammetric%20data&amp;journal=Electroanalysis&amp;doi=10.1002%2Felan.200302851&amp;volume=16&amp;pages=505-506&amp;publication_year=2004&amp;author=Lavagnini%2CI&amp;author=Antiochia%2CR&amp;author=Magno%2CF">
                    Google Scholar</a> 
                </p></li><li data-counter="48."><p id="ref-CR48">Robinson, M. &amp; Smyth, G. Small-sample estimation of negative binomial dispersion, with applications to SAGE data. <i>Biostatistics</i> <b>9</b>, 321–332 (2008).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1093/biostatistics/kxm030" data-track-action="article reference" href="https://doi.org/10.1093%2Fbiostatistics%2Fkxm030" aria-label="Article reference 48" data-doi="10.1093/biostatistics/kxm030">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 48" href="http://scholar.google.com/scholar_lookup?&amp;title=Small-sample%20estimation%20of%20negative%20binomial%20dispersion%2C%20with%20applications%20to%20SAGE%20data&amp;journal=Biostatistics&amp;doi=10.1093%2Fbiostatistics%2Fkxm030&amp;volume=9&amp;pages=321-332&amp;publication_year=2008&amp;author=Robinson%2CM&amp;author=Smyth%2CG">
                    Google Scholar</a> 
                </p></li><li data-counter="49."><p id="ref-CR49">Robinson, M., McCarthy, D. &amp; Smyth, G. edgeR: a Bioconductor package for differential expression analysis of digital geneexpression data. <i>Bioinformatics</i> <b>26</b>, 139–140 (2010).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1093/bioinformatics/btp616" data-track-action="article reference" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtp616" aria-label="Article reference 49" data-doi="10.1093/bioinformatics/btp616">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://bytes.zone/articles/cas-redirect/1:CAS:528:DC%2BD1MXhs1WlurvO" aria-label="CAS reference 49">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 49" href="http://scholar.google.com/scholar_lookup?&amp;title=edgeR%3A%20a%20Bioconductor%20package%20for%20differential%20expression%20analysis%20of%20digital%20geneexpression%20data&amp;journal=Bioinformatics&amp;doi=10.1093%2Fbioinformatics%2Fbtp616&amp;volume=26&amp;pages=139-140&amp;publication_year=2010&amp;author=Robinson%2CM&amp;author=McCarthy%2CD&amp;author=Smyth%2CG">
                    Google Scholar</a> 
                </p></li><li data-counter="50."><p id="ref-CR50">Reinhardt, J. et al. MAPK signaling and inflammation link melanoma phenotype switching to induction of CD73 during immunotherapy. <i>Cancer Res.</i> <b>77</b>, 4697–4709 (2017).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1158/0008-5472.CAN-17-0395" data-track-action="article reference" href="https://doi.org/10.1158%2F0008-5472.CAN-17-0395" aria-label="Article reference 50" data-doi="10.1158/0008-5472.CAN-17-0395">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://bytes.zone/articles/cas-redirect/1:CAS:528:DC%2BC2sXhsV2mu73P" aria-label="CAS reference 50">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 50" href="http://scholar.google.com/scholar_lookup?&amp;title=MAPK%20signaling%20and%20inflammation%20link%20melanoma%20phenotype%20switching%20to%20induction%20of%20CD73%20during%20immunotherapy&amp;journal=Cancer%20Res.&amp;doi=10.1158%2F0008-5472.CAN-17-0395&amp;volume=77&amp;pages=4697-4709&amp;publication_year=2017&amp;author=Reinhardt%2CJ">
                    Google Scholar</a> 
                </p></li><li data-counter="51."><p id="ref-CR51">Subramanian, A. et al. Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles. <i>Proc. Natl Acad. Sci. USA</i> <b>102</b>, 15545–15550 (2005).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1073/pnas.0506580102" data-track-action="article reference" href="https://doi.org/10.1073%2Fpnas.0506580102" aria-label="Article reference 51" data-doi="10.1073/pnas.0506580102">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://bytes.zone/articles/cas-redirect/1:CAS:528:DC%2BD2MXht1ShtrnO" aria-label="CAS reference 51">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 51" href="http://scholar.google.com/scholar_lookup?&amp;title=Gene%20set%20enrichment%20analysis%3A%20a%20knowledge-based%20approach%20for%20interpreting%20genome-wide%20expression%20profiles&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;doi=10.1073%2Fpnas.0506580102&amp;volume=102&amp;pages=15545-15550&amp;publication_year=2005&amp;author=Subramanian%2CA">
                    Google Scholar</a> 
                </p></li><li data-counter="52."><p id="ref-CR52">Liberzon, A. et al. The Molecular Signatures Database hallmark gene set collection. <i>Cell Syst.</i> <b>1</b>, 417–425 (2015).</p><p><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.cels.2015.12.004" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.cels.2015.12.004" aria-label="Article reference 52" data-doi="10.1016/j.cels.2015.12.004">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="https://bytes.zone/articles/cas-redirect/1:CAS:528:DC%2BC2sXhtFaltLc%3D" aria-label="CAS reference 52">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 52" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20Molecular%20Signatures%20Database%20hallmark%20gene%20set%20collection&amp;journal=Cell%20Syst.&amp;doi=10.1016%2Fj.cels.2015.12.004&amp;volume=1&amp;pages=417-425&amp;publication_year=2015&amp;author=Liberzon%2CA">
                    Google Scholar</a> 
                </p></li></ol><p><a data-track="click" data-track-action="download citation references" data-track-label="link" rel="nofollow" href="https://citation-needed.springer.com/v2/references/10.1038/s41565-023-01496-y?format=refman&amp;flavour=references">Download references<svg width="16" height="16" focusable="false" role="img" aria-hidden="true"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-download"></use></svg></a></p></div></div></div></section></div><section data-title="Acknowledgements"><div id="Ack1-section"><h2 id="Ack1">Acknowledgements</h2><p>This work was supported by the Engineering and Physical Sciences Research Council (grant number EP/R004072/1 to F.J.R.) and Royal Society of Chemistry Enablement Grant no. E21-1135058786 to F.J.R. and A.J. A.J. acknowledges the University of Nottingham internal funding schemes NanoPrime and UNICAS. J.G., M.F. and L.T. acknowledge the Engineering and Physical Sciences Research Council (grant number EP/P031684/1). L.P.-G. thanks project PID2020-115663GB-C3, which was funded by MCIN/AEI/10.13039/501100011033 and AGAUR (Generalitat de Catalunya), for a grant to consolidated research groups 2021 SGR 01085. We thank B. Xie at Nanjing University, China, for plasmon resonance scattering and DFM (Dark Field Microscopy) analysis; D. Onion at the University of Nottingham, Flow Cytometry Unit, for assistance with flow cytometry analysis; M. Sueca-Comes for providing HIBEpiC cells; M. Fay for help with TEM; J. Potts for assistance with particle analysis; and L. Martínez Núñez for the illustration in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bytes.zone/articles/s41565-023-01496-y#Fig1">1</a>. We would also like to thank Prof. Nieves Casañ-Pastor for her feedback on the manuscript.</p></div></section><section aria-labelledby="author-information" data-title="Author information"><div id="author-information-section"><h2 id="author-information">Author information</h2><div id="author-information-content"><h3 id="affiliations">Authors and Affiliations</h3><ol><li id="Aff1"><p>Bioelectronics Laboratory, Division of Regenerative Medicine and Cellular Therapies, School of Pharmacy, Biodiscovery Institute, University of Nottingham, Nottingham, UK</p><p>Akhil Jain &amp; Frankie J. Rawson</p></li><li id="Aff2"><p>Faculty of Engineering, University of Nottingham, Nottingham, UK</p><p>Jonathan Gosling &amp; Lyudmila Turyanska</p></li><li id="Aff3"><p>State Key Laboratory of Analytical Chemistry for Life Science, School of Chemistry and Chemical Engineering, Nanjing University, Nanjing, China</p><p>Shaochuang Liu, Haowei Wang &amp; Yi-Tao Long</p></li><li id="Aff4"><p>School of Pharmacy, University of Nottingham, Nottingham, UK</p><p>Eloise M. Stone &amp; Lluïsa Pérez-García</p></li><li id="Aff5"><p>Institute of Medical Bioinformatics and Systems Medicine, Medical Center - University of Freiburg Faculty of Medicine, University of Freiburg, Freiburg, Germany</p><p>Sajib Chakraborty</p></li><li id="Aff6"><p>School of Medicine, Biodiscovery Institute, University of Nottingham, Nottingham, UK</p><p>Padma-Sheela Jayaraman</p></li><li id="Aff7"><p>Children’s Brain Tumour Research Centre, School of Medicine, Biodiscovery Institute, University of Nottingham, Nottingham, UK</p><p>Stuart Smith &amp; Ruman Rahman</p></li><li id="Aff8"><p>Department of Neurosurgery, Nottingham University Hospitals, Nottingham, UK</p><p>Stuart Smith</p></li><li id="Aff9"><p>Institut de Ciència de Materials de Barcelona (ICMAB-CSIC), Campus Universitari de Cerdanyola, Barcelona, Spain</p><p>David B. Amabilino</p></li><li id="Aff10"><p>School of Chemistry, University of Nottingham, Nottingham, UK</p><p>David B. Amabilino</p></li><li id="Aff11"><p>School of Physics and Astronomy, University of Nottingham, Nottingham, UK</p><p>Mark Fromhold</p></li><li id="Aff12"><p>Departament de Farmacologia, Toxicologia i Química Terapèutica, Facultat de Farmàcia i Ciències de l’Alimentació, Universitat de Barcelona, Barcelona, Spain</p><p>Lluïsa Pérez-García</p></li><li id="Aff13"><p>Institut de Nanociència i Nanotecnologia, Universitat de Barcelona (IN2UB), Barcelona, Spain</p><p>Lluïsa Pérez-García</p></li></ol><h3 id="contributions">Contributions</h3><p>A.J., L.P.-G., D.B.A., S.S., R.R. and F.J.R. conceptualized the work. A.J., L.P.-G., D.B.A., R.R., L.T. and F.J.R. designed the experiments and performed the analysis. A.J. synthesized and characterized the bio-nanoantennae. A.J. performed all the ES experiments in vitro and analyzed the data together with L.P.-G., D.B.A., L.T. and F.J.R.; J.G. and M.F. performed the mathematical modelling. S.L. and H.W. performed the plasmon resonance scattering. Y.-T.L. supervised S.L. and H.W.; A.J. and E.M.S. analyzed the cyclic voltammetry data. S.C. and R.R. performed the gene ontology analysis. S.S. and P.-S.J. supplied the primary GBM and normal liver bile duct cells, respectively. F.J.R. supervised A.J.; A.J. and F.J.R. wrote the first draft of the manuscript. All the authors corrected the manuscript.</p><h3 id="corresponding-author">Corresponding author</h3><p id="corresponding-author-list">Correspondence to
                <a id="corresp-c1" href="mailto:Frankie.Rawson@nottingham.ac.uk">Frankie J. Rawson</a>.</p></div></div></section><section data-title="Ethics declarations"><div id="ethics-section"><h2 id="ethics">Ethics declarations</h2><div id="ethics-content">
            
              <h3 id="FPar4">Competing interests</h3>
              <p>The research described in this manuscript has been filed for a UK patent (application no. 2302102.5) by The University of Nottingham, UK. Inventors listed are Dr Frankie Rawson, Dr Akhil Jain, Prof David Amabilino and Prof Luisa Pérez-García. All parts of the main paper have been included in the patent. The authors have no other conflict of interest to declare. All the authors read the manuscript and agreed on submission.</p>
            
          </div></div></section><section data-title="Peer review"><div id="peer-review-section"><h2 id="peer-review">Peer review</h2><div id="peer-review-content">
            
            
              <h3 id="FPar3">Peer review information</h3>
              <p><i>Nature Nanotechnology</i> thanks Jun-Tao Cao, Binbin Huang and João Pessoa for their contribution to the peer review of this work.</p>
            
          </div></div></section><section data-title="Additional information"><div id="additional-information-section"><h2 id="additional-information">Additional information</h2><p><b>Publisher’s note</b> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div></section><section data-title="Extended data"><div id="Sec35-section"><h2 id="Sec35">Extended data</h2><div id="Sec35-content"><div data-test="supplementary-info"><div data-test="supp-item" id="Fig7"><h3><a data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 1 gene-set enrichment analysis " href="https://bytes.zone/articles/s41565-023-01496-y/figures/7" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41565-023-01496-y/MediaObjects/41565_2023_1496_Fig7_ESM.jpg">Extended Data Fig. 1 Gene-set Enrichment analysis (GSEA) of Gene Ontology Biological Processes (GOBP).</a></h3><div data-component="thumbnail-container"><p><b>(a)</b> GSEA of GOBP across all cell-lines (Cortical-Astrocytes, GCE-31, and GIN-31) is represented as a heatmap. The topmost GOBP terms based on the adjusted p-value are shown. <b>(b)</b> Cortical Astrocyte Response to Treatment: GSEA of GOBP across all cell-lines (Cortical-Astrocytes, GCE-31, and GIN-31) is represented as a heatmap. The topmost GOBP terms that are differentially regulated in cortical astrocytes (based on the adjusted p-value) compared to GCE-31 and GIN-31 cell-lines are shown. <b>(c)</b> GCE/GIN 31 Cells Response to Treatment: GSEA of GOBP across all cell-lines (Cortical-Astrocytes, GCE-31, and GIN-31) is represented as a heatmap. The topmost GOBP terms that are differentially regulated in GCE-31 and GIN-31 cell-lines (based on the adjusted <i>p</i>-value) compared to cortical astrocytes are shown. In Fig. b-d: the colour code is based on the normalized enrichment score (NES). The significantly enriched terms (adj <i>p</i>-value &lt; 0.05) are indicated by (*) symbol. For the GSEA, we used one-tailed Fisher’s exact test with 95% of confidence interval. The p-values were <i>corrected</i> for <i>multiple comparisons</i> with the Benjamini-Hochberg method. The exact <i>p</i> values can be found in Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://bytes.zone/articles/s41565-023-01496-y#MOESM1">7</a>. The treatment codes are as follow: Control = no treatment with either bio-nanoantennae or AC EFs); EF = cells treated with AC EFs; EF_NP1 = cell treated with GNP100@r.Cyt <i>c</i>, and EF_NP2 = cells treated with GNP100@r.Cyt <i>c</i>@Z, for 8 h followed by 2-hour a.c. EFs.</p><p>
                        <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://bytes.zone/articles/s41565-023-01496-y#MOESM8">Source data</a>
                      </p></div></div><div data-test="supp-item" id="Fig8"><h3><a data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 2 plasmon resonance scattering " href="https://bytes.zone/articles/s41565-023-01496-y/figures/8" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41565-023-01496-y/MediaObjects/41565_2023_1496_Fig8_ESM.jpg">Extended Data Fig. 2 Plasmon resonance scattering spectra and quantized peaks to demonstrate QBET.</a></h3><p>Two-dimensional heat map of scattering spectral changes obtained from dark field microscopy, <b>(a)</b> in the absence of EF and <b>(b</b>) presence of EF.</p></div></div></div></div></section><section data-title="Supplementary information"><div id="Sec36-section"><h2 id="Sec36">Supplementary information</h2></div></section><section data-title="Source data"><div id="Sec37-section"><h2 id="Sec37">Source data</h2><div id="Sec37-content"><div data-test="supplementary-info"><div data-test="supp-item" id="MOESM3"><h3><a data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="source data fig. 2" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41565-023-01496-y/MediaObjects/41565_2023_1496_MOESM3_ESM.xlsx" data-supp-info-image="">Source Data Fig. 2</a></h3><p>Source data associated with UV–visible measurements of bio-nanoantennae before ES (Fig. 2b,c), cyclic voltammetry scan rate studies (Fig. 2d) and ICP-MS analysis (Fig. 2e).</p></div><div data-test="supp-item" id="MOESM4"><h3><a data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="source data fig. 3" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41565-023-01496-y/MediaObjects/41565_2023_1496_MOESM4_ESM.xlsx" data-supp-info-image="">Source Data Fig. 3</a></h3><p>Source data associated with metabolic activity of GIN 31 and GCE 31 cells (Fig. 3a), GIN 28 and GCE 28 cells (Fig. 3b) and cortical astrocytes (Fig. 3c).</p></div><div data-test="supp-item" id="MOESM5"><h3><a data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="source data fig. 4" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41565-023-01496-y/MediaObjects/41565_2023_1496_MOESM5_ESM.xlsx" data-supp-info-image="">Source Data Fig. 4</a></h3><p>Source data associated with the differential gene expression shown in Fig. 4.</p></div><div data-test="supp-item" id="MOESM6"><h3><a data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="source data fig. 5" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41565-023-01496-y/MediaObjects/41565_2023_1496_MOESM6_ESM.xlsx" data-supp-info-image="">Source Data Fig. 5</a></h3><p>Source data associated with circular dichroism (Fig. 5a), UV–visible measurements of bio-nanoantennae after ES (Fig. 5b,c) and the metabolic activity of GIN 31 cells as a function of different size bio-nanoantennae (20 nm, 50 nm and 100 nm; GNP20@r.Cyt <i>c</i>@Z, GNP50@r.Cyt <i>c</i>@Z and GNP100@r.Cyt <i>c</i>@Z) synthesized using various linker lengths (1, 2, 3.5 and 5 kDa; Fig. 5d–f).</p></div><div data-test="supp-item" id="MOESM7"><h3><a data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="source data fig. 6" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41565-023-01496-y/MediaObjects/41565_2023_1496_MOESM7_ESM.xlsx" data-supp-info-image="">Source Data Fig. 6</a></h3><p>Source data associated with mathematical modelling to determine the rate of donor charging, <i>r</i><sub>d</sub>, calculated using the metabolic activity, using equation (1), compared to the PEG linker length, <i>L</i>, for GNP20@r.Cyt <i>c</i>@Z (Fig. 6a) and GNP100@r.Cyt <i>c</i>@Z (Fig. 6b) in GIN 31 cell samples. Source data for the scattering spectra and spectra difference (Fig. 6c) and quantized peaks (Fig. 6d) for QBET obtained for GNP100@r.Cyt <i>c</i>@Z bio-nanoantennae.</p></div><div data-test="supp-item" id="MOESM8"><h3><a data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="source data extended data fig. 1" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41565-023-01496-y/MediaObjects/41565_2023_1496_MOESM8_ESM.xlsx" data-supp-info-image="">Source Data Extended Data Fig. 1</a></h3><p>Source data associated with gene-set enrichment analysis of gene ontology biological processes shown in Extended Data Fig. 1a–c.</p></div></div></div></div></section><section data-title="Rights and permissions"><div id="rightslink-section"><h2 id="rightslink">Rights and permissions</h2><div id="rightslink-content">
              <p><b>Open Access</b>  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this license, visit <a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>.</p>
            <p><a data-track="click" data-track-action="view rights and permissions" data-track-label="link" href="https://s100.copyright.com/AppDispatchServlet?title=Wireless%20electrical%E2%80%93molecular%20quantum%20signalling%20for%20cancer%20cell%20apoptosis&amp;author=Akhil%20Jain%20et%20al&amp;contentID=10.1038%2Fs41565-023-01496-y&amp;copyright=The%20Author%28s%29&amp;publication=1748-3387&amp;publicationDate=2023-09-14&amp;publisherName=SpringerNature&amp;orderBeanReset=true&amp;oa=CC%20BY">Reprints and Permissions</a></p></div></div></section><section aria-labelledby="article-info" data-title="About this article"><div id="article-info-section"><h2 id="article-info">About this article</h2><div id="article-info-content"><div><p><a data-crossmark="10.1038/s41565-023-01496-y" target="_blank" rel="noopener" href="https://crossmark.crossref.org/dialog/?doi=10.1038/s41565-023-01496-y" data-track="click" data-track-action="Click Crossmark" data-track-label="link" data-test="crossmark"><img width="57" height="81" alt="Check for updates. Verify currency and authenticity via CrossMark" src="data:image/svg+xml;base64,<svg height="81" width="57" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="m17.35 35.45 21.3-14.2v-17.03h-21.3" fill="#989898"/><path d="m38.65 35.45-21.3-14.2v-17.03h21.3" fill="#747474"/><path d="m28 .5c-12.98 0-23.5 10.52-23.5 23.5s10.52 23.5 23.5 23.5 23.5-10.52 23.5-23.5c0-6.23-2.48-12.21-6.88-16.62-4.41-4.4-10.39-6.88-16.62-6.88zm0 41.25c-9.8 0-17.75-7.95-17.75-17.75s7.95-17.75 17.75-17.75 17.75 7.95 17.75 17.75c0 4.71-1.87 9.22-5.2 12.55s-7.84 5.2-12.55 5.2z" fill="#535353"/><path d="m41 36c-5.81 6.23-15.23 7.45-22.43 2.9-7.21-4.55-10.16-13.57-7.03-21.5l-4.92-3.11c-4.95 10.7-1.19 23.42 8.78 29.71 9.97 6.3 23.07 4.22 30.6-4.86z" fill="#9c9c9c"/><path d="m.2 58.45c0-.75.11-1.42.33-2.01s.52-1.09.91-1.5c.38-.41.83-.73 1.34-.94.51-.22 1.06-.32 1.65-.32.56 0 1.06.11 1.51.35.44.23.81.5 1.1.81l-.91 1.01c-.24-.24-.49-.42-.75-.56-.27-.13-.58-.2-.93-.2-.39 0-.73.08-1.05.23-.31.16-.58.37-.81.66-.23.28-.41.63-.53 1.04-.13.41-.19.88-.19 1.39 0 1.04.23 1.86.68 2.46.45.59 1.06.88 1.84.88.41 0 .77-.07 1.07-.23s.59-.39.85-.68l.91 1c-.38.43-.8.76-1.28.99-.47.22-1 .34-1.58.34-.59 0-1.13-.1-1.64-.31-.5-.2-.94-.51-1.31-.91-.38-.4-.67-.9-.88-1.48-.22-.59-.33-1.26-.33-2.02zm8.4-5.33h1.61v2.54l-.05 1.33c.29-.27.61-.51.96-.72s.76-.31 1.24-.31c.73 0 1.27.23 1.61.71.33.47.5 1.14.5 2.02v4.31h-1.61v-4.1c0-.57-.08-.97-.25-1.21-.17-.23-.45-.35-.83-.35-.3 0-.56.08-.79.22-.23.15-.49.36-.78.64v4.8h-1.61zm7.37 6.45c0-.56.09-1.06.26-1.51.18-.45.42-.83.71-1.14.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.36c.07.62.29 1.1.65 1.44.36.33.82.5 1.38.5.29 0 .57-.04.83-.13s.51-.21.76-.37l.55 1.01c-.33.21-.69.39-1.09.53-.41.14-.83.21-1.26.21-.48 0-.92-.08-1.34-.25-.41-.16-.76-.4-1.07-.7-.31-.31-.55-.69-.72-1.13-.18-.44-.26-.95-.26-1.52zm4.6-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.07.45-.31.29-.5.73-.58 1.3zm2.5.62c0-.57.09-1.08.28-1.53.18-.44.43-.82.75-1.13s.69-.54 1.1-.71c.42-.16.85-.24 1.31-.24.45 0 .84.08 1.17.23s.61.34.85.57l-.77 1.02c-.19-.16-.38-.28-.56-.37-.19-.09-.39-.14-.61-.14-.56 0-1.01.21-1.35.63-.35.41-.52.97-.52 1.67 0 .69.17 1.24.51 1.66.34.41.78.62 1.32.62.28 0 .54-.06.78-.17.24-.12.45-.26.64-.42l.67 1.03c-.33.29-.69.51-1.08.65-.39.15-.78.23-1.18.23-.46 0-.9-.08-1.31-.24-.4-.16-.75-.39-1.05-.7s-.53-.69-.7-1.13c-.17-.45-.25-.96-.25-1.53zm6.91-6.45h1.58v6.17h.05l2.54-3.16h1.77l-2.35 2.8 2.59 4.07h-1.75l-1.77-2.98-1.08 1.23v1.75h-1.58zm13.69 1.27c-.25-.11-.5-.17-.75-.17-.58 0-.87.39-.87 1.16v.75h1.34v1.27h-1.34v5.6h-1.61v-5.6h-.92v-1.2l.92-.07v-.72c0-.35.04-.68.13-.98.08-.31.21-.57.4-.79s.42-.39.71-.51c.28-.12.63-.18 1.04-.18.24 0 .48.02.69.07.22.05.41.1.57.17zm.48 5.18c0-.57.09-1.08.27-1.53.17-.44.41-.82.72-1.13.3-.31.65-.54 1.04-.71.39-.16.8-.24 1.23-.24s.84.08 1.24.24c.4.17.74.4 1.04.71s.54.69.72 1.13c.19.45.28.96.28 1.53s-.09 1.08-.28 1.53c-.18.44-.42.82-.72 1.13s-.64.54-1.04.7-.81.24-1.24.24-.84-.08-1.23-.24-.74-.39-1.04-.7c-.31-.31-.55-.69-.72-1.13-.18-.45-.27-.96-.27-1.53zm1.65 0c0 .69.14 1.24.43 1.66.28.41.68.62 1.18.62.51 0 .9-.21 1.19-.62.29-.42.44-.97.44-1.66 0-.7-.15-1.26-.44-1.67-.29-.42-.68-.63-1.19-.63-.5 0-.9.21-1.18.63-.29.41-.43.97-.43 1.67zm6.48-3.44h1.33l.12 1.21h.05c.24-.44.54-.79.88-1.02.35-.24.7-.36 1.07-.36.32 0 .59.05.78.14l-.28 1.4-.33-.09c-.11-.01-.23-.02-.38-.02-.27 0-.56.1-.86.31s-.55.58-.77 1.1v4.2h-1.61zm-47.87 15h1.61v4.1c0 .57.08.97.25 1.2.17.24.44.35.81.35.3 0 .57-.07.8-.22.22-.15.47-.39.73-.73v-4.7h1.61v6.87h-1.32l-.12-1.01h-.04c-.3.36-.63.64-.98.86-.35.21-.76.32-1.24.32-.73 0-1.27-.24-1.61-.71-.33-.47-.5-1.14-.5-2.02zm9.46 7.43v2.16h-1.61v-9.59h1.33l.12.72h.05c.29-.24.61-.45.97-.63.35-.17.72-.26 1.1-.26.43 0 .81.08 1.15.24.33.17.61.4.84.71.24.31.41.68.53 1.11.13.42.19.91.19 1.44 0 .59-.09 1.11-.25 1.57-.16.47-.38.85-.65 1.16-.27.32-.58.56-.94.73-.35.16-.72.25-1.1.25-.3 0-.6-.07-.9-.2s-.59-.31-.87-.56zm0-2.3c.26.22.5.37.73.45.24.09.46.13.66.13.46 0 .84-.2 1.15-.6.31-.39.46-.98.46-1.77 0-.69-.12-1.22-.35-1.61-.23-.38-.61-.57-1.13-.57-.49 0-.99.26-1.52.77zm5.87-1.69c0-.56.08-1.06.25-1.51.16-.45.37-.83.65-1.14.27-.3.58-.54.93-.71s.71-.25 1.08-.25c.39 0 .73.07 1 .2.27.14.54.32.81.55l-.06-1.1v-2.49h1.61v9.88h-1.33l-.11-.74h-.06c-.25.25-.54.46-.88.64-.33.18-.69.27-1.06.27-.87 0-1.56-.32-2.07-.95s-.76-1.51-.76-2.65zm1.67-.01c0 .74.13 1.31.4 1.7.26.38.65.58 1.15.58.51 0 .99-.26 1.44-.77v-3.21c-.24-.21-.48-.36-.7-.45-.23-.08-.46-.12-.7-.12-.45 0-.82.19-1.13.59-.31.39-.46.95-.46 1.68zm6.35 1.59c0-.73.32-1.3.97-1.71.64-.4 1.67-.68 3.08-.84 0-.17-.02-.34-.07-.51-.05-.16-.12-.3-.22-.43s-.22-.22-.38-.3c-.15-.06-.34-.1-.58-.1-.34 0-.68.07-1 .2s-.63.29-.93.47l-.59-1.08c.39-.24.81-.45 1.28-.63.47-.17.99-.26 1.54-.26.86 0 1.51.25 1.93.76s.63 1.25.63 2.21v4.07h-1.32l-.12-.76h-.05c-.3.27-.63.48-.98.66s-.73.27-1.14.27c-.61 0-1.1-.19-1.48-.56-.38-.36-.57-.85-.57-1.46zm1.57-.12c0 .3.09.53.27.67.19.14.42.21.71.21.28 0 .54-.07.77-.2s.48-.31.73-.56v-1.54c-.47.06-.86.13-1.18.23-.31.09-.57.19-.76.31s-.33.25-.41.4c-.09.15-.13.31-.13.48zm6.29-3.63h-.98v-1.2l1.06-.07.2-1.88h1.34v1.88h1.75v1.27h-1.75v3.28c0 .8.32 1.2.97 1.2.12 0 .24-.01.37-.04.12-.03.24-.07.34-.11l.28 1.19c-.19.06-.4.12-.64.17-.23.05-.49.08-.76.08-.4 0-.74-.06-1.02-.18-.27-.13-.49-.3-.67-.52-.17-.21-.3-.48-.37-.78-.08-.3-.12-.64-.12-1.01zm4.36 2.17c0-.56.09-1.06.27-1.51s.41-.83.71-1.14c.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.37c.08.62.29 1.1.65 1.44.36.33.82.5 1.38.5.3 0 .58-.04.84-.13.25-.09.51-.21.76-.37l.54 1.01c-.32.21-.69.39-1.09.53s-.82.21-1.26.21c-.47 0-.92-.08-1.33-.25-.41-.16-.77-.4-1.08-.7-.3-.31-.54-.69-.72-1.13-.17-.44-.26-.95-.26-1.52zm4.61-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.08.45-.31.29-.5.73-.57 1.3zm3.01 2.23c.31.24.61.43.92.57.3.13.63.2.98.2.38 0 .65-.08.83-.23s.27-.35.27-.6c0-.14-.05-.26-.13-.37-.08-.1-.2-.2-.34-.28-.14-.09-.29-.16-.47-.23l-.53-.22c-.23-.09-.46-.18-.69-.3-.23-.11-.44-.24-.62-.4s-.33-.35-.45-.55c-.12-.21-.18-.46-.18-.75 0-.61.23-1.1.68-1.49.44-.38 1.06-.57 1.83-.57.48 0 .91.08 1.29.25s.71.36.99.57l-.74.98c-.24-.17-.49-.32-.73-.42-.25-.11-.51-.16-.78-.16-.35 0-.6.07-.76.21-.17.15-.25.33-.25.54 0 .14.04.26.12.36s.18.18.31.26c.14.07.29.14.46.21l.54.19c.23.09.47.18.7.29s.44.24.64.4c.19.16.34.35.46.58.11.23.17.5.17.82 0 .3-.06.58-.17.83-.12.26-.29.48-.51.68-.23.19-.51.34-.84.45-.34.11-.72.17-1.15.17-.48 0-.95-.09-1.41-.27-.46-.19-.86-.41-1.2-.68z" fill="#535353"/></g></svg>"/></a></p><div><h3 id="citeas">Cite this article</h3><p>Jain, A., Gosling, J., Liu, S. <i>et al.</i> Wireless electrical–molecular quantum signalling for cancer cell apoptosis.
                    <i>Nat. Nanotechnol.</i>  (2023). https://doi.org/10.1038/s41565-023-01496-y</p><p><a data-test="citation-link" data-track="click" data-track-action="download article citation" data-track-label="link" data-track-external="" rel="nofollow" href="https://citation-needed.springer.com/v2/references/10.1038/s41565-023-01496-y?format=refman&amp;flavour=citation">Download citation<svg width="16" height="16" focusable="false" role="img" aria-hidden="true"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-download"></use></svg></a></p><ul data-test="publication-history"><li><p>Received<span>: </span><span><time datetime="2023-03-26">26 March 2023</time></span></p></li><li><p>Accepted<span>: </span><span><time datetime="2023-08-01">01 August 2023</time></span></p></li><li><p>Published<span>: </span><span><time datetime="2023-09-14">14 September 2023</time></span></p></li><li><p><abbr title="Digital Object Identifier">DOI</abbr><span>: </span><span>https://doi.org/10.1038/s41565-023-01496-y</span></p></li></ul></div></div></div></div></section>
            </div></div>
  </body>
</html>
